
<html lang="en"     class="pb-page"  data-request-id="cf86e817-e9b0-498f-97ea-645f802191e9"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;article:article:10.1021/acs.jmedchem.1c00319;pageGroup:string:Publication Pages;subPage:string:Full Text;issue:issue:10.1021/jmcmar.2021.64.issue-14"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Structure-Guided Design of Conformationally Constrained Cyclohexane Inhibitors of Severe Acute Respiratory Syndrome Coronavirus-2 3CL Protease" /></meta><meta name="dc.Creator" content="Chamandi S.  Dampalla" /></meta><meta name="dc.Creator" content="Yunjeong  Kim" /></meta><meta name="dc.Creator" content="Naemi  Bickmeier" /></meta><meta name="dc.Creator" content="Athri D.  Rathnayake" /></meta><meta name="dc.Creator" content="Harry Nhat  Nguyen" /></meta><meta name="dc.Creator" content="Jian  Zheng" /></meta><meta name="dc.Creator" content="Maithri M.  Kashipathy" /></meta><meta name="dc.Creator" content="Matthew A.  Baird" /></meta><meta name="dc.Creator" content="Kevin P.  Battaile" /></meta><meta name="dc.Creator" content="Scott  Lovell" /></meta><meta name="dc.Creator" content="Stanley  Perlman" /></meta><meta name="dc.Creator" content="Kyeong-Ok  Chang" /></meta><meta name="dc.Creator" content="William C.  Groutas" /></meta><meta name="dc.Description" content="A series of nondeuterated and deuterated dipeptidyl aldehyde and masked aldehyde inhibitors that incorporate in their structure a conformationally constrained cyclohexane moiety was synthesized and..." /></meta><meta name="Description" content="A series of nondeuterated and deuterated dipeptidyl aldehyde and masked aldehyde inhibitors that incorporate in their structure a conformationally constrained cyclohexane moiety was synthesized and..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="July 2, 2021" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.1c00319" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2021 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.1c00319" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.1c00319" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.1c00319" /></link>
        
    
    

<title>Structure-Guided Design of Conformationally Constrained Cyclohexane Inhibitors of Severe Acute Respiratory Syndrome Coronavirus-2 3CL Protease | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.1c00319" /></meta><meta property="og:title" content="Structure-Guided Design of Conformationally Constrained Cyclohexane Inhibitors of Severe Acute Respiratory Syndrome Coronavirus-2 3CL Protease" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00319/20210716/images/large/jm1c00319_0008.jpeg" /></meta><meta property="og:description" content="A series of nondeuterated and deuterated dipeptidyl aldehyde and masked aldehyde inhibitors that incorporate in their structure a conformationally constrained cyclohexane moiety was synthesized and found to potently inhibit severe acute respiratory syndrome coronavirus-2 3CL protease in biochemical and cell-based assays. Several of the inhibitors were also found to be nanomolar inhibitors of Middle East respiratory syndrome coronavirus 3CL protease. The corresponding latent aldehyde bisulfite adducts were found to be equipotent to the precursor aldehydes. High-resolution cocrystal structures confirmed the mechanism of action and illuminated the structural determinants involved in binding. The spatial disposition of the compounds disclosed herein provides an effective means of accessing new chemical space and optimizing pharmacological activity. The cellular permeability of the identified inhibitors and lack of cytotoxicity warrant their advancement as potential therapeutics for COVID-19." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.1c00319"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00319">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.1c00319&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.1c00319&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.1c00319&amp;href=/doi/10.1021/acs.jmedchem.1c00319" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, 64</span><span class="cit-fg-issue">, 14</span><span class="cit-fg-pageRange">, 10047-10058</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/64/14" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.1c00315" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.1c00347" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Structure-Guided Design of Conformationally Constrained Cyclohexane Inhibitors of Severe Acute Respiratory Syndrome Coronavirus-2 3CL Protease</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Chamandi S. Dampalla</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Chamandi S. Dampalla</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, Wichita State University, Wichita, Kansas 67260, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Chamandi+S.++Dampalla">Chamandi S. Dampalla</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-8199-2376" title="Orcid link">https://orcid.org/0000-0002-8199-2376</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yunjeong Kim</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yunjeong Kim</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Diagnostic Medicine & Pathobiology, College of Veterinary Medicine, Kansas State University, Manhattan, Kansas 66506, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yunjeong++Kim">Yunjeong Kim</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Naemi Bickmeier</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Naemi Bickmeier</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Diagnostic Medicine & Pathobiology, College of Veterinary Medicine, Kansas State University, Manhattan, Kansas 66506, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Naemi++Bickmeier">Naemi Bickmeier</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Athri D. Rathnayake</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Athri D. Rathnayake</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, Wichita State University, Wichita, Kansas 67260, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Athri+D.++Rathnayake">Athri D. Rathnayake</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-2588-7624" title="Orcid link">https://orcid.org/0000-0003-2588-7624</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Harry Nhat Nguyen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Harry Nhat Nguyen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, Wichita State University, Wichita, Kansas 67260, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Harry+Nhat++Nguyen">Harry Nhat Nguyen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jian Zheng</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jian Zheng</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Microbiology and Immunology, Carver College of Medicine, University of Iowa, Iowa City, Iowa 52242, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jian++Zheng">Jian Zheng</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Maithri M. Kashipathy</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Maithri M. Kashipathy</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Protein Structure Laboratory, The University of Kansas, Lawrence, Kansas 66047, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Maithri+M.++Kashipathy">Maithri M. Kashipathy</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Matthew A. Baird</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Matthew A. Baird</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, Wichita State University, Wichita, Kansas 67260, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Matthew+A.++Baird">Matthew A. Baird</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Kevin P. Battaile</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kevin P. Battaile</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">NYX, New York Structural Biology Center, Upton, New York 11973, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kevin+P.++Battaile">Kevin P. Battaile</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-0833-3259" title="Orcid link">https://orcid.org/0000-0003-0833-3259</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Scott Lovell</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Scott Lovell</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Protein Structure Laboratory, The University of Kansas, Lawrence, Kansas 66047, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Scott++Lovell">Scott Lovell</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Stanley Perlman</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Stanley Perlman</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Microbiology and Immunology, Carver College of Medicine, University of Iowa, Iowa City, Iowa 52242, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Stanley++Perlman">Stanley Perlman</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Kyeong-Ok Chang</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kyeong-Ok Chang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Diagnostic Medicine & Pathobiology, College of Veterinary Medicine, Kansas State University, Manhattan, Kansas 66506, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#88eae1e4e4a6effae7fdfce9fbc8ffe1ebe0e1fce9a6edecfd"><span class="__cf_email__" data-cfemail="98faf1f4f4b6ffeaf7edecf9ebd8eff1fbf0f1ecf9b6fdfced">[email protected]</span></a>. Phone: (316) 978 7374.</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kyeong-Ok++Chang">Kyeong-Ok Chang</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">William C. Groutas</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">William C. Groutas</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, Wichita State University, Wichita, Kansas 67260, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#d7bcb4bfb6b9b097a1b2a3f9bca4a2f9b2b3a2"><span class="__cf_email__" data-cfemail="412a2229202f26013724356f2a32346f242534">[email protected]</span></a>. Phone: (785) 532 3849.</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=William+C.++Groutas">William C. Groutas</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0001-5248-7912" title="Orcid link">https://orcid.org/0000-0001-5248-7912</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.1c00319&amp;href=/doi/10.1021%2Facs.jmedchem.1c00319" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, 64</span><span class="cit-issue">, 14</span><span class="cit-pageRange">, 10047–10058</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">July 2, 2021</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>20 February 2021</li><li><span class="item_label"><b>Published</b> online</span>2 July 2021</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 22 July 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.1c00319" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00319</a></div><div class="article_header-article-copyright"><strong>Copyright © 2021 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D10047%26pageCount%3D12%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DChamandi%2BS.%2BDampalla%252C%2BYunjeong%2BKim%252C%2BNaemi%2BBickmeier%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D64%26issueNum%3D14%26contentID%3Dacs.jmedchem.1c00319%26title%3DStructure-Guided%2BDesign%2Bof%2BConformationally%2BConstrained%2BCyclohexane%2BInhibitors%2Bof%2BSevere%2BAcute%2BRespiratory%2BSyndrome%2BCoronavirus-2%2B3CL%2BProtease%26numPages%3D12%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D10058%26publicationDate%3DJuly%2B2021">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.1c00319"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1252</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val">-</div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.1c00319" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Structure-Guided Design of Conformationally Constrained Cyclohexane Inhibitors of Severe Acute Respiratory Syndrome Coronavirus-2 3CL Protease&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Chamandi&quot;,&quot;last_name&quot;:&quot;S. Dampalla&quot;},{&quot;first_name&quot;:&quot;Yunjeong&quot;,&quot;last_name&quot;:&quot;Kim&quot;},{&quot;first_name&quot;:&quot;Naemi&quot;,&quot;last_name&quot;:&quot;Bickmeier&quot;},{&quot;first_name&quot;:&quot;Athri&quot;,&quot;last_name&quot;:&quot;D. Rathnayake&quot;},{&quot;first_name&quot;:&quot;Harry&quot;,&quot;last_name&quot;:&quot;Nhat Nguyen&quot;},{&quot;first_name&quot;:&quot;Jian&quot;,&quot;last_name&quot;:&quot;Zheng&quot;},{&quot;first_name&quot;:&quot;Maithri&quot;,&quot;last_name&quot;:&quot;M. Kashipathy&quot;},{&quot;first_name&quot;:&quot;Matthew&quot;,&quot;last_name&quot;:&quot;A. Baird&quot;},{&quot;first_name&quot;:&quot;Kevin&quot;,&quot;last_name&quot;:&quot;P. Battaile&quot;},{&quot;first_name&quot;:&quot;Scott&quot;,&quot;last_name&quot;:&quot;Lovell&quot;},{&quot;first_name&quot;:&quot;Stanley&quot;,&quot;last_name&quot;:&quot;Perlman&quot;},{&quot;first_name&quot;:&quot;Kyeong-Ok&quot;,&quot;last_name&quot;:&quot;Chang&quot;},{&quot;first_name&quot;:&quot;William&quot;,&quot;last_name&quot;:&quot;C. Groutas&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2021&quot;,&quot;month&quot;:&quot;07&quot;,&quot;day&quot;:&quot;02&quot;,&quot;issue&quot;:&quot;14&quot;,&quot;volume&quot;:&quot;64&quot;,&quot;pages&quot;:&quot;10047-10058&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.1c00319&quot;},&quot;abstract&quot;:&quot;A series of nondeuterated and deuterated dipeptidyl aldehyde and masked aldehyde inhibitors that incorporate in their structure a conformationally constrained cyclohexane moiety was synthesized and found to potently inhibit severe acute respiratory syndrome coronavirus-2 3CL protease in biochemical and cell-based assays. Several of the inhibitors were also found to be nanomolar inhibitors of Middle East respiratory syndrome coronavirus 3CL protease. The corresponding latent aldehyde bisulfite adducts were found to be equipotent to the precursor aldehydes. High-resolution cocrystal structures confirmed the mechanism of action and illuminated the structural determinants involved in binding. The spatial disposition of the compounds disclosed herein provides an effective means of accessing new chemical space and optimizing pharmacological activity. The cellular permeability of the identified inhibitors and lack of cytotoxicity warrant their advancement as potential therapeutics for COVID-19.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00319&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00319" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00319&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00319" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00319&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00319" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.1c00319&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00319&amp;href=/doi/10.1021/acs.jmedchem.1c00319" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.1c00319" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.1c00319" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (7 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00319&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.1c00319%26sid%3Dliteratum%253Aachs%26pmid%3D34213885%26genre%3Darticle%26aulast%3DDampalla%26date%3D2021%26atitle%3DStructure-Guided%2BDesign%2Bof%2BConformationally%2BConstrained%2BCyclohexane%2BInhibitors%2Bof%2BSevere%2BAcute%2BRespiratory%2BSyndrome%2BCoronavirus-2%2B3CL%2BProtease%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D64%26issue%3D14%26spage%3D10047%26epage%3D10058%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292644" title="SARS-CoV-2">SARS-CoV-2</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291650" title="Alcohols">Alcohols</a>,</li><li><a href="/action/doSearch?ConceptID=292236" title="Aldehydes">Aldehydes</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/64/14" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00319/20210716/images/medium/jm1c00319_0008.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00319/20210716/images/large/jm1c00319_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00319&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">A series of nondeuterated and deuterated dipeptidyl aldehyde and masked aldehyde inhibitors that incorporate in their structure a conformationally constrained cyclohexane moiety was synthesized and found to potently inhibit severe acute respiratory syndrome coronavirus-2 3CL protease in biochemical and cell-based assays. Several of the inhibitors were also found to be nanomolar inhibitors of Middle East respiratory syndrome coronavirus 3CL protease. The corresponding latent aldehyde bisulfite adducts were found to be equipotent to the precursor aldehydes. High-resolution cocrystal structures confirmed the mechanism of action and illuminated the structural determinants involved in binding. The spatial disposition of the compounds disclosed herein provides an effective means of accessing new chemical space and optimizing pharmacological activity. The cellular permeability of the identified inhibitors and lack of cytotoxicity warrant their advancement as potential therapeutics for COVID-19.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div class="extra-info-sec articleNote"><h4><span></span>  Note</h4><p class="first last">This article is made available via the <a href="/page/vi/chemistry_coronavirus_research" class="ext-link">ACS COVID-19 subset</a> for unrestricted RESEARCH re-use and analyses in any form or by any means with acknowledgement of the original source. These permissions are granted for the duration of the World Health Organization (WHO) declaration of COVID-19 as a global pandemic.</p></div><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_87564" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_87564" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Coronaviruses are enveloped, positive-sense, single-stranded RNA viruses that belong to the family <i>Coronaviridae</i>.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Among human coronaviruses, several strains (229E, NL63, OC43, and KHU1) are the cause of mild upper respiratory infections; however, a few coronaviruses have emerged from animals that cause severe respiratory disease, including severe acute respiratory syndrome coronavirus (SARS-CoV), Middle East respiratory syndrome coronavirus (MERS-CoV), and SARS-CoV-2.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> Of particular concern is SARS-CoV-2, the highly pathogenic causative agent of COVID-19 which is associated with high infectivity and is a significant threat to public health worldwide.<a onclick="showRef(event, 'ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref3 ref4">(3,4)</a> The problem is further compounded by the current lack of effective vaccines or small molecule therapeutics for the treatment of SARS-CoV-2 infections, underscoring the urgent and dire need for the development of prophylactic and therapeutic countermeasures to combat infections by pathogenic coronaviruses.<a onclick="showRef(event, 'ref5 ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref5 ref6 ref7">(5−7)</a></div><div class="NLM_p">The SARS-CoV-2 genome is large(∼30 kb) and similar to the genomes of SARS-CoV and MERS-CoV (∼80 and ∼50% sequence identity, respectively). It contains two open reading frames (ORF1a and ORF1b) and encodes multiple structural and nonstructural proteins.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Translation of the genomic mRNA of ORF1a yields a polyprotein (pp1a), while a second polyprotein (pp1ab) is the product of a ribosomal frameshift that joins ORF1a together with ORF1b. The two polyproteins are processed by a 3C-like protease (3CLpro, also referred to as main protease, M<sup>pro</sup>) (11 cleavage sites) and a papain-like cysteine protease (PLpro), resulting in 16 mature nonstructural proteins which are involved in the replication–transcription complex. The two proteases are essential for viral replication, making them attractive targets for therapeutic intervention.<a onclick="showRef(event, 'ref8 ref9 ref10 ref11 ref12 ref13 ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref8 ref9 ref10 ref11 ref12 ref13 ref14 ref15">(8−15)</a></div><div class="NLM_p">SARS-CoV-2 3CLpro is a homodimer with a catalytic Cys–His dyad (Cys145–His41) and an extended binding cleft. Substrate specificity profiling studies<a onclick="showRef(event, 'ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref12 ref13">(12,13)</a> have shown that the protease displays a strong preference for a −Y–Z–Leu–Gln–X sequence, where X is a small amino acid, Y is a hydrophobic amino acid, and Z is solvent-exposed and fairly diverse (V/T/K), corresponding to the amino acid residues −P<sub>4</sub>–P<sub>3</sub>–P<sub>2</sub>–P<sub>1</sub>–P<sub>1</sub>′-of a substrate or inhibitor.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> Cleavage is at the P<sub>1</sub>–P<sub>1</sub>′ scissile bond. The 3D structure of SARS-CoV-2 3CLpro is similar to that of SARS-CoV 3CLpro; however, the S<sub>2</sub> subsite of SARS-CoV-2 3CLpro displays considerable plasticity and can accommodate natural and unnatural amino acids with smaller side chains.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> Similarly, the active-site topography of MERS-CoV 3CLpro closely resembles that of SARS-CoV-2 3CLpro.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> High-resolution crystal structures with bound inhibitors have been determined, enabling the use of structure-guided approaches in the design of inhibitors. In continuing our foray in this area,<a onclick="showRef(event, 'ref17 ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref17 ref18 ref19">(17−19)</a> we report herein the results of preliminary studies related to the inhibition of SARS-CoV-2 protease by a series of inhibitors (I) (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>) that incorporate in their structure a conformationally constrained cyclohexane moiety envisaged to exploit new chemical space and to optimally engage in favorable binding interactions with the active site of the protease. Furthermore, several deuterated variants of the inhibitor were also synthesized to potentially improve the PK properties and ancillary parameters of the inhibitors.<a onclick="showRef(event, 'ref20 ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref20 ref21 ref22">(20−22)</a></div><figure id="sch1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00319/20210716/images/medium/jm1c00319_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00319/20210716/images/large/jm1c00319_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Inhibitors <b>2(a–o)</b> and <b>3(a–o)</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00319/20210716/images/large/jm1c00319_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00319&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_63656" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_63656" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Inhibitor Design Rationale</h3><div class="NLM_p last">The design of inhibitor (I) (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>) included the use of a P<sub>1</sub> glutamine surrogate residue and a P<sub>2</sub> Leu residue as recognition elements congruent with the substrate specificity of the protease,<a onclick="showRef(event, 'ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref12 ref13">(12,13)</a> as well as an aldehyde warhead or a latent aldehyde bisulfite adduct. The design of the inhibitor (I) was further abetted by insights gained from examining the available X-ray crystal structures of the protease with inhibitors<a onclick="showRef(event, 'ref12 ref13 ref17'); return false;" href="javascript:void(0);" class="ref ref12 ref13 ref17">(12,13,17)</a> and the results of recent studies with cyclohexyl-derived inhibitors with demonstrated efficacy in a mouse model of MERS-CoV-2 infection and potent inhibition against SARS-CoV-2 3CL protease.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a></div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Chemistry</h3><div class="NLM_p">The synthesis of inhibitors <b>2(a–o)</b> and <b>3(a–o)</b> was readily accomplished by activating the precursor primary or secondary alcohol inputs (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>) with <i>N</i>,<i>N</i>′-disuccinimidyl carbonate<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> and coupling the mixed carbonate with the readily accessible Leu–Gln surrogate amino alcohol to yield alcohol product <b>1</b> which was oxidized with Dess-Martin periodinane to generate the corresponding aldehyde (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>, Z = CHO <b>2</b>). The aldehydes were subsequently converted to the corresponding bisulfite adducts (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>, Z = CH(OH)SO<sub>3</sub>Na <b>3</b>).<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a></div><figure id="sch2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00319/20210716/images/medium/jm1c00319_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00319/20210716/images/large/jm1c00319_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Alcohol Inputs <b>(a–o)</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00319/20210716/images/large/jm1c00319_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00319&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Biochemical Studies</h3><div class="NLM_p">The inhibitory activity of compounds <b>2–3 (a–o)</b> against SARS-CoV-2 3CL protease<a onclick="showRef(event, 'ref17 ref25'); return false;" href="javascript:void(0);" class="ref ref17 ref25">(17,25)</a> and their activity in a cell-based system were determined as described in the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>. The IC<sub>50</sub> values, EC<sub>50</sub> values for two representative inhibitors (<b>2a/3a</b>), and the CC<sub>50</sub> values in Huh-7, CRFK, or CCL1 cells<a onclick="showRef(event, 'ref17 ref25'); return false;" href="javascript:void(0);" class="ref ref17 ref25">(17,25)</a> are summarized in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, and they are the average of at least two determinations. The inhibitory activity of compounds <b>2a/3a</b>, <b>2f/3f</b>, and <b>2k/3k</b> against MERS-CoV 3CL protease was also determined as described previously,<a onclick="showRef(event, 'ref17 ref18 ref25'); return false;" href="javascript:void(0);" class="ref ref17 ref18 ref25">(17,18,25)</a> and the IC<sub>50</sub> values are listed in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. IC<sub>50</sub> and CC<sub>50</sub> Values of SARS-CoV-2 3CLpro Inhibitors <b>2–3 (a–o)</b></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="±" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="±" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center" char="±">IC<sub>50</sub> (μM)</th><th class="colsep0 rowsep0" align="center">CC<sub>50</sub> (μM)</th><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center" char="±">IC50 (μM)</th><th class="colsep0 rowsep0" align="center">CC<sub>50</sub> (μM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2a</b></td><td class="colsep0 rowsep0" align="char" char="±">0.18 ± 0.03<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left"><b>3h</b></td><td class="colsep0 rowsep0" align="char" char="±">0.37 ± 0.03</td><td class="colsep0 rowsep0" align="left">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3a</b></td><td class="colsep0 rowsep0" align="char" char="±">0.17 ± 0.02<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left"><b>2i</b></td><td class="colsep0 rowsep0" align="char" char="±">0.27 ± 0.02</td><td class="colsep0 rowsep0" align="left">21 ± 1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2b</b></td><td class="colsep0 rowsep0" align="char" char="±">0.29 ± 0.05</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left"><b>3i</b></td><td class="colsep0 rowsep0" align="char" char="±">0.30 ± 0.02</td><td class="colsep0 rowsep0" align="left">21 ± 3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3b</b></td><td class="colsep0 rowsep0" align="char" char="±">0.29 ± 0.01</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left"><b>2j</b></td><td class="colsep0 rowsep0" align="char" char="±">0.31 ± 0.03</td><td class="colsep0 rowsep0" align="left">20 ± 3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2c</b></td><td class="colsep0 rowsep0" align="char" char="±">0.31 ± 0.06</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left"><b>3j</b></td><td class="colsep0 rowsep0" align="char" char="±">0.31 ± 0.06</td><td class="colsep0 rowsep0" align="left">23 ± 1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3c</b></td><td class="colsep0 rowsep0" align="char" char="±">0.26 ± 0.05</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left"><b>2k</b></td><td class="colsep0 rowsep0" align="char" char="±">0.18 ± 0.04</td><td class="colsep0 rowsep0" align="left">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2d</b></td><td class="colsep0 rowsep0" align="char" char="±">0.28 ± 0.08</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left"><b>3k</b></td><td class="colsep0 rowsep0" align="char" char="±">0.15 ± 0.04</td><td class="colsep0 rowsep0" align="left">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3d</b></td><td class="colsep0 rowsep0" align="char" char="±">0.30 ± 0.03</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left"><b>2l</b></td><td class="colsep0 rowsep0" align="char" char="±">0.29 ± 0.04</td><td class="colsep0 rowsep0" align="left">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2e</b></td><td class="colsep0 rowsep0" align="char" char="±">0.26 ± 0.05</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left"><b>3l</b></td><td class="colsep0 rowsep0" align="char" char="±">0.27 ± 0.04</td><td class="colsep0 rowsep0" align="left">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3e</b></td><td class="colsep0 rowsep0" align="char" char="±">0.28 ± 0.03</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left"><b>2m</b></td><td class="colsep0 rowsep0" align="char" char="±">0.27 ± 0.02</td><td class="colsep0 rowsep0" align="left">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2f</b></td><td class="colsep0 rowsep0" align="char" char="±">0.14 ± 0.02</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left"><b>3m</b></td><td class="colsep0 rowsep0" align="char" char="±">0.25 ± 0.04</td><td class="colsep0 rowsep0" align="left">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3f</b></td><td class="colsep0 rowsep0" align="char" char="±">0.10 ± 0.01</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left"><b>2n</b></td><td class="colsep0 rowsep0" align="char" char="±">0.82 ± 0.19</td><td class="colsep0 rowsep0" align="left">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2g</b></td><td class="colsep0 rowsep0" align="char" char="±">1.90 ± 0.14</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left"><b>3n</b></td><td class="colsep0 rowsep0" align="char" char="±">1.03 ± 0.24</td><td class="colsep0 rowsep0" align="left">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3g</b></td><td class="colsep0 rowsep0" align="char" char="±">1.71 ± 0.16</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left"><b>2o</b></td><td class="colsep0 rowsep0" align="char" char="±">0.74 ± 0.26</td><td class="colsep0 rowsep0" align="left">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2h</b></td><td class="colsep0 rowsep0" align="char" char="±">0.39 ± 0.02</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left"><b>3o</b></td><td class="colsep0 rowsep0" align="char" char="±">0.78 ± 0.18</td><td class="colsep0 rowsep0" align="left">>100</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">The EC<sub>50</sub> values for inhibitors <b>2a</b> and <b>3a</b> against SARS-CoV-2 in Vero E6 cells were 0.035 ± 0.001 and 0.032 ± 0.001 μM, respectively.</p></div></div></div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. IC<sub>50</sub> Values of MERS-CoV 3CLpro Inhibitors <b>2a/3a</b>, <b>2f/3f</b>, and <b>2k/3k</b></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="±" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center" char="±">IC<sub>50</sub> (μM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2a</b></td><td class="colsep0 rowsep0" align="char" char="±">0.052 ± 0.001</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3a</b></td><td class="colsep0 rowsep0" align="char" char="±">0.049 ± 0.002</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2f</b></td><td class="colsep0 rowsep0" align="char" char="±">0.063 ± 0.003</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3f</b></td><td class="colsep0 rowsep0" align="char" char="±">0.058 ± 0.002</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2k</b></td><td class="colsep0 rowsep0" align="char" char="±">0.055 ± 0.002</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3k</b></td><td class="colsep0 rowsep0" align="char" char="±">0.053 ± 0.003</td></tr></tbody></table></div></div><div class="NLM_p">It is clearly evident from the results shown in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> that the synthesized compounds display high potency in biochemical assays, with most IC<sub>50</sub> values in the sub-micromolar range. Furthermore, the inhibitors were found to be devoid of cytotoxicity and the safety index (SI), defined as the CC<sub>50</sub>/IC<sub>50</sub> ratio, ranged between ∼78 and 1110. The potency of deuterated variants <b>2b/3b</b> decreased ∼1.6-fold (aldehydes) and ∼1.7-fold (bisulfite adducts) as compared to the respective nondeuterated compounds <b>2a/3a</b> and remained essentially the same in the case of nondeuterated <b>2n/3n</b> and deuterated <b>2o/3o</b> inhibitors, respectively. A change in geometry from a cyclohexene (<b>2e/3e</b>) to a cyclohexane (<b>2f/3f</b>) resulted in a two- to threefold increase in potency. The approximately fivefold decrease in potency of compounds <b>2n/3n</b> compared to <b>2k/3k</b> presumably reflects the inimical effect on potency of the 3° hydroxyl group. Importantly, the EC<sub>50</sub> values of two representative inhibitors (<b>2a/3a</b>) against SARS-CoV-2 in Vero E6 cells were found to be ∼4.6-fold lower (EC<sub>50</sub> 0.035 and 0.032 μM, respectively) than the corresponding IC<sub>50</sub> values, and the selectivity indices of compounds <b>2a/3a</b> were very high (2857 and 3125, respectively). The significance of these findings was further augmented by the notable inhibition of MERS-CoV 3CL protease by a select number of inhibitors (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, compounds <b>2a/3a</b>, <b>2f/3f</b>, and <b>2k/3k</b>), demonstrating the broad spectrum of antiviral activity displayed by this series of compounds.</div><div class="NLM_p last">Emergence of viral resistance to antiviral drugs is a major concern. We previously reported that GC376 has a high barrier to resistance to feline infectious peritonitis virus (FIPV) in cell culture and naturally infected animals with long-term treatment.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> We also examined several compounds similar to the series in this report for emergence of viral resistance by serial passaging FIPV in the presence of each compound in cell culture. The EC<sub>50</sub> values of the compounds did not increase at up to 10 passage number, and the 3CLpro of viruses passaged with each compound has the same sequence as mock-passaged viruses. These results suggest that this series of compounds have a high barrier to resistance.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> X-ray Crystallography Studies</h3><div class="NLM_p">In order to elucidate the mechanism of action of the inhibitors and identify the structural determinants associated with the binding of inhibitors to the active site of SARS-CoV-2 3CL protease, high-resolution cocrystal structures were determined for inhibitors <b>2a</b>, <b>3b</b>, <b>2f</b>, <b>2k</b>, <b>3c</b>, <b>3d</b> and <b>3e</b>. The structure of SARS-CoV-2 3CLpro in complex with compound <b>2a</b> contained a prominent difference in electron density consistent with the inhibitor covalently bound to Cys145 in each subunit (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A,B). The electron density was consistent with the inhibitor aldehyde carbon covalently bound to the Sγ atom of the catalytic Cys145 residue and the formation of a tetrahedral hemithioacetal, confirming the mechanism of action. Both the R- and S-enantiomers were observed at the newly formed stereocenter, and each enantiomer was modeled with 0.5 occupancy and was observed for all structures described here. Interestingly, <b>2a</b> adopts two conformations in which the bicyclic ring is projected away from the S<sub>4</sub> subsite in subunit A and is positioned in the S<sub>4</sub> subsite in subunit B. The inhibitor engages in multiple favorable binding interactions with the enzyme, including direct hydrogen bond interactions with His163 and Glu166 (γ-lactam C═O and N–H, respectively), His41, Phe140, Ser144, His164, and Gln189 (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C,D). The isobutyl side chain of Leu is ensconced in the hydrophobic S<sub>2</sub> pocket, and the γ-lactam ring of the P<sub>1</sub> Gln surrogate is nestled in the S<sub>1</sub> subsite forming hydrogen bonds with His163 and Glu166. In addition, the lipophilic bicyclic ring in subunit A is directed toward the surface, whereas in subunit B, it is anchored in the vicinity of the hydrophobic S<sub>4</sub> pocket that is lined by Ala191, Leu197, and Pro168 (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>E,F). It should be noted that the 11 sites in the pp1a and pp1ab polyproteins cleaved by the protease are all characterized by the presence of a P<sub>1</sub> Gln residue, which is conserved in all known coronavirus 3CLpro cleavage sites. Interestingly, the deuterated analogue <b>3b</b> adopts the same binding mode and superimposes nearly identical to <b>2a</b>, as shown in <a href="/doi/suppl/10.1021/acs.jmedchem.1c00319/suppl_file/jm1c00319_si_001.pdf" class="ext-link">Figure S1</a>. The root mean square deviation (RMSD) between the Cα atoms of <b>2a</b> and <b>3b</b> was 0.27 Å for 594 residues aligned.</div><figure id="fig1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00319/20210716/images/medium/jm1c00319_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00319/20210716/images/large/jm1c00319_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Binding mode of <b>2a</b> (gray) with SARS-CoV-2 3CLpro associated with subunit A (A,C) and subunit B (B,D). Fo–Fc polder omit map (green mesh) contoured at 3σ (A,B). Hydrogen bond interactions (dashed lines) (C,D). Surface representations showing the orientation of <b>2a</b> in subunit A (E) and subunit B (F) near the S<sub>4</sub> subsite of SARS-CoV-2 3CLpro with neighboring residues colored yellow (nonpolar), cyan (polar), and white (weakly polar).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00319/20210716/images/large/jm1c00319_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00319&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Likewise, the structure of <b>3c</b> shows similar binding mode properties to those observed for <b>2a</b> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A,C,E). The inhibitor engages in multiple favorable binding interactions with the enzyme, including direct hydrogen bond interactions with His163 and Glu166 (γ-lactam C═O and N–H, respectively), His41, Ser144, His164, and Gln189. The bicyclic ring is oriented within the hydrophobic S<sub>4</sub> pocket, in both subunits. Likewise, the structure of SARS-CoV-2 3CL protease with deuterated inhibitor <b>3d</b> adopts a very similar binding mode (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>C,D,F). The main difference is that the electron density for <b>3c</b> is the most consistent with an axial conformation of the carbon atom attached to the bicyclic ring, whereas <b>3d</b> appears to adopt an equatorial orientation (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00319/suppl_file/jm1c00319_si_001.pdf" class="ext-link">Figure S2</a>). The structures are very similar overall, and the superposition yielded an RMSD between the Cα atoms of 0.25 Å for 596 residues aligned.</div><figure id="fig2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00319/20210716/images/medium/jm1c00319_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00319/20210716/images/large/jm1c00319_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Binding mode of <b>3c</b> (A,B,E) and its deuterated analogue <b>3d</b> (C,D,F) with SARS-CoV-2 3CLpro. Fo–Fc polder omit map (green mesh) contoured at 3σ (A,B). Hydrogen bond interactions (dashed lines) (C,D). surface representation showing the orientation of <b>3c</b> (E) and <b>3d</b> (F) near the S<sub>4</sub> subsite of SARS-CoV-2 3CLpro with neighboring residues colored yellow (nonpolar), cyan (polar), and white (weakly polar).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00319/20210716/images/large/jm1c00319_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00319&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Similarly, inhibitors <b>2f</b>, <b>2k</b>, and <b>3e</b> in complex with SARS-CoV-2 were found to adopt similar binding modes in the active site of the protease, as shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.1c00319/suppl_file/jm1c00319_si_001.pdf" class="ext-link">S3</a>. Collectively, the bicyclic rings of the inhibitors span a relatively small region in the active site and cover a space of approximately 6.3 Å (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). As such, these cocrystal structures provide valuable insights for further structure-guided multiparameter optimization.</div><figure id="fig3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00319/20210716/images/medium/jm1c00319_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00319/20210716/images/large/jm1c00319_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Binding mode of <b>2f</b> (A,B), <b>2k</b> (C,D), and <b>3e</b> (E,F) with SARS-CoV-2 3CLpro. Fo–Fc polder omit map (green mesh) contoured at 3σ (A,C,E). Hydrogen bond interactions (dashed lines) (B,D,F).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00319/20210716/images/large/jm1c00319_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00319&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00319/20210716/images/medium/jm1c00319_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00319/20210716/images/large/jm1c00319_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Superposition of all seven inhibitor-bound structures <b>2a</b> (red), <b>3b</b> (blue), <b>2f</b> (cyan), <b>2k</b> (yellow), <b>3c</b> (coral), <b>3d</b> (magenta), and <b>3e</b> (green). The bicyclic rings cover a space of approximately 6.3 Å in the S<sub>4</sub> subsite.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00319/20210716/images/large/jm1c00319_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00319&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i12">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_22463" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_22463" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">Given the major clinical importance associated with the SARS-CoV-2 pandemic and the current paucity of effective countermeasures, the results of the studies described herein can serve as a launching pad for conducting further preclinical studies. Most of the compounds exhibited high potency in biochemical assays and, for two of the compounds tested, in cellular assays. Furthermore, members of this series were also found to potently inhibit MERS-CoV 3CL protease, suggesting that the compounds can be developed into broad-spectrum antivirals. Since there are no known human proteases that have a primary substrate specificity P<sub>1</sub> residue, that is, Gln, these inhibitors could also display high selectivity and diminished off-target effects. Furthermore, the utilization of an aldehyde warhead, or a latent aldehyde functionality that can rapidly generate the aldehyde <i>in vivo</i>, in the design of transition state inhibitors is advantageous for several reasons, including rapid engagement with the target, leading to the reversible formation of a covalent adduct. The high reactivity of aldehydes is generally viewed as a toxicity alert; however, the safety indices for most of the compounds reported herein were found to be high. Indeed, a number of pharmaceuticals that incorporate in their structure an aldehyde functionality are currently in clinical use and, furthermore, toxicity arising from the presence of the aldehyde is context-specific,<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> as is presumably the case here. Finally, the present study also sought to exploit the kinetic isotope effect associated with the H/D bioisosteric replacement<a onclick="showRef(event, 'ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref27 ref28">(27,28)</a> in order to dampen oxidative metabolism at the −CH<sub>2</sub>O– metabolic soft spot in the inhibitors,<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> as well as to reduce toxicity. Thus, the availability of equipotent deuterated analogues that display improved PK characteristics enhances further the significance of the results reported herein. Evaluation of a select number of inhibitors in a mouse model of SARS-CoV-2 infection is in progress, and the results will be reported in due course. In conclusion, a series of potent transition state inhibitors of SARS-CoV-2 3CL protease that incorporate in their structures a conformationally constrained cyclohexyl moiety is reported.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i13">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_07614" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_07614" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> General</h3><div class="NLM_p last">Reagents and dry solvents were purchased from various chemical suppliers (Sigma-Aldrich, Acros Organics, Chem-Impex, TCI America, Oakwood chemical, APExBIO, Cambridge Isotopes, Alpha Aesar, Fisher, and Advanced Chemblocks) and were used as obtained. Silica gel (230–450 mesh) used for flash chromatography was purchased from Sorbent Technologies (Atlanta, GA). Thin layer chromatography was performed using Analtech silica gel plates. Visualization was accomplished using UV light and/or iodine. Nuclear magnetic resonance (NMR) spectra were recorded in CDCl<sub>3</sub> or dimethyl sulfoxide (DMSO)-<i>d</i><sub>6</sub> using a Varian XL-400 spectrometer. High-resolution mass spectrometry (HRMS) was performed at the Wichita State University mass spectrometry laboratory using an Orbitrap Velos Pro mass spectrometer (ThermoFisher, Waltham, MA) equipped with an electrospray ion source. The purity of all final compounds was >95% as evidenced by NMR analysis.</div></div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Synthesis of Compounds</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_01804" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_01804" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Preparation of Compounds <b>1(a–o)</b></h3><div id="sec5_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i17" class="anchor-spacer"></div><h4 class="article-section__title" id="_i17"> General Procedure</h4><div class="NLM_p">To a solution of alcohol (1 equiv) (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>) in anhydrous acetonitrile (10 mL/g alcohol) was added DSC (1.2 equiv) and TEA (3.0 equiv), and the reaction mixture was stirred for 4 h at room temperature. The solvent was removed <i>in vacuo</i>, and the residue was dissolved in ethyl acetate (40 mL/g alcohol). The organic phase was washed with saturated aqueous NaHCO<sub>3</sub> (2 × 20 mL/g alcohol), followed by brine (20 mL/g alcohol). The organic layers were combined and dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated <i>in vacuo</i> to yield the mixed carbonate which was used in the next step without further purification.</div><div class="NLM_p last">To a solution of Leu–Gln surrogate amino alcohol (1.0 equiv) in dry methylene chloride (10 mL/g amino alcohol) was added TEA (1.5 equiv), and the reaction mixture was stirred for 20 min at room temperature (solution 1). In a separate flask, the mixed carbonate was dissolved in dry methylene chloride (10 mL/g carbonate) (solution 2). Solution 1 was added to solution 2, and the reaction mixture was stirred for 3 h at room temperature. Methylene chloride was added to the organic phase (40 mL/g carbonate) and then washed with saturated aqueous NaHCO<sub>3</sub> (2 × 20 mL/g alcohol), followed by brine (20 mL/g alcohol). The organic phase was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated <i>in vacuo</i>. The resultant crude product was purified by flash chromatography (hexane/ethyl acetate) to yield dipeptidyl alcohol <b>1</b> as a white solid.</div></div><div id="sec5_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18"> ((1<i>R</i>,5<i>S</i>)-Bicyclo[3.3.1]nonan-3-yl)methyl((<i>S</i>)-1-(((<i>S</i>)-1-hydroxy-3-((<i>S</i>)-2-oxopyrrolidin-3-yl)propan-2-yl)amino)-4-methyl-1-oxopentan-2-yl)carbamate (<b>1a</b>)</h4><div class="NLM_p last">Yield (35%); <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.75 (d, <i>J</i> = 7.2 Hz, 1H), 6.24 (s, 1H), 5.31 (d, <i>J</i> = 9.1 Hz, 1H), 4.29–4.10 (m, 1H), 4.10–3.93 (m, 1H), 3.89 (d, <i>J</i> = 6.3 Hz, 1H), 3.71–3.54 (m, 2H), 3.48 (d, <i>J</i> = 2.0 Hz, 1H), 3.39–3.27 (m, 2H), 2.55–2.25 (m, 3H), 2.03 (d, <i>J</i> = 11.9 Hz, 3H), 1.96–1.77 (m, 4H), 1.77–1.57 (m, 4H), 1.57–1.44 (m, 1H), 1.37 (d, <i>J</i> = 9.5 Hz, 4H), 1.27–1.15 (m, 1H), 1.09 (dd, <i>J</i> = 12.8, 2.6 Hz, 1H), 0.95 (d, <i>J</i> = 6.3 Hz, 6H), 0.91–0.80 (m, 2H).</div></div><div id="sec5_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> ((1<i>R</i>,3<i>S</i>,5<i>S</i>)-Bicyclo[3.3.1]nonan-3-yl)methyl-<i>d</i><sub>2</sub>((<i>S</i>)-1-(((<i>S</i>)-1-hydroxy-3-((<i>S</i>)-2-oxopyrrolidin-3-yl)propan-2-yl)amino)-4-methyl-1-oxopentan-2-yl)carbamate (<b>1b</b>)</h4><div class="NLM_p last">Yield (36%); <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.77 (d, <i>J</i> = 7.1 Hz, 1H), 6.25 (s, 1H), 5.30 (s, 1H), 4.27–3.93 (m, 2H), 3.69–3.55 (m, 2H), 3.39–3.32 (m, 2H), 2.53–2.35 (m, 2H), 2.09–1.96 (m, 4H), 1.96–1.77 (m, 3H), 1.77–1.58 (m, 5H), 1.58–1.47 (m, 1H), 1.45–1.30 (m, 5H), 1.09 (d, <i>J</i> = 12.8 Hz, 1H), 0.95 (d, <i>J</i> = 6.4 Hz, 6H), 0.88 (d, <i>J</i> = 13.1 Hz, 2H).</div></div><div id="sec5_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> ((1<i>S</i>,5<i>R</i>)-Bicyclo[3.3.1]non-6-en-3-yl)methyl((<i>S</i>)-1-(((<i>S</i>)-1-hydroxy-3-((<i>S</i>)-2-oxopyrrolidin-3-yl)propan-2-yl)amino)-4-methyl-1-oxopentan-2-yl)carbamate (<b>1c</b>)</h4><div class="NLM_p last">Yield (35%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.74 (s, 1H), 5.98 (s, 1H), 5.85 (t, <i>J</i> = 8.0 Hz, 1H), 5.58 (t, <i>J</i> = 9.7 Hz, 1H), 5.19 (s, 1H), 4.21–4.16 (m, 1H), 4.07 (d, <i>J</i> = 2.5 Hz, 2H), 4.02–3.95 (m, 1H), 3.68–3.56 (m, 2H), 3.38–3.32 (m, 2H), 2.50–2.36 (m, 2H), 2.36–2.31 (m, 1H), 2.31–2.25 (m, 2H), 2.17–2.09 (m, 1H), 2.03–1.95 (m, 1H), 1.95–1.87 (m, 1H), 1.87–1.81 (m, 2H), 1.81–1.73 (m, 2H), 1.73–1.59 (m, 3H), 1.56–1.46 (m, 2H), 1.46–1.35 (m, 2H), 0.95 (d, <i>J</i> = 6.5 Hz, 6H).</div></div><div id="sec5_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> ((1<i>S</i>,3<i>S</i>,5<i>R</i>)-Bicyclo[3.3.1]non-6-en-3-yl)methyl-<i>d</i><sub>2</sub>((<i>S</i>)-1-(((<i>S</i>)-1-hydroxy-3-((<i>S</i>)-2-oxopyrrolidin-3-yl)propan-2-yl)amino)-4-methyl-1-oxopentan-2-yl)carbamate (<b>1d</b>)</h4><div class="NLM_p last">Yield (36%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.72 (s, 1H), 6.04 (s, 1H), 5.85 (t, <i>J</i> = 8.1 Hz, 1H), 5.62–5.55 (m, 1H), 5.20 (s, 1H), 4.23–4.15 (m, 1H), 4.02–3.93 (m, 1H), 3.68–3.54 (m, 2H), 3.39–3.33 (m, 2H), 2.49–2.31 (m, 3H), 2.30–2.26 (m, 1H), 2.16–2.09 (m, 1H), 2.03–1.95 (m, 2H), 1.93–1.78 (m, 4H), 1.78–1.59 (m, 4H), 1.59–1.46 (m, 2H), 1.46–1.35 (m, 2H), 0.95 (d, <i>J</i> = 6.5 Hz, 6H).</div></div><div id="sec5_1_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> Bicyclo[2.2.1]hept-5-en-2-ylmethyl((<i>S</i>)-1-(((<i>S</i>)-1-hydroxy-3-((S)-2-oxopyrrolidin-3-yl)propan-2-yl)amino)-4-methyl-1-oxopentan-2-yl)carbamate (<b>1e</b>)</h4><div class="NLM_p last">Yield (44%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.74 (s, 1H), 6.38 (s, 1H), 6.14 (dd, <i>J</i> = 5.7, 3.0 Hz, 1H), 5.93 (dd, <i>J</i> = 5.8, 2.9 Hz, 1H), 5.41–5.32 (m, 1H), 4.27–4.19 (m, 1H), 4.19–4.07 (m, 1H), 4.06–3.89 (m, 1H), 3.88–3.80 (m, 1H), 3.69–3.53 (m, 3H), 3.35 (dd, <i>J</i> = 10.6, 4.2 Hz, 2H), 2.87–2.77 (m, 2H), 2.52–2.34 (m, 2H), 2.10–1.87 (m, 1H), 1.87–1.77 (m, 1H), 1.77–1.57 (m, 3H), 1.57–1.47 (m, 1H), 1.47–1.40 (m, 1H), 1.37–1.18 (m, 1H), 1.18–1.10 (m, 1H), 0.96 (d, <i>J</i> = 6.5 Hz, 6H), 0.53 (ddd, <i>J</i> = 11.7, 4.5, 2.6 Hz, 1H).</div></div><div id="sec5_1_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> Bicyclo[2.2.1]heptan-2-ylmethyl((<i>S</i>)-1-(((<i>S</i>)-1-hydroxy-3-((<i>S</i>)-2-oxopyrrolidin-3-yl)propan-2-yl)amino)-4-methyl-1-oxopentan-2-yl)carbamate (<b>1f</b>)</h4><div class="NLM_p last">Yield (44%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.73 (s, 1H), 6.30 (s, 1H), 5.30 (s, 1H), 4.30–4.15 (m, 1H), 4.15–4.04 (m, 1H), 4.04–3.95 (m, 1H), 3.95–3.85 (m, 1H), 3.78 (d, <i>J</i> = 8.0 Hz, 1H), 3.71–3.52 (m, 2H), 3.39–3.32 (m, 2H), 2.48–2.36 (m, 2H), 2.24–2.17 (m, 1H), 2.17–1.96 (m, 1H), 1.97–1.75 (m, 1H), 1.75–1.56 (m, 3H), 1.56–1.41 (m, 3H), 1.41–1.21 (m, 3H), 1.21–0.99 (m, 3H), 0.95 (d, <i>J</i> = 6.4 Hz, 6H), 0.66 (dd, <i>J</i> = 12.6, 5.1 Hz, 1H).</div></div><div id="sec5_1_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> <i>tert</i>-Butyl (1<i>R</i>,3<i>s</i>,5<i>S</i>)-3-((((<i>S</i>)-1-(((<i>S</i>)-1-Hydroxy-3-((<i>S</i>)-2-oxopyrrolidin-3-yl)propan-2-yl)amino)-4-methyl-1-oxopentan-2-yl)carbamoyl)oxy)-8-azabicyclo[3.2.1]octane-8-carboxylate (<b>1g</b>)</h4><div class="NLM_p last">Yield (45%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.85 (s, 1H), 6.29 (s, 1H), 5.31 (d, <i>J</i> = 9.3 Hz, 1H), 4.94 (s, 1H), 4.29–4.02 (m, 3H), 4.01–3.97 (m, 1H), 3.64–3.59 (m, 2H), 3.39–3.31 (m, 2H), 2.44–2.39 (m, 2H), 2.20–1.89 (m, 8H), 1.84 (dd, <i>J</i> = 11.3, 9.0 Hz, 1H), 1.76 (d, <i>J</i> = 15.2 Hz, 2H), 1.73–1.60 (m, 2H), 1.60–1.48 (m, 1H), 1.46 (s, 9H), 0.99–0.91 (m, 6H).</div></div><div id="sec5_1_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> Benzyl (1<i>R</i>,3<i>s</i>,5<i>S</i>)-3-((((<i>S</i>)-1-(((<i>S</i>)-1-hydroxy-3-((<i>S</i>)-2-oxopyrrolidin-3-yl)propan-2-yl)amino)-4-methyl-1-oxopentan-2-yl)carbamoyl)oxy)-8-azabicyclo[3.2.1]octane-8-carboxylate (<b>1h</b>)</h4><div class="NLM_p last">Yield (49%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.60–7.56 (m, 2H), 7.47–7.10 (m, 6H), 5.11 (d, <i>J</i> = 10.53 Hz, 2H), 5.02–4.88 (m, 1H), 4.83–4.61 (m, 1H), 4.30–4.15 (m, 2H), 4.05 (d, <i>J</i> = 7.11 Hz, 1H), 3.97–3.88 (m, 2H), 3.79–3.71 (m, 1H), 3.65–3.56 (m, 2H), 2.79–2.71 (m, 1H), 2.30–1.35 (m, 5H), 1.35–0.90 (m, 9H), 0.87 (td, <i>J</i> = 9.48, 8.02, 8.02 Hz, 6H).</div></div><div id="sec5_1_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> (4-Pentylbicyclo[2.2.2]octan-1-yl)methyl ((<i>S</i>)-1-(((<i>S</i>)-1-Hydroxy-3-((S)-2-oxopyrrolidin-3-yl)propan-2-yl)amino)-4-methyl-1-oxopentan-2-yl)carbamate (<b>1i</b>)</h4><div class="NLM_p last">Yield (33%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.63–7.45 (m, 1H), 7.09–6.97 (m, 2H), 6.94–4.71 (m, 2H), 4.69–4.57 (m, 5H), 2.90 (s, 1H), 2.78–2.70 (m, 1H), 1.95–1.87 (m, 5H), 1.32 (d, <i>J</i> = 7.74 Hz, 6H), 1.23–1.15 (m, 19H), 0.86 (dd, <i>J</i> = 13.86, 6.91 Hz, 6H).</div></div><div id="sec5_1_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> (4-Pentylbicyclo[2.2.2]octan-1-yl)methyl-<i>d</i><sub>2</sub> ((<i>S</i>)-1-(((<i>S</i>)-1-hydroxy-3-((<i>S</i>)-2-oxopyrrolidin-3-yl)propan-2-yl)amino)-4-methyl-1-oxopentan-2-yl)carbamate (<b>1j</b>)</h4><div class="NLM_p last">Yield (20%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.62–7.47 (m, 1H), 7.10–6.97 (m, 2H), 6.97–4.75 (m, 2H), 4.69–4.59 (m, 3H), 2.92 (s, 1H), 2.78–2.76 (m, 1H), 1.95–1.87 (m, 5H), 1.39 (d, <i>J</i> = 7.74 Hz, 6H), 1.23–1.17 (m, 19H), 0.86 (dd, <i>J</i> = 13.86, 6.91 Hz, 6H).</div></div><div id="sec5_1_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> ((3<i>S</i>,5<i>S</i>,7<i>S</i>)-Adamantan-1-yl)methyl ((<i>S</i>)-1-(((<i>S</i>)-1-Hydroxy-3-((S)-2-oxopyrrolidin-3-yl)propan-2-yl)amino)-4-methyl-1-oxopentan-2-yl)carbamate (<b>1k</b>)</h4><div class="NLM_p last">Yield (75%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.75 (d, <i>J</i> = 7.1 Hz, 1H), 6.15 (s, 1H), 5.27 (d, <i>J</i> = 8.1 Hz, 1H), 4.23–4.18 (m, 1H), 4.03–3.96 (m, 1H), 3.74–3.57 (m, 4H), 3.39–3.31 (m, 2H), 2.49–2.34 (m, 2H), 1.99–1.95 (m, 4H), 1.88–1.79 (m, 1H), 1.76–1.60 (m, 9H), 1.58–1.46 (m, 7H), 0.96 (dd, <i>J</i> = 6.4, 2.4 Hz, 6H).</div></div><div id="sec5_1_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> ((3<i>S</i>,5<i>S</i>,7<i>S</i>)-Adamantan-1-yl)methyl-<i>d</i><sub>2</sub> ((<i>S</i>)-1-(((<i>S</i>)-1-hydroxy-3-((<i>S</i>)-2-oxopyrrolidin-3-yl)propan-2-yl)amino)-4-methyl-1-oxopentan-2-yl)carbamate (<b>1l</b>)</h4><div class="NLM_p last">Yield (26%). <sup>1</sup>H NMR (400 MHz, cdcl<sub>3</sub>): δ 7.73 (d, <i>J</i> = 7.3 Hz, 1H), 6.19 (s, 1H), 5.26 (d, <i>J</i> = 8.1 Hz, 1H), 4.22–4.18 (m, 1H), 4.03–3.96 (m, 1H), 3.67–3.54 (m, 2H), 3.39–3.31 (m, 2H), 2.49–2.33 (m, 2H), 2.06–2.02 (m, 1H), 1.99–1.95 (m, 3H), 1.88–1.79 (m, 2H), 1.78–1.60 (m, 9H), 1.57–1.48 (m, 6H), 0.96 (dd, <i>J</i> = 6.4, 2.1 Hz, 6H).</div></div><div id="sec5_1_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> 2-((3<i>S</i>,5<i>S</i>,7<i>S</i>)-Adamantan-1-yl)ethyl ((<i>S</i>)-1-(((<i>S</i>)-1-Hydroxy-3-((<i>S</i>)-2-oxopyrrolidin-3-yl)propan-2-yl)amino)-4-methyl-1-oxopentan-2-yl)carbamate (<b>1m</b>)</h4><div class="NLM_p last">Yield (65%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.73 (d, <i>J</i> = 7.0 Hz, 1H), 6.24 (s, 1H), 5.28 (d, <i>J</i> = 8.3 Hz, 1H), 4.25–4.20 (m, 1H), 4.17–4.07 (m, 2H), 4.04–3.96 (m, 1H), 3.70–3.44 (m, 2H), 3.39–3.31 (m, 2H), 2.52–2.34 (m, 2H), 2.09–1.97 (m, 1H), 1.97–1.91 (m, 3H), 1.90–1.76 (m, 1H), 1.74–1.57 (m, 9H), 1.55–1.45 (m, 7H), 1.40 (t, <i>J</i> = 7.5 Hz, 2H), 0.95 (d, <i>J</i> = 6.4 Hz, 6H).</div></div><div id="sec5_1_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> ((1<i>R</i>,3<i>R</i>,5<i>R</i>,7<i>S</i>)-3-Hydroxyadamantan-1-yl)methyl ((<i>S</i>)-1-(((<i>S</i>)-1-Hydroxy-3-((<i>S</i>)-2-oxopyrrolidin-3-yl)propan-2-yl)amino)-4-methyl-1-oxopentan-2-yl)carbamate (<b>1n</b>)</h4><div class="NLM_p last">Yield (8%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.69 (d, <i>J</i> = 7.3 Hz, 1H), 6.34 (s, 1H), 5.45 (d, <i>J</i> = 8.0 Hz, 1H), 4.20–4.16 (m, 1H), 4.05–3.87 (m, 1H), 3.74–3.54 (m, 4H), 3.37–3.28 (m, 2H), 2.45–2.40 (m, 2H), 2.21 (s, 2H), 2.11–1.94 (m, 1H), 1.88–1.79 (m, 1H), 1.79–1.59 (m, 8H), 1.59–1.54 (m, 2H), 1.54–1.47 (m, 3H), 1.47–1.32 (m, 3H), 0.99–0.88 (m, 6H).</div></div><div id="sec5_1_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> ((1<i>r</i>,3<i>R</i>,5<i>R</i>,7<i>S</i>)-3-Hydroxyadamantan-1-yl)methyl-d<sub>2</sub> ((<i>S</i>)-1-(((<i>S</i>)-1-hydroxy-3-((<i>S</i>)-2-oxopyrrolidin-3-yl)propan-2-yl)amino)-4-methyl-1-oxopentan-2-yl)carbamate (<b>1o</b>)</h4><div class="NLM_p last">Yield (7%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.70 (d, <i>J</i> = 7.1 Hz, 1H), 6.26 (s, 1H), 5.41 (d, <i>J</i> = 7.9 Hz, 1H), 4.23–4.14 (m, 1H), 4.03–3.96 (m, 1H), 3.68–3.54 (m, 2H), 3.39–3.32 (m, 2H), 2.43 (s, 2H), 2.27–2.19 (m, 2H), 2.11–1.91 (m, 1H), 1.88–1.80 (m, 1H), 1.77–1.60 (m, 8H), 1.60–1.54 (m, 1H), 1.54–1.47 (m, 4H), 1.43–1.38 (m, 3H), 0.99–0.88 (m, 6H).</div></div></div></div><div id="sec6" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i33">Preparation of Compounds <b>2(a–o)</b></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_48670" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_48670" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec6_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> General Procedure</h3><div class="NLM_p">To a solution of dipeptidyl alcohol <b>1</b> (1 equiv) in anhydrous dichloromethane (300 mL/g dipeptidyl alcohol) kept at 0–5 °C under a N<sub>2</sub> atmosphere was added the DMP reagent (3.0 equiv), and the reaction mixture was stirred for 3 h at 15–20 °C. The organic phase was washed with 10% aq Na<sub>2</sub>S<sub>2</sub>O<sub>3</sub> (2 × 100 mL/g dipeptidyl alcohol), followed by saturated aqueous NaHCO<sub>3</sub> (2 × 100 mL/g dipeptidyl alcohol), distilled water (2 × 100 mL/g dipeptidyl alcohol), and brine (100 mL/g dipeptidyl alcohol). The organic phase was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated <i>in vacuo</i>. The resulting crude product was purified by flash chromatography (hexane/ethyl acetate) to yield aldehyde <b>2</b> as a white solid.</div><div id="sec6_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> ((1<i>R</i>,3<i>s</i>,5<i>S</i>)-Bicyclo[3.3.1]nonan-3-yl)methyl ((<i>S</i>)-4-Methyl-1-oxo-1-(((<i>S</i>)-1-oxo-3-((<i>S</i>)-2-oxopyrrolidin-3-yl)propan-2-yl)amino)pentan-2-yl)carbamate (<b>2a</b>)</h4><div class="NLM_p last">Yield (86%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 9.50 (s, 1H), 8.31 (s, 1H), 5.21 (d, <i>J</i> = 8.7 Hz, 1H), 4.38–4.28 (m, 2H), 3.96–3.87 (m, 2H), 3.40–3.31 (m, 2H), 2.54–2.36 (m, 2H), 2.08–1.99 (m, 3H), 1.99–1.81 (m, 5H), 1.76–1.63 (m, 6H), 1.60–1.49 (m, 1H), 1.46–1.29 (m, 4H), 1.15–1.05 (m, 1H), 0.97 (d, <i>J</i> = 6.3 Hz, 6H), 0.88 (d, <i>J</i> = 13.4 Hz, 2H). HRMS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>24</sub>H<sub>40</sub>N<sub>3</sub>O<sub>5</sub>, 450.2968; found, 450.2958, <i>m</i>/<i>z</i>: [M + Na]<sup>+</sup> calcd for C<sub>24</sub>H<sub>39</sub>N<sub>3</sub>NaO<sub>5</sub>, 472.2788; found, 472.2776.</div></div><div id="sec6_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> ((1<i>R</i>,3<i>s</i>,5<i>S</i>)-Bicyclo[3.3.1]nonan-3-yl)methyl-<i>d</i><sub>2</sub> ((<i>S</i>)-4-Methyl-1-oxo-1-(((<i>S</i>)-1-oxo-3-((<i>S</i>)-2-oxopyrrolidin-3-yl)propan-2-yl)amino)pentan-2-yl)carbamate (<b>2b</b>)</h4><div class="NLM_p last">Yield (85%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 9.50 (s, 1H), 8.30 (s, 1H), 6.16 (s, 1H), 5.24 (d, <i>J</i> = 8.6 Hz, 1H), 4.38–4.29 (m, 2H), 3.41–3.30 (m, 2H), 2.52–2.34 (m, 2H), 2.12–1.99 (m, 3H), 1.98–1.78 (m, 4H), 1.78–1.62 (m, 4H), 1.61–1.51 (m, 1H), 1.41–1.30 (m, 6H), 1.13–1.06 (m, 1H), 0.97 (d, <i>J</i> = 6.3 Hz, 6H), 0.96–0.83 (m, 2H). HRMS <i>m</i>/<i>z</i>: [M + Na]<sup>+</sup> calcd for C<sub>24</sub>H<sub>37</sub>D<sub>2</sub>N<sub>3</sub>NaO<sub>5</sub>, 474.2913; found, 474.2897.</div></div><div id="sec6_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> ((1<i>S</i>,5<i>R</i>)-Bicyclo[3.3.1]non-6-en-3-yl)methyl ((<i>S</i>)-4-Methyl-1-oxo-1-(((<i>S</i>)-1-oxo-3-((S)-2-oxopyrrolidin-3-yl)propan-2-yl)amino)pentan-2-yl)carbamate (<b>2c</b>)</h4><div class="NLM_p last">Yield (53%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 9.49 (s, 1H), 8.27 (s, 1H), 6.04 (s, 1H), 5.86 (t, <i>J</i> = 7.1 Hz, 1H), 5.63–5.55 (m, 1H), 5.18 (d, <i>J</i> = 8.5 Hz, 1H), 4.38–4.27 (m, 2H), 4.10–4.03 (m, 1H), 4.03–3.94 (m, 1H), 3.41–3.31 (m, 2H), 2.52–2.37 (m, 1H), 2.28 (s, 2H), 2.18–2.08 (m, 1H), 2.01–1.92 (m, 3H), 1.92–1.83 (m, 3H), 1.83–1.71 (m, 1H), 1.71–1.66 (m, 4H), 1.62–1.49 (m, 3H), 1.49–1.36 (m, 1H), 0.97 (d, <i>J</i> = 6.4 Hz, 6H). HRMS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>24</sub>H<sub>38</sub>N<sub>3</sub>O<sub>5</sub>, 448.2811; found, 448.2810, <i>m</i>/<i>z</i>: [M + Na]<sup>+</sup> calcd for C<sub>24</sub>H<sub>37</sub>N<sub>3</sub>NaO<sub>5</sub>, 470.2631; found, 470.2628.</div></div><div id="sec6_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> ((1<i>S</i>,3<i>S</i>,5<i>R</i>)-Bicyclo[3.3.1]non-6-en-3-yl)methyl-<i>d</i><sub>2</sub> ((<i>S</i>)-4-methyl-1-oxo-1-(((<i>S</i>)-1-oxo-3-((<i>S</i>)-2-oxopyrrolidin-3-yl)propan-2-yl)amino)pentan-2-yl)carbamate (<b>2d</b>)</h4><div class="NLM_p last">Yield (88%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 9.49 (s, 1H), 8.28 (d, <i>J</i> = 6.1 Hz, 1H), 6.33 (s, 1H), 5.85 (t, <i>J</i> = 8.0 Hz, 1H), 5.58 (d, <i>J</i> = 9.9 Hz, 1H), 5.24 (d, <i>J</i> = 8.7 Hz, 1H), 4.38–4.29 (m, 2H), 3.43–3.30 (m, 2H), 2.54–2.32 (m, 2H), 2.31–2.26 (m, 1H), 2.20–2.08 (m, 1H), 2.08–1.62 (m, 9H), 1.62–1.46 (m, 4H), 1.45–1.38 (m, 2H), 0.97 (d, <i>J</i> = 6.1 Hz, 6H). HRMS <i>m</i>/<i>z</i>: [M + Na]<sup>+</sup> calcd for C<sub>24</sub>H<sub>35</sub>D<sub>2</sub>N<sub>3</sub>NaO<sub>5</sub>, 472.2757; found, 472.2743.</div></div><div id="sec6_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> Bicyclo[2.2.1]hept-5-en-2-ylmethyl ((<i>S</i>)-4-Methyl-1-oxo-1-(((<i>S</i>)-1-oxo-3-((<i>S</i>)-2-oxopyrrolidin-3-yl)propan-2-yl)amino)pentan-2-yl)carbamate (<b>2e</b>)</h4><div class="NLM_p last">Yield (84%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 9.50 (s, 1H), 8.30 (d, <i>J</i> = 6.7 Hz, 1H), 6.39 (s, 1H), 6.14 (dd, <i>J</i> = 5.8, 3.1 Hz, 1H), 5.93 (dd, <i>J</i> = 5.7, 2.9 Hz, 1H), 5.27 (s, 1H), 4.38–4.31 (m, 2H), 3.89–3.80 (m, 1H), 3.69–3.60 (m, 1H), 3.43–3.30 (m, 2H), 2.89–2.78 (m, 1H), 2.56–2.31 (m, 3H), 2.11–1.96 (m, 1H), 1.96–1.90 (m, 1H), 1.90–1.78 (m, 2H), 1.78–1.63 (m, 2H), 1.63–1.49 (m, 1H), 1.48–1.33 (m, 1H), 1.29–1.18 (m, 1H), 1.19–1.10 (m, 1H), 0.97 (d, <i>J</i> = 6.3 Hz, 6H), 0.58–0.50 (m, 1H). HRMS <i>m</i>/<i>z</i>: [M + Na]<sup>+</sup> calcd for C<sub>22</sub>H<sub>33</sub>N<sub>3</sub>NaO<sub>5</sub>, 442.2318; found, 442.2310.</div></div><div id="sec6_1_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> Bicyclo[2.2.1]heptan-2-ylmethyl ((<i>S</i>)-4-Methyl-1-oxo-1-(((<i>S</i>)-1-oxo-3-((<i>S</i>)-2-oxopyrrolidin-3-yl)propan-2-yl)amino)pentan-2-yl)carbamate (<b>2f</b>)</h4><div class="NLM_p last">Yield (88%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 9.50 (s, 1H), 8.29 (s, 1H), 6.34 (s, 1H), 5.26 (s, 1H), 4.39–4.29 (m, 2H), 4.14–4.03 (m, 1H), 3.97–3.86 (m, 1H), 3.84–3.72 (m, 1H), 3.43–3.30 (m, 2H), 2.54–2.34 (m, 2H), 2.29–2.15 (m, 2H), 2.16–2.08 (m, 1H), 2.08–1.77 (m, 1H), 1.77–1.63 (m, 3H), 1.59–1.43 (m, 2H), 1.41–1.23 (m, 4H), 1.22–1.02 (m, 2H), 0.97 (d, <i>J</i> = 6.3 Hz, 6H), 0.67 (ddd, <i>J</i> = 12.5, 5.4, 2.3 Hz, 1H). HRMS <i>m</i>/<i>z</i>: [M + Na]<sup>+</sup> calcd for C<sub>22</sub>H<sub>35</sub>N<sub>3</sub>NaO<sub>5</sub>, 444.2475; found, 444.2467.</div></div><div id="sec6_1_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> tert-Butyl (1<i>R</i>,3<i>s</i>,5<i>S</i>)-3-((((<i>S</i>)-4-methyl-1-oxo-1-(((<i>S</i>)-1-oxo-3-((<i>S</i>)-2-oxopyrrolidin-3-yl)propan-2-yl)amino)pentan-2-yl)carbamoyl)oxy)-8-azabicyclo[3.2.1]octane-8-carboxylate (<b>2g</b>)</h4><div class="NLM_p last">Yield (90%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 9.49 (s, 1H), 8.43 (d, <i>J</i> = 5.6 Hz, 1H), 6.07 (s, 1H), 5.20 (d, <i>J</i> = 8.6 Hz, 1H), 4.96 (s, 1H), 4.35–4.28 (m, 2H), 4.25–4.09 (m, 2H), 3.41–3.33 (m, 2H), 2.53–2.36 (m, 2H), 2.23–1.94 (m, 8H), 1.94–1.82 (m, 1H), 1.81–1.64 (m, 4H), 1.61–1.52 (m, 1H), 1.46 (s, 9H), 0.98 (d, <i>J</i> = 5.0 Hz, 6H). HRMS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>26</sub>H<sub>43</sub>N<sub>4</sub>O<sub>7</sub>, 523.3131; found, 523.3116. HRMS <i>m</i>/<i>z</i>: [M + Na]<sup>+</sup> calcd for C<sub>26</sub>H<sub>42</sub>N<sub>4</sub>NaO<sub>7</sub>, 545.2951; found, 545.2938.</div></div><div id="sec6_1_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> Benzyl (1<i>R</i>,3<i>s</i>,5<i>S</i>)-3-((((<i>S</i>)-4-methyl-1-oxo-1-(((<i>S</i>)-1-oxo-3-((<i>S</i>)-2-oxopyrrolidin-3-yl)propan-2-yl)amino)pentan-2-yl)carbamoyl)oxy)-8-azabicyclo[3.2.1]octane-8-carboxylate (<b>2h</b>)</h4><div class="NLM_p last">Yield (66%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.39 (s, 1H), 8.55–8.35 (m, 1H), 8.11–8.02 (m, 1H), 7.64 (s, 1H), 7.45–7.19 (m, 5H), 5.75 (s, 1H), 5.09 (d, <i>J</i> = 10.47 Hz, 2H), 4.83–4.71 (m, 2H), 4.21 (s, 2H), 3.73–3.57 (m, 2H), 3.24–3.00 (m, 2H), 2.34–1.79 (m, 3H), 1.79–1.34 (m, 2H), 1.28–1.12 (m, 9H), 0.98–0.77 (m, 6H). HRMS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>29</sub>H<sub>41</sub>N<sub>4</sub>O<sub>7</sub>, 557.2970; found, 557.2962. HRMS <i>m</i>/<i>z</i>: [M + Na]<sup>+</sup> calcd for C<sub>29</sub>H<sub>40</sub>N<sub>4</sub>NaO<sub>7</sub>, 579.2789; found, 579.2773.</div></div><div id="sec6_1_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> (4-Pentylbicyclo[2.2.2]octan-1-yl)methyl ((<i>S</i>)-4-Methyl-1-oxo-1-(((<i>S</i>)-1-oxo-3-((<i>S</i>)-2-oxopyrrolidin-3-yl)propan-2-yl)amino)pentan-2-yl)carbamate (<b>2i</b>)</h4><div class="NLM_p last">Yield (59%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.49 (s, 1H), 7.66–7.58 (m, 1H), 7.54–7.44 (m, 2H), 5.72–5.64 (m, 2H), 3.71–3.53 (m, 5H), 3.21–3.13 (m, 1H), 2.93–2.85 (m, 1H), 2.77–2.69 (m, 1H), 2.30–2.22 (m, 1H), 1.91 (s, 1H), 1.63–1.55 (m, 5H), 1.42–0.96 (m, 20H), 0.87 (td, <i>J</i> = 19.27, 6.96, 6.96 Hz, 6H). HRMS <i>m</i>/<i>z</i>: [M + Na]<sup>+</sup> calcd for C<sub>28</sub>H<sub>47</sub>N<sub>3</sub>NaO<sub>5</sub>, 528.3414; found, 528.3391.</div></div><div id="sec6_1_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> (4-Pentylbicyclo[2.2.2]octan-1-yl)methyl-<i>d</i><sub>2</sub> ((<i>S</i>)-4-Methyl-1-oxo-1-(((<i>S</i>)-1-oxo-3-((<i>S</i>)-2-oxopyrrolidin-3-yl)propan-2-yl)amino)pentan-2-yl)carbamate (<b>2j</b>)</h4><div class="NLM_p last">Yield (49%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.49 (s, 1H), 7.66–7.58 (m, 1H), 7.54–7.44 (m, 2H), 5.72–5.64 (m, 2H), 3.71–3.63 (m, 3H), 3.31 (s, 1H), 2.93–2.85 (m, 1H), 2.85–2.69 (m, 1H), 2.30–2.22 (m, 1H), 1.91 (s, 1H), 1.42–0.96 (m, 25H), 0.87 (td, <i>J</i> = 19.21, 6.96, 6.96 Hz, 6H). HRMS <i>m</i>/<i>z</i>: [M + Na]<sup>+</sup> calcd for C<sub>28</sub>H<sub>45</sub>D<sub>2</sub>N<sub>3</sub>NaO<sub>5</sub>, 530.3539; found, 530.3536.</div></div><div id="sec6_1_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> ((3<i>S</i>,5<i>S</i>,7<i>S</i>)-Adamantan-1-yl)methyl((<i>S</i>)-4-methyl-1-oxo-1-(((<i>S</i>)-1-oxo-3-((<i>S</i>)-2-oxopyrrolidin-3-yl)propan-2-yl)amino)pentan-2-yl)carbamate (<b>2k</b>)</h4><div class="NLM_p last">Yield (90%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 9.50 (s, 1H), 8.31 (d, <i>J</i> = 5.8 Hz, 1H), 6.36 (s, 1H), 5.27 (s, 1H), 4.39–4.29 (m, 2H), 3.66 (s, 2H), 3.41–3.30 (m, 2H), 2.55–2.34 (m, 2H), 2.08–1.92 (m, 3H), 1.92–1.78 (m, 2H), 1.78–1.60 (m, 9H), 1.52 (d, <i>J</i> = 2.9 Hz, 7H), 1.00–0.94 (m, 6H). HRMS <i>m</i>/<i>z</i>: [M + Na]<sup>+</sup> calcd for C<sub>25</sub>H<sub>39</sub>N<sub>3</sub>NaO<sub>5</sub>, 484.2788; found, 484.2780.</div></div><div id="sec6_1_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> ((3<i>S</i>,5<i>S</i>,7<i>S</i>)-Adamantan-1-yl)methyl-<i>d</i><sub>2</sub> ((<i>S</i>)-4-methyl-1-oxo-1-(((<i>S</i>)-1-oxo-3-((<i>S</i>)-2-oxopyrrolidin-3-yl)propan-2-yl)amino)pentan-2-yl)carbamate (<b>2l</b>)</h4><div class="NLM_p last">Yield (88%). <sup>1</sup>H NMR (400 MHz, cdcl<sub>3</sub>): δ 9.49 (s, 1H), 8.32 (d, <i>J</i> = 5.9 Hz, 1H), 6.13 (d, <i>J</i> = 8.1 Hz, 1H), 5.23 (s, 1H), 4.68–4.58 (m, 1H), 4.38–4.25 (m, 1H), 3.41–3.27 (m, 2H), 2.52–2.34 (m, 2H), 2.06–2.01 (m, 2H), 2.00–1.89 (m, 3H), 1.88–1.78 (m, 4H), 1.78–1.55 (m, 9H), 1.52 (d, <i>J</i> = 2.9 Hz, 3H), 1.00–0.91 (m, 6H). HRMS <i>m</i>/<i>z</i>: [M + Na]<sup>+</sup> calcd for C<sub>25</sub>H<sub>37</sub>D<sub>2</sub>N<sub>3</sub>NaO<sub>5</sub>, 486.2913; found, 486.2910.</div></div><div id="sec6_1_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> 2-((3<i>S</i>,5<i>S</i>,7<i>S</i>)-Adamantan-1-yl)ethyl ((<i>S</i>)-4-Methyl-1-oxo-1-(((<i>S</i>)-1-oxo-3-((<i>S</i>)-2-oxopyrrolidin-3-yl)propan-2-yl)amino)pentan-2-yl)carbamate (<b>2m</b>)</h4><div class="NLM_p last">Yield (83%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 9.50 (s, 1H), 8.30 (d, <i>J</i> = 5.9 Hz, 1H), 6.09 (d, <i>J</i> = 11.7 Hz, 1H), 5.19 (d, <i>J</i> = 8.7 Hz, 1H), 4.38–4.28 (m, 2H), 4.26–4.05 (m, 2H), 3.41–3.33 (m, 2H), 2.54–2.35 (m, 2H), 2.01–1.91 (m, 4H), 1.91–1.79 (m, 1H), 1.74–1.58 (m, 9H), 1.51 (d, <i>J</i> = 2.9 Hz, 6H), 1.41 (t, <i>J</i> = 7.4 Hz, 2H), 1.28–1.23 (m, 1H), 0.97 (d, <i>J</i> = 6.4 Hz, 6H). HRMS <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>26</sub>H<sub>42</sub>N<sub>3</sub>O<sub>5</sub>, 476.3124; found, 476.3124. HRMS <i>m</i>/<i>z</i>: [M + Na]<sup>+</sup> calcd for C<sub>26</sub>H<sub>41</sub>N<sub>3</sub>NaO<sub>5</sub>, 498.2944; found, 498.2938.</div></div><div id="sec6_1_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> ((1<i>r</i>,3<i>R</i>,5<i>R</i>,7<i>S</i>)-3-Hydroxyadamantan-1-yl)methyl ((<i>S</i>)-4-Methyl-1-oxo-1-(((<i>S</i>)-1-oxo-3-((<i>S</i>)-2-oxopyrrolidin-3-yl)propan-2-yl)amino)pentan-2-yl)carbamate (<b>2n</b>)</h4><div class="NLM_p last">Yield (47%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 9.49 (s, 1H), 8.28 (d, <i>J</i> = 23.9 Hz, 1H), 6.19 (s, 1H), 5.19 (s, 1H), 4.34–4.03 (m, 2H), 4.02–3.61 (m, 2H), 2.89–2.58 (m, 2H), 2.58–2.28 (m, 2H), 2.28–2.13 (m, 3H), 2.09–1.76 (m, 2H), 1.78–1.60 (m, 6H), 1.57–1.31 (m, 9H), 1.01–0.91 (m, 6H). HRMS <i>m</i>/<i>z</i>: [M + Na]<sup>+</sup> calcd for C<sub>25</sub>H<sub>39</sub>N<sub>3</sub>NaO<sub>6</sub>, 500.2737; found, 500.2739.</div></div><div id="sec6_1_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> ((1<i>r</i>,3<i>R</i>,5<i>R</i>,7<i>S</i>)-3-Hydroxyadamantan-1-yl)methyl-<i>d</i><sub>2</sub> ((<i>S</i>)-4-Methyl-1-oxo-1-(((<i>S</i>)-1-oxo-3-((<i>S</i>)-2-oxopyrrolidin-3-yl)propan-2-yl)amino)pentan-2-yl)carbamate (<b>2o</b>)</h4><div class="NLM_p last">Yield (44%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 9.48 (s, 1H), 8.28 (d, <i>J</i> = 6.5 Hz, 1H), 6.39 (s, 1H), 5.12 (s, 1H), 4.40–4.09 (m, 2H), 2.89–2.59 (m, 2H), 2.56–2.30 (m, 2H), 2.30–2.20 (m, 3H), 2.18–1.77 (m, 2H), 1.77–1.61 (m, 6H), 1.59–1.39 (m, 9H), 1.01–0.89 (m, 6H). HRMS <i>m</i>/<i>z</i>: [M + Na]<sup>+</sup> calcd for C<sub>25</sub>H<sub>37</sub>D<sub>2</sub>N<sub>3</sub>NaO<sub>6</sub>, 502.2862; found, 502.2860.</div></div></div></div><div id="sec7" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i50">Preparation of Compounds <b>3(a–o)</b></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_00833" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_00833" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec7_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i51" class="anchor-spacer"></div><h3 class="article-section__title" id="_i51"> General Procedure</h3><div class="NLM_p">To a solution of dipeptidyl aldehyde <b>2</b> (1 equiv) in ethyl acetate (10 mL/g dipeptidyl aldehyde) was added absolute ethanol (5 mL/g dipeptidyl aldehyde) with stirring, followed by a solution of sodium bisulfite (1 equiv) in water (1 mL/g dipeptidyl aldehyde). The reaction mixture was stirred for 3 h at 50 °C. The reaction mixture was allowed to cool to room temperature and then vacuum-filtered. The solid was thoroughly washed with absolute ethanol, and the filtrate was dried over anhydrous sodium sulfate, filtered, and concentrated to yield a white solid. The white solid was stirred with dry ethyl ether (3 × 10 mL/g dipeptidyl aldehyde), followed by careful removal of the solvent using a pipette and dried using a vacuum pump for 2 h to yield dipeptidyl bisulfite adduct <b>3</b> as a white solid.</div><div id="sec7_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> Sodium (2<i>S</i>)-2-((2<i>S</i>)-2-(((((1<i>R</i>,5<i>S</i>)-Bicyclo[3.3.1]nonan-3-yl)methoxy)carbonyl)amino)-4-methylpentanamido)-1-hydroxy-3-((<i>S</i>)-2-oxopyrrolidin-3-yl)propane-1-sulfonate (<b>3a</b>)</h4><div class="NLM_p last">Yield (66%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.61 (d, <i>J</i> = 9.0 Hz, 1H), 7.57–7.44 (m, 1H), 7.28–7.08 (m, 1H), 5.62 (d, <i>J</i> = 6.1 Hz, 1H), 5.47 (d, <i>J</i> = 5.9 Hz, 1H), 4.12–3.91 (m, 2H), 3.91–3.73 (m, 2H), 3.16–3.00 (m, 2H), 2.24–2.07 (m, 2H), 2.06–1.95 (m, 2H), 1.83 (dt, <i>J</i> = 17.2, 7.2 Hz, 4H), 1.74–1.51 (m, 5H), 1.51–1.38 (m, 2H), 1.35–1.15 (m, 6H), 1.14–1.00 (m, 2H), 0.99–0.81 (m, 6H). HRMS <i>m</i>/<i>z</i>: [M + Na]<sup>+</sup> calcd for C<sub>24</sub>H<sub>40</sub>N<sub>3</sub>Na<sub>2</sub>O<sub>8</sub>S, 576.2332; found, 576.2329, <i>m</i>/<i>z</i>: [M]<sup>−</sup> calcd for C<sub>24</sub>H<sub>40</sub>N<sub>3</sub>O<sub>8</sub>S, 530.2536; found, 530.2529.</div></div><div id="sec7_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> Sodium (2<i>S</i>)-2-((<i>S</i>)-2-(((((1<i>R</i>,3<i>s</i>,5<i>S</i>)-Bicyclo[3.3.1]nonan-3-yl)methoxy-<i>d</i><sub>2</sub>)carbonyl)amino)-4-methylpentanamido)-1-hydroxy-3-((<i>S</i>)-2-oxopyrrolidin-3-yl)propane-1-sulfonate (<b>3b</b>)</h4><div class="NLM_p last">Yield (75%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.63–7.43 (m, 2H), 7.20 (dd, <i>J</i> = 20.3, 8.2 Hz, 1H), 5.52–5.33 (m, 1H), 4.06–3.78 (m, 2H), 3.20–2.99 (m, 2H), 2.33–2.04 (m, 3H), 2.04–1.93 (m, 2H), 1.94–1.74 (m, 2H), 1.74–1.50 (m, 3H), 1.50–1.38 (m, 2H), 1.38–1.19 (m, 5H), 1.11–0.98 (m, 2H), 0.97–0.80 (m, 10H). HRMS <i>m</i>/<i>z</i>: [M + Na]<sup>+</sup> calcd for C<sub>24</sub>H<sub>38</sub>D<sub>2</sub>N<sub>3</sub>Na<sub>2</sub>O<sub>8</sub>S, 578.2457; found, 578.2432.</div></div><div id="sec7_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> Sodium (2<i>S</i>)-2-((2<i>S</i>)-2-(((((1<i>S</i>,5<i>R</i>)-Bicyclo[3.3.1]non-6-en-3-yl)methoxy)carbonyl)amino)-4-methylpentanamido)-1-hydroxy-3-((<i>S</i>)-2-oxopyrrolidin-3-yl)propane-1-sulfonate (<b>3c</b>)</h4><div class="NLM_p last">Yield (66%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.52 (d, <i>J</i> = 9.5 Hz, 1H), 7.47 (d, <i>J</i> = 4.5 Hz, 1H), 7.18 (ddd, <i>J</i> = 19.5, 8.5, 2.7 Hz, 1H), 5.89–5.79 (m, 1H), 5.58–5.52 (m, 1H), 3.97 (h, <i>J</i> = 8.1 Hz, 1H), 3.90–3.70 (m, 1H), 3.27–3.00 (m, 2H), 2.35–2.19 (m, 3H), 2.19–2.02 (m, 3H), 2.02–1.68 (m, 6H), 1.68–1.49 (m, 4H), 1.49–1.25 (m, 6H), 1.13–1.04 (m, 1H), 0.91–0.79 (m, 6H). HRMS <i>m</i>/<i>z</i>: [M + Na]<sup>+</sup> calcd for C<sub>24</sub>H<sub>38</sub>N<sub>3</sub>Na<sub>2</sub>O<sub>8</sub>S, 574.2175; found, 574.2163, <i>m</i>/<i>z</i>: [M]<sup>−</sup> calcd for C<sub>24</sub>H<sub>38</sub>N<sub>3</sub>O<sub>8</sub>S, 528.2379; found, 528.2367.</div></div><div id="sec7_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> Sodium (2<i>S</i>)-2-((<i>S</i>)-2-(((((1<i>S</i>,3<i>S</i>,5<i>R</i>)-Bicyclo[3.3.1]non-6-en-3-yl)methoxy-<i>d</i><sub>2</sub>)carbonyl)amino)-4-methylpentanamido)-1-hydroxy-3-((<i>S</i>)-2-oxopyrrolidin-3-yl)propane-1-sulfonate (<b>3d</b>)</h4><div class="NLM_p last">Yield (53%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.62–7.39 (m, 2H), 7.18 (dd, <i>J</i> = 24.9, 8.5 Hz, 1H), 5.89–5.80 (m, 1H), 5.59–5.52 (m, 1H), 5.43–5.26 (m, 1H), 4.06–3.67 (m, 2H), 3.22–2.98 (m, 2H), 2.37–2.03 (m, 5H), 2.03–1.64 (m, 5H), 1.64–1.52 (m, 3H), 1.52–1.29 (m, 5H), 1.13–1.02 (m, 1H), 0.89–0.80 (m, 6H). HRMS <i>m</i>/<i>z</i>: [M + Na]<sup>+</sup> calcd for C<sub>24</sub>H<sub>36</sub>D<sub>2</sub>N<sub>3</sub>Na<sub>2</sub>O<sub>8</sub>S, 576.2301; found, 576.2275.</div></div><div id="sec7_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> Sodium (2<i>S</i>)-2-((2<i>S</i>)-2-(((Bicyclo[2.2.1]hept-5-en-2-ylmethoxy)carbonyl)amino)-4-methylpentanamido)-1-hydroxy-3-((<i>S</i>)-2-oxopyrrolidin-3-yl)propane-1-sulfonate (<b>3e</b>)</h4><div class="NLM_p last">Yield (56%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.68–7.37 (m, 2H), 7.32–7.17 (m, 1H), 6.20–6.02 (m, 1H), 6.02–5.86 (m, 1H), 5.59–5.30 (m, 1H), 4.11–3.79 (m, 2H), 3.79–3.41 (m, 2H), 3.21–2.98 (m, 2H), 2.87–2.66 (m, 2H), 2.39–2.04 (m, 3H), 2.04–1.85 (m, 1H), 1.85–1.70 (m, 1H), 1.70–1.51 (m, 2H), 1.50–1.36 (m, 2H), 1.28 (dd, <i>J</i> = 39.4, 7.9 Hz, 2H), 1.18–1.00 (m, 1H), 0.93–0.79 (m, 6H), 0.47 (d, <i>J</i> = 11.6 Hz, 1H). HRMS <i>m</i>/<i>z</i>: [M + Na]<sup>+</sup> calcd for C<sub>22</sub>H<sub>34</sub>N<sub>3</sub>Na<sub>2</sub>O<sub>8</sub>S, 546.1862; found, 546.1842.</div></div><div id="sec7_1_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> Sodium (2<i>S</i>)-2-((2<i>S</i>)-2-(((Bicyclo[2.2.1]heptan-2-ylmethoxy)carbonyl)amino)-4-methylpentanamido)-1-hydroxy-3-((<i>S</i>)-2-oxopyrrolidin-3-yl)propane-1-sulfonate (<b>3f</b>)</h4><div class="NLM_p last">Yield (66%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.61–7.39 (m, 2H), 7.32–7.08 (m, 1H), 5.41 (dd, <i>J</i> = 55.1, 5.9 Hz, 1H), 4.07–3.55 (m, 2H), 3.18–2.99 (m, 2H), 2.28–2.04 (m, 6H), 2.04–1.81 (m, 1H), 1.81–1.53 (m, 3H), 1.53–1.36 (m, 5H), 1.36–1.18 (m, 3H), 1.18–0.91 (m, 2H), 0.90–0.75 (m, 6H), 0.70–0.59 (m, 1H). HRMS <i>m</i>/<i>z</i>: [M + Na]<sup>+</sup> calcd for C<sub>22</sub>H<sub>36</sub>N<sub>3</sub>Na<sub>2</sub>O<sub>8</sub>S, 548.2019; found, 548.1999.</div></div><div id="sec7_1_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> Sodium (2<i>S</i>)-2-((<i>S</i>)-2-(((((1<i>R</i>,3<i>s</i>,5<i>S</i>)-8-(<i>tert</i>-Butoxycarbonyl)-8-azabicyclo[3.2.1]octan-3-yl)oxy)carbonyl)amino)-4-methylpentanamido)-1-hydroxy-3-((<i>S</i>)-2-oxopyrrolidin-3-yl)propane-1-sulfonate (<b>3g</b>)</h4><div class="NLM_p last">Yield (52%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.64–7.51 (m, 1H), 7.46 (d, <i>J</i> = 3.8 Hz, 1H), 7.30 (dd, <i>J</i> = 19.2, 8.3 Hz, 1H), 5.55–5.31 (m, 1H), 4.76 (s, 1H), 4.02–3.78 (m, 4H), 3.22–2.98 (m, 2H), 2.27–1.68 (m, 9H), 1.66–1.57 (m, 4H), 1.51–1.43 (m, 2H), 1.40 (s, 9H), 1.13–1.02 (m, 1H), 0.91–0.81 (m, 6H). HRMS <i>m</i>/<i>z</i>: [M + Na]<sup>+</sup> calcd for C<sub>26</sub>H<sub>43</sub>N<sub>4</sub>Na<sub>2</sub>O<sub>10</sub>S, 649.2496; found, 649.2500.</div></div><div id="sec7_1_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> Sodium (2<i>S</i>)-2-((<i>S</i>)-2-(((((1<i>R</i>,3<i>s</i>,5<i>S</i>)-8-((Benzyloxy)carbonyl)-8-azabicyclo[3.2.1]octan-3-yl)oxy)carbonyl)amino)-4-methylpentanamido)-1-hydroxy-3-((<i>S</i>)-2-oxopyrrolidin-3-yl)propane-1-sulfonate (<b>3h</b>)</h4><div class="NLM_p last">Yield (53%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.37 (t, <i>J</i> = 4.74, 4.74 Hz, 8H), 5.09 (d, <i>J</i> = 11.06 Hz, 2H), 4.29–4.13 (m, 5H), 3.96–3.82 (m, 1H), 3.79–3.69 (m, 1H), 3.17–3.09 (m, 1H), 3.06–2.98 (m, 1H), 2.05–1.81 (m, 5H), 1.74–1.63 (m, 3H), 1.58–1.49 (m, 4H), 1.47–1.33 (m, 2H), 1.08 (dd, <i>J</i> = 13.72, 6.78 Hz, 1H), 0.84 (ddd, <i>J</i> = 10.35, 8.11, 4.59 Hz, 6H). HRMS <i>m</i>/<i>z</i>: [M + Na]<sup>+</sup> calcd for C<sub>29</sub>H<sub>41</sub>N<sub>4</sub>Na<sub>2</sub>O<sub>10</sub>S, 683.2339; found, 683.2317.</div></div><div id="sec7_1_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> Sodium (2<i>S</i>)-1-Hydroxy-2-((<i>S</i>)-4-methyl-2-((((4-pentylbicyclo[2.2.2]octan-1-yl)methoxy)carbonyl)amino)pentanamido)-3-((<i>S</i>)-2-oxopyrrolidin-3-yl)propane-1-sulfonate (<b>3i</b>)</h4><div class="NLM_p last">Yield (57%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.51–7.40 (m, 3H), 4.00–3.53 (m, 5H), 3.49–3.40 (m, 1H), 3.16–2.98 (m, 4H), 1.94–1.85 (m, 3H), 1.65–0.96 (m, 25H), 0.85 (dd, <i>J</i> = 12.73, 5.76 Hz, 6H). HRMS <i>m</i>/<i>z</i>: [M + Na]<sup>+</sup> calcd for C<sub>28</sub>H<sub>48</sub>N<sub>3</sub>Na<sub>2</sub>O<sub>8</sub>S, 632.2958; found, 632.2932. HRMS <i>m</i>/<i>z</i>: [M]<sup>−</sup> calcd for C<sub>28</sub>H<sub>48</sub>N<sub>3</sub>O<sub>8</sub>S, 586.3168; found, 586.3163.</div></div><div id="sec7_1_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61"> Sodium (2<i>S</i>)-1-Hydroxy-2-((<i>S</i>)-4-methyl-2-((((4-pentylbicyclo[2.2.2]octan-1-yl)methoxy-<i>d</i><sub>2</sub>)carbonyl)amino)pentanamido)-3-((<i>S</i>)-2-oxopyrrolidin-3-yl)propane-1-sulfonate (<b>3j</b>)</h4><div class="NLM_p last">Yield (50%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.51–7.40 (m, 3H), 4.00–3.53 (m, 3H), 3.40–3.26 (m, 1H), 3.16–2.90 (m, 3H), 2.56–2.44 (m, 1H), 1.95–1.85 (m, 3H), 1.65–0.96 (m, 25H), 0.85 (dd, <i>J</i> = 12.73, 5.76 Hz, 6H). HRMS <i>m</i>/<i>z</i>: [M + Na]<sup>+</sup> calcd for C<sub>28</sub>H<sub>46</sub>D<sub>2</sub>N<sub>3</sub>Na<sub>2</sub>O<sub>8</sub>S, 634.3083; found, 634.3071. HRMS <i>m</i>/<i>z</i>: [M]<sup>−</sup> calcd for C<sub>28</sub>H<sub>46</sub>D<sub>2</sub>N<sub>3</sub>O<sub>8</sub>S, 588.3287; found, 588.3399.</div></div><div id="sec7_1_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62"> Sodium (2<i>S</i>)-2-((<i>S</i>)-2-(((((3<i>S</i>,5<i>S</i>,7<i>S</i>)-Adamantan-1-yl)methoxy)carbonyl)amino)-4-methylpentanamido)-1-hydroxy-3-((<i>S</i>)-2-oxopyrrolidin-3-yl)propane-1-sulfonate (<b>3k</b>)</h4><div class="NLM_p last">Yield (52%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.66–7.41 (m, 2H), 7.25–7.10 (m, 1H), 5.40 (dd, <i>J</i> = 47.1, 6.0 Hz, 1H), 4.10–3.70 (m, 2H), 3.69–3.47 (m, 2H), 3.18–3.00 (m, 2H), 2.24–2.03 (m, 2H), 2.03–1.83 (m, 4H), 1.83–1.55 (m, 8H), 1.55–1.33 (m, 6H), 1.13–1.02 (m, 1H), 0.93–0.80 (m, 6H). HRMS <i>m</i>/<i>z</i>: [M + Na]<sup>+</sup> calcd for C<sub>25</sub>H<sub>40</sub>N<sub>3</sub>Na<sub>2</sub>O<sub>8</sub>S, 588.2332; found, 588.2310.</div></div><div id="sec7_1_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i63" class="anchor-spacer"></div><h4 class="article-section__title" id="_i63"> Sodium (2<i>S</i>)-2-((<i>S</i>)-2-(((((3<i>S</i>,5<i>S</i>,7<i>S</i>)-Adamantan-1-yl)methoxy-<i>d</i><sub>2</sub>)carbonyl)amino)-4-methylpentanamido)-1-hydroxy-3-((<i>S</i>)-2-oxopyrrolidin-3-yl)propane-1-sulfonate (<b>3l</b>)</h4><div class="NLM_p last">Yield (39%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.63–7.50 (m, 1H), 7.47–7.36 (m, 1H), 7.38–7.28 (m, 1H), 5.41 (dd, <i>J</i> = 36.5, 6.0 Hz, 1H), 4.27–4.19 (m, 1H), 3.99–3.94 (m, 1H), 3.49–3.34 (m, 1H), 3.21–3.01 (m, 2H), 2.25–1.99 (m, 2H), 1.99–1.90 (m, 3H), 1.89–1.73 (m, 3H), 1.72–1.52 (m, 9H), 1.48 (s, 5H), 0.90–0.76 (m, 6H). HRMS <i>m</i>/<i>z</i>: [M + Na]<sup>+</sup> calcd for C<sub>25</sub>H<sub>38</sub>D<sub>2</sub>N<sub>3</sub>Na<sub>2</sub>O<sub>8</sub>S, 590.2457; found, 590.2447.</div></div><div id="sec7_1_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i64" class="anchor-spacer"></div><h4 class="article-section__title" id="_i64"> Sodium (2<i>S</i>)-2-((<i>S</i>)-2-(((2-((3<i>S</i>,5<i>S</i>,7<i>S</i>)-Adamantan-1-yl)ethoxy)carbonyl)amino)-4-methylpentanamido)-1-hydroxy-3-((<i>S</i>)-2-oxopyrrolidin-3-yl)propane-1-sulfonate (<b>3m</b>)</h4><div class="NLM_p last">Yield (60%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.62–7.49 (m, 1H), 7.49–7.43 (m, 1H), 7.24–7.07 (m, 1H), 5.40 (dd, <i>J</i> = 48.6, 6.1 Hz, 1H), 4.05–3.81 (m, 4H), 3.20–3.00 (m, 2H), 2.26–2.06 (m, 2H), 2.01–1.87 (m, 3H), 1.84–1.72 (m, 1H), 1.70–1.54 (m, 10H), 1.49 (d, <i>J</i> = 3.0 Hz, 6H), 1.45–1.38 (m, 1H), 1.38–1.28 (m, 2H), 0.89–0.80 (m, 6H). HRMS <i>m</i>/<i>z</i>: [M + Na]<sup>+</sup> calcd for C<sub>26</sub>H<sub>42</sub>N<sub>3</sub>Na<sub>2</sub>O<sub>8</sub>S, 602.2488; found, 602.2480.</div></div><div id="sec7_1_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i65" class="anchor-spacer"></div><h4 class="article-section__title" id="_i65"> Sodium (2<i>S</i>)-1-Hydroxy-2-((<i>S</i>)-2-(((((1<i>r</i>,3<i>R</i>,5<i>R</i>,7<i>S</i>)-3-hydroxyadamantan-1-yl)methoxy)carbonyl)amino)-4-methylpentanamido)-3-((<i>S</i>)-2-oxopyrrolidin-3-yl)propane-1-sulfonate (<b>3n</b>)</h4><div class="NLM_p last">Yield (69%). <sup>1</sup>H NMR (400 MHz, acetone): δ 7.08–7.03 (m, 1H), 7.00–6.95 (m, 1H), 6.45–6.41 (m, 1H), 4.38–3.89 (m, 2H), 3.89–3.53 (m, 2H), 2.96–2.53 (m, 1H), 2.53–2.25 (m, 2H), 2.16 (s, 2H), 1.99–1.70 (m, 4H), 1.70–1.62 (m, 5H), 1.62–1.53 (m, 4H), 1.53–1.35 (m, 5H), 1.34–1.16 (m, 1H), 0.99–0.85 (m, 6H). HRMS <i>m</i>/<i>z</i>: [M + Na]<sup>+</sup> calcd for C<sub>25</sub>H<sub>40</sub>N<sub>3</sub>Na<sub>2</sub>O<sub>9</sub>S, 604.2282; found, 604.2273.</div></div><div id="sec7_1_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i66" class="anchor-spacer"></div><h4 class="article-section__title" id="_i66"> Sodium (2<i>S</i>)-1-Hydroxy-2-((<i>S</i>)-2-(((((1<i>r</i>,3<i>R</i>,5<i>R</i>,7<i>S</i>)-3-hydroxyadamantan-1-yl)methoxy-<i>d</i><sub>2</sub>)carbonyl)amino)-4-methylpentanamido)-3-((<i>S</i>)-2-oxopyrrolidin-3-yl)propane-1-sulfonate (<b>3o</b>)</h4><div class="NLM_p last">Yield (73%). <sup>1</sup>H NMR (400 MHz, acetone): δ 7.10–7.05 (m, 1H), 6.91–6.87 (m, 1H), 6.43–6.38 (m, 1H), 4.33–4.06 (m, 2H), 3.33–3.19 (m, 2H), 2.50–2.24 (m, 1H), 2.22–2.10 (m, 3H), 2.03–1.71 (m, 2H), 1.70–1.61 (m, 5H), 1.61–1.52 (m, 5H), 1.52–1.38 (m, 6H), 1.01–0.85 (m, 6H). HRMS <i>m</i>/<i>z</i>: [M + Na]<sup>+</sup> calcd for C<sub>25</sub>H<sub>38</sub>D<sub>2</sub>N<sub>3</sub>Na<sub>2</sub>O<sub>9</sub>S, 606.2406; found, 606.2409.</div></div></div></div><div id="sec8" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i67">Biochemical Studies</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_43341" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_43341" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec8_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i68" class="anchor-spacer"></div><h3 class="article-section__title" id="_i68"> Enzyme Assays and Inhibition Studies</h3><div id="sec8_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i69" class="anchor-spacer"></div><h4 class="article-section__title" id="_i69"> Cloning and Expression of the 3CL Protease of SARS-CoV-2 and FRET Enzyme Assays</h4><div class="NLM_p last">The codon-optimized cDNA of full length of 3CLpro of SARS-CoV-2 (GenBank number MN908947.3) fused with sequences encoding six histidines at the N-terminal was synthesized by Integrated DNA (Coralville, IA). The synthesized gene was subcloned into the pET-28a(+) vector. The expression and purification of SARS-CoV-2 3CLpro were conducted following a standard procedure described previously.<a onclick="showRef(event, 'ref17 ref18 ref25'); return false;" href="javascript:void(0);" class="ref ref17 ref18 ref25">(17,18,25)</a> Briefly, a stock solution of an inhibitor was prepared in DMSO and diluted in assay buffer composed of 20 mM HEPES buffer, pH 8, containing NaCl (200 mM), EDTA (0.4 mM), glycerol (60%), and 6 mM dithiothreitol. The SARS-CoV-2 protease was mixed with serial dilutions of the inhibitor or with DMSO in 25 μL of assay buffer and incubated at 37 °C for 1 h, followed by the addition of 25 μL of assay buffer containing the substrate (FAM-SAVLQ/SG-QXL520, AnaSpec, Fremont, CA). The substrate was derived from the cleavage sites on the viral polyproteins of SARS-CoV. Fluorescence readings were obtained using an excitation wavelength of 480 nm and an emission wavelength of 520 nm on a fluorescence microplate reader (FLx800; Biotec, Winoosk, VT) for 1 h following the addition of the substrate. Relative fluorescence units were determined by subtracting background values (substrate-containing well without protease) from the raw fluorescence values, as described previously.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> The dose-dependent FRET inhibition curves were fitted with a variable slope using GraphPad Prism software (GraphPad, La Jolla, CA) in order to determine the IC<sub>50</sub> values of the compounds. The expression and purification of the 3CLpro of MERS-CoV and the FRET enzyme assays were performed as described previously.<a onclick="showRef(event, 'ref17 ref18 ref25'); return false;" href="javascript:void(0);" class="ref ref17 ref18 ref25">(17,18,25)</a></div></div></div><div id="sec8_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i70" class="anchor-spacer"></div><h3 class="article-section__title" id="_i70"> Cell-Based Assay for Antiviral Activity</h3><div class="NLM_p last">Compounds <b>2a</b> and <b>3a</b> were investigated for their antiviral activity against the replication of SARS-CoV-2. Briefly, confluent Vero E6 cells were inoculated with SARS-CoV-2 at 50–100 plaque-forming units/well, and medium containing various concentrations of each compound and agar was applied to the cells. After 48–72 h, plaques in each well were counted. The 50% effective concentration (EC<sub>50</sub>) values were determined using GraphPad Prism software using a variable slope (GraphPad, La Jolla, CA).</div></div><div id="sec8_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i71" class="anchor-spacer"></div><h3 class="article-section__title" id="_i71"> Nonspecific Cytotoxic Effects/<i>in Vitro</i> Cytotoxicity</h3><div class="NLM_p last">Confluent cells grown in 96-well plates were incubated with various concentrations (1–100 μM) of each compound for 72 h. Cell cytotoxicity was measured using a CytoTox 96 nonradioactive cytotoxicity assay kit (Promega, Madison, WI), and the CC<sub>50</sub> values were calculated using a variable slope using GraphPad Prism software. The <i>in vitro</i> SI was calculated by dividing the CC<sub>50</sub> by the IC<sub>50.</sub></div></div></div><div id="sec9" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i72">X-ray Crystallographic Studies</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_33587" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_33587" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec9_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i73" class="anchor-spacer"></div><h3 class="article-section__title" id="_i73"> Crystallization and Data Collection</h3><div class="NLM_p last">Purified SARS-2 3CL protease (SARS-2 3CLpro) in 100 mM NaCl and 20 mM Tris buffer, pH 8.0, was concentrated to 9.6 mg/mL (0.28 mM) for crystallization screening. All crystallization experiments were set up using an NT8 drop-setting robot (Formulatrix Inc.) and UVXPO MRC (Molecular Dimensions) sitting drop vapor diffusion plates at 18 °C. Protein (100 nL) and 100 nL of crystallization solution were dispensed and equilibrated against 50 μL of the latter. Stock solutions of the inhibitors (100 mM) were prepared in DMSO, and the complexes were obtained by mixing 1 μL of the ligand (2 mM) with 49 μL (0.28 mM) of SARS-2 3CLpro and incubating on ice for 1 h. Crystals were obtained in 1–2 days from the following conditions. <b>2a</b> and <b>3b</b>: Berkeley screen (Rigaku Reagents) condition C5 (20% (w/v) PEG 4000, 100 mM Tris pH 8.0), <b>2f</b>: Index HT screen (Hampton Research) condition H6 (20% (w/v) PEG 3350, 200 mM sodium formate), <b>2k</b>: Proplex HT screen (Molecular Dimensions) condition D7 (15% (w/v) PEG 6000, 100 mM sodium citrate pH 5.5), <b>3c</b> and <b>3d</b>: the Berkeley screen (Rigaku Reagents) condition D9 (20% (w/v) PEG 3350, 100 mM Bis-Tris pH 6.5, 100 mM ammonium phosphate dibasic, 5% (v/v) 2-propanol), and <b>3e:</b> Index HT screen (Hampton Research) condition C5 (15% (w/v) PEG 3350, 100 mM succinic acid pH 7.0). Samples were transferred to cryoprotectant solutions, prior to plunging in liquid nitrogen, composed of 80% crystallization solution and 20% (v/v) PEG 200 except for <b>3c</b> and <b>3d</b> for which 20% (v/v) ethylene glycol was used as the cryoprotectant. X-ray diffraction data were collected at the Advanced Photon Source IMCA-CAT beamline 17-ID except for the data for the complex with <b>3c</b> which were collected at the National Synchrotron Light Source II (NSLS-II) AMX beamline 17-ID-1.</div></div><div id="sec9_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i74" class="anchor-spacer"></div><h3 class="article-section__title" id="_i74"> Structure Solution and Refinement</h3><div class="NLM_p last">Intensities were integrated using XDS<a onclick="showRef(event, 'ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref30 ref31">(30,31)</a> via Autoproc,<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> and the Laue class analysis and data scaling were performed with Aimless.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> Structure solution was conducted by molecular replacement with Phaser<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> using a previously determined structure of SARS-2 3CLpro (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6XMK">6XMK</a>) as the search model. Structure refinement and manual model building were conducted with Phenix<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> and Coot,<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> respectively. Disordered side chains were truncated to the point for which electron density could be observed. Structure validation was conducted with MolProbity,<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> and structure analysis/figure preparation were carried out using the CCP4mg package.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> Crystallographic data are provided in <a href="/doi/suppl/10.1021/acs.jmedchem.1c00319/suppl_file/jm1c00319_si_001.pdf" class="ext-link">Table S1</a>.<a onclick="showRef(event, 'ref39 ref40 ref41 ref42 ref43'); return false;" href="javascript:void(0);" class="ref ref39 ref40 ref41 ref42 ref43">(39−43)</a></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i75"><a href="/doi/suppl/10.1021/acs.jmedchem.1c00319" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_02118" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_02118" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.1c00319?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00319</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Binding mode of the 3b with SARS-CoV-2 3CLpro; Superposition of 3b with 2a, 3c with 3d; Surface representation showing the orientation 2f, 2k and 3e in subunit B near the S4 subsite of SARS-CoV-2 3CLpro; and Crystallographic data for SARS-CoV-2 3CLpro inhibitor complexes (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00319/suppl_file/jm1c00319_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings—SMILES codes (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00319/suppl_file/jm1c00319_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00319/suppl_file/jm1c00319_si_001.pdf">jm1c00319_si_001.pdf (389.59 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00319/suppl_file/jm1c00319_si_002.csv">jm1c00319_si_002.csv (3.54 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Coordinates and structure factors for the following SARS2 3CLpro complexes with inhibitors were deposited to the Worldwide Protein Databank (wwPDB) with the accession codes 7LKR (<b>2a</b>), 7LKS (<b>2f</b>), 7LKT (<b>2k</b>), 7LKU (<b>3b</b>), 7LKV (<b>3c</b>), 7LKW (<b>3d</b>), and 7LKX (<b>3e</b>). Authors will release the atomic coordinates upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.1c00319" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_44031" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_44031" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kyeong-Ok Chang</span> - <span class="hlFld-Affiliation affiliation">Department
of Diagnostic Medicine & Pathobiology, College of Veterinary Medicine, Kansas State University, Manhattan, Kansas 66506, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#1c7e757070327b6e7369687d6f5c6b757f7475687d32797869"><span class="__cf_email__" data-cfemail="e7858e8b8bc980958892938694a7908e848f8e9386c9828392">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">William C. Groutas</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, Wichita State University, Wichita, Kansas 67260, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0001-5248-7912" title="Orcid link">https://orcid.org/0000-0001-5248-7912</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#355e565d545b52754350411b5e46401b505140"><span class="__cf_email__" data-cfemail="1d767e757c737a5d6b786933766e6833787968">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chamandi S. Dampalla</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, Wichita State University, Wichita, Kansas 67260, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-8199-2376" title="Orcid link">https://orcid.org/0000-0002-8199-2376</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yunjeong Kim</span> - <span class="hlFld-Affiliation affiliation">Department
of Diagnostic Medicine & Pathobiology, College of Veterinary Medicine, Kansas State University, Manhattan, Kansas 66506, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Naemi Bickmeier</span> - <span class="hlFld-Affiliation affiliation">Department
of Diagnostic Medicine & Pathobiology, College of Veterinary Medicine, Kansas State University, Manhattan, Kansas 66506, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Athri D. Rathnayake</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, Wichita State University, Wichita, Kansas 67260, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-2588-7624" title="Orcid link">https://orcid.org/0000-0003-2588-7624</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Harry Nhat Nguyen</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, Wichita State University, Wichita, Kansas 67260, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jian Zheng</span> - <span class="hlFld-Affiliation affiliation">Department
of Microbiology and Immunology, Carver College of Medicine, University of Iowa, Iowa City, Iowa 52242, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Maithri M. Kashipathy</span> - <span class="hlFld-Affiliation affiliation">Protein
Structure Laboratory, The University of
Kansas, Lawrence, Kansas 66047, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Matthew A. Baird</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, Wichita State University, Wichita, Kansas 67260, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kevin P. Battaile</span> - <span class="hlFld-Affiliation affiliation">NYX,
New York Structural Biology Center, Upton, New York 11973, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-0833-3259" title="Orcid link">https://orcid.org/0000-0003-0833-3259</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Scott Lovell</span> - <span class="hlFld-Affiliation affiliation">Protein
Structure Laboratory, The University of
Kansas, Lawrence, Kansas 66047, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Stanley Perlman</span> - <span class="hlFld-Affiliation affiliation">Department
of Microbiology and Immunology, Carver College of Medicine, University of Iowa, Iowa City, Iowa 52242, United States</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i77">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_45481" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_45481" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This research was supported in part by grants from the National Institutes of Health (NIH) (R01 AI109039 to K.O.C. and P01 AI060699 and R01 AI129269 to S.P.). The use of the University of Kansas Protein Structure Laboratory was supported by a grant from the National Institute of General Medical Sciences (P30GM110761) of the NIH. The use of the IMCA-CAT beamline 17-ID at the Advanced Photon Source was supported by the companies of the Industrial Macromolecular Crystallography Association through a contract with Hauptman-Woodward Medical Research Institute. The use of the Advanced Photon Source was supported by the U.S. Department of Energy, Office of Science, Office of Basic Energy Sciences under contract no. DE-AC02-06CH11357. This research used the AMX beamline of the National Synchrotron Light Source II, a U.S. Department of Energy (DOE) Office of Science User Facility operated for the DOE Office of Science by Brookhaven National Laboratory under Contract no. DE-SC0012704. The Center for BioMolecular Structure (CBMS) is primarily supported by the National Institutes of Health, National Institute of General Medical Sciences (NIGMS) through a Center Core P30 grant (P30GM133893), and the DOE Office of Biological and Environmental Research (KP1605010).</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">ORF</td><td class="NLM_def"><p class="first last">open reading frame</p></td></tr><tr><td class="NLM_term">DSC</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N</i>′-disuccinimidyl carbonate</p></td></tr><tr><td class="NLM_term">TEA</td><td class="NLM_def"><p class="first last">triethyl amine</p></td></tr><tr><td class="NLM_term">DMP</td><td class="NLM_def"><p class="first last">Dess-Martin periodinane</p></td></tr><tr><td class="NLM_term">DTT</td><td class="NLM_def"><p class="first last">dithiothreitol</p></td></tr><tr><td class="NLM_term">DMSO</td><td class="NLM_def"><p class="first last">dimethyl sulfoxide</p></td></tr><tr><td class="NLM_term">MNV</td><td class="NLM_def"><p class="first last">murine norovirus</p></td></tr><tr><td class="NLM_term">MOI</td><td class="NLM_def"><p class="first last">multiplicity of infection</p></td></tr><tr><td class="NLM_term">CPE</td><td class="NLM_def"><p class="first last">cytopathic effects</p></td></tr><tr><td class="NLM_term">TCID<sub>50</sub></td><td class="NLM_def"><p class="first last">the 50% tissue culture infectious dose</p></td></tr><tr><td class="NLM_term">IC<sub>50</sub></td><td class="NLM_def"><p class="first last">the 50% inhibitory concentration in the enzyme assay</p></td></tr><tr><td class="NLM_term">EC<sub>50</sub></td><td class="NLM_def"><p class="first last">the 50% effective concentration in cell culture</p></td></tr><tr><td class="NLM_term">CC<sub>50</sub></td><td class="NLM_def"><p class="first last">50% cytotoxic concentration in cell-based assays</p></td></tr><tr><td class="NLM_term">GESAMT</td><td class="NLM_def"><p class="first last">general efficient structural alignment of macromolecular targets</p></td></tr><tr><td class="NLM_term">RMSD</td><td class="NLM_def"><p class="first last">root mean square deviation</p></td></tr><tr><td class="NLM_term">XDS</td><td class="NLM_def"><p class="first last">X-ray detector software</p></td></tr><tr><td class="NLM_term">MME</td><td class="NLM_def"><p class="first last">monomethyl ether</p></td></tr><tr><td class="NLM_term">PK</td><td class="NLM_def"><p class="first last">pharmacokinetics.</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i79">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_65580" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_65580" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 43 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Perlman, S.</span>, <span class="NLM_string-name">Masters, P. S.</span></span> <span> </span><span class="NLM_article-title">Coronaviridae: The Viruses and Their Replication</span>. In  <i>Fields Virology: Emerging Viruses</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howley, P. M.</span>, <span class="NLM_string-name">Knipe, D. M.</span>, <span class="NLM_string-name">Whelan, S.</span></span>, Eds.; <span class="NLM_publisher-name">Wolters Kluwer: Wichita</span>, <span class="NLM_year">2020</span>; Vol.  <span class="NLM_volume">1</span>, pp  <span class="NLM_fpage">410</span>– <span class="NLM_lpage">448</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&pages=410-448&author=S.+Perlman&author=P.+S.+Mastersauthor=P.+M.+Howley&author=D.+M.+Knipe&author=S.+Whelan&title=Fields+Virology%3A+Emerging+Viruses"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DPerlman%26aufirst%3DS.%26atitle%3DCoronaviridae%253A%2520The%2520Viruses%2520and%2520Their%2520Replication%26btitle%3DFields%2520Virology%253A%2520Emerging%2520Viruses%26aulast%3DHowley%26aufirst%3DP.%2BM.%26pub%3DWolters%2520Kluwer%253A%2520Wichita%26date%3D2020%26volume%3D1%26spage%3D410%26epage%3D448" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhuang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, Q.</span></span> <span> </span><span class="NLM_article-title">A comparison of COVID-19, SARS and MERS</span>. <i>PeerJ</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">e9725</span> <span class="refDoi"> DOI: 10.7717/peerj.9725</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=10.7717%2Fpeerj.9725" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=32879801" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2020&author=T.+Huauthor=Y.+Liuauthor=M.+Zhaoauthor=Q.+Zhuangauthor=L.+Xuauthor=Q.+He&title=A+comparison+of+COVID-19%2C+SARS+and+MERS&doi=10.7717%2Fpeerj.9725"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.7717%2Fpeerj.9725&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7717%252Fpeerj.9725%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DT.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DM.%26aulast%3DZhuang%26aufirst%3DQ.%26aulast%3DXu%26aufirst%3DL.%26aulast%3DHe%26aufirst%3DQ.%26atitle%3DA%2520comparison%2520of%2520COVID-19%252C%2520SARS%2520and%2520MERS%26jtitle%3DPeerJ%26date%3D2020%26volume%3D8%26doi%3D10.7717%2Fpeerj.9725" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Z.-G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, Z.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, J.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, Y.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, M.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, F.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, J.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holmes, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.-Z.</span></span> <span> </span><span class="NLM_article-title">A new coronavirus associated with human respiratory disease in China</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>579</i></span>,  <span class="NLM_fpage">265</span>– <span class="NLM_lpage">269</span>, <span class="refDoi"> DOI: 10.1038/s41586-020-2008-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=10.1038%2Fs41586-020-2008-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=32015508" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A528%3ADC%252BB3cXksFKlsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=579&publication_year=2020&pages=265-269&author=F.+Wuauthor=S.+Zhaoauthor=B.+Yuauthor=Y.-M.+Chenauthor=W.+Wangauthor=Z.-G.+Songauthor=Y.+Huauthor=Z.-W.+Taoauthor=J.-H.+Tianauthor=Y.-Y.+Peiauthor=M.-L.+Yuanauthor=Y.-L.+Zhangauthor=F.-H.+Daiauthor=Y.+Liuauthor=Q.-M.+Wangauthor=J.-J.+Zhengauthor=L.+Xuauthor=E.+C.+Holmesauthor=Y.-Z.+Zhang&title=A+new+coronavirus+associated+with+human+respiratory+disease+in+China&doi=10.1038%2Fs41586-020-2008-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">A new coronavirus associated with human respiratory disease in China</span></div><div class="casAuthors">Wu, Fan; Zhao, Su; Yu, Bin; Chen, Yan-Mei; Wang, Wen; Song, Zhi-Gang; Hu, Yi; Tao, Zhao-Wu; Tian, Jun-Hua; Pei, Yuan-Yuan; Yuan, Ming-Li; Zhang, Yu-Ling; Dai, Fa-Hui; Liu, Yi; Wang, Qi-Min; Zheng, Jiao-Jiao; Xu, Lin; Holmes, Edward C.; Zhang, Yong-Zhen</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">579</span>
        (<span class="NLM_cas:issue">7798</span>),
    <span class="NLM_cas:pages">265-269</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Emerging infectious diseases, such as severe acute respiratory syndrome (SARS) and Zika virus disease, present a major threat to public health.  Despite intense research efforts, how, when and where new diseases appear are still a source of considerable uncertainty.  A severe respiratory disease was recently reported in Wuhan, Hubei province, China.  As of 25 Jan. 2020, at least 1,975 cases had been reported since the first patient was hospitalized on 12 Dec. 2019.  Epidemiol. investigations have suggested that the outbreak was assocd. with a seafood market in Wuhan.  Here we study a single patient who was a worker at the market and who was admitted to the Central Hospital of Wuhan on 26 Dec. 2019 while experiencing a severe respiratory syndrome that included fever, dizziness and a cough.  Metagenomic RNA sequencing of a sample of bronchoalveolar lavage fluid from the patient identified a new RNA virus strain from the family Coronaviridae, which is designated here 'WH-Human 1' coronavirus (and has also been referred to as '2019-nCoV').  Phylogenetic anal. of the complete viral genome (29,903 nucleotides) revealed that the virus was most closely related (89.1% nucleotide similarity) to a group of SARS-like coronaviruses (genus Betacoronavirus, subgenus Sarbecovirus) that had previously been found in bats in China.  This outbreak highlights the ongoing ability of viral spill-over from animals to cause severe disease in humans.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5h_VHfuDuTbVg90H21EOLACvtfcHk0li-Y39vTx-DDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXksFKlsLc%253D&md5=0163a684829e880a0c3347e19f0ce52a</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fs41586-020-2008-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-020-2008-3%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DF.%26aulast%3DZhao%26aufirst%3DS.%26aulast%3DYu%26aufirst%3DB.%26aulast%3DChen%26aufirst%3DY.-M.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DSong%26aufirst%3DZ.-G.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DTao%26aufirst%3DZ.-W.%26aulast%3DTian%26aufirst%3DJ.-H.%26aulast%3DPei%26aufirst%3DY.-Y.%26aulast%3DYuan%26aufirst%3DM.-L.%26aulast%3DZhang%26aufirst%3DY.-L.%26aulast%3DDai%26aufirst%3DF.-H.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DQ.-M.%26aulast%3DZheng%26aufirst%3DJ.-J.%26aulast%3DXu%26aufirst%3DL.%26aulast%3DHolmes%26aufirst%3DE.%2BC.%26aulast%3DZhang%26aufirst%3DY.-Z.%26atitle%3DA%2520new%2520coronavirus%2520associated%2520with%2520human%2520respiratory%2520disease%2520in%2520China%26jtitle%3DNature%26date%3D2020%26volume%3D579%26spage%3D265%26epage%3D269%26doi%3D10.1038%2Fs41586-020-2008-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dong, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gardner, L.</span></span> <span> </span><span class="NLM_article-title">An interactive web-based dashboard to track COVID-19 in real time</span>. <i>Lancet Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">533</span>– <span class="NLM_lpage">534</span>, <span class="refDoi"> DOI: 10.1016/s1473-3099(20)30120-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=10.1016%2Fs1473-3099%2820%2930120-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=32087114" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A528%3ADC%252BB3cXksVaisbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2020&pages=533-534&author=E.+Dongauthor=H.+Duauthor=L.+Gardner&title=An+interactive+web-based+dashboard+to+track+COVID-19+in+real+time&doi=10.1016%2Fs1473-3099%2820%2930120-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">An interactive web-based dashboard to track COVID-19 in real time</span></div><div class="casAuthors">Dong, Ensheng; Du, Hongru; Gardner, Lauren</div><div class="citationInfo"><span class="NLM_cas:title">Lancet Infectious Diseases</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">533-534</span>CODEN:
                <span class="NLM_cas:coden">LIDABP</span>;
        ISSN:<span class="NLM_cas:issn">1473-3099</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The authors describe the development of an online interactive dashboard, hosted by the Center for Systems Science and Engineering (CSSE) at Johns Hopkins University, Baltimore, MD, USA, to visualize and track reported cases of coronavirus disease 2019 (COVID-19) in real time.  The dashboard, first shared publicly on Jan 22, illustrates the location and no. of confirmed COVID-19 cases, deaths, and recoveries for all affected countries.  It was developed to provide researchers, public health authorities, and the general public with a user-friendly tool to track the outbreak as it unfolds.  All data collected and displayed are made freely available, initially through Google Sheets and now through a GitHub repository, along with the feature layers of the dashboard, which are now included in the Esri Living Atlas.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDJFa57DS5AbVg90H21EOLACvtfcHk0li-Y39vTx-DDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXksVaisbs%253D&md5=7f528642a4d2b2fcc3575e37085819a8</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fs1473-3099%2820%2930120-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fs1473-3099%252820%252930120-1%26sid%3Dliteratum%253Aachs%26aulast%3DDong%26aufirst%3DE.%26aulast%3DDu%26aufirst%3DH.%26aulast%3DGardner%26aufirst%3DL.%26atitle%3DAn%2520interactive%2520web-based%2520dashboard%2520to%2520track%2520COVID-19%2520in%2520real%2520time%26jtitle%3DLancet%2520Infect.%2520Dis.%26date%3D2020%26volume%3D20%26spage%3D533%26epage%3D534%26doi%3D10.1016%2Fs1473-3099%2820%2930120-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brindisi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shahabi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chapman, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mesecar, A. D.</span></span> <span> </span><span class="NLM_article-title">Drug Development and Medicinal Chemistry Efforts toward SARS-Coronavirus and Covid-19 Therapeutics</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">907</span>– <span class="NLM_lpage">932</span>, <span class="refDoi"> DOI: 10.1002/cmdc.202000223</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=10.1002%2Fcmdc.202000223" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=32324951" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A528%3ADC%252BB3cXos1GmsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2020&pages=907-932&author=A.+K.+Ghoshauthor=M.+Brindisiauthor=D.+Shahabiauthor=M.+E.+Chapmanauthor=A.+D.+Mesecar&title=Drug+Development+and+Medicinal+Chemistry+Efforts+toward+SARS-Coronavirus+and+Covid-19+Therapeutics&doi=10.1002%2Fcmdc.202000223"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Drug Development and Medicinal Chemistry Efforts toward SARS-Coronavirus and Covid-19 Therapeutics</span></div><div class="casAuthors">Ghosh, Arun K.; Brindisi, Margherita; Shahabi, Dana; Chapman, Mackenzie E.; Mesecar, Andrew D.</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">907-932</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  The COVID-19 pandemic caused by SARS-CoV-2 infection is spreading at an alarming rate and has created an unprecedented health emergency around the globe.  There is no effective vaccine or approved drug treatment against COVID-19 and other pathogenic coronaviruses.  The development of antiviral agents is an urgent priority.  Biochem. events crit. to the coronavirus replication cycle provided a no. of attractive targets for drug development.  These include, spike protein for binding to host cell-surface receptors, proteolytic enzymes that are essential for processing polyproteins into mature viruses, and RNA-dependent RNA polymerase for RNA replication.  There has been a lot of ground work for drug discovery and development against these targets.  Also, high-throughput screening efforts have led to the identification of diverse lead structures, including natural product-derived mols.  This review highlights past and present drug discovery and medicinal-chem. approaches against SARS-CoV, MERS-CoV and COVID-19 targets.  The review hopes to stimulate further research and will be a useful guide to the development of effective therapies against COVID-19 and other pathogenic coronaviruses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqcgVQF9N3zf7Vg90H21EOLACvtfcHk0lg0bX0lKRObAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXos1GmsL4%253D&md5=8b4c3c83c05a7b64f39ba7d803ea1709</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.202000223&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.202000223%26sid%3Dliteratum%253Aachs%26aulast%3DGhosh%26aufirst%3DA.%2BK.%26aulast%3DBrindisi%26aufirst%3DM.%26aulast%3DShahabi%26aufirst%3DD.%26aulast%3DChapman%26aufirst%3DM.%2BE.%26aulast%3DMesecar%26aufirst%3DA.%2BD.%26atitle%3DDrug%2520Development%2520and%2520Medicinal%2520Chemistry%2520Efforts%2520toward%2520SARS-Coronavirus%2520and%2520Covid-19%2520Therapeutics%26jtitle%3DChemMedChem%26date%3D2020%26volume%3D15%26spage%3D907%26epage%3D932%26doi%3D10.1002%2Fcmdc.202000223" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gil, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ginex, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maestro, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nozal, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrado-Gil, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuesta-Geijo, M. Á.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urquiza, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramírez, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alonso, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campillo, N. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, A.</span></span> <span> </span><span class="NLM_article-title">COVID-19: Drug targets and potential treatments</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">12359</span>– <span class="NLM_lpage">12386</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00606</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00606" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtFWmsL3I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=12359-12386&author=C.+Gilauthor=T.+Ginexauthor=I.+Maestroauthor=V.+Nozalauthor=L.+Barrado-Gilauthor=M.+%C3%81.+Cuesta-Geijoauthor=J.+Urquizaauthor=D.+Ram%C3%ADrezauthor=C.+Alonsoauthor=N.+E.+Campilloauthor=A.+Martinez&title=COVID-19%3A+Drug+targets+and+potential+treatments&doi=10.1021%2Facs.jmedchem.0c00606"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">COVID-19: Drug Targets and Potential Treatments</span></div><div class="casAuthors">Gil, Carmen; Ginex, Tiziana; Maestro, Ines; Nozal, Vanesa; Barrado-Gil, Lucia; Cuesta-Geijo, Miguel Angel; Urquiza, Jesus; Ramirez, David; Alonso, Covadonga; Campillo, Nuria E.; Martinez, Ana</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">12359-12386</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Currently, the authors are immersed in a pandemic caused by the emerging severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), which severely threatens public health worldwide.  Until now, no drug or vaccine has been approved to treat the severe disease caused by this coronavirus, COVID-19.  The authors will focus on the main virus-based and host-based targets that can guide medicinal chem. efforts to discover new drugs for this devastating disease.  In principle, all CoVs enzymes and proteins involved in viral replication and the control of host cellular machineries are potentially druggable targets in the search for therapeutic options for SARS-CoV-2.  This perspective provides an overview of the main targets from a structural point of view, together with reported therapeutic compds. with activity against SARS-CoV-2 and/or other CoVs.  Also, the role of innate immune response to coronavirus infection and the related therapeutic options will be presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsg-YRktOknrVg90H21EOLACvtfcHk0lg0bX0lKRObAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtFWmsL3I&md5=6a0dc17c5f601f09a92da933f81bbbea</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00606&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00606%26sid%3Dliteratum%253Aachs%26aulast%3DGil%26aufirst%3DC.%26aulast%3DGinex%26aufirst%3DT.%26aulast%3DMaestro%26aufirst%3DI.%26aulast%3DNozal%26aufirst%3DV.%26aulast%3DBarrado-Gil%26aufirst%3DL.%26aulast%3DCuesta-Geijo%26aufirst%3DM.%2B%25C3%2581.%26aulast%3DUrquiza%26aufirst%3DJ.%26aulast%3DRam%25C3%25ADrez%26aufirst%3DD.%26aulast%3DAlonso%26aufirst%3DC.%26aulast%3DCampillo%26aufirst%3DN.%2BE.%26aulast%3DMartinez%26aufirst%3DA.%26atitle%3DCOVID-19%253A%2520Drug%2520targets%2520and%2520potential%2520treatments%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D12359%26epage%3D12386%26doi%3D10.1021%2Facs.jmedchem.0c00606" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tiwari, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deb, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MARTY, J. L.</span></span> <span> </span><span class="NLM_article-title">Severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2): a global pandemic and treatment strategies</span>. <i>Int. J. Antimicrob. Agents</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">106054</span>, <span class="refDoi"> DOI: 10.1016/j.ijantimicag.2020.106054</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=10.1016%2Fj.ijantimicag.2020.106054" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=32534188" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtlyjtrvL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2020&pages=106054&author=A.+Sharmaauthor=S.+Tiwariauthor=M.+K.+Debauthor=J.+L.+MARTY&title=Severe+acute+respiratory+syndrome+coronavirus-2+%28SARS-CoV-2%29%3A+a+global+pandemic+and+treatment+strategies&doi=10.1016%2Fj.ijantimicag.2020.106054"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2): a global pandemic and treatment strategies</span></div><div class="casAuthors">Sharma, Atul; Tiwari, Swapnil; Deb, Manas Kanti; Marty, Jean Louis</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Antimicrobial Agents</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">106054</span>CODEN:
                <span class="NLM_cas:coden">IAAGEA</span>;
        ISSN:<span class="NLM_cas:issn">0924-8579</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The emergence and rapid spread of coronavirus disease 2019 (COVID-19), caused by severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2), a potentially fatal disease, is swiftly leading to public health crises worldwide.  The origin of SARS-CoV-2 infection was first reported in people exposed to a seafood market in Wuhan City, China in Dec. 2019.  It has been suggested that the infection is likely to be of zoonotic origin and transmitted to humans through a not-yet-known intermediary.  As of 22 May 2020, the World Health Organization reported that there were approx. 4995,996 confirmed cases and 327,821 deaths.  SARS-CoV-2 is transmitted via inhalation or direct contact with droplets from infected people.  It has an incubation period ranging from 2 to ≥14 days.  The rate of spread of SARS-CoV-2 is greater than that for severe acute respiratory syndrome coronavirus and Middle East respiratory coronavirus.  The symptoms are similar to influenza (i.e. breathlessness, sore throat and fatigue) and infected cases are isolated and treated.  Infection is mild in most cases, but in elderly (>50 years) patients and those with cardiac and respiratory disorders, it may progress to pneumonia, acute respiratory distress syndrome and multi-organ failure.  People with strong immunity or those who have developed herd immunity are asymptomatic.  The fatality rate ranges from 3% to 4%.  Recommended methods for diagnosis of COVID-19 are mol. tests (e.g. polymerase chain reaction) on respiratory secretions, chest scan and common lab. diagnosis.  Currently, treatment is essentially supportive, and the role of antiviral agents is yet to be established as a vaccine is not yet available.  This review will focus on epidemiol., symptoms, transmission, pathogenesis, 210 1 ongoing available treatments and future perspectives of SARS-CoV-2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBUcBXUfUAJLVg90H21EOLACvtfcHk0lg0bX0lKRObAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtlyjtrvL&md5=24d5155d11e2f97f4b14c19545b72235</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.ijantimicag.2020.106054&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ijantimicag.2020.106054%26sid%3Dliteratum%253Aachs%26aulast%3DSharma%26aufirst%3DA.%26aulast%3DTiwari%26aufirst%3DS.%26aulast%3DDeb%26aufirst%3DM.%2BK.%26aulast%3DMARTY%26aufirst%3DJ.%2BL.%26atitle%3DSevere%2520acute%2520respiratory%2520syndrome%2520coronavirus-2%2520%2528SARS-CoV-2%2529%253A%2520a%2520global%2520pandemic%2520and%2520treatment%2520strategies%26jtitle%3DInt.%2520J.%2520Antimicrob.%2520Agents%26date%3D2020%26volume%3D56%26spage%3D106054%26doi%3D10.1016%2Fj.ijantimicag.2020.106054" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cannalire, R.</span>; <span class="NLM_string-name">Cerchia, C.</span>; <span class="NLM_string-name">Beccari, A. R.</span>; <span class="NLM_string-name">Di Leva, F. S.</span>; <span class="NLM_string-name">Summa, V.</span></span> <span> </span><span class="NLM_article-title">Targeting SARS-CoV-2 proteases and polymerase for COVID-19 treatment: State of the art and future opportunities</span>.  <i>J. Med. Chem.</i> [Online early access].<span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c01140</span> . Published online: Nov 13, <span class="NLM_year" style="font-weight: bold;">2020</span>. <a href="https://doi.org/10.1021/acs.jmedchem.0c01140" class="extLink">https://doi.org/10.1021/acs.jmedchem.0c01140</a> (accessed Nov 13, 2020).</span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c01140" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&author=R.+Cannalire&author=C.+Cerchia&author=A.+R.+Beccari&author=F.+S.+Di+Leva&author=V.+Summa&title=Targeting+SARS-CoV-2+proteases+and+polymerase+for+COVID-19+treatment%3A+State+of+the+art+and+future+opportunities&doi=10.1021%2Facs.jmedchem.0c01140"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c01140&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c01140%26sid%3Dliteratum%253Aachs%26aulast%3DCannalire%26aufirst%3DR.%26atitle%3DTargeting%2520SARS-CoV-2%2520proteases%2520and%2520polymerase%2520for%2520COVID-19%2520treatment%253A%2520State%2520of%2520the%2520art%2520and%2520future%2520opportunities%26jtitle%3DJ.%2520Med.%2520Chem.%26doi%3D10.1021%2Facs.jmedchem.0c01140%26date%3D2020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ullrich, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nitsche, C.</span></span> <span> </span><span class="NLM_article-title">The SARS-CoV-2 main protease as drug target</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">127377</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2020.127377</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=10.1016%2Fj.bmcl.2020.127377" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=32738988" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtlaisrbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2020&pages=127377&author=S.+Ullrichauthor=C.+Nitsche&title=The+SARS-CoV-2+main+protease+as+drug+target&doi=10.1016%2Fj.bmcl.2020.127377"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">The SARS-CoV-2 main protease as drug target</span></div><div class="casAuthors">Ullrich, Sven; Nitsche, Christoph</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">127377</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The unprecedented pandemic of the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is threatening global health.  The virus emerged in late 2019 and can cause a severe disease assocd. with significant mortality.  Several vaccine development and drug discovery campaigns are underway.  The SARS-CoV-2 main protease is considered a promising drug target, as it is dissimilar to human proteases.  Sequence and structure of the main protease are closely related to those from other betacoronaviruses, facilitating drug discovery attempts based on previous lead compds.  Covalently binding peptidomimetics and small mols. are investigated.  Various compds. show antiviral activity in infected human cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVeovBguTwDbVg90H21EOLACvtfcHk0ljG6Hb0ZU7WLA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtlaisrbJ&md5=b2fd8ea99ddf6d3e553b137904d406a9</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2020.127377&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2020.127377%26sid%3Dliteratum%253Aachs%26aulast%3DUllrich%26aufirst%3DS.%26aulast%3DNitsche%26aufirst%3DC.%26atitle%3DThe%2520SARS-CoV-2%2520main%2520protease%2520as%2520drug%2520target%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2020%26volume%3D30%26spage%3D127377%26doi%3D10.1016%2Fj.bmcl.2020.127377" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rut, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lv, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zmudzinski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patchett, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nayak, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snipas, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El Oualid, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bekes, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drag, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olsen, S. K.</span></span> <span> </span><span class="NLM_article-title">Activity profiling and crystal structures of inhibitor-bound SARS-CoV-2 papain-like protease: A framework for anti-COVID-19 drug design</span>. <i>Sci. Adv.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>6</i></span>, <span class="NLM_elocation-id">eabd4596</span> <span class="refDoi"> DOI: 10.1126/sciadv.abd4596</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=10.1126%2Fsciadv.abd4596" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=33067239" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2020&author=W.+Rutauthor=Z.+Lvauthor=M.+Zmudzinskiauthor=S.+Patchettauthor=D.+Nayakauthor=S.+J.+Snipasauthor=F.+El+Oualidauthor=T.+T.+Huangauthor=M.+Bekesauthor=M.+Dragauthor=S.+K.+Olsen&title=Activity+profiling+and+crystal+structures+of+inhibitor-bound+SARS-CoV-2+papain-like+protease%3A+A+framework+for+anti-COVID-19+drug+design&doi=10.1126%2Fsciadv.abd4596"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1126%2Fsciadv.abd4596&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fsciadv.abd4596%26sid%3Dliteratum%253Aachs%26aulast%3DRut%26aufirst%3DW.%26aulast%3DLv%26aufirst%3DZ.%26aulast%3DZmudzinski%26aufirst%3DM.%26aulast%3DPatchett%26aufirst%3DS.%26aulast%3DNayak%26aufirst%3DD.%26aulast%3DSnipas%26aufirst%3DS.%2BJ.%26aulast%3DEl%2BOualid%26aufirst%3DF.%26aulast%3DHuang%26aufirst%3DT.%2BT.%26aulast%3DBekes%26aufirst%3DM.%26aulast%3DDrag%26aufirst%3DM.%26aulast%3DOlsen%26aufirst%3DS.%2BK.%26atitle%3DActivity%2520profiling%2520and%2520crystal%2520structures%2520of%2520inhibitor-bound%2520SARS-CoV-2%2520papain-like%2520protease%253A%2520A%2520framework%2520for%2520anti-COVID-19%2520drug%2520design%26jtitle%3DSci.%2520Adv.%26date%3D2020%26volume%3D6%26doi%3D10.1126%2Fsciadv.abd4596" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jin, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guddat, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span> <span> </span><span class="NLM_article-title">Structure of Mpro from SARS-CoV-2 and discovery of its inhibitors</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>582</i></span>,  <span class="NLM_fpage">289</span>– <span class="NLM_lpage">293</span>, <span class="refDoi"> DOI: 10.1038/s41586-020-2223-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=10.1038%2Fs41586-020-2223-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=32272481" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtVyhsrrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=582&publication_year=2020&pages=289-293&author=Z.+Jinauthor=X.+Duauthor=Y.+Xuauthor=Y.+Dengauthor=M.+Liuauthor=Y.+Zhaoauthor=B.+Zhangauthor=X.+Liauthor=L.+Zhangauthor=C.+Pengauthor=Y.+Duanauthor=J.+Yuauthor=L.+Wangauthor=K.+Yangauthor=F.+Liuauthor=R.+Jiangauthor=X.+Yangauthor=T.+Youauthor=X.+Liuauthor=X.+Yangauthor=F.+Baiauthor=H.+Liuauthor=X.+Liuauthor=L.+W.+Guddatauthor=W.+Xuauthor=G.+Xiaoauthor=C.+Qinauthor=Z.+Shiauthor=H.+Jiangauthor=Z.+Raoauthor=H.+Yang&title=Structure+of+Mpro+from+SARS-CoV-2+and+discovery+of+its+inhibitors&doi=10.1038%2Fs41586-020-2223-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of Mpro from SARS-CoV-2 and discovery of its inhibitors</span></div><div class="casAuthors">Jin, Zhenming; Du, Xiaoyu; Xu, Yechun; Deng, Yongqiang; Liu, Meiqin; Zhao, Yao; Zhang, Bing; Li, Xiaofeng; Zhang, Leike; Peng, Chao; Duan, Yinkai; Yu, Jing; Wang, Lin; Yang, Kailin; Liu, Fengjiang; Jiang, Rendi; Yang, Xinglou; You, Tian; Liu, Xiaoce; Yang, Xiuna; Bai, Fang; Liu, Hong; Liu, Xiang; Guddat, Luke W.; Xu, Wenqing; Xiao, Gengfu; Qin, Chengfeng; Shi, Zhengli; Jiang, Hualiang; Rao, Zihe; Yang, Haitao</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">582</span>
        (<span class="NLM_cas:issue">7811</span>),
    <span class="NLM_cas:pages">289-293</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: A new coronavirus, known as severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), is the etiol. agent responsible for the 2019-2020 viral pneumonia outbreak of coronavirus disease 2019 (COVID-19).  Currently, there are no targeted therapeutic agents for the treatment of this disease, and effective treatment options remain very limited.  Here, we describe the results of a program that aimed to rapidly discover lead compds. for clin. use, by combining structure-assisted drug design, virtual drug screening and high-throughput screening.  This program focused on identifying drug leads that target main protease (Mpro) of SARS-CoV-2: Mpro is a key enzyme of coronaviruses and has a pivotal role in mediating viral replication and transcription, making it an attractive drug target for SARS-CoV-2.  We identified a mechanism-based inhibitor (N3) by computer-aided drug design, and then detd. the crystal structure of Mpro of SARS-CoV-2 in complex with this compd.  Through a combination of structure-based virtual and high-throughput screening, we assayed more than 10,000 compds.-including approved drugs, drug candidates in clin. trials and other pharmacol. active compds.-as inhibitors of Mpro.  Six of these compds. inhibited Mpro, showing half-maximal inhibitory concn. values that ranged from 0.67 to 21.4μM.  One of these compds. (ebselen) also exhibited promising antiviral activity in cell-based assays.  Our results demonstrate the efficacy of our screening strategy, which can lead to the rapid discovery of drug leads with clin. potential in response to new infectious diseases for which no specific drugs or vaccines are available.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFupuh6WIyW7Vg90H21EOLACvtfcHk0ljG6Hb0ZU7WLA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtVyhsrrO&md5=b84f350fe9ce1109485df6caf814ba82</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fs41586-020-2223-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-020-2223-y%26sid%3Dliteratum%253Aachs%26aulast%3DJin%26aufirst%3DZ.%26aulast%3DDu%26aufirst%3DX.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DDeng%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DM.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DB.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DPeng%26aufirst%3DC.%26aulast%3DDuan%26aufirst%3DY.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DJiang%26aufirst%3DR.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DYou%26aufirst%3DT.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DBai%26aufirst%3DF.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DGuddat%26aufirst%3DL.%2BW.%26aulast%3DXu%26aufirst%3DW.%26aulast%3DXiao%26aufirst%3DG.%26aulast%3DQin%26aufirst%3DC.%26aulast%3DShi%26aufirst%3DZ.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DRao%26aufirst%3DZ.%26aulast%3DYang%26aufirst%3DH.%26atitle%3DStructure%2520of%2520Mpro%2520from%2520SARS-CoV-2%2520and%2520discovery%2520of%2520its%2520inhibitors%26jtitle%3DNature%26date%3D2020%26volume%3D582%26spage%3D289%26epage%3D293%26doi%3D10.1038%2Fs41586-020-2223-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curth, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drosten, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sauerhering, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rox, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilgenfeld, R.</span></span> <span> </span><span class="NLM_article-title">Crystal structure of SARS-CoV-2 main protease provides a basis for design of improved α-ketoamide inhibitors</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>368</i></span>,  <span class="NLM_fpage">409</span>– <span class="NLM_lpage">412</span>, <span class="refDoi"> DOI: 10.1126/science.abb3405</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=10.1126%2Fscience.abb3405" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=32198291" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A528%3ADC%252BB3cXnslKrtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=368&publication_year=2020&pages=409-412&author=L.+Zhangauthor=D.+Linauthor=X.+Sunauthor=U.+Curthauthor=C.+Drostenauthor=L.+Sauerheringauthor=S.+Beckerauthor=K.+Roxauthor=R.+Hilgenfeld&title=Crystal+structure+of+SARS-CoV-2+main+protease+provides+a+basis+for+design+of+improved+%CE%B1-ketoamide+inhibitors&doi=10.1126%2Fscience.abb3405"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of SARS-CoV-2 main protease provides a basis for design of improved α-ketoamide inhibitors</span></div><div class="casAuthors">Zhang, Linlin; Lin, Daizong; Sun, Xinyuanyuan; Curth, Ute; Drosten, Christian; Sauerhering, Lucie; Becker, Stephan; Rox, Katharina; Hilgenfeld, Rolf</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">368</span>
        (<span class="NLM_cas:issue">6489</span>),
    <span class="NLM_cas:pages">409-412</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">1095-9203</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The coronavirus disease 2019 (COVID-19) pandemic caused by severe acute respiratory syndrome-coronavirus 2 (SARS-CoV-2) is a global health emergency.  An attractive drug target among coronaviruses is the main protease (Mpro, also called 3CLpro) because of its essential role in processing the polyproteins that are translated from the viral RNA.  We report the x-ray structures of the unliganded SARS-CoV-2 Mpro and its complex with an α-ketoamide inhibitor.  This was derived from a previously designed inhibitor but with the P3-P2 amide bond incorporated into a pyridone ring to enhance the half-life of the compd. in plasma.  On the basis of the unliganded structure, we developed the lead compd. into a potent inhibitor of the SARS-CoV-2 Mpro.  The pharmacokinetic characterization of the optimized inhibitor reveals a pronounced lung tropism and suitability for administration by the inhalative route.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrT531N47m4BbVg90H21EOLACvtfcHk0liUcBhPBjtS8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXnslKrtL8%253D&md5=9ac417c20f54c3327f9de9088b512d52</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1126%2Fscience.abb3405&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.abb3405%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DLin%26aufirst%3DD.%26aulast%3DSun%26aufirst%3DX.%26aulast%3DCurth%26aufirst%3DU.%26aulast%3DDrosten%26aufirst%3DC.%26aulast%3DSauerhering%26aufirst%3DL.%26aulast%3DBecker%26aufirst%3DS.%26aulast%3DRox%26aufirst%3DK.%26aulast%3DHilgenfeld%26aufirst%3DR.%26atitle%3DCrystal%2520structure%2520of%2520SARS-CoV-2%2520main%2520protease%2520provides%2520a%2520basis%2520for%2520design%2520of%2520improved%2520%25CE%25B1-ketoamide%2520inhibitors%26jtitle%3DScience%26date%3D2020%26volume%3D368%26spage%3D409%26epage%3D412%26doi%3D10.1126%2Fscience.abb3405" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dai, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, X.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span> <span> </span><span class="NLM_article-title">Structure-based design of antiviral drug candidates targeting the SARS-CoV-2 main protease</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>368</i></span>,  <span class="NLM_fpage">1331</span>– <span class="NLM_lpage">1335</span>, <span class="refDoi"> DOI: 10.1126/science.abb4489</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=10.1126%2Fscience.abb4489" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=32321856" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A528%3ADC%252BB3cXht1alsrzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=368&publication_year=2020&pages=1331-1335&author=W.+Daiauthor=B.+Zhangauthor=X.-M.+Jiangauthor=H.+Suauthor=J.+Liauthor=Y.+Zhaoauthor=X.+Xieauthor=Z.+Jinauthor=J.+Pengauthor=F.+Liuauthor=C.+Liauthor=Y.+Liauthor=F.+Baiauthor=H.+Wangauthor=X.+Chengauthor=X.+Cenauthor=S.+Huauthor=X.+Yangauthor=J.+Wangauthor=X.+Liuauthor=G.+Xiaoauthor=H.+Jiangauthor=Z.+Raoauthor=L.-K.+Zhangauthor=Y.+Xuauthor=H.+Yangauthor=H.+Liu&title=Structure-based+design+of+antiviral+drug+candidates+targeting+the+SARS-CoV-2+main+protease&doi=10.1126%2Fscience.abb4489"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based design of antiviral drug candidates targeting the SARS-CoV-2 main protease</span></div><div class="casAuthors">Dai, Wenhao; Zhang, Bing; Jiang, Xia-Ming; Su, Haixia; Li, Jian; Zhao, Yao; Xie, Xiong; Jin, Zhenming; Peng, Jingjing; Liu, Fengjiang; Li, Chunpu; Li, You; Bai, Fang; Wang, Haofeng; Cheng, Xi; Cen, Xiaobo; Hu, Shulei; Yang, Xiuna; Wang, Jiang; Liu, Xiang; Xiao, Gengfu; Jiang, Hualiang; Rao, Zihe; Zhang, Lei-Ke; Xu, Yechun; Yang, Haitao; Liu, Hong</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">368</span>
        (<span class="NLM_cas:issue">6497</span>),
    <span class="NLM_cas:pages">1331-1335</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">1095-9203</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">SARS-CoV-2 (severe acute respiratory syndrome coronavirus 2) is the etiol. agent responsible for the global COVID-19 (coronavirus disease 2019) outbreak.  The main protease of SARS-CoV-2, Mpro, is a key enzyme that plays a pivotal role in mediating viral replication and transcription.  We designed and synthesized two lead compds. (11a and 11b) targeting Mpro.  Both exhibited excellent inhibitory activity and potent anti-SARS-CoV-2 infection activity.  The x-ray crystal structures of SARS-CoV-2 Mpro in complex with 11a or 11b, both detd. at a resoln. of 1.5 angstroms, showed that the aldehyde groups of 11a and 11b are covalently bound to cysteine 145 of Mpro.  Both compds. showed good pharmacokinetic properties in vivo, and 11a also exhibited low toxicity, which suggests that these compds. are promising drug candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFFvzOwCOpZrVg90H21EOLACvtfcHk0liUcBhPBjtS8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXht1alsrzL&md5=ec87c340410edb919d2584a2b4d33e1a</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1126%2Fscience.abb4489&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.abb4489%26sid%3Dliteratum%253Aachs%26aulast%3DDai%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DB.%26aulast%3DJiang%26aufirst%3DX.-M.%26aulast%3DSu%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DXie%26aufirst%3DX.%26aulast%3DJin%26aufirst%3DZ.%26aulast%3DPeng%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DBai%26aufirst%3DF.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DCheng%26aufirst%3DX.%26aulast%3DCen%26aufirst%3DX.%26aulast%3DHu%26aufirst%3DS.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DXiao%26aufirst%3DG.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DRao%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DL.-K.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DH.%26atitle%3DStructure-based%2520design%2520of%2520antiviral%2520drug%2520candidates%2520targeting%2520the%2520SARS-CoV-2%2520main%2520protease%26jtitle%3DScience%26date%3D2020%26volume%3D368%26spage%3D1331%26epage%3D1335%26doi%3D10.1126%2Fscience.abb4489" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kania, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brothers, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferre, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gajiwala, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hogan, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozminski, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lockner, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marra, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, L. J.,  Jr</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nieman, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Planken, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rowe, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, G. J.,  III</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solowiej, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steppan, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taggart, B.</span></span> <span> </span><span class="NLM_article-title">Discovery of ketone-based covalent inhibitors of coronavirus 3CL proteases for the potential therapeutic treatment of COVID-19</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">12725</span>– <span class="NLM_lpage">12747</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c01063</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c01063" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A528%3ADC%252BB3cXitVKnur3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=12725-12747&author=R.+L.+Hoffmanauthor=R.+S.+Kaniaauthor=M.+A.+Brothersauthor=J.+F.+Daviesauthor=R.+A.+Ferreauthor=K.+S.+Gajiwalaauthor=M.+Heauthor=R.+J.+Hoganauthor=K.+Kozminskiauthor=L.+Y.+Liauthor=J.+W.+Locknerauthor=J.+Louauthor=M.+T.+Marraauthor=L.+J.+Mitchellauthor=B.+W.+Murrayauthor=J.+A.+Niemanauthor=S.+Noellauthor=S.+P.+Plankenauthor=T.+Roweauthor=K.+Ryanauthor=G.+J.+Smithauthor=J.+E.+Solowiejauthor=C.+M.+Steppanauthor=B.+Taggart&title=Discovery+of+ketone-based+covalent+inhibitors+of+coronavirus+3CL+proteases+for+the+potential+therapeutic+treatment+of+COVID-19&doi=10.1021%2Facs.jmedchem.0c01063"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of ketone-based covalent inhibitors of coronavirus 3CL proteases for the potential therapeutic treatment of COVID-19</span></div><div class="casAuthors">Hoffman, Robert L.; Kania, Robert S.; Brothers, Mary A.; Davies, Jay F.; Ferre, Rose A.; Gajiwala, Ketan S.; He, Mingying; Hogan, Robert J.; Kozminski, Kirk; Li, Lilian Y.; Lockner, Jonathan W.; Lou, Jihong; Marra, Michelle T.; Mitchell Jr., Lennert J.; Murray, Brion W.; Nieman, James A.; Noell, Stephen; Planken, Simon P.; Rowe, Thomas; Ryan, Kevin; Smith III, George J.; Solowiej, James E.; Steppan, Claire M.; Taggart, Barbara</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">12725-12747</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The novel coronavirus disease COVID-19 that emerged in 2019 is caused by the virus SARS CoV-2 and named for its close genetic similarity to SARS CoV-1 that caused severe acute respiratory syndrome (SARS) in 2002.  Both SARS coronavirus genomes encode two overlapping large polyproteins, which are cleaved at specific sites by a 3C-like cysteine protease (3CLpro) in a post-translational processing step that is crit. for coronavirus replication.  The 3CLpro sequences for CoV-1 and CoV-2 viruses are 100% identical in the catalytic domain that carries out protein cleavage.  A research effort that focused on the discovery of reversible and irreversible ketone-based inhibitors of SARS CoV-1 3CLpro employing ligand-protease structures solved by X-ray crystallog. led to the identification of 3 and 4.  Preclin. expts. reveal 4 (PF-00835231) as a potent inhibitor of CoV-2 3CLpro with suitable pharmaceutical properties to warrant further development as an i.v. treatment for COVID-19.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqP_zn3ZDefOrVg90H21EOLACvtfcHk0lj1FhhQ3ABagg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXitVKnur3E&md5=7d5be169ad3de528496c862934d59881</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c01063%26sid%3Dliteratum%253Aachs%26aulast%3DHoffman%26aufirst%3DR.%2BL.%26aulast%3DKania%26aufirst%3DR.%2BS.%26aulast%3DBrothers%26aufirst%3DM.%2BA.%26aulast%3DDavies%26aufirst%3DJ.%2BF.%26aulast%3DFerre%26aufirst%3DR.%2BA.%26aulast%3DGajiwala%26aufirst%3DK.%2BS.%26aulast%3DHe%26aufirst%3DM.%26aulast%3DHogan%26aufirst%3DR.%2BJ.%26aulast%3DKozminski%26aufirst%3DK.%26aulast%3DLi%26aufirst%3DL.%2BY.%26aulast%3DLockner%26aufirst%3DJ.%2BW.%26aulast%3DLou%26aufirst%3DJ.%26aulast%3DMarra%26aufirst%3DM.%2BT.%26aulast%3DMitchell%26aufirst%3DL.%2BJ.%26aulast%3DMurray%26aufirst%3DB.%2BW.%26aulast%3DNieman%26aufirst%3DJ.%2BA.%26aulast%3DNoell%26aufirst%3DS.%26aulast%3DPlanken%26aufirst%3DS.%2BP.%26aulast%3DRowe%26aufirst%3DT.%26aulast%3DRyan%26aufirst%3DK.%26aulast%3DSmith%26aufirst%3DG.%2BJ.%26aulast%3DSolowiej%26aufirst%3DJ.%2BE.%26aulast%3DSteppan%26aufirst%3DC.%2BM.%26aulast%3DTaggart%26aufirst%3DB.%26atitle%3DDiscovery%2520of%2520ketone-based%2520covalent%2520inhibitors%2520of%2520coronavirus%25203CL%2520proteases%2520for%2520the%2520potential%2520therapeutic%2520treatment%2520of%2520COVID-19%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D12725%26epage%3D12747%26doi%3D10.1021%2Facs.jmedchem.0c01063" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Qiao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, R.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ni, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, Z.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, Q.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, Y.-H.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, M.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, R.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, G.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, G.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, Y.-Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.</span></span> <span> </span><span class="NLM_article-title">SARS-CoV-2 M<sup>pro</sup> inhibitors with antiviral activity in a transgenic mouse model</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>371</i></span>,  <span class="NLM_fpage">1374</span>– <span class="NLM_lpage">1378</span>, <span class="refDoi"> DOI: 10.1126/science.abf1611</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=10.1126%2Fscience.abf1611" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=33602867" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A528%3ADC%252BB3MXnsVyru7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=371&publication_year=2021&pages=1374-1378&author=J.+Qiaoauthor=Y.-S.+Liauthor=R.+Zengauthor=F.-L.+Liuauthor=R.-H.+Luoauthor=C.+Huangauthor=Y.-F.+Wangauthor=J.+Zhangauthor=B.+Quanauthor=C.+Shenauthor=X.+Maoauthor=X.+Liuauthor=W.+Sunauthor=W.+Yangauthor=X.+Niauthor=K.+Wangauthor=L.+Xuauthor=Z.-L.+Duanauthor=Q.-C.+Zouauthor=H.-L.+Zhangauthor=W.+Quauthor=Y.-H.-P.+Longauthor=M.-H.+Liauthor=R.-C.+Yangauthor=X.+Liuauthor=J.+Youauthor=Y.+Zhouauthor=R.+Yaoauthor=W.-P.+Liauthor=J.-M.+Liuauthor=P.+Chenauthor=Y.+Liuauthor=G.-F.+Linauthor=X.+Yangauthor=J.+Zouauthor=L.+Liauthor=Y.+Huauthor=G.-W.+Luauthor=W.-M.+Liauthor=Y.-Q.+Weiauthor=Y.-T.+Zhengauthor=J.+Leiauthor=S.+Yang&title=SARS-CoV-2+Mpro+inhibitors+with+antiviral+activity+in+a+transgenic+mouse+model&doi=10.1126%2Fscience.abf1611"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">SARS-CoV-2 Mpro inhibitors with antiviral activity in a transgenic mouse model</span></div><div class="casAuthors">Qiao, Jingxin; Li, Yue-Shan; Zeng, Rui; Liu, Feng-Liang; Luo, Rong-Hua; Huang, Chong; Wang, Yi-Fei; Zhang, Jie; Quan, Baoxue; Shen, Chenjian; Mao, Xin; Liu, Xinlei; Sun, Weining; Yang, Wei; Ni, Xincheng; Wang, Kai; Xu, Ling; Duan, Zi-Lei; Zou, Qing-Cui; Zhang, Hai-Lin; Qu, Wang; Long, Yang-Hao-Peng; Li, Ming-Hua; Yang, Rui-Cheng; Liu, Xiaolong; You, Jing; Zhou, Yangli; Yao, Rui; Li, Wen-Pei; Liu, Jing-Ming; Chen, Pei; Liu, Yang; Lin, Gui-Feng; Yang, Xin; Zou, Jun; Li, Linli; Hu, Yiguo; Lu, Guang-Wen; Li, Wei-Min; Wei, Yu-Quan; Zheng, Yong-Tang; Lei, Jian; Yang, Shengyong</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">371</span>
        (<span class="NLM_cas:issue">6536</span>),
    <span class="NLM_cas:pages">1374-1378</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">1095-9203</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The COVID-19 pandemic caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) continually poses serious threats to global public health.  The main protease (Mpro) of SARS-CoV-2 plays a central role in viral replication.  We designed and synthesized 32 new bicycloproline-contg. Mpro inhibitors derived from either boceprevir or telaprevir, both of which are approved antivirals.  All compds. inhibited SARS-CoV-2 Mpro activity in vitro, with 50% inhibitory concn. values ranging from 7.6 to 748.5 nM.  The cocrystal structure of Mpro in complex with MI-23, one of the most potent compds., revealed its interaction mode.  Two compds. (MI-09 and MI-30) showed excellent antiviral activity in cell-based assays.  In a transgenic mouse model of SARS-CoV-2 infection, oral or i.p. treatment with MI-09 or MI-30 significantly reduced lung viral loads and lung lesions.  Both also displayed good pharmacokinetic properties and safety in rats.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVMv8s8s-oWLVg90H21EOLACvtfcHk0lj1FhhQ3ABagg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXnsVyru7g%253D&md5=555937b1776c8353e59601d567e2ebb9</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1126%2Fscience.abf1611&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.abf1611%26sid%3Dliteratum%253Aachs%26aulast%3DQiao%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DY.-S.%26aulast%3DZeng%26aufirst%3DR.%26aulast%3DLiu%26aufirst%3DF.-L.%26aulast%3DLuo%26aufirst%3DR.-H.%26aulast%3DHuang%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DY.-F.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DQuan%26aufirst%3DB.%26aulast%3DShen%26aufirst%3DC.%26aulast%3DMao%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DSun%26aufirst%3DW.%26aulast%3DYang%26aufirst%3DW.%26aulast%3DNi%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DK.%26aulast%3DXu%26aufirst%3DL.%26aulast%3DDuan%26aufirst%3DZ.-L.%26aulast%3DZou%26aufirst%3DQ.-C.%26aulast%3DZhang%26aufirst%3DH.-L.%26aulast%3DQu%26aufirst%3DW.%26aulast%3DLong%26aufirst%3DY.-H.-P.%26aulast%3DLi%26aufirst%3DM.-H.%26aulast%3DYang%26aufirst%3DR.-C.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DYou%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DYao%26aufirst%3DR.%26aulast%3DLi%26aufirst%3DW.-P.%26aulast%3DLiu%26aufirst%3DJ.-M.%26aulast%3DChen%26aufirst%3DP.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DLin%26aufirst%3DG.-F.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DZou%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DLu%26aufirst%3DG.-W.%26aulast%3DLi%26aufirst%3DW.-M.%26aulast%3DWei%26aufirst%3DY.-Q.%26aulast%3DZheng%26aufirst%3DY.-T.%26aulast%3DLei%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DS.%26atitle%3DSARS-CoV-2%2520Mpro%2520inhibitors%2520with%2520antiviral%2520activity%2520in%2520a%2520transgenic%2520mouse%2520model%26jtitle%3DScience%26date%3D2021%26volume%3D371%26spage%3D1374%26epage%3D1378%26doi%3D10.1126%2Fscience.abf1611" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schechter, I.</span></span> <span> </span><span class="NLM_article-title">Reprint of “on the size of the active site in proteases. I. Papain”</span>. <i>Biochem. Biophys. Res. Com.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>425</i></span>,  <span class="NLM_fpage">497</span>– <span class="NLM_lpage">502</span>, <span class="refDoi"> DOI: 10.1016/j.bbrc.2012.08.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=10.1016%2Fj.bbrc.2012.08.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=22925665" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=425&publication_year=2012&pages=497-502&author=I.+Schechter&title=Reprint+of+%E2%80%9Con+the+size+of+the+active+site+in+proteases.+I.+Papain%E2%80%9D&doi=10.1016%2Fj.bbrc.2012.08.015"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2012.08.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2012.08.015%26sid%3Dliteratum%253Aachs%26aulast%3DSchechter%26aufirst%3DI.%26atitle%3DReprint%2520of%2520%25E2%2580%259Con%2520the%2520size%2520of%2520the%2520active%2520site%2520in%2520proteases.%2520I.%2520Papain%25E2%2580%259D%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Com.%26date%3D2012%26volume%3D425%26spage%3D497%26epage%3D502%26doi%3D10.1016%2Fj.bbrc.2012.08.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="note"><p class="first last">. The nomenclature used is that of Schechter, I. and Berger, A., where the residues on the N-terminus side of the peptide bond that is cleaved are designated as P<sub>1</sub>–P<sub>n</sub> and those on the C-terminus side are designated P<sub>1</sub>–P<sub>1</sub>′. The corresponding active-site subsites are designated S<sub>1</sub>–S<sub>n</sub> and S<sub>1</sub>–S<sub>n</sub>′. S<sub>1</sub> is the primary substrate specificity subsite, and P<sub>1</sub>–P<sub>1</sub>′ is the scissile bond</p></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rathnayake, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perera, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mackin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyerholz, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kashipathy, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Battaile, K. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lovell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perlman, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groutas, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, K.-O.</span></span> <span> </span><span class="NLM_article-title">3C-like protease inhibitors block coronavirus replication in vitro and improve survival in MERS-CoV-infected mice</span>. <i>Sci. Transl. Med.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>12</i></span>, <span class="NLM_elocation-id">eabc5332</span> <span class="refDoi"> DOI: 10.1126/scitranslmed.abc5332</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=10.1126%2Fscitranslmed.abc5332" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=32747425" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2020&author=A.+D.+Rathnayakeauthor=J.+Zhengauthor=Y.+Kimauthor=K.+D.+Pereraauthor=S.+Mackinauthor=D.+K.+Meyerholzauthor=M.+M.+Kashipathyauthor=K.+P.+Battaileauthor=S.+Lovellauthor=S.+Perlmanauthor=W.+C.+Groutasauthor=K.-O.+Chang&title=3C-like+protease+inhibitors+block+coronavirus+replication+in+vitro+and+improve+survival+in+MERS-CoV-infected+mice&doi=10.1126%2Fscitranslmed.abc5332"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1126%2Fscitranslmed.abc5332&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscitranslmed.abc5332%26sid%3Dliteratum%253Aachs%26aulast%3DRathnayake%26aufirst%3DA.%2BD.%26aulast%3DZheng%26aufirst%3DJ.%26aulast%3DKim%26aufirst%3DY.%26aulast%3DPerera%26aufirst%3DK.%2BD.%26aulast%3DMackin%26aufirst%3DS.%26aulast%3DMeyerholz%26aufirst%3DD.%2BK.%26aulast%3DKashipathy%26aufirst%3DM.%2BM.%26aulast%3DBattaile%26aufirst%3DK.%2BP.%26aulast%3DLovell%26aufirst%3DS.%26aulast%3DPerlman%26aufirst%3DS.%26aulast%3DGroutas%26aufirst%3DW.%2BC.%26aulast%3DChang%26aufirst%3DK.-O.%26atitle%3D3C-like%2520protease%2520inhibitors%2520block%2520coronavirus%2520replication%2520in%2520vitro%2520and%2520improve%2520survival%2520in%2520MERS-CoV-infected%2520mice%26jtitle%3DSci.%2520Transl.%2520Med.%26date%3D2020%26volume%3D12%26doi%3D10.1126%2Fscitranslmed.abc5332" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galasiti
Kankanamalage, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weerasekara, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hua, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groutas, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, K.-O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pedersen, N. C.</span></span> <span> </span><span class="NLM_article-title">Reversal of the progression of fatal coronavirus infection in cats by a broad-spectrum coronavirus protease inhibitor</span>. <i>PLoS Pathog.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>, <span class="NLM_elocation-id">e1005531</span> <span class="refDoi"> DOI: 10.1371/journal.ppat.1005531</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=10.1371%2Fjournal.ppat.1005531" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=27027316" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlGru7fN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&author=Y.+Kimauthor=H.+Liuauthor=A.+C.+Galasiti%0AKankanamalageauthor=S.+Weerasekaraauthor=D.+H.+Huaauthor=W.+C.+Groutasauthor=K.-O.+Changauthor=N.+C.+Pedersen&title=Reversal+of+the+progression+of+fatal+coronavirus+infection+in+cats+by+a+broad-spectrum+coronavirus+protease+inhibitor&doi=10.1371%2Fjournal.ppat.1005531"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Reversal of the progression of fatal coronavirus infection in cats by a broad-spectrum coronavirus protease inhibitor</span></div><div class="casAuthors">Kim, Yunjeong; Liu, Hongwei; Kankanamalage, Anushka C. Galasiti; Weerasekara, Sahani; Hua, Duy H.; Groutas, William C.; Chang, Kyeong-Ok; Pedersen, Niels C.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Pathogens</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">e1005531/1-e1005531/18</span>CODEN:
                <span class="NLM_cas:coden">PPLACN</span>;
        ISSN:<span class="NLM_cas:issn">1553-7374</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Coronaviruses infect animals and humans causing a wide range of diseases.  The diversity of coronaviruses in many mammalian species is contributed by relatively high mutation and recombination rates during replication.  This dynamic nature of coronaviruses may facilitate cross-species transmission and shifts in tissue or cell tropism in a host, resulting in substantial change in virulence.  Feline enteric coronavirus (FECV) causes inapparent or mild enteritis in cats, but a highly fatal disease, called feline infectious peritonitis (FIP), can arise through mutation of FECV to FIP virus (FIPV).  The pathogenesis of FIP is intimately assocd. with immune responses and involves depletion of T cells, features shared by some other coronaviruses like Severe Acute Respiratory Syndrome Coronavirus.  The increasing risks of highly virulent coronavirus infections in humans or animals call for effective antiviral drugs, but no such measures are yet available.  Previously, we have reported the inhibitors that target 3C-like protease (3CLpro) with broad-spectrum activity against important human and animal coronaviruses.  Here, we evaluated the therapeutic efficacy of our 3CLpro inhibitor in lab. cats with FIP.  Exptl. FIP is 100% fatal once certain clin. and lab. signs become apparent.We found that antiviral treatment led to full recovery of cats when treatment was started at a stage of disease that would be otherwise fatal if left untreated.  Antiviral treatment was assocd. with a rapid improvement in fever, ascites, lymphopenia and gross signs of illness and cats returned to normal health within 20 days or less of treatment.  Significant redn. in viral titers was also obsd. in cats.  These results indicate that continuous virus replication is required for progression of immune-mediated inflammatory disease of FIP.  These findings may provide important insights into devising therapeutic strategies and selection of antiviral compds. for further development for important coronaviruses in animals and humans.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprSH3BnhFwZLVg90H21EOLACvtfcHk0ljILJJtR9reqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlGru7fN&md5=7900b2718caeaf528f10f553e0e8fa16</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1371%2Fjournal.ppat.1005531&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.ppat.1005531%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DGalasiti%2BKankanamalage%26aufirst%3DA.%2BC.%26aulast%3DWeerasekara%26aufirst%3DS.%26aulast%3DHua%26aufirst%3DD.%2BH.%26aulast%3DGroutas%26aufirst%3DW.%2BC.%26aulast%3DChang%26aufirst%3DK.-O.%26aulast%3DPedersen%26aufirst%3DN.%2BC.%26atitle%3DReversal%2520of%2520the%2520progression%2520of%2520fatal%2520coronavirus%2520infection%2520in%2520cats%2520by%2520a%2520broad-spectrum%2520coronavirus%2520protease%2520inhibitor%26jtitle%3DPLoS%2520Pathog.%26date%3D2016%26volume%3D12%26doi%3D10.1371%2Fjournal.ppat.1005531" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pedersen, N. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galasiti Kankanamalage, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eckstrand, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groutas, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bannasch, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meadows, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, K.-O.</span></span> <span> </span><span class="NLM_article-title">Efficacy of a 3C-like protease inhibitor in treating various forms of acquired feline infectious peritonitis</span>. <i>J. Feline Med. Surg.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">378</span>– <span class="NLM_lpage">392</span>, <span class="refDoi"> DOI: 10.1177/1098612x17729626</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=10.1177%2F1098612x17729626" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=28901812" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A280%3ADC%252BC1cbns12gsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2018&pages=378-392&author=N.+C.+Pedersenauthor=Y.+Kimauthor=H.+Liuauthor=A.+C.+Galasiti+Kankanamalageauthor=C.+Eckstrandauthor=W.+C.+Groutasauthor=M.+Bannaschauthor=J.+M.+Meadowsauthor=K.-O.+Chang&title=Efficacy+of+a+3C-like+protease+inhibitor+in+treating+various+forms+of+acquired+feline+infectious+peritonitis&doi=10.1177%2F1098612x17729626"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Efficacy of a 3C-like protease inhibitor in treating various forms of acquired feline infectious peritonitis</span></div><div class="casAuthors">Pedersen Niels C; Liu Hongwei; Bannasch Michael; Kim Yunjeong; Chang Kyeong-Ok; Galasiti Kankanamalage Anushka C; Groutas William C; Eckstrand Chrissy; Meadows Juliana M</div><div class="citationInfo"><span class="NLM_cas:title">Journal of feline medicine and surgery</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">378-392</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Objectives The safety and efficacy of the 3C-like protease inhibitor GC376 was tested on a cohort of client-owned cats with various forms of feline infectious peritonitis (FIP).  Methods Twenty cats from 3.3-82 months of age (mean 10.4 months) with various forms of FIP were accepted into a field trial.  Fourteen cats presented with wet or dry-to-wet FIP and six cats presented with dry FIP.  GC376 was administered subcutaneously every 12 h at a dose of 15 mg/kg.  Cats with neurologic signs were excluded from the study.  Results Nineteen of 20 cats treated with GC376 regained outward health within 2 weeks of initial treatment.  However, disease signs recurred 1-7 weeks after primary treatment and relapses and new cases were ultimately treated for a minimum of 12 weeks.  Relapses no longer responsive to treatment occurred in 13 of these 19 cats within 1-7 weeks of initial or repeat treatment(s).  Severe neurologic disease occurred in 8/13 cats that failed treatment and five cats had recurrences of abdominal lesions.  At the time of writing, seven cats were in disease remission.  Five kittens aged 3.3-4.4 months with wet FIP were treated for 12 weeks and have been in disease remission after stopping treatment and at the time of writing for 5-14 months (mean 11.2 months).  A sixth kitten was in remission for 10 weeks after 12 weeks of treatment, relapsed and is responding to a second round of GC376.  The seventh was a 6.8-year-old cat with only mesenteric lymph node involvement that went into remission after three relapses that required progressively longer repeat treatments over a 10 month period.  Side effects of treatment included transient stinging upon injection and occasional foci of subcutaneous fibrosis and hair loss.  There was retarded development and abnormal eruption of permanent teeth in cats treated before 16-18 weeks of age.  Conclusions and relevance GC376 showed promise in treating cats with certain presentations of FIP and has opened the door to targeted antiviral drug therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTFFwhHX-Ibonqj_yg7twvpfW6udTcc2ea2iL3jxenYnrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cbns12gsg%253D%253D&md5=69b4ebebb24b351bfb0f409e275b3153</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1177%2F1098612x17729626&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1098612x17729626%26sid%3Dliteratum%253Aachs%26aulast%3DPedersen%26aufirst%3DN.%2BC.%26aulast%3DKim%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DGalasiti%2BKankanamalage%26aufirst%3DA.%2BC.%26aulast%3DEckstrand%26aufirst%3DC.%26aulast%3DGroutas%26aufirst%3DW.%2BC.%26aulast%3DBannasch%26aufirst%3DM.%26aulast%3DMeadows%26aufirst%3DJ.%2BM.%26aulast%3DChang%26aufirst%3DK.-O.%26atitle%3DEfficacy%2520of%2520a%25203C-like%2520protease%2520inhibitor%2520in%2520treating%2520various%2520forms%2520of%2520acquired%2520feline%2520infectious%2520peritonitis%26jtitle%3DJ.%2520Feline%2520Med.%2520Surg.%26date%3D2018%26volume%3D20%26spage%3D378%26epage%3D392%26doi%3D10.1177%2F1098612x17729626" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pirali, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serafini, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cargnin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Genazzani, A. A.</span></span> <span> </span><span class="NLM_article-title">Applications of deuterium in medicinal chemistry</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">5276</span>– <span class="NLM_lpage">5297</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01808</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01808" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A528%3ADC%252BC1MXpvFKjtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=5276-5297&author=T.+Piraliauthor=M.+Serafiniauthor=S.+Cargninauthor=A.+A.+Genazzani&title=Applications+of+deuterium+in+medicinal+chemistry&doi=10.1021%2Facs.jmedchem.8b01808"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Applications of deuterium in medicinal chemistry</span></div><div class="casAuthors">Pirali, Tracey; Serafini, Marta; Cargnin, Sarah; Genazzani, Armando A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">5276-5297</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The use of deuteration in medicinal chem. has exploded in the past years, and the FDA has recently approved the first deuterium-labeled drug.  Precision deuteration goes beyond the pure and simple amelioration of the pharmacokinetic parameters of a drug and might provide an opportunity when facing problems in terms of metab.-mediated toxicity, drug interactions, and low bioactivation.  The use of deuterium is even broader, offering the opportunity to lower the degree of epimerization, reduce the dose of coadministered boosters, and discover compds. where deuterium is the basis for the mechanism of action.  Nevertheless, designing, synthesizing, and developing a successful deuterated drug is far from straightforward, and the translation from concept to practice is often unpredictable.  This Perspective provides an overview of the recent developments of deuteration, with a focus on deuterated clin. candidates, and highlights both opportunities and challenges of this strategy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1ICy0AH9zGLVg90H21EOLACvtfcHk0ljmKbV0PLMU-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXpvFKjtw%253D%253D&md5=1ce58dbff268894dc319bfc00a1ae01f</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01808&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01808%26sid%3Dliteratum%253Aachs%26aulast%3DPirali%26aufirst%3DT.%26aulast%3DSerafini%26aufirst%3DM.%26aulast%3DCargnin%26aufirst%3DS.%26aulast%3DGenazzani%26aufirst%3DA.%2BA.%26atitle%3DApplications%2520of%2520deuterium%2520in%2520medicinal%2520chemistry%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D5276%26epage%3D5297%26doi%3D10.1021%2Facs.jmedchem.8b01808" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Timmins, G. S.</span></span> <span> </span><span class="NLM_article-title">Deuterated drugs; updates and obviousness analysis</span>. <i>Expert Opin. Ther. Pat.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">1353</span>– <span class="NLM_lpage">1361</span>, <span class="refDoi"> DOI: 10.1080/13543776.2017.1378350</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=10.1080%2F13543776.2017.1378350" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=28885861" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFSmtbjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2017&pages=1353-1361&author=G.+S.+Timmins&title=Deuterated+drugs%3B+updates+and+obviousness+analysis&doi=10.1080%2F13543776.2017.1378350"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Deuterated drugs; updates and obviousness analysis</span></div><div class="casAuthors">Timmins, Graham S.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Patents</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1353-1361</span>CODEN:
                <span class="NLM_cas:coden">EOTPEG</span>;
        ISSN:<span class="NLM_cas:issn">1354-3776</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">The pharmacokinetics and/or toxicity of many known drugs can be modified by selective deuteration, an area of significant com. interest and scientific and regulatory progress.  This review firstly discusses recent developments in deuterated drugs including the FDA approval of deutetrabenazine.  Secondly, it discusses 35 U.S.C. Sec. 103 'obviousness' as it relates to recent patent prosecution, and also to Inter Partes Review (IPR).  IPR is a new post-award review of patentability under Sec. 102 or Sec. 103, two IPR petitions upon deuterated drugs have been instituted and included Sec. 103 arguments.  Finally, an extended anal. of Sec. 103 obviousness based upon the practices of major pharmaceutical companies is provided, that supports rather late priority dates, while Sec. 102 is also discussed.  The total value of transactions involving deuterated drugs is close to $5 billion.  While the importance of Sec. 103 'obviousness' rejections remains in patent applications under current prosecution, IPR of issued patents is developing and will affect likely affect Sec. 103 interpretations in this area.  However, patents are still issuing with later priority dates, and further litigation will likely occur.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplgaJXt_pSD7Vg90H21EOLACvtfcHk0ljmKbV0PLMU-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFSmtbjO&md5=d29367a0daec2c70367a58327fbccd9c</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1080%2F13543776.2017.1378350&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F13543776.2017.1378350%26sid%3Dliteratum%253Aachs%26aulast%3DTimmins%26aufirst%3DG.%2BS.%26atitle%3DDeuterated%2520drugs%253B%2520updates%2520and%2520obviousness%2520analysis%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Pat.%26date%3D2017%26volume%3D27%26spage%3D1353%26epage%3D1361%26doi%3D10.1080%2F13543776.2017.1378350" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harbeson, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brummel, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tung, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silverman, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doller, D.</span></span> <span> </span><span class="NLM_article-title">A decade of deuteration in medicinal chemistry</span>. <i>Annu. Rep. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">519</span>– <span class="NLM_lpage">542</span>, <span class="refDoi"> DOI: 10.1016/bs.armc.2017.08.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=10.1016%2Fbs.armc.2017.08.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFOlsrvF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2017&pages=519-542&author=J.+F.+Liuauthor=S.+L.+Harbesonauthor=C.+L.+Brummelauthor=R.+Tungauthor=R.+Silvermanauthor=D.+Doller&title=A+decade+of+deuteration+in+medicinal+chemistry&doi=10.1016%2Fbs.armc.2017.08.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">A decade of deuteration in medicinal chemistry</span></div><div class="casAuthors">Liu, Julie F.; Harbeson, Scott L.; Brummel, Christopher L.; Tung, Roger; Silverman, Robert; Doller, Dario</div><div class="citationInfo"><span class="NLM_cas:title">Annual Reports in Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">519-542</span>CODEN:
                <span class="NLM_cas:coden">ARMCBI</span>;
        ISSN:<span class="NLM_cas:issn">0065-7743</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  The use of deuterium to drive DIEs and favorably alter pharmacokinetics and metab. in drug discovery has accelerated in the last decade.  The first major wave of patent applications came from small biotechnol. organizations and were aimed at improving pharmacokinetic properties of marketed drugs or enhancing properties offailed drug candidates.  A substantial no. of patent applications were published beginning in 2006, and the deuterium chem. patent estate continues to expand.  Deuterated drug candidates have progressed from early patent filings to successful proof-of-concept human studies, including the first FDA-approved deuterated drug.  The use of DIEs in drug design has expanded, and now deuteration of drug candidates and tool compds., including imaging ligands, has become a welcome instrument in the arsenal of medicinal chemists.  It is expected that experimentation with strategic introduction of deuterium into the structures of org. compds. will continue to increase and may enable the future development ofa variety ofnovel drugs and chem. tools.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRWRQHDkKt5LVg90H21EOLACvtfcHk0ljmKbV0PLMU-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFOlsrvF&md5=18499ed5ccfb021913c4dea9e51946b7</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fbs.armc.2017.08.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fbs.armc.2017.08.010%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DJ.%2BF.%26aulast%3DHarbeson%26aufirst%3DS.%2BL.%26aulast%3DBrummel%26aufirst%3DC.%2BL.%26aulast%3DTung%26aufirst%3DR.%26aulast%3DSilverman%26aufirst%3DR.%26aulast%3DDoller%26aufirst%3DD.%26atitle%3DA%2520decade%2520of%2520deuteration%2520in%2520medicinal%2520chemistry%26jtitle%3DAnnu.%2520Rep.%2520Med.%2520Chem.%26date%3D2017%26volume%3D50%26spage%3D519%26epage%3D542%26doi%3D10.1016%2Fbs.armc.2017.08.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doung, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKee, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, W. J.</span></span> <span> </span><span class="NLM_article-title">N,N-dissuccinimidyl carbonate: a useful reagent for alkoxycarbonylation of amines</span>. <i>Tetrahedron Lett.</i> <span class="NLM_year" style="font-weight: bold;">1992</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">2781</span>– <span class="NLM_lpage">2784</span>, <span class="refDoi"> DOI: 10.1016/s0040-4039(00)78856-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=10.1016%2Fs0040-4039%2800%2978856-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=30310243" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A528%3ADyaK38XkvFSlsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=1992&pages=2781-2784&author=A.+K.+Ghoshauthor=T.+T.+Doungauthor=S.+P.+McKeeauthor=W.+J.+Thompson&title=N%2CN-dissuccinimidyl+carbonate%3A+a+useful+reagent+for+alkoxycarbonylation+of+amines&doi=10.1016%2Fs0040-4039%2800%2978856-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">N,N'-Disuccinimidyl carbonate:  a useful reagent for alkoxycarbonylation of amines</span></div><div class="casAuthors">Ghosh, Arun K.; Duong, Tien T.; McKee, Sean P.; Thompson, Wayne J.</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron Letters</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">2781-4</span>CODEN:
                <span class="NLM_cas:coden">TELEAY</span>;
        ISSN:<span class="NLM_cas:issn">0040-4039</span>.
    </div><div class="casAbstract">An efficient and mild method for alkoxycarbonylation of amines is described, utilizing com. available N,N'-disuccinimidyl carbonate (I).  Transesterification of I with 1,2-O-isopropylidene-D-xylofuranose gave the activated ester II which upon treatment with amines gave carbamates, e.g. III.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpR1G8uUUw3crVg90H21EOLACvtfcHk0lgJ-pjmcN6u8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38XkvFSlsrY%253D&md5=167660e9aebebc06a9020024d50825f6</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fs0040-4039%2800%2978856-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fs0040-4039%252800%252978856-3%26sid%3Dliteratum%253Aachs%26aulast%3DGhosh%26aufirst%3DA.%2BK.%26aulast%3DDoung%26aufirst%3DT.%2BT.%26aulast%3DMcKee%26aufirst%3DS.%2BP.%26aulast%3DThompson%26aufirst%3DW.%2BJ.%26atitle%3DN%252CN-dissuccinimidyl%2520carbonate%253A%2520a%2520useful%2520reagent%2520for%2520alkoxycarbonylation%2520of%2520amines%26jtitle%3DTetrahedron%2520Lett.%26date%3D1992%26volume%3D33%26spage%3D2781%26epage%3D2784%26doi%3D10.1016%2Fs0040-4039%2800%2978856-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kjell, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slattery, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Semo, M. J.</span></span> <span> </span><span class="NLM_article-title">A novel, nonaqueous method for regeneration of aldehydes from bisulfite adducts</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>64</i></span>,  <span class="NLM_fpage">5722</span>– <span class="NLM_lpage">5724</span>, <span class="refDoi"> DOI: 10.1021/jo990543v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo990543v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A528%3ADyaK1MXktFSkur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=1999&pages=5722-5724&author=D.+P.+Kjellauthor=B.+J.+Slatteryauthor=M.+J.+Semo&title=A+novel%2C+nonaqueous+method+for+regeneration+of+aldehydes+from+bisulfite+adducts&doi=10.1021%2Fjo990543v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">A Novel, Nonaqueous Method for Regeneration of Aldehydes from Bisulfite Adducts</span></div><div class="casAuthors">Kjell, Douglas P.; Slattery, Brian J.; Semo, Michael J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">5722-5724</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Aldehydes have been generated from their bisulfite adducts under nonaq. conditions by treatment of the bisulfite addn. compd. with excess trimethylsilyl chloride in acetonitrile to give the aldehydes in 95-100% yield.  E.g., in the prepn. of an oncolytic agent, intermediate aldehyde I, prepd. by the palladium-catalyzed coupling of Me 4-iodobenzoate with 3-buten-1-ol, and susceptible to ester hydrolysis in the presence of water and base, is purified from a reaction mixt. by treatment with sodium bisulfite in ethanol-water to give the bisulfite addn. compd. II; treatment of II with >2 equiv. Me3SiCl in acetonitrile and heating for 2h regenerated I in 90% yield from II.  E.g., Et formylcyclopropanecarboxylate III (as an 88:12 mixt. with the cis diastereomer) was treated with sodium bisulfite to give the bisulfite adduct IV in 61% yield as a single diastereomer; treatment of IV with >2 equiv. Me3SiCl in MeCN gave III in 98% yield.  Acetonitrile was the only effective solvent; methylene chloride, Et acetate, and THF give incomplete reaction, while more polar alc. solvents, DMF and DMSO all react with Me3SiCl and are incompatible.  Other trimethylsilyl compds., such as 1-trimethylsilylimidazole and N,O-bis(trimethylsilyl)acetamide, are also effective in the generation of aldehydes from bisulfite addn. compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRzgg7pFpbpLVg90H21EOLACvtfcHk0lgJ-pjmcN6u8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXktFSkur8%253D&md5=30e4e9dd7a6b8ef13427bfc1050f8fa5</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fjo990543v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo990543v%26sid%3Dliteratum%253Aachs%26aulast%3DKjell%26aufirst%3DD.%2BP.%26aulast%3DSlattery%26aufirst%3DB.%2BJ.%26aulast%3DSemo%26aufirst%3DM.%2BJ.%26atitle%3DA%2520novel%252C%2520nonaqueous%2520method%2520for%2520regeneration%2520of%2520aldehydes%2520from%2520bisulfite%2520adducts%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D1999%26volume%3D64%26spage%3D5722%26epage%3D5724%26doi%3D10.1021%2Fjo990543v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lovell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tiew, K.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mandadapu, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alliston, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Battaile, K. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groutas, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, K.-O.</span></span> <span> </span><span class="NLM_article-title">Broad-spectrum antivirals against 3C or 3C-like proteases of picornaviruses, noroviruses, and coronaviruses</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>86</i></span>,  <span class="NLM_fpage">11754</span>– <span class="NLM_lpage">11762</span>, <span class="refDoi"> DOI: 10.1128/jvi.01348-12</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=10.1128%2Fjvi.01348-12" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=22915796" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFCmur3M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=86&publication_year=2012&pages=11754-11762&author=Y.+Kimauthor=S.+Lovellauthor=K.-C.+Tiewauthor=S.+R.+Mandadapuauthor=K.+R.+Allistonauthor=K.+P.+Battaileauthor=W.+C.+Groutasauthor=K.-O.+Chang&title=Broad-spectrum+antivirals+against+3C+or+3C-like+proteases+of+picornaviruses%2C+noroviruses%2C+and+coronaviruses&doi=10.1128%2Fjvi.01348-12"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Broad-spectrum antivirals against 3C or 3C-like proteases of picornaviruses, noroviruses, and coronaviruses</span></div><div class="casAuthors">Kim, Yunjeong; Lovell, Scott; Tiew, Kok-Chuan; Mandadapu, Sivakoteswara Rao; Alliston, Kevin R.; Battaile, Kevin P.; Groutas, William C.; Chang, Kyeong-Ok</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">86</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">11754-11762</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Phylogenetic anal. has demonstrated that some pos.-sense RNA viruses can be classified into the picornavirus-like super-cluster, which includes picornaviruses, caliciviruses, and coronaviruses.  These viruses possess 3C or 3C-like proteases (3Cpro or 3CLpro, resp.), which contain a typical chymotrypsin-like fold and a catalytic triad (or dyad) with a Cys residue as a nucleophile.  The conserved key sites of 3Cpro or 3CLpro may serve as attractive targets for the design of broad-spectrum antivirals for multiple viruses in the supercluster.  We previously reported the structure-based design and synthesis of potent protease inhibitors of Norwalk virus (NV), a member of the Caliciviridae family.  We report herein the broad-spectrum antiviral activities of three compds. possessing a common dipeptidyl residue with different warheads, i.e., an aldehyde (GC373), a bisulfite adduct (GC376), and an α-ketoamide (GC375), against viruses that belong to the supercluster.  All compds. were highly effective against the majority of tested viruses, with half-maximal inhibitory concns. in the high nanomolar or low micromolar range in enzyme- and/or cell-based assays and with high therapeutic indexes.  We also report the high-resoln. X-ray cocrystal structures of NV 3CLpro-, poliovirus 3Cpro-, and transmissible gastroenteritis virus 3CLpro- GC376 inhibitor complexes, which show the compd. covalently bound to a nucleophilic Cys residue in the catalytic site of the corresponding protease.  We conclude that these compds. have the potential to be developed as antiviral therapeutics aimed at a single virus or multiple viruses in the picornavirus-like supercluster by targeting 3Cpro or 3CLpro.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDn4MUH7bIlLVg90H21EOLACvtfcHk0lgJ-pjmcN6u8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFCmur3M&md5=c7fe984b52490199a2c2ef333c22fb98</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1128%2Fjvi.01348-12&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252Fjvi.01348-12%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DY.%26aulast%3DLovell%26aufirst%3DS.%26aulast%3DTiew%26aufirst%3DK.-C.%26aulast%3DMandadapu%26aufirst%3DS.%2BR.%26aulast%3DAlliston%26aufirst%3DK.%2BR.%26aulast%3DBattaile%26aufirst%3DK.%2BP.%26aulast%3DGroutas%26aufirst%3DW.%2BC.%26aulast%3DChang%26aufirst%3DK.-O.%26atitle%3DBroad-spectrum%2520antivirals%2520against%25203C%2520or%25203C-like%2520proteases%2520of%2520picornaviruses%252C%2520noroviruses%252C%2520and%2520coronaviruses%26jtitle%3DJ.%2520Virol.%26date%3D2012%26volume%3D86%26spage%3D11754%26epage%3D11762%26doi%3D10.1128%2Fjvi.01348-12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gampe, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verma, V. A.</span></span> <span> </span><span class="NLM_article-title">Curse or cure? A perspective on the developability of aldehydes as active pharmaceutical ingredients</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">14357</span>– <span class="NLM_lpage">14381</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c01177</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c01177" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhvVWhs7vE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=14357-14381&author=C.+Gampeauthor=V.+A.+Verma&title=Curse+or+cure%3F+A+perspective+on+the+developability+of+aldehydes+as+active+pharmaceutical+ingredients&doi=10.1021%2Facs.jmedchem.0c01177"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Curse or Cure? A Perspective on the Developability of Aldehydes as Active Pharmaceutical Ingredients</span></div><div class="casAuthors">Gampe, Christian; Verma, Vishal A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">14357-14381</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  There is a dichotomy between the increasing utility of aldehydes as tool mols. that bind to "undruggable" protein sites and the designation of the aldehyde functional group as a structural alert by the medicinal chem. community.  Herein we compile information about pharmacol. active aldehydes that are being used in humans.  We summarize the learnings from these data, discuss advantages and challenges assocd. with aldehydes, and derive strategies for the successful development of aldehydes within drug discovery programs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppz1oqp1WGV7Vg90H21EOLACvtfcHk0ljmi_h0xdiLiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhvVWhs7vE&md5=4696123fdaba8f20895bb11f2b78ff84</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c01177&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c01177%26sid%3Dliteratum%253Aachs%26aulast%3DGampe%26aufirst%3DC.%26aulast%3DVerma%26aufirst%3DV.%2BA.%26atitle%3DCurse%2520or%2520cure%253F%2520A%2520perspective%2520on%2520the%2520developability%2520of%2520aldehydes%2520as%2520active%2520pharmaceutical%2520ingredients%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D14357%26epage%3D14381%26doi%3D10.1021%2Facs.jmedchem.0c01177" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cargnin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serafini, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pirali, T.</span></span> <span> </span><span class="NLM_article-title">A primer of deuterium in drug design</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">2039</span>– <span class="NLM_lpage">2042</span>, <span class="refDoi"> DOI: 10.4155/fmc-2019-0183</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=10.4155%2Ffmc-2019-0183" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=31538524" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvVKis7fN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2019&pages=2039-2042&author=S.+Cargninauthor=M.+Serafiniauthor=T.+Pirali&title=A+primer+of+deuterium+in+drug+design&doi=10.4155%2Ffmc-2019-0183"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">A primer of deuterium in drug design</span></div><div class="casAuthors">Cargnin, Sarah; Serafini, Marta; Pirali, Tracey</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">2039-2042</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  This article discusses about role of deuterated compounds, heavy and nonradioactive isotope of hydrogen in clinical development and effects of deuteration incorporation in drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnDF9mfxcU77Vg90H21EOLACvtfcHk0ljmi_h0xdiLiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvVKis7fN&md5=2c7c1c4a9820456566886af749d28068</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.4155%2Ffmc-2019-0183&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc-2019-0183%26sid%3Dliteratum%253Aachs%26aulast%3DCargnin%26aufirst%3DS.%26aulast%3DSerafini%26aufirst%3DM.%26aulast%3DPirali%26aufirst%3DT.%26atitle%3DA%2520primer%2520of%2520deuterium%2520in%2520drug%2520design%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2019%26volume%3D11%26spage%3D2039%26epage%3D2042%26doi%3D10.4155%2Ffmc-2019-0183" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Meanwell, N. A.</span></span> <span> </span><span class="NLM_article-title">Synopsis of some recent tactical application of bioisosteres in drug design</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">2529</span>– <span class="NLM_lpage">2591</span>, <span class="refDoi"> DOI: 10.1021/jm1013693</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm1013693" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjsVCnu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=2529-2591&author=N.+A.+Meanwell&title=Synopsis+of+some+recent+tactical+application+of+bioisosteres+in+drug+design&doi=10.1021%2Fjm1013693"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Synopsis of Some Recent Tactical Application of Bioisosteres in Drug Design</span></div><div class="casAuthors">Meanwell, Nicholas A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2529-2591</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The established utility of bioisosteres is broad in nature, extending to improving potency, enhancing selectivity, altering phys. properties, reducing or redirecting metab., eliminating or modifying toxicophores, and acquiring novel intellectual property.  In this Perspective, some contemporary themes exploring the role of isosteres in drug design are sampled, with an emphasis placed on tactical applications designed to solve the kinds of problems that impinge on compd. optimization and the long-term success of drug candidates.  Interesting concepts that may have been poorly effective in the context examd. are captured, since the ideas may have merit in alternative circumstances.  A comprehensive cataloging of bioisosteres is beyond the scope of what will be provided, although a synopsis of relevant isosteres of a particular functionality is summarized in a succinct fashion in several sections.  Isosterism has also found productive application in the design and optimization of organocatalysts, and there are several examples in which functional mimicry established initially in a medicinal chem. setting has been adopted by this community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZ9z_aPcU4ubVg90H21EOLACvtfcHk0li0kH3Rd-hJ0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjsVCnu7o%253D&md5=0637ab60c379bef535f3f709b26f3582</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fjm1013693&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm1013693%26sid%3Dliteratum%253Aachs%26aulast%3DMeanwell%26aufirst%3DN.%2BA.%26atitle%3DSynopsis%2520of%2520some%2520recent%2520tactical%2520application%2520of%2520bioisosteres%2520in%2520drug%2520design%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D2529%26epage%3D2591%26doi%3D10.1021%2Fjm1013693" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chessari, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hardcastle, I. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahn, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anil, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anscombe, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bawn, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bevan, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blackburn, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buck, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cano, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carbain, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castro, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cons, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cully, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Endicott, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Endicott, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fazal, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golding, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffin, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haggerty, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harnor, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hearn, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hobson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holvey, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jennings, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lunec, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Newell, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noble, M. E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reeks, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Revill, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riedinger, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denis, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamanini, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, N. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vinković, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wedge, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilsher, N. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span> <span> </span><span class="NLM_article-title">Structure-based design of potent and orally active isoindolinone inhibitors of MDM2-p53 protein-protein interaction</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>64</i></span>,  <span class="NLM_fpage">4071</span>– <span class="NLM_lpage">4088</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c02188</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c02188" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A528%3ADC%252BB3MXntValtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2021&pages=4071-4088&author=G.+Chessariauthor=I.+R.+Hardcastleauthor=J.+S.+Ahnauthor=B.+Anilauthor=E.+Anscombeauthor=R.+H.+Bawnauthor=L.+D.+Bevanauthor=T.+J.+Blackburnauthor=I.+Buckauthor=C.+Canoauthor=B.+Carbainauthor=J.+Castroauthor=B.+Consauthor=S.+J.+Cullyauthor=J.+A.+Endicottauthor=J.+A.+Endicottauthor=L.+Fazalauthor=B.+T.+Goldingauthor=R.+J.+Griffinauthor=K.+Haggertyauthor=S.+J.+Harnorauthor=K.+Hearnauthor=S.+Hobsonauthor=R.+S.+Holveyauthor=S.+Howardauthor=C.+E.+Jenningsauthor=C.+N.+Johnsonauthor=J.+Lunecauthor=D.+C.+Millerauthor=D.+R.+Newellauthor=M.+E.+M.+Nobleauthor=J.+Reeksauthor=C.+H.+Revillauthor=C.+Riedingerauthor=J.+D.+Denisauthor=E.+Tamaniniauthor=H.+Thomasauthor=N.+T.+Thompsonauthor=M.+Vinkovi%C4%87author=S.+R.+Wedgeauthor=P.+A.+Williamsauthor=N.+E.+Wilsherauthor=B.+Zhangauthor=Y.+Zhao&title=Structure-based+design+of+potent+and+orally+active+isoindolinone+inhibitors+of+MDM2-p53+protein-protein+interaction&doi=10.1021%2Facs.jmedchem.0c02188"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design of Potent and Orally Active Isoindolinone Inhibitors of MDM2-p53 Protein-Protein Interaction</span></div><div class="casAuthors">Chessari, Gianni; Hardcastle, Ian R.; Ahn, Jong Sook; Anil, Burcu; Anscombe, Elizabeth; Bawn, Ruth H.; Bevan, Luke D.; Blackburn, Timothy J.; Buck, Ildiko; Cano, Celine; Carbain, Benoit; Castro, Juan; Cons, Ben; Cully, Sarah J.; Endicott, Jane A.; Fazal, Lynsey; Golding, Bernard T.; Griffin, Roger J.; Haggerty, Karen; Harnor, Suzannah J.; Hearn, Keisha; Hobson, Stephen; Holvey, Rhian S.; Howard, Steven; Jennings, Claire E.; Johnson, Christopher N.; Lunec, John; Miller, Duncan C.; Newell, David R.; Noble, Martin E. M.; Reeks, Judith; Revill, Charlotte H.; Riedinger, Christiane; St. Denis, Jeffrey D.; Tamanini, Emiliano; Thomas, Huw; Thompson, Neil T.; Vinkovic, Mladen; Wedge, Stephen R.; Williams, Pamela A.; Wilsher, Nicola E.; Zhang, Bian; Zhao, Yan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">4071-4088</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Inhibition of murine double minute 2 (MDM2)-p53 protein-protein interaction with small mols. has been shown to reactivate p53 and inhibit tumor growth.  Here, we describe rational, structure-guided, design of novel isoindolinone-based MDM2 inhibitors.  MDM2 X-ray crystallog., quantum mechanics ligand-based design, and metabolite identification all contributed toward the discovery of potent in vitro and in vivo inhibitors of the MDM2-p53 interaction with representative compds. inducing cytostasis in an SJSA-1 osteosarcoma xenograft model following once-daily oral administration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdhfpb86qe6bVg90H21EOLACvtfcHk0li0kH3Rd-hJ0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXntValtr4%253D&md5=5e7cc87b8b8aa3f4359b4c3b059c9910</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c02188&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c02188%26sid%3Dliteratum%253Aachs%26aulast%3DChessari%26aufirst%3DG.%26aulast%3DHardcastle%26aufirst%3DI.%2BR.%26aulast%3DAhn%26aufirst%3DJ.%2BS.%26aulast%3DAnil%26aufirst%3DB.%26aulast%3DAnscombe%26aufirst%3DE.%26aulast%3DBawn%26aufirst%3DR.%2BH.%26aulast%3DBevan%26aufirst%3DL.%2BD.%26aulast%3DBlackburn%26aufirst%3DT.%2BJ.%26aulast%3DBuck%26aufirst%3DI.%26aulast%3DCano%26aufirst%3DC.%26aulast%3DCarbain%26aufirst%3DB.%26aulast%3DCastro%26aufirst%3DJ.%26aulast%3DCons%26aufirst%3DB.%26aulast%3DCully%26aufirst%3DS.%2BJ.%26aulast%3DEndicott%26aufirst%3DJ.%2BA.%26aulast%3DEndicott%26aufirst%3DJ.%2BA.%26aulast%3DFazal%26aufirst%3DL.%26aulast%3DGolding%26aufirst%3DB.%2BT.%26aulast%3DGriffin%26aufirst%3DR.%2BJ.%26aulast%3DHaggerty%26aufirst%3DK.%26aulast%3DHarnor%26aufirst%3DS.%2BJ.%26aulast%3DHearn%26aufirst%3DK.%26aulast%3DHobson%26aufirst%3DS.%26aulast%3DHolvey%26aufirst%3DR.%2BS.%26aulast%3DHoward%26aufirst%3DS.%26aulast%3DJennings%26aufirst%3DC.%2BE.%26aulast%3DJohnson%26aufirst%3DC.%2BN.%26aulast%3DLunec%26aufirst%3DJ.%26aulast%3DMiller%26aufirst%3DD.%2BC.%26aulast%3DNewell%26aufirst%3DD.%2BR.%26aulast%3DNoble%26aufirst%3DM.%2BE.%2BM.%26aulast%3DReeks%26aufirst%3DJ.%26aulast%3DRevill%26aufirst%3DC.%2BH.%26aulast%3DRiedinger%26aufirst%3DC.%26aulast%3DDenis%26aufirst%3DJ.%2BD.%26aulast%3DTamanini%26aufirst%3DE.%26aulast%3DThomas%26aufirst%3DH.%26aulast%3DThompson%26aufirst%3DN.%2BT.%26aulast%3DVinkovi%25C4%2587%26aufirst%3DM.%26aulast%3DWedge%26aufirst%3DS.%2BR.%26aulast%3DWilliams%26aufirst%3DP.%2BA.%26aulast%3DWilsher%26aufirst%3DN.%2BE.%26aulast%3DZhang%26aufirst%3DB.%26aulast%3DZhao%26aufirst%3DY.%26atitle%3DStructure-based%2520design%2520of%2520potent%2520and%2520orally%2520active%2520isoindolinone%2520inhibitors%2520of%2520MDM2-p53%2520protein-protein%2520interaction%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2021%26volume%3D64%26spage%3D4071%26epage%3D4088%26doi%3D10.1021%2Facs.jmedchem.0c02188" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">Automatic indexing of rotation diffraction patterns</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1988</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">67</span>– <span class="NLM_lpage">72</span>, <span class="refDoi"> DOI: 10.1107/s0021889887009737</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=10.1107%2Fs0021889887009737" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A528%3ADyaL1cXhtVCgsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=1988&pages=67-72&author=W.+Kabsch&title=Automatic+indexing+of+rotation+diffraction+patterns&doi=10.1107%2Fs0021889887009737"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Automatic indexing of rotation diffraction patterns</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">67-71</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    </div><div class="casAbstract">A method is described which assigns indexes to a set of single-crystal reflections recorded by the rotation-oscillation technique using a fixed x-ray wavelength.  The space group and approx. unit-cell parameters are known.  The unknown crystal orientation is detd. directly from the obsd. diffraction pattern of one or several oscillation data records.  A local indexing procedure is described which allows for large initial errors in the parameters controlling the diffraction pattern.  These parameters are refined subsequently, thereby satisfying the constraints imposed by the space-group symmetry.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9WshttyE38LVg90H21EOLACvtfcHk0li0kH3Rd-hJ0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1cXhtVCgsb8%253D&md5=268552b3f6069d2465b3916a985b840f</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1107%2Fs0021889887009737&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs0021889887009737%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DAutomatic%2520indexing%2520of%2520rotation%2520diffraction%2520patterns%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D1988%26volume%3D21%26spage%3D67%26epage%3D72%26doi%3D10.1107%2Fs0021889887009737" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">XDS</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1107/s0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=XDS&doi=10.1107%2Fs0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0lgFanA6Z_Ll5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXDS%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2Fs0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vonrhein, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flensburg, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keller, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharff, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smart, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paciorek, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Womack, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bricogne, G.</span></span> <span> </span><span class="NLM_article-title">Data processing and analysis with theautoPROCtoolbox</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">293</span>– <span class="NLM_lpage">302</span>, <span class="refDoi"> DOI: 10.1107/s0907444911007773</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=10.1107%2Fs0907444911007773" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=21460447" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=293-302&author=C.+Vonrheinauthor=C.+Flensburgauthor=P.+Kellerauthor=A.+Sharffauthor=O.+Smartauthor=W.+Paciorekauthor=T.+Womackauthor=G.+Bricogne&title=Data+processing+and+analysis+with+theautoPROCtoolbox&doi=10.1107%2Fs0907444911007773"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Data processing and analysis with the autoPROC toolbox</span></div><div class="casAuthors">Vonrhein, Clemens; Flensburg, Claus; Keller, Peter; Sharff, Andrew; Smart, Oliver; Paciorek, Wlodek; Womack, Thomas; Bricogne, Gerard</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">293-302</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A typical diffraction expt. will generate many images and data sets from different crystals in a very short time.  This creates a challenge for the high-throughput operation of modern synchrotron beamlines as well as for the subsequent data processing.  Novice users in particular may feel overwhelmed by the tables, plots and nos. that the different data-processing programs and software packages present to them.  Here, some of the more common problems that a user has to deal with when processing a set of images that will finally make up a processed data set are shown, concg. on difficulties that may often show up during the first steps along the path of turning the expt. (i.e. data collection) into a model (i.e. interpreted electron d.).  Difficulties such as unexpected crystal forms, issues in crystal handling and suboptimal choices of data-collection strategies can often be dealt with, or at least diagnosed, by analyzing specific data characteristics during processing.  In the end, one wants to distinguish problems over which one has no immediate control once the expt. is finished from problems that can be remedied a posteriori.  A new software package, autoPROC, is also presented that combines third-party processing programs with new tools and an automated workflow script that is intended to provide users with both guidance and insight into the offline processing of data affected by the difficulties mentioned above, with particular emphasis on the automated treatment of multi-sweep data sets collected on multi-axis goniostats.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmah7iUrMLXLVg90H21EOLACvtfcHk0lgFanA6Z_Ll5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLs%253D&md5=a0a3579f6b371b6312819883864aa80d</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1107%2Fs0907444911007773&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs0907444911007773%26sid%3Dliteratum%253Aachs%26aulast%3DVonrhein%26aufirst%3DC.%26aulast%3DFlensburg%26aufirst%3DC.%26aulast%3DKeller%26aufirst%3DP.%26aulast%3DSharff%26aufirst%3DA.%26aulast%3DSmart%26aufirst%3DO.%26aulast%3DPaciorek%26aufirst%3DW.%26aulast%3DWomack%26aufirst%3DT.%26aulast%3DBricogne%26aufirst%3DG.%26atitle%3DData%2520processing%2520and%2520analysis%2520with%2520theautoPROCtoolbox%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D293%26epage%3D302%26doi%3D10.1107%2Fs0907444911007773" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span> <span> </span><span class="NLM_article-title">An introduction to data reduction: space-group determination, scaling and intensity statistics</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">282</span>– <span class="NLM_lpage">292</span>, <span class="refDoi"> DOI: 10.1107/s090744491003982x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=10.1107%2Fs090744491003982x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=21460446" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=282-292&author=P.+R.+Evans&title=An+introduction+to+data+reduction%3A+space-group+determination%2C+scaling+and+intensity+statistics&doi=10.1107%2Fs090744491003982x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">An introduction to data reduction: Space-group determination, scaling and intensity statistics</span></div><div class="casAuthors">Evans, Philip R.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">282-292</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  This paper presents an overview of how to run the CCP4 programs for data redn. (SCALA, POINTLESS and CTRUNCATE) through the CCP4 graphical interface ccp4i and points out some issues that need to be considered, together with a few examples.  It covers detn. of the point-group symmetry of the diffraction data (the Laue group), which is required for the subsequent scaling step, examn. of systematic absences, which in many cases will allow inference of the space group, putting multiple data sets on a common indexing system when there are alternatives, the scaling step itself, which produces a large set of data-quality indicators, estn. of |F| from intensity and finally examn. of intensity statistics to detect crystal pathologies such as twinning.  An appendix outlines the scoring schemes used by the program POINTLESS to assign probabilities to possible Laue and space groups.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEq8RhfQOW8LVg90H21EOLACvtfcHk0lgFanA6Z_Ll5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLo%253D&md5=f425388a87744d721d324118d50e4f9a</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1107%2Fs090744491003982x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs090744491003982x%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DP.%2BR.%26atitle%3DAn%2520introduction%2520to%2520data%2520reduction%253A%2520space-group%2520determination%252C%2520scaling%2520and%2520intensity%2520statistics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D282%26epage%3D292%26doi%3D10.1107%2Fs090744491003982x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Storoni, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span> <span> </span><span class="NLM_article-title">Phasercrystallographic software</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">658</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1107/s0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=10.1107%2Fs0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phasercrystallographic+software&doi=10.1107%2Fs0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0lgFanA6Z_Ll5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1107%2Fs0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhasercrystallographic%2520software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2Fs0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunkóczi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echols, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oeffner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, P. H.</span></span> <span> </span><span class="NLM_article-title">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">213</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1107/s0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunk%C3%B3cziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.-W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+a+comprehensive+Python-based+system+for+macromolecular+structure+solution&doi=10.1107%2Fs0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0liATe-v1iRewg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunk%25C3%25B3czi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.-W.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520a%2520comprehensive%2520Python-based%2520system%2520for%2520macromolecular%2520structure%2520solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2Fs0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and development ofCoot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/s0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=10.1107%2Fs0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+ofCoot&doi=10.1107%2Fs0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0liATe-v1iRewg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1107%2Fs0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520ofCoot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2Fs0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arendall, W. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keedy, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Immormino, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span> <span> </span><span class="NLM_article-title">MolProbity: all-atom structure validation for macromolecular crystallography</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">12</span>– <span class="NLM_lpage">21</span>, <span class="refDoi"> DOI: 10.1107/s0907444909042073</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=10.1107%2Fs0907444909042073" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=20057044" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=12-21&author=V.+B.+Chenauthor=W.+B.+Arendallauthor=J.+J.+Headdauthor=D.+A.+Keedyauthor=R.+M.+Immorminoauthor=G.+J.+Kapralauthor=L.+W.+Murrayauthor=J.+S.+Richardsonauthor=D.+C.+Richardson&title=MolProbity%3A+all-atom+structure+validation+for+macromolecular+crystallography&doi=10.1107%2Fs0907444909042073"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">MolProbity: all-atom structure validation for macromolecular crystallography</span></div><div class="casAuthors">Chen, Vincent B.; Arendall, W. Bryan, III; Headd, Jeffrey J.; Keedy, Daniel A.; Immormino, Robert M.; Kapral, Gary J.; Murray, Laura W.; Richardson, Jane S.; Richardson, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">12-21</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">MolProbity is a structure-validation web service that provides broad-spectrum solidly based evaluation of model quality at both the global and local levels for both proteins and nucleic acids.  It relies heavily on the power and sensitivity provided by optimized hydrogen placement and all-atom contact anal., complemented by updated versions of covalent-geometry and torsion-angle criteria.  Some of the local corrections can be performed automatically in MolProbity and all of the diagnostics are presented in chart and graphical forms that help guide manual rebuilding.  X-ray crystallog. provides a wealth of biol. important mol. data in the form of at. three-dimensional structures of proteins, nucleic acids and increasingly large complexes in multiple forms and states.  Advances in automation, in everything from crystn. to data collection to phasing to model building to refinement, have made solving a structure using crystallog. easier than ever.  However, despite these improvements, local errors that can affect biol. interpretation are widespread at low resoln. and even high-resoln. structures nearly all contain at least a few local errors such as Ramachandran outliers, flipped branched protein side chains and incorrect sugar puckers.  It is crit. both for the crystallographer and for the end user that there are easy and reliable methods to diagnose and correct these sorts of errors in structures.  MolProbity is the authors' contribution to helping solve this problem and this article reviews its general capabilities, reports on recent enhancements and usage, and presents evidence that the resulting improvements are now beneficially affecting the global database.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIjgsh59dO-7Vg90H21EOLACvtfcHk0lgK2ZEShRr79g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D&md5=b5fc7574f43f01dd6e43c3663ca4f779</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1107%2Fs0907444909042073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs0907444909042073%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DArendall%26aufirst%3DW.%2BB.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DKeedy%26aufirst%3DD.%2BA.%26aulast%3DImmormino%26aufirst%3DR.%2BM.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DMurray%26aufirst%3DL.%2BW.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26atitle%3DMolProbity%253A%2520all-atom%2520structure%2520validation%2520for%2520macromolecular%2520crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D12%26epage%3D21%26doi%3D10.1107%2Fs0907444909042073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Potterton, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNicholas, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krissinel, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gruber, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perrakis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noble, M.</span></span> <span> </span><span class="NLM_article-title">Developments in theCCP4 molecular-graphics project</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2288</span>– <span class="NLM_lpage">2294</span>, <span class="refDoi"> DOI: 10.1107/s0907444904023716</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=10.1107%2Fs0907444904023716" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=15572783" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars7vN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2288-2294&author=L.+Pottertonauthor=S.+McNicholasauthor=E.+Krissinelauthor=J.+Gruberauthor=K.+Cowtanauthor=P.+Emsleyauthor=G.+N.+Murshudovauthor=S.+Cohenauthor=A.+Perrakisauthor=M.+Noble&title=Developments+in+theCCP4+molecular-graphics+project&doi=10.1107%2Fs0907444904023716"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Developments in the CCP4 molecular-graphics project</span></div><div class="casAuthors">Potterton, Liz; McNicholas, Stuart; Krissinel, Eugene; Gruber, Jan; Cowtan, Kevin; Emsley, Paul; Murshudov, Garib N.; Cohen, Serge; Perrakis, Anastassis; Noble, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2288-2294</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">Progress towards structure detn. that is both high-throughput and high-value is dependent on the development of integrated and automatic tools for electron-d. map interpretation and for the anal. of the resulting at. models.  Advances in map-interpretation algorithms are extending the resoln. regime in which fully automatic tools can work reliably, but at present human intervention is required to interpret poor regions of macromol. electron d., particularly where crystallog. data is only available to modest resoln. [for example, I/σ(I) < 2.0 for min. resoln. 2.5 Å].  In such cases, a set of manual and semi-manual model-building mol.-graphics tools is needed.  At the same time, converting the knowledge encapsulated in a mol. structure into understanding is dependent upon visualization tools, which must be able to communicate that understanding to others by both static and dynamic representations.  CCP4mg is a program designed to meet these needs in a way that is closely integrated with the ongoing development of CCP4 as a program suite suitable for both low- and high-intervention computational structural biol.  As well as providing a carefully designed user interface to advanced algorithms of model building and anal., CCP4mg is intended to present a graphical toolkit to developers of novel algorithms in these fields.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptPa3hk2BiOrVg90H21EOLACvtfcHk0lgK2ZEShRr79g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars7vN&md5=6bdc27bb8dc2d7c5ed37ef306e686001</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1107%2Fs0907444904023716&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs0907444904023716%26sid%3Dliteratum%253Aachs%26aulast%3DPotterton%26aufirst%3DL.%26aulast%3DMcNicholas%26aufirst%3DS.%26aulast%3DKrissinel%26aufirst%3DE.%26aulast%3DGruber%26aufirst%3DJ.%26aulast%3DCowtan%26aufirst%3DK.%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DCohen%26aufirst%3DS.%26aulast%3DPerrakis%26aufirst%3DA.%26aulast%3DNoble%26aufirst%3DM.%26atitle%3DDevelopments%2520in%2520theCCP4%2520molecular-graphics%2520project%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2288%26epage%3D2294%26doi%3D10.1107%2Fs0907444904023716" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P.</span></span> <span> </span><span class="NLM_article-title">Scaling and assessment of data quality</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">72</span>– <span class="NLM_lpage">82</span>, <span class="refDoi"> DOI: 10.1107/s0907444905036693</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=10.1107%2Fs0907444905036693" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=16369096" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlant7jM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2006&pages=72-82&author=P.+Evans&title=Scaling+and+assessment+of+data+quality&doi=10.1107%2Fs0907444905036693"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Scaling and assessment of data quality</span></div><div class="casAuthors">Evans, Philip</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">D62</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">72-82</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">The various phys. factors affecting measured diffraction intensities are discussed, as are the scaling models which may be used to put the data on a consistent scale.  After scaling, the intensities can be analyzed to set the real resoln. of the data set, to detect bad regions (e.g. bad images), to analyze radiation damage and to assess the overall quality of the data set.  The significance of any anomalous signal may be assessed by probability and correlation anal.  The algorithms used by the CCP4 scaling program SCALA are described.  A requirement for the scaling and merging of intensities is knowledge of the Laue group and point-group symmetries: the possible symmetry of the diffraction pattern may be detd. from scores such as correlation coeffs. between observations which might be symmetry-related.  These scoring functions are implemented in a new program POINTLESS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGravLSLU62RhLVg90H21EOLACvtfcHk0lgK2ZEShRr79g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlant7jM&md5=293d3876e534c0c57813990515bb3c76</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1107%2Fs0907444905036693&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs0907444905036693%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DP.%26atitle%3DScaling%2520and%2520assessment%2520of%2520data%2520quality%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2006%26volume%3D62%26spage%3D72%26epage%3D82%26doi%3D10.1107%2Fs0907444905036693" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Diederichs, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karplus, P. A.</span></span> <span> </span><span class="NLM_article-title">Improved R-factors for diffraction data analysis in macromolecular crystallography</span>. <i>Nat. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">269</span>– <span class="NLM_lpage">275</span>, <span class="refDoi"> DOI: 10.1038/nsb0497-269</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=10.1038%2Fnsb0497-269" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=9095194" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A528%3ADyaK2sXitl2ntbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=1997&pages=269-275&author=K.+Diederichsauthor=P.+A.+Karplus&title=Improved+R-factors+for+diffraction+data+analysis+in+macromolecular+crystallography&doi=10.1038%2Fnsb0497-269"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Improved R-factors for diffraction data analysis in macromolecular crystallography</span></div><div class="casAuthors">Diederichs, Kay; Karplus, P. Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural Biology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">269-275</span>CODEN:
                <span class="NLM_cas:coden">NSBIEW</span>;
        ISSN:<span class="NLM_cas:issn">1072-8368</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Co.</span>)
        </div><div class="casAbstract">The quantity Rsym (also called Rmerge) is almost universally used for describing x-ray diffraction data quality.  Rsym is seriously flawed, because it has an implicit dependence on the redundancy of the data.  A cor. R-factor, Rmeas, is introduced as the equiv. robust indicator of data consistency.  Rmrgd, an R-factor that reflects the gain in accuracy upon averaging of equiv. reflections, is introduced as a useful indicator of the quality of reduced data.  These new data quality indicators better reveal the benefits of highly redundant data and should stimulate improvements in data quality through increased merging of data from multiple crystals.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpquf8R-kmFX7Vg90H21EOLACvtfcHk0lhn3I_Sgb5wgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXitl2ntbY%253D&md5=b7e94299b5b1267903d3eb08ad46068c</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1038%2Fnsb0497-269&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsb0497-269%26sid%3Dliteratum%253Aachs%26aulast%3DDiederichs%26aufirst%3DK.%26aulast%3DKarplus%26aufirst%3DP.%2BA.%26atitle%3DImproved%2520R-factors%2520for%2520diffraction%2520data%2520analysis%2520in%2520macromolecular%2520crystallography%26jtitle%3DNat.%2520Struct.%2520Biol.%26date%3D1997%26volume%3D4%26spage%3D269%26epage%3D275%26doi%3D10.1038%2Fnsb0497-269" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, M. S.</span></span> <span> </span><span class="NLM_article-title">Global indicators of X-ray data quality</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">130</span>– <span class="NLM_lpage">135</span>, <span class="refDoi"> DOI: 10.1107/s0021889800018227</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=10.1107%2Fs0021889800018227" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A528%3ADC%252BD3MXit1yhsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2001&pages=130-135&author=M.+S.+Weiss&title=Global+indicators+of+X-ray+data+quality&doi=10.1107%2Fs0021889800018227"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Global indicators of X-ray data quality</span></div><div class="casAuthors">Weiss, Manfred S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">130-135</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard International Publishers Ltd.</span>)
        </div><div class="casAbstract">Global indicators of the quality of diffraction data are presented and discussed, and are evaluated in terms of their performance with respect to various tasks.  Based on the results obtained, probably some of the conventional indicators still in use in the crystallog. community should be abandoned, such as the nominal resoln. dmin or the merging R factor Rmerge, and replaced by more objective and more meaningful nos., such as the effective optical resoln. deff,opt and the redundancy-independent merging R factor Rr.i.m.  Also, it is recommended that the precision-indicating merging R factor Rp.i.m. should be reported with every diffraction data set published, because it describes the precision of the averaged measurements, which are the quantities normally used in crystallog. as observables.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4sLhqfAbBZ7Vg90H21EOLACvtfcHk0lhn3I_Sgb5wgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXit1yhsLk%253D&md5=4b4d934beea8b3479f5fd54c802d34af</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1107%2Fs0021889800018227&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs0021889800018227%26sid%3Dliteratum%253Aachs%26aulast%3DWeiss%26aufirst%3DM.%2BS.%26atitle%3DGlobal%2520indicators%2520of%2520X-ray%2520data%2520quality%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2001%26volume%3D34%26spage%3D130%26epage%3D135%26doi%3D10.1107%2Fs0021889800018227" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Karplus, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diederichs, K.</span></span> <span> </span><span class="NLM_article-title">Linking crystallographic model and data quality</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>336</i></span>,  <span class="NLM_fpage">1030</span>– <span class="NLM_lpage">1033</span>, <span class="refDoi"> DOI: 10.1126/science.1218231</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=10.1126%2Fscience.1218231" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=22628654" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A528%3ADC%252BC38Xnt1Kgt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=336&publication_year=2012&pages=1030-1033&author=P.+A.+Karplusauthor=K.+Diederichs&title=Linking+crystallographic+model+and+data+quality&doi=10.1126%2Fscience.1218231"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Linking Crystallographic Model and Data Quality</span></div><div class="casAuthors">Karplus, P. Andrew; Diederichs, Kay</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">336</span>
        (<span class="NLM_cas:issue">6084</span>),
    <span class="NLM_cas:pages">1030-1033</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">In macromol. x-ray crystallog., refinement R values measure the agreement between obsd. and calcd. data.  Analogously, Rmerge values reporting on the agreement between multiple measurements of a given reflection are used to assess data quality.  Here, we show that despite their widespread use, Rmerge values are poorly suited for detg. the high-resoln. limit and that current std. protocols discard much useful data.  We introduce a statistic that ests. the correlation of an obsd. data set with the underlying (not measurable) true signal; this quantity, CC*, provides a single statistically valid guide for deciding which data are useful.  CC* also can be used to assess model and data quality on the same scale, and this reveals when data quality is limiting model improvement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0u3BGdcdUzbVg90H21EOLACvtfcHk0lhn3I_Sgb5wgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xnt1Kgt7Y%253D&md5=b8ee6edacea44364cc66c44dd3a505fd</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1126%2Fscience.1218231&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1218231%26sid%3Dliteratum%253Aachs%26aulast%3DKarplus%26aufirst%3DP.%2BA.%26aulast%3DDiederichs%26aufirst%3DK.%26atitle%3DLinking%2520crystallographic%2520model%2520and%2520data%2520quality%26jtitle%3DScience%26date%3D2012%26volume%3D336%26spage%3D1030%26epage%3D1033%26doi%3D10.1126%2Fscience.1218231" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P.</span></span> <span> </span><span class="NLM_article-title">Resolving Some Old Problems in Protein Crystallography</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>336</i></span>,  <span class="NLM_fpage">986</span>– <span class="NLM_lpage">987</span>, <span class="refDoi"> DOI: 10.1126/science.1222162</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=10.1126%2Fscience.1222162" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=22628641" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A528%3ADC%252BC38XptVClsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=336&publication_year=2012&pages=986-987&author=P.+Evans&title=Resolving+Some+Old+Problems+in+Protein+Crystallography&doi=10.1126%2Fscience.1222162"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Resolving some old problems in protein crystallography</span></div><div class="casAuthors">Evans, Phil</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">336</span>
        (<span class="NLM_cas:issue">6084</span>),
    <span class="NLM_cas:pages">986-987</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3-ErE43zPCLVg90H21EOLACvtfcHk0lhn3I_Sgb5wgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XptVClsbo%253D&md5=fcffa3a74929e5c11b8a2ef3c890cfd5</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1126%2Fscience.1222162&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1222162%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DP.%26atitle%3DResolving%2520Some%2520Old%2520Problems%2520in%2520Protein%2520Crystallography%26jtitle%3DScience%26date%3D2012%26volume%3D336%26spage%3D986%26epage%3D987%26doi%3D10.1126%2Fscience.1222162" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article has not yet been cited by other publications.</p></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00319/20210716/images/medium/jm1c00319_0008.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00319/20210716/images/large/jm1c00319_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00319&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00319/20210716/images/medium/jm1c00319_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00319/20210716/images/large/jm1c00319_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Inhibitors <b>2(a–o)</b> and <b>3(a–o)</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00319/20210716/images/large/jm1c00319_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00319&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00319/20210716/images/medium/jm1c00319_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00319/20210716/images/large/jm1c00319_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Alcohol Inputs <b>(a–o)</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00319/20210716/images/large/jm1c00319_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00319&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00319/20210716/images/medium/jm1c00319_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00319/20210716/images/large/jm1c00319_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Binding mode of <b>2a</b> (gray) with SARS-CoV-2 3CLpro associated with subunit A (A,C) and subunit B (B,D). Fo–Fc polder omit map (green mesh) contoured at 3σ (A,B). Hydrogen bond interactions (dashed lines) (C,D). Surface representations showing the orientation of <b>2a</b> in subunit A (E) and subunit B (F) near the S<sub>4</sub> subsite of SARS-CoV-2 3CLpro with neighboring residues colored yellow (nonpolar), cyan (polar), and white (weakly polar).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00319/20210716/images/large/jm1c00319_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00319&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00319/20210716/images/medium/jm1c00319_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00319/20210716/images/large/jm1c00319_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Binding mode of <b>3c</b> (A,B,E) and its deuterated analogue <b>3d</b> (C,D,F) with SARS-CoV-2 3CLpro. Fo–Fc polder omit map (green mesh) contoured at 3σ (A,B). Hydrogen bond interactions (dashed lines) (C,D). surface representation showing the orientation of <b>3c</b> (E) and <b>3d</b> (F) near the S<sub>4</sub> subsite of SARS-CoV-2 3CLpro with neighboring residues colored yellow (nonpolar), cyan (polar), and white (weakly polar).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00319/20210716/images/large/jm1c00319_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00319&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00319/20210716/images/medium/jm1c00319_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00319/20210716/images/large/jm1c00319_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Binding mode of <b>2f</b> (A,B), <b>2k</b> (C,D), and <b>3e</b> (E,F) with SARS-CoV-2 3CLpro. Fo–Fc polder omit map (green mesh) contoured at 3σ (A,C,E). Hydrogen bond interactions (dashed lines) (B,D,F).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00319/20210716/images/large/jm1c00319_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00319&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00319/20210716/images/medium/jm1c00319_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00319/20210716/images/large/jm1c00319_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Superposition of all seven inhibitor-bound structures <b>2a</b> (red), <b>3b</b> (blue), <b>2f</b> (cyan), <b>2k</b> (yellow), <b>3c</b> (coral), <b>3d</b> (magenta), and <b>3e</b> (green). The bicyclic rings cover a space of approximately 6.3 Å in the S<sub>4</sub> subsite.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00319/20210716/images/large/jm1c00319_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00319&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i79">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_14823" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_14823" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 43 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Perlman, S.</span>, <span class="NLM_string-name">Masters, P. S.</span></span> <span> </span><span class="NLM_article-title">Coronaviridae: The Viruses and Their Replication</span>. In  <i>Fields Virology: Emerging Viruses</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howley, P. M.</span>, <span class="NLM_string-name">Knipe, D. M.</span>, <span class="NLM_string-name">Whelan, S.</span></span>, Eds.; <span class="NLM_publisher-name">Wolters Kluwer: Wichita</span>, <span class="NLM_year">2020</span>; Vol.  <span class="NLM_volume">1</span>, pp  <span class="NLM_fpage">410</span>– <span class="NLM_lpage">448</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&pages=410-448&author=S.+Perlman&author=P.+S.+Mastersauthor=P.+M.+Howley&author=D.+M.+Knipe&author=S.+Whelan&title=Fields+Virology%3A+Emerging+Viruses"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DPerlman%26aufirst%3DS.%26atitle%3DCoronaviridae%253A%2520The%2520Viruses%2520and%2520Their%2520Replication%26btitle%3DFields%2520Virology%253A%2520Emerging%2520Viruses%26aulast%3DHowley%26aufirst%3DP.%2BM.%26pub%3DWolters%2520Kluwer%253A%2520Wichita%26date%3D2020%26volume%3D1%26spage%3D410%26epage%3D448" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhuang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, Q.</span></span> <span> </span><span class="NLM_article-title">A comparison of COVID-19, SARS and MERS</span>. <i>PeerJ</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">e9725</span> <span class="refDoi"> DOI: 10.7717/peerj.9725</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=10.7717%2Fpeerj.9725" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=32879801" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2020&author=T.+Huauthor=Y.+Liuauthor=M.+Zhaoauthor=Q.+Zhuangauthor=L.+Xuauthor=Q.+He&title=A+comparison+of+COVID-19%2C+SARS+and+MERS&doi=10.7717%2Fpeerj.9725"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.7717%2Fpeerj.9725&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7717%252Fpeerj.9725%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DT.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DM.%26aulast%3DZhuang%26aufirst%3DQ.%26aulast%3DXu%26aufirst%3DL.%26aulast%3DHe%26aufirst%3DQ.%26atitle%3DA%2520comparison%2520of%2520COVID-19%252C%2520SARS%2520and%2520MERS%26jtitle%3DPeerJ%26date%3D2020%26volume%3D8%26doi%3D10.7717%2Fpeerj.9725" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Z.-G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, Z.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, J.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, Y.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, M.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, F.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, J.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holmes, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.-Z.</span></span> <span> </span><span class="NLM_article-title">A new coronavirus associated with human respiratory disease in China</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>579</i></span>,  <span class="NLM_fpage">265</span>– <span class="NLM_lpage">269</span>, <span class="refDoi"> DOI: 10.1038/s41586-020-2008-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=10.1038%2Fs41586-020-2008-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=32015508" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A528%3ADC%252BB3cXksFKlsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=579&publication_year=2020&pages=265-269&author=F.+Wuauthor=S.+Zhaoauthor=B.+Yuauthor=Y.-M.+Chenauthor=W.+Wangauthor=Z.-G.+Songauthor=Y.+Huauthor=Z.-W.+Taoauthor=J.-H.+Tianauthor=Y.-Y.+Peiauthor=M.-L.+Yuanauthor=Y.-L.+Zhangauthor=F.-H.+Daiauthor=Y.+Liuauthor=Q.-M.+Wangauthor=J.-J.+Zhengauthor=L.+Xuauthor=E.+C.+Holmesauthor=Y.-Z.+Zhang&title=A+new+coronavirus+associated+with+human+respiratory+disease+in+China&doi=10.1038%2Fs41586-020-2008-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">A new coronavirus associated with human respiratory disease in China</span></div><div class="casAuthors">Wu, Fan; Zhao, Su; Yu, Bin; Chen, Yan-Mei; Wang, Wen; Song, Zhi-Gang; Hu, Yi; Tao, Zhao-Wu; Tian, Jun-Hua; Pei, Yuan-Yuan; Yuan, Ming-Li; Zhang, Yu-Ling; Dai, Fa-Hui; Liu, Yi; Wang, Qi-Min; Zheng, Jiao-Jiao; Xu, Lin; Holmes, Edward C.; Zhang, Yong-Zhen</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">579</span>
        (<span class="NLM_cas:issue">7798</span>),
    <span class="NLM_cas:pages">265-269</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Emerging infectious diseases, such as severe acute respiratory syndrome (SARS) and Zika virus disease, present a major threat to public health.  Despite intense research efforts, how, when and where new diseases appear are still a source of considerable uncertainty.  A severe respiratory disease was recently reported in Wuhan, Hubei province, China.  As of 25 Jan. 2020, at least 1,975 cases had been reported since the first patient was hospitalized on 12 Dec. 2019.  Epidemiol. investigations have suggested that the outbreak was assocd. with a seafood market in Wuhan.  Here we study a single patient who was a worker at the market and who was admitted to the Central Hospital of Wuhan on 26 Dec. 2019 while experiencing a severe respiratory syndrome that included fever, dizziness and a cough.  Metagenomic RNA sequencing of a sample of bronchoalveolar lavage fluid from the patient identified a new RNA virus strain from the family Coronaviridae, which is designated here 'WH-Human 1' coronavirus (and has also been referred to as '2019-nCoV').  Phylogenetic anal. of the complete viral genome (29,903 nucleotides) revealed that the virus was most closely related (89.1% nucleotide similarity) to a group of SARS-like coronaviruses (genus Betacoronavirus, subgenus Sarbecovirus) that had previously been found in bats in China.  This outbreak highlights the ongoing ability of viral spill-over from animals to cause severe disease in humans.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5h_VHfuDuTbVg90H21EOLACvtfcHk0ljcZE8KVeHLag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXksFKlsLc%253D&md5=0163a684829e880a0c3347e19f0ce52a</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fs41586-020-2008-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-020-2008-3%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DF.%26aulast%3DZhao%26aufirst%3DS.%26aulast%3DYu%26aufirst%3DB.%26aulast%3DChen%26aufirst%3DY.-M.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DSong%26aufirst%3DZ.-G.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DTao%26aufirst%3DZ.-W.%26aulast%3DTian%26aufirst%3DJ.-H.%26aulast%3DPei%26aufirst%3DY.-Y.%26aulast%3DYuan%26aufirst%3DM.-L.%26aulast%3DZhang%26aufirst%3DY.-L.%26aulast%3DDai%26aufirst%3DF.-H.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DQ.-M.%26aulast%3DZheng%26aufirst%3DJ.-J.%26aulast%3DXu%26aufirst%3DL.%26aulast%3DHolmes%26aufirst%3DE.%2BC.%26aulast%3DZhang%26aufirst%3DY.-Z.%26atitle%3DA%2520new%2520coronavirus%2520associated%2520with%2520human%2520respiratory%2520disease%2520in%2520China%26jtitle%3DNature%26date%3D2020%26volume%3D579%26spage%3D265%26epage%3D269%26doi%3D10.1038%2Fs41586-020-2008-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dong, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gardner, L.</span></span> <span> </span><span class="NLM_article-title">An interactive web-based dashboard to track COVID-19 in real time</span>. <i>Lancet Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">533</span>– <span class="NLM_lpage">534</span>, <span class="refDoi"> DOI: 10.1016/s1473-3099(20)30120-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=10.1016%2Fs1473-3099%2820%2930120-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=32087114" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A528%3ADC%252BB3cXksVaisbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2020&pages=533-534&author=E.+Dongauthor=H.+Duauthor=L.+Gardner&title=An+interactive+web-based+dashboard+to+track+COVID-19+in+real+time&doi=10.1016%2Fs1473-3099%2820%2930120-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">An interactive web-based dashboard to track COVID-19 in real time</span></div><div class="casAuthors">Dong, Ensheng; Du, Hongru; Gardner, Lauren</div><div class="citationInfo"><span class="NLM_cas:title">Lancet Infectious Diseases</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">533-534</span>CODEN:
                <span class="NLM_cas:coden">LIDABP</span>;
        ISSN:<span class="NLM_cas:issn">1473-3099</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The authors describe the development of an online interactive dashboard, hosted by the Center for Systems Science and Engineering (CSSE) at Johns Hopkins University, Baltimore, MD, USA, to visualize and track reported cases of coronavirus disease 2019 (COVID-19) in real time.  The dashboard, first shared publicly on Jan 22, illustrates the location and no. of confirmed COVID-19 cases, deaths, and recoveries for all affected countries.  It was developed to provide researchers, public health authorities, and the general public with a user-friendly tool to track the outbreak as it unfolds.  All data collected and displayed are made freely available, initially through Google Sheets and now through a GitHub repository, along with the feature layers of the dashboard, which are now included in the Esri Living Atlas.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDJFa57DS5AbVg90H21EOLACvtfcHk0ljcZE8KVeHLag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXksVaisbs%253D&md5=7f528642a4d2b2fcc3575e37085819a8</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fs1473-3099%2820%2930120-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fs1473-3099%252820%252930120-1%26sid%3Dliteratum%253Aachs%26aulast%3DDong%26aufirst%3DE.%26aulast%3DDu%26aufirst%3DH.%26aulast%3DGardner%26aufirst%3DL.%26atitle%3DAn%2520interactive%2520web-based%2520dashboard%2520to%2520track%2520COVID-19%2520in%2520real%2520time%26jtitle%3DLancet%2520Infect.%2520Dis.%26date%3D2020%26volume%3D20%26spage%3D533%26epage%3D534%26doi%3D10.1016%2Fs1473-3099%2820%2930120-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brindisi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shahabi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chapman, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mesecar, A. D.</span></span> <span> </span><span class="NLM_article-title">Drug Development and Medicinal Chemistry Efforts toward SARS-Coronavirus and Covid-19 Therapeutics</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">907</span>– <span class="NLM_lpage">932</span>, <span class="refDoi"> DOI: 10.1002/cmdc.202000223</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=10.1002%2Fcmdc.202000223" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=32324951" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A528%3ADC%252BB3cXos1GmsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2020&pages=907-932&author=A.+K.+Ghoshauthor=M.+Brindisiauthor=D.+Shahabiauthor=M.+E.+Chapmanauthor=A.+D.+Mesecar&title=Drug+Development+and+Medicinal+Chemistry+Efforts+toward+SARS-Coronavirus+and+Covid-19+Therapeutics&doi=10.1002%2Fcmdc.202000223"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Drug Development and Medicinal Chemistry Efforts toward SARS-Coronavirus and Covid-19 Therapeutics</span></div><div class="casAuthors">Ghosh, Arun K.; Brindisi, Margherita; Shahabi, Dana; Chapman, Mackenzie E.; Mesecar, Andrew D.</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">907-932</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  The COVID-19 pandemic caused by SARS-CoV-2 infection is spreading at an alarming rate and has created an unprecedented health emergency around the globe.  There is no effective vaccine or approved drug treatment against COVID-19 and other pathogenic coronaviruses.  The development of antiviral agents is an urgent priority.  Biochem. events crit. to the coronavirus replication cycle provided a no. of attractive targets for drug development.  These include, spike protein for binding to host cell-surface receptors, proteolytic enzymes that are essential for processing polyproteins into mature viruses, and RNA-dependent RNA polymerase for RNA replication.  There has been a lot of ground work for drug discovery and development against these targets.  Also, high-throughput screening efforts have led to the identification of diverse lead structures, including natural product-derived mols.  This review highlights past and present drug discovery and medicinal-chem. approaches against SARS-CoV, MERS-CoV and COVID-19 targets.  The review hopes to stimulate further research and will be a useful guide to the development of effective therapies against COVID-19 and other pathogenic coronaviruses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqcgVQF9N3zf7Vg90H21EOLACvtfcHk0ljcZE8KVeHLag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXos1GmsL4%253D&md5=8b4c3c83c05a7b64f39ba7d803ea1709</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.202000223&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.202000223%26sid%3Dliteratum%253Aachs%26aulast%3DGhosh%26aufirst%3DA.%2BK.%26aulast%3DBrindisi%26aufirst%3DM.%26aulast%3DShahabi%26aufirst%3DD.%26aulast%3DChapman%26aufirst%3DM.%2BE.%26aulast%3DMesecar%26aufirst%3DA.%2BD.%26atitle%3DDrug%2520Development%2520and%2520Medicinal%2520Chemistry%2520Efforts%2520toward%2520SARS-Coronavirus%2520and%2520Covid-19%2520Therapeutics%26jtitle%3DChemMedChem%26date%3D2020%26volume%3D15%26spage%3D907%26epage%3D932%26doi%3D10.1002%2Fcmdc.202000223" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gil, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ginex, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maestro, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nozal, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrado-Gil, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuesta-Geijo, M. Á.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urquiza, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramírez, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alonso, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campillo, N. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, A.</span></span> <span> </span><span class="NLM_article-title">COVID-19: Drug targets and potential treatments</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">12359</span>– <span class="NLM_lpage">12386</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00606</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00606" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtFWmsL3I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=12359-12386&author=C.+Gilauthor=T.+Ginexauthor=I.+Maestroauthor=V.+Nozalauthor=L.+Barrado-Gilauthor=M.+%C3%81.+Cuesta-Geijoauthor=J.+Urquizaauthor=D.+Ram%C3%ADrezauthor=C.+Alonsoauthor=N.+E.+Campilloauthor=A.+Martinez&title=COVID-19%3A+Drug+targets+and+potential+treatments&doi=10.1021%2Facs.jmedchem.0c00606"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">COVID-19: Drug Targets and Potential Treatments</span></div><div class="casAuthors">Gil, Carmen; Ginex, Tiziana; Maestro, Ines; Nozal, Vanesa; Barrado-Gil, Lucia; Cuesta-Geijo, Miguel Angel; Urquiza, Jesus; Ramirez, David; Alonso, Covadonga; Campillo, Nuria E.; Martinez, Ana</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">12359-12386</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Currently, the authors are immersed in a pandemic caused by the emerging severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), which severely threatens public health worldwide.  Until now, no drug or vaccine has been approved to treat the severe disease caused by this coronavirus, COVID-19.  The authors will focus on the main virus-based and host-based targets that can guide medicinal chem. efforts to discover new drugs for this devastating disease.  In principle, all CoVs enzymes and proteins involved in viral replication and the control of host cellular machineries are potentially druggable targets in the search for therapeutic options for SARS-CoV-2.  This perspective provides an overview of the main targets from a structural point of view, together with reported therapeutic compds. with activity against SARS-CoV-2 and/or other CoVs.  Also, the role of innate immune response to coronavirus infection and the related therapeutic options will be presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsg-YRktOknrVg90H21EOLACvtfcHk0lgupv4GTwRmYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtFWmsL3I&md5=6a0dc17c5f601f09a92da933f81bbbea</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00606&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00606%26sid%3Dliteratum%253Aachs%26aulast%3DGil%26aufirst%3DC.%26aulast%3DGinex%26aufirst%3DT.%26aulast%3DMaestro%26aufirst%3DI.%26aulast%3DNozal%26aufirst%3DV.%26aulast%3DBarrado-Gil%26aufirst%3DL.%26aulast%3DCuesta-Geijo%26aufirst%3DM.%2B%25C3%2581.%26aulast%3DUrquiza%26aufirst%3DJ.%26aulast%3DRam%25C3%25ADrez%26aufirst%3DD.%26aulast%3DAlonso%26aufirst%3DC.%26aulast%3DCampillo%26aufirst%3DN.%2BE.%26aulast%3DMartinez%26aufirst%3DA.%26atitle%3DCOVID-19%253A%2520Drug%2520targets%2520and%2520potential%2520treatments%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D12359%26epage%3D12386%26doi%3D10.1021%2Facs.jmedchem.0c00606" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tiwari, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deb, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MARTY, J. L.</span></span> <span> </span><span class="NLM_article-title">Severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2): a global pandemic and treatment strategies</span>. <i>Int. J. Antimicrob. Agents</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">106054</span>, <span class="refDoi"> DOI: 10.1016/j.ijantimicag.2020.106054</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=10.1016%2Fj.ijantimicag.2020.106054" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=32534188" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtlyjtrvL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2020&pages=106054&author=A.+Sharmaauthor=S.+Tiwariauthor=M.+K.+Debauthor=J.+L.+MARTY&title=Severe+acute+respiratory+syndrome+coronavirus-2+%28SARS-CoV-2%29%3A+a+global+pandemic+and+treatment+strategies&doi=10.1016%2Fj.ijantimicag.2020.106054"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2): a global pandemic and treatment strategies</span></div><div class="casAuthors">Sharma, Atul; Tiwari, Swapnil; Deb, Manas Kanti; Marty, Jean Louis</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Antimicrobial Agents</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">106054</span>CODEN:
                <span class="NLM_cas:coden">IAAGEA</span>;
        ISSN:<span class="NLM_cas:issn">0924-8579</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The emergence and rapid spread of coronavirus disease 2019 (COVID-19), caused by severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2), a potentially fatal disease, is swiftly leading to public health crises worldwide.  The origin of SARS-CoV-2 infection was first reported in people exposed to a seafood market in Wuhan City, China in Dec. 2019.  It has been suggested that the infection is likely to be of zoonotic origin and transmitted to humans through a not-yet-known intermediary.  As of 22 May 2020, the World Health Organization reported that there were approx. 4995,996 confirmed cases and 327,821 deaths.  SARS-CoV-2 is transmitted via inhalation or direct contact with droplets from infected people.  It has an incubation period ranging from 2 to ≥14 days.  The rate of spread of SARS-CoV-2 is greater than that for severe acute respiratory syndrome coronavirus and Middle East respiratory coronavirus.  The symptoms are similar to influenza (i.e. breathlessness, sore throat and fatigue) and infected cases are isolated and treated.  Infection is mild in most cases, but in elderly (>50 years) patients and those with cardiac and respiratory disorders, it may progress to pneumonia, acute respiratory distress syndrome and multi-organ failure.  People with strong immunity or those who have developed herd immunity are asymptomatic.  The fatality rate ranges from 3% to 4%.  Recommended methods for diagnosis of COVID-19 are mol. tests (e.g. polymerase chain reaction) on respiratory secretions, chest scan and common lab. diagnosis.  Currently, treatment is essentially supportive, and the role of antiviral agents is yet to be established as a vaccine is not yet available.  This review will focus on epidemiol., symptoms, transmission, pathogenesis, 210 1 ongoing available treatments and future perspectives of SARS-CoV-2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBUcBXUfUAJLVg90H21EOLACvtfcHk0lgupv4GTwRmYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtlyjtrvL&md5=24d5155d11e2f97f4b14c19545b72235</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.ijantimicag.2020.106054&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ijantimicag.2020.106054%26sid%3Dliteratum%253Aachs%26aulast%3DSharma%26aufirst%3DA.%26aulast%3DTiwari%26aufirst%3DS.%26aulast%3DDeb%26aufirst%3DM.%2BK.%26aulast%3DMARTY%26aufirst%3DJ.%2BL.%26atitle%3DSevere%2520acute%2520respiratory%2520syndrome%2520coronavirus-2%2520%2528SARS-CoV-2%2529%253A%2520a%2520global%2520pandemic%2520and%2520treatment%2520strategies%26jtitle%3DInt.%2520J.%2520Antimicrob.%2520Agents%26date%3D2020%26volume%3D56%26spage%3D106054%26doi%3D10.1016%2Fj.ijantimicag.2020.106054" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cannalire, R.</span>; <span class="NLM_string-name">Cerchia, C.</span>; <span class="NLM_string-name">Beccari, A. R.</span>; <span class="NLM_string-name">Di Leva, F. S.</span>; <span class="NLM_string-name">Summa, V.</span></span> <span> </span><span class="NLM_article-title">Targeting SARS-CoV-2 proteases and polymerase for COVID-19 treatment: State of the art and future opportunities</span>.  <i>J. Med. Chem.</i> [Online early access].<span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c01140</span> . Published online: Nov 13, <span class="NLM_year" style="font-weight: bold;">2020</span>. <a href="https://doi.org/10.1021/acs.jmedchem.0c01140" class="extLink">https://doi.org/10.1021/acs.jmedchem.0c01140</a> (accessed Nov 13, 2020).</span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c01140" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&author=R.+Cannalire&author=C.+Cerchia&author=A.+R.+Beccari&author=F.+S.+Di+Leva&author=V.+Summa&title=Targeting+SARS-CoV-2+proteases+and+polymerase+for+COVID-19+treatment%3A+State+of+the+art+and+future+opportunities&doi=10.1021%2Facs.jmedchem.0c01140"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c01140&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c01140%26sid%3Dliteratum%253Aachs%26aulast%3DCannalire%26aufirst%3DR.%26atitle%3DTargeting%2520SARS-CoV-2%2520proteases%2520and%2520polymerase%2520for%2520COVID-19%2520treatment%253A%2520State%2520of%2520the%2520art%2520and%2520future%2520opportunities%26jtitle%3DJ.%2520Med.%2520Chem.%26doi%3D10.1021%2Facs.jmedchem.0c01140%26date%3D2020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ullrich, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nitsche, C.</span></span> <span> </span><span class="NLM_article-title">The SARS-CoV-2 main protease as drug target</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">127377</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2020.127377</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=10.1016%2Fj.bmcl.2020.127377" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=32738988" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtlaisrbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2020&pages=127377&author=S.+Ullrichauthor=C.+Nitsche&title=The+SARS-CoV-2+main+protease+as+drug+target&doi=10.1016%2Fj.bmcl.2020.127377"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">The SARS-CoV-2 main protease as drug target</span></div><div class="casAuthors">Ullrich, Sven; Nitsche, Christoph</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">127377</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The unprecedented pandemic of the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is threatening global health.  The virus emerged in late 2019 and can cause a severe disease assocd. with significant mortality.  Several vaccine development and drug discovery campaigns are underway.  The SARS-CoV-2 main protease is considered a promising drug target, as it is dissimilar to human proteases.  Sequence and structure of the main protease are closely related to those from other betacoronaviruses, facilitating drug discovery attempts based on previous lead compds.  Covalently binding peptidomimetics and small mols. are investigated.  Various compds. show antiviral activity in infected human cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVeovBguTwDbVg90H21EOLACvtfcHk0lgupv4GTwRmYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtlaisrbJ&md5=b2fd8ea99ddf6d3e553b137904d406a9</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2020.127377&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2020.127377%26sid%3Dliteratum%253Aachs%26aulast%3DUllrich%26aufirst%3DS.%26aulast%3DNitsche%26aufirst%3DC.%26atitle%3DThe%2520SARS-CoV-2%2520main%2520protease%2520as%2520drug%2520target%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2020%26volume%3D30%26spage%3D127377%26doi%3D10.1016%2Fj.bmcl.2020.127377" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rut, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lv, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zmudzinski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patchett, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nayak, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snipas, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El Oualid, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bekes, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drag, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olsen, S. K.</span></span> <span> </span><span class="NLM_article-title">Activity profiling and crystal structures of inhibitor-bound SARS-CoV-2 papain-like protease: A framework for anti-COVID-19 drug design</span>. <i>Sci. Adv.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>6</i></span>, <span class="NLM_elocation-id">eabd4596</span> <span class="refDoi"> DOI: 10.1126/sciadv.abd4596</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=10.1126%2Fsciadv.abd4596" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=33067239" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2020&author=W.+Rutauthor=Z.+Lvauthor=M.+Zmudzinskiauthor=S.+Patchettauthor=D.+Nayakauthor=S.+J.+Snipasauthor=F.+El+Oualidauthor=T.+T.+Huangauthor=M.+Bekesauthor=M.+Dragauthor=S.+K.+Olsen&title=Activity+profiling+and+crystal+structures+of+inhibitor-bound+SARS-CoV-2+papain-like+protease%3A+A+framework+for+anti-COVID-19+drug+design&doi=10.1126%2Fsciadv.abd4596"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1126%2Fsciadv.abd4596&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fsciadv.abd4596%26sid%3Dliteratum%253Aachs%26aulast%3DRut%26aufirst%3DW.%26aulast%3DLv%26aufirst%3DZ.%26aulast%3DZmudzinski%26aufirst%3DM.%26aulast%3DPatchett%26aufirst%3DS.%26aulast%3DNayak%26aufirst%3DD.%26aulast%3DSnipas%26aufirst%3DS.%2BJ.%26aulast%3DEl%2BOualid%26aufirst%3DF.%26aulast%3DHuang%26aufirst%3DT.%2BT.%26aulast%3DBekes%26aufirst%3DM.%26aulast%3DDrag%26aufirst%3DM.%26aulast%3DOlsen%26aufirst%3DS.%2BK.%26atitle%3DActivity%2520profiling%2520and%2520crystal%2520structures%2520of%2520inhibitor-bound%2520SARS-CoV-2%2520papain-like%2520protease%253A%2520A%2520framework%2520for%2520anti-COVID-19%2520drug%2520design%26jtitle%3DSci.%2520Adv.%26date%3D2020%26volume%3D6%26doi%3D10.1126%2Fsciadv.abd4596" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jin, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guddat, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span> <span> </span><span class="NLM_article-title">Structure of Mpro from SARS-CoV-2 and discovery of its inhibitors</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>582</i></span>,  <span class="NLM_fpage">289</span>– <span class="NLM_lpage">293</span>, <span class="refDoi"> DOI: 10.1038/s41586-020-2223-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=10.1038%2Fs41586-020-2223-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=32272481" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtVyhsrrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=582&publication_year=2020&pages=289-293&author=Z.+Jinauthor=X.+Duauthor=Y.+Xuauthor=Y.+Dengauthor=M.+Liuauthor=Y.+Zhaoauthor=B.+Zhangauthor=X.+Liauthor=L.+Zhangauthor=C.+Pengauthor=Y.+Duanauthor=J.+Yuauthor=L.+Wangauthor=K.+Yangauthor=F.+Liuauthor=R.+Jiangauthor=X.+Yangauthor=T.+Youauthor=X.+Liuauthor=X.+Yangauthor=F.+Baiauthor=H.+Liuauthor=X.+Liuauthor=L.+W.+Guddatauthor=W.+Xuauthor=G.+Xiaoauthor=C.+Qinauthor=Z.+Shiauthor=H.+Jiangauthor=Z.+Raoauthor=H.+Yang&title=Structure+of+Mpro+from+SARS-CoV-2+and+discovery+of+its+inhibitors&doi=10.1038%2Fs41586-020-2223-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of Mpro from SARS-CoV-2 and discovery of its inhibitors</span></div><div class="casAuthors">Jin, Zhenming; Du, Xiaoyu; Xu, Yechun; Deng, Yongqiang; Liu, Meiqin; Zhao, Yao; Zhang, Bing; Li, Xiaofeng; Zhang, Leike; Peng, Chao; Duan, Yinkai; Yu, Jing; Wang, Lin; Yang, Kailin; Liu, Fengjiang; Jiang, Rendi; Yang, Xinglou; You, Tian; Liu, Xiaoce; Yang, Xiuna; Bai, Fang; Liu, Hong; Liu, Xiang; Guddat, Luke W.; Xu, Wenqing; Xiao, Gengfu; Qin, Chengfeng; Shi, Zhengli; Jiang, Hualiang; Rao, Zihe; Yang, Haitao</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">582</span>
        (<span class="NLM_cas:issue">7811</span>),
    <span class="NLM_cas:pages">289-293</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: A new coronavirus, known as severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), is the etiol. agent responsible for the 2019-2020 viral pneumonia outbreak of coronavirus disease 2019 (COVID-19).  Currently, there are no targeted therapeutic agents for the treatment of this disease, and effective treatment options remain very limited.  Here, we describe the results of a program that aimed to rapidly discover lead compds. for clin. use, by combining structure-assisted drug design, virtual drug screening and high-throughput screening.  This program focused on identifying drug leads that target main protease (Mpro) of SARS-CoV-2: Mpro is a key enzyme of coronaviruses and has a pivotal role in mediating viral replication and transcription, making it an attractive drug target for SARS-CoV-2.  We identified a mechanism-based inhibitor (N3) by computer-aided drug design, and then detd. the crystal structure of Mpro of SARS-CoV-2 in complex with this compd.  Through a combination of structure-based virtual and high-throughput screening, we assayed more than 10,000 compds.-including approved drugs, drug candidates in clin. trials and other pharmacol. active compds.-as inhibitors of Mpro.  Six of these compds. inhibited Mpro, showing half-maximal inhibitory concn. values that ranged from 0.67 to 21.4μM.  One of these compds. (ebselen) also exhibited promising antiviral activity in cell-based assays.  Our results demonstrate the efficacy of our screening strategy, which can lead to the rapid discovery of drug leads with clin. potential in response to new infectious diseases for which no specific drugs or vaccines are available.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFupuh6WIyW7Vg90H21EOLACvtfcHk0ljcj2V6Si1PWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtVyhsrrO&md5=b84f350fe9ce1109485df6caf814ba82</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fs41586-020-2223-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-020-2223-y%26sid%3Dliteratum%253Aachs%26aulast%3DJin%26aufirst%3DZ.%26aulast%3DDu%26aufirst%3DX.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DDeng%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DM.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DB.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DPeng%26aufirst%3DC.%26aulast%3DDuan%26aufirst%3DY.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DJiang%26aufirst%3DR.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DYou%26aufirst%3DT.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DBai%26aufirst%3DF.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DGuddat%26aufirst%3DL.%2BW.%26aulast%3DXu%26aufirst%3DW.%26aulast%3DXiao%26aufirst%3DG.%26aulast%3DQin%26aufirst%3DC.%26aulast%3DShi%26aufirst%3DZ.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DRao%26aufirst%3DZ.%26aulast%3DYang%26aufirst%3DH.%26atitle%3DStructure%2520of%2520Mpro%2520from%2520SARS-CoV-2%2520and%2520discovery%2520of%2520its%2520inhibitors%26jtitle%3DNature%26date%3D2020%26volume%3D582%26spage%3D289%26epage%3D293%26doi%3D10.1038%2Fs41586-020-2223-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curth, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drosten, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sauerhering, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rox, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilgenfeld, R.</span></span> <span> </span><span class="NLM_article-title">Crystal structure of SARS-CoV-2 main protease provides a basis for design of improved α-ketoamide inhibitors</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>368</i></span>,  <span class="NLM_fpage">409</span>– <span class="NLM_lpage">412</span>, <span class="refDoi"> DOI: 10.1126/science.abb3405</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=10.1126%2Fscience.abb3405" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=32198291" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A528%3ADC%252BB3cXnslKrtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=368&publication_year=2020&pages=409-412&author=L.+Zhangauthor=D.+Linauthor=X.+Sunauthor=U.+Curthauthor=C.+Drostenauthor=L.+Sauerheringauthor=S.+Beckerauthor=K.+Roxauthor=R.+Hilgenfeld&title=Crystal+structure+of+SARS-CoV-2+main+protease+provides+a+basis+for+design+of+improved+%CE%B1-ketoamide+inhibitors&doi=10.1126%2Fscience.abb3405"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of SARS-CoV-2 main protease provides a basis for design of improved α-ketoamide inhibitors</span></div><div class="casAuthors">Zhang, Linlin; Lin, Daizong; Sun, Xinyuanyuan; Curth, Ute; Drosten, Christian; Sauerhering, Lucie; Becker, Stephan; Rox, Katharina; Hilgenfeld, Rolf</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">368</span>
        (<span class="NLM_cas:issue">6489</span>),
    <span class="NLM_cas:pages">409-412</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">1095-9203</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The coronavirus disease 2019 (COVID-19) pandemic caused by severe acute respiratory syndrome-coronavirus 2 (SARS-CoV-2) is a global health emergency.  An attractive drug target among coronaviruses is the main protease (Mpro, also called 3CLpro) because of its essential role in processing the polyproteins that are translated from the viral RNA.  We report the x-ray structures of the unliganded SARS-CoV-2 Mpro and its complex with an α-ketoamide inhibitor.  This was derived from a previously designed inhibitor but with the P3-P2 amide bond incorporated into a pyridone ring to enhance the half-life of the compd. in plasma.  On the basis of the unliganded structure, we developed the lead compd. into a potent inhibitor of the SARS-CoV-2 Mpro.  The pharmacokinetic characterization of the optimized inhibitor reveals a pronounced lung tropism and suitability for administration by the inhalative route.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrT531N47m4BbVg90H21EOLACvtfcHk0ljcj2V6Si1PWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXnslKrtL8%253D&md5=9ac417c20f54c3327f9de9088b512d52</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1126%2Fscience.abb3405&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.abb3405%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DLin%26aufirst%3DD.%26aulast%3DSun%26aufirst%3DX.%26aulast%3DCurth%26aufirst%3DU.%26aulast%3DDrosten%26aufirst%3DC.%26aulast%3DSauerhering%26aufirst%3DL.%26aulast%3DBecker%26aufirst%3DS.%26aulast%3DRox%26aufirst%3DK.%26aulast%3DHilgenfeld%26aufirst%3DR.%26atitle%3DCrystal%2520structure%2520of%2520SARS-CoV-2%2520main%2520protease%2520provides%2520a%2520basis%2520for%2520design%2520of%2520improved%2520%25CE%25B1-ketoamide%2520inhibitors%26jtitle%3DScience%26date%3D2020%26volume%3D368%26spage%3D409%26epage%3D412%26doi%3D10.1126%2Fscience.abb3405" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dai, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, X.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span> <span> </span><span class="NLM_article-title">Structure-based design of antiviral drug candidates targeting the SARS-CoV-2 main protease</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>368</i></span>,  <span class="NLM_fpage">1331</span>– <span class="NLM_lpage">1335</span>, <span class="refDoi"> DOI: 10.1126/science.abb4489</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=10.1126%2Fscience.abb4489" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=32321856" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A528%3ADC%252BB3cXht1alsrzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=368&publication_year=2020&pages=1331-1335&author=W.+Daiauthor=B.+Zhangauthor=X.-M.+Jiangauthor=H.+Suauthor=J.+Liauthor=Y.+Zhaoauthor=X.+Xieauthor=Z.+Jinauthor=J.+Pengauthor=F.+Liuauthor=C.+Liauthor=Y.+Liauthor=F.+Baiauthor=H.+Wangauthor=X.+Chengauthor=X.+Cenauthor=S.+Huauthor=X.+Yangauthor=J.+Wangauthor=X.+Liuauthor=G.+Xiaoauthor=H.+Jiangauthor=Z.+Raoauthor=L.-K.+Zhangauthor=Y.+Xuauthor=H.+Yangauthor=H.+Liu&title=Structure-based+design+of+antiviral+drug+candidates+targeting+the+SARS-CoV-2+main+protease&doi=10.1126%2Fscience.abb4489"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based design of antiviral drug candidates targeting the SARS-CoV-2 main protease</span></div><div class="casAuthors">Dai, Wenhao; Zhang, Bing; Jiang, Xia-Ming; Su, Haixia; Li, Jian; Zhao, Yao; Xie, Xiong; Jin, Zhenming; Peng, Jingjing; Liu, Fengjiang; Li, Chunpu; Li, You; Bai, Fang; Wang, Haofeng; Cheng, Xi; Cen, Xiaobo; Hu, Shulei; Yang, Xiuna; Wang, Jiang; Liu, Xiang; Xiao, Gengfu; Jiang, Hualiang; Rao, Zihe; Zhang, Lei-Ke; Xu, Yechun; Yang, Haitao; Liu, Hong</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">368</span>
        (<span class="NLM_cas:issue">6497</span>),
    <span class="NLM_cas:pages">1331-1335</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">1095-9203</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">SARS-CoV-2 (severe acute respiratory syndrome coronavirus 2) is the etiol. agent responsible for the global COVID-19 (coronavirus disease 2019) outbreak.  The main protease of SARS-CoV-2, Mpro, is a key enzyme that plays a pivotal role in mediating viral replication and transcription.  We designed and synthesized two lead compds. (11a and 11b) targeting Mpro.  Both exhibited excellent inhibitory activity and potent anti-SARS-CoV-2 infection activity.  The x-ray crystal structures of SARS-CoV-2 Mpro in complex with 11a or 11b, both detd. at a resoln. of 1.5 angstroms, showed that the aldehyde groups of 11a and 11b are covalently bound to cysteine 145 of Mpro.  Both compds. showed good pharmacokinetic properties in vivo, and 11a also exhibited low toxicity, which suggests that these compds. are promising drug candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFFvzOwCOpZrVg90H21EOLACvtfcHk0lhM8tkvc_8hig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXht1alsrzL&md5=ec87c340410edb919d2584a2b4d33e1a</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1126%2Fscience.abb4489&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.abb4489%26sid%3Dliteratum%253Aachs%26aulast%3DDai%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DB.%26aulast%3DJiang%26aufirst%3DX.-M.%26aulast%3DSu%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DXie%26aufirst%3DX.%26aulast%3DJin%26aufirst%3DZ.%26aulast%3DPeng%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DBai%26aufirst%3DF.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DCheng%26aufirst%3DX.%26aulast%3DCen%26aufirst%3DX.%26aulast%3DHu%26aufirst%3DS.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DXiao%26aufirst%3DG.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DRao%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DL.-K.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DH.%26atitle%3DStructure-based%2520design%2520of%2520antiviral%2520drug%2520candidates%2520targeting%2520the%2520SARS-CoV-2%2520main%2520protease%26jtitle%3DScience%26date%3D2020%26volume%3D368%26spage%3D1331%26epage%3D1335%26doi%3D10.1126%2Fscience.abb4489" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kania, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brothers, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferre, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gajiwala, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hogan, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozminski, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lockner, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marra, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, L. J.,  Jr</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nieman, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Planken, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rowe, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, G. J.,  III</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solowiej, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steppan, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taggart, B.</span></span> <span> </span><span class="NLM_article-title">Discovery of ketone-based covalent inhibitors of coronavirus 3CL proteases for the potential therapeutic treatment of COVID-19</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">12725</span>– <span class="NLM_lpage">12747</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c01063</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c01063" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A528%3ADC%252BB3cXitVKnur3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=12725-12747&author=R.+L.+Hoffmanauthor=R.+S.+Kaniaauthor=M.+A.+Brothersauthor=J.+F.+Daviesauthor=R.+A.+Ferreauthor=K.+S.+Gajiwalaauthor=M.+Heauthor=R.+J.+Hoganauthor=K.+Kozminskiauthor=L.+Y.+Liauthor=J.+W.+Locknerauthor=J.+Louauthor=M.+T.+Marraauthor=L.+J.+Mitchellauthor=B.+W.+Murrayauthor=J.+A.+Niemanauthor=S.+Noellauthor=S.+P.+Plankenauthor=T.+Roweauthor=K.+Ryanauthor=G.+J.+Smithauthor=J.+E.+Solowiejauthor=C.+M.+Steppanauthor=B.+Taggart&title=Discovery+of+ketone-based+covalent+inhibitors+of+coronavirus+3CL+proteases+for+the+potential+therapeutic+treatment+of+COVID-19&doi=10.1021%2Facs.jmedchem.0c01063"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of ketone-based covalent inhibitors of coronavirus 3CL proteases for the potential therapeutic treatment of COVID-19</span></div><div class="casAuthors">Hoffman, Robert L.; Kania, Robert S.; Brothers, Mary A.; Davies, Jay F.; Ferre, Rose A.; Gajiwala, Ketan S.; He, Mingying; Hogan, Robert J.; Kozminski, Kirk; Li, Lilian Y.; Lockner, Jonathan W.; Lou, Jihong; Marra, Michelle T.; Mitchell Jr., Lennert J.; Murray, Brion W.; Nieman, James A.; Noell, Stephen; Planken, Simon P.; Rowe, Thomas; Ryan, Kevin; Smith III, George J.; Solowiej, James E.; Steppan, Claire M.; Taggart, Barbara</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">12725-12747</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The novel coronavirus disease COVID-19 that emerged in 2019 is caused by the virus SARS CoV-2 and named for its close genetic similarity to SARS CoV-1 that caused severe acute respiratory syndrome (SARS) in 2002.  Both SARS coronavirus genomes encode two overlapping large polyproteins, which are cleaved at specific sites by a 3C-like cysteine protease (3CLpro) in a post-translational processing step that is crit. for coronavirus replication.  The 3CLpro sequences for CoV-1 and CoV-2 viruses are 100% identical in the catalytic domain that carries out protein cleavage.  A research effort that focused on the discovery of reversible and irreversible ketone-based inhibitors of SARS CoV-1 3CLpro employing ligand-protease structures solved by X-ray crystallog. led to the identification of 3 and 4.  Preclin. expts. reveal 4 (PF-00835231) as a potent inhibitor of CoV-2 3CLpro with suitable pharmaceutical properties to warrant further development as an i.v. treatment for COVID-19.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqP_zn3ZDefOrVg90H21EOLACvtfcHk0lhM8tkvc_8hig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXitVKnur3E&md5=7d5be169ad3de528496c862934d59881</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c01063%26sid%3Dliteratum%253Aachs%26aulast%3DHoffman%26aufirst%3DR.%2BL.%26aulast%3DKania%26aufirst%3DR.%2BS.%26aulast%3DBrothers%26aufirst%3DM.%2BA.%26aulast%3DDavies%26aufirst%3DJ.%2BF.%26aulast%3DFerre%26aufirst%3DR.%2BA.%26aulast%3DGajiwala%26aufirst%3DK.%2BS.%26aulast%3DHe%26aufirst%3DM.%26aulast%3DHogan%26aufirst%3DR.%2BJ.%26aulast%3DKozminski%26aufirst%3DK.%26aulast%3DLi%26aufirst%3DL.%2BY.%26aulast%3DLockner%26aufirst%3DJ.%2BW.%26aulast%3DLou%26aufirst%3DJ.%26aulast%3DMarra%26aufirst%3DM.%2BT.%26aulast%3DMitchell%26aufirst%3DL.%2BJ.%26aulast%3DMurray%26aufirst%3DB.%2BW.%26aulast%3DNieman%26aufirst%3DJ.%2BA.%26aulast%3DNoell%26aufirst%3DS.%26aulast%3DPlanken%26aufirst%3DS.%2BP.%26aulast%3DRowe%26aufirst%3DT.%26aulast%3DRyan%26aufirst%3DK.%26aulast%3DSmith%26aufirst%3DG.%2BJ.%26aulast%3DSolowiej%26aufirst%3DJ.%2BE.%26aulast%3DSteppan%26aufirst%3DC.%2BM.%26aulast%3DTaggart%26aufirst%3DB.%26atitle%3DDiscovery%2520of%2520ketone-based%2520covalent%2520inhibitors%2520of%2520coronavirus%25203CL%2520proteases%2520for%2520the%2520potential%2520therapeutic%2520treatment%2520of%2520COVID-19%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D12725%26epage%3D12747%26doi%3D10.1021%2Facs.jmedchem.0c01063" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Qiao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, R.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ni, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, Z.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, Q.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, Y.-H.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, M.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, R.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, G.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, G.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, Y.-Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.</span></span> <span> </span><span class="NLM_article-title">SARS-CoV-2 M<sup>pro</sup> inhibitors with antiviral activity in a transgenic mouse model</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>371</i></span>,  <span class="NLM_fpage">1374</span>– <span class="NLM_lpage">1378</span>, <span class="refDoi"> DOI: 10.1126/science.abf1611</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=10.1126%2Fscience.abf1611" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=33602867" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A528%3ADC%252BB3MXnsVyru7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=371&publication_year=2021&pages=1374-1378&author=J.+Qiaoauthor=Y.-S.+Liauthor=R.+Zengauthor=F.-L.+Liuauthor=R.-H.+Luoauthor=C.+Huangauthor=Y.-F.+Wangauthor=J.+Zhangauthor=B.+Quanauthor=C.+Shenauthor=X.+Maoauthor=X.+Liuauthor=W.+Sunauthor=W.+Yangauthor=X.+Niauthor=K.+Wangauthor=L.+Xuauthor=Z.-L.+Duanauthor=Q.-C.+Zouauthor=H.-L.+Zhangauthor=W.+Quauthor=Y.-H.-P.+Longauthor=M.-H.+Liauthor=R.-C.+Yangauthor=X.+Liuauthor=J.+Youauthor=Y.+Zhouauthor=R.+Yaoauthor=W.-P.+Liauthor=J.-M.+Liuauthor=P.+Chenauthor=Y.+Liuauthor=G.-F.+Linauthor=X.+Yangauthor=J.+Zouauthor=L.+Liauthor=Y.+Huauthor=G.-W.+Luauthor=W.-M.+Liauthor=Y.-Q.+Weiauthor=Y.-T.+Zhengauthor=J.+Leiauthor=S.+Yang&title=SARS-CoV-2+Mpro+inhibitors+with+antiviral+activity+in+a+transgenic+mouse+model&doi=10.1126%2Fscience.abf1611"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">SARS-CoV-2 Mpro inhibitors with antiviral activity in a transgenic mouse model</span></div><div class="casAuthors">Qiao, Jingxin; Li, Yue-Shan; Zeng, Rui; Liu, Feng-Liang; Luo, Rong-Hua; Huang, Chong; Wang, Yi-Fei; Zhang, Jie; Quan, Baoxue; Shen, Chenjian; Mao, Xin; Liu, Xinlei; Sun, Weining; Yang, Wei; Ni, Xincheng; Wang, Kai; Xu, Ling; Duan, Zi-Lei; Zou, Qing-Cui; Zhang, Hai-Lin; Qu, Wang; Long, Yang-Hao-Peng; Li, Ming-Hua; Yang, Rui-Cheng; Liu, Xiaolong; You, Jing; Zhou, Yangli; Yao, Rui; Li, Wen-Pei; Liu, Jing-Ming; Chen, Pei; Liu, Yang; Lin, Gui-Feng; Yang, Xin; Zou, Jun; Li, Linli; Hu, Yiguo; Lu, Guang-Wen; Li, Wei-Min; Wei, Yu-Quan; Zheng, Yong-Tang; Lei, Jian; Yang, Shengyong</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">371</span>
        (<span class="NLM_cas:issue">6536</span>),
    <span class="NLM_cas:pages">1374-1378</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">1095-9203</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The COVID-19 pandemic caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) continually poses serious threats to global public health.  The main protease (Mpro) of SARS-CoV-2 plays a central role in viral replication.  We designed and synthesized 32 new bicycloproline-contg. Mpro inhibitors derived from either boceprevir or telaprevir, both of which are approved antivirals.  All compds. inhibited SARS-CoV-2 Mpro activity in vitro, with 50% inhibitory concn. values ranging from 7.6 to 748.5 nM.  The cocrystal structure of Mpro in complex with MI-23, one of the most potent compds., revealed its interaction mode.  Two compds. (MI-09 and MI-30) showed excellent antiviral activity in cell-based assays.  In a transgenic mouse model of SARS-CoV-2 infection, oral or i.p. treatment with MI-09 or MI-30 significantly reduced lung viral loads and lung lesions.  Both also displayed good pharmacokinetic properties and safety in rats.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVMv8s8s-oWLVg90H21EOLACvtfcHk0liCnErCnturJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXnsVyru7g%253D&md5=555937b1776c8353e59601d567e2ebb9</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1126%2Fscience.abf1611&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.abf1611%26sid%3Dliteratum%253Aachs%26aulast%3DQiao%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DY.-S.%26aulast%3DZeng%26aufirst%3DR.%26aulast%3DLiu%26aufirst%3DF.-L.%26aulast%3DLuo%26aufirst%3DR.-H.%26aulast%3DHuang%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DY.-F.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DQuan%26aufirst%3DB.%26aulast%3DShen%26aufirst%3DC.%26aulast%3DMao%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DSun%26aufirst%3DW.%26aulast%3DYang%26aufirst%3DW.%26aulast%3DNi%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DK.%26aulast%3DXu%26aufirst%3DL.%26aulast%3DDuan%26aufirst%3DZ.-L.%26aulast%3DZou%26aufirst%3DQ.-C.%26aulast%3DZhang%26aufirst%3DH.-L.%26aulast%3DQu%26aufirst%3DW.%26aulast%3DLong%26aufirst%3DY.-H.-P.%26aulast%3DLi%26aufirst%3DM.-H.%26aulast%3DYang%26aufirst%3DR.-C.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DYou%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DYao%26aufirst%3DR.%26aulast%3DLi%26aufirst%3DW.-P.%26aulast%3DLiu%26aufirst%3DJ.-M.%26aulast%3DChen%26aufirst%3DP.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DLin%26aufirst%3DG.-F.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DZou%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DLu%26aufirst%3DG.-W.%26aulast%3DLi%26aufirst%3DW.-M.%26aulast%3DWei%26aufirst%3DY.-Q.%26aulast%3DZheng%26aufirst%3DY.-T.%26aulast%3DLei%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DS.%26atitle%3DSARS-CoV-2%2520Mpro%2520inhibitors%2520with%2520antiviral%2520activity%2520in%2520a%2520transgenic%2520mouse%2520model%26jtitle%3DScience%26date%3D2021%26volume%3D371%26spage%3D1374%26epage%3D1378%26doi%3D10.1126%2Fscience.abf1611" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schechter, I.</span></span> <span> </span><span class="NLM_article-title">Reprint of “on the size of the active site in proteases. I. Papain”</span>. <i>Biochem. Biophys. Res. Com.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>425</i></span>,  <span class="NLM_fpage">497</span>– <span class="NLM_lpage">502</span>, <span class="refDoi"> DOI: 10.1016/j.bbrc.2012.08.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=10.1016%2Fj.bbrc.2012.08.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=22925665" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=425&publication_year=2012&pages=497-502&author=I.+Schechter&title=Reprint+of+%E2%80%9Con+the+size+of+the+active+site+in+proteases.+I.+Papain%E2%80%9D&doi=10.1016%2Fj.bbrc.2012.08.015"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2012.08.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2012.08.015%26sid%3Dliteratum%253Aachs%26aulast%3DSchechter%26aufirst%3DI.%26atitle%3DReprint%2520of%2520%25E2%2580%259Con%2520the%2520size%2520of%2520the%2520active%2520site%2520in%2520proteases.%2520I.%2520Papain%25E2%2580%259D%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Com.%26date%3D2012%26volume%3D425%26spage%3D497%26epage%3D502%26doi%3D10.1016%2Fj.bbrc.2012.08.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="note"><p class="first last">. The nomenclature used is that of Schechter, I. and Berger, A., where the residues on the N-terminus side of the peptide bond that is cleaved are designated as P<sub>1</sub>–P<sub>n</sub> and those on the C-terminus side are designated P<sub>1</sub>–P<sub>1</sub>′. The corresponding active-site subsites are designated S<sub>1</sub>–S<sub>n</sub> and S<sub>1</sub>–S<sub>n</sub>′. S<sub>1</sub> is the primary substrate specificity subsite, and P<sub>1</sub>–P<sub>1</sub>′ is the scissile bond</p></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rathnayake, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perera, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mackin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyerholz, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kashipathy, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Battaile, K. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lovell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perlman, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groutas, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, K.-O.</span></span> <span> </span><span class="NLM_article-title">3C-like protease inhibitors block coronavirus replication in vitro and improve survival in MERS-CoV-infected mice</span>. <i>Sci. Transl. Med.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>12</i></span>, <span class="NLM_elocation-id">eabc5332</span> <span class="refDoi"> DOI: 10.1126/scitranslmed.abc5332</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=10.1126%2Fscitranslmed.abc5332" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=32747425" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2020&author=A.+D.+Rathnayakeauthor=J.+Zhengauthor=Y.+Kimauthor=K.+D.+Pereraauthor=S.+Mackinauthor=D.+K.+Meyerholzauthor=M.+M.+Kashipathyauthor=K.+P.+Battaileauthor=S.+Lovellauthor=S.+Perlmanauthor=W.+C.+Groutasauthor=K.-O.+Chang&title=3C-like+protease+inhibitors+block+coronavirus+replication+in+vitro+and+improve+survival+in+MERS-CoV-infected+mice&doi=10.1126%2Fscitranslmed.abc5332"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1126%2Fscitranslmed.abc5332&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscitranslmed.abc5332%26sid%3Dliteratum%253Aachs%26aulast%3DRathnayake%26aufirst%3DA.%2BD.%26aulast%3DZheng%26aufirst%3DJ.%26aulast%3DKim%26aufirst%3DY.%26aulast%3DPerera%26aufirst%3DK.%2BD.%26aulast%3DMackin%26aufirst%3DS.%26aulast%3DMeyerholz%26aufirst%3DD.%2BK.%26aulast%3DKashipathy%26aufirst%3DM.%2BM.%26aulast%3DBattaile%26aufirst%3DK.%2BP.%26aulast%3DLovell%26aufirst%3DS.%26aulast%3DPerlman%26aufirst%3DS.%26aulast%3DGroutas%26aufirst%3DW.%2BC.%26aulast%3DChang%26aufirst%3DK.-O.%26atitle%3D3C-like%2520protease%2520inhibitors%2520block%2520coronavirus%2520replication%2520in%2520vitro%2520and%2520improve%2520survival%2520in%2520MERS-CoV-infected%2520mice%26jtitle%3DSci.%2520Transl.%2520Med.%26date%3D2020%26volume%3D12%26doi%3D10.1126%2Fscitranslmed.abc5332" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galasiti
Kankanamalage, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weerasekara, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hua, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groutas, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, K.-O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pedersen, N. C.</span></span> <span> </span><span class="NLM_article-title">Reversal of the progression of fatal coronavirus infection in cats by a broad-spectrum coronavirus protease inhibitor</span>. <i>PLoS Pathog.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>, <span class="NLM_elocation-id">e1005531</span> <span class="refDoi"> DOI: 10.1371/journal.ppat.1005531</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=10.1371%2Fjournal.ppat.1005531" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=27027316" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlGru7fN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&author=Y.+Kimauthor=H.+Liuauthor=A.+C.+Galasiti%0AKankanamalageauthor=S.+Weerasekaraauthor=D.+H.+Huaauthor=W.+C.+Groutasauthor=K.-O.+Changauthor=N.+C.+Pedersen&title=Reversal+of+the+progression+of+fatal+coronavirus+infection+in+cats+by+a+broad-spectrum+coronavirus+protease+inhibitor&doi=10.1371%2Fjournal.ppat.1005531"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Reversal of the progression of fatal coronavirus infection in cats by a broad-spectrum coronavirus protease inhibitor</span></div><div class="casAuthors">Kim, Yunjeong; Liu, Hongwei; Kankanamalage, Anushka C. Galasiti; Weerasekara, Sahani; Hua, Duy H.; Groutas, William C.; Chang, Kyeong-Ok; Pedersen, Niels C.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Pathogens</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">e1005531/1-e1005531/18</span>CODEN:
                <span class="NLM_cas:coden">PPLACN</span>;
        ISSN:<span class="NLM_cas:issn">1553-7374</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Coronaviruses infect animals and humans causing a wide range of diseases.  The diversity of coronaviruses in many mammalian species is contributed by relatively high mutation and recombination rates during replication.  This dynamic nature of coronaviruses may facilitate cross-species transmission and shifts in tissue or cell tropism in a host, resulting in substantial change in virulence.  Feline enteric coronavirus (FECV) causes inapparent or mild enteritis in cats, but a highly fatal disease, called feline infectious peritonitis (FIP), can arise through mutation of FECV to FIP virus (FIPV).  The pathogenesis of FIP is intimately assocd. with immune responses and involves depletion of T cells, features shared by some other coronaviruses like Severe Acute Respiratory Syndrome Coronavirus.  The increasing risks of highly virulent coronavirus infections in humans or animals call for effective antiviral drugs, but no such measures are yet available.  Previously, we have reported the inhibitors that target 3C-like protease (3CLpro) with broad-spectrum activity against important human and animal coronaviruses.  Here, we evaluated the therapeutic efficacy of our 3CLpro inhibitor in lab. cats with FIP.  Exptl. FIP is 100% fatal once certain clin. and lab. signs become apparent.We found that antiviral treatment led to full recovery of cats when treatment was started at a stage of disease that would be otherwise fatal if left untreated.  Antiviral treatment was assocd. with a rapid improvement in fever, ascites, lymphopenia and gross signs of illness and cats returned to normal health within 20 days or less of treatment.  Significant redn. in viral titers was also obsd. in cats.  These results indicate that continuous virus replication is required for progression of immune-mediated inflammatory disease of FIP.  These findings may provide important insights into devising therapeutic strategies and selection of antiviral compds. for further development for important coronaviruses in animals and humans.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprSH3BnhFwZLVg90H21EOLACvtfcHk0ljeGjcdOJPjPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlGru7fN&md5=7900b2718caeaf528f10f553e0e8fa16</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1371%2Fjournal.ppat.1005531&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.ppat.1005531%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DGalasiti%2BKankanamalage%26aufirst%3DA.%2BC.%26aulast%3DWeerasekara%26aufirst%3DS.%26aulast%3DHua%26aufirst%3DD.%2BH.%26aulast%3DGroutas%26aufirst%3DW.%2BC.%26aulast%3DChang%26aufirst%3DK.-O.%26aulast%3DPedersen%26aufirst%3DN.%2BC.%26atitle%3DReversal%2520of%2520the%2520progression%2520of%2520fatal%2520coronavirus%2520infection%2520in%2520cats%2520by%2520a%2520broad-spectrum%2520coronavirus%2520protease%2520inhibitor%26jtitle%3DPLoS%2520Pathog.%26date%3D2016%26volume%3D12%26doi%3D10.1371%2Fjournal.ppat.1005531" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pedersen, N. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galasiti Kankanamalage, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eckstrand, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groutas, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bannasch, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meadows, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, K.-O.</span></span> <span> </span><span class="NLM_article-title">Efficacy of a 3C-like protease inhibitor in treating various forms of acquired feline infectious peritonitis</span>. <i>J. Feline Med. Surg.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">378</span>– <span class="NLM_lpage">392</span>, <span class="refDoi"> DOI: 10.1177/1098612x17729626</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=10.1177%2F1098612x17729626" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=28901812" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A280%3ADC%252BC1cbns12gsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2018&pages=378-392&author=N.+C.+Pedersenauthor=Y.+Kimauthor=H.+Liuauthor=A.+C.+Galasiti+Kankanamalageauthor=C.+Eckstrandauthor=W.+C.+Groutasauthor=M.+Bannaschauthor=J.+M.+Meadowsauthor=K.-O.+Chang&title=Efficacy+of+a+3C-like+protease+inhibitor+in+treating+various+forms+of+acquired+feline+infectious+peritonitis&doi=10.1177%2F1098612x17729626"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Efficacy of a 3C-like protease inhibitor in treating various forms of acquired feline infectious peritonitis</span></div><div class="casAuthors">Pedersen Niels C; Liu Hongwei; Bannasch Michael; Kim Yunjeong; Chang Kyeong-Ok; Galasiti Kankanamalage Anushka C; Groutas William C; Eckstrand Chrissy; Meadows Juliana M</div><div class="citationInfo"><span class="NLM_cas:title">Journal of feline medicine and surgery</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">378-392</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Objectives The safety and efficacy of the 3C-like protease inhibitor GC376 was tested on a cohort of client-owned cats with various forms of feline infectious peritonitis (FIP).  Methods Twenty cats from 3.3-82 months of age (mean 10.4 months) with various forms of FIP were accepted into a field trial.  Fourteen cats presented with wet or dry-to-wet FIP and six cats presented with dry FIP.  GC376 was administered subcutaneously every 12 h at a dose of 15 mg/kg.  Cats with neurologic signs were excluded from the study.  Results Nineteen of 20 cats treated with GC376 regained outward health within 2 weeks of initial treatment.  However, disease signs recurred 1-7 weeks after primary treatment and relapses and new cases were ultimately treated for a minimum of 12 weeks.  Relapses no longer responsive to treatment occurred in 13 of these 19 cats within 1-7 weeks of initial or repeat treatment(s).  Severe neurologic disease occurred in 8/13 cats that failed treatment and five cats had recurrences of abdominal lesions.  At the time of writing, seven cats were in disease remission.  Five kittens aged 3.3-4.4 months with wet FIP were treated for 12 weeks and have been in disease remission after stopping treatment and at the time of writing for 5-14 months (mean 11.2 months).  A sixth kitten was in remission for 10 weeks after 12 weeks of treatment, relapsed and is responding to a second round of GC376.  The seventh was a 6.8-year-old cat with only mesenteric lymph node involvement that went into remission after three relapses that required progressively longer repeat treatments over a 10 month period.  Side effects of treatment included transient stinging upon injection and occasional foci of subcutaneous fibrosis and hair loss.  There was retarded development and abnormal eruption of permanent teeth in cats treated before 16-18 weeks of age.  Conclusions and relevance GC376 showed promise in treating cats with certain presentations of FIP and has opened the door to targeted antiviral drug therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTFFwhHX-Ibonqj_yg7twvpfW6udTcc2eY_4KvPlqHX4bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cbns12gsg%253D%253D&md5=69b4ebebb24b351bfb0f409e275b3153</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1177%2F1098612x17729626&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1098612x17729626%26sid%3Dliteratum%253Aachs%26aulast%3DPedersen%26aufirst%3DN.%2BC.%26aulast%3DKim%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DGalasiti%2BKankanamalage%26aufirst%3DA.%2BC.%26aulast%3DEckstrand%26aufirst%3DC.%26aulast%3DGroutas%26aufirst%3DW.%2BC.%26aulast%3DBannasch%26aufirst%3DM.%26aulast%3DMeadows%26aufirst%3DJ.%2BM.%26aulast%3DChang%26aufirst%3DK.-O.%26atitle%3DEfficacy%2520of%2520a%25203C-like%2520protease%2520inhibitor%2520in%2520treating%2520various%2520forms%2520of%2520acquired%2520feline%2520infectious%2520peritonitis%26jtitle%3DJ.%2520Feline%2520Med.%2520Surg.%26date%3D2018%26volume%3D20%26spage%3D378%26epage%3D392%26doi%3D10.1177%2F1098612x17729626" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pirali, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serafini, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cargnin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Genazzani, A. A.</span></span> <span> </span><span class="NLM_article-title">Applications of deuterium in medicinal chemistry</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">5276</span>– <span class="NLM_lpage">5297</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01808</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01808" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A528%3ADC%252BC1MXpvFKjtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=5276-5297&author=T.+Piraliauthor=M.+Serafiniauthor=S.+Cargninauthor=A.+A.+Genazzani&title=Applications+of+deuterium+in+medicinal+chemistry&doi=10.1021%2Facs.jmedchem.8b01808"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Applications of deuterium in medicinal chemistry</span></div><div class="casAuthors">Pirali, Tracey; Serafini, Marta; Cargnin, Sarah; Genazzani, Armando A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">5276-5297</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The use of deuteration in medicinal chem. has exploded in the past years, and the FDA has recently approved the first deuterium-labeled drug.  Precision deuteration goes beyond the pure and simple amelioration of the pharmacokinetic parameters of a drug and might provide an opportunity when facing problems in terms of metab.-mediated toxicity, drug interactions, and low bioactivation.  The use of deuterium is even broader, offering the opportunity to lower the degree of epimerization, reduce the dose of coadministered boosters, and discover compds. where deuterium is the basis for the mechanism of action.  Nevertheless, designing, synthesizing, and developing a successful deuterated drug is far from straightforward, and the translation from concept to practice is often unpredictable.  This Perspective provides an overview of the recent developments of deuteration, with a focus on deuterated clin. candidates, and highlights both opportunities and challenges of this strategy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1ICy0AH9zGLVg90H21EOLACvtfcHk0ljeGjcdOJPjPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXpvFKjtw%253D%253D&md5=1ce58dbff268894dc319bfc00a1ae01f</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01808&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01808%26sid%3Dliteratum%253Aachs%26aulast%3DPirali%26aufirst%3DT.%26aulast%3DSerafini%26aufirst%3DM.%26aulast%3DCargnin%26aufirst%3DS.%26aulast%3DGenazzani%26aufirst%3DA.%2BA.%26atitle%3DApplications%2520of%2520deuterium%2520in%2520medicinal%2520chemistry%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D5276%26epage%3D5297%26doi%3D10.1021%2Facs.jmedchem.8b01808" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Timmins, G. S.</span></span> <span> </span><span class="NLM_article-title">Deuterated drugs; updates and obviousness analysis</span>. <i>Expert Opin. Ther. Pat.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">1353</span>– <span class="NLM_lpage">1361</span>, <span class="refDoi"> DOI: 10.1080/13543776.2017.1378350</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=10.1080%2F13543776.2017.1378350" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=28885861" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFSmtbjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2017&pages=1353-1361&author=G.+S.+Timmins&title=Deuterated+drugs%3B+updates+and+obviousness+analysis&doi=10.1080%2F13543776.2017.1378350"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Deuterated drugs; updates and obviousness analysis</span></div><div class="casAuthors">Timmins, Graham S.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Patents</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1353-1361</span>CODEN:
                <span class="NLM_cas:coden">EOTPEG</span>;
        ISSN:<span class="NLM_cas:issn">1354-3776</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">The pharmacokinetics and/or toxicity of many known drugs can be modified by selective deuteration, an area of significant com. interest and scientific and regulatory progress.  This review firstly discusses recent developments in deuterated drugs including the FDA approval of deutetrabenazine.  Secondly, it discusses 35 U.S.C. Sec. 103 'obviousness' as it relates to recent patent prosecution, and also to Inter Partes Review (IPR).  IPR is a new post-award review of patentability under Sec. 102 or Sec. 103, two IPR petitions upon deuterated drugs have been instituted and included Sec. 103 arguments.  Finally, an extended anal. of Sec. 103 obviousness based upon the practices of major pharmaceutical companies is provided, that supports rather late priority dates, while Sec. 102 is also discussed.  The total value of transactions involving deuterated drugs is close to $5 billion.  While the importance of Sec. 103 'obviousness' rejections remains in patent applications under current prosecution, IPR of issued patents is developing and will affect likely affect Sec. 103 interpretations in this area.  However, patents are still issuing with later priority dates, and further litigation will likely occur.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplgaJXt_pSD7Vg90H21EOLACvtfcHk0lh68Bi6IlxPFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFSmtbjO&md5=d29367a0daec2c70367a58327fbccd9c</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1080%2F13543776.2017.1378350&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F13543776.2017.1378350%26sid%3Dliteratum%253Aachs%26aulast%3DTimmins%26aufirst%3DG.%2BS.%26atitle%3DDeuterated%2520drugs%253B%2520updates%2520and%2520obviousness%2520analysis%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Pat.%26date%3D2017%26volume%3D27%26spage%3D1353%26epage%3D1361%26doi%3D10.1080%2F13543776.2017.1378350" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harbeson, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brummel, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tung, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silverman, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doller, D.</span></span> <span> </span><span class="NLM_article-title">A decade of deuteration in medicinal chemistry</span>. <i>Annu. Rep. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">519</span>– <span class="NLM_lpage">542</span>, <span class="refDoi"> DOI: 10.1016/bs.armc.2017.08.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=10.1016%2Fbs.armc.2017.08.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFOlsrvF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2017&pages=519-542&author=J.+F.+Liuauthor=S.+L.+Harbesonauthor=C.+L.+Brummelauthor=R.+Tungauthor=R.+Silvermanauthor=D.+Doller&title=A+decade+of+deuteration+in+medicinal+chemistry&doi=10.1016%2Fbs.armc.2017.08.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">A decade of deuteration in medicinal chemistry</span></div><div class="casAuthors">Liu, Julie F.; Harbeson, Scott L.; Brummel, Christopher L.; Tung, Roger; Silverman, Robert; Doller, Dario</div><div class="citationInfo"><span class="NLM_cas:title">Annual Reports in Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">519-542</span>CODEN:
                <span class="NLM_cas:coden">ARMCBI</span>;
        ISSN:<span class="NLM_cas:issn">0065-7743</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  The use of deuterium to drive DIEs and favorably alter pharmacokinetics and metab. in drug discovery has accelerated in the last decade.  The first major wave of patent applications came from small biotechnol. organizations and were aimed at improving pharmacokinetic properties of marketed drugs or enhancing properties offailed drug candidates.  A substantial no. of patent applications were published beginning in 2006, and the deuterium chem. patent estate continues to expand.  Deuterated drug candidates have progressed from early patent filings to successful proof-of-concept human studies, including the first FDA-approved deuterated drug.  The use of DIEs in drug design has expanded, and now deuteration of drug candidates and tool compds., including imaging ligands, has become a welcome instrument in the arsenal of medicinal chemists.  It is expected that experimentation with strategic introduction of deuterium into the structures of org. compds. will continue to increase and may enable the future development ofa variety ofnovel drugs and chem. tools.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRWRQHDkKt5LVg90H21EOLACvtfcHk0lh68Bi6IlxPFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFOlsrvF&md5=18499ed5ccfb021913c4dea9e51946b7</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fbs.armc.2017.08.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fbs.armc.2017.08.010%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DJ.%2BF.%26aulast%3DHarbeson%26aufirst%3DS.%2BL.%26aulast%3DBrummel%26aufirst%3DC.%2BL.%26aulast%3DTung%26aufirst%3DR.%26aulast%3DSilverman%26aufirst%3DR.%26aulast%3DDoller%26aufirst%3DD.%26atitle%3DA%2520decade%2520of%2520deuteration%2520in%2520medicinal%2520chemistry%26jtitle%3DAnnu.%2520Rep.%2520Med.%2520Chem.%26date%3D2017%26volume%3D50%26spage%3D519%26epage%3D542%26doi%3D10.1016%2Fbs.armc.2017.08.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doung, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKee, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, W. J.</span></span> <span> </span><span class="NLM_article-title">N,N-dissuccinimidyl carbonate: a useful reagent for alkoxycarbonylation of amines</span>. <i>Tetrahedron Lett.</i> <span class="NLM_year" style="font-weight: bold;">1992</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">2781</span>– <span class="NLM_lpage">2784</span>, <span class="refDoi"> DOI: 10.1016/s0040-4039(00)78856-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=10.1016%2Fs0040-4039%2800%2978856-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=30310243" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A528%3ADyaK38XkvFSlsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=1992&pages=2781-2784&author=A.+K.+Ghoshauthor=T.+T.+Doungauthor=S.+P.+McKeeauthor=W.+J.+Thompson&title=N%2CN-dissuccinimidyl+carbonate%3A+a+useful+reagent+for+alkoxycarbonylation+of+amines&doi=10.1016%2Fs0040-4039%2800%2978856-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">N,N'-Disuccinimidyl carbonate:  a useful reagent for alkoxycarbonylation of amines</span></div><div class="casAuthors">Ghosh, Arun K.; Duong, Tien T.; McKee, Sean P.; Thompson, Wayne J.</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron Letters</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">2781-4</span>CODEN:
                <span class="NLM_cas:coden">TELEAY</span>;
        ISSN:<span class="NLM_cas:issn">0040-4039</span>.
    </div><div class="casAbstract">An efficient and mild method for alkoxycarbonylation of amines is described, utilizing com. available N,N'-disuccinimidyl carbonate (I).  Transesterification of I with 1,2-O-isopropylidene-D-xylofuranose gave the activated ester II which upon treatment with amines gave carbamates, e.g. III.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpR1G8uUUw3crVg90H21EOLACvtfcHk0lh68Bi6IlxPFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38XkvFSlsrY%253D&md5=167660e9aebebc06a9020024d50825f6</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fs0040-4039%2800%2978856-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fs0040-4039%252800%252978856-3%26sid%3Dliteratum%253Aachs%26aulast%3DGhosh%26aufirst%3DA.%2BK.%26aulast%3DDoung%26aufirst%3DT.%2BT.%26aulast%3DMcKee%26aufirst%3DS.%2BP.%26aulast%3DThompson%26aufirst%3DW.%2BJ.%26atitle%3DN%252CN-dissuccinimidyl%2520carbonate%253A%2520a%2520useful%2520reagent%2520for%2520alkoxycarbonylation%2520of%2520amines%26jtitle%3DTetrahedron%2520Lett.%26date%3D1992%26volume%3D33%26spage%3D2781%26epage%3D2784%26doi%3D10.1016%2Fs0040-4039%2800%2978856-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kjell, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slattery, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Semo, M. J.</span></span> <span> </span><span class="NLM_article-title">A novel, nonaqueous method for regeneration of aldehydes from bisulfite adducts</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>64</i></span>,  <span class="NLM_fpage">5722</span>– <span class="NLM_lpage">5724</span>, <span class="refDoi"> DOI: 10.1021/jo990543v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo990543v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A528%3ADyaK1MXktFSkur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=1999&pages=5722-5724&author=D.+P.+Kjellauthor=B.+J.+Slatteryauthor=M.+J.+Semo&title=A+novel%2C+nonaqueous+method+for+regeneration+of+aldehydes+from+bisulfite+adducts&doi=10.1021%2Fjo990543v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">A Novel, Nonaqueous Method for Regeneration of Aldehydes from Bisulfite Adducts</span></div><div class="casAuthors">Kjell, Douglas P.; Slattery, Brian J.; Semo, Michael J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">5722-5724</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Aldehydes have been generated from their bisulfite adducts under nonaq. conditions by treatment of the bisulfite addn. compd. with excess trimethylsilyl chloride in acetonitrile to give the aldehydes in 95-100% yield.  E.g., in the prepn. of an oncolytic agent, intermediate aldehyde I, prepd. by the palladium-catalyzed coupling of Me 4-iodobenzoate with 3-buten-1-ol, and susceptible to ester hydrolysis in the presence of water and base, is purified from a reaction mixt. by treatment with sodium bisulfite in ethanol-water to give the bisulfite addn. compd. II; treatment of II with >2 equiv. Me3SiCl in acetonitrile and heating for 2h regenerated I in 90% yield from II.  E.g., Et formylcyclopropanecarboxylate III (as an 88:12 mixt. with the cis diastereomer) was treated with sodium bisulfite to give the bisulfite adduct IV in 61% yield as a single diastereomer; treatment of IV with >2 equiv. Me3SiCl in MeCN gave III in 98% yield.  Acetonitrile was the only effective solvent; methylene chloride, Et acetate, and THF give incomplete reaction, while more polar alc. solvents, DMF and DMSO all react with Me3SiCl and are incompatible.  Other trimethylsilyl compds., such as 1-trimethylsilylimidazole and N,O-bis(trimethylsilyl)acetamide, are also effective in the generation of aldehydes from bisulfite addn. compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRzgg7pFpbpLVg90H21EOLACvtfcHk0lhJsKA5qm-qvg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXktFSkur8%253D&md5=30e4e9dd7a6b8ef13427bfc1050f8fa5</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fjo990543v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo990543v%26sid%3Dliteratum%253Aachs%26aulast%3DKjell%26aufirst%3DD.%2BP.%26aulast%3DSlattery%26aufirst%3DB.%2BJ.%26aulast%3DSemo%26aufirst%3DM.%2BJ.%26atitle%3DA%2520novel%252C%2520nonaqueous%2520method%2520for%2520regeneration%2520of%2520aldehydes%2520from%2520bisulfite%2520adducts%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D1999%26volume%3D64%26spage%3D5722%26epage%3D5724%26doi%3D10.1021%2Fjo990543v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lovell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tiew, K.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mandadapu, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alliston, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Battaile, K. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groutas, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, K.-O.</span></span> <span> </span><span class="NLM_article-title">Broad-spectrum antivirals against 3C or 3C-like proteases of picornaviruses, noroviruses, and coronaviruses</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>86</i></span>,  <span class="NLM_fpage">11754</span>– <span class="NLM_lpage">11762</span>, <span class="refDoi"> DOI: 10.1128/jvi.01348-12</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=10.1128%2Fjvi.01348-12" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=22915796" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFCmur3M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=86&publication_year=2012&pages=11754-11762&author=Y.+Kimauthor=S.+Lovellauthor=K.-C.+Tiewauthor=S.+R.+Mandadapuauthor=K.+R.+Allistonauthor=K.+P.+Battaileauthor=W.+C.+Groutasauthor=K.-O.+Chang&title=Broad-spectrum+antivirals+against+3C+or+3C-like+proteases+of+picornaviruses%2C+noroviruses%2C+and+coronaviruses&doi=10.1128%2Fjvi.01348-12"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Broad-spectrum antivirals against 3C or 3C-like proteases of picornaviruses, noroviruses, and coronaviruses</span></div><div class="casAuthors">Kim, Yunjeong; Lovell, Scott; Tiew, Kok-Chuan; Mandadapu, Sivakoteswara Rao; Alliston, Kevin R.; Battaile, Kevin P.; Groutas, William C.; Chang, Kyeong-Ok</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">86</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">11754-11762</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Phylogenetic anal. has demonstrated that some pos.-sense RNA viruses can be classified into the picornavirus-like super-cluster, which includes picornaviruses, caliciviruses, and coronaviruses.  These viruses possess 3C or 3C-like proteases (3Cpro or 3CLpro, resp.), which contain a typical chymotrypsin-like fold and a catalytic triad (or dyad) with a Cys residue as a nucleophile.  The conserved key sites of 3Cpro or 3CLpro may serve as attractive targets for the design of broad-spectrum antivirals for multiple viruses in the supercluster.  We previously reported the structure-based design and synthesis of potent protease inhibitors of Norwalk virus (NV), a member of the Caliciviridae family.  We report herein the broad-spectrum antiviral activities of three compds. possessing a common dipeptidyl residue with different warheads, i.e., an aldehyde (GC373), a bisulfite adduct (GC376), and an α-ketoamide (GC375), against viruses that belong to the supercluster.  All compds. were highly effective against the majority of tested viruses, with half-maximal inhibitory concns. in the high nanomolar or low micromolar range in enzyme- and/or cell-based assays and with high therapeutic indexes.  We also report the high-resoln. X-ray cocrystal structures of NV 3CLpro-, poliovirus 3Cpro-, and transmissible gastroenteritis virus 3CLpro- GC376 inhibitor complexes, which show the compd. covalently bound to a nucleophilic Cys residue in the catalytic site of the corresponding protease.  We conclude that these compds. have the potential to be developed as antiviral therapeutics aimed at a single virus or multiple viruses in the picornavirus-like supercluster by targeting 3Cpro or 3CLpro.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDn4MUH7bIlLVg90H21EOLACvtfcHk0lhJsKA5qm-qvg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFCmur3M&md5=c7fe984b52490199a2c2ef333c22fb98</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1128%2Fjvi.01348-12&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252Fjvi.01348-12%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DY.%26aulast%3DLovell%26aufirst%3DS.%26aulast%3DTiew%26aufirst%3DK.-C.%26aulast%3DMandadapu%26aufirst%3DS.%2BR.%26aulast%3DAlliston%26aufirst%3DK.%2BR.%26aulast%3DBattaile%26aufirst%3DK.%2BP.%26aulast%3DGroutas%26aufirst%3DW.%2BC.%26aulast%3DChang%26aufirst%3DK.-O.%26atitle%3DBroad-spectrum%2520antivirals%2520against%25203C%2520or%25203C-like%2520proteases%2520of%2520picornaviruses%252C%2520noroviruses%252C%2520and%2520coronaviruses%26jtitle%3DJ.%2520Virol.%26date%3D2012%26volume%3D86%26spage%3D11754%26epage%3D11762%26doi%3D10.1128%2Fjvi.01348-12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gampe, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verma, V. A.</span></span> <span> </span><span class="NLM_article-title">Curse or cure? A perspective on the developability of aldehydes as active pharmaceutical ingredients</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">14357</span>– <span class="NLM_lpage">14381</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c01177</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c01177" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhvVWhs7vE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=14357-14381&author=C.+Gampeauthor=V.+A.+Verma&title=Curse+or+cure%3F+A+perspective+on+the+developability+of+aldehydes+as+active+pharmaceutical+ingredients&doi=10.1021%2Facs.jmedchem.0c01177"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Curse or Cure? A Perspective on the Developability of Aldehydes as Active Pharmaceutical Ingredients</span></div><div class="casAuthors">Gampe, Christian; Verma, Vishal A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">14357-14381</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  There is a dichotomy between the increasing utility of aldehydes as tool mols. that bind to "undruggable" protein sites and the designation of the aldehyde functional group as a structural alert by the medicinal chem. community.  Herein we compile information about pharmacol. active aldehydes that are being used in humans.  We summarize the learnings from these data, discuss advantages and challenges assocd. with aldehydes, and derive strategies for the successful development of aldehydes within drug discovery programs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppz1oqp1WGV7Vg90H21EOLACvtfcHk0lhJsKA5qm-qvg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhvVWhs7vE&md5=4696123fdaba8f20895bb11f2b78ff84</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c01177&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c01177%26sid%3Dliteratum%253Aachs%26aulast%3DGampe%26aufirst%3DC.%26aulast%3DVerma%26aufirst%3DV.%2BA.%26atitle%3DCurse%2520or%2520cure%253F%2520A%2520perspective%2520on%2520the%2520developability%2520of%2520aldehydes%2520as%2520active%2520pharmaceutical%2520ingredients%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D14357%26epage%3D14381%26doi%3D10.1021%2Facs.jmedchem.0c01177" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cargnin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serafini, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pirali, T.</span></span> <span> </span><span class="NLM_article-title">A primer of deuterium in drug design</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">2039</span>– <span class="NLM_lpage">2042</span>, <span class="refDoi"> DOI: 10.4155/fmc-2019-0183</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=10.4155%2Ffmc-2019-0183" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=31538524" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvVKis7fN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2019&pages=2039-2042&author=S.+Cargninauthor=M.+Serafiniauthor=T.+Pirali&title=A+primer+of+deuterium+in+drug+design&doi=10.4155%2Ffmc-2019-0183"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">A primer of deuterium in drug design</span></div><div class="casAuthors">Cargnin, Sarah; Serafini, Marta; Pirali, Tracey</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">2039-2042</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  This article discusses about role of deuterated compounds, heavy and nonradioactive isotope of hydrogen in clinical development and effects of deuteration incorporation in drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnDF9mfxcU77Vg90H21EOLACvtfcHk0ljazo2UNwdr1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvVKis7fN&md5=2c7c1c4a9820456566886af749d28068</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.4155%2Ffmc-2019-0183&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc-2019-0183%26sid%3Dliteratum%253Aachs%26aulast%3DCargnin%26aufirst%3DS.%26aulast%3DSerafini%26aufirst%3DM.%26aulast%3DPirali%26aufirst%3DT.%26atitle%3DA%2520primer%2520of%2520deuterium%2520in%2520drug%2520design%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2019%26volume%3D11%26spage%3D2039%26epage%3D2042%26doi%3D10.4155%2Ffmc-2019-0183" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Meanwell, N. A.</span></span> <span> </span><span class="NLM_article-title">Synopsis of some recent tactical application of bioisosteres in drug design</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">2529</span>– <span class="NLM_lpage">2591</span>, <span class="refDoi"> DOI: 10.1021/jm1013693</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm1013693" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjsVCnu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=2529-2591&author=N.+A.+Meanwell&title=Synopsis+of+some+recent+tactical+application+of+bioisosteres+in+drug+design&doi=10.1021%2Fjm1013693"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Synopsis of Some Recent Tactical Application of Bioisosteres in Drug Design</span></div><div class="casAuthors">Meanwell, Nicholas A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2529-2591</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The established utility of bioisosteres is broad in nature, extending to improving potency, enhancing selectivity, altering phys. properties, reducing or redirecting metab., eliminating or modifying toxicophores, and acquiring novel intellectual property.  In this Perspective, some contemporary themes exploring the role of isosteres in drug design are sampled, with an emphasis placed on tactical applications designed to solve the kinds of problems that impinge on compd. optimization and the long-term success of drug candidates.  Interesting concepts that may have been poorly effective in the context examd. are captured, since the ideas may have merit in alternative circumstances.  A comprehensive cataloging of bioisosteres is beyond the scope of what will be provided, although a synopsis of relevant isosteres of a particular functionality is summarized in a succinct fashion in several sections.  Isosterism has also found productive application in the design and optimization of organocatalysts, and there are several examples in which functional mimicry established initially in a medicinal chem. setting has been adopted by this community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZ9z_aPcU4ubVg90H21EOLACvtfcHk0ljazo2UNwdr1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjsVCnu7o%253D&md5=0637ab60c379bef535f3f709b26f3582</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fjm1013693&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm1013693%26sid%3Dliteratum%253Aachs%26aulast%3DMeanwell%26aufirst%3DN.%2BA.%26atitle%3DSynopsis%2520of%2520some%2520recent%2520tactical%2520application%2520of%2520bioisosteres%2520in%2520drug%2520design%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D2529%26epage%3D2591%26doi%3D10.1021%2Fjm1013693" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chessari, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hardcastle, I. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahn, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anil, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anscombe, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bawn, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bevan, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blackburn, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buck, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cano, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carbain, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castro, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cons, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cully, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Endicott, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Endicott, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fazal, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golding, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffin, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haggerty, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harnor, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hearn, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hobson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holvey, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jennings, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lunec, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Newell, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noble, M. E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reeks, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Revill, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riedinger, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denis, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamanini, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, N. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vinković, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wedge, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilsher, N. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span> <span> </span><span class="NLM_article-title">Structure-based design of potent and orally active isoindolinone inhibitors of MDM2-p53 protein-protein interaction</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>64</i></span>,  <span class="NLM_fpage">4071</span>– <span class="NLM_lpage">4088</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c02188</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c02188" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A528%3ADC%252BB3MXntValtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2021&pages=4071-4088&author=G.+Chessariauthor=I.+R.+Hardcastleauthor=J.+S.+Ahnauthor=B.+Anilauthor=E.+Anscombeauthor=R.+H.+Bawnauthor=L.+D.+Bevanauthor=T.+J.+Blackburnauthor=I.+Buckauthor=C.+Canoauthor=B.+Carbainauthor=J.+Castroauthor=B.+Consauthor=S.+J.+Cullyauthor=J.+A.+Endicottauthor=J.+A.+Endicottauthor=L.+Fazalauthor=B.+T.+Goldingauthor=R.+J.+Griffinauthor=K.+Haggertyauthor=S.+J.+Harnorauthor=K.+Hearnauthor=S.+Hobsonauthor=R.+S.+Holveyauthor=S.+Howardauthor=C.+E.+Jenningsauthor=C.+N.+Johnsonauthor=J.+Lunecauthor=D.+C.+Millerauthor=D.+R.+Newellauthor=M.+E.+M.+Nobleauthor=J.+Reeksauthor=C.+H.+Revillauthor=C.+Riedingerauthor=J.+D.+Denisauthor=E.+Tamaniniauthor=H.+Thomasauthor=N.+T.+Thompsonauthor=M.+Vinkovi%C4%87author=S.+R.+Wedgeauthor=P.+A.+Williamsauthor=N.+E.+Wilsherauthor=B.+Zhangauthor=Y.+Zhao&title=Structure-based+design+of+potent+and+orally+active+isoindolinone+inhibitors+of+MDM2-p53+protein-protein+interaction&doi=10.1021%2Facs.jmedchem.0c02188"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design of Potent and Orally Active Isoindolinone Inhibitors of MDM2-p53 Protein-Protein Interaction</span></div><div class="casAuthors">Chessari, Gianni; Hardcastle, Ian R.; Ahn, Jong Sook; Anil, Burcu; Anscombe, Elizabeth; Bawn, Ruth H.; Bevan, Luke D.; Blackburn, Timothy J.; Buck, Ildiko; Cano, Celine; Carbain, Benoit; Castro, Juan; Cons, Ben; Cully, Sarah J.; Endicott, Jane A.; Fazal, Lynsey; Golding, Bernard T.; Griffin, Roger J.; Haggerty, Karen; Harnor, Suzannah J.; Hearn, Keisha; Hobson, Stephen; Holvey, Rhian S.; Howard, Steven; Jennings, Claire E.; Johnson, Christopher N.; Lunec, John; Miller, Duncan C.; Newell, David R.; Noble, Martin E. M.; Reeks, Judith; Revill, Charlotte H.; Riedinger, Christiane; St. Denis, Jeffrey D.; Tamanini, Emiliano; Thomas, Huw; Thompson, Neil T.; Vinkovic, Mladen; Wedge, Stephen R.; Williams, Pamela A.; Wilsher, Nicola E.; Zhang, Bian; Zhao, Yan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">4071-4088</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Inhibition of murine double minute 2 (MDM2)-p53 protein-protein interaction with small mols. has been shown to reactivate p53 and inhibit tumor growth.  Here, we describe rational, structure-guided, design of novel isoindolinone-based MDM2 inhibitors.  MDM2 X-ray crystallog., quantum mechanics ligand-based design, and metabolite identification all contributed toward the discovery of potent in vitro and in vivo inhibitors of the MDM2-p53 interaction with representative compds. inducing cytostasis in an SJSA-1 osteosarcoma xenograft model following once-daily oral administration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdhfpb86qe6bVg90H21EOLACvtfcHk0ljazo2UNwdr1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXntValtr4%253D&md5=5e7cc87b8b8aa3f4359b4c3b059c9910</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c02188&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c02188%26sid%3Dliteratum%253Aachs%26aulast%3DChessari%26aufirst%3DG.%26aulast%3DHardcastle%26aufirst%3DI.%2BR.%26aulast%3DAhn%26aufirst%3DJ.%2BS.%26aulast%3DAnil%26aufirst%3DB.%26aulast%3DAnscombe%26aufirst%3DE.%26aulast%3DBawn%26aufirst%3DR.%2BH.%26aulast%3DBevan%26aufirst%3DL.%2BD.%26aulast%3DBlackburn%26aufirst%3DT.%2BJ.%26aulast%3DBuck%26aufirst%3DI.%26aulast%3DCano%26aufirst%3DC.%26aulast%3DCarbain%26aufirst%3DB.%26aulast%3DCastro%26aufirst%3DJ.%26aulast%3DCons%26aufirst%3DB.%26aulast%3DCully%26aufirst%3DS.%2BJ.%26aulast%3DEndicott%26aufirst%3DJ.%2BA.%26aulast%3DEndicott%26aufirst%3DJ.%2BA.%26aulast%3DFazal%26aufirst%3DL.%26aulast%3DGolding%26aufirst%3DB.%2BT.%26aulast%3DGriffin%26aufirst%3DR.%2BJ.%26aulast%3DHaggerty%26aufirst%3DK.%26aulast%3DHarnor%26aufirst%3DS.%2BJ.%26aulast%3DHearn%26aufirst%3DK.%26aulast%3DHobson%26aufirst%3DS.%26aulast%3DHolvey%26aufirst%3DR.%2BS.%26aulast%3DHoward%26aufirst%3DS.%26aulast%3DJennings%26aufirst%3DC.%2BE.%26aulast%3DJohnson%26aufirst%3DC.%2BN.%26aulast%3DLunec%26aufirst%3DJ.%26aulast%3DMiller%26aufirst%3DD.%2BC.%26aulast%3DNewell%26aufirst%3DD.%2BR.%26aulast%3DNoble%26aufirst%3DM.%2BE.%2BM.%26aulast%3DReeks%26aufirst%3DJ.%26aulast%3DRevill%26aufirst%3DC.%2BH.%26aulast%3DRiedinger%26aufirst%3DC.%26aulast%3DDenis%26aufirst%3DJ.%2BD.%26aulast%3DTamanini%26aufirst%3DE.%26aulast%3DThomas%26aufirst%3DH.%26aulast%3DThompson%26aufirst%3DN.%2BT.%26aulast%3DVinkovi%25C4%2587%26aufirst%3DM.%26aulast%3DWedge%26aufirst%3DS.%2BR.%26aulast%3DWilliams%26aufirst%3DP.%2BA.%26aulast%3DWilsher%26aufirst%3DN.%2BE.%26aulast%3DZhang%26aufirst%3DB.%26aulast%3DZhao%26aufirst%3DY.%26atitle%3DStructure-based%2520design%2520of%2520potent%2520and%2520orally%2520active%2520isoindolinone%2520inhibitors%2520of%2520MDM2-p53%2520protein-protein%2520interaction%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2021%26volume%3D64%26spage%3D4071%26epage%3D4088%26doi%3D10.1021%2Facs.jmedchem.0c02188" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">Automatic indexing of rotation diffraction patterns</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1988</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">67</span>– <span class="NLM_lpage">72</span>, <span class="refDoi"> DOI: 10.1107/s0021889887009737</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=10.1107%2Fs0021889887009737" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A528%3ADyaL1cXhtVCgsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=1988&pages=67-72&author=W.+Kabsch&title=Automatic+indexing+of+rotation+diffraction+patterns&doi=10.1107%2Fs0021889887009737"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Automatic indexing of rotation diffraction patterns</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">67-71</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    </div><div class="casAbstract">A method is described which assigns indexes to a set of single-crystal reflections recorded by the rotation-oscillation technique using a fixed x-ray wavelength.  The space group and approx. unit-cell parameters are known.  The unknown crystal orientation is detd. directly from the obsd. diffraction pattern of one or several oscillation data records.  A local indexing procedure is described which allows for large initial errors in the parameters controlling the diffraction pattern.  These parameters are refined subsequently, thereby satisfying the constraints imposed by the space-group symmetry.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9WshttyE38LVg90H21EOLACvtfcHk0likhi8MIepqGw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1cXhtVCgsb8%253D&md5=268552b3f6069d2465b3916a985b840f</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1107%2Fs0021889887009737&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs0021889887009737%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DAutomatic%2520indexing%2520of%2520rotation%2520diffraction%2520patterns%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D1988%26volume%3D21%26spage%3D67%26epage%3D72%26doi%3D10.1107%2Fs0021889887009737" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">XDS</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1107/s0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=XDS&doi=10.1107%2Fs0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0likhi8MIepqGw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXDS%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2Fs0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vonrhein, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flensburg, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keller, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharff, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smart, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paciorek, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Womack, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bricogne, G.</span></span> <span> </span><span class="NLM_article-title">Data processing and analysis with theautoPROCtoolbox</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">293</span>– <span class="NLM_lpage">302</span>, <span class="refDoi"> DOI: 10.1107/s0907444911007773</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=10.1107%2Fs0907444911007773" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=21460447" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=293-302&author=C.+Vonrheinauthor=C.+Flensburgauthor=P.+Kellerauthor=A.+Sharffauthor=O.+Smartauthor=W.+Paciorekauthor=T.+Womackauthor=G.+Bricogne&title=Data+processing+and+analysis+with+theautoPROCtoolbox&doi=10.1107%2Fs0907444911007773"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Data processing and analysis with the autoPROC toolbox</span></div><div class="casAuthors">Vonrhein, Clemens; Flensburg, Claus; Keller, Peter; Sharff, Andrew; Smart, Oliver; Paciorek, Wlodek; Womack, Thomas; Bricogne, Gerard</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">293-302</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A typical diffraction expt. will generate many images and data sets from different crystals in a very short time.  This creates a challenge for the high-throughput operation of modern synchrotron beamlines as well as for the subsequent data processing.  Novice users in particular may feel overwhelmed by the tables, plots and nos. that the different data-processing programs and software packages present to them.  Here, some of the more common problems that a user has to deal with when processing a set of images that will finally make up a processed data set are shown, concg. on difficulties that may often show up during the first steps along the path of turning the expt. (i.e. data collection) into a model (i.e. interpreted electron d.).  Difficulties such as unexpected crystal forms, issues in crystal handling and suboptimal choices of data-collection strategies can often be dealt with, or at least diagnosed, by analyzing specific data characteristics during processing.  In the end, one wants to distinguish problems over which one has no immediate control once the expt. is finished from problems that can be remedied a posteriori.  A new software package, autoPROC, is also presented that combines third-party processing programs with new tools and an automated workflow script that is intended to provide users with both guidance and insight into the offline processing of data affected by the difficulties mentioned above, with particular emphasis on the automated treatment of multi-sweep data sets collected on multi-axis goniostats.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmah7iUrMLXLVg90H21EOLACvtfcHk0lhOV6u4ZRJ9lA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLs%253D&md5=a0a3579f6b371b6312819883864aa80d</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1107%2Fs0907444911007773&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs0907444911007773%26sid%3Dliteratum%253Aachs%26aulast%3DVonrhein%26aufirst%3DC.%26aulast%3DFlensburg%26aufirst%3DC.%26aulast%3DKeller%26aufirst%3DP.%26aulast%3DSharff%26aufirst%3DA.%26aulast%3DSmart%26aufirst%3DO.%26aulast%3DPaciorek%26aufirst%3DW.%26aulast%3DWomack%26aufirst%3DT.%26aulast%3DBricogne%26aufirst%3DG.%26atitle%3DData%2520processing%2520and%2520analysis%2520with%2520theautoPROCtoolbox%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D293%26epage%3D302%26doi%3D10.1107%2Fs0907444911007773" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span> <span> </span><span class="NLM_article-title">An introduction to data reduction: space-group determination, scaling and intensity statistics</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">282</span>– <span class="NLM_lpage">292</span>, <span class="refDoi"> DOI: 10.1107/s090744491003982x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=10.1107%2Fs090744491003982x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=21460446" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=282-292&author=P.+R.+Evans&title=An+introduction+to+data+reduction%3A+space-group+determination%2C+scaling+and+intensity+statistics&doi=10.1107%2Fs090744491003982x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">An introduction to data reduction: Space-group determination, scaling and intensity statistics</span></div><div class="casAuthors">Evans, Philip R.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">282-292</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  This paper presents an overview of how to run the CCP4 programs for data redn. (SCALA, POINTLESS and CTRUNCATE) through the CCP4 graphical interface ccp4i and points out some issues that need to be considered, together with a few examples.  It covers detn. of the point-group symmetry of the diffraction data (the Laue group), which is required for the subsequent scaling step, examn. of systematic absences, which in many cases will allow inference of the space group, putting multiple data sets on a common indexing system when there are alternatives, the scaling step itself, which produces a large set of data-quality indicators, estn. of |F| from intensity and finally examn. of intensity statistics to detect crystal pathologies such as twinning.  An appendix outlines the scoring schemes used by the program POINTLESS to assign probabilities to possible Laue and space groups.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEq8RhfQOW8LVg90H21EOLACvtfcHk0ljImshO0Ai_kg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLo%253D&md5=f425388a87744d721d324118d50e4f9a</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1107%2Fs090744491003982x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs090744491003982x%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DP.%2BR.%26atitle%3DAn%2520introduction%2520to%2520data%2520reduction%253A%2520space-group%2520determination%252C%2520scaling%2520and%2520intensity%2520statistics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D282%26epage%3D292%26doi%3D10.1107%2Fs090744491003982x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Storoni, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span> <span> </span><span class="NLM_article-title">Phasercrystallographic software</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">658</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1107/s0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=10.1107%2Fs0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phasercrystallographic+software&doi=10.1107%2Fs0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0ljImshO0Ai_kg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1107%2Fs0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhasercrystallographic%2520software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2Fs0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunkóczi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echols, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oeffner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, P. H.</span></span> <span> </span><span class="NLM_article-title">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">213</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1107/s0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunk%C3%B3cziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.-W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+a+comprehensive+Python-based+system+for+macromolecular+structure+solution&doi=10.1107%2Fs0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0ljImshO0Ai_kg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunk%25C3%25B3czi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.-W.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520a%2520comprehensive%2520Python-based%2520system%2520for%2520macromolecular%2520structure%2520solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2Fs0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and development ofCoot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/s0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=10.1107%2Fs0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+ofCoot&doi=10.1107%2Fs0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lgjmGQCbZcmIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1107%2Fs0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520ofCoot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2Fs0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arendall, W. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keedy, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Immormino, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span> <span> </span><span class="NLM_article-title">MolProbity: all-atom structure validation for macromolecular crystallography</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">12</span>– <span class="NLM_lpage">21</span>, <span class="refDoi"> DOI: 10.1107/s0907444909042073</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=10.1107%2Fs0907444909042073" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=20057044" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=12-21&author=V.+B.+Chenauthor=W.+B.+Arendallauthor=J.+J.+Headdauthor=D.+A.+Keedyauthor=R.+M.+Immorminoauthor=G.+J.+Kapralauthor=L.+W.+Murrayauthor=J.+S.+Richardsonauthor=D.+C.+Richardson&title=MolProbity%3A+all-atom+structure+validation+for+macromolecular+crystallography&doi=10.1107%2Fs0907444909042073"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">MolProbity: all-atom structure validation for macromolecular crystallography</span></div><div class="casAuthors">Chen, Vincent B.; Arendall, W. Bryan, III; Headd, Jeffrey J.; Keedy, Daniel A.; Immormino, Robert M.; Kapral, Gary J.; Murray, Laura W.; Richardson, Jane S.; Richardson, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">12-21</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">MolProbity is a structure-validation web service that provides broad-spectrum solidly based evaluation of model quality at both the global and local levels for both proteins and nucleic acids.  It relies heavily on the power and sensitivity provided by optimized hydrogen placement and all-atom contact anal., complemented by updated versions of covalent-geometry and torsion-angle criteria.  Some of the local corrections can be performed automatically in MolProbity and all of the diagnostics are presented in chart and graphical forms that help guide manual rebuilding.  X-ray crystallog. provides a wealth of biol. important mol. data in the form of at. three-dimensional structures of proteins, nucleic acids and increasingly large complexes in multiple forms and states.  Advances in automation, in everything from crystn. to data collection to phasing to model building to refinement, have made solving a structure using crystallog. easier than ever.  However, despite these improvements, local errors that can affect biol. interpretation are widespread at low resoln. and even high-resoln. structures nearly all contain at least a few local errors such as Ramachandran outliers, flipped branched protein side chains and incorrect sugar puckers.  It is crit. both for the crystallographer and for the end user that there are easy and reliable methods to diagnose and correct these sorts of errors in structures.  MolProbity is the authors' contribution to helping solve this problem and this article reviews its general capabilities, reports on recent enhancements and usage, and presents evidence that the resulting improvements are now beneficially affecting the global database.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIjgsh59dO-7Vg90H21EOLACvtfcHk0lgjmGQCbZcmIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D&md5=b5fc7574f43f01dd6e43c3663ca4f779</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1107%2Fs0907444909042073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs0907444909042073%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DArendall%26aufirst%3DW.%2BB.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DKeedy%26aufirst%3DD.%2BA.%26aulast%3DImmormino%26aufirst%3DR.%2BM.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DMurray%26aufirst%3DL.%2BW.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26atitle%3DMolProbity%253A%2520all-atom%2520structure%2520validation%2520for%2520macromolecular%2520crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D12%26epage%3D21%26doi%3D10.1107%2Fs0907444909042073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Potterton, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNicholas, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krissinel, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gruber, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perrakis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noble, M.</span></span> <span> </span><span class="NLM_article-title">Developments in theCCP4 molecular-graphics project</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2288</span>– <span class="NLM_lpage">2294</span>, <span class="refDoi"> DOI: 10.1107/s0907444904023716</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=10.1107%2Fs0907444904023716" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=15572783" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars7vN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2288-2294&author=L.+Pottertonauthor=S.+McNicholasauthor=E.+Krissinelauthor=J.+Gruberauthor=K.+Cowtanauthor=P.+Emsleyauthor=G.+N.+Murshudovauthor=S.+Cohenauthor=A.+Perrakisauthor=M.+Noble&title=Developments+in+theCCP4+molecular-graphics+project&doi=10.1107%2Fs0907444904023716"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Developments in the CCP4 molecular-graphics project</span></div><div class="casAuthors">Potterton, Liz; McNicholas, Stuart; Krissinel, Eugene; Gruber, Jan; Cowtan, Kevin; Emsley, Paul; Murshudov, Garib N.; Cohen, Serge; Perrakis, Anastassis; Noble, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2288-2294</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">Progress towards structure detn. that is both high-throughput and high-value is dependent on the development of integrated and automatic tools for electron-d. map interpretation and for the anal. of the resulting at. models.  Advances in map-interpretation algorithms are extending the resoln. regime in which fully automatic tools can work reliably, but at present human intervention is required to interpret poor regions of macromol. electron d., particularly where crystallog. data is only available to modest resoln. [for example, I/σ(I) < 2.0 for min. resoln. 2.5 Å].  In such cases, a set of manual and semi-manual model-building mol.-graphics tools is needed.  At the same time, converting the knowledge encapsulated in a mol. structure into understanding is dependent upon visualization tools, which must be able to communicate that understanding to others by both static and dynamic representations.  CCP4mg is a program designed to meet these needs in a way that is closely integrated with the ongoing development of CCP4 as a program suite suitable for both low- and high-intervention computational structural biol.  As well as providing a carefully designed user interface to advanced algorithms of model building and anal., CCP4mg is intended to present a graphical toolkit to developers of novel algorithms in these fields.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptPa3hk2BiOrVg90H21EOLACvtfcHk0lgjmGQCbZcmIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars7vN&md5=6bdc27bb8dc2d7c5ed37ef306e686001</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1107%2Fs0907444904023716&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs0907444904023716%26sid%3Dliteratum%253Aachs%26aulast%3DPotterton%26aufirst%3DL.%26aulast%3DMcNicholas%26aufirst%3DS.%26aulast%3DKrissinel%26aufirst%3DE.%26aulast%3DGruber%26aufirst%3DJ.%26aulast%3DCowtan%26aufirst%3DK.%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DCohen%26aufirst%3DS.%26aulast%3DPerrakis%26aufirst%3DA.%26aulast%3DNoble%26aufirst%3DM.%26atitle%3DDevelopments%2520in%2520theCCP4%2520molecular-graphics%2520project%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2288%26epage%3D2294%26doi%3D10.1107%2Fs0907444904023716" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P.</span></span> <span> </span><span class="NLM_article-title">Scaling and assessment of data quality</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">72</span>– <span class="NLM_lpage">82</span>, <span class="refDoi"> DOI: 10.1107/s0907444905036693</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=10.1107%2Fs0907444905036693" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=16369096" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlant7jM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2006&pages=72-82&author=P.+Evans&title=Scaling+and+assessment+of+data+quality&doi=10.1107%2Fs0907444905036693"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Scaling and assessment of data quality</span></div><div class="casAuthors">Evans, Philip</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">D62</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">72-82</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">The various phys. factors affecting measured diffraction intensities are discussed, as are the scaling models which may be used to put the data on a consistent scale.  After scaling, the intensities can be analyzed to set the real resoln. of the data set, to detect bad regions (e.g. bad images), to analyze radiation damage and to assess the overall quality of the data set.  The significance of any anomalous signal may be assessed by probability and correlation anal.  The algorithms used by the CCP4 scaling program SCALA are described.  A requirement for the scaling and merging of intensities is knowledge of the Laue group and point-group symmetries: the possible symmetry of the diffraction pattern may be detd. from scores such as correlation coeffs. between observations which might be symmetry-related.  These scoring functions are implemented in a new program POINTLESS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGravLSLU62RhLVg90H21EOLACvtfcHk0ljsipH5YH56ww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlant7jM&md5=293d3876e534c0c57813990515bb3c76</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1107%2Fs0907444905036693&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs0907444905036693%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DP.%26atitle%3DScaling%2520and%2520assessment%2520of%2520data%2520quality%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2006%26volume%3D62%26spage%3D72%26epage%3D82%26doi%3D10.1107%2Fs0907444905036693" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Diederichs, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karplus, P. A.</span></span> <span> </span><span class="NLM_article-title">Improved R-factors for diffraction data analysis in macromolecular crystallography</span>. <i>Nat. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">269</span>– <span class="NLM_lpage">275</span>, <span class="refDoi"> DOI: 10.1038/nsb0497-269</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=10.1038%2Fnsb0497-269" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=9095194" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A528%3ADyaK2sXitl2ntbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=1997&pages=269-275&author=K.+Diederichsauthor=P.+A.+Karplus&title=Improved+R-factors+for+diffraction+data+analysis+in+macromolecular+crystallography&doi=10.1038%2Fnsb0497-269"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Improved R-factors for diffraction data analysis in macromolecular crystallography</span></div><div class="casAuthors">Diederichs, Kay; Karplus, P. Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural Biology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">269-275</span>CODEN:
                <span class="NLM_cas:coden">NSBIEW</span>;
        ISSN:<span class="NLM_cas:issn">1072-8368</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Co.</span>)
        </div><div class="casAbstract">The quantity Rsym (also called Rmerge) is almost universally used for describing x-ray diffraction data quality.  Rsym is seriously flawed, because it has an implicit dependence on the redundancy of the data.  A cor. R-factor, Rmeas, is introduced as the equiv. robust indicator of data consistency.  Rmrgd, an R-factor that reflects the gain in accuracy upon averaging of equiv. reflections, is introduced as a useful indicator of the quality of reduced data.  These new data quality indicators better reveal the benefits of highly redundant data and should stimulate improvements in data quality through increased merging of data from multiple crystals.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpquf8R-kmFX7Vg90H21EOLACvtfcHk0ljsipH5YH56ww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXitl2ntbY%253D&md5=b7e94299b5b1267903d3eb08ad46068c</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1038%2Fnsb0497-269&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsb0497-269%26sid%3Dliteratum%253Aachs%26aulast%3DDiederichs%26aufirst%3DK.%26aulast%3DKarplus%26aufirst%3DP.%2BA.%26atitle%3DImproved%2520R-factors%2520for%2520diffraction%2520data%2520analysis%2520in%2520macromolecular%2520crystallography%26jtitle%3DNat.%2520Struct.%2520Biol.%26date%3D1997%26volume%3D4%26spage%3D269%26epage%3D275%26doi%3D10.1038%2Fnsb0497-269" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, M. S.</span></span> <span> </span><span class="NLM_article-title">Global indicators of X-ray data quality</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">130</span>– <span class="NLM_lpage">135</span>, <span class="refDoi"> DOI: 10.1107/s0021889800018227</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=10.1107%2Fs0021889800018227" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A528%3ADC%252BD3MXit1yhsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2001&pages=130-135&author=M.+S.+Weiss&title=Global+indicators+of+X-ray+data+quality&doi=10.1107%2Fs0021889800018227"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Global indicators of X-ray data quality</span></div><div class="casAuthors">Weiss, Manfred S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">130-135</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard International Publishers Ltd.</span>)
        </div><div class="casAbstract">Global indicators of the quality of diffraction data are presented and discussed, and are evaluated in terms of their performance with respect to various tasks.  Based on the results obtained, probably some of the conventional indicators still in use in the crystallog. community should be abandoned, such as the nominal resoln. dmin or the merging R factor Rmerge, and replaced by more objective and more meaningful nos., such as the effective optical resoln. deff,opt and the redundancy-independent merging R factor Rr.i.m.  Also, it is recommended that the precision-indicating merging R factor Rp.i.m. should be reported with every diffraction data set published, because it describes the precision of the averaged measurements, which are the quantities normally used in crystallog. as observables.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4sLhqfAbBZ7Vg90H21EOLACvtfcHk0ljsipH5YH56ww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXit1yhsLk%253D&md5=4b4d934beea8b3479f5fd54c802d34af</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1107%2Fs0021889800018227&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs0021889800018227%26sid%3Dliteratum%253Aachs%26aulast%3DWeiss%26aufirst%3DM.%2BS.%26atitle%3DGlobal%2520indicators%2520of%2520X-ray%2520data%2520quality%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2001%26volume%3D34%26spage%3D130%26epage%3D135%26doi%3D10.1107%2Fs0021889800018227" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Karplus, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diederichs, K.</span></span> <span> </span><span class="NLM_article-title">Linking crystallographic model and data quality</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>336</i></span>,  <span class="NLM_fpage">1030</span>– <span class="NLM_lpage">1033</span>, <span class="refDoi"> DOI: 10.1126/science.1218231</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=10.1126%2Fscience.1218231" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=22628654" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A528%3ADC%252BC38Xnt1Kgt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=336&publication_year=2012&pages=1030-1033&author=P.+A.+Karplusauthor=K.+Diederichs&title=Linking+crystallographic+model+and+data+quality&doi=10.1126%2Fscience.1218231"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Linking Crystallographic Model and Data Quality</span></div><div class="casAuthors">Karplus, P. Andrew; Diederichs, Kay</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">336</span>
        (<span class="NLM_cas:issue">6084</span>),
    <span class="NLM_cas:pages">1030-1033</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">In macromol. x-ray crystallog., refinement R values measure the agreement between obsd. and calcd. data.  Analogously, Rmerge values reporting on the agreement between multiple measurements of a given reflection are used to assess data quality.  Here, we show that despite their widespread use, Rmerge values are poorly suited for detg. the high-resoln. limit and that current std. protocols discard much useful data.  We introduce a statistic that ests. the correlation of an obsd. data set with the underlying (not measurable) true signal; this quantity, CC*, provides a single statistically valid guide for deciding which data are useful.  CC* also can be used to assess model and data quality on the same scale, and this reveals when data quality is limiting model improvement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0u3BGdcdUzbVg90H21EOLACvtfcHk0ljsipH5YH56ww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xnt1Kgt7Y%253D&md5=b8ee6edacea44364cc66c44dd3a505fd</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1126%2Fscience.1218231&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1218231%26sid%3Dliteratum%253Aachs%26aulast%3DKarplus%26aufirst%3DP.%2BA.%26aulast%3DDiederichs%26aufirst%3DK.%26atitle%3DLinking%2520crystallographic%2520model%2520and%2520data%2520quality%26jtitle%3DScience%26date%3D2012%26volume%3D336%26spage%3D1030%26epage%3D1033%26doi%3D10.1126%2Fscience.1218231" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P.</span></span> <span> </span><span class="NLM_article-title">Resolving Some Old Problems in Protein Crystallography</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>336</i></span>,  <span class="NLM_fpage">986</span>– <span class="NLM_lpage">987</span>, <span class="refDoi"> DOI: 10.1126/science.1222162</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=10.1126%2Fscience.1222162" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=22628641" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;key=1%3ACAS%3A528%3ADC%252BC38XptVClsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=336&publication_year=2012&pages=986-987&author=P.+Evans&title=Resolving+Some+Old+Problems+in+Protein+Crystallography&doi=10.1126%2Fscience.1222162"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Resolving some old problems in protein crystallography</span></div><div class="casAuthors">Evans, Phil</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">336</span>
        (<span class="NLM_cas:issue">6084</span>),
    <span class="NLM_cas:pages">986-987</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3-ErE43zPCLVg90H21EOLACvtfcHk0lggjKOiEmpc-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XptVClsbo%253D&md5=fcffa3a74929e5c11b8a2ef3c890cfd5</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1126%2Fscience.1222162&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1222162%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DP.%26atitle%3DResolving%2520Some%2520Old%2520Problems%2520in%2520Protein%2520Crystallography%26jtitle%3DScience%26date%3D2012%26volume%3D336%26spage%3D986%26epage%3D987%26doi%3D10.1126%2Fscience.1222162" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6XMK" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6XMK','PDB','6XMK'); return false;">PDB: 6XMK</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i75"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00319">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_71351"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.1c00319?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00319</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Binding mode of the 3b with SARS-CoV-2 3CLpro; Superposition of 3b with 2a, 3c with 3d; Surface representation showing the orientation 2f, 2k and 3e in subunit B near the S4 subsite of SARS-CoV-2 3CLpro; and Crystallographic data for SARS-CoV-2 3CLpro inhibitor complexes (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00319/suppl_file/jm1c00319_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings—SMILES codes (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00319/suppl_file/jm1c00319_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00319/suppl_file/jm1c00319_si_001.pdf">jm1c00319_si_001.pdf (389.59 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00319/suppl_file/jm1c00319_si_002.csv">jm1c00319_si_002.csv (3.54 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Coordinates and structure factors for the following SARS2 3CLpro complexes with inhibitors were deposited to the Worldwide Protein Databank (wwPDB) with the accession codes 7LKR (<b>2a</b>), 7LKS (<b>2f</b>), 7LKT (<b>2k</b>), 7LKU (<b>3b</b>), 7LKV (<b>3c</b>), 7LKW (<b>3d</b>), and 7LKX (<b>3e</b>). Authors will release the atomic coordinates upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.1c00319&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.1c00319%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2021.64.issue-14" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.1c00319" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                7MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679a3d211ac4195b","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
