
<html lang="en"     class="pb-page"  data-request-id="bed38f78-609a-4c31-8049-f0c20fa539b2"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;article:article:10.1021/acs.jmedchem.9b01421;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;issue:issue:10.1021/jmcmar.2019.62.issue-22"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="A New Approach of Mitigating CYP3A4 Induction Led to the Discovery of Potent Hepatitis B Virus (HBV) Capsid Inhibitor with Optimal ADMET Profiles" /></meta><meta name="dc.Creator" content="Xianfeng  Lin" /></meta><meta name="dc.Creator" content="Houguang  Shi" /></meta><meta name="dc.Creator" content="Weixing  Zhang" /></meta><meta name="dc.Creator" content="Zongxing  Qiu" /></meta><meta name="dc.Creator" content="Zheng  Zhou" /></meta><meta name="dc.Creator" content="Fabian  Dey" /></meta><meta name="dc.Creator" content="Sheng  Zhong" /></meta><meta name="dc.Creator" content="Hongxia  Qiu" /></meta><meta name="dc.Creator" content="Jianxun  Xie" /></meta><meta name="dc.Creator" content="Xue  Zhou" /></meta><meta name="dc.Creator" content="Guang  Yang" /></meta><meta name="dc.Creator" content="Guozhi  Tang" /></meta><meta name="dc.Creator" content="Hong C.  Shen" /></meta><meta name="dc.Creator" content="Wei  Zhu" /></meta><meta name="dc.Description" content="Described herein is a new approach to mitigate CYP3A4 induction. In this unconventional approach, a fine-tuning of the dihedral angle between the C4 phenyl and the dihydropyrimidine core of the het..." /></meta><meta name="Description" content="Described herein is a new approach to mitigate CYP3A4 induction. In this unconventional approach, a fine-tuning of the dihedral angle between the C4 phenyl and the dihydropyrimidine core of the het..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="November 5, 2019" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b01421" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2019 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b01421" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b01421" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b01421" /></link>
        
    
    

<title>A New Approach of Mitigating CYP3A4 Induction Led to the Discovery of Potent Hepatitis B Virus (HBV) Capsid Inhibitor with Optimal ADMET Profiles | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b01421" /></meta><meta property="og:title" content="A New Approach of Mitigating CYP3A4 Induction Led to the Discovery of Potent Hepatitis B Virus (HBV) Capsid Inhibitor with Optimal ADMET Profiles" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01421/20191122/images/large/jm9b01421_0007.jpeg" /></meta><meta property="og:description" content="Described herein is a new approach to mitigate CYP3A4 induction. In this unconventional approach, a fine-tuning of the dihedral angle between the C4 phenyl and the dihydropyrimidine core of the heteroaryldihydropyrimidine (HAP) class of capsid inhibitors successfully altered the structure–activity−relationships (SARs) of the unwanted CYP3A4 induction and the desired HBV capsid inhibition to more favorable values. This eventually led to the discovery of a new capsid inhibitor with significantly reduced CYP3A4 induction, excellent anti-HBV activity, favorable preclinical PK/PD profiles, and no early safety flags." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b01421"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01421">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b01421&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b01421&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b01421&amp;href=/doi/10.1021/acs.jmedchem.9b01421" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2019</span><span class="cit-fg-volume">, 62</span><span class="cit-fg-issue">, 22</span><span class="cit-fg-pageRange">, 10352-10361</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/62/22" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.9b01400" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.9b01429" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">A New Approach of Mitigating CYP3A4 Induction Led to the Discovery of Potent Hepatitis B Virus (HBV) Capsid Inhibitor with Optimal ADMET Profiles</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Xianfeng Lin</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xianfeng Lin</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Roche Innovation Center Shanghai,  Medicinal Chemistry, Roche Pharma Research and Early Development, Building 5, 720 Cailun Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xianfeng++Lin">Xianfeng Lin</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Houguang Shi</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Houguang Shi</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Roche Innovation Center Shanghai,  Medicinal Chemistry, Roche Pharma Research and Early Development, Building 5, 720 Cailun Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Houguang++Shi">Houguang Shi</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Weixing Zhang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Weixing Zhang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Roche Innovation Center Shanghai,  Medicinal Chemistry, Roche Pharma Research and Early Development, Building 5, 720 Cailun Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Weixing++Zhang">Weixing Zhang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Zongxing Qiu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Zongxing Qiu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Roche Innovation Center Shanghai,  Medicinal Chemistry, Roche Pharma Research and Early Development, Building 5, 720 Cailun Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Zongxing++Qiu">Zongxing Qiu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Zheng Zhou</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Zheng Zhou</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Roche Innovation Center Shanghai,  Lead Discovery, Roche Pharma Research and Early Development, Building 5, 720 Cailun Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Zheng++Zhou">Zheng Zhou</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Fabian Dey</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Fabian Dey</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Roche Innovation Center Basel,  Lead Discovery, Roche Pharma Research and Early Development, Building 5, 720 Cailun Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Fabian++Dey">Fabian Dey</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Sheng Zhong</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sheng Zhong</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Roche Innovation Center Shanghai,  Pharmaceutical Sciences, Roche Pharma Research and Early Development, Building 5, 720 Cailun Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Sheng++Zhong">Sheng Zhong</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Hongxia Qiu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hongxia Qiu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Roche Innovation Center Shanghai,  Pharmaceutical Sciences, Roche Pharma Research and Early Development, Building 5, 720 Cailun Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hongxia++Qiu">Hongxia Qiu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jianxun Xie</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jianxun Xie</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Roche Innovation Center Shanghai,  Pharmaceutical Sciences, Roche Pharma Research and Early Development, Building 5, 720 Cailun Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jianxun++Xie">Jianxun Xie</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Xue Zhou</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xue Zhou</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Roche Innovation Center Shanghai,  Discovery Virology, Roche Pharma Research and Early Development, Building 5, 720 Cailun Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xue++Zhou">Xue Zhou</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Guang Yang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Guang Yang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Roche Innovation Center Shanghai,  Discovery Virology, Roche Pharma Research and Early Development, Building 5, 720 Cailun Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Guang++Yang">Guang Yang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Guozhi Tang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Guozhi Tang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Roche Innovation Center Shanghai,  Medicinal Chemistry, Roche Pharma Research and Early Development, Building 5, 720 Cailun Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Guozhi++Tang">Guozhi Tang</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-7131-4678" title="Orcid link">http://orcid.org/0000-0002-7131-4678</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Hong C. Shen</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hong C. Shen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Roche Innovation Center Shanghai,  Medicinal Chemistry, Roche Pharma Research and Early Development, Building 5, 720 Cailun Road, Shanghai 201203, China</div></div><span class="conrtib-corresp"><strong>*</strong>H.C.S.: phone, +86 21 28946738; e-mail, <a href="/cdn-cgi/l/email-protection#88e0e7e6efa6fbe0ede6a6e0fbb9c8fae7ebe0eda6ebe7e5"><span class="__cf_email__" data-cfemail="40282f2e276e3328252e6e28337100322f2328256e232f2d">[email protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hong+C.++Shen">Hong C. Shen</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-3477-8932" title="Orcid link">http://orcid.org/0000-0002-3477-8932</a></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Wei Zhu</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Wei Zhu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Roche Innovation Center Shanghai,  Medicinal Chemistry, Roche Pharma Research and Early Development, Building 5, 720 Cailun Road, Shanghai 201203, China</div></div><span class="conrtib-corresp"><strong>*</strong>W.Z. (primary contact): phone, +86 21 28946743; fax, +86 21 50790293; e-mail, <a href="/cdn-cgi/l/email-protection#601705094e1a08154e171a5120120f0308054e030f0d"><span class="__cf_email__" data-cfemail="fb8c9e92d581938ed58c81cabb899498939ed5989496">[email protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Wei++Zhu">Wei Zhu</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-4701-2474" title="Orcid link">http://orcid.org/0000-0003-4701-2474</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b01421&amp;href=/doi/10.1021%2Facs.jmedchem.9b01421" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2019</span></span><span class="cit-volume">, 62</span><span class="cit-issue">, 22</span><span class="cit-pageRange">, 10352–10361</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">November 5, 2019</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>27 August 2019</li><li><span class="item_label"><b>Published</b> online</span>5 November 2019</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 27 November 2019</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b01421" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01421</a></div><div class="article_header-article-copyright"><strong>Copyright © 2019 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D10352%26pageCount%3D10%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DXianfeng%2BLin%252C%2BHouguang%2BShi%252C%2BWeixing%2BZhang%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D62%26issueNum%3D22%26contentID%3Dacs.jmedchem.9b01421%26title%3DA%2BNew%2BApproach%2Bof%2BMitigating%2BCYP3A4%2BInduction%2BLed%2Bto%2Bthe%2BDiscovery%2Bof%2BPotent%2BHepatitis%2BB%2BVirus%2B%2528HBV%2529%2BCapsid%2BInhibitor%2Bwith%2BOptimal%2BADMET%2BProfiles%26numPages%3D10%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D10361%26publicationDate%3DNovember%2B2019">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b01421"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1429</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">1</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b01421" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;A New Approach of Mitigating CYP3A4 Induction Led to the Discovery of Potent Hepatitis B Virus (HBV) Capsid Inhibitor with Optimal ADMET Profiles&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Xianfeng&quot;,&quot;last_name&quot;:&quot;Lin&quot;},{&quot;first_name&quot;:&quot;Houguang&quot;,&quot;last_name&quot;:&quot;Shi&quot;},{&quot;first_name&quot;:&quot;Weixing&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;Zongxing&quot;,&quot;last_name&quot;:&quot;Qiu&quot;},{&quot;first_name&quot;:&quot;Zheng&quot;,&quot;last_name&quot;:&quot;Zhou&quot;},{&quot;first_name&quot;:&quot;Fabian&quot;,&quot;last_name&quot;:&quot;Dey&quot;},{&quot;first_name&quot;:&quot;Sheng&quot;,&quot;last_name&quot;:&quot;Zhong&quot;},{&quot;first_name&quot;:&quot;Hongxia&quot;,&quot;last_name&quot;:&quot;Qiu&quot;},{&quot;first_name&quot;:&quot;Jianxun&quot;,&quot;last_name&quot;:&quot;Xie&quot;},{&quot;first_name&quot;:&quot;Xue&quot;,&quot;last_name&quot;:&quot;Zhou&quot;},{&quot;first_name&quot;:&quot;Guang&quot;,&quot;last_name&quot;:&quot;Yang&quot;},{&quot;first_name&quot;:&quot;Guozhi&quot;,&quot;last_name&quot;:&quot;Tang&quot;},{&quot;first_name&quot;:&quot;Hong&quot;,&quot;last_name&quot;:&quot;C. Shen&quot;},{&quot;first_name&quot;:&quot;Wei&quot;,&quot;last_name&quot;:&quot;Zhu&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2019&quot;,&quot;month&quot;:&quot;11&quot;,&quot;day&quot;:&quot;05&quot;,&quot;issue&quot;:&quot;22&quot;,&quot;volume&quot;:&quot;62&quot;,&quot;pages&quot;:&quot;10352-10361&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b01421&quot;},&quot;abstract&quot;:&quot;Described herein is a new approach to mitigate CYP3A4 induction. In this unconventional approach, a fine-tuning of the dihedral angle between the C4 phenyl and the dihydropyrimidine core of the heteroaryldihydropyrimidine (HAP) class of capsid inhibitors successfully altered the structure–activity−relationships (SARs) of the unwanted CYP3A4 induction and the desired HBV capsid inhibition to more favorable values. This eventually led to the discovery of a new capsid inhibitor with significantly reduced CYP3A4 induction, excellent anti-HBV activity, favorable preclinical PK/PD profiles, and no early safety flags.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01421&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01421" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01421&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01421" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01421&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01421" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b01421&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01421&amp;href=/doi/10.1021/acs.jmedchem.9b01421" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b01421" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b01421" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (2 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01421&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b01421%26sid%3Dliteratum%253Aachs%26pmid%3D31689116%26genre%3Darticle%26aulast%3DLin%26date%3D2019%26atitle%3DA%2BNew%2BApproach%2Bof%2BMitigating%2BCYP3A4%2BInduction%2BLed%2Bto%2Bthe%2BDiscovery%2Bof%2BPotent%2BHepatitis%2BB%2BVirus%2B%2528HBV%2529%2BCapsid%2BInhibitor%2Bwith%2BOptimal%2BADMET%2BProfiles%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D62%26issue%3D22%26spage%3D10352%26epage%3D10361%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291880" title="Crystal structure">Crystal structure</a>,</li><li><a href="/action/doSearch?ConceptID=291241" title="Phenyls">Phenyls</a>,</li><li><a href="/action/doSearch?ConceptID=292464" title="Rodent models">Rodent models</a>,</li><li><a href="/action/doSearch?ConceptID=292178" title="Organic compounds">Organic compounds</a>,</li><li><a href="/action/doSearch?ConceptID=291026" title="Conformation">Conformation</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/62/22" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/jmcmar.2019.62.issue-22/20191127/jmcmar.2019.62.issue-22.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01421/20191122/images/medium/jm9b01421_0007.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01421/20191122/images/large/jm9b01421_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01421&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Described herein is a new approach to mitigate CYP3A4 induction. In this unconventional approach, a fine-tuning of the dihedral angle between the C4 phenyl and the dihydropyrimidine core of the heteroaryldihydropyrimidine (HAP) class of capsid inhibitors successfully altered the structure–activity−relationships (SARs) of the unwanted CYP3A4 induction and the desired HBV capsid inhibition to more favorable values. This eventually led to the discovery of a new capsid inhibitor with significantly reduced CYP3A4 induction, excellent anti-HBV activity, favorable preclinical PK/PD profiles, and no early safety flags.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_51133" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_51133" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Hepatitis B virus (HBV), which chronically infects approximately 240 million people worldwide, is an important causative agent of liver fibrosis, cirrhosis, and cancer.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Despite widespread implementation of the vaccination program, HBV infection remains a global health burden. However, the current standard of care (SoC), namely, interferon and nucleos(t)ide analogues, cannot lead to a substantial HBV functional cure rate (typically much less than 10%), defined by the complete loss of HBV surface antigen (HBsAg) after a finite treatment period. It is also known that SoCs can be associated with either viral resistance or side effects.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> Consequently, there has been a great demand in the discovery and development of non-nucleos(t)ide, direct-acting anti-HBV drugs with novel mechanism of action in order to improve disease cure rate, as a standalone therapy or, more likely, in combination with other antiviral agents.<a onclick="showRef(event, 'ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref3 ref4">(3,4)</a></div><div class="NLM_p">The HBV core protein plays critical roles in the HBV lifecycle, with the most prominent one as the essential component of the nucleocapsid.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> The core protein is conserved across all HBV genotypes and is thus considered as an attractive target for small molecule intervention. We previously reported a new HAP class of core protein inhibitors represented by <b>1</b> (HAP_R01) and <b>2</b> (HAP_R10), as shown in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> These compounds prevented the HBV core proteins from assembling into a normal nucleocapsid. The misassembled nucleocapsid was presumably subsequently degraded through the host proteasome pathway, resulting in the interruption of HBV replication.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> The lead compound <b>2</b> demonstrated good <i>in vitro</i> and <i>in vivo</i> anti-HBV activity, favorable preclinical PK, and early safety profiles. However, further development of compound <b>2</b> was hampered due to its strong CYP3A4 enzyme induction observed in human hepatocytes, which was later confirmed in the phase I clinical trial with healthy volunteers (unpublished).</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Data of Anti-HBV EC<sub>50</sub> and CYP3A4 Induction of Compounds <b>1</b>–<b>20</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01421/20191122/images/medium/jm9b01421_0005.gif" alt="" id="GRAPHIC-d7e356-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01421/20191122/images/medium/jm9b01421_0006.gif" alt="" id="gr8" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">EC<sub>50</sub> is the mean value for reduction of HBV DNA by 50% in HepG.2.2.15 cells. Experiments were run in duplicate, with variation of <15%.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">CC<sub>50</sub> is the mean value for cytotoxicity to HepDE19 cells. Experiments were in duplicate runs with variation of <15%.</p></div><div class="footnote" id="t1fn3"><sup><sup>c</sup></sup><p class="last">mRNA level of CYP3A4 in percentage to that of rifampicin with compound treatment at 10 μM.</p></div><div class="footnote" id="t1fn4"><sup><sup>d</sup></sup><p class="last">Not determined.</p></div></div><div></div></div><div class="NLM_p">The induction of CYP3A4 enzyme represents a significant clinical issue because CYP3A4 is involved in the metabolism of more than 50% of marketed drugs.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> Specifically, CYP3A4 induction can lead to the reduction of therapeutic efficacy of a co-medication or the CYP3A4 inducer itself (a phenomenon known as autoinduction if the inducer itself is metabolized by CYP3A4). The induction of CYP3A4 enzyme can occur through several mechanisms, with the predominant one occurring through the activation of pregnane X receptor (PXR), a nuclear hormone receptor.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> In the human PXR transactivation assay, both compounds <b>1</b> and <b>2</b> were found to be strong PXR activators, indicating PXR activation as the mechanism of CYP3A4 induction for this chemical class of capsid inhibitors. Several crystal structures of the human PXR ligand binding domain (LBD) complexed with structurally diversified PXR activators were reported, which revealed an unusually large and flexible binding pocket.<a onclick="showRef(event, 'ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref8 ref9">(8,9)</a> A plausible pharmacophore model of PXR activators was proposed, which comprised one H-bond acceptor and at least two flanking hydrophobic groups, in particular aromatic groups.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> The structure of the HAP class of capsid inhibitors represented by compounds <b>1</b> and <b>2</b> coincidently contains all the pharmacophore elements favorable for hPXR binding, in which the C2 thiazoyl and the C4 phenyl play the part of hydrophobic groups and the nitrogen of the dihydropyrimidine or the carbonyl oxygen of the C5 methyl ester accounts for the key H-bonding interaction with PXR, thus potentially explaining the strong CYP3A4 induction effect observed for both compounds <b>1</b> and <b>2</b>. The well adopted strategies to reduce hPXR activation include (1) introducing a polar substituent to the lipophilic group to destabilize the hydrophobic interactions and (2) removing the central H-bond acceptor.<a onclick="showRef(event, 'ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref8 ref9">(8,9)</a> However, from the cocrystal structure of Cp149 Y132A mutant hexamer in complex with <b>1</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5WRE">5WRE</a>) (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>a), both the C2 thiazoyl and the C4 phenyl were located in hydrophobic pockets, where introduction of polar substituent would essentially lead to a loss of binding affinity to core proteins. Moreover, the removal of potential H-bond acceptors, e.g., replacing the dihydropyrimidine with dihydropyridine or C5 ester with trifluoromethyl, led to a complete loss of anti-HBV activity. In this manuscript, we report a new approach to mitigate hPXR activation, which culminated with the discovery of a new capsid inhibitor with significantly reduced CYP3A4 induction, excellent anti-HBV activity, favorable preclinical PK/PD profiles, and no early safety flags.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01421/20191122/images/medium/jm9b01421_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01421/20191122/images/large/jm9b01421_0001.jpeg" id="GRAPHIC-d7e461-autogenerated" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. (a) Crystal structure of Cp149 Y132A in complex with compound <b>1</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5WRE">5WRE</a>). Compound <b>1</b> is shown in stick with cyan carbons. The molecular surface of the protein is shown in gray in the vicinity of the ligand. The core protein monomer forming the top of the binding pocket is omitted for clarity. The dihedral angle constituted by the bonds labeled in red in the schematic representation was −64.3° in the cocrystal structure. (b) X-ray of <b>1a</b>. Compound <b>1a</b> is shown in stick with the green carbons. The dihedral angle constituted by the bonds labeled in red in the schematic representation was −64.5°. (c) Overlay of <b>1</b> (cyan) and <b>1a</b> (green).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01421/20191122/images/large/jm9b01421_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01421&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_23853" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_23853" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Working Hypothesis</h3><div class="NLM_p">In theory, when a ligand binds to its targeted protein, the closer the binding conformation is to its lowest energy conformation in water, the higher binding affinity it could exert. In the crystal structure of Cp149 Y132A in complex with compound <b>1</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5WRE">5WRE</a>), compound <b>1</b> binds to core proteins in a conformation where the C4 phenyl is nearly perpendicular to the dihydropyrimidine plane with a dihedral angle of −64.3° (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>a).<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> This is almost identical to the conformation observed in the single crystal structure of compound <b>1a</b> (the core structure of compound <b>1</b>), where a dihedral angle of −64.5° is observed (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>b and <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>c) except for an 180° flip of the C4 phenyl ring in the cocrystal structure (the −Cl substituent will clash with the residues forming the cap if it orients as that in the cocrystal structure of <b>1a</b>).<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> Interestingly, in the CSD five out of the seven total related small molecule crystal structures (codes 002548, 002919, 002920, FAGDAS, FAGDEW) show a similar conformation as observed in the protein/ligand crystal structure with dihedrals in the range between −62.3° and −88.5°.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> The remaining two (codes 200608 and HAYPOP) have the phenyl flipped with the chlorine above the dihydropyrimidine plane and the ester flipped by 180° as well.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> The concordance of the dihedral angles between C4 phenyl and dihydropyrimidine plane in water and in the binding pocket could nicely explain the high binding affinity of compound <b>1</b> to core proteins and thus its strong anti-HBV activity. Likewise, it was hypothesized that the strong CYP3A4 induction of compound <b>1</b> might also be ascribed to its favorable binding to hPXR LBD in this conformation.</div><div class="NLM_p last">As evident in the binding pocket (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>a), there is still a bit of room around the C4 phenyl of compound <b>1</b> to allow it to move slightly around the C–C bond connecting to the dihydropyrimidine core. As such, the SAR of capsid inhibition might tolerate small changes of the dihedral angle between the C4 phenyl and the dihydropyrimidine plane. The medicinal chemistry efforts were thus focused on the modulation of this dihedral angle in order to separate SARs of CYP3A4 induction and capsid inhibition.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> CYP3A4 Induction SAR</h3><div class="NLM_p">The 2′-substituent of C4 phenyl of compound <b>1</b> was assumed to be crucial in determining the dihedral angle between the C4 phenyl and the dihydropyrimidine plane. In the presence of an ester at the C5 position, the smaller the 2′-substituent is, the smaller the dihedral angle will likely be inferred from modeling results (<i>vide infra</i>). As such, we set out to explore smaller substituents than −Cl, e.g., −H and −F.</div><div class="NLM_p">The selected compounds were evaluated for their ability to induce CYP3A4 mRNA using the cryopreserved primary human hepatocytes. The induction was determined in cells exposed to the individual compounds at a concentration of 10 μM by measuring CYP3A4 mRNA levels, in comparison with the positive control rifampicin (RIF) at the same concentration. The choice of 10 μM is for detection of induction signals as much as possible. An increase in CYP3A4 mRNA contents that was >40% of the level stimulated by rifampicin was clearly considered as a risk for induction in clinic based on internal experience and FDA guidance.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a></div><div class="NLM_p">As seen in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, the 2′-F analogues <b>5</b> and <b>6</b> showed slightly reduced CYP3A4 induction compared to the 2′-Cl analogues <b>1</b> and <b>4</b>. The smaller 2′-H analogue <b>7</b> exhibited more pronounced effect on reducing CYP3A4 induction with a value of 85% (<b>7</b> vs <b>4</b>) while still maintaining potent anti-HBV activity with an EC<sub>50</sub> of 15 nM in HepG2.2.15 cells, which boosted our confidence in the exploration of this novel approach to reduce CYP3A4 induction.</div><div class="NLM_p">With the preliminary success, we next set out to explore the effect of the 3′-substituent on CYP3A4 induction. However, the effect on core protein binding was uncertain though there was a bit of room in the binding pocket. As such, we prepared the −CN, −F, −Cl, −Br substituted analogues at the 3′-position. From the SAR results, the substituents with a van der Waals radii larger than −F were not tolerated in the capsid binding pocket, which led to a significant loss of anti-HBV activity (>1 μM for all three analogues, examples <b>9a</b>–<b>c</b>). However, the 3′,4′-difluoro substituted analogue <b>8</b> maintained potent anti-HBV activity with an EC<sub>50</sub> of 11 nM. More importantly it demonstrated an increased selectivity of anti-HBV activity vs CYP3A4 induction with the latter decreased to 41% of rifampicin (an average of >3 tests). Subsequently, we synthesized the 3′,4′,5′-trifluoro substituted analogue <b>10</b>, which, however, displayed a decrease of anti-HBV activity and an elevation of CYP3A4 induction. The introduction of one additional −F substituent at the 2′-position could further improve anti-HBV activity with an EC<sub>50</sub> of 5 nM but at the cost of the increased CYP3A4 induction (example <b>11</b>).</div><div class="NLM_p">In order to further increase the selectivity of anti-HBV activity vs CYP3A4 induction, the substituent at the 4′-position was explored. From our previous SAR findings, we knew that the 4′-F of C4 phenyl could be successfully replaced with −Me, −Cl, and −Br with comparable or even better anti-HBV activity.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> Indeed, all three analogues (examples <b>12</b>–<b>14</b>) displayed improved anti-HBV activity in the single digit nM range. Regarding CYP3A4 induction potential, compound <b>12</b> showed a value of 73% relative to rifampicin. However, considering its cytotoxicity profile (CC<sub>50</sub> is 14 μM in HepDE19 cells), this result may not accurately reflect its induction potential in human hepatocytes as cytotoxicity can affect CYP3A4 induction readout also at the tested concentration of 10 μM. Compound <b>13</b> also showed an unfavorable CC<sub>50</sub> of 24 μM. The 4′-Me analogue <b>14</b> has potent anti-HBV EC<sub>50</sub> of 5 nM, good CC<sub>50</sub> of 85 μM, and similar CYP3A4 induction level (40% relative to RIF) to compound <b>8</b>. The 4′-H analogue <b>15</b>, however, demonstrated an increased CYP3A4 induction potential (94% relative to RIF) albeit a similar anti-HBV EC<sub>50</sub> to compound <b>8</b>.</div><div class="NLM_p last">Both analogues <b>8</b> and <b>14</b> demonstrated reduced CYP3A4 induction at a level around 40% relative to RIF, which met our preliminary goal. In order to minimize the concerns of DDI and autoinduction in clinical use, we were keen to explore whether the CYP3A4 induction could be further reduced for this HAP chemical series. By comparing the C5 methyl ester to C5 ethyl ester analogues (<b>1</b> vs <b>3</b>, and <b>2</b> vs <b>4</b>), it was clear that the methyl ester analogues had lower potential of CYP3A4 induction likely due to their reduced lipophilicity.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> As such, the corresponding C5 methyl ester analogues <b>16</b>–<b>20</b> were synthesized and the CYP3A4 induction data are summarized in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>. Indeed, they all demonstrated reduced CYP3A4 induction in human hepatocytes as compared to their corresponding ethyl ester analogues. As observed in previous SAR, they also showed decreased anti-HBV activity compared to the ethyl ester analogues, of which the 4′-Cl and 4′-Br analogues <b>19</b> and <b>20</b>, however, still displayed potent anti-HBV activity. Compound <b>19</b> was the most interesting one with an anti-HBV EC<sub>50</sub> of 10 nM, a high CC<sub>50</sub> of >100 μM, and a low CYP3A4 induction value of 25%.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> CYP3A4, CYP1A2, and CYP2B6 Induction Assessments of <b>19</b></h3><div class="NLM_p">With a low CYP3A4 induction potential at 10 μM, compound <b>19</b> was further tested in human hepatocytes for its induction potentials in a dose dependent manner for various CYP enzymes including CYP3A4, 1A2, and 2B6. As seen in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, compound <b>19</b> showed dose dependent decrease of CYP3A4 induction from 10 to 0.1 μM (25% at 10 μM, 7.5% at 1 μM, and 2% at 0.1 μM), which indicated indeed a low risk of DDI and autoinduction in clinical use. In contrast, compound <b>2</b> showed CYP3A4 inductions of 111%, 83%, and 22% of rifampicin at 10, 1, and 0.1 μM, respectively. Moreover, compound <b>19</b> had neither CYP1A2 nor CYP2B6 induction liabilities as determined by the changes of individual mRNAs compared with that of corresponding positive controls (<40% was accepted as a low risk of induction).</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Induction Data of <b>19</b> on CYP3A4, CYP1A2, and CYP2B6 mRNAs</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">enzyme subtype</th><th class="colsep0 rowsep0" align="center">concn (μM)</th><th class="colsep0 rowsep0" align="center">% of positive control</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CYP3A4<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">0.1</td><td class="colsep0 rowsep0" align="left">2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">7.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">10</td><td class="colsep0 rowsep0" align="left">25</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CYP1A2<a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="left">0.1</td><td class="colsep0 rowsep0" align="left">1.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">0.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">10</td><td class="colsep0 rowsep0" align="left">1.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CYP2B6<a class="ref internalNav" href="#t2fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left">0.1</td><td class="colsep0 rowsep0" align="left">1.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">1.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">10</td><td class="colsep0 rowsep0" align="left">1.4</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">Data were reported as CYP3A4 mRNA change with the treatment of <b>19</b> in percentage to that of rifampicin at 10 μM.</p></div><div class="footnote" id="t2fn2"><sup><sup>b</sup></sup><p class="last">Data were reported as CYP1A2 mRNA change with the treatment of <b>19</b> in percentage to that of omeprazole at 10 μM.</p></div><div class="footnote" id="t2fn3"><sup><sup>c</sup></sup><p class="last">Data were reported as CYP2B6 mRNA change with the treatment of <b>19</b> in percentage to that of phenobarbital at 10 μM.</p></div></div></div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Chemistry</h3><div class="NLM_p">The synthetic route of preparing this new series of 3′,4′-disubstituted C4 phenyl HAP analogues was shown in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. As exemplified by the synthesis of compound <b>19</b>, a three-component condensation reaction of methyl acetoacetate, 3-fluoro-4-chlorobenzaldehyde, and thiazole-2-carboxamidine afforded the key intermediate dihydropyrimidine <b>19a</b> as a racemic mixture. An SFC (supercritical fluid chromatography) chiral separation of <b>19a</b> afforded two enantiomerically pure intermediates, of which <b>(<i>S</i>)-19a</b> was determined as the desired enantiomer by X-ray diffraction study. <b>(<i>S</i>)-19a</b> was then reacted with NBS to give bromide <b>19b</b>, which was finally coupled with (3<i>S</i>)-morpholine-3-carboxylic acid to provide <b>19</b> as a single isomer after HPLC purification.</div><figure id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01421/20191122/images/medium/jm9b01421_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01421/20191122/images/large/jm9b01421_0004.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Preparation of Compound <b>19</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01421/20191122/images/large/jm9b01421_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01421&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <i>i</i>-PrOH, HOAc, piperidine, 48 °C, 2 h, followed by the addition of thiazole-2-carboxamidine, then 90 °C, 3 h, 52%; (b) SFC chiral separation; (c) NBS, CCl<sub>4</sub>, rt, 1 h, 44%; (d) DIPEA, DCM, ∼1 h, 57%.</p></p></figure></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Putative Explanation of the Reduced CYP3A4 Induction of <b>19</b></h3><div class="NLM_p">In order to prove that the 3′,4′-disubstituted C4 phenyl analogues such as compound <b>19</b> indeed has a smaller dihedral angle between C4 phenyl and dihydropyrimidine plane than that of 2′,4′-disubstituted C4 phenyl HAP analogues, we solved the single crystal structure of intermediate <b>(<i>S</i>)-19a</b>. Indeed, as shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>, compound <b>(<i>S</i>)-19a</b> has a much smaller dihedral angle of 1.3° than that of compound <b>1a</b> (64.5°) (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>a and <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>b). Interestingly, with this drastic conformation change, <b>(<i>S</i>)-19a</b> shows an opposite optical rotation to <b>1a</b>, where <b>(<i>S</i>)-19a</b> has [α]<sub>D</sub><sup>20</sup> of +50°, while <b>1a</b> has [α]<sub>D</sub><sup>20</sup> of −55°. However, in the absence of any structural information on the binding of these compounds to hPXR, it remains speculative to ascribe the decreased CYP3A4 induction to the decreased dihedral angle though this dihedral angle change had a consistent and pronounced effect on hPXR activation. Two plausible explanations were proposed. On one hand, the preferable conformation of <b>19</b> in water with a small dihedral angle between C4 phenyl and the dihydropyrimidine core may fit poorly or lead to enthalpy penalty if the C4 phenyl has to be oriented to the conformation of a larger dihedral angle in the binding pocket of hPXR LBD. On the other hand, the nearly planar orientation of the C4 phenyl to the dihydropyrimidine plane might prevent either the 3-N of dihydropyrimidine or the carbonyl of C5 ester from forming a favorable hydrogen bonding interaction with the corresponding residues of hPXR, e.g., Ser247 or His407 as described in the literature.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a></div><figure id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01421/20191122/images/medium/jm9b01421_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01421/20191122/images/large/jm9b01421_0002.jpeg" id="GRAPHIC-d7e981-autogenerated" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (a) X-ray of <b>(<i>S</i>)-19a</b>. Compound <b>(<i>S</i>)-19a</b> is shown as ball and stick with the color of carbon in purple. The dihedral angle consisting of the bonds labeled in red in the schematic representation was 1.3°. (b) Overlay of <b>1a</b> (green) and <b>(<i>S</i>)-19a</b> (purple).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01421/20191122/images/large/jm9b01421_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01421&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> ADMET and Preclinical PK/PD Studies of <b>19</b></h3><div class="NLM_p">With potent ant-HBV EC<sub>50</sub>, higher CC<sub>50</sub>, and low CYP enzymes induction potential, <b>19</b> was further profiled in ADMET and preclinical PK/PD studies in order to evaluate its potential for further development. The physicochemical and ADME data were summarized in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>. Compound <b>19</b> had a log <i>D</i> of 0.38 and two p<i>K</i><sub>a</sub> values of 1.8 and 5.9, respectively. The ionizable carboxylic acid group of compound <b>19</b> offered the advantage of a high solubility of >666 μg/mL. Moreover, compound <b>19</b> showed good permeability under the conditions of the PAMPA assay at pH 6.5. This compound displayed a low human hepatic clearance and a moderate mouse hepatic clearance. Incubation in human hepatocytes revealed that the unchanged parent was the major form with a minor portion of a phase II metabolite as a result of glucuronide conjugation of the carboxylic acid group (the acylglucuronide reactivity was assessed low). Compound <b>19</b> also demonstrated a sufficient free fraction of 3.1% and 4.0% in human and mouse plasmas, respectively.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Physicochemical, ADME, and PK Properties of <b>19</b></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" colspan="2" align="center">Physicochemical Properties</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">log <i>D</i><a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">0.38</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">p<i>K</i><sub>a</sub><a class="ref internalNav" href="#t3fn2" aria-label="b">b</a> (acid/base)</td><td class="colsep0 rowsep0" align="left">1.8/5.9</td></tr></tbody></table><table class="table "><colgroup><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" colspan="2" align="center">ADME Properties</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Lysa<a class="ref internalNav" href="#t3fn3" aria-label="c">c</a>(μg/mL)</td><td class="colsep0 rowsep0" align="left">>631</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PAMPA<a class="ref internalNav" href="#t3fn4" aria-label="d">d</a> (10<sup>–6</sup> cm/s)</td><td class="colsep0 rowsep0" align="left">2.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">human/mouse Hep_CL (mL min<sup>–1</sup> kg<sup>–1</sup>)<a class="ref internalNav" href="#t3fn5" aria-label="e">e</a></td><td class="colsep0 rowsep0" align="left">3.6/48.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PPB (human/mouse)<a class="ref internalNav" href="#t3fn6" aria-label="f">f</a></td><td class="colsep0 rowsep0" align="left">3.1/4.0</td></tr></tbody></table><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" colspan="3" align="center">Mouse SDPK profiles<a class="ref internalNav" href="#t3fn7" aria-label="g">g</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">plasma</td><td class="colsep0 rowsep0" align="left"><i>t</i><sub>1/2</sub> (iv, H)</td><td class="colsep0 rowsep0" align="left">2.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"><i>V</i><sub>ss</sub> (iv, L/kg)</td><td class="colsep0 rowsep0" align="left">2.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">CL (iv) (mL min<sup>–1</sup> kg<sup>–1</sup>)</td><td class="colsep0 rowsep0" align="left">40.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"><i>C</i><sub>max</sub> (po, μg/L)</td><td class="colsep0 rowsep0" align="left">254</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">AUC<sub>(0-<b>∞</b>)</sub> (iv, (μg/L)·h)</td><td class="colsep0 rowsep0" align="left">408</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">AUC<sub>(0–<b>∞</b>)</sub> (po, (μg/L)·h)</td><td class="colsep0 rowsep0" align="left">527</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"><i>F</i> (%)</td><td class="colsep0 rowsep0" align="left">43.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">liver</td><td class="colsep0 rowsep0" align="left">AUC<sub>(0–<b>∞</b>)</sub> (po, (μg/L)·h)</td><td class="colsep0 rowsep0" align="left">19985</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="2" align="left">AUC<sub>(0–∞),liver</sub>/AUC<sub>(0–∞),plasma</sub> (po)</td><td class="colsep0 rowsep0" align="left">38</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">Apparent partition coefficients at pH 7.4.</p></div><div class="footnote" id="t3fn2"><sup><sup>b</sup></sup><p class="last">Ionization constant.</p></div><div class="footnote" id="t3fn3"><sup><sup>c</sup></sup><p class="last">Lyophilization solubility assay (LYSA) (μg/mL).</p></div><div class="footnote" id="t3fn4"><sup><sup>d</sup></sup><p class="last">Parallel artificial membrane permeability assay (10<sup>–6</sup> cm/s).</p></div><div class="footnote" id="t3fn5"><sup><sup>e</sup></sup><p class="last">Predicted hepatic clearance obtained with human and mouse hepatocytes (mL min<sup>–1</sup> kg<sup>–1</sup>).</p></div><div class="footnote" id="t3fn6"><sup><sup>f</sup></sup><p class="last">Plasma protein binding assay (% unbound fraction, average of three tests).</p></div><div class="footnote" id="t3fn7"><sup><sup>g</sup></sup><p class="last">The single-dose pharmacokinetics (SDPK) study of <b>19</b> was carried out in mice as indicated according to the standard procedures (iv, 1 mg/kg; po, 3 mg/kg). Major parameters, including plasma clearance (Cl), volume of distribution at steady state (<i>V</i><sub>ss</sub>), <i>T</i><sub>1/2</sub> (iv), maximal concentration (<i>C</i><sub>max</sub>), area under the curve (AUC), and oral bioavailability (<i>F</i>) are reported.</p></div></div></div><div class="NLM_p">In mouse SDPK studies, compound <b>19</b> showed a moderate clearance similar to the <i>in vitro</i> prediction in mouse hepatocytes, a moderate half-life, and oral bioavailability. Despite a relatively low oral exposure in plasma, <b>19</b> demonstrated a high liver exposure (liver-to-plasma exposure ratio was 38), which was a desirable property for HBV infection treatment as liver is the disease organ.</div><div class="NLM_p">Supported by potent <i>in vitro</i> anti-HBV activity and favorable mouse PK profiles, compound <b>19</b> was assessed against HBV infection in the hydrodynamic injection (HDI) mouse model.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> In this study, female BALB/c mice were hydrodynamically injected with replication-competent HBV DNA plasmid through the tail vein. Twenty hours after plasmid injection, mice were orally dosed by blank vehicle, entecavir (ETV, 0.1 mg/kg, once daily as the positive control), or different doses (3 and 12 mg/kg per dose, twice daily) of compound <b>19</b> for 5 days. Plasma and liver samples were collected at the indicated time points for HBV DNA quantification by real-time quantitative polymerase chain reaction (qPCR). As shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>, with treatment of compound <b>19</b>, HBV DNA copy numbers were rapidly and markedly reduced in plasma and were hardly detectable in liver tissues, demonstrating a strong anti-HBV effect of compound <b>19</b> upon oral administration. The maximum efficacy of compound <b>19</b> was comparable to the maximum efficacy of ETV in this model.</div><figure id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01421/20191122/images/medium/jm9b01421_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01421/20191122/images/large/jm9b01421_0003.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Levels of HBV DNA in plasma (A, left) and liver (B, right) of HBV HDI mice upon treatment with the vehicle control, ETV control, and compound <b>19</b>. Error bars indicate standard error of the mean.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01421/20191122/images/large/jm9b01421_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01421&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Like its close analogue <b>2</b>, compound <b>19</b> was also clean in the following <i>in vitro</i> safety assessment including hERG, GSH (glutathione adduction), mutagenicity (Ames assay), and clastogenicity (micronucleus assay). The 5-day minitox study in rats showed sufficient safety margin as defined by ratio of the plasma exposure (<i>C</i><sub>max</sub> and AUC) in rat at NOAEL (no observed adverse effect level) versus the exposure at a projected human efficacious dose, which all supported further development of <b>19</b>.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i14">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_53700" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_53700" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">In summary, we have described herein a novel approach to reduce CYP3A4 induction, which relied on a finely designed conformational change between two close analogues in the same chemical series. This seemingly unexpected finding might remind medicinal chemists of the importance of looking at the ligand in a 3D (three-dimensional) way (they may look very similar in 2D but differ dramatically in 3D!). Plausible explanations of the conformation effect on CYP3A4 induction were provided also. However, given the promiscuous nature of the ligand-binding site of PXR which can accommodate structurally diverse small molecules in multiple ways, additional strategies have been identified to completely abolish hPXR binding while retaining anti-HBV activity in the HAP class of capsid inhibitors. Further research in this area will be reported in due course.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_55885" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_55885" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Synthetic Chemistry General Comments</h3><div class="NLM_p last">All of the intermediates were purified by silica gel chromatography using either a Biotage SP1 system or an ISCO CombiFlash chromatography instrument. All of the final compounds were purified by preparative HPLC (prep-HPLC) on a reversed-phase column using a Waters XBridge OBD Phenyl (30 mm × 100 mm, 5 μm) or OBD RP18 (30 mm × 100 mm, 5 μm) column under acidic conditions (A, 0.1% formic acid in H<sub>2</sub>O; B, 0.1% formic acid in acetonitrile) or basic conditions (A, 0.1% ammonia in H<sub>2</sub>O; B, acetonitrile). For SFC chiral separation, the intermediates were separated using a chiral column (Daicel Chiralpak IC, 30 mm × 250 mm, 5 μm) on a Mettler Teledo SFC-Multigram system (solvent system of 95% CO<sub>2</sub> and 5% IPA (0.5% TEA in IPA), backpressure of 100 bar, UV detection at 254 nm). Optical rotation was measured using a Rudolph Autopol V automatic polarimeter at a wavelength of 589 nm. LC–MS spectra were obtained using a MicroMass Platform LC (Waters Alliance 2795-ZQ2000). NMR spectra were obtained using Bruker AVANCE 400 MHz spectrometer, operating at 400.13 MHz (<sup>1</sup>H) and 100.62 MHz (<sup>13</sup>C). <sup>1</sup>H NMR spectra were obtained using the single pulse zg30. <sup>13</sup>C{<sup>1</sup>H} NMR spectra were obtained using composite pulse zgpg30 for proton decoupling. The chemical shifts were referenced against internal TMS (<sup>1</sup>H, <sup>13</sup>C) and the coupling constants were calculated as a whole of H,H and H,F. All of the starting materials were obtained commercially. All of the final compounds had purities greater than 95% based upon HPLC, LC–MS, and <sup>1</sup>H NMR analyses. All of the reported yields are for isolated products and are not optimized.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> General Synthetic Procedure, Exemplified by (3<i>S</i>)-4-[[(4<i>S</i>)-4-(4-Chloro-3-fluorophenyl)-5-methoxycarbonyl-2-thiazol-2-yl-1,4-dihydropyrimidin-6-yl]methyl]morpholine-3-carboxylic Acid (<b>19</b>)</h3><div class="NLM_p">To a stirred solution of 4-chloro-3-fluorobenzaldehyde (0.500 g, 0.32 mmol) and methyl acetoacetate (0.366 g, 3.2 mmol) in <i>i</i>-PrOH (6 mL) were added acetic acid (0.020 g, 0.034 mmol) and piperdine (0.027 g, 0.032 mmol) at 48 °C. The resulting mixture was stirred at 48 °C for 2 h. Then thiazole-2-carboxamidine hydrochloride (0.516 g, crude, ∼2.52 mmol) and triethylamine (0.318 g, 3.2 mmol) were added to the reaction mixture. The mixture was heated at 90 °C for 3 h. After it was cooled to room temperature, the reaction mixture was concentrated and the residue was dissolved in ethyl acetate and washed with brine. The organic layer was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>. The solvent was removed and the residue was purified by silica gel column chromatography (ethyl acetate/petroleum ether: 1/4 to 1/2) to afford methyl 4-(4-chloro-3-fluorophenyl)-6-methyl-2-thiazol-2-yl-1,4-dihydropyrimidine-5-carboxylate (compound <b>19a</b>) as a yellow solid (600 mg, 52%). MS: calcd (MH<sup>+</sup>) 366, measured (MH<sup>+</sup>) 366. <sup>1</sup>H NMR (MeOD-<i>d</i><sub>4</sub>, 400 MHz) δ ppm 7.96 (m, 1H), 7.77 (m, 1H), 7.42 (m, 1H), 7.18 (m, 2H), 5.69 (br s, 1H), 3.69 (s, 3H), 2.49 (s, 3H).</div><div class="NLM_p">The racemic compound <b>19a</b> was subjected to SFC chiral separation. The desired (−)-enantiomer methyl (4<i>S</i>)-4-(4-chloro-3-fluorophenyl)-6-methyl-2-thiazol-2-yl-1,4-dihydropyrimidine-5-carboxylate (<b>(<i>S</i>)-19a</b>) has a relatively short retention time. The absolute stereochemistry of <b>(<i>S</i>)-19a</b> was determined by X-ray diffraction study (see <a href="/doi/suppl/10.1021/acs.jmedchem.9b01421/suppl_file/jm9b01421_si_001.pdf" class="ext-link">Supporting Information</a>). <b>(<i>S</i>)-19a</b>: [α]<sub>D</sub><sup>20</sup> +50° (<i>c</i> 0.21 g/100 mL, MeOH).</div><div class="NLM_p">To a stirred solution of compound <b>(<i>S</i>)-19a</b> (0.160 g, 0.44 mmol) in CCl<sub>4</sub> (5 mL) was added NBS (0.070 g, 0.39 mmol) in portions. After the reaction mixture was stirred at 70 °C for 3 h, the solvent was removed <i>in vacuo</i> and the residue was purified by column chromatography to give methyl (4<i>S</i>)-6-(bromomethyl)-4-(4-chloro-3-fluorophenyl)-2-thiazol-2-yl-1,4-dihydropyrimidine-5-carboxylate (compound <b>19b</b>) as a yellow solid (0.086 g, 44%). MS: calcd 445 (MH<sup>+</sup>), measured 445 (MH<sup>+</sup>).</div><div class="NLM_p">To a stirred solution of compound <b>19b</b> (0.086 g, 0.19 mmol) and (3<i>S</i>)-morpholine-3-carboxylic acid hydrochloride salt (CAS no. 1187929-04-9, 0.042 g, 0.25 mmol) in dichloromethane (1 mL) was added dropwise DIPEA (0.13 mL, 0.77 mmol). The reaction mixture was stirred at room temperature until the disappearance of <b>19b</b>. The mixture was diluted with EtOAc (10 mL), washed successively with saturated aqueous NH<sub>4</sub>Cl solution and brine. The organic layer was separated and dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>. The solvent was concentrated <i>in vacuo</i> and the residue was purified by prep-HPLC to give (3<i>S</i>)-4-[[(4<i>S</i>)-4-(4-chloro-3-fluorophenyl)-5-methoxycarbonyl-2-thiazol-2-yl-1,4-dihydropyrimidin-6-yl]methyl]morpholine-3-carboxylic acid (compound <b>19</b>) as a light yellow solid (0.055 mg, 57%). Melting point: 136.5 °C. MS: calcd (MH<sup>+</sup>) 495, measured (MH<sup>+</sup>) 495. HRMS: calcd (MH<sup>+</sup>) 495.08997, measured (MH<sup>+</sup>) 495.08958. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz) δ ppm 12.9–13.0 (m, 1H), 12.9–13.0 (m, 1H), 12.4–13.0 (m, 1H), 9.8–10.0 (m, 1H), 8.0–8.1 (m, 1H), 7.9–8.0 (m, 1H), 7.5–7.6 (m, 1H), 7.3–7.3 (m, 1H), 7.2–7.2 (m, 1H), 5.6–5.7 (m, 1H), 4.25 (br d, 1H, <i>J</i> = 17.6 Hz), 3.9–4.0 (m, 2H), 3.8–3.8 (m, 1H), 3.6–3.7 (m, 2H), 3.6–3.6 (m, 4H), 3.0–3.1 (m, 1H), 2.35 (br d, 1H, <i>J</i> = 12.1 Hz). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz): δ ppm 172.8, 166.3, 162.4, 158.9, 156.4, 147.5, 146.7(d, <sup>2</sup><i>J</i><sub><i>CF</i></sub> = 20 Hz, FC), 145.4, 144.2, 131.2, 125.3, 124.7 (d, <sup>3</sup><i>J</i><sub><i>CF</i></sub> = 12 HZ, FC), 118.7, 118.6, 115.8, 115.6, 97.1, 69.4, 67.2, 61.9, 58.6, 54.8, 51.5, 48.5.</div><div class="NLM_p last"><b>The following compounds are prepared in analogy to 19 from commercially available building blocks</b>.</div></div><div id="sec4_3_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> (<i>R</i>)-6-((<i>S</i>)-2-Carboxy-4,4-difluoropyrrolidin-1-ylmethyl)-4-(2-chloro-4-fluorophenyl)-2-thiazol-2-yl-1,4-dihydropyrimidine-5-carboxylic Acid Ethyl Ester (<b>3</b>)</h3><div class="NLM_p last">MS: calcd 529 (MH<sup>+</sup>), exp 529 (MH<sup>+</sup>). <sup>1</sup>H NMR (MeOD-<i>d</i><sub>4</sub>, 400 MHz): δ ppm 8.09 (d, <i>J</i> = 3.01 Hz, 1H), 8.02 (d, <i>J</i> = 3.01 Hz, 1H), 7.57 (dd, <i>J</i> = 8.78, 6.02 Hz, 1H), 7.32 (dd, <i>J</i> = 8.78, 2.51 Hz, 1H), 7.16 (td, <i>J</i> = 8.41, 2.51 Hz, 1H), 6.25 (s, 1H), 4.53 (d, <i>J</i> = 16.06 Hz, 1H), 4.00–4.18 (m, 4H), 3.68 (q, <i>J</i> = 11.13 Hz, 1H), 3.20–3.30 (m,1H), 2.77–2.94 (m, 1H), 2.49–2.65 (m, 1H), 1.17 (t, <i>J</i> = 7.15 Hz, 3H).</div></div><div id="sec4_3_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> (<i>S</i>)-4-[(<i>R</i>)-6-(2-Chloro-4-fluorophenyl)-5-ethoxycarbonyl-2-thiazol-2-yl-3,6-dihydropyrimidin-4-ylmethyl]morpholine-3-carboxylic Acid (<b>4</b>)</h3><div class="NLM_p last">MS: calcd (MH<sup>+</sup>) 509, measured (MH<sup>+</sup>) 509. <sup>1</sup>H NMR (MeOD-<i>d</i><sub>4</sub>, 400 MHz): δ ppm 8.08 (d, <i>J</i> = 3.01 Hz, 1H), 8.01 (d, <i>J</i> = 3.01 Hz, 1H), 7.60 (dd, <i>J</i> = 8.53, 6.02 Hz, 1H), 7.31 (dd, <i>J</i> = 8.66, 2.38 Hz, 1H), 7.15 (td, <i>J</i> = 8.34, 2.38 Hz, 1H), 6.26 (s, 1H), 4.88 (d, <i>J</i> = 16.31 Hz, 1H), 4.59 (d, <i>J</i> = 16.56 Hz, 1H), 4.26–4.33 (m, 1H), 4.22 (d, <i>J</i> = 4.02 Hz, 2H), 4.11 (q, <i>J</i> = 6.94 Hz, 2H), 3.93–4.05 (m, 2H), 3.78 (ddd, <i>J</i> = 12.42, 8.28, 3.64 Hz, 1H), 3.27 (d, <i>J</i> = 13.05 Hz, 1H), 1.16 (t, <i>J</i> = 7.15 Hz, 3H).</div></div><div id="sec4_3_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> (2<i>S</i>)-1-[[(4<i>R</i>)-4-(2,4-Difluorophenyl)-5-methoxycarbonyl-2-thiazol-2-yl-1,4-dihydropyrimidin-6-yl]methyl]-4,4-difluoropyrrolidine-2-carboxylic Acid (<b>5</b>)</h3><div class="NLM_p last">MS: calcd 499 (MH<sup>+</sup>), exp 499 (MH<sup>+</sup>). <sup>1</sup>H NMR (MeOD-<i>d</i><sub>4</sub>, 400 MHz): δ ppm 8.14 (dd, <i>J</i> = 3.1, 12.9 Hz, 2H), 7.53 (dt, <i>J</i> = 6.5, 8.5 Hz, 1H), 7.0–7.1 (m, 2H), 6.04 (s, 1H), 4.54 (d, <i>J</i> = 16.1 Hz, 1H), 4.0–4.2 (m, 2H), 3.70 (s, 3H), 3.5–3.7 (m, 1H), 3.2–3.3 (m, 1H), 2.8–2.9 (m, 1H), 2.5–2.6 (m, 1H).</div></div><div id="sec4_3_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> (<i>S</i>)-4-[(<i>R</i>)-6-(2,4-Difluorophenyl)-5-ethoxycarbonyl-2-thiazol-2-yl-3,6-dihydropyrimidin-4-ylmethyl]morpholine-3-carboxylic Acid (<b>6</b>)</h3><div class="NLM_p last">MS: calcd 493 (MH<sup>+</sup>), exp 493 (MH<sup>+</sup>). <sup>1</sup>H NMR (MeOD-<i>d</i><sub>4</sub>, 400 MHz): δ ppm 7.98 (d, <i>J</i> = 2.8 Hz, 1H), 7.79 (d, <i>J</i> = 2.8 Hz, 1H), 7.50–7.36 (m, 1H), 7.04–6.87 (m, 2H), 6.02–5.93 (m, 1H), 4.38–4.19 (m, 2H), 4.17–4.00 (m, 4H), 3.97–3.78 (m, 3H), 3.58 (br s, 1H), 2.71 (br s, 1H), 1.19 (t, <i>J</i> = 7.0 Hz, 3H).</div></div><div id="sec4_3_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> (<i>S</i>)-4-[(<i>S</i>)-5-Ethoxycarbonyl-6-(4-fluorophenyl)-2-thiazol-2-yl-3,6-dihydropyrimidin-4-ylmethyl]morpholine-3-carboxylic Acid (<b>7</b>)</h3><div class="NLM_p last">MS: calcd 475 (MH<sup>+</sup>), exp 475 (MH<sup>+</sup>). <sup>1</sup>H NMR (MeOD-<i>d</i><sub>4</sub>, 400 MHz): δ ppm 8.00 (d, <i>J</i> = 3.0 Hz, 1H), 7.84 (d, <i>J</i> = 3.0 Hz, 1H), 7.41 (dd, <i>J</i> = 5.4, 8.7 Hz, 2H), 7.07 (t, <i>J</i> = 8.8 Hz, 2H), 5.71 (s, 1H), 4.58 (d, <i>J</i> = 16.1 Hz, 1H), 4.31 (br s, 1H), 4.21–4.03 (m, 4H), 3.98–3.81 (m, 2H), 3.71 (br s, 1H), 3.50 (dd, <i>J</i> = 1.6, 3.4 Hz, 1H), 2.85 (br s, 1H), 1.23 (t, <i>J</i> = 7.2 Hz, 3H).</div></div><div id="sec4_3_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> (<i>S</i>)-4-[(<i>S</i>)-6-(3,4-Difluorophenyl)-5-ethoxycarbonyl-2-thiazol-2-yl-3,6-dihydropyrimidin-4-ylmethyl]morpholine-3-carboxylic Acid (<b>8</b>)</h3><div class="NLM_p last">MS: calcd 493 (MH<sup>+</sup>), exp 493 (MH<sup>+</sup>). <sup>1</sup>H NMR (MeOD-<i>d</i><sub>4</sub>, 400 MHz): δ ppm 8.00 (d, <i>J</i> = 3.26 Hz, 1H), 7.83 (d, <i>J</i> = 3.01 Hz, 1H), 7.14–7.31 (m, 3H), 5.71 (s, 1H), 4.54 (d, <i>J</i> = 17.07 Hz, 1H), 4.02–4.30 (m, 5H), 3.78–3.98 (m, 2H), 3.67 (t, <i>J</i> = 3.89 Hz, 1H), 3.41 (br s, 1H), 2.70–2.86 (m, 1H), 1.24 (t, <i>J</i> = 7.15 Hz, 3H).</div></div><div id="sec4_3_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> (<i>S</i>)-4-[(<i>S</i>)-5-Ethoxycarbonyl-2-thiazol-2-yl-6-(3,4,5-trifluorophenyl)-3,6-dihydropyrimidin-4-ylmethyl]morpholine-3-carboxylic Acid (<b>10</b>)</h3><div class="NLM_p last">MS: calcd 511 (MH<sup>+</sup>), exp 511 (MH<sup>+</sup>). <sup>1</sup>H NMR (MeOD-<i>d</i><sub>4</sub>, 400 MHz): δ ppm 8.01 (d, <i>J</i> = 3.26 Hz, 1H), 7.83 (d, <i>J</i> = 3.26 Hz, 1H), 7.13 (dd, <i>J</i> = 8.66, 6.65 Hz, 2H), 5.72 (s, 1H), 4.55 (d, <i>J</i> = 17.32 Hz, 1H), 4.01–4.26 (m, 5H), 3.78–3.96 (m, 2H), 3.67 (t, <i>J</i> = 3.76 Hz, 1H), 3.35–3.47 (m, 1H), 2.61–2.79 (m, 1H), 1.25 (t, <i>J</i> = 7.03 Hz, 3H).</div></div><div id="sec4_3_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> (<i>S</i>)-4-[(<i>R</i>)-5-Ethoxycarbonyl-2-thiazol-2-yl-6-(2,3,4-trifluorophenyl)-3,6-dihydropyrimidin-4-ylmethyl]morpholine-3-carboxylic Acid (<b>11</b>)</h3><div class="NLM_p last">MS: calcd 511 (MH<sup>+</sup>), exp 511 (MH<sup>+</sup>). <sup>1</sup>H NMR (MeOD-<i>d</i><sub>4</sub>, 400 MHz): δ ppm 7.93–8.08 (m, 1H), 7.76–7.90 (m, 1H), 7.19–7.31 (m, 1H), 7.04–7.16 (m, 1H), 6.02 (br s, 1H), 4.35–4.54 (m, 1H), 4.10 (d, <i>J</i> = 4.77 Hz, 4H), 3.79–3.97 (m, 2H), 3.56–3.70 (m, 1H), 2.61–2.81 (m, 1H), 1.09–1.30 (m, 3H).</div></div><div id="sec4_3_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> (<i>S</i>)-4-[(<i>S</i>)-6-(4-Bromo-3-fluorophenyl)-5-ethoxycarbonyl-2-thiazol-2-yl-3,6-dihydropyrimidin-4-ylmethyl]morpholine-3-carboxylic Acid (<b>12</b>)</h3><div class="NLM_p last">MS: calcd 553 (MH<sup>+</sup>), exp 553 (MH<sup>+</sup>). <sup>1</sup>H NMR (MeOD-<i>d</i><sub>4</sub>, 400 MHz): δ ppm 8.01 (d, <i>J</i> = 3.3 Hz, 1H), 7.83 (d, <i>J</i> = 3.0 Hz, 1H), 7.59 (dd, <i>J</i> = 7.3, 8.3 Hz, 1H), 7.22 (dd, <i>J</i> = 2.0, 9.8 Hz, 1H), 7.15 (dd, <i>J</i> = 2.0, 8.3 Hz, 1H), 5.73 (s, 1H), 4.53 (d, <i>J</i> = 16.8 Hz, 1H), 4.27–4.01 (m, 5H), 3.98–3.80 (m, 2H), 3.67 (br s, 1H), 3.36 (d, <i>J</i> = 6.0 Hz, 1H), 2.73 (br s, 1H), 1.24 (t, <i>J</i> = 7.2 Hz, 3H).</div></div><div id="sec4_3_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> (<i>S</i>)-4-[(<i>S</i>)-6-(4-Chloro-3-fluorophenyl)-5-ethoxycarbonyl-2-thiazol-2-yl-3,6-dihydropyrimidin-4-ylmethyl]morpholine-3-carboxylic Acid (<b>13</b>)</h3><div class="NLM_p last">MS: calcd (MH<sup>+</sup>) 509, measured (MH<sup>+</sup>) 509. <sup>1</sup>H NMR (MeOD-<i>d</i><sub>4</sub>, 400 MHz): δ ppm 8.06–8.04 (m, 1H), 7.97–7.95 (m, 1H), 7.46–7.42 (m, 1H), 7.25–7.20 (m, 2H), 5.73 (s, 1H), 4.25–4.09 (m, 5H), 3.95–3.81 (m, 3H), 3.75–3.61 (m, 3H), 1.25 (t, <i>J</i> = 7.2 Hz, 3H).</div></div><div id="sec4_3_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> (3<i>S</i>)-4-[[(4<i>S</i>)-5-Ethoxycarbonyl-4-(3-fluoro-4-methylphenyl)-2-thiazol-2-yl-1,4-dihydropyrimidin-6-yl]methyl]morpholine-3-carboxylic Acid (<b>14</b>)</h3><div class="NLM_p last">MS: calcd (MH<sup>+</sup>) 489, measured (MH<sup>+</sup>) 489. <sup>1</sup>H NMR (MeOD-<i>d</i><sub>4</sub>, 400 MHz): δ ppm 8.00 (d, <i>J</i> = 3.0 Hz, 1H), 7.84 (d, <i>J</i> = 3.3 Hz, 1H), 7.16–7.26 (m, 1H), 6.95–7.13 (m, 2H), 5.69 (s, 1H), 4.46–4.71 (m, 1H), 4.21–4.38 (m, 1H), 4.03–4.22 (m, 4H), 3.78–3.97 (m, 2H), 3.60–3.75 (m, 1H), 3.38–3.54 (m, 1H), 2.69–2.94 (m, 1H), 2.24 (d, <i>J</i> = 1.5 Hz, 3H), 1.24 (t, <i>J</i> = 7.2 Hz, 3H).</div></div><div id="sec4_3_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> (<i>S</i>)-4-[(<i>S</i>)-5-Ethoxycarbonyl-6-(3-fluorophenyl)-2-thiazol-2-yl-3,6-dihydropyrimidin-4-ylmethyl]morpholine-3-carboxylic Acid (<b>15</b>)</h3><div class="NLM_p last">MS: calcd 475 (MH<sup>+</sup>), exp 475 (MH<sup>+</sup>). <sup>1</sup>H NMR (MeOD-<i>d</i><sub>4</sub>, 400 MHz): δ ppm 8.01 (d, <i>J</i> = 3.0 Hz, 1H), 7.83 (d, <i>J</i> = 3.3 Hz, 1H), 7.35 (dt, <i>J</i> = 6.0, 7.9 Hz, 1H), 7.22 (d, <i>J</i> = 7.5 Hz, 1H), 7.10 (td, <i>J</i> = 2.0, 9.9 Hz, 1H), 7.01 (dt, <i>J</i> = 2.0, 8.4 Hz, 1H), 5.74 (s, 1H), 4.53 (d, <i>J</i> = 14.1 Hz, 1H), 4.34–4.02 (m, 5H), 3.98–3.80 (m, 2H), 3.66 (br s, 1H), 3.47–3.36 (m, 1H), 2.77 (br s, 1H), 1.24 (t, <i>J</i> = 7.2 Hz, 3H).</div></div><div id="sec4_3_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> (3<i>S</i>)-4-[[(4<i>S</i>)-4-(3,4-Difluorophenyl)-5-methoxycarbonyl-2-thiazol-2-yl-1,4-dihydropyrimidin-6-yl]methyl]morpholine-3-carboxylic Acid (<b>16</b>)</h3><div class="NLM_p last">MS: calcd 499 (MH<sup>+</sup>), exp 499 (MH<sup>+</sup>). <sup>1</sup>H NMR (MeOD-<i>d</i><sub>4</sub>, 400 MHz): δ ppm 8.57 (s, 1H), 7.99 (d, <i>J</i> = 3.3 Hz, 1H), 7.75 (d, <i>J</i> = 3.1 Hz, 1H), 7.1–7.3 (m, 3H), 5.68 (s, 1H), 3.9–4.1 (m, 3H), 3.7–3.9 (m, 3H), 3.68 (s, 3H), 3.24 (dd, <i>J</i> = 3.5, 8.3 Hz, 1H), 2.97 (br d, <i>J</i> = 11.8 Hz, 1H), 2.48 (ddd, <i>J</i> = 3.3, 8.9, 11.8 Hz, 1H).</div></div><div id="sec4_3_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> (2<i>S</i>)-1-[[(4<i>S</i>)-4-(3,4-Difluorophenyl)-5-methoxycarbonyl-2-thiazol-2-yl-1,4-dihydropyrimidin-6-yl]methyl]-4,4-difluoropyrrolidine-2-carboxylic Acid (<b>17</b>)</h3><div class="NLM_p last">MS: calcd 479 (MH<sup>+</sup>), exp 479 (MH<sup>+</sup>). <sup>1</sup>H NMR (MeOD-<i>d</i><sub>4</sub>, 400 MHz): δ ppm 8.07 (d, <i>J</i> = 3.0 Hz, 1H), 7.98 (d, <i>J</i> = 3.0 Hz, 1H), 7.2–7.4 (m, 3H), 5.76 (s, 1H), 4.45 (d, <i>J</i> = 16.3 Hz, 1H), 4.22 (d, <i>J</i> = 16.3 Hz, 1H), 4.04 (t, <i>J</i> = 8.2 Hz, 1H), 3.71 (s, 3H), 3.6–3.7 (m, 1H), 3.2–3.3 (m, 1H), 2.84 (ddt, <i>J</i> = 8.9, 11.5, 14.9 Hz, 1H), 2.5–2.6 (m, 1H).</div></div><div id="sec4_3_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> (3<i>S</i>)-4-[[(4<i>S</i>)-4-(3-Fluoro-4-methyl-phenyl)-5-methoxycarbonyl-2-thiazol-2-yl-1,4-dihydropyrimidin-6-yl]methyl]morpholine-3-carboxylic Acid (<b>18</b>)</h3><div class="NLM_p last">MS: calcd (MH<sup>+</sup>) 475, measured (MH<sup>+</sup>) 475. <sup>1</sup>H NMR (MeOD-<i>d</i><sub>4</sub>, 400 MHz): δ ppm 8.00 (d, <i>J</i> = 3.0 Hz, 1H), 7.84 (d, <i>J</i> = 3.3 Hz, 1H), 7.15–7.26 (m, 1H), 6.97–7.12 (m, 2H), 5.69 (s, 1H), 4.47–4.67 (m, 1H), 4.23–4.34 (m, 1H), 4.01–4.19 (m, 2H), 3.81–3.98 (m, 2H), 3.64–3.73 (m, 1H), 3.40–3.54 (m, 1H), 2.71–2.94 (m, 1H), 2.24 (d, <i>J</i> = 1.5 Hz, 3H), 1.24 (t, <i>J</i> = 7.2 Hz, 3H).</div></div><div id="sec4_3_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> (3<i>S</i>)-4-[[(4<i>S</i>)-4-(4-Bromo-3-fluorophenyl)-5-methoxycarbonyl-2-thiazol-2-yl-1,4-dihydropyrimidin-6-yl]methyl]morpholine-3-carboxylic Acid (<b>20</b>)</h3><div class="NLM_p last">MS: calcd (MH<sup>+</sup>) 539, measured (MH<sup>+</sup>) 539. <sup>1</sup>H NMR (MeOD-<i>d</i><sub>4</sub>, 400 MHz): δ ppm 8.01 (d, <i>J</i> = 3.3 Hz, 1H), 7.84 (d, <i>J</i> = 3.0 Hz, 1H), 7.59 (dd, <i>J</i> = 8.3, 7.3 Hz, 1H), 7.09–7.28 (m, 2H), 5.73 (s, 1H), 4.44–4.70 (m, 1H), 4.02–4.30 (m, 3H), 3.77–3.96 (m, 2H), 3.71 (s, 3H), 3.36–3.47 (m, 2H), 2.64–2.91 (m, 1H).</div></div><div id="sec4_3_16_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> Cells and Culture Conditions</h3><div class="NLM_p last">HepG2.2.15 and HepDE19 are stably transfected cell lines containing the HBV genome. Both cell lines are derived from the hepatoblastoma cell line Hep G2 (American Type Culture Collection, ATCC HB-8065) by the published procedures.<a onclick="showRef(event, 'ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref14 ref15">(14,15)</a> Both cell lines were maintained in Dulbecco’s modified Eagle’s medium (DMEM)-F12 medium supplemented with 10% fetal bovine serum, 100 U/mL penicillin, 100 μg/mL streptomycin, and 0.5 mg/mL of G418. While HepG2.2.15 cells constitutively support HBV replication and production of virus particles, HepDE19 cells are inducible by tetracycline. Addition of 1 μg/mL tetracycline in culture medium suppresses HBV replication in HepDE19 cells, whereas switching to tetracycline-free medium resumes this process.</div></div><div id="sec4_3_16_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> Anti-HBV Activity <i>in Vitro</i></h3><div class="NLM_p last">HepG2.2.15 cells were seeded into 96-well plates (3 × 10<sup>4</sup> cells in 100 μL media per well) and incubated overnight at 37 °C. The test compounds were serially half-log diluted in DMSO, then diluted 100 times in culture media. An amount of 100 μL of diluted compounds was added into the plates to reach 0.5% final concentration of DMSO in every well. Five days after compound treatment, culture supernatant was collected for further analysis. For quantitative PCR detection of extracellular HBV DNA, 100 μL of culture supernatant was collected and processed in MagNA Pure 96 nucleic acid purification system (Roche Applied Science) for viral DNA extraction. The extracted samples were subjected to HBV DNA quantification by qPCR. The effective compound concentration at which HBV replication is inhibited by 50% (EC<sub>50</sub>) was determined.</div></div><div id="sec4_3_16_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> Cytotoxicity Assay</h3><div class="NLM_p last">HepDE19 cells were seeded into 96-well plates (5 × 10<sup>3</sup> cells per well) and treated with compounds for CC<sub>50</sub> determination. Five days after treatment, cell viability was measured by addition of 20 μL of CCK-8 reagent. Two hours after incubation at 37 °C, the absorbance at wavelengths of 450 and 630 nm (OD<sub>450</sub> and OD<sub>630</sub>) were recorded by a plate reader. The concentration results in the death of 50% of the host cells (CC<sub>50</sub>) of each compound were determined.</div></div><div id="sec4_3_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> LYSA Solubility Assay</h3><div class="NLM_p last">Samples are prepared in duplicate from 10 mM DMSO stock solution. After evaporation of DMSO with a centrifugal vacuum evaporator, compounds are dissolved in a 0.05 M phosphate buffer (pH 6.5), stirred for 1 h, and shaken for 2 h. After one night, the solutions are filtered using a microtiter filter plate. Then the filtrate and its 1/10 dilution are analyzed by HPLC-UV. In addition, a four-point calibration curve is prepared from the 10 mM stock solutions and used for the solubility determination of the compounds. The results are in μg/mL. In case the percentage of sample measured in solution after evaporation divided by the calculated maximum of sample amount is bigger than 80%, the solubility is reported as bigger than this value.</div></div><div id="sec4_3_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> Hepatocytes Stability Assay</h3><div class="NLM_p last">Cryopreserved hepatocytes suspension was diluted to 2 × 10<sup>6</sup> viable cells/mL for human and 1 × 10<sup>6</sup> viable cells/mL for mouse in the incubation medium and transferred 50 μL into each well of 96-well plates. These plates were incubated at 37 °C, 5% CO<sub>2</sub> with shaking at 900 rpm in the incubator. After 30 min preincubation, reactions were initiated by addition of 50 μL of prewarmed incubation medium containing 2 μM test compound to obtain a final compound concentration of 1 μM and a final cell density of 1 × 10<sup>6</sup> cells/mL for human and 0.5 × 10<sup>6</sup> cells/mL for mouse. After 2, 10, 20, 40, 60, and 120 min, 200 μL of ice cold acetonitrile containing an internal standard was added to terminate the reaction. Following precipitation and centrifugation, the amount of compound remaining in the samples were determined by LC–MS/MS.</div></div><div id="sec4_3_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> Parallel Artificial Membrane Permeability Assay</h3><div class="NLM_p last">The permeation of drugs is measured using a “sandwich” construction. Sample stock solution (DMSO) was diluted to 150 μM (DMSO% < 2%) with donor buffer, and after the filtration the solution was added into donor plate. An acceptor plate coated with phospholipids (membrane) was placed onto the donor plate containing drug solution. Finally, the upper plate was filled with buffer solution. The donor concentration was determined at <i>t</i>-start as reference concentration. The drug concentrations in acceptor and donor plate were detected at <i>t</i>-end (∼18 h). Donor buffer is 0.05 M MOPSO buffer at pH 6.5 with 0.5% (w/v) glycocholic acid, acceptor buffer is 0.05 M MOPSO buffer at pH 6.5, and the membrane is 10% (w/v) egg lecithin with 0.5% (w/v) cholesterol in dodecane.</div></div><div id="sec4_3_20" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> Plasma Protein Binding Assay</h3><div class="NLM_p last">The determination of unbound compound is performed using a 96-well format equilibrium dialysis (HTDialysis, Gales Ferry, CT, USA) with a molecular weight cutoff membrane of 12–14 kDa (HTDialysis, Gales Ferry, CT, USA), and diazepam was used as positive controls. Pooled human and mouse plasma were purchased from Biopredic (Rennes, France). The equilibrium dialysis device itself is made of Teflon to minimize nonspecific binding of the test substance. For the HTD device, the dialysis membranes are conditioned as recommended by the supplier and the dialysis plate is assembled by placing conditioned membranes between rows of half-wells and tightly clamping the assembled apparatus. Compounds are tested in cassettes of 2–5 with an initial total concentration of 1000 nM, one of the cassette compound being the positive control. By determination of the unbound fraction values for the positive control in each well, the integrity of membranes is tested. Equal volumes of matrix samples containing substances and blank dialysis buffer (Soerensen buffer at pH 7.4) are loaded into the donor and acceptor compartment, respectively. The HTD dialysis block is sealed and kept for 5 h at a temperature of 37 °C and 5% CO<sub>2</sub> environment in an incubator. At the end of dialysis, the plasma and buffer samples were retrieved and the drug concentrations quantified by LC–MS/MS.</div></div><div id="sec4_3_21" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> Pharmacokinetic (PK) Analysis</h3><div class="NLM_p last">Compound was evaluated in female BALB/c mice (16–22 g. Twelve mice in iv group were used for blood collection and 21 mice in po group were used for blood and liver collection) at iv 1 mg/kg and po 3 mg/kg. Compound was dissolved in 5% DMSO, 40% PEG400, 55% saline for iv dose and 2% Klucel, Tween 80, 0.1% Parabens in water for oral dose. Blood samples were collected at each time point and placed into tubes containing EDTA-K2 and centrifuged at 2000<i>g</i> for 5 min at 4 °C to separate plasma from the samples. Following centrifugation, the resulting plasma was transferred to clean tubes and stored frozen in dry ice and then transferred into −80 °C fridge pending bioanalysis. Immediately after the collection of blood, liver samples of each animal in the po group will be harvested, rinsed with cold saline, dried with filter paper, and placed into a tube and weighed. Liver samples will be snap frozen in dry ice and then stored at −80 °C until bioanalysis. Plasma concentrations and liver concentration were determined by LC–MS/MS, and the data were analyzed by a noncompartmental module of WinNonlin Professional version 6.2. Any BLQs were omitted from the calculation.</div></div><div id="sec4_3_22" class="NLM_sec NLM_sec_level_2"><div id="ac_i42" class="anchor-spacer"></div><h3 class="article-section__title" id="_i42"> <i>In Vivo</i> Efficacy in HDI Mouse Model</h3><div class="NLM_p last">The study was conducted by WuXi AppTec (Shanghai) Co., Ltd. All procedures in the study were in compliance with local animal welfare legislation, the Guide for the Care and Use of Laboratory Animals, and protocols approved by the WuXi AppTec (Shanghai) IACUC (Institutional Animal Care and Use Committee). Female BALB/c mice (6–8 weeks old) were injected through tail vein with 20 μg of plasmid pcDNA3.1HBV encoding a 1.1-mer HBV genome (Genotype D) in a volume of normal saline equivalent to 8% of the mouse body weight. The total volume was delivered within a few seconds. Seven hours after injection, mice were orally dosed by blank vehicle, 0.1 mg/kg of entecavir, or 3 mg/kg, and 12.5 mg/kg of <b>10</b> at indicated frequency for 5 days, respectively. Plasma and liver samples were collected at the indicated time points for HBV DNA quantification by real-time PCR.</div></div><div id="sec4_4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> CYP Induction Assay. High Throughput CYP Induction Screening Assay</h3><div class="NLM_p last">Cryopreserved hepatocytes were seeded on a 96-well collagen coated plate at a seeding density of 1 × 10<sup>6</sup> cells/mL. After 24 h culture in Williams E medium without serum, the test compounds at 0.1, 1.0, and 10 μM in serum-free culture media (0.1% DMSO content) were added in triplicates. A specific positive control inducer dependent on isoform (omeprazole for CYP1A2, phenobarbital for CYP2B6, or rifampicin for CYP3A4) was incubated alongside the test compounds. Negative control wells are also included on each plate consisting of vehicle alone (0.1% DMSO in Williams E media). Upon completion of the 24 h incubation, the hepatocytes are washed twice with Williams E media and then lysed by adding 100 μL of lysis binding solution. The lysates were stored at −80 °C until the qRT-PCR assay. Quantitative PCR analysis was performed on cDNA, using Applied Biosystems designed TaqMan gene expression assay for the target genes CYP1A2, CYP2B6, or CYP3A4. Peptidylprolyl isomerase A (PPIA) was used as a housekeeping gene. Samples were analyzed using an ABI 7900 HT real time PCR system. Relative fold mRNA expression levels of the target genes were determined based on the threshold cycle (CT) data of target genes relative to housekeeping gene for each reaction (DCT), normalized to the negative control (DDCT). Fold induction was calculated using the 2<sup>–ΔΔCT</sup> method. The cytotoxicity assays were not conducted; however, cell morphology during the test drug exposure period was monitored. Otherwise, a high CT number shift of PPIA from a baseline value of approximately 20 was considered as a consequence of cell toxicity indirectly indicated from this study.</div></div><div id="sec4_4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> Assay Validation</h3><div class="NLM_p last">The high throughput CYP induction screening assay was conducted for 24 h, which is shorter than the standard incubation time 48–72 h of a regulatory assay. Only one human liver donor was used to ensure detecting relatively strong positive induction flags. Positive induction signals of omeprazole (1, 10, and 100 μM) on CYP1A2 mRNA, phenobarbital (10, 100, and 1000 μM) on CYP2B6 mRNA, and rifampicin (0.1, 1, and 10 μM) on CYP3A4 mRNA could be obtained in a concentration-dependent manner. Internal historical data showed the mRNA fold induction of the positive control inducers at the highest concentrations was approximately >10, >2–6, and >10, respectively.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i45"><a href="/doi/suppl/10.1021/acs.jmedchem.9b01421" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_80531" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_80531" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.9b01421" class="ext-link">10.1021/acs.jmedchem.9b01421</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">X-ray structure of <b>(<i>S</i>)-19a</b>; <sup>1</sup>H NMR and <sup>13</sup>C NMR spectra of <b>19</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01421/suppl_file/jm9b01421_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and some data (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01421/suppl_file/jm9b01421_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01421/suppl_file/jm9b01421_si_001.pdf">jm9b01421_si_001.pdf (391.8 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01421/suppl_file/jm9b01421_si_002.csv">jm9b01421_si_002.csv (1.63 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.9b01421" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_68207" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_68207" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hong C. Shen</span> - <span class="hlFld-Affiliation affiliation">Roche
Innovation Center Shanghai, 
    Medicinal Chemistry, 
    , , , , , and , Roche Pharma Research and Early Development, Building 5, 720 Cailun Road, Shanghai 201203, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-3477-8932" title="Orcid link">http://orcid.org/0000-0002-3477-8932</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#066e69686128756e6368286e7537467469656e632865696b"><span class="__cf_email__" data-cfemail="6e06010009401d060b0040061d5f2e1c010d060b400d0103">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Wei Zhu</span> - <span class="hlFld-Affiliation affiliation">Roche
Innovation Center Shanghai, 
    Medicinal Chemistry, 
    , , , , , and , Roche Pharma Research and Early Development, Building 5, 720 Cailun Road, Shanghai 201203, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-4701-2474" title="Orcid link">http://orcid.org/0000-0003-4701-2474</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#a4d3c1cd8adeccd18ad3de95e4d6cbc7ccc18ac7cbc9"><span class="__cf_email__" data-cfemail="4730222e693d2f3269303d76073528242f226924282a">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xianfeng Lin</span> - <span class="hlFld-Affiliation affiliation">Roche
Innovation Center Shanghai, 
    Medicinal Chemistry, 
    , , , , , and , Roche Pharma Research and Early Development, Building 5, 720 Cailun Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Houguang Shi</span> - <span class="hlFld-Affiliation affiliation">Roche
Innovation Center Shanghai, 
    Medicinal Chemistry, 
    , , , , , and , Roche Pharma Research and Early Development, Building 5, 720 Cailun Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Weixing Zhang</span> - <span class="hlFld-Affiliation affiliation">Roche
Innovation Center Shanghai, 
    Medicinal Chemistry, 
    , , , , , and , Roche Pharma Research and Early Development, Building 5, 720 Cailun Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Zongxing Qiu</span> - <span class="hlFld-Affiliation affiliation">Roche
Innovation Center Shanghai, 
    Medicinal Chemistry, 
    , , , , , and , Roche Pharma Research and Early Development, Building 5, 720 Cailun Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Zheng Zhou</span> - <span class="hlFld-Affiliation affiliation">Roche
Innovation Center Shanghai, 
    Lead Discovery, 
    , , , , , and , Roche Pharma Research and Early Development, Building 5, 720 Cailun Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Fabian Dey</span> - <span class="hlFld-Affiliation affiliation">Roche Innovation Center Basel, 
    Lead Discovery, 
    , , , , , and , Roche Pharma Research and Early Development, Building 5, 720 Cailun Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sheng Zhong</span> - <span class="hlFld-Affiliation affiliation">Roche
Innovation Center Shanghai, 
    Pharmaceutical Sciences, 
    , , , , , and , Roche Pharma Research and Early Development, Building 5, 720 Cailun Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hongxia Qiu</span> - <span class="hlFld-Affiliation affiliation">Roche
Innovation Center Shanghai, 
    Pharmaceutical Sciences, 
    , , , , , and , Roche Pharma Research and Early Development, Building 5, 720 Cailun Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jianxun Xie</span> - <span class="hlFld-Affiliation affiliation">Roche
Innovation Center Shanghai, 
    Pharmaceutical Sciences, 
    , , , , , and , Roche Pharma Research and Early Development, Building 5, 720 Cailun Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xue Zhou</span> - <span class="hlFld-Affiliation affiliation">Roche
Innovation Center Shanghai, 
    Discovery Virology, 
    , , , , , and , Roche Pharma Research and Early Development, Building 5, 720 Cailun Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Guang Yang</span> - <span class="hlFld-Affiliation affiliation">Roche
Innovation Center Shanghai, 
    Discovery Virology, 
    , , , , , and , Roche Pharma Research and Early Development, Building 5, 720 Cailun Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Guozhi Tang</span> - <span class="hlFld-Affiliation affiliation">Roche
Innovation Center Shanghai, 
    Medicinal Chemistry, 
    , , , , , and , Roche Pharma Research and Early Development, Building 5, 720 Cailun Road, Shanghai 201203, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-7131-4678" title="Orcid link">http://orcid.org/0000-0002-7131-4678</a></span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>X.L. and H.S. contributed equally to this work.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br><p class="inlineNote">All animal experiments performed in the manuscript were conducted in compliance with institutional guidelines.</p></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i48">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_50010" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_50010" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We are grateful to Qinshan Gao, Yang Lu, Wenzhi Cheng, Liqin Chen, Peilan Ding, and Leimin Wang for the purification of final compounds and the analytical assistance. We thank the process group, including Jin She, Guocai Zhang, and Junli Chen, for their assistance with large-scale synthesis campaigns. We thank Jieyu Dai and Kenichi Umehara for preparing the CYP induction assay protocol. We also thank André Alker for the analysis of the single crystal X-ray structure of <b>(<i>S</i>)-19a</b>. This work was funded by F. Hoffmann-La Roche AG.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">HBV</td><td class="NLM_def"><p class="first last">hepatitis B virus</p></td></tr><tr><td class="NLM_term">ADMET</td><td class="NLM_def"><p class="first last">absorption, distribution, metabolism, excretion, and toxicity</p></td></tr><tr><td class="NLM_term">HAP</td><td class="NLM_def"><p class="first last">heteroaryldihydropyrimidine</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity-relationship</p></td></tr><tr><td class="NLM_term">PK</td><td class="NLM_def"><p class="first last">pharmacokinetic</p></td></tr><tr><td class="NLM_term">PD</td><td class="NLM_def"><p class="first last">pharmacodynamics</p></td></tr><tr><td class="NLM_term">CYP</td><td class="NLM_def"><p class="first last">cytochrome P450</p></td></tr><tr><td class="NLM_term">HBsAg</td><td class="NLM_def"><p class="first last">HBV surface antigen</p></td></tr><tr><td class="NLM_term">PXR</td><td class="NLM_def"><p class="first last">pregnane X receptor</p></td></tr><tr><td class="NLM_term">RIF</td><td class="NLM_def"><p class="first last">rifampicin</p></td></tr><tr><td class="NLM_term">CSD</td><td class="NLM_def"><p class="first last">Cambridge Structural Database</p></td></tr><tr><td class="NLM_term">SFC</td><td class="NLM_def"><p class="first last">supercritical fluid chromatography</p></td></tr><tr><td class="NLM_term">HPLC</td><td class="NLM_def"><p class="first last">high performance liquid chromatography</p></td></tr><tr><td class="NLM_term">PAMPA</td><td class="NLM_def"><p class="first last">parallel artificial membrane permeability assay</p></td></tr><tr><td class="NLM_term">LYSA</td><td class="NLM_def"><p class="first last">lyophilization solubility assay</p></td></tr><tr><td class="NLM_term">PPB</td><td class="NLM_def"><p class="first last">plasma protein binding</p></td></tr><tr><td class="NLM_term">SDPK</td><td class="NLM_def"><p class="first last">single-dose pharmacokinetics</p></td></tr><tr><td class="NLM_term">HDI</td><td class="NLM_def"><p class="first last">hydrodynamic injection</p></td></tr><tr><td class="NLM_term">ETV</td><td class="NLM_def"><p class="first last">entecavir</p></td></tr><tr><td class="NLM_term">hERG</td><td class="NLM_def"><p class="first last">human ether-a-go-go-related gene</p></td></tr><tr><td class="NLM_term">GSH</td><td class="NLM_def"><p class="first last">glutathione adduction</p></td></tr><tr><td class="NLM_term">NOAEL</td><td class="NLM_def"><p class="first last">no observed adverse effect level</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i50">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_65016" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_65016" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 15 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span>World Health
Organization (WHO). Fact Sheet No. 204. <a href="http://www.who.int/mediacentre/factsheets/fs204/en/" class="extLink">http://www.who.int/mediacentre/factsheets/fs204/en/</a> (accessed Apr 16, <span class="NLM_year">2019</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=World+Health%0AOrganization+%28WHO%29.+Fact+Sheet+No.+204.+http%3A%2F%2Fwww.who.int%2Fmediacentre%2Ffactsheets%2Ffs204%2Fen%2F+%28accessed+Apr+16%2C+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Feng, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, J. A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yun, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, H. C.</span></span> <span> </span><span class="NLM_article-title">Discovery of small molecule therapeutics for treatment of chronic HBV infection</span>. <i>ACS Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">257</span>– <span class="NLM_lpage">277</span>, and references cited therein <span class="refDoi"> DOI: 10.1021/acsinfecdis.7b00144</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsinfecdis.7b00144" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01421&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs1yrtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2018&pages=257-277&author=S.+Fengauthor=L.+Gaoauthor=X.+Hanauthor=T.+Huauthor=Y.+Huauthor=H.+Liuauthor=A.+W.+Thomasauthor=Z.+Yanauthor=S.+Yangauthor=J.+A.+T.+Youngauthor=H.+Yunauthor=W.+Zhuauthor=H.+C.+Shen&title=Discovery+of+small+molecule+therapeutics+for+treatment+of+chronic+HBV+infection&doi=10.1021%2Facsinfecdis.7b00144"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Small Molecule Therapeutics for Treatment of Chronic HBV Infection</span></div><div class="casAuthors">Feng, Song; Gao, Lu; Han, Xingchun; Hu, Taishan; Hu, Yimin; Liu, Haixia; Thomas, Andrew W.; Yan, Zhipeng; Yang, Song; Young, John A. T.; Yun, Hongying; Zhu, Wei; Shen, Hong C.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Infectious Diseases</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">257-277</span>CODEN:
                <span class="NLM_cas:coden">AIDCBC</span>;
        ISSN:<span class="NLM_cas:issn">2373-8227</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The chronic infection of hepatitis B virus (HBV) inflicts 250 million people worldwide representing a major public health threat.  A significant sub-population of chronically infected HBV patients eventually develop cirrhosis and hepatocellular carcinoma (HCC) which aggravate the issues caused by hepatitis alone.  Unfortunately, none of the current std. therapies for chronic Hepatitis B (CHB) result in a satisfactory clin. cure rate.  Driven by a highly unmet medical need for novel and more effective treatments of HBV infection, multiple pharmaceutical companies and research institutions have been engaged in drug discovery and development to improve the CHB functional cure rate, which is defined by sustainable viral suppression and HBsAg clearance after a finite treatment.  This review summarizes the recent significant advances in the discovery and development of novel small mol. anti-HBV compds., highlighted by representative structures, mechanism of action (MoA), and biol. activities.  Lastly, it is believed that improved CHB functional cure rate may be accomplished via the combination of mols. with distinct MoAs.  Thus some of the mols. may evolve into key components of a suitable combination therapy leading to superior outcome of clin. efficacy in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1zsMqu16n17Vg90H21EOLACvtfcHk0lg41ywkrT25kg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs1yrtbY%253D&md5=45e6f4740904bd92d82193455a2d97ac</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1021%2Facsinfecdis.7b00144&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsinfecdis.7b00144%26sid%3Dliteratum%253Aachs%26aulast%3DFeng%26aufirst%3DS.%26aulast%3DGao%26aufirst%3DL.%26aulast%3DHan%26aufirst%3DX.%26aulast%3DHu%26aufirst%3DT.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DThomas%26aufirst%3DA.%2BW.%26aulast%3DYan%26aufirst%3DZ.%26aulast%3DYang%26aufirst%3DS.%26aulast%3DYoung%26aufirst%3DJ.%2BA.%2BT.%26aulast%3DYun%26aufirst%3DH.%26aulast%3DZhu%26aufirst%3DW.%26aulast%3DShen%26aufirst%3DH.%2BC.%26atitle%3DDiscovery%2520of%2520small%2520molecule%2520therapeutics%2520for%2520treatment%2520of%2520chronic%2520HBV%2520infection%26jtitle%3DACS%2520Infect.%2520Dis.%26date%3D2018%26volume%3D4%26spage%3D257%26epage%3D277%26doi%3D10.1021%2Facsinfecdis.7b00144" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Testoni, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durantel, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoulim, F.</span></span> <span> </span><span class="NLM_article-title">Novel targets for hepatitis B virus therapy</span>. <i>Liver Int.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>37</i></span> (<span class="NLM_issue">S1</span>),  <span class="NLM_fpage">33</span>– <span class="NLM_lpage">39</span>, <span class="refDoi"> DOI: 10.1111/liv.13307</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01421&amp;key=10.1111%2Fliv.13307" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01421&amp;key=28052622" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01421&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjvFWntA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2017&pages=33-39&issue=S1&author=B.+Testoniauthor=D.+Durantelauthor=F.+Zoulim&title=Novel+targets+for+hepatitis+B+virus+therapy&doi=10.1111%2Fliv.13307"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Novel targets for hepatitis B virus therapy</span></div><div class="casAuthors">Testoni, Barbara; Durantel, David; Zoulim, Fabien</div><div class="citationInfo"><span class="NLM_cas:title">Liver International</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">S1</span>),
    <span class="NLM_cas:pages">33-39</span>CODEN:
                <span class="NLM_cas:coden">LIINCM</span>;
        ISSN:<span class="NLM_cas:issn">1478-3223</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Treatment with either pegylated interferon-alpha (pegIFN-α) or last generation nucleos(t)ide analogs (NAs) successfully leads to serum viral load suppression in most chronically infected hepatitis B (CHB) patients, but HBsAg loss is only achieved in 10% of the cases after a 5-yr follow-up.  Thus, therapy must be administered long-term and it will not completely eliminate infection because of the persistent hepatitis B virus (HBV) minichromosome in infected cells, and cannot completely abolish the risk of developing severe sequelae such as cirrhosis and hepatocellular carcinoma.  Recent progress in the development of in vitro and in vivo models of HBV infection have helped renew interest in the investigation of the viral life cycle, as well as specific virus-host cell interactions to identify new targets for the development of new antiviral drugs.  This includes either direct inhibition of viral replication by targeting fundamental steps such as entry, cccDNA formation/stability, viral transcripts, capsid assembly and secretion or the manipulation of the host immune system for better defense against infection.  Multiple strategies are currently under investigation, including boosting endogenous innate responses and/or restoring adaptive immunity via engineering of HBV-specific T cells or via the use of inhibitors of neg. regulators, as well as therapeutic vaccines.  It is increasingly clear that multiple therapeutic strategies must be combined to reach a cure of HBV and that the definition of clin., virol. and immunol. correlates for the management of treatment are urgently needed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCpuA-v_62UbVg90H21EOLACvtfcHk0lg41ywkrT25kg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjvFWntA%253D%253D&md5=d6631b3098076a3795070480c7016998</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1111%2Fliv.13307&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fliv.13307%26sid%3Dliteratum%253Aachs%26aulast%3DTestoni%26aufirst%3DB.%26aulast%3DDurantel%26aufirst%3DD.%26aulast%3DZoulim%26aufirst%3DF.%26atitle%3DNovel%2520targets%2520for%2520hepatitis%2520B%2520virus%2520therapy%26jtitle%3DLiver%2520Int.%26date%3D2017%26volume%3D37%26issue%3DS1%26spage%3D33%26epage%3D39%26doi%3D10.1111%2Fliv.13307" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, J.-T.</span></span> <span> </span><span class="NLM_article-title">The current status and future directions of hepatitis B antiviral drug discovery</span>. <i>Expert Opin. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">5</span>– <span class="NLM_lpage">15</span>, <span class="refDoi"> DOI: 10.1080/17460441.2017.1255195</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01421&amp;key=10.1080%2F17460441.2017.1255195" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01421&amp;key=27797587" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01421&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFSls7%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=5-15&author=L.+Tangauthor=Q.+Zhaoauthor=S.+Wuauthor=J.+Chengauthor=J.+Changauthor=J.-T.+Guo&title=The+current+status+and+future+directions+of+hepatitis+B+antiviral+drug+discovery&doi=10.1080%2F17460441.2017.1255195"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">The current status and future directions of hepatitis B antiviral drug discovery</span></div><div class="casAuthors">Tang, Liudi; Zhao, Qiong; Wu, Shuo; Cheng, Junjun; Chang, Jinhong; Guo, Ju-Tao</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">5-15</span>CODEN:
                <span class="NLM_cas:coden">EODDBX</span>;
        ISSN:<span class="NLM_cas:issn">1746-0441</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">The current std. care of chronic hepatitis B fails to induce a durable off-drug control of hepatitis B virus (HBV) replication in the majority of treated patients.  The primary reasons are its inability to eliminate the covalently closed circular (ccc) DNA, the nuclear form of HBV genome, and restoration of the dysfunctional host antiviral immune response against the virus.  Accordingly, discovery and development of therapeutics to completely stop HBV replication, eliminate or functionally inactivate cccDNA as well as activate a functional antiviral immune response against HBV are the primary efforts for finding a cure for chronic hepatitis B.  Herein, the authors highlight the current efforts of HBV drug discovery and offer their opinions for the future directions of this research.  The authors believe that through a consecutive or overlapping three-stage antiviral and immunotherapy program to: (i) completely stop HBV replication and cccDNA amplification; (ii) reduce viral antigen load and induce HBV surface antigen (HBsAg) seroclearance through eradication or inactivation of cccDNA and RNA interference-mediated degrdn. of viral mRNA and (iii) activate a functional antiviral immune response against HBV through therapeutic immunization or immunotherapy, a functional cure of chronic HBV infection can be achieved in the majority of chronic HBV carriers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3PtMZrdkoM7Vg90H21EOLACvtfcHk0lg41ywkrT25kg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFSls7%252FF&md5=e1425fe2bd4b239065211a78d19b2b1f</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1080%2F17460441.2017.1255195&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F17460441.2017.1255195%26sid%3Dliteratum%253Aachs%26aulast%3DTang%26aufirst%3DL.%26aulast%3DZhao%26aufirst%3DQ.%26aulast%3DWu%26aufirst%3DS.%26aulast%3DCheng%26aufirst%3DJ.%26aulast%3DChang%26aufirst%3DJ.%26aulast%3DGuo%26aufirst%3DJ.-T.%26atitle%3DThe%2520current%2520status%2520and%2520future%2520directions%2520of%2520hepatitis%2520B%2520antiviral%2520drug%2520discovery%26jtitle%3DExpert%2520Opin.%2520Drug%2520Discovery%26date%3D2017%26volume%3D12%26spage%3D5%26epage%3D15%26doi%3D10.1080%2F17460441.2017.1255195" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deres, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schröder, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paessens, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldmann, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hacker, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weber, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krämer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niewöhner, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pleiss, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stoltefuss, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graef, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koletzki, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masantschek, R. N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reimann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaeger, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groβ, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beckermann, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlemmer, K.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haebich, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rübsamen-Waigmann, H.</span></span> <span> </span><span class="NLM_article-title">Inhibition of hepatitis B virus replication by drug-induced depletion of nucleocapsids</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>299</i></span>,  <span class="NLM_fpage">893</span>– <span class="NLM_lpage">896</span>, <span class="refDoi"> DOI: 10.1126/science.1077215</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01421&amp;key=10.1126%2Fscience.1077215" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01421&amp;key=12574631" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01421&amp;key=1%3ACAS%3A528%3ADC%252BD3sXpt1Sgsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=299&publication_year=2003&pages=893-896&author=K.+Deresauthor=C.+H.+Schr%C3%B6derauthor=A.+Paessensauthor=S.+Goldmannauthor=H.+J.+Hackerauthor=O.+Weberauthor=T.+Kr%C3%A4merauthor=U.+Niew%C3%B6hnerauthor=U.+Pleissauthor=J.+Stoltefussauthor=E.+Graefauthor=D.+Koletzkiauthor=R.+N.+A.+Masantschekauthor=A.+Reimannauthor=R.+Jaegerauthor=R.+Gro%CE%B2author=B.+Beckermannauthor=K.-H.+Schlemmerauthor=D.+Haebichauthor=H.+R%C3%BCbsamen-Waigmann&title=Inhibition+of+hepatitis+B+virus+replication+by+drug-induced+depletion+of+nucleocapsids&doi=10.1126%2Fscience.1077215"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of Hepatitis B Virus Replication by Drug-Induced Depletion of Nucleocapsids</span></div><div class="casAuthors">Deres, Karl; Schroeder, Claus H.; Paessens, Arnold; Goldmann, Siegfried; Hacker, Hans Joerg; Weber, Olaf; Kraemer, Thomas; Niewoehner, Ulrich; Pleiss, Ulrich; Stoltefuss, Juergen; Graef, Erwin; Koletzki, Diana; Masantschek, Ralf N. A.; Reimann, Anja; Jaeger, Rainer; Gro, Rainer; Beckermann, Bernhard; Schlemmer, Karl-Heinz; Haebich, Dieter; Ruebsamen-Waigmann, Helga</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">299</span>
        (<span class="NLM_cas:issue">5608</span>),
    <span class="NLM_cas:pages">893-896</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Chronic hepatitis B virus (HBV) infection is a major cause of liver disease.  Only interferon-α and the nucleosidic inhibitors of the viral polymerase, 3TC and adefovir, are approved for therapy.  However, these therapies are limited by the side effects of interferon and the substantial resistance of the virus to nucleosidic inhibitors.  Potent new antiviral compds. suitable for monotherapy or combination therapy are highly desired.  We describe non-nucleosidic inhibitors of HBV nucleocapsid maturation that possess in vitro and in vivo antiviral activity.  These inhibitors have potential for future therapeutic regimens to combat chronic HBV infection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAlJUFASSEhLVg90H21EOLACvtfcHk0ljIXQTY94rMug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXpt1Sgsg%253D%253D&md5=a5e74b45d335f74ba17ec9fe4d31e24b</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1126%2Fscience.1077215&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1077215%26sid%3Dliteratum%253Aachs%26aulast%3DDeres%26aufirst%3DK.%26aulast%3DSchr%25C3%25B6der%26aufirst%3DC.%2BH.%26aulast%3DPaessens%26aufirst%3DA.%26aulast%3DGoldmann%26aufirst%3DS.%26aulast%3DHacker%26aufirst%3DH.%2BJ.%26aulast%3DWeber%26aufirst%3DO.%26aulast%3DKr%25C3%25A4mer%26aufirst%3DT.%26aulast%3DNiew%25C3%25B6hner%26aufirst%3DU.%26aulast%3DPleiss%26aufirst%3DU.%26aulast%3DStoltefuss%26aufirst%3DJ.%26aulast%3DGraef%26aufirst%3DE.%26aulast%3DKoletzki%26aufirst%3DD.%26aulast%3DMasantschek%26aufirst%3DR.%2BN.%2BA.%26aulast%3DReimann%26aufirst%3DA.%26aulast%3DJaeger%26aufirst%3DR.%26aulast%3DGro%25CE%25B2%26aufirst%3DR.%26aulast%3DBeckermann%26aufirst%3DB.%26aulast%3DSchlemmer%26aufirst%3DK.-H.%26aulast%3DHaebich%26aufirst%3DD.%26aulast%3DR%25C3%25BCbsamen-Waigmann%26aufirst%3DH.%26atitle%3DInhibition%2520of%2520hepatitis%2520B%2520virus%2520replication%2520by%2520drug-induced%2520depletion%2520of%2520nucleocapsids%26jtitle%3DScience%26date%3D2003%26volume%3D299%26spage%3D893%26epage%3D896%26doi%3D10.1126%2Fscience.1077215" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kocer, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kou, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottaviani, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, J. A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayweg, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, W.</span></span> <span> </span><span class="NLM_article-title">Discovery and pre-clinical characterization of third-generation 4-H heteroaryldihydropyrimidine (HAP) analogues as hepatitis B virus (HBV) capsid inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">3352</span>– <span class="NLM_lpage">3371</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00083</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00083" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01421&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkvFWjtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=3352-3371&author=Z.+Qiuauthor=X.+Linauthor=W.+Zhangauthor=M.+Zhouauthor=L.+Guoauthor=B.+Kocerauthor=G.+Wuauthor=Z.+Zhangauthor=H.+Liuauthor=H.+Shiauthor=B.+Kouauthor=T.+Huauthor=Y.+Huauthor=M.+Huangauthor=S.+F.+Yanauthor=Z.+Xuauthor=Z.+Zhouauthor=N.+Qinauthor=Y.+F.+Wangauthor=S.+Renauthor=H.+Qiuauthor=Y.+Zhangauthor=Y.+Zhangauthor=X.+Wuauthor=K.+Sunauthor=S.+Zhongauthor=J.+Xieauthor=G.+Ottavianiauthor=Y.+Zhouauthor=L.+Zhuauthor=X.+Tianauthor=L.+Shiauthor=F.+Shenauthor=Y.+Maoauthor=X.+Zhouauthor=L.+Gaoauthor=J.+A.+T.+Youngauthor=J.+Z.+Wuauthor=G.+Yangauthor=A.+V.+Maywegauthor=H.+C.+Shenauthor=G.+Tangauthor=W.+Zhu&title=Discovery+and+pre-clinical+characterization+of+third-generation+4-H+heteroaryldihydropyrimidine+%28HAP%29+analogues+as+hepatitis+B+virus+%28HBV%29+capsid+inhibitors&doi=10.1021%2Facs.jmedchem.7b00083"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Pre-Clinical Characterization of Third-Generation 4-H Heteroaryldihydropyrimidine (HAP) Analogues as Hepatitis B Virus (HBV) Capsid Inhibitors</span></div><div class="casAuthors">Qiu, Zongxing; Lin, Xianfeng; Zhang, Weixing; Zhou, Mingwei; Guo, Lei; Kocer, Buelent; Wu, Guolong; Zhang, Zhisen; Liu, Haixia; Shi, Houguang; Kou, Buyu; Hu, Taishan; Hu, Yimin; Huang, Mengwei; Yan, S. Frank; Xu, Zhiheng; Zhou, Zheng; Qin, Ning; Wang, Yue Fen; Ren, Shuang; Qiu, Hongxia; Zhang, Yuxia; Zhang, Yi; Wu, Xiaoyue; Sun, Kai; Zhong, Sheng; Xie, Jianxun; Ottaviani, Giorgio; Zhou, Yuan; Zhu, Lina; Tian, Xiaojun; Shi, Liping; Shen, Fang; Mao, Yi; Zhou, Xue; Gao, Lu; Young, John A. T.; Wu, Jim Zhen; Yang, Guang; Mayweg, Alexander V.; Shen, Hong C.; Tang, Guozhi; Zhu, Wei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3352-3371</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Described herein are the discovery and structure-activity relationship (SAR) studies of the third-generation 4-H heteroaryldihydropyrimidines (4-H HAPs) (I) featuring the introduction of a C6 carboxyl group as novel HBV capsid inhibitors.  This new series of 4-H HAPs showed improved anti-HBV activity and better drug-like properties compared to the first- and second-generation 4-H HAPs.  X-ray crystallog. study of analog 12 (HAP_R01) with Cp149 Y132A mutant hexamer clearly elucidated the role of C6 carboxyl group played for the increased binding affinity, which formed strong hydrogen bonding interactions with capsid protein and coordinated waters.  The representative analog 10 (HAP_R10) was extensively characterized in vitro (ADMET) and in vivo (mouse PK and PD) and subsequently selected for further development as oral anti-HBV infection agent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolNkEF9n_vqrVg90H21EOLACvtfcHk0ljPPaYr0lN69A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkvFWjtbw%253D&md5=6c2a89c594f2c735855f64aca2240837</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00083&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00083%26sid%3Dliteratum%253Aachs%26aulast%3DQiu%26aufirst%3DZ.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DZhou%26aufirst%3DM.%26aulast%3DGuo%26aufirst%3DL.%26aulast%3DKocer%26aufirst%3DB.%26aulast%3DWu%26aufirst%3DG.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DShi%26aufirst%3DH.%26aulast%3DKou%26aufirst%3DB.%26aulast%3DHu%26aufirst%3DT.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DM.%26aulast%3DYan%26aufirst%3DS.%2BF.%26aulast%3DXu%26aufirst%3DZ.%26aulast%3DZhou%26aufirst%3DZ.%26aulast%3DQin%26aufirst%3DN.%26aulast%3DWang%26aufirst%3DY.%2BF.%26aulast%3DRen%26aufirst%3DS.%26aulast%3DQiu%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DSun%26aufirst%3DK.%26aulast%3DZhong%26aufirst%3DS.%26aulast%3DXie%26aufirst%3DJ.%26aulast%3DOttaviani%26aufirst%3DG.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DZhu%26aufirst%3DL.%26aulast%3DTian%26aufirst%3DX.%26aulast%3DShi%26aufirst%3DL.%26aulast%3DShen%26aufirst%3DF.%26aulast%3DMao%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DX.%26aulast%3DGao%26aufirst%3DL.%26aulast%3DYoung%26aufirst%3DJ.%2BA.%2BT.%26aulast%3DWu%26aufirst%3DJ.%2BZ.%26aulast%3DYang%26aufirst%3DG.%26aulast%3DMayweg%26aufirst%3DA.%2BV.%26aulast%3DShen%26aufirst%3DH.%2BC.%26aulast%3DTang%26aufirst%3DG.%26aulast%3DZhu%26aufirst%3DW.%26atitle%3DDiscovery%2520and%2520pre-clinical%2520characterization%2520of%2520third-generation%25204-H%2520heteroaryldihydropyrimidine%2520%2528HAP%2529%2520analogues%2520as%2520hepatitis%2520B%2520virus%2520%2528HBV%2529%2520capsid%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D3352%26epage%3D3371%26doi%3D10.1021%2Facs.jmedchem.7b00083" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, S. F.</span></span> <span> </span><span class="NLM_article-title">Drugs behave as substrates inhibitors and inducers of human cytochrome P450 3A4</span>. <i>Curr. Drug Metab.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">310</span>– <span class="NLM_lpage">322</span>, <span class="refDoi"> DOI: 10.2174/138920008784220664</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01421&amp;key=10.2174%2F138920008784220664" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01421&amp;key=18473749" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01421&amp;key=1%3ACAS%3A528%3ADC%252BD1cXls1aitLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2008&pages=310-322&author=S.+F.+Zhou&title=Drugs+behave+as+substrates+inhibitors+and+inducers+of+human+cytochrome+P450+3A4&doi=10.2174%2F138920008784220664"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Drugs behave as substrates, inhibitors and inducers of human cytochrome P450 3A4</span></div><div class="casAuthors">Zhou, Shu-Feng</div><div class="citationInfo"><span class="NLM_cas:title">Current Drug Metabolism</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">310-322</span>CODEN:
                <span class="NLM_cas:coden">CDMUBU</span>;
        ISSN:<span class="NLM_cas:issn">1389-2002</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Human cytochrome P 450 (CYP) 3A4 is the most abundant hepatic and intestinal phase I enzyme that metabolizes approx. 50% marketed drugs.  The crystal structure of bound and unbound CYP3A4 has been recently constructed, and a small active site and a peripheral binding site are identified.  A recent study indicates that CYP3A4 undergoes dramatic conformational changes upon binding to ketoconazole or erythromycin with a differential but substantial (>80%) increase in the active site vol., providing a structural basis for ligand promiscuity of CYP3A4.  A no. of important drugs have been identified as substrates, inducers and/or inhibitors of CYP3A4.  The ability of drugs to act as inducers, inhibitors, or substrates for CYP3A is predictive of whether concurrent administration of these compds. with a known CYP3A substrate might lead to altered drug disposition, efficacy or toxicity.  The substrates of CYP3A4 considerably overlap with those of P-glycoprotein (P-gp).  To date, the identified clin. important CYP3A4 inhibitors mainly include macrolide antibiotics (e.g., clarithromycin, and erythromycin), anti-HIV agents (e.g., ritonavir and delavirdine), antidepressants (E.g. fluoxetine and fluvoxamine), calcium channel blockers (e.g. verapamil and diltiazem), steroids and their modulators (e.g., gestodene and mifepristone), and several herbal and dietary components.  Many of these drugs are also mechanism-based inhibitors of CYP3A4, which involves formation of reactive metabolites, binding to CYP3A4 and irreversible enzyme inactivation.  A small no. of drugs such as rifampin, phenytoin and ritonavir are identified as inducers of CYP3A4.  The orphan nuclear receptor, pregnane X receptor (PXR), have been found to play a crit. role in the induction of CYP3A4.  The inhibition or induction of CYP3A4 by drugs often causes unfavorable and long-lasting drug-drug interactions and probably fatal toxicity, depending on many factors assocd. with the enzyme, drugs and the patients.  The study of interactions of newly synthesized compds. with CYP3A4 has been incorporated into drug development and detection of possible CYP3A4 inhibitors and inducers during the early stages of drug development is crit. in preventing potential drug-drug interactions and side effects.  Clinicians are encouraged to have a sound knowledge on drugs that behave as substrates, inhibitors or inducers of CYP3A4, and take proper cautions and close monitoring for potential drug interactions when using drugs that are CYP3A4 inhibitors or inducers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrv7PWdiHlenbVg90H21EOLACvtfcHk0ljPPaYr0lN69A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXls1aitLY%253D&md5=df7e7825fd46cd35b8937245771cb972</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.2174%2F138920008784220664&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138920008784220664%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DS.%2BF.%26atitle%3DDrugs%2520behave%2520as%2520substrates%2520inhibitors%2520and%2520inducers%2520of%2520human%2520cytochrome%2520P450%25203A4%26jtitle%3DCurr.%2520Drug%2520Metab.%26date%3D2008%26volume%3D9%26spage%3D310%26epage%3D322%26doi%3D10.2174%2F138920008784220664" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Y.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balkovec, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Royo, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yabut, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evers, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, E. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartley, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mosley, R. T.</span></span> <span> </span><span class="NLM_article-title">Attenuating pregnane X receptor (PXR) activation: A molecular modelling approach</span>. <i>Xenobiotica</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">124</span>– <span class="NLM_lpage">138</span>, <span class="refDoi"> DOI: 10.1080/00498250601050412</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01421&amp;key=10.1080%2F00498250601050412" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01421&amp;key=17484516" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01421&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjt1ChsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2007&pages=124-138&author=Y.-D.+Gaoauthor=S.+H.+Olsonauthor=J.+M.+Balkovecauthor=Y.+Zhuauthor=I.+Royoauthor=J.+Yabutauthor=R.+Eversauthor=E.+Y.+Tanauthor=W.+Tangauthor=D.+P.+Hartleyauthor=R.+T.+Mosley&title=Attenuating+pregnane+X+receptor+%28PXR%29+activation%3A+A+molecular+modelling+approach&doi=10.1080%2F00498250601050412"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Attenuating pregnane X receptor (PXR) activation: a molecular modelling approach</span></div><div class="casAuthors">Gao, Y.-D.; Olson, S. H.; Balkovec, J. M.; Zhu, Y.; Royo, I.; Yabut, J.; Evers, R.; Tan, E. Y.; Tang, W.; Hartley, D. P.; Mosley, R. T.</div><div class="citationInfo"><span class="NLM_cas:title">Xenobiotica</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">124-138</span>CODEN:
                <span class="NLM_cas:coden">XENOBH</span>;
        ISSN:<span class="NLM_cas:issn">0049-8254</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">Recent studies have demonstrated that the pregnane X receptor (PXR) is a key regulator of cytochromes P 450 3A (e.g. CYP3A4 in human) gene expression.  As a result, activation of PXR may lead to CYP3A4 protein over-expression.  Because induction of CYP3A4 could result in clin. important drug-drug interactions, there has been a great interest in reducing the possibility of PXR activation by drug candidates in drug-discovery programs.  In order to provide structural insight for attenuating drug candidate-mediated PXR activation, we used a docking approach to study the structure-activity relationship for PXR activators.  Based on our docking models, it is proposed that introducing polar groups to the end of an activator should reduce its human PXR (hPXR) activity via destabilizing interactions in the hydrophobic areas of the PXR ligand-binding pocket.  A no. of analogs that incorporate these structural features then were designed and synthesized, and they exhibited significantly lower hPXR activation in a transactivation assay and decreased CYP3A4 induction in a human hepatocytes-based assay.  In addn., an example in which attenuating hPXR activation was achieved by sterically destabilizing the helixes 11 and 12 of the receptor is presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBZSC0secQoLVg90H21EOLACvtfcHk0ljPPaYr0lN69A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjt1ChsrY%253D&md5=400688510dd165a4a9632b6ba10bf267</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1080%2F00498250601050412&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F00498250601050412%26sid%3Dliteratum%253Aachs%26aulast%3DGao%26aufirst%3DY.-D.%26aulast%3DOlson%26aufirst%3DS.%2BH.%26aulast%3DBalkovec%26aufirst%3DJ.%2BM.%26aulast%3DZhu%26aufirst%3DY.%26aulast%3DRoyo%26aufirst%3DI.%26aulast%3DYabut%26aufirst%3DJ.%26aulast%3DEvers%26aufirst%3DR.%26aulast%3DTan%26aufirst%3DE.%2BY.%26aulast%3DTang%26aufirst%3DW.%26aulast%3DHartley%26aufirst%3DD.%2BP.%26aulast%3DMosley%26aufirst%3DR.%2BT.%26atitle%3DAttenuating%2520pregnane%2520X%2520receptor%2520%2528PXR%2529%2520activation%253A%2520A%2520molecular%2520modelling%2520approach%26jtitle%3DXenobiotica%26date%3D2007%26volume%3D37%26spage%3D124%26epage%3D138%26doi%3D10.1080%2F00498250601050412" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sinz, M. W.</span></span> <span> </span><span class="NLM_article-title">Avoiding PXR and CAR activation and CYP3A4 enzyme induction</span>. <i>Top. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">159</span>– <span class="NLM_lpage">190</span>, <span class="refDoi"> DOI: 10.1007/7355_2013_24</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01421&amp;key=10.1007%2F7355_2013_24" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=159-190&author=M.+W.+Sinz&title=Avoiding+PXR+and+CAR+activation+and+CYP3A4+enzyme+induction&doi=10.1007%2F7355_2013_24"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1007%2F7355_2013_24&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F7355_2013_24%26sid%3Dliteratum%253Aachs%26aulast%3DSinz%26aufirst%3DM.%2BW.%26atitle%3DAvoiding%2520PXR%2520and%2520CAR%2520activation%2520and%2520CYP3A4%2520enzyme%2520induction%26jtitle%3DTop.%2520Med.%2520Chem.%26date%3D2013%26volume%3D9%26spage%3D159%26epage%3D190%26doi%3D10.1007%2F7355_2013_24" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Groom, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bruno, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lightfoot, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, S. C.</span></span> <span> </span><span class="NLM_article-title">The Cambridgestructural database</span>. <i>Acta Crystallogr., Sect. B: Struct. Sci., Cryst. Eng. Mater.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>B72</i></span>,  <span class="NLM_fpage">171</span>– <span class="NLM_lpage">179</span>, <span class="refDoi"> DOI: 10.1107/S2052520616003954</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01421&amp;key=10.1107%2FS2052520616003954" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=B72&publication_year=2016&pages=171-179&author=C.+R.+Groomauthor=I.+J.+Brunoauthor=M.+P.+Lightfootauthor=S.+C.+Ward&title=The+Cambridgestructural+database&doi=10.1107%2FS2052520616003954"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1107%2FS2052520616003954&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS2052520616003954%26sid%3Dliteratum%253Aachs%26aulast%3DGroom%26aufirst%3DC.%2BR.%26aulast%3DBruno%26aufirst%3DI.%2BJ.%26aulast%3DLightfoot%26aufirst%3DM.%2BP.%26aulast%3DWard%26aufirst%3DS.%2BC.%26atitle%3DThe%2520Cambridgestructural%2520database%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520B%253A%2520Struct.%2520Sci.%252C%2520Cryst.%2520Eng.%2520Mater.%26date%3D2016%26volume%3DB72%26spage%3D171%26epage%3D179%26doi%3D10.1107%2FS2052520616003954" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span>In Vitro
Metabolism and Transporter Mediated Drug-Drug Interaction Studies
Guidance for Industry. <a href="https://www.fda.gov/downloads/Drugs/Guidances/UCM581965.pdf" class="extLink">https://www.fda.gov/downloads/Drugs/Guidances/UCM581965.pdf</a> (accessed Apr 16, <span class="NLM_year">2019</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=In+Vitro%0AMetabolism+and+Transporter+Mediated+Drug-Drug+Interaction+Studies%0AGuidance+for+Industry.+https%3A%2F%2Fwww.fda.gov%2Fdownloads%2FDrugs%2FGuidances%2FUCM581965.pdf+%28accessed+Apr+16%2C+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nagai, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konno, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Satsukawa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamashita, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshinari, K.</span></span> <span> </span><span class="NLM_article-title">Establishment of in silico prediction models for CYP3A4 and CYP2B6 induction in human hepatocytes by multiple regressionanalysis using azole compounds</span>. <i>Drug Metab. Dispos.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">1390</span>– <span class="NLM_lpage">1398</span>, <span class="refDoi"> DOI: 10.1124/dmd.115.068619</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01421&amp;key=10.1124%2Fdmd.115.068619" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01421&amp;key=27208383" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01421&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVajtLfP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=1390-1398&author=M.+Nagaiauthor=Y.+Konnoauthor=M.+Satsukawaauthor=S.+Yamashitaauthor=K.+Yoshinari&title=Establishment+of+in+silico+prediction+models+for+CYP3A4+and+CYP2B6+induction+in+human+hepatocytes+by+multiple+regressionanalysis+using+azole+compounds&doi=10.1124%2Fdmd.115.068619"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Establishment of in silico prediction models for CYP3A4 and CYP2B6 induction in human hepatocytes by multiple regression analysis using azole compounds</span></div><div class="casAuthors">Nagai, Mika; Konno, Yoshihiro; Satsukawa, Masahiro; Yamashita, Shinji; Yoshinari, Kouichi</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism & Disposition</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1390-1398</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">1521-009X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Drug-drug interactions (DDIs) via cytochrome P 450 (P 450) induction are one clin. problem leading to increased risk of adverse effects and the need for dosage adjustments and addnl. therapeutic monitoring.  In silico models for predicting P 450 induction are useful for avoiding DDI risk.  In this study, we have established regression models for CYP3A4 and CYP2B6 induction in human hepatocytes using several physicochem. parameters for a set of azole compds. with different P 450 induction as characteristics as model compds.  To obtain a well-correlated regression model, the compds. for CYP3A4 or CYP2B6 induction were independently selected from the tested azole compds. using principal component anal. with fold-induction data.  Both of the multiple linear regression models obtained for CYP3A4 and CYP2B6 induction are represented by different sets of physicochem. parameters.  The adjusted coeffs. of detn. for these models were of 0.8 and 0.9, resp.  The fold-induction of the validation compds., another set of 12 azole-contg. compds., were predicted within twofold limits for both CYP3A4 and CYP2B6.  The concordance for the prediction of CYP3A4 induction was 87% with another validation set, 23 marketed drugs.  However, the prediction of CYP2B6 induction tended to be overestimated for these marketed drugs.  The regression models show that lipophilicity mostly contributes to CYP3A4 induction, whereas not only the lipophilicity but also the mol. polarity is important for CYP2B6 induction.  Our regression models, esp. that for CYP3A4 induction, might provide useful methods to avoid potent CYP3A4 or CYP2B6 inducers during the lead optimization stage without performing induction assays in human hepatocytes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJ-HLYUJTykbVg90H21EOLACvtfcHk0lhGPEiSgVjXMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVajtLfP&md5=cb2a43dde2f98fc2d4bd20f966f89c3e</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1124%2Fdmd.115.068619&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.115.068619%26sid%3Dliteratum%253Aachs%26aulast%3DNagai%26aufirst%3DM.%26aulast%3DKonno%26aufirst%3DY.%26aulast%3DSatsukawa%26aufirst%3DM.%26aulast%3DYamashita%26aufirst%3DS.%26aulast%3DYoshinari%26aufirst%3DK.%26atitle%3DEstablishment%2520of%2520in%2520silico%2520prediction%2520models%2520for%2520CYP3A4%2520and%2520CYP2B6%2520induction%2520in%2520human%2520hepatocytes%2520by%2520multiple%2520regressionanalysis%2520using%2520azole%2520compounds%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2016%26volume%3D44%26spage%3D1390%26epage%3D1398%26doi%3D10.1124%2Fdmd.115.068619" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Althage, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chisari, F. V.</span></span> <span> </span><span class="NLM_article-title">Hydrodynamic injection of viral DNA: a mouse model of acute hepatitis B virus infection</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>99</i></span>,  <span class="NLM_fpage">13825</span>– <span class="NLM_lpage">13830</span>, <span class="refDoi"> DOI: 10.1073/pnas.202398599</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01421&amp;key=10.1073%2Fpnas.202398599" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01421&amp;key=12374864" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01421&amp;key=1%3ACAS%3A528%3ADC%252BD38XotVKmtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2002&pages=13825-13830&author=P.+L.+Yangauthor=A.+Althageauthor=J.+Chungauthor=F.+V.+Chisari&title=Hydrodynamic+injection+of+viral+DNA%3A+a+mouse+model+of+acute+hepatitis+B+virus+infection&doi=10.1073%2Fpnas.202398599"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Hydrodynamic injection of viral DNA: a mouse model of acute hepatitis B virus infection</span></div><div class="casAuthors">Yang, Priscilla L.; Althage, Alana; Chung, Josan; Chisari, Francis V.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">99</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">13825-13830</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Hepatitis B virus (HBV) is a prototype for liver-specific pathogens in which the failure of the immune system to mount an effective response leads to chronic infection.  The authors' understanding of the immune response to HBV is incomplete, largely due to the narrow host restriction of this pathogen and the limitations of existing exptl. models.  The authors have developed a murine model for studying human HBV replication, immunogenicity, and control.  After transfection of hepatocytes in vivo with a replication-competent, over-length, linear HBV genome, viral antigens and replicative intermediates were synthesized and virus was secreted into the blood.  Viral antigens disappeared from the blood as early as 7 days after transfection, coincident with the appearance of antiviral antibodies.  HBV transcripts and replicative intermediates disappeared from the liver by day 15, after the appearance of antiviral CD8 + T cells.  In contrast, the virus persisted for at least 81 days after transfection of NOD/Scid mice, which lack functional T cells, B cells, and natural killer (NK) cells.  Thus, the outcome of hydrodynamic transfection of HBV depends on the host immune response, as it is during a natural infection.  The methods the authors describe will allow the examn. of viral dynamics in a tightly controlled in vivo system, the application of mutagenesis methods to the study of the HBV life cycle in vivo, and the dissection of the immune response to HBV using genetically modified mice whose immunoregulatory and immune effector functions have been deleted or overexpressed.  In addn., this methodol. represents a prototype for the study of other known and to-be-discovered liver-specific pathogens.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo98nNA_w5YxrVg90H21EOLACvtfcHk0lhGPEiSgVjXMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XotVKmtbw%253D&md5=0c0f40c386fe47f6c57c413fccdaf3e5</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1073%2Fpnas.202398599&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.202398599%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DP.%2BL.%26aulast%3DAlthage%26aufirst%3DA.%26aulast%3DChung%26aufirst%3DJ.%26aulast%3DChisari%26aufirst%3DF.%2BV.%26atitle%3DHydrodynamic%2520injection%2520of%2520viral%2520DNA%253A%2520a%2520mouse%2520model%2520of%2520acute%2520hepatitis%2520B%2520virus%2520infection%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2002%26volume%3D99%26spage%3D13825%26epage%3D13830%26doi%3D10.1073%2Fpnas.202398599" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sells, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Acs, G.</span></span> <span> </span><span class="NLM_article-title">Production of hepatitis B virus particles in HepG2 cells transfected with cloned hepatitis B virus DNA</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">1987</span>,  <span class="NLM_volume"><i>84</i></span>,  <span class="NLM_fpage">1005</span>– <span class="NLM_lpage">1009</span>, <span class="refDoi"> DOI: 10.1073/pnas.84.4.1005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01421&amp;key=10.1073%2Fpnas.84.4.1005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01421&amp;key=3029758" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01421&amp;key=1%3ACAS%3A528%3ADyaL2sXhvV2qt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=84&publication_year=1987&pages=1005-1009&author=M.+A.+Sellsauthor=M.+L.+Chenauthor=G.+Acs&title=Production+of+hepatitis+B+virus+particles+in+HepG2+cells+transfected+with+cloned+hepatitis+B+virus+DNA&doi=10.1073%2Fpnas.84.4.1005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Production of hepatitis B virus particles in Hep G2 cells transfected with cloned hepatitis B virus DNA</span></div><div class="casAuthors">Sells, Mary Ann; Chen, Mei Ling; Acs, George</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1987</span>),
    <span class="NLM_cas:volume">84</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1005-9</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    </div><div class="casAbstract">The hepatoblastoma cell line Hep G2 was transfected with a plasmid carrying the gene that confers resistance to G418 and four 5'-3' tandem copies of the hepatitis B virus (HBV) genome positioned such that two dimers of the genomic DNA are 3'-3' with respect to one another.  Cells of one clone that grew in the presence of G418 produce high levels of hepatitis B e antigen and of hepatitis B surface antigen.  HBV DNA is carried by these cells as chromosomally integrated sequences and episomally as relaxed circular, covalently closed, and incomplete copies of the HBV genome.  Viral DNA was detected also in conditioned growth medium at the buoyant densities characteristic for infectious Dane and immature core particles.  Finally, HBV-specific components morphol. identical to the 22-nm spherical and filamentous hepatitis B surface antigen particles as well as 42-nm Dane particles were visualized by immunoelectron microscopic anal.  Therefore, the Hep G2 cell line can support the assembly and secretion not only of several of the replicative intermediates of HBV DNA but also of Dane-like particles.  This in vitro system can now be used to study the life cycle of HBV and the reaction of immunocompetent cells with cells carrying HBV.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGogbjqX6YU-W7Vg90H21EOLACvtfcHk0likaNa2V2AqWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2sXhvV2qt7c%253D&md5=27744d6955d4701586243f651c79ab1e</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1073%2Fpnas.84.4.1005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.84.4.1005%26sid%3Dliteratum%253Aachs%26aulast%3DSells%26aufirst%3DM.%2BA.%26aulast%3DChen%26aufirst%3DM.%2BL.%26aulast%3DAcs%26aufirst%3DG.%26atitle%3DProduction%2520of%2520hepatitis%2520B%2520virus%2520particles%2520in%2520HepG2%2520cells%2520transfected%2520with%2520cloned%2520hepatitis%2520B%2520virus%2520DNA%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1987%26volume%3D84%26spage%3D1005%26epage%3D1009%26doi%3D10.1073%2Fpnas.84.4.1005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuconati, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Block, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, J.-T.</span></span> <span> </span><span class="NLM_article-title">Characterization of the intracellular deproteinized relaxed circular DNA of hepatitis B virus: an intermediate of covalently closed circular DNA formation</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>81</i></span>,  <span class="NLM_fpage">12472</span>– <span class="NLM_lpage">12484</span>, <span class="refDoi"> DOI: 10.1128/JVI.01123-07</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01421&amp;key=10.1128%2FJVI.01123-07" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01421&amp;key=17804499" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01421&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlKku7zJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=2007&pages=12472-12484&author=H.+Guoauthor=D.+Jiangauthor=T.+Zhouauthor=A.+Cuconatiauthor=T.+M.+Blockauthor=J.-T.+Guo&title=Characterization+of+the+intracellular+deproteinized+relaxed+circular+DNA+of+hepatitis+B+virus%3A+an+intermediate+of+covalently+closed+circular+DNA+formation&doi=10.1128%2FJVI.01123-07"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of the intracellular deproteinized relaxed circular DNA of hepatitis B virus: an intermediate of covalently closed circular DNA formation</span></div><div class="casAuthors">Guo, Haitao; Jiang, Dong; Zhou, Tianlun; Cuconati, Andrea; Block, Timothy M.; Guo, Ju-Tao</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">81</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">12472-12484</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Covalently closed circular DNA (cccDNA) of hepatitis B virus (HBV) is formed by conversion of capsid-assocd. relaxed circular DNA (rcDNA) via unknown mechanisms and exists in the nucleus of the infected hepatocyte as a minichromosome that serves as the transcription template for viral RNAs.  To study the mol. pathway of cccDNA formation and its regulation by viral and cellular factors, we have established a cell line that supports the replication of an envelope protein-deficient HBV genome in a tetracycline-inducible manner.  Following induction of HBV replication, the cells accumulate higher levels of cccDNA as well as larger amts. of deproteinized rcDNA (DP-rcDNA) than cells that replicate wild-type HBV genomes.  These results indicate that HBV envelope proteins neg. regulate cccDNA formation, and conversion of DP-rcDNA into cccDNA is a rate-limiting step of cccDNA formation in HepG2 cells.  Detailed analyses reveal the following: (i) DP-rcDNA exists in both cytoplasm and nucleus; (ii) while nuclear DP-rcDNA is sensitive to DNase I digestion, a small fraction of cytoplasmic DP-rcDNA is DNase I resistant; (iii) both DNase I-sensitive and -resistant cytoplasmic DP-rcDNAs cosediment with capsids and can be immunopptd. with HBV core antibody; and (iv) a primer extension assay maps the 5' end of the minus strand of DP-rcDNA at the authentic end of virion rcDNA.  Hence, our results favor a hypothesis that the removal of viral polymerase protein covalently linked to the 5' end of the minus-strand DNA occurs inside the capsid in the cytoplasm and most possibly via a reaction that cleaves the phosphodiester bond between the tyrosine of the polymerase and the 5' phosphoryl group of minus-strand DNA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGot_WiB1hJ4zbVg90H21EOLACvtfcHk0likaNa2V2AqWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlKku7zJ&md5=91d6a9284b406680c85be2e35048d4ae</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1128%2FJVI.01123-07&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.01123-07%26sid%3Dliteratum%253Aachs%26aulast%3DGuo%26aufirst%3DH.%26aulast%3DJiang%26aufirst%3DD.%26aulast%3DZhou%26aufirst%3DT.%26aulast%3DCuconati%26aufirst%3DA.%26aulast%3DBlock%26aufirst%3DT.%2BM.%26aulast%3DGuo%26aufirst%3DJ.-T.%26atitle%3DCharacterization%2520of%2520the%2520intracellular%2520deproteinized%2520relaxed%2520circular%2520DNA%2520of%2520hepatitis%2520B%2520virus%253A%2520an%2520intermediate%2520of%2520covalently%2520closed%2520circular%2520DNA%2520formation%26jtitle%3DJ.%2520Virol.%26date%3D2007%26volume%3D81%26spage%3D12472%26epage%3D12484%26doi%3D10.1128%2FJVI.01123-07" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 1 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Adrian Hall, Hugues Chanteux, Karelle Ménochet, Marie Ledecq, <span class="NLM_string-name hlFld-ContribAuthor">Monika-Sarah E. D. Schulze</span>. </span><span class="cited-content_cbyCitation_article-title">Designing Out PXR Activity on Drug Discovery Projects: A Review of Structure-Based Methods, Empirical and Computational Approaches. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (10)
                                     , 6413-6522. <a href="https://doi.org/10.1021/acs.jmedchem.0c02245" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02245</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02245&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c02245%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDesigning%252BOut%252BPXR%252BActivity%252Bon%252BDrug%252BDiscovery%252BProjects%25253A%252BA%252BReview%252Bof%252BStructure-Based%252BMethods%25252C%252BEmpirical%252Band%252BComputational%252BApproaches%26aulast%3DHall%26aufirst%3DAdrian%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D27122020%26date%3D18052021%26volume%3D64%26issue%3D10%26spage%3D6413%26epage%3D6522" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01421/20191122/images/medium/jm9b01421_0007.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01421/20191122/images/large/jm9b01421_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01421&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01421/20191122/images/medium/jm9b01421_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01421/20191122/images/large/jm9b01421_0001.jpeg" id="rightTab-GRAPHIC-d7e461-autogenerated" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. (a) Crystal structure of Cp149 Y132A in complex with compound <b>1</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5WRE">5WRE</a>). Compound <b>1</b> is shown in stick with cyan carbons. The molecular surface of the protein is shown in gray in the vicinity of the ligand. The core protein monomer forming the top of the binding pocket is omitted for clarity. The dihedral angle constituted by the bonds labeled in red in the schematic representation was −64.3° in the cocrystal structure. (b) X-ray of <b>1a</b>. Compound <b>1a</b> is shown in stick with the green carbons. The dihedral angle constituted by the bonds labeled in red in the schematic representation was −64.5°. (c) Overlay of <b>1</b> (cyan) and <b>1a</b> (green).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01421/20191122/images/large/jm9b01421_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01421&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01421/20191122/images/medium/jm9b01421_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01421/20191122/images/large/jm9b01421_0004.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Preparation of Compound <b>19</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01421/20191122/images/large/jm9b01421_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01421&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <i>i</i>-PrOH, HOAc, piperidine, 48 °C, 2 h, followed by the addition of thiazole-2-carboxamidine, then 90 °C, 3 h, 52%; (b) SFC chiral separation; (c) NBS, CCl<sub>4</sub>, rt, 1 h, 44%; (d) DIPEA, DCM, ∼1 h, 57%.</p></p></figure><figure data-id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01421/20191122/images/medium/jm9b01421_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01421/20191122/images/large/jm9b01421_0002.jpeg" id="rightTab-GRAPHIC-d7e981-autogenerated" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (a) X-ray of <b>(<i>S</i>)-19a</b>. Compound <b>(<i>S</i>)-19a</b> is shown as ball and stick with the color of carbon in purple. The dihedral angle consisting of the bonds labeled in red in the schematic representation was 1.3°. (b) Overlay of <b>1a</b> (green) and <b>(<i>S</i>)-19a</b> (purple).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01421/20191122/images/large/jm9b01421_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01421&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01421/20191122/images/medium/jm9b01421_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01421/20191122/images/large/jm9b01421_0003.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Levels of HBV DNA in plasma (A, left) and liver (B, right) of HBV HDI mice upon treatment with the vehicle control, ETV control, and compound <b>19</b>. Error bars indicate standard error of the mean.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01421/20191122/images/large/jm9b01421_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01421&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i50">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_40067" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_40067" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 15 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span>World Health
Organization (WHO). Fact Sheet No. 204. <a href="http://www.who.int/mediacentre/factsheets/fs204/en/" class="extLink">http://www.who.int/mediacentre/factsheets/fs204/en/</a> (accessed Apr 16, <span class="NLM_year">2019</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=World+Health%0AOrganization+%28WHO%29.+Fact+Sheet+No.+204.+http%3A%2F%2Fwww.who.int%2Fmediacentre%2Ffactsheets%2Ffs204%2Fen%2F+%28accessed+Apr+16%2C+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Feng, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, J. A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yun, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, H. C.</span></span> <span> </span><span class="NLM_article-title">Discovery of small molecule therapeutics for treatment of chronic HBV infection</span>. <i>ACS Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">257</span>– <span class="NLM_lpage">277</span>, and references cited therein <span class="refDoi"> DOI: 10.1021/acsinfecdis.7b00144</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsinfecdis.7b00144" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01421&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs1yrtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2018&pages=257-277&author=S.+Fengauthor=L.+Gaoauthor=X.+Hanauthor=T.+Huauthor=Y.+Huauthor=H.+Liuauthor=A.+W.+Thomasauthor=Z.+Yanauthor=S.+Yangauthor=J.+A.+T.+Youngauthor=H.+Yunauthor=W.+Zhuauthor=H.+C.+Shen&title=Discovery+of+small+molecule+therapeutics+for+treatment+of+chronic+HBV+infection&doi=10.1021%2Facsinfecdis.7b00144"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Small Molecule Therapeutics for Treatment of Chronic HBV Infection</span></div><div class="casAuthors">Feng, Song; Gao, Lu; Han, Xingchun; Hu, Taishan; Hu, Yimin; Liu, Haixia; Thomas, Andrew W.; Yan, Zhipeng; Yang, Song; Young, John A. T.; Yun, Hongying; Zhu, Wei; Shen, Hong C.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Infectious Diseases</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">257-277</span>CODEN:
                <span class="NLM_cas:coden">AIDCBC</span>;
        ISSN:<span class="NLM_cas:issn">2373-8227</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The chronic infection of hepatitis B virus (HBV) inflicts 250 million people worldwide representing a major public health threat.  A significant sub-population of chronically infected HBV patients eventually develop cirrhosis and hepatocellular carcinoma (HCC) which aggravate the issues caused by hepatitis alone.  Unfortunately, none of the current std. therapies for chronic Hepatitis B (CHB) result in a satisfactory clin. cure rate.  Driven by a highly unmet medical need for novel and more effective treatments of HBV infection, multiple pharmaceutical companies and research institutions have been engaged in drug discovery and development to improve the CHB functional cure rate, which is defined by sustainable viral suppression and HBsAg clearance after a finite treatment.  This review summarizes the recent significant advances in the discovery and development of novel small mol. anti-HBV compds., highlighted by representative structures, mechanism of action (MoA), and biol. activities.  Lastly, it is believed that improved CHB functional cure rate may be accomplished via the combination of mols. with distinct MoAs.  Thus some of the mols. may evolve into key components of a suitable combination therapy leading to superior outcome of clin. efficacy in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1zsMqu16n17Vg90H21EOLACvtfcHk0lhJcX3srlbvPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs1yrtbY%253D&md5=45e6f4740904bd92d82193455a2d97ac</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1021%2Facsinfecdis.7b00144&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsinfecdis.7b00144%26sid%3Dliteratum%253Aachs%26aulast%3DFeng%26aufirst%3DS.%26aulast%3DGao%26aufirst%3DL.%26aulast%3DHan%26aufirst%3DX.%26aulast%3DHu%26aufirst%3DT.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DThomas%26aufirst%3DA.%2BW.%26aulast%3DYan%26aufirst%3DZ.%26aulast%3DYang%26aufirst%3DS.%26aulast%3DYoung%26aufirst%3DJ.%2BA.%2BT.%26aulast%3DYun%26aufirst%3DH.%26aulast%3DZhu%26aufirst%3DW.%26aulast%3DShen%26aufirst%3DH.%2BC.%26atitle%3DDiscovery%2520of%2520small%2520molecule%2520therapeutics%2520for%2520treatment%2520of%2520chronic%2520HBV%2520infection%26jtitle%3DACS%2520Infect.%2520Dis.%26date%3D2018%26volume%3D4%26spage%3D257%26epage%3D277%26doi%3D10.1021%2Facsinfecdis.7b00144" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Testoni, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durantel, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoulim, F.</span></span> <span> </span><span class="NLM_article-title">Novel targets for hepatitis B virus therapy</span>. <i>Liver Int.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>37</i></span> (<span class="NLM_issue">S1</span>),  <span class="NLM_fpage">33</span>– <span class="NLM_lpage">39</span>, <span class="refDoi"> DOI: 10.1111/liv.13307</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01421&amp;key=10.1111%2Fliv.13307" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01421&amp;key=28052622" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01421&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjvFWntA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2017&pages=33-39&issue=S1&author=B.+Testoniauthor=D.+Durantelauthor=F.+Zoulim&title=Novel+targets+for+hepatitis+B+virus+therapy&doi=10.1111%2Fliv.13307"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Novel targets for hepatitis B virus therapy</span></div><div class="casAuthors">Testoni, Barbara; Durantel, David; Zoulim, Fabien</div><div class="citationInfo"><span class="NLM_cas:title">Liver International</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">S1</span>),
    <span class="NLM_cas:pages">33-39</span>CODEN:
                <span class="NLM_cas:coden">LIINCM</span>;
        ISSN:<span class="NLM_cas:issn">1478-3223</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Treatment with either pegylated interferon-alpha (pegIFN-α) or last generation nucleos(t)ide analogs (NAs) successfully leads to serum viral load suppression in most chronically infected hepatitis B (CHB) patients, but HBsAg loss is only achieved in 10% of the cases after a 5-yr follow-up.  Thus, therapy must be administered long-term and it will not completely eliminate infection because of the persistent hepatitis B virus (HBV) minichromosome in infected cells, and cannot completely abolish the risk of developing severe sequelae such as cirrhosis and hepatocellular carcinoma.  Recent progress in the development of in vitro and in vivo models of HBV infection have helped renew interest in the investigation of the viral life cycle, as well as specific virus-host cell interactions to identify new targets for the development of new antiviral drugs.  This includes either direct inhibition of viral replication by targeting fundamental steps such as entry, cccDNA formation/stability, viral transcripts, capsid assembly and secretion or the manipulation of the host immune system for better defense against infection.  Multiple strategies are currently under investigation, including boosting endogenous innate responses and/or restoring adaptive immunity via engineering of HBV-specific T cells or via the use of inhibitors of neg. regulators, as well as therapeutic vaccines.  It is increasingly clear that multiple therapeutic strategies must be combined to reach a cure of HBV and that the definition of clin., virol. and immunol. correlates for the management of treatment are urgently needed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCpuA-v_62UbVg90H21EOLACvtfcHk0lhJcX3srlbvPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjvFWntA%253D%253D&md5=d6631b3098076a3795070480c7016998</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1111%2Fliv.13307&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fliv.13307%26sid%3Dliteratum%253Aachs%26aulast%3DTestoni%26aufirst%3DB.%26aulast%3DDurantel%26aufirst%3DD.%26aulast%3DZoulim%26aufirst%3DF.%26atitle%3DNovel%2520targets%2520for%2520hepatitis%2520B%2520virus%2520therapy%26jtitle%3DLiver%2520Int.%26date%3D2017%26volume%3D37%26issue%3DS1%26spage%3D33%26epage%3D39%26doi%3D10.1111%2Fliv.13307" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, J.-T.</span></span> <span> </span><span class="NLM_article-title">The current status and future directions of hepatitis B antiviral drug discovery</span>. <i>Expert Opin. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">5</span>– <span class="NLM_lpage">15</span>, <span class="refDoi"> DOI: 10.1080/17460441.2017.1255195</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01421&amp;key=10.1080%2F17460441.2017.1255195" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01421&amp;key=27797587" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01421&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFSls7%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=5-15&author=L.+Tangauthor=Q.+Zhaoauthor=S.+Wuauthor=J.+Chengauthor=J.+Changauthor=J.-T.+Guo&title=The+current+status+and+future+directions+of+hepatitis+B+antiviral+drug+discovery&doi=10.1080%2F17460441.2017.1255195"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">The current status and future directions of hepatitis B antiviral drug discovery</span></div><div class="casAuthors">Tang, Liudi; Zhao, Qiong; Wu, Shuo; Cheng, Junjun; Chang, Jinhong; Guo, Ju-Tao</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">5-15</span>CODEN:
                <span class="NLM_cas:coden">EODDBX</span>;
        ISSN:<span class="NLM_cas:issn">1746-0441</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">The current std. care of chronic hepatitis B fails to induce a durable off-drug control of hepatitis B virus (HBV) replication in the majority of treated patients.  The primary reasons are its inability to eliminate the covalently closed circular (ccc) DNA, the nuclear form of HBV genome, and restoration of the dysfunctional host antiviral immune response against the virus.  Accordingly, discovery and development of therapeutics to completely stop HBV replication, eliminate or functionally inactivate cccDNA as well as activate a functional antiviral immune response against HBV are the primary efforts for finding a cure for chronic hepatitis B.  Herein, the authors highlight the current efforts of HBV drug discovery and offer their opinions for the future directions of this research.  The authors believe that through a consecutive or overlapping three-stage antiviral and immunotherapy program to: (i) completely stop HBV replication and cccDNA amplification; (ii) reduce viral antigen load and induce HBV surface antigen (HBsAg) seroclearance through eradication or inactivation of cccDNA and RNA interference-mediated degrdn. of viral mRNA and (iii) activate a functional antiviral immune response against HBV through therapeutic immunization or immunotherapy, a functional cure of chronic HBV infection can be achieved in the majority of chronic HBV carriers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3PtMZrdkoM7Vg90H21EOLACvtfcHk0liL3DKBp7rqaQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFSls7%252FF&md5=e1425fe2bd4b239065211a78d19b2b1f</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1080%2F17460441.2017.1255195&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F17460441.2017.1255195%26sid%3Dliteratum%253Aachs%26aulast%3DTang%26aufirst%3DL.%26aulast%3DZhao%26aufirst%3DQ.%26aulast%3DWu%26aufirst%3DS.%26aulast%3DCheng%26aufirst%3DJ.%26aulast%3DChang%26aufirst%3DJ.%26aulast%3DGuo%26aufirst%3DJ.-T.%26atitle%3DThe%2520current%2520status%2520and%2520future%2520directions%2520of%2520hepatitis%2520B%2520antiviral%2520drug%2520discovery%26jtitle%3DExpert%2520Opin.%2520Drug%2520Discovery%26date%3D2017%26volume%3D12%26spage%3D5%26epage%3D15%26doi%3D10.1080%2F17460441.2017.1255195" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deres, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schröder, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paessens, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldmann, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hacker, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weber, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krämer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niewöhner, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pleiss, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stoltefuss, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graef, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koletzki, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masantschek, R. N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reimann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaeger, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groβ, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beckermann, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlemmer, K.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haebich, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rübsamen-Waigmann, H.</span></span> <span> </span><span class="NLM_article-title">Inhibition of hepatitis B virus replication by drug-induced depletion of nucleocapsids</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>299</i></span>,  <span class="NLM_fpage">893</span>– <span class="NLM_lpage">896</span>, <span class="refDoi"> DOI: 10.1126/science.1077215</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01421&amp;key=10.1126%2Fscience.1077215" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01421&amp;key=12574631" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01421&amp;key=1%3ACAS%3A528%3ADC%252BD3sXpt1Sgsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=299&publication_year=2003&pages=893-896&author=K.+Deresauthor=C.+H.+Schr%C3%B6derauthor=A.+Paessensauthor=S.+Goldmannauthor=H.+J.+Hackerauthor=O.+Weberauthor=T.+Kr%C3%A4merauthor=U.+Niew%C3%B6hnerauthor=U.+Pleissauthor=J.+Stoltefussauthor=E.+Graefauthor=D.+Koletzkiauthor=R.+N.+A.+Masantschekauthor=A.+Reimannauthor=R.+Jaegerauthor=R.+Gro%CE%B2author=B.+Beckermannauthor=K.-H.+Schlemmerauthor=D.+Haebichauthor=H.+R%C3%BCbsamen-Waigmann&title=Inhibition+of+hepatitis+B+virus+replication+by+drug-induced+depletion+of+nucleocapsids&doi=10.1126%2Fscience.1077215"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of Hepatitis B Virus Replication by Drug-Induced Depletion of Nucleocapsids</span></div><div class="casAuthors">Deres, Karl; Schroeder, Claus H.; Paessens, Arnold; Goldmann, Siegfried; Hacker, Hans Joerg; Weber, Olaf; Kraemer, Thomas; Niewoehner, Ulrich; Pleiss, Ulrich; Stoltefuss, Juergen; Graef, Erwin; Koletzki, Diana; Masantschek, Ralf N. A.; Reimann, Anja; Jaeger, Rainer; Gro, Rainer; Beckermann, Bernhard; Schlemmer, Karl-Heinz; Haebich, Dieter; Ruebsamen-Waigmann, Helga</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">299</span>
        (<span class="NLM_cas:issue">5608</span>),
    <span class="NLM_cas:pages">893-896</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Chronic hepatitis B virus (HBV) infection is a major cause of liver disease.  Only interferon-α and the nucleosidic inhibitors of the viral polymerase, 3TC and adefovir, are approved for therapy.  However, these therapies are limited by the side effects of interferon and the substantial resistance of the virus to nucleosidic inhibitors.  Potent new antiviral compds. suitable for monotherapy or combination therapy are highly desired.  We describe non-nucleosidic inhibitors of HBV nucleocapsid maturation that possess in vitro and in vivo antiviral activity.  These inhibitors have potential for future therapeutic regimens to combat chronic HBV infection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAlJUFASSEhLVg90H21EOLACvtfcHk0liL3DKBp7rqaQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXpt1Sgsg%253D%253D&md5=a5e74b45d335f74ba17ec9fe4d31e24b</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1126%2Fscience.1077215&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1077215%26sid%3Dliteratum%253Aachs%26aulast%3DDeres%26aufirst%3DK.%26aulast%3DSchr%25C3%25B6der%26aufirst%3DC.%2BH.%26aulast%3DPaessens%26aufirst%3DA.%26aulast%3DGoldmann%26aufirst%3DS.%26aulast%3DHacker%26aufirst%3DH.%2BJ.%26aulast%3DWeber%26aufirst%3DO.%26aulast%3DKr%25C3%25A4mer%26aufirst%3DT.%26aulast%3DNiew%25C3%25B6hner%26aufirst%3DU.%26aulast%3DPleiss%26aufirst%3DU.%26aulast%3DStoltefuss%26aufirst%3DJ.%26aulast%3DGraef%26aufirst%3DE.%26aulast%3DKoletzki%26aufirst%3DD.%26aulast%3DMasantschek%26aufirst%3DR.%2BN.%2BA.%26aulast%3DReimann%26aufirst%3DA.%26aulast%3DJaeger%26aufirst%3DR.%26aulast%3DGro%25CE%25B2%26aufirst%3DR.%26aulast%3DBeckermann%26aufirst%3DB.%26aulast%3DSchlemmer%26aufirst%3DK.-H.%26aulast%3DHaebich%26aufirst%3DD.%26aulast%3DR%25C3%25BCbsamen-Waigmann%26aufirst%3DH.%26atitle%3DInhibition%2520of%2520hepatitis%2520B%2520virus%2520replication%2520by%2520drug-induced%2520depletion%2520of%2520nucleocapsids%26jtitle%3DScience%26date%3D2003%26volume%3D299%26spage%3D893%26epage%3D896%26doi%3D10.1126%2Fscience.1077215" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kocer, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kou, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottaviani, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, J. A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayweg, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, W.</span></span> <span> </span><span class="NLM_article-title">Discovery and pre-clinical characterization of third-generation 4-H heteroaryldihydropyrimidine (HAP) analogues as hepatitis B virus (HBV) capsid inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">3352</span>– <span class="NLM_lpage">3371</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00083</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00083" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01421&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkvFWjtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=3352-3371&author=Z.+Qiuauthor=X.+Linauthor=W.+Zhangauthor=M.+Zhouauthor=L.+Guoauthor=B.+Kocerauthor=G.+Wuauthor=Z.+Zhangauthor=H.+Liuauthor=H.+Shiauthor=B.+Kouauthor=T.+Huauthor=Y.+Huauthor=M.+Huangauthor=S.+F.+Yanauthor=Z.+Xuauthor=Z.+Zhouauthor=N.+Qinauthor=Y.+F.+Wangauthor=S.+Renauthor=H.+Qiuauthor=Y.+Zhangauthor=Y.+Zhangauthor=X.+Wuauthor=K.+Sunauthor=S.+Zhongauthor=J.+Xieauthor=G.+Ottavianiauthor=Y.+Zhouauthor=L.+Zhuauthor=X.+Tianauthor=L.+Shiauthor=F.+Shenauthor=Y.+Maoauthor=X.+Zhouauthor=L.+Gaoauthor=J.+A.+T.+Youngauthor=J.+Z.+Wuauthor=G.+Yangauthor=A.+V.+Maywegauthor=H.+C.+Shenauthor=G.+Tangauthor=W.+Zhu&title=Discovery+and+pre-clinical+characterization+of+third-generation+4-H+heteroaryldihydropyrimidine+%28HAP%29+analogues+as+hepatitis+B+virus+%28HBV%29+capsid+inhibitors&doi=10.1021%2Facs.jmedchem.7b00083"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Pre-Clinical Characterization of Third-Generation 4-H Heteroaryldihydropyrimidine (HAP) Analogues as Hepatitis B Virus (HBV) Capsid Inhibitors</span></div><div class="casAuthors">Qiu, Zongxing; Lin, Xianfeng; Zhang, Weixing; Zhou, Mingwei; Guo, Lei; Kocer, Buelent; Wu, Guolong; Zhang, Zhisen; Liu, Haixia; Shi, Houguang; Kou, Buyu; Hu, Taishan; Hu, Yimin; Huang, Mengwei; Yan, S. Frank; Xu, Zhiheng; Zhou, Zheng; Qin, Ning; Wang, Yue Fen; Ren, Shuang; Qiu, Hongxia; Zhang, Yuxia; Zhang, Yi; Wu, Xiaoyue; Sun, Kai; Zhong, Sheng; Xie, Jianxun; Ottaviani, Giorgio; Zhou, Yuan; Zhu, Lina; Tian, Xiaojun; Shi, Liping; Shen, Fang; Mao, Yi; Zhou, Xue; Gao, Lu; Young, John A. T.; Wu, Jim Zhen; Yang, Guang; Mayweg, Alexander V.; Shen, Hong C.; Tang, Guozhi; Zhu, Wei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3352-3371</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Described herein are the discovery and structure-activity relationship (SAR) studies of the third-generation 4-H heteroaryldihydropyrimidines (4-H HAPs) (I) featuring the introduction of a C6 carboxyl group as novel HBV capsid inhibitors.  This new series of 4-H HAPs showed improved anti-HBV activity and better drug-like properties compared to the first- and second-generation 4-H HAPs.  X-ray crystallog. study of analog 12 (HAP_R01) with Cp149 Y132A mutant hexamer clearly elucidated the role of C6 carboxyl group played for the increased binding affinity, which formed strong hydrogen bonding interactions with capsid protein and coordinated waters.  The representative analog 10 (HAP_R10) was extensively characterized in vitro (ADMET) and in vivo (mouse PK and PD) and subsequently selected for further development as oral anti-HBV infection agent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolNkEF9n_vqrVg90H21EOLACvtfcHk0ljkfyAGKKxI9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkvFWjtbw%253D&md5=6c2a89c594f2c735855f64aca2240837</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00083&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00083%26sid%3Dliteratum%253Aachs%26aulast%3DQiu%26aufirst%3DZ.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DZhou%26aufirst%3DM.%26aulast%3DGuo%26aufirst%3DL.%26aulast%3DKocer%26aufirst%3DB.%26aulast%3DWu%26aufirst%3DG.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DShi%26aufirst%3DH.%26aulast%3DKou%26aufirst%3DB.%26aulast%3DHu%26aufirst%3DT.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DM.%26aulast%3DYan%26aufirst%3DS.%2BF.%26aulast%3DXu%26aufirst%3DZ.%26aulast%3DZhou%26aufirst%3DZ.%26aulast%3DQin%26aufirst%3DN.%26aulast%3DWang%26aufirst%3DY.%2BF.%26aulast%3DRen%26aufirst%3DS.%26aulast%3DQiu%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DSun%26aufirst%3DK.%26aulast%3DZhong%26aufirst%3DS.%26aulast%3DXie%26aufirst%3DJ.%26aulast%3DOttaviani%26aufirst%3DG.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DZhu%26aufirst%3DL.%26aulast%3DTian%26aufirst%3DX.%26aulast%3DShi%26aufirst%3DL.%26aulast%3DShen%26aufirst%3DF.%26aulast%3DMao%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DX.%26aulast%3DGao%26aufirst%3DL.%26aulast%3DYoung%26aufirst%3DJ.%2BA.%2BT.%26aulast%3DWu%26aufirst%3DJ.%2BZ.%26aulast%3DYang%26aufirst%3DG.%26aulast%3DMayweg%26aufirst%3DA.%2BV.%26aulast%3DShen%26aufirst%3DH.%2BC.%26aulast%3DTang%26aufirst%3DG.%26aulast%3DZhu%26aufirst%3DW.%26atitle%3DDiscovery%2520and%2520pre-clinical%2520characterization%2520of%2520third-generation%25204-H%2520heteroaryldihydropyrimidine%2520%2528HAP%2529%2520analogues%2520as%2520hepatitis%2520B%2520virus%2520%2528HBV%2529%2520capsid%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D3352%26epage%3D3371%26doi%3D10.1021%2Facs.jmedchem.7b00083" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, S. F.</span></span> <span> </span><span class="NLM_article-title">Drugs behave as substrates inhibitors and inducers of human cytochrome P450 3A4</span>. <i>Curr. Drug Metab.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">310</span>– <span class="NLM_lpage">322</span>, <span class="refDoi"> DOI: 10.2174/138920008784220664</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01421&amp;key=10.2174%2F138920008784220664" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01421&amp;key=18473749" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01421&amp;key=1%3ACAS%3A528%3ADC%252BD1cXls1aitLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2008&pages=310-322&author=S.+F.+Zhou&title=Drugs+behave+as+substrates+inhibitors+and+inducers+of+human+cytochrome+P450+3A4&doi=10.2174%2F138920008784220664"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Drugs behave as substrates, inhibitors and inducers of human cytochrome P450 3A4</span></div><div class="casAuthors">Zhou, Shu-Feng</div><div class="citationInfo"><span class="NLM_cas:title">Current Drug Metabolism</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">310-322</span>CODEN:
                <span class="NLM_cas:coden">CDMUBU</span>;
        ISSN:<span class="NLM_cas:issn">1389-2002</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Human cytochrome P 450 (CYP) 3A4 is the most abundant hepatic and intestinal phase I enzyme that metabolizes approx. 50% marketed drugs.  The crystal structure of bound and unbound CYP3A4 has been recently constructed, and a small active site and a peripheral binding site are identified.  A recent study indicates that CYP3A4 undergoes dramatic conformational changes upon binding to ketoconazole or erythromycin with a differential but substantial (>80%) increase in the active site vol., providing a structural basis for ligand promiscuity of CYP3A4.  A no. of important drugs have been identified as substrates, inducers and/or inhibitors of CYP3A4.  The ability of drugs to act as inducers, inhibitors, or substrates for CYP3A is predictive of whether concurrent administration of these compds. with a known CYP3A substrate might lead to altered drug disposition, efficacy or toxicity.  The substrates of CYP3A4 considerably overlap with those of P-glycoprotein (P-gp).  To date, the identified clin. important CYP3A4 inhibitors mainly include macrolide antibiotics (e.g., clarithromycin, and erythromycin), anti-HIV agents (e.g., ritonavir and delavirdine), antidepressants (E.g. fluoxetine and fluvoxamine), calcium channel blockers (e.g. verapamil and diltiazem), steroids and their modulators (e.g., gestodene and mifepristone), and several herbal and dietary components.  Many of these drugs are also mechanism-based inhibitors of CYP3A4, which involves formation of reactive metabolites, binding to CYP3A4 and irreversible enzyme inactivation.  A small no. of drugs such as rifampin, phenytoin and ritonavir are identified as inducers of CYP3A4.  The orphan nuclear receptor, pregnane X receptor (PXR), have been found to play a crit. role in the induction of CYP3A4.  The inhibition or induction of CYP3A4 by drugs often causes unfavorable and long-lasting drug-drug interactions and probably fatal toxicity, depending on many factors assocd. with the enzyme, drugs and the patients.  The study of interactions of newly synthesized compds. with CYP3A4 has been incorporated into drug development and detection of possible CYP3A4 inhibitors and inducers during the early stages of drug development is crit. in preventing potential drug-drug interactions and side effects.  Clinicians are encouraged to have a sound knowledge on drugs that behave as substrates, inhibitors or inducers of CYP3A4, and take proper cautions and close monitoring for potential drug interactions when using drugs that are CYP3A4 inhibitors or inducers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrv7PWdiHlenbVg90H21EOLACvtfcHk0ljkfyAGKKxI9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXls1aitLY%253D&md5=df7e7825fd46cd35b8937245771cb972</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.2174%2F138920008784220664&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138920008784220664%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DS.%2BF.%26atitle%3DDrugs%2520behave%2520as%2520substrates%2520inhibitors%2520and%2520inducers%2520of%2520human%2520cytochrome%2520P450%25203A4%26jtitle%3DCurr.%2520Drug%2520Metab.%26date%3D2008%26volume%3D9%26spage%3D310%26epage%3D322%26doi%3D10.2174%2F138920008784220664" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Y.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balkovec, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Royo, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yabut, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evers, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, E. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartley, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mosley, R. T.</span></span> <span> </span><span class="NLM_article-title">Attenuating pregnane X receptor (PXR) activation: A molecular modelling approach</span>. <i>Xenobiotica</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">124</span>– <span class="NLM_lpage">138</span>, <span class="refDoi"> DOI: 10.1080/00498250601050412</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01421&amp;key=10.1080%2F00498250601050412" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01421&amp;key=17484516" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01421&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjt1ChsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2007&pages=124-138&author=Y.-D.+Gaoauthor=S.+H.+Olsonauthor=J.+M.+Balkovecauthor=Y.+Zhuauthor=I.+Royoauthor=J.+Yabutauthor=R.+Eversauthor=E.+Y.+Tanauthor=W.+Tangauthor=D.+P.+Hartleyauthor=R.+T.+Mosley&title=Attenuating+pregnane+X+receptor+%28PXR%29+activation%3A+A+molecular+modelling+approach&doi=10.1080%2F00498250601050412"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Attenuating pregnane X receptor (PXR) activation: a molecular modelling approach</span></div><div class="casAuthors">Gao, Y.-D.; Olson, S. H.; Balkovec, J. M.; Zhu, Y.; Royo, I.; Yabut, J.; Evers, R.; Tan, E. Y.; Tang, W.; Hartley, D. P.; Mosley, R. T.</div><div class="citationInfo"><span class="NLM_cas:title">Xenobiotica</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">124-138</span>CODEN:
                <span class="NLM_cas:coden">XENOBH</span>;
        ISSN:<span class="NLM_cas:issn">0049-8254</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">Recent studies have demonstrated that the pregnane X receptor (PXR) is a key regulator of cytochromes P 450 3A (e.g. CYP3A4 in human) gene expression.  As a result, activation of PXR may lead to CYP3A4 protein over-expression.  Because induction of CYP3A4 could result in clin. important drug-drug interactions, there has been a great interest in reducing the possibility of PXR activation by drug candidates in drug-discovery programs.  In order to provide structural insight for attenuating drug candidate-mediated PXR activation, we used a docking approach to study the structure-activity relationship for PXR activators.  Based on our docking models, it is proposed that introducing polar groups to the end of an activator should reduce its human PXR (hPXR) activity via destabilizing interactions in the hydrophobic areas of the PXR ligand-binding pocket.  A no. of analogs that incorporate these structural features then were designed and synthesized, and they exhibited significantly lower hPXR activation in a transactivation assay and decreased CYP3A4 induction in a human hepatocytes-based assay.  In addn., an example in which attenuating hPXR activation was achieved by sterically destabilizing the helixes 11 and 12 of the receptor is presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBZSC0secQoLVg90H21EOLACvtfcHk0lj0RrBNWqvjcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjt1ChsrY%253D&md5=400688510dd165a4a9632b6ba10bf267</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1080%2F00498250601050412&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F00498250601050412%26sid%3Dliteratum%253Aachs%26aulast%3DGao%26aufirst%3DY.-D.%26aulast%3DOlson%26aufirst%3DS.%2BH.%26aulast%3DBalkovec%26aufirst%3DJ.%2BM.%26aulast%3DZhu%26aufirst%3DY.%26aulast%3DRoyo%26aufirst%3DI.%26aulast%3DYabut%26aufirst%3DJ.%26aulast%3DEvers%26aufirst%3DR.%26aulast%3DTan%26aufirst%3DE.%2BY.%26aulast%3DTang%26aufirst%3DW.%26aulast%3DHartley%26aufirst%3DD.%2BP.%26aulast%3DMosley%26aufirst%3DR.%2BT.%26atitle%3DAttenuating%2520pregnane%2520X%2520receptor%2520%2528PXR%2529%2520activation%253A%2520A%2520molecular%2520modelling%2520approach%26jtitle%3DXenobiotica%26date%3D2007%26volume%3D37%26spage%3D124%26epage%3D138%26doi%3D10.1080%2F00498250601050412" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sinz, M. W.</span></span> <span> </span><span class="NLM_article-title">Avoiding PXR and CAR activation and CYP3A4 enzyme induction</span>. <i>Top. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">159</span>– <span class="NLM_lpage">190</span>, <span class="refDoi"> DOI: 10.1007/7355_2013_24</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01421&amp;key=10.1007%2F7355_2013_24" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=159-190&author=M.+W.+Sinz&title=Avoiding+PXR+and+CAR+activation+and+CYP3A4+enzyme+induction&doi=10.1007%2F7355_2013_24"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1007%2F7355_2013_24&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F7355_2013_24%26sid%3Dliteratum%253Aachs%26aulast%3DSinz%26aufirst%3DM.%2BW.%26atitle%3DAvoiding%2520PXR%2520and%2520CAR%2520activation%2520and%2520CYP3A4%2520enzyme%2520induction%26jtitle%3DTop.%2520Med.%2520Chem.%26date%3D2013%26volume%3D9%26spage%3D159%26epage%3D190%26doi%3D10.1007%2F7355_2013_24" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Groom, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bruno, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lightfoot, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, S. C.</span></span> <span> </span><span class="NLM_article-title">The Cambridgestructural database</span>. <i>Acta Crystallogr., Sect. B: Struct. Sci., Cryst. Eng. Mater.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>B72</i></span>,  <span class="NLM_fpage">171</span>– <span class="NLM_lpage">179</span>, <span class="refDoi"> DOI: 10.1107/S2052520616003954</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01421&amp;key=10.1107%2FS2052520616003954" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=B72&publication_year=2016&pages=171-179&author=C.+R.+Groomauthor=I.+J.+Brunoauthor=M.+P.+Lightfootauthor=S.+C.+Ward&title=The+Cambridgestructural+database&doi=10.1107%2FS2052520616003954"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1107%2FS2052520616003954&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS2052520616003954%26sid%3Dliteratum%253Aachs%26aulast%3DGroom%26aufirst%3DC.%2BR.%26aulast%3DBruno%26aufirst%3DI.%2BJ.%26aulast%3DLightfoot%26aufirst%3DM.%2BP.%26aulast%3DWard%26aufirst%3DS.%2BC.%26atitle%3DThe%2520Cambridgestructural%2520database%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520B%253A%2520Struct.%2520Sci.%252C%2520Cryst.%2520Eng.%2520Mater.%26date%3D2016%26volume%3DB72%26spage%3D171%26epage%3D179%26doi%3D10.1107%2FS2052520616003954" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span>In Vitro
Metabolism and Transporter Mediated Drug-Drug Interaction Studies
Guidance for Industry. <a href="https://www.fda.gov/downloads/Drugs/Guidances/UCM581965.pdf" class="extLink">https://www.fda.gov/downloads/Drugs/Guidances/UCM581965.pdf</a> (accessed Apr 16, <span class="NLM_year">2019</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=In+Vitro%0AMetabolism+and+Transporter+Mediated+Drug-Drug+Interaction+Studies%0AGuidance+for+Industry.+https%3A%2F%2Fwww.fda.gov%2Fdownloads%2FDrugs%2FGuidances%2FUCM581965.pdf+%28accessed+Apr+16%2C+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nagai, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konno, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Satsukawa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamashita, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshinari, K.</span></span> <span> </span><span class="NLM_article-title">Establishment of in silico prediction models for CYP3A4 and CYP2B6 induction in human hepatocytes by multiple regressionanalysis using azole compounds</span>. <i>Drug Metab. Dispos.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">1390</span>– <span class="NLM_lpage">1398</span>, <span class="refDoi"> DOI: 10.1124/dmd.115.068619</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01421&amp;key=10.1124%2Fdmd.115.068619" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01421&amp;key=27208383" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01421&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVajtLfP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=1390-1398&author=M.+Nagaiauthor=Y.+Konnoauthor=M.+Satsukawaauthor=S.+Yamashitaauthor=K.+Yoshinari&title=Establishment+of+in+silico+prediction+models+for+CYP3A4+and+CYP2B6+induction+in+human+hepatocytes+by+multiple+regressionanalysis+using+azole+compounds&doi=10.1124%2Fdmd.115.068619"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Establishment of in silico prediction models for CYP3A4 and CYP2B6 induction in human hepatocytes by multiple regression analysis using azole compounds</span></div><div class="casAuthors">Nagai, Mika; Konno, Yoshihiro; Satsukawa, Masahiro; Yamashita, Shinji; Yoshinari, Kouichi</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism & Disposition</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1390-1398</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">1521-009X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Drug-drug interactions (DDIs) via cytochrome P 450 (P 450) induction are one clin. problem leading to increased risk of adverse effects and the need for dosage adjustments and addnl. therapeutic monitoring.  In silico models for predicting P 450 induction are useful for avoiding DDI risk.  In this study, we have established regression models for CYP3A4 and CYP2B6 induction in human hepatocytes using several physicochem. parameters for a set of azole compds. with different P 450 induction as characteristics as model compds.  To obtain a well-correlated regression model, the compds. for CYP3A4 or CYP2B6 induction were independently selected from the tested azole compds. using principal component anal. with fold-induction data.  Both of the multiple linear regression models obtained for CYP3A4 and CYP2B6 induction are represented by different sets of physicochem. parameters.  The adjusted coeffs. of detn. for these models were of 0.8 and 0.9, resp.  The fold-induction of the validation compds., another set of 12 azole-contg. compds., were predicted within twofold limits for both CYP3A4 and CYP2B6.  The concordance for the prediction of CYP3A4 induction was 87% with another validation set, 23 marketed drugs.  However, the prediction of CYP2B6 induction tended to be overestimated for these marketed drugs.  The regression models show that lipophilicity mostly contributes to CYP3A4 induction, whereas not only the lipophilicity but also the mol. polarity is important for CYP2B6 induction.  Our regression models, esp. that for CYP3A4 induction, might provide useful methods to avoid potent CYP3A4 or CYP2B6 inducers during the lead optimization stage without performing induction assays in human hepatocytes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJ-HLYUJTykbVg90H21EOLACvtfcHk0lgdL46ZH3hu9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVajtLfP&md5=cb2a43dde2f98fc2d4bd20f966f89c3e</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1124%2Fdmd.115.068619&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.115.068619%26sid%3Dliteratum%253Aachs%26aulast%3DNagai%26aufirst%3DM.%26aulast%3DKonno%26aufirst%3DY.%26aulast%3DSatsukawa%26aufirst%3DM.%26aulast%3DYamashita%26aufirst%3DS.%26aulast%3DYoshinari%26aufirst%3DK.%26atitle%3DEstablishment%2520of%2520in%2520silico%2520prediction%2520models%2520for%2520CYP3A4%2520and%2520CYP2B6%2520induction%2520in%2520human%2520hepatocytes%2520by%2520multiple%2520regressionanalysis%2520using%2520azole%2520compounds%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2016%26volume%3D44%26spage%3D1390%26epage%3D1398%26doi%3D10.1124%2Fdmd.115.068619" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Althage, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chisari, F. V.</span></span> <span> </span><span class="NLM_article-title">Hydrodynamic injection of viral DNA: a mouse model of acute hepatitis B virus infection</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>99</i></span>,  <span class="NLM_fpage">13825</span>– <span class="NLM_lpage">13830</span>, <span class="refDoi"> DOI: 10.1073/pnas.202398599</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01421&amp;key=10.1073%2Fpnas.202398599" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01421&amp;key=12374864" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01421&amp;key=1%3ACAS%3A528%3ADC%252BD38XotVKmtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2002&pages=13825-13830&author=P.+L.+Yangauthor=A.+Althageauthor=J.+Chungauthor=F.+V.+Chisari&title=Hydrodynamic+injection+of+viral+DNA%3A+a+mouse+model+of+acute+hepatitis+B+virus+infection&doi=10.1073%2Fpnas.202398599"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Hydrodynamic injection of viral DNA: a mouse model of acute hepatitis B virus infection</span></div><div class="casAuthors">Yang, Priscilla L.; Althage, Alana; Chung, Josan; Chisari, Francis V.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">99</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">13825-13830</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Hepatitis B virus (HBV) is a prototype for liver-specific pathogens in which the failure of the immune system to mount an effective response leads to chronic infection.  The authors' understanding of the immune response to HBV is incomplete, largely due to the narrow host restriction of this pathogen and the limitations of existing exptl. models.  The authors have developed a murine model for studying human HBV replication, immunogenicity, and control.  After transfection of hepatocytes in vivo with a replication-competent, over-length, linear HBV genome, viral antigens and replicative intermediates were synthesized and virus was secreted into the blood.  Viral antigens disappeared from the blood as early as 7 days after transfection, coincident with the appearance of antiviral antibodies.  HBV transcripts and replicative intermediates disappeared from the liver by day 15, after the appearance of antiviral CD8 + T cells.  In contrast, the virus persisted for at least 81 days after transfection of NOD/Scid mice, which lack functional T cells, B cells, and natural killer (NK) cells.  Thus, the outcome of hydrodynamic transfection of HBV depends on the host immune response, as it is during a natural infection.  The methods the authors describe will allow the examn. of viral dynamics in a tightly controlled in vivo system, the application of mutagenesis methods to the study of the HBV life cycle in vivo, and the dissection of the immune response to HBV using genetically modified mice whose immunoregulatory and immune effector functions have been deleted or overexpressed.  In addn., this methodol. represents a prototype for the study of other known and to-be-discovered liver-specific pathogens.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo98nNA_w5YxrVg90H21EOLACvtfcHk0lgdL46ZH3hu9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XotVKmtbw%253D&md5=0c0f40c386fe47f6c57c413fccdaf3e5</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1073%2Fpnas.202398599&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.202398599%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DP.%2BL.%26aulast%3DAlthage%26aufirst%3DA.%26aulast%3DChung%26aufirst%3DJ.%26aulast%3DChisari%26aufirst%3DF.%2BV.%26atitle%3DHydrodynamic%2520injection%2520of%2520viral%2520DNA%253A%2520a%2520mouse%2520model%2520of%2520acute%2520hepatitis%2520B%2520virus%2520infection%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2002%26volume%3D99%26spage%3D13825%26epage%3D13830%26doi%3D10.1073%2Fpnas.202398599" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sells, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Acs, G.</span></span> <span> </span><span class="NLM_article-title">Production of hepatitis B virus particles in HepG2 cells transfected with cloned hepatitis B virus DNA</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">1987</span>,  <span class="NLM_volume"><i>84</i></span>,  <span class="NLM_fpage">1005</span>– <span class="NLM_lpage">1009</span>, <span class="refDoi"> DOI: 10.1073/pnas.84.4.1005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01421&amp;key=10.1073%2Fpnas.84.4.1005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01421&amp;key=3029758" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01421&amp;key=1%3ACAS%3A528%3ADyaL2sXhvV2qt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=84&publication_year=1987&pages=1005-1009&author=M.+A.+Sellsauthor=M.+L.+Chenauthor=G.+Acs&title=Production+of+hepatitis+B+virus+particles+in+HepG2+cells+transfected+with+cloned+hepatitis+B+virus+DNA&doi=10.1073%2Fpnas.84.4.1005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Production of hepatitis B virus particles in Hep G2 cells transfected with cloned hepatitis B virus DNA</span></div><div class="casAuthors">Sells, Mary Ann; Chen, Mei Ling; Acs, George</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1987</span>),
    <span class="NLM_cas:volume">84</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1005-9</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    </div><div class="casAbstract">The hepatoblastoma cell line Hep G2 was transfected with a plasmid carrying the gene that confers resistance to G418 and four 5'-3' tandem copies of the hepatitis B virus (HBV) genome positioned such that two dimers of the genomic DNA are 3'-3' with respect to one another.  Cells of one clone that grew in the presence of G418 produce high levels of hepatitis B e antigen and of hepatitis B surface antigen.  HBV DNA is carried by these cells as chromosomally integrated sequences and episomally as relaxed circular, covalently closed, and incomplete copies of the HBV genome.  Viral DNA was detected also in conditioned growth medium at the buoyant densities characteristic for infectious Dane and immature core particles.  Finally, HBV-specific components morphol. identical to the 22-nm spherical and filamentous hepatitis B surface antigen particles as well as 42-nm Dane particles were visualized by immunoelectron microscopic anal.  Therefore, the Hep G2 cell line can support the assembly and secretion not only of several of the replicative intermediates of HBV DNA but also of Dane-like particles.  This in vitro system can now be used to study the life cycle of HBV and the reaction of immunocompetent cells with cells carrying HBV.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGogbjqX6YU-W7Vg90H21EOLACvtfcHk0lgdL46ZH3hu9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2sXhvV2qt7c%253D&md5=27744d6955d4701586243f651c79ab1e</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1073%2Fpnas.84.4.1005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.84.4.1005%26sid%3Dliteratum%253Aachs%26aulast%3DSells%26aufirst%3DM.%2BA.%26aulast%3DChen%26aufirst%3DM.%2BL.%26aulast%3DAcs%26aufirst%3DG.%26atitle%3DProduction%2520of%2520hepatitis%2520B%2520virus%2520particles%2520in%2520HepG2%2520cells%2520transfected%2520with%2520cloned%2520hepatitis%2520B%2520virus%2520DNA%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1987%26volume%3D84%26spage%3D1005%26epage%3D1009%26doi%3D10.1073%2Fpnas.84.4.1005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuconati, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Block, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, J.-T.</span></span> <span> </span><span class="NLM_article-title">Characterization of the intracellular deproteinized relaxed circular DNA of hepatitis B virus: an intermediate of covalently closed circular DNA formation</span>. <i>J. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>81</i></span>,  <span class="NLM_fpage">12472</span>– <span class="NLM_lpage">12484</span>, <span class="refDoi"> DOI: 10.1128/JVI.01123-07</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01421&amp;key=10.1128%2FJVI.01123-07" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01421&amp;key=17804499" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01421&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlKku7zJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=2007&pages=12472-12484&author=H.+Guoauthor=D.+Jiangauthor=T.+Zhouauthor=A.+Cuconatiauthor=T.+M.+Blockauthor=J.-T.+Guo&title=Characterization+of+the+intracellular+deproteinized+relaxed+circular+DNA+of+hepatitis+B+virus%3A+an+intermediate+of+covalently+closed+circular+DNA+formation&doi=10.1128%2FJVI.01123-07"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of the intracellular deproteinized relaxed circular DNA of hepatitis B virus: an intermediate of covalently closed circular DNA formation</span></div><div class="casAuthors">Guo, Haitao; Jiang, Dong; Zhou, Tianlun; Cuconati, Andrea; Block, Timothy M.; Guo, Ju-Tao</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">81</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">12472-12484</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Covalently closed circular DNA (cccDNA) of hepatitis B virus (HBV) is formed by conversion of capsid-assocd. relaxed circular DNA (rcDNA) via unknown mechanisms and exists in the nucleus of the infected hepatocyte as a minichromosome that serves as the transcription template for viral RNAs.  To study the mol. pathway of cccDNA formation and its regulation by viral and cellular factors, we have established a cell line that supports the replication of an envelope protein-deficient HBV genome in a tetracycline-inducible manner.  Following induction of HBV replication, the cells accumulate higher levels of cccDNA as well as larger amts. of deproteinized rcDNA (DP-rcDNA) than cells that replicate wild-type HBV genomes.  These results indicate that HBV envelope proteins neg. regulate cccDNA formation, and conversion of DP-rcDNA into cccDNA is a rate-limiting step of cccDNA formation in HepG2 cells.  Detailed analyses reveal the following: (i) DP-rcDNA exists in both cytoplasm and nucleus; (ii) while nuclear DP-rcDNA is sensitive to DNase I digestion, a small fraction of cytoplasmic DP-rcDNA is DNase I resistant; (iii) both DNase I-sensitive and -resistant cytoplasmic DP-rcDNAs cosediment with capsids and can be immunopptd. with HBV core antibody; and (iv) a primer extension assay maps the 5' end of the minus strand of DP-rcDNA at the authentic end of virion rcDNA.  Hence, our results favor a hypothesis that the removal of viral polymerase protein covalently linked to the 5' end of the minus-strand DNA occurs inside the capsid in the cytoplasm and most possibly via a reaction that cleaves the phosphodiester bond between the tyrosine of the polymerase and the 5' phosphoryl group of minus-strand DNA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGot_WiB1hJ4zbVg90H21EOLACvtfcHk0lhozqIVCHstyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlKku7zJ&md5=91d6a9284b406680c85be2e35048d4ae</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1128%2FJVI.01123-07&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.01123-07%26sid%3Dliteratum%253Aachs%26aulast%3DGuo%26aufirst%3DH.%26aulast%3DJiang%26aufirst%3DD.%26aulast%3DZhou%26aufirst%3DT.%26aulast%3DCuconati%26aufirst%3DA.%26aulast%3DBlock%26aufirst%3DT.%2BM.%26aulast%3DGuo%26aufirst%3DJ.-T.%26atitle%3DCharacterization%2520of%2520the%2520intracellular%2520deproteinized%2520relaxed%2520circular%2520DNA%2520of%2520hepatitis%2520B%2520virus%253A%2520an%2520intermediate%2520of%2520covalently%2520closed%2520circular%2520DNA%2520formation%26jtitle%3DJ.%2520Virol.%26date%3D2007%26volume%3D81%26spage%3D12472%26epage%3D12484%26doi%3D10.1128%2FJVI.01123-07" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5WRE" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5WRE','PDB','5WRE'); return false;">PDB: 5WRE</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i45"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01421">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_44252"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.9b01421">10.1021/acs.jmedchem.9b01421</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">X-ray structure of <b>(<i>S</i>)-19a</b>; <sup>1</sup>H NMR and <sup>13</sup>C NMR spectra of <b>19</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01421/suppl_file/jm9b01421_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and some data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01421/suppl_file/jm9b01421_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01421/suppl_file/jm9b01421_si_001.pdf">jm9b01421_si_001.pdf (391.8 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01421/suppl_file/jm9b01421_si_002.csv">jm9b01421_si_002.csv (1.63 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b01421&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b01421%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2019.62.issue-22" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b01421" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                2MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679974e6fd183cf4","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
