
<html lang="en"     class="pb-page"  data-request-id="01ca0dfa-4fc8-44ab-878b-601fa0521d52"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";issue:issue:10.1021/jmcmar.2020.63.issue-13;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;article:article:10.1021/acs.jmedchem.0c00456"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Novel Pyrrolopyridone Bromodomain and Extra-Terminal Motif (BET) Inhibitors Effective in Endocrine-Resistant ER+ Breast Cancer with Acquired Resistance to Fulvestrant and Palbociclib" /></meta><meta name="dc.Creator" content="Yangfeng  Li" /></meta><meta name="dc.Creator" content="Jiong  Zhao" /></meta><meta name="dc.Creator" content="Lauren M.  Gutgesell" /></meta><meta name="dc.Creator" content="Zhengnan  Shen" /></meta><meta name="dc.Creator" content="Kiira  Ratia" /></meta><meta name="dc.Creator" content="Katherine  Dye" /></meta><meta name="dc.Creator" content="Oleksii  Dubrovskyi" /></meta><meta name="dc.Creator" content="Huiping  Zhao" /></meta><meta name="dc.Creator" content="Fei  Huang" /></meta><meta name="dc.Creator" content="Debra A.  Tonetti" /></meta><meta name="dc.Creator" content="Gregory R. J.  Thatcher" /></meta><meta name="dc.Creator" content="Rui  Xiong" /></meta><meta name="dc.Description" content="Acquired resistance to fulvestrant and palbociclib is a new challenge to treatment of estrogen receptor positive (ER+) breast cancer. ER is expressed in most resistance settings; thus, bromodomain ..." /></meta><meta name="Description" content="Acquired resistance to fulvestrant and palbociclib is a new challenge to treatment of estrogen receptor positive (ER+) breast cancer. ER is expressed in most resistance settings; thus, bromodomain ..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="May 26, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c00456" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2020 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00456" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c00456" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00456" /></link>
        
    
    

<title>Novel Pyrrolopyridone Bromodomain and Extra-Terminal Motif (BET) Inhibitors Effective in Endocrine-Resistant ER+ Breast Cancer with Acquired Resistance to Fulvestrant and Palbociclib | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c00456" /></meta><meta property="og:title" content="Novel Pyrrolopyridone Bromodomain and Extra-Terminal Motif (BET) Inhibitors Effective in Endocrine-Resistant ER+ Breast Cancer with Acquired Resistance to Fulvestrant and Palbociclib" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00456/20200701/images/large/jm0c00456_0007.jpeg" /></meta><meta property="og:description" content="Acquired resistance to fulvestrant and palbociclib is a new challenge to treatment of estrogen receptor positive (ER+) breast cancer. ER is expressed in most resistance settings; thus, bromodomain and extra-terminal protein inhibitors (BETi) that target BET-amplified ER-mediated transcription have therapeutic potential. Novel pyrrolopyridone BETi leveraged novel interactions with L92/L94 confirmed by a cocrystal structure of 27 with BRD4. Optimization of BETi using growth inhibition in fulvestrant-resistant (MCF-7:CFR) cells was confirmed in endocrine-resistant, palbociclib-resistant, and ESR1 mutant cell lines. 27 was more potent in MCF-7:CFR cells than six BET inhibitors in clinical trials. Transcriptomic analysis differentiated 27 from the benchmark BETi, JQ-1, showing downregulation of oncogenes and upregulation of tumor suppressors and apoptosis. The therapeutic approach was validated by oral administration of 27 in orthotopic xenografts of endocrine-resistant breast cancer in monotherapy and in combination with fulvestrant. Importantly, at an equivalent dose in rats, thrombocytopenia was mitigated." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c00456"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00456">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c00456&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c00456&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c00456&amp;href=/doi/10.1021/acs.jmedchem.0c00456" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 13</span><span class="cit-fg-pageRange">, 7186-7210</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/13" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c00450" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c00463" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Novel Pyrrolopyridone Bromodomain and Extra-Terminal Motif (BET) Inhibitors Effective in Endocrine-Resistant ER+ Breast Cancer with Acquired Resistance to Fulvestrant and Palbociclib</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Yangfeng Li</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yangfeng Li</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">UICentre (Drug Discovery @ UIC), University of Illinois at Chicago, 833 S. Wood Street, Chicago, Illinois 60612, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yangfeng++Li">Yangfeng Li</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jiong Zhao</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jiong Zhao</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmaceutical Sciences, University of Illinois College of Pharmacy, University of Illinois at Chicago, 833 S. Wood Street, Chicago, Illinois 60612, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jiong++Zhao">Jiong Zhao</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Lauren M. Gutgesell</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lauren M. Gutgesell</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmaceutical Sciences, University of Illinois College of Pharmacy, University of Illinois at Chicago, 833 S. Wood Street, Chicago, Illinois 60612, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lauren+M.++Gutgesell">Lauren M. Gutgesell</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-7645-9227" title="Orcid link">http://orcid.org/0000-0002-7645-9227</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Zhengnan Shen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Zhengnan Shen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">UICentre (Drug Discovery @ UIC), University of Illinois at Chicago, 833 S. Wood Street, Chicago, Illinois 60612, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Zhengnan++Shen">Zhengnan Shen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Kiira Ratia</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kiira Ratia</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">UICentre (Drug Discovery @ UIC), University of Illinois at Chicago, 833 S. Wood Street, Chicago, Illinois 60612, United States</div><div class="loa-info-affiliations-info">Department of Pharmaceutical Sciences, University of Illinois College of Pharmacy, University of Illinois at Chicago, 833 S. Wood Street, Chicago, Illinois 60612, United States</div><div class="loa-info-affiliations-info">Research Resources Center, University of Illinois at Chicago, Chicago, Illinois 60612, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kiira++Ratia">Kiira Ratia</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Katherine Dye</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Katherine Dye</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">UICentre (Drug Discovery @ UIC), University of Illinois at Chicago, 833 S. Wood Street, Chicago, Illinois 60612, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Katherine++Dye">Katherine Dye</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Oleksii Dubrovskyi</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Oleksii Dubrovskyi</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">UICentre (Drug Discovery @ UIC), University of Illinois at Chicago, 833 S. Wood Street, Chicago, Illinois 60612, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Oleksii++Dubrovskyi">Oleksii Dubrovskyi</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Huiping Zhao</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Huiping Zhao</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmaceutical Sciences, University of Illinois College of Pharmacy, University of Illinois at Chicago, 833 S. Wood Street, Chicago, Illinois 60612, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Huiping++Zhao">Huiping Zhao</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Fei Huang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Fei Huang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">UICentre (Drug Discovery @ UIC), University of Illinois at Chicago, 833 S. Wood Street, Chicago, Illinois 60612, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Fei++Huang">Fei Huang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Debra A. Tonetti</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Debra A. Tonetti</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmaceutical Sciences, University of Illinois College of Pharmacy, University of Illinois at Chicago, 833 S. Wood Street, Chicago, Illinois 60612, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Debra+A.++Tonetti">Debra A. Tonetti</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Gregory R. J. Thatcher</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Gregory R. J. Thatcher</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmaceutical Sciences, University of Illinois College of Pharmacy, University of Illinois at Chicago, 833 S. Wood Street, Chicago, Illinois 60612, United States</div><div class="loa-info-affiliations-info">UICentre (Drug Discovery @ UIC), University of Illinois at Chicago, 833 S. Wood Street, Chicago, Illinois 60612, United States</div></div><span class="conrtib-corresp"><strong>*</strong>G.R.J.T.: phone, 312-355-5282; email, <a href="/cdn-cgi/l/email-protection#a6d2cec7d2c5cec3d4e6d3cfc588c3c2d3"><span class="__cf_email__" data-cfemail="b0c4d8d1c4d3d8d5c2f0c5d9d39ed5d4c5">[email protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Gregory+R.+J.++Thatcher">Gregory R. J. Thatcher</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-7757-1739" title="Orcid link">http://orcid.org/0000-0002-7757-1739</a></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Rui Xiong</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Rui Xiong</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmaceutical Sciences, University of Illinois College of Pharmacy, University of Illinois at Chicago, 833 S. Wood Street, Chicago, Illinois 60612, United States</div><div class="loa-info-affiliations-info">UICentre (Drug Discovery @ UIC), University of Illinois at Chicago, 833 S. Wood Street, Chicago, Illinois 60612, United States</div></div><span class="conrtib-corresp"><strong>*</strong>R.X.: phone, 312-996-9710; email, <a href="/cdn-cgi/l/email-protection#3d4f455452535a0e7d48545e13585948"><span class="__cf_email__" data-cfemail="7d0f051412131a4e3d08141e53181908">[email protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Rui++Xiong">Rui Xiong</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6350-9037" title="Orcid link">http://orcid.org/0000-0002-6350-9037</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00456&amp;href=/doi/10.1021%2Facs.jmedchem.0c00456" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 13</span><span class="cit-pageRange">, 7186–7210</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">May 26, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>18 March 2020</li><li><span class="item_label"><b>Published</b> online</span>26 May 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 9 July 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c00456" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00456</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D7186%26pageCount%3D25%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DYangfeng%2BLi%252C%2BJiong%2BZhao%252C%2BLauren%2BM.%2BGutgesell%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D13%26contentID%3Dacs.jmedchem.0c00456%26title%3DNovel%2BPyrrolopyridone%2BBromodomain%2Band%2BExtra-Terminal%2BMotif%2B%2528BET%2529%2BInhibitors%2BEffective%2Bin%2BEndocrine-Resistant%2BER%252B%2BBreast%2BCancer%2Bwith%2BAcquired%2BResistance%2Bto%2BFulvestrant%2Band%2BPalbociclib%26numPages%3D25%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D7210%26publicationDate%3DJuly%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c00456"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1727</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">3</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c00456" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Novel Pyrrolopyridone Bromodomain and Extra-Terminal Motif (BET) Inhibitors Effective in Endocrine-Resistant ER+ Breast Cancer with Acquired Resistance to Fulvestrant and Palbociclib&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Yangfeng&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Jiong&quot;,&quot;last_name&quot;:&quot;Zhao&quot;},{&quot;first_name&quot;:&quot;Lauren&quot;,&quot;last_name&quot;:&quot;M. Gutgesell&quot;},{&quot;first_name&quot;:&quot;Zhengnan&quot;,&quot;last_name&quot;:&quot;Shen&quot;},{&quot;first_name&quot;:&quot;Kiira&quot;,&quot;last_name&quot;:&quot;Ratia&quot;},{&quot;first_name&quot;:&quot;Katherine&quot;,&quot;last_name&quot;:&quot;Dye&quot;},{&quot;first_name&quot;:&quot;Oleksii&quot;,&quot;last_name&quot;:&quot;Dubrovskyi&quot;},{&quot;first_name&quot;:&quot;Huiping&quot;,&quot;last_name&quot;:&quot;Zhao&quot;},{&quot;first_name&quot;:&quot;Fei&quot;,&quot;last_name&quot;:&quot;Huang&quot;},{&quot;first_name&quot;:&quot;Debra&quot;,&quot;last_name&quot;:&quot;A. Tonetti&quot;},{&quot;first_name&quot;:&quot;Gregory&quot;,&quot;last_name&quot;:&quot;R. J. Thatcher&quot;},{&quot;first_name&quot;:&quot;Rui&quot;,&quot;last_name&quot;:&quot;Xiong&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;05&quot;,&quot;day&quot;:&quot;26&quot;,&quot;issue&quot;:&quot;13&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;7186-7210&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c00456&quot;},&quot;abstract&quot;:&quot;Acquired resistance to fulvestrant and palbociclib is a new challenge to treatment of estrogen receptor positive (ER+) breast cancer. ER is expressed in most resistance settings; thus, bromodomain and extra-terminal protein inhibitors (BETi) that target BET-amplified ER-mediated transcription have therapeutic potential. Novel pyrrolopyridone BETi leveraged novel interactions with L92/L94 confirmed by a cocrystal structure of 27 with BRD4. Optimization of BETi using growth inhibition in fulvestrant-resistant (MCF-7:CFR) cells was confirmed in endocrine-resistant, palbociclib-resistant, and ESR1 mutant cell lines. 27 was more potent in MCF-7:CFR cells than six BET inhibitors in clinical trials. Transcriptomic analysis differentiated 27 from the benchmark BETi, JQ-1, showing downregulation of oncogenes and upregulation of tumor suppressors and apoptosis. The therapeutic approach was validated by oral administration of 27 in orthotopic xenografts of endocrine-resistant breast cancer in monotherapy and in combinat&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00456&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00456" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00456&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00456" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00456&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00456" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00456&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00456&amp;href=/doi/10.1021/acs.jmedchem.0c00456" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c00456" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c00456" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (6 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00456&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c00456%26sid%3Dliteratum%253Aachs%26pmid%3D32453591%26genre%3Darticle%26aulast%3DLi%26date%3D2020%26atitle%3DNovel%2BPyrrolopyridone%2BBromodomain%2Band%2BExtra-Terminal%2BMotif%2B%2528BET%2529%2BInhibitors%2BEffective%2Bin%2BEndocrine-Resistant%2BER%252B%2BBreast%2BCancer%2Bwith%2BAcquired%2BResistance%2Bto%2BFulvestrant%2Band%2BPalbociclib%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D13%26spage%3D7186%26epage%3D7210%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (3)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=290896" title="Cells">Cells</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=292258" title="Cancer">Cancer</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/13" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/jmcmar.2020.63.issue-13/20200709/jmcmar.2020.63.issue-13.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00456/20200701/images/medium/jm0c00456_0007.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00456/20200701/images/large/jm0c00456_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00456&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Acquired resistance to fulvestrant and palbociclib is a new challenge to treatment of estrogen receptor positive (ER+) breast cancer. ER is expressed in most resistance settings; thus, bromodomain and extra-terminal protein inhibitors (BETi) that target BET-amplified ER-mediated transcription have therapeutic potential. Novel pyrrolopyridone BETi leveraged novel interactions with L92/L94 confirmed by a cocrystal structure of <b>27</b> with BRD4. Optimization of BETi using growth inhibition in fulvestrant-resistant (MCF-7:CFR) cells was confirmed in endocrine-resistant, palbociclib-resistant, and <i>ESR1</i> mutant cell lines. <b>27</b> was more potent in MCF-7:CFR cells than six BET inhibitors in clinical trials. Transcriptomic analysis differentiated <b>27</b> from the benchmark BETi, JQ-1, showing downregulation of oncogenes and upregulation of tumor suppressors and apoptosis. The therapeutic approach was validated by oral administration of <b>27</b> in orthotopic xenografts of endocrine-resistant breast cancer in monotherapy and in combination with fulvestrant. Importantly, at an equivalent dose in rats, thrombocytopenia was mitigated.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_25364" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_25364" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Breast cancer is the second leading cause of cancer death in women worldwide.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> One out of every eight women in the U.S. will develop invasive breast cancer during her lifetime. Approximately 70% of all breast cancers are estrogen receptor positive (ER+), where dysregulated ER signaling fuels cancer growth.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> The mainstay of disease control is endocrine therapy using tamoxifen and aromatase inhibitors (AIs) (e.g., anastrazole, letrazole). Tumors with acquired resistance to tamoxifen and AIs maintain functional ER signaling,<a onclick="showRef(event, 'ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref3 ref4">(3,4)</a> and therefore ER remains a therapeutic target. Thus, the selective ER degrader (SERD),<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> fulvestrant, has become a key player in the therapeutic response to endocrine resistant, metastatic breast cancer. The standard-of-care in metastatic breast cancer has changed rapidly with the introduction of CDK4/6 inhibitors, such as palbociclib,<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> used in first line therapy in combination with fulvestrant or AIs.<a onclick="showRef(event, 'ref7 ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref7 ref8 ref9">(7−9)</a> However, the majority of patients with ER+ metastatic breast cancer will experience disease progression.</div><div class="NLM_p">The emerging clinical resistance to both fulvestrant and CDK4/6 inhibitors presents a new and immediate challenge for targeted drug discovery. Fulvestrant is effective in the endocrine resistant setting, that is, in ER+ breast cancer resistant to tamoxifen and AIs.<a onclick="showRef(event, 'ref10 ref11 ref12 ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref10 ref11 ref12 ref13 ref14">(10−14)</a> Therefore, the specific mechanisms of resistance to fulvestrant, though not fully defined, are clearly different from mechanisms that drive resistance to endocrine therapy. The optimization of targeted therapies for fulvestrant-resistant breast cancer must therefore be driven by activity in fulvestrant-resistant ER+ breast cancer cells. This work is the first example of iterative lead optimization driven by antiproliferative activity in fulvestrant and endocrine-resistant ER+ breast cancer cells. Moreover, activity was confirmed in an endocrine-resistant cell line stably resistant to the CDK4/6 inhibitor palbociclib.</div><div class="NLM_p">Bromodomain and extra-terminal domain (BET) family proteins are “readers” of epigenetic regulation that recognize acetylated lysines in histones H3 and H4.<a onclick="showRef(event, 'ref15 ref16 ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref15 ref16 ref17 ref18">(15−18)</a> The BET family comprises bromodomain-containing proteins, BRD2, BRD3, BRD4, and bromodomain testis-specific protein (BRDT), all featuring two tandem N-terminal bromodomains and an extra C-terminal domain (ET).<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> Each bromodomain comprises ∼110 amino acids that structurally consists of four α-helices (αZ, αA, αB, and αC) and two loops, connecting αZ and αA (ZA loop) and αB and αC (BC loop).<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> Early research on BET proteins, predominantly BRD4, revealed a transcriptional coactivator function in enhancer and superenhancer complexes,<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> facilitating transcription from master regulatory transcription factors, such as MYC, to RNA polymerase II (RNAPII).<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> BET proteins then trigger RNAPII release from the promoter,<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> driving transcriptional elongation and transcription of oncogenes. BRD4 fusion proteins were recognized to drive the aberrant growth of lung, head, and neck cancer.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a></div><div class="NLM_p">Recently, BRD4 was reported to be recruited to distal estrogen response elements (ERE) and to coordinate with ERα in a transcriptional complex that regulates gene expression in breast cancer.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> The first generation BET inhibitor, JQ1 (<b>1</b>; <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>),<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> was reported to suppress ER signaling and to inhibit estrogen-mediated breast cancer cell growth.<a onclick="showRef(event, 'ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref27 ref28">(27,28)</a> Specifically, BRD4 recruitment to the transcriptional complex formed on ERα binding, histone acetylation, and cohesin recruitment is necessary for elongation-associated phosphorylation of RNAPII (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A). Since ER is expressed and plays a ligand-independent role in supporting growth of endocrine-resistant breast tumors, these findings support the use of a BET inhibitor in endocrine and fulvestrant-resistant breast cancer.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00456/20200701/images/medium/jm0c00456_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00456/20200701/images/large/jm0c00456_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. BRD4 is a coregulator of ER signaling in breast cancer. (A) ER+ breast cancer is a transcriptionally driven cancer: E2-liganded ER localizes to ERE forming a transcriptional complex, including proteins such as MED1, P300, and FOXA1; BRD4 recruitment to this complex at acetylated histones and cohesin recruitment causes transcriptional elongation via P-TEFb and RNA polymerase II. Targeting the ER transcriptional complex through BRD4 provides an alternative approach either as monotherapy or in combination with SERDs to treat ER+ breast cancer. (B) Structures of benchmark BET inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00456/20200701/images/large/jm0c00456_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00456&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Small molecules with diverse bromodomain-targeting scaffolds<a onclick="showRef(event, 'ref29 ref30 ref31 ref32 ref33 ref34 ref35 ref36 ref37 ref38 ref39 ref40 ref41 ref42 ref43'); return false;" href="javascript:void(0);" class="ref ref29 ref30 ref31 ref32 ref33 ref34 ref35 ref36 ref37 ref38 ref39 ref40 ref41 ref42 ref43">(29−43)</a> have been reported over the past decade, since the discovery of <b>1</b>,<a onclick="showRef(event, 'ref26 ref44'); return false;" href="javascript:void(0);" class="ref ref26 ref44">(26,44)</a> including >20 BET inhibitors that entered clinical trials for various therapeutic applications such as advanced solid tumors and hematological malignancies (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B).<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> Benzodiazepine-based GSK-525762 (<b>2</b>),<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> pyridine-based ABBV-075 (<b>3</b>),<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> triazole-based BMS-986158 (<b>4</b>),<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> and bivalent AZD-5153 (<b>5</b>)<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> all feature an acetylated lysine mimic that forms a hydrogen bond with a conserved Asn140 in the BD1 domain of BRD4. Herein, we report a novel series of potent pyrrolopyridone BET inhibitors, leveraging a novel interaction in the BD1 domain, confirmed by cocrystal structures.</div><div class="NLM_p last">A unique iterative optimization strategy, driven by antiproliferative potency in a fulvestrant-resistant ER+ cell line (MCF-7:CFR), led to BET inhibitor <b>27</b>, which demonstrated superior potency to other clinical-stage BET ihhibitors in this cell line. <b>27</b> was also shown to be a potent inhibitor of growth of breast cancer cells resistant to tamoxifen, aromatase inhibitors, and palbociclib. This strategy was designed specifically to target emerging treatment-resistant ER+ breast cancers and was supported by transcriptomic analysis in MCF-7:CFR cells identifying downregulated oncogenes and upregulated tumor suppressor and proapoptotic genes. <b>27</b> was orally bioavailable and inhibited growth of endocrine-resistant tumors <i>in vivo</i>, without thrombocytopenia.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Design and Optimization</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_56363" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_56363" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Initial structure-based design of inhibitors was driven by literature crystal structures. The design strategy was validated by obtaining a new crystal structure of our best BET inhibitor bound to BRD4. Inhibitor optimization was driven by two assays: (1) growth inhibition of the MCF-7:CFR cell line and (2) measurement of binding to BRD4-BD1 using TR-FRET. The ER+ MCF-7:CFR cell line is resistant to endocrine therapy and resistant to treatment with fulvestrant, modeling the clinical resistance setting wherein ERα continues to drive tumor survival, but has acquired resistance to all ER-targeted clinical therapeutics. The MCF-7:CFR cell line was obtained by long-term exposure of the endocrine-resistant MCF-7:5C cell line to fulvestrant and provides a stable phenotype across multiple passages. This is a unique optimization strategy directed at addressing the emerging clinical resistance in metastatic breast cancer.</div><div class="NLM_p">A pyrrolopyridone scaffold was identified from virtual screening by Vidler et al.<a onclick="showRef(event, 'ref46 ref47'); return false;" href="javascript:void(0);" class="ref ref46 ref47">(46,47)</a> The pyrrolopyridone ring acts as an acetylated lysine mimic (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A), forming a bidentate hydrogen bond with Asn140 (BRD4-BD1). We postulated that the 3 position, amenable to functionalization, would permit optimization and drew inspiration from earlier studies: the sulfonamide group of <b>3</b><a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> and bromosporine (<b>8</b>)<a onclick="showRef(event, 'ref48 ref49'); return false;" href="javascript:void(0);" class="ref ref48 ref49">(48,49)</a> and the heteroaryl moiety of I-BET151 (<b>9</b>).<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> The sulfonamide group of <b>8</b> is proposed to interact with the ZA loop, whereas the heteroaryl moiety of <b>9</b> is proposed to bind to the hydrophobic WPF shelf, both important structural features of the BD1 binding site (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B). Compound <b>10</b>, combining these two key design elements in a “trans” arrangement, was anticipated to engage both the ZA loop and WPF shelf, increasing affinity and potency (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A,B). Superposition of the predicted docking pose of <b>10</b> with that of <b>9</b> from a cocrystal structure with BRD4-BD1 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ZYU">3ZYU</a>) showed that <b>10</b> captures most of the binding interactions of <b>9</b> with BRD4-BD1; moreover, <b>10</b> also forms an extra hydrogen bond with Asn140 that might enhance the binding affinity (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>C). Further designs were also tested in this preliminary round of SAR, including the regioisomers <b>12</b>–<b>14</b>, benzimidazoles <b>15</b> and <b>16</b>, and indazoles <b>17</b> and <b>18</b>.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00456/20200701/images/medium/jm0c00456_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00456/20200701/images/large/jm0c00456_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Design of pyrrolopyridone-based BET inhibitors. (A) Design concept inspired by earlier reported BET inhibitors. Pyrrolopyridone, <b>7</b>, identified from a virtual screen forms bidentate hydrogen-bonding interaction with Asn140, highlighted in green. The sulfonamide group of <b>8</b> interacts with the ZA loop via a putative hydrogen-bonding interaction, highlighted in blue. The heteroaryl moiety of <b>9</b> interacts with the hydrophobic WPF shelf, highlighted in yellow. Combining these design parameters leads to the design of <b>10</b> and <b>12</b> as novel BET inhibitors. (B) Predicted binding pose of compound <b>10</b> binding to BRD4-BD1 using Glide (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ZYU">3ZYU</a>). Key interacting residues from docking are highlighted and labeled. (C) Overlay of the predicted binding pose of <b>10</b> (blue) with <b>9</b> (beige) from PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ZYU">3ZYU</a>. (D) New cocrystal of compound <b>27</b> bound to BRD4-BD1 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6P05">6P05</a>), showing four hydrogen bonds with N140, D88, and K91 and one additional water-mediated hydrogen bond with K91. The “leucine clamp” (L92 and L94) is highlighted in green, and other key interacting residues are highlighted and labeled in red. (E) Overlay of the predicted binding pose of <b>10</b> (blue) with the experimentally confirmed binding pose of <b>27</b> (beige) clearly showing the second pyridine ring of <b>27</b> (boxed in green) that forms a hydrophobic interaction with L92 and L94.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00456/20200701/images/large/jm0c00456_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00456&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The design strategy for compound <b>10</b> was substantiated by TR-FRET assay of binding to BRD4-BD1 (IC<sub>50</sub> = 96 nM), while the positive control <b>1</b> showed slightly weaker binding affinity in this assay (IC<sub>50</sub> = 199 nM) (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). In contrast, the 7-pyrrolopyridone substituted indole <b>12</b>, the regioisomer of <b>10</b>, showed a dramatic loss of potency (IC<sub>50</sub> = 1.18 μM). In MCF-7:CFR cells, the high potency of <b>10</b> in inhibiting cell growth (IC<sub>50</sub> = 17 nM) was confirmed, comparing favorably with <b>1</b> (IC<sub>50</sub> = 90 nM) and in accord with TR-FRET binding data. A further attempt to optimize the phenyl ring in the 7-pyrrolopyridone substituted indole series yielded compounds <b>13</b> and <b>14</b>, which again lacked potency in both binding and growth inhibition assays (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Two further, alternative heterocycle designs were studied: the benzimidazole <b>15</b> exhibited a significant decrease in BRD4-BD1 binding by TR-FRET and reduced potency in MCF-7:CFR cells (IC<sub>50</sub> = 0.7 μM). The benzimidazole <b>16</b> suffered a further loss of potency, indicating a preference for a monosulfonamide in interacting with the ZA loop. Finally, <b>17</b> and <b>18</b> were designed to probe the effect of an indazole scaffold in comparison to the indoles, <b>10</b> and <b>11</b>, showing similar potency in biochemical and cell-based assays.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Biochemical and Cellular Potency of Compounds <b>10</b>–<b>18</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00456/20200701/images/medium/jm0c00456_0013.gif" alt="" id="GRAPHIC-d7e696-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00456/20200701/images/medium/jm0c00456_0014.gif" alt="" id="GRAPHIC-d7e697-autogenerated" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">TR-FRET assay BRD4(1) bromodomain IC<sub>50</sub> values are reported as the mean ± SD.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">Cell survival was normalized to DMSO control (100%) and JQ1 (3 μM) as 0% assessed in a 5-day cell proliferation assay with one time drug treatment. Data show the mean ± SEM from at least two cell passages (triplicates in each passage).</p></div><div class="footnote" id="t1fn3"><sup><sup>c</sup></sup><p class="last">LLE (LipE) = pIC<sub>50</sub> – cLogP; IC<sub>50</sub> of growth inhibition was used for calculations.</p></div></div><div></div></div><div class="NLM_p">Ligand-lipophilicity efficiency (LLE)<a onclick="showRef(event, 'ref51 ref52'); return false;" href="javascript:void(0);" class="ref ref51 ref52">(51,52)</a> for compounds in the first round of SAR was calculated using cell based potency to estimate drug-likeness (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>), with <b>10</b>, <b>17</b>, and <b>18</b> showing superior metrics. However, a preliminary pharmacokinetic study showed that <b>10</b> had oral bioavailability clearly superior to the indazoles (data not shown); therefore, 6-pyrrolopyridone substituted indole <b>10</b> was selected for further optimization.</div><div class="NLM_p">Compounds <b>19</b>–<b>22</b> were designed to alter the torsional angle between the phenyl ring and the indole moiety by derivatizing the phenyl ring, leading to comparable potency to <b>10</b> (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>), suggesting that the WPF region tolerates varied ring substituents. Efforts to improve potency and increase sp<sup>3</sup> count were made by replacement with nonaromatic substituents, namely, oxetane and difluorocyclobutane rings, as in compounds <b>23</b> and <b>24</b>, respectively. These changes were tolerated without significant loss of potency and with an improvement in the LLE metric. To overcome the potential metabolic liability of the benzylic position of <b>10</b>, we introduced a second phenyl group at this position in compound <b>25</b>, which gratifyingly led to improved potency relative to <b>10</b>. The high potency in inhibition of MCF7:CFR cell growth (IC<sub>50</sub> = 15 nM) was counterbalanced by a less favorable LLE.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Biochemical and Cellular Potency of Substituted Indoles</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00456/20200701/images/medium/jm0c00456_0015.gif" alt="" id="GRAPHIC-d7e793-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00456/20200701/images/medium/jm0c00456_0016.gif" alt="" id="GRAPHIC-d7e794-autogenerated" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">TR-FRET assay BRD4(1) bromodomain IC<sub>50</sub> values are reported as the mean ± SD.</p></div><div class="footnote" id="t2fn2"><sup><sup>b</sup></sup><p class="last">Cell survival was normalized to DMSO control (100%) and JQ1 (3 μM) as 0% assessed in a 5-day cell proliferation assay with one time drug treatment. Data show the mean ± SEM from at least two cell passages (triplicates in each passage).</p></div><div class="footnote" id="t2fn3"><sup><sup>c</sup></sup><p class="last">LLE (LipE) = pIC<sub>50</sub> – cLogP; IC<sub>50</sub> of growth inhibition was used for calculations.</p></div></div><div></div></div><div class="NLM_p">The gain in potency of <b>25</b> suggested the existence of extra stabilizing interactions with the second phenyl ring, which could be further optimized. The less favorable LLE also indicated the need to explore heterocycles in place of benzene rings. A significant improvement in biochemical and cell-based assay potency was achieved by replacing the diphenyl substituent with a bipyridyl analog and simultaneously converting the benzylic position into a quaternary carbon center in compounds <b>26</b>–<b>28</b>. The ethyl substitution at this position in compound <b>28</b> was inspired by molecular modeling indicating a small hydrophobic pocket; however, compound <b>28</b> was inferior to <b>26</b> and <b>27</b> in binding potency and LLE, suggesting a binding penalty in further modifying this site. An exemplar TR-FRET binding curve is shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A for compound <b>27</b>.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00456/20200701/images/medium/jm0c00456_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00456/20200701/images/large/jm0c00456_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Binding to BRD4-BD1 correlates with cellular antiproliferative potency: (A) BRD4-BD1 and BRD4-BD2 binding measured by TR-FRET for <b>1</b> compared to <b>27</b> (data normalized to DMSO as 100%); (B) BROMOscan of compound <b>27</b> against 40 bromodomain proteins; (C) correlation between BRD4-BD1 binding (TR-FRET) assay and potency for growth inhibition of MCF-7:CFR cells.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00456/20200701/images/large/jm0c00456_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00456&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">With compound <b>27</b> demonstrating a good combination of potency and LLE, we obtained a cocrystal structure of <b>27</b> in complex with BRD4-BD1 to better understand the structural basis of binding to BRD4 and the high potency in biochemical and cell growth inhibition assays (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>D). The structure confirmed our initial design hypotheses: (1) the pyrrolopyridone scaffold forms a bidentate hydrogen-bonding interaction with Asn140; (2) the sulfonamide moiety inserts into the ZA loop and its neighboring residues; (3) one pyridine ring interacts with the hydrophobic WPF shelf. However, less anticipated was the observed hydrogen-bonding network induced by the sulfonamide moiety both directly with Asp88 and Lys91 and via an additional water-mediated hydrogen bond with Lys91.</div><div class="NLM_p">In addition to the sulfonamide-induced hydrogen bonding network, we also observed that the second pyridine ring forms an edge-to-face interaction with a “leucine clamp” (Leu92 and Leu94) in the ZA loop (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>E), which we postulated to contribute to the high affinity and potency of compound <b>27</b>. A similar interaction was previously reported (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00456/suppl_file/jm0c00456_si_001.pdf" class="ext-link">Supporting Information Figure S2</a>).<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> In compound <b>27</b>, the sterically congested and conformationally restricted methinylpyridine likely contributes to the increased binding affinity. In summary, our new cocrystal structure showed a unique edge-to-face interaction with the “leucine clamp” that boosted the potency by an order of magnitude. This interaction could be incorporated into other scaffolds, serving as a generalizable design strategy toward next-generation BET inhibitors.</div><div class="NLM_p">The binding interactions of <b>27</b> at the BD1 site revealed a hydrogen-bond network with the ZA loop, involving Asp88 and Lys91. Interestingly, Lys91 is one of the key residues that differentiate the BD1 domain from the BD2 domain of BET proteins.<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> Therefore, these interactions provide an opportunity for further optimization for both potency and selectivity. We explored substitutions at the 4-position of the indole ring in the bipyridyl series. Replacing the sulfonamide with amides in <b>29</b> and <b>30</b> led to BET inhibitors with similar potency but an inferior LLE, whereas replacement with a 2-hydroxypropyl group resulted in 2- to 3-fold loss of potency in biochemical and cell-based assays (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). Most compounds, except <b>31</b>, showed comparable binding between BD1 and BD2 in TR-FRET assays, indicating a similar hydrogen bonding interaction with the ZA loop. Compound <b>31</b> displayed a 6-fold selectivity for BD1 versus BD2, likely resulting from a selective interaction with Lys91 and provided the foundation for BD1-selective inhibitors that will be reported in a separate paper. However, given the excellent potency and superior LLE, compound <b>27</b> was selected for further study.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Exploration of 4-Position of Indole-Based Analogs</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00456/20200701/images/medium/jm0c00456_0017.gif" alt="" id="GRAPHIC-d7e930-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00456/20200701/images/medium/jm0c00456_0018.gif" alt="" id="GRAPHIC-d7e931-autogenerated" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">TR-FRET assay BRD4(1) bromodomain IC<sub>50</sub> values are reported as the mean ± SD. CFR IC<sub>50</sub>: Cell survival was normalized to DMSO control (100%) and JQ1 (3 μM) as 0% assessed in a 5-day cell proliferation assay with one-time drug treatment; data show the mean ± SEM from at least two cell passages (triplicates in each passage).</p></div><div class="footnote" id="t3fn3"><sup><sup>b</sup></sup><p class="last">LLE (LipE) = pIC<sub>50</sub> – cLogP; IC<sub>50</sub> of growth inhibition was used for calculations.</p></div></div><div></div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i7">Mechanism and Efficacy in Fulvestrant Resistance</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_23501" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_23501" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The ER+ MCF-7:CFR cell line, resistant to tamoxifen, fulvestrant, and AIs, drove the iterative optimization of BET inhibitors leading to <b>27</b>. To test the dependence of growth inhibition of MCF-7:CFR cells on binding to BRD4 over other bromodomain proteins, the binding affinity for BRD4-BD1 from TR-FRET was plotted against antiproliferative potency in MCF-7:CFR cells (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C). The excellent correlation (<i>R</i><sup>2</sup> = 0.85), observed across the 23 BET inhibitors created and studied herein, strongly supports the hypothesis that growth inhibition in MCF-7:CFR cells is driven by on-target engagement of BD1. In an orthogonal BROMOscan binding assay against a panel of 40 bromodomains, <b>27</b> showed high potency and selectivity (<i>K</i><sub>d</sub> < 1 nM) across BET family proteins (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B and <a href="/doi/suppl/10.1021/acs.jmedchem.0c00456/suppl_file/jm0c00456_si_001.pdf" class="ext-link">Table S1</a>). Selectivity was 90-fold for BRD4 binding (full length, short-iso, <i>K</i><sub>d</sub> = 0.1 nM) over TAF1 (<i>K</i><sub>d</sub> = 9.1 nM) and 370-fold for BRD4 binding over EP300 (<i>K</i><sub>d</sub> = 37 nM). Selectivity for BRD proteins over all other bromodomain proteins was >400-fold. There is no compelling evidence for EP300 inhibition contributing to the antiproliferative actions of <b>27</b> in MCF-7:CFR cells; however, it is noted that NEO2734, a BET inhibitor with activity at EP300, was recently reported to have efficacy in a prostate cancer xenograft model.<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a></div><div class="NLM_p">Cell cultures grown as 3D spheroids better reflect the influence of a tumor microenvironment in supporting cell survival; therefore, it was important to test our observations in 2D cell cultures by studying effects in MCF-7:CFR cells in 3D. Cells were grown as spheroids in low adhesion plates for 1 day, before addition of compound or vehicle, and cell viability was measured at day 9 (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>E). Neither 4-hydroxytamoxifen (4OHT) nor fulvestrant inhibited MCF-7:CFR cell growth, as anticipated for a tamoxifen-resistant cell line passaged for a year in media containing fulvestrant. BET inhibitors <b>1</b> and <b>5</b> significantly inhibited growth, and at the same concentration (100 nM), <b>27</b> caused complete loss of spheroid viability, compatible with the observed high potency in 2D cultures.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00456/20200701/images/medium/jm0c00456_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00456/20200701/images/large/jm0c00456_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Inhibition of growth and transcriptional regulation of MCF-7:CFR cells by <b>27</b> compared to other BET inhibitors: (A) comparison of compound <b>27</b> with benchmark BET inhibitors in cell proliferation assay of MCF-7:CFR 2D cell cultures showing resistance to fulvestrant and 4OHT (TAM) (1 μM); (B) heatmap comparing gene expression changes among control (DMSO), <b>27</b> (100 nM), and <b>1</b> (100 nM) treated MCF-7:CFR cells after 24 h; (C) volcano plot showing top 10 downregulated genes from RNA-seq comparing <b>27</b> with vehicle control; (D) log 2-fold change of significantly regulated oncogenes and tumor suppressors in various oncogenic (Myc, cell cycle) and apoptotic pathways; (E) representative images on day 9 of treatment with DMSO (0.01%), fulvestrant (100 nM), JQ1 (<b>1</b>, 100 nM), AZD5153 (<b>5</b>, 100 nM), and compound <b>27</b> (10 and 100 nM) in MCF7:CFR 3D spheroids; scale bar = 1 mm.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00456/20200701/images/large/jm0c00456_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00456&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In the MCF-7:CFR cell line, the potency of compound <b>27</b> was compared to JQ1 (<b>1</b>), four BET inhibitors currently in clinical trials (<b>2</b>–<b>5</b>), and the BET-PROTAC <b>6</b> (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A and <a href="/doi/suppl/10.1021/acs.jmedchem.0c00456/suppl_file/jm0c00456_si_001.pdf" class="ext-link">Table S3</a>). Compound <b>27</b> was more potent than all other BET inhibitors and degraders tested. The measured potency for growth inhibition by <b>27</b> (IC<sub>50</sub> = 0.9 nM) is compatible with the potency and affinity shown in the BROMOscan and TR-FRET assays. In addition, the observed potency is 10- to 40-fold greater than measured for inhibition of TAF1 and EP300 from BROMOscan data and therefore is incompatible with growth inhibition driven by inhibition of TAF1 or EP300. The exceptional potency and superiority in growth inhibition of MCF-7:CFR cells by <b>27</b> compared to six other BET inhibitors that have reached clinical trials can be explained by the additional interactions of <b>27</b> with Lys91 and the leucine clamp. Further, it should also be noted that the MCF-7:CFR cell line was used in the iterative optimization of compound <b>27</b>.</div><div class="NLM_p">To further explore the mechanism of action of <b>27</b> in inhibiting growth of fulvestrant-resistant MCF-7:CFR cells, we investigated genome-wide gene expression through RNA-seq, comparing <b>27</b> to the benchmark inhibitor <b>1</b> (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B–D). The genome-wide effect of <b>27</b> was significantly greater than <b>1</b>, despite both compounds being studied at a concentration greater than IC<sub>50</sub> (100 nM). BET inhibitor <b>27</b> broadly regulated the gene expression profile in MCF-7:CFR cells, with 1637 genes showing >2-fold downregulation and 897 genes >2-fold upregulation (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B). The top 10 genes regulated by <b>27</b> were IGFBP5, IFI44L, BCAS1, NUPR1, CLEC3A, PARP10, ZMYND8, KRT83, HIST1H3B, and CCDC170 (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C). However, we focused our analysis on three gene pathway clusters: Myc signaling, cell cycle regulation, and apoptosis (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>D).</div><div class="NLM_p">The oncogenic Myc pathway has been shown to be a critical target of BET inhibitor action in the anticancer arena. Consistent with these findings, we found <b>27</b> significantly decreased expression of Myc pathway oncogenes such as c-MYC and n-MYC. Focusing on cell cycle genes, <b>27</b> increased expression of CDKN1B, a tumor suppressor that prevents the activation of the cyclin-D/CDK4 complex, and <b>27</b> also reduced expression of cyclin-D (CCND1) itself. Although Myc pathway genes were regulated in the same direction by <b>27</b> and <b>1</b>, cell cycle genes were differentially regulated. Differential regulation was also observed in the RTK/RAS pathway, with NRAS, ERBB3, and c-MET downregulated by <b>27</b>. The most significantly downregulated oncogene by <b>27</b> is c-MET, the overexpression of which is reported to correlate with poor prognosis and treatment resistance from many reports.<a onclick="showRef(event, 'ref56 ref57'); return false;" href="javascript:void(0);" class="ref ref56 ref57">(56,57)</a> BET inhibition has been shown to induce apoptosis in various cancer cells, and we observed that <b>27</b> upregulated proapoptotic genes such as RIPK1, APAF1, and CASP3/6/9 and at the same time decreased expression of antiapoptotic genes, including BCL2L1 and BCL2. In regulation of estrogen-related genes (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00456/suppl_file/jm0c00456_si_001.pdf" class="ext-link">Figure S4</a>), <i>ESR1</i> itself was transcriptionally downregulated by <b>27</b>.</div><div class="NLM_p last">Together, the RNA-seq analysis of <b>27</b> in fulvestrant-resistant breast cancer cells revealed suppression of Myc pathway signaling, inhibition of cylin-D/CDK4 complex, and a profound shift in the pro- versus antiapoptotic signaling bias toward apoptosis. Moreover, <b>27</b> was clearly differentiated from <b>1</b> in its gene regulation profile, despite the concentration of <b>1</b> tested being 10-fold greater than the IC<sub>50</sub> for growth inhibition by <b>1</b> in MCF-7:CFR cells. This differentiation was amplified in the response of genes controlling cell cycle and apoptosis. These observations are compatible with the superior potency of <b>27</b> in inhibiting growth of MCF-7:CFR cells, relative to <b>1</b>, and highlight the efficacy of <b>27</b> in downregulation of oncogenes and antiapoptotic genes and in upregulation of tumor suppressor and proapoptotic genes.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i8">Efficacy in Endocrine and Palbociclib Resistance</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_75004" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_75004" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">BET inhibitor <b>27</b> was further profiled in ER+ cell lines modeling tumors resistant to multiple therapeutic agents in the clinical setting, including resistance to tamoxifen, AIs, and palbociclib. A scheme showing the derivatization of cell lines is provided in <a href="/doi/suppl/10.1021/acs.jmedchem.0c00456/suppl_file/jm0c00456_si_001.pdf" class="ext-link">Figure S1</a>. In general, long-term estrogen deprivation (LTED) and long-term exposure to 4OHT (the active metabolite of tamoxifen) are used to model AI and tamoxifen resistance, respectively. The majority of cell lines obtained by these methods remain ER+ (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00456/suppl_file/jm0c00456_si_001.pdf" class="ext-link">Figure S3</a>)and either lose sensitivity to 4OHT or grow more rapidly in the presence of 4OHT. The parental MCF-7:WS8 cell line, from which all MCF-7 resistant lines were obtained, is sensitive to growth inhibition induced by treatment with 4OHT (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A). Compound <b>27</b> dose-dependently inhibited the proliferation of these tamoxifen-sensitive cells with potency almost 2 orders of magnitude superior to benchmark inhibitor <b>1</b>. A similar pattern was observed for growth inhibition of the tamoxifen-resistant cell lines, MCF-7:5C and MCF-7:TAM1, wherein the potency of <b>27</b> was 1–2 orders of magnitude superior to <b>1</b> (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B,C).</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00456/20200701/images/medium/jm0c00456_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00456/20200701/images/large/jm0c00456_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. BET inhibitor <b>27</b> is a potent antiproliferative agent in breast cancer cell lines resistant to tamoxifen, AIs, CDK4/6 inhibitors, and fulvestrant. (A–E) Concentration-dependent response to compound <b>27</b> (red) compared to <b>1</b> (blue) and 4OHT (TAM, brown), with potency for <b>27</b> annotated. (A) <b>27</b> dose-dependently inhibited the proliferation of parental MCF-7:WS8 cells that are sensitive to growth inhibition by tamoxifen. Compound <b>27</b> dose-dependently inhibited the proliferation of tamoxifen resistant MCF-7:5C (B) and MCF-7:TAM1 (C) cells with superior potency to <b>1</b>. Compounds <b>27</b> and <b>1</b> dose-dependently inhibited the proliferation of tamoxifen sensitive parental T47D cells (D) and <i>ESR1</i> D538G mutant T47D/TDG cells (E), with no significant right-shift in potency in the <i>ESR1</i> mutant cell line. (F) Compound <b>27</b> dose-dependently inhibited the proliferation of MCF-7:CPR cells that are resistant to tamoxifen (brown. 1 μM) and palbociclib (green. 1 μM). (G) Comparison of <b>27</b> with <b>1</b> and <b>5</b> in fulvestrant-resistant MCF-7:CFR cell proliferation assay using drug washout after 3 h, by replacing media after incubation with drug for 3 h and measuring cell viability after 5 days. Solid bars show no washout. Hashed bars show washout treatment.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00456/20200701/images/large/jm0c00456_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00456&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The tamoxifen-sensitive and endocrine-dependent T-47D cell line was sensitive to growth inhibition by <b>27</b> with potency almost 100-fold greater than <b>1</b>, demonstrating that the potency of <b>1</b> extended beyond MCF-7 breast cancer cell lines (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>D). The <i>ESR1</i> D538G mutation is one of three mutations most commonly observed in patients with acquired resistance to AI therapy, conferring estrogen-independent cancer cell survival.<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> The T47D/TDG cell line expresses only mutant D538G ERα, and the parental T47D cell line studied herein provides the appropriate estrogen-dependent comparator (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>E versus 5D). Compound <b>27</b> dose-dependently inhibited the proliferation of T47D/TDG cells with no loss of potency compared to the parent T47D cell line and was again much more potent than benchmark <b>1.</b></div><div class="NLM_p">Resistance to CDK4/6 inhibitors is emerging in the clinic.<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a> Palbociclib is the first and most prescribed drug in this class; therefore, MCF-7:5C cells were passaged in the presence of increasing concentrations of palbociclib for 6 months to obtain an endocrine-independent cell line stably resistant to tamoxifen and palbociclib. In this MCF-7:CPR cell line, potent, concentration-dependent growth inhibition was observed after <b>27</b> treatment (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>F).</div><div class="NLM_p last">BET inhibitor <b>27</b> showed both high efficacy and potency (IC<sub>50</sub> < 10 nM) across all cell lines with significantly superior potency compared to <b>1</b> in all cell lines studied and superior potency to a variety of BET inhibitors in the MCF-7:CFR cell line. Although many more potent BET inhibitors have been discovered since the discovery of JQ1 (<b>1</b>), this BET inhibitor remains the benchmark with >140 papers annually reporting data on JQ1 from 2017 to 2019.</div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i9">Admet and <i>in Vivo</i> Efficacy</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_40510" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_40510" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Profiling of <b>27</b> in ADMET and physicochemical studies demonstrated acceptable solubility (73.4 μM, pH = 7.4) and human hepatocyte stability (<i>T</i><sub>1/2</sub> = 202 min; clearance rate = 6.86 (μL/min)/10<sup>6</sup> cells) (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). The IC<sub>50</sub> for hERG inhibition was 10 000-fold right-shifted from the potency observed for inhibition of breast cancer cell growth, indicating an acceptable therapeutic index with respect to hERG. Similarly, across seven human CYP450 enzymes, weak inhibition was observed with IC<sub>50</sub> for CYP3A4 estimated as ≥10 μM and CYP2C8 showing 84% inhibition at 10 μM. In the <i>in vivo</i> mouse PK studies (<a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>), compound <b>27</b> had moderate clearance and acceptable volume of distribution (1.49 L/kg). The elimination half-life was 2.9 h, and the oral bioavailability was 49%.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Properties of <b>27</b> (HCl Salt) <i>in Vitro</i></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col /></col><col /></col><col /></col><col /></col><col /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">solubility (PBS, pH 7.4, μM)<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center">hERG inhibition, IC<sub>50</sub> (μM)<a class="ref internalNav" href="#t4fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center">CYP inhibition (% at 10 μM)<a class="ref internalNav" href="#t4fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center">metabolic stability in human hepatocytes<a class="ref internalNav" href="#t4fn4" aria-label="d">d</a></th><th class="colsep0 rowsep0" align="center">protein binding (%)<a class="ref internalNav" href="#t4fn5" aria-label="e">e</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="7">73.43</td><td class="colsep0 rowsep0" rowspan="7">29.258</td><td class="colsep0 rowsep0">1A2 (2.29)</td><td class="colsep0 rowsep0" rowspan="3"><i>in vitro</i><i>T</i><sub>1/2</sub> (min): 202.00</td><td class="colsep0 rowsep0" rowspan="7">97.98</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0">2A6 (7.47)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0">2C8 (84.23)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0">2C9 (30.74)</td><td class="colsep0 rowsep0" rowspan="4"><i>in vitro</i> Cl<sub>int</sub> ((μL/min)/10<sup>6</sup> cells): 6.86</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0">2C19 (31.79)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0">2D6 (14.50)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0">3A4 (50.61)</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">Kinetic solubility. For reference, the solubility of progesterone is 22.37 μM.</p></div><div class="footnote" id="t4fn2"><sup><sup>b</sup></sup><p class="last">HEK293 cell line stably transfected with hERG gene is employed, and dofetilide is used as a positive control (IC<sub>50</sub> = 0.015 μM).</p></div><div class="footnote" id="t4fn3"><sup><sup>c</sup></sup><p class="last">For reference, the inhibitory percentages of subtypes of CYP by known inhibitors at 10 μM are CYP1A2 (phenacetin, 83.68), CYP2A6 (coumarin, 98.19), CYP2C8 (paclitaxel, 80.89), CYP2C9 (tolbutamide, 87.68), CYP2C19 (mephenytoin, 96.83), CYP2D6 (dextromethorphan, 94.37), and CYP3A4 (midazolam, 99.46).</p></div><div class="footnote" id="t4fn4"><sup><sup>d</sup></sup><p class="last">For reference, <i>in vitro</i><i>T</i><sub>1/2</sub> and <i>in vitro</i> Cl<sub>int</sub> of verapamil is 26.19 min and 52.91 (μL/min)/10<sup>6</sup> cells, respectively.</p></div><div class="footnote" id="t4fn5"><sup><sup>e</sup></sup><p class="last">For reference, protein binding of ketoconazole is 98.35%.</p></div></div></div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Pharmacokinetic Properties of <b>27</b> (HCl Salt) in Mice<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">route/dose</th><th class="colsep0 rowsep0" align="center" char="."><i>T</i><sub>1/2</sub> (h)</th><th class="colsep0 rowsep0" align="center" char="."><i>T</i><sub>max</sub> (h)</th><th class="colsep0 rowsep0" align="center" char="."><i>C</i><sub>max</sub> (ng/mL)</th><th class="colsep0 rowsep0" align="center" char=".">AUC (h·ng/mL)</th><th class="colsep0 rowsep0" align="center" char=".">Cl ((mL/min)/kg)</th><th class="colsep0 rowsep0" align="center" char="."><i>V</i><sub>ss</sub> (L/kg)</th><th class="colsep0 rowsep0" align="center" char="."><i>F</i> (%)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">iv 5 mg/kg</td><td class="colsep0 rowsep0" align="char" char=".">1.95</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char=".">3208</td><td class="colsep0 rowsep0" align="char" char=".">25.5</td><td class="colsep0 rowsep0" align="char" char=".">1.49</td><td class="colsep0 rowsep0" align="char" char="."> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">po 30 mg/kg</td><td class="colsep0 rowsep0" align="char" char=".">2.90</td><td class="colsep0 rowsep0" align="char" char=".">0.25</td><td class="colsep0 rowsep0" align="char" char=".">3090</td><td class="colsep0 rowsep0" align="char" char=".">9598</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char=".">49.0</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup><sup>a</sup></sup><p class="last">PK parameters from measurement of plasma concentration of <b>27</b> (po 30 mg/kg) measured at 0.25, 0.5, 1, 2, 4, 8, and 24 h (<i>N</i> = 3) in a formulation of 30% PEG400/70% HP-β-CD (10% in water) (v/v).</p></div></div></div><div class="NLM_p">Early readouts from clinical trials showed that oncogene downregulation by BET inhibitors occurs within 3 h of drug administration and suggest that a continuous exposure to BET inhibitor is not required to maintain oncogene suppression and antiproliferative activity.<a onclick="showRef(event, 'ref60 ref61 ref62'); return false;" href="javascript:void(0);" class="ref ref60 ref61 ref62">(60−62)</a> Furthermore, it is widely believed that toxicity associated with BET inhibitors is on-target, and therefore continuous exposure may be undesirable.<a onclick="showRef(event, 'ref63 ref64'); return false;" href="javascript:void(0);" class="ref ref63 ref64">(63,64)</a> To test if short-term exposure to <b>27</b> on would maintain efficacy, we conducted a cell culture experiment with drug washout, showing that the efficacy of compound <b>27</b> was only slightly diminished (right-shfted; <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>G). Comparison was made in washout experiments with BET inhibitors <b>1</b> and <b>5</b>, showing similar reduced efficacy after washout, with <b>1</b> showing no significant efficacy at <300 nM. Although this cell culture model has limitations, the results suggest short-term exposure to <b>27</b> preserves efficacy.</div><div class="NLM_p">Thrombocytopenia is a potential dose-limiting toxicity associated with anticancer agents including BET inhibitors. Several publications of BET inhibitor clinical trials have reported human thrombocytopenia readouts: for example, CPI-0610 was well tolerated, with the principal toxicity being reversible, noncumulative, dose-dependent thrombocytopenia.<a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a> In contrast, there are few reports of <i>preclinical</i> analysis of thromombocytopenia induced by BET inhibitors. However, platelet data on BET inhibitor, <b>3</b> (3 (mg/kg)/day)has very recently been published in rats, which provides a validated model of clinical, dose-limiting thrombocytopenia.<a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(64)</a> Therefore, we measured platelet count in rats administered <b>27</b> po at a dose (25 (mg/kg)/day) in rats equivalent to the dose in mice used in the xenograft study, compared rats administered compound <b>3</b>. The dose and response to <b>3</b> were comparable to the published report,<a onclick="showRef(event, 'ref64 ref66'); return false;" href="javascript:void(0);" class="ref ref64 ref66">(64,66)</a> and after 10 days of drug treatment, the platelet count was significantly reduced in the <b>3</b> treatment group compared to the <b>27</b> treatment group (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A). The mean platelet count in rats before drug administration was (1088 ± 77) × 10<sup>9</sup> cells/mL compared to (173 ± 69) × 10<sup>9</sup> cells/mL and (1052 ± 85) × 10<sup>9</sup> cells/mL 10 days after treatment initiation for <b>3</b> and <b>27</b>, respectively.</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00456/20200701/images/medium/jm0c00456_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00456/20200701/images/large/jm0c00456_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Platelet count in rats and inhibition of tumor growth in mice of tamoxifen-resistant MCF7:TAM1 cell-derived xenografts. (A) Platelet counts from rats administered <b>27</b> (25 mg/kg) or <b>3</b> (3 mg/kg) starting at day 0 showing data for individual rats (<i>n</i> = 3): <i>p</i> < 0.05 at day 10 by two-tailed paired <i>t</i> test. (B) Tumors grown to 0.32 cm<sup>2</sup> were treated with vehicle, TAM (1.5 (mg/mouse)/day), or compound <b>27</b> (HCl salt) (50 (mg/kg)/day) by daily oral gavage. Data show mean and SD. (C) Average body weight changes compared to pretreatment. (D) Changes in individual tumor size at 4 weeks. (E) Low dose fulvestrant (1 (mg/mouse)/week by sc) in combination with <b>2</b> 50 (mg/kg)/day (formulated in 0.5% (w/v) HPMC/TWEEN80 1% in water<a onclick="showRef(event, 'ref69'); return false;" href="javascript:void(0);" class="ref ref69">(69)</a>) or <b>27</b> 30 (mg/kg)/day (formulated in 10% PEG400/90% HP-β-CD (10%)). Changes of individual tumor size at 4 weeks were plotted: data show the mean and SD with one-way ANOVA analysis with Dunnett’s multiple comparisons test.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00456/20200701/images/large/jm0c00456_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00456&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The reported half-life for <b>3</b> varies across species (mice 2.7 h, rat 2.6 h, human 25.1 h). Thus, although we observed a promising, neutral effect of <b>27</b> on platelet numbers in rats, it is impossible to attribute this to the somewhat short half-life of <b>27</b>. Although <b>27</b> was observed to have favorable characteristics in human liver hepatocytes and CYP450 inhibition, the extrapolation of ADME data in mice and platelet count in rats cannot be used to predict safety in humans without further ADMET studies.</div><div class="NLM_p last">Compound <b>27</b> was studied in the tamoxifen-resistant MCF-7:TAM1 cell-derived xenograft model in nude mice. This tumorigenic cell line has been used to validate oral SERDs that have progressed to clinical trials in women with ER+ advanced breast cancer (ABC) who have failed on endocrine therapy.<a onclick="showRef(event, 'ref67'); return false;" href="javascript:void(0);" class="ref ref67">(67)</a> Compound <b>27</b> (50 (mg/kg)/day po) was compared with tamoxifen (1.5 mg/mouse/day po) and vehicle control. The BET inhibitor significantly inhibited growth of tumors (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B) without any effect on animal body weight (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>C). At day 28 of treatment, tamoxifen had no significant effect on tumor growth relative to the vehicle control group, whereas <b>27</b> significantly inhibited tumor growth relative to both the tamoxifen group and the vehicle control group (tumor growth inhibition: 106%, <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>D). In addition to no loss of weight, no overt signs of toxicity were observed for once daily administration of <b>27</b> for 4 weeks. In a follow-up xenograft study, we compared <b>27</b> (30 mg/kg, po) with <b>2</b> (50 mg/kg, po) in combination with fulvestrant (sc 1 (mg/mouse)/week). Compound <b>2</b> was selected as an appropriate comparison because it is the only BET inhibitor in clinical trials for advanced breast cancer in combination with fulvestrant (NCT02964507).<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a> Both compounds showed comparable and significant tumor growth inhibition versus the vehicle control group (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>E).</div></div><div id="sec5_1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i12">Chemistry</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_82821" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_82821" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The convergent synthesis of pyrrolopyridone BET inhibitors was mainly based on a Suzuki–Miyaura cross coupling reaction to construct a C–C bond between pyrrolopyridone moiety and the substituted indole moiety. The general synthesis of <b>10</b>, <b>11</b>, and <b>19</b>–<b>24</b> is summarized in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. The two components of the Suzuki–Miyaura coupling reaction were derived from commercially available compounds <b>32</b> and 6-bromo-1<i>H</i>-indol-4-amine (<b>37</b>). <b>32</b> was converted to <b>34</b> via a two-step sequence involving N-methylation followed by iodination of the 3-pyrrolo position. Iodo-substitued pyrrolopyridone, <b>34</b>, was then transformed to the organoboronic ester <b>35</b> in 65% yield by using one-pot lithiation–borylation reaction. Aryl bromide, the other component of the Suzuki–Miyaura reaction, is prepared through a sequence of reactions starting from selective sulfonamidation of <b>37</b> to give <b>38a</b>,<b>b</b>, followed by S<sub>N</sub>2 reaction with different benzyl or alkyl bromides to afford the corresponding <b>39a</b>–<b>h</b>. Suzuki–Miyaura coupling of precursor bromides with <b>35</b> using Pd(dppf)Cl<sub>2</sub>–DCM as the catalyst gave <b>40a</b>–<b>h</b>. A mild condition for deprotection of the methyl ether using TMSCl and NaI generated the final products in near-quantitative yield.</div><figure id="sch1" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00456/20200701/images/medium/jm0c00456_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00456/20200701/images/large/jm0c00456_0010.jpeg" id="GRAPHIC-d7e1927-autogenerated" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthetic Route of Compounds <b>10</b>, <b>11</b>, and <b>19</b>–<b>24</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00456/20200701/images/large/jm0c00456_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00456&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) MeI, 60% NaH, DMF, RT, 1 h; (b) NIS, DMF, RT, 12 h, 92% over two steps; (c) <i>n</i>-BuLi, 2-isopropoxy-4,4,5,5-tetramethyl-1,3,2-dioxaborolane, THF, −78 °C to RT, overnight, 65%; (d) <i>n</i>-BuLi, DMF, THF, −78 °C, 2 h, 73%; (e) R<sub>1</sub>SO<sub>2</sub>Cl, pyridine, RT, 2 h, 75–96%; (f) R<sub>2</sub>Br, K<sub>2</sub>CO<sub>3</sub>, DMF, 50 °C, 4–16 h, 66–92%; (g) <b>35</b>, Pd(dppf)Cl<sub>2</sub>–DCM, K<sub>3</sub>PO<sub>4</sub>, dioxane/H<sub>2</sub>O, 6–24 h, 90 °C, 77–94%; (h) TMSCl, NaI, MeCN/H<sub>2</sub>O, 3 h, 50 °C, 82–96%.</p></p></figure><div class="NLM_p">The 7-indole pyrrolopyridone (<b>12</b>–<b>14</b>) and indazole derivatives (<b>17</b>, <b>18</b>) were prepared by using an analogous approach outlined in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>. Alkylation with the appropriate aryl bromide generated <b>42a</b>–<b>c</b> and <b>47</b>, followed by reduction of nitro group to provide anilines. Subsequent sulfonamidation of <b>43a</b>–<b>c</b> and <b>48</b> afforded corresponding Suzuki–Miyaura precusor bromides, which were then coupled with <b>35</b>. Compounds <b>12</b>–<b>14</b> and <b>17</b> and <b>18</b> were produced by deprotection of the corresponding methyl ethers.</div><figure id="sch2" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00456/20200701/images/medium/jm0c00456_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00456/20200701/images/large/jm0c00456_0011.jpeg" id="GRAPHIC-d7e2040-autogenerated" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthetic Route of Compounds <b>12</b>–<b>18</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00456/20200701/images/large/jm0c00456_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00456&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) aryl bromide, K<sub>2</sub>CO<sub>3</sub>, DMF, RT, 2 h, 75–96%; (b) Fe, saturated NH<sub>4</sub>Cl (aq), EtOH, reflux, 2 h, 66–92%; (c) sulfonyl chloride, pyridine, RT, 1–5 h, 83–90%; (d) <b>35</b>, Pd(dppf)Cl<sub>2</sub>–DCM, K<sub>3</sub>PO<sub>4</sub>, dioxane/H<sub>2</sub>O, 6–24 h, 90 °C, 75–95%; (e) TMSCl, NaI, MeCN/H<sub>2</sub>O, 2 h, 50 °C, 87–95%; (f) <b>36</b>, AcOH, NaBO<sub>3</sub>, 50 °C, 1 h, 67%; (g) ethanesulfonyl chloride, triethylamine, DCM, RT, 3 h, 83–90%.</p></p></figure><div class="NLM_p">Preparation of benzimidazole analogs, <b>15</b> and <b>16</b>, took a more linear synthesis because direct C–C bond construction between benzimidazole and pyrrolopyridone was not achieved with standard conditions. The 3-nitrobenzene-1,2-diamine, <b>51</b>, was treated with 4-fluorobenzyl bromide <b>52</b> to give the mono N-alkylation product <b>53</b>. The key oxidative cyclization of <b>53</b> with aldehyde <b>36</b> (prepared from <b>35</b> through formylation shown in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>) was achieved by heating with NaBO<sub>3</sub> in AcOH to yield <b>54</b>. Reduction of the nitro group afforded the corresponding aniline <b>55</b>, which was further derivatized with ethanesulfonyl chloride to provide <b>56a</b>,<b>b</b>. Demethylation led to the final products <b>15</b> and <b>16</b>.</div><div class="NLM_p">Bipyridine analogs, <b>25</b>–<b>30</b>, were prepared by using a convergent strategy similar to that depicted in <a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>. Treatment of <b>57</b> with 2,2′-(bromomethylene)dipyridine afforded <b>58a</b>,<b>b</b>, which was converted to <b>59a</b>,<b>b</b> in the presence of MeI and <i>t</i>-BuOK. It is noted that <i>t</i>-BuOK should be added to the THF solution of MeI and substrate <b>58</b> to prevent unwanted initial formation of the debenzylation product. Reduction of nitro group using Fe/NH<sub>4</sub>Cl system gave anilines <b>60a</b>–<b>c</b>, which then underwent acylation or sulfonylation to afford <b>61a</b>–<b>f</b>. With both boronic ester <b>35</b> and bromide <b>61c</b>, we directed our attention to optimize the Suzuki–Miyaura coupling reaction. A variety of conditions were explored, and we identified Pd(dppf)Cl<sub>2</sub>–DCM as the optimal palladium catalyst with K<sub>3</sub>PO<sub>4</sub> as the base in 1,4-dioxane/water (4:1, v/v) as solvent, at 90 °C (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00456/suppl_file/jm0c00456_si_001.pdf" class="ext-link">Table S2</a>). Demethylation gave the final products <b>25</b>–<b>30</b> in near-quantitative yield. Gram-scale synthesis of <b>27</b> was achieved through this same route.</div><figure id="sch3" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00456/20200701/images/medium/jm0c00456_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00456/20200701/images/large/jm0c00456_0012.jpeg" id="GRAPHIC-d7e2228-autogenerated" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthetic Route of Compounds <b>25</b>–<b>30</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00456/20200701/images/large/jm0c00456_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00456&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) 2,2′-(bromomethylene)dipyridine or bromodiphenylmethane, K<sub>2</sub>CO<sub>3</sub>, DMF, 50 °C, 5–16 h, 72–91%; (b) R<sub>3</sub>I, <i>t</i>-BuOK, THF, RT, 2 h, 73–93%; (c) Fe, saturated NH<sub>4</sub>Cl (aq), EtOH, reflux, 2 h; (d) sulfonyl chloride or acyl chloride, pyridine, RT, 1–5 h, 83–90%; (e) <b>35</b>, Pd(dppf)Cl<sub>2</sub>–DCM, K<sub>3</sub>PO<sub>4</sub>, dioxane/H<sub>2</sub>O, 90 °C, 6–24 h, 66–93%; (f) TMSCl, NaI, MeCN/H<sub>2</sub>O, 50 °C, 2 h, 85–94%; (g) MeLi, THF, −42 °C, 2 h, 55%.</p></p></figure><div class="NLM_p last">Synthesis of compound <b>31</b> also used the Suzuki–Miyaura coupling reaction by starting from the commercially available compound <b>63</b>. Substitution followed by methylation of <b>64</b> furnished <b>65</b>, which was then coupled with <b>35</b> to yield <b>66</b>. Acid sensitive tertiary alcohol group forced us to carry out TMSCl-mediated deprotection first before it was introduced into the scaffold. Demethylation provided facile access to compound <b>67</b> without further purification. The final tertiary alcohol derivative, <b>31</b>, was obtained by treating <b>67</b> with methyllithium in THF at −42 °C.</div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_33357" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_33357" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">ER+ breast cancer is a transcriptionally driven cancer dependent on ER signaling.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> Endocrine therapy using tamoxifen and AIs attempts to block ER signaling, either by selective modulation/antagonism of ER in breast tissues or by inhibiting production of circulating estrogens. However, endocrine resistance occurs in 50% of breast cancer patients. The SERD, fulvestrant, is currently the most efficacious ER-targeted therapy in metastatic disease, causing antagonism and loss of ER through proteasomal degradation. Together with the emergence of CDK4/6 inhibitors, the increasing use of fulvestrant has rapidly changed the standard of care for advanced and metastatic ER+ breast cancer. Resistance to fulvestrant and CDK4/6 inhibitors is an emerging clinical phenomenon. We envisioned that targeting of ER-driven transcription, via inhibition of bromodomain proteins, essential activating components of the ER transcriptional complex, would provide an alternative therapeutic approach to ER+ breast cancer.</div><div class="NLM_p">Epigenetic dysregulation is a hallmark of cancer.<a onclick="showRef(event, 'ref70'); return false;" href="javascript:void(0);" class="ref ref70">(70)</a> The discovery of the benchmark BET inhibitor, JQ1, provides the foundation for regulation of oncogene expression through “readers” of histone marks. The combination of a BRD-targeting BET inhibitor with a SERD would afford a more robust ablation of ER-driven transcription and circumvent resistance by attacking two different components of the ER transcriptional complex. The first generation BET inhibitor, <b>2</b>, has entered breast cancer clinical trials in combination with fulvestrant (NCT02964507). Herein, we report a novel BET inhibitor, <b>27</b>, with high binding affinity (<i>K</i><sub>d</sub> = 0.29 nM) and cellular potency (IC<sub>50</sub> = 0.9 nM and >100-fold over JQ1). In two orthotopic xenograft models of endocrine-resistant ER+ breast cancer, oral administration of <b>27</b> significantly inhibited tumor growth, either as monotherapy or compared with <b>2</b> in combination with fulvestrant.</div><div class="NLM_p">Most importantly, <b>27</b> is the first BET inhibitor to undergo iterative optimization using an ER+ endocrine-resistant breast cancer cell line, MCF-7:CFR, that is also resistant to fulvestrant. This approach strongly supports the ability of <b>27</b> in combination with fulvestrant to block cancer cell escape pathways normally associated with fulvestrant resistance. Moreover, <b>27</b> maintained high antiproliferative potency in a range of cell lines modeling resistance to current therapies, including <i>ESR1</i> mutations and resistance to the leading CDK4/6 inhibitor, palbociclib. Iterative optimization was also guided by measuring BRD4-BD1 binding affinity by TR-FRET. The excellent correlation between binding affinity and cellular potency across the 23 BET inhibitors discovered herein supports BRD4 as a primary target eliciting the observed antiproliferative phenotype.</div><div class="NLM_p">The mechanisms by which <b>27</b> exerted antiproliferative activity in fulvestrant-resistant MCF-7:CFR cells were analyzed by RNA-seq. These mechanisms include downregulation of oncogenes and upregulation of tumor suppressor genes. Consistent with an early report on BET inhibitors, <b>27</b> downregulated components of both oncogenic MYC signaling and cell cycle signaling.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> The transcriptomic response to <b>27</b> was enhanced and differentiated relative to <b>1</b>, including a dramatic downregulation of antiapoptotic and upregulation of apoptotic genes. These observations in a cell line resistant to endocrine therapy and fulvestrant provide a solid mechanistic basis for the further development of <b>27</b>, and BET inhibitors in general, for metastatic ER+ breast cancer.</div><div class="NLM_p">Optimization was structure-based, driven by pyrrolopyridone and sulfonamide groups forming multiple hydrogen bonds with BRD4. The high potency of <b>27</b> was the result of adding a second pyridine group to the benzylic position leading to a novel binding interaction of this ring with the “leucine clamp” (L92 and L94), which greatly enhanced the binding affinity to BRD4. The unique approach and the rationale for high potency were supported by obtaining a crystal structure of <b>27</b> bound to BRD4. The increased affinity gained from the “leucine clamp” interaction provides a platform for further elaboration of the core of <b>27</b> to obtain domain-selective BET inhibitors and PROTACs while maintaining good affinity and potency (these will be the subject of future reports).</div><div class="NLM_p last">Although early clinical trials have revealed the activity of BET inhibitors in both hematological malignancies and solid tumors,<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> the clinical development of BET inhibitors is complicated by the roles of various BET proteins in multiple gene regulatory networks and signaling pathways. Many BET inhibitors in clinical development have suffered from reversible, dose-limiting thrombocytopenia in clinical trials.<a onclick="showRef(event, 'ref63 ref64'); return false;" href="javascript:void(0);" class="ref ref63 ref64">(63,64)</a> Evidence suggests that oncogene downregulation is induced by BET inhibitors within 3 h of administration.<a onclick="showRef(event, 'ref60 ref61 ref62'); return false;" href="javascript:void(0);" class="ref ref60 ref61 ref62">(60−62)</a> Therefore, thrombocytopenia may be resolved by a “dosing holiday” to allow systemic recovery<a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a> or may be mitigated with supportive care agents such as romiplostim.<a onclick="showRef(event, 'ref71'); return false;" href="javascript:void(0);" class="ref ref71">(71)</a> The PK characteristics of each BET inhibitor, in addition to selectivity and potency, therefore become determinants of both efficacy and tolerability. Although further studies are needed, <i>in vitro</i> human and <i>in vivo</i> mouse ADME properties, combined with a effect on platelet count in rats, support the further development of <b>27</b> as a new targeted therapeutic in ER+ breast cancer, in particular given the efficacy in cancer cells resistant to fulvestrant and palbociclib.</div></div><div id="sec5_3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i17">Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_52171" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_52171" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">In summary, a family of novel, orally bioavailable pyrrolopyridone BET inhibitors was discovered, culminating in <b>27</b> that displayed high selectivity and affinity for BRD proteins and high antiproliferative potency in tamoxifen, fulvestrant, and palbociclib-resistant and <i>ESR1</i> mutant breast cancer cell cultures. Data support a mechanism of action in fulvestrant-resistant cells mediated by binding to BRD4 with antiproliferative actions driven by downregulation of MYC and cylinD-cdk4 oncogenes and a significant shift from anti- to proapoptotic gene expression. The high potency and affinity of <b>27</b> for BRD4 was amply explained by novel interactions within a new cocrystal structure of <b>27</b>. ADMET and PK parameters showing oral bioavailability supported tumor xenograft studies that demonstrated growth inhibition of tamoxifen-resistant breast tumors in mice, with no body weight loss or overt toxicity. Similarly, studies in rats showed attenuated thrombocytopenia risk. Collectively, our data suggest that <b>27</b> is representative of a novel class of BET inhibitors with potential in cancer therapy.</div></div><div id="sec6" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i18">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_86054" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_86054" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec6_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Cell Lines and Culture Conditions</h3><div class="NLM_p last">A detailed derivatization of MCF-7 cell lines is summarized in <a href="/doi/suppl/10.1021/acs.jmedchem.0c00456/suppl_file/jm0c00456_si_001.pdf" class="ext-link">Supporting Information Figure S1</a>. The parental, tamoxifen-resistant cell line, MCF7:5C, was derived from a single cell clone by the Jordan group at Northwestern University as previously described.<a onclick="showRef(event, 'ref72 ref73'); return false;" href="javascript:void(0);" class="ref ref72 ref73">(72,73)</a> These cells were maintained in phenol red-free RPMI 1640 medium supplemented with 10% charcoal–dextran treated FBS, 1% antibiotic–antimycotic, and 1% GlutaMAX at 37 °C and 5% CO<sub>2</sub>. The MCF7:CFR cell line was generated through long-term exposure to increasing concentrations of fulvestrant (200 nM to 1 μM) until resistance was established (>25 passages). MCF7:CFR cells are maintained in fulvestrant (200 nM) and phenol red-free, stripped RPMI-1640. MCF7:ws8 cells were cultured in RPMI-1640 media with 10% FBS, 1% nonessential amino acids (NEAA), antibiotic-antimitotic (AbAm), and GlutaMAX, and 10 ng/mL insulin. MCF7:TAM1 cells were maintained in the same media as MCF7:5C with the addition of 100 nM 4OHT. MCF7:CPR cells were generated with increasing concentrations of CDK4/6 inhibitor (100–1000 nM), palbociclib, and maintained in the same media as MCF7:5C with the addition of 200 nM palbociclib. T47D parental and T47D/TDG are gifts from David Shapiro’s lab at University of Illinois at Urbana—Champaign.<a onclick="showRef(event, 'ref74'); return false;" href="javascript:void(0);" class="ref ref74">(74)</a></div></div><div id="sec6_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Cell Growth Assay</h3><div class="NLM_p last">Compounds <b>1</b>–<b>6</b> were purchased from MedChemExpress. MCF-7:CFR cells were grown in phenol red-free RPMI 1640 medium and 10% charcoal–dextran treated FBS for 2 days prior to each experiment. One day before the treatment, cells were seeded in a 96-well clear, flat bottom plate at a density of 2000 cells/well and treated with either 0.1% (v/v) DMSO or compounds of interest (0.1 nM to 10 μM). All compounds were dissolved in DMSO and added to the medium at a final 1:1000 dilution. Cell number was monitored daily via Celigo imaging system (Nexcelom Bioscience). On day 5, the medium was removed, and cells were quantified by using Hoechst 33342 dye staining and quantified by Celigo. Cell growth assay in other cells were performed in a similar manner with corresponding medium. The 3 h treatment and washout study in MCF-7:CFR cell cultures was carried out in similar fashion except switching treatment medium after 3 h to normal growth medium.</div></div><div id="sec6_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> 3D-Spheroid Growth Assay</h3><div class="NLM_p last">MCF7:CFR cells were plated at a density of 1000 cells/well in Corning 96-well black, clear round-bottom, ultralow attachment spheroid microplates and grown in the absence of treatment for 24 h. Spheroids were then treated with 2× treatment media following the removal of half of the media (100 μL) from each well. Treatment was repeated every 2–3 days for 14 days. CellTiter-Glo 3D cell viability assay protocol was used to determine growth inhibition of the spheroids, as per manufacturer’s instructions. Luminescence signal was read using a Synergy Neo (Biotek). Data were normalized to blank (media with CellTiter Glo 3D reagent).</div></div><div id="sec6_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Time-Resolved Fluorescence Energy Transfer (TR-FRET) Bromodomain Binding Assay</h3><div class="NLM_p last">The binding affinity of our chemical library was assayed measured using Cayman BRD4 bromodomain 1 TR-FRET assay kit (catalog no. 600520) as per manufacturer’s instructions. Briefly, 5 μL of tested compound and 10 μL of BRD4 BD1 europium chelate were added to Corning 384-well flat, clear plate and preincubated for 15 min. Subsequently, 5 μL of the BRD4 bromodomain 1 ligand/APC acceptor mixture was added to each tested well. The plate was then sealed and incubated at room temperature in the dark for 1 h. The TR-FRET signal was read using a Synergy Neo (Biotek) by measuring the dual emissions at 620 and 670 nm using an excitation at 340 nm. The data are analyzed using TR-FRET ratio (670 nm/620 nm emission) and normalized to DMSO as 100%.</div></div><div id="sec6_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Protein Purification</h3><div class="NLM_p last">The gene of human BRD4-BD1 (residues 44–168) with a His<sub>6</sub>-SUMO-tag at the N-terminus was codon-optimized, synthesized (Genscript), and cloned into a pET15b vector between XhoI and <i>Bam</i>HI. The plasmid was transformed into <i>E. coli</i> BL21(DE3) cell for protein expression. One liter of BL21(DE3) cells containing the recombinant BRD4-BD1 gene was grown to an OD<sub>600</sub> of 0.65–0.7 at 37 °C in LB medium containing 100 μg/mL ampicillin. The cells were then induced with 0.5 mM IPTG for 14 h at 25 °C. Harvested cells were lysed by sonication in lysis buffer (1 mg/mL lysozyme, 0.01% TritonX-100, 0.025 mg/mL DNase I, and one tablet of EDTA-free protease inhibitor cocktail in buffer A of 50 mM HEPES, pH 8.0, 500 mM NaCl, 5 mM imidazole, 5% glycerol, and 0.5 mM TCEP). The His<sub>6</sub>-SUMO fused BRD4-BD1 was purified by a two-step protocol, a HisTrap HP column (GE Healthcare) with a stepwise gradient of buffer B (50 mM HEPES, pH 8.0, 500 mM NaCl, 500 mM imidazole, 5% glycerol, and 0.5 mM TCEP) and a 16/60 Superdex 75 size exclusion column with buffer C (50 mM HEPES, pH 8.0, 150 mM NaCl, 5% glycerol, and 0.5 mM TCEP). The His<sub>6</sub>-SUMO tag was cleaved by 2 units/mL of SUMO protease for 2 h at room temperature, producing a BRD4-BD1 with only two extra residues (LE) at the N-terminus. Finally, digested sample was again loaded onto the HisTrap column to remove SUMO protease, cleaved His-SUMO tags, and uncleaved His<sub>6</sub>-SUMO fused BRD4-BD1. Purified BRD4-BD1 protein with approximately 95% purity was either used to set up crystals or flash frozen in liquid nitrogen for storage at −80 °C.</div></div><div id="sec6_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> Protein Crystallography</h3><div class="NLM_p last">Crystals of BRD4(BD1) complexed with <b>27</b> were grown by hanging drop vapor diffusion at 4 °C. Prior to crystallization, 10 mg/mL BRD4(BD1) was incubated with 2.5 mM <b>27</b> for 30 min at room temperature and then centrifuged to remove precipitate. Crystals of the complex were grown by mixing 2 μL of BRD4(BD1): <b>27</b> with 1–2 μL of reservoir solution containing 16–20% PEG 4000, 0.2 M Li<sub>2</sub>SO<sub>4</sub>, and 0.1 M Tris-HCl, pH 8.5. Crystals were cryopreserved by soaking in mother liquor containing 10% glycerol and 200 μM <b>27</b> and frozen directly in the liquid nitrogen stream of the crystal mount prior to data collection. Data were collected on a MAR300 detector at 0.979 Å at the Life Sciences Collaborative Access Team beamline 21-ID-F at the Advanced Photon Source, Argonne National Laboratory. Data indexing, integration, and scaling were performed using XDS<a onclick="showRef(event, 'ref75'); return false;" href="javascript:void(0);" class="ref ref75">(75)</a> (<b>1</b>), and phases were determined by molecular replacement using Molrep<a onclick="showRef(event, 'ref76'); return false;" href="javascript:void(0);" class="ref ref76">(76)</a> (<b>2</b>) and a BRD4(BD1) structure (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3MXF">3MXF</a>) as search model. Rigid body refinement followed by iterative rounds of restrained refinement and model building was performed with CCP4i<a onclick="showRef(event, 'ref77'); return false;" href="javascript:void(0);" class="ref ref77">(77)</a> (<b>3</b>) modules Refmac5<a onclick="showRef(event, 'ref78'); return false;" href="javascript:void(0);" class="ref ref78">(78)</a> (<b>4</b>) and Coot<a onclick="showRef(event, 'ref79'); return false;" href="javascript:void(0);" class="ref ref79">(79)</a> (<b>5</b>). The coordinates and structure factors have been deposited with PDB accession code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6P05">6P05</a>.</div></div><div id="sec6_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> BROMOscan Bromodomain Profiling</h3><div class="NLM_p last">BROMOscan bromodomain profiling was provided by DiscoverX Corp. <i>K</i><sub>d</sub> of test compounds with DNA-tagged bromodomains was determined through binding against a proprietary reference immobilized ligand.</div></div><div id="sec6_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> RNA-seq Analysis</h3><div class="NLM_p last">MCF-7:CFR cells were treated with DMSO (0.01%), <b>1</b> (100 nM), and <b>27</b> (100 nM) for 24 h. RNA was extracted using the RNeasy Plus Mini Kit (Qiagen) as per the manufacturer’s directions. QC, sequence, and data analysis were performed by Novogene.</div></div><div id="sec6_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> Pharmacokinetics Studies</h3><div class="NLM_p last">The Animal Care and Use Committee of the University of Illinois at Chicago approved all the procedures involving animals. Kinetic solubility, hERG inhibition, hepatocyte stability, and human human CYP450 enzymes inhibition were performed by Pharmaron Inc. Rigorous mouse PK profiling was also conducted by Pharmaron Inc. Briefly, HCl salt of <b>27</b> was formulated in 30% PEG400 and 70% “10% HP-β-CD in water” (v/v) in a concentration of 1 mg/mL for iv injection and 3 mg/mL for oral gavage. Blood samples (100 μL) were collected from CD1 mice at 0, 0.0833, 0.25, 0.5, 1, 2, 4, 8, and 24 h for iv and 0, 0.25, 0.5, 1, 2, 4, 8, and 24 h for po (<i>n</i> = 3 for each time point). Plasma concentrations of the compounds were determined by AB API 6500 QTRAP LC/MS/MS instrument developed and validated for this study.</div></div><div id="sec6_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> Animal Experiments</h3><div class="NLM_p last">The Animal Care and Use Committee of the University of Illinois at Chicago approved all the procedures involving animals. For the monotherapy animal study, MCF-7:Tam1 tumors were established in 4–6 week old female ovariectomized athymic nude mice (Harlan Laboratories; strains, nude/Foxn1) with a daily tamoxifen treatment of 1.5 mg/mouse as previously described.<a onclick="showRef(event, 'ref80 ref81 ref82'); return false;" href="javascript:void(0);" class="ref ref80 ref81 ref82">(80−82)</a><b>27</b> was administered by gavage at a dose of 50 (mg/kg)/day for 4 weeks in a formulation of 10% PEG400 and 90% HP-β-CD (10% in water) (v/v). For the combinational therapy animal study, MCF-7:Tam1 tumors were established in 4–6 week old female ovariectomized athymic nude mice (Harlan Laboratories; strains, nude/Foxn1) with a daily tamoxifen treatment of 1.5 mg/mouse as previously described.<a onclick="showRef(event, 'ref80 ref81 ref82'); return false;" href="javascript:void(0);" class="ref ref80 ref81 ref82">(80−82)</a><b>27</b> was administered by gavage at a dose of 30 (mg/kg)/day for 4 weeks in a formulation of 10% PEG400 and 90% HP-β-CD (10% in water) (v/v). <b>2</b> was administered by gavage at a dose of 50 (mg/kg)/day for 4 weeks in a formulation of 0.5% (w/v) HPMC/TWEEN80 1% in water.<a onclick="showRef(event, 'ref69'); return false;" href="javascript:void(0);" class="ref ref69">(69)</a> Tumor volume (TV) was determined weekly using Vernier calipers and calculated using the formula (length) × (width)<sup>2</sup> × π/6. Mean tumor area was plotted against time (in weeks) to monitor tumor growth. Tumor growth inhibition was calculated by the formula (1 – (TV<sub>txDay28</sub> – TV<sub>txInitial</sub>)/(TV<sub>vehDay 28</sub> – TV<sub>vehInitial</sub>)) × 100%. For the thrombocytopenia study, 7-week-old male Sprague-Dawley rats were obtained from Taconic (Rensselaer, NY, USA) and housed two rats per cage. Water and food were provided ad libitum. One-week acclimation period was allowed before treatment started. During the acclimation period blood was collected twice to determine the baseline platelet count. Drug treatment began at 8 weeks of age. All compounds were administrated by oral gavage (10 mL/kg) daily from 4:00 to 5:00 p.m. During the treatment period, the blood was collected 72, 168, and 264 h after treatment initiation and platelets were counted. Blood collection was performed from 11:00 to 12:00. For the blood collection, animals were anesthetized with 2.5% isoflurane inhalation and no more than 3% of estimated circulating blood volume has been withdrawn from tail vein using standard IACUC approved procedure for tail vein blood collection. Animal weight, body, and behavior were monitored daily.</div></div><div id="sec6_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> General Chemical Experimental Information</h3><div class="NLM_p last">Unless otherwise specified, reactions were performed under an inert atmosphere of argon and monitored by thin-layer chromatography (TLC) and/or LCMS. All reagents were purchased from commercial suppliers and used as provided. Synthetic intermediates were purified using CombiFlash chromatography system on 230–400 mesh silica gel. <sup>1</sup>H and <sup>13</sup>C NMR spectra were obtained using Bruker DPX-400 or AVANCE-400 spectrometer at 400 and 100 MHz, respectively. NMR chemical shifts were described in δ (ppm) using residual solvent peaks as standard (chloroform-<i>d</i>, 7.26 ppm (<sup>1</sup>H), 77.16 ppm (<sup>13</sup>C); methanol-<i>d</i><sub>4</sub>, 3.31 ppm (<sup>1</sup>H), 49.00 ppm (<sup>13</sup>C); DMSO-<i>d</i><sub>6</sub>, 2.50 ppm (<sup>1</sup>H), 39.52 ppm (<sup>13</sup>C); acetone-<i>d</i><sub>6</sub>, 2.05 ppm (<sup>1</sup>H), 29.84 ppm (<sup>13</sup>C)). Data were reported in a format as follows: chemical shift, multiplicity (s = singlet, d = doublet, dd = doublet of doublet, t = triplet, q = quartet, br = broad, m = multiplet, abq = ab quartet), number of protons, and coupling constants. High resolution mass spectral data were measured in-house using a Shimadzu IT-TOF LC/MS for all final compounds. All compounds submitted for biological testing were confirmed to be ≥95% pure by analytical HPLC. Synthetic methods, spectral data, and HRMS for novel compounds are described in detail below.</div></div><div id="sec6_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> 7-Methoxy-1-methyl-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridine (<b>33</b>)</h3><div class="NLM_p last">To a solution of <b>32</b> (2.96 g, 20 mmol) in DMF (50 mL) at 0 °C were added 60% NaH (0.96 g, 24 mmol) and methyl iodide (1.37 mL, 22 mmol). The reaction mixture was stirred at room temperature (RT) for 1 h. Water was then added to quench the reaction. The solution was extracted with ethyl acetate, and the combined organic layer was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>. Filtration and concentration in vacuo gave <b>33</b> (3.21 g) without further purification.</div></div><div id="sec6_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> 3-Iodo-7-methoxy-1-methyl-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridine (<b>34</b>)</h3><div class="NLM_p last">To a solution of <b>33</b> (from previous step) in DMF (60 mL) at RT was added NIS (4.95 g, 22 mmol), and then the mixture was stirred at RT for 12 h. Water was added to quench the reaction. The solution was extracted with ethyl acetate, and the combined organic layer was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>. Filtration and concentration in vacuo gave a residue, which was purified by flash column chromatography on silica gel (hexanes–EtOAc, 3:1) to obtain <b>34</b> as a white solid (5.3 g, yield 92% over the two steps). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 7.76 (d, <i>J</i> = 5.7 Hz, 1H), 7.11 (s, 1H), 6.92 (d, <i>J</i> = 5.6 Hz, 1H), 4.08 (s, 3H), 4.06 (s, 3H). LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> = 289.0.</div></div><div id="sec6_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> 7-Methoxy-1-methyl-3-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridine (<b>35</b>)</h3><div class="NLM_p last">To a solution of <b>34</b> (2.88 g, 10 mmol) in THF (20 mL) was added <i>n</i>-BuLi (12 mmol) at −78 °C. The reaction mixture was stirred for 1 h, followed by addition of 2-isopropoxy-4,4,5,5-tetramethyl-1,3,2-dioxaborolane (2.24 mL, 11 mmol) at −78 °C. The reaction mixture was allowed to warm to room temperatue and stirred overnight. Upon the completion of the reaction monitored by TLC, water was added to quench the reaction. The solution was extracted with ethyl acetate, and the combined organic layer was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>. Filtration and concentration in vacuo gave a residue, which was purified by flash column chromatography on silica gel (hexanes–EtOAc, 10:1) to obtain 1.87 g of white solid <b>35</b> with a 65% yield. <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 7.75 (d, <i>J</i> = 5.6 Hz, 1H), 7.48 (d, <i>J</i> = 5.6 Hz, 1H), 7.44 (s, 1H), 4.07 (s, 3H), 4.04 (s, 3H), 1.34 (s, 12H). LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> = 289.2.</div></div><div id="sec6_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> <i>N</i>-(6-Bromo-1<i>H</i>-indol-4-yl)methanesulfonamide (<b>38a</b>)</h3><div class="NLM_p last">To a solution of <b>37</b> (4.22 g, 20 mmol) in pyridine (60 mL) at 0 °C was added MsCl (3.0 mL, 40 mmol). The reaction mixture was allowed to slowly warm to room temperatue. The solution was further stirred at RT for 2 h and then quenched by adding water. The mixture was extracted with ethyl acetate, and the combined organic layer was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>. And then filtration and concentration in vacuo gave a residue, which was purified by flash column chromatography on silica gel (hexanes–EtOAc, 1:1) to obtain <b>38a</b> (5.26 g, yield 91%) as a brown solid. <sup>1</sup>H NMR (400 MHz, acetone-<i>d</i><sub>6</sub>) δ 10.55 (s, 1H), 8.55 (s, 1H), 7.49 (dd, <i>J</i> = 1.6, 1.0 Hz, 1H), 7.38 (dd, <i>J</i> = 3.3, 2.4 Hz, 1H), 7.34 (d, <i>J</i> = 1.6 Hz, 1H), 6.88 (ddt, <i>J</i> = 3.2, 2.1, 1.3 Hz, 1H), 3.01 (d, <i>J</i> = 1.1 Hz, 3H). LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> = 289.1.</div></div><div id="sec6_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> <i>N</i>-(6-Bromo-1<i>H</i>-indol-4-yl)ethanesulfonamide (<b>38b</b>)</h3><div class="NLM_p last">This compound was obtained by a procedure similar to the preparation of <b>38a</b> (860 mg, yield 95%). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 8.33 (s, 1H), 7.41 (t, <i>J</i> = 1.3 Hz, 1H), 7.39 (d, <i>J</i> = 1.4 Hz, 1H), 7.22 (dd, <i>J</i> = 3.4, 2.4 Hz, 1H), 6.58 (s, 1H), 6.54 (dt, <i>J</i> = 2.9, 1.5 Hz, 1H), 3.19 (q, <i>J</i> = 7.4 Hz, 2H), 1.38 (t, <i>J</i> = 7.4 Hz, 3H). LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> = 303.0.</div></div><div id="sec6_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> <i>N</i>-(6-Bromo-1-(4-fluorobenzyl)-1<i>H</i>-indol-4-yl)methanesulfonamide (<b>39a</b>)</h3><div class="NLM_p last">To a solution of <b>38a</b> (4.34 g, 15 mmol) in DMF (50 mL) at RT were added K<sub>2</sub>CO<sub>3</sub> (3.11 g, 22.5 mmol) and <b>52</b> (2.24 mL, 18 mmol). The solution was stirred at 50 °C for 6 h, followed by quenching in water. The mixture was then extracted with ethyl acetate, and the combined organic layer was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>. Filtration and concentration in vacuo gave a residue, which was purified by flash column chromatography on silica gel (hexanes–EtOAc, 2:1) to obtain <b>39a</b> as a yellow solid (5.48 g, yield 92%). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 7.37 (d, <i>J</i> = 1.4 Hz, 1H), 7.29 (t, <i>J</i> = 1.1 Hz, 1H), 7.11 (d, <i>J</i> = 3.3 Hz, 1H), 7.08 (dd, <i>J</i> = 8.6, 5.4 Hz, 2H), 7.05–6.98 (m, 2H), 6.65 (s, 1H), 6.54 (dd, <i>J</i> = 3.3, 0.9 Hz, 1H), 5.24 (s, 2H), 3.07 (s, 3H). LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> = 397.2.</div></div><div id="sec6_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> <i>N</i>-(6-Bromo-1-(4-fluorobenzyl)-1<i>H</i>-indol-4-yl)ethanesulfonamide (<b>39b</b>)</h3><div class="NLM_p last">This compound was obtained by a procedure similar to the preparation of <b>39a</b> (221 mg, yield 90%). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 8.40 (s, 1H), 7.42 (t, <i>J</i> = 1.2 Hz, 1H), 7.21–7.15 (m, 2H), 7.12 (dd, <i>J</i> = 3.3, 2.4 Hz, 1H), 7.09 (d, <i>J</i> = 1.5 Hz, 1H), 6.93–6.87 (m, 2H), 6.43 (ddd, <i>J</i> = 3.2, 2.1, 1.0 Hz, 1H), 4.89 (s, 2H), 3.13 (q, <i>J</i> = 7.4 Hz, 2H), 1.43 (t, <i>J</i> = 7.4 Hz, 3H). LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> = 411.0.</div></div><div id="sec6_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> <i>N</i>-(6-Bromo-1-(2,6-dimethylbenzyl)-1<i>H</i>-indol-4-yl)ethanesulfonamide (<b>39c</b>)</h3><div class="NLM_p last">This compound was obtained by a procedure similar to the preparation of <b>39a</b> (178 mg, yield 76%). <sup>1</sup>H NMR (400 MHz, acetone-<i>d</i><sub>6</sub>) δ 7.55 (dd, <i>J</i> = 1.6, 0.9 Hz, 1H), 7.31 (dd, <i>J</i> = 3.2, 2.5 Hz, 1H), 7.21 (d, <i>J</i> = 1.6 Hz, 1H), 6.89 (dd, <i>J</i> = 8.3, 6.6 Hz, 1H), 6.81 (d, <i>J</i> = 7.5 Hz, 2H), 6.55 (ddd, <i>J</i> = 3.1, 2.1, 0.9 Hz, 1H), 5.14 (s, 2H), 3.26 (q, <i>J</i> = 7.4 Hz, 2H), 2.27 (s, 6H), 1.37 (t, <i>J</i> = 7.4 Hz, 3H). LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> = 421.1.</div></div><div id="sec6_20" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> <i>N</i>-(6-Bromo-1-(2,6-dimethylbenzyl)-1<i>H</i>-indol-4-yl)methanesulfonamide (<b>39d</b>)</h3><div class="NLM_p last">This compound was obtained by a procedure similar to the preparation of <b>39a</b> (95 mg, yield 75%). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 7.28–7.23 (m, 1H), 7.20–7.14 (m, 3H), 6.60 (d, <i>J</i> = 1.5 Hz, 1H), 6.49 (d, <i>J</i> = 3.3 Hz, 1H), 6.30 (dd, <i>J</i> = 3.3, 0.9 Hz, 1H), 5.16 (s, 2H), 3.99 (s, 3H), 2.29 (s, 6H). LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> = 407.0.</div></div><div id="sec6_21" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> <i>N</i>-(6-Bromo-1-(4-fluoro-2-methylbenzyl)-1<i>H</i>-indol-4-yl)methanesulfonamide (<b>39e</b>)</h3><div class="NLM_p last">This compound was obtained by a procedure similar to the preparation of <b>39a</b> (113 mg, yield 84%). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 7.38 (dd, <i>J</i> = 3.4, 1.5 Hz, 1H), 7.30 (t, <i>J</i> = 1.2 Hz, 1H), 6.95 (m, 2H), 6.83 (td, <i>J</i> = 8.4, 2.7 Hz, 1H), 6.78–6.71 (m, 2H), 6.53 (dd, <i>J</i> = 3.4, 0.9 Hz, 1H), 5.18 (s, 2H), 3.08 (s, 3H), 2.27 (s, 3H). LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> = 411.0.</div></div><div id="sec6_22" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> <i>N</i>-(6-Bromo-1-(3-fluorobenzyl)-1<i>H</i>-indol-4-yl)methanesulfonamide (<b>39f</b>)</h3><div class="NLM_p last">This compound was obtained by a procedure similar to the preparation of <b>39a</b> (89 mg, yield 92%). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 8.53 (s, 1H), 7.34 (t, <i>J</i> = 1.2 Hz, 1H), 7.18 (td, <i>J</i> = 7.9, 5.8 Hz, 1H), 7.11 (d, <i>J</i> = 1.5 Hz, 1H), 7.06 (dd, <i>J</i> = 3.3, 2.4 Hz, 1H), 7.02 (dt, <i>J</i> = 7.7, 1.2 Hz, 1H), 6.98 (dt, <i>J</i> = 9.5, 2.2 Hz, 1H), 6.89 (tdd, <i>J</i> = 8.5, 2.7, 1.0 Hz, 1H), 6.43 (ddd, <i>J</i> = 3.2, 2.0, 1.0 Hz, 1H), 4.90 (s, 2H), 3.01 (s, 3H). LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> = 397.1.</div></div><div id="sec6_23" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> <i>N</i>-(6-Bromo-1-((3-methyloxetan-3-yl)methyl)-1<i>H</i>-indol-4-yl)methanesulfonamide (<b>39g</b>)</h3><div class="NLM_p last">This compound was obtained by a procedure similar to the preparation of <b>39a</b> (86 mg, yield 93%). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 7.37 (s, 2H), 7.25 (s, 1H), 7.06 (d, <i>J</i> = 3.3 Hz, 1H), 6.62 (d, <i>J</i> = 3.2 Hz, 1H), 4.63 (d, <i>J</i> = 6.1 Hz, 2H), 4.41 (d, <i>J</i> = 6.1 Hz, 2H), 4.31 (s, 2H), 3.06 (s, 3H), 1.30 (s, 3H). LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> = 373.1.</div></div><div id="sec6_24" class="NLM_sec NLM_sec_level_2"><div id="ac_i42" class="anchor-spacer"></div><h3 class="article-section__title" id="_i42"> <i>N</i>-(6-Bromo-1-((3,3-difluorocyclobutyl)methyl)-1<i>H</i>-indol-4-yl)methanesulfonamide (<b>39h</b>)</h3><div class="NLM_p last">This compound was obtained by a procedure similar to the preparation of <b>39a</b> (158 mg, yield 96%). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 7.52 (s, 1H), 7.37 (d, <i>J</i> = 1.5 Hz, 1H), 7.34 (t, <i>J</i> = 1.1 Hz, 1H), 7.06 (d, <i>J</i> = 3.3 Hz, 1H), 6.67 (dd, <i>J</i> = 3.3, 0.9 Hz, 1H), 4.17 (d, <i>J</i> = 5.9 Hz, 2H), 3.07 (s, 3H), 2.74–2.60 (m, 3H), 2.42–2.26 (m, 2H). LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> = 392.9.</div></div><div id="sec6_25" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> <i>N</i>-(1-(4-Fluorobenzyl)-6-(7-methoxy-1-methyl-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridin-3-yl)-1<i>H</i>-indol-4-yl)methanesulfonamide (<b>40a</b>)</h3><div class="NLM_p last">A mixture of <b>39a</b> (4.77 g, 12 mmol), Pd(dppf)Cl<sub>2</sub>–DCM (490 mg, 0.6 mmol), <b>35</b> (4.16 g, 14.4 mmol), and K<sub>3</sub>PO<sub>4</sub> (6.62 g, 31.2 mmol) in dioxane–water (25 mL, V/V, 4:1) was heated at 90 °C for 24 h. The mixture was then diluted with ethyl acetate and filtered to obtain a clear solution, which was subsequently washed three times with brine. The organic layer was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated. The crude product was purified by flash column chromatography on silica gel (hexanes–acetone, 1:1) to afford <b>40a</b> as a gray solid (5.16 g, yield 90%). <sup>1</sup>H NMR (400 MHz, acetone-<i>d</i><sub>6</sub>) δ 8.47 (s, 1H), 7.67 (d, <i>J</i> = 5.8 Hz, 1H), 7.54–7.48 (m, 3H), 7.42 (d, <i>J</i> = 3.3 Hz, 1H), 7.33 (dd, <i>J</i> = 8.7, 5.6 Hz, 3H), 7.11 (t, <i>J</i> = 8.8 Hz, 2H), 6.94–6.88 (m, 1H), 5.51 (s, 2H), 4.11 (s, 3H), 4.05 (s, 3H), 3.02 (s, 3H). LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> = 479.2.</div></div><div id="sec6_26" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> <i>N</i>-(1-(4-Fluorobenzyl)-6-(7-methoxy-1-methyl-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridin-3-yl)-1<i>H</i>-indol-4-yl)ethanesulfonamide (<b>40b</b>)</h3><div class="NLM_p last">This compound was obtained by a procedure similar to the preparation of <b>40a</b> (102 mg, yield 94%). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 8.20 (d, <i>J</i> = 0.9 Hz, 1H), 7.77 (d, <i>J</i> = 5.8 Hz, 1H), 7.57 (s, 1H), 7.46 (d, <i>J</i> = 1.0 Hz, 1H), 7.27 (d, <i>J</i> = 1.5 Hz, 2H), 7.25–7.20 (m, 3H), 7.04–6.98 (m, 2H), 5.57 (s, 2H), 4.11 (s, 3H), 4.10 (s, 3H), 3.25 (q, <i>J</i> = 7.4 Hz, 2H), 1.41 (t, <i>J</i> = 7.4 Hz, 3H). LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> = 493.1.</div></div><div id="sec6_27" class="NLM_sec NLM_sec_level_2"><div id="ac_i45" class="anchor-spacer"></div><h3 class="article-section__title" id="_i45"> <i>N</i>-(1-(2,6-Dimethylbenzyl)-6-(7-methoxy-1-methyl-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridin-3-yl)-1<i>H</i>-indol-4-yl)methanesulfonamide (<b>40c</b>)</h3><div class="NLM_p last">This compound was obtained by a procedure similar to the preparation of <b>40a</b> (98 mg, yield 83%). <sup>1</sup>H NMR (400 MHz, acetone-<i>d</i><sub>6</sub>) δ 7.71 (d, <i>J</i> = 5.7 Hz, 1H), 7.59 (dd, <i>J</i> = 3.8, 1.1 Hz, 2H), 7.57 (s, 1H), 7.42 (d, <i>J</i> = 5.7 Hz, 1H), 7.23 (dd, <i>J</i> = 8.4, 6.5 Hz, 1H), 7.16 (d, <i>J</i> = 7.5 Hz, 2H), 6.83 (d, <i>J</i> = 3.3 Hz, 1H), 6.78 (d, <i>J</i> = 3.3 Hz, 1H), 5.45 (s, 2H), 4.15 (s, 3H), 4.06 (d, <i>J</i> = 1.8 Hz, 3H), 3.13 (q, <i>J</i> = 7.4 Hz, 2H), 2.30 (s, 6H), 1.30 (t, <i>J</i> = 7.4 Hz, 3H). LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> = 503.2.</div></div><div id="sec6_28" class="NLM_sec NLM_sec_level_2"><div id="ac_i46" class="anchor-spacer"></div><h3 class="article-section__title" id="_i46"> <i>N</i>-(1-(2,6-Dimethylbenzyl)-6-(7-methoxy-1-methyl-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridin-3-yl)-1<i>H</i>-indol-4-yl)methanesulfonamide (<b>40d</b>)</h3><div class="NLM_p last">This compound was obtained by a procedure similar to the preparation of <b>40a</b> (91 mg, yield 90%). <sup>1</sup>H NMR (400 MHz, acetone-<i>d</i><sub>6</sub>) δ 8.44 (s, 1H), 7.71 (dd, <i>J</i> = 5.7, 1.3 Hz, 1H), 7.62 (d, <i>J</i> = 1.2 Hz, 1H), 7.58 (s, 1H), 7.55 (d, <i>J</i> = 1.3 Hz, 1H), 7.41 (d, <i>J</i> = 5.7 Hz, 1H), 7.25–7.21 (m, 1H), 7.17 (s, 1H), 7.15 (s, 1H), 6.79 (s, 2H), 5.46 (s, 2H), 4.15 (d, <i>J</i> = 1.5 Hz, 3H), 4.07 (s, 3H), 3.01 (s, 3H), 2.30 (d, <i>J</i> = 2.2 Hz, 6H). LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> = 489.1.</div></div><div id="sec6_29" class="NLM_sec NLM_sec_level_2"><div id="ac_i47" class="anchor-spacer"></div><h3 class="article-section__title" id="_i47"> <i>N</i>-(1-(4-Fluoro-2-methylbenzyl)-6-(7-methoxy-1-methyl-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridin-3-yl)-1<i>H</i>-indol-4-yl)methanesulfonamide (<b>40e</b>)</h3><div class="NLM_p last">This compound was obtained by a procedure similar to the preparation of <b>40a</b> (75 mg, yield 77%). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 7.74 (d, <i>J</i> = 5.7 Hz, 1H), 7.49 (d, <i>J</i> = 1.2 Hz, 1H), 7.31 (d, <i>J</i> = 1.0 Hz, 1H), 7.24 (d, <i>J</i> = 5.8 Hz, 1H), 7.22 (s, 1H), 7.00 (d, <i>J</i> = 3.3 Hz, 1H), 6.94 (dd, <i>J</i> = 9.5, 2.4 Hz, 1H), 6.87–6.82 (m, 2H), 6.63 (t, <i>J</i> = 2.5 Hz, 1H), 5.25 (s, 2H), 4.09 (s, 3H), 4.07 (s, 3H), 3.08 (s, 3H), 2.28 (s, 3H). LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> = 493.1.</div></div><div id="sec6_30" class="NLM_sec NLM_sec_level_2"><div id="ac_i48" class="anchor-spacer"></div><h3 class="article-section__title" id="_i48"> <i>N</i>-(1-(3-Fluorobenzyl)-6-(7-methoxy-1-methyl-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridin-3-yl)-1<i>H</i>-indol-4-yl)methanesulfonamide (<b>40f</b>)</h3><div class="NLM_p last">This compound was obtained by a procedure similar to the preparation of <b>40a</b> (104 mg, yield 84%). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 8.96 (t, <i>J</i> = 2.3 Hz, 1H), 7.71 (d, <i>J</i> = 5.7 Hz, 1H), 7.39 (t, <i>J</i> = 1.1 Hz, 1H), 7.21 (d, <i>J</i> = 1.3 Hz, 1H), 7.20–7.14 (m, 3H), 7.11–7.03 (m, 2H), 7.00 (s, 1H), 6.91–6.85 (m, 1H), 6.50 (ddd, <i>J</i> = 3.1, 2.0, 1.0 Hz, 1H), 4.99 (s, 2H), 4.08 (s, 3H), 3.96 (s, 3H), 3.03 (s, 3H). LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> = 479.1.</div></div><div id="sec6_31" class="NLM_sec NLM_sec_level_2"><div id="ac_i49" class="anchor-spacer"></div><h3 class="article-section__title" id="_i49"> <i>N</i>-(6-(7-Methoxy-1-methyl-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridin-3-yl)-1-((3-methyloxetan-3-yl)methyl)-1<i>H</i>-indol-4-yl)methanesulfonamide (<b>40g</b>)</h3><div class="NLM_p last">This compound was obtained by a procedure similar to the preparation of <b>40a</b> (129 mg, yield 90%). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 7.78 (d, <i>J</i> = 5.7 Hz, 1H), 7.44 (d, <i>J</i> = 22.0 Hz, 2H), 7.39 (d, <i>J</i> = 5.7 Hz, 1H), 7.09 (d, <i>J</i> = 3.3 Hz, 1H), 6.72 (s, 1H), 6.56 (d, <i>J</i> = 3.3 Hz, 1H), 4.70 (d, <i>J</i> = 6.1 Hz, 2H), 4.44 (d, <i>J</i> = 6.1 Hz, 2H), 4.40 (s, 2H), 4.11 (s, 3H), 4.10 (s, 3H), 3.06 (s, 3H), 1.35 (s, 3H). LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> = 455.2.</div></div><div id="sec6_32" class="NLM_sec NLM_sec_level_2"><div id="ac_i50" class="anchor-spacer"></div><h3 class="article-section__title" id="_i50"> <i>N</i>-(1-((3,3-Difluorocyclobutyl)methyl)-6-(7-methoxy-1-methyl-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridin-3-yl)-1<i>H</i>-indol-4-yl)methanesulfonamide (<b>40h</b>)</h3><div class="NLM_p last">This compound was obtained by a procedure similar to the preparation of <b>40a</b> (95 mg, yield 86%). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 7.78 (d, <i>J</i> = 5.8 Hz, 1H), 7.59 (s, 1H), 7.53–7.49 (m, 1H), 7.43 (d, <i>J</i> = 5.8 Hz, 1H), 7.37 (s, 1H), 7.06 (d, <i>J</i> = 3.2 Hz, 1H), 6.68 (d, <i>J</i> = 3.2 Hz, 1H), 4.22 (d, <i>J</i> = 6.5 Hz, 2H), 4.10 (s, 3H), 4.06 (s, 3H), 3.06 (s, 3H), 2.75–2.63 (m, 3H), 2.41–2.29 (m, 2H). LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> = 475.2.</div></div><div id="sec6_33" class="NLM_sec NLM_sec_level_2"><div id="ac_i51" class="anchor-spacer"></div><h3 class="article-section__title" id="_i51"> <i>N</i>-(1-(4-Fluorobenzyl)-6-(1-methyl-7-oxo-6,7-dihydro-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridin-3-yl)-1<i>H</i>-indol-4-yl)methanesulfonamide (<b>10</b>)</h3><div class="NLM_p last">A mixture of <b>40a</b> (478 mg, 1 mmol), TMSCl (203 μL, 1.6 mmol), NaI (240 mg, 1.6 mmol), and H<sub>2</sub>O (9 μL, 0.5 mmol) in acetonitrile (6 mL) was heated at 50 °C for 3 h. The reaction mixture was then cooled to RT, diluted with ethyl acetate, and washed three times with aqueous NaHCO<sub>3</sub>. The organic layer was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated. The crude product was purified by preparative-HPLC system to obtain the final product <b>10</b> (408 mg, yield 88%). <sup>1</sup>H NMR (400 MHz, acetone-<i>d</i><sub>6</sub>) δ 10.07 (s, 1H), 8.51 (s, 1H), 7.45 (d, <i>J</i> = 1.2 Hz, 1H), 7.42 (t, <i>J</i> = 3.5 Hz, 3H), 7.33 (dd, <i>J</i> = 8.5, 5.5 Hz, 2H), 7.14–7.07 (m, 2H), 6.98 (d, <i>J</i> = 7.1 Hz, 1H), 6.90 (d, <i>J</i> = 3.2 Hz, 1H), 6.64 (d, <i>J</i> = 7.1 Hz, 1H), 5.51 (s, 2H), 4.19 (s, 3H), 3.01 (s, 3H). <sup>13</sup>C NMR (101 MHz, acetone-<i>d</i><sub>6</sub>) δ 163.0 (d, <i>J</i> = 243.9 Hz), 157.1, 138.7, 135.2 (d, <i>J</i> = 3.2 Hz), 131.3, 130.1, 130.0 (d, <i>J</i> = 8.2 Hz), 129.8, 129.5, 125.8, 124.8, 122.3, 119.2, 116.2 (d, <i>J</i> = 21.6 Hz), 113.4, 106.5, 100.8, 99.9, 50.0, 39.7, 35.8. HRMS (ESI) calcd for C<sub>24</sub>H<sub>22</sub>FN<sub>4</sub>O<sub>3</sub>S [M + H]<sup>+</sup>, 465.1391; found, 465.1377.</div></div><div id="sec6_34" class="NLM_sec NLM_sec_level_2"><div id="ac_i52" class="anchor-spacer"></div><h3 class="article-section__title" id="_i52"> <i>N</i>-(1-(4-Fluorobenzyl)-6-(1-methyl-7-oxo-6,7-dihydro-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridin-3-yl)-1<i>H</i>-indol-4-yl)ethanesulfonamide (<b>11</b>)</h3><div class="NLM_p last">This compound was obtained by a procedure similar to the preparation of <b>10</b> (52 mg, yield 89%). <sup>1</sup>H NMR (400 MHz, acetone-<i>d</i><sub>6</sub>) δ 10.85 (s, 1H), 8.58 (s, 1H), 7.49 (s, 2H), 7.45–7.38 (m, 2H), 7.36–7.26 (m, 2H), 7.15–7.04 (m, 3H), 6.96 (d, <i>J</i> = 3.0 Hz, 1H), 6.77 (d, <i>J</i> = 7.0 Hz, 1H), 5.47 (s, 2H), 4.20 (s, 3H), 3.14 (q, <i>J</i> = 7.3 Hz, 2H), 1.31 (t, <i>J</i> = 7.4 Hz, 3H). <sup>13</sup>C NMR (101 MHz, acetone-<i>d</i><sub>6</sub>) δ 163.0 (d, <i>J</i> = 244.1 Hz), 157.0, 138.7, 135.2 (d, <i>J</i> = 3.3 Hz), 131.4, 130.9, 130.7, 130.0 (d, <i>J</i> = 8.2 Hz), 129.8, 129.4, 125.7, 124.3, 121.9, 119.4, 116.2 (d, <i>J</i> = 21.8 Hz), 112.4, 106.2, 102.1, 99.8, 50.0, 46.4, 36.0, 8.7. HRMS (ESI) calcd for C<sub>25</sub>H<sub>24</sub>FN<sub>4</sub>O<sub>3</sub>S [M + H]<sup>+</sup>, 479.1548; found, 479.1525.</div></div><div id="sec6_35" class="NLM_sec NLM_sec_level_2"><div id="ac_i53" class="anchor-spacer"></div><h3 class="article-section__title" id="_i53"> <i>N</i>-(1-(2,6-Dimethylbenzyl)-6-(1-methyl-7-oxo-6,7-dihydro-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridin-3-yl)-1<i>H</i>-indol-4-yl)ethanesulfonamide (<b>19</b>)</h3><div class="NLM_p last">This compound was obtained by a procedure similar to the preparation of <b>10</b> (35 mg, yield 94%). <sup>1</sup>H NMR (400 MHz, acetone-<i>d</i><sub>6</sub>) δ 10.18 (s, 1H), 8.48 (s, 1H), 7.55–7.49 (m, 2H), 7.47 (s, 1H), 7.23 (dd, <i>J</i> = 8.5, 6.5 Hz, 1H), 7.16 (s, 1H), 7.14 (s, 1H), 7.02 (d, <i>J</i> = 7.0 Hz, 1H), 6.83 (d, <i>J</i> = 3.2 Hz, 1H), 6.77 (d, <i>J</i> = 3.3 Hz, 1H), 6.74 (d, <i>J</i> = 7.0 Hz, 1H), 5.44 (s, 2H), 4.22 (s, 3H), 3.13 (q, <i>J</i> = 7.4 Hz, 2H), 2.29 (s, 6H), 1.30 (t, <i>J</i> = 7.3 Hz, 3H). <sup>13</sup>C NMR (101 MHz, acetone-<i>d</i><sub>6</sub>) δ 155.3, 139.1, 138.9, 133.5, 131.3, 129.9, 129.5, 129.2, 127.3, 127.2, 125.8, 124.9, 121.6, 119.5, 112.5, 112.4, 105.8, 101.1, 99.3, 46.4, 45.2, 35.9, 20.0, 8.7. HRMS (ESI) calcd for C<sub>27</sub>H<sub>29</sub>N<sub>4</sub>O<sub>3</sub>S [M + H]<sup>+</sup>, 489.1955; found, 489.1954.</div></div><div id="sec6_36" class="NLM_sec NLM_sec_level_2"><div id="ac_i54" class="anchor-spacer"></div><h3 class="article-section__title" id="_i54"> <i>N</i>-(1-(2,6-Dimethylbenzyl)-6-(1-methyl-7-oxo-6,7-dihydro-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridin-3-yl)-1<i>H</i>-indol-4-yl)methanesulfonamide (<b>20</b>)</h3><div class="NLM_p last">This compound was obtained by a procedure similar to the preparation of <b>10</b> (22 mg, yield 90%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.96 (d, <i>J</i> = 5.6 Hz, 1H), 9.58 (s, 1H), 7.56 (s, 1H), 7.43 (s, 1H), 7.25–7.17 (m, 2H), 7.12 (d, <i>J</i> = 7.5 Hz, 2H), 6.94–6.88 (m, 1H), 6.75 (dd, <i>J</i> = 28.1, 3.2 Hz, 2H), 6.56 (d, <i>J</i> = 7.0 Hz, 1H), 5.36 (s, 2H), 4.13 (s, 3H), 2.97 (s, 3H), 2.22 (s, 6H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 155.9, 137.7, 137.6, 132.6, 130.0, 129.3, 128.5, 128.3, 128.3, 128.1, 126.7, 125.3, 123.4, 120.9, 117.5, 112.2, 105.0, 99.6, 99.3, 44.1, 35.4, 19.5. HRMS (ESI) calcd for C<sub>26</sub>H<sub>27</sub>N<sub>4</sub>O<sub>3</sub>S [M + H]<sup>+</sup>, 475.1798; found, 475.1792.</div></div><div id="sec6_37" class="NLM_sec NLM_sec_level_2"><div id="ac_i55" class="anchor-spacer"></div><h3 class="article-section__title" id="_i55"> <i>N</i>-(1-(4-Fluoro-2-methylbenzyl)-6-(1-methyl-7-oxo-6,7-dihydro-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridin-3-yl)-1<i>H</i>-indol-4-yl)methanesulfonamide (<b>21</b>)</h3><div class="NLM_p last">This compound was obtained by a procedure similar to the preparation of <b>10</b> (43 mg, yield 92%). <sup>1</sup>H NMR (400 MHz, acetone-<i>d</i><sub>6</sub>) δ 10.51 (s, 1H), 8.54 (s, 1H), 7.47 (d, <i>J</i> = 1.3 Hz, 1H), 7.45 (s, 1H), 7.38 (t, <i>J</i> = 1.0 Hz, 1H), 7.27 (d, <i>J</i> = 3.3 Hz, 1H), 7.08–7.00 (m, 2H), 6.92 (dd, <i>J</i> = 3.3, 0.9 Hz, 1H), 6.92–6.81 (m, 2H), 6.73 (d, <i>J</i> = 7.0 Hz, 1H), 5.48 (s, 2H), 4.19 (s, 3H), 3.03 (s, 3H), 2.36 (s, 3H). <sup>13</sup>C NMR (101 MHz, acetone-<i>d</i><sub>6</sub>) δ 163.0 (d, <i>J</i> = 243.8 Hz), 157.0, 139.7 (d, <i>J</i> = 8.1 Hz), 139.0, 132.8 (d, <i>J</i> = 2.9 Hz), 131.3, 130.5, 130.3, 130.0 (d, <i>J</i> = 8.7 Hz), 129.9, 129.4, 125.8, 124.5, 122.2, 119.3, 117.8 (d, <i>J</i> = 21.6 Hz), 113.5 (d, <i>J</i> = 21.2 Hz), 113.3, 106.4, 101.6, 100.0, 48.4, 39.8, 35.9, 19.2. HRMS (ESI) calcd for C<sub>25</sub>H<sub>24</sub>FN<sub>4</sub>O<sub>3</sub>S [M + H]<sup>+</sup>, 479.1548; found, 479.1515.</div></div><div id="sec6_38" class="NLM_sec NLM_sec_level_2"><div id="ac_i56" class="anchor-spacer"></div><h3 class="article-section__title" id="_i56"> <i>N</i>-(1-(3-Fluorobenzyl)-6-(1-methyl-7-oxo-6,7-dihydro-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridin-3-yl)-1<i>H</i>-indol-4-yl)methanesulfonamide (<b>22</b>)</h3><div class="NLM_p last">This compound was obtained by a procedure similar to the preparation of <b>10</b> (45 mg, yield 97%). <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 10.73 (s, 1H), 8.10 (s, 1H), 7.46 (t, <i>J</i> = 1.1 Hz, 1H), 7.31 (dd, <i>J</i> = 3.0, 1.6 Hz, 1H), 7.26 (s, 1H), 7.15 (td, <i>J</i> = 8.2, 6.1 Hz, 1H), 7.09 (d, <i>J</i> = 1.3 Hz, 1H), 7.07–6.99 (m, 2H), 6.91–6.84 (m, 2H), 6.59 (d, <i>J</i> = 7.0 Hz, 1H), 6.56 (dt, <i>J</i> = 3.1, 1.2 Hz, 1H), 4.98 (s, 2H), 4.07 (s, 3H), 3.07 (s, 3H). <sup>13</sup>C NMR (101 MHz, acetone-<i>d</i><sub>6</sub>) δ 163.9 (d, <i>J</i> = 244.6 Hz), 156.4, 142.1 (d, <i>J</i> = 6.8 Hz), 138.8, 132.3, 132, 131.5 (d, <i>J</i> = 8.3 Hz), 129.8, 129.4, 125.6, 123.8 (d, <i>J</i> = 2.3 Hz), 123.6, 122.4, 119.5, 115.1 (d, <i>J</i> = 21.3 Hz), 114.7 (d, <i>J</i> = 22.0 Hz), 113.4, 106.6, 103.8, 103.7, 100.1, 50.2, 39.8, 36.2. HRMS (ESI) calcd for C<sub>24</sub>H<sub>22</sub>FN<sub>4</sub>O<sub>3</sub>S [M + H]<sup>+</sup>, 465.1391; found, 465.1373.</div></div><div id="sec6_39" class="NLM_sec NLM_sec_level_2"><div id="ac_i57" class="anchor-spacer"></div><h3 class="article-section__title" id="_i57"> <i>N</i>-(6-(1-Methyl-7-oxo-6,7-dihydro-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridin-3-yl)-1-((3-methyloxetan-3-yl)methyl)-1<i>H</i>-indol-4-yl)methanesulfonamide (<b>23</b>)</h3><div class="NLM_p last">This compound was obtained by a procedure similar to the preparation of 10 (30 mg, yield 92%). <sup>1</sup>H NMR (400 MHz, acetone-<i>d</i><sub>6</sub>) δ 10.81 (s, br, 1H), 8.49 (s, 1H), 7.68 (s, 1H), 7.55 (s, 1H), 7.46 (d, <i>J</i> = 1.1 Hz, 1H), 7.37 (d, <i>J</i> = 3.3 Hz, 1H), 7.10 (d, <i>J</i> = 7.0 Hz, 1H), 7.01 (d, <i>J</i> = 7.0 Hz, 1H), 6.90 (d, <i>J</i> = 2.8 Hz, 1H), 4.39–4.28 (m, 2H), 4.22 (s, 3H), 3.52–3.46 (m, 3H), 3.40 (d, <i>J</i> = 10.0 Hz, 1H), 3.01 (s, 3H), 1.03 (s, 3H). <sup>13</sup>C NMR (101 MHz, acetone-<i>d</i><sub>6</sub>) δ 157.4, 140.2, 131.04, 130.98, 130.8, 130.5, 130.4, 129.7, 125.7, 121.8, 119.5, 113.3, 106.8, 102.3, 99.9, 65.9, 50.0, 42.0, 39.8, 36.0, 21.7. HRMS (ESI) calcd for C<sub>22</sub>H<sub>25</sub>N<sub>4</sub>O<sub>4</sub>S [M + H]<sup>+</sup>, 441.1591; found, 441.1632.</div></div><div id="sec6_40" class="NLM_sec NLM_sec_level_2"><div id="ac_i58" class="anchor-spacer"></div><h3 class="article-section__title" id="_i58"> <i>N</i>-(1-((3,3-Difluorocyclobutyl)methyl)-6-(1-methyl-7-oxo-6,7-dihydro-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridin-3-yl)-1<i>H</i>-indol-4-yl)methanesulfonamide (<b>24</b>)</h3><div class="NLM_p last">This compound was obtained by a procedure similar to the preparation of <b>10</b> (41 mg, yield 89%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.97 (d, <i>J</i> = 5.6 Hz, 1H), 9.61 (d, <i>J</i> = 1.7 Hz, 1H), 7.59 (s, 1H), 7.44 (s, 1H), 7.40 (d, <i>J</i> = 3.2 Hz, 1H), 7.24 (d, <i>J</i> = 1.2 Hz, 1H), 6.93 (dd, <i>J</i> = 7.0, 5.7 Hz, 1H), 6.76 (d, <i>J</i> = 3.2 Hz, 1H), 6.71 (d, <i>J</i> = 6.9 Hz, 1H), 4.32 (d, <i>J</i> = 6.7 Hz, 2H), 4.13 (s, 3H), 2.98 (s, 3H), 2.74–2.55 (m, 3H), 2.48–2.35 (m, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 155.9, 137.4, 130.0, 129.4, 128.4, 128.3, 125.4, 123.3, 120.8, 120.4 (dd, <i>J</i> = 283.8, 273.4 Hz), 117.5, 112.0, 104.8, 99.6, 99.3, 49.5, 38.2 (t, <i>J</i> = 22.0 Hz), 35.4, 23.2 (dd, <i>J</i> = 12.6, 5.9 Hz). HRMS (ESI) calcd for C<sub>22</sub>H<sub>23</sub>F<sub>2</sub>N<sub>4</sub>O<sub>3</sub>S [M + H]<sup>+</sup>, 461.1453; found, 461.1462.</div></div><div id="sec6_41" class="NLM_sec NLM_sec_level_2"><div id="ac_i59" class="anchor-spacer"></div><h3 class="article-section__title" id="_i59"> 7-Bromo-1-(4-fluorobenzyl)-5-nitro-1<i>H</i>-indole (<b>42a</b>)</h3><div class="NLM_p last">To a solution of <b>41</b> (1.21 g, 5 mmol) in DMF (20 mL) at RT were added K<sub>2</sub>CO<sub>3</sub> (968 mg, 7 mmol) and <b>52</b> (0.75 mL, 6 mmol). The mixture was then stirred at RT for 2 h, followed by quenching in water. The resulting solution was extracted with ethyl acetate, and the combined organic layer was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>. Filtration and concentration in vacuo gave a residue, which was purified by flash column chromatography on silica gel (hexanes–EtOAc, 4:1) to obtain <b>42a</b> (1.53 g, yield 88%) as a pale yellow solid. <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 8.53 (d, <i>J</i> = 2.1 Hz, 1H), 8.28 (d, <i>J</i> = 2.1 Hz, 1H), 7.24 (d, <i>J</i> = 3.3 Hz, 1H), 7.02–6.96 (m, 4H), 6.78 (d, <i>J</i> = 3.3 Hz, 1H), 5.83 (s, 2H). LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> = 349.1.</div></div><div id="sec6_42" class="NLM_sec NLM_sec_level_2"><div id="ac_i60" class="anchor-spacer"></div><h3 class="article-section__title" id="_i60"> 7-Bromo-1-(4-fluoro-2-methylbenzyl)-5-nitro-1<i>H</i>-indole (<b>42b</b>)</h3><div class="NLM_p last">This compound was obtained by a procedure similar to the preparation of <b>42a</b> (430 mg, yield 96%). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 8.54 (t, <i>J</i> = 2.2 Hz, 1H), 8.28 (d, <i>J</i> = 2.7 Hz, 1H), 7.14 (d, <i>J</i> = 3.2 Hz, 1H), 6.95 (d, <i>J</i> = 9.5 Hz, 1H), 6.77 (q, <i>J</i> = 3.4, 2.7 Hz, 2H), 6.40 (d, <i>J</i> = 6.9 Hz, 1H), 5.76 (s, 2H), 2.33 (s, 3H). LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> = 363.1.</div></div><div id="sec6_43" class="NLM_sec NLM_sec_level_2"><div id="ac_i61" class="anchor-spacer"></div><h3 class="article-section__title" id="_i61"> 7-Bromo-1-(4-fluorobenzyl)-1<i>H</i>-indol-5-amine (<b>43a</b>)</h3><div class="NLM_p last">To a solution of <b>42a</b> (1.04 g, 3 mmol) in EtOH (15 mL) at RT were added iron powder (1.34 g, 24 mmol) and saturated ammonia chloride (3 mL). The reaction was refluxed for 2 h and then cooled to room temperature. The mixture was filtered to obtain a clear solution, followed by extraction with ethyl acetate. Organic layers were combined and dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>. Then filtration and concentration in vacuo gave <b>43a</b> as a brown solid without further purification (940 mg). LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> = 319.1.</div></div><div id="sec6_44" class="NLM_sec NLM_sec_level_2"><div id="ac_i62" class="anchor-spacer"></div><h3 class="article-section__title" id="_i62"> 7-Bromo-1-(4-fluoro-2-methylbenzyl)-1<i>H</i>-indol-5-amine (<b>43b</b>)</h3><div class="NLM_p last">This compound was obtained by a procedure similar to the preparation of <b>43a</b> (160 mg, yield 66%). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 6.94–6.87 (m, 3H), 6.83 (d, <i>J</i> = 2.4 Hz, 1H), 6.72 (t, <i>J</i> = 8.7 Hz, 1H), 6.35 (q, <i>J</i> = 5.9, 4.5 Hz, 2H), 5.60 (s, 2H), 2.32 (d, <i>J</i> = 2.0 Hz, 3H). LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> = 333.1.</div></div><div id="sec6_45" class="NLM_sec NLM_sec_level_2"><div id="ac_i63" class="anchor-spacer"></div><h3 class="article-section__title" id="_i63"> 7-Bromo-1-(4-fluoro-2,6-dimethylbenzyl)-1<i>H</i>-indol-5-amine (<b>43c</b>)</h3><div class="NLM_p last">This compound was obtained by a procedure similar to the preparation of <b>43a</b> (129 mg, yield 90%). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 8.49 (d, <i>J</i> = 2.2 Hz, 1H), 8.35 (d, <i>J</i> = 2.2 Hz, 1H), 6.87 (d, <i>J</i> = 9.2 Hz, 2H), 6.67 (d, <i>J</i> = 3.4 Hz, 1H), 6.58 (d, <i>J</i> = 3.4 Hz, 1H), 5.87 (s, 2H), 2.28 (s, 6H). LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> = 347.1.</div></div><div id="sec6_46" class="NLM_sec NLM_sec_level_2"><div id="ac_i64" class="anchor-spacer"></div><h3 class="article-section__title" id="_i64"> <i>N</i>-(7-Bromo-1-(4-fluorobenzyl)-1<i>H</i>-indol-5-yl)methanesulfonamide (<b>44a</b>)</h3><div class="NLM_p last">To a solution of <b>43a</b> (957 mg, 3 mmol) in pyridine (10 mL) at 0 °C was added MsCl (0.675 mL, 9 mmol). The reaction mixture was allowed to warm to RT and stirred for 3 h. Water was then added to quench the reaction. The mixture was extracted with ethyl acetate, and the combined organic layer was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>. Filtration and concentration in vacuo gave a residue, which was purified by flash column chromatography on silica gel (hexanes–EtOAc, 1:1) to obtain <b>44a</b> (1.06 g, yield 89%) as a gray solid. <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 7.53 (d, <i>J</i> = 2.0 Hz, 1H), 7.27 (s, 2H), 7.15 (d, <i>J</i> = 3.2 Hz, 1H), 6.99 (s, 1H), 6.97 (s, 2H), 6.56 (d, <i>J</i> = 3.2 Hz, 1H), 5.76 (d, <i>J</i> = 1.1 Hz, 2H), 3.00 (s, 3H). LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> = 396.9.</div></div><div id="sec6_47" class="NLM_sec NLM_sec_level_2"><div id="ac_i65" class="anchor-spacer"></div><h3 class="article-section__title" id="_i65"> <i>N</i>-(7-Bromo-1-(4-fluoro-2-methylbenzyl)-1<i>H</i>-indol-5-yl)methanesulfonamide (<b>44b</b>)</h3><div class="NLM_p last">This compound was obtained by a procedure similar to the preparation of <b>44a</b> (118 mg, yield 83%). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 7.55 (d, <i>J</i> = 2.0 Hz, 1H), 7.27 (s, 1H), 7.04 (d, <i>J</i> = 3.3 Hz, 1H), 6.94 (dd, <i>J</i> = 9.5, 2.7 Hz, 1H), 6.75 (td, <i>J</i> = 8.4, 2.8 Hz, 1H), 6.55 (d, <i>J</i> = 3.2 Hz, 1H), 6.47 (s, 1H), 6.37 (dd, <i>J</i> = 8.6, 5.7 Hz, 1H), 5.69 (s, 2H), 3.01 (s, 3H), 2.34 (s, 3H). LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> = 411.1.</div></div><div id="sec6_48" class="NLM_sec NLM_sec_level_2"><div id="ac_i66" class="anchor-spacer"></div><h3 class="article-section__title" id="_i66"> <i>N</i>-(7-Bromo-1-(4-fluoro-2,6-dimethylbenzyl)-1<i>H</i>-indol-5-yl)methanesulfonamide (<b>44c</b>)</h3><div class="NLM_p last">This compound was obtained by a procedure similar to the preparation of <b>44a</b> (103 mg, yield 90%). <sup>1</sup>H NMR (400 MHz, acetone-<i>d</i><sub>6</sub>) δ 8.58 (d, <i>J</i> = 2.2 Hz, 1H), 8.28 (d, <i>J</i> = 2.2 Hz, 1H), 6.99 (d, <i>J</i> = 9.5 Hz, 2H), 6.86 (d, <i>J</i> = 3.4 Hz, 1H), 6.78 (d, <i>J</i> = 3.4 Hz, 1H), 6.00 (s, 2H), 2.80–2.76 (m, 4H), 2.32 (d, <i>J</i> = 0.8 Hz, 6H). LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> = 424.1.</div></div><div id="sec6_49" class="NLM_sec NLM_sec_level_2"><div id="ac_i67" class="anchor-spacer"></div><h3 class="article-section__title" id="_i67"> <i>N</i>-(1-(4-Fluorobenzyl)-7-(7-methoxy-1-methyl-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridin-3-yl)-1<i>H</i>-indol-5-yl)methanesulfonamide (<b>45a</b>)</h3><div class="NLM_p last">A mixture of <b>44a</b> (792 mg, 2 mmol), Pd(dppf)Cl<sub>2</sub>–DCM (82 mg, 0.1 mmol), <b>35</b> (694 mg, 2.4 mmol), and K<sub>3</sub>PO<sub>4</sub> (1.1 g, 5.2 mmol) in dioxane–water (15 mL, v/v, 4:1) was heated at 90 °C for 24 h. Upon completion of the reaction, the reaction mixture was then diluted with ethyl acetate and filtered to obtain a clear solution, which was subsequently washed three times with brine. The organic layer was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated. The crude product was purified by flash column chromatography on silica gel (hexanes–acetone, 1:1) to yield <b>45a</b> (832 mg, yield 87%) as a gray solid. <sup>1</sup>H NMR (400 MHz, acetone-<i>d</i><sub>6</sub>) δ 8.24 (s, 1H), 7.63 (d, <i>J</i> = 2.1 Hz, 1H), 7.58 (d, <i>J</i> = 5.7 Hz, 1H), 7.38 (d, <i>J</i> = 3.2 Hz, 1H), 7.01 (d, <i>J</i> = 2.2 Hz, 1H), 6.94 (s, 1H), 6.83–6.78 (m, 2H), 6.69 (d, <i>J</i> = 5.6 Hz, 1H), 6.63 (d, <i>J</i> = 3.2 Hz, 1H), 6.43–6.38 (m, 2H), 4.07 (s, 3H), 4.05 (s, 3H), 2.93 (d, <i>J</i> = 1.1 Hz, 3H). LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> = 479.2.</div></div><div id="sec6_50" class="NLM_sec NLM_sec_level_2"><div id="ac_i68" class="anchor-spacer"></div><h3 class="article-section__title" id="_i68"> <i>N</i>-(1-(4-Fluoro-2-methylbenzyl)-7-(7-methoxy-1-methyl-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridin-3-yl)-1<i>H</i>-indol-5-yl)methanesulfonamide (<b>45b</b>)</h3><div class="NLM_p last">This compound was obtained by a procedure similar to the preparation of <b>45a</b> (123 mg, yield 91%). <sup>1</sup>H NMR (400 MHz, acetone-<i>d</i><sub>6</sub>) δ 8.25 (s, 1H), 7.67 (d, <i>J</i> = 2.2 Hz, 1H), 7.57 (d, <i>J</i> = 5.6 Hz, 1H), 7.28 (d, <i>J</i> = 3.2 Hz, 1H), 7.00 (d, <i>J</i> = 2.2 Hz, 1H), 6.78 (dd, <i>J</i> = 9.8, 2.7 Hz, 1H), 6.72–6.66 (m, 3H), 6.65 (d, <i>J</i> = 3.2 Hz, 1H), 6.04 (dd, <i>J</i> = 8.6, 5.9 Hz, 1H), 4.91 (s, 2H), 4.06 (s, 3H), 3.93 (s, 3H), 2.94 (s, 3H), 1.62 (s, 3H). LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> = 493.1.</div></div><div id="sec6_51" class="NLM_sec NLM_sec_level_2"><div id="ac_i69" class="anchor-spacer"></div><h3 class="article-section__title" id="_i69"> <i>N</i>-(1-(4-Fluoro-2,6-dimethylbenzyl)-7-(7-methoxy-1-methyl-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridin-3-yl)-1<i>H</i>-indol-5-yl)methanesulfonamide (<b>45c</b>)</h3><div class="NLM_p last">This compound was obtained by a procedure similar to the preparation of <b>45a</b> (104 mg, yield 89%). <sup>1</sup>H NMR (400 MHz, acetone-<i>d</i><sub>6</sub>) δ 8.27 (s, 1H), 7.69 (d, <i>J</i> = 5.7 Hz, 1H), 7.60 (d, <i>J</i> = 2.2 Hz, 1H), 7.55 (s, 1H), 7.12 (d, <i>J</i> = 2.2 Hz, 1H), 6.95 (d, <i>J</i> = 5.6 Hz, 1H), 6.81 (d, <i>J</i> = 9.6 Hz, 2H), 6.52 (d, <i>J</i> = 3.3 Hz, 1H), 6.45 (d, <i>J</i> = 3.3 Hz, 1H), 4.18 (s, 3H), 4.06 (s, 3H), 2.95 (s, 3H), 2.02 (s, 6H). LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> = 507.2.</div></div><div id="sec6_52" class="NLM_sec NLM_sec_level_2"><div id="ac_i70" class="anchor-spacer"></div><h3 class="article-section__title" id="_i70"> <i>N</i>-(1-(4-Fluorobenzyl)-7-(1-methyl-7-oxo-6,7-dihydro-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridin-3-yl)-1<i>H</i>-indol-5-yl)methanesulfonamide (<b>12</b>)</h3><div class="NLM_p last">A mixture of <b>45a</b> (478 mg, 1 mmol), TMSCl (0.2 mL, 1.6 mmol), NaI (240 mg, 1.6 mmol), and H<sub>2</sub>O (9 μL, 0.5 mmol) in acetonitrile (8 mL) was heated at 50 °C for 2 h. The reaction was then cooled to room temperature. The reaction mixture was diluted with ethyl acetate and washed three times with aqueous NaHCO<sub>3</sub>. The organic layer was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated. The crude product was purified by preparative-LC system to obtain the final product <b>12</b> (417 mg) with a 90% yield. <sup>1</sup>H NMR (400 MHz, acetone-<i>d</i><sub>6</sub>) δ 10.18 (s, 1H), 8.30 (s, 1H), 7.62 (d, <i>J</i> = 2.1 Hz, 1H), 7.40 (d, <i>J</i> = 3.2 Hz, 1H), 6.98 (d, <i>J</i> = 2.2 Hz, 1H), 6.93–6.87 (m, 2H), 6.84 (t, <i>J</i> = 8.8 Hz, 2H), 6.63 (d, <i>J</i> = 3.2 Hz, 1H), 6.45 (dd, <i>J</i> = 8.4, 5.4 Hz, 2H), 6.00 (d, <i>J</i> = 6.8 Hz, 1H), 5.10 (s, 2H), 4.11 (s, 3H), 2.92 (s, 3H). <sup>13</sup>C NMR (101 MHz, acetone-<i>d</i><sub>6</sub>) δ 162.5 (d, <i>J</i> = 242.9 Hz), 143.3, 136.1 (d, <i>J</i> = 2.1 Hz), 133.2, 133.0, 132.0, 131.7, 131.2, 127.9 (d, <i>J</i> = 8.1 Hz), 126.0, 123.4, 122.2, 120.0, 115.6 (d, <i>J</i> = 21.7 Hz), 115.0, 114.7, 102.7, 100.7, 51.6, 38.8, 35.8. HRMS (ESI) calcd for C<sub>24</sub>H<sub>22</sub>FN<sub>4</sub>O<sub>3</sub>S [M + H]<sup>+</sup>, 465.1391; found, 465.1380.</div></div><div id="sec6_53" class="NLM_sec NLM_sec_level_2"><div id="ac_i71" class="anchor-spacer"></div><h3 class="article-section__title" id="_i71"> <i>N</i>-(1-(4-Fluoro-2-methylbenzyl)-7-(1-methyl-7-oxo-6,7-dihydro-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridin-3-yl)-1<i>H</i>-indol-5-yl)methanesulfonamide (<b>13</b>)</h3><div class="NLM_p last">This compound was obtained by a procedure similar to the preparation of <b>12</b> (55 mg, yield 95%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.84 (d, <i>J</i> = 5.6 Hz, 1H), 9.38 (s, 1H), 7.49 (d, <i>J</i> = 2.1 Hz, 1H), 7.38 (d, <i>J</i> = 3.1 Hz, 1H), 6.85 (dd, <i>J</i> = 10.0, 2.7 Hz, 1H), 6.75 (d, <i>J</i> = 2.1 Hz, 1H), 6.74–6.66 (m, 3H), 6.61 (d, <i>J</i> = 3.1 Hz, 1H), 5.86–5.78 (m, 2H), 4.91 (s, 2H), 3.90 (s, 3H), 2.92 (s, 3H), 1.70 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 160.7 (d, <i>J</i> = 241.7 Hz), 155.6, 136.1 (d, <i>J</i> = 7.9 Hz), 133.3 (d, <i>J</i> = 2.0 Hz), 132.3, 131.7, 131.2, 130.4, 130.1, 130.0, 125.8 (d, <i>J</i> = 8.7 Hz), 125.3, 121.8, 120.6, 118.5, 115.8 (d, <i>J</i> = 21.0 Hz), 113.0, 112.8, 112.2 (d, <i>J</i> = 21.0 Hz), 101.5, 99.0, 48.4, 38.6, 35.0, 17.5. HRMS (ESI) calcd for C<sub>25</sub>H<sub>24</sub>FN<sub>4</sub>O<sub>3</sub>S [M + H]<sup>+</sup>, 479.1548; found, 479.1560.</div></div><div id="sec6_54" class="NLM_sec NLM_sec_level_2"><div id="ac_i72" class="anchor-spacer"></div><h3 class="article-section__title" id="_i72"> <i>N</i>-(1-(4-Fluoro-2,6-dimethylbenzyl)-7-(1-methyl-7-oxo-6,7-dihydro-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridin-3-yl)-1<i>H</i>-indol-5-yl)methanesulfonamide (<b>14</b>)</h3><div class="NLM_p last">This compound was obtained by a procedure similar to the preparation of <b>12</b> (47 mg, yield 92%). <sup>1</sup>H NMR (400 MHz, acetone-<i>d</i><sub>6</sub>) δ 10.03 (s, 1H), 8.30 (s, 1H), 7.59 (d, <i>J</i> = 2.2 Hz, 1H), 7.46 (s, 1H), 7.10 (d, <i>J</i> = 2.2 Hz, 1H), 7.03–6.98 (m, 1H), 6.82 (d, <i>J</i> = 9.6 Hz, 2H), 6.52 (d, <i>J</i> = 3.3 Hz, 1H), 6.44 (d, <i>J</i> = 3.3 Hz, 1H), 6.25 (d, <i>J</i> = 7.0 Hz, 1H), 4.88 (s, 2H), 4.23 (s, 3H), 2.95 (s, 3H). <sup>13</sup>C NMR (101 MHz, acetone-<i>d</i><sub>6</sub>) δ 162.9 (d, <i>J</i> = 244.3 Hz), 157.0, 141.6 (d, <i>J</i> = 8.5 Hz), 134.4, 132.7, 131.5, 131.3, 131.0, 130.1 (d, <i>J</i> = 2.5 Hz), 128.4, 126.3, 123.9, 122.3, 119.7, 115.8, 115.5 (d, <i>J</i> = 21.0 Hz), 114.8, 102.4, 100.5, 46.3, 38.8, 35.9, 19.7. HRMS (ESI) calcd for C<sub>26</sub>H<sub>25</sub>FN<sub>4</sub>O<sub>3</sub>S [M + H]<sup>+</sup>, 493.1704; found, 493.1686.</div></div><div id="sec6_55" class="NLM_sec NLM_sec_level_2"><div id="ac_i73" class="anchor-spacer"></div><h3 class="article-section__title" id="_i73"> 6-Bromo-1-(4-fluorobenzyl)-4-nitro-1<i>H</i>-indazole (<b>47</b>)</h3><div class="NLM_p last">This compound was obtained by a procedure similar to the preparation of <b>42a</b> (301 mg, yield 83%). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 8.62 (d, <i>J</i> = 1.0 Hz, 1H), 8.22 (d, <i>J</i> = 1.4 Hz, 1H), 7.84 (t, <i>J</i> = 1.1 Hz, 1H), 7.24–7.17 (m, 2H), 7.08–6.99 (m, 2H), 5.60 (s, 2H). LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> = 349.9.</div></div><div id="sec6_56" class="NLM_sec NLM_sec_level_2"><div id="ac_i74" class="anchor-spacer"></div><h3 class="article-section__title" id="_i74"> <i>N</i>-(6-Bromo-1-(4-fluorobenzyl)-1<i>H</i>-indazol-4-yl)methanesulfonamide (<b>49a</b>)</h3><div class="NLM_p last">This compound was obtained by a procedure similar to the preparation of <b>44a</b> (55 mg, yield 95%). LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> = 397.9</div></div><div id="sec6_57" class="NLM_sec NLM_sec_level_2"><div id="ac_i75" class="anchor-spacer"></div><h3 class="article-section__title" id="_i75"> <i>N</i>-(6-Bromo-1-(4-fluorobenzyl)-1<i>H</i>-indazol-4-yl)ethanesulfonamide (<b>49b</b>)</h3><div class="NLM_p last">This compound was obtained by a procedure similar to the preparation of <b>44a</b> (166 mg, yield 86%). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 8.22 (d, <i>J</i> = 1.0 Hz, 1H), 7.87 (s, 1H), 7.34 (d, <i>J</i> = 1.3 Hz, 1H), 7.30 (t, <i>J</i> = 1.2 Hz, 1H), 7.21–7.15 (m, 2H), 7.03–6.96 (m, 2H), 5.48 (s, 2H), 3.26 (q, <i>J</i> = 7.4 Hz, 2H), 1.39 (t, <i>J</i> = 7.4 Hz, 3H). LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> = 412.0.</div></div><div id="sec6_58" class="NLM_sec NLM_sec_level_2"><div id="ac_i76" class="anchor-spacer"></div><h3 class="article-section__title" id="_i76"> <i>N</i>-(1-(4-Fluorobenzyl)-6-(7-methoxy-1-methyl-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridin-3-yl)-1<i>H</i>-indazol-4-yl)methanesulfonamide (<b>50a</b>)</h3><div class="NLM_p last">This compound was obtained by a procedure similar to the preparation of <b>45a</b> (98 mg, yield 90%). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 8.19–8.14 (m, 1H), 7.80 (d, <i>J</i> = 5.8 Hz, 1H), 7.46 (d, <i>J</i> = 1.1 Hz, 1H), 7.33 (s, 1H), 7.30 (s, 1H), 7.28–7.24 (m, 2H), 7.23 (d, <i>J</i> = 5.8 Hz, 1H), 7.04 (t, <i>J</i> = 8.6 Hz, 2H), 5.60 (s, 2H), 4.13 (s, 6H), 3.15 (s, 3H). LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> = 480.1.</div></div><div id="sec6_59" class="NLM_sec NLM_sec_level_2"><div id="ac_i77" class="anchor-spacer"></div><h3 class="article-section__title" id="_i77"> <i>N</i>-(1-(4-Fluorobenzyl)-6-(7-methoxy-1-methyl-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridin-3-yl)-1<i>H</i>-indazol-4-yl)ethanesulfonamide (<b>50b</b>)</h3><div class="NLM_p last">This compound was obtained by a procedure similar to the preparation of <b>45a</b> (87 mg, yield 77%). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 8.20 (d, <i>J</i> = 0.9 Hz, 1H), 7.77 (d, <i>J</i> = 5.8 Hz, 1H), 7.57 (s, 1H), 7.46 (d, <i>J</i> = 1.1 Hz, 1H), 7.27 (d, <i>J</i> = 1.5 Hz, 2H), 7.25–7.20 (m, 3H), 7.01 (t, <i>J</i> = 8.6 Hz, 2H), 5.57 (s, 2H), 4.10 (d, <i>J</i> = 2.0 Hz, 6H), 3.25 (q, <i>J</i> = 7.4 Hz, 2H), 1.41 (t, <i>J</i> = 7.4 Hz, 3H). LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> = 494.2.</div></div><div id="sec6_60" class="NLM_sec NLM_sec_level_2"><div id="ac_i78" class="anchor-spacer"></div><h3 class="article-section__title" id="_i78"> <i>N</i>-(1-(4-Fluorobenzyl)-6-(1-methyl-7-oxo-6,7-dihydro-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridin-3-yl)-1<i>H</i>-indazol-4-yl)methanesulfonamide (<b>17</b>)</h3><div class="NLM_p last">This compound was obtained by a procedure similar to the preparation of <b>12</b> (38 mg, yield 91%). <sup>1</sup>H NMR (400 MHz, acetone-<i>d</i><sub>6</sub>) δ 10.05 (s, 1H), 8.94 (s, 1H), 8.34 (d, <i>J</i> = 0.9 Hz, 1H), 7.54 (s, 1H), 7.53 (s, 1H), 7.49 (d, <i>J</i> = 1.1 Hz, 1H), 7.45–7.36 (m, 2H), 7.13–7.07 (m, 2H), 7.02 (d, <i>J</i> = 7.1 Hz, 1H), 6.70 (d, <i>J</i> = 7.1 Hz, 1H), 5.71 (s, 2H), 4.20 (s, 3H), 3.11 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 161.6 (d, <i>J</i> = 243.4 Hz), 155.9, 141.1, 133.8, 133.7 (d, <i>J</i> = 3.4 Hz), 131.9, 131.2, 130.1, 129.6 (d, <i>J</i> = 8.4 Hz), 128.5, 125.8, 123.6, 116.4, 116.0, 115.4 (d, <i>J</i> = 21.5 Hz), 110.3, 102.7, 99.4, 51.2, 39.6, 35.5. HRMS (ESI) calcd for C<sub>23</sub>H<sub>21</sub>FN<sub>5</sub>O<sub>3</sub>S [M + H]<sup>+</sup>, 466.1344; found, 466.1353.</div></div><div id="sec6_61" class="NLM_sec NLM_sec_level_2"><div id="ac_i79" class="anchor-spacer"></div><h3 class="article-section__title" id="_i79"> <i>N</i>-(1-(4-Fluorobenzyl)-6-(1-methyl-7-oxo-6,7-dihydro-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridin-3-yl)-1<i>H</i>-indazol-4-yl)ethanesulfonamide (<b>18</b>)</h3><div class="NLM_p last">This compound was obtained by a procedure similar to the preparation of <b>12</b> (31 mg, yield 90%). <sup>1</sup>H NMR (400 MHz, acetone-<i>d</i><sub>6</sub>) δ 10.21 (s, 1H), 9.00 (s, 1H), 8.38 (s, 1H), 7.60–7.48 (m, 3H), 7.44–7.35 (m, 2H), 7.10 (t, <i>J</i> = 8.8 Hz, 2H), 7.05 (d, <i>J</i> = 7.1 Hz, 1H), 6.73 (d, <i>J</i> = 7.0 Hz, 1H), 5.70 (s, 2H), 4.20 (s, 3H), 3.23 (q, <i>J</i> = 7.4 Hz, 2H), 1.32 (t, <i>J</i> = 7.4 Hz, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 161.6 (d, <i>J</i> = 243.4 Hz), 155.9, 141.1, 133.8, 133.7 (d, <i>J</i> = 3.2 Hz), 131.8, 131.2, 130.1, 129.7 (d, <i>J</i> = 8.3 Hz), 128.5, 125.8, 123.6, 116.4, 115.7, 115.4 (d, <i>J</i> = 21.4 Hz), 109.8, 102.6, 99.3, 51.1, 45.5, 35.5, 8.2. HRMS (ESI) calcd for C<sub>24</sub>H<sub>23</sub>FN<sub>5</sub>O<sub>3</sub>S [M + H]<sup>+</sup>, 480.1500; found, 480.1487.</div></div><div id="sec6_62" class="NLM_sec NLM_sec_level_2"><div id="ac_i80" class="anchor-spacer"></div><h3 class="article-section__title" id="_i80"> 7-Methoxy-1-methyl-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridine-3-carbaldehyde (<b>36</b>)</h3><div class="NLM_p last">To a solution of <b>34</b> (288 mg, 1 mmol) in THF (6 mL) at −78 °C was added <i>n</i>-BuLi (1.2 mmol) for 1 h. The reaction was cooled to −78 °C, and DMF (0.1 mL) was added. Dry ice bath was removed, and the mixture was stirred for 2 h at room temperature. Water was added to quench the reaction. The mixture was extracted with ethyl acetate, and the combined organic layer was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>. Then filtration and concentration in vacuo gave a residue, which was purified by flash column chromatography on silica gel (hexanes–EtOAc, 4:1) to obtain <b>36</b> (138 mg, yield 73%) as a white solid. <sup>1</sup>H NMR (400 MHz, acetone-<i>d</i><sub>6</sub>) δ 9.97 (s, 1H), 8.12–8.03 (m, 1H), 7.83 (dd, <i>J</i> = 5.5, 1.4 Hz, 1H), 7.66 (dt, <i>J</i> = 5.8, 1.5 Hz, 1H), 4.14 (s, 3H), 4.05 (s, 3H). LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> = 191.2.</div></div><div id="sec6_63" class="NLM_sec NLM_sec_level_2"><div id="ac_i81" class="anchor-spacer"></div><h3 class="article-section__title" id="_i81"> <i>N</i><sup>1</sup>-(4-Fluorobenzyl)-3-nitrobenzene-1,2-diamine (<b>53</b>)</h3><div class="NLM_p last">To a solution of <b>51</b> (153 mg, 1 mmol) in DMF (5 mL) at RT were added K<sub>2</sub>CO<sub>3</sub> (276 mg, 2 mmol) and <b>52</b> (137 μL, 1.1 mmol). The solution was stirred at RT. After 2 h, the reaction was quenched by addition of water. The mixture was extracted with ethyl acetate, and the combined organic layer was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>. Then filtration and concentration in vacuo gave a residue, which was purified by flash column chromatography on silica gel (hexanes–EtOAc, 2:1) to obtain <b>53</b> (196 mg, yield 75%) as a yellow solid. LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> = 262.2.</div></div><div id="sec6_63_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i82" class="anchor-spacer"></div><h3 class="article-section__title" id="_i82"> 1-(4-Fluorobenzyl)-2-(7-methoxy-1-methyl-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridin-3-yl)-4-nitro-1<i>H</i>-benzo[<i>d</i>]imidazole (<b>54</b>)</h3><div class="NLM_p last">To a solution of <b>53</b> (261 mg, 1 mmol) in HOAc-H<sub>2</sub>O (6 mL, v/v, 2:1) at RT was added NaBO<sub>3</sub>·4H<sub>2</sub>O (462 mg, 3 mmol). The solution was stirred at 50 °C for 1 h and then quenched by addition of water. The mixture was extracted with ethyl acetate, and the combined organic layer was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>. Then filtration and concentration in vacuo gave a residue, which was purified by flash column chromatography on silica gel (hexanes–acetone, 1:1) to obtain <b>54</b> (289 mg, yield 67%) as a yellow solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.15–8.04 (m, 3H), 7.96 (d, <i>J</i> = 8.1 Hz, 1H), 7.85 (d, <i>J</i> = 5.6 Hz, 1H), 7.38 (t, <i>J</i> = 8.0 Hz, 1H), 7.13 (d, <i>J</i> = 7.1 Hz, 4H), 5.86 (s, 2H), 4.07 (s, 3H), 4.03 (s, 3H). LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> = 432.1.</div></div><div id="sec6_64" class="NLM_sec NLM_sec_level_2"><div id="ac_i83" class="anchor-spacer"></div><h3 class="article-section__title" id="_i83"> 1-(4-Fluorobenzyl)-2-(7-methoxy-1-methyl-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridin-3-yl)-1<i>H</i>-benzo[<i>d</i>]imidazol-4-amine (<b>55</b>)</h3><div class="NLM_p last">To a solution of <b>54</b> (1.29 g, 3 mmol) in EtOH (15 mL) at RT were added iron powder (1.34 g, 24 mmol) and saturated ammonia chloride (3 mL). The reaction was refluxed for 2 h and then cooled to room temperature. The mixture was filtered to obtain a clear solution, followed by extraction with ethyl acetate. Organic layers were combined and dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>. Then filtration and concentration in vacuo gave <b>55</b> as a brown solid without further purification (905 mg). LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> = 402.1.</div></div><div id="sec6_65" class="NLM_sec NLM_sec_level_2"><div id="ac_i84" class="anchor-spacer"></div><h3 class="article-section__title" id="_i84"> <i>N</i>-(1-Benzyl-2-(7-methoxy-1-methyl-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridin-3-yl)-1<i>H</i>-benzo[<i>d</i>]imidazol-4-yl)ethanesulfonamide (<b>56a</b>). <i>N</i>-(1-Benzyl-2-(7-methoxy-1-methyl-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridin-3-yl)-1<i>H</i>-benzo[<i>d</i>]imidazol-4-yl)-<i>N</i>-(ethylsulfonyl)ethanesulfonamide (<b>56b</b>)</h3><div class="NLM_p last">To a solution of <b>55</b> (431 mg, 1 mmol) in DCM (5 mL) at 0 °C was added EtSO<sub>2</sub>Cl (0.28 mL, 3 mmol). Then ice bath was removed. After stirring for 3 h at RT, the reaction contents were quenched by the addition of water. The mixture was extracted with ethyl acetate, and the combined organic layer was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>. Filtration and concentration in vacuo gave a residue without further purification. For <b>56a</b>, LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> = 494.2, For <b>56b</b>, LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> = 586.1.</div></div><div id="sec6_66" class="NLM_sec NLM_sec_level_2"><div id="ac_i85" class="anchor-spacer"></div><h3 class="article-section__title" id="_i85"> <i>N</i>-(1-(4-Fluorobenzyl)-2-(1-methyl-7-oxo-6,7-dihydro-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridin-3-yl)-1<i>H</i>-benzo[<i>d</i>]imidazol-4-yl)ethanesulfonamide (<b>15</b>)</h3><div class="NLM_p last">This compound was obtained by a procedure similar to the preparation of <b>10</b> (26 mg, yield 37% for two steps). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.11 (d, <i>J</i> = 5.9 Hz, 1H), 9.60 (s, 1H), 7.72 (s, 1H), 7.31–7.27 (m, 2H), 7.17–7.09 (m, 6H), 6.99 (dd, <i>J</i> = 7.0, 5.7 Hz, 1H), 5.68 (s, 2H),3.37 (q, <i>J</i> = 7.4 Hz, 2H), 1.34 (t, <i>J</i> = 7.3 Hz, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 161.4 (d, <i>J</i> = 243.4 Hz), 155.6, 147.8, 136.8, 136.6, 136.4, 133.1 (d, <i>J</i> = 2.5 Hz), 131.3, 131.0, 128.4 (d, <i>J</i> = 8.3 Hz), 128.1, 126.1, 123.4, 122.4, 115.7 (d, <i>J</i> = 21.9 Hz), 107.2, 105.0, 101.3, 46.7, 46.6, 35.8, 8.3. HRMS (ESI) calcd for C<sub>24</sub>H<sub>23</sub>FN<sub>5</sub>O<sub>3</sub>S [M + H]<sup>+</sup>, 480.1500; found, 480.1496.</div></div><div id="sec6_67" class="NLM_sec NLM_sec_level_2"><div id="ac_i86" class="anchor-spacer"></div><h3 class="article-section__title" id="_i86"> <i>N</i>-(Ethylsulfonyl)-<i>N</i>-(1-(4-fluorobenzyl)-2-(1-methyl-7-oxo-6,7-dihydro-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridin-3-yl)-1<i>H</i>-benzo[<i>d</i>]imidazol-4-yl)ethanesulfonamide (<b>16</b>)</h3><div class="NLM_p last">This compound was obtained by a procedure similar to the preparation of <b>10</b> (32 mg, yield 31% for two steps). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.15 (d, <i>J</i> = 5.6 Hz, 1H), 7.79 (s, 1H), 7.64–7.59 (m, 1H), 7.34 (dd, <i>J</i> = 7.8, 1.0 Hz, 1H), 7.26 (t, <i>J</i> = 7.9 Hz, 1H), 7.19–7.11 (m, 5H), 7.02 (dd, <i>J</i> = 6.9, 5.6 Hz, 1H), 5.74 (s, 2H), 4.10 (s, 3H), 3.95–3.78 (m, <i>J</i> = 7.2 Hz, 4H), 1.38 (t, <i>J</i> = 7.4 Hz, 6H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 161.5 (d, <i>J</i> = 243.5 Hz), 155.6, 149.6, 142.0, 136.8, 132.8 (d, <i>J</i> = 2.1 Hz), 131.5, 131.4, 128.5 (d, <i>J</i> = 8.2 Hz), 126.6, 125.4, 123.6, 123.5, 122.2, 115.8 (d, <i>J</i> = 21.6 Hz), 112.5, 104.6, 100.7, 50.5, 46.8, 35.9, 7.7. HRMS (ESI) calcd for C<sub>26</sub>H<sub>27</sub>FN<sub>5</sub>O<sub>5</sub>S<sub>2</sub> [M + H]<sup>+</sup>, 572.1432; found, 572.1424.</div></div><div id="sec6_68" class="NLM_sec NLM_sec_level_2"><div id="ac_i87" class="anchor-spacer"></div><h3 class="article-section__title" id="_i87"> 6-Bromo-1-(di(pyridin-2-yl)methyl)-4-nitro-1<i>H</i>-indole (<b>58a</b>)</h3><div class="NLM_p last">To a solution of <b>57</b> (7.23 g, 30 mmol) in DMF (60 mL) at RT were added K<sub>2</sub>CO<sub>3</sub> (6.22 g, 45 mmol) and 2,2′-(bromomethylene)dipyridine (8.97 g, 36 mmol),<a onclick="showRef(event, 'ref83'); return false;" href="javascript:void(0);" class="ref ref83">(83)</a> and then the mixture was stirred at 50 °C for 6 h. Water was added to quench the reaction. The solution was extracted with ethyl acetate, and the combined organic layer was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>. Filtration and concentration in vacuo gave a residue, which was purified by flash column chromatography on silica gel (hexanes–EtOAc, 2:1) to obtain <b>58a</b> (11.17 g, yield 91%) as a yellow solid. <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 8.70–8.64 (m, 2H), 8.25 (d, <i>J</i> = 1.5 Hz, 1H), 7.85 (d, <i>J</i> = 1.4 Hz, 1H), 7.72 (td, <i>J</i> = 7.8, 1.8 Hz, 2H), 7.52 (d, <i>J</i> = 3.4 Hz, 1H), 7.33–7.25 (m, 3H), 7.16 (d, <i>J</i> = 7.9 Hz, 2H), 6.97 (s, 1H). LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> = 409.0.</div></div><div id="sec6_69" class="NLM_sec NLM_sec_level_2"><div id="ac_i88" class="anchor-spacer"></div><h3 class="article-section__title" id="_i88"> 1-Benzhydryl-6-bromo-4-nitro-1<i>H</i>-indole (<b>58b</b>)</h3><div class="NLM_p last">This compound was obtained by a procedure similar to the preparation of <b>58a</b> (425 mg, yield 72%). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 8.26 (d, <i>J</i> = 1.5 Hz, 1H), 7.69 (t, <i>J</i> = 1.1 Hz, 1H), 7.40–7.34 (m, 6H), 7.21 (dd, <i>J</i> = 3.3, 0.8 Hz, 1H), 7.11–7.05 (m, 5H), 6.82 (s, 1H). LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> = 407.0.</div></div><div id="sec6_70" class="NLM_sec NLM_sec_level_2"><div id="ac_i89" class="anchor-spacer"></div><h3 class="article-section__title" id="_i89"> 6-Bromo-1-(1,1-di(pyridin-2-yl)ethyl)-4-nitro-1<i>H</i>-indole (<b>59a</b>)</h3><div class="NLM_p last">To a solution of <b>58a</b> (10.23 g, 25 mmol) in THF (80 mL) at 0 °C were added <i>t</i>-BuOK (5.61 g, 50 mmol) and MeI (4.67 mL, 75 mmol) successively. Then the ice bath was removed. The solution was stirred at RT for 2 h, and water was added to quench the reaction. The mixture was extracted with ethyl acetate, and the combined organic layer was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>. Filtration and concentration in vacuo gave a residue, which was purified by flash column chromatography on silica gel (hexanes–EtOAc, 3:1) to obtain <b>59a</b> (9.84 g, yield 93%) as a yellow solid. <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 8.66 (d, <i>J</i> = 4.7 Hz, 2H), 8.15 (d, <i>J</i> = 1.5 Hz, 1H), 7.66 (td, <i>J</i> = 7.8, 1.9 Hz, 2H), 7.28 (t, <i>J</i> = 3.4 Hz, 3H), 7.26 (s, 1H), 7.11 (d, <i>J</i> = 8.1 Hz, 2H), 6.91 (s, 1H), 2.55 (s, 3H). LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> = 423.1.</div></div><div id="sec6_71" class="NLM_sec NLM_sec_level_2"><div id="ac_i90" class="anchor-spacer"></div><h3 class="article-section__title" id="_i90"> 6-Bromo-1-(1,1-di(pyridin-2-yl)propyl)-4-nitro-1<i>H</i>-indole (<b>59b</b>)</h3><div class="NLM_p last">This compound was obtained by a procedure similar to the preparation of <b>59a</b> (276 mg, yield 73%). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 8.65–8.59 (m, 2H), 8.15 (s, 1H), 7.66 (td, <i>J</i> = 7.8, 1.9 Hz, 2H), 7.51–7.43 (m, 3H), 7.28–7.18 (m, 3H), 6.82 (s, 1H), 3.10 (q, <i>J</i> = 7.3 Hz, 2H), 0.70 (t, <i>J</i> = 7.3 Hz, 3H). LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> = 437.1.</div></div><div id="sec6_72" class="NLM_sec NLM_sec_level_2"><div id="ac_i91" class="anchor-spacer"></div><h3 class="article-section__title" id="_i91"> 6-Bromo-1-(1,1-di(pyridin-2-yl)ethyl)-1<i>H</i>-indol-4-amine (<b>60a</b>)</h3><div class="NLM_p last">To a solution of <b>59a</b> (9.30 g, 22 mmol) in EtOH (60 mL) at RT were added iron powder (9.86 g, 176 mmol) and saturated ammonia chloride (15 mL). The reaction was refluxed for 2 h and then cooled to room temperature. The mixture was filtered to obtain a clear solution, followed by extraction with ethyl acetate. Organic layers were combined and dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>. Then filtration and concentration in vacuo gave <b>60a</b> (7.93 g) as a brown solid without further purification. LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> = 393.1.</div></div><div id="sec6_73" class="NLM_sec NLM_sec_level_2"><div id="ac_i92" class="anchor-spacer"></div><h3 class="article-section__title" id="_i92"> <i>N</i>-(6-Bromo-1-(1,1-diphenylethyl)-7,7a-dihydro-1<i>H</i>-indol-4-yl)ethanesulfonamide (<b>61a</b>)</h3><div class="NLM_p last">This compound was obtained by a procedure similar to the preparation of <b>61c</b> (202 mg, yield 83%). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 7.42–7.40 (m, 2H), 7.34 (dd, <i>J</i> = 5.2, 1.9 Hz, 6H), 7.25 (t, <i>J</i> = 1.1 Hz, 1H), 7.11–7.05 (m, 4H), 6.84 (d, <i>J</i> = 3.4 Hz, 1H), 6.74 (s, 1H), 6.65 (d, <i>J</i> = 3.4 Hz, 1H), 3.20 (q, <i>J</i> = 7.4 Hz, 2H), 1.37 (t, <i>J</i> = 7.4 Hz, 3H). LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> = 469.1.</div></div><div id="sec6_74" class="NLM_sec NLM_sec_level_2"><div id="ac_i93" class="anchor-spacer"></div><h3 class="article-section__title" id="_i93"> <i>N</i>-(6-Bromo-1-(1,1-di(pyridin-2-yl)ethyl)-1<i>H</i>-indol-4-yl)methanesulfonamide (<b>61b</b>)</h3><div class="NLM_p last">This compound was obtained by a procedure similar to the preparation of <b>61c</b> (299 mg, yield 90%). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 8.66 (ddd, <i>J</i> = 4.8, 1.9, 0.9 Hz, 2H), 7.63 (td, <i>J</i> = 7.8, 1.9 Hz, 2H), 7.29 (d, <i>J</i> = 1.4 Hz, 1H), 7.24 (dd, <i>J</i> = 2.8, 1.1 Hz, 1H), 7.23 (d, <i>J</i> = 1.1 Hz, 1H), 7.16 (s, 1H), 7.07 (s, 1H), 7.05 (s, 1H), 6.98 (d, <i>J</i> = 3.5 Hz, 1H), 6.57 (dd, <i>J</i> = 3.6, 0.9 Hz, 1H), 6.46 (t, <i>J</i> = 1.2 Hz, 1H), 3.04 (s, 3H), 2.52 (d, <i>J</i> = 1.8 Hz, 3H). LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> = 471.1.</div></div><div id="sec6_75" class="NLM_sec NLM_sec_level_2"><div id="ac_i94" class="anchor-spacer"></div><h3 class="article-section__title" id="_i94"> <i>N</i>-(6-Bromo-1-(1,1-di(pyridin-2-yl)ethyl)-1<i>H</i>-indol-4-yl)ethanesulfonamide (<b>61c</b>)</h3><div class="NLM_p last">To a solution of <b>60a</b> (from previous step) in pyridine (40 mL) at 0 °C was added EtSO<sub>2</sub>Cl (4.17 mL, 44 mmol). Upon completion monitored by TLC, the reaction contents were quenched by the addition of water. The mixture was extracted with ethyl acetate, and the combined organic layer was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>. Filtration and concentration in vacuo gave a residue, which was purified by flash column chromatography on silica gel (hexanes–EtOAc, 1:1) to obtain <b>61c</b> (9.18 g, yield 86% for two steps) as a dark yellow solid. <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 8.68 (ddd, <i>J</i> = 4.8, 2.0, 1.0 Hz, 2H), 7.65 (tt, <i>J</i> = 7.8, 1.6 Hz, 2H), 7.33 (d, <i>J</i> = 1.3 Hz, 1H), 7.27 (ddt, <i>J</i> = 7.4, 4.8, 1.2 Hz, 2H), 7.07 (d, <i>J</i> = 7.9 Hz, 2H), 6.98 (dd, <i>J</i> = 3.5, 1.3 Hz, 1H), 6.64 (ddd, <i>J</i> = 7.1, 3.6, 0.9 Hz, 1H), 6.46 (t, <i>J</i> = 1.1 Hz, 1H), 3.20 (q, <i>J</i> = 7.4 Hz, 2H), 2.54 (s, 3H), 1.39 (td, <i>J</i> = 7.4, 1.2 Hz, 3H). LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> = 485.2.</div></div><div id="sec6_76" class="NLM_sec NLM_sec_level_2"><div id="ac_i95" class="anchor-spacer"></div><h3 class="article-section__title" id="_i95"> <i>N</i>-(6-Bromo-1-(1,1-di(pyridin-2-yl)propyl)-7,7a-dihydro-1<i>H</i>-indol-4-yl)ethanesulfonamide (<b>61d</b>)</h3><div class="NLM_p last">This compound was obtained by a procedure similar to the preparation of <b>61c</b> (183 mg, yield 85%). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 8.60 (dd, <i>J</i> = 5.1, 1.8 Hz, 2H), 7.61 (td, <i>J</i> = 7.8, 1.9 Hz, 2H), 7.47 (s, 1H), 7.43 (dd, <i>J</i> = 8.0, 1.3 Hz, 2H), 7.31 (d, <i>J</i> = 1.4 Hz, 1H), 7.20–7.15 (m, 3H), 6.66 (d, <i>J</i> = 3.5 Hz, 1H), 6.34 (s, 1H), 3.18 (q, <i>J</i> = 7.4 Hz, 2H), 3.08 (q, <i>J</i> = 7.2 Hz, 2H), 1.35 (t, <i>J</i> = 7.4 Hz, 3H), 0.66 (t, <i>J</i> = 7.3 Hz, 3H). LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> = 501.1.</div></div><div id="sec6_77" class="NLM_sec NLM_sec_level_2"><div id="ac_i96" class="anchor-spacer"></div><h3 class="article-section__title" id="_i96"> <i>N</i>-(6-Bromo-1-(1,1-di(pyridin-2-yl)ethyl)-1<i>H</i>-indol-4-yl)isobutyramide (<b>61e</b>)</h3><div class="NLM_p last">This compound was obtained by a procedure similar to the preparation of <b>61c</b> (138 mg, yield 88%). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 8.64 (dd, <i>J</i> = 4.7, 1.8 Hz, 2H), 7.95 (s, 1H), 7.59 (td, <i>J</i> = 7.8, 1.9 Hz, 2H), 7.55 (s, 1H), 7.21 (dd, <i>J</i> = 7.6, 4.8 Hz, 2H), 7.02 (d, <i>J</i> = 8.0 Hz, 2H), 6.94 (d, <i>J</i> = 3.5 Hz, 1H), 6.42 (d, <i>J</i> = 3.6 Hz, 1H), 6.39 (s, 1H), 2.60 (p, <i>J</i> = 6.9 Hz, 1H), 2.50 (s, 3H), 1.25 (d, <i>J</i> = 6.9 Hz, 6H). LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> = 463.1.</div></div><div id="sec6_78" class="NLM_sec NLM_sec_level_2"><div id="ac_i97" class="anchor-spacer"></div><h3 class="article-section__title" id="_i97"> <i>N</i>-(6-Bromo-1-(1,1-di(pyridin-2-yl)ethyl)-1<i>H</i>-indol-4-yl)acetamide (<b>61f</b>)</h3><div class="NLM_p last">This compound was obtained by a procedure similar to the preparation of <b>61c</b> (106 mg, yield 86%). <sup>1</sup>H NMR (400 MHz, acetone-<i>d</i><sub>6</sub>) δ 9.06 (s, 1H), 8.62 (ddd, <i>J</i> = 4.8, 1.9, 1.0 Hz, 2H), 8.10 (s, 1H), 7.76 (td, <i>J</i> = 7.8, 1.9 Hz, 2H), 7.34 (ddd, <i>J</i> = 7.5, 4.8, 1.1 Hz, 2H), 7.15 (dt, <i>J</i> = 8.1, 1.1 Hz, 2H), 7.02 (d, <i>J</i> = 3.6 Hz, 1H), 6.81–6.72 (m, 1H), 6.33 (t, <i>J</i> = 1.3 Hz, 1H), 2.50 (s, 3H), 2.17 (s, 3H). LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> = 435.0.</div></div><div id="sec6_79" class="NLM_sec NLM_sec_level_2"><div id="ac_i98" class="anchor-spacer"></div><h3 class="article-section__title" id="_i98"> <i>N</i>-(1-Benzhydryl-6-(7-methoxy-1-methyl-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridin-3-yl)-1<i>H</i>-indol-4-yl)ethanesulfonamide (<b>62a</b>)</h3><div class="NLM_p last">This compound was obtained by a procedure similar to the preparation of <b>62c</b> (75 mg, yield 66%). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 7.66 (d, <i>J</i> = 5.8 Hz, 1H), 7.48 (d, <i>J</i> = 1.2 Hz, 1H), 7.35 (qd, <i>J</i> = 4.0, 2.3 Hz, 7H), 7.25 (d, <i>J</i> = 1.1 Hz, 1H), 7.18–7.15 (m, 5H), 6.96 (d, <i>J</i> = 5.8 Hz, 1H), 6.84 (d, <i>J</i> = 3.4 Hz, 1H), 6.81 (s, 1H), 6.64 (d, <i>J</i> = 3.1 Hz, 1H), 4.07 (s, 3H), 4.03 (s, 3H), 3.20 (q, <i>J</i> = 7.4 Hz, 2H), 1.38 (t, <i>J</i> = 7.4 Hz, 3H). LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> = 551.2.</div></div><div id="sec6_80" class="NLM_sec NLM_sec_level_2"><div id="ac_i99" class="anchor-spacer"></div><h3 class="article-section__title" id="_i99"> <i>N</i>-(1-(1,1-Di(pyridin-2-yl)ethyl)-6-(7-methoxy-1-methyl-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridin-3-yl)-1<i>H</i>-indol-4-yl)methanesulfonamide (<b>62b</b>)</h3><div class="NLM_p last">This compound was obtained by a procedure similar to the preparation of <b>62c</b> (109 mg, yield 91%). <sup>1</sup>H NMR (400 MHz, acetone-<i>d</i><sub>6</sub>) δ 8.66 (ddd, <i>J</i> = 4.7, 1.9, 0.9 Hz, 2H), 8.45 (s, 1H), 7.78 (td, <i>J</i> = 7.8, 1.9 Hz, 2H), 7.57 (d, <i>J</i> = 5.7 Hz, 1H), 7.40 (d, <i>J</i> = 1.2 Hz, 1H), 7.37 (ddd, <i>J</i> = 7.6, 4.8, 1.1 Hz, 2H), 7.30 (d, <i>J</i> = 1.1 Hz, 1H), 7.27 (s, 2H), 7.07 (d, <i>J</i> = 3.5 Hz, 1H), 6.89 (d, <i>J</i> = 3.5 Hz, 1H), 6.71 (d, <i>J</i> = 5.8 Hz, 1H), 6.66 (t, <i>J</i> = 1.1 Hz, 1H), 4.05 (s, 3H), 4.01 (s, 3H), 3.03 (s, 3H), 2.55 (s, 3H). LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> = 553.2.</div></div><div id="sec6_81" class="NLM_sec NLM_sec_level_2"><div id="ac_i100" class="anchor-spacer"></div><h3 class="article-section__title" id="_i100"> <i>N</i>-(1-(1,1-Di(pyridin-2-yl)ethyl)-6-(7-methoxy-1-methyl-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridin-3-yl)-1<i>H</i>-indol-4-yl)ethanesulfonamide (<b>62c</b>)</h3><div class="NLM_p last">A mixture of <b>61c</b> (7.28 g, 15 mmol), Pd(dppf)Cl<sub>2</sub>–DCM (612 mg, 0.75 mmol), <b>35</b> (5.20 g, 18 mmol), and K<sub>3</sub>PO<sub>4</sub> (8.28 g, 39 mmol) in dioxane–water (80 mL, 4:1) was heated at 90 °C for 24 h. The mixture was then diluted with ethyl acetate and filtered to obtain a clear solution, which was washed three times with brine. The organic layer was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated. The crude product was purified by flash column chromatography on silica gel (hexanes–acetone, 1:1) to yield <b>62c</b> (7.48 g, yield 88%) as a gray solid. <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 8.69 (ddd, <i>J</i> = 4.8, 1.9, 0.9 Hz, 2H), 7.64 (td, <i>J</i> = 7.7, 1.9 Hz, 2H), 7.58 (d, <i>J</i> = 5.6 Hz, 1H), 7.39 (d, <i>J</i> = 1.2 Hz, 1H), 7.27–7.23 (m, 2H), 7.18 (dt, <i>J</i> = 8.0, 1.1 Hz, 2H), 7.03 (s, 1H), 7.02 (d, <i>J</i> = 3.7 Hz, 1H), 6.83 (s, 1H), 6.64–6.60 (m, 2H), 6.57 (d, <i>J</i> = 3.6 Hz, 1H), 4.05 (s, 3H), 4.02 (s, 3H), 3.20 (q, <i>J</i> = 7.3 Hz, 2H), 2.55 (s, 3H), 1.40 (t, <i>J</i> = 7.3 Hz, 3H). LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> = 567.1.</div></div><div id="sec6_82" class="NLM_sec NLM_sec_level_2"><div id="ac_i101" class="anchor-spacer"></div><h3 class="article-section__title" id="_i101"> <i>N</i>-(1-(1,1-Di(pyridin-2-yl)propyl)-6-(7-methoxy-1-methyl-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridin-3-yl)-1<i>H</i>-indol-4-yl)ethanesulfonamide (<b>62d</b>)</h3><div class="NLM_p last">This compound was obtained by a procedure similar to the preparation of <b>62c</b> (96 mg, yield 79%). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 8.64 (s, 2H), 7.72–7.48 (m, 5H), 7.38 (s, 1H), 7.25–7.13 (m, 3H), 7.04 (s, 1H), 6.80 (s, 1H), 6.56 (d, <i>J</i> = 15.4 Hz, 3H), 4.03 (d, <i>J</i> = 13.9 Hz, 6H), 3.17 (dq, <i>J</i> = 15.5, 7.4 Hz, 4H), 1.42–1.19 (m, 6H). LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> = 581.1.</div></div><div id="sec6_83" class="NLM_sec NLM_sec_level_2"><div id="ac_i102" class="anchor-spacer"></div><h3 class="article-section__title" id="_i102"> <i>N</i>-(1-(1,1-Di(pyridin-2-yl)ethyl)-6-(7-methoxy-1-methyl-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridin-3-yl)-1<i>H</i>-indol-4-yl)isobutyramide (<b>62e</b>)</h3><div class="NLM_p last">This compound was obtained by a procedure similar to the preparation of <b>62c</b> (78 mg, yield 82%). <sup>1</sup>H NMR (400 MHz, acetone-<i>d</i><sub>6</sub>) δ 8.88 (s, 1H), 8.65 (ddd, <i>J</i> = 4.8, 1.9, 0.9 Hz, 2H), 8.17 (s, 1H), 7.75 (td, <i>J</i> = 7.8, 1.9 Hz, 2H), 7.57 (d, <i>J</i> = 5.7 Hz, 1H), 7.35 (ddd, <i>J</i> = 7.6, 4.7, 1.1 Hz, 2H), 7.26 (d, <i>J</i> = 1.0 Hz, 1H), 7.25–7.22 (m, 2H), 7.00 (d, <i>J</i> = 3.5 Hz, 1H), 6.78–6.73 (m, 2H), 6.57 (s, 1H), 4.04 (s, 3H), 4.01 (s, 3H), 2.84 (hept, <i>J</i> = 6.8 Hz, 1H), 2.53 (s, 3H), 1.22 (d, <i>J</i> = 6.8 Hz, 6H). LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> = 545.2.</div></div><div id="sec6_84" class="NLM_sec NLM_sec_level_2"><div id="ac_i103" class="anchor-spacer"></div><h3 class="article-section__title" id="_i103"> <i>N</i>-(1-(1,1-Di(pyridin-2-yl)ethyl)-6-(7-methoxy-1-methyl-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridin-3-yl)-1<i>H</i>-indol-4-yl)acetamide (<b>62f</b>)</h3><div class="NLM_p last">This compound was obtained by a procedure similar to the preparation of <b>62c</b> (98 mg, yield 81%). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 8.69 (d, <i>J</i> = 3.6 Hz, 2H), 7.96 (s, 1H), 7.62 (td, <i>J</i> = 7.8, 1.9 Hz, 2H), 7.56 (d, <i>J</i> = 5.8 Hz, 1H), 7.44 (s, 1H), 7.28–7.20 (m, 2H), 7.16 (d, <i>J</i> = 8.0 Hz, 2H), 7.03 (d, <i>J</i> = 3.4 Hz, 2H), 6.64 (d, <i>J</i> = 5.8 Hz, 1H), 6.58 (s, 1H), 6.48 (d, <i>J</i> = 3.4 Hz, 1H), 4.04 (s, 3H), 4.00 (s, 3H), 2.55 (s, 3H), 2.28 (s, 3H). LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> = 517.2.</div></div><div id="sec6_85" class="NLM_sec NLM_sec_level_2"><div id="ac_i104" class="anchor-spacer"></div><h3 class="article-section__title" id="_i104"> <i>N</i>-(1-Benzhydryl-6-(1-methyl-7-oxo-6,7-dihydro-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridin-3-yl)-1<i>H</i>-indol-4-yl)ethanesulfonamide (<b>25</b>)</h3><div class="NLM_p last">This compound was obtained by a procedure similar to the preparation of <b>27</b> (33 mg, yield 94%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.93 (d, <i>J</i> = 5.6 Hz, 1H), 9.68 (s, 1H), 7.47 (s, 1H), 7.43–7.30 (m, 7H), 7.24 (d, <i>J</i> = 1.2 Hz, 1H), 7.22–7.15 (m, 5H), 6.97 (d, <i>J</i> = 3.3 Hz, 1H), 6.87–6.80 (m, 2H), 6.36 (dd, <i>J</i> = 7.0, 1.1 Hz, 1H), 4.09 (s, 3H), 3.09 (q, <i>J</i> = 7.3 Hz, 2H), 1.22 (t, <i>J</i> = 7.3 Hz, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 155.9, 139.9, 137.7, 130.2, 129.1, 128.8, 128.3, 128.2, 127.9, 126.6, 125.2, 123.4, 120.7, 117.3, 111.6, 105.6, 99.6, 99.4, 62.6, 45.5, 35.4, 8.2. HRMS (ESI) calcd for C<sub>31</sub>H<sub>29</sub>N<sub>4</sub>O<sub>3</sub>S [M + H]<sup>+</sup>, 537.1955; found, 537.1942.</div></div><div id="sec6_86" class="NLM_sec NLM_sec_level_2"><div id="ac_i105" class="anchor-spacer"></div><h3 class="article-section__title" id="_i105"> <i>N</i>-(1-(1,1-Di(pyridin-2-yl)ethyl)-6-(1-methyl-7-oxo-6,7-dihydro-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridin-3-yl)-1<i>H</i>-indol-4-yl)methanesulfonamide (<b>26</b>)</h3><div class="NLM_p last">This compound was obtained by a procedure similar to the preparation of <b>27</b> (56 mg, yield 92%). <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 8.60 (d, <i>J</i> = 4.1 Hz, 2H), 7.77–7.67 (m, 2H), 7.34 (dd, <i>J</i> = 7.6, 4.9 Hz, 2H), 7.23 (s, 1H), 7.16 (d, <i>J</i> = 4.2 Hz, 2H), 7.14 (s, 2H), 6.82 (d, <i>J</i> = 7.0 Hz, 1H), 6.79 (d, <i>J</i> = 3.5 Hz, 1H), 6.42 (s, 1H), 6.16 (d, <i>J</i> = 6.9 Hz, 1H), 4.08 (s, 3H), 2.98 (s, 3H), 2.51 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 161.1, 155.8, 148.9, 137.1, 137.0, 130.5, 128.9, 127.9, 127.5, 127.4, 125.1, 123.2, 122.8, 122.5, 122.2, 117.2, 111.9, 108.2, 99.5, 99.1, 69.9, 35.3, 30.7, 27.9. HRMS (ESI) calcd for C<sub>29</sub>H<sub>27</sub>FN<sub>6</sub>O<sub>3</sub>S [M + H]<sup>+</sup>, 539.1860; found, 539.1836.</div></div><div id="sec6_87" class="NLM_sec NLM_sec_level_2"><div id="ac_i106" class="anchor-spacer"></div><h3 class="article-section__title" id="_i106"> <i>N</i>-(1-(1,1-Di(pyridin-2-yl)ethyl)-6-(1-methyl-7-oxo-6,7-dihydro-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridin-3-yl)-1<i>H</i>-indol-4-yl)ethanesulfonamide (<b>27</b>)</h3><div class="NLM_p last">A mixture of <b>62c</b> (5.66 g, 10 mmol), TMSCl (2.03 mL, 16 mmol), NaI (2.40 g, 16 mmol), and H<sub>2</sub>O (90 μL, 5 mmol) in acetonitrile (30 mL) was heated at 50 °C for 3 h and then cooled to room temperature. The mixture was then diluted with ethyl acetate and washed three times with aqueous NaHCO<sub>3</sub>. The organic layer was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated. The crude product was recrystallized (methanol/ethyl acetate) to obtain the final product <b>27</b> (4.7 g, yield 85%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.92 (d, <i>J</i> = 5.4 Hz, 1H), 9.66 (s, 1H), 8.63 (d, <i>J</i> = 3.9 Hz, 2H), 7.79 (td, <i>J</i> = 7.8, 1.9 Hz, 2H), 7.38 (dd, <i>J</i> = 7.5, 4.8 Hz, 2H), 7.22 (s, 1H), 7.16 (d, <i>J</i> = 8.0 Hz, 2H), 7.12 (s, 1H), 7.10 (d, <i>J</i> = 3.4 Hz, 1H), 6.87 (d, <i>J</i> = 3.4 Hz, 1H), 6.77 (t, <i>J</i> = 6.3 Hz, 1H), 6.32 (s, 1H), 5.78 (d, <i>J</i> = 7.0 Hz, 1H), 4.03 (s, 3H), 3.12 (q, <i>J</i> = 7.3 Hz, 2H), 2.43 (s, 3H), 1.25 (t, <i>J</i> = 7.3 Hz, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 161.1, 155.8, 148.9, 137.1, 137.0, 130.3, 128.9, 127.9, 127.5, 127.4, 125.1, 123.2, 122.8, 122.5, 121.8, 117.2, 111.4, 108.2, 99.4, 99.1, 69.9, 45.6, 35.3, 27.9, 8.2. HRMS (ESI) calcd for C<sub>30</sub>H<sub>29</sub>FN<sub>6</sub>O<sub>3</sub>S [M + H]<sup>+</sup>, 553.2016; found, 553.2017.</div></div><div id="sec6_88" class="NLM_sec NLM_sec_level_2"><div id="ac_i107" class="anchor-spacer"></div><h3 class="article-section__title" id="_i107"> <i>N</i>-(1-(1,1-Di(pyridin-2-yl)propyl)-6-(1-methyl-7-oxo-6,7-dihydro-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridin-3-yl)-1<i>H</i>-indol-4-yl)ethanesulfonamide (<b>28</b>)</h3><div class="NLM_p last">This compound was obtained by a procedure similar to the preparation of <b>27</b> (62 mg, yield 91%). <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 8.93 (dd, <i>J</i> = 5.4, 1.7 Hz, 2H), 8.13 (td, <i>J</i> = 8.0, 1.7 Hz, 2H), 8.07 (s, 1H), 7.95 (d, <i>J</i> = 3.6 Hz, 1H), 7.75 (dd, <i>J</i> = 7.6, 5.3 Hz, 2H), 7.25 (d, <i>J</i> = 1.2 Hz, 1H), 7.21 (d, <i>J</i> = 8.2 Hz, 2H), 7.09 (d, <i>J</i> = 3.5 Hz, 1H), 7.05 (s, 1H), 6.80 (d, <i>J</i> = 7.0 Hz, 1H), 6.05 (s, 1H), 5.76 (d, <i>J</i> = 7.0 Hz, 1H), 4.03 (s, 3H), 3.17 (q, <i>J</i> = 7.4 Hz, 2H), 2.96 (q, <i>J</i> = 7.3 Hz, 2H), 1.37 (t, <i>J</i> = 7.4 Hz, 3H), 0.84 (t, <i>J</i> = 7.2 Hz, 3H). <sup>13</sup>C NMR (101 MHz, methanol-<i>d</i><sub>4</sub>) δ 164.4, 160.5, 158.0, 146.0, 144.0, 138.1, 132.2, 131.24, 131.17, 130.7, 128.0, 126.3, 125.8, 125.7, 124.6, 124.0, 118.9, 114.6, 109.0, 102.2, 102.0, 68.5, 47.2, 38.4, 36.1, 9.0, 8.6. HRMS (ESI) calcd for C<sub>31</sub>H<sub>31</sub>N<sub>6</sub>O<sub>3</sub>S [M + H]<sup>+</sup>, 567.2173; found, 567.2151.</div></div><div id="sec6_89" class="NLM_sec NLM_sec_level_2"><div id="ac_i108" class="anchor-spacer"></div><h3 class="article-section__title" id="_i108"> <i>N</i>-(1-(1,1-Di(pyridin-2-yl)ethyl)-6-(1-methyl-7-oxo-6,7-dihydro-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridin-3-yl)-1<i>H</i>-indol-4-yl)isobutyramide (<b>29</b>)</h3><div class="NLM_p last">This compound was obtained by a procedure similar to the preparation of <b>27</b> (39 mg, yield 93%). <sup>1</sup>H NMR (400 MHz, acetone-<i>d</i><sub>6</sub>) δ 10.11 (s, 1H), 8.89 (s, 1H), 8.64 (ddd, <i>J</i> = 4.7, 1.9, 0.9 Hz, 2H), 8.10 (s, 1H), 7.76 (td, <i>J</i> = 7.8, 1.9 Hz, 2H), 7.35 (ddd, <i>J</i> = 7.6, 4.7, 1.1 Hz, 2H), 7.23 (dt, <i>J</i> = 8.1, 1.1 Hz, 2H), 7.12 (s, 1H), 7.00 (d, <i>J</i> = 3.5 Hz, 1H), 6.87 (d, <i>J</i> = 7.1 Hz, 1H), 6.76 (dd, <i>J</i> = 3.5, 0.9 Hz, 1H), 6.47 (t, <i>J</i> = 1.1 Hz, 1H), 6.08 (d, <i>J</i> = 7.1 Hz, 1H), 4.13 (s, 3H), 2.82 (m, 1H), 2.53 (s, 3H), 1.22 (d, <i>J</i> = 6.8 Hz, 6H). <sup>13</sup>C NMR (101 MHz, acetone-<i>d</i><sub>6</sub>) δ 176.2, 162.7, 157.1, 149.9, 138.1, 137.5, 132.6, 132.0, 129.5, 128.9, 127.6, 125.5, 124.6, 123.8, 123.5, 121.4, 119.8, 111.8, 108.8, 101.0, 98.9, 71.2, 36.4, 35.7, 27.9, 20.2. HRMS (ESI) calcd for C<sub>32</sub>H<sub>30</sub>N<sub>6</sub>O<sub>2</sub> [M + H]<sup>+</sup>, 531.2503; found, 531.2489.</div></div><div id="sec6_90" class="NLM_sec NLM_sec_level_2"><div id="ac_i109" class="anchor-spacer"></div><h3 class="article-section__title" id="_i109"> <i>N</i>-(1-(1,1-Di(pyridin-2-yl)ethyl)-6-(1-methyl-7-oxo-6,7-dihydro-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridin-3-yl)-1<i>H</i>-indol-4-yl)acetamide (<b>30</b>)</h3><div class="NLM_p last">This compound was obtained by a procedure similar to the preparation of <b>27</b> (44 mg, yield 96%). <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 9.17 (s, 1H), 8.74–8.63 (m, 2H), 7.93 (s, 1H), 7.63 (t, <i>J</i> = 7.8 Hz, 2H), 7.40 (s, 1H), 7.25–7.20 (m, 2H), 7.14 (d, <i>J</i> = 8.1 Hz, 2H), 7.03 (d, <i>J</i> = 3.4 Hz, 1H), 6.95 (s, 1H), 6.71 (t, <i>J</i> = 6.1 Hz, 1H), 6.47 (s, 1H), 6.15 (d, <i>J</i> = 7.0 Hz, 1H), 4.14 (s, 3H), 2.56 (s, 3H), 2.29 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 168.6, 161.2, 155.8, 148.9, 136.9, 136.8, 131.3, 128.7, 127.9, 127.2, 126.7, 125.0, 123.1, 122.8, 122.5, 120.3, 117.7, 110.8, 107.3, 99.3, 99.1, 69.8, 35.2, 27.9, 23.9. HRMS (ESI) calcd for C<sub>30</sub>H<sub>27</sub>N<sub>6</sub>O<sub>2</sub> [M + H]<sup>+</sup>, 503.2190; found, 503.2170.</div></div><div id="sec6_91" class="NLM_sec NLM_sec_level_2"><div id="ac_i110" class="anchor-spacer"></div><h3 class="article-section__title" id="_i110"> Methyl 6-Bromo-1-(di(pyridin-2-yl)methyl)-1<i>H</i>-indole-4-carboxylate (<b>64</b>)</h3><div class="NLM_p last">To a solution of <b>63</b> (1.27 g, 5 mmol) in DMF (20 mL) at RT were added K<sub>2</sub>CO<sub>3</sub> (1.38 g, 10 mmol) and 2,2′-(bromomethylene)dipyridine (1.87 g, 7.5 mmol), and then the mixture was stirred at 50 °C and quenched after 16 h by addition of water. The solution was extracted with ethyl acetate, and the combined organic layer was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>. Filtration and concentration in vacuo gave a residue, which was purified by flash column chromatography on silica gel (hexanes–EtOAc, 2:1) to obtain <b>64</b> (1.52 g, yield 72%) as a yellow solid. <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 8.55–8.52 (m, 2H), 7.95 (d, <i>J</i> = 1.7 Hz, 1H), 7.70 (t, <i>J</i> = 1.2 Hz, 1H), 7.54 (td, <i>J</i> = 7.7, 1.8 Hz, 2H), 7.32 (d, <i>J</i> = 3.3 Hz, 1H), 7.16–7.09 (m, 3H), 7.06–7.01 (m, 2H), 6.93 (s, 1H), 3.87 (s, 3H). LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> = 422.0.</div></div><div id="sec6_92" class="NLM_sec NLM_sec_level_2"><div id="ac_i111" class="anchor-spacer"></div><h3 class="article-section__title" id="_i111"> Methyl 6-Bromo-1-(1,1-di(pyridin-2-yl)ethyl)-1<i>H</i>-indole-4-carboxylate (<b>65</b>)</h3><div class="NLM_p last">To a solution of <b>64</b> (1.27 g, 3 mmol) in THF (10 mL) at 0 °C were added <i>t</i>-BuOK (1.01 g, 9 mmol) and MeI (0.56 mL, 9 mmol) successively. Then the solution was stirred at RT for 2 h, and water was added to quench the reaction. The mixture was extracted with ethyl acetate, and the combined organic layer was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>. Filtration and concentration in vacuo gave a residue, which was purified by flash column chromatography on silica gel (hexanes–EtOAc, 3:1) to obtain <b>65</b> (1.15 g, yield 88%) as a yellow solid. <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 8.63 (ddd, <i>J</i> = 4.8, 1.8, 0.9 Hz, 2H), 8.00 (d, <i>J</i> = 1.7 Hz, 1H), 7.70 (s, 1H), 7.67 (td, <i>J</i> = 7.8, 1.9 Hz, 2H), 7.32 (d, <i>J</i> = 3.3 Hz, 1H), 7.27 (d, <i>J</i> = 1.1 Hz, 1H), 7.28–7.21 (m, 1H), 7.16 (dd, <i>J</i> = 3.3, 0.8 Hz, 1H), 7.10 (d, <i>J</i> = 1.1 Hz, 1H), 7.08 (d, <i>J</i> = 1.0 Hz, 1H), 6.95 (s, 1H), 3.97 (s, 3H). LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> = 436.0.</div></div><div id="sec6_93" class="NLM_sec NLM_sec_level_2"><div id="ac_i112" class="anchor-spacer"></div><h3 class="article-section__title" id="_i112"> Methyl 1-(1,1-Di(pyridin-2-yl)ethyl)-6-(7-methoxy-1-methyl-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridin-3-yl)-1<i>H</i>-indole-4-carboxylate (<b>66</b>)</h3><div class="NLM_p last">A mixture of <b>65</b> (870 mg, 2 mmol), Pd(dppf)Cl<sub>2</sub>–DCM (163 mg, 0.2 mmol), <b>35</b> (693 mg, 2.4 mmol), and K<sub>3</sub>PO<sub>4</sub> (1.06 g, 5 mmol) in dioxane–water (10 mL, 4:1) was heated at 90 °C for 24 h. The mixture was then diluted with ethyl acetate and filtered to obtain a clear solution, which was washed three times with brine. The organic layer was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated. The crude product was purified by flash column chromatography on silica gel (hexanes–acetone, 1:1) to yield <b>66</b> (880 mg, yield 85%) as a gray solid. <sup>1</sup>H NMR (400 MHz, chloroform-<i>d</i>) δ 8.69 (ddd, <i>J</i> = 4.8, 1.9, 0.9 Hz, 2H), 8.02 (d, <i>J</i> = 1.4 Hz, 1H), 7.63 (td, <i>J</i> = 7.8, 1.9 Hz, 2H), 7.58 (d, <i>J</i> = 5.8 Hz, 1H), 7.25 (dd, <i>J</i> = 4.9, 1.2 Hz, 1H), 7.23 (dd, <i>J</i> = 4.8, 1.1 Hz, 1H), 7.21–7.16 (m, 4H), 7.05 (s, 1H), 7.01 (t, <i>J</i> = 1.1 Hz, 1H), 6.59 (d, <i>J</i> = 5.8 Hz, 1H), 4.06 (s, 3H), 4.03 (s, 3H), 4.00 (s, 3H), 2.57 (s, 3H). LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> = 518.3.</div></div><div id="sec6_94" class="NLM_sec NLM_sec_level_2"><div id="ac_i113" class="anchor-spacer"></div><h3 class="article-section__title" id="_i113"> 2-(1-(1,1-Di(pyridin-2-yl)ethyl)-6-(7-methoxy-1-methyl-1<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridin-3-yl)-1<i>H</i>-indol-4-yl)propan-2-ol (<b>67</b>)</h3><div class="NLM_p last">A mixture of <b>66</b> (776 mg, 1.5 mmol), TMSCl (0.31 mL, 2.4 mmol), NaI (360 mg, 2.4 mmol), and H<sub>2</sub>O (15 μL, 0.8 mmol) in acetonitrile (5 mL) was heated at 50 °C for 3 h and then cooled to room temperature. The mixture was then diluted with ethyl acetate and washed three times with aqueous NaHCO<sub>3</sub>. The organic layer was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated. The crude product <b>67</b> (720 mg) was obtained without further purification. LCMS (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup> = 504.2.</div></div><div id="sec6_95" class="NLM_sec NLM_sec_level_2"><div id="ac_i114" class="anchor-spacer"></div><h3 class="article-section__title" id="_i114"> 3-(1-(1,1-Di(pyridin-2-yl)ethyl)-4-(2-hydroxypropan-2-yl)-1<i>H</i>-indol-6-yl)-1-methyl-1,6-dihydro-7<i>H</i>-pyrrolo[2,3-<i>c</i>]pyridin-7-one (<b>31</b>)</h3><div class="NLM_p last">To a solution of <b>67</b> (50 mg, 0.1 mmol) in THF (3 mL) at −42 °C was added MeLi (0.8 mmol), and the mixture was stirred for 2 h. Water was added to quench the reaction. The mixture was extracted with ethyl acetate, and the combined organic layer was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>. Then filtration and concentration in vacuo gave a residue, which was purified by preparative TLC (DCM–MeOH, 20:1) to obtain <b>31</b> (27 mg, yield 55%) as a gray solid. <sup>1</sup>H NMR (400 MHz, acetone-<i>d</i><sub>6</sub>) δ 10.06 (s, 1H), 8.65 (ddd, <i>J</i> = 4.8, 1.9, 0.9 Hz, 2H), 7.77 (td, <i>J</i> = 7.8, 1.9 Hz, 2H), 7.37–7.35 (m, 2H), 7.34 (dd, <i>J</i> = 4.8, 1.1 Hz, 1H), 7.28 (dt, <i>J</i> = 8.0, 1.0 Hz, 2H), 7.16 (s, 1H), 6.98 (d, <i>J</i> = 3.5 Hz, 1H), 6.85 (d, <i>J</i> = 7.3 Hz, 2H), 6.83 (dd, <i>J</i> = 3.5, 0.9 Hz, 1H), 6.65 (t, <i>J</i> = 1.2 Hz, 1H), 5.96 (d, <i>J</i> = 7.0 Hz, 1H), 4.11 (s, 3H), 2.53 (s, 3H), 1.73 (s, 6H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 161.3, 155.8, 148.9, 142.6, 136.9, 136.7, 128.8, 128.0, 126.6, 126.3, 124.9, 124.7, 123.1, 122.7, 122.7, 118.0, 115.0, 109.8, 102.4, 99.4, 71.4, 69.7, 35.2, 31.1, 27.6. HRMS (ESI) calcd for C<sub>31</sub>H<sub>30</sub>N<sub>5</sub>O<sub>2</sub> [M + H]<sup>+</sup>, 504.2394; found, 504.2358.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i115"><a href="/doi/suppl/10.1021/acs.jmedchem.0c00456" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_38076" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_38076" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c00456?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00456</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Synthesis and characterization of all compounds and further experimental details (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00456/suppl_file/jm0c00456_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and some data (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00456/suppl_file/jm0c00456_si_002.xlsx" class="ext-link">XLSX</a>)</p></li><li><p class="inline">Coordinates information for structure representation (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00456/suppl_file/jm0c00456_si_003.pdb" class="ext-link">PDB</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00456/suppl_file/jm0c00456_si_001.pdf">jm0c00456_si_001.pdf (1.19 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00456/suppl_file/jm0c00456_si_002.xlsx">jm0c00456_si_002.xlsx (11.33 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00456/suppl_file/jm0c00456_si_003.pdb">jm0c00456_si_003.pdb (6.08 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The X-ray coordinates have been deposited with the Protein Data Bank. The code is <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6P05">6P05</a> (BRD4 BD1 domain with <b>27</b>). Authors will release the atomic coordinates and experimental data upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.0c00456" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_73563" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_73563" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Gregory R. J. Thatcher</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmaceutical Sciences, University of Illinois College of Pharmacy, University of Illinois at Chicago, 833 S. Wood Street, Chicago, Illinois 60612, United States</span>; 
    <span class="hlFld-Affiliation affiliation">UICentre
(Drug Discovery @ UIC), University of Illinois
at Chicago, 833 S. Wood Street, Chicago, Illinois 60612, United
States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-7757-1739" title="Orcid link">http://orcid.org/0000-0002-7757-1739</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#314559504552595443714458521f545544"><span class="__cf_email__" data-cfemail="710519100512191403310418125f141504">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rui Xiong</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmaceutical Sciences, University of Illinois College of Pharmacy, University of Illinois at Chicago, 833 S. Wood Street, Chicago, Illinois 60612, United States</span>; 
    <span class="hlFld-Affiliation affiliation">UICentre
(Drug Discovery @ UIC), University of Illinois
at Chicago, 833 S. Wood Street, Chicago, Illinois 60612, United
States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6350-9037" title="Orcid link">http://orcid.org/0000-0002-6350-9037</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#dfada7b6b0b1b8ec9faab6bcf1babbaa"><span class="__cf_email__" data-cfemail="215359484e4f4612615448420f444554">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yangfeng Li</span> - <span class="hlFld-Affiliation affiliation">UICentre
(Drug Discovery @ UIC), University of Illinois
at Chicago, 833 S. Wood Street, Chicago, Illinois 60612, United
States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jiong Zhao</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmaceutical Sciences, University of Illinois College of Pharmacy, University of Illinois at Chicago, 833 S. Wood Street, Chicago, Illinois 60612, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lauren M. Gutgesell</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmaceutical Sciences, University of Illinois College of Pharmacy, University of Illinois at Chicago, 833 S. Wood Street, Chicago, Illinois 60612, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-7645-9227" title="Orcid link">http://orcid.org/0000-0002-7645-9227</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Zhengnan Shen</span> - <span class="hlFld-Affiliation affiliation">UICentre
(Drug Discovery @ UIC), University of Illinois
at Chicago, 833 S. Wood Street, Chicago, Illinois 60612, United
States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kiira Ratia</span> - <span class="hlFld-Affiliation affiliation">UICentre
(Drug Discovery @ UIC), University of Illinois
at Chicago, 833 S. Wood Street, Chicago, Illinois 60612, United
States</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Pharmaceutical Sciences, University of Illinois College of Pharmacy, University of Illinois at Chicago, 833 S. Wood Street, Chicago, Illinois 60612, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Research
Resources Center, University of Illinois
at Chicago, Chicago, Illinois 60612, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Katherine Dye</span> - <span class="hlFld-Affiliation affiliation">UICentre
(Drug Discovery @ UIC), University of Illinois
at Chicago, 833 S. Wood Street, Chicago, Illinois 60612, United
States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Oleksii Dubrovskyi</span> - <span class="hlFld-Affiliation affiliation">UICentre
(Drug Discovery @ UIC), University of Illinois
at Chicago, 833 S. Wood Street, Chicago, Illinois 60612, United
States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Huiping Zhao</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmaceutical Sciences, University of Illinois College of Pharmacy, University of Illinois at Chicago, 833 S. Wood Street, Chicago, Illinois 60612, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Fei Huang</span> - <span class="hlFld-Affiliation affiliation">UICentre
(Drug Discovery @ UIC), University of Illinois
at Chicago, 833 S. Wood Street, Chicago, Illinois 60612, United
States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Debra A. Tonetti</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmaceutical Sciences, University of Illinois College of Pharmacy, University of Illinois at Chicago, 833 S. Wood Street, Chicago, Illinois 60612, United States</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>The manuscript was written and the research directed by R.X. and G.R.J.T. Drug design and synthesis was directed by R.X. The largest contribution to synthesis was from Y.L. and R.X. Intermediate synthesis, scale-up, synthetic optimization, and purification were supported by Z.S. Cell line and bioassay development was led by J.Z. and R.X. Animal study was led by D.A.T. and J.Z. Biological support was provided by L.M.G. and F.H. PK was supported by K.D. and O.D. X-ray cocrystal structure was led by K.R. All authors have given approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i118">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_12326" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_12326" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This study is supported in part by UICentre via NIH Grant UL1RR029879, a sponsored research grant from G1 Therapeutics Inc., and an Accelerator Award from Chicago Biomedical Consortium. We acknowledge Dr. Hyun Lee and Isabel Ojeda for protein purification support.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">ADMET</td><td class="NLM_def"><p class="first last">absorption, distribution, metabolism, excretion, and toxicity</p></td></tr><tr><td class="NLM_term">AI</td><td class="NLM_def"><p class="first last">aromatase inhibitor</p></td></tr><tr><td class="NLM_term">BET</td><td class="NLM_def"><p class="first last">bromodomain and extra-terminal</p></td></tr><tr><td class="NLM_term">CDK</td><td class="NLM_def"><p class="first last">cyclin-dependent kinase</p></td></tr><tr><td class="NLM_term">CYP450</td><td class="NLM_def"><p class="first last">cytochrome P450</p></td></tr><tr><td class="NLM_term">DMF</td><td class="NLM_def"><p class="first last">dimethylformamide</p></td></tr><tr><td class="NLM_term">E2</td><td class="NLM_def"><p class="first last">estradiol</p></td></tr><tr><td class="NLM_term">EP300</td><td class="NLM_def"><p class="first last">E1A-associated protein p300</p></td></tr><tr><td class="NLM_term">ER</td><td class="NLM_def"><p class="first last">estrogen receptor</p></td></tr><tr><td class="NLM_term">ERE</td><td class="NLM_def"><p class="first last">estrogen response element</p></td></tr><tr><td class="NLM_term">ET</td><td class="NLM_def"><p class="first last">extra C-terminal domain</p></td></tr><tr><td class="NLM_term">hERG</td><td class="NLM_def"><p class="first last">human ether-a-go-go-related gene</p></td></tr><tr><td class="NLM_term">Ful</td><td class="NLM_def"><p class="first last">fulvestrant</p></td></tr><tr><td class="NLM_term">HPBCD</td><td class="NLM_def"><p class="first last">hydroxypropyl β-cyclodextrin</p></td></tr><tr><td class="NLM_term">LLE</td><td class="NLM_def"><p class="first last">ligand-lipophilicity efficiency</p></td></tr><tr><td class="NLM_term">MED1</td><td class="NLM_def"><p class="first last">mediator complex subunit 1</p></td></tr><tr><td class="NLM_term">nM</td><td class="NLM_def"><p class="first last">nanomolar</p></td></tr><tr><td class="NLM_term">PEG400</td><td class="NLM_def"><p class="first last">polyethylene glycol 400</p></td></tr><tr><td class="NLM_term">PK</td><td class="NLM_def"><p class="first last">pharmacokinetics</p></td></tr><tr><td class="NLM_term">PROTAC</td><td class="NLM_def"><p class="first last">proteolysis targeting chimera</p></td></tr><tr><td class="NLM_term">P-TEFb</td><td class="NLM_def"><p class="first last">positive transcription elongation factor b</p></td></tr><tr><td class="NLM_term">RNAPII</td><td class="NLM_def"><p class="first last">RNA polymerase II</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr><tr><td class="NLM_term">SERD</td><td class="NLM_def"><p class="first last">selective estrogen receptor degrader</p></td></tr><tr><td class="NLM_term">TAF-1</td><td class="NLM_def"><p class="first last">TBP-associated factor 1</p></td></tr><tr><td class="NLM_term">TAM</td><td class="NLM_def"><p class="first last">tamoxifen</p></td></tr><tr><td class="NLM_term">TR-FRET</td><td class="NLM_def"><p class="first last">time-resolved fluorescence resonance energy transfer</p></td></tr><tr><td class="NLM_term">YY1</td><td class="NLM_def"><p class="first last">Yin Yang 1</p></td></tr><tr><td class="NLM_term">μM</td><td class="NLM_def"><p class="first last">micromolar</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i120">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_25856" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_25856" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 83 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span>U.S. Breast
Cancer Statistics. <a href="https://www.breastcancer.org/symptoms/understand_bc/statistics" class="extLink">https://www.breastcancer.org/symptoms/understand_bc/statistics</a> (accessed Mar 7, <span class="NLM_year">2019</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=U.S.+Breast%0ACancer+Statistics.+https%3A%2F%2Fwww.breastcancer.org%2Fsymptoms%2Funderstand_bc%2Fstatistics+%28accessed+Mar+7%2C+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Early
Breast Cancer Trialists’ Collaborative Group.</span> <span> </span><span class="NLM_article-title">Tamoxifen for early breast cancer:
an overview of the randomised trials</span>. <i>Lancet</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>351</i></span>,  <span class="NLM_fpage">1451</span>– <span class="NLM_lpage">1467</span>, <span class="refDoi"> DOI: 10.1016/S0140-6736(97)11423-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1016%2FS0140-6736%2897%2911423-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=9605801" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=351&publication_year=1998&pages=1451-1467&author=Early%0ABreast+Cancer+Trialists%E2%80%99+Collaborative+Group.&title=Tamoxifen+for+early+breast+cancer%3A%0Aan+overview+of+the+randomised+trials&doi=10.1016%2FS0140-6736%2897%2911423-4"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2FS0140-6736%2897%2911423-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0140-6736%252897%252911423-4%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DTamoxifen%2520for%2520early%2520breast%2520cancer%253A%250Aan%2520overview%2520of%2520the%2520randomised%2520trials%26jtitle%3DLancet%26date%3D1998%26volume%3D351%26spage%3D1451%26epage%3D1467%26doi%3D10.1016%2FS0140-6736%2897%2911423-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roodi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bailey, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kao, W. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verrier, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yee, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dupont, W. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parl, F. F.</span></span> <span> </span><span class="NLM_article-title">Estrogen receptor gene analysis in estrogen receptor-positive and receptor-negative primary breast cancer</span>. <i>J. Natl. Cancer Inst.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>87</i></span>,  <span class="NLM_fpage">446</span>– <span class="NLM_lpage">451</span>, <span class="refDoi"> DOI: 10.1093/jnci/87.6.446</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1093%2Fjnci%2F87.6.446" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=7861463" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADyaK2MXkvVGntL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=87&publication_year=1995&pages=446-451&author=N.+Roodiauthor=L.+R.+Baileyauthor=W.+Y.+Kaoauthor=C.+S.+Verrierauthor=C.+J.+Yeeauthor=W.+D.+Dupontauthor=F.+F.+Parl&title=Estrogen+receptor+gene+analysis+in+estrogen+receptor-positive+and+receptor-negative+primary+breast+cancer&doi=10.1093%2Fjnci%2F87.6.446"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Estrogen receptor gene analysis in estrogen receptor-positive and receptor-negative primary breast cancer</span></div><div class="casAuthors">Roodi, Nady; Bailey, L. Renee; Kao, Woei-Yau; Verrier, Carmel S.; Yee, Cindy J.; Dupont, William D.; Parl, Fritz F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the National Cancer Institute</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">87</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">446-51</span>CODEN:
                <span class="NLM_cas:coden">JNCIEQ</span>;
        ISSN:<span class="NLM_cas:issn">0027-8874</span>.
    </div><div class="casAbstract">In breast cancer patients, about two thirds of the tumors are estrogen receptor (ER)-pos. and one third are ER-neg.  The mol. mechanisms leading to the ER-neg. phenotype are poorly understood.  Nearly all ER-neg. and about 40% of ER-pos. cancers are resistant to endocrine therapy.  In this study, the authors examd. the entire coding region of the ER gene in ER-pos. and ER-neg. primary breast tumors to det. whether deletions/insertions or point mutations might account for the ER-neg. phenotype.  The authors amplified exons 1 through 8 of the ER gene in 118 ER-pos. and 70 ER-neg. primary breast tumors and searched for mutations by single-strand conformation polymorphism anal., denaturing gradient gel electrophoresis, and DNA sequencing.  Both ER-neg. and ER-pos. tumors contained neutral polymorphisms in codons 10 [TCT→TCC (Ser)], 87 [GCG→GCC (Ala)], 243 [CGC→CGT (Arg)], 325 [CCC→CCG (Pro)], and 594 [ACA→ACG (Thr)].  There was no correlation of any of the polymorphic alleles with the ER phenotype or other clinicopathol. parameters including tumor type, size, grade, or stage.  However, the polymorphism in codon 325 showed a strong assocn. with a family history of breast cancer.  This assocn. was obsd. both in premenopausal and postmenopausal patients.  Despite extensive searching in exons 1 through 8, the authors found no deletions/insertions and only two missense mutations in codons 69 [AAC (Asn)→AAG (Lys)] and 396 [ATG (Met)→GTG (Val)] of the same ER-neg. tumor.  Thus, only 1% of the primary breast cancers had point mutations in the ER gene.  In the majority of primary breast cancers, the ER-neg. phenotype is not the result of mutations in the coding region of the ER gene, but is due to deficient ER expression at the transcriptional or post-transcriptional level.  The correlation reported previously, as well as the current findings, suggest that further investigations are warranted to understand the possible linkage of the ER gene locus to hereditary breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0jnOyp24XmbVg90H21EOLACvtfcHk0lgWsoN6Pw3weA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXkvVGntL4%253D&md5=c45eb7d3beda848c2dcbad1538a8296f</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1093%2Fjnci%2F87.6.446&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjnci%252F87.6.446%26sid%3Dliteratum%253Aachs%26aulast%3DRoodi%26aufirst%3DN.%26aulast%3DBailey%26aufirst%3DL.%2BR.%26aulast%3DKao%26aufirst%3DW.%2BY.%26aulast%3DVerrier%26aufirst%3DC.%2BS.%26aulast%3DYee%26aufirst%3DC.%2BJ.%26aulast%3DDupont%26aufirst%3DW.%2BD.%26aulast%3DParl%26aufirst%3DF.%2BF.%26atitle%3DEstrogen%2520receptor%2520gene%2520analysis%2520in%2520estrogen%2520receptor-positive%2520and%2520receptor-negative%2520primary%2520breast%2520cancer%26jtitle%3DJ.%2520Natl.%2520Cancer%2520Inst.%26date%3D1995%26volume%3D87%26spage%3D446%26epage%3D451%26doi%3D10.1093%2Fjnci%2F87.6.446" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Riggins, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schrecengost, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guerrero, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouton, A. H.</span></span> <span> </span><span class="NLM_article-title">Pathways to tamoxifen resistance</span>. <i>Cancer Lett.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>256</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">24</span>, <span class="refDoi"> DOI: 10.1016/j.canlet.2007.03.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1016%2Fj.canlet.2007.03.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=17475399" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BD2sXpvFGrtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=256&publication_year=2007&pages=1-24&author=R.+B.+Rigginsauthor=R.+S.+Schrecengostauthor=M.+S.+Guerreroauthor=A.+H.+Bouton&title=Pathways+to+tamoxifen+resistance&doi=10.1016%2Fj.canlet.2007.03.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Pathways to tamoxifen resistance</span></div><div class="casAuthors">Riggins, Rebecca B.; Schrecengost, Randy S.; Guerrero, Michael S.; Bouton, Amy H.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Letters (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">256</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-24</span>CODEN:
                <span class="NLM_cas:coden">CALEDQ</span>;
        ISSN:<span class="NLM_cas:issn">0304-3835</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Therapies that target the synthesis of estrogen or the function of estrogen receptor(s) have been developed to treat breast cancer.  While these approaches have proven to be beneficial to a large no. of patients, both de novo and acquired resistance to these drugs is a significant problem.  Recent advances in our understanding of the mol. mechanisms that contribute to resistance have provided a means to begin to predict patient responses to these drugs and develop rational approaches for combining therapeutic agents to circumvent or desensitize the resistant phenotype.  Here, we review common mechanisms of antiestrogen resistance and discuss the implications for prediction of response and design of effective combinatorial treatments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreh_AisSEW9LVg90H21EOLACvtfcHk0lhIRBItZ8urHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXpvFGrtLo%253D&md5=25203d3e50ade6508421835d29bf048c</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.canlet.2007.03.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.canlet.2007.03.016%26sid%3Dliteratum%253Aachs%26aulast%3DRiggins%26aufirst%3DR.%2BB.%26aulast%3DSchrecengost%26aufirst%3DR.%2BS.%26aulast%3DGuerrero%26aufirst%3DM.%2BS.%26aulast%3DBouton%26aufirst%3DA.%2BH.%26atitle%3DPathways%2520to%2520tamoxifen%2520resistance%26jtitle%3DCancer%2520Lett.%26date%3D2007%26volume%3D256%26spage%3D1%26epage%3D24%26doi%3D10.1016%2Fj.canlet.2007.03.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Robertson, J. F. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bondarenko, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trishkina, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dvorkin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panasci, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manikhas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shparyk, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cardona-Huerta, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philco-Salas, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruiz-Borrego, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noguchi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rowbottom, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuart, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grinsted, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fazal, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, M. J.</span></span> <span> </span><span class="NLM_article-title">Fulvestrant 500 mg versus anastrozole 1 mg for hormone receptor-positive advanced breast cancer (FALCON): an international, randomised, double-blind, phase 3 trial</span>. <i>Lancet</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>388</i></span>,  <span class="NLM_fpage">2997</span>– <span class="NLM_lpage">3005</span>, <span class="refDoi"> DOI: 10.1016/S0140-6736(16)32389-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1016%2FS0140-6736%2816%2932389-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=27908454" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFKnurnK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=388&publication_year=2016&pages=2997-3005&author=J.+F.+R.+Robertsonauthor=I.+M.+Bondarenkoauthor=E.+Trishkinaauthor=M.+Dvorkinauthor=L.+Panasciauthor=A.+Manikhasauthor=Y.+Shparykauthor=S.+Cardona-Huertaauthor=K.+L.+Cheungauthor=M.+J.+Philco-Salasauthor=M.+Ruiz-Borregoauthor=Z.+Shaoauthor=S.+Noguchiauthor=J.+Rowbottomauthor=M.+Stuartauthor=L.+M.+Grinstedauthor=M.+Fazalauthor=M.+J.+Ellis&title=Fulvestrant+500+mg+versus+anastrozole+1+mg+for+hormone+receptor-positive+advanced+breast+cancer+%28FALCON%29%3A+an+international%2C+randomised%2C+double-blind%2C+phase+3+trial&doi=10.1016%2FS0140-6736%2816%2932389-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Fulvestrant 500 mg versus anastrozole 1 mg for hormone receptor-positive advanced breast cancer (FALCON): an international, randomised, double-blind, phase 3 trial</span></div><div class="casAuthors">Robertson, John F. R.; Bondarenko, Igor M.; Trishkina, Ekaterina; Dvorkin, Mikhail; Panasci, Lawrence; Manikhas, Alexey; Shparyk, Yaroslav; Cardona-Huerta, Servando; Cheung, Kwok-Leung; Philco-Salas, Manuel Jesus; Ruiz-Borrego, Manuel; Shao, Zhimin; Noguchi, Shinzaburo; Rowbottom, Jacqui; Stuart, Mary; Grinsted, Lynda M.; Fazal, Mehdi; Ellis, Matthew J.</div><div class="citationInfo"><span class="NLM_cas:title">Lancet</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">388</span>
        (<span class="NLM_cas:issue">10063</span>),
    <span class="NLM_cas:pages">2997-3005</span>CODEN:
                <span class="NLM_cas:coden">LANCAO</span>;
        ISSN:<span class="NLM_cas:issn">0140-6736</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Aromatase inhibitors are a std. of care for hormone receptor-pos. locally advanced or metastatic breast cancer.  We investigated whether the selective estrogen receptor degrader fulvestrant could improve progression-free survival compared with anastrozole in postmenopausal patients who had not received previous endocrine therapy.  In this phase 3, randomised, double-blind trial, we recruited eligible patients with histol. confirmed estrogen receptor-pos. or progesterone receptor-pos., or both, locally advanced or metastatic breast cancer from 113 academic hospitals and community centers in 20 countries.  Eligible patients were endocrine therapy-naive, with WHO performance status 0-2, and at least one measurable or non-measurable lesion.  Patients were randomly assigned (1:1) to fulvestrant (500 mg i.m. injection; on days 0, 14, 28, then every 28 days thereafter) or anastrozole (1 mg orally daily) using a computer-generated randomisation scheme.  The primary endpoint was progression-free survival, detd. by Response Evaluation Criteria in Solid Tumors version 1·1, intervention by surgery or radiotherapy because of disease deterioration, or death from any cause, assessed in the intention-to-treat population.  Safety outcomes were assessed in all patients who received at least one dose of randomised treatment (including placebo).  This trial is registered with ClinicalTrials.gov, no. NCT01602380.  Between Oct 17, 2012, and July 11, 2014, 524 patients were enrolled to this study.  Of these, 462 patients were randomised (230 to receive fulvestrant and 232 to receive anastrozole).  Progression-free survival was significantly longer in the fulvestrant group than in the anastrozole group (hazard ratio [HR] 0·797, 95% CI 0·637-0·999, p=0·0486).  Median progression-free survival was 16·6 mo (95% CI 13·83-20·99) in the fulvestrant group vs. 13·8 mo (11·99-16·59) in the anastrozole group.  The most common adverse events were arthralgia (38 [17%] in the fulvestrant group vs 24 [10%] in the anastrozole group) and hot flushes (26 [11%] in the fulvestrant group vs 24 [10%] in the anastrozole group). 16 (7%) of 228 patients in in the fulvestrant group and 11 (5%) of 232 patients in the anastrozole group discontinued because of adverse events.  Fulvestrant has superior efficacy and is a preferred treatment option for patients with hormone receptor-pos. locally advanced or metastatic breast cancer who have not received previous endocrine therapy compared with a third-generation aromatase inhibitor, a std. of care for first-line treatment of these patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGry16EI3NgNOLVg90H21EOLACvtfcHk0lhIRBItZ8urHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFKnurnK&md5=7d8295fdf38fc5eb5720a7d3cd97d6e9</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2FS0140-6736%2816%2932389-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0140-6736%252816%252932389-3%26sid%3Dliteratum%253Aachs%26aulast%3DRobertson%26aufirst%3DJ.%2BF.%2BR.%26aulast%3DBondarenko%26aufirst%3DI.%2BM.%26aulast%3DTrishkina%26aufirst%3DE.%26aulast%3DDvorkin%26aufirst%3DM.%26aulast%3DPanasci%26aufirst%3DL.%26aulast%3DManikhas%26aufirst%3DA.%26aulast%3DShparyk%26aufirst%3DY.%26aulast%3DCardona-Huerta%26aufirst%3DS.%26aulast%3DCheung%26aufirst%3DK.%2BL.%26aulast%3DPhilco-Salas%26aufirst%3DM.%2BJ.%26aulast%3DRuiz-Borrego%26aufirst%3DM.%26aulast%3DShao%26aufirst%3DZ.%26aulast%3DNoguchi%26aufirst%3DS.%26aulast%3DRowbottom%26aufirst%3DJ.%26aulast%3DStuart%26aufirst%3DM.%26aulast%3DGrinsted%26aufirst%3DL.%2BM.%26aulast%3DFazal%26aufirst%3DM.%26aulast%3DEllis%26aufirst%3DM.%2BJ.%26atitle%3DFulvestrant%2520500%2520mg%2520versus%2520anastrozole%25201%2520mg%2520for%2520hormone%2520receptor-positive%2520advanced%2520breast%2520cancer%2520%2528FALCON%2529%253A%2520an%2520international%252C%2520randomised%252C%2520double-blind%252C%2520phase%25203%2520trial%26jtitle%3DLancet%26date%3D2016%26volume%3D388%26spage%3D2997%26epage%3D3005%26doi%3D10.1016%2FS0140-6736%2816%2932389-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Toogood, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harvey, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Repine, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheehan, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">VanderWel, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keller, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNamara, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherry, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brodfuehrer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barvian, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fry, D. W.</span></span> <span> </span><span class="NLM_article-title">Discovery of a potent and selective inhibitor of cyclin-dependent kinase 4/6</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">2388</span>– <span class="NLM_lpage">2406</span>, <span class="refDoi"> DOI: 10.1021/jm049354h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm049354h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BD2MXitVGisL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=2388-2406&author=P.+L.+Toogoodauthor=P.+J.+Harveyauthor=J.+T.+Repineauthor=D.+J.+Sheehanauthor=S.+N.+VanderWelauthor=H.+Zhouauthor=P.+R.+Kellerauthor=D.+J.+McNamaraauthor=D.+Sherryauthor=T.+Zhuauthor=J.+Brodfuehrerauthor=C.+Choiauthor=M.+R.+Barvianauthor=D.+W.+Fry&title=Discovery+of+a+potent+and+selective+inhibitor+of+cyclin-dependent+kinase+4%2F6&doi=10.1021%2Fjm049354h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Potent and Selective Inhibitor of Cyclin-Dependent Kinase 4/6</span></div><div class="casAuthors">Toogood, Peter L.; Harvey, Patricia J.; Repine, Joseph T.; Sheehan, Derek J.; VanderWel, Scott N.; Zhou, Hairong; Keller, Paul R.; McNamara, Dennis J.; Sherry, Debra; Zhu, Tong; Brodfuehrer, Joanne; Choi, Chung; Barvian, Mark R.; Fry, David W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2388-2406</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A pharmacol. approach to inhibition of cyclin-dependent kinases 4 and 6 (Cdk4/6) using highly selective small mol. inhibitors has the potential to provide novel cancer therapies for clin. use.  Achieving high levels of selectivity for Cdk4/6, vs. other ATP-dependent kinases, presents a significant challenge.  The pyrido[2,3-d]pyrimidin-7-one template provides an effective platform for the inhibition of a broad cross-section of kinases, including Cdks.  It is now demonstrated that the modification of pyrido[2,3-d]pyrimidin-7-ones to include a 2-aminopyridine side chain at the C2-position provides inhibitors with exquisite selectivity for Cdk4/6 in vitro.  This selectivity profile is recapitulated in cells where the most selective inhibitors create a G1 block at concns. up to 100-fold the IC50 for cell proliferation.  On the basis of its selectivity profile and pharmacokinetic profile, compd. I (PD 0332991) was identified as a drug candidate for the treatment of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoX577J5U0r97Vg90H21EOLACvtfcHk0lgdjXcS3lCWIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXitVGisL0%253D&md5=23cbe58426ff1975695712db7c1b128a</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Fjm049354h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm049354h%26sid%3Dliteratum%253Aachs%26aulast%3DToogood%26aufirst%3DP.%2BL.%26aulast%3DHarvey%26aufirst%3DP.%2BJ.%26aulast%3DRepine%26aufirst%3DJ.%2BT.%26aulast%3DSheehan%26aufirst%3DD.%2BJ.%26aulast%3DVanderWel%26aufirst%3DS.%2BN.%26aulast%3DZhou%26aufirst%3DH.%26aulast%3DKeller%26aufirst%3DP.%2BR.%26aulast%3DMcNamara%26aufirst%3DD.%2BJ.%26aulast%3DSherry%26aufirst%3DD.%26aulast%3DZhu%26aufirst%3DT.%26aulast%3DBrodfuehrer%26aufirst%3DJ.%26aulast%3DChoi%26aufirst%3DC.%26aulast%3DBarvian%26aufirst%3DM.%2BR.%26aulast%3DFry%26aufirst%3DD.%2BW.%26atitle%3DDiscovery%2520of%2520a%2520potent%2520and%2520selective%2520inhibitor%2520of%2520cyclin-dependent%2520kinase%25204%252F6%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D2388%26epage%3D2406%26doi%3D10.1021%2Fjm049354h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Finn, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crown, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lang, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boer, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bondarenko, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kulyk, S. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ettl, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinter, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shparyk, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thummala, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voytko, N. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fowst, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Randolph, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slamon, D. J.</span></span> <span> </span><span class="NLM_article-title">The cyclin-dependent kinase 4/6 inhibitor palbociclib in combination with letrozole versus letrozole alone as first-line treatment of oestrogen receptor-positive, HER2-negative, advanced breast cancer (PALOMA-1/TRIO-18): a randomised phase 2 study</span>. <i>Lancet Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">25</span>– <span class="NLM_lpage">35</span>, <span class="refDoi"> DOI: 10.1016/S1470-2045(14)71159-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1016%2FS1470-2045%2814%2971159-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=25524798" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1yhsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2015&pages=25-35&author=R.+S.+Finnauthor=J.+P.+Crownauthor=I.+Langauthor=K.+Boerauthor=I.+M.+Bondarenkoauthor=S.+O.+Kulykauthor=J.+Ettlauthor=R.+Patelauthor=T.+Pinterauthor=M.+Schmidtauthor=Y.+Shparykauthor=A.+R.+Thummalaauthor=N.+L.+Voytkoauthor=C.+Fowstauthor=X.+Huangauthor=S.+T.+Kimauthor=S.+Randolphauthor=D.+J.+Slamon&title=The+cyclin-dependent+kinase+4%2F6+inhibitor+palbociclib+in+combination+with+letrozole+versus+letrozole+alone+as+first-line+treatment+of+oestrogen+receptor-positive%2C+HER2-negative%2C+advanced+breast+cancer+%28PALOMA-1%2FTRIO-18%29%3A+a+randomised+phase+2+study&doi=10.1016%2FS1470-2045%2814%2971159-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">The cyclin-dependent kinase 4/6 inhibitor palbociclib in combination with letrozole versus letrozole alone as first-line treatment of oestrogen receptor-positive, HER2-negative, advanced breast cancer (PALOMA-1/TRIO-18): a randomised phase 2 study</span></div><div class="casAuthors">Finn, Richard S.; Crown, John P.; Lang, Istvan; Boer, Katalin; Bondarenko, Igor M.; Kulyk, Sergey O.; Ettl, Johannes; Patel, Ravindranath; Pinter, Tamas; Schmidt, Marcus; Shparyk, Yaroslav; Thummala, Anu R.; Voytko, Nataliya L.; Fowst, Camilla; Huang, Xin; Kim, Sindy T.; Randolph, Sophia; Slamon, Dennis J.</div><div class="citationInfo"><span class="NLM_cas:title">Lancet Oncology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">25-35</span>CODEN:
                <span class="NLM_cas:coden">LOANBN</span>;
        ISSN:<span class="NLM_cas:issn">1470-2045</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Palbociclib (PD-0332991) is an oral, small-mol. inhibitor of cyclin-dependent kinases (CDKs) 4 and 6 with preclin. evidence of growth-inhibitory activity in estrogen receptor-pos. breast cancer cells and synergy with anti-estrogens.  We aimed to assess the safety and efficacy of palbociclib in combination with letrozole as first-line treatment of patients with advanced, estrogen receptor-pos., HER2-neg. breast cancer.  In this open-label, randomised phase 2 study, postmenopausal women with advanced estrogen receptor-pos. and HER2-neg. breast cancer who had not received any systemic treatment for their advanced disease were eligible to participate.  Patients were enrolled in two sep. cohorts that accrued sequentially: in cohort 1, patients were enrolled on the basis of their estrogen receptor-pos. and HER2-neg. biomarker status alone, whereas in cohort 2 they were also required to have cancers with amplification of cyclin D1 (CCND1), loss of p16 (INK4A or CDKN2A), or both.  In both cohorts, patients were randomly assigned 1:1 via an interactive web-based randomization system, stratified by disease site and disease-free interval, to receive continuous oral letrozole 2·5 mg daily or continuous oral letrozole 2·5 mg daily plus oral palbociclib 125 mg, given once daily for 3 wk followed by 1 wk off over 28-day cycles.  The primary endpoint was investigator-assessed progression-free survival in the intention-to-treat population.  Accrual to cohort 2 was stopped after an unplanned interim anal. of cohort 1 and the statistical anal. plan for the primary endpoint was amended to a combined anal. of cohorts 1 and 2 (instead of cohort 2 alone).  The study is ongoing but closed to accrual; these are the results of the final anal. of progression-free survival.  The study is registered with the ClinicalTrials.gov, no. NCT00721409.  Between Dec 22, 2009, and May 12, 2012, we randomly assigned 165 patients, 84 to palbociclib plus letrozole and 81 to letrozole alone.  At the time of the final anal. for progression-free survival (median follow-up 29·6 mo [95% CI 27·9-36·0] for the palbociclib plus letrozole group and 27·9 mo [25·5-31·1] for the letrozole group), 41 progression-free survival events had occurred in the palbociclib plus letrozole group and 59 in the letrozole group.  Median progression-free survival was 10·2 mo (95% CI 5·7-12·6) for the letrozole group and 20·2 mo (13·8-27·5) for the palbociclib plus letrozole group (HR 0·488, 95% CI 0·319-0·748; one-sided p=0·0004).  In cohort 1 (n=66), median progression-free survival was 5·7 mo (2·6-10·5) for the letrozole group and 26·1 mo (11·2-not estimable) for the palbociclib plus letrozole group (HR 0·299, 0·156-0·572; one-sided p<0·0001); in cohort 2 (n=99), median progression-free survival was 11·1 mo (7·1-16·4) for the letrozole group and 18·1 mo (13·1-27·5) for the palbociclib plus letrozole group (HR 0·508, 0·303-0·853; one-sided p=0·0046).  Grade 3-4 neutropenia was reported in 45 (54%) of 83 patients in the palbociclib plus letrozole group vs. one (1%) of 77 patients in the letrozole group, leucopenia in 16 (19%) vs. none, and fatigue in four (4%) vs. one (1%).  Serious adverse events that occurred in more than one patient in the palbociclib plus letrozole group were pulmonary embolism (three [4%] patients), back pain (two [2%]), and diarrhoea (two [2%]).  No cases of febrile neutropenia or neutropenia-related infections were reported during the study. 11 (13%) patients in the palbociclib plus letrozole group and two (2%) in the letrozole group discontinued the study because of adverse events.  The addn. of palbociclib to letrozole in this phase 2 study significantly improved progression-free survival in women with advanced estrogen receptor-pos. and HER2-neg. breast cancer.  A phase 3 trial is currently underway.Pfizer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBgFyFPQQlcbVg90H21EOLACvtfcHk0lgdjXcS3lCWIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1yhsb8%253D&md5=9163a35e22c50fb25782d439aff88bcb</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2FS1470-2045%2814%2971159-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1470-2045%252814%252971159-3%26sid%3Dliteratum%253Aachs%26aulast%3DFinn%26aufirst%3DR.%2BS.%26aulast%3DCrown%26aufirst%3DJ.%2BP.%26aulast%3DLang%26aufirst%3DI.%26aulast%3DBoer%26aufirst%3DK.%26aulast%3DBondarenko%26aufirst%3DI.%2BM.%26aulast%3DKulyk%26aufirst%3DS.%2BO.%26aulast%3DEttl%26aufirst%3DJ.%26aulast%3DPatel%26aufirst%3DR.%26aulast%3DPinter%26aufirst%3DT.%26aulast%3DSchmidt%26aufirst%3DM.%26aulast%3DShparyk%26aufirst%3DY.%26aulast%3DThummala%26aufirst%3DA.%2BR.%26aulast%3DVoytko%26aufirst%3DN.%2BL.%26aulast%3DFowst%26aufirst%3DC.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DKim%26aufirst%3DS.%2BT.%26aulast%3DRandolph%26aufirst%3DS.%26aulast%3DSlamon%26aufirst%3DD.%2BJ.%26atitle%3DThe%2520cyclin-dependent%2520kinase%25204%252F6%2520inhibitor%2520palbociclib%2520in%2520combination%2520with%2520letrozole%2520versus%2520letrozole%2520alone%2520as%2520first-line%2520treatment%2520of%2520oestrogen%2520receptor-positive%252C%2520HER2-negative%252C%2520advanced%2520breast%2520cancer%2520%2528PALOMA-1%252FTRIO-18%2529%253A%2520a%2520randomised%2520phase%25202%2520study%26jtitle%3DLancet%2520Oncol.%26date%3D2015%26volume%3D16%26spage%3D25%26epage%3D35%26doi%3D10.1016%2FS1470-2045%2814%2971159-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Perey, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paridaens, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hawle, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nole, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wildiers, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fiche, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dietrich, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clement, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koberle, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldhirsch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thurlimann, B.</span></span> <span> </span><span class="NLM_article-title">Clinical benefit of fulvestrant in postmenopausal women with advanced breast cancer and primary or acquired resistance to aromatase inhibitors: final results of phase II Swiss Group for Clinical Cancer Research Trial (SAKK 21/00)</span>. <i>Ann. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">64</span>– <span class="NLM_lpage">69</span>, <span class="refDoi"> DOI: 10.1093/annonc/mdl341</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1093%2Fannonc%2Fmdl341" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=17030543" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A280%3ADC%252BD2s%252FisVWntQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2007&pages=64-69&author=L.+Pereyauthor=R.+Paridaensauthor=H.+Hawleauthor=K.+Zamanauthor=F.+Noleauthor=H.+Wildiersauthor=M.+Ficheauthor=D.+Dietrichauthor=P.+Clementauthor=D.+Koberleauthor=A.+Goldhirschauthor=B.+Thurlimann&title=Clinical+benefit+of+fulvestrant+in+postmenopausal+women+with+advanced+breast+cancer+and+primary+or+acquired+resistance+to+aromatase+inhibitors%3A+final+results+of+phase+II+Swiss+Group+for+Clinical+Cancer+Research+Trial+%28SAKK+21%2F00%29&doi=10.1093%2Fannonc%2Fmdl341"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Clinical benefit of fulvestrant in postmenopausal women with advanced breast cancer and primary or acquired resistance to aromatase inhibitors: final results of phase II Swiss Group for Clinical Cancer Research Trial (SAKK 21/00)</span></div><div class="casAuthors">Perey L; Paridaens R; Hawle H; Zaman K; Nole F; Wildiers H; Fiche M; Dietrich D; Clement P; Koberle D; Goldhirsch A; Thurlimann B</div><div class="citationInfo"><span class="NLM_cas:title">Annals of oncology : official journal of the European Society for Medical Oncology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">64-9</span>
        ISSN:<span class="NLM_cas:issn">0923-7534</span>.
    </div><div class="casAbstract">BACKGROUND:  The aim of this study was to evaluate the efficacy and tolerability of fulvestrant, an estrogen receptor antagonist, in postmenopausal women with hormone-responsive tumors progressing after aromatase inhibitor (AI) treatment.  PATIENTS AND METHODS:  This is a phase II, open, multicenter, noncomparative study.  Two patient groups were prospectively considered: group A (n=70) with AI-responsive disease and group B (n=20) with AI-resistant disease.  Fulvestrant 250 mg was administered as intramuscular injection every 28 (+/-3) days.  RESULTS:  All patients were pretreated with AI and 84% also with tamoxifen or toremifene; 67% had bone metastases and 45% liver metastases.  Fulvestrant administration was well tolerated and yielded a clinical benefit (CB; defined as objective response or stable disease [SD] for >or=24 weeks) in 28% (90% confidence interval [CI] 19% to 39%) of patients in group A and 37% (90% CI 19% to 58%) of patients in group B.  Median time to progression (TTP) was 3.6 (95% CI 3.0 to 4.8) months in group A and 3.4 (95% CI 2.5 to 6.7) months in group B.  CONCLUSIONS:  Overall, 30% of patients who had progressed following prior AI treatment gained CB with fulvestrant, thereby delaying indication to start chemotherapy.  Prior response to an AI did not appear to be predictive for benefit with fulvestrant.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQgdgcuVoVvhr7NkxioyDXTfW6udTcc2ebbW05t6drUxbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2s%252FisVWntQ%253D%253D&md5=33f52d40ebeba134195a6311c0e8d854</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1093%2Fannonc%2Fmdl341&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fannonc%252Fmdl341%26sid%3Dliteratum%253Aachs%26aulast%3DPerey%26aufirst%3DL.%26aulast%3DParidaens%26aufirst%3DR.%26aulast%3DHawle%26aufirst%3DH.%26aulast%3DZaman%26aufirst%3DK.%26aulast%3DNole%26aufirst%3DF.%26aulast%3DWildiers%26aufirst%3DH.%26aulast%3DFiche%26aufirst%3DM.%26aulast%3DDietrich%26aufirst%3DD.%26aulast%3DClement%26aufirst%3DP.%26aulast%3DKoberle%26aufirst%3DD.%26aulast%3DGoldhirsch%26aufirst%3DA.%26aulast%3DThurlimann%26aufirst%3DB.%26atitle%3DClinical%2520benefit%2520of%2520fulvestrant%2520in%2520postmenopausal%2520women%2520with%2520advanced%2520breast%2520cancer%2520and%2520primary%2520or%2520acquired%2520resistance%2520to%2520aromatase%2520inhibitors%253A%2520final%2520results%2520of%2520phase%2520II%2520Swiss%2520Group%2520for%2520Clinical%2520Cancer%2520Research%2520Trial%2520%2528SAKK%252021%252F00%2529%26jtitle%3DAnn.%2520Oncol.%26date%3D2007%26volume%3D18%26spage%3D64%26epage%3D69%26doi%3D10.1093%2Fannonc%2Fmdl341" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ingle, J. N.</span></span> <span> </span><span class="NLM_article-title">Estrogen as therapy for breast cancer</span>. <i>Breast Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">133</span>– <span class="NLM_lpage">136</span>, <span class="refDoi"> DOI: 10.1186/bcr436</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1186%2Fbcr436" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=12100736" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BD38XltlWksL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2002&pages=133-136&author=J.+N.+Ingle&title=Estrogen+as+therapy+for+breast+cancer&doi=10.1186%2Fbcr436"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Estrogen as therapy for breast cancer</span></div><div class="casAuthors">Ingle, James N.</div><div class="citationInfo"><span class="NLM_cas:title">Breast Cancer Research [online computer file]</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">133-136</span>CODEN:
                <span class="NLM_cas:coden">BRCRFS</span>;
        ISSN:<span class="NLM_cas:issn">1465-542X</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">A review.  High-dose estrogen was generally considered the endocrine therapy of choice for postmenopausal women with breast cancer prior to the introduction of tamoxifen.  Subsequently, the use of estrogen was largely abandoned.  Recent clin. trial data have shown clin. meaningful efficacy for high-dose estrogen even in patients with extensive prior endocrine therapy.  Preclin. research has demonstrated that the estrogen dose-response curve for breast cancer cells can be shifted by modification of the estrogen environment.  Clin. and lab. data together provide the basis for developing testable hypotheses of management strategies, with the potential of increasing the value of endocrine therapy in women with breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_shZXrDZkBrVg90H21EOLACvtfcHk0litc7kXrKt2VQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XltlWksL8%253D&md5=5d65ac9d3527c51b9977071c3af41c2e</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1186%2Fbcr436&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fbcr436%26sid%3Dliteratum%253Aachs%26aulast%3DIngle%26aufirst%3DJ.%2BN.%26atitle%3DEstrogen%2520as%2520therapy%2520for%2520breast%2520cancer%26jtitle%3DBreast%2520Cancer%2520Res.%26date%3D2002%26volume%3D4%26spage%3D133%26epage%3D136%26doi%3D10.1186%2Fbcr436" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dowsett, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicholson, R. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pietras, R. J.</span></span> <span> </span><span class="NLM_article-title">Biological characteristics of the pure antiestrogen fulvestrant: overcoming endocrine resistance</span>. <i>Breast Cancer Res. Treat.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>93</i></span> (<span class="NLM_issue">Suppl. 1</span>),  <span class="NLM_fpage">11</span>– <span class="NLM_lpage">18</span>, <span class="refDoi"> DOI: 10.1007/s10549-005-9037-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1007%2Fs10549-005-9037-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtFCrtb3P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=2005&pages=11-18&issue=Suppl.+1&author=M.+Dowsettauthor=R.+I.+Nicholsonauthor=R.+J.+Pietras&title=Biological+characteristics+of+the+pure+antiestrogen+fulvestrant%3A+overcoming+endocrine+resistance&doi=10.1007%2Fs10549-005-9037-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Biological characteristics of the pure antiestrogen fulvestrant: overcoming endocrine resistance</span></div><div class="casAuthors">Dowsett, Mitch; Nicholson, Robert I.; Pietras, Richard J.</div><div class="citationInfo"><span class="NLM_cas:title">Breast Cancer Research and Treatment</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue">Suppl. 1</span>),
    <span class="NLM_cas:pages">S11-S18</span>CODEN:
                <span class="NLM_cas:coden">BCTRD6</span>;
        ISSN:<span class="NLM_cas:issn">0167-6806</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Understanding the underlying mechanisms responsible for endocrine resistance remains a challenge in improving the treatment of breast cancer.  The discovery that growth factor and estrogen receptor (ER) signaling pathways interact in endocrine resistant breast cancer has provided a rationale for disrupting these signaling cascades in ER-pos., endocrine-resistant tumors.  In postmenopausal women, the ER signaling pathway may be targeted using fulvestrant ('Faslodex'), a new type of ER antagonist with no agonist effects.  Fulvestrant binds, blocks and causes degrdn. of the ER, culminating in complete abrogation of estrogen-sensitive gene transcription.  This unique mechanism of action may result in a lack of cross-resistance with other endocrine agents.  Preclin. studies have confirmed the potential of fulvestrant to inhibit the growth of tamoxifen-resistant, as well as tamoxifen-sensitive, human breast cancer cell lines.  Clin. studies have demonstrated that fulvestrant is an effective treatment option in postmenopausal women with advanced breast cancer who have progressed on prior endocrine therapy.  Furthermore, preclin. studies indicate that combining fulvestrant with growth factor targeted agents, such as the epidermal growth factor receptor (EGFR/HER1) tyrosine kinase inhibitor gefitinib (IRESSA) or the anti-human HER2 monoclonal antibody trastuzumab ('Herceptin'), may result in greater anti-tumor activity than either agent alone.  A range of clin. trials are now ongoing to det. whether the combination of growth factor-targeting agents with fulvestrant will delay the onset of endocrine resistance and so provide new strategy for women with hormone receptor-pos. advanced breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4LULVK2HFH7Vg90H21EOLACvtfcHk0litc7kXrKt2VQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtFCrtb3P&md5=e082bfa908da811b2b78afd04138971a</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1007%2Fs10549-005-9037-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10549-005-9037-3%26sid%3Dliteratum%253Aachs%26aulast%3DDowsett%26aufirst%3DM.%26aulast%3DNicholson%26aufirst%3DR.%2BI.%26aulast%3DPietras%26aufirst%3DR.%2BJ.%26atitle%3DBiological%2520characteristics%2520of%2520the%2520pure%2520antiestrogen%2520fulvestrant%253A%2520overcoming%2520endocrine%2520resistance%26jtitle%3DBreast%2520Cancer%2520Res.%2520Treat.%26date%3D2005%26volume%3D93%26issue%3DSuppl.%25201%26spage%3D11%26epage%3D18%26doi%3D10.1007%2Fs10549-005-9037-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Osborne, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiff, R.</span></span> <span> </span><span class="NLM_article-title">Mechanisms of Endocrine Resistance in Breast Cancer</span>. <i>Annu. Rev. Med.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">233</span>– <span class="NLM_lpage">247</span>, <span class="refDoi"> DOI: 10.1146/annurev-med-070909-182917</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1146%2Fannurev-med-070909-182917" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=20887199" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC3MXivVWhtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2011&pages=233-247&author=C.+K.+Osborneauthor=R.+Schiff&title=Mechanisms+of+Endocrine+Resistance+in+Breast+Cancer&doi=10.1146%2Fannurev-med-070909-182917"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms of endocrine resistance in breast cancer</span></div><div class="casAuthors">Osborne, C. Kent; Schiff, Rachel</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Medicine</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">233-247</span>CODEN:
                <span class="NLM_cas:coden">ARMCAH</span>;
        ISSN:<span class="NLM_cas:issn">0066-4219</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  The estrogen receptor (ER) pathway plays a pivotal role in breast cancer development and progression.  Endocrine therapy to block the ER pathway is highly effective, but its usefulness is limited by common intrinsic and acquired resistance.  Multiple mechanisms responsible for endocrine resistance have been proposed and include deregulation of various components of the ER pathway itself, alterations in cell cycle and cell survival signaling mols., and the activation of escape pathways that can provide tumors with alternative proliferative and survival stimuli.  Among these, increased expression or signaling of growth factor receptor pathways, esp. the EGFR/HER2 pathway, has been assocd. with both exptl. and clin. endocrine therapy resistance.  New treatment combinations targeting both ER and growth factor receptor signaling to block the crosstalk between these pathways and eliminate escape routes have been proven highly effective in preclin. models.  Results of recent clin. studies, while partly supporting this approach, also highlight the need to better identify a priori the patients whose tumors are most likely to benefit from these specific cotargeting strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp23PYELtZJrbVg90H21EOLACvtfcHk0lhMXykGqFC8VQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXivVWhtLo%253D&md5=48573e39ea81057b53b5b43b1d72d2d9</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1146%2Fannurev-med-070909-182917&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-med-070909-182917%26sid%3Dliteratum%253Aachs%26aulast%3DOsborne%26aufirst%3DC.%2BK.%26aulast%3DSchiff%26aufirst%3DR.%26atitle%3DMechanisms%2520of%2520Endocrine%2520Resistance%2520in%2520Breast%2520Cancer%26jtitle%3DAnnu.%2520Rev.%2520Med.%26date%3D2011%26volume%3D62%26spage%3D233%26epage%3D247%26doi%3D10.1146%2Fannurev-med-070909-182917" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Frogne, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benjaminsen, R. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sonne-Hansen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sorensen, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nexo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laenkholm, A.-V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rasmussen, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riese, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Cremoux, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stenvang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lykkesfeldt, A. E.</span></span> <span> </span><span class="NLM_article-title">Activation of ErbB3, EGFR and Erk is essential for growth of human breast cancer cell lines with acquired resistance to fulvestrant</span>. <i>Breast Cancer Res. Treat.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">263</span>– <span class="NLM_lpage">275</span>, <span class="refDoi"> DOI: 10.1007/s10549-008-0011-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1007%2Fs10549-008-0011-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=18409071" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFWnu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2009&pages=263-275&author=T.+Frogneauthor=R.+V.+Benjaminsenauthor=K.+Sonne-Hansenauthor=B.+S.+Sorensenauthor=E.+Nexoauthor=A.-V.+Laenkholmauthor=L.+M.+Rasmussenauthor=D.+J.+Rieseauthor=P.+de+Cremouxauthor=J.+Stenvangauthor=A.+E.+Lykkesfeldt&title=Activation+of+ErbB3%2C+EGFR+and+Erk+is+essential+for+growth+of+human+breast+cancer+cell+lines+with+acquired+resistance+to+fulvestrant&doi=10.1007%2Fs10549-008-0011-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Activation of ErbB3, EGFR and Erk is essential for growth of human breast cancer cell lines with acquired resistance to fulvestrant</span></div><div class="casAuthors">Frogne, Thomas; Benjaminsen, Rikke V.; Sonne-Hansen, Katrine; Sorensen, Boe S.; Nexo, Ebba; Laenkholm, Anne-Vibeke; Rasmussen, Louise M.; Riese, David J., II; de Cremoux, Patricia; Stenvang, Jan; Lykkesfeldt, Anne E.</div><div class="citationInfo"><span class="NLM_cas:title">Breast Cancer Research and Treatment</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">263-275</span>CODEN:
                <span class="NLM_cas:coden">BCTRD6</span>;
        ISSN:<span class="NLM_cas:issn">0167-6806</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Seven fulvestrant resistant cell lines derived from the estrogen receptor α pos. MCF-7 human breast cancer cell line were used to investigate the importance of epidermal growth factor receptor (ErbB1-4) signaling.  We found an increase in mRNA expression of EGFR and the ErbB3/ErbB4 ligand heregulin2 (hrg2) and a decrease of ErbB4 in all resistant cell lines.  Western analyses confirmed the upregulation of EGFR and hrg2 and the downregulation of ErbB4.  Elevated activation of EGFR and ErbB3 was seen in all resistant cell lines and the ErbB3 activation occurred by an autocrine mechanism.  ErbB4 activation was obsd. only in the parental MCF-7 cells.  The downstream kinases pAkt and pErk were increased in five of seven and in all seven resistant cell lines, resp.  Treatment with the EGFR inhibitor gefitinib preferentially inhibited growth and reduced the S phase fraction in the resistant cell lines concomitant with inhibition of Erk and unaltered Akt activation.  In concert, inhibition of Erk with U0126 preferentially reduced growth of resistant cell lines.  Treatment with ErbB3 neutralizing antibodies inhibited ErbB3 activation and resulted in a modest but statistically significant growth inhibition of two resistant cell lines.  These data indicate that ligand activated ErbB3 and EGFR, and Erk signaling play important roles in fulvestrant resistant cell growth.  Furthermore, the decreased level of ErbB4 in resistant cells may facilitate heterodimerization of ErbB3 with EGFR and ErbB2.  Our data support that a concerted action against EGFR, ErbB2 and ErbB3 may be required to obtain complete growth suppression of fulvestrant resistant cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_Dd87hGSoHLVg90H21EOLACvtfcHk0lhMXykGqFC8VQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFWnu74%253D&md5=3a477e5836c0056d9af0bd67e120e2f5</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1007%2Fs10549-008-0011-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10549-008-0011-8%26sid%3Dliteratum%253Aachs%26aulast%3DFrogne%26aufirst%3DT.%26aulast%3DBenjaminsen%26aufirst%3DR.%2BV.%26aulast%3DSonne-Hansen%26aufirst%3DK.%26aulast%3DSorensen%26aufirst%3DB.%2BS.%26aulast%3DNexo%26aufirst%3DE.%26aulast%3DLaenkholm%26aufirst%3DA.-V.%26aulast%3DRasmussen%26aufirst%3DL.%2BM.%26aulast%3DRiese%26aufirst%3DD.%2BJ.%26aulast%3Dde%2BCremoux%26aufirst%3DP.%26aulast%3DStenvang%26aufirst%3DJ.%26aulast%3DLykkesfeldt%26aufirst%3DA.%2BE.%26atitle%3DActivation%2520of%2520ErbB3%252C%2520EGFR%2520and%2520Erk%2520is%2520essential%2520for%2520growth%2520of%2520human%2520breast%2520cancer%2520cell%2520lines%2520with%2520acquired%2520resistance%2520to%2520fulvestrant%26jtitle%3DBreast%2520Cancer%2520Res.%2520Treat.%26date%3D2009%26volume%3D114%26spage%3D263%26epage%3D275%26doi%3D10.1007%2Fs10549-008-0011-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Massarweh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osborne, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wakeling, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rimawi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohsin, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilsenbeck, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiff, R.</span></span> <span> </span><span class="NLM_article-title">Mechanisms of tumor regression and resistance to estrogen deprivation and fulvestrant in a model of estrogen receptor-positive, HER-2/neu-positive breast cancer</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">8266</span>– <span class="NLM_lpage">8273</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-05-4045</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1158%2F0008-5472.CAN-05-4045" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=16912207" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BD28XotFWqurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2006&pages=8266-8273&author=S.+Massarwehauthor=C.+K.+Osborneauthor=S.+Jiangauthor=A.+E.+Wakelingauthor=M.+Rimawiauthor=S.+K.+Mohsinauthor=S.+Hilsenbeckauthor=R.+Schiff&title=Mechanisms+of+tumor+regression+and+resistance+to+estrogen+deprivation+and+fulvestrant+in+a+model+of+estrogen+receptor-positive%2C+HER-2%2Fneu-positive+breast+cancer&doi=10.1158%2F0008-5472.CAN-05-4045"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms of Tumor Regression and Resistance to Estrogen Deprivation and Fulvestrant in a Model of Estrogen Receptor-Positive, HER-2/neu-Positive Breast Cancer</span></div><div class="casAuthors">Massarweh, Suleiman; Osborne, C. Kent; Jiang, Shou; Wakeling, Alan E.; Rimawi, Mothaffar; Mohsin, Syed K.; Hilsenbeck, Susan; Schiff, Rachel</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">8266-8273</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">HER-2/neu in breast cancer is assocd. with tamoxifen resistance, but little data exist on its interaction with estrogen deprivation or fulvestrant.  Here, we used an in vivo xenograft model of estrogen receptor (ER)-pos. breast cancer with HER-2/neu overexpression (MCF7/HER-2/neu-18) to investigate mechanisms of growth inhibition and treatment resistance.  MCF7/HER-2/neu-18 tumors were growth inhibited by estrogen deprivation and with fulvestrant, but resistance developed in 2 to 3 mo.  Inhibited tumors had redns. in ER, insulin-like growth factor-I receptor (IGF-IR), phosphorylated HER-2/neu (p-HER-2/neu), and phosphorylated p42/44 mitogen-activated protein kinase (p-MAPK).  P27 was increased esp. in tumors sensitive to estrogen deprivation.  Tumors with acquired resistance to these therapies had complete loss of ER, increased p-HER-2/neu, increased p-MAPK, and reduced p27.  In contrast, IGF-IR and phosphorylated AKT (p-AKT) levels were markedly reduced in these resistant tumors.  The epidermal growth factor receptor (EGFR) tyrosine kinase inhibitor gefitinib, which can block EGFR/HER-2/neu signaling, significantly delayed the emergence of resistance to both estrogen deprivation and fulvestrant.  Levels of p-MAPK and p-AKT decreased with gefitinib, whereas high ER levels were restored.  Eventually, however, tumors progressed in mice treated with gefitinib combined with estrogen deprivation or fulvestrant accompanied again by loss of ER and IGF-IR, increased p-HER-2/neu, high p-MAPK, and now increased p-AKT.  Thus, estrogen deprivation and fulvestrant can effectively inhibit HER-2/neu-overexpressing tumors but resistance develops quickly.  EGFR/HER-2/neu inhibitors can delay resistance, but reactivation of HER-2/neu and signaling through AKT leads to tumor regrowth.  Combining endocrine therapy with EGFR/HER-2/neu inhibitors should be tested in clin. breast cancer, but a more complete blockade of EGFR/HER-2/neu may be optimal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5TEeU_-NI5bVg90H21EOLACvtfcHk0lhMXykGqFC8VQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XotFWqurs%253D&md5=23e976f76a73ac68f7e63bec37f33127</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-05-4045&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-05-4045%26sid%3Dliteratum%253Aachs%26aulast%3DMassarweh%26aufirst%3DS.%26aulast%3DOsborne%26aufirst%3DC.%2BK.%26aulast%3DJiang%26aufirst%3DS.%26aulast%3DWakeling%26aufirst%3DA.%2BE.%26aulast%3DRimawi%26aufirst%3DM.%26aulast%3DMohsin%26aufirst%3DS.%2BK.%26aulast%3DHilsenbeck%26aufirst%3DS.%26aulast%3DSchiff%26aufirst%3DR.%26atitle%3DMechanisms%2520of%2520tumor%2520regression%2520and%2520resistance%2520to%2520estrogen%2520deprivation%2520and%2520fulvestrant%2520in%2520a%2520model%2520of%2520estrogen%2520receptor-positive%252C%2520HER-2%252Fneu-positive%2520breast%2520cancer%26jtitle%3DCancer%2520Res.%26date%3D2006%26volume%3D66%26spage%3D8266%26epage%3D8273%26doi%3D10.1158%2F0008-5472.CAN-05-4045" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Leva, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Devlin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartman-Frey, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burow, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ivan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Croce, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nephew, K. P.</span></span> <span> </span><span class="NLM_article-title">MicroRNA-221/222 confers breast cancer fulvestrant resistance by regulating multiple signaling pathways</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">1082</span>– <span class="NLM_lpage">1097</span>, <span class="refDoi"> DOI: 10.1038/onc.2010.487</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1038%2Fonc.2010.487" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=21057537" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtl2jtrfF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2011&pages=1082-1097&author=X.+Raoauthor=G.+Di+Levaauthor=M.+Liauthor=F.+Fangauthor=C.+Devlinauthor=C.+Hartman-Freyauthor=M.+E.+Burowauthor=M.+Ivanauthor=C.+M.+Croceauthor=K.+P.+Nephew&title=MicroRNA-221%2F222+confers+breast+cancer+fulvestrant+resistance+by+regulating+multiple+signaling+pathways&doi=10.1038%2Fonc.2010.487"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">MicroRNA-221/222 confers breast cancer fulvestrant resistance by regulating multiple signaling pathways</span></div><div class="casAuthors">Rao, X.; Di Leva, G.; Li, M.; Fang, F.; Devlin, C.; Hartman-Frey, C.; Burow, M. E.; Ivan, M.; Croce, C. M.; Nephew, K. P.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1082-1097</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Fulvestrant is a selective estrogen receptor downregulator (SERD) and highly effective antagonist to hormone-sensitive breast cancers following failure of previous tamoxifen or aromatase inhibitor therapies.  However, after prolonged fulvestrant therapy, acquired resistance eventually occurs in the majority of breast cancer patients, due to poorly understood mechanisms.  To examine a possible role(s) of aberrantly expressed microRNAs (miRNAs) in acquired fulvestrant resistance, we compared antiestrogen-resistant and -sensitive breast cancer cells, revealing the overexpression of miR-221/222 in the SERD-resistant cell lines.  Fulvestrant treatment of estradiol (E2)- and fulvestrant-sensitive MCF7 cells resulted in increased expression of endogenous miR-221/222.  Ectopic upregulation of miR-221/222 in estrogen receptor-α (ERα)-pos. cell lines counteracted the effects of E2 depletion or fulvestrant-induced cell death, thus also conferring hormone-independent growth and fulvestrant resistance.  In cells with acquired resistance to fulvestrant, miR-221/222 expression was essential for cell growth and cell cycle progression.  To identify possible miR-221/222 targets, miR-221- or miR-222- induced alterations in global gene expression profiles and target gene expression at distinct time points were detd., revealing that miR-221/222 overexpression resulted in deregulation of multiple oncogenic signaling pathways previously assocd. with drug resistance.  Activation of β-catenin by miR-221/222 contributed to estrogen-independent growth and fulvestrant resistance, whereas TGF-β-mediated growth inhibition was repressed by the 2 miRNAs.  This first in-depth investigation into the role of miR-221/222 in acquired fulvestrant resistance, a clin. important problem, demonstrates that these 2 oncomirs' may represent promising therapeutic targets for treating hormone-independent, SERD-resistant breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqh79Pw5CWhrVg90H21EOLACvtfcHk0lhX9RLaU1TVPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtl2jtrfF&md5=24f2c05c330c8735230247ba02fc9856</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fonc.2010.487&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2010.487%26sid%3Dliteratum%253Aachs%26aulast%3DRao%26aufirst%3DX.%26aulast%3DDi%2BLeva%26aufirst%3DG.%26aulast%3DLi%26aufirst%3DM.%26aulast%3DFang%26aufirst%3DF.%26aulast%3DDevlin%26aufirst%3DC.%26aulast%3DHartman-Frey%26aufirst%3DC.%26aulast%3DBurow%26aufirst%3DM.%2BE.%26aulast%3DIvan%26aufirst%3DM.%26aulast%3DCroce%26aufirst%3DC.%2BM.%26aulast%3DNephew%26aufirst%3DK.%2BP.%26atitle%3DMicroRNA-221%252F222%2520confers%2520breast%2520cancer%2520fulvestrant%2520resistance%2520by%2520regulating%2520multiple%2520signaling%2520pathways%26jtitle%3DOncogene%26date%3D2011%26volume%3D30%26spage%3D1082%26epage%3D1097%26doi%3D10.1038%2Fonc.2010.487" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Belkina, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denis, G. V.</span></span> <span> </span><span class="NLM_article-title">BET domain co-regulators in obesity, inflammation and cancer</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">465</span>– <span class="NLM_lpage">477</span>, <span class="refDoi"> DOI: 10.1038/nrc3256</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1038%2Fnrc3256" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=22722403" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC38XovFyks7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2012&pages=465-477&author=A.+C.+Belkinaauthor=G.+V.+Denis&title=BET+domain+co-regulators+in+obesity%2C+inflammation+and+cancer&doi=10.1038%2Fnrc3256"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">BET domain co-regulators in obesity, inflammation and cancer</span></div><div class="casAuthors">Belkina, Anna C.; Denis, Gerald V.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">465-477</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The bromodomain is a highly conserved motif of 110 amino acids that is bundled into four anti-parallel α-helixes and found in proteins that interact with chromatin, such as transcription factors, histone acetylases and nucleosome remodelling complexes.  Bromodomain proteins are chromatin 'readers'; they recruit chromatin-regulating enzymes, including 'writers' and 'erasers' of histone modification, to target promoters and to regulate gene expression.  Conventional wisdom held that complexes involved in chromatin dynamics are not 'druggable' targets.  However, small mols. that inhibit bromodomain and extraterminal (BET) proteins have been described.  We examine these developments and discuss the implications for small mol. epigenetic targeting of chromatin networks in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1x-bD67oukrVg90H21EOLACvtfcHk0lhX9RLaU1TVPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XovFyks7s%253D&md5=d06d88ef0d5d4d71115f40a2f901f53d</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fnrc3256&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3256%26sid%3Dliteratum%253Aachs%26aulast%3DBelkina%26aufirst%3DA.%2BC.%26aulast%3DDenis%26aufirst%3DG.%2BV.%26atitle%3DBET%2520domain%2520co-regulators%2520in%2520obesity%252C%2520inflammation%2520and%2520cancer%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2012%26volume%3D12%26spage%3D465%26epage%3D477%26doi%3D10.1038%2Fnrc3256" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bountra, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fish, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span> <span> </span><span class="NLM_article-title">Epigenetic protein families: a new frontier for drug discovery</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">384</span>– <span class="NLM_lpage">400</span>, <span class="refDoi"> DOI: 10.1038/nrd3674</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1038%2Fnrd3674" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=22498752" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC38XlsFWjsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=384-400&author=C.+H.+Arrowsmithauthor=C.+Bountraauthor=P.+V.+Fishauthor=K.+Leeauthor=M.+Schapira&title=Epigenetic+protein+families%3A+a+new+frontier+for+drug+discovery&doi=10.1038%2Fnrd3674"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetic protein families: a new frontier for drug discovery</span></div><div class="casAuthors">Arrowsmith, Cheryl H.; Bountra, Chas; Fish, Paul V.; Lee, Kevin; Schapira, Matthieu</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">384-400</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Epigenetic regulation of gene expression is a dynamic and reversible process that establishes normal cellular phenotypes but also contributes to human diseases.  At the mol. level, epigenetic regulation involves hierarchical covalent modification of DNA and the proteins that package DNA, such as histones.  Here, we review the key protein families that mediate epigenetic signalling through the acetylation and methylation of histones, including histone deacetylases, protein methyltransferases, lysine demethylases, bromodomain-contg. proteins and proteins that bind to methylated histones.  These protein families are emerging as druggable classes of enzymes and druggable classes of protein-protein interaction domains.  In this article, we discuss the known links with disease, basic mol. mechanisms of action and recent progress in the pharmacol. modulation of each class of proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcD99eCigNtLVg90H21EOLACvtfcHk0lhX9RLaU1TVPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XlsFWjsbs%253D&md5=eb82034466a43107aa74a18ccf6d29f5</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fnrd3674&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3674%26sid%3Dliteratum%253Aachs%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DFish%26aufirst%3DP.%2BV.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DSchapira%26aufirst%3DM.%26atitle%3DEpigenetic%2520protein%2520families%253A%2520a%2520new%2520frontier%2520for%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2012%26volume%3D11%26spage%3D384%26epage%3D400%26doi%3D10.1038%2Fnrd3674" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mangos, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keates, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambert, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Volkmer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pawson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gingras, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Histone recognition and large-scale structural analysis of the human bromodomain family</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>149</i></span>,  <span class="NLM_fpage">214</span>– <span class="NLM_lpage">231</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2012.02.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1016%2Fj.cell.2012.02.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=22464331" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=149&publication_year=2012&pages=214-231&author=P.+Filippakopoulosauthor=S.+Picaudauthor=M.+Mangosauthor=T.+Keatesauthor=J.+P.+Lambertauthor=D.+Barsyte-Lovejoyauthor=I.+Felletarauthor=R.+Volkmerauthor=S.+Mullerauthor=T.+Pawsonauthor=A.+C.+Gingrasauthor=C.+H.+Arrowsmithauthor=S.+Knapp&title=Histone+recognition+and+large-scale+structural+analysis+of+the+human+bromodomain+family&doi=10.1016%2Fj.cell.2012.02.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Histone Recognition and Large-Scale Structural Analysis of the Human Bromodomain Family</span></div><div class="casAuthors">Filippakopoulos, Panagis; Picaud, Sarah; Mangos, Maria; Keates, Tracy; Lambert, Jean-Philippe; Barsyte-Lovejoy, Dalia; Felletar, Ildiko; Volkmer, Rudolf; Muller, Susanne; Pawson, Tony; Gingras, Anne-Claude; Arrowsmith, Cheryl H.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">149</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">214-231</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Bromodomains (BRDs) are protein interaction modules that specifically recognize ε-N-lysine acetylation motifs, a key event in the reading process of epigenetic marks.  The 61 BRDs in the human genome cluster into eight families based on structure/sequence similarity.  Here, we present 29 high-resoln. crystal structures, covering all BRD families.  These proteins are: ASH1L, ATAD2, BAZ2B, BPTF, BRD1, BRD3(1), BRD3(2), BRD4(1), BRD4(2), BRD9, BRDT(1), CECR2, EP300, CREBBP, GCN5L2, KIAA1240, PB1(1), PB1(2), PB1(3), PB1(4), PB1(5), PB1(6), PCAF, PHIP(2), TAF1(2), WDR9(2), BRD4(1).  Comprehensive crossfamily structural anal. identifies conserved and family-specific structural features that are necessary for specific acetylation-dependent substrate recognition.  Screening of more than 30 representative BRDs against systematic histone-peptide arrays identifies new BRD substrates and reveals a strong influence of flanking posttranslational modifications, such as acetylation and phosphorylation, suggesting that BRDs recognize combinations of marks rather than singly acetylated sequences.  We further uncovered a structural mechanism for the simultaneous binding and recognition of diverse diacetyl-contg. peptides by BRD4.  These data provide a foundation for structure-based drug design of specific inhibitors for this emerging target family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoK2FSIcPoXZLVg90H21EOLACvtfcHk0lhX9RLaU1TVPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D&md5=05583a01a1cd4c3b59fc59c00e4c4777</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2012.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2012.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DMangos%26aufirst%3DM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DLambert%26aufirst%3DJ.%2BP.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DVolkmer%26aufirst%3DR.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DPawson%26aufirst%3DT.%26aulast%3DGingras%26aufirst%3DA.%2BC.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DHistone%2520recognition%2520and%2520large-scale%2520structural%2520analysis%2520of%2520the%2520human%2520bromodomain%2520family%26jtitle%3DCell%26date%3D2012%26volume%3D149%26spage%3D214%26epage%3D231%26doi%3D10.1016%2Fj.cell.2012.02.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Targeting bromodomains: epigenetic readers of lysine acetylation</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">337</span>– <span class="NLM_lpage">356</span>, <span class="refDoi"> DOI: 10.1038/nrd4286</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1038%2Fnrd4286" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=24751816" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=337-356&author=P.+Filippakopoulosauthor=S.+Knapp&title=Targeting+bromodomains%3A+epigenetic+readers+of+lysine+acetylation&doi=10.1038%2Fnrd4286"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting bromodomains: epigenetic readers of lysine acetylation</span></div><div class="casAuthors">Filippakopoulos, Panagis; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">337-356</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Lysine acetylation is a key mechanism that regulates chromatin structure; aberrant acetylation levels have been linked to the development of several diseases.  Acetyl-lysine modifications create docking sites for bromodomains, which are small interaction modules found on diverse proteins, some of which have a key role in the acetylation-dependent assembly of transcriptional regulator complexes.  These complexes can then initiate transcriptional programs that result in phenotypic changes.  The recent discovery of potent and highly specific inhibitors for the BET (bromodomain and extra-terminal) family of bromodomains has stimulated intensive research activity in diverse therapeutic areas, particularly in oncol., where BET proteins regulate the expression of key oncogenes and anti-apoptotic proteins.  In addn., targeting BET bromodomains could hold potential for the treatment of inflammation and viral infection.  Here, we highlight recent progress in the development of bromodomain inhibitors, and their potential applications in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOSIcwBJKj4LVg90H21EOLACvtfcHk0lhd4J-kddm5UQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D&md5=77049c82ce6a19b4685e3b7950399a07</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fnrd4286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4286%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DTargeting%2520bromodomains%253A%2520epigenetic%2520readers%2520of%2520lysine%2520acetylation%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D337%26epage%3D356%26doi%3D10.1038%2Fnrd4286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dhalluin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlson, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aggarwal, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M. M.</span></span> <span> </span><span class="NLM_article-title">Structure and ligand of a histone acetyltransferase bromodomain</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>399</i></span>,  <span class="NLM_fpage">491</span>– <span class="NLM_lpage">496</span>, <span class="refDoi"> DOI: 10.1038/20974</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1038%2F20974" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10365964" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADyaK1MXjvV2isLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=399&publication_year=1999&pages=491-496&author=C.+Dhalluinauthor=J.+E.+Carlsonauthor=L.+Zengauthor=C.+Heauthor=A.+K.+Aggarwalauthor=M.+M.+Zhou&title=Structure+and+ligand+of+a+histone+acetyltransferase+bromodomain&doi=10.1038%2F20974"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and ligand of a histone acetyltransferase bromodomain</span></div><div class="casAuthors">Dhalluin, Christophe; Carlson, Justin E.; Zeng, Lei; He, Cheng; Aggarwal, Aneel K.; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">399</span>
        (<span class="NLM_cas:issue">6735</span>),
    <span class="NLM_cas:pages">491-496</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines</span>)
        </div><div class="casAbstract">Histone acetylation is important in chromatin remodelling and gene activation.  Nearly all known histone-acetyltransferase (HAT)-assocd. transcriptional co-activators contain bromodomains, which are ∼110-amino-acid modules found in many chromatin-assocd. proteins.  Despite the wide occurrence of these bromodomains, their three-dimensional structure and binding partners remain unknown.  Here we report the soln. structure of the bromodomain of the HAT co-activator P/CAF (p300/CBP-assocd. factor).  The structure reveals an unusual left-handed up-and-down four-helix bundle.  In addn., we show by a combination of structural and site-directed mutagenesis studies that bromodomains can interact specifically with acetylated lysine, making them the first known protein modules to do so.  The nature of the recognition of acetyl-lysine by the P/CAF bromodomain is similar to that of acetyl-CoA by histone acetyltransferase.  Thus, the bromodomain is functionally linked to the HAT activity of co-activators in the regulation of gene transcription.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsTbwcjeBG_rVg90H21EOLACvtfcHk0lhd4J-kddm5UQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXjvV2isLo%253D&md5=ff1063ba81d81b431065dd6e3751ba79</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2F20974&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F20974%26sid%3Dliteratum%253Aachs%26aulast%3DDhalluin%26aufirst%3DC.%26aulast%3DCarlson%26aufirst%3DJ.%2BE.%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DHe%26aufirst%3DC.%26aulast%3DAggarwal%26aufirst%3DA.%2BK.%26aulast%3DZhou%26aufirst%3DM.%2BM.%26atitle%3DStructure%2520and%2520ligand%2520of%2520a%2520histone%2520acetyltransferase%2520bromodomain%26jtitle%3DNature%26date%3D1999%26volume%3D399%26spage%3D491%26epage%3D496%26doi%3D10.1038%2F20974" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M. M.</span></span> <span> </span><span class="NLM_article-title">Bromodomain: an acetyl-lysine binding domain</span>. <i>FEBS Lett.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>513</i></span>,  <span class="NLM_fpage">124</span>– <span class="NLM_lpage">128</span>, <span class="refDoi"> DOI: 10.1016/S0014-5793(01)03309-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1016%2FS0014-5793%2801%2903309-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=11911891" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BD38XitFejtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=513&publication_year=2002&pages=124-128&author=L.+Zengauthor=M.+M.+Zhou&title=Bromodomain%3A+an+acetyl-lysine+binding+domain&doi=10.1016%2FS0014-5793%2801%2903309-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomain: an acetyl-lysine binding domain</span></div><div class="casAuthors">Zeng, Lei; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Letters</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">513</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">124-128</span>CODEN:
                <span class="NLM_cas:coden">FEBLAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-5793</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">A review with 43 refs.  Bromodomains, an extensive family of evolutionarily conserved protein modules originally found in proteins assocd. with chromatin and in nearly all nuclear histone acetyltransferases, have been recently discovered to function as acetyllysine-binding domains.  More recent structural studies of bromodomain/peptide ligand complexes have enriched the understanding of differences in ligand selectivity of bromodomains.  These new findings demonstrate that bromodomain/acetyllysine recognition can serve as a pivotal mechanism for regulating protein-protein interactions in numerous cellular processes including chromatin remodeling and transcriptional activation, and reinforce the concept that functional diversity of a conserved protein modular structure is achieved by evolutionary changes of amino acid sequences in the ligand binding site.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMESL8iZ9agLVg90H21EOLACvtfcHk0lhd4J-kddm5UQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XitFejtbo%253D&md5=79070a63cd171f846ab577aa9f7092bb</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2FS0014-5793%2801%2903309-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0014-5793%252801%252903309-9%26sid%3Dliteratum%253Aachs%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DZhou%26aufirst%3DM.%2BM.%26atitle%3DBromodomain%253A%2520an%2520acetyl-lysine%2520binding%2520domain%26jtitle%3DFEBS%2520Lett.%26date%3D2002%26volume%3D513%26spage%3D124%26epage%3D128%26doi%3D10.1016%2FS0014-5793%2801%2903309-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Loven, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoke, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lau, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orlando, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vakoc, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, T. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. A.</span></span> <span> </span><span class="NLM_article-title">Selective inhibition of tumor oncogenes by disruption of super-enhancers</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>153</i></span>,  <span class="NLM_fpage">320</span>– <span class="NLM_lpage">334</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2013.03.036</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1016%2Fj.cell.2013.03.036" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=23582323" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlvVCgsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=153&publication_year=2013&pages=320-334&author=J.+Lovenauthor=H.+A.+Hokeauthor=C.+Y.+Linauthor=A.+Lauauthor=D.+A.+Orlandoauthor=C.+R.+Vakocauthor=J.+E.+Bradnerauthor=T.+I.+Leeauthor=R.+A.+Young&title=Selective+inhibition+of+tumor+oncogenes+by+disruption+of+super-enhancers&doi=10.1016%2Fj.cell.2013.03.036"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Inhibition of Tumor Oncogenes by Disruption of Super-Enhancers</span></div><div class="casAuthors">Loven, Jakob; Hoke, Heather A.; Lin, Charles Y.; Lau, Ashley; Orlando, David A.; Vakoc, Christopher R.; Bradner, James E.; Lee, Tong Ihn; Young, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">153</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">320-334</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Chromatin regulators have become attractive targets for cancer therapy, but it is unclear why inhibition of these ubiquitous regulators should have gene-specific effects in tumor cells.  Here, we investigate how inhibition of the widely expressed transcriptional coactivator BRD4 leads to selective inhibition of the MYC oncogene in multiple myeloma (MM).  BRD4 and Mediator were found to co-occupy thousands of enhancers assocd. with active genes.  They also co-occupied a small set of exceptionally large super-enhancers assocd. with genes that feature prominently in MM biol., including the MYC oncogene.  Treatment of MM tumor cells with the BET-bromodomain inhibitor JQ1 led to preferential loss of BRD4 at super-enhancers and consequent transcription elongation defects that preferentially impacted genes with super-enhancers, including MYC.  Super-enhancers were found at key oncogenic drivers in many other tumor cells.  These observations have implications for the discovery of cancer therapeutics directed at components of super-enhancers in diverse tumor types.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrM-NqFYuGMqrVg90H21EOLACvtfcHk0lhNuctoR_Ld-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlvVCgsrY%253D&md5=66b6ccb5b8a6039832910443fd8b4664</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2013.03.036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2013.03.036%26sid%3Dliteratum%253Aachs%26aulast%3DLoven%26aufirst%3DJ.%26aulast%3DHoke%26aufirst%3DH.%2BA.%26aulast%3DLin%26aufirst%3DC.%2BY.%26aulast%3DLau%26aufirst%3DA.%26aulast%3DOrlando%26aufirst%3DD.%2BA.%26aulast%3DVakoc%26aufirst%3DC.%2BR.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DLee%26aufirst%3DT.%2BI.%26aulast%3DYoung%26aufirst%3DR.%2BA.%26atitle%3DSelective%2520inhibition%2520of%2520tumor%2520oncogenes%2520by%2520disruption%2520of%2520super-enhancers%26jtitle%3DCell%26date%3D2013%26volume%3D153%26spage%3D320%26epage%3D334%26doi%3D10.1016%2Fj.cell.2013.03.036" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hnisz, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. A.</span></span> <span> </span><span class="NLM_article-title">Transcriptional Addiction in Cancer</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>168</i></span>,  <span class="NLM_fpage">629</span>– <span class="NLM_lpage">643</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2016.12.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1016%2Fj.cell.2016.12.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=28187285" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC2sXis1ygtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=168&publication_year=2017&pages=629-643&author=J.+E.+Bradnerauthor=D.+Hniszauthor=R.+A.+Young&title=Transcriptional+Addiction+in+Cancer&doi=10.1016%2Fj.cell.2016.12.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Transcriptional addiction in cancer</span></div><div class="casAuthors">Bradner, James E.; Hnisz, Denes; Young, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">168</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">629-643</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Cancer arises from genetic alterations that invariably lead to dysregulated transcriptional programs.  These dysregulated programs can cause cancer cells to become highly dependent on certain regulators of gene expression.  Here, we discuss how transcriptional control is disrupted by genetic alterations in cancer cells, why transcriptional dependencies can develop as a consequence of dysregulated programs, and how these dependencies provide opportunities for novel therapeutic interventions in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAb-krWUlAJLVg90H21EOLACvtfcHk0lhNuctoR_Ld-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXis1ygtbk%253D&md5=d6f9840f701f233b14e89306113c828c</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2016.12.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2016.12.013%26sid%3Dliteratum%253Aachs%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DHnisz%26aufirst%3DD.%26aulast%3DYoung%26aufirst%3DR.%2BA.%26atitle%3DTranscriptional%2520Addiction%2520in%2520Cancer%26jtitle%3DCell%26date%3D2017%26volume%3D168%26spage%3D629%26epage%3D643%26doi%3D10.1016%2Fj.cell.2016.12.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prakash, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sum, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwok, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thiessen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pettersson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chin, K. C.</span></span> <span> </span><span class="NLM_article-title">Bromodomain-containing protein 4 (BRD4) regulates RNA polymerase II serine 2 phosphorylation in human CD4+ T cells</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>287</i></span>,  <span class="NLM_fpage">43137</span>– <span class="NLM_lpage">43155</span>, <span class="refDoi"> DOI: 10.1074/jbc.M112.413047</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1074%2Fjbc.M112.413047" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=23086925" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVeksbnL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2012&pages=43137-43155&author=W.+Zhangauthor=C.+Prakashauthor=C.+Sumauthor=Y.+Gongauthor=Y.+Liauthor=J.+J.+Kwokauthor=N.+Thiessenauthor=S.+Petterssonauthor=S.+J.+Jonesauthor=S.+Knappauthor=H.+Yangauthor=K.+C.+Chin&title=Bromodomain-containing+protein+4+%28BRD4%29+regulates+RNA+polymerase+II+serine+2+phosphorylation+in+human+CD4%2B+T+cells&doi=10.1074%2Fjbc.M112.413047"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomain-containing Protein 4 (BRD4) Regulates RNA Polymerase II Serine 2 Phosphorylation in Human CD4+ T Cells</span></div><div class="casAuthors">Zhang, Weishi; Prakash, Celine; Sum, Calvin; Gong, Yue; Li, Yinghui; Kwok, Jeffrey J. T.; Thiessen, Nina; Pettersson, Sven; Jones, Steven J. M.; Knapp, Stefan; Yang, Henry; Chin, Keh-Chuang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">287</span>
        (<span class="NLM_cas:issue">51</span>),
    <span class="NLM_cas:pages">43137-43155</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Transcriptional elongation by RNA polymerase II (Pol II) is regulated by pos. transcription elongation factor b (P-TEFb) in assocn. with bromodomain-contg. protein 4 (BRD4).  The authors used genome-wide chromatin immunopptn. sequencing in primary human CD4+ T cells to reveal that BRD4 co-localizes with Ser-2-phosphorylated Pol II (Pol II Ser-2) at both enhancers and promoters of active genes.  Disruption of bromodomain-histone acetylation interactions by JQ1, a small-mol. bromodomain inhibitor, resulted in decreased BRD4 binding, reduced Pol II Ser-2, and reduced expression of lineage-specific genes in primary human CD4+ T cells.  A large no. of JQ1-disrupted BRD4 binding regions exhibited diacetylated H4 (lysine 5 and -8) and H3K27 acetylation (H3K27ac), which correlated with the presence of histone acetyltransferases and deacetylases.  Genes assocd. with BRD4/H3K27ac co-occupancy exhibited significantly higher activity than those assocd. with H3K27ac or BRD4 binding alone.  Comparison of BRD4 binding in T cells and in human embryonic stem cells revealed that enhancer BRD4 binding sites were predominantly lineage-specific.  These findings suggest that BRD4-driven Pol II phosphorylation at serine 2 plays an important role in regulating lineage-specific gene transcription in human CD4+ T cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMZ7-itp_IObVg90H21EOLACvtfcHk0lhNuctoR_Ld-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVeksbnL&md5=79541de2c23a5c164fc8c4295ffebf53</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M112.413047&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M112.413047%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DPrakash%26aufirst%3DC.%26aulast%3DSum%26aufirst%3DC.%26aulast%3DGong%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DKwok%26aufirst%3DJ.%2BJ.%26aulast%3DThiessen%26aufirst%3DN.%26aulast%3DPettersson%26aufirst%3DS.%26aulast%3DJones%26aufirst%3DS.%2BJ.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DChin%26aufirst%3DK.%2BC.%26atitle%3DBromodomain-containing%2520protein%25204%2520%2528BRD4%2529%2520regulates%2520RNA%2520polymerase%2520II%2520serine%25202%2520phosphorylation%2520in%2520human%2520CD4%252B%2520T%2520cells%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2012%26volume%3D287%26spage%3D43137%26epage%3D43155%26doi%3D10.1074%2Fjbc.M112.413047" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">French, C. A.</span></span> <span> </span><span class="NLM_article-title">NUT midline carcinoma</span>. <i>Cancer Genet. Cytogenet.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>203</i></span>,  <span class="NLM_fpage">16</span>– <span class="NLM_lpage">20</span>, <span class="refDoi"> DOI: 10.1016/j.cancergencyto.2010.06.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1016%2Fj.cancergencyto.2010.06.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=20951314" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A280%3ADC%252BC3cbgtVeluw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=203&publication_year=2010&pages=16-20&author=C.+A.+French&title=NUT+midline+carcinoma&doi=10.1016%2Fj.cancergencyto.2010.06.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">NUT midline carcinoma</span></div><div class="casAuthors">French Christopher Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Cancer genetics and cytogenetics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">203</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">16-20</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">NUT midline carcinoma (NMC) is a rare, aggressive human cancer, genetically defined by rearrangements of the gene NUT (HUGO symbol: C15orf55).  In the majority (∼75%) of NMCs, most of the coding sequence of NUT on chromosome 15q14 is fused with BRD4 creating chimeric genes that encode BRD-NUT fusion proteins.  In the remaining cases, NUT is fused to BRD3 or an unknown partner gene; these tumors are termed NUT-variant.  Diagnosis of NMC is made by demonstration of expression of the NUT-fusion protein using a monoclonal antibody to NUT for immunohistochemistry, and confirmation of the fusion (BRD-NUT or NUT-variant) by fluorescent in situ hybridization or reverse transcriptase-polymerase chain reaction.  BRD-NUT functions to block cellular differentiation and promote uncontrolled growth of carcinoma cells.  Because the reagents and expertise required to diagnose NMC are not available in most laboratories, and because of incomplete awareness of this disease, NMC is frequently undiagnosed or misdiagnosed, and its actual prevalence is unknown.  NUT midline carcinoma does not arise from any specific tissue type or organ.  It presents as a poorly differentiated carcinoma originating from midline locations such as the head, neck or mediastinum.  Although rare, NMCs occur throughout life, and advanced local disease is frequently accompanied by distant hematogenous metastases.  There still is no effective treatment for NMC, there are no guidelines, and current approaches to treatment are based on discussions among a few oncologists who each have had a single experience treating this disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTU75vAesIXfgMthz5_Rx5rfW6udTcc2eZeBiLYkuYGOrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3cbgtVeluw%253D%253D&md5=65592b3ac99151b498d9b10f9aab6346</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.cancergencyto.2010.06.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cancergencyto.2010.06.007%26sid%3Dliteratum%253Aachs%26aulast%3DFrench%26aufirst%3DC.%2BA.%26atitle%3DNUT%2520midline%2520carcinoma%26jtitle%3DCancer%2520Genet.%2520Cytogenet.%26date%3D2010%26volume%3D203%26spage%3D16%26epage%3D20%26doi%3D10.1016%2Fj.cancergencyto.2010.06.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nagarajan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hossan, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alawi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Najafova, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Indenbirken, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedi, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taipaleenmaki, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ben-Batalla, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scheller, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loges, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hesse, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grundhoff, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnsen, S. A.</span></span> <span> </span><span class="NLM_article-title">Bromodomain protein BRD4 is required for estrogen receptor-dependent enhancer activation and gene transcription</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">460</span>– <span class="NLM_lpage">469</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2014.06.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1016%2Fj.celrep.2014.06.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=25017071" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFChs7%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2014&pages=460-469&author=S.+Nagarajanauthor=T.+Hossanauthor=M.+Alawiauthor=Z.+Najafovaauthor=D.+Indenbirkenauthor=U.+Bediauthor=H.+Taipaleenmakiauthor=I.+Ben-Batallaauthor=M.+Schellerauthor=S.+Logesauthor=S.+Knappauthor=E.+Hesseauthor=C.+M.+Chiangauthor=A.+Grundhoffauthor=S.+A.+Johnsen&title=Bromodomain+protein+BRD4+is+required+for+estrogen+receptor-dependent+enhancer+activation+and+gene+transcription&doi=10.1016%2Fj.celrep.2014.06.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomain Protein BRD4 Is Required for Estrogen Receptor-Dependent Enhancer Activation and Gene Transcription</span></div><div class="casAuthors">Nagarajan, Sankari; Hossan, Tareq; Alawi, Malik; Najafova, Zeynab; Indenbirken, Daniela; Bedi, Upasana; Taipaleenmaeki, Hanna; Ben-Batalla, Isabel; Scheller, Marina; Loges, Sonja; Knapp, Stefan; Hesse, Eric; Chiang, Cheng-Ming; Grundhoff, Adam; Johnsen, Steven A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">460-469</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The estrogen receptor α (ERα) controls cell proliferation and tumorigenesis by recruiting various cofactors to estrogen response elements (EREs) to control gene transcription.  A deeper understanding of these transcriptional mechanisms may uncover therapeutic targets for ERα-dependent cancers.  We show that BRD4 regulates ERα-induced gene expression by affecting elongation-assocd. phosphorylation of RNA polymerase II (RNAPII) and histone H2B monoubiquitination.  Consistently, BRD4 activity is required for proliferation of ER+ breast and endometrial cancer cells and uterine growth in mice.  Genome-wide studies revealed an enrichment of BRD4 on transcriptional start sites of active genes and a requirement of BRD4 for H2B monoubiquitination in the transcribed region of estrogen-responsive genes.  Importantly, we demonstrate that BRD4 occupancy on distal EREs enriched for H3K27ac is required for recruitment and elongation of RNAPII on EREs and the prodn. of ERα-dependent enhancer RNAs.  These results uncover BRD4 as a central regulator of ERα function and potential therapeutic target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0fNk5Ueff1bVg90H21EOLACvtfcHk0liDzOWmH0rlcw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFChs7%252FM&md5=301a96567713aa777911a158736adb6d</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2014.06.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2014.06.016%26sid%3Dliteratum%253Aachs%26aulast%3DNagarajan%26aufirst%3DS.%26aulast%3DHossan%26aufirst%3DT.%26aulast%3DAlawi%26aufirst%3DM.%26aulast%3DNajafova%26aufirst%3DZ.%26aulast%3DIndenbirken%26aufirst%3DD.%26aulast%3DBedi%26aufirst%3DU.%26aulast%3DTaipaleenmaki%26aufirst%3DH.%26aulast%3DBen-Batalla%26aufirst%3DI.%26aulast%3DScheller%26aufirst%3DM.%26aulast%3DLoges%26aufirst%3DS.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DHesse%26aufirst%3DE.%26aulast%3DChiang%26aufirst%3DC.%2BM.%26aulast%3DGrundhoff%26aufirst%3DA.%26aulast%3DJohnsen%26aufirst%3DS.%2BA.%26atitle%3DBromodomain%2520protein%2520BRD4%2520is%2520required%2520for%2520estrogen%2520receptor-dependent%2520enhancer%2520activation%2520and%2520gene%2520transcription%26jtitle%3DCell%2520Rep.%26date%3D2014%26volume%3D8%26spage%3D460%26epage%3D469%26doi%3D10.1016%2Fj.celrep.2014.06.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, W. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morse, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keates, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickman, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munro, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKeown, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christie, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cameron, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heightman, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">La Thangue, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">French, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiest, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kung, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Selective inhibition of BET bromodomains</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>468</i></span>,  <span class="NLM_fpage">1067</span>– <span class="NLM_lpage">1073</span>, <span class="refDoi"> DOI: 10.1038/nature09504</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1038%2Fnature09504" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=20871596" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1067-1073&author=P.+Filippakopoulosauthor=J.+Qiauthor=S.+Picaudauthor=Y.+Shenauthor=W.+B.+Smithauthor=O.+Fedorovauthor=E.+M.+Morseauthor=T.+Keatesauthor=T.+T.+Hickmanauthor=I.+Felletarauthor=M.+Philpottauthor=S.+Munroauthor=M.+R.+McKeownauthor=Y.+Wangauthor=A.+L.+Christieauthor=N.+Westauthor=M.+J.+Cameronauthor=B.+Schwartzauthor=T.+D.+Heightmanauthor=N.+La+Thangueauthor=C.+A.+Frenchauthor=O.+Wiestauthor=A.+L.+Kungauthor=S.+Knappauthor=J.+E.+Bradner&title=Selective+inhibition+of+BET+bromodomains&doi=10.1038%2Fnature09504"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibition of BET bromodomains</span></div><div class="casAuthors">Filippakopoulos, Panagis; Qi, Jun; Picaud, Sarah; Shen, Yao; Smith, William B.; Fedorov, Oleg; Morse, Elizabeth M.; Keates, Tracey; Hickman, Tyler T.; Felletar, Ildiko; Philpott, Martin; Munro, Shongah; McKeown, Michael R.; Wang, Yuchuan; Christie, Amanda L.; West, Nathan; Cameron, Michael J.; Schwartz, Brian; Heightman, Tom D.; La Thangue, Nicholas; French, Christopher; Wiest, Olaf; Kung, Andrew L.; Knapp, Stefan; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7327</span>),
    <span class="NLM_cas:pages">1067-1073</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Epigenetic proteins are intently pursued targets in ligand discovery.  So far, successful efforts have been limited to chromatin modifying enzymes, or so-called epigenetic 'writers' and 'erasers'.  Potent inhibitors of histone binding modules have not yet been described.  Here the authors report a cell-permeable small mol. (I,JQ1) that binds competitively to acetyl-lysine recognition motifs, or bromodomains.  High potency and specificity towards a subset of human bromodomains is explained by co-crystal structures with bromodomain and extra-terminal (BET) family member BRD4, revealing excellent shape complementarity with the acetyl-lysine binding cavity.  Recurrent translocation of BRD4 is obsd. in a genetically-defined, incurable subtype of human squamous carcinoma.  Competitive binding by JQ1 displaces the BRD4 fusion oncoprotein from chromatin, prompting squamous differentiation and specific antiproliferative effects in BRD4-dependent cell lines and patient-derived xenograft models.  These data establish proof-of-concept for targeting protein-protein interactions of epigenetic 'readers', and provide a versatile chem. scaffold for the development of chem. probes more broadly throughout the bromodomain family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2ZbtmZM8m_LVg90H21EOLACvtfcHk0liDzOWmH0rlcw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D&md5=bd9cadac86124d2c6c3cd5b2a92d68c2</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fnature09504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09504%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DSmith%26aufirst%3DW.%2BB.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMorse%26aufirst%3DE.%2BM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DHickman%26aufirst%3DT.%2BT.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DMunro%26aufirst%3DS.%26aulast%3DMcKeown%26aufirst%3DM.%2BR.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DChristie%26aufirst%3DA.%2BL.%26aulast%3DWest%26aufirst%3DN.%26aulast%3DCameron%26aufirst%3DM.%2BJ.%26aulast%3DSchwartz%26aufirst%3DB.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DLa%2BThangue%26aufirst%3DN.%26aulast%3DFrench%26aufirst%3DC.%2BA.%26aulast%3DWiest%26aufirst%3DO.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DSelective%2520inhibition%2520of%2520BET%2520bromodomains%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D1067%26epage%3D1073%26doi%3D10.1038%2Fnature09504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sengupta, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biarnes, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jordan, V. C.</span></span> <span> </span><span class="NLM_article-title">Inhibition of BET proteins impairs estrogen-mediated growth and transcription in breast cancers by pausing RNA polymerase advancement</span>. <i>Breast Cancer Res. Treat.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>150</i></span>,  <span class="NLM_fpage">265</span>– <span class="NLM_lpage">278</span>, <span class="refDoi"> DOI: 10.1007/s10549-015-3319-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1007%2Fs10549-015-3319-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=25721606" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjs1GrsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=150&publication_year=2015&pages=265-278&author=S.+Senguptaauthor=M.+C.+Biarnesauthor=R.+Clarkeauthor=V.+C.+Jordan&title=Inhibition+of+BET+proteins+impairs+estrogen-mediated+growth+and+transcription+in+breast+cancers+by+pausing+RNA+polymerase+advancement&doi=10.1007%2Fs10549-015-3319-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of BET proteins impairs estrogen-mediated growth and transcription in breast cancers by pausing RNA polymerase advancement</span></div><div class="casAuthors">Sengupta, Surojeet; Biarnes, Michael C.; Clarke, Robert; Jordan, V. Craig</div><div class="citationInfo"><span class="NLM_cas:title">Breast Cancer Research and Treatment</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">150</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">265-278</span>CODEN:
                <span class="NLM_cas:coden">BCTRD6</span>;
        ISSN:<span class="NLM_cas:issn">0167-6806</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Estrogen (E2)-induced transcription requires coordinated recruitment of estrogen receptor α (ER) and multiple factors at the promoter of activated genes.  However, the precise mechanism by which this complex stimulates the RNA polymerase II activity required to execute transcription is largely unresolved.  We investigated the role of bromodomain (BRD) contg. bromodomain and extra-terminal (BET) proteins, in E2-induced growth and gene activation.  JQ1, a specific BET protein inhibitor, was used to block BET protein function in two different ER-pos. breast cancer cell lines (MCF7 and T47D).  Real-time PCR and ChIP assays were used to measure RNA expression and to detect recruitment of various factors on the genes, resp.  Protein levels were measured by Western blotting.  JQ1 suppressed E2-induced growth and transcription in both MCF7 and T47D cells.  The combination of E2 and JQ1 down-regulated the levels of ER protein in MCF7 cells but the loss of ER was not responsible for JQ1-mediated inhibition of E2 signaling.  JQ1 did not disrupt E2-induced recruitment of ER and co-activator (SRC3) at the E2-responsive DNA elements.  The E2-induced increase in histone acetylation was also not altered by JQ1.  However, JQ1 blocked the E2-induced transition of RNA polymerase II from initiation to elongation by stalling it at the promoter region of the responsive genes upstream of the transcription start site.  This study establishes BET proteins as the key mediators of E2-induced transcriptional activation.  This adds another layer of complexity to the regulation of estrogen-induced gene activation that can potentially be targeted for therapeutic intervention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo54AxgJqtE0LVg90H21EOLACvtfcHk0liDzOWmH0rlcw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjs1GrsLw%253D&md5=12afeb1211a6d4112438b4c3afb2c392</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1007%2Fs10549-015-3319-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10549-015-3319-1%26sid%3Dliteratum%253Aachs%26aulast%3DSengupta%26aufirst%3DS.%26aulast%3DBiarnes%26aufirst%3DM.%2BC.%26aulast%3DClarke%26aufirst%3DR.%26aulast%3DJordan%26aufirst%3DV.%2BC.%26atitle%3DInhibition%2520of%2520BET%2520proteins%2520impairs%2520estrogen-mediated%2520growth%2520and%2520transcription%2520in%2520breast%2520cancers%2520by%2520pausing%2520RNA%2520polymerase%2520advancement%26jtitle%3DBreast%2520Cancer%2520Res.%2520Treat.%26date%3D2015%26volume%3D150%26spage%3D265%26epage%3D278%26doi%3D10.1007%2Fs10549-015-3319-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shea, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Creighton, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coarfa, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilsenbeck, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lanz, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nardone, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitsiades, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitsiades, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osborne, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiff, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Malley, B. W.</span></span> <span> </span><span class="NLM_article-title">An epigenomic approach to therapy for tamoxifen-resistant breast cancer</span>. <i>Cell Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">809</span>– <span class="NLM_lpage">819</span>, <span class="refDoi"> DOI: 10.1038/cr.2014.71</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1038%2Fcr.2014.71" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=24874954" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC2cXptFyltLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2014&pages=809-819&author=Q.+Fengauthor=Z.+Zhangauthor=M.+J.+Sheaauthor=C.+J.+Creightonauthor=C.+Coarfaauthor=S.+G.+Hilsenbeckauthor=R.+Lanzauthor=B.+Heauthor=L.+Wangauthor=X.+Fuauthor=A.+Nardoneauthor=Y.+Songauthor=J.+Bradnerauthor=N.+Mitsiadesauthor=C.+S.+Mitsiadesauthor=C.+K.+Osborneauthor=R.+Schiffauthor=B.+W.+O%E2%80%99Malley&title=An+epigenomic+approach+to+therapy+for+tamoxifen-resistant+breast+cancer&doi=10.1038%2Fcr.2014.71"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">An epigenomic approach to therapy for tamoxifen-resistant breast cancer</span></div><div class="casAuthors">Feng, Qin; Zhang, Zheng; Shea, Martin J.; Creighton, Chad J.; Coarfa, Cristian; Hilsenbeck, Susan G.; Lanz, Rainer; He, Bin; Wang, Lei; Fu, Xiaoyong; Nardone, Agostina; Song, Yongcheng; Bradner, James; Mitsiades, Nicholas; Mitsiades, Constantine S.; Osborne, C. Kent; Schiff, Rachel; O'Malley, Bert W.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">809-819</span>CODEN:
                <span class="NLM_cas:coden">CREEB6</span>;
        ISSN:<span class="NLM_cas:issn">1001-0602</span>.
    
            (<span class="NLM_cas:orgname">NPG Nature Asia-Pacific</span>)
        </div><div class="casAbstract">Tamoxifen has been a frontline treatment for estrogen receptor alpha (ERα)-pos. breast tumors in premenopausal women.  However, resistance to tamoxifen occurs in many patients.  ER still plays a crit. role in the growth of breast cancer cells with acquired tamoxifen resistance, suggesting that ERα remains a valid target for treatment of tamoxifen-resistant (Tam-R) breast cancer.  In an effort to identify novel regulators of ERα signaling, through a small-scale siRNA screen against histone Me modifiers, we found WHSC1, a histone H3K36 methyltransferase, as a pos. regulator of ERα signaling in breast cancer cells.  We demonstrated that WHSC1 is recruited to the ERα gene by the BET protein BRD3/4, and facilitates ERα gene expression.  The small-mol. BET protein inhibitor JQ1 potently suppressed the classic ERα signaling pathway and the growth of Tam-R breast cancer cells in culture.  Using a Tam-R breast cancer xenograft mouse model, we demonstrated in vivo anti-breast cancer activity by JQ1 and a strong long-lasting effect of combination therapy with JQ1 and the ER degrader fulvestrant.  Taken together, we provide evidence that the epigenomic proteins BRD3/4 and WHSC1 are essential regulators of estrogen receptor signaling and are novel therapeutic targets for treatment of Tam-R breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqv-Z8nViYZsbVg90H21EOLACvtfcHk0lgISYmPoN5oag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXptFyltLo%253D&md5=aaf9d8656e0ebc529d63ae2b217539b3</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1038%2Fcr.2014.71&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fcr.2014.71%26sid%3Dliteratum%253Aachs%26aulast%3DFeng%26aufirst%3DQ.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DShea%26aufirst%3DM.%2BJ.%26aulast%3DCreighton%26aufirst%3DC.%2BJ.%26aulast%3DCoarfa%26aufirst%3DC.%26aulast%3DHilsenbeck%26aufirst%3DS.%2BG.%26aulast%3DLanz%26aufirst%3DR.%26aulast%3DHe%26aufirst%3DB.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DFu%26aufirst%3DX.%26aulast%3DNardone%26aufirst%3DA.%26aulast%3DSong%26aufirst%3DY.%26aulast%3DBradner%26aufirst%3DJ.%26aulast%3DMitsiades%26aufirst%3DN.%26aulast%3DMitsiades%26aufirst%3DC.%2BS.%26aulast%3DOsborne%26aufirst%3DC.%2BK.%26aulast%3DSchiff%26aufirst%3DR.%26aulast%3DO%25E2%2580%2599Malley%26aufirst%3DB.%2BW.%26atitle%3DAn%2520epigenomic%2520approach%2520to%2520therapy%2520for%2520tamoxifen-resistant%2520breast%2520cancer%26jtitle%3DCell%2520Res.%26date%3D2014%26volume%3D24%26spage%3D809%26epage%3D819%26doi%3D10.1038%2Fcr.2014.71" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wold, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brasier, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span> <span> </span><span class="NLM_article-title">Drug Discovery Targeting Bromodomain-Containing Protein 4</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">4533</span>– <span class="NLM_lpage">4558</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01761</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01761" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC2sXis1yiurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=4533-4558&author=Z.+Liuauthor=P.+Wangauthor=H.+Chenauthor=E.+A.+Woldauthor=B.+Tianauthor=A.+R.+Brasierauthor=J.+Zhou&title=Drug+Discovery+Targeting+Bromodomain-Containing+Protein+4&doi=10.1021%2Facs.jmedchem.6b01761"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Drug Discovery Targeting Bromodomain-Containing Protein 4</span></div><div class="casAuthors">Liu, Zhiqing; Wang, Pingyuan; Chen, Haiying; Wold, Eric A.; Tian, Bing; Brasier, Allan R.; Zhou, Jia</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4533-4558</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  BRD4, the most extensively studied member of BET family, is an epigenetic regulator that localizes to DNA via binding acetylated histones and controls the expression of therapeutically important gene regulatory networks through recruiting transcription factors to form mediator complexes, phosphorylating RNA polymerase II and by its intrinsic histone acetyltransferase activity.  Disrupting the protein-protein interactions between BRD4 and acetyl-lysine has been shown to effectively block cell proliferation in cancer, cytokine prodn. in acute inflammation, etc.  To date, significant efforts have been devoted to the development of BRD4 inhibitors, and consequently, a dozen have progressed into human clin. trials.  Herein, the authors summarize the advances in drug discovery and development of BRD4 inhibitors by focusing on their chemotypes, in vitro and in vivo activity, selectivity, relevant mechanisms of action and therapeutic potential.  Opportunities and challenges to achieve selective and efficacious BRD4 inhibitors as a viable therapeutic strategy for human diseases are also highlighted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6IAERH4iggrVg90H21EOLACvtfcHk0lgISYmPoN5oag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXis1yiurg%253D&md5=6d7ba5844073fac8c916b1249af402e5</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01761&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01761%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DP.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DWold%26aufirst%3DE.%2BA.%26aulast%3DTian%26aufirst%3DB.%26aulast%3DBrasier%26aufirst%3DA.%2BR.%26aulast%3DZhou%26aufirst%3DJ.%26atitle%3DDrug%2520Discovery%2520Targeting%2520Bromodomain-Containing%2520Protein%25204%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D4533%26epage%3D4558%26doi%3D10.1021%2Facs.jmedchem.6b01761" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brotherton, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savitsky, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diez-Dacal, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bountra, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lingard, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span> <span> </span><span class="NLM_article-title">RVX-208, an inhibitor of BET transcriptional regulators with selectivity for the second bromodomain</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">19754</span>– <span class="NLM_lpage">19759</span>, <span class="refDoi"> DOI: 10.1073/pnas.1310658110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1073%2Fpnas.1310658110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=24248379" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFKms7zN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=19754-19759&author=S.+Picaudauthor=C.+Wellsauthor=I.+Felletarauthor=D.+Brothertonauthor=S.+Martinauthor=P.+Savitskyauthor=B.+Diez-Dacalauthor=M.+Philpottauthor=C.+Bountraauthor=H.+Lingardauthor=O.+Fedorovauthor=S.+M%C3%BCllerauthor=P.+E.+Brennanauthor=S.+Knappauthor=P.+Filippakopoulos&title=RVX-208%2C+an+inhibitor+of+BET+transcriptional+regulators+with+selectivity+for+the+second+bromodomain&doi=10.1073%2Fpnas.1310658110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">RVX-208, an inhibitor of BET transcriptional regulators with selectivity for the second bromodomain</span></div><div class="casAuthors">Picaud, Sarah; Wells, Christopher; Felletar, Ildiko; Brotherton, Deborah; Martin, Sarah; Savitsky, Pavel; Diez-Dacal, Beatriz; Philpott, Martin; Bountra, Chas; Lingard, Hannah; Fedorov, Oleg; Muller, Susanne; Brennan, Paul E.; Knapp, Stefan; Filippakopoulos, Panagis</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">19754-19759,S19754/1-S19754/10</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Bromodomains have emerged as attractive candidates for the development of inhibitors targeting gene transcription.  Inhibitors of the bromo and extraterminal (BET) family recently showed promising activity in diverse disease models.  However, the pleiotropic nature of BET proteins regulating tissue-specific transcription has raised safety concerns and suggested that attempts should be made for domain-specific targeting.  Here, we report that RVX-208, a compd. currently in phase II clin. trials, is a BET bromodomain inhibitor specific for second bromodomains (BD2s).  Cocrystal structures revealed binding modes of RVX-208 and its synthetic precursor, and fluorescent recovery after photobleaching demonstrated that RVX-208 displaces BET proteins from chromatin.  However, gene-expression data showed that BD2 inhibition only modestly affects BET-dependent gene transcription.  Our data demonstrate the feasibility of specific targeting within the BET family resulting in different transcriptional outcomes and highlight the importance of BD1 in transcriptional regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqamW8vnIopKrVg90H21EOLACvtfcHk0lgISYmPoN5oag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFKms7zN&md5=c918702c9fdb91241bc837732730f7f6</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1310658110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1310658110%26sid%3Dliteratum%253Aachs%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DBrotherton%26aufirst%3DD.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DSavitsky%26aufirst%3DP.%26aulast%3DDiez-Dacal%26aufirst%3DB.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DLingard%26aufirst%3DH.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26atitle%3DRVX-208%252C%2520an%2520inhibitor%2520of%2520BET%2520transcriptional%2520regulators%2520with%2520selectivity%2520for%2520the%2520second%2520bromodomain%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2013%26volume%3D110%26spage%3D19754%26epage%3D19759%26doi%3D10.1073%2Fpnas.1310658110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mirguet, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gosmini, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toum, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clement, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnathan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brusq, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mordaunt, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimes, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crowe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pineau, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ajakane, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daugan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeffrey, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cutler, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haynes, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smithers, N. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uings, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witherington, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parr, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicodeme, E.</span></span> <span> </span><span class="NLM_article-title">Discovery of epigenetic regulator I-BET762: lead optimization to afford a clinical candidate inhibitor of the BET bromodomains</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">7501</span>– <span class="NLM_lpage">7515</span>, <span class="refDoi"> DOI: 10.1021/jm401088k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401088k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVSls7jJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=7501-7515&author=O.+Mirguetauthor=R.+Gosminiauthor=J.+Toumauthor=C.+A.+Clementauthor=M.+Barnathanauthor=J.+M.+Brusqauthor=J.+E.+Mordauntauthor=R.+M.+Grimesauthor=M.+Croweauthor=O.+Pineauauthor=M.+Ajakaneauthor=A.+Dauganauthor=P.+Jeffreyauthor=L.+Cutlerauthor=A.+C.+Haynesauthor=N.+N.+Smithersauthor=C.+W.+Chungauthor=P.+Bamboroughauthor=I.+J.+Uingsauthor=A.+Lewisauthor=J.+Witheringtonauthor=N.+Parrauthor=R.+K.+Prinjhaauthor=E.+Nicodeme&title=Discovery+of+epigenetic+regulator+I-BET762%3A+lead+optimization+to+afford+a+clinical+candidate+inhibitor+of+the+BET+bromodomains&doi=10.1021%2Fjm401088k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Epigenetic Regulator I-BET762: Lead Optimization to Afford a Clinical Candidate Inhibitor of the BET Bromodomains</span></div><div class="casAuthors">Mirguet, Olivier; Gosmini, Romain; Toum, Jerome; Clement, Catherine A.; Barnathan, Melanie; Brusq, Jean-Marie; Mordaunt, Jacqueline E.; Grimes, Richard M.; Crowe, Miriam; Pineau, Olivier; Ajakane, Myriam; Daugan, Alain; Jeffrey, Phillip; Cutler, Leanne; Haynes, Andrea C.; Smithers, Nicholas N.; Chung, Chun-wa; Bamborough, Paul; Uings, Iain J.; Lewis, Antonia; Witherington, Jason; Parr, Nigel; Prinjha, Rab K.; Nicodeme, Edwige</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">7501-7515</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bromo and extra C-terminal domain (BET) family of bromodomains are involved in binding epigenetic marks on histone proteins, more specifically acetylated lysine residues.  This paper describes the discovery and structure-activity relationships (SAR) of potent benzodiazepine inhibitors that disrupt the function of the BET family of bromodomains (BRD2, BRD3, and BRD4).  This work has yielded a potent, selective compd. I-BET762 (I) that is now under evaluation in a phase I/II clin. trial for nuclear protein in testis (NUT) midline carcinoma and other cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDitaXJSnFE7Vg90H21EOLACvtfcHk0ljveOzqZpnIeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVSls7jJ&md5=5d059b0be08a74a5721b72a120f7e090</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fjm401088k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401088k%26sid%3Dliteratum%253Aachs%26aulast%3DMirguet%26aufirst%3DO.%26aulast%3DGosmini%26aufirst%3DR.%26aulast%3DToum%26aufirst%3DJ.%26aulast%3DClement%26aufirst%3DC.%2BA.%26aulast%3DBarnathan%26aufirst%3DM.%26aulast%3DBrusq%26aufirst%3DJ.%2BM.%26aulast%3DMordaunt%26aufirst%3DJ.%2BE.%26aulast%3DGrimes%26aufirst%3DR.%2BM.%26aulast%3DCrowe%26aufirst%3DM.%26aulast%3DPineau%26aufirst%3DO.%26aulast%3DAjakane%26aufirst%3DM.%26aulast%3DDaugan%26aufirst%3DA.%26aulast%3DJeffrey%26aufirst%3DP.%26aulast%3DCutler%26aufirst%3DL.%26aulast%3DHaynes%26aufirst%3DA.%2BC.%26aulast%3DSmithers%26aufirst%3DN.%2BN.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DUings%26aufirst%3DI.%2BJ.%26aulast%3DLewis%26aufirst%3DA.%26aulast%3DWitherington%26aufirst%3DJ.%26aulast%3DParr%26aufirst%3DN.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DNicodeme%26aufirst%3DE.%26atitle%3DDiscovery%2520of%2520epigenetic%2520regulator%2520I-BET762%253A%2520lead%2520optimization%2520to%2520afford%2520a%2520clinical%2520candidate%2520inhibitor%2520of%2520the%2520BET%2520bromodomains%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D7501%26epage%3D7515%26doi%3D10.1021%2Fjm401088k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McDaniel, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soltwedel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fidanze, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasvold, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mantei, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pratt, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bui, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faivre, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warder, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilcox, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albert, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magoc, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajaraman, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hutchins, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edalji, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elmore, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kati, W. M.</span></span> <span> </span><span class="NLM_article-title">Discovery of N-(4-(2,4-Difluorophenoxy)-3-(6-methyl-7-oxo-6,7-dihydro-1H-pyrrolo[2,3-c]pyridin-4-yl)phenyl)ethanesulfonamide (ABBV-075/Mivebresib), a Potent and Orally Available Bromodomain and Extraterminal Domain (BET) Family Bromodomain Inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">8369</span>– <span class="NLM_lpage">8384</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00746</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00746" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFOjtrbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=8369-8384&author=K.+F.+McDanielauthor=L.+Wangauthor=T.+Soltwedelauthor=S.+D.+Fidanzeauthor=L.+A.+Hasvoldauthor=D.+Liuauthor=R.+A.+Manteiauthor=J.+K.+Prattauthor=G.+S.+Sheppardauthor=M.+H.+Buiauthor=E.+J.+Faivreauthor=X.+Huangauthor=L.+Liauthor=X.+Linauthor=R.+Wangauthor=S.+E.+Warderauthor=D.+Wilcoxauthor=D.+H.+Albertauthor=T.+J.+Magocauthor=G.+Rajaramanauthor=C.+H.+Parkauthor=C.+W.+Hutchinsauthor=J.+J.+Shenauthor=R.+P.+Edaljiauthor=C.+C.+Sunauthor=R.+Martinauthor=W.+Gaoauthor=S.+Wongauthor=G.+Fangauthor=S.+W.+Elmoreauthor=Y.+Shenauthor=W.+M.+Kati&title=Discovery+of+N-%284-%282%2C4-Difluorophenoxy%29-3-%286-methyl-7-oxo-6%2C7-dihydro-1H-pyrrolo%5B2%2C3-c%5Dpyridin-4-yl%29phenyl%29ethanesulfonamide+%28ABBV-075%2FMivebresib%29%2C+a+Potent+and+Orally+Available+Bromodomain+and+Extraterminal+Domain+%28BET%29+Family+Bromodomain+Inhibitor&doi=10.1021%2Facs.jmedchem.7b00746"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of N-(4-(2,4-Difluorophenoxy)-3-(6-methyl-7-oxo-6,7-dihydro-1H-pyrrolo[2,3-c]pyridin-4-yl)phenyl)ethanesulfonamide (ABBV-075/Mivebresib), a Potent and Orally Available Bromodomain and Extraterminal Domain (BET) Family Bromodomain Inhibitor</span></div><div class="casAuthors">McDaniel, Keith F.; Wang, Le; Soltwedel, Todd; Fidanze, Steven D.; Hasvold, Lisa A.; Liu, Dachun; Mantei, Robert A.; Pratt, John K.; Sheppard, George S.; Bui, Mai H.; Faivre, Emily J.; Huang, Xiaoli; Li, Leiming; Lin, Xiaoyu; Wang, Rongqi; Warder, Scott E.; Wilcox, Denise; Albert, Daniel H.; Magoc, Terrance J.; Rajaraman, Ganesh; Park, Chang H.; Hutchins, Charles W.; Shen, Jianwei J.; Edalji, Rohinton P.; Sun, Chaohong C.; Martin, Ruth; Gao, Wenqing; Wong, Shekman; Fang, Guowei; Elmore, Steven W.; Shen, Yu; Kati, Warren M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">8369-8384</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The development of bromodomain and extraterminal domain (BET) bromodomain inhibitors and their examn. in clin. studies, particularly in oncol. settings, has garnered substantial recent interest.  An effort to generate novel BET bromodomain inhibitors with excellent potency and drug metab. and pharmacokinetics (DMPK) properties was initiated based upon elaboration of a simple pyridone core.  Efforts to develop a bidentate interaction with a crit. asparagine residue resulted in the incorporation of a pyrrolopyridone core, which improved potency by 9-19-fold.  Addnl. structure-activity relationship (SAR) efforts aimed both at increasing potency and improving pharmacokinetic properties led to the discovery of the clin. candidate I (ABBV-075/mivebresib), which demonstrates excellent potency in biochem. and cellular assays, advantageous exposures and half-life both in animal models and in humans, and in vivo efficacy in mouse models of cancer progression and inflammation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGom-8BkipNknbVg90H21EOLACvtfcHk0ljveOzqZpnIeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFOjtrbF&md5=16ea9e9ade8a22a6e99aeae16fd1bf9c</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00746&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00746%26sid%3Dliteratum%253Aachs%26aulast%3DMcDaniel%26aufirst%3DK.%2BF.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DSoltwedel%26aufirst%3DT.%26aulast%3DFidanze%26aufirst%3DS.%2BD.%26aulast%3DHasvold%26aufirst%3DL.%2BA.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DMantei%26aufirst%3DR.%2BA.%26aulast%3DPratt%26aufirst%3DJ.%2BK.%26aulast%3DSheppard%26aufirst%3DG.%2BS.%26aulast%3DBui%26aufirst%3DM.%2BH.%26aulast%3DFaivre%26aufirst%3DE.%2BJ.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DR.%26aulast%3DWarder%26aufirst%3DS.%2BE.%26aulast%3DWilcox%26aufirst%3DD.%26aulast%3DAlbert%26aufirst%3DD.%2BH.%26aulast%3DMagoc%26aufirst%3DT.%2BJ.%26aulast%3DRajaraman%26aufirst%3DG.%26aulast%3DPark%26aufirst%3DC.%2BH.%26aulast%3DHutchins%26aufirst%3DC.%2BW.%26aulast%3DShen%26aufirst%3DJ.%2BJ.%26aulast%3DEdalji%26aufirst%3DR.%2BP.%26aulast%3DSun%26aufirst%3DC.%2BC.%26aulast%3DMartin%26aufirst%3DR.%26aulast%3DGao%26aufirst%3DW.%26aulast%3DWong%26aufirst%3DS.%26aulast%3DFang%26aufirst%3DG.%26aulast%3DElmore%26aufirst%3DS.%2BW.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DKati%26aufirst%3DW.%2BM.%26atitle%3DDiscovery%2520of%2520N-%25284-%25282%252C4-Difluorophenoxy%2529-3-%25286-methyl-7-oxo-6%252C7-dihydro-1H-pyrrolo%255B2%252C3-c%255Dpyridin-4-yl%2529phenyl%2529ethanesulfonamide%2520%2528ABBV-075%252FMivebresib%2529%252C%2520a%2520Potent%2520and%2520Orally%2520Available%2520Bromodomain%2520and%2520Extraterminal%2520Domain%2520%2528BET%2529%2520Family%2520Bromodomain%2520Inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D8369%26epage%3D8384%26doi%3D10.1021%2Facs.jmedchem.7b00746" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gavai, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norris, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tortolani, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Malley, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quesnelle, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gill, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaccaro, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huynh, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahuja, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodd, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mussari, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harikrishnan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamau, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tokarski, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheriff, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rampulla, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, D.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yip, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathur, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ranasinghe, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Arienzo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Everlof, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raghavan, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poss, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vite, G. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westhouse, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wee, S.</span></span> <span> </span><span class="NLM_article-title">Abstract 5789: Discovery of clinical candidate BMS-986158, an oral BET inhibitor, for the treatment of cancer</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>78</i></span>,  <span class="NLM_fpage">5789</span>– <span class="NLM_lpage">5789</span>, <span class="refDoi"> DOI: 10.1158/1538-7445.AM2018-5789</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1158%2F1538-7445.AM2018-5789" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=2018&pages=5789-5789&author=A.+V.+Gavaiauthor=D.+Norrisauthor=D.+Tortolaniauthor=D.+O%E2%80%99Malleyauthor=Y.+Zhaoauthor=C.+Quesnelleauthor=P.+Gillauthor=W.+Vaccaroauthor=T.+Huynhauthor=V.+Ahujaauthor=D.+Doddauthor=C.+Mussariauthor=L.+Harikrishnanauthor=M.+Kamauauthor=J.+S.+Tokarskiauthor=S.+Sheriffauthor=R.+Rampullaauthor=D.-R.+Wuauthor=J.+Liauthor=H.+Zhangauthor=P.+Liauthor=D.+Sunauthor=H.+Yipauthor=Y.+Zhangauthor=A.+Mathurauthor=H.+Zhangauthor=C.+Huangauthor=Z.+Yangauthor=A.+Ranasingheauthor=C.+D%E2%80%99Arienzoauthor=C.+Suauthor=G.+Everlofauthor=L.+Zhangauthor=N.+Raghavanauthor=J.+T.+Huntauthor=M.+Possauthor=G.+D.+Viteauthor=R.+A.+Westhouseauthor=S.+Wee&title=Abstract+5789%3A+Discovery+of+clinical+candidate+BMS-986158%2C+an+oral+BET+inhibitor%2C+for+the+treatment+of+cancer&doi=10.1158%2F1538-7445.AM2018-5789"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1158%2F1538-7445.AM2018-5789&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1538-7445.AM2018-5789%26sid%3Dliteratum%253Aachs%26aulast%3DGavai%26aufirst%3DA.%2BV.%26aulast%3DNorris%26aufirst%3DD.%26aulast%3DTortolani%26aufirst%3DD.%26aulast%3DO%25E2%2580%2599Malley%26aufirst%3DD.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DQuesnelle%26aufirst%3DC.%26aulast%3DGill%26aufirst%3DP.%26aulast%3DVaccaro%26aufirst%3DW.%26aulast%3DHuynh%26aufirst%3DT.%26aulast%3DAhuja%26aufirst%3DV.%26aulast%3DDodd%26aufirst%3DD.%26aulast%3DMussari%26aufirst%3DC.%26aulast%3DHarikrishnan%26aufirst%3DL.%26aulast%3DKamau%26aufirst%3DM.%26aulast%3DTokarski%26aufirst%3DJ.%2BS.%26aulast%3DSheriff%26aufirst%3DS.%26aulast%3DRampulla%26aufirst%3DR.%26aulast%3DWu%26aufirst%3DD.-R.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DP.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DYip%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DMathur%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DHuang%26aufirst%3DC.%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DRanasinghe%26aufirst%3DA.%26aulast%3DD%25E2%2580%2599Arienzo%26aufirst%3DC.%26aulast%3DSu%26aufirst%3DC.%26aulast%3DEverlof%26aufirst%3DG.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DRaghavan%26aufirst%3DN.%26aulast%3DHunt%26aufirst%3DJ.%2BT.%26aulast%3DPoss%26aufirst%3DM.%26aulast%3DVite%26aufirst%3DG.%2BD.%26aulast%3DWesthouse%26aufirst%3DR.%2BA.%26aulast%3DWee%26aufirst%3DS.%26atitle%3DAbstract%25205789%253A%2520Discovery%2520of%2520clinical%2520candidate%2520BMS-986158%252C%2520an%2520oral%2520BET%2520inhibitor%252C%2520for%2520the%2520treatment%2520of%2520cancer%26jtitle%3DCancer%2520Res.%26date%3D2018%26volume%3D78%26spage%3D5789%26epage%3D5789%26doi%3D10.1158%2F1538-7445.AM2018-5789" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bradbury, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Callis, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carr, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feron, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glossop, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graham, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hattersley, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamont, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouvry, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabow, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stokes, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stratton, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whalley, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whittaker, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wrigley, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waring, M. J.</span></span> <span> </span><span class="NLM_article-title">Optimization of a Series of Bivalent Triazolopyridazine Based Bromodomain and Extraterminal Inhibitors: The Discovery of (3R)-4-[2-[4-[1-(3-Methoxy-[1,2,4]triazolo[4,3-b]pyridazin-6-yl)-4-piperidyl]phenoxy]ethyl]-1,3-dimethyl-piperazin-2-one (AZD5153)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">7801</span>– <span class="NLM_lpage">7817</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00070</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00070" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhtleiu7rK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=7801-7817&author=R.+H.+Bradburyauthor=R.+Callisauthor=G.+R.+Carrauthor=H.+Chenauthor=E.+Clarkauthor=L.+Feronauthor=S.+Glossopauthor=M.+A.+Grahamauthor=M.+Hattersleyauthor=C.+Jonesauthor=S.+G.+Lamontauthor=G.+Ouvryauthor=A.+Patelauthor=J.+Patelauthor=A.+A.+Rabowauthor=C.+A.+Robertsauthor=S.+Stokesauthor=N.+Strattonauthor=G.+E.+Walkerauthor=L.+Wardauthor=D.+Whalleyauthor=D.+Whittakerauthor=G.+Wrigleyauthor=M.+J.+Waring&title=Optimization+of+a+Series+of+Bivalent+Triazolopyridazine+Based+Bromodomain+and+Extraterminal+Inhibitors%3A+The+Discovery+of+%283R%29-4-%5B2-%5B4-%5B1-%283-Methoxy-%5B1%2C2%2C4%5Dtriazolo%5B4%2C3-b%5Dpyridazin-6-yl%29-4-piperidyl%5Dphenoxy%5Dethyl%5D-1%2C3-dimethyl-piperazin-2-one+%28AZD5153%29&doi=10.1021%2Facs.jmedchem.6b00070"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of a Series of Bivalent Triazolopyridazine Based Bromodomain and Extraterminal Inhibitors: The Discovery of (3R)-4-[2-[4-[1-(3-Methoxy-[1,2,4]triazolo[4,3-b]pyridazin-6-yl)-4-piperidyl]phenoxy]ethyl]-1,3-dimethyl-piperazin-2-one (AZD5153)</span></div><div class="casAuthors">Bradbury, Robert H.; Callis, Rowena; Carr, Gregory R.; Chen, Huawei; Clark, Edwin; Feron, Lyman; Glossop, Steve; Graham, Mark A.; Hattersley, Maureen; Jones, Chris; Lamont, Scott G.; Ouvry, Gilles; Patel, Anil; Patel, Joe; Rabow, Alfred A.; Roberts, Craig A.; Stokes, Stephen; Stratton, Natalie; Walker, Graeme E.; Ward, Lara; Whalley, David; Whittaker, David; Wrigley, Gail; Waring, Michael J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">7801-7817</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Here the authors report the discovery and optimization of a series of bivalent bromodomain and extraterminal inhibitors.  Starting with the observation of BRD4 activity of compds. from a previous program, the compds. were optimized for BRD4 potency and phys. properties.  The optimized compd. from this campaign exhibited excellent pharmacokinetic profile and exhibited high potency in vitro and in vivo effecting c-Myc downregulation and tumor growth inhibition in xenograft studies.  This compd. was selected as the development candidate AZD5153.  The series showed enhanced potency as a result of bivalent binding and a clear correlation between BRD4 activity and cellular potency.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFRSE33Jdb17Vg90H21EOLACvtfcHk0lggyGjnGlDAvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhtleiu7rK&md5=a91cdef7369dd1888c4948ba4fc9d72d</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00070%26sid%3Dliteratum%253Aachs%26aulast%3DBradbury%26aufirst%3DR.%2BH.%26aulast%3DCallis%26aufirst%3DR.%26aulast%3DCarr%26aufirst%3DG.%2BR.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DClark%26aufirst%3DE.%26aulast%3DFeron%26aufirst%3DL.%26aulast%3DGlossop%26aufirst%3DS.%26aulast%3DGraham%26aufirst%3DM.%2BA.%26aulast%3DHattersley%26aufirst%3DM.%26aulast%3DJones%26aufirst%3DC.%26aulast%3DLamont%26aufirst%3DS.%2BG.%26aulast%3DOuvry%26aufirst%3DG.%26aulast%3DPatel%26aufirst%3DA.%26aulast%3DPatel%26aufirst%3DJ.%26aulast%3DRabow%26aufirst%3DA.%2BA.%26aulast%3DRoberts%26aufirst%3DC.%2BA.%26aulast%3DStokes%26aufirst%3DS.%26aulast%3DStratton%26aufirst%3DN.%26aulast%3DWalker%26aufirst%3DG.%2BE.%26aulast%3DWard%26aufirst%3DL.%26aulast%3DWhalley%26aufirst%3DD.%26aulast%3DWhittaker%26aufirst%3DD.%26aulast%3DWrigley%26aufirst%3DG.%26aulast%3DWaring%26aufirst%3DM.%2BJ.%26atitle%3DOptimization%2520of%2520a%2520Series%2520of%2520Bivalent%2520Triazolopyridazine%2520Based%2520Bromodomain%2520and%2520Extraterminal%2520Inhibitors%253A%2520The%2520Discovery%2520of%2520%25283R%2529-4-%255B2-%255B4-%255B1-%25283-Methoxy-%255B1%252C2%252C4%255Dtriazolo%255B4%252C3-b%255Dpyridazin-6-yl%2529-4-piperidyl%255Dphenoxy%255Dethyl%255D-1%252C3-dimethyl-piperazin-2-one%2520%2528AZD5153%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D7801%26epage%3D7817%26doi%3D10.1021%2Facs.jmedchem.6b00070" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Coude, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braun, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berrou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dupont, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertrand, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masse, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raffoux, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Itzykson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delord, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riveiro, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herait, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baruchel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dombret, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gardin, C.</span></span> <span> </span><span class="NLM_article-title">BET inhibitor OTX015 targets BRD2 and BRD4 and decreases c-MYC in acute leukemia cells</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">17698</span>– <span class="NLM_lpage">17712</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.4131</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.18632%2Foncotarget.4131" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=25989842" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A280%3ADC%252BC2MflsVChsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=17698-17712&author=M.+M.+Coudeauthor=T.+Braunauthor=J.+Berrouauthor=M.+Dupontauthor=S.+Bertrandauthor=A.+Masseauthor=E.+Raffouxauthor=R.+Itzyksonauthor=M.+Delordauthor=M.+E.+Riveiroauthor=P.+Heraitauthor=A.+Baruchelauthor=H.+Dombretauthor=C.+Gardin&title=BET+inhibitor+OTX015+targets+BRD2+and+BRD4+and+decreases+c-MYC+in+acute+leukemia+cells&doi=10.18632%2Foncotarget.4131"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">BET inhibitor OTX015 targets BRD2 and BRD4 and decreases c-MYC in acute leukemia cells</span></div><div class="casAuthors">Coude Marie-Magdelaine; Braun Thorsten; Berrou Jeannig; Dupont Melanie; Bertrand Sibyl; Masse Aline; Raffoux Emmanuel; Itzykson Raphael; Baruchel Andre; Dombret Herve; Gardin Claude; Coude Marie-Magdelaine; Braun Thorsten; Gardin Claude; Raffoux Emmanuel; Itzykson Raphael; Dombret Herve; Delord Marc; Riveiro Maria E; Herait Patrice; Baruchel Andre</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">17698-712</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The bromodomain (BRD) and extraterminal (BET) proteins including BRD2, BRD3 and BRD4 have been identified as key targets for leukemia maintenance.  A novel oral inhibitor of BRD2/3/4, the thienotriazolodiazepine compound OTX015, suitable for human use, is available.  Here we report its biological effects in AML and ALL cell lines and leukemic samples.  Exposure to OTX015 lead to cell growth inhibition, cell cycle arrest and apoptosis at submicromolar concentrations in acute leukemia cell lines and patient-derived leukemic cells, as described with the canonical JQ1 BET inhibitor.  Treatment with JQ1 and OTX15 induces similar gene expression profiles in sensitive cell lines, including a c-MYC decrease and an HEXIM1 increase.  OTX015 exposure also induced a strong decrease of BRD2, BRD4 and c-MYC and increase of HEXIM1 proteins, while BRD3 expression was unchanged. c-MYC, BRD2, BRD3, BRD4 and HEXIM1 mRNA levels did not correlate however with viability following exposure to OTX015.  Sequential combinations of OTX015 with other epigenetic modifying drugs, panobinostat and azacitidine have a synergic effect on growth of the KASUMI cell line.  Our results indicate that OTX015 and JQ1 have similar biological effects in leukemic cells, supporting OTX015 evaluation in a Phase Ib trial in relapsed/refractory leukemia patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSlHRu3Td8hIa_dlEzl_fBafW6udTcc2eaZXOdokavXK7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MflsVChsA%253D%253D&md5=0db953a0a24b12ec2918a02d1a2df00b</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.4131&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.4131%26sid%3Dliteratum%253Aachs%26aulast%3DCoude%26aufirst%3DM.%2BM.%26aulast%3DBraun%26aufirst%3DT.%26aulast%3DBerrou%26aufirst%3DJ.%26aulast%3DDupont%26aufirst%3DM.%26aulast%3DBertrand%26aufirst%3DS.%26aulast%3DMasse%26aufirst%3DA.%26aulast%3DRaffoux%26aufirst%3DE.%26aulast%3DItzykson%26aufirst%3DR.%26aulast%3DDelord%26aufirst%3DM.%26aulast%3DRiveiro%26aufirst%3DM.%2BE.%26aulast%3DHerait%26aufirst%3DP.%26aulast%3DBaruchel%26aufirst%3DA.%26aulast%3DDombret%26aufirst%3DH.%26aulast%3DGardin%26aufirst%3DC.%26atitle%3DBET%2520inhibitor%2520OTX015%2520targets%2520BRD2%2520and%2520BRD4%2520and%2520decreases%2520c-MYC%2520in%2520acute%2520leukemia%2520cells%26jtitle%3DOncotarget%26date%3D2015%26volume%3D6%26spage%3D17698%26epage%3D17712%26doi%3D10.18632%2Foncotarget.4131" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Albrecht, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gehling, V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hewitt, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaswani, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Côté, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leblanc, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nasveschuk, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergeron, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantone, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jayaram, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joshi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mertz, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neiss, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Normant, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Meara, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pardo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poy, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sandy, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Supko, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sims, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harmange, J.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Audia, J. E.</span></span> <span> </span><span class="NLM_article-title">Identification of a Benzoisoxazoloazepine Inhibitor (CPI-0610) of the Bromodomain and Extra-Terminal (BET) Family as a Candidate for Human Clinical Trials</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1330</span>– <span class="NLM_lpage">1339</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01882</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01882" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFKjs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1330-1339&author=B.+K.+Albrechtauthor=V.+S.+Gehlingauthor=M.+C.+Hewittauthor=R.+G.+Vaswaniauthor=A.+C%C3%B4t%C3%A9author=Y.+Leblancauthor=C.+G.+Nasveschukauthor=S.+Bellonauthor=L.+Bergeronauthor=R.+Campbellauthor=N.+Cantoneauthor=M.+R.+Cooperauthor=R.+T.+Cummingsauthor=H.+Jayaramauthor=S.+Joshiauthor=J.+A.+Mertzauthor=A.+Neissauthor=E.+Normantauthor=M.+O%E2%80%99Mearaauthor=E.+Pardoauthor=F.+Poyauthor=P.+Sandyauthor=J.+Supkoauthor=R.+J.+Simsauthor=J.-C.+Harmangeauthor=A.+M.+Taylorauthor=J.+E.+Audia&title=Identification+of+a+Benzoisoxazoloazepine+Inhibitor+%28CPI-0610%29+of+the+Bromodomain+and+Extra-Terminal+%28BET%29+Family+as+a+Candidate+for+Human+Clinical+Trials&doi=10.1021%2Facs.jmedchem.5b01882"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a Benzoisoxazoloazepine Inhibitor (CPI-0610) of the Bromodomain and Extra-Terminal (BET) Family as a Candidate for Human Clinical Trials</span></div><div class="casAuthors">Albrecht, Brian K.; Gehling, Victor S.; Hewitt, Michael C.; Vaswani, Rishi G.; Cote, Alexandre; Leblanc, Yves; Nasveschuk, Christopher G.; Bellon, Steve; Bergeron, Louise; Campbell, Robert; Cantone, Nico; Cooper, Michael R.; Cummings, Richard T.; Jayaram, Hariharan; Joshi, Shivangi; Mertz, Jennifer A.; Neiss, Adrianne; Normant, Emmanuel; O'Meara, Michael; Pardo, Eneida; Poy, Florence; Sandy, Peter; Supko, Jeffrey; Sims, Robert J.; Harmange, Jean-Christophe; Taylor, Alexander M.; Audia, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1330-1339</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In recent years, inhibition of the interaction between the bromodomain and extra-terminal domain (BET) family of chromatin adaptors and acetyl-lysine residues on chromatin has emerged as a promising approach to regulate the expression of important disease-relevant genes, including MYC, BCL-2, and NF-κB.  Here we describe the identification and characterization of a potent and selective benzoisoxazoloazepine BET bromodomain inhibitor that attenuates BET-dependent gene expression in vivo, demonstrates antitumor efficacy in an MV-4-11 mouse xenograft model, and is currently undergoing human clin. trials for hematol. malignancies (CPI-0610).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp40qjnYl4EZLVg90H21EOLACvtfcHk0limhu-5zgWg1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFKjs7g%253D&md5=5e1c5aab26aa58844a9f6670a7bbd78b</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01882&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01882%26sid%3Dliteratum%253Aachs%26aulast%3DAlbrecht%26aufirst%3DB.%2BK.%26aulast%3DGehling%26aufirst%3DV.%2BS.%26aulast%3DHewitt%26aufirst%3DM.%2BC.%26aulast%3DVaswani%26aufirst%3DR.%2BG.%26aulast%3DC%25C3%25B4t%25C3%25A9%26aufirst%3DA.%26aulast%3DLeblanc%26aufirst%3DY.%26aulast%3DNasveschuk%26aufirst%3DC.%2BG.%26aulast%3DBellon%26aufirst%3DS.%26aulast%3DBergeron%26aufirst%3DL.%26aulast%3DCampbell%26aufirst%3DR.%26aulast%3DCantone%26aufirst%3DN.%26aulast%3DCooper%26aufirst%3DM.%2BR.%26aulast%3DCummings%26aufirst%3DR.%2BT.%26aulast%3DJayaram%26aufirst%3DH.%26aulast%3DJoshi%26aufirst%3DS.%26aulast%3DMertz%26aufirst%3DJ.%2BA.%26aulast%3DNeiss%26aufirst%3DA.%26aulast%3DNormant%26aufirst%3DE.%26aulast%3DO%25E2%2580%2599Meara%26aufirst%3DM.%26aulast%3DPardo%26aufirst%3DE.%26aulast%3DPoy%26aufirst%3DF.%26aulast%3DSandy%26aufirst%3DP.%26aulast%3DSupko%26aufirst%3DJ.%26aulast%3DSims%26aufirst%3DR.%2BJ.%26aulast%3DHarmange%26aufirst%3DJ.-C.%26aulast%3DTaylor%26aufirst%3DA.%2BM.%26aulast%3DAudia%26aufirst%3DJ.%2BE.%26atitle%3DIdentification%2520of%2520a%2520Benzoisoxazoloazepine%2520Inhibitor%2520%2528CPI-0610%2529%2520of%2520the%2520Bromodomain%2520and%2520Extra-Terminal%2520%2528BET%2529%2520Family%2520as%2520a%2520Candidate%2520for%2520Human%2520Clinical%2520Trials%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1330%26epage%3D1339%26doi%3D10.1021%2Facs.jmedchem.5b01882" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tontsch-Grunt, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savarese, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerlach, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gianni, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baum, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scharn, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engelhardt, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaya, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schweifer, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerstberger, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraut, N.</span></span> <span> </span><span class="NLM_article-title">Abstract B79: BI 894999, a novel BET inhibitor: Treatment of hematological malignancies by repression of super-enhancer driven oncogenes</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">B79</span>– <span class="NLM_lpage">79</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.TARG-15-B79</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1158%2F1535-7163.TARG-15-B79" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2015&pages=B79-79&author=U.+Tontsch-Gruntauthor=F.+Savareseauthor=D.+Gerlachauthor=D.+Gianniauthor=A.+Baumauthor=D.+Scharnauthor=H.+Engelhardtauthor=O.+Kayaauthor=N.+Schweiferauthor=T.+Gerstbergerauthor=N.+Kraut&title=Abstract+B79%3A+BI+894999%2C+a+novel+BET+inhibitor%3A+Treatment+of+hematological+malignancies+by+repression+of+super-enhancer+driven+oncogenes&doi=10.1158%2F1535-7163.TARG-15-B79"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.TARG-15-B79&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.TARG-15-B79%26sid%3Dliteratum%253Aachs%26aulast%3DTontsch-Grunt%26aufirst%3DU.%26aulast%3DSavarese%26aufirst%3DF.%26aulast%3DGerlach%26aufirst%3DD.%26aulast%3DGianni%26aufirst%3DD.%26aulast%3DBaum%26aufirst%3DA.%26aulast%3DScharn%26aufirst%3DD.%26aulast%3DEngelhardt%26aufirst%3DH.%26aulast%3DKaya%26aufirst%3DO.%26aulast%3DSchweifer%26aufirst%3DN.%26aulast%3DGerstberger%26aufirst%3DT.%26aulast%3DKraut%26aufirst%3DN.%26atitle%3DAbstract%2520B79%253A%2520BI%2520894999%252C%2520a%2520novel%2520BET%2520inhibitor%253A%2520Treatment%2520of%2520hematological%2520malignancies%2520by%2520repression%2520of%2520super-enhancer%2520driven%2520oncogenes%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2015%26volume%3D14%26spage%3DB79%26epage%3D79%26doi%3D10.1158%2F1535-7163.TARG-15-B79" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, P. C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stubbs, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maduskuie, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sparks, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zolotarjova, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Favata, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feldman, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Volgina, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DiMatteo, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Falahatpisheh, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polam, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Covington, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diamond-Fosbenner, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wynn, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burn, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaddi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeleswaram, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Combs, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scherle, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hollis, G.</span></span> <span> </span><span class="NLM_article-title">Abstract 3523: Discovery of a novel BET inhibitor INCB054329</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>75</i></span>,  <span class="NLM_fpage">3523</span>– <span class="NLM_lpage">3523</span>, <span class="refDoi"> DOI: 10.1158/1538-7445.AM2015-3523</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1158%2F1538-7445.AM2015-3523" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2015&pages=3523-3523&author=P.+C.+C.+Liuauthor=X.+M.+Liuauthor=M.+C.+Stubbsauthor=T.+Maduskuieauthor=R.+Sparksauthor=N.+Zolotarjovaauthor=J.+Liauthor=X.+Wenauthor=M.+Favataauthor=P.+Feldmanauthor=A.+Volginaauthor=D.+DiMatteoauthor=R.+Collinsauthor=N.+Falahatpishehauthor=P.+Polamauthor=Y.+Liauthor=M.+Covingtonauthor=S.+Diamond-Fosbennerauthor=R.+Wynnauthor=T.+Burnauthor=K.+Vaddiauthor=S.+Yeleswaramauthor=A.+P.+Combsauthor=W.+Yaoauthor=R.+Huberauthor=P.+Scherleauthor=G.+Hollis&title=Abstract+3523%3A+Discovery+of+a+novel+BET+inhibitor+INCB054329&doi=10.1158%2F1538-7445.AM2015-3523"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1158%2F1538-7445.AM2015-3523&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1538-7445.AM2015-3523%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DP.%2BC.%2BC.%26aulast%3DLiu%26aufirst%3DX.%2BM.%26aulast%3DStubbs%26aufirst%3DM.%2BC.%26aulast%3DMaduskuie%26aufirst%3DT.%26aulast%3DSparks%26aufirst%3DR.%26aulast%3DZolotarjova%26aufirst%3DN.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DWen%26aufirst%3DX.%26aulast%3DFavata%26aufirst%3DM.%26aulast%3DFeldman%26aufirst%3DP.%26aulast%3DVolgina%26aufirst%3DA.%26aulast%3DDiMatteo%26aufirst%3DD.%26aulast%3DCollins%26aufirst%3DR.%26aulast%3DFalahatpisheh%26aufirst%3DN.%26aulast%3DPolam%26aufirst%3DP.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DCovington%26aufirst%3DM.%26aulast%3DDiamond-Fosbenner%26aufirst%3DS.%26aulast%3DWynn%26aufirst%3DR.%26aulast%3DBurn%26aufirst%3DT.%26aulast%3DVaddi%26aufirst%3DK.%26aulast%3DYeleswaram%26aufirst%3DS.%26aulast%3DCombs%26aufirst%3DA.%2BP.%26aulast%3DYao%26aufirst%3DW.%26aulast%3DHuber%26aufirst%3DR.%26aulast%3DScherle%26aufirst%3DP.%26aulast%3DHollis%26aufirst%3DG.%26atitle%3DAbstract%25203523%253A%2520Discovery%2520of%2520a%2520novel%2520BET%2520inhibitor%2520INCB054329%26jtitle%3DCancer%2520Res.%26date%3D2015%26volume%3D75%26spage%3D3523%26epage%3D3523%26doi%3D10.1158%2F1538-7445.AM2015-3523" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Attwell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campeau, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jahagirdar, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kharenko, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norek, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsujikawa, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calosing, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gesner, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lakhotia, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, H.</span></span> <span> </span><span class="NLM_article-title">Abstract C86: The clinical candidate ZEN-3694, a novel BET bromodomain inhibitor, is efficacious in the treatment of a variety of solid tumor and hematological malignancies, alone or in combination with several standard of care and targeted therapies</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">C86</span>– <span class="NLM_lpage">86</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.TARG-15-C86</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1158%2F1535-7163.TARG-15-C86" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2015&pages=C86-86&author=S.+Attwellauthor=E.+Campeauauthor=R.+Jahagirdarauthor=O.+Kharenkoauthor=K.+Norekauthor=L.+Tsujikawaauthor=C.+Calosingauthor=R.+Patelauthor=E.+Gesnerauthor=S.+Lakhotiaauthor=H.+Hansen&title=Abstract+C86%3A+The+clinical+candidate+ZEN-3694%2C+a+novel+BET+bromodomain+inhibitor%2C+is+efficacious+in+the+treatment+of+a+variety+of+solid+tumor+and+hematological+malignancies%2C+alone+or+in+combination+with+several+standard+of+care+and+targeted+therapies&doi=10.1158%2F1535-7163.TARG-15-C86"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.TARG-15-C86&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.TARG-15-C86%26sid%3Dliteratum%253Aachs%26aulast%3DAttwell%26aufirst%3DS.%26aulast%3DCampeau%26aufirst%3DE.%26aulast%3DJahagirdar%26aufirst%3DR.%26aulast%3DKharenko%26aufirst%3DO.%26aulast%3DNorek%26aufirst%3DK.%26aulast%3DTsujikawa%26aufirst%3DL.%26aulast%3DCalosing%26aufirst%3DC.%26aulast%3DPatel%26aufirst%3DR.%26aulast%3DGesner%26aufirst%3DE.%26aulast%3DLakhotia%26aufirst%3DS.%26aulast%3DHansen%26aufirst%3DH.%26atitle%3DAbstract%2520C86%253A%2520The%2520clinical%2520candidate%2520ZEN-3694%252C%2520a%2520novel%2520BET%2520bromodomain%2520inhibitor%252C%2520is%2520efficacious%2520in%2520the%2520treatment%2520of%2520a%2520variety%2520of%2520solid%2520tumor%2520and%2520hematological%2520malignancies%252C%2520alone%2520or%2520in%2520combination%2520with%2520several%2520standard%2520of%2520care%2520and%2520targeted%2520therapies%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2015%26volume%3D14%26spage%3DC86%26epage%3D86%26doi%3D10.1158%2F1535-7163.TARG-15-C86" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bates, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kusam, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tannheimer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breckenridge, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tumas, D.</span></span> <span> </span><span class="NLM_article-title">The Combination of a BET Inhibitor (GS-5829) and a BTK Inhibitor (GS-4059) Potentiates DLBCL Cell Line Cell Death and Reduces Expression of MYC, IL-10, and IL-6 in Vitro</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>128</i></span>,  <span class="NLM_fpage">5116</span>– <span class="NLM_lpage">5116</span>, <span class="refDoi"> DOI: 10.1182/blood.V128.22.5116.5116</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1182%2Fblood.V128.22.5116.5116" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2016&pages=5116-5116&author=J.+Batesauthor=S.+Kusamauthor=S.+Tannheimerauthor=J.+Chanauthor=Y.+Liauthor=D.+Breckenridgeauthor=D.+Tumas&title=The+Combination+of+a+BET+Inhibitor+%28GS-5829%29+and+a+BTK+Inhibitor+%28GS-4059%29+Potentiates+DLBCL+Cell+Line+Cell+Death+and+Reduces+Expression+of+MYC%2C+IL-10%2C+and+IL-6+in+Vitro&doi=10.1182%2Fblood.V128.22.5116.5116"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1182%2Fblood.V128.22.5116.5116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood.V128.22.5116.5116%26sid%3Dliteratum%253Aachs%26aulast%3DBates%26aufirst%3DJ.%26aulast%3DKusam%26aufirst%3DS.%26aulast%3DTannheimer%26aufirst%3DS.%26aulast%3DChan%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DBreckenridge%26aufirst%3DD.%26aulast%3DTumas%26aufirst%3DD.%26atitle%3DThe%2520Combination%2520of%2520a%2520BET%2520Inhibitor%2520%2528GS-5829%2529%2520and%2520a%2520BTK%2520Inhibitor%2520%2528GS-4059%2529%2520Potentiates%2520DLBCL%2520Cell%2520Line%2520Cell%2520Death%2520and%2520Reduces%2520Expression%2520of%2520MYC%252C%2520IL-10%252C%2520and%2520IL-6%2520in%2520Vitro%26jtitle%3DBlood%26date%3D2016%26volume%3D128%26spage%3D5116%26epage%3D5116%26doi%3D10.1182%2Fblood.V128.22.5116.5116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ran, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meagher, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaswamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Design of γ-Carboline Analogues as Potent and Specific BET Bromodomain Inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">4927</span>– <span class="NLM_lpage">4939</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00613</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00613" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVagu7rM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=4927-4939&author=X.+Ranauthor=Y.+Zhaoauthor=L.+Liuauthor=L.+Baiauthor=C.-Y.+Yangauthor=B.+Zhouauthor=J.+L.+Meagherauthor=K.+Chinnaswamyauthor=J.+A.+Stuckeyauthor=S.+Wang&title=Structure-Based+Design+of+%CE%B3-Carboline+Analogues+as+Potent+and+Specific+BET+Bromodomain+Inhibitors&doi=10.1021%2Facs.jmedchem.5b00613"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design of γ-Carboline Analogues as Potent and Specific BET Bromodomain Inhibitors</span></div><div class="casAuthors">Ran, Xu; Zhao, Yujun; Liu, Liu; Bai, Longchuan; Yang, Chao-Yie; Zhou, Bing; Meagher, Jennifer L.; Chinnaswamy, Krishnapriya; Stuckey, Jeanne A.; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4927-4939</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Small-mol. inhibitors of bromodomain and extra terminal proteins (BET), including BRD2, BRD3, and BRD4 proteins have therapeutic potential for the treatment of human cancers and other diseases and conditions.  In this paper, the authors report the design, synthesis, and evaluation of γ-carboline-contg. compds. as a new class of small-mol. BET inhibitors.  The most potent inhibitor I (RX-37) obtained from this study binds to BET bromodomain proteins (BRD2, BRD3, and BRD4) with Ki values of 3.2-24.7 nM and demonstrates high selectivity over other non-BET bromodomain-contg. proteins.  Compd. I potently and selectively inhibits cell growth in human acute leukemia cell lines harboring the rearranged mixed lineage leukemia 1 gene.  The authors have detd. a cocrystal structure of I in complex with BRD4 BD2 at 1.4 Å resoln., which provides a solid structural basis for the compd.'s high binding affinity and for its further structure-based optimization.  Compd. I represents a promising lead compd. for the development of a new class of therapeutics for the treatment of human cancer and other conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv-VCfWLBWYLVg90H21EOLACvtfcHk0lgqP--ul8X-VQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVagu7rM&md5=e6e150169812e439c1d7d91416eb3ee9</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00613&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00613%26sid%3Dliteratum%253Aachs%26aulast%3DRan%26aufirst%3DX.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DBai%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DC.-Y.%26aulast%3DZhou%26aufirst%3DB.%26aulast%3DMeagher%26aufirst%3DJ.%2BL.%26aulast%3DChinnaswamy%26aufirst%3DK.%26aulast%3DStuckey%26aufirst%3DJ.%2BA.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DStructure-Based%2520Design%2520of%2520%25CE%25B3-Carboline%2520Analogues%2520as%2520Potent%2520and%2520Specific%2520BET%2520Bromodomain%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D4927%26epage%3D4939%26doi%3D10.1021%2Facs.jmedchem.5b00613" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ayoub, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hawk, L. M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herzig, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wisniewski, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gee, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, J.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berndt, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Offei-Addo, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schönbrunn, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Georg, G. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pomerantz, W. C. K.</span></span> <span> </span><span class="NLM_article-title">BET Bromodomain Inhibitors with One-Step Synthesis Discovered from Virtual Screen</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">4805</span>– <span class="NLM_lpage">4817</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01336</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01336" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC2sXot1ylsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=4805-4817&author=A.+M.+Ayoubauthor=L.+M.+L.+Hawkauthor=R.+J.+Herzigauthor=J.+Jiangauthor=A.+J.+Wisniewskiauthor=C.+T.+Geeauthor=P.+Zhaoauthor=J.-Y.+Zhuauthor=N.+Berndtauthor=N.+K.+Offei-Addoauthor=T.+G.+Scottauthor=J.+Qiauthor=J.+E.+Bradnerauthor=T.+R.+Wardauthor=E.+Sch%C3%B6nbrunnauthor=G.+I.+Georgauthor=W.+C.+K.+Pomerantz&title=BET+Bromodomain+Inhibitors+with+One-Step+Synthesis+Discovered+from+Virtual+Screen&doi=10.1021%2Facs.jmedchem.6b01336"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">BET Bromodomain Inhibitors with One-Step Synthesis Discovered from Virtual Screen</span></div><div class="casAuthors">Ayoub, Alex M.; Hawk, Laura M. L.; Herzig, Ryan J.; Jiang, Jiewei; Wisniewski, Andrea J.; Gee, Clifford T.; Zhao, Peiliang; Zhu, Jin-Yi; Berndt, Norbert; Offei-Addo, Nana K.; Scott, Thomas G.; Qi, Jun; Bradner, James E.; Ward, Timothy R.; Schonbrunn, Ernst; Georg, Gunda I.; Pomerantz, William C. K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4805-4817</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Chem. inhibition of epigenetic regulatory proteins BrdT and Brd4 is emerging as a promising therapeutic strategy in contraception, cancer, and heart disease.  We report an easily synthesized dihydropyridopyrimidine pan-BET inhibitor scaffold, which was uncovered via a virtual screen followed by testing in a fluorescence anisotropy assay.  Dihydropyridopyimidine 3 (III) was subjected to further characterization and is highly selective for the BET family of bromodomains.  Structure-activity relationship data and ligand deconstruction highlight the importance of the substitution of the uracil moiety for potency and selectivity.  Compd. 3 was also cocrystd. with Brd4 for detg. the ligand binding pose and rationalizing subsequent structure-activity data.  An addnl. series of dihydropyridopyrimidines was synthesized to exploit the proximity of a channel near the ZA loop of Brd4, leading to compds. with submicromolar affinity and cellular target engagement.  Given these findings, novel and easily synthesized inhibitors are being introduced to the growing field of bromodomain inhibitor development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpr1keKj3fUOLVg90H21EOLACvtfcHk0lgqP--ul8X-VQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXot1ylsLo%253D&md5=2d537cb4dea3e8155688f183fca5612a</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01336%26sid%3Dliteratum%253Aachs%26aulast%3DAyoub%26aufirst%3DA.%2BM.%26aulast%3DHawk%26aufirst%3DL.%2BM.%2BL.%26aulast%3DHerzig%26aufirst%3DR.%2BJ.%26aulast%3DJiang%26aufirst%3DJ.%26aulast%3DWisniewski%26aufirst%3DA.%2BJ.%26aulast%3DGee%26aufirst%3DC.%2BT.%26aulast%3DZhao%26aufirst%3DP.%26aulast%3DZhu%26aufirst%3DJ.-Y.%26aulast%3DBerndt%26aufirst%3DN.%26aulast%3DOffei-Addo%26aufirst%3DN.%2BK.%26aulast%3DScott%26aufirst%3DT.%2BG.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DWard%26aufirst%3DT.%2BR.%26aulast%3DSch%25C3%25B6nbrunn%26aufirst%3DE.%26aulast%3DGeorg%26aufirst%3DG.%2BI.%26aulast%3DPomerantz%26aufirst%3DW.%2BC.%2BK.%26atitle%3DBET%2520Bromodomain%2520Inhibitors%2520with%2520One-Step%2520Synthesis%2520Discovered%2520from%2520Virtual%2520Screen%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D4805%26epage%3D4817%26doi%3D10.1021%2Facs.jmedchem.6b01336" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaudio, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonetti, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwee, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernasconi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarantelli, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rinaldi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Testoni, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cascione, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ponzoni, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mensah, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stathis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stussi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riveiro, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herait, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inghirami, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cvitkovic, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zucca, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertoni, F.</span></span> <span> </span><span class="NLM_article-title">The BET Bromodomain Inhibitor OTX015 Affects Pathogenetic Pathways in Preclinical B-cell Tumor Models and Synergizes with Targeted Drugs</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">1628</span>– <span class="NLM_lpage">1638</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.CCR-14-1561</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1158%2F1078-0432.CCR-14-1561" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=25623213" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmtVahsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=1628-1638&author=M.+Boiauthor=E.+Gaudioauthor=P.+Bonettiauthor=I.+Kweeauthor=E.+Bernasconiauthor=C.+Tarantelliauthor=A.+Rinaldiauthor=M.+Testoniauthor=L.+Cascioneauthor=M.+Ponzoniauthor=A.+A.+Mensahauthor=A.+Stathisauthor=G.+Stussiauthor=M.+E.+Riveiroauthor=P.+Heraitauthor=G.+Inghiramiauthor=E.+Cvitkovicauthor=E.+Zuccaauthor=F.+Bertoni&title=The+BET+Bromodomain+Inhibitor+OTX015+Affects+Pathogenetic+Pathways+in+Preclinical+B-cell+Tumor+Models+and+Synergizes+with+Targeted+Drugs&doi=10.1158%2F1078-0432.CCR-14-1561"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">The BET Bromodomain Inhibitor OTX015 Affects Pathogenetic Pathways in Preclinical B-cell Tumor Models and Synergizes with Targeted Drugs</span></div><div class="casAuthors">Boi, Michela; Gaudio, Eugenio; Bonetti, Paola; Kwee, Ivo; Bernasconi, Elena; Tarantelli, Chiara; Rinaldi, Andrea; Testoni, Monica; Cascione, Luciano; Ponzoni, Maurilio; Mensah, Afua Adjeiwaa; Stathis, Anastasios; Stussi, Georg; Riveiro, Maria Eugenia; Herait, Patrice; Inghirami, Giorgio; Cvitkovic, Esteban; Zucca, Emanuele; Bertoni, Francesco</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1628-1638</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">In cancer cells, the epigenome is often deregulated, and inhibition of the bromodomain and extra-terminal (BET) family of bromodomain-contg. proteins is a novel epigenetic therapeutic approach.  Preliminary results of an ongoing phase I trial have reported promising activity and tolerability with the new BET bromodomain inhibitor OTX015.  We assessed the preclin. activity of OTX015 as single agent and in combination in mature B-cell lymphoma models and performed in vitro and in vivo expts. to identify the mechanism of action and the genetic features assocd. with sensitivity to the compd.  OTX015 showed antiproliferative activity in a large panel of cell lines derived from mature B-cell lymphoid tumors with median IC50 of 240 nmol/L, without significant differences among the different histotypes.  In vitro and in vivo expts. showed that OTX015 targeted NFKB/TLR/JAK/STAT signaling pathways, MYC- and E2F1-regulated genes, cell-cycle regulation, and chromatin structure.  OTX015 presented in vitro synergism with several anticancer agents, esp. with mTOR and BTK inhibitors.  Gene expression signatures assocd. with different degrees of sensitivity to OTX015 were identified.  Although OTX015 was mostly cytostatic, the compd. induced apoptosis in a genetically defined subgroup of cells, derived from activated B-cell-like diffuse large B-cell lymphoma, bearing wtTP53, mutations in MYD88, and CD79B or CARD11.  Together with the data coming from the ongoing phase I study, the in vitro and in vivo data presented here provide the basis for further clin. investigation of OTX015 as single agent and in combination therapies.  Clin Cancer Res; 21(7); 1628-38. ©2015 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJcy3O1ElDJ7Vg90H21EOLACvtfcHk0lj7n4pgeBEuIQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmtVahsbc%253D&md5=5742b4d5884261d546588a339f3c2ea0</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-14-1561&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-14-1561%26sid%3Dliteratum%253Aachs%26aulast%3DBoi%26aufirst%3DM.%26aulast%3DGaudio%26aufirst%3DE.%26aulast%3DBonetti%26aufirst%3DP.%26aulast%3DKwee%26aufirst%3DI.%26aulast%3DBernasconi%26aufirst%3DE.%26aulast%3DTarantelli%26aufirst%3DC.%26aulast%3DRinaldi%26aufirst%3DA.%26aulast%3DTestoni%26aufirst%3DM.%26aulast%3DCascione%26aufirst%3DL.%26aulast%3DPonzoni%26aufirst%3DM.%26aulast%3DMensah%26aufirst%3DA.%2BA.%26aulast%3DStathis%26aufirst%3DA.%26aulast%3DStussi%26aufirst%3DG.%26aulast%3DRiveiro%26aufirst%3DM.%2BE.%26aulast%3DHerait%26aufirst%3DP.%26aulast%3DInghirami%26aufirst%3DG.%26aulast%3DCvitkovic%26aufirst%3DE.%26aulast%3DZucca%26aufirst%3DE.%26aulast%3DBertoni%26aufirst%3DF.%26atitle%3DThe%2520BET%2520Bromodomain%2520Inhibitor%2520OTX015%2520Affects%2520Pathogenetic%2520Pathways%2520in%2520Preclinical%2520B-cell%2520Tumor%2520Models%2520and%2520Synergizes%2520with%2520Targeted%2520Drugs%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2015%26volume%3D21%26spage%3D1628%26epage%3D1638%26doi%3D10.1158%2F1078-0432.CCR-14-1561" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miyoshi, S.</span>; <span class="NLM_string-name">Ooike, S.</span>; <span class="NLM_string-name">Iwata, K.</span>; <span class="NLM_string-name">Hikawa, H.</span>; <span class="NLM_string-name">Sugahara, K.</span></span> <span> </span><span class="NLM_article-title">Antitumor agent</span>. <span class="NLM_patent">WO 2009084693</span>, <span class="NLM_year">2009</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2009&author=S.+Miyoshi&author=S.+Ooike&author=K.+Iwata&author=H.+Hikawa&author=K.+Sugahara&title=Antitumor+agent"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DMiyoshi%26aufirst%3DS.%26atitle%3DAntitumor%2520agent%26date%3D2009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Doroshow, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eder, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LoRusso, P. M.</span></span> <span> </span><span class="NLM_article-title">BET inhibitors: a novel epigenetic approach</span>. <i>Ann. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">1776</span>– <span class="NLM_lpage">1787</span>, <span class="refDoi"> DOI: 10.1093/annonc/mdx157</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1093%2Fannonc%2Fmdx157" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=28838216" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A280%3ADC%252BC1cbhtlOisw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2017&pages=1776-1787&author=D.+B.+Doroshowauthor=J.+P.+Ederauthor=P.+M.+LoRusso&title=BET+inhibitors%3A+a+novel+epigenetic+approach&doi=10.1093%2Fannonc%2Fmdx157"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">BET inhibitors: a novel epigenetic approach</span></div><div class="casAuthors">Doroshow D B; Eder J P; LoRusso P M</div><div class="citationInfo"><span class="NLM_cas:title">Annals of oncology : official journal of the European Society for Medical Oncology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1776-1787</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Epigenetics has been defined as 'the structural adaptation of chromosomal regions so as to register, signal or perpetuate altered activity states.' Currently, several classes of anticancer drugs function at the epigenetic level, including inhibitors of DNA methyltransferase, histone deacetylase (HDAC), lysine-specific demethylase 1, zeste homolog 2, and bromodomain and extra-terminal motif (BET) proteins.BET proteins have multiple functions, including the initiation and elongation of transcription and cell cycle regulation.  In recent years, inhibitors of BET proteins have been developed as anticancer agents.  These inhibitors exhibit selectivity for tumor cells by preferentially binding to superenhancers, noncoding regions of DNA critical for the transcription of genes that determine a cell's identity.  Preclinical research on BET inhibitors has identified them as a potential means of targeting MYC.Early clinical trials with BET inhibitors have had mixed results, with few responses in both hematologic and solid tumors that tend to be short-lived.  Toxicities have included severe, thrombocytopenia, fatigue, nausea, vomiting, and diarrhea; GI side-effects, fatigue, and low-grade dysgeusia have limited compliance.  However, preclinical data suggest that BET inhibitors may have a promising future in combination with other agents.  They appear to be able to overcome resistance to targeted agents and have strong synergy with immune checkpoint inhibitors as well as with multiple epigenetic agents, particularly HDAC inhibitors.  In many instances, BET and HDAC inhibitors were synergistic at reduced doses, suggesting a potential means of avoiding the overlapping toxicities of the two drug classes.BET inhibitors provide a novel approach to epigenetic anticancer therapy.  However, to date they appear to have limited efficacy as single agents.  A focus on BET inhibitors in combination with other drugs such as targeted and/or as other epigenetic agents is warranted, due to limited monotherapy activity, including pharmacodynamic correlatives differential activity amongst select drug combinations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSttah97bhI2UPI-fCgwosgfW6udTcc2eZTl9a4nsp15bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cbhtlOisw%253D%253D&md5=580ac90cf48f18db8313471ed61af173</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1093%2Fannonc%2Fmdx157&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fannonc%252Fmdx157%26sid%3Dliteratum%253Aachs%26aulast%3DDoroshow%26aufirst%3DD.%2BB.%26aulast%3DEder%26aufirst%3DJ.%2BP.%26aulast%3DLoRusso%26aufirst%3DP.%2BM.%26atitle%3DBET%2520inhibitors%253A%2520a%2520novel%2520epigenetic%2520approach%26jtitle%3DAnn.%2520Oncol.%26date%3D2017%26volume%3D28%26spage%3D1776%26epage%3D1787%26doi%3D10.1093%2Fannonc%2Fmdx157" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vidler, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomsett, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodward, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoelder, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of novel small-molecule inhibitors of BRD4 using structure-based virtual screening</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">8073</span>– <span class="NLM_lpage">8088</span>, <span class="refDoi"> DOI: 10.1021/jm4011302</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm4011302" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFOjurrI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=8073-8088&author=L.+R.+Vidlerauthor=P.+Filippakopoulosauthor=O.+Fedorovauthor=S.+Picaudauthor=S.+Martinauthor=M.+Tomsettauthor=H.+Woodwardauthor=N.+Brownauthor=S.+Knappauthor=S.+Hoelder&title=Discovery+of+novel+small-molecule+inhibitors+of+BRD4+using+structure-based+virtual+screening&doi=10.1021%2Fjm4011302"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Novel Small-Molecule Inhibitors of BRD4 Using Structure-Based Virtual Screening</span></div><div class="casAuthors">Vidler, Lewis R.; Filippakopoulos, Panagis; Fedorov, Oleg; Picaud, Sarah; Martin, Sarah; Tomsett, Michael; Woodward, Hannah; Brown, Nathan; Knapp, Stefan; Hoelder, Swen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">8073-8088</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomains (BRDs) are epigenetic readers that recognize acetylated-lysine (KAc) on proteins and are implicated in a no. of diseases.  We describe a virtual screening approach to identify BRD inhibitors.  Key elements of this approach are the extensive design and use of substructure queries to compile a set of com. available compds. featuring novel putative KAc mimetics and docking this set for final compd. selection.  We describe the validation of this approach by applying it to the first BRD of BRD4.  The selection and testing of 143 compds. lead to the discovery of six novel hits, including four unprecedented KAc mimetics.  We solved the crystal structure of four hits, detd. their binding mode, and improved their potency through synthesis and the purchase of derivs.  This work provides a validated virtual screening approach that is applicable to other BRDs and describes novel KAc mimetics that can be further explored to design more potent inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGow5yj8bnohUbVg90H21EOLACvtfcHk0lj7n4pgeBEuIQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFOjurrI&md5=c1f76e4838713335be7a93ebe4c68701</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Fjm4011302&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm4011302%26sid%3Dliteratum%253Aachs%26aulast%3DVidler%26aufirst%3DL.%2BR.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DTomsett%26aufirst%3DM.%26aulast%3DWoodward%26aufirst%3DH.%26aulast%3DBrown%26aufirst%3DN.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DHoelder%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520novel%2520small-molecule%2520inhibitors%2520of%2520BRD4%2520using%2520structure-based%2520virtual%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D8073%26epage%3D8088%26doi%3D10.1021%2Fjm4011302" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hasvold, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fidanze, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pratt, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mantei, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wada, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warder, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilcox, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buchanan, F. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smithee, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albert, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magoc, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petros, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panchal, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kovar, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soni, N. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elmore, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kati, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDaniel, K. F.</span></span> <span> </span><span class="NLM_article-title">Methylpyrrole inhibitors of BET bromodomains</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">2225</span>– <span class="NLM_lpage">2233</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2017.02.057</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1016%2Fj.bmcl.2017.02.057" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=28268136" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjsl2gsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2017&pages=2225-2233&author=L.+A.+Hasvoldauthor=G.+S.+Sheppardauthor=L.+Wangauthor=S.+D.+Fidanzeauthor=D.+Liuauthor=J.+K.+Prattauthor=R.+A.+Manteiauthor=C.+K.+Wadaauthor=R.+Hubbardauthor=Y.+Shenauthor=X.+Linauthor=X.+Huangauthor=S.+E.+Warderauthor=D.+Wilcoxauthor=L.+Liauthor=F.+G.+Buchananauthor=L.+Smitheeauthor=D.+H.+Albertauthor=T.+J.+Magocauthor=C.+H.+Parkauthor=A.+M.+Petrosauthor=S.+C.+Panchalauthor=C.+Sunauthor=P.+Kovarauthor=N.+B.+Soniauthor=S.+W.+Elmoreauthor=W.+M.+Katiauthor=K.+F.+McDaniel&title=Methylpyrrole+inhibitors+of+BET+bromodomains&doi=10.1016%2Fj.bmcl.2017.02.057"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Methylpyrrole inhibitors of BET bromodomains</span></div><div class="casAuthors">Hasvold, Lisa A.; Sheppard, George S.; Wang, Le; Fidanze, Steven D.; Liu, Dachun; Pratt, John K.; Mantei, Robert A.; Wada, Carol K.; Hubbard, Robbert; Shen, Yu; Lin, Xiaoyu; Huang, Xiaoli; Warder, Scott E.; Wilcox, Denise; Li, Leiming; Buchanan, F. Greg; Smithee, Lauren; Albert, Daniel H.; Magoc, Terrance J.; Park, Chang H.; Petros, Andrew M.; Panchal, Sanjay C.; Sun, Chaohong; Kovar, Peter; Soni, Nirupama B.; Elmore, Steven W.; Kati, Warren M.; McDaniel, Keith F.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2225-2233</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">An NMR fragment screen for binders to the bromodomains of BRD4 identified 2-methyl-3-ketopyrroles (I) and (II).  Elaboration of these fragments guided by structure-based design provided lead mols. with significant activity in a mouse tumor model.  Further modifications to the methylpyrrole core provided compds. with improved properties and enhanced activity in a mouse model of multiple myeloma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9iHIAhR06LrVg90H21EOLACvtfcHk0li6nUojJRSRNw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjsl2gsLc%253D&md5=80d50b9ce910e4ed8233e31e463456fd</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2017.02.057&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2017.02.057%26sid%3Dliteratum%253Aachs%26aulast%3DHasvold%26aufirst%3DL.%2BA.%26aulast%3DSheppard%26aufirst%3DG.%2BS.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DFidanze%26aufirst%3DS.%2BD.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DPratt%26aufirst%3DJ.%2BK.%26aulast%3DMantei%26aufirst%3DR.%2BA.%26aulast%3DWada%26aufirst%3DC.%2BK.%26aulast%3DHubbard%26aufirst%3DR.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DWarder%26aufirst%3DS.%2BE.%26aulast%3DWilcox%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DBuchanan%26aufirst%3DF.%2BG.%26aulast%3DSmithee%26aufirst%3DL.%26aulast%3DAlbert%26aufirst%3DD.%2BH.%26aulast%3DMagoc%26aufirst%3DT.%2BJ.%26aulast%3DPark%26aufirst%3DC.%2BH.%26aulast%3DPetros%26aufirst%3DA.%2BM.%26aulast%3DPanchal%26aufirst%3DS.%2BC.%26aulast%3DSun%26aufirst%3DC.%26aulast%3DKovar%26aufirst%3DP.%26aulast%3DSoni%26aufirst%3DN.%2BB.%26aulast%3DElmore%26aufirst%3DS.%2BW.%26aulast%3DKati%26aufirst%3DW.%2BM.%26aulast%3DMcDaniel%26aufirst%3DK.%2BF.%26atitle%3DMethylpyrrole%2520inhibitors%2520of%2520BET%2520bromodomains%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D27%26spage%3D2225%26epage%3D2233%26doi%3D10.1016%2Fj.bmcl.2017.02.057" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gallenkamp, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gelato, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haendler, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinmann, H.</span></span> <span> </span><span class="NLM_article-title">Bromodomains and their pharmacological inhibitors</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">438</span>– <span class="NLM_lpage">464</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201300434</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1002%2Fcmdc.201300434" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=24497428" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsl2gtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=438-464&author=D.+Gallenkampauthor=K.+A.+Gelatoauthor=B.+Haendlerauthor=H.+Weinmann&title=Bromodomains+and+their+pharmacological+inhibitors&doi=10.1002%2Fcmdc.201300434"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomains and Their Pharmacological Inhibitors</span></div><div class="casAuthors">Gallenkamp, Daniel; Gelato, Kathy A.; Haendler, Bernard; Weinmann, Hilmar</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">438-464</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Over 60 bromodomains belonging to proteins with very different functions have been identified in humans.  Several of them interact with acetylated lysine residues, leading to the recruitment and stabilization of protein complexes.  The bromodomain and extra-terminal domain (BET) proteins contain tandem bromodomains which bind to acetylated histones and are thereby implicated in a no. of DNA-centered processes, including the regulation of gene expression.  The recent identification of inhibitors of BET and non-BET bromodomains is one of the few examples in which effective blockade of a protein-protein interaction can be achieved with a small mol.  This has led to major strides in the understanding of the function of bromodomain-contg. proteins and their involvement in diseases such as cancer and inflammation.  Indeed, BET bromodomain inhibitors are now being clin. evaluated for the treatment of hematol. tumors and have also been tested in clin. trials for the relatively rare BRD-NUT midline carcinoma.  This review gives an overview of the newest developments in the field, with a focus on the biol. of selected bromodomain proteins on the one hand, and on reported pharmacol. inhibitors on the other, including recent examples from the patent literature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGon-8_ymytCPbVg90H21EOLACvtfcHk0li6nUojJRSRNw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsl2gtL4%253D&md5=2ce7c6b5c37001fb8fb901fe04aeb606</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201300434&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201300434%26sid%3Dliteratum%253Aachs%26aulast%3DGallenkamp%26aufirst%3DD.%26aulast%3DGelato%26aufirst%3DK.%2BA.%26aulast%3DHaendler%26aufirst%3DB.%26aulast%3DWeinmann%26aufirst%3DH.%26atitle%3DBromodomains%2520and%2520their%2520pharmacological%2520inhibitors%26jtitle%3DChemMedChem%26date%3D2014%26volume%3D9%26spage%3D438%26epage%3D464%26doi%3D10.1002%2Fcmdc.201300434" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leonards, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambert, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dovey, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monteiro, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujisawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lingard, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikbin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guetzoyan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ingham, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ley, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samsonova, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gingras, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwaller, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vassiliou, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span> <span> </span><span class="NLM_article-title">Promiscuous targeting of bromodomains by bromosporine identifies BET proteins as master regulators of primary transcription response in leukemia</span>. <i>Sci. Adv.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>2</i></span>, <span class="NLM_elocation-id">e1600760</span> <span class="refDoi"> DOI: 10.1126/sciadv.1600760</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1126%2Fsciadv.1600760" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=27757418" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC1cXkvFyjtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2016&author=S.+Picaudauthor=K.+Leonardsauthor=J.+P.+Lambertauthor=O.+Doveyauthor=C.+Wellsauthor=O.+Fedorovauthor=O.+Monteiroauthor=T.+Fujisawaauthor=C.+Y.+Wangauthor=H.+Lingardauthor=C.+Tallantauthor=N.+Nikbinauthor=L.+Guetzoyanauthor=R.+Inghamauthor=S.+V.+Leyauthor=P.+Brennanauthor=S.+Mullerauthor=A.+Samsonovaauthor=A.+C.+Gingrasauthor=J.+Schwallerauthor=G.+Vassiliouauthor=S.+Knappauthor=P.+Filippakopoulos&title=Promiscuous+targeting+of+bromodomains+by+bromosporine+identifies+BET+proteins+as+master+regulators+of+primary+transcription+response+in+leukemia&doi=10.1126%2Fsciadv.1600760"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Promiscuous targeting of bromodomains by bromosporine identifies BET proteins as master regulators of primary transcription response in leukemia</span></div><div class="casAuthors">Picaud, Sarah; Leonards, Katharina; Lambert, Jean-Philippe; Dovey, Oliver; Wells, Christopher; Fedorov, Oleg; Monteiro, Octovia; Fujisawa, Takao; Wang, Chen-Yi; Lingard, Hannah; Tallant, Cynthia; Nikbin, Nikzad; Guetzoyan, Lucie; Ingham, Richard; Ley, Steven V.; Brennan, Paul; Muller, Susanne; Samsonova, Anastasia; Gingras, Anne-Claude; Schwaller, Juerg; Vassiliou, George; Knapp, Stefan; Filippakopoulos, Panagis</div><div class="citationInfo"><span class="NLM_cas:title">Science Advances</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">e1600760/1-e1600760/16</span>CODEN:
                <span class="NLM_cas:coden">SACDAF</span>;
        ISSN:<span class="NLM_cas:issn">2375-2548</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Bromodomains (BRDs) have emerged as compelling targets for cancer therapy.  The development of selective and potent BET (bromo and extra-terminal) inhibitors and their significant activity in diverse tumor models have rapidly translated into clin. studies and have motivated drug development efforts targeting non-BET BRDs.  However, the complexmultidomain/subunit architecture of BRD protein complexes complicates predictions of the consequences of their pharmacol. targeting.  To address this issue, we developed a promiscuous BRD inhibitor [bromosporine (BSP)] that broadly targets BRDs (including BETs) with nanomolar affinity, creating a tool for the identification of cellular processes and diseases where BRDs have a regulatory function.  As a proof of principle, we studied the effects of BSP on leukemic cell lines known to be sensitive to BET inhibition and found, as expected, strong antiproliferative activity.  Comparison of the modulation of transcriptional profiles by BSP after a short exposure to the inhibitor resulted in a BET inhibitor signature but no significant addnl. changes in transcription that could account for inhibition of other BRDs.  Thus, nonselective targeting of BRDs identified BETs, but not other BRDs, as master regulators of contextdependent primary transcription response.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr28artLYYsqrVg90H21EOLACvtfcHk0lheYXkiGPDMGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXkvFyjtbw%253D&md5=3060e8fd66c657f70e0ec3acea273057</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1126%2Fsciadv.1600760&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fsciadv.1600760%26sid%3Dliteratum%253Aachs%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DLeonards%26aufirst%3DK.%26aulast%3DLambert%26aufirst%3DJ.%2BP.%26aulast%3DDovey%26aufirst%3DO.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMonteiro%26aufirst%3DO.%26aulast%3DFujisawa%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DC.%2BY.%26aulast%3DLingard%26aufirst%3DH.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DNikbin%26aufirst%3DN.%26aulast%3DGuetzoyan%26aufirst%3DL.%26aulast%3DIngham%26aufirst%3DR.%26aulast%3DLey%26aufirst%3DS.%2BV.%26aulast%3DBrennan%26aufirst%3DP.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DSamsonova%26aufirst%3DA.%26aulast%3DGingras%26aufirst%3DA.%2BC.%26aulast%3DSchwaller%26aufirst%3DJ.%26aulast%3DVassiliou%26aufirst%3DG.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26atitle%3DPromiscuous%2520targeting%2520of%2520bromodomains%2520by%2520bromosporine%2520identifies%2520BET%2520proteins%2520as%2520master%2520regulators%2520of%2520primary%2520transcription%2520response%2520in%2520leukemia%26jtitle%3DSci.%2520Adv.%26date%3D2016%26volume%3D2%26doi%3D10.1126%2Fsciadv.1600760" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dittmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giotopoulos, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bantscheff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, W. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robson, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hopf, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savitski, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huthmacher, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gudgin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lugo, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beinke, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chapman, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soden, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Auger, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirguet, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doehner, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delwel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burnett, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeffrey, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewes, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huntly, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kouzarides, T.</span></span> <span> </span><span class="NLM_article-title">Inhibition of BET recruitment to chromatin as an effective treatment for MLL-fusion leukaemia</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>478</i></span>,  <span class="NLM_fpage">529</span>– <span class="NLM_lpage">533</span>, <span class="refDoi"> DOI: 10.1038/nature10509</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1038%2Fnature10509" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=21964340" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1GqsL%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=478&publication_year=2011&pages=529-533&author=M.+A.+Dawsonauthor=R.+K.+Prinjhaauthor=A.+Dittmannauthor=G.+Giotopoulosauthor=M.+Bantscheffauthor=W.+I.+Chanauthor=S.+C.+Robsonauthor=C.+W.+Chungauthor=C.+Hopfauthor=M.+M.+Savitskiauthor=C.+Huthmacherauthor=E.+Gudginauthor=D.+Lugoauthor=S.+Beinkeauthor=T.+D.+Chapmanauthor=E.+J.+Robertsauthor=P.+E.+Sodenauthor=K.+R.+Augerauthor=O.+Mirguetauthor=K.+Doehnerauthor=R.+Delwelauthor=A.+K.+Burnettauthor=P.+Jeffreyauthor=G.+Drewesauthor=K.+Leeauthor=B.+J.+Huntlyauthor=T.+Kouzarides&title=Inhibition+of+BET+recruitment+to+chromatin+as+an+effective+treatment+for+MLL-fusion+leukaemia&doi=10.1038%2Fnature10509"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of BET recruitment to chromatin as an effective treatment for MLL-fusion leukaemia</span></div><div class="casAuthors">Dawson, Mark A.; Prinjha, Rab K.; Dittman, Antje; Giotopoulos, George; Bantscheff, Marcus; Chan, Wai-In; Robson, Samuel C.; Chung, Chun-wa; Hopf, Carsten; Savitski, Mikhail M.; Huthmacher, Carola; Gudgin, Emma; Lugo, Dave; Beinke, Soren; Chapman, Trevor D.; Roberts, Emma J.; Soden, Peter E.; Auger, Kurt R.; Mirguet, Olivier; Doehner, Konstanze; Delwel, Ruud; Burnett, Alan K.; Jeffrey, Phillip; Drewes, Gerard; Lee, Kevin; Huntly, Brian J. P.; Kouzarides, Tony</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">478</span>
        (<span class="NLM_cas:issue">7370</span>),
    <span class="NLM_cas:pages">529-533</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Recurrent chromosomal translocations involving the mixed lineage leukemia (MLL) gene initiate aggressive forms of leukemia, which are often refractory to conventional therapies.  Many MLL-fusion partners are members of the super elongation complex (SEC), a crit. regulator of transcriptional elongation, suggesting that aberrant control of this process has an important role in leukemia induction.  Here we use a global proteomic strategy to demonstrate that MLL fusions, as part of SEC and the polymerase-assocd. factor complex (PAFc), are assocd. with the BET family of acetyl-lysine recognizing, chromatin adaptor' proteins.  These data provided the basis for therapeutic intervention in MLL-fusion leukemia, via the displacement of the BET family of proteins from chromatin.  We show that a novel small mol. inhibitor of the BET family, GSK1210151A (I-BET151), has profound efficacy against human and murine MLL-fusion leukemic cell lines, through the induction of early cell cycle arrest and apoptosis.  I-BET151 treatment in two human leukemia cell lines with different MLL fusions alters the expression of a common set of genes whose function may account for these phenotypic changes.  The mode of action of I-BET151 is, at least in part, due to the inhibition of transcription at key genes (BCL2, C-MYC and CDK6) through the displacement of BRD3/4, PAFc and SEC components from chromatin.  In vivo studies indicate that I-BET151 has significant therapeutic value, providing survival benefit in two distinct mouse models of murine MLL-AF9 and human MLL-AF4 leukemia.  Finally, the efficacy of I-BET151 against human leukemia stem cells is demonstrated, providing further evidence of its potent therapeutic potential.  These findings establish the displacement of BET proteins from chromatin as a promising epigenetic therapy for these aggressive leukemias.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4FbgJpjftv7Vg90H21EOLACvtfcHk0lheYXkiGPDMGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1GqsL%252FO&md5=dffb4b81da4ca8291c27139d0df86324</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1038%2Fnature10509&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature10509%26sid%3Dliteratum%253Aachs%26aulast%3DDawson%26aufirst%3DM.%2BA.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DDittmann%26aufirst%3DA.%26aulast%3DGiotopoulos%26aufirst%3DG.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DChan%26aufirst%3DW.%2BI.%26aulast%3DRobson%26aufirst%3DS.%2BC.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DHopf%26aufirst%3DC.%26aulast%3DSavitski%26aufirst%3DM.%2BM.%26aulast%3DHuthmacher%26aufirst%3DC.%26aulast%3DGudgin%26aufirst%3DE.%26aulast%3DLugo%26aufirst%3DD.%26aulast%3DBeinke%26aufirst%3DS.%26aulast%3DChapman%26aufirst%3DT.%2BD.%26aulast%3DRoberts%26aufirst%3DE.%2BJ.%26aulast%3DSoden%26aufirst%3DP.%2BE.%26aulast%3DAuger%26aufirst%3DK.%2BR.%26aulast%3DMirguet%26aufirst%3DO.%26aulast%3DDoehner%26aufirst%3DK.%26aulast%3DDelwel%26aufirst%3DR.%26aulast%3DBurnett%26aufirst%3DA.%2BK.%26aulast%3DJeffrey%26aufirst%3DP.%26aulast%3DDrewes%26aufirst%3DG.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DHuntly%26aufirst%3DB.%2BJ.%26aulast%3DKouzarides%26aufirst%3DT.%26atitle%3DInhibition%2520of%2520BET%2520recruitment%2520to%2520chromatin%2520as%2520an%2520effective%2520treatment%2520for%2520MLL-fusion%2520leukaemia%26jtitle%3DNature%26date%3D2011%26volume%3D478%26spage%3D529%26epage%3D533%26doi%3D10.1038%2Fnature10509" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ryckmans, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horne, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Correia, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owen, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tutt, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, T.</span></span> <span> </span><span class="NLM_article-title">Rapid assessment of a novel series of selective CB(2) agonists using parallel synthesis protocols: A Lipophilic Efficiency (LipE) analysis</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">4406</span>– <span class="NLM_lpage">4409</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2009.05.062</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1016%2Fj.bmcl.2009.05.062" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=19500981" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BD1MXosVGlsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=4406-4409&author=T.+Ryckmansauthor=M.+P.+Edwardsauthor=V.+A.+Horneauthor=A.+M.+Correiaauthor=D.+R.+Owenauthor=L.+R.+Thompsonauthor=I.+Tranauthor=M.+F.+Tuttauthor=T.+Young&title=Rapid+assessment+of+a+novel+series+of+selective+CB%282%29+agonists+using+parallel+synthesis+protocols%3A+A+Lipophilic+Efficiency+%28LipE%29+analysis&doi=10.1016%2Fj.bmcl.2009.05.062"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid assessment of a novel series of selective CB2 agonists using parallel synthesis protocols: A Lipophilic Efficiency (LipE) analysis</span></div><div class="casAuthors">Ryckmans, Thomas; Edwards, Martin P.; Horne, Val A.; Correia, Ana Monica; Owen, Dafydd R.; Thompson, Lisa R.; Tran, Isabelle; Tutt, Michelle F.; Young, Tim</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">4406-4409</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A series of libraries were designed using the 1-(cyclopropylmethyl)-2-alkyl-4,5,6,7-tetrahydro-1H-imidazo[4,5-c]pyridin-5-ium templates 2a-b, and Sulfonamide derivs. 11a-n proved to be potent agonists of the CB2 receptor.  Anal. of the Lipophilic Efficiency (LipE) of potent compds. provided new insight for the design of potent, metabolically stable CB2 agonists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1cihq9YHuO7Vg90H21EOLACvtfcHk0lheYXkiGPDMGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXosVGlsrg%253D&md5=b62d97e5f52a594cfdd510f32d2e566f</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2009.05.062&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2009.05.062%26sid%3Dliteratum%253Aachs%26aulast%3DRyckmans%26aufirst%3DT.%26aulast%3DEdwards%26aufirst%3DM.%2BP.%26aulast%3DHorne%26aufirst%3DV.%2BA.%26aulast%3DCorreia%26aufirst%3DA.%2BM.%26aulast%3DOwen%26aufirst%3DD.%2BR.%26aulast%3DThompson%26aufirst%3DL.%2BR.%26aulast%3DTran%26aufirst%3DI.%26aulast%3DTutt%26aufirst%3DM.%2BF.%26aulast%3DYoung%26aufirst%3DT.%26atitle%3DRapid%2520assessment%2520of%2520a%2520novel%2520series%2520of%2520selective%2520CB%25282%2529%2520agonists%2520using%2520parallel%2520synthesis%2520protocols%253A%2520A%2520Lipophilic%2520Efficiency%2520%2528LipE%2529%2520analysis%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2009%26volume%3D19%26spage%3D4406%26epage%3D4409%26doi%3D10.1016%2Fj.bmcl.2009.05.062" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, M. P.</span>; <span class="NLM_string-name">Price, D. A.</span></span> <span> </span><span class="NLM_article-title">Role of Physicochemical Properties and Ligand Lipophilicity Efficiency in Addressing Drug Safety Risks</span>. In  <i>Annual Reports in Medicinal Chemistry</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macor, J. E.</span></span>, Ed.; <span class="NLM_publisher-name">Academic Press</span>, <span class="NLM_year">2010</span>; Vol.  <span class="NLM_volume">45</span>, Chapter 23, pp  <span class="NLM_fpage">380</span>– <span class="NLM_lpage">391</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1016%2FS0065-7743%2810%2945023-X" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2010&pages=380-391&author=M.+P.+Edwards&author=D.+A.+Priceauthor=J.+E.+Macor&title=Annual+Reports+in+Medicinal+Chemistry"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1016%2FS0065-7743%2810%2945023-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1016%252FS0065-7743%252810%252945023-X%26sid%3Dliteratum%253Aachs%26aulast%3DEdwards%26aufirst%3DM.%2BP.%26atitle%3DRole%2520of%2520Physicochemical%2520Properties%2520and%2520Ligand%2520Lipophilicity%2520Efficiency%2520in%2520Addressing%2520Drug%2520Safety%2520Risks%26btitle%3DAnnual%2520Reports%2520in%2520Medicinal%2520Chemistry%26aulast%3DMacor%26aufirst%3DJ.%2BE.%26pub%3DAcademic%2520Press%26date%3D2010%26volume%3D45%26spage%3D380%26epage%3D391" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coste, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirguet, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilde, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gosmini, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delves, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magny, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodward, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boursier, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flynn, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouillot, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brusq, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gellibert, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riou, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Homes, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uings, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toum, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clement, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boullay, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimley, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blandel, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirilovsky, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicodeme, E.</span></span> <span> </span><span class="NLM_article-title">Discovery and characterization of small molecule inhibitors of the BET family bromodomains</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">3827</span>– <span class="NLM_lpage">3838</span>, <span class="refDoi"> DOI: 10.1021/jm200108t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm200108t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmtVWnurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=3827-3838&author=C.+W.+Chungauthor=H.+Costeauthor=J.+H.+Whiteauthor=O.+Mirguetauthor=J.+Wildeauthor=R.+L.+Gosminiauthor=C.+Delvesauthor=S.+M.+Magnyauthor=R.+Woodwardauthor=S.+A.+Hughesauthor=E.+V.+Boursierauthor=H.+Flynnauthor=A.+M.+Bouillotauthor=P.+Bamboroughauthor=J.+M.+Brusqauthor=F.+J.+Gellibertauthor=E.+J.+Jonesauthor=A.+M.+Riouauthor=P.+Homesauthor=S.+L.+Martinauthor=I.+J.+Uingsauthor=J.+Toumauthor=C.+A.+Clementauthor=A.+B.+Boullayauthor=R.+L.+Grimleyauthor=F.+M.+Blandelauthor=R.+K.+Prinjhaauthor=K.+Leeauthor=J.+Kirilovskyauthor=E.+Nicodeme&title=Discovery+and+characterization+of+small+molecule+inhibitors+of+the+BET+family+bromodomains&doi=10.1021%2Fjm200108t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and characterization of small molecule inhibitors of the BET family bromodomains</span></div><div class="casAuthors">Chung, Chun-wa; Coste, Herve; White, Julia H.; Mirguet, Olivier; Wilde, Jonathan; Gosmini, Romain L.; Delves, Chris; Magny, Sylvie M.; Woodward, Robert; Hughes, Stephen A.; Boursier, Eric V.; Flynn, Helen; Bouillot, Anne M.; Bamborough, Paul; Brusq, Jean-Marie G.; Gellibert, Francoise J.; Jones, Emma J.; Riou, Alizon M.; Homes, Paul; Martin, Sandrine L.; Uings, Iain J.; Toum, Jerome; Clement, Catherine A.; Boullay, Anne-Benedicte; Grimley, Rachel L.; Blandel, Florence M.; Prinjha, Rab K.; Lee, Kevin; Kirilovsky, Jorge; Nicodeme, Edwige</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3827-3838</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Epigenetic mechanisms of gene regulation have a profound role in normal development and disease processes.  An integral part of this mechanism occurs through lysine acetylation of histone tails which are recognized by bromodomains.  While the biol. and structural characterization of many bromodomain contg. proteins has advanced considerably, the therapeutic tractability of this protein family is only now becoming understood.  This paper describes the discovery and mol. characterization of potent (nM) small mol. inhibitors that disrupt the function of the BET family of bromodomains (Brd2, Brd3, and Brd4).  By using a combination of phenotypic screening, chemoproteomics, and biophys. studies, we have discovered that the protein-protein interactions between bromodomains and acetylated histones can be antagonized by selective small mols. that bind at the acetylated lysine recognition pocket.  X-ray crystal structures of compds. bound into bromodomains of Brd2 and Brd4 elucidate the mol. interactions of binding and explain the precisely defined stereochem. required for activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8VauuYkT5YLVg90H21EOLACvtfcHk0lgq8Y_KY4aeHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmtVWnurs%253D&md5=ebc0f6062321dece877259d775f1c19b</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Fjm200108t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200108t%26sid%3Dliteratum%253Aachs%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DCoste%26aufirst%3DH.%26aulast%3DWhite%26aufirst%3DJ.%2BH.%26aulast%3DMirguet%26aufirst%3DO.%26aulast%3DWilde%26aufirst%3DJ.%26aulast%3DGosmini%26aufirst%3DR.%2BL.%26aulast%3DDelves%26aufirst%3DC.%26aulast%3DMagny%26aufirst%3DS.%2BM.%26aulast%3DWoodward%26aufirst%3DR.%26aulast%3DHughes%26aufirst%3DS.%2BA.%26aulast%3DBoursier%26aufirst%3DE.%2BV.%26aulast%3DFlynn%26aufirst%3DH.%26aulast%3DBouillot%26aufirst%3DA.%2BM.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DBrusq%26aufirst%3DJ.%2BM.%26aulast%3DGellibert%26aufirst%3DF.%2BJ.%26aulast%3DJones%26aufirst%3DE.%2BJ.%26aulast%3DRiou%26aufirst%3DA.%2BM.%26aulast%3DHomes%26aufirst%3DP.%26aulast%3DMartin%26aufirst%3DS.%2BL.%26aulast%3DUings%26aufirst%3DI.%2BJ.%26aulast%3DToum%26aufirst%3DJ.%26aulast%3DClement%26aufirst%3DC.%2BA.%26aulast%3DBoullay%26aufirst%3DA.%2BB.%26aulast%3DGrimley%26aufirst%3DR.%2BL.%26aulast%3DBlandel%26aufirst%3DF.%2BM.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DKirilovsky%26aufirst%3DJ.%26aulast%3DNicodeme%26aufirst%3DE.%26atitle%3DDiscovery%2520and%2520characterization%2520of%2520small%2520molecule%2520inhibitors%2520of%2520the%2520BET%2520family%2520bromodomains%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D3827%26epage%3D3838%26doi%3D10.1021%2Fjm200108t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, C. M.</span></span> <span> </span><span class="NLM_article-title">The double bromodomain-containing chromatin adaptor Brd4 and transcriptional regulation</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>282</i></span>,  <span class="NLM_fpage">13141</span>– <span class="NLM_lpage">13145</span>, <span class="refDoi"> DOI: 10.1074/jbc.R700001200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1074%2Fjbc.R700001200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=17329240" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BD2sXks1Ghtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=282&publication_year=2007&pages=13141-13145&author=S.+Y.+Wuauthor=C.+M.+Chiang&title=The+double+bromodomain-containing+chromatin+adaptor+Brd4+and+transcriptional+regulation&doi=10.1074%2Fjbc.R700001200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">The Double Bromodomain-containing Chromatin Adaptor Brd4 and Transcriptional Regulation</span></div><div class="casAuthors">Wu, Shwu-Yuan; Chiang, Cheng-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">282</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">13141-13145</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">A review.  Brd4 is a double bromodomain-contg. protein that binds preferentially to acetylated chromatin.  It belongs to the BET (bromodomains and extraterminal) family that includes mammalian Brd2, Brd3, Brd4, Brdt, Drosophila Fsh, yeast Bdf1, Bdf2, and corresponding homologues in other species.  Brd4 is essential for cellular growth and has been implicated in cell cycle control, DNA replication, and gene rearrangement found in t(15;19)-assocd. carcinomas.  Recently, Brd4 has been found in several transcription complexes, including the general cofactor Mediator and the P-TEFb elongation factor, and is capable of stimulating HIV-1 transcription in a Tat-independent manner.  In addn., Brd4 is used as a cellular adaptor by some animal and human papillomaviruses (HPV) for anchoring viral genomes to mitotic chromosomes.  This tethering, mediated by Brd4 interaction with virus-encoded E2 protein, facilitates viral genome segregation during mitosis.  Interestingly, Brd4 is also identified in a transcriptional silencing complex assembled by HPV E2 and turns out to be the long sought cellular corepressor that inhibits the expression of HPV-encoded E6 and E7 oncoproteins that antagonize p53 and pRB tumor suppressor activity, resp.  The dual role of Brd4 in gene activation and repression illustrates how a dynamic chromatin-binding adaptor is able to recruit distinct transcriptional regulators to modulate promoter activity through cell cycle progression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohccDdZC9iKLVg90H21EOLACvtfcHk0lgq8Y_KY4aeHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXks1Ghtbo%253D&md5=b025a5222b510a7009eca4289370b1d4</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1074%2Fjbc.R700001200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.R700001200%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DS.%2BY.%26aulast%3DChiang%26aufirst%3DC.%2BM.%26atitle%3DThe%2520double%2520bromodomain-containing%2520chromatin%2520adaptor%2520Brd4%2520and%2520transcriptional%2520regulation%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2007%26volume%3D282%26spage%3D13141%26epage%3D13145%26doi%3D10.1074%2Fjbc.R700001200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giles, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H.</span></span> <span> </span><span class="NLM_article-title">Activity of NEO2734, a novel dual inhibitor of both BET and CBP-P300, in SPOP-mutated prostate cancer</span>. <i>J. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">62</span>– <span class="NLM_lpage">62</span>, <span class="refDoi"> DOI: 10.1200/JCO.2019.37.7_suppl.62</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1200%2FJCO.2019.37.7_suppl.62" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2019&pages=62-62&author=Y.+Yanauthor=D.+Wangauthor=J.+Maauthor=X.+Pangauthor=D.+Linauthor=F.+Gilesauthor=Y.+Wangauthor=H.+Huang&title=Activity+of+NEO2734%2C+a+novel+dual+inhibitor+of+both+BET+and+CBP-P300%2C+in+SPOP-mutated+prostate+cancer&doi=10.1200%2FJCO.2019.37.7_suppl.62"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1200%2FJCO.2019.37.7_suppl.62&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1200%252FJCO.2019.37.7_suppl.62%26sid%3Dliteratum%253Aachs%26aulast%3DYan%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DD.%26aulast%3DMa%26aufirst%3DJ.%26aulast%3DPang%26aufirst%3DX.%26aulast%3DLin%26aufirst%3DD.%26aulast%3DGiles%26aufirst%3DF.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DH.%26atitle%3DActivity%2520of%2520NEO2734%252C%2520a%2520novel%2520dual%2520inhibitor%2520of%2520both%2520BET%2520and%2520CBP-P300%252C%2520in%2520SPOP-mutated%2520prostate%2520cancer%26jtitle%3DJ.%2520Clin.%2520Oncol.%26date%3D2019%26volume%3D37%26spage%3D62%26epage%3D62%26doi%3D10.1200%2FJCO.2019.37.7_suppl.62" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Raghav, K. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chavez-MacGregor, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hortobagyi, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mills, G. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meric-Bernstam, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blumenschein, G. R.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gonzalez-Angulo, A. M.</span></span> <span> </span><span class="NLM_article-title">cMET and phospho-cMET protein levels in breast cancers and survival outcomes</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">2269</span>– <span class="NLM_lpage">2277</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.CCR-11-2830</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1158%2F1078-0432.CCR-11-2830" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=22374333" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC38XmsFOntbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2012&pages=2269-2277&author=K.+P.+Raghavauthor=W.+Wangauthor=S.+Liuauthor=M.+Chavez-MacGregorauthor=X.+Mengauthor=G.+N.+Hortobagyiauthor=G.+B.+Millsauthor=F.+Meric-Bernstamauthor=G.+R.+Blumenscheinauthor=A.+M.+Gonzalez-Angulo&title=cMET+and+phospho-cMET+protein+levels+in+breast+cancers+and+survival+outcomes&doi=10.1158%2F1078-0432.CCR-11-2830"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">cMET and Phospho-cMET Protein Levels in Breast Cancers and Survival Outcomes</span></div><div class="casAuthors">Raghav, Kanwal P.; Wang, Wenting; Liu, Shuying; Chavez-MacGregor, Mariana; Meng, Xiaolong; Hortobagyi, Gabriel N.; Mills, Gordon B.; Meric-Bernstam, Funda; Blumenschein, George R., Jr.; Gonzalez-Angulo, Ana M.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2269-2277</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">PURPOSE: To evaluate cMET (mesenchymal-epithelial transition factor gene) and phospho-cMET (p-cMET) levels in breast cancer subtypes and its impact on survival outcomes.  Exptl. Design: We measured protein levels of cMET and p-cMET in 257 breast cancers using reverse phase protein array.  Regression tree method and Martingale residual plots were applied to find best cutoff point for high and low levels.  Kaplan-Meier survival curves were used to est. relapse-free (RFS) and overall (OS) survival.  Cox proportional hazards models were fit to det. assocns. of cMET/p-cMET with outcomes after adjustment for other characteristics.  RESULTS: Median age was 51 years.  There were 140 (54.5%) hormone receptor (HR) pos., 53 (20.6%) HER2 pos., and 64 (24.9%) triple-neg. tumors.  Using selected cutoffs, 181 (70.4%) and 123 (47.9%) cancers had high levels of cMET and p-cMET, resp.  There were no significant differences in mean expression of cMET (P < 0.128) and p-cMET (P < 0.088) by breast cancer subtype.  Dichotomized cMET and p-cMET level was a significant prognostic factor for RFS [HR: 2.44, 95% confidence interval (CI): 1.34-4.44, P = 0.003 and HR: 1.64, 95% CI: 1.04-2.60, P = 0.033] and OS (HR: 3.18, 95% CI: 1.43-7.11, P = 0.003 and HR: 1.92, 95% CI: 1.08-3.44, P = 0.025).  Within breast cancer subtypes, high cMET levels were assocd. with worse RFS (P = 0.014) and OS (P = 0.006) in HR-pos. tumors, and high p-cMET levels were assocd. with worse RFS (P = 0.019) and OS (P = 0.014) in HER2-pos. breast cancers.  In multivariable anal., patients with high cMET had a significantly higher risk of recurrence (HR: 2.06, 95% CI: 1.08-3.94, P = 0.028) and death (HR: 2.81, 95% CI: 1.19-6.64, P = 0.019).  High p-cMET level was assocd. with higher risk of recurrence (HR: 1.79, 95% CI: 1.08-2.95.77, P = 0.020).  CONCLUSIONS: High levels of cMET and p-cMET were seen in all breast cancer subtypes and correlated with poor prognosis.  Clin Cancer Res; 18(8); 2269-77.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowG62mI7wLAbVg90H21EOLACvtfcHk0lh67eumBnplhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmsFOntbk%253D&md5=c3ccdd6329f649fefa6c8bf62c259168</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-11-2830&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-11-2830%26sid%3Dliteratum%253Aachs%26aulast%3DRaghav%26aufirst%3DK.%2BP.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DChavez-MacGregor%26aufirst%3DM.%26aulast%3DMeng%26aufirst%3DX.%26aulast%3DHortobagyi%26aufirst%3DG.%2BN.%26aulast%3DMills%26aufirst%3DG.%2BB.%26aulast%3DMeric-Bernstam%26aufirst%3DF.%26aulast%3DBlumenschein%26aufirst%3DG.%2BR.%26aulast%3DGonzalez-Angulo%26aufirst%3DA.%2BM.%26atitle%3DcMET%2520and%2520phospho-cMET%2520protein%2520levels%2520in%2520breast%2520cancers%2520and%2520survival%2520outcomes%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2012%26volume%3D18%26spage%3D2269%26epage%3D2277%26doi%3D10.1158%2F1078-0432.CCR-11-2830" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ho-Yen, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kermorgant, S.</span></span> <span> </span><span class="NLM_article-title">The clinical and functional significance of c-Met in breast cancer: a review</span>. <i>Breast Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">17</span>– <span class="NLM_lpage">52</span>, <span class="refDoi"> DOI: 10.1186/s13058-015-0547-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1186%2Fs13058-015-0547-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=25849559" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2015&pages=17-52&author=C.+M.+Ho-Yenauthor=J.+L.+Jonesauthor=S.+Kermorgant&title=The+clinical+and+functional+significance+of+c-Met+in+breast+cancer%3A+a+review&doi=10.1186%2Fs13058-015-0547-6"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1186%2Fs13058-015-0547-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13058-015-0547-6%26sid%3Dliteratum%253Aachs%26aulast%3DHo-Yen%26aufirst%3DC.%2BM.%26aulast%3DJones%26aufirst%3DJ.%2BL.%26aulast%3DKermorgant%26aufirst%3DS.%26atitle%3DThe%2520clinical%2520and%2520functional%2520significance%2520of%2520c-Met%2520in%2520breast%2520cancer%253A%2520a%2520review%26jtitle%3DBreast%2520Cancer%2520Res.%26date%3D2015%26volume%3D17%26spage%3D17%26epage%3D52%26doi%3D10.1186%2Fs13058-015-0547-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Merenbakh-Lamin, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ben-Baruch, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeheskel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dvir, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soussan-Gutman, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeselsohn, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yelensky, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sarid, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rizel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rubinek, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolf, I.</span></span> <span> </span><span class="NLM_article-title">D538G mutation in estrogen receptor-alpha: A novel mechanism for acquired endocrine resistance in breast cancer</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">6856</span>– <span class="NLM_lpage">6864</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-13-1197</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1158%2F0008-5472.CAN-13-1197" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=24217577" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVKhurfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2013&pages=6856-6864&author=K.+Merenbakh-Laminauthor=N.+Ben-Baruchauthor=A.+Yeheskelauthor=A.+Dvirauthor=L.+Soussan-Gutmanauthor=R.+Jeselsohnauthor=R.+Yelenskyauthor=M.+Brownauthor=V.+A.+Millerauthor=D.+Saridauthor=S.+Rizelauthor=B.+Kleinauthor=T.+Rubinekauthor=I.+Wolf&title=D538G+mutation+in+estrogen+receptor-alpha%3A+A+novel+mechanism+for+acquired+endocrine+resistance+in+breast+cancer&doi=10.1158%2F0008-5472.CAN-13-1197"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">D538G Mutation in Estrogen Receptor-α: A Novel Mechanism for Acquired Endocrine Resistance in Breast Cancer</span></div><div class="casAuthors">Merenbakh-Lamin, Keren; Ben-Baruch, Noa; Yeheskel, Adva; Dvir, Addie; Soussan-Gutman, Lior; Jeselsohn, Rinath; Yelensky, Roman; Brown, Myles; Miller, Vincent A.; Sarid, David; Rizel, Shulamith; Klein, Baruch; Rubinek, Tami; Wolf, Ido</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">6856-6864</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Resistance to endocrine therapy occurs in virtually all patients with estrogen receptor a (ERa)-pos. metastatic breast cancer, and is attributed to various mechanisms including loss of ERα expression, altered activity of coregulators, and cross-talk between the ERa and growth factor signaling pathways.  To our knowledge, acquired mutations of the ERα have not been described as mediating endocrine resistance.  Samples of 13 patients with metastatic breast cancer were analyzed for mutations in cancer-related genes.  In five patients who developed resistance to hormonal therapy, a mutation of A to G at position 1,613 of ERα, resulting in a substitution of aspartic acid at position 538 to glycine (D538G), was identified in liver metastases.  Importantly, the mutation was not detected in the primary tumors obtained prior to endocrine treatment.  Structural modeling indicated that D538G substitution leads to a conformational change in the ligand-binding domain, which mimics the conformation of activated ligand-bound receptor and alters binding of tamoxifen.  Indeed, expts. in breast cancer cells indicated constitutive, ligand-independent transcriptional activity of the D538G receptor, and overexpression of it enhanced proliferation and conferred resistance to tamoxifen.  These data indicate a novel mechanism of acquired endocrine resistance in breast cancer.  Further studies are needed to assess the frequency of D538G-ERα among patients with breast cancer and explore ways to inhibit its activity and restore endocrine sensitivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWljQ_odNNUrVg90H21EOLACvtfcHk0lh67eumBnplhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvVKhurfN&md5=6dca8ee6cdb77bfc8a94d02fe5630ea8</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-13-1197&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-13-1197%26sid%3Dliteratum%253Aachs%26aulast%3DMerenbakh-Lamin%26aufirst%3DK.%26aulast%3DBen-Baruch%26aufirst%3DN.%26aulast%3DYeheskel%26aufirst%3DA.%26aulast%3DDvir%26aufirst%3DA.%26aulast%3DSoussan-Gutman%26aufirst%3DL.%26aulast%3DJeselsohn%26aufirst%3DR.%26aulast%3DYelensky%26aufirst%3DR.%26aulast%3DBrown%26aufirst%3DM.%26aulast%3DMiller%26aufirst%3DV.%2BA.%26aulast%3DSarid%26aufirst%3DD.%26aulast%3DRizel%26aufirst%3DS.%26aulast%3DKlein%26aufirst%3DB.%26aulast%3DRubinek%26aufirst%3DT.%26aulast%3DWolf%26aufirst%3DI.%26atitle%3DD538G%2520mutation%2520in%2520estrogen%2520receptor-alpha%253A%2520A%2520novel%2520mechanism%2520for%2520acquired%2520endocrine%2520resistance%2520in%2520breast%2520cancer%26jtitle%3DCancer%2520Res.%26date%3D2013%26volume%3D73%26spage%3D6856%26epage%3D6864%26doi%3D10.1158%2F0008-5472.CAN-13-1197" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Del
Re, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertolini, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crucitta, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fontanelli, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rofi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Angelis, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diodati, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavallero, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gianfilippo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salvadori, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fogli, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Falcone, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scatena, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naccarato, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roncella, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghilli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morganti, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fontana, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danesi, R.</span></span> <span> </span><span class="NLM_article-title">Overexpression of TK1 and CDK9 in plasma-derived exosomes is associated with clinical resistance to CDK4/6 inhibitors in metastatic breast cancer patients</span>. <i>Breast Cancer Res. Treat.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>178</i></span>,  <span class="NLM_fpage">57</span>– <span class="NLM_lpage">62</span>, <span class="refDoi"> DOI: 10.1007/s10549-019-05365-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1007%2Fs10549-019-05365-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=31346846" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsVGrtbzM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=178&publication_year=2019&pages=57-62&author=M.+Del%0AReauthor=I.+Bertoliniauthor=S.+Crucittaauthor=L.+Fontanelliauthor=E.+Rofiauthor=C.+De+Angelisauthor=L.+Diodatiauthor=D.+Cavalleroauthor=G.+Gianfilippoauthor=B.+Salvadoriauthor=S.+Fogliauthor=A.+Falconeauthor=C.+Scatenaauthor=A.+G.+Naccaratoauthor=M.+Roncellaauthor=M.+Ghilliauthor=R.+Morgantiauthor=A.+Fontanaauthor=R.+Danesi&title=Overexpression+of+TK1+and+CDK9+in+plasma-derived+exosomes+is+associated+with+clinical+resistance+to+CDK4%2F6+inhibitors+in+metastatic+breast+cancer+patients&doi=10.1007%2Fs10549-019-05365-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Overexpression of TK1 and CDK9 in plasma-derived exosomes is associated with clinical resistance to CDK4/6 inhibitors in metastatic breast cancer patients</span></div><div class="casAuthors">Del Re, Marzia; Bertolini, Ilaria; Crucitta, Stefania; Fontanelli, Lorenzo; Rofi, Eleonora; De Angelis, Claudia; Diodati, Lucrezia; Cavallero, Diletta; Gianfilippo, Giulia; Salvadori, Barbara; Fogli, Stefano; Falcone, Alfredo; Scatena, Cristian; Naccarato, Antonio Giuseppe; Roncella, Manuela; Ghilli, Matteo; Morganti, Riccardo; Fontana, Andrea; Danesi, Romano</div><div class="citationInfo"><span class="NLM_cas:title">Breast Cancer Research and Treatment</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">178</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">57-62</span>CODEN:
                <span class="NLM_cas:coden">BCTRD6</span>;
        ISSN:<span class="NLM_cas:issn">0167-6806</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The present article addresses the role of the mRNA expression of thymidine kinase 1 (TK1), CDK4, 6 and 9 in plasma-derived exosomes and their relevance in the pharmacol. activity of CDK4/6i.  Methods: Blood samples of 40 HR+/HER2- advanced breast cancer patients were collected before (T0) the administration of palbociclib plus hormonal therapy and after 3 mo (T1).  RNA was isolated from exosomes and analyzed for the expression of TK1, CDK 4, 6 and 9 by digital droplet PCR (ddPCR).  Results: A higher value of TK1 copies/mL at baseline (T0) was significantly assocd. with the no. of previous lines of chemotherapy (p = 0.009).  In patients with PD, a significant increase was obsd. in the no. of copies/mL of TK1 (p = 0.01) and CDK9 (p = 0.03) comparing T1 vs.  T0 values. No significant correlations between response to treatment and clin. parameters were found at univariate anal.  High baseline CDK4 expression was significantly correlated with longer PFS in patients treated with fulvestrant + palbociclib (low vs. high: 6.45 mo vs. not reached, p = 0.01).  Conclusions: The present study demonstrates that, in plasma-derived exosomes, high baseline CDK4 mRNA levels are assocd. with response to palbociclib plus hormonal therapy, while the increase in TK1 and CDK9 mRNA copies/mL is assocd. with clin. resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAoJIlNIJ7NLVg90H21EOLACvtfcHk0lgcB9ETlNYCuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsVGrtbzM&md5=e5684c7965c4527fe92c7c5c9869e428</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1007%2Fs10549-019-05365-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10549-019-05365-y%26sid%3Dliteratum%253Aachs%26aulast%3DDel%2BRe%26aufirst%3DM.%26aulast%3DBertolini%26aufirst%3DI.%26aulast%3DCrucitta%26aufirst%3DS.%26aulast%3DFontanelli%26aufirst%3DL.%26aulast%3DRofi%26aufirst%3DE.%26aulast%3DDe%2BAngelis%26aufirst%3DC.%26aulast%3DDiodati%26aufirst%3DL.%26aulast%3DCavallero%26aufirst%3DD.%26aulast%3DGianfilippo%26aufirst%3DG.%26aulast%3DSalvadori%26aufirst%3DB.%26aulast%3DFogli%26aufirst%3DS.%26aulast%3DFalcone%26aufirst%3DA.%26aulast%3DScatena%26aufirst%3DC.%26aulast%3DNaccarato%26aufirst%3DA.%2BG.%26aulast%3DRoncella%26aufirst%3DM.%26aulast%3DGhilli%26aufirst%3DM.%26aulast%3DMorganti%26aufirst%3DR.%26aulast%3DFontana%26aufirst%3DA.%26aulast%3DDanesi%26aufirst%3DR.%26atitle%3DOverexpression%2520of%2520TK1%2520and%2520CDK9%2520in%2520plasma-derived%2520exosomes%2520is%2520associated%2520with%2520clinical%2520resistance%2520to%2520CDK4%252F6%2520inhibitors%2520in%2520metastatic%2520breast%2520cancer%2520patients%26jtitle%3DBreast%2520Cancer%2520Res.%2520Treat.%26date%3D2019%26volume%3D178%26spage%3D57%26epage%3D62%26doi%3D10.1007%2Fs10549-019-05365-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witwicki, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tabassum, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janiszewska, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huh, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doherty, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohammed, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stover, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ekram, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peluffo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Santos, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krop, I. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dillon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKeown, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ott, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ni, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duarte, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anders, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winer, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Letai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barry, W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carroll, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polyak, K.</span></span> <span> </span><span class="NLM_article-title">Response and resistance to BET bromodomain inhibitors in triple-negative breast cancer</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>529</i></span>,  <span class="NLM_fpage">413</span>– <span class="NLM_lpage">417</span>, <span class="refDoi"> DOI: 10.1038/nature16508</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1038%2Fnature16508" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=26735014" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC28Xns1Gmtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=529&publication_year=2016&pages=413-417&author=S.+Shuauthor=C.+Y.+Linauthor=H.+H.+Heauthor=R.+M.+Witwickiauthor=D.+P.+Tabassumauthor=J.+M.+Robertsauthor=M.+Janiszewskaauthor=S.+J.+Huhauthor=Y.+Liangauthor=J.+Ryanauthor=E.+Dohertyauthor=H.+Mohammedauthor=H.+Guoauthor=D.+G.+Stoverauthor=M.+B.+Ekramauthor=G.+Peluffoauthor=J.+Brownauthor=C.+D%E2%80%99Santosauthor=I.+E.+Kropauthor=D.+Dillonauthor=M.+McKeownauthor=C.+Ottauthor=J.+Qiauthor=M.+Niauthor=P.+K.+Raoauthor=M.+Duarteauthor=S.+Y.+Wuauthor=C.+M.+Chiangauthor=L.+Andersauthor=R.+A.+Youngauthor=E.+Winerauthor=A.+Letaiauthor=W.+T.+Barryauthor=J.+S.+Carrollauthor=H.+Longauthor=M.+Brownauthor=X.+S.+Liuauthor=C.+A.+Meyerauthor=J.+E.+Bradnerauthor=K.+Polyak&title=Response+and+resistance+to+BET+bromodomain+inhibitors+in+triple-negative+breast+cancer&doi=10.1038%2Fnature16508"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Response and resistance to BET bromodomain inhibitors in triple-negative breast cancer</span></div><div class="casAuthors">Shu, Shaokun; Lin, Charles Y.; He, Housheng Hansen; Witwicki, Robert M.; Tabassum, Doris P.; Roberts, Justin M.; Janiszewska, Michalina; Huh, Sung Jin; Liang, Yi; Ryan, Jeremy; Doherty, Ernest; Mohammed, Hisham; Guo, Hao; Stover, Daniel G.; Ekram, Muhammad B.; Peluffo, Guillermo; Brown, Jonathan; D'Santos, Clive; Krop, Ian E.; Dillon, Deborah; McKeown, Michael; Ott, Christopher; Qi, Jun; Ni, Min; Rao, Prakash K.; Duarte, Melissa; Wu, Shwu-Yuan; Chiang, Cheng-Ming; Anders, Lars; Young, Richard A.; Winer, Eric P.; Letai, Antony; Barry, William T.; Carroll, Jason S.; Long, Henry W.; Brown, Myles; Shirley Liu, X.; Meyer, Clifford A.; Bradner, James E.; Polyak, Kornelia</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">529</span>
        (<span class="NLM_cas:issue">7586</span>),
    <span class="NLM_cas:pages">413-417</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Triple-neg. breast cancer (TNBC) is a heterogeneous and clin. aggressive disease for which there is no targeted therapy.  BET bromodomain inhibitors, which have shown efficacy in several models of cancer, have not been evaluated in TNBC.  These inhibitors displace BET bromodomain proteins such as BRD4 from chromatin by competing with their acetyl-lysine recognition modules, leading to inhibition of oncogenic transcriptional programs.  Here we report the preferential sensitivity of TNBCs to BET bromodomain inhibition in vitro and in vivo, establishing a rationale for clin. investigation and further motivation to understand mechanisms of resistance.  In paired cell lines selected for acquired resistance to BET inhibition from previously sensitive TNBCs, we failed to identify gatekeeper mutations, new driver events or drug pump activation.  BET-resistant TNBC cells remain dependent on wild-type BRD4, which supports transcription and cell proliferation in a bromodomain-independent manner.  Proteomic studies of resistant TNBC identify strong assocn. with MED1 and hyper-phosphorylation of BRD4 attributable to decreased activity of PP2A, identified here as a principal BRD4 serine phosphatase.  Together, these studies provide a rationale for BET inhibition in TNBC and present mechanism-based combination strategies to anticipate clin. drug resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZu5A85pdQDbVg90H21EOLACvtfcHk0lgcB9ETlNYCuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xns1Gmtw%253D%253D&md5=50a03861d9156ee6e344e95c5f5361b9</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1038%2Fnature16508&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature16508%26sid%3Dliteratum%253Aachs%26aulast%3DShu%26aufirst%3DS.%26aulast%3DLin%26aufirst%3DC.%2BY.%26aulast%3DHe%26aufirst%3DH.%2BH.%26aulast%3DWitwicki%26aufirst%3DR.%2BM.%26aulast%3DTabassum%26aufirst%3DD.%2BP.%26aulast%3DRoberts%26aufirst%3DJ.%2BM.%26aulast%3DJaniszewska%26aufirst%3DM.%26aulast%3DHuh%26aufirst%3DS.%2BJ.%26aulast%3DLiang%26aufirst%3DY.%26aulast%3DRyan%26aufirst%3DJ.%26aulast%3DDoherty%26aufirst%3DE.%26aulast%3DMohammed%26aufirst%3DH.%26aulast%3DGuo%26aufirst%3DH.%26aulast%3DStover%26aufirst%3DD.%2BG.%26aulast%3DEkram%26aufirst%3DM.%2BB.%26aulast%3DPeluffo%26aufirst%3DG.%26aulast%3DBrown%26aufirst%3DJ.%26aulast%3DD%25E2%2580%2599Santos%26aufirst%3DC.%26aulast%3DKrop%26aufirst%3DI.%2BE.%26aulast%3DDillon%26aufirst%3DD.%26aulast%3DMcKeown%26aufirst%3DM.%26aulast%3DOtt%26aufirst%3DC.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DNi%26aufirst%3DM.%26aulast%3DRao%26aufirst%3DP.%2BK.%26aulast%3DDuarte%26aufirst%3DM.%26aulast%3DWu%26aufirst%3DS.%2BY.%26aulast%3DChiang%26aufirst%3DC.%2BM.%26aulast%3DAnders%26aufirst%3DL.%26aulast%3DYoung%26aufirst%3DR.%2BA.%26aulast%3DWiner%26aufirst%3DE.%26aulast%3DLetai%26aufirst%3DA.%26aulast%3DBarry%26aufirst%3DW.%2BT.%26aulast%3DCarroll%26aufirst%3DJ.%2BS.%26aulast%3DLong%26aufirst%3DH.%26aulast%3DBrown%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DX.%2BS.%26aulast%3DMeyer%26aufirst%3DC.%2BA.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DPolyak%26aufirst%3DK.%26atitle%3DResponse%2520and%2520resistance%2520to%2520BET%2520bromodomain%2520inhibitors%2520in%2520triple-negative%2520breast%2520cancer%26jtitle%3DNature%26date%3D2016%26volume%3D529%26spage%3D413%26epage%3D417%26doi%3D10.1038%2Fnature16508" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ozer, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El-Gamal, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hing, Z. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blachly, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrington, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grieselhuber, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alinari, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baiocchi, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brinton, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baskin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cannon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beaver, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goettl, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucas, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woyach, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sampath, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lehman, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spevak, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Severson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shellooe, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carias, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ewing, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marimuthu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tantoy, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walters, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanftner, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rezaei, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nespi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matusow, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Habets, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ibrahim, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathe, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bollag, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Byrd, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lapalombella, R.</span></span> <span> </span><span class="NLM_article-title">BRD4 Profiling Identifies Critical Chronic Lymphocytic Leukemia Oncogenic Circuits and Reveals Sensitivity to PLX51107, a Novel Structurally Distinct BET Inhibitor</span>. <i>Cancer Discovery</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">458</span>– <span class="NLM_lpage">477</span>, <span class="refDoi"> DOI: 10.1158/2159-8290.CD-17-0902</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1158%2F2159-8290.CD-17-0902" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=29386193" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC1cXms1emsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&pages=458-477&author=H.+G.+Ozerauthor=D.+El-Gamalauthor=B.+Powellauthor=Z.+A.+Hingauthor=J.+S.+Blachlyauthor=B.+Harringtonauthor=S.+Mitchellauthor=N.+R.+Grieselhuberauthor=K.+Williamsauthor=T.+H.+Laiauthor=L.+Alinariauthor=R.+A.+Baiocchiauthor=L.+Brintonauthor=E.+Baskinauthor=M.+Cannonauthor=L.+Beaverauthor=V.+M.+Goettlauthor=D.+M.+Lucasauthor=J.+A.+Woyachauthor=D.+Sampathauthor=A.+M.+Lehmanauthor=L.+Yuauthor=J.+Zhangauthor=Y.+Maauthor=Y.+Zhangauthor=W.+Spevakauthor=S.+Shiauthor=P.+Seversonauthor=R.+Shellooeauthor=H.+Cariasauthor=G.+Tsangauthor=K.+Dongauthor=T.+Ewingauthor=A.+Marimuthuauthor=C.+Tantoyauthor=J.+Waltersauthor=L.+Sanftnerauthor=H.+Rezaeiauthor=M.+Nespiauthor=B.+Matusowauthor=G.+Habetsauthor=P.+Ibrahimauthor=C.+Zhangauthor=E.+A.+Matheauthor=G.+Bollagauthor=J.+C.+Byrdauthor=R.+Lapalombella&title=BRD4+Profiling+Identifies+Critical+Chronic+Lymphocytic+Leukemia+Oncogenic+Circuits+and+Reveals+Sensitivity+to+PLX51107%2C+a+Novel+Structurally+Distinct+BET+Inhibitor&doi=10.1158%2F2159-8290.CD-17-0902"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">BRD4 Profiling Identifies Critical Chronic Lymphocytic Leukemia Oncogenic Circuits and Reveals Sensitivity to PLX51107, a Novel Structurally Distinct BET Inhibitor</span></div><div class="casAuthors">Ozer, Hatice Gulcin; El-Gamal, Dalia; Powell, Ben; Hing, Zachary A.; Blachly, James S.; Harrington, Bonnie; Mitchell, Shaneice; Grieselhuber, Nicole R.; Williams, Katie; Lai, Tzung-Huei; Alinari, Lapo; Baiocchi, Robert A.; Brinton, Lindsey; Baskin, Elizabeth; Cannon, Matthew; Beaver, Larry; Goettl, Virginia M.; Lucas, David M.; Woyach, Jennifer A.; Sampath, Deepa; Lehman, Amy M.; Yu, Lianbo; Zhang, Jiazhong; Ma, Yan; Zhang, Ying; Spevak, Wayne; Shi, Songyuan; Severson, Paul; Shellooe, Rafe; Carias, Heidi; Tsang, Garson; Dong, Ken; Ewing, Todd; Marimuthu, Adhirai; Tantoy, Christina; Walters, Jason; Sanftner, Laura; Rezaei, Hamid; Nespi, Marika; Matusow, Bernice; Habets, Gaston; Ibrahim, Prabha; Zhang, Chao; Mathe, Ewy A.; Bollag, Gideon; Byrd, John C.; Lapalombella, Rosa</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Discovery</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">458-477</span>CODEN:
                <span class="NLM_cas:coden">CDAIB2</span>;
        ISSN:<span class="NLM_cas:issn">2159-8274</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Bromodomain and extra-terminal (BET) family proteins are key regulators of gene expression in cancer.  Herein, we utilize BRD4 profiling to identify crit. pathways involved in pathogenesis of chronic lymphocytic leukemia (CLL).  BRD4 is overexpressed in CLL and is enriched proximal to genes upregulated or de novo expressed in CLL with known functions in disease pathogenesis and progression.  These genes, including key members of the B-cell receptor (BCR) signaling pathway, provide a rationale for this therapeutic approach to identify new targets in alternative types of cancer.  Addnl., we describe PLX51107, a structurally distinct BET inhibitor with novel in vitro and in vivo pharmacol. properties that emulates or exceeds the efficacy of BCR signaling agents in preclin. models of CLL.  Herein, the discovery of the involvement of BRD4 in the core CLL transcriptional program provides a compelling rationale for clin. investigation of PLX51107 as epigenetic therapy in CLL and application of BRD4 profiling in other cancers.  Significance: To date, functional studies of BRD4 in CLL are lacking.  Through integrated genomic, functional, and pharmacol. analyses, we uncover the existence of BRD4-regulated core CLL transcriptional programs and present preclin. proof-of-concept studies validating BET inhibition as an epigenetic approach to target BCR signaling in CLL.  Cancer Discov; 8(4); 458-77. ©2018 AACR.  This article is highlighted in the In This Issue feature, p.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqpvzZtZKkb5rVg90H21EOLACvtfcHk0liOShUMrQ83EA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXms1emsr4%253D&md5=3715a65bb8cee2a5fe53994a3d106303</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1158%2F2159-8290.CD-17-0902&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F2159-8290.CD-17-0902%26sid%3Dliteratum%253Aachs%26aulast%3DOzer%26aufirst%3DH.%2BG.%26aulast%3DEl-Gamal%26aufirst%3DD.%26aulast%3DPowell%26aufirst%3DB.%26aulast%3DHing%26aufirst%3DZ.%2BA.%26aulast%3DBlachly%26aufirst%3DJ.%2BS.%26aulast%3DHarrington%26aufirst%3DB.%26aulast%3DMitchell%26aufirst%3DS.%26aulast%3DGrieselhuber%26aufirst%3DN.%2BR.%26aulast%3DWilliams%26aufirst%3DK.%26aulast%3DLai%26aufirst%3DT.%2BH.%26aulast%3DAlinari%26aufirst%3DL.%26aulast%3DBaiocchi%26aufirst%3DR.%2BA.%26aulast%3DBrinton%26aufirst%3DL.%26aulast%3DBaskin%26aufirst%3DE.%26aulast%3DCannon%26aufirst%3DM.%26aulast%3DBeaver%26aufirst%3DL.%26aulast%3DGoettl%26aufirst%3DV.%2BM.%26aulast%3DLucas%26aufirst%3DD.%2BM.%26aulast%3DWoyach%26aufirst%3DJ.%2BA.%26aulast%3DSampath%26aufirst%3DD.%26aulast%3DLehman%26aufirst%3DA.%2BM.%26aulast%3DYu%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DMa%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DSpevak%26aufirst%3DW.%26aulast%3DShi%26aufirst%3DS.%26aulast%3DSeverson%26aufirst%3DP.%26aulast%3DShellooe%26aufirst%3DR.%26aulast%3DCarias%26aufirst%3DH.%26aulast%3DTsang%26aufirst%3DG.%26aulast%3DDong%26aufirst%3DK.%26aulast%3DEwing%26aufirst%3DT.%26aulast%3DMarimuthu%26aufirst%3DA.%26aulast%3DTantoy%26aufirst%3DC.%26aulast%3DWalters%26aufirst%3DJ.%26aulast%3DSanftner%26aufirst%3DL.%26aulast%3DRezaei%26aufirst%3DH.%26aulast%3DNespi%26aufirst%3DM.%26aulast%3DMatusow%26aufirst%3DB.%26aulast%3DHabets%26aufirst%3DG.%26aulast%3DIbrahim%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DMathe%26aufirst%3DE.%2BA.%26aulast%3DBollag%26aufirst%3DG.%26aulast%3DByrd%26aufirst%3DJ.%2BC.%26aulast%3DLapalombella%26aufirst%3DR.%26atitle%3DBRD4%2520Profiling%2520Identifies%2520Critical%2520Chronic%2520Lymphocytic%2520Leukemia%2520Oncogenic%2520Circuits%2520and%2520Reveals%2520Sensitivity%2520to%2520PLX51107%252C%2520a%2520Novel%2520Structurally%2520Distinct%2520BET%2520Inhibitor%26jtitle%3DCancer%2520Discovery%26date%3D2018%26volume%3D8%26spage%3D458%26epage%3D477%26doi%3D10.1158%2F2159-8290.CD-17-0902" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Patnaik, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carvajal, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komatsubara, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Britten, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wesolowski, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michelson, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alcantar, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Severson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orloff, M. M.</span></span> <span> </span><span class="NLM_article-title">Phase ib/2a study of PLX51107, a small molecule BET inhibitor, in subjects with advanced hematological malignancies and solid tumors</span>. <i>J. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">2550</span>– <span class="NLM_lpage">2550</span>, <span class="refDoi"> DOI: 10.1200/JCO.2018.36.15_suppl.2550</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1200%2FJCO.2018.36.15_suppl.2550" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2018&pages=2550-2550&author=A.+Patnaikauthor=R.+D.+Carvajalauthor=K.+M.+Komatsubaraauthor=C.+D.+Brittenauthor=R.+Wesolowskiauthor=G.+Michelsonauthor=O.+Alcantarauthor=C.+Zhangauthor=B.+Powellauthor=P.+Seversonauthor=E.+Martinauthor=M.+M.+Orloff&title=Phase+ib%2F2a+study+of+PLX51107%2C+a+small+molecule+BET+inhibitor%2C+in+subjects+with+advanced+hematological+malignancies+and+solid+tumors&doi=10.1200%2FJCO.2018.36.15_suppl.2550"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1200%2FJCO.2018.36.15_suppl.2550&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1200%252FJCO.2018.36.15_suppl.2550%26sid%3Dliteratum%253Aachs%26aulast%3DPatnaik%26aufirst%3DA.%26aulast%3DCarvajal%26aufirst%3DR.%2BD.%26aulast%3DKomatsubara%26aufirst%3DK.%2BM.%26aulast%3DBritten%26aufirst%3DC.%2BD.%26aulast%3DWesolowski%26aufirst%3DR.%26aulast%3DMichelson%26aufirst%3DG.%26aulast%3DAlcantar%26aufirst%3DO.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DPowell%26aufirst%3DB.%26aulast%3DSeverson%26aufirst%3DP.%26aulast%3DMartin%26aufirst%3DE.%26aulast%3DOrloff%26aufirst%3DM.%2BM.%26atitle%3DPhase%2520ib%252F2a%2520study%2520of%2520PLX51107%252C%2520a%2520small%2520molecule%2520BET%2520inhibitor%252C%2520in%2520subjects%2520with%2520advanced%2520hematological%2520malignancies%2520and%2520solid%2520tumors%26jtitle%3DJ.%2520Clin.%2520Oncol.%26date%3D2018%26volume%3D36%26spage%3D2550%26epage%3D2550%26doi%3D10.1200%2FJCO.2018.36.15_suppl.2550" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lewin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soria, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stathis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delord, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peters, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Awada, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aftimos, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bekradda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rezai, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hussain, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perez, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siu, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Massard, C.</span></span> <span> </span><span class="NLM_article-title">Phase Ib Trial With Birabresib, a Small-Molecule Inhibitor of Bromodomain and Extraterminal Proteins, in Patients With Selected Advanced Solid Tumors</span>. <i>J. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">3007</span>– <span class="NLM_lpage">3014</span>, <span class="refDoi"> DOI: 10.1200/JCO.2018.78.2292</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1200%2FJCO.2018.78.2292" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=29733771" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjvFymurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2018&pages=3007-3014&author=J.+Lewinauthor=J.+C.+Soriaauthor=A.+Stathisauthor=J.+P.+Delordauthor=S.+Petersauthor=A.+Awadaauthor=P.+G.+Aftimosauthor=M.+Bekraddaauthor=K.+Rezaiauthor=Z.+Zengauthor=A.+Hussainauthor=S.+Perezauthor=L.+L.+Siuauthor=C.+Massard&title=Phase+Ib+Trial+With+Birabresib%2C+a+Small-Molecule+Inhibitor+of+Bromodomain+and+Extraterminal+Proteins%2C+in+Patients+With+Selected+Advanced+Solid+Tumors&doi=10.1200%2FJCO.2018.78.2292"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Phase Ib trial with birabresib, a small-molecule inhibitor of bromodomain and extraterminal proteins, in patients with selected advanced solid tumors</span></div><div class="casAuthors">Lewin, Jeremy; Soria, Jean-Charles; Stathis, Anastasios; Delord, Jean-Pierre; Peters, Solange; Awada, Ahmad; Aftimos, Philippe G.; Bekradda, Mohamed; Rezai, Keyvan; Zeng, Zhen; Hussain, Azher; Perez, Susan; Siu, Lillian L.; Massard, Christophe</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Oncology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">3007-3014</span>CODEN:
                <span class="NLM_cas:coden">JCONDN</span>;
        ISSN:<span class="NLM_cas:issn">1527-7755</span>.
    
            (<span class="NLM_cas:orgname">American Society of Clinical Oncology</span>)
        </div><div class="casAbstract">Purpose Birabresib (MK-8628/OTX015) is a first-in-class bromodomain inhibitor with activity in select hematol. tumors.  Safety, efficacy, and pharmacokinetics of birabresib were evaluated in patients with castrate-resistant prostate cancer, nuclear protein in testis midline carcinoma (NMC), and non-small-cell lung cancer in this phase Ib study.  Patients and Methods Forty-seven patients were enrolled to receive birabresib once daily at starting doses of 80 mg continuously (cohort A) or 100 mg for 7 consecutive days (cohort B) in 21-day cycles using a parallel dose escalation 3 + 3 design.  The primary objective was occurrence of dose-limiting toxicities (DLTs) and detn. of the recommended phase II dose.  Results Of 46 treated patients, 26 had castrate-resistant prostate cancer, 10 NMC, and 10 non-small-cell lung cancer.  For cohort A, four of 19 (21%) evaluable patients had DLTs at 80 mg once daily (grade 3 thrombocytopenia [n = 3], ALT/hyperbilirubinemia [n = 1]) and two of three had DLTs at 100 mg once daily (grade 2 anorexia and nausea with treatment delay > 7 days [n = 1], grade 4 thrombocytopenia [n = 1]).  No DLTs occurred in cohort B.  Of 46 patients, 38 (83%) had treatment-related adverse events (diarrhea, 17 [37%]; nausea, 17 [37%]; anorexia, 14 [30%]; vomiting, 12 [26%]; thrombocytopenia 10 [22%]).  Three patients with NMC (80 mg once daily) had a partial response (Response Evaluation Criteria in Solid Tumors [RECIST] version 1.1) with duration of 1.4 to 8.4 mo.  Pharmacokinetic anal. indicated a dose-proportional increase in birabresib exposure and rapid absorption.  Conclusion The recommended phase II dose of birabresib in patients with select solid tumors is 80 mg once daily with continuous dosing.  Birabresib has dose-proportional exposure and a favorable safety profile, with clin. activity obsd. in NMC.  Future studies of birabresib must consider intermittent scheduling to possibly mitigate the toxicities of chronic dosing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzewa4HJkOK7Vg90H21EOLACvtfcHk0liOShUMrQ83EA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjvFymurg%253D&md5=e602f260adf9fb34b0544fff8599129b</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1200%2FJCO.2018.78.2292&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1200%252FJCO.2018.78.2292%26sid%3Dliteratum%253Aachs%26aulast%3DLewin%26aufirst%3DJ.%26aulast%3DSoria%26aufirst%3DJ.%2BC.%26aulast%3DStathis%26aufirst%3DA.%26aulast%3DDelord%26aufirst%3DJ.%2BP.%26aulast%3DPeters%26aufirst%3DS.%26aulast%3DAwada%26aufirst%3DA.%26aulast%3DAftimos%26aufirst%3DP.%2BG.%26aulast%3DBekradda%26aufirst%3DM.%26aulast%3DRezai%26aufirst%3DK.%26aulast%3DZeng%26aufirst%3DZ.%26aulast%3DHussain%26aufirst%3DA.%26aulast%3DPerez%26aufirst%3DS.%26aulast%3DSiu%26aufirst%3DL.%2BL.%26aulast%3DMassard%26aufirst%3DC.%26atitle%3DPhase%2520Ib%2520Trial%2520With%2520Birabresib%252C%2520a%2520Small-Molecule%2520Inhibitor%2520of%2520Bromodomain%2520and%2520Extraterminal%2520Proteins%252C%2520in%2520Patients%2520With%2520Selected%2520Advanced%2520Solid%2520Tumors%26jtitle%3DJ.%2520Clin.%2520Oncol.%26date%3D2018%26volume%3D36%26spage%3D3007%26epage%3D3014%26doi%3D10.1200%2FJCO.2018.78.2292" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Faivre, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDaniel, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albert, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mantena, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plotnik, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilcox, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bui, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sehgal, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uziel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hessler, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, L. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellin, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mehta, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fidanze, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pratt, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasvold, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panchal, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicolette, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fossey, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Longenecker, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bigelow, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torrent, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenberg, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kati, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span> <span> </span><span class="NLM_article-title">Selective inhibition of the BD2 bromodomain of BET proteins in prostate cancer</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>578</i></span>,  <span class="NLM_fpage">306</span>– <span class="NLM_lpage">310</span>, <span class="refDoi"> DOI: 10.1038/s41586-020-1930-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1038%2Fs41586-020-1930-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=31969702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BB3cXis1Smsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=578&publication_year=2020&pages=306-310&author=E.+J.+Faivreauthor=K.+F.+McDanielauthor=D.+H.+Albertauthor=S.+R.+Mantenaauthor=J.+P.+Plotnikauthor=D.+Wilcoxauthor=L.+Zhangauthor=M.+H.+Buiauthor=G.+S.+Sheppardauthor=L.+Wangauthor=V.+Sehgalauthor=X.+Linauthor=X.+Huangauthor=X.+Luauthor=T.+Uzielauthor=P.+Hesslerauthor=L.+T.+Lamauthor=R.+J.+Bellinauthor=G.+Mehtaauthor=S.+Fidanzeauthor=J.+K.+Prattauthor=D.+Liuauthor=L.+A.+Hasvoldauthor=C.+Sunauthor=S.+C.+Panchalauthor=J.+J.+Nicoletteauthor=S.+L.+Fosseyauthor=C.+H.+Parkauthor=K.+Longeneckerauthor=L.+Bigelowauthor=M.+Torrentauthor=S.+H.+Rosenbergauthor=W.+M.+Katiauthor=Y.+Shen&title=Selective+inhibition+of+the+BD2+bromodomain+of+BET+proteins+in+prostate+cancer&doi=10.1038%2Fs41586-020-1930-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibition of the BD2 bromodomain of BET proteins in prostate cancer</span></div><div class="casAuthors">Faivre, Emily J.; McDaniel, Keith F.; Albert, Daniel H.; Mantena, Srinivasa R.; Plotnik, Joshua P.; Wilcox, Denise; Zhang, Lu; Bui, Mai H.; Sheppard, George S.; Wang, Le; Sehgal, Vasudha; Lin, Xiaoyu; Huang, Xiaoli; Lu, Xin; Uziel, Tamar; Hessler, Paul; Lam, Lloyd T.; Bellin, Richard J.; Mehta, Gaurav; Fidanze, Steve; Pratt, John K.; Liu, Dachun; Hasvold, Lisa A.; Sun, Chaohong; Panchal, Sanjay C.; Nicolette, John J.; Fossey, Stacey L.; Park, Chang H.; Longenecker, Kenton; Bigelow, Lance; Torrent, Maricel; Rosenberg, Saul H.; Kati, Warren M.; Shen, Yu</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">578</span>
        (<span class="NLM_cas:issue">7794</span>),
    <span class="NLM_cas:pages">306-310</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: Proteins of the bromodomain and extra-terminal (BET) domain family are epigenetic readers that bind acetylated histones through their bromodomains to regulate gene transcription.  Dual-bromodomain BET inhibitors (DbBi) that bind with similar affinities to the first (BD1) and second (BD2) bromodomains of BRD2, BRD3, BRD4 and BRDt have displayed modest clin. activity in monotherapy cancer trials.  A reduced no. of thrombocytes in the blood (thrombocytopenia) as well as symptoms of gastrointestinal toxicity are dose-limiting adverse events for some types of DbBi1-5.  Given that similar haematol. and gastrointestinal defects were obsd. after genetic silencing of Brd4 in mice6, the platelet and gastrointestinal toxicities may represent on-target activities assocd. with BET inhibition.  The two individual bromodomains in BET family proteins may have distinct functions7-9 and different cellular phenotypes after pharmacol. inhibition of one or both bromodomains have been reported10,11, suggesting that selectively targeting one of the bromodomains may result in a different efficacy and tolerability profile compared with DbBi.  Available compds. that are selective to individual domains lack sufficient potency and the pharmacokinetics properties that are required for in vivo efficacy and tolerability assessment10-13.  Here we carried out a medicinal chem. campaign that led to the discovery of ABBV-744, a highly potent and selective inhibitor of the BD2 domain of BET family proteins with drug-like properties.  In contrast to the broad range of cell growth inhibition induced by DbBi, the antiproliferative activity of ABBV-744 was largely, but not exclusively, restricted to cell lines of acute myeloid leukemia and prostate cancer that expressed the full-length androgen receptor (AR).  ABBV-744 retained robust activity in prostate cancer xenografts, and showed fewer platelet and gastrointestinal toxicities than the DbBi ABBV-07514.  Analyses of RNA expression and chromatin immunopptn. followed by sequencing revealed that ABBV-744 displaced BRD4 from AR-contg. super-enhancers and inhibited AR-dependent transcription, with less impact on global transcription compared with ABBV-075.  These results underscore the potential value of selectively targeting the BD2 domain of BET family proteins for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrF98kER1y9pLVg90H21EOLACvtfcHk0lj97oEIFdQAVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXis1Smsr0%253D&md5=800d9106c33084390fd8cbf1ac7f1766</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1038%2Fs41586-020-1930-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-020-1930-8%26sid%3Dliteratum%253Aachs%26aulast%3DFaivre%26aufirst%3DE.%2BJ.%26aulast%3DMcDaniel%26aufirst%3DK.%2BF.%26aulast%3DAlbert%26aufirst%3DD.%2BH.%26aulast%3DMantena%26aufirst%3DS.%2BR.%26aulast%3DPlotnik%26aufirst%3DJ.%2BP.%26aulast%3DWilcox%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DBui%26aufirst%3DM.%2BH.%26aulast%3DSheppard%26aufirst%3DG.%2BS.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DSehgal%26aufirst%3DV.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DLu%26aufirst%3DX.%26aulast%3DUziel%26aufirst%3DT.%26aulast%3DHessler%26aufirst%3DP.%26aulast%3DLam%26aufirst%3DL.%2BT.%26aulast%3DBellin%26aufirst%3DR.%2BJ.%26aulast%3DMehta%26aufirst%3DG.%26aulast%3DFidanze%26aufirst%3DS.%26aulast%3DPratt%26aufirst%3DJ.%2BK.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DHasvold%26aufirst%3DL.%2BA.%26aulast%3DSun%26aufirst%3DC.%26aulast%3DPanchal%26aufirst%3DS.%2BC.%26aulast%3DNicolette%26aufirst%3DJ.%2BJ.%26aulast%3DFossey%26aufirst%3DS.%2BL.%26aulast%3DPark%26aufirst%3DC.%2BH.%26aulast%3DLongenecker%26aufirst%3DK.%26aulast%3DBigelow%26aufirst%3DL.%26aulast%3DTorrent%26aufirst%3DM.%26aulast%3DRosenberg%26aufirst%3DS.%2BH.%26aulast%3DKati%26aufirst%3DW.%2BM.%26aulast%3DShen%26aufirst%3DY.%26atitle%3DSelective%2520inhibition%2520of%2520the%2520BD2%2520bromodomain%2520of%2520BET%2520proteins%2520in%2520prostate%2520cancer%26jtitle%3DNature%26date%3D2020%26volume%3D578%26spage%3D306%26epage%3D310%26doi%3D10.1038%2Fs41586-020-1930-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abramson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blum, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flinn, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gutierrez, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maris, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Meara, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borger, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mertz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sims, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeffrey, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Younes, A.</span></span> <span> </span><span class="NLM_article-title">BET Inhibitor CPI-0610 Is Well Tolerated and Induces Responses in Diffuse Large B-Cell Lymphoma and Follicular Lymphoma: Preliminary Analysis of an Ongoing Phase 1 Study</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>126</i></span>,  <span class="NLM_fpage">1491</span>– <span class="NLM_lpage">1491</span>, <span class="refDoi"> DOI: 10.1182/blood.V126.23.1491.1491</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1182%2Fblood.V126.23.1491.1491" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=126&publication_year=2015&pages=1491-1491&author=J.+S.+Abramsonauthor=K.+A.+Blumauthor=I.+W.+Flinnauthor=M.+Gutierrezauthor=A.+Goyauthor=M.+Marisauthor=M.+Cooperauthor=M.+O%E2%80%99Mearaauthor=D.+Borgerauthor=J.+Mertzauthor=R.+J.+Simsauthor=S.+Jeffreyauthor=A.+Younes&title=BET+Inhibitor+CPI-0610+Is+Well+Tolerated+and+Induces+Responses+in+Diffuse+Large+B-Cell+Lymphoma+and+Follicular+Lymphoma%3A+Preliminary+Analysis+of+an+Ongoing+Phase+1+Study&doi=10.1182%2Fblood.V126.23.1491.1491"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1182%2Fblood.V126.23.1491.1491&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood.V126.23.1491.1491%26sid%3Dliteratum%253Aachs%26aulast%3DAbramson%26aufirst%3DJ.%2BS.%26aulast%3DBlum%26aufirst%3DK.%2BA.%26aulast%3DFlinn%26aufirst%3DI.%2BW.%26aulast%3DGutierrez%26aufirst%3DM.%26aulast%3DGoy%26aufirst%3DA.%26aulast%3DMaris%26aufirst%3DM.%26aulast%3DCooper%26aufirst%3DM.%26aulast%3DO%25E2%2580%2599Meara%26aufirst%3DM.%26aulast%3DBorger%26aufirst%3DD.%26aulast%3DMertz%26aufirst%3DJ.%26aulast%3DSims%26aufirst%3DR.%2BJ.%26aulast%3DJeffrey%26aufirst%3DS.%26aulast%3DYounes%26aufirst%3DA.%26atitle%3DBET%2520Inhibitor%2520CPI-0610%2520Is%2520Well%2520Tolerated%2520and%2520Induces%2520Responses%2520in%2520Diffuse%2520Large%2520B-Cell%2520Lymphoma%2520and%2520Follicular%2520Lymphoma%253A%2520Preliminary%2520Analysis%2520of%2520an%2520Ongoing%2520Phase%25201%2520Study%26jtitle%3DBlood%26date%3D2015%26volume%3D126%26spage%3D1491%26epage%3D1491%26doi%3D10.1182%2Fblood.V126.23.1491.1491" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Odenike, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolff, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borthakur, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aldoss, I. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rizzieri, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prebet, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dinh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Modi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freise, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jonas, B. A.</span></span> <span> </span><span class="NLM_article-title">Results from the first-in-human study of mivebresib (ABBV-075), a pan-inhibitor of bromodomain and extra terminal proteins, in patients with relapsed/refractory acute myeloid leukemia</span>. <i>J. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">7030</span>– <span class="NLM_lpage">7030</span>, <span class="refDoi"> DOI: 10.1200/JCO.2019.37.15_suppl.7030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1200%2FJCO.2019.37.15_suppl.7030" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2019&pages=7030-7030&author=O.+Odenikeauthor=J.+E.+Wolffauthor=G.+Borthakurauthor=I.+T.+Aldossauthor=D.+Rizzieriauthor=T.+Prebetauthor=B.+Huauthor=M.+Dinhauthor=X.+Chenauthor=D.+Modiauthor=K.+J.+Freiseauthor=B.+A.+Jonas&title=Results+from+the+first-in-human+study+of+mivebresib+%28ABBV-075%29%2C+a+pan-inhibitor+of+bromodomain+and+extra+terminal+proteins%2C+in+patients+with+relapsed%2Frefractory+acute+myeloid+leukemia&doi=10.1200%2FJCO.2019.37.15_suppl.7030"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1200%2FJCO.2019.37.15_suppl.7030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1200%252FJCO.2019.37.15_suppl.7030%26sid%3Dliteratum%253Aachs%26aulast%3DOdenike%26aufirst%3DO.%26aulast%3DWolff%26aufirst%3DJ.%2BE.%26aulast%3DBorthakur%26aufirst%3DG.%26aulast%3DAldoss%26aufirst%3DI.%2BT.%26aulast%3DRizzieri%26aufirst%3DD.%26aulast%3DPrebet%26aufirst%3DT.%26aulast%3DHu%26aufirst%3DB.%26aulast%3DDinh%26aufirst%3DM.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DModi%26aufirst%3DD.%26aulast%3DFreise%26aufirst%3DK.%2BJ.%26aulast%3DJonas%26aufirst%3DB.%2BA.%26atitle%3DResults%2520from%2520the%2520first-in-human%2520study%2520of%2520mivebresib%2520%2528ABBV-075%2529%252C%2520a%2520pan-inhibitor%2520of%2520bromodomain%2520and%2520extra%2520terminal%2520proteins%252C%2520in%2520patients%2520with%2520relapsed%252Frefractory%2520acute%2520myeloid%2520leukemia%26jtitle%3DJ.%2520Clin.%2520Oncol.%26date%3D2019%26volume%3D37%26spage%3D7030%26epage%3D7030%26doi%3D10.1200%2FJCO.2019.37.15_suppl.7030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gutgesell, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karumudi, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tonetti, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thatcher, G. R.</span></span> <span> </span><span class="NLM_article-title">Novel Selective Estrogen Receptor Downregulators (SERDs) Developed against Treatment-Resistant Breast Cancer</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">1325</span>– <span class="NLM_lpage">1342</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01355</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01355" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFWqu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=1325-1342&author=R.+Xiongauthor=J.+Zhaoauthor=L.+M.+Gutgesellauthor=Y.+Wangauthor=S.+Leeauthor=B.+Karumudiauthor=H.+Zhaoauthor=Y.+Luauthor=D.+A.+Tonettiauthor=G.+R.+Thatcher&title=Novel+Selective+Estrogen+Receptor+Downregulators+%28SERDs%29+Developed+against+Treatment-Resistant+Breast+Cancer&doi=10.1021%2Facs.jmedchem.6b01355"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Selective Estrogen Receptor Downregulators (SERDs) Developed against Treatment-Resistant Breast Cancer</span></div><div class="casAuthors">Xiong, Rui; Zhao, Jiong; Gutgesell, Lauren M.; Wang, Yueting; Lee, Sue; Karumudi, Bhargava; Zhao, Huiping; Lu, Yunlong; Tonetti, Debra A.; Thatcher, Gregory R. J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1325-1342</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Resistance to the selective estrogen receptor modulator (SERM) tamoxifen and to aromatase inhibitors that lower circulating estradiol occurs in up to 50% of patients, generally leading to an endocrine-independent ER+ phenotype.  Selective ER downregulators (SERDs) are able to ablate ER and thus theor. to prevent survival of both endocrine-dependent and independent ER+ tumors.  The clin. SERD, fulvestrant, is hampered by i.m. administration and undesirable pharmacokinetics.  Novel SERDs were designed using the 6-OH-benzothiophene (BT) scaffold common to arzoxifene and raloxifene.  Treatment-resistant (TR) ER+ cell lines (MCF-7:5C and MCF-7:TAM1) were used for optimization, followed by validation in the parent endocrine-dependent cell line (MCF-7:WS8), in 2D and 3D cultures, using ERα in-cell westerns, ERE-luciferase, and cell viability assays, with GDC-0810 (ARN-810) used for comparison.  Two BT SERDs with superior in vitro activity to GDC-0810 were studied for bioavailability and shown to cause regression of a TR, endocrine-independent ER+ xenograft superior to GDC-0810.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLqe0A8df78rVg90H21EOLACvtfcHk0lj97oEIFdQAVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFWqu70%253D&md5=71dbdfa2152f9e9019e8d138ed41313e</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01355&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01355%26sid%3Dliteratum%253Aachs%26aulast%3DXiong%26aufirst%3DR.%26aulast%3DZhao%26aufirst%3DJ.%26aulast%3DGutgesell%26aufirst%3DL.%2BM.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DLee%26aufirst%3DS.%26aulast%3DKarumudi%26aufirst%3DB.%26aulast%3DZhao%26aufirst%3DH.%26aulast%3DLu%26aufirst%3DY.%26aulast%3DTonetti%26aufirst%3DD.%2BA.%26aulast%3DThatcher%26aufirst%3DG.%2BR.%26atitle%3DNovel%2520Selective%2520Estrogen%2520Receptor%2520Downregulators%2520%2528SERDs%2529%2520Developed%2520against%2520Treatment-Resistant%2520Breast%2520Cancer%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D1325%26epage%3D1342%26doi%3D10.1021%2Facs.jmedchem.6b01355" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pluard, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliveira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cescon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan-Chiu, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carpenter, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cunningham, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sparano, J.</span></span> <span> </span><span class="NLM_article-title">Abstract OT3-06-07: A phase I/II dose escalation and expansion study to investigate the safety, pharmacokinetics, pharmacodynamics and clinical activity of GSK525762 in combination with fulvestrant in subjects with ER+ breast cancer</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>78</i></span>,  <span class="NLM_fpage">OT3-06-07</span>, <span class="refDoi"> DOI: 10.1158/1538-7445.SABCS17-OT3-06-07</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1158%2F1538-7445.SABCS17-OT3-06-07" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=2018&pages=OT3-06-07&author=T.+Pluardauthor=S.+Ohauthor=M.+Oliveiraauthor=D.+Cesconauthor=E.+Tan-Chiuauthor=Y.+Wuauthor=C.+Carpenterauthor=E.+Cunninghamauthor=M.+Ballasauthor=A.+Dharauthor=J.+Sparano&title=Abstract+OT3-06-07%3A+A+phase+I%2FII+dose+escalation+and+expansion+study+to+investigate+the+safety%2C+pharmacokinetics%2C+pharmacodynamics+and+clinical+activity+of+GSK525762+in+combination+with+fulvestrant+in+subjects+with+ER%2B+breast+cancer&doi=10.1158%2F1538-7445.SABCS17-OT3-06-07"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1158%2F1538-7445.SABCS17-OT3-06-07&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1538-7445.SABCS17-OT3-06-07%26sid%3Dliteratum%253Aachs%26aulast%3DPluard%26aufirst%3DT.%26aulast%3DOh%26aufirst%3DS.%26aulast%3DOliveira%26aufirst%3DM.%26aulast%3DCescon%26aufirst%3DD.%26aulast%3DTan-Chiu%26aufirst%3DE.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DCarpenter%26aufirst%3DC.%26aulast%3DCunningham%26aufirst%3DE.%26aulast%3DBallas%26aufirst%3DM.%26aulast%3DDhar%26aufirst%3DA.%26aulast%3DSparano%26aufirst%3DJ.%26atitle%3DAbstract%2520OT3-06-07%253A%2520A%2520phase%2520I%252FII%2520dose%2520escalation%2520and%2520expansion%2520study%2520to%2520investigate%2520the%2520safety%252C%2520pharmacokinetics%252C%2520pharmacodynamics%2520and%2520clinical%2520activity%2520of%2520GSK525762%2520in%2520combination%2520with%2520fulvestrant%2520in%2520subjects%2520with%2520ER%252B%2520breast%2520cancer%26jtitle%3DCancer%2520Res.%26date%3D2018%26volume%3D78%26spage%3DOT3-06-07%26doi%3D10.1158%2F1538-7445.SABCS17-OT3-06-07" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mirguet, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gosmini, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toum, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clément, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnathan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brusq, J.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mordaunt, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimes, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crowe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pineau, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ajakane, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daugan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeffrey, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cutler, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haynes, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smithers, N. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uings, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witherington, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parr, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicodème, E.</span></span> <span> </span><span class="NLM_article-title">Discovery of Epigenetic Regulator I-BET762: Lead Optimization to Afford a Clinical Candidate Inhibitor of the BET Bromodomains</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">7501</span>– <span class="NLM_lpage">7515</span>, <span class="refDoi"> DOI: 10.1021/jm401088k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401088k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVSls7jJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=7501-7515&author=O.+Mirguetauthor=R.+Gosminiauthor=J.+Toumauthor=C.+A.+Cl%C3%A9mentauthor=M.+Barnathanauthor=J.-M.+Brusqauthor=J.+E.+Mordauntauthor=R.+M.+Grimesauthor=M.+Croweauthor=O.+Pineauauthor=M.+Ajakaneauthor=A.+Dauganauthor=P.+Jeffreyauthor=L.+Cutlerauthor=A.+C.+Haynesauthor=N.+N.+Smithersauthor=C.-w.+Chungauthor=P.+Bamboroughauthor=I.+J.+Uingsauthor=A.+Lewisauthor=J.+Witheringtonauthor=N.+Parrauthor=R.+K.+Prinjhaauthor=E.+Nicod%C3%A8me&title=Discovery+of+Epigenetic+Regulator+I-BET762%3A+Lead+Optimization+to+Afford+a+Clinical+Candidate+Inhibitor+of+the+BET+Bromodomains&doi=10.1021%2Fjm401088k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Epigenetic Regulator I-BET762: Lead Optimization to Afford a Clinical Candidate Inhibitor of the BET Bromodomains</span></div><div class="casAuthors">Mirguet, Olivier; Gosmini, Romain; Toum, Jerome; Clement, Catherine A.; Barnathan, Melanie; Brusq, Jean-Marie; Mordaunt, Jacqueline E.; Grimes, Richard M.; Crowe, Miriam; Pineau, Olivier; Ajakane, Myriam; Daugan, Alain; Jeffrey, Phillip; Cutler, Leanne; Haynes, Andrea C.; Smithers, Nicholas N.; Chung, Chun-wa; Bamborough, Paul; Uings, Iain J.; Lewis, Antonia; Witherington, Jason; Parr, Nigel; Prinjha, Rab K.; Nicodeme, Edwige</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">7501-7515</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bromo and extra C-terminal domain (BET) family of bromodomains are involved in binding epigenetic marks on histone proteins, more specifically acetylated lysine residues.  This paper describes the discovery and structure-activity relationships (SAR) of potent benzodiazepine inhibitors that disrupt the function of the BET family of bromodomains (BRD2, BRD3, and BRD4).  This work has yielded a potent, selective compd. I-BET762 (I) that is now under evaluation in a phase I/II clin. trial for nuclear protein in testis (NUT) midline carcinoma and other cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDitaXJSnFE7Vg90H21EOLACvtfcHk0lgBuiI_8cBD1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVSls7jJ&md5=5d059b0be08a74a5721b72a120f7e090</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1021%2Fjm401088k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401088k%26sid%3Dliteratum%253Aachs%26aulast%3DMirguet%26aufirst%3DO.%26aulast%3DGosmini%26aufirst%3DR.%26aulast%3DToum%26aufirst%3DJ.%26aulast%3DCl%25C3%25A9ment%26aufirst%3DC.%2BA.%26aulast%3DBarnathan%26aufirst%3DM.%26aulast%3DBrusq%26aufirst%3DJ.-M.%26aulast%3DMordaunt%26aufirst%3DJ.%2BE.%26aulast%3DGrimes%26aufirst%3DR.%2BM.%26aulast%3DCrowe%26aufirst%3DM.%26aulast%3DPineau%26aufirst%3DO.%26aulast%3DAjakane%26aufirst%3DM.%26aulast%3DDaugan%26aufirst%3DA.%26aulast%3DJeffrey%26aufirst%3DP.%26aulast%3DCutler%26aufirst%3DL.%26aulast%3DHaynes%26aufirst%3DA.%2BC.%26aulast%3DSmithers%26aufirst%3DN.%2BN.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DUings%26aufirst%3DI.%2BJ.%26aulast%3DLewis%26aufirst%3DA.%26aulast%3DWitherington%26aufirst%3DJ.%26aulast%3DParr%26aufirst%3DN.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DNicod%25C3%25A8me%26aufirst%3DE.%26atitle%3DDiscovery%2520of%2520Epigenetic%2520Regulator%2520I-BET762%253A%2520Lead%2520Optimization%2520to%2520Afford%2520a%2520Clinical%2520Candidate%2520Inhibitor%2520of%2520the%2520BET%2520Bromodomains%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D7501%26epage%3D7515%26doi%3D10.1021%2Fjm401088k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref70"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref70'); return false;" data-citation="" class="refNumLink">70</a></strong><div class="NLM_citation" id="cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hanahan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinberg, R. A.</span></span> <span> </span><span class="NLM_article-title">Hallmarks of cancer: the next generation</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>144</i></span>,  <span class="NLM_fpage">646</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2011.02.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1016%2Fj.cell.2011.02.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=21376230" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjsFeqtrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=144&publication_year=2011&pages=646-674&author=D.+Hanahanauthor=R.+A.+Weinberg&title=Hallmarks+of+cancer%3A+the+next+generation&doi=10.1016%2Fj.cell.2011.02.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Hallmarks of cancer: the next generation</span></div><div class="casAuthors">Hanahan, Douglas; Weinberg, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">144</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">646-674</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The hallmarks of cancer comprise six biol. capabilities acquired during the multistep development of human tumors.  The hallmarks constitute an organizing principle for rationalizing the complexities of neoplastic disease.  They include sustaining proliferative signaling, evading growth suppressors, resisting cell death, enabling replicative immortality, inducing angiogenesis, and activating invasion and metastasis.  Underlying these hallmarks are genome instability, which generates the genetic diversity that expedites their acquisition, and inflammation, which fosters multiple hallmark functions.  Conceptual progress in the last decade has added two emerging hallmarks of potential generality to this list-reprogramming of energy metab. and evading immune destruction.  In addn. to cancer cells, tumors exhibit another dimension of complexity: they contain a repertoire of recruited, ostensibly normal cells that contribute to the acquisition of hallmark traits by creating the "tumor microenvironment.".  Recognition of the widespread applicability of these concepts will increasingly affect the development of new means to treat human cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTcpqXALcb6LVg90H21EOLACvtfcHk0lgBuiI_8cBD1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjsFeqtrk%253D&md5=b36f160c417a712e83cb83c577f0018e</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2011.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2011.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DHanahan%26aufirst%3DD.%26aulast%3DWeinberg%26aufirst%3DR.%2BA.%26atitle%3DHallmarks%2520of%2520cancer%253A%2520the%2520next%2520generation%26jtitle%3DCell%26date%3D2011%26volume%3D144%26spage%3D646%26epage%3D674%26doi%3D10.1016%2Fj.cell.2011.02.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref71"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref71'); return false;" data-citation="" class="refNumLink">71</a></strong><div class="NLM_citation" id="cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mantena, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dinh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mittelstadt, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kati, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klapczynski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDaniel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koeniger, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolff, J. E.</span></span> <span> </span><span class="NLM_article-title">Preclinical evidence for management of thrombocytopenia associated with bromodomain extra-terminal (BET) inhibition therapy</span>. <i>J. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>36</i></span>, <span class="NLM_elocation-id">e24207</span> <span class="refDoi"> DOI: 10.1200/JCO.2018.36.15_suppl.e24207</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1200%2FJCO.2018.36.15_suppl.e24207" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2018&author=S.+R.+Mantenaauthor=M.+Dinhauthor=S.+Mittelstadtauthor=W.+R.+Katiauthor=M.+Klapczynskiauthor=K.+McDanielauthor=S.+Koenigerauthor=J.+E.+Wolff&title=Preclinical+evidence+for+management+of+thrombocytopenia+associated+with+bromodomain+extra-terminal+%28BET%29+inhibition+therapy&doi=10.1200%2FJCO.2018.36.15_suppl.e24207"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1200%2FJCO.2018.36.15_suppl.e24207&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1200%252FJCO.2018.36.15_suppl.e24207%26sid%3Dliteratum%253Aachs%26aulast%3DMantena%26aufirst%3DS.%2BR.%26aulast%3DDinh%26aufirst%3DM.%26aulast%3DMittelstadt%26aufirst%3DS.%26aulast%3DKati%26aufirst%3DW.%2BR.%26aulast%3DKlapczynski%26aufirst%3DM.%26aulast%3DMcDaniel%26aufirst%3DK.%26aulast%3DKoeniger%26aufirst%3DS.%26aulast%3DWolff%26aufirst%3DJ.%2BE.%26atitle%3DPreclinical%2520evidence%2520for%2520management%2520of%2520thrombocytopenia%2520associated%2520with%2520bromodomain%2520extra-terminal%2520%2528BET%2529%2520inhibition%2520therapy%26jtitle%3DJ.%2520Clin.%2520Oncol.%26date%3D2018%26volume%3D36%26doi%3D10.1200%2FJCO.2018.36.15_suppl.e24207" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref72"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref72'); return false;" data-citation="" class="refNumLink">72</a></strong><div class="NLM_citation" id="cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meeke, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osipo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kidawi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandel, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jordan, V. C.</span></span> <span> </span><span class="NLM_article-title">Intrinsic mechanism of estradiol-induced apoptosis in breast cancer cells resistant to estrogen deprivation</span>. <i>J. Natl. Cancer Inst.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>97</i></span>,  <span class="NLM_fpage">1746</span>– <span class="NLM_lpage">1759</span>, <span class="refDoi"> DOI: 10.1093/jnci/dji400</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1093%2Fjnci%2Fdji400" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=16333030" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlSjurnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=97&publication_year=2005&pages=1746-1759&author=J.+S.+Lewisauthor=K.+Meekeauthor=C.+Osipoauthor=E.+A.+Rossauthor=N.+Kidawiauthor=T.+Liauthor=E.+Bellauthor=N.+S.+Chandelauthor=V.+C.+Jordan&title=Intrinsic+mechanism+of+estradiol-induced+apoptosis+in+breast+cancer+cells+resistant+to+estrogen+deprivation&doi=10.1093%2Fjnci%2Fdji400"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Intrinsic Mechanism of Estradiol-Induced Apoptosis in Breast Cancer Cells Resistant to Estrogen Deprivation</span></div><div class="casAuthors">Lewis, Joan S.; Meeke, Kathleen; Osipo, Clodia; Ross, Eric A.; Kidawi, Noman; Li, Tianyu; Bell, Eric; Chandel, Navdeep S.; Jordan, V. Craig</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the National Cancer Institute</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">97</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">1746-1759</span>CODEN:
                <span class="NLM_cas:coden">JNCIEQ</span>;
        ISSN:<span class="NLM_cas:issn">0027-8874</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Background: We previously developed an estrogen receptor (ER)-pos. breast cancer cell line (MCF-7:5C) that is resistant to long-term estrogen deprivation and undergoes rapid and complete apoptosis in the presence of physiol. concns. of 17β-estradiol.  Here, we investigated the role of the mitochondrial apoptotic pathway in this process.  Methods: Apoptosis in MCF-7:5C cells treated with estradiol, fulvestrant, or vehicle (control) was investigated by annexin V-propidium iodide double staining and 4',6-diamidino-2-phenylindole (DAPI) staining.  Apoptosis was also analyzed in MCF-7:5C cells transiently transfected with small interfering RNAs (siRNAs) to apoptotic pathway components.  Expression of apoptotic pathway intermediates was measured by western blot anal.  Mitochondrial transmembrane potential (ψm) was detd. by rhodamine-123 retention assay.  Mitochondrial pathway activity was detd. by cytochrome c release and cleavage of poly(ADP-ribose) polymerase (PARP) protein.  Tumorigenesis was studied in ovariectomized athymic mice that were injected with MCF-7:5C cells.  Differences between the treatment groups and control group were detd. by two-sample t test or one-factor anal. of variance.  All statistical tests were two-sided.  Results: MCF-7:5C cells treated with estradiol underwent apoptosis and showed increased expression of proapoptotic proteins, decreased ψm, enhanced cytochrome c release, and PARP cleavage compared with cells treated with fulvestrant or vehicle.  Blockade of Bax, Bim, and p53 mRNA expression by siRNA reduced estradiol-induced apoptosis relative to control by 76% [95% confidence interval (CI) = 73% to 79%, P<.001], 85% [95% CI = 90% to 80%, P<.001], and 40% [95% CI = 45% to 35%, P<.001], resp., whereas blockade of FasL by siRNA had no effect.  Estradiol caused complete regression of MCF-7:5C tumors in vivo.  Conclusion: The mitochondrial pathway of apoptosis plays a crit. role in estradiol-induced apoptosis in long-term estrogen-deprived breast cancer cells.  Physiol. concns. of estradiol could potentially be used to induce apoptosis and tumor regression in tumors that have developed resistance to aromatase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0fGAoI50tabVg90H21EOLACvtfcHk0lje1zNlyJveEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlSjurnN&md5=8107a7d125731549936a7268fb9e1138</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1093%2Fjnci%2Fdji400&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjnci%252Fdji400%26sid%3Dliteratum%253Aachs%26aulast%3DLewis%26aufirst%3DJ.%2BS.%26aulast%3DMeeke%26aufirst%3DK.%26aulast%3DOsipo%26aufirst%3DC.%26aulast%3DRoss%26aufirst%3DE.%2BA.%26aulast%3DKidawi%26aufirst%3DN.%26aulast%3DLi%26aufirst%3DT.%26aulast%3DBell%26aufirst%3DE.%26aulast%3DChandel%26aufirst%3DN.%2BS.%26aulast%3DJordan%26aufirst%3DV.%2BC.%26atitle%3DIntrinsic%2520mechanism%2520of%2520estradiol-induced%2520apoptosis%2520in%2520breast%2520cancer%2520cells%2520resistant%2520to%2520estrogen%2520deprivation%26jtitle%3DJ.%2520Natl.%2520Cancer%2520Inst.%26date%3D2005%26volume%3D97%26spage%3D1746%26epage%3D1759%26doi%3D10.1093%2Fjnci%2Fdji400" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref73"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref73'); return false;" data-citation="" class="refNumLink">73</a></strong><div class="NLM_citation" id="cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osipo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meeke, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jordan, V. C.</span></span> <span> </span><span class="NLM_article-title">Estrogen-induced apoptosis in a breast cancer model resistant to long-term estrogen withdrawal</span>. <i>J. Steroid Biochem. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>94</i></span>,  <span class="NLM_fpage">131</span>– <span class="NLM_lpage">141</span>, <span class="refDoi"> DOI: 10.1016/j.jsbmb.2004.12.032</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1016%2Fj.jsbmb.2004.12.032" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=15862958" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BD2MXjvVKks74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=2005&pages=131-141&author=J.+S.+Lewisauthor=C.+Osipoauthor=K.+Meekeauthor=V.+C.+Jordan&title=Estrogen-induced+apoptosis+in+a+breast+cancer+model+resistant+to+long-term+estrogen+withdrawal&doi=10.1016%2Fj.jsbmb.2004.12.032"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">Estrogen-induced apoptosis in a breast cancer model resistant to long-term estrogen withdrawal</span></div><div class="casAuthors">Lewis, J. S.; Osipo, C.; Meeke, K.; Jordan, V. C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Steroid Biochemistry and Molecular Biology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">94</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">131-141</span>CODEN:
                <span class="NLM_cas:coden">JSBBEZ</span>;
        ISSN:<span class="NLM_cas:issn">0960-0760</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Estrogen suppression through the use of an aromatase inhibitor is an effective endocrine treatment option for postmenopausal breast cancer patients with estrogen receptor (ER)-pos. disease, however, there are concerns that long-term estrogen deprivation will inevitably lead to resistance.  To address the issue of acquired resistance to long-term estrogen deprivation our lab. has developed an ER+/PR- hormone-independent breast cancer cell line, MCF-7:5C which is a variant clone of wild-type MCF-7 cells.  Originally, these cells were cultured in estrogen-free MEM contg. 5% charcoal-stripped calf serum and were found to be resistant to both estradiol (E2) and antiestrogens.  Interestingly, a completely different phenomenon was obsd. when MCF-7:5C cells were cultured in phenol red-free RPMI 1640 medium contg. 10% charcoal-stripped fetal bovine serum (SFS).  Using DNA quantitation assays, the authors examd. the effect of E2 on the growth of MCF-7:5C cells under different media conditions.  These results showed that 10-9 M E2 caused a dramatic 90% redn. in the growth of MCF-7:5C cells cultured in RPMI medium contg. 10% SFS but did not have any significant inhibitory effects on cells cultured in MEM media.  Addnl. expts. were performed to det. whether the medium or the serum facilitated the inhibitory effects of E2 and the results indicated that it was the serum.  Annexin V and DAPI staining confirmed that the E2-induced growth inhibition of MCF-7:5C cells was due to apoptosis.  We also examd. the tumorigenic potential of MCF-7:5C cells by injecting 1 × 107 cells/site into ovariectomized athymic mice and found that these cells, previously cultured in RPMI media, spontaneously grew into tumors in the absence of E2.  Overall, these results show that low concns. (>10-11 M) of E2 are capable of inducing apoptosis in an aromatase resistant breast cancer cell model and that this effect is highly influenced by the medium in which the cells are grown.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhS-VZhkUY2rVg90H21EOLACvtfcHk0lje1zNlyJveEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXjvVKks74%253D&md5=5b11562cc6ecd12fc4056f8446f9f47f</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1016%2Fj.jsbmb.2004.12.032&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jsbmb.2004.12.032%26sid%3Dliteratum%253Aachs%26aulast%3DLewis%26aufirst%3DJ.%2BS.%26aulast%3DOsipo%26aufirst%3DC.%26aulast%3DMeeke%26aufirst%3DK.%26aulast%3DJordan%26aufirst%3DV.%2BC.%26atitle%3DEstrogen-induced%2520apoptosis%2520in%2520a%2520breast%2520cancer%2520model%2520resistant%2520to%2520long-term%2520estrogen%2520withdrawal%26jtitle%3DJ.%2520Steroid%2520Biochem.%2520Mol.%2520Biol.%26date%3D2005%26volume%3D94%26spage%3D131%26epage%3D141%26doi%3D10.1016%2Fj.jsbmb.2004.12.032" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref74"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref74'); return false;" data-citation="" class="refNumLink">74</a></strong><div class="NLM_citation" id="cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Livezey, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shapiro, D. J.</span></span> <span> </span><span class="NLM_article-title">Antiestrogen Resistant Cell Lines Expressing Estrogen Receptor α Mutations Upregulate the Unfolded Protein Response and are Killed by BHPI</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">34753</span>, <span class="refDoi"> DOI: 10.1038/srep34753</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1038%2Fsrep34753" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=27713477" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1Kmu7zF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&pages=34753&author=C.+Maoauthor=M.+Livezeyauthor=J.+E.+Kimauthor=D.+J.+Shapiro&title=Antiestrogen+Resistant+Cell+Lines+Expressing+Estrogen+Receptor+%CE%B1+Mutations+Upregulate+the+Unfolded+Protein+Response+and+are+Killed+by+BHPI&doi=10.1038%2Fsrep34753"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">Antiestrogen Resistant Cell Lines Expressing Estrogen Receptor α Mutations Upregulate the Unfolded Protein Response and are Killed by BHPI</span></div><div class="casAuthors">Mao, Chengjian; Livezey, Mara; Kim, Ji Eun; Shapiro, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">34753</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Outgrowth of metastases expressing ERα mutations Y537S and D538G is common after endocrine therapy for estrogen receptor α (ERα) pos. breast cancer.  The effect of replacing wild type ERα in breast cancer cells with these mutations was unclear.  We used the CRISPR-Cas9 genome editing system and homol. directed repair to isolate and characterize 14 T47D cell lines in which ERαY537S or ERαD538G replace one or both wild-type ERα genes.  In 2-dimensional, and in quant. anchorage-independent 3-dimensional cell culture, ERαY537S and ERαD538G cells exhibited estrogen-independent growth.  A progestin further increased their already substantial proliferation in micromolar 4-hydroxytamoxifen and fulvestrant/ICI 182,780 (ICI).  Our recently described ERα biomodulator, BHPI, which hyperactivates the unfolded protein response (UPR), completely blocked proliferation.  In ERαY537S and ERαD538G cells, estrogen-ERα target genes were constitutively active and partially antiestrogen resistant.  The UPR marker sp-XBP1 was constitutively activated in ERαY537S cells and further induced by progesterone in both cell lines.  UPR-regulated genes assocd. with tamoxifen resistance, including the oncogenic chaperone BiP/GRP78, were upregulated.  ICI displayed a greater than 2 fold redn. in its ability to induce ERαY537S and ERαD538G degrdn.  Progestins, UPR activation and perhaps reduced ICI-stimulated ERα degrdn. likely contribute to antiestrogen resistance seen in ERαY537S and ERαD538G cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnk6TbVl3BX7Vg90H21EOLACvtfcHk0lje1zNlyJveEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1Kmu7zF&md5=a823b3cbf233db08a026b76f615424e4</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1038%2Fsrep34753&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep34753%26sid%3Dliteratum%253Aachs%26aulast%3DMao%26aufirst%3DC.%26aulast%3DLivezey%26aufirst%3DM.%26aulast%3DKim%26aufirst%3DJ.%2BE.%26aulast%3DShapiro%26aufirst%3DD.%2BJ.%26atitle%3DAntiestrogen%2520Resistant%2520Cell%2520Lines%2520Expressing%2520Estrogen%2520Receptor%2520%25CE%25B1%2520Mutations%2520Upregulate%2520the%2520Unfolded%2520Protein%2520Response%2520and%2520are%2520Killed%2520by%2520BHPI%26jtitle%3DSci.%2520Rep.%26date%3D2016%26volume%3D6%26spage%3D34753%26doi%3D10.1038%2Fsrep34753" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref75"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref75'); return false;" data-citation="" class="refNumLink">75</a></strong><div class="NLM_citation" id="cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">Integration, scaling, space-group assignment and post-refinement</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">133</span>– <span class="NLM_lpage">144</span>, <span class="refDoi"> DOI: 10.1107/S0907444909047374</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1107%2FS0907444909047374" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=20124693" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=133-144&author=W.+Kabsch&title=Integration%2C+scaling%2C+space-group+assignment+and+post-refinement&doi=10.1107%2FS0907444909047374"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">Integration, scaling, space-group assignment and post-refinement</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">133-144</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Important steps in the processing of rotation data are described that are common to most software packages.  These programs differ in the details and in the methods implemented to carry out the tasks.  Here, the working principles underlying the data-redn. package XDS are explained, including the new features of automatic detn. of spot size and reflecting range, recognition and assignment of crystal symmetry and a highly efficient algorithm for the detn. of correction/scaling factors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGW7t3Bb1P_rVg90H21EOLACvtfcHk0ljBPYjB8jTcoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisb4%253D&md5=eb1db34f3c959b4921771eb9f57b2159</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047374&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047374%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DIntegration%252C%2520scaling%252C%2520space-group%2520assignment%2520and%2520post-refinement%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D133%26epage%3D144%26doi%3D10.1107%2FS0907444909047374" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref76"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref76'); return false;" data-citation="" class="refNumLink">76</a></strong><div class="NLM_citation" id="cit76"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teplyakov, A.</span></span> <span> </span><span class="NLM_article-title">Molecular replacement with MOLREP</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">22</span>– <span class="NLM_lpage">25</span>, <span class="refDoi"> DOI: 10.1107/S0907444909042589</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1107%2FS0907444909042589" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=20057045" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC3cXit1Kktw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=22-25&author=A.+Vaginauthor=A.+Teplyakov&title=Molecular+replacement+with+MOLREP&doi=10.1107%2FS0907444909042589"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular replacement with MOLREP</span></div><div class="casAuthors">Vagin, Alexei; Teplyakov, Alexei</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">22-25</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">MOLREP is an automated program for mol. replacement that utilizes a no. of original approaches to rotational and translational search and data prepn.  Since the first publication describing the program, MOLREP has acquired a variety of features that include weighting of the X-ray data and search models, multi-copy search, fitting the model into electron d., structural superposition of two models and rigid-body refinement.  The program can run in a fully automatic mode using optimized parameters calcd. from the input data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzj5SzYvyu_7Vg90H21EOLACvtfcHk0ljBPYjB8jTcoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXit1Kktw%253D%253D&md5=820d114719aca209994ffb0403e3b20d</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1107%2FS0907444909042589&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909042589%26sid%3Dliteratum%253Aachs%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DTeplyakov%26aufirst%3DA.%26atitle%3DMolecular%2520replacement%2520with%2520MOLREP%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D22%26epage%3D25%26doi%3D10.1107%2FS0907444909042589" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref77"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref77'); return false;" data-citation="" class="refNumLink">77</a></strong><div class="NLM_citation" id="cit77"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Potterton, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Briggs, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turkenburg, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E.</span></span> <span> </span><span class="NLM_article-title">A graphical user interface to the CCP4 program suite</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1131</span>– <span class="NLM_lpage">1137</span>, <span class="refDoi"> DOI: 10.1107/S0907444903008126</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1107%2FS0907444903008126" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=12832755" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BD3sXkvFSksLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2003&pages=1131-1137&author=E.+Pottertonauthor=P.+Briggsauthor=M.+Turkenburgauthor=E.+Dodson&title=A+graphical+user+interface+to+the+CCP4+program+suite&doi=10.1107%2FS0907444903008126"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">A graphical user interface to the CCP4 program suite</span></div><div class="casAuthors">Potterton, Elizabeth; Briggs, Peter; Turkenburg, Maria; Dodson, Eleanor</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">D59</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1131-1137</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4i is a graphical user interface that makes running programs from the CCP4 suite simpler and quicker.  It is particularly directed at inexperienced users and tightly linked to introductory and scientific documentation.  It also provides a simple project-management system and visualization tools.  The system is readily extensible and not specific to CCP4 software.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDH-BQSvi5qbVg90H21EOLACvtfcHk0ljBPYjB8jTcoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXkvFSksLs%253D&md5=25a0168dd7ea9f930036453a829e1e3d</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1107%2FS0907444903008126&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444903008126%26sid%3Dliteratum%253Aachs%26aulast%3DPotterton%26aufirst%3DE.%26aulast%3DBriggs%26aufirst%3DP.%26aulast%3DTurkenburg%26aufirst%3DM.%26aulast%3DDodson%26aufirst%3DE.%26atitle%3DA%2520graphical%2520user%2520interface%2520to%2520the%2520CCP4%2520program%2520suite%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2003%26volume%3D59%26spage%3D1131%26epage%3D1137%26doi%3D10.1107%2FS0907444903008126" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref78"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref78'); return false;" data-citation="" class="refNumLink">78</a></strong><div class="NLM_citation" id="cit78"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span> <span> </span><span class="NLM_article-title">Refinement of Macromolecular Structures by the Maximum-Likelihood Method</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">240</span>– <span class="NLM_lpage">255</span>, <span class="refDoi"> DOI: 10.1107/S0907444996012255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-255&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+Macromolecular+Structures+by+the+Maximum-Likelihood+Method&doi=10.1107%2FS0907444996012255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. σA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial α-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0ljBPYjB8jTcoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520Macromolecular%2520Structures%2520by%2520the%2520Maximum-Likelihood%2520Method%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1997%26volume%3D53%26spage%3D240%26epage%3D255%26doi%3D10.1107%2FS0907444996012255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref79"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref79'); return false;" data-citation="" class="refNumLink">79</a></strong><div class="NLM_citation" id="cit79"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and development of Coot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lhEFaVrnFoAhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref80"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref80'); return false;" data-citation="" class="refNumLink">80</a></strong><div class="NLM_citation" id="cit80"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chisamore, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahmed, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bentrem, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jordan, V. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tonetti, D. A.</span></span> <span> </span><span class="NLM_article-title">Novel antitumor effect of estradiol in athymic mice injected with a T47D breast cancer cell line overexpressing protein kinase Calpha</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">3156</span>– <span class="NLM_lpage">3165</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=11595710" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BD3MXnvF2rsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2001&pages=3156-3165&author=M.+J.+Chisamoreauthor=Y.+Ahmedauthor=D.+J.+Bentremauthor=V.+C.+Jordanauthor=D.+A.+Tonetti&title=Novel+antitumor+effect+of+estradiol+in+athymic+mice+injected+with+a+T47D+breast+cancer+cell+line+overexpressing+protein+kinase+Calpha"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">Novel antitumor effect of estradiol in athymic mice injected with a T47D breast cancer cell line overexpressing protein kinase cα</span></div><div class="casAuthors">Chisamore, Michael J.; Ahmed, Yasmin; Bentrem, David J.; Jordan, V. Craig; Tonetti, Debra A.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3156-3165</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Resistance to tamoxifen (TAM) represents a significant challenge to the management of breast cancer.  We previously reported that the estrogen receptor (ER)-neg. hormone-independent T47D:C42 cell line has both elevated protein kinase Cα (PKCα) protein expression and basal activator protein-1 activity compared with the parental ER+ (hormone-dependent) T47D:A18 cell line.  Stable transfection of PKCα to the T47D:A18 breast cancer cell line results in increased basal activator protein-1 activity, reduced ER function, increased proliferation rate, and hormone-independent growth (Tonetti et al., Br. J. Cancer, 83: 782-791, 2000).  In this report, we further characterize the role of PKCα overexpression in vivo to elucidate a possible mol. mechanism of tamoxifen resistance.  To det. whether the T47D:A18/PKCα cell line would produce hormone-independent tumors in athymic mice, we injected T47D:A18, T47D:A18/neo, or the T47D:A18/PKCα20 cell clones bilaterally into the mammary fat pads of athymic mice.  Tumor growth was evaluated following treatment with estradiol (E2), TAM, and the pure antiestrogen, ICI 182780.  Mice receiving either T47D:A18 or T47D:A18/neo cells produced tumors that grew in response to E2 treatment, whereas the untreated control and TAM-treated groups showed no tumor growth.  Interestingly, mice receiving the T47D:A18/PKCα20 clone produced tumors in both the control and TAM groups, whereas tumor growth was inhibited in mice treated with E2.  PKCα was also overexpressed in an MCF-7 tumor model that also exhibited TAM-stimulated and E2-induced regression.  These results suggest that overexpression of PKCα in breast tumors results in hormone-independent tumor growth that cannot be inhibited by TAM treatment.  Furthermore, the finding that E2 has an antitumor effect on breast tumors overexpressing PKCα is a novel observation that may have important therapeutic implications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4Yd2zilR7XbVg90H21EOLACvtfcHk0lhEFaVrnFoAhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXnvF2rsr0%253D&md5=e132ea7de154480f42b7f701b796401c</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DChisamore%26aufirst%3DM.%2BJ.%26aulast%3DAhmed%26aufirst%3DY.%26aulast%3DBentrem%26aufirst%3DD.%2BJ.%26aulast%3DJordan%26aufirst%3DV.%2BC.%26aulast%3DTonetti%26aufirst%3DD.%2BA.%26atitle%3DNovel%2520antitumor%2520effect%2520of%2520estradiol%2520in%2520athymic%2520mice%2520injected%2520with%2520a%2520T47D%2520breast%2520cancer%2520cell%2520line%2520overexpressing%2520protein%2520kinase%2520Calpha%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2001%26volume%3D7%26spage%3D3156%26epage%3D3165" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref81"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref81'); return false;" data-citation="" class="refNumLink">81</a></strong><div class="NLM_citation" id="cit81"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Perez
White, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molloy, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tonetti, D. A.</span></span> <span> </span><span class="NLM_article-title">RETRACTED ARTICLE: Extranuclear ERalpha is associated with regression of T47D PKCalpha-overexpressing, tamoxifen-resistant breast cancer</span>. <i>Mol. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">34</span>, <span class="refDoi"> DOI: 10.1186/1476-4598-12-34</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1186%2F1476-4598-12-34" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=23634843" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpt12qt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2013&pages=34&author=B.+Perez%0AWhiteauthor=M.+E.+Molloyauthor=H.+Zhaoauthor=Y.+Zhangauthor=D.+A.+Tonetti&title=RETRACTED+ARTICLE%3A+Extranuclear+ERalpha+is+associated+with+regression+of+T47D+PKCalpha-overexpressing%2C+tamoxifen-resistant+breast+cancer&doi=10.1186%2F1476-4598-12-34"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">Extranuclear ERα is associated with regression of T47D PKCα-overexpressing, tamoxifen-resistant breast cancer</span></div><div class="casAuthors">Perez White, Bethany; Molloy, Mary Ellen; Zhao, Huiping; Zhang, Yiyun; Tonetti, Debra A.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">34</span>CODEN:
                <span class="NLM_cas:coden">MCOACG</span>;
        ISSN:<span class="NLM_cas:issn">1476-4598</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Background: Prior to the introduction of tamoxifen, high dose estradiol was used to treat breast cancer patients with similar efficacy as tamoxifen, albeit with some undesirable side effects.  There is renewed interest to utilize estradiol to treat endocrine resistant breast cancers, esp. since findings from several preclin. models and clin. trials indicate that estradiol may be a rational second-line therapy in patients exhibiting resistance to tamoxifen and/or aromatase inhibitors.  We and others reported that breast cancer patients bearing protein kinase C alpha (PKCα)-expressing tumors exhibit endocrine resistance and tumor aggressiveness.  Our T47D:A18/PKCα preclin. model is tamoxifen-resistant, hormone-independent, yet is inhibited by 17β-estradiol (E2) in vivo.  We previously reported that E2-induced T47D:A18/PKCα tumor regression requires extranuclear ERα and interaction with the extracellular matrix.  Methods: T47D:A18/PKCα cells were grown in vitro using two-dimensional (2D) cell culture, three-dimensional (3D) Matrigel and in vivo by establishing xenografts in athymic mice.  Immunofluoresence confocal microscopy and co-localization were applied to det. estrogen receptor alpha (ERα) subcellular localization.  Co-immunopptn. and western blot were used to examine interaction of ERα with caveolin-1.  Results: We report that although T47D:A18/PKCα cells are cross-resistant to raloxifene in cell culture and in Matrigel, raloxifene induces regression of tamoxifen-resistant tumors.  ERα rapidly translocates to extranuclear sites during T47D:A18/PKCα tumor regression in response to both raloxifene and E2, whereas ERα is primarily localized in the nucleus in proliferating tumors.  E2 treatment induced complete tumor regression whereas cessation of raloxifene treatment resulted in tumor regrowth accompanied by re-localization of ERα to the nucleus.  T47D:A18/neo tumors that do not overexpress PKCα maintain ERα in the nucleus during tamoxifen-mediated regression.  An assocn. between ERα and caveolin-1 increases in tumors regressing in response to E2.  Conclusions: Extranuclear ERα plays a role in the regression of PKCα-overexpressing tamoxifen-resistant tumors.  These studies underline the unique role of extranuclear ERα in E2- and raloxifene-induced tumor regression that may have implications for treatment of endocrine-resistant PKCα-expressing tumors encountered in the clinic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3dl-xeGYbWbVg90H21EOLACvtfcHk0lhEFaVrnFoAhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXpt12qt7w%253D&md5=cc902496a93ae81c99847d5f16b89512</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1186%2F1476-4598-12-34&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1476-4598-12-34%26sid%3Dliteratum%253Aachs%26aulast%3DPerez%2BWhite%26aufirst%3DB.%26aulast%3DMolloy%26aufirst%3DM.%2BE.%26aulast%3DZhao%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DTonetti%26aufirst%3DD.%2BA.%26atitle%3DRETRACTED%2520ARTICLE%253A%2520Extranuclear%2520ERalpha%2520is%2520associated%2520with%2520regression%2520of%2520T47D%2520PKCalpha-overexpressing%252C%2520tamoxifen-resistant%2520breast%2520cancer%26jtitle%3DMol.%2520Cancer%26date%3D2013%26volume%3D12%26spage%3D34%26doi%3D10.1186%2F1476-4598-12-34" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref82"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref82'); return false;" data-citation="" class="refNumLink">82</a></strong><div class="NLM_citation" id="cit82"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Molloy, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gherezghiher, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michalsen, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maximov, P. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jordan, V. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thatcher, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tonetti, D. A.</span></span> <span> </span><span class="NLM_article-title">Novel selective estrogen mimics for the treatment of tamoxifen-resistant breast cancer</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">2515</span>– <span class="NLM_lpage">2526</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.MCT-14-0319</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1158%2F1535-7163.MCT-14-0319" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=25205655" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVGju73N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=2515-2526&author=M.+E.+Molloyauthor=B.+E.+Whiteauthor=T.+Gherezghiherauthor=B.+T.+Michalsenauthor=R.+Xiongauthor=H.+Patelauthor=H.+Zhaoauthor=P.+Y.+Maximovauthor=V.+C.+Jordanauthor=G.+R.+Thatcherauthor=D.+A.+Tonetti&title=Novel+selective+estrogen+mimics+for+the+treatment+of+tamoxifen-resistant+breast+cancer&doi=10.1158%2F1535-7163.MCT-14-0319"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit82R"><div class="casContent"><span class="casTitleNuber">82</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Selective Estrogen Mimics for the Treatment of Tamoxifen-Resistant Breast Cancer</span></div><div class="casAuthors">Molloy, Mary Ellen; White, Bethany E. Perez; Gherezghiher, Teshome; Michalsen, Bradley T.; Xiong, Rui; Patel, Hitisha; Zhao, Huiping; Maximov, Philipp Y.; Jordan, V. Craig; Thatcher, Gregory R. J.; Tonetti, Debra A.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2515-2526</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Endocrine-resistant breast cancer is a major clin. obstacle.  The use of 17β-estradiol (E2) has reemerged as a potential treatment option following exhaustive use of tamoxifen or aromatase inhibitors, although side effects have hindered its clin. usage.  Protein kinase C alpha (PKCα) expression was shown to be a predictor of disease outcome for patients receiving endocrine therapy and may predict a pos. response to an estrogenic treatment.  Here, we have investigated the use of novel benzothiophene selective estrogen mimics (SEM) as an alternative to E2 for the treatment of tamoxifen-resistant breast cancer.  Following in vitro characterization of SEMs, a panel of clin. relevant PKCα-expressing, tamoxifen-resistant models were used to investigate the antitumor effects of these compds.  SEM treatment resulted in growth inhibition and apoptosis of tamoxifen-resistant cell lines in vitro.  In vivo SEM treatment induced tumor regression of tamoxifen-resistant T47D:A18/PKCα and T47D:A18-TAM1 tumor models.  T47D:A18/PKCα tumor regression was accompanied by translocation of estrogen receptor (ER) α to extranuclear sites, possibly defining a mechanism through which these SEMs initiate tumor regression.  SEM treatment did not stimulate growth of E2-dependent T47D:A18/neo tumors.  In addn., unlike E2 or tamoxifen, treatment with SEMs did not stimulate uterine wt. gain.  These findings suggest the further development of SEMs as a feasible therapeutic strategy for the treatment of endocrine-resistant breast cancer without the side effects assocd. with E2.  Mol Cancer Ther; 13(11); 2515-26. cpr2014 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp64x3RCcDeYbVg90H21EOLACvtfcHk0lhST5JGQq7yzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVGju73N&md5=203055e50477bcb1c98ea4c7844cf4b9</span></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-14-0319&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-14-0319%26sid%3Dliteratum%253Aachs%26aulast%3DMolloy%26aufirst%3DM.%2BE.%26aulast%3DWhite%26aufirst%3DB.%2BE.%26aulast%3DGherezghiher%26aufirst%3DT.%26aulast%3DMichalsen%26aufirst%3DB.%2BT.%26aulast%3DXiong%26aufirst%3DR.%26aulast%3DPatel%26aufirst%3DH.%26aulast%3DZhao%26aufirst%3DH.%26aulast%3DMaximov%26aufirst%3DP.%2BY.%26aulast%3DJordan%26aufirst%3DV.%2BC.%26aulast%3DThatcher%26aufirst%3DG.%2BR.%26aulast%3DTonetti%26aufirst%3DD.%2BA.%26atitle%3DNovel%2520selective%2520estrogen%2520mimics%2520for%2520the%2520treatment%2520of%2520tamoxifen-resistant%2520breast%2520cancer%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2014%26volume%3D13%26spage%3D2515%26epage%3D2526%26doi%3D10.1158%2F1535-7163.MCT-14-0319" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref83"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref83'); return false;" data-citation="" class="refNumLink">83</a></strong><div class="NLM_citation" id="cit83"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Uenishi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishiwaki, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wakabayashi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oae, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsukube, H.</span></span> <span> </span><span class="NLM_article-title">Synthesis of ι-(bromomethyl)bipyridines and related ι-(bromomethyl)pyridinoheteroaromatics: useful functional tools for ligands in host molecules</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">4382</span>– <span class="NLM_lpage">4388</span>, <span class="refDoi"> DOI: 10.1021/jo00068a037</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo00068a037" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADyaK3sXltl2jsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=1993&pages=4382-4388&author=J.+Uenishiauthor=T.+Tanakaauthor=K.+Nishiwakiauthor=S.+Wakabayashiauthor=S.+Oaeauthor=H.+Tsukube&title=Synthesis+of+%CE%B9-%28bromomethyl%29bipyridines+and+related+%CE%B9-%28bromomethyl%29pyridinoheteroaromatics%3A+useful+functional+tools+for+ligands+in+host+molecules&doi=10.1021%2Fjo00068a037"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit83R"><div class="casContent"><span class="casTitleNuber">83</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of ω-(bromomethyl)bipyridines and related ω-(bromomethyl)pyridinoheteroaromatics: useful functional tools for ligands in host molecules</span></div><div class="casAuthors">Uenishi, Junichi; Tanaka, Takakazu; Nishiwaki, Kenji; Wakabayashi, Shoji; Oae, Shigeru; Tsukube, Hiroshi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">4382-8</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    </div><div class="casAbstract">Pyridines and 2,2'-bipyridines have been employed as useful ligands in mol. recognition chemistries.  Halomethyl-substituted bipyridine or oligopyridine derivs. were required for the assembly of bipyridine or oligopyridine units with a supporting mother functional part in artificial biofunctional mols.  A series of ω-(bromomethyl)bipyridines and related ω-(bromomethyl)pyridinoheteroarom. compds., e.g., I, II, and III, were synthesized.  Prepn. of oligopyridines and pyridinoheteroarom. compds. have been carried out by either intermol. ligand coupling of alkyl heteroaryl sulfoxide with pyridyllithium or intramol. ligand coupling of pyridyl heteroaryl sulfoxide with methylmagnesium bromide for the type I compds.  The type II and III compds. were synthesized by addn. of pyridyllithium to pyridinecarboxaldehyde.  The ω-bromo group was introduced by radical bromination reaction of methylpyridyl group using N-bromosuccinimide and dibenzyl peroxide or bromination of ω-(hydroxymethyl)pyridine using a combination of CBr4 and Ph3P.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWejYii-fOv7Vg90H21EOLACvtfcHk0lhST5JGQq7yzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXltl2jsb0%253D&md5=4ffdb2da0fd33506259b3b5ac86e2d15</span></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.1021%2Fjo00068a037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo00068a037%26sid%3Dliteratum%253Aachs%26aulast%3DUenishi%26aufirst%3DJ.%26aulast%3DTanaka%26aufirst%3DT.%26aulast%3DNishiwaki%26aufirst%3DK.%26aulast%3DWakabayashi%26aufirst%3DS.%26aulast%3DOae%26aufirst%3DS.%26aulast%3DTsukube%26aufirst%3DH.%26atitle%3DSynthesis%2520of%2520%25CE%25B9-%2528bromomethyl%2529bipyridines%2520and%2520related%2520%25CE%25B9-%2528bromomethyl%2529pyridinoheteroaromatics%253A%2520useful%2520functional%2520tools%2520for%2520ligands%2520in%2520host%2520molecules%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D1993%26volume%3D58%26spage%3D4382%26epage%3D4388%26doi%3D10.1021%2Fjo00068a037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68'],'ref69':['cit69'],'ref70':['cit70'],'ref71':['cit71'],'ref72':['cit72'],'ref73':['cit73'],'ref74':['cit74'],'ref75':['cit75'],'ref76':['cit76'],'ref77':['cit77'],'ref78':['cit78'],'ref79':['cit79'],'ref80':['cit80'],'ref81':['cit81'],'ref82':['cit82'],'ref83':['cit83']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 3 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Xiaoyi Zhu, Ruibo Li, Hequan Yao, <span class="NLM_string-name hlFld-ContribAuthor">Aijun Lin</span>. </span><span class="cited-content_cbyCitation_article-title">Palladium-Catalyzed Allenamide Carbopalladation/Allylation with Active Methine Compounds. </span><span class="cited-content_cbyCitation_journal-name">Organic Letters</span><span> <strong>2021,</strong> <em>23 </em>
                                    (12)
                                     , 4630-4634. <a href="https://doi.org/10.1021/acs.orglett.1c01369" title="DOI URL">https://doi.org/10.1021/acs.orglett.1c01369</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.orglett.1c01369&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.orglett.1c01369%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520Letters%26atitle%3DPalladium-Catalyzed%252BAllenamide%252BCarbopalladation%25252FAllylation%252Bwith%252BActive%252BMethine%252BCompounds%26aulast%3DZhu%26aufirst%3DXiaoyi%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D20042021%26date%3D28052021%26volume%3D23%26issue%3D12%26spage%3D4630%26epage%3D4634" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="NLM_string-name hlFld-ContribAuthor">Tanaji T. Talele</span>. </span><span class="cited-content_cbyCitation_article-title">Opportunities for Tapping into Three-Dimensional Chemical Space through a Quaternary Carbon. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (22)
                                     , 13291-13315. <a href="https://doi.org/10.1021/acs.jmedchem.0c00829" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00829</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00829&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00829%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DOpportunities%252Bfor%252BTapping%252Binto%252BThree-Dimensional%252BChemical%252BSpace%252Bthrough%252Ba%252BQuaternary%252BCarbon%26aulast%3DTalele%26aufirst%3DTanaji%2BT.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D15052020%26date%3D31082020%26date%3D17082020%26volume%3D63%26issue%3D22%26spage%3D13291%26epage%3D13315" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhangxu  He</span>, <span class="hlFld-ContribAuthor ">Haomiao  Jiao</span>, <span class="hlFld-ContribAuthor ">Qi  An</span>, <span class="hlFld-ContribAuthor ">Xin  Zhang</span>, <span class="hlFld-ContribAuthor ">Dan  Zengyangzong</span>, <span class="hlFld-ContribAuthor ">Jiale  Xu</span>, <span class="hlFld-ContribAuthor ">Hongmin  Liu</span>, <span class="hlFld-ContribAuthor ">Liying  Ma</span>, <span class="hlFld-ContribAuthor ">Wen  Zhao</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of novel 4-phenylquinazoline-based BRD4 inhibitors for cardiac fibrosis. </span><span class="cited-content_cbyCitation_journal-name">Acta Pharmaceutica Sinica B</span><span> <strong>2021,</strong> <em>399 </em><a href="https://doi.org/10.1016/j.apsb.2021.07.018" title="DOI URL">https://doi.org/10.1016/j.apsb.2021.07.018</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.apsb.2021.07.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.apsb.2021.07.018%26sid%3Dliteratum%253Aachs%26jtitle%3DActa%2520Pharmaceutica%2520Sinica%2520B%26atitle%3DDiscovery%252Bof%252Bnovel%252B4-phenylquinazoline-based%252BBRD4%252Binhibitors%252Bfor%252Bcardiac%252Bfibrosis%26aulast%3DHe%26aufirst%3DZhangxu%26date%3D2021%26volume%3D399" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00456/20200701/images/medium/jm0c00456_0007.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00456/20200701/images/large/jm0c00456_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00456&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00456/20200701/images/medium/jm0c00456_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00456/20200701/images/large/jm0c00456_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. BRD4 is a coregulator of ER signaling in breast cancer. (A) ER+ breast cancer is a transcriptionally driven cancer: E2-liganded ER localizes to ERE forming a transcriptional complex, including proteins such as MED1, P300, and FOXA1; BRD4 recruitment to this complex at acetylated histones and cohesin recruitment causes transcriptional elongation via P-TEFb and RNA polymerase II. Targeting the ER transcriptional complex through BRD4 provides an alternative approach either as monotherapy or in combination with SERDs to treat ER+ breast cancer. (B) Structures of benchmark BET inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00456/20200701/images/large/jm0c00456_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00456&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00456/20200701/images/medium/jm0c00456_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00456/20200701/images/large/jm0c00456_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Design of pyrrolopyridone-based BET inhibitors. (A) Design concept inspired by earlier reported BET inhibitors. Pyrrolopyridone, <b>7</b>, identified from a virtual screen forms bidentate hydrogen-bonding interaction with Asn140, highlighted in green. The sulfonamide group of <b>8</b> interacts with the ZA loop via a putative hydrogen-bonding interaction, highlighted in blue. The heteroaryl moiety of <b>9</b> interacts with the hydrophobic WPF shelf, highlighted in yellow. Combining these design parameters leads to the design of <b>10</b> and <b>12</b> as novel BET inhibitors. (B) Predicted binding pose of compound <b>10</b> binding to BRD4-BD1 using Glide (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ZYU">3ZYU</a>). Key interacting residues from docking are highlighted and labeled. (C) Overlay of the predicted binding pose of <b>10</b> (blue) with <b>9</b> (beige) from PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ZYU">3ZYU</a>. (D) New cocrystal of compound <b>27</b> bound to BRD4-BD1 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6P05">6P05</a>), showing four hydrogen bonds with N140, D88, and K91 and one additional water-mediated hydrogen bond with K91. The “leucine clamp” (L92 and L94) is highlighted in green, and other key interacting residues are highlighted and labeled in red. (E) Overlay of the predicted binding pose of <b>10</b> (blue) with the experimentally confirmed binding pose of <b>27</b> (beige) clearly showing the second pyridine ring of <b>27</b> (boxed in green) that forms a hydrophobic interaction with L92 and L94.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00456/20200701/images/large/jm0c00456_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00456&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00456/20200701/images/medium/jm0c00456_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00456/20200701/images/large/jm0c00456_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Binding to BRD4-BD1 correlates with cellular antiproliferative potency: (A) BRD4-BD1 and BRD4-BD2 binding measured by TR-FRET for <b>1</b> compared to <b>27</b> (data normalized to DMSO as 100%); (B) BROMOscan of compound <b>27</b> against 40 bromodomain proteins; (C) correlation between BRD4-BD1 binding (TR-FRET) assay and potency for growth inhibition of MCF-7:CFR cells.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00456/20200701/images/large/jm0c00456_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00456&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00456/20200701/images/medium/jm0c00456_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00456/20200701/images/large/jm0c00456_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Inhibition of growth and transcriptional regulation of MCF-7:CFR cells by <b>27</b> compared to other BET inhibitors: (A) comparison of compound <b>27</b> with benchmark BET inhibitors in cell proliferation assay of MCF-7:CFR 2D cell cultures showing resistance to fulvestrant and 4OHT (TAM) (1 μM); (B) heatmap comparing gene expression changes among control (DMSO), <b>27</b> (100 nM), and <b>1</b> (100 nM) treated MCF-7:CFR cells after 24 h; (C) volcano plot showing top 10 downregulated genes from RNA-seq comparing <b>27</b> with vehicle control; (D) log 2-fold change of significantly regulated oncogenes and tumor suppressors in various oncogenic (Myc, cell cycle) and apoptotic pathways; (E) representative images on day 9 of treatment with DMSO (0.01%), fulvestrant (100 nM), JQ1 (<b>1</b>, 100 nM), AZD5153 (<b>5</b>, 100 nM), and compound <b>27</b> (10 and 100 nM) in MCF7:CFR 3D spheroids; scale bar = 1 mm.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00456/20200701/images/large/jm0c00456_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00456&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00456/20200701/images/medium/jm0c00456_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00456/20200701/images/large/jm0c00456_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. BET inhibitor <b>27</b> is a potent antiproliferative agent in breast cancer cell lines resistant to tamoxifen, AIs, CDK4/6 inhibitors, and fulvestrant. (A–E) Concentration-dependent response to compound <b>27</b> (red) compared to <b>1</b> (blue) and 4OHT (TAM, brown), with potency for <b>27</b> annotated. (A) <b>27</b> dose-dependently inhibited the proliferation of parental MCF-7:WS8 cells that are sensitive to growth inhibition by tamoxifen. Compound <b>27</b> dose-dependently inhibited the proliferation of tamoxifen resistant MCF-7:5C (B) and MCF-7:TAM1 (C) cells with superior potency to <b>1</b>. Compounds <b>27</b> and <b>1</b> dose-dependently inhibited the proliferation of tamoxifen sensitive parental T47D cells (D) and <i>ESR1</i> D538G mutant T47D/TDG cells (E), with no significant right-shift in potency in the <i>ESR1</i> mutant cell line. (F) Compound <b>27</b> dose-dependently inhibited the proliferation of MCF-7:CPR cells that are resistant to tamoxifen (brown. 1 μM) and palbociclib (green. 1 μM). (G) Comparison of <b>27</b> with <b>1</b> and <b>5</b> in fulvestrant-resistant MCF-7:CFR cell proliferation assay using drug washout after 3 h, by replacing media after incubation with drug for 3 h and measuring cell viability after 5 days. Solid bars show no washout. Hashed bars show washout treatment.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00456/20200701/images/large/jm0c00456_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00456&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00456/20200701/images/medium/jm0c00456_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00456/20200701/images/large/jm0c00456_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Platelet count in rats and inhibition of tumor growth in mice of tamoxifen-resistant MCF7:TAM1 cell-derived xenografts. (A) Platelet counts from rats administered <b>27</b> (25 mg/kg) or <b>3</b> (3 mg/kg) starting at day 0 showing data for individual rats (<i>n</i> = 3): <i>p</i> < 0.05 at day 10 by two-tailed paired <i>t</i> test. (B) Tumors grown to 0.32 cm<sup>2</sup> were treated with vehicle, TAM (1.5 (mg/mouse)/day), or compound <b>27</b> (HCl salt) (50 (mg/kg)/day) by daily oral gavage. Data show mean and SD. (C) Average body weight changes compared to pretreatment. (D) Changes in individual tumor size at 4 weeks. (E) Low dose fulvestrant (1 (mg/mouse)/week by sc) in combination with <b>2</b> 50 (mg/kg)/day (formulated in 0.5% (w/v) HPMC/TWEEN80 1% in water<a onclick="showRef(event, 'ref69'); return false;" href="javascript:void(0);" class="ref ref69">(69)</a>) or <b>27</b> 30 (mg/kg)/day (formulated in 10% PEG400/90% HP-β-CD (10%)). Changes of individual tumor size at 4 weeks were plotted: data show the mean and SD with one-way ANOVA analysis with Dunnett’s multiple comparisons test.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00456/20200701/images/large/jm0c00456_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00456&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00456/20200701/images/medium/jm0c00456_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00456/20200701/images/large/jm0c00456_0010.jpeg" id="rightTab-GRAPHIC-d7e1927-autogenerated" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthetic Route of Compounds <b>10</b>, <b>11</b>, and <b>19</b>–<b>24</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00456/20200701/images/large/jm0c00456_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00456&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) MeI, 60% NaH, DMF, RT, 1 h; (b) NIS, DMF, RT, 12 h, 92% over two steps; (c) <i>n</i>-BuLi, 2-isopropoxy-4,4,5,5-tetramethyl-1,3,2-dioxaborolane, THF, −78 °C to RT, overnight, 65%; (d) <i>n</i>-BuLi, DMF, THF, −78 °C, 2 h, 73%; (e) R<sub>1</sub>SO<sub>2</sub>Cl, pyridine, RT, 2 h, 75–96%; (f) R<sub>2</sub>Br, K<sub>2</sub>CO<sub>3</sub>, DMF, 50 °C, 4–16 h, 66–92%; (g) <b>35</b>, Pd(dppf)Cl<sub>2</sub>–DCM, K<sub>3</sub>PO<sub>4</sub>, dioxane/H<sub>2</sub>O, 6–24 h, 90 °C, 77–94%; (h) TMSCl, NaI, MeCN/H<sub>2</sub>O, 3 h, 50 °C, 82–96%.</p></p></figure><figure data-id="sch2" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00456/20200701/images/medium/jm0c00456_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00456/20200701/images/large/jm0c00456_0011.jpeg" id="rightTab-GRAPHIC-d7e2040-autogenerated" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthetic Route of Compounds <b>12</b>–<b>18</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00456/20200701/images/large/jm0c00456_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00456&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) aryl bromide, K<sub>2</sub>CO<sub>3</sub>, DMF, RT, 2 h, 75–96%; (b) Fe, saturated NH<sub>4</sub>Cl (aq), EtOH, reflux, 2 h, 66–92%; (c) sulfonyl chloride, pyridine, RT, 1–5 h, 83–90%; (d) <b>35</b>, Pd(dppf)Cl<sub>2</sub>–DCM, K<sub>3</sub>PO<sub>4</sub>, dioxane/H<sub>2</sub>O, 6–24 h, 90 °C, 75–95%; (e) TMSCl, NaI, MeCN/H<sub>2</sub>O, 2 h, 50 °C, 87–95%; (f) <b>36</b>, AcOH, NaBO<sub>3</sub>, 50 °C, 1 h, 67%; (g) ethanesulfonyl chloride, triethylamine, DCM, RT, 3 h, 83–90%.</p></p></figure><figure data-id="sch3" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00456/20200701/images/medium/jm0c00456_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00456/20200701/images/large/jm0c00456_0012.jpeg" id="rightTab-GRAPHIC-d7e2228-autogenerated" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthetic Route of Compounds <b>25</b>–<b>30</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-13/acs.jmedchem.0c00456/20200701/images/large/jm0c00456_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00456&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) 2,2′-(bromomethylene)dipyridine or bromodiphenylmethane, K<sub>2</sub>CO<sub>3</sub>, DMF, 50 °C, 5–16 h, 72–91%; (b) R<sub>3</sub>I, <i>t</i>-BuOK, THF, RT, 2 h, 73–93%; (c) Fe, saturated NH<sub>4</sub>Cl (aq), EtOH, reflux, 2 h; (d) sulfonyl chloride or acyl chloride, pyridine, RT, 1–5 h, 83–90%; (e) <b>35</b>, Pd(dppf)Cl<sub>2</sub>–DCM, K<sub>3</sub>PO<sub>4</sub>, dioxane/H<sub>2</sub>O, 90 °C, 6–24 h, 66–93%; (f) TMSCl, NaI, MeCN/H<sub>2</sub>O, 50 °C, 2 h, 85–94%; (g) MeLi, THF, −42 °C, 2 h, 55%.</p></p></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i120">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_08782" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_08782" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 83 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span>U.S. Breast
Cancer Statistics. <a href="https://www.breastcancer.org/symptoms/understand_bc/statistics" class="extLink">https://www.breastcancer.org/symptoms/understand_bc/statistics</a> (accessed Mar 7, <span class="NLM_year">2019</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=U.S.+Breast%0ACancer+Statistics.+https%3A%2F%2Fwww.breastcancer.org%2Fsymptoms%2Funderstand_bc%2Fstatistics+%28accessed+Mar+7%2C+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Early
Breast Cancer Trialists’ Collaborative Group.</span> <span> </span><span class="NLM_article-title">Tamoxifen for early breast cancer:
an overview of the randomised trials</span>. <i>Lancet</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>351</i></span>,  <span class="NLM_fpage">1451</span>– <span class="NLM_lpage">1467</span>, <span class="refDoi"> DOI: 10.1016/S0140-6736(97)11423-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1016%2FS0140-6736%2897%2911423-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=9605801" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=351&publication_year=1998&pages=1451-1467&author=Early%0ABreast+Cancer+Trialists%E2%80%99+Collaborative+Group.&title=Tamoxifen+for+early+breast+cancer%3A%0Aan+overview+of+the+randomised+trials&doi=10.1016%2FS0140-6736%2897%2911423-4"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2FS0140-6736%2897%2911423-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0140-6736%252897%252911423-4%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DTamoxifen%2520for%2520early%2520breast%2520cancer%253A%250Aan%2520overview%2520of%2520the%2520randomised%2520trials%26jtitle%3DLancet%26date%3D1998%26volume%3D351%26spage%3D1451%26epage%3D1467%26doi%3D10.1016%2FS0140-6736%2897%2911423-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roodi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bailey, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kao, W. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verrier, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yee, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dupont, W. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parl, F. F.</span></span> <span> </span><span class="NLM_article-title">Estrogen receptor gene analysis in estrogen receptor-positive and receptor-negative primary breast cancer</span>. <i>J. Natl. Cancer Inst.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>87</i></span>,  <span class="NLM_fpage">446</span>– <span class="NLM_lpage">451</span>, <span class="refDoi"> DOI: 10.1093/jnci/87.6.446</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1093%2Fjnci%2F87.6.446" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=7861463" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADyaK2MXkvVGntL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=87&publication_year=1995&pages=446-451&author=N.+Roodiauthor=L.+R.+Baileyauthor=W.+Y.+Kaoauthor=C.+S.+Verrierauthor=C.+J.+Yeeauthor=W.+D.+Dupontauthor=F.+F.+Parl&title=Estrogen+receptor+gene+analysis+in+estrogen+receptor-positive+and+receptor-negative+primary+breast+cancer&doi=10.1093%2Fjnci%2F87.6.446"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Estrogen receptor gene analysis in estrogen receptor-positive and receptor-negative primary breast cancer</span></div><div class="casAuthors">Roodi, Nady; Bailey, L. Renee; Kao, Woei-Yau; Verrier, Carmel S.; Yee, Cindy J.; Dupont, William D.; Parl, Fritz F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the National Cancer Institute</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">87</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">446-51</span>CODEN:
                <span class="NLM_cas:coden">JNCIEQ</span>;
        ISSN:<span class="NLM_cas:issn">0027-8874</span>.
    </div><div class="casAbstract">In breast cancer patients, about two thirds of the tumors are estrogen receptor (ER)-pos. and one third are ER-neg.  The mol. mechanisms leading to the ER-neg. phenotype are poorly understood.  Nearly all ER-neg. and about 40% of ER-pos. cancers are resistant to endocrine therapy.  In this study, the authors examd. the entire coding region of the ER gene in ER-pos. and ER-neg. primary breast tumors to det. whether deletions/insertions or point mutations might account for the ER-neg. phenotype.  The authors amplified exons 1 through 8 of the ER gene in 118 ER-pos. and 70 ER-neg. primary breast tumors and searched for mutations by single-strand conformation polymorphism anal., denaturing gradient gel electrophoresis, and DNA sequencing.  Both ER-neg. and ER-pos. tumors contained neutral polymorphisms in codons 10 [TCT→TCC (Ser)], 87 [GCG→GCC (Ala)], 243 [CGC→CGT (Arg)], 325 [CCC→CCG (Pro)], and 594 [ACA→ACG (Thr)].  There was no correlation of any of the polymorphic alleles with the ER phenotype or other clinicopathol. parameters including tumor type, size, grade, or stage.  However, the polymorphism in codon 325 showed a strong assocn. with a family history of breast cancer.  This assocn. was obsd. both in premenopausal and postmenopausal patients.  Despite extensive searching in exons 1 through 8, the authors found no deletions/insertions and only two missense mutations in codons 69 [AAC (Asn)→AAG (Lys)] and 396 [ATG (Met)→GTG (Val)] of the same ER-neg. tumor.  Thus, only 1% of the primary breast cancers had point mutations in the ER gene.  In the majority of primary breast cancers, the ER-neg. phenotype is not the result of mutations in the coding region of the ER gene, but is due to deficient ER expression at the transcriptional or post-transcriptional level.  The correlation reported previously, as well as the current findings, suggest that further investigations are warranted to understand the possible linkage of the ER gene locus to hereditary breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0jnOyp24XmbVg90H21EOLACvtfcHk0ljtAh9rJ_EqVQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXkvVGntL4%253D&md5=c45eb7d3beda848c2dcbad1538a8296f</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1093%2Fjnci%2F87.6.446&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjnci%252F87.6.446%26sid%3Dliteratum%253Aachs%26aulast%3DRoodi%26aufirst%3DN.%26aulast%3DBailey%26aufirst%3DL.%2BR.%26aulast%3DKao%26aufirst%3DW.%2BY.%26aulast%3DVerrier%26aufirst%3DC.%2BS.%26aulast%3DYee%26aufirst%3DC.%2BJ.%26aulast%3DDupont%26aufirst%3DW.%2BD.%26aulast%3DParl%26aufirst%3DF.%2BF.%26atitle%3DEstrogen%2520receptor%2520gene%2520analysis%2520in%2520estrogen%2520receptor-positive%2520and%2520receptor-negative%2520primary%2520breast%2520cancer%26jtitle%3DJ.%2520Natl.%2520Cancer%2520Inst.%26date%3D1995%26volume%3D87%26spage%3D446%26epage%3D451%26doi%3D10.1093%2Fjnci%2F87.6.446" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Riggins, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schrecengost, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guerrero, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouton, A. H.</span></span> <span> </span><span class="NLM_article-title">Pathways to tamoxifen resistance</span>. <i>Cancer Lett.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>256</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">24</span>, <span class="refDoi"> DOI: 10.1016/j.canlet.2007.03.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1016%2Fj.canlet.2007.03.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=17475399" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BD2sXpvFGrtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=256&publication_year=2007&pages=1-24&author=R.+B.+Rigginsauthor=R.+S.+Schrecengostauthor=M.+S.+Guerreroauthor=A.+H.+Bouton&title=Pathways+to+tamoxifen+resistance&doi=10.1016%2Fj.canlet.2007.03.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Pathways to tamoxifen resistance</span></div><div class="casAuthors">Riggins, Rebecca B.; Schrecengost, Randy S.; Guerrero, Michael S.; Bouton, Amy H.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Letters (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">256</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-24</span>CODEN:
                <span class="NLM_cas:coden">CALEDQ</span>;
        ISSN:<span class="NLM_cas:issn">0304-3835</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Therapies that target the synthesis of estrogen or the function of estrogen receptor(s) have been developed to treat breast cancer.  While these approaches have proven to be beneficial to a large no. of patients, both de novo and acquired resistance to these drugs is a significant problem.  Recent advances in our understanding of the mol. mechanisms that contribute to resistance have provided a means to begin to predict patient responses to these drugs and develop rational approaches for combining therapeutic agents to circumvent or desensitize the resistant phenotype.  Here, we review common mechanisms of antiestrogen resistance and discuss the implications for prediction of response and design of effective combinatorial treatments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreh_AisSEW9LVg90H21EOLACvtfcHk0lh7WpxzbW-tig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXpvFGrtLo%253D&md5=25203d3e50ade6508421835d29bf048c</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.canlet.2007.03.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.canlet.2007.03.016%26sid%3Dliteratum%253Aachs%26aulast%3DRiggins%26aufirst%3DR.%2BB.%26aulast%3DSchrecengost%26aufirst%3DR.%2BS.%26aulast%3DGuerrero%26aufirst%3DM.%2BS.%26aulast%3DBouton%26aufirst%3DA.%2BH.%26atitle%3DPathways%2520to%2520tamoxifen%2520resistance%26jtitle%3DCancer%2520Lett.%26date%3D2007%26volume%3D256%26spage%3D1%26epage%3D24%26doi%3D10.1016%2Fj.canlet.2007.03.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Robertson, J. F. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bondarenko, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trishkina, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dvorkin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panasci, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manikhas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shparyk, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cardona-Huerta, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philco-Salas, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruiz-Borrego, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noguchi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rowbottom, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuart, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grinsted, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fazal, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, M. J.</span></span> <span> </span><span class="NLM_article-title">Fulvestrant 500 mg versus anastrozole 1 mg for hormone receptor-positive advanced breast cancer (FALCON): an international, randomised, double-blind, phase 3 trial</span>. <i>Lancet</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>388</i></span>,  <span class="NLM_fpage">2997</span>– <span class="NLM_lpage">3005</span>, <span class="refDoi"> DOI: 10.1016/S0140-6736(16)32389-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1016%2FS0140-6736%2816%2932389-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=27908454" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFKnurnK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=388&publication_year=2016&pages=2997-3005&author=J.+F.+R.+Robertsonauthor=I.+M.+Bondarenkoauthor=E.+Trishkinaauthor=M.+Dvorkinauthor=L.+Panasciauthor=A.+Manikhasauthor=Y.+Shparykauthor=S.+Cardona-Huertaauthor=K.+L.+Cheungauthor=M.+J.+Philco-Salasauthor=M.+Ruiz-Borregoauthor=Z.+Shaoauthor=S.+Noguchiauthor=J.+Rowbottomauthor=M.+Stuartauthor=L.+M.+Grinstedauthor=M.+Fazalauthor=M.+J.+Ellis&title=Fulvestrant+500+mg+versus+anastrozole+1+mg+for+hormone+receptor-positive+advanced+breast+cancer+%28FALCON%29%3A+an+international%2C+randomised%2C+double-blind%2C+phase+3+trial&doi=10.1016%2FS0140-6736%2816%2932389-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Fulvestrant 500 mg versus anastrozole 1 mg for hormone receptor-positive advanced breast cancer (FALCON): an international, randomised, double-blind, phase 3 trial</span></div><div class="casAuthors">Robertson, John F. R.; Bondarenko, Igor M.; Trishkina, Ekaterina; Dvorkin, Mikhail; Panasci, Lawrence; Manikhas, Alexey; Shparyk, Yaroslav; Cardona-Huerta, Servando; Cheung, Kwok-Leung; Philco-Salas, Manuel Jesus; Ruiz-Borrego, Manuel; Shao, Zhimin; Noguchi, Shinzaburo; Rowbottom, Jacqui; Stuart, Mary; Grinsted, Lynda M.; Fazal, Mehdi; Ellis, Matthew J.</div><div class="citationInfo"><span class="NLM_cas:title">Lancet</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">388</span>
        (<span class="NLM_cas:issue">10063</span>),
    <span class="NLM_cas:pages">2997-3005</span>CODEN:
                <span class="NLM_cas:coden">LANCAO</span>;
        ISSN:<span class="NLM_cas:issn">0140-6736</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Aromatase inhibitors are a std. of care for hormone receptor-pos. locally advanced or metastatic breast cancer.  We investigated whether the selective estrogen receptor degrader fulvestrant could improve progression-free survival compared with anastrozole in postmenopausal patients who had not received previous endocrine therapy.  In this phase 3, randomised, double-blind trial, we recruited eligible patients with histol. confirmed estrogen receptor-pos. or progesterone receptor-pos., or both, locally advanced or metastatic breast cancer from 113 academic hospitals and community centers in 20 countries.  Eligible patients were endocrine therapy-naive, with WHO performance status 0-2, and at least one measurable or non-measurable lesion.  Patients were randomly assigned (1:1) to fulvestrant (500 mg i.m. injection; on days 0, 14, 28, then every 28 days thereafter) or anastrozole (1 mg orally daily) using a computer-generated randomisation scheme.  The primary endpoint was progression-free survival, detd. by Response Evaluation Criteria in Solid Tumors version 1·1, intervention by surgery or radiotherapy because of disease deterioration, or death from any cause, assessed in the intention-to-treat population.  Safety outcomes were assessed in all patients who received at least one dose of randomised treatment (including placebo).  This trial is registered with ClinicalTrials.gov, no. NCT01602380.  Between Oct 17, 2012, and July 11, 2014, 524 patients were enrolled to this study.  Of these, 462 patients were randomised (230 to receive fulvestrant and 232 to receive anastrozole).  Progression-free survival was significantly longer in the fulvestrant group than in the anastrozole group (hazard ratio [HR] 0·797, 95% CI 0·637-0·999, p=0·0486).  Median progression-free survival was 16·6 mo (95% CI 13·83-20·99) in the fulvestrant group vs. 13·8 mo (11·99-16·59) in the anastrozole group.  The most common adverse events were arthralgia (38 [17%] in the fulvestrant group vs 24 [10%] in the anastrozole group) and hot flushes (26 [11%] in the fulvestrant group vs 24 [10%] in the anastrozole group). 16 (7%) of 228 patients in in the fulvestrant group and 11 (5%) of 232 patients in the anastrozole group discontinued because of adverse events.  Fulvestrant has superior efficacy and is a preferred treatment option for patients with hormone receptor-pos. locally advanced or metastatic breast cancer who have not received previous endocrine therapy compared with a third-generation aromatase inhibitor, a std. of care for first-line treatment of these patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGry16EI3NgNOLVg90H21EOLACvtfcHk0lh7WpxzbW-tig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFKnurnK&md5=7d8295fdf38fc5eb5720a7d3cd97d6e9</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2FS0140-6736%2816%2932389-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0140-6736%252816%252932389-3%26sid%3Dliteratum%253Aachs%26aulast%3DRobertson%26aufirst%3DJ.%2BF.%2BR.%26aulast%3DBondarenko%26aufirst%3DI.%2BM.%26aulast%3DTrishkina%26aufirst%3DE.%26aulast%3DDvorkin%26aufirst%3DM.%26aulast%3DPanasci%26aufirst%3DL.%26aulast%3DManikhas%26aufirst%3DA.%26aulast%3DShparyk%26aufirst%3DY.%26aulast%3DCardona-Huerta%26aufirst%3DS.%26aulast%3DCheung%26aufirst%3DK.%2BL.%26aulast%3DPhilco-Salas%26aufirst%3DM.%2BJ.%26aulast%3DRuiz-Borrego%26aufirst%3DM.%26aulast%3DShao%26aufirst%3DZ.%26aulast%3DNoguchi%26aufirst%3DS.%26aulast%3DRowbottom%26aufirst%3DJ.%26aulast%3DStuart%26aufirst%3DM.%26aulast%3DGrinsted%26aufirst%3DL.%2BM.%26aulast%3DFazal%26aufirst%3DM.%26aulast%3DEllis%26aufirst%3DM.%2BJ.%26atitle%3DFulvestrant%2520500%2520mg%2520versus%2520anastrozole%25201%2520mg%2520for%2520hormone%2520receptor-positive%2520advanced%2520breast%2520cancer%2520%2528FALCON%2529%253A%2520an%2520international%252C%2520randomised%252C%2520double-blind%252C%2520phase%25203%2520trial%26jtitle%3DLancet%26date%3D2016%26volume%3D388%26spage%3D2997%26epage%3D3005%26doi%3D10.1016%2FS0140-6736%2816%2932389-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Toogood, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harvey, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Repine, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheehan, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">VanderWel, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keller, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNamara, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherry, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brodfuehrer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barvian, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fry, D. W.</span></span> <span> </span><span class="NLM_article-title">Discovery of a potent and selective inhibitor of cyclin-dependent kinase 4/6</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">2388</span>– <span class="NLM_lpage">2406</span>, <span class="refDoi"> DOI: 10.1021/jm049354h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm049354h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BD2MXitVGisL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=2388-2406&author=P.+L.+Toogoodauthor=P.+J.+Harveyauthor=J.+T.+Repineauthor=D.+J.+Sheehanauthor=S.+N.+VanderWelauthor=H.+Zhouauthor=P.+R.+Kellerauthor=D.+J.+McNamaraauthor=D.+Sherryauthor=T.+Zhuauthor=J.+Brodfuehrerauthor=C.+Choiauthor=M.+R.+Barvianauthor=D.+W.+Fry&title=Discovery+of+a+potent+and+selective+inhibitor+of+cyclin-dependent+kinase+4%2F6&doi=10.1021%2Fjm049354h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Potent and Selective Inhibitor of Cyclin-Dependent Kinase 4/6</span></div><div class="casAuthors">Toogood, Peter L.; Harvey, Patricia J.; Repine, Joseph T.; Sheehan, Derek J.; VanderWel, Scott N.; Zhou, Hairong; Keller, Paul R.; McNamara, Dennis J.; Sherry, Debra; Zhu, Tong; Brodfuehrer, Joanne; Choi, Chung; Barvian, Mark R.; Fry, David W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2388-2406</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A pharmacol. approach to inhibition of cyclin-dependent kinases 4 and 6 (Cdk4/6) using highly selective small mol. inhibitors has the potential to provide novel cancer therapies for clin. use.  Achieving high levels of selectivity for Cdk4/6, vs. other ATP-dependent kinases, presents a significant challenge.  The pyrido[2,3-d]pyrimidin-7-one template provides an effective platform for the inhibition of a broad cross-section of kinases, including Cdks.  It is now demonstrated that the modification of pyrido[2,3-d]pyrimidin-7-ones to include a 2-aminopyridine side chain at the C2-position provides inhibitors with exquisite selectivity for Cdk4/6 in vitro.  This selectivity profile is recapitulated in cells where the most selective inhibitors create a G1 block at concns. up to 100-fold the IC50 for cell proliferation.  On the basis of its selectivity profile and pharmacokinetic profile, compd. I (PD 0332991) was identified as a drug candidate for the treatment of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoX577J5U0r97Vg90H21EOLACvtfcHk0lgDGYyP3WOzSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXitVGisL0%253D&md5=23cbe58426ff1975695712db7c1b128a</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Fjm049354h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm049354h%26sid%3Dliteratum%253Aachs%26aulast%3DToogood%26aufirst%3DP.%2BL.%26aulast%3DHarvey%26aufirst%3DP.%2BJ.%26aulast%3DRepine%26aufirst%3DJ.%2BT.%26aulast%3DSheehan%26aufirst%3DD.%2BJ.%26aulast%3DVanderWel%26aufirst%3DS.%2BN.%26aulast%3DZhou%26aufirst%3DH.%26aulast%3DKeller%26aufirst%3DP.%2BR.%26aulast%3DMcNamara%26aufirst%3DD.%2BJ.%26aulast%3DSherry%26aufirst%3DD.%26aulast%3DZhu%26aufirst%3DT.%26aulast%3DBrodfuehrer%26aufirst%3DJ.%26aulast%3DChoi%26aufirst%3DC.%26aulast%3DBarvian%26aufirst%3DM.%2BR.%26aulast%3DFry%26aufirst%3DD.%2BW.%26atitle%3DDiscovery%2520of%2520a%2520potent%2520and%2520selective%2520inhibitor%2520of%2520cyclin-dependent%2520kinase%25204%252F6%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D2388%26epage%3D2406%26doi%3D10.1021%2Fjm049354h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Finn, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crown, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lang, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boer, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bondarenko, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kulyk, S. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ettl, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinter, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shparyk, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thummala, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voytko, N. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fowst, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Randolph, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slamon, D. J.</span></span> <span> </span><span class="NLM_article-title">The cyclin-dependent kinase 4/6 inhibitor palbociclib in combination with letrozole versus letrozole alone as first-line treatment of oestrogen receptor-positive, HER2-negative, advanced breast cancer (PALOMA-1/TRIO-18): a randomised phase 2 study</span>. <i>Lancet Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">25</span>– <span class="NLM_lpage">35</span>, <span class="refDoi"> DOI: 10.1016/S1470-2045(14)71159-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1016%2FS1470-2045%2814%2971159-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=25524798" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1yhsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2015&pages=25-35&author=R.+S.+Finnauthor=J.+P.+Crownauthor=I.+Langauthor=K.+Boerauthor=I.+M.+Bondarenkoauthor=S.+O.+Kulykauthor=J.+Ettlauthor=R.+Patelauthor=T.+Pinterauthor=M.+Schmidtauthor=Y.+Shparykauthor=A.+R.+Thummalaauthor=N.+L.+Voytkoauthor=C.+Fowstauthor=X.+Huangauthor=S.+T.+Kimauthor=S.+Randolphauthor=D.+J.+Slamon&title=The+cyclin-dependent+kinase+4%2F6+inhibitor+palbociclib+in+combination+with+letrozole+versus+letrozole+alone+as+first-line+treatment+of+oestrogen+receptor-positive%2C+HER2-negative%2C+advanced+breast+cancer+%28PALOMA-1%2FTRIO-18%29%3A+a+randomised+phase+2+study&doi=10.1016%2FS1470-2045%2814%2971159-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">The cyclin-dependent kinase 4/6 inhibitor palbociclib in combination with letrozole versus letrozole alone as first-line treatment of oestrogen receptor-positive, HER2-negative, advanced breast cancer (PALOMA-1/TRIO-18): a randomised phase 2 study</span></div><div class="casAuthors">Finn, Richard S.; Crown, John P.; Lang, Istvan; Boer, Katalin; Bondarenko, Igor M.; Kulyk, Sergey O.; Ettl, Johannes; Patel, Ravindranath; Pinter, Tamas; Schmidt, Marcus; Shparyk, Yaroslav; Thummala, Anu R.; Voytko, Nataliya L.; Fowst, Camilla; Huang, Xin; Kim, Sindy T.; Randolph, Sophia; Slamon, Dennis J.</div><div class="citationInfo"><span class="NLM_cas:title">Lancet Oncology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">25-35</span>CODEN:
                <span class="NLM_cas:coden">LOANBN</span>;
        ISSN:<span class="NLM_cas:issn">1470-2045</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Palbociclib (PD-0332991) is an oral, small-mol. inhibitor of cyclin-dependent kinases (CDKs) 4 and 6 with preclin. evidence of growth-inhibitory activity in estrogen receptor-pos. breast cancer cells and synergy with anti-estrogens.  We aimed to assess the safety and efficacy of palbociclib in combination with letrozole as first-line treatment of patients with advanced, estrogen receptor-pos., HER2-neg. breast cancer.  In this open-label, randomised phase 2 study, postmenopausal women with advanced estrogen receptor-pos. and HER2-neg. breast cancer who had not received any systemic treatment for their advanced disease were eligible to participate.  Patients were enrolled in two sep. cohorts that accrued sequentially: in cohort 1, patients were enrolled on the basis of their estrogen receptor-pos. and HER2-neg. biomarker status alone, whereas in cohort 2 they were also required to have cancers with amplification of cyclin D1 (CCND1), loss of p16 (INK4A or CDKN2A), or both.  In both cohorts, patients were randomly assigned 1:1 via an interactive web-based randomization system, stratified by disease site and disease-free interval, to receive continuous oral letrozole 2·5 mg daily or continuous oral letrozole 2·5 mg daily plus oral palbociclib 125 mg, given once daily for 3 wk followed by 1 wk off over 28-day cycles.  The primary endpoint was investigator-assessed progression-free survival in the intention-to-treat population.  Accrual to cohort 2 was stopped after an unplanned interim anal. of cohort 1 and the statistical anal. plan for the primary endpoint was amended to a combined anal. of cohorts 1 and 2 (instead of cohort 2 alone).  The study is ongoing but closed to accrual; these are the results of the final anal. of progression-free survival.  The study is registered with the ClinicalTrials.gov, no. NCT00721409.  Between Dec 22, 2009, and May 12, 2012, we randomly assigned 165 patients, 84 to palbociclib plus letrozole and 81 to letrozole alone.  At the time of the final anal. for progression-free survival (median follow-up 29·6 mo [95% CI 27·9-36·0] for the palbociclib plus letrozole group and 27·9 mo [25·5-31·1] for the letrozole group), 41 progression-free survival events had occurred in the palbociclib plus letrozole group and 59 in the letrozole group.  Median progression-free survival was 10·2 mo (95% CI 5·7-12·6) for the letrozole group and 20·2 mo (13·8-27·5) for the palbociclib plus letrozole group (HR 0·488, 95% CI 0·319-0·748; one-sided p=0·0004).  In cohort 1 (n=66), median progression-free survival was 5·7 mo (2·6-10·5) for the letrozole group and 26·1 mo (11·2-not estimable) for the palbociclib plus letrozole group (HR 0·299, 0·156-0·572; one-sided p<0·0001); in cohort 2 (n=99), median progression-free survival was 11·1 mo (7·1-16·4) for the letrozole group and 18·1 mo (13·1-27·5) for the palbociclib plus letrozole group (HR 0·508, 0·303-0·853; one-sided p=0·0046).  Grade 3-4 neutropenia was reported in 45 (54%) of 83 patients in the palbociclib plus letrozole group vs. one (1%) of 77 patients in the letrozole group, leucopenia in 16 (19%) vs. none, and fatigue in four (4%) vs. one (1%).  Serious adverse events that occurred in more than one patient in the palbociclib plus letrozole group were pulmonary embolism (three [4%] patients), back pain (two [2%]), and diarrhoea (two [2%]).  No cases of febrile neutropenia or neutropenia-related infections were reported during the study. 11 (13%) patients in the palbociclib plus letrozole group and two (2%) in the letrozole group discontinued the study because of adverse events.  The addn. of palbociclib to letrozole in this phase 2 study significantly improved progression-free survival in women with advanced estrogen receptor-pos. and HER2-neg. breast cancer.  A phase 3 trial is currently underway.Pfizer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBgFyFPQQlcbVg90H21EOLACvtfcHk0lgDGYyP3WOzSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1yhsb8%253D&md5=9163a35e22c50fb25782d439aff88bcb</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2FS1470-2045%2814%2971159-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1470-2045%252814%252971159-3%26sid%3Dliteratum%253Aachs%26aulast%3DFinn%26aufirst%3DR.%2BS.%26aulast%3DCrown%26aufirst%3DJ.%2BP.%26aulast%3DLang%26aufirst%3DI.%26aulast%3DBoer%26aufirst%3DK.%26aulast%3DBondarenko%26aufirst%3DI.%2BM.%26aulast%3DKulyk%26aufirst%3DS.%2BO.%26aulast%3DEttl%26aufirst%3DJ.%26aulast%3DPatel%26aufirst%3DR.%26aulast%3DPinter%26aufirst%3DT.%26aulast%3DSchmidt%26aufirst%3DM.%26aulast%3DShparyk%26aufirst%3DY.%26aulast%3DThummala%26aufirst%3DA.%2BR.%26aulast%3DVoytko%26aufirst%3DN.%2BL.%26aulast%3DFowst%26aufirst%3DC.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DKim%26aufirst%3DS.%2BT.%26aulast%3DRandolph%26aufirst%3DS.%26aulast%3DSlamon%26aufirst%3DD.%2BJ.%26atitle%3DThe%2520cyclin-dependent%2520kinase%25204%252F6%2520inhibitor%2520palbociclib%2520in%2520combination%2520with%2520letrozole%2520versus%2520letrozole%2520alone%2520as%2520first-line%2520treatment%2520of%2520oestrogen%2520receptor-positive%252C%2520HER2-negative%252C%2520advanced%2520breast%2520cancer%2520%2528PALOMA-1%252FTRIO-18%2529%253A%2520a%2520randomised%2520phase%25202%2520study%26jtitle%3DLancet%2520Oncol.%26date%3D2015%26volume%3D16%26spage%3D25%26epage%3D35%26doi%3D10.1016%2FS1470-2045%2814%2971159-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Perey, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paridaens, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hawle, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nole, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wildiers, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fiche, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dietrich, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clement, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koberle, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldhirsch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thurlimann, B.</span></span> <span> </span><span class="NLM_article-title">Clinical benefit of fulvestrant in postmenopausal women with advanced breast cancer and primary or acquired resistance to aromatase inhibitors: final results of phase II Swiss Group for Clinical Cancer Research Trial (SAKK 21/00)</span>. <i>Ann. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">64</span>– <span class="NLM_lpage">69</span>, <span class="refDoi"> DOI: 10.1093/annonc/mdl341</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1093%2Fannonc%2Fmdl341" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=17030543" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A280%3ADC%252BD2s%252FisVWntQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2007&pages=64-69&author=L.+Pereyauthor=R.+Paridaensauthor=H.+Hawleauthor=K.+Zamanauthor=F.+Noleauthor=H.+Wildiersauthor=M.+Ficheauthor=D.+Dietrichauthor=P.+Clementauthor=D.+Koberleauthor=A.+Goldhirschauthor=B.+Thurlimann&title=Clinical+benefit+of+fulvestrant+in+postmenopausal+women+with+advanced+breast+cancer+and+primary+or+acquired+resistance+to+aromatase+inhibitors%3A+final+results+of+phase+II+Swiss+Group+for+Clinical+Cancer+Research+Trial+%28SAKK+21%2F00%29&doi=10.1093%2Fannonc%2Fmdl341"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Clinical benefit of fulvestrant in postmenopausal women with advanced breast cancer and primary or acquired resistance to aromatase inhibitors: final results of phase II Swiss Group for Clinical Cancer Research Trial (SAKK 21/00)</span></div><div class="casAuthors">Perey L; Paridaens R; Hawle H; Zaman K; Nole F; Wildiers H; Fiche M; Dietrich D; Clement P; Koberle D; Goldhirsch A; Thurlimann B</div><div class="citationInfo"><span class="NLM_cas:title">Annals of oncology : official journal of the European Society for Medical Oncology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">64-9</span>
        ISSN:<span class="NLM_cas:issn">0923-7534</span>.
    </div><div class="casAbstract">BACKGROUND:  The aim of this study was to evaluate the efficacy and tolerability of fulvestrant, an estrogen receptor antagonist, in postmenopausal women with hormone-responsive tumors progressing after aromatase inhibitor (AI) treatment.  PATIENTS AND METHODS:  This is a phase II, open, multicenter, noncomparative study.  Two patient groups were prospectively considered: group A (n=70) with AI-responsive disease and group B (n=20) with AI-resistant disease.  Fulvestrant 250 mg was administered as intramuscular injection every 28 (+/-3) days.  RESULTS:  All patients were pretreated with AI and 84% also with tamoxifen or toremifene; 67% had bone metastases and 45% liver metastases.  Fulvestrant administration was well tolerated and yielded a clinical benefit (CB; defined as objective response or stable disease [SD] for >or=24 weeks) in 28% (90% confidence interval [CI] 19% to 39%) of patients in group A and 37% (90% CI 19% to 58%) of patients in group B.  Median time to progression (TTP) was 3.6 (95% CI 3.0 to 4.8) months in group A and 3.4 (95% CI 2.5 to 6.7) months in group B.  CONCLUSIONS:  Overall, 30% of patients who had progressed following prior AI treatment gained CB with fulvestrant, thereby delaying indication to start chemotherapy.  Prior response to an AI did not appear to be predictive for benefit with fulvestrant.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQgdgcuVoVvhr7NkxioyDXTfW6udTcc2ebNrXBkKIA-Dbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2s%252FisVWntQ%253D%253D&md5=33f52d40ebeba134195a6311c0e8d854</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1093%2Fannonc%2Fmdl341&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fannonc%252Fmdl341%26sid%3Dliteratum%253Aachs%26aulast%3DPerey%26aufirst%3DL.%26aulast%3DParidaens%26aufirst%3DR.%26aulast%3DHawle%26aufirst%3DH.%26aulast%3DZaman%26aufirst%3DK.%26aulast%3DNole%26aufirst%3DF.%26aulast%3DWildiers%26aufirst%3DH.%26aulast%3DFiche%26aufirst%3DM.%26aulast%3DDietrich%26aufirst%3DD.%26aulast%3DClement%26aufirst%3DP.%26aulast%3DKoberle%26aufirst%3DD.%26aulast%3DGoldhirsch%26aufirst%3DA.%26aulast%3DThurlimann%26aufirst%3DB.%26atitle%3DClinical%2520benefit%2520of%2520fulvestrant%2520in%2520postmenopausal%2520women%2520with%2520advanced%2520breast%2520cancer%2520and%2520primary%2520or%2520acquired%2520resistance%2520to%2520aromatase%2520inhibitors%253A%2520final%2520results%2520of%2520phase%2520II%2520Swiss%2520Group%2520for%2520Clinical%2520Cancer%2520Research%2520Trial%2520%2528SAKK%252021%252F00%2529%26jtitle%3DAnn.%2520Oncol.%26date%3D2007%26volume%3D18%26spage%3D64%26epage%3D69%26doi%3D10.1093%2Fannonc%2Fmdl341" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ingle, J. N.</span></span> <span> </span><span class="NLM_article-title">Estrogen as therapy for breast cancer</span>. <i>Breast Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">133</span>– <span class="NLM_lpage">136</span>, <span class="refDoi"> DOI: 10.1186/bcr436</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1186%2Fbcr436" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=12100736" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BD38XltlWksL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2002&pages=133-136&author=J.+N.+Ingle&title=Estrogen+as+therapy+for+breast+cancer&doi=10.1186%2Fbcr436"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Estrogen as therapy for breast cancer</span></div><div class="casAuthors">Ingle, James N.</div><div class="citationInfo"><span class="NLM_cas:title">Breast Cancer Research [online computer file]</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">133-136</span>CODEN:
                <span class="NLM_cas:coden">BRCRFS</span>;
        ISSN:<span class="NLM_cas:issn">1465-542X</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">A review.  High-dose estrogen was generally considered the endocrine therapy of choice for postmenopausal women with breast cancer prior to the introduction of tamoxifen.  Subsequently, the use of estrogen was largely abandoned.  Recent clin. trial data have shown clin. meaningful efficacy for high-dose estrogen even in patients with extensive prior endocrine therapy.  Preclin. research has demonstrated that the estrogen dose-response curve for breast cancer cells can be shifted by modification of the estrogen environment.  Clin. and lab. data together provide the basis for developing testable hypotheses of management strategies, with the potential of increasing the value of endocrine therapy in women with breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_shZXrDZkBrVg90H21EOLACvtfcHk0lhTqovxDRfdvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XltlWksL8%253D&md5=5d65ac9d3527c51b9977071c3af41c2e</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1186%2Fbcr436&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fbcr436%26sid%3Dliteratum%253Aachs%26aulast%3DIngle%26aufirst%3DJ.%2BN.%26atitle%3DEstrogen%2520as%2520therapy%2520for%2520breast%2520cancer%26jtitle%3DBreast%2520Cancer%2520Res.%26date%3D2002%26volume%3D4%26spage%3D133%26epage%3D136%26doi%3D10.1186%2Fbcr436" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dowsett, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicholson, R. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pietras, R. J.</span></span> <span> </span><span class="NLM_article-title">Biological characteristics of the pure antiestrogen fulvestrant: overcoming endocrine resistance</span>. <i>Breast Cancer Res. Treat.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>93</i></span> (<span class="NLM_issue">Suppl. 1</span>),  <span class="NLM_fpage">11</span>– <span class="NLM_lpage">18</span>, <span class="refDoi"> DOI: 10.1007/s10549-005-9037-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1007%2Fs10549-005-9037-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtFCrtb3P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=2005&pages=11-18&issue=Suppl.+1&author=M.+Dowsettauthor=R.+I.+Nicholsonauthor=R.+J.+Pietras&title=Biological+characteristics+of+the+pure+antiestrogen+fulvestrant%3A+overcoming+endocrine+resistance&doi=10.1007%2Fs10549-005-9037-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Biological characteristics of the pure antiestrogen fulvestrant: overcoming endocrine resistance</span></div><div class="casAuthors">Dowsett, Mitch; Nicholson, Robert I.; Pietras, Richard J.</div><div class="citationInfo"><span class="NLM_cas:title">Breast Cancer Research and Treatment</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue">Suppl. 1</span>),
    <span class="NLM_cas:pages">S11-S18</span>CODEN:
                <span class="NLM_cas:coden">BCTRD6</span>;
        ISSN:<span class="NLM_cas:issn">0167-6806</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Understanding the underlying mechanisms responsible for endocrine resistance remains a challenge in improving the treatment of breast cancer.  The discovery that growth factor and estrogen receptor (ER) signaling pathways interact in endocrine resistant breast cancer has provided a rationale for disrupting these signaling cascades in ER-pos., endocrine-resistant tumors.  In postmenopausal women, the ER signaling pathway may be targeted using fulvestrant ('Faslodex'), a new type of ER antagonist with no agonist effects.  Fulvestrant binds, blocks and causes degrdn. of the ER, culminating in complete abrogation of estrogen-sensitive gene transcription.  This unique mechanism of action may result in a lack of cross-resistance with other endocrine agents.  Preclin. studies have confirmed the potential of fulvestrant to inhibit the growth of tamoxifen-resistant, as well as tamoxifen-sensitive, human breast cancer cell lines.  Clin. studies have demonstrated that fulvestrant is an effective treatment option in postmenopausal women with advanced breast cancer who have progressed on prior endocrine therapy.  Furthermore, preclin. studies indicate that combining fulvestrant with growth factor targeted agents, such as the epidermal growth factor receptor (EGFR/HER1) tyrosine kinase inhibitor gefitinib (IRESSA) or the anti-human HER2 monoclonal antibody trastuzumab ('Herceptin'), may result in greater anti-tumor activity than either agent alone.  A range of clin. trials are now ongoing to det. whether the combination of growth factor-targeting agents with fulvestrant will delay the onset of endocrine resistance and so provide new strategy for women with hormone receptor-pos. advanced breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4LULVK2HFH7Vg90H21EOLACvtfcHk0lhTqovxDRfdvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtFCrtb3P&md5=e082bfa908da811b2b78afd04138971a</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1007%2Fs10549-005-9037-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10549-005-9037-3%26sid%3Dliteratum%253Aachs%26aulast%3DDowsett%26aufirst%3DM.%26aulast%3DNicholson%26aufirst%3DR.%2BI.%26aulast%3DPietras%26aufirst%3DR.%2BJ.%26atitle%3DBiological%2520characteristics%2520of%2520the%2520pure%2520antiestrogen%2520fulvestrant%253A%2520overcoming%2520endocrine%2520resistance%26jtitle%3DBreast%2520Cancer%2520Res.%2520Treat.%26date%3D2005%26volume%3D93%26issue%3DSuppl.%25201%26spage%3D11%26epage%3D18%26doi%3D10.1007%2Fs10549-005-9037-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Osborne, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiff, R.</span></span> <span> </span><span class="NLM_article-title">Mechanisms of Endocrine Resistance in Breast Cancer</span>. <i>Annu. Rev. Med.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">233</span>– <span class="NLM_lpage">247</span>, <span class="refDoi"> DOI: 10.1146/annurev-med-070909-182917</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1146%2Fannurev-med-070909-182917" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=20887199" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC3MXivVWhtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2011&pages=233-247&author=C.+K.+Osborneauthor=R.+Schiff&title=Mechanisms+of+Endocrine+Resistance+in+Breast+Cancer&doi=10.1146%2Fannurev-med-070909-182917"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms of endocrine resistance in breast cancer</span></div><div class="casAuthors">Osborne, C. Kent; Schiff, Rachel</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Medicine</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">233-247</span>CODEN:
                <span class="NLM_cas:coden">ARMCAH</span>;
        ISSN:<span class="NLM_cas:issn">0066-4219</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  The estrogen receptor (ER) pathway plays a pivotal role in breast cancer development and progression.  Endocrine therapy to block the ER pathway is highly effective, but its usefulness is limited by common intrinsic and acquired resistance.  Multiple mechanisms responsible for endocrine resistance have been proposed and include deregulation of various components of the ER pathway itself, alterations in cell cycle and cell survival signaling mols., and the activation of escape pathways that can provide tumors with alternative proliferative and survival stimuli.  Among these, increased expression or signaling of growth factor receptor pathways, esp. the EGFR/HER2 pathway, has been assocd. with both exptl. and clin. endocrine therapy resistance.  New treatment combinations targeting both ER and growth factor receptor signaling to block the crosstalk between these pathways and eliminate escape routes have been proven highly effective in preclin. models.  Results of recent clin. studies, while partly supporting this approach, also highlight the need to better identify a priori the patients whose tumors are most likely to benefit from these specific cotargeting strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp23PYELtZJrbVg90H21EOLACvtfcHk0lhTqovxDRfdvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXivVWhtLo%253D&md5=48573e39ea81057b53b5b43b1d72d2d9</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1146%2Fannurev-med-070909-182917&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-med-070909-182917%26sid%3Dliteratum%253Aachs%26aulast%3DOsborne%26aufirst%3DC.%2BK.%26aulast%3DSchiff%26aufirst%3DR.%26atitle%3DMechanisms%2520of%2520Endocrine%2520Resistance%2520in%2520Breast%2520Cancer%26jtitle%3DAnnu.%2520Rev.%2520Med.%26date%3D2011%26volume%3D62%26spage%3D233%26epage%3D247%26doi%3D10.1146%2Fannurev-med-070909-182917" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Frogne, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benjaminsen, R. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sonne-Hansen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sorensen, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nexo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laenkholm, A.-V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rasmussen, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riese, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Cremoux, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stenvang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lykkesfeldt, A. E.</span></span> <span> </span><span class="NLM_article-title">Activation of ErbB3, EGFR and Erk is essential for growth of human breast cancer cell lines with acquired resistance to fulvestrant</span>. <i>Breast Cancer Res. Treat.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">263</span>– <span class="NLM_lpage">275</span>, <span class="refDoi"> DOI: 10.1007/s10549-008-0011-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1007%2Fs10549-008-0011-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=18409071" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFWnu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2009&pages=263-275&author=T.+Frogneauthor=R.+V.+Benjaminsenauthor=K.+Sonne-Hansenauthor=B.+S.+Sorensenauthor=E.+Nexoauthor=A.-V.+Laenkholmauthor=L.+M.+Rasmussenauthor=D.+J.+Rieseauthor=P.+de+Cremouxauthor=J.+Stenvangauthor=A.+E.+Lykkesfeldt&title=Activation+of+ErbB3%2C+EGFR+and+Erk+is+essential+for+growth+of+human+breast+cancer+cell+lines+with+acquired+resistance+to+fulvestrant&doi=10.1007%2Fs10549-008-0011-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Activation of ErbB3, EGFR and Erk is essential for growth of human breast cancer cell lines with acquired resistance to fulvestrant</span></div><div class="casAuthors">Frogne, Thomas; Benjaminsen, Rikke V.; Sonne-Hansen, Katrine; Sorensen, Boe S.; Nexo, Ebba; Laenkholm, Anne-Vibeke; Rasmussen, Louise M.; Riese, David J., II; de Cremoux, Patricia; Stenvang, Jan; Lykkesfeldt, Anne E.</div><div class="citationInfo"><span class="NLM_cas:title">Breast Cancer Research and Treatment</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">263-275</span>CODEN:
                <span class="NLM_cas:coden">BCTRD6</span>;
        ISSN:<span class="NLM_cas:issn">0167-6806</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Seven fulvestrant resistant cell lines derived from the estrogen receptor α pos. MCF-7 human breast cancer cell line were used to investigate the importance of epidermal growth factor receptor (ErbB1-4) signaling.  We found an increase in mRNA expression of EGFR and the ErbB3/ErbB4 ligand heregulin2 (hrg2) and a decrease of ErbB4 in all resistant cell lines.  Western analyses confirmed the upregulation of EGFR and hrg2 and the downregulation of ErbB4.  Elevated activation of EGFR and ErbB3 was seen in all resistant cell lines and the ErbB3 activation occurred by an autocrine mechanism.  ErbB4 activation was obsd. only in the parental MCF-7 cells.  The downstream kinases pAkt and pErk were increased in five of seven and in all seven resistant cell lines, resp.  Treatment with the EGFR inhibitor gefitinib preferentially inhibited growth and reduced the S phase fraction in the resistant cell lines concomitant with inhibition of Erk and unaltered Akt activation.  In concert, inhibition of Erk with U0126 preferentially reduced growth of resistant cell lines.  Treatment with ErbB3 neutralizing antibodies inhibited ErbB3 activation and resulted in a modest but statistically significant growth inhibition of two resistant cell lines.  These data indicate that ligand activated ErbB3 and EGFR, and Erk signaling play important roles in fulvestrant resistant cell growth.  Furthermore, the decreased level of ErbB4 in resistant cells may facilitate heterodimerization of ErbB3 with EGFR and ErbB2.  Our data support that a concerted action against EGFR, ErbB2 and ErbB3 may be required to obtain complete growth suppression of fulvestrant resistant cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_Dd87hGSoHLVg90H21EOLACvtfcHk0lhTqovxDRfdvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFWnu74%253D&md5=3a477e5836c0056d9af0bd67e120e2f5</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1007%2Fs10549-008-0011-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10549-008-0011-8%26sid%3Dliteratum%253Aachs%26aulast%3DFrogne%26aufirst%3DT.%26aulast%3DBenjaminsen%26aufirst%3DR.%2BV.%26aulast%3DSonne-Hansen%26aufirst%3DK.%26aulast%3DSorensen%26aufirst%3DB.%2BS.%26aulast%3DNexo%26aufirst%3DE.%26aulast%3DLaenkholm%26aufirst%3DA.-V.%26aulast%3DRasmussen%26aufirst%3DL.%2BM.%26aulast%3DRiese%26aufirst%3DD.%2BJ.%26aulast%3Dde%2BCremoux%26aufirst%3DP.%26aulast%3DStenvang%26aufirst%3DJ.%26aulast%3DLykkesfeldt%26aufirst%3DA.%2BE.%26atitle%3DActivation%2520of%2520ErbB3%252C%2520EGFR%2520and%2520Erk%2520is%2520essential%2520for%2520growth%2520of%2520human%2520breast%2520cancer%2520cell%2520lines%2520with%2520acquired%2520resistance%2520to%2520fulvestrant%26jtitle%3DBreast%2520Cancer%2520Res.%2520Treat.%26date%3D2009%26volume%3D114%26spage%3D263%26epage%3D275%26doi%3D10.1007%2Fs10549-008-0011-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Massarweh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osborne, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wakeling, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rimawi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohsin, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilsenbeck, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiff, R.</span></span> <span> </span><span class="NLM_article-title">Mechanisms of tumor regression and resistance to estrogen deprivation and fulvestrant in a model of estrogen receptor-positive, HER-2/neu-positive breast cancer</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">8266</span>– <span class="NLM_lpage">8273</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-05-4045</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1158%2F0008-5472.CAN-05-4045" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=16912207" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BD28XotFWqurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2006&pages=8266-8273&author=S.+Massarwehauthor=C.+K.+Osborneauthor=S.+Jiangauthor=A.+E.+Wakelingauthor=M.+Rimawiauthor=S.+K.+Mohsinauthor=S.+Hilsenbeckauthor=R.+Schiff&title=Mechanisms+of+tumor+regression+and+resistance+to+estrogen+deprivation+and+fulvestrant+in+a+model+of+estrogen+receptor-positive%2C+HER-2%2Fneu-positive+breast+cancer&doi=10.1158%2F0008-5472.CAN-05-4045"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms of Tumor Regression and Resistance to Estrogen Deprivation and Fulvestrant in a Model of Estrogen Receptor-Positive, HER-2/neu-Positive Breast Cancer</span></div><div class="casAuthors">Massarweh, Suleiman; Osborne, C. Kent; Jiang, Shou; Wakeling, Alan E.; Rimawi, Mothaffar; Mohsin, Syed K.; Hilsenbeck, Susan; Schiff, Rachel</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">8266-8273</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">HER-2/neu in breast cancer is assocd. with tamoxifen resistance, but little data exist on its interaction with estrogen deprivation or fulvestrant.  Here, we used an in vivo xenograft model of estrogen receptor (ER)-pos. breast cancer with HER-2/neu overexpression (MCF7/HER-2/neu-18) to investigate mechanisms of growth inhibition and treatment resistance.  MCF7/HER-2/neu-18 tumors were growth inhibited by estrogen deprivation and with fulvestrant, but resistance developed in 2 to 3 mo.  Inhibited tumors had redns. in ER, insulin-like growth factor-I receptor (IGF-IR), phosphorylated HER-2/neu (p-HER-2/neu), and phosphorylated p42/44 mitogen-activated protein kinase (p-MAPK).  P27 was increased esp. in tumors sensitive to estrogen deprivation.  Tumors with acquired resistance to these therapies had complete loss of ER, increased p-HER-2/neu, increased p-MAPK, and reduced p27.  In contrast, IGF-IR and phosphorylated AKT (p-AKT) levels were markedly reduced in these resistant tumors.  The epidermal growth factor receptor (EGFR) tyrosine kinase inhibitor gefitinib, which can block EGFR/HER-2/neu signaling, significantly delayed the emergence of resistance to both estrogen deprivation and fulvestrant.  Levels of p-MAPK and p-AKT decreased with gefitinib, whereas high ER levels were restored.  Eventually, however, tumors progressed in mice treated with gefitinib combined with estrogen deprivation or fulvestrant accompanied again by loss of ER and IGF-IR, increased p-HER-2/neu, high p-MAPK, and now increased p-AKT.  Thus, estrogen deprivation and fulvestrant can effectively inhibit HER-2/neu-overexpressing tumors but resistance develops quickly.  EGFR/HER-2/neu inhibitors can delay resistance, but reactivation of HER-2/neu and signaling through AKT leads to tumor regrowth.  Combining endocrine therapy with EGFR/HER-2/neu inhibitors should be tested in clin. breast cancer, but a more complete blockade of EGFR/HER-2/neu may be optimal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5TEeU_-NI5bVg90H21EOLACvtfcHk0li9t7aZTE8RHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XotFWqurs%253D&md5=23e976f76a73ac68f7e63bec37f33127</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-05-4045&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-05-4045%26sid%3Dliteratum%253Aachs%26aulast%3DMassarweh%26aufirst%3DS.%26aulast%3DOsborne%26aufirst%3DC.%2BK.%26aulast%3DJiang%26aufirst%3DS.%26aulast%3DWakeling%26aufirst%3DA.%2BE.%26aulast%3DRimawi%26aufirst%3DM.%26aulast%3DMohsin%26aufirst%3DS.%2BK.%26aulast%3DHilsenbeck%26aufirst%3DS.%26aulast%3DSchiff%26aufirst%3DR.%26atitle%3DMechanisms%2520of%2520tumor%2520regression%2520and%2520resistance%2520to%2520estrogen%2520deprivation%2520and%2520fulvestrant%2520in%2520a%2520model%2520of%2520estrogen%2520receptor-positive%252C%2520HER-2%252Fneu-positive%2520breast%2520cancer%26jtitle%3DCancer%2520Res.%26date%3D2006%26volume%3D66%26spage%3D8266%26epage%3D8273%26doi%3D10.1158%2F0008-5472.CAN-05-4045" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Leva, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Devlin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartman-Frey, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burow, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ivan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Croce, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nephew, K. P.</span></span> <span> </span><span class="NLM_article-title">MicroRNA-221/222 confers breast cancer fulvestrant resistance by regulating multiple signaling pathways</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">1082</span>– <span class="NLM_lpage">1097</span>, <span class="refDoi"> DOI: 10.1038/onc.2010.487</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1038%2Fonc.2010.487" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=21057537" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtl2jtrfF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2011&pages=1082-1097&author=X.+Raoauthor=G.+Di+Levaauthor=M.+Liauthor=F.+Fangauthor=C.+Devlinauthor=C.+Hartman-Freyauthor=M.+E.+Burowauthor=M.+Ivanauthor=C.+M.+Croceauthor=K.+P.+Nephew&title=MicroRNA-221%2F222+confers+breast+cancer+fulvestrant+resistance+by+regulating+multiple+signaling+pathways&doi=10.1038%2Fonc.2010.487"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">MicroRNA-221/222 confers breast cancer fulvestrant resistance by regulating multiple signaling pathways</span></div><div class="casAuthors">Rao, X.; Di Leva, G.; Li, M.; Fang, F.; Devlin, C.; Hartman-Frey, C.; Burow, M. E.; Ivan, M.; Croce, C. M.; Nephew, K. P.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1082-1097</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Fulvestrant is a selective estrogen receptor downregulator (SERD) and highly effective antagonist to hormone-sensitive breast cancers following failure of previous tamoxifen or aromatase inhibitor therapies.  However, after prolonged fulvestrant therapy, acquired resistance eventually occurs in the majority of breast cancer patients, due to poorly understood mechanisms.  To examine a possible role(s) of aberrantly expressed microRNAs (miRNAs) in acquired fulvestrant resistance, we compared antiestrogen-resistant and -sensitive breast cancer cells, revealing the overexpression of miR-221/222 in the SERD-resistant cell lines.  Fulvestrant treatment of estradiol (E2)- and fulvestrant-sensitive MCF7 cells resulted in increased expression of endogenous miR-221/222.  Ectopic upregulation of miR-221/222 in estrogen receptor-α (ERα)-pos. cell lines counteracted the effects of E2 depletion or fulvestrant-induced cell death, thus also conferring hormone-independent growth and fulvestrant resistance.  In cells with acquired resistance to fulvestrant, miR-221/222 expression was essential for cell growth and cell cycle progression.  To identify possible miR-221/222 targets, miR-221- or miR-222- induced alterations in global gene expression profiles and target gene expression at distinct time points were detd., revealing that miR-221/222 overexpression resulted in deregulation of multiple oncogenic signaling pathways previously assocd. with drug resistance.  Activation of β-catenin by miR-221/222 contributed to estrogen-independent growth and fulvestrant resistance, whereas TGF-β-mediated growth inhibition was repressed by the 2 miRNAs.  This first in-depth investigation into the role of miR-221/222 in acquired fulvestrant resistance, a clin. important problem, demonstrates that these 2 oncomirs' may represent promising therapeutic targets for treating hormone-independent, SERD-resistant breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqh79Pw5CWhrVg90H21EOLACvtfcHk0li9t7aZTE8RHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtl2jtrfF&md5=24f2c05c330c8735230247ba02fc9856</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fonc.2010.487&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2010.487%26sid%3Dliteratum%253Aachs%26aulast%3DRao%26aufirst%3DX.%26aulast%3DDi%2BLeva%26aufirst%3DG.%26aulast%3DLi%26aufirst%3DM.%26aulast%3DFang%26aufirst%3DF.%26aulast%3DDevlin%26aufirst%3DC.%26aulast%3DHartman-Frey%26aufirst%3DC.%26aulast%3DBurow%26aufirst%3DM.%2BE.%26aulast%3DIvan%26aufirst%3DM.%26aulast%3DCroce%26aufirst%3DC.%2BM.%26aulast%3DNephew%26aufirst%3DK.%2BP.%26atitle%3DMicroRNA-221%252F222%2520confers%2520breast%2520cancer%2520fulvestrant%2520resistance%2520by%2520regulating%2520multiple%2520signaling%2520pathways%26jtitle%3DOncogene%26date%3D2011%26volume%3D30%26spage%3D1082%26epage%3D1097%26doi%3D10.1038%2Fonc.2010.487" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Belkina, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denis, G. V.</span></span> <span> </span><span class="NLM_article-title">BET domain co-regulators in obesity, inflammation and cancer</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">465</span>– <span class="NLM_lpage">477</span>, <span class="refDoi"> DOI: 10.1038/nrc3256</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1038%2Fnrc3256" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=22722403" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC38XovFyks7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2012&pages=465-477&author=A.+C.+Belkinaauthor=G.+V.+Denis&title=BET+domain+co-regulators+in+obesity%2C+inflammation+and+cancer&doi=10.1038%2Fnrc3256"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">BET domain co-regulators in obesity, inflammation and cancer</span></div><div class="casAuthors">Belkina, Anna C.; Denis, Gerald V.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">465-477</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The bromodomain is a highly conserved motif of 110 amino acids that is bundled into four anti-parallel α-helixes and found in proteins that interact with chromatin, such as transcription factors, histone acetylases and nucleosome remodelling complexes.  Bromodomain proteins are chromatin 'readers'; they recruit chromatin-regulating enzymes, including 'writers' and 'erasers' of histone modification, to target promoters and to regulate gene expression.  Conventional wisdom held that complexes involved in chromatin dynamics are not 'druggable' targets.  However, small mols. that inhibit bromodomain and extraterminal (BET) proteins have been described.  We examine these developments and discuss the implications for small mol. epigenetic targeting of chromatin networks in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1x-bD67oukrVg90H21EOLACvtfcHk0li9t7aZTE8RHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XovFyks7s%253D&md5=d06d88ef0d5d4d71115f40a2f901f53d</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fnrc3256&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3256%26sid%3Dliteratum%253Aachs%26aulast%3DBelkina%26aufirst%3DA.%2BC.%26aulast%3DDenis%26aufirst%3DG.%2BV.%26atitle%3DBET%2520domain%2520co-regulators%2520in%2520obesity%252C%2520inflammation%2520and%2520cancer%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2012%26volume%3D12%26spage%3D465%26epage%3D477%26doi%3D10.1038%2Fnrc3256" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bountra, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fish, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span> <span> </span><span class="NLM_article-title">Epigenetic protein families: a new frontier for drug discovery</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">384</span>– <span class="NLM_lpage">400</span>, <span class="refDoi"> DOI: 10.1038/nrd3674</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1038%2Fnrd3674" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=22498752" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC38XlsFWjsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=384-400&author=C.+H.+Arrowsmithauthor=C.+Bountraauthor=P.+V.+Fishauthor=K.+Leeauthor=M.+Schapira&title=Epigenetic+protein+families%3A+a+new+frontier+for+drug+discovery&doi=10.1038%2Fnrd3674"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetic protein families: a new frontier for drug discovery</span></div><div class="casAuthors">Arrowsmith, Cheryl H.; Bountra, Chas; Fish, Paul V.; Lee, Kevin; Schapira, Matthieu</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">384-400</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Epigenetic regulation of gene expression is a dynamic and reversible process that establishes normal cellular phenotypes but also contributes to human diseases.  At the mol. level, epigenetic regulation involves hierarchical covalent modification of DNA and the proteins that package DNA, such as histones.  Here, we review the key protein families that mediate epigenetic signalling through the acetylation and methylation of histones, including histone deacetylases, protein methyltransferases, lysine demethylases, bromodomain-contg. proteins and proteins that bind to methylated histones.  These protein families are emerging as druggable classes of enzymes and druggable classes of protein-protein interaction domains.  In this article, we discuss the known links with disease, basic mol. mechanisms of action and recent progress in the pharmacol. modulation of each class of proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpcD99eCigNtLVg90H21EOLACvtfcHk0liSC4QRRejK2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XlsFWjsbs%253D&md5=eb82034466a43107aa74a18ccf6d29f5</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fnrd3674&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3674%26sid%3Dliteratum%253Aachs%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DFish%26aufirst%3DP.%2BV.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DSchapira%26aufirst%3DM.%26atitle%3DEpigenetic%2520protein%2520families%253A%2520a%2520new%2520frontier%2520for%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2012%26volume%3D11%26spage%3D384%26epage%3D400%26doi%3D10.1038%2Fnrd3674" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mangos, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keates, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambert, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Volkmer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pawson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gingras, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Histone recognition and large-scale structural analysis of the human bromodomain family</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>149</i></span>,  <span class="NLM_fpage">214</span>– <span class="NLM_lpage">231</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2012.02.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1016%2Fj.cell.2012.02.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=22464331" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=149&publication_year=2012&pages=214-231&author=P.+Filippakopoulosauthor=S.+Picaudauthor=M.+Mangosauthor=T.+Keatesauthor=J.+P.+Lambertauthor=D.+Barsyte-Lovejoyauthor=I.+Felletarauthor=R.+Volkmerauthor=S.+Mullerauthor=T.+Pawsonauthor=A.+C.+Gingrasauthor=C.+H.+Arrowsmithauthor=S.+Knapp&title=Histone+recognition+and+large-scale+structural+analysis+of+the+human+bromodomain+family&doi=10.1016%2Fj.cell.2012.02.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Histone Recognition and Large-Scale Structural Analysis of the Human Bromodomain Family</span></div><div class="casAuthors">Filippakopoulos, Panagis; Picaud, Sarah; Mangos, Maria; Keates, Tracy; Lambert, Jean-Philippe; Barsyte-Lovejoy, Dalia; Felletar, Ildiko; Volkmer, Rudolf; Muller, Susanne; Pawson, Tony; Gingras, Anne-Claude; Arrowsmith, Cheryl H.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">149</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">214-231</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Bromodomains (BRDs) are protein interaction modules that specifically recognize ε-N-lysine acetylation motifs, a key event in the reading process of epigenetic marks.  The 61 BRDs in the human genome cluster into eight families based on structure/sequence similarity.  Here, we present 29 high-resoln. crystal structures, covering all BRD families.  These proteins are: ASH1L, ATAD2, BAZ2B, BPTF, BRD1, BRD3(1), BRD3(2), BRD4(1), BRD4(2), BRD9, BRDT(1), CECR2, EP300, CREBBP, GCN5L2, KIAA1240, PB1(1), PB1(2), PB1(3), PB1(4), PB1(5), PB1(6), PCAF, PHIP(2), TAF1(2), WDR9(2), BRD4(1).  Comprehensive crossfamily structural anal. identifies conserved and family-specific structural features that are necessary for specific acetylation-dependent substrate recognition.  Screening of more than 30 representative BRDs against systematic histone-peptide arrays identifies new BRD substrates and reveals a strong influence of flanking posttranslational modifications, such as acetylation and phosphorylation, suggesting that BRDs recognize combinations of marks rather than singly acetylated sequences.  We further uncovered a structural mechanism for the simultaneous binding and recognition of diverse diacetyl-contg. peptides by BRD4.  These data provide a foundation for structure-based drug design of specific inhibitors for this emerging target family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoK2FSIcPoXZLVg90H21EOLACvtfcHk0liSC4QRRejK2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D&md5=05583a01a1cd4c3b59fc59c00e4c4777</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2012.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2012.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DMangos%26aufirst%3DM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DLambert%26aufirst%3DJ.%2BP.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DVolkmer%26aufirst%3DR.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DPawson%26aufirst%3DT.%26aulast%3DGingras%26aufirst%3DA.%2BC.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DHistone%2520recognition%2520and%2520large-scale%2520structural%2520analysis%2520of%2520the%2520human%2520bromodomain%2520family%26jtitle%3DCell%26date%3D2012%26volume%3D149%26spage%3D214%26epage%3D231%26doi%3D10.1016%2Fj.cell.2012.02.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Targeting bromodomains: epigenetic readers of lysine acetylation</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">337</span>– <span class="NLM_lpage">356</span>, <span class="refDoi"> DOI: 10.1038/nrd4286</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1038%2Fnrd4286" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=24751816" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=337-356&author=P.+Filippakopoulosauthor=S.+Knapp&title=Targeting+bromodomains%3A+epigenetic+readers+of+lysine+acetylation&doi=10.1038%2Fnrd4286"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting bromodomains: epigenetic readers of lysine acetylation</span></div><div class="casAuthors">Filippakopoulos, Panagis; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">337-356</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Lysine acetylation is a key mechanism that regulates chromatin structure; aberrant acetylation levels have been linked to the development of several diseases.  Acetyl-lysine modifications create docking sites for bromodomains, which are small interaction modules found on diverse proteins, some of which have a key role in the acetylation-dependent assembly of transcriptional regulator complexes.  These complexes can then initiate transcriptional programs that result in phenotypic changes.  The recent discovery of potent and highly specific inhibitors for the BET (bromodomain and extra-terminal) family of bromodomains has stimulated intensive research activity in diverse therapeutic areas, particularly in oncol., where BET proteins regulate the expression of key oncogenes and anti-apoptotic proteins.  In addn., targeting BET bromodomains could hold potential for the treatment of inflammation and viral infection.  Here, we highlight recent progress in the development of bromodomain inhibitors, and their potential applications in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOSIcwBJKj4LVg90H21EOLACvtfcHk0liSC4QRRejK2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D&md5=77049c82ce6a19b4685e3b7950399a07</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fnrd4286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4286%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DTargeting%2520bromodomains%253A%2520epigenetic%2520readers%2520of%2520lysine%2520acetylation%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D337%26epage%3D356%26doi%3D10.1038%2Fnrd4286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dhalluin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlson, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aggarwal, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M. M.</span></span> <span> </span><span class="NLM_article-title">Structure and ligand of a histone acetyltransferase bromodomain</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>399</i></span>,  <span class="NLM_fpage">491</span>– <span class="NLM_lpage">496</span>, <span class="refDoi"> DOI: 10.1038/20974</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1038%2F20974" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10365964" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADyaK1MXjvV2isLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=399&publication_year=1999&pages=491-496&author=C.+Dhalluinauthor=J.+E.+Carlsonauthor=L.+Zengauthor=C.+Heauthor=A.+K.+Aggarwalauthor=M.+M.+Zhou&title=Structure+and+ligand+of+a+histone+acetyltransferase+bromodomain&doi=10.1038%2F20974"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and ligand of a histone acetyltransferase bromodomain</span></div><div class="casAuthors">Dhalluin, Christophe; Carlson, Justin E.; Zeng, Lei; He, Cheng; Aggarwal, Aneel K.; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">399</span>
        (<span class="NLM_cas:issue">6735</span>),
    <span class="NLM_cas:pages">491-496</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines</span>)
        </div><div class="casAbstract">Histone acetylation is important in chromatin remodelling and gene activation.  Nearly all known histone-acetyltransferase (HAT)-assocd. transcriptional co-activators contain bromodomains, which are ∼110-amino-acid modules found in many chromatin-assocd. proteins.  Despite the wide occurrence of these bromodomains, their three-dimensional structure and binding partners remain unknown.  Here we report the soln. structure of the bromodomain of the HAT co-activator P/CAF (p300/CBP-assocd. factor).  The structure reveals an unusual left-handed up-and-down four-helix bundle.  In addn., we show by a combination of structural and site-directed mutagenesis studies that bromodomains can interact specifically with acetylated lysine, making them the first known protein modules to do so.  The nature of the recognition of acetyl-lysine by the P/CAF bromodomain is similar to that of acetyl-CoA by histone acetyltransferase.  Thus, the bromodomain is functionally linked to the HAT activity of co-activators in the regulation of gene transcription.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsTbwcjeBG_rVg90H21EOLACvtfcHk0liV6mdCt7valA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXjvV2isLo%253D&md5=ff1063ba81d81b431065dd6e3751ba79</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2F20974&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F20974%26sid%3Dliteratum%253Aachs%26aulast%3DDhalluin%26aufirst%3DC.%26aulast%3DCarlson%26aufirst%3DJ.%2BE.%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DHe%26aufirst%3DC.%26aulast%3DAggarwal%26aufirst%3DA.%2BK.%26aulast%3DZhou%26aufirst%3DM.%2BM.%26atitle%3DStructure%2520and%2520ligand%2520of%2520a%2520histone%2520acetyltransferase%2520bromodomain%26jtitle%3DNature%26date%3D1999%26volume%3D399%26spage%3D491%26epage%3D496%26doi%3D10.1038%2F20974" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M. M.</span></span> <span> </span><span class="NLM_article-title">Bromodomain: an acetyl-lysine binding domain</span>. <i>FEBS Lett.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>513</i></span>,  <span class="NLM_fpage">124</span>– <span class="NLM_lpage">128</span>, <span class="refDoi"> DOI: 10.1016/S0014-5793(01)03309-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1016%2FS0014-5793%2801%2903309-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=11911891" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BD38XitFejtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=513&publication_year=2002&pages=124-128&author=L.+Zengauthor=M.+M.+Zhou&title=Bromodomain%3A+an+acetyl-lysine+binding+domain&doi=10.1016%2FS0014-5793%2801%2903309-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomain: an acetyl-lysine binding domain</span></div><div class="casAuthors">Zeng, Lei; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Letters</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">513</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">124-128</span>CODEN:
                <span class="NLM_cas:coden">FEBLAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-5793</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">A review with 43 refs.  Bromodomains, an extensive family of evolutionarily conserved protein modules originally found in proteins assocd. with chromatin and in nearly all nuclear histone acetyltransferases, have been recently discovered to function as acetyllysine-binding domains.  More recent structural studies of bromodomain/peptide ligand complexes have enriched the understanding of differences in ligand selectivity of bromodomains.  These new findings demonstrate that bromodomain/acetyllysine recognition can serve as a pivotal mechanism for regulating protein-protein interactions in numerous cellular processes including chromatin remodeling and transcriptional activation, and reinforce the concept that functional diversity of a conserved protein modular structure is achieved by evolutionary changes of amino acid sequences in the ligand binding site.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMESL8iZ9agLVg90H21EOLACvtfcHk0liV6mdCt7valA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XitFejtbo%253D&md5=79070a63cd171f846ab577aa9f7092bb</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2FS0014-5793%2801%2903309-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0014-5793%252801%252903309-9%26sid%3Dliteratum%253Aachs%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DZhou%26aufirst%3DM.%2BM.%26atitle%3DBromodomain%253A%2520an%2520acetyl-lysine%2520binding%2520domain%26jtitle%3DFEBS%2520Lett.%26date%3D2002%26volume%3D513%26spage%3D124%26epage%3D128%26doi%3D10.1016%2FS0014-5793%2801%2903309-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Loven, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoke, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lau, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orlando, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vakoc, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, T. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. A.</span></span> <span> </span><span class="NLM_article-title">Selective inhibition of tumor oncogenes by disruption of super-enhancers</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>153</i></span>,  <span class="NLM_fpage">320</span>– <span class="NLM_lpage">334</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2013.03.036</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1016%2Fj.cell.2013.03.036" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=23582323" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlvVCgsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=153&publication_year=2013&pages=320-334&author=J.+Lovenauthor=H.+A.+Hokeauthor=C.+Y.+Linauthor=A.+Lauauthor=D.+A.+Orlandoauthor=C.+R.+Vakocauthor=J.+E.+Bradnerauthor=T.+I.+Leeauthor=R.+A.+Young&title=Selective+inhibition+of+tumor+oncogenes+by+disruption+of+super-enhancers&doi=10.1016%2Fj.cell.2013.03.036"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Inhibition of Tumor Oncogenes by Disruption of Super-Enhancers</span></div><div class="casAuthors">Loven, Jakob; Hoke, Heather A.; Lin, Charles Y.; Lau, Ashley; Orlando, David A.; Vakoc, Christopher R.; Bradner, James E.; Lee, Tong Ihn; Young, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">153</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">320-334</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Chromatin regulators have become attractive targets for cancer therapy, but it is unclear why inhibition of these ubiquitous regulators should have gene-specific effects in tumor cells.  Here, we investigate how inhibition of the widely expressed transcriptional coactivator BRD4 leads to selective inhibition of the MYC oncogene in multiple myeloma (MM).  BRD4 and Mediator were found to co-occupy thousands of enhancers assocd. with active genes.  They also co-occupied a small set of exceptionally large super-enhancers assocd. with genes that feature prominently in MM biol., including the MYC oncogene.  Treatment of MM tumor cells with the BET-bromodomain inhibitor JQ1 led to preferential loss of BRD4 at super-enhancers and consequent transcription elongation defects that preferentially impacted genes with super-enhancers, including MYC.  Super-enhancers were found at key oncogenic drivers in many other tumor cells.  These observations have implications for the discovery of cancer therapeutics directed at components of super-enhancers in diverse tumor types.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrM-NqFYuGMqrVg90H21EOLACvtfcHk0liV6mdCt7valA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlvVCgsrY%253D&md5=66b6ccb5b8a6039832910443fd8b4664</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2013.03.036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2013.03.036%26sid%3Dliteratum%253Aachs%26aulast%3DLoven%26aufirst%3DJ.%26aulast%3DHoke%26aufirst%3DH.%2BA.%26aulast%3DLin%26aufirst%3DC.%2BY.%26aulast%3DLau%26aufirst%3DA.%26aulast%3DOrlando%26aufirst%3DD.%2BA.%26aulast%3DVakoc%26aufirst%3DC.%2BR.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DLee%26aufirst%3DT.%2BI.%26aulast%3DYoung%26aufirst%3DR.%2BA.%26atitle%3DSelective%2520inhibition%2520of%2520tumor%2520oncogenes%2520by%2520disruption%2520of%2520super-enhancers%26jtitle%3DCell%26date%3D2013%26volume%3D153%26spage%3D320%26epage%3D334%26doi%3D10.1016%2Fj.cell.2013.03.036" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hnisz, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. A.</span></span> <span> </span><span class="NLM_article-title">Transcriptional Addiction in Cancer</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>168</i></span>,  <span class="NLM_fpage">629</span>– <span class="NLM_lpage">643</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2016.12.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1016%2Fj.cell.2016.12.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=28187285" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC2sXis1ygtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=168&publication_year=2017&pages=629-643&author=J.+E.+Bradnerauthor=D.+Hniszauthor=R.+A.+Young&title=Transcriptional+Addiction+in+Cancer&doi=10.1016%2Fj.cell.2016.12.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Transcriptional addiction in cancer</span></div><div class="casAuthors">Bradner, James E.; Hnisz, Denes; Young, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">168</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">629-643</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Cancer arises from genetic alterations that invariably lead to dysregulated transcriptional programs.  These dysregulated programs can cause cancer cells to become highly dependent on certain regulators of gene expression.  Here, we discuss how transcriptional control is disrupted by genetic alterations in cancer cells, why transcriptional dependencies can develop as a consequence of dysregulated programs, and how these dependencies provide opportunities for novel therapeutic interventions in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAb-krWUlAJLVg90H21EOLACvtfcHk0liV6mdCt7valA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXis1ygtbk%253D&md5=d6f9840f701f233b14e89306113c828c</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2016.12.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2016.12.013%26sid%3Dliteratum%253Aachs%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DHnisz%26aufirst%3DD.%26aulast%3DYoung%26aufirst%3DR.%2BA.%26atitle%3DTranscriptional%2520Addiction%2520in%2520Cancer%26jtitle%3DCell%26date%3D2017%26volume%3D168%26spage%3D629%26epage%3D643%26doi%3D10.1016%2Fj.cell.2016.12.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prakash, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sum, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwok, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thiessen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pettersson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chin, K. C.</span></span> <span> </span><span class="NLM_article-title">Bromodomain-containing protein 4 (BRD4) regulates RNA polymerase II serine 2 phosphorylation in human CD4+ T cells</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>287</i></span>,  <span class="NLM_fpage">43137</span>– <span class="NLM_lpage">43155</span>, <span class="refDoi"> DOI: 10.1074/jbc.M112.413047</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1074%2Fjbc.M112.413047" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=23086925" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVeksbnL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2012&pages=43137-43155&author=W.+Zhangauthor=C.+Prakashauthor=C.+Sumauthor=Y.+Gongauthor=Y.+Liauthor=J.+J.+Kwokauthor=N.+Thiessenauthor=S.+Petterssonauthor=S.+J.+Jonesauthor=S.+Knappauthor=H.+Yangauthor=K.+C.+Chin&title=Bromodomain-containing+protein+4+%28BRD4%29+regulates+RNA+polymerase+II+serine+2+phosphorylation+in+human+CD4%2B+T+cells&doi=10.1074%2Fjbc.M112.413047"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomain-containing Protein 4 (BRD4) Regulates RNA Polymerase II Serine 2 Phosphorylation in Human CD4+ T Cells</span></div><div class="casAuthors">Zhang, Weishi; Prakash, Celine; Sum, Calvin; Gong, Yue; Li, Yinghui; Kwok, Jeffrey J. T.; Thiessen, Nina; Pettersson, Sven; Jones, Steven J. M.; Knapp, Stefan; Yang, Henry; Chin, Keh-Chuang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">287</span>
        (<span class="NLM_cas:issue">51</span>),
    <span class="NLM_cas:pages">43137-43155</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Transcriptional elongation by RNA polymerase II (Pol II) is regulated by pos. transcription elongation factor b (P-TEFb) in assocn. with bromodomain-contg. protein 4 (BRD4).  The authors used genome-wide chromatin immunopptn. sequencing in primary human CD4+ T cells to reveal that BRD4 co-localizes with Ser-2-phosphorylated Pol II (Pol II Ser-2) at both enhancers and promoters of active genes.  Disruption of bromodomain-histone acetylation interactions by JQ1, a small-mol. bromodomain inhibitor, resulted in decreased BRD4 binding, reduced Pol II Ser-2, and reduced expression of lineage-specific genes in primary human CD4+ T cells.  A large no. of JQ1-disrupted BRD4 binding regions exhibited diacetylated H4 (lysine 5 and -8) and H3K27 acetylation (H3K27ac), which correlated with the presence of histone acetyltransferases and deacetylases.  Genes assocd. with BRD4/H3K27ac co-occupancy exhibited significantly higher activity than those assocd. with H3K27ac or BRD4 binding alone.  Comparison of BRD4 binding in T cells and in human embryonic stem cells revealed that enhancer BRD4 binding sites were predominantly lineage-specific.  These findings suggest that BRD4-driven Pol II phosphorylation at serine 2 plays an important role in regulating lineage-specific gene transcription in human CD4+ T cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMZ7-itp_IObVg90H21EOLACvtfcHk0ljOaX1h_CBcdg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVeksbnL&md5=79541de2c23a5c164fc8c4295ffebf53</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M112.413047&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M112.413047%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DPrakash%26aufirst%3DC.%26aulast%3DSum%26aufirst%3DC.%26aulast%3DGong%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DKwok%26aufirst%3DJ.%2BJ.%26aulast%3DThiessen%26aufirst%3DN.%26aulast%3DPettersson%26aufirst%3DS.%26aulast%3DJones%26aufirst%3DS.%2BJ.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DChin%26aufirst%3DK.%2BC.%26atitle%3DBromodomain-containing%2520protein%25204%2520%2528BRD4%2529%2520regulates%2520RNA%2520polymerase%2520II%2520serine%25202%2520phosphorylation%2520in%2520human%2520CD4%252B%2520T%2520cells%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2012%26volume%3D287%26spage%3D43137%26epage%3D43155%26doi%3D10.1074%2Fjbc.M112.413047" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">French, C. A.</span></span> <span> </span><span class="NLM_article-title">NUT midline carcinoma</span>. <i>Cancer Genet. Cytogenet.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>203</i></span>,  <span class="NLM_fpage">16</span>– <span class="NLM_lpage">20</span>, <span class="refDoi"> DOI: 10.1016/j.cancergencyto.2010.06.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1016%2Fj.cancergencyto.2010.06.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=20951314" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A280%3ADC%252BC3cbgtVeluw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=203&publication_year=2010&pages=16-20&author=C.+A.+French&title=NUT+midline+carcinoma&doi=10.1016%2Fj.cancergencyto.2010.06.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">NUT midline carcinoma</span></div><div class="casAuthors">French Christopher Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Cancer genetics and cytogenetics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">203</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">16-20</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">NUT midline carcinoma (NMC) is a rare, aggressive human cancer, genetically defined by rearrangements of the gene NUT (HUGO symbol: C15orf55).  In the majority (∼75%) of NMCs, most of the coding sequence of NUT on chromosome 15q14 is fused with BRD4 creating chimeric genes that encode BRD-NUT fusion proteins.  In the remaining cases, NUT is fused to BRD3 or an unknown partner gene; these tumors are termed NUT-variant.  Diagnosis of NMC is made by demonstration of expression of the NUT-fusion protein using a monoclonal antibody to NUT for immunohistochemistry, and confirmation of the fusion (BRD-NUT or NUT-variant) by fluorescent in situ hybridization or reverse transcriptase-polymerase chain reaction.  BRD-NUT functions to block cellular differentiation and promote uncontrolled growth of carcinoma cells.  Because the reagents and expertise required to diagnose NMC are not available in most laboratories, and because of incomplete awareness of this disease, NMC is frequently undiagnosed or misdiagnosed, and its actual prevalence is unknown.  NUT midline carcinoma does not arise from any specific tissue type or organ.  It presents as a poorly differentiated carcinoma originating from midline locations such as the head, neck or mediastinum.  Although rare, NMCs occur throughout life, and advanced local disease is frequently accompanied by distant hematogenous metastases.  There still is no effective treatment for NMC, there are no guidelines, and current approaches to treatment are based on discussions among a few oncologists who each have had a single experience treating this disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTU75vAesIXfgMthz5_Rx5rfW6udTcc2eYD1s8Ksyv5-7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3cbgtVeluw%253D%253D&md5=65592b3ac99151b498d9b10f9aab6346</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.cancergencyto.2010.06.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cancergencyto.2010.06.007%26sid%3Dliteratum%253Aachs%26aulast%3DFrench%26aufirst%3DC.%2BA.%26atitle%3DNUT%2520midline%2520carcinoma%26jtitle%3DCancer%2520Genet.%2520Cytogenet.%26date%3D2010%26volume%3D203%26spage%3D16%26epage%3D20%26doi%3D10.1016%2Fj.cancergencyto.2010.06.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nagarajan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hossan, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alawi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Najafova, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Indenbirken, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedi, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taipaleenmaki, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ben-Batalla, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scheller, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loges, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hesse, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grundhoff, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnsen, S. A.</span></span> <span> </span><span class="NLM_article-title">Bromodomain protein BRD4 is required for estrogen receptor-dependent enhancer activation and gene transcription</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">460</span>– <span class="NLM_lpage">469</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2014.06.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1016%2Fj.celrep.2014.06.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=25017071" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFChs7%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2014&pages=460-469&author=S.+Nagarajanauthor=T.+Hossanauthor=M.+Alawiauthor=Z.+Najafovaauthor=D.+Indenbirkenauthor=U.+Bediauthor=H.+Taipaleenmakiauthor=I.+Ben-Batallaauthor=M.+Schellerauthor=S.+Logesauthor=S.+Knappauthor=E.+Hesseauthor=C.+M.+Chiangauthor=A.+Grundhoffauthor=S.+A.+Johnsen&title=Bromodomain+protein+BRD4+is+required+for+estrogen+receptor-dependent+enhancer+activation+and+gene+transcription&doi=10.1016%2Fj.celrep.2014.06.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomain Protein BRD4 Is Required for Estrogen Receptor-Dependent Enhancer Activation and Gene Transcription</span></div><div class="casAuthors">Nagarajan, Sankari; Hossan, Tareq; Alawi, Malik; Najafova, Zeynab; Indenbirken, Daniela; Bedi, Upasana; Taipaleenmaeki, Hanna; Ben-Batalla, Isabel; Scheller, Marina; Loges, Sonja; Knapp, Stefan; Hesse, Eric; Chiang, Cheng-Ming; Grundhoff, Adam; Johnsen, Steven A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">460-469</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The estrogen receptor α (ERα) controls cell proliferation and tumorigenesis by recruiting various cofactors to estrogen response elements (EREs) to control gene transcription.  A deeper understanding of these transcriptional mechanisms may uncover therapeutic targets for ERα-dependent cancers.  We show that BRD4 regulates ERα-induced gene expression by affecting elongation-assocd. phosphorylation of RNA polymerase II (RNAPII) and histone H2B monoubiquitination.  Consistently, BRD4 activity is required for proliferation of ER+ breast and endometrial cancer cells and uterine growth in mice.  Genome-wide studies revealed an enrichment of BRD4 on transcriptional start sites of active genes and a requirement of BRD4 for H2B monoubiquitination in the transcribed region of estrogen-responsive genes.  Importantly, we demonstrate that BRD4 occupancy on distal EREs enriched for H3K27ac is required for recruitment and elongation of RNAPII on EREs and the prodn. of ERα-dependent enhancer RNAs.  These results uncover BRD4 as a central regulator of ERα function and potential therapeutic target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0fNk5Ueff1bVg90H21EOLACvtfcHk0ljOaX1h_CBcdg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFChs7%252FM&md5=301a96567713aa777911a158736adb6d</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2014.06.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2014.06.016%26sid%3Dliteratum%253Aachs%26aulast%3DNagarajan%26aufirst%3DS.%26aulast%3DHossan%26aufirst%3DT.%26aulast%3DAlawi%26aufirst%3DM.%26aulast%3DNajafova%26aufirst%3DZ.%26aulast%3DIndenbirken%26aufirst%3DD.%26aulast%3DBedi%26aufirst%3DU.%26aulast%3DTaipaleenmaki%26aufirst%3DH.%26aulast%3DBen-Batalla%26aufirst%3DI.%26aulast%3DScheller%26aufirst%3DM.%26aulast%3DLoges%26aufirst%3DS.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DHesse%26aufirst%3DE.%26aulast%3DChiang%26aufirst%3DC.%2BM.%26aulast%3DGrundhoff%26aufirst%3DA.%26aulast%3DJohnsen%26aufirst%3DS.%2BA.%26atitle%3DBromodomain%2520protein%2520BRD4%2520is%2520required%2520for%2520estrogen%2520receptor-dependent%2520enhancer%2520activation%2520and%2520gene%2520transcription%26jtitle%3DCell%2520Rep.%26date%3D2014%26volume%3D8%26spage%3D460%26epage%3D469%26doi%3D10.1016%2Fj.celrep.2014.06.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, W. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morse, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keates, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickman, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munro, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKeown, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christie, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cameron, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heightman, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">La Thangue, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">French, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiest, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kung, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Selective inhibition of BET bromodomains</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>468</i></span>,  <span class="NLM_fpage">1067</span>– <span class="NLM_lpage">1073</span>, <span class="refDoi"> DOI: 10.1038/nature09504</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1038%2Fnature09504" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=20871596" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=1067-1073&author=P.+Filippakopoulosauthor=J.+Qiauthor=S.+Picaudauthor=Y.+Shenauthor=W.+B.+Smithauthor=O.+Fedorovauthor=E.+M.+Morseauthor=T.+Keatesauthor=T.+T.+Hickmanauthor=I.+Felletarauthor=M.+Philpottauthor=S.+Munroauthor=M.+R.+McKeownauthor=Y.+Wangauthor=A.+L.+Christieauthor=N.+Westauthor=M.+J.+Cameronauthor=B.+Schwartzauthor=T.+D.+Heightmanauthor=N.+La+Thangueauthor=C.+A.+Frenchauthor=O.+Wiestauthor=A.+L.+Kungauthor=S.+Knappauthor=J.+E.+Bradner&title=Selective+inhibition+of+BET+bromodomains&doi=10.1038%2Fnature09504"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibition of BET bromodomains</span></div><div class="casAuthors">Filippakopoulos, Panagis; Qi, Jun; Picaud, Sarah; Shen, Yao; Smith, William B.; Fedorov, Oleg; Morse, Elizabeth M.; Keates, Tracey; Hickman, Tyler T.; Felletar, Ildiko; Philpott, Martin; Munro, Shongah; McKeown, Michael R.; Wang, Yuchuan; Christie, Amanda L.; West, Nathan; Cameron, Michael J.; Schwartz, Brian; Heightman, Tom D.; La Thangue, Nicholas; French, Christopher; Wiest, Olaf; Kung, Andrew L.; Knapp, Stefan; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7327</span>),
    <span class="NLM_cas:pages">1067-1073</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Epigenetic proteins are intently pursued targets in ligand discovery.  So far, successful efforts have been limited to chromatin modifying enzymes, or so-called epigenetic 'writers' and 'erasers'.  Potent inhibitors of histone binding modules have not yet been described.  Here the authors report a cell-permeable small mol. (I,JQ1) that binds competitively to acetyl-lysine recognition motifs, or bromodomains.  High potency and specificity towards a subset of human bromodomains is explained by co-crystal structures with bromodomain and extra-terminal (BET) family member BRD4, revealing excellent shape complementarity with the acetyl-lysine binding cavity.  Recurrent translocation of BRD4 is obsd. in a genetically-defined, incurable subtype of human squamous carcinoma.  Competitive binding by JQ1 displaces the BRD4 fusion oncoprotein from chromatin, prompting squamous differentiation and specific antiproliferative effects in BRD4-dependent cell lines and patient-derived xenograft models.  These data establish proof-of-concept for targeting protein-protein interactions of epigenetic 'readers', and provide a versatile chem. scaffold for the development of chem. probes more broadly throughout the bromodomain family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2ZbtmZM8m_LVg90H21EOLACvtfcHk0lgcjKJ00v37ng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXotlGmtA%253D%253D&md5=bd9cadac86124d2c6c3cd5b2a92d68c2</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fnature09504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09504%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DSmith%26aufirst%3DW.%2BB.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMorse%26aufirst%3DE.%2BM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DHickman%26aufirst%3DT.%2BT.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DMunro%26aufirst%3DS.%26aulast%3DMcKeown%26aufirst%3DM.%2BR.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DChristie%26aufirst%3DA.%2BL.%26aulast%3DWest%26aufirst%3DN.%26aulast%3DCameron%26aufirst%3DM.%2BJ.%26aulast%3DSchwartz%26aufirst%3DB.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DLa%2BThangue%26aufirst%3DN.%26aulast%3DFrench%26aufirst%3DC.%2BA.%26aulast%3DWiest%26aufirst%3DO.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DSelective%2520inhibition%2520of%2520BET%2520bromodomains%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D1067%26epage%3D1073%26doi%3D10.1038%2Fnature09504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sengupta, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biarnes, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jordan, V. C.</span></span> <span> </span><span class="NLM_article-title">Inhibition of BET proteins impairs estrogen-mediated growth and transcription in breast cancers by pausing RNA polymerase advancement</span>. <i>Breast Cancer Res. Treat.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>150</i></span>,  <span class="NLM_fpage">265</span>– <span class="NLM_lpage">278</span>, <span class="refDoi"> DOI: 10.1007/s10549-015-3319-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1007%2Fs10549-015-3319-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=25721606" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjs1GrsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=150&publication_year=2015&pages=265-278&author=S.+Senguptaauthor=M.+C.+Biarnesauthor=R.+Clarkeauthor=V.+C.+Jordan&title=Inhibition+of+BET+proteins+impairs+estrogen-mediated+growth+and+transcription+in+breast+cancers+by+pausing+RNA+polymerase+advancement&doi=10.1007%2Fs10549-015-3319-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of BET proteins impairs estrogen-mediated growth and transcription in breast cancers by pausing RNA polymerase advancement</span></div><div class="casAuthors">Sengupta, Surojeet; Biarnes, Michael C.; Clarke, Robert; Jordan, V. Craig</div><div class="citationInfo"><span class="NLM_cas:title">Breast Cancer Research and Treatment</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">150</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">265-278</span>CODEN:
                <span class="NLM_cas:coden">BCTRD6</span>;
        ISSN:<span class="NLM_cas:issn">0167-6806</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Estrogen (E2)-induced transcription requires coordinated recruitment of estrogen receptor α (ER) and multiple factors at the promoter of activated genes.  However, the precise mechanism by which this complex stimulates the RNA polymerase II activity required to execute transcription is largely unresolved.  We investigated the role of bromodomain (BRD) contg. bromodomain and extra-terminal (BET) proteins, in E2-induced growth and gene activation.  JQ1, a specific BET protein inhibitor, was used to block BET protein function in two different ER-pos. breast cancer cell lines (MCF7 and T47D).  Real-time PCR and ChIP assays were used to measure RNA expression and to detect recruitment of various factors on the genes, resp.  Protein levels were measured by Western blotting.  JQ1 suppressed E2-induced growth and transcription in both MCF7 and T47D cells.  The combination of E2 and JQ1 down-regulated the levels of ER protein in MCF7 cells but the loss of ER was not responsible for JQ1-mediated inhibition of E2 signaling.  JQ1 did not disrupt E2-induced recruitment of ER and co-activator (SRC3) at the E2-responsive DNA elements.  The E2-induced increase in histone acetylation was also not altered by JQ1.  However, JQ1 blocked the E2-induced transition of RNA polymerase II from initiation to elongation by stalling it at the promoter region of the responsive genes upstream of the transcription start site.  This study establishes BET proteins as the key mediators of E2-induced transcriptional activation.  This adds another layer of complexity to the regulation of estrogen-induced gene activation that can potentially be targeted for therapeutic intervention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo54AxgJqtE0LVg90H21EOLACvtfcHk0lgcjKJ00v37ng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjs1GrsLw%253D&md5=12afeb1211a6d4112438b4c3afb2c392</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1007%2Fs10549-015-3319-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10549-015-3319-1%26sid%3Dliteratum%253Aachs%26aulast%3DSengupta%26aufirst%3DS.%26aulast%3DBiarnes%26aufirst%3DM.%2BC.%26aulast%3DClarke%26aufirst%3DR.%26aulast%3DJordan%26aufirst%3DV.%2BC.%26atitle%3DInhibition%2520of%2520BET%2520proteins%2520impairs%2520estrogen-mediated%2520growth%2520and%2520transcription%2520in%2520breast%2520cancers%2520by%2520pausing%2520RNA%2520polymerase%2520advancement%26jtitle%3DBreast%2520Cancer%2520Res.%2520Treat.%26date%3D2015%26volume%3D150%26spage%3D265%26epage%3D278%26doi%3D10.1007%2Fs10549-015-3319-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shea, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Creighton, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coarfa, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilsenbeck, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lanz, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nardone, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitsiades, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitsiades, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osborne, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiff, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Malley, B. W.</span></span> <span> </span><span class="NLM_article-title">An epigenomic approach to therapy for tamoxifen-resistant breast cancer</span>. <i>Cell Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">809</span>– <span class="NLM_lpage">819</span>, <span class="refDoi"> DOI: 10.1038/cr.2014.71</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1038%2Fcr.2014.71" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=24874954" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC2cXptFyltLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2014&pages=809-819&author=Q.+Fengauthor=Z.+Zhangauthor=M.+J.+Sheaauthor=C.+J.+Creightonauthor=C.+Coarfaauthor=S.+G.+Hilsenbeckauthor=R.+Lanzauthor=B.+Heauthor=L.+Wangauthor=X.+Fuauthor=A.+Nardoneauthor=Y.+Songauthor=J.+Bradnerauthor=N.+Mitsiadesauthor=C.+S.+Mitsiadesauthor=C.+K.+Osborneauthor=R.+Schiffauthor=B.+W.+O%E2%80%99Malley&title=An+epigenomic+approach+to+therapy+for+tamoxifen-resistant+breast+cancer&doi=10.1038%2Fcr.2014.71"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">An epigenomic approach to therapy for tamoxifen-resistant breast cancer</span></div><div class="casAuthors">Feng, Qin; Zhang, Zheng; Shea, Martin J.; Creighton, Chad J.; Coarfa, Cristian; Hilsenbeck, Susan G.; Lanz, Rainer; He, Bin; Wang, Lei; Fu, Xiaoyong; Nardone, Agostina; Song, Yongcheng; Bradner, James; Mitsiades, Nicholas; Mitsiades, Constantine S.; Osborne, C. Kent; Schiff, Rachel; O'Malley, Bert W.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">809-819</span>CODEN:
                <span class="NLM_cas:coden">CREEB6</span>;
        ISSN:<span class="NLM_cas:issn">1001-0602</span>.
    
            (<span class="NLM_cas:orgname">NPG Nature Asia-Pacific</span>)
        </div><div class="casAbstract">Tamoxifen has been a frontline treatment for estrogen receptor alpha (ERα)-pos. breast tumors in premenopausal women.  However, resistance to tamoxifen occurs in many patients.  ER still plays a crit. role in the growth of breast cancer cells with acquired tamoxifen resistance, suggesting that ERα remains a valid target for treatment of tamoxifen-resistant (Tam-R) breast cancer.  In an effort to identify novel regulators of ERα signaling, through a small-scale siRNA screen against histone Me modifiers, we found WHSC1, a histone H3K36 methyltransferase, as a pos. regulator of ERα signaling in breast cancer cells.  We demonstrated that WHSC1 is recruited to the ERα gene by the BET protein BRD3/4, and facilitates ERα gene expression.  The small-mol. BET protein inhibitor JQ1 potently suppressed the classic ERα signaling pathway and the growth of Tam-R breast cancer cells in culture.  Using a Tam-R breast cancer xenograft mouse model, we demonstrated in vivo anti-breast cancer activity by JQ1 and a strong long-lasting effect of combination therapy with JQ1 and the ER degrader fulvestrant.  Taken together, we provide evidence that the epigenomic proteins BRD3/4 and WHSC1 are essential regulators of estrogen receptor signaling and are novel therapeutic targets for treatment of Tam-R breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqv-Z8nViYZsbVg90H21EOLACvtfcHk0lgcjKJ00v37ng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXptFyltLo%253D&md5=aaf9d8656e0ebc529d63ae2b217539b3</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1038%2Fcr.2014.71&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fcr.2014.71%26sid%3Dliteratum%253Aachs%26aulast%3DFeng%26aufirst%3DQ.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DShea%26aufirst%3DM.%2BJ.%26aulast%3DCreighton%26aufirst%3DC.%2BJ.%26aulast%3DCoarfa%26aufirst%3DC.%26aulast%3DHilsenbeck%26aufirst%3DS.%2BG.%26aulast%3DLanz%26aufirst%3DR.%26aulast%3DHe%26aufirst%3DB.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DFu%26aufirst%3DX.%26aulast%3DNardone%26aufirst%3DA.%26aulast%3DSong%26aufirst%3DY.%26aulast%3DBradner%26aufirst%3DJ.%26aulast%3DMitsiades%26aufirst%3DN.%26aulast%3DMitsiades%26aufirst%3DC.%2BS.%26aulast%3DOsborne%26aufirst%3DC.%2BK.%26aulast%3DSchiff%26aufirst%3DR.%26aulast%3DO%25E2%2580%2599Malley%26aufirst%3DB.%2BW.%26atitle%3DAn%2520epigenomic%2520approach%2520to%2520therapy%2520for%2520tamoxifen-resistant%2520breast%2520cancer%26jtitle%3DCell%2520Res.%26date%3D2014%26volume%3D24%26spage%3D809%26epage%3D819%26doi%3D10.1038%2Fcr.2014.71" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wold, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brasier, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span> <span> </span><span class="NLM_article-title">Drug Discovery Targeting Bromodomain-Containing Protein 4</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">4533</span>– <span class="NLM_lpage">4558</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01761</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01761" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC2sXis1yiurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=4533-4558&author=Z.+Liuauthor=P.+Wangauthor=H.+Chenauthor=E.+A.+Woldauthor=B.+Tianauthor=A.+R.+Brasierauthor=J.+Zhou&title=Drug+Discovery+Targeting+Bromodomain-Containing+Protein+4&doi=10.1021%2Facs.jmedchem.6b01761"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Drug Discovery Targeting Bromodomain-Containing Protein 4</span></div><div class="casAuthors">Liu, Zhiqing; Wang, Pingyuan; Chen, Haiying; Wold, Eric A.; Tian, Bing; Brasier, Allan R.; Zhou, Jia</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4533-4558</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  BRD4, the most extensively studied member of BET family, is an epigenetic regulator that localizes to DNA via binding acetylated histones and controls the expression of therapeutically important gene regulatory networks through recruiting transcription factors to form mediator complexes, phosphorylating RNA polymerase II and by its intrinsic histone acetyltransferase activity.  Disrupting the protein-protein interactions between BRD4 and acetyl-lysine has been shown to effectively block cell proliferation in cancer, cytokine prodn. in acute inflammation, etc.  To date, significant efforts have been devoted to the development of BRD4 inhibitors, and consequently, a dozen have progressed into human clin. trials.  Herein, the authors summarize the advances in drug discovery and development of BRD4 inhibitors by focusing on their chemotypes, in vitro and in vivo activity, selectivity, relevant mechanisms of action and therapeutic potential.  Opportunities and challenges to achieve selective and efficacious BRD4 inhibitors as a viable therapeutic strategy for human diseases are also highlighted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6IAERH4iggrVg90H21EOLACvtfcHk0ljLZvys5JHC5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXis1yiurg%253D&md5=6d7ba5844073fac8c916b1249af402e5</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01761&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01761%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DP.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DWold%26aufirst%3DE.%2BA.%26aulast%3DTian%26aufirst%3DB.%26aulast%3DBrasier%26aufirst%3DA.%2BR.%26aulast%3DZhou%26aufirst%3DJ.%26atitle%3DDrug%2520Discovery%2520Targeting%2520Bromodomain-Containing%2520Protein%25204%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D4533%26epage%3D4558%26doi%3D10.1021%2Facs.jmedchem.6b01761" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brotherton, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savitsky, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diez-Dacal, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Philpott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bountra, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lingard, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span> <span> </span><span class="NLM_article-title">RVX-208, an inhibitor of BET transcriptional regulators with selectivity for the second bromodomain</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">19754</span>– <span class="NLM_lpage">19759</span>, <span class="refDoi"> DOI: 10.1073/pnas.1310658110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1073%2Fpnas.1310658110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=24248379" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFKms7zN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=19754-19759&author=S.+Picaudauthor=C.+Wellsauthor=I.+Felletarauthor=D.+Brothertonauthor=S.+Martinauthor=P.+Savitskyauthor=B.+Diez-Dacalauthor=M.+Philpottauthor=C.+Bountraauthor=H.+Lingardauthor=O.+Fedorovauthor=S.+M%C3%BCllerauthor=P.+E.+Brennanauthor=S.+Knappauthor=P.+Filippakopoulos&title=RVX-208%2C+an+inhibitor+of+BET+transcriptional+regulators+with+selectivity+for+the+second+bromodomain&doi=10.1073%2Fpnas.1310658110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">RVX-208, an inhibitor of BET transcriptional regulators with selectivity for the second bromodomain</span></div><div class="casAuthors">Picaud, Sarah; Wells, Christopher; Felletar, Ildiko; Brotherton, Deborah; Martin, Sarah; Savitsky, Pavel; Diez-Dacal, Beatriz; Philpott, Martin; Bountra, Chas; Lingard, Hannah; Fedorov, Oleg; Muller, Susanne; Brennan, Paul E.; Knapp, Stefan; Filippakopoulos, Panagis</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">19754-19759,S19754/1-S19754/10</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Bromodomains have emerged as attractive candidates for the development of inhibitors targeting gene transcription.  Inhibitors of the bromo and extraterminal (BET) family recently showed promising activity in diverse disease models.  However, the pleiotropic nature of BET proteins regulating tissue-specific transcription has raised safety concerns and suggested that attempts should be made for domain-specific targeting.  Here, we report that RVX-208, a compd. currently in phase II clin. trials, is a BET bromodomain inhibitor specific for second bromodomains (BD2s).  Cocrystal structures revealed binding modes of RVX-208 and its synthetic precursor, and fluorescent recovery after photobleaching demonstrated that RVX-208 displaces BET proteins from chromatin.  However, gene-expression data showed that BD2 inhibition only modestly affects BET-dependent gene transcription.  Our data demonstrate the feasibility of specific targeting within the BET family resulting in different transcriptional outcomes and highlight the importance of BD1 in transcriptional regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqamW8vnIopKrVg90H21EOLACvtfcHk0ljLZvys5JHC5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFKms7zN&md5=c918702c9fdb91241bc837732730f7f6</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1310658110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1310658110%26sid%3Dliteratum%253Aachs%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DBrotherton%26aufirst%3DD.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DSavitsky%26aufirst%3DP.%26aulast%3DDiez-Dacal%26aufirst%3DB.%26aulast%3DPhilpott%26aufirst%3DM.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DLingard%26aufirst%3DH.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26atitle%3DRVX-208%252C%2520an%2520inhibitor%2520of%2520BET%2520transcriptional%2520regulators%2520with%2520selectivity%2520for%2520the%2520second%2520bromodomain%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2013%26volume%3D110%26spage%3D19754%26epage%3D19759%26doi%3D10.1073%2Fpnas.1310658110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mirguet, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gosmini, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toum, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clement, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnathan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brusq, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mordaunt, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimes, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crowe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pineau, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ajakane, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daugan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeffrey, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cutler, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haynes, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smithers, N. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uings, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witherington, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parr, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicodeme, E.</span></span> <span> </span><span class="NLM_article-title">Discovery of epigenetic regulator I-BET762: lead optimization to afford a clinical candidate inhibitor of the BET bromodomains</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">7501</span>– <span class="NLM_lpage">7515</span>, <span class="refDoi"> DOI: 10.1021/jm401088k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401088k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVSls7jJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=7501-7515&author=O.+Mirguetauthor=R.+Gosminiauthor=J.+Toumauthor=C.+A.+Clementauthor=M.+Barnathanauthor=J.+M.+Brusqauthor=J.+E.+Mordauntauthor=R.+M.+Grimesauthor=M.+Croweauthor=O.+Pineauauthor=M.+Ajakaneauthor=A.+Dauganauthor=P.+Jeffreyauthor=L.+Cutlerauthor=A.+C.+Haynesauthor=N.+N.+Smithersauthor=C.+W.+Chungauthor=P.+Bamboroughauthor=I.+J.+Uingsauthor=A.+Lewisauthor=J.+Witheringtonauthor=N.+Parrauthor=R.+K.+Prinjhaauthor=E.+Nicodeme&title=Discovery+of+epigenetic+regulator+I-BET762%3A+lead+optimization+to+afford+a+clinical+candidate+inhibitor+of+the+BET+bromodomains&doi=10.1021%2Fjm401088k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Epigenetic Regulator I-BET762: Lead Optimization to Afford a Clinical Candidate Inhibitor of the BET Bromodomains</span></div><div class="casAuthors">Mirguet, Olivier; Gosmini, Romain; Toum, Jerome; Clement, Catherine A.; Barnathan, Melanie; Brusq, Jean-Marie; Mordaunt, Jacqueline E.; Grimes, Richard M.; Crowe, Miriam; Pineau, Olivier; Ajakane, Myriam; Daugan, Alain; Jeffrey, Phillip; Cutler, Leanne; Haynes, Andrea C.; Smithers, Nicholas N.; Chung, Chun-wa; Bamborough, Paul; Uings, Iain J.; Lewis, Antonia; Witherington, Jason; Parr, Nigel; Prinjha, Rab K.; Nicodeme, Edwige</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">7501-7515</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bromo and extra C-terminal domain (BET) family of bromodomains are involved in binding epigenetic marks on histone proteins, more specifically acetylated lysine residues.  This paper describes the discovery and structure-activity relationships (SAR) of potent benzodiazepine inhibitors that disrupt the function of the BET family of bromodomains (BRD2, BRD3, and BRD4).  This work has yielded a potent, selective compd. I-BET762 (I) that is now under evaluation in a phase I/II clin. trial for nuclear protein in testis (NUT) midline carcinoma and other cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDitaXJSnFE7Vg90H21EOLACvtfcHk0ljLZvys5JHC5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVSls7jJ&md5=5d059b0be08a74a5721b72a120f7e090</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fjm401088k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401088k%26sid%3Dliteratum%253Aachs%26aulast%3DMirguet%26aufirst%3DO.%26aulast%3DGosmini%26aufirst%3DR.%26aulast%3DToum%26aufirst%3DJ.%26aulast%3DClement%26aufirst%3DC.%2BA.%26aulast%3DBarnathan%26aufirst%3DM.%26aulast%3DBrusq%26aufirst%3DJ.%2BM.%26aulast%3DMordaunt%26aufirst%3DJ.%2BE.%26aulast%3DGrimes%26aufirst%3DR.%2BM.%26aulast%3DCrowe%26aufirst%3DM.%26aulast%3DPineau%26aufirst%3DO.%26aulast%3DAjakane%26aufirst%3DM.%26aulast%3DDaugan%26aufirst%3DA.%26aulast%3DJeffrey%26aufirst%3DP.%26aulast%3DCutler%26aufirst%3DL.%26aulast%3DHaynes%26aufirst%3DA.%2BC.%26aulast%3DSmithers%26aufirst%3DN.%2BN.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DUings%26aufirst%3DI.%2BJ.%26aulast%3DLewis%26aufirst%3DA.%26aulast%3DWitherington%26aufirst%3DJ.%26aulast%3DParr%26aufirst%3DN.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DNicodeme%26aufirst%3DE.%26atitle%3DDiscovery%2520of%2520epigenetic%2520regulator%2520I-BET762%253A%2520lead%2520optimization%2520to%2520afford%2520a%2520clinical%2520candidate%2520inhibitor%2520of%2520the%2520BET%2520bromodomains%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D7501%26epage%3D7515%26doi%3D10.1021%2Fjm401088k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McDaniel, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soltwedel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fidanze, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasvold, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mantei, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pratt, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bui, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faivre, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warder, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilcox, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albert, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magoc, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajaraman, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hutchins, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edalji, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elmore, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kati, W. M.</span></span> <span> </span><span class="NLM_article-title">Discovery of N-(4-(2,4-Difluorophenoxy)-3-(6-methyl-7-oxo-6,7-dihydro-1H-pyrrolo[2,3-c]pyridin-4-yl)phenyl)ethanesulfonamide (ABBV-075/Mivebresib), a Potent and Orally Available Bromodomain and Extraterminal Domain (BET) Family Bromodomain Inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">8369</span>– <span class="NLM_lpage">8384</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00746</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00746" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFOjtrbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=8369-8384&author=K.+F.+McDanielauthor=L.+Wangauthor=T.+Soltwedelauthor=S.+D.+Fidanzeauthor=L.+A.+Hasvoldauthor=D.+Liuauthor=R.+A.+Manteiauthor=J.+K.+Prattauthor=G.+S.+Sheppardauthor=M.+H.+Buiauthor=E.+J.+Faivreauthor=X.+Huangauthor=L.+Liauthor=X.+Linauthor=R.+Wangauthor=S.+E.+Warderauthor=D.+Wilcoxauthor=D.+H.+Albertauthor=T.+J.+Magocauthor=G.+Rajaramanauthor=C.+H.+Parkauthor=C.+W.+Hutchinsauthor=J.+J.+Shenauthor=R.+P.+Edaljiauthor=C.+C.+Sunauthor=R.+Martinauthor=W.+Gaoauthor=S.+Wongauthor=G.+Fangauthor=S.+W.+Elmoreauthor=Y.+Shenauthor=W.+M.+Kati&title=Discovery+of+N-%284-%282%2C4-Difluorophenoxy%29-3-%286-methyl-7-oxo-6%2C7-dihydro-1H-pyrrolo%5B2%2C3-c%5Dpyridin-4-yl%29phenyl%29ethanesulfonamide+%28ABBV-075%2FMivebresib%29%2C+a+Potent+and+Orally+Available+Bromodomain+and+Extraterminal+Domain+%28BET%29+Family+Bromodomain+Inhibitor&doi=10.1021%2Facs.jmedchem.7b00746"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of N-(4-(2,4-Difluorophenoxy)-3-(6-methyl-7-oxo-6,7-dihydro-1H-pyrrolo[2,3-c]pyridin-4-yl)phenyl)ethanesulfonamide (ABBV-075/Mivebresib), a Potent and Orally Available Bromodomain and Extraterminal Domain (BET) Family Bromodomain Inhibitor</span></div><div class="casAuthors">McDaniel, Keith F.; Wang, Le; Soltwedel, Todd; Fidanze, Steven D.; Hasvold, Lisa A.; Liu, Dachun; Mantei, Robert A.; Pratt, John K.; Sheppard, George S.; Bui, Mai H.; Faivre, Emily J.; Huang, Xiaoli; Li, Leiming; Lin, Xiaoyu; Wang, Rongqi; Warder, Scott E.; Wilcox, Denise; Albert, Daniel H.; Magoc, Terrance J.; Rajaraman, Ganesh; Park, Chang H.; Hutchins, Charles W.; Shen, Jianwei J.; Edalji, Rohinton P.; Sun, Chaohong C.; Martin, Ruth; Gao, Wenqing; Wong, Shekman; Fang, Guowei; Elmore, Steven W.; Shen, Yu; Kati, Warren M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">8369-8384</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The development of bromodomain and extraterminal domain (BET) bromodomain inhibitors and their examn. in clin. studies, particularly in oncol. settings, has garnered substantial recent interest.  An effort to generate novel BET bromodomain inhibitors with excellent potency and drug metab. and pharmacokinetics (DMPK) properties was initiated based upon elaboration of a simple pyridone core.  Efforts to develop a bidentate interaction with a crit. asparagine residue resulted in the incorporation of a pyrrolopyridone core, which improved potency by 9-19-fold.  Addnl. structure-activity relationship (SAR) efforts aimed both at increasing potency and improving pharmacokinetic properties led to the discovery of the clin. candidate I (ABBV-075/mivebresib), which demonstrates excellent potency in biochem. and cellular assays, advantageous exposures and half-life both in animal models and in humans, and in vivo efficacy in mouse models of cancer progression and inflammation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGom-8BkipNknbVg90H21EOLACvtfcHk0liscx7Gg3Q3Pw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFOjtrbF&md5=16ea9e9ade8a22a6e99aeae16fd1bf9c</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00746&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00746%26sid%3Dliteratum%253Aachs%26aulast%3DMcDaniel%26aufirst%3DK.%2BF.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DSoltwedel%26aufirst%3DT.%26aulast%3DFidanze%26aufirst%3DS.%2BD.%26aulast%3DHasvold%26aufirst%3DL.%2BA.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DMantei%26aufirst%3DR.%2BA.%26aulast%3DPratt%26aufirst%3DJ.%2BK.%26aulast%3DSheppard%26aufirst%3DG.%2BS.%26aulast%3DBui%26aufirst%3DM.%2BH.%26aulast%3DFaivre%26aufirst%3DE.%2BJ.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DR.%26aulast%3DWarder%26aufirst%3DS.%2BE.%26aulast%3DWilcox%26aufirst%3DD.%26aulast%3DAlbert%26aufirst%3DD.%2BH.%26aulast%3DMagoc%26aufirst%3DT.%2BJ.%26aulast%3DRajaraman%26aufirst%3DG.%26aulast%3DPark%26aufirst%3DC.%2BH.%26aulast%3DHutchins%26aufirst%3DC.%2BW.%26aulast%3DShen%26aufirst%3DJ.%2BJ.%26aulast%3DEdalji%26aufirst%3DR.%2BP.%26aulast%3DSun%26aufirst%3DC.%2BC.%26aulast%3DMartin%26aufirst%3DR.%26aulast%3DGao%26aufirst%3DW.%26aulast%3DWong%26aufirst%3DS.%26aulast%3DFang%26aufirst%3DG.%26aulast%3DElmore%26aufirst%3DS.%2BW.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DKati%26aufirst%3DW.%2BM.%26atitle%3DDiscovery%2520of%2520N-%25284-%25282%252C4-Difluorophenoxy%2529-3-%25286-methyl-7-oxo-6%252C7-dihydro-1H-pyrrolo%255B2%252C3-c%255Dpyridin-4-yl%2529phenyl%2529ethanesulfonamide%2520%2528ABBV-075%252FMivebresib%2529%252C%2520a%2520Potent%2520and%2520Orally%2520Available%2520Bromodomain%2520and%2520Extraterminal%2520Domain%2520%2528BET%2529%2520Family%2520Bromodomain%2520Inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D8369%26epage%3D8384%26doi%3D10.1021%2Facs.jmedchem.7b00746" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gavai, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norris, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tortolani, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Malley, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quesnelle, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gill, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaccaro, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huynh, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahuja, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodd, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mussari, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harikrishnan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamau, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tokarski, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheriff, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rampulla, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, D.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yip, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathur, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ranasinghe, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Arienzo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Everlof, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raghavan, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poss, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vite, G. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westhouse, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wee, S.</span></span> <span> </span><span class="NLM_article-title">Abstract 5789: Discovery of clinical candidate BMS-986158, an oral BET inhibitor, for the treatment of cancer</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>78</i></span>,  <span class="NLM_fpage">5789</span>– <span class="NLM_lpage">5789</span>, <span class="refDoi"> DOI: 10.1158/1538-7445.AM2018-5789</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1158%2F1538-7445.AM2018-5789" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=2018&pages=5789-5789&author=A.+V.+Gavaiauthor=D.+Norrisauthor=D.+Tortolaniauthor=D.+O%E2%80%99Malleyauthor=Y.+Zhaoauthor=C.+Quesnelleauthor=P.+Gillauthor=W.+Vaccaroauthor=T.+Huynhauthor=V.+Ahujaauthor=D.+Doddauthor=C.+Mussariauthor=L.+Harikrishnanauthor=M.+Kamauauthor=J.+S.+Tokarskiauthor=S.+Sheriffauthor=R.+Rampullaauthor=D.-R.+Wuauthor=J.+Liauthor=H.+Zhangauthor=P.+Liauthor=D.+Sunauthor=H.+Yipauthor=Y.+Zhangauthor=A.+Mathurauthor=H.+Zhangauthor=C.+Huangauthor=Z.+Yangauthor=A.+Ranasingheauthor=C.+D%E2%80%99Arienzoauthor=C.+Suauthor=G.+Everlofauthor=L.+Zhangauthor=N.+Raghavanauthor=J.+T.+Huntauthor=M.+Possauthor=G.+D.+Viteauthor=R.+A.+Westhouseauthor=S.+Wee&title=Abstract+5789%3A+Discovery+of+clinical+candidate+BMS-986158%2C+an+oral+BET+inhibitor%2C+for+the+treatment+of+cancer&doi=10.1158%2F1538-7445.AM2018-5789"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1158%2F1538-7445.AM2018-5789&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1538-7445.AM2018-5789%26sid%3Dliteratum%253Aachs%26aulast%3DGavai%26aufirst%3DA.%2BV.%26aulast%3DNorris%26aufirst%3DD.%26aulast%3DTortolani%26aufirst%3DD.%26aulast%3DO%25E2%2580%2599Malley%26aufirst%3DD.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DQuesnelle%26aufirst%3DC.%26aulast%3DGill%26aufirst%3DP.%26aulast%3DVaccaro%26aufirst%3DW.%26aulast%3DHuynh%26aufirst%3DT.%26aulast%3DAhuja%26aufirst%3DV.%26aulast%3DDodd%26aufirst%3DD.%26aulast%3DMussari%26aufirst%3DC.%26aulast%3DHarikrishnan%26aufirst%3DL.%26aulast%3DKamau%26aufirst%3DM.%26aulast%3DTokarski%26aufirst%3DJ.%2BS.%26aulast%3DSheriff%26aufirst%3DS.%26aulast%3DRampulla%26aufirst%3DR.%26aulast%3DWu%26aufirst%3DD.-R.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DP.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DYip%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DMathur%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DHuang%26aufirst%3DC.%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DRanasinghe%26aufirst%3DA.%26aulast%3DD%25E2%2580%2599Arienzo%26aufirst%3DC.%26aulast%3DSu%26aufirst%3DC.%26aulast%3DEverlof%26aufirst%3DG.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DRaghavan%26aufirst%3DN.%26aulast%3DHunt%26aufirst%3DJ.%2BT.%26aulast%3DPoss%26aufirst%3DM.%26aulast%3DVite%26aufirst%3DG.%2BD.%26aulast%3DWesthouse%26aufirst%3DR.%2BA.%26aulast%3DWee%26aufirst%3DS.%26atitle%3DAbstract%25205789%253A%2520Discovery%2520of%2520clinical%2520candidate%2520BMS-986158%252C%2520an%2520oral%2520BET%2520inhibitor%252C%2520for%2520the%2520treatment%2520of%2520cancer%26jtitle%3DCancer%2520Res.%26date%3D2018%26volume%3D78%26spage%3D5789%26epage%3D5789%26doi%3D10.1158%2F1538-7445.AM2018-5789" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bradbury, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Callis, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carr, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feron, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glossop, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graham, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hattersley, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamont, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouvry, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabow, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stokes, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stratton, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whalley, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whittaker, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wrigley, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waring, M. J.</span></span> <span> </span><span class="NLM_article-title">Optimization of a Series of Bivalent Triazolopyridazine Based Bromodomain and Extraterminal Inhibitors: The Discovery of (3R)-4-[2-[4-[1-(3-Methoxy-[1,2,4]triazolo[4,3-b]pyridazin-6-yl)-4-piperidyl]phenoxy]ethyl]-1,3-dimethyl-piperazin-2-one (AZD5153)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">7801</span>– <span class="NLM_lpage">7817</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00070</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00070" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhtleiu7rK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=7801-7817&author=R.+H.+Bradburyauthor=R.+Callisauthor=G.+R.+Carrauthor=H.+Chenauthor=E.+Clarkauthor=L.+Feronauthor=S.+Glossopauthor=M.+A.+Grahamauthor=M.+Hattersleyauthor=C.+Jonesauthor=S.+G.+Lamontauthor=G.+Ouvryauthor=A.+Patelauthor=J.+Patelauthor=A.+A.+Rabowauthor=C.+A.+Robertsauthor=S.+Stokesauthor=N.+Strattonauthor=G.+E.+Walkerauthor=L.+Wardauthor=D.+Whalleyauthor=D.+Whittakerauthor=G.+Wrigleyauthor=M.+J.+Waring&title=Optimization+of+a+Series+of+Bivalent+Triazolopyridazine+Based+Bromodomain+and+Extraterminal+Inhibitors%3A+The+Discovery+of+%283R%29-4-%5B2-%5B4-%5B1-%283-Methoxy-%5B1%2C2%2C4%5Dtriazolo%5B4%2C3-b%5Dpyridazin-6-yl%29-4-piperidyl%5Dphenoxy%5Dethyl%5D-1%2C3-dimethyl-piperazin-2-one+%28AZD5153%29&doi=10.1021%2Facs.jmedchem.6b00070"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of a Series of Bivalent Triazolopyridazine Based Bromodomain and Extraterminal Inhibitors: The Discovery of (3R)-4-[2-[4-[1-(3-Methoxy-[1,2,4]triazolo[4,3-b]pyridazin-6-yl)-4-piperidyl]phenoxy]ethyl]-1,3-dimethyl-piperazin-2-one (AZD5153)</span></div><div class="casAuthors">Bradbury, Robert H.; Callis, Rowena; Carr, Gregory R.; Chen, Huawei; Clark, Edwin; Feron, Lyman; Glossop, Steve; Graham, Mark A.; Hattersley, Maureen; Jones, Chris; Lamont, Scott G.; Ouvry, Gilles; Patel, Anil; Patel, Joe; Rabow, Alfred A.; Roberts, Craig A.; Stokes, Stephen; Stratton, Natalie; Walker, Graeme E.; Ward, Lara; Whalley, David; Whittaker, David; Wrigley, Gail; Waring, Michael J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">7801-7817</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Here the authors report the discovery and optimization of a series of bivalent bromodomain and extraterminal inhibitors.  Starting with the observation of BRD4 activity of compds. from a previous program, the compds. were optimized for BRD4 potency and phys. properties.  The optimized compd. from this campaign exhibited excellent pharmacokinetic profile and exhibited high potency in vitro and in vivo effecting c-Myc downregulation and tumor growth inhibition in xenograft studies.  This compd. was selected as the development candidate AZD5153.  The series showed enhanced potency as a result of bivalent binding and a clear correlation between BRD4 activity and cellular potency.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFRSE33Jdb17Vg90H21EOLACvtfcHk0lggthiKK5-elQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhtleiu7rK&md5=a91cdef7369dd1888c4948ba4fc9d72d</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00070&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00070%26sid%3Dliteratum%253Aachs%26aulast%3DBradbury%26aufirst%3DR.%2BH.%26aulast%3DCallis%26aufirst%3DR.%26aulast%3DCarr%26aufirst%3DG.%2BR.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DClark%26aufirst%3DE.%26aulast%3DFeron%26aufirst%3DL.%26aulast%3DGlossop%26aufirst%3DS.%26aulast%3DGraham%26aufirst%3DM.%2BA.%26aulast%3DHattersley%26aufirst%3DM.%26aulast%3DJones%26aufirst%3DC.%26aulast%3DLamont%26aufirst%3DS.%2BG.%26aulast%3DOuvry%26aufirst%3DG.%26aulast%3DPatel%26aufirst%3DA.%26aulast%3DPatel%26aufirst%3DJ.%26aulast%3DRabow%26aufirst%3DA.%2BA.%26aulast%3DRoberts%26aufirst%3DC.%2BA.%26aulast%3DStokes%26aufirst%3DS.%26aulast%3DStratton%26aufirst%3DN.%26aulast%3DWalker%26aufirst%3DG.%2BE.%26aulast%3DWard%26aufirst%3DL.%26aulast%3DWhalley%26aufirst%3DD.%26aulast%3DWhittaker%26aufirst%3DD.%26aulast%3DWrigley%26aufirst%3DG.%26aulast%3DWaring%26aufirst%3DM.%2BJ.%26atitle%3DOptimization%2520of%2520a%2520Series%2520of%2520Bivalent%2520Triazolopyridazine%2520Based%2520Bromodomain%2520and%2520Extraterminal%2520Inhibitors%253A%2520The%2520Discovery%2520of%2520%25283R%2529-4-%255B2-%255B4-%255B1-%25283-Methoxy-%255B1%252C2%252C4%255Dtriazolo%255B4%252C3-b%255Dpyridazin-6-yl%2529-4-piperidyl%255Dphenoxy%255Dethyl%255D-1%252C3-dimethyl-piperazin-2-one%2520%2528AZD5153%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D7801%26epage%3D7817%26doi%3D10.1021%2Facs.jmedchem.6b00070" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Coude, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braun, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berrou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dupont, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertrand, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masse, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raffoux, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Itzykson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delord, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riveiro, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herait, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baruchel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dombret, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gardin, C.</span></span> <span> </span><span class="NLM_article-title">BET inhibitor OTX015 targets BRD2 and BRD4 and decreases c-MYC in acute leukemia cells</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">17698</span>– <span class="NLM_lpage">17712</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.4131</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.18632%2Foncotarget.4131" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=25989842" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A280%3ADC%252BC2MflsVChsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=17698-17712&author=M.+M.+Coudeauthor=T.+Braunauthor=J.+Berrouauthor=M.+Dupontauthor=S.+Bertrandauthor=A.+Masseauthor=E.+Raffouxauthor=R.+Itzyksonauthor=M.+Delordauthor=M.+E.+Riveiroauthor=P.+Heraitauthor=A.+Baruchelauthor=H.+Dombretauthor=C.+Gardin&title=BET+inhibitor+OTX015+targets+BRD2+and+BRD4+and+decreases+c-MYC+in+acute+leukemia+cells&doi=10.18632%2Foncotarget.4131"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">BET inhibitor OTX015 targets BRD2 and BRD4 and decreases c-MYC in acute leukemia cells</span></div><div class="casAuthors">Coude Marie-Magdelaine; Braun Thorsten; Berrou Jeannig; Dupont Melanie; Bertrand Sibyl; Masse Aline; Raffoux Emmanuel; Itzykson Raphael; Baruchel Andre; Dombret Herve; Gardin Claude; Coude Marie-Magdelaine; Braun Thorsten; Gardin Claude; Raffoux Emmanuel; Itzykson Raphael; Dombret Herve; Delord Marc; Riveiro Maria E; Herait Patrice; Baruchel Andre</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">17698-712</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The bromodomain (BRD) and extraterminal (BET) proteins including BRD2, BRD3 and BRD4 have been identified as key targets for leukemia maintenance.  A novel oral inhibitor of BRD2/3/4, the thienotriazolodiazepine compound OTX015, suitable for human use, is available.  Here we report its biological effects in AML and ALL cell lines and leukemic samples.  Exposure to OTX015 lead to cell growth inhibition, cell cycle arrest and apoptosis at submicromolar concentrations in acute leukemia cell lines and patient-derived leukemic cells, as described with the canonical JQ1 BET inhibitor.  Treatment with JQ1 and OTX15 induces similar gene expression profiles in sensitive cell lines, including a c-MYC decrease and an HEXIM1 increase.  OTX015 exposure also induced a strong decrease of BRD2, BRD4 and c-MYC and increase of HEXIM1 proteins, while BRD3 expression was unchanged. c-MYC, BRD2, BRD3, BRD4 and HEXIM1 mRNA levels did not correlate however with viability following exposure to OTX015.  Sequential combinations of OTX015 with other epigenetic modifying drugs, panobinostat and azacitidine have a synergic effect on growth of the KASUMI cell line.  Our results indicate that OTX015 and JQ1 have similar biological effects in leukemic cells, supporting OTX015 evaluation in a Phase Ib trial in relapsed/refractory leukemia patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSlHRu3Td8hIa_dlEzl_fBafW6udTcc2eav3YV-Ufnc3bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MflsVChsA%253D%253D&md5=0db953a0a24b12ec2918a02d1a2df00b</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.4131&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.4131%26sid%3Dliteratum%253Aachs%26aulast%3DCoude%26aufirst%3DM.%2BM.%26aulast%3DBraun%26aufirst%3DT.%26aulast%3DBerrou%26aufirst%3DJ.%26aulast%3DDupont%26aufirst%3DM.%26aulast%3DBertrand%26aufirst%3DS.%26aulast%3DMasse%26aufirst%3DA.%26aulast%3DRaffoux%26aufirst%3DE.%26aulast%3DItzykson%26aufirst%3DR.%26aulast%3DDelord%26aufirst%3DM.%26aulast%3DRiveiro%26aufirst%3DM.%2BE.%26aulast%3DHerait%26aufirst%3DP.%26aulast%3DBaruchel%26aufirst%3DA.%26aulast%3DDombret%26aufirst%3DH.%26aulast%3DGardin%26aufirst%3DC.%26atitle%3DBET%2520inhibitor%2520OTX015%2520targets%2520BRD2%2520and%2520BRD4%2520and%2520decreases%2520c-MYC%2520in%2520acute%2520leukemia%2520cells%26jtitle%3DOncotarget%26date%3D2015%26volume%3D6%26spage%3D17698%26epage%3D17712%26doi%3D10.18632%2Foncotarget.4131" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Albrecht, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gehling, V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hewitt, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaswani, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Côté, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leblanc, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nasveschuk, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergeron, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantone, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jayaram, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joshi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mertz, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neiss, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Normant, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Meara, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pardo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poy, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sandy, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Supko, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sims, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harmange, J.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Audia, J. E.</span></span> <span> </span><span class="NLM_article-title">Identification of a Benzoisoxazoloazepine Inhibitor (CPI-0610) of the Bromodomain and Extra-Terminal (BET) Family as a Candidate for Human Clinical Trials</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1330</span>– <span class="NLM_lpage">1339</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01882</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01882" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFKjs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1330-1339&author=B.+K.+Albrechtauthor=V.+S.+Gehlingauthor=M.+C.+Hewittauthor=R.+G.+Vaswaniauthor=A.+C%C3%B4t%C3%A9author=Y.+Leblancauthor=C.+G.+Nasveschukauthor=S.+Bellonauthor=L.+Bergeronauthor=R.+Campbellauthor=N.+Cantoneauthor=M.+R.+Cooperauthor=R.+T.+Cummingsauthor=H.+Jayaramauthor=S.+Joshiauthor=J.+A.+Mertzauthor=A.+Neissauthor=E.+Normantauthor=M.+O%E2%80%99Mearaauthor=E.+Pardoauthor=F.+Poyauthor=P.+Sandyauthor=J.+Supkoauthor=R.+J.+Simsauthor=J.-C.+Harmangeauthor=A.+M.+Taylorauthor=J.+E.+Audia&title=Identification+of+a+Benzoisoxazoloazepine+Inhibitor+%28CPI-0610%29+of+the+Bromodomain+and+Extra-Terminal+%28BET%29+Family+as+a+Candidate+for+Human+Clinical+Trials&doi=10.1021%2Facs.jmedchem.5b01882"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a Benzoisoxazoloazepine Inhibitor (CPI-0610) of the Bromodomain and Extra-Terminal (BET) Family as a Candidate for Human Clinical Trials</span></div><div class="casAuthors">Albrecht, Brian K.; Gehling, Victor S.; Hewitt, Michael C.; Vaswani, Rishi G.; Cote, Alexandre; Leblanc, Yves; Nasveschuk, Christopher G.; Bellon, Steve; Bergeron, Louise; Campbell, Robert; Cantone, Nico; Cooper, Michael R.; Cummings, Richard T.; Jayaram, Hariharan; Joshi, Shivangi; Mertz, Jennifer A.; Neiss, Adrianne; Normant, Emmanuel; O'Meara, Michael; Pardo, Eneida; Poy, Florence; Sandy, Peter; Supko, Jeffrey; Sims, Robert J.; Harmange, Jean-Christophe; Taylor, Alexander M.; Audia, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1330-1339</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In recent years, inhibition of the interaction between the bromodomain and extra-terminal domain (BET) family of chromatin adaptors and acetyl-lysine residues on chromatin has emerged as a promising approach to regulate the expression of important disease-relevant genes, including MYC, BCL-2, and NF-κB.  Here we describe the identification and characterization of a potent and selective benzoisoxazoloazepine BET bromodomain inhibitor that attenuates BET-dependent gene expression in vivo, demonstrates antitumor efficacy in an MV-4-11 mouse xenograft model, and is currently undergoing human clin. trials for hematol. malignancies (CPI-0610).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp40qjnYl4EZLVg90H21EOLACvtfcHk0lhyd3Jd-a5UYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFKjs7g%253D&md5=5e1c5aab26aa58844a9f6670a7bbd78b</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01882&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01882%26sid%3Dliteratum%253Aachs%26aulast%3DAlbrecht%26aufirst%3DB.%2BK.%26aulast%3DGehling%26aufirst%3DV.%2BS.%26aulast%3DHewitt%26aufirst%3DM.%2BC.%26aulast%3DVaswani%26aufirst%3DR.%2BG.%26aulast%3DC%25C3%25B4t%25C3%25A9%26aufirst%3DA.%26aulast%3DLeblanc%26aufirst%3DY.%26aulast%3DNasveschuk%26aufirst%3DC.%2BG.%26aulast%3DBellon%26aufirst%3DS.%26aulast%3DBergeron%26aufirst%3DL.%26aulast%3DCampbell%26aufirst%3DR.%26aulast%3DCantone%26aufirst%3DN.%26aulast%3DCooper%26aufirst%3DM.%2BR.%26aulast%3DCummings%26aufirst%3DR.%2BT.%26aulast%3DJayaram%26aufirst%3DH.%26aulast%3DJoshi%26aufirst%3DS.%26aulast%3DMertz%26aufirst%3DJ.%2BA.%26aulast%3DNeiss%26aufirst%3DA.%26aulast%3DNormant%26aufirst%3DE.%26aulast%3DO%25E2%2580%2599Meara%26aufirst%3DM.%26aulast%3DPardo%26aufirst%3DE.%26aulast%3DPoy%26aufirst%3DF.%26aulast%3DSandy%26aufirst%3DP.%26aulast%3DSupko%26aufirst%3DJ.%26aulast%3DSims%26aufirst%3DR.%2BJ.%26aulast%3DHarmange%26aufirst%3DJ.-C.%26aulast%3DTaylor%26aufirst%3DA.%2BM.%26aulast%3DAudia%26aufirst%3DJ.%2BE.%26atitle%3DIdentification%2520of%2520a%2520Benzoisoxazoloazepine%2520Inhibitor%2520%2528CPI-0610%2529%2520of%2520the%2520Bromodomain%2520and%2520Extra-Terminal%2520%2528BET%2529%2520Family%2520as%2520a%2520Candidate%2520for%2520Human%2520Clinical%2520Trials%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1330%26epage%3D1339%26doi%3D10.1021%2Facs.jmedchem.5b01882" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tontsch-Grunt, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savarese, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerlach, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gianni, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baum, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scharn, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engelhardt, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaya, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schweifer, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerstberger, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraut, N.</span></span> <span> </span><span class="NLM_article-title">Abstract B79: BI 894999, a novel BET inhibitor: Treatment of hematological malignancies by repression of super-enhancer driven oncogenes</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">B79</span>– <span class="NLM_lpage">79</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.TARG-15-B79</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1158%2F1535-7163.TARG-15-B79" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2015&pages=B79-79&author=U.+Tontsch-Gruntauthor=F.+Savareseauthor=D.+Gerlachauthor=D.+Gianniauthor=A.+Baumauthor=D.+Scharnauthor=H.+Engelhardtauthor=O.+Kayaauthor=N.+Schweiferauthor=T.+Gerstbergerauthor=N.+Kraut&title=Abstract+B79%3A+BI+894999%2C+a+novel+BET+inhibitor%3A+Treatment+of+hematological+malignancies+by+repression+of+super-enhancer+driven+oncogenes&doi=10.1158%2F1535-7163.TARG-15-B79"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.TARG-15-B79&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.TARG-15-B79%26sid%3Dliteratum%253Aachs%26aulast%3DTontsch-Grunt%26aufirst%3DU.%26aulast%3DSavarese%26aufirst%3DF.%26aulast%3DGerlach%26aufirst%3DD.%26aulast%3DGianni%26aufirst%3DD.%26aulast%3DBaum%26aufirst%3DA.%26aulast%3DScharn%26aufirst%3DD.%26aulast%3DEngelhardt%26aufirst%3DH.%26aulast%3DKaya%26aufirst%3DO.%26aulast%3DSchweifer%26aufirst%3DN.%26aulast%3DGerstberger%26aufirst%3DT.%26aulast%3DKraut%26aufirst%3DN.%26atitle%3DAbstract%2520B79%253A%2520BI%2520894999%252C%2520a%2520novel%2520BET%2520inhibitor%253A%2520Treatment%2520of%2520hematological%2520malignancies%2520by%2520repression%2520of%2520super-enhancer%2520driven%2520oncogenes%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2015%26volume%3D14%26spage%3DB79%26epage%3D79%26doi%3D10.1158%2F1535-7163.TARG-15-B79" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, P. C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stubbs, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maduskuie, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sparks, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zolotarjova, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Favata, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feldman, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Volgina, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DiMatteo, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Falahatpisheh, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polam, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Covington, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diamond-Fosbenner, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wynn, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burn, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaddi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeleswaram, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Combs, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scherle, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hollis, G.</span></span> <span> </span><span class="NLM_article-title">Abstract 3523: Discovery of a novel BET inhibitor INCB054329</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>75</i></span>,  <span class="NLM_fpage">3523</span>– <span class="NLM_lpage">3523</span>, <span class="refDoi"> DOI: 10.1158/1538-7445.AM2015-3523</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1158%2F1538-7445.AM2015-3523" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2015&pages=3523-3523&author=P.+C.+C.+Liuauthor=X.+M.+Liuauthor=M.+C.+Stubbsauthor=T.+Maduskuieauthor=R.+Sparksauthor=N.+Zolotarjovaauthor=J.+Liauthor=X.+Wenauthor=M.+Favataauthor=P.+Feldmanauthor=A.+Volginaauthor=D.+DiMatteoauthor=R.+Collinsauthor=N.+Falahatpishehauthor=P.+Polamauthor=Y.+Liauthor=M.+Covingtonauthor=S.+Diamond-Fosbennerauthor=R.+Wynnauthor=T.+Burnauthor=K.+Vaddiauthor=S.+Yeleswaramauthor=A.+P.+Combsauthor=W.+Yaoauthor=R.+Huberauthor=P.+Scherleauthor=G.+Hollis&title=Abstract+3523%3A+Discovery+of+a+novel+BET+inhibitor+INCB054329&doi=10.1158%2F1538-7445.AM2015-3523"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1158%2F1538-7445.AM2015-3523&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1538-7445.AM2015-3523%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DP.%2BC.%2BC.%26aulast%3DLiu%26aufirst%3DX.%2BM.%26aulast%3DStubbs%26aufirst%3DM.%2BC.%26aulast%3DMaduskuie%26aufirst%3DT.%26aulast%3DSparks%26aufirst%3DR.%26aulast%3DZolotarjova%26aufirst%3DN.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DWen%26aufirst%3DX.%26aulast%3DFavata%26aufirst%3DM.%26aulast%3DFeldman%26aufirst%3DP.%26aulast%3DVolgina%26aufirst%3DA.%26aulast%3DDiMatteo%26aufirst%3DD.%26aulast%3DCollins%26aufirst%3DR.%26aulast%3DFalahatpisheh%26aufirst%3DN.%26aulast%3DPolam%26aufirst%3DP.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DCovington%26aufirst%3DM.%26aulast%3DDiamond-Fosbenner%26aufirst%3DS.%26aulast%3DWynn%26aufirst%3DR.%26aulast%3DBurn%26aufirst%3DT.%26aulast%3DVaddi%26aufirst%3DK.%26aulast%3DYeleswaram%26aufirst%3DS.%26aulast%3DCombs%26aufirst%3DA.%2BP.%26aulast%3DYao%26aufirst%3DW.%26aulast%3DHuber%26aufirst%3DR.%26aulast%3DScherle%26aufirst%3DP.%26aulast%3DHollis%26aufirst%3DG.%26atitle%3DAbstract%25203523%253A%2520Discovery%2520of%2520a%2520novel%2520BET%2520inhibitor%2520INCB054329%26jtitle%3DCancer%2520Res.%26date%3D2015%26volume%3D75%26spage%3D3523%26epage%3D3523%26doi%3D10.1158%2F1538-7445.AM2015-3523" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Attwell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campeau, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jahagirdar, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kharenko, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norek, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsujikawa, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calosing, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gesner, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lakhotia, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, H.</span></span> <span> </span><span class="NLM_article-title">Abstract C86: The clinical candidate ZEN-3694, a novel BET bromodomain inhibitor, is efficacious in the treatment of a variety of solid tumor and hematological malignancies, alone or in combination with several standard of care and targeted therapies</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">C86</span>– <span class="NLM_lpage">86</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.TARG-15-C86</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1158%2F1535-7163.TARG-15-C86" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2015&pages=C86-86&author=S.+Attwellauthor=E.+Campeauauthor=R.+Jahagirdarauthor=O.+Kharenkoauthor=K.+Norekauthor=L.+Tsujikawaauthor=C.+Calosingauthor=R.+Patelauthor=E.+Gesnerauthor=S.+Lakhotiaauthor=H.+Hansen&title=Abstract+C86%3A+The+clinical+candidate+ZEN-3694%2C+a+novel+BET+bromodomain+inhibitor%2C+is+efficacious+in+the+treatment+of+a+variety+of+solid+tumor+and+hematological+malignancies%2C+alone+or+in+combination+with+several+standard+of+care+and+targeted+therapies&doi=10.1158%2F1535-7163.TARG-15-C86"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.TARG-15-C86&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.TARG-15-C86%26sid%3Dliteratum%253Aachs%26aulast%3DAttwell%26aufirst%3DS.%26aulast%3DCampeau%26aufirst%3DE.%26aulast%3DJahagirdar%26aufirst%3DR.%26aulast%3DKharenko%26aufirst%3DO.%26aulast%3DNorek%26aufirst%3DK.%26aulast%3DTsujikawa%26aufirst%3DL.%26aulast%3DCalosing%26aufirst%3DC.%26aulast%3DPatel%26aufirst%3DR.%26aulast%3DGesner%26aufirst%3DE.%26aulast%3DLakhotia%26aufirst%3DS.%26aulast%3DHansen%26aufirst%3DH.%26atitle%3DAbstract%2520C86%253A%2520The%2520clinical%2520candidate%2520ZEN-3694%252C%2520a%2520novel%2520BET%2520bromodomain%2520inhibitor%252C%2520is%2520efficacious%2520in%2520the%2520treatment%2520of%2520a%2520variety%2520of%2520solid%2520tumor%2520and%2520hematological%2520malignancies%252C%2520alone%2520or%2520in%2520combination%2520with%2520several%2520standard%2520of%2520care%2520and%2520targeted%2520therapies%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2015%26volume%3D14%26spage%3DC86%26epage%3D86%26doi%3D10.1158%2F1535-7163.TARG-15-C86" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bates, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kusam, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tannheimer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breckenridge, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tumas, D.</span></span> <span> </span><span class="NLM_article-title">The Combination of a BET Inhibitor (GS-5829) and a BTK Inhibitor (GS-4059) Potentiates DLBCL Cell Line Cell Death and Reduces Expression of MYC, IL-10, and IL-6 in Vitro</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>128</i></span>,  <span class="NLM_fpage">5116</span>– <span class="NLM_lpage">5116</span>, <span class="refDoi"> DOI: 10.1182/blood.V128.22.5116.5116</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1182%2Fblood.V128.22.5116.5116" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2016&pages=5116-5116&author=J.+Batesauthor=S.+Kusamauthor=S.+Tannheimerauthor=J.+Chanauthor=Y.+Liauthor=D.+Breckenridgeauthor=D.+Tumas&title=The+Combination+of+a+BET+Inhibitor+%28GS-5829%29+and+a+BTK+Inhibitor+%28GS-4059%29+Potentiates+DLBCL+Cell+Line+Cell+Death+and+Reduces+Expression+of+MYC%2C+IL-10%2C+and+IL-6+in+Vitro&doi=10.1182%2Fblood.V128.22.5116.5116"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1182%2Fblood.V128.22.5116.5116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood.V128.22.5116.5116%26sid%3Dliteratum%253Aachs%26aulast%3DBates%26aufirst%3DJ.%26aulast%3DKusam%26aufirst%3DS.%26aulast%3DTannheimer%26aufirst%3DS.%26aulast%3DChan%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DBreckenridge%26aufirst%3DD.%26aulast%3DTumas%26aufirst%3DD.%26atitle%3DThe%2520Combination%2520of%2520a%2520BET%2520Inhibitor%2520%2528GS-5829%2529%2520and%2520a%2520BTK%2520Inhibitor%2520%2528GS-4059%2529%2520Potentiates%2520DLBCL%2520Cell%2520Line%2520Cell%2520Death%2520and%2520Reduces%2520Expression%2520of%2520MYC%252C%2520IL-10%252C%2520and%2520IL-6%2520in%2520Vitro%26jtitle%3DBlood%26date%3D2016%26volume%3D128%26spage%3D5116%26epage%3D5116%26doi%3D10.1182%2Fblood.V128.22.5116.5116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ran, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meagher, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaswamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Design of γ-Carboline Analogues as Potent and Specific BET Bromodomain Inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">4927</span>– <span class="NLM_lpage">4939</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00613</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00613" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVagu7rM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=4927-4939&author=X.+Ranauthor=Y.+Zhaoauthor=L.+Liuauthor=L.+Baiauthor=C.-Y.+Yangauthor=B.+Zhouauthor=J.+L.+Meagherauthor=K.+Chinnaswamyauthor=J.+A.+Stuckeyauthor=S.+Wang&title=Structure-Based+Design+of+%CE%B3-Carboline+Analogues+as+Potent+and+Specific+BET+Bromodomain+Inhibitors&doi=10.1021%2Facs.jmedchem.5b00613"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design of γ-Carboline Analogues as Potent and Specific BET Bromodomain Inhibitors</span></div><div class="casAuthors">Ran, Xu; Zhao, Yujun; Liu, Liu; Bai, Longchuan; Yang, Chao-Yie; Zhou, Bing; Meagher, Jennifer L.; Chinnaswamy, Krishnapriya; Stuckey, Jeanne A.; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4927-4939</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Small-mol. inhibitors of bromodomain and extra terminal proteins (BET), including BRD2, BRD3, and BRD4 proteins have therapeutic potential for the treatment of human cancers and other diseases and conditions.  In this paper, the authors report the design, synthesis, and evaluation of γ-carboline-contg. compds. as a new class of small-mol. BET inhibitors.  The most potent inhibitor I (RX-37) obtained from this study binds to BET bromodomain proteins (BRD2, BRD3, and BRD4) with Ki values of 3.2-24.7 nM and demonstrates high selectivity over other non-BET bromodomain-contg. proteins.  Compd. I potently and selectively inhibits cell growth in human acute leukemia cell lines harboring the rearranged mixed lineage leukemia 1 gene.  The authors have detd. a cocrystal structure of I in complex with BRD4 BD2 at 1.4 Å resoln., which provides a solid structural basis for the compd.'s high binding affinity and for its further structure-based optimization.  Compd. I represents a promising lead compd. for the development of a new class of therapeutics for the treatment of human cancer and other conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv-VCfWLBWYLVg90H21EOLACvtfcHk0ljMThwHshQGMg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVagu7rM&md5=e6e150169812e439c1d7d91416eb3ee9</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00613&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00613%26sid%3Dliteratum%253Aachs%26aulast%3DRan%26aufirst%3DX.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DBai%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DC.-Y.%26aulast%3DZhou%26aufirst%3DB.%26aulast%3DMeagher%26aufirst%3DJ.%2BL.%26aulast%3DChinnaswamy%26aufirst%3DK.%26aulast%3DStuckey%26aufirst%3DJ.%2BA.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DStructure-Based%2520Design%2520of%2520%25CE%25B3-Carboline%2520Analogues%2520as%2520Potent%2520and%2520Specific%2520BET%2520Bromodomain%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D4927%26epage%3D4939%26doi%3D10.1021%2Facs.jmedchem.5b00613" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ayoub, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hawk, L. M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herzig, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wisniewski, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gee, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, J.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berndt, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Offei-Addo, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schönbrunn, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Georg, G. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pomerantz, W. C. K.</span></span> <span> </span><span class="NLM_article-title">BET Bromodomain Inhibitors with One-Step Synthesis Discovered from Virtual Screen</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">4805</span>– <span class="NLM_lpage">4817</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01336</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01336" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC2sXot1ylsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=4805-4817&author=A.+M.+Ayoubauthor=L.+M.+L.+Hawkauthor=R.+J.+Herzigauthor=J.+Jiangauthor=A.+J.+Wisniewskiauthor=C.+T.+Geeauthor=P.+Zhaoauthor=J.-Y.+Zhuauthor=N.+Berndtauthor=N.+K.+Offei-Addoauthor=T.+G.+Scottauthor=J.+Qiauthor=J.+E.+Bradnerauthor=T.+R.+Wardauthor=E.+Sch%C3%B6nbrunnauthor=G.+I.+Georgauthor=W.+C.+K.+Pomerantz&title=BET+Bromodomain+Inhibitors+with+One-Step+Synthesis+Discovered+from+Virtual+Screen&doi=10.1021%2Facs.jmedchem.6b01336"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">BET Bromodomain Inhibitors with One-Step Synthesis Discovered from Virtual Screen</span></div><div class="casAuthors">Ayoub, Alex M.; Hawk, Laura M. L.; Herzig, Ryan J.; Jiang, Jiewei; Wisniewski, Andrea J.; Gee, Clifford T.; Zhao, Peiliang; Zhu, Jin-Yi; Berndt, Norbert; Offei-Addo, Nana K.; Scott, Thomas G.; Qi, Jun; Bradner, James E.; Ward, Timothy R.; Schonbrunn, Ernst; Georg, Gunda I.; Pomerantz, William C. K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4805-4817</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Chem. inhibition of epigenetic regulatory proteins BrdT and Brd4 is emerging as a promising therapeutic strategy in contraception, cancer, and heart disease.  We report an easily synthesized dihydropyridopyrimidine pan-BET inhibitor scaffold, which was uncovered via a virtual screen followed by testing in a fluorescence anisotropy assay.  Dihydropyridopyimidine 3 (III) was subjected to further characterization and is highly selective for the BET family of bromodomains.  Structure-activity relationship data and ligand deconstruction highlight the importance of the substitution of the uracil moiety for potency and selectivity.  Compd. 3 was also cocrystd. with Brd4 for detg. the ligand binding pose and rationalizing subsequent structure-activity data.  An addnl. series of dihydropyridopyrimidines was synthesized to exploit the proximity of a channel near the ZA loop of Brd4, leading to compds. with submicromolar affinity and cellular target engagement.  Given these findings, novel and easily synthesized inhibitors are being introduced to the growing field of bromodomain inhibitor development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpr1keKj3fUOLVg90H21EOLACvtfcHk0ljMThwHshQGMg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXot1ylsLo%253D&md5=2d537cb4dea3e8155688f183fca5612a</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01336&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01336%26sid%3Dliteratum%253Aachs%26aulast%3DAyoub%26aufirst%3DA.%2BM.%26aulast%3DHawk%26aufirst%3DL.%2BM.%2BL.%26aulast%3DHerzig%26aufirst%3DR.%2BJ.%26aulast%3DJiang%26aufirst%3DJ.%26aulast%3DWisniewski%26aufirst%3DA.%2BJ.%26aulast%3DGee%26aufirst%3DC.%2BT.%26aulast%3DZhao%26aufirst%3DP.%26aulast%3DZhu%26aufirst%3DJ.-Y.%26aulast%3DBerndt%26aufirst%3DN.%26aulast%3DOffei-Addo%26aufirst%3DN.%2BK.%26aulast%3DScott%26aufirst%3DT.%2BG.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DWard%26aufirst%3DT.%2BR.%26aulast%3DSch%25C3%25B6nbrunn%26aufirst%3DE.%26aulast%3DGeorg%26aufirst%3DG.%2BI.%26aulast%3DPomerantz%26aufirst%3DW.%2BC.%2BK.%26atitle%3DBET%2520Bromodomain%2520Inhibitors%2520with%2520One-Step%2520Synthesis%2520Discovered%2520from%2520Virtual%2520Screen%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D4805%26epage%3D4817%26doi%3D10.1021%2Facs.jmedchem.6b01336" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaudio, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonetti, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwee, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernasconi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarantelli, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rinaldi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Testoni, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cascione, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ponzoni, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mensah, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stathis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stussi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riveiro, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herait, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inghirami, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cvitkovic, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zucca, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertoni, F.</span></span> <span> </span><span class="NLM_article-title">The BET Bromodomain Inhibitor OTX015 Affects Pathogenetic Pathways in Preclinical B-cell Tumor Models and Synergizes with Targeted Drugs</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">1628</span>– <span class="NLM_lpage">1638</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.CCR-14-1561</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1158%2F1078-0432.CCR-14-1561" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=25623213" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmtVahsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=1628-1638&author=M.+Boiauthor=E.+Gaudioauthor=P.+Bonettiauthor=I.+Kweeauthor=E.+Bernasconiauthor=C.+Tarantelliauthor=A.+Rinaldiauthor=M.+Testoniauthor=L.+Cascioneauthor=M.+Ponzoniauthor=A.+A.+Mensahauthor=A.+Stathisauthor=G.+Stussiauthor=M.+E.+Riveiroauthor=P.+Heraitauthor=G.+Inghiramiauthor=E.+Cvitkovicauthor=E.+Zuccaauthor=F.+Bertoni&title=The+BET+Bromodomain+Inhibitor+OTX015+Affects+Pathogenetic+Pathways+in+Preclinical+B-cell+Tumor+Models+and+Synergizes+with+Targeted+Drugs&doi=10.1158%2F1078-0432.CCR-14-1561"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">The BET Bromodomain Inhibitor OTX015 Affects Pathogenetic Pathways in Preclinical B-cell Tumor Models and Synergizes with Targeted Drugs</span></div><div class="casAuthors">Boi, Michela; Gaudio, Eugenio; Bonetti, Paola; Kwee, Ivo; Bernasconi, Elena; Tarantelli, Chiara; Rinaldi, Andrea; Testoni, Monica; Cascione, Luciano; Ponzoni, Maurilio; Mensah, Afua Adjeiwaa; Stathis, Anastasios; Stussi, Georg; Riveiro, Maria Eugenia; Herait, Patrice; Inghirami, Giorgio; Cvitkovic, Esteban; Zucca, Emanuele; Bertoni, Francesco</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1628-1638</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">In cancer cells, the epigenome is often deregulated, and inhibition of the bromodomain and extra-terminal (BET) family of bromodomain-contg. proteins is a novel epigenetic therapeutic approach.  Preliminary results of an ongoing phase I trial have reported promising activity and tolerability with the new BET bromodomain inhibitor OTX015.  We assessed the preclin. activity of OTX015 as single agent and in combination in mature B-cell lymphoma models and performed in vitro and in vivo expts. to identify the mechanism of action and the genetic features assocd. with sensitivity to the compd.  OTX015 showed antiproliferative activity in a large panel of cell lines derived from mature B-cell lymphoid tumors with median IC50 of 240 nmol/L, without significant differences among the different histotypes.  In vitro and in vivo expts. showed that OTX015 targeted NFKB/TLR/JAK/STAT signaling pathways, MYC- and E2F1-regulated genes, cell-cycle regulation, and chromatin structure.  OTX015 presented in vitro synergism with several anticancer agents, esp. with mTOR and BTK inhibitors.  Gene expression signatures assocd. with different degrees of sensitivity to OTX015 were identified.  Although OTX015 was mostly cytostatic, the compd. induced apoptosis in a genetically defined subgroup of cells, derived from activated B-cell-like diffuse large B-cell lymphoma, bearing wtTP53, mutations in MYD88, and CD79B or CARD11.  Together with the data coming from the ongoing phase I study, the in vitro and in vivo data presented here provide the basis for further clin. investigation of OTX015 as single agent and in combination therapies.  Clin Cancer Res; 21(7); 1628-38. ©2015 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJcy3O1ElDJ7Vg90H21EOLACvtfcHk0ljMThwHshQGMg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmtVahsbc%253D&md5=5742b4d5884261d546588a339f3c2ea0</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-14-1561&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-14-1561%26sid%3Dliteratum%253Aachs%26aulast%3DBoi%26aufirst%3DM.%26aulast%3DGaudio%26aufirst%3DE.%26aulast%3DBonetti%26aufirst%3DP.%26aulast%3DKwee%26aufirst%3DI.%26aulast%3DBernasconi%26aufirst%3DE.%26aulast%3DTarantelli%26aufirst%3DC.%26aulast%3DRinaldi%26aufirst%3DA.%26aulast%3DTestoni%26aufirst%3DM.%26aulast%3DCascione%26aufirst%3DL.%26aulast%3DPonzoni%26aufirst%3DM.%26aulast%3DMensah%26aufirst%3DA.%2BA.%26aulast%3DStathis%26aufirst%3DA.%26aulast%3DStussi%26aufirst%3DG.%26aulast%3DRiveiro%26aufirst%3DM.%2BE.%26aulast%3DHerait%26aufirst%3DP.%26aulast%3DInghirami%26aufirst%3DG.%26aulast%3DCvitkovic%26aufirst%3DE.%26aulast%3DZucca%26aufirst%3DE.%26aulast%3DBertoni%26aufirst%3DF.%26atitle%3DThe%2520BET%2520Bromodomain%2520Inhibitor%2520OTX015%2520Affects%2520Pathogenetic%2520Pathways%2520in%2520Preclinical%2520B-cell%2520Tumor%2520Models%2520and%2520Synergizes%2520with%2520Targeted%2520Drugs%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2015%26volume%3D21%26spage%3D1628%26epage%3D1638%26doi%3D10.1158%2F1078-0432.CCR-14-1561" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miyoshi, S.</span>; <span class="NLM_string-name">Ooike, S.</span>; <span class="NLM_string-name">Iwata, K.</span>; <span class="NLM_string-name">Hikawa, H.</span>; <span class="NLM_string-name">Sugahara, K.</span></span> <span> </span><span class="NLM_article-title">Antitumor agent</span>. <span class="NLM_patent">WO 2009084693</span>, <span class="NLM_year">2009</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2009&author=S.+Miyoshi&author=S.+Ooike&author=K.+Iwata&author=H.+Hikawa&author=K.+Sugahara&title=Antitumor+agent"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DMiyoshi%26aufirst%3DS.%26atitle%3DAntitumor%2520agent%26date%3D2009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Doroshow, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eder, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LoRusso, P. M.</span></span> <span> </span><span class="NLM_article-title">BET inhibitors: a novel epigenetic approach</span>. <i>Ann. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">1776</span>– <span class="NLM_lpage">1787</span>, <span class="refDoi"> DOI: 10.1093/annonc/mdx157</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1093%2Fannonc%2Fmdx157" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=28838216" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A280%3ADC%252BC1cbhtlOisw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2017&pages=1776-1787&author=D.+B.+Doroshowauthor=J.+P.+Ederauthor=P.+M.+LoRusso&title=BET+inhibitors%3A+a+novel+epigenetic+approach&doi=10.1093%2Fannonc%2Fmdx157"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">BET inhibitors: a novel epigenetic approach</span></div><div class="casAuthors">Doroshow D B; Eder J P; LoRusso P M</div><div class="citationInfo"><span class="NLM_cas:title">Annals of oncology : official journal of the European Society for Medical Oncology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1776-1787</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Epigenetics has been defined as 'the structural adaptation of chromosomal regions so as to register, signal or perpetuate altered activity states.' Currently, several classes of anticancer drugs function at the epigenetic level, including inhibitors of DNA methyltransferase, histone deacetylase (HDAC), lysine-specific demethylase 1, zeste homolog 2, and bromodomain and extra-terminal motif (BET) proteins.BET proteins have multiple functions, including the initiation and elongation of transcription and cell cycle regulation.  In recent years, inhibitors of BET proteins have been developed as anticancer agents.  These inhibitors exhibit selectivity for tumor cells by preferentially binding to superenhancers, noncoding regions of DNA critical for the transcription of genes that determine a cell's identity.  Preclinical research on BET inhibitors has identified them as a potential means of targeting MYC.Early clinical trials with BET inhibitors have had mixed results, with few responses in both hematologic and solid tumors that tend to be short-lived.  Toxicities have included severe, thrombocytopenia, fatigue, nausea, vomiting, and diarrhea; GI side-effects, fatigue, and low-grade dysgeusia have limited compliance.  However, preclinical data suggest that BET inhibitors may have a promising future in combination with other agents.  They appear to be able to overcome resistance to targeted agents and have strong synergy with immune checkpoint inhibitors as well as with multiple epigenetic agents, particularly HDAC inhibitors.  In many instances, BET and HDAC inhibitors were synergistic at reduced doses, suggesting a potential means of avoiding the overlapping toxicities of the two drug classes.BET inhibitors provide a novel approach to epigenetic anticancer therapy.  However, to date they appear to have limited efficacy as single agents.  A focus on BET inhibitors in combination with other drugs such as targeted and/or as other epigenetic agents is warranted, due to limited monotherapy activity, including pharmacodynamic correlatives differential activity amongst select drug combinations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSttah97bhI2UPI-fCgwosgfW6udTcc2ebFaySmxJrWB7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cbhtlOisw%253D%253D&md5=580ac90cf48f18db8313471ed61af173</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1093%2Fannonc%2Fmdx157&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fannonc%252Fmdx157%26sid%3Dliteratum%253Aachs%26aulast%3DDoroshow%26aufirst%3DD.%2BB.%26aulast%3DEder%26aufirst%3DJ.%2BP.%26aulast%3DLoRusso%26aufirst%3DP.%2BM.%26atitle%3DBET%2520inhibitors%253A%2520a%2520novel%2520epigenetic%2520approach%26jtitle%3DAnn.%2520Oncol.%26date%3D2017%26volume%3D28%26spage%3D1776%26epage%3D1787%26doi%3D10.1093%2Fannonc%2Fmdx157" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vidler, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomsett, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodward, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoelder, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of novel small-molecule inhibitors of BRD4 using structure-based virtual screening</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">8073</span>– <span class="NLM_lpage">8088</span>, <span class="refDoi"> DOI: 10.1021/jm4011302</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm4011302" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFOjurrI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=8073-8088&author=L.+R.+Vidlerauthor=P.+Filippakopoulosauthor=O.+Fedorovauthor=S.+Picaudauthor=S.+Martinauthor=M.+Tomsettauthor=H.+Woodwardauthor=N.+Brownauthor=S.+Knappauthor=S.+Hoelder&title=Discovery+of+novel+small-molecule+inhibitors+of+BRD4+using+structure-based+virtual+screening&doi=10.1021%2Fjm4011302"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Novel Small-Molecule Inhibitors of BRD4 Using Structure-Based Virtual Screening</span></div><div class="casAuthors">Vidler, Lewis R.; Filippakopoulos, Panagis; Fedorov, Oleg; Picaud, Sarah; Martin, Sarah; Tomsett, Michael; Woodward, Hannah; Brown, Nathan; Knapp, Stefan; Hoelder, Swen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">8073-8088</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomains (BRDs) are epigenetic readers that recognize acetylated-lysine (KAc) on proteins and are implicated in a no. of diseases.  We describe a virtual screening approach to identify BRD inhibitors.  Key elements of this approach are the extensive design and use of substructure queries to compile a set of com. available compds. featuring novel putative KAc mimetics and docking this set for final compd. selection.  We describe the validation of this approach by applying it to the first BRD of BRD4.  The selection and testing of 143 compds. lead to the discovery of six novel hits, including four unprecedented KAc mimetics.  We solved the crystal structure of four hits, detd. their binding mode, and improved their potency through synthesis and the purchase of derivs.  This work provides a validated virtual screening approach that is applicable to other BRDs and describes novel KAc mimetics that can be further explored to design more potent inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGow5yj8bnohUbVg90H21EOLACvtfcHk0liACO0-NSyufg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFOjurrI&md5=c1f76e4838713335be7a93ebe4c68701</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Fjm4011302&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm4011302%26sid%3Dliteratum%253Aachs%26aulast%3DVidler%26aufirst%3DL.%2BR.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DTomsett%26aufirst%3DM.%26aulast%3DWoodward%26aufirst%3DH.%26aulast%3DBrown%26aufirst%3DN.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DHoelder%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520novel%2520small-molecule%2520inhibitors%2520of%2520BRD4%2520using%2520structure-based%2520virtual%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D8073%26epage%3D8088%26doi%3D10.1021%2Fjm4011302" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hasvold, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fidanze, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pratt, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mantei, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wada, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warder, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilcox, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buchanan, F. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smithee, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albert, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magoc, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petros, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panchal, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kovar, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soni, N. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elmore, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kati, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDaniel, K. F.</span></span> <span> </span><span class="NLM_article-title">Methylpyrrole inhibitors of BET bromodomains</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">2225</span>– <span class="NLM_lpage">2233</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2017.02.057</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1016%2Fj.bmcl.2017.02.057" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=28268136" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjsl2gsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2017&pages=2225-2233&author=L.+A.+Hasvoldauthor=G.+S.+Sheppardauthor=L.+Wangauthor=S.+D.+Fidanzeauthor=D.+Liuauthor=J.+K.+Prattauthor=R.+A.+Manteiauthor=C.+K.+Wadaauthor=R.+Hubbardauthor=Y.+Shenauthor=X.+Linauthor=X.+Huangauthor=S.+E.+Warderauthor=D.+Wilcoxauthor=L.+Liauthor=F.+G.+Buchananauthor=L.+Smitheeauthor=D.+H.+Albertauthor=T.+J.+Magocauthor=C.+H.+Parkauthor=A.+M.+Petrosauthor=S.+C.+Panchalauthor=C.+Sunauthor=P.+Kovarauthor=N.+B.+Soniauthor=S.+W.+Elmoreauthor=W.+M.+Katiauthor=K.+F.+McDaniel&title=Methylpyrrole+inhibitors+of+BET+bromodomains&doi=10.1016%2Fj.bmcl.2017.02.057"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Methylpyrrole inhibitors of BET bromodomains</span></div><div class="casAuthors">Hasvold, Lisa A.; Sheppard, George S.; Wang, Le; Fidanze, Steven D.; Liu, Dachun; Pratt, John K.; Mantei, Robert A.; Wada, Carol K.; Hubbard, Robbert; Shen, Yu; Lin, Xiaoyu; Huang, Xiaoli; Warder, Scott E.; Wilcox, Denise; Li, Leiming; Buchanan, F. Greg; Smithee, Lauren; Albert, Daniel H.; Magoc, Terrance J.; Park, Chang H.; Petros, Andrew M.; Panchal, Sanjay C.; Sun, Chaohong; Kovar, Peter; Soni, Nirupama B.; Elmore, Steven W.; Kati, Warren M.; McDaniel, Keith F.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2225-2233</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">An NMR fragment screen for binders to the bromodomains of BRD4 identified 2-methyl-3-ketopyrroles (I) and (II).  Elaboration of these fragments guided by structure-based design provided lead mols. with significant activity in a mouse tumor model.  Further modifications to the methylpyrrole core provided compds. with improved properties and enhanced activity in a mouse model of multiple myeloma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9iHIAhR06LrVg90H21EOLACvtfcHk0liACO0-NSyufg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjsl2gsLc%253D&md5=80d50b9ce910e4ed8233e31e463456fd</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2017.02.057&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2017.02.057%26sid%3Dliteratum%253Aachs%26aulast%3DHasvold%26aufirst%3DL.%2BA.%26aulast%3DSheppard%26aufirst%3DG.%2BS.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DFidanze%26aufirst%3DS.%2BD.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DPratt%26aufirst%3DJ.%2BK.%26aulast%3DMantei%26aufirst%3DR.%2BA.%26aulast%3DWada%26aufirst%3DC.%2BK.%26aulast%3DHubbard%26aufirst%3DR.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DWarder%26aufirst%3DS.%2BE.%26aulast%3DWilcox%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DBuchanan%26aufirst%3DF.%2BG.%26aulast%3DSmithee%26aufirst%3DL.%26aulast%3DAlbert%26aufirst%3DD.%2BH.%26aulast%3DMagoc%26aufirst%3DT.%2BJ.%26aulast%3DPark%26aufirst%3DC.%2BH.%26aulast%3DPetros%26aufirst%3DA.%2BM.%26aulast%3DPanchal%26aufirst%3DS.%2BC.%26aulast%3DSun%26aufirst%3DC.%26aulast%3DKovar%26aufirst%3DP.%26aulast%3DSoni%26aufirst%3DN.%2BB.%26aulast%3DElmore%26aufirst%3DS.%2BW.%26aulast%3DKati%26aufirst%3DW.%2BM.%26aulast%3DMcDaniel%26aufirst%3DK.%2BF.%26atitle%3DMethylpyrrole%2520inhibitors%2520of%2520BET%2520bromodomains%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D27%26spage%3D2225%26epage%3D2233%26doi%3D10.1016%2Fj.bmcl.2017.02.057" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gallenkamp, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gelato, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haendler, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinmann, H.</span></span> <span> </span><span class="NLM_article-title">Bromodomains and their pharmacological inhibitors</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">438</span>– <span class="NLM_lpage">464</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201300434</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1002%2Fcmdc.201300434" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=24497428" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsl2gtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=438-464&author=D.+Gallenkampauthor=K.+A.+Gelatoauthor=B.+Haendlerauthor=H.+Weinmann&title=Bromodomains+and+their+pharmacological+inhibitors&doi=10.1002%2Fcmdc.201300434"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomains and Their Pharmacological Inhibitors</span></div><div class="casAuthors">Gallenkamp, Daniel; Gelato, Kathy A.; Haendler, Bernard; Weinmann, Hilmar</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">438-464</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Over 60 bromodomains belonging to proteins with very different functions have been identified in humans.  Several of them interact with acetylated lysine residues, leading to the recruitment and stabilization of protein complexes.  The bromodomain and extra-terminal domain (BET) proteins contain tandem bromodomains which bind to acetylated histones and are thereby implicated in a no. of DNA-centered processes, including the regulation of gene expression.  The recent identification of inhibitors of BET and non-BET bromodomains is one of the few examples in which effective blockade of a protein-protein interaction can be achieved with a small mol.  This has led to major strides in the understanding of the function of bromodomain-contg. proteins and their involvement in diseases such as cancer and inflammation.  Indeed, BET bromodomain inhibitors are now being clin. evaluated for the treatment of hematol. tumors and have also been tested in clin. trials for the relatively rare BRD-NUT midline carcinoma.  This review gives an overview of the newest developments in the field, with a focus on the biol. of selected bromodomain proteins on the one hand, and on reported pharmacol. inhibitors on the other, including recent examples from the patent literature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGon-8_ymytCPbVg90H21EOLACvtfcHk0lgOYe9lLNoIOQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsl2gtL4%253D&md5=2ce7c6b5c37001fb8fb901fe04aeb606</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201300434&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201300434%26sid%3Dliteratum%253Aachs%26aulast%3DGallenkamp%26aufirst%3DD.%26aulast%3DGelato%26aufirst%3DK.%2BA.%26aulast%3DHaendler%26aufirst%3DB.%26aulast%3DWeinmann%26aufirst%3DH.%26atitle%3DBromodomains%2520and%2520their%2520pharmacological%2520inhibitors%26jtitle%3DChemMedChem%26date%3D2014%26volume%3D9%26spage%3D438%26epage%3D464%26doi%3D10.1002%2Fcmdc.201300434" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leonards, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambert, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dovey, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monteiro, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujisawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lingard, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikbin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guetzoyan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ingham, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ley, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samsonova, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gingras, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwaller, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vassiliou, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span> <span> </span><span class="NLM_article-title">Promiscuous targeting of bromodomains by bromosporine identifies BET proteins as master regulators of primary transcription response in leukemia</span>. <i>Sci. Adv.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>2</i></span>, <span class="NLM_elocation-id">e1600760</span> <span class="refDoi"> DOI: 10.1126/sciadv.1600760</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1126%2Fsciadv.1600760" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=27757418" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC1cXkvFyjtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2016&author=S.+Picaudauthor=K.+Leonardsauthor=J.+P.+Lambertauthor=O.+Doveyauthor=C.+Wellsauthor=O.+Fedorovauthor=O.+Monteiroauthor=T.+Fujisawaauthor=C.+Y.+Wangauthor=H.+Lingardauthor=C.+Tallantauthor=N.+Nikbinauthor=L.+Guetzoyanauthor=R.+Inghamauthor=S.+V.+Leyauthor=P.+Brennanauthor=S.+Mullerauthor=A.+Samsonovaauthor=A.+C.+Gingrasauthor=J.+Schwallerauthor=G.+Vassiliouauthor=S.+Knappauthor=P.+Filippakopoulos&title=Promiscuous+targeting+of+bromodomains+by+bromosporine+identifies+BET+proteins+as+master+regulators+of+primary+transcription+response+in+leukemia&doi=10.1126%2Fsciadv.1600760"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Promiscuous targeting of bromodomains by bromosporine identifies BET proteins as master regulators of primary transcription response in leukemia</span></div><div class="casAuthors">Picaud, Sarah; Leonards, Katharina; Lambert, Jean-Philippe; Dovey, Oliver; Wells, Christopher; Fedorov, Oleg; Monteiro, Octovia; Fujisawa, Takao; Wang, Chen-Yi; Lingard, Hannah; Tallant, Cynthia; Nikbin, Nikzad; Guetzoyan, Lucie; Ingham, Richard; Ley, Steven V.; Brennan, Paul; Muller, Susanne; Samsonova, Anastasia; Gingras, Anne-Claude; Schwaller, Juerg; Vassiliou, George; Knapp, Stefan; Filippakopoulos, Panagis</div><div class="citationInfo"><span class="NLM_cas:title">Science Advances</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">e1600760/1-e1600760/16</span>CODEN:
                <span class="NLM_cas:coden">SACDAF</span>;
        ISSN:<span class="NLM_cas:issn">2375-2548</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Bromodomains (BRDs) have emerged as compelling targets for cancer therapy.  The development of selective and potent BET (bromo and extra-terminal) inhibitors and their significant activity in diverse tumor models have rapidly translated into clin. studies and have motivated drug development efforts targeting non-BET BRDs.  However, the complexmultidomain/subunit architecture of BRD protein complexes complicates predictions of the consequences of their pharmacol. targeting.  To address this issue, we developed a promiscuous BRD inhibitor [bromosporine (BSP)] that broadly targets BRDs (including BETs) with nanomolar affinity, creating a tool for the identification of cellular processes and diseases where BRDs have a regulatory function.  As a proof of principle, we studied the effects of BSP on leukemic cell lines known to be sensitive to BET inhibition and found, as expected, strong antiproliferative activity.  Comparison of the modulation of transcriptional profiles by BSP after a short exposure to the inhibitor resulted in a BET inhibitor signature but no significant addnl. changes in transcription that could account for inhibition of other BRDs.  Thus, nonselective targeting of BRDs identified BETs, but not other BRDs, as master regulators of contextdependent primary transcription response.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr28artLYYsqrVg90H21EOLACvtfcHk0lgOYe9lLNoIOQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXkvFyjtbw%253D&md5=3060e8fd66c657f70e0ec3acea273057</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1126%2Fsciadv.1600760&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fsciadv.1600760%26sid%3Dliteratum%253Aachs%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DLeonards%26aufirst%3DK.%26aulast%3DLambert%26aufirst%3DJ.%2BP.%26aulast%3DDovey%26aufirst%3DO.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DMonteiro%26aufirst%3DO.%26aulast%3DFujisawa%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DC.%2BY.%26aulast%3DLingard%26aufirst%3DH.%26aulast%3DTallant%26aufirst%3DC.%26aulast%3DNikbin%26aufirst%3DN.%26aulast%3DGuetzoyan%26aufirst%3DL.%26aulast%3DIngham%26aufirst%3DR.%26aulast%3DLey%26aufirst%3DS.%2BV.%26aulast%3DBrennan%26aufirst%3DP.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DSamsonova%26aufirst%3DA.%26aulast%3DGingras%26aufirst%3DA.%2BC.%26aulast%3DSchwaller%26aufirst%3DJ.%26aulast%3DVassiliou%26aufirst%3DG.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26atitle%3DPromiscuous%2520targeting%2520of%2520bromodomains%2520by%2520bromosporine%2520identifies%2520BET%2520proteins%2520as%2520master%2520regulators%2520of%2520primary%2520transcription%2520response%2520in%2520leukemia%26jtitle%3DSci.%2520Adv.%26date%3D2016%26volume%3D2%26doi%3D10.1126%2Fsciadv.1600760" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dittmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giotopoulos, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bantscheff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, W. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robson, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hopf, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savitski, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huthmacher, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gudgin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lugo, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beinke, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chapman, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soden, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Auger, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirguet, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doehner, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delwel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burnett, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeffrey, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewes, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huntly, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kouzarides, T.</span></span> <span> </span><span class="NLM_article-title">Inhibition of BET recruitment to chromatin as an effective treatment for MLL-fusion leukaemia</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>478</i></span>,  <span class="NLM_fpage">529</span>– <span class="NLM_lpage">533</span>, <span class="refDoi"> DOI: 10.1038/nature10509</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1038%2Fnature10509" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=21964340" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1GqsL%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=478&publication_year=2011&pages=529-533&author=M.+A.+Dawsonauthor=R.+K.+Prinjhaauthor=A.+Dittmannauthor=G.+Giotopoulosauthor=M.+Bantscheffauthor=W.+I.+Chanauthor=S.+C.+Robsonauthor=C.+W.+Chungauthor=C.+Hopfauthor=M.+M.+Savitskiauthor=C.+Huthmacherauthor=E.+Gudginauthor=D.+Lugoauthor=S.+Beinkeauthor=T.+D.+Chapmanauthor=E.+J.+Robertsauthor=P.+E.+Sodenauthor=K.+R.+Augerauthor=O.+Mirguetauthor=K.+Doehnerauthor=R.+Delwelauthor=A.+K.+Burnettauthor=P.+Jeffreyauthor=G.+Drewesauthor=K.+Leeauthor=B.+J.+Huntlyauthor=T.+Kouzarides&title=Inhibition+of+BET+recruitment+to+chromatin+as+an+effective+treatment+for+MLL-fusion+leukaemia&doi=10.1038%2Fnature10509"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of BET recruitment to chromatin as an effective treatment for MLL-fusion leukaemia</span></div><div class="casAuthors">Dawson, Mark A.; Prinjha, Rab K.; Dittman, Antje; Giotopoulos, George; Bantscheff, Marcus; Chan, Wai-In; Robson, Samuel C.; Chung, Chun-wa; Hopf, Carsten; Savitski, Mikhail M.; Huthmacher, Carola; Gudgin, Emma; Lugo, Dave; Beinke, Soren; Chapman, Trevor D.; Roberts, Emma J.; Soden, Peter E.; Auger, Kurt R.; Mirguet, Olivier; Doehner, Konstanze; Delwel, Ruud; Burnett, Alan K.; Jeffrey, Phillip; Drewes, Gerard; Lee, Kevin; Huntly, Brian J. P.; Kouzarides, Tony</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">478</span>
        (<span class="NLM_cas:issue">7370</span>),
    <span class="NLM_cas:pages">529-533</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Recurrent chromosomal translocations involving the mixed lineage leukemia (MLL) gene initiate aggressive forms of leukemia, which are often refractory to conventional therapies.  Many MLL-fusion partners are members of the super elongation complex (SEC), a crit. regulator of transcriptional elongation, suggesting that aberrant control of this process has an important role in leukemia induction.  Here we use a global proteomic strategy to demonstrate that MLL fusions, as part of SEC and the polymerase-assocd. factor complex (PAFc), are assocd. with the BET family of acetyl-lysine recognizing, chromatin adaptor' proteins.  These data provided the basis for therapeutic intervention in MLL-fusion leukemia, via the displacement of the BET family of proteins from chromatin.  We show that a novel small mol. inhibitor of the BET family, GSK1210151A (I-BET151), has profound efficacy against human and murine MLL-fusion leukemic cell lines, through the induction of early cell cycle arrest and apoptosis.  I-BET151 treatment in two human leukemia cell lines with different MLL fusions alters the expression of a common set of genes whose function may account for these phenotypic changes.  The mode of action of I-BET151 is, at least in part, due to the inhibition of transcription at key genes (BCL2, C-MYC and CDK6) through the displacement of BRD3/4, PAFc and SEC components from chromatin.  In vivo studies indicate that I-BET151 has significant therapeutic value, providing survival benefit in two distinct mouse models of murine MLL-AF9 and human MLL-AF4 leukemia.  Finally, the efficacy of I-BET151 against human leukemia stem cells is demonstrated, providing further evidence of its potent therapeutic potential.  These findings establish the displacement of BET proteins from chromatin as a promising epigenetic therapy for these aggressive leukemias.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4FbgJpjftv7Vg90H21EOLACvtfcHk0lgOYe9lLNoIOQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1GqsL%252FO&md5=dffb4b81da4ca8291c27139d0df86324</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1038%2Fnature10509&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature10509%26sid%3Dliteratum%253Aachs%26aulast%3DDawson%26aufirst%3DM.%2BA.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DDittmann%26aufirst%3DA.%26aulast%3DGiotopoulos%26aufirst%3DG.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DChan%26aufirst%3DW.%2BI.%26aulast%3DRobson%26aufirst%3DS.%2BC.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DHopf%26aufirst%3DC.%26aulast%3DSavitski%26aufirst%3DM.%2BM.%26aulast%3DHuthmacher%26aufirst%3DC.%26aulast%3DGudgin%26aufirst%3DE.%26aulast%3DLugo%26aufirst%3DD.%26aulast%3DBeinke%26aufirst%3DS.%26aulast%3DChapman%26aufirst%3DT.%2BD.%26aulast%3DRoberts%26aufirst%3DE.%2BJ.%26aulast%3DSoden%26aufirst%3DP.%2BE.%26aulast%3DAuger%26aufirst%3DK.%2BR.%26aulast%3DMirguet%26aufirst%3DO.%26aulast%3DDoehner%26aufirst%3DK.%26aulast%3DDelwel%26aufirst%3DR.%26aulast%3DBurnett%26aufirst%3DA.%2BK.%26aulast%3DJeffrey%26aufirst%3DP.%26aulast%3DDrewes%26aufirst%3DG.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DHuntly%26aufirst%3DB.%2BJ.%26aulast%3DKouzarides%26aufirst%3DT.%26atitle%3DInhibition%2520of%2520BET%2520recruitment%2520to%2520chromatin%2520as%2520an%2520effective%2520treatment%2520for%2520MLL-fusion%2520leukaemia%26jtitle%3DNature%26date%3D2011%26volume%3D478%26spage%3D529%26epage%3D533%26doi%3D10.1038%2Fnature10509" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ryckmans, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horne, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Correia, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owen, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tutt, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, T.</span></span> <span> </span><span class="NLM_article-title">Rapid assessment of a novel series of selective CB(2) agonists using parallel synthesis protocols: A Lipophilic Efficiency (LipE) analysis</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">4406</span>– <span class="NLM_lpage">4409</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2009.05.062</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1016%2Fj.bmcl.2009.05.062" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=19500981" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BD1MXosVGlsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=4406-4409&author=T.+Ryckmansauthor=M.+P.+Edwardsauthor=V.+A.+Horneauthor=A.+M.+Correiaauthor=D.+R.+Owenauthor=L.+R.+Thompsonauthor=I.+Tranauthor=M.+F.+Tuttauthor=T.+Young&title=Rapid+assessment+of+a+novel+series+of+selective+CB%282%29+agonists+using+parallel+synthesis+protocols%3A+A+Lipophilic+Efficiency+%28LipE%29+analysis&doi=10.1016%2Fj.bmcl.2009.05.062"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid assessment of a novel series of selective CB2 agonists using parallel synthesis protocols: A Lipophilic Efficiency (LipE) analysis</span></div><div class="casAuthors">Ryckmans, Thomas; Edwards, Martin P.; Horne, Val A.; Correia, Ana Monica; Owen, Dafydd R.; Thompson, Lisa R.; Tran, Isabelle; Tutt, Michelle F.; Young, Tim</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">4406-4409</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A series of libraries were designed using the 1-(cyclopropylmethyl)-2-alkyl-4,5,6,7-tetrahydro-1H-imidazo[4,5-c]pyridin-5-ium templates 2a-b, and Sulfonamide derivs. 11a-n proved to be potent agonists of the CB2 receptor.  Anal. of the Lipophilic Efficiency (LipE) of potent compds. provided new insight for the design of potent, metabolically stable CB2 agonists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1cihq9YHuO7Vg90H21EOLACvtfcHk0ljrS9RkgGqFNw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXosVGlsrg%253D&md5=b62d97e5f52a594cfdd510f32d2e566f</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2009.05.062&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2009.05.062%26sid%3Dliteratum%253Aachs%26aulast%3DRyckmans%26aufirst%3DT.%26aulast%3DEdwards%26aufirst%3DM.%2BP.%26aulast%3DHorne%26aufirst%3DV.%2BA.%26aulast%3DCorreia%26aufirst%3DA.%2BM.%26aulast%3DOwen%26aufirst%3DD.%2BR.%26aulast%3DThompson%26aufirst%3DL.%2BR.%26aulast%3DTran%26aufirst%3DI.%26aulast%3DTutt%26aufirst%3DM.%2BF.%26aulast%3DYoung%26aufirst%3DT.%26atitle%3DRapid%2520assessment%2520of%2520a%2520novel%2520series%2520of%2520selective%2520CB%25282%2529%2520agonists%2520using%2520parallel%2520synthesis%2520protocols%253A%2520A%2520Lipophilic%2520Efficiency%2520%2528LipE%2529%2520analysis%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2009%26volume%3D19%26spage%3D4406%26epage%3D4409%26doi%3D10.1016%2Fj.bmcl.2009.05.062" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, M. P.</span>; <span class="NLM_string-name">Price, D. A.</span></span> <span> </span><span class="NLM_article-title">Role of Physicochemical Properties and Ligand Lipophilicity Efficiency in Addressing Drug Safety Risks</span>. In  <i>Annual Reports in Medicinal Chemistry</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macor, J. E.</span></span>, Ed.; <span class="NLM_publisher-name">Academic Press</span>, <span class="NLM_year">2010</span>; Vol.  <span class="NLM_volume">45</span>, Chapter 23, pp  <span class="NLM_fpage">380</span>– <span class="NLM_lpage">391</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1016%2FS0065-7743%2810%2945023-X" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2010&pages=380-391&author=M.+P.+Edwards&author=D.+A.+Priceauthor=J.+E.+Macor&title=Annual+Reports+in+Medicinal+Chemistry"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1016%2FS0065-7743%2810%2945023-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1016%252FS0065-7743%252810%252945023-X%26sid%3Dliteratum%253Aachs%26aulast%3DEdwards%26aufirst%3DM.%2BP.%26atitle%3DRole%2520of%2520Physicochemical%2520Properties%2520and%2520Ligand%2520Lipophilicity%2520Efficiency%2520in%2520Addressing%2520Drug%2520Safety%2520Risks%26btitle%3DAnnual%2520Reports%2520in%2520Medicinal%2520Chemistry%26aulast%3DMacor%26aufirst%3DJ.%2BE.%26pub%3DAcademic%2520Press%26date%3D2010%26volume%3D45%26spage%3D380%26epage%3D391" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coste, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirguet, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilde, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gosmini, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delves, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magny, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodward, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boursier, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flynn, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouillot, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brusq, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gellibert, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riou, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Homes, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uings, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toum, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clement, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boullay, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimley, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blandel, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirilovsky, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicodeme, E.</span></span> <span> </span><span class="NLM_article-title">Discovery and characterization of small molecule inhibitors of the BET family bromodomains</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">3827</span>– <span class="NLM_lpage">3838</span>, <span class="refDoi"> DOI: 10.1021/jm200108t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm200108t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmtVWnurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=3827-3838&author=C.+W.+Chungauthor=H.+Costeauthor=J.+H.+Whiteauthor=O.+Mirguetauthor=J.+Wildeauthor=R.+L.+Gosminiauthor=C.+Delvesauthor=S.+M.+Magnyauthor=R.+Woodwardauthor=S.+A.+Hughesauthor=E.+V.+Boursierauthor=H.+Flynnauthor=A.+M.+Bouillotauthor=P.+Bamboroughauthor=J.+M.+Brusqauthor=F.+J.+Gellibertauthor=E.+J.+Jonesauthor=A.+M.+Riouauthor=P.+Homesauthor=S.+L.+Martinauthor=I.+J.+Uingsauthor=J.+Toumauthor=C.+A.+Clementauthor=A.+B.+Boullayauthor=R.+L.+Grimleyauthor=F.+M.+Blandelauthor=R.+K.+Prinjhaauthor=K.+Leeauthor=J.+Kirilovskyauthor=E.+Nicodeme&title=Discovery+and+characterization+of+small+molecule+inhibitors+of+the+BET+family+bromodomains&doi=10.1021%2Fjm200108t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and characterization of small molecule inhibitors of the BET family bromodomains</span></div><div class="casAuthors">Chung, Chun-wa; Coste, Herve; White, Julia H.; Mirguet, Olivier; Wilde, Jonathan; Gosmini, Romain L.; Delves, Chris; Magny, Sylvie M.; Woodward, Robert; Hughes, Stephen A.; Boursier, Eric V.; Flynn, Helen; Bouillot, Anne M.; Bamborough, Paul; Brusq, Jean-Marie G.; Gellibert, Francoise J.; Jones, Emma J.; Riou, Alizon M.; Homes, Paul; Martin, Sandrine L.; Uings, Iain J.; Toum, Jerome; Clement, Catherine A.; Boullay, Anne-Benedicte; Grimley, Rachel L.; Blandel, Florence M.; Prinjha, Rab K.; Lee, Kevin; Kirilovsky, Jorge; Nicodeme, Edwige</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3827-3838</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Epigenetic mechanisms of gene regulation have a profound role in normal development and disease processes.  An integral part of this mechanism occurs through lysine acetylation of histone tails which are recognized by bromodomains.  While the biol. and structural characterization of many bromodomain contg. proteins has advanced considerably, the therapeutic tractability of this protein family is only now becoming understood.  This paper describes the discovery and mol. characterization of potent (nM) small mol. inhibitors that disrupt the function of the BET family of bromodomains (Brd2, Brd3, and Brd4).  By using a combination of phenotypic screening, chemoproteomics, and biophys. studies, we have discovered that the protein-protein interactions between bromodomains and acetylated histones can be antagonized by selective small mols. that bind at the acetylated lysine recognition pocket.  X-ray crystal structures of compds. bound into bromodomains of Brd2 and Brd4 elucidate the mol. interactions of binding and explain the precisely defined stereochem. required for activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8VauuYkT5YLVg90H21EOLACvtfcHk0ljrS9RkgGqFNw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmtVWnurs%253D&md5=ebc0f6062321dece877259d775f1c19b</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Fjm200108t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200108t%26sid%3Dliteratum%253Aachs%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DCoste%26aufirst%3DH.%26aulast%3DWhite%26aufirst%3DJ.%2BH.%26aulast%3DMirguet%26aufirst%3DO.%26aulast%3DWilde%26aufirst%3DJ.%26aulast%3DGosmini%26aufirst%3DR.%2BL.%26aulast%3DDelves%26aufirst%3DC.%26aulast%3DMagny%26aufirst%3DS.%2BM.%26aulast%3DWoodward%26aufirst%3DR.%26aulast%3DHughes%26aufirst%3DS.%2BA.%26aulast%3DBoursier%26aufirst%3DE.%2BV.%26aulast%3DFlynn%26aufirst%3DH.%26aulast%3DBouillot%26aufirst%3DA.%2BM.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DBrusq%26aufirst%3DJ.%2BM.%26aulast%3DGellibert%26aufirst%3DF.%2BJ.%26aulast%3DJones%26aufirst%3DE.%2BJ.%26aulast%3DRiou%26aufirst%3DA.%2BM.%26aulast%3DHomes%26aufirst%3DP.%26aulast%3DMartin%26aufirst%3DS.%2BL.%26aulast%3DUings%26aufirst%3DI.%2BJ.%26aulast%3DToum%26aufirst%3DJ.%26aulast%3DClement%26aufirst%3DC.%2BA.%26aulast%3DBoullay%26aufirst%3DA.%2BB.%26aulast%3DGrimley%26aufirst%3DR.%2BL.%26aulast%3DBlandel%26aufirst%3DF.%2BM.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DKirilovsky%26aufirst%3DJ.%26aulast%3DNicodeme%26aufirst%3DE.%26atitle%3DDiscovery%2520and%2520characterization%2520of%2520small%2520molecule%2520inhibitors%2520of%2520the%2520BET%2520family%2520bromodomains%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D3827%26epage%3D3838%26doi%3D10.1021%2Fjm200108t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, C. M.</span></span> <span> </span><span class="NLM_article-title">The double bromodomain-containing chromatin adaptor Brd4 and transcriptional regulation</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>282</i></span>,  <span class="NLM_fpage">13141</span>– <span class="NLM_lpage">13145</span>, <span class="refDoi"> DOI: 10.1074/jbc.R700001200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1074%2Fjbc.R700001200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=17329240" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BD2sXks1Ghtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=282&publication_year=2007&pages=13141-13145&author=S.+Y.+Wuauthor=C.+M.+Chiang&title=The+double+bromodomain-containing+chromatin+adaptor+Brd4+and+transcriptional+regulation&doi=10.1074%2Fjbc.R700001200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">The Double Bromodomain-containing Chromatin Adaptor Brd4 and Transcriptional Regulation</span></div><div class="casAuthors">Wu, Shwu-Yuan; Chiang, Cheng-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">282</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">13141-13145</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">A review.  Brd4 is a double bromodomain-contg. protein that binds preferentially to acetylated chromatin.  It belongs to the BET (bromodomains and extraterminal) family that includes mammalian Brd2, Brd3, Brd4, Brdt, Drosophila Fsh, yeast Bdf1, Bdf2, and corresponding homologues in other species.  Brd4 is essential for cellular growth and has been implicated in cell cycle control, DNA replication, and gene rearrangement found in t(15;19)-assocd. carcinomas.  Recently, Brd4 has been found in several transcription complexes, including the general cofactor Mediator and the P-TEFb elongation factor, and is capable of stimulating HIV-1 transcription in a Tat-independent manner.  In addn., Brd4 is used as a cellular adaptor by some animal and human papillomaviruses (HPV) for anchoring viral genomes to mitotic chromosomes.  This tethering, mediated by Brd4 interaction with virus-encoded E2 protein, facilitates viral genome segregation during mitosis.  Interestingly, Brd4 is also identified in a transcriptional silencing complex assembled by HPV E2 and turns out to be the long sought cellular corepressor that inhibits the expression of HPV-encoded E6 and E7 oncoproteins that antagonize p53 and pRB tumor suppressor activity, resp.  The dual role of Brd4 in gene activation and repression illustrates how a dynamic chromatin-binding adaptor is able to recruit distinct transcriptional regulators to modulate promoter activity through cell cycle progression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohccDdZC9iKLVg90H21EOLACvtfcHk0ljW1ODi4dQmQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXks1Ghtbo%253D&md5=b025a5222b510a7009eca4289370b1d4</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1074%2Fjbc.R700001200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.R700001200%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DS.%2BY.%26aulast%3DChiang%26aufirst%3DC.%2BM.%26atitle%3DThe%2520double%2520bromodomain-containing%2520chromatin%2520adaptor%2520Brd4%2520and%2520transcriptional%2520regulation%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2007%26volume%3D282%26spage%3D13141%26epage%3D13145%26doi%3D10.1074%2Fjbc.R700001200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giles, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H.</span></span> <span> </span><span class="NLM_article-title">Activity of NEO2734, a novel dual inhibitor of both BET and CBP-P300, in SPOP-mutated prostate cancer</span>. <i>J. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">62</span>– <span class="NLM_lpage">62</span>, <span class="refDoi"> DOI: 10.1200/JCO.2019.37.7_suppl.62</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1200%2FJCO.2019.37.7_suppl.62" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2019&pages=62-62&author=Y.+Yanauthor=D.+Wangauthor=J.+Maauthor=X.+Pangauthor=D.+Linauthor=F.+Gilesauthor=Y.+Wangauthor=H.+Huang&title=Activity+of+NEO2734%2C+a+novel+dual+inhibitor+of+both+BET+and+CBP-P300%2C+in+SPOP-mutated+prostate+cancer&doi=10.1200%2FJCO.2019.37.7_suppl.62"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1200%2FJCO.2019.37.7_suppl.62&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1200%252FJCO.2019.37.7_suppl.62%26sid%3Dliteratum%253Aachs%26aulast%3DYan%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DD.%26aulast%3DMa%26aufirst%3DJ.%26aulast%3DPang%26aufirst%3DX.%26aulast%3DLin%26aufirst%3DD.%26aulast%3DGiles%26aufirst%3DF.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DH.%26atitle%3DActivity%2520of%2520NEO2734%252C%2520a%2520novel%2520dual%2520inhibitor%2520of%2520both%2520BET%2520and%2520CBP-P300%252C%2520in%2520SPOP-mutated%2520prostate%2520cancer%26jtitle%3DJ.%2520Clin.%2520Oncol.%26date%3D2019%26volume%3D37%26spage%3D62%26epage%3D62%26doi%3D10.1200%2FJCO.2019.37.7_suppl.62" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Raghav, K. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chavez-MacGregor, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hortobagyi, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mills, G. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meric-Bernstam, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blumenschein, G. R.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gonzalez-Angulo, A. M.</span></span> <span> </span><span class="NLM_article-title">cMET and phospho-cMET protein levels in breast cancers and survival outcomes</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">2269</span>– <span class="NLM_lpage">2277</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.CCR-11-2830</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1158%2F1078-0432.CCR-11-2830" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=22374333" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC38XmsFOntbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2012&pages=2269-2277&author=K.+P.+Raghavauthor=W.+Wangauthor=S.+Liuauthor=M.+Chavez-MacGregorauthor=X.+Mengauthor=G.+N.+Hortobagyiauthor=G.+B.+Millsauthor=F.+Meric-Bernstamauthor=G.+R.+Blumenscheinauthor=A.+M.+Gonzalez-Angulo&title=cMET+and+phospho-cMET+protein+levels+in+breast+cancers+and+survival+outcomes&doi=10.1158%2F1078-0432.CCR-11-2830"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">cMET and Phospho-cMET Protein Levels in Breast Cancers and Survival Outcomes</span></div><div class="casAuthors">Raghav, Kanwal P.; Wang, Wenting; Liu, Shuying; Chavez-MacGregor, Mariana; Meng, Xiaolong; Hortobagyi, Gabriel N.; Mills, Gordon B.; Meric-Bernstam, Funda; Blumenschein, George R., Jr.; Gonzalez-Angulo, Ana M.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2269-2277</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">PURPOSE: To evaluate cMET (mesenchymal-epithelial transition factor gene) and phospho-cMET (p-cMET) levels in breast cancer subtypes and its impact on survival outcomes.  Exptl. Design: We measured protein levels of cMET and p-cMET in 257 breast cancers using reverse phase protein array.  Regression tree method and Martingale residual plots were applied to find best cutoff point for high and low levels.  Kaplan-Meier survival curves were used to est. relapse-free (RFS) and overall (OS) survival.  Cox proportional hazards models were fit to det. assocns. of cMET/p-cMET with outcomes after adjustment for other characteristics.  RESULTS: Median age was 51 years.  There were 140 (54.5%) hormone receptor (HR) pos., 53 (20.6%) HER2 pos., and 64 (24.9%) triple-neg. tumors.  Using selected cutoffs, 181 (70.4%) and 123 (47.9%) cancers had high levels of cMET and p-cMET, resp.  There were no significant differences in mean expression of cMET (P < 0.128) and p-cMET (P < 0.088) by breast cancer subtype.  Dichotomized cMET and p-cMET level was a significant prognostic factor for RFS [HR: 2.44, 95% confidence interval (CI): 1.34-4.44, P = 0.003 and HR: 1.64, 95% CI: 1.04-2.60, P = 0.033] and OS (HR: 3.18, 95% CI: 1.43-7.11, P = 0.003 and HR: 1.92, 95% CI: 1.08-3.44, P = 0.025).  Within breast cancer subtypes, high cMET levels were assocd. with worse RFS (P = 0.014) and OS (P = 0.006) in HR-pos. tumors, and high p-cMET levels were assocd. with worse RFS (P = 0.019) and OS (P = 0.014) in HER2-pos. breast cancers.  In multivariable anal., patients with high cMET had a significantly higher risk of recurrence (HR: 2.06, 95% CI: 1.08-3.94, P = 0.028) and death (HR: 2.81, 95% CI: 1.19-6.64, P = 0.019).  High p-cMET level was assocd. with higher risk of recurrence (HR: 1.79, 95% CI: 1.08-2.95.77, P = 0.020).  CONCLUSIONS: High levels of cMET and p-cMET were seen in all breast cancer subtypes and correlated with poor prognosis.  Clin Cancer Res; 18(8); 2269-77.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowG62mI7wLAbVg90H21EOLACvtfcHk0liY_I637zsLjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmsFOntbk%253D&md5=c3ccdd6329f649fefa6c8bf62c259168</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-11-2830&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-11-2830%26sid%3Dliteratum%253Aachs%26aulast%3DRaghav%26aufirst%3DK.%2BP.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DChavez-MacGregor%26aufirst%3DM.%26aulast%3DMeng%26aufirst%3DX.%26aulast%3DHortobagyi%26aufirst%3DG.%2BN.%26aulast%3DMills%26aufirst%3DG.%2BB.%26aulast%3DMeric-Bernstam%26aufirst%3DF.%26aulast%3DBlumenschein%26aufirst%3DG.%2BR.%26aulast%3DGonzalez-Angulo%26aufirst%3DA.%2BM.%26atitle%3DcMET%2520and%2520phospho-cMET%2520protein%2520levels%2520in%2520breast%2520cancers%2520and%2520survival%2520outcomes%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2012%26volume%3D18%26spage%3D2269%26epage%3D2277%26doi%3D10.1158%2F1078-0432.CCR-11-2830" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ho-Yen, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kermorgant, S.</span></span> <span> </span><span class="NLM_article-title">The clinical and functional significance of c-Met in breast cancer: a review</span>. <i>Breast Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">17</span>– <span class="NLM_lpage">52</span>, <span class="refDoi"> DOI: 10.1186/s13058-015-0547-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1186%2Fs13058-015-0547-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=25849559" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2015&pages=17-52&author=C.+M.+Ho-Yenauthor=J.+L.+Jonesauthor=S.+Kermorgant&title=The+clinical+and+functional+significance+of+c-Met+in+breast+cancer%3A+a+review&doi=10.1186%2Fs13058-015-0547-6"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1186%2Fs13058-015-0547-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13058-015-0547-6%26sid%3Dliteratum%253Aachs%26aulast%3DHo-Yen%26aufirst%3DC.%2BM.%26aulast%3DJones%26aufirst%3DJ.%2BL.%26aulast%3DKermorgant%26aufirst%3DS.%26atitle%3DThe%2520clinical%2520and%2520functional%2520significance%2520of%2520c-Met%2520in%2520breast%2520cancer%253A%2520a%2520review%26jtitle%3DBreast%2520Cancer%2520Res.%26date%3D2015%26volume%3D17%26spage%3D17%26epage%3D52%26doi%3D10.1186%2Fs13058-015-0547-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Merenbakh-Lamin, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ben-Baruch, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeheskel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dvir, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soussan-Gutman, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeselsohn, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yelensky, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sarid, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rizel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rubinek, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolf, I.</span></span> <span> </span><span class="NLM_article-title">D538G mutation in estrogen receptor-alpha: A novel mechanism for acquired endocrine resistance in breast cancer</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">6856</span>– <span class="NLM_lpage">6864</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-13-1197</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1158%2F0008-5472.CAN-13-1197" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=24217577" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVKhurfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2013&pages=6856-6864&author=K.+Merenbakh-Laminauthor=N.+Ben-Baruchauthor=A.+Yeheskelauthor=A.+Dvirauthor=L.+Soussan-Gutmanauthor=R.+Jeselsohnauthor=R.+Yelenskyauthor=M.+Brownauthor=V.+A.+Millerauthor=D.+Saridauthor=S.+Rizelauthor=B.+Kleinauthor=T.+Rubinekauthor=I.+Wolf&title=D538G+mutation+in+estrogen+receptor-alpha%3A+A+novel+mechanism+for+acquired+endocrine+resistance+in+breast+cancer&doi=10.1158%2F0008-5472.CAN-13-1197"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">D538G Mutation in Estrogen Receptor-α: A Novel Mechanism for Acquired Endocrine Resistance in Breast Cancer</span></div><div class="casAuthors">Merenbakh-Lamin, Keren; Ben-Baruch, Noa; Yeheskel, Adva; Dvir, Addie; Soussan-Gutman, Lior; Jeselsohn, Rinath; Yelensky, Roman; Brown, Myles; Miller, Vincent A.; Sarid, David; Rizel, Shulamith; Klein, Baruch; Rubinek, Tami; Wolf, Ido</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">6856-6864</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Resistance to endocrine therapy occurs in virtually all patients with estrogen receptor a (ERa)-pos. metastatic breast cancer, and is attributed to various mechanisms including loss of ERα expression, altered activity of coregulators, and cross-talk between the ERa and growth factor signaling pathways.  To our knowledge, acquired mutations of the ERα have not been described as mediating endocrine resistance.  Samples of 13 patients with metastatic breast cancer were analyzed for mutations in cancer-related genes.  In five patients who developed resistance to hormonal therapy, a mutation of A to G at position 1,613 of ERα, resulting in a substitution of aspartic acid at position 538 to glycine (D538G), was identified in liver metastases.  Importantly, the mutation was not detected in the primary tumors obtained prior to endocrine treatment.  Structural modeling indicated that D538G substitution leads to a conformational change in the ligand-binding domain, which mimics the conformation of activated ligand-bound receptor and alters binding of tamoxifen.  Indeed, expts. in breast cancer cells indicated constitutive, ligand-independent transcriptional activity of the D538G receptor, and overexpression of it enhanced proliferation and conferred resistance to tamoxifen.  These data indicate a novel mechanism of acquired endocrine resistance in breast cancer.  Further studies are needed to assess the frequency of D538G-ERα among patients with breast cancer and explore ways to inhibit its activity and restore endocrine sensitivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWljQ_odNNUrVg90H21EOLACvtfcHk0liY_I637zsLjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvVKhurfN&md5=6dca8ee6cdb77bfc8a94d02fe5630ea8</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-13-1197&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-13-1197%26sid%3Dliteratum%253Aachs%26aulast%3DMerenbakh-Lamin%26aufirst%3DK.%26aulast%3DBen-Baruch%26aufirst%3DN.%26aulast%3DYeheskel%26aufirst%3DA.%26aulast%3DDvir%26aufirst%3DA.%26aulast%3DSoussan-Gutman%26aufirst%3DL.%26aulast%3DJeselsohn%26aufirst%3DR.%26aulast%3DYelensky%26aufirst%3DR.%26aulast%3DBrown%26aufirst%3DM.%26aulast%3DMiller%26aufirst%3DV.%2BA.%26aulast%3DSarid%26aufirst%3DD.%26aulast%3DRizel%26aufirst%3DS.%26aulast%3DKlein%26aufirst%3DB.%26aulast%3DRubinek%26aufirst%3DT.%26aulast%3DWolf%26aufirst%3DI.%26atitle%3DD538G%2520mutation%2520in%2520estrogen%2520receptor-alpha%253A%2520A%2520novel%2520mechanism%2520for%2520acquired%2520endocrine%2520resistance%2520in%2520breast%2520cancer%26jtitle%3DCancer%2520Res.%26date%3D2013%26volume%3D73%26spage%3D6856%26epage%3D6864%26doi%3D10.1158%2F0008-5472.CAN-13-1197" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Del
Re, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertolini, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crucitta, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fontanelli, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rofi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Angelis, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diodati, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavallero, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gianfilippo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salvadori, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fogli, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Falcone, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scatena, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naccarato, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roncella, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghilli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morganti, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fontana, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danesi, R.</span></span> <span> </span><span class="NLM_article-title">Overexpression of TK1 and CDK9 in plasma-derived exosomes is associated with clinical resistance to CDK4/6 inhibitors in metastatic breast cancer patients</span>. <i>Breast Cancer Res. Treat.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>178</i></span>,  <span class="NLM_fpage">57</span>– <span class="NLM_lpage">62</span>, <span class="refDoi"> DOI: 10.1007/s10549-019-05365-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1007%2Fs10549-019-05365-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=31346846" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsVGrtbzM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=178&publication_year=2019&pages=57-62&author=M.+Del%0AReauthor=I.+Bertoliniauthor=S.+Crucittaauthor=L.+Fontanelliauthor=E.+Rofiauthor=C.+De+Angelisauthor=L.+Diodatiauthor=D.+Cavalleroauthor=G.+Gianfilippoauthor=B.+Salvadoriauthor=S.+Fogliauthor=A.+Falconeauthor=C.+Scatenaauthor=A.+G.+Naccaratoauthor=M.+Roncellaauthor=M.+Ghilliauthor=R.+Morgantiauthor=A.+Fontanaauthor=R.+Danesi&title=Overexpression+of+TK1+and+CDK9+in+plasma-derived+exosomes+is+associated+with+clinical+resistance+to+CDK4%2F6+inhibitors+in+metastatic+breast+cancer+patients&doi=10.1007%2Fs10549-019-05365-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Overexpression of TK1 and CDK9 in plasma-derived exosomes is associated with clinical resistance to CDK4/6 inhibitors in metastatic breast cancer patients</span></div><div class="casAuthors">Del Re, Marzia; Bertolini, Ilaria; Crucitta, Stefania; Fontanelli, Lorenzo; Rofi, Eleonora; De Angelis, Claudia; Diodati, Lucrezia; Cavallero, Diletta; Gianfilippo, Giulia; Salvadori, Barbara; Fogli, Stefano; Falcone, Alfredo; Scatena, Cristian; Naccarato, Antonio Giuseppe; Roncella, Manuela; Ghilli, Matteo; Morganti, Riccardo; Fontana, Andrea; Danesi, Romano</div><div class="citationInfo"><span class="NLM_cas:title">Breast Cancer Research and Treatment</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">178</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">57-62</span>CODEN:
                <span class="NLM_cas:coden">BCTRD6</span>;
        ISSN:<span class="NLM_cas:issn">0167-6806</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The present article addresses the role of the mRNA expression of thymidine kinase 1 (TK1), CDK4, 6 and 9 in plasma-derived exosomes and their relevance in the pharmacol. activity of CDK4/6i.  Methods: Blood samples of 40 HR+/HER2- advanced breast cancer patients were collected before (T0) the administration of palbociclib plus hormonal therapy and after 3 mo (T1).  RNA was isolated from exosomes and analyzed for the expression of TK1, CDK 4, 6 and 9 by digital droplet PCR (ddPCR).  Results: A higher value of TK1 copies/mL at baseline (T0) was significantly assocd. with the no. of previous lines of chemotherapy (p = 0.009).  In patients with PD, a significant increase was obsd. in the no. of copies/mL of TK1 (p = 0.01) and CDK9 (p = 0.03) comparing T1 vs.  T0 values. No significant correlations between response to treatment and clin. parameters were found at univariate anal.  High baseline CDK4 expression was significantly correlated with longer PFS in patients treated with fulvestrant + palbociclib (low vs. high: 6.45 mo vs. not reached, p = 0.01).  Conclusions: The present study demonstrates that, in plasma-derived exosomes, high baseline CDK4 mRNA levels are assocd. with response to palbociclib plus hormonal therapy, while the increase in TK1 and CDK9 mRNA copies/mL is assocd. with clin. resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAoJIlNIJ7NLVg90H21EOLACvtfcHk0liRU3ufgDt4ZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsVGrtbzM&md5=e5684c7965c4527fe92c7c5c9869e428</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1007%2Fs10549-019-05365-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10549-019-05365-y%26sid%3Dliteratum%253Aachs%26aulast%3DDel%2BRe%26aufirst%3DM.%26aulast%3DBertolini%26aufirst%3DI.%26aulast%3DCrucitta%26aufirst%3DS.%26aulast%3DFontanelli%26aufirst%3DL.%26aulast%3DRofi%26aufirst%3DE.%26aulast%3DDe%2BAngelis%26aufirst%3DC.%26aulast%3DDiodati%26aufirst%3DL.%26aulast%3DCavallero%26aufirst%3DD.%26aulast%3DGianfilippo%26aufirst%3DG.%26aulast%3DSalvadori%26aufirst%3DB.%26aulast%3DFogli%26aufirst%3DS.%26aulast%3DFalcone%26aufirst%3DA.%26aulast%3DScatena%26aufirst%3DC.%26aulast%3DNaccarato%26aufirst%3DA.%2BG.%26aulast%3DRoncella%26aufirst%3DM.%26aulast%3DGhilli%26aufirst%3DM.%26aulast%3DMorganti%26aufirst%3DR.%26aulast%3DFontana%26aufirst%3DA.%26aulast%3DDanesi%26aufirst%3DR.%26atitle%3DOverexpression%2520of%2520TK1%2520and%2520CDK9%2520in%2520plasma-derived%2520exosomes%2520is%2520associated%2520with%2520clinical%2520resistance%2520to%2520CDK4%252F6%2520inhibitors%2520in%2520metastatic%2520breast%2520cancer%2520patients%26jtitle%3DBreast%2520Cancer%2520Res.%2520Treat.%26date%3D2019%26volume%3D178%26spage%3D57%26epage%3D62%26doi%3D10.1007%2Fs10549-019-05365-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witwicki, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tabassum, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janiszewska, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huh, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doherty, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohammed, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stover, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ekram, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peluffo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Santos, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krop, I. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dillon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKeown, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ott, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ni, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duarte, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anders, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winer, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Letai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barry, W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carroll, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polyak, K.</span></span> <span> </span><span class="NLM_article-title">Response and resistance to BET bromodomain inhibitors in triple-negative breast cancer</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>529</i></span>,  <span class="NLM_fpage">413</span>– <span class="NLM_lpage">417</span>, <span class="refDoi"> DOI: 10.1038/nature16508</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1038%2Fnature16508" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=26735014" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC28Xns1Gmtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=529&publication_year=2016&pages=413-417&author=S.+Shuauthor=C.+Y.+Linauthor=H.+H.+Heauthor=R.+M.+Witwickiauthor=D.+P.+Tabassumauthor=J.+M.+Robertsauthor=M.+Janiszewskaauthor=S.+J.+Huhauthor=Y.+Liangauthor=J.+Ryanauthor=E.+Dohertyauthor=H.+Mohammedauthor=H.+Guoauthor=D.+G.+Stoverauthor=M.+B.+Ekramauthor=G.+Peluffoauthor=J.+Brownauthor=C.+D%E2%80%99Santosauthor=I.+E.+Kropauthor=D.+Dillonauthor=M.+McKeownauthor=C.+Ottauthor=J.+Qiauthor=M.+Niauthor=P.+K.+Raoauthor=M.+Duarteauthor=S.+Y.+Wuauthor=C.+M.+Chiangauthor=L.+Andersauthor=R.+A.+Youngauthor=E.+Winerauthor=A.+Letaiauthor=W.+T.+Barryauthor=J.+S.+Carrollauthor=H.+Longauthor=M.+Brownauthor=X.+S.+Liuauthor=C.+A.+Meyerauthor=J.+E.+Bradnerauthor=K.+Polyak&title=Response+and+resistance+to+BET+bromodomain+inhibitors+in+triple-negative+breast+cancer&doi=10.1038%2Fnature16508"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Response and resistance to BET bromodomain inhibitors in triple-negative breast cancer</span></div><div class="casAuthors">Shu, Shaokun; Lin, Charles Y.; He, Housheng Hansen; Witwicki, Robert M.; Tabassum, Doris P.; Roberts, Justin M.; Janiszewska, Michalina; Huh, Sung Jin; Liang, Yi; Ryan, Jeremy; Doherty, Ernest; Mohammed, Hisham; Guo, Hao; Stover, Daniel G.; Ekram, Muhammad B.; Peluffo, Guillermo; Brown, Jonathan; D'Santos, Clive; Krop, Ian E.; Dillon, Deborah; McKeown, Michael; Ott, Christopher; Qi, Jun; Ni, Min; Rao, Prakash K.; Duarte, Melissa; Wu, Shwu-Yuan; Chiang, Cheng-Ming; Anders, Lars; Young, Richard A.; Winer, Eric P.; Letai, Antony; Barry, William T.; Carroll, Jason S.; Long, Henry W.; Brown, Myles; Shirley Liu, X.; Meyer, Clifford A.; Bradner, James E.; Polyak, Kornelia</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">529</span>
        (<span class="NLM_cas:issue">7586</span>),
    <span class="NLM_cas:pages">413-417</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Triple-neg. breast cancer (TNBC) is a heterogeneous and clin. aggressive disease for which there is no targeted therapy.  BET bromodomain inhibitors, which have shown efficacy in several models of cancer, have not been evaluated in TNBC.  These inhibitors displace BET bromodomain proteins such as BRD4 from chromatin by competing with their acetyl-lysine recognition modules, leading to inhibition of oncogenic transcriptional programs.  Here we report the preferential sensitivity of TNBCs to BET bromodomain inhibition in vitro and in vivo, establishing a rationale for clin. investigation and further motivation to understand mechanisms of resistance.  In paired cell lines selected for acquired resistance to BET inhibition from previously sensitive TNBCs, we failed to identify gatekeeper mutations, new driver events or drug pump activation.  BET-resistant TNBC cells remain dependent on wild-type BRD4, which supports transcription and cell proliferation in a bromodomain-independent manner.  Proteomic studies of resistant TNBC identify strong assocn. with MED1 and hyper-phosphorylation of BRD4 attributable to decreased activity of PP2A, identified here as a principal BRD4 serine phosphatase.  Together, these studies provide a rationale for BET inhibition in TNBC and present mechanism-based combination strategies to anticipate clin. drug resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZu5A85pdQDbVg90H21EOLACvtfcHk0liRU3ufgDt4ZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xns1Gmtw%253D%253D&md5=50a03861d9156ee6e344e95c5f5361b9</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1038%2Fnature16508&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature16508%26sid%3Dliteratum%253Aachs%26aulast%3DShu%26aufirst%3DS.%26aulast%3DLin%26aufirst%3DC.%2BY.%26aulast%3DHe%26aufirst%3DH.%2BH.%26aulast%3DWitwicki%26aufirst%3DR.%2BM.%26aulast%3DTabassum%26aufirst%3DD.%2BP.%26aulast%3DRoberts%26aufirst%3DJ.%2BM.%26aulast%3DJaniszewska%26aufirst%3DM.%26aulast%3DHuh%26aufirst%3DS.%2BJ.%26aulast%3DLiang%26aufirst%3DY.%26aulast%3DRyan%26aufirst%3DJ.%26aulast%3DDoherty%26aufirst%3DE.%26aulast%3DMohammed%26aufirst%3DH.%26aulast%3DGuo%26aufirst%3DH.%26aulast%3DStover%26aufirst%3DD.%2BG.%26aulast%3DEkram%26aufirst%3DM.%2BB.%26aulast%3DPeluffo%26aufirst%3DG.%26aulast%3DBrown%26aufirst%3DJ.%26aulast%3DD%25E2%2580%2599Santos%26aufirst%3DC.%26aulast%3DKrop%26aufirst%3DI.%2BE.%26aulast%3DDillon%26aufirst%3DD.%26aulast%3DMcKeown%26aufirst%3DM.%26aulast%3DOtt%26aufirst%3DC.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DNi%26aufirst%3DM.%26aulast%3DRao%26aufirst%3DP.%2BK.%26aulast%3DDuarte%26aufirst%3DM.%26aulast%3DWu%26aufirst%3DS.%2BY.%26aulast%3DChiang%26aufirst%3DC.%2BM.%26aulast%3DAnders%26aufirst%3DL.%26aulast%3DYoung%26aufirst%3DR.%2BA.%26aulast%3DWiner%26aufirst%3DE.%26aulast%3DLetai%26aufirst%3DA.%26aulast%3DBarry%26aufirst%3DW.%2BT.%26aulast%3DCarroll%26aufirst%3DJ.%2BS.%26aulast%3DLong%26aufirst%3DH.%26aulast%3DBrown%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DX.%2BS.%26aulast%3DMeyer%26aufirst%3DC.%2BA.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DPolyak%26aufirst%3DK.%26atitle%3DResponse%2520and%2520resistance%2520to%2520BET%2520bromodomain%2520inhibitors%2520in%2520triple-negative%2520breast%2520cancer%26jtitle%3DNature%26date%3D2016%26volume%3D529%26spage%3D413%26epage%3D417%26doi%3D10.1038%2Fnature16508" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ozer, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El-Gamal, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hing, Z. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blachly, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrington, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grieselhuber, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alinari, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baiocchi, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brinton, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baskin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cannon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beaver, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goettl, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucas, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woyach, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sampath, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lehman, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spevak, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Severson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shellooe, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carias, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ewing, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marimuthu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tantoy, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walters, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanftner, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rezaei, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nespi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matusow, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Habets, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ibrahim, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathe, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bollag, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Byrd, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lapalombella, R.</span></span> <span> </span><span class="NLM_article-title">BRD4 Profiling Identifies Critical Chronic Lymphocytic Leukemia Oncogenic Circuits and Reveals Sensitivity to PLX51107, a Novel Structurally Distinct BET Inhibitor</span>. <i>Cancer Discovery</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">458</span>– <span class="NLM_lpage">477</span>, <span class="refDoi"> DOI: 10.1158/2159-8290.CD-17-0902</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1158%2F2159-8290.CD-17-0902" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=29386193" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC1cXms1emsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&pages=458-477&author=H.+G.+Ozerauthor=D.+El-Gamalauthor=B.+Powellauthor=Z.+A.+Hingauthor=J.+S.+Blachlyauthor=B.+Harringtonauthor=S.+Mitchellauthor=N.+R.+Grieselhuberauthor=K.+Williamsauthor=T.+H.+Laiauthor=L.+Alinariauthor=R.+A.+Baiocchiauthor=L.+Brintonauthor=E.+Baskinauthor=M.+Cannonauthor=L.+Beaverauthor=V.+M.+Goettlauthor=D.+M.+Lucasauthor=J.+A.+Woyachauthor=D.+Sampathauthor=A.+M.+Lehmanauthor=L.+Yuauthor=J.+Zhangauthor=Y.+Maauthor=Y.+Zhangauthor=W.+Spevakauthor=S.+Shiauthor=P.+Seversonauthor=R.+Shellooeauthor=H.+Cariasauthor=G.+Tsangauthor=K.+Dongauthor=T.+Ewingauthor=A.+Marimuthuauthor=C.+Tantoyauthor=J.+Waltersauthor=L.+Sanftnerauthor=H.+Rezaeiauthor=M.+Nespiauthor=B.+Matusowauthor=G.+Habetsauthor=P.+Ibrahimauthor=C.+Zhangauthor=E.+A.+Matheauthor=G.+Bollagauthor=J.+C.+Byrdauthor=R.+Lapalombella&title=BRD4+Profiling+Identifies+Critical+Chronic+Lymphocytic+Leukemia+Oncogenic+Circuits+and+Reveals+Sensitivity+to+PLX51107%2C+a+Novel+Structurally+Distinct+BET+Inhibitor&doi=10.1158%2F2159-8290.CD-17-0902"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">BRD4 Profiling Identifies Critical Chronic Lymphocytic Leukemia Oncogenic Circuits and Reveals Sensitivity to PLX51107, a Novel Structurally Distinct BET Inhibitor</span></div><div class="casAuthors">Ozer, Hatice Gulcin; El-Gamal, Dalia; Powell, Ben; Hing, Zachary A.; Blachly, James S.; Harrington, Bonnie; Mitchell, Shaneice; Grieselhuber, Nicole R.; Williams, Katie; Lai, Tzung-Huei; Alinari, Lapo; Baiocchi, Robert A.; Brinton, Lindsey; Baskin, Elizabeth; Cannon, Matthew; Beaver, Larry; Goettl, Virginia M.; Lucas, David M.; Woyach, Jennifer A.; Sampath, Deepa; Lehman, Amy M.; Yu, Lianbo; Zhang, Jiazhong; Ma, Yan; Zhang, Ying; Spevak, Wayne; Shi, Songyuan; Severson, Paul; Shellooe, Rafe; Carias, Heidi; Tsang, Garson; Dong, Ken; Ewing, Todd; Marimuthu, Adhirai; Tantoy, Christina; Walters, Jason; Sanftner, Laura; Rezaei, Hamid; Nespi, Marika; Matusow, Bernice; Habets, Gaston; Ibrahim, Prabha; Zhang, Chao; Mathe, Ewy A.; Bollag, Gideon; Byrd, John C.; Lapalombella, Rosa</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Discovery</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">458-477</span>CODEN:
                <span class="NLM_cas:coden">CDAIB2</span>;
        ISSN:<span class="NLM_cas:issn">2159-8274</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Bromodomain and extra-terminal (BET) family proteins are key regulators of gene expression in cancer.  Herein, we utilize BRD4 profiling to identify crit. pathways involved in pathogenesis of chronic lymphocytic leukemia (CLL).  BRD4 is overexpressed in CLL and is enriched proximal to genes upregulated or de novo expressed in CLL with known functions in disease pathogenesis and progression.  These genes, including key members of the B-cell receptor (BCR) signaling pathway, provide a rationale for this therapeutic approach to identify new targets in alternative types of cancer.  Addnl., we describe PLX51107, a structurally distinct BET inhibitor with novel in vitro and in vivo pharmacol. properties that emulates or exceeds the efficacy of BCR signaling agents in preclin. models of CLL.  Herein, the discovery of the involvement of BRD4 in the core CLL transcriptional program provides a compelling rationale for clin. investigation of PLX51107 as epigenetic therapy in CLL and application of BRD4 profiling in other cancers.  Significance: To date, functional studies of BRD4 in CLL are lacking.  Through integrated genomic, functional, and pharmacol. analyses, we uncover the existence of BRD4-regulated core CLL transcriptional programs and present preclin. proof-of-concept studies validating BET inhibition as an epigenetic approach to target BCR signaling in CLL.  Cancer Discov; 8(4); 458-77. ©2018 AACR.  This article is highlighted in the In This Issue feature, p.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqpvzZtZKkb5rVg90H21EOLACvtfcHk0ljYgkRPvyTeyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXms1emsr4%253D&md5=3715a65bb8cee2a5fe53994a3d106303</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1158%2F2159-8290.CD-17-0902&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F2159-8290.CD-17-0902%26sid%3Dliteratum%253Aachs%26aulast%3DOzer%26aufirst%3DH.%2BG.%26aulast%3DEl-Gamal%26aufirst%3DD.%26aulast%3DPowell%26aufirst%3DB.%26aulast%3DHing%26aufirst%3DZ.%2BA.%26aulast%3DBlachly%26aufirst%3DJ.%2BS.%26aulast%3DHarrington%26aufirst%3DB.%26aulast%3DMitchell%26aufirst%3DS.%26aulast%3DGrieselhuber%26aufirst%3DN.%2BR.%26aulast%3DWilliams%26aufirst%3DK.%26aulast%3DLai%26aufirst%3DT.%2BH.%26aulast%3DAlinari%26aufirst%3DL.%26aulast%3DBaiocchi%26aufirst%3DR.%2BA.%26aulast%3DBrinton%26aufirst%3DL.%26aulast%3DBaskin%26aufirst%3DE.%26aulast%3DCannon%26aufirst%3DM.%26aulast%3DBeaver%26aufirst%3DL.%26aulast%3DGoettl%26aufirst%3DV.%2BM.%26aulast%3DLucas%26aufirst%3DD.%2BM.%26aulast%3DWoyach%26aufirst%3DJ.%2BA.%26aulast%3DSampath%26aufirst%3DD.%26aulast%3DLehman%26aufirst%3DA.%2BM.%26aulast%3DYu%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DMa%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DSpevak%26aufirst%3DW.%26aulast%3DShi%26aufirst%3DS.%26aulast%3DSeverson%26aufirst%3DP.%26aulast%3DShellooe%26aufirst%3DR.%26aulast%3DCarias%26aufirst%3DH.%26aulast%3DTsang%26aufirst%3DG.%26aulast%3DDong%26aufirst%3DK.%26aulast%3DEwing%26aufirst%3DT.%26aulast%3DMarimuthu%26aufirst%3DA.%26aulast%3DTantoy%26aufirst%3DC.%26aulast%3DWalters%26aufirst%3DJ.%26aulast%3DSanftner%26aufirst%3DL.%26aulast%3DRezaei%26aufirst%3DH.%26aulast%3DNespi%26aufirst%3DM.%26aulast%3DMatusow%26aufirst%3DB.%26aulast%3DHabets%26aufirst%3DG.%26aulast%3DIbrahim%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DMathe%26aufirst%3DE.%2BA.%26aulast%3DBollag%26aufirst%3DG.%26aulast%3DByrd%26aufirst%3DJ.%2BC.%26aulast%3DLapalombella%26aufirst%3DR.%26atitle%3DBRD4%2520Profiling%2520Identifies%2520Critical%2520Chronic%2520Lymphocytic%2520Leukemia%2520Oncogenic%2520Circuits%2520and%2520Reveals%2520Sensitivity%2520to%2520PLX51107%252C%2520a%2520Novel%2520Structurally%2520Distinct%2520BET%2520Inhibitor%26jtitle%3DCancer%2520Discovery%26date%3D2018%26volume%3D8%26spage%3D458%26epage%3D477%26doi%3D10.1158%2F2159-8290.CD-17-0902" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Patnaik, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carvajal, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komatsubara, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Britten, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wesolowski, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michelson, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alcantar, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Severson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orloff, M. M.</span></span> <span> </span><span class="NLM_article-title">Phase ib/2a study of PLX51107, a small molecule BET inhibitor, in subjects with advanced hematological malignancies and solid tumors</span>. <i>J. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">2550</span>– <span class="NLM_lpage">2550</span>, <span class="refDoi"> DOI: 10.1200/JCO.2018.36.15_suppl.2550</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1200%2FJCO.2018.36.15_suppl.2550" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2018&pages=2550-2550&author=A.+Patnaikauthor=R.+D.+Carvajalauthor=K.+M.+Komatsubaraauthor=C.+D.+Brittenauthor=R.+Wesolowskiauthor=G.+Michelsonauthor=O.+Alcantarauthor=C.+Zhangauthor=B.+Powellauthor=P.+Seversonauthor=E.+Martinauthor=M.+M.+Orloff&title=Phase+ib%2F2a+study+of+PLX51107%2C+a+small+molecule+BET+inhibitor%2C+in+subjects+with+advanced+hematological+malignancies+and+solid+tumors&doi=10.1200%2FJCO.2018.36.15_suppl.2550"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1200%2FJCO.2018.36.15_suppl.2550&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1200%252FJCO.2018.36.15_suppl.2550%26sid%3Dliteratum%253Aachs%26aulast%3DPatnaik%26aufirst%3DA.%26aulast%3DCarvajal%26aufirst%3DR.%2BD.%26aulast%3DKomatsubara%26aufirst%3DK.%2BM.%26aulast%3DBritten%26aufirst%3DC.%2BD.%26aulast%3DWesolowski%26aufirst%3DR.%26aulast%3DMichelson%26aufirst%3DG.%26aulast%3DAlcantar%26aufirst%3DO.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DPowell%26aufirst%3DB.%26aulast%3DSeverson%26aufirst%3DP.%26aulast%3DMartin%26aufirst%3DE.%26aulast%3DOrloff%26aufirst%3DM.%2BM.%26atitle%3DPhase%2520ib%252F2a%2520study%2520of%2520PLX51107%252C%2520a%2520small%2520molecule%2520BET%2520inhibitor%252C%2520in%2520subjects%2520with%2520advanced%2520hematological%2520malignancies%2520and%2520solid%2520tumors%26jtitle%3DJ.%2520Clin.%2520Oncol.%26date%3D2018%26volume%3D36%26spage%3D2550%26epage%3D2550%26doi%3D10.1200%2FJCO.2018.36.15_suppl.2550" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lewin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soria, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stathis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delord, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peters, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Awada, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aftimos, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bekradda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rezai, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hussain, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perez, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siu, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Massard, C.</span></span> <span> </span><span class="NLM_article-title">Phase Ib Trial With Birabresib, a Small-Molecule Inhibitor of Bromodomain and Extraterminal Proteins, in Patients With Selected Advanced Solid Tumors</span>. <i>J. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">3007</span>– <span class="NLM_lpage">3014</span>, <span class="refDoi"> DOI: 10.1200/JCO.2018.78.2292</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1200%2FJCO.2018.78.2292" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=29733771" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjvFymurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2018&pages=3007-3014&author=J.+Lewinauthor=J.+C.+Soriaauthor=A.+Stathisauthor=J.+P.+Delordauthor=S.+Petersauthor=A.+Awadaauthor=P.+G.+Aftimosauthor=M.+Bekraddaauthor=K.+Rezaiauthor=Z.+Zengauthor=A.+Hussainauthor=S.+Perezauthor=L.+L.+Siuauthor=C.+Massard&title=Phase+Ib+Trial+With+Birabresib%2C+a+Small-Molecule+Inhibitor+of+Bromodomain+and+Extraterminal+Proteins%2C+in+Patients+With+Selected+Advanced+Solid+Tumors&doi=10.1200%2FJCO.2018.78.2292"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Phase Ib trial with birabresib, a small-molecule inhibitor of bromodomain and extraterminal proteins, in patients with selected advanced solid tumors</span></div><div class="casAuthors">Lewin, Jeremy; Soria, Jean-Charles; Stathis, Anastasios; Delord, Jean-Pierre; Peters, Solange; Awada, Ahmad; Aftimos, Philippe G.; Bekradda, Mohamed; Rezai, Keyvan; Zeng, Zhen; Hussain, Azher; Perez, Susan; Siu, Lillian L.; Massard, Christophe</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Oncology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">3007-3014</span>CODEN:
                <span class="NLM_cas:coden">JCONDN</span>;
        ISSN:<span class="NLM_cas:issn">1527-7755</span>.
    
            (<span class="NLM_cas:orgname">American Society of Clinical Oncology</span>)
        </div><div class="casAbstract">Purpose Birabresib (MK-8628/OTX015) is a first-in-class bromodomain inhibitor with activity in select hematol. tumors.  Safety, efficacy, and pharmacokinetics of birabresib were evaluated in patients with castrate-resistant prostate cancer, nuclear protein in testis midline carcinoma (NMC), and non-small-cell lung cancer in this phase Ib study.  Patients and Methods Forty-seven patients were enrolled to receive birabresib once daily at starting doses of 80 mg continuously (cohort A) or 100 mg for 7 consecutive days (cohort B) in 21-day cycles using a parallel dose escalation 3 + 3 design.  The primary objective was occurrence of dose-limiting toxicities (DLTs) and detn. of the recommended phase II dose.  Results Of 46 treated patients, 26 had castrate-resistant prostate cancer, 10 NMC, and 10 non-small-cell lung cancer.  For cohort A, four of 19 (21%) evaluable patients had DLTs at 80 mg once daily (grade 3 thrombocytopenia [n = 3], ALT/hyperbilirubinemia [n = 1]) and two of three had DLTs at 100 mg once daily (grade 2 anorexia and nausea with treatment delay > 7 days [n = 1], grade 4 thrombocytopenia [n = 1]).  No DLTs occurred in cohort B.  Of 46 patients, 38 (83%) had treatment-related adverse events (diarrhea, 17 [37%]; nausea, 17 [37%]; anorexia, 14 [30%]; vomiting, 12 [26%]; thrombocytopenia 10 [22%]).  Three patients with NMC (80 mg once daily) had a partial response (Response Evaluation Criteria in Solid Tumors [RECIST] version 1.1) with duration of 1.4 to 8.4 mo.  Pharmacokinetic anal. indicated a dose-proportional increase in birabresib exposure and rapid absorption.  Conclusion The recommended phase II dose of birabresib in patients with select solid tumors is 80 mg once daily with continuous dosing.  Birabresib has dose-proportional exposure and a favorable safety profile, with clin. activity obsd. in NMC.  Future studies of birabresib must consider intermittent scheduling to possibly mitigate the toxicities of chronic dosing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzewa4HJkOK7Vg90H21EOLACvtfcHk0ljYgkRPvyTeyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjvFymurg%253D&md5=e602f260adf9fb34b0544fff8599129b</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1200%2FJCO.2018.78.2292&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1200%252FJCO.2018.78.2292%26sid%3Dliteratum%253Aachs%26aulast%3DLewin%26aufirst%3DJ.%26aulast%3DSoria%26aufirst%3DJ.%2BC.%26aulast%3DStathis%26aufirst%3DA.%26aulast%3DDelord%26aufirst%3DJ.%2BP.%26aulast%3DPeters%26aufirst%3DS.%26aulast%3DAwada%26aufirst%3DA.%26aulast%3DAftimos%26aufirst%3DP.%2BG.%26aulast%3DBekradda%26aufirst%3DM.%26aulast%3DRezai%26aufirst%3DK.%26aulast%3DZeng%26aufirst%3DZ.%26aulast%3DHussain%26aufirst%3DA.%26aulast%3DPerez%26aufirst%3DS.%26aulast%3DSiu%26aufirst%3DL.%2BL.%26aulast%3DMassard%26aufirst%3DC.%26atitle%3DPhase%2520Ib%2520Trial%2520With%2520Birabresib%252C%2520a%2520Small-Molecule%2520Inhibitor%2520of%2520Bromodomain%2520and%2520Extraterminal%2520Proteins%252C%2520in%2520Patients%2520With%2520Selected%2520Advanced%2520Solid%2520Tumors%26jtitle%3DJ.%2520Clin.%2520Oncol.%26date%3D2018%26volume%3D36%26spage%3D3007%26epage%3D3014%26doi%3D10.1200%2FJCO.2018.78.2292" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Faivre, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDaniel, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albert, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mantena, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plotnik, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilcox, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bui, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sehgal, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uziel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hessler, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, L. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellin, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mehta, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fidanze, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pratt, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasvold, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panchal, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicolette, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fossey, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Longenecker, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bigelow, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torrent, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenberg, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kati, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span> <span> </span><span class="NLM_article-title">Selective inhibition of the BD2 bromodomain of BET proteins in prostate cancer</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>578</i></span>,  <span class="NLM_fpage">306</span>– <span class="NLM_lpage">310</span>, <span class="refDoi"> DOI: 10.1038/s41586-020-1930-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1038%2Fs41586-020-1930-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=31969702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BB3cXis1Smsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=578&publication_year=2020&pages=306-310&author=E.+J.+Faivreauthor=K.+F.+McDanielauthor=D.+H.+Albertauthor=S.+R.+Mantenaauthor=J.+P.+Plotnikauthor=D.+Wilcoxauthor=L.+Zhangauthor=M.+H.+Buiauthor=G.+S.+Sheppardauthor=L.+Wangauthor=V.+Sehgalauthor=X.+Linauthor=X.+Huangauthor=X.+Luauthor=T.+Uzielauthor=P.+Hesslerauthor=L.+T.+Lamauthor=R.+J.+Bellinauthor=G.+Mehtaauthor=S.+Fidanzeauthor=J.+K.+Prattauthor=D.+Liuauthor=L.+A.+Hasvoldauthor=C.+Sunauthor=S.+C.+Panchalauthor=J.+J.+Nicoletteauthor=S.+L.+Fosseyauthor=C.+H.+Parkauthor=K.+Longeneckerauthor=L.+Bigelowauthor=M.+Torrentauthor=S.+H.+Rosenbergauthor=W.+M.+Katiauthor=Y.+Shen&title=Selective+inhibition+of+the+BD2+bromodomain+of+BET+proteins+in+prostate+cancer&doi=10.1038%2Fs41586-020-1930-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibition of the BD2 bromodomain of BET proteins in prostate cancer</span></div><div class="casAuthors">Faivre, Emily J.; McDaniel, Keith F.; Albert, Daniel H.; Mantena, Srinivasa R.; Plotnik, Joshua P.; Wilcox, Denise; Zhang, Lu; Bui, Mai H.; Sheppard, George S.; Wang, Le; Sehgal, Vasudha; Lin, Xiaoyu; Huang, Xiaoli; Lu, Xin; Uziel, Tamar; Hessler, Paul; Lam, Lloyd T.; Bellin, Richard J.; Mehta, Gaurav; Fidanze, Steve; Pratt, John K.; Liu, Dachun; Hasvold, Lisa A.; Sun, Chaohong; Panchal, Sanjay C.; Nicolette, John J.; Fossey, Stacey L.; Park, Chang H.; Longenecker, Kenton; Bigelow, Lance; Torrent, Maricel; Rosenberg, Saul H.; Kati, Warren M.; Shen, Yu</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">578</span>
        (<span class="NLM_cas:issue">7794</span>),
    <span class="NLM_cas:pages">306-310</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: Proteins of the bromodomain and extra-terminal (BET) domain family are epigenetic readers that bind acetylated histones through their bromodomains to regulate gene transcription.  Dual-bromodomain BET inhibitors (DbBi) that bind with similar affinities to the first (BD1) and second (BD2) bromodomains of BRD2, BRD3, BRD4 and BRDt have displayed modest clin. activity in monotherapy cancer trials.  A reduced no. of thrombocytes in the blood (thrombocytopenia) as well as symptoms of gastrointestinal toxicity are dose-limiting adverse events for some types of DbBi1-5.  Given that similar haematol. and gastrointestinal defects were obsd. after genetic silencing of Brd4 in mice6, the platelet and gastrointestinal toxicities may represent on-target activities assocd. with BET inhibition.  The two individual bromodomains in BET family proteins may have distinct functions7-9 and different cellular phenotypes after pharmacol. inhibition of one or both bromodomains have been reported10,11, suggesting that selectively targeting one of the bromodomains may result in a different efficacy and tolerability profile compared with DbBi.  Available compds. that are selective to individual domains lack sufficient potency and the pharmacokinetics properties that are required for in vivo efficacy and tolerability assessment10-13.  Here we carried out a medicinal chem. campaign that led to the discovery of ABBV-744, a highly potent and selective inhibitor of the BD2 domain of BET family proteins with drug-like properties.  In contrast to the broad range of cell growth inhibition induced by DbBi, the antiproliferative activity of ABBV-744 was largely, but not exclusively, restricted to cell lines of acute myeloid leukemia and prostate cancer that expressed the full-length androgen receptor (AR).  ABBV-744 retained robust activity in prostate cancer xenografts, and showed fewer platelet and gastrointestinal toxicities than the DbBi ABBV-07514.  Analyses of RNA expression and chromatin immunopptn. followed by sequencing revealed that ABBV-744 displaced BRD4 from AR-contg. super-enhancers and inhibited AR-dependent transcription, with less impact on global transcription compared with ABBV-075.  These results underscore the potential value of selectively targeting the BD2 domain of BET family proteins for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrF98kER1y9pLVg90H21EOLACvtfcHk0ljya-tpahbmCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXis1Smsr0%253D&md5=800d9106c33084390fd8cbf1ac7f1766</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1038%2Fs41586-020-1930-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-020-1930-8%26sid%3Dliteratum%253Aachs%26aulast%3DFaivre%26aufirst%3DE.%2BJ.%26aulast%3DMcDaniel%26aufirst%3DK.%2BF.%26aulast%3DAlbert%26aufirst%3DD.%2BH.%26aulast%3DMantena%26aufirst%3DS.%2BR.%26aulast%3DPlotnik%26aufirst%3DJ.%2BP.%26aulast%3DWilcox%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DBui%26aufirst%3DM.%2BH.%26aulast%3DSheppard%26aufirst%3DG.%2BS.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DSehgal%26aufirst%3DV.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DLu%26aufirst%3DX.%26aulast%3DUziel%26aufirst%3DT.%26aulast%3DHessler%26aufirst%3DP.%26aulast%3DLam%26aufirst%3DL.%2BT.%26aulast%3DBellin%26aufirst%3DR.%2BJ.%26aulast%3DMehta%26aufirst%3DG.%26aulast%3DFidanze%26aufirst%3DS.%26aulast%3DPratt%26aufirst%3DJ.%2BK.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DHasvold%26aufirst%3DL.%2BA.%26aulast%3DSun%26aufirst%3DC.%26aulast%3DPanchal%26aufirst%3DS.%2BC.%26aulast%3DNicolette%26aufirst%3DJ.%2BJ.%26aulast%3DFossey%26aufirst%3DS.%2BL.%26aulast%3DPark%26aufirst%3DC.%2BH.%26aulast%3DLongenecker%26aufirst%3DK.%26aulast%3DBigelow%26aufirst%3DL.%26aulast%3DTorrent%26aufirst%3DM.%26aulast%3DRosenberg%26aufirst%3DS.%2BH.%26aulast%3DKati%26aufirst%3DW.%2BM.%26aulast%3DShen%26aufirst%3DY.%26atitle%3DSelective%2520inhibition%2520of%2520the%2520BD2%2520bromodomain%2520of%2520BET%2520proteins%2520in%2520prostate%2520cancer%26jtitle%3DNature%26date%3D2020%26volume%3D578%26spage%3D306%26epage%3D310%26doi%3D10.1038%2Fs41586-020-1930-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abramson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blum, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flinn, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gutierrez, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maris, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Meara, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borger, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mertz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sims, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeffrey, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Younes, A.</span></span> <span> </span><span class="NLM_article-title">BET Inhibitor CPI-0610 Is Well Tolerated and Induces Responses in Diffuse Large B-Cell Lymphoma and Follicular Lymphoma: Preliminary Analysis of an Ongoing Phase 1 Study</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>126</i></span>,  <span class="NLM_fpage">1491</span>– <span class="NLM_lpage">1491</span>, <span class="refDoi"> DOI: 10.1182/blood.V126.23.1491.1491</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1182%2Fblood.V126.23.1491.1491" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=126&publication_year=2015&pages=1491-1491&author=J.+S.+Abramsonauthor=K.+A.+Blumauthor=I.+W.+Flinnauthor=M.+Gutierrezauthor=A.+Goyauthor=M.+Marisauthor=M.+Cooperauthor=M.+O%E2%80%99Mearaauthor=D.+Borgerauthor=J.+Mertzauthor=R.+J.+Simsauthor=S.+Jeffreyauthor=A.+Younes&title=BET+Inhibitor+CPI-0610+Is+Well+Tolerated+and+Induces+Responses+in+Diffuse+Large+B-Cell+Lymphoma+and+Follicular+Lymphoma%3A+Preliminary+Analysis+of+an+Ongoing+Phase+1+Study&doi=10.1182%2Fblood.V126.23.1491.1491"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1182%2Fblood.V126.23.1491.1491&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood.V126.23.1491.1491%26sid%3Dliteratum%253Aachs%26aulast%3DAbramson%26aufirst%3DJ.%2BS.%26aulast%3DBlum%26aufirst%3DK.%2BA.%26aulast%3DFlinn%26aufirst%3DI.%2BW.%26aulast%3DGutierrez%26aufirst%3DM.%26aulast%3DGoy%26aufirst%3DA.%26aulast%3DMaris%26aufirst%3DM.%26aulast%3DCooper%26aufirst%3DM.%26aulast%3DO%25E2%2580%2599Meara%26aufirst%3DM.%26aulast%3DBorger%26aufirst%3DD.%26aulast%3DMertz%26aufirst%3DJ.%26aulast%3DSims%26aufirst%3DR.%2BJ.%26aulast%3DJeffrey%26aufirst%3DS.%26aulast%3DYounes%26aufirst%3DA.%26atitle%3DBET%2520Inhibitor%2520CPI-0610%2520Is%2520Well%2520Tolerated%2520and%2520Induces%2520Responses%2520in%2520Diffuse%2520Large%2520B-Cell%2520Lymphoma%2520and%2520Follicular%2520Lymphoma%253A%2520Preliminary%2520Analysis%2520of%2520an%2520Ongoing%2520Phase%25201%2520Study%26jtitle%3DBlood%26date%3D2015%26volume%3D126%26spage%3D1491%26epage%3D1491%26doi%3D10.1182%2Fblood.V126.23.1491.1491" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Odenike, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolff, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borthakur, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aldoss, I. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rizzieri, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prebet, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dinh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Modi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freise, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jonas, B. A.</span></span> <span> </span><span class="NLM_article-title">Results from the first-in-human study of mivebresib (ABBV-075), a pan-inhibitor of bromodomain and extra terminal proteins, in patients with relapsed/refractory acute myeloid leukemia</span>. <i>J. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">7030</span>– <span class="NLM_lpage">7030</span>, <span class="refDoi"> DOI: 10.1200/JCO.2019.37.15_suppl.7030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1200%2FJCO.2019.37.15_suppl.7030" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2019&pages=7030-7030&author=O.+Odenikeauthor=J.+E.+Wolffauthor=G.+Borthakurauthor=I.+T.+Aldossauthor=D.+Rizzieriauthor=T.+Prebetauthor=B.+Huauthor=M.+Dinhauthor=X.+Chenauthor=D.+Modiauthor=K.+J.+Freiseauthor=B.+A.+Jonas&title=Results+from+the+first-in-human+study+of+mivebresib+%28ABBV-075%29%2C+a+pan-inhibitor+of+bromodomain+and+extra+terminal+proteins%2C+in+patients+with+relapsed%2Frefractory+acute+myeloid+leukemia&doi=10.1200%2FJCO.2019.37.15_suppl.7030"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1200%2FJCO.2019.37.15_suppl.7030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1200%252FJCO.2019.37.15_suppl.7030%26sid%3Dliteratum%253Aachs%26aulast%3DOdenike%26aufirst%3DO.%26aulast%3DWolff%26aufirst%3DJ.%2BE.%26aulast%3DBorthakur%26aufirst%3DG.%26aulast%3DAldoss%26aufirst%3DI.%2BT.%26aulast%3DRizzieri%26aufirst%3DD.%26aulast%3DPrebet%26aufirst%3DT.%26aulast%3DHu%26aufirst%3DB.%26aulast%3DDinh%26aufirst%3DM.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DModi%26aufirst%3DD.%26aulast%3DFreise%26aufirst%3DK.%2BJ.%26aulast%3DJonas%26aufirst%3DB.%2BA.%26atitle%3DResults%2520from%2520the%2520first-in-human%2520study%2520of%2520mivebresib%2520%2528ABBV-075%2529%252C%2520a%2520pan-inhibitor%2520of%2520bromodomain%2520and%2520extra%2520terminal%2520proteins%252C%2520in%2520patients%2520with%2520relapsed%252Frefractory%2520acute%2520myeloid%2520leukemia%26jtitle%3DJ.%2520Clin.%2520Oncol.%26date%3D2019%26volume%3D37%26spage%3D7030%26epage%3D7030%26doi%3D10.1200%2FJCO.2019.37.15_suppl.7030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gutgesell, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karumudi, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tonetti, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thatcher, G. R.</span></span> <span> </span><span class="NLM_article-title">Novel Selective Estrogen Receptor Downregulators (SERDs) Developed against Treatment-Resistant Breast Cancer</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">1325</span>– <span class="NLM_lpage">1342</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01355</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01355" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFWqu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=1325-1342&author=R.+Xiongauthor=J.+Zhaoauthor=L.+M.+Gutgesellauthor=Y.+Wangauthor=S.+Leeauthor=B.+Karumudiauthor=H.+Zhaoauthor=Y.+Luauthor=D.+A.+Tonettiauthor=G.+R.+Thatcher&title=Novel+Selective+Estrogen+Receptor+Downregulators+%28SERDs%29+Developed+against+Treatment-Resistant+Breast+Cancer&doi=10.1021%2Facs.jmedchem.6b01355"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Selective Estrogen Receptor Downregulators (SERDs) Developed against Treatment-Resistant Breast Cancer</span></div><div class="casAuthors">Xiong, Rui; Zhao, Jiong; Gutgesell, Lauren M.; Wang, Yueting; Lee, Sue; Karumudi, Bhargava; Zhao, Huiping; Lu, Yunlong; Tonetti, Debra A.; Thatcher, Gregory R. J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1325-1342</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Resistance to the selective estrogen receptor modulator (SERM) tamoxifen and to aromatase inhibitors that lower circulating estradiol occurs in up to 50% of patients, generally leading to an endocrine-independent ER+ phenotype.  Selective ER downregulators (SERDs) are able to ablate ER and thus theor. to prevent survival of both endocrine-dependent and independent ER+ tumors.  The clin. SERD, fulvestrant, is hampered by i.m. administration and undesirable pharmacokinetics.  Novel SERDs were designed using the 6-OH-benzothiophene (BT) scaffold common to arzoxifene and raloxifene.  Treatment-resistant (TR) ER+ cell lines (MCF-7:5C and MCF-7:TAM1) were used for optimization, followed by validation in the parent endocrine-dependent cell line (MCF-7:WS8), in 2D and 3D cultures, using ERα in-cell westerns, ERE-luciferase, and cell viability assays, with GDC-0810 (ARN-810) used for comparison.  Two BT SERDs with superior in vitro activity to GDC-0810 were studied for bioavailability and shown to cause regression of a TR, endocrine-independent ER+ xenograft superior to GDC-0810.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLqe0A8df78rVg90H21EOLACvtfcHk0ljya-tpahbmCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFWqu70%253D&md5=71dbdfa2152f9e9019e8d138ed41313e</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01355&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01355%26sid%3Dliteratum%253Aachs%26aulast%3DXiong%26aufirst%3DR.%26aulast%3DZhao%26aufirst%3DJ.%26aulast%3DGutgesell%26aufirst%3DL.%2BM.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DLee%26aufirst%3DS.%26aulast%3DKarumudi%26aufirst%3DB.%26aulast%3DZhao%26aufirst%3DH.%26aulast%3DLu%26aufirst%3DY.%26aulast%3DTonetti%26aufirst%3DD.%2BA.%26aulast%3DThatcher%26aufirst%3DG.%2BR.%26atitle%3DNovel%2520Selective%2520Estrogen%2520Receptor%2520Downregulators%2520%2528SERDs%2529%2520Developed%2520against%2520Treatment-Resistant%2520Breast%2520Cancer%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D1325%26epage%3D1342%26doi%3D10.1021%2Facs.jmedchem.6b01355" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pluard, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliveira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cescon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan-Chiu, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carpenter, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cunningham, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sparano, J.</span></span> <span> </span><span class="NLM_article-title">Abstract OT3-06-07: A phase I/II dose escalation and expansion study to investigate the safety, pharmacokinetics, pharmacodynamics and clinical activity of GSK525762 in combination with fulvestrant in subjects with ER+ breast cancer</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>78</i></span>,  <span class="NLM_fpage">OT3-06-07</span>, <span class="refDoi"> DOI: 10.1158/1538-7445.SABCS17-OT3-06-07</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1158%2F1538-7445.SABCS17-OT3-06-07" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=2018&pages=OT3-06-07&author=T.+Pluardauthor=S.+Ohauthor=M.+Oliveiraauthor=D.+Cesconauthor=E.+Tan-Chiuauthor=Y.+Wuauthor=C.+Carpenterauthor=E.+Cunninghamauthor=M.+Ballasauthor=A.+Dharauthor=J.+Sparano&title=Abstract+OT3-06-07%3A+A+phase+I%2FII+dose+escalation+and+expansion+study+to+investigate+the+safety%2C+pharmacokinetics%2C+pharmacodynamics+and+clinical+activity+of+GSK525762+in+combination+with+fulvestrant+in+subjects+with+ER%2B+breast+cancer&doi=10.1158%2F1538-7445.SABCS17-OT3-06-07"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1158%2F1538-7445.SABCS17-OT3-06-07&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1538-7445.SABCS17-OT3-06-07%26sid%3Dliteratum%253Aachs%26aulast%3DPluard%26aufirst%3DT.%26aulast%3DOh%26aufirst%3DS.%26aulast%3DOliveira%26aufirst%3DM.%26aulast%3DCescon%26aufirst%3DD.%26aulast%3DTan-Chiu%26aufirst%3DE.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DCarpenter%26aufirst%3DC.%26aulast%3DCunningham%26aufirst%3DE.%26aulast%3DBallas%26aufirst%3DM.%26aulast%3DDhar%26aufirst%3DA.%26aulast%3DSparano%26aufirst%3DJ.%26atitle%3DAbstract%2520OT3-06-07%253A%2520A%2520phase%2520I%252FII%2520dose%2520escalation%2520and%2520expansion%2520study%2520to%2520investigate%2520the%2520safety%252C%2520pharmacokinetics%252C%2520pharmacodynamics%2520and%2520clinical%2520activity%2520of%2520GSK525762%2520in%2520combination%2520with%2520fulvestrant%2520in%2520subjects%2520with%2520ER%252B%2520breast%2520cancer%26jtitle%3DCancer%2520Res.%26date%3D2018%26volume%3D78%26spage%3DOT3-06-07%26doi%3D10.1158%2F1538-7445.SABCS17-OT3-06-07" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mirguet, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gosmini, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toum, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clément, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnathan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brusq, J.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mordaunt, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimes, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crowe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pineau, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ajakane, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daugan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeffrey, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cutler, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haynes, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smithers, N. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uings, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witherington, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parr, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicodème, E.</span></span> <span> </span><span class="NLM_article-title">Discovery of Epigenetic Regulator I-BET762: Lead Optimization to Afford a Clinical Candidate Inhibitor of the BET Bromodomains</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">7501</span>– <span class="NLM_lpage">7515</span>, <span class="refDoi"> DOI: 10.1021/jm401088k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401088k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVSls7jJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=7501-7515&author=O.+Mirguetauthor=R.+Gosminiauthor=J.+Toumauthor=C.+A.+Cl%C3%A9mentauthor=M.+Barnathanauthor=J.-M.+Brusqauthor=J.+E.+Mordauntauthor=R.+M.+Grimesauthor=M.+Croweauthor=O.+Pineauauthor=M.+Ajakaneauthor=A.+Dauganauthor=P.+Jeffreyauthor=L.+Cutlerauthor=A.+C.+Haynesauthor=N.+N.+Smithersauthor=C.-w.+Chungauthor=P.+Bamboroughauthor=I.+J.+Uingsauthor=A.+Lewisauthor=J.+Witheringtonauthor=N.+Parrauthor=R.+K.+Prinjhaauthor=E.+Nicod%C3%A8me&title=Discovery+of+Epigenetic+Regulator+I-BET762%3A+Lead+Optimization+to+Afford+a+Clinical+Candidate+Inhibitor+of+the+BET+Bromodomains&doi=10.1021%2Fjm401088k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Epigenetic Regulator I-BET762: Lead Optimization to Afford a Clinical Candidate Inhibitor of the BET Bromodomains</span></div><div class="casAuthors">Mirguet, Olivier; Gosmini, Romain; Toum, Jerome; Clement, Catherine A.; Barnathan, Melanie; Brusq, Jean-Marie; Mordaunt, Jacqueline E.; Grimes, Richard M.; Crowe, Miriam; Pineau, Olivier; Ajakane, Myriam; Daugan, Alain; Jeffrey, Phillip; Cutler, Leanne; Haynes, Andrea C.; Smithers, Nicholas N.; Chung, Chun-wa; Bamborough, Paul; Uings, Iain J.; Lewis, Antonia; Witherington, Jason; Parr, Nigel; Prinjha, Rab K.; Nicodeme, Edwige</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">7501-7515</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bromo and extra C-terminal domain (BET) family of bromodomains are involved in binding epigenetic marks on histone proteins, more specifically acetylated lysine residues.  This paper describes the discovery and structure-activity relationships (SAR) of potent benzodiazepine inhibitors that disrupt the function of the BET family of bromodomains (BRD2, BRD3, and BRD4).  This work has yielded a potent, selective compd. I-BET762 (I) that is now under evaluation in a phase I/II clin. trial for nuclear protein in testis (NUT) midline carcinoma and other cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDitaXJSnFE7Vg90H21EOLACvtfcHk0ljDTDZyFua7sw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVSls7jJ&md5=5d059b0be08a74a5721b72a120f7e090</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1021%2Fjm401088k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401088k%26sid%3Dliteratum%253Aachs%26aulast%3DMirguet%26aufirst%3DO.%26aulast%3DGosmini%26aufirst%3DR.%26aulast%3DToum%26aufirst%3DJ.%26aulast%3DCl%25C3%25A9ment%26aufirst%3DC.%2BA.%26aulast%3DBarnathan%26aufirst%3DM.%26aulast%3DBrusq%26aufirst%3DJ.-M.%26aulast%3DMordaunt%26aufirst%3DJ.%2BE.%26aulast%3DGrimes%26aufirst%3DR.%2BM.%26aulast%3DCrowe%26aufirst%3DM.%26aulast%3DPineau%26aufirst%3DO.%26aulast%3DAjakane%26aufirst%3DM.%26aulast%3DDaugan%26aufirst%3DA.%26aulast%3DJeffrey%26aufirst%3DP.%26aulast%3DCutler%26aufirst%3DL.%26aulast%3DHaynes%26aufirst%3DA.%2BC.%26aulast%3DSmithers%26aufirst%3DN.%2BN.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DUings%26aufirst%3DI.%2BJ.%26aulast%3DLewis%26aufirst%3DA.%26aulast%3DWitherington%26aufirst%3DJ.%26aulast%3DParr%26aufirst%3DN.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DNicod%25C3%25A8me%26aufirst%3DE.%26atitle%3DDiscovery%2520of%2520Epigenetic%2520Regulator%2520I-BET762%253A%2520Lead%2520Optimization%2520to%2520Afford%2520a%2520Clinical%2520Candidate%2520Inhibitor%2520of%2520the%2520BET%2520Bromodomains%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D7501%26epage%3D7515%26doi%3D10.1021%2Fjm401088k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref70"><div class="reference"><strong class="refLabel"><a href="#ref70" class="rightTabRefNumLink">70</a></strong><div class="NLM_citation" id="rightTab-cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hanahan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinberg, R. A.</span></span> <span> </span><span class="NLM_article-title">Hallmarks of cancer: the next generation</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>144</i></span>,  <span class="NLM_fpage">646</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2011.02.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1016%2Fj.cell.2011.02.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=21376230" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjsFeqtrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=144&publication_year=2011&pages=646-674&author=D.+Hanahanauthor=R.+A.+Weinberg&title=Hallmarks+of+cancer%3A+the+next+generation&doi=10.1016%2Fj.cell.2011.02.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Hallmarks of cancer: the next generation</span></div><div class="casAuthors">Hanahan, Douglas; Weinberg, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">144</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">646-674</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The hallmarks of cancer comprise six biol. capabilities acquired during the multistep development of human tumors.  The hallmarks constitute an organizing principle for rationalizing the complexities of neoplastic disease.  They include sustaining proliferative signaling, evading growth suppressors, resisting cell death, enabling replicative immortality, inducing angiogenesis, and activating invasion and metastasis.  Underlying these hallmarks are genome instability, which generates the genetic diversity that expedites their acquisition, and inflammation, which fosters multiple hallmark functions.  Conceptual progress in the last decade has added two emerging hallmarks of potential generality to this list-reprogramming of energy metab. and evading immune destruction.  In addn. to cancer cells, tumors exhibit another dimension of complexity: they contain a repertoire of recruited, ostensibly normal cells that contribute to the acquisition of hallmark traits by creating the "tumor microenvironment.".  Recognition of the widespread applicability of these concepts will increasingly affect the development of new means to treat human cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTcpqXALcb6LVg90H21EOLACvtfcHk0lgDy08m59BgqQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjsFeqtrk%253D&md5=b36f160c417a712e83cb83c577f0018e</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2011.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2011.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DHanahan%26aufirst%3DD.%26aulast%3DWeinberg%26aufirst%3DR.%2BA.%26atitle%3DHallmarks%2520of%2520cancer%253A%2520the%2520next%2520generation%26jtitle%3DCell%26date%3D2011%26volume%3D144%26spage%3D646%26epage%3D674%26doi%3D10.1016%2Fj.cell.2011.02.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref71"><div class="reference"><strong class="refLabel"><a href="#ref71" class="rightTabRefNumLink">71</a></strong><div class="NLM_citation" id="rightTab-cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mantena, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dinh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mittelstadt, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kati, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klapczynski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDaniel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koeniger, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolff, J. E.</span></span> <span> </span><span class="NLM_article-title">Preclinical evidence for management of thrombocytopenia associated with bromodomain extra-terminal (BET) inhibition therapy</span>. <i>J. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>36</i></span>, <span class="NLM_elocation-id">e24207</span> <span class="refDoi"> DOI: 10.1200/JCO.2018.36.15_suppl.e24207</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1200%2FJCO.2018.36.15_suppl.e24207" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2018&author=S.+R.+Mantenaauthor=M.+Dinhauthor=S.+Mittelstadtauthor=W.+R.+Katiauthor=M.+Klapczynskiauthor=K.+McDanielauthor=S.+Koenigerauthor=J.+E.+Wolff&title=Preclinical+evidence+for+management+of+thrombocytopenia+associated+with+bromodomain+extra-terminal+%28BET%29+inhibition+therapy&doi=10.1200%2FJCO.2018.36.15_suppl.e24207"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1200%2FJCO.2018.36.15_suppl.e24207&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1200%252FJCO.2018.36.15_suppl.e24207%26sid%3Dliteratum%253Aachs%26aulast%3DMantena%26aufirst%3DS.%2BR.%26aulast%3DDinh%26aufirst%3DM.%26aulast%3DMittelstadt%26aufirst%3DS.%26aulast%3DKati%26aufirst%3DW.%2BR.%26aulast%3DKlapczynski%26aufirst%3DM.%26aulast%3DMcDaniel%26aufirst%3DK.%26aulast%3DKoeniger%26aufirst%3DS.%26aulast%3DWolff%26aufirst%3DJ.%2BE.%26atitle%3DPreclinical%2520evidence%2520for%2520management%2520of%2520thrombocytopenia%2520associated%2520with%2520bromodomain%2520extra-terminal%2520%2528BET%2529%2520inhibition%2520therapy%26jtitle%3DJ.%2520Clin.%2520Oncol.%26date%3D2018%26volume%3D36%26doi%3D10.1200%2FJCO.2018.36.15_suppl.e24207" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref72"><div class="reference"><strong class="refLabel"><a href="#ref72" class="rightTabRefNumLink">72</a></strong><div class="NLM_citation" id="rightTab-cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meeke, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osipo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kidawi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandel, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jordan, V. C.</span></span> <span> </span><span class="NLM_article-title">Intrinsic mechanism of estradiol-induced apoptosis in breast cancer cells resistant to estrogen deprivation</span>. <i>J. Natl. Cancer Inst.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>97</i></span>,  <span class="NLM_fpage">1746</span>– <span class="NLM_lpage">1759</span>, <span class="refDoi"> DOI: 10.1093/jnci/dji400</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1093%2Fjnci%2Fdji400" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=16333030" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlSjurnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=97&publication_year=2005&pages=1746-1759&author=J.+S.+Lewisauthor=K.+Meekeauthor=C.+Osipoauthor=E.+A.+Rossauthor=N.+Kidawiauthor=T.+Liauthor=E.+Bellauthor=N.+S.+Chandelauthor=V.+C.+Jordan&title=Intrinsic+mechanism+of+estradiol-induced+apoptosis+in+breast+cancer+cells+resistant+to+estrogen+deprivation&doi=10.1093%2Fjnci%2Fdji400"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Intrinsic Mechanism of Estradiol-Induced Apoptosis in Breast Cancer Cells Resistant to Estrogen Deprivation</span></div><div class="casAuthors">Lewis, Joan S.; Meeke, Kathleen; Osipo, Clodia; Ross, Eric A.; Kidawi, Noman; Li, Tianyu; Bell, Eric; Chandel, Navdeep S.; Jordan, V. Craig</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the National Cancer Institute</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">97</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">1746-1759</span>CODEN:
                <span class="NLM_cas:coden">JNCIEQ</span>;
        ISSN:<span class="NLM_cas:issn">0027-8874</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Background: We previously developed an estrogen receptor (ER)-pos. breast cancer cell line (MCF-7:5C) that is resistant to long-term estrogen deprivation and undergoes rapid and complete apoptosis in the presence of physiol. concns. of 17β-estradiol.  Here, we investigated the role of the mitochondrial apoptotic pathway in this process.  Methods: Apoptosis in MCF-7:5C cells treated with estradiol, fulvestrant, or vehicle (control) was investigated by annexin V-propidium iodide double staining and 4',6-diamidino-2-phenylindole (DAPI) staining.  Apoptosis was also analyzed in MCF-7:5C cells transiently transfected with small interfering RNAs (siRNAs) to apoptotic pathway components.  Expression of apoptotic pathway intermediates was measured by western blot anal.  Mitochondrial transmembrane potential (ψm) was detd. by rhodamine-123 retention assay.  Mitochondrial pathway activity was detd. by cytochrome c release and cleavage of poly(ADP-ribose) polymerase (PARP) protein.  Tumorigenesis was studied in ovariectomized athymic mice that were injected with MCF-7:5C cells.  Differences between the treatment groups and control group were detd. by two-sample t test or one-factor anal. of variance.  All statistical tests were two-sided.  Results: MCF-7:5C cells treated with estradiol underwent apoptosis and showed increased expression of proapoptotic proteins, decreased ψm, enhanced cytochrome c release, and PARP cleavage compared with cells treated with fulvestrant or vehicle.  Blockade of Bax, Bim, and p53 mRNA expression by siRNA reduced estradiol-induced apoptosis relative to control by 76% [95% confidence interval (CI) = 73% to 79%, P<.001], 85% [95% CI = 90% to 80%, P<.001], and 40% [95% CI = 45% to 35%, P<.001], resp., whereas blockade of FasL by siRNA had no effect.  Estradiol caused complete regression of MCF-7:5C tumors in vivo.  Conclusion: The mitochondrial pathway of apoptosis plays a crit. role in estradiol-induced apoptosis in long-term estrogen-deprived breast cancer cells.  Physiol. concns. of estradiol could potentially be used to induce apoptosis and tumor regression in tumors that have developed resistance to aromatase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0fGAoI50tabVg90H21EOLACvtfcHk0lgDy08m59BgqQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlSjurnN&md5=8107a7d125731549936a7268fb9e1138</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1093%2Fjnci%2Fdji400&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjnci%252Fdji400%26sid%3Dliteratum%253Aachs%26aulast%3DLewis%26aufirst%3DJ.%2BS.%26aulast%3DMeeke%26aufirst%3DK.%26aulast%3DOsipo%26aufirst%3DC.%26aulast%3DRoss%26aufirst%3DE.%2BA.%26aulast%3DKidawi%26aufirst%3DN.%26aulast%3DLi%26aufirst%3DT.%26aulast%3DBell%26aufirst%3DE.%26aulast%3DChandel%26aufirst%3DN.%2BS.%26aulast%3DJordan%26aufirst%3DV.%2BC.%26atitle%3DIntrinsic%2520mechanism%2520of%2520estradiol-induced%2520apoptosis%2520in%2520breast%2520cancer%2520cells%2520resistant%2520to%2520estrogen%2520deprivation%26jtitle%3DJ.%2520Natl.%2520Cancer%2520Inst.%26date%3D2005%26volume%3D97%26spage%3D1746%26epage%3D1759%26doi%3D10.1093%2Fjnci%2Fdji400" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref73"><div class="reference"><strong class="refLabel"><a href="#ref73" class="rightTabRefNumLink">73</a></strong><div class="NLM_citation" id="rightTab-cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osipo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meeke, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jordan, V. C.</span></span> <span> </span><span class="NLM_article-title">Estrogen-induced apoptosis in a breast cancer model resistant to long-term estrogen withdrawal</span>. <i>J. Steroid Biochem. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>94</i></span>,  <span class="NLM_fpage">131</span>– <span class="NLM_lpage">141</span>, <span class="refDoi"> DOI: 10.1016/j.jsbmb.2004.12.032</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1016%2Fj.jsbmb.2004.12.032" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=15862958" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BD2MXjvVKks74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=2005&pages=131-141&author=J.+S.+Lewisauthor=C.+Osipoauthor=K.+Meekeauthor=V.+C.+Jordan&title=Estrogen-induced+apoptosis+in+a+breast+cancer+model+resistant+to+long-term+estrogen+withdrawal&doi=10.1016%2Fj.jsbmb.2004.12.032"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">Estrogen-induced apoptosis in a breast cancer model resistant to long-term estrogen withdrawal</span></div><div class="casAuthors">Lewis, J. S.; Osipo, C.; Meeke, K.; Jordan, V. C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Steroid Biochemistry and Molecular Biology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">94</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">131-141</span>CODEN:
                <span class="NLM_cas:coden">JSBBEZ</span>;
        ISSN:<span class="NLM_cas:issn">0960-0760</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Estrogen suppression through the use of an aromatase inhibitor is an effective endocrine treatment option for postmenopausal breast cancer patients with estrogen receptor (ER)-pos. disease, however, there are concerns that long-term estrogen deprivation will inevitably lead to resistance.  To address the issue of acquired resistance to long-term estrogen deprivation our lab. has developed an ER+/PR- hormone-independent breast cancer cell line, MCF-7:5C which is a variant clone of wild-type MCF-7 cells.  Originally, these cells were cultured in estrogen-free MEM contg. 5% charcoal-stripped calf serum and were found to be resistant to both estradiol (E2) and antiestrogens.  Interestingly, a completely different phenomenon was obsd. when MCF-7:5C cells were cultured in phenol red-free RPMI 1640 medium contg. 10% charcoal-stripped fetal bovine serum (SFS).  Using DNA quantitation assays, the authors examd. the effect of E2 on the growth of MCF-7:5C cells under different media conditions.  These results showed that 10-9 M E2 caused a dramatic 90% redn. in the growth of MCF-7:5C cells cultured in RPMI medium contg. 10% SFS but did not have any significant inhibitory effects on cells cultured in MEM media.  Addnl. expts. were performed to det. whether the medium or the serum facilitated the inhibitory effects of E2 and the results indicated that it was the serum.  Annexin V and DAPI staining confirmed that the E2-induced growth inhibition of MCF-7:5C cells was due to apoptosis.  We also examd. the tumorigenic potential of MCF-7:5C cells by injecting 1 × 107 cells/site into ovariectomized athymic mice and found that these cells, previously cultured in RPMI media, spontaneously grew into tumors in the absence of E2.  Overall, these results show that low concns. (>10-11 M) of E2 are capable of inducing apoptosis in an aromatase resistant breast cancer cell model and that this effect is highly influenced by the medium in which the cells are grown.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhS-VZhkUY2rVg90H21EOLACvtfcHk0lgDy08m59BgqQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXjvVKks74%253D&md5=5b11562cc6ecd12fc4056f8446f9f47f</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1016%2Fj.jsbmb.2004.12.032&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jsbmb.2004.12.032%26sid%3Dliteratum%253Aachs%26aulast%3DLewis%26aufirst%3DJ.%2BS.%26aulast%3DOsipo%26aufirst%3DC.%26aulast%3DMeeke%26aufirst%3DK.%26aulast%3DJordan%26aufirst%3DV.%2BC.%26atitle%3DEstrogen-induced%2520apoptosis%2520in%2520a%2520breast%2520cancer%2520model%2520resistant%2520to%2520long-term%2520estrogen%2520withdrawal%26jtitle%3DJ.%2520Steroid%2520Biochem.%2520Mol.%2520Biol.%26date%3D2005%26volume%3D94%26spage%3D131%26epage%3D141%26doi%3D10.1016%2Fj.jsbmb.2004.12.032" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref74"><div class="reference"><strong class="refLabel"><a href="#ref74" class="rightTabRefNumLink">74</a></strong><div class="NLM_citation" id="rightTab-cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Livezey, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shapiro, D. J.</span></span> <span> </span><span class="NLM_article-title">Antiestrogen Resistant Cell Lines Expressing Estrogen Receptor α Mutations Upregulate the Unfolded Protein Response and are Killed by BHPI</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">34753</span>, <span class="refDoi"> DOI: 10.1038/srep34753</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1038%2Fsrep34753" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=27713477" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1Kmu7zF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&pages=34753&author=C.+Maoauthor=M.+Livezeyauthor=J.+E.+Kimauthor=D.+J.+Shapiro&title=Antiestrogen+Resistant+Cell+Lines+Expressing+Estrogen+Receptor+%CE%B1+Mutations+Upregulate+the+Unfolded+Protein+Response+and+are+Killed+by+BHPI&doi=10.1038%2Fsrep34753"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">Antiestrogen Resistant Cell Lines Expressing Estrogen Receptor α Mutations Upregulate the Unfolded Protein Response and are Killed by BHPI</span></div><div class="casAuthors">Mao, Chengjian; Livezey, Mara; Kim, Ji Eun; Shapiro, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">34753</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Outgrowth of metastases expressing ERα mutations Y537S and D538G is common after endocrine therapy for estrogen receptor α (ERα) pos. breast cancer.  The effect of replacing wild type ERα in breast cancer cells with these mutations was unclear.  We used the CRISPR-Cas9 genome editing system and homol. directed repair to isolate and characterize 14 T47D cell lines in which ERαY537S or ERαD538G replace one or both wild-type ERα genes.  In 2-dimensional, and in quant. anchorage-independent 3-dimensional cell culture, ERαY537S and ERαD538G cells exhibited estrogen-independent growth.  A progestin further increased their already substantial proliferation in micromolar 4-hydroxytamoxifen and fulvestrant/ICI 182,780 (ICI).  Our recently described ERα biomodulator, BHPI, which hyperactivates the unfolded protein response (UPR), completely blocked proliferation.  In ERαY537S and ERαD538G cells, estrogen-ERα target genes were constitutively active and partially antiestrogen resistant.  The UPR marker sp-XBP1 was constitutively activated in ERαY537S cells and further induced by progesterone in both cell lines.  UPR-regulated genes assocd. with tamoxifen resistance, including the oncogenic chaperone BiP/GRP78, were upregulated.  ICI displayed a greater than 2 fold redn. in its ability to induce ERαY537S and ERαD538G degrdn.  Progestins, UPR activation and perhaps reduced ICI-stimulated ERα degrdn. likely contribute to antiestrogen resistance seen in ERαY537S and ERαD538G cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnk6TbVl3BX7Vg90H21EOLACvtfcHk0lgmAsKT33cHew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1Kmu7zF&md5=a823b3cbf233db08a026b76f615424e4</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1038%2Fsrep34753&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep34753%26sid%3Dliteratum%253Aachs%26aulast%3DMao%26aufirst%3DC.%26aulast%3DLivezey%26aufirst%3DM.%26aulast%3DKim%26aufirst%3DJ.%2BE.%26aulast%3DShapiro%26aufirst%3DD.%2BJ.%26atitle%3DAntiestrogen%2520Resistant%2520Cell%2520Lines%2520Expressing%2520Estrogen%2520Receptor%2520%25CE%25B1%2520Mutations%2520Upregulate%2520the%2520Unfolded%2520Protein%2520Response%2520and%2520are%2520Killed%2520by%2520BHPI%26jtitle%3DSci.%2520Rep.%26date%3D2016%26volume%3D6%26spage%3D34753%26doi%3D10.1038%2Fsrep34753" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref75"><div class="reference"><strong class="refLabel"><a href="#ref75" class="rightTabRefNumLink">75</a></strong><div class="NLM_citation" id="rightTab-cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">Integration, scaling, space-group assignment and post-refinement</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">133</span>– <span class="NLM_lpage">144</span>, <span class="refDoi"> DOI: 10.1107/S0907444909047374</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1107%2FS0907444909047374" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=20124693" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=133-144&author=W.+Kabsch&title=Integration%2C+scaling%2C+space-group+assignment+and+post-refinement&doi=10.1107%2FS0907444909047374"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">Integration, scaling, space-group assignment and post-refinement</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">133-144</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Important steps in the processing of rotation data are described that are common to most software packages.  These programs differ in the details and in the methods implemented to carry out the tasks.  Here, the working principles underlying the data-redn. package XDS are explained, including the new features of automatic detn. of spot size and reflecting range, recognition and assignment of crystal symmetry and a highly efficient algorithm for the detn. of correction/scaling factors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGW7t3Bb1P_rVg90H21EOLACvtfcHk0lgmAsKT33cHew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisb4%253D&md5=eb1db34f3c959b4921771eb9f57b2159</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047374&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047374%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DIntegration%252C%2520scaling%252C%2520space-group%2520assignment%2520and%2520post-refinement%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D133%26epage%3D144%26doi%3D10.1107%2FS0907444909047374" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref76"><div class="reference"><strong class="refLabel"><a href="#ref76" class="rightTabRefNumLink">76</a></strong><div class="NLM_citation" id="rightTab-cit76"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teplyakov, A.</span></span> <span> </span><span class="NLM_article-title">Molecular replacement with MOLREP</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">22</span>– <span class="NLM_lpage">25</span>, <span class="refDoi"> DOI: 10.1107/S0907444909042589</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1107%2FS0907444909042589" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=20057045" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC3cXit1Kktw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=22-25&author=A.+Vaginauthor=A.+Teplyakov&title=Molecular+replacement+with+MOLREP&doi=10.1107%2FS0907444909042589"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular replacement with MOLREP</span></div><div class="casAuthors">Vagin, Alexei; Teplyakov, Alexei</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">22-25</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">MOLREP is an automated program for mol. replacement that utilizes a no. of original approaches to rotational and translational search and data prepn.  Since the first publication describing the program, MOLREP has acquired a variety of features that include weighting of the X-ray data and search models, multi-copy search, fitting the model into electron d., structural superposition of two models and rigid-body refinement.  The program can run in a fully automatic mode using optimized parameters calcd. from the input data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzj5SzYvyu_7Vg90H21EOLACvtfcHk0lgmAsKT33cHew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXit1Kktw%253D%253D&md5=820d114719aca209994ffb0403e3b20d</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1107%2FS0907444909042589&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909042589%26sid%3Dliteratum%253Aachs%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DTeplyakov%26aufirst%3DA.%26atitle%3DMolecular%2520replacement%2520with%2520MOLREP%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D22%26epage%3D25%26doi%3D10.1107%2FS0907444909042589" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref77"><div class="reference"><strong class="refLabel"><a href="#ref77" class="rightTabRefNumLink">77</a></strong><div class="NLM_citation" id="rightTab-cit77"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Potterton, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Briggs, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turkenburg, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E.</span></span> <span> </span><span class="NLM_article-title">A graphical user interface to the CCP4 program suite</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1131</span>– <span class="NLM_lpage">1137</span>, <span class="refDoi"> DOI: 10.1107/S0907444903008126</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1107%2FS0907444903008126" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=12832755" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BD3sXkvFSksLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2003&pages=1131-1137&author=E.+Pottertonauthor=P.+Briggsauthor=M.+Turkenburgauthor=E.+Dodson&title=A+graphical+user+interface+to+the+CCP4+program+suite&doi=10.1107%2FS0907444903008126"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">A graphical user interface to the CCP4 program suite</span></div><div class="casAuthors">Potterton, Elizabeth; Briggs, Peter; Turkenburg, Maria; Dodson, Eleanor</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">D59</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1131-1137</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4i is a graphical user interface that makes running programs from the CCP4 suite simpler and quicker.  It is particularly directed at inexperienced users and tightly linked to introductory and scientific documentation.  It also provides a simple project-management system and visualization tools.  The system is readily extensible and not specific to CCP4 software.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDH-BQSvi5qbVg90H21EOLACvtfcHk0lgmAsKT33cHew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXkvFSksLs%253D&md5=25a0168dd7ea9f930036453a829e1e3d</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1107%2FS0907444903008126&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444903008126%26sid%3Dliteratum%253Aachs%26aulast%3DPotterton%26aufirst%3DE.%26aulast%3DBriggs%26aufirst%3DP.%26aulast%3DTurkenburg%26aufirst%3DM.%26aulast%3DDodson%26aufirst%3DE.%26atitle%3DA%2520graphical%2520user%2520interface%2520to%2520the%2520CCP4%2520program%2520suite%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2003%26volume%3D59%26spage%3D1131%26epage%3D1137%26doi%3D10.1107%2FS0907444903008126" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref78"><div class="reference"><strong class="refLabel"><a href="#ref78" class="rightTabRefNumLink">78</a></strong><div class="NLM_citation" id="rightTab-cit78"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span> <span> </span><span class="NLM_article-title">Refinement of Macromolecular Structures by the Maximum-Likelihood Method</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">240</span>– <span class="NLM_lpage">255</span>, <span class="refDoi"> DOI: 10.1107/S0907444996012255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-255&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+Macromolecular+Structures+by+the+Maximum-Likelihood+Method&doi=10.1107%2FS0907444996012255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. σA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial α-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0li02XcYAWrdgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520Macromolecular%2520Structures%2520by%2520the%2520Maximum-Likelihood%2520Method%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1997%26volume%3D53%26spage%3D240%26epage%3D255%26doi%3D10.1107%2FS0907444996012255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref79"><div class="reference"><strong class="refLabel"><a href="#ref79" class="rightTabRefNumLink">79</a></strong><div class="NLM_citation" id="rightTab-cit79"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and development of Coot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0li02XcYAWrdgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref80"><div class="reference"><strong class="refLabel"><a href="#ref80" class="rightTabRefNumLink">80</a></strong><div class="NLM_citation" id="rightTab-cit80"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chisamore, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahmed, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bentrem, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jordan, V. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tonetti, D. A.</span></span> <span> </span><span class="NLM_article-title">Novel antitumor effect of estradiol in athymic mice injected with a T47D breast cancer cell line overexpressing protein kinase Calpha</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">3156</span>– <span class="NLM_lpage">3165</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=11595710" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BD3MXnvF2rsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2001&pages=3156-3165&author=M.+J.+Chisamoreauthor=Y.+Ahmedauthor=D.+J.+Bentremauthor=V.+C.+Jordanauthor=D.+A.+Tonetti&title=Novel+antitumor+effect+of+estradiol+in+athymic+mice+injected+with+a+T47D+breast+cancer+cell+line+overexpressing+protein+kinase+Calpha"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">Novel antitumor effect of estradiol in athymic mice injected with a T47D breast cancer cell line overexpressing protein kinase cα</span></div><div class="casAuthors">Chisamore, Michael J.; Ahmed, Yasmin; Bentrem, David J.; Jordan, V. Craig; Tonetti, Debra A.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3156-3165</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Resistance to tamoxifen (TAM) represents a significant challenge to the management of breast cancer.  We previously reported that the estrogen receptor (ER)-neg. hormone-independent T47D:C42 cell line has both elevated protein kinase Cα (PKCα) protein expression and basal activator protein-1 activity compared with the parental ER+ (hormone-dependent) T47D:A18 cell line.  Stable transfection of PKCα to the T47D:A18 breast cancer cell line results in increased basal activator protein-1 activity, reduced ER function, increased proliferation rate, and hormone-independent growth (Tonetti et al., Br. J. Cancer, 83: 782-791, 2000).  In this report, we further characterize the role of PKCα overexpression in vivo to elucidate a possible mol. mechanism of tamoxifen resistance.  To det. whether the T47D:A18/PKCα cell line would produce hormone-independent tumors in athymic mice, we injected T47D:A18, T47D:A18/neo, or the T47D:A18/PKCα20 cell clones bilaterally into the mammary fat pads of athymic mice.  Tumor growth was evaluated following treatment with estradiol (E2), TAM, and the pure antiestrogen, ICI 182780.  Mice receiving either T47D:A18 or T47D:A18/neo cells produced tumors that grew in response to E2 treatment, whereas the untreated control and TAM-treated groups showed no tumor growth.  Interestingly, mice receiving the T47D:A18/PKCα20 clone produced tumors in both the control and TAM groups, whereas tumor growth was inhibited in mice treated with E2.  PKCα was also overexpressed in an MCF-7 tumor model that also exhibited TAM-stimulated and E2-induced regression.  These results suggest that overexpression of PKCα in breast tumors results in hormone-independent tumor growth that cannot be inhibited by TAM treatment.  Furthermore, the finding that E2 has an antitumor effect on breast tumors overexpressing PKCα is a novel observation that may have important therapeutic implications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4Yd2zilR7XbVg90H21EOLACvtfcHk0li02XcYAWrdgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXnvF2rsr0%253D&md5=e132ea7de154480f42b7f701b796401c</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DChisamore%26aufirst%3DM.%2BJ.%26aulast%3DAhmed%26aufirst%3DY.%26aulast%3DBentrem%26aufirst%3DD.%2BJ.%26aulast%3DJordan%26aufirst%3DV.%2BC.%26aulast%3DTonetti%26aufirst%3DD.%2BA.%26atitle%3DNovel%2520antitumor%2520effect%2520of%2520estradiol%2520in%2520athymic%2520mice%2520injected%2520with%2520a%2520T47D%2520breast%2520cancer%2520cell%2520line%2520overexpressing%2520protein%2520kinase%2520Calpha%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2001%26volume%3D7%26spage%3D3156%26epage%3D3165" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref81"><div class="reference"><strong class="refLabel"><a href="#ref81" class="rightTabRefNumLink">81</a></strong><div class="NLM_citation" id="rightTab-cit81"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Perez
White, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molloy, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tonetti, D. A.</span></span> <span> </span><span class="NLM_article-title">RETRACTED ARTICLE: Extranuclear ERalpha is associated with regression of T47D PKCalpha-overexpressing, tamoxifen-resistant breast cancer</span>. <i>Mol. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">34</span>, <span class="refDoi"> DOI: 10.1186/1476-4598-12-34</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1186%2F1476-4598-12-34" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=23634843" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpt12qt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2013&pages=34&author=B.+Perez%0AWhiteauthor=M.+E.+Molloyauthor=H.+Zhaoauthor=Y.+Zhangauthor=D.+A.+Tonetti&title=RETRACTED+ARTICLE%3A+Extranuclear+ERalpha+is+associated+with+regression+of+T47D+PKCalpha-overexpressing%2C+tamoxifen-resistant+breast+cancer&doi=10.1186%2F1476-4598-12-34"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">Extranuclear ERα is associated with regression of T47D PKCα-overexpressing, tamoxifen-resistant breast cancer</span></div><div class="casAuthors">Perez White, Bethany; Molloy, Mary Ellen; Zhao, Huiping; Zhang, Yiyun; Tonetti, Debra A.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">34</span>CODEN:
                <span class="NLM_cas:coden">MCOACG</span>;
        ISSN:<span class="NLM_cas:issn">1476-4598</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Background: Prior to the introduction of tamoxifen, high dose estradiol was used to treat breast cancer patients with similar efficacy as tamoxifen, albeit with some undesirable side effects.  There is renewed interest to utilize estradiol to treat endocrine resistant breast cancers, esp. since findings from several preclin. models and clin. trials indicate that estradiol may be a rational second-line therapy in patients exhibiting resistance to tamoxifen and/or aromatase inhibitors.  We and others reported that breast cancer patients bearing protein kinase C alpha (PKCα)-expressing tumors exhibit endocrine resistance and tumor aggressiveness.  Our T47D:A18/PKCα preclin. model is tamoxifen-resistant, hormone-independent, yet is inhibited by 17β-estradiol (E2) in vivo.  We previously reported that E2-induced T47D:A18/PKCα tumor regression requires extranuclear ERα and interaction with the extracellular matrix.  Methods: T47D:A18/PKCα cells were grown in vitro using two-dimensional (2D) cell culture, three-dimensional (3D) Matrigel and in vivo by establishing xenografts in athymic mice.  Immunofluoresence confocal microscopy and co-localization were applied to det. estrogen receptor alpha (ERα) subcellular localization.  Co-immunopptn. and western blot were used to examine interaction of ERα with caveolin-1.  Results: We report that although T47D:A18/PKCα cells are cross-resistant to raloxifene in cell culture and in Matrigel, raloxifene induces regression of tamoxifen-resistant tumors.  ERα rapidly translocates to extranuclear sites during T47D:A18/PKCα tumor regression in response to both raloxifene and E2, whereas ERα is primarily localized in the nucleus in proliferating tumors.  E2 treatment induced complete tumor regression whereas cessation of raloxifene treatment resulted in tumor regrowth accompanied by re-localization of ERα to the nucleus.  T47D:A18/neo tumors that do not overexpress PKCα maintain ERα in the nucleus during tamoxifen-mediated regression.  An assocn. between ERα and caveolin-1 increases in tumors regressing in response to E2.  Conclusions: Extranuclear ERα plays a role in the regression of PKCα-overexpressing tamoxifen-resistant tumors.  These studies underline the unique role of extranuclear ERα in E2- and raloxifene-induced tumor regression that may have implications for treatment of endocrine-resistant PKCα-expressing tumors encountered in the clinic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3dl-xeGYbWbVg90H21EOLACvtfcHk0li02XcYAWrdgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXpt12qt7w%253D&md5=cc902496a93ae81c99847d5f16b89512</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1186%2F1476-4598-12-34&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1476-4598-12-34%26sid%3Dliteratum%253Aachs%26aulast%3DPerez%2BWhite%26aufirst%3DB.%26aulast%3DMolloy%26aufirst%3DM.%2BE.%26aulast%3DZhao%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DTonetti%26aufirst%3DD.%2BA.%26atitle%3DRETRACTED%2520ARTICLE%253A%2520Extranuclear%2520ERalpha%2520is%2520associated%2520with%2520regression%2520of%2520T47D%2520PKCalpha-overexpressing%252C%2520tamoxifen-resistant%2520breast%2520cancer%26jtitle%3DMol.%2520Cancer%26date%3D2013%26volume%3D12%26spage%3D34%26doi%3D10.1186%2F1476-4598-12-34" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref82"><div class="reference"><strong class="refLabel"><a href="#ref82" class="rightTabRefNumLink">82</a></strong><div class="NLM_citation" id="rightTab-cit82"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Molloy, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gherezghiher, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michalsen, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maximov, P. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jordan, V. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thatcher, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tonetti, D. A.</span></span> <span> </span><span class="NLM_article-title">Novel selective estrogen mimics for the treatment of tamoxifen-resistant breast cancer</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">2515</span>– <span class="NLM_lpage">2526</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.MCT-14-0319</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=10.1158%2F1535-7163.MCT-14-0319" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=25205655" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVGju73N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=2515-2526&author=M.+E.+Molloyauthor=B.+E.+Whiteauthor=T.+Gherezghiherauthor=B.+T.+Michalsenauthor=R.+Xiongauthor=H.+Patelauthor=H.+Zhaoauthor=P.+Y.+Maximovauthor=V.+C.+Jordanauthor=G.+R.+Thatcherauthor=D.+A.+Tonetti&title=Novel+selective+estrogen+mimics+for+the+treatment+of+tamoxifen-resistant+breast+cancer&doi=10.1158%2F1535-7163.MCT-14-0319"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit82R"><div class="casContent"><span class="casTitleNuber">82</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Selective Estrogen Mimics for the Treatment of Tamoxifen-Resistant Breast Cancer</span></div><div class="casAuthors">Molloy, Mary Ellen; White, Bethany E. Perez; Gherezghiher, Teshome; Michalsen, Bradley T.; Xiong, Rui; Patel, Hitisha; Zhao, Huiping; Maximov, Philipp Y.; Jordan, V. Craig; Thatcher, Gregory R. J.; Tonetti, Debra A.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2515-2526</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Endocrine-resistant breast cancer is a major clin. obstacle.  The use of 17β-estradiol (E2) has reemerged as a potential treatment option following exhaustive use of tamoxifen or aromatase inhibitors, although side effects have hindered its clin. usage.  Protein kinase C alpha (PKCα) expression was shown to be a predictor of disease outcome for patients receiving endocrine therapy and may predict a pos. response to an estrogenic treatment.  Here, we have investigated the use of novel benzothiophene selective estrogen mimics (SEM) as an alternative to E2 for the treatment of tamoxifen-resistant breast cancer.  Following in vitro characterization of SEMs, a panel of clin. relevant PKCα-expressing, tamoxifen-resistant models were used to investigate the antitumor effects of these compds.  SEM treatment resulted in growth inhibition and apoptosis of tamoxifen-resistant cell lines in vitro.  In vivo SEM treatment induced tumor regression of tamoxifen-resistant T47D:A18/PKCα and T47D:A18-TAM1 tumor models.  T47D:A18/PKCα tumor regression was accompanied by translocation of estrogen receptor (ER) α to extranuclear sites, possibly defining a mechanism through which these SEMs initiate tumor regression.  SEM treatment did not stimulate growth of E2-dependent T47D:A18/neo tumors.  In addn., unlike E2 or tamoxifen, treatment with SEMs did not stimulate uterine wt. gain.  These findings suggest the further development of SEMs as a feasible therapeutic strategy for the treatment of endocrine-resistant breast cancer without the side effects assocd. with E2.  Mol Cancer Ther; 13(11); 2515-26. cpr2014 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp64x3RCcDeYbVg90H21EOLACvtfcHk0lgIR4XhM8KhPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVGju73N&md5=203055e50477bcb1c98ea4c7844cf4b9</span></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-14-0319&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-14-0319%26sid%3Dliteratum%253Aachs%26aulast%3DMolloy%26aufirst%3DM.%2BE.%26aulast%3DWhite%26aufirst%3DB.%2BE.%26aulast%3DGherezghiher%26aufirst%3DT.%26aulast%3DMichalsen%26aufirst%3DB.%2BT.%26aulast%3DXiong%26aufirst%3DR.%26aulast%3DPatel%26aufirst%3DH.%26aulast%3DZhao%26aufirst%3DH.%26aulast%3DMaximov%26aufirst%3DP.%2BY.%26aulast%3DJordan%26aufirst%3DV.%2BC.%26aulast%3DThatcher%26aufirst%3DG.%2BR.%26aulast%3DTonetti%26aufirst%3DD.%2BA.%26atitle%3DNovel%2520selective%2520estrogen%2520mimics%2520for%2520the%2520treatment%2520of%2520tamoxifen-resistant%2520breast%2520cancer%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2014%26volume%3D13%26spage%3D2515%26epage%3D2526%26doi%3D10.1158%2F1535-7163.MCT-14-0319" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref83"><div class="reference"><strong class="refLabel"><a href="#ref83" class="rightTabRefNumLink">83</a></strong><div class="NLM_citation" id="rightTab-cit83"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Uenishi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishiwaki, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wakabayashi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oae, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsukube, H.</span></span> <span> </span><span class="NLM_article-title">Synthesis of ι-(bromomethyl)bipyridines and related ι-(bromomethyl)pyridinoheteroaromatics: useful functional tools for ligands in host molecules</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">4382</span>– <span class="NLM_lpage">4388</span>, <span class="refDoi"> DOI: 10.1021/jo00068a037</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo00068a037" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;key=1%3ACAS%3A528%3ADyaK3sXltl2jsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=1993&pages=4382-4388&author=J.+Uenishiauthor=T.+Tanakaauthor=K.+Nishiwakiauthor=S.+Wakabayashiauthor=S.+Oaeauthor=H.+Tsukube&title=Synthesis+of+%CE%B9-%28bromomethyl%29bipyridines+and+related+%CE%B9-%28bromomethyl%29pyridinoheteroaromatics%3A+useful+functional+tools+for+ligands+in+host+molecules&doi=10.1021%2Fjo00068a037"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit83R"><div class="casContent"><span class="casTitleNuber">83</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of ω-(bromomethyl)bipyridines and related ω-(bromomethyl)pyridinoheteroaromatics: useful functional tools for ligands in host molecules</span></div><div class="casAuthors">Uenishi, Junichi; Tanaka, Takakazu; Nishiwaki, Kenji; Wakabayashi, Shoji; Oae, Shigeru; Tsukube, Hiroshi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">4382-8</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    </div><div class="casAbstract">Pyridines and 2,2'-bipyridines have been employed as useful ligands in mol. recognition chemistries.  Halomethyl-substituted bipyridine or oligopyridine derivs. were required for the assembly of bipyridine or oligopyridine units with a supporting mother functional part in artificial biofunctional mols.  A series of ω-(bromomethyl)bipyridines and related ω-(bromomethyl)pyridinoheteroarom. compds., e.g., I, II, and III, were synthesized.  Prepn. of oligopyridines and pyridinoheteroarom. compds. have been carried out by either intermol. ligand coupling of alkyl heteroaryl sulfoxide with pyridyllithium or intramol. ligand coupling of pyridyl heteroaryl sulfoxide with methylmagnesium bromide for the type I compds.  The type II and III compds. were synthesized by addn. of pyridyllithium to pyridinecarboxaldehyde.  The ω-bromo group was introduced by radical bromination reaction of methylpyridyl group using N-bromosuccinimide and dibenzyl peroxide or bromination of ω-(hydroxymethyl)pyridine using a combination of CBr4 and Ph3P.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWejYii-fOv7Vg90H21EOLACvtfcHk0lgIR4XhM8KhPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXltl2jsb0%253D&md5=4ffdb2da0fd33506259b3b5ac86e2d15</span></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.1021%2Fjo00068a037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo00068a037%26sid%3Dliteratum%253Aachs%26aulast%3DUenishi%26aufirst%3DJ.%26aulast%3DTanaka%26aufirst%3DT.%26aulast%3DNishiwaki%26aufirst%3DK.%26aulast%3DWakabayashi%26aufirst%3DS.%26aulast%3DOae%26aufirst%3DS.%26aulast%3DTsukube%26aufirst%3DH.%26atitle%3DSynthesis%2520of%2520%25CE%25B9-%2528bromomethyl%2529bipyridines%2520and%2520related%2520%25CE%25B9-%2528bromomethyl%2529pyridinoheteroaromatics%253A%2520useful%2520functional%2520tools%2520for%2520ligands%2520in%2520host%2520molecules%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D1993%26volume%3D58%26spage%3D4382%26epage%3D4388%26doi%3D10.1021%2Fjo00068a037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ZYU" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ZYU','PDB','3ZYU'); return false;">PDB: 3ZYU</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6P05" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6P05','PDB','6P05'); return false;">PDB: 6P05</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3MXF" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3MXF','PDB','3MXF'); return false;">PDB: 3MXF</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i115"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00456">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_73315"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c00456?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00456</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Synthesis and characterization of all compounds and further experimental details (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00456/suppl_file/jm0c00456_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and some data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00456/suppl_file/jm0c00456_si_002.xlsx">XLSX</a>)</p></li><li><p class="inline">Coordinates information for structure representation (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00456/suppl_file/jm0c00456_si_003.pdb">PDB</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00456/suppl_file/jm0c00456_si_001.pdf">jm0c00456_si_001.pdf (1.19 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00456/suppl_file/jm0c00456_si_002.xlsx">jm0c00456_si_002.xlsx (11.33 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00456/suppl_file/jm0c00456_si_003.pdb">jm0c00456_si_003.pdb (6.08 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The X-ray coordinates have been deposited with the Protein Data Bank. The code is <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6P05">6P05</a> (BRD4 BD1 domain with <b>27</b>). Authors will release the atomic coordinates and experimental data upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c00456&amp;pbContext=%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-13%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c00456" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c00456" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                6MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679a47ad5d463cee","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
