
<html lang="en"     class="pb-page"  data-request-id="92de77c1-daa3-4bf3-a915-b299105fa1bb"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;issue:issue:10.1021/jmcmar.2016.59.issue-4;wgroup:string:ACHS website Group;website:website:acspubs;article:article:10.1021/acs.jmedchem.5b01527;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Docking and Linking of Fragments To Discover Jumonji Histone Demethylase Inhibitors" /></meta><meta name="dc.Creator" content="Magdalena  Korczynska" /></meta><meta name="dc.Creator" content="Daniel D.  Le" /></meta><meta name="dc.Creator" content="Noah  Younger" /></meta><meta name="dc.Creator" content="Elisabet  Gregori-Puigjané" /></meta><meta name="dc.Creator" content="Anthony  Tumber" /></meta><meta name="dc.Creator" content="Tobias  Krojer" /></meta><meta name="dc.Creator" content="Srikannathasan  Velupillai" /></meta><meta name="dc.Creator" content="Carina  Gileadi" /></meta><meta name="dc.Creator" content="Radosław P.  Nowak" /></meta><meta name="dc.Creator" content="Eriko  Iwasa" /></meta><meta name="dc.Creator" content="Samuel B.  Pollock" /></meta><meta name="dc.Creator" content="Idelisse  Ortiz Torres" /></meta><meta name="dc.Creator" content="Udo  Oppermann" /></meta><meta name="dc.Creator" content="Brian K.  Shoichet" /></meta><meta name="dc.Creator" content="Danica Galonić  Fujimori" /></meta><meta name="dc.Description" content="Development of tool molecules that inhibit Jumonji demethylases allows for the investigation of cancer-associated transcription. While scaffolds such as 2,4-pyridinedicarboxylic acid (2,4-PDCA) are..." /></meta><meta name="Description" content="Development of tool molecules that inhibit Jumonji demethylases allows for the investigation of cancer-associated transcription. While scaffolds such as 2,4-pyridinedicarboxylic acid (2,4-PDCA) are..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="December 23, 2015" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.5b01527" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2015 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.5b01527" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.5b01527" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.5b01527" /></link>
        
    
    

<title>Docking and Linking of Fragments To Discover Jumonji Histone Demethylase Inhibitors | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.5b01527" /></meta><meta property="og:title" content="Docking and Linking of Fragments To Discover Jumonji Histone Demethylase Inhibitors" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/large/jm-2015-015277_0013.jpeg" /></meta><meta property="og:description" content="Development of tool molecules that inhibit Jumonji demethylases allows for the investigation of cancer-associated transcription. While scaffolds such as 2,4-pyridinedicarboxylic acid (2,4-PDCA) are potent inhibitors, they exhibit limited selectivity. To discover new inhibitors for the KDM4 demethylases, enzymes overexpressed in several cancers, we docked a library of 600 000 fragments into the high-resolution structure of KDM4A. Among the most interesting chemotypes were the 5-aminosalicylates, which docked in two distinct but overlapping orientations. Docking poses informed the design of covalently linked fragment compounds, which were further derivatized. This combined approach improved affinity by ∼3 log-orders to yield compound 35 (Ki = 43 nM). Several hybrid inhibitors were selective for KDM4C over the related enzymes FIH, KDM2A, and KDM6B while lacking selectivity against the KDM3 and KDM5 subfamilies. Cocrystal structures corroborated the docking predictions. This study extends the use of structure-based docking from fragment discovery to fragment linking optimization, yielding novel KDM4 inhibitors." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.5b01527"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.5b01527">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.5b01527&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.5b01527&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.5b01527&amp;href=/doi/10.1021/acs.jmedchem.5b01527" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2016</span><span class="cit-fg-volume">, 59</span><span class="cit-fg-issue">, 4</span><span class="cit-fg-pageRange">, 1580-1598</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/59/4" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.5b01511" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.5b01517" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Docking and Linking of Fragments To Discover Jumonji Histone Demethylase Inhibitors</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Magdalena++Korczynska">Magdalena Korczynska</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Daniel+D.++Le">Daniel D. Le</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Noah++Younger">Noah Younger</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Elisabet++Gregori-Puigjan%C3%A9">Elisabet Gregori-Puigjané</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Anthony++Tumber">Anthony Tumber</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>¶</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Tobias++Krojer">Tobias Krojer</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Srikannathasan++Velupillai">Srikannathasan Velupillai</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Carina++Gileadi">Carina Gileadi</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Rados%C5%82aw+P.++Nowak">Radosław P. Nowak</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Eriko++Iwasa">Eriko Iwasa</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Samuel+B.++Pollock">Samuel B. Pollock</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Idelisse++Ortiz+Torres">Idelisse Ortiz Torres</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Udo++Oppermann">Udo Oppermann</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>⊥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Brian+K.++Shoichet">Brian K. Shoichet</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Danica+Galoni%C4%87++Fujimori">Danica Galonić Fujimori</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-text"><sup>†</sup>Department of Pharmaceutical Chemistry, <sup>‡</sup>Chemistry and Chemical Biology Graduate Program, and <sup>§</sup>Department of Cellular and Molecular Pharmacology, University of California, San Francisco, Genentech Hall, 600 16th Street, MC2280, San Francisco, California 94158-2280, United States</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">∥</span> <span class="aff-text">Structural Genomics Consortium, University of Oxford, Oxford OX3 7DQ, U.K.</span></div><div class="aff-info" id="aff3"><span class="aff-symbol">⊥</span> <span class="aff-text">Botnar Research Center, University of Oxford, Oxford OX3 7LD, U.K.</span></div><div class="aff-info" id="aff4"><span class="aff-symbol">¶</span> <span class="aff-text">Nuffield Department of Clinical Medicine, Target Discovery Institute (TDI), University of Oxford, Oxford OX3 7BN, U.K.</span></div><div class="corresp-info"><strong>*</strong>U.D.: e-mail, <a href="/cdn-cgi/l/email-protection#5f2a3b3071302f2f3a2d323e31311f2c383c713027713e3c712a34"><span class="__cf_email__" data-cfemail="7500111a5b1a0505100718141b1b350612165b1a0d5b14165b001e">[email protected]</span></a>; phone, +44 (0)1865 227306.</div><div class="corresp-info"><strong>*</strong>B.K.S.: e-mail, <a href="/cdn-cgi/l/email-protection#9efcedf6f1f7fdf6fbeadef9f3fff7f2b0fdf1f3"><span class="__cf_email__" data-cfemail="cfadbca7a0a6aca7aabb8fa8a2aea6a3e1aca0a2">[email protected]</span></a>; phone, 415-514-4126.</div><div class="corresp-info"><strong>*</strong>D.G.F.: e-mail, <a href="/cdn-cgi/l/email-protection#bcf8ddd2d5dfdd92fac9d6d5d1d3ced5fcc9dfcfda92d9d8c9"><span class="__cf_email__" data-cfemail="f6b297989f9597d8b0839c9f9b99849fb683958590d8939283">[email protected]</span></a>; phone, 415-514-0147.</div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.5b01527&amp;href=/doi/10.1021%2Facs.jmedchem.5b01527" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2016</span></span><span class="cit-volume">, 59</span><span class="cit-issue">, 4</span><span class="cit-pageRange">, 1580–1598</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">December 23, 2015</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>30 September 2015</li><li><span class="item_label"><b>Published</b> online</span>23 December 2015</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 25 February 2016</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.5b01527" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.5b01527</a></div><div class="article_header-article-copyright"><strong>Copyright © 2015 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D1580%26pageCount%3D19%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DMagdalena%2BKorczynska%252C%2BDaniel%2BD.%2BLe%252C%2BNoah%2BYounger%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D59%26issueNum%3D4%26contentID%3Dacs.jmedchem.5b01527%26title%3DDocking%2Band%2BLinking%2Bof%2BFragments%2BTo%2BDiscover%2BJumonji%2BHistone%2BDemethylase%2BInhibitors%26numPages%3D19%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D1598%26publicationDate%3DFebruary%2B2016">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.5b01527"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">3320</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">32</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.5b01527" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Docking and Linking of Fragments To Discover Jumonji Histone Demethylase Inhibitors&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Magdalena&quot;,&quot;last_name&quot;:&quot;Korczynska&quot;},{&quot;first_name&quot;:&quot;Daniel&quot;,&quot;last_name&quot;:&quot;D. Le&quot;},{&quot;first_name&quot;:&quot;Noah&quot;,&quot;last_name&quot;:&quot;Younger&quot;},{&quot;first_name&quot;:&quot;Elisabet&quot;,&quot;last_name&quot;:&quot;Gregori-Puigjané&quot;},{&quot;first_name&quot;:&quot;Anthony&quot;,&quot;last_name&quot;:&quot;Tumber&quot;},{&quot;first_name&quot;:&quot;Tobias&quot;,&quot;last_name&quot;:&quot;Krojer&quot;},{&quot;first_name&quot;:&quot;Srikannathasan&quot;,&quot;last_name&quot;:&quot;Velupillai&quot;},{&quot;first_name&quot;:&quot;Carina&quot;,&quot;last_name&quot;:&quot;Gileadi&quot;},{&quot;first_name&quot;:&quot;Radosław&quot;,&quot;last_name&quot;:&quot;P. Nowak&quot;},{&quot;first_name&quot;:&quot;Eriko&quot;,&quot;last_name&quot;:&quot;Iwasa&quot;},{&quot;first_name&quot;:&quot;Samuel&quot;,&quot;last_name&quot;:&quot;B. Pollock&quot;},{&quot;first_name&quot;:&quot;Idelisse&quot;,&quot;last_name&quot;:&quot;Ortiz Torres&quot;},{&quot;first_name&quot;:&quot;Udo&quot;,&quot;last_name&quot;:&quot;Oppermann&quot;},{&quot;first_name&quot;:&quot;Brian&quot;,&quot;last_name&quot;:&quot;K. Shoichet&quot;},{&quot;first_name&quot;:&quot;Danica&quot;,&quot;last_name&quot;:&quot;Galonić Fujimori&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2015&quot;,&quot;month&quot;:&quot;12&quot;,&quot;day&quot;:&quot;23&quot;,&quot;issue&quot;:&quot;4&quot;,&quot;volume&quot;:&quot;59&quot;,&quot;pages&quot;:&quot;1580-1598&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.5b01527&quot;},&quot;abstract&quot;:&quot;Development of tool molecules that inhibit Jumonji demethylases allows for the investigation of cancer-associated transcription. While scaffolds such as 2,4-pyridinedicarboxylic acid (2,4-PDCA) are potent inhibitors, they exhibit limited selectivity. To discover new inhibitors for the KDM4 demethylases, enzymes overexpressed in several cancers, we docked a library of 600 000 fragments into the high-resolution structure of KDM4A. Among the most interesting chemotypes were the 5-aminosalicylates, which docked in two distinct but overlapping orientations. Docking poses informed the design of covalently linked fragment compounds, which were further derivatized. This combined approach improved affinity by ∼3 log-orders to yield compound 35 (Ki = 43 nM). Several hybrid inhibitors were selective for KDM4C over the related enzymes FIH, KDM2A, and KDM6B while lacking selectivity against the KDM3 and KDM5 subfamilies. Cocrystal structures corroborated the docking predictions. This study extends the use of structure-bas&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b01527&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01527" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b01527&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01527" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b01527&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01527" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.5b01527&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01527&amp;href=/doi/10.1021/acs.jmedchem.5b01527" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.5b01527" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.5b01527" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (8 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.5b01527&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.5b01527%26sid%3Dliteratum%253Aachs%26pmid%3D26699912%26genre%3Darticle%26aulast%3DKorczynska%26date%3D2016%26atitle%3DDocking%2Band%2BLinking%2Bof%2BFragments%2BTo%2BDiscover%2BJumonji%2BHistone%2BDemethylase%2BInhibitors%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D59%26issue%3D4%26spage%3D1580%26epage%3D1598%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291711" title="Selectivity">Selectivity</a>,</li><li><a href="/action/doSearch?ConceptID=292201" title="Scaffolds">Scaffolds</a>,</li><li><a href="/action/doSearch?ConceptID=291754" title="Molecules">Molecules</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/59/4" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/jmcmar.2016.59.issue-4/20160225/jmcmar.2016.59.issue-4.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/medium/jm-2015-015277_0013.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/large/jm-2015-015277_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01527&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Development of tool molecules that inhibit Jumonji demethylases allows for the investigation of cancer-associated transcription. While scaffolds such as 2,4-pyridinedicarboxylic acid (2,4-PDCA) are potent inhibitors, they exhibit limited selectivity. To discover new inhibitors for the KDM4 demethylases, enzymes overexpressed in several cancers, we docked a library of 600 000 fragments into the high-resolution structure of KDM4A. Among the most interesting chemotypes were the 5-aminosalicylates, which docked in two distinct but overlapping orientations. Docking poses informed the design of covalently linked fragment compounds, which were further derivatized. This combined approach improved affinity by ∼3 log-orders to yield compound <b>35</b> (<i>K</i><sub>i</sub> = 43 nM). Several hybrid inhibitors were selective for KDM4C over the related enzymes FIH, KDM2A, and KDM6B while lacking selectivity against the KDM3 and KDM5 subfamilies. Cocrystal structures corroborated the docking predictions. This study extends the use of structure-based docking from fragment discovery to fragment linking optimization, yielding novel KDM4 inhibitors.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="specialIssueNotice" class="extra-info-sec articleNote"><h4>SPECIAL ISSUE</h4><p class="last">This article is part of the <a href="/toc/jmcmar/59/4"><issue-title>Epigenetics</issue-title></a> special issue.</p></div><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_43582" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_43582" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Methylation of lysine residues in histone proteins profoundly impacts the regulation of cellular processes such as transcription, formation of heterochromatin, genomic imprinting, and X-chromosome inactivation. The discovery of lysine-specific demethylase 1 (LSD1)<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> and Jumonji C (JmjC) domain-containing histone lysine demethylases (KDMs)<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> uncovered the dynamic character of methyllysine modifications in regulating gene expression. The JmjC domain-containing protein family comprises 33 members in humans,<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> of which 24 are classified as histone lysine demethylases.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> In the context of lysine demethylation, these enzymes use iron(II), α-ketoglutarate (α-KG), and molecular oxygen to hydroxylate methyl groups attached to the ε-amino group of lysines. Subsequent decomposition of the hemiaminal intermediate releases formaldehyde and a demethylated residue.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> Accordingly, these enzymes are considered “erasers” of histone marks.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a></div><div class="NLM_p">Aberrant lysine methylation, caused by mutation or misregulation of histone demethylases and histone methyltransferases, profoundly impacts cell physiology. Of particular interest is the hyperactivity of the KDM4 enzyme subfamily (also known as JMJD2). This subfamily consists of five enzymes, KDM4A–E,<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> which erase methylation marks on several lysine residues in both histone and non-histone proteins (e.g., polycomb 2 protein (Pc2)).<a onclick="showRef(event, 'ref7 ref8 ref9 ref10 ref11 ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref7 ref8 ref9 ref10 ref11 ref12 ref13">(7-13)</a> High expression of individual isoforms of KDM4 is thought to promote oncogenesis in human tumors, especially in prostate cancer<a onclick="showRef(event, 'ref8 ref14'); return false;" href="javascript:void(0);" class="ref ref8 ref14">(8, 14)</a> but also in colon<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> and some types of breast<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> cancers.</div><div class="NLM_p">The availability of selective chemical probes against individual Jumonji C demethylase subfamilies is critical for exploring their physiological and pathological roles. Since the discovery of Jumonji demethylases in 2006, several inhibitor classes have been identified (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01527/suppl_file/jm5b01527_si_001.pdf" class="ext-link">Figure S1A</a>) (for reviews, see refs <a onclick="showRef(event, 'ref4 ref17 ref18 ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref4 ref17 ref18 ref19 ref20">4, 17−20</a>). These molecules are largely pan-Jumonji inhibitors, often with only modest selectivity among subfamilies and activity against related oxygen-sensing enzymes,<a onclick="showRef(event, 'ref21 ref22 ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref21 ref22 ref23 ref24">(-21-24)</a> such as factor-inhibiting hypoxia-inducible factor (FIH)<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> and prolyl hydroxylases.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> Despite recent advances in KDM inhibitor development, resulting in molecules such as 5-chloro-2-[(<i>E</i>)-2-[phenyl(pyridin-2-yl)methylidene]hydrazin-1-yl]pyridine (JIB-04)<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> and methylstat,<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> the challenge of identifying selective molecules remains.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> This is further highlighted by <i>N</i>-[2-(2-pyridinyl)-6-(1,2,4,5-tetrahydro-3<i>H</i>-3-benzazepin-3-yl)-4-pyrimidinyl]-β-alanine (GSK-J1),<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> a dual specificity inhibitor that potently inhibits both the KDM6 and KDM5 subfamilies.<a onclick="showRef(event, 'ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref28 ref29">(28, 29)</a> Among the only class of molecules with excellent selectivity are the 8-mer peptide substrate analogs,<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> but their peptidic nature limits their cellular permeability and utility as chemical probes. To develop new molecules, we employed a structure-based discovery approach that broadly samples chemical space as a method to identify novel and diverse chemotypes targeting the KDM4 demethylases.<a onclick="showRef(event, 'ref31 ref32 ref33 ref34 ref35 ref36 ref37'); return false;" href="javascript:void(0);" class="ref ref31 ref32 ref33 ref34 ref35 ref36 ref37">(31-37)</a></div><div class="NLM_p last">We began with molecular docking screens focused on a library of 600 000 commercially available fragments.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> This screen resulted in the identification of unique lead fragments that inhibited KDM4C with good ligand efficiency (LE) values. The docking poses informed subsequent fragment linking, leading to hybrid compounds with as much as 700-fold improvement in inhibition relative to the parent fragments. Further improvement in potency was achieved by iterative cycles of optimization through chemical synthesis, isozyme profiling, and docking. Importantly, these optimized compounds displayed excellent selectivity against FIH and had substantial selectivity over several other Jumonji demethylase subfamilies. Beyond the discovery of novel fragments, this work suggests that docking can inform the design of fused fragments. This docking approach and the corroborating X-ray structures provide atomic-resolution insight into the activity of this new family of demethylase inhibitors.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_17676" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_17676" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3"> Fragment Docking and Initial Testing</h3><div class="NLM_p">Molecular docking was performed on the model structure of KDM4A, since the residues lining the active site cavity are conserved between KDM4A and KDM4C, and extensive high-resolution structural data were available for KDM4A. The ZINC fragment library, containing over 600 000 molecules, was docked into the model active site using DOCK3.6. Compounds in this library share common features: commercial availability, xLogP value of ≤3.5, molecular weight of ≤250 Da, and fewer than five rotatable bonds. Each fragment was docked in an average of 21 857 orientations and 42 conformations, resulting in approximately 1.5 × 10<sup>12</sup> calculated complexes. Partial ligand desolvation was accounted for in the docking calculations using a context-dependent implementation of GB/SA solvation values from AMSOL.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a></div><div class="NLM_p">The docked fragments were scored for van der Waals interactions using the AMBER potential function<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> and for electrostatic complementarity using a point charge model with precalculated maps generated by Poisson–Boltzmann calculations using Qnifft.<a onclick="showRef(event, 'ref40 ref41'); return false;" href="javascript:void(0);" class="ref ref40 ref41">(40, 41)</a> Importantly, all candidate compounds selected were among the very top-ranking 0.1% of the docking-ranked library. The selection criteria following docking included the following: (1) formation of favorable interactions with the active site iron center, which was not an explicit parameter in the initial scoring function;<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> (2) identification of novel chemotypes; (3) elimination of poses with high internal energies, which are not always captured by Omega program calculations used to generate docking conformations.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> Such postscreening criteria are widely used for “hit-picking”, both in docking<a onclick="showRef(event, 'ref43 ref44'); return false;" href="javascript:void(0);" class="ref ref43 ref44">(43, 44)</a> and in high-throughput screening.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> On the basis of these criteria, 14 fragments were selected for testing in an antibody-based demethylation assay coupled with time-resolved FRET (TR-FRET) detection (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>), which revealed four compounds that inhibited KDM4C with an IC<sub>50</sub> below 100 μM (LE values 0.32–0.36, <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>, compounds <b>2</b>–<b>5</b>). Three other compounds exhibited IC<sub>50</sub> values in the 100–200 μM range (LE values of 0.3–0.41, <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>, compounds <b>6</b>–<b>8</b>). Finally, the remaining seven molecules had detectable inhibition, albeit above 200 μM (LE values of 0.2–0.31, <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>, compounds <b>9</b>–<b>15</b>). In this study, 50% of the tested candidate compounds demonstrated IC<sub>50</sub> values lower than 200 μM and LE values of 0.3 or higher.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Fragment Docking Hits Tested for Inhibition of KDM4C<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/medium/jm-2015-015277_0007.gif" alt="" id="fx48" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup>Table a</sup><p class="last">IC<sub>50</sub> values determined by TR-FRET assay with 2 μM α-KG. (∗) Control compound, 2,4-PDCA. (∗∗) Compound <b>3</b> is the enantiomer of compound <b>2</b> and has no docking rank associated with it.</p></div></div><div></div></div><div class="NLM_p">Among the eight fragments with the highest ligand efficiency, compounds <b>4</b>, <b>5</b>, <b>7</b> share a common 5-aminosalicylate scaffold (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>, <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>). The 5-aminosalicylate scaffold docking poses predicted coordination to the active site Fe(II) via the carboxylate and hydroxyl moieties. The 5-amino group was predicted not to participate in metal coordination but rather to form a hydrogen bond with Asp135 (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>). KDM4C inhibition analysis by TR-FRET yielded IC<sub>50</sub> values ranging from 58 to 165 μM for compounds with this scaffold (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>).</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/medium/jm-2015-015277_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/large/jm-2015-015277_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. 5-Aminosalicylate docking hits. Poses of the 5-aminosalicylates identified in the initial docking screens. Fragments <b>4</b>, <b>5</b>, and <b>7</b> are predicted to bind the metal in a bivalent fashion through the carboxylate and hydroxyl groups, while the amide proton is predicted to hydrogen-bond with Asp135 of KDM4A.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/large/jm-2015-015277_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01527&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Initial efforts to optimize the 5-aminosalicylate scaffold explored 80 analogs, which were either commercially available or de novo synthesized. Among these analogs, the smallest compound tested was <i>N</i>-acetyl-5-aminosalicylic acid, which showed no inhibition. However, 13 molecules (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01527/suppl_file/jm5b01527_si_001.pdf" class="ext-link">Table S2</a>) showed well-behaved dose–repose curves and a narrow confidence interval for IC<sub>50</sub> values (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01527/suppl_file/jm5b01527_si_001.pdf" class="ext-link">Table S5</a>). Importantly, the 5-aminosalicylate chemotype, represented by compound <b>20</b>, stabilized the enzyme in thermal shift assays and did not exhibit nonspecific inhibition (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01527/suppl_file/jm5b01527_si_001.pdf" class="ext-link">Figure S2, Table S1</a>). Compound <b>21</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01527/suppl_file/jm5b01527_si_001.pdf" class="ext-link">Table S2</a>) was the most potent molecule in this series with an IC<sub>50</sub> of 5.2 μM, showing a 10-fold increase in activity over starting fragment <b>4</b>. However, once we arrived at these low micromolar compounds, we could not optimize them any further. In general these new analogs showed relatively flat SAR, and no derivative emerged with affinity better than low μM.</div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Controls for Colloidal Aggregation</h3><div class="NLM_p last">Surprisingly, though the molecules developed and tested here were almost without exception polar and small (many of them fragments) a substantial number had steep concentration–response curves in inhibition assays, a harbinger for colloidal aggregation<a onclick="showRef(event, 'ref51 ref52'); return false;" href="javascript:void(0);" class="ref ref51 ref52">(46, 47)</a> that is precedented for fragments.<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(48)</a> For two of the initial docking hits, compounds <b>4</b> and <b>5</b>, we specifically controlled for this artifact by counterscreening them against the unrelated enzyme AmpC β-lactamase and testing them for particle formation by DLS; neither of the two inhibitors acted as aggregators in a relevant concentration range (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01527/suppl_file/jm5b01527_si_001.pdf" class="ext-link">Table S1</a>). However, of the 80 5-aminosalicylate inhibitors optimized, 18 had steep Hill slopes of above 3. For these compounds the molecular weight varied between 192 and 333 Da and the median xLogP was 2.39. Compounds that had unfavorable concentration–response curves, often a sign of artifactual activity, often by aggregation (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01527/suppl_file/jm5b01527_si_001.pdf" class="ext-link">Table S1C</a>), were not further pursued.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Optimization by Fragment Linking</h3><div class="NLM_p">As derivatization of the amino moiety of the 5-aminosalicylates scaffold did not substantially improve potency, we considered alternative approaches to improve ligand binding. Specifically, we observed that the 5-aminosalicylates could adopt two high-scoring docking poses in the active site, leading us to explore ligand optimization through fragment linking. While most highly ranked 5-aminosalicylates docked in a pose dominated by iron chelation via the carboxylate and alcohol moieties (compound <b>4</b>, <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>A), a subset of 5-aminosalicylates paired the carboxylate with Lys206 and Tyr132 in the α-KG binding site of the enzyme (e.g., compound <b>29</b>, <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>A). Intriguingly, on the basis of the docking poses, a linking strategy for the two fragments appeared feasible. Specifically, we envisioned that the hybrid molecule should retain the 1,5-bidentate coordination of iron, as in fragment <b>4</b>. In addition, the hybrid scaffold should contain a carboxylate group to engage Lys206-Tyr132, as present in fragment <b>29</b> (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>B). To evaluate if a hybrid molecule could exploit both sets of interactions, 12 scaffolds were created computationally (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01527/suppl_file/jm5b01527_si_001.pdf" class="ext-link">Table S3</a>). Each molecule incorporated a 1,5-bidentate Fe(II) ligand in addition to a carboxylate (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01527/suppl_file/jm5b01527_si_001.pdf" class="ext-link">Table S3</a>). Docking of these hybrid molecules revealed that scaffolds I, II, and III (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>C, <a href="/doi/suppl/10.1021/acs.jmedchem.5b01527/suppl_file/jm5b01527_si_001.pdf" class="ext-link">Table S3</a>, entries 1, 2, 3) had favorable binding geometries and minimal internal strain. Among these, the most appealing was scaffold II, where the iron coordination is achieved by phenol and pyridine moieties. The position of the pyridine ring in this scaffold, as predicted by docking, is reminiscent of 2,4-pyridinedicarboxylic acid (2,4-PDCA) in its cocrystal structure with KDM4A (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2VD7">2VD7</a>),<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> providing further encouragement for our fragment fusion strategy. Furthermore, this scaffold can be synthetically accessed by a cross-coupling reaction between two aromatic rings leading to the core compound <b>30</b> (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>) that can be further derivatized to explore the chemical space around the hybrid scaffold.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/medium/jm-2015-015277_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/large/jm-2015-015277_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Docking-based fragment linking. (A) Superimposition of the two major 5-aminosalicylate poses (compound <b>4</b>, orange; compound <b>29</b>, magenta). (B) Structures of 5-aminosalicylate fragments used to create hybrid scaffolds. Key interacting groups are highlighted with a box colored to match the fragments. Iron coordinating atoms are marked by asterisk, and possible ring positions are highlighted. (C) Among 12 virtual hybrid scaffolds that were docked (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01527/suppl_file/jm5b01527_si_001.pdf" class="ext-link">Table S3</a>), three had good scores and maintained the hypothesized network of interactions. (D) Overlay of a docking pose of <b>46</b> (orange) and a crystal structure of <b>30</b> (green), a representative hybrid scaffold synthesized in this study.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/large/jm-2015-015277_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01527&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Synthesis and Structure–Activity Relationships of the Acylated Hybrid Scaffolds<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/medium/jm-2015-015277_0008.gif" alt="" id="fx49" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/medium/jm-2015-015277_0009.gif" alt="" id="GRAPHIC-d1398e755-autogenerated" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup>Table a</sup><p class="last">Top: synthetic route to <i>N</i>-acyl hybrid derivatives: (a) CombiPhos-Pd6 mixed catalyst, Cs<sub>2</sub>CO<sub>3</sub>, 20:1 acetonitrile/water, 100 °C, 24 h; (b) NaI, HBr, 110 °C, 4 days; (c) typically, (i) NHS, R-COCl, MeCN, 20 °C, 1 h; (ii) compound <b>30</b>, 1:1:1 MeCN/H<sub>2</sub>O/MeOH, 20 °C, 16 h. Bottom: IC<sub>50</sub> values for selected hybrid derivatives tested for inhibition of KDM4C using TR-FRET with the indicated concentration of α-KG. ND = not determined, NI = not inhibitory.</p></div></div><div></div></div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Hybrid Scaffold Structure–Activity Relationship</h3><div class="NLM_p">The core biaryl hybrid scaffold was generated via Suzuki–Miyaura coupling followed by simultaneous nitro group reduction and deprotection of the methyl ester and methyl ether groups using HBr/NaI at reflux (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>). Acylation of the resultant amino group yielded the N-acylated hybrid analogs (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>). Several hybrid derivatives strongly inhibited KDM4C: compounds <b>35</b>, <b>36</b>, and <b>39</b> exhibited IC<sub>50</sub> values below 70 nM using the TR-FRET assay. A subset of the hybrid compounds was tested under stringent competition with 50 μM α-KG to differentiate relative IC<sub>50</sub> values among the most potent derivatives. Under these conditions, the tested molecules demonstrated a range of low micromolar values at half-maximal KDM4C inhibition (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>).</div><div class="NLM_p">The FDH-coupled assay was used for reciprocal (Lineweaver–Burk) plot analysis of selected potent acylated hybrid derivatives, <b>35</b> and <b>42</b>. The <i>K</i><sub>i</sub> constants were calculated based on the Morrison equation using the experimentally determined <i>K</i><sub>M</sub> of 5.1 μM for α-KG (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01527/suppl_file/jm5b01527_si_001.pdf" class="ext-link">Figure S3C</a>). The established nonselective α-KG competitive inhibitor, 2,4-PDCA (<b>1</b>), exhibited a characteristic competitive reciprocal plot profile and a <i>K</i><sub>i</sub> of 2 nM (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>C, <a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>). Compounds <b>35</b> and <b>42</b> displayed <i>K</i><sub>i</sub> values of 43 and 680 nM, respectively (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>, <a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>). Importantly, these inhibition constants allow for direct comparison of the compound potency irrespective of the assay conditions. While these two compounds bind competitively with respect to α-KG, they do not show competition with the peptide substrate (<a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>).</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/medium/jm-2015-015277_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/large/jm-2015-015277_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Lineweaver–Burke plots. Plots of 1/[α-KG] vs 1/<i>V</i><sub>i</sub> for a range of concentrations of (A) <b>42</b>, (B) <b>35</b>, and (C) <b>1</b>: ● = 19 μM inhibitor, ■ = 6 μM inhibitor, ◆ = 2 μM inhibitor, ▲ = 0.7 μM inhibitor, ▼ = 0 μM inhibitor.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/large/jm-2015-015277_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01527&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Competition Experiments Performed with a Range of α-KG and Peptide Concentrations<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/medium/jm-2015-015277_0010.gif" alt="" id="fx50" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup>Table a</sup><p class="last">Compounds <b>1</b>, <b>35</b>, and <b>42</b> were tested over a range of concentrations of α-KG and peptide to determine inhibitory constants (<i>K</i><sub>i</sub>) and competition with respect to both substrates. The <i>K</i><sub>M</sub> for H3K9me3 peptide is 76 μM,<a onclick="showRef(event, 'ref76'); return false;" href="javascript:void(0);" class="ref ref76">(76)</a> and the determined <i>K</i><sub>M</sub> for α-KG is 5.1 μM (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01527/suppl_file/jm5b01527_si_001.pdf" class="ext-link">Figure S3C</a>). Measurements were performed in triplicates.</p></div></div><div></div></div><div class="NLM_p">The potency improvement conferred by the hybrid scaffold over the lead 5-aminosalicylates is illustrated by direct comparison of the corresponding derivatized compounds (<a class="ref internalNav" href="#tbl4" aria-label="Table 4">Table 4</a>). Relative to the 5-aminosalicylate compounds, the potencies of the hybrid analogs improved substantially. The greatest increase in potency was observed for <b>42</b> vs <b>5</b> with IC<sub>50</sub> values of 0.14 and 98 μM, respectively (∼700-fold improvement, <a class="ref internalNav" href="#tbl4" aria-label="Table 4">Table 4</a>). Several other corresponding pairs of compounds, such as <b>4</b> vs <b>45</b> and <b>28</b> vs <b>46</b>, showed improvements in affinity. Naturally, there was an exception where the hybrid scaffold did not improve activity (<b>21</b> vs <b>47</b>, <a class="ref internalNav" href="#tbl4" aria-label="Table 4">Table 4</a>).</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Potency Comparison between 5-Aminosalicylates and the Corresponding Hybrid Molecules<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/medium/jm-2015-015277_0011.gif" alt="" id="fx51" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup>Table a</sup><p class="last">Selected 5-aminosalicylate and hybrid derivatives tested for inhibition of KDM4C using TR-FRET with 2 μM α-KG.</p></div></div><div></div></div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Selectivity Counterscreens</h3><div class="NLM_p">The in vitro selectivity profiles of derivatives <b>35</b>, <b>36</b>, <b>40</b>, <b>42</b>, and <b>44</b> (<a class="ref internalNav" href="#tbl5" aria-label="Table 5">Table 5</a>) were determined using representative members from distinct phylogenetic groups within the JmjC domain-containing enzyme family (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>A) and were screened using the MALDI or AlphaScreen assays. We tested each inhibitor against the H3K4 demethylases KDM5A (JARID1A, RBP2) and KDM5B (JARID1B, PLU-1), the H3K27 demethylase KDM6B (JMJD3), the H3K36me2 demethylase KDM2A (FBXL11), the H3K9me1/2 demethylase KDM3A (JMJD1A), and the asparagine hydroxylase FIH. While the parent hybrid scaffold <b>30</b> exhibited 6-fold selectivity for KDM4C over FIH (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>B), this selectivity was improved for all <i>N</i>-acyl derivatives, with compound <b>35</b> demonstrating >85-fold selectivity against FIH (IC<sub>50</sub> > 500 μM) (<a class="ref internalNav" href="#tbl5" aria-label="Table 5">Table 5</a>). Given the important physiological role of FIH in regulating broad transcriptional networks, the excellent selectivity of the hybrid analogs for KDM4C over FIH inhibition is critical. All tested derivatives exhibited 6-fold or greater selectivity for KDM4C vs the H3K36me2 demethylase KDM2A, with the best discrimination (24-fold, <a class="ref internalNav" href="#tbl5" aria-label="Table 5">Table 5</a>) achieved by <b>42</b>. Moreover, in this series compound <b>42</b> was also the most selective inhibitor of KDM4C over the H3K27 demethylase KDM6B (>26-fold). The tested hybrid compounds demonstrated poor discrimination of KDM4C over the H3K4 demethylases KDM5A and KDM5B and the H3K9me1/2 demethylase KDM3A. As expected, these molecules do not discriminate among the members of the KDM4 family (<a class="ref internalNav" href="#tbl5" aria-label="Table 5">Table 5</a>).</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/medium/jm-2015-015277_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/large/jm-2015-015277_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Jumonji family enzymes and hybrid compound selectivity against FIH. (A) Phylogenetic diagram based on the catalytic domains of JmjC proteins, with KDM4C highlighted in red, and enzymes used in the counterscreening experiments in this study highlighted in blue. (B) Selectivity profile obtained using the MALDI assay with 100 μM α-KG against KDM4C (red) and FIH (green) for hybrid scaffold <b>30</b> indicating an IC<sub>50</sub> of 135 and 828 μM, respectively, and (C) its acyl derivative <b>44</b> showing an IC<sub>50</sub> of 63 μM for KDM4C (red) and no inhibition for FIH (green).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/large/jm-2015-015277_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01527&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Selectivity Profile of Selected Inhibitors<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/medium/jm-2015-015277_0012.gif" alt="" id="fx52" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup>Table a</sup><p class="last">Selectivity of inhibitors for KDM4C compared to FIH and KDM5A was determined by MALDI assay with 100 μM α-KG. Selectivity against other JmjC domain containing proteins was determined by AlphaScreen, wherein α-KG concentrations used are at approximately the <i>K</i><sub>M</sub> for each of the enzymes. No inhibition was observed at maximum concentrations of compounds tested for each of the enzymes: (″) KDM2A, [inhibitor] = 100 μM; (∧) KDM6B, [inhibitor] = 100 μM; (∗) FIH, [inhibitor] = 500 μM. The reported IC<sub>50</sub> are therefore an underestimate. (∗∗) IC<sub>50</sub> value was inferred by extrapolating the IC<sub>50</sub> curve (see <a href="/doi/suppl/10.1021/acs.jmedchem.5b01527/suppl_file/jm5b01527_si_001.pdf" class="ext-link">Figure S3B</a>).</p></div></div><div></div></div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Crystallography</h3><div class="NLM_p">To investigate the agreement between docking predictions and experimentally observed orientations and to enable future elaboration of the hybrid scaffold, we turned to X-ray crystallography. Crystal structures of KDM4A, a representative KDM4 enzyme that is well suited for crystallographic studies,<a onclick="showRef(event, 'ref23 ref25 ref46 ref47 ref48'); return false;" href="javascript:void(0);" class="ref ref23 ref25 ref46 ref47 ref48">(23, 25, 49-51)</a> were determined in complex with hybrid compounds <b>30</b>, <b>35</b>, <b>36</b>, <b>40</b>, <b>42</b>, <b>43</b>, and <b>44</b> (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>A–G). The resolution of the structures ranged from 2.0 to 2.39 Å (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>A–G), and other statistics can be found in <a href="/doi/suppl/10.1021/acs.jmedchem.5b01527/suppl_file/jm5b01527_si_001.pdf" class="ext-link">Supporting Information Table S4</a>. Unambiguous positions for the ligands in the structures were identified in unbiased difference density maps (<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> for compound <b>36</b> contoured at 2.5σ (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>H) and contoured at 3σ (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01527/suppl_file/jm5b01527_si_001.pdf" class="ext-link">Figure S4G</a>)), and the inhibitors placement refined well (2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> map for compound <b>36</b>, contoured at 1σ (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01527/suppl_file/jm5b01527_si_001.pdf" class="ext-link">Figure S4G</a>, and composite omit map contoured at 1σ (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>I)).</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/medium/jm-2015-015277_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/large/jm-2015-015277_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Crystal structures of hybrid compounds in complex with KDM4A. Seven cocrystal structures were obtained with KDM4A and the hybrid molecules (A) <b>43</b>, (B) <b>30</b>, (C) <b>44</b>, (D) <b>42</b>, (E) <b>35</b>, (F) <b>40</b>, and (G) <b>36</b> to a resolution of 2.39, 2.00, 2.20, 2.15, 2.27, 2.16, and 2.28 Å, respectively. Interacting residues are shown as sticks. (H) Omit map (green) for compound <b>36</b> contoured at 2.5σ. (I) 2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> composite omit map (gray) contoured at 1σ showing residues 5 Å around compound <b>36</b>. (J) As a representative structure, compound <b>36</b> (wheat) is superimposed with the docked 5-aminosalicylate compound <b>4</b> (orange) and the corresponding docked hybrid compound <b>45</b> (green). (K) Hydrogen bond network with compound <b>36</b>. (L) Stacking interactions with compound <b>36</b>; the hydrophobic centers are indicated by a green sphere.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/large/jm-2015-015277_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01527&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In each of the seven complexes, the hybrid core of the compounds superimposes well with the docked pose (rmsd range from 0.45 to 0.77 Å, represented by <b>42</b> and <b>43</b>, respectively), forming nearly identical key interactions with the metal and α-KG pocket (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>J). As predicted by the docking poses, in all seven structures the inhibitor carboxylate salt-bridges with Lys206 N<sup>ε</sup> (2.6 Å) and hydrogen-bonds with Tyr132 OH (2.6 Å) (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>K), mimicking the interactions observed between the carboxylic acid of the α-KG cofactor in the active site<a onclick="showRef(event, 'ref49 ref50'); return false;" href="javascript:void(0);" class="ref ref49 ref50">(52, 53)</a>(PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2OX0">2OX0</a>). Similarly, in both the docking and the crystal structures, the inhibitor pyridine ring stacks with Phe185 (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>L), while the pyridine nitrogen, as expected, chelates the active site metal (Mn(II) was used as a mimic of oxygen sensitive Fe(II)) (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>K), an interaction analogous to the interaction between the metal and previously identified inhibitors such as 2,4-PDCA (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2VD7">2VD7</a>)<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> and 4′-[(2-aminoethyl)carbamoyl]-2,2′-bipyridine-4-carboxylic acid (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PDQ">3PDQ</a>).<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(51)</a> The second interaction formed with the metal is with the hydroxyl moiety (2.1 Å) of the <i>N</i>-acylamino substituted phenol ring (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>K). As in the docked structures, in the crystal structures this coordination forms an octahedral geometry, with an angle of 171° between the inhibitor’s hydroxyl group, the metal, and the N<sup>ε</sup> of His276; Glu190, His188, and a water molecule provide the remaining three metal coordinations. Finally, as anticipated by docking, the phenol ring of the hybrid molecule is sandwiched between the hydroxyl moiety of Tyr177 and the side chain of Lys241, while the pyridine ring is positioned between Phe185 and the aromatic ring of Tyr177 (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>L). We note that in several of the structures there is unexplained electron density that superimposes well with the position occupied by the trimethylated N<sup>ε</sup> of the lysine peptide substrate. This electron density is approximately 4 Å from the phenol ring and may be modeled as a DMSO molecule that could make π–σ stacking interactions with the inhibitors (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>L).</div><div class="NLM_p last">The one substantial difference between the docking poses and the crystallographic results is in the position of the exocyclic amide substituent, common to the five compounds crystallized (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>C–G). Whereas this difference has little effect on the overall placement of the core scaffold in the site (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>J), the details of the hydrogen-bonding to the enzyme change. In the docking predictions, the amide proton is predicted to hydrogen bond directly with Asp135. While a hydrogen bond between this amide and the protein is observed crystallographically, in some of the complexes (for example, <b>35</b>, <b>40</b>, and <b>42</b>) the nitrogen engages both Tyr177 and Asp135 through a bridging water molecule (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>D–F). In contrast, in the crystal structures of compounds <b>36</b> and <b>44</b>, the Tyr177 and Asp135 form a water-mediated hydrogen bond with the oxygen atom of the exocyclic amide of the inhibitor (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>C,G). Compound <b>40</b> is the largest compound for which a structure was solved; however, poor density is observed for its acyl substituent, which occupies different orientations in each crystallographic monomer (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01527/suppl_file/jm5b01527_si_001.pdf" class="ext-link">Figure S4F</a>). The acyl moieties of these inhibitors reach the peptide binding pocket and mostly occupy the area in which Ser10, Thr11, and Gly12 of the histone H3 substrate bind (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01527/suppl_file/jm5b01527_si_001.pdf" class="ext-link">Figure S5</a>).<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(52)</a> For example, the oxygen atom of the isoxazolyl moiety of compound <b>36</b> forms a hydrogen bond with the side chain nitrogen of Asn86 (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>K), consistent with docking poses of 5-aminosalicylate fragments (fragment <b>4</b>, <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>J).</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i14">Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_37727" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_37727" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In this study we applied fragment-based docking screens to identify novel KDM4 inhibitor chemotypes. Subsequent fragment optimization (typically requiring numerous iterations of structure determination, modeling, and synthesis) was streamlined by the use of docked geometries to inform fragment linking and the design of a hybrid scaffold. While fragment linking is considered more difficult than fragment elaboration,<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> it has been successfully implemented.<a onclick="showRef(event, 'ref55 ref56 ref57'); return false;" href="javascript:void(0);" class="ref ref55 ref56 ref57">(55-57)</a> Typically, fragment linking is guided by experimental binding geometries, either from NMR or from crystallography;<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> this work establishes the use of docking geometries to effectively guide fragment fusion. The success of the strategy here (supported by the 2 log-orders of affinity gained by the fused molecules and the correspondence of the docking predictions to the subsequent crystallographic results) support the use of docking not only to prioritize initial hits for testing but also to guide their optimization. This is further supported by earlier studies that suggest that docked fragments can pose in orientations that accurately represent experimental structures<a onclick="showRef(event, 'ref59 ref60 ref61 ref62'); return false;" href="javascript:void(0);" class="ref ref59 ref60 ref61 ref62">(59-62)</a> and that docking can prioritize among multiple binding modes sometimes suggested by experimental structures.<a onclick="showRef(event, 'ref63'); return false;" href="javascript:void(0);" class="ref ref63">(63)</a> A detailed analysis of representative hybrid salicylate compounds revealed a competitive binding mode with respect to α-KG that is supported by seven crystal structures (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>A–G), with <i>K</i><sub>i</sub> values of 43 nM for <b>35</b> and 680 nM for <b>42</b> (<a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>). Acyl derivatives of the hybrid scaffold exhibited increased inhibition of KDM4 demethylases in addition to increased selectivity versus the asparagine hydroxylase FIH (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>B,C, <a class="ref internalNav" href="#tbl5" aria-label="Table 5">Table 5</a>). Substantial selectivity was achieved against the H3K36 demethylase KDM2A and the H3K27 demethylase KDM6B, for example, compound <b>42</b> showed levels of selectivity ≥24-fold against these enzymes. However, selectivity versus the KDM5 and KDM3 subfamilies remains limited for all tested compounds.</div><div class="NLM_p">Due to the presence of the pyridyl-4-carboxylate moiety, our inhibitors resemble some of the known Jumonji inhibitor scaffolds<a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(64)</a> that also use this moiety, such as 2,4-PDCA derivatives,<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> bipyridyl derivatives,<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(51)</a> and 8-hydroxyquinoline derivatives<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01527/suppl_file/jm5b01527_si_001.pdf" class="ext-link">Figure S1A</a>). Indeed, since its discovery as a collagen prolyl hydroxylase inhibitor in the 1980s,<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> the 2,4-PDCA scaffold and its derivatives have been used for the inhibition of Fe(II) α-KG dependent dioxygenases. However, in contrast to previously reported KDM4 demethylase inhibitors, a unique feature of the molecules described in this study is the coordination geometry of the iron chelating atoms. While other iron-coordinating demethylase inhibitors are usually 1,4-bidentate ligands (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01527/suppl_file/jm5b01527_si_001.pdf" class="ext-link">Figure S1A</a>), molecules that resulted from this study are 1,5-bidentate ligands. This ligand architecture imposes an out of plane rotation of the two aromatic rings, with a median dihedral angle between the two planes of 36°, as determined in cocrystal structures of KDM4A with the inhibitors (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>). The observation that the hybrid ligands can adopt different dihedral angles between the two aromatic rings, which accommodate the interactions between the acyl substituent and the enzyme, suggests flexibility of the hybrid inhibitors. Importantly, the identification of these novel hybrid molecules was only possible through fragment discovery of the 5-aminosalicylate chemotype. The favorable ligand efficiency and the small size of the 5-aminosalicylate chemotype allowed us to grow this fragment to gain potency.</div><div class="NLM_p">Two key observations emerge from this study. First, against a soluble enzyme with little ligand precedence, a structure-based docking screen of 600 000 fragments revealed new inhibitors. All 14 of the docking-prioritized molecules demonstrated KDM4C inhibition. Half of the candidate inhibitor fragments showed IC<sub>50</sub> values better than 200 μM and LE values of 0.3 or better. Whereas structure-based docking has been shown to be effective in fragment prioritization, this was often against model enzymes, like β-lactamase<a onclick="showRef(event, 'ref59 ref60 ref63'); return false;" href="javascript:void(0);" class="ref ref59 ref60 ref63">(59, 60, 63)</a> for which there was substantial ligand precedence; this is much less true for KDM4C, a target of active biological interest for which such ligand precedence is much reduced. Second, and more ambitiously, this study represents the first successful use of docking poses to guide a fragment-linking and synthetic elaboration strategy. This approach was born out of the affinity maturation of the early compounds, and the subsequent crystallography verified the docking predictions.</div><div class="NLM_p">An important caveat to consider is that we do not suggest that docking can replace the cycles of structure determination and synthesis that are widely practiced in the field of fragment discovery and optimization; indeed, this study itself uses such cycles. We used docking to guide the fusion and optimization because we were unable to determine the cocomplex structures of the initial 5-aminosalicylate fragment hits, which bind in the 50–200 μM range. Thus, we do not advocate docking as a first strategy to guide fragment fusion and optimization. However, when determining initial fragment structures is difficult, this study suggests that docking may be a viable alternative.</div><div class="NLM_p last">In summary, this study has revealed how discovery of the novel midmicromolar 5-aminosalicylate series of inhibitors has been used for fragment linking to yield a potent hybrid scaffold. Optimization of this new scaffold has led to nanomolar hybrid inhibitors of the KDM4 family of epigenetic erasers. This work sets the stage for evaluation of these molecules in cellular assays and for further elaboration of these molecules to improve their selectivity with the ultimate goal of using them as potent and selective chemical probes of KDM4 family demethylase function in physiology and disease.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_74184" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_74184" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Docking and Fragment Identification for Testing</h3><div class="NLM_p">For docking we considered the demethylase domain of KDM4A and KDM4C enzymes to be identical, as the residues lining the active site cavity are conserved. Hence 20 chains from the structures of KDM4A (PDB codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2GP5">2GP5</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2GP3">2GP3</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2OQ7">2OQ7</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2OQ6">2OQ6</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2Q8C">2Q8C</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2VD7">2VD7</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2YBK">2YBK</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2WWJ">2WWJ</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3NJY">3NJY</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PDQ">3PDQ</a>) and KDM4C (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2XML">2XML</a>) were superimposed (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01527/suppl_file/jm5b01527_si_001.pdf" class="ext-link">Figure S1B</a>). Chain B of structure <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PDQ">3PDQ</a> was chosen as the docking model (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01527/suppl_file/jm5b01527_si_001.pdf" class="ext-link">Figure S1C</a>) because it was the highest resolution (1.99 Å) structure among the five inhibitor complexes that were used for pose recapitulation. Additionally, this chain had a well-defined rim of the active site, encompassing loop residues Leu153 to Thr173 that were unstructured in other models.</div><div class="NLM_p last">Chain B of <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PDQ">3PDQ</a> was prepared by replacing the Ni(II) metal in the active site with the catalytically relevant Fe(II) metal. Docking was calibrated by screening 40 known inhibitors (extracted from PDB codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2VD7">2VD7</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2WWJ">2WWJ</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2YBK">2YBK</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3NJY">3NJY</a>, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PDQ">3PDQ</a> and selected compounds from literature with IC<sub>50</sub> of 0.6–8.2 μM)<a onclick="showRef(event, 'ref23 ref25 ref27 ref46 ref47 ref48 ref65'); return false;" href="javascript:void(0);" class="ref ref23 ref25 ref27 ref46 ref47 ref48 ref65">(23, 25, 27, 49-51, 65)</a> vs 10 200 physically matched decoys. Results were evaluated based on known inhibitor enrichment and pose recapitulation. In the optimization, we explored representing the Fe(II) metal charge as +1.1, +1.2, +1.3, and +1.4 electrons vs +2, thereby displacing the extra charge from the Fe(II) to the metal chelating residues His188, His276, and Glu190, as previously described.<a onclick="showRef(event, 'ref42 ref66 ref67'); return false;" href="javascript:void(0);" class="ref ref42 ref66 ref67">(42, 66, 67)</a> The hydrogen placement on the enzyme was optimized to interact with known inhibitors. A good log AUC enrichment of 20.28 was obtained with Fe(II) at a charge of +1.3, with the charge on His188 and His276 increased by +0.25 each and the charge on Glu190 increased by +0.2 (the net charge of the iron system was unchanged). In DOCK3.6 spheres are typically calculated to represent the negative image of the binding site, on which ligand atoms are placed in calculating initial poses.<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a> Here, these spheres were calculated automatically<a onclick="showRef(event, 'ref69'); return false;" href="javascript:void(0);" class="ref ref69">(69)</a> and a few spheres found to be distant from the metal center were manually removed. For the docking screen that led to the 5-aminosalicylates, the partial atomic charge on the phenolic oxygen and hydrogen of Tyr132 was changed by 0.4 electron in reciprocal directions (the overall charge was again unchanged), a technique we have used previously to slightly compensate for the polar component of ligand recognition.<a onclick="showRef(event, 'ref70'); return false;" href="javascript:void(0);" class="ref ref70">(70)</a> A ligand and receptor bin size of 0.4, with an overlap of 0.1 was used to dock fragments from the ZINC database<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> (<a href="http://zinc.docking.org/subsets/fragment-like" class="extLink">http://zinc.docking.org/subsets/fragment-like</a>).</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> KDM4C Time-Resolved FRET Assay</h3><div class="NLM_p last">3-Fold serial dilutions of compound stocks in DMSO were added to 5 μL of assay buffer (50 mM HEPES, pH 7.0, 0.01 v/v % Tween-20, 0.01 m/v % BSA) supplemented with 10 μM iron(II) ammonium sulfate hexahydrate, 4 μM or 100 μM α-KG, 900 nM histone H3 (residues 1–21) lysine 9 trimethylated peptide with biotin tag (Anaspec), and 200 μM ascorbic acid in a 384-well white opaque OptiPlate (PerkinElmer). The reaction was started by adding 5 μL of 20 nM KDM4C (residues 1–352; see <a href="/doi/suppl/10.1021/acs.jmedchem.5b01527/suppl_file/jm5b01527_si_001.pdf" class="ext-link">Supporting Information methods</a> for cloning and purification) in assay buffer, then sealed and incubated for 45 min at ambient temperature. The final concentration of DMSO was 2%. The reaction was quenched by addition of 10 μL of detection mix containing 2 nM europium-conjugated anti-H3K9(me2) antibody (PerkinElmer), 100 nM Ulight-streptavadin conjugate (PerkinElmer), 2 mM EDTA, pH 8.0, and 1× LANCE detection buffer (PerkinElmer) in water. The quenched reaction was covered and incubated for 1 h in the dark and then analyzed by a SpectraMax M5e plate reader using TR-FRET mode with an excitation wavelength of 320 nm, emission wavelengths of 665 and 615 nm, 50 μs delay, 500 μs integration, and 100 reads per well. Signal was calculated as <i>E</i><sub>665</sub>/<i>E</i><sub>615</sub>. Values were plotted in GraphPad Prism and fit by nonlinear regression to calculate IC<sub>50</sub> values.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Crystallography</h3><div class="NLM_p last">The paralog KDM4A was expressed, purified, and used for crystallography as described.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(52)</a> Typically, protein solution at 1.8 mg/mL containing 500 μM MnCl<sub>2</sub> and 400 μM compound was incubated for 2 h on ice and spun for 5 min at 10 000 rcf at 4 °C to remove precipitate. The supernatant was collected and concentrated to 14–16 mg/mL. Crystals were grown by vapor diffusion by mixing 100 nL of protein solution with 50 nL of reservoir solution at 4 °C. Crystals appeared in conditions containing 22–30% PEG3350, 0.1 M Bis-Tris, pH 5.5–7.5, or 0.1 M Tris, pH 8.5, 0.15–0.25 M ammonium sulfate. Crystals were cryoprotected by addition of 25% ethylene glycol based on crystallization solution and flash-cooled in N<sub>2</sub>. Data were collected at beamlines Diamond-I04-1 or I02 and I04. Data sets were processed using Xia2.<a onclick="showRef(event, 'ref71'); return false;" href="javascript:void(0);" class="ref ref71">(71)</a> Iterative model building with COOT and refinement with PHENIX and BUSTER resulted in final models.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> FDH-Coupled Assay for Generation of Lineweaver–Burk Plots</h3><div class="NLM_p last">The following components were added to a black 96-well round-bottom Microfluor 1 plate (Thermo Scientific): reaction buffer (10 mM HEPES, pH 7.9, 50 mM NaCl, 0.01 v/v % Tween-20), enzymes, cofactors, additives (vide infra), and 3-fold compound dilutions from DMSO stocks (1 v/v % final DMSO concentration) to a final volume of 90 μL. Each well was then mixed thoroughly by manual pipetting, followed by the immediate addition of 10 μL of ARK(Me3)STGGK peptide substrate. Typical final concentrations are 50 μM iron(II) ammonium sulfate hexahydrate, 500 μM ascorbate, 2 mM NAD<sup>+</sup>, 0.0252 U formaldehyde dehydrogenase (FDH) per reaction, 1 μM KDM4C (residues 1–352), 50 μM peptide substrate, and variable concentration of α-KG (2.5–50 μM). The reaction was monitored by measuring the change in fluorescence intensity over time on a SpectraMax M5e plate reader with an excitation wavelength of 350 nm and emission wavelength of 460 nm.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Determination of <i>K</i><sub>i</sub> Values</h3><div class="NLM_p"><i>K</i><sub>i</sub> values were determined by FDH-coupled assay under identical conditions to those described above except that the inhibitor was incubated for 15 min in reaction mixture containing 500 nM enzyme concentration and lacking the peptide and α-KG. Following incubation, the reaction was initiated by addition of α-KG (10–100 μM) and peptide substrate.</div><div class="NLM_p last">The rate of the first 2 min of the reaction was calculated by a linear fit, normalized to the fluorescence intensity of 1 μM NADH. Values were plotted in GraphPad Prism. Owing to the high concentration of enzyme, the Morrison equation for tight-binding inhibition was used to globally fit data derived from testing inhibition in the presence of a range of concentrations of α-KG:<a onclick="showRef(event, 'ref72'); return false;" href="javascript:void(0);" class="ref ref72">(72)</a><span class="NLM_disp-formula" id="ueq1"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/jm-2015-015277_m001.gif" alt="" /></img></span>with the form of <i>K</i><sub class="stack">i</sub><sup class="stack">app</sup> for competitive inhibitors,<span class="NLM_disp-formula" id="ueq2"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/jm-2015-015277_m002.gif" alt="" /></img></span></div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Counterscreening with MALDI Assay</h3><div class="NLM_p last">General methods for counter screening of inhibitors against FIH and KDM5A can be found in the <a href="/doi/suppl/10.1021/acs.jmedchem.5b01527/suppl_file/jm5b01527_si_001.pdf" class="ext-link">methods section in Supporting Information</a>.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Counterscreening of KDM4C with AlphaScreen Assay</h3><div class="NLM_p last">Antibody based AlphaScreen assays were used to detect demethylated peptide product<a onclick="showRef(event, 'ref73'); return false;" href="javascript:void(0);" class="ref ref73">(73)</a> to counterscreen the KDM4C inhibitors against KDM2A, KDM3A, KDM4D, KDM5B, KDM6B. Details of enzyme purification and assay conditions are described in the <a href="/doi/suppl/10.1021/acs.jmedchem.5b01527/suppl_file/jm5b01527_si_001.pdf" class="ext-link">methods section in Supporting Information</a>.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Controls for Colloidal Aggregation</h3><div class="NLM_p last">Two of the initial docking fragment hits, compounds <b>4</b> and <b>5</b>, were tested for colloidal aggregation in an AmpC β-lactamase counterscreen and by dynamic light scattering (DLS).<a onclick="showRef(event, 'ref74 ref75'); return false;" href="javascript:void(0);" class="ref ref74 ref75">(74, 75)</a> Synthesized inhibitors that had steep concentration–response curves were dropped from further consideration, as we were concerned that they were likely aggregating and inhibiting nonspecifically.</div></div><div id="sec4_9_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> Synthesis of Hybrid Scaffold. 2-(5-Amino-2-hydroxyphenyl)isonicotinic Acid (<b>30</b>)</h3><div class="NLM_p last">1.28 mmol of methyl 2-bromoisonicotinate (277 mg) (Combi-Blocks), 0.85 mmol of 2-methoxy-5-nitrophenylboronic acid pinacol ester (237 mg) (Frontier Scientific), Cs<sub>2</sub>CO<sub>3</sub> (1.63 g), 0.1 mmol of CombiPhos-Pd6 (50 mg) (CombiPhos Catalysts), and 10 mL of DMF were combined in a 15 mL pressure vial and stirred for 20 h at 100 °C. The mixture was then diluted in 75 mL of water, acidified to pH 2 with 2 N HCl, and extracted three times with 50 mL of EtOAc. The combined organic phase was washed with 5% citric acid, followed by 50 mL of saturated NaCl, which included 1 mL of 2 N HCl, and concentrated under reduced pressure. The crude mixture was transferred to a new 15 mL pressure vial. 6 mL of 48% aqueous HBr (Sigma) and 4 mmol NaI (600 mg) were added, and the mixture was stirred at 100 °C for 4 days. The mixture was then diluted in 50 mL of water and washed once with 50 mL of EtOAc, and the aqueous fraction was concentrated under reduced pressure. HPLC purification (0.1% TFA, acetonitrile/water gradient of 0–10 min, 10% B; 10–89 min, 10–40% B; 89–90 min, 40–100% B; 90–100 min, 100% B) was followed by freeze-drying and yielded 38 mg of product (0.165 mmol, 19%). LCMS [M + H]<sup>+</sup><i>m</i>/<i>z</i> = 231.54. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, ppm) δ 8.85 (d, <i>J</i> = 5.1 Hz, 1H), 8.52 (d, <i>J</i> = 0.8 Hz, 1H), 8.07 (d, <i>J</i> = 2.1 Hz, 1H), 7.92–7.79 (m, 1H), 7.38–7.27 (m, 1H), 7.10 (dd, <i>J</i> = 8.7, 0.8 Hz, 1H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>, ppm) δ 165.9, 158.0, 156.4, 156.0, 148.7, 139.8, 125.2, 121.5, 120.8, 120.3, 118.7.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> Typical Reductive Amination</h3><div class="NLM_p last">Hybrid scaffold <b>30</b> (1 equiv) and aldehyde (2 equiv) were combined in 0.02 M ethanol and refluxed for 2 h. NaBH<sub>4</sub> (4 equiv) was then added and the resulting mixture was refluxed for an additional 30 min. This mixture was diluted in water and purified by HPLC.</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> Typical Acylation</h3><div class="NLM_p last">Acid chloride (5 equiv) and <i>N</i>-hydroxysuccinimide (10 equiv) were combined in 0.15 M acetonitrile and stirred at rt for 1 h. <b>30</b> (1 equiv) was added in an equal volume of water/acetonitrile/methanol (1:1:1) and stirred at rt for 16 h. This mixture was diluted in water and purified by HPLC.</div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> General Methods for Synthesis of Compounds</h3><div class="NLM_p last">All reagents and solvents were purchased as the highest available grade from Sigma-Aldrich and used without further purification, unless otherwise indicated. Reverse-phase high performance liquid chromatography (RP-HPLC) was performed with a Varian ProStar solvent delivery system equipped with a Phenomenex Luna 10 μm C18(2) 100 Å column. Separation was achieved using a gradient of acetonitrile or methanol in water with 0.1% TFA, at a flow rate of 15 mL/min. Compounds were purified by RP-HPLC to >95% purity, as assessed by UPLC–MS peak integration and proton NMR. <sup>1</sup>H NMR data were recorded with a Varian Innova 400 MHz spectrometer. <sup>13</sup>C NMR data were recorded either with a Varian Innova 400 MHz spectrometer or a 500 MHz Bruker AVANCE DRX500 spectrometer equipped with an actively shielded Z-gradient QCI cryoprobe (H-P/C/N-D). <sup>13</sup>C spectra were analyzed following multipoint baseline correction. Mass spectrometry (ESI-MS) was performed using a Waters Acquity UPLC/ESI-TQD equipped with a 2.1 mm × 50 mm Acquity UPLC BEH C18 column.</div></div><div id="sec4_12_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> <b>1</b></h3><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/medium/jm-2015-015277_0014.gif" alt="" id="fx1" /></img><div class="NLM_p last">Purchased from Acros.</div></div><div id="sec4_12_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> <b>2</b></h3><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/medium/jm-2015-015277_0015.gif" alt="" id="fx2" /></img><div class="NLM_p last">Molecular weight: 246.26. <span class="smallcaps smallerCapital">l</span>-tryptophan (6 mmol, 1.226 g), urea (10.3 mmol, 0.661 g), and 1.8 mL of a 3.3 N aqueous solution of NaOH were combined in a 50 mL beaker and mixed. Mixture was microwaved (1100 W) on high for 4 min along with a 500 mL beaker full of water. Mixture was cooled to 0 °C, and 3 mL of a 2 N aqueous solution of HCl was added. The precipitate was isolated by filtration and washed with water. Crude yield was 848.5 mg (3.43 mmol, 57%). 100 mg of crude material was HPLC purified (0.1% TFA, acetonitrile/water gradient of 0–5 min, 20% B; 5–30 min, 20–100% B; 30–60 min, 100% B) followed by freeze-drying and yielded 92.9 mg of solid (0.38 mmol, 53%). LCMS [ES]<sup>−</sup><i>m</i>/<i>z</i> = 246.08. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, ppm) δ 12.51 (s, 1H), 10.85 (s, 1H), 7.51 (d, <i>J</i> = 7.8 Hz, 1H), 7.33 (d, <i>J</i> = 8.1 Hz, 1H), 7.09 (d, <i>J</i> = 2.4 Hz, 1H), 7.06 (ddd, <i>J</i> = 8.1, 7.0, 1.2 Hz, 1H), 6.97 (ddd, <i>J</i> = 7.9, 7.1, 1.0 Hz, 1H), 6.11 (d, <i>J</i> = 8.1 Hz, 1H), 5.61 (s, 2H), 4.46–4.32 (m, 1H), 3.10 (dd, <i>J</i> = 14.6, 5.3 Hz, 1H), 3.01 (dd, <i>J</i> = 14.7, 7.0 Hz, 1H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>, ppm) δ 174.4, 158.2, 136.1, 127.4, 123.6, 120.9, 118.4, 118.3, 111.3, 109.7, 53.1, 27.9.</div></div><div id="sec4_12_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> <b>3</b></h3><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/medium/jm-2015-015277_0016.gif" alt="" id="fx3" /></img><div class="NLM_p last">Molecular weight: 246.26. <span class="smallcaps smallerCapital">d</span>-Tryptophan (3 mmol, 0.614 g), urea (4.99 mmol, 0.300 g), and 0.9 mL of a 3.3 N aqueous solution of NaOH were combined in a 50 mL beaker and mixed. The mixture was microwaved (1100 W) on high for 2 min along with a 500 mL beaker full of water. The mixture was then cooled to 0 °C. 1.5 mL of a 2 N aqueous solution of HCl was then added and the precipitate was isolated by filtration and washed with water, yielding 942.7 mg of solid (3.82 mmol, 128%). LCMS [ES]<sup>−</sup><i>m</i>/<i>z</i> = 248.63. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, ppm) δ 10.83 (s, 1H), 7.51 (d, <i>J</i> = 8.2 Hz, 1H), 7.32 (d, <i>J</i> = 8.1 Hz, 1H), 7.09 (d, <i>J</i> = 2.3 Hz, 1H), 7.04 (t, <i>J</i> = 7.0 Hz, 1H), 7.00–6.91 (m, 1H), 6.08 (d, <i>J</i> = 8.1 Hz, 1H), 5.58 (s, 2H), 5.39 (s, 1H), 4.31 (s, 1H), 3.10 (dd, <i>J</i> = 14.6, 5.5 Hz, 1H), 3.00 (dd, <i>J</i> = 14.5, 6.7 Hz, 1H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>, ppm) δ 174.4, 158.2, 136.1, 127.4, 123.6, 120.9, 118.4, 118.3, 111.3, 109.7, 53.1, 27.9.</div></div><div id="sec4_12_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> <b>4</b></h3><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/medium/jm-2015-015277_0017.gif" alt="" id="fx4" /></img><div class="NLM_p last">Molecular weight: 247.25. 5-Aminosalicylate (1 mmol, 153 mg), 1-methyl-1<i>H</i>-pyrazole-5-carbaldehyde (0.5 mmol, 49.2 μL), and 10 mL of EtOH were combined in a flame-dried round-bottom flask fitted with a reflux condenser, under an argon atmosphere. The mixture was stirred for 2 h at 100 °C. Sodium borohydride (1 mmol, 38 mg) was added, and mixture was stirred for another 15 h at ambient temperature. 20 mL of saturated NH<sub>4</sub>Cl was used to quench the reaction, and the product was extracted into 2 × 50 mL of EtOAc. Organic layers were combined and washed with saturated NaCl, dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure, yielding 218.8 mg of crude product (0.88 mmol, 177%). 22.2 mg of crude product was HPLC purified (0.1% TFA, acetonitrile/water gradient of 0–5 min, 15% B; 5–25 min, 15–80% B; 25–35 min, 80–100% B; 35–50 min, 100% B) and the purified product freeze-dried. Final yield was 12.3 mg of solid (0.050 mmol, 98%). LCMS [M – H]<sup>−</sup><i>m</i>/<i>z</i> = 246.45. <sup>1</sup>H NMR (400 MHz, acetonitrile-<i>d</i><sub>3</sub>, ppm) δ 7.71 (s, 1H), 7.13 (d, <i>J</i> = 2.8 Hz, 1H), 6.98 (dd, <i>J</i> = 8.9, 2.8 Hz, 1H), 6.83 (d, <i>J</i> = 8.9 Hz, 1H), 6.42 (s, 1H), 4.38 (s, 2H), 3.95 (s, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>, ppm) δ 171.8, 158.5, 158.2, 137.3, 117.7, 116.5, 114.2, 112.7, 106.0, 36.4, 36.3.</div></div><div id="sec4_12_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> <b>5</b></h3><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/medium/jm-2015-015277_0018.gif" alt="" id="fx5" /></img><div class="NLM_p last">Purchased from Enamine, Ltd.</div></div><div id="sec4_12_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> <b>6</b></h3><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/medium/jm-2015-015277_0019.gif" alt="" id="fx6" /></img><div class="NLM_p last">Molecular weight: 188.22. 2-Aminoisobutyric acid (2 mmol, 206 mg), 1-isocyanatopropane (2.1 mmol, 200 uL), and 3 mL of 0.33 N NaOH were combined in a 10 mL flask and stirred at ambient temperature for 2 days. Precipitate was filtered and the solution was acidified to pH 2.0 with 2 N HCl, extracted with EtOAc, and concentrate under reduced pressure. The compound was then HPLC purified (0.1% TFA, acetonitrile/water gradient of 0–5 min, 20% B; 5–30 min, 20–100% B; 30–60 min, 100% B) followed by freeze-drying, yielding 67 mg of solid (0.36 mmol, 18%). LCMS [M + H]<sup>+</sup><i>m</i>/<i>z</i> = 189.11. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, ppm) δ 8.23 (s, 1H), 3.29 (t, <i>J</i> = 7.0 Hz, 2H), 1.50 (p, <i>J</i> = 7.2 Hz, 2H), 1.27 (s, 6H), 0.80 (t, <i>J</i> = 7.4 Hz, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>, ppm) δ 177.56, 155.50, 57.63, 24.69, 20.92, 10.96.</div></div><div id="sec4_12_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> <b>7</b></h3><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/medium/jm-2015-015277_0020.gif" alt="" id="fx7" /></img><div class="NLM_p last">Molecular weight: 240.28. 5-Aminosalicylic acid (1 mmol, 153 mg), ethyl isocyanate (1 mmol, 90 uL), and 2 mL of acetonitrile were combined in a 50 mL flask and refluxed for 1.5 h. The mixture was then cooled to ambient temperature, 20 mL of DMF was added, and the mixture was refluxed for an additional 75 min. The mixture was again cooled to ambient temperature and concentrated under reduced pressure. Crude material was suspended in 20 mL of EtOAc and extracted with 10 mL of saturated aqueous NaHCO<sub>3</sub>. The aqueous layer was washed with 2 × 20 mL EtOAc, and the combined EtOAc washes were re-extracted with 2 × 10 mL saturated aqueous NaHCO<sub>3</sub>. The aqueous layer was acidified with 60 mL of 2 N HCl and then extracted with 3 × 20 mL of EtOAc. Organic fractions were combined and washed with 50 mL of brine, dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. The product was HPLC purified (0.1% TFA, acetonitrile/water gradient of 0–10 min, 20% B; 10–25 min, 20–80% B; 25–30 min, 80% B; 35–60 min, 100% B), followed by freeze-drying, yielding 156.3 mg of solid (0.65 mmol, 65%). LCMS [M – H]<sup>−</sup><i>m</i>/<i>z</i> = 239.4. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, ppm) δ 11.12 (s, 1H), 9.25 (s, 1H), 7.77–7.53 (m, 1H), 7.45 (d, <i>J</i> = 8.8 Hz, 1H), 6.91 (d, <i>J</i> = 8.8 Hz, 1H), 3.45 (s, 2H), 1.09 (t, <i>J</i> = 7.1 Hz, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>, ppm) δ 180.7, 171.6, 158.3, 132.8, 130.6, 126.0, 117.2, 112.5, 56.0, 18.6, 14.3.</div></div><div id="sec4_12_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> <b>8</b></h3><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/medium/jm-2015-015277_0021.gif" alt="" id="fx8" /></img><div class="NLM_p last">Molecular weight: 234.20. Citric acid (5.04 mmol, 968 mg) was added portionwise to 1.4 mL of H<sub>2</sub>SO<sub>4</sub> and stirred for 40 min at ambient temperature, followed by stirring for 70 min at 70 °C. The mixture was cooled to 0 °C, and 5-methylbenzene-1,3-diol (3.97 mmol, 492 mg) was added portionwise over 15 min. The mixture was stirred at 0 °C for 2 h and then poured onto 30 g of ice. The filtrate was collected, washed with 4 × 5 mL of water, and eluted with 5 × 5 mL of saturated aqueous NaHCO<sub>3</sub>. The filtrate was acidified with 30 mL of 2 N HCl and extracted with 2 × 30 mL of EtOAc. EtOAc fractions were combined and washed with 2 × 50 mL of brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated under reduced pressure, yielding 152.2 mg of solid (0.65 mmol, 16%). LCMS [M – H]<sup>−</sup><i>m</i>/<i>z</i> = 233.50. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>, ppm) δ 6.94 (dd, <i>J</i> = 2.1, 1.2 Hz, 1H), 6.81 (s, 1H), 6.15 (t, <i>J</i> = 1.8 Hz, 1H), 4.03 (d, <i>J</i> = 1.7 Hz, 2H), 2.49–2.44 (m, 3H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>, ppm) δ 216.5, 162.5, 159.9, 153.2, 149.6, 146.2, 144.3, 113.0, 112.3, 110.9, 102.3, 33.1, 22.5.</div></div><div id="sec4_12_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> <b>9</b></h3><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/medium/jm-2015-015277_0022.gif" alt="" id="fx9" /></img><div class="NLM_p last">Molecular weight: 252.27. 1-(2-Methoxy-2-oxoethyl)pyrazole-3-carboxylic acid (Aurora) (0.16 mmol, 30 mg), HATU (0.187 mmol, 71 mg), DIPEA (0.172 mmol, 30 μL), <i>N</i>-methylpiperazine (0.27 mmol, 30 μL), and dry DMF (0.7 mL) were added to a 5 mL flask and stirred at ambient temperature overnight. The reaction was concentrated under reduced pressure, then partitioned between water (1 mL) and ethyl acetate (2 mL). The aqueous layer was washed twice with ethyl acetate (2 mL). The aqueous layer was HPLC purified (0.1% TFA, acetonitrile/water gradient of 0–5 min, 0% B; 5–30 min, 0–30% B; 30–60 min, 100% B) followed by freeze-drying, yielding 19.0 mg of solid (0.070 mmol, 46%). LCMS [M + H]<sup>+</sup><i>m</i>/<i>z</i> = 253. <sup>1</sup>H NMR (400 MHz, D<sub>2</sub>O, ppm) δ 7.80 (dd, <i>J</i> = 2.4, 0.9 Hz, 1H), 6.72 (dd, <i>J</i> = 2.3, 1.0 Hz, 1H), 5.14 (d, <i>J</i> = 0.7 Hz, 2H), 4.79–4.68 (m, 2H), 3.66–3.57 (m, 2H), 3.37–3.17 (m, 2H), 2.10–1.98 (s, 3H). <sup>13</sup>C NMR (100 MHz, D<sub>2</sub>O, ppm) δ 171.8, 164.9, 145.0, 133.9, 108.4, 53.3, 53.0, 43.0, 39.7.</div></div><div id="sec4_12_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> <b>10</b></h3><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/medium/jm-2015-015277_0023.gif" alt="" id="fx10" /></img><div class="NLM_p last">Molecular weight: 241.24. Methyl 2-chloroisonicotinate (0.76 mmol, 130 mg), 2-acetylphenylboronic acid (1.39 mmol, 229 mg) (Alpha Aesar), 2.3 mL of aqueous 2 M K<sub>2</sub>CO<sub>3</sub>, and 3 mL of 1,4-dioxane were combined in a 50 mL flask and bubbled with argon for 10 min. Pd(PPh<sub>3</sub>)<sub>4</sub> (0.073 mmol, 84 mg) was added, and the mixture was stirred for 12 h at reflux. The mixture was cooled to ambient temperature, diluted with 30 mL of EtOAc, and the organic layer was washed with 2 × 10 mL of saturated aqueous NaHCO<sub>3</sub>. The aqueous layers were combined, acidified with 2 N HCl, extracted with 2 × 15 mL of <i>n</i>-BuOH, concentrated under reduced pressure, and finally HPLC purified (0.1% TFA, acetonitrile/water gradient of 0–10 min, 20% B; 10–25 min, 20–80% B; 25–30 min, 80% B; 35–60 min, 100% B). The product was freeze-dried, yielding 73.8 mg of solid (0.31 mmol, 40%). LCMS [M – H]<sup>−</sup><i>m</i>/<i>z</i> = 240.5. <sup>1</sup>H NMR (400 MHz, D<sub>2</sub>O, ppm) δ 9.19 (d, <i>J</i> = 6.4 Hz, 1H), 8.72 (s, 1H), 8.30–8.22 (m, 2H), 7.91–7.77 (m, 3H), 2.09 (d, <i>J</i> = 0.9 Hz, 3H). <sup>13</sup>C NMR (126 MHz, D<sub>2</sub>O, ppm) δ 166.7, 151.0, 150.3, 143.3, 139.1, 134.4, 131.6, 128.2, 125.2, 123.9, 123.4, 120.2, 101.6, 25.7.</div></div><div id="sec4_12_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> <b>11</b></h3><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/medium/jm-2015-015277_0024.gif" alt="" id="fx11" /></img><div class="NLM_p last">Purchased from Enamine, Ltd.</div></div><div id="sec4_12_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> <b>12</b></h3><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/medium/jm-2015-015277_0025.gif" alt="" id="fx12" /></img><div class="NLM_p last">Molecular weight: 228.63. 4-Chloropicolinic acid (0.52 mmol, 81 mg), <span class="smallcaps smallerCapital">l</span>-alanine methyl ester HCl (0.58 mmol, 81 mg), 1-ethyl-3-(3-dimethylaminopropyl)carbodiimide HCl (0.55 mmol, 105 mg), hydroxybenzotriazole (0.57 mmol, 77 mg), diisopropylethylamine (0.80 mmol, 140 uL), and 1 mL DMF were combined in a 10 mL Schlenk flask and stirred at ambient temperature for 18 h. The mixture was concentrated under reduced pressure and the crude material dissolved in 5 mL of EtOAc. The solution was subsequently washed with 2 × 5 mL of 5% aqueous citric acid; 2 × 5 mL of 5% aqueous NaHCO<sub>3</sub>; 2 × 5 mL of water; followed by 10 mL of brine. The organic layer was dried over Na<sub>2</sub>SO<sub>4</sub>, concentrated under reduced pressure, and purified by flash chromatography (eluting with EtOAc/hexanes (20/80)), yielding 72 mg of crude intermediate. 44 mg of this material was dissolved in 1.2 mL of 6 N HCl and 1.2 mL of 1,4-dioxane in a 10 mL flask and stirred at 60 °C for 9 h. Volatiles were removed under reduced pressure. HPLC purification (0.1% TFA, acetonitrile/water gradient of 0–5 min, 25% B; 5–30 min, 25–100% B; 30–60 min, 100% B) was followed by freeze-drying, yielding 21.7 mg of solid (0.095 mmol, 53%). LCMS [M – H]<sup>−</sup><i>m</i>/<i>z</i> = 227.43. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, ppm) δ 8.90 (s, 1H), 8.66 (d, <i>J</i> = 5.3 Hz, 1H), 8.04 (s, 1H), 7.80 (dd, <i>J</i> = 5.3, 0.9 Hz, 1H), 4.60–4.40 (m, 1H), 1.43 (d, <i>J</i> = 7.0 Hz, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>, ppm) δ 173.6, 162.25, 151.2, 150.2, 144.7, 126.7, 122.0, 47.9, 17.2.</div></div><div id="sec4_12_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> <b>13</b></h3><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/medium/jm-2015-015277_0026.gif" alt="" id="fx13" /></img><div class="NLM_p last">Purchased from Oakwood.</div></div><div id="sec4_12_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> <b>14</b></h3><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/medium/jm-2015-015277_0027.gif" alt="" id="fx14" /></img><div class="NLM_p last">Purchased from Otava.</div></div><div id="sec4_12_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i42" class="anchor-spacer"></div><h3 class="article-section__title" id="_i42"> <b>15</b></h3><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/medium/jm-2015-015277_0028.gif" alt="" id="fx15" /></img><div class="NLM_p last">Molecular weight: 230.22. Quinoline-2-carbonyl chloride (1.07 mmol, 205 mg), glycine methyl ester HCl (1.06 mmol, 134 mg), and 20 mL of CH<sub>2</sub>Cl<sub>2</sub> were combined in a 100 mL Schlenk flask and stirred. Into this solution, Et<sub>3</sub>N (24.4 mmol, 3.4 mL) was added dropwise over 5 min. The mixture was stirred at ambient temperature for 2 days. The reaction mixture was then washed with 2 × 20 mL of water, dried over Na<sub>2</sub>SO<sub>4</sub>, concentrated under reduced pressure, and purified by flash chromatography (eluting in EtOAc/hexanes (25/75)), yielding 147 mg of crude intermediate. 70 mg of this material was dissolved in 1.5 mL of 6 N HCl and 1.5 mL of 1,4-dioxane and stirred at 60 °C for 1 h. Volatiles were removed under reduced pressure. HPLC purification (0.1% TFA, acetonitrile/water gradient of 0–5 min, 30% B; 5–30 min, 30–100% B; 30–60 min, 100% B), followed by freeze-drying, yielded 54.4 mg of solid (0.24 mmol, 46%). LCMS [M + H]<sup>+</sup><i>m</i>/<i>z</i> = 231.57. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, ppm) δ 12.74 (s, 1H), 9.13 (t, <i>J</i> = 6.1 Hz, 1H), 8.59 (dd, <i>J</i> = 8.5, 0.7 Hz, 1H), 8.19–8.08 (m, 3H), 7.89 (ddd, <i>J</i> = 8.4, 6.9, 1.5 Hz, 1H), 7.74 (ddd, <i>J</i> = 8.1, 6.9, 1.2 Hz, 1H), 4.05 (d, <i>J</i> = 6.1 Hz, 2H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>, ppm) δ 171.1, 164.2, 149.6, 146.0, 138.0, 130.7, 129.2, 128.9, 128.2, 128.2, 118.6, 41.2.</div></div><div id="sec4_12_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> <b>16</b></h3><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/medium/jm-2015-015277_0029.gif" alt="" id="fx16" /></img><div class="NLM_p last">Molecular weight: 287.27. 3-Methoxybenzoyl chloride (0.13 mmol, 22 mg), <i>N</i>-hydroxysuccinamide (0.26 mmol, 30 mg), diisopropylethylamine (0.5 mmol, 87 μL), and 1.5 mL of acetonitrile were combined and stirred at ambient temperature for 30 min. Subsequently 5-aminosalicylate (0.13 mmol, 20 mg) was added and the mixture was stirred at ambient temperature for 5 h. The reaction was acidified with 2 M HCl, extracted with EtOAc, and concentrated under reduced pressure. HPLC purification (0.1% TFA, acetonitrile/water gradient of 0–5 min, 20% B; 5–25 min, 20–100% B), followed by freeze-drying, yielded 13.7 mg of white solid (0.048 mmol, 37%). LCMS [M + H]<sup>+</sup><i>m</i>/<i>z</i> = 288. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, ppm) δ 11.06 (s, 1H), 10.19 (s, 1H), 8.27 (d, <i>J</i> = 2.7 Hz, 1H), 7.88 (dd, <i>J</i> = 8.9, 2.7 Hz, 1H), 7.58–7.47 (m, 1H), 7.44 (t, <i>J</i> = 7.9 Hz, 1H), 7.15 (ddd, <i>J</i> = 8.2, 2.6, 0.9 Hz, 1H), 6.97 (d, <i>J</i> = 8.9 Hz, 1H), 4.15 (s, 1H), 3.84 (s, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>, ppm) δ 171.9, 165.1, 159.4, 157.6, 136.3, 131.0, 129.8, 128.9, 122.1, 120.0, 117.5, 117.3, 113.0, 112.6, 55.5.</div></div><div id="sec4_12_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> <b>17</b></h3><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/medium/jm-2015-015277_0030.gif" alt="" id="fx17" /></img><div class="NLM_p last">Molecular weight: 315.36. 1-Adamantanecarbonyl chloride (0.13 mmol, 26 mg), <i>N</i>-hydroxysuccinamide (0.26 mmol, 30 mg), diisopropylethylamine (0.5 mmol, 87 μL), and 1.5 mL of acetonitrile were combined and stirred at ambient temperature for 30 min. Then 5-aminosalicylate (0.13 mmol, 20 mg) was added and the mixture was stirred at ambient temperature for 5 h. The reaction was acidified with 2 M HCl, extracted with EtOAc, and concentrated under reduced pressure. HPLC purification (0.1% TFA, acetonitrile/water gradient of 0–5 min, 50% B; 5–30 min, 50–100% B), followed by freeze-drying, yielded 1.0 mg of white solid (0.0032 mmol, 2.5%). LCMS [M + H]<sup>+</sup><i>m</i>/<i>z</i> = 316. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, ppm) δ 9.06 (s, 1H), 7.96 (d, <i>J</i> = 2.8 Hz, 1H), 7.68 (dd, <i>J</i> = 8.9, 2.8 Hz, 1H), 6.74 (d, <i>J</i> = 8.8 Hz, 1H), 2.02 (s, 2H), 1.91 (s, 3H), 1.70 (s, 3H).</div></div><div id="sec4_12_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i45" class="anchor-spacer"></div><h3 class="article-section__title" id="_i45"> <b>18</b></h3><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/medium/jm-2015-015277_0031.gif" alt="" id="fx18" /></img><div class="NLM_p last">Molecular weight: 257.28. 0.25 mmol paraformaldehyde (0.25 mmol, 7.5 mg), <i>N</i>-benzyl-5-aminosalicylate (0.12 mmol, 29 mg), NaBH<sub>3</sub>CN (0.32 mmol, 20 mg), and 2 mL of EtOH were combined in a round-bottom flask fitted with a reflux condenser. The mixture was heated for 2 h at 100 °C and subsequently cooled to ambient temperature and acidified with 2 M HCl. HPLC purification (0.1% TFA, acetonitrile/water gradient of 0–5 min, 10% B; 5–30 min, 10–80% B), followed by freeze-drying, yielded 11.9 mg of white solid (0.046 mmol, 39%). LCMS [M + H]<sup>+</sup><i>m</i>/<i>z</i> = 258. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, ppm) δ 11.08 (s, 1H), 7.47–7.07 (m, 7H), 6.90 (d, <i>J</i> = 7.7 Hz, 1H), 4.55 (s, 2H), 3.01 (s, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>, ppm) δ 171.5, 158.6, 158.2, 128.5, 128.4, 127.6, 124.3, 118.0, 117.0, 114.1, 112.9, 58.3, 40.6.</div></div><div id="sec4_12_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i46" class="anchor-spacer"></div><h3 class="article-section__title" id="_i46"> <b>19</b></h3><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/medium/jm-2015-015277_0032.gif" alt="" id="fx19" /></img><div class="NLM_p last">Molecular weight: 365.38. 4-(4-Methoxyphenoxy)benzaldehyde (0.25 mmol, 57 mg), 5-aminosalicylate (0.12 mmol, 18 mg), NaCNBH<sub>3</sub> (0.32 mmol, 20 mg), and 2 mL of EtOH were combined in a round-bottom flask fitted with a reflux condenser. The mixture was heated for 2 h at 100 °C, then cooled to ambient temperature and acidified with 2 M HCl. HPLC purification (0.1% TFA, acetonitrile/water gradient of 0–5 min, 30% B; 5–30 min, 30–100% B), followed by freeze-drying, yielded 17.2 mg of yellow solid (0.047 mmol, 23%). LCMS [M + H]<sup>+</sup><i>m</i>/<i>z</i> = 366. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, ppm) δ 7.34 (dd, <i>J</i> = 8.4, 3.0 Hz, 1H), 7.17 (s, 1H), 7.04 (d, <i>J</i> = 8.7 Hz, 1H), 6.99–6.92 (m, 2H), 6.89 (dd, <i>J</i> = 8.6, 3.3 Hz, 1H), 6.81 (dd, <i>J</i> = 8.9, 3.3 Hz, 1H), 4.25 (s, 2H), 3.74 (d, <i>J</i> = 3.6 Hz, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>, ppm) δ 171.7, 157.2, 155.6, 154.7, 154.6, 149.5, 129.6, 123.9, 120.7, 120.6, 117.7, 117.3, 115.1, 112.8, 55.4, 48.5.</div></div><div id="sec4_12_20" class="NLM_sec NLM_sec_level_2"><div id="ac_i47" class="anchor-spacer"></div><h3 class="article-section__title" id="_i47"> <b>20</b></h3><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/medium/jm-2015-015277_0033.gif" alt="" id="fx20" /></img><div class="NLM_p last">Molecular weight: 257.28. 2-Methylbenzaldehyde (0.25 mmol, 30 mg), 5-aminosalicylate (0.12 mmol, 18 mg), NaCNBH<sub>3</sub> (0.32 mmol, 20 mg), and 2 mL of EtOH were combined in a round-bottom flask fitted with a reflux condenser. The mixture was heated for 2 h at 100 °C, then cooled to ambient temperature and acidified with 2 M HCl. HPLC purification (0.1% TFA, acetonitrile/water gradient of 0–5 min, 10% B; 5–30 min, 10–80% B), followed by freeze-drying, yielded 36.2 mg of white solid (0.14 mmol, 70%). LCMS [M + H]<sup>+</sup><i>m</i>/<i>z</i> = 258.28. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, ppm) δ 7.34–7.27 (m, 1H), 7.20–7.11 (m, 4H), 7.03 (dd, <i>J</i> = 8.9, 2.7 Hz, 1H), 6.81 (d, <i>J</i> = 8.9 Hz, 1H), 4.23 (s, 2H), 2.31 (s, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>, ppm) δ 171.8, 154.4, 138.7, 136.3, 136.0, 130.1, 128.1, 127.2, 125.8, 123.5, 117.7, 114.0, 112.8, 46.8, 18.7.</div></div><div id="sec4_12_21" class="NLM_sec NLM_sec_level_2"><div id="ac_i48" class="anchor-spacer"></div><h3 class="article-section__title" id="_i48"> <b>21</b></h3><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/medium/jm-2015-015277_0034.gif" alt="" id="fx21" /></img><div class="NLM_p last">Molecular weight: 322.15. 5-Aminosalicylate (0.2 mmol, 31 mg), 4-bromobenzaldehyde (0.2 mmol, 37 mg), and 2 mL of EtOH were combined in a round-bottom flask fitted with a reflux condenser. The mixture was heated for 1 h at 100 °C, then sodium borohydride (0.4 mmol, 15 mg) was added and the mixture was maintained at 100 °C for 3 h. The reaction was acidified with 2 M HCl. HPLC purification (0.1% TFA, acetonitrile/water gradient of 0–5 min, 10% B; 5–20 min, 10–35% B; 20–30 min, 35–75% B), followed by freeze-drying, yielded 35.5 mg of orange solid (0.11 mmol, 55%). LCMS [M + H]<sup>+</sup><i>m</i>/<i>z</i> = 322.13, 324.11. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, ppm) δ 10.50 (s, 1H), 7.51 (s, 1H), 7.49 (s, 1H), 7.32 (s, 1H), 7.29 (s, 1H), 6.92 (d, <i>J</i> = 2.9 Hz, 1H), 6.85 (dd, <i>J</i> = 8.8, 3.0 Hz, 1H), 6.72 (d, <i>J</i> = 8.9 Hz, 1H), 4.19 (s, 2H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>, ppm) δ 172.0, 152.8, 141.0, 139.7, 131.1 (2C), 129.4 (2C), 121.9, 119.6, 117.5, 112.5, 111.5, 46.5.</div></div><div id="sec4_12_22" class="NLM_sec NLM_sec_level_2"><div id="ac_i49" class="anchor-spacer"></div><h3 class="article-section__title" id="_i49"> <b>22</b></h3><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/medium/jm-2015-015277_0035.gif" alt="" id="fx22" /></img><div class="NLM_p last">Molecular weight: 273.28. 4-Methoxybenzaldehyde (0.25 mmol, 30.4 μL), 5-aminosalicylate (0.12 mmol, 18 mg), NaCNBH<sub>3</sub> (0.32 mmol, 20 mg), and 2 mL of EtOH were combined in a round-bottom flask fitted with a reflux condenser. The mixture was heated for 2 h at 100 °C, then cooled to ambient temperature and acidified with 2 M HCl. HPLC purification (0.1% TFA, acetonitrile/water gradient of 0–5 min, 10% B; 5–30 min, 10–80% B), followed by freeze-drying, yielded 46.9 mg of white solid (0.17 mmol, 86%). LCMS [M – H]<sup>−</sup><i>m</i>/<i>z</i> = 272.19. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, ppm) δ 7.56 (s, 1H), 7.32 (dd, <i>J</i> = 8.5, 3.4 Hz, 3H), 7.00–6.87 (m, 3H), 4.36 (d, <i>J</i> = 3.0 Hz, 2H), 3.73 (d, <i>J</i> = 3.8 Hz, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>, ppm) δ 171.3, 159.3, 158.9, 158.1, 130.8, 127.5, 126.3, 120.6, 118.2, 113.9, 113.5, 55.2, 51.6.</div></div><div id="sec4_12_23" class="NLM_sec NLM_sec_level_2"><div id="ac_i50" class="anchor-spacer"></div><h3 class="article-section__title" id="_i50"> <b>23</b></h3><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/medium/jm-2015-015277_0036.gif" alt="" id="fx23" /></img><div class="NLM_p last">Molecular weight: 282.29. 5-Aminosalicylate (0.2 mmol, 31 mg), 1<i>H</i>-indole-7-carbaldehyde (0.2 mmol, 29 mg) (Acros), and 2 mL of EtOH were combined in a round-bottom flask fitted with a reflux condenser. The mixture was heated for 1 h at 100 °C, then sodium borohydride (0.4 mmol, 15 mg) was added and the mixture was maintained at 100 °C for 3 h. The reaction was acidified with 2 M HCl. HPLC purification (0.1% TFA, acetonitrile/water gradient of 0–5 min, 10% B; 5–30 min, 10–80% B), followed by freeze-drying, yielded 14.3 mg of yellow solid (0.051 mmol, 25%). LCMS [M – H]<sup>−</sup><i>m</i>/<i>z</i> = 281.21. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, ppm) δ 11.24 (s, 1H), 7.51 (d, <i>J</i> = 7.8 Hz, 2H), 7.42 (s, 1H), 7.41–7.38 (m, 2H), 7.23 (d, <i>J</i> = 7.9 Hz, 2H), 7.16 (d, <i>J</i> = 7.0 Hz, 2H), 6.99 (t, <i>J</i> = 7.6 Hz, 2H), 6.88 (d, <i>J</i> = 8.7 Hz, 1H), 6.48 (dd, <i>J</i> = 2.9, 1.5 Hz, 1H), 4.61 (s, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>, ppm) δ 171.3, 162.0, 134.2, 127.8, 125.2, 121.0, 120.7, 119.6, 118.6, 117.6, 117.2, 114.3, 112.7, 101.3, 93.8, 60.6.</div></div><div id="sec4_12_24" class="NLM_sec NLM_sec_level_2"><div id="ac_i51" class="anchor-spacer"></div><h3 class="article-section__title" id="_i51"> <b>24</b></h3><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/medium/jm-2015-015277_0037.gif" alt="" id="fx24" /></img><div class="NLM_p last">Molecular weight 322.15. 5-Aminosalicylate (0.2 mmol, 31 mg), 3-bromobenzaldehyde (0.2 mmol, 23.3 μL), and 2 mL of EtOH were added in a round-bottom flask fitted with a reflux condenser. The mixture was heated for 1 h at 100 °C, then sodium borohydride (0.4 mmol, 15 mg) was added and the mixture was maintained at 100 °C for 3 h. The reaction was acidified with 2 M HCl. HPLC purification (0.1% TFA, acetonitrile/water gradient of 0–5 min, 10% B; 5–20 min, 10–35% B; 20–30 min, 35–75% B), followed by freeze-drying, yielded 23.7 mg of orange solid (0.074 mmol, 37%). LCMS [M + H]<sup>+</sup><i>m</i>/<i>z</i> = 322.09, 324.08. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, ppm) δ 7.66 (s,1H), 7.50 (d, <i>J</i> = 7.9 Hz, 1H), 7.46–7.39 (m, 1H), 7.39–7.22 (m, 2H), 7.05 (d, <i>J</i> = 27.5 Hz, 1H), 6.76 (dd, <i>J</i> = 8.6, 3.2 Hz, 1H), 4.37 (s, 2H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>, ppm) δ 171.9, 130.6, 130.5, 130.1, 129.7, 126.6, 121.7, 121.7, 117.6, 112.6, 47.2.</div></div><div id="sec4_12_25" class="NLM_sec NLM_sec_level_2"><div id="ac_i52" class="anchor-spacer"></div><h3 class="article-section__title" id="_i52"> <b>25</b></h3><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/medium/jm-2015-015277_0038.gif" alt="" id="fx25" /></img><div class="NLM_p last">Molecular weight 277.70. 5-Aminosalicylate (0.2 mmol, 31 mg), 2-chlorobenzaldehyde (0.2 mmol, 22.5 μL), and 2 mL of EtOH were combined in a round-bottom flask fitted with a reflux condenser. The mixture was heated for 1 h at 100 °C, then sodium borohydride (0.4 mmol, 15 mg) was added and the mixture was maintained at 100 °C for 3 h. The reaction was acidified with 2 M HCl. HPLC purification (0.1% TFA, acetonitrile/water gradient of 0–5 min, 10% B; 5–10 min, 10–35% B; 10–30 min, 35–80% B), followed by freeze-drying, yielded 39.2 mg of white solid (0.14 mmol, 70%). LCMS [M + H]<sup>+</sup><i>m</i>/<i>z</i> = 276.13, 278.13. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, ppm) δ 10.53 (s, 1H), 7.48–7.38 (m, 2H), 7.34–7.25 (m, 2H), 6.95 (d, <i>J</i> = 2.9 Hz, 1H), 6.88 (dd, <i>J</i> = 8.8, 3.0 Hz, 1H), 6.76 (dd, <i>J</i> = 8.8, 0.4 Hz, 1H), 4.30 (s, 2H).</div></div><div id="sec4_12_26" class="NLM_sec NLM_sec_level_2"><div id="ac_i53" class="anchor-spacer"></div><h3 class="article-section__title" id="_i53"> <b>26</b></h3><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/medium/jm-2015-015277_0039.gif" alt="" id="fx26" /></img><div class="NLM_p last">Molecular weight: 366.16. 5-Aminosalicylate (0.065 mmol, 10 mg), 6-bromo-1,3-benzodioxole-5-carboxaldehyde (0.13 mmol, 30 mg), 1 mL of EtOH, and 100 μL of acetic acid were combined in a round-bottom flask fitted with a reflux condenser. The mixture was stirred for 2 h at 100 °C, then sodium borohydride (0.26 mmol, 10 mg) was added and the mixture was stirred at 100 °C for another 30 min. HPLC purification (0.1% TFA, methanol/water gradient of 0–8 min, 5% B; 8–22 min, 5–75% B; 22–34 min, 100% B), followed by freeze-drying, yielded 9.7 mg of product (0.027 mmol, 41%). LCMS [M – H]<sup>−</sup><i>m</i>/<i>z</i> = 364.00, 365.95. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, ppm) δ 10.53 (s, 1H), 7.21 (d, <i>J</i> = 0.5 Hz, 1H), 6.94 (d, <i>J</i> = 4.6 Hz, 2H), 6.85 (d, <i>J</i> = 2.9 Hz, 1H), 6.77 (s, 1H), 6.03 (s, 2H), 4.15 (s, 2H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>, ppm) δ 171.9, 153.3, 147.3, 147.2, 131.5, 122.1, 117.6, 112.8, 112.6, 112.3, 112.3, 108.7, 102.0, 101.9, 47.5.</div></div><div id="sec4_12_27" class="NLM_sec NLM_sec_level_2"><div id="ac_i54" class="anchor-spacer"></div><h3 class="article-section__title" id="_i54"> <b>27</b></h3><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/medium/jm-2015-015277_0040.gif" alt="" id="fx27" /></img><div class="NLM_p last">Molecular weight: 321.71. 5-Aminosalicylate (0.065 mmol, 10 mg), 6-chloro-1,3-benzodioxole-5-carboxaldehyde (0.13 mmol, 24 mg), 1 mL of EtOH, and 100 μL of acetic acid were combined in a round-bottom flask fitted with a reflux condenser. The mixture was stirred for 2 h at 100 °C, then sodium borohydride (0.26 mmol, 10 mg) was added and mixture was stirred for 30 min at 100 °C. HPLC purification (0.1% TFA, methanol/water gradient of 0–8 min, 5% B; 8–22 min, 5–75% B; 22–34 min, 100% B) was followed by freeze-drying, yielding 10.1 mg of product (0.031 mmol, 48%). LCMS [M – H]<sup>−</sup><i>m</i>/<i>z</i> = 320.08. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, ppm) δ 10.55 (s, 1H), 7.07 (s, 1H), 6.96 (d, <i>J</i> = 2.9 Hz, 1H), 6.94 (s, 1H), 6.87 (dd, <i>J</i> = 8.8, 2.9 Hz, 1H), 6.76 (d, <i>J</i> = 8.9 Hz, 1H), 6.03 (d, <i>J</i> = 0.4 Hz, 2H), 4.19 (s, 2H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>, ppm) δ 171.9, 153.3, 147.0, 146.7, 140.1, 129.9, 123.8, 122.2, 117.6, 112.6, 111.9, 109.5, 108.5, 101.9, 45.0.</div></div><div id="sec4_12_28" class="NLM_sec NLM_sec_level_2"><div id="ac_i55" class="anchor-spacer"></div><h3 class="article-section__title" id="_i55"> <b>28</b></h3><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/medium/jm-2015-015277_0041.gif" alt="" id="fx28" /></img><div class="NLM_p last">Molecular weight: 287.27. 5-Aminosalicylate (0.4 mmol, 61 mg), 2,3-(methylenedioxy)benzaldehyde (0.4 mmol, 45.8 μL) (Combi-Blocks), and 4 mL of EtOH were combined in a round-bottom flask fitted with a reflux condenser. The mixture was stirred for 30 min at 100 °C, then sodium borohydride (0.8 mol, 30 mg) was added and stirring was continued for 30 min at 100 °C. HPLC purification (0.1% TFA, acetonitrile/water gradient of 0–10 min, 20% B; 10–40 min, 20–100% B; 40–50 min, 100% B) was followed by freeze-drying, yielding 85.3 mg of product (0.30 mmol, 74%). LCMS [M – H]<sup>−</sup><i>m</i>/<i>z</i> = 286.15. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, ppm) δ 7.31 (s, 1H), 7.12–7.05 (m, 1H), 6.98–6.73 (m, 5H), 6.01 (d, <i>J</i> = 2.4 Hz, 2H), 4.31 (s, 2H), 4.18 (s, 1H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>, ppm) δ 172.0, 153.5, 146.7, 145.1, 122.5, 121.5, 121.4, 121.2, 120.9, 117.6, 112.6, 107.3, 107.1, 100.8, 57.2, 41.8</div></div><div id="sec4_12_29" class="NLM_sec NLM_sec_level_2"><div id="ac_i56" class="anchor-spacer"></div><h3 class="article-section__title" id="_i56"> <b>29</b></h3><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/medium/jm-2015-015277_0042.gif" alt="" id="fx29" /></img><div class="NLM_p last">Purchased from ChemBridge.</div></div><div id="sec4_12_30" class="NLM_sec NLM_sec_level_2"><div id="ac_i57" class="anchor-spacer"></div><h3 class="article-section__title" id="_i57"> <b>30</b></h3><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/medium/jm-2015-015277_0043.gif" alt="" id="fx30" /></img><div class="NLM_p last">Molecular weight: 230.22. See section “<a class="ref internalNav" href="#sec4_9_1" aria-label="Synthesis of Hybrid Scaffold. 2-(5-Amino-2-hydroxyphenyl)isonicotinic Acid (30)">Synthesis of Hybrid Scaffold. 2-(5-Amino-2-hydroxyphenyl)isonicotinic Acid (30)</a>”.</div></div><div id="sec4_12_31" class="NLM_sec NLM_sec_level_2"><div id="ac_i58" class="anchor-spacer"></div><h3 class="article-section__title" id="_i58"> <b>31</b></h3><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/medium/jm-2015-015277_0044.gif" alt="" id="fx31" /></img><div class="NLM_p last">Molecular weight: 244.25. This compound was purified as a side product of the reaction to produce compound <b>30</b>. LCMS [M + H]<sup>+</sup><i>m</i>/<i>z</i> = 245.19. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, ppm) δ 8.86 (dd, <i>J</i> = 5.0, 0.9 Hz, 1H), 8.38 (dd, <i>J</i> = 1.6, 0.9 Hz, 1H), 7.78 (dd, <i>J</i> = 5.0, 1.6 Hz, 1H), 7.76 (d, <i>J</i> = 2.7 Hz, 1H), 7.31 (dd, <i>J</i> = 8.8, 2.7 Hz, 1H), 7.25 (d, <i>J</i> = 8.9 Hz, 1H), 3.88 (s, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>, ppm) δ 166.3, 154.9, 154.7, 150.4, 138.3, 127.9, 123.6, 123.3, 123.2, 121.4, 113.5, 56.3.</div></div><div id="sec4_12_32" class="NLM_sec NLM_sec_level_2"><div id="ac_i59" class="anchor-spacer"></div><h3 class="article-section__title" id="_i59"> <b>32</b></h3><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/medium/jm-2015-015277_0045.gif" alt="" id="fx32" /></img><div class="NLM_p last">Purchased from Sigma.</div></div><div id="sec4_12_33" class="NLM_sec NLM_sec_level_2"><div id="ac_i60" class="anchor-spacer"></div><h3 class="article-section__title" id="_i60"> <b>33</b></h3><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/medium/jm-2015-015277_0046.gif" alt="" id="fx33" /></img><div class="NLM_p last">Molecular weight: 350.32. 2-(Acetoxy)benzoyl chloride (0.071 mmol, 14 mg), <i>N</i>-hydroxysuccinimide (0.13 mmol, 15 mg), and 0.5 mL of acetonitrile were added to a glass vial. The mixture was stirred at ambient temperature for 1 h. Compound <b>30</b> (0.013 mmol, 3 mg) in 0.5 mL of 1:1:1 water/acetonitrile/methanol was then added, and the mixture was stirred at ambient temperature for 16 h. The intermediate was isolated by HPLC (0.1% TFA, MeOH/water gradient of 0–5 min, 15% B; 5–10 min, 15–40% B; 10–35 min, 40–80% B), followed by concentration at reduced pressure. The concentrate was transferred to a sealed pressure vial with 0.5 mL of 2 M NaOH and heated to 100 °C for 1 h. The reaction was quenched with 2 M HCl. The final product was HPLC purified (0.1% TFA, MeOH/water gradient of 0–5 min, 15% B; 5–10 min, 15–50% B; 10–35 min, 50–100% B), followed by freeze-drying, yielding 0.9 mg of yellow solid (0.003 mmol, 20%). LCMS [M + H]<sup>+</sup><i>m</i>/<i>z</i> = 351.26. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, ppm) δ 10.35 (s, 1H), 8.84 (d, <i>J</i> = 5.4 Hz, 1H), 8.50 (s, 1H), 8.37 (s, 1H), 8.00 (d, <i>J</i> = 7.9 Hz, 1H), 7.84 (d, <i>J</i> = 5.3 Hz, 1H), 7.72 (dd, <i>J</i> = 8.2, 1.5 Hz, 1H), 7.44 (s, 1H), 7.00 (s, 1H), 6.99 (d, <i>J</i> = 2.8 Hz, 1H), 6.97 (s, 1H).</div></div><div id="sec4_12_34" class="NLM_sec NLM_sec_level_2"><div id="ac_i61" class="anchor-spacer"></div><h3 class="article-section__title" id="_i61"> <b>34</b></h3><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/medium/jm-2015-015277_0047.gif" alt="" id="fx34" /></img><div class="NLM_p last">Molecular weight: 350.32. 3-(Acetoxy)benzoyl chloride (0.071 mmol, 14 mg), <i>N</i>-hydroxysuccinimide (0.13 mmol, 15 mg), and 0.5 mL of acetonitrile were added to a glass vial, and the mixture was stirred at ambient temperature for 1 h. Compound <b>30</b> (0.013 mmol, 3 mg) in 0.5 mL of 1:1:1 water/acetonitrile/methanol was then added, and stirring was continued at ambient temperature for 16 h. The reaction mixture was then transferred to a sealed pressure vial with 0.5 mL of 2 M NaOH and heated to 100 °C for 1 h. The reaction was quenched with 2 M HCl. HPLC purification (0.1% TFA, MeOH/water gradient of 0–5 min, 15% B; 5–10 min, 15–50% B; 10–35 min, 50–100% B), followed by freeze-drying, yielded 1.9 mg of yellow solid (0.005 mmol, 32%). LCMS [M + H]<sup>+</sup><i>m</i>/<i>z</i> = 351.16. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, ppm) δ 10.11 (s, 1H), 9.74 (s, 1H), 8.83 (dd, <i>J</i> = 5.3, 1.1 Hz, 1H), 8.45 (d, <i>J</i> = 16.9 Hz, 1H), 7.88–7.80 (m, 1H), 7.45–7.39 (m, 1H), 7.38–7.27 (m, 2H), 6.97 (ddt, <i>J</i> = 4.9, 2.6, 1.3 Hz, 2H), 6.95 (d, <i>J</i> = 1.2 Hz, 1H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>, ppm) δ 165.9, 165.2, 157.4, 154.7, 148.2, 140.1, 136.4, 131.3, 129.4, 124.5, 121.1, 119.3. 119.1, 118.8, 118.4, 118.1, 117.7, 114.4.</div></div><div id="sec4_12_35" class="NLM_sec NLM_sec_level_2"><div id="ac_i62" class="anchor-spacer"></div><h3 class="article-section__title" id="_i62"> <b>35</b></h3><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/medium/jm-2015-015277_0048.gif" alt="" id="fx35" /></img><div class="NLM_p last">Molecular weight: 350.32. 4-(Acetoxy)benzoyl chloride (0.071 mmol, 14 mg) and 0.5 mL of acetonitrile were added to a glass vial, and the mixture was stirred at ambient temperature for 1 h. Compound <b>30</b> (0.013 mmol, 3 mg) in 0.5 mL of 1:1:1 water/acetonitrile/methanol was then added, and the mixture was stirred at ambient temperature for 16 h. The reaction mixture was then transferred to a sealed pressure vial with 0.5 mL of 2 M NaOH and heated to 100 °C for 1 h. The reaction was quenched with 2 M HCl. HPLC purification (0.1% TFA, MeOH/water gradient of 0–5 min, 15% B; 5–10 min, 15–50% B; 10–35 min, 50–100% B), followed by freeze-drying, yielded 2.0 mg of yellow solid (0.006 mmol, 34%). LCMS [M + H]<sup>+</sup><i>m</i>/<i>z</i> = 351.18. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, ppm) δ 12.99 (s, 1H), 10.08 (s, 1H), 9.95 (s, 1H), 8.83 (dd, <i>J</i> = 5.1, 0.8 Hz, 1H), 8.53–8.35 (m, 2H), 7.87 (d, <i>J</i> = 8.7 Hz, 2H), 7.85–7.80 (m, 2H), 6.95 (d, <i>J</i> = 8.8 Hz, 1H), 6.87 (d, <i>J</i> = 8.7 Hz, 2H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>, ppm) δ 165.9, 164.8, 160.4, 157.4, 154.5, 148.1, 140.1, 131.5, 129.5 (2C), 125.4, 124.5, 121.1, 119.2, 118.9, 118.7, 117.7, 114.9 (2C).</div></div><div id="sec4_12_36" class="NLM_sec NLM_sec_level_2"><div id="ac_i63" class="anchor-spacer"></div><h3 class="article-section__title" id="_i63"> <b>36</b></h3><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/medium/jm-2015-015277_0049.gif" alt="" id="fx36" /></img><div class="NLM_p last">Molecular weight: 339.30. 5-Methylisoxazole-3-carboxyl chloride (0.071 mmol, 10 mg) (Oakwood Research), <i>N</i>-hydroxysuccinimide (0.13 mmol, 15 mg), and 0.5 mL of acetonitrile were added to a glass vial, and the mixture was stirred at ambient temperature for 1 h. Compound <b>30</b> (0.013 mmol, 3 mg) in 0.5 mL of 1:1:1 water/acetonitrile/methanol was then added, and the mixture was stirred at ambient temperature for an additional 16 h. HPLC purification (0.1% TFA, MeOH/water gradient of 0–10 min, 15% B; 10–15 min, 15–40% B; 15–35 min, 40–80% B) was followed by freeze-drying, yielding 1.5 mg of yellow solid (0.004 mmol, 40%). LCMS [M + H]<sup>+</sup><i>m</i>/<i>z</i> = 340.16. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, ppm) δ 13.16 (s, 1H), 10.57 (s, 1H), 8.82 (d, <i>J</i> = 5.1 Hz, 1H), 8.49 (d, <i>J</i> = 2.5 Hz, 1H), 8.46 (s, 1H), 7.85 (d, <i>J</i> = 2.5 Hz, 1H), 7.85–7.81 (m, 2H), 6.97 (d, <i>J</i> = 8.8 Hz, 1H), 6.67 (d, <i>J</i> = 1.1 Hz, 1H), 2.54 (s, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>, ppm) δ 171.4, 165.9, 159.3, 157.1, 157.0, 155.3, 148.0, 130.1, 124.4, 121.3, 119.3, 119.3, 118.8, 117.8, 101.6, 11.9.</div></div><div id="sec4_12_37" class="NLM_sec NLM_sec_level_2"><div id="ac_i64" class="anchor-spacer"></div><h3 class="article-section__title" id="_i64"> <b>37</b></h3><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/medium/jm-2015-015277_0050.gif" alt="" id="fx37" /></img><div class="NLM_p last">Molecular weight: 348.35. Phenylacetyl chloride (0.071 mmol, 9.4 μL) (Sigma), <i>N</i>-hydroxysuccinimide (0.13 mmol, 15 mg), and 0.5 mL of acetonitrile were added to a glass vial, and the mixture was stirred at ambient temperature for 1 h. Compound <b>30</b> (0.013 mmol, 3 mg) in 0.5 mL of 1:1:1 water/acetonitrile/methanol was then added, and stirring was continued at ambient temperature for 16 h. HPLC purification (0.1% TFA, MeOH/water gradient of 0–10 min, 15% B; 10–15 min, 15–50% B; 15–35 min, 50–90% B) was followed by freeze-drying, yielding 0.3 mg of yellow solid (0.001 mmol, 8%). LCMS [M + H]<sup>+</sup><i>m</i>/<i>z</i> = 349.18. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, ppm) δ 10.11 (s, 1H), 8.81 (d, <i>J</i> = 5.2 Hz, 1H), 8.40 (s, 1H), 8.28 (d, <i>J</i> = 2.7 Hz, 1H), 7.84–7.75 (m, 1H), 7.62–7.52 (m, 1H), 7.34 (d, <i>J</i> = 6.7 Hz, 3H), 7.21 (s, 1H), 7.08 (s, 1H), 7.00–6.86 (m, 2H), 3.62 (s, 2H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>, ppm) δ 168.7, 165.9, 157.2, 154.3, 148.3, 139.8, 136.1, 131.4, 129.1 (2C), 128.3 (2C), 126.5, 123.2, 121.1, 119.4, 119.1, 117.9, 117.9, 43.4.</div></div><div id="sec4_12_38" class="NLM_sec NLM_sec_level_2"><div id="ac_i65" class="anchor-spacer"></div><h3 class="article-section__title" id="_i65"> <b>38</b></h3><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/medium/jm-2015-015277_0051.gif" alt="" id="fx38" /></img><div class="NLM_p last">Molecular weight: 427.25. 2-(3-Bromophenyl)acetyl chloride (0.71 mmol, 17 mg), <i>N</i>-hydroxysuccinimide (0.13 mmol, 15 mg), and 0.5 mL of acetonitrile were added to a glass vial, and the mixture was stirred at ambient temperature for 1 h. Compound <b>30</b> (0.013 mmol, 3 mg) in 0.5 mL of 1:1:1 water/acetonitrile/methanol was then added, and the mixture was stirred at ambient temperature for 16 h. HPLC purification (0.1% TFA, MeOH/water gradient of 0–10 min, 15% B; 10–15 min, 15–50% B; 15–35 min, 50–90% B), followed by freeze-drying, yielded 0.6 mg of yellow solid (0.001 mmol, 13%). LCMS [M + H]<sup>+</sup><i>m</i>/<i>z</i> = 427.09, 429.05. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, ppm) δ 10.13 (d, <i>J</i> = 1.5 Hz, 1H), 8.81 (dd, <i>J</i> = 5.3, 1.3 Hz, 1H), 8.41 (s, 1H), 8.27 (s, 1H), 7.84–7.77 (m, 1H), 7.62–7.51 (m, 2H), 7.50–7.42 (m, 1H), 7.39–7.25 (m, 2H), 6.92 (dd, <i>J</i> = 8.8, 1.5 Hz, 1H), 3.65 (d, <i>J</i> = 1.6 Hz, 2H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>, ppm) δ 219.0, 168.1, 165.9, 157.1, 154.3, 148.3, 139.9, 138.7, 131.9, 131.2, 130.5, 129.4, 128.3, 123.2, 121.5, 121.1, 119.4, 118.0, 117.9, 42.6.</div></div><div id="sec4_12_39" class="NLM_sec NLM_sec_level_2"><div id="ac_i66" class="anchor-spacer"></div><h3 class="article-section__title" id="_i66"> <b>39</b></h3><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/medium/jm-2015-015277_0052.gif" alt="" id="fx39" /></img><div class="NLM_p last">Molecular weight: 378.38. 2-(4-Methoxyphenyl)acetyl chloride (0.071 mmol, 10.9 μL), <i>N</i>-hydroxysuccinimide (0.13 mmol, 15 mg), and 0.5 mL of acetonitrile were added to a glass vial, and the mixture was stirred at ambient temperature for 1 h. Compound <b>30</b> (0.013 mmol, 3 mg) in 0.5 mL of 1:1:1 water/acetonitrile/methanol was then added, and the mixture was stirred at ambient temperature for 16 h. HPLC purification (0.1% TFA, MeOH/water gradient of 0–10 min, 15% B; 10–15 min, 15–50% B; 15–35 min, 50–90% B) was followed by freeze-drying, yielding 1.1 mg of yellow solid (0.003 mmol, 29%). LCMS [M + H]<sup>+</sup><i>m</i>/<i>z</i> = 379. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, ppm) δ 10.05 (s, 1H), 8.87–8.76 (m, 1H), 8.41 (s, 1H), 8.27 (s, 1H), 7.82 (d, <i>J</i> = 5.3 Hz, 1H), 7.58 (d, <i>J</i> = 8.5 Hz, 1H), 7.29–7.24 (m, 2H), 6.94–6.86 (m, 2H), 3.73 (d, <i>J</i> = 1.9 Hz, 3H), 3.54 (d, <i>J</i> = 1.3 Hz, 2H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>, ppm) δ 169.1, 165.9, 158.0, 157.2, 154.2, 148.2, 131.4, 130.1 (2C), 128.0, 123.2, 121.1, 119.4, 119.0, 117.8, 113.8 (2C), 55.0, 42.5.</div></div><div id="sec4_12_40" class="NLM_sec NLM_sec_level_2"><div id="ac_i67" class="anchor-spacer"></div><h3 class="article-section__title" id="_i67"> <b>40</b></h3><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/medium/jm-2015-015277_0053.gif" alt="" id="fx40" /></img><div class="NLM_p last">Molecular weight: 378.38. 2-(3-Methoxyphenyl)acetyl chloride (0.071 mmol, 11.1 μL), <i>N</i>-hydroxysuccinimide (0.13 mmol, 15 mg), and 0.5 mL of acetonitrile were added to a glass vial, and the mixture was stirred at ambient temperature for 1 h. Compound <b>30</b> (0.013 mmol, 3 mg) in 0.5 mL of 1:1:1 water/acetonitrile/methanol was then added, and the mixture was stirred at ambient temperature for 16 h. HPLC purification (0.1% TFA, MeOH/water gradient of 0–5 min, 15% B; 5–10 min, 15–50% B; 10–35 min, 50–100% B) was followed by freeze-drying, yielding 1.6 mg of yellow solid (0.004 mmol, 25%). LCMS [M + H]<sup>+</sup><i>m</i>/<i>z</i> = 379.20. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, ppm) δ 10.09 (s, 1H), 8.81 (d, <i>J</i> = 5.2 Hz, 1H), 8.41 (d, <i>J</i> = 1.3 Hz, 1H), 8.27 (d, <i>J</i> = 2.6 Hz, 1H), 7.81 (dd, <i>J</i> = 5.2, 1.4 Hz, 1H), 7.59 (dd, <i>J</i> = 8.9, 2.5 Hz, 1H), 7.23 (d, <i>J</i> = 8.1 Hz, 1H), 6.94–6.91 (m, 2H), 6.91 (s, 1H), 6.85–6.79 (m, 1H), 3.75 (s, 3H), 3.59 (s, 2H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>, ppm) δ 169.0, 166.3, 159.7, 157.6, 154.8, 138.0, 131.8, 129.8, 123.7, 121.8, 121.6, 119.8, 119.5, 118.3, 118.3, 115.4, 112.4, 55.5, 43.9.</div></div><div id="sec4_12_41" class="NLM_sec NLM_sec_level_2"><div id="ac_i68" class="anchor-spacer"></div><h3 class="article-section__title" id="_i68"> <b>41</b></h3><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/medium/jm-2015-015277_0054.gif" alt="" id="fx41" /></img><div class="NLM_p last">Molecular weight: 325.28. Isoxazole-5-carbonyl chloride (0.071 mmol, 9.3 mg) (Maybridge), <i>N</i>-hydroxysuccinimide (0.13 mmol, 15 mg), and 0.5 mL of acetonitrile were added to a glass vial, and the mixture was stirred at ambient temperature for 1 h. Compound <b>30</b> (0.013 mmol, 3 mg) in 0.5 mL of 1:1:1 water/acetonitrile/methanol was then added, and the mixture was stirred at ambient temperature for 1 h. HPLC purification (0.1% TFA, MeOH/water gradient of 0–5 min, 15% B; 5–10 min, 15–40% B; 10–35 min, 40–100% B), followed by freeze-drying, yielded 1.0 mg of orange solid (0.004 mmol, 33%). LCMS [M + H]<sup>+</sup><i>m</i>/<i>z</i> = 326.18. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, ppm) δ 10.72 (s, 1H), 8.84 (dd, <i>J</i> = 5.2, 0.8 Hz, 1H), 8.82 (d, <i>J</i> = 1.9 Hz, 1H), 8.48 (dd, <i>J</i> = 1.5, 0.9 Hz, 1H), 8.43 (d, <i>J</i> = 2.6 Hz, 1H), 7.86–7.81 (m, 2H), 7.24 (d, <i>J</i> = 1.9 Hz, 1H), 7.00 (d, <i>J</i> = 8.9 Hz, 1H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>, ppm) δ 165.9, 162.8, 157.1, 155.4, 153.8, 151.9, 148.3, 140.0, 129.8, 124.6, 121.3, 119.8, 119.5, 119.1, 118.0, 106.6.</div></div><div id="sec4_12_42" class="NLM_sec NLM_sec_level_2"><div id="ac_i69" class="anchor-spacer"></div><h3 class="article-section__title" id="_i69"> <b>42</b></h3><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/medium/jm-2015-015277_0055.gif" alt="" id="fx42" /></img><div class="NLM_p last">Molecular weight: 302.28. 2-Methoxyacetyl chloride (0.071 mmol, 6.5 μL), <i>N</i>-hydroxysuccinimide (0.13 mmol, 15 mg), and 0.5 mL of acetonitrile were added to a glass vial, and the mixture was stirred at ambient temperature for 1 h. Compound <b>30</b> (0.013 mmol, 3 mg) in 0.5 mL of 1:1:1 water/acetonitrile/methanol was then added, and the mixture was stirred at ambient temperature for 1 h. HPLC purification (0.1% TFA, MeOH/water gradient of 0–10 min, 15% B; 10–15 min, 15–40% B; 15–35 min, 40–80% B), followed by freeze-drying, yielded 1.5 mg of yellow solid (0.005 mmol, 50%). LCMS [M + H]<sup>+</sup><i>m</i>/<i>z</i> = 303.22. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, ppm) δ 9.70 (s, 1H), 8.82 (d, <i>J</i> = 5.2 Hz, 1H), 8.45 (s, 1H), 8.34 (d, <i>J</i> = 2.5 Hz, 1H), 7.83 (dd, <i>J</i> = 5.2, 1.5 Hz, 1H), 7.74 (dd, <i>J</i> = 9.1, 2.3 Hz, 1H), 6.92 (d, <i>J</i> = 8.8 Hz, 1H), 3.99 (s, 2H), 3.40 (s, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>, ppm) δ 167.7, 165.9, 157.3, 154.7, 148.1, 140.0, 130.5, 124.0, 121.1, 119.3, 118.7, 118.7, 117.7, 71.8, 58.7.</div></div><div id="sec4_12_43" class="NLM_sec NLM_sec_level_2"><div id="ac_i70" class="anchor-spacer"></div><h3 class="article-section__title" id="_i70"> <b>43</b></h3><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/medium/jm-2015-015277_0056.gif" alt="" id="fx43" /></img><div class="NLM_p last">Molecular weight: 240.21. Methyl 2-bromo-4-pyridinecarboxylate (0.19 mmol, 30 mg), 5-cyano-2-hydroxyphenylboronic acid (0.28 mmol, 61 mg) (Combi-Blocks), cesium carbonate (0.70 mmol, 228 mg), Pd6 mixed catalyst (0.014 mmol, 7 mg) (CombiPhos), and 2 mL of DMF were added to a sealed pressure vial. The mixture was then heated to 120 °C for 5 h. Upon cooling to ambient temperature, the mixture was acidified with 2 M HCl, extracted with EtOAc, washed with brine, and concentrated. The product was isolated by HPLC purification (0.1% TFA, acetonitrile/water gradient of 0–5 min, 10% B; 5–20 min, 10–40% B; 20–30 min, 40–100% B), followed by freeze-drying, yielding 3.0 mg of gray solid (0.013 mmol, 7%), and the unhydrolyzed methyl ester was not collected. LCMS [M + H]<sup>+</sup><i>m</i>/<i>z</i> = 241.18. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, ppm) δ 8.85 (d, <i>J</i> = 5.1 Hz, 1H), 8.68 (s, 1H), 8.62 (d, <i>J</i> = 2.1 Hz, 1H), 7.88 (dd, <i>J</i> = 5.2, 0.9 Hz, 1H), 7.76 (dd, <i>J</i> = 8.5, 2.1 Hz, 1H), 7.12 (d, <i>J</i> = 8.5 Hz, 1H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>, ppm) δ 166.0, 162.2, 155.7, 148.4, 140.4, 139.9, 135.1, 133.3, 122.2, 121.2, 120.9, 119.1, 101.9.</div></div><div id="sec4_12_44" class="NLM_sec NLM_sec_level_2"><div id="ac_i71" class="anchor-spacer"></div><h3 class="article-section__title" id="_i71"> <b>44</b></h3><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/medium/jm-2015-015277_0057.gif" alt="" id="fx44" /></img><div class="NLM_p last">Molecular weight: 272.26. Compound <b>30</b> (2 mg, 0.009 mmol), acetic anhydride (5 μL, 0.053 mmol, Sigma), and 250 μL of water were added to a glass vial. The reaction was stirred at ambient temperature for 30 min and then diluted in HPLC solvent. HPLC purification (0.1% TFA, MeOH/water gradient of 0–5 min, 15% B; 5–10 min, 15–50% B; 10–35 min, 50–90% B) was followed by freeze-drying, yielding 2.2 mg of yellow solid (0.008 mmol, 90%). LCMS [M + H]<sup>+</sup><i>m</i>/<i>z</i> = 273.57. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.86 (s, 1H), 8.82 (dd, <i>J</i> = 5.2, 0.8 Hz, 1H), 8.41 (dd, <i>J</i> = 1.5, 0.9 Hz, 1H), 8.23 (d, <i>J</i> = 2.6 Hz, 1H), 7.82 (dd, <i>J</i> = 5.1, 1.4 Hz, 1H), 7.57 (dd, <i>J</i> = 8.8, 2.6 Hz, 1H), 6.91 (d, <i>J</i> = 8.8 Hz, 1H), 2.03 (s, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.9, 165.9, 157.2, 154.1, 148.2, 139.9, 131.5, 123.3, 121.1, 119.4, 118.9, 117.9, 117.8, 23.9.</div></div><div id="sec4_12_45" class="NLM_sec NLM_sec_level_2"><div id="ac_i72" class="anchor-spacer"></div><h3 class="article-section__title" id="_i72"> <b>45</b></h3><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/medium/jm-2015-015277_0058.gif" alt="" id="fx45" /></img><div class="NLM_p last">Molecular weight: 324.33. Compound <b>30</b> (0.017 mmol, 4 mg), 1-methyl-1<i>H</i>-pyrazole-5-carbaldehyde (0.034 mmol, 3.3 μL), and 1 mL of EtOH were combined in a round-bottom flask fitted with a reflux condenser. The mixture was stirred for 2 h at 100 °C, then sodium borohydride (0.068 mmol, 2.6 mg) was added and the mixture was stirred for 30 min at 100 °C. HPLC purification (0.1% TFA, methanol/water gradient of 0–8 min, 5% B; 5–22 min, 5–75% B; 22–34 min, 100% B), followed by freeze-drying, yielded 1.8 mg of product (0.00356 mmol, 33%). LCMS [M + H]<sup>+</sup><i>m</i>/<i>z</i> = 325.16. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD, ppm) δ 8.79 (dd, <i>J</i> = 5.3, 0.9 Hz, 1H), 8.53 (t, <i>J</i> = 1.2 Hz, 1H), 7.98 (dd, <i>J</i> = 5.3, 1.4 Hz, 1H), 7.76 (d, <i>J</i> = 2.7 Hz, 1H), 7.53 (d, <i>J</i> = 2.0 Hz, 1H), 7.23 (dd, <i>J</i> = 8.8, 2.7 Hz, 1H), 7.07 (d, <i>J</i> = 8.8 Hz, 1H), 6.51 (d, <i>J</i> = 2.0 Hz, 1H), 4.75 (s, 2H), 3.70 (s, 3H). <sup>13</sup>C NMR (126 MHz, CD<sub>3</sub>OD, ppm) δ 165.8, 157.2, 155.8, 147.0, 140.7, 138.0, 137.2, 122.4, 121.4, 119.7, 119.3, 119.0, 116.3, 107.0, 42.4, 35.2.</div></div><div id="sec4_12_46" class="NLM_sec NLM_sec_level_2"><div id="ac_i73" class="anchor-spacer"></div><h3 class="article-section__title" id="_i73"> <b>46</b></h3><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/medium/jm-2015-015277_0059.gif" alt="" id="fx46" /></img><div class="NLM_p last">Molecular weight: 364.35. Compound <b>30</b> (0.038 mmol, 9 mg), 2,3-(methylenedioxy)benzaldehyde (0.075 mmol, 8.6 μL) (Combi-Blocks), 2 mL of EtOH, and 100 μL of glacial acetic acid were combined in a round-bottom flask fitted with a reflux condenser. The mixture was stirred for 3 h at ambient temperature, then sodium borohydride (0.15 mmol, 6 mg) was added and the mixture was stirred for an additional 30 min at ambient temperature. The reaction was quenched with ∼10 drops of 2 N HCl. HPLC purification (0.1% TFA, methanol/water gradient of 0–8 min, 5% B; 5–22 min, 5–75% B; 22–34 min, 100% B) was followed by freeze-drying, yielding 6.4 mg of product (0.018 mmol, 47%). LCMS [M – H]<sup>−</sup><i>m</i>/<i>z</i> = 363.14. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, ppm) δ 8.79 (dd, <i>J</i> = 5.2, 0.8 Hz, 1H), 8.43 (d, <i>J</i> = 1.4 Hz, 1H), 7.80 (dd, <i>J</i> = 5.2, 1.4 Hz, 1H), 7.47 (s, 1H), 7.04–6.74 (m, 5H), 6.03 (s, 2H), 4.32 (s, 2H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>, ppm) δ 166.0, 158.2, 157.9, 157.5, 148.1, 146.8, 145.3, 139.9, 121.9, 121.6, 120.9, 119.3, 118.5, 117.4, 115.1, 107.6, 100.8.</div></div><div id="sec4_12_47" class="NLM_sec NLM_sec_level_2"><div id="ac_i74" class="anchor-spacer"></div><h3 class="article-section__title" id="_i74"> <b>47</b></h3><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/medium/jm-2015-015277_0060.gif" alt="" id="fx47" /></img><div class="NLM_p last">Molecular weight: 399.24. Compound <b>30</b> (0.02 mmol, 4.6 mg), 4-bromobenzaldehyde (0.08 mmol, 15 mg), and 1 mL of EtOH were combined in a round-bottom flask fitted with a reflux condenser. The mixture was stirred for 2 h at 100 °C, then sodium borohydride (0.08 mmol, 3 mg) was added and the mixture was stirred for 30 min at 100 °C. HPLC purification (0.1% TFA, methanol/water gradient of 0–8 min, 5% B; 5–22 min, 5–75% B; 22–34 min, 100% B), followed by freeze-drying, yielded 3.2 mg of product (0.008 mmol, 40%). LCMS [M + H]<sup>+</sup><i>m</i>/<i>z</i> = 399.13, 401.11. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, ppm) δ 8.79 (dd, <i>J</i> = 5.1, 0.8 Hz, 1H), 8.41 (s, 1H), 7.79 (dd, <i>J</i> = 5.2, 1.4 Hz, 1H), 7.55 (d, <i>J</i> = 8.4 Hz, 2H), 7.39 (d, <i>J</i> = 8.4 Hz, 2H), 7.26–6.95 (m, 1H), 6.83 (s, 1H), 4.36 (s, 2H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>, ppm) δ 171.9, 166.4, 158.6, 158.4, 157.9, 148.6, 140.3, 131.7 (2C), 130.8 (2C), 121.4, 120.1, 118.8, 118.1, 115.7, 23.0.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i75"><a href="/doi/suppl/10.1021/acs.jmedchem.5b01527">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_81388" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_81388" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.5b01527" class="ext-link">10.1021/acs.jmedchem.5b01527</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Experimental procedure for cloning, expression for all enzymes, MALDI and AlphaScreen counterscreening; IC<sub>50</sub> for selected 5-aminosalicylates series of inhibitors; structures of hybrid scaffolds that were virtually screened; crystallography statistics for compounds <b>43</b>, <b>30</b>, <b>44</b>, <b>42</b>, <b>35</b>, <b>40</b>, and <b>36</b>; omit maps and 2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> maps for all compounds; figures showing docking targets used, docking binding site used, and superimposition of hybrid compounds with structure <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2OQ6">2OQ6</a> showing H3 histone peptide; counterscreening confirming specific inhibition and binding of compounds through an aggregation test, DLS test, and enzymes <i>T</i><sub>m</sub> stabilization curves (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01527/suppl_file/jm5b01527_si_001.pdf" class="ext-link">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b01527/suppl_file/jm5b01527_si_001.pdf">jm5b01527_si_001.pdf (2.18 MB)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Atomic coordinates and structure factors for the KDM4A complexes have been deposited in the Protein Data Bank as entries <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5A7N">5A7N</a> in complex with compound <b>43</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5A7O">5A7O</a> with <b>42</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5A7P">5A7P</a> with <b>36</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5A7Q">5A7Q</a> with <b>30</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5A7S">5A7S</a> with <b>44</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5A7W">5A7W</a> with <b>35</b>, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5A80">5A80</a> with <b>40</b>.</p><div class="testing" data-doi="10.1021/acs.jmedchem.5b01527" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_11812" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_11812" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Udo Oppermann</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Oxford, Oxford OX3 7DQ, U.K.</span>; 
    <span class="hlFld-Affiliation affiliation">Botnar
Research Center, University of Oxford, Oxford OX3 7LD, U.K.</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#b7c2d3d899d8c7c7d2c5dad6d9d9f7c4d0d499d8cf99d6d499c2dc"><span class="__cf_email__" data-cfemail="790c1d16571609091c0b14181717390a1e1a57160157181a570c12">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Brian K. Shoichet</span> - <span class="hlFld-Affiliation affiliation">†Department of Pharmaceutical Chemistry, ‡Chemistry and Chemical
Biology
Graduate Program, and §Department of Cellular and Molecular Pharmacology, University of California, San Francisco, Genentech Hall, 600 16th Street,
MC2280, San Francisco, California 94158-2280, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Oxford, Oxford OX3 7DQ, U.K.</span>; 
    <span class="hlFld-Affiliation affiliation">Botnar
Research Center, University of Oxford, Oxford OX3 7LD, U.K.</span>; 
    <span class="hlFld-Affiliation affiliation">Nuffield
Department of Clinical Medicine, Target Discovery Institute (TDI), University of Oxford, Oxford OX3 7BN, U.K.</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#86e4f5eee9efe5eee3f2c6e1ebe7efeaa8e5e9eb"><span class="__cf_email__" data-cfemail="bedccdd6d1d7ddd6dbcafed9d3dfd7d290ddd1d3">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Danica Galonić Fujimori</span> - <span class="hlFld-Affiliation affiliation">†Department of Pharmaceutical Chemistry, ‡Chemistry and Chemical
Biology
Graduate Program, and §Department of Cellular and Molecular Pharmacology, University of California, San Francisco, Genentech Hall, 600 16th Street,
MC2280, San Francisco, California 94158-2280, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Oxford, Oxford OX3 7DQ, U.K.</span>; 
    <span class="hlFld-Affiliation affiliation">Botnar
Research Center, University of Oxford, Oxford OX3 7LD, U.K.</span>; 
    <span class="hlFld-Affiliation affiliation">Nuffield
Department of Clinical Medicine, Target Discovery Institute (TDI), University of Oxford, Oxford OX3 7BN, U.K.</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#81c5e0efe8e2e0afc7f4ebe8eceef3e8c1f4e2f2e7afe4e5f4"><span class="__cf_email__" data-cfemail="7337121d1a10125d3506191a1e1c011a33061000155d161706">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Magdalena Korczynska</span> - <span class="hlFld-Affiliation affiliation">†Department of Pharmaceutical Chemistry, ‡Chemistry and Chemical
Biology
Graduate Program, and §Department of Cellular and Molecular Pharmacology, University of California, San Francisco, Genentech Hall, 600 16th Street,
MC2280, San Francisco, California 94158-2280, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Oxford, Oxford OX3 7DQ, U.K.</span>; 
    <span class="hlFld-Affiliation affiliation">Botnar
Research Center, University of Oxford, Oxford OX3 7LD, U.K.</span>; 
    <span class="hlFld-Affiliation affiliation">Nuffield
Department of Clinical Medicine, Target Discovery Institute (TDI), University of Oxford, Oxford OX3 7BN, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Daniel D. Le</span> - <span class="hlFld-Affiliation affiliation">†Department of Pharmaceutical Chemistry, ‡Chemistry and Chemical
Biology
Graduate Program, and §Department of Cellular and Molecular Pharmacology, University of California, San Francisco, Genentech Hall, 600 16th Street,
MC2280, San Francisco, California 94158-2280, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Oxford, Oxford OX3 7DQ, U.K.</span>; 
    <span class="hlFld-Affiliation affiliation">Botnar
Research Center, University of Oxford, Oxford OX3 7LD, U.K.</span>; 
    <span class="hlFld-Affiliation affiliation">Nuffield
Department of Clinical Medicine, Target Discovery Institute (TDI), University of Oxford, Oxford OX3 7BN, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Noah Younger</span> - <span class="hlFld-Affiliation affiliation">†Department of Pharmaceutical Chemistry, ‡Chemistry and Chemical
Biology
Graduate Program, and §Department of Cellular and Molecular Pharmacology, University of California, San Francisco, Genentech Hall, 600 16th Street,
MC2280, San Francisco, California 94158-2280, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Oxford, Oxford OX3 7DQ, U.K.</span>; 
    <span class="hlFld-Affiliation affiliation">Botnar
Research Center, University of Oxford, Oxford OX3 7LD, U.K.</span>; 
    <span class="hlFld-Affiliation affiliation">Nuffield
Department of Clinical Medicine, Target Discovery Institute (TDI), University of Oxford, Oxford OX3 7BN, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Elisabet Gregori-Puigjané</span> - <span class="hlFld-Affiliation affiliation">†Department of Pharmaceutical Chemistry, ‡Chemistry and Chemical
Biology
Graduate Program, and §Department of Cellular and Molecular Pharmacology, University of California, San Francisco, Genentech Hall, 600 16th Street,
MC2280, San Francisco, California 94158-2280, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Oxford, Oxford OX3 7DQ, U.K.</span>; 
    <span class="hlFld-Affiliation affiliation">Botnar
Research Center, University of Oxford, Oxford OX3 7LD, U.K.</span>; 
    <span class="hlFld-Affiliation affiliation">Nuffield
Department of Clinical Medicine, Target Discovery Institute (TDI), University of Oxford, Oxford OX3 7BN, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Anthony Tumber</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Oxford, Oxford OX3 7DQ, U.K.</span>; 
    <span class="hlFld-Affiliation affiliation">Nuffield
Department of Clinical Medicine, Target Discovery Institute (TDI), University of Oxford, Oxford OX3 7BN, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tobias Krojer</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Oxford, Oxford OX3 7DQ, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Srikannathasan Velupillai</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Oxford, Oxford OX3 7DQ, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Carina Gileadi</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Oxford, Oxford OX3 7DQ, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Radosław P. Nowak</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Oxford, Oxford OX3 7DQ, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Eriko Iwasa</span> - <span class="hlFld-Affiliation affiliation">†Department of Pharmaceutical Chemistry, ‡Chemistry and Chemical
Biology
Graduate Program, and §Department of Cellular and Molecular Pharmacology, University of California, San Francisco, Genentech Hall, 600 16th Street,
MC2280, San Francisco, California 94158-2280, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Oxford, Oxford OX3 7DQ, U.K.</span>; 
    <span class="hlFld-Affiliation affiliation">Botnar
Research Center, University of Oxford, Oxford OX3 7LD, U.K.</span>; 
    <span class="hlFld-Affiliation affiliation">Nuffield
Department of Clinical Medicine, Target Discovery Institute (TDI), University of Oxford, Oxford OX3 7BN, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Samuel B. Pollock</span> - <span class="hlFld-Affiliation affiliation">†Department of Pharmaceutical Chemistry, ‡Chemistry and Chemical
Biology
Graduate Program, and §Department of Cellular and Molecular Pharmacology, University of California, San Francisco, Genentech Hall, 600 16th Street,
MC2280, San Francisco, California 94158-2280, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Oxford, Oxford OX3 7DQ, U.K.</span>; 
    <span class="hlFld-Affiliation affiliation">Botnar
Research Center, University of Oxford, Oxford OX3 7LD, U.K.</span>; 
    <span class="hlFld-Affiliation affiliation">Nuffield
Department of Clinical Medicine, Target Discovery Institute (TDI), University of Oxford, Oxford OX3 7BN, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Idelisse Ortiz Torres</span> - <span class="hlFld-Affiliation affiliation">†Department of Pharmaceutical Chemistry, ‡Chemistry and Chemical
Biology
Graduate Program, and §Department of Cellular and Molecular Pharmacology, University of California, San Francisco, Genentech Hall, 600 16th Street,
MC2280, San Francisco, California 94158-2280, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Oxford, Oxford OX3 7DQ, U.K.</span>; 
    <span class="hlFld-Affiliation affiliation">Botnar
Research Center, University of Oxford, Oxford OX3 7LD, U.K.</span>; 
    <span class="hlFld-Affiliation affiliation">Nuffield
Department of Clinical Medicine, Target Discovery Institute (TDI), University of Oxford, Oxford OX3 7BN, U.K.</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>M.K., D.D.L., and N.Y. contributed equally. All authors contributed to writing the manuscript, and all approve the final version of the manuscript. </p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d1398e4106-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i78">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_14220" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_14220" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The authors thank Dr. Wendell Lim and Dr. Denise Chan for the kind gift of expression plasmid and of FIH gene, respectively, Hayarpi Torosyan for help with aggregation assays, and Dr. Anat Levit, Dr. Trent E. Balius, Lindsey Pack, and Christina Fitzsimmons for valuable comments on the manuscript. This work was supported by Grant GM59957 (for docking) and Grant GM71630 (for colloidal aggregation controls) (to B.K.S.), Searle Scholar award, V Foundation Scholar awards, and UCSF Prostate Cancer Developmental Research Program (to D.G.F.) and by Arthritis Research UK (program Grant number 20522), the NIHR Oxford Biomedical Research Unit and the Rosetrees Trust (to U.O.). The Structural Genomics Consortium is a registered charity (No. 1097737) that receives funds from Abbvie, Bayer Healthcare, Boehringer Ingelheim, the Canadian Institutes for Health Research, the Canadian Foundation for Innovation, Eli Lilly and Company, Genome Canada, GlaxoSmithKline, the Ontario Ministry of Economic Development and Innovation, Janssen, the Novartis Research Foundation, Pfizer, Takeda, and the Wellcome Trust.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i79" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i79"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i80" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i80"> Abbreviations Used</h2><tr><td class="NLM_term">2,4-PDCA</td><td class="NLM_def"><p class="first last">2,4-pyridinedicarboxylic acid</p></td></tr><tr><td class="NLM_term">LSD1</td><td class="NLM_def"><p class="first last">lysine-specific demethylase 1</p></td></tr><tr><td class="NLM_term">JmjC</td><td class="NLM_def"><p class="first last">Jumonji C domain</p></td></tr><tr><td class="NLM_term">α-KG</td><td class="NLM_def"><p class="first last">α-ketoglutarate</p></td></tr><tr><td class="NLM_term">FIH</td><td class="NLM_def"><p class="first last">factor-inhibiting hypoxia-inducible factor</p></td></tr><tr><td class="NLM_term">LE</td><td class="NLM_def"><p class="first last">ligand efficiency</p></td></tr><tr><td class="NLM_term">xLogP</td><td class="NLM_def"><p class="first last">predicted octanol–water partitioning coefficient</p></td></tr><tr><td class="NLM_term">H3</td><td class="NLM_def"><p class="first last">histone H3</p></td></tr><tr><td class="NLM_term">KDM</td><td class="NLM_def"><p class="first last">lysine-specific demethylase</p></td></tr><tr><td class="NLM_term">NHS</td><td class="NLM_def"><p class="first last"><i>N</i>-hydroxysuccinimide</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i81">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_68412" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_68412" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 76 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Shi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lan, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mulligan, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whetstine, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cole, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casero, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Y.</span><span> </span><span class="NLM_article-title">Histone demethylation mediated by the nuclear amine oxidase homolog LSD1</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">119</span><span class="NLM_x">, </span> <span class="NLM_fpage">941</span><span class="NLM_x">–</span> <span class="NLM_lpage">953</span><span class="refDoi"> DOI: 10.1016/j.cell.2004.12.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1016%2Fj.cell.2004.12.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=15620353" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BD2MXltlSitg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2004&pages=941-953&author=Y.+Shiauthor=F.+Lanauthor=C.+Matsonauthor=P.+Mulliganauthor=J.+R.+Whetstineauthor=P.+A.+Coleauthor=R.+A.+Caseroauthor=Y.+Shi&title=Histone+demethylation+mediated+by+the+nuclear+amine+oxidase+homolog+LSD1&doi=10.1016%2Fj.cell.2004.12.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Histone demethylation mediated by the nuclear amine oxidase homolog LSD1</span></div><div class="casAuthors">Shi, Yujiang; Lan, Fei; Matson, Caitlin; Mulligan, Peter; Whetstine, Johnathan R.; Cole, Philip A.; Casero, Robert A.; Shi, Yang</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">941-953</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Posttranslational modifications of histone N-terminal tails impact chromatin structure and gene transcription.  While the extent of histone acetylation is detd. by both acetyltransferases and deacetylases, it has been unclear whether histone methylation is also regulated by enzymes with opposing activities.  Here, we provide evidence that LSD1 (KIAA0601), a nuclear homolog of amine oxidases, functions as a histone demethylase and transcriptional corepressor.  LSD1 specifically demethylates histone H3 lysine 4, which is linked to active transcription.  Lysine demethylation occurs via an oxidn. reaction that generates formaldehyde.  Importantly, RNAi inhibition of LSD1 causes an increase in H3 lysine 4 methylation and concomitant derepression of target genes, suggesting that LSD1 represses transcription via histone demethylation.  The results thus identify a histone demethylase conserved from S. pombe to human and reveal dynamic regulation of histone methylation by both histone methylases and demethylases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXHr-8kwGdXrVg90H21EOLACvtfcHk0ljrMu2zm1G5yg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXltlSitg%253D%253D&md5=3474569204255e90a832667ba654748b</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2004.12.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2004.12.012%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DY.%26aulast%3DLan%26aufirst%3DF.%26aulast%3DMatson%26aufirst%3DC.%26aulast%3DMulligan%26aufirst%3DP.%26aulast%3DWhetstine%26aufirst%3DJ.%2BR.%26aulast%3DCole%26aufirst%3DP.%2BA.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26aulast%3DShi%26aufirst%3DY.%26atitle%3DHistone%2520demethylation%2520mediated%2520by%2520the%2520nuclear%2520amine%2520oxidase%2520homolog%2520LSD1%26jtitle%3DCell%26date%3D2004%26volume%3D119%26spage%3D941%26epage%3D953%26doi%3D10.1016%2Fj.cell.2004.12.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Tsukada, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Erdjument-Bromage, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Warren, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borchers, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tempst, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span> </span><span class="NLM_article-title">Histone demethylation by a family of JmjC domain-containing proteins</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">439</span><span class="NLM_x">, </span> <span class="NLM_fpage">811</span><span class="NLM_x">–</span> <span class="NLM_lpage">816</span><span class="refDoi"> DOI: 10.1038/nature04433</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1038%2Fnature04433" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=16362057" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BD28XhsVSkuro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=439&publication_year=2006&pages=811-816&author=Y.+Tsukadaauthor=J.+Fangauthor=H.+Erdjument-Bromageauthor=M.+E.+Warrenauthor=C.+H.+Borchersauthor=P.+Tempstauthor=Y.+Zhang&title=Histone+demethylation+by+a+family+of+JmjC+domain-containing+proteins&doi=10.1038%2Fnature04433"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Histone demethylation by a family of JmjC domain-containing proteins</span></div><div class="casAuthors">Tsukada, Yu-ichi; Fang, Jia; Erdjument-Bromage, Hediye; Warren, Maria E.; Borchers, Christoph H.; Tempst, Paul; Zhang, Yi</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">439</span>
        (<span class="NLM_cas:issue">7078</span>),
    <span class="NLM_cas:pages">811-816</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Covalent modification of histones has an important role in regulating chromatin dynamics and transcription.  Whereas most covalent histone modifications are reversible, until recently it was unknown whether Me groups could be actively removed from histones.  Using a biochem. assay coupled with chromatog., we have purified a novel JmjC domain-contg. protein, JHDM1 (JmjC domain-contg. histone demethylase 1), that specifically demethylates histone H3 at lysine 36 (H3-K36).  In the presence of Fe(II) and α-ketoglutarate (αKG), JHDM1 demethylates H3-methyl-K36 and generates formaldehyde and succinate.  Overexpression of JHDM1 reduced the level of dimethyl-H3-K36 (H3K36me2) in vivo.  The demethylase activity of the JmjC domain-contg. proteins is conserved, as a JHDM1 homolog in Saccharomyces cerevisiae also has H3-K36 demethylase activity.  Thus, we identify the JmjC domain as a novel demethylase signature motif and uncover a protein demethylation mechanism that is conserved from yeast to human.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKQSpeu3QBl7Vg90H21EOLACvtfcHk0ljRhPnTvd_5kQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhsVSkuro%253D&md5=08f672cc5f81ca88288496f5d8069e3e</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnature04433&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature04433%26sid%3Dliteratum%253Aachs%26aulast%3DTsukada%26aufirst%3DY.%26aulast%3DFang%26aufirst%3DJ.%26aulast%3DErdjument-Bromage%26aufirst%3DH.%26aulast%3DWarren%26aufirst%3DM.%2BE.%26aulast%3DBorchers%26aufirst%3DC.%2BH.%26aulast%3DTempst%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DY.%26atitle%3DHistone%2520demethylation%2520by%2520a%2520family%2520of%2520JmjC%2520domain-containing%2520proteins%26jtitle%3DNature%26date%3D2006%26volume%3D439%26spage%3D811%26epage%3D816%26doi%3D10.1038%2Fnature04433" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Johansson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tumber, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Che, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cain, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nowak, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gileadi, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oppermann, U.</span><span> </span><span class="NLM_article-title">The roles of Jumonji-type oxygenases in human disease</span> <span class="citation_source-journal">Epigenomics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">89</span><span class="NLM_x">–</span> <span class="NLM_lpage">120</span><span class="refDoi"> DOI: 10.2217/epi.13.79</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.2217%2Fepi.13.79" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=24579949" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BC2cXnslClu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=89-120&author=C.+Johanssonauthor=A.+Tumberauthor=K.+Cheauthor=P.+Cainauthor=R.+Nowakauthor=C.+Gileadiauthor=U.+Oppermann&title=The+roles+of+Jumonji-type+oxygenases+in+human+disease&doi=10.2217%2Fepi.13.79"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The roles of Jumonji-type oxygenases in human disease</span></div><div class="casAuthors">Johansson, Catrine; Tumber, Anthony; Che, Ka Hing; Cain, Peter; Nowak, Radoslaw; Gileadi, Carina; Oppermann, Udo</div><div class="citationInfo"><span class="NLM_cas:title">Epigenomics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">89-120</span>CODEN:
                <span class="NLM_cas:coden">EPIGC7</span>;
        ISSN:<span class="NLM_cas:issn">1750-192X</span>.
    
            (<span class="NLM_cas:orgname">Future Medicine Ltd.</span>)
        </div><div class="casAbstract">A review.  The iron- and 2-oxoglutarate-dependent oxygenases constitute a phylogenetically conserved class of enzymes that catalyze hydroxylation reactions in humans by acting on various types of substrates, including metabolic intermediates, amino acid residues in different proteins and various types of nucleic acids.  The discovery of jumonji (Jmj), the founding member of a class of Jmj-type chromatin modifying enzymes and transcriptional regulators, has culminated in the discovery of several branches of histone lysine demethylases, with essential functions in regulating the epigenetic landscape of the chromatin environment.  This work has now been considerably expanded into other aspects of epigenetic biol. and includes the discovery of enzymic steps required for methyl-cytosine demethylation as well as modification of RNA and ribosomal proteins.  This overview aims to summarize the current knowledge on the human Jmj-type enzymes and their involvement in human pathol. processes, including development, cancer, inflammation and metabolic diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2w6QxrC2kT7Vg90H21EOLACvtfcHk0ljRhPnTvd_5kQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXnslClu70%253D&md5=a9a897d70db8a17dc1ae424ce4c40e9f</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.2217%2Fepi.13.79&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2217%252Fepi.13.79%26sid%3Dliteratum%253Aachs%26aulast%3DJohansson%26aufirst%3DC.%26aulast%3DTumber%26aufirst%3DA.%26aulast%3DChe%26aufirst%3DK.%26aulast%3DCain%26aufirst%3DP.%26aulast%3DNowak%26aufirst%3DR.%26aulast%3DGileadi%26aufirst%3DC.%26aulast%3DOppermann%26aufirst%3DU.%26atitle%3DThe%2520roles%2520of%2520Jumonji-type%2520oxygenases%2520in%2520human%2520disease%26jtitle%3DEpigenomics%26date%3D2014%26volume%3D6%26spage%3D89%26epage%3D120%26doi%3D10.2217%2Fepi.13.79" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Labbe, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holowatyj, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, Z. Q.</span><span> </span><span class="NLM_article-title">Histone lysine demethylase (KDM) subfamily 4: structures, functions and therapeutic potential</span> <span class="citation_source-journal">Am. J. Transl. Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">15</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=24349617" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A280%3ADC%252BC2c3pslKitw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2013&pages=1-15&author=R.+M.+Labbeauthor=A.+Holowatyjauthor=Z.+Q.+Yang&title=Histone+lysine+demethylase+%28KDM%29+subfamily+4%3A+structures%2C+functions+and+therapeutic+potential"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Histone lysine demethylase (KDM) subfamily 4: structures, functions and therapeutic potential</span></div><div class="casAuthors">Labbe Roselyne M; Holowatyj Andreana; Yang Zeng-Quan</div><div class="citationInfo"><span class="NLM_cas:title">American journal of translational research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-15</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">KDM4 histone demethylases catalyze the removal of methyl marks from histone lysine residues to epigenetically regulate chromatin structure and gene expression.  KDM4 expression is tightly regulated to insure proper function in diverse biological processes, such as cellular differentiation.  Mounting evidence has shown that disrupting KDM4 expression is implicated in the establishment and progression of multiple diseases including cancer.  In particular, genomic regions encoding the KDM4A, B and C genes are often amplified, disrupting normal cellular proliferation.  Furthermore, KDM4 demethylases are promising druggable targets.  In this review, we highlight the latest advances in characterizing the structures and regulatory mechanisms of KDM4 proteins, as well as our current understanding of their alterations and roles in tumorigenesis.  We also review the reported KDM4 inhibitors and discuss their potential as therapeutic agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRlRv7E2abvvjW1zwg2yUK5fW6udTcc2eZM0NcSdf68X7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2c3pslKitw%253D%253D&md5=60009f9b59e7110ce4ff57998493a128</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLabbe%26aufirst%3DR.%2BM.%26aulast%3DHolowatyj%26aufirst%3DA.%26aulast%3DYang%26aufirst%3DZ.%2BQ.%26atitle%3DHistone%2520lysine%2520demethylase%2520%2528KDM%2529%2520subfamily%25204%253A%2520structures%252C%2520functions%2520and%2520therapeutic%2520potential%26jtitle%3DAm.%2520J.%2520Transl.%2520Res.%26date%3D2013%26volume%3D6%26spage%3D1%26epage%3D15" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Jenuwein, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allis, C. D.</span><span> </span><span class="NLM_article-title">Translating the histone code</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">293</span><span class="NLM_x">, </span> <span class="NLM_fpage">1074</span><span class="NLM_x">–</span> <span class="NLM_lpage">1080</span><span class="refDoi"> DOI: 10.1126/science.1063127</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1126%2Fscience.1063127" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=11498575" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BD3MXmtVWltro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=293&publication_year=2001&pages=1074-1080&author=T.+Jenuweinauthor=C.+D.+Allis&title=Translating+the+histone+code&doi=10.1126%2Fscience.1063127"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Translating the histone code</span></div><div class="casAuthors">Jenuwein, Thomas; Allis, C. David</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">293</span>
        (<span class="NLM_cas:issue">5532</span>),
    <span class="NLM_cas:pages">1074-1080</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The review.  Chromatin, the physiol. template of all eukaryotic genetic information, is subject to a diverse array of posttranslational modifications that largely impinge on histone amino termini, thereby regulating access to the underlying DNA.  Distinct histone amino-terminal modifications can generate synergistic or antagonistic interaction affinities for chromatin-assocd. proteins, which in turn dictate dynamic transitions between transcriptionally active or transcriptionally silent chromatin states.  The combinatorial nature of histone amino-terminal modifications thus reveals a "histone code" that considerably extends the information potential of the genetic code.  We propose that this epigenetic marking system represents a fundamental regulatory mechanism that has an impact on most, if not all; chromatin-templated processes, with far-reaching consequences for cell fate decisions and both normal and pathol. development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUFyHNepKcUbVg90H21EOLACvtfcHk0liOAytSrxIi9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXmtVWltro%253D&md5=021dcd98a5e5bf2ed7eedef1897b2269</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1126%2Fscience.1063127&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1063127%26sid%3Dliteratum%253Aachs%26aulast%3DJenuwein%26aufirst%3DT.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26atitle%3DTranslating%2520the%2520histone%2520code%26jtitle%3DScience%26date%3D2001%26volume%3D293%26spage%3D1074%26epage%3D1080%26doi%3D10.1126%2Fscience.1063127" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Katoh, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Katoh, M.</span><span> </span><span class="NLM_article-title">Identification and characterization of JMJD2 family genes in silico</span> <span class="citation_source-journal">Int. J. Oncol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">1623</span><span class="NLM_x">–</span> <span class="NLM_lpage">1628</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=15138608" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BD2cXkvFaisbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2004&pages=1623-1628&author=M.+Katohauthor=M.+Katoh&title=Identification+and+characterization+of+JMJD2+family+genes+in+silico"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and characterization of JMJD2 family genes in silico</span></div><div class="casAuthors">Katoh, Masuko; Katoh, Masaru</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Oncology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1623-1628</span>CODEN:
                <span class="NLM_cas:coden">IJONES</span>;
        ISSN:<span class="NLM_cas:issn">1019-6439</span>.
    
            (<span class="NLM_cas:orgname">International Journal of Oncology</span>)
        </div><div class="casAbstract">JMJD1 family genes, including JMJD1 (TSGA), TRIP8, and C5orf7 (5qNCA), encode nuclear proteins with Jumonji C (JmjC), TRI8H1 and TRI8H2 domains.  Here, we identified JMJD2 family genes.  Human genes corresponding to KIAA0677, KIAA0876, KIAA0780 and FLJ10251 cDNAs were designated JMJD2A, JMJD2B, JMJD2C, and JMJD2D, resp.  In addn., JMJD2D homologous genes within human genome sequences AP002383.3 and AP001264.4 were designated JMJD2E and JMJD2F, resp.  JMJD2A gene was mapped to human chromosome 1p34.1, JMJD2B gene to 19p13.3, and JMJD2C gene to 9p24.1.  JMJD2D, JMJD2E and JMJD2F genes were clustered at human chromosome 11q21.  Coding region of JMJD2A, JMJD2B, JMJD2C genes was located on multiple exons, while that of JMJD2D, JMJD2E and JMJD2F genes was located on a single exon.  These facts strongly indicate that JMJD2D, JMJD2E and JMJD2F genes are locally triplicated retrotransposed elements derived from JMJD2 family gene.  JD2H domain with C2HC2HC2- and C5HC2-type Cys (His) clusters was identified as the region conserved among JMJD2A (1064 aa), JMJD2B (1096 aa), and JMJD2C (1056 aa) proteins.  JMJD2A, JMJD2B and JMJD2C consist of JmjN, JmjC, JD2H, and two TUDOR domains, while JMJD2D (523 aa), JMJD2E (506 aa) and JMJD2F (638 aa) consist of JmjN and JmjC domains.  JMJD2 family proteins were classified into one group with JD2H and TUDOR domains and another group without JD2H or TUDOR domains.  Because JMJD2C gene (also known as GASC1 gene) is amplified in esophageal squamous cell carcinoma (ESCC), JMJD2 family genes are cancer-assocd. genes.  This is the first report on identification and characterization of human JMJD2 gene family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2QrOZpXEa9rVg90H21EOLACvtfcHk0liOAytSrxIi9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXkvFaisbo%253D&md5=b6ab9ea6d3aebb1c242870726103a449</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKatoh%26aufirst%3DM.%26aulast%3DKatoh%26aufirst%3DM.%26atitle%3DIdentification%2520and%2520characterization%2520of%2520JMJD2%2520family%2520genes%2520in%2520silico%26jtitle%3DInt.%2520J.%2520Oncol.%26date%3D2004%26volume%3D24%26spage%3D1623%26epage%3D1628" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Whetstine, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nottke, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lan, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huarte, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smolikov, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spooner, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Colaiacovo, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Y.</span><span> </span><span class="NLM_article-title">Reversal of histone lysine trimethylation by the JMJD2 family of histone demethylases</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">125</span><span class="NLM_x">, </span> <span class="NLM_fpage">467</span><span class="NLM_x">–</span> <span class="NLM_lpage">481</span><span class="refDoi"> DOI: 10.1016/j.cell.2006.03.028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1016%2Fj.cell.2006.03.028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=16603238" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BD28XkslCjtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2006&pages=467-481&author=J.+R.+Whetstineauthor=A.+Nottkeauthor=F.+Lanauthor=M.+Huarteauthor=S.+Smolikovauthor=Z.+Chenauthor=E.+Spoonerauthor=E.+Liauthor=G.+Zhangauthor=M.+Colaiacovoauthor=Y.+Shi&title=Reversal+of+histone+lysine+trimethylation+by+the+JMJD2+family+of+histone+demethylases&doi=10.1016%2Fj.cell.2006.03.028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Reversal of histone lysine trimethylation by the JMJD2 family of histone demethylases</span></div><div class="casAuthors">Whetstine, Johnathan R.; Nottke, Amanda; Lan, Fei; Huarte, Maite; Smolikov, Sarit; Chen, Zhongzhou; Spooner, Eric; Li, En; Zhang, Gongyi; Colaiacovo, Monica; Shi, Yang</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">125</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">467-481</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Histone methylation regulates chromatin structure, transcription, and epigenetic state of the cell.  Histone methylation is dynamically regulated by histone methylases and demethylases such as LSD1 and JHDM1, which mediate demethylation of di- and monomethylated histones.  It has been unclear whether demethylases exist that reverse lysine trimethylation.  We show the JmjC domain-contg. protein JMJD2A reversed trimethylated H3-K9/K36 to di- but not mono- or unmethylated products.  Overexpression of JMJD2A but not a catalytically inactive mutant reduced H3-K9/K36 trimethylation levels in cultured cells.  In contrast, RNAi depletion of the C. elegans JMJD2A homolog resulted in an increase in general H3-K9Me3 and localized H3-K36Me3 levels on meiotic chromosomes and triggered p53-dependent germline apoptosis.  Addnl., other human JMJD2 subfamily members also functioned as trimethylation-specific demethylases, converting H3-K9Me3 to H3-K9Me2 and H3-K9Me1, resp.  Our finding that this family of demethylases generates different methylated states at the same lysine residue provides a mechanism for fine-tuning histone methylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopUntpFohMorVg90H21EOLACvtfcHk0liOAytSrxIi9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XkslCjtLo%253D&md5=e73fcf5b9f5f3127bae7c9f5be8a269b</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2006.03.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2006.03.028%26sid%3Dliteratum%253Aachs%26aulast%3DWhetstine%26aufirst%3DJ.%2BR.%26aulast%3DNottke%26aufirst%3DA.%26aulast%3DLan%26aufirst%3DF.%26aulast%3DHuarte%26aufirst%3DM.%26aulast%3DSmolikov%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DSpooner%26aufirst%3DE.%26aulast%3DLi%26aufirst%3DE.%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DColaiacovo%26aufirst%3DM.%26aulast%3DShi%26aufirst%3DY.%26atitle%3DReversal%2520of%2520histone%2520lysine%2520trimethylation%2520by%2520the%2520JMJD2%2520family%2520of%2520histone%2520demethylases%26jtitle%3DCell%26date%3D2006%26volume%3D125%26spage%3D467%26epage%3D481%26doi%3D10.1016%2Fj.cell.2006.03.028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Cloos, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Christensen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Agger, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maiolica, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rappsilber, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Antal, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hansen, K. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Helin, K.</span><span> </span><span class="NLM_article-title">The putative oncogene GASC1 demethylates tri- and dimethylated lysine 9 on histone H3</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">442</span><span class="NLM_x">, </span> <span class="NLM_fpage">307</span><span class="NLM_x">–</span> <span class="NLM_lpage">311</span><span class="refDoi"> DOI: 10.1038/nature04837</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1038%2Fnature04837" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=442&publication_year=2006&pages=307-311&author=P.+A.+Cloosauthor=J.+Christensenauthor=K.+Aggerauthor=A.+Maiolicaauthor=J.+Rappsilberauthor=T.+Antalauthor=K.+H.+Hansenauthor=K.+Helin&title=The+putative+oncogene+GASC1+demethylates+tri-+and+dimethylated+lysine+9+on+histone+H3&doi=10.1038%2Fnature04837"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnature04837&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature04837%26sid%3Dliteratum%253Aachs%26aulast%3DCloos%26aufirst%3DP.%2BA.%26aulast%3DChristensen%26aufirst%3DJ.%26aulast%3DAgger%26aufirst%3DK.%26aulast%3DMaiolica%26aufirst%3DA.%26aulast%3DRappsilber%26aufirst%3DJ.%26aulast%3DAntal%26aufirst%3DT.%26aulast%3DHansen%26aufirst%3DK.%2BH.%26aulast%3DHelin%26aufirst%3DK.%26atitle%3DThe%2520putative%2520oncogene%2520GASC1%2520demethylates%2520tri-%2520and%2520dimethylated%2520lysine%25209%2520on%2520histone%2520H3%26jtitle%3DNature%26date%3D2006%26volume%3D442%26spage%3D307%26epage%3D311%26doi%3D10.1038%2Fnature04837" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Klose, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamane, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bae, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Erdjument-Bromage, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tempst, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span> </span><span class="NLM_article-title">The transcriptional repressor JHDM3A demethylates trimethyl histone H3 lysine 9 and lysine 36</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">442</span><span class="NLM_x">, </span> <span class="NLM_fpage">312</span><span class="NLM_x">–</span> <span class="NLM_lpage">316</span><span class="refDoi"> DOI: 10.1038/nature04853</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1038%2Fnature04853" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=16732292" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BD28XmvF2jsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=442&publication_year=2006&pages=312-316&author=R.+J.+Kloseauthor=K.+Yamaneauthor=Y.+Baeauthor=D.+Zhangauthor=H.+Erdjument-Bromageauthor=P.+Tempstauthor=J.+Wongauthor=Y.+Zhang&title=The+transcriptional+repressor+JHDM3A+demethylates+trimethyl+histone+H3+lysine+9+and+lysine+36&doi=10.1038%2Fnature04853"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">The transcriptional repressor JHDM3A demethylates trimethyl histone H3 lysine 9 and lysine 36</span></div><div class="casAuthors">Klose, Robert J.; Yamane, Kenichi; Bae, Yangjin; Zhang, Dianzheng; Erdjument-Bromage, Hediye; Tempst, Paul; Wong, Jiemin; Zhang, Yi</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">442</span>
        (<span class="NLM_cas:issue">7100</span>),
    <span class="NLM_cas:pages">312-316</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Post-translational modification of chromatin has profound effects on many biol. processes including transcriptional regulation, heterochromatin organization, and X-chromosome inactivation.  Recent studies indicate that methylation on specific histone lysine (K) residues participates in many of these processes.  Lysine methylation occurs in three distinct states, having either one (me1), two (me2) or three (me3) Me groups attached to the amine group of the lysine side chain.  These differences in modification state have an important role in defining how methylated chromatin is recognized and interpreted.  Until recently, histone lysine methylation was considered a stable modification, but the identification of histone demethylase enzymes has demonstrated the reversibility of this epigenetic mark.  So far, all characterized histone demethylases show enzymic activity towards lysine residues modified in the me1 or me2 state, leaving open the possibility that me3 constitutes an irreversible modification.  Here we demonstrate that JHDM3A (jumonji C (JmjC)-domain-contg. histone demethylase 3A; also known as JMJD2A) is capable of removing the me3 group from modified H3 lysine 9 (H3K9) and H3 lysine 36 (H3K36).  Overexpression of JHDM3A abrogates recruitment of HP1 (heterochromatin protein 1) to heterochromatin, indicating a role for JHDM3A in antagonizing methylated H3K9 nucleated events.  SiRNA-mediated knockdown of JHDM3A leads to increased levels of H3K9 methylation and upregulation of a JHDM3A target gene, ASCL2, indicating that JHDM3A may function in euchromatin to remove histone methylation marks that are assocd. with active transcription.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7-qskfqfg47Vg90H21EOLACvtfcHk0lgM00Da8RLIgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XmvF2jsr0%253D&md5=d9ab4dd44f1ddcc3bcf3472fd540dd60</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnature04853&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature04853%26sid%3Dliteratum%253Aachs%26aulast%3DKlose%26aufirst%3DR.%2BJ.%26aulast%3DYamane%26aufirst%3DK.%26aulast%3DBae%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DErdjument-Bromage%26aufirst%3DH.%26aulast%3DTempst%26aufirst%3DP.%26aulast%3DWong%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DY.%26atitle%3DThe%2520transcriptional%2520repressor%2520JHDM3A%2520demethylates%2520trimethyl%2520histone%2520H3%2520lysine%25209%2520and%2520lysine%252036%26jtitle%3DNature%26date%3D2006%26volume%3D442%26spage%3D312%26epage%3D316%26doi%3D10.1038%2Fnature04853" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Fodor, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kubicek, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yonezawa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Sullivan, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sengupta, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perez-Burgos, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Opravil, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mechtler, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schotta, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenuwein, T.</span><span> </span><span class="NLM_article-title">Jmjd2b antagonizes H3K9 trimethylation at pericentric heterochromatin in mammalian cells</span> <span class="citation_source-journal">Genes Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">1557</span><span class="NLM_x">–</span> <span class="NLM_lpage">1562</span><span class="refDoi"> DOI: 10.1101/gad.388206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1101%2Fgad.388206" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2006&pages=1557-1562&author=B.+D.+Fodorauthor=S.+Kubicekauthor=M.+Yonezawaauthor=R.+J.+O%E2%80%99Sullivanauthor=R.+Senguptaauthor=L.+Perez-Burgosauthor=S.+Opravilauthor=K.+Mechtlerauthor=G.+Schottaauthor=T.+Jenuwein&title=Jmjd2b+antagonizes+H3K9+trimethylation+at+pericentric+heterochromatin+in+mammalian+cells&doi=10.1101%2Fgad.388206"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1101%2Fgad.388206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.388206%26sid%3Dliteratum%253Aachs%26aulast%3DFodor%26aufirst%3DB.%2BD.%26aulast%3DKubicek%26aufirst%3DS.%26aulast%3DYonezawa%26aufirst%3DM.%26aulast%3DO%25E2%2580%2599Sullivan%26aufirst%3DR.%2BJ.%26aulast%3DSengupta%26aufirst%3DR.%26aulast%3DPerez-Burgos%26aufirst%3DL.%26aulast%3DOpravil%26aufirst%3DS.%26aulast%3DMechtler%26aufirst%3DK.%26aulast%3DSchotta%26aufirst%3DG.%26aulast%3DJenuwein%26aufirst%3DT.%26atitle%3DJmjd2b%2520antagonizes%2520H3K9%2520trimethylation%2520at%2520pericentric%2520heterochromatin%2520in%2520mammalian%2520cells%26jtitle%3DGenes%2520Dev.%26date%3D2006%26volume%3D20%26spage%3D1557%26epage%3D1562%26doi%3D10.1101%2Fgad.388206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Shin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janknecht, R.</span><span> </span><span class="NLM_article-title">Activation of androgen receptor by histone demethylases JMJD2A and JMJD2D</span> <span class="citation_source-journal">Biochem. Biophys. Res. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">359</span><span class="NLM_x">, </span> <span class="NLM_fpage">742</span><span class="NLM_x">–</span> <span class="NLM_lpage">746</span><span class="refDoi"> DOI: 10.1016/j.bbrc.2007.05.179</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1016%2Fj.bbrc.2007.05.179" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=17555712" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BD2sXmsFykurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=359&publication_year=2007&pages=742-746&author=S.+Shinauthor=R.+Janknecht&title=Activation+of+androgen+receptor+by+histone+demethylases+JMJD2A+and+JMJD2D&doi=10.1016%2Fj.bbrc.2007.05.179"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Activation of androgen receptor by histone demethylases JMJD2A and JMJD2D</span></div><div class="casAuthors">Shin, Sook; Janknecht, Ralf</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">359</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">742-746</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">The androgen receptor (AR) is a transcription factor that is pivotal for the development of prostate cancer.  Here, we have identified two related histone demethylases, JMJD2A and JMJD2D, which form complexes with ligand-bound AR.  We found that AR interacts through its ligand binding domain with JMJD2A and JMJD2D.  On the other hand, JMJD2A utilizes its catalytic domain or C-terminus to bind to AR, and JMJD2D does so via its C-terminus.  Further, overexpression of JMJD2A or D stimulates AR function and this is dependent on JMJD2 catalytic activity.  Conversely, downregulation of JMJD2A, which is often overexpressed in prostate tumors, reduces basal transcription of the AR target gene, prostate-specific antigen, in LNCaP prostate cancer cells.  Altogether, our data have identified a novel class of AR coactivators, whose (over)expression in prostate tumors could contribute to the constitutive activation of AR and thus to androgen-depletion independency of advanced prostate cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkS2bbLidpDbVg90H21EOLACvtfcHk0lgM00Da8RLIgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXmsFykurk%253D&md5=56e0c5770876f4ce54e46fe940425b7a</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2007.05.179&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2007.05.179%26sid%3Dliteratum%253Aachs%26aulast%3DShin%26aufirst%3DS.%26aulast%3DJanknecht%26aufirst%3DR.%26atitle%3DActivation%2520of%2520androgen%2520receptor%2520by%2520histone%2520demethylases%2520JMJD2A%2520and%2520JMJD2D%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2007%26volume%3D359%26spage%3D742%26epage%3D746%26doi%3D10.1016%2Fj.bbrc.2007.05.179" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Trojer, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yonezawa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmidt, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenuwein, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reinberg, D.</span><span> </span><span class="NLM_article-title">Dynamic Histone H1 Isotype 4 Methylation and Demethylation by Histone Lysine Methyltransferase G9a/KMT1C and the Jumonji Domain-containing JMJD2/KDM4 Proteins</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">284</span><span class="NLM_x">, </span> <span class="NLM_fpage">8395</span><span class="NLM_x">–</span> <span class="NLM_lpage">8405</span><span class="refDoi"> DOI: 10.1074/jbc.M807818200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1074%2Fjbc.M807818200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=284&publication_year=2009&pages=8395-8405&author=P.+Trojerauthor=J.+Zhangauthor=M.+Yonezawaauthor=A.+Schmidtauthor=H.+Zhengauthor=T.+Jenuweinauthor=D.+Reinberg&title=Dynamic+Histone+H1+Isotype+4+Methylation+and+Demethylation+by+Histone+Lysine+Methyltransferase+G9a%2FKMT1C+and+the+Jumonji+Domain-containing+JMJD2%2FKDM4+Proteins&doi=10.1074%2Fjbc.M807818200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M807818200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M807818200%26sid%3Dliteratum%253Aachs%26aulast%3DTrojer%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DYonezawa%26aufirst%3DM.%26aulast%3DSchmidt%26aufirst%3DA.%26aulast%3DZheng%26aufirst%3DH.%26aulast%3DJenuwein%26aufirst%3DT.%26aulast%3DReinberg%26aufirst%3DD.%26atitle%3DDynamic%2520Histone%2520H1%2520Isotype%25204%2520Methylation%2520and%2520Demethylation%2520by%2520Histone%2520Lysine%2520Methyltransferase%2520G9a%252FKMT1C%2520and%2520the%2520Jumonji%2520Domain-containing%2520JMJD2%252FKDM4%2520Proteins%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2009%26volume%3D284%26spage%3D8395%26epage%3D8405%26doi%3D10.1074%2Fjbc.M807818200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Yang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohgi, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grinstein, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dorrestein, P. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosenfeld, M. G.</span><span> </span><span class="NLM_article-title">ncRNA- and Pc2 methylation-dependent gene relocation between nuclear structures mediates gene activation programs</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">147</span><span class="NLM_x">, </span> <span class="NLM_fpage">773</span><span class="NLM_x">–</span> <span class="NLM_lpage">788</span><span class="refDoi"> DOI: 10.1016/j.cell.2011.08.054</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1016%2Fj.cell.2011.08.054" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=147&publication_year=2011&pages=773-788&author=L.+Yangauthor=C.+Linauthor=W.+Liuauthor=J.+Zhangauthor=K.+A.+Ohgiauthor=J.+D.+Grinsteinauthor=P.+C.+Dorresteinauthor=M.+G.+Rosenfeld&title=ncRNA-+and+Pc2+methylation-dependent+gene+relocation+between+nuclear+structures+mediates+gene+activation+programs&doi=10.1016%2Fj.cell.2011.08.054"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2011.08.054&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2011.08.054%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DL.%26aulast%3DLin%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DOhgi%26aufirst%3DK.%2BA.%26aulast%3DGrinstein%26aufirst%3DJ.%2BD.%26aulast%3DDorrestein%26aufirst%3DP.%2BC.%26aulast%3DRosenfeld%26aufirst%3DM.%2BG.%26atitle%3DncRNA-%2520and%2520Pc2%2520methylation-dependent%2520gene%2520relocation%2520between%2520nuclear%2520structures%2520mediates%2520gene%2520activation%2520programs%26jtitle%3DCell%26date%3D2011%26volume%3D147%26spage%3D773%26epage%3D788%26doi%3D10.1016%2Fj.cell.2011.08.054" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Chu, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hsu, K. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, H. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, L. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hung, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kung, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, W. C.</span><span> </span><span class="NLM_article-title">KDM4B as a target for prostate cancer: structural analysis and selective inhibition by a novel inhibitor</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">5975</span><span class="NLM_x">–</span> <span class="NLM_lpage">5985</span><span class="refDoi"> DOI: 10.1021/jm500249n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm500249n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVKnsLnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=5975-5985&author=C.+H.+Chuauthor=L.+Y.+Wangauthor=K.+C.+Hsuauthor=C.+C.+Chenauthor=H.+H.+Chengauthor=S.+M.+Wangauthor=C.+M.+Wuauthor=T.+J.+Chenauthor=L.+T.+Liauthor=R.+Liuauthor=C.+L.+Hungauthor=J.+M.+Yangauthor=H.+J.+Kungauthor=W.+C.+Wang&title=KDM4B+as+a+target+for+prostate+cancer%3A+structural+analysis+and+selective+inhibition+by+a+novel+inhibitor&doi=10.1021%2Fjm500249n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">KDM4B as a Target for Prostate Cancer: Structural Analysis and Selective Inhibition by a Novel Inhibitor</span></div><div class="casAuthors">Chu, Chia-Han; Wang, Ling-Yu; Hsu, Kai-Cheng; Chen, Chung-Chin; Cheng, Hsing-Hung; Wang, Szu-Min; Wu, Chien-Ming; Chen, Tsan-Jan; Li, Ling-Ting; Liu, Ruiwu; Hung, Chiu-Lien; Yang, Jing-Moon; Kung, Hsing-Jien; Wang, Wen-Ching</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">5975-5985</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The KDM4/JMJD2 Jumonji C-contg. histone lysine demethylases (KDM4A-KDM4D), which selectively remove the Me group(s) from tri/dimethylated lysine 9/36 of H3, modulate transcriptional activation and genome stability.  The overexpression of KDM4A/KDM4B in prostate cancer and their assocn. with androgen receptor suggest that KDM4A/KDM4B are potential progression factors for prostate cancer.  Here, we report the crystal structure of the KDM4B·pyridine 2,4-dicarboxylic acid·H3K9me3 ternary complex, revealing the core active-site region and a selective K9/K36 site.  A selective KDM4A/KDM4B inhibitor, 4, that occupies three subsites in the binding pocket is identified by virtual screening.  Pharmacol. and genetic inhibition of KDM4A/KDM4B significantly blocks the viability of cultured prostate cancer cells, which is accompanied by increased H3K9me3 staining and transcriptional silencing of growth-related genes.  Significantly, a substantial portion of differentially expressed genes are AR-responsive, consistent with the roles of KDM4s as crit. AR activators.  Our results point to KDM4 as a useful therapeutic target and identify a new inhibitor scaffold.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoR0KwURfrUnbVg90H21EOLACvtfcHk0ljHSl96Yd5cog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVKnsLnO&md5=789ecb59ed2e8d965ef220d92c9a2ce2</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fjm500249n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm500249n%26sid%3Dliteratum%253Aachs%26aulast%3DChu%26aufirst%3DC.%2BH.%26aulast%3DWang%26aufirst%3DL.%2BY.%26aulast%3DHsu%26aufirst%3DK.%2BC.%26aulast%3DChen%26aufirst%3DC.%2BC.%26aulast%3DCheng%26aufirst%3DH.%2BH.%26aulast%3DWang%26aufirst%3DS.%2BM.%26aulast%3DWu%26aufirst%3DC.%2BM.%26aulast%3DChen%26aufirst%3DT.%2BJ.%26aulast%3DLi%26aufirst%3DL.%2BT.%26aulast%3DLiu%26aufirst%3DR.%26aulast%3DHung%26aufirst%3DC.%2BL.%26aulast%3DYang%26aufirst%3DJ.%2BM.%26aulast%3DKung%26aufirst%3DH.%2BJ.%26aulast%3DWang%26aufirst%3DW.%2BC.%26atitle%3DKDM4B%2520as%2520a%2520target%2520for%2520prostate%2520cancer%253A%2520structural%2520analysis%2520and%2520selective%2520inhibition%2520by%2520a%2520novel%2520inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D5975%26epage%3D5985%26doi%3D10.1021%2Fjm500249n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ji, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yao, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, J.</span><span> </span><span class="NLM_article-title">An epigenetic role for PRL-3 as a regulator of H3K9 methylation in colorectal cancer</span> <span class="citation_source-journal">Gut</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">62</span><span class="NLM_x">, </span> <span class="NLM_fpage">571</span><span class="NLM_x">–</span> <span class="NLM_lpage">581</span><span class="refDoi"> DOI: 10.1136/gutjnl-2011-301059</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1136%2Fgutjnl-2011-301059" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2013&pages=571-581&author=Y.+Liuauthor=P.+Zhengauthor=Y.+Liuauthor=T.+Jiauthor=X.+Liuauthor=S.+Yaoauthor=X.+Chengauthor=Y.+Liauthor=L.+Chenauthor=Z.+Xiaoauthor=J.+Zhouauthor=J.+Li&title=An+epigenetic+role+for+PRL-3+as+a+regulator+of+H3K9+methylation+in+colorectal+cancer&doi=10.1136%2Fgutjnl-2011-301059"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1136%2Fgutjnl-2011-301059&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1136%252Fgutjnl-2011-301059%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DZheng%26aufirst%3DP.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DJi%26aufirst%3DT.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DYao%26aufirst%3DS.%26aulast%3DCheng%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DXiao%26aufirst%3DZ.%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DJ.%26atitle%3DAn%2520epigenetic%2520role%2520for%2520PRL-3%2520as%2520a%2520regulator%2520of%2520H3K9%2520methylation%2520in%2520colorectal%2520cancer%26jtitle%3DGut%26date%3D2013%26volume%3D62%26spage%3D571%26epage%3D581%26doi%3D10.1136%2Fgutjnl-2011-301059" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Yang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jubb, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pike, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buffa, F. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Turley, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baban, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leek, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gatter, K. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ragoussis, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harris, A. L.</span><span> </span><span class="NLM_article-title">The histone demethylase JMJD2B is regulated by estrogen receptor alpha and hypoxia, and is a key mediator of estrogen induced growth</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">70</span><span class="NLM_x">, </span> <span class="NLM_fpage">6456</span><span class="NLM_x">–</span> <span class="NLM_lpage">6466</span><span class="refDoi"> DOI: 10.1158/0008-5472.CAN-10-0413</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1158%2F0008-5472.CAN-10-0413" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=2010&pages=6456-6466&author=J.+Yangauthor=A.+M.+Jubbauthor=L.+Pikeauthor=F.+M.+Buffaauthor=H.+Turleyauthor=D.+Babanauthor=R.+Leekauthor=K.+C.+Gatterauthor=J.+Ragoussisauthor=A.+L.+Harris&title=The+histone+demethylase+JMJD2B+is+regulated+by+estrogen+receptor+alpha+and+hypoxia%2C+and+is+a+key+mediator+of+estrogen+induced+growth&doi=10.1158%2F0008-5472.CAN-10-0413"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-10-0413&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-10-0413%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DJubb%26aufirst%3DA.%2BM.%26aulast%3DPike%26aufirst%3DL.%26aulast%3DBuffa%26aufirst%3DF.%2BM.%26aulast%3DTurley%26aufirst%3DH.%26aulast%3DBaban%26aufirst%3DD.%26aulast%3DLeek%26aufirst%3DR.%26aulast%3DGatter%26aufirst%3DK.%2BC.%26aulast%3DRagoussis%26aufirst%3DJ.%26aulast%3DHarris%26aufirst%3DA.%2BL.%26atitle%3DThe%2520histone%2520demethylase%2520JMJD2B%2520is%2520regulated%2520by%2520estrogen%2520receptor%2520alpha%2520and%2520hypoxia%252C%2520and%2520is%2520a%2520key%2520mediator%2520of%2520estrogen%2520induced%2520growth%26jtitle%3DCancer%2520Res.%26date%3D2010%26volume%3D70%26spage%3D6456%26epage%3D6466%26doi%3D10.1158%2F0008-5472.CAN-10-0413" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Walport, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hopkinson, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span> </span><span class="NLM_article-title">Mechanisms of human histone and nucleic acid demethylases</span> <span class="citation_source-journal">Curr. Opin. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">525</span><span class="NLM_x">–</span> <span class="NLM_lpage">534</span><span class="refDoi"> DOI: 10.1016/j.cbpa.2012.09.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1016%2Fj.cbpa.2012.09.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=23063108" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsV2qtL%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2012&pages=525-534&author=L.+J.+Walportauthor=R.+J.+Hopkinsonauthor=C.+J.+Schofield&title=Mechanisms+of+human+histone+and+nucleic+acid+demethylases&doi=10.1016%2Fj.cbpa.2012.09.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms of human histone and nucleic acid demethylases</span></div><div class="casAuthors">Walport, Louise J.; Hopkinson, Richard J.; Schofield, Christopher J.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">5-6</span>),
    <span class="NLM_cas:pages">525-534</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The discovery that protein and nucleic acid demethylation is common opens up the possibility of 'methylation cycles' of functional importance, including in the regulation of gene expression.  The mechanisms of known demethylases can be broadly divided into those involving nucleophilic catalysis and those involving oxidative catalysis.  The latter group appear more common; they produce formaldehyde as a co-product.  Nucleophilic demethylases include those proceeding via irreversible S-methylation and Me esterases.  In addn. to the direct reversal of methylation, demethylation can occur concurrent with loss of other groups, such as in methylarginine hydrolysis, oxidn. of Nε-methyllysine to allysine, and indirectly, for example via base-excision repair.  Here, the authors discuss chem. viable mechanisms for biol. demethylation and summarize mechanistic knowledge of the major known families of demethylases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9kS9XUCbZTrVg90H21EOLACvtfcHk0liTwZsjmijD-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsV2qtL%252FI&md5=0e4c08d79aaca4ee74de3f222757cd9a</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2012.09.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2012.09.015%26sid%3Dliteratum%253Aachs%26aulast%3DWalport%26aufirst%3DL.%2BJ.%26aulast%3DHopkinson%26aufirst%3DR.%2BJ.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26atitle%3DMechanisms%2520of%2520human%2520histone%2520and%2520nucleic%2520acid%2520demethylases%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2012%26volume%3D16%26spage%3D525%26epage%3D534%26doi%3D10.1016%2Fj.cbpa.2012.09.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">McDonough, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loenarz, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chowdhury, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clifton, I. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span> </span><span class="NLM_article-title">Structural studies on human 2-oxoglutarate dependent oxygenases</span> <span class="citation_source-journal">Curr. Opin. Struct. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">659</span><span class="NLM_x">–</span> <span class="NLM_lpage">672</span><span class="refDoi"> DOI: 10.1016/j.sbi.2010.08.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1016%2Fj.sbi.2010.08.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=20888218" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsV2jsrfI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2010&pages=659-672&author=M.+A.+McDonoughauthor=C.+Loenarzauthor=R.+Chowdhuryauthor=I.+J.+Cliftonauthor=C.+J.+Schofield&title=Structural+studies+on+human+2-oxoglutarate+dependent+oxygenases&doi=10.1016%2Fj.sbi.2010.08.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Structural studies on human 2-oxoglutarate dependent oxygenases</span></div><div class="casAuthors">McDonough, Michael A.; Loenarz, Christoph; Chowdhury, Rasheduzzaman; Clifton, Ian J.; Schofield, Christopher J.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">659-672</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  2-Oxoglutarate- and ferrous iron-dependent oxygenases have emerged as an important family of human enzymes that catalyze hydroxylations and related demethylation reactions.  Their substrates in humans include proteins, nucleic acids, lipids, and small mols.  They play roles in collagen biosynthesis, hypoxic sensing, regulation of gene expression, and lipid biosynthesis/metab.  Structural analyses, principally employing crystallog., have revealed that all of these oxygenases possess a double-stranded β-helix core fold that supports a highly conserved triad of iron binding residues and a less well conserved 2-oxoglutarate co-substrate binding site.  The 2-oxoglutarate binds to the iron in a bidentate manner via its 1-carboxylate and 2-oxo groups.  The primary substrate binding elements are more variable and can involve mobile elements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrkeQ46qZSLnbVg90H21EOLACvtfcHk0liTwZsjmijD-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsV2jsrfI&md5=1540abfc06c091944a9514a42de68a1f</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.sbi.2010.08.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.sbi.2010.08.006%26sid%3Dliteratum%253Aachs%26aulast%3DMcDonough%26aufirst%3DM.%2BA.%26aulast%3DLoenarz%26aufirst%3DC.%26aulast%3DChowdhury%26aufirst%3DR.%26aulast%3DClifton%26aufirst%3DI.%2BJ.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26atitle%3DStructural%2520studies%2520on%2520human%25202-oxoglutarate%2520dependent%2520oxygenases%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2010%26volume%3D20%26spage%3D659%26epage%3D672%26doi%3D10.1016%2Fj.sbi.2010.08.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Rose, N. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDonough, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">King, O. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawamura, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span> </span><span class="NLM_article-title">Inhibition of 2-oxoglutarate dependent oxygenases</span> <span class="citation_source-journal">Chem. Soc. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">4364</span><span class="NLM_x">–</span> <span class="NLM_lpage">4397</span><span class="refDoi"> DOI: 10.1039/c0cs00203h</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1039%2Fc0cs00203h" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=21390379" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BC3MXovVemur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2011&pages=4364-4397&author=N.+R.+Roseauthor=M.+A.+McDonoughauthor=O.+N.+Kingauthor=A.+Kawamuraauthor=C.+J.+Schofield&title=Inhibition+of+2-oxoglutarate+dependent+oxygenases&doi=10.1039%2Fc0cs00203h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of 2-oxoglutarate dependent oxygenases</span></div><div class="casAuthors">Rose, Nathan R.; McDonough, Michael A.; King, Oliver N. F.; Kawamura, Akane; Schofield, Christopher J.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Society Reviews</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">4364-4397</span>CODEN:
                <span class="NLM_cas:coden">CSRVBR</span>;
        ISSN:<span class="NLM_cas:issn">0306-0012</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  2-Oxoglutarate (2OG) dependent oxygenases are ubiquitous iron enzymes that couple substrate oxidn. to the conversion of 2OG to succinate and carbon dioxide.  In humans their roles include collagen biosynthesis, fatty acid metab., DNA repair, RNA and chromatin modifications, and hypoxic sensing.  Com. applications of 2OG oxygenase inhibitors began with plant growth retardants, and now extend to a clin. used pharmaceutical compd. for cardioprotection.  Several 2OG oxygenases are now being targeted for therapeutic intervention for diseases including anemia, inflammation and cancer.  In this crit. review, we describe studies on the inhibition of 2OG oxygenases, focusing on small mols., and discuss the potential of 2OG oxygenases as therapeutic targets (295 refs.).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqp4Z13BJAo17Vg90H21EOLACvtfcHk0lhZ-PdS5OB0Mw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXovVemur8%253D&md5=f7a87e0bc9392d9a29f4ffb7d6f92b44</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1039%2Fc0cs00203h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc0cs00203h%26sid%3Dliteratum%253Aachs%26aulast%3DRose%26aufirst%3DN.%2BR.%26aulast%3DMcDonough%26aufirst%3DM.%2BA.%26aulast%3DKing%26aufirst%3DO.%2BN.%26aulast%3DKawamura%26aufirst%3DA.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26atitle%3DInhibition%2520of%25202-oxoglutarate%2520dependent%2520oxygenases%26jtitle%3DChem.%2520Soc.%2520Rev.%26date%3D2011%26volume%3D40%26spage%3D4364%26epage%3D4397%26doi%3D10.1039%2Fc0cs00203h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Thinnes, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">England, K. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawamura, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chowdhury, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hopkinson, R. J.</span><span> </span><span class="NLM_article-title">Targeting histone lysine demethylases - Progress, challenges, and the future</span> <span class="citation_source-journal">Biochim. Biophys. Acta, Gene Regul. Mech.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">1839</span><span class="NLM_x">, </span> <span class="NLM_fpage">1416</span><span class="NLM_x">–</span> <span class="NLM_lpage">1432</span><span class="refDoi"> DOI: 10.1016/j.bbagrm.2014.05.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1016%2Fj.bbagrm.2014.05.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=24859458" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVemtbrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1839&publication_year=2014&pages=1416-1432&author=C.+C.+Thinnesauthor=K.+S.+Englandauthor=A.+Kawamuraauthor=R.+Chowdhuryauthor=C.+J.+Schofieldauthor=R.+J.+Hopkinson&title=Targeting+histone+lysine+demethylases+-+Progress%2C+challenges%2C+and+the+future&doi=10.1016%2Fj.bbagrm.2014.05.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting histone lysine demethylases - Progress, challenges, and the future</span></div><div class="casAuthors">Thinnes, Cyrille C.; England, Katherine S.; Kawamura, Akane; Chowdhury, Rasheduzzaman; Schofield, Christopher J.; Hopkinson, Richard J.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Gene Regulatory Mechanisms</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">1839</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1416-1432</span>CODEN:
                <span class="NLM_cas:coden">BBAGC6</span>;
        ISSN:<span class="NLM_cas:issn">1874-9399</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  N-Methylation of lysine and arginine residues has emerged as a major mechanism of transcriptional regulation in eukaryotes.  In humans, Nε-methyllysine residue demethylation is catalyzed by two distinct subfamilies of demethylases (KDMs), the flavin-dependent KDM1 subfamily and the 2-oxoglutarate- (2OG) dependent JmjC subfamily, which both employ oxidative mechanisms.  Modulation of histone methylation status is proposed to be important in epigenetic regulation and has substantial medicinal potential for the treatment of diseases including cancer and genetic disorders.  This article provides an introduction to the enzymol. of the KDMs and the therapeutic possibilities and challenges assocd. with targeting them, followed by a review of reported KDM inhibitors and their mechanisms of action from kinetic and structural perspectives.  This article is part of a Special Issue entitled: Methylation: A Multifaceted Modification - looking at transcription and beyond.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_ltK_pdkkCrVg90H21EOLACvtfcHk0lhZ-PdS5OB0Mw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVemtbrE&md5=d5d70e8873451dbedb4f36c66defc040</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.bbagrm.2014.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbagrm.2014.05.009%26sid%3Dliteratum%253Aachs%26aulast%3DThinnes%26aufirst%3DC.%2BC.%26aulast%3DEngland%26aufirst%3DK.%2BS.%26aulast%3DKawamura%26aufirst%3DA.%26aulast%3DChowdhury%26aufirst%3DR.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26aulast%3DHopkinson%26aufirst%3DR.%2BJ.%26atitle%3DTargeting%2520histone%2520lysine%2520demethylases%2520-%2520Progress%252C%2520challenges%252C%2520and%2520the%2520future%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Gene%2520Regul.%2520Mech.%26date%3D2014%26volume%3D1839%26spage%3D1416%26epage%3D1432%26doi%3D10.1016%2Fj.bbagrm.2014.05.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Thalhammer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mecinovic, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loenarz, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tumber, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rose, N. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heightman, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span> </span><span class="NLM_article-title">Inhibition of the histone demethylase JMJD2E by 3-substituted pyridine 2,4-dicarboxylates</span> <span class="citation_source-journal">Org. Biomol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">127</span><span class="NLM_x">–</span> <span class="NLM_lpage">135</span><span class="refDoi"> DOI: 10.1039/C0OB00592D</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1039%2FC0OB00592D" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=21076780" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsFCqtL%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2011&pages=127-135&author=A.+Thalhammerauthor=J.+Mecinovicauthor=C.+Loenarzauthor=A.+Tumberauthor=N.+R.+Roseauthor=T.+D.+Heightmanauthor=C.+J.+Schofield&title=Inhibition+of+the+histone+demethylase+JMJD2E+by+3-substituted+pyridine+2%2C4-dicarboxylates&doi=10.1039%2FC0OB00592D"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of the histone demethylase JMJD2E by 3-substituted pyridine 2,4-dicarboxylates</span></div><div class="casAuthors">Thalhammer, Armin; Mecinovic, Jasmin; Loenarz, Christoph; Tumber, Anthony; Rose, Nathan R.; Heightman, Tom D.; Schofield, Christopher J.</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">127-135</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Based on structural anal. of the human 2-oxoglutarate (2OG) dependent JMJD2 histone Nε-Me lysyl demethylase family, 3-substituted pyridine 2,4-dicarboxylic acids were identified as potential inhibitors with possible selectivity over other human 2OG oxygenases.  Microwave-assisted palladium-catalyzed cross coupling methodol. was developed to install a diverse set of substituents on the sterically demanding C-3 position of a pyridine 2,4-dicarboxylate scaffold.  The subsequently prepd. di-acids were tested for in vitro inhibition of the histone demethylase JMJD2E and another human 2OG oxygenase, prolyl-hydroxylase domain isoform 2 (PHD2, EGLN1).  A subset of substitution patterns yielded inhibitors with selectivity for JMJD2E over PHD2, demonstrating that structure-based inhibitor design can enable selective inhibition of histone demethylases over related human 2OG oxygenases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqB-fCd3fbQqrVg90H21EOLACvtfcHk0lhZ-PdS5OB0Mw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsFCqtL%252FJ&md5=c4fdff0f41b967e6138b44d1849a05a7</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1039%2FC0OB00592D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC0OB00592D%26sid%3Dliteratum%253Aachs%26aulast%3DThalhammer%26aufirst%3DA.%26aulast%3DMecinovic%26aufirst%3DJ.%26aulast%3DLoenarz%26aufirst%3DC.%26aulast%3DTumber%26aufirst%3DA.%26aulast%3DRose%26aufirst%3DN.%2BR.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26atitle%3DInhibition%2520of%2520the%2520histone%2520demethylase%2520JMJD2E%2520by%25203-substituted%2520pyridine%25202%252C4-dicarboxylates%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2011%26volume%3D9%26spage%3D127%26epage%3D135%26doi%3D10.1039%2FC0OB00592D" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Majamaa, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hanauske-Abel, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gunzler, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kivirikko, K. I.</span><span> </span><span class="NLM_article-title">The 2-oxoglutarate binding site of prolyl 4-hydroxylase. Identification of distinct subsites and evidence for 2-oxoglutarate decarboxylation in a ligand reaction at the enzyme-bound ferrous ion</span> <span class="citation_source-journal">Eur. J. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1984</span><span class="NLM_x">, </span> <span class="NLM_volume">138</span><span class="NLM_x">, </span> <span class="NLM_fpage">239</span><span class="NLM_x">–</span> <span class="NLM_lpage">245</span><span class="refDoi"> DOI: 10.1111/j.1432-1033.1984.tb07907.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1111%2Fj.1432-1033.1984.tb07907.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=6321169" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADyaL2cXmsVSrsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=138&publication_year=1984&pages=239-245&author=K.+Majamaaauthor=H.+M.+Hanauske-Abelauthor=V.+Gunzlerauthor=K.+I.+Kivirikko&title=The+2-oxoglutarate+binding+site+of+prolyl+4-hydroxylase.+Identification+of+distinct+subsites+and+evidence+for+2-oxoglutarate+decarboxylation+in+a+ligand+reaction+at+the+enzyme-bound+ferrous+ion&doi=10.1111%2Fj.1432-1033.1984.tb07907.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">The 2-oxoglutarate-binding site of prolyl 4-hydroxylase. Identification of distinct subsites and evidence for 2-oxoglutarate decarboxylation in a ligand reaction at the enzyme-bound ferrous ion</span></div><div class="casAuthors">Majamaa, Kari; Hanauske-Abel, Hartmut M.; Guenzler, Volkmar; Kivirikko, Kari I.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Biochemistry</span>
        (<span class="NLM_cas:date">1984</span>),
    <span class="NLM_cas:volume">138</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">239-45</span>CODEN:
                <span class="NLM_cas:coden">EJBCAI</span>;
        ISSN:<span class="NLM_cas:issn">0014-2956</span>.
    </div><div class="casAbstract">The structure and function of the 2-oxoglutarate-binding site of prolyl 4-hydroxylase (I) was studied by assaying the inhibitory potential of 24 selected aliph. or arom. compds.  All except one of them inhibited I competitively with respect to 2-oxoglutarate and noncompetitively with respect to Fe2+, the Ki values ranging from 0.8 μM to >15 mM.  The Ki values for the 2 most effective inhibitors, pyridine 2,5-dicarboxylate and 2,4-dicarboxylate, were ∼0.8 and ∼2 μM, resp., these compds. being the most potent inhibitors of I with respect to 2-oxoglutarate known so far.  Only 1 of the compds. tested, 2-oxoadipinate, was able to support hydroxylation by replacing 2-oxoglutarate as a cosubstrate.  The data suggest that the 2-oxoglutarate-binding site can be divided into 3 distinct subsites.  Subsite I is probably a pos. charged side chain of the enzyme that ionically binds the C5 carboxyl group of the 2-oxoglutarate, subsite II consists of 2 cis-positioned equatorial coordination sites of the enzyme-bound Fe2+ and is chelated by the C1-C2 moiety, whereas subsite III involves a hydrophobic binding site in the C3-C4 region of the cosubstrate.  The sp3 rehybridization of C2 within the chelating moiety of the cosubstrate appears to be a crucial event during decarboxylation that proceeds in the form of a ligand reaction inside the Fe2+ coordination sphere.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0K7FKUN6as7Vg90H21EOLACvtfcHk0ljHSLTlgAyK0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2cXmsVSrsw%253D%253D&md5=8fc6c41b2a619c8443a92636a6d64488</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1111%2Fj.1432-1033.1984.tb07907.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1432-1033.1984.tb07907.x%26sid%3Dliteratum%253Aachs%26aulast%3DMajamaa%26aufirst%3DK.%26aulast%3DHanauske-Abel%26aufirst%3DH.%2BM.%26aulast%3DGunzler%26aufirst%3DV.%26aulast%3DKivirikko%26aufirst%3DK.%2BI.%26atitle%3DThe%25202-oxoglutarate%2520binding%2520site%2520of%2520prolyl%25204-hydroxylase.%2520Identification%2520of%2520distinct%2520subsites%2520and%2520evidence%2520for%25202-oxoglutarate%2520decarboxylation%2520in%2520a%2520ligand%2520reaction%2520at%2520the%2520enzyme-bound%2520ferrous%2520ion%26jtitle%3DEur.%2520J.%2520Biochem.%26date%3D1984%26volume%3D138%26spage%3D239%26epage%3D245%26doi%3D10.1111%2Fj.1432-1033.1984.tb07907.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">King, O. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, X. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sakurai, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawamura, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rose, N. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ng, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quinn, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rai, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mott, B. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beswick, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klose, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oppermann, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jadhav, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heightman, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maloney, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeonov, A.</span><span> </span><span class="NLM_article-title">Quantitative high-throughput screening identifies 8-hydroxyquinolines as cell-active histone demethylase inhibitors</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">e15535</span><span class="refDoi"> DOI: 10.1371/journal.pone.0015535</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1371%2Fjournal.pone.0015535" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=21124847" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A280%3ADC%252BC3M%252Fhs1Onsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2010&pages=e15535&author=O.+N.+Kingauthor=X.+S.+Liauthor=M.+Sakuraiauthor=A.+Kawamuraauthor=N.+R.+Roseauthor=S.+S.+Ngauthor=A.+M.+Quinnauthor=G.+Raiauthor=B.+T.+Mottauthor=P.+Beswickauthor=R.+J.+Kloseauthor=U.+Oppermannauthor=A.+Jadhavauthor=T.+D.+Heightmanauthor=D.+J.+Maloneyauthor=C.+J.+Schofieldauthor=A.+Simeonov&title=Quantitative+high-throughput+screening+identifies+8-hydroxyquinolines+as+cell-active+histone+demethylase+inhibitors&doi=10.1371%2Fjournal.pone.0015535"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Quantitative high-throughput screening identifies 8-hydroxyquinolines as cell-active histone demethylase inhibitors</span></div><div class="casAuthors">King Oliver N F; Li Xuan Shirley; Sakurai Masaaki; Kawamura Akane; Rose Nathan R; Ng Stanley S; Quinn Amy M; Rai Ganesha; Mott Bryan T; Beswick Paul; Klose Robert J; Oppermann Udo; Jadhav Ajit; Heightman Tom D; Maloney David J; Schofield Christopher J; Simeonov Anton</div><div class="citationInfo"><span class="NLM_cas:title">PloS one</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">e15535</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Small molecule modulators of epigenetic processes are currently sought as basic probes for biochemical mechanisms, and as starting points for development of therapeutic agents.  N(ε)-Methylation of lysine residues on histone tails is one of a number of post-translational modifications that together enable transcriptional regulation.  Histone lysine demethylases antagonize the action of histone methyltransferases in a site- and methylation state-specific manner.  N(ε)-Methyllysine demethylases that use 2-oxoglutarate as co-factor are associated with diverse human diseases, including cancer, inflammation and X-linked mental retardation; they are proposed as targets for the therapeutic modulation of transcription.  There are few reports on the identification of templates that are amenable to development as potent inhibitors in vivo and large diverse collections have yet to be exploited for the discovery of demethylase inhibitors.  PRINCIPAL FINDINGS:  High-throughput screening of a ∼236,000-member collection of diverse molecules arrayed as dilution series was used to identify inhibitors of the JMJD2 (KDM4) family of 2-oxoglutarate-dependent histone demethylases.  Initial screening hits were prioritized by a combination of cheminformatics, counterscreening using a coupled assay enzyme, and orthogonal confirmatory detection of inhibition by mass spectrometric assays.  Follow-up studies were carried out on one of the series identified, 8-hydroxyquinolines, which were shown by crystallographic analyses to inhibit by binding to the active site Fe(II) and to modulate demethylation at the H3K9 locus in a cell-based assay.  CONCLUSIONS:  These studies demonstrate that diverse compound screening can yield novel inhibitors of 2OG dependent histone demethylases and provide starting points for the development of potent and selective agents to interrogate epigenetic regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTwpX9dy1vWgwxyojvcjibhfW6udTcc2eaNYExrjWzobbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3M%252Fhs1Onsw%253D%253D&md5=d2d242c04dfb29ed4f2e00843101ae64</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0015535&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0015535%26sid%3Dliteratum%253Aachs%26aulast%3DKing%26aufirst%3DO.%2BN.%26aulast%3DLi%26aufirst%3DX.%2BS.%26aulast%3DSakurai%26aufirst%3DM.%26aulast%3DKawamura%26aufirst%3DA.%26aulast%3DRose%26aufirst%3DN.%2BR.%26aulast%3DNg%26aufirst%3DS.%2BS.%26aulast%3DQuinn%26aufirst%3DA.%2BM.%26aulast%3DRai%26aufirst%3DG.%26aulast%3DMott%26aufirst%3DB.%2BT.%26aulast%3DBeswick%26aufirst%3DP.%26aulast%3DKlose%26aufirst%3DR.%2BJ.%26aulast%3DOppermann%26aufirst%3DU.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DMaloney%26aufirst%3DD.%2BJ.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26aulast%3DSimeonov%26aufirst%3DA.%26atitle%3DQuantitative%2520high-throughput%2520screening%2520identifies%25208-hydroxyquinolines%2520as%2520cell-active%2520histone%2520demethylase%2520inhibitors%26jtitle%3DPLoS%2520One%26date%3D2010%26volume%3D5%26spage%3De15535%26doi%3D10.1371%2Fjournal.pone.0015535" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Hopkinson, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tumber, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yapp, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chowdhury, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aik, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Che, K. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, X. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kristensen, J. B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">King, O. N. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yeoh, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choi, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walport, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thinnes, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bush, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lejeune, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rydzik, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rose, N. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bagg, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDonough, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krojer, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yue, W. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ng, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olsen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brennan, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oppermann, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muller, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klose, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ratcliffe, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawamura, A.</span><span> </span><span class="NLM_article-title">5-Carboxy-8-hydroxyquinoline is a broad spectrum 2-oxoglutarate oxygenase inhibitor which causes iron translocation</span> <span class="citation_source-journal">Chem. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">3110</span><span class="NLM_x">–</span> <span class="NLM_lpage">3117</span><span class="refDoi"> DOI: 10.1039/c3sc51122g</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1039%2Fc3sc51122g" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=26682036" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVehs73L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=3110-3117&author=R.+J.+Hopkinsonauthor=A.+Tumberauthor=C.+Yappauthor=R.+Chowdhuryauthor=W.+Aikauthor=K.+H.+Cheauthor=X.+S.+Liauthor=J.+B.+L.+Kristensenauthor=O.+N.+F.+Kingauthor=M.+C.+Chanauthor=K.+K.+Yeohauthor=H.+Choiauthor=L.+J.+Walportauthor=C.+C.+Thinnesauthor=J.+T.+Bushauthor=C.+Lejeuneauthor=A.+M.+Rydzikauthor=N.+R.+Roseauthor=E.+A.+Baggauthor=M.+A.+McDonoughauthor=T.+J.+Krojerauthor=W.+W.+Yueauthor=S.+S.+Ngauthor=L.+Olsenauthor=P.+E.+Brennanauthor=U.+Oppermannauthor=S.+Mullerauthor=R.+J.+Kloseauthor=P.+J.+Ratcliffeauthor=C.+J.+Schofieldauthor=A.+Kawamura&title=5-Carboxy-8-hydroxyquinoline+is+a+broad+spectrum+2-oxoglutarate+oxygenase+inhibitor+which+causes+iron+translocation&doi=10.1039%2Fc3sc51122g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">5-Carboxy-8-hydroxyquinoline is a broad spectrum 2-oxoglutarate oxygenase inhibitor which causes iron translocation</span></div><div class="casAuthors">Hopkinson, Richard J.; Tumber, Anthony; Yapp, Clarence; Chowdhury, Rasheduzzaman; Aik, WeiShen; Che, Ka Hing; Li, Xuan Shirley; Kristensen, Jan B. L.; King, Oliver N. F.; Chan, Mun Chiang; Yeoh, Kar Kheng; Choi, Hwanho; Walport, Louise J.; Thinnes, Cyrille C.; Bush, Jacob T.; Lejeune, Clarisse; Rydzik, Anna M.; Rose, Nathan R.; Bagg, Eleanor A.; McDonough, Michael A.; Krojer, Tobias J.; Yue, Wyatt W.; Ng, Stanley S.; Olsen, Lars; Brennan, Paul E.; Oppermann, Udo; Mueller, Susanne; Klose, Robert J.; Ratcliffe, Peter J.; Schofield, Christopher J.; Kawamura, Akane</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3110-3117</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">2-Oxoglutarate and iron dependent oxygenases are therapeutic targets for human diseases.  Using a representative 2OG oxygenase panel, the authors compare the inhibitory activities of 5-carboxy-8-hydroxyquinoline (IOX1) and 4-carboxy-8-hydroxyquinoline (4C8HQ) with that of two other commonly used 2OG oxygenase inhibitors, N-oxalylglycine (NOG) and 2,4-pyridinedicarboxylic acid (2,4-PDCA).  IOX1 has a broad spectrum of activity, as demonstrated by the inhibition of transcription factor hydroxylases, representatives of all 2OG dependent histone demethylase subfamilies, nucleic acid demethylases and γ-butyrobetaine hydroxylase.  Cellular assays show that, unlike NOG and 2,4-PDCA, IOX1 is active against both cytosolic and nuclear 2OG oxygenases without ester derivatization.  Unexpectedly, crystallog. studies on these oxygenases demonstrate that IOX1, but not 4C8HQ, can cause translocation of the active site metal, revealing a rare example of protein ligand-induced metal movement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_NBtJvNEBOLVg90H21EOLACvtfcHk0ljZym2duprJrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVehs73L&md5=506fb4e75971c57a7a968f4d37805856</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1039%2Fc3sc51122g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc3sc51122g%26sid%3Dliteratum%253Aachs%26aulast%3DHopkinson%26aufirst%3DR.%2BJ.%26aulast%3DTumber%26aufirst%3DA.%26aulast%3DYapp%26aufirst%3DC.%26aulast%3DChowdhury%26aufirst%3DR.%26aulast%3DAik%26aufirst%3DW.%26aulast%3DChe%26aufirst%3DK.%2BH.%26aulast%3DLi%26aufirst%3DX.%2BS.%26aulast%3DKristensen%26aufirst%3DJ.%2BB.%2BL.%26aulast%3DKing%26aufirst%3DO.%2BN.%2BF.%26aulast%3DChan%26aufirst%3DM.%2BC.%26aulast%3DYeoh%26aufirst%3DK.%2BK.%26aulast%3DChoi%26aufirst%3DH.%26aulast%3DWalport%26aufirst%3DL.%2BJ.%26aulast%3DThinnes%26aufirst%3DC.%2BC.%26aulast%3DBush%26aufirst%3DJ.%2BT.%26aulast%3DLejeune%26aufirst%3DC.%26aulast%3DRydzik%26aufirst%3DA.%2BM.%26aulast%3DRose%26aufirst%3DN.%2BR.%26aulast%3DBagg%26aufirst%3DE.%2BA.%26aulast%3DMcDonough%26aufirst%3DM.%2BA.%26aulast%3DKrojer%26aufirst%3DT.%2BJ.%26aulast%3DYue%26aufirst%3DW.%2BW.%26aulast%3DNg%26aufirst%3DS.%2BS.%26aulast%3DOlsen%26aufirst%3DL.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DOppermann%26aufirst%3DU.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DKlose%26aufirst%3DR.%2BJ.%26aulast%3DRatcliffe%26aufirst%3DP.%2BJ.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26aulast%3DKawamura%26aufirst%3DA.%26atitle%3D5-Carboxy-8-hydroxyquinoline%2520is%2520a%2520broad%2520spectrum%25202-oxoglutarate%2520oxygenase%2520inhibitor%2520which%2520causes%2520iron%2520translocation%26jtitle%3DChem.%2520Sci.%26date%3D2013%26volume%3D4%26spage%3D3110%26epage%3D3117%26doi%3D10.1039%2Fc3sc51122g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Rose, N. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ng, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mecinovic, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lienard, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bello, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDonough, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oppermann, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span> </span><span class="NLM_article-title">Inhibitor scaffolds for 2-oxoglutarate-dependent histone lysine demethylases</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">7053</span><span class="NLM_x">–</span> <span class="NLM_lpage">7056</span><span class="refDoi"> DOI: 10.1021/jm800936s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm800936s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BD1cXht1Ors7rI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=7053-7056&author=N.+R.+Roseauthor=S.+S.+Ngauthor=J.+Mecinovicauthor=B.+M.+Lienardauthor=S.+H.+Belloauthor=Z.+Sunauthor=M.+A.+McDonoughauthor=U.+Oppermannauthor=C.+J.+Schofield&title=Inhibitor+scaffolds+for+2-oxoglutarate-dependent+histone+lysine+demethylases&doi=10.1021%2Fjm800936s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitor Scaffolds for 2-Oxoglutarate-Dependent Histone Lysine Demethylases</span></div><div class="casAuthors">Rose, Nathan R.; Ng, Stanley S.; Mecinovic, Jasmin; Lienard, Benoit M. R.; Bello, Simon H.; Sun, Zhe; McDonough, Michael A.; Oppermann, Udo; Schofield, Christopher J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">7053-7056</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The dynamic methylation of histone lysyl residues plays an important role in biol. by regulating transcription, maintaining genomic integrity, and by contributing to epigenetic effects.  Here we describe a variety of inhibitor scaffolds that inhibit the human 2-oxoglutarate-dependent JMJD2 subfamily of histone demethylases.  Combined with structural data, these chem. starting points will be useful to generate small-mol. probes to analyze the physiol. roles of these enzymes in epigenetic signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr90HQ8nfBLFrVg90H21EOLACvtfcHk0ljZym2duprJrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXht1Ors7rI&md5=155c830b0544a6c1d5e6eb81d14389de</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fjm800936s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm800936s%26sid%3Dliteratum%253Aachs%26aulast%3DRose%26aufirst%3DN.%2BR.%26aulast%3DNg%26aufirst%3DS.%2BS.%26aulast%3DMecinovic%26aufirst%3DJ.%26aulast%3DLienard%26aufirst%3DB.%2BM.%26aulast%3DBello%26aufirst%3DS.%2BH.%26aulast%3DSun%26aufirst%3DZ.%26aulast%3DMcDonough%26aufirst%3DM.%2BA.%26aulast%3DOppermann%26aufirst%3DU.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26atitle%3DInhibitor%2520scaffolds%2520for%25202-oxoglutarate-dependent%2520histone%2520lysine%2520demethylases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D7053%26epage%3D7056%26doi%3D10.1021%2Fjm800936s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varghese, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dellinger, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kumar, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Best, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruiz, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bruick, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pena-Llopis, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Babinski, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frantz, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brekken, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quinn, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeonov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Easmon, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martinez, E. D.</span><span> </span><span class="NLM_article-title">A small molecule modulates Jumonji histone demethylase activity and selectively inhibits cancer growth</span> <span class="citation_source-journal">Nat. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">2035</span><span class="refDoi"> DOI: 10.1038/ncomms3035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1038%2Fncomms3035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=23792809" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A280%3ADC%252BC3sjksV2iug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=2035&author=L.+Wangauthor=J.+Changauthor=D.+Vargheseauthor=M.+Dellingerauthor=S.+Kumarauthor=A.+M.+Bestauthor=J.+Ruizauthor=R.+Bruickauthor=S.+Pena-Llopisauthor=J.+Xuauthor=D.+J.+Babinskiauthor=D.+E.+Frantzauthor=R.+A.+Brekkenauthor=A.+M.+Quinnauthor=A.+Simeonovauthor=J.+Easmonauthor=E.+D.+Martinez&title=A+small+molecule+modulates+Jumonji+histone+demethylase+activity+and+selectively+inhibits+cancer+growth&doi=10.1038%2Fncomms3035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">A small molecule modulates Jumonji histone demethylase activity and selectively inhibits cancer growth</span></div><div class="casAuthors">Wang Lei; Chang Jianjun; Varghese Diana; Dellinger Michael; Kumar Subodh; Best Anne M; Ruiz Julio; Bruick Richard; Pena-Llopis Samuel; Xu Junjie; Babinski David J; Frantz Doug E; Brekken Rolf A; Quinn Amy M; Simeonov Anton; Easmon Johnny; Martinez Elisabeth D</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">2035</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The pharmacological inhibition of general transcriptional regulators has the potential to block growth through targeting multiple tumorigenic signalling pathways simultaneously.  Here, using an innovative cell-based screen, we identify a structurally unique small molecule (named JIB-04) that specifically inhibits the activity of the Jumonji family of histone demethylases in vitro, in cancer cells, and in tumours in vivo.  Unlike known inhibitors, JIB-04 is not a competitive inhibitor of α-ketoglutarate.  In cancer, but not in patient-matched normal cells, JIB-04 alters a subset of transcriptional pathways and blocks viability.  In mice, JIB-04 reduces tumour burden and prolongs survival.  Importantly, we find that patients with breast tumours that overexpress Jumonji demethylases have significantly lower survival.  Thus, JIB-04, a novel inhibitor of Jumonji demethylases in vitro and in vivo, constitutes a unique potential therapeutic and research tool against cancer, and validates the use of unbiased cellular screens to discover chemical modulators with disease relevance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ5QxHbn8WTZg20zIbnCCI3fW6udTcc2ebzgtZOuklK_rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3sjksV2iug%253D%253D&md5=37aa303d173d027fd711bc4ef7fd841b</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fncomms3035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms3035%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.%26aulast%3DChang%26aufirst%3DJ.%26aulast%3DVarghese%26aufirst%3DD.%26aulast%3DDellinger%26aufirst%3DM.%26aulast%3DKumar%26aufirst%3DS.%26aulast%3DBest%26aufirst%3DA.%2BM.%26aulast%3DRuiz%26aufirst%3DJ.%26aulast%3DBruick%26aufirst%3DR.%26aulast%3DPena-Llopis%26aufirst%3DS.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DBabinski%26aufirst%3DD.%2BJ.%26aulast%3DFrantz%26aufirst%3DD.%2BE.%26aulast%3DBrekken%26aufirst%3DR.%2BA.%26aulast%3DQuinn%26aufirst%3DA.%2BM.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DEasmon%26aufirst%3DJ.%26aulast%3DMartinez%26aufirst%3DE.%2BD.%26atitle%3DA%2520small%2520molecule%2520modulates%2520Jumonji%2520histone%2520demethylase%2520activity%2520and%2520selectively%2520inhibits%2520cancer%2520growth%26jtitle%3DNat.%2520Commun.%26date%3D2013%26volume%3D4%26spage%3D2035%26doi%3D10.1038%2Fncomms3035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Luo, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kubicek, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Myllyharju, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tumber, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ng, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Che, K. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Podoll, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heightman, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oppermann, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schreiber, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, X.</span><span> </span><span class="NLM_article-title">A selective inhibitor and probe of the cellular functions of Jumonji C domain-containing histone demethylases</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">133</span><span class="NLM_x">, </span> <span class="NLM_fpage">9451</span><span class="NLM_x">–</span> <span class="NLM_lpage">9456</span><span class="refDoi"> DOI: 10.1021/ja201597b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja201597b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmslGmurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2011&pages=9451-9456&author=X.+Luoauthor=Y.+Liuauthor=S.+Kubicekauthor=J.+Myllyharjuauthor=A.+Tumberauthor=S.+Ngauthor=K.+H.+Cheauthor=J.+Podollauthor=T.+D.+Heightmanauthor=U.+Oppermannauthor=S.+L.+Schreiberauthor=X.+Wang&title=A+selective+inhibitor+and+probe+of+the+cellular+functions+of+Jumonji+C+domain-containing+histone+demethylases&doi=10.1021%2Fja201597b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">A Selective Inhibitor and Probe of the Cellular Functions of Jumonji C Domain-Containing Histone Demethylases</span></div><div class="casAuthors">Luo, Xue-Lai; Liu, Yong-Xiang; Kubicek, Stefan; Myllyharju, Johanna; Tumber, Anthony; Ng, Stanley; Che, Ka-Hing; Podoll, Jessica; Heightman, Tom D.; Oppermann, Udo; Schreiber, Stuart L.; Wang, Xiang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">9451-9456</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Histone methylations are important chromatin marks that regulate gene expression, genomic stability, DNA repair, and genomic imprinting.  Histone demethylases are the most recent family of histone-modifying enzymes discovered.  Here, we report the characterization of a small-mol. inhibitor of Jumonji C domain-contg. histone demethylases.  The inhibitor derives from a structure-based design and preferentially inhibits the subfamily of tri-Me lysine demethylases.  Its Me ester prodrug, methylstat, selectively inhibits Jumonji C domain-contg. his-tone demethylases in cells and may be a useful small-mol. probe of chromatin and its role in epigenetics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcmCdVvaIpq7Vg90H21EOLACvtfcHk0lhi_hjoa_KafA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmslGmurg%253D&md5=ad50a692dfab85e1bcce77acc14b7b5a</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fja201597b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja201597b%26sid%3Dliteratum%253Aachs%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DKubicek%26aufirst%3DS.%26aulast%3DMyllyharju%26aufirst%3DJ.%26aulast%3DTumber%26aufirst%3DA.%26aulast%3DNg%26aufirst%3DS.%26aulast%3DChe%26aufirst%3DK.%2BH.%26aulast%3DPodoll%26aufirst%3DJ.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DOppermann%26aufirst%3DU.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26aulast%3DWang%26aufirst%3DX.%26atitle%3DA%2520selective%2520inhibitor%2520and%2520probe%2520of%2520the%2520cellular%2520functions%2520of%2520Jumonji%2520C%2520domain-containing%2520histone%2520demethylases%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2011%26volume%3D133%26spage%3D9451%26epage%3D9456%26doi%3D10.1021%2Fja201597b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Kruidenier, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liddle, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Che, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Joberty, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bantscheff, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bountra, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bridges, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diallo, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eberhard, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hutchinson, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Katso, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leveridge, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mander, P. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mosley, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramirez-Molina, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rowland, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sheppard, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swales, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tanner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomas, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tumber, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drewes, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oppermann, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, D. M.</span><span> </span><span class="NLM_article-title">A selective jumonji H3K27 demethylase inhibitor modulates the proinflammatory macrophage response</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">488</span><span class="NLM_x">, </span> <span class="NLM_fpage">404</span><span class="NLM_x">–</span> <span class="NLM_lpage">408</span><span class="refDoi"> DOI: 10.1038/nature11262</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1038%2Fnature11262" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=22842901" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFWjtbfI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=488&publication_year=2012&pages=404-408&author=L.+Kruidenierauthor=C.+W.+Chungauthor=Z.+Chengauthor=J.+Liddleauthor=K.+Cheauthor=G.+Jobertyauthor=M.+Bantscheffauthor=C.+Bountraauthor=A.+Bridgesauthor=H.+Dialloauthor=D.+Eberhardauthor=S.+Hutchinsonauthor=E.+Jonesauthor=R.+Katsoauthor=M.+Leveridgeauthor=P.+K.+Manderauthor=J.+Mosleyauthor=C.+Ramirez-Molinaauthor=P.+Rowlandauthor=C.+J.+Schofieldauthor=R.+J.+Sheppardauthor=J.+E.+Smithauthor=C.+Swalesauthor=R.+Tannerauthor=P.+Thomasauthor=A.+Tumberauthor=G.+Drewesauthor=U.+Oppermannauthor=D.+J.+Patelauthor=K.+Leeauthor=D.+M.+Wilson&title=A+selective+jumonji+H3K27+demethylase+inhibitor+modulates+the+proinflammatory+macrophage+response&doi=10.1038%2Fnature11262"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">A selective jumonji H3K27 demethylase inhibitor modulates the proinflammatory macrophage response</span></div><div class="casAuthors">Kruidenier, Laurens; Chung, Chun-wa; Cheng, Zhongjun; Liddle, John; Che, Ka Hing; Joberty, Gerard; Bantscheff, Marcus; Bountra, Chas; Bridges, Angela; Diallo, Hawa; Eberhard, Dirk; Hutchinson, Sue; Jones, Emma; Katso, Roy; Leveridge, Melanie; Mander, Palwinder K.; Mosley, Julie; Ramirez-Molina, Cesar; Rowland, Paul; Schofield, Christopher J.; Sheppard, Robert J.; Smith, Julia E.; Swales, Catherine; Tanner, Robert; Thomas, Pamela; Tumber, Anthony; Drewes, Gerard; Oppermann, Udo; Patel, Dinshaw J.; Lee, Kevin; Wilson, David M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">488</span>
        (<span class="NLM_cas:issue">7411</span>),
    <span class="NLM_cas:pages">404-408</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The jumonji (JMJ) family of histone demethylases are Fe2+- and α-ketoglutarate-dependent oxygenases that are essential components of regulatory transcriptional chromatin complexes.  These enzymes demethylate lysine residues in histones in a methylation-state and sequence-specific context.  Considerable effort has been devoted to gaining a mechanistic understanding of the roles of histone lysine demethylases in eukaryotic transcription, genome integrity and epigenetic inheritance, as well as in development, physiol. and disease.  However, because of the absence of any selective inhibitors, the relevance of the demethylase activity of JMJ enzymes in regulating cellular responses remains poorly understood.  Here we present a structure-guided small-mol. and chemoproteomics approach to elucidating the functional role of the H3K27me3-specific demethylase subfamily (KDM6 subfamily members JMJD3 and UTX).  The liganded structures of human and mouse JMJD3 provide novel insight into the specificity determinants for cofactor, substrate and inhibitor recognition by the KDM6 subfamily of demethylases.  We exploited these structural features to generate the first small-mol. catalytic site inhibitor that is selective for the H3K27me3-specific JMJ subfamily.  We demonstrate that this inhibitor binds in a novel manner and reduces lipopolysaccharide-induced proinflammatory cytokine prodn. by human primary macrophages, a process that depends on both JMJD3 and UTX.  Our results resolve the ambiguity assocd. with the catalytic function of H3K27-specific JMJs in regulating disease-relevant inflammatory responses and provide encouragement for designing small-mol. inhibitors to allow selective pharmacol. intervention across the JMJ family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-lxSNmxjfsbVg90H21EOLACvtfcHk0lhi_hjoa_KafA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFWjtbfI&md5=a4fa470a86070b4777e16a71e51763c8</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1038%2Fnature11262&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11262%26sid%3Dliteratum%253Aachs%26aulast%3DKruidenier%26aufirst%3DL.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DCheng%26aufirst%3DZ.%26aulast%3DLiddle%26aufirst%3DJ.%26aulast%3DChe%26aufirst%3DK.%26aulast%3DJoberty%26aufirst%3DG.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DBridges%26aufirst%3DA.%26aulast%3DDiallo%26aufirst%3DH.%26aulast%3DEberhard%26aufirst%3DD.%26aulast%3DHutchinson%26aufirst%3DS.%26aulast%3DJones%26aufirst%3DE.%26aulast%3DKatso%26aufirst%3DR.%26aulast%3DLeveridge%26aufirst%3DM.%26aulast%3DMander%26aufirst%3DP.%2BK.%26aulast%3DMosley%26aufirst%3DJ.%26aulast%3DRamirez-Molina%26aufirst%3DC.%26aulast%3DRowland%26aufirst%3DP.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26aulast%3DSheppard%26aufirst%3DR.%2BJ.%26aulast%3DSmith%26aufirst%3DJ.%2BE.%26aulast%3DSwales%26aufirst%3DC.%26aulast%3DTanner%26aufirst%3DR.%26aulast%3DThomas%26aufirst%3DP.%26aulast%3DTumber%26aufirst%3DA.%26aulast%3DDrewes%26aufirst%3DG.%26aulast%3DOppermann%26aufirst%3DU.%26aulast%3DPatel%26aufirst%3DD.%2BJ.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DWilson%26aufirst%3DD.%2BM.%26atitle%3DA%2520selective%2520jumonji%2520H3K27%2520demethylase%2520inhibitor%2520modulates%2520the%2520proinflammatory%2520macrophage%2520response%26jtitle%3DNature%26date%3D2012%26volume%3D488%26spage%3D404%26epage%3D408%26doi%3D10.1038%2Fnature11262" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Heinemann, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nielsen, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hudlebusch, H. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lees, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Larsen, D. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boesen, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Labelle, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerlach, L. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Birk, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Helin, K.</span><span> </span><span class="NLM_article-title">Inhibition of demethylases by GSK-J1/J4</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">514</span><span class="NLM_x">, </span> <span class="NLM_fpage">E1</span><span class="NLM_x">–</span> <span class="NLM_lpage">2</span><span class="refDoi"> DOI: 10.1038/nature13688</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1038%2Fnature13688" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=514&publication_year=2014&pages=E1-2&author=B.+Heinemannauthor=J.+M.+Nielsenauthor=H.+R.+Hudlebuschauthor=M.+J.+Leesauthor=D.+V.+Larsenauthor=T.+Boesenauthor=M.+Labelleauthor=L.+O.+Gerlachauthor=P.+Birkauthor=K.+Helin&title=Inhibition+of+demethylases+by+GSK-J1%2FJ4&doi=10.1038%2Fnature13688"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1038%2Fnature13688&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13688%26sid%3Dliteratum%253Aachs%26aulast%3DHeinemann%26aufirst%3DB.%26aulast%3DNielsen%26aufirst%3DJ.%2BM.%26aulast%3DHudlebusch%26aufirst%3DH.%2BR.%26aulast%3DLees%26aufirst%3DM.%2BJ.%26aulast%3DLarsen%26aufirst%3DD.%2BV.%26aulast%3DBoesen%26aufirst%3DT.%26aulast%3DLabelle%26aufirst%3DM.%26aulast%3DGerlach%26aufirst%3DL.%2BO.%26aulast%3DBirk%26aufirst%3DP.%26aulast%3DHelin%26aufirst%3DK.%26atitle%3DInhibition%2520of%2520demethylases%2520by%2520GSK-J1%252FJ4%26jtitle%3DNature%26date%3D2014%26volume%3D514%26spage%3DE1%26epage%3D2%26doi%3D10.1038%2Fnature13688" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Woon, E. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tumber, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawamura, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hillringhaus, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ge, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rose, N. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walport, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Che, K. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ng, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marsden, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oppermann, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDonough, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span> </span><span class="NLM_article-title">Linking of 2-oxoglutarate and substrate binding sites enables potent and highly selective inhibition of JmjC histone demethylases</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">1631</span><span class="NLM_x">–</span> <span class="NLM_lpage">1634</span><span class="refDoi"> DOI: 10.1002/anie.201107833</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1002%2Fanie.201107833" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2012&pages=1631-1634&author=E.+C.+Woonauthor=A.+Tumberauthor=A.+Kawamuraauthor=L.+Hillringhausauthor=W.+Geauthor=N.+R.+Roseauthor=J.+H.+Maauthor=M.+C.+Chanauthor=L.+J.+Walportauthor=K.+H.+Cheauthor=S.+S.+Ngauthor=B.+D.+Marsdenauthor=U.+Oppermannauthor=M.+A.+McDonoughauthor=C.+J.+Schofield&title=Linking+of+2-oxoglutarate+and+substrate+binding+sites+enables+potent+and+highly+selective+inhibition+of+JmjC+histone+demethylases&doi=10.1002%2Fanie.201107833"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1002%2Fanie.201107833&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201107833%26sid%3Dliteratum%253Aachs%26aulast%3DWoon%26aufirst%3DE.%2BC.%26aulast%3DTumber%26aufirst%3DA.%26aulast%3DKawamura%26aufirst%3DA.%26aulast%3DHillringhaus%26aufirst%3DL.%26aulast%3DGe%26aufirst%3DW.%26aulast%3DRose%26aufirst%3DN.%2BR.%26aulast%3DMa%26aufirst%3DJ.%2BH.%26aulast%3DChan%26aufirst%3DM.%2BC.%26aulast%3DWalport%26aufirst%3DL.%2BJ.%26aulast%3DChe%26aufirst%3DK.%2BH.%26aulast%3DNg%26aufirst%3DS.%2BS.%26aulast%3DMarsden%26aufirst%3DB.%2BD.%26aulast%3DOppermann%26aufirst%3DU.%26aulast%3DMcDonough%26aufirst%3DM.%2BA.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26atitle%3DLinking%2520of%25202-oxoglutarate%2520and%2520substrate%2520binding%2520sites%2520enables%2520potent%2520and%2520highly%2520selective%2520inhibition%2520of%2520JmjC%2520histone%2520demethylases%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2012%26volume%3D51%26spage%3D1631%26epage%3D1634%26doi%3D10.1002%2Fanie.201107833" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Murray, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blundell, T. L.</span><span> </span><span class="NLM_article-title">Structural biology in fragment-based drug design</span> <span class="citation_source-journal">Curr. Opin. Struct. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">497</span><span class="NLM_x">–</span> <span class="NLM_lpage">507</span><span class="refDoi"> DOI: 10.1016/j.sbi.2010.04.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1016%2Fj.sbi.2010.04.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=20471246" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVKhtLvF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2010&pages=497-507&author=C.+W.+Murrayauthor=T.+L.+Blundell&title=Structural+biology+in+fragment-based+drug+design&doi=10.1016%2Fj.sbi.2010.04.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Structural biology in fragment-based drug design</span></div><div class="casAuthors">Murray, Christopher W.; Blundell, Tom L.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">497-507</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Fragment-based ligand screening is now established as an emerging paradigm for drug discovery.  Here we examine the recent literature looking at how structural biol. has been used in a variety of successful fragment-screening applications.  We argue that the detn. of exptl. binding modes has proved to be one of the mainstays of successful fragment-based approaches and that this reflects the difficulty in optimizing a fragment to a lead mol. in the absence of structural information.  We focus on antimicrobial research where fragment-based drug discovery allows control of the phys. properties of the emerging lead mol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1S2lx8J-FX7Vg90H21EOLACvtfcHk0lj_ELTUhlgnlg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVKhtLvF&md5=ae8cfc0f8be47a316bd92073e95e0c8a</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.sbi.2010.04.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.sbi.2010.04.003%26sid%3Dliteratum%253Aachs%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DBlundell%26aufirst%3DT.%2BL.%26atitle%3DStructural%2520biology%2520in%2520fragment-based%2520drug%2520design%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2010%26volume%3D20%26spage%3D497%26epage%3D507%26doi%3D10.1016%2Fj.sbi.2010.04.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Hamada, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Itoh, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsumoto, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakagawa, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janknecht, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyata, N.</span><span> </span><span class="NLM_article-title">Synthesis and activity of N-oxalylglycine and its derivatives as Jumonji C-domain-containing histone lysine demethylase inhibitors</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">2852</span><span class="NLM_x">–</span> <span class="NLM_lpage">2855</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2009.03.098</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1016%2Fj.bmcl.2009.03.098" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=2852-2855&author=S.+Hamadaauthor=T.+D.+Kimauthor=T.+Suzukiauthor=Y.+Itohauthor=H.+Tsumotoauthor=H.+Nakagawaauthor=R.+Janknechtauthor=N.+Miyata&title=Synthesis+and+activity+of+N-oxalylglycine+and+its+derivatives+as+Jumonji+C-domain-containing+histone+lysine+demethylase+inhibitors&doi=10.1016%2Fj.bmcl.2009.03.098"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2009.03.098&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2009.03.098%26sid%3Dliteratum%253Aachs%26aulast%3DHamada%26aufirst%3DS.%26aulast%3DKim%26aufirst%3DT.%2BD.%26aulast%3DSuzuki%26aufirst%3DT.%26aulast%3DItoh%26aufirst%3DY.%26aulast%3DTsumoto%26aufirst%3DH.%26aulast%3DNakagawa%26aufirst%3DH.%26aulast%3DJanknecht%26aufirst%3DR.%26aulast%3DMiyata%26aufirst%3DN.%26atitle%3DSynthesis%2520and%2520activity%2520of%2520N-oxalylglycine%2520and%2520its%2520derivatives%2520as%2520Jumonji%2520C-domain-containing%2520histone%2520lysine%2520demethylase%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2009%26volume%3D19%26spage%3D2852%26epage%3D2855%26doi%3D10.1016%2Fj.bmcl.2009.03.098" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Hann, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leach, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harper, G.</span><span> </span><span class="NLM_article-title">Molecular complexity and its impact on the probability of finding leads for drug discovery</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">856</span><span class="NLM_x">–</span> <span class="NLM_lpage">864</span><span class="refDoi"> DOI: 10.1021/ci000403i</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci000403i" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BD3MXivFynt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2001&pages=856-864&author=M.+M.+Hannauthor=A.+R.+Leachauthor=G.+Harper&title=Molecular+complexity+and+its+impact+on+the+probability+of+finding+leads+for+drug+discovery&doi=10.1021%2Fci000403i"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Complexity and Its Impact on the Probability of Finding Leads for Drug Discovery</span></div><div class="casAuthors">Hann, Michael M.; Leach, Andrew R.; Harper, Gavin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">856-864</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Using a simple model of ligand-receptor interactions, the interactions between ligands and receptors of varying complexities are studied and the probabilities of binding calcd.  It is obsd. that as the systems become more complex the chance of observing a useful interaction for a randomly chosen ligand falls dramatically.  The implications of this for the design of combinatorial libraries is explored.  A large set of drug leads and optimized compds. is profiled using several different properties relevant to mol. recognition.  The changes obsd. for these properties during the drug optimization phase support the hypothesis that less complex mols. are more common starting points for the discovery of drugs.  An extreme example of the use of simple mols. for directed screening against thrombin is provided.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7ed40mt9qb7Vg90H21EOLACvtfcHk0lj_ELTUhlgnlg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXivFynt7w%253D&md5=e89e76c8fb83fc5012369ae30f20e6b7</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fci000403i&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci000403i%26sid%3Dliteratum%253Aachs%26aulast%3DHann%26aufirst%3DM.%2BM.%26aulast%3DLeach%26aufirst%3DA.%2BR.%26aulast%3DHarper%26aufirst%3DG.%26atitle%3DMolecular%2520complexity%2520and%2520its%2520impact%2520on%2520the%2520probability%2520of%2520finding%2520leads%2520for%2520drug%2520discovery%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2001%26volume%3D41%26spage%3D856%26epage%3D864%26doi%3D10.1021%2Fci000403i" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Hann, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keseru, G. M.</span><span> </span><span class="NLM_article-title">Finding the sweet spot: the role of nature and nurture in medicinal chemistry</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">355</span><span class="NLM_x">–</span> <span class="NLM_lpage">365</span><span class="refDoi"> DOI: 10.1038/nrd3701</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1038%2Fnrd3701" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=22543468" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BC38Xmt1GqsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=355-365&author=M.+M.+Hannauthor=G.+M.+Keseru&title=Finding+the+sweet+spot%3A+the+role+of+nature+and+nurture+in+medicinal+chemistry&doi=10.1038%2Fnrd3701"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Finding the sweet spot: the role of nature and nurture in medicinal chemistry</span></div><div class="casAuthors">Hann, Michael M.; Keserue, Gyoergy M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">355-365</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Given its position at the heart of small-mol. drug discovery, medicinal chem. has an important role in tackling the well-known productivity challenges in pharmaceutical research and development.  In recent years, extensive analyses of successful and failed discovery compds. and drug candidates have improved our understanding of the role of physicochem. properties in drug attrition.  Based on the clarified challenges in finding the 'sweet spot' in medicinal chem. programs, we suggest that this goal can be achieved through a combination of first identifying chem. starting points with appropriate 'nature' and then rigorously 'nurturing' them during lead optimization.  Here, we discuss scientific, strategic, organizational and cultural considerations for medicinal chem. practices, with the aim of promoting more effective use of what is already known, as well as a wider appreciation of the risks of pursuing suboptimal compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrURciGjB2A3LVg90H21EOLACvtfcHk0ljljktEN2aRfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xmt1GqsLY%253D&md5=90170e3f4e69e52bfa38667df380516a</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1038%2Fnrd3701&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3701%26sid%3Dliteratum%253Aachs%26aulast%3DHann%26aufirst%3DM.%2BM.%26aulast%3DKeseru%26aufirst%3DG.%2BM.%26atitle%3DFinding%2520the%2520sweet%2520spot%253A%2520the%2520role%2520of%2520nature%2520and%2520nurture%2520in%2520medicinal%2520chemistry%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2012%26volume%3D11%26spage%3D355%26epage%3D365%26doi%3D10.1038%2Fnrd3701" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Murray, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carr, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Callaghan, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chessari, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Congreve, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coyle, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Downham, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Figueroa, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frederickson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Graham, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McMenamin, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Brien, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Phillips, T. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woodhead, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woolford, A. J.</span><span> </span><span class="NLM_article-title">Fragment-based drug discovery applied to Hsp90. Discovery of two lead series with high ligand efficiency</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">5942</span><span class="NLM_x">–</span> <span class="NLM_lpage">5955</span><span class="refDoi"> DOI: 10.1021/jm100059d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm100059d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BC3cXpsVChsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=5942-5955&author=C.+W.+Murrayauthor=M.+G.+Carrauthor=O.+Callaghanauthor=G.+Chessariauthor=M.+Congreveauthor=S.+Cowanauthor=J.+E.+Coyleauthor=R.+Downhamauthor=E.+Figueroaauthor=M.+Fredericksonauthor=B.+Grahamauthor=R.+McMenaminauthor=M.+A.+O%E2%80%99Brienauthor=S.+Patelauthor=T.+R.+Phillipsauthor=G.+Williamsauthor=A.+J.+Woodheadauthor=A.+J.+Woolford&title=Fragment-based+drug+discovery+applied+to+Hsp90.+Discovery+of+two+lead+series+with+high+ligand+efficiency&doi=10.1021%2Fjm100059d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Drug Discovery Applied to Hsp90. Discovery of Two Lead Series with High Ligand Efficiency</span></div><div class="casAuthors">Murray, Christopher W.; Carr, Maria G.; Callaghan, Owen; Chessari, Gianni; Congreve, Miles; Cowan, Suzanna; Coyle, Joseph E.; Downham, Robert; Figueroa, Eva; Frederickson, Martyn; Graham, Brent; McMenamin, Rachel; O'Brien, M. Alistair; Patel, Sahil; Phillips, Theresa R.; Williams, Glyn; Woodhead, Andrew J.; Woolford, Alison J.-A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">5942-5955</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Inhibitors of the chaperone Hsp90 are potentially useful as chemotherapeutic agents in cancer.  This paper describes an application of fragment screening to Hsp90 using a combination of NMR and high throughput x-ray crystallog.  The screening identified an aminopyrimidine with affinity in the high micromolar range and subsequent structure-based design allowed its optimization into a low nanomolar series with good ligand efficiency.  A phenolic chemotype was also identified in fragment screening and was found to bind with affinity close to 1 mM.  This fragment was optimized using structure based design into a resorcinol lead which has subnanomolar affinity for Hsp90, excellent cell potency, and good ligand efficiency.  This fragment to lead campaign improved affinity for Hsp90 by over 1,000,000 fold with the addn. of only 6 heavy atoms.  The companion paper (DOI: 10.1021/jm100060b) describes how the resorcinol lead was optimized into a compd. that is now in clin. trials for the treatment of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqISC3cw2labbVg90H21EOLACvtfcHk0ljljktEN2aRfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXpsVChsb0%253D&md5=d1ad900aa5a45ba6ba3a5e9ccdcd96b3</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fjm100059d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100059d%26sid%3Dliteratum%253Aachs%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DCarr%26aufirst%3DM.%2BG.%26aulast%3DCallaghan%26aufirst%3DO.%26aulast%3DChessari%26aufirst%3DG.%26aulast%3DCongreve%26aufirst%3DM.%26aulast%3DCowan%26aufirst%3DS.%26aulast%3DCoyle%26aufirst%3DJ.%2BE.%26aulast%3DDownham%26aufirst%3DR.%26aulast%3DFigueroa%26aufirst%3DE.%26aulast%3DFrederickson%26aufirst%3DM.%26aulast%3DGraham%26aufirst%3DB.%26aulast%3DMcMenamin%26aufirst%3DR.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DM.%2BA.%26aulast%3DPatel%26aufirst%3DS.%26aulast%3DPhillips%26aufirst%3DT.%2BR.%26aulast%3DWilliams%26aufirst%3DG.%26aulast%3DWoodhead%26aufirst%3DA.%2BJ.%26aulast%3DWoolford%26aufirst%3DA.%2BJ.%26atitle%3DFragment-based%2520drug%2520discovery%2520applied%2520to%2520Hsp90.%2520Discovery%2520of%2520two%2520lead%2520series%2520with%2520high%2520ligand%2520efficiency%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D5942%26epage%3D5955%26doi%3D10.1021%2Fjm100059d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Bembenek, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tounge, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reynolds, C. H.</span><span> </span><span class="NLM_article-title">Ligand efficiency and fragment-based drug discovery</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">278</span><span class="NLM_x">–</span> <span class="NLM_lpage">283</span><span class="refDoi"> DOI: 10.1016/j.drudis.2008.11.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1016%2Fj.drudis.2008.11.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=19073276" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjt1Sht78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2009&pages=278-283&author=S.+D.+Bembenekauthor=B.+A.+Toungeauthor=C.+H.+Reynolds&title=Ligand+efficiency+and+fragment-based+drug+discovery&doi=10.1016%2Fj.drudis.2008.11.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand efficiency and fragment-based drug discovery</span></div><div class="casAuthors">Bembenek, Scott D.; Tounge, Brett A.; Reynolds, Charles H.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">5/6</span>),
    <span class="NLM_cas:pages">278-283</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The use of fragment-based drug discovery (FBDD) has increased in recent years since it is more likely to produce a better optimized compd. of lower mol. wt.  Ligand efficiency (LE) has become important for assessing fragments, HTS hits, and resulting optimized ligands.  LE is useful for comparing ligands of equal mol. wt., but is ineffective for comparisons of ligands of differing mol. wt.  LE has a strong dependence on mol. size, which has led us to develop a size-independent efficiency score termed fit quality.  Evaluating FBDD examples from the literature using LE and fit quality, we find that, in general, the LEs of starting fragments are greater than those of larger, more elaborated, structures.  Fit quality scores, however, tend to improve upon optimization of the fragments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrIWfRm1zWl87Vg90H21EOLACvtfcHk0liq7wRviX4Swg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjt1Sht78%253D&md5=55b35499fb26249d535bbe586f74f38f</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2008.11.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2008.11.007%26sid%3Dliteratum%253Aachs%26aulast%3DBembenek%26aufirst%3DS.%2BD.%26aulast%3DTounge%26aufirst%3DB.%2BA.%26aulast%3DReynolds%26aufirst%3DC.%2BH.%26atitle%3DLigand%2520efficiency%2520and%2520fragment-based%2520drug%2520discovery%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2009%26volume%3D14%26spage%3D278%26epage%3D283%26doi%3D10.1016%2Fj.drudis.2008.11.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Lipinski, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hopkins, A.</span><span> </span><span class="NLM_article-title">Navigating chemical space for biology and medicine</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">432</span><span class="NLM_x">, </span> <span class="NLM_fpage">855</span><span class="NLM_x">–</span> <span class="NLM_lpage">861</span><span class="refDoi"> DOI: 10.1038/nature03193</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1038%2Fnature03193" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=15602551" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVOht7jO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=432&publication_year=2004&pages=855-861&author=C.+Lipinskiauthor=A.+Hopkins&title=Navigating+chemical+space+for+biology+and+medicine&doi=10.1038%2Fnature03193"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Navigating chemical space for biology and medicine</span></div><div class="casAuthors">Lipinski, Christopher; Hopkins, Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">432</span>
        (<span class="NLM_cas:issue">7019</span>),
    <span class="NLM_cas:pages">855-861</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Despite over a century of applying org. synthesis to the search for drugs, the authors are still far from even a cursory examn. of the vast no. of possible small mols. that could be created.  Indeed, a thorough examn. of all 'chem. space' is practically impossible.  Given this, what are the best strategies for identifying small mols. that modulate biol. targets And how might such strategies differ, depending on whether the primary goal is to understand biol. systems or to develop potential drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3hQm7xaX5MLVg90H21EOLACvtfcHk0liq7wRviX4Swg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVOht7jO&md5=077237d55fa1225fd0aaee481e376172</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1038%2Fnature03193&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature03193%26sid%3Dliteratum%253Aachs%26aulast%3DLipinski%26aufirst%3DC.%26aulast%3DHopkins%26aufirst%3DA.%26atitle%3DNavigating%2520chemical%2520space%2520for%2520biology%2520and%2520medicine%26jtitle%3DNature%26date%3D2004%26volume%3D432%26spage%3D855%26epage%3D861%26doi%3D10.1038%2Fnature03193" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sterling, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mysinger, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bolstad, E. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coleman, R. G.</span><span> </span><span class="NLM_article-title">ZINC: a free tool to discover chemistry for biology</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">1757</span><span class="NLM_x">–</span> <span class="NLM_lpage">1768</span><span class="refDoi"> DOI: 10.1021/ci3001277</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci3001277" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BC38XmvFGnsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=1757-1768&author=J.+J.+Irwinauthor=T.+Sterlingauthor=M.+M.+Mysingerauthor=E.+S.+Bolstadauthor=R.+G.+Coleman&title=ZINC%3A+a+free+tool+to+discover+chemistry+for+biology&doi=10.1021%2Fci3001277"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">ZINC: A Free Tool to Discover Chemistry for Biology</span></div><div class="casAuthors">Irwin, John J.; Sterling, Teague; Mysinger, Michael M.; Bolstad, Erin S.; Coleman, Ryan G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1757-1768</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">ZINC is a free public resource for ligand discovery.  The database contains over twenty million com. available mols. in biol. relevant representations that may be downloaded in popular ready-to-dock formats and subsets.  The Web site also enables searches by structure, biol. activity, phys. property, vendor, catalog no., name, and CAS no.  Small custom subsets may be created, edited, shared, docked, downloaded, and conveyed to a vendor for purchase.  The database is maintained and curated for a high purchasing success rate and is freely available at zinc.docking.org.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotyhntxgvRpbVg90H21EOLACvtfcHk0liq7wRviX4Swg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmvFGnsrg%253D&md5=97f2ede64afc6b5e3ea2f279e38e32a0</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fci3001277&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci3001277%26sid%3Dliteratum%253Aachs%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DSterling%26aufirst%3DT.%26aulast%3DMysinger%26aufirst%3DM.%2BM.%26aulast%3DBolstad%26aufirst%3DE.%2BS.%26aulast%3DColeman%26aufirst%3DR.%2BG.%26atitle%3DZINC%253A%2520a%2520free%2520tool%2520to%2520discover%2520chemistry%2520for%2520biology%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D1757%26epage%3D1768%26doi%3D10.1021%2Fci3001277" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Mysinger, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Rapid context-dependent ligand desolvation in molecular docking</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">1561</span><span class="NLM_x">–</span> <span class="NLM_lpage">1573</span><span class="refDoi"> DOI: 10.1021/ci100214a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100214a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVGrsLvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=1561-1573&author=M.+M.+Mysingerauthor=B.+K.+Shoichet&title=Rapid+context-dependent+ligand+desolvation+in+molecular+docking&doi=10.1021%2Fci100214a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid Context-Dependent Ligand Desolvation in Molecular Docking</span></div><div class="casAuthors">Mysinger, Michael M.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1561-1573</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In structure-based screens for new ligands, a mol. docking algorithm must rapidly score many mols. in multiple configurations, accounting for both the ligand's interactions with receptor and its competing interactions with solvent.  Here the authors explore a context-dependent ligand desolvation scoring term for mol. docking.  The authors relate the Generalized-Born effective Born radii for every ligand atom to a fractional desolvation and then use this fraction to scale an atom-by-atom decompn. of the full transfer free energy.  The fractional desolvation is precomputed on a scoring grid by numerically integrating over the vol. of receptor proximal to a ligand atom, weighted by distance.  To test this method's performance, the authors dock ligands vs. property-matched decoys over 40 DUD targets.  Context-dependent desolvation better enriches ligands compared to both the raw full transfer free energy penalty and compared to ignoring desolvation altogether, though the improvement is modest.  More compellingly, the new method improves docking performance across receptor types.  Thus, whereas entirely ignoring desolvation works best for charged sites and overpenalizing with full desolvation works well for neutral sites, the phys. more correct context-dependent ligand desolvation is competitive across both types of targets.  The method also reliably discriminates ligands from highly charged mols., where ignoring desolvation performs poorly.  Since this context-dependent ligand desolvation may be precalcd., it improves docking reliability with minimal cost to calcn. time and may be readily incorporated into any physics-based docking program.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtPnS0XiKpYLVg90H21EOLACvtfcHk0liq7wRviX4Swg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVGrsLvI&md5=5124d2c4618e07b4f1e41609c9e61362</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fci100214a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100214a%26sid%3Dliteratum%253Aachs%26aulast%3DMysinger%26aufirst%3DM.%2BM.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DRapid%2520context-dependent%2520ligand%2520desolvation%2520in%2520molecular%2520docking%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D1561%26epage%3D1573%26doi%3D10.1021%2Fci100214a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Meng, E. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuntz, I. D.</span><span> </span><span class="NLM_article-title">Automated Docking with Grid-Based Energy Evaluation</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1992</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">505</span><span class="NLM_x">–</span> <span class="NLM_lpage">524</span><span class="refDoi"> DOI: 10.1002/jcc.540130412</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1002%2Fjcc.540130412" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADyaK38Xit1Omt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=1992&pages=505-524&author=E.+C.+Mengauthor=B.+K.+Shoichetauthor=I.+D.+Kuntz&title=Automated+Docking+with+Grid-Based+Energy+Evaluation&doi=10.1002%2Fjcc.540130412"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Automated docking with grid-based energy evaluation</span></div><div class="casAuthors">Meng, Elaine C.; Shoichet, Brian K.; Kuntz, Irwin D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">505-24</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    </div><div class="casAbstract">The ability to generate feasible binding orientations of a small mol. within a site of known structure is important for ligand design.  The authors present a method that combines a rapid, geometric docking algorithm with the evaluation of mol. mechanics interaction energies.  The computational costs of evaluation are minimal because the authors precalc. the receptor-dependent terms in the potential function at points on a three-dimensional grid.  In four test cases where the components of crystallog. detd. complexes are redocked, the "force field" score correctly identifies the family of orientations closest to the exptl. binding geometry.  Scoring functions that consider only steric factors or only electrostatic factors are less successful.  The force field function will play an important role in efforts to search databases for potential lead compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKAw7Zj_QFSbVg90H21EOLACvtfcHk0lgmMQGBwHAhNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38Xit1Omt7c%253D&md5=f472fffa0c9a61652b4c20e4dbbba69e</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1002%2Fjcc.540130412&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.540130412%26sid%3Dliteratum%253Aachs%26aulast%3DMeng%26aufirst%3DE.%2BC.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DKuntz%26aufirst%3DI.%2BD.%26atitle%3DAutomated%2520Docking%2520with%2520Grid-Based%2520Energy%2520Evaluation%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1992%26volume%3D13%26spage%3D505%26epage%3D524%26doi%3D10.1002%2Fjcc.540130412" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">Gilson, M. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Honig, B.</span><span> </span><span class="NLM_article-title">The inclusion of electrostatic hydration energies in molecular mechanics calculations</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1991</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">5</span><span class="NLM_x">–</span> <span class="NLM_lpage">20</span><span class="refDoi"> DOI: 10.1007/BF00173467</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1007%2FBF00173467" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=2072125" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADyaK3MXlt1Shtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=1991&pages=5-20&author=M.+K.+Gilsonauthor=B.+Honig&title=The+inclusion+of+electrostatic+hydration+energies+in+molecular+mechanics+calculations&doi=10.1007%2FBF00173467"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">The inclusion of electrostatic hydration energies in molecular mechanics calculations</span></div><div class="casAuthors">Gilson, Michael K.; Honig, Barry</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">5-20</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    </div><div class="casAbstract">The problem of including solvent effects in mol. mechanics calcns. is discussed.  It is argued that the neglect of charge-solvent (solvation) interactions can introduce significant errors.  The finite difference Poisson-Boltzmann (FDPB) method for calcg. electrostatic interactions is summarized and is used as a basis for introducing a new pairwise energy term which accounts for charge-solvent interactions.  This term acts between all pairs of atoms usually considered in mol. mechanics calcns. and can be easily incorporated into existing force fields.  As an example, a parameterization is developed for the CHARMm force field and the results compared to the predictions of the FDPB method.  An approach to the realistic incorporation of solvent screening into force fields is also outlined.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhzyjxAJyAZrVg90H21EOLACvtfcHk0lgmMQGBwHAhNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXlt1Shtr0%253D&md5=844bc05c190cb622320aeb0564d922a7</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1007%2FBF00173467&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF00173467%26sid%3Dliteratum%253Aachs%26aulast%3DGilson%26aufirst%3DM.%2BK.%26aulast%3DHonig%26aufirst%3DB.%26atitle%3DThe%2520inclusion%2520of%2520electrostatic%2520hydration%2520energies%2520in%2520molecular%2520mechanics%2520calculations%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D1991%26volume%3D5%26spage%3D5%26epage%3D20%26doi%3D10.1007%2FBF00173467" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raushel, F. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Virtual screening against metalloenzymes for inhibitors and substrates</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">12316</span><span class="NLM_x">–</span> <span class="NLM_lpage">12328</span><span class="refDoi"> DOI: 10.1021/bi050801k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi050801k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BD2MXovFWhtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2005&pages=12316-12328&author=J.+J.+Irwinauthor=F.+M.+Raushelauthor=B.+K.+Shoichet&title=Virtual+screening+against+metalloenzymes+for+inhibitors+and+substrates&doi=10.1021%2Fbi050801k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual Screening against Metalloenzymes for Inhibitors and Substrates</span></div><div class="casAuthors">Irwin, John J.; Raushel, Frank M.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">37</span>),
    <span class="NLM_cas:pages">12316-12328</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. docking uses the three-dimensional structure of a receptor to screen databases of small mols. for potential ligands, often based on energetic complementarity.  For many docking scoring functions, which calc. nonbonded interactions, metalloenzymes are challenging because of the partial covalent nature of metal-ligand interactions.  To investigate how well mol. docking can identify potential ligands of metalloenzymes using a "std." scoring function, we have docked the MDL Drug Data Report (MDDR), a functionally annotated database of 95 000 small mols., against the X-ray crystal structures of five metalloenzymes.  These enzymes included three zinc proteases, the nickel analog of an iron enzyme, and a molybdenum metalloenzyme.  The ability of the docking program to retrospectively enrich the annotated ligands as high-scoring hits for each enzyme and to calc. proper geometries was evaluated.  In all five systems, the annotated ligands within the MDDR were enriched at least 20 times over random.  To test the approach prospectively, a sixth target, the zinc β-lactamase from Bacteroides fragilis, was screened against the fragment-like subset of the ZINC database.  We purchased and tested 15 compds. from among the top 50 top-ranked ligands from docking, and found 5 inhibitors with apparent Ki values less than 120 μM, the best of which was 2 μM.  A more ambitious test still was predicting actual substrates for a seventh target, a Zn-dependent phosphotriesterase from Pseudomonas diminuta.  Screening the Available Chems. Directory (ACD) identified 25 thiophosphate esters as potential substrates within the top 100 ranked compds.  Eight of these, all previously uncharacterized for this enzyme, were acquired and tested, and all were confirmed exptl. as substrates.  These results suggest that a simple, noncovalent scoring function may be used to identify inhibitors of at least some metalloenzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhSirdbjLQSrVg90H21EOLACvtfcHk0lgmMQGBwHAhNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXovFWhtrc%253D&md5=f0bd25b13e227ed670ad2a2abf25abef</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fbi050801k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi050801k%26sid%3Dliteratum%253Aachs%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DRaushel%26aufirst%3DF.%2BM.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DVirtual%2520screening%2520against%2520metalloenzymes%2520for%2520inhibitors%2520and%2520substrates%26jtitle%3DBiochemistry%26date%3D2005%26volume%3D44%26spage%3D12316%26epage%3D12328%26doi%3D10.1021%2Fbi050801k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group">Babaoglu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeonov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nelson, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomas, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cancian, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Costi, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maltby, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jadhav, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Inglese, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Austin, C. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Comprehensive mechanistic analysis of hits from high-throughput and docking screens against beta-lactamase</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">2502</span><span class="NLM_x">–</span> <span class="NLM_lpage">2511</span><span class="refDoi"> DOI: 10.1021/jm701500e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm701500e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjtFSksb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=2502-2511&author=K.+Babaogluauthor=A.+Simeonovauthor=J.+J.+Irwinauthor=M.+E.+Nelsonauthor=B.+Fengauthor=C.+J.+Thomasauthor=L.+Cancianauthor=M.+P.+Costiauthor=D.+A.+Maltbyauthor=A.+Jadhavauthor=J.+Ingleseauthor=C.+P.+Austinauthor=B.+K.+Shoichet&title=Comprehensive+mechanistic+analysis+of+hits+from+high-throughput+and+docking+screens+against+beta-lactamase&doi=10.1021%2Fjm701500e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive Mechanistic Analysis of Hits from High-Throughput and Docking Screens against β-Lactamase</span></div><div class="casAuthors">Babaoglu, Kerim; Simeonov, Anton; Irwin, John J.; Nelson, Michael E.; Feng, Brian; Thomas, Craig J.; Cancian, Laura; Costi, M. Paola; Maltby, David A.; Jadhav, Ajit; Inglese, James; Austin, Christopher P.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2502-2511</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">High-throughput screening (HTS) is widely used in drug discovery.  Esp. for screens of unbiased libraries, false positives can dominate "hit lists"; their origins are much debated.  Here we det. the mechanism of every active hit from a screen of 70,563 unbiased mols. against β-lactamase using quant. HTS (qHTS).  Of the 1274 initial inhibitors, 95% were detergent-sensitive and were classified as aggregators.  Among the 70 remaining were 25 potent, covalent-acting β-lactams.  Mass spectra, counter-screens, and crystallog. identified 12 as promiscuous covalent inhibitors.  The remaining 33 were either aggregators or irreproducible.  No specific reversible inhibitors were found.  We turned to mol. docking to prioritize mols. from the same library for testing at higher concns.  Of 16 tested, 2 were modest inhibitors.  Subsequent X-ray structures corresponded to the docking prediction.  Analog synthesis improved affinity to 8 μM.  These results suggest that it may be the phys. behavior of org. mols., not their reactivity, that accounts for most screening artifacts.  Structure-based methods may prioritize weak-but-novel chemotypes in unbiased library screens.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNtnaCegS8aLVg90H21EOLACvtfcHk0lghMTWGIdEqQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjtFSksb0%253D&md5=7736429aeafc6b17a5d0182e24df0dd4</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Fjm701500e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm701500e%26sid%3Dliteratum%253Aachs%26aulast%3DBabaoglu%26aufirst%3DK.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DNelson%26aufirst%3DM.%2BE.%26aulast%3DFeng%26aufirst%3DB.%26aulast%3DThomas%26aufirst%3DC.%2BJ.%26aulast%3DCancian%26aufirst%3DL.%26aulast%3DCosti%26aufirst%3DM.%2BP.%26aulast%3DMaltby%26aufirst%3DD.%2BA.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DInglese%26aufirst%3DJ.%26aulast%3DAustin%26aufirst%3DC.%2BP.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DComprehensive%2520mechanistic%2520analysis%2520of%2520hits%2520from%2520high-throughput%2520and%2520docking%2520screens%2520against%2520beta-lactamase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D2502%26epage%3D2511%26doi%3D10.1021%2Fjm701500e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group">Mysinger, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weiss, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ziarek, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gravel, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doak, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karpiak, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heveker, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Volkman, B. F.</span><span> </span><span class="NLM_article-title">Structure-based ligand discovery for the protein-protein interface of chemokine receptor CXCR4</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">109</span><span class="NLM_x">, </span> <span class="NLM_fpage">5517</span><span class="NLM_x">–</span> <span class="NLM_lpage">5522</span><span class="refDoi"> DOI: 10.1073/pnas.1120431109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1073%2Fpnas.1120431109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=22431600" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BC38XlslOjtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=5517-5522&author=M.+M.+Mysingerauthor=D.+R.+Weissauthor=J.+J.+Ziarekauthor=S.+Gravelauthor=A.+K.+Doakauthor=J.+Karpiakauthor=N.+Hevekerauthor=B.+K.+Shoichetauthor=B.+F.+Volkman&title=Structure-based+ligand+discovery+for+the+protein-protein+interface+of+chemokine+receptor+CXCR4&doi=10.1073%2Fpnas.1120431109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based ligand discovery for the protein-protein interface of chemokine receptor CXCR4</span></div><div class="casAuthors">Mysinger, Michael M.; Weiss, Dahlia R.; Ziarek, Joshua J.; Gravel, Stephanie; Doak, Allison K.; Karpiak, Joel; Heveker, Nikolaus; Shoichet, Brian K.; Volkman, Brian F.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">5517-5522, S5517/1-S5517/28</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">G-protein-coupled receptors (GPCRs) are key signaling mols. and are intensely studied.  Whereas GPCRs recognizing small-mols. have been successfully targeted for drug discovery, protein-recognizing GPCRs, such as the chemokine receptors, claim few drugs or even useful small mol. reagents.  This reflects both the difficulties that attend protein-protein interface inhibitor discovery, and the lack of structures for these targets.  Imminent structure detn. of chemokine receptor CXCR4 motivated docking screens for new ligands against a homol. model and subsequently the crystal structure.  More than 3 million mols. were docked against the model and then against the crystal structure; 24 and 23 high-scoring compds. from the resp. screens were tested exptl.  Docking against the model yielded only one antagonist, which resembled known ligands and lacked specificity, whereas the crystal structure docking yielded four that were dissimilar to previously known scaffolds and apparently specific.  Intriguingly, several were potent and relatively small, with IC50 values as low as 306 nM, ligand efficiencies as high as 0.36, and with efficacy in cellular chemotaxis.  The potency and efficiency of these mols. has few precedents among protein-protein interface inhibitors, and supports structure-based efforts to discover leads for chemokine GPCRs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqeJKWh0aH7FLVg90H21EOLACvtfcHk0lghMTWGIdEqQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XlslOjtrg%253D&md5=762616e6eb0eebc0d983964d71499e75</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1120431109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1120431109%26sid%3Dliteratum%253Aachs%26aulast%3DMysinger%26aufirst%3DM.%2BM.%26aulast%3DWeiss%26aufirst%3DD.%2BR.%26aulast%3DZiarek%26aufirst%3DJ.%2BJ.%26aulast%3DGravel%26aufirst%3DS.%26aulast%3DDoak%26aufirst%3DA.%2BK.%26aulast%3DKarpiak%26aufirst%3DJ.%26aulast%3DHeveker%26aufirst%3DN.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DVolkman%26aufirst%3DB.%2BF.%26atitle%3DStructure-based%2520ligand%2520discovery%2520for%2520the%2520protein-protein%2520interface%2520of%2520chemokine%2520receptor%2520CXCR4%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2012%26volume%3D109%26spage%3D5517%26epage%3D5522%26doi%3D10.1073%2Fpnas.1120431109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group">Peng, Z. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gillespie, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weisel, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">So, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">So, W. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kondru, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Narayanan, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hermann, J. C.</span><span> </span><span class="NLM_article-title">A Crowd-Based Process and Tool for HTS Hit Triage</span> <span class="citation_source-journal">Mol. Inf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x">, </span> <span class="NLM_fpage">337</span><span class="NLM_x">–</span> <span class="NLM_lpage">345</span><span class="refDoi"> DOI: 10.1002/minf.201200154</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1002%2Fminf.201200154" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=27481590" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BC3sXls1SjtL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2013&pages=337-345&author=Z.+W.+Pengauthor=P.+Gillespieauthor=M.+Weiselauthor=S.+S.+Soauthor=W.+V.+Soauthor=R.+Kondruauthor=A.+Narayananauthor=J.+C.+Hermann&title=A+Crowd-Based+Process+and+Tool+for+HTS+Hit+Triage&doi=10.1002%2Fminf.201200154"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">A Crowd-Based Process and Tool for HTS Hit Triage</span></div><div class="casAuthors">Peng, Zhengwei; Gillespie, Paul; Weisel, Martin; So, Sung-Sau; So, W. Venus; Kondru, Rama; Narayanan, Arjun; Hermann, Johannes C.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Informatics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">337-345</span>CODEN:
                <span class="NLM_cas:coden">MIONBS</span>;
        ISSN:<span class="NLM_cas:issn">1868-1743</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">This article presents a high-throughput screening (HTS) hit triage process based on a wisdom of crowds approach.  Participants provide votes and comments on suggested compds. of confirmed HTS hits through a web voting tool in an independent and decentralized way.  Insights gained and lessons learned through the actual application of this process and the assocd. tool on several drug discovery projects will be discussed.  Finally this web voting tool could be easily modified and adapted for any crowd-based decision making for compd. selections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5La7mNfdDJLVg90H21EOLACvtfcHk0lghMTWGIdEqQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXls1SjtL0%253D&md5=ad8166a0740aa6e118afa659e2de1f94</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1002%2Fminf.201200154&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fminf.201200154%26sid%3Dliteratum%253Aachs%26aulast%3DPeng%26aufirst%3DZ.%2BW.%26aulast%3DGillespie%26aufirst%3DP.%26aulast%3DWeisel%26aufirst%3DM.%26aulast%3DSo%26aufirst%3DS.%2BS.%26aulast%3DSo%26aufirst%3DW.%2BV.%26aulast%3DKondru%26aufirst%3DR.%26aulast%3DNarayanan%26aufirst%3DA.%26aulast%3DHermann%26aufirst%3DJ.%2BC.%26atitle%3DA%2520Crowd-Based%2520Process%2520and%2520Tool%2520for%2520HTS%2520Hit%2520Triage%26jtitle%3DMol.%2520Inf.%26date%3D2013%26volume%3D32%26spage%3D337%26epage%3D345%26doi%3D10.1002%2Fminf.201200154" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group">Feng, B. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeonov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jadhav, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Babaoglu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Inglese, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Austin, C. P.</span><span> </span><span class="NLM_article-title">A high-throughput screen for aggregation-based inhibition in a large compound library</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">2385</span><span class="NLM_x">–</span> <span class="NLM_lpage">2390</span><span class="refDoi"> DOI: 10.1021/jm061317y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm061317y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BD2sXksVSlsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2007&pages=2385-2390&author=B.+Y.+Fengauthor=A.+Simeonovauthor=A.+Jadhavauthor=K.+Babaogluauthor=J.+Ingleseauthor=B.+K.+Shoichetauthor=C.+P.+Austin&title=A+high-throughput+screen+for+aggregation-based+inhibition+in+a+large+compound+library&doi=10.1021%2Fjm061317y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">A High-Throughput Screen for Aggregation-Based Inhibition in a Large Compound Library</span></div><div class="casAuthors">Feng, Brian Y.; Simeonov, Anton; Jadhav, Ajit; Babaoglu, Kerim; Inglese, James; Shoichet, Brian K.; Austin, Christopher P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2385-2390</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">High-throughput screening (HTS) is the primary technique for new lead identification in drug discovery and chem. biol.  Unfortunately, it is susceptible to false-pos. hits.  One common mechanism for such false-positives is the congregation of org. mols. into colloidal aggregates, which nonspecifically inhibit enzymes.  To both evaluate the feasibility of large-scale identification of aggregate-based inhibition and quantify its prevalence among screening hits, the authors tested 70 563 mols. from the National Institutes of Health Chem. Genomics Center (NCGC) library for detergent-sensitive inhibition.  Each mol. was screened in at least seven concns., such that dose-response curves were obtained for all mols. in the library.  There were 1274 inhibitors identified in total, of which 1204 were unambiguously detergent-sensitive.  The authors identified these as aggregate-based inhibitors.  Thirty-one library mols. were independently purchased and retested in secondary low-throughput expts.; 29 of these were confirmed as either aggregators or nonaggregators, as appropriate.  Finally, with the dose-response information collected for every compd., the authors could examine the correlation between aggregate-based inhibition and steep dose-response curves.  Three key results emerge from this study: first, detergent-dependent identification of aggregate-based inhibition is feasible on the large scale.  Second, 95% of the actives obtained in this screen are aggregate-based inhibitors.  Third, aggregate-based inhibition is correlated with steep dose-response curves, although not absolutely.  The results of this screen are being released publicly via the PubChem database.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpn9eDfL7UswbVg90H21EOLACvtfcHk0ljdQUGLliqsrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXksVSlsrw%253D&md5=33399bcd4998faf830f54a6f0d1f636a</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Fjm061317y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm061317y%26sid%3Dliteratum%253Aachs%26aulast%3DFeng%26aufirst%3DB.%2BY.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DBabaoglu%26aufirst%3DK.%26aulast%3DInglese%26aufirst%3DJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DAustin%26aufirst%3DC.%2BP.%26atitle%3DA%2520high-throughput%2520screen%2520for%2520aggregation-based%2520inhibition%2520in%2520a%2520large%2520compound%2520library%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2007%26volume%3D50%26spage%3D2385%26epage%3D2390%26doi%3D10.1021%2Fjm061317y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Screening in a spirit haunted world</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">607</span><span class="NLM_x">–</span> <span class="NLM_lpage">615</span><span class="refDoi"> DOI: 10.1016/j.drudis.2006.05.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1016%2Fj.drudis.2006.05.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=16793529" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BD28XlvFGqtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2006&pages=607-615&author=B.+K.+Shoichet&title=Screening+in+a+spirit+haunted+world&doi=10.1016%2Fj.drudis.2006.05.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Screening in a spirit haunted world</span></div><div class="casAuthors">Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">13 & 14</span>),
    <span class="NLM_cas:pages">607-615</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  High-throughput screening (HTS) campaigns can be dominated by hits that ultimately turn out to be non-drug-like.  These 'nuisance' compds. often behave strangely, with steep dose-response curves, absence of clear structure-activity relationships, and high sensitivity to assay conditions.  Several mechanisms contribute to these artifacts, including chem. reactive mols., those that absorb light in assays and those that affect redox conditions.  One of the most common mechanisms behind artifactual inhibition is discussed in this review: at micromolar concns. org. mols. can aggregate to form particles in aq. buffers, and these aggregates can sequester and thereby inhibit protein targets.  Aggregation-based inhibition is baffling from a chem. perspective, but viewed biophys. such behavior is expected.  The range of mols. that behave this way, their rapid detection in a screening environment and their possible biol. implications will be considered here.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzoEkwI2izy7Vg90H21EOLACvtfcHk0ljdQUGLliqsrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XlvFGqtLk%253D&md5=08d22b6b06215ac435824b7664b3674c</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2006.05.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2006.05.014%26sid%3Dliteratum%253Aachs%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DScreening%2520in%2520a%2520spirit%2520haunted%2520world%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2006%26volume%3D11%26spage%3D607%26epage%3D615%26doi%3D10.1016%2Fj.drudis.2006.05.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group">Giannetti, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koch, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Browner, M. F.</span><span> </span><span class="NLM_article-title">Surface plasmon resonance based assay for the detection and characterization of promiscuous inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">574</span><span class="NLM_x">–</span> <span class="NLM_lpage">580</span><span class="refDoi"> DOI: 10.1021/jm700952v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm700952v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjt1entg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=574-580&author=A.+M.+Giannettiauthor=B.+D.+Kochauthor=M.+F.+Browner&title=Surface+plasmon+resonance+based+assay+for+the+detection+and+characterization+of+promiscuous+inhibitors&doi=10.1021%2Fjm700952v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Surface Plasmon Resonance Based Assay for the Detection and Characterization of Promiscuous Inhibitors</span></div><div class="casAuthors">Giannetti, Anthony M.; Koch, Bruce D.; Browner, Michelle F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">574-580</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Promiscuous binders achieve enzyme inhibition using a nonspecific aggregation-type binding mechanism to proteins.  These compds. are a source of false-pos. hits in biochem. inhibition assays and should be removed from screening hit lists because they are not good candidates to initiate medicinal chem. programs.  The authors introduce a robust approach to identify these mols. early in the lead generation process using real time surface plasmon resonance based biosensors to observe the behavior of the binding interactions between promiscuous compds. and proteins.  Furthermore, the time resoln. of the assay reveals a no. of distinct mechanisms that promiscuous compds. employ to inhibit enzyme function and indicate that the type of mechanism can vary depending on the protein target.  A classification scheme for these compds. is presented that can be used to rapidly characterize the hits from high-throughput screens and eliminate compds. with a nonspecific mechanism of inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVTh2kYyROf7Vg90H21EOLACvtfcHk0ljdQUGLliqsrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjt1entg%253D%253D&md5=b2869c5be1cd6f455503f8e812ed6f23</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Fjm700952v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm700952v%26sid%3Dliteratum%253Aachs%26aulast%3DGiannetti%26aufirst%3DA.%2BM.%26aulast%3DKoch%26aufirst%3DB.%2BD.%26aulast%3DBrowner%26aufirst%3DM.%2BF.%26atitle%3DSurface%2520plasmon%2520resonance%2520based%2520assay%2520for%2520the%2520detection%2520and%2520characterization%2520of%2520promiscuous%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D574%26epage%3D580%26doi%3D10.1021%2Fjm700952v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group">Chowdhury, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yeoh, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tian, Y. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hillringhaus, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bagg, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rose, N. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leung, I. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, X. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woon, E. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDonough, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">King, O. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clifton, I. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klose, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Claridge, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ratcliffe, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawamura, A.</span><span> </span><span class="NLM_article-title">The oncometabolite 2-hydroxyglutarate inhibits histone lysine demethylases</span> <span class="citation_source-journal">EMBO Rep.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">463</span><span class="NLM_x">–</span> <span class="NLM_lpage">469</span><span class="refDoi"> DOI: 10.1038/embor.2011.43</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1038%2Fembor.2011.43" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=21460794" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFahs74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=463-469&author=R.+Chowdhuryauthor=K.+K.+Yeohauthor=Y.+M.+Tianauthor=L.+Hillringhausauthor=E.+A.+Baggauthor=N.+R.+Roseauthor=I.+K.+Leungauthor=X.+S.+Liauthor=E.+C.+Woonauthor=M.+Yangauthor=M.+A.+McDonoughauthor=O.+N.+Kingauthor=I.+J.+Cliftonauthor=R.+J.+Kloseauthor=T.+D.+Claridgeauthor=P.+J.+Ratcliffeauthor=C.+J.+Schofieldauthor=A.+Kawamura&title=The+oncometabolite+2-hydroxyglutarate+inhibits+histone+lysine+demethylases&doi=10.1038%2Fembor.2011.43"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">The oncometabolite 2-hydroxyglutarate inhibits histone lysine demethylases</span></div><div class="casAuthors">Chowdhury, Rasheduzzaman; Yeoh, Kar-Kheng; Tian, Ya-Min; Hillringhaus, Lars; Bagg, Eleanor A.; Rose, Nathan R.; Leung, Ivanhoe K.-H.; Li, Xuan-S.; Woon, Esther C.-Y.; Yang, Ming; McDonough, Michael A.; King, Oliver N.; Clifton, Ian J.; Klose, Robert J.; Claridge, Timothy D. W.; Ratcliffe, Peter J.; Schofield, Christopher J.; Kawamura, Akane</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Reports</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">463-469</span>CODEN:
                <span class="NLM_cas:coden">ERMEAX</span>;
        ISSN:<span class="NLM_cas:issn">1469-221X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Mutations in isocitrate dehydrogenases (IDHs) have a gain-of-function effect leading to R(-)-2-hydroxyglutarate (R-2HG) accumulation.  By using biochem., structural and cellular assays, we show that either or both R- and S-2HG inhibit 2-oxoglutarate (2OG)-dependent oxygenases with varying potencies.  Half-maximal inhibitory concn. (IC50) values for the R-form of 2HG varied from approx. 25 μM for the histone N.vepsiln.-lysine demethylase JMJD2A to more than 5 mM for the hypoxia-inducible factor (HIF) prolyl hydroxylase.  The results indicate that candidate oncogenic pathways in IDH-assocd. malignancy should include those that are regulated by other 2OG oxygenases than HIF hydroxylases, in particular those involving the regulation of histone methylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGHxuB6sKkTbVg90H21EOLACvtfcHk0lgu-GV83BxBXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFahs74%253D&md5=1b862b1337346af6b796093cea34056f</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1038%2Fembor.2011.43&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fembor.2011.43%26sid%3Dliteratum%253Aachs%26aulast%3DChowdhury%26aufirst%3DR.%26aulast%3DYeoh%26aufirst%3DK.%2BK.%26aulast%3DTian%26aufirst%3DY.%2BM.%26aulast%3DHillringhaus%26aufirst%3DL.%26aulast%3DBagg%26aufirst%3DE.%2BA.%26aulast%3DRose%26aufirst%3DN.%2BR.%26aulast%3DLeung%26aufirst%3DI.%2BK.%26aulast%3DLi%26aufirst%3DX.%2BS.%26aulast%3DWoon%26aufirst%3DE.%2BC.%26aulast%3DYang%26aufirst%3DM.%26aulast%3DMcDonough%26aufirst%3DM.%2BA.%26aulast%3DKing%26aufirst%3DO.%2BN.%26aulast%3DClifton%26aufirst%3DI.%2BJ.%26aulast%3DKlose%26aufirst%3DR.%2BJ.%26aulast%3DClaridge%26aufirst%3DT.%2BD.%26aulast%3DRatcliffe%26aufirst%3DP.%2BJ.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26aulast%3DKawamura%26aufirst%3DA.%26atitle%3DThe%2520oncometabolite%25202-hydroxyglutarate%2520inhibits%2520histone%2520lysine%2520demethylases%26jtitle%3DEMBO%2520Rep.%26date%3D2011%26volume%3D12%26spage%3D463%26epage%3D469%26doi%3D10.1038%2Fembor.2011.43" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group">Rose, N. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woon, E. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kingham, G. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">King, O. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mecinovic, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clifton, I. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ng, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Talib-Hardy, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oppermann, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDonough, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span> </span><span class="NLM_article-title">Selective inhibitors of the JMJD2 histone demethylases: combined nondenaturing mass spectrometric screening and crystallographic approaches</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">1810</span><span class="NLM_x">–</span> <span class="NLM_lpage">1818</span><span class="refDoi"> DOI: 10.1021/jm901680b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901680b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BC3cXnvFWnsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=1810-1818&author=N.+R.+Roseauthor=E.+C.+Woonauthor=G.+L.+Kinghamauthor=O.+N.+Kingauthor=J.+Mecinovicauthor=I.+J.+Cliftonauthor=S.+S.+Ngauthor=J.+Talib-Hardyauthor=U.+Oppermannauthor=M.+A.+McDonoughauthor=C.+J.+Schofield&title=Selective+inhibitors+of+the+JMJD2+histone+demethylases%3A+combined+nondenaturing+mass+spectrometric+screening+and+crystallographic+approaches&doi=10.1021%2Fjm901680b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Inhibitors of the JMJD2 Histone Demethylases: Combined Nondenaturing Mass Spectrometric Screening and Crystallographic Approaches</span></div><div class="casAuthors">Rose, Nathan R.; Woon, Esther C. Y.; Kingham, Guy L.; King, Oliver N. F.; Mecinovic, Jasmin; Clifton, Ian J.; Ng, Stanley S.; Talib-Hardy, Jobina; Oppermann, Udo; McDonough, Michael A.; Schofield, Christopher J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1810-1818</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Ferrous ion and 2-oxoglutarate (2OG) oxygenases catalyze the demethylation of Nε-methylated lysine residues in histones.  Here, studies on the inhibition of the JMJD2 subfamily of histone demethylases are reported, employing binding analyses by non-denaturing mass spectrometry (MS), dynamic combinatorial chem. coupled to MS, turnover assays, and crystallog.  The results of initial binding and inhibition assays directed the prodn. and anal. of a set of N-oxalyl-D-tyrosine derivs. to explore the extent of a sub-pocket at the JMJD2 active site.  Some of the inhibitors were shown to be selective for JMJD2 over the hypoxia-inducible factor prolyl hydroxylase PHD2.  A crystal structure of JMJD2A in complex with one of the potent inhibitors was obtained; modeling other inhibitors based on this structure predicts interactions that enable improved inhibition for some compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoa6dzV_ed-qrVg90H21EOLACvtfcHk0lgu-GV83BxBXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXnvFWnsQ%253D%253D&md5=3eb3c36d91f46193946f64288ea9ccae</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Fjm901680b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901680b%26sid%3Dliteratum%253Aachs%26aulast%3DRose%26aufirst%3DN.%2BR.%26aulast%3DWoon%26aufirst%3DE.%2BC.%26aulast%3DKingham%26aufirst%3DG.%2BL.%26aulast%3DKing%26aufirst%3DO.%2BN.%26aulast%3DMecinovic%26aufirst%3DJ.%26aulast%3DClifton%26aufirst%3DI.%2BJ.%26aulast%3DNg%26aufirst%3DS.%2BS.%26aulast%3DTalib-Hardy%26aufirst%3DJ.%26aulast%3DOppermann%26aufirst%3DU.%26aulast%3DMcDonough%26aufirst%3DM.%2BA.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26atitle%3DSelective%2520inhibitors%2520of%2520the%2520JMJD2%2520histone%2520demethylases%253A%2520combined%2520nondenaturing%2520mass%2520spectrometric%2520screening%2520and%2520crystallographic%2520approaches%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D1810%26epage%3D1818%26doi%3D10.1021%2Fjm901680b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group">Chang, K. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">King, O. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tumber, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woon, E. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heightman, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDonough, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rose, N. R.</span><span> </span><span class="NLM_article-title">Inhibition of histone demethylases by 4-carboxy-2,2′-bipyridyl compounds</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">759</span><span class="NLM_x">–</span> <span class="NLM_lpage">764</span><span class="refDoi"> DOI: 10.1002/cmdc.201100026</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1002%2Fcmdc.201100026" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=759-764&author=K.+H.+Changauthor=O.+N.+Kingauthor=A.+Tumberauthor=E.+C.+Woonauthor=T.+D.+Heightmanauthor=M.+A.+McDonoughauthor=C.+J.+Schofieldauthor=N.+R.+Rose&title=Inhibition+of+histone+demethylases+by+4-carboxy-2%2C2%E2%80%B2-bipyridyl+compounds&doi=10.1002%2Fcmdc.201100026"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201100026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201100026%26sid%3Dliteratum%253Aachs%26aulast%3DChang%26aufirst%3DK.%2BH.%26aulast%3DKing%26aufirst%3DO.%2BN.%26aulast%3DTumber%26aufirst%3DA.%26aulast%3DWoon%26aufirst%3DE.%2BC.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DMcDonough%26aufirst%3DM.%2BA.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26aulast%3DRose%26aufirst%3DN.%2BR.%26atitle%3DInhibition%2520of%2520histone%2520demethylases%2520by%25204-carboxy-2%252C2%25E2%2580%25B2-bipyridyl%2520compounds%26jtitle%3DChemMedChem%26date%3D2011%26volume%3D6%26spage%3D759%26epage%3D764%26doi%3D10.1002%2Fcmdc.201100026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group">Ng, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kavanagh, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDonough, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Butler, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pilka, E. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lienard, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bray, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Savitsky, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gileadi, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">von Delft, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rose, N. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Offer, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scheinost, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borowski, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sundstrom, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oppermann, U.</span><span> </span><span class="NLM_article-title">Crystal structures of histone demethylase JMJD2A reveal basis for substrate specificity</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">448</span><span class="NLM_x">, </span> <span class="NLM_fpage">87</span><span class="NLM_x">–</span> <span class="NLM_lpage">91</span><span class="refDoi"> DOI: 10.1038/nature05971</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1038%2Fnature05971" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=17589501" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnt1ent7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=448&publication_year=2007&pages=87-91&author=S.+S.+Ngauthor=K.+L.+Kavanaghauthor=M.+A.+McDonoughauthor=D.+Butlerauthor=E.+S.+Pilkaauthor=B.+M.+Lienardauthor=J.+E.+Brayauthor=P.+Savitskyauthor=O.+Gileadiauthor=F.+von+Delftauthor=N.+R.+Roseauthor=J.+Offerauthor=J.+C.+Scheinostauthor=T.+Borowskiauthor=M.+Sundstromauthor=C.+J.+Schofieldauthor=U.+Oppermann&title=Crystal+structures+of+histone+demethylase+JMJD2A+reveal+basis+for+substrate+specificity&doi=10.1038%2Fnature05971"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structures of histone demethylase JMJD2A reveal basis for substrate specificity</span></div><div class="casAuthors">Ng, Stanley S.; Kavanagh, Kathryn L.; McDonough, Michael A.; Butler, Danica; Pilka, Ewa S.; Lienard, Benoit M. R.; Bray, James E.; Savitsky, Pavel; Gileadi, Opher; von Delft, Frank; Rose, Nathan R.; Offer, John; Scheinost, Johanna C.; Borowski, Tomasz; Sundstrom, Michael; Schofield, Christopher J.; Oppermann, Udo</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">448</span>
        (<span class="NLM_cas:issue">7149</span>),
    <span class="NLM_cas:pages">87-91</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Post-translational histone modification has a fundamental role in chromatin biol. and is proposed to constitute a 'histone code' in epigenetic regulation.  Differential methylation of histone H3 and H4 lysyl residues regulates processes including heterochromatin formation, X-chromosome inactivation, genome imprinting, DNA repair and transcriptional regulation.  The discovery of lysyl demethylases using flavin (amine oxidases) or Fe(ii) and 2-oxoglutarate as cofactors (2OG oxygenases) has changed the view of methylation as a stable epigenetic marker.  However, little is known about how the demethylases are selective for particular lysyl-contg. sequences in specific methylation states, a key to understanding their functions.  Here we reveal how human JMJD2A (jumonji domain contg. 2A), which is selective towards tri- and dimethylated histone H3 lysyl residues 9 and 36 (H3K9me3/me2 and H3K36me3/me2), discriminates between methylation states and achieves sequence selectivity for H3K9.  We report structures of JMJD2A-Ni(ii)-Zn(ii) inhibitor complexes bound to tri-, di- and monomethyl forms of H3K9 and the tri-Me form of H3K36.  The structures reveal a lysyl-binding pocket in which substrates are bound in distinct bent conformations involving the Zn-binding site.  We propose a mechanism for achieving methylation state selectivity involving the orientation of the substrate Me groups towards a ferryl intermediate.  The results suggest distinct recognition mechanisms in different demethylase subfamilies and provide a starting point to develop chem. tools for drug discovery and to study and dissect the complexity of reversible histone methylation and its role in chromatin biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-lZrFTDTWwbVg90H21EOLACvtfcHk0lgu-GV83BxBXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnt1ent7c%253D&md5=c6429fefef17f7ae14b3d01c11617d7f</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1038%2Fnature05971&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature05971%26sid%3Dliteratum%253Aachs%26aulast%3DNg%26aufirst%3DS.%2BS.%26aulast%3DKavanagh%26aufirst%3DK.%2BL.%26aulast%3DMcDonough%26aufirst%3DM.%2BA.%26aulast%3DButler%26aufirst%3DD.%26aulast%3DPilka%26aufirst%3DE.%2BS.%26aulast%3DLienard%26aufirst%3DB.%2BM.%26aulast%3DBray%26aufirst%3DJ.%2BE.%26aulast%3DSavitsky%26aufirst%3DP.%26aulast%3DGileadi%26aufirst%3DO.%26aulast%3Dvon%2BDelft%26aufirst%3DF.%26aulast%3DRose%26aufirst%3DN.%2BR.%26aulast%3DOffer%26aufirst%3DJ.%26aulast%3DScheinost%26aufirst%3DJ.%2BC.%26aulast%3DBorowski%26aufirst%3DT.%26aulast%3DSundstrom%26aufirst%3DM.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26aulast%3DOppermann%26aufirst%3DU.%26atitle%3DCrystal%2520structures%2520of%2520histone%2520demethylase%2520JMJD2A%2520reveal%2520basis%2520for%2520substrate%2520specificity%26jtitle%3DNature%26date%3D2007%26volume%3D448%26spage%3D87%26epage%3D91%26doi%3D10.1038%2Fnature05971" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group">Chen, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whetstine, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hong, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davrazou, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kutateladze, T. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simpson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mao, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pan, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dai, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hagman, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hansen, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, G.</span><span> </span><span class="NLM_article-title">Structural insights into histone demethylation by JMJD2 family members</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">125</span><span class="NLM_x">, </span> <span class="NLM_fpage">691</span><span class="NLM_x">–</span> <span class="NLM_lpage">702</span><span class="refDoi"> DOI: 10.1016/j.cell.2006.04.024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1016%2Fj.cell.2006.04.024" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2006&pages=691-702&author=Z.+Chenauthor=J.+Zangauthor=J.+Whetstineauthor=X.+Hongauthor=F.+Davrazouauthor=T.+G.+Kutateladzeauthor=M.+Simpsonauthor=Q.+Maoauthor=C.+H.+Panauthor=S.+Daiauthor=J.+Hagmanauthor=K.+Hansenauthor=Y.+Shiauthor=G.+Zhang&title=Structural+insights+into+histone+demethylation+by+JMJD2+family+members&doi=10.1016%2Fj.cell.2006.04.024"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2006.04.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2006.04.024%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DZang%26aufirst%3DJ.%26aulast%3DWhetstine%26aufirst%3DJ.%26aulast%3DHong%26aufirst%3DX.%26aulast%3DDavrazou%26aufirst%3DF.%26aulast%3DKutateladze%26aufirst%3DT.%2BG.%26aulast%3DSimpson%26aufirst%3DM.%26aulast%3DMao%26aufirst%3DQ.%26aulast%3DPan%26aufirst%3DC.%2BH.%26aulast%3DDai%26aufirst%3DS.%26aulast%3DHagman%26aufirst%3DJ.%26aulast%3DHansen%26aufirst%3DK.%26aulast%3DShi%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DG.%26atitle%3DStructural%2520insights%2520into%2520histone%2520demethylation%2520by%2520JMJD2%2520family%2520members%26jtitle%3DCell%26date%3D2006%26volume%3D125%26spage%3D691%26epage%3D702%26doi%3D10.1016%2Fj.cell.2006.04.024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group">Shuker, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajduk, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meadows, R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fesik, S. W.</span><span> </span><span class="NLM_article-title">Discovering high-affinity ligands for proteins: SAR by NMR</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">274</span><span class="NLM_x">, </span> <span class="NLM_fpage">1531</span><span class="NLM_x">–</span> <span class="NLM_lpage">1534</span><span class="refDoi"> DOI: 10.1126/science.274.5292.1531</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1126%2Fscience.274.5292.1531" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=8929414" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADyaK28Xnt1ansLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=274&publication_year=1996&pages=1531-1534&author=S.+B.+Shukerauthor=P.+J.+Hajdukauthor=R.+P.+Meadowsauthor=S.+W.+Fesik&title=Discovering+high-affinity+ligands+for+proteins%3A+SAR+by+NMR&doi=10.1126%2Fscience.274.5292.1531"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Discovering high-affinity ligands for proteins: SAR by NMR</span></div><div class="casAuthors">Shuker, Suzanne B.; Hajduk, Philip J.; Meadows, Robert P.; Fesik, Stephen W.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">274</span>
        (<span class="NLM_cas:issue">5292</span>),
    <span class="NLM_cas:pages">1531-1534</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A NMR-based method is described in which small org. mols. that bind to proximal subsites of a protein are identified, optimized, and linked together to produce high-affinity ligands.  The approach is called "SAR by NMR" because structure-activity relationships (SAR) are obtained from NMR.  With this technique, compds. with nanomolar affinities for the FK506 binding protein were rapidly discovered by tethering two ligands with micromolar affinities.  The method reduces the amt. of chem. synthesis and time required for the discovery of high-affinity ligands and appears particularly useful in target-directed drug research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwkuE4EVU_yrVg90H21EOLACvtfcHk0lh4TcxDBf9lUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28Xnt1ansLs%253D&md5=1b9a5a5e4f2d6ee4b1c249e3e1faf2ec</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1126%2Fscience.274.5292.1531&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.274.5292.1531%26sid%3Dliteratum%253Aachs%26aulast%3DShuker%26aufirst%3DS.%2BB.%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26aulast%3DMeadows%26aufirst%3DR.%2BP.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26atitle%3DDiscovering%2520high-affinity%2520ligands%2520for%2520proteins%253A%2520SAR%2520by%2520NMR%26jtitle%3DScience%26date%3D1996%26volume%3D274%26spage%3D1531%26epage%3D1534%26doi%3D10.1126%2Fscience.274.5292.1531" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group">Friberg, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vigil, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Daniels, R. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burke, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garcia-Barrantes, P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Camper, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chauder, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olejniczak, E. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fesik, S. W.</span><span> </span><span class="NLM_article-title">Discovery of potent myeloid cell leukemia 1 (Mcl-1) inhibitors using fragment-based methods and structure-based design</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">15</span><span class="NLM_x">–</span> <span class="NLM_lpage">30</span><span class="refDoi"> DOI: 10.1021/jm301448p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm301448p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVCjur3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=15-30&author=A.+Fribergauthor=D.+Vigilauthor=B.+Zhaoauthor=R.+N.+Danielsauthor=J.+P.+Burkeauthor=P.+M.+Garcia-Barrantesauthor=D.+Camperauthor=B.+A.+Chauderauthor=T.+Leeauthor=E.+T.+Olejniczakauthor=S.+W.+Fesik&title=Discovery+of+potent+myeloid+cell+leukemia+1+%28Mcl-1%29+inhibitors+using+fragment-based+methods+and+structure-based+design&doi=10.1021%2Fjm301448p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Potent Myeloid Cell Leukemia 1 (Mcl-1) Inhibitors Using Fragment-Based Methods and Structure-Based Design</span></div><div class="casAuthors">Friberg, Anders; Vigil, Dominico; Zhao, Bin; Daniels, R. Nathan; Burke, Jason P.; Garcia-Barrantes, Pedro M.; Camper, DeMarco; Chauder, Brian A.; Lee, Taekyu; Olejniczak, Edward T.; Fesik, Stephen W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">15-30</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Myeloid cell leukemia 1 (Mcl-1), a member of the Bcl-2 family of proteins, is overexpressed and amplified in various cancers and promotes the aberrant survival of tumor cells that otherwise would undergo apoptosis.  Here we describe the discovery of potent and selective Mcl-1 inhibitors using fragment-based methods and structure-based design.  NMR-based screening of a large fragment library identified two chem. distinct hit series that bind to different sites on Mcl-1.  Members of the two fragment classes were merged together to produce lead compds. that bind to Mcl-1 with a dissocn. const. of <100 nM with selectivity for Mcl-1 over Bcl-xL and Bcl-2.  Structures of merged compds. when complexed to Mcl-1 were obtained by x-ray crystallog. and provide detailed information about the mol. recognition of small-mol. ligands binding Mcl-1.  The compds. represent starting points for the discovery of clin. useful Mcl-1 inhibitors for the treatment of a wide variety of cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-bJGKKNPbErVg90H21EOLACvtfcHk0lh4TcxDBf9lUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVCjur3J&md5=74e72c3d56cdff85d263bd974319b2db</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Fjm301448p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm301448p%26sid%3Dliteratum%253Aachs%26aulast%3DFriberg%26aufirst%3DA.%26aulast%3DVigil%26aufirst%3DD.%26aulast%3DZhao%26aufirst%3DB.%26aulast%3DDaniels%26aufirst%3DR.%2BN.%26aulast%3DBurke%26aufirst%3DJ.%2BP.%26aulast%3DGarcia-Barrantes%26aufirst%3DP.%2BM.%26aulast%3DCamper%26aufirst%3DD.%26aulast%3DChauder%26aufirst%3DB.%2BA.%26aulast%3DLee%26aufirst%3DT.%26aulast%3DOlejniczak%26aufirst%3DE.%2BT.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26atitle%3DDiscovery%2520of%2520potent%2520myeloid%2520cell%2520leukemia%25201%2520%2528Mcl-1%2529%2520inhibitors%2520using%2520fragment-based%2520methods%2520and%2520structure-based%2520design%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D15%26epage%3D30%26doi%3D10.1021%2Fjm301448p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group">Barker, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barker, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Courtney, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gardiner, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hesterkamp, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ichihara, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mather, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Montalbetti, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muller, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varasi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whittaker, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yarnold, C. J.</span><span> </span><span class="NLM_article-title">Discovery of a novel Hsp90 inhibitor by fragment linking</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">1697</span><span class="NLM_x">–</span> <span class="NLM_lpage">1700</span><span class="refDoi"> DOI: 10.1002/cmdc.201000219</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1002%2Fcmdc.201000219" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=20665758" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlCkurzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2010&pages=1697-1700&author=J.+J.+Barkerauthor=O.+Barkerauthor=S.+M.+Courtneyauthor=M.+Gardinerauthor=T.+Hesterkampauthor=O.+Ichiharaauthor=O.+Matherauthor=C.+A.+Montalbettiauthor=A.+Mullerauthor=M.+Varasiauthor=M.+Whittakerauthor=C.+J.+Yarnold&title=Discovery+of+a+novel+Hsp90+inhibitor+by+fragment+linking&doi=10.1002%2Fcmdc.201000219"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a novel Hsp90 inhibitor by fragment linking</span></div><div class="casAuthors">Barker, John J.; Barker, Oliver; Courtney, Stephen M.; Gardiner, Mihaly; Hesterkamp, Thomas; Ichihara, Osamu; Mather, Owen; Montalbetti, Christian A. G. N.; Muller, Annett; Varasi, Mario; Whittaker, Mark; Yarnold, Christopher J.</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1697-1700</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Following a high-throughput biochem. fragment screen, we have identified novel fragment inhibitors of Hsp90.  Two fragment hits were combined to give a dual-fragment Hsp90 complex.  The compd., I, with the lowest strain energy was synthesized, and a 1000-fold improvement in activity was achieved.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq05FWLIYmt6bVg90H21EOLACvtfcHk0lh4TcxDBf9lUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlCkurzI&md5=5ae93af1444d6819d8cc4bf2854bff4f</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201000219&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201000219%26sid%3Dliteratum%253Aachs%26aulast%3DBarker%26aufirst%3DJ.%2BJ.%26aulast%3DBarker%26aufirst%3DO.%26aulast%3DCourtney%26aufirst%3DS.%2BM.%26aulast%3DGardiner%26aufirst%3DM.%26aulast%3DHesterkamp%26aufirst%3DT.%26aulast%3DIchihara%26aufirst%3DO.%26aulast%3DMather%26aufirst%3DO.%26aulast%3DMontalbetti%26aufirst%3DC.%2BA.%26aulast%3DMuller%26aufirst%3DA.%26aulast%3DVarasi%26aufirst%3DM.%26aulast%3DWhittaker%26aufirst%3DM.%26aulast%3DYarnold%26aufirst%3DC.%2BJ.%26atitle%3DDiscovery%2520of%2520a%2520novel%2520Hsp90%2520inhibitor%2520by%2520fragment%2520linking%26jtitle%3DChemMedChem%26date%3D2010%26volume%3D5%26spage%3D1697%26epage%3D1700%26doi%3D10.1002%2Fcmdc.201000219" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group">Villemagne, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flipo, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blondiaux, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crauste, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Malaquin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leroux, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Piveteau, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Villeret, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brodin, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Villoutreix, B. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sperandio, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soror, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wohlkonig, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wintjens, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deprez, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baulard, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willand, N.</span><span> </span><span class="NLM_article-title">Ligand Efficiency Driven Design of New Inhibitors of Mycobacterium tuberculosis Transcriptional Repressor EthR Using Fragment Growing, Merging, and Linking Approaches</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">4876</span><span class="NLM_x">–</span> <span class="NLM_lpage">4888</span><span class="refDoi"> DOI: 10.1021/jm500422b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm500422b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BC2cXnvVShtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=4876-4888&author=B.+Villemagneauthor=M.+Flipoauthor=N.+Blondiauxauthor=C.+Crausteauthor=S.+Malaquinauthor=F.+Lerouxauthor=C.+Piveteauauthor=V.+Villeretauthor=P.+Brodinauthor=B.+O.+Villoutreixauthor=O.+Sperandioauthor=S.+H.+Sororauthor=A.+Wohlkonigauthor=R.+Wintjensauthor=B.+Deprezauthor=A.+R.+Baulardauthor=N.+Willand&title=Ligand+Efficiency+Driven+Design+of+New+Inhibitors+of+Mycobacterium+tuberculosis+Transcriptional+Repressor+EthR+Using+Fragment+Growing%2C+Merging%2C+and+Linking+Approaches&doi=10.1021%2Fjm500422b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand Efficiency Driven Design of New Inhibitors of Mycobacterium tuberculosis Transcriptional Repressor EthR Using Fragment Growing, Merging, and Linking Approaches</span></div><div class="casAuthors">Villemagne, Baptiste; Flipo, Marion; Blondiaux, Nicolas; Crauste, Celine; Malaquin, Sandra; Leroux, Florence; Piveteau, Catherine; Villeret, Vincent; Brodin, Priscille; Villoutreix, Bruno O.; Sperandio, Olivier; Soror, Sameh H.; Wohlkonig, Alexandre; Wintjens, Rene; Deprez, Benoit; Baulard, Alain R.; Willand, Nicolas</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4876-4888</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Tuberculosis remains a major cause of mortality and morbidity, killing each year more than one million people.  Although the combined use of first line antibiotics (isoniazid, rifampicin, pyrazinamide, and ethambutol) is efficient to treat most patients, the rapid emergence of multidrug resistant strains of Mycobacterium tuberculosis stresses the need for alternative therapies.  Mycobacterial transcriptional repressor EthR is a key player in the control of second-line drugs bioactivation such as ethionamide and has been shown to impair the sensitivity of the human pathogen Mycobacterium tuberculosis to this antibiotic.  As a way to identify new potent ligands of this protein, we have developed fragment-based approaches.  In the current study, we combined surface plasmon resonance assay, x-ray crystallog., and ligand efficiency driven design for the rapid discovery and optimization of new chemotypes of EthR ligands starting from a fragment.  The design, synthesis, and in vitro and ex vivo activities of these compds. will be discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp77Ygs3mf7prVg90H21EOLACvtfcHk0lj-UFYgQC7UZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXnvVShtrY%253D&md5=36763fe306287529745b7c5a48272d6a</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Fjm500422b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm500422b%26sid%3Dliteratum%253Aachs%26aulast%3DVillemagne%26aufirst%3DB.%26aulast%3DFlipo%26aufirst%3DM.%26aulast%3DBlondiaux%26aufirst%3DN.%26aulast%3DCrauste%26aufirst%3DC.%26aulast%3DMalaquin%26aufirst%3DS.%26aulast%3DLeroux%26aufirst%3DF.%26aulast%3DPiveteau%26aufirst%3DC.%26aulast%3DVilleret%26aufirst%3DV.%26aulast%3DBrodin%26aufirst%3DP.%26aulast%3DVilloutreix%26aufirst%3DB.%2BO.%26aulast%3DSperandio%26aufirst%3DO.%26aulast%3DSoror%26aufirst%3DS.%2BH.%26aulast%3DWohlkonig%26aufirst%3DA.%26aulast%3DWintjens%26aufirst%3DR.%26aulast%3DDeprez%26aufirst%3DB.%26aulast%3DBaulard%26aufirst%3DA.%2BR.%26aulast%3DWilland%26aufirst%3DN.%26atitle%3DLigand%2520Efficiency%2520Driven%2520Design%2520of%2520New%2520Inhibitors%2520of%2520Mycobacterium%2520tuberculosis%2520Transcriptional%2520Repressor%2520EthR%2520Using%2520Fragment%2520Growing%252C%2520Merging%252C%2520and%2520Linking%2520Approaches%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D4876%26epage%3D4888%26doi%3D10.1021%2Fjm500422b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group">Erlanson, D. A.</span><span> </span><span class="NLM_article-title">Introduction to fragment-based drug discovery</span> <span class="citation_source-journal">Top. Curr. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">317</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">32</span><span class="refDoi"> DOI: 10.1007/128_2011_180</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1007%2F128_2011_180" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=317&publication_year=2011&pages=1-32&author=D.+A.+Erlanson&title=Introduction+to+fragment-based+drug+discovery&doi=10.1007%2F128_2011_180"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1007%2F128_2011_180&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F128_2011_180%26sid%3Dliteratum%253Aachs%26aulast%3DErlanson%26aufirst%3DD.%2BA.%26atitle%3DIntroduction%2520to%2520fragment-based%2520drug%2520discovery%26jtitle%3DTop.%2520Curr.%2520Chem.%26date%3D2011%26volume%3D317%26spage%3D1%26epage%3D32%26doi%3D10.1007%2F128_2011_180" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group">Teotico, D. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Babaoglu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rocklin, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferreira, R. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giannetti, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Docking for fragment inhibitors of AmpC beta-lactamase</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">106</span><span class="NLM_x">, </span> <span class="NLM_fpage">7455</span><span class="NLM_x">–</span> <span class="NLM_lpage">7460</span><span class="refDoi"> DOI: 10.1073/pnas.0813029106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1073%2Fpnas.0813029106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=19416920" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BD1MXmt1Khtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2009&pages=7455-7460&author=D.+G.+Teoticoauthor=K.+Babaogluauthor=G.+J.+Rocklinauthor=R.+S.+Ferreiraauthor=A.+M.+Giannettiauthor=B.+K.+Shoichet&title=Docking+for+fragment+inhibitors+of+AmpC+beta-lactamase&doi=10.1073%2Fpnas.0813029106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Docking for fragment inhibitors of AmpC 83-lactamase</span></div><div class="casAuthors">Teotico, Denise G.; Babaoglu, Kerim; Rocklin, Gabriel J.; Ferreira, Rafaela S.; Giannetti, Anthony M.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">7455-7460</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Fragment screens for new ligands have had wide success, not withstanding their constraint to libraries of 1000-10,000 mols.  Larger libraries would be addressable were mol. docking reliable for fragment screens, but this has not been widely accepted.  To investigate docking's ability to prioritize fragments, a library of >137,000 such mols. were docked against the structure of β-lactamase.  Forty-eight fragments highly ranked by docking were acquired and tested; 23 had Ki values ranging from 0.7 to 9.2 mM.  X-ray crystal structures of the enzyme-bound complexes were detd. for 8 of the fragments.  For 4, the correspondence between the predicted and exptl. structures was high (RMSD between 1.2 and 1.4 Å), whereas for another 2, the fidelity was lower but retained most key interactions (RMSD 2.4-2.6 Å).  Two of the 8 fragments adopted very different poses in the active site owing to enzyme conformational changes.  The 48% hit rate of the fragment docking compares very favorably with "lead-like" docking and high-throughput screening against the same enzyme.  To understand this, we investigated the occurrence of the fragment scaffolds among larger, lead-like mols.  Approx. 1% of com. available fragments contain these inhibitors whereas only 10-7% of lead-like mols. do.  This suggests that many more chemotypes and combinations of chemotypes are present among fragments than are available among lead-like mols., contributing to the higher hit rates.  The ability of docking to prioritize these fragments suggests that the technique can be used to exploit the better chemotype coverage that exists at the fragment level.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprpafzXv4ydrVg90H21EOLACvtfcHk0lj-UFYgQC7UZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXmt1Khtrg%253D&md5=d2df01432d5381c226e8aa7e008457b5</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0813029106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0813029106%26sid%3Dliteratum%253Aachs%26aulast%3DTeotico%26aufirst%3DD.%2BG.%26aulast%3DBabaoglu%26aufirst%3DK.%26aulast%3DRocklin%26aufirst%3DG.%2BJ.%26aulast%3DFerreira%26aufirst%3DR.%2BS.%26aulast%3DGiannetti%26aufirst%3DA.%2BM.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DDocking%2520for%2520fragment%2520inhibitors%2520of%2520AmpC%2520beta-lactamase%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2009%26volume%3D106%26spage%3D7455%26epage%3D7460%26doi%3D10.1073%2Fpnas.0813029106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group">Chen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Molecular docking and ligand specificity in fragment-based inhibitor discovery</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">358</span><span class="NLM_x">–</span> <span class="NLM_lpage">364</span><span class="refDoi"> DOI: 10.1038/nchembio.155</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1038%2Fnchembio.155" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=19305397" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjsVCqt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=358-364&author=Y.+Chenauthor=B.+K.+Shoichet&title=Molecular+docking+and+ligand+specificity+in+fragment-based+inhibitor+discovery&doi=10.1038%2Fnchembio.155"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular docking and ligand specificity in fragment-based inhibitor discovery</span></div><div class="casAuthors">Chen, Yu; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">358-364</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Fragment screens have successfully identified new scaffolds in drug discovery, often with relatively high hit rates (5%) using small screening libraries (1,000-10,000 compds.).  This raises two questions: would other noteworthy chemotypes be found were one to screen all com. available fragments (>300,000), and does the success rate imply low specificity of fragments.  We used mol. docking to screen large libraries of fragments against CTX-M β-lactamase.  We identified ten millimolar-range inhibitors from the 69 compds. tested.  The docking poses corresponded closely to the crystallog. structures subsequently detd.  Notably, these initial low-affinity hits showed little specificity between CTX-M and an unrelated β-lactamase, AmpC, which is unusual among β-lactamase inhibitors.  This is consistent with the idea that the high hit rates among fragments correlate to a low initial specificity.  As the inhibitors were progressed, both specificity and affinity rose together, yielding to our knowledge the first micromolar-range noncovalent inhibitors against a class A β-lactamase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqO8zTkq9kMAbVg90H21EOLACvtfcHk0li7rBYWJFAw_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjsVCqt7k%253D&md5=6ee8dd19d0b99599ea74eaf735483b1d</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.155&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.155%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DY.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DMolecular%2520docking%2520and%2520ligand%2520specificity%2520in%2520fragment-based%2520inhibitor%2520discovery%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2009%26volume%3D5%26spage%3D358%26epage%3D364%26doi%3D10.1038%2Fnchembio.155" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group">Tiefenbrunn, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Forli, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baksh, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, M. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Happer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, Y. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perryman, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rhee, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Torbett, B. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olson, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elder, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Finn, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stout, C. D.</span><span> </span><span class="NLM_article-title">Small Molecule Regulation of Protein Conformation by Binding in the Flap of HIV Protease</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">1223</span><span class="NLM_x">–</span> <span class="NLM_lpage">1231</span><span class="refDoi"> DOI: 10.1021/cb300611p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb300611p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=1223-1231&author=T.+Tiefenbrunnauthor=S.+Forliauthor=M.+M.+Bakshauthor=M.+W.+Changauthor=M.+Happerauthor=Y.+C.+Linauthor=A.+L.+Perrymanauthor=J.+K.+Rheeauthor=B.+E.+Torbettauthor=A.+J.+Olsonauthor=J.+H.+Elderauthor=M.+G.+Finnauthor=C.+D.+Stout&title=Small+Molecule+Regulation+of+Protein+Conformation+by+Binding+in+the+Flap+of+HIV+Protease&doi=10.1021%2Fcb300611p"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1021%2Fcb300611p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb300611p%26sid%3Dliteratum%253Aachs%26aulast%3DTiefenbrunn%26aufirst%3DT.%26aulast%3DForli%26aufirst%3DS.%26aulast%3DBaksh%26aufirst%3DM.%2BM.%26aulast%3DChang%26aufirst%3DM.%2BW.%26aulast%3DHapper%26aufirst%3DM.%26aulast%3DLin%26aufirst%3DY.%2BC.%26aulast%3DPerryman%26aufirst%3DA.%2BL.%26aulast%3DRhee%26aufirst%3DJ.%2BK.%26aulast%3DTorbett%26aufirst%3DB.%2BE.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26aulast%3DElder%26aufirst%3DJ.%2BH.%26aulast%3DFinn%26aufirst%3DM.%2BG.%26aulast%3DStout%26aufirst%3DC.%2BD.%26atitle%3DSmall%2520Molecule%2520Regulation%2520of%2520Protein%2520Conformation%2520by%2520Binding%2520in%2520the%2520Flap%2520of%2520HIV%2520Protease%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2013%26volume%3D8%26spage%3D1223%26epage%3D1231%26doi%3D10.1021%2Fcb300611p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group">Nichols, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Renslo, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y.</span><span> </span><span class="NLM_article-title">Fragment-based inhibitor discovery against beta-lactamase</span> <span class="citation_source-journal">Future Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">413</span><span class="NLM_x">–</span> <span class="NLM_lpage">427</span><span class="refDoi"> DOI: 10.4155/fmc.14.10</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.4155%2Ffmc.14.10" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=24635522" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BC2cXksV2qtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=413-427&author=D.+A.+Nicholsauthor=A.+R.+Rensloauthor=Y.+Chen&title=Fragment-based+inhibitor+discovery+against+beta-lactamase&doi=10.4155%2Ffmc.14.10"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-based inhibitor discovery against β-lactamase</span></div><div class="casAuthors">Nichols, Derek A.; Renslo, Adam R.; Chen, Yu</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">413-427</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  The prodn. of β-lactamase is one of the primary resistance mechanisms used by Gram-neg. bacterial pathogens to counter β-lactam antibiotics, such as penicillins, cephalosporins and carbapenems.  There is an urgent need to develop novel β-lactamase inhibitors in response to ever evolving β-lactamases possessing an expanded spectrum of β-lactam hydrolyzing activity.  Whereas traditional high-throughput screening proved ineffective against serine β-lactamases, fragment-based approaches have been successfully employed to identify novel chem. matter, which in turn revealed much about the specific mol. interactions possible in the active site of serine and metallo β-lactamases.  In this review, the authors summarize recent progress in the field, particularly: the identification of novel inhibitor chemotypes through fragment-based screening; the use of fragment-protein structures to understand key features of binding hot spots and inform the design of improved leads; lessons learned and new prospects for β-lactamase inhibitor development using fragment-based approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9O0Za50-8K7Vg90H21EOLACvtfcHk0li7rBYWJFAw_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXksV2qtb0%253D&md5=6e8278707450cc1fe4c33f71efe41f8c</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.4155%2Ffmc.14.10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.14.10%26sid%3Dliteratum%253Aachs%26aulast%3DNichols%26aufirst%3DD.%2BA.%26aulast%3DRenslo%26aufirst%3DA.%2BR.%26aulast%3DChen%26aufirst%3DY.%26atitle%3DFragment-based%2520inhibitor%2520discovery%2520against%2520beta-lactamase%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2014%26volume%3D6%26spage%3D413%26epage%3D427%26doi%3D10.4155%2Ffmc.14.10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group">Barelier, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eidam, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fish, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hollander, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Figaroa, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nachane, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siegal, G.</span><span> </span><span class="NLM_article-title">Increasing chemical space coverage by combining empirical and computational fragment screens</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">1528</span><span class="NLM_x">–</span> <span class="NLM_lpage">1535</span><span class="refDoi"> DOI: 10.1021/cb5001636</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb5001636" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BC2cXnsFSksbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=1528-1535&author=S.+Barelierauthor=O.+Eidamauthor=I.+Fishauthor=J.+Hollanderauthor=F.+Figaroaauthor=R.+Nachaneauthor=J.+J.+Irwinauthor=B.+K.+Shoichetauthor=G.+Siegal&title=Increasing+chemical+space+coverage+by+combining+empirical+and+computational+fragment+screens&doi=10.1021%2Fcb5001636"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Increasing Chemical Space Coverage by Combining Empirical and Computational Fragment Screens</span></div><div class="casAuthors">Barelier, Sarah; Eidam, Oliv; Fish, Inbar; Hollander, Johan; Figaroa, Francis; Nachane, Ruta; Irwin, John J.; Shoichet, Brian K.; Siegal, Gregg</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1528-1535</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Most libraries for fragment-based drug discovery are restricted to 1,000-10,000 compds., but over 500,000 fragments are com. available and potentially accessible by virtual screening.  Whether this larger set would increase chemotype coverage, and whether a computational screen can pragmatically prioritize them, is debated.  To investigate this question, a 1281-fragment library was screened by NMR (NMR) against AmpC β-lactamase, and hits were confirmed by surface plasmon resonance (SPR).  Nine hits with novel chemotypes were confirmed biochem. with KI values from 0.2 to low mM.  We also computationally docked 290,000 purchasable fragments with chemotypes unrepresented in the empirical library, finding 10 that had KI values from 0.03 to low mM.  Though less novel than those discovered by NMR, the docking-derived fragments filled chemotype holes from the empirical library.  Crystal structures of nine of the fragments in complex with AmpC β-lactamase revealed new binding sites and explained the relatively high affinity of the docking-derived fragments.  The existence of chemotype holes is likely a general feature of fragment libraries, as calcn. suggests that to represent the fragment substructures of even known biogenic mols. would demand a library of minimally over 32,000 fragments.  Combining computational and empirical fragment screens enables the discovery of unexpected chemotypes, here by the NMR screen, while capturing chemotypes missing from the empirical library and tailored to the target, with little extra cost in resources.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5DNyK9V50nLVg90H21EOLACvtfcHk0lhSmTvCxjog7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXnsFSksbg%253D&md5=7c75e304e70fec6d9209c21a87a06a17</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1021%2Fcb5001636&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb5001636%26sid%3Dliteratum%253Aachs%26aulast%3DBarelier%26aufirst%3DS.%26aulast%3DEidam%26aufirst%3DO.%26aulast%3DFish%26aufirst%3DI.%26aulast%3DHollander%26aufirst%3DJ.%26aulast%3DFigaroa%26aufirst%3DF.%26aulast%3DNachane%26aufirst%3DR.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DSiegal%26aufirst%3DG.%26atitle%3DIncreasing%2520chemical%2520space%2520coverage%2520by%2520combining%2520empirical%2520and%2520computational%2520fragment%2520screens%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2014%26volume%3D9%26spage%3D1528%26epage%3D1535%26doi%3D10.1021%2Fcb5001636" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group">Suzuki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyata, N.</span><span> </span><span class="NLM_article-title">Lysine demethylases inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">8236</span><span class="NLM_x">–</span> <span class="NLM_lpage">8250</span><span class="refDoi"> DOI: 10.1021/jm201048w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm201048w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1OrtrnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=8236-8250&author=T.+Suzukiauthor=N.+Miyata&title=Lysine+demethylases+inhibitors&doi=10.1021%2Fjm201048w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine Demethylases Inhibitors</span></div><div class="casAuthors">Suzuki, Takayoshi; Miyata, Naoki</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">8236-8250</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review, on the use of lysine demethylases inhibitors in therapy of various diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplp9MjSdvnn7Vg90H21EOLACvtfcHk0lhSmTvCxjog7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1OrtrnF&md5=194d4bbc2e52672f5d5f019de0bb760c</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1021%2Fjm201048w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm201048w%26sid%3Dliteratum%253Aachs%26aulast%3DSuzuki%26aufirst%3DT.%26aulast%3DMiyata%26aufirst%3DN.%26atitle%3DLysine%2520demethylases%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D8236%26epage%3D8250%26doi%3D10.1021%2Fjm201048w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rival-Gervier, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Labrie, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wigle, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dimaggio, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wasney, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siarheyeva, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tempel, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chau, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mangano, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, X. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simpson, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pattenden, S. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norris, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kireev, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tripathy, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edwards, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janzen, W. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garcia, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petronis, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ellis, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">A chemical probe selectively inhibits G9a and GLP methyltransferase activity in cells</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">566</span><span class="NLM_x">–</span> <span class="NLM_lpage">574</span><span class="refDoi"> DOI: 10.1038/nchembio.599</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1038%2Fnchembio.599" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=21743462" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BC3MXos1OmtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=566-574&author=M.+Vedadiauthor=D.+Barsyte-Lovejoyauthor=F.+Liuauthor=S.+Rival-Gervierauthor=A.+Allali-Hassaniauthor=V.+Labrieauthor=T.+J.+Wigleauthor=P.+A.+Dimaggioauthor=G.+A.+Wasneyauthor=A.+Siarheyevaauthor=A.+Dongauthor=W.+Tempelauthor=S.+C.+Wangauthor=X.+Chenauthor=I.+Chauauthor=T.+J.+Manganoauthor=X.+P.+Huangauthor=C.+D.+Simpsonauthor=S.+G.+Pattendenauthor=J.+L.+Norrisauthor=D.+B.+Kireevauthor=A.+Tripathyauthor=A.+Edwardsauthor=B.+L.+Rothauthor=W.+P.+Janzenauthor=B.+A.+Garciaauthor=A.+Petronisauthor=J.+Ellisauthor=P.+J.+Brownauthor=S.+V.+Fryeauthor=C.+H.+Arrowsmithauthor=J.+Jin&title=A+chemical+probe+selectively+inhibits+G9a+and+GLP+methyltransferase+activity+in+cells&doi=10.1038%2Fnchembio.599"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">A chemical probe selectively inhibits G9a and GLP methyltransferase activity in cells</span></div><div class="casAuthors">Vedadi, Masoud; Barsyte-Lovejoy, Dalia; Liu, Feng; Rival-Gervier, Sylvie; Allali-Hassani, Abdellah; Labrie, Viviane; Wigle, Tim J.; DiMaggio, Peter A.; Wasney, Gregory A.; Siarheyeva, Alena; Dong, Aiping; Tempel, Wolfram; Wang, Sun-Chong; Chen, Xin; Chau, Irene; Mangano, Thomas J.; Huang, Xi-ping; Simpson, Catherine D.; Pattenden, Samantha G.; Norris, Jacqueline L.; Kireev, Dmitri B.; Tripathy, Ashutosh; Edwards, Aled; Roth, Bryan L.; Janzen, William P.; Garcia, Benjamin A.; Petronis, Arturas; Ellis, James; Brown, Peter J.; Frye, Stephen V.; Arrowsmith, Cheryl H.; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">566-574</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein lysine methyltransferases G9a and GLP modulate the transcriptional repression of a variety of genes via dimethylation of Lys9 on histone H3 (H3K9me2) as well as dimethylation of non-histone targets.  Here we report the discovery of UNC0638, an inhibitor of G9a and GLP with excellent potency and selectivity over a wide range of epigenetic and non-epigenetic targets.  UNC0638 treatment of a variety of cell lines resulted in lower global H3K9me2 levels, equiv. to levels obsd. for small hairpin RNA knockdown of G9a and GLP with the functional potency of UNC0638 being well sepd. from its toxicity.  UNC0638 markedly reduced the clonogenicity of MCF7 cells, reduced the abundance of H3K9me2 marks at promoters of known G9a-regulated endogenous genes and disproportionately affected several genomic loci encoding microRNAs.  In mouse embryonic stem cells, UNC0638 reactivated G9a-silenced genes and a retroviral reporter gene in a concn.-dependent manner without promoting differentiation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphv-2bZcBqwLVg90H21EOLACvtfcHk0lhBoFuk0v5svw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXos1OmtLs%253D&md5=a944e99a910194c1a15240fc6dc8f852</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.599&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.599%26sid%3Dliteratum%253Aachs%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DRival-Gervier%26aufirst%3DS.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DLabrie%26aufirst%3DV.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DDimaggio%26aufirst%3DP.%2BA.%26aulast%3DWasney%26aufirst%3DG.%2BA.%26aulast%3DSiarheyeva%26aufirst%3DA.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DTempel%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DS.%2BC.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DMangano%26aufirst%3DT.%2BJ.%26aulast%3DHuang%26aufirst%3DX.%2BP.%26aulast%3DSimpson%26aufirst%3DC.%2BD.%26aulast%3DPattenden%26aufirst%3DS.%2BG.%26aulast%3DNorris%26aufirst%3DJ.%2BL.%26aulast%3DKireev%26aufirst%3DD.%2BB.%26aulast%3DTripathy%26aufirst%3DA.%26aulast%3DEdwards%26aufirst%3DA.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DJanzen%26aufirst%3DW.%2BP.%26aulast%3DGarcia%26aufirst%3DB.%2BA.%26aulast%3DPetronis%26aufirst%3DA.%26aulast%3DEllis%26aufirst%3DJ.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DA%2520chemical%2520probe%2520selectively%2520inhibits%2520G9a%2520and%2520GLP%2520methyltransferase%2520activity%2520in%2520cells%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2011%26volume%3D7%26spage%3D566%26epage%3D574%26doi%3D10.1038%2Fnchembio.599" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group">Hermann, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ghanem, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raushel, F. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Predicting substrates by docking high-energy intermediates to enzyme structures</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">128</span><span class="NLM_x">, </span> <span class="NLM_fpage">15882</span><span class="NLM_x">–</span> <span class="NLM_lpage">15891</span><span class="refDoi"> DOI: 10.1021/ja065860f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja065860f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1WhsLjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2006&pages=15882-15891&author=J.+C.+Hermannauthor=E.+Ghanemauthor=Y.+C.+Liauthor=F.+M.+Raushelauthor=J.+J.+Irwinauthor=B.+K.+Shoichet&title=Predicting+substrates+by+docking+high-energy+intermediates+to+enzyme+structures&doi=10.1021%2Fja065860f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Predicting Substrates by Docking High-Energy Intermediates to Enzyme Structures</span></div><div class="casAuthors">Hermann, Johannes C.; Ghanem, Eman; Li, Yingchun; Raushel, Frank M.; Irwin, John J.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">15882-15891</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">With the emergence of sequences and even structures for proteins of unknown function, structure-based prediction of enzyme activity has become a pragmatic as well as an interesting question.  Here we investigate a method to predict substrates for enzymes of known structure by docking high-energy intermediate forms of the potential substrates.  A database of such high-energy transition-state analogs was created from the KEGG metabolites.  To reduce the no. of possible reactions to consider, we restricted ourselves to enzymes of the amidohydrolase superfamily.  We docked each metabolite into seven different amidohydrolases in both the ground-state and the high-energy intermediate forms.  Docking the high-energy intermediates improved the discrimination between decoys and substrates significantly over the corresponding std. ground-state database, both by enrichment of the true substrates and by geometric fidelity.  To test this method prospectively, we attempted to predict the enantioselectivity of a set of chiral substrates for phosphotriesterase, for both wild-type and mutant forms of this enzyme.  The stereoselectivity ratios of the six enzymes considered for those four substrate enantiomer pairs differed over a range of 10- to 10,000-fold and underwent 20 switches in stereoselectivities for favored enantiomers, compared to the wild type.  The docking of the high-energy intermediates correctly predicted the stereoselectivities for 18 of the 20 substrate/enzyme combinations when compared to subsequent exptl. synthesis and testing.  The possible applications of this approach to other enzymes are considered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXBSzfDFP1ZLVg90H21EOLACvtfcHk0lhBoFuk0v5svw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1WhsLjL&md5=e34d1979bf873173f3c21c2de4275b5b</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1021%2Fja065860f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja065860f%26sid%3Dliteratum%253Aachs%26aulast%3DHermann%26aufirst%3DJ.%2BC.%26aulast%3DGhanem%26aufirst%3DE.%26aulast%3DLi%26aufirst%3DY.%2BC.%26aulast%3DRaushel%26aufirst%3DF.%2BM.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DPredicting%2520substrates%2520by%2520docking%2520high-energy%2520intermediates%2520to%2520enzyme%2520structures%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2006%26volume%3D128%26spage%3D15882%26epage%3D15891%26doi%3D10.1021%2Fja065860f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group">Hermann, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marti-Arbona, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Almo, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raushel, F. M.</span><span> </span><span class="NLM_article-title">Structure-based activity prediction for an enzyme of unknown function</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">448</span><span class="NLM_x">, </span> <span class="NLM_fpage">775</span><span class="NLM_x">–</span> <span class="NLM_lpage">779</span><span class="refDoi"> DOI: 10.1038/nature05981</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1038%2Fnature05981" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=17603473" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BD2sXptVGrs7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=448&publication_year=2007&pages=775-779&author=J.+C.+Hermannauthor=R.+Marti-Arbonaauthor=A.+A.+Fedorovauthor=E.+Fedorovauthor=S.+C.+Almoauthor=B.+K.+Shoichetauthor=F.+M.+Raushel&title=Structure-based+activity+prediction+for+an+enzyme+of+unknown+function&doi=10.1038%2Fnature05981"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based activity prediction for an enzyme of unknown function</span></div><div class="casAuthors">Hermann, Johannes C.; Marti-Arbona, Ricardo; Fedorov, Alexander A.; Fedorov, Elena; Almo, Steven C.; Shoichet, Brian K.; Raushel, Frank M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">448</span>
        (<span class="NLM_cas:issue">7155</span>),
    <span class="NLM_cas:pages">775-779</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">With many genomes sequenced, a pressing challenge in biol. is predicting the function of the proteins that the genes encode.  When proteins are unrelated to others of known activity, bioinformatics inference for function becomes problematic.  It would thus be useful to interrogate protein structures for function directly.  Here, we predict the function of an enzyme of unknown activity, Tm0936 from Thermotoga maritima, by docking high-energy intermediate forms of thousands of candidate metabolites.  The docking hit list was dominated by adenine analogs, which appeared to undergo C6-deamination.  Four of these, including 5-methylthioadenosine and S-adenosylhomocysteine (SAH), were tested as substrates, and three had substantial catalytic rate consts. (105 M-1 s-1).  The x-ray crystal structure of the complex between Tm0936 and the product resulting from the deamination of SAH, S-inosylhomocysteine, was detd., and it corresponded closely to the predicted structure.  The deaminated products can be further metabolized by T. maritima in a previously uncharacterized SAH degrdn. pathway.  Structure-based docking with high-energy forms of potential substrates may be a useful tool to annotate enzymes for function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomC_L0C9hsf7Vg90H21EOLACvtfcHk0lhBoFuk0v5svw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXptVGrs7w%253D&md5=b0bd6612147fb848f49f8b9ccfd8384e</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1038%2Fnature05981&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature05981%26sid%3Dliteratum%253Aachs%26aulast%3DHermann%26aufirst%3DJ.%2BC.%26aulast%3DMarti-Arbona%26aufirst%3DR.%26aulast%3DFedorov%26aufirst%3DA.%2BA.%26aulast%3DFedorov%26aufirst%3DE.%26aulast%3DAlmo%26aufirst%3DS.%2BC.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DRaushel%26aufirst%3DF.%2BM.%26atitle%3DStructure-based%2520activity%2520prediction%2520for%2520an%2520enzyme%2520of%2520unknown%2520function%26jtitle%3DNature%26date%3D2007%26volume%3D448%26spage%3D775%26epage%3D779%26doi%3D10.1038%2Fnature05981" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group">Kuntz, I. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blaney, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oatley, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Langridge, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferrin, T. E.</span><span> </span><span class="NLM_article-title">A geometric approach to macromolecule-ligand interactions</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1982</span><span class="NLM_x">, </span> <span class="NLM_volume">161</span><span class="NLM_x">, </span> <span class="NLM_fpage">269</span><span class="NLM_x">–</span> <span class="NLM_lpage">88</span><span class="refDoi"> DOI: 10.1016/0022-2836(82)90153-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1016%2F0022-2836%2882%2990153-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=7154081" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADyaL38XmtFajsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=161&publication_year=1982&pages=269-88&author=I.+D.+Kuntzauthor=J.+M.+Blaneyauthor=S.+J.+Oatleyauthor=R.+Langridgeauthor=T.+E.+Ferrin&title=A+geometric+approach+to+macromolecule-ligand+interactions&doi=10.1016%2F0022-2836%2882%2990153-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">A geometric approach to macromolecule-ligand interactions</span></div><div class="casAuthors">Kuntz, Irwin D.; Blaney, Jeffrey M.; Oatley, Stuart J.; Langridge, Robert; Ferrin, Thomas E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1982</span>),
    <span class="NLM_cas:volume">161</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">269-88</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    </div><div class="casAbstract">A method is described to explore geometrically feasible alignments of ligands and receptors of known structure.  Algorithms are presented that examine many binding geometries and evaluate them in terms of steric overlap.  The procedure uses specific mol. conformations.  A method is included for finding putative binding sites on a macromol. surface.  Results are reported for heme-myoglobin interaction and the binding of thyroid hormone analogs to prealbumin.  In each case, the program finds structures within 1 Å of the x-ray results and also finds distinctly different geometries that provide good steric fits.  The approach seems well-suited for generating conformations for energy refinement programs and interactive computer graphics routines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWsp3KKxuQf7Vg90H21EOLACvtfcHk0lhBoFuk0v5svw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL38XmtFajsbw%253D&md5=8a4234b24356ea5340f33d906cd71d3e</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1016%2F0022-2836%2882%2990153-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0022-2836%252882%252990153-X%26sid%3Dliteratum%253Aachs%26aulast%3DKuntz%26aufirst%3DI.%2BD.%26aulast%3DBlaney%26aufirst%3DJ.%2BM.%26aulast%3DOatley%26aufirst%3DS.%2BJ.%26aulast%3DLangridge%26aufirst%3DR.%26aulast%3DFerrin%26aufirst%3DT.%2BE.%26atitle%3DA%2520geometric%2520approach%2520to%2520macromolecule-ligand%2520interactions%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1982%26volume%3D161%26spage%3D269%26epage%3D88%26doi%3D10.1016%2F0022-2836%2882%2990153-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit69"><span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mysinger, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Colizzi, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wassam, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, Y.</span><span> </span><span class="NLM_article-title">Automated docking screens: a feasibility study</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">5712</span><span class="NLM_x">–</span> <span class="NLM_lpage">5720</span><span class="refDoi"> DOI: 10.1021/jm9006966</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm9006966" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVOjurjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=5712-5720&author=J.+J.+Irwinauthor=B.+K.+Shoichetauthor=M.+M.+Mysingerauthor=N.+Huangauthor=F.+Colizziauthor=P.+Wassamauthor=Y.+Cao&title=Automated+docking+screens%3A+a+feasibility+study&doi=10.1021%2Fjm9006966"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Automated Docking Screens: A Feasibility Study</span></div><div class="casAuthors">Irwin, John J.; Shoichet, Brian K.; Mysinger, Michael M.; Huang, Niu; Colizzi, Francesco; Wassam, Pascal; Cao, Yiqun</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">5712-5720</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. docking is the most practical approach to leverage protein structure for ligand discovery, but the technique retains important liabilities that make it challenging to deploy on a large scale.  We have therefore created an expert system, DOCK Blaster, to investigate the feasibility of full automation.  The method requires a PDB code, sometimes with a ligand structure, and from that alone can launch a full screen of large libraries.  A crit. feature is self-assessment, which ests. the anticipated reliability of the automated screening results using pose fidelity and enrichment.  Against common benchmarks, DOCK Blaster recapitulates the crystal ligand pose within 2 Å rmsd 50-60% of the time; inferior to an expert, but respectable.  Half the time the ligand also ranked among the top 5% of 100 phys. matched decoys chosen on the fly.  Further tests were undertaken culminating in a study of 7755 eligible PDB structures.  In 1398 cases, the redocked ligand ranked in the top 5% of 100 property-matched decoys while also posing within 2 Å rmsd, suggesting that unsupervised prospective docking is viable.  DOCK Blaster is available at http://blaster.docking.org.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPZAd6CrHGp7Vg90H21EOLACvtfcHk0lg5gNhdSoZKsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVOjurjN&md5=f302db1f77b87f26d1fa2fece55bca96</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1021%2Fjm9006966&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm9006966%26sid%3Dliteratum%253Aachs%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DMysinger%26aufirst%3DM.%2BM.%26aulast%3DHuang%26aufirst%3DN.%26aulast%3DColizzi%26aufirst%3DF.%26aulast%3DWassam%26aufirst%3DP.%26aulast%3DCao%26aufirst%3DY.%26atitle%3DAutomated%2520docking%2520screens%253A%2520a%2520feasibility%2520study%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D5712%26epage%3D5720%26doi%3D10.1021%2Fjm9006966" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref70"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref70'); return false;" data-citation="" class="refNumLink">70</a></strong><div class="NLM_citation" id="cit70"><span><span class="NLM_contrib-group">Powers, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morandi, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Structure-based discovery of a novel, noncovalent inhibitor of AmpC beta-lactamase</span> <span class="citation_source-journal">Structure</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">1013</span><span class="NLM_x">–</span> <span class="NLM_lpage">1023</span><span class="refDoi"> DOI: 10.1016/S0969-2126(02)00799-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1016%2FS0969-2126%2802%2900799-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=12121656" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BD38XlsVyjsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2002&pages=1013-1023&author=R.+A.+Powersauthor=F.+Morandiauthor=B.+K.+Shoichet&title=Structure-based+discovery+of+a+novel%2C+noncovalent+inhibitor+of+AmpC+beta-lactamase&doi=10.1016%2FS0969-2126%2802%2900799-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Discovery of a Novel, Noncovalent Inhibitor of AmpC β-Lactamase</span></div><div class="casAuthors">Powers, Rachel A.; Morandi, Federica; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1013-1023</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">β-Lactamases are the most widespread resistance mechanisms to β-lactam antibiotics, and there is a pressing need for novel, non-β-lactam drugs.  A database of over 200,000 compds. was docked to the active site of AmpC β-lactamase to identify potential inhibitors.  Fifty-six compds. were tested, and three had Ki values of 650 μM or better.  The best of these, 3-[(4-chloroanilino)sulfonyl]thiophene-2-carboxylic acid, was a competitive noncovalent inhibitor (Ki = 26 μM), which also reversed resistance to β-lactams in bacteria expressing AmpC.  The structure of AmpC in complex with this compd. was detd. by x-ray crystallog. to 1.94 A and reveals that the inhibitor interacts with key active-site residues in sites targeted in the docking calcn.  Indeed, the exptl. detd. conformation of the inhibitor closely resembles the prediction.  The structure of the enzyme-inhibitor complex presents an opportunity to improve binding affinity in a novel series of inhibitors discovered by structure-based methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKX-gft1lXA7Vg90H21EOLACvtfcHk0lg5gNhdSoZKsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XlsVyjsrk%253D&md5=5f539484d08191f0c65626493fc1b262</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1016%2FS0969-2126%2802%2900799-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0969-2126%252802%252900799-2%26sid%3Dliteratum%253Aachs%26aulast%3DPowers%26aufirst%3DR.%2BA.%26aulast%3DMorandi%26aufirst%3DF.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DStructure-based%2520discovery%2520of%2520a%2520novel%252C%2520noncovalent%2520inhibitor%2520of%2520AmpC%2520beta-lactamase%26jtitle%3DStructure%26date%3D2002%26volume%3D10%26spage%3D1013%26epage%3D1023%26doi%3D10.1016%2FS0969-2126%2802%2900799-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref71"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref71'); return false;" data-citation="" class="refNumLink">71</a></strong><div class="NLM_citation" id="cit71"><span><span class="NLM_contrib-group">Winter, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lobley, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prince, S. M.</span><span> </span><span class="NLM_article-title">Decision making in xia2</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">69</span><span class="NLM_x">, </span> <span class="NLM_fpage">1260</span><span class="NLM_x">–</span> <span class="NLM_lpage">1273</span><span class="refDoi"> DOI: 10.1107/S0907444913015308</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1107%2FS0907444913015308" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=23793152" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpvVejsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2013&pages=1260-1273&author=G.+Winterauthor=C.+M.+Lobleyauthor=S.+M.+Prince&title=Decision+making+in+xia2&doi=10.1107%2FS0907444913015308"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Decision making in xia2</span></div><div class="casAuthors">Winter, Graeme; Lobley, Carina M. C.; Prince, Stephen M.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1260-1273</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Xia2 is an expert system for the automated redn. of macromol. crystallog. (MX) data employing well trusted existing software.  The system can process a full MX data set consisting of one or more sequences of images at one or more wavelengths from images to structure-factor amplitudes with no user input.  To achieve this many decisions are made, the rationale for which is described here.  In addn., it is crit. to support the testing of hypotheses and to allow feedback of results from later stages in the anal. to earlier points where decisions were made: the flexible framework employed by xia2 to support this feedback is summarized here.  While the decision-making protocols described here were developed for xia2, they are equally applicable to interactive data redn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrMCxz3Jkl1CrVg90H21EOLACvtfcHk0lj-lvymh4MhJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXpvVejsro%253D&md5=48d02842ee5967636a5f6042cfd90b25</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1107%2FS0907444913015308&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444913015308%26sid%3Dliteratum%253Aachs%26aulast%3DWinter%26aufirst%3DG.%26aulast%3DLobley%26aufirst%3DC.%2BM.%26aulast%3DPrince%26aufirst%3DS.%2BM.%26atitle%3DDecision%2520making%2520in%2520xia2%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2013%26volume%3D69%26spage%3D1260%26epage%3D1273%26doi%3D10.1107%2FS0907444913015308" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref72"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref72'); return false;" data-citation="" class="refNumLink">72</a></strong><div class="NLM_citation" id="cit72"><span><span class="NLM_contrib-group">Copeland, R. A.</span> <span class="citation_source-book">Enzymes: A Practical Introduction to Structure, Mechanism, and Data Analysis</span>, <span class="NLM_edition">2</span>nd ed.; <span class="NLM_publisher-name">Wiley</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">2000</span><span class="NLM_x">; </span>p  <span class="NLM_fpage">xvi</span>,  <span class="NLM_fpage">397</span> pp.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1002%2F0471220639" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2000&pages=xvi+397&author=R.+A.+Copeland&title=Enzymes%3A+A+Practical+Introduction+to+Structure%2C+Mechanism%2C+and+Data+Analysis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1002%2F0471220639&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F0471220639%26sid%3Dliteratum%253Aachs%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26btitle%3DEnzymes%253A%2520A%2520Practical%2520Introduction%2520to%2520Structure%252C%2520Mechanism%252C%2520and%2520Data%2520Analysis%26pub%3DWiley%26date%3D2000%26spage%3Dxvi%26spage%3D397" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref73"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref73'); return false;" data-citation="" class="refNumLink">73</a></strong><div class="NLM_citation" id="cit73"><span><span class="NLM_contrib-group">Kawamura, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tumber, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rose, N. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">King, O. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Daniel, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oppermann, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heightman, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C.</span><span> </span><span class="NLM_article-title">Development of homogeneous luminescence assays for histone demethylase catalysis and binding</span> <span class="citation_source-journal">Anal. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">404</span><span class="NLM_x">, </span> <span class="NLM_fpage">86</span><span class="NLM_x">–</span> <span class="NLM_lpage">93</span><span class="refDoi"> DOI: 10.1016/j.ab.2010.04.030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1016%2Fj.ab.2010.04.030" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=404&publication_year=2010&pages=86-93&author=A.+Kawamuraauthor=A.+Tumberauthor=N.+R.+Roseauthor=O.+N.+Kingauthor=M.+Danielauthor=U.+Oppermannauthor=T.+D.+Heightmanauthor=C.+Schofield&title=Development+of+homogeneous+luminescence+assays+for+histone+demethylase+catalysis+and+binding&doi=10.1016%2Fj.ab.2010.04.030"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1016%2Fj.ab.2010.04.030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ab.2010.04.030%26sid%3Dliteratum%253Aachs%26aulast%3DKawamura%26aufirst%3DA.%26aulast%3DTumber%26aufirst%3DA.%26aulast%3DRose%26aufirst%3DN.%2BR.%26aulast%3DKing%26aufirst%3DO.%2BN.%26aulast%3DDaniel%26aufirst%3DM.%26aulast%3DOppermann%26aufirst%3DU.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DSchofield%26aufirst%3DC.%26atitle%3DDevelopment%2520of%2520homogeneous%2520luminescence%2520assays%2520for%2520histone%2520demethylase%2520catalysis%2520and%2520binding%26jtitle%3DAnal.%2520Biochem.%26date%3D2010%26volume%3D404%26spage%3D86%26epage%3D93%26doi%3D10.1016%2Fj.ab.2010.04.030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref74"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref74'); return false;" data-citation="" class="refNumLink">74</a></strong><div class="NLM_citation" id="cit74"><span><span class="NLM_contrib-group">McGovern, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Helfand, B. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">A specific mechanism of nonspecific inhibition</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">4265</span><span class="NLM_x">–</span> <span class="NLM_lpage">4272</span><span class="refDoi"> DOI: 10.1021/jm030266r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm030266r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BD3sXmsFamsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2003&pages=4265-4272&author=S.+L.+McGovernauthor=B.+T.+Helfandauthor=B.+Fengauthor=B.+K.+Shoichet&title=A+specific+mechanism+of+nonspecific+inhibition&doi=10.1021%2Fjm030266r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">A Specific Mechanism of Nonspecific Inhibition</span></div><div class="casAuthors">McGovern, Susan L.; Helfand, Brian T.; Feng, Brian; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">4265-4272</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Promiscuous small mols. plague screening libraries and hit lists.  Previous work has found that several nonspecific compds. form submicrometer aggregates, and it has been suggested that this aggregate species is responsible for the inhibition of many different enzymes.  It is not understood how aggregates inhibit their targets.  To address this question, biophys., kinetic, and microscopy methods were used to study the interaction of promiscuous, aggregate-forming inhibitors with model proteins.  By use of centrifugation and gel electrophoresis, aggregates and protein were found to directly interact.  This is consistent with a subsequent observation from confocal fluorescence microscopy that aggregates conc. green fluorescent protein.  β-Lactamase mutants with increased or decreased thermodn. stability relative to wild-type enzyme were equally inhibited by an aggregate-forming compd., suggesting that denaturation by unfolding was not the primary mechanism of interaction.  Instead, visualization by electron microscopy revealed that enzyme assocs. with the surface of inhibitor aggregates.  This assocn. could be reversed or prevented by the addn. of Triton X-100.  These observations suggest that the aggregates formed by promiscuous compds. reversibly sequester enzyme, resulting in apparent inhibition.  They also suggest a simple method to identify or reverse the action of aggregate-based inhibitors, which appear to be widespread.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppdLVG_GeLZbVg90H21EOLACvtfcHk0lj-lvymh4MhJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXmsFamsbs%253D&md5=cd0fd6bf0a64ceb7d7219dae2406dd21</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1021%2Fjm030266r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030266r%26sid%3Dliteratum%253Aachs%26aulast%3DMcGovern%26aufirst%3DS.%2BL.%26aulast%3DHelfand%26aufirst%3DB.%2BT.%26aulast%3DFeng%26aufirst%3DB.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DA%2520specific%2520mechanism%2520of%2520nonspecific%2520inhibition%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2003%26volume%3D46%26spage%3D4265%26epage%3D4272%26doi%3D10.1021%2Fjm030266r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref75"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref75'); return false;" data-citation="" class="refNumLink">75</a></strong><div class="NLM_citation" id="cit75"><span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duan, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Torosyan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doak, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ziebart, K. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sterling, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tumanian, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">An Aggregation Advisor for Ligand Discovery</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">7076</span><span class="NLM_x">–</span> <span class="NLM_lpage">7087</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01105</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01105" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlOqu7zM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=7076-7087&author=J.+J.+Irwinauthor=D.+Duanauthor=H.+Torosyanauthor=A.+K.+Doakauthor=K.+T.+Ziebartauthor=T.+Sterlingauthor=G.+Tumanianauthor=B.+K.+Shoichet&title=An+Aggregation+Advisor+for+Ligand+Discovery&doi=10.1021%2Facs.jmedchem.5b01105"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">An Aggregation Advisor for Ligand Discovery</span></div><div class="casAuthors">Irwin, John J.; Duan, Da; Torosyan, Hayarpi; Doak, Allison K.; Ziebart, Kristin T.; Sterling, Teague; Tumanian, Gurgen; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">7076-7087</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Colloidal aggregation of org. mols. is the dominant mechanism for artifactual inhibition of proteins, and controls against it are widely deployed.  Notwithstanding an increasingly detailed understanding of this phenomenon, a method to reliably predict aggregation has remained elusive.  Correspondingly, active mols. that act via aggregation continue to be found in early discovery campaigns and remain common in the literature.  Over the past decade, over 12 thousand aggregating org. mols. have been identified, potentially enabling a precedent-based approach to match known aggregators with new mols. that may be expected to aggregate and lead to artifacts.  We investigate an approach that uses lipophilicity, affinity, and similarity to known aggregators to advise on the likelihood that a candidate compd. is an aggregator.  In prospective exptl. testing, five of seven new mols. with Tanimoto coeffs. (Tc's) between 0.95 and 0.99 to known aggregators aggregated at relevant concns.  Ten of 19 with Tc's between 0.94 and 0.90 and three of seven with Tc's between 0.89 and 0.85 also aggregated.  Another three of the predicted compds. aggregated at higher concns.  This method finds that 61 827 or 5.1% of the ligands acting in the 0.1 to 10 μM range in the medicinal chem. literature are at least 85% similar to a known aggregator with these phys. properties and may aggregate at relevant concns.  Intriguingly, only 0.73% of all drug-like com. available compds. resemble the known aggregators, suggesting that colloidal aggregators are enriched in the literature.  As a percentage of the literature, aggregator-like compds. have increased 9-fold since 1995, partly reflecting the advent of high-throughput and virtual screens against mol. targets.  Emerging from this study is an aggregator advisor database and tool (http://advisor.bkslab.org), free to the community, that may help distinguish between fruitful and artifactual screening hits acting by this mechanism.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTKGJSfIz30bVg90H21EOLACvtfcHk0lifc5XqR26oeA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlOqu7zM&md5=d3112f9f3a57a2ac61a4b32f31778b1d</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01105%26sid%3Dliteratum%253Aachs%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DDuan%26aufirst%3DD.%26aulast%3DTorosyan%26aufirst%3DH.%26aulast%3DDoak%26aufirst%3DA.%2BK.%26aulast%3DZiebart%26aufirst%3DK.%2BT.%26aulast%3DSterling%26aufirst%3DT.%26aulast%3DTumanian%26aufirst%3DG.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DAn%2520Aggregation%2520Advisor%2520for%2520Ligand%2520Discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D7076%26epage%3D7087%26doi%3D10.1021%2Facs.jmedchem.5b01105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref76"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref76'); return false;" data-citation="" class="refNumLink">76</a></strong><div class="NLM_citation" id="cit76"><span><span class="NLM_contrib-group">Cascella, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mirica, L. M.</span><span> </span><span class="NLM_article-title">Kinetic analysis of iron-dependent histone demethylases: alpha-ketoglutarate substrate inhibition and potential relevance to the regulation of histone demethylation in cancer cells</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">8699</span><span class="NLM_x">–</span> <span class="NLM_lpage">8701</span><span class="refDoi"> DOI: 10.1021/bi3012466</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi3012466" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2012&pages=8699-8701&author=B.+Cascellaauthor=L.+M.+Mirica&title=Kinetic+analysis+of+iron-dependent+histone+demethylases%3A+alpha-ketoglutarate+substrate+inhibition+and+potential+relevance+to+the+regulation+of+histone+demethylation+in+cancer+cells&doi=10.1021%2Fbi3012466"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1021%2Fbi3012466&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi3012466%26sid%3Dliteratum%253Aachs%26aulast%3DCascella%26aufirst%3DB.%26aulast%3DMirica%26aufirst%3DL.%2BM.%26atitle%3DKinetic%2520analysis%2520of%2520iron-dependent%2520histone%2520demethylases%253A%2520alpha-ketoglutarate%2520substrate%2520inhibition%2520and%2520potential%2520relevance%2520to%2520the%2520regulation%2520of%2520histone%2520demethylation%2520in%2520cancer%2520cells%26jtitle%3DBiochemistry%26date%3D2012%26volume%3D51%26spage%3D8699%26epage%3D8701%26doi%3D10.1021%2Fbi3012466" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68'],'ref69':['cit69'],'ref70':['cit70'],'ref71':['cit71'],'ref72':['cit72'],'ref73':['cit73'],'ref74':['cit74'],'ref75':['cit75'],'ref76':['cit76']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 32 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Mengyang Xu, Chao Zhao, Biying Zhu, Liangyue Wang, Huihao Zhou, Daoguang Yan, Qiong Gu, <span class="NLM_string-name hlFld-ContribAuthor">Jun Xu</span>. </span><span class="cited-content_cbyCitation_article-title">Discovering High Potent Hsp90 Inhibitors as Antinasopharyngeal Carcinoma Agents through Fragment Assembling Approach. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (4)
                                     , 2010-2023. <a href="https://doi.org/10.1021/acs.jmedchem.0c01521" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01521</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01521&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01521%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovering%252BHigh%252BPotent%252BHsp90%252BInhibitors%252Bas%252BAntinasopharyngeal%252BCarcinoma%252BAgents%252Bthrough%252BFragment%252BAssembling%252BApproach%26aulast%3DXu%26aufirst%3DMengyang%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D03092020%26date%3D05022021%26volume%3D64%26issue%3D4%26spage%3D2010%26epage%3D2023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Ting Ran, Rongquan Xiao, Qixuan Huang, Haoliang Yuan, Tao Lu, <span class="NLM_string-name hlFld-ContribAuthor">Wen Liu</span>. </span><span class="cited-content_cbyCitation_article-title">In Silico Discovery of JMJD6 Inhibitors for Cancer Treatment. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2019,</strong> <em>10 </em>
                                    (12)
                                     , 1609-1613. <a href="https://doi.org/10.1021/acsmedchemlett.9b00264" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.9b00264</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.9b00264&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.9b00264%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DIn%252BSilico%252BDiscovery%252Bof%252BJMJD6%252BInhibitors%252Bfor%252BCancer%252BTreatment%26aulast%3DRan%26aufirst%3DTing%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D12062019%26date%3D19112019%26date%3D22112019%26date%3D19112019%26volume%3D10%26issue%3D12%26spage%3D1609%26epage%3D1613" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">C. Esposito, L. Wiedmer, <span class="NLM_string-name hlFld-ContribAuthor">A. Caflisch</span>. </span><span class="cited-content_cbyCitation_article-title">In Silico Identification of JMJD3 Demethylase Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2018,</strong> <em>58 </em>
                                    (10)
                                     , 2151-2163. <a href="https://doi.org/10.1021/acs.jcim.8b00539" title="DOI URL">https://doi.org/10.1021/acs.jcim.8b00539</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.8b00539&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.8b00539%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DIn%252BSilico%252BIdentification%252Bof%252BJMJD3%252BDemethylase%252BInhibitors%26aulast%3DEsposito%26aufirst%3DC.%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D10082018%26date%3D03102018%26date%3D18092018%26volume%3D58%26issue%3D10%26spage%3D2151%26epage%3D2163" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Pradeep
K. Jaiswal, Vashundhra Sharma, Manas Mathur, <span class="NLM_string-name hlFld-ContribAuthor">Sandeep Chaudhary</span>. </span><span class="cited-content_cbyCitation_article-title">Organocatalytic Modified Guareschi–Thorpe Type Regioselective Synthesis: A Unified Direct Access to 5,6,7,8-Tetrahydroquinolines and Other Alicyclic[b]-Fused Pyridines. </span><span class="cited-content_cbyCitation_journal-name">Organic Letters</span><span> <strong>2018,</strong> <em>20 </em>
                                    (19)
                                     , 6059-6063. <a href="https://doi.org/10.1021/acs.orglett.8b02132" title="DOI URL">https://doi.org/10.1021/acs.orglett.8b02132</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.orglett.8b02132&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.orglett.8b02132%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520Letters%26atitle%3DOrganocatalytic%252BModified%252BGuareschi%2525E2%252580%252593Thorpe%252BType%252BRegioselective%252BSynthesis%25253A%252BA%252BUnified%252BDirect%252BAccess%252Bto%252B5%25252C6%25252C7%25252C8-Tetrahydroquinolines%252Band%252BOther%252BAlicyclic%25255Bb%25255D-Fused%252BPyridines%26aulast%3DJaiswal%26aufirst%3DPradeep%2BK.%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D01082018%26date%3D14092018%26volume%3D20%26issue%3D19%26spage%3D6059%26epage%3D6063" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Christopher N. Johnson, Daniel A. Erlanson, Wolfgang Jahnke, Paul N. Mortenson, <span class="NLM_string-name hlFld-ContribAuthor">David C. Rees</span>. </span><span class="cited-content_cbyCitation_article-title">Fragment-to-Lead Medicinal Chemistry Publications in 2016. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (5)
                                     , 1774-1784. <a href="https://doi.org/10.1021/acs.jmedchem.7b01298" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b01298</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b01298&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.7b01298%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DFragment-to-Lead%252BMedicinal%252BChemistry%252BPublications%252Bin%252B2016%26aulast%3DJohnson%26aufirst%3DChristopher%2BN.%26date%3D2018%26date%3D2017%26date%3D2017%26date%3D31082017%26date%3D31102017%26volume%3D61%26issue%3D5%26spage%3D1774%26epage%3D1784" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">H. Ümit  Kaniskan</span>, <span class="hlFld-ContribAuthor ">Michael L.  Martini</span>, and <span class="hlFld-ContribAuthor ">Jian  Jin</span>  . </span><span class="cited-content_cbyCitation_article-title">Inhibitors of Protein Methyltransferases and Demethylases. </span><span class="cited-content_cbyCitation_journal-name">Chemical Reviews</span><span> <strong>2018,</strong> <em>118 </em>
                                    (3)
                                     , 989-1068. <a href="https://doi.org/10.1021/acs.chemrev.6b00801" title="DOI URL">https://doi.org/10.1021/acs.chemrev.6b00801</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.chemrev.6b00801&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.chemrev.6b00801%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Reviews%26atitle%3DInhibitors%252Bof%252BProtein%252BMethyltransferases%252Band%252BDemethylases%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259Cmit%26date%3D2018%26date%3D2017%26date%3D2016%26date%3D30112016%26date%3D24032017%26date%3D14022018%26volume%3D118%26issue%3D3%26spage%3D989%26epage%3D1068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Guan-Jun  Yang</span>, <span class="hlFld-ContribAuthor ">Ming-Hui  Zhu</span>, <span class="hlFld-ContribAuthor ">Xin-Jiang  Lu</span>, <span class="hlFld-ContribAuthor ">Yan-Jun  Liu</span>, <span class="hlFld-ContribAuthor ">Jian-Fei  Lu</span>, <span class="hlFld-ContribAuthor ">Chung-Hang  Leung</span>, <span class="hlFld-ContribAuthor ">Dik-Lung  Ma</span>, <span class="hlFld-ContribAuthor ">Jiong  Chen</span>. </span><span class="cited-content_cbyCitation_article-title">The emerging role of KDM5A in human cancer. </span><span class="cited-content_cbyCitation_journal-name">Journal of Hematology & Oncology</span><span> <strong>2021,</strong> <em>14 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13045-021-01041-1" title="DOI URL">https://doi.org/10.1186/s13045-021-01041-1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13045-021-01041-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13045-021-01041-1%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Hematology%2520%2526%2520Oncology%26atitle%3DThe%252Bemerging%252Brole%252Bof%252BKDM5A%252Bin%252Bhuman%252Bcancer%26aulast%3DYang%26aufirst%3DGuan-Jun%26date%3D2021%26date%3D2021%26volume%3D14%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhen  Fang</span>, <span class="hlFld-ContribAuthor ">Yang  Liu</span>, <span class="hlFld-ContribAuthor ">Rong  Zhang</span>, <span class="hlFld-ContribAuthor ">Qiang  Chen</span>, <span class="hlFld-ContribAuthor ">Tianqi  Wang</span>, <span class="hlFld-ContribAuthor ">Wei  Yang</span>, <span class="hlFld-ContribAuthor ">Yan  Fan</span>, <span class="hlFld-ContribAuthor ">Chundong  Yu</span>, <span class="hlFld-ContribAuthor ">Rong  Xiang</span>, <span class="hlFld-ContribAuthor ">Shengyong  Yang</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of a potent and selective inhibitor of histone lysine demethylase KDM4D. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>223 </em>, 113662. <a href="https://doi.org/10.1016/j.ejmech.2021.113662" title="DOI URL">https://doi.org/10.1016/j.ejmech.2021.113662</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2021.113662&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2021.113662%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Ba%252Bpotent%252Band%252Bselective%252Binhibitor%252Bof%252Bhistone%252Blysine%252Bdemethylase%252BKDM4D%26aulast%3DFang%26aufirst%3DZhen%26date%3D2021%26volume%3D223%26spage%3D113662" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Stephin  Baby</span>, <span class="hlFld-ContribAuthor ">Durgesh  Gurukkala Valapil</span>, <span class="hlFld-ContribAuthor ">Nagula  Shankaraiah</span>. </span><span class="cited-content_cbyCitation_article-title">Unravelling KDM4 histone demethylase inhibitors for cancer therapy. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today</span><span> <strong>2021,</strong> <em>21 </em><a href="https://doi.org/10.1016/j.drudis.2021.05.015" title="DOI URL">https://doi.org/10.1016/j.drudis.2021.05.015</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.drudis.2021.05.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.drudis.2021.05.015%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%26atitle%3DUnravelling%252BKDM4%252Bhistone%252Bdemethylase%252Binhibitors%252Bfor%252Bcancer%252Btherapy%26aulast%3DBaby%26aufirst%3DStephin%26date%3D2021%26volume%3D21" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yudibeth  Sixto-López</span>, <span class="hlFld-ContribAuthor ">José  Correa-Basurto</span>. </span><span class="cited-content_cbyCitation_article-title">Docking Approaches Used in Epigenetic Drug Investigations. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 367-390. <a href="https://doi.org/10.1016/B978-0-12-822312-3.00016-3" title="DOI URL">https://doi.org/10.1016/B978-0-12-822312-3.00016-3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-822312-3.00016-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-822312-3.00016-3%26sid%3Dliteratum%253Aachs%26atitle%3DDocking%252BApproaches%252BUsed%252Bin%252BEpigenetic%252BDrug%252BInvestigations%26aulast%3DSixto-L%25C3%25B3pez%26aufirst%3DYudibeth%26date%3D2021%26spage%3D367%26epage%3D390%26pub%3DElsevier%26atitle%3DMolecular%252BDocking%252Bfor%252BComputer-Aided%252BDrug%252BDesign%26date%3D2021%26volume%3D2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ronald W  Brown</span>. </span><span class="cited-content_cbyCitation_article-title">Abiotic scaffolds in medicinal chemistry: not a waste of chemical space. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2021,</strong> <em>13 </em>
                                    (2)
                                     , 211-224. <a href="https://doi.org/10.4155/fmc-2020-0294" title="DOI URL">https://doi.org/10.4155/fmc-2020-0294</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc-2020-0294&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc-2020-0294%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DAbiotic%252Bscaffolds%252Bin%252Bmedicinal%252Bchemistry%25253A%252Bnot%252Ba%252Bwaste%252Bof%252Bchemical%252Bspace%26aulast%3DBrown%26aufirst%3DRonald%2BW%26date%3D2021%26volume%3D13%26issue%3D2%26spage%3D211%26epage%3D224" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Qingxin  Li</span>. </span><span class="cited-content_cbyCitation_article-title">Application of Fragment-Based Drug Discovery to Versatile Targets. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Molecular Biosciences</span><span> <strong>2020,</strong> <em>7 </em><a href="https://doi.org/10.3389/fmolb.2020.00180" title="DOI URL">https://doi.org/10.3389/fmolb.2020.00180</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fmolb.2020.00180&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffmolb.2020.00180%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Molecular%2520Biosciences%26atitle%3DApplication%252Bof%252BFragment-Based%252BDrug%252BDiscovery%252Bto%252BVersatile%252BTargets%26aulast%3DLi%26aufirst%3DQingxin%26date%3D2020%26date%3D2020%26volume%3D7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">José A.  Souto</span>, <span class="hlFld-ContribAuthor ">Federica  Sarno</span>, <span class="hlFld-ContribAuthor ">Angela  Nebbioso</span>, <span class="hlFld-ContribAuthor ">Chiara  Papulino</span>, <span class="hlFld-ContribAuthor ">Rosana  Álvarez</span>, <span class="hlFld-ContribAuthor ">Jessica  Lombino</span>, <span class="hlFld-ContribAuthor ">Ugo  Perricone</span>, <span class="hlFld-ContribAuthor ">Alessandro  Padova</span>, <span class="hlFld-ContribAuthor ">Lucia  Altucci</span>, <span class="hlFld-ContribAuthor ">Ángel R.  de Lera</span>. </span><span class="cited-content_cbyCitation_article-title">A New Family of Jumonji C Domain-Containing KDM Inhibitors Inspired by Natural Product Purpurogallin. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Chemistry</span><span> <strong>2020,</strong> <em>8 </em><a href="https://doi.org/10.3389/fchem.2020.00312" title="DOI URL">https://doi.org/10.3389/fchem.2020.00312</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fchem.2020.00312&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffchem.2020.00312%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Chemistry%26atitle%3DA%252BNew%252BFamily%252Bof%252BJumonji%252BC%252BDomain-Containing%252BKDM%252BInhibitors%252BInspired%252Bby%252BNatural%252BProduct%252BPurpurogallin%26aulast%3DSouto%26aufirst%3DJos%25C3%25A9%2BA.%26date%3D2020%26date%3D2020%26volume%3D8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Piotr H.  Małecki</span>, <span class="hlFld-ContribAuthor ">Nicole  Rüger</span>, <span class="hlFld-ContribAuthor ">Martin  Roatsch</span>, <span class="hlFld-ContribAuthor ">Oxana  Krylova</span>, <span class="hlFld-ContribAuthor ">Andreas  Link</span>, <span class="hlFld-ContribAuthor ">Manfred  Jung</span>, <span class="hlFld-ContribAuthor ">Udo  Heinemann</span>, <span class="hlFld-ContribAuthor ">Manfred S.  Weiss</span>. </span><span class="cited-content_cbyCitation_article-title">Structure‐Based Screening of Tetrazolylhydrazide Inhibitors versus KDM4 Histone Demethylases. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2019,</strong> <em>14 </em>
                                    (21)
                                     , 1828-1839. <a href="https://doi.org/10.1002/cmdc.201900441" title="DOI URL">https://doi.org/10.1002/cmdc.201900441</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.201900441&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.201900441%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DStructure%2525E2%252580%252590Based%252BScreening%252Bof%252BTetrazolylhydrazide%252BInhibitors%252Bversus%252BKDM4%252BHistone%252BDemethylases%26aulast%3DMa%25C5%2582ecki%26aufirst%3DPiotr%2BH.%26date%3D2019%26date%3D2019%26volume%3D14%26issue%3D21%26spage%3D1828%26epage%3D1839" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Elena  Lenci</span>, <span class="hlFld-ContribAuthor ">Andrea  Trabocchi</span>. </span><span class="cited-content_cbyCitation_article-title">Smart Design of Small‐Molecule Libraries: When Organic Synthesis Meets Cheminformatics. </span><span class="cited-content_cbyCitation_journal-name">ChemBioChem</span><span> <strong>2019,</strong> <em>20 </em>
                                    (9)
                                     , 1115-1123. <a href="https://doi.org/10.1002/cbic.201800751" title="DOI URL">https://doi.org/10.1002/cbic.201800751</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cbic.201800751&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcbic.201800751%26sid%3Dliteratum%253Aachs%26jtitle%3DChemBioChem%26atitle%3DSmart%252BDesign%252Bof%252BSmall%2525E2%252580%252590Molecule%252BLibraries%25253A%252BWhen%252BOrganic%252BSynthesis%252BMeets%252BCheminformatics%26aulast%3DLenci%26aufirst%3DElena%26date%3D2019%26date%3D2019%26volume%3D20%26issue%3D9%26spage%3D1115%26epage%3D1123" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dina  Robaa</span>, <span class="hlFld-ContribAuthor ">Jelena  Melesina</span>, <span class="hlFld-ContribAuthor ">Chiara  Luise</span>, <span class="hlFld-ContribAuthor ">Wolfgang  Sippl</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-Based Design of Epigenetic Inhibitors. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 455-486. <a href="https://doi.org/10.1007/7355_2019_80" title="DOI URL">https://doi.org/10.1007/7355_2019_80</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/7355_2019_80&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F7355_2019_80%26sid%3Dliteratum%253Aachs%26atitle%3DStructure-Based%252BDesign%252Bof%252BEpigenetic%252BInhibitors%26aulast%3DRobaa%26aufirst%3DDina%26date%3D2019%26date%3D2019%26spage%3D455%26epage%3D486%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DChemical%252BEpigenetics%26aulast%3DMai%26aufirst%3DAntonello%26date%3D2020%26volume%3D33" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Daniel A.  Erlanson</span>, <span class="hlFld-ContribAuthor ">Ben J.  Davis</span>, <span class="hlFld-ContribAuthor ">Wolfgang  Jahnke</span>. </span><span class="cited-content_cbyCitation_article-title">Fragment-Based Drug Discovery: Advancing Fragments in the Absence of Crystal Structures. </span><span class="cited-content_cbyCitation_journal-name">Cell Chemical Biology</span><span> <strong>2019,</strong> <em>26 </em>
                                    (1)
                                     , 9-15. <a href="https://doi.org/10.1016/j.chembiol.2018.10.001" title="DOI URL">https://doi.org/10.1016/j.chembiol.2018.10.001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.chembiol.2018.10.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.chembiol.2018.10.001%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Chemical%2520Biology%26atitle%3DFragment-Based%252BDrug%252BDiscovery%25253A%252BAdvancing%252BFragments%252Bin%252Bthe%252BAbsence%252Bof%252BCrystal%252BStructures%26aulast%3DErlanson%26aufirst%3DDaniel%2BA.%26date%3D2019%26volume%3D26%26issue%3D1%26spage%3D9%26epage%3D15" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Guan-Jun  Yang</span>, <span class="hlFld-ContribAuthor ">Chung-Nga  Ko</span>, <span class="hlFld-ContribAuthor ">Hai-Jing  Zhong</span>, <span class="hlFld-ContribAuthor ">Chung-Hang  Leung</span>, <span class="hlFld-ContribAuthor ">Dik-Lung  Ma</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-Based Discovery of a Selective KDM5A Inhibitor that Exhibits Anti-Cancer Activity via Inducing Cell Cycle Arrest and Senescence in Breast Cancer Cell Lines. </span><span class="cited-content_cbyCitation_journal-name">Cancers</span><span> <strong>2019,</strong> <em>11 </em>
                                    (1)
                                     , 92. <a href="https://doi.org/10.3390/cancers11010092" title="DOI URL">https://doi.org/10.3390/cancers11010092</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/cancers11010092&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fcancers11010092%26sid%3Dliteratum%253Aachs%26jtitle%3DCancers%26atitle%3DStructure-Based%252BDiscovery%252Bof%252Ba%252BSelective%252BKDM5A%252BInhibitor%252Bthat%252BExhibits%252BAnti-Cancer%252BActivity%252Bvia%252BInducing%252BCell%252BCycle%252BArrest%252Band%252BSenescence%252Bin%252BBreast%252BCancer%252BCell%252BLines%26aulast%3DYang%26aufirst%3DGuan-Jun%26date%3D2019%26date%3D2019%26volume%3D11%26issue%3D1%26spage%3D92" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lucía  Guillade</span>, <span class="hlFld-ContribAuthor ">Federica  Sarno</span>, <span class="hlFld-ContribAuthor ">Hanna  Tarhonskaya</span>, <span class="hlFld-ContribAuthor ">Angela  Nebbioso</span>, <span class="hlFld-ContribAuthor ">Susana  Alvarez</span>, <span class="hlFld-ContribAuthor ">Akane  Kawamura</span>, <span class="hlFld-ContribAuthor ">Christopher J.  Schofield</span>, <span class="hlFld-ContribAuthor ">Lucia  Altucci</span>, <span class="hlFld-ContribAuthor ">Ángel R.  de Lera</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis and Biological Evaluation of Tripartin, a Putative KDM4 Natural Product Inhibitor, and 1-Dichloromethylinden-1-ol Analogues. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2018,</strong> <em>13 </em>
                                    (18)
                                     , 1949-1956. <a href="https://doi.org/10.1002/cmdc.201800377" title="DOI URL">https://doi.org/10.1002/cmdc.201800377</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.201800377&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.201800377%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DSynthesis%252Band%252BBiological%252BEvaluation%252Bof%252BTripartin%25252C%252Ba%252BPutative%252BKDM4%252BNatural%252BProduct%252BInhibitor%25252C%252Band%252B1-Dichloromethylinden-1-ol%252BAnalogues%26aulast%3DGuillade%26aufirst%3DLuc%25C3%25ADa%26date%3D2018%26date%3D2018%26volume%3D13%26issue%3D18%26spage%3D1949%26epage%3D1956" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alexander-Thomas  Hauser</span>, <span class="hlFld-ContribAuthor ">Dina  Robaa</span>, <span class="hlFld-ContribAuthor ">Manfred  Jung</span>. </span><span class="cited-content_cbyCitation_article-title">Epigenetic small molecule modulators of histone and DNA methylation. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Chemical Biology</span><span> <strong>2018,</strong> <em>45 </em>, 73-85. <a href="https://doi.org/10.1016/j.cbpa.2018.03.003" title="DOI URL">https://doi.org/10.1016/j.cbpa.2018.03.003</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cbpa.2018.03.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cbpa.2018.03.003%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Chemical%2520Biology%26atitle%3DEpigenetic%252Bsmall%252Bmolecule%252Bmodulators%252Bof%252Bhistone%252Band%252BDNA%252Bmethylation%26aulast%3DHauser%26aufirst%3DAlexander-Thomas%26date%3D2018%26volume%3D45%26spage%3D73%26epage%3D85" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Anup K.  Upadhyay</span>, <span class="hlFld-ContribAuthor ">Russell A.  Judge</span>, <span class="hlFld-ContribAuthor ">Leiming  Li</span>, <span class="hlFld-ContribAuthor ">Ron  Pithawalla</span>, <span class="hlFld-ContribAuthor ">Justin  Simanis</span>, <span class="hlFld-ContribAuthor ">Pierre M.  Bodelle</span>, <span class="hlFld-ContribAuthor ">Violeta L.  Marin</span>, <span class="hlFld-ContribAuthor ">Rodger F.  Henry</span>, <span class="hlFld-ContribAuthor ">Andrew M.  Petros</span>, <span class="hlFld-ContribAuthor ">Chaohong  Sun</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting lysine specific demethylase 4A (KDM4A) tandem TUDOR domain – A fragment based approach. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2018,</strong> <em>28 </em>
                                    (10)
                                     , 1708-1713. <a href="https://doi.org/10.1016/j.bmcl.2018.04.050" title="DOI URL">https://doi.org/10.1016/j.bmcl.2018.04.050</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2018.04.050&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2018.04.050%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DTargeting%252Blysine%252Bspecific%252Bdemethylase%252B4A%252B%252528KDM4A%252529%252Btandem%252BTUDOR%252Bdomain%252B%2525E2%252580%252593%252BA%252Bfragment%252Bbased%252Bapproach%26aulast%3DUpadhyay%26aufirst%3DAnup%2BK.%26date%3D2018%26volume%3D28%26issue%3D10%26spage%3D1708%26epage%3D1713" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Wenchao  Lu</span>, <span class="hlFld-ContribAuthor ">Rukang  Zhang</span>, <span class="hlFld-ContribAuthor ">Hao  Jiang</span>, <span class="hlFld-ContribAuthor ">Huimin  Zhang</span>, <span class="hlFld-ContribAuthor ">Cheng  Luo</span>. </span><span class="cited-content_cbyCitation_article-title">Computer-Aided Drug Design in Epigenetics. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Chemistry</span><span> <strong>2018,</strong> <em>6 </em><a href="https://doi.org/10.3389/fchem.2018.00057" title="DOI URL">https://doi.org/10.3389/fchem.2018.00057</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fchem.2018.00057&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffchem.2018.00057%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Chemistry%26atitle%3DComputer-Aided%252BDrug%252BDesign%252Bin%252BEpigenetics%26aulast%3DLu%26aufirst%3DWenchao%26date%3D2018%26date%3D2018%26volume%3D6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hongzhi  Lin</span>, <span class="hlFld-ContribAuthor ">Qihang  Li</span>, <span class="hlFld-ContribAuthor ">Qi  Li</span>, <span class="hlFld-ContribAuthor ">Jie  Zhu</span>, <span class="hlFld-ContribAuthor ">Kai  Gu</span>, <span class="hlFld-ContribAuthor ">Xueyang  Jiang</span>, <span class="hlFld-ContribAuthor ">Qianqian  Hu</span>, <span class="hlFld-ContribAuthor ">Feng  Feng</span>, <span class="hlFld-ContribAuthor ">Wei  Qu</span>, <span class="hlFld-ContribAuthor ">Yao  Chen</span>, <span class="hlFld-ContribAuthor ">Haopeng  Sun</span>. </span><span class="cited-content_cbyCitation_article-title">Small molecule KDM4s inhibitors as anti-cancer agents. </span><span class="cited-content_cbyCitation_journal-name">Journal of Enzyme Inhibition and Medicinal Chemistry</span><span> <strong>2018,</strong> <em>33 </em>
                                    (1)
                                     , 777-793. <a href="https://doi.org/10.1080/14756366.2018.1455676" title="DOI URL">https://doi.org/10.1080/14756366.2018.1455676</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/14756366.2018.1455676&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F14756366.2018.1455676%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Enzyme%2520Inhibition%2520and%2520Medicinal%2520Chemistry%26atitle%3DSmall%252Bmolecule%252BKDM4s%252Binhibitors%252Bas%252Banti-cancer%252Bagents%26aulast%3DLin%26aufirst%3DHongzhi%26date%3D2018%26date%3D2018%26volume%3D33%26issue%3D1%26spage%3D777%26epage%3D793" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Martina  Bigatti</span>, <span class="hlFld-ContribAuthor ">Alberto  Dal Corso</span>, <span class="hlFld-ContribAuthor ">Sara  Vanetti</span>, <span class="hlFld-ContribAuthor ">Samuele  Cazzamalli</span>, <span class="hlFld-ContribAuthor ">Ulrike  Rieder</span>, <span class="hlFld-ContribAuthor ">Jörg  Scheuermann</span>, <span class="hlFld-ContribAuthor ">Dario  Neri</span>, <span class="hlFld-ContribAuthor ">Filippo  Sladojevich</span>. </span><span class="cited-content_cbyCitation_article-title">Impact of a Central Scaffold on the Binding Affinity of Fragment Pairs Isolated from DNA-Encoded Self-Assembling Chemical Libraries. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2017,</strong> <em>12 </em>
                                    (21)
                                     , 1748-1752. <a href="https://doi.org/10.1002/cmdc.201700569" title="DOI URL">https://doi.org/10.1002/cmdc.201700569</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.201700569&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.201700569%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DImpact%252Bof%252Ba%252BCentral%252BScaffold%252Bon%252Bthe%252BBinding%252BAffinity%252Bof%252BFragment%252BPairs%252BIsolated%252Bfrom%252BDNA-Encoded%252BSelf-Assembling%252BChemical%252BLibraries%26aulast%3DBigatti%26aufirst%3DMartina%26date%3D2017%26date%3D2017%26volume%3D12%26issue%3D21%26spage%3D1748%26epage%3D1752" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Damian  W Young</span>. </span><span class="cited-content_cbyCitation_article-title">Using Fragment Based Drug Discovery to Target Epigenetic Regulators in Cancer. </span><span class="cited-content_cbyCitation_journal-name">MOJ Bioequivalence & Bioavailability</span><span> <strong>2017,</strong> <em>4 </em>
                                    (1)
                                     <a href="https://doi.org/10.15406/mojbb.2017.04.00062" title="DOI URL">https://doi.org/10.15406/mojbb.2017.04.00062</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.15406/mojbb.2017.04.00062&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.15406%2Fmojbb.2017.04.00062%26sid%3Dliteratum%253Aachs%26jtitle%3DMOJ%2520Bioequivalence%2520%2526%2520Bioavailability%26atitle%3DUsing%252BFragment%252BBased%252BDrug%252BDiscovery%252Bto%252BTarget%252BEpigenetic%252BRegulators%252Bin%252BCancer%26aulast%3DW%2BYoung%26aufirst%3DDamian%26date%3D2017%26date%3D2017%26volume%3D4%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Daniel J.  Foley</span>, <span class="hlFld-ContribAuthor ">Philip G. E.  Craven</span>, <span class="hlFld-ContribAuthor ">Patrick M.  Collins</span>, <span class="hlFld-ContribAuthor ">Richard G.  Doveston</span>, <span class="hlFld-ContribAuthor ">Anthony  Aimon</span>, <span class="hlFld-ContribAuthor ">Romain  Talon</span>, <span class="hlFld-ContribAuthor ">Ian  Churcher</span>, <span class="hlFld-ContribAuthor ">Frank  von Delft</span>, <span class="hlFld-ContribAuthor ">Stephen P.  Marsden</span>, <span class="hlFld-ContribAuthor ">Adam  Nelson</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis and Demonstration of the Biological Relevance of sp
              3
              -rich Scaffolds Distantly Related to Natural Product Frameworks. </span><span class="cited-content_cbyCitation_journal-name">Chemistry - A European Journal</span><span> <strong>2017,</strong> <em>23 </em>
                                    (60)
                                     , 15227-15232. <a href="https://doi.org/10.1002/chem.201704169" title="DOI URL">https://doi.org/10.1002/chem.201704169</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/chem.201704169&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fchem.201704169%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistry%2520-%2520A%2520European%2520Journal%26atitle%3DSynthesis%252Band%252BDemonstration%252Bof%252Bthe%252BBiological%252BRelevance%252Bof%252Bsp%252B3%252B-rich%252BScaffolds%252BDistantly%252BRelated%252Bto%252BNatural%252BProduct%252BFrameworks%26aulast%3DFoley%26aufirst%3DDaniel%2BJ.%26date%3D2017%26date%3D2017%26volume%3D23%26issue%3D60%26spage%3D15227%26epage%3D15232" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhen  Fang</span>, <span class="hlFld-ContribAuthor ">Tian-qi  Wang</span>, <span class="hlFld-ContribAuthor ">Hui  Li</span>, <span class="hlFld-ContribAuthor ">Guo  Zhang</span>, <span class="hlFld-ContribAuthor ">Xiao-ai  Wu</span>, <span class="hlFld-ContribAuthor ">Li  Yang</span>, <span class="hlFld-ContribAuthor ">Yu-lan  Peng</span>, <span class="hlFld-ContribAuthor ">Jun  Zou</span>, <span class="hlFld-ContribAuthor ">Lin-li  Li</span>, <span class="hlFld-ContribAuthor ">Rong  Xiang</span>, <span class="hlFld-ContribAuthor ">Sheng-yong  Yang</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of pyrazolo[1,5-a]pyrimidine-3-carbonitrile derivatives as a new class of histone lysine demethylase 4D (KDM4D) inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2017,</strong> <em>27 </em>
                                    (14)
                                     , 3201-3204. <a href="https://doi.org/10.1016/j.bmcl.2017.05.002" title="DOI URL">https://doi.org/10.1016/j.bmcl.2017.05.002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2017.05.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2017.05.002%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DDiscovery%252Bof%252Bpyrazolo%25255B1%25252C5-a%25255Dpyrimidine-3-carbonitrile%252Bderivatives%252Bas%252Ba%252Bnew%252Bclass%252Bof%252Bhistone%252Blysine%252Bdemethylase%252B4D%252B%252528KDM4D%252529%252Binhibitors%26aulast%3DFang%26aufirst%3DZhen%26date%3D2017%26volume%3D27%26issue%3D14%26spage%3D3201%26epage%3D3204" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">John R.  Horton</span>, <span class="hlFld-ContribAuthor ">Molly  Gale</span>, <span class="hlFld-ContribAuthor ">Qin  Yan</span>, <span class="hlFld-ContribAuthor ">Xiaodong  Cheng</span>. </span><span class="cited-content_cbyCitation_article-title">The Molecular Basis of Histone Demethylation. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2017,</strong>,, 151-219. <a href="https://doi.org/10.1007/978-3-319-59786-7_7" title="DOI URL">https://doi.org/10.1007/978-3-319-59786-7_7</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-3-319-59786-7_7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-3-319-59786-7_7%26sid%3Dliteratum%253Aachs%26atitle%3DThe%252BMolecular%252BBasis%252Bof%252BHistone%252BDemethylation%26aulast%3DHorton%26aufirst%3DJohn%2BR.%26date%3D2017%26date%3D2017%26spage%3D151%26epage%3D219%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DDNA%252Band%252BHistone%252BMethylation%252Bas%252BCancer%252BTargets%26aulast%3DKaneda%26aufirst%3DAtsushi%26date%3D2017%26volume%3D376" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Florian  Thaler</span>, <span class="hlFld-ContribAuthor ">Ciro  Mercurio</span>. </span><span class="cited-content_cbyCitation_article-title">Compounds and methods for inhibiting histone demethylases: a patent evaluation of US20160102096A1. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Therapeutic Patents</span><span> <strong>2016,</strong> <em>26 </em>
                                    (12)
                                     , 1367-1370. <a href="https://doi.org/10.1080/13543776.2016.1246539" title="DOI URL">https://doi.org/10.1080/13543776.2016.1246539</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/13543776.2016.1246539&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F13543776.2016.1246539%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Therapeutic%2520Patents%26atitle%3DCompounds%252Band%252Bmethods%252Bfor%252Binhibiting%252Bhistone%252Bdemethylases%25253A%252Ba%252Bpatent%252Bevaluation%252Bof%252BUS20160102096A1%26aulast%3DThaler%26aufirst%3DFlorian%26date%3D2016%26date%3D2016%26volume%3D26%26issue%3D12%26spage%3D1367%26epage%3D1370" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Bradley C.  Doak</span>, <span class="hlFld-ContribAuthor ">Raymond S.  Norton</span>, <span class="hlFld-ContribAuthor ">Martin J.  Scanlon</span>. </span><span class="cited-content_cbyCitation_article-title">The ways and means of fragment-based drug design. </span><span class="cited-content_cbyCitation_journal-name">Pharmacology & Therapeutics</span><span> <strong>2016,</strong> <em>167 </em>, 28-37. <a href="https://doi.org/10.1016/j.pharmthera.2016.07.003" title="DOI URL">https://doi.org/10.1016/j.pharmthera.2016.07.003</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.pharmthera.2016.07.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.pharmthera.2016.07.003%26sid%3Dliteratum%253Aachs%26jtitle%3DPharmacology%2520%2526%2520Therapeutics%26atitle%3DThe%252Bways%252Band%252Bmeans%252Bof%252Bfragment-based%252Bdrug%252Bdesign%26aulast%3DDoak%26aufirst%3DBradley%2BC.%26date%3D2016%26volume%3D167%26spage%3D28%26epage%3D37" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Daniel A.  Erlanson</span>, <span class="hlFld-ContribAuthor ">Stephen W.  Fesik</span>, <span class="hlFld-ContribAuthor ">Roderick E.  Hubbard</span>, <span class="hlFld-ContribAuthor ">Wolfgang  Jahnke</span>, <span class="hlFld-ContribAuthor ">Harren  Jhoti</span>. </span><span class="cited-content_cbyCitation_article-title">Twenty years on: the impact of fragments on drug discovery. </span><span class="cited-content_cbyCitation_journal-name">Nature Reviews Drug Discovery</span><span> <strong>2016,</strong> <em>15 </em>
                                    (9)
                                     , 605-619. <a href="https://doi.org/10.1038/nrd.2016.109" title="DOI URL">https://doi.org/10.1038/nrd.2016.109</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/nrd.2016.109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fnrd.2016.109%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Reviews%2520Drug%2520Discovery%26atitle%3DTwenty%252Byears%252Bon%25253A%252Bthe%252Bimpact%252Bof%252Bfragments%252Bon%252Bdrug%252Bdiscovery%26aulast%3DErlanson%26aufirst%3DDaniel%2BA.%26date%3D2016%26date%3D2016%26volume%3D15%26issue%3D9%26spage%3D605%26epage%3D619" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ryan  Lillico</span>, <span class="hlFld-ContribAuthor ">Nicholas  Stesco</span>, <span class="hlFld-ContribAuthor ">Tina  Khorshid Amhad</span>, <span class="hlFld-ContribAuthor ">Claudia  Cortes</span>, <span class="hlFld-ContribAuthor ">Mike P  Namaka</span>, <span class="hlFld-ContribAuthor ">Ted M  Lakowski</span>. </span><span class="cited-content_cbyCitation_article-title">Inhibitors of enzymes catalyzing modifications to histone lysine residues: structure, function and activity. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2016,</strong> <em>8 </em>
                                    (8)
                                     , 879-897. <a href="https://doi.org/10.4155/fmc-2016-0021" title="DOI URL">https://doi.org/10.4155/fmc-2016-0021</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc-2016-0021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc-2016-0021%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DInhibitors%252Bof%252Benzymes%252Bcatalyzing%252Bmodifications%252Bto%252Bhistone%252Blysine%252Bresidues%25253A%252Bstructure%25252C%252Bfunction%252Band%252Bactivity%26aulast%3DLillico%26aufirst%3DRyan%26date%3D2016%26volume%3D8%26issue%3D8%26spage%3D879%26epage%3D897" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/medium/jm-2015-015277_0013.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/large/jm-2015-015277_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01527&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/medium/jm-2015-015277_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/large/jm-2015-015277_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. 5-Aminosalicylate docking hits. Poses of the 5-aminosalicylates identified in the initial docking screens. Fragments <b>4</b>, <b>5</b>, and <b>7</b> are predicted to bind the metal in a bivalent fashion through the carboxylate and hydroxyl groups, while the amide proton is predicted to hydrogen-bond with Asp135 of KDM4A.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/large/jm-2015-015277_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01527&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/medium/jm-2015-015277_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/large/jm-2015-015277_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Docking-based fragment linking. (A) Superimposition of the two major 5-aminosalicylate poses (compound <b>4</b>, orange; compound <b>29</b>, magenta). (B) Structures of 5-aminosalicylate fragments used to create hybrid scaffolds. Key interacting groups are highlighted with a box colored to match the fragments. Iron coordinating atoms are marked by asterisk, and possible ring positions are highlighted. (C) Among 12 virtual hybrid scaffolds that were docked (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01527/suppl_file/jm5b01527_si_001.pdf" class="ext-link">Table S3</a>), three had good scores and maintained the hypothesized network of interactions. (D) Overlay of a docking pose of <b>46</b> (orange) and a crystal structure of <b>30</b> (green), a representative hybrid scaffold synthesized in this study.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/large/jm-2015-015277_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01527&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/medium/jm-2015-015277_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/large/jm-2015-015277_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Lineweaver–Burke plots. Plots of 1/[α-KG] vs 1/<i>V</i><sub>i</sub> for a range of concentrations of (A) <b>42</b>, (B) <b>35</b>, and (C) <b>1</b>: ● = 19 μM inhibitor, ■ = 6 μM inhibitor, ◆ = 2 μM inhibitor, ▲ = 0.7 μM inhibitor, ▼ = 0 μM inhibitor.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/large/jm-2015-015277_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01527&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/medium/jm-2015-015277_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/large/jm-2015-015277_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Jumonji family enzymes and hybrid compound selectivity against FIH. (A) Phylogenetic diagram based on the catalytic domains of JmjC proteins, with KDM4C highlighted in red, and enzymes used in the counterscreening experiments in this study highlighted in blue. (B) Selectivity profile obtained using the MALDI assay with 100 μM α-KG against KDM4C (red) and FIH (green) for hybrid scaffold <b>30</b> indicating an IC<sub>50</sub> of 135 and 828 μM, respectively, and (C) its acyl derivative <b>44</b> showing an IC<sub>50</sub> of 63 μM for KDM4C (red) and no inhibition for FIH (green).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/large/jm-2015-015277_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01527&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/medium/jm-2015-015277_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/large/jm-2015-015277_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Crystal structures of hybrid compounds in complex with KDM4A. Seven cocrystal structures were obtained with KDM4A and the hybrid molecules (A) <b>43</b>, (B) <b>30</b>, (C) <b>44</b>, (D) <b>42</b>, (E) <b>35</b>, (F) <b>40</b>, and (G) <b>36</b> to a resolution of 2.39, 2.00, 2.20, 2.15, 2.27, 2.16, and 2.28 Å, respectively. Interacting residues are shown as sticks. (H) Omit map (green) for compound <b>36</b> contoured at 2.5σ. (I) 2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> composite omit map (gray) contoured at 1σ showing residues 5 Å around compound <b>36</b>. (J) As a representative structure, compound <b>36</b> (wheat) is superimposed with the docked 5-aminosalicylate compound <b>4</b> (orange) and the corresponding docked hybrid compound <b>45</b> (green). (K) Hydrogen bond network with compound <b>36</b>. (L) Stacking interactions with compound <b>36</b>; the hydrophobic centers are indicated by a green sphere.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-4/acs.jmedchem.5b01527/20160219/images/large/jm-2015-015277_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01527&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i81">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_51181" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_51181" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 76 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Shi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lan, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mulligan, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whetstine, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cole, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casero, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Y.</span><span> </span><span class="NLM_article-title">Histone demethylation mediated by the nuclear amine oxidase homolog LSD1</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">119</span><span class="NLM_x">, </span> <span class="NLM_fpage">941</span><span class="NLM_x">–</span> <span class="NLM_lpage">953</span><span class="refDoi"> DOI: 10.1016/j.cell.2004.12.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1016%2Fj.cell.2004.12.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=15620353" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BD2MXltlSitg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2004&pages=941-953&author=Y.+Shiauthor=F.+Lanauthor=C.+Matsonauthor=P.+Mulliganauthor=J.+R.+Whetstineauthor=P.+A.+Coleauthor=R.+A.+Caseroauthor=Y.+Shi&title=Histone+demethylation+mediated+by+the+nuclear+amine+oxidase+homolog+LSD1&doi=10.1016%2Fj.cell.2004.12.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Histone demethylation mediated by the nuclear amine oxidase homolog LSD1</span></div><div class="casAuthors">Shi, Yujiang; Lan, Fei; Matson, Caitlin; Mulligan, Peter; Whetstine, Johnathan R.; Cole, Philip A.; Casero, Robert A.; Shi, Yang</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">941-953</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Posttranslational modifications of histone N-terminal tails impact chromatin structure and gene transcription.  While the extent of histone acetylation is detd. by both acetyltransferases and deacetylases, it has been unclear whether histone methylation is also regulated by enzymes with opposing activities.  Here, we provide evidence that LSD1 (KIAA0601), a nuclear homolog of amine oxidases, functions as a histone demethylase and transcriptional corepressor.  LSD1 specifically demethylates histone H3 lysine 4, which is linked to active transcription.  Lysine demethylation occurs via an oxidn. reaction that generates formaldehyde.  Importantly, RNAi inhibition of LSD1 causes an increase in H3 lysine 4 methylation and concomitant derepression of target genes, suggesting that LSD1 represses transcription via histone demethylation.  The results thus identify a histone demethylase conserved from S. pombe to human and reveal dynamic regulation of histone methylation by both histone methylases and demethylases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXHr-8kwGdXrVg90H21EOLACvtfcHk0lgBbwpDbsfRfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXltlSitg%253D%253D&md5=3474569204255e90a832667ba654748b</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2004.12.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2004.12.012%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DY.%26aulast%3DLan%26aufirst%3DF.%26aulast%3DMatson%26aufirst%3DC.%26aulast%3DMulligan%26aufirst%3DP.%26aulast%3DWhetstine%26aufirst%3DJ.%2BR.%26aulast%3DCole%26aufirst%3DP.%2BA.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26aulast%3DShi%26aufirst%3DY.%26atitle%3DHistone%2520demethylation%2520mediated%2520by%2520the%2520nuclear%2520amine%2520oxidase%2520homolog%2520LSD1%26jtitle%3DCell%26date%3D2004%26volume%3D119%26spage%3D941%26epage%3D953%26doi%3D10.1016%2Fj.cell.2004.12.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Tsukada, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Erdjument-Bromage, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Warren, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borchers, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tempst, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span> </span><span class="NLM_article-title">Histone demethylation by a family of JmjC domain-containing proteins</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">439</span><span class="NLM_x">, </span> <span class="NLM_fpage">811</span><span class="NLM_x">–</span> <span class="NLM_lpage">816</span><span class="refDoi"> DOI: 10.1038/nature04433</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1038%2Fnature04433" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=16362057" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BD28XhsVSkuro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=439&publication_year=2006&pages=811-816&author=Y.+Tsukadaauthor=J.+Fangauthor=H.+Erdjument-Bromageauthor=M.+E.+Warrenauthor=C.+H.+Borchersauthor=P.+Tempstauthor=Y.+Zhang&title=Histone+demethylation+by+a+family+of+JmjC+domain-containing+proteins&doi=10.1038%2Fnature04433"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Histone demethylation by a family of JmjC domain-containing proteins</span></div><div class="casAuthors">Tsukada, Yu-ichi; Fang, Jia; Erdjument-Bromage, Hediye; Warren, Maria E.; Borchers, Christoph H.; Tempst, Paul; Zhang, Yi</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">439</span>
        (<span class="NLM_cas:issue">7078</span>),
    <span class="NLM_cas:pages">811-816</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Covalent modification of histones has an important role in regulating chromatin dynamics and transcription.  Whereas most covalent histone modifications are reversible, until recently it was unknown whether Me groups could be actively removed from histones.  Using a biochem. assay coupled with chromatog., we have purified a novel JmjC domain-contg. protein, JHDM1 (JmjC domain-contg. histone demethylase 1), that specifically demethylates histone H3 at lysine 36 (H3-K36).  In the presence of Fe(II) and α-ketoglutarate (αKG), JHDM1 demethylates H3-methyl-K36 and generates formaldehyde and succinate.  Overexpression of JHDM1 reduced the level of dimethyl-H3-K36 (H3K36me2) in vivo.  The demethylase activity of the JmjC domain-contg. proteins is conserved, as a JHDM1 homolog in Saccharomyces cerevisiae also has H3-K36 demethylase activity.  Thus, we identify the JmjC domain as a novel demethylase signature motif and uncover a protein demethylation mechanism that is conserved from yeast to human.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKQSpeu3QBl7Vg90H21EOLACvtfcHk0lgBbwpDbsfRfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhsVSkuro%253D&md5=08f672cc5f81ca88288496f5d8069e3e</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnature04433&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature04433%26sid%3Dliteratum%253Aachs%26aulast%3DTsukada%26aufirst%3DY.%26aulast%3DFang%26aufirst%3DJ.%26aulast%3DErdjument-Bromage%26aufirst%3DH.%26aulast%3DWarren%26aufirst%3DM.%2BE.%26aulast%3DBorchers%26aufirst%3DC.%2BH.%26aulast%3DTempst%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DY.%26atitle%3DHistone%2520demethylation%2520by%2520a%2520family%2520of%2520JmjC%2520domain-containing%2520proteins%26jtitle%3DNature%26date%3D2006%26volume%3D439%26spage%3D811%26epage%3D816%26doi%3D10.1038%2Fnature04433" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Johansson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tumber, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Che, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cain, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nowak, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gileadi, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oppermann, U.</span><span> </span><span class="NLM_article-title">The roles of Jumonji-type oxygenases in human disease</span> <span class="citation_source-journal">Epigenomics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">89</span><span class="NLM_x">–</span> <span class="NLM_lpage">120</span><span class="refDoi"> DOI: 10.2217/epi.13.79</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.2217%2Fepi.13.79" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=24579949" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BC2cXnslClu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=89-120&author=C.+Johanssonauthor=A.+Tumberauthor=K.+Cheauthor=P.+Cainauthor=R.+Nowakauthor=C.+Gileadiauthor=U.+Oppermann&title=The+roles+of+Jumonji-type+oxygenases+in+human+disease&doi=10.2217%2Fepi.13.79"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The roles of Jumonji-type oxygenases in human disease</span></div><div class="casAuthors">Johansson, Catrine; Tumber, Anthony; Che, Ka Hing; Cain, Peter; Nowak, Radoslaw; Gileadi, Carina; Oppermann, Udo</div><div class="citationInfo"><span class="NLM_cas:title">Epigenomics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">89-120</span>CODEN:
                <span class="NLM_cas:coden">EPIGC7</span>;
        ISSN:<span class="NLM_cas:issn">1750-192X</span>.
    
            (<span class="NLM_cas:orgname">Future Medicine Ltd.</span>)
        </div><div class="casAbstract">A review.  The iron- and 2-oxoglutarate-dependent oxygenases constitute a phylogenetically conserved class of enzymes that catalyze hydroxylation reactions in humans by acting on various types of substrates, including metabolic intermediates, amino acid residues in different proteins and various types of nucleic acids.  The discovery of jumonji (Jmj), the founding member of a class of Jmj-type chromatin modifying enzymes and transcriptional regulators, has culminated in the discovery of several branches of histone lysine demethylases, with essential functions in regulating the epigenetic landscape of the chromatin environment.  This work has now been considerably expanded into other aspects of epigenetic biol. and includes the discovery of enzymic steps required for methyl-cytosine demethylation as well as modification of RNA and ribosomal proteins.  This overview aims to summarize the current knowledge on the human Jmj-type enzymes and their involvement in human pathol. processes, including development, cancer, inflammation and metabolic diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2w6QxrC2kT7Vg90H21EOLACvtfcHk0lhBzhJu2p7VEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXnslClu70%253D&md5=a9a897d70db8a17dc1ae424ce4c40e9f</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.2217%2Fepi.13.79&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2217%252Fepi.13.79%26sid%3Dliteratum%253Aachs%26aulast%3DJohansson%26aufirst%3DC.%26aulast%3DTumber%26aufirst%3DA.%26aulast%3DChe%26aufirst%3DK.%26aulast%3DCain%26aufirst%3DP.%26aulast%3DNowak%26aufirst%3DR.%26aulast%3DGileadi%26aufirst%3DC.%26aulast%3DOppermann%26aufirst%3DU.%26atitle%3DThe%2520roles%2520of%2520Jumonji-type%2520oxygenases%2520in%2520human%2520disease%26jtitle%3DEpigenomics%26date%3D2014%26volume%3D6%26spage%3D89%26epage%3D120%26doi%3D10.2217%2Fepi.13.79" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Labbe, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holowatyj, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, Z. Q.</span><span> </span><span class="NLM_article-title">Histone lysine demethylase (KDM) subfamily 4: structures, functions and therapeutic potential</span> <span class="citation_source-journal">Am. J. Transl. Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">15</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=24349617" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A280%3ADC%252BC2c3pslKitw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2013&pages=1-15&author=R.+M.+Labbeauthor=A.+Holowatyjauthor=Z.+Q.+Yang&title=Histone+lysine+demethylase+%28KDM%29+subfamily+4%3A+structures%2C+functions+and+therapeutic+potential"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Histone lysine demethylase (KDM) subfamily 4: structures, functions and therapeutic potential</span></div><div class="casAuthors">Labbe Roselyne M; Holowatyj Andreana; Yang Zeng-Quan</div><div class="citationInfo"><span class="NLM_cas:title">American journal of translational research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-15</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">KDM4 histone demethylases catalyze the removal of methyl marks from histone lysine residues to epigenetically regulate chromatin structure and gene expression.  KDM4 expression is tightly regulated to insure proper function in diverse biological processes, such as cellular differentiation.  Mounting evidence has shown that disrupting KDM4 expression is implicated in the establishment and progression of multiple diseases including cancer.  In particular, genomic regions encoding the KDM4A, B and C genes are often amplified, disrupting normal cellular proliferation.  Furthermore, KDM4 demethylases are promising druggable targets.  In this review, we highlight the latest advances in characterizing the structures and regulatory mechanisms of KDM4 proteins, as well as our current understanding of their alterations and roles in tumorigenesis.  We also review the reported KDM4 inhibitors and discuss their potential as therapeutic agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRlRv7E2abvvjW1zwg2yUK5fW6udTcc2ebX88DNGhS377ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2c3pslKitw%253D%253D&md5=60009f9b59e7110ce4ff57998493a128</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLabbe%26aufirst%3DR.%2BM.%26aulast%3DHolowatyj%26aufirst%3DA.%26aulast%3DYang%26aufirst%3DZ.%2BQ.%26atitle%3DHistone%2520lysine%2520demethylase%2520%2528KDM%2529%2520subfamily%25204%253A%2520structures%252C%2520functions%2520and%2520therapeutic%2520potential%26jtitle%3DAm.%2520J.%2520Transl.%2520Res.%26date%3D2013%26volume%3D6%26spage%3D1%26epage%3D15" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Jenuwein, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allis, C. D.</span><span> </span><span class="NLM_article-title">Translating the histone code</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">293</span><span class="NLM_x">, </span> <span class="NLM_fpage">1074</span><span class="NLM_x">–</span> <span class="NLM_lpage">1080</span><span class="refDoi"> DOI: 10.1126/science.1063127</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1126%2Fscience.1063127" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=11498575" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BD3MXmtVWltro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=293&publication_year=2001&pages=1074-1080&author=T.+Jenuweinauthor=C.+D.+Allis&title=Translating+the+histone+code&doi=10.1126%2Fscience.1063127"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Translating the histone code</span></div><div class="casAuthors">Jenuwein, Thomas; Allis, C. David</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">293</span>
        (<span class="NLM_cas:issue">5532</span>),
    <span class="NLM_cas:pages">1074-1080</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The review.  Chromatin, the physiol. template of all eukaryotic genetic information, is subject to a diverse array of posttranslational modifications that largely impinge on histone amino termini, thereby regulating access to the underlying DNA.  Distinct histone amino-terminal modifications can generate synergistic or antagonistic interaction affinities for chromatin-assocd. proteins, which in turn dictate dynamic transitions between transcriptionally active or transcriptionally silent chromatin states.  The combinatorial nature of histone amino-terminal modifications thus reveals a "histone code" that considerably extends the information potential of the genetic code.  We propose that this epigenetic marking system represents a fundamental regulatory mechanism that has an impact on most, if not all; chromatin-templated processes, with far-reaching consequences for cell fate decisions and both normal and pathol. development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUFyHNepKcUbVg90H21EOLACvtfcHk0lhBzhJu2p7VEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXmtVWltro%253D&md5=021dcd98a5e5bf2ed7eedef1897b2269</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1126%2Fscience.1063127&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1063127%26sid%3Dliteratum%253Aachs%26aulast%3DJenuwein%26aufirst%3DT.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26atitle%3DTranslating%2520the%2520histone%2520code%26jtitle%3DScience%26date%3D2001%26volume%3D293%26spage%3D1074%26epage%3D1080%26doi%3D10.1126%2Fscience.1063127" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Katoh, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Katoh, M.</span><span> </span><span class="NLM_article-title">Identification and characterization of JMJD2 family genes in silico</span> <span class="citation_source-journal">Int. J. Oncol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">1623</span><span class="NLM_x">–</span> <span class="NLM_lpage">1628</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=15138608" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BD2cXkvFaisbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2004&pages=1623-1628&author=M.+Katohauthor=M.+Katoh&title=Identification+and+characterization+of+JMJD2+family+genes+in+silico"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and characterization of JMJD2 family genes in silico</span></div><div class="casAuthors">Katoh, Masuko; Katoh, Masaru</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Oncology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1623-1628</span>CODEN:
                <span class="NLM_cas:coden">IJONES</span>;
        ISSN:<span class="NLM_cas:issn">1019-6439</span>.
    
            (<span class="NLM_cas:orgname">International Journal of Oncology</span>)
        </div><div class="casAbstract">JMJD1 family genes, including JMJD1 (TSGA), TRIP8, and C5orf7 (5qNCA), encode nuclear proteins with Jumonji C (JmjC), TRI8H1 and TRI8H2 domains.  Here, we identified JMJD2 family genes.  Human genes corresponding to KIAA0677, KIAA0876, KIAA0780 and FLJ10251 cDNAs were designated JMJD2A, JMJD2B, JMJD2C, and JMJD2D, resp.  In addn., JMJD2D homologous genes within human genome sequences AP002383.3 and AP001264.4 were designated JMJD2E and JMJD2F, resp.  JMJD2A gene was mapped to human chromosome 1p34.1, JMJD2B gene to 19p13.3, and JMJD2C gene to 9p24.1.  JMJD2D, JMJD2E and JMJD2F genes were clustered at human chromosome 11q21.  Coding region of JMJD2A, JMJD2B, JMJD2C genes was located on multiple exons, while that of JMJD2D, JMJD2E and JMJD2F genes was located on a single exon.  These facts strongly indicate that JMJD2D, JMJD2E and JMJD2F genes are locally triplicated retrotransposed elements derived from JMJD2 family gene.  JD2H domain with C2HC2HC2- and C5HC2-type Cys (His) clusters was identified as the region conserved among JMJD2A (1064 aa), JMJD2B (1096 aa), and JMJD2C (1056 aa) proteins.  JMJD2A, JMJD2B and JMJD2C consist of JmjN, JmjC, JD2H, and two TUDOR domains, while JMJD2D (523 aa), JMJD2E (506 aa) and JMJD2F (638 aa) consist of JmjN and JmjC domains.  JMJD2 family proteins were classified into one group with JD2H and TUDOR domains and another group without JD2H or TUDOR domains.  Because JMJD2C gene (also known as GASC1 gene) is amplified in esophageal squamous cell carcinoma (ESCC), JMJD2 family genes are cancer-assocd. genes.  This is the first report on identification and characterization of human JMJD2 gene family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2QrOZpXEa9rVg90H21EOLACvtfcHk0lhKpWdk4pNGkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXkvFaisbo%253D&md5=b6ab9ea6d3aebb1c242870726103a449</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKatoh%26aufirst%3DM.%26aulast%3DKatoh%26aufirst%3DM.%26atitle%3DIdentification%2520and%2520characterization%2520of%2520JMJD2%2520family%2520genes%2520in%2520silico%26jtitle%3DInt.%2520J.%2520Oncol.%26date%3D2004%26volume%3D24%26spage%3D1623%26epage%3D1628" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Whetstine, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nottke, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lan, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huarte, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smolikov, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spooner, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Colaiacovo, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Y.</span><span> </span><span class="NLM_article-title">Reversal of histone lysine trimethylation by the JMJD2 family of histone demethylases</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">125</span><span class="NLM_x">, </span> <span class="NLM_fpage">467</span><span class="NLM_x">–</span> <span class="NLM_lpage">481</span><span class="refDoi"> DOI: 10.1016/j.cell.2006.03.028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1016%2Fj.cell.2006.03.028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=16603238" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BD28XkslCjtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2006&pages=467-481&author=J.+R.+Whetstineauthor=A.+Nottkeauthor=F.+Lanauthor=M.+Huarteauthor=S.+Smolikovauthor=Z.+Chenauthor=E.+Spoonerauthor=E.+Liauthor=G.+Zhangauthor=M.+Colaiacovoauthor=Y.+Shi&title=Reversal+of+histone+lysine+trimethylation+by+the+JMJD2+family+of+histone+demethylases&doi=10.1016%2Fj.cell.2006.03.028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Reversal of histone lysine trimethylation by the JMJD2 family of histone demethylases</span></div><div class="casAuthors">Whetstine, Johnathan R.; Nottke, Amanda; Lan, Fei; Huarte, Maite; Smolikov, Sarit; Chen, Zhongzhou; Spooner, Eric; Li, En; Zhang, Gongyi; Colaiacovo, Monica; Shi, Yang</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">125</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">467-481</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Histone methylation regulates chromatin structure, transcription, and epigenetic state of the cell.  Histone methylation is dynamically regulated by histone methylases and demethylases such as LSD1 and JHDM1, which mediate demethylation of di- and monomethylated histones.  It has been unclear whether demethylases exist that reverse lysine trimethylation.  We show the JmjC domain-contg. protein JMJD2A reversed trimethylated H3-K9/K36 to di- but not mono- or unmethylated products.  Overexpression of JMJD2A but not a catalytically inactive mutant reduced H3-K9/K36 trimethylation levels in cultured cells.  In contrast, RNAi depletion of the C. elegans JMJD2A homolog resulted in an increase in general H3-K9Me3 and localized H3-K36Me3 levels on meiotic chromosomes and triggered p53-dependent germline apoptosis.  Addnl., other human JMJD2 subfamily members also functioned as trimethylation-specific demethylases, converting H3-K9Me3 to H3-K9Me2 and H3-K9Me1, resp.  Our finding that this family of demethylases generates different methylated states at the same lysine residue provides a mechanism for fine-tuning histone methylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopUntpFohMorVg90H21EOLACvtfcHk0lhKpWdk4pNGkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XkslCjtLo%253D&md5=e73fcf5b9f5f3127bae7c9f5be8a269b</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2006.03.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2006.03.028%26sid%3Dliteratum%253Aachs%26aulast%3DWhetstine%26aufirst%3DJ.%2BR.%26aulast%3DNottke%26aufirst%3DA.%26aulast%3DLan%26aufirst%3DF.%26aulast%3DHuarte%26aufirst%3DM.%26aulast%3DSmolikov%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DSpooner%26aufirst%3DE.%26aulast%3DLi%26aufirst%3DE.%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DColaiacovo%26aufirst%3DM.%26aulast%3DShi%26aufirst%3DY.%26atitle%3DReversal%2520of%2520histone%2520lysine%2520trimethylation%2520by%2520the%2520JMJD2%2520family%2520of%2520histone%2520demethylases%26jtitle%3DCell%26date%3D2006%26volume%3D125%26spage%3D467%26epage%3D481%26doi%3D10.1016%2Fj.cell.2006.03.028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Cloos, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Christensen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Agger, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maiolica, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rappsilber, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Antal, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hansen, K. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Helin, K.</span><span> </span><span class="NLM_article-title">The putative oncogene GASC1 demethylates tri- and dimethylated lysine 9 on histone H3</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">442</span><span class="NLM_x">, </span> <span class="NLM_fpage">307</span><span class="NLM_x">–</span> <span class="NLM_lpage">311</span><span class="refDoi"> DOI: 10.1038/nature04837</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1038%2Fnature04837" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=442&publication_year=2006&pages=307-311&author=P.+A.+Cloosauthor=J.+Christensenauthor=K.+Aggerauthor=A.+Maiolicaauthor=J.+Rappsilberauthor=T.+Antalauthor=K.+H.+Hansenauthor=K.+Helin&title=The+putative+oncogene+GASC1+demethylates+tri-+and+dimethylated+lysine+9+on+histone+H3&doi=10.1038%2Fnature04837"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnature04837&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature04837%26sid%3Dliteratum%253Aachs%26aulast%3DCloos%26aufirst%3DP.%2BA.%26aulast%3DChristensen%26aufirst%3DJ.%26aulast%3DAgger%26aufirst%3DK.%26aulast%3DMaiolica%26aufirst%3DA.%26aulast%3DRappsilber%26aufirst%3DJ.%26aulast%3DAntal%26aufirst%3DT.%26aulast%3DHansen%26aufirst%3DK.%2BH.%26aulast%3DHelin%26aufirst%3DK.%26atitle%3DThe%2520putative%2520oncogene%2520GASC1%2520demethylates%2520tri-%2520and%2520dimethylated%2520lysine%25209%2520on%2520histone%2520H3%26jtitle%3DNature%26date%3D2006%26volume%3D442%26spage%3D307%26epage%3D311%26doi%3D10.1038%2Fnature04837" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Klose, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamane, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bae, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Erdjument-Bromage, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tempst, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span> </span><span class="NLM_article-title">The transcriptional repressor JHDM3A demethylates trimethyl histone H3 lysine 9 and lysine 36</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">442</span><span class="NLM_x">, </span> <span class="NLM_fpage">312</span><span class="NLM_x">–</span> <span class="NLM_lpage">316</span><span class="refDoi"> DOI: 10.1038/nature04853</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1038%2Fnature04853" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=16732292" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BD28XmvF2jsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=442&publication_year=2006&pages=312-316&author=R.+J.+Kloseauthor=K.+Yamaneauthor=Y.+Baeauthor=D.+Zhangauthor=H.+Erdjument-Bromageauthor=P.+Tempstauthor=J.+Wongauthor=Y.+Zhang&title=The+transcriptional+repressor+JHDM3A+demethylates+trimethyl+histone+H3+lysine+9+and+lysine+36&doi=10.1038%2Fnature04853"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">The transcriptional repressor JHDM3A demethylates trimethyl histone H3 lysine 9 and lysine 36</span></div><div class="casAuthors">Klose, Robert J.; Yamane, Kenichi; Bae, Yangjin; Zhang, Dianzheng; Erdjument-Bromage, Hediye; Tempst, Paul; Wong, Jiemin; Zhang, Yi</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">442</span>
        (<span class="NLM_cas:issue">7100</span>),
    <span class="NLM_cas:pages">312-316</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Post-translational modification of chromatin has profound effects on many biol. processes including transcriptional regulation, heterochromatin organization, and X-chromosome inactivation.  Recent studies indicate that methylation on specific histone lysine (K) residues participates in many of these processes.  Lysine methylation occurs in three distinct states, having either one (me1), two (me2) or three (me3) Me groups attached to the amine group of the lysine side chain.  These differences in modification state have an important role in defining how methylated chromatin is recognized and interpreted.  Until recently, histone lysine methylation was considered a stable modification, but the identification of histone demethylase enzymes has demonstrated the reversibility of this epigenetic mark.  So far, all characterized histone demethylases show enzymic activity towards lysine residues modified in the me1 or me2 state, leaving open the possibility that me3 constitutes an irreversible modification.  Here we demonstrate that JHDM3A (jumonji C (JmjC)-domain-contg. histone demethylase 3A; also known as JMJD2A) is capable of removing the me3 group from modified H3 lysine 9 (H3K9) and H3 lysine 36 (H3K36).  Overexpression of JHDM3A abrogates recruitment of HP1 (heterochromatin protein 1) to heterochromatin, indicating a role for JHDM3A in antagonizing methylated H3K9 nucleated events.  SiRNA-mediated knockdown of JHDM3A leads to increased levels of H3K9 methylation and upregulation of a JHDM3A target gene, ASCL2, indicating that JHDM3A may function in euchromatin to remove histone methylation marks that are assocd. with active transcription.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7-qskfqfg47Vg90H21EOLACvtfcHk0lhKpWdk4pNGkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XmvF2jsr0%253D&md5=d9ab4dd44f1ddcc3bcf3472fd540dd60</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnature04853&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature04853%26sid%3Dliteratum%253Aachs%26aulast%3DKlose%26aufirst%3DR.%2BJ.%26aulast%3DYamane%26aufirst%3DK.%26aulast%3DBae%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DErdjument-Bromage%26aufirst%3DH.%26aulast%3DTempst%26aufirst%3DP.%26aulast%3DWong%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DY.%26atitle%3DThe%2520transcriptional%2520repressor%2520JHDM3A%2520demethylates%2520trimethyl%2520histone%2520H3%2520lysine%25209%2520and%2520lysine%252036%26jtitle%3DNature%26date%3D2006%26volume%3D442%26spage%3D312%26epage%3D316%26doi%3D10.1038%2Fnature04853" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Fodor, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kubicek, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yonezawa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Sullivan, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sengupta, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perez-Burgos, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Opravil, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mechtler, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schotta, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenuwein, T.</span><span> </span><span class="NLM_article-title">Jmjd2b antagonizes H3K9 trimethylation at pericentric heterochromatin in mammalian cells</span> <span class="citation_source-journal">Genes Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">1557</span><span class="NLM_x">–</span> <span class="NLM_lpage">1562</span><span class="refDoi"> DOI: 10.1101/gad.388206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1101%2Fgad.388206" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2006&pages=1557-1562&author=B.+D.+Fodorauthor=S.+Kubicekauthor=M.+Yonezawaauthor=R.+J.+O%E2%80%99Sullivanauthor=R.+Senguptaauthor=L.+Perez-Burgosauthor=S.+Opravilauthor=K.+Mechtlerauthor=G.+Schottaauthor=T.+Jenuwein&title=Jmjd2b+antagonizes+H3K9+trimethylation+at+pericentric+heterochromatin+in+mammalian+cells&doi=10.1101%2Fgad.388206"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1101%2Fgad.388206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.388206%26sid%3Dliteratum%253Aachs%26aulast%3DFodor%26aufirst%3DB.%2BD.%26aulast%3DKubicek%26aufirst%3DS.%26aulast%3DYonezawa%26aufirst%3DM.%26aulast%3DO%25E2%2580%2599Sullivan%26aufirst%3DR.%2BJ.%26aulast%3DSengupta%26aufirst%3DR.%26aulast%3DPerez-Burgos%26aufirst%3DL.%26aulast%3DOpravil%26aufirst%3DS.%26aulast%3DMechtler%26aufirst%3DK.%26aulast%3DSchotta%26aufirst%3DG.%26aulast%3DJenuwein%26aufirst%3DT.%26atitle%3DJmjd2b%2520antagonizes%2520H3K9%2520trimethylation%2520at%2520pericentric%2520heterochromatin%2520in%2520mammalian%2520cells%26jtitle%3DGenes%2520Dev.%26date%3D2006%26volume%3D20%26spage%3D1557%26epage%3D1562%26doi%3D10.1101%2Fgad.388206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Shin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janknecht, R.</span><span> </span><span class="NLM_article-title">Activation of androgen receptor by histone demethylases JMJD2A and JMJD2D</span> <span class="citation_source-journal">Biochem. Biophys. Res. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">359</span><span class="NLM_x">, </span> <span class="NLM_fpage">742</span><span class="NLM_x">–</span> <span class="NLM_lpage">746</span><span class="refDoi"> DOI: 10.1016/j.bbrc.2007.05.179</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1016%2Fj.bbrc.2007.05.179" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=17555712" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BD2sXmsFykurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=359&publication_year=2007&pages=742-746&author=S.+Shinauthor=R.+Janknecht&title=Activation+of+androgen+receptor+by+histone+demethylases+JMJD2A+and+JMJD2D&doi=10.1016%2Fj.bbrc.2007.05.179"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Activation of androgen receptor by histone demethylases JMJD2A and JMJD2D</span></div><div class="casAuthors">Shin, Sook; Janknecht, Ralf</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">359</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">742-746</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">The androgen receptor (AR) is a transcription factor that is pivotal for the development of prostate cancer.  Here, we have identified two related histone demethylases, JMJD2A and JMJD2D, which form complexes with ligand-bound AR.  We found that AR interacts through its ligand binding domain with JMJD2A and JMJD2D.  On the other hand, JMJD2A utilizes its catalytic domain or C-terminus to bind to AR, and JMJD2D does so via its C-terminus.  Further, overexpression of JMJD2A or D stimulates AR function and this is dependent on JMJD2 catalytic activity.  Conversely, downregulation of JMJD2A, which is often overexpressed in prostate tumors, reduces basal transcription of the AR target gene, prostate-specific antigen, in LNCaP prostate cancer cells.  Altogether, our data have identified a novel class of AR coactivators, whose (over)expression in prostate tumors could contribute to the constitutive activation of AR and thus to androgen-depletion independency of advanced prostate cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkS2bbLidpDbVg90H21EOLACvtfcHk0liliwE0yjYqwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXmsFykurk%253D&md5=56e0c5770876f4ce54e46fe940425b7a</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2007.05.179&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2007.05.179%26sid%3Dliteratum%253Aachs%26aulast%3DShin%26aufirst%3DS.%26aulast%3DJanknecht%26aufirst%3DR.%26atitle%3DActivation%2520of%2520androgen%2520receptor%2520by%2520histone%2520demethylases%2520JMJD2A%2520and%2520JMJD2D%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2007%26volume%3D359%26spage%3D742%26epage%3D746%26doi%3D10.1016%2Fj.bbrc.2007.05.179" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Trojer, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yonezawa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmidt, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenuwein, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reinberg, D.</span><span> </span><span class="NLM_article-title">Dynamic Histone H1 Isotype 4 Methylation and Demethylation by Histone Lysine Methyltransferase G9a/KMT1C and the Jumonji Domain-containing JMJD2/KDM4 Proteins</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">284</span><span class="NLM_x">, </span> <span class="NLM_fpage">8395</span><span class="NLM_x">–</span> <span class="NLM_lpage">8405</span><span class="refDoi"> DOI: 10.1074/jbc.M807818200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1074%2Fjbc.M807818200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=284&publication_year=2009&pages=8395-8405&author=P.+Trojerauthor=J.+Zhangauthor=M.+Yonezawaauthor=A.+Schmidtauthor=H.+Zhengauthor=T.+Jenuweinauthor=D.+Reinberg&title=Dynamic+Histone+H1+Isotype+4+Methylation+and+Demethylation+by+Histone+Lysine+Methyltransferase+G9a%2FKMT1C+and+the+Jumonji+Domain-containing+JMJD2%2FKDM4+Proteins&doi=10.1074%2Fjbc.M807818200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M807818200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M807818200%26sid%3Dliteratum%253Aachs%26aulast%3DTrojer%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DYonezawa%26aufirst%3DM.%26aulast%3DSchmidt%26aufirst%3DA.%26aulast%3DZheng%26aufirst%3DH.%26aulast%3DJenuwein%26aufirst%3DT.%26aulast%3DReinberg%26aufirst%3DD.%26atitle%3DDynamic%2520Histone%2520H1%2520Isotype%25204%2520Methylation%2520and%2520Demethylation%2520by%2520Histone%2520Lysine%2520Methyltransferase%2520G9a%252FKMT1C%2520and%2520the%2520Jumonji%2520Domain-containing%2520JMJD2%252FKDM4%2520Proteins%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2009%26volume%3D284%26spage%3D8395%26epage%3D8405%26doi%3D10.1074%2Fjbc.M807818200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Yang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohgi, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grinstein, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dorrestein, P. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosenfeld, M. G.</span><span> </span><span class="NLM_article-title">ncRNA- and Pc2 methylation-dependent gene relocation between nuclear structures mediates gene activation programs</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">147</span><span class="NLM_x">, </span> <span class="NLM_fpage">773</span><span class="NLM_x">–</span> <span class="NLM_lpage">788</span><span class="refDoi"> DOI: 10.1016/j.cell.2011.08.054</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1016%2Fj.cell.2011.08.054" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=147&publication_year=2011&pages=773-788&author=L.+Yangauthor=C.+Linauthor=W.+Liuauthor=J.+Zhangauthor=K.+A.+Ohgiauthor=J.+D.+Grinsteinauthor=P.+C.+Dorresteinauthor=M.+G.+Rosenfeld&title=ncRNA-+and+Pc2+methylation-dependent+gene+relocation+between+nuclear+structures+mediates+gene+activation+programs&doi=10.1016%2Fj.cell.2011.08.054"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2011.08.054&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2011.08.054%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DL.%26aulast%3DLin%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DOhgi%26aufirst%3DK.%2BA.%26aulast%3DGrinstein%26aufirst%3DJ.%2BD.%26aulast%3DDorrestein%26aufirst%3DP.%2BC.%26aulast%3DRosenfeld%26aufirst%3DM.%2BG.%26atitle%3DncRNA-%2520and%2520Pc2%2520methylation-dependent%2520gene%2520relocation%2520between%2520nuclear%2520structures%2520mediates%2520gene%2520activation%2520programs%26jtitle%3DCell%26date%3D2011%26volume%3D147%26spage%3D773%26epage%3D788%26doi%3D10.1016%2Fj.cell.2011.08.054" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Chu, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hsu, K. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, H. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, L. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hung, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kung, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, W. C.</span><span> </span><span class="NLM_article-title">KDM4B as a target for prostate cancer: structural analysis and selective inhibition by a novel inhibitor</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">5975</span><span class="NLM_x">–</span> <span class="NLM_lpage">5985</span><span class="refDoi"> DOI: 10.1021/jm500249n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm500249n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVKnsLnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=5975-5985&author=C.+H.+Chuauthor=L.+Y.+Wangauthor=K.+C.+Hsuauthor=C.+C.+Chenauthor=H.+H.+Chengauthor=S.+M.+Wangauthor=C.+M.+Wuauthor=T.+J.+Chenauthor=L.+T.+Liauthor=R.+Liuauthor=C.+L.+Hungauthor=J.+M.+Yangauthor=H.+J.+Kungauthor=W.+C.+Wang&title=KDM4B+as+a+target+for+prostate+cancer%3A+structural+analysis+and+selective+inhibition+by+a+novel+inhibitor&doi=10.1021%2Fjm500249n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">KDM4B as a Target for Prostate Cancer: Structural Analysis and Selective Inhibition by a Novel Inhibitor</span></div><div class="casAuthors">Chu, Chia-Han; Wang, Ling-Yu; Hsu, Kai-Cheng; Chen, Chung-Chin; Cheng, Hsing-Hung; Wang, Szu-Min; Wu, Chien-Ming; Chen, Tsan-Jan; Li, Ling-Ting; Liu, Ruiwu; Hung, Chiu-Lien; Yang, Jing-Moon; Kung, Hsing-Jien; Wang, Wen-Ching</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">5975-5985</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The KDM4/JMJD2 Jumonji C-contg. histone lysine demethylases (KDM4A-KDM4D), which selectively remove the Me group(s) from tri/dimethylated lysine 9/36 of H3, modulate transcriptional activation and genome stability.  The overexpression of KDM4A/KDM4B in prostate cancer and their assocn. with androgen receptor suggest that KDM4A/KDM4B are potential progression factors for prostate cancer.  Here, we report the crystal structure of the KDM4B·pyridine 2,4-dicarboxylic acid·H3K9me3 ternary complex, revealing the core active-site region and a selective K9/K36 site.  A selective KDM4A/KDM4B inhibitor, 4, that occupies three subsites in the binding pocket is identified by virtual screening.  Pharmacol. and genetic inhibition of KDM4A/KDM4B significantly blocks the viability of cultured prostate cancer cells, which is accompanied by increased H3K9me3 staining and transcriptional silencing of growth-related genes.  Significantly, a substantial portion of differentially expressed genes are AR-responsive, consistent with the roles of KDM4s as crit. AR activators.  Our results point to KDM4 as a useful therapeutic target and identify a new inhibitor scaffold.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoR0KwURfrUnbVg90H21EOLACvtfcHk0liliwE0yjYqwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVKnsLnO&md5=789ecb59ed2e8d965ef220d92c9a2ce2</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fjm500249n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm500249n%26sid%3Dliteratum%253Aachs%26aulast%3DChu%26aufirst%3DC.%2BH.%26aulast%3DWang%26aufirst%3DL.%2BY.%26aulast%3DHsu%26aufirst%3DK.%2BC.%26aulast%3DChen%26aufirst%3DC.%2BC.%26aulast%3DCheng%26aufirst%3DH.%2BH.%26aulast%3DWang%26aufirst%3DS.%2BM.%26aulast%3DWu%26aufirst%3DC.%2BM.%26aulast%3DChen%26aufirst%3DT.%2BJ.%26aulast%3DLi%26aufirst%3DL.%2BT.%26aulast%3DLiu%26aufirst%3DR.%26aulast%3DHung%26aufirst%3DC.%2BL.%26aulast%3DYang%26aufirst%3DJ.%2BM.%26aulast%3DKung%26aufirst%3DH.%2BJ.%26aulast%3DWang%26aufirst%3DW.%2BC.%26atitle%3DKDM4B%2520as%2520a%2520target%2520for%2520prostate%2520cancer%253A%2520structural%2520analysis%2520and%2520selective%2520inhibition%2520by%2520a%2520novel%2520inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D5975%26epage%3D5985%26doi%3D10.1021%2Fjm500249n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ji, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yao, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, J.</span><span> </span><span class="NLM_article-title">An epigenetic role for PRL-3 as a regulator of H3K9 methylation in colorectal cancer</span> <span class="citation_source-journal">Gut</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">62</span><span class="NLM_x">, </span> <span class="NLM_fpage">571</span><span class="NLM_x">–</span> <span class="NLM_lpage">581</span><span class="refDoi"> DOI: 10.1136/gutjnl-2011-301059</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1136%2Fgutjnl-2011-301059" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2013&pages=571-581&author=Y.+Liuauthor=P.+Zhengauthor=Y.+Liuauthor=T.+Jiauthor=X.+Liuauthor=S.+Yaoauthor=X.+Chengauthor=Y.+Liauthor=L.+Chenauthor=Z.+Xiaoauthor=J.+Zhouauthor=J.+Li&title=An+epigenetic+role+for+PRL-3+as+a+regulator+of+H3K9+methylation+in+colorectal+cancer&doi=10.1136%2Fgutjnl-2011-301059"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1136%2Fgutjnl-2011-301059&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1136%252Fgutjnl-2011-301059%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DZheng%26aufirst%3DP.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DJi%26aufirst%3DT.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DYao%26aufirst%3DS.%26aulast%3DCheng%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DXiao%26aufirst%3DZ.%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DJ.%26atitle%3DAn%2520epigenetic%2520role%2520for%2520PRL-3%2520as%2520a%2520regulator%2520of%2520H3K9%2520methylation%2520in%2520colorectal%2520cancer%26jtitle%3DGut%26date%3D2013%26volume%3D62%26spage%3D571%26epage%3D581%26doi%3D10.1136%2Fgutjnl-2011-301059" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Yang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jubb, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pike, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buffa, F. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Turley, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baban, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leek, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gatter, K. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ragoussis, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harris, A. L.</span><span> </span><span class="NLM_article-title">The histone demethylase JMJD2B is regulated by estrogen receptor alpha and hypoxia, and is a key mediator of estrogen induced growth</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">70</span><span class="NLM_x">, </span> <span class="NLM_fpage">6456</span><span class="NLM_x">–</span> <span class="NLM_lpage">6466</span><span class="refDoi"> DOI: 10.1158/0008-5472.CAN-10-0413</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1158%2F0008-5472.CAN-10-0413" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=2010&pages=6456-6466&author=J.+Yangauthor=A.+M.+Jubbauthor=L.+Pikeauthor=F.+M.+Buffaauthor=H.+Turleyauthor=D.+Babanauthor=R.+Leekauthor=K.+C.+Gatterauthor=J.+Ragoussisauthor=A.+L.+Harris&title=The+histone+demethylase+JMJD2B+is+regulated+by+estrogen+receptor+alpha+and+hypoxia%2C+and+is+a+key+mediator+of+estrogen+induced+growth&doi=10.1158%2F0008-5472.CAN-10-0413"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-10-0413&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-10-0413%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DJubb%26aufirst%3DA.%2BM.%26aulast%3DPike%26aufirst%3DL.%26aulast%3DBuffa%26aufirst%3DF.%2BM.%26aulast%3DTurley%26aufirst%3DH.%26aulast%3DBaban%26aufirst%3DD.%26aulast%3DLeek%26aufirst%3DR.%26aulast%3DGatter%26aufirst%3DK.%2BC.%26aulast%3DRagoussis%26aufirst%3DJ.%26aulast%3DHarris%26aufirst%3DA.%2BL.%26atitle%3DThe%2520histone%2520demethylase%2520JMJD2B%2520is%2520regulated%2520by%2520estrogen%2520receptor%2520alpha%2520and%2520hypoxia%252C%2520and%2520is%2520a%2520key%2520mediator%2520of%2520estrogen%2520induced%2520growth%26jtitle%3DCancer%2520Res.%26date%3D2010%26volume%3D70%26spage%3D6456%26epage%3D6466%26doi%3D10.1158%2F0008-5472.CAN-10-0413" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Walport, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hopkinson, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span> </span><span class="NLM_article-title">Mechanisms of human histone and nucleic acid demethylases</span> <span class="citation_source-journal">Curr. Opin. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">525</span><span class="NLM_x">–</span> <span class="NLM_lpage">534</span><span class="refDoi"> DOI: 10.1016/j.cbpa.2012.09.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1016%2Fj.cbpa.2012.09.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=23063108" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsV2qtL%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2012&pages=525-534&author=L.+J.+Walportauthor=R.+J.+Hopkinsonauthor=C.+J.+Schofield&title=Mechanisms+of+human+histone+and+nucleic+acid+demethylases&doi=10.1016%2Fj.cbpa.2012.09.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms of human histone and nucleic acid demethylases</span></div><div class="casAuthors">Walport, Louise J.; Hopkinson, Richard J.; Schofield, Christopher J.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">5-6</span>),
    <span class="NLM_cas:pages">525-534</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The discovery that protein and nucleic acid demethylation is common opens up the possibility of 'methylation cycles' of functional importance, including in the regulation of gene expression.  The mechanisms of known demethylases can be broadly divided into those involving nucleophilic catalysis and those involving oxidative catalysis.  The latter group appear more common; they produce formaldehyde as a co-product.  Nucleophilic demethylases include those proceeding via irreversible S-methylation and Me esterases.  In addn. to the direct reversal of methylation, demethylation can occur concurrent with loss of other groups, such as in methylarginine hydrolysis, oxidn. of Nε-methyllysine to allysine, and indirectly, for example via base-excision repair.  Here, the authors discuss chem. viable mechanisms for biol. demethylation and summarize mechanistic knowledge of the major known families of demethylases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9kS9XUCbZTrVg90H21EOLACvtfcHk0lgXS5z83JNiJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsV2qtL%252FI&md5=0e4c08d79aaca4ee74de3f222757cd9a</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2012.09.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2012.09.015%26sid%3Dliteratum%253Aachs%26aulast%3DWalport%26aufirst%3DL.%2BJ.%26aulast%3DHopkinson%26aufirst%3DR.%2BJ.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26atitle%3DMechanisms%2520of%2520human%2520histone%2520and%2520nucleic%2520acid%2520demethylases%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2012%26volume%3D16%26spage%3D525%26epage%3D534%26doi%3D10.1016%2Fj.cbpa.2012.09.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">McDonough, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loenarz, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chowdhury, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clifton, I. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span> </span><span class="NLM_article-title">Structural studies on human 2-oxoglutarate dependent oxygenases</span> <span class="citation_source-journal">Curr. Opin. Struct. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">659</span><span class="NLM_x">–</span> <span class="NLM_lpage">672</span><span class="refDoi"> DOI: 10.1016/j.sbi.2010.08.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1016%2Fj.sbi.2010.08.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=20888218" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsV2jsrfI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2010&pages=659-672&author=M.+A.+McDonoughauthor=C.+Loenarzauthor=R.+Chowdhuryauthor=I.+J.+Cliftonauthor=C.+J.+Schofield&title=Structural+studies+on+human+2-oxoglutarate+dependent+oxygenases&doi=10.1016%2Fj.sbi.2010.08.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Structural studies on human 2-oxoglutarate dependent oxygenases</span></div><div class="casAuthors">McDonough, Michael A.; Loenarz, Christoph; Chowdhury, Rasheduzzaman; Clifton, Ian J.; Schofield, Christopher J.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">659-672</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  2-Oxoglutarate- and ferrous iron-dependent oxygenases have emerged as an important family of human enzymes that catalyze hydroxylations and related demethylation reactions.  Their substrates in humans include proteins, nucleic acids, lipids, and small mols.  They play roles in collagen biosynthesis, hypoxic sensing, regulation of gene expression, and lipid biosynthesis/metab.  Structural analyses, principally employing crystallog., have revealed that all of these oxygenases possess a double-stranded β-helix core fold that supports a highly conserved triad of iron binding residues and a less well conserved 2-oxoglutarate co-substrate binding site.  The 2-oxoglutarate binds to the iron in a bidentate manner via its 1-carboxylate and 2-oxo groups.  The primary substrate binding elements are more variable and can involve mobile elements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrkeQ46qZSLnbVg90H21EOLACvtfcHk0lgXS5z83JNiJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsV2jsrfI&md5=1540abfc06c091944a9514a42de68a1f</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.sbi.2010.08.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.sbi.2010.08.006%26sid%3Dliteratum%253Aachs%26aulast%3DMcDonough%26aufirst%3DM.%2BA.%26aulast%3DLoenarz%26aufirst%3DC.%26aulast%3DChowdhury%26aufirst%3DR.%26aulast%3DClifton%26aufirst%3DI.%2BJ.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26atitle%3DStructural%2520studies%2520on%2520human%25202-oxoglutarate%2520dependent%2520oxygenases%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2010%26volume%3D20%26spage%3D659%26epage%3D672%26doi%3D10.1016%2Fj.sbi.2010.08.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Rose, N. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDonough, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">King, O. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawamura, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span> </span><span class="NLM_article-title">Inhibition of 2-oxoglutarate dependent oxygenases</span> <span class="citation_source-journal">Chem. Soc. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">4364</span><span class="NLM_x">–</span> <span class="NLM_lpage">4397</span><span class="refDoi"> DOI: 10.1039/c0cs00203h</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1039%2Fc0cs00203h" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=21390379" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BC3MXovVemur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2011&pages=4364-4397&author=N.+R.+Roseauthor=M.+A.+McDonoughauthor=O.+N.+Kingauthor=A.+Kawamuraauthor=C.+J.+Schofield&title=Inhibition+of+2-oxoglutarate+dependent+oxygenases&doi=10.1039%2Fc0cs00203h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of 2-oxoglutarate dependent oxygenases</span></div><div class="casAuthors">Rose, Nathan R.; McDonough, Michael A.; King, Oliver N. F.; Kawamura, Akane; Schofield, Christopher J.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Society Reviews</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">4364-4397</span>CODEN:
                <span class="NLM_cas:coden">CSRVBR</span>;
        ISSN:<span class="NLM_cas:issn">0306-0012</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  2-Oxoglutarate (2OG) dependent oxygenases are ubiquitous iron enzymes that couple substrate oxidn. to the conversion of 2OG to succinate and carbon dioxide.  In humans their roles include collagen biosynthesis, fatty acid metab., DNA repair, RNA and chromatin modifications, and hypoxic sensing.  Com. applications of 2OG oxygenase inhibitors began with plant growth retardants, and now extend to a clin. used pharmaceutical compd. for cardioprotection.  Several 2OG oxygenases are now being targeted for therapeutic intervention for diseases including anemia, inflammation and cancer.  In this crit. review, we describe studies on the inhibition of 2OG oxygenases, focusing on small mols., and discuss the potential of 2OG oxygenases as therapeutic targets (295 refs.).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqp4Z13BJAo17Vg90H21EOLACvtfcHk0lh7wdz1DgFhSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXovVemur8%253D&md5=f7a87e0bc9392d9a29f4ffb7d6f92b44</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1039%2Fc0cs00203h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc0cs00203h%26sid%3Dliteratum%253Aachs%26aulast%3DRose%26aufirst%3DN.%2BR.%26aulast%3DMcDonough%26aufirst%3DM.%2BA.%26aulast%3DKing%26aufirst%3DO.%2BN.%26aulast%3DKawamura%26aufirst%3DA.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26atitle%3DInhibition%2520of%25202-oxoglutarate%2520dependent%2520oxygenases%26jtitle%3DChem.%2520Soc.%2520Rev.%26date%3D2011%26volume%3D40%26spage%3D4364%26epage%3D4397%26doi%3D10.1039%2Fc0cs00203h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Thinnes, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">England, K. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawamura, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chowdhury, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hopkinson, R. J.</span><span> </span><span class="NLM_article-title">Targeting histone lysine demethylases - Progress, challenges, and the future</span> <span class="citation_source-journal">Biochim. Biophys. Acta, Gene Regul. Mech.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">1839</span><span class="NLM_x">, </span> <span class="NLM_fpage">1416</span><span class="NLM_x">–</span> <span class="NLM_lpage">1432</span><span class="refDoi"> DOI: 10.1016/j.bbagrm.2014.05.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1016%2Fj.bbagrm.2014.05.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=24859458" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVemtbrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1839&publication_year=2014&pages=1416-1432&author=C.+C.+Thinnesauthor=K.+S.+Englandauthor=A.+Kawamuraauthor=R.+Chowdhuryauthor=C.+J.+Schofieldauthor=R.+J.+Hopkinson&title=Targeting+histone+lysine+demethylases+-+Progress%2C+challenges%2C+and+the+future&doi=10.1016%2Fj.bbagrm.2014.05.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting histone lysine demethylases - Progress, challenges, and the future</span></div><div class="casAuthors">Thinnes, Cyrille C.; England, Katherine S.; Kawamura, Akane; Chowdhury, Rasheduzzaman; Schofield, Christopher J.; Hopkinson, Richard J.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Gene Regulatory Mechanisms</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">1839</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1416-1432</span>CODEN:
                <span class="NLM_cas:coden">BBAGC6</span>;
        ISSN:<span class="NLM_cas:issn">1874-9399</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  N-Methylation of lysine and arginine residues has emerged as a major mechanism of transcriptional regulation in eukaryotes.  In humans, Nε-methyllysine residue demethylation is catalyzed by two distinct subfamilies of demethylases (KDMs), the flavin-dependent KDM1 subfamily and the 2-oxoglutarate- (2OG) dependent JmjC subfamily, which both employ oxidative mechanisms.  Modulation of histone methylation status is proposed to be important in epigenetic regulation and has substantial medicinal potential for the treatment of diseases including cancer and genetic disorders.  This article provides an introduction to the enzymol. of the KDMs and the therapeutic possibilities and challenges assocd. with targeting them, followed by a review of reported KDM inhibitors and their mechanisms of action from kinetic and structural perspectives.  This article is part of a Special Issue entitled: Methylation: A Multifaceted Modification - looking at transcription and beyond.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_ltK_pdkkCrVg90H21EOLACvtfcHk0lh7wdz1DgFhSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVemtbrE&md5=d5d70e8873451dbedb4f36c66defc040</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.bbagrm.2014.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbagrm.2014.05.009%26sid%3Dliteratum%253Aachs%26aulast%3DThinnes%26aufirst%3DC.%2BC.%26aulast%3DEngland%26aufirst%3DK.%2BS.%26aulast%3DKawamura%26aufirst%3DA.%26aulast%3DChowdhury%26aufirst%3DR.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26aulast%3DHopkinson%26aufirst%3DR.%2BJ.%26atitle%3DTargeting%2520histone%2520lysine%2520demethylases%2520-%2520Progress%252C%2520challenges%252C%2520and%2520the%2520future%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Gene%2520Regul.%2520Mech.%26date%3D2014%26volume%3D1839%26spage%3D1416%26epage%3D1432%26doi%3D10.1016%2Fj.bbagrm.2014.05.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Thalhammer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mecinovic, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loenarz, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tumber, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rose, N. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heightman, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span> </span><span class="NLM_article-title">Inhibition of the histone demethylase JMJD2E by 3-substituted pyridine 2,4-dicarboxylates</span> <span class="citation_source-journal">Org. Biomol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">127</span><span class="NLM_x">–</span> <span class="NLM_lpage">135</span><span class="refDoi"> DOI: 10.1039/C0OB00592D</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1039%2FC0OB00592D" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=21076780" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsFCqtL%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2011&pages=127-135&author=A.+Thalhammerauthor=J.+Mecinovicauthor=C.+Loenarzauthor=A.+Tumberauthor=N.+R.+Roseauthor=T.+D.+Heightmanauthor=C.+J.+Schofield&title=Inhibition+of+the+histone+demethylase+JMJD2E+by+3-substituted+pyridine+2%2C4-dicarboxylates&doi=10.1039%2FC0OB00592D"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of the histone demethylase JMJD2E by 3-substituted pyridine 2,4-dicarboxylates</span></div><div class="casAuthors">Thalhammer, Armin; Mecinovic, Jasmin; Loenarz, Christoph; Tumber, Anthony; Rose, Nathan R.; Heightman, Tom D.; Schofield, Christopher J.</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">127-135</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Based on structural anal. of the human 2-oxoglutarate (2OG) dependent JMJD2 histone Nε-Me lysyl demethylase family, 3-substituted pyridine 2,4-dicarboxylic acids were identified as potential inhibitors with possible selectivity over other human 2OG oxygenases.  Microwave-assisted palladium-catalyzed cross coupling methodol. was developed to install a diverse set of substituents on the sterically demanding C-3 position of a pyridine 2,4-dicarboxylate scaffold.  The subsequently prepd. di-acids were tested for in vitro inhibition of the histone demethylase JMJD2E and another human 2OG oxygenase, prolyl-hydroxylase domain isoform 2 (PHD2, EGLN1).  A subset of substitution patterns yielded inhibitors with selectivity for JMJD2E over PHD2, demonstrating that structure-based inhibitor design can enable selective inhibition of histone demethylases over related human 2OG oxygenases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqB-fCd3fbQqrVg90H21EOLACvtfcHk0lh7wdz1DgFhSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsFCqtL%252FJ&md5=c4fdff0f41b967e6138b44d1849a05a7</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1039%2FC0OB00592D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC0OB00592D%26sid%3Dliteratum%253Aachs%26aulast%3DThalhammer%26aufirst%3DA.%26aulast%3DMecinovic%26aufirst%3DJ.%26aulast%3DLoenarz%26aufirst%3DC.%26aulast%3DTumber%26aufirst%3DA.%26aulast%3DRose%26aufirst%3DN.%2BR.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26atitle%3DInhibition%2520of%2520the%2520histone%2520demethylase%2520JMJD2E%2520by%25203-substituted%2520pyridine%25202%252C4-dicarboxylates%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2011%26volume%3D9%26spage%3D127%26epage%3D135%26doi%3D10.1039%2FC0OB00592D" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Majamaa, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hanauske-Abel, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gunzler, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kivirikko, K. I.</span><span> </span><span class="NLM_article-title">The 2-oxoglutarate binding site of prolyl 4-hydroxylase. Identification of distinct subsites and evidence for 2-oxoglutarate decarboxylation in a ligand reaction at the enzyme-bound ferrous ion</span> <span class="citation_source-journal">Eur. J. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1984</span><span class="NLM_x">, </span> <span class="NLM_volume">138</span><span class="NLM_x">, </span> <span class="NLM_fpage">239</span><span class="NLM_x">–</span> <span class="NLM_lpage">245</span><span class="refDoi"> DOI: 10.1111/j.1432-1033.1984.tb07907.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1111%2Fj.1432-1033.1984.tb07907.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=6321169" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADyaL2cXmsVSrsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=138&publication_year=1984&pages=239-245&author=K.+Majamaaauthor=H.+M.+Hanauske-Abelauthor=V.+Gunzlerauthor=K.+I.+Kivirikko&title=The+2-oxoglutarate+binding+site+of+prolyl+4-hydroxylase.+Identification+of+distinct+subsites+and+evidence+for+2-oxoglutarate+decarboxylation+in+a+ligand+reaction+at+the+enzyme-bound+ferrous+ion&doi=10.1111%2Fj.1432-1033.1984.tb07907.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">The 2-oxoglutarate-binding site of prolyl 4-hydroxylase. Identification of distinct subsites and evidence for 2-oxoglutarate decarboxylation in a ligand reaction at the enzyme-bound ferrous ion</span></div><div class="casAuthors">Majamaa, Kari; Hanauske-Abel, Hartmut M.; Guenzler, Volkmar; Kivirikko, Kari I.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Biochemistry</span>
        (<span class="NLM_cas:date">1984</span>),
    <span class="NLM_cas:volume">138</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">239-45</span>CODEN:
                <span class="NLM_cas:coden">EJBCAI</span>;
        ISSN:<span class="NLM_cas:issn">0014-2956</span>.
    </div><div class="casAbstract">The structure and function of the 2-oxoglutarate-binding site of prolyl 4-hydroxylase (I) was studied by assaying the inhibitory potential of 24 selected aliph. or arom. compds.  All except one of them inhibited I competitively with respect to 2-oxoglutarate and noncompetitively with respect to Fe2+, the Ki values ranging from 0.8 μM to >15 mM.  The Ki values for the 2 most effective inhibitors, pyridine 2,5-dicarboxylate and 2,4-dicarboxylate, were ∼0.8 and ∼2 μM, resp., these compds. being the most potent inhibitors of I with respect to 2-oxoglutarate known so far.  Only 1 of the compds. tested, 2-oxoadipinate, was able to support hydroxylation by replacing 2-oxoglutarate as a cosubstrate.  The data suggest that the 2-oxoglutarate-binding site can be divided into 3 distinct subsites.  Subsite I is probably a pos. charged side chain of the enzyme that ionically binds the C5 carboxyl group of the 2-oxoglutarate, subsite II consists of 2 cis-positioned equatorial coordination sites of the enzyme-bound Fe2+ and is chelated by the C1-C2 moiety, whereas subsite III involves a hydrophobic binding site in the C3-C4 region of the cosubstrate.  The sp3 rehybridization of C2 within the chelating moiety of the cosubstrate appears to be a crucial event during decarboxylation that proceeds in the form of a ligand reaction inside the Fe2+ coordination sphere.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0K7FKUN6as7Vg90H21EOLACvtfcHk0lhbyV7wOJhoCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2cXmsVSrsw%253D%253D&md5=8fc6c41b2a619c8443a92636a6d64488</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1111%2Fj.1432-1033.1984.tb07907.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1432-1033.1984.tb07907.x%26sid%3Dliteratum%253Aachs%26aulast%3DMajamaa%26aufirst%3DK.%26aulast%3DHanauske-Abel%26aufirst%3DH.%2BM.%26aulast%3DGunzler%26aufirst%3DV.%26aulast%3DKivirikko%26aufirst%3DK.%2BI.%26atitle%3DThe%25202-oxoglutarate%2520binding%2520site%2520of%2520prolyl%25204-hydroxylase.%2520Identification%2520of%2520distinct%2520subsites%2520and%2520evidence%2520for%25202-oxoglutarate%2520decarboxylation%2520in%2520a%2520ligand%2520reaction%2520at%2520the%2520enzyme-bound%2520ferrous%2520ion%26jtitle%3DEur.%2520J.%2520Biochem.%26date%3D1984%26volume%3D138%26spage%3D239%26epage%3D245%26doi%3D10.1111%2Fj.1432-1033.1984.tb07907.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">King, O. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, X. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sakurai, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawamura, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rose, N. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ng, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quinn, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rai, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mott, B. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beswick, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klose, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oppermann, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jadhav, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heightman, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maloney, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeonov, A.</span><span> </span><span class="NLM_article-title">Quantitative high-throughput screening identifies 8-hydroxyquinolines as cell-active histone demethylase inhibitors</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">e15535</span><span class="refDoi"> DOI: 10.1371/journal.pone.0015535</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1371%2Fjournal.pone.0015535" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=21124847" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A280%3ADC%252BC3M%252Fhs1Onsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2010&pages=e15535&author=O.+N.+Kingauthor=X.+S.+Liauthor=M.+Sakuraiauthor=A.+Kawamuraauthor=N.+R.+Roseauthor=S.+S.+Ngauthor=A.+M.+Quinnauthor=G.+Raiauthor=B.+T.+Mottauthor=P.+Beswickauthor=R.+J.+Kloseauthor=U.+Oppermannauthor=A.+Jadhavauthor=T.+D.+Heightmanauthor=D.+J.+Maloneyauthor=C.+J.+Schofieldauthor=A.+Simeonov&title=Quantitative+high-throughput+screening+identifies+8-hydroxyquinolines+as+cell-active+histone+demethylase+inhibitors&doi=10.1371%2Fjournal.pone.0015535"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Quantitative high-throughput screening identifies 8-hydroxyquinolines as cell-active histone demethylase inhibitors</span></div><div class="casAuthors">King Oliver N F; Li Xuan Shirley; Sakurai Masaaki; Kawamura Akane; Rose Nathan R; Ng Stanley S; Quinn Amy M; Rai Ganesha; Mott Bryan T; Beswick Paul; Klose Robert J; Oppermann Udo; Jadhav Ajit; Heightman Tom D; Maloney David J; Schofield Christopher J; Simeonov Anton</div><div class="citationInfo"><span class="NLM_cas:title">PloS one</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">e15535</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Small molecule modulators of epigenetic processes are currently sought as basic probes for biochemical mechanisms, and as starting points for development of therapeutic agents.  N(ε)-Methylation of lysine residues on histone tails is one of a number of post-translational modifications that together enable transcriptional regulation.  Histone lysine demethylases antagonize the action of histone methyltransferases in a site- and methylation state-specific manner.  N(ε)-Methyllysine demethylases that use 2-oxoglutarate as co-factor are associated with diverse human diseases, including cancer, inflammation and X-linked mental retardation; they are proposed as targets for the therapeutic modulation of transcription.  There are few reports on the identification of templates that are amenable to development as potent inhibitors in vivo and large diverse collections have yet to be exploited for the discovery of demethylase inhibitors.  PRINCIPAL FINDINGS:  High-throughput screening of a ∼236,000-member collection of diverse molecules arrayed as dilution series was used to identify inhibitors of the JMJD2 (KDM4) family of 2-oxoglutarate-dependent histone demethylases.  Initial screening hits were prioritized by a combination of cheminformatics, counterscreening using a coupled assay enzyme, and orthogonal confirmatory detection of inhibition by mass spectrometric assays.  Follow-up studies were carried out on one of the series identified, 8-hydroxyquinolines, which were shown by crystallographic analyses to inhibit by binding to the active site Fe(II) and to modulate demethylation at the H3K9 locus in a cell-based assay.  CONCLUSIONS:  These studies demonstrate that diverse compound screening can yield novel inhibitors of 2OG dependent histone demethylases and provide starting points for the development of potent and selective agents to interrogate epigenetic regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTwpX9dy1vWgwxyojvcjibhfW6udTcc2eZL4hX_z70tPLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3M%252Fhs1Onsw%253D%253D&md5=d2d242c04dfb29ed4f2e00843101ae64</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0015535&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0015535%26sid%3Dliteratum%253Aachs%26aulast%3DKing%26aufirst%3DO.%2BN.%26aulast%3DLi%26aufirst%3DX.%2BS.%26aulast%3DSakurai%26aufirst%3DM.%26aulast%3DKawamura%26aufirst%3DA.%26aulast%3DRose%26aufirst%3DN.%2BR.%26aulast%3DNg%26aufirst%3DS.%2BS.%26aulast%3DQuinn%26aufirst%3DA.%2BM.%26aulast%3DRai%26aufirst%3DG.%26aulast%3DMott%26aufirst%3DB.%2BT.%26aulast%3DBeswick%26aufirst%3DP.%26aulast%3DKlose%26aufirst%3DR.%2BJ.%26aulast%3DOppermann%26aufirst%3DU.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DMaloney%26aufirst%3DD.%2BJ.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26aulast%3DSimeonov%26aufirst%3DA.%26atitle%3DQuantitative%2520high-throughput%2520screening%2520identifies%25208-hydroxyquinolines%2520as%2520cell-active%2520histone%2520demethylase%2520inhibitors%26jtitle%3DPLoS%2520One%26date%3D2010%26volume%3D5%26spage%3De15535%26doi%3D10.1371%2Fjournal.pone.0015535" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Hopkinson, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tumber, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yapp, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chowdhury, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aik, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Che, K. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, X. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kristensen, J. B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">King, O. N. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yeoh, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choi, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walport, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thinnes, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bush, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lejeune, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rydzik, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rose, N. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bagg, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDonough, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krojer, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yue, W. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ng, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olsen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brennan, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oppermann, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muller, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klose, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ratcliffe, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawamura, A.</span><span> </span><span class="NLM_article-title">5-Carboxy-8-hydroxyquinoline is a broad spectrum 2-oxoglutarate oxygenase inhibitor which causes iron translocation</span> <span class="citation_source-journal">Chem. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">3110</span><span class="NLM_x">–</span> <span class="NLM_lpage">3117</span><span class="refDoi"> DOI: 10.1039/c3sc51122g</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1039%2Fc3sc51122g" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=26682036" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVehs73L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=3110-3117&author=R.+J.+Hopkinsonauthor=A.+Tumberauthor=C.+Yappauthor=R.+Chowdhuryauthor=W.+Aikauthor=K.+H.+Cheauthor=X.+S.+Liauthor=J.+B.+L.+Kristensenauthor=O.+N.+F.+Kingauthor=M.+C.+Chanauthor=K.+K.+Yeohauthor=H.+Choiauthor=L.+J.+Walportauthor=C.+C.+Thinnesauthor=J.+T.+Bushauthor=C.+Lejeuneauthor=A.+M.+Rydzikauthor=N.+R.+Roseauthor=E.+A.+Baggauthor=M.+A.+McDonoughauthor=T.+J.+Krojerauthor=W.+W.+Yueauthor=S.+S.+Ngauthor=L.+Olsenauthor=P.+E.+Brennanauthor=U.+Oppermannauthor=S.+Mullerauthor=R.+J.+Kloseauthor=P.+J.+Ratcliffeauthor=C.+J.+Schofieldauthor=A.+Kawamura&title=5-Carboxy-8-hydroxyquinoline+is+a+broad+spectrum+2-oxoglutarate+oxygenase+inhibitor+which+causes+iron+translocation&doi=10.1039%2Fc3sc51122g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">5-Carboxy-8-hydroxyquinoline is a broad spectrum 2-oxoglutarate oxygenase inhibitor which causes iron translocation</span></div><div class="casAuthors">Hopkinson, Richard J.; Tumber, Anthony; Yapp, Clarence; Chowdhury, Rasheduzzaman; Aik, WeiShen; Che, Ka Hing; Li, Xuan Shirley; Kristensen, Jan B. L.; King, Oliver N. F.; Chan, Mun Chiang; Yeoh, Kar Kheng; Choi, Hwanho; Walport, Louise J.; Thinnes, Cyrille C.; Bush, Jacob T.; Lejeune, Clarisse; Rydzik, Anna M.; Rose, Nathan R.; Bagg, Eleanor A.; McDonough, Michael A.; Krojer, Tobias J.; Yue, Wyatt W.; Ng, Stanley S.; Olsen, Lars; Brennan, Paul E.; Oppermann, Udo; Mueller, Susanne; Klose, Robert J.; Ratcliffe, Peter J.; Schofield, Christopher J.; Kawamura, Akane</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3110-3117</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">2-Oxoglutarate and iron dependent oxygenases are therapeutic targets for human diseases.  Using a representative 2OG oxygenase panel, the authors compare the inhibitory activities of 5-carboxy-8-hydroxyquinoline (IOX1) and 4-carboxy-8-hydroxyquinoline (4C8HQ) with that of two other commonly used 2OG oxygenase inhibitors, N-oxalylglycine (NOG) and 2,4-pyridinedicarboxylic acid (2,4-PDCA).  IOX1 has a broad spectrum of activity, as demonstrated by the inhibition of transcription factor hydroxylases, representatives of all 2OG dependent histone demethylase subfamilies, nucleic acid demethylases and γ-butyrobetaine hydroxylase.  Cellular assays show that, unlike NOG and 2,4-PDCA, IOX1 is active against both cytosolic and nuclear 2OG oxygenases without ester derivatization.  Unexpectedly, crystallog. studies on these oxygenases demonstrate that IOX1, but not 4C8HQ, can cause translocation of the active site metal, revealing a rare example of protein ligand-induced metal movement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_NBtJvNEBOLVg90H21EOLACvtfcHk0lhbyV7wOJhoCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVehs73L&md5=506fb4e75971c57a7a968f4d37805856</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1039%2Fc3sc51122g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc3sc51122g%26sid%3Dliteratum%253Aachs%26aulast%3DHopkinson%26aufirst%3DR.%2BJ.%26aulast%3DTumber%26aufirst%3DA.%26aulast%3DYapp%26aufirst%3DC.%26aulast%3DChowdhury%26aufirst%3DR.%26aulast%3DAik%26aufirst%3DW.%26aulast%3DChe%26aufirst%3DK.%2BH.%26aulast%3DLi%26aufirst%3DX.%2BS.%26aulast%3DKristensen%26aufirst%3DJ.%2BB.%2BL.%26aulast%3DKing%26aufirst%3DO.%2BN.%2BF.%26aulast%3DChan%26aufirst%3DM.%2BC.%26aulast%3DYeoh%26aufirst%3DK.%2BK.%26aulast%3DChoi%26aufirst%3DH.%26aulast%3DWalport%26aufirst%3DL.%2BJ.%26aulast%3DThinnes%26aufirst%3DC.%2BC.%26aulast%3DBush%26aufirst%3DJ.%2BT.%26aulast%3DLejeune%26aufirst%3DC.%26aulast%3DRydzik%26aufirst%3DA.%2BM.%26aulast%3DRose%26aufirst%3DN.%2BR.%26aulast%3DBagg%26aufirst%3DE.%2BA.%26aulast%3DMcDonough%26aufirst%3DM.%2BA.%26aulast%3DKrojer%26aufirst%3DT.%2BJ.%26aulast%3DYue%26aufirst%3DW.%2BW.%26aulast%3DNg%26aufirst%3DS.%2BS.%26aulast%3DOlsen%26aufirst%3DL.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DOppermann%26aufirst%3DU.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DKlose%26aufirst%3DR.%2BJ.%26aulast%3DRatcliffe%26aufirst%3DP.%2BJ.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26aulast%3DKawamura%26aufirst%3DA.%26atitle%3D5-Carboxy-8-hydroxyquinoline%2520is%2520a%2520broad%2520spectrum%25202-oxoglutarate%2520oxygenase%2520inhibitor%2520which%2520causes%2520iron%2520translocation%26jtitle%3DChem.%2520Sci.%26date%3D2013%26volume%3D4%26spage%3D3110%26epage%3D3117%26doi%3D10.1039%2Fc3sc51122g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Rose, N. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ng, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mecinovic, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lienard, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bello, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDonough, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oppermann, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span> </span><span class="NLM_article-title">Inhibitor scaffolds for 2-oxoglutarate-dependent histone lysine demethylases</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">7053</span><span class="NLM_x">–</span> <span class="NLM_lpage">7056</span><span class="refDoi"> DOI: 10.1021/jm800936s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm800936s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BD1cXht1Ors7rI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=7053-7056&author=N.+R.+Roseauthor=S.+S.+Ngauthor=J.+Mecinovicauthor=B.+M.+Lienardauthor=S.+H.+Belloauthor=Z.+Sunauthor=M.+A.+McDonoughauthor=U.+Oppermannauthor=C.+J.+Schofield&title=Inhibitor+scaffolds+for+2-oxoglutarate-dependent+histone+lysine+demethylases&doi=10.1021%2Fjm800936s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitor Scaffolds for 2-Oxoglutarate-Dependent Histone Lysine Demethylases</span></div><div class="casAuthors">Rose, Nathan R.; Ng, Stanley S.; Mecinovic, Jasmin; Lienard, Benoit M. R.; Bello, Simon H.; Sun, Zhe; McDonough, Michael A.; Oppermann, Udo; Schofield, Christopher J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">7053-7056</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The dynamic methylation of histone lysyl residues plays an important role in biol. by regulating transcription, maintaining genomic integrity, and by contributing to epigenetic effects.  Here we describe a variety of inhibitor scaffolds that inhibit the human 2-oxoglutarate-dependent JMJD2 subfamily of histone demethylases.  Combined with structural data, these chem. starting points will be useful to generate small-mol. probes to analyze the physiol. roles of these enzymes in epigenetic signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr90HQ8nfBLFrVg90H21EOLACvtfcHk0lhPqYjckNqKfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXht1Ors7rI&md5=155c830b0544a6c1d5e6eb81d14389de</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fjm800936s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm800936s%26sid%3Dliteratum%253Aachs%26aulast%3DRose%26aufirst%3DN.%2BR.%26aulast%3DNg%26aufirst%3DS.%2BS.%26aulast%3DMecinovic%26aufirst%3DJ.%26aulast%3DLienard%26aufirst%3DB.%2BM.%26aulast%3DBello%26aufirst%3DS.%2BH.%26aulast%3DSun%26aufirst%3DZ.%26aulast%3DMcDonough%26aufirst%3DM.%2BA.%26aulast%3DOppermann%26aufirst%3DU.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26atitle%3DInhibitor%2520scaffolds%2520for%25202-oxoglutarate-dependent%2520histone%2520lysine%2520demethylases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D7053%26epage%3D7056%26doi%3D10.1021%2Fjm800936s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varghese, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dellinger, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kumar, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Best, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruiz, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bruick, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pena-Llopis, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Babinski, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frantz, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brekken, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quinn, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeonov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Easmon, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martinez, E. D.</span><span> </span><span class="NLM_article-title">A small molecule modulates Jumonji histone demethylase activity and selectively inhibits cancer growth</span> <span class="citation_source-journal">Nat. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">2035</span><span class="refDoi"> DOI: 10.1038/ncomms3035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1038%2Fncomms3035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=23792809" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A280%3ADC%252BC3sjksV2iug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=2035&author=L.+Wangauthor=J.+Changauthor=D.+Vargheseauthor=M.+Dellingerauthor=S.+Kumarauthor=A.+M.+Bestauthor=J.+Ruizauthor=R.+Bruickauthor=S.+Pena-Llopisauthor=J.+Xuauthor=D.+J.+Babinskiauthor=D.+E.+Frantzauthor=R.+A.+Brekkenauthor=A.+M.+Quinnauthor=A.+Simeonovauthor=J.+Easmonauthor=E.+D.+Martinez&title=A+small+molecule+modulates+Jumonji+histone+demethylase+activity+and+selectively+inhibits+cancer+growth&doi=10.1038%2Fncomms3035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">A small molecule modulates Jumonji histone demethylase activity and selectively inhibits cancer growth</span></div><div class="casAuthors">Wang Lei; Chang Jianjun; Varghese Diana; Dellinger Michael; Kumar Subodh; Best Anne M; Ruiz Julio; Bruick Richard; Pena-Llopis Samuel; Xu Junjie; Babinski David J; Frantz Doug E; Brekken Rolf A; Quinn Amy M; Simeonov Anton; Easmon Johnny; Martinez Elisabeth D</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">2035</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The pharmacological inhibition of general transcriptional regulators has the potential to block growth through targeting multiple tumorigenic signalling pathways simultaneously.  Here, using an innovative cell-based screen, we identify a structurally unique small molecule (named JIB-04) that specifically inhibits the activity of the Jumonji family of histone demethylases in vitro, in cancer cells, and in tumours in vivo.  Unlike known inhibitors, JIB-04 is not a competitive inhibitor of α-ketoglutarate.  In cancer, but not in patient-matched normal cells, JIB-04 alters a subset of transcriptional pathways and blocks viability.  In mice, JIB-04 reduces tumour burden and prolongs survival.  Importantly, we find that patients with breast tumours that overexpress Jumonji demethylases have significantly lower survival.  Thus, JIB-04, a novel inhibitor of Jumonji demethylases in vitro and in vivo, constitutes a unique potential therapeutic and research tool against cancer, and validates the use of unbiased cellular screens to discover chemical modulators with disease relevance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ5QxHbn8WTZg20zIbnCCI3fW6udTcc2eYxihy117lonrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3sjksV2iug%253D%253D&md5=37aa303d173d027fd711bc4ef7fd841b</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fncomms3035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms3035%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.%26aulast%3DChang%26aufirst%3DJ.%26aulast%3DVarghese%26aufirst%3DD.%26aulast%3DDellinger%26aufirst%3DM.%26aulast%3DKumar%26aufirst%3DS.%26aulast%3DBest%26aufirst%3DA.%2BM.%26aulast%3DRuiz%26aufirst%3DJ.%26aulast%3DBruick%26aufirst%3DR.%26aulast%3DPena-Llopis%26aufirst%3DS.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DBabinski%26aufirst%3DD.%2BJ.%26aulast%3DFrantz%26aufirst%3DD.%2BE.%26aulast%3DBrekken%26aufirst%3DR.%2BA.%26aulast%3DQuinn%26aufirst%3DA.%2BM.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DEasmon%26aufirst%3DJ.%26aulast%3DMartinez%26aufirst%3DE.%2BD.%26atitle%3DA%2520small%2520molecule%2520modulates%2520Jumonji%2520histone%2520demethylase%2520activity%2520and%2520selectively%2520inhibits%2520cancer%2520growth%26jtitle%3DNat.%2520Commun.%26date%3D2013%26volume%3D4%26spage%3D2035%26doi%3D10.1038%2Fncomms3035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Luo, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kubicek, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Myllyharju, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tumber, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ng, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Che, K. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Podoll, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heightman, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oppermann, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schreiber, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, X.</span><span> </span><span class="NLM_article-title">A selective inhibitor and probe of the cellular functions of Jumonji C domain-containing histone demethylases</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">133</span><span class="NLM_x">, </span> <span class="NLM_fpage">9451</span><span class="NLM_x">–</span> <span class="NLM_lpage">9456</span><span class="refDoi"> DOI: 10.1021/ja201597b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja201597b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmslGmurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2011&pages=9451-9456&author=X.+Luoauthor=Y.+Liuauthor=S.+Kubicekauthor=J.+Myllyharjuauthor=A.+Tumberauthor=S.+Ngauthor=K.+H.+Cheauthor=J.+Podollauthor=T.+D.+Heightmanauthor=U.+Oppermannauthor=S.+L.+Schreiberauthor=X.+Wang&title=A+selective+inhibitor+and+probe+of+the+cellular+functions+of+Jumonji+C+domain-containing+histone+demethylases&doi=10.1021%2Fja201597b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">A Selective Inhibitor and Probe of the Cellular Functions of Jumonji C Domain-Containing Histone Demethylases</span></div><div class="casAuthors">Luo, Xue-Lai; Liu, Yong-Xiang; Kubicek, Stefan; Myllyharju, Johanna; Tumber, Anthony; Ng, Stanley; Che, Ka-Hing; Podoll, Jessica; Heightman, Tom D.; Oppermann, Udo; Schreiber, Stuart L.; Wang, Xiang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">9451-9456</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Histone methylations are important chromatin marks that regulate gene expression, genomic stability, DNA repair, and genomic imprinting.  Histone demethylases are the most recent family of histone-modifying enzymes discovered.  Here, we report the characterization of a small-mol. inhibitor of Jumonji C domain-contg. histone demethylases.  The inhibitor derives from a structure-based design and preferentially inhibits the subfamily of tri-Me lysine demethylases.  Its Me ester prodrug, methylstat, selectively inhibits Jumonji C domain-contg. his-tone demethylases in cells and may be a useful small-mol. probe of chromatin and its role in epigenetics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcmCdVvaIpq7Vg90H21EOLACvtfcHk0lhPqYjckNqKfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmslGmurg%253D&md5=ad50a692dfab85e1bcce77acc14b7b5a</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fja201597b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja201597b%26sid%3Dliteratum%253Aachs%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DKubicek%26aufirst%3DS.%26aulast%3DMyllyharju%26aufirst%3DJ.%26aulast%3DTumber%26aufirst%3DA.%26aulast%3DNg%26aufirst%3DS.%26aulast%3DChe%26aufirst%3DK.%2BH.%26aulast%3DPodoll%26aufirst%3DJ.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DOppermann%26aufirst%3DU.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26aulast%3DWang%26aufirst%3DX.%26atitle%3DA%2520selective%2520inhibitor%2520and%2520probe%2520of%2520the%2520cellular%2520functions%2520of%2520Jumonji%2520C%2520domain-containing%2520histone%2520demethylases%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2011%26volume%3D133%26spage%3D9451%26epage%3D9456%26doi%3D10.1021%2Fja201597b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Kruidenier, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liddle, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Che, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Joberty, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bantscheff, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bountra, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bridges, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diallo, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eberhard, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hutchinson, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Katso, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leveridge, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mander, P. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mosley, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramirez-Molina, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rowland, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sheppard, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swales, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tanner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomas, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tumber, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drewes, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oppermann, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, D. M.</span><span> </span><span class="NLM_article-title">A selective jumonji H3K27 demethylase inhibitor modulates the proinflammatory macrophage response</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">488</span><span class="NLM_x">, </span> <span class="NLM_fpage">404</span><span class="NLM_x">–</span> <span class="NLM_lpage">408</span><span class="refDoi"> DOI: 10.1038/nature11262</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1038%2Fnature11262" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=22842901" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFWjtbfI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=488&publication_year=2012&pages=404-408&author=L.+Kruidenierauthor=C.+W.+Chungauthor=Z.+Chengauthor=J.+Liddleauthor=K.+Cheauthor=G.+Jobertyauthor=M.+Bantscheffauthor=C.+Bountraauthor=A.+Bridgesauthor=H.+Dialloauthor=D.+Eberhardauthor=S.+Hutchinsonauthor=E.+Jonesauthor=R.+Katsoauthor=M.+Leveridgeauthor=P.+K.+Manderauthor=J.+Mosleyauthor=C.+Ramirez-Molinaauthor=P.+Rowlandauthor=C.+J.+Schofieldauthor=R.+J.+Sheppardauthor=J.+E.+Smithauthor=C.+Swalesauthor=R.+Tannerauthor=P.+Thomasauthor=A.+Tumberauthor=G.+Drewesauthor=U.+Oppermannauthor=D.+J.+Patelauthor=K.+Leeauthor=D.+M.+Wilson&title=A+selective+jumonji+H3K27+demethylase+inhibitor+modulates+the+proinflammatory+macrophage+response&doi=10.1038%2Fnature11262"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">A selective jumonji H3K27 demethylase inhibitor modulates the proinflammatory macrophage response</span></div><div class="casAuthors">Kruidenier, Laurens; Chung, Chun-wa; Cheng, Zhongjun; Liddle, John; Che, Ka Hing; Joberty, Gerard; Bantscheff, Marcus; Bountra, Chas; Bridges, Angela; Diallo, Hawa; Eberhard, Dirk; Hutchinson, Sue; Jones, Emma; Katso, Roy; Leveridge, Melanie; Mander, Palwinder K.; Mosley, Julie; Ramirez-Molina, Cesar; Rowland, Paul; Schofield, Christopher J.; Sheppard, Robert J.; Smith, Julia E.; Swales, Catherine; Tanner, Robert; Thomas, Pamela; Tumber, Anthony; Drewes, Gerard; Oppermann, Udo; Patel, Dinshaw J.; Lee, Kevin; Wilson, David M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">488</span>
        (<span class="NLM_cas:issue">7411</span>),
    <span class="NLM_cas:pages">404-408</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The jumonji (JMJ) family of histone demethylases are Fe2+- and α-ketoglutarate-dependent oxygenases that are essential components of regulatory transcriptional chromatin complexes.  These enzymes demethylate lysine residues in histones in a methylation-state and sequence-specific context.  Considerable effort has been devoted to gaining a mechanistic understanding of the roles of histone lysine demethylases in eukaryotic transcription, genome integrity and epigenetic inheritance, as well as in development, physiol. and disease.  However, because of the absence of any selective inhibitors, the relevance of the demethylase activity of JMJ enzymes in regulating cellular responses remains poorly understood.  Here we present a structure-guided small-mol. and chemoproteomics approach to elucidating the functional role of the H3K27me3-specific demethylase subfamily (KDM6 subfamily members JMJD3 and UTX).  The liganded structures of human and mouse JMJD3 provide novel insight into the specificity determinants for cofactor, substrate and inhibitor recognition by the KDM6 subfamily of demethylases.  We exploited these structural features to generate the first small-mol. catalytic site inhibitor that is selective for the H3K27me3-specific JMJ subfamily.  We demonstrate that this inhibitor binds in a novel manner and reduces lipopolysaccharide-induced proinflammatory cytokine prodn. by human primary macrophages, a process that depends on both JMJD3 and UTX.  Our results resolve the ambiguity assocd. with the catalytic function of H3K27-specific JMJs in regulating disease-relevant inflammatory responses and provide encouragement for designing small-mol. inhibitors to allow selective pharmacol. intervention across the JMJ family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-lxSNmxjfsbVg90H21EOLACvtfcHk0ljHX2xiAmww0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFWjtbfI&md5=a4fa470a86070b4777e16a71e51763c8</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1038%2Fnature11262&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11262%26sid%3Dliteratum%253Aachs%26aulast%3DKruidenier%26aufirst%3DL.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DCheng%26aufirst%3DZ.%26aulast%3DLiddle%26aufirst%3DJ.%26aulast%3DChe%26aufirst%3DK.%26aulast%3DJoberty%26aufirst%3DG.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DBountra%26aufirst%3DC.%26aulast%3DBridges%26aufirst%3DA.%26aulast%3DDiallo%26aufirst%3DH.%26aulast%3DEberhard%26aufirst%3DD.%26aulast%3DHutchinson%26aufirst%3DS.%26aulast%3DJones%26aufirst%3DE.%26aulast%3DKatso%26aufirst%3DR.%26aulast%3DLeveridge%26aufirst%3DM.%26aulast%3DMander%26aufirst%3DP.%2BK.%26aulast%3DMosley%26aufirst%3DJ.%26aulast%3DRamirez-Molina%26aufirst%3DC.%26aulast%3DRowland%26aufirst%3DP.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26aulast%3DSheppard%26aufirst%3DR.%2BJ.%26aulast%3DSmith%26aufirst%3DJ.%2BE.%26aulast%3DSwales%26aufirst%3DC.%26aulast%3DTanner%26aufirst%3DR.%26aulast%3DThomas%26aufirst%3DP.%26aulast%3DTumber%26aufirst%3DA.%26aulast%3DDrewes%26aufirst%3DG.%26aulast%3DOppermann%26aufirst%3DU.%26aulast%3DPatel%26aufirst%3DD.%2BJ.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DWilson%26aufirst%3DD.%2BM.%26atitle%3DA%2520selective%2520jumonji%2520H3K27%2520demethylase%2520inhibitor%2520modulates%2520the%2520proinflammatory%2520macrophage%2520response%26jtitle%3DNature%26date%3D2012%26volume%3D488%26spage%3D404%26epage%3D408%26doi%3D10.1038%2Fnature11262" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Heinemann, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nielsen, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hudlebusch, H. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lees, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Larsen, D. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boesen, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Labelle, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerlach, L. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Birk, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Helin, K.</span><span> </span><span class="NLM_article-title">Inhibition of demethylases by GSK-J1/J4</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">514</span><span class="NLM_x">, </span> <span class="NLM_fpage">E1</span><span class="NLM_x">–</span> <span class="NLM_lpage">2</span><span class="refDoi"> DOI: 10.1038/nature13688</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1038%2Fnature13688" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=514&publication_year=2014&pages=E1-2&author=B.+Heinemannauthor=J.+M.+Nielsenauthor=H.+R.+Hudlebuschauthor=M.+J.+Leesauthor=D.+V.+Larsenauthor=T.+Boesenauthor=M.+Labelleauthor=L.+O.+Gerlachauthor=P.+Birkauthor=K.+Helin&title=Inhibition+of+demethylases+by+GSK-J1%2FJ4&doi=10.1038%2Fnature13688"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1038%2Fnature13688&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13688%26sid%3Dliteratum%253Aachs%26aulast%3DHeinemann%26aufirst%3DB.%26aulast%3DNielsen%26aufirst%3DJ.%2BM.%26aulast%3DHudlebusch%26aufirst%3DH.%2BR.%26aulast%3DLees%26aufirst%3DM.%2BJ.%26aulast%3DLarsen%26aufirst%3DD.%2BV.%26aulast%3DBoesen%26aufirst%3DT.%26aulast%3DLabelle%26aufirst%3DM.%26aulast%3DGerlach%26aufirst%3DL.%2BO.%26aulast%3DBirk%26aufirst%3DP.%26aulast%3DHelin%26aufirst%3DK.%26atitle%3DInhibition%2520of%2520demethylases%2520by%2520GSK-J1%252FJ4%26jtitle%3DNature%26date%3D2014%26volume%3D514%26spage%3DE1%26epage%3D2%26doi%3D10.1038%2Fnature13688" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Woon, E. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tumber, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawamura, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hillringhaus, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ge, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rose, N. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walport, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Che, K. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ng, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marsden, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oppermann, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDonough, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span> </span><span class="NLM_article-title">Linking of 2-oxoglutarate and substrate binding sites enables potent and highly selective inhibition of JmjC histone demethylases</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">1631</span><span class="NLM_x">–</span> <span class="NLM_lpage">1634</span><span class="refDoi"> DOI: 10.1002/anie.201107833</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1002%2Fanie.201107833" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2012&pages=1631-1634&author=E.+C.+Woonauthor=A.+Tumberauthor=A.+Kawamuraauthor=L.+Hillringhausauthor=W.+Geauthor=N.+R.+Roseauthor=J.+H.+Maauthor=M.+C.+Chanauthor=L.+J.+Walportauthor=K.+H.+Cheauthor=S.+S.+Ngauthor=B.+D.+Marsdenauthor=U.+Oppermannauthor=M.+A.+McDonoughauthor=C.+J.+Schofield&title=Linking+of+2-oxoglutarate+and+substrate+binding+sites+enables+potent+and+highly+selective+inhibition+of+JmjC+histone+demethylases&doi=10.1002%2Fanie.201107833"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1002%2Fanie.201107833&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201107833%26sid%3Dliteratum%253Aachs%26aulast%3DWoon%26aufirst%3DE.%2BC.%26aulast%3DTumber%26aufirst%3DA.%26aulast%3DKawamura%26aufirst%3DA.%26aulast%3DHillringhaus%26aufirst%3DL.%26aulast%3DGe%26aufirst%3DW.%26aulast%3DRose%26aufirst%3DN.%2BR.%26aulast%3DMa%26aufirst%3DJ.%2BH.%26aulast%3DChan%26aufirst%3DM.%2BC.%26aulast%3DWalport%26aufirst%3DL.%2BJ.%26aulast%3DChe%26aufirst%3DK.%2BH.%26aulast%3DNg%26aufirst%3DS.%2BS.%26aulast%3DMarsden%26aufirst%3DB.%2BD.%26aulast%3DOppermann%26aufirst%3DU.%26aulast%3DMcDonough%26aufirst%3DM.%2BA.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26atitle%3DLinking%2520of%25202-oxoglutarate%2520and%2520substrate%2520binding%2520sites%2520enables%2520potent%2520and%2520highly%2520selective%2520inhibition%2520of%2520JmjC%2520histone%2520demethylases%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2012%26volume%3D51%26spage%3D1631%26epage%3D1634%26doi%3D10.1002%2Fanie.201107833" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Murray, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blundell, T. L.</span><span> </span><span class="NLM_article-title">Structural biology in fragment-based drug design</span> <span class="citation_source-journal">Curr. Opin. Struct. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">497</span><span class="NLM_x">–</span> <span class="NLM_lpage">507</span><span class="refDoi"> DOI: 10.1016/j.sbi.2010.04.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1016%2Fj.sbi.2010.04.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=20471246" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVKhtLvF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2010&pages=497-507&author=C.+W.+Murrayauthor=T.+L.+Blundell&title=Structural+biology+in+fragment-based+drug+design&doi=10.1016%2Fj.sbi.2010.04.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Structural biology in fragment-based drug design</span></div><div class="casAuthors">Murray, Christopher W.; Blundell, Tom L.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">497-507</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Fragment-based ligand screening is now established as an emerging paradigm for drug discovery.  Here we examine the recent literature looking at how structural biol. has been used in a variety of successful fragment-screening applications.  We argue that the detn. of exptl. binding modes has proved to be one of the mainstays of successful fragment-based approaches and that this reflects the difficulty in optimizing a fragment to a lead mol. in the absence of structural information.  We focus on antimicrobial research where fragment-based drug discovery allows control of the phys. properties of the emerging lead mol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1S2lx8J-FX7Vg90H21EOLACvtfcHk0ljHX2xiAmww0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVKhtLvF&md5=ae8cfc0f8be47a316bd92073e95e0c8a</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.sbi.2010.04.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.sbi.2010.04.003%26sid%3Dliteratum%253Aachs%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DBlundell%26aufirst%3DT.%2BL.%26atitle%3DStructural%2520biology%2520in%2520fragment-based%2520drug%2520design%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2010%26volume%3D20%26spage%3D497%26epage%3D507%26doi%3D10.1016%2Fj.sbi.2010.04.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Hamada, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Itoh, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsumoto, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakagawa, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janknecht, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyata, N.</span><span> </span><span class="NLM_article-title">Synthesis and activity of N-oxalylglycine and its derivatives as Jumonji C-domain-containing histone lysine demethylase inhibitors</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">2852</span><span class="NLM_x">–</span> <span class="NLM_lpage">2855</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2009.03.098</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1016%2Fj.bmcl.2009.03.098" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=2852-2855&author=S.+Hamadaauthor=T.+D.+Kimauthor=T.+Suzukiauthor=Y.+Itohauthor=H.+Tsumotoauthor=H.+Nakagawaauthor=R.+Janknechtauthor=N.+Miyata&title=Synthesis+and+activity+of+N-oxalylglycine+and+its+derivatives+as+Jumonji+C-domain-containing+histone+lysine+demethylase+inhibitors&doi=10.1016%2Fj.bmcl.2009.03.098"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2009.03.098&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2009.03.098%26sid%3Dliteratum%253Aachs%26aulast%3DHamada%26aufirst%3DS.%26aulast%3DKim%26aufirst%3DT.%2BD.%26aulast%3DSuzuki%26aufirst%3DT.%26aulast%3DItoh%26aufirst%3DY.%26aulast%3DTsumoto%26aufirst%3DH.%26aulast%3DNakagawa%26aufirst%3DH.%26aulast%3DJanknecht%26aufirst%3DR.%26aulast%3DMiyata%26aufirst%3DN.%26atitle%3DSynthesis%2520and%2520activity%2520of%2520N-oxalylglycine%2520and%2520its%2520derivatives%2520as%2520Jumonji%2520C-domain-containing%2520histone%2520lysine%2520demethylase%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2009%26volume%3D19%26spage%3D2852%26epage%3D2855%26doi%3D10.1016%2Fj.bmcl.2009.03.098" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Hann, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leach, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harper, G.</span><span> </span><span class="NLM_article-title">Molecular complexity and its impact on the probability of finding leads for drug discovery</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">856</span><span class="NLM_x">–</span> <span class="NLM_lpage">864</span><span class="refDoi"> DOI: 10.1021/ci000403i</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci000403i" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BD3MXivFynt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2001&pages=856-864&author=M.+M.+Hannauthor=A.+R.+Leachauthor=G.+Harper&title=Molecular+complexity+and+its+impact+on+the+probability+of+finding+leads+for+drug+discovery&doi=10.1021%2Fci000403i"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Complexity and Its Impact on the Probability of Finding Leads for Drug Discovery</span></div><div class="casAuthors">Hann, Michael M.; Leach, Andrew R.; Harper, Gavin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">856-864</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Using a simple model of ligand-receptor interactions, the interactions between ligands and receptors of varying complexities are studied and the probabilities of binding calcd.  It is obsd. that as the systems become more complex the chance of observing a useful interaction for a randomly chosen ligand falls dramatically.  The implications of this for the design of combinatorial libraries is explored.  A large set of drug leads and optimized compds. is profiled using several different properties relevant to mol. recognition.  The changes obsd. for these properties during the drug optimization phase support the hypothesis that less complex mols. are more common starting points for the discovery of drugs.  An extreme example of the use of simple mols. for directed screening against thrombin is provided.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7ed40mt9qb7Vg90H21EOLACvtfcHk0lhYz4UAaulyKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXivFynt7w%253D&md5=e89e76c8fb83fc5012369ae30f20e6b7</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fci000403i&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci000403i%26sid%3Dliteratum%253Aachs%26aulast%3DHann%26aufirst%3DM.%2BM.%26aulast%3DLeach%26aufirst%3DA.%2BR.%26aulast%3DHarper%26aufirst%3DG.%26atitle%3DMolecular%2520complexity%2520and%2520its%2520impact%2520on%2520the%2520probability%2520of%2520finding%2520leads%2520for%2520drug%2520discovery%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2001%26volume%3D41%26spage%3D856%26epage%3D864%26doi%3D10.1021%2Fci000403i" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Hann, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keseru, G. M.</span><span> </span><span class="NLM_article-title">Finding the sweet spot: the role of nature and nurture in medicinal chemistry</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">355</span><span class="NLM_x">–</span> <span class="NLM_lpage">365</span><span class="refDoi"> DOI: 10.1038/nrd3701</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1038%2Fnrd3701" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=22543468" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BC38Xmt1GqsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=355-365&author=M.+M.+Hannauthor=G.+M.+Keseru&title=Finding+the+sweet+spot%3A+the+role+of+nature+and+nurture+in+medicinal+chemistry&doi=10.1038%2Fnrd3701"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Finding the sweet spot: the role of nature and nurture in medicinal chemistry</span></div><div class="casAuthors">Hann, Michael M.; Keserue, Gyoergy M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">355-365</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Given its position at the heart of small-mol. drug discovery, medicinal chem. has an important role in tackling the well-known productivity challenges in pharmaceutical research and development.  In recent years, extensive analyses of successful and failed discovery compds. and drug candidates have improved our understanding of the role of physicochem. properties in drug attrition.  Based on the clarified challenges in finding the 'sweet spot' in medicinal chem. programs, we suggest that this goal can be achieved through a combination of first identifying chem. starting points with appropriate 'nature' and then rigorously 'nurturing' them during lead optimization.  Here, we discuss scientific, strategic, organizational and cultural considerations for medicinal chem. practices, with the aim of promoting more effective use of what is already known, as well as a wider appreciation of the risks of pursuing suboptimal compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrURciGjB2A3LVg90H21EOLACvtfcHk0lhYz4UAaulyKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xmt1GqsLY%253D&md5=90170e3f4e69e52bfa38667df380516a</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1038%2Fnrd3701&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3701%26sid%3Dliteratum%253Aachs%26aulast%3DHann%26aufirst%3DM.%2BM.%26aulast%3DKeseru%26aufirst%3DG.%2BM.%26atitle%3DFinding%2520the%2520sweet%2520spot%253A%2520the%2520role%2520of%2520nature%2520and%2520nurture%2520in%2520medicinal%2520chemistry%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2012%26volume%3D11%26spage%3D355%26epage%3D365%26doi%3D10.1038%2Fnrd3701" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Murray, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carr, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Callaghan, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chessari, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Congreve, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coyle, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Downham, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Figueroa, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frederickson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Graham, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McMenamin, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Brien, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Phillips, T. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woodhead, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woolford, A. J.</span><span> </span><span class="NLM_article-title">Fragment-based drug discovery applied to Hsp90. Discovery of two lead series with high ligand efficiency</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">5942</span><span class="NLM_x">–</span> <span class="NLM_lpage">5955</span><span class="refDoi"> DOI: 10.1021/jm100059d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm100059d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BC3cXpsVChsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=5942-5955&author=C.+W.+Murrayauthor=M.+G.+Carrauthor=O.+Callaghanauthor=G.+Chessariauthor=M.+Congreveauthor=S.+Cowanauthor=J.+E.+Coyleauthor=R.+Downhamauthor=E.+Figueroaauthor=M.+Fredericksonauthor=B.+Grahamauthor=R.+McMenaminauthor=M.+A.+O%E2%80%99Brienauthor=S.+Patelauthor=T.+R.+Phillipsauthor=G.+Williamsauthor=A.+J.+Woodheadauthor=A.+J.+Woolford&title=Fragment-based+drug+discovery+applied+to+Hsp90.+Discovery+of+two+lead+series+with+high+ligand+efficiency&doi=10.1021%2Fjm100059d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-Based Drug Discovery Applied to Hsp90. Discovery of Two Lead Series with High Ligand Efficiency</span></div><div class="casAuthors">Murray, Christopher W.; Carr, Maria G.; Callaghan, Owen; Chessari, Gianni; Congreve, Miles; Cowan, Suzanna; Coyle, Joseph E.; Downham, Robert; Figueroa, Eva; Frederickson, Martyn; Graham, Brent; McMenamin, Rachel; O'Brien, M. Alistair; Patel, Sahil; Phillips, Theresa R.; Williams, Glyn; Woodhead, Andrew J.; Woolford, Alison J.-A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">5942-5955</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Inhibitors of the chaperone Hsp90 are potentially useful as chemotherapeutic agents in cancer.  This paper describes an application of fragment screening to Hsp90 using a combination of NMR and high throughput x-ray crystallog.  The screening identified an aminopyrimidine with affinity in the high micromolar range and subsequent structure-based design allowed its optimization into a low nanomolar series with good ligand efficiency.  A phenolic chemotype was also identified in fragment screening and was found to bind with affinity close to 1 mM.  This fragment was optimized using structure based design into a resorcinol lead which has subnanomolar affinity for Hsp90, excellent cell potency, and good ligand efficiency.  This fragment to lead campaign improved affinity for Hsp90 by over 1,000,000 fold with the addn. of only 6 heavy atoms.  The companion paper (DOI: 10.1021/jm100060b) describes how the resorcinol lead was optimized into a compd. that is now in clin. trials for the treatment of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqISC3cw2labbVg90H21EOLACvtfcHk0lhYz4UAaulyKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXpsVChsb0%253D&md5=d1ad900aa5a45ba6ba3a5e9ccdcd96b3</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fjm100059d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100059d%26sid%3Dliteratum%253Aachs%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DCarr%26aufirst%3DM.%2BG.%26aulast%3DCallaghan%26aufirst%3DO.%26aulast%3DChessari%26aufirst%3DG.%26aulast%3DCongreve%26aufirst%3DM.%26aulast%3DCowan%26aufirst%3DS.%26aulast%3DCoyle%26aufirst%3DJ.%2BE.%26aulast%3DDownham%26aufirst%3DR.%26aulast%3DFigueroa%26aufirst%3DE.%26aulast%3DFrederickson%26aufirst%3DM.%26aulast%3DGraham%26aufirst%3DB.%26aulast%3DMcMenamin%26aufirst%3DR.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DM.%2BA.%26aulast%3DPatel%26aufirst%3DS.%26aulast%3DPhillips%26aufirst%3DT.%2BR.%26aulast%3DWilliams%26aufirst%3DG.%26aulast%3DWoodhead%26aufirst%3DA.%2BJ.%26aulast%3DWoolford%26aufirst%3DA.%2BJ.%26atitle%3DFragment-based%2520drug%2520discovery%2520applied%2520to%2520Hsp90.%2520Discovery%2520of%2520two%2520lead%2520series%2520with%2520high%2520ligand%2520efficiency%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D5942%26epage%3D5955%26doi%3D10.1021%2Fjm100059d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Bembenek, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tounge, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reynolds, C. H.</span><span> </span><span class="NLM_article-title">Ligand efficiency and fragment-based drug discovery</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">278</span><span class="NLM_x">–</span> <span class="NLM_lpage">283</span><span class="refDoi"> DOI: 10.1016/j.drudis.2008.11.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1016%2Fj.drudis.2008.11.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=19073276" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjt1Sht78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2009&pages=278-283&author=S.+D.+Bembenekauthor=B.+A.+Toungeauthor=C.+H.+Reynolds&title=Ligand+efficiency+and+fragment-based+drug+discovery&doi=10.1016%2Fj.drudis.2008.11.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand efficiency and fragment-based drug discovery</span></div><div class="casAuthors">Bembenek, Scott D.; Tounge, Brett A.; Reynolds, Charles H.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">5/6</span>),
    <span class="NLM_cas:pages">278-283</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The use of fragment-based drug discovery (FBDD) has increased in recent years since it is more likely to produce a better optimized compd. of lower mol. wt.  Ligand efficiency (LE) has become important for assessing fragments, HTS hits, and resulting optimized ligands.  LE is useful for comparing ligands of equal mol. wt., but is ineffective for comparisons of ligands of differing mol. wt.  LE has a strong dependence on mol. size, which has led us to develop a size-independent efficiency score termed fit quality.  Evaluating FBDD examples from the literature using LE and fit quality, we find that, in general, the LEs of starting fragments are greater than those of larger, more elaborated, structures.  Fit quality scores, however, tend to improve upon optimization of the fragments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrIWfRm1zWl87Vg90H21EOLACvtfcHk0lg0s1SB6sji9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjt1Sht78%253D&md5=55b35499fb26249d535bbe586f74f38f</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2008.11.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2008.11.007%26sid%3Dliteratum%253Aachs%26aulast%3DBembenek%26aufirst%3DS.%2BD.%26aulast%3DTounge%26aufirst%3DB.%2BA.%26aulast%3DReynolds%26aufirst%3DC.%2BH.%26atitle%3DLigand%2520efficiency%2520and%2520fragment-based%2520drug%2520discovery%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2009%26volume%3D14%26spage%3D278%26epage%3D283%26doi%3D10.1016%2Fj.drudis.2008.11.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Lipinski, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hopkins, A.</span><span> </span><span class="NLM_article-title">Navigating chemical space for biology and medicine</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">432</span><span class="NLM_x">, </span> <span class="NLM_fpage">855</span><span class="NLM_x">–</span> <span class="NLM_lpage">861</span><span class="refDoi"> DOI: 10.1038/nature03193</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1038%2Fnature03193" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=15602551" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVOht7jO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=432&publication_year=2004&pages=855-861&author=C.+Lipinskiauthor=A.+Hopkins&title=Navigating+chemical+space+for+biology+and+medicine&doi=10.1038%2Fnature03193"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Navigating chemical space for biology and medicine</span></div><div class="casAuthors">Lipinski, Christopher; Hopkins, Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">432</span>
        (<span class="NLM_cas:issue">7019</span>),
    <span class="NLM_cas:pages">855-861</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Despite over a century of applying org. synthesis to the search for drugs, the authors are still far from even a cursory examn. of the vast no. of possible small mols. that could be created.  Indeed, a thorough examn. of all 'chem. space' is practically impossible.  Given this, what are the best strategies for identifying small mols. that modulate biol. targets And how might such strategies differ, depending on whether the primary goal is to understand biol. systems or to develop potential drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3hQm7xaX5MLVg90H21EOLACvtfcHk0lg0s1SB6sji9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVOht7jO&md5=077237d55fa1225fd0aaee481e376172</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1038%2Fnature03193&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature03193%26sid%3Dliteratum%253Aachs%26aulast%3DLipinski%26aufirst%3DC.%26aulast%3DHopkins%26aufirst%3DA.%26atitle%3DNavigating%2520chemical%2520space%2520for%2520biology%2520and%2520medicine%26jtitle%3DNature%26date%3D2004%26volume%3D432%26spage%3D855%26epage%3D861%26doi%3D10.1038%2Fnature03193" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sterling, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mysinger, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bolstad, E. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coleman, R. G.</span><span> </span><span class="NLM_article-title">ZINC: a free tool to discover chemistry for biology</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">1757</span><span class="NLM_x">–</span> <span class="NLM_lpage">1768</span><span class="refDoi"> DOI: 10.1021/ci3001277</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci3001277" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BC38XmvFGnsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=1757-1768&author=J.+J.+Irwinauthor=T.+Sterlingauthor=M.+M.+Mysingerauthor=E.+S.+Bolstadauthor=R.+G.+Coleman&title=ZINC%3A+a+free+tool+to+discover+chemistry+for+biology&doi=10.1021%2Fci3001277"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">ZINC: A Free Tool to Discover Chemistry for Biology</span></div><div class="casAuthors">Irwin, John J.; Sterling, Teague; Mysinger, Michael M.; Bolstad, Erin S.; Coleman, Ryan G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1757-1768</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">ZINC is a free public resource for ligand discovery.  The database contains over twenty million com. available mols. in biol. relevant representations that may be downloaded in popular ready-to-dock formats and subsets.  The Web site also enables searches by structure, biol. activity, phys. property, vendor, catalog no., name, and CAS no.  Small custom subsets may be created, edited, shared, docked, downloaded, and conveyed to a vendor for purchase.  The database is maintained and curated for a high purchasing success rate and is freely available at zinc.docking.org.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotyhntxgvRpbVg90H21EOLACvtfcHk0lg0s1SB6sji9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmvFGnsrg%253D&md5=97f2ede64afc6b5e3ea2f279e38e32a0</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fci3001277&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci3001277%26sid%3Dliteratum%253Aachs%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DSterling%26aufirst%3DT.%26aulast%3DMysinger%26aufirst%3DM.%2BM.%26aulast%3DBolstad%26aufirst%3DE.%2BS.%26aulast%3DColeman%26aufirst%3DR.%2BG.%26atitle%3DZINC%253A%2520a%2520free%2520tool%2520to%2520discover%2520chemistry%2520for%2520biology%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D1757%26epage%3D1768%26doi%3D10.1021%2Fci3001277" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Mysinger, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Rapid context-dependent ligand desolvation in molecular docking</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">1561</span><span class="NLM_x">–</span> <span class="NLM_lpage">1573</span><span class="refDoi"> DOI: 10.1021/ci100214a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100214a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVGrsLvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=1561-1573&author=M.+M.+Mysingerauthor=B.+K.+Shoichet&title=Rapid+context-dependent+ligand+desolvation+in+molecular+docking&doi=10.1021%2Fci100214a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid Context-Dependent Ligand Desolvation in Molecular Docking</span></div><div class="casAuthors">Mysinger, Michael M.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1561-1573</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In structure-based screens for new ligands, a mol. docking algorithm must rapidly score many mols. in multiple configurations, accounting for both the ligand's interactions with receptor and its competing interactions with solvent.  Here the authors explore a context-dependent ligand desolvation scoring term for mol. docking.  The authors relate the Generalized-Born effective Born radii for every ligand atom to a fractional desolvation and then use this fraction to scale an atom-by-atom decompn. of the full transfer free energy.  The fractional desolvation is precomputed on a scoring grid by numerically integrating over the vol. of receptor proximal to a ligand atom, weighted by distance.  To test this method's performance, the authors dock ligands vs. property-matched decoys over 40 DUD targets.  Context-dependent desolvation better enriches ligands compared to both the raw full transfer free energy penalty and compared to ignoring desolvation altogether, though the improvement is modest.  More compellingly, the new method improves docking performance across receptor types.  Thus, whereas entirely ignoring desolvation works best for charged sites and overpenalizing with full desolvation works well for neutral sites, the phys. more correct context-dependent ligand desolvation is competitive across both types of targets.  The method also reliably discriminates ligands from highly charged mols., where ignoring desolvation performs poorly.  Since this context-dependent ligand desolvation may be precalcd., it improves docking reliability with minimal cost to calcn. time and may be readily incorporated into any physics-based docking program.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtPnS0XiKpYLVg90H21EOLACvtfcHk0lgN3MC5Ej50Hw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVGrsLvI&md5=5124d2c4618e07b4f1e41609c9e61362</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fci100214a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100214a%26sid%3Dliteratum%253Aachs%26aulast%3DMysinger%26aufirst%3DM.%2BM.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DRapid%2520context-dependent%2520ligand%2520desolvation%2520in%2520molecular%2520docking%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D1561%26epage%3D1573%26doi%3D10.1021%2Fci100214a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Meng, E. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuntz, I. D.</span><span> </span><span class="NLM_article-title">Automated Docking with Grid-Based Energy Evaluation</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1992</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">505</span><span class="NLM_x">–</span> <span class="NLM_lpage">524</span><span class="refDoi"> DOI: 10.1002/jcc.540130412</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1002%2Fjcc.540130412" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADyaK38Xit1Omt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=1992&pages=505-524&author=E.+C.+Mengauthor=B.+K.+Shoichetauthor=I.+D.+Kuntz&title=Automated+Docking+with+Grid-Based+Energy+Evaluation&doi=10.1002%2Fjcc.540130412"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Automated docking with grid-based energy evaluation</span></div><div class="casAuthors">Meng, Elaine C.; Shoichet, Brian K.; Kuntz, Irwin D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">505-24</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    </div><div class="casAbstract">The ability to generate feasible binding orientations of a small mol. within a site of known structure is important for ligand design.  The authors present a method that combines a rapid, geometric docking algorithm with the evaluation of mol. mechanics interaction energies.  The computational costs of evaluation are minimal because the authors precalc. the receptor-dependent terms in the potential function at points on a three-dimensional grid.  In four test cases where the components of crystallog. detd. complexes are redocked, the "force field" score correctly identifies the family of orientations closest to the exptl. binding geometry.  Scoring functions that consider only steric factors or only electrostatic factors are less successful.  The force field function will play an important role in efforts to search databases for potential lead compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKAw7Zj_QFSbVg90H21EOLACvtfcHk0lgN3MC5Ej50Hw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38Xit1Omt7c%253D&md5=f472fffa0c9a61652b4c20e4dbbba69e</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1002%2Fjcc.540130412&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.540130412%26sid%3Dliteratum%253Aachs%26aulast%3DMeng%26aufirst%3DE.%2BC.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DKuntz%26aufirst%3DI.%2BD.%26atitle%3DAutomated%2520Docking%2520with%2520Grid-Based%2520Energy%2520Evaluation%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1992%26volume%3D13%26spage%3D505%26epage%3D524%26doi%3D10.1002%2Fjcc.540130412" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">Gilson, M. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Honig, B.</span><span> </span><span class="NLM_article-title">The inclusion of electrostatic hydration energies in molecular mechanics calculations</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1991</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">5</span><span class="NLM_x">–</span> <span class="NLM_lpage">20</span><span class="refDoi"> DOI: 10.1007/BF00173467</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1007%2FBF00173467" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=2072125" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADyaK3MXlt1Shtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=1991&pages=5-20&author=M.+K.+Gilsonauthor=B.+Honig&title=The+inclusion+of+electrostatic+hydration+energies+in+molecular+mechanics+calculations&doi=10.1007%2FBF00173467"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">The inclusion of electrostatic hydration energies in molecular mechanics calculations</span></div><div class="casAuthors">Gilson, Michael K.; Honig, Barry</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">5-20</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    </div><div class="casAbstract">The problem of including solvent effects in mol. mechanics calcns. is discussed.  It is argued that the neglect of charge-solvent (solvation) interactions can introduce significant errors.  The finite difference Poisson-Boltzmann (FDPB) method for calcg. electrostatic interactions is summarized and is used as a basis for introducing a new pairwise energy term which accounts for charge-solvent interactions.  This term acts between all pairs of atoms usually considered in mol. mechanics calcns. and can be easily incorporated into existing force fields.  As an example, a parameterization is developed for the CHARMm force field and the results compared to the predictions of the FDPB method.  An approach to the realistic incorporation of solvent screening into force fields is also outlined.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhzyjxAJyAZrVg90H21EOLACvtfcHk0lgN3MC5Ej50Hw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXlt1Shtr0%253D&md5=844bc05c190cb622320aeb0564d922a7</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1007%2FBF00173467&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF00173467%26sid%3Dliteratum%253Aachs%26aulast%3DGilson%26aufirst%3DM.%2BK.%26aulast%3DHonig%26aufirst%3DB.%26atitle%3DThe%2520inclusion%2520of%2520electrostatic%2520hydration%2520energies%2520in%2520molecular%2520mechanics%2520calculations%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D1991%26volume%3D5%26spage%3D5%26epage%3D20%26doi%3D10.1007%2FBF00173467" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raushel, F. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Virtual screening against metalloenzymes for inhibitors and substrates</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">12316</span><span class="NLM_x">–</span> <span class="NLM_lpage">12328</span><span class="refDoi"> DOI: 10.1021/bi050801k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi050801k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BD2MXovFWhtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2005&pages=12316-12328&author=J.+J.+Irwinauthor=F.+M.+Raushelauthor=B.+K.+Shoichet&title=Virtual+screening+against+metalloenzymes+for+inhibitors+and+substrates&doi=10.1021%2Fbi050801k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual Screening against Metalloenzymes for Inhibitors and Substrates</span></div><div class="casAuthors">Irwin, John J.; Raushel, Frank M.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">37</span>),
    <span class="NLM_cas:pages">12316-12328</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. docking uses the three-dimensional structure of a receptor to screen databases of small mols. for potential ligands, often based on energetic complementarity.  For many docking scoring functions, which calc. nonbonded interactions, metalloenzymes are challenging because of the partial covalent nature of metal-ligand interactions.  To investigate how well mol. docking can identify potential ligands of metalloenzymes using a "std." scoring function, we have docked the MDL Drug Data Report (MDDR), a functionally annotated database of 95 000 small mols., against the X-ray crystal structures of five metalloenzymes.  These enzymes included three zinc proteases, the nickel analog of an iron enzyme, and a molybdenum metalloenzyme.  The ability of the docking program to retrospectively enrich the annotated ligands as high-scoring hits for each enzyme and to calc. proper geometries was evaluated.  In all five systems, the annotated ligands within the MDDR were enriched at least 20 times over random.  To test the approach prospectively, a sixth target, the zinc β-lactamase from Bacteroides fragilis, was screened against the fragment-like subset of the ZINC database.  We purchased and tested 15 compds. from among the top 50 top-ranked ligands from docking, and found 5 inhibitors with apparent Ki values less than 120 μM, the best of which was 2 μM.  A more ambitious test still was predicting actual substrates for a seventh target, a Zn-dependent phosphotriesterase from Pseudomonas diminuta.  Screening the Available Chems. Directory (ACD) identified 25 thiophosphate esters as potential substrates within the top 100 ranked compds.  Eight of these, all previously uncharacterized for this enzyme, were acquired and tested, and all were confirmed exptl. as substrates.  These results suggest that a simple, noncovalent scoring function may be used to identify inhibitors of at least some metalloenzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhSirdbjLQSrVg90H21EOLACvtfcHk0lgN3MC5Ej50Hw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXovFWhtrc%253D&md5=f0bd25b13e227ed670ad2a2abf25abef</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fbi050801k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi050801k%26sid%3Dliteratum%253Aachs%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DRaushel%26aufirst%3DF.%2BM.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DVirtual%2520screening%2520against%2520metalloenzymes%2520for%2520inhibitors%2520and%2520substrates%26jtitle%3DBiochemistry%26date%3D2005%26volume%3D44%26spage%3D12316%26epage%3D12328%26doi%3D10.1021%2Fbi050801k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group">Babaoglu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeonov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nelson, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomas, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cancian, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Costi, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maltby, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jadhav, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Inglese, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Austin, C. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Comprehensive mechanistic analysis of hits from high-throughput and docking screens against beta-lactamase</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">2502</span><span class="NLM_x">–</span> <span class="NLM_lpage">2511</span><span class="refDoi"> DOI: 10.1021/jm701500e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm701500e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjtFSksb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=2502-2511&author=K.+Babaogluauthor=A.+Simeonovauthor=J.+J.+Irwinauthor=M.+E.+Nelsonauthor=B.+Fengauthor=C.+J.+Thomasauthor=L.+Cancianauthor=M.+P.+Costiauthor=D.+A.+Maltbyauthor=A.+Jadhavauthor=J.+Ingleseauthor=C.+P.+Austinauthor=B.+K.+Shoichet&title=Comprehensive+mechanistic+analysis+of+hits+from+high-throughput+and+docking+screens+against+beta-lactamase&doi=10.1021%2Fjm701500e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive Mechanistic Analysis of Hits from High-Throughput and Docking Screens against β-Lactamase</span></div><div class="casAuthors">Babaoglu, Kerim; Simeonov, Anton; Irwin, John J.; Nelson, Michael E.; Feng, Brian; Thomas, Craig J.; Cancian, Laura; Costi, M. Paola; Maltby, David A.; Jadhav, Ajit; Inglese, James; Austin, Christopher P.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2502-2511</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">High-throughput screening (HTS) is widely used in drug discovery.  Esp. for screens of unbiased libraries, false positives can dominate "hit lists"; their origins are much debated.  Here we det. the mechanism of every active hit from a screen of 70,563 unbiased mols. against β-lactamase using quant. HTS (qHTS).  Of the 1274 initial inhibitors, 95% were detergent-sensitive and were classified as aggregators.  Among the 70 remaining were 25 potent, covalent-acting β-lactams.  Mass spectra, counter-screens, and crystallog. identified 12 as promiscuous covalent inhibitors.  The remaining 33 were either aggregators or irreproducible.  No specific reversible inhibitors were found.  We turned to mol. docking to prioritize mols. from the same library for testing at higher concns.  Of 16 tested, 2 were modest inhibitors.  Subsequent X-ray structures corresponded to the docking prediction.  Analog synthesis improved affinity to 8 μM.  These results suggest that it may be the phys. behavior of org. mols., not their reactivity, that accounts for most screening artifacts.  Structure-based methods may prioritize weak-but-novel chemotypes in unbiased library screens.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNtnaCegS8aLVg90H21EOLACvtfcHk0lhAlequ3UGHPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjtFSksb0%253D&md5=7736429aeafc6b17a5d0182e24df0dd4</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Fjm701500e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm701500e%26sid%3Dliteratum%253Aachs%26aulast%3DBabaoglu%26aufirst%3DK.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DNelson%26aufirst%3DM.%2BE.%26aulast%3DFeng%26aufirst%3DB.%26aulast%3DThomas%26aufirst%3DC.%2BJ.%26aulast%3DCancian%26aufirst%3DL.%26aulast%3DCosti%26aufirst%3DM.%2BP.%26aulast%3DMaltby%26aufirst%3DD.%2BA.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DInglese%26aufirst%3DJ.%26aulast%3DAustin%26aufirst%3DC.%2BP.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DComprehensive%2520mechanistic%2520analysis%2520of%2520hits%2520from%2520high-throughput%2520and%2520docking%2520screens%2520against%2520beta-lactamase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D2502%26epage%3D2511%26doi%3D10.1021%2Fjm701500e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group">Mysinger, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weiss, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ziarek, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gravel, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doak, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karpiak, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heveker, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Volkman, B. F.</span><span> </span><span class="NLM_article-title">Structure-based ligand discovery for the protein-protein interface of chemokine receptor CXCR4</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">109</span><span class="NLM_x">, </span> <span class="NLM_fpage">5517</span><span class="NLM_x">–</span> <span class="NLM_lpage">5522</span><span class="refDoi"> DOI: 10.1073/pnas.1120431109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1073%2Fpnas.1120431109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=22431600" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BC38XlslOjtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=5517-5522&author=M.+M.+Mysingerauthor=D.+R.+Weissauthor=J.+J.+Ziarekauthor=S.+Gravelauthor=A.+K.+Doakauthor=J.+Karpiakauthor=N.+Hevekerauthor=B.+K.+Shoichetauthor=B.+F.+Volkman&title=Structure-based+ligand+discovery+for+the+protein-protein+interface+of+chemokine+receptor+CXCR4&doi=10.1073%2Fpnas.1120431109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based ligand discovery for the protein-protein interface of chemokine receptor CXCR4</span></div><div class="casAuthors">Mysinger, Michael M.; Weiss, Dahlia R.; Ziarek, Joshua J.; Gravel, Stephanie; Doak, Allison K.; Karpiak, Joel; Heveker, Nikolaus; Shoichet, Brian K.; Volkman, Brian F.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">5517-5522, S5517/1-S5517/28</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">G-protein-coupled receptors (GPCRs) are key signaling mols. and are intensely studied.  Whereas GPCRs recognizing small-mols. have been successfully targeted for drug discovery, protein-recognizing GPCRs, such as the chemokine receptors, claim few drugs or even useful small mol. reagents.  This reflects both the difficulties that attend protein-protein interface inhibitor discovery, and the lack of structures for these targets.  Imminent structure detn. of chemokine receptor CXCR4 motivated docking screens for new ligands against a homol. model and subsequently the crystal structure.  More than 3 million mols. were docked against the model and then against the crystal structure; 24 and 23 high-scoring compds. from the resp. screens were tested exptl.  Docking against the model yielded only one antagonist, which resembled known ligands and lacked specificity, whereas the crystal structure docking yielded four that were dissimilar to previously known scaffolds and apparently specific.  Intriguingly, several were potent and relatively small, with IC50 values as low as 306 nM, ligand efficiencies as high as 0.36, and with efficacy in cellular chemotaxis.  The potency and efficiency of these mols. has few precedents among protein-protein interface inhibitors, and supports structure-based efforts to discover leads for chemokine GPCRs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqeJKWh0aH7FLVg90H21EOLACvtfcHk0lhAlequ3UGHPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XlslOjtrg%253D&md5=762616e6eb0eebc0d983964d71499e75</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1120431109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1120431109%26sid%3Dliteratum%253Aachs%26aulast%3DMysinger%26aufirst%3DM.%2BM.%26aulast%3DWeiss%26aufirst%3DD.%2BR.%26aulast%3DZiarek%26aufirst%3DJ.%2BJ.%26aulast%3DGravel%26aufirst%3DS.%26aulast%3DDoak%26aufirst%3DA.%2BK.%26aulast%3DKarpiak%26aufirst%3DJ.%26aulast%3DHeveker%26aufirst%3DN.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DVolkman%26aufirst%3DB.%2BF.%26atitle%3DStructure-based%2520ligand%2520discovery%2520for%2520the%2520protein-protein%2520interface%2520of%2520chemokine%2520receptor%2520CXCR4%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2012%26volume%3D109%26spage%3D5517%26epage%3D5522%26doi%3D10.1073%2Fpnas.1120431109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group">Peng, Z. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gillespie, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weisel, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">So, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">So, W. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kondru, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Narayanan, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hermann, J. C.</span><span> </span><span class="NLM_article-title">A Crowd-Based Process and Tool for HTS Hit Triage</span> <span class="citation_source-journal">Mol. Inf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x">, </span> <span class="NLM_fpage">337</span><span class="NLM_x">–</span> <span class="NLM_lpage">345</span><span class="refDoi"> DOI: 10.1002/minf.201200154</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1002%2Fminf.201200154" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=27481590" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BC3sXls1SjtL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2013&pages=337-345&author=Z.+W.+Pengauthor=P.+Gillespieauthor=M.+Weiselauthor=S.+S.+Soauthor=W.+V.+Soauthor=R.+Kondruauthor=A.+Narayananauthor=J.+C.+Hermann&title=A+Crowd-Based+Process+and+Tool+for+HTS+Hit+Triage&doi=10.1002%2Fminf.201200154"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">A Crowd-Based Process and Tool for HTS Hit Triage</span></div><div class="casAuthors">Peng, Zhengwei; Gillespie, Paul; Weisel, Martin; So, Sung-Sau; So, W. Venus; Kondru, Rama; Narayanan, Arjun; Hermann, Johannes C.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Informatics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">337-345</span>CODEN:
                <span class="NLM_cas:coden">MIONBS</span>;
        ISSN:<span class="NLM_cas:issn">1868-1743</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">This article presents a high-throughput screening (HTS) hit triage process based on a wisdom of crowds approach.  Participants provide votes and comments on suggested compds. of confirmed HTS hits through a web voting tool in an independent and decentralized way.  Insights gained and lessons learned through the actual application of this process and the assocd. tool on several drug discovery projects will be discussed.  Finally this web voting tool could be easily modified and adapted for any crowd-based decision making for compd. selections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5La7mNfdDJLVg90H21EOLACvtfcHk0lhAlequ3UGHPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXls1SjtL0%253D&md5=ad8166a0740aa6e118afa659e2de1f94</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1002%2Fminf.201200154&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fminf.201200154%26sid%3Dliteratum%253Aachs%26aulast%3DPeng%26aufirst%3DZ.%2BW.%26aulast%3DGillespie%26aufirst%3DP.%26aulast%3DWeisel%26aufirst%3DM.%26aulast%3DSo%26aufirst%3DS.%2BS.%26aulast%3DSo%26aufirst%3DW.%2BV.%26aulast%3DKondru%26aufirst%3DR.%26aulast%3DNarayanan%26aufirst%3DA.%26aulast%3DHermann%26aufirst%3DJ.%2BC.%26atitle%3DA%2520Crowd-Based%2520Process%2520and%2520Tool%2520for%2520HTS%2520Hit%2520Triage%26jtitle%3DMol.%2520Inf.%26date%3D2013%26volume%3D32%26spage%3D337%26epage%3D345%26doi%3D10.1002%2Fminf.201200154" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group">Feng, B. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeonov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jadhav, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Babaoglu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Inglese, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Austin, C. P.</span><span> </span><span class="NLM_article-title">A high-throughput screen for aggregation-based inhibition in a large compound library</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">2385</span><span class="NLM_x">–</span> <span class="NLM_lpage">2390</span><span class="refDoi"> DOI: 10.1021/jm061317y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm061317y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BD2sXksVSlsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2007&pages=2385-2390&author=B.+Y.+Fengauthor=A.+Simeonovauthor=A.+Jadhavauthor=K.+Babaogluauthor=J.+Ingleseauthor=B.+K.+Shoichetauthor=C.+P.+Austin&title=A+high-throughput+screen+for+aggregation-based+inhibition+in+a+large+compound+library&doi=10.1021%2Fjm061317y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">A High-Throughput Screen for Aggregation-Based Inhibition in a Large Compound Library</span></div><div class="casAuthors">Feng, Brian Y.; Simeonov, Anton; Jadhav, Ajit; Babaoglu, Kerim; Inglese, James; Shoichet, Brian K.; Austin, Christopher P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2385-2390</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">High-throughput screening (HTS) is the primary technique for new lead identification in drug discovery and chem. biol.  Unfortunately, it is susceptible to false-pos. hits.  One common mechanism for such false-positives is the congregation of org. mols. into colloidal aggregates, which nonspecifically inhibit enzymes.  To both evaluate the feasibility of large-scale identification of aggregate-based inhibition and quantify its prevalence among screening hits, the authors tested 70 563 mols. from the National Institutes of Health Chem. Genomics Center (NCGC) library for detergent-sensitive inhibition.  Each mol. was screened in at least seven concns., such that dose-response curves were obtained for all mols. in the library.  There were 1274 inhibitors identified in total, of which 1204 were unambiguously detergent-sensitive.  The authors identified these as aggregate-based inhibitors.  Thirty-one library mols. were independently purchased and retested in secondary low-throughput expts.; 29 of these were confirmed as either aggregators or nonaggregators, as appropriate.  Finally, with the dose-response information collected for every compd., the authors could examine the correlation between aggregate-based inhibition and steep dose-response curves.  Three key results emerge from this study: first, detergent-dependent identification of aggregate-based inhibition is feasible on the large scale.  Second, 95% of the actives obtained in this screen are aggregate-based inhibitors.  Third, aggregate-based inhibition is correlated with steep dose-response curves, although not absolutely.  The results of this screen are being released publicly via the PubChem database.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpn9eDfL7UswbVg90H21EOLACvtfcHk0ljtIfr3Rx4ZaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXksVSlsrw%253D&md5=33399bcd4998faf830f54a6f0d1f636a</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Fjm061317y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm061317y%26sid%3Dliteratum%253Aachs%26aulast%3DFeng%26aufirst%3DB.%2BY.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DBabaoglu%26aufirst%3DK.%26aulast%3DInglese%26aufirst%3DJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DAustin%26aufirst%3DC.%2BP.%26atitle%3DA%2520high-throughput%2520screen%2520for%2520aggregation-based%2520inhibition%2520in%2520a%2520large%2520compound%2520library%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2007%26volume%3D50%26spage%3D2385%26epage%3D2390%26doi%3D10.1021%2Fjm061317y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Screening in a spirit haunted world</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">607</span><span class="NLM_x">–</span> <span class="NLM_lpage">615</span><span class="refDoi"> DOI: 10.1016/j.drudis.2006.05.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1016%2Fj.drudis.2006.05.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=16793529" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BD28XlvFGqtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2006&pages=607-615&author=B.+K.+Shoichet&title=Screening+in+a+spirit+haunted+world&doi=10.1016%2Fj.drudis.2006.05.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Screening in a spirit haunted world</span></div><div class="casAuthors">Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">13 & 14</span>),
    <span class="NLM_cas:pages">607-615</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  High-throughput screening (HTS) campaigns can be dominated by hits that ultimately turn out to be non-drug-like.  These 'nuisance' compds. often behave strangely, with steep dose-response curves, absence of clear structure-activity relationships, and high sensitivity to assay conditions.  Several mechanisms contribute to these artifacts, including chem. reactive mols., those that absorb light in assays and those that affect redox conditions.  One of the most common mechanisms behind artifactual inhibition is discussed in this review: at micromolar concns. org. mols. can aggregate to form particles in aq. buffers, and these aggregates can sequester and thereby inhibit protein targets.  Aggregation-based inhibition is baffling from a chem. perspective, but viewed biophys. such behavior is expected.  The range of mols. that behave this way, their rapid detection in a screening environment and their possible biol. implications will be considered here.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzoEkwI2izy7Vg90H21EOLACvtfcHk0ljtIfr3Rx4ZaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XlvFGqtLk%253D&md5=08d22b6b06215ac435824b7664b3674c</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2006.05.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2006.05.014%26sid%3Dliteratum%253Aachs%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DScreening%2520in%2520a%2520spirit%2520haunted%2520world%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2006%26volume%3D11%26spage%3D607%26epage%3D615%26doi%3D10.1016%2Fj.drudis.2006.05.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group">Giannetti, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koch, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Browner, M. F.</span><span> </span><span class="NLM_article-title">Surface plasmon resonance based assay for the detection and characterization of promiscuous inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">574</span><span class="NLM_x">–</span> <span class="NLM_lpage">580</span><span class="refDoi"> DOI: 10.1021/jm700952v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm700952v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjt1entg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=574-580&author=A.+M.+Giannettiauthor=B.+D.+Kochauthor=M.+F.+Browner&title=Surface+plasmon+resonance+based+assay+for+the+detection+and+characterization+of+promiscuous+inhibitors&doi=10.1021%2Fjm700952v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Surface Plasmon Resonance Based Assay for the Detection and Characterization of Promiscuous Inhibitors</span></div><div class="casAuthors">Giannetti, Anthony M.; Koch, Bruce D.; Browner, Michelle F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">574-580</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Promiscuous binders achieve enzyme inhibition using a nonspecific aggregation-type binding mechanism to proteins.  These compds. are a source of false-pos. hits in biochem. inhibition assays and should be removed from screening hit lists because they are not good candidates to initiate medicinal chem. programs.  The authors introduce a robust approach to identify these mols. early in the lead generation process using real time surface plasmon resonance based biosensors to observe the behavior of the binding interactions between promiscuous compds. and proteins.  Furthermore, the time resoln. of the assay reveals a no. of distinct mechanisms that promiscuous compds. employ to inhibit enzyme function and indicate that the type of mechanism can vary depending on the protein target.  A classification scheme for these compds. is presented that can be used to rapidly characterize the hits from high-throughput screens and eliminate compds. with a nonspecific mechanism of inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVTh2kYyROf7Vg90H21EOLACvtfcHk0ljtIfr3Rx4ZaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjt1entg%253D%253D&md5=b2869c5be1cd6f455503f8e812ed6f23</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Fjm700952v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm700952v%26sid%3Dliteratum%253Aachs%26aulast%3DGiannetti%26aufirst%3DA.%2BM.%26aulast%3DKoch%26aufirst%3DB.%2BD.%26aulast%3DBrowner%26aufirst%3DM.%2BF.%26atitle%3DSurface%2520plasmon%2520resonance%2520based%2520assay%2520for%2520the%2520detection%2520and%2520characterization%2520of%2520promiscuous%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D574%26epage%3D580%26doi%3D10.1021%2Fjm700952v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group">Chowdhury, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yeoh, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tian, Y. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hillringhaus, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bagg, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rose, N. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leung, I. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, X. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woon, E. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDonough, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">King, O. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clifton, I. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klose, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Claridge, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ratcliffe, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawamura, A.</span><span> </span><span class="NLM_article-title">The oncometabolite 2-hydroxyglutarate inhibits histone lysine demethylases</span> <span class="citation_source-journal">EMBO Rep.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">463</span><span class="NLM_x">–</span> <span class="NLM_lpage">469</span><span class="refDoi"> DOI: 10.1038/embor.2011.43</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1038%2Fembor.2011.43" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=21460794" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFahs74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=463-469&author=R.+Chowdhuryauthor=K.+K.+Yeohauthor=Y.+M.+Tianauthor=L.+Hillringhausauthor=E.+A.+Baggauthor=N.+R.+Roseauthor=I.+K.+Leungauthor=X.+S.+Liauthor=E.+C.+Woonauthor=M.+Yangauthor=M.+A.+McDonoughauthor=O.+N.+Kingauthor=I.+J.+Cliftonauthor=R.+J.+Kloseauthor=T.+D.+Claridgeauthor=P.+J.+Ratcliffeauthor=C.+J.+Schofieldauthor=A.+Kawamura&title=The+oncometabolite+2-hydroxyglutarate+inhibits+histone+lysine+demethylases&doi=10.1038%2Fembor.2011.43"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">The oncometabolite 2-hydroxyglutarate inhibits histone lysine demethylases</span></div><div class="casAuthors">Chowdhury, Rasheduzzaman; Yeoh, Kar-Kheng; Tian, Ya-Min; Hillringhaus, Lars; Bagg, Eleanor A.; Rose, Nathan R.; Leung, Ivanhoe K.-H.; Li, Xuan-S.; Woon, Esther C.-Y.; Yang, Ming; McDonough, Michael A.; King, Oliver N.; Clifton, Ian J.; Klose, Robert J.; Claridge, Timothy D. W.; Ratcliffe, Peter J.; Schofield, Christopher J.; Kawamura, Akane</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Reports</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">463-469</span>CODEN:
                <span class="NLM_cas:coden">ERMEAX</span>;
        ISSN:<span class="NLM_cas:issn">1469-221X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Mutations in isocitrate dehydrogenases (IDHs) have a gain-of-function effect leading to R(-)-2-hydroxyglutarate (R-2HG) accumulation.  By using biochem., structural and cellular assays, we show that either or both R- and S-2HG inhibit 2-oxoglutarate (2OG)-dependent oxygenases with varying potencies.  Half-maximal inhibitory concn. (IC50) values for the R-form of 2HG varied from approx. 25 μM for the histone N.vepsiln.-lysine demethylase JMJD2A to more than 5 mM for the hypoxia-inducible factor (HIF) prolyl hydroxylase.  The results indicate that candidate oncogenic pathways in IDH-assocd. malignancy should include those that are regulated by other 2OG oxygenases than HIF hydroxylases, in particular those involving the regulation of histone methylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGHxuB6sKkTbVg90H21EOLACvtfcHk0liHtbi272hR-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFahs74%253D&md5=1b862b1337346af6b796093cea34056f</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1038%2Fembor.2011.43&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fembor.2011.43%26sid%3Dliteratum%253Aachs%26aulast%3DChowdhury%26aufirst%3DR.%26aulast%3DYeoh%26aufirst%3DK.%2BK.%26aulast%3DTian%26aufirst%3DY.%2BM.%26aulast%3DHillringhaus%26aufirst%3DL.%26aulast%3DBagg%26aufirst%3DE.%2BA.%26aulast%3DRose%26aufirst%3DN.%2BR.%26aulast%3DLeung%26aufirst%3DI.%2BK.%26aulast%3DLi%26aufirst%3DX.%2BS.%26aulast%3DWoon%26aufirst%3DE.%2BC.%26aulast%3DYang%26aufirst%3DM.%26aulast%3DMcDonough%26aufirst%3DM.%2BA.%26aulast%3DKing%26aufirst%3DO.%2BN.%26aulast%3DClifton%26aufirst%3DI.%2BJ.%26aulast%3DKlose%26aufirst%3DR.%2BJ.%26aulast%3DClaridge%26aufirst%3DT.%2BD.%26aulast%3DRatcliffe%26aufirst%3DP.%2BJ.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26aulast%3DKawamura%26aufirst%3DA.%26atitle%3DThe%2520oncometabolite%25202-hydroxyglutarate%2520inhibits%2520histone%2520lysine%2520demethylases%26jtitle%3DEMBO%2520Rep.%26date%3D2011%26volume%3D12%26spage%3D463%26epage%3D469%26doi%3D10.1038%2Fembor.2011.43" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group">Rose, N. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woon, E. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kingham, G. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">King, O. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mecinovic, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clifton, I. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ng, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Talib-Hardy, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oppermann, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDonough, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span> </span><span class="NLM_article-title">Selective inhibitors of the JMJD2 histone demethylases: combined nondenaturing mass spectrometric screening and crystallographic approaches</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">1810</span><span class="NLM_x">–</span> <span class="NLM_lpage">1818</span><span class="refDoi"> DOI: 10.1021/jm901680b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901680b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BC3cXnvFWnsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=1810-1818&author=N.+R.+Roseauthor=E.+C.+Woonauthor=G.+L.+Kinghamauthor=O.+N.+Kingauthor=J.+Mecinovicauthor=I.+J.+Cliftonauthor=S.+S.+Ngauthor=J.+Talib-Hardyauthor=U.+Oppermannauthor=M.+A.+McDonoughauthor=C.+J.+Schofield&title=Selective+inhibitors+of+the+JMJD2+histone+demethylases%3A+combined+nondenaturing+mass+spectrometric+screening+and+crystallographic+approaches&doi=10.1021%2Fjm901680b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Inhibitors of the JMJD2 Histone Demethylases: Combined Nondenaturing Mass Spectrometric Screening and Crystallographic Approaches</span></div><div class="casAuthors">Rose, Nathan R.; Woon, Esther C. Y.; Kingham, Guy L.; King, Oliver N. F.; Mecinovic, Jasmin; Clifton, Ian J.; Ng, Stanley S.; Talib-Hardy, Jobina; Oppermann, Udo; McDonough, Michael A.; Schofield, Christopher J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1810-1818</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Ferrous ion and 2-oxoglutarate (2OG) oxygenases catalyze the demethylation of Nε-methylated lysine residues in histones.  Here, studies on the inhibition of the JMJD2 subfamily of histone demethylases are reported, employing binding analyses by non-denaturing mass spectrometry (MS), dynamic combinatorial chem. coupled to MS, turnover assays, and crystallog.  The results of initial binding and inhibition assays directed the prodn. and anal. of a set of N-oxalyl-D-tyrosine derivs. to explore the extent of a sub-pocket at the JMJD2 active site.  Some of the inhibitors were shown to be selective for JMJD2 over the hypoxia-inducible factor prolyl hydroxylase PHD2.  A crystal structure of JMJD2A in complex with one of the potent inhibitors was obtained; modeling other inhibitors based on this structure predicts interactions that enable improved inhibition for some compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoa6dzV_ed-qrVg90H21EOLACvtfcHk0liHtbi272hR-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXnvFWnsQ%253D%253D&md5=3eb3c36d91f46193946f64288ea9ccae</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Fjm901680b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901680b%26sid%3Dliteratum%253Aachs%26aulast%3DRose%26aufirst%3DN.%2BR.%26aulast%3DWoon%26aufirst%3DE.%2BC.%26aulast%3DKingham%26aufirst%3DG.%2BL.%26aulast%3DKing%26aufirst%3DO.%2BN.%26aulast%3DMecinovic%26aufirst%3DJ.%26aulast%3DClifton%26aufirst%3DI.%2BJ.%26aulast%3DNg%26aufirst%3DS.%2BS.%26aulast%3DTalib-Hardy%26aufirst%3DJ.%26aulast%3DOppermann%26aufirst%3DU.%26aulast%3DMcDonough%26aufirst%3DM.%2BA.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26atitle%3DSelective%2520inhibitors%2520of%2520the%2520JMJD2%2520histone%2520demethylases%253A%2520combined%2520nondenaturing%2520mass%2520spectrometric%2520screening%2520and%2520crystallographic%2520approaches%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D1810%26epage%3D1818%26doi%3D10.1021%2Fjm901680b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group">Chang, K. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">King, O. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tumber, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woon, E. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heightman, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDonough, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rose, N. R.</span><span> </span><span class="NLM_article-title">Inhibition of histone demethylases by 4-carboxy-2,2′-bipyridyl compounds</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">759</span><span class="NLM_x">–</span> <span class="NLM_lpage">764</span><span class="refDoi"> DOI: 10.1002/cmdc.201100026</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1002%2Fcmdc.201100026" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=759-764&author=K.+H.+Changauthor=O.+N.+Kingauthor=A.+Tumberauthor=E.+C.+Woonauthor=T.+D.+Heightmanauthor=M.+A.+McDonoughauthor=C.+J.+Schofieldauthor=N.+R.+Rose&title=Inhibition+of+histone+demethylases+by+4-carboxy-2%2C2%E2%80%B2-bipyridyl+compounds&doi=10.1002%2Fcmdc.201100026"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201100026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201100026%26sid%3Dliteratum%253Aachs%26aulast%3DChang%26aufirst%3DK.%2BH.%26aulast%3DKing%26aufirst%3DO.%2BN.%26aulast%3DTumber%26aufirst%3DA.%26aulast%3DWoon%26aufirst%3DE.%2BC.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DMcDonough%26aufirst%3DM.%2BA.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26aulast%3DRose%26aufirst%3DN.%2BR.%26atitle%3DInhibition%2520of%2520histone%2520demethylases%2520by%25204-carboxy-2%252C2%25E2%2580%25B2-bipyridyl%2520compounds%26jtitle%3DChemMedChem%26date%3D2011%26volume%3D6%26spage%3D759%26epage%3D764%26doi%3D10.1002%2Fcmdc.201100026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group">Ng, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kavanagh, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDonough, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Butler, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pilka, E. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lienard, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bray, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Savitsky, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gileadi, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">von Delft, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rose, N. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Offer, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scheinost, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borowski, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sundstrom, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oppermann, U.</span><span> </span><span class="NLM_article-title">Crystal structures of histone demethylase JMJD2A reveal basis for substrate specificity</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">448</span><span class="NLM_x">, </span> <span class="NLM_fpage">87</span><span class="NLM_x">–</span> <span class="NLM_lpage">91</span><span class="refDoi"> DOI: 10.1038/nature05971</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1038%2Fnature05971" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=17589501" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnt1ent7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=448&publication_year=2007&pages=87-91&author=S.+S.+Ngauthor=K.+L.+Kavanaghauthor=M.+A.+McDonoughauthor=D.+Butlerauthor=E.+S.+Pilkaauthor=B.+M.+Lienardauthor=J.+E.+Brayauthor=P.+Savitskyauthor=O.+Gileadiauthor=F.+von+Delftauthor=N.+R.+Roseauthor=J.+Offerauthor=J.+C.+Scheinostauthor=T.+Borowskiauthor=M.+Sundstromauthor=C.+J.+Schofieldauthor=U.+Oppermann&title=Crystal+structures+of+histone+demethylase+JMJD2A+reveal+basis+for+substrate+specificity&doi=10.1038%2Fnature05971"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structures of histone demethylase JMJD2A reveal basis for substrate specificity</span></div><div class="casAuthors">Ng, Stanley S.; Kavanagh, Kathryn L.; McDonough, Michael A.; Butler, Danica; Pilka, Ewa S.; Lienard, Benoit M. R.; Bray, James E.; Savitsky, Pavel; Gileadi, Opher; von Delft, Frank; Rose, Nathan R.; Offer, John; Scheinost, Johanna C.; Borowski, Tomasz; Sundstrom, Michael; Schofield, Christopher J.; Oppermann, Udo</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">448</span>
        (<span class="NLM_cas:issue">7149</span>),
    <span class="NLM_cas:pages">87-91</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Post-translational histone modification has a fundamental role in chromatin biol. and is proposed to constitute a 'histone code' in epigenetic regulation.  Differential methylation of histone H3 and H4 lysyl residues regulates processes including heterochromatin formation, X-chromosome inactivation, genome imprinting, DNA repair and transcriptional regulation.  The discovery of lysyl demethylases using flavin (amine oxidases) or Fe(ii) and 2-oxoglutarate as cofactors (2OG oxygenases) has changed the view of methylation as a stable epigenetic marker.  However, little is known about how the demethylases are selective for particular lysyl-contg. sequences in specific methylation states, a key to understanding their functions.  Here we reveal how human JMJD2A (jumonji domain contg. 2A), which is selective towards tri- and dimethylated histone H3 lysyl residues 9 and 36 (H3K9me3/me2 and H3K36me3/me2), discriminates between methylation states and achieves sequence selectivity for H3K9.  We report structures of JMJD2A-Ni(ii)-Zn(ii) inhibitor complexes bound to tri-, di- and monomethyl forms of H3K9 and the tri-Me form of H3K36.  The structures reveal a lysyl-binding pocket in which substrates are bound in distinct bent conformations involving the Zn-binding site.  We propose a mechanism for achieving methylation state selectivity involving the orientation of the substrate Me groups towards a ferryl intermediate.  The results suggest distinct recognition mechanisms in different demethylase subfamilies and provide a starting point to develop chem. tools for drug discovery and to study and dissect the complexity of reversible histone methylation and its role in chromatin biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-lZrFTDTWwbVg90H21EOLACvtfcHk0liHtbi272hR-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnt1ent7c%253D&md5=c6429fefef17f7ae14b3d01c11617d7f</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1038%2Fnature05971&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature05971%26sid%3Dliteratum%253Aachs%26aulast%3DNg%26aufirst%3DS.%2BS.%26aulast%3DKavanagh%26aufirst%3DK.%2BL.%26aulast%3DMcDonough%26aufirst%3DM.%2BA.%26aulast%3DButler%26aufirst%3DD.%26aulast%3DPilka%26aufirst%3DE.%2BS.%26aulast%3DLienard%26aufirst%3DB.%2BM.%26aulast%3DBray%26aufirst%3DJ.%2BE.%26aulast%3DSavitsky%26aufirst%3DP.%26aulast%3DGileadi%26aufirst%3DO.%26aulast%3Dvon%2BDelft%26aufirst%3DF.%26aulast%3DRose%26aufirst%3DN.%2BR.%26aulast%3DOffer%26aufirst%3DJ.%26aulast%3DScheinost%26aufirst%3DJ.%2BC.%26aulast%3DBorowski%26aufirst%3DT.%26aulast%3DSundstrom%26aufirst%3DM.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26aulast%3DOppermann%26aufirst%3DU.%26atitle%3DCrystal%2520structures%2520of%2520histone%2520demethylase%2520JMJD2A%2520reveal%2520basis%2520for%2520substrate%2520specificity%26jtitle%3DNature%26date%3D2007%26volume%3D448%26spage%3D87%26epage%3D91%26doi%3D10.1038%2Fnature05971" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group">Chen, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whetstine, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hong, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davrazou, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kutateladze, T. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simpson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mao, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pan, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dai, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hagman, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hansen, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, G.</span><span> </span><span class="NLM_article-title">Structural insights into histone demethylation by JMJD2 family members</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">125</span><span class="NLM_x">, </span> <span class="NLM_fpage">691</span><span class="NLM_x">–</span> <span class="NLM_lpage">702</span><span class="refDoi"> DOI: 10.1016/j.cell.2006.04.024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1016%2Fj.cell.2006.04.024" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2006&pages=691-702&author=Z.+Chenauthor=J.+Zangauthor=J.+Whetstineauthor=X.+Hongauthor=F.+Davrazouauthor=T.+G.+Kutateladzeauthor=M.+Simpsonauthor=Q.+Maoauthor=C.+H.+Panauthor=S.+Daiauthor=J.+Hagmanauthor=K.+Hansenauthor=Y.+Shiauthor=G.+Zhang&title=Structural+insights+into+histone+demethylation+by+JMJD2+family+members&doi=10.1016%2Fj.cell.2006.04.024"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2006.04.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2006.04.024%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DZang%26aufirst%3DJ.%26aulast%3DWhetstine%26aufirst%3DJ.%26aulast%3DHong%26aufirst%3DX.%26aulast%3DDavrazou%26aufirst%3DF.%26aulast%3DKutateladze%26aufirst%3DT.%2BG.%26aulast%3DSimpson%26aufirst%3DM.%26aulast%3DMao%26aufirst%3DQ.%26aulast%3DPan%26aufirst%3DC.%2BH.%26aulast%3DDai%26aufirst%3DS.%26aulast%3DHagman%26aufirst%3DJ.%26aulast%3DHansen%26aufirst%3DK.%26aulast%3DShi%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DG.%26atitle%3DStructural%2520insights%2520into%2520histone%2520demethylation%2520by%2520JMJD2%2520family%2520members%26jtitle%3DCell%26date%3D2006%26volume%3D125%26spage%3D691%26epage%3D702%26doi%3D10.1016%2Fj.cell.2006.04.024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group">Shuker, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajduk, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meadows, R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fesik, S. W.</span><span> </span><span class="NLM_article-title">Discovering high-affinity ligands for proteins: SAR by NMR</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">274</span><span class="NLM_x">, </span> <span class="NLM_fpage">1531</span><span class="NLM_x">–</span> <span class="NLM_lpage">1534</span><span class="refDoi"> DOI: 10.1126/science.274.5292.1531</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1126%2Fscience.274.5292.1531" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=8929414" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADyaK28Xnt1ansLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=274&publication_year=1996&pages=1531-1534&author=S.+B.+Shukerauthor=P.+J.+Hajdukauthor=R.+P.+Meadowsauthor=S.+W.+Fesik&title=Discovering+high-affinity+ligands+for+proteins%3A+SAR+by+NMR&doi=10.1126%2Fscience.274.5292.1531"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Discovering high-affinity ligands for proteins: SAR by NMR</span></div><div class="casAuthors">Shuker, Suzanne B.; Hajduk, Philip J.; Meadows, Robert P.; Fesik, Stephen W.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">274</span>
        (<span class="NLM_cas:issue">5292</span>),
    <span class="NLM_cas:pages">1531-1534</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A NMR-based method is described in which small org. mols. that bind to proximal subsites of a protein are identified, optimized, and linked together to produce high-affinity ligands.  The approach is called "SAR by NMR" because structure-activity relationships (SAR) are obtained from NMR.  With this technique, compds. with nanomolar affinities for the FK506 binding protein were rapidly discovered by tethering two ligands with micromolar affinities.  The method reduces the amt. of chem. synthesis and time required for the discovery of high-affinity ligands and appears particularly useful in target-directed drug research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwkuE4EVU_yrVg90H21EOLACvtfcHk0ljhyfJXXzl2WA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28Xnt1ansLs%253D&md5=1b9a5a5e4f2d6ee4b1c249e3e1faf2ec</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1126%2Fscience.274.5292.1531&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.274.5292.1531%26sid%3Dliteratum%253Aachs%26aulast%3DShuker%26aufirst%3DS.%2BB.%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26aulast%3DMeadows%26aufirst%3DR.%2BP.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26atitle%3DDiscovering%2520high-affinity%2520ligands%2520for%2520proteins%253A%2520SAR%2520by%2520NMR%26jtitle%3DScience%26date%3D1996%26volume%3D274%26spage%3D1531%26epage%3D1534%26doi%3D10.1126%2Fscience.274.5292.1531" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group">Friberg, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vigil, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Daniels, R. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burke, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garcia-Barrantes, P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Camper, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chauder, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olejniczak, E. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fesik, S. W.</span><span> </span><span class="NLM_article-title">Discovery of potent myeloid cell leukemia 1 (Mcl-1) inhibitors using fragment-based methods and structure-based design</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">15</span><span class="NLM_x">–</span> <span class="NLM_lpage">30</span><span class="refDoi"> DOI: 10.1021/jm301448p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm301448p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVCjur3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=15-30&author=A.+Fribergauthor=D.+Vigilauthor=B.+Zhaoauthor=R.+N.+Danielsauthor=J.+P.+Burkeauthor=P.+M.+Garcia-Barrantesauthor=D.+Camperauthor=B.+A.+Chauderauthor=T.+Leeauthor=E.+T.+Olejniczakauthor=S.+W.+Fesik&title=Discovery+of+potent+myeloid+cell+leukemia+1+%28Mcl-1%29+inhibitors+using+fragment-based+methods+and+structure-based+design&doi=10.1021%2Fjm301448p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Potent Myeloid Cell Leukemia 1 (Mcl-1) Inhibitors Using Fragment-Based Methods and Structure-Based Design</span></div><div class="casAuthors">Friberg, Anders; Vigil, Dominico; Zhao, Bin; Daniels, R. Nathan; Burke, Jason P.; Garcia-Barrantes, Pedro M.; Camper, DeMarco; Chauder, Brian A.; Lee, Taekyu; Olejniczak, Edward T.; Fesik, Stephen W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">15-30</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Myeloid cell leukemia 1 (Mcl-1), a member of the Bcl-2 family of proteins, is overexpressed and amplified in various cancers and promotes the aberrant survival of tumor cells that otherwise would undergo apoptosis.  Here we describe the discovery of potent and selective Mcl-1 inhibitors using fragment-based methods and structure-based design.  NMR-based screening of a large fragment library identified two chem. distinct hit series that bind to different sites on Mcl-1.  Members of the two fragment classes were merged together to produce lead compds. that bind to Mcl-1 with a dissocn. const. of <100 nM with selectivity for Mcl-1 over Bcl-xL and Bcl-2.  Structures of merged compds. when complexed to Mcl-1 were obtained by x-ray crystallog. and provide detailed information about the mol. recognition of small-mol. ligands binding Mcl-1.  The compds. represent starting points for the discovery of clin. useful Mcl-1 inhibitors for the treatment of a wide variety of cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-bJGKKNPbErVg90H21EOLACvtfcHk0ljhyfJXXzl2WA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVCjur3J&md5=74e72c3d56cdff85d263bd974319b2db</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Fjm301448p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm301448p%26sid%3Dliteratum%253Aachs%26aulast%3DFriberg%26aufirst%3DA.%26aulast%3DVigil%26aufirst%3DD.%26aulast%3DZhao%26aufirst%3DB.%26aulast%3DDaniels%26aufirst%3DR.%2BN.%26aulast%3DBurke%26aufirst%3DJ.%2BP.%26aulast%3DGarcia-Barrantes%26aufirst%3DP.%2BM.%26aulast%3DCamper%26aufirst%3DD.%26aulast%3DChauder%26aufirst%3DB.%2BA.%26aulast%3DLee%26aufirst%3DT.%26aulast%3DOlejniczak%26aufirst%3DE.%2BT.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26atitle%3DDiscovery%2520of%2520potent%2520myeloid%2520cell%2520leukemia%25201%2520%2528Mcl-1%2529%2520inhibitors%2520using%2520fragment-based%2520methods%2520and%2520structure-based%2520design%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D15%26epage%3D30%26doi%3D10.1021%2Fjm301448p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group">Barker, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barker, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Courtney, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gardiner, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hesterkamp, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ichihara, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mather, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Montalbetti, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muller, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varasi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whittaker, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yarnold, C. J.</span><span> </span><span class="NLM_article-title">Discovery of a novel Hsp90 inhibitor by fragment linking</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">1697</span><span class="NLM_x">–</span> <span class="NLM_lpage">1700</span><span class="refDoi"> DOI: 10.1002/cmdc.201000219</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1002%2Fcmdc.201000219" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=20665758" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlCkurzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2010&pages=1697-1700&author=J.+J.+Barkerauthor=O.+Barkerauthor=S.+M.+Courtneyauthor=M.+Gardinerauthor=T.+Hesterkampauthor=O.+Ichiharaauthor=O.+Matherauthor=C.+A.+Montalbettiauthor=A.+Mullerauthor=M.+Varasiauthor=M.+Whittakerauthor=C.+J.+Yarnold&title=Discovery+of+a+novel+Hsp90+inhibitor+by+fragment+linking&doi=10.1002%2Fcmdc.201000219"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a novel Hsp90 inhibitor by fragment linking</span></div><div class="casAuthors">Barker, John J.; Barker, Oliver; Courtney, Stephen M.; Gardiner, Mihaly; Hesterkamp, Thomas; Ichihara, Osamu; Mather, Owen; Montalbetti, Christian A. G. N.; Muller, Annett; Varasi, Mario; Whittaker, Mark; Yarnold, Christopher J.</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1697-1700</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Following a high-throughput biochem. fragment screen, we have identified novel fragment inhibitors of Hsp90.  Two fragment hits were combined to give a dual-fragment Hsp90 complex.  The compd., I, with the lowest strain energy was synthesized, and a 1000-fold improvement in activity was achieved.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq05FWLIYmt6bVg90H21EOLACvtfcHk0lhJ3Mh-S_Ln6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlCkurzI&md5=5ae93af1444d6819d8cc4bf2854bff4f</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201000219&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201000219%26sid%3Dliteratum%253Aachs%26aulast%3DBarker%26aufirst%3DJ.%2BJ.%26aulast%3DBarker%26aufirst%3DO.%26aulast%3DCourtney%26aufirst%3DS.%2BM.%26aulast%3DGardiner%26aufirst%3DM.%26aulast%3DHesterkamp%26aufirst%3DT.%26aulast%3DIchihara%26aufirst%3DO.%26aulast%3DMather%26aufirst%3DO.%26aulast%3DMontalbetti%26aufirst%3DC.%2BA.%26aulast%3DMuller%26aufirst%3DA.%26aulast%3DVarasi%26aufirst%3DM.%26aulast%3DWhittaker%26aufirst%3DM.%26aulast%3DYarnold%26aufirst%3DC.%2BJ.%26atitle%3DDiscovery%2520of%2520a%2520novel%2520Hsp90%2520inhibitor%2520by%2520fragment%2520linking%26jtitle%3DChemMedChem%26date%3D2010%26volume%3D5%26spage%3D1697%26epage%3D1700%26doi%3D10.1002%2Fcmdc.201000219" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group">Villemagne, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flipo, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blondiaux, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crauste, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Malaquin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leroux, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Piveteau, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Villeret, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brodin, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Villoutreix, B. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sperandio, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soror, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wohlkonig, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wintjens, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deprez, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baulard, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willand, N.</span><span> </span><span class="NLM_article-title">Ligand Efficiency Driven Design of New Inhibitors of Mycobacterium tuberculosis Transcriptional Repressor EthR Using Fragment Growing, Merging, and Linking Approaches</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">4876</span><span class="NLM_x">–</span> <span class="NLM_lpage">4888</span><span class="refDoi"> DOI: 10.1021/jm500422b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm500422b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BC2cXnvVShtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=4876-4888&author=B.+Villemagneauthor=M.+Flipoauthor=N.+Blondiauxauthor=C.+Crausteauthor=S.+Malaquinauthor=F.+Lerouxauthor=C.+Piveteauauthor=V.+Villeretauthor=P.+Brodinauthor=B.+O.+Villoutreixauthor=O.+Sperandioauthor=S.+H.+Sororauthor=A.+Wohlkonigauthor=R.+Wintjensauthor=B.+Deprezauthor=A.+R.+Baulardauthor=N.+Willand&title=Ligand+Efficiency+Driven+Design+of+New+Inhibitors+of+Mycobacterium+tuberculosis+Transcriptional+Repressor+EthR+Using+Fragment+Growing%2C+Merging%2C+and+Linking+Approaches&doi=10.1021%2Fjm500422b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand Efficiency Driven Design of New Inhibitors of Mycobacterium tuberculosis Transcriptional Repressor EthR Using Fragment Growing, Merging, and Linking Approaches</span></div><div class="casAuthors">Villemagne, Baptiste; Flipo, Marion; Blondiaux, Nicolas; Crauste, Celine; Malaquin, Sandra; Leroux, Florence; Piveteau, Catherine; Villeret, Vincent; Brodin, Priscille; Villoutreix, Bruno O.; Sperandio, Olivier; Soror, Sameh H.; Wohlkonig, Alexandre; Wintjens, Rene; Deprez, Benoit; Baulard, Alain R.; Willand, Nicolas</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4876-4888</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Tuberculosis remains a major cause of mortality and morbidity, killing each year more than one million people.  Although the combined use of first line antibiotics (isoniazid, rifampicin, pyrazinamide, and ethambutol) is efficient to treat most patients, the rapid emergence of multidrug resistant strains of Mycobacterium tuberculosis stresses the need for alternative therapies.  Mycobacterial transcriptional repressor EthR is a key player in the control of second-line drugs bioactivation such as ethionamide and has been shown to impair the sensitivity of the human pathogen Mycobacterium tuberculosis to this antibiotic.  As a way to identify new potent ligands of this protein, we have developed fragment-based approaches.  In the current study, we combined surface plasmon resonance assay, x-ray crystallog., and ligand efficiency driven design for the rapid discovery and optimization of new chemotypes of EthR ligands starting from a fragment.  The design, synthesis, and in vitro and ex vivo activities of these compds. will be discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp77Ygs3mf7prVg90H21EOLACvtfcHk0lhJ3Mh-S_Ln6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXnvVShtrY%253D&md5=36763fe306287529745b7c5a48272d6a</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Fjm500422b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm500422b%26sid%3Dliteratum%253Aachs%26aulast%3DVillemagne%26aufirst%3DB.%26aulast%3DFlipo%26aufirst%3DM.%26aulast%3DBlondiaux%26aufirst%3DN.%26aulast%3DCrauste%26aufirst%3DC.%26aulast%3DMalaquin%26aufirst%3DS.%26aulast%3DLeroux%26aufirst%3DF.%26aulast%3DPiveteau%26aufirst%3DC.%26aulast%3DVilleret%26aufirst%3DV.%26aulast%3DBrodin%26aufirst%3DP.%26aulast%3DVilloutreix%26aufirst%3DB.%2BO.%26aulast%3DSperandio%26aufirst%3DO.%26aulast%3DSoror%26aufirst%3DS.%2BH.%26aulast%3DWohlkonig%26aufirst%3DA.%26aulast%3DWintjens%26aufirst%3DR.%26aulast%3DDeprez%26aufirst%3DB.%26aulast%3DBaulard%26aufirst%3DA.%2BR.%26aulast%3DWilland%26aufirst%3DN.%26atitle%3DLigand%2520Efficiency%2520Driven%2520Design%2520of%2520New%2520Inhibitors%2520of%2520Mycobacterium%2520tuberculosis%2520Transcriptional%2520Repressor%2520EthR%2520Using%2520Fragment%2520Growing%252C%2520Merging%252C%2520and%2520Linking%2520Approaches%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D4876%26epage%3D4888%26doi%3D10.1021%2Fjm500422b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group">Erlanson, D. A.</span><span> </span><span class="NLM_article-title">Introduction to fragment-based drug discovery</span> <span class="citation_source-journal">Top. Curr. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">317</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">32</span><span class="refDoi"> DOI: 10.1007/128_2011_180</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1007%2F128_2011_180" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=317&publication_year=2011&pages=1-32&author=D.+A.+Erlanson&title=Introduction+to+fragment-based+drug+discovery&doi=10.1007%2F128_2011_180"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1007%2F128_2011_180&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F128_2011_180%26sid%3Dliteratum%253Aachs%26aulast%3DErlanson%26aufirst%3DD.%2BA.%26atitle%3DIntroduction%2520to%2520fragment-based%2520drug%2520discovery%26jtitle%3DTop.%2520Curr.%2520Chem.%26date%3D2011%26volume%3D317%26spage%3D1%26epage%3D32%26doi%3D10.1007%2F128_2011_180" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group">Teotico, D. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Babaoglu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rocklin, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferreira, R. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giannetti, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Docking for fragment inhibitors of AmpC beta-lactamase</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">106</span><span class="NLM_x">, </span> <span class="NLM_fpage">7455</span><span class="NLM_x">–</span> <span class="NLM_lpage">7460</span><span class="refDoi"> DOI: 10.1073/pnas.0813029106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1073%2Fpnas.0813029106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=19416920" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BD1MXmt1Khtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2009&pages=7455-7460&author=D.+G.+Teoticoauthor=K.+Babaogluauthor=G.+J.+Rocklinauthor=R.+S.+Ferreiraauthor=A.+M.+Giannettiauthor=B.+K.+Shoichet&title=Docking+for+fragment+inhibitors+of+AmpC+beta-lactamase&doi=10.1073%2Fpnas.0813029106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Docking for fragment inhibitors of AmpC 83-lactamase</span></div><div class="casAuthors">Teotico, Denise G.; Babaoglu, Kerim; Rocklin, Gabriel J.; Ferreira, Rafaela S.; Giannetti, Anthony M.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">7455-7460</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Fragment screens for new ligands have had wide success, not withstanding their constraint to libraries of 1000-10,000 mols.  Larger libraries would be addressable were mol. docking reliable for fragment screens, but this has not been widely accepted.  To investigate docking's ability to prioritize fragments, a library of >137,000 such mols. were docked against the structure of β-lactamase.  Forty-eight fragments highly ranked by docking were acquired and tested; 23 had Ki values ranging from 0.7 to 9.2 mM.  X-ray crystal structures of the enzyme-bound complexes were detd. for 8 of the fragments.  For 4, the correspondence between the predicted and exptl. structures was high (RMSD between 1.2 and 1.4 Å), whereas for another 2, the fidelity was lower but retained most key interactions (RMSD 2.4-2.6 Å).  Two of the 8 fragments adopted very different poses in the active site owing to enzyme conformational changes.  The 48% hit rate of the fragment docking compares very favorably with "lead-like" docking and high-throughput screening against the same enzyme.  To understand this, we investigated the occurrence of the fragment scaffolds among larger, lead-like mols.  Approx. 1% of com. available fragments contain these inhibitors whereas only 10-7% of lead-like mols. do.  This suggests that many more chemotypes and combinations of chemotypes are present among fragments than are available among lead-like mols., contributing to the higher hit rates.  The ability of docking to prioritize these fragments suggests that the technique can be used to exploit the better chemotype coverage that exists at the fragment level.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprpafzXv4ydrVg90H21EOLACvtfcHk0lhJ3Mh-S_Ln6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXmt1Khtrg%253D&md5=d2df01432d5381c226e8aa7e008457b5</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0813029106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0813029106%26sid%3Dliteratum%253Aachs%26aulast%3DTeotico%26aufirst%3DD.%2BG.%26aulast%3DBabaoglu%26aufirst%3DK.%26aulast%3DRocklin%26aufirst%3DG.%2BJ.%26aulast%3DFerreira%26aufirst%3DR.%2BS.%26aulast%3DGiannetti%26aufirst%3DA.%2BM.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DDocking%2520for%2520fragment%2520inhibitors%2520of%2520AmpC%2520beta-lactamase%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2009%26volume%3D106%26spage%3D7455%26epage%3D7460%26doi%3D10.1073%2Fpnas.0813029106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group">Chen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Molecular docking and ligand specificity in fragment-based inhibitor discovery</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">358</span><span class="NLM_x">–</span> <span class="NLM_lpage">364</span><span class="refDoi"> DOI: 10.1038/nchembio.155</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1038%2Fnchembio.155" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=19305397" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjsVCqt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=358-364&author=Y.+Chenauthor=B.+K.+Shoichet&title=Molecular+docking+and+ligand+specificity+in+fragment-based+inhibitor+discovery&doi=10.1038%2Fnchembio.155"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular docking and ligand specificity in fragment-based inhibitor discovery</span></div><div class="casAuthors">Chen, Yu; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">358-364</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Fragment screens have successfully identified new scaffolds in drug discovery, often with relatively high hit rates (5%) using small screening libraries (1,000-10,000 compds.).  This raises two questions: would other noteworthy chemotypes be found were one to screen all com. available fragments (>300,000), and does the success rate imply low specificity of fragments.  We used mol. docking to screen large libraries of fragments against CTX-M β-lactamase.  We identified ten millimolar-range inhibitors from the 69 compds. tested.  The docking poses corresponded closely to the crystallog. structures subsequently detd.  Notably, these initial low-affinity hits showed little specificity between CTX-M and an unrelated β-lactamase, AmpC, which is unusual among β-lactamase inhibitors.  This is consistent with the idea that the high hit rates among fragments correlate to a low initial specificity.  As the inhibitors were progressed, both specificity and affinity rose together, yielding to our knowledge the first micromolar-range noncovalent inhibitors against a class A β-lactamase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqO8zTkq9kMAbVg90H21EOLACvtfcHk0lhJAk-s5DTErg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjsVCqt7k%253D&md5=6ee8dd19d0b99599ea74eaf735483b1d</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.155&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.155%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DY.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DMolecular%2520docking%2520and%2520ligand%2520specificity%2520in%2520fragment-based%2520inhibitor%2520discovery%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2009%26volume%3D5%26spage%3D358%26epage%3D364%26doi%3D10.1038%2Fnchembio.155" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group">Tiefenbrunn, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Forli, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baksh, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, M. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Happer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, Y. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perryman, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rhee, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Torbett, B. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olson, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elder, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Finn, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stout, C. D.</span><span> </span><span class="NLM_article-title">Small Molecule Regulation of Protein Conformation by Binding in the Flap of HIV Protease</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">1223</span><span class="NLM_x">–</span> <span class="NLM_lpage">1231</span><span class="refDoi"> DOI: 10.1021/cb300611p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb300611p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=1223-1231&author=T.+Tiefenbrunnauthor=S.+Forliauthor=M.+M.+Bakshauthor=M.+W.+Changauthor=M.+Happerauthor=Y.+C.+Linauthor=A.+L.+Perrymanauthor=J.+K.+Rheeauthor=B.+E.+Torbettauthor=A.+J.+Olsonauthor=J.+H.+Elderauthor=M.+G.+Finnauthor=C.+D.+Stout&title=Small+Molecule+Regulation+of+Protein+Conformation+by+Binding+in+the+Flap+of+HIV+Protease&doi=10.1021%2Fcb300611p"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1021%2Fcb300611p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb300611p%26sid%3Dliteratum%253Aachs%26aulast%3DTiefenbrunn%26aufirst%3DT.%26aulast%3DForli%26aufirst%3DS.%26aulast%3DBaksh%26aufirst%3DM.%2BM.%26aulast%3DChang%26aufirst%3DM.%2BW.%26aulast%3DHapper%26aufirst%3DM.%26aulast%3DLin%26aufirst%3DY.%2BC.%26aulast%3DPerryman%26aufirst%3DA.%2BL.%26aulast%3DRhee%26aufirst%3DJ.%2BK.%26aulast%3DTorbett%26aufirst%3DB.%2BE.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26aulast%3DElder%26aufirst%3DJ.%2BH.%26aulast%3DFinn%26aufirst%3DM.%2BG.%26aulast%3DStout%26aufirst%3DC.%2BD.%26atitle%3DSmall%2520Molecule%2520Regulation%2520of%2520Protein%2520Conformation%2520by%2520Binding%2520in%2520the%2520Flap%2520of%2520HIV%2520Protease%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2013%26volume%3D8%26spage%3D1223%26epage%3D1231%26doi%3D10.1021%2Fcb300611p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group">Nichols, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Renslo, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y.</span><span> </span><span class="NLM_article-title">Fragment-based inhibitor discovery against beta-lactamase</span> <span class="citation_source-journal">Future Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">413</span><span class="NLM_x">–</span> <span class="NLM_lpage">427</span><span class="refDoi"> DOI: 10.4155/fmc.14.10</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.4155%2Ffmc.14.10" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=24635522" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BC2cXksV2qtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=413-427&author=D.+A.+Nicholsauthor=A.+R.+Rensloauthor=Y.+Chen&title=Fragment-based+inhibitor+discovery+against+beta-lactamase&doi=10.4155%2Ffmc.14.10"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-based inhibitor discovery against β-lactamase</span></div><div class="casAuthors">Nichols, Derek A.; Renslo, Adam R.; Chen, Yu</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">413-427</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  The prodn. of β-lactamase is one of the primary resistance mechanisms used by Gram-neg. bacterial pathogens to counter β-lactam antibiotics, such as penicillins, cephalosporins and carbapenems.  There is an urgent need to develop novel β-lactamase inhibitors in response to ever evolving β-lactamases possessing an expanded spectrum of β-lactam hydrolyzing activity.  Whereas traditional high-throughput screening proved ineffective against serine β-lactamases, fragment-based approaches have been successfully employed to identify novel chem. matter, which in turn revealed much about the specific mol. interactions possible in the active site of serine and metallo β-lactamases.  In this review, the authors summarize recent progress in the field, particularly: the identification of novel inhibitor chemotypes through fragment-based screening; the use of fragment-protein structures to understand key features of binding hot spots and inform the design of improved leads; lessons learned and new prospects for β-lactamase inhibitor development using fragment-based approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9O0Za50-8K7Vg90H21EOLACvtfcHk0lhJAk-s5DTErg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXksV2qtb0%253D&md5=6e8278707450cc1fe4c33f71efe41f8c</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.4155%2Ffmc.14.10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.14.10%26sid%3Dliteratum%253Aachs%26aulast%3DNichols%26aufirst%3DD.%2BA.%26aulast%3DRenslo%26aufirst%3DA.%2BR.%26aulast%3DChen%26aufirst%3DY.%26atitle%3DFragment-based%2520inhibitor%2520discovery%2520against%2520beta-lactamase%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2014%26volume%3D6%26spage%3D413%26epage%3D427%26doi%3D10.4155%2Ffmc.14.10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group">Barelier, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eidam, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fish, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hollander, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Figaroa, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nachane, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siegal, G.</span><span> </span><span class="NLM_article-title">Increasing chemical space coverage by combining empirical and computational fragment screens</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">1528</span><span class="NLM_x">–</span> <span class="NLM_lpage">1535</span><span class="refDoi"> DOI: 10.1021/cb5001636</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb5001636" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BC2cXnsFSksbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=1528-1535&author=S.+Barelierauthor=O.+Eidamauthor=I.+Fishauthor=J.+Hollanderauthor=F.+Figaroaauthor=R.+Nachaneauthor=J.+J.+Irwinauthor=B.+K.+Shoichetauthor=G.+Siegal&title=Increasing+chemical+space+coverage+by+combining+empirical+and+computational+fragment+screens&doi=10.1021%2Fcb5001636"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Increasing Chemical Space Coverage by Combining Empirical and Computational Fragment Screens</span></div><div class="casAuthors">Barelier, Sarah; Eidam, Oliv; Fish, Inbar; Hollander, Johan; Figaroa, Francis; Nachane, Ruta; Irwin, John J.; Shoichet, Brian K.; Siegal, Gregg</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1528-1535</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Most libraries for fragment-based drug discovery are restricted to 1,000-10,000 compds., but over 500,000 fragments are com. available and potentially accessible by virtual screening.  Whether this larger set would increase chemotype coverage, and whether a computational screen can pragmatically prioritize them, is debated.  To investigate this question, a 1281-fragment library was screened by NMR (NMR) against AmpC β-lactamase, and hits were confirmed by surface plasmon resonance (SPR).  Nine hits with novel chemotypes were confirmed biochem. with KI values from 0.2 to low mM.  We also computationally docked 290,000 purchasable fragments with chemotypes unrepresented in the empirical library, finding 10 that had KI values from 0.03 to low mM.  Though less novel than those discovered by NMR, the docking-derived fragments filled chemotype holes from the empirical library.  Crystal structures of nine of the fragments in complex with AmpC β-lactamase revealed new binding sites and explained the relatively high affinity of the docking-derived fragments.  The existence of chemotype holes is likely a general feature of fragment libraries, as calcn. suggests that to represent the fragment substructures of even known biogenic mols. would demand a library of minimally over 32,000 fragments.  Combining computational and empirical fragment screens enables the discovery of unexpected chemotypes, here by the NMR screen, while capturing chemotypes missing from the empirical library and tailored to the target, with little extra cost in resources.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5DNyK9V50nLVg90H21EOLACvtfcHk0lhJAk-s5DTErg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXnsFSksbg%253D&md5=7c75e304e70fec6d9209c21a87a06a17</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1021%2Fcb5001636&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb5001636%26sid%3Dliteratum%253Aachs%26aulast%3DBarelier%26aufirst%3DS.%26aulast%3DEidam%26aufirst%3DO.%26aulast%3DFish%26aufirst%3DI.%26aulast%3DHollander%26aufirst%3DJ.%26aulast%3DFigaroa%26aufirst%3DF.%26aulast%3DNachane%26aufirst%3DR.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DSiegal%26aufirst%3DG.%26atitle%3DIncreasing%2520chemical%2520space%2520coverage%2520by%2520combining%2520empirical%2520and%2520computational%2520fragment%2520screens%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2014%26volume%3D9%26spage%3D1528%26epage%3D1535%26doi%3D10.1021%2Fcb5001636" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group">Suzuki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyata, N.</span><span> </span><span class="NLM_article-title">Lysine demethylases inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">8236</span><span class="NLM_x">–</span> <span class="NLM_lpage">8250</span><span class="refDoi"> DOI: 10.1021/jm201048w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm201048w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1OrtrnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=8236-8250&author=T.+Suzukiauthor=N.+Miyata&title=Lysine+demethylases+inhibitors&doi=10.1021%2Fjm201048w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine Demethylases Inhibitors</span></div><div class="casAuthors">Suzuki, Takayoshi; Miyata, Naoki</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">8236-8250</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review, on the use of lysine demethylases inhibitors in therapy of various diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplp9MjSdvnn7Vg90H21EOLACvtfcHk0lhwhnSQN8KSHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1OrtrnF&md5=194d4bbc2e52672f5d5f019de0bb760c</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1021%2Fjm201048w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm201048w%26sid%3Dliteratum%253Aachs%26aulast%3DSuzuki%26aufirst%3DT.%26aulast%3DMiyata%26aufirst%3DN.%26atitle%3DLysine%2520demethylases%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D8236%26epage%3D8250%26doi%3D10.1021%2Fjm201048w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rival-Gervier, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Labrie, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wigle, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dimaggio, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wasney, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siarheyeva, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tempel, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chau, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mangano, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, X. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simpson, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pattenden, S. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norris, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kireev, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tripathy, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edwards, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janzen, W. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garcia, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petronis, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ellis, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">A chemical probe selectively inhibits G9a and GLP methyltransferase activity in cells</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">566</span><span class="NLM_x">–</span> <span class="NLM_lpage">574</span><span class="refDoi"> DOI: 10.1038/nchembio.599</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1038%2Fnchembio.599" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=21743462" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BC3MXos1OmtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=566-574&author=M.+Vedadiauthor=D.+Barsyte-Lovejoyauthor=F.+Liuauthor=S.+Rival-Gervierauthor=A.+Allali-Hassaniauthor=V.+Labrieauthor=T.+J.+Wigleauthor=P.+A.+Dimaggioauthor=G.+A.+Wasneyauthor=A.+Siarheyevaauthor=A.+Dongauthor=W.+Tempelauthor=S.+C.+Wangauthor=X.+Chenauthor=I.+Chauauthor=T.+J.+Manganoauthor=X.+P.+Huangauthor=C.+D.+Simpsonauthor=S.+G.+Pattendenauthor=J.+L.+Norrisauthor=D.+B.+Kireevauthor=A.+Tripathyauthor=A.+Edwardsauthor=B.+L.+Rothauthor=W.+P.+Janzenauthor=B.+A.+Garciaauthor=A.+Petronisauthor=J.+Ellisauthor=P.+J.+Brownauthor=S.+V.+Fryeauthor=C.+H.+Arrowsmithauthor=J.+Jin&title=A+chemical+probe+selectively+inhibits+G9a+and+GLP+methyltransferase+activity+in+cells&doi=10.1038%2Fnchembio.599"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">A chemical probe selectively inhibits G9a and GLP methyltransferase activity in cells</span></div><div class="casAuthors">Vedadi, Masoud; Barsyte-Lovejoy, Dalia; Liu, Feng; Rival-Gervier, Sylvie; Allali-Hassani, Abdellah; Labrie, Viviane; Wigle, Tim J.; DiMaggio, Peter A.; Wasney, Gregory A.; Siarheyeva, Alena; Dong, Aiping; Tempel, Wolfram; Wang, Sun-Chong; Chen, Xin; Chau, Irene; Mangano, Thomas J.; Huang, Xi-ping; Simpson, Catherine D.; Pattenden, Samantha G.; Norris, Jacqueline L.; Kireev, Dmitri B.; Tripathy, Ashutosh; Edwards, Aled; Roth, Bryan L.; Janzen, William P.; Garcia, Benjamin A.; Petronis, Arturas; Ellis, James; Brown, Peter J.; Frye, Stephen V.; Arrowsmith, Cheryl H.; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">566-574</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein lysine methyltransferases G9a and GLP modulate the transcriptional repression of a variety of genes via dimethylation of Lys9 on histone H3 (H3K9me2) as well as dimethylation of non-histone targets.  Here we report the discovery of UNC0638, an inhibitor of G9a and GLP with excellent potency and selectivity over a wide range of epigenetic and non-epigenetic targets.  UNC0638 treatment of a variety of cell lines resulted in lower global H3K9me2 levels, equiv. to levels obsd. for small hairpin RNA knockdown of G9a and GLP with the functional potency of UNC0638 being well sepd. from its toxicity.  UNC0638 markedly reduced the clonogenicity of MCF7 cells, reduced the abundance of H3K9me2 marks at promoters of known G9a-regulated endogenous genes and disproportionately affected several genomic loci encoding microRNAs.  In mouse embryonic stem cells, UNC0638 reactivated G9a-silenced genes and a retroviral reporter gene in a concn.-dependent manner without promoting differentiation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphv-2bZcBqwLVg90H21EOLACvtfcHk0lhwhnSQN8KSHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXos1OmtLs%253D&md5=a944e99a910194c1a15240fc6dc8f852</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.599&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.599%26sid%3Dliteratum%253Aachs%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DRival-Gervier%26aufirst%3DS.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DLabrie%26aufirst%3DV.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DDimaggio%26aufirst%3DP.%2BA.%26aulast%3DWasney%26aufirst%3DG.%2BA.%26aulast%3DSiarheyeva%26aufirst%3DA.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DTempel%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DS.%2BC.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DMangano%26aufirst%3DT.%2BJ.%26aulast%3DHuang%26aufirst%3DX.%2BP.%26aulast%3DSimpson%26aufirst%3DC.%2BD.%26aulast%3DPattenden%26aufirst%3DS.%2BG.%26aulast%3DNorris%26aufirst%3DJ.%2BL.%26aulast%3DKireev%26aufirst%3DD.%2BB.%26aulast%3DTripathy%26aufirst%3DA.%26aulast%3DEdwards%26aufirst%3DA.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DJanzen%26aufirst%3DW.%2BP.%26aulast%3DGarcia%26aufirst%3DB.%2BA.%26aulast%3DPetronis%26aufirst%3DA.%26aulast%3DEllis%26aufirst%3DJ.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DA%2520chemical%2520probe%2520selectively%2520inhibits%2520G9a%2520and%2520GLP%2520methyltransferase%2520activity%2520in%2520cells%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2011%26volume%3D7%26spage%3D566%26epage%3D574%26doi%3D10.1038%2Fnchembio.599" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group">Hermann, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ghanem, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raushel, F. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Predicting substrates by docking high-energy intermediates to enzyme structures</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">128</span><span class="NLM_x">, </span> <span class="NLM_fpage">15882</span><span class="NLM_x">–</span> <span class="NLM_lpage">15891</span><span class="refDoi"> DOI: 10.1021/ja065860f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja065860f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1WhsLjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2006&pages=15882-15891&author=J.+C.+Hermannauthor=E.+Ghanemauthor=Y.+C.+Liauthor=F.+M.+Raushelauthor=J.+J.+Irwinauthor=B.+K.+Shoichet&title=Predicting+substrates+by+docking+high-energy+intermediates+to+enzyme+structures&doi=10.1021%2Fja065860f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Predicting Substrates by Docking High-Energy Intermediates to Enzyme Structures</span></div><div class="casAuthors">Hermann, Johannes C.; Ghanem, Eman; Li, Yingchun; Raushel, Frank M.; Irwin, John J.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">15882-15891</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">With the emergence of sequences and even structures for proteins of unknown function, structure-based prediction of enzyme activity has become a pragmatic as well as an interesting question.  Here we investigate a method to predict substrates for enzymes of known structure by docking high-energy intermediate forms of the potential substrates.  A database of such high-energy transition-state analogs was created from the KEGG metabolites.  To reduce the no. of possible reactions to consider, we restricted ourselves to enzymes of the amidohydrolase superfamily.  We docked each metabolite into seven different amidohydrolases in both the ground-state and the high-energy intermediate forms.  Docking the high-energy intermediates improved the discrimination between decoys and substrates significantly over the corresponding std. ground-state database, both by enrichment of the true substrates and by geometric fidelity.  To test this method prospectively, we attempted to predict the enantioselectivity of a set of chiral substrates for phosphotriesterase, for both wild-type and mutant forms of this enzyme.  The stereoselectivity ratios of the six enzymes considered for those four substrate enantiomer pairs differed over a range of 10- to 10,000-fold and underwent 20 switches in stereoselectivities for favored enantiomers, compared to the wild type.  The docking of the high-energy intermediates correctly predicted the stereoselectivities for 18 of the 20 substrate/enzyme combinations when compared to subsequent exptl. synthesis and testing.  The possible applications of this approach to other enzymes are considered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXBSzfDFP1ZLVg90H21EOLACvtfcHk0lhwhnSQN8KSHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1WhsLjL&md5=e34d1979bf873173f3c21c2de4275b5b</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1021%2Fja065860f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja065860f%26sid%3Dliteratum%253Aachs%26aulast%3DHermann%26aufirst%3DJ.%2BC.%26aulast%3DGhanem%26aufirst%3DE.%26aulast%3DLi%26aufirst%3DY.%2BC.%26aulast%3DRaushel%26aufirst%3DF.%2BM.%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DPredicting%2520substrates%2520by%2520docking%2520high-energy%2520intermediates%2520to%2520enzyme%2520structures%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2006%26volume%3D128%26spage%3D15882%26epage%3D15891%26doi%3D10.1021%2Fja065860f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group">Hermann, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marti-Arbona, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Almo, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raushel, F. M.</span><span> </span><span class="NLM_article-title">Structure-based activity prediction for an enzyme of unknown function</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">448</span><span class="NLM_x">, </span> <span class="NLM_fpage">775</span><span class="NLM_x">–</span> <span class="NLM_lpage">779</span><span class="refDoi"> DOI: 10.1038/nature05981</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1038%2Fnature05981" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=17603473" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BD2sXptVGrs7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=448&publication_year=2007&pages=775-779&author=J.+C.+Hermannauthor=R.+Marti-Arbonaauthor=A.+A.+Fedorovauthor=E.+Fedorovauthor=S.+C.+Almoauthor=B.+K.+Shoichetauthor=F.+M.+Raushel&title=Structure-based+activity+prediction+for+an+enzyme+of+unknown+function&doi=10.1038%2Fnature05981"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based activity prediction for an enzyme of unknown function</span></div><div class="casAuthors">Hermann, Johannes C.; Marti-Arbona, Ricardo; Fedorov, Alexander A.; Fedorov, Elena; Almo, Steven C.; Shoichet, Brian K.; Raushel, Frank M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">448</span>
        (<span class="NLM_cas:issue">7155</span>),
    <span class="NLM_cas:pages">775-779</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">With many genomes sequenced, a pressing challenge in biol. is predicting the function of the proteins that the genes encode.  When proteins are unrelated to others of known activity, bioinformatics inference for function becomes problematic.  It would thus be useful to interrogate protein structures for function directly.  Here, we predict the function of an enzyme of unknown activity, Tm0936 from Thermotoga maritima, by docking high-energy intermediate forms of thousands of candidate metabolites.  The docking hit list was dominated by adenine analogs, which appeared to undergo C6-deamination.  Four of these, including 5-methylthioadenosine and S-adenosylhomocysteine (SAH), were tested as substrates, and three had substantial catalytic rate consts. (105 M-1 s-1).  The x-ray crystal structure of the complex between Tm0936 and the product resulting from the deamination of SAH, S-inosylhomocysteine, was detd., and it corresponded closely to the predicted structure.  The deaminated products can be further metabolized by T. maritima in a previously uncharacterized SAH degrdn. pathway.  Structure-based docking with high-energy forms of potential substrates may be a useful tool to annotate enzymes for function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomC_L0C9hsf7Vg90H21EOLACvtfcHk0ljYHgPB4IhWLg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXptVGrs7w%253D&md5=b0bd6612147fb848f49f8b9ccfd8384e</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1038%2Fnature05981&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature05981%26sid%3Dliteratum%253Aachs%26aulast%3DHermann%26aufirst%3DJ.%2BC.%26aulast%3DMarti-Arbona%26aufirst%3DR.%26aulast%3DFedorov%26aufirst%3DA.%2BA.%26aulast%3DFedorov%26aufirst%3DE.%26aulast%3DAlmo%26aufirst%3DS.%2BC.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DRaushel%26aufirst%3DF.%2BM.%26atitle%3DStructure-based%2520activity%2520prediction%2520for%2520an%2520enzyme%2520of%2520unknown%2520function%26jtitle%3DNature%26date%3D2007%26volume%3D448%26spage%3D775%26epage%3D779%26doi%3D10.1038%2Fnature05981" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group">Kuntz, I. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blaney, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oatley, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Langridge, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferrin, T. E.</span><span> </span><span class="NLM_article-title">A geometric approach to macromolecule-ligand interactions</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1982</span><span class="NLM_x">, </span> <span class="NLM_volume">161</span><span class="NLM_x">, </span> <span class="NLM_fpage">269</span><span class="NLM_x">–</span> <span class="NLM_lpage">88</span><span class="refDoi"> DOI: 10.1016/0022-2836(82)90153-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1016%2F0022-2836%2882%2990153-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=7154081" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADyaL38XmtFajsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=161&publication_year=1982&pages=269-88&author=I.+D.+Kuntzauthor=J.+M.+Blaneyauthor=S.+J.+Oatleyauthor=R.+Langridgeauthor=T.+E.+Ferrin&title=A+geometric+approach+to+macromolecule-ligand+interactions&doi=10.1016%2F0022-2836%2882%2990153-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">A geometric approach to macromolecule-ligand interactions</span></div><div class="casAuthors">Kuntz, Irwin D.; Blaney, Jeffrey M.; Oatley, Stuart J.; Langridge, Robert; Ferrin, Thomas E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1982</span>),
    <span class="NLM_cas:volume">161</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">269-88</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    </div><div class="casAbstract">A method is described to explore geometrically feasible alignments of ligands and receptors of known structure.  Algorithms are presented that examine many binding geometries and evaluate them in terms of steric overlap.  The procedure uses specific mol. conformations.  A method is included for finding putative binding sites on a macromol. surface.  Results are reported for heme-myoglobin interaction and the binding of thyroid hormone analogs to prealbumin.  In each case, the program finds structures within 1 Å of the x-ray results and also finds distinctly different geometries that provide good steric fits.  The approach seems well-suited for generating conformations for energy refinement programs and interactive computer graphics routines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWsp3KKxuQf7Vg90H21EOLACvtfcHk0ljYHgPB4IhWLg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL38XmtFajsbw%253D&md5=8a4234b24356ea5340f33d906cd71d3e</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1016%2F0022-2836%2882%2990153-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0022-2836%252882%252990153-X%26sid%3Dliteratum%253Aachs%26aulast%3DKuntz%26aufirst%3DI.%2BD.%26aulast%3DBlaney%26aufirst%3DJ.%2BM.%26aulast%3DOatley%26aufirst%3DS.%2BJ.%26aulast%3DLangridge%26aufirst%3DR.%26aulast%3DFerrin%26aufirst%3DT.%2BE.%26atitle%3DA%2520geometric%2520approach%2520to%2520macromolecule-ligand%2520interactions%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1982%26volume%3D161%26spage%3D269%26epage%3D88%26doi%3D10.1016%2F0022-2836%2882%2990153-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit69"><span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mysinger, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Colizzi, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wassam, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, Y.</span><span> </span><span class="NLM_article-title">Automated docking screens: a feasibility study</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">5712</span><span class="NLM_x">–</span> <span class="NLM_lpage">5720</span><span class="refDoi"> DOI: 10.1021/jm9006966</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm9006966" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVOjurjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=5712-5720&author=J.+J.+Irwinauthor=B.+K.+Shoichetauthor=M.+M.+Mysingerauthor=N.+Huangauthor=F.+Colizziauthor=P.+Wassamauthor=Y.+Cao&title=Automated+docking+screens%3A+a+feasibility+study&doi=10.1021%2Fjm9006966"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Automated Docking Screens: A Feasibility Study</span></div><div class="casAuthors">Irwin, John J.; Shoichet, Brian K.; Mysinger, Michael M.; Huang, Niu; Colizzi, Francesco; Wassam, Pascal; Cao, Yiqun</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">5712-5720</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. docking is the most practical approach to leverage protein structure for ligand discovery, but the technique retains important liabilities that make it challenging to deploy on a large scale.  We have therefore created an expert system, DOCK Blaster, to investigate the feasibility of full automation.  The method requires a PDB code, sometimes with a ligand structure, and from that alone can launch a full screen of large libraries.  A crit. feature is self-assessment, which ests. the anticipated reliability of the automated screening results using pose fidelity and enrichment.  Against common benchmarks, DOCK Blaster recapitulates the crystal ligand pose within 2 Å rmsd 50-60% of the time; inferior to an expert, but respectable.  Half the time the ligand also ranked among the top 5% of 100 phys. matched decoys chosen on the fly.  Further tests were undertaken culminating in a study of 7755 eligible PDB structures.  In 1398 cases, the redocked ligand ranked in the top 5% of 100 property-matched decoys while also posing within 2 Å rmsd, suggesting that unsupervised prospective docking is viable.  DOCK Blaster is available at http://blaster.docking.org.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPZAd6CrHGp7Vg90H21EOLACvtfcHk0ljYHgPB4IhWLg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVOjurjN&md5=f302db1f77b87f26d1fa2fece55bca96</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1021%2Fjm9006966&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm9006966%26sid%3Dliteratum%253Aachs%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DMysinger%26aufirst%3DM.%2BM.%26aulast%3DHuang%26aufirst%3DN.%26aulast%3DColizzi%26aufirst%3DF.%26aulast%3DWassam%26aufirst%3DP.%26aulast%3DCao%26aufirst%3DY.%26atitle%3DAutomated%2520docking%2520screens%253A%2520a%2520feasibility%2520study%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D5712%26epage%3D5720%26doi%3D10.1021%2Fjm9006966" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref70"><div class="reference"><strong class="refLabel"><a href="#ref70" class="rightTabRefNumLink">70</a></strong><div class="NLM_citation" id="rightTab-cit70"><span><span class="NLM_contrib-group">Powers, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morandi, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Structure-based discovery of a novel, noncovalent inhibitor of AmpC beta-lactamase</span> <span class="citation_source-journal">Structure</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">1013</span><span class="NLM_x">–</span> <span class="NLM_lpage">1023</span><span class="refDoi"> DOI: 10.1016/S0969-2126(02)00799-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1016%2FS0969-2126%2802%2900799-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=12121656" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BD38XlsVyjsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2002&pages=1013-1023&author=R.+A.+Powersauthor=F.+Morandiauthor=B.+K.+Shoichet&title=Structure-based+discovery+of+a+novel%2C+noncovalent+inhibitor+of+AmpC+beta-lactamase&doi=10.1016%2FS0969-2126%2802%2900799-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Discovery of a Novel, Noncovalent Inhibitor of AmpC β-Lactamase</span></div><div class="casAuthors">Powers, Rachel A.; Morandi, Federica; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1013-1023</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">β-Lactamases are the most widespread resistance mechanisms to β-lactam antibiotics, and there is a pressing need for novel, non-β-lactam drugs.  A database of over 200,000 compds. was docked to the active site of AmpC β-lactamase to identify potential inhibitors.  Fifty-six compds. were tested, and three had Ki values of 650 μM or better.  The best of these, 3-[(4-chloroanilino)sulfonyl]thiophene-2-carboxylic acid, was a competitive noncovalent inhibitor (Ki = 26 μM), which also reversed resistance to β-lactams in bacteria expressing AmpC.  The structure of AmpC in complex with this compd. was detd. by x-ray crystallog. to 1.94 A and reveals that the inhibitor interacts with key active-site residues in sites targeted in the docking calcn.  Indeed, the exptl. detd. conformation of the inhibitor closely resembles the prediction.  The structure of the enzyme-inhibitor complex presents an opportunity to improve binding affinity in a novel series of inhibitors discovered by structure-based methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKX-gft1lXA7Vg90H21EOLACvtfcHk0lj0RbiVeoOGww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XlsVyjsrk%253D&md5=5f539484d08191f0c65626493fc1b262</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1016%2FS0969-2126%2802%2900799-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0969-2126%252802%252900799-2%26sid%3Dliteratum%253Aachs%26aulast%3DPowers%26aufirst%3DR.%2BA.%26aulast%3DMorandi%26aufirst%3DF.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DStructure-based%2520discovery%2520of%2520a%2520novel%252C%2520noncovalent%2520inhibitor%2520of%2520AmpC%2520beta-lactamase%26jtitle%3DStructure%26date%3D2002%26volume%3D10%26spage%3D1013%26epage%3D1023%26doi%3D10.1016%2FS0969-2126%2802%2900799-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref71"><div class="reference"><strong class="refLabel"><a href="#ref71" class="rightTabRefNumLink">71</a></strong><div class="NLM_citation" id="rightTab-cit71"><span><span class="NLM_contrib-group">Winter, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lobley, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prince, S. M.</span><span> </span><span class="NLM_article-title">Decision making in xia2</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">69</span><span class="NLM_x">, </span> <span class="NLM_fpage">1260</span><span class="NLM_x">–</span> <span class="NLM_lpage">1273</span><span class="refDoi"> DOI: 10.1107/S0907444913015308</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1107%2FS0907444913015308" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=23793152" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpvVejsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2013&pages=1260-1273&author=G.+Winterauthor=C.+M.+Lobleyauthor=S.+M.+Prince&title=Decision+making+in+xia2&doi=10.1107%2FS0907444913015308"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Decision making in xia2</span></div><div class="casAuthors">Winter, Graeme; Lobley, Carina M. C.; Prince, Stephen M.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1260-1273</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Xia2 is an expert system for the automated redn. of macromol. crystallog. (MX) data employing well trusted existing software.  The system can process a full MX data set consisting of one or more sequences of images at one or more wavelengths from images to structure-factor amplitudes with no user input.  To achieve this many decisions are made, the rationale for which is described here.  In addn., it is crit. to support the testing of hypotheses and to allow feedback of results from later stages in the anal. to earlier points where decisions were made: the flexible framework employed by xia2 to support this feedback is summarized here.  While the decision-making protocols described here were developed for xia2, they are equally applicable to interactive data redn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrMCxz3Jkl1CrVg90H21EOLACvtfcHk0lj0RbiVeoOGww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXpvVejsro%253D&md5=48d02842ee5967636a5f6042cfd90b25</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1107%2FS0907444913015308&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444913015308%26sid%3Dliteratum%253Aachs%26aulast%3DWinter%26aufirst%3DG.%26aulast%3DLobley%26aufirst%3DC.%2BM.%26aulast%3DPrince%26aufirst%3DS.%2BM.%26atitle%3DDecision%2520making%2520in%2520xia2%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2013%26volume%3D69%26spage%3D1260%26epage%3D1273%26doi%3D10.1107%2FS0907444913015308" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref72"><div class="reference"><strong class="refLabel"><a href="#ref72" class="rightTabRefNumLink">72</a></strong><div class="NLM_citation" id="rightTab-cit72"><span><span class="NLM_contrib-group">Copeland, R. A.</span> <span class="citation_source-book">Enzymes: A Practical Introduction to Structure, Mechanism, and Data Analysis</span>, <span class="NLM_edition">2</span>nd ed.; <span class="NLM_publisher-name">Wiley</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">2000</span><span class="NLM_x">; </span>p  <span class="NLM_fpage">xvi</span>,  <span class="NLM_fpage">397</span> pp.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1002%2F0471220639" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2000&pages=xvi+397&author=R.+A.+Copeland&title=Enzymes%3A+A+Practical+Introduction+to+Structure%2C+Mechanism%2C+and+Data+Analysis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1002%2F0471220639&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F0471220639%26sid%3Dliteratum%253Aachs%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26btitle%3DEnzymes%253A%2520A%2520Practical%2520Introduction%2520to%2520Structure%252C%2520Mechanism%252C%2520and%2520Data%2520Analysis%26pub%3DWiley%26date%3D2000%26spage%3Dxvi%26spage%3D397" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref73"><div class="reference"><strong class="refLabel"><a href="#ref73" class="rightTabRefNumLink">73</a></strong><div class="NLM_citation" id="rightTab-cit73"><span><span class="NLM_contrib-group">Kawamura, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tumber, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rose, N. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">King, O. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Daniel, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oppermann, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heightman, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C.</span><span> </span><span class="NLM_article-title">Development of homogeneous luminescence assays for histone demethylase catalysis and binding</span> <span class="citation_source-journal">Anal. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">404</span><span class="NLM_x">, </span> <span class="NLM_fpage">86</span><span class="NLM_x">–</span> <span class="NLM_lpage">93</span><span class="refDoi"> DOI: 10.1016/j.ab.2010.04.030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=10.1016%2Fj.ab.2010.04.030" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=404&publication_year=2010&pages=86-93&author=A.+Kawamuraauthor=A.+Tumberauthor=N.+R.+Roseauthor=O.+N.+Kingauthor=M.+Danielauthor=U.+Oppermannauthor=T.+D.+Heightmanauthor=C.+Schofield&title=Development+of+homogeneous+luminescence+assays+for+histone+demethylase+catalysis+and+binding&doi=10.1016%2Fj.ab.2010.04.030"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1016%2Fj.ab.2010.04.030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ab.2010.04.030%26sid%3Dliteratum%253Aachs%26aulast%3DKawamura%26aufirst%3DA.%26aulast%3DTumber%26aufirst%3DA.%26aulast%3DRose%26aufirst%3DN.%2BR.%26aulast%3DKing%26aufirst%3DO.%2BN.%26aulast%3DDaniel%26aufirst%3DM.%26aulast%3DOppermann%26aufirst%3DU.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DSchofield%26aufirst%3DC.%26atitle%3DDevelopment%2520of%2520homogeneous%2520luminescence%2520assays%2520for%2520histone%2520demethylase%2520catalysis%2520and%2520binding%26jtitle%3DAnal.%2520Biochem.%26date%3D2010%26volume%3D404%26spage%3D86%26epage%3D93%26doi%3D10.1016%2Fj.ab.2010.04.030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref74"><div class="reference"><strong class="refLabel"><a href="#ref74" class="rightTabRefNumLink">74</a></strong><div class="NLM_citation" id="rightTab-cit74"><span><span class="NLM_contrib-group">McGovern, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Helfand, B. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">A specific mechanism of nonspecific inhibition</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">4265</span><span class="NLM_x">–</span> <span class="NLM_lpage">4272</span><span class="refDoi"> DOI: 10.1021/jm030266r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm030266r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BD3sXmsFamsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2003&pages=4265-4272&author=S.+L.+McGovernauthor=B.+T.+Helfandauthor=B.+Fengauthor=B.+K.+Shoichet&title=A+specific+mechanism+of+nonspecific+inhibition&doi=10.1021%2Fjm030266r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">A Specific Mechanism of Nonspecific Inhibition</span></div><div class="casAuthors">McGovern, Susan L.; Helfand, Brian T.; Feng, Brian; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">4265-4272</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Promiscuous small mols. plague screening libraries and hit lists.  Previous work has found that several nonspecific compds. form submicrometer aggregates, and it has been suggested that this aggregate species is responsible for the inhibition of many different enzymes.  It is not understood how aggregates inhibit their targets.  To address this question, biophys., kinetic, and microscopy methods were used to study the interaction of promiscuous, aggregate-forming inhibitors with model proteins.  By use of centrifugation and gel electrophoresis, aggregates and protein were found to directly interact.  This is consistent with a subsequent observation from confocal fluorescence microscopy that aggregates conc. green fluorescent protein.  β-Lactamase mutants with increased or decreased thermodn. stability relative to wild-type enzyme were equally inhibited by an aggregate-forming compd., suggesting that denaturation by unfolding was not the primary mechanism of interaction.  Instead, visualization by electron microscopy revealed that enzyme assocs. with the surface of inhibitor aggregates.  This assocn. could be reversed or prevented by the addn. of Triton X-100.  These observations suggest that the aggregates formed by promiscuous compds. reversibly sequester enzyme, resulting in apparent inhibition.  They also suggest a simple method to identify or reverse the action of aggregate-based inhibitors, which appear to be widespread.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppdLVG_GeLZbVg90H21EOLACvtfcHk0lj0RbiVeoOGww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXmsFamsbs%253D&md5=cd0fd6bf0a64ceb7d7219dae2406dd21</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1021%2Fjm030266r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030266r%26sid%3Dliteratum%253Aachs%26aulast%3DMcGovern%26aufirst%3DS.%2BL.%26aulast%3DHelfand%26aufirst%3DB.%2BT.%26aulast%3DFeng%26aufirst%3DB.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DA%2520specific%2520mechanism%2520of%2520nonspecific%2520inhibition%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2003%26volume%3D46%26spage%3D4265%26epage%3D4272%26doi%3D10.1021%2Fjm030266r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref75"><div class="reference"><strong class="refLabel"><a href="#ref75" class="rightTabRefNumLink">75</a></strong><div class="NLM_citation" id="rightTab-cit75"><span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duan, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Torosyan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doak, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ziebart, K. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sterling, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tumanian, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">An Aggregation Advisor for Ligand Discovery</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">7076</span><span class="NLM_x">–</span> <span class="NLM_lpage">7087</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01105</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01105" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlOqu7zM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=7076-7087&author=J.+J.+Irwinauthor=D.+Duanauthor=H.+Torosyanauthor=A.+K.+Doakauthor=K.+T.+Ziebartauthor=T.+Sterlingauthor=G.+Tumanianauthor=B.+K.+Shoichet&title=An+Aggregation+Advisor+for+Ligand+Discovery&doi=10.1021%2Facs.jmedchem.5b01105"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">An Aggregation Advisor for Ligand Discovery</span></div><div class="casAuthors">Irwin, John J.; Duan, Da; Torosyan, Hayarpi; Doak, Allison K.; Ziebart, Kristin T.; Sterling, Teague; Tumanian, Gurgen; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">7076-7087</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Colloidal aggregation of org. mols. is the dominant mechanism for artifactual inhibition of proteins, and controls against it are widely deployed.  Notwithstanding an increasingly detailed understanding of this phenomenon, a method to reliably predict aggregation has remained elusive.  Correspondingly, active mols. that act via aggregation continue to be found in early discovery campaigns and remain common in the literature.  Over the past decade, over 12 thousand aggregating org. mols. have been identified, potentially enabling a precedent-based approach to match known aggregators with new mols. that may be expected to aggregate and lead to artifacts.  We investigate an approach that uses lipophilicity, affinity, and similarity to known aggregators to advise on the likelihood that a candidate compd. is an aggregator.  In prospective exptl. testing, five of seven new mols. with Tanimoto coeffs. (Tc's) between 0.95 and 0.99 to known aggregators aggregated at relevant concns.  Ten of 19 with Tc's between 0.94 and 0.90 and three of seven with Tc's between 0.89 and 0.85 also aggregated.  Another three of the predicted compds. aggregated at higher concns.  This method finds that 61 827 or 5.1% of the ligands acting in the 0.1 to 10 μM range in the medicinal chem. literature are at least 85% similar to a known aggregator with these phys. properties and may aggregate at relevant concns.  Intriguingly, only 0.73% of all drug-like com. available compds. resemble the known aggregators, suggesting that colloidal aggregators are enriched in the literature.  As a percentage of the literature, aggregator-like compds. have increased 9-fold since 1995, partly reflecting the advent of high-throughput and virtual screens against mol. targets.  Emerging from this study is an aggregator advisor database and tool (http://advisor.bkslab.org), free to the community, that may help distinguish between fruitful and artifactual screening hits acting by this mechanism.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTKGJSfIz30bVg90H21EOLACvtfcHk0ljRR1E1QFbR6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlOqu7zM&md5=d3112f9f3a57a2ac61a4b32f31778b1d</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01105%26sid%3Dliteratum%253Aachs%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DDuan%26aufirst%3DD.%26aulast%3DTorosyan%26aufirst%3DH.%26aulast%3DDoak%26aufirst%3DA.%2BK.%26aulast%3DZiebart%26aufirst%3DK.%2BT.%26aulast%3DSterling%26aufirst%3DT.%26aulast%3DTumanian%26aufirst%3DG.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DAn%2520Aggregation%2520Advisor%2520for%2520Ligand%2520Discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D7076%26epage%3D7087%26doi%3D10.1021%2Facs.jmedchem.5b01105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref76"><div class="reference"><strong class="refLabel"><a href="#ref76" class="rightTabRefNumLink">76</a></strong><div class="NLM_citation" id="rightTab-cit76"><span><span class="NLM_contrib-group">Cascella, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mirica, L. M.</span><span> </span><span class="NLM_article-title">Kinetic analysis of iron-dependent histone demethylases: alpha-ketoglutarate substrate inhibition and potential relevance to the regulation of histone demethylation in cancer cells</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">8699</span><span class="NLM_x">–</span> <span class="NLM_lpage">8701</span><span class="refDoi"> DOI: 10.1021/bi3012466</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi3012466" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2012&pages=8699-8701&author=B.+Cascellaauthor=L.+M.+Mirica&title=Kinetic+analysis+of+iron-dependent+histone+demethylases%3A+alpha-ketoglutarate+substrate+inhibition+and+potential+relevance+to+the+regulation+of+histone+demethylation+in+cancer+cells&doi=10.1021%2Fbi3012466"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1021%2Fbi3012466&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi3012466%26sid%3Dliteratum%253Aachs%26aulast%3DCascella%26aufirst%3DB.%26aulast%3DMirica%26aufirst%3DL.%2BM.%26atitle%3DKinetic%2520analysis%2520of%2520iron-dependent%2520histone%2520demethylases%253A%2520alpha-ketoglutarate%2520substrate%2520inhibition%2520and%2520potential%2520relevance%2520to%2520the%2520regulation%2520of%2520histone%2520demethylation%2520in%2520cancer%2520cells%26jtitle%3DBiochemistry%26date%3D2012%26volume%3D51%26spage%3D8699%26epage%3D8701%26doi%3D10.1021%2Fbi3012466" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2VD7" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2VD7','PDB','2VD7'); return false;">PDB: 2VD7</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2OX0" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2OX0','PDB','2OX0'); return false;">PDB: 2OX0</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PDQ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PDQ','PDB','3PDQ'); return false;">PDB: 3PDQ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2GP5" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2GP5','PDB','2GP5'); return false;">PDB: 2GP5</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2GP3" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2GP3','PDB','2GP3'); return false;">PDB: 2GP3</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2OQ7" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2OQ7','PDB','2OQ7'); return false;">PDB: 2OQ7</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2OQ6" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2OQ6','PDB','2OQ6'); return false;">PDB: 2OQ6</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2Q8C" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2Q8C','PDB','2Q8C'); return false;">PDB: 2Q8C</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2YBK" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2YBK','PDB','2YBK'); return false;">PDB: 2YBK</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2WWJ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2WWJ','PDB','2WWJ'); return false;">PDB: 2WWJ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3NJY" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3NJY','PDB','3NJY'); return false;">PDB: 3NJY</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2XML" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2XML','PDB','2XML'); return false;">PDB: 2XML</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5A7N" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5A7N','PDB','5A7N'); return false;">PDB: 5A7N</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5A7O" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5A7O','PDB','5A7O'); return false;">PDB: 5A7O</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5A7P" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5A7P','PDB','5A7P'); return false;">PDB: 5A7P</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5A7Q" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5A7Q','PDB','5A7Q'); return false;">PDB: 5A7Q</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5A7S" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5A7S','PDB','5A7S'); return false;">PDB: 5A7S</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5A7W" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5A7W','PDB','5A7W'); return false;">PDB: 5A7W</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5A80" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5A80','PDB','5A80'); return false;">PDB: 5A80</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i75"><a href="/doi/suppl/10.1021/acs.jmedchem.5b01527">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_11086"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.5b01527">10.1021/acs.jmedchem.5b01527</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Experimental procedure for cloning, expression for all enzymes, MALDI and AlphaScreen counterscreening; IC<sub>50</sub> for selected 5-aminosalicylates series of inhibitors; structures of hybrid scaffolds that were virtually screened; crystallography statistics for compounds <b>43</b>, <b>30</b>, <b>44</b>, <b>42</b>, <b>35</b>, <b>40</b>, and <b>36</b>; omit maps and 2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> maps for all compounds; figures showing docking targets used, docking binding site used, and superimposition of hybrid compounds with structure <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2OQ6">2OQ6</a> showing H3 histone peptide; counterscreening confirming specific inhibition and binding of compounds through an aggregation test, DLS test, and enzymes <i>T</i><sub>m</sub> stabilization curves (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.5b01527/suppl_file/jm5b01527_si_001.pdf">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b01527/suppl_file/jm5b01527_si_001.pdf">jm5b01527_si_001.pdf (2.18 MB)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Atomic coordinates and structure factors for the KDM4A complexes have been deposited in the Protein Data Bank as entries <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5A7N">5A7N</a> in complex with compound <b>43</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5A7O">5A7O</a> with <b>42</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5A7P">5A7P</a> with <b>36</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5A7Q">5A7Q</a> with <b>30</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5A7S">5A7S</a> with <b>44</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5A7W">5A7W</a> with <b>35</b>, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5A80">5A80</a> with <b>40</b>.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.5b01527&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2016.59.issue-4%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.5b01527%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.5b01527" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                8MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679a54ea8f153da6","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
