
<html lang="en"     class="pb-page"  data-request-id="f2f2170d-d893-4d92-87aa-f34a3f67344b"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;issue:issue:10.1021/jmcmar.2014.57.issue-14;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;article:article:10.1021/jm500249n"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="KDM4B as a Target for Prostate Cancer: Structural Analysis and Selective Inhibition by a Novel Inhibitor" /></meta><meta name="dc.Creator" content="Chia-Han  Chu" /></meta><meta name="dc.Creator" content="Ling-Yu  Wang" /></meta><meta name="dc.Creator" content="Kai-Cheng  Hsu" /></meta><meta name="dc.Creator" content="Chung-Chin  Chen" /></meta><meta name="dc.Creator" content="Hsing-Hung  Cheng" /></meta><meta name="dc.Creator" content="Szu-Min  Wang" /></meta><meta name="dc.Creator" content="Chien-Ming  Wu" /></meta><meta name="dc.Creator" content="Tsan-Jan  Chen" /></meta><meta name="dc.Creator" content="Ling-Ting  Li" /></meta><meta name="dc.Creator" content="Ruiwu  Liu" /></meta><meta name="dc.Creator" content="Chiu-Lien  Hung" /></meta><meta name="dc.Creator" content="Jing-Moon  Yang" /></meta><meta name="dc.Creator" content="Hsing-Jien  Kung" /></meta><meta name="dc.Creator" content="Wen-Ching  Wang" /></meta><meta name="dc.Description" content="The KDM4/JMJD2 Jumonji C-containing histone lysine demethylases (KDM4A–KDM4D), which selectively remove the methyl group(s) from tri/dimethylated lysine 9/36 of H3, modulate transcriptional activat..." /></meta><meta name="Description" content="The KDM4/JMJD2 Jumonji C-containing histone lysine demethylases (KDM4A–KDM4D), which selectively remove the methyl group(s) from tri/dimethylated lysine 9/36 of H3, modulate transcriptional activat..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="July 9, 2014" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/jm500249n" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2014 American Chemical Society" /></meta><meta name="citation_fulltext_world_readable" content=""/></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Fjm500249n" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Fjm500249n" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Fjm500249n" /></link>
        
    
    

<title>KDM4B as a Target for Prostate Cancer: Structural Analysis and Selective Inhibition by a Novel Inhibitor | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/jm500249n" /></meta><meta property="og:title" content="KDM4B as a Target for Prostate Cancer: Structural Analysis and Selective Inhibition by a Novel Inhibitor" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm500249n/20150627/images/large/jm-2014-00249n_0010.jpeg" /></meta><meta property="og:description" content="The KDM4/JMJD2 Jumonji C-containing histone lysine demethylases (KDM4A–KDM4D), which selectively remove the methyl group(s) from tri/dimethylated lysine 9/36 of H3, modulate transcriptional activation and genome stability. The overexpression of KDM4A/KDM4B in prostate cancer and their association with androgen receptor suggest that KDM4A/KDM4B are potential progression factors for prostate cancer. Here, we report the crystal structure of the KDM4B·pyridine 2,4-dicarboxylic acid·H3K9me3 ternary complex, revealing the core active-site region and a selective K9/K36 site. A selective KDM4A/KDM4B inhibitor, 4, that occupies three subsites in the binding pocket is identified by virtual screening. Pharmacological and genetic inhibition of KDM4A/KDM4B significantly blocks the viability of cultured prostate cancer cells, which is accompanied by increased H3K9me3 staining and transcriptional silencing of growth-related genes. Significantly, a substantial portion of differentially expressed genes are AR-responsive, consistent with the roles of KDM4s as critical AR activators. Our results point to KDM4 as a useful therapeutic target and identify a new inhibitor scaffold." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/jm500249n"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/jm500249n">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Fjm500249n&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Fjm500249n&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/jm500249n&amp;href=/doi/10.1021/jm500249n" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2014</span><span class="cit-fg-volume">, 57</span><span class="cit-fg-issue">, 14</span><span class="cit-fg-pageRange">, 5975-5985</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/57/14" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/jm500218x" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/jm500272r" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">KDM4B as a Target for Prostate Cancer: Structural Analysis and Selective Inhibition by a Novel Inhibitor</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Chia-Han++Chu">Chia-Han Chu</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Ling-Yu++Wang">Ling-Yu Wang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Kai-Cheng++Hsu">Kai-Cheng Hsu</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Chung-Chin++Chen">Chung-Chin Chen</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Hsing-Hung++Cheng">Hsing-Hung Cheng</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Szu-Min++Wang">Szu-Min Wang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Chien-Ming++Wu">Chien-Ming Wu</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Tsan-Jan++Chen">Tsan-Jan Chen</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Ling-Ting++Li">Ling-Ting Li</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Ruiwu++Liu">Ruiwu Liu</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Chiu-Lien++Hung">Chiu-Lien Hung</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jing-Moon++Yang">Jing-Moon Yang</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Hsing-Jien++Kung">Hsing-Jien Kung</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>⊥</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Wen-Ching++Wang">Wen-Ching Wang</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-text"><sup>†</sup>Institute of Molecular and Cellular Biology and Department of Life Sciences and <sup>‡</sup>Biomedical Science and Engineering Center, National Tsing-Hua University, Hsinchu, 30013, Taiwan</span></div><div class="aff-info" id="aff3"><span class="aff-symbol">§</span> <span class="aff-text">Department of Biochemistry and Molecular Medicine, University of California Davis School of Medicine, University of California Davis Cancer Center, Sacramento, California 95817, United States</span></div><div class="aff-info" id="aff4"><span class="aff-symbol">∥</span> <span class="aff-text">Institute of Bioinformatics and Systems Biology, National Chiao Tung University, Hsinchu, 30050, Taiwan</span></div><div class="aff-info" id="aff5"><span class="aff-symbol">⊥</span> <span class="aff-text">National Health Research Institutes, Miaoli, 35053, Taiwan</span></div><div class="corresp-info"><strong>*</strong>(J.-M.Y.) E-mail: <a href="/cdn-cgi/l/email-protection#4f222020210f292e2c3a233b3661212c3b3a612a2b3a613b38"><span class="__cf_email__" data-cfemail="7c111313123c1a1d1f0910080552121f08095219180952080b">[email protected]</span></a>; Phone: +886-3-5715131 (ext 56942); Fax: +886-3-5729288.</div><div class="corresp-info"><strong>*</strong>(H.-J.K.) E-mail: <a href="/cdn-cgi/l/email-protection#bbd3d0ced5dcfbd5d3c9d295d4c9dc95cfcc"><span class="__cf_email__" data-cfemail="137b78667d74537d7b617a3d7c61743d6764">[email protected]</span></a>; Phone: +886-3-7246166 (ext 31000); Fax: +886-3-7586402.</div><div class="corresp-info"><strong>*</strong>(W.-C.W.) E-mail: <a href="/cdn-cgi/l/email-protection#73041004121d14331f1a15165d1d071b065d1617065d0704"><span class="__cf_email__" data-cfemail="27504450464940674b4e41420949534f5209424352095350">[email protected]</span></a> and <a href="/cdn-cgi/l/email-protection#d3a4b6bdb0bbbabdb4fda4b093b4beb2babffdb0bcbe"><span class="__cf_email__" data-cfemail="196e7c777a7170777e376e7a597e74787075377a7674">[email protected]</span></a>; Phone: +886-3-5742766; Fax: +886-3-5742766 and +886-3-5717237.</div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Fjm500249n&amp;href=/doi/10.1021%2Fjm500249n" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2014</span></span><span class="cit-volume">, 57</span><span class="cit-issue">, 14</span><span class="cit-pageRange">, 5975–5985</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">June 27, 2014</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>15 February 2014</li><li><span class="item_label"><b>Published</b> online</span>9 July 2014</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 24 July 2014</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/jm500249n" title="DOI URL">https://doi.org/10.1021/jm500249n</a></div><div class="article_header-article-copyright"><strong>Copyright © 2014 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/page/rightslinkno.jsp">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"><a href="/page/policy/authorchoice_termsofuse.html"><img src="/pb-assets/images/programsLogos/OA_Icons_Ac-1612971054130.svg" alt="ACS AuthorChoice" /></img><span>ACS AuthorChoice</span></a></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/jm500249n"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">4338</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">56</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/jm500249n" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;KDM4B as a Target for Prostate Cancer: Structural Analysis and Selective Inhibition by a Novel Inhibitor&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Chia-Han&quot;,&quot;last_name&quot;:&quot;Chu&quot;},{&quot;first_name&quot;:&quot;Ling-Yu&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Kai-Cheng&quot;,&quot;last_name&quot;:&quot;Hsu&quot;},{&quot;first_name&quot;:&quot;Chung-Chin&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Hsing-Hung&quot;,&quot;last_name&quot;:&quot;Cheng&quot;},{&quot;first_name&quot;:&quot;Szu-Min&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Chien-Ming&quot;,&quot;last_name&quot;:&quot;Wu&quot;},{&quot;first_name&quot;:&quot;Tsan-Jan&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Ling-Ting&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Ruiwu&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Chiu-Lien&quot;,&quot;last_name&quot;:&quot;Hung&quot;},{&quot;first_name&quot;:&quot;Jing-Moon&quot;,&quot;last_name&quot;:&quot;Yang&quot;},{&quot;first_name&quot;:&quot;Hsing-Jien&quot;,&quot;last_name&quot;:&quot;Kung&quot;},{&quot;first_name&quot;:&quot;Wen-Ching&quot;,&quot;last_name&quot;:&quot;Wang&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2014&quot;,&quot;month&quot;:&quot;07&quot;,&quot;day&quot;:&quot;09&quot;,&quot;issue&quot;:&quot;14&quot;,&quot;volume&quot;:&quot;57&quot;,&quot;pages&quot;:&quot;5975-5985&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/jm500249n&quot;},&quot;abstract&quot;:&quot;The KDM4/JMJD2 Jumonji C-containing histone lysine demethylases (KDM4A–KDM4D), which selectively remove the methyl group(s) from tri/dimethylated lysine 9/36 of H3, modulate transcriptional activation and genome stability. The overexpression of KDM4A/KDM4B in prostate cancer and their association with androgen receptor suggest that KDM4A/KDM4B are potential progression factors for prostate cancer. Here, we report the crystal structure of the KDM4B·pyridine 2,4-dicarboxylic acid·H3K9me3 ternary complex, revealing the core active-site region and a selective K9/K36 site. A selective KDM4A/KDM4B inhibitor, 4, that occupies three subsites in the binding pocket is identified by virtual screening. Pharmacological and genetic inhibition of KDM4A/KDM4B significantly blocks the viability of cultured prostate cancer cells, which is accompanied by increased H3K9me3 staining and transcriptional silencing of growth-related genes. Significantly, a substantial portion of differentially expressed genes are AR-responsive, cons&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Fjm500249n&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm500249n" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm500249n&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=jm500249n" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm500249n&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=jm500249n" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Fjm500249n&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm500249n&amp;href=/doi/10.1021/jm500249n" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/jm500249n" /></input><a href="/doi/pdf/10.1021/jm500249n" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (5 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/jm500249n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Fjm500249n%26sid%3Dliteratum%253Aachs%26pmid%3D24971742%26genre%3Darticle%26aulast%3DChu%26date%3D2014%26atitle%3DKDM4B%2Bas%2Ba%2BTarget%2Bfor%2BProstate%2BCancer%253A%2BStructural%2BAnalysis%2Band%2BSelective%2BInhibition%2Bby%2Ba%2BNovel%2BInhibitor%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D57%26issue%3D14%26spage%3D5975%26epage%3D5985%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=290896" title="Cells">Cells</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=292258" title="Cancer">Cancer</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291439" title="Genetics">Genetics</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/57/14" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jmcmar.2014.57.issue-14/production/jmcmar.2014.57.issue-14.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm500249n/20150627/images/medium/jm-2014-00249n_0010.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm500249n/20150627/images/large/jm-2014-00249n_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500249n&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">The KDM4/JMJD2 Jumonji C-containing histone lysine demethylases (KDM4A–KDM4D), which selectively remove the methyl group(s) from tri/dimethylated lysine 9/36 of H3, modulate transcriptional activation and genome stability. The overexpression of KDM4A/KDM4B in prostate cancer and their association with androgen receptor suggest that KDM4A/KDM4B are potential progression factors for prostate cancer. Here, we report the crystal structure of the KDM4B·pyridine 2,4-dicarboxylic acid·H3K9me3 ternary complex, revealing the core active-site region and a selective K9/K36 site. A selective KDM4A/KDM4B inhibitor, <b>4</b>, that occupies three subsites in the binding pocket is identified by virtual screening. Pharmacological and genetic inhibition of KDM4A/KDM4B significantly blocks the viability of cultured prostate cancer cells, which is accompanied by increased H3K9me3 staining and transcriptional silencing of growth-related genes. Significantly, a substantial portion of differentially expressed genes are AR-responsive, consistent with the roles of KDM4s as critical AR activators. Our results point to KDM4 as a useful therapeutic target and identify a new inhibitor scaffold.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_01805" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_01805" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Histone lysine demethylases (KDMs), which regulate a dynamic, reversible status of “methyl” histone codes, have gained much attention since the first discovery of KDM1/LSD1 in 2004.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Mutations, amplifications, deletions, and aberrant expression of KDMs have been identified in a variety of cancers, and their roles in modulating the behavior of cancer cells have been substantiated.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> As such, increasing attention has been paid to evaluating KDMs as potential therapeutic targets for cancer.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> There are now eight KDM families, including 28 members that have been identified (for a review, see ref <a onclick="showRef(event, 'cit3b'); return false;" href="javascript:void(0);" class="ref cit3b">3b</a>). KDM2–KDM8 constitute a large superfamily that shares a Jumonji C (JmjC) domain, which functions as an α-ketoglutarate (AKG) and Fe(II)-dependent demethylase. Notably, each family exhibits its exquisite substrate specificity toward different histone lysine residues, thereby effectively integrating upstream signals and modulating chromatin conformation.<a onclick="showRef(event, 'cit1c'); return false;" href="javascript:void(0);" class="ref cit1c">(1c)</a></div><div class="NLM_p">Among these, the largest gene family, KDM4 (four paralogues, KDM4A–KDM4D, and two pseudogenes, KDM4E and KDM4F), has been shown to be an eraser of a repressive mark, H3K9me3/me2, whereas its subfamily, KDM4A–KDM4C, also demethylates H3K36me3/me2.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> KDM4A and KDM4B are overexpressed in a variety of cancers including prostate, breast, colorectal, lung, gastric, esophageal, lymphoma, renal, and medulloblastoma.<a onclick="showRef(event, 'cit4b'); return false;" href="javascript:void(0);" class="ref cit4b">(4b)</a> For prostate and breast cancers, this family of demethylases has the added significance of being coactivators of androgen receptor (AR) (KDM4A–KDM4D)<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> and estrogen receptor (ER) (KDM4A and KDM4B).<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> They function to stimulate the transcriptional potential of the receptors. KDM4B also regulates the turnover of AR.<a onclick="showRef(event, 'cit5c'); return false;" href="javascript:void(0);" class="ref cit5c">(5c)</a> Given the important roles of AR and ER in prostate and breast carcinogenesis, KDM4A/KDM4B are considered to be promising drug targets for intervening in these malignancies.<a onclick="showRef(event, 'cit3b cit4b'); return false;" href="javascript:void(0);" class="ref cit3b cit4b">(3b, 4b)</a></div><div class="NLM_p last">Thus far, the inhibitors described for KDM4 proteins are largely AKG analogues: <i>N</i>-oxalylglycines (OGAs), including OGA (<b>1</b>), which inhibits KDM4A, KDM4C, and KDM4D,<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> pyridine 2,4-dicarboxylic acids (PD2s) developed based on KDM4E,<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> and 8-hydroxyquinolines (8HQs), including <b>3</b> (8HQ).<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> Yet, as a prodrug, the cytotoxic IC<sub>50</sub> of compound <b>2</b> (PD2) is in the millimolar range in cultured cells because of its poor cell-penetrating ability,<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> whereas 5-carboxy-8HQ, with a potent inhibitory effect on KDM4E in vitro (IC<sub>50</sub> = 0.2 μM), exhibits a relatively high cytotoxic IC<sub>50</sub> in HeLa cells (86.5 μM).<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> In this investigation, we determined the KDM4B·<b>2</b>·H3K9me3 ternary complex structure and applied a structure-guided strategy to identify a selective inhibitor, 1,5-bis[(<i>E</i>)-2-(3,4-dichlorophenyl)ethenyl]-2,4-dinitrobenzene (NSC636819), toward KDM4A and KDM4B. Kinetic analysis reveals that compound <b>4</b> (NSC636819) is a competitive inhibitor of KDM4A/KDM4B. Furthermore, we report that pharmacological and genetic inhibition of KDM4A/KDM4B significantly lowers the viability of prostate cancer cells, principally because of its potency toward inhibiting the AR transcriptional network.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Materials and Methods</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_01857" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_01857" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3"> Cloning, Expression, and Purification</h3><div class="NLM_p last">Human KDM4B (1–348) and KDM4A (1–347) were PCR-amplified from cDNAs using the following primers: KDM4B 1–348 forward, 5′-AAACATATGGGGTCTGAGGACCACGGCGCC-3′ (<i>Nde</i>I); KDM4B 1–348 reverse, 5′-AAAAAACTCGGGGCTCTCGAGCTACGTGGGCCG-3′ (<i>Xho</i>I); KDM4A 1–347 forward, 5′-AAACATATGGCGAGCGAAAGCGAAACTCTG-3′ (<i>Nde</i>I); and KDM4A reverse, 5′-AAAGGATCCCTACGTGGGCAGAGTATGGTC-3′ (<i>Bam</i>HI). The KDM4E catalytic domain (residues 1–347) was subcloned from pcDNA-KDM4E, a gift of Dr. Yoshihiro Izumiya (University of California Davis School of Medicine, Sacramento, CA), using the primers 5′-GGGGCTAGCATGAAGTCTGTGCACTC-3′ (<i>Nhe</i>I) and 5′-CCTCTCGAGCTAGGGCTCTGTGTGTTC-3′ (<i>Xho</i>I). PCR was performed with the HiFi DNA polymerase kit using a C1000 Touch thermal cycler (Bio-Rad Laboratories, Inc., USA). The amplified products were inserted into pET28a or pET30a (Novagen, Inc., USA) to generate pET28a-KDM4A(1–347), pET28a-KDM4B(1–348), and pET28a-KDM4E(1–347). The KDM4D construct containing the catalytic domain (residues 1–350) was a gift from Dr. Shi Yang (Harvard Medical School, Boston, MA). Expression of protein in <i>Escherichia coli</i> BL21 (DE3) was induced by addition of 0.5 mM isopropyl-β-<span class="smallcaps smallerCapital">d</span>-thiogalactopyranoside (IPTG) at 16 °C for 21 h. The His6-tagged proteins were purified by a nickel affinity column (Ni Sepharose high performance, GE Healthcare). The protein was concentrated and further purified by a 16/60 Superdex 75 gel filtration column equilibrated with 50 mM HEPES, pH 7.5, and 500 mM NaCl. The protein purity was analyzed by SDS-PAGE. Protein concentration was assayed by the Bradford method using bovine serum albumin as the standard.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a></div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Enzyme Assay</h3><div class="NLM_p last">A formaldehyde dehydrogenase (FDH)-coupled demethylase assay was used to determine demethylase activity and to select potent inhibitors. All inhibitors were dissolved in dimethyl sulfoxide (DMSO) at various concentrations and added to the mixture such that the final DMSO concentration was 5%. The reagents for the demethylase reactions were dissolved in HEPES buffer (50 mM, pH 7.5), with the exception of Fe(II) solutions, which were made using (NH4)<sub>2</sub>Fe(SO<sub>4</sub>)<sub>2</sub> dissolved in 20 mM HCl to make a 400 mM stock solution. All reagents were stored at −30 °C. FDH, NAD+, TKQTARK(Me)<sub>3</sub>STGGKAPR (H3<sub>3–17</sub>K9me3), STGGVK(Me)<sub>3</sub>KPHRY (H3<sub>31–41</sub>K36me3), or ARTK(Me)<sub>3</sub>QTARK(Me)<sub>2</sub>STGGKAPRKQLATKA (H3<sub>1–24</sub>K4me3K9me2) peptides (Kelowna Int. Sci. Inc.), DMSO, and the demethylase enzyme were added first to 96-well black immuno plate (SPL Life Science) and incubated together on ice for 15 min. Then, the plate was put into a FLUOStar OPTIMA ELISA reader (BMG LABTECH) at 37 °C, and the reaction was started by adding ascorbic acid (ascorbate), Fe(II), and AKG to final concentrations of 50 mM HEPES, pH 7.5, 2 μM of KDM4B, 5% DMSO, 0.01 U FDH (Sigma), 1 mM NAD+, 1 mM AKG, 2 mM ascorbate, 50 μM Fe(II), and various concentration of H3K9me3 peptide; the final volume was 50 μL. Each reaction was incubated at 37 °C for 30 min, and the production of NADH was detected by fluorescence (ex 360/em 470).</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Crystallization</h3><div class="NLM_p last">Crystallization was performed by the hanging-drop vapor-diffusion method at 4 °C. Equal volumes of a protein sample and the reservoir solution were mixed. Initial crystallization screening was automated using a Oryx8 robot (Douglas Instruments, UK) and the reagents of seven sets of crystallization kits: Crystal Screen I and II kits (Hampton Research), Index kit (Hampton Research), Clear Strategy Screen I and II kits (Molecular Dimension), Wizard kit (Emerald), and JB Screen classic HTS I and II kits (Jena Bioscience). Crystals of KDM4B (10 mg/mL protein, 4 mM compound <b>2</b> and 5 mM H3K9me3 peptide) were grown in 0.1 M MES (pH 6.5), 0.2 M magnesium acetate, and 20% (w/v) poly(ethylene glycol) (PEG) 8000. Optimized crystals used for diffraction (12 mg/mL within 4 mM <b>2</b> and 5 mM H3K9me3 peptide) were grown in 0.1 M MES (pH 6.5), 0.2 M magnesium acetate, 24% (w/v) PEG 8000. The crystal diffracted to 1.87 Å, belonged to space group <i>P</i>212121, and had unit cell dimensions of <i>a</i> = 54.36, <i>b</i> = 78.48, <i>c</i> = 83.89 Å. The asymmetric unit contained one molecule.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> X-ray Data Collection and Processing</h3><div class="NLM_p last">Crystals were flash-frozen in a stream of liquid nitrogen and then screened and characterized using an RU-300 rotating-anode X-ray generator (Rigaku/MSC Inc., USA) at the Macromolecular X-ray Crystallographic Laboratory of National Tsing Hua University, Taiwan. The KDM4B·<b>2</b>·H3K9me3 data set was collected at the SPring-8 BL12B2 beamline, Japan, with an ADSC Quantum 4R detector. All data sets were indexed, integrated, and scaled using HKL-2000.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> Data collection statistics are shown in Table S2 of the <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Structure Determination and Refinement</h3><div class="NLM_p last">Crystallographic refinement used the maximum-likelihood target function module in REFMAC5.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> The KDM4B·<b>2</b>·H3K9me3 structures were constructed by MOLREP with KDM4A (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2YBS">2YBS</a>) as the template<a onclick="showRef(event, 'ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref13 ref14">(13, 14)</a> and were refined using REFMAC5 coupled with ARP/wARP,<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> which automatically added water molecules. The 2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> electron density maps were generated by FFT and plotted by PyMOL. The validities of the KDM4B·<b>2</b>·H3K9me3 structure were assessed by PROCHECK.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a></div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Structural Comparison</h3><div class="NLM_p last">The KDM4B structure was compared with protein structures in the DALI server (<a href="http://ekhidna.biocenter.helsinki.fi/dali_server/" class="extLink">http://ekhidna.biocenter.helsinki.fi/dali_server/</a>). The structures of KDM4A·H3<sub>1–17</sub>K9me3 (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2P5B">2P5B</a>),<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> KDM4B·<b>2</b> (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4LXL">4LXL</a>; this study), KDM4C·<b>1</b> (OGA) (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2XML">2XML</a>), KDM4D·AKG·H3<sub>6–15</sub>K9me3 (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4HON">4HON</a>),<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> and KDM4E·<b>2</b> (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2W2I">2W2I</a>) were superimposed by LSQMAN in O.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> ESPript was used for the combined sequence as well as for secondary structure alignments and figure preparation.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> PyMol (<a href="http://www.pymol.org" class="extLink">http://www.pymol.org</a>) was used to prepare the figures.</div></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Virtual Screening</h3><div class="NLM_p last">The binding site for virtual docking screening of putative inhibitors was prepared by including protein atoms located in a ≤10 Å radius sphere centered around the bound ligand of KDM4A (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2YBK">2YBK</a><a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a>). We utilized GEMDOCK<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> to screen the NCI database (236 962 compounds). Top ranked compounds available from the Developmental Therapeutics Program of the National Cancer Institute were selected for testing in the KDM4A/KDM4B inhibitory assay.</div></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Cell Culture</h3><div class="NLM_p last">Primary PrEC cells were purchased from Clonetics (Walkersville, MD) and cultured in serum-free prostate epithelial cell growth medium following the vendor’s directions. Cell lines RWPE1, LNCaP (LNCaP-FGC), CWR22Rv1 (22Rv1), VCaP, DU145, PC3 (all purchased from ATCC), and PNT2 (Sigma-Aldrich, MO) were cultured under their recommended conditions. CWR-R1<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(23)</a> and LNCaP derived C4-2 and C4-2B cells<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(24)</a> were cultured in RPMI1640 medium containing 10% FBS.</div></div><div id="sec2_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> RNA Interference and Quantitative RT-PCR</h3><div class="NLM_p last">Lentiviral vector pLKO.1 carrying sequences encoding a shRNA that specifically targets KDM4A and KDM4B gene (TRC library clone nos. TRCN0000234910 and TRCN000018014) were cotransfected with viral packaging plasmids in 293T cells to generate the shRNA lentiviral particles. Empty pLKO.1 plasmid was used as negative control. The lentiviral supernatant was collected 48 h after transfection and concentrated by Lenti-X Concentrator (Clontech, CA). The precipitated viral particles were resuspended in fresh RPMI1640 medium with 10% FBS for subsequent LNCaP infection and were transduced into LNCaP cells for 72 h. Cells were then harvested, and total RNA was isolated, followed by cDNA synthesis and real-time PCR analysis using iQ5 iCycler thermal cycler (Bio-Rad, CA). Threshold cycle values were normalized against actin transcript level. Individual samples were performed in triplicate and converted to relative gene expression using QGene96 software (<a href="http://www.gene-quantification.de/download.html#qgene" class="extLink">http://www.gene-quantification.de/download.html#qgene</a>). Primer sequences used are as follows: KDM4A-F, 5′-AGGAGAGTGAACTGCCTCCA-3′; KDM4A-R, 5′-GGTCTCCTTCCTCTCCATCC; KDM4B-F, 5′-TCACGCAGTACAATATCCAG-3′; KDM4B-R, 5′-TCGTCATCATACAAAGAGCC; actin-F, 5′-GTACCACTGGCATCGTGATGGACT-3′; and actin-R, 5′CCGCTCATTGCCAATGGTGAT-3′.</div></div><div id="sec2_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Cell Proliferation Assay</h3><div class="NLM_p last">LNCaP cells were seeded in a 48-well plate 1 day prior to lentivirus infection. After being subjected to the shRNA lentivirus (day 0), cell proliferation was measured every 2 days by MTT colorimetric assay according to the manufacturer’s instruction (Roche, IN).</div></div><div id="sec2_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Immunoblotting and Flow Cytometry</h3><div class="NLM_p last">Total cell lysates were obtained by lysing the cells with buffer (50 mM Tris-HCl, pH7.5, 150 mM NaCl, 0.5% Triton X-100, 10% glycerol, 1 mM EDTA, protease inhibitors) for 15 min on ice, followed by 10 min of sonication cycling (30 s on, 30 s off) on ice. The level of total histone H3 and trimethylated histone H3 Lys9 was analyzed by western blotting using anti-histone H3 (Cell Signaling, 4499) and anti-H3K9me3 (Millipore, 61021) antibodies. Mock- and inhibitor-treated cells were harvested and fixed with 70% ethanol for >4 h at −20 °C, followed by propidium iodide (Sigma-Aldrich) staining. The DNA content was analyzed by Becton Dickinson FACScan flow cytometry, and the sub-G1 population was quantified by WinMDI 2.9.</div></div><div id="sec2_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Microarray</h3><div class="NLM_p last">LNCaP cells treated with mock or inhibitor <b>4</b> for 3 days were harvested, and the total RNA was extracted using TRIzol reagent (Life Technologies, NY). Microarray analysis was performed by the University of California Davis Cancer Center Gene Expression Resource, using Affymetrix Human Genome U133A (HG-U133A) GeneChip arrays (Affymetrix, CA), which permits expression analysis of the entire Genbank RefSeq database. Array scanning and generation of raw signal data files were done with GeneChip operating software (Affymetrix). Subsequent data analysis was done by GeneSpring (Agilent Technologies, CA) and DAVID bioinformatic resources 6.7 (NIH).</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_80740" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_80740" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec3_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> KDM4B·<b>2</b>·H3K9me3 Crystal Structure</h3><div class="NLM_p">In an effort to understand the detailed structure–function relationship of KDM4B at an atomic resolution, the recombinant catalytic domain of KDM4B was subjected to crystallization in the presence of a peptide, Ni(II), and AKG or compound <b>2</b>. After extensive trials, a well-diffracting crystal was found to consist of a large piece of residual density in the binding pocket, which could be modeled as the H3K9me3 peptide, compound <b>2</b>, and Ni (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>A). The final crystal structure shows a 1.87 Å resolution monomer (<i>R</i> = 21.8%; <i>R</i><sub>free</sub> = 26.2%) that consists of the KDM4B catalytic domain (residues 9–337), compound <b>2</b>, and an H3K9me3 peptide (residues 7–14) within the active site (Table S2 of the <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>). A Ni(II) ion is located on a site corresponding to the Fe(II) position on the bottom of the catalytic pocket. The JmjC domain of KDM4B folds into a β-jellyroll structure, characteristic of members in the KDM4 family (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>B).<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(25)</a> Superposition of KDM4A·<b>1</b>·H3K9me3 (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2OQ6">2OQ6</a>), KDM4B·<b>2</b>·H3K9me3, KDM4C·<b>1</b> (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2XML">2XML</a>), and KDM4D·AKG·H3K9me3 (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4HON">4HON</a>) shows limited conformational change in overall Cα atoms. The RMSD between KDM4A and KDM4B is 0.54 Å (residues 9–337 of KDM4B).</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm500249n/20150627/images/medium/jm-2014-00249n_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm500249n/20150627/images/large/jm-2014-00249n_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. KDM4B·<b>2</b>·H3K9me3 crystal structure. (A) Electron density map for Ni(II), <b>2</b>, and H3K9me3 peptide. The 2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> electron density maps are contoured at 1.0σ. (B) KDM4A (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2OQ6">2OQ6</a>), KDM4B (this study; PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4LXL">4LXL</a>), KDM4C (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2XML">2XML</a>), and KDM4D (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4HON">4HON</a>) are colored green, orange, blue, and gray, respectively. The stick models of compound <b>2</b> (pyridine 2,4-dicarboxylic acid) and the peptide are colored orange and cyan, respectively. Oxygen, nitrogen, and nickel atoms are colored red, blue, and magenta, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm500249n/20150627/images/large/jm-2014-00249n_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500249n&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In the active site, compound <b>2</b> is situated at a position nearly overlapped with AKG in which one of its carboxyl moieties contacts H189, E191, and K242, while the other H bonds with Y133 and K207, similar to those that contact with AKG (Y132, N198, and K206 in KDM4A). Ni(II), which occupies the site of Fe(II), makes contacts with three strictly conserved residues (H189, E191, and H277 in KDM4B). G171, Y176, T290, and N291 in KDM4B, which surround the methylated lysine, are also strictly conserved (Figure S1 of the <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>).</div><div class="NLM_p">The most prominent feature of the KDM4 family is its potent catalytic activity toward H3K9me3/me2.<a onclick="showRef(event, 'ref18 ref30'); return false;" href="javascript:void(0);" class="ref ref18 ref30">(18, 26)</a> Analysis of superimposed H3K9me3 liganded structures including KDM4A, KDM4B, and KDM4D [KDM4A·Fe(II)·<b>1</b>·H3K9me3 (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2OQ6">2OQ6</a>), KDM4B·Ni(II)·<b>2</b>·H3K9me3 (this study; PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4LXL">4LXL</a>), KDM4D·Ni(II)·AKG·H3K9me3 (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4HON">4HON</a>)] reveals a conserved region to accommodate R8 (−1) and the methylated K9 of H3. Notably, three conserved residues (KDM4A: D135, E169, Y175; KDM4B: D136, E170, Y176; KDM4D: D139, E173, Y179) from β7 and β8 make H contacts with the guanidinium group of R8, the peptide O and N atoms of K9, and the peptide N atom of T11 from H3. In the interior of this cleft, a lysyl side chain (KDM4A: K241; KDM4B: K242; KDM4D: K245) forms a strong bond to the peptide O atom of S10 (Figure S1 of the <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>), which together properly orient H3K9me3/me2 for similarly efficient catalysis in KDMs.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> Interestingly, the KDM4A/KDM4B/KDM4C subfamily, but not KDM4D, exhibits additional specificity to demethylate H3K36me3/me2.<a onclick="showRef(event, 'ref18 ref30'); return false;" href="javascript:void(0);" class="ref ref18 ref30">(18, 26)</a> Consistent with the structural analysis for KDM4D,<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> we observe two heterogeneous regions in KDM4B that account for the substrate specificity: (1) RKDM versus GEAR and (2) NIQ versus HKK (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>).</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm500249n/20150627/images/medium/jm-2014-00249n_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm500249n/20150627/images/large/jm-2014-00249n_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Differential ligand binding region between KDM4A/KDM4B/KDM4C and KDM4D. Superposition of KDM4A, KDM4B, KDM4C, and KDM4D reveals two heterogeneous regions, RKDM and NIQ. Surface representation of KDM4A, KDM4B, and KDM4D shows that the RKDM and NIQ regions and a crucial isoleucine (KDMA, I71; KDM4B, I72) make several contacts with the peptide at (−2) and (+3 and +4) sites in KDM4A and KDM4B. The corresponding GEAR and HKK sites deviate away from H3K9me3. The following PDB codes were used: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2OQ6">2OQ6</a> (KDM4A), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4LXL">4LXL</a> (KDM4B), and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4HON">4HON</a> (KDM4D).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm500249n/20150627/images/large/jm-2014-00249n_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500249n&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The RKDM site (residues 310–313 in KDM4B) from a long U-shaped loop resides near the (−1 and −2) site of the peptide-binding cleft. The aspartate side chain of RKDM (D311 in KDM4A) faces toward the peptide (−1 and −2 sites) and could make contacts with the peptide (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>). Additionally, the long and positively charged side chain of R from RKDM contributes to contact with the plus side of the peptide, as demonstrated in two liganded structures: KDM4A·AKG·H3K9 (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2Q8C">2Q8C</a>) [KDM4A/R309 (NH1)–H3/G12 (O): 3.8 Å]<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(27)</a> and KDM4A·<i>N</i>-oxalylglycine·H3K36 (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2P5B">2P5B</a>) [KDM4A/R309 (NH1)–H3/H39 (N): 3.8 Å].<a onclick="showRef(event, 'cit29a'); return false;" href="javascript:void(0);" class="ref cit29a">(25a)</a> A subtle difference is also noted at the other side of this U loop between KDM4A and KDM4B; there is a T308–D236 contact in KDM4B but not in KDM4A (the corresponding residues are S307 and E235). In contrast, the GEAR motif deviates away from the peptide-binding cleft; hence, it makes no contacts with the peptide.</div><div class="NLM_p">The other region is the NIQ site from the β4−β5 segment shared in KDM4A/KDM4B/KDM4C (residues 87–89 in KDM4B); KDM4D has HKK at the corresponding region. Q89 is noted to contact H3H39 and R40 (+3 and +4),<a onclick="showRef(event, 'cit29a'); return false;" href="javascript:void(0);" class="ref cit29a">(25a)</a> whereas KDM4D consists of HKK with positively charged side chains at the corresponding site (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>), which is likely to yield steric hindrance and electrostatic repulsion against H39 and R40 of H3K36me3.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> I71, which is near NIQ (KDMA, I71; KDM4B, I72), also plays a crucial role.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a></div><div class="NLM_p">We used the FDH-demethylase coupled continuous fluorescent demethylase method<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(28)</a> to assess the enzymatic activity of recombinant KDM4A and KDM4B expressed in <i>E. coli</i>. Using an H3K9me3 peptide (residues 3–17) as the substrate, KDM4A and KDM4B exhibited comparable catalytic activity (Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>), consistent with Hillringhaus et al.<a onclick="showRef(event, 'cit29b'); return false;" href="javascript:void(0);" class="ref cit29b">(25b)</a> We were able to measure the kinetic parameters with an H3K36me3 peptide (H3<sub>31–41</sub>K36me3) and obtained an analogous <i>k</i><sub>cat</sub> value and a higher <i>K</i><sub>m</sub> value compared with those for the H3K9me3 peptide, suggesting that KDM4A/KDM4B have a lower binding affinity toward H3K36me3 than H3K9me3.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Kinetic Parameters for KDM4A and KDM4B Using H3<sub>3–17</sub>K9me3 or H3<sub>31–41</sub>K36me3 as the Substrate</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="3" align="center" char=".">H3<sub>3–17</sub>K9me3</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">KDM4</th><th class="colsep0 rowsep0" align="center" char="."><i>k</i><sub>cat</sub> (s<sup>–1</sup>)</th><th class="colsep0 rowsep0" align="center" char="."><i>K</i><sub>m</sub> (μM)</th><th class="colsep0 rowsep0" align="center" char="."><i>k</i><sub>cat</sub>/<i>K</i><sub>m</sub> (s<sup>–1</sup> μM<sup>–1</sup>)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">KDM4A</td><td class="colsep0 rowsep0" align="char" char=".">0.017 ± 0.001</td><td class="colsep0 rowsep0" align="char" char=".">92.5 ± 5.9</td><td class="colsep0 rowsep0" align="char" char=".">1.8 × 10<sup>–4</sup></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">KDM4B</td><td class="colsep0 rowsep0" align="char" char=".">0.014 ± 0.001</td><td class="colsep0 rowsep0" align="char" char=".">88.3 ± 8.6</td><td class="colsep0 rowsep0" align="char" char=".">1.6 × 10<sup>–4</sup></td></tr></tbody></table><table class="table" border="0"><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="3" align="center" char=".">H3<sub>31–41</sub>K36me3</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">KDM4A</td><td class="colsep0 rowsep0" align="char" char=".">0.015 ± 0.001</td><td class="colsep0 rowsep0" align="char" char=".">169.9 ± 19.9</td><td class="colsep0 rowsep0" align="char" char=".">8.8 × 10<sup>–5</sup></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">KDM4B</td><td class="colsep0 rowsep0" align="char" char=".">0.013 ± 0.001</td><td class="colsep0 rowsep0" align="char" char=".">138.5 ± 14.1</td><td class="colsep0 rowsep0" align="char" char=".">9.4 × 10<sup>–5</sup></td></tr></tbody></table></div></div><div class="NLM_p last">We further utilized calf thymus histones as the substrate and probed for H3K9, H3K27, and H3K36me3/me2/me1 in the presence of recombinant KDM4A or KDM4B using western blot analysis. As shown in <a class="ref internalNav" href="#notes-1" aria-label="Figure S2">Figure S2</a>, the signal of H3K9me3/me2 was significantly reduced and that of H3K9me1 was increased in the presence KDM4A (upper panel) or KDM4B (lower panel) in a time-dependent manner compared with that in the controls, indicating that an active KDM4A/KDM4B forms to remove the methyl group from H3K9me3/me2. For the H3K36 signal, a longer time was needed to remove the signal for H3K36me3/me2. No difference was found for H3K27me3 or H3K27me1. These results collectively suggest that KDM4A and KDM4B demethylate H3K9me3/me2 more efficiently than H3K36me3/me2 and that there was no activity toward H3K27me3/me2/me1, confirming our results in Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>.</div></div><div id="sec3_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Virtual Screening To Identify <b>4</b> as a Novel Active-Site Inhibitor toward KDM4A and KDM4B</h3><div class="NLM_p">We utilized GEMDOCK<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> to screen for putative hits against the NCI database. The known inhibitor <b>2</b> was used as a positive control, which showed significant inhibition [21% (KDM4A) and 24% (KDM4B) of residual activity].<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(29)</a> We selected 10 compounds from the top-ranked 3000 compounds based on their ranking, availability, domain knowledge, and docked-pose analysis (<a class="ref internalNav" href="#notes-1" aria-label="Table S1">Table S1</a>). Of these, compound <b>4</b>, which was docked into the active site (<a class="ref internalNav" href="#notes-1" aria-label="Figure S3">Figure S3</a>), exhibited the highest inhibitory effect toward both KDM4A (28%) and KDM4B (35%) (<a class="ref internalNav" href="#notes-1" aria-label="Table S1">Table S1</a>). To confirm the FDH-demethylase coupled results, we utilized histones as the substrate and probed for H3K9me3 and H3K36me3 in the absence or presence of recombinant KDM4A or KDM4B using western blot analysis. Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>A shows that compounds <b>2</b> and <b>4</b> indeed blocked the demethylation activity. Further kinetic inhibition characterization of compound <b>4</b> demonstrated a competitive inhibitory mode against H3<sub>3–17</sub>K9me3 for KDM4A [IC<sub>50</sub> = 6.4 μM; <i>K</i><sub>i</sub> (H3K9me3) = 5.5 ± 1.6 μM; Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>B). KDM4B also showed analogous inhibition kinetics [IC<sub>50</sub> = 9.3 μM; <i>K</i><sub>i</sub> (H3K9me3) = 3.0 ± 1.1 μM).</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm500249n/20150627/images/medium/jm-2014-00249n_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm500249n/20150627/images/large/jm-2014-00249n_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Compound <b>4</b> inhibits demethylase activity of KDM4A and KDM4B. (A) Demethylation of calf thymus H3 by bacteria-expressed KDM4A (upper panel) and KDM4B (lower panel) was determined in the presence of <b>4</b> by western blot analysis. The reaction mixture containing 10 μM enzyme, 100 μM inhibitor or blank buffer, and 5 μM of H3 in 50 mM HEPES, pH 7.5, 1 mM AKG, 2 mM ascorbate, and 50 μM Fe(II) was incubated at 37 °C for 30 min, followed by western blot analysis. H3 lysine modifications were probed with H3K9me3 and H3K36me3 antisera, respectively. (B) Inhibition kinetics of KDM4A/KDM4B demethylation activity by <b>4</b>. The inset in each panel shows the double-reciprocal form, where the 1/relative activity is plotted versus 1/[H3K9me3] at various fixed concentrations of the inhibitor.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm500249n/20150627/images/large/jm-2014-00249n_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500249n&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">We next tested whether compound <b>4</b> could inhibit purified, recombinant KDM4D and KDM4E. Using the in vitro FDH-demethylase coupled assay, compound <b>4</b> exhibited a much weaker inhibitory effect toward KDM4D and KDM4E than did compound <b>2</b> (<a class="ref internalNav" href="#notes-1" aria-label="Figure S4A">Figure S4A</a>). We further characterized the methylated status of H3 in LNCaP cells treated with compound <b>4</b>. As shown in <a class="ref internalNav" href="#notes-1" aria-label="Figure S4B">Figure S4B</a>, only the level of H3K9me3 was significantly increased in compound <b>4</b>-treated LNCaP cells as opposed to essentially comparable signals of H3K4me2, H3K27me3, H3K27me2, H3K36me3, H3K36me2, and H3K79me2 between control and treated cells. Together, these provide strong evidence that compound <b>4</b> is a potent, selective inhibitor against H3K9me3 demethylation by KDM4A/KDM4B in LNCaP cells.</div></div><div id="sec3_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Genetic and Pharmacological Inhibition of KDM4A and KDM4B Induces Apoptosis</h3><div class="NLM_p">Several studies have reported that KDM4 family members are overexpressed in various cancers.<a onclick="showRef(event, 'cit3b'); return false;" href="javascript:void(0);" class="ref cit3b">(3b)</a> To further support the clinical relevance of KDM4A and KDM4B in prostate cancer, we took advantage of the comprehensive database collection of OncomineTM (Compendia Bioscience, Ann Arbor, MI, USA) (<a href="http://www.oncomine.org/" class="extLink">http://www.oncomine.org/</a>) to examine their expression profiles between normal prostate glands and tumor tissues. Among the 14 data sets available, a statistically significant (<i>p</i> < 0.05) elevation of KDM4A was seen in 6 data sets and KDM4B was elevated in 8 data sets in prostate cancer (PCa) compared to their levels in normal/benign samples (Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>A). The rest of the data sets also exhibited higher median values of KDM4A/KDM4B expression in PCa sites, despite not being statistically significant (<a class="ref internalNav" href="#notes-1" aria-label="Figure S5">Figure S5</a>). Strikingly, the level of KDM4A and KDM4B expression is positively correlated with prostate cancer progression (normal, primary PCa, and metastatic PCa).<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(30)</a></div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm500249n/20150627/images/medium/jm-2014-00249n_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm500249n/20150627/images/large/jm-2014-00249n_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. KDM4A and KDM4B are highly expressed in prostate cancer cells. (A) Expression values of KDM4A and KDM4B in normal prostate glands and tumor tissues from the selected studies were obtained from the Oncomine (Compendia Bioscience, Ann Arbor, MI, USA) database (<a href="http://www.oncomine.org/" class="extLink">http://www.oncomine.org/</a>). The box and whisker plots show a box that encompasses the 25th–75th percentiles, the median as a line within the box, and the 10th and 90th percentiles as error bars. <i>p</i> values were determined by one-tailed Student’s <i>t</i> test and were calculated on the basis of the comparison of normal vs cancer; normal vs primary sites; or normal vs metastasis. (B) qRT-PCR analysis of KDM4A and KDM4B expression in normal prostate primary cells (PrEC), normal prostate epithelial cell lines (RWPE-1 and PNT2), and prostate cancer cell lines (LNCaP, C4-2, C4-2B, CWR22rv1, CWR-R1, VCaP, DU145, and PC3). Asterisks indicate significant overexpression compared to that in normal cells.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm500249n/20150627/images/large/jm-2014-00249n_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500249n&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We also examined the expression of KDM4A and KDM4B in several laboratory-cultured prostate cancer cell models: normal prostate epithelial cells (PrEC, RWPE-1, and PNT2) and a number of prostate cancer cell lines (LNCaP, C4-2, C4-2B, CWR22rv1, CWR-R1, VCaP, DU145, and PC3). Essentially, all prostate cancer cells exhibited higher expression of KDM4A compared with its level in normal prostate epithelial cell lines, which was statistically significant for LNCaP, C4-2, C4-2B, CWR22rv1, and VCaP cells (Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>B). Similarly, with the exceptions of DU145 and PC3, KDM4B is overexpressed in all other malignant cell lines tested.</div><div class="NLM_p">To assess whether KDM4A or KDM4B was crucial for prostate cancer cell growth, LNCaP cells were treated with sh-KDM4A or sh-KDM4B to knockdown the expression of KDM4A or KDM4B, respectively (Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>A). A significantly reduced level of cell growth was found in KDM4A and KDM4B knockdown cells (Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>A). Correspondingly, there was an increased level of H3K9me3 signal in either of KDM4A or KDM4B knockdown cells (Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>B). Flow cytometry analysis showed that there was a significantly increased population of apoptotic cells in sh-KDM4A (40.1%) or sh-KDM4B (64.2%) cells compared to that in control cells (23.6%) (Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>C). These results suggest that KDM4A and KDM4B are critical to the viability of the cancer cells and thus are potentially useful targets for intervention.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm500249n/20150627/images/medium/jm-2014-00249n_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm500249n/20150627/images/large/jm-2014-00249n_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (A) KDM4A and KDM4B are crucial for the growth of LNCaP cells. LNCaP cells were infected with lentivirus encoding control shRNA (control), sh-KDM4A, or sh-KDM4B, as indicated (left panel). qRT-PCR analysis was performed to evaluate the expression of KDM4s (right panel). (B) H3K9me3 levels in KDM4A and KDM4B knockdown cells. LNCaP cells were infected by lentivius carrying control vector, sh-KDM4A, or sh-KDM4B for 3 days. The KDM4A, KDM4B, and H3K9me3 signals were detected in cell lysates by western blot analysis as indicated. (C) Flow cytometry analysis of the DNA content in sh-KDM4A and sh-KDM4B LNCaP cells.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm500249n/20150627/images/large/jm-2014-00249n_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500249n&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The knockdown data for KDM4A and KDM4B prompted us to test whether compound <b>4</b>, which inhibits both KDM4A and KDM4B, would similarly reduce the viability of LNCaP cells. Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>A shows that this compound had hardly any effect on PNT2 cells over 6 days (5–20 μM), whereas there was slightly reduced growth for the other normal cell line, RWPE1, upon 20 μM treatment. By contrast, the growth of LNCaP cells was significantly affected by inhibitor <b>4</b>. As shown in Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>B, inhibitor <b>4</b> effectively kills LNCaP cells after 3 day culture (cytotoxicity IC<sub>50</sub> = 16.5 μM). To validate the effect of inhibitor <b>4</b>, we examined the cellular level of H3K9me3 in the treated cells. Inhibitor <b>4</b>-treated cells had a notable increase in the level of H3K9me3 in a dose-dependent manner: 5 μM inhibitor <b>4</b> treatment almost completely blocked the demethylating activity toward H3K9me3. Flow cytometry analysis showed that there were nearly 4-fold more apoptotic LNCaP cells produced upon treatment with inhibitor <b>4</b> compared to that for mock-treated cells (11.9 vs 39.5%) (Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>C). These results collectively suggest that inhibition of KDM4A/KDM4B by shRNA or by inhibitor <b>4</b> specifically inhibits the demethylating activity of H3K9me3 and strongly blocks cell growth. As a comparison, we utilized the dimethyl ester form of compound <b>2</b>, the most potent inhibitor against KDM4E (in vitro assay), which allows penetration into the cells.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> Consistent with the other results, a high CC<sub>50</sub> value was found in LNCaP cells treated with <b>2</b> (588.7 μM).</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm500249n/20150627/images/medium/jm-2014-00249n_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm500249n/20150627/images/large/jm-2014-00249n_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Compound <b>4</b> exhibits anticancer effects on LNCaP cells. (A) Treatment of LNCaP or normal prostate (PNT2 and RWPE1) cells with different concentrations of inhibitor <b>4</b> over 5 days shows inhibitor <b>4</b>’s selective anticancer properties toward LNCaP cells. (B) Inhibition of viability by inhibitor <b>4</b> in normal prostate and LNCaP cells. LNCaP and normal prostate (PNT2 and RWPE1) cells were treated with inhibitor <b>4</b> for 3 days, and viability was measured by viable cell count. Standard deviation is derived from biological triplicates. The H3K9me3 signal (lower panel) was detected in cell lysates treated with different concentrations (1–100 μM) of inhibitor <b>4</b> for 24 h, followed by estern blot analysis using anti-H3K9me3 antibody. The different concentrations (1–100 μM) of inhibitor <b>4</b> are indicated below the blot. The level of H3K9me3 was detected and quantified by AlphaView SA (Cell Biosciences Inc.). The level of H3K9me3 is shown by bar graph. (C) Flow cytometry analysis of the DNA content in LNCaP cells treated with DMSO (mock) or 50 μM inhibitor <b>4</b> for 3 days.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm500249n/20150627/images/large/jm-2014-00249n_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500249n&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec3_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Inhibition of KDM4 by <b>4</b> Negatively Regulates AR Responsive Genes</h3><div class="NLM_p">To understand the mechanisms associated with growth inhibition and apoptosis induction by inhibitor <b>4</b>, we characterized the differential gene expression profiles in LNCaP cells treated with or without <b>4</b> using microarray analysis (≥2-fold alterations). As shown in Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a>A, functional annotations indicated a number of differentially expressed genes related to cell division and DNA processes. Most intriguingly, a significant portion (27% = 178/656) of the altered genes were found to be androgen-responsive genes (Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a>B). In addition to the alteration of androgen-responsive genes, inhibitor <b>4</b> induced upregulation of tumor suppressors RB1 and CDH1 as well as downregulation of oncogenes IGF1R, FGFR3, CCNE2, AURKA, and AURKB (Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a>C), which may contribute to a loss of proliferation and survival advantages for the tumor cell. The mRNA expression of AR-signature genes was shown in Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a>.</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm500249n/20150627/images/medium/jm-2014-00249n_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm500249n/20150627/images/large/jm-2014-00249n_0008.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Microarray analysis of LNCaP cells treated with inhibitor <b>4</b> (50 μM) compared to that of mock-treated cells. (A) DAVID functional annotation of the genes that showed 2-fold alterations in expression. GO terms associated with the altered genes that show statistically strong enrichment with low <i>p</i> values are listed. The numbers next to each bar in the graph indicate the gene count for each pathway. Percent of hits indicates the percentage of genes that are altered in each GO classification. (B) Expression of tumor suppressors and oncogenes that are up- and downregulated in the inhibitor-treated cells, respectively. (C) Venn diagram of the overlap of the inhibitor-altered genes with androgen-responsive genes.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm500249n/20150627/images/large/jm-2014-00249n_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500249n&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm500249n/20150627/images/medium/jm-2014-00249n_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm500249n/20150627/images/large/jm-2014-00249n_0009.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. AR-signature genes are differentially expressed in inhibitor <b>4</b>-treated LNCaP cells.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm500249n/20150627/images/large/jm-2014-00249n_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500249n&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">We further compared the expression profiles of selected AR-responsive genes (CDC6, CDC25A, CLDN8, FZD3, KLK3, MCM5, NKX3.1, RRM2, TMEPA1, and TMPRSS2) in two sets of experiments using qRT-PCR analysis: (i) control, sh-KDM4A, and sh-KDM4B cells and (ii) control and <b>4</b>-treated LNCaP cells (<a class="ref internalNav" href="#notes-1" aria-label="Figure S6">Figure S6</a>). Overall, there were similarly downregulated expression profiles for these genes in cells treated with inhibitor <b>4</b>, sh-KDM4A, and sh-KDM4B, indicating that the targets of inhibitor <b>4</b> were indeed KDM4A/KDM4B. This is also consistent with previous results showing that both KDM4A and KDM4B (as well as KDM4C) are coactivators of AR.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> Thus, inhibitor <b>4</b> specifically inhibits the expression of genes involved in DNA-dependent processes, cell proliferation, and AR-dependent signaling in prostate cancer cells. Given the importance of AR in prostate carcinogenesis, compounds that inhibit KDM4A and KDM4B may be beneficially used to overcome castration-resistant prostate cancer.</div></div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i21">Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_12607" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_12607" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In this study, we report a peptide-liganded KDM4B structure that demonstrates a well-conserved core active site with that of KDM4A and KDM4C, supporting the efficient catalytic power of this subfamily to demethylate both H3K9me3/me2 and H3K36me3/me2. This structure also presents two heterogeneous regions in the peptide-binding cleft compared with that of KDM4D, which accounts for the sole H3K9me3, but not H3K36me3, specificity of KDM4D: (i) RKDM vs GEAR and (ii) NIQ vs HKK.</div><div class="NLM_p">Importantly, we identified an inhibitor, <b>4</b>, that selectively blocks in vitro demethylation activity (H3K9me3 and H3K36me3) by KDM4A/KDM4B based on a virtual structure-guided screening method.<a onclick="showRef(event, 'cit22a ref35'); return false;" href="javascript:void(0);" class="ref cit22a ref35">(22a, 31)</a> Compound <b>4</b> also significantly inhibits histone demethylase activity on H3K9me3 and induces apoptosis in cultured LNCaP cells. Knocking down KDM4B expression by shRNA treatment also reduces the viability of LNCaP cells. This inhibitory effect appears to be largely due to its transcriptional reprogramming of genes that control cell proliferation, particularly those involved in the cell cycle and mitosis. Also affected are genes involved in DNA repair, DNA recombination, and chromosomal organization, suggesting that KDM4 plays a role in regulating genome stability. Our results are echoed by a recent report that KDM4A overexpression leads to genomic amplification.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(32)</a> Perhaps the most striking finding of the microarray analysis is that a significant portion (close to ∼30%) of <b>4</b>-altered genes are androgen-responsive, consistent with the roles of KDM4A–C as critical coactivators of AR and suggesting its potential to be used to treat castration-resistant prostate cancer. Furthermore, compound <b>4</b> exhibits a competitive inhibitory mode against the H3K9me3 peptide. It specifically blocked KDM4A/KDM4B but only weakly suppressed KDM4D and KDM4E. Importantly, cell-based results revealed that compound <b>4</b>-treated LNCaP cells significantly lost demethylation capability toward H3K9me3 but not toward other methyl marks (H3K4me2, H3K27me3, H3K27me2, H3K36me3, H3K36me2, and H3K79me2). Together, these provide strong evidence that compound <b>4</b> is a potent, selective inhibitor against H3K9me3 demethylation by KDM4A/KDM4B in LNCaP cells.</div><div class="NLM_p">Much attention has been focused on developing KDM4-targeted inhibitors because of their importance in biological processes including carcinogenesis.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(33)</a> Currently, the developed inhibitors include analogues of <b>1</b>–<b>3</b> that all occupy the AKG site (PDB codes: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2P58">2P58</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2VD7">2VD7</a>, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3NJY">3NJY</a>). Despite the high potencies demonstrated by these inhibitors in vitro, their clinical outcomes are not yet clear. A recent cell-based screening study reported a potent pan-selective inhibitor, JIB-04, a nonsymmetrical pyridine hydrazone that consists of E and Z isomers. It is noted that its E isomer (not Z) shows pan inhibition toward JmjC-containing KDMs in vitro and anticancer effects in cells and in a breast cancer model.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(34)</a> However, this molecule also blocks prolyl hydroxylase, albeit weakly, and thus may have more global effects on cells than intended. Inhibitor <b>4</b>, although not as potent as JIB-04, offers a new structural framework that lies on the selective peptide-binding region apart from the AKG site. Further structure–activity relationship (SAR) studies on JIB-04 and <b>4</b> are thus likely to contribute to the development of new selective agents against KDM-dependent functions.</div><div class="NLM_p last">In summary, inhibition of KDM4A/KDM4B by pharmacological (inhibitor <b>4</b>) or genetic (sh-KDM4A and sh-KDM4B) means selectively downmodulates cell growth-promoting genes, including androgen-responsive genes, resulting in the effective killing of prostate cancer cells. To our knowledge, this is the first KDM4A/KDM4B inhibitor that shows anticancer properties in prostate cancer cells. This inhibitor thus provides an attractive scaffold for further SAR development of new KDM4 modulators to control its coactivation for transcriptional programming of prostate cancer cells.</div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i22"><a href="/doi/suppl/10.1021/jm500249n">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_24223" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_24223" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">Binding pocket analysis in KDM4s (Figure S1); western blot of KDM4A and KDM4B that demethylate H3K9me3/me2 and H3K36me3/me2 but not H3K27me3 (Figure S2); superposition of the KDM4B·<b>2</b>·H3K9me3 structure and the KDM4B·<b>4</b> docked pose (Figure S3); selective inhibitory effect of compound <b>4</b> (Figure S4); KDM4A and KDM4B expression data in PCa sites (Figure S5); downregulation of AR target genes in KDM4A/KDM4B-knockdown and compound <b>4</b>-treated cells (Figure S6); inhibition effects of selected hits on KDM4A and KDM4B (Table S1); and crystallographic data and refinement statistics for KDM4B·<b>2</b>·H3K9me3 (Table S2). This material is available free of charge via the Internet at <a href="http://pubs.acs.org" class="extLink">http://pubs.acs.org</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm500249n/suppl_file/jm500249n_si_001.pdf">jm500249n_si_001.pdf (1.03 MB)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The atomic coordinates of KDM4B have been deposited as Protein Data Bank entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4LXL">4LXL</a>.</p><div class="testing" data-doi="10.1021/jm500249n" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_53167" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_53167" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jing-Moon Yang</span> - <span class="hlFld-Affiliation affiliation">Institute
of Bioinformatics and Systems Biology, National
Chiao Tung University, Hsinchu, 30050, Taiwan</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#f69b999998b6909795839a828fd898958283d8939283d88281"><span class="__cf_email__" data-cfemail="deb3b1b1b09eb8bfbdabb2aaa7f0b0bdaaabf0bbbaabf0aaa9">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hsing-Jien Kung</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Molecular Medicine, University of California Davis School of Medicine, University of
California Davis Cancer Center, Sacramento, California 95817, United States</span>; 
    <span class="hlFld-Affiliation affiliation">National
Health Research Institutes, Miaoli, 35053, Taiwan</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#1b73706e757c5b757369723574697c356f6c"><span class="__cf_email__" data-cfemail="abc3c0dec5ccebc5c3d9c285c4d9cc85dfdc">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Wen-Ching Wang</span> - <span class="hlFld-Affiliation affiliation">†Institute of Molecular and Cellular Biology
and Department of Life
Sciences and ‡Biomedical Science and Engineering Center, National Tsing-Hua University, Hsinchu, 30013, Taiwan</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Molecular Medicine, University of California Davis School of Medicine, University of
California Davis Cancer Center, Sacramento, California 95817, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Institute
of Bioinformatics and Systems Biology, National
Chiao Tung University, Hsinchu, 30050, Taiwan</span>; 
    <span class="hlFld-Affiliation affiliation">National
Health Research Institutes, Miaoli, 35053, Taiwan</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#04736773656a6344686d62612a6a706c712a6160712a70732473616a676c6d6a632a7367446369656d682a676b69"><span class="__cf_email__" data-cfemail="a1d6c2d6c0cfc6e1cdc8c7c48fcfd5c9d48fc4c5d48fd5d6">[email protected]</span> <span class="__cf_email__" data-cfemail="f7809299949f9e9990d98094b7909a969e9bd994989a">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chia-Han Chu</span> - <span class="hlFld-Affiliation affiliation">†Institute of Molecular and Cellular Biology
and Department of Life
Sciences and ‡Biomedical Science and Engineering Center, National Tsing-Hua University, Hsinchu, 30013, Taiwan</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Molecular Medicine, University of California Davis School of Medicine, University of
California Davis Cancer Center, Sacramento, California 95817, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Institute
of Bioinformatics and Systems Biology, National
Chiao Tung University, Hsinchu, 30050, Taiwan</span>; 
    <span class="hlFld-Affiliation affiliation">National
Health Research Institutes, Miaoli, 35053, Taiwan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ling-Yu Wang</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Molecular Medicine, University of California Davis School of Medicine, University of
California Davis Cancer Center, Sacramento, California 95817, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kai-Cheng Hsu</span> - <span class="hlFld-Affiliation affiliation">Institute
of Bioinformatics and Systems Biology, National
Chiao Tung University, Hsinchu, 30050, Taiwan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chung-Chin Chen</span> - <span class="hlFld-Affiliation affiliation">†Institute of Molecular and Cellular Biology
and Department of Life
Sciences and ‡Biomedical Science and Engineering Center, National Tsing-Hua University, Hsinchu, 30013, Taiwan</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Molecular Medicine, University of California Davis School of Medicine, University of
California Davis Cancer Center, Sacramento, California 95817, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Institute
of Bioinformatics and Systems Biology, National
Chiao Tung University, Hsinchu, 30050, Taiwan</span>; 
    <span class="hlFld-Affiliation affiliation">National
Health Research Institutes, Miaoli, 35053, Taiwan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hsing-Hung Cheng</span> - <span class="hlFld-Affiliation affiliation">†Institute of Molecular and Cellular Biology
and Department of Life
Sciences and ‡Biomedical Science and Engineering Center, National Tsing-Hua University, Hsinchu, 30013, Taiwan</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Molecular Medicine, University of California Davis School of Medicine, University of
California Davis Cancer Center, Sacramento, California 95817, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Institute
of Bioinformatics and Systems Biology, National
Chiao Tung University, Hsinchu, 30050, Taiwan</span>; 
    <span class="hlFld-Affiliation affiliation">National
Health Research Institutes, Miaoli, 35053, Taiwan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Szu-Min Wang</span> - <span class="hlFld-Affiliation affiliation">†Institute of Molecular and Cellular Biology
and Department of Life
Sciences and ‡Biomedical Science and Engineering Center, National Tsing-Hua University, Hsinchu, 30013, Taiwan</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Molecular Medicine, University of California Davis School of Medicine, University of
California Davis Cancer Center, Sacramento, California 95817, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Institute
of Bioinformatics and Systems Biology, National
Chiao Tung University, Hsinchu, 30050, Taiwan</span>; 
    <span class="hlFld-Affiliation affiliation">National
Health Research Institutes, Miaoli, 35053, Taiwan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chien-Ming Wu</span> - <span class="hlFld-Affiliation affiliation">†Institute of Molecular and Cellular Biology
and Department of Life
Sciences and ‡Biomedical Science and Engineering Center, National Tsing-Hua University, Hsinchu, 30013, Taiwan</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Molecular Medicine, University of California Davis School of Medicine, University of
California Davis Cancer Center, Sacramento, California 95817, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Institute
of Bioinformatics and Systems Biology, National
Chiao Tung University, Hsinchu, 30050, Taiwan</span>; 
    <span class="hlFld-Affiliation affiliation">National
Health Research Institutes, Miaoli, 35053, Taiwan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tsan-Jan Chen</span> - <span class="hlFld-Affiliation affiliation">†Institute of Molecular and Cellular Biology
and Department of Life
Sciences and ‡Biomedical Science and Engineering Center, National Tsing-Hua University, Hsinchu, 30013, Taiwan</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Molecular Medicine, University of California Davis School of Medicine, University of
California Davis Cancer Center, Sacramento, California 95817, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Institute
of Bioinformatics and Systems Biology, National
Chiao Tung University, Hsinchu, 30050, Taiwan</span>; 
    <span class="hlFld-Affiliation affiliation">National
Health Research Institutes, Miaoli, 35053, Taiwan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ling-Ting Li</span> - <span class="hlFld-Affiliation affiliation">Institute
of Bioinformatics and Systems Biology, National
Chiao Tung University, Hsinchu, 30050, Taiwan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ruiwu Liu</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Molecular Medicine, University of California Davis School of Medicine, University of
California Davis Cancer Center, Sacramento, California 95817, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chiu-Lien Hung</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Molecular Medicine, University of California Davis School of Medicine, University of
California Davis Cancer Center, Sacramento, California 95817, United States</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>C.-H.C., L.-Y.W., and K.-C.H. are co-first authors.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d570e1451-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i25">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_22177" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_22177" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was supported by NSC, Taiwan (NSC102-2321-B-007–003 and NSC102-2325-B-007–001), and by “Aim for the Top University Project” of NTHU and MOE, Taiwan, ROC. We acknowledge the support of grants from the NIH, USA, as well as NHRI (02A1MGPP20-014) and MOHW (DOH102-TD-M-111-100001), Taiwan, to H.-J.K. We thank the Macromolecular X-ray Crystallographic Center at the National Tsing Hua University Instrument Center at Hsinchu, Taiwan, for access to the BL13B1 beamline at NSRRC, Taiwan, and the SP12B2 Taiwan beamline at SPring-8, Japan, for data collection. We also thank the staff at NSRRC for their excellent support.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i26" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i26"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i27" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i27"> Abbreviations Used</h2><tr><td class="NLM_term">AKG</td><td class="NLM_def"><p class="first last">α-ketoglutarate</p></td></tr><tr><td class="NLM_term">JmjC</td><td class="NLM_def"><p class="first last">Jumonji C</p></td></tr><tr><td class="NLM_term">AR</td><td class="NLM_def"><p class="first last">androgen receptor of KDM4s</p></td></tr><tr><td class="NLM_term">ER</td><td class="NLM_def"><p class="first last">estrogen receptor of KDM4A and B</p></td></tr><tr><td class="NLM_term">OGA</td><td class="NLM_def"><p class="first last"><i>N</i>-oxalylglycine</p></td></tr><tr><td class="NLM_term">PD2</td><td class="NLM_def"><p class="first last">2,4-dicarboxylic acids</p></td></tr><tr><td class="NLM_term">8HQ</td><td class="NLM_def"><p class="first last">8-hydroxyquinolines</p></td></tr><tr><td class="NLM_term">DE3</td><td class="NLM_def"><p class="first last"><i>Escherichia coli</i> BL21</p></td></tr><tr><td class="NLM_term">IPTG</td><td class="NLM_def"><p class="first last">isopropyl-β-<span class="smallcaps smallerCapital">d</span>-thiogalactopyranoside</p></td></tr><tr><td class="NLM_term">DMSO</td><td class="NLM_def"><p class="first last">dimethyl sulfoxide</p></td></tr><tr><td class="NLM_term">ascorbate</td><td class="NLM_def"><p class="first last">ascorbic acid</p></td></tr><tr><td class="NLM_term">PEG</td><td class="NLM_def"><p class="first last">poly(ethylene glycol)</p></td></tr><tr><td class="NLM_term">RMSD</td><td class="NLM_def"><p class="first last">root-mean-squared deviation</p></td></tr><tr><td class="NLM_term">FDH</td><td class="NLM_def"><p class="first last">formaldehyde dehydrogenase</p></td></tr><tr><td class="NLM_term">PCa</td><td class="NLM_def"><p class="first last">prostate cancer</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i28">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_37524" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_37524" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 34 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Shi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lan, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mulligan, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whetstine, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cole, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casero, R. A.</span><span> </span><span class="NLM_article-title">Histone demethylation mediated by the nuclear amine oxidase homolog LSD1</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">119</span><span class="NLM_x">, </span> <span class="NLM_fpage">941</span><span class="NLM_x">–</span> <span class="NLM_lpage">53</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1a&amp;dbid=16&amp;doi=10.1021%2Fjm500249n&amp;key=10.1016%2Fj.cell.2004.12.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1a&amp;dbid=8&amp;doi=10.1021%2Fjm500249n&amp;key=15620353" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1a&amp;dbid=32&amp;doi=10.1021%2Fjm500249n&amp;key=1%3ACAS%3A528%3ADC%252BD2MXltlSitg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2004&pages=941-53&author=Y.+Shiauthor=F.+Lanauthor=C.+Matsonauthor=P.+Mulliganauthor=J.+R.+Whetstineauthor=P.+A.+Coleauthor=R.+A.+Casero&title=Histone+demethylation+mediated+by+the+nuclear+amine+oxidase+homolog+LSD1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1aR"><div class="casContent"><span class="casTitleNuber">1a</span><div class="casTitle"><span class="NLM_cas:atitle">Histone demethylation mediated by the nuclear amine oxidase homolog LSD1</span></div><div class="casAuthors">Shi, Yujiang; Lan, Fei; Matson, Caitlin; Mulligan, Peter; Whetstine, Johnathan R.; Cole, Philip A.; Casero, Robert A.; Shi, Yang</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">941-953</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Posttranslational modifications of histone N-terminal tails impact chromatin structure and gene transcription.  While the extent of histone acetylation is detd. by both acetyltransferases and deacetylases, it has been unclear whether histone methylation is also regulated by enzymes with opposing activities.  Here, we provide evidence that LSD1 (KIAA0601), a nuclear homolog of amine oxidases, functions as a histone demethylase and transcriptional corepressor.  LSD1 specifically demethylates histone H3 lysine 4, which is linked to active transcription.  Lysine demethylation occurs via an oxidn. reaction that generates formaldehyde.  Importantly, RNAi inhibition of LSD1 causes an increase in H3 lysine 4 methylation and concomitant derepression of target genes, suggesting that LSD1 represses transcription via histone demethylation.  The results thus identify a histone demethylase conserved from S. pombe to human and reveal dynamic regulation of histone methylation by both histone methylases and demethylases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXHr-8kwGdXrVg90H21EOLACvtfcHk0lgk7V0vVbulUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXltlSitg%253D%253D&md5=3474569204255e90a832667ba654748b</span></div><a href="/servlet/linkout?suffix=cit1a&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2004.12.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2004.12.012%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DY.%26aulast%3DLan%26aufirst%3DF.%26aulast%3DMatson%26aufirst%3DC.%26aulast%3DMulligan%26aufirst%3DP.%26aulast%3DWhetstine%26aufirst%3DJ.%2BR.%26aulast%3DCole%26aufirst%3DP.%2BA.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26atitle%3DHistone%2520demethylation%2520mediated%2520by%2520the%2520nuclear%2520amine%2520oxidase%2520homolog%2520LSD1%26jtitle%3DCell%26date%3D2004%26volume%3D119%26spage%3D941%26epage%3D53" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit1b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Kouzarides, T.</span><span> </span><span class="NLM_article-title">Chromatin modifications and their function</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">128</span><span class="NLM_x">, </span> <span class="NLM_fpage">693</span><span class="NLM_x">–</span> <span class="NLM_lpage">705</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1b&amp;dbid=16&amp;doi=10.1021%2Fjm500249n&amp;key=10.1016%2Fj.cell.2007.02.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1b&amp;dbid=8&amp;doi=10.1021%2Fjm500249n&amp;key=17320507" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1b&amp;dbid=32&amp;doi=10.1021%2Fjm500249n&amp;key=1%3ACAS%3A528%3ADC%252BD2sXis12ju7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2007&pages=693-705&author=T.+Kouzarides&title=Chromatin+modifications+and+their+function"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1bR"><div class="casContent"><span class="casTitleNuber">1b</span><div class="casTitle"><span class="NLM_cas:atitle">Chromatin modifications and their function</span></div><div class="casAuthors">Kouzarides, Tony</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">693-705</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The surface of nucleosomes is studded with a multiplicity of modifications.  At least eight different classes have been characterized to date and many different sites have been identified for each class.  Operationally, modifications function either by disrupting chromatin contacts or by affecting the recruitment of nonhistone proteins to chromatin.  Their presence on histones can dictate the higher-order chromatin structure in which DNA is packaged and can orchestrate the ordered recruitment of enzyme complexes to manipulate DNA.  In this way, histone modifications have the potential to influence many fundamental biol. processes, some of which may be epigenetically inherited.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAH9AX2dGicbVg90H21EOLACvtfcHk0lgk7V0vVbulUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXis12ju7Y%253D&md5=c5995fe68fd9473f4268d59c01d3b9de</span></div><a href="/servlet/linkout?suffix=cit1b&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2007.02.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2007.02.005%26sid%3Dliteratum%253Aachs%26aulast%3DKouzarides%26aufirst%3DT.%26atitle%3DChromatin%2520modifications%2520and%2520their%2520function%26jtitle%3DCell%26date%3D2007%26volume%3D128%26spage%3D693%26epage%3D705" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit1c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Mosammaparast, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Y.</span><span> </span><span class="NLM_article-title">Reversal of histone methylation: biochemical and molecular mechanisms of histone demethylases</span> <span class="citation_source-journal">Annu. Rev. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">79</span><span class="NLM_x">, </span> <span class="NLM_fpage">155</span><span class="NLM_x">–</span> <span class="NLM_lpage">79</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1c&amp;dbid=16&amp;doi=10.1021%2Fjm500249n&amp;key=10.1146%2Fannurev.biochem.78.070907.103946" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1c&amp;dbid=8&amp;doi=10.1021%2Fjm500249n&amp;key=20373914" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1c&amp;dbid=32&amp;doi=10.1021%2Fjm500249n&amp;key=1%3ACAS%3A528%3ADC%252BC3cXpslShsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=2010&pages=155-79&author=N.+Mosammaparastauthor=Y.+Shi&title=Reversal+of+histone+methylation%3A+biochemical+and+molecular+mechanisms+of+histone+demethylases"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1cR"><div class="casContent"><span class="casTitleNuber">1c</span><div class="casTitle"><span class="NLM_cas:atitle">Reversal of histone methylation: biochemical and molecular mechanisms of histone demethylases</span></div><div class="casAuthors">Mosammaparast, Nima; Shi, Yang</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biochemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">155-179</span>CODEN:
                <span class="NLM_cas:coden">ARBOAW</span>;
        ISSN:<span class="NLM_cas:issn">0066-4154</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  The importance of histone methylation in gene regulation was suggested over 40 years ago.  Yet, the dynamic nature of this histone modification was recognized only recently, with the discovery of the first histone demethylase nearly five years ago.  Since then, our insight into the mechanisms, structures, and macromol. complexes of these enzymes has grown exponentially.  Overall, the evidence strongly supports a key role for histone demethylases in eukaryotic transcription and other chromatin-dependent processes.  Here, we examine these and related facets of histone demethylases discovered to date, focusing on their biochem., structure, and enzymol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGquQJN8-1hp07Vg90H21EOLACvtfcHk0lgk7V0vVbulUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXpslShsbk%253D&md5=781bebbd65cc2f68ebe31490e93da09c</span></div><a href="/servlet/linkout?suffix=cit1c&amp;dbid=16384&amp;doi=10.1146%2Fannurev.biochem.78.070907.103946&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.biochem.78.070907.103946%26sid%3Dliteratum%253Aachs%26aulast%3DMosammaparast%26aufirst%3DN.%26aulast%3DShi%26aufirst%3DY.%26atitle%3DReversal%2520of%2520histone%2520methylation%253A%2520biochemical%2520and%2520molecular%2520mechanisms%2520of%2520histone%2520demethylases%26jtitle%3DAnnu.%2520Rev.%2520Biochem.%26date%3D2010%26volume%3D79%26spage%3D155%26epage%3D79" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Breccia, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alimena, G.</span><span> </span><span class="NLM_article-title">NFκB as a potential therapeutic target in myelodysplastic syndromes and acute myeloid leukemia</span> <span class="citation_source-journal">Expert Opin. Ther. Targets</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">1157</span><span class="NLM_x">–</span> <span class="NLM_lpage">76</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2a&amp;dbid=16&amp;doi=10.1021%2Fjm500249n&amp;key=10.1517%2F14728222.2010.522570" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2a&amp;dbid=8&amp;doi=10.1021%2Fjm500249n&amp;key=20858024" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2a&amp;dbid=32&amp;doi=10.1021%2Fjm500249n&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht12ns7zE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2010&pages=1157-76&author=M.+Brecciaauthor=G.+Alimena&title=NF%CE%BAB+as+a+potential+therapeutic+target+in+myelodysplastic+syndromes+and+acute+myeloid+leukemia"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2aR"><div class="casContent"><span class="casTitleNuber">2a</span><div class="casTitle"><span class="NLM_cas:atitle">NF-κB as a potential therapeutic target in myelodysplastic syndromes and acute myeloid leukemia</span></div><div class="casAuthors">Breccia, Massimo; Alimena, Giuliana</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Targets</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1157-1176</span>CODEN:
                <span class="NLM_cas:coden">EOTTAO</span>;
        ISSN:<span class="NLM_cas:issn">1472-8222</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Importance of the field: The inactive NF-κB-inhibitor of NF-κB (IκB) complex is activated by stimuli including pro-inflammatory cytokines, mitogens, growth factors and stress-inducing agents.  The release of NF-κB facilitates its translocation to the nucleus, where it promotes cell survival by initiating transcription of genes encoding stress-response enzymes, cell-adhesion mols., pro-inflammatory cytokines and anti-apoptotic proteins.  NF-κB and assocd. regulatory factors (IκB kinase subunits and bcl-3) are implicated in hematol. and solid tumor malignancies.  NF-κB appears to be involved in cell proliferation control, apoptosis control, angiogenesis promotion and possibly regulation of diffusion of metastases.  There are several reports that inhibition of NF-κB as a therapeutic target may have a role in tumor cell death or growth inhibition.  Area covered in this review: We review data about inhibition of NF-κB in acute myeloid leukemia (AML) and myelodysplastic syndromes (MDS).  We describe the mol. mechanisms underlying NF-κB deregulation in these haematol. malignancies.  What the reader will gain: Constitutive activation of NF-κB in the nucleus has been reported in some varieties of MDS/AML.  The in vitro and in vivo results of NF-κB inhibition in myeloid malignancies are highlighted.  Take home message: NF-κB selective inhibitory drugs may be useful, either as single agents or assocd. with conventional chemotherapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpvyIKKXSfGnLVg90H21EOLACvtfcHk0li2VUwlT58jhg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht12ns7zE&md5=aea9295b4ea76817d6b81b582146b75d</span></div><a href="/servlet/linkout?suffix=cit2a&amp;dbid=16384&amp;doi=10.1517%2F14728222.2010.522570&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F14728222.2010.522570%26sid%3Dliteratum%253Aachs%26aulast%3DBreccia%26aufirst%3DM.%26aulast%3DAlimena%26aufirst%3DG.%26atitle%3DNF%25CE%25BAB%2520as%2520a%2520potential%2520therapeutic%2520target%2520in%2520myelodysplastic%2520syndromes%2520and%2520acute%2520myeloid%2520leukemia%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Targets%26date%3D2010%26volume%3D14%26spage%3D1157%26epage%3D76" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit2b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Rodriguez-Paredes, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Esteller, M.</span><span> </span><span class="NLM_article-title">Cancer epigenetics reaches mainstream oncology</span> <span class="citation_source-journal">Nat. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">330</span><span class="NLM_x">–</span> <span class="NLM_lpage">9</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2b&amp;dbid=16&amp;doi=10.1021%2Fjm500249n&amp;key=10.1038%2Fnm.2305" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2b&amp;dbid=8&amp;doi=10.1021%2Fjm500249n&amp;key=21386836" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2b&amp;dbid=32&amp;doi=10.1021%2Fjm500249n&amp;key=1%3ACAS%3A528%3ADC%252BC3MXivVGgtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2011&pages=330-9&author=M.+Rodriguez-Paredesauthor=M.+Esteller&title=Cancer+epigenetics+reaches+mainstream+oncology"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2bR"><div class="casContent"><span class="casTitleNuber">2b</span><div class="casTitle"><span class="NLM_cas:atitle">Cancer epigenetics reaches mainstream oncology</span></div><div class="casAuthors">Rodriguez-Paredes, Manuel; Esteller, Manel</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">330-339</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Epigenetics is one of the most promising and expanding fields in the current biomedical research landscape.  Since the inception of epigenetics in the 1940s, the discoveries regarding its implications in normal and disease biol. have not stopped, compiling a vast amt. of knowledge in the past decade.  The field has moved from just one recognized marker, DNA methylation, to a variety of others, including a wide spectrum of histone modifications.  From the methodol. standpoint, the successful initial single gene candidate approaches have been complemented by the current comprehensive epigenomic approaches that allow the interrogation of genomes to search for translational applications in an unbiased manner.  Most important, the discovery of mutations in the epigenetic machinery and the approval of the first epigenetic drugs for the treatment of subtypes of leukemias and lymphomas has been an eye-opener for many biomedical scientists and clinicians.  Herein, we will summarize the progress in the field of cancer epigenetics research that has reached mainstream oncol. in the development of new biomarkers of the disease and new pharmacol. strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxyADVbZ4KHrVg90H21EOLACvtfcHk0li2VUwlT58jhg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXivVGgtb8%253D&md5=07b6faaddb8bce884261990dbc927331</span></div><a href="/servlet/linkout?suffix=cit2b&amp;dbid=16384&amp;doi=10.1038%2Fnm.2305&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.2305%26sid%3Dliteratum%253Aachs%26aulast%3DRodriguez-Paredes%26aufirst%3DM.%26aulast%3DEsteller%26aufirst%3DM.%26atitle%3DCancer%2520epigenetics%2520reaches%2520mainstream%2520oncology%26jtitle%3DNat.%2520Med.%26date%3D2011%26volume%3D17%26spage%3D330%26epage%3D9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit2c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Chi, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allis, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, G. G.</span><span> </span><span class="NLM_article-title">Covalent histone modifications--miswritten, misinterpreted and mis-erased in human cancers</span> <span class="citation_source-journal">Nat. Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">457</span><span class="NLM_x">–</span> <span class="NLM_lpage">69</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2c&amp;dbid=16&amp;doi=10.1021%2Fjm500249n&amp;key=10.1038%2Fnrc2876" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2c&amp;dbid=8&amp;doi=10.1021%2Fjm500249n&amp;key=20574448" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2c&amp;dbid=32&amp;doi=10.1021%2Fjm500249n&amp;key=1%3ACAS%3A528%3ADC%252BC3cXnvVShurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2010&pages=457-69&author=P.+Chiauthor=C.+D.+Allisauthor=G.+G.+Wang&title=Covalent+histone+modifications%2D%2Dmiswritten%2C+misinterpreted+and+mis-erased+in+human+cancers"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2cR"><div class="casContent"><span class="casTitleNuber">2c</span><div class="casTitle"><span class="NLM_cas:atitle">Covalent histone modifications - miswritten, misinterpreted and mis-erased in human cancers</span></div><div class="casAuthors">Chi, Ping; Allis, C. David; Wang, Gang Greg</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">457-469</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Post-translational modification of histones provides an important regulatory platform for processes such as gene transcription and DNA damage repair.  It has become increasingly apparent that the misregulation of histone modification, which is caused by the deregulation of factors that mediate the modification installation, removal and/or interpretation, actively contributes to human cancer.  In this review, we summarize recent advances in understanding the interpretation of certain histone methylations by plant homeodomain finger-contg. proteins, and how misreading, miswriting and mis-erasing of histone methylation marks can be assocd. with oncogenesis and progression.  These observations provide us with a greater mechanistic understanding of epigenetic alterations in human cancers and might also help direct new therapeutic interventions in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGMmMSNkuTSrVg90H21EOLACvtfcHk0li2VUwlT58jhg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXnvVShurc%253D&md5=08cd0d4dcd62c2fc90f4768a8eb29e3c</span></div><a href="/servlet/linkout?suffix=cit2c&amp;dbid=16384&amp;doi=10.1038%2Fnrc2876&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2876%26sid%3Dliteratum%253Aachs%26aulast%3DChi%26aufirst%3DP.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26aulast%3DWang%26aufirst%3DG.%2BG.%26atitle%3DCovalent%2520histone%2520modifications--miswritten%252C%2520misinterpreted%2520and%2520mis-erased%2520in%2520human%2520cancers%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2010%26volume%3D10%26spage%3D457%26epage%3D69" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Kampranis, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsichlis, P. N.</span><span> </span><span class="NLM_article-title">Histone demethylases and cancer</span> <span class="citation_source-journal">Adv. Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">102</span><span class="NLM_x">, </span> <span class="NLM_fpage">103</span><span class="NLM_x">–</span> <span class="NLM_lpage">69</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2009&pages=103-69&author=S.+C.+Kampranisauthor=P.+N.+Tsichlis&title=Histone+demethylases+and+cancer"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3a&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKampranis%26aufirst%3DS.%2BC.%26aulast%3DTsichlis%26aufirst%3DP.%2BN.%26atitle%3DHistone%2520demethylases%2520and%2520cancer%26jtitle%3DAdv.%2520Cancer%2520Res.%26date%3D2009%26volume%3D102%26spage%3D103%26epage%3D69" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit3b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Wang, L.-Y.; Guo, W.; Kim, K.; Pochampalli, M.; Hung, C.-L.; Izumiya, Y.; Kung, H.-J.</span><span> </span><span class="NLM_article-title">Histone demethylases in prostate cancer</span>. In  <span class="citation_source-book">Nuclear Signaling Pathways and Targeting Transcription in Cancer</span>; <span class="NLM_contrib-group">Kumer, R.</span>, Ed; <span class="NLM_publisher-name">Springer Science+Business Media</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">2013</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=L.-Y.+Wang&author=W.+Guo&author=K.+Kim&author=M.+Pochampalli&author=C.-L.+Hung&author=Y.+Izumiya&author=H.-J.+Kungauthor=R.+Kumer&title=Nuclear+Signaling+Pathways+and+Targeting+Transcription+in+Cancer"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3b&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.-Y.%26atitle%3DHistone%2520demethylases%2520in%2520prostate%2520cancer%26btitle%3DNuclear%2520Signaling%2520Pathways%2520and%2520Targeting%2520Transcription%2520in%2520Cancer%26aulast%3DKumer%26aufirst%3DR.%26pub%3DSpringer%2520Science%252BBusiness%2520Media%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Katoh, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Katoh, M.</span><span> </span><span class="NLM_article-title">Identification and characterization of JMJD2 family genes in silico</span> <span class="citation_source-journal">Int. J. Oncol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">1623</span><span class="NLM_x">–</span> <span class="NLM_lpage">8</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4a&amp;dbid=8&amp;doi=10.1021%2Fjm500249n&amp;key=15138608" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4a&amp;dbid=32&amp;doi=10.1021%2Fjm500249n&amp;key=1%3ACAS%3A528%3ADC%252BD2cXkvFaisbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2004&pages=1623-8&author=M.+Katohauthor=M.+Katoh&title=Identification+and+characterization+of+JMJD2+family+genes+in+silico"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4aR"><div class="casContent"><span class="casTitleNuber">4a</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and characterization of JMJD2 family genes in silico</span></div><div class="casAuthors">Katoh, Masuko; Katoh, Masaru</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Oncology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1623-1628</span>CODEN:
                <span class="NLM_cas:coden">IJONES</span>;
        ISSN:<span class="NLM_cas:issn">1019-6439</span>.
    
            (<span class="NLM_cas:orgname">International Journal of Oncology</span>)
        </div><div class="casAbstract">JMJD1 family genes, including JMJD1 (TSGA), TRIP8, and C5orf7 (5qNCA), encode nuclear proteins with Jumonji C (JmjC), TRI8H1 and TRI8H2 domains.  Here, we identified JMJD2 family genes.  Human genes corresponding to KIAA0677, KIAA0876, KIAA0780 and FLJ10251 cDNAs were designated JMJD2A, JMJD2B, JMJD2C, and JMJD2D, resp.  In addn., JMJD2D homologous genes within human genome sequences AP002383.3 and AP001264.4 were designated JMJD2E and JMJD2F, resp.  JMJD2A gene was mapped to human chromosome 1p34.1, JMJD2B gene to 19p13.3, and JMJD2C gene to 9p24.1.  JMJD2D, JMJD2E and JMJD2F genes were clustered at human chromosome 11q21.  Coding region of JMJD2A, JMJD2B, JMJD2C genes was located on multiple exons, while that of JMJD2D, JMJD2E and JMJD2F genes was located on a single exon.  These facts strongly indicate that JMJD2D, JMJD2E and JMJD2F genes are locally triplicated retrotransposed elements derived from JMJD2 family gene.  JD2H domain with C2HC2HC2- and C5HC2-type Cys (His) clusters was identified as the region conserved among JMJD2A (1064 aa), JMJD2B (1096 aa), and JMJD2C (1056 aa) proteins.  JMJD2A, JMJD2B and JMJD2C consist of JmjN, JmjC, JD2H, and two TUDOR domains, while JMJD2D (523 aa), JMJD2E (506 aa) and JMJD2F (638 aa) consist of JmjN and JmjC domains.  JMJD2 family proteins were classified into one group with JD2H and TUDOR domains and another group without JD2H or TUDOR domains.  Because JMJD2C gene (also known as GASC1 gene) is amplified in esophageal squamous cell carcinoma (ESCC), JMJD2 family genes are cancer-assocd. genes.  This is the first report on identification and characterization of human JMJD2 gene family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2QrOZpXEa9rVg90H21EOLACvtfcHk0lg5LlgxHQ3FYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXkvFaisbo%253D&md5=b6ab9ea6d3aebb1c242870726103a449</span></div><a href="/servlet/linkout?suffix=cit4a&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKatoh%26aufirst%3DM.%26aulast%3DKatoh%26aufirst%3DM.%26atitle%3DIdentification%2520and%2520characterization%2520of%2520JMJD2%2520family%2520genes%2520in%2520silico%26jtitle%3DInt.%2520J.%2520Oncol.%26date%3D2004%26volume%3D24%26spage%3D1623%26epage%3D8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit4b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Berry, W. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janknecht, R.</span><span> </span><span class="NLM_article-title">KDM4/JMJD2 histone demethylases: epigenetic regulators in cancer cells</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">73</span><span class="NLM_x">, </span> <span class="NLM_fpage">2936</span><span class="NLM_x">–</span> <span class="NLM_lpage">42</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2013&pages=2936-42&author=W.+L.+Berryauthor=R.+Janknecht&title=KDM4%2FJMJD2+histone+demethylases%3A+epigenetic+regulators+in+cancer+cells"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4b&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBerry%26aufirst%3DW.%2BL.%26aulast%3DJanknecht%26aufirst%3DR.%26atitle%3DKDM4%252FJMJD2%2520histone%2520demethylases%253A%2520epigenetic%2520regulators%2520in%2520cancer%2520cells%26jtitle%3DCancer%2520Res.%26date%3D2013%26volume%3D73%26spage%3D2936%26epage%3D42" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Shin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janknecht, R.</span><span> </span><span class="NLM_article-title">Activation of androgen receptor by histone demethylases JMJD2A and JMJD2D</span> <span class="citation_source-journal">Biochem. Biophys. Res. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">359</span><span class="NLM_x">, </span> <span class="NLM_fpage">742</span><span class="NLM_x">–</span> <span class="NLM_lpage">6</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=359&publication_year=2007&pages=742-6&author=S.+Shinauthor=R.+Janknecht&title=Activation+of+androgen+receptor+by+histone+demethylases+JMJD2A+and+JMJD2D"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5a&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DShin%26aufirst%3DS.%26aulast%3DJanknecht%26aufirst%3DR.%26atitle%3DActivation%2520of%2520androgen%2520receptor%2520by%2520histone%2520demethylases%2520JMJD2A%2520and%2520JMJD2D%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2007%26volume%3D359%26spage%3D742%26epage%3D6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit5b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Wissmann, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yin, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muller, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greschik, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fodor, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenuwein, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vogler, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneider, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gunther, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buettner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metzger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schule, R.</span><span> </span><span class="NLM_article-title">Cooperative demethylation by JMJD2C and LSD1 promotes androgen receptor-dependent gene expression</span> <span class="citation_source-journal">Nat. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">347</span><span class="NLM_x">–</span> <span class="NLM_lpage">53</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5b&amp;dbid=16&amp;doi=10.1021%2Fjm500249n&amp;key=10.1038%2Fncb1546" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5b&amp;dbid=8&amp;doi=10.1021%2Fjm500249n&amp;key=17277772" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5b&amp;dbid=32&amp;doi=10.1021%2Fjm500249n&amp;key=1%3ACAS%3A528%3ADC%252BD2sXit12rsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2007&pages=347-53&author=M.+Wissmannauthor=N.+Yinauthor=J.+M.+Mullerauthor=H.+Greschikauthor=B.+D.+Fodorauthor=T.+Jenuweinauthor=C.+Voglerauthor=R.+Schneiderauthor=T.+Guntherauthor=R.+Buettnerauthor=E.+Metzgerauthor=R.+Schule&title=Cooperative+demethylation+by+JMJD2C+and+LSD1+promotes+androgen+receptor-dependent+gene+expression"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5bR"><div class="casContent"><span class="casTitleNuber">5b</span><div class="casTitle"><span class="NLM_cas:atitle">Cooperative demethylation by JMJD2C and LSD1 promotes androgen receptor-dependent gene expression</span></div><div class="casAuthors">Wissmann, Melanie; Yin, Na; Mueller, Judith M.; Greschik, Holger; Fodor, Barna D.; Jenuwein, Thomas; Vogler, Christine; Schneider, Robert; Guenther, Thomas; Buettner, Reinhard; Metzger, Eric; Schuele, Roland</div><div class="citationInfo"><span class="NLM_cas:title">Nature Cell Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">347-353</span>CODEN:
                <span class="NLM_cas:coden">NCBIFN</span>;
        ISSN:<span class="NLM_cas:issn">1465-7392</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Posttranslational modifications of histones, such as methylation, regulate chromatin structure and gene expression.  Recently, lysine-specific demethylase 1 (LSD1), the first histone demethylase, was identified.  LSD1 interacts with the androgen receptor and promotes androgen-dependent transcription of target genes by ligand-induced demethylation of mono- and dimethylated histone H3 at Lys 9 (H3K9) only.  Here, we identify the Jumonji C (JMJC) domain-contg. protein JMJD2C as the first histone tridemethylase regulating androgen receptor function.  JMJD2C interacts with androgen receptor in vitro and in vivo.  Assembly of ligand-bound androgen receptor and JMJD2C on androgen receptor-target genes results in demethylation of tri-Me H3K9 and in stimulation of androgen receptor-dependent transcription.  Conversely, knockdown of JMJD2C inhibits androgen-induced removal of tri-Me H3K9, transcriptional activation and tumor cell proliferation.  Importantly, JMJD2C colocalizes with androgen receptor and LSD1 in normal prostate and in prostate carcinomas.  JMJD2C and LSD1 interact and both demethylases cooperatively stimulate androgen receptor-dependent gene transcription.  In addn., androgen receptor, JMJD2C and LSD1 assemble on chromatin to remove Me groups from mono, di and trimethylated H3K9.  Thus, our data suggest that specific gene regulation requires the assembly and coordinate action of demethylases with distinct substrate specificities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2b5sXBcoCRrVg90H21EOLACvtfcHk0lg5LlgxHQ3FYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXit12rsrw%253D&md5=d1f7ec6fbaadf301372ebe644a2e978a</span></div><a href="/servlet/linkout?suffix=cit5b&amp;dbid=16384&amp;doi=10.1038%2Fncb1546&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncb1546%26sid%3Dliteratum%253Aachs%26aulast%3DWissmann%26aufirst%3DM.%26aulast%3DYin%26aufirst%3DN.%26aulast%3DMuller%26aufirst%3DJ.%2BM.%26aulast%3DGreschik%26aufirst%3DH.%26aulast%3DFodor%26aufirst%3DB.%2BD.%26aulast%3DJenuwein%26aufirst%3DT.%26aulast%3DVogler%26aufirst%3DC.%26aulast%3DSchneider%26aufirst%3DR.%26aulast%3DGunther%26aufirst%3DT.%26aulast%3DBuettner%26aufirst%3DR.%26aulast%3DMetzger%26aufirst%3DE.%26aulast%3DSchule%26aufirst%3DR.%26atitle%3DCooperative%2520demethylation%2520by%2520JMJD2C%2520and%2520LSD1%2520promotes%2520androgen%2520receptor-dependent%2520gene%2520expression%26jtitle%3DNat.%2520Cell%2520Biol.%26date%3D2007%26volume%3D9%26spage%3D347%26epage%3D53" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit5c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Coffey, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rogerson, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ryan-Munden, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alkharaif, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stockley, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heer, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sahadevan, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Neill, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Darby, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Staller, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mantilla, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gaughan, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robson, C. N.</span><span> </span><span class="NLM_article-title">The lysine demethylase, KDM4B, is a key molecule in androgen receptor signalling and turnover</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">4433</span><span class="NLM_x">–</span> <span class="NLM_lpage">46</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2013&pages=4433-46&author=K.+Coffeyauthor=L.+Rogersonauthor=C.+Ryan-Mundenauthor=D.+Alkharaifauthor=J.+Stockleyauthor=R.+Heerauthor=K.+Sahadevanauthor=D.+O%E2%80%99Neillauthor=D.+Jonesauthor=S.+Darbyauthor=P.+Stallerauthor=A.+Mantillaauthor=L.+Gaughanauthor=C.+N.+Robson&title=The+lysine+demethylase%2C+KDM4B%2C+is+a+key+molecule+in+androgen+receptor+signalling+and+turnover"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5c&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCoffey%26aufirst%3DK.%26aulast%3DRogerson%26aufirst%3DL.%26aulast%3DRyan-Munden%26aufirst%3DC.%26aulast%3DAlkharaif%26aufirst%3DD.%26aulast%3DStockley%26aufirst%3DJ.%26aulast%3DHeer%26aufirst%3DR.%26aulast%3DSahadevan%26aufirst%3DK.%26aulast%3DO%25E2%2580%2599Neill%26aufirst%3DD.%26aulast%3DJones%26aufirst%3DD.%26aulast%3DDarby%26aufirst%3DS.%26aulast%3DStaller%26aufirst%3DP.%26aulast%3DMantilla%26aufirst%3DA.%26aulast%3DGaughan%26aufirst%3DL.%26aulast%3DRobson%26aufirst%3DC.%2BN.%26atitle%3DThe%2520lysine%2520demethylase%252C%2520KDM4B%252C%2520is%2520a%2520key%2520molecule%2520in%2520androgen%2520receptor%2520signalling%2520and%2520turnover%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2013%26volume%3D41%26spage%3D4433%26epage%3D46" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Shi, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yi, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Han, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xuan, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yao, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shang, Y.</span><span> </span><span class="NLM_article-title">Histone demethylase JMJD2B coordinates H3K4/H3K9 methylation and promotes hormonally responsive breast carcinogenesis</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">108</span><span class="NLM_x">, </span> <span class="NLM_fpage">7541</span><span class="NLM_x">–</span> <span class="NLM_lpage">6</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2011&pages=7541-6&author=L.+Shiauthor=L.+Sunauthor=Q.+Liauthor=J.+Liangauthor=W.+Yuauthor=X.+Yiauthor=X.+Yangauthor=Y.+Liauthor=X.+Hanauthor=Y.+Zhangauthor=C.+Xuanauthor=Z.+Yaoauthor=Y.+Shang&title=Histone+demethylase+JMJD2B+coordinates+H3K4%2FH3K9+methylation+and+promotes+hormonally+responsive+breast+carcinogenesis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6a&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DL.%26aulast%3DSun%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DLiang%26aufirst%3DJ.%26aulast%3DYu%26aufirst%3DW.%26aulast%3DYi%26aufirst%3DX.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DHan%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DXuan%26aufirst%3DC.%26aulast%3DYao%26aufirst%3DZ.%26aulast%3DShang%26aufirst%3DY.%26atitle%3DHistone%2520demethylase%2520JMJD2B%2520coordinates%2520H3K4%252FH3K9%2520methylation%2520and%2520promotes%2520hormonally%2520responsive%2520breast%2520carcinogenesis%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2011%26volume%3D108%26spage%3D7541%26epage%3D6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit6b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Kawazu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saso, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tong, K. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McQuire, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goto, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Son, D. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wakeham, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyagishi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mak, T. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Okada, H.</span><span> </span><span class="NLM_article-title">Histone demethylase JMJD2B functions as a co-factor of estrogen receptor in breast cancer proliferation and mammary gland development</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">e17830</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=e17830&author=M.+Kawazuauthor=K.+Sasoauthor=K.+I.+Tongauthor=T.+McQuireauthor=K.+Gotoauthor=D.+O.+Sonauthor=A.+Wakehamauthor=M.+Miyagishiauthor=T.+W.+Makauthor=H.+Okada&title=Histone+demethylase+JMJD2B+functions+as+a+co-factor+of+estrogen+receptor+in+breast+cancer+proliferation+and+mammary+gland+development"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6b&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKawazu%26aufirst%3DM.%26aulast%3DSaso%26aufirst%3DK.%26aulast%3DTong%26aufirst%3DK.%2BI.%26aulast%3DMcQuire%26aufirst%3DT.%26aulast%3DGoto%26aufirst%3DK.%26aulast%3DSon%26aufirst%3DD.%2BO.%26aulast%3DWakeham%26aufirst%3DA.%26aulast%3DMiyagishi%26aufirst%3DM.%26aulast%3DMak%26aufirst%3DT.%2BW.%26aulast%3DOkada%26aufirst%3DH.%26atitle%3DHistone%2520demethylase%2520JMJD2B%2520functions%2520as%2520a%2520co-factor%2520of%2520estrogen%2520receptor%2520in%2520breast%2520cancer%2520proliferation%2520and%2520mammary%2520gland%2520development%26jtitle%3DPLoS%2520One%26date%3D2011%26volume%3D6%26spage%3De17830" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit6c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Berry, W. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lightfoot, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janknecht, R.</span><span> </span><span class="NLM_article-title">Oncogenic features of the JMJD2A histone demethylase in breast cancer</span> <span class="citation_source-journal">Int. J. Oncol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">1701</span><span class="NLM_x">–</span> <span class="NLM_lpage">6</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6c&amp;dbid=16&amp;doi=10.1021%2Fjm500249n&amp;key=10.3892%2Fijo.2012.1618" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6c&amp;dbid=8&amp;doi=10.1021%2Fjm500249n&amp;key=22948256" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6c&amp;dbid=32&amp;doi=10.1021%2Fjm500249n&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslWisLrF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2012&pages=1701-6&author=W.+L.+Berryauthor=S.+Shinauthor=S.+A.+Lightfootauthor=R.+Janknecht&title=Oncogenic+features+of+the+JMJD2A+histone+demethylase+in+breast+cancer"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6cR"><div class="casContent"><span class="casTitleNuber">6c</span><div class="casTitle"><span class="NLM_cas:atitle">Oncogenic features of the JMJD2A histone demethylase in breast cancer</span></div><div class="casAuthors">Berry, William L.; Shin, Sook; Lightfoot, Stan A.; Janknecht, Ralf</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Oncology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1701-1706</span>CODEN:
                <span class="NLM_cas:coden">IJONES</span>;
        ISSN:<span class="NLM_cas:issn">1019-6439</span>.
    
            (<span class="NLM_cas:orgname">International Journal of Oncology</span>)
        </div><div class="casAbstract">Estrogen receptor α (ERα) plays a pivotal role in the genesis of the majority of breast tumors.  Consequently, endocrine therapy is now routinely utilized in the clinic for the treatment of ERα-pos. breast cancer patients.  However, how ERα activity becomes dysregulated in breast cancer cells remains to be elucidated.  The aim of this study was to show that the histone demethylase JMJD2A, also known as KDM4A, is capable of forming a complex with ERα in vivo.  Moreover, wild-type JMJD2A, but not a catalytically impaired mutant, was able to strongly coactivate ERα-mediated transcription.  Consistently, the downregulation of JMJD2A in human T47D breast cancer cells led to a decreased expression of cyclin D1, a prominent ERα target gene and cell cycle regulator.  The downregulation of JMJD2A induced a redn. in the growth of T47D cells.  In addn., we found that JMJD2A is overexpressed in human breast tumors both at the mRNA and protein level.  Taken together, these data indicate that the overexpression of JMJD2A may contribute to breast tumor formation by stimulating ERα activity and that JMJD2A may be a breast-relevant oncoprotein.  As such, small mol. drugs targeting the catalytic center of JMJD2A might be useful in breast cancer adjuvant therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpubrKLORIlcrVg90H21EOLACvtfcHk0lje80mHrfan7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslWisLrF&md5=13f012c96876d1b49e2f10f08985d853</span></div><a href="/servlet/linkout?suffix=cit6c&amp;dbid=16384&amp;doi=10.3892%2Fijo.2012.1618&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3892%252Fijo.2012.1618%26sid%3Dliteratum%253Aachs%26aulast%3DBerry%26aufirst%3DW.%2BL.%26aulast%3DShin%26aufirst%3DS.%26aulast%3DLightfoot%26aufirst%3DS.%2BA.%26aulast%3DJanknecht%26aufirst%3DR.%26atitle%3DOncogenic%2520features%2520of%2520the%2520JMJD2A%2520histone%2520demethylase%2520in%2520breast%2520cancer%26jtitle%3DInt.%2520J.%2520Oncol.%26date%3D2012%26volume%3D41%26spage%3D1701%26epage%3D6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Hamada, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Itoh, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsumoto, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakagawa, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janknecht, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyata, N.</span><span> </span><span class="NLM_article-title">Synthesis and activity of N-oxalylglycine and its derivatives as Jumonji C-domain-containing histone lysine demethylase inhibitors</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">2852</span><span class="NLM_x">–</span> <span class="NLM_lpage">5</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=2852-5&author=S.+Hamadaauthor=T.+D.+Kimauthor=T.+Suzukiauthor=Y.+Itohauthor=H.+Tsumotoauthor=H.+Nakagawaauthor=R.+Janknechtauthor=N.+Miyata&title=Synthesis+and+activity+of+N-oxalylglycine+and+its+derivatives+as+Jumonji+C-domain-containing+histone+lysine+demethylase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7a&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHamada%26aufirst%3DS.%26aulast%3DKim%26aufirst%3DT.%2BD.%26aulast%3DSuzuki%26aufirst%3DT.%26aulast%3DItoh%26aufirst%3DY.%26aulast%3DTsumoto%26aufirst%3DH.%26aulast%3DNakagawa%26aufirst%3DH.%26aulast%3DJanknecht%26aufirst%3DR.%26aulast%3DMiyata%26aufirst%3DN.%26atitle%3DSynthesis%2520and%2520activity%2520of%2520N-oxalylglycine%2520and%2520its%2520derivatives%2520as%2520Jumonji%2520C-domain-containing%2520histone%2520lysine%2520demethylase%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2009%26volume%3D19%26spage%3D2852%26epage%3D5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit7b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Rose, N. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woon, E. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kingham, G. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">King, O. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mecinovic, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clifton, I. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ng, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Talib-Hardy, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oppermann, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDonough, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span> </span><span class="NLM_article-title">Selective inhibitors of the JMJD2 histone demethylases: combined nondenaturing mass spectrometric screening and crystallographic approaches</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">1810</span><span class="NLM_x">–</span> <span class="NLM_lpage">8</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm901680b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref7/cit7b&amp;dbid=32&amp;doi=10.1021%2Fjm500249n&amp;key=1%3ACAS%3A528%3ADC%252BC3cXnvFWnsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=1810-8&author=N.+R.+Roseauthor=E.+C.+Woonauthor=G.+L.+Kinghamauthor=O.+N.+Kingauthor=J.+Mecinovicauthor=I.+J.+Cliftonauthor=S.+S.+Ngauthor=J.+Talib-Hardyauthor=U.+Oppermannauthor=M.+A.+McDonoughauthor=C.+J.+Schofield&title=Selective+inhibitors+of+the+JMJD2+histone+demethylases%3A+combined+nondenaturing+mass+spectrometric+screening+and+crystallographic+approaches"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7bR"><div class="casContent"><span class="casTitleNuber">7b</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Inhibitors of the JMJD2 Histone Demethylases: Combined Nondenaturing Mass Spectrometric Screening and Crystallographic Approaches</span></div><div class="casAuthors">Rose, Nathan R.; Woon, Esther C. Y.; Kingham, Guy L.; King, Oliver N. F.; Mecinovic, Jasmin; Clifton, Ian J.; Ng, Stanley S.; Talib-Hardy, Jobina; Oppermann, Udo; McDonough, Michael A.; Schofield, Christopher J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1810-1818</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Ferrous ion and 2-oxoglutarate (2OG) oxygenases catalyze the demethylation of Nε-methylated lysine residues in histones.  Here, studies on the inhibition of the JMJD2 subfamily of histone demethylases are reported, employing binding analyses by non-denaturing mass spectrometry (MS), dynamic combinatorial chem. coupled to MS, turnover assays, and crystallog.  The results of initial binding and inhibition assays directed the prodn. and anal. of a set of N-oxalyl-D-tyrosine derivs. to explore the extent of a sub-pocket at the JMJD2 active site.  Some of the inhibitors were shown to be selective for JMJD2 over the hypoxia-inducible factor prolyl hydroxylase PHD2.  A crystal structure of JMJD2A in complex with one of the potent inhibitors was obtained; modeling other inhibitors based on this structure predicts interactions that enable improved inhibition for some compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoa6dzV_ed-qrVg90H21EOLACvtfcHk0lgWLRhlLJ4lwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXnvFWnsQ%253D%253D&md5=3eb3c36d91f46193946f64288ea9ccae</span></div><a href="/servlet/linkout?suffix=cit7b&amp;dbid=16384&amp;doi=10.1021%2Fjm901680b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901680b%26sid%3Dliteratum%253Aachs%26aulast%3DRose%26aufirst%3DN.%2BR.%26aulast%3DWoon%26aufirst%3DE.%2BC.%26aulast%3DKingham%26aufirst%3DG.%2BL.%26aulast%3DKing%26aufirst%3DO.%2BN.%26aulast%3DMecinovic%26aufirst%3DJ.%26aulast%3DClifton%26aufirst%3DI.%2BJ.%26aulast%3DNg%26aufirst%3DS.%2BS.%26aulast%3DTalib-Hardy%26aufirst%3DJ.%26aulast%3DOppermann%26aufirst%3DU.%26aulast%3DMcDonough%26aufirst%3DM.%2BA.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26atitle%3DSelective%2520inhibitors%2520of%2520the%2520JMJD2%2520histone%2520demethylases%253A%2520combined%2520nondenaturing%2520mass%2520spectrometric%2520screening%2520and%2520crystallographic%2520approaches%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D1810%26epage%3D8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Thalhammer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mecinovic, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loenarz, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tumber, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rose, N. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heightman, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span> </span><span class="NLM_article-title">Inhibition of the histone demethylase JMJD2E by 3-substituted pyridine 2,4-dicarboxylates</span> <span class="citation_source-journal">Org. Biomol Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">127</span><span class="NLM_x">–</span> <span class="NLM_lpage">35</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2011&pages=127-35&author=A.+Thalhammerauthor=J.+Mecinovicauthor=C.+Loenarzauthor=A.+Tumberauthor=N.+R.+Roseauthor=T.+D.+Heightmanauthor=C.+J.+Schofield&title=Inhibition+of+the+histone+demethylase+JMJD2E+by+3-substituted+pyridine+2%2C4-dicarboxylates"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DThalhammer%26aufirst%3DA.%26aulast%3DMecinovic%26aufirst%3DJ.%26aulast%3DLoenarz%26aufirst%3DC.%26aulast%3DTumber%26aufirst%3DA.%26aulast%3DRose%26aufirst%3DN.%2BR.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26atitle%3DInhibition%2520of%2520the%2520histone%2520demethylase%2520JMJD2E%2520by%25203-substituted%2520pyridine%25202%252C4-dicarboxylates%26jtitle%3DOrg.%2520Biomol%2520Chem.%26date%3D2011%26volume%3D9%26spage%3D127%26epage%3D35" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">King, O. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, X. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sakurai, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawamura, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rose, N. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ng, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quinn, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rai, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mott, B. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beswick, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klose, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oppermann, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jadhav, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heightman, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maloney, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeonov, A.</span><span> </span><span class="NLM_article-title">Quantitative high-throughput screening identifies 8-hydroxyquinolines as cell-active histone demethylase inhibitors</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">e15535</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2010&pages=e15535&author=O.+N.+Kingauthor=X.+S.+Liauthor=M.+Sakuraiauthor=A.+Kawamuraauthor=N.+R.+Roseauthor=S.+S.+Ngauthor=A.+M.+Quinnauthor=G.+Raiauthor=B.+T.+Mottauthor=P.+Beswickauthor=R.+J.+Kloseauthor=U.+Oppermannauthor=A.+Jadhavauthor=T.+D.+Heightmanauthor=D.+J.+Maloneyauthor=C.+J.+Schofieldauthor=A.+Simeonov&title=Quantitative+high-throughput+screening+identifies+8-hydroxyquinolines+as+cell-active+histone+demethylase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKing%26aufirst%3DO.%2BN.%26aulast%3DLi%26aufirst%3DX.%2BS.%26aulast%3DSakurai%26aufirst%3DM.%26aulast%3DKawamura%26aufirst%3DA.%26aulast%3DRose%26aufirst%3DN.%2BR.%26aulast%3DNg%26aufirst%3DS.%2BS.%26aulast%3DQuinn%26aufirst%3DA.%2BM.%26aulast%3DRai%26aufirst%3DG.%26aulast%3DMott%26aufirst%3DB.%2BT.%26aulast%3DBeswick%26aufirst%3DP.%26aulast%3DKlose%26aufirst%3DR.%2BJ.%26aulast%3DOppermann%26aufirst%3DU.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DMaloney%26aufirst%3DD.%2BJ.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26aulast%3DSimeonov%26aufirst%3DA.%26atitle%3DQuantitative%2520high-throughput%2520screening%2520identifies%25208-hydroxyquinolines%2520as%2520cell-active%2520histone%2520demethylase%2520inhibitors%26jtitle%3DPLoS%2520One%26date%3D2010%26volume%3D5%26spage%3De15535" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Mackeen, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kramer, H. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, K. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coleman, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hopkinson, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kessler, B. M.</span><span> </span><span class="NLM_article-title">Small-molecule-based inhibition of histone demethylation in cells assessed by quantitative mass spectrometry</span> <span class="citation_source-journal">J. Proteome Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">4082</span><span class="NLM_x">–</span> <span class="NLM_lpage">92</span></span><div class="citationLinks">[<a href="/doi/10.1021/pr100269b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Fjm500249n&amp;key=1%3ACAS%3A528%3ADC%252BC3cXptVygu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2010&pages=4082-92&author=M.+M.+Mackeenauthor=H.+B.+Kramerauthor=K.+H.+Changauthor=M.+L.+Colemanauthor=R.+J.+Hopkinsonauthor=C.+J.+Schofieldauthor=B.+M.+Kessler&title=Small-molecule-based+inhibition+of+histone+demethylation+in+cells+assessed+by+quantitative+mass+spectrometry"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule-Based Inhibition of Histone Demethylation in Cells Assessed by Quantitative Mass Spectrometry</span></div><div class="casAuthors">MacKeen, Mukram M.; Kramer, Holger B.; Chang, Kai-Hsuan; Coleman, Matthew L.; Hopkinson, Richard J.; Schofield, Christopher J.; Kessler, Benedikt M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Proteome Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">4082-4092</span>CODEN:
                <span class="NLM_cas:coden">JPROBS</span>;
        ISSN:<span class="NLM_cas:issn">1535-3893</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Post-translational modifications on histones are an important mechanism for the regulation of gene expression and are involved in all aspects of cell growth and differentiation, as well as pathol. processes including neurodegeneration, autoimmunity, and cancer.  A major challenge within the chromatin field is to develop methods for the quant. anal. of histone modifications.  Here the authors report a mass spectrometry (MS) approach based on ultraperformance liq. chromatog. high/low collision switching (UPLC-MSE) to monitor histone modifications in cells.  This approach is exemplified by the anal. of trimethylated lysine-9 levels in histone H3, following a simple chem. derivatization procedure with d6-acetic anhydride.  This method was used to study the inhibition of histone demethylases with pyridine-2,4-dicarboxylic acid (PDCA) derivs. in cells.  The authors' results show that the PDCA-dimethyl ester inhibits JMJD2A catalyzed demethylation of lysine-9 on histone H3 in human HEK 293T cells.  Demethylase inhibition, as obsd. by MS analyses, was supported by immunoblotting with modification-specific antibodies.  The results demonstrate that PDCA derived small mols. are cell permeable demethylase inhibitors and reveal that quant. MS is a useful tool for measuring post-translational histone modifications in cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrysm0DR-5KtbVg90H21EOLACvtfcHk0ljLlv5PY4KNMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXptVygu7o%253D&md5=ef022b8021283895612a34bd0f75a1d5</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fpr100269b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fpr100269b%26sid%3Dliteratum%253Aachs%26aulast%3DMackeen%26aufirst%3DM.%2BM.%26aulast%3DKramer%26aufirst%3DH.%2BB.%26aulast%3DChang%26aufirst%3DK.%2BH.%26aulast%3DColeman%26aufirst%3DM.%2BL.%26aulast%3DHopkinson%26aufirst%3DR.%2BJ.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26aulast%3DKessler%26aufirst%3DB.%2BM.%26atitle%3DSmall-molecule-based%2520inhibition%2520of%2520histone%2520demethylation%2520in%2520cells%2520assessed%2520by%2520quantitative%2520mass%2520spectrometry%26jtitle%3DJ.%2520Proteome%2520Res.%26date%3D2010%26volume%3D9%26spage%3D4082%26epage%3D92" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Bradford, M. M.</span><span> </span><span class="NLM_article-title">A rapid and sensitive method for the quantitation of microgram quantities of protein utilizing the principle of protein-dye binding</span> <span class="citation_source-journal">Anal. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1976</span><span class="NLM_x">, </span> <span class="NLM_volume">72</span><span class="NLM_x">, </span> <span class="NLM_fpage">248</span><span class="NLM_x">–</span> <span class="NLM_lpage">54</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Fjm500249n&amp;key=10.1016%2F0003-2697%2876%2990527-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Fjm500249n&amp;key=942051" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Fjm500249n&amp;key=1%3ACAS%3A528%3ADyaE28XksVehtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=1976&pages=248-54&author=M.+M.+Bradford&title=A+rapid+and+sensitive+method+for+the+quantitation+of+microgram+quantities+of+protein+utilizing+the+principle+of+protein-dye+binding"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">A rapid and sensitive method for the quantitation of microgram quantities of protein utilizing the principle of protein-dye binding</span></div><div class="casAuthors">Bradford, Marion M.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">1976</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">248-54</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    </div><div class="casAbstract">A protein detn. method that involves the binding of coomassie Brilliant Blue G 250 to protein is described.  The binding of the dye to protein causes a shift in the absorption max. of the dye from 465 to 595 nm, and it is the increase in absorption at 595 nm that is monitored.  This assay is very reproducible and rapid with the dye binding process virtually complete in ∼ 2 min with good color stability for 1 hr.  There is little or no interference from cations such as Na+ or K+ nor from carbohydrates such as sucrose.  A small amt. of color is developed in the presence of strongly alk. buffering agents, but the assay may be run accurately by the use of proper buffer controls.  The only components found to give excessive interfering color in the assay are relatively large amts. of detergents such as Na dodecyl sulfate, Triton X 100, and commercial glassware detergents.  Interference by small amts. of detergent may be eliminated by the use of proper control.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrN_T_d0trcNrVg90H21EOLACvtfcHk0ljLlv5PY4KNMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE28XksVehtrY%253D&md5=43f388a493becc4193c7a49d9de0e1db</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2F0003-2697%2876%2990527-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0003-2697%252876%252990527-3%26sid%3Dliteratum%253Aachs%26aulast%3DBradford%26aufirst%3DM.%2BM.%26atitle%3DA%2520rapid%2520and%2520sensitive%2520method%2520for%2520the%2520quantitation%2520of%2520microgram%2520quantities%2520of%2520protein%2520utilizing%2520the%2520principle%2520of%2520protein-dye%2520binding%26jtitle%3DAnal.%2520Biochem.%26date%3D1976%26volume%3D72%26spage%3D248%26epage%3D54" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Otwinowski, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Minor, W.</span><span> </span><span class="NLM_article-title">Processing of X-ray diffraction data collected in oscillation mode</span> <span class="citation_source-journal">Methods Enzymol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">276</span><span class="NLM_x">, </span> <span class="NLM_fpage">307</span><span class="NLM_x">–</span> <span class="NLM_lpage">26</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Fjm500249n&amp;key=10.1016%2FS0076-6879%2897%2976066-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Fjm500249n&amp;key=27754618" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Fjm500249n&amp;key=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=1997&pages=307-26&author=Z.+Otwinowskiauthor=W.+Minor&title=Processing+of+X-ray+diffraction+data+collected+in+oscillation+mode"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Processing of x-ray diffraction data collected in oscillation mode</span></div><div class="casAuthors">Otwinowski, Zbyszek; Minor, Wladek</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">276</span>
        (<span class="NLM_cas:issue">Macromolecular Crystallography, Part A</span>),
    <span class="NLM_cas:pages">307-326</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Macromol. crystallog. is an iterative process.  Rarely do the first crystals provide all the necessary data to solve the biol. problem being studied.  Each step benefits from experience learned in previous steps.  To monitor the progress, the HKL package provides 2 tools: (1) statistics, both weighted (χ2) and unweighted (R-merge), are provided, and the Bayesian reasoning and multicomponent error model facilitates obtaining the proper error ests. and (2) visualization of the process plays a double role by helping the operator to confirm that the process of data redn., including the resulting statistics, is correct, and allowing one to evaluate problems for which there are no good statistical criteria.  Visualization also provides confidence that the point of diminishing returns in data collection and redn. has been reached.  At that point, the effort should be directed to solving the structure.  The methods presented here have been applied to solve a large variety of problems, from inorg. mols. with 5 Å unit cell to rotavirus of 700 Å diam. crystd. in 700 × 1000 × 1400 Å cell.  Overall quality of the method was tested by many researchers by successful application of the programs to MAD structure detns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrA3fvBOHqhhrVg90H21EOLACvtfcHk0lgmrp-Yd-19yg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D&md5=c9536971d4e32cc35352c40fb9368131</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2FS0076-6879%2897%2976066-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0076-6879%252897%252976066-X%26sid%3Dliteratum%253Aachs%26aulast%3DOtwinowski%26aufirst%3DZ.%26aulast%3DMinor%26aufirst%3DW.%26atitle%3DProcessing%2520of%2520X-ray%2520diffraction%2520data%2520collected%2520in%2520oscillation%2520mode%26jtitle%3DMethods%2520Enzymol.%26date%3D1997%26volume%3D276%26spage%3D307%26epage%3D26" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Collaborative Computational Project</span><span> </span><span class="NLM_article-title">The CCP4 suite: programs for protein crystallography</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">760</span><span class="NLM_x">–</span> <span class="NLM_lpage">3</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13a&amp;dbid=16&amp;doi=10.1021%2Fjm500249n&amp;key=10.1107%2FS0907444994003112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13a&amp;dbid=8&amp;doi=10.1021%2Fjm500249n&amp;key=15299374" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=1994&pages=760-3&author=Collaborative+Computational+Project&title=The+CCP4+suite%3A+programs+for+protein+crystallography"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13a&amp;dbid=16384&amp;doi=10.1107%2FS0907444994003112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444994003112%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DThe%2520CCP4%2520suite%253A%2520programs%2520for%2520protein%2520crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1994%26volume%3D50%26spage%3D760%26epage%3D3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit13b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Murshudov, G. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vagin, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dodson, E. J.</span><span> </span><span class="NLM_article-title">Refinement of macromolecular structures by the maximum-likelihood method</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">240</span><span class="NLM_x">–</span> <span class="NLM_lpage">55</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13b&amp;dbid=16&amp;doi=10.1021%2Fjm500249n&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13b&amp;dbid=8&amp;doi=10.1021%2Fjm500249n&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13b&amp;dbid=32&amp;doi=10.1021%2Fjm500249n&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-55&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+macromolecular+structures+by+the+maximum-likelihood+method"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13bR"><div class="casContent"><span class="casTitleNuber">13b</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. σA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial α-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0lgmrp-Yd-19yg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit13b&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520macromolecular%2520structures%2520by%2520the%2520maximum-likelihood%2520method%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1997%26volume%3D53%26spage%3D240%26epage%3D55" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Vagin, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teplyakov, A.</span><span> </span><span class="NLM_article-title">MOLREP: an automated program for molecular replacement</span> <span class="citation_source-journal">J. Appl. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">1022</span><span class="NLM_x">–</span> <span class="NLM_lpage">5</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Fjm500249n&amp;key=10.1107%2FS0021889897006766" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Fjm500249n&amp;key=1%3ACAS%3A528%3ADyaK1cXhslehu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=1997&pages=1022-5&author=A.+Vaginauthor=A.+Teplyakov&title=MOLREP%3A+an+automated+program+for+molecular+replacement"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">MOLREP: an automated program for molecular replacement</span></div><div class="casAuthors">Vagin, Alexei; Teplyakov, Alexei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1022-1025</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard International Publishers Ltd.</span>)
        </div><div class="casAbstract">MOLREP is an automated program for mol. replacement which uses effective new approaches in data processing and rotational and translational searching.  These include an automatic choice of all parameters, scaling by Patterson origin peaks and soft resoln. cutoff.  One of the cornerstones of the program is an original full-symmetry translation function combined with a packing function.  Information from the model already placed in the cell is incorporated in both translation and packing functions.  A no. of tests using exptl. data proved the ability of the program to find the correct soln. in difficult cases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkGXdZ-nTKxLVg90H21EOLACvtfcHk0lgmrp-Yd-19yg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXhslehu7Y%253D&md5=96af64aedd8a8f018c3c18b57a6b4b21</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1107%2FS0021889897006766&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889897006766%26sid%3Dliteratum%253Aachs%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DTeplyakov%26aufirst%3DA.%26atitle%3DMOLREP%253A%2520an%2520automated%2520program%2520for%2520molecular%2520replacement%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D1997%26volume%3D30%26spage%3D1022%26epage%3D5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Lamzin, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, K. S.</span><span> </span><span class="NLM_article-title">Automated refinement of protein models</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">127</span><span class="NLM_x">–</span> <span class="NLM_lpage">47</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=1993&pages=127-47&author=V.+Lamzinauthor=K.+S.+Wilson&title=Automated+refinement+of+protein+models"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLamzin%26aufirst%3DV.%26aulast%3DWilson%26aufirst%3DK.%2BS.%26atitle%3DAutomated%2520refinement%2520of%2520protein%2520models%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1993%26volume%3D49%26spage%3D127%26epage%3D47" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Laskowski, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacArthur, M. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moss, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thornton, J. M.</span><span> </span><span class="NLM_article-title">PROCHECK: a program to check the stereochemical quality of protein structures</span> <span class="citation_source-journal">J. Appl. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">283</span><span class="NLM_x">–</span> <span class="NLM_lpage">91</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Fjm500249n&amp;key=10.1107%2FS0021889892009944" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Fjm500249n&amp;key=1%3ACAS%3A528%3ADyaK3sXit12lurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=1993&pages=283-91&author=R.+A.+Laskowskiauthor=M.+W.+MacArthurauthor=D.+S.+Mossauthor=J.+M.+Thornton&title=PROCHECK%3A+a+program+to+check+the+stereochemical+quality+of+protein+structures"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">PROCHECK: a program to check the stereochemical quality of protein structures</span></div><div class="casAuthors">Laskowski, Roman A.; MacArthur, Malcolm W.; Moss, David S.; Thornton, Janet M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">283-91</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    </div><div class="casAbstract">The PROCHECK suite of programs provides a detailed check on the stereochem. of a protein structure.  Its outputs comprise a no. of plots in PostScript format and a comprehensive residue-by-residue listing.  These give an assessment of the overall quality of the structure as compared with well-refined structures of the same resoln. and also highlight regions that may need further investigation.  The PROCHECK programs are useful for assessing the quality not only of protein structures in the process of being solved but also of existing structures and of those being modeled on known structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCe5FgJf4KH7Vg90H21EOLACvtfcHk0liBS5xasfi-2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXit12lurY%253D&md5=fb69fc4410cd716aaaa7cc0db06b3ed2</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1107%2FS0021889892009944&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889892009944%26sid%3Dliteratum%253Aachs%26aulast%3DLaskowski%26aufirst%3DR.%2BA.%26aulast%3DMacArthur%26aufirst%3DM.%2BW.%26aulast%3DMoss%26aufirst%3DD.%2BS.%26aulast%3DThornton%26aufirst%3DJ.%2BM.%26atitle%3DPROCHECK%253A%2520a%2520program%2520to%2520check%2520the%2520stereochemical%2520quality%2520of%2520protein%2520structures%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D1993%26volume%3D26%26spage%3D283%26epage%3D91" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Chen, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kappler, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hong, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crawford, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lan, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whetstine, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dai, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hansen, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, G.</span><span> </span><span class="NLM_article-title">Structural basis of the recognition of a methylated histone tail by JMJD2A</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">104</span><span class="NLM_x">, </span> <span class="NLM_fpage">10818</span><span class="NLM_x">–</span> <span class="NLM_lpage">23</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2007&pages=10818-23&author=Z.+Chenauthor=J.+Zangauthor=J.+Kapplerauthor=X.+Hongauthor=F.+Crawfordauthor=Q.+Wangauthor=F.+Lanauthor=C.+Jiangauthor=J.+Whetstineauthor=S.+Daiauthor=K.+Hansenauthor=Y.+Shiauthor=G.+Zhang&title=Structural+basis+of+the+recognition+of+a+methylated+histone+tail+by+JMJD2A"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DZang%26aufirst%3DJ.%26aulast%3DKappler%26aufirst%3DJ.%26aulast%3DHong%26aufirst%3DX.%26aulast%3DCrawford%26aufirst%3DF.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DLan%26aufirst%3DF.%26aulast%3DJiang%26aufirst%3DC.%26aulast%3DWhetstine%26aufirst%3DJ.%26aulast%3DDai%26aufirst%3DS.%26aulast%3DHansen%26aufirst%3DK.%26aulast%3DShi%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DG.%26atitle%3DStructural%2520basis%2520of%2520the%2520recognition%2520of%2520a%2520methylated%2520histone%2520tail%2520by%2520JMJD2A%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2007%26volume%3D104%26spage%3D10818%26epage%3D23" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Krishnan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trievel, R. C.</span><span> </span><span class="NLM_article-title">Structural and functional analysis of JMJD2D reveals molecular basis for site-specific demethylation among JMJD2 demethylases</span> <span class="citation_source-journal">Structure</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">98</span><span class="NLM_x">–</span> <span class="NLM_lpage">108</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2013&pages=98-108&author=S.+Krishnanauthor=R.+C.+Trievel&title=Structural+and+functional+analysis+of+JMJD2D+reveals+molecular+basis+for+site-specific+demethylation+among+JMJD2+demethylases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKrishnan%26aufirst%3DS.%26aulast%3DTrievel%26aufirst%3DR.%2BC.%26atitle%3DStructural%2520and%2520functional%2520analysis%2520of%2520JMJD2D%2520reveals%2520molecular%2520basis%2520for%2520site-specific%2520demethylation%2520among%2520JMJD2%2520demethylases%26jtitle%3DStructure%26date%3D2013%26volume%3D21%26spage%3D98%26epage%3D108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Jones, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zou, J. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowan, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kjeldgaard, M.</span><span> </span><span class="NLM_article-title">Improved methods for building protein models in electron density maps and the location of errors in these models</span> <span class="citation_source-journal">Acta Crystallogr., Sect. A</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1991</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">110</span><span class="NLM_x">–</span> <span class="NLM_lpage">9</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Fjm500249n&amp;key=10.1107%2FS0108767390010224" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Fjm500249n&amp;key=2025413" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Fjm500249n&amp;key=1%3ACAS%3A280%3ADyaK3M3it1CmsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=1991&pages=110-9&author=T.+A.+Jonesauthor=J.+Y.+Zouauthor=S.+W.+Cowanauthor=M.+Kjeldgaard&title=Improved+methods+for+building+protein+models+in+electron+density+maps+and+the+location+of+errors+in+these+models"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Improved methods for building protein models in electron density maps and the location of errors in these models</span></div><div class="casAuthors">Jones T A; Zou J Y; Cowan S W; Kjeldgaard M</div><div class="citationInfo"><span class="NLM_cas:title">Acta crystallographica. Section A, Foundations of crystallography</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">47 ( Pt 2)</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">110-9</span>
        ISSN:<span class="NLM_cas:issn">0108-7673</span>.
    </div><div class="casAbstract">Map interpretation remains a critical step in solving the structure of a macromolecule.  Errors introduced at this early stage may persist throughout crystallographic refinement and result in an incorrect structure.  The normally quoted crystallographic residual is often a poor description for the quality of the model.  Strategies and tools are described that help to alleviate this problem.  These simplify the model-building process, quantify the goodness of fit of the model on a per-residue basis and locate possible errors in peptide and side-chain conformations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQhby6yNdW8QMKL0FlyGHDofW6udTcc2eYk7TJsuUv7qLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK3M3it1CmsQ%253D%253D&md5=fd4d4a3cdb4ecb3f05bbad8b2300ef45</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1107%2FS0108767390010224&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0108767390010224%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DT.%2BA.%26aulast%3DZou%26aufirst%3DJ.%2BY.%26aulast%3DCowan%26aufirst%3DS.%2BW.%26aulast%3DKjeldgaard%26aufirst%3DM.%26atitle%3DImproved%2520methods%2520for%2520building%2520protein%2520models%2520in%2520electron%2520density%2520maps%2520and%2520the%2520location%2520of%2520errors%2520in%2520these%2520models%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520A%26date%3D1991%26volume%3D47%26spage%3D110%26epage%3D9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Raghuraman, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mosier, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Desai, U. R.</span><span> </span><span class="NLM_article-title">Finding a needle in a haystack: development of a combinatorial virtual screening approach for identifying high specificity heparin/heparan sulfate sequence(s)</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">3553</span><span class="NLM_x">–</span> <span class="NLM_lpage">62</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm060092o" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=3553-62&author=A.+Raghuramanauthor=P.+D.+Mosierauthor=U.+R.+Desai&title=Finding+a+needle+in+a+haystack%3A+development+of+a+combinatorial+virtual+screening+approach+for+identifying+high+specificity+heparin%2Fheparan+sulfate+sequence%28s%29"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm060092o&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm060092o%26sid%3Dliteratum%253Aachs%26aulast%3DRaghuraman%26aufirst%3DA.%26aulast%3DMosier%26aufirst%3DP.%2BD.%26aulast%3DDesai%26aufirst%3DU.%2BR.%26atitle%3DFinding%2520a%2520needle%2520in%2520a%2520haystack%253A%2520development%2520of%2520a%2520combinatorial%2520virtual%2520screening%2520approach%2520for%2520identifying%2520high%2520specificity%2520heparin%252Fheparan%2520sulfate%2520sequence%2528s%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D3553%26epage%3D62" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Chowdhury, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yeoh, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tian, Y. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hillringhaus, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bagg, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rose, N. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leung, I. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, X. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woon, E. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDonough, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">King, O. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clifton, I. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klose, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Claridge, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ratcliffe, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawamura, A.</span><span> </span><span class="NLM_article-title">The oncometabolite 2-hydroxyglutarate inhibits histone lysine demethylases</span> <span class="citation_source-journal">EMBO Rep.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">463</span><span class="NLM_x">–</span> <span class="NLM_lpage">9</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=463-9&author=R.+Chowdhuryauthor=K.+K.+Yeohauthor=Y.+M.+Tianauthor=L.+Hillringhausauthor=E.+A.+Baggauthor=N.+R.+Roseauthor=I.+K.+Leungauthor=X.+S.+Liauthor=E.+C.+Woonauthor=M.+Yangauthor=M.+A.+McDonoughauthor=O.+N.+Kingauthor=I.+J.+Cliftonauthor=R.+J.+Kloseauthor=T.+D.+Claridgeauthor=P.+J.+Ratcliffeauthor=C.+J.+Schofieldauthor=A.+Kawamura&title=The+oncometabolite+2-hydroxyglutarate+inhibits+histone+lysine+demethylases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DChowdhury%26aufirst%3DR.%26aulast%3DYeoh%26aufirst%3DK.%2BK.%26aulast%3DTian%26aufirst%3DY.%2BM.%26aulast%3DHillringhaus%26aufirst%3DL.%26aulast%3DBagg%26aufirst%3DE.%2BA.%26aulast%3DRose%26aufirst%3DN.%2BR.%26aulast%3DLeung%26aufirst%3DI.%2BK.%26aulast%3DLi%26aufirst%3DX.%2BS.%26aulast%3DWoon%26aufirst%3DE.%2BC.%26aulast%3DYang%26aufirst%3DM.%26aulast%3DMcDonough%26aufirst%3DM.%2BA.%26aulast%3DKing%26aufirst%3DO.%2BN.%26aulast%3DClifton%26aufirst%3DI.%2BJ.%26aulast%3DKlose%26aufirst%3DR.%2BJ.%26aulast%3DClaridge%26aufirst%3DT.%2BD.%26aulast%3DRatcliffe%26aufirst%3DP.%2BJ.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26aulast%3DKawamura%26aufirst%3DA.%26atitle%3DThe%2520oncometabolite%25202-hydroxyglutarate%2520inhibits%2520histone%2520lysine%2520demethylases%26jtitle%3DEMBO%2520Rep.%26date%3D2011%26volume%3D12%26spage%3D463%26epage%3D9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Hsu, K. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, J. M.</span><span> </span><span class="NLM_article-title">iGEMDOCK: a graphical environment of enhancing GEMDOCK using pharmacological interactions and post-screening analysis</span> <span class="citation_source-journal">BMC Bioinformatics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">S33</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=S33&author=K.+C.+Hsuauthor=Y.+F.+Chenauthor=S.+R.+Linauthor=J.+M.+Yang&title=iGEMDOCK%3A+a+graphical+environment+of+enhancing+GEMDOCK+using+pharmacological+interactions+and+post-screening+analysis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22a&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHsu%26aufirst%3DK.%2BC.%26aulast%3DChen%26aufirst%3DY.%2BF.%26aulast%3DLin%26aufirst%3DS.%2BR.%26aulast%3DYang%26aufirst%3DJ.%2BM.%26atitle%3DiGEMDOCK%253A%2520a%2520graphical%2520environment%2520of%2520enhancing%2520GEMDOCK%2520using%2520pharmacological%2520interactions%2520and%2520post-screening%2520analysis%26jtitle%3DBMC%2520Bioinformatics%26date%3D2011%26volume%3D12%26spage%3DS33" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit22b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Yang, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, C. C.</span><span> </span><span class="NLM_article-title">GEMDOCK: a generic evolutionary method for molecular docking</span> <span class="citation_source-journal">Proteins: Struct., Funct., Bioinf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">288</span><span class="NLM_x">–</span> <span class="NLM_lpage">304</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2004&pages=288-304&author=J.+M.+Yangauthor=C.+C.+Chen&title=GEMDOCK%3A+a+generic+evolutionary+method+for+molecular+docking"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22b&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DJ.%2BM.%26aulast%3DChen%26aufirst%3DC.%2BC.%26atitle%3DGEMDOCK%253A%2520a%2520generic%2520evolutionary%2520method%2520for%2520molecular%2520docking%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Bioinf.%26date%3D2004%26volume%3D55%26spage%3D288%26epage%3D304" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Gregory, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, R. T.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mohler, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">French, F. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, E. M.</span><span> </span><span class="NLM_article-title">Androgen receptor stabilization in recurrent prostate cancer is associated with hypersensitivity to low androgen</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">61</span><span class="NLM_x">, </span> <span class="NLM_fpage">2892</span><span class="NLM_x">–</span> <span class="NLM_lpage">8</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2001&pages=2892-8&author=C.+W.+Gregoryauthor=R.+T.+Johnsonauthor=J.+L.+Mohlerauthor=F.+S.+Frenchauthor=E.+M.+Wilson&title=Androgen+receptor+stabilization+in+recurrent+prostate+cancer+is+associated+with+hypersensitivity+to+low+androgen"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGregory%26aufirst%3DC.%2BW.%26aulast%3DJohnson%26aufirst%3DR.%2BT.%26aulast%3DMohler%26aufirst%3DJ.%2BL.%26aulast%3DFrench%26aufirst%3DF.%2BS.%26aulast%3DWilson%26aufirst%3DE.%2BM.%26atitle%3DAndrogen%2520receptor%2520stabilization%2520in%2520recurrent%2520prostate%2520cancer%2520is%2520associated%2520with%2520hypersensitivity%2520to%2520low%2520androgen%26jtitle%3DCancer%2520Res.%26date%3D2001%26volume%3D61%26spage%3D2892%26epage%3D8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Thalmann, G. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anezinis, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhau, H. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, E. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hopwood, V. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pathak, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">von Eschenbach, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, L. W.</span><span> </span><span class="NLM_article-title">Androgen-independent cancer progression and bone metastasis in the LNCaP model of human prostate cancer</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">2577</span><span class="NLM_x">–</span> <span class="NLM_lpage">81</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=1994&pages=2577-81&author=G.+N.+Thalmannauthor=P.+E.+Anezinisauthor=S.+M.+Changauthor=H.+E.+Zhauauthor=E.+E.+Kimauthor=V.+L.+Hopwoodauthor=S.+Pathakauthor=A.+C.+von+Eschenbachauthor=L.+W.+Chung&title=Androgen-independent+cancer+progression+and+bone+metastasis+in+the+LNCaP+model+of+human+prostate+cancer"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DThalmann%26aufirst%3DG.%2BN.%26aulast%3DAnezinis%26aufirst%3DP.%2BE.%26aulast%3DChang%26aufirst%3DS.%2BM.%26aulast%3DZhau%26aufirst%3DH.%2BE.%26aulast%3DKim%26aufirst%3DE.%2BE.%26aulast%3DHopwood%26aufirst%3DV.%2BL.%26aulast%3DPathak%26aufirst%3DS.%26aulast%3Dvon%2BEschenbach%26aufirst%3DA.%2BC.%26aulast%3DChung%26aufirst%3DL.%2BW.%26atitle%3DAndrogen-independent%2520cancer%2520progression%2520and%2520bone%2520metastasis%2520in%2520the%2520LNCaP%2520model%2520of%2520human%2520prostate%2520cancer%26jtitle%3DCancer%2520Res.%26date%3D1994%26volume%3D54%26spage%3D2577%26epage%3D81" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit29a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Chen, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whetstine, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hong, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davrazou, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kutateladze, T. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simpson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mao, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pan, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dai, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hagman, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hansen, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, G.</span><span> </span><span class="NLM_article-title">Structural insights into histone demethylation by JMJD2 family members</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">125</span><span class="NLM_x">, </span> <span class="NLM_fpage">691</span><span class="NLM_x">–</span> <span class="NLM_lpage">702</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2006&pages=691-702&author=Z.+Chenauthor=J.+Zangauthor=J.+Whetstineauthor=X.+Hongauthor=F.+Davrazouauthor=T.+G.+Kutateladzeauthor=M.+Simpsonauthor=Q.+Maoauthor=C.+H.+Panauthor=S.+Daiauthor=J.+Hagmanauthor=K.+Hansenauthor=Y.+Shiauthor=G.+Zhang&title=Structural+insights+into+histone+demethylation+by+JMJD2+family+members"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29a&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DZang%26aufirst%3DJ.%26aulast%3DWhetstine%26aufirst%3DJ.%26aulast%3DHong%26aufirst%3DX.%26aulast%3DDavrazou%26aufirst%3DF.%26aulast%3DKutateladze%26aufirst%3DT.%2BG.%26aulast%3DSimpson%26aufirst%3DM.%26aulast%3DMao%26aufirst%3DQ.%26aulast%3DPan%26aufirst%3DC.%2BH.%26aulast%3DDai%26aufirst%3DS.%26aulast%3DHagman%26aufirst%3DJ.%26aulast%3DHansen%26aufirst%3DK.%26aulast%3DShi%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DG.%26atitle%3DStructural%2520insights%2520into%2520histone%2520demethylation%2520by%2520JMJD2%2520family%2520members%26jtitle%3DCell%26date%3D2006%26volume%3D125%26spage%3D691%26epage%3D702" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit29b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Hillringhaus, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yue, W. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rose, N. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ng, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gileadi, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loenarz, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bello, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bray, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oppermann, U.</span><span> </span><span class="NLM_article-title">Structural and evolutionary basis for the dual substrate selectivity of human KDM4 histone demethylase family</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">286</span><span class="NLM_x">, </span> <span class="NLM_fpage">41616</span><span class="NLM_x">–</span> <span class="NLM_lpage">25</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=286&publication_year=2011&pages=41616-25&author=L.+Hillringhausauthor=W.+W.+Yueauthor=N.+R.+Roseauthor=S.+S.+Ngauthor=C.+Gileadiauthor=C.+Loenarzauthor=S.+H.+Belloauthor=J.+E.+Brayauthor=C.+J.+Schofieldauthor=U.+Oppermann&title=Structural+and+evolutionary+basis+for+the+dual+substrate+selectivity+of+human+KDM4+histone+demethylase+family"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29b&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHillringhaus%26aufirst%3DL.%26aulast%3DYue%26aufirst%3DW.%2BW.%26aulast%3DRose%26aufirst%3DN.%2BR.%26aulast%3DNg%26aufirst%3DS.%2BS.%26aulast%3DGileadi%26aufirst%3DC.%26aulast%3DLoenarz%26aufirst%3DC.%26aulast%3DBello%26aufirst%3DS.%2BH.%26aulast%3DBray%26aufirst%3DJ.%2BE.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26aulast%3DOppermann%26aufirst%3DU.%26atitle%3DStructural%2520and%2520evolutionary%2520basis%2520for%2520the%2520dual%2520substrate%2520selectivity%2520of%2520human%2520KDM4%2520histone%2520demethylase%2520family%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2011%26volume%3D286%26spage%3D41616%26epage%3D25" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Whetstine, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nottke, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lan, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huarte, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smolikov, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spooner, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Colaiacovo, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Y.</span><span> </span><span class="NLM_article-title">Reversal of histone lysine trimethylation by the JMJD2 family of histone demethylases</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">125</span><span class="NLM_x">, </span> <span class="NLM_fpage">467</span><span class="NLM_x">–</span> <span class="NLM_lpage">81</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2006&pages=467-81&author=J.+R.+Whetstineauthor=A.+Nottkeauthor=F.+Lanauthor=M.+Huarteauthor=S.+Smolikovauthor=Z.+Chenauthor=E.+Spoonerauthor=E.+Liauthor=G.+Zhangauthor=M.+Colaiacovoauthor=Y.+Shi&title=Reversal+of+histone+lysine+trimethylation+by+the+JMJD2+family+of+histone+demethylases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWhetstine%26aufirst%3DJ.%2BR.%26aulast%3DNottke%26aufirst%3DA.%26aulast%3DLan%26aufirst%3DF.%26aulast%3DHuarte%26aufirst%3DM.%26aulast%3DSmolikov%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DSpooner%26aufirst%3DE.%26aulast%3DLi%26aufirst%3DE.%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DColaiacovo%26aufirst%3DM.%26aulast%3DShi%26aufirst%3DY.%26atitle%3DReversal%2520of%2520histone%2520lysine%2520trimethylation%2520by%2520the%2520JMJD2%2520family%2520of%2520histone%2520demethylases%26jtitle%3DCell%26date%3D2006%26volume%3D125%26spage%3D467%26epage%3D81" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Couture, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Collazo, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ortiz-Tello, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brunzelle, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trievel, R. C.</span><span> </span><span class="NLM_article-title">Specificity and mechanism of JMJD2A, a trimethyllysine-specific histone demethylase</span> <span class="citation_source-journal">Nat. Struct. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">689</span><span class="NLM_x">–</span> <span class="NLM_lpage">95</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Fjm500249n&amp;key=10.1038%2Fnsmb1273" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Fjm500249n&amp;key=17589523" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Fjm500249n&amp;key=1%3ACAS%3A528%3ADC%252BD2sXosVyktLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2007&pages=689-95&author=J.+F.+Coutureauthor=E.+Collazoauthor=P.+A.+Ortiz-Telloauthor=J.+S.+Brunzelleauthor=R.+C.+Trievel&title=Specificity+and+mechanism+of+JMJD2A%2C+a+trimethyllysine-specific+histone+demethylase"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Specificity and mechanism of JMJD2A, a trimethyllysine-specific histone demethylase</span></div><div class="casAuthors">Couture, Jean-Francois; Collazo, Evys; Ortiz-Tello, Patricia A.; Brunzelle, Joseph S.; Trievel, Raymond C.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">689-695</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">JMJD2A is a JmjC histone demethylase (HDM) that catalyzes the demethylation of di- and trimethylated Lys9 and Lys36 in histone H3 (H3K9me2/3 and H3K36me2/3).  Here we present the crystal structures of the JMJD2A catalytic domain in complex with H3K9me3, H3K36me2 and H3K36me3 peptides.  The structures reveal that histone substrates are recognized through a network of backbone hydrogen bonds and hydrophobic interactions that deposit the trimethyllysine into the active site.  The trimethylated ε-ammonium cation is coordinated within a methylammonium-binding pocket through carbon-oxygen (CH···O) hydrogen bonds that position one of the ζ-Me groups adjacent to the Fe(II) center for hydroxylation and demethylation.  Mutations of the residues comprising this pocket abrogate demethylation by JMJD2A, with the exception of an S288A substitution, which augments activity, particularly toward H3K9me2.  We propose that this residue modulates the methylation-state specificities of JMJD2 enzymes and other trimethyllysine-specific JmjC HDMs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZ8SMMLvfZErVg90H21EOLACvtfcHk0lj0WSsFxWdyrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXosVyktLY%253D&md5=86d9464f761cbd77c22abc1b3a35478c</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1038%2Fnsmb1273&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb1273%26sid%3Dliteratum%253Aachs%26aulast%3DCouture%26aufirst%3DJ.%2BF.%26aulast%3DCollazo%26aufirst%3DE.%26aulast%3DOrtiz-Tello%26aufirst%3DP.%2BA.%26aulast%3DBrunzelle%26aufirst%3DJ.%2BS.%26aulast%3DTrievel%26aufirst%3DR.%2BC.%26atitle%3DSpecificity%2520and%2520mechanism%2520of%2520JMJD2A%252C%2520a%2520trimethyllysine-specific%2520histone%2520demethylase%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2007%26volume%3D14%26spage%3D689%26epage%3D95" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Winters, D. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cederbaum, A. I.</span><span> </span><span class="NLM_article-title">Oxidation of glycerol to formaldehyde by rat liver microsomes. Effects of cytochrome P-450 inducing agents</span> <span class="citation_source-journal">Biochem. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1990</span><span class="NLM_x">, </span> <span class="NLM_volume">39</span><span class="NLM_x">, </span> <span class="NLM_fpage">697</span><span class="NLM_x">–</span> <span class="NLM_lpage">705</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1990&pages=697-705&author=D.+K.+Wintersauthor=A.+I.+Cederbaum&title=Oxidation+of+glycerol+to+formaldehyde+by+rat+liver+microsomes.+Effects+of+cytochrome+P-450+inducing+agents"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWinters%26aufirst%3DD.%2BK.%26aulast%3DCederbaum%26aufirst%3DA.%2BI.%26atitle%3DOxidation%2520of%2520glycerol%2520to%2520formaldehyde%2520by%2520rat%2520liver%2520microsomes.%2520Effects%2520of%2520cytochrome%2520P-450%2520inducing%2520agents%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D1990%26volume%3D39%26spage%3D697%26epage%3D705" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Rose, N. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ng, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mecinovic, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lienard, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bello, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDonough, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oppermann, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span> </span><span class="NLM_article-title">Inhibitor scaffolds for 2-oxoglutarate-dependent histone lysine demethylases</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">7053</span><span class="NLM_x">–</span> <span class="NLM_lpage">6</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm800936s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Fjm500249n&amp;key=1%3ACAS%3A528%3ADC%252BD1cXht1Ors7rI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=7053-6&author=N.+R.+Roseauthor=S.+S.+Ngauthor=J.+Mecinovicauthor=B.+M.+Lienardauthor=S.+H.+Belloauthor=Z.+Sunauthor=M.+A.+McDonoughauthor=U.+Oppermannauthor=C.+J.+Schofield&title=Inhibitor+scaffolds+for+2-oxoglutarate-dependent+histone+lysine+demethylases"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitor Scaffolds for 2-Oxoglutarate-Dependent Histone Lysine Demethylases</span></div><div class="casAuthors">Rose, Nathan R.; Ng, Stanley S.; Mecinovic, Jasmin; Lienard, Benoit M. R.; Bello, Simon H.; Sun, Zhe; McDonough, Michael A.; Oppermann, Udo; Schofield, Christopher J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">7053-7056</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The dynamic methylation of histone lysyl residues plays an important role in biol. by regulating transcription, maintaining genomic integrity, and by contributing to epigenetic effects.  Here we describe a variety of inhibitor scaffolds that inhibit the human 2-oxoglutarate-dependent JMJD2 subfamily of histone demethylases.  Combined with structural data, these chem. starting points will be useful to generate small-mol. probes to analyze the physiol. roles of these enzymes in epigenetic signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr90HQ8nfBLFrVg90H21EOLACvtfcHk0liOnwOnWASZWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXht1Ors7rI&md5=155c830b0544a6c1d5e6eb81d14389de</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fjm800936s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm800936s%26sid%3Dliteratum%253Aachs%26aulast%3DRose%26aufirst%3DN.%2BR.%26aulast%3DNg%26aufirst%3DS.%2BS.%26aulast%3DMecinovic%26aufirst%3DJ.%26aulast%3DLienard%26aufirst%3DB.%2BM.%26aulast%3DBello%26aufirst%3DS.%2BH.%26aulast%3DSun%26aufirst%3DZ.%26aulast%3DMcDonough%26aufirst%3DM.%2BA.%26aulast%3DOppermann%26aufirst%3DU.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26atitle%3DInhibitor%2520scaffolds%2520for%25202-oxoglutarate-dependent%2520histone%2520lysine%2520demethylases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D7053%26epage%3D6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit34a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Grasso, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, Y.-M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robinson, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dhanasekaran, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khan, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quist, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jing, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lonigro, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brenner, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Asangani, I. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ateeq, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chun, S. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siddiqui, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sam, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anstett, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mehra, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prensner, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palanisamy, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ryslik, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vandin, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raphael, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kunju, L. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rhodes, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pienta, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chinnaiyan, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tomlins, S. A.</span><span> </span><span class="NLM_article-title">The mutational landscape of lethal castration-resistant prostate cancer</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">487</span><span class="NLM_x">, </span> <span class="NLM_fpage">239</span><span class="NLM_x">–</span> <span class="NLM_lpage">43</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34a&amp;dbid=16&amp;doi=10.1021%2Fjm500249n&amp;key=10.1038%2Fnature11125" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34a&amp;dbid=8&amp;doi=10.1021%2Fjm500249n&amp;key=22722839" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34a&amp;dbid=32&amp;doi=10.1021%2Fjm500249n&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVWisLfP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=487&publication_year=2012&pages=239-43&author=C.+S.+Grassoauthor=Y.-M.+Wuauthor=D.+R.+Robinsonauthor=X.+Caoauthor=S.+M.+Dhanasekaranauthor=A.+P.+Khanauthor=M.+J.+Quistauthor=X.+Jingauthor=R.+J.+Lonigroauthor=J.+C.+Brennerauthor=I.+A.+Asanganiauthor=B.+Ateeqauthor=S.+Y.+Chunauthor=J.+Siddiquiauthor=L.+Samauthor=M.+Anstettauthor=R.+Mehraauthor=J.+R.+Prensnerauthor=N.+Palanisamyauthor=G.+A.+Ryslikauthor=F.+Vandinauthor=B.+J.+Raphaelauthor=L.+P.+Kunjuauthor=D.+R.+Rhodesauthor=K.+J.+Pientaauthor=A.+M.+Chinnaiyanauthor=S.+A.+Tomlins&title=The+mutational+landscape+of+lethal+castration-resistant+prostate+cancer"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34aR"><div class="casContent"><span class="casTitleNuber">30a</span><div class="casTitle"><span class="NLM_cas:atitle">The mutational landscape of lethal castration-resistant prostate cancer</span></div><div class="casAuthors">Grasso, Catherine S.; Wu, Yi-Mi; Robinson, Dan R.; Cao, Xuhong; Dhanasekaran, Saravana M.; Khan, Amjad P.; Quist, Michael J.; Jing, Xiaojun; Lonigro, Robert J.; Brenner, J. Chad; Asangani, Irfan A.; Ateeq, Bushra; Chun, Sang Y.; Siddiqui, Javed; Sam, Lee; Anstett, Matt; Mehra, Rohit; Prensner, John R.; Palanisamy, Nallasivam; Ryslik, Gregory A.; Vandin, Fabio; Raphael, Benjamin J.; Kunju, Lakshmi P.; Rhodes, Daniel R.; Pienta, Kenneth J.; Chinnaiyan, Arul M.; Tomlins, Scott A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">487</span>
        (<span class="NLM_cas:issue">7406</span>),
    <span class="NLM_cas:pages">239-243</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Characterization of the prostate cancer transcriptome and genome has identified chromosomal rearrangements and copy no. gains and losses, including ETS gene family fusions, PTEN loss and androgen receptor (AR) amplification, which drive prostate cancer development and progression to lethal, metastatic castration-resistant prostate cancer (CRPC).  However, less is known about the role of mutations.  Here we sequenced the exomes of 50 lethal, heavily pre-treated metastatic CRPCs obtained at rapid autopsy (including three different foci from the same patient) and 11 treatment-naive, high-grade localized prostate cancers.  We identified low overall mutation rates even in heavily treated CRPCs (2.00 per megabase) and confirmed the monoclonal origin of lethal CRPC.  Integrating exome copy no. anal. identified disruptions of CHD1 that define a subtype of ETS gene family fusion-neg. prostate cancer.  Similarly, we demonstrate that ETS2, which is deleted in approx. one-third of CRPCs (commonly through TMPRSS2:ERG fusions), is also deregulated through mutation.  Furthermore, we identified recurrent mutations in multiple chromatin- and histone-modifying genes, including MLL2 (mutated in 8.6% of prostate cancers), and demonstrate interaction of the MLL complex with the AR, which is required for AR-mediated signaling.  We also identified novel recurrent mutations in the AR collaborating factor FOXA1, which is mutated in 5 of 147 (3.4%) prostate cancers (both untreated localized prostate cancer and CRPC), and showed that mutated FOXA1 represses androgen signaling and increases tumor growth.  Proteins that phys. interact with the AR, such as the ERG gene fusion product, FOXA1, MLL2, UTX (also known as KDM6A) and ASXL1 were found to be mutated in CRPC.  In summary, we describe the mutational landscape of a heavily treated metastatic cancer, identify novel mechanisms of AR signaling deregulated in prostate cancer, and prioritize candidates for future study.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgzVYOnnaqVLVg90H21EOLACvtfcHk0liOnwOnWASZWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVWisLfP&md5=15b1780d9cd2189325baf4e4fec84687</span></div><a href="/servlet/linkout?suffix=cit34a&amp;dbid=16384&amp;doi=10.1038%2Fnature11125&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11125%26sid%3Dliteratum%253Aachs%26aulast%3DGrasso%26aufirst%3DC.%2BS.%26aulast%3DWu%26aufirst%3DY.-M.%26aulast%3DRobinson%26aufirst%3DD.%2BR.%26aulast%3DCao%26aufirst%3DX.%26aulast%3DDhanasekaran%26aufirst%3DS.%2BM.%26aulast%3DKhan%26aufirst%3DA.%2BP.%26aulast%3DQuist%26aufirst%3DM.%2BJ.%26aulast%3DJing%26aufirst%3DX.%26aulast%3DLonigro%26aufirst%3DR.%2BJ.%26aulast%3DBrenner%26aufirst%3DJ.%2BC.%26aulast%3DAsangani%26aufirst%3DI.%2BA.%26aulast%3DAteeq%26aufirst%3DB.%26aulast%3DChun%26aufirst%3DS.%2BY.%26aulast%3DSiddiqui%26aufirst%3DJ.%26aulast%3DSam%26aufirst%3DL.%26aulast%3DAnstett%26aufirst%3DM.%26aulast%3DMehra%26aufirst%3DR.%26aulast%3DPrensner%26aufirst%3DJ.%2BR.%26aulast%3DPalanisamy%26aufirst%3DN.%26aulast%3DRyslik%26aufirst%3DG.%2BA.%26aulast%3DVandin%26aufirst%3DF.%26aulast%3DRaphael%26aufirst%3DB.%2BJ.%26aulast%3DKunju%26aufirst%3DL.%2BP.%26aulast%3DRhodes%26aufirst%3DD.%2BR.%26aulast%3DPienta%26aufirst%3DK.%2BJ.%26aulast%3DChinnaiyan%26aufirst%3DA.%2BM.%26aulast%3DTomlins%26aufirst%3DS.%2BA.%26atitle%3DThe%2520mutational%2520landscape%2520of%2520lethal%2520castration-resistant%2520prostate%2520cancer%26jtitle%3DNature%26date%3D2012%26volume%3D487%26spage%3D239%26epage%3D43" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit34b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Lapointe, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Higgins, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van de Rijn, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bair, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Montgomery, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferrari, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Egevad, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rayford, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bergerheim, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ekman, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DeMarzo, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tibshirani, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Botstein, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brooks, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollack, J. R.</span><span> </span><span class="NLM_article-title">Gene expression profiling identifies clinically relevant subtypes of prostate cancer</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">101</span><span class="NLM_x">, </span> <span class="NLM_fpage">811</span><span class="NLM_x">–</span> <span class="NLM_lpage">6</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2004&pages=811-6&author=J.+Lapointeauthor=C.+Liauthor=J.+P.+Higginsauthor=M.+van+de+Rijnauthor=E.+Bairauthor=K.+Montgomeryauthor=M.+Ferrariauthor=L.+Egevadauthor=W.+Rayfordauthor=U.+Bergerheimauthor=P.+Ekmanauthor=A.+M.+DeMarzoauthor=R.+Tibshiraniauthor=D.+Botsteinauthor=P.+O.+Brownauthor=J.+D.+Brooksauthor=J.+R.+Pollack&title=Gene+expression+profiling+identifies+clinically+relevant+subtypes+of+prostate+cancer"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34b&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLapointe%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DHiggins%26aufirst%3DJ.%2BP.%26aulast%3Dvan%2Bde%2BRijn%26aufirst%3DM.%26aulast%3DBair%26aufirst%3DE.%26aulast%3DMontgomery%26aufirst%3DK.%26aulast%3DFerrari%26aufirst%3DM.%26aulast%3DEgevad%26aufirst%3DL.%26aulast%3DRayford%26aufirst%3DW.%26aulast%3DBergerheim%26aufirst%3DU.%26aulast%3DEkman%26aufirst%3DP.%26aulast%3DDeMarzo%26aufirst%3DA.%2BM.%26aulast%3DTibshirani%26aufirst%3DR.%26aulast%3DBotstein%26aufirst%3DD.%26aulast%3DBrown%26aufirst%3DP.%2BO.%26aulast%3DBrooks%26aufirst%3DJ.%2BD.%26aulast%3DPollack%26aufirst%3DJ.%2BR.%26atitle%3DGene%2520expression%2520profiling%2520identifies%2520clinically%2520relevant%2520subtypes%2520of%2520prostate%2520cancer%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2004%26volume%3D101%26spage%3D811%26epage%3D6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Hsu, K. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, L. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, J. M.</span><span> </span><span class="NLM_article-title">Core site-moiety maps reveal inhibitors and binding mechanisms of orthologous proteins by screening compound libraries</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">e32142</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=e32142&author=K.+C.+Hsuauthor=W.+C.+Chengauthor=Y.+F.+Chenauthor=H.+J.+Wangauthor=L.+T.+Liauthor=W.+C.+Wangauthor=J.+M.+Yang&title=Core+site-moiety+maps+reveal+inhibitors+and+binding+mechanisms+of+orthologous+proteins+by+screening+compound+libraries"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHsu%26aufirst%3DK.%2BC.%26aulast%3DCheng%26aufirst%3DW.%2BC.%26aulast%3DChen%26aufirst%3DY.%2BF.%26aulast%3DWang%26aufirst%3DH.%2BJ.%26aulast%3DLi%26aufirst%3DL.%2BT.%26aulast%3DWang%26aufirst%3DW.%2BC.%26aulast%3DYang%26aufirst%3DJ.%2BM.%26atitle%3DCore%2520site-moiety%2520maps%2520reveal%2520inhibitors%2520and%2520binding%2520mechanisms%2520of%2520orthologous%2520proteins%2520by%2520screening%2520compound%2520libraries%26jtitle%3DPLoS%2520One%26date%3D2012%26volume%3D7%26spage%3De32142" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Black, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Manning, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Rechem, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ladd, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cho, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pineda, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murphy, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Daniels, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Montagna, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lewis, P. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glass, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allis, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dyson, N. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Getz, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whetstine, J. R.</span><span> </span><span class="NLM_article-title">KDM4A lysine demethylase induces site-specific copy gain and rereplication of regions amplified in tumors</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">154</span><span class="NLM_x">, </span> <span class="NLM_fpage">541</span><span class="NLM_x">–</span> <span class="NLM_lpage">55</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=154&publication_year=2013&pages=541-55&author=J.+C.+Blackauthor=A.+L.+Manningauthor=C.+Van+Rechemauthor=J.+Kimauthor=B.+Laddauthor=J.+Choauthor=C.+M.+Pinedaauthor=N.+Murphyauthor=D.+L.+Danielsauthor=C.+Montagnaauthor=P.+W.+Lewisauthor=K.+Glassauthor=C.+D.+Allisauthor=N.+J.+Dysonauthor=G.+Getzauthor=J.+R.+Whetstine&title=KDM4A+lysine+demethylase+induces+site-specific+copy+gain+and+rereplication+of+regions+amplified+in+tumors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBlack%26aufirst%3DJ.%2BC.%26aulast%3DManning%26aufirst%3DA.%2BL.%26aulast%3DVan%2BRechem%26aufirst%3DC.%26aulast%3DKim%26aufirst%3DJ.%26aulast%3DLadd%26aufirst%3DB.%26aulast%3DCho%26aufirst%3DJ.%26aulast%3DPineda%26aufirst%3DC.%2BM.%26aulast%3DMurphy%26aufirst%3DN.%26aulast%3DDaniels%26aufirst%3DD.%2BL.%26aulast%3DMontagna%26aufirst%3DC.%26aulast%3DLewis%26aufirst%3DP.%2BW.%26aulast%3DGlass%26aufirst%3DK.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26aulast%3DDyson%26aufirst%3DN.%2BJ.%26aulast%3DGetz%26aufirst%3DG.%26aulast%3DWhetstine%26aufirst%3DJ.%2BR.%26atitle%3DKDM4A%2520lysine%2520demethylase%2520induces%2520site-specific%2520copy%2520gain%2520and%2520rereplication%2520of%2520regions%2520amplified%2520in%2520tumors%26jtitle%3DCell%26date%3D2013%26volume%3D154%26spage%3D541%26epage%3D55" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Rose, N. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDonough, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">King, O. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawamura, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span> </span><span class="NLM_article-title">Inhibition of 2-oxoglutarate dependent oxygenases</span> <span class="citation_source-journal">Chem. Soc. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">4364</span><span class="NLM_x">–</span> <span class="NLM_lpage">97</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Fjm500249n&amp;key=10.1039%2Fc0cs00203h" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Fjm500249n&amp;key=21390379" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Fjm500249n&amp;key=1%3ACAS%3A528%3ADC%252BC3MXovVemur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2011&pages=4364-97&author=N.+R.+Roseauthor=M.+A.+McDonoughauthor=O.+N.+Kingauthor=A.+Kawamuraauthor=C.+J.+Schofield&title=Inhibition+of+2-oxoglutarate+dependent+oxygenases"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of 2-oxoglutarate dependent oxygenases</span></div><div class="casAuthors">Rose, Nathan R.; McDonough, Michael A.; King, Oliver N. F.; Kawamura, Akane; Schofield, Christopher J.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Society Reviews</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">4364-4397</span>CODEN:
                <span class="NLM_cas:coden">CSRVBR</span>;
        ISSN:<span class="NLM_cas:issn">0306-0012</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  2-Oxoglutarate (2OG) dependent oxygenases are ubiquitous iron enzymes that couple substrate oxidn. to the conversion of 2OG to succinate and carbon dioxide.  In humans their roles include collagen biosynthesis, fatty acid metab., DNA repair, RNA and chromatin modifications, and hypoxic sensing.  Com. applications of 2OG oxygenase inhibitors began with plant growth retardants, and now extend to a clin. used pharmaceutical compd. for cardioprotection.  Several 2OG oxygenases are now being targeted for therapeutic intervention for diseases including anemia, inflammation and cancer.  In this crit. review, we describe studies on the inhibition of 2OG oxygenases, focusing on small mols., and discuss the potential of 2OG oxygenases as therapeutic targets (295 refs.).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqp4Z13BJAo17Vg90H21EOLACvtfcHk0lhoA5BeqaV-dA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXovVemur8%253D&md5=f7a87e0bc9392d9a29f4ffb7d6f92b44</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1039%2Fc0cs00203h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc0cs00203h%26sid%3Dliteratum%253Aachs%26aulast%3DRose%26aufirst%3DN.%2BR.%26aulast%3DMcDonough%26aufirst%3DM.%2BA.%26aulast%3DKing%26aufirst%3DO.%2BN.%26aulast%3DKawamura%26aufirst%3DA.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26atitle%3DInhibition%2520of%25202-oxoglutarate%2520dependent%2520oxygenases%26jtitle%3DChem.%2520Soc.%2520Rev.%26date%3D2011%26volume%3D40%26spage%3D4364%26epage%3D97" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varghese, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dellinger, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kumar, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Best, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruiz, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bruick, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pena-Llopis, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Babinski, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frantz, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brekken, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quinn, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeonov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Easmon, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martinez, E. D.</span><span> </span><span class="NLM_article-title">A small molecule modulates Jumonji histone demethylase activity and selectively inhibits cancer growth</span> <span class="citation_source-journal">Nat. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">2035</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Fjm500249n&amp;key=10.1038%2Fncomms3035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Fjm500249n&amp;key=23792809" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Fjm500249n&amp;key=1%3ACAS%3A280%3ADC%252BC3sjksV2iug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=2035&author=L.+Wangauthor=J.+Changauthor=D.+Vargheseauthor=M.+Dellingerauthor=S.+Kumarauthor=A.+M.+Bestauthor=J.+Ruizauthor=R.+Bruickauthor=S.+Pena-Llopisauthor=J.+Xuauthor=D.+J.+Babinskiauthor=D.+E.+Frantzauthor=R.+A.+Brekkenauthor=A.+M.+Quinnauthor=A.+Simeonovauthor=J.+Easmonauthor=E.+D.+Martinez&title=A+small+molecule+modulates+Jumonji+histone+demethylase+activity+and+selectively+inhibits+cancer+growth"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">A small molecule modulates Jumonji histone demethylase activity and selectively inhibits cancer growth</span></div><div class="casAuthors">Wang Lei; Chang Jianjun; Varghese Diana; Dellinger Michael; Kumar Subodh; Best Anne M; Ruiz Julio; Bruick Richard; Pena-Llopis Samuel; Xu Junjie; Babinski David J; Frantz Doug E; Brekken Rolf A; Quinn Amy M; Simeonov Anton; Easmon Johnny; Martinez Elisabeth D</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">2035</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The pharmacological inhibition of general transcriptional regulators has the potential to block growth through targeting multiple tumorigenic signalling pathways simultaneously.  Here, using an innovative cell-based screen, we identify a structurally unique small molecule (named JIB-04) that specifically inhibits the activity of the Jumonji family of histone demethylases in vitro, in cancer cells, and in tumours in vivo.  Unlike known inhibitors, JIB-04 is not a competitive inhibitor of α-ketoglutarate.  In cancer, but not in patient-matched normal cells, JIB-04 alters a subset of transcriptional pathways and blocks viability.  In mice, JIB-04 reduces tumour burden and prolongs survival.  Importantly, we find that patients with breast tumours that overexpress Jumonji demethylases have significantly lower survival.  Thus, JIB-04, a novel inhibitor of Jumonji demethylases in vitro and in vivo, constitutes a unique potential therapeutic and research tool against cancer, and validates the use of unbiased cellular screens to discover chemical modulators with disease relevance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ5QxHbn8WTZg20zIbnCCI3fW6udTcc2ebk3VJg5Uf5vrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3sjksV2iug%253D%253D&md5=37aa303d173d027fd711bc4ef7fd841b</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1038%2Fncomms3035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms3035%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.%26aulast%3DChang%26aufirst%3DJ.%26aulast%3DVarghese%26aufirst%3DD.%26aulast%3DDellinger%26aufirst%3DM.%26aulast%3DKumar%26aufirst%3DS.%26aulast%3DBest%26aufirst%3DA.%2BM.%26aulast%3DRuiz%26aufirst%3DJ.%26aulast%3DBruick%26aufirst%3DR.%26aulast%3DPena-Llopis%26aufirst%3DS.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DBabinski%26aufirst%3DD.%2BJ.%26aulast%3DFrantz%26aufirst%3DD.%2BE.%26aulast%3DBrekken%26aufirst%3DR.%2BA.%26aulast%3DQuinn%26aufirst%3DA.%2BM.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DEasmon%26aufirst%3DJ.%26aulast%3DMartinez%26aufirst%3DE.%2BD.%26atitle%3DA%2520small%2520molecule%2520modulates%2520Jumonji%2520histone%2520demethylase%2520activity%2520and%2520selectively%2520inhibits%2520cancer%2520growth%26jtitle%3DNat.%2520Commun.%26date%3D2013%26volume%3D4%26spage%3D2035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1a','cit1b','cit1c'],'ref2':['cit2a','cit2b','cit2c'],'ref3':['cit3a','cit3b'],'ref4':['cit4a','cit4b'],'ref5':['cit5a','cit5b','cit5c'],'ref6':['cit6a','cit6b','cit6c'],'ref7':['cit7a','cit7b'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13a','cit13b'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22a','cit22b'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29a','cit29b'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34a','cit34b'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 56 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Sankhya Bhattacharya, Pijus Kundu, J.S. Liu, Wen-Ching Wang, <span class="NLM_string-name hlFld-ContribAuthor">Fan-Gang Tseng</span>. </span><span class="cited-content_cbyCitation_article-title">Feedback-System-Control Integrated Microfluidic System for Fast Screening of Protein Crystallization Conditions. </span><span class="cited-content_cbyCitation_journal-name">Crystal Growth & Design</span><span> <strong>2020,</strong> <em>20 </em>
                                    (7)
                                     , 4325-4334. <a href="https://doi.org/10.1021/acs.cgd.0c00011" title="DOI URL">https://doi.org/10.1021/acs.cgd.0c00011</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.cgd.0c00011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.cgd.0c00011%26sid%3Dliteratum%253Aachs%26jtitle%3DCrystal%2520Growth%2520%2526%2520Design%26atitle%3DFeedback-System-Control%252BIntegrated%252BMicrofluidic%252BSystem%252Bfor%252BFast%252BScreening%252Bof%252BProtein%252BCrystallization%252BConditions%26aulast%3DBhattacharya%26aufirst%3DSankhya%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D05012020%26date%3D13052020%26date%3D01062020%26date%3D14052020%26volume%3D20%26issue%3D7%26spage%3D4325%26epage%3D4334" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">C. Esposito, L. Wiedmer, <span class="NLM_string-name hlFld-ContribAuthor">A. Caflisch</span>. </span><span class="cited-content_cbyCitation_article-title">In Silico Identification of JMJD3 Demethylase Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2018,</strong> <em>58 </em>
                                    (10)
                                     , 2151-2163. <a href="https://doi.org/10.1021/acs.jcim.8b00539" title="DOI URL">https://doi.org/10.1021/acs.jcim.8b00539</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.8b00539&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.8b00539%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DIn%252BSilico%252BIdentification%252Bof%252BJMJD3%252BDemethylase%252BInhibitors%26aulast%3DEsposito%26aufirst%3DC.%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D10082018%26date%3D03102018%26date%3D18092018%26volume%3D58%26issue%3D10%26spage%3D2151%26epage%3D2163" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">H. Ümit  Kaniskan</span>, <span class="hlFld-ContribAuthor ">Michael L.  Martini</span>, and <span class="hlFld-ContribAuthor ">Jian  Jin</span>  . </span><span class="cited-content_cbyCitation_article-title">Inhibitors of Protein Methyltransferases and Demethylases. </span><span class="cited-content_cbyCitation_journal-name">Chemical Reviews</span><span> <strong>2018,</strong> <em>118 </em>
                                    (3)
                                     , 989-1068. <a href="https://doi.org/10.1021/acs.chemrev.6b00801" title="DOI URL">https://doi.org/10.1021/acs.chemrev.6b00801</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.chemrev.6b00801&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.chemrev.6b00801%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Reviews%26atitle%3DInhibitors%252Bof%252BProtein%252BMethyltransferases%252Band%252BDemethylases%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259Cmit%26date%3D2018%26date%3D2017%26date%3D2016%26date%3D30112016%26date%3D24032017%26date%3D14022018%26volume%3D118%26issue%3D3%26spage%3D989%26epage%3D1068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Magdalena  Korczynska</span>, <span class="hlFld-ContribAuthor ">Daniel D.  Le</span>, <span class="hlFld-ContribAuthor ">Noah  Younger</span>, <span class="hlFld-ContribAuthor ">Elisabet  Gregori-Puigjané</span>, <span class="hlFld-ContribAuthor ">Anthony  Tumber</span>, <span class="hlFld-ContribAuthor ">Tobias  Krojer</span>, <span class="hlFld-ContribAuthor ">Srikannathasan  Velupillai</span>, <span class="hlFld-ContribAuthor ">Carina  Gileadi</span>, <span class="hlFld-ContribAuthor ">Radosław P.  Nowak</span>, <span class="hlFld-ContribAuthor ">Eriko  Iwasa</span>, <span class="hlFld-ContribAuthor ">Samuel B.  Pollock</span>, <span class="hlFld-ContribAuthor ">Idelisse  Ortiz Torres</span>, <span class="hlFld-ContribAuthor ">Udo  Oppermann</span>, <span class="hlFld-ContribAuthor ">Brian K.  Shoichet</span>, and <span class="hlFld-ContribAuthor ">Danica Galonić  Fujimori</span>  . </span><span class="cited-content_cbyCitation_article-title">Docking and Linking of Fragments To Discover Jumonji Histone Demethylase Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2016,</strong> <em>59 </em>
                                    (4)
                                     , 1580-1598. <a href="https://doi.org/10.1021/acs.jmedchem.5b01527" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.5b01527</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.5b01527&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.5b01527%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDocking%252Band%252BLinking%252Bof%252BFragments%252BTo%252BDiscover%252BJumonji%252BHistone%252BDemethylase%252BInhibitors%26aulast%3DKorczynska%26aufirst%3DMagdalena%26date%3D2016%26date%3D2015%26date%3D2015%26date%3D30092015%26date%3D23122015%26date%3D25022016%26volume%3D59%26issue%3D4%26spage%3D1580%26epage%3D1598" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Vassilios  Bavetsias</span>, <span class="hlFld-ContribAuthor ">Rachel M.  Lanigan</span>, <span class="hlFld-ContribAuthor ">Gian Filippo  Ruda</span>, <span class="hlFld-ContribAuthor ">Butrus  Atrash</span>, <span class="hlFld-ContribAuthor ">Mark G.  McLaughlin</span>, <span class="hlFld-ContribAuthor ">Anthony  Tumber</span>, <span class="hlFld-ContribAuthor ">N. Yi  Mok</span>, <span class="hlFld-ContribAuthor ">Yann-Vaï  Le Bihan</span>, <span class="hlFld-ContribAuthor ">Sally  Dempster</span>, <span class="hlFld-ContribAuthor ">Katherine J.  Boxall</span>, <span class="hlFld-ContribAuthor ">Fiona  Jeganathan</span>, <span class="hlFld-ContribAuthor ">Stephanie B.  Hatch</span>, <span class="hlFld-ContribAuthor ">Pavel  Savitsky</span>, <span class="hlFld-ContribAuthor ">Srikannathasan  Velupillai</span>, <span class="hlFld-ContribAuthor ">Tobias  Krojer</span>, <span class="hlFld-ContribAuthor ">Katherine S.  England</span>, <span class="hlFld-ContribAuthor ">Jimmy  Sejberg</span>, <span class="hlFld-ContribAuthor ">Ching  Thai</span>, <span class="hlFld-ContribAuthor ">Adam  Donovan</span>, <span class="hlFld-ContribAuthor ">Akos  Pal</span>, <span class="hlFld-ContribAuthor ">Giuseppe  Scozzafava</span>, <span class="hlFld-ContribAuthor ">James M.  Bennett</span>, <span class="hlFld-ContribAuthor ">Akane  Kawamura</span>, <span class="hlFld-ContribAuthor ">Catrine  Johansson</span>, <span class="hlFld-ContribAuthor ">Aleksandra  Szykowska</span>, <span class="hlFld-ContribAuthor ">Carina  Gileadi</span>, <span class="hlFld-ContribAuthor ">Nicola A.  Burgess-Brown</span>, <span class="hlFld-ContribAuthor ">Frank  von Delft</span>, <span class="hlFld-ContribAuthor ">Udo  Oppermann</span>, <span class="hlFld-ContribAuthor ">Zoe  Walters</span>, <span class="hlFld-ContribAuthor ">Janet  Shipley</span>, <span class="hlFld-ContribAuthor ">Florence I.  Raynaud</span>, <span class="hlFld-ContribAuthor ">Susan M.  Westaway</span>, <span class="hlFld-ContribAuthor ">Rab K.  Prinjha</span>, <span class="hlFld-ContribAuthor ">Oleg  Fedorov</span>, <span class="hlFld-ContribAuthor ">Rosemary  Burke</span>, <span class="hlFld-ContribAuthor ">Christopher J.  Schofield</span>, <span class="hlFld-ContribAuthor ">Isaac M.  Westwood</span>, <span class="hlFld-ContribAuthor ">Chas  Bountra</span>, <span class="hlFld-ContribAuthor ">Susanne  Müller</span>, <span class="hlFld-ContribAuthor ">Rob L. M.  van Montfort</span>, <span class="hlFld-ContribAuthor ">Paul E.  Brennan</span>, and <span class="hlFld-ContribAuthor ">Julian  Blagg</span>  . </span><span class="cited-content_cbyCitation_article-title">8-Substituted Pyrido[3,4-d]pyrimidin-4(3H)-one Derivatives As Potent, Cell Permeable, KDM4 (JMJD2) and KDM5 (JARID1) Histone Lysine Demethylase Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2016,</strong> <em>59 </em>
                                    (4)
                                     , 1388-1409. <a href="https://doi.org/10.1021/acs.jmedchem.5b01635" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.5b01635</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.5b01635&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.5b01635%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3D8-Substituted%252BPyrido%25255B3%25252C4-d%25255Dpyrimidin-4%2525283H%252529-one%252BDerivatives%252BAs%252BPotent%25252C%252BCell%252BPermeable%25252C%252BKDM4%252B%252528JMJD2%252529%252Band%252BKDM5%252B%252528JARID1%252529%252BHistone%252BLysine%252BDemethylase%252BInhibitors%26aulast%3DBavetsias%26aufirst%3DVassilios%26date%3D2016%26date%3D2016%26date%3D2015%26date%3D18102015%26date%3D07012016%26date%3D25022016%26volume%3D59%26issue%3D4%26spage%3D1388%26epage%3D1409" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tom E.  McAllister</span>, <span class="hlFld-ContribAuthor ">Katherine S.  England</span>, <span class="hlFld-ContribAuthor ">Richard J.  Hopkinson</span>, <span class="hlFld-ContribAuthor ">Paul E.  Brennan</span>, <span class="hlFld-ContribAuthor ">Akane  Kawamura</span>, and <span class="hlFld-ContribAuthor ">Christopher J.  Schofield</span>  . </span><span class="cited-content_cbyCitation_article-title">Recent Progress in Histone Demethylase Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2016,</strong> <em>59 </em>
                                    (4)
                                     , 1308-1329. <a href="https://doi.org/10.1021/acs.jmedchem.5b01758" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.5b01758</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.5b01758&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.5b01758%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DRecent%252BProgress%252Bin%252BHistone%252BDemethylase%252BInhibitors%26aulast%3DMcAllister%26aufirst%3DTom%2BE.%26date%3D2016%26date%3D2016%26date%3D2015%26date%3D10112015%26date%3D02022016%26date%3D25022016%26date%3D28122015%26volume%3D59%26issue%3D4%26spage%3D1308%26epage%3D1329" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Arvind Kumar  Yadav</span>, <span class="hlFld-ContribAuthor ">Tiratha Raj  Singh</span>. </span><span class="cited-content_cbyCitation_article-title">Novel inhibitors design through structural investigations and simulation studies for human PKMTs (SMYD2) involved in cancer. </span><span class="cited-content_cbyCitation_journal-name">Molecular Simulation</span><span> <strong>2021,</strong> <em>16 </em>, 1-10. <a href="https://doi.org/10.1080/08927022.2021.1957882" title="DOI URL">https://doi.org/10.1080/08927022.2021.1957882</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/08927022.2021.1957882&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F08927022.2021.1957882%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Simulation%26atitle%3DNovel%252Binhibitors%252Bdesign%252Bthrough%252Bstructural%252Binvestigations%252Band%252Bsimulation%252Bstudies%252Bfor%252Bhuman%252BPKMTs%252B%252528SMYD2%252529%252Binvolved%252Bin%252Bcancer%26aulast%3DYadav%26aufirst%3DArvind%2BKumar%26date%3D2021%26date%3D2021%26volume%3D16%26spage%3D1%26epage%3D10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Stephin  Baby</span>, <span class="hlFld-ContribAuthor ">Durgesh  Gurukkala Valapil</span>, <span class="hlFld-ContribAuthor ">Nagula  Shankaraiah</span>. </span><span class="cited-content_cbyCitation_article-title">Unravelling KDM4 histone demethylase inhibitors for cancer therapy. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today</span><span> <strong>2021,</strong> <em>21 </em><a href="https://doi.org/10.1016/j.drudis.2021.05.015" title="DOI URL">https://doi.org/10.1016/j.drudis.2021.05.015</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.drudis.2021.05.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.drudis.2021.05.015%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%26atitle%3DUnravelling%252BKDM4%252Bhistone%252Bdemethylase%252Binhibitors%252Bfor%252Bcancer%252Btherapy%26aulast%3DBaby%26aufirst%3DStephin%26date%3D2021%26volume%3D21" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shivendra  Singh</span>, <span class="hlFld-ContribAuthor ">Ahmed  Abu-Zaid</span>, <span class="hlFld-ContribAuthor ">Wenwei  Lin</span>, <span class="hlFld-ContribAuthor ">Jonathan  Low</span>, <span class="hlFld-ContribAuthor ">Alireza  Abdolvahabi</span>, <span class="hlFld-ContribAuthor ">Hongjian  Jin</span>, <span class="hlFld-ContribAuthor ">Qiong  Wu</span>, <span class="hlFld-ContribAuthor ">Bailey  Cooke</span>, <span class="hlFld-ContribAuthor ">Jie  Fang</span>, <span class="hlFld-ContribAuthor ">John  Bowling</span>, <span class="hlFld-ContribAuthor ">Sivaraja  Vaithiyalingam</span>, <span class="hlFld-ContribAuthor ">Duane  Currier</span>, <span class="hlFld-ContribAuthor ">Mi-Kyung  Yun</span>, <span class="hlFld-ContribAuthor ">Dinesh M.  Fernando</span>, <span class="hlFld-ContribAuthor ">Julie  Maier</span>, <span class="hlFld-ContribAuthor ">Heather  Tillman</span>, <span class="hlFld-ContribAuthor ">Purva  Bulsara</span>, <span class="hlFld-ContribAuthor ">Zhaohua  Lu</span>, <span class="hlFld-ContribAuthor ">Sourav  Das</span>, <span class="hlFld-ContribAuthor ">Anang  Shelat</span>, <span class="hlFld-ContribAuthor ">Zhenmei  Li</span>, <span class="hlFld-ContribAuthor ">Brandon  Young</span>, <span class="hlFld-ContribAuthor ">Richard  Lee</span>, <span class="hlFld-ContribAuthor ">Zoran  Rankovic</span>, <span class="hlFld-ContribAuthor ">Andrew J.  Murphy</span>, <span class="hlFld-ContribAuthor ">Stephen W.  White</span>, <span class="hlFld-ContribAuthor ">Andrew M.  Davidoff</span>, <span class="hlFld-ContribAuthor ">Taosheng  Chen</span>, <span class="hlFld-ContribAuthor ">Jun  Yang</span>. </span><span class="cited-content_cbyCitation_article-title">17-DMAG dually inhibits Hsp90 and histone lysine demethylases in alveolar rhabdomyosarcoma. </span><span class="cited-content_cbyCitation_journal-name">iScience</span><span> <strong>2021,</strong> <em>24 </em>
                                    (1)
                                     , 101996. <a href="https://doi.org/10.1016/j.isci.2020.101996" title="DOI URL">https://doi.org/10.1016/j.isci.2020.101996</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.isci.2020.101996&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.isci.2020.101996%26sid%3Dliteratum%253Aachs%26jtitle%3DiScience%26atitle%3D17-DMAG%252Bdually%252Binhibits%252BHsp90%252Band%252Bhistone%252Blysine%252Bdemethylases%252Bin%252Balveolar%252Brhabdomyosarcoma%26aulast%3DSingh%26aufirst%3DShivendra%26date%3D2021%26volume%3D24%26issue%3D1%26spage%3D101996" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shu‐Zhong  Cui</span>, <span class="hlFld-ContribAuthor ">Zi‐Ying  Lei</span>, <span class="hlFld-ContribAuthor ">Tian‐Pei  Guan</span>, <span class="hlFld-ContribAuthor ">Ling‐Ling  Fan</span>, <span class="hlFld-ContribAuthor ">You‐Qiang  Li</span>, <span class="hlFld-ContribAuthor ">Xin‐Yan  Geng</span>, <span class="hlFld-ContribAuthor ">De‐Xue  Fu</span>, <span class="hlFld-ContribAuthor ">Hao‐Wu  Jiang</span>, <span class="hlFld-ContribAuthor ">Song‐Hui  Xu</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting USP1‐dependent KDM4A protein stability as a potential prostate cancer therapy. </span><span class="cited-content_cbyCitation_journal-name">Cancer Science</span><span> <strong>2020,</strong> <em>111 </em>
                                    (5)
                                     , 1567-1581. <a href="https://doi.org/10.1111/cas.14375" title="DOI URL">https://doi.org/10.1111/cas.14375</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1111/cas.14375&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1111%2Fcas.14375%26sid%3Dliteratum%253Aachs%26jtitle%3DCancer%2520Science%26atitle%3DTargeting%252BUSP1%2525E2%252580%252590dependent%252BKDM4A%252Bprotein%252Bstability%252Bas%252Ba%252Bpotential%252Bprostate%252Bcancer%252Btherapy%26aulast%3DCui%26aufirst%3DShu%25E2%2580%2590Zhong%26date%3D2020%26date%3D2020%26volume%3D111%26issue%3D5%26spage%3D1567%26epage%3D1581" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dong Hoon  Lee</span>, <span class="hlFld-ContribAuthor ">Go Woon  Kim</span>, <span class="hlFld-ContribAuthor ">Yu Hyun  Jeon</span>, <span class="hlFld-ContribAuthor ">Jung  Yoo</span>, <span class="hlFld-ContribAuthor ">Sang Wu  Lee</span>, <span class="hlFld-ContribAuthor ">So Hee  Kwon</span>. </span><span class="cited-content_cbyCitation_article-title">Advances in histone demethylase KDM4 as cancer therapeutic targets. </span><span class="cited-content_cbyCitation_journal-name">The FASEB Journal</span><span> <strong>2020,</strong> <em>34 </em>
                                    (3)
                                     , 3461-3484. <a href="https://doi.org/10.1096/fj.201902584R" title="DOI URL">https://doi.org/10.1096/fj.201902584R</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1096/fj.201902584R&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1096%2Ffj.201902584R%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520FASEB%2520Journal%26atitle%3DAdvances%252Bin%252Bhistone%252Bdemethylase%252BKDM4%252Bas%252Bcancer%252Btherapeutic%252Btargets%26aulast%3DLee%26aufirst%3DDong%2BHoon%26date%3D2020%26date%3D2020%26volume%3D34%26issue%3D3%26spage%3D3461%26epage%3D3484" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hyerin  Song</span>, <span class="hlFld-ContribAuthor ">Jung Woong  Chu</span>, <span class="hlFld-ContribAuthor ">Su Chan  Park</span>, <span class="hlFld-ContribAuthor ">Hyuntae  Im</span>, <span class="hlFld-ContribAuthor ">Il-Geun  Park</span>, <span class="hlFld-ContribAuthor ">Hyunkyung  Kim</span>, <span class="hlFld-ContribAuthor ">Ji Min  Lee</span>. </span><span class="cited-content_cbyCitation_article-title">Isoform-Specific Lysine Methylation of RORα2 by SETD7 Is Required for Association of the TIP60 Coactivator Complex in Prostate Cancer Progression. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2020,</strong> <em>21 </em>
                                    (5)
                                     , 1622. <a href="https://doi.org/10.3390/ijms21051622" title="DOI URL">https://doi.org/10.3390/ijms21051622</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms21051622&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms21051622%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DIsoform-Specific%252BLysine%252BMethylation%252Bof%252BROR%2525CE%2525B12%252Bby%252BSETD7%252BIs%252BRequired%252Bfor%252BAssociation%252Bof%252Bthe%252BTIP60%252BCoactivator%252BComplex%252Bin%252BProstate%252BCancer%252BProgression%26aulast%3DSong%26aufirst%3DHyerin%26date%3D2020%26date%3D2020%26volume%3D21%26issue%3D5%26spage%3D1622" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jianjun  Sha</span>, <span class="hlFld-ContribAuthor ">Qing  Han</span>, <span class="hlFld-ContribAuthor ">Chenfei  Chi</span>, <span class="hlFld-ContribAuthor ">Yinjie  Zhu</span>, <span class="hlFld-ContribAuthor ">Jiahua  Pan</span>, <span class="hlFld-ContribAuthor ">Baijun  Dong</span>, <span class="hlFld-ContribAuthor ">Yiran  Huang</span>, <span class="hlFld-ContribAuthor ">Weiliang  Xia</span>, <span class="hlFld-ContribAuthor ">Wei  Xue</span>. </span><span class="cited-content_cbyCitation_article-title">Upregulated KDM4B promotes prostate cancer cell proliferation by activating autophagy. </span><span class="cited-content_cbyCitation_journal-name">Journal of Cellular Physiology</span><span> <strong>2020,</strong> <em>235 </em>
                                    (3)
                                     , 2129-2138. <a href="https://doi.org/10.1002/jcp.29117" title="DOI URL">https://doi.org/10.1002/jcp.29117</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/jcp.29117&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fjcp.29117%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Cellular%2520Physiology%26atitle%3DUpregulated%252BKDM4B%252Bpromotes%252Bprostate%252Bcancer%252Bcell%252Bproliferation%252Bby%252Bactivating%252Bautophagy%26aulast%3DSha%26aufirst%3DJianjun%26date%3D2020%26date%3D2019%26volume%3D235%26issue%3D3%26spage%3D2129%26epage%3D2138" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jung Nyeo  Chun</span>, <span class="hlFld-ContribAuthor ">Minsoo  Cho</span>, <span class="hlFld-ContribAuthor ">Soonbum  Park</span>, <span class="hlFld-ContribAuthor ">Insuk  So</span>, <span class="hlFld-ContribAuthor ">Ju-Hong  Jeon</span>. </span><span class="cited-content_cbyCitation_article-title">The conflicting role of E2F1 in prostate cancer: A matter of cell context or interpretational flexibility?. </span><span class="cited-content_cbyCitation_journal-name">Biochimica et Biophysica Acta (BBA) - Reviews on Cancer</span><span> <strong>2020,</strong> <em>1873 </em>
                                    (1)
                                     , 188336. <a href="https://doi.org/10.1016/j.bbcan.2019.188336" title="DOI URL">https://doi.org/10.1016/j.bbcan.2019.188336</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bbcan.2019.188336&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bbcan.2019.188336%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochimica%2520et%2520Biophysica%2520Acta%2520%2528BBA%2529%2520-%2520Reviews%2520on%2520Cancer%26atitle%3DThe%252Bconflicting%252Brole%252Bof%252BE2F1%252Bin%252Bprostate%252Bcancer%25253A%252BA%252Bmatter%252Bof%252Bcell%252Bcontext%252Bor%252Binterpretational%252Bflexibility%25253F%26aulast%3DChun%26aufirst%3DJung%2BNyeo%26date%3D2020%26volume%3D1873%26issue%3D1%26spage%3D188336" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lin</span>, <span class="hlFld-ContribAuthor ">Wang</span>, <span class="hlFld-ContribAuthor ">Chen</span>, <span class="hlFld-ContribAuthor ">Tseng</span>, <span class="hlFld-ContribAuthor ">Jiang</span>, <span class="hlFld-ContribAuthor ">Tsai</span>, <span class="hlFld-ContribAuthor ">Shen</span>, <span class="hlFld-ContribAuthor ">Yuh</span>, <span class="hlFld-ContribAuthor ">Sie</span>, <span class="hlFld-ContribAuthor ">Wang</span>, <span class="hlFld-ContribAuthor ">Kung</span>, <span class="hlFld-ContribAuthor ">Chuu</span>. </span><span class="cited-content_cbyCitation_article-title">Histone Demethylase KDM4C Stimulates the Proliferation of Prostate Cancer Cells via Activation of AKT and c-Myc. </span><span class="cited-content_cbyCitation_journal-name">Cancers</span><span> <strong>2019,</strong> <em>11 </em>
                                    (11)
                                     , 1785. <a href="https://doi.org/10.3390/cancers11111785" title="DOI URL">https://doi.org/10.3390/cancers11111785</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/cancers11111785&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fcancers11111785%26sid%3Dliteratum%253Aachs%26jtitle%3DCancers%26atitle%3DHistone%252BDemethylase%252BKDM4C%252BStimulates%252Bthe%252BProliferation%252Bof%252BProstate%252BCancer%252BCells%252Bvia%252BActivation%252Bof%252BAKT%252Band%252Bc-Myc%26aulast%3DLin%26date%3D2019%26date%3D2019%26volume%3D11%26issue%3D11%26spage%3D1785" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Laura  Monaghan</span>, <span class="hlFld-ContribAuthor ">Matthew E.  Massett</span>, <span class="hlFld-ContribAuthor ">Roderick P.  Bunschoten</span>, <span class="hlFld-ContribAuthor ">Alex  Hoose</span>, <span class="hlFld-ContribAuthor ">Petrisor-Alin  Pirvan</span>, <span class="hlFld-ContribAuthor ">Robert M. J.  Liskamp</span>, <span class="hlFld-ContribAuthor ">Heather G.  Jørgensen</span>, <span class="hlFld-ContribAuthor ">Xu  Huang</span>. </span><span class="cited-content_cbyCitation_article-title">The Emerging Role of H3K9me3 as a Potential Therapeutic Target in Acute Myeloid Leukemia. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Oncology</span><span> <strong>2019,</strong> <em>9 </em><a href="https://doi.org/10.3389/fonc.2019.00705" title="DOI URL">https://doi.org/10.3389/fonc.2019.00705</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fonc.2019.00705&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffonc.2019.00705%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Oncology%26atitle%3DThe%252BEmerging%252BRole%252Bof%252BH3K9me3%252Bas%252Ba%252BPotential%252BTherapeutic%252BTarget%252Bin%252BAcute%252BMyeloid%252BLeukemia%26aulast%3DMonaghan%26aufirst%3DLaura%26date%3D2019%26date%3D2019%26volume%3D9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jie  Li</span>, <span class="hlFld-ContribAuthor ">Jeong Hyun  Ahn</span>, <span class="hlFld-ContribAuthor ">Gang Greg  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Understanding histone H3 lysine 36 methylation and its deregulation in disease. </span><span class="cited-content_cbyCitation_journal-name">Cellular and Molecular Life Sciences</span><span> <strong>2019,</strong> <em>76 </em>
                                    (15)
                                     , 2899-2916. <a href="https://doi.org/10.1007/s00018-019-03144-y" title="DOI URL">https://doi.org/10.1007/s00018-019-03144-y</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s00018-019-03144-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs00018-019-03144-y%26sid%3Dliteratum%253Aachs%26jtitle%3DCellular%2520and%2520Molecular%2520Life%2520Sciences%26atitle%3DUnderstanding%252Bhistone%252BH3%252Blysine%252B36%252Bmethylation%252Band%252Bits%252Bderegulation%252Bin%252Bdisease%26aulast%3DLi%26aufirst%3DJie%26date%3D2019%26date%3D2019%26volume%3D76%26issue%3D15%26spage%3D2899%26epage%3D2916" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jones</span>, <span class="hlFld-ContribAuthor ">Wilson</span>, <span class="hlFld-ContribAuthor ">Thomas</span>, <span class="hlFld-ContribAuthor ">Gaughan</span>, <span class="hlFld-ContribAuthor ">Wade</span>. </span><span class="cited-content_cbyCitation_article-title">The Histone Demethylase Enzymes KDM3A and KDM4B Co-Operatively Regulate Chromatin Transactions of the Estrogen Receptor in Breast Cancer. </span><span class="cited-content_cbyCitation_journal-name">Cancers</span><span> <strong>2019,</strong> <em>11 </em>
                                    (8)
                                     , 1122. <a href="https://doi.org/10.3390/cancers11081122" title="DOI URL">https://doi.org/10.3390/cancers11081122</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/cancers11081122&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fcancers11081122%26sid%3Dliteratum%253Aachs%26jtitle%3DCancers%26atitle%3DThe%252BHistone%252BDemethylase%252BEnzymes%252BKDM3A%252Band%252BKDM4B%252BCo-Operatively%252BRegulate%252BChromatin%252BTransactions%252Bof%252Bthe%252BEstrogen%252BReceptor%252Bin%252BBreast%252BCancer%26aulast%3DJones%26date%3D2019%26date%3D2019%26volume%3D11%26issue%3D8%26spage%3D1122" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jia‐Chen  Jing</span>, <span class="hlFld-ContribAuthor ">Zhen  Feng</span>, <span class="hlFld-ContribAuthor ">Zhong‐Hua  Chen</span>, <span class="hlFld-ContribAuthor ">Bei‐Na  Ji</span>, <span class="hlFld-ContribAuthor ">Jing  Hong</span>, <span class="hlFld-ContribAuthor ">Nan  Tang</span>, <span class="hlFld-ContribAuthor ">Jin Ling  Yu</span>, <span class="hlFld-ContribAuthor ">Shao‐Ying  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">KDM4B promotes gastric cancer metastasis by regulating miR‐125b‐mediated activation of Wnt signaling. </span><span class="cited-content_cbyCitation_journal-name">Journal of Cellular Biochemistry</span><span> <strong>2019,</strong> <em>120 </em>
                                    (5)
                                     , 7897-7906. <a href="https://doi.org/10.1002/jcb.28065" title="DOI URL">https://doi.org/10.1002/jcb.28065</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/jcb.28065&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fjcb.28065%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Cellular%2520Biochemistry%26atitle%3DKDM4B%252Bpromotes%252Bgastric%252Bcancer%252Bmetastasis%252Bby%252Bregulating%252BmiR%2525E2%252580%252590125b%2525E2%252580%252590mediated%252Bactivation%252Bof%252BWnt%252Bsignaling%26aulast%3DJing%26aufirst%3DJia%25E2%2580%2590Chen%26date%3D2019%26date%3D2018%26volume%3D120%26issue%3D5%26spage%3D7897%26epage%3D7906" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chiara  Luise</span>, <span class="hlFld-ContribAuthor ">Tino  Heimburg</span>, <span class="hlFld-ContribAuthor ">Berin  Karaman</span>, <span class="hlFld-ContribAuthor ">Dina  Robaa</span>, <span class="hlFld-ContribAuthor ">Wolfgang  Sippl</span>. </span><span class="cited-content_cbyCitation_article-title">Computer‐based Lead Identification for Epigenetic Targets. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 45-77. <a href="https://doi.org/10.1002/9783527809257.ch3" title="DOI URL">https://doi.org/10.1002/9783527809257.ch3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9783527809257.ch3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9783527809257.ch3%26sid%3Dliteratum%253Aachs%26atitle%3DComputer%2525E2%252580%252590based%252BLead%252BIdentification%252Bfor%252BEpigenetic%252BTargets%26aulast%3DLuise%26aufirst%3DChiara%26date%3D2019%26date%3D2019%26spage%3D45%26epage%3D77%26pub%3DWiley%26atitle%3DEpigenetic%252BDrug%252BDiscovery%26aulast%3DSippl%26aufirst%3DWolfgang%26date%3D2019%26date%3D2019%26volume%3D286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Meng-Chen  Wu</span>, <span class="hlFld-ContribAuthor ">Hsin-Hung  Cheng</span>, <span class="hlFld-ContribAuthor ">Ta-Sen  Yeh</span>, <span class="hlFld-ContribAuthor ">Yi-Chen  Li</span>, <span class="hlFld-ContribAuthor ">Tsan-Jan  Chen</span>, <span class="hlFld-ContribAuthor ">Wei Yang  Sit</span>, <span class="hlFld-ContribAuthor ">Chih-Pin  Chuu</span>, <span class="hlFld-ContribAuthor ">Hsing-Jien  Kung</span>, <span class="hlFld-ContribAuthor ">Shu  Chien</span>, <span class="hlFld-ContribAuthor ">Wen-Ching  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">KDM4B is a coactivator of c-Jun and involved in gastric carcinogenesis. </span><span class="cited-content_cbyCitation_journal-name">Cell Death & Disease</span><span> <strong>2019,</strong> <em>10 </em>
                                    (2)
                                     <a href="https://doi.org/10.1038/s41419-019-1305-y" title="DOI URL">https://doi.org/10.1038/s41419-019-1305-y</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41419-019-1305-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41419-019-1305-y%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Death%2520%2526%2520Disease%26atitle%3DKDM4B%252Bis%252Ba%252Bcoactivator%252Bof%252Bc-Jun%252Band%252Binvolved%252Bin%252Bgastric%252Bcarcinogenesis%26aulast%3DWu%26aufirst%3DMeng-Chen%26date%3D2019%26date%3D2019%26volume%3D10%26issue%3D2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kevin  Dzobo</span>. </span><span class="cited-content_cbyCitation_article-title">Epigenomics-Guided Drug Development: Recent Advances in Solving the Cancer Treatment “jigsaw puzzle”. </span><span class="cited-content_cbyCitation_journal-name">OMICS: A Journal of Integrative Biology</span><span> <strong>2019,</strong> <em>23 </em>
                                    (2)
                                     , 70-85. <a href="https://doi.org/10.1089/omi.2018.0206" title="DOI URL">https://doi.org/10.1089/omi.2018.0206</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1089/omi.2018.0206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1089%2Fomi.2018.0206%26sid%3Dliteratum%253Aachs%26jtitle%3DOMICS%253A%2520A%2520Journal%2520of%2520Integrative%2520Biology%26atitle%3DEpigenomics-Guided%252BDrug%252BDevelopment%25253A%252BRecent%252BAdvances%252Bin%252BSolving%252Bthe%252BCancer%252BTreatment%252B%2525E2%252580%25259Cjigsaw%252Bpuzzle%2525E2%252580%25259D%26aulast%3DDzobo%26aufirst%3DKevin%26date%3D2019%26volume%3D23%26issue%3D2%26spage%3D70%26epage%3D85" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Cailin  Wilson</span>, <span class="hlFld-ContribAuthor ">Adam J.  Krieg</span>. </span><span class="cited-content_cbyCitation_article-title">KDM4B: A Nail for Every Hammer?. </span><span class="cited-content_cbyCitation_journal-name">Genes</span><span> <strong>2019,</strong> <em>10 </em>
                                    (2)
                                     , 134. <a href="https://doi.org/10.3390/genes10020134" title="DOI URL">https://doi.org/10.3390/genes10020134</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/genes10020134&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fgenes10020134%26sid%3Dliteratum%253Aachs%26jtitle%3DGenes%26atitle%3DKDM4B%25253A%252BA%252BNail%252Bfor%252BEvery%252BHammer%25253F%26aulast%3DWilson%26aufirst%3DCailin%26date%3D2019%26date%3D2019%26volume%3D10%26issue%3D2%26spage%3D134" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Miranda  Wright</span>, <span class="hlFld-ContribAuthor ">Paul E.  Brennan</span>, <span class="hlFld-ContribAuthor ">Akane  Kawamura</span>. </span><span class="cited-content_cbyCitation_article-title">Inhibitors of JmjC-Containing Histone Demethylases. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 221-253. <a href="https://doi.org/10.1007/7355_2019_75" title="DOI URL">https://doi.org/10.1007/7355_2019_75</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/7355_2019_75&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F7355_2019_75%26sid%3Dliteratum%253Aachs%26atitle%3DInhibitors%252Bof%252BJmjC-Containing%252BHistone%252BDemethylases%26aulast%3DWright%26aufirst%3DMiranda%26date%3D2019%26date%3D2020%26spage%3D221%26epage%3D253%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DChemical%252BEpigenetics%26aulast%3DMai%26aufirst%3DAntonello%26date%3D2020%26volume%3D33" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dina  Robaa</span>, <span class="hlFld-ContribAuthor ">Jelena  Melesina</span>, <span class="hlFld-ContribAuthor ">Chiara  Luise</span>, <span class="hlFld-ContribAuthor ">Wolfgang  Sippl</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-Based Design of Epigenetic Inhibitors. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 455-486. <a href="https://doi.org/10.1007/7355_2019_80" title="DOI URL">https://doi.org/10.1007/7355_2019_80</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/7355_2019_80&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F7355_2019_80%26sid%3Dliteratum%253Aachs%26atitle%3DStructure-Based%252BDesign%252Bof%252BEpigenetic%252BInhibitors%26aulast%3DRobaa%26aufirst%3DDina%26date%3D2019%26date%3D2019%26spage%3D455%26epage%3D486%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DChemical%252BEpigenetics%26aulast%3DMai%26aufirst%3DAntonello%26date%3D2020%26volume%3D33" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hung-Jung  Wang</span>, <span class="hlFld-ContribAuthor ">Mamata  Pochampalli</span>, <span class="hlFld-ContribAuthor ">Ling-Yu  Wang</span>, <span class="hlFld-ContribAuthor ">June X  Zou</span>, <span class="hlFld-ContribAuthor ">Pei-Shan  Li</span>, <span class="hlFld-ContribAuthor ">Sheng-Chieh  Hsu</span>, <span class="hlFld-ContribAuthor ">Bi-Juan  Wang</span>, <span class="hlFld-ContribAuthor ">Shih-Han  Huang</span>, <span class="hlFld-ContribAuthor ">Ping  Yang</span>, <span class="hlFld-ContribAuthor ">Joy C.  Yang</span>, <span class="hlFld-ContribAuthor ">Cheng-Ying  Chu</span>, <span class="hlFld-ContribAuthor ">Chia-Ling  Hsieh</span>, <span class="hlFld-ContribAuthor ">Shian-Ying  Sung</span>, <span class="hlFld-ContribAuthor ">Chien-Feng  Li</span>, <span class="hlFld-ContribAuthor ">Clifford G.  Tepper</span>, <span class="hlFld-ContribAuthor ">David K.  Ann</span>, <span class="hlFld-ContribAuthor ">Allen C.  Gao</span>, <span class="hlFld-ContribAuthor ">Christopher P.  Evans</span>, <span class="hlFld-ContribAuthor ">Yoshihiro  Izumiya</span>, <span class="hlFld-ContribAuthor ">Chi-Pin  Chuu</span>, <span class="hlFld-ContribAuthor ">Wen-Ching  Wang</span>, <span class="hlFld-ContribAuthor ">Hong-Wu  Chen</span>, <span class="hlFld-ContribAuthor ">Hsing-Jien  Kung</span>. </span><span class="cited-content_cbyCitation_article-title">KDM8/JMJD5 as a dual coactivator of AR and PKM2 integrates AR/EZH2 network and tumor metabolism in CRPC. </span><span class="cited-content_cbyCitation_journal-name">Oncogene</span><span> <strong>2019,</strong> <em>38 </em>
                                    (1)
                                     , 17-32. <a href="https://doi.org/10.1038/s41388-018-0414-x" title="DOI URL">https://doi.org/10.1038/s41388-018-0414-x</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41388-018-0414-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41388-018-0414-x%26sid%3Dliteratum%253Aachs%26jtitle%3DOncogene%26atitle%3DKDM8%25252FJMJD5%252Bas%252Ba%252Bdual%252Bcoactivator%252Bof%252BAR%252Band%252BPKM2%252Bintegrates%252BAR%25252FEZH2%252Bnetwork%252Band%252Btumor%252Bmetabolism%252Bin%252BCRPC%26aulast%3DWang%26aufirst%3DHung-Jung%26date%3D2019%26date%3D2018%26volume%3D38%26issue%3D1%26spage%3D17%26epage%3D32" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ying  Xiang</span>, <span class="hlFld-ContribAuthor ">Kai  Yan</span>, <span class="hlFld-ContribAuthor ">Qian  Zheng</span>, <span class="hlFld-ContribAuthor ">Haiqiang  Ke</span>, <span class="hlFld-ContribAuthor ">Jie  Cheng</span>, <span class="hlFld-ContribAuthor ">Wenjun  Xiong</span>, <span class="hlFld-ContribAuthor ">Xin  Shi</span>, <span class="hlFld-ContribAuthor ">Lei  Wei</span>, <span class="hlFld-ContribAuthor ">Min  Zhao</span>, <span class="hlFld-ContribAuthor ">Fei  Yang</span>, <span class="hlFld-ContribAuthor ">Ping  Wang</span>, <span class="hlFld-ContribAuthor ">Xing  Lu</span>, <span class="hlFld-ContribAuthor ">Li  Fu</span>, <span class="hlFld-ContribAuthor ">Xuemei  Lu</span>, <span class="hlFld-ContribAuthor ">Feng  Li</span>. </span><span class="cited-content_cbyCitation_article-title">Histone Demethylase KDM4B Promotes DNA Damage by Activating Long Interspersed Nuclear Element-1. </span><span class="cited-content_cbyCitation_journal-name">Cancer Research</span><span> <strong>2019,</strong> <em>79 </em>
                                    (1)
                                     , 86-98. <a href="https://doi.org/10.1158/0008-5472.CAN-18-1310" title="DOI URL">https://doi.org/10.1158/0008-5472.CAN-18-1310</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1158/0008-5472.CAN-18-1310&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1158%2F0008-5472.CAN-18-1310%26sid%3Dliteratum%253Aachs%26jtitle%3DCancer%2520Research%26atitle%3DHistone%252BDemethylase%252BKDM4B%252BPromotes%252BDNA%252BDamage%252Bby%252BActivating%252BLong%252BInterspersed%252BNuclear%252BElement-1%26aulast%3DXiang%26aufirst%3DYing%26date%3D2019%26date%3D2018%26volume%3D79%26issue%3D1%26spage%3D86%26epage%3D98" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">S. P.  Rowbotham</span>, <span class="hlFld-ContribAuthor ">F.  Li</span>, <span class="hlFld-ContribAuthor ">A. F. M.  Dost</span>, <span class="hlFld-ContribAuthor ">S. M.  Louie</span>, <span class="hlFld-ContribAuthor ">B. P.  Marsh</span>, <span class="hlFld-ContribAuthor ">P.  Pessina</span>, <span class="hlFld-ContribAuthor ">C. R.  Anbarasu</span>, <span class="hlFld-ContribAuthor ">C. F.  Brainson</span>, <span class="hlFld-ContribAuthor ">S. J.  Tuminello</span>, <span class="hlFld-ContribAuthor ">A.  Lieberman</span>, <span class="hlFld-ContribAuthor ">S.  Ryeom</span>, <span class="hlFld-ContribAuthor ">T. M.  Schlaeger</span>, <span class="hlFld-ContribAuthor ">B. J.  Aronow</span>, <span class="hlFld-ContribAuthor ">H.  Watanabe</span>, <span class="hlFld-ContribAuthor ">K. K.  Wong</span>, <span class="hlFld-ContribAuthor ">C. F.  Kim</span>. </span><span class="cited-content_cbyCitation_article-title">H3K9 methyltransferases and demethylases control lung tumor-propagating cells and lung cancer progression. </span><span class="cited-content_cbyCitation_journal-name">Nature Communications</span><span> <strong>2018,</strong> <em>9 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41467-018-07077-1" title="DOI URL">https://doi.org/10.1038/s41467-018-07077-1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41467-018-07077-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41467-018-07077-1%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Communications%26atitle%3DH3K9%252Bmethyltransferases%252Band%252Bdemethylases%252Bcontrol%252Blung%252Btumor-propagating%252Bcells%252Band%252Blung%252Bcancer%252Bprogression%26aulast%3DRowbotham%26aufirst%3DS.%2BP.%26date%3D2018%26date%3D2018%26volume%3D9%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Adam J.  Krieg</span>, <span class="hlFld-ContribAuthor ">Sarah R.  Mullinax</span>, <span class="hlFld-ContribAuthor ">Frances  Grimstad</span>, <span class="hlFld-ContribAuthor ">Kaitlin  Marquis</span>, <span class="hlFld-ContribAuthor ">Elizabeth  Constance</span>, <span class="hlFld-ContribAuthor ">Yan  Hong</span>, <span class="hlFld-ContribAuthor ">Sacha A.  Krieg</span>, <span class="hlFld-ContribAuthor ">Katherine F.  Roby</span>. </span><span class="cited-content_cbyCitation_article-title">Histone demethylase KDM4A and KDM4B expression in granulosa cells from women undergoing in vitro fertilization. </span><span class="cited-content_cbyCitation_journal-name">Journal of Assisted Reproduction and Genetics</span><span> <strong>2018,</strong> <em>35 </em>
                                    (6)
                                     , 993-1003. <a href="https://doi.org/10.1007/s10815-018-1151-3" title="DOI URL">https://doi.org/10.1007/s10815-018-1151-3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10815-018-1151-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10815-018-1151-3%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Assisted%2520Reproduction%2520and%2520Genetics%26atitle%3DHistone%252Bdemethylase%252BKDM4A%252Band%252BKDM4B%252Bexpression%252Bin%252Bgranulosa%252Bcells%252Bfrom%252Bwomen%252Bundergoing%252Bin%252Bvitro%252Bfertilization%26aulast%3DKrieg%26aufirst%3DAdam%2BJ.%26date%3D2018%26date%3D2018%26volume%3D35%26issue%3D6%26spage%3D993%26epage%3D1003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xing  Lu</span>, <span class="hlFld-ContribAuthor ">Yi-Ming  Wu</span>, <span class="hlFld-ContribAuthor ">Jing-Mei  Yang</span>, <span class="hlFld-ContribAuthor ">Feng-E.  Ma</span>, <span class="hlFld-ContribAuthor ">Liang-Ping  Li</span>, <span class="hlFld-ContribAuthor ">Sheng  Chen</span>, <span class="hlFld-ContribAuthor ">Ye  Zhang</span>, <span class="hlFld-ContribAuthor ">Qing-Ling  Ni</span>, <span class="hlFld-ContribAuthor ">Ying-Ming  Pan</span>, <span class="hlFld-ContribAuthor ">Xue  Hong</span>, <span class="hlFld-ContribAuthor ">Yan  Peng</span>. </span><span class="cited-content_cbyCitation_article-title">Preparation of Rhodium(III) complexes with 2(1H)-quinolinone derivatives and evaluation of their in vitro and in vivo antitumor activity. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>151 </em>, 226-236. <a href="https://doi.org/10.1016/j.ejmech.2018.03.074" title="DOI URL">https://doi.org/10.1016/j.ejmech.2018.03.074</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2018.03.074&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2018.03.074%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DPreparation%252Bof%252BRhodium%252528III%252529%252Bcomplexes%252Bwith%252B2%2525281H%252529-quinolinone%252Bderivatives%252Band%252Bevaluation%252Bof%252Btheir%252Bin%2525C2%2525A0vitro%252Band%252Bin%2525C2%2525A0vivo%252Bantitumor%252Bactivity%26aulast%3DLu%26aufirst%3DXing%26date%3D2018%26volume%3D151%26spage%3D226%26epage%3D236" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Wenchao  Lu</span>, <span class="hlFld-ContribAuthor ">Rukang  Zhang</span>, <span class="hlFld-ContribAuthor ">Hao  Jiang</span>, <span class="hlFld-ContribAuthor ">Huimin  Zhang</span>, <span class="hlFld-ContribAuthor ">Cheng  Luo</span>. </span><span class="cited-content_cbyCitation_article-title">Computer-Aided Drug Design in Epigenetics. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Chemistry</span><span> <strong>2018,</strong> <em>6 </em><a href="https://doi.org/10.3389/fchem.2018.00057" title="DOI URL">https://doi.org/10.3389/fchem.2018.00057</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fchem.2018.00057&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffchem.2018.00057%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Chemistry%26atitle%3DComputer-Aided%252BDrug%252BDesign%252Bin%252BEpigenetics%26aulast%3DLu%26aufirst%3DWenchao%26date%3D2018%26date%3D2018%26volume%3D6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Qianqian  Wang</span>, <span class="hlFld-ContribAuthor ">Jiahui  Xu</span>, <span class="hlFld-ContribAuthor ">Ying  Li</span>, <span class="hlFld-ContribAuthor ">Jumin  Huang</span>, <span class="hlFld-ContribAuthor ">Zebo  Jiang</span>, <span class="hlFld-ContribAuthor ">Yuwei  Wang</span>, <span class="hlFld-ContribAuthor ">Liang  Liu</span>, <span class="hlFld-ContribAuthor ">Elaine Lai Han  Leung</span>, <span class="hlFld-ContribAuthor ">Xiaojun  Yao</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of a Novel Protein Arginine Methyltransferase 5 Inhibitor in Non-small Cell Lung Cancer by Structure-Based Virtual Screening. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Pharmacology</span><span> <strong>2018,</strong> <em>9 </em><a href="https://doi.org/10.3389/fphar.2018.00173" title="DOI URL">https://doi.org/10.3389/fphar.2018.00173</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fphar.2018.00173&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffphar.2018.00173%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Pharmacology%26atitle%3DIdentification%252Bof%252Ba%252BNovel%252BProtein%252BArginine%252BMethyltransferase%252B5%252BInhibitor%252Bin%252BNon-small%252BCell%252BLung%252BCancer%252Bby%252BStructure-Based%252BVirtual%252BScreening%26aulast%3DWang%26aufirst%3DQianqian%26date%3D2018%26date%3D2018%26volume%3D9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hongzhi  Lin</span>, <span class="hlFld-ContribAuthor ">Qihang  Li</span>, <span class="hlFld-ContribAuthor ">Qi  Li</span>, <span class="hlFld-ContribAuthor ">Jie  Zhu</span>, <span class="hlFld-ContribAuthor ">Kai  Gu</span>, <span class="hlFld-ContribAuthor ">Xueyang  Jiang</span>, <span class="hlFld-ContribAuthor ">Qianqian  Hu</span>, <span class="hlFld-ContribAuthor ">Feng  Feng</span>, <span class="hlFld-ContribAuthor ">Wei  Qu</span>, <span class="hlFld-ContribAuthor ">Yao  Chen</span>, <span class="hlFld-ContribAuthor ">Haopeng  Sun</span>. </span><span class="cited-content_cbyCitation_article-title">Small molecule KDM4s inhibitors as anti-cancer agents. </span><span class="cited-content_cbyCitation_journal-name">Journal of Enzyme Inhibition and Medicinal Chemistry</span><span> <strong>2018,</strong> <em>33 </em>
                                    (1)
                                     , 777-793. <a href="https://doi.org/10.1080/14756366.2018.1455676" title="DOI URL">https://doi.org/10.1080/14756366.2018.1455676</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/14756366.2018.1455676&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F14756366.2018.1455676%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Enzyme%2520Inhibition%2520and%2520Medicinal%2520Chemistry%26atitle%3DSmall%252Bmolecule%252BKDM4s%252Binhibitors%252Bas%252Banti-cancer%252Bagents%26aulast%3DLin%26aufirst%3DHongzhi%26date%3D2018%26date%3D2018%26volume%3D33%26issue%3D1%26spage%3D777%26epage%3D793" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jun  Yang</span>, <span class="hlFld-ContribAuthor ">Adrian  Harris</span>, <span class="hlFld-ContribAuthor ">Andrew  Davidoff</span>. </span><span class="cited-content_cbyCitation_article-title">Hypoxia and Hormone-Mediated Pathways Converge at the Histone Demethylase KDM4B in Cancer. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2018,</strong> <em>19 </em>
                                    (1)
                                     , 240. <a href="https://doi.org/10.3390/ijms19010240" title="DOI URL">https://doi.org/10.3390/ijms19010240</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms19010240&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms19010240%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DHypoxia%252Band%252BHormone-Mediated%252BPathways%252BConverge%252Bat%252Bthe%252BHistone%252BDemethylase%252BKDM4B%252Bin%252BCancer%26aulast%3DYang%26aufirst%3DJun%26date%3D2018%26date%3D2018%26volume%3D19%26issue%3D1%26spage%3D240" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Eric  Metzger</span>, <span class="hlFld-ContribAuthor ">Stella S.  Stepputtis</span>, <span class="hlFld-ContribAuthor ">Juliane  Strietz</span>, <span class="hlFld-ContribAuthor ">Bogdan-Tiberius  Preca</span>, <span class="hlFld-ContribAuthor ">Sylvia  Urban</span>, <span class="hlFld-ContribAuthor ">Dominica  Willmann</span>, <span class="hlFld-ContribAuthor ">Anita  Allen</span>, <span class="hlFld-ContribAuthor ">Fides  Zenk</span>, <span class="hlFld-ContribAuthor ">Nicola  Iovino</span>, <span class="hlFld-ContribAuthor ">Peter  Bronsert</span>, <span class="hlFld-ContribAuthor ">Amelie  Proske</span>, <span class="hlFld-ContribAuthor ">Marie  Follo</span>, <span class="hlFld-ContribAuthor ">Melanie  Boerries</span>, <span class="hlFld-ContribAuthor ">Elmar  Stickeler</span>, <span class="hlFld-ContribAuthor ">Jiangchun  Xu</span>, <span class="hlFld-ContribAuthor ">Michael B.  Wallace</span>, <span class="hlFld-ContribAuthor ">Jeffrey A.  Stafford</span>, <span class="hlFld-ContribAuthor ">Toufike  Kanouni</span>, <span class="hlFld-ContribAuthor ">Jochen  Maurer</span>, <span class="hlFld-ContribAuthor ">Roland  Schüle</span>. </span><span class="cited-content_cbyCitation_article-title">KDM4 Inhibition Targets Breast Cancer Stem–like Cells. </span><span class="cited-content_cbyCitation_journal-name">Cancer Research</span><span> <strong>2017,</strong> <em>77 </em>
                                    (21)
                                     , 5900-5912. <a href="https://doi.org/10.1158/0008-5472.CAN-17-1754" title="DOI URL">https://doi.org/10.1158/0008-5472.CAN-17-1754</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1158/0008-5472.CAN-17-1754&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1158%2F0008-5472.CAN-17-1754%26sid%3Dliteratum%253Aachs%26jtitle%3DCancer%2520Research%26atitle%3DKDM4%252BInhibition%252BTargets%252BBreast%252BCancer%252BStem%2525E2%252580%252593like%252BCells%26aulast%3DMetzger%26aufirst%3DEric%26date%3D2017%26date%3D2017%26volume%3D77%26issue%3D21%26spage%3D5900%26epage%3D5912" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hugo  Sepulveda</span>, <span class="hlFld-ContribAuthor ">Alejandro  Villagra</span>, <span class="hlFld-ContribAuthor ">Martin  Montecino</span>. </span><span class="cited-content_cbyCitation_article-title">Tet-Mediated DNA Demethylation Is Required for SWI/SNF-Dependent Chromatin Remodeling and Histone-Modifying Activities That Trigger Expression of the Sp7 Osteoblast Master Gene during Mesenchymal Lineage Commitment. </span><span class="cited-content_cbyCitation_journal-name">Molecular and Cellular Biology</span><span> <strong>2017,</strong> <em>37 </em>
                                    (20)
                                     <a href="https://doi.org/10.1128/MCB.00177-17" title="DOI URL">https://doi.org/10.1128/MCB.00177-17</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1128/MCB.00177-17&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1128%2FMCB.00177-17%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520and%2520Cellular%2520Biology%26atitle%3DTet-Mediated%252BDNA%252BDemethylation%252BIs%252BRequired%252Bfor%252BSWI%25252FSNF-Dependent%252BChromatin%252BRemodeling%252Band%252BHistone-Modifying%252BActivities%252BThat%252BTrigger%252BExpression%252Bof%252Bthe%252BSp7%252BOsteoblast%252BMaster%252BGene%252Bduring%252BMesenchymal%252BLineage%252BCommitment%26aulast%3DSepulveda%26aufirst%3DHugo%26date%3D2017%26volume%3D37%26issue%3D20" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">David M.  Carter</span>, <span class="hlFld-ContribAuthor ">Edgar  Specker</span>, <span class="hlFld-ContribAuthor ">Jessica  Przygodda</span>, <span class="hlFld-ContribAuthor ">Martin  Neuenschwander</span>, <span class="hlFld-ContribAuthor ">Jens Peter  von Kries</span>, <span class="hlFld-ContribAuthor ">Udo  Heinemann</span>, <span class="hlFld-ContribAuthor ">Marc  Nazaré</span>, <span class="hlFld-ContribAuthor ">Ulrich  Gohlke</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of a Novel Benzimidazole Pyrazolone Scaffold That Inhibits KDM4 Lysine Demethylases and Reduces Proliferation of Prostate Cancer Cells. </span><span class="cited-content_cbyCitation_journal-name">SLAS DISCOVERY: Advancing the Science of Drug Discovery</span><span> <strong>2017,</strong> <em>22 </em>
                                    (7)
                                     , 801-812. <a href="https://doi.org/10.1177/2472555217699157" title="DOI URL">https://doi.org/10.1177/2472555217699157</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1177/2472555217699157&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1177%2F2472555217699157%26sid%3Dliteratum%253Aachs%26jtitle%3DSLAS%2520DISCOVERY%253A%2520Advancing%2520the%2520Science%2520of%2520Drug%2520Discovery%26atitle%3DIdentification%252Bof%252Ba%252BNovel%252BBenzimidazole%252BPyrazolone%252BScaffold%252BThat%252BInhibits%252BKDM4%252BLysine%252BDemethylases%252Band%252BReduces%252BProliferation%252Bof%252BProstate%252BCancer%252BCells%26aulast%3DCarter%26aufirst%3DDavid%2BM.%26date%3D2017%26date%3D2017%26volume%3D22%26issue%3D7%26spage%3D801%26epage%3D812" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sophia Xiao  Pfister</span>, <span class="hlFld-ContribAuthor ">Alan  Ashworth</span>. </span><span class="cited-content_cbyCitation_article-title">Marked for death: targeting epigenetic changes in cancer. </span><span class="cited-content_cbyCitation_journal-name">Nature Reviews Drug Discovery</span><span> <strong>2017,</strong> <em>16 </em>
                                    (4)
                                     , 241-263. <a href="https://doi.org/10.1038/nrd.2016.256" title="DOI URL">https://doi.org/10.1038/nrd.2016.256</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/nrd.2016.256&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fnrd.2016.256%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Reviews%2520Drug%2520Discovery%26atitle%3DMarked%252Bfor%252Bdeath%25253A%252Btargeting%252Bepigenetic%252Bchanges%252Bin%252Bcancer%26aulast%3DPfister%26aufirst%3DSophia%2BXiao%26date%3D2017%26date%3D2017%26volume%3D16%26issue%3D4%26spage%3D241%26epage%3D263" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Parker L.  Sulkowski</span>, <span class="hlFld-ContribAuthor ">Christopher D.  Corso</span>, <span class="hlFld-ContribAuthor ">Nathaniel D.  Robinson</span>, <span class="hlFld-ContribAuthor ">Susan E.  Scanlon</span>, <span class="hlFld-ContribAuthor ">Karin R.  Purshouse</span>, <span class="hlFld-ContribAuthor ">Hanwen  Bai</span>, <span class="hlFld-ContribAuthor ">Yanfeng  Liu</span>, <span class="hlFld-ContribAuthor ">Ranjini K.  Sundaram</span>, <span class="hlFld-ContribAuthor ">Denise C.  Hegan</span>, <span class="hlFld-ContribAuthor ">Nathan R.  Fons</span>, <span class="hlFld-ContribAuthor ">Gregory A.  Breuer</span>, <span class="hlFld-ContribAuthor ">Yuanbin  Song</span>, <span class="hlFld-ContribAuthor ">Ketu  Mishra-Gorur</span>, <span class="hlFld-ContribAuthor ">Henk M.  De Feyter</span>, <span class="hlFld-ContribAuthor ">Robin A.  de Graaf</span>, <span class="hlFld-ContribAuthor ">Yulia V.  Surovtseva</span>, <span class="hlFld-ContribAuthor ">Maureen  Kachman</span>, <span class="hlFld-ContribAuthor ">Stephanie  Halene</span>, <span class="hlFld-ContribAuthor ">Murat  Günel</span>, <span class="hlFld-ContribAuthor ">Peter M.  Glazer</span>, <span class="hlFld-ContribAuthor ">Ranjit S.  Bindra</span>. </span><span class="cited-content_cbyCitation_article-title">2-Hydroxyglutarate produced by neomorphic IDH mutations suppresses homologous recombination and induces PARP inhibitor sensitivity. </span><span class="cited-content_cbyCitation_journal-name">Science Translational Medicine</span><span> <strong>2017,</strong> <em>9 </em>
                                    (375)
                                     , eaal2463. <a href="https://doi.org/10.1126/scitranslmed.aal2463" title="DOI URL">https://doi.org/10.1126/scitranslmed.aal2463</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1126/scitranslmed.aal2463&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1126%2Fscitranslmed.aal2463%26sid%3Dliteratum%253Aachs%26jtitle%3DScience%2520Translational%2520Medicine%26atitle%3D2-Hydroxyglutarate%252Bproduced%252Bby%252Bneomorphic%252BIDH%252Bmutations%252Bsuppresses%252Bhomologous%252Brecombination%252Band%252Binduces%252BPARP%252Binhibitor%252Bsensitivity%26aulast%3DSulkowski%26aufirst%3DParker%2BL.%26date%3D2017%26date%3D2017%26volume%3D9%26issue%3D375%26spage%3Deaal2463" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">John R.  Horton</span>, <span class="hlFld-ContribAuthor ">Molly  Gale</span>, <span class="hlFld-ContribAuthor ">Qin  Yan</span>, <span class="hlFld-ContribAuthor ">Xiaodong  Cheng</span>. </span><span class="cited-content_cbyCitation_article-title">The Molecular Basis of Histone Demethylation. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2017,</strong>,, 151-219. <a href="https://doi.org/10.1007/978-3-319-59786-7_7" title="DOI URL">https://doi.org/10.1007/978-3-319-59786-7_7</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-3-319-59786-7_7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-3-319-59786-7_7%26sid%3Dliteratum%253Aachs%26atitle%3DThe%252BMolecular%252BBasis%252Bof%252BHistone%252BDemethylation%26aulast%3DHorton%26aufirst%3DJohn%2BR.%26date%3D2017%26date%3D2017%26spage%3D151%26epage%3D219%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DDNA%252Band%252BHistone%252BMethylation%252Bas%252BCancer%252BTargets%26aulast%3DKaneda%26aufirst%3DAtsushi%26date%3D2017%26volume%3D376" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Steve  Lefever</span>, <span class="hlFld-ContribAuthor ">Jasper  Anckaert</span>, <span class="hlFld-ContribAuthor ">Pieter-Jan  Volders</span>, <span class="hlFld-ContribAuthor ">Manuel  Luypaert</span>, <span class="hlFld-ContribAuthor ">Jo  Vandesompele</span>, <span class="hlFld-ContribAuthor ">Pieter  Mestdagh</span>. </span><span class="cited-content_cbyCitation_article-title">decodeRNA— predicting non-coding RNA functions using guilt-by-association. </span><span class="cited-content_cbyCitation_journal-name">Database</span><span> <strong>2017,</strong> <em>2017 </em><a href="https://doi.org/10.1093/database/bax042" title="DOI URL">https://doi.org/10.1093/database/bax042</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/database/bax042&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fdatabase%2Fbax042%26sid%3Dliteratum%253Aachs%26jtitle%3DDatabase%26atitle%3DdecodeRNA%2525E2%252580%252594%252Bpredicting%252Bnon-coding%252BRNA%252Bfunctions%252Busing%252Bguilt-by-association%26aulast%3DLefever%26aufirst%3DSteve%26date%3D2017%26date%3D2017%26volume%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ashwini  Jambhekar</span>, <span class="hlFld-ContribAuthor ">Jamie N.  Anastas</span>, <span class="hlFld-ContribAuthor ">Yang  Shi</span>. </span><span class="cited-content_cbyCitation_article-title">Histone Lysine Demethylase Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Cold Spring Harbor Perspectives in Medicine</span><span> <strong>2017,</strong> <em>7 </em>
                                    (1)
                                     , a026484. <a href="https://doi.org/10.1101/cshperspect.a026484" title="DOI URL">https://doi.org/10.1101/cshperspect.a026484</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1101/cshperspect.a026484&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1101%2Fcshperspect.a026484%26sid%3Dliteratum%253Aachs%26jtitle%3DCold%2520Spring%2520Harbor%2520Perspectives%2520in%2520Medicine%26atitle%3DHistone%252BLysine%252BDemethylase%252BInhibitors%26aulast%3DJambhekar%26aufirst%3DAshwini%26date%3D2017%26date%3D2016%26volume%3D7%26issue%3D1%26spage%3Da026484" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Junjian  Wang</span>, <span class="hlFld-ContribAuthor ">Haibin  Wang</span>, <span class="hlFld-ContribAuthor ">Ling-Yu  Wang</span>, <span class="hlFld-ContribAuthor ">Demin  Cai</span>, <span class="hlFld-ContribAuthor ">Zhijian  Duan</span>, <span class="hlFld-ContribAuthor ">Yanhong  Zhang</span>, <span class="hlFld-ContribAuthor ">Peng  Chen</span>, <span class="hlFld-ContribAuthor ">June X  Zou</span>, <span class="hlFld-ContribAuthor ">Jianzhen  Xu</span>, <span class="hlFld-ContribAuthor ">Xinbin  Chen</span>, <span class="hlFld-ContribAuthor ">Hsing-Jien  Kung</span>, <span class="hlFld-ContribAuthor ">Hong-Wu  Chen</span>. </span><span class="cited-content_cbyCitation_article-title">Silencing the epigenetic silencer KDM4A for TRAIL and DR5 simultaneous induction and antitumor therapy. </span><span class="cited-content_cbyCitation_journal-name">Cell Death & Differentiation</span><span> <strong>2016,</strong> <em>23 </em>
                                    (11)
                                     , 1886-1896. <a href="https://doi.org/10.1038/cdd.2016.92" title="DOI URL">https://doi.org/10.1038/cdd.2016.92</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/cdd.2016.92&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fcdd.2016.92%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Death%2520%2526%2520Differentiation%26atitle%3DSilencing%252Bthe%252Bepigenetic%252Bsilencer%252BKDM4A%252Bfor%252BTRAIL%252Band%252BDR5%252Bsimultaneous%252Binduction%252Band%252Bantitumor%252Btherapy%26aulast%3DWang%26aufirst%3DJunjian%26date%3D2016%26date%3D2016%26volume%3D23%26issue%3D11%26spage%3D1886%26epage%3D1896" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ludovica  Morera</span>, <span class="hlFld-ContribAuthor ">Martin  Roatsch</span>, <span class="hlFld-ContribAuthor ">Michael C. D.  Fürst</span>, <span class="hlFld-ContribAuthor ">Inga  Hoffmann</span>, <span class="hlFld-ContribAuthor ">Johanna  Senger</span>, <span class="hlFld-ContribAuthor ">Mirjam  Hau</span>, <span class="hlFld-ContribAuthor ">Henriette  Franz</span>, <span class="hlFld-ContribAuthor ">Roland  Schüle</span>, <span class="hlFld-ContribAuthor ">Markus R.  Heinrich</span>, <span class="hlFld-ContribAuthor ">Manfred  Jung</span>. </span><span class="cited-content_cbyCitation_article-title">4-Biphenylalanine- and 3-Phenyltyrosine-Derived Hydroxamic Acids as Inhibitors of the JumonjiC-Domain-Containing Histone Demethylase KDM4A. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2016,</strong> <em>11 </em>
                                    (18)
                                     , 2063-2083. <a href="https://doi.org/10.1002/cmdc.201600218" title="DOI URL">https://doi.org/10.1002/cmdc.201600218</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.201600218&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.201600218%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3D4-Biphenylalanine-%252Band%252B3-Phenyltyrosine-Derived%252BHydroxamic%252BAcids%252Bas%252BInhibitors%252Bof%252Bthe%252BJumonjiC-Domain-Containing%252BHistone%252BDemethylase%252BKDM4A%26aulast%3DMorera%26aufirst%3DLudovica%26date%3D2016%26date%3D2016%26volume%3D11%26issue%3D18%26spage%3D2063%26epage%3D2083" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ling-Yu  Wang</span>, <span class="hlFld-ContribAuthor ">Chiu-Lien  Hung</span>, <span class="hlFld-ContribAuthor ">Yun-Ru  Chen</span>, <span class="hlFld-ContribAuthor ">Joy C.  Yang</span>, <span class="hlFld-ContribAuthor ">Junjian  Wang</span>, <span class="hlFld-ContribAuthor ">Mel  Campbell</span>, <span class="hlFld-ContribAuthor ">Yoshihiro  Izumiya</span>, <span class="hlFld-ContribAuthor ">Hong-Wu  Chen</span>, <span class="hlFld-ContribAuthor ">Wen-Ching  Wang</span>, <span class="hlFld-ContribAuthor ">David K.  Ann</span>, <span class="hlFld-ContribAuthor ">Hsing-Jien  Kung</span>. </span><span class="cited-content_cbyCitation_article-title">KDM4A Coactivates E2F1 to Regulate the PDK-Dependent Metabolic Switch between Mitochondrial Oxidation and Glycolysis. </span><span class="cited-content_cbyCitation_journal-name">Cell Reports</span><span> <strong>2016,</strong> <em>16 </em>
                                    (11)
                                     , 3016-3027. <a href="https://doi.org/10.1016/j.celrep.2016.08.018" title="DOI URL">https://doi.org/10.1016/j.celrep.2016.08.018</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.celrep.2016.08.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.celrep.2016.08.018%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Reports%26atitle%3DKDM4A%252BCoactivates%252BE2F1%252Bto%252BRegulate%252Bthe%252BPDK-Dependent%252BMetabolic%252BSwitch%252Bbetween%252BMitochondrial%252BOxidation%252Band%252BGlycolysis%26aulast%3DWang%26aufirst%3DLing-Yu%26date%3D2016%26volume%3D16%26issue%3D11%26spage%3D3016%26epage%3D3027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sharada S.  Labadie</span>, <span class="hlFld-ContribAuthor ">Peter S.  Dragovich</span>, <span class="hlFld-ContribAuthor ">Richard T.  Cummings</span>, <span class="hlFld-ContribAuthor ">Gauri  Deshmukh</span>, <span class="hlFld-ContribAuthor ">Amy  Gustafson</span>, <span class="hlFld-ContribAuthor ">Ning  Han</span>, <span class="hlFld-ContribAuthor ">Jean-Christophe  Harmange</span>, <span class="hlFld-ContribAuthor ">James R.  Kiefer</span>, <span class="hlFld-ContribAuthor ">Yue  Li</span>, <span class="hlFld-ContribAuthor ">Jun  Liang</span>, <span class="hlFld-ContribAuthor ">Bianca M.  Liederer</span>, <span class="hlFld-ContribAuthor ">Yichin  Liu</span>, <span class="hlFld-ContribAuthor ">Wanda  Manieri</span>, <span class="hlFld-ContribAuthor ">Wiefeng  Mao</span>, <span class="hlFld-ContribAuthor ">Lesley  Murray</span>, <span class="hlFld-ContribAuthor ">Daniel F.  Ortwine</span>, <span class="hlFld-ContribAuthor ">Patrick  Trojer</span>, <span class="hlFld-ContribAuthor ">Erica  VanderPorten</span>, <span class="hlFld-ContribAuthor ">Maia  Vinogradova</span>, <span class="hlFld-ContribAuthor ">Li  Wen</span>. </span><span class="cited-content_cbyCitation_article-title">Design and evaluation of 1,7-naphthyridones as novel KDM5 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2016,</strong> <em>26 </em>
                                    (18)
                                     , 4492-4496. <a href="https://doi.org/10.1016/j.bmcl.2016.07.070" title="DOI URL">https://doi.org/10.1016/j.bmcl.2016.07.070</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2016.07.070&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2016.07.070%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DDesign%252Band%252Bevaluation%252Bof%252B1%25252C7-naphthyridones%252Bas%252Bnovel%252BKDM5%252Binhibitors%26aulast%3DLabadie%26aufirst%3DSharada%2BS.%26date%3D2016%26volume%3D26%26issue%3D18%26spage%3D4492%26epage%3D4496" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kathy A.  Gelato</span>, <span class="hlFld-ContribAuthor ">Zaki  Shaikhibrahim</span>, <span class="hlFld-ContribAuthor ">Matthias  Ocker</span>, <span class="hlFld-ContribAuthor ">Bernard  Haendler</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting epigenetic regulators for cancer therapy: modulation of bromodomain proteins, methyltransferases, demethylases, and microRNAs. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Therapeutic Targets</span><span> <strong>2016,</strong> <em>20 </em>
                                    (7)
                                     , 783-799. <a href="https://doi.org/10.1517/14728222.2016.1134490" title="DOI URL">https://doi.org/10.1517/14728222.2016.1134490</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1517/14728222.2016.1134490&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1517%2F14728222.2016.1134490%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Therapeutic%2520Targets%26atitle%3DTargeting%252Bepigenetic%252Bregulators%252Bfor%252Bcancer%252Btherapy%25253A%252Bmodulation%252Bof%252Bbromodomain%252Bproteins%25252C%252Bmethyltransferases%25252C%252Bdemethylases%25252C%252Band%252BmicroRNAs%26aulast%3DGelato%26aufirst%3DKathy%2BA.%26date%3D2016%26date%3D2016%26volume%3D20%26issue%3D7%26spage%3D783%26epage%3D799" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Matthew G.  Quesne</span>, <span class="hlFld-ContribAuthor ">Tomasz  Borowski</span>, <span class="hlFld-ContribAuthor ">Sam P.  de Visser</span>. </span><span class="cited-content_cbyCitation_article-title">Quantum Mechanics/Molecular Mechanics Modeling of Enzymatic Processes: Caveats and Breakthroughs. </span><span class="cited-content_cbyCitation_journal-name">Chemistry - A European Journal</span><span> <strong>2016,</strong> <em>22 </em>
                                    (8)
                                     , 2562-2581. <a href="https://doi.org/10.1002/chem.201503802" title="DOI URL">https://doi.org/10.1002/chem.201503802</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/chem.201503802&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fchem.201503802%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistry%2520-%2520A%2520European%2520Journal%26atitle%3DQuantum%252BMechanics%25252FMolecular%252BMechanics%252BModeling%252Bof%252BEnzymatic%252BProcesses%25253A%252BCaveats%252Band%252BBreakthroughs%26aulast%3DQuesne%26aufirst%3DMatthew%2BG.%26date%3D2016%26date%3D2015%26volume%3D22%26issue%3D8%26spage%3D2562%26epage%3D2581" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shuo  Li</span>, <span class="hlFld-ContribAuthor ">Linlin  Wu</span>, <span class="hlFld-ContribAuthor ">Qingbing  Wang</span>, <span class="hlFld-ContribAuthor ">Yingyi  Li</span>, <span class="hlFld-ContribAuthor ">Xiaolin  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">KDM4B promotes epithelial-mesenchymal transition through up-regulation of ZEB1 in pancreatic cancer. </span><span class="cited-content_cbyCitation_journal-name">Acta Biochimica et Biophysica Sinica</span><span> <strong>2015,</strong> <em>1855 </em>, gmv107. <a href="https://doi.org/10.1093/abbs/gmv107" title="DOI URL">https://doi.org/10.1093/abbs/gmv107</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/abbs/gmv107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fabbs%2Fgmv107%26sid%3Dliteratum%253Aachs%26jtitle%3DActa%2520Biochimica%2520et%2520Biophysica%2520Sinica%26atitle%3DKDM4B%252Bpromotes%252Bepithelial-mesenchymal%252Btransition%252Bthrough%252Bup-regulation%252Bof%252BZEB1%252Bin%252Bpancreatic%252Bcancer%26aulast%3DLi%26aufirst%3DShuo%26date%3D2015%26date%3D2015%26volume%3D1855%26spage%3Dgmv107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lingling  Duan</span>, <span class="hlFld-ContribAuthor ">Ganesha  Rai</span>, <span class="hlFld-ContribAuthor ">Carlos  Roggero</span>, <span class="hlFld-ContribAuthor ">Qing-Jun  Zhang</span>, <span class="hlFld-ContribAuthor ">Qun  Wei</span>, <span class="hlFld-ContribAuthor ">Shi Hong  Ma</span>, <span class="hlFld-ContribAuthor ">Yunyun  Zhou</span>, <span class="hlFld-ContribAuthor ">John  Santoyo</span>, <span class="hlFld-ContribAuthor ">Elisabeth D.  Martinez</span>, <span class="hlFld-ContribAuthor ">Guanghua  Xiao</span>, <span class="hlFld-ContribAuthor ">Ganesh V.  Raj</span>, <span class="hlFld-ContribAuthor ">Ajit  Jadhav</span>, <span class="hlFld-ContribAuthor ">Anton  Simeonov</span>, <span class="hlFld-ContribAuthor ">David J.  Maloney</span>, <span class="hlFld-ContribAuthor ">Josep  Rizo</span>, <span class="hlFld-ContribAuthor ">Jer-Tsong  Hsieh</span>, <span class="hlFld-ContribAuthor ">Zhi-Ping  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">KDM4/JMJD2 Histone Demethylase Inhibitors Block Prostate Tumor Growth by Suppressing the Expression of AR and BMYB-Regulated Genes. </span><span class="cited-content_cbyCitation_journal-name">Chemistry & Biology</span><span> <strong>2015,</strong> <em>22 </em>
                                    (9)
                                     , 1185-1196. <a href="https://doi.org/10.1016/j.chembiol.2015.08.007" title="DOI URL">https://doi.org/10.1016/j.chembiol.2015.08.007</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.chembiol.2015.08.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.chembiol.2015.08.007%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistry%2520%2526%2520Biology%26atitle%3DKDM4%25252FJMJD2%252BHistone%252BDemethylase%252BInhibitors%252BBlock%252BProstate%252BTumor%252BGrowth%252Bby%252BSuppressing%252Bthe%252BExpression%252Bof%252BAR%252Band%252BBMYB-Regulated%252BGenes%26aulast%3DDuan%26aufirst%3DLingling%26date%3D2015%26volume%3D22%26issue%3D9%26spage%3D1185%26epage%3D1196" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tamara  Maes</span>, <span class="hlFld-ContribAuthor ">Elena  Carceller</span>, <span class="hlFld-ContribAuthor ">Jordi  Salas</span>, <span class="hlFld-ContribAuthor ">Alberto  Ortega</span>, <span class="hlFld-ContribAuthor ">Carlos  Buesa</span>. </span><span class="cited-content_cbyCitation_article-title">Advances in the development of histone lysine demethylase inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Pharmacology</span><span> <strong>2015,</strong> <em>23 </em>, 52-60. <a href="https://doi.org/10.1016/j.coph.2015.05.009" title="DOI URL">https://doi.org/10.1016/j.coph.2015.05.009</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.coph.2015.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.coph.2015.05.009%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Pharmacology%26atitle%3DAdvances%252Bin%252Bthe%252Bdevelopment%252Bof%252Bhistone%252Blysine%252Bdemethylase%252Binhibitors%26aulast%3DMaes%26aufirst%3DTamara%26date%3D2015%26volume%3D23%26spage%3D52%26epage%3D60" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ke  Liu</span>, <span class="hlFld-ContribAuthor ">Yanli  Liu</span>, <span class="hlFld-ContribAuthor ">Johnathan L.  Lau</span>, <span class="hlFld-ContribAuthor ">Jinrong  Min</span>. </span><span class="cited-content_cbyCitation_article-title">Epigenetic targets and drug discovery Part 2: Histone demethylation and DNA methylation. </span><span class="cited-content_cbyCitation_journal-name">Pharmacology & Therapeutics</span><span> <strong>2015,</strong> <em>151 </em>, 121-140. <a href="https://doi.org/10.1016/j.pharmthera.2015.04.001" title="DOI URL">https://doi.org/10.1016/j.pharmthera.2015.04.001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.pharmthera.2015.04.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.pharmthera.2015.04.001%26sid%3Dliteratum%253Aachs%26jtitle%3DPharmacology%2520%2526%2520Therapeutics%26atitle%3DEpigenetic%252Btargets%252Band%252Bdrug%252Bdiscovery%252BPart%252B2%25253A%252BHistone%252Bdemethylation%252Band%252BDNA%252Bmethylation%26aulast%3DLiu%26aufirst%3DKe%26date%3D2015%26volume%3D151%26spage%3D121%26epage%3D140" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Masaru  Katoh</span>. </span><span class="cited-content_cbyCitation_article-title">Functional proteomics of the epigenetic regulators ASXL1, ASXL2 and ASXL3: a convergence of proteomics and epigenetics for translational medicine. </span><span class="cited-content_cbyCitation_journal-name">Expert Review of Proteomics</span><span> <strong>2015,</strong> <em>12 </em>
                                    (3)
                                     , 317-328. <a href="https://doi.org/10.1586/14789450.2015.1033409" title="DOI URL">https://doi.org/10.1586/14789450.2015.1033409</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1586/14789450.2015.1033409&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1586%2F14789450.2015.1033409%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Review%2520of%2520Proteomics%26atitle%3DFunctional%252Bproteomics%252Bof%252Bthe%252Bepigenetic%252Bregulators%252BASXL1%25252C%252BASXL2%252Band%252BASXL3%25253A%252Ba%252Bconvergence%252Bof%252Bproteomics%252Band%252Bepigenetics%252Bfor%252Btranslational%252Bmedicine%26aulast%3DKatoh%26aufirst%3DMasaru%26date%3D2015%26date%3D2015%26volume%3D12%26issue%3D3%26spage%3D317%26epage%3D328" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Young-Won  Chin</span>, <span class="hlFld-ContribAuthor ">Sun-Young  Han</span>. </span><span class="cited-content_cbyCitation_article-title">KDM4 histone demethylase inhibitors for anti-cancer agents: a patent review. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Therapeutic Patents</span><span> <strong>2015,</strong> <em>25 </em>
                                    (2)
                                     , 135-144. <a href="https://doi.org/10.1517/13543776.2014.991310" title="DOI URL">https://doi.org/10.1517/13543776.2014.991310</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1517/13543776.2014.991310&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1517%2F13543776.2014.991310%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Therapeutic%2520Patents%26atitle%3DKDM4%252Bhistone%252Bdemethylase%252Binhibitors%252Bfor%252Banti-cancer%252Bagents%25253A%252Ba%252Bpatent%252Breview%26aulast%3DChin%26aufirst%3DYoung-Won%26date%3D2015%26date%3D2014%26volume%3D25%26issue%3D2%26spage%3D135%26epage%3D144" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mark A.  Wade</span>, <span class="hlFld-ContribAuthor ">Dominic  Jones</span>, <span class="hlFld-ContribAuthor ">Laura  Wilson</span>, <span class="hlFld-ContribAuthor ">Jacqueline  Stockley</span>, <span class="hlFld-ContribAuthor ">Kelly  Coffey</span>, <span class="hlFld-ContribAuthor ">Craig N.  Robson</span>, <span class="hlFld-ContribAuthor ">Luke  Gaughan</span>. </span><span class="cited-content_cbyCitation_article-title">The histone demethylase enzyme KDM3A is a key estrogen receptor regulator in breast cancer. </span><span class="cited-content_cbyCitation_journal-name">Nucleic Acids Research</span><span> <strong>2015,</strong> <em>43 </em>
                                    (1)
                                     , 196-207. <a href="https://doi.org/10.1093/nar/gku1298" title="DOI URL">https://doi.org/10.1093/nar/gku1298</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/nar/gku1298&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fnar%2Fgku1298%26sid%3Dliteratum%253Aachs%26jtitle%3DNucleic%2520Acids%2520Research%26atitle%3DThe%252Bhistone%252Bdemethylase%252Benzyme%252BKDM3A%252Bis%252Ba%252Bkey%252Bestrogen%252Breceptor%252Bregulator%252Bin%252Bbreast%252Bcancer%26aulast%3DWade%26aufirst%3DMark%2BA.%26date%3D2015%26date%3D2014%26volume%3D43%26issue%3D1%26spage%3D196%26epage%3D207" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Peter  Staller</span>. </span><span class="cited-content_cbyCitation_article-title">Inhibitors of Jumonji C-Domain Histone Demethylases. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2015,</strong>,, 439-469. <a href="https://doi.org/10.1016/B978-0-12-800206-3.00020-3" title="DOI URL">https://doi.org/10.1016/B978-0-12-800206-3.00020-3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-800206-3.00020-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-800206-3.00020-3%26sid%3Dliteratum%253Aachs%26atitle%3DInhibitors%252Bof%252BJumonji%252BC-Domain%252BHistone%252BDemethylases%26aulast%3DStaller%26aufirst%3DPeter%26date%3D2015%26spage%3D439%26epage%3D469%26pub%3DElsevier%26atitle%3DEpigenetic%252BCancer%252BTherapy%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm500249n/20150627/images/medium/jm-2014-00249n_0010.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm500249n/20150627/images/large/jm-2014-00249n_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500249n&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm500249n/20150627/images/medium/jm-2014-00249n_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm500249n/20150627/images/large/jm-2014-00249n_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. KDM4B·<b>2</b>·H3K9me3 crystal structure. (A) Electron density map for Ni(II), <b>2</b>, and H3K9me3 peptide. The 2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> electron density maps are contoured at 1.0σ. (B) KDM4A (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2OQ6">2OQ6</a>), KDM4B (this study; PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4LXL">4LXL</a>), KDM4C (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2XML">2XML</a>), and KDM4D (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4HON">4HON</a>) are colored green, orange, blue, and gray, respectively. The stick models of compound <b>2</b> (pyridine 2,4-dicarboxylic acid) and the peptide are colored orange and cyan, respectively. Oxygen, nitrogen, and nickel atoms are colored red, blue, and magenta, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm500249n/20150627/images/large/jm-2014-00249n_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500249n&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm500249n/20150627/images/medium/jm-2014-00249n_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm500249n/20150627/images/large/jm-2014-00249n_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Differential ligand binding region between KDM4A/KDM4B/KDM4C and KDM4D. Superposition of KDM4A, KDM4B, KDM4C, and KDM4D reveals two heterogeneous regions, RKDM and NIQ. Surface representation of KDM4A, KDM4B, and KDM4D shows that the RKDM and NIQ regions and a crucial isoleucine (KDMA, I71; KDM4B, I72) make several contacts with the peptide at (−2) and (+3 and +4) sites in KDM4A and KDM4B. The corresponding GEAR and HKK sites deviate away from H3K9me3. The following PDB codes were used: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2OQ6">2OQ6</a> (KDM4A), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4LXL">4LXL</a> (KDM4B), and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4HON">4HON</a> (KDM4D).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm500249n/20150627/images/large/jm-2014-00249n_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500249n&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm500249n/20150627/images/medium/jm-2014-00249n_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm500249n/20150627/images/large/jm-2014-00249n_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Compound <b>4</b> inhibits demethylase activity of KDM4A and KDM4B. (A) Demethylation of calf thymus H3 by bacteria-expressed KDM4A (upper panel) and KDM4B (lower panel) was determined in the presence of <b>4</b> by western blot analysis. The reaction mixture containing 10 μM enzyme, 100 μM inhibitor or blank buffer, and 5 μM of H3 in 50 mM HEPES, pH 7.5, 1 mM AKG, 2 mM ascorbate, and 50 μM Fe(II) was incubated at 37 °C for 30 min, followed by western blot analysis. H3 lysine modifications were probed with H3K9me3 and H3K36me3 antisera, respectively. (B) Inhibition kinetics of KDM4A/KDM4B demethylation activity by <b>4</b>. The inset in each panel shows the double-reciprocal form, where the 1/relative activity is plotted versus 1/[H3K9me3] at various fixed concentrations of the inhibitor.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm500249n/20150627/images/large/jm-2014-00249n_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500249n&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm500249n/20150627/images/medium/jm-2014-00249n_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm500249n/20150627/images/large/jm-2014-00249n_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. KDM4A and KDM4B are highly expressed in prostate cancer cells. (A) Expression values of KDM4A and KDM4B in normal prostate glands and tumor tissues from the selected studies were obtained from the Oncomine (Compendia Bioscience, Ann Arbor, MI, USA) database (<a href="http://www.oncomine.org/" class="extLink">http://www.oncomine.org/</a>). The box and whisker plots show a box that encompasses the 25th–75th percentiles, the median as a line within the box, and the 10th and 90th percentiles as error bars. <i>p</i> values were determined by one-tailed Student’s <i>t</i> test and were calculated on the basis of the comparison of normal vs cancer; normal vs primary sites; or normal vs metastasis. (B) qRT-PCR analysis of KDM4A and KDM4B expression in normal prostate primary cells (PrEC), normal prostate epithelial cell lines (RWPE-1 and PNT2), and prostate cancer cell lines (LNCaP, C4-2, C4-2B, CWR22rv1, CWR-R1, VCaP, DU145, and PC3). Asterisks indicate significant overexpression compared to that in normal cells.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm500249n/20150627/images/large/jm-2014-00249n_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500249n&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm500249n/20150627/images/medium/jm-2014-00249n_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm500249n/20150627/images/large/jm-2014-00249n_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (A) KDM4A and KDM4B are crucial for the growth of LNCaP cells. LNCaP cells were infected with lentivirus encoding control shRNA (control), sh-KDM4A, or sh-KDM4B, as indicated (left panel). qRT-PCR analysis was performed to evaluate the expression of KDM4s (right panel). (B) H3K9me3 levels in KDM4A and KDM4B knockdown cells. LNCaP cells were infected by lentivius carrying control vector, sh-KDM4A, or sh-KDM4B for 3 days. The KDM4A, KDM4B, and H3K9me3 signals were detected in cell lysates by western blot analysis as indicated. (C) Flow cytometry analysis of the DNA content in sh-KDM4A and sh-KDM4B LNCaP cells.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm500249n/20150627/images/large/jm-2014-00249n_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500249n&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm500249n/20150627/images/medium/jm-2014-00249n_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm500249n/20150627/images/large/jm-2014-00249n_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Compound <b>4</b> exhibits anticancer effects on LNCaP cells. (A) Treatment of LNCaP or normal prostate (PNT2 and RWPE1) cells with different concentrations of inhibitor <b>4</b> over 5 days shows inhibitor <b>4</b>’s selective anticancer properties toward LNCaP cells. (B) Inhibition of viability by inhibitor <b>4</b> in normal prostate and LNCaP cells. LNCaP and normal prostate (PNT2 and RWPE1) cells were treated with inhibitor <b>4</b> for 3 days, and viability was measured by viable cell count. Standard deviation is derived from biological triplicates. The H3K9me3 signal (lower panel) was detected in cell lysates treated with different concentrations (1–100 μM) of inhibitor <b>4</b> for 24 h, followed by estern blot analysis using anti-H3K9me3 antibody. The different concentrations (1–100 μM) of inhibitor <b>4</b> are indicated below the blot. The level of H3K9me3 was detected and quantified by AlphaView SA (Cell Biosciences Inc.). The level of H3K9me3 is shown by bar graph. (C) Flow cytometry analysis of the DNA content in LNCaP cells treated with DMSO (mock) or 50 μM inhibitor <b>4</b> for 3 days.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm500249n/20150627/images/large/jm-2014-00249n_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500249n&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm500249n/20150627/images/medium/jm-2014-00249n_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm500249n/20150627/images/large/jm-2014-00249n_0008.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Microarray analysis of LNCaP cells treated with inhibitor <b>4</b> (50 μM) compared to that of mock-treated cells. (A) DAVID functional annotation of the genes that showed 2-fold alterations in expression. GO terms associated with the altered genes that show statistically strong enrichment with low <i>p</i> values are listed. The numbers next to each bar in the graph indicate the gene count for each pathway. Percent of hits indicates the percentage of genes that are altered in each GO classification. (B) Expression of tumor suppressors and oncogenes that are up- and downregulated in the inhibitor-treated cells, respectively. (C) Venn diagram of the overlap of the inhibitor-altered genes with androgen-responsive genes.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm500249n/20150627/images/large/jm-2014-00249n_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500249n&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm500249n/20150627/images/medium/jm-2014-00249n_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm500249n/20150627/images/large/jm-2014-00249n_0009.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. AR-signature genes are differentially expressed in inhibitor <b>4</b>-treated LNCaP cells.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-14/jm500249n/20150627/images/large/jm-2014-00249n_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm500249n&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i28">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_65841" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_65841" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 34 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Shi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lan, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mulligan, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whetstine, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cole, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casero, R. A.</span><span> </span><span class="NLM_article-title">Histone demethylation mediated by the nuclear amine oxidase homolog LSD1</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">119</span><span class="NLM_x">, </span> <span class="NLM_fpage">941</span><span class="NLM_x">–</span> <span class="NLM_lpage">53</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1a&amp;dbid=16&amp;doi=10.1021%2Fjm500249n&amp;key=10.1016%2Fj.cell.2004.12.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1a&amp;dbid=8&amp;doi=10.1021%2Fjm500249n&amp;key=15620353" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1a&amp;dbid=32&amp;doi=10.1021%2Fjm500249n&amp;key=1%3ACAS%3A528%3ADC%252BD2MXltlSitg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2004&pages=941-53&author=Y.+Shiauthor=F.+Lanauthor=C.+Matsonauthor=P.+Mulliganauthor=J.+R.+Whetstineauthor=P.+A.+Coleauthor=R.+A.+Casero&title=Histone+demethylation+mediated+by+the+nuclear+amine+oxidase+homolog+LSD1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1aR"><div class="casContent"><span class="casTitleNuber">1a</span><div class="casTitle"><span class="NLM_cas:atitle">Histone demethylation mediated by the nuclear amine oxidase homolog LSD1</span></div><div class="casAuthors">Shi, Yujiang; Lan, Fei; Matson, Caitlin; Mulligan, Peter; Whetstine, Johnathan R.; Cole, Philip A.; Casero, Robert A.; Shi, Yang</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">941-953</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Posttranslational modifications of histone N-terminal tails impact chromatin structure and gene transcription.  While the extent of histone acetylation is detd. by both acetyltransferases and deacetylases, it has been unclear whether histone methylation is also regulated by enzymes with opposing activities.  Here, we provide evidence that LSD1 (KIAA0601), a nuclear homolog of amine oxidases, functions as a histone demethylase and transcriptional corepressor.  LSD1 specifically demethylates histone H3 lysine 4, which is linked to active transcription.  Lysine demethylation occurs via an oxidn. reaction that generates formaldehyde.  Importantly, RNAi inhibition of LSD1 causes an increase in H3 lysine 4 methylation and concomitant derepression of target genes, suggesting that LSD1 represses transcription via histone demethylation.  The results thus identify a histone demethylase conserved from S. pombe to human and reveal dynamic regulation of histone methylation by both histone methylases and demethylases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXHr-8kwGdXrVg90H21EOLACvtfcHk0liliT4RCTslXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXltlSitg%253D%253D&md5=3474569204255e90a832667ba654748b</span></div><a href="/servlet/linkout?suffix=cit1a&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2004.12.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2004.12.012%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DY.%26aulast%3DLan%26aufirst%3DF.%26aulast%3DMatson%26aufirst%3DC.%26aulast%3DMulligan%26aufirst%3DP.%26aulast%3DWhetstine%26aufirst%3DJ.%2BR.%26aulast%3DCole%26aufirst%3DP.%2BA.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26atitle%3DHistone%2520demethylation%2520mediated%2520by%2520the%2520nuclear%2520amine%2520oxidase%2520homolog%2520LSD1%26jtitle%3DCell%26date%3D2004%26volume%3D119%26spage%3D941%26epage%3D53" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit1b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Kouzarides, T.</span><span> </span><span class="NLM_article-title">Chromatin modifications and their function</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">128</span><span class="NLM_x">, </span> <span class="NLM_fpage">693</span><span class="NLM_x">–</span> <span class="NLM_lpage">705</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1b&amp;dbid=16&amp;doi=10.1021%2Fjm500249n&amp;key=10.1016%2Fj.cell.2007.02.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1b&amp;dbid=8&amp;doi=10.1021%2Fjm500249n&amp;key=17320507" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1b&amp;dbid=32&amp;doi=10.1021%2Fjm500249n&amp;key=1%3ACAS%3A528%3ADC%252BD2sXis12ju7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2007&pages=693-705&author=T.+Kouzarides&title=Chromatin+modifications+and+their+function"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1bR"><div class="casContent"><span class="casTitleNuber">1b</span><div class="casTitle"><span class="NLM_cas:atitle">Chromatin modifications and their function</span></div><div class="casAuthors">Kouzarides, Tony</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">693-705</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The surface of nucleosomes is studded with a multiplicity of modifications.  At least eight different classes have been characterized to date and many different sites have been identified for each class.  Operationally, modifications function either by disrupting chromatin contacts or by affecting the recruitment of nonhistone proteins to chromatin.  Their presence on histones can dictate the higher-order chromatin structure in which DNA is packaged and can orchestrate the ordered recruitment of enzyme complexes to manipulate DNA.  In this way, histone modifications have the potential to influence many fundamental biol. processes, some of which may be epigenetically inherited.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAH9AX2dGicbVg90H21EOLACvtfcHk0ljAb7E5Prxnkw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXis12ju7Y%253D&md5=c5995fe68fd9473f4268d59c01d3b9de</span></div><a href="/servlet/linkout?suffix=cit1b&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2007.02.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2007.02.005%26sid%3Dliteratum%253Aachs%26aulast%3DKouzarides%26aufirst%3DT.%26atitle%3DChromatin%2520modifications%2520and%2520their%2520function%26jtitle%3DCell%26date%3D2007%26volume%3D128%26spage%3D693%26epage%3D705" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit1c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Mosammaparast, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Y.</span><span> </span><span class="NLM_article-title">Reversal of histone methylation: biochemical and molecular mechanisms of histone demethylases</span> <span class="citation_source-journal">Annu. Rev. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">79</span><span class="NLM_x">, </span> <span class="NLM_fpage">155</span><span class="NLM_x">–</span> <span class="NLM_lpage">79</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1c&amp;dbid=16&amp;doi=10.1021%2Fjm500249n&amp;key=10.1146%2Fannurev.biochem.78.070907.103946" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1c&amp;dbid=8&amp;doi=10.1021%2Fjm500249n&amp;key=20373914" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1c&amp;dbid=32&amp;doi=10.1021%2Fjm500249n&amp;key=1%3ACAS%3A528%3ADC%252BC3cXpslShsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=2010&pages=155-79&author=N.+Mosammaparastauthor=Y.+Shi&title=Reversal+of+histone+methylation%3A+biochemical+and+molecular+mechanisms+of+histone+demethylases"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1cR"><div class="casContent"><span class="casTitleNuber">1c</span><div class="casTitle"><span class="NLM_cas:atitle">Reversal of histone methylation: biochemical and molecular mechanisms of histone demethylases</span></div><div class="casAuthors">Mosammaparast, Nima; Shi, Yang</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biochemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">155-179</span>CODEN:
                <span class="NLM_cas:coden">ARBOAW</span>;
        ISSN:<span class="NLM_cas:issn">0066-4154</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  The importance of histone methylation in gene regulation was suggested over 40 years ago.  Yet, the dynamic nature of this histone modification was recognized only recently, with the discovery of the first histone demethylase nearly five years ago.  Since then, our insight into the mechanisms, structures, and macromol. complexes of these enzymes has grown exponentially.  Overall, the evidence strongly supports a key role for histone demethylases in eukaryotic transcription and other chromatin-dependent processes.  Here, we examine these and related facets of histone demethylases discovered to date, focusing on their biochem., structure, and enzymol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGquQJN8-1hp07Vg90H21EOLACvtfcHk0ljAb7E5Prxnkw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXpslShsbk%253D&md5=781bebbd65cc2f68ebe31490e93da09c</span></div><a href="/servlet/linkout?suffix=cit1c&amp;dbid=16384&amp;doi=10.1146%2Fannurev.biochem.78.070907.103946&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.biochem.78.070907.103946%26sid%3Dliteratum%253Aachs%26aulast%3DMosammaparast%26aufirst%3DN.%26aulast%3DShi%26aufirst%3DY.%26atitle%3DReversal%2520of%2520histone%2520methylation%253A%2520biochemical%2520and%2520molecular%2520mechanisms%2520of%2520histone%2520demethylases%26jtitle%3DAnnu.%2520Rev.%2520Biochem.%26date%3D2010%26volume%3D79%26spage%3D155%26epage%3D79" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Breccia, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alimena, G.</span><span> </span><span class="NLM_article-title">NFκB as a potential therapeutic target in myelodysplastic syndromes and acute myeloid leukemia</span> <span class="citation_source-journal">Expert Opin. Ther. Targets</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">1157</span><span class="NLM_x">–</span> <span class="NLM_lpage">76</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2a&amp;dbid=16&amp;doi=10.1021%2Fjm500249n&amp;key=10.1517%2F14728222.2010.522570" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2a&amp;dbid=8&amp;doi=10.1021%2Fjm500249n&amp;key=20858024" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2a&amp;dbid=32&amp;doi=10.1021%2Fjm500249n&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht12ns7zE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2010&pages=1157-76&author=M.+Brecciaauthor=G.+Alimena&title=NF%CE%BAB+as+a+potential+therapeutic+target+in+myelodysplastic+syndromes+and+acute+myeloid+leukemia"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2aR"><div class="casContent"><span class="casTitleNuber">2a</span><div class="casTitle"><span class="NLM_cas:atitle">NF-κB as a potential therapeutic target in myelodysplastic syndromes and acute myeloid leukemia</span></div><div class="casAuthors">Breccia, Massimo; Alimena, Giuliana</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Targets</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1157-1176</span>CODEN:
                <span class="NLM_cas:coden">EOTTAO</span>;
        ISSN:<span class="NLM_cas:issn">1472-8222</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Importance of the field: The inactive NF-κB-inhibitor of NF-κB (IκB) complex is activated by stimuli including pro-inflammatory cytokines, mitogens, growth factors and stress-inducing agents.  The release of NF-κB facilitates its translocation to the nucleus, where it promotes cell survival by initiating transcription of genes encoding stress-response enzymes, cell-adhesion mols., pro-inflammatory cytokines and anti-apoptotic proteins.  NF-κB and assocd. regulatory factors (IκB kinase subunits and bcl-3) are implicated in hematol. and solid tumor malignancies.  NF-κB appears to be involved in cell proliferation control, apoptosis control, angiogenesis promotion and possibly regulation of diffusion of metastases.  There are several reports that inhibition of NF-κB as a therapeutic target may have a role in tumor cell death or growth inhibition.  Area covered in this review: We review data about inhibition of NF-κB in acute myeloid leukemia (AML) and myelodysplastic syndromes (MDS).  We describe the mol. mechanisms underlying NF-κB deregulation in these haematol. malignancies.  What the reader will gain: Constitutive activation of NF-κB in the nucleus has been reported in some varieties of MDS/AML.  The in vitro and in vivo results of NF-κB inhibition in myeloid malignancies are highlighted.  Take home message: NF-κB selective inhibitory drugs may be useful, either as single agents or assocd. with conventional chemotherapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpvyIKKXSfGnLVg90H21EOLACvtfcHk0lg-D-inV4UkYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht12ns7zE&md5=aea9295b4ea76817d6b81b582146b75d</span></div><a href="/servlet/linkout?suffix=cit2a&amp;dbid=16384&amp;doi=10.1517%2F14728222.2010.522570&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F14728222.2010.522570%26sid%3Dliteratum%253Aachs%26aulast%3DBreccia%26aufirst%3DM.%26aulast%3DAlimena%26aufirst%3DG.%26atitle%3DNF%25CE%25BAB%2520as%2520a%2520potential%2520therapeutic%2520target%2520in%2520myelodysplastic%2520syndromes%2520and%2520acute%2520myeloid%2520leukemia%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Targets%26date%3D2010%26volume%3D14%26spage%3D1157%26epage%3D76" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit2b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Rodriguez-Paredes, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Esteller, M.</span><span> </span><span class="NLM_article-title">Cancer epigenetics reaches mainstream oncology</span> <span class="citation_source-journal">Nat. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">330</span><span class="NLM_x">–</span> <span class="NLM_lpage">9</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2b&amp;dbid=16&amp;doi=10.1021%2Fjm500249n&amp;key=10.1038%2Fnm.2305" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2b&amp;dbid=8&amp;doi=10.1021%2Fjm500249n&amp;key=21386836" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2b&amp;dbid=32&amp;doi=10.1021%2Fjm500249n&amp;key=1%3ACAS%3A528%3ADC%252BC3MXivVGgtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2011&pages=330-9&author=M.+Rodriguez-Paredesauthor=M.+Esteller&title=Cancer+epigenetics+reaches+mainstream+oncology"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2bR"><div class="casContent"><span class="casTitleNuber">2b</span><div class="casTitle"><span class="NLM_cas:atitle">Cancer epigenetics reaches mainstream oncology</span></div><div class="casAuthors">Rodriguez-Paredes, Manuel; Esteller, Manel</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">330-339</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Epigenetics is one of the most promising and expanding fields in the current biomedical research landscape.  Since the inception of epigenetics in the 1940s, the discoveries regarding its implications in normal and disease biol. have not stopped, compiling a vast amt. of knowledge in the past decade.  The field has moved from just one recognized marker, DNA methylation, to a variety of others, including a wide spectrum of histone modifications.  From the methodol. standpoint, the successful initial single gene candidate approaches have been complemented by the current comprehensive epigenomic approaches that allow the interrogation of genomes to search for translational applications in an unbiased manner.  Most important, the discovery of mutations in the epigenetic machinery and the approval of the first epigenetic drugs for the treatment of subtypes of leukemias and lymphomas has been an eye-opener for many biomedical scientists and clinicians.  Herein, we will summarize the progress in the field of cancer epigenetics research that has reached mainstream oncol. in the development of new biomarkers of the disease and new pharmacol. strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxyADVbZ4KHrVg90H21EOLACvtfcHk0lg-D-inV4UkYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXivVGgtb8%253D&md5=07b6faaddb8bce884261990dbc927331</span></div><a href="/servlet/linkout?suffix=cit2b&amp;dbid=16384&amp;doi=10.1038%2Fnm.2305&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.2305%26sid%3Dliteratum%253Aachs%26aulast%3DRodriguez-Paredes%26aufirst%3DM.%26aulast%3DEsteller%26aufirst%3DM.%26atitle%3DCancer%2520epigenetics%2520reaches%2520mainstream%2520oncology%26jtitle%3DNat.%2520Med.%26date%3D2011%26volume%3D17%26spage%3D330%26epage%3D9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit2c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Chi, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allis, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, G. G.</span><span> </span><span class="NLM_article-title">Covalent histone modifications--miswritten, misinterpreted and mis-erased in human cancers</span> <span class="citation_source-journal">Nat. Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">457</span><span class="NLM_x">–</span> <span class="NLM_lpage">69</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2c&amp;dbid=16&amp;doi=10.1021%2Fjm500249n&amp;key=10.1038%2Fnrc2876" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2c&amp;dbid=8&amp;doi=10.1021%2Fjm500249n&amp;key=20574448" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2c&amp;dbid=32&amp;doi=10.1021%2Fjm500249n&amp;key=1%3ACAS%3A528%3ADC%252BC3cXnvVShurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2010&pages=457-69&author=P.+Chiauthor=C.+D.+Allisauthor=G.+G.+Wang&title=Covalent+histone+modifications%2D%2Dmiswritten%2C+misinterpreted+and+mis-erased+in+human+cancers"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2cR"><div class="casContent"><span class="casTitleNuber">2c</span><div class="casTitle"><span class="NLM_cas:atitle">Covalent histone modifications - miswritten, misinterpreted and mis-erased in human cancers</span></div><div class="casAuthors">Chi, Ping; Allis, C. David; Wang, Gang Greg</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">457-469</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Post-translational modification of histones provides an important regulatory platform for processes such as gene transcription and DNA damage repair.  It has become increasingly apparent that the misregulation of histone modification, which is caused by the deregulation of factors that mediate the modification installation, removal and/or interpretation, actively contributes to human cancer.  In this review, we summarize recent advances in understanding the interpretation of certain histone methylations by plant homeodomain finger-contg. proteins, and how misreading, miswriting and mis-erasing of histone methylation marks can be assocd. with oncogenesis and progression.  These observations provide us with a greater mechanistic understanding of epigenetic alterations in human cancers and might also help direct new therapeutic interventions in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGMmMSNkuTSrVg90H21EOLACvtfcHk0lg-D-inV4UkYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXnvVShurc%253D&md5=08cd0d4dcd62c2fc90f4768a8eb29e3c</span></div><a href="/servlet/linkout?suffix=cit2c&amp;dbid=16384&amp;doi=10.1038%2Fnrc2876&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2876%26sid%3Dliteratum%253Aachs%26aulast%3DChi%26aufirst%3DP.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26aulast%3DWang%26aufirst%3DG.%2BG.%26atitle%3DCovalent%2520histone%2520modifications--miswritten%252C%2520misinterpreted%2520and%2520mis-erased%2520in%2520human%2520cancers%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2010%26volume%3D10%26spage%3D457%26epage%3D69" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Kampranis, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsichlis, P. N.</span><span> </span><span class="NLM_article-title">Histone demethylases and cancer</span> <span class="citation_source-journal">Adv. Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">102</span><span class="NLM_x">, </span> <span class="NLM_fpage">103</span><span class="NLM_x">–</span> <span class="NLM_lpage">69</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2009&pages=103-69&author=S.+C.+Kampranisauthor=P.+N.+Tsichlis&title=Histone+demethylases+and+cancer"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3a&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKampranis%26aufirst%3DS.%2BC.%26aulast%3DTsichlis%26aufirst%3DP.%2BN.%26atitle%3DHistone%2520demethylases%2520and%2520cancer%26jtitle%3DAdv.%2520Cancer%2520Res.%26date%3D2009%26volume%3D102%26spage%3D103%26epage%3D69" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit3b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Wang, L.-Y.; Guo, W.; Kim, K.; Pochampalli, M.; Hung, C.-L.; Izumiya, Y.; Kung, H.-J.</span><span> </span><span class="NLM_article-title">Histone demethylases in prostate cancer</span>. In  <span class="citation_source-book">Nuclear Signaling Pathways and Targeting Transcription in Cancer</span>; <span class="NLM_contrib-group">Kumer, R.</span>, Ed; <span class="NLM_publisher-name">Springer Science+Business Media</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">2013</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=L.-Y.+Wang&author=W.+Guo&author=K.+Kim&author=M.+Pochampalli&author=C.-L.+Hung&author=Y.+Izumiya&author=H.-J.+Kungauthor=R.+Kumer&title=Nuclear+Signaling+Pathways+and+Targeting+Transcription+in+Cancer"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3b&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.-Y.%26atitle%3DHistone%2520demethylases%2520in%2520prostate%2520cancer%26btitle%3DNuclear%2520Signaling%2520Pathways%2520and%2520Targeting%2520Transcription%2520in%2520Cancer%26aulast%3DKumer%26aufirst%3DR.%26pub%3DSpringer%2520Science%252BBusiness%2520Media%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Katoh, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Katoh, M.</span><span> </span><span class="NLM_article-title">Identification and characterization of JMJD2 family genes in silico</span> <span class="citation_source-journal">Int. J. Oncol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">1623</span><span class="NLM_x">–</span> <span class="NLM_lpage">8</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4a&amp;dbid=8&amp;doi=10.1021%2Fjm500249n&amp;key=15138608" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4a&amp;dbid=32&amp;doi=10.1021%2Fjm500249n&amp;key=1%3ACAS%3A528%3ADC%252BD2cXkvFaisbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2004&pages=1623-8&author=M.+Katohauthor=M.+Katoh&title=Identification+and+characterization+of+JMJD2+family+genes+in+silico"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4aR"><div class="casContent"><span class="casTitleNuber">4a</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and characterization of JMJD2 family genes in silico</span></div><div class="casAuthors">Katoh, Masuko; Katoh, Masaru</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Oncology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1623-1628</span>CODEN:
                <span class="NLM_cas:coden">IJONES</span>;
        ISSN:<span class="NLM_cas:issn">1019-6439</span>.
    
            (<span class="NLM_cas:orgname">International Journal of Oncology</span>)
        </div><div class="casAbstract">JMJD1 family genes, including JMJD1 (TSGA), TRIP8, and C5orf7 (5qNCA), encode nuclear proteins with Jumonji C (JmjC), TRI8H1 and TRI8H2 domains.  Here, we identified JMJD2 family genes.  Human genes corresponding to KIAA0677, KIAA0876, KIAA0780 and FLJ10251 cDNAs were designated JMJD2A, JMJD2B, JMJD2C, and JMJD2D, resp.  In addn., JMJD2D homologous genes within human genome sequences AP002383.3 and AP001264.4 were designated JMJD2E and JMJD2F, resp.  JMJD2A gene was mapped to human chromosome 1p34.1, JMJD2B gene to 19p13.3, and JMJD2C gene to 9p24.1.  JMJD2D, JMJD2E and JMJD2F genes were clustered at human chromosome 11q21.  Coding region of JMJD2A, JMJD2B, JMJD2C genes was located on multiple exons, while that of JMJD2D, JMJD2E and JMJD2F genes was located on a single exon.  These facts strongly indicate that JMJD2D, JMJD2E and JMJD2F genes are locally triplicated retrotransposed elements derived from JMJD2 family gene.  JD2H domain with C2HC2HC2- and C5HC2-type Cys (His) clusters was identified as the region conserved among JMJD2A (1064 aa), JMJD2B (1096 aa), and JMJD2C (1056 aa) proteins.  JMJD2A, JMJD2B and JMJD2C consist of JmjN, JmjC, JD2H, and two TUDOR domains, while JMJD2D (523 aa), JMJD2E (506 aa) and JMJD2F (638 aa) consist of JmjN and JmjC domains.  JMJD2 family proteins were classified into one group with JD2H and TUDOR domains and another group without JD2H or TUDOR domains.  Because JMJD2C gene (also known as GASC1 gene) is amplified in esophageal squamous cell carcinoma (ESCC), JMJD2 family genes are cancer-assocd. genes.  This is the first report on identification and characterization of human JMJD2 gene family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2QrOZpXEa9rVg90H21EOLACvtfcHk0ljtzxyVR5z6yg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXkvFaisbo%253D&md5=b6ab9ea6d3aebb1c242870726103a449</span></div><a href="/servlet/linkout?suffix=cit4a&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKatoh%26aufirst%3DM.%26aulast%3DKatoh%26aufirst%3DM.%26atitle%3DIdentification%2520and%2520characterization%2520of%2520JMJD2%2520family%2520genes%2520in%2520silico%26jtitle%3DInt.%2520J.%2520Oncol.%26date%3D2004%26volume%3D24%26spage%3D1623%26epage%3D8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit4b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Berry, W. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janknecht, R.</span><span> </span><span class="NLM_article-title">KDM4/JMJD2 histone demethylases: epigenetic regulators in cancer cells</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">73</span><span class="NLM_x">, </span> <span class="NLM_fpage">2936</span><span class="NLM_x">–</span> <span class="NLM_lpage">42</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2013&pages=2936-42&author=W.+L.+Berryauthor=R.+Janknecht&title=KDM4%2FJMJD2+histone+demethylases%3A+epigenetic+regulators+in+cancer+cells"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4b&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBerry%26aufirst%3DW.%2BL.%26aulast%3DJanknecht%26aufirst%3DR.%26atitle%3DKDM4%252FJMJD2%2520histone%2520demethylases%253A%2520epigenetic%2520regulators%2520in%2520cancer%2520cells%26jtitle%3DCancer%2520Res.%26date%3D2013%26volume%3D73%26spage%3D2936%26epage%3D42" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Shin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janknecht, R.</span><span> </span><span class="NLM_article-title">Activation of androgen receptor by histone demethylases JMJD2A and JMJD2D</span> <span class="citation_source-journal">Biochem. Biophys. Res. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">359</span><span class="NLM_x">, </span> <span class="NLM_fpage">742</span><span class="NLM_x">–</span> <span class="NLM_lpage">6</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=359&publication_year=2007&pages=742-6&author=S.+Shinauthor=R.+Janknecht&title=Activation+of+androgen+receptor+by+histone+demethylases+JMJD2A+and+JMJD2D"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5a&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DShin%26aufirst%3DS.%26aulast%3DJanknecht%26aufirst%3DR.%26atitle%3DActivation%2520of%2520androgen%2520receptor%2520by%2520histone%2520demethylases%2520JMJD2A%2520and%2520JMJD2D%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2007%26volume%3D359%26spage%3D742%26epage%3D6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit5b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Wissmann, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yin, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muller, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greschik, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fodor, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenuwein, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vogler, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneider, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gunther, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buettner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metzger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schule, R.</span><span> </span><span class="NLM_article-title">Cooperative demethylation by JMJD2C and LSD1 promotes androgen receptor-dependent gene expression</span> <span class="citation_source-journal">Nat. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">347</span><span class="NLM_x">–</span> <span class="NLM_lpage">53</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5b&amp;dbid=16&amp;doi=10.1021%2Fjm500249n&amp;key=10.1038%2Fncb1546" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5b&amp;dbid=8&amp;doi=10.1021%2Fjm500249n&amp;key=17277772" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5b&amp;dbid=32&amp;doi=10.1021%2Fjm500249n&amp;key=1%3ACAS%3A528%3ADC%252BD2sXit12rsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2007&pages=347-53&author=M.+Wissmannauthor=N.+Yinauthor=J.+M.+Mullerauthor=H.+Greschikauthor=B.+D.+Fodorauthor=T.+Jenuweinauthor=C.+Voglerauthor=R.+Schneiderauthor=T.+Guntherauthor=R.+Buettnerauthor=E.+Metzgerauthor=R.+Schule&title=Cooperative+demethylation+by+JMJD2C+and+LSD1+promotes+androgen+receptor-dependent+gene+expression"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5bR"><div class="casContent"><span class="casTitleNuber">5b</span><div class="casTitle"><span class="NLM_cas:atitle">Cooperative demethylation by JMJD2C and LSD1 promotes androgen receptor-dependent gene expression</span></div><div class="casAuthors">Wissmann, Melanie; Yin, Na; Mueller, Judith M.; Greschik, Holger; Fodor, Barna D.; Jenuwein, Thomas; Vogler, Christine; Schneider, Robert; Guenther, Thomas; Buettner, Reinhard; Metzger, Eric; Schuele, Roland</div><div class="citationInfo"><span class="NLM_cas:title">Nature Cell Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">347-353</span>CODEN:
                <span class="NLM_cas:coden">NCBIFN</span>;
        ISSN:<span class="NLM_cas:issn">1465-7392</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Posttranslational modifications of histones, such as methylation, regulate chromatin structure and gene expression.  Recently, lysine-specific demethylase 1 (LSD1), the first histone demethylase, was identified.  LSD1 interacts with the androgen receptor and promotes androgen-dependent transcription of target genes by ligand-induced demethylation of mono- and dimethylated histone H3 at Lys 9 (H3K9) only.  Here, we identify the Jumonji C (JMJC) domain-contg. protein JMJD2C as the first histone tridemethylase regulating androgen receptor function.  JMJD2C interacts with androgen receptor in vitro and in vivo.  Assembly of ligand-bound androgen receptor and JMJD2C on androgen receptor-target genes results in demethylation of tri-Me H3K9 and in stimulation of androgen receptor-dependent transcription.  Conversely, knockdown of JMJD2C inhibits androgen-induced removal of tri-Me H3K9, transcriptional activation and tumor cell proliferation.  Importantly, JMJD2C colocalizes with androgen receptor and LSD1 in normal prostate and in prostate carcinomas.  JMJD2C and LSD1 interact and both demethylases cooperatively stimulate androgen receptor-dependent gene transcription.  In addn., androgen receptor, JMJD2C and LSD1 assemble on chromatin to remove Me groups from mono, di and trimethylated H3K9.  Thus, our data suggest that specific gene regulation requires the assembly and coordinate action of demethylases with distinct substrate specificities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2b5sXBcoCRrVg90H21EOLACvtfcHk0lgBK5EazZinMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXit12rsrw%253D&md5=d1f7ec6fbaadf301372ebe644a2e978a</span></div><a href="/servlet/linkout?suffix=cit5b&amp;dbid=16384&amp;doi=10.1038%2Fncb1546&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncb1546%26sid%3Dliteratum%253Aachs%26aulast%3DWissmann%26aufirst%3DM.%26aulast%3DYin%26aufirst%3DN.%26aulast%3DMuller%26aufirst%3DJ.%2BM.%26aulast%3DGreschik%26aufirst%3DH.%26aulast%3DFodor%26aufirst%3DB.%2BD.%26aulast%3DJenuwein%26aufirst%3DT.%26aulast%3DVogler%26aufirst%3DC.%26aulast%3DSchneider%26aufirst%3DR.%26aulast%3DGunther%26aufirst%3DT.%26aulast%3DBuettner%26aufirst%3DR.%26aulast%3DMetzger%26aufirst%3DE.%26aulast%3DSchule%26aufirst%3DR.%26atitle%3DCooperative%2520demethylation%2520by%2520JMJD2C%2520and%2520LSD1%2520promotes%2520androgen%2520receptor-dependent%2520gene%2520expression%26jtitle%3DNat.%2520Cell%2520Biol.%26date%3D2007%26volume%3D9%26spage%3D347%26epage%3D53" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit5c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Coffey, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rogerson, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ryan-Munden, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alkharaif, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stockley, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heer, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sahadevan, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Neill, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Darby, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Staller, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mantilla, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gaughan, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robson, C. N.</span><span> </span><span class="NLM_article-title">The lysine demethylase, KDM4B, is a key molecule in androgen receptor signalling and turnover</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">4433</span><span class="NLM_x">–</span> <span class="NLM_lpage">46</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2013&pages=4433-46&author=K.+Coffeyauthor=L.+Rogersonauthor=C.+Ryan-Mundenauthor=D.+Alkharaifauthor=J.+Stockleyauthor=R.+Heerauthor=K.+Sahadevanauthor=D.+O%E2%80%99Neillauthor=D.+Jonesauthor=S.+Darbyauthor=P.+Stallerauthor=A.+Mantillaauthor=L.+Gaughanauthor=C.+N.+Robson&title=The+lysine+demethylase%2C+KDM4B%2C+is+a+key+molecule+in+androgen+receptor+signalling+and+turnover"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5c&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCoffey%26aufirst%3DK.%26aulast%3DRogerson%26aufirst%3DL.%26aulast%3DRyan-Munden%26aufirst%3DC.%26aulast%3DAlkharaif%26aufirst%3DD.%26aulast%3DStockley%26aufirst%3DJ.%26aulast%3DHeer%26aufirst%3DR.%26aulast%3DSahadevan%26aufirst%3DK.%26aulast%3DO%25E2%2580%2599Neill%26aufirst%3DD.%26aulast%3DJones%26aufirst%3DD.%26aulast%3DDarby%26aufirst%3DS.%26aulast%3DStaller%26aufirst%3DP.%26aulast%3DMantilla%26aufirst%3DA.%26aulast%3DGaughan%26aufirst%3DL.%26aulast%3DRobson%26aufirst%3DC.%2BN.%26atitle%3DThe%2520lysine%2520demethylase%252C%2520KDM4B%252C%2520is%2520a%2520key%2520molecule%2520in%2520androgen%2520receptor%2520signalling%2520and%2520turnover%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2013%26volume%3D41%26spage%3D4433%26epage%3D46" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Shi, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yi, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Han, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xuan, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yao, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shang, Y.</span><span> </span><span class="NLM_article-title">Histone demethylase JMJD2B coordinates H3K4/H3K9 methylation and promotes hormonally responsive breast carcinogenesis</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">108</span><span class="NLM_x">, </span> <span class="NLM_fpage">7541</span><span class="NLM_x">–</span> <span class="NLM_lpage">6</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2011&pages=7541-6&author=L.+Shiauthor=L.+Sunauthor=Q.+Liauthor=J.+Liangauthor=W.+Yuauthor=X.+Yiauthor=X.+Yangauthor=Y.+Liauthor=X.+Hanauthor=Y.+Zhangauthor=C.+Xuanauthor=Z.+Yaoauthor=Y.+Shang&title=Histone+demethylase+JMJD2B+coordinates+H3K4%2FH3K9+methylation+and+promotes+hormonally+responsive+breast+carcinogenesis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6a&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DL.%26aulast%3DSun%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DLiang%26aufirst%3DJ.%26aulast%3DYu%26aufirst%3DW.%26aulast%3DYi%26aufirst%3DX.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DHan%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DXuan%26aufirst%3DC.%26aulast%3DYao%26aufirst%3DZ.%26aulast%3DShang%26aufirst%3DY.%26atitle%3DHistone%2520demethylase%2520JMJD2B%2520coordinates%2520H3K4%252FH3K9%2520methylation%2520and%2520promotes%2520hormonally%2520responsive%2520breast%2520carcinogenesis%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2011%26volume%3D108%26spage%3D7541%26epage%3D6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit6b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Kawazu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saso, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tong, K. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McQuire, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goto, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Son, D. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wakeham, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyagishi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mak, T. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Okada, H.</span><span> </span><span class="NLM_article-title">Histone demethylase JMJD2B functions as a co-factor of estrogen receptor in breast cancer proliferation and mammary gland development</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">e17830</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=e17830&author=M.+Kawazuauthor=K.+Sasoauthor=K.+I.+Tongauthor=T.+McQuireauthor=K.+Gotoauthor=D.+O.+Sonauthor=A.+Wakehamauthor=M.+Miyagishiauthor=T.+W.+Makauthor=H.+Okada&title=Histone+demethylase+JMJD2B+functions+as+a+co-factor+of+estrogen+receptor+in+breast+cancer+proliferation+and+mammary+gland+development"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6b&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKawazu%26aufirst%3DM.%26aulast%3DSaso%26aufirst%3DK.%26aulast%3DTong%26aufirst%3DK.%2BI.%26aulast%3DMcQuire%26aufirst%3DT.%26aulast%3DGoto%26aufirst%3DK.%26aulast%3DSon%26aufirst%3DD.%2BO.%26aulast%3DWakeham%26aufirst%3DA.%26aulast%3DMiyagishi%26aufirst%3DM.%26aulast%3DMak%26aufirst%3DT.%2BW.%26aulast%3DOkada%26aufirst%3DH.%26atitle%3DHistone%2520demethylase%2520JMJD2B%2520functions%2520as%2520a%2520co-factor%2520of%2520estrogen%2520receptor%2520in%2520breast%2520cancer%2520proliferation%2520and%2520mammary%2520gland%2520development%26jtitle%3DPLoS%2520One%26date%3D2011%26volume%3D6%26spage%3De17830" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit6c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Berry, W. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lightfoot, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janknecht, R.</span><span> </span><span class="NLM_article-title">Oncogenic features of the JMJD2A histone demethylase in breast cancer</span> <span class="citation_source-journal">Int. J. Oncol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">1701</span><span class="NLM_x">–</span> <span class="NLM_lpage">6</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6c&amp;dbid=16&amp;doi=10.1021%2Fjm500249n&amp;key=10.3892%2Fijo.2012.1618" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6c&amp;dbid=8&amp;doi=10.1021%2Fjm500249n&amp;key=22948256" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6c&amp;dbid=32&amp;doi=10.1021%2Fjm500249n&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslWisLrF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2012&pages=1701-6&author=W.+L.+Berryauthor=S.+Shinauthor=S.+A.+Lightfootauthor=R.+Janknecht&title=Oncogenic+features+of+the+JMJD2A+histone+demethylase+in+breast+cancer"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6cR"><div class="casContent"><span class="casTitleNuber">6c</span><div class="casTitle"><span class="NLM_cas:atitle">Oncogenic features of the JMJD2A histone demethylase in breast cancer</span></div><div class="casAuthors">Berry, William L.; Shin, Sook; Lightfoot, Stan A.; Janknecht, Ralf</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Oncology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1701-1706</span>CODEN:
                <span class="NLM_cas:coden">IJONES</span>;
        ISSN:<span class="NLM_cas:issn">1019-6439</span>.
    
            (<span class="NLM_cas:orgname">International Journal of Oncology</span>)
        </div><div class="casAbstract">Estrogen receptor α (ERα) plays a pivotal role in the genesis of the majority of breast tumors.  Consequently, endocrine therapy is now routinely utilized in the clinic for the treatment of ERα-pos. breast cancer patients.  However, how ERα activity becomes dysregulated in breast cancer cells remains to be elucidated.  The aim of this study was to show that the histone demethylase JMJD2A, also known as KDM4A, is capable of forming a complex with ERα in vivo.  Moreover, wild-type JMJD2A, but not a catalytically impaired mutant, was able to strongly coactivate ERα-mediated transcription.  Consistently, the downregulation of JMJD2A in human T47D breast cancer cells led to a decreased expression of cyclin D1, a prominent ERα target gene and cell cycle regulator.  The downregulation of JMJD2A induced a redn. in the growth of T47D cells.  In addn., we found that JMJD2A is overexpressed in human breast tumors both at the mRNA and protein level.  Taken together, these data indicate that the overexpression of JMJD2A may contribute to breast tumor formation by stimulating ERα activity and that JMJD2A may be a breast-relevant oncoprotein.  As such, small mol. drugs targeting the catalytic center of JMJD2A might be useful in breast cancer adjuvant therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpubrKLORIlcrVg90H21EOLACvtfcHk0lg6YlKklE_JYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslWisLrF&md5=13f012c96876d1b49e2f10f08985d853</span></div><a href="/servlet/linkout?suffix=cit6c&amp;dbid=16384&amp;doi=10.3892%2Fijo.2012.1618&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3892%252Fijo.2012.1618%26sid%3Dliteratum%253Aachs%26aulast%3DBerry%26aufirst%3DW.%2BL.%26aulast%3DShin%26aufirst%3DS.%26aulast%3DLightfoot%26aufirst%3DS.%2BA.%26aulast%3DJanknecht%26aufirst%3DR.%26atitle%3DOncogenic%2520features%2520of%2520the%2520JMJD2A%2520histone%2520demethylase%2520in%2520breast%2520cancer%26jtitle%3DInt.%2520J.%2520Oncol.%26date%3D2012%26volume%3D41%26spage%3D1701%26epage%3D6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Hamada, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Itoh, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsumoto, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakagawa, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janknecht, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyata, N.</span><span> </span><span class="NLM_article-title">Synthesis and activity of N-oxalylglycine and its derivatives as Jumonji C-domain-containing histone lysine demethylase inhibitors</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">2852</span><span class="NLM_x">–</span> <span class="NLM_lpage">5</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=2852-5&author=S.+Hamadaauthor=T.+D.+Kimauthor=T.+Suzukiauthor=Y.+Itohauthor=H.+Tsumotoauthor=H.+Nakagawaauthor=R.+Janknechtauthor=N.+Miyata&title=Synthesis+and+activity+of+N-oxalylglycine+and+its+derivatives+as+Jumonji+C-domain-containing+histone+lysine+demethylase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7a&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHamada%26aufirst%3DS.%26aulast%3DKim%26aufirst%3DT.%2BD.%26aulast%3DSuzuki%26aufirst%3DT.%26aulast%3DItoh%26aufirst%3DY.%26aulast%3DTsumoto%26aufirst%3DH.%26aulast%3DNakagawa%26aufirst%3DH.%26aulast%3DJanknecht%26aufirst%3DR.%26aulast%3DMiyata%26aufirst%3DN.%26atitle%3DSynthesis%2520and%2520activity%2520of%2520N-oxalylglycine%2520and%2520its%2520derivatives%2520as%2520Jumonji%2520C-domain-containing%2520histone%2520lysine%2520demethylase%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2009%26volume%3D19%26spage%3D2852%26epage%3D5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit7b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Rose, N. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woon, E. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kingham, G. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">King, O. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mecinovic, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clifton, I. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ng, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Talib-Hardy, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oppermann, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDonough, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span> </span><span class="NLM_article-title">Selective inhibitors of the JMJD2 histone demethylases: combined nondenaturing mass spectrometric screening and crystallographic approaches</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">1810</span><span class="NLM_x">–</span> <span class="NLM_lpage">8</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm901680b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref7/cit7b&amp;dbid=32&amp;doi=10.1021%2Fjm500249n&amp;key=1%3ACAS%3A528%3ADC%252BC3cXnvFWnsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=1810-8&author=N.+R.+Roseauthor=E.+C.+Woonauthor=G.+L.+Kinghamauthor=O.+N.+Kingauthor=J.+Mecinovicauthor=I.+J.+Cliftonauthor=S.+S.+Ngauthor=J.+Talib-Hardyauthor=U.+Oppermannauthor=M.+A.+McDonoughauthor=C.+J.+Schofield&title=Selective+inhibitors+of+the+JMJD2+histone+demethylases%3A+combined+nondenaturing+mass+spectrometric+screening+and+crystallographic+approaches"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7bR"><div class="casContent"><span class="casTitleNuber">7b</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Inhibitors of the JMJD2 Histone Demethylases: Combined Nondenaturing Mass Spectrometric Screening and Crystallographic Approaches</span></div><div class="casAuthors">Rose, Nathan R.; Woon, Esther C. Y.; Kingham, Guy L.; King, Oliver N. F.; Mecinovic, Jasmin; Clifton, Ian J.; Ng, Stanley S.; Talib-Hardy, Jobina; Oppermann, Udo; McDonough, Michael A.; Schofield, Christopher J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1810-1818</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Ferrous ion and 2-oxoglutarate (2OG) oxygenases catalyze the demethylation of Nε-methylated lysine residues in histones.  Here, studies on the inhibition of the JMJD2 subfamily of histone demethylases are reported, employing binding analyses by non-denaturing mass spectrometry (MS), dynamic combinatorial chem. coupled to MS, turnover assays, and crystallog.  The results of initial binding and inhibition assays directed the prodn. and anal. of a set of N-oxalyl-D-tyrosine derivs. to explore the extent of a sub-pocket at the JMJD2 active site.  Some of the inhibitors were shown to be selective for JMJD2 over the hypoxia-inducible factor prolyl hydroxylase PHD2.  A crystal structure of JMJD2A in complex with one of the potent inhibitors was obtained; modeling other inhibitors based on this structure predicts interactions that enable improved inhibition for some compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoa6dzV_ed-qrVg90H21EOLACvtfcHk0lg6YlKklE_JYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXnvFWnsQ%253D%253D&md5=3eb3c36d91f46193946f64288ea9ccae</span></div><a href="/servlet/linkout?suffix=cit7b&amp;dbid=16384&amp;doi=10.1021%2Fjm901680b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901680b%26sid%3Dliteratum%253Aachs%26aulast%3DRose%26aufirst%3DN.%2BR.%26aulast%3DWoon%26aufirst%3DE.%2BC.%26aulast%3DKingham%26aufirst%3DG.%2BL.%26aulast%3DKing%26aufirst%3DO.%2BN.%26aulast%3DMecinovic%26aufirst%3DJ.%26aulast%3DClifton%26aufirst%3DI.%2BJ.%26aulast%3DNg%26aufirst%3DS.%2BS.%26aulast%3DTalib-Hardy%26aufirst%3DJ.%26aulast%3DOppermann%26aufirst%3DU.%26aulast%3DMcDonough%26aufirst%3DM.%2BA.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26atitle%3DSelective%2520inhibitors%2520of%2520the%2520JMJD2%2520histone%2520demethylases%253A%2520combined%2520nondenaturing%2520mass%2520spectrometric%2520screening%2520and%2520crystallographic%2520approaches%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D1810%26epage%3D8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Thalhammer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mecinovic, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loenarz, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tumber, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rose, N. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heightman, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span> </span><span class="NLM_article-title">Inhibition of the histone demethylase JMJD2E by 3-substituted pyridine 2,4-dicarboxylates</span> <span class="citation_source-journal">Org. Biomol Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">127</span><span class="NLM_x">–</span> <span class="NLM_lpage">35</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2011&pages=127-35&author=A.+Thalhammerauthor=J.+Mecinovicauthor=C.+Loenarzauthor=A.+Tumberauthor=N.+R.+Roseauthor=T.+D.+Heightmanauthor=C.+J.+Schofield&title=Inhibition+of+the+histone+demethylase+JMJD2E+by+3-substituted+pyridine+2%2C4-dicarboxylates"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DThalhammer%26aufirst%3DA.%26aulast%3DMecinovic%26aufirst%3DJ.%26aulast%3DLoenarz%26aufirst%3DC.%26aulast%3DTumber%26aufirst%3DA.%26aulast%3DRose%26aufirst%3DN.%2BR.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26atitle%3DInhibition%2520of%2520the%2520histone%2520demethylase%2520JMJD2E%2520by%25203-substituted%2520pyridine%25202%252C4-dicarboxylates%26jtitle%3DOrg.%2520Biomol%2520Chem.%26date%3D2011%26volume%3D9%26spage%3D127%26epage%3D35" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">King, O. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, X. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sakurai, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawamura, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rose, N. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ng, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quinn, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rai, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mott, B. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beswick, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klose, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oppermann, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jadhav, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heightman, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maloney, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeonov, A.</span><span> </span><span class="NLM_article-title">Quantitative high-throughput screening identifies 8-hydroxyquinolines as cell-active histone demethylase inhibitors</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">e15535</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2010&pages=e15535&author=O.+N.+Kingauthor=X.+S.+Liauthor=M.+Sakuraiauthor=A.+Kawamuraauthor=N.+R.+Roseauthor=S.+S.+Ngauthor=A.+M.+Quinnauthor=G.+Raiauthor=B.+T.+Mottauthor=P.+Beswickauthor=R.+J.+Kloseauthor=U.+Oppermannauthor=A.+Jadhavauthor=T.+D.+Heightmanauthor=D.+J.+Maloneyauthor=C.+J.+Schofieldauthor=A.+Simeonov&title=Quantitative+high-throughput+screening+identifies+8-hydroxyquinolines+as+cell-active+histone+demethylase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKing%26aufirst%3DO.%2BN.%26aulast%3DLi%26aufirst%3DX.%2BS.%26aulast%3DSakurai%26aufirst%3DM.%26aulast%3DKawamura%26aufirst%3DA.%26aulast%3DRose%26aufirst%3DN.%2BR.%26aulast%3DNg%26aufirst%3DS.%2BS.%26aulast%3DQuinn%26aufirst%3DA.%2BM.%26aulast%3DRai%26aufirst%3DG.%26aulast%3DMott%26aufirst%3DB.%2BT.%26aulast%3DBeswick%26aufirst%3DP.%26aulast%3DKlose%26aufirst%3DR.%2BJ.%26aulast%3DOppermann%26aufirst%3DU.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DMaloney%26aufirst%3DD.%2BJ.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26aulast%3DSimeonov%26aufirst%3DA.%26atitle%3DQuantitative%2520high-throughput%2520screening%2520identifies%25208-hydroxyquinolines%2520as%2520cell-active%2520histone%2520demethylase%2520inhibitors%26jtitle%3DPLoS%2520One%26date%3D2010%26volume%3D5%26spage%3De15535" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Mackeen, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kramer, H. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, K. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coleman, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hopkinson, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kessler, B. M.</span><span> </span><span class="NLM_article-title">Small-molecule-based inhibition of histone demethylation in cells assessed by quantitative mass spectrometry</span> <span class="citation_source-journal">J. Proteome Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">4082</span><span class="NLM_x">–</span> <span class="NLM_lpage">92</span></span><div class="citationLinks">[<a href="/doi/10.1021/pr100269b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Fjm500249n&amp;key=1%3ACAS%3A528%3ADC%252BC3cXptVygu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2010&pages=4082-92&author=M.+M.+Mackeenauthor=H.+B.+Kramerauthor=K.+H.+Changauthor=M.+L.+Colemanauthor=R.+J.+Hopkinsonauthor=C.+J.+Schofieldauthor=B.+M.+Kessler&title=Small-molecule-based+inhibition+of+histone+demethylation+in+cells+assessed+by+quantitative+mass+spectrometry"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule-Based Inhibition of Histone Demethylation in Cells Assessed by Quantitative Mass Spectrometry</span></div><div class="casAuthors">MacKeen, Mukram M.; Kramer, Holger B.; Chang, Kai-Hsuan; Coleman, Matthew L.; Hopkinson, Richard J.; Schofield, Christopher J.; Kessler, Benedikt M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Proteome Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">4082-4092</span>CODEN:
                <span class="NLM_cas:coden">JPROBS</span>;
        ISSN:<span class="NLM_cas:issn">1535-3893</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Post-translational modifications on histones are an important mechanism for the regulation of gene expression and are involved in all aspects of cell growth and differentiation, as well as pathol. processes including neurodegeneration, autoimmunity, and cancer.  A major challenge within the chromatin field is to develop methods for the quant. anal. of histone modifications.  Here the authors report a mass spectrometry (MS) approach based on ultraperformance liq. chromatog. high/low collision switching (UPLC-MSE) to monitor histone modifications in cells.  This approach is exemplified by the anal. of trimethylated lysine-9 levels in histone H3, following a simple chem. derivatization procedure with d6-acetic anhydride.  This method was used to study the inhibition of histone demethylases with pyridine-2,4-dicarboxylic acid (PDCA) derivs. in cells.  The authors' results show that the PDCA-dimethyl ester inhibits JMJD2A catalyzed demethylation of lysine-9 on histone H3 in human HEK 293T cells.  Demethylase inhibition, as obsd. by MS analyses, was supported by immunoblotting with modification-specific antibodies.  The results demonstrate that PDCA derived small mols. are cell permeable demethylase inhibitors and reveal that quant. MS is a useful tool for measuring post-translational histone modifications in cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrysm0DR-5KtbVg90H21EOLACvtfcHk0liKG-8SpQaBGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXptVygu7o%253D&md5=ef022b8021283895612a34bd0f75a1d5</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fpr100269b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fpr100269b%26sid%3Dliteratum%253Aachs%26aulast%3DMackeen%26aufirst%3DM.%2BM.%26aulast%3DKramer%26aufirst%3DH.%2BB.%26aulast%3DChang%26aufirst%3DK.%2BH.%26aulast%3DColeman%26aufirst%3DM.%2BL.%26aulast%3DHopkinson%26aufirst%3DR.%2BJ.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26aulast%3DKessler%26aufirst%3DB.%2BM.%26atitle%3DSmall-molecule-based%2520inhibition%2520of%2520histone%2520demethylation%2520in%2520cells%2520assessed%2520by%2520quantitative%2520mass%2520spectrometry%26jtitle%3DJ.%2520Proteome%2520Res.%26date%3D2010%26volume%3D9%26spage%3D4082%26epage%3D92" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Bradford, M. M.</span><span> </span><span class="NLM_article-title">A rapid and sensitive method for the quantitation of microgram quantities of protein utilizing the principle of protein-dye binding</span> <span class="citation_source-journal">Anal. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1976</span><span class="NLM_x">, </span> <span class="NLM_volume">72</span><span class="NLM_x">, </span> <span class="NLM_fpage">248</span><span class="NLM_x">–</span> <span class="NLM_lpage">54</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Fjm500249n&amp;key=10.1016%2F0003-2697%2876%2990527-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Fjm500249n&amp;key=942051" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Fjm500249n&amp;key=1%3ACAS%3A528%3ADyaE28XksVehtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=1976&pages=248-54&author=M.+M.+Bradford&title=A+rapid+and+sensitive+method+for+the+quantitation+of+microgram+quantities+of+protein+utilizing+the+principle+of+protein-dye+binding"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">A rapid and sensitive method for the quantitation of microgram quantities of protein utilizing the principle of protein-dye binding</span></div><div class="casAuthors">Bradford, Marion M.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">1976</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">248-54</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    </div><div class="casAbstract">A protein detn. method that involves the binding of coomassie Brilliant Blue G 250 to protein is described.  The binding of the dye to protein causes a shift in the absorption max. of the dye from 465 to 595 nm, and it is the increase in absorption at 595 nm that is monitored.  This assay is very reproducible and rapid with the dye binding process virtually complete in ∼ 2 min with good color stability for 1 hr.  There is little or no interference from cations such as Na+ or K+ nor from carbohydrates such as sucrose.  A small amt. of color is developed in the presence of strongly alk. buffering agents, but the assay may be run accurately by the use of proper buffer controls.  The only components found to give excessive interfering color in the assay are relatively large amts. of detergents such as Na dodecyl sulfate, Triton X 100, and commercial glassware detergents.  Interference by small amts. of detergent may be eliminated by the use of proper control.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrN_T_d0trcNrVg90H21EOLACvtfcHk0lg0ouPHeUPJOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE28XksVehtrY%253D&md5=43f388a493becc4193c7a49d9de0e1db</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2F0003-2697%2876%2990527-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0003-2697%252876%252990527-3%26sid%3Dliteratum%253Aachs%26aulast%3DBradford%26aufirst%3DM.%2BM.%26atitle%3DA%2520rapid%2520and%2520sensitive%2520method%2520for%2520the%2520quantitation%2520of%2520microgram%2520quantities%2520of%2520protein%2520utilizing%2520the%2520principle%2520of%2520protein-dye%2520binding%26jtitle%3DAnal.%2520Biochem.%26date%3D1976%26volume%3D72%26spage%3D248%26epage%3D54" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Otwinowski, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Minor, W.</span><span> </span><span class="NLM_article-title">Processing of X-ray diffraction data collected in oscillation mode</span> <span class="citation_source-journal">Methods Enzymol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">276</span><span class="NLM_x">, </span> <span class="NLM_fpage">307</span><span class="NLM_x">–</span> <span class="NLM_lpage">26</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Fjm500249n&amp;key=10.1016%2FS0076-6879%2897%2976066-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Fjm500249n&amp;key=27754618" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Fjm500249n&amp;key=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=1997&pages=307-26&author=Z.+Otwinowskiauthor=W.+Minor&title=Processing+of+X-ray+diffraction+data+collected+in+oscillation+mode"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Processing of x-ray diffraction data collected in oscillation mode</span></div><div class="casAuthors">Otwinowski, Zbyszek; Minor, Wladek</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">276</span>
        (<span class="NLM_cas:issue">Macromolecular Crystallography, Part A</span>),
    <span class="NLM_cas:pages">307-326</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Macromol. crystallog. is an iterative process.  Rarely do the first crystals provide all the necessary data to solve the biol. problem being studied.  Each step benefits from experience learned in previous steps.  To monitor the progress, the HKL package provides 2 tools: (1) statistics, both weighted (χ2) and unweighted (R-merge), are provided, and the Bayesian reasoning and multicomponent error model facilitates obtaining the proper error ests. and (2) visualization of the process plays a double role by helping the operator to confirm that the process of data redn., including the resulting statistics, is correct, and allowing one to evaluate problems for which there are no good statistical criteria.  Visualization also provides confidence that the point of diminishing returns in data collection and redn. has been reached.  At that point, the effort should be directed to solving the structure.  The methods presented here have been applied to solve a large variety of problems, from inorg. mols. with 5 Å unit cell to rotavirus of 700 Å diam. crystd. in 700 × 1000 × 1400 Å cell.  Overall quality of the method was tested by many researchers by successful application of the programs to MAD structure detns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrA3fvBOHqhhrVg90H21EOLACvtfcHk0lg0ouPHeUPJOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D&md5=c9536971d4e32cc35352c40fb9368131</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2FS0076-6879%2897%2976066-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0076-6879%252897%252976066-X%26sid%3Dliteratum%253Aachs%26aulast%3DOtwinowski%26aufirst%3DZ.%26aulast%3DMinor%26aufirst%3DW.%26atitle%3DProcessing%2520of%2520X-ray%2520diffraction%2520data%2520collected%2520in%2520oscillation%2520mode%26jtitle%3DMethods%2520Enzymol.%26date%3D1997%26volume%3D276%26spage%3D307%26epage%3D26" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Collaborative Computational Project</span><span> </span><span class="NLM_article-title">The CCP4 suite: programs for protein crystallography</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">760</span><span class="NLM_x">–</span> <span class="NLM_lpage">3</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13a&amp;dbid=16&amp;doi=10.1021%2Fjm500249n&amp;key=10.1107%2FS0907444994003112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13a&amp;dbid=8&amp;doi=10.1021%2Fjm500249n&amp;key=15299374" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=1994&pages=760-3&author=Collaborative+Computational+Project&title=The+CCP4+suite%3A+programs+for+protein+crystallography"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13a&amp;dbid=16384&amp;doi=10.1107%2FS0907444994003112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444994003112%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DThe%2520CCP4%2520suite%253A%2520programs%2520for%2520protein%2520crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1994%26volume%3D50%26spage%3D760%26epage%3D3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit13b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Murshudov, G. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vagin, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dodson, E. J.</span><span> </span><span class="NLM_article-title">Refinement of macromolecular structures by the maximum-likelihood method</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">240</span><span class="NLM_x">–</span> <span class="NLM_lpage">55</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13b&amp;dbid=16&amp;doi=10.1021%2Fjm500249n&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13b&amp;dbid=8&amp;doi=10.1021%2Fjm500249n&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13b&amp;dbid=32&amp;doi=10.1021%2Fjm500249n&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-55&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+macromolecular+structures+by+the+maximum-likelihood+method"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13bR"><div class="casContent"><span class="casTitleNuber">13b</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. σA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial α-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0lgNwnoAvvrmqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit13b&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520macromolecular%2520structures%2520by%2520the%2520maximum-likelihood%2520method%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1997%26volume%3D53%26spage%3D240%26epage%3D55" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Vagin, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teplyakov, A.</span><span> </span><span class="NLM_article-title">MOLREP: an automated program for molecular replacement</span> <span class="citation_source-journal">J. Appl. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">1022</span><span class="NLM_x">–</span> <span class="NLM_lpage">5</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Fjm500249n&amp;key=10.1107%2FS0021889897006766" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Fjm500249n&amp;key=1%3ACAS%3A528%3ADyaK1cXhslehu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=1997&pages=1022-5&author=A.+Vaginauthor=A.+Teplyakov&title=MOLREP%3A+an+automated+program+for+molecular+replacement"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">MOLREP: an automated program for molecular replacement</span></div><div class="casAuthors">Vagin, Alexei; Teplyakov, Alexei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1022-1025</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard International Publishers Ltd.</span>)
        </div><div class="casAbstract">MOLREP is an automated program for mol. replacement which uses effective new approaches in data processing and rotational and translational searching.  These include an automatic choice of all parameters, scaling by Patterson origin peaks and soft resoln. cutoff.  One of the cornerstones of the program is an original full-symmetry translation function combined with a packing function.  Information from the model already placed in the cell is incorporated in both translation and packing functions.  A no. of tests using exptl. data proved the ability of the program to find the correct soln. in difficult cases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkGXdZ-nTKxLVg90H21EOLACvtfcHk0lgNwnoAvvrmqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXhslehu7Y%253D&md5=96af64aedd8a8f018c3c18b57a6b4b21</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1107%2FS0021889897006766&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889897006766%26sid%3Dliteratum%253Aachs%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DTeplyakov%26aufirst%3DA.%26atitle%3DMOLREP%253A%2520an%2520automated%2520program%2520for%2520molecular%2520replacement%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D1997%26volume%3D30%26spage%3D1022%26epage%3D5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Lamzin, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, K. S.</span><span> </span><span class="NLM_article-title">Automated refinement of protein models</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">127</span><span class="NLM_x">–</span> <span class="NLM_lpage">47</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=1993&pages=127-47&author=V.+Lamzinauthor=K.+S.+Wilson&title=Automated+refinement+of+protein+models"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLamzin%26aufirst%3DV.%26aulast%3DWilson%26aufirst%3DK.%2BS.%26atitle%3DAutomated%2520refinement%2520of%2520protein%2520models%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1993%26volume%3D49%26spage%3D127%26epage%3D47" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Laskowski, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacArthur, M. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moss, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thornton, J. M.</span><span> </span><span class="NLM_article-title">PROCHECK: a program to check the stereochemical quality of protein structures</span> <span class="citation_source-journal">J. Appl. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">283</span><span class="NLM_x">–</span> <span class="NLM_lpage">91</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Fjm500249n&amp;key=10.1107%2FS0021889892009944" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Fjm500249n&amp;key=1%3ACAS%3A528%3ADyaK3sXit12lurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=1993&pages=283-91&author=R.+A.+Laskowskiauthor=M.+W.+MacArthurauthor=D.+S.+Mossauthor=J.+M.+Thornton&title=PROCHECK%3A+a+program+to+check+the+stereochemical+quality+of+protein+structures"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">PROCHECK: a program to check the stereochemical quality of protein structures</span></div><div class="casAuthors">Laskowski, Roman A.; MacArthur, Malcolm W.; Moss, David S.; Thornton, Janet M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">283-91</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    </div><div class="casAbstract">The PROCHECK suite of programs provides a detailed check on the stereochem. of a protein structure.  Its outputs comprise a no. of plots in PostScript format and a comprehensive residue-by-residue listing.  These give an assessment of the overall quality of the structure as compared with well-refined structures of the same resoln. and also highlight regions that may need further investigation.  The PROCHECK programs are useful for assessing the quality not only of protein structures in the process of being solved but also of existing structures and of those being modeled on known structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCe5FgJf4KH7Vg90H21EOLACvtfcHk0liuFXn6mRUgFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXit12lurY%253D&md5=fb69fc4410cd716aaaa7cc0db06b3ed2</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1107%2FS0021889892009944&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889892009944%26sid%3Dliteratum%253Aachs%26aulast%3DLaskowski%26aufirst%3DR.%2BA.%26aulast%3DMacArthur%26aufirst%3DM.%2BW.%26aulast%3DMoss%26aufirst%3DD.%2BS.%26aulast%3DThornton%26aufirst%3DJ.%2BM.%26atitle%3DPROCHECK%253A%2520a%2520program%2520to%2520check%2520the%2520stereochemical%2520quality%2520of%2520protein%2520structures%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D1993%26volume%3D26%26spage%3D283%26epage%3D91" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Chen, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kappler, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hong, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crawford, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lan, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whetstine, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dai, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hansen, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, G.</span><span> </span><span class="NLM_article-title">Structural basis of the recognition of a methylated histone tail by JMJD2A</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">104</span><span class="NLM_x">, </span> <span class="NLM_fpage">10818</span><span class="NLM_x">–</span> <span class="NLM_lpage">23</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2007&pages=10818-23&author=Z.+Chenauthor=J.+Zangauthor=J.+Kapplerauthor=X.+Hongauthor=F.+Crawfordauthor=Q.+Wangauthor=F.+Lanauthor=C.+Jiangauthor=J.+Whetstineauthor=S.+Daiauthor=K.+Hansenauthor=Y.+Shiauthor=G.+Zhang&title=Structural+basis+of+the+recognition+of+a+methylated+histone+tail+by+JMJD2A"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DZang%26aufirst%3DJ.%26aulast%3DKappler%26aufirst%3DJ.%26aulast%3DHong%26aufirst%3DX.%26aulast%3DCrawford%26aufirst%3DF.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DLan%26aufirst%3DF.%26aulast%3DJiang%26aufirst%3DC.%26aulast%3DWhetstine%26aufirst%3DJ.%26aulast%3DDai%26aufirst%3DS.%26aulast%3DHansen%26aufirst%3DK.%26aulast%3DShi%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DG.%26atitle%3DStructural%2520basis%2520of%2520the%2520recognition%2520of%2520a%2520methylated%2520histone%2520tail%2520by%2520JMJD2A%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2007%26volume%3D104%26spage%3D10818%26epage%3D23" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Krishnan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trievel, R. C.</span><span> </span><span class="NLM_article-title">Structural and functional analysis of JMJD2D reveals molecular basis for site-specific demethylation among JMJD2 demethylases</span> <span class="citation_source-journal">Structure</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">98</span><span class="NLM_x">–</span> <span class="NLM_lpage">108</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2013&pages=98-108&author=S.+Krishnanauthor=R.+C.+Trievel&title=Structural+and+functional+analysis+of+JMJD2D+reveals+molecular+basis+for+site-specific+demethylation+among+JMJD2+demethylases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKrishnan%26aufirst%3DS.%26aulast%3DTrievel%26aufirst%3DR.%2BC.%26atitle%3DStructural%2520and%2520functional%2520analysis%2520of%2520JMJD2D%2520reveals%2520molecular%2520basis%2520for%2520site-specific%2520demethylation%2520among%2520JMJD2%2520demethylases%26jtitle%3DStructure%26date%3D2013%26volume%3D21%26spage%3D98%26epage%3D108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Jones, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zou, J. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowan, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kjeldgaard, M.</span><span> </span><span class="NLM_article-title">Improved methods for building protein models in electron density maps and the location of errors in these models</span> <span class="citation_source-journal">Acta Crystallogr., Sect. A</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1991</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">110</span><span class="NLM_x">–</span> <span class="NLM_lpage">9</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Fjm500249n&amp;key=10.1107%2FS0108767390010224" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Fjm500249n&amp;key=2025413" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Fjm500249n&amp;key=1%3ACAS%3A280%3ADyaK3M3it1CmsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=1991&pages=110-9&author=T.+A.+Jonesauthor=J.+Y.+Zouauthor=S.+W.+Cowanauthor=M.+Kjeldgaard&title=Improved+methods+for+building+protein+models+in+electron+density+maps+and+the+location+of+errors+in+these+models"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Improved methods for building protein models in electron density maps and the location of errors in these models</span></div><div class="casAuthors">Jones T A; Zou J Y; Cowan S W; Kjeldgaard M</div><div class="citationInfo"><span class="NLM_cas:title">Acta crystallographica. Section A, Foundations of crystallography</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">47 ( Pt 2)</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">110-9</span>
        ISSN:<span class="NLM_cas:issn">0108-7673</span>.
    </div><div class="casAbstract">Map interpretation remains a critical step in solving the structure of a macromolecule.  Errors introduced at this early stage may persist throughout crystallographic refinement and result in an incorrect structure.  The normally quoted crystallographic residual is often a poor description for the quality of the model.  Strategies and tools are described that help to alleviate this problem.  These simplify the model-building process, quantify the goodness of fit of the model on a per-residue basis and locate possible errors in peptide and side-chain conformations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQhby6yNdW8QMKL0FlyGHDofW6udTcc2eb5Cehxuj5Ulbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK3M3it1CmsQ%253D%253D&md5=fd4d4a3cdb4ecb3f05bbad8b2300ef45</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1107%2FS0108767390010224&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0108767390010224%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DT.%2BA.%26aulast%3DZou%26aufirst%3DJ.%2BY.%26aulast%3DCowan%26aufirst%3DS.%2BW.%26aulast%3DKjeldgaard%26aufirst%3DM.%26atitle%3DImproved%2520methods%2520for%2520building%2520protein%2520models%2520in%2520electron%2520density%2520maps%2520and%2520the%2520location%2520of%2520errors%2520in%2520these%2520models%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520A%26date%3D1991%26volume%3D47%26spage%3D110%26epage%3D9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Raghuraman, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mosier, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Desai, U. R.</span><span> </span><span class="NLM_article-title">Finding a needle in a haystack: development of a combinatorial virtual screening approach for identifying high specificity heparin/heparan sulfate sequence(s)</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">3553</span><span class="NLM_x">–</span> <span class="NLM_lpage">62</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm060092o" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=3553-62&author=A.+Raghuramanauthor=P.+D.+Mosierauthor=U.+R.+Desai&title=Finding+a+needle+in+a+haystack%3A+development+of+a+combinatorial+virtual+screening+approach+for+identifying+high+specificity+heparin%2Fheparan+sulfate+sequence%28s%29"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm060092o&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm060092o%26sid%3Dliteratum%253Aachs%26aulast%3DRaghuraman%26aufirst%3DA.%26aulast%3DMosier%26aufirst%3DP.%2BD.%26aulast%3DDesai%26aufirst%3DU.%2BR.%26atitle%3DFinding%2520a%2520needle%2520in%2520a%2520haystack%253A%2520development%2520of%2520a%2520combinatorial%2520virtual%2520screening%2520approach%2520for%2520identifying%2520high%2520specificity%2520heparin%252Fheparan%2520sulfate%2520sequence%2528s%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D3553%26epage%3D62" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Chowdhury, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yeoh, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tian, Y. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hillringhaus, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bagg, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rose, N. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leung, I. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, X. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woon, E. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDonough, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">King, O. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clifton, I. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klose, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Claridge, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ratcliffe, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawamura, A.</span><span> </span><span class="NLM_article-title">The oncometabolite 2-hydroxyglutarate inhibits histone lysine demethylases</span> <span class="citation_source-journal">EMBO Rep.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">463</span><span class="NLM_x">–</span> <span class="NLM_lpage">9</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=463-9&author=R.+Chowdhuryauthor=K.+K.+Yeohauthor=Y.+M.+Tianauthor=L.+Hillringhausauthor=E.+A.+Baggauthor=N.+R.+Roseauthor=I.+K.+Leungauthor=X.+S.+Liauthor=E.+C.+Woonauthor=M.+Yangauthor=M.+A.+McDonoughauthor=O.+N.+Kingauthor=I.+J.+Cliftonauthor=R.+J.+Kloseauthor=T.+D.+Claridgeauthor=P.+J.+Ratcliffeauthor=C.+J.+Schofieldauthor=A.+Kawamura&title=The+oncometabolite+2-hydroxyglutarate+inhibits+histone+lysine+demethylases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DChowdhury%26aufirst%3DR.%26aulast%3DYeoh%26aufirst%3DK.%2BK.%26aulast%3DTian%26aufirst%3DY.%2BM.%26aulast%3DHillringhaus%26aufirst%3DL.%26aulast%3DBagg%26aufirst%3DE.%2BA.%26aulast%3DRose%26aufirst%3DN.%2BR.%26aulast%3DLeung%26aufirst%3DI.%2BK.%26aulast%3DLi%26aufirst%3DX.%2BS.%26aulast%3DWoon%26aufirst%3DE.%2BC.%26aulast%3DYang%26aufirst%3DM.%26aulast%3DMcDonough%26aufirst%3DM.%2BA.%26aulast%3DKing%26aufirst%3DO.%2BN.%26aulast%3DClifton%26aufirst%3DI.%2BJ.%26aulast%3DKlose%26aufirst%3DR.%2BJ.%26aulast%3DClaridge%26aufirst%3DT.%2BD.%26aulast%3DRatcliffe%26aufirst%3DP.%2BJ.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26aulast%3DKawamura%26aufirst%3DA.%26atitle%3DThe%2520oncometabolite%25202-hydroxyglutarate%2520inhibits%2520histone%2520lysine%2520demethylases%26jtitle%3DEMBO%2520Rep.%26date%3D2011%26volume%3D12%26spage%3D463%26epage%3D9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Hsu, K. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, J. M.</span><span> </span><span class="NLM_article-title">iGEMDOCK: a graphical environment of enhancing GEMDOCK using pharmacological interactions and post-screening analysis</span> <span class="citation_source-journal">BMC Bioinformatics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">S33</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=S33&author=K.+C.+Hsuauthor=Y.+F.+Chenauthor=S.+R.+Linauthor=J.+M.+Yang&title=iGEMDOCK%3A+a+graphical+environment+of+enhancing+GEMDOCK+using+pharmacological+interactions+and+post-screening+analysis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22a&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHsu%26aufirst%3DK.%2BC.%26aulast%3DChen%26aufirst%3DY.%2BF.%26aulast%3DLin%26aufirst%3DS.%2BR.%26aulast%3DYang%26aufirst%3DJ.%2BM.%26atitle%3DiGEMDOCK%253A%2520a%2520graphical%2520environment%2520of%2520enhancing%2520GEMDOCK%2520using%2520pharmacological%2520interactions%2520and%2520post-screening%2520analysis%26jtitle%3DBMC%2520Bioinformatics%26date%3D2011%26volume%3D12%26spage%3DS33" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit22b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Yang, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, C. C.</span><span> </span><span class="NLM_article-title">GEMDOCK: a generic evolutionary method for molecular docking</span> <span class="citation_source-journal">Proteins: Struct., Funct., Bioinf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">288</span><span class="NLM_x">–</span> <span class="NLM_lpage">304</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2004&pages=288-304&author=J.+M.+Yangauthor=C.+C.+Chen&title=GEMDOCK%3A+a+generic+evolutionary+method+for+molecular+docking"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22b&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DJ.%2BM.%26aulast%3DChen%26aufirst%3DC.%2BC.%26atitle%3DGEMDOCK%253A%2520a%2520generic%2520evolutionary%2520method%2520for%2520molecular%2520docking%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Bioinf.%26date%3D2004%26volume%3D55%26spage%3D288%26epage%3D304" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Gregory, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, R. T.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mohler, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">French, F. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, E. M.</span><span> </span><span class="NLM_article-title">Androgen receptor stabilization in recurrent prostate cancer is associated with hypersensitivity to low androgen</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">61</span><span class="NLM_x">, </span> <span class="NLM_fpage">2892</span><span class="NLM_x">–</span> <span class="NLM_lpage">8</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2001&pages=2892-8&author=C.+W.+Gregoryauthor=R.+T.+Johnsonauthor=J.+L.+Mohlerauthor=F.+S.+Frenchauthor=E.+M.+Wilson&title=Androgen+receptor+stabilization+in+recurrent+prostate+cancer+is+associated+with+hypersensitivity+to+low+androgen"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGregory%26aufirst%3DC.%2BW.%26aulast%3DJohnson%26aufirst%3DR.%2BT.%26aulast%3DMohler%26aufirst%3DJ.%2BL.%26aulast%3DFrench%26aufirst%3DF.%2BS.%26aulast%3DWilson%26aufirst%3DE.%2BM.%26atitle%3DAndrogen%2520receptor%2520stabilization%2520in%2520recurrent%2520prostate%2520cancer%2520is%2520associated%2520with%2520hypersensitivity%2520to%2520low%2520androgen%26jtitle%3DCancer%2520Res.%26date%3D2001%26volume%3D61%26spage%3D2892%26epage%3D8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Thalmann, G. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anezinis, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhau, H. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, E. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hopwood, V. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pathak, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">von Eschenbach, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, L. W.</span><span> </span><span class="NLM_article-title">Androgen-independent cancer progression and bone metastasis in the LNCaP model of human prostate cancer</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">2577</span><span class="NLM_x">–</span> <span class="NLM_lpage">81</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=1994&pages=2577-81&author=G.+N.+Thalmannauthor=P.+E.+Anezinisauthor=S.+M.+Changauthor=H.+E.+Zhauauthor=E.+E.+Kimauthor=V.+L.+Hopwoodauthor=S.+Pathakauthor=A.+C.+von+Eschenbachauthor=L.+W.+Chung&title=Androgen-independent+cancer+progression+and+bone+metastasis+in+the+LNCaP+model+of+human+prostate+cancer"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DThalmann%26aufirst%3DG.%2BN.%26aulast%3DAnezinis%26aufirst%3DP.%2BE.%26aulast%3DChang%26aufirst%3DS.%2BM.%26aulast%3DZhau%26aufirst%3DH.%2BE.%26aulast%3DKim%26aufirst%3DE.%2BE.%26aulast%3DHopwood%26aufirst%3DV.%2BL.%26aulast%3DPathak%26aufirst%3DS.%26aulast%3Dvon%2BEschenbach%26aufirst%3DA.%2BC.%26aulast%3DChung%26aufirst%3DL.%2BW.%26atitle%3DAndrogen-independent%2520cancer%2520progression%2520and%2520bone%2520metastasis%2520in%2520the%2520LNCaP%2520model%2520of%2520human%2520prostate%2520cancer%26jtitle%3DCancer%2520Res.%26date%3D1994%26volume%3D54%26spage%3D2577%26epage%3D81" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit29a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Chen, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whetstine, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hong, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davrazou, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kutateladze, T. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simpson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mao, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pan, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dai, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hagman, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hansen, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, G.</span><span> </span><span class="NLM_article-title">Structural insights into histone demethylation by JMJD2 family members</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">125</span><span class="NLM_x">, </span> <span class="NLM_fpage">691</span><span class="NLM_x">–</span> <span class="NLM_lpage">702</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2006&pages=691-702&author=Z.+Chenauthor=J.+Zangauthor=J.+Whetstineauthor=X.+Hongauthor=F.+Davrazouauthor=T.+G.+Kutateladzeauthor=M.+Simpsonauthor=Q.+Maoauthor=C.+H.+Panauthor=S.+Daiauthor=J.+Hagmanauthor=K.+Hansenauthor=Y.+Shiauthor=G.+Zhang&title=Structural+insights+into+histone+demethylation+by+JMJD2+family+members"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29a&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DZang%26aufirst%3DJ.%26aulast%3DWhetstine%26aufirst%3DJ.%26aulast%3DHong%26aufirst%3DX.%26aulast%3DDavrazou%26aufirst%3DF.%26aulast%3DKutateladze%26aufirst%3DT.%2BG.%26aulast%3DSimpson%26aufirst%3DM.%26aulast%3DMao%26aufirst%3DQ.%26aulast%3DPan%26aufirst%3DC.%2BH.%26aulast%3DDai%26aufirst%3DS.%26aulast%3DHagman%26aufirst%3DJ.%26aulast%3DHansen%26aufirst%3DK.%26aulast%3DShi%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DG.%26atitle%3DStructural%2520insights%2520into%2520histone%2520demethylation%2520by%2520JMJD2%2520family%2520members%26jtitle%3DCell%26date%3D2006%26volume%3D125%26spage%3D691%26epage%3D702" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit29b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Hillringhaus, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yue, W. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rose, N. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ng, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gileadi, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loenarz, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bello, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bray, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oppermann, U.</span><span> </span><span class="NLM_article-title">Structural and evolutionary basis for the dual substrate selectivity of human KDM4 histone demethylase family</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">286</span><span class="NLM_x">, </span> <span class="NLM_fpage">41616</span><span class="NLM_x">–</span> <span class="NLM_lpage">25</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=286&publication_year=2011&pages=41616-25&author=L.+Hillringhausauthor=W.+W.+Yueauthor=N.+R.+Roseauthor=S.+S.+Ngauthor=C.+Gileadiauthor=C.+Loenarzauthor=S.+H.+Belloauthor=J.+E.+Brayauthor=C.+J.+Schofieldauthor=U.+Oppermann&title=Structural+and+evolutionary+basis+for+the+dual+substrate+selectivity+of+human+KDM4+histone+demethylase+family"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29b&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHillringhaus%26aufirst%3DL.%26aulast%3DYue%26aufirst%3DW.%2BW.%26aulast%3DRose%26aufirst%3DN.%2BR.%26aulast%3DNg%26aufirst%3DS.%2BS.%26aulast%3DGileadi%26aufirst%3DC.%26aulast%3DLoenarz%26aufirst%3DC.%26aulast%3DBello%26aufirst%3DS.%2BH.%26aulast%3DBray%26aufirst%3DJ.%2BE.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26aulast%3DOppermann%26aufirst%3DU.%26atitle%3DStructural%2520and%2520evolutionary%2520basis%2520for%2520the%2520dual%2520substrate%2520selectivity%2520of%2520human%2520KDM4%2520histone%2520demethylase%2520family%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2011%26volume%3D286%26spage%3D41616%26epage%3D25" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Whetstine, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nottke, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lan, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huarte, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smolikov, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spooner, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Colaiacovo, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Y.</span><span> </span><span class="NLM_article-title">Reversal of histone lysine trimethylation by the JMJD2 family of histone demethylases</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">125</span><span class="NLM_x">, </span> <span class="NLM_fpage">467</span><span class="NLM_x">–</span> <span class="NLM_lpage">81</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2006&pages=467-81&author=J.+R.+Whetstineauthor=A.+Nottkeauthor=F.+Lanauthor=M.+Huarteauthor=S.+Smolikovauthor=Z.+Chenauthor=E.+Spoonerauthor=E.+Liauthor=G.+Zhangauthor=M.+Colaiacovoauthor=Y.+Shi&title=Reversal+of+histone+lysine+trimethylation+by+the+JMJD2+family+of+histone+demethylases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWhetstine%26aufirst%3DJ.%2BR.%26aulast%3DNottke%26aufirst%3DA.%26aulast%3DLan%26aufirst%3DF.%26aulast%3DHuarte%26aufirst%3DM.%26aulast%3DSmolikov%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DSpooner%26aufirst%3DE.%26aulast%3DLi%26aufirst%3DE.%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DColaiacovo%26aufirst%3DM.%26aulast%3DShi%26aufirst%3DY.%26atitle%3DReversal%2520of%2520histone%2520lysine%2520trimethylation%2520by%2520the%2520JMJD2%2520family%2520of%2520histone%2520demethylases%26jtitle%3DCell%26date%3D2006%26volume%3D125%26spage%3D467%26epage%3D81" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Couture, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Collazo, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ortiz-Tello, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brunzelle, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trievel, R. C.</span><span> </span><span class="NLM_article-title">Specificity and mechanism of JMJD2A, a trimethyllysine-specific histone demethylase</span> <span class="citation_source-journal">Nat. Struct. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">689</span><span class="NLM_x">–</span> <span class="NLM_lpage">95</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Fjm500249n&amp;key=10.1038%2Fnsmb1273" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Fjm500249n&amp;key=17589523" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Fjm500249n&amp;key=1%3ACAS%3A528%3ADC%252BD2sXosVyktLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2007&pages=689-95&author=J.+F.+Coutureauthor=E.+Collazoauthor=P.+A.+Ortiz-Telloauthor=J.+S.+Brunzelleauthor=R.+C.+Trievel&title=Specificity+and+mechanism+of+JMJD2A%2C+a+trimethyllysine-specific+histone+demethylase"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Specificity and mechanism of JMJD2A, a trimethyllysine-specific histone demethylase</span></div><div class="casAuthors">Couture, Jean-Francois; Collazo, Evys; Ortiz-Tello, Patricia A.; Brunzelle, Joseph S.; Trievel, Raymond C.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">689-695</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">JMJD2A is a JmjC histone demethylase (HDM) that catalyzes the demethylation of di- and trimethylated Lys9 and Lys36 in histone H3 (H3K9me2/3 and H3K36me2/3).  Here we present the crystal structures of the JMJD2A catalytic domain in complex with H3K9me3, H3K36me2 and H3K36me3 peptides.  The structures reveal that histone substrates are recognized through a network of backbone hydrogen bonds and hydrophobic interactions that deposit the trimethyllysine into the active site.  The trimethylated ε-ammonium cation is coordinated within a methylammonium-binding pocket through carbon-oxygen (CH···O) hydrogen bonds that position one of the ζ-Me groups adjacent to the Fe(II) center for hydroxylation and demethylation.  Mutations of the residues comprising this pocket abrogate demethylation by JMJD2A, with the exception of an S288A substitution, which augments activity, particularly toward H3K9me2.  We propose that this residue modulates the methylation-state specificities of JMJD2 enzymes and other trimethyllysine-specific JmjC HDMs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZ8SMMLvfZErVg90H21EOLACvtfcHk0lhnEZ9ggl4fSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXosVyktLY%253D&md5=86d9464f761cbd77c22abc1b3a35478c</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1038%2Fnsmb1273&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb1273%26sid%3Dliteratum%253Aachs%26aulast%3DCouture%26aufirst%3DJ.%2BF.%26aulast%3DCollazo%26aufirst%3DE.%26aulast%3DOrtiz-Tello%26aufirst%3DP.%2BA.%26aulast%3DBrunzelle%26aufirst%3DJ.%2BS.%26aulast%3DTrievel%26aufirst%3DR.%2BC.%26atitle%3DSpecificity%2520and%2520mechanism%2520of%2520JMJD2A%252C%2520a%2520trimethyllysine-specific%2520histone%2520demethylase%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2007%26volume%3D14%26spage%3D689%26epage%3D95" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Winters, D. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cederbaum, A. I.</span><span> </span><span class="NLM_article-title">Oxidation of glycerol to formaldehyde by rat liver microsomes. Effects of cytochrome P-450 inducing agents</span> <span class="citation_source-journal">Biochem. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1990</span><span class="NLM_x">, </span> <span class="NLM_volume">39</span><span class="NLM_x">, </span> <span class="NLM_fpage">697</span><span class="NLM_x">–</span> <span class="NLM_lpage">705</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1990&pages=697-705&author=D.+K.+Wintersauthor=A.+I.+Cederbaum&title=Oxidation+of+glycerol+to+formaldehyde+by+rat+liver+microsomes.+Effects+of+cytochrome+P-450+inducing+agents"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWinters%26aufirst%3DD.%2BK.%26aulast%3DCederbaum%26aufirst%3DA.%2BI.%26atitle%3DOxidation%2520of%2520glycerol%2520to%2520formaldehyde%2520by%2520rat%2520liver%2520microsomes.%2520Effects%2520of%2520cytochrome%2520P-450%2520inducing%2520agents%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D1990%26volume%3D39%26spage%3D697%26epage%3D705" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Rose, N. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ng, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mecinovic, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lienard, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bello, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDonough, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oppermann, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span> </span><span class="NLM_article-title">Inhibitor scaffolds for 2-oxoglutarate-dependent histone lysine demethylases</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">7053</span><span class="NLM_x">–</span> <span class="NLM_lpage">6</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm800936s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Fjm500249n&amp;key=1%3ACAS%3A528%3ADC%252BD1cXht1Ors7rI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=7053-6&author=N.+R.+Roseauthor=S.+S.+Ngauthor=J.+Mecinovicauthor=B.+M.+Lienardauthor=S.+H.+Belloauthor=Z.+Sunauthor=M.+A.+McDonoughauthor=U.+Oppermannauthor=C.+J.+Schofield&title=Inhibitor+scaffolds+for+2-oxoglutarate-dependent+histone+lysine+demethylases"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitor Scaffolds for 2-Oxoglutarate-Dependent Histone Lysine Demethylases</span></div><div class="casAuthors">Rose, Nathan R.; Ng, Stanley S.; Mecinovic, Jasmin; Lienard, Benoit M. R.; Bello, Simon H.; Sun, Zhe; McDonough, Michael A.; Oppermann, Udo; Schofield, Christopher J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">7053-7056</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The dynamic methylation of histone lysyl residues plays an important role in biol. by regulating transcription, maintaining genomic integrity, and by contributing to epigenetic effects.  Here we describe a variety of inhibitor scaffolds that inhibit the human 2-oxoglutarate-dependent JMJD2 subfamily of histone demethylases.  Combined with structural data, these chem. starting points will be useful to generate small-mol. probes to analyze the physiol. roles of these enzymes in epigenetic signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr90HQ8nfBLFrVg90H21EOLACvtfcHk0li2OZ9OdI2gCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXht1Ors7rI&md5=155c830b0544a6c1d5e6eb81d14389de</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fjm800936s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm800936s%26sid%3Dliteratum%253Aachs%26aulast%3DRose%26aufirst%3DN.%2BR.%26aulast%3DNg%26aufirst%3DS.%2BS.%26aulast%3DMecinovic%26aufirst%3DJ.%26aulast%3DLienard%26aufirst%3DB.%2BM.%26aulast%3DBello%26aufirst%3DS.%2BH.%26aulast%3DSun%26aufirst%3DZ.%26aulast%3DMcDonough%26aufirst%3DM.%2BA.%26aulast%3DOppermann%26aufirst%3DU.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26atitle%3DInhibitor%2520scaffolds%2520for%25202-oxoglutarate-dependent%2520histone%2520lysine%2520demethylases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D7053%26epage%3D6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit34a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Grasso, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, Y.-M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robinson, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dhanasekaran, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khan, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quist, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jing, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lonigro, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brenner, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Asangani, I. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ateeq, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chun, S. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siddiqui, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sam, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anstett, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mehra, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prensner, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palanisamy, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ryslik, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vandin, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raphael, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kunju, L. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rhodes, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pienta, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chinnaiyan, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tomlins, S. A.</span><span> </span><span class="NLM_article-title">The mutational landscape of lethal castration-resistant prostate cancer</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">487</span><span class="NLM_x">, </span> <span class="NLM_fpage">239</span><span class="NLM_x">–</span> <span class="NLM_lpage">43</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34a&amp;dbid=16&amp;doi=10.1021%2Fjm500249n&amp;key=10.1038%2Fnature11125" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34a&amp;dbid=8&amp;doi=10.1021%2Fjm500249n&amp;key=22722839" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34a&amp;dbid=32&amp;doi=10.1021%2Fjm500249n&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVWisLfP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=487&publication_year=2012&pages=239-43&author=C.+S.+Grassoauthor=Y.-M.+Wuauthor=D.+R.+Robinsonauthor=X.+Caoauthor=S.+M.+Dhanasekaranauthor=A.+P.+Khanauthor=M.+J.+Quistauthor=X.+Jingauthor=R.+J.+Lonigroauthor=J.+C.+Brennerauthor=I.+A.+Asanganiauthor=B.+Ateeqauthor=S.+Y.+Chunauthor=J.+Siddiquiauthor=L.+Samauthor=M.+Anstettauthor=R.+Mehraauthor=J.+R.+Prensnerauthor=N.+Palanisamyauthor=G.+A.+Ryslikauthor=F.+Vandinauthor=B.+J.+Raphaelauthor=L.+P.+Kunjuauthor=D.+R.+Rhodesauthor=K.+J.+Pientaauthor=A.+M.+Chinnaiyanauthor=S.+A.+Tomlins&title=The+mutational+landscape+of+lethal+castration-resistant+prostate+cancer"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34aR"><div class="casContent"><span class="casTitleNuber">30a</span><div class="casTitle"><span class="NLM_cas:atitle">The mutational landscape of lethal castration-resistant prostate cancer</span></div><div class="casAuthors">Grasso, Catherine S.; Wu, Yi-Mi; Robinson, Dan R.; Cao, Xuhong; Dhanasekaran, Saravana M.; Khan, Amjad P.; Quist, Michael J.; Jing, Xiaojun; Lonigro, Robert J.; Brenner, J. Chad; Asangani, Irfan A.; Ateeq, Bushra; Chun, Sang Y.; Siddiqui, Javed; Sam, Lee; Anstett, Matt; Mehra, Rohit; Prensner, John R.; Palanisamy, Nallasivam; Ryslik, Gregory A.; Vandin, Fabio; Raphael, Benjamin J.; Kunju, Lakshmi P.; Rhodes, Daniel R.; Pienta, Kenneth J.; Chinnaiyan, Arul M.; Tomlins, Scott A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">487</span>
        (<span class="NLM_cas:issue">7406</span>),
    <span class="NLM_cas:pages">239-243</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Characterization of the prostate cancer transcriptome and genome has identified chromosomal rearrangements and copy no. gains and losses, including ETS gene family fusions, PTEN loss and androgen receptor (AR) amplification, which drive prostate cancer development and progression to lethal, metastatic castration-resistant prostate cancer (CRPC).  However, less is known about the role of mutations.  Here we sequenced the exomes of 50 lethal, heavily pre-treated metastatic CRPCs obtained at rapid autopsy (including three different foci from the same patient) and 11 treatment-naive, high-grade localized prostate cancers.  We identified low overall mutation rates even in heavily treated CRPCs (2.00 per megabase) and confirmed the monoclonal origin of lethal CRPC.  Integrating exome copy no. anal. identified disruptions of CHD1 that define a subtype of ETS gene family fusion-neg. prostate cancer.  Similarly, we demonstrate that ETS2, which is deleted in approx. one-third of CRPCs (commonly through TMPRSS2:ERG fusions), is also deregulated through mutation.  Furthermore, we identified recurrent mutations in multiple chromatin- and histone-modifying genes, including MLL2 (mutated in 8.6% of prostate cancers), and demonstrate interaction of the MLL complex with the AR, which is required for AR-mediated signaling.  We also identified novel recurrent mutations in the AR collaborating factor FOXA1, which is mutated in 5 of 147 (3.4%) prostate cancers (both untreated localized prostate cancer and CRPC), and showed that mutated FOXA1 represses androgen signaling and increases tumor growth.  Proteins that phys. interact with the AR, such as the ERG gene fusion product, FOXA1, MLL2, UTX (also known as KDM6A) and ASXL1 were found to be mutated in CRPC.  In summary, we describe the mutational landscape of a heavily treated metastatic cancer, identify novel mechanisms of AR signaling deregulated in prostate cancer, and prioritize candidates for future study.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgzVYOnnaqVLVg90H21EOLACvtfcHk0li2OZ9OdI2gCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVWisLfP&md5=15b1780d9cd2189325baf4e4fec84687</span></div><a href="/servlet/linkout?suffix=cit34a&amp;dbid=16384&amp;doi=10.1038%2Fnature11125&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11125%26sid%3Dliteratum%253Aachs%26aulast%3DGrasso%26aufirst%3DC.%2BS.%26aulast%3DWu%26aufirst%3DY.-M.%26aulast%3DRobinson%26aufirst%3DD.%2BR.%26aulast%3DCao%26aufirst%3DX.%26aulast%3DDhanasekaran%26aufirst%3DS.%2BM.%26aulast%3DKhan%26aufirst%3DA.%2BP.%26aulast%3DQuist%26aufirst%3DM.%2BJ.%26aulast%3DJing%26aufirst%3DX.%26aulast%3DLonigro%26aufirst%3DR.%2BJ.%26aulast%3DBrenner%26aufirst%3DJ.%2BC.%26aulast%3DAsangani%26aufirst%3DI.%2BA.%26aulast%3DAteeq%26aufirst%3DB.%26aulast%3DChun%26aufirst%3DS.%2BY.%26aulast%3DSiddiqui%26aufirst%3DJ.%26aulast%3DSam%26aufirst%3DL.%26aulast%3DAnstett%26aufirst%3DM.%26aulast%3DMehra%26aufirst%3DR.%26aulast%3DPrensner%26aufirst%3DJ.%2BR.%26aulast%3DPalanisamy%26aufirst%3DN.%26aulast%3DRyslik%26aufirst%3DG.%2BA.%26aulast%3DVandin%26aufirst%3DF.%26aulast%3DRaphael%26aufirst%3DB.%2BJ.%26aulast%3DKunju%26aufirst%3DL.%2BP.%26aulast%3DRhodes%26aufirst%3DD.%2BR.%26aulast%3DPienta%26aufirst%3DK.%2BJ.%26aulast%3DChinnaiyan%26aufirst%3DA.%2BM.%26aulast%3DTomlins%26aufirst%3DS.%2BA.%26atitle%3DThe%2520mutational%2520landscape%2520of%2520lethal%2520castration-resistant%2520prostate%2520cancer%26jtitle%3DNature%26date%3D2012%26volume%3D487%26spage%3D239%26epage%3D43" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit34b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Lapointe, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Higgins, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van de Rijn, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bair, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Montgomery, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferrari, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Egevad, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rayford, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bergerheim, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ekman, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DeMarzo, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tibshirani, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Botstein, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brooks, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollack, J. R.</span><span> </span><span class="NLM_article-title">Gene expression profiling identifies clinically relevant subtypes of prostate cancer</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">101</span><span class="NLM_x">, </span> <span class="NLM_fpage">811</span><span class="NLM_x">–</span> <span class="NLM_lpage">6</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2004&pages=811-6&author=J.+Lapointeauthor=C.+Liauthor=J.+P.+Higginsauthor=M.+van+de+Rijnauthor=E.+Bairauthor=K.+Montgomeryauthor=M.+Ferrariauthor=L.+Egevadauthor=W.+Rayfordauthor=U.+Bergerheimauthor=P.+Ekmanauthor=A.+M.+DeMarzoauthor=R.+Tibshiraniauthor=D.+Botsteinauthor=P.+O.+Brownauthor=J.+D.+Brooksauthor=J.+R.+Pollack&title=Gene+expression+profiling+identifies+clinically+relevant+subtypes+of+prostate+cancer"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34b&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLapointe%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DHiggins%26aufirst%3DJ.%2BP.%26aulast%3Dvan%2Bde%2BRijn%26aufirst%3DM.%26aulast%3DBair%26aufirst%3DE.%26aulast%3DMontgomery%26aufirst%3DK.%26aulast%3DFerrari%26aufirst%3DM.%26aulast%3DEgevad%26aufirst%3DL.%26aulast%3DRayford%26aufirst%3DW.%26aulast%3DBergerheim%26aufirst%3DU.%26aulast%3DEkman%26aufirst%3DP.%26aulast%3DDeMarzo%26aufirst%3DA.%2BM.%26aulast%3DTibshirani%26aufirst%3DR.%26aulast%3DBotstein%26aufirst%3DD.%26aulast%3DBrown%26aufirst%3DP.%2BO.%26aulast%3DBrooks%26aufirst%3DJ.%2BD.%26aulast%3DPollack%26aufirst%3DJ.%2BR.%26atitle%3DGene%2520expression%2520profiling%2520identifies%2520clinically%2520relevant%2520subtypes%2520of%2520prostate%2520cancer%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2004%26volume%3D101%26spage%3D811%26epage%3D6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Hsu, K. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, L. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, J. M.</span><span> </span><span class="NLM_article-title">Core site-moiety maps reveal inhibitors and binding mechanisms of orthologous proteins by screening compound libraries</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">e32142</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=e32142&author=K.+C.+Hsuauthor=W.+C.+Chengauthor=Y.+F.+Chenauthor=H.+J.+Wangauthor=L.+T.+Liauthor=W.+C.+Wangauthor=J.+M.+Yang&title=Core+site-moiety+maps+reveal+inhibitors+and+binding+mechanisms+of+orthologous+proteins+by+screening+compound+libraries"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHsu%26aufirst%3DK.%2BC.%26aulast%3DCheng%26aufirst%3DW.%2BC.%26aulast%3DChen%26aufirst%3DY.%2BF.%26aulast%3DWang%26aufirst%3DH.%2BJ.%26aulast%3DLi%26aufirst%3DL.%2BT.%26aulast%3DWang%26aufirst%3DW.%2BC.%26aulast%3DYang%26aufirst%3DJ.%2BM.%26atitle%3DCore%2520site-moiety%2520maps%2520reveal%2520inhibitors%2520and%2520binding%2520mechanisms%2520of%2520orthologous%2520proteins%2520by%2520screening%2520compound%2520libraries%26jtitle%3DPLoS%2520One%26date%3D2012%26volume%3D7%26spage%3De32142" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Black, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Manning, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Rechem, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ladd, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cho, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pineda, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murphy, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Daniels, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Montagna, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lewis, P. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glass, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allis, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dyson, N. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Getz, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whetstine, J. R.</span><span> </span><span class="NLM_article-title">KDM4A lysine demethylase induces site-specific copy gain and rereplication of regions amplified in tumors</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">154</span><span class="NLM_x">, </span> <span class="NLM_fpage">541</span><span class="NLM_x">–</span> <span class="NLM_lpage">55</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=154&publication_year=2013&pages=541-55&author=J.+C.+Blackauthor=A.+L.+Manningauthor=C.+Van+Rechemauthor=J.+Kimauthor=B.+Laddauthor=J.+Choauthor=C.+M.+Pinedaauthor=N.+Murphyauthor=D.+L.+Danielsauthor=C.+Montagnaauthor=P.+W.+Lewisauthor=K.+Glassauthor=C.+D.+Allisauthor=N.+J.+Dysonauthor=G.+Getzauthor=J.+R.+Whetstine&title=KDM4A+lysine+demethylase+induces+site-specific+copy+gain+and+rereplication+of+regions+amplified+in+tumors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBlack%26aufirst%3DJ.%2BC.%26aulast%3DManning%26aufirst%3DA.%2BL.%26aulast%3DVan%2BRechem%26aufirst%3DC.%26aulast%3DKim%26aufirst%3DJ.%26aulast%3DLadd%26aufirst%3DB.%26aulast%3DCho%26aufirst%3DJ.%26aulast%3DPineda%26aufirst%3DC.%2BM.%26aulast%3DMurphy%26aufirst%3DN.%26aulast%3DDaniels%26aufirst%3DD.%2BL.%26aulast%3DMontagna%26aufirst%3DC.%26aulast%3DLewis%26aufirst%3DP.%2BW.%26aulast%3DGlass%26aufirst%3DK.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26aulast%3DDyson%26aufirst%3DN.%2BJ.%26aulast%3DGetz%26aufirst%3DG.%26aulast%3DWhetstine%26aufirst%3DJ.%2BR.%26atitle%3DKDM4A%2520lysine%2520demethylase%2520induces%2520site-specific%2520copy%2520gain%2520and%2520rereplication%2520of%2520regions%2520amplified%2520in%2520tumors%26jtitle%3DCell%26date%3D2013%26volume%3D154%26spage%3D541%26epage%3D55" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Rose, N. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDonough, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">King, O. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawamura, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span> </span><span class="NLM_article-title">Inhibition of 2-oxoglutarate dependent oxygenases</span> <span class="citation_source-journal">Chem. Soc. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">4364</span><span class="NLM_x">–</span> <span class="NLM_lpage">97</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Fjm500249n&amp;key=10.1039%2Fc0cs00203h" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Fjm500249n&amp;key=21390379" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Fjm500249n&amp;key=1%3ACAS%3A528%3ADC%252BC3MXovVemur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2011&pages=4364-97&author=N.+R.+Roseauthor=M.+A.+McDonoughauthor=O.+N.+Kingauthor=A.+Kawamuraauthor=C.+J.+Schofield&title=Inhibition+of+2-oxoglutarate+dependent+oxygenases"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of 2-oxoglutarate dependent oxygenases</span></div><div class="casAuthors">Rose, Nathan R.; McDonough, Michael A.; King, Oliver N. F.; Kawamura, Akane; Schofield, Christopher J.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Society Reviews</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">4364-4397</span>CODEN:
                <span class="NLM_cas:coden">CSRVBR</span>;
        ISSN:<span class="NLM_cas:issn">0306-0012</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  2-Oxoglutarate (2OG) dependent oxygenases are ubiquitous iron enzymes that couple substrate oxidn. to the conversion of 2OG to succinate and carbon dioxide.  In humans their roles include collagen biosynthesis, fatty acid metab., DNA repair, RNA and chromatin modifications, and hypoxic sensing.  Com. applications of 2OG oxygenase inhibitors began with plant growth retardants, and now extend to a clin. used pharmaceutical compd. for cardioprotection.  Several 2OG oxygenases are now being targeted for therapeutic intervention for diseases including anemia, inflammation and cancer.  In this crit. review, we describe studies on the inhibition of 2OG oxygenases, focusing on small mols., and discuss the potential of 2OG oxygenases as therapeutic targets (295 refs.).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqp4Z13BJAo17Vg90H21EOLACvtfcHk0ljePaKlD2uXrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXovVemur8%253D&md5=f7a87e0bc9392d9a29f4ffb7d6f92b44</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1039%2Fc0cs00203h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc0cs00203h%26sid%3Dliteratum%253Aachs%26aulast%3DRose%26aufirst%3DN.%2BR.%26aulast%3DMcDonough%26aufirst%3DM.%2BA.%26aulast%3DKing%26aufirst%3DO.%2BN.%26aulast%3DKawamura%26aufirst%3DA.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26atitle%3DInhibition%2520of%25202-oxoglutarate%2520dependent%2520oxygenases%26jtitle%3DChem.%2520Soc.%2520Rev.%26date%3D2011%26volume%3D40%26spage%3D4364%26epage%3D97" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varghese, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dellinger, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kumar, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Best, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruiz, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bruick, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pena-Llopis, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Babinski, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frantz, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brekken, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quinn, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeonov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Easmon, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martinez, E. D.</span><span> </span><span class="NLM_article-title">A small molecule modulates Jumonji histone demethylase activity and selectively inhibits cancer growth</span> <span class="citation_source-journal">Nat. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">2035</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Fjm500249n&amp;key=10.1038%2Fncomms3035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Fjm500249n&amp;key=23792809" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Fjm500249n&amp;key=1%3ACAS%3A280%3ADC%252BC3sjksV2iug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=2035&author=L.+Wangauthor=J.+Changauthor=D.+Vargheseauthor=M.+Dellingerauthor=S.+Kumarauthor=A.+M.+Bestauthor=J.+Ruizauthor=R.+Bruickauthor=S.+Pena-Llopisauthor=J.+Xuauthor=D.+J.+Babinskiauthor=D.+E.+Frantzauthor=R.+A.+Brekkenauthor=A.+M.+Quinnauthor=A.+Simeonovauthor=J.+Easmonauthor=E.+D.+Martinez&title=A+small+molecule+modulates+Jumonji+histone+demethylase+activity+and+selectively+inhibits+cancer+growth"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">A small molecule modulates Jumonji histone demethylase activity and selectively inhibits cancer growth</span></div><div class="casAuthors">Wang Lei; Chang Jianjun; Varghese Diana; Dellinger Michael; Kumar Subodh; Best Anne M; Ruiz Julio; Bruick Richard; Pena-Llopis Samuel; Xu Junjie; Babinski David J; Frantz Doug E; Brekken Rolf A; Quinn Amy M; Simeonov Anton; Easmon Johnny; Martinez Elisabeth D</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">2035</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The pharmacological inhibition of general transcriptional regulators has the potential to block growth through targeting multiple tumorigenic signalling pathways simultaneously.  Here, using an innovative cell-based screen, we identify a structurally unique small molecule (named JIB-04) that specifically inhibits the activity of the Jumonji family of histone demethylases in vitro, in cancer cells, and in tumours in vivo.  Unlike known inhibitors, JIB-04 is not a competitive inhibitor of α-ketoglutarate.  In cancer, but not in patient-matched normal cells, JIB-04 alters a subset of transcriptional pathways and blocks viability.  In mice, JIB-04 reduces tumour burden and prolongs survival.  Importantly, we find that patients with breast tumours that overexpress Jumonji demethylases have significantly lower survival.  Thus, JIB-04, a novel inhibitor of Jumonji demethylases in vitro and in vivo, constitutes a unique potential therapeutic and research tool against cancer, and validates the use of unbiased cellular screens to discover chemical modulators with disease relevance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ5QxHbn8WTZg20zIbnCCI3fW6udTcc2eYw_bDlo8Bd7Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3sjksV2iug%253D%253D&md5=37aa303d173d027fd711bc4ef7fd841b</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1038%2Fncomms3035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms3035%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.%26aulast%3DChang%26aufirst%3DJ.%26aulast%3DVarghese%26aufirst%3DD.%26aulast%3DDellinger%26aufirst%3DM.%26aulast%3DKumar%26aufirst%3DS.%26aulast%3DBest%26aufirst%3DA.%2BM.%26aulast%3DRuiz%26aufirst%3DJ.%26aulast%3DBruick%26aufirst%3DR.%26aulast%3DPena-Llopis%26aufirst%3DS.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DBabinski%26aufirst%3DD.%2BJ.%26aulast%3DFrantz%26aufirst%3DD.%2BE.%26aulast%3DBrekken%26aufirst%3DR.%2BA.%26aulast%3DQuinn%26aufirst%3DA.%2BM.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DEasmon%26aufirst%3DJ.%26aulast%3DMartinez%26aufirst%3DE.%2BD.%26atitle%3DA%2520small%2520molecule%2520modulates%2520Jumonji%2520histone%2520demethylase%2520activity%2520and%2520selectively%2520inhibits%2520cancer%2520growth%26jtitle%3DNat.%2520Commun.%26date%3D2013%26volume%3D4%26spage%3D2035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2YBS" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2YBS','PDB','2YBS'); return false;">PDB: 2YBS</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2P5B" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2P5B','PDB','2P5B'); return false;">PDB: 2P5B</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4LXL" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4LXL','PDB','4LXL'); return false;">PDB: 4LXL</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2XML" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2XML','PDB','2XML'); return false;">PDB: 2XML</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4HON" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4HON','PDB','4HON'); return false;">PDB: 4HON</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2W2I" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2W2I','PDB','2W2I'); return false;">PDB: 2W2I</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2YBK" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2YBK','PDB','2YBK'); return false;">PDB: 2YBK</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2OQ6" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2OQ6','PDB','2OQ6'); return false;">PDB: 2OQ6</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2Q8C" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2Q8C','PDB','2Q8C'); return false;">PDB: 2Q8C</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2P58" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2P58','PDB','2P58'); return false;">PDB: 2P58</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2VD7" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2VD7','PDB','2VD7'); return false;">PDB: 2VD7</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3NJY" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3NJY','PDB','3NJY'); return false;">PDB: 3NJY</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i22"><a href="/doi/suppl/10.1021/jm500249n">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_68887"></div></div></div></div></div><hr /></hr><p class="last">Binding pocket analysis in KDM4s (Figure S1); western blot of KDM4A and KDM4B that demethylate H3K9me3/me2 and H3K36me3/me2 but not H3K27me3 (Figure S2); superposition of the KDM4B·<b>2</b>·H3K9me3 structure and the KDM4B·<b>4</b> docked pose (Figure S3); selective inhibitory effect of compound <b>4</b> (Figure S4); KDM4A and KDM4B expression data in PCa sites (Figure S5); downregulation of AR target genes in KDM4A/KDM4B-knockdown and compound <b>4</b>-treated cells (Figure S6); inhibition effects of selected hits on KDM4A and KDM4B (Table S1); and crystallographic data and refinement statistics for KDM4B·<b>2</b>·H3K9me3 (Table S2). This material is available free of charge via the Internet at <a class="extLink" href="http://pubs.acs.org">http://pubs.acs.org</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm500249n/suppl_file/jm500249n_si_001.pdf">jm500249n_si_001.pdf (1.03 MB)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The atomic coordinates of KDM4B have been deposited as Protein Data Bank entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4LXL">4LXL</a>.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Fjm500249n&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2014.57.issue-14%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Barticle%3Aarticle%3A10.1021%2Fjm500249n" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/jm500249n" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                5MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679a410d1ed73cca","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
