
<html lang="en"     class="pb-page"  data-request-id="82fc29b1-9a40-49d8-bd38-88b10fa520a3"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";article:article:10.1021/acs.jmedchem.0c01133;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;issue:issue:10.1021/jmcmar.2021.64.issue-2;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="GPR52 Antagonist Reduces Huntingtin Levels and Ameliorates Huntington’s Disease-Related Phenotypes" /></meta><meta name="dc.Creator" content="Congcong  Wang" /></meta><meta name="dc.Creator" content="Yu-Fang  Zhang" /></meta><meta name="dc.Creator" content="Shimeng  Guo" /></meta><meta name="dc.Creator" content="Quan  Zhao" /></meta><meta name="dc.Creator" content="Yanping  Zeng" /></meta><meta name="dc.Creator" content="Zhicheng  Xie" /></meta><meta name="dc.Creator" content="Xin  Xie" /></meta><meta name="dc.Creator" content="Boxun  Lu" /></meta><meta name="dc.Creator" content="Youhong  Hu" /></meta><meta name="dc.Description" content="GPR52 is an orphan G protein-coupled receptor (GPCR) that has been recently implicated as a potential drug target of Huntington’s disease (HD), an incurable monogenic neurodegenerative disorder. In..." /></meta><meta name="Description" content="GPR52 is an orphan G protein-coupled receptor (GPCR) that has been recently implicated as a potential drug target of Huntington’s disease (HD), an incurable monogenic neurodegenerative disorder. In..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="November 13, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c01133" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2020 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01133" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c01133" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01133" /></link>
        
    
    

<title>GPR52 Antagonist Reduces Huntingtin Levels and Ameliorates Huntington’s Disease-Related Phenotypes | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c01133" /></meta><meta property="og:title" content="GPR52 Antagonist Reduces Huntingtin Levels and Ameliorates Huntington’s Disease-Related Phenotypes" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01133/20210121/images/large/jm0c01133_0011.jpeg" /></meta><meta property="og:description" content="GPR52 is an orphan G protein-coupled receptor (GPCR) that has been recently implicated as a potential drug target of Huntington’s disease (HD), an incurable monogenic neurodegenerative disorder. In this research, we found that striatal knockdown of GPR52 reduces mHTT levels in adult HdhQ140 mice, validating GPR52 as an HD target. In addition, we discovered a highly potent and specific GPR52 antagonist Comp-43 with an IC50 value of 0.63 μM by a structure–activity relationship (SAR) study. Further studies showed that Comp-43 reduces mHTT levels by targeting GPR52 and promotes survival of mouse primary striatal neurons. Moreover, in vivo study showed that Comp-43 not only reduces mHTT levels but also rescues HD-related phenotypes in HdhQ140 mice. Taken together, our study confirms that inhibition of GPR52 is a promising strategy for HD therapy, and the GPR52 antagonist Comp-43 might serve as a lead compound for further investigation." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c01133"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01133">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c01133&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c01133&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c01133&amp;href=/doi/10.1021/acs.jmedchem.0c01133" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, 64</span><span class="cit-fg-issue">, 2</span><span class="cit-fg-pageRange">, 941-957</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/64/2" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c02235" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Featured Article</span><a href="/doi/10.1021/acs.jmedchem.0c01026" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="article_relation"><span class="article_relation-correction"><i class="icon-asterisk"></i><span class="relation-text"><a href="/doi/10.1021/acs.jmedchem.0c01981"><span class="relation-type">ADDITION / CORRECTION</span><span class="relation-desc">This article has been corrected. View the notice.</span></a></span></span></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">GPR52 Antagonist Reduces Huntingtin Levels and Ameliorates Huntington’s Disease-Related Phenotypes</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Congcong Wang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Congcong Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Neurology Department at Huashan Hospital, State Key Laboratory of Medical Neurobiology and MOE Frontiers Center for Brain Science, School of Life Sciences, Fudan University, Shanghai 200438, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Congcong++Wang">Congcong Wang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yu-Fang Zhang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yu-Fang Zhang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yu-Fang++Zhang">Yu-Fang Zhang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Shimeng Guo</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Shimeng Guo</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Chinese Materia Medica, Nanjing University of Chinese Medicine, Nanjing 210046, China</div><div class="loa-info-affiliations-info">CAS Key Laboratory of Receptor Research, National Center for Drug Screening, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Shimeng++Guo">Shimeng Guo</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Quan Zhao</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Quan Zhao</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Neurology Department at Huashan Hospital, State Key Laboratory of Medical Neurobiology and MOE Frontiers Center for Brain Science, School of Life Sciences, Fudan University, Shanghai 200438, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Quan++Zhao">Quan Zhao</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yanping Zeng</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yanping Zeng</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yanping++Zeng">Yanping Zeng</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Zhicheng Xie</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Zhicheng Xie</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Zhicheng++Xie">Zhicheng Xie</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Xin Xie</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xin Xie</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Chinese Materia Medica, Nanjing University of Chinese Medicine, Nanjing 210046, China</div><div class="loa-info-affiliations-info">CAS Key Laboratory of Receptor Research, National Center for Drug Screening, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, Shanghai 201203, China</div><div class="loa-info-affiliations-info">School of Life Science and Technology, ShanghaiTech University, Shanghai 201210, China</div><div class="loa-info-affiliations-info">School of Pharmaceutical Science and Technology, Hangzhou Institute for Advanced Study, UCAS, Hangzhou 310024, China</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#374f4f5e5277445e5a5a195654195459"><span class="__cf_email__" data-cfemail="582020313d182b31353576393b763b36">[email protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xin++Xie">Xin Xie</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Boxun Lu</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Boxun Lu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Neurology Department at Huashan Hospital, State Key Laboratory of Medical Neurobiology and MOE Frontiers Center for Brain Science, School of Life Sciences, Fudan University, Shanghai 200438, China</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#432f36212c3b362d03253627222d6d2627366d202d"><span class="__cf_email__" data-cfemail="98f4edfaf7e0edf6d8feedfcf9f6b6fdfcedb6fbf6">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Boxun++Lu">Boxun Lu</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Youhong Hu</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Youhong Hu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</div><div class="loa-info-affiliations-info">School of Pharmaceutical Science and Technology, Hangzhou Institute for Advanced Study, UCAS, Hangzhou 310024, China</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#abd2c3c3deebd8c2c6c685cac885c8c5"><span class="__cf_email__" data-cfemail="a4ddccccd1e4d7cdc9c98ac5c78ac7ca">[email protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Youhong++Hu">Youhong Hu</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-1770-6272" title="Orcid link">http://orcid.org/0000-0003-1770-6272</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01133&amp;href=/doi/10.1021%2Facs.jmedchem.0c01133" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, 64</span><span class="cit-issue">, 2</span><span class="cit-pageRange">, 941–957</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">November 13, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>2 July 2020</li><li><span class="item_label"><b>Published</b> online</span>13 November 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 28 January 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c01133" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01133</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D941%26pageCount%3D17%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DCongcong%2BWang%252C%2BYu-Fang%2BZhang%252C%2BShimeng%2BGuo%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D64%26issueNum%3D2%26contentID%3Dacs.jmedchem.0c01133%26title%3DGPR52%2BAntagonist%2BReduces%2BHuntingtin%2BLevels%2Band%2BAmeliorates%2BHuntington%25E2%2580%2599s%2BDisease-Related%2BPhenotypes%26numPages%3D17%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D957%26publicationDate%3DJanuary%2B2021">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c01133"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2000</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">1</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c01133" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;GPR52 Antagonist Reduces Huntingtin Levels and Ameliorates Huntington’s Disease-Related Phenotypes&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Congcong&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Yu-Fang&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;Shimeng&quot;,&quot;last_name&quot;:&quot;Guo&quot;},{&quot;first_name&quot;:&quot;Quan&quot;,&quot;last_name&quot;:&quot;Zhao&quot;},{&quot;first_name&quot;:&quot;Yanping&quot;,&quot;last_name&quot;:&quot;Zeng&quot;},{&quot;first_name&quot;:&quot;Zhicheng&quot;,&quot;last_name&quot;:&quot;Xie&quot;},{&quot;first_name&quot;:&quot;Xin&quot;,&quot;last_name&quot;:&quot;Xie&quot;},{&quot;first_name&quot;:&quot;Boxun&quot;,&quot;last_name&quot;:&quot;Lu&quot;},{&quot;first_name&quot;:&quot;Youhong&quot;,&quot;last_name&quot;:&quot;Hu&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;11&quot;,&quot;day&quot;:&quot;13&quot;,&quot;issue&quot;:&quot;2&quot;,&quot;volume&quot;:&quot;64&quot;,&quot;pages&quot;:&quot;941-957&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c01133&quot;},&quot;abstract&quot;:&quot;GPR52 is an orphan G protein-coupled receptor (GPCR) that has been recently implicated as a potential drug target of Huntington’s disease (HD), an incurable monogenic neurodegenerative disorder. In this research, we found that striatal knockdown of GPR52 reduces mHTT levels in adult HdhQ140 mice, validating GPR52 as an HD target. In addition, we discovered a highly potent and specific GPR52 antagonist Comp-43 with an IC50 value of 0.63 μM by a structure–activity relationship (SAR) study. Further studies showed that Comp-43 reduces mHTT levels by targeting GPR52 and promotes survival of mouse primary striatal neurons. Moreover, in vivo study showed that Comp-43 not only reduces mHTT levels but also rescues HD-related phenotypes in HdhQ140 mice. Taken together, our study confirms that inhibition of GPR52 is a promising strategy for HD therapy, and the GPR52 antagonist Comp-43 might serve as a lead compound for further investigation.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01133&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01133" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01133&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01133" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01133&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01133" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01133&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01133&amp;href=/doi/10.1021/acs.jmedchem.0c01133" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c01133" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c01133" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (7 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01133&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c01133%26sid%3Dliteratum%253Aachs%26pmid%3D33185430%26genre%3Darticle%26aulast%3DWang%26date%3D2021%26atitle%3DGPR52%2BAntagonist%2BReduces%2BHuntingtin%2BLevels%2Band%2BAmeliorates%2BHuntington%25E2%2580%2599s%2BDisease-Related%2BPhenotypes%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D64%26issue%3D2%26spage%3D941%26epage%3D957%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291909" title="Immunology">Immunology</a>,</li><li><a href="/action/doSearch?ConceptID=291171" title="Antagonists">Antagonists</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292464" title="Rodent models">Rodent models</a>,</li><li><a href="/action/doSearch?ConceptID=290975" title="Biopolymers">Biopolymers</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/64/2" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/jmcmar.2021.64.issue-2/20210128/jmcmar.2021.64.issue-2.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01133/20210121/images/medium/jm0c01133_0011.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01133/20210121/images/large/jm0c01133_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01133&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">GPR52 is an orphan G protein-coupled receptor (GPCR) that has been recently implicated as a potential drug target of Huntington’s disease (HD), an incurable monogenic neurodegenerative disorder. In this research, we found that striatal knockdown of GPR52 reduces mHTT levels in adult HdhQ140 mice, validating GPR52 as an HD target. In addition, we discovered a highly potent and specific GPR52 antagonist Comp-<b>43</b> with an IC<sub>50</sub> value of 0.63 μM by a structure–activity relationship (SAR) study. Further studies showed that Comp-<b>43</b> reduces mHTT levels by targeting GPR52 and promotes survival of mouse primary striatal neurons. Moreover, <i>in vivo</i> study showed that Comp-<b>43</b> not only reduces mHTT levels but also rescues HD-related phenotypes in HdhQ140 mice. Taken together, our study confirms that inhibition of GPR52 is a promising strategy for HD therapy, and the GPR52 antagonist Comp-<b>43</b> might serve as a lead compound for further investigation.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_35174" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_35174" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Huntington’s disease (HD) is a monogenetic neurodegenerative disease caused by the mutation of the huntingtin gene, which encodes the mutant huntingtin protein (mHTT) with expanded polyglutamine tract (polyQ).<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Multiple toxic mHTT species, from soluble monomeric/oligomeric to aggregates and inclusions, contribute to the neuronal death found predominantly in the striatum that leads to HD pathogenesis.<a onclick="showRef(event, 'ref2 ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref2 ref3 ref4">(2−4)</a> Numerous groups have also suggested that the reduction of mHTT through mitigated production or enhanced degradation rescues HD-associated phenotypes.<a onclick="showRef(event, 'ref5 ref6 ref7 ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref5 ref6 ref7 ref8 ref9">(5−9)</a></div><div class="NLM_p">Current mHTT reduction approaches focus on direct delivery of <i>Htt</i> silencing modalities into the brain, including small interference RNAs (siRNAs), short hairpin RNAs (shRNAs), and antisense oligonucleotides (ASO).<a onclick="showRef(event, 'ref10 ref11 ref12 ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref10 ref11 ref12 ref13 ref14">(10−14)</a> Notably, both allele-selective and nonselective reduction of mHTT and wild-type HTT (wtHTT) show beneficial effects.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> However, these gene therapies have limitations such as delivery efficiency. Recently, several signal pathways have been demonstrated to modulate HTT levels.<a onclick="showRef(event, 'ref8 ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref8 ref16 ref17">(8,16,17)</a> Among them, GPR52, an orphan G protein-coupled receptor (GPCR) highly expressed in the striatum,<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> has been demonstrated to modulate mHTT levels in various models.<a onclick="showRef(event, 'ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref18 ref19">(18,19)</a> Embryonic knockdown of <i>Gpr</i>52 rescues disease-associated phenotypes in HD mice. In addition, a novel GPR52 antagonist <b>E7</b> has been found to improve HD-related pathogenesis through the intracerebroventricular (ICV) delivery.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> However, it remains unclear whether the regulatory role of GPR52 exists in adult HD. It would also be interesting to discover more potent and druggable GPR52 antagonists, which may cross the blood–brain barrier to reduce mHTT levels and rescue HD-related pathology <i>in vivo</i>.</div><div class="NLM_p last">In this research, we found that knockdown of <i>Gpr</i>52 in adult HdhQ140 mice reduced mHTT levels. We also performed the high-throughput screening from libraries containing about 10 000 compounds and identified another GPR52 antagonist <b>F11</b> ((<i>E</i>)-1,7-diphenylhept-4-en-3-one). Based on <b>F11</b>, we explored the structure–activity relationship (SAR) and found a highly potent compound <b>43</b> (Comp-<b>43</b>), which could reduce mHTT levels and promote neuronal survival in striatal cells. More importantly, Comp-<b>43</b> reached the brain at a therapeutically relevant level after intraperitoneal treatment and reduced mHTT levels as well as the disease-related phenotypes in HdhQ140 mice.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_53530" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_53530" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Striatal Knockdown of GPR52 Reduces mHTT Levels in Adult HdhQ140 Mice</h3><div class="NLM_p">We have previously shown heterozygous or homozygous embryonic knockout of <i>Gpr</i>52 rescues disease-associated phenotypes in HD mice.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> To further confirm the therapeutic potential of lowering GPR52 in adult HD, we investigated the effects of lowering GPR52 in the striatum of adult HD mice (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A). First, adeno-associated viruses (AAV) expressing either shRNAs targeting <i>Gpr</i>52 (sh<i>Gpr</i>52) or the nontargeting shRNA control (shNT) were generated and the efficiencies were evaluated in STHdhQ7/Q111 cells, a mouse striatal cell line expressing both wtHTT and mHTT with 7Q and 111Q mutations.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> As shown in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B, <i>Gpr</i>52 mRNA levels were significantly reduced in STHdhQ7/Q111 cells expressing shRNAs targeting GPR52 (shG7 and shG10), compared to the shNT control. Then, we respectively injected shG7, shG10, and the shNT control into bilateral striata of 4-month-old HdhQ140 mice, a well-established knock-in HD mouse model expressing mHTT proteins with 140Q.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> After 6 weeks, we observed that injection of shG7 and shG10, but not shNT, remarkably decreased GPR52 mRNA levels in the striatum, indicating the effective striatal downregulation of GPR52 (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C).</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01133/20210121/images/medium/jm0c01133_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01133/20210121/images/large/jm0c01133_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Striatal knockdown of GPR52 reduces mHTT levels in adult HdhQ140 mice. (A) Experimental strategy for striatal knockdown of GPR52 in adult HdhQ140 mice. Generally, adeno-associated viruses (AAV) expressing either shRNAs targeting GPR52 (shGPR52) or the nontargeting shRNA control (shNT) were generated, followed by bi-striatal injection into 4-month-old HdhQ140 mice. After 6 weeks, mice striata were collected for further analysis. (B) QPCR analysis of GPR52 mRNA levels in STHdhQ7/Q111 cells after the viral infection with shGPR52-7 (shG7) or shGPR52-10 (shG10) for 72 h. Plots represent mean ± standard error of the mean (S.E.M.) <i>N</i> = 3–4 and the one-way analysis of variance (ANOVA) with the post hoc Bonferroni’s multiple comparisons was used for statistical analysis. *** <i>P</i> < 0.001, versus the shNT control. (C) mRNA levels of GPR52 in striata of adult HdhQ140 mice 6 weeks post bi-striatal injection with shG7 or shG10. Plots represent mean ± S.E.M. <i>N</i> = 3–4 mice per group. ** <i>P</i> < 0.01 as determined by the one-way ANOVA with the post hoc Bonferroni’s multiple comparisons. (D) TR-FRET analysis of mHTT levels with 2B7/2166 antibody pair in striata of HdhQ140 mice. Mice were bi-striatally injected with shG7 or shG10. Six weeks later, mice striata were obtained for further analysis. Plots represent mean ± S.E.M. <i>N</i> = 3–4 per group. The one-way ANOVA with the post hoc Bonferroni’s multiple comparisons was used. * <i>P</i> < 0.05, ** <i>P</i> < 0.01. (E) WB analysis of mHTT levels using Ab1 antibody in striata of HdhQ140 mice. Representative images are shown. (F) Quantification of WB analysis of mHTT levels in (A). Plots represent mean ± S.E.M. <i>N</i> = 3 per group. One-way ANOVA with the post hoc Bonferroni’s multiple comparisons was used. * <i>P</i> < 0.05.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01133/20210121/images/large/jm0c01133_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01133&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Further, we applied the time-resolved fluorescence resonance energy transfer (TR-FRET) assay to examine the total HTT levels.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> Two antibodies targeting HTT are labeled with terbium or D2 fluorophores, respectively, and the energy transfer between them occurs when they are in spatial proximity by binding to the same HTT molecular. As a result, the homogeneous time-resolved fluorescence (HTRF) efficiency is proportional to the HTT levels. However, considering the mHTT-specific antibody pair 2B7/MW1 is not suitable for tissue or cellular samples from mice because of nonspecific signals as reported previously,<a onclick="showRef(event, 'ref8 ref23'); return false;" href="javascript:void(0);" class="ref ref8 ref23">(8,23)</a> we applied 2B7-terbium and 2166-D2 antibodies, which target the N-terminal 17 amino acids and 181–810 amino acids of HTT, respectively. Thus, the 2B7/2166 antibody pair examines the total HTT levels including mHTT and/or wtHTT depending on the genotypes in all TR-FRET assays. As shown in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>D, in homozygous HdhQ140 mice, which express mHTT proteins with 140Q, we found that the expression of shG7 and shG10 significantly decreased mHTT levels compared to shNT in the striatum. We also observed a remarkable reduction of mHTT levels by western blot (WB) (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>E,F). Together, our data demonstrate that striatal knockdown of GPR52 reduces mHTT levels in adult HD mice, further validating GPR52 as a therapeutic target.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Design and Synthesis of Specific GPR52 Antagonists</h3><div class="NLM_p">Our previous study identified a sesquiterpene lactone <b>E7</b> as a novel GPR52 inhibitor, which rescued HD symptoms by the intracerebroventricular (ICV) delivery <i>in vivo</i>.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> However, the naturally extracted <b>E7</b> demonstrates a relatively low potency with multiple Michael acceptors, which might cause off-target effects.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> To identify more potent GPR52 antagonists with structural diversity, we performed the high-throughput screening and identified compound <b>F11</b> ((<i>E</i>)-1,7-diphenylhept-4-en-3-one), which displayed moderate potency with the half-maximal inhibitory concentration (IC<sub>50</sub>) of 5.45 μM. Compared to <b>E7</b>, compound <b>F11</b> has relatively low molecular weight and extended chemical space containing an α,β-unsaturated carbonyl group, a linker, and terminal aryl-substitutes, which is more suitable for further optimization.</div><div class="NLM_p">First, we investigated the SAR on the α,β-unsaturated carbonyl moiety of <b>F11</b>. The broken α,β-unsaturated carbonyl system, by reducing double bond or carbonyl group and changing with other conjugated moiety, resulted in the loss of GPR52 inhibitory activity (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01133/suppl_file/jm0c01133_si_001.pdf" class="ext-link">Table S1</a>, compounds <b>1</b>–<b>6</b>). These results indicate that the α,β-unsaturated carbonyl group is critical and the flexible terminal aryl group at the side of double bond is required to keep the suitable direction. Shortening the linker length between α,β-unsaturated carbonyl group and two phenyl rings could maintain or even increase the inhibitory activity of GPR52, as shown in <a href="/doi/suppl/10.1021/acs.jmedchem.0c01133/suppl_file/jm0c01133_si_001.pdf" class="ext-link">Table S2</a> (compounds <b>7</b>–<b>12</b>). Notably, compound <b>8</b> (Comp-<b>8</b>) with ketone moiety significantly inhibited GPR52 activity with an IC<sub>50</sub> value of 0.65 μM (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A,B). Although with the Michael system, Comp-<b>8</b> exhibited the selectivity inhibitory activity targeting GPR52 but not other GPCRs (β-2 adrenergic receptor (β2AR), glucagon receptor (GCGR), glucagon-like peptide 1 receptor (GLP-1R)), as shown in <a href="/doi/suppl/10.1021/acs.jmedchem.0c01133/suppl_file/jm0c01133_si_001.pdf" class="ext-link">Figure S1</a>. However, when the left-hand-side linker in Comp-<b>8</b> became too short or too long, the compound potency decreased (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01133/suppl_file/jm0c01133_si_001.pdf" class="ext-link">Table S2</a>, compounds <b>9</b>–<b>12</b>). We further modified the α,β-unsaturated carbonyl group system based on Comp-<b>8</b> in <a href="/doi/suppl/10.1021/acs.jmedchem.0c01133/suppl_file/jm0c01133_si_001.pdf" class="ext-link">Table S3</a> (compounds <b>13</b>–<b>21</b>). All compounds including the added methyl group on the double bond (compound <b>14</b>), and the replacement of the double bond with three-membered carbocyclic ring (compound <b>19</b>) resulted in a dramatic loss of inhibitory activity targeting GPR52. These findings indicate that this key pharmacophore of α,β-unsaturated carbonyl group is preserved, which might fit the narrow binding pocket in GPR52.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a></div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01133/20210121/images/medium/jm0c01133_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01133/20210121/images/large/jm0c01133_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. GPR52-specific antagonists <b>F11</b>, Comp-<b>8</b>, and Comp-<b>43</b>. (A) Chemical structures of (<i>E</i>)-1,7-diphenylhept-4-en-3-one (<b>F11</b>), Comp-<b>8</b>, and Comp-<b>43</b>. (B) Dose responses of <b>F11</b>, Comp-<b>8</b>, and Comp-<b>43</b> on <b>WO-459</b> (100 nM)-stimulated cAMP accumulation in HEK293 cells stably expressing GPR52. Plots represent mean ± S.E.M. <i>N</i> = 3 for independently plated and treated wells. (C) Reversibility of <b>F11</b>, Comp-<b>8</b>, and Comp-<b>43</b> for targeting GPR52 in cells. First, HEK293 cells expressing β2AR serve as the positive control. Cells were treated with dimethyl sulfoxide (DMSO) or propranolol (Pro, 100 nM) for 15 min. For the recovery group, the cells were washed to remove propranolol and then incubated with fresh medium, followed by treatment with β2AR agonist isoproterenol (ISO, 100 nM) for 30 min. Cell lysates were then analyzed by cAMP kit. Then, HEK293 cells expressing GPR52 were treated with DMSO or GPR52 antagonists (10 μM <b>F11</b>, 3 μM Comp-<b>8</b>, and 3 μM Comp-<b>43</b>). For the recovery group, the cells were washed to remove GRP52 antagonists and then incubated with fresh medium. Further, the cells were treated with GPR52 agonist WO-459 (100 nM) and cell lysates were analyzed by cAMP kit. Cells treated with DMSO serve as 100% response, and cells treated with propranolol or GPR52 antagonists without washing as indicated serve as 0% response. Plots represent mean ± S.E.M. <i>N</i> = 3 for independently plated and treated wells.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01133/20210121/images/large/jm0c01133_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01133&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Then, we investigated the terminal aryl tails based on Comp<b>-8</b>, as shown in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> (compounds <b>22</b>–<b>48</b>). After replacing the left phenyl ring with hydrogen, compound <b>22</b> lost the inhibitory activity completely. With different substitutions on the left aryl group, compounds <b>23</b> and <b>24</b> slightly decreased the activity. When changing phenyl with piperidine on the right-hand-side ring, compound <b>25</b> also lost the activity. These results indicate that the aryl substitutions on the two terminals might form the π–π stacking interactions with the residues of GPR52 protein. Then, the different aryl groups at the ketone substitutions with electronic and steric effects were synthesized. The substituted phenyl rings (compounds <b>26</b>–<b>38</b>) and naphthyl rings (compounds <b>39</b> and <b>40</b>) could maintain or lower inhibitory activity of GPR52 compared with Comp-<b>8</b>. Surprisingly, compound <b>35</b> with the 4-F substitution on the benzene ring lost the activity completely. After further screening for a variety of five- and six-membered heteroaromatic rings (compounds <b>41</b>–<b>48</b>), we found compound <b>43</b> (Comp-<b>43</b>) bearing a thiophene ring exhibited high potency targeting GPR52 (IC<sub>50</sub> = 0.63 μM), as shown in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A,B. In addition, as shown in <a href="/doi/suppl/10.1021/acs.jmedchem.0c01133/suppl_file/jm0c01133_si_001.pdf" class="ext-link">Figures S1 and S2</a>, Comp-<b>43</b> showed no effects on the cAMP elevation or calcium response stimulated by other GPCRs (β2AR, GCGR, GLP-1R, C–C motif chemokine receptor 5 (CCR5), α-1D adrenergic receptor (α1DAR), and dopamine D2 receptor (DRD2)), except at a very high concentration.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. SAR of the Terminal Groups of Comp-<b>8</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01133/20210121/images/medium/jm0c01133_0010.gif" alt="" id="fx1" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">comp</th><th class="colsep0 rowsep0" align="center">R</th><th class="colsep0 rowsep0" align="center">R′</th><th class="colsep0 rowsep0" align="center">IC<sub>50</sub> (μM)<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center">comp</th><th class="colsep0 rowsep0" align="center">R</th><th class="colsep0 rowsep0" align="center">R″</th><th class="colsep0 rowsep0" align="center">IC<sub>50</sub> (μM)<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8</b></td><td class="colsep0 rowsep0" align="left">Ph</td><td class="colsep0 rowsep0" align="left">Ph</td><td class="colsep0 rowsep0" align="left">0.65</td><td class="colsep0 rowsep0" align="left"><b>35</b></td><td class="colsep0 rowsep0" align="left">Ph</td><td class="colsep0 rowsep0" align="left">4-F-C<sub>6</sub>H<sub>4</sub></td><td class="colsep0 rowsep0" align="left">NA<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>22</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">Ph</td><td class="colsep0 rowsep0" align="left">NA<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="left"><b>36</b></td><td class="colsep0 rowsep0" align="left">Ph</td><td class="colsep0 rowsep0" align="left">2-Ph-C<sub>6</sub>H<sub>4</sub></td><td class="colsep0 rowsep0" align="left">1.23</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>23</b></td><td class="colsep0 rowsep0" align="left">2-Br-C<sub>6</sub>H<sub>4</sub></td><td class="colsep0 rowsep0" align="left">Ph</td><td class="colsep0 rowsep0" align="left">1.64</td><td class="colsep0 rowsep0" align="left"><b>37</b></td><td class="colsep0 rowsep0" align="left">Ph</td><td class="colsep0 rowsep0" align="left">3-Ph-C<sub>6</sub>H<sub>4</sub></td><td class="colsep0 rowsep0" align="left">8.13</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>24</b></td><td class="colsep0 rowsep0" align="left">4-Br-C<sub>6</sub>H<sub>4</sub></td><td class="colsep0 rowsep0" align="left">Ph</td><td class="colsep0 rowsep0" align="left">1.81</td><td class="colsep0 rowsep0" align="left"><b>38</b></td><td class="colsep0 rowsep0" align="left">Ph</td><td class="colsep0 rowsep0" align="left">4-Ph-C<sub>6</sub>H<sub>4</sub></td><td class="colsep0 rowsep0" align="left">7.64</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>25</b></td><td class="colsep0 rowsep0" align="left">Ph</td><td class="colsep0 rowsep0" align="left">piperidyl</td><td class="colsep0 rowsep0" align="left">NA</td><td class="colsep0 rowsep0" align="left"><b>39</b></td><td class="colsep0 rowsep0" align="left">Ph</td><td class="colsep0 rowsep0" align="left">1-Naph</td><td class="colsep0 rowsep0" align="left">3.29</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>26</b></td><td class="colsep0 rowsep0" align="left">Ph</td><td class="colsep0 rowsep0" align="left">2-Me-C<sub>6</sub>H<sub>4</sub></td><td class="colsep0 rowsep0" align="left">2.63</td><td class="colsep0 rowsep0" align="left"><b>40</b></td><td class="colsep0 rowsep0" align="left">Ph</td><td class="colsep0 rowsep0" align="left">2-Naph</td><td class="colsep0 rowsep0" align="left">3.54</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>27</b></td><td class="colsep0 rowsep0" align="left">Ph</td><td class="colsep0 rowsep0" align="left">2,6-Me-C<sub>6</sub>H<sub>4</sub></td><td class="colsep0 rowsep0" align="left">2.82</td><td class="colsep0 rowsep0" align="left"><b>41</b></td><td class="colsep0 rowsep0" align="left">Ph</td><td class="colsep0 rowsep0" align="left">2-furyl</td><td class="colsep0 rowsep0" align="left">0.88</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>28</b></td><td class="colsep0 rowsep0" align="left">Ph</td><td class="colsep0 rowsep0" align="left">3-Me-C<sub>6</sub>H<sub>4</sub></td><td class="colsep0 rowsep0" align="left">2.57</td><td class="colsep0 rowsep0" align="left"><b>42</b></td><td class="colsep0 rowsep0" align="left">Ph</td><td class="colsep0 rowsep0" align="left">3-furyl</td><td class="colsep0 rowsep0" align="left">0.91</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>29</b></td><td class="colsep0 rowsep0" align="left">Ph</td><td class="colsep0 rowsep0" align="left">4-Me-C<sub>6</sub>H<sub>4</sub></td><td class="colsep0 rowsep0" align="left">3.02</td><td class="colsep0 rowsep0" align="left"><b>43</b></td><td class="colsep0 rowsep0" align="left"><b>Ph</b></td><td class="colsep0 rowsep0" align="left"><b>2-thienyl</b></td><td class="colsep0 rowsep0" align="left"><b>0.63</b></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>30</b></td><td class="colsep0 rowsep0" align="left">Ph</td><td class="colsep0 rowsep0" align="left">2-MeO-C<sub>6</sub>H<sub>4</sub></td><td class="colsep0 rowsep0" align="left">1.72</td><td class="colsep0 rowsep0" align="left"><b>44</b></td><td class="colsep0 rowsep0" align="left">Ph</td><td class="colsep0 rowsep0" align="left">3-thienyl</td><td class="colsep0 rowsep0" align="left">1.52</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>31</b></td><td class="colsep0 rowsep0" align="left">Ph</td><td class="colsep0 rowsep0" align="left">3-MeO-C<sub>6</sub>H<sub>4</sub></td><td class="colsep0 rowsep0" align="left">1.49</td><td class="colsep0 rowsep0" align="left"><b>45</b></td><td class="colsep0 rowsep0" align="left">Ph</td><td class="colsep0 rowsep0" align="left">5-(1-Me)pyrazolyl</td><td class="colsep0 rowsep0" align="left">1.08</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>32</b></td><td class="colsep0 rowsep0" align="left">Ph</td><td class="colsep0 rowsep0" align="left">4-MeO-C<sub>6</sub>H<sub>4</sub></td><td class="colsep0 rowsep0" align="left">3.87</td><td class="colsep0 rowsep0" align="left"><b>46</b></td><td class="colsep0 rowsep0" align="left">Ph</td><td class="colsep0 rowsep0" align="left">2-Py</td><td class="colsep0 rowsep0" align="left">1.82</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>33</b></td><td class="colsep0 rowsep0" align="left">Ph</td><td class="colsep0 rowsep0" align="left">2-F-C<sub>6</sub>H<sub>4</sub></td><td class="colsep0 rowsep0" align="left">1.76</td><td class="colsep0 rowsep0" align="left"><b>47</b></td><td class="colsep0 rowsep0" align="left">Ph</td><td class="colsep0 rowsep0" align="left">3-Py</td><td class="colsep0 rowsep0" align="left">1.01</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>34</b></td><td class="colsep0 rowsep0" align="left">Ph</td><td class="colsep0 rowsep0" align="left">3-F-C<sub>6</sub>H<sub>4</sub></td><td class="colsep0 rowsep0" align="left">2.37</td><td class="colsep0 rowsep0" align="left"><b>48</b></td><td class="colsep0 rowsep0" align="left">Ph</td><td class="colsep0 rowsep0" align="left">4-Py</td><td class="colsep0 rowsep0" align="left">1.81</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">IC<sub>50</sub> is 50% inhibitory concentration. The GPR52 inhibitor <b>F11</b> was used as the positive control, exhibiting an IC<sub>50</sub> value of 5.45 μM.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">NA, not active.</p></div></div></div><div class="NLM_p last">The current data based on the SAR study suggest that the active compounds might bind into the specific narrow pocket of GPR52,<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> which is probably mediated by the two hydrophobic π–π stacking interactions. In addition, compound <b>22</b> with the Michael system structure type (one terminal is H) is inactive, reflecting the importance of aromatic rings at both terminals. Further, we examined the reversibility of GPR52 antagonists <b>F11</b>, Comp-<b>8</b>, and Comp-<b>43</b> in cells for targeting GPR52 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>C). After removing the antagonists, GPR52 could restore the protein function by treatment with its agonist WO-549 significantly. Together, by SAR study, we identified novel, specific, and highly potent GPR52 antagonists Comp-<b>8</b> and Comp-<b>43</b>, which exhibit none of the properties of pan assay interference compounds (PAINS) and thus deserve further evaluation.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> SAR Analysis of Comp-<b>43</b> by Docking with GPR52 Crystal Structure</h3><div class="NLM_p">Recently, the crystal structure of GPR52 with an exogenous agonist was reported (PDB identification code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6LI0">6LI0</a>).<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> To better understand the SAR of these novel GPR52 antagonists, computational investigations by docking with GPR52 crystal structure were conducted. First, the parent compound <b>F11</b> was docked into the potential binding pocket of GPR52.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> As shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A, the phenyl rings of <b>F11</b> stretch into hydrophobic pockets and form key π–π interaction with Phe300 and Tyr34 in GPR52. The carbonyl oxygen atom of <b>F11</b> forms a critical hydrogen bond with Ser299 in GPR52, which explains any modification of carbonyl group led to complete loss of activity (compounds <b>1</b>–<b>2</b>). Also, the replacement of the double bond changes the phenyl ring conformation, leading to activity loss (compounds <b>3</b>–<b>6</b>). Interestingly, as shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B, adjusting to the suitable distance of carbonyl group to benzene ring (Comp-<b>8</b>) resulted in a remarkable improvement of potency against GPR52 (IC<sub>50</sub> = 0.65 μM). The docking result of Comp-<b>8</b> with GPR52 suggests that this optimized molecular conformation with carbonyl oxygen provides two new hydrogen-bonding patterns to fit the binding pocket properly.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01133/20210121/images/medium/jm0c01133_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01133/20210121/images/large/jm0c01133_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Structural alignments of GPR52 antagonists (wheat) in complex with GPR52 (green). (A–D) Structural alignments of GPR52 with compound <b>F11</b> (A), Comp-<b>8</b> (B), Comp-<b>14</b> (C), and Comp-<b>43</b> (D). The PDB code for GPR52 is <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6LI0">6LI0</a><a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> All of the procedures were performed in Autodock Vina,<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> and the figures were generated using PyMOL software.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01133/20210121/images/large/jm0c01133_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01133&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">On the basis of Comp-<b>8</b>, computational investigation of other derivatives suggests that either the shortened (compound <b>10</b>, <a href="/doi/suppl/10.1021/acs.jmedchem.0c01133/suppl_file/jm0c01133_si_001.pdf" class="ext-link">Figure S3A</a>) or lengthened (compound <b>12</b>, <a href="/doi/suppl/10.1021/acs.jmedchem.0c01133/suppl_file/jm0c01133_si_001.pdf" class="ext-link">Figure S3B</a>) chain is unfavorable for the terminal phenyl rings to form π–π interactions with the residues in GPR52. In addition, the linker in a relatively narrow pocket and the double bond are optimal for the unique steric and conformational profiles. Modifications of this double bond by introducing steric hindrance or changing conformation led to the total loss of activity (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01133/suppl_file/jm0c01133_si_001.pdf" class="ext-link">Table S3</a>). A docking study of compound <b>14</b> with GPR52 shows that the steric methyl group disturbs the formation of the crucial hydrogen bond due to its steric repulsion against the side chains of Pro28 and Phe300 in GPR52 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C). Moreover, π–π stacking interactions between terminal phenyl rings and GPR52 are disrupted by the spatial conformation changes induced by the modifications of double bond. As a result, compound <b>14</b> exhibited no potency against GPR52. Of note, in comparison to Comp-<b>8</b>, the binding mode of Comp-<b>43</b> with GPR52 showed the thiophene ring with less steric resistance, the restored π–π stacking interaction with Phe300, and the specific electronic effects with His186 in GPR52 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>D). Thus, our data indicate that Comp-43 might be more favorable for binding with GPR52 and worthy for further study of biological function.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> GPR52-Specific Antagonist Comp-<b>43</b> Decreases HTT Levels in Neuronal Cells</h3><div class="NLM_p">As the representative Comp-<b>43</b> shows specific and high potency targeting GPR52, we further examined the effect of Comp-<b>43</b> on HTT levels in STHdhQ7/Q111 cells. First, we observed that Comp-<b>43</b> showed little effects on the cell viability of STHdhQ7/Q111 at the concentration of 10 μM or less, which served as the recommended doses in further evaluations (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01133/suppl_file/jm0c01133_si_001.pdf" class="ext-link">Figure S4</a>). As shown in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A, upon treatment for 48 h, Comp-<b>43</b> significantly reduced the HTRF signals in a dose-dependent manner, which is proportional to total HTT levels. Similarly, by WB analysis, we observed a remarkable decrease on both mHTT and wtHTT levels by Comp-<b>43</b>, which is comparable to that by Rapamycin, a well-known autophagy enhancer (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B,C, and <a href="/doi/suppl/10.1021/acs.jmedchem.0c01133/suppl_file/jm0c01133_si_001.pdf" class="ext-link">S5A</a>).</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01133/20210121/images/medium/jm0c01133_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01133/20210121/images/large/jm0c01133_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. GPR52-specific antagonist Comp-<b>43</b> decreases mHTT levels in neuronal cells. (A) TR-FRET analysis of HTT levels using 2B7/2166 antibody pair in STHdhQ7/Q111. Cells were treated with Comp-<b>43</b> at the concentrations of 0.3–10 μM for 48 h. Plots represent mean ± S.E.M. <i>N</i> = 3–4, and the one-way ANOVA with the post hoc Bonferroni’s multiple comparisons was used for statistical analysis. * <i>P</i> < 0.05, ** <i>P</i> < 0.01, *** <i>P</i> < 0.001, versus the DMSO control. (B) WB analysis of HTT levels in STHdhQ7/Q111 cells after treatment with Comp-<b>43</b> at the concentration of 1–10 μM for 48 h. Ab1 antibody detects both mHTT and wtHTT. Rapa: Rapamycin, an autophagy enhancer as the positive control. (C) Quantification of WB analysis of mHTT levels in (B). Plots represent mean ± S.E.M. <i>N</i> = 3 per group. One-way ANOVA with the post hoc Bonferroni’s multiple comparisons was used. * <i>P</i> < 0.05, ** <i>P</i> < 0.01, versus the DMSO control. (D) TR-FRET analysis of HTT levels using 2B7/2166 antibody pair in primary HdhQ7/Q140 neurons. Cells were treated with Comp-<b>43</b> at the concentration of 0.1–3 μM for 48 h. Plots represent mean ± S.E.M. <i>N</i> = 5–6 for independently experiments. ** <i>P</i> < 0.01, *** <i>P</i> < 0.001, versus the DMSO control, as determined by the one-way ANOVA with the post hoc Bonferroni’s multiple comparisons. (E) WB analysis of HTT levels in primary HdhQ7/Q140 neurons after treatment with Comp-<b>43</b> at the concentrations of 0.3–3 μM for 48 h. (F) Quantification of WB analysis of mHTT levels in (E). Plots represent mean ± S.E.M. <i>N</i> = 4 per group. One-way ANOVA with the post hoc Donnet’s multiple comparisons was used. * <i>P</i> < 0.05, versus the DMSO control. (G) TR-FRET analysis of HTT levels using 2B7/2166 antibody pair in STHdhQ7/Q111. Cells were transfected with siNC or siGPR52 for 24 h, respectively, followed by treatment with Comp-<b>43</b> at the concentration of 1–10 μM for another 48 h. Plots represent mean ± S.E.M. N = 4, and two-way ANOVA with the post hoc Bonferroni’s multiple comparisons was used for statistical analysis. *** P < 0.001, versus the DMSO control in the siNC group. (H) WB analysis of HTT levels using Ab1 antibody. STHdhQ7/Q111 cells were transfected with siNC or siGPR52 for 24 h and then treated with Comp-<b>43</b> at the concentration of 3 and10 μM for 48 h. (I) Quantification of WB analysis of mHTT levels in (H). Plots represent mean ± S.E.M. <i>N</i> = 4 per group. One-way ANOVA with the post hoc Bonferroni’s multiple comparisons was used. * <i>P</i> < 0.05, ** <i>P</i> < 0.01, versus the DMSO control in the siNC group.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01133/20210121/images/large/jm0c01133_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01133&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To confirm the regulatory roles of Comp-<b>43</b> on HTT levels, we performed TR-FRET and WB analysis in mouse primary neurons obtained from HdhQ7/Q140 mice. As shown in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>D–F and <a href="/doi/suppl/10.1021/acs.jmedchem.0c01133/suppl_file/jm0c01133_si_001.pdf" class="ext-link">S5B</a>, Comp-<b>43</b> also remarkably reduced both mHTT and wtHTT levels in a dose-dependent manner, which is consistent with the observations in STHdhQ7/Q111 cells. Our data suggest that Comp-<b>43</b> decreases HTT levels in a nonselective manner.</div><div class="NLM_p last">To explore whether Comp-<b>43</b> regulates HTT levels via GPR52, we transiently expressed either the siRNA targeting GPR52 (siGPR52) or the negative control (siNC), respectively, in STHdhQ7/Q111 cells followed by Comp-<b>43</b> treatment. As shown in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>G–I and <a href="/doi/suppl/10.1021/acs.jmedchem.0c01133/suppl_file/jm0c01133_si_001.pdf" class="ext-link">S5C</a>, the knockdown of GPR52 significantly reduced both mHTT and wtHTT levels compared to that of siNC control, as previously reported.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> Furthermore, we observed that knockdown of GPR52 abolished the effect of Comp-<b>43</b> on lowering HTT, suggesting that Comp-<b>43</b> could decrease HTT levels via targeting GPR52.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Comp-<b>43</b> Protects Mouse Primary Striatal Neurons from mHTT Toxicity</h3><div class="NLM_p">Numerous studies have shown that mHTT exerts toxic effects on neurons especially in the striatum, which leads to progressive HD phenotypes.<a onclick="showRef(event, 'ref28 ref29 ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref28 ref29 ref30 ref31">(28−31)</a> Therefore, we further investigated the effects of Comp-<b>43</b> on neuronal survival. Primary P1 striatal neurons were cultured in vitro for 3 days and then challenged by removal of protective growth factors, which could induce HD-specific neuronal shrinkage and death.<a onclick="showRef(event, 'ref32 ref33'); return false;" href="javascript:void(0);" class="ref ref32 ref33">(32,33)</a> The neuronal shrinkage and death could also be rescued by knockdown of mHTT (siHTT), confirming the mHTT-dependent toxicity.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a></div><div class="NLM_p">Thus, we assessed this mHTT-dependent neuronal shrinkage and death by immunostaining for NeuN, a neuron-specific marker. As shown in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A–C and <a href="/doi/suppl/10.1021/acs.jmedchem.0c01133/suppl_file/jm0c01133_si_001.pdf" class="ext-link">S6A</a>, in comparison to primary striatal neurons from wild-type mice, primary striatal neurons from HdhQ7/Q140 mice showed remarkable neuronal loss, confirming the neuron deficits caused by mHTT as reported.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> Furthermore, treatment with Comp-<b>43</b> at 3 μM significantly rescued the neuronal loss in HdhQ7/Q140 striatal neurons, which is comparable to the expressing of siHTT. Similar results were observed in the neuronal apoptosis analysis by examining the time series of caspase 3 activity. As shown in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>D and <a href="/doi/suppl/10.1021/acs.jmedchem.0c01133/suppl_file/jm0c01133_si_001.pdf" class="ext-link">S6B</a>, primary HdhQ7/Q140 striatal neurons showed significant caspase 3 activity compared to wild-type neurons. In addition, a remarkable reduction of caspase 3 activity was observed after treatment with Comp-<b>43</b> as well as siHTT. Together, these data suggest that Comp-<b>43</b> protects HD mouse primary striatal neurons from mHTT toxicity.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01133/20210121/images/medium/jm0c01133_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01133/20210121/images/large/jm0c01133_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Comp-<b>43</b> protects mouse primary striatal neurons from mHTT toxicity. (A) Immunostaining of the neuronal-specific marker NeuN in primary striatal neurons. Primary neurons were obtained from HdhQ7/Q140 P1 mice. At DIV3-DIV5, cells were treated with Comp-<b>43</b> at 3 μM or transfected with siHTT, and then subjected to starvation by withdrawing the growth factors N2 and B27 for another 48 h. Representative images are shown. Scale bar: 100 μm. (B, C) Quantifications of the NeuN<sup>+</sup> intensity (B) and NeuN<sup>+</sup> area (C) in (A). <i>N</i> = 18, 18, and 9 for DMSO, siHTT, and Comp-<b>43</b>, respectively. Plots represent mean ± S.E.M. * <i>P</i> < 0.05, ** <i>P</i> < 0.01, *** <i>P</i> < 0.001, versus the DMSO control, as determined by one-way ANOVA with the post hoc Bonferroni’s multiple comparisons. (D) Neuronal apoptosis at different time points determined by caspase 3 activity in HdhQ7/Q140 primary striatal neurons. Cells were treated with indicated conditions and starved, and then the caspase 3 activity within the cells was monitored using a green fluorescent dye (NucView 488) and corrected by cell confluency. <i>N</i> = 12 per condition. Plots represent mean ± S.E.M. Two-way ANOVA with post hoc Bonferroni’s multiple comparisons was used for statistical analysis. *** <i>P</i> < 0.001, versus the DMSO control.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01133/20210121/images/large/jm0c01133_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01133&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Intracerebroventricular Treatment of Comp-<b>43</b> Decreases mHTT Levels in HdhQ140 Mice</h3><div class="NLM_p">To further explore the effects of Comp-<b>43</b><i>in vivo</i>, we performed direct intracerebroventricular administration of Comp-<b>43</b> into 9-month-old wild-type and HdhQ140 mice at a concentration of 0.5 mM, which is 5-fold lower than <b>E7</b><a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> considering the relatively high efficiency of Comp-<b>43</b> antagonizing GPR52 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B). After once-a-day 3 consecutive weeks of treatment, Comp-<b>43</b> significantly reduced mHTT levels in the striatum of HdhQ140 mice, which express endogenous mHTT proteins with 140Q. Moreover, Comp-<b>43</b> showed no detectable effect on mHTT levels in the cortex of HD mice (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A), possibly because of GPR52’s high expression specificity in the striatum and Comp-<b>43</b>-mediated mHTT reduction through GPR52 targeting, which further indicates on-target effects. In addition, Comp-<b>43</b> showed no obvious influence on wtHTT levels in wild-type mice (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B). Hence, our data demonstrate that Comp-<b>43</b> reduces mHTT levels in HD mice.</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01133/20210121/images/medium/jm0c01133_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01133/20210121/images/large/jm0c01133_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Intracerebroventricular treatment of Comp-<b>43</b> decreases mHTT levels in HdhQ140 mice. (A, B) TR-FRET analysis of HTT levels with 2B7/2166 antibody pair in striata and cortices of HdhQ140 mice (A) and wild-type mice (B), respectively. Nine-month-old mice were subjected to intracerebroventricular injection of Comp-<b>43</b> for 2 μL at 0.5 mM once a day for 3 consecutive weeks. <i>N</i> = 4–5 for each group. Plots represent mean ± S.E.M. * <i>P</i> < 0.05 as determined by one-way ANOVA with post hoc Bonferroni’s multiple comparisons. (C) Representative immunofluorescent images of S830 (left), and the quantification of S830<sup>+</sup> intensity (right) in striatal slices of HdhQ140 mice treated with DMSO or Comp-<b>43</b>. S830 antibody detects aggregated mHTT. Scale bar: 100 μm. For each group, four to five mice were tested with nine images taken from three independent slices. Plots represent mean ± S.E.M. An unpaired two-tailed Student’s <i>t</i> test was used. ** <i>P</i> < 0.01. (D) Representative immunostaining images of Iba1 (left) and the quantification of Iba1<sup>+</sup> intensity (right) in striatal slices of HdhQ140 mice treated with DMSO or Comp-<b>43</b>. Iba1 antibody serves as an activated microglial marker. Scale bar: 100 μm. For each group, four to five mice were tested with nine images taken from three independent slices Plots represent mean ± S.E.M. *** <i>P</i> < 0.001 as determined by an unpaired two-tailed Student’s <i>t</i> test.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01133/20210121/images/large/jm0c01133_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01133&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">To further examine the effect of Comp-<b>43</b> on mHTT aggregates in HD mice, we performed immunostaining against S830, an antibody specific for the aggregated forms of mHTT.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> As shown in <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>C, treatment with Comp-<b>43</b> significantly reduced mHTT aggregates in the striatum of HdhQ140 mice, suggesting that Comp-<b>43</b> decreased not only the soluble mHTT but also the aggregates in HD mice. In addition, mounting evidence has shown that microglial-mediated proinflammation plays critical roles in the pathogenesis of neurodegenerative diseases including HD.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> As shown in <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>D, we noted that the staining intensity of Iba1-positive microglia was significantly decreased after treatment with Comp-<b>43</b> in the striatum of HdhQ140 mice, suggesting that HD-associated microgliosis is ameliorated by Comp-<b>43</b> treatment. Together, these results indicate that intracerebroventricular administration of Comp-<b>43</b> attenuates disease-related pathology in HdhQ140 mice.</div></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Intraperitoneal Treatment of Comp-<b>43</b> Attenuates HD-Associated Phenotypes in HdhQ140 Mice</h3><div class="NLM_p">Considering the challenging and limited application of delivering Comp-<b>43</b> intracerebroventricularly, we further explored whether Comp-<b>43</b> crosses the blood–brain barrier. Adult wild-type mice were subjected to intravenous injection with Comp-<b>43</b> at 5 mg/kg, and then the concentrations of Comp-<b>43</b> in heart blood plasma and brain tissues were analyzed at the designated time. As shown in <a href="/doi/suppl/10.1021/acs.jmedchem.0c01133/suppl_file/jm0c01133_si_001.pdf" class="ext-link">Figure S7</a>, 0.5–6 h after injection, Comp-<b>43</b> at the concentration of 5 mg/kg passed the blood–brain barrier and reached the wild-type brain at concentrations of approximately 20–60 nM, suggesting similar blood–brain barrier permeability in the more vulnerable HdhQ140 mice.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a></div><div class="NLM_p">Then, we performed intraperitoneal injection of Comp-<b>43</b> into 9-month-old HdhQ140 and the wild-type control mice at a dose of 5 mg/kg. After treatment once a day for continuous 4 weeks, we investigated the disease-associated motor performance in these mice, including rotarod and balance beam tests. As shown in <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>A, HdhQ140 mice exhibited obvious motor deficits quantified by the latency to fall in a rotarod test. Intraperitoneal treatment of Comp-<b>43</b> significantly improved the rotarod performance in HdhQ140 mice, without influencing the wild-type mice. Similar results were observed in a balance beam test (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>B). Compared to wild-type mice, evident motor deficiency determined by increased balance beam crossing times was observed in HdhQ140 mice. Comp-<b>43</b> rescued this deficit in the balance beam test in HdhQ140 mice while exhibiting no obvious effect in wild-type mice. Thus, these data suggest that the treatment of Comp-<b>43</b> alleviates the motor deficits in HdhQ140 mice.</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01133/20210121/images/medium/jm0c01133_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01133/20210121/images/large/jm0c01133_0008.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Intraperitoneal treatment of Comp-<b>43</b> attenuates HD-associated phenotypes in HdhQ140 mice. (A) Rotarod test results for 9-month-old HdhQ140 and the wild-type control mice. Mice were intraperitoneally injected with Comp-<b>43</b> at a dose of 5 mg/kg or the vehicle control once a day for 4 continuous weeks. After that, the mice were trained for 3 days and then tested for 5 days. <i>N</i> = 7–10 per group. Plots represent mean ± S.E.M. Two-way ANOVA with the post hoc Bonferroni’s multiple comparisons was used for statistical analysis. * <i>P</i> < 0.05, versus the result for HdhQ140 mice treatment with DMSO. (B) Balance beam results for 9-month-old HdhQ140 and the wild-type control mice after intraperitoneal treatment with Comp-<b>43</b> or the vehicle control. Balance beam test was performed after training for 2 consecutive days. <i>N</i> = 7–10 per group. Plots represent mean ± S.E.M. * <i>P</i> < 0.05 as determined by two-way ANOVA with the post hoc Bonferroni’s multiple comparisons. (C) Soluble mHTT levels were measured with 2B7/2166 antibody pair in HdhQ140 mice by TR-FRET. After the behavioral tests, mice striata and cortices were obtained, and then the soluble supernatant proteins were extracted after lysed, homogenized, and centrifuged at 20 000<i>g</i> at 4 °C for 20 min. <i>N</i> = 5–6 for each group. Plots represent mean ± S.E.M. * <i>P</i> < 0.05 as determined by an unpaired two-tailed Student’s <i>t</i> test. (D) mHTT aggregation levels were measured with 4C9/4C9 antibody pair in HdhQ140 mice by TR-FRET. After the soluble supernatant proteins from striata and cortices were extracted, the aggregates were resuspended with 2% sodium dodecyl sulfate (SDS) followed by further sonication. After another centrifugation at 20 000<i>g</i> for 20 min, the supernatants were collected and subjected to HTRF analysis. <i>N</i> = 5–6 for each group. Plots represent mean ± S.E.M. An unpaired two-tailed Student’s <i>t</i> test was used. * <i>P</i> < 0.05. (E) Representative immunostaining images of DARPP32 (top) and Iba1 (bottom) in striatal slices of wild-type or HdhQ140 mice, treated with DMSO or Comp-<b>43</b>, respectively. Mice striata slices were obtained and stained using antibodies against DARPP32 or Iba1, followed by DAPI staining. DARPP32 antibody serves as a specific striatal neuron marker. Scale bar: 50 μm. (F) Quantifications of (E). For each group, six to seven mice were tested with nine images taken from three independent slices. Plots represent mean ± S.E.M. * <i>P</i> < 0.05, ** <i>P</i> < 0.01, *** <i>P</i> < 0.001 as determined by the two-way ANOVA with the post hoc Bonferroni’s multiple comparisons.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01133/20210121/images/large/jm0c01133_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01133&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We further examined the effect of Comp-<b>43</b> on mHTT levels in HdhQ140 mice, which express mHTT proteins with 140Q. In the TR-FRET assay, Comp-<b>43</b> significantly reduced the soluble mHTT levels in the striatum of HdhQ140 mice without influencing cortex mHTT levels in HD mice (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>C) or wtHTT levels in wild-type mice (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01133/suppl_file/jm0c01133_si_001.pdf" class="ext-link">Figure S8B</a>).</div><div class="NLM_p">Furthermore, the aggregate fractions were resuspended with 2% SDS-contained buffer and subjected to TR-FRET test for mHTT aggregates measurement.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> When mHTT forms insoluble aggregates, the epitope specific to 2166 is sterically blocked and can no longer bind with the antibody, resulting in loss of TR-FRET signal. Instead, by targeting the polyproline region in exon 1 of mHTT, a mixture of 4C9 antibody labeled with either terbium or D2 fluorophore, respectively, results in a TR-FRET signal for aggregated, but not soluble mHTT.<a onclick="showRef(event, 'ref7 ref38'); return false;" href="javascript:void(0);" class="ref ref7 ref38">(7,38)</a> As shown in <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>D and <a href="/doi/suppl/10.1021/acs.jmedchem.0c01133/suppl_file/jm0c01133_si_001.pdf" class="ext-link">S9C</a>, a remarkable reduction of mHTT aggregation in the striatum of HD mice was observed after the treatment with Comp-<b>43</b>, whereas no obvious effects in the cortex of HD or wild-type mice were noted. These data demonstrate that intraperitoneal administration of Comp-<b>43</b> reduces both soluble and aggregated mHTT levels in the striatum of HdhQ140 mice.</div><div class="NLM_p">In addition, we investigated the neuronal deficits by immunostaining of DAPRR32, a selective marker of medium spiny neurons in the striatum. As shown in <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>E and <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>F, the neuronal loss in the striatum of HdhQ140 mice was rescued by the treatment of Comp-<b>43</b>. Moreover, the intensity of Iba1-positive microglia was reduced after treatment with Comp-<b>43</b> in the striatum of HdhQ140 mice, further supporting that Comp-<b>43</b> ameliorates HD-associated pathology.</div><div class="NLM_p">Taken together, our data demonstrate that intraperitoneal treatment of Comp-<b>43</b> attenuates HD-associated phenotypes, including the motor performance, both soluble and aggregated mHTT levels, and neuronal deficits in HD mice (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>).</div><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01133/20210121/images/medium/jm0c01133_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01133/20210121/images/large/jm0c01133_0009.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Model illustrating the function of GPR52 antagonist Comp-<b>43</b>. GPR52-specific antagonist Comp-<b>43</b> reduces mHTT levels via GPR52 and protects striatal neurons from mHTT toxicity, and eventually ameliorates HD-related phenotypes in HdhQ140 mice.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01133/20210121/images/large/jm0c01133_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01133&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i12">Discussion and Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_50606" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_50606" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">GPR52 plays a critical role in HD pathogenesis and serves as a potential drug target for HD therapy. Previous studies showed that embryonic knockdown of <i>Gpr</i>52, which is highly expressed in the striatum, lowers HTT levels and rescues HD-associated phenotypes. This research further confirms the therapeutic potential of GPR52 in adult-onset HD, the most common form of this disease.<a onclick="showRef(event, 'ref39 ref40'); return false;" href="javascript:void(0);" class="ref ref39 ref40">(39,40)</a> We found that AAV-mediated striatal knockdown of GPR52 reduces mHTT levels in adult HdhQ140 mice by TR-FRET and WB assays. While other <i>in vivo</i> phenotypes need to be further explored, our findings provide critical evidence for a GPR52 targeting HD therapy considering well-established mHTT reduction observed in HD phenotypic rescue.</div><div class="NLM_p">We have previously identified a novel GPR52 antagonist <b>E7</b> and conducted the proof-of-concept study to demonstrate the potential of targeting GPR52.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> In our current study, we performed high-throughput screening and identified <b>F11</b>, which showed moderate potency (IC<sub>50</sub>: 5.45 μM for <b>F11</b> vs 16.59 μM for <b>E7</b>). The compound <b>F11</b> contains an α,β-unsaturated carbonyl group, a linker, and terminal aryl-substitutes, with easier access to modify compared to <b>E7</b>. After the modification of <b>F11</b> with the three parts, the SAR study indicates that the key pharmacophore of α,β-unsaturated carbonyl group is preserved and the aryl substitutions on the two terminals might form the π–π stacking interactions with the residues of GPR52 protein in the narrow binding pocket.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> Thus, we identified Comp-<b>43</b> with highly potent and specific GPR52 antagonist activity (IC<sub>50</sub> = 0.63 μM), which does not exhibit the property of PAINS based on the cellular bioactive selectivity and SAR study. Further, by the molecular docking study of the GPR52 antagonists with GPR52 crystal structure,<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> the active GPR52 antagonists bind with the same binding pocket in GPR52 as the agonist, which might explain the results of the SAR study. Among them, Comp-<b>43</b> shows an optimal binding mode with GPR52, which is worth for further functional evaluations in cellular and <i>in vivo</i> levels.</div><div class="NLM_p">In both STHdhQ7/Q111 cell lines and primary HdhQ7/Q140 neurons, we observed that the treatment with Comp-<b>43</b> at a concentration of 0.3–3 μM significantly decreased mHTT levels, which is roughly consistent with the effective concentration antagonizing GPR52. Furthermore, knockdown of GPR52 almost abolished Comp-<b>43</b>-mediated HTT reduction, indicating that Comp-<b>43</b> regulated mHTT levels <i>via</i> GPR52. While Comp-<b>43</b> showed relatively mild effects on mHTT reduction, it is considered that ∼20% mHTT reduction was sufficient for phenotypic rescue in various HD mice models.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> Notably, in addition to mHTT, a comparable reduction on wtHTT by Comp-<b>43</b> was also observed. Multiple groups have also shown that ∼50% reduction of wtHTT is well tolerated <i>in vivo</i>.<a onclick="showRef(event, 'ref13 ref42 ref43 ref44'); return false;" href="javascript:void(0);" class="ref ref13 ref42 ref43 ref44">(13,42−44)</a> Moreover, with individual strengths and limitations, both nonselective and allele-selective HTT reduction approaches have been investigated in clinical trials for HD.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> In addition, we observed that Comp-<b>43</b> rescued the neuronal loss and apoptosis in primary HdhQ7/Q140 striatal neurons. As neurodegeneration in HD occurs mainly in the striatum, especially in early stages of pathology, this neuroprotective effect might be sufficient to attenuate the disease-associated phenotypes <i>in vivo</i>.<a onclick="showRef(event, 'ref14 ref15 ref45'); return false;" href="javascript:void(0);" class="ref ref14 ref15 ref45">(14,15,45)</a></div><div class="NLM_p last">By delivering this compound via intracerebroventricular administration into HdhQ140 mice, we observed that Comp-<b>43</b> significantly reduced both soluble and aggregated mHTT, as well as the Iba1<sup>+</sup> microglia in the striatum. Moreover, we found that Comp-<b>43</b> showed favorable lipophilicity and could cross the blood–brain barrier. Further intraperitoneal injection of Comp-<b>43</b> alleviated the motor deficits and disease-associated pathology in HdhQ140 mice, including the reduction of soluble and aggregated mHTT levels, neuronal loss, and neuroinflammation-related microgliosis. Considering both intraperitoneal injection and intracerebroventricular administration of Comp-<b>43</b> exhibited consistent biological beneficial effects in HdhQ140 mice, our findings provide a potential lead compound and confirm the possibility of targeting GPR52 for HD therapy (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>).</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i13">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_52155" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_52155" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> General Information</h3><div class="NLM_p last">NMR spectral data were recorded on a Bruker spectrometer (400 or 300 MHz for <sup>1</sup>H, 125 MHz or 150 MHz for <sup>13</sup>C). Chemical shifts (δ) were reported in parts per million (ppm), and the signals were described as brs (broad single), s (singlet), d (doublet), dd (double of double), t (triplet), q (quartet), and m (multiple). Coupling constants (<i>J</i> values) were given in hertz. HRMS were obtained using a Q-TOF mass spectrometer with ESI resource. Finally, the purity of main compounds was determined by high-performance liquid chromatography (HPLC) and confirmed to be more than 95%. HPLC analyses were performed on an Agilent 1260 series LC system (Agilent Eclipse XDB-C18, 4.6 mm × 150 mm, 5 μm). Conditions were as follows: CH<sub>3</sub>OH/H<sub>2</sub>O (v/v = 10:90) eluent [containing 0.1% trifluoroacetic] at 1.0 mL/min flow; UV wavelength, maximal absorbance at 254 nm; temperature, ambient; injection volume, 3 μL. Thin-layer chromatography (TLC) spots were visualized under UV light. Column chromatography was carried out on silica gel. Unless otherwise noted, all reagents and solvents were used directly as obtained commercially.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> General Synthesis</h3><div class="NLM_p">The synthesis route for most analogues is shown in <a href="/doi/suppl/10.1021/acs.jmedchem.0c01133/suppl_file/jm0c01133_si_001.pdf" class="ext-link">Scheme S1</a>. Starting from an appropriate substituted carboxylic acid (<b>S-1</b>), conversion to the ester (<b>S-2</b>) followed by HWE reaction under basic conditions gave intermediate <b>S-3</b>. The title products were prepared by condensation of intermediates <b>S-3</b> and <b>S-4</b>.</div><div id="sec4_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i16" class="anchor-spacer"></div><h4 class="article-section__title" id="_i16"> (<i>E</i>)-1,7-Diphenylhept-4-en-3-one (<b>F11</b>)</h4><div class="NLM_p last">Step a: A suspension of 3-phenylpropanoic acid (3 g, 20 mmol) and sulfuric acid (1.5 mL) in methyl alcohol (30 mL) was refluxed for 2 h (monitored by TLC). The reaction mixture was cooled to room temperature and evaporated, then diluted with ethyl acetate, and washed with saturated sodium bicarbonate solution. The organic phase was separated, washed with brine, dried over sodium sulfate, and evaporated to afford the methyl 3-phenylpropanoate as a colorless oil (3.6 g, 96% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.32–7.23 (m, 2H), 7.22–7.14 (m, 3H), 3.65 (s, 3H), 2.94 (t, <i>J</i> = 7.9 Hz, 2H), 2.62 (dd, <i>J</i> = 8.4, 7.3 Hz, 2H). Step b: To a solution of dimethyl methyl phosphonate (1.3 mL, 12.2 mmol) in dry tetrahydrofuran (15 mL) at −78 °C, <i>n</i>-BuLi (5.6 mL, 2.4 mol in tetrahydrofuran (THF)) was added dropwise under nitrogen atmosphere. After stirring for 30 min at this temperature, methyl 3-phenylpropanoate (1.0 g, 6.1 mmol in THF) was added and stirred for 1 h. Then, the system was warmed to 0 °C and quenched with a saturated solution of NH<sub>4</sub>Cl. The mixture was diluted with ethyl acetate. The organic phase was separated, washed with brine, dried over sodium sulfate, filtered, and evaporated under reduced pressure. The residue was purified by silica gel column chromatography using dichloromethane/methanol (20:1, v/v) as an eluent to afford the intermediate dimethyl (2-oxo-4-phenylbutyl)phosphonate as a colorless oil (1.5 g, 98% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.30–7.24 (m, 2H), 7.19 (d, <i>J</i> = 6.9 Hz, 3H), 3.75 (d, <i>J</i> = 0.4 Hz, 3H), 3.72 (d, <i>J</i> = 0.4 Hz, 3H), 3.07 (d, <i>J</i> = 22.7 Hz, 2H), 2.93 (dd, <i>J</i> = 12.3, 5.8 Hz, 4H). Step c: A suspension of dimethyl (2-oxo-4-phenylbutyl)phosphonate (190 mg, 0.8 mmol) in dry THF was treated with K<sub>2</sub>CO<sub>3</sub> (110 mg, 0.8 mmol). The system was purged with nitrogen three times and stirred at room temperature for 40 min; then, 3-phenylpropanal (100 mg, 0.8 mmol) in THF was added. The reaction mixture was warmed to 40 °C and stirred for 12 h, cooled down to room temperature, filtered, and evaporated. The residue was purified by silica gel column chromatography using petroleum ether/ethyl acetate (50:1–20:1, v/v) as an eluent to afford the title product <b>F11</b> as a colorless oil (182 mg, 92% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.44–7.15 (m, 10H), 6.91 (dt, <i>J</i> = 15.7, 6.8 Hz, 1H), 6.19 (d, <i>J</i> = 15.9 Hz, 1H), 3.01 (t, <i>J</i> = 7.1 Hz, 2H), 2.95–2.88 (m, 2H), 2.84 (t, <i>J</i> = 7.7 Hz, 2H), 2.62–2.57 (m, 2H). Retention time 2.68 min, HPLC purity > 98%.</div></div><div id="sec4_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i17" class="anchor-spacer"></div><h4 class="article-section__title" id="_i17"> (<i>E</i>)-1,7-Diphenylhept-4-en-3-ol (<b>1</b>)</h4><div class="NLM_p last">To a solution of (<i>E</i>)-1,7-diphenylhept-4-en-3-one <b>F11</b> (200 mg, 0.76 mmol) in ethyl alcohol (10 mL) was added sodium borohydride (43 mg, 1.14 mmol). After stirring at room temperature for 30 min (monitored by TLC), the mixture was quenched with water, extracted by ethyl acetate, and washed with brine. The organic layer was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub> and concentrated under reduced pressure. The residue was purified by silica gel column chromatography using petroleum ether/ethyl acetate (20:1–10:1, v/v) as an eluent to afford (<i>E</i>)-1,7-diphenylhept-4-en-3-ol <b>1</b> as a colorless oil (138 mg, 68% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.30–7.26 (m, 4H), 7.20–7.17 (m, 6H), 5.75–5.64 (m, 1H), 5.51 (dd, <i>J</i> = 15.5, 7.0 Hz, 1H), 4.06 (q, <i>J</i> = 6.5 Hz, 1H), 2.73–2.60 (m, 4H), 2.38 (dd, <i>J</i> = 14.8, 7.2 Hz, 2H), 1.90–1.70 (m, 2H), 1.47 (brs, 1H).</div></div><div id="sec4_2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18"> (<i>E</i>)-(5-Methoxyhept-3-ene-1,7-diyl)dibenzene (<b>2</b>)</h4><div class="NLM_p last">To a solution of (<i>E</i>)-1,7-diphenylhept-4-en-3-ol 1 (236 mg, 0.9 mmol) in dry THF (3 mL) at 0 °C was added sodium hydride (43 mg, 1.1 mmol) under nitrogen atmosphere. After stirring at 0 °C for 5 min, the mixture was warmed to room temperature. After stirring for 10 min, methyl iodide was added, and the resulting mixture was stirred for 2 h at room temperature (monitored by TLC). The mixture was diluted with water, extracted with ethyl acetate, washed with brine, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, and concentrated to give the crude product, which was purified by silica gel column chromatography using petroleum ether/ethyl acetate (20:1, v/v) as an eluent to afford the product <b>2</b> as a colorless oil (78 mg, 31% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.33–7.29 (m, 4H), 7.23–7.19 (m, 6H), 5.71–5.62 (m, 1H), 5.33 (dd, <i>J</i> = 15.5, 8.3 Hz, 1H), 3.48 (dd, <i>J</i> = 14.3, 6.9 Hz, 1H), 3.23 (s, 3H), 2.76 (t, <i>J</i> = 7.6 Hz, 2H), 2.67–2.62 (m, 2H), 2.44 (dd, <i>J</i> = 14.6, 7.3 Hz, 2H), 1.99–1.87 (m, 1H), 1.75 (ddd, <i>J</i> = 20.4, 14.1, 6.6 Hz, 1H). HRMS (ESI): <i>m</i>/<i>z</i> [M + Na]<sup>+</sup> calcd for C<sub>20</sub>H<sub>24</sub>ONa: 303.1725; found: 303.1717.</div></div><div id="sec4_2_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> 1,7-Diphenylheptan-3-one (<b>3</b>)</h4><div class="NLM_p last">To a solution of (<i>E</i>)-1,7-diphenylhept-4-en-3-one <b>F11</b> (100 mg, 0.4 mmol) in ethyl alcohol (10 mL) was added Pd/C (10 mg, 10% wt). After stirring at room temperature for 2 h (monitored by TLC), the mixture was filtered and washed with ethyl alcohol. The organic residue was concentrated and purified by column chromatography on silica gel to afford 1,7-diphenylheptan-3-one <b>3</b> as a colorless oil (54 mg, 51% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.24–7.19 (m, 4H), 7.17–7.12 (m, 6H), 2.86–2.79 (m, 2H), 2.75–2.71 (m, 2H), 2.58–2.54 (m, 2H), 2.44–2.41 (m, 2H), 1.55–1.51 (m, 4H).</div></div><div id="sec4_2_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> 3-Phenyl-<i>N</i>-(3-phenylpropyl)propenamide (<b>4</b>)</h4><div class="NLM_p last">To a suspension of 3-phenylpropanoic acid (1.0 g, 6.7 mmol) in DMF (20 mL) was added EDCI (1.27 g, 6.7 mmol), HOBt (900 mg, 6.7 mmol), and <i>N</i>,<i>N</i>-diisopropylethylamine (DIPEA, 2.6 mL, 20 mmol). After stirring at room temperature for 30 min, 3-phenylpropan-1-aminethe (0.9 mL, 6.7 mmol) was added and stirred at room temperature for 12 h (monitored by TLC) before being concentrated and purified by column chromatography on silica gel to afford 3-phenyl-N-(3-phenylpropyl)propenamide <b>4</b> as a white solid (1.4 g, 79% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.30–7.25 (m, 4H), 7.21–7.18 (m, 4H), 7.15–7.11 (m, 2H), 3.24 (dd, <i>J</i> = 13.0, 7.0 Hz, 2H), 2.94 (t, <i>J</i> = 7.7 Hz, 2H), 2.62–2.52 (m, 2H), 2.46–2.38 (m, 2H), 1.76 (dt, <i>J</i> = 14.6, 7.5 Hz, 2H).</div></div><div id="sec4_2_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> 1,7-Diphenylhept-4-yn-3-one (<b>5</b>)</h4><div class="NLM_p last">To a solution of but-3-yn-1-ylbenzene (1.0 mL, 7.7 mmol) in dry tetrahydrofuran (3 mL) at −78 °C, <i>n</i>-BuLi (3.2 mL, 2.4 mol in hexane) was added dropwise under nitrogen atmosphere. After stirring for 30 min at this temperature, 3-phenylpropanal (0.9 mL, 7.0 mmol) was added, then the system was warmed to room temperature and stirred for 3 h (monitored by TLC). The mixture was quenched with a saturated solution of NH<sub>4</sub>Cl and extracted with ethyl acetate. The combined organic layer was washed with brine, dried over sodium sulfate, filtered, and evaporated to give the crude intermediate as a yellow oil. Next, the alcohol intermediate was oxidized using general procedure (PCC/CH<sub>2</sub>Cl<sub>2</sub>) to give the title compound as a colorless oil (586 mg, 32% yield over two steps). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.32–7.18 (m, 8H), 7.17–7.13 (m, 2H), 2.97–2.78 (m, 6H), 2.64 (t, <i>J</i> = 7.4 Hz, 2H).</div></div><div id="sec4_2_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> (4<i>E</i>, 6<i>E</i>)-1,7-Diphenylhepta-4,6-dien-3-one (<b>6</b>)</h4><div class="NLM_p last">The title compound was prepared as a yellow solid using a method similar to that described for compound <b>F11</b> in 13% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.45 (d, <i>J</i> = 7.8 Hz, 2H), 7.38–7.25 (m, 6H), 7.22–7.18 (m, 3H), 6.95–6.80 (m, 2H), 6.27 (d, <i>J</i> = 15.5 Hz, 1H), 3.01–2.95 (m, 2H), 2.94–2.87 (m, 2H).</div></div><div id="sec4_2_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> (<i>E</i>)-1,6-Diphenylhex-3-en-2-one (<b>7</b>)</h4><div class="NLM_p last">The title compound was prepared as a colorless oil using a method similar to that described for compound <b>F11</b> in 41% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.35–7.22 (m, 5H), 7.20–7.16 (m, 3H), 7.11 (d, <i>J</i> = 7.3 Hz, 2H), 6.91 (dt, <i>J</i> = 15.6, 6.8 Hz, 1H), 6.13 (d, <i>J</i> = 15.8 Hz, 1H), 3.78 (s, 2H), 2.73 (t, <i>J</i> = 7.6 Hz, 2H), 2.49 (q, <i>J</i> = 7.2 Hz, 2H). Retention time 2.62 min, HPLC purity > 97%.</div></div><div id="sec4_2_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> (<i>E</i>)-1,5-Diphenylpent-2-en-1-one (<b>8</b>)</h4><div class="NLM_p last">The title compound was prepared as a colorless oil using a method similar to that described for compound <b>F11</b> in 77% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.94 (d, <i>J</i> = 7.4 Hz, 2H), 7.58 (t, <i>J</i> = 7.3 Hz, 1H), 7.48 (t, <i>J</i> = 7.7 Hz, 2H), 7.39–7.32 (m, 2H), 7.28–7.25 (m, 3H), 7.18–7.09 (m, 1H), 6.92 (d, <i>J</i> = 15.4 Hz, 1H), 2.88 (t, <i>J</i> = 7.6 Hz, 2H), 2.67 (dd, <i>J</i> = 14.8, 7.2 Hz, 2H). Retention time 2.46 min, HPLC purity = 100%.</div></div><div id="sec4_2_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> (<i>E</i>)-1,4-Diphenylbut-2-en-1-one (<b>9</b>)</h4><div class="NLM_p last">The title compound was prepared as a white solid using a method similar to that described for compound <b>F11</b> in 76% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.01 (d, <i>J</i> = 7.3 Hz, 2H), 7.58 (t, <i>J</i> = 7.4 Hz, 1H), 7.49 (t, <i>J</i> = 6.3 Hz, 2H), 7.39 (d, <i>J</i> = 7.4 Hz, 2H), 7.35–7.25 (m, 3H), 6.86 (d, <i>J</i> = 15.3 Hz, 1H), 6.59–6.50 (m, 1H), 3.92 (d, <i>J</i> = 6.2 Hz, 2H). Retention time 2.61 min, HPLC purity > 98%.</div></div><div id="sec4_2_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> (<i>E</i>)-Chalcone (<b>10</b>)</h4><div class="NLM_p last">The title compound was prepared as a yellow solid using a method similar to that described for compound <b>F11</b> in 92% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.05–7.98 (m, 2H), 7.81 (d, <i>J</i> = 15.7 Hz, 1H), 7.64 (dd, <i>J</i> = 6.2, 2.6 Hz, 2H), 7.61–7.46 (m, 4H), 7.44–7.36 (m, 3H). Retention time 2.57 min, HPLC purity > 99%.</div></div><div id="sec4_2_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> (<i>E</i>)-1,6-Diphenylhex-2-en-1-one (<b>11</b>)</h4><div class="NLM_p last">The title compound was prepared as a white solid using a method similar to that described for compound <b>F11</b> in 86% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.92 (d, <i>J</i> = 7.0 Hz, 2H), 7.55 (t, <i>J</i> = 7.4 Hz, 1H), 7.46 (t, <i>J</i> = 7.5 Hz, 2H), 7.29 (t, <i>J</i> = 7.7 Hz, 2H), 7.19 (d, <i>J</i> = 8.0 Hz, 3H), 7.08 (dt, <i>J</i> = 15.4, 6.9 Hz, 1H), 6.89 (d, <i>J</i> = 15.4 Hz, 1H), 2.71–2.66 (m, 2H), 2.35 (td, <i>J</i> = 8.2, 1.4 Hz, 2H), 1.92–1.82 (m, 2H). Retention time 3.23 min, HPLC purity > 98%.</div></div><div id="sec4_2_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> (<i>E</i>)-1,7-Diphenylhept-2-en-1-one (<b>12</b>)</h4><div class="NLM_p last">The title compound was prepared as a white solid using a method similar to that described for compound <b>F11</b> in 88% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.91 (d, <i>J</i> = 7.1 Hz, 2H), 7.54 (t, <i>J</i> = 7.4 Hz, 1H), 7.45 (t, <i>J</i> = 7.5 Hz, 2H), 7.31–7.25 (m, 2H), 7.20–7.16 (m, 3H), 7.05 (dt, <i>J</i> = 15.3, 6.9 Hz, 1H), 6.86 (d, <i>J</i> = 15.4 Hz, 1H), 2.64 (t, <i>J</i> = 7.5 Hz, 2H), 2.39–2.30 (m, 2H), 1.71–1.65 (m, 2H), 1.61–1.52 (m, 2H). Retention time 3.82 min, HPLC purity > 92%.</div></div><div id="sec4_2_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> (<i>E</i>)-Pent-2-ene-1,5-diyldibenzene (<b>13</b>)</h4><div class="NLM_p last">Step a: To a solution of 3-phenylpropanal (1.0 g, 7.5 mmol) in THF (10 mL) at 0 °C was added slowly a solution of vinylmagnesium bromide solution (9 mmol, 1.0 M in THF). The resulting mixture was allowed to warm to room temperature and stirred for 6 h (monitored by TLC). Upon completion, the reaction was quenched with a saturated solution of NH<sub>4</sub>Cl and extracted with ethyl acetate. The organic phase was separated, washed with brine, dried over sodium sulfate, filtered, and evaporated to give the intermediate 5-phenylpent-1-en-3-ol as a colorless oil, which then was converted to the corresponding allylic acetate (5-phenylpent-1-en-3-yl acetate, 0.9 g, 62% yield) by the acetylation reaction (Ac<sub>2</sub>O/pyridine/CH<sub>2</sub>Cl<sub>2</sub>). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.33–7.23 (m, 2H), 7.21–7.14 (m, 3H), 5.81 (ddd, <i>J</i> = 17.0, 10.5, 6.3 Hz, 1H), 5.29–5.18 (m, 3H), 2.65 (ddd, <i>J</i> = 9.1, 6.5, 4.6 Hz, 2H), 2.06 (s, 3H), 2.03–1.84 (m, 2H). Step b: A suspension of Pd(OAc)<sub>2</sub> (11 mg, 5% mmol), 1,10-phenanthroline (11 mg, 6% mmol), and AgSbF<sub>6</sub> (17 mg, 5% mmol) in THF (5 mL) was treated with ultrasonic energy at rt for 1 min and then stirred at rt for 3 min to give a yellow suspension. Phenylboronic acid (90 mg, 0.7 mmol) and 5-phenylpent-1-en-3-yl acetate (100 mg, 0.5 mmol) were sequentially added at rt. The resulting mixture was heated at 60 °C for 12 h (monitored by TLC) and cooled to room temperature. The mixture was diluted with CH<sub>2</sub>Cl<sub>2</sub>, concentrated, and purified by column chromatography on silica gel to afford (<i>E</i>)-pent-2-ene-1,5-diyldibenzene <b>13</b> as a colorless oil (89 mg, 82% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.35–7.31 (m, 4H), 7.26–7.22 (m, 4H), 7.20–7.16 (m, 2H), 5.69–5.55 (m, 2H), 3.37 (d, <i>J</i> = 5.6 Hz, 2H), 2.80–2.71 (m, 2H), 2.44–2.38 (m, 2H).</div></div><div id="sec4_2_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> (<i>E</i>)-3-Methyl-1,5-diphenylpent-2-en-1-one (<b>14</b>) and (<i>Z</i>)-3-methyl-1,5-diphenylpent-2-en-1-one (<b>15</b>)</h4><div class="NLM_p last">To a suspension of acetophenone (0.6 mL, 5 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (10 mL) at −78 °C was added a solution of TiCl<sub>4</sub> (7.5 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (10 mL) and Bu<sub>3</sub>N (2.1 mL, 9 mmol). After stirring for 1.5 h at −78 °C, 2-oxo-4-phenylbutan-1-ylium (0.7 mL, 5 mmol) was added slowly; then, the solution was warmed to rt and stirred for additional 1.5 h. The reaction mixture was cooled to 0 °C, and mesyl chloride (0.8 mL, 10 mmol) and Et<sub>3</sub>N (0.9 mL, 6 mmol) were added. After being stirred for 3 h at rt, the reaction was quenched with saturated aqueous NaHCO<sub>3</sub> and extracted with CH<sub>2</sub>Cl<sub>2</sub>. The organic phase was separated, washed with brine, dried over sodium sulfate, filtered, and concentrated in vacuo. Purification by column chromatography on silica gel gave each stereoisomer of 3-methyl-1,5-diphenylpent-2-en-1-one in 8% yield for (<i>E</i>)-isomer <b>14</b> (100 mg, a yellow oil) and in 2% yield for (<i>Z</i>)-isomer <b>15</b> (25 mg, white solid). (<i>E</i>)-isomer <b>14</b>:<sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.84–7.77 (m, 2H), 7.47–7.43 (m, 3H), 7.36 (t, <i>J</i> = 7.3 Hz, 2H), 7.27–7.21 (m, 3H), 6.66 (d, <i>J</i> = 1.0 Hz, 1H), 2.97–2.88 (m, 2H), 2.64–2.57 (m, 2H), 2.27 (d, <i>J</i> = 1.1 Hz, 3H); <sup>13</sup>C{1H} NMR (126 MHz, CDCl<sub>3</sub>) δ 192.0, 158.2, 141.2, 139.3, 132.5, 128.7, 128.6, 128.5, 128.4, 126.3, 121.7, 43.1, 34.1, 19.9. HRMS (ESI): <i>m</i>/<i>z</i> [M + Na]<sup>+</sup> calcd for C<sub>18</sub>H<sub>18</sub>ONa: 273.1255; found: 273.1253. (<i>Z</i>)-isomer <b>15</b>:<sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.91 (d, <i>J</i> = 7.1 Hz, 2H), 7.53–7.48 (m, 1H), 7.46–7.42 (m, 2H), 7.32–7.22 (m, 4H), 7.18–7.14 (m, 1H), 6.76 (s, 1H), 2.95–2.81 (ddtt, <i>J</i> = 1.3 Hz, 4H), 1.99 (d, <i>J</i> = 1.3 Hz, 3H); <sup>13</sup>C{1H} NMR (126 MHz, CDCl<sub>3</sub>) δ 191.3, 159.8, 141.9, 139.4, 132.4, 128.64, 128.57, 128.5, 128.3, 126.1, 121.8, 36.6, 34.7, 26.2. HRmS (ESI): <i>m</i>/<i>z</i> [M + Na]<sup>+</sup> calcd for C<sub>18</sub>H<sub>18</sub>ONa: 273.1255; found: 273.1253.</div></div><div id="sec4_2_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> (4-Benzylpiperazin-1-yl)(phenyl)methanone (<b>16</b>)</h4><div class="NLM_p last">A suspension of (chloromethyl)benzene (120 μL, 1.1 mmol), phenyl(piperazin-1-yl)methanone (200 mg, 1.1 mmol), and Et<sub>3</sub>N (150 μL, 1.1 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (10 mL) was refluxed overnight. The reaction mixture was cooled to rt, concentrated, and purified by column chromatography on silica gel to afford (4-benzylpiperazin-1-yl)(phenyl)methanone <b>16</b> as a yellow oil (70 mg, 24% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.39 (s, 5H), 7.34–7.23 (m, 5H), 3.79 (brs, 2H), 3.54 (s, 2H), 3.43 (s, 2H), 2.54–2.38 (m, 4H).</div></div><div id="sec4_2_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> (4-Phenethylpiperazin-1-yl)(phenyl)methanone (<b>17</b>)</h4><div class="NLM_p last">A suspension of (2-chloroethyl)benzene (0.1 mL, 0.8 mmol) and phenyl(piperazin-1-yl)methanone (100 mg, 0.5 mmol) was treated with K<sub>2</sub>CO<sub>3</sub> (73 mg, 0.5 mmol) and KI (9 mg, 5% mmol). After stirring at 50 °C overnight, the reaction mixture was cooled to rt, concentrated, and purified by column chromatography on silica gel to afford (4-benzylpiperazin-1-yl)(phenyl)methanone <b>17</b> as a yellow solid (294 mg, 72% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.53–7.37 (m, 5H), 7.31 (dd, <i>J</i> = 13.5, 6.4 Hz, 2H), 7.26–7.22 (m, 3H), 4.34 (t, <i>J</i> = 6.9 Hz, 2H), 3.85–3.27 (m, 8H), 2.97 (t, <i>J</i> = 6.7 Hz, 2H).</div></div><div id="sec4_2_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> (1-Benzylpiperidin-4-yl)(phenyl)methanone (<b>18</b>)</h4><div class="NLM_p last">To a suspension of 1-benzylpiperidine-4-carboxylic acid (500 mg, 2.3 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (10 mL) was added oxalyl chloride (212 μL, 2.5 mmol) at rt. After stirring at this temperature for 15 h, the reaction mixture was concentrated to afford the crude acyl chloride product. A solution of 1-benzylpiperidine-4-carbonyl chloride in benzene (2 mL) was treated with AlCl<sub>3</sub> (456 mg, 3.42 mmol). The reaction mixture was heated to 90 °C for 8, then cooled to rt, poured into water, and diluted with CH<sub>2</sub>Cl<sub>2</sub>. The organic phase was concentrated and purified by column chromatography on silica gel to afford (1-benzylpiperidin-4-yl)(phenyl)methanone <b>18</b> as a yellow solid (39 mg, 62% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.93 (d, <i>J</i> = 7.2 Hz, 2H), 7.55 (ddd, <i>J</i> = 6.6, 3.9, 1.3 Hz, 1H), 7.46 (t, <i>J</i> = 7.5 Hz, 2H), 7.35–7.23 (m, 5H), 3.55 (s, 2H), 3.29–3.21 (m, 1H), 2.98 (dt, <i>J</i> = 5.6, 2.5 Hz, 2H), 2.20–2.08 (m, 2H), 1.87–1.83 (m, 2H).</div></div><div id="sec4_2_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> (2-Phenethylcyclopropyl)(phenyl)methanone (<b>19</b>)</h4><div class="NLM_p last">A mixture of compound 8 (95 mg, 0.4 mmol), trimethylsulfoxonium iodide (185 mg, 0.8 mmol), and tetrabutylammonium bromide (27 mg, 8% mmol) in CH<sub>2</sub>Cl<sub>2</sub> (10 mL) was stirred at room temperature for 15 min. 50% aqueous NaOH solution (5 mL) was added dropwise, and the reaction mixture was refluxed at 80 °C for 4 h. The reaction mixture was cooled to rt, diluted with CH<sub>2</sub>Cl<sub>2</sub>, and washed twice with H<sub>2</sub>O. The organic layer was dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated in vacuo. The crude product was purified by flash chromatography on silica gel to give the title product <b>19</b> as a colorless oil (69 mg, 66% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.91 (d, <i>J</i> = 7.4 Hz, 2H), 7.55 (t, <i>J</i> = 7.4 Hz, 1H), 7.45 (t, <i>J</i> = 7.6 Hz, 2H), 7.22 (d, <i>J</i> = 7.0 Hz, 2H), 7.14–7.13 (m, 3H), 2.76 (dd, <i>J</i> = 14.5, 6.7 Hz, 2H), 2.40 (dt, <i>J</i> = 8.2, 4.2 Hz, 1H), 1.83–1.65 (m, 2H), 1.68–1.62 (m, 1H), 1.46 (dt, <i>J</i> = 8.3, 4.1 Hz, 1H), 0.91 (ddd, <i>J</i> = 7.9, 6.4, 3.6 Hz, 1H).</div></div><div id="sec4_2_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> 4-Phenethyl-2-phenylpyridine (<b>20</b>)</h4><div class="NLM_p last">Step a: To a solution of 4-bromo-2-chloropyridine (1.0 g, 5.22 mmol) in dioxane/H<sub>2</sub>O (25 mL, v/v = 4:1) were added Pd(dppf)Cl<sub>2</sub> (425 mg, 0.5 mmol), K<sub>3</sub>PO<sub>4</sub> (2.2 g, 10.4 mmol), and (<i>E</i>)-4,4,5,5-tetramethyl-2-styryl-1,3,2-dioxaborolane (1.2 g, 5.2 mmol) under nitrogen atmosphere. After refluxing for 6 h (monitored by TLC), the mixture was purified by column chromatography on silica gel to afford (<i>E</i>)-2-chloro-4-styrylpyridine (293 mg, 30% yield). Step b: To a solution of (<i>E</i>)-2-chloro-4-styrylpyridine (300 mg, 1.4 mmol) in toluene/H<sub>2</sub>O/EtOH (10 mL, v/v/v = 4:4:1) were added Pd(PPh<sub>3</sub>)<sub>4</sub> (146 mg, 0.1 mmol), Na<sub>2</sub>CO<sub>3</sub> (2.2 g, 10.4 mmol), and phenylboronic acid (512 mg, 4.2 mmol) under nitrogen atmosphere. After refluxing for 12 h (monitored by TLC), the mixture was purified by column chromatography on silica gel to afford (<i>E</i>)-2-phenyl-4-styrylpyridine (142 mg, 39% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.54 (d, <i>J</i> = 5.0 Hz, 1H), 7.90 (d, <i>J</i> = 7.4 Hz, 2H), 7.67 (s, 1H), 7.45 (d, <i>J</i> = 7.4 Hz, 2H), 7.40–7.31 (m, 3H), 7.27–7.25 (m, 3H), 7.09–7.06 (m, 1H), 6.97 (d, <i>J</i> = 16.3 Hz, 1H), 6.80–6.64 (m, 1H). Step c: To a solution of (<i>E</i>)-2-phenyl-4-styrylpyridine (100 mg, 0.4 mmol) in methyl alcohol (10 mL) was added Pd/C (10 mg, 10% wt). The mixture was treated with H<sub>2</sub>. After stirring at room temperature for 8 h (monitored by TLC), the mixture was filtered and washed with methyl alcohol. The organic residue was concentrated and purified by column chromatography on silica gel to afford 4-phenethyl-2-phenylpyridine <b>20</b> as a colorless oil (89 mg, 86% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.60 (d, <i>J</i> = 5.0 Hz, 1H), 7.96 (d, <i>J</i> = 7.0 Hz, 2H), 7.54–7.40 (m, 4H), 7.33 (t, <i>J</i> = 7.3 Hz, 2H), 7.27–7.18 (m, 3H), 7.07 (dd, <i>J</i> = 5.0, 1.4 Hz, 1H), 3.01 (s, 4H).</div></div><div id="sec4_2_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> 3-Phenethyl-5-phenylpyridine (<b>21</b>)</h4><div class="NLM_p last">The title compound was prepared as a white solid using a method similar to that described for compound <b>20</b> in 91% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.68 (s, 1H), 8.41 (s, 1H), 7.59 (s, 1H), 7.53 (d, <i>J</i> = 7.3 Hz, 2H), 7.47 (t, <i>J</i> = 7.4 Hz, 2H), 7.40 (t, <i>J</i> = 7.3 Hz, 1H), 7.31 (t, <i>J</i> = 7.2 Hz, 2H), 7.23 (t, <i>J</i> = 7.3 Hz, 1H), 7.18 (d, <i>J</i> = 6.8 Hz, 2H), 3.01–2.98 (m, 4H).</div></div><div id="sec4_2_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> (<i>E</i>)-1-Phenylpent-2-en-1-one (<b>22</b>)</h4><div class="NLM_p last">The title compound was prepared as a colorless oil using a method similar to that described for compound <b>F11</b> in 89% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.93 (d, <i>J</i> = 7.1 Hz, 2H), 7.59–7.52 (m, 1H), 7.47 (t, <i>J</i> = 7.5 Hz, 2H), 7.12 (dt, <i>J</i> = 15.4, 6.4 Hz, 1H), 6.88 (dt, <i>J</i> = 15.4, 1.6 Hz, 1H), 2.40–2.29 (m, 2H), 1.15 (t, <i>J</i> = 7.4 Hz, 3H).</div></div><div id="sec4_2_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> (<i>E</i>)-5-(2-Bromophenyl)-1-phenylpent-2-en-1-one (<b>23</b>)</h4><div class="NLM_p last">The title compound was prepared as a colorless oil using a method similar to that described for compound <b>F11</b> in 55% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.62–7.58 (m, 1H), 7.38–7.34 (m, 1H), 7.32–7.25 (m, 4H), 7.23–7.15 (m, 3H), 6.70 (dt, <i>J</i> = 15.8, 6.8 Hz, 1H), 6.44 (dt, <i>J</i> = 15.8, 1.4 Hz, 1H), 2.80 (t, <i>J</i> = 7.7 Hz, 2H), 2.66–2.56 (m, 2H). <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ 195.2, 151.7, 141.1, 140.7, 133.5, 131.3, 130.9, 129.1, 128.7, 128.5, 127.3, 126.4, 119.5, 34.6, 34.3. HRMS (ESI): <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>17</sub>H<sub>16</sub>BrO: 315.0385; found: 315.0367. Retention time 2.80 min, HPLC purity > 99%.</div></div><div id="sec4_2_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> (<i>E</i>)-5-(4-Bromophenyl)-1-phenylpent-2-en-1-one (<b>24</b>)</h4><div class="NLM_p last">The title compound was prepared as a yellow solid using a method similar to that described for compound <b>F11</b> in 77% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.91–7.85 (m, 2H), 7.56 (dd, <i>J</i> = 10.5, 4.3 Hz, 1H), 7.51–7.40 (m, 4H), 7.12–6.99 (m, 3H), 6.86 (t, <i>J</i> = 8.4 Hz, 1H), 2.81 (t, <i>J</i> = 7.6 Hz, 2H), 2.66–2.59 (m, 2H). <sup>13</sup>C{1H} NMR (125 MHz, CDCl<sub>3</sub>) δ 190.7, 147.7, 139.7, 137.8, 132.8, 131.6, 130.2, 128.6, 126.8, 120.0, 34.3, 33.9. HRMS (ESI): <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>17</sub>H<sub>16</sub>BrO: 315.0385; found: 315.0373. Retention time 3.32 min, HPLC purity > 98%.</div></div><div id="sec4_2_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> (<i>E</i>)-5-Phenyl-1-(piperidin-1-yl)pent-2-en-1-one (<b>25</b>)</h4><div class="NLM_p last">To a suspension of sodium hydride (0.9 g, 60 wt% in mineral oil) in dry THF (30 mL) at 0 °C was added ethyl 2-(diethoxyphosphoryl)acetate (4.4 mL, 22.4 mmol). After stirring at this temperature for 1.5 h, a solution of 3-phenylpropanal (2.0 g, 14.9 mmol) in THF was added. Then, the reaction mixture was allowed to warm to rt and stirred for 4 h. Upon completion, the reaction was quenched with a saturated solution of NH<sub>4</sub>Cl and extracted with diethyl ether. The organic phase was separated, washed with brine, dried over sodium sulfate, filtered, and evaporated to give the intermediate ester, which was hydrolyzed using the general procedure (NaOH/EtOH) to yield (<i>E</i>)-5-phenylpent-2-enoic acid as a yellow solid (2.4 g, 90% yield for two steps), <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ 7.34–7.23 (m, 2H), 7.20–7.14 (m, 3H), 6.96 (dt, <i>J</i> = 15.3, 6.9 Hz, 1H), 5.78 (d, <i>J</i> = 15.6 Hz, 1H), 2.77 (t, <i>J</i> = 7.6 Hz, 2H), 2.52 (q, <i>J</i> = 7.4 Hz, 2H). To a suspension (<i>E</i>)-5-phenylpent-2-enoic acid (200 mg, 1.1 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (10 mL) at rt were added EDCI (383 mg, 2.0 mmol), DMAP (122 mg, 1.0 mmol), and Et<sub>3</sub>N (0.6 mL, 4.0 mmol). After stirring at rt for 30 min, piperidine (92 μL, 1.0 mmol) was added and stirred overnight. The mixture was diluted with H<sub>2</sub>O and extracted by CH<sub>2</sub>Cl<sub>2</sub>. The organic phase was concentrated and purified by column chromatography on silica gel to afford (<i>E</i>)-5-phenyl-1-(piperidin-1-yl)pent-2-en-1-one as a yellow oil (60.8 mg, 25%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.27–7.21 (m, 2H), 7.19–7.14 (m, 3H), 6.87–6.74 (m, 1H), 6.18 (d, <i>J</i> = 15.1 Hz, 1H), 3.61–3.47 (m, 2H), 3.37–3.29 (m, 2H), 2.74 (t, <i>J</i> = 7.7 Hz, 2H), 2.49 (dd, <i>J</i> = 14.5, 7.3 Hz, 2H), 1.65–1.40 (m, 6H).</div></div><div id="sec4_2_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> (<i>E</i>)-5-Phenyl-1-(o-tolyl)pent-2-en-1-one (<b>26</b>)</h4><div class="NLM_p last">The title compound was prepared as a colorless oil using a method similar to that described for compound <b>F11</b> in 82% yield. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 7.36–7.23 (m, 4H), 7.21–7.13 (m, 5H), 6.72 (dt, <i>J</i> = 15.7, 6.7 Hz, 1H), 6.45 (d, <i>J</i> = 15.8 Hz, 1H), 2.77 (t, <i>J</i> = 7.6 Hz, 2H), 2.56 (dd, <i>J</i> = 14.9, 7.0 Hz, 2H), 2.35 (s, 3H). <sup>13</sup>C{1H} NMR (125 MHz, CDCl<sub>3</sub>) δ 197.2, 150.33, 150.31, 140.8, 138.9, 136.8, 131.4, 131.3, 130.4, 128.6, 128.5, 128.2, 126.3, 125.4, 34.5, 34.4, 20.2. HRMS (ESI): <i>m</i>/<i>z</i> [M + Na]<sup>+</sup> calcd for C<sub>18</sub>H<sub>18</sub>ONa: 273.1255; found: 273.1243. Retention time 2.63 min, HPLC purity > 96%.</div></div><div id="sec4_2_27" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> (<i>E</i>)-1-(2,6-Dimethylphenyl)-5-phenylpent-2-en-1-one (<b>27</b>)</h4><div class="NLM_p last">The title compound was prepared as a colorless oil using a method similar to that described for compound <b>F11</b> in 80% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.26 (t, <i>J</i> = 7.4 Hz, 2H), 7.21–7.08 (m, 4H), 7.00 (d, <i>J</i> = 7.5 Hz, 2H), 6.49 (dt, <i>J</i> = 15.9, 6.8 Hz, 1H), 6.28 (dt, <i>J</i> = 15.9, 1.3 Hz, 1H), 2.75 (t, <i>J</i> = 7.6 Hz, 2H), 2.59–2.51 (m, 2H), 2.12 (s, 6H). <sup>13</sup>C{1H} NMR (125 MHz, CDCl<sub>3</sub>) δ 201.4, 151.5, 140.6, 140.0, 134.0, 133.0, 128.7, 128.6, 128.5, 127.6, 126.4, 34.38, 34.35, 19.4. HRMS (ESI): <i>m</i>/<i>z</i> [M + Na]<sup>+</sup> calcd for C<sub>19</sub>H<sub>20</sub>ONa: 287.1412; found: 287.1405. Retention time 3.08 min, HPLC purity > 97%.</div></div><div id="sec4_2_28" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> (<i>E</i>)-5-Phenyl-1-(m-tolyl)pent-2-en-1-one (<b>28</b>)</h4><div class="NLM_p last">The title compound was prepared as a colorless oil using a method similar to that described for compound <b>F11</b> in 92% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.72–7.69 (m, 2H), 7.41–7.31 (m, 4H), 7.27–7.24 (m, 3H), 7.10 (dt, <i>J</i> = 15.4, 6.8 Hz, 1H), 6.89 (dt, <i>J</i> = 15.4, 1.3 Hz, 1H), 2.88 (t, <i>J</i> = 7.7 Hz, 2H), 2.71–2.64 (m, 2H), 2.44 (s, 3H). <sup>13</sup>C{1H} NMR (125 MHz, CDCl<sub>3</sub>) δ 191.2, 148.3, 141.0, 138.4, 138.1, 133.5, 129.2, 128.6, 128.5, 128.5, 126.9, 126.3, 125.9, 34.6, 34.6, 21.5. HRMS (ESI): <i>m</i>/<i>z</i> [M + Na]<sup>+</sup> calcd for C<sub>18</sub>H<sub>18</sub>ONa: 273.1255; found: 273.1245. Retention time 2.86 min, HPLC purity > 96%.</div></div><div id="sec4_2_29" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> (<i>E</i>)-5-Phenyl-1-(p-tolyl)pent-2-en-1-one (<b>29</b>)</h4><div class="NLM_p last">The title compound was prepared as a colorless oil using a method similar to that described for compound <b>F11</b> in 90% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.85 (d, <i>J</i> = 8.2 Hz, 2H), 7.33–7.37 (m, 2H), 7.30–7.24 (m, 5H), 7.11 (dt, <i>J</i> = 15.3, 6.8 Hz, 1H), 6.91 (dt, <i>J</i> = 15.4, 1.3 Hz, 1H), 2.88 (t, <i>J</i> = 7.7 Hz, 2H), 2.71–2.64 (m, 2H), 2.45 (s, 3H). LR-MS (ESI) <i>m</i>/<i>z</i> 273.1 [M + Na]<sup>+</sup>. Retention time 3.23 min, HPLC purity > 95%.</div></div><div id="sec4_2_30" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> (<i>E</i>)-1-(2-Methoxyphenyl)-5-phenylpent-2-en-1-one (<b>30</b>)</h4><div class="NLM_p last">The title compound was prepared as a colorless oil using a method similar to that described for compound <b>F11</b> in 92% yield. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 7.52–7.40 (m, 2H), 7.35–7.16 (m, 5H), 7.05–6.84 (m, 3H), 6.75–6.65 (m, 1H), 3.83 (s, 3H), 2.81 (t, <i>J</i> = 7.7 Hz, 2H), 2.59 (dd, <i>J</i> = 14.6, 7.4 Hz, 2H). Retention time 2.27 min, HPLC purity > 98%.</div></div><div id="sec4_2_31" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> (<i>E</i>)-1-(3-Methoxyphenyl)-5-phenylpent-2-en-1-one (<b>31</b>)</h4><div class="NLM_p last">The title compound was prepared as a colorless oil using a method similar to that described for compound <b>F11</b> in 94% yield.<sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 7.45–7.27 (m, 5H), 7.25–7.20 (m, 3H), 7.07 (dd, <i>J</i> = 14.9, 7.3 Hz, 2H), 6.85 (d, <i>J</i> = 15.3 Hz, 1H), 3.86 (s, 3H), 2.85 (t, <i>J</i> = 7.7 Hz, 2H), 2.65 (dd, <i>J</i> = 14.6, 7.6 Hz, 2H). Retention time 2.59 min, HPLC purity > 96%.</div></div><div id="sec4_2_32" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> (<i>E</i>)-1-(4-Methoxyphenyl)-5-phenylpent-2-en-1-one (<b>32</b>)</h4><div class="NLM_p last">The title compound was prepared as a colorless oil using a method similar to that described for compound <b>F11</b> in 88% yield. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 7.89 (d, <i>J</i> = 8.9 Hz, 2H), 7.34–7.13 (m, 5H), 7.12–6.81 (m, 4H), 3.83 (s, 3H), 2.83 (t, <i>J</i> = 7.6 Hz, 2H), 2.67–2.56 (m, 2H). Retention time 2.76 min, HPLC purity > 99%.</div></div><div id="sec4_2_33" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> (<i>E</i>)-1-(2-Fluorophenyl)-5-phenylpent-2-en-1-one (<b>33</b>)</h4><div class="NLM_p last">The title compound was prepared as a colorless oil using a method similar to that described for compound <b>F11</b> in 85% yield. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 7.68 (t, <i>J</i> = 7.5 Hz, 1H), 7.48 (dd, <i>J</i> = 12.9, 6.0 Hz, 1H), 7.36–6.94 (m, 8H), 6.74 (d, <i>J</i> = 15.5 Hz, 1H), 2.88–2.79 (m, 2H), 2.62 (dd, <i>J</i> = 14.6, 7.6 Hz, 2H). <sup>13</sup>C{1H} NMR (125 MHz, CDCl<sub>3</sub>) δ 189.7, 189.6, 162.1 (d, <i>J</i><sub>CF</sub> = 252.5 Hz), 149.2, 140.9, 133.7 (d, <i>J</i><sub>CF</sub> = 8.7 Hz), 130.9 (d, <i>J</i><sub>CF</sub> = 2.6 Hz), 130.1 (d, <i>J</i><sub>CF</sub> = 5.6 Hz), 128.54 (d, <i>J</i><sub>CF</sub> = 38.8 Hz), 128.52 (d, <i>J</i><sub>CF</sub> = 16.3 Hz), 128.45, 128.4, 127.1 (d, <i>J</i><sub>CF</sub> = 13.6 Hz), 126.3, 124.5 (d, <i>J</i><sub>CF</sub> = 3.4 Hz), 116.5 (d, <i>J</i><sub>CF</sub> = 22.5 Hz), 34.5. HRMS (ESI): <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>17</sub>H<sub>16</sub>FO: 255.1180; found: 255.1179. Retention time 2.84 min, HPLC purity > 96%.</div></div><div id="sec4_2_34" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> (<i>E</i>)-1-(3-Fluorophenyl)-5-phenylpent-2-en-1-one (<b>34</b>)</h4><div class="NLM_p last">The title compound was prepared as a colorless oil using a method similar to that described for compound <b>F11</b> in 89% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.64 (d, <i>J</i> = 7.7 Hz, 1H), 7.55 (ddd, <i>J</i> = 9.5, 2.5, 1.6 Hz, 1H), 7.42 (td, <i>J</i> = 8.0, 5.5 Hz, 1H), 7.31 (t, <i>J</i> = 7.4 Hz, 2H), 7.26–7.18 (m, 4H), 7.09 (dt, <i>J</i> = 15.4, 6.9 Hz, 1H), 6.80 (dt, <i>J</i> = 15.4, 1.4 Hz, 1H), 2.85 (t, <i>J</i> = 7.6 Hz, 2H), 2.68–2.60 (m, 2H). <sup>13</sup>C{1H} NMR (125 MHz, CDCl<sub>3</sub>) δ 189.6 (d, <i>J</i><sub>CF</sub> = 1.9 Hz), 162.9 (d, <i>J</i><sub>CF</sub> = 247.8 Hz), 149.36, 140.77, 140.1 (d, <i>J</i><sub>CF</sub> = 6.2 Hz), 130.3 (d, <i>J</i><sub>CF</sub> = 7.7 Hz), 128.6 (d, <i>J</i><sub>CF</sub> = 17.9 Hz), 126.3 (d, <i>J</i><sub>CF</sub> = 5.4 Hz), 124.3 (d, <i>J</i><sub>CF</sub> = 2.8 Hz), 119.7 (d, <i>J</i><sub>CF</sub> = 21.5 Hz), 115.4 (d, <i>J</i><sub>CF</sub> = 22.3 Hz), 34.6, 34.5. HRMS (ESI): <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>17</sub>H<sub>16</sub>FO: 255.1180; found: 255.1186. Retention time 2.60 min, HPLC purity > 99%.</div></div><div id="sec4_2_35" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> (<i>E</i>)-1-(4-Fluorophenyl)-5-phenylpent-2-en-1-one (<b>35</b>)</h4><div class="NLM_p last">The title compound was prepared as a colorless oil using a method similar to that described for compound <b>F11</b> in 81% yield. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 7.92–7.81 (m, 2H), 7.30–7.13 (m, 4H), 7.11–7.03 (m, 3H), 7.02–6.95 (m, 2H), 3.20 (ddd, <i>J</i> = 7.2, 2.8, 1.6 Hz, 2H), 2.47 (t, <i>J</i> = 7.9 Hz, 2H). Retention time 2.51 min, HPLC purity > 97%.</div></div><div id="sec4_2_36" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> (<i>E</i>)-1-([1,1′-Biphenyl]-2-yl)-5-phenylpent-2-en-1-one (<b>36</b>)</h4><div class="NLM_p last">The title compound was prepared as a colorless oil using a method similar to that described for compound <b>F11</b> in 60% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.49–7.53 (m, 2H), 7.42 (d, <i>J</i> = 7.4 Hz, 2H), 7.37–7.24 (m, 8H), 7.04 (d, <i>J</i> = 6.9 Hz, 2H), 6.58 (dt, <i>J</i> = 15.7, 6.8 Hz, 1H), 5.96 (d, <i>J</i> = 15.7 Hz, 1H), 2.49–2.41 (m, 2H), 2.30–2.22 (m, 2H). <sup>13</sup>C{1H} NMR (150 MHz, CDCl<sub>3</sub>) δ 197.4, 148.6, 140.93, 140.90, 140.7, 139.7, 131.1, 130.6, 130.1, 129.3, 128.8, 128.7, 128.6, 128.6, 128.4, 128.3, 127.8, 127.4, 126.2, 34.4, 34.2. HRMS (ESI): <i>m</i>/<i>z</i> [M + Na]<sup>+</sup> calcd for C<sub>23</sub>H<sub>20</sub>ONa: 335.1412; found: 335.1403. Retention time 3.54 min, HPLC purity > 99%.</div></div><div id="sec4_2_37" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> (<i>E</i>)-1-([1,1′-Biphenyl]-3-yl)-5-phenylpent-2-en-1-one (<b>37</b>)</h4><div class="NLM_p last">The title compound was prepared as a colorless oil using a method similar to that described for compound <b>F11</b> in 80% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.09 (s, 1H), 7.83 (d, <i>J</i> = 7.7 Hz, 1H), 7.76 (d, <i>J</i> = 8.0 Hz, 1H), 7.60 (d, <i>J</i> = 7.3 Hz, 2H), 7.49–7.37 (m, 3H), 7.37 (t, <i>J</i> = 7.3 Hz, 1H), 7.33–7.26 (m, 2H), 7.24–7.16 (m, 3H), 7.11 (dt, <i>J</i> = 15.3, 6.8 Hz, 1H), 6.89 (d, <i>J</i> = 15.4 Hz, 1H), 2.85 (t, <i>J</i> = 7.6 Hz, 2H), 2.69–2.61 (m, 2H). HRMS (ESI): <i>m</i>/<i>z</i> [M + Na]<sup>+</sup> calcd for C<sub>23</sub>H<sub>20</sub>ONa: 335.1412; found: 335.1407. Retention time 2.64 min, HPLC purity > 96%.</div></div><div id="sec4_2_38" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> (<i>E</i>)-1-([1,1′-Biphenyl]-4-yl)-5-phenylpent-2-en-1-one (<b>38</b>)</h4><div class="NLM_p last">The title compound was prepared as a colorless oil using a method similar to that described for compound <b>F11</b> in 81% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.96 (d, <i>J</i> = 8.4 Hz, 2H), 7.68 (d, <i>J</i> = 8.4 Hz, 2H), 7.63 (d, <i>J</i> = 7.2 Hz, 2H), 7.47 (t, <i>J</i> = 7.4 Hz, 2H), 7.40 (t, <i>J</i> = 7.3 Hz, 1H), 7.35–7.28 (m, 2H), 7.25–7.20 (m, 3H), 7.12 (dt, <i>J</i> = 15.3, 6.8 Hz, 1H), 6.91 (d, <i>J</i> = 15.4 Hz, 1H), 2.87 (t, <i>J</i> = 7.7 Hz, 2H), 2.71–2.62 (m, 2H). HRMS (ESI): <i>m</i>/<i>z</i> [M + Na]<sup>+</sup> calcd for C<sub>23</sub>H<sub>20</sub>ONa: 335.1412; found: 335.1401. Retention time 3.92 min, HPLC purity > 97%.</div></div><div id="sec4_2_39" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> (<i>E</i>)-1-(Naphthalen-1-yl)-5-phenylpent-2-en-1-one (<b>39</b>)</h4><div class="NLM_p last">The title compound was prepared as a colorless oil using a method similar to that described for compound <b>F11</b> in 82% yield. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 8.21 (d, <i>J</i> = 9.4 Hz, 1H), 7.92 (d, <i>J</i> = 8.2 Hz, 1H), 7.89–7.79 (m, 1H), 7.57–7.42 (m, 4H), 7.28 (t, <i>J</i> = 7.1 Hz, 2H), 7.16 (t, <i>J</i> = 8.9 Hz, 3H), 6.91–6.76 (m, 1H), 6.62 (d, <i>J</i> = 15.7 Hz, 1H), 2.83–2.74 (m, 2H), 2.58 (dd, <i>J</i> = 14.8, 6.9 Hz, 2H). HRMS (ESI): <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>21</sub>H<sub>19</sub>O: 287.1436; found: 287.1434. Retention time 3.44 min, HPLC purity > 99%.</div></div><div id="sec4_2_40" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> (<i>E</i>)-1-(Naphthalen-2-yl)-5-phenylpent-2-en-1-one (<b>40</b>)</h4><div class="NLM_p last">The title compound was prepared as a colorless oil using a method similar to that described for compound <b>F11</b> in 84% yield. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 8.34 (s, 1H), 8.02–7.79 (m, 4H), 7.61–7.47 (m, 2H), 7.29 (d, <i>J</i> = 6.9 Hz, 2H), 7.24–7.14 (m, 3H), 7.10 (t, <i>J</i> = 6.6 Hz, 1H), 6.99 (d, <i>J</i> = 15.4 Hz, 1H), 2.86 (t, <i>J</i> = 7.6 Hz, 2H), 2.66 (dd, <i>J</i> = 14.5, 7.4 Hz, 2H). Retention time 3.70 min, HPLC purity > 99%.</div></div><div id="sec4_2_41" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> (<i>E</i>)-5-Phenyl-1-(thiophen-2-yl)pent-2-en-1-one (<b>41</b>)</h4><div class="NLM_p last">The title compound was prepared as a colorless oil using a method similar to that described for compound <b>F11</b> in 80% yield. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 7.68 (dd, <i>J</i> = 3.8, 1.1 Hz, 1H), 7.60 (dd, <i>J</i> = 4.9, 1.1 Hz, 1H), 7.29 (dd, <i>J</i> = 10.4, 4.5 Hz, 2H), 7.23–7.14 (m, 3H), 7.11 (ddd, <i>J</i> = 8.8, 5.1, 1.6 Hz, 2H), 6.77 (dd, <i>J</i> = 15.3, 1.4 Hz, 1H), 2.82 (t, <i>J</i> = 7.6 Hz, 2H), 2.67–2.54 (m, 2H). Retention time 1.99 min, HPLC purity > 99%.</div></div><div id="sec4_2_42" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> (<i>E</i>)-5-Phenyl-1-(thiophen-3-yl)pent-2-en-1-one (<b>42</b>)</h4><div class="NLM_p last">The title compound was prepared as a white solid using a method similar to that described for compound <b>F11</b> in 82% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.99 (dd, <i>J</i> = 2.8, 1.1 Hz, 1H), 7.57 (dd, <i>J</i> = 5.1, 1.1 Hz, 1H), 7.35–7.27 (m, 3H), 7.23–7.20 (m, 3H), 7.10 (dt, <i>J</i> = 15.2, 6.9 Hz, 1H), 6.75 (d, <i>J</i> = 15.4 Hz, 1H), 2.84 (t, <i>J</i> = 7.7 Hz, 2H), 2.67–2.59 (m, 2H). HRMS (ESI): <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>15</sub>H<sub>15</sub>OS:243.0844; found: 243.0841. Retention time 2.01 min, HPLC purity > 99%.</div></div><div id="sec4_2_43" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> (<i>E</i>)-1-(Furan-2-yl)-5-phenylpent-2-en-1-one (<b>43</b>)</h4><div class="NLM_p last">The title compound was prepared as a colorless oil using a method similar to that described for compound <b>F11</b> in 85% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.67–7.60 (m, 1H), 7.35–7.30 (m, 2H), 7.26–7.17 (m, 5H), 6.83 (dt, <i>J</i> = 15.5, 1.4 Hz, 1H), 6.58 (dd, <i>J</i> = 3.6, 1.7 Hz, 1H), 2.90–2.84 (m, 2H), 2.69–2.62 (m, 2H). Retention time 2.20 min, HPLC purity > 99%.</div></div><div id="sec4_2_44" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> (<i>E</i>)-1-(Furan-3-yl)-5-phenylpent-2-en-1-one (<b>44</b>)</h4><div class="NLM_p last">The title compound was prepared as a colorless oil using a method similar to that described for compound <b>F11</b> in 80% yield. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 7.61 (s, 1H), 7.36–7.11 (m, 8H), 6.80 (d, <i>J</i> = 15.5 Hz, 1H), 6.54 (s, 1H), 2.84 (t, <i>J</i> = 7.6 Hz, 2H), 2.63 (dd, <i>J</i> = 14.8, 6.9 Hz, 2H). HRMS (ESI): <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>15</sub>H<sub>15</sub>O<sub>2</sub>: 227.1072; found: 227.1066. Retention time 2.23 min, HPLC purity > 97%.</div></div><div id="sec4_2_45" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> (<i>E</i>)-1-(1-Methyl-1<i>H</i>-pyrazol-5-yl)-5-phenylpent-2-en-1-one (<b>45</b>)</h4><div class="NLM_p last">The title compound was prepared as a colorless oil using a method similar to that described for compound <b>F11</b> in 71% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.48 (d, <i>J</i> = 2.1 Hz, 1H), 7.34–7.27 (m, 2H), 7.24–7.18 (m, 3H), 7.12 (dt, <i>J</i> = 15.3, 6.9 Hz, 1H), 6.76 (d, <i>J</i> = 2.1 Hz, 1H), 6.68 (dt, <i>J</i> = 15.4, 1.5 Hz, 1H), 4.19 (s, 3H), 2.84 (t, <i>J</i> = 7.6 Hz, 2H), 2.68–2.57 (m, 2H). <sup>13</sup>C{1H} NMR (150 MHz, CDCl<sub>3</sub>) δ 180.7, 148.5, 140.8, 139.1, 137.7, 128.7, 128.5, 127.6, 126.4, 111.3, 40.3, 34.5, 34.4. HRMS (ESI): <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>15</sub>H<sub>17</sub>N<sub>2</sub>O: 241.1314; found: 241.1333. Retention time 2.16 min, HPLC purity > 99%.</div></div><div id="sec4_2_46" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61"> (<i>E</i>)-5-Phenyl-1-(pyridin-2-yl)pent-2-en-1-one (<b>46</b>)</h4><div class="NLM_p last">The title compound was prepared as a brown oil using a method similar to that described for compound <b>F11</b> in 70% yield. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 8.70 (ddd, <i>J</i> = 4.8, 1.7, 0.9 Hz, 1H), 8.17–8.07 (m, 1H), 7.85 (td, <i>J</i> = 7.7, 1.7 Hz, 1H), 7.64 (dt, <i>J</i> = 15.7, 1.5 Hz, 1H), 7.46 (ddd, <i>J</i> = 7.5, 4.8, 1.2 Hz, 1H), 7.36–7.15 (m, 6H), 2.94–2.81 (m, 2H), 2.73–2.61 (m, 2H). Retention time 2.65 min, HPLC purity > 95%.</div></div><div id="sec4_2_47" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62"> (<i>E</i>)-5-Phenyl-1-(pyridin-3-yl)pent-2-en-1-one (<b>47</b>)</h4><div class="NLM_p last">The title compound was prepared as a brown oil using a method similar to that described for compound <b>F11</b> in 75% yield. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 8.77 (d, <i>J</i> = 5.9 Hz, 2H), 7.61 (d, <i>J</i> = 5.9 Hz, 2H), 7.34–7.29 (m, 2H), 7.25–7.19 (m, 3H), 7.16–7.03 (m, 1H), 6.75 (d, <i>J</i> = 15.5 Hz, 1H), 2.86 (t, <i>J</i> = 7.6 Hz, 2H), 2.67 (dd, <i>J</i> = 14.7, 7.1 Hz, 2H). HRMS (ESI): <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>16</sub>H<sub>16</sub>NO: 238.1232; found: 238.1224. Retention time 2.63 min, HPLC purity > 93%.</div></div><div id="sec4_2_48" class="NLM_sec NLM_sec_level_3"><div id="ac_i63" class="anchor-spacer"></div><h4 class="article-section__title" id="_i63"> (<i>E</i>)-5-Phenyl-1-(pyridin-4-yl)pent-2-en-1-one (<b>48</b>)</h4><div class="NLM_p last">The title compound was prepared as a brown oil using a method similar to that described for compound <b>F11</b> in 78% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 9.10–9.03 (m, 1H), 8.77 (dd, <i>J</i> = 4.8, 1.7 Hz, 1H), 8.19–8.14 (m, 1H), 7.43 (ddd, <i>J</i> = 7.9, 4.8, 0.9 Hz, 1H), 7.31 (d, <i>J</i> = 7.0 Hz, 2H), 7.25–7.20 (m, 3H), 7.13 (dt, <i>J</i> = 15.4, 6.9 Hz, 1H), 6.83 (dt, <i>J</i> = 15.4, 1.5 Hz, 1H), 2.87 (t, <i>J</i> = 7.6 Hz, 2H), 2.73–2.63 (m, 2H). HRMS (ESI): <i>m</i>/<i>z</i> [M + H]<sup>+</sup> calcd for C<sub>16</sub>H<sub>16</sub>NO: 238.1232; found: 238.1223. Retention time 2.25 min, HPLC purity > 99%.</div></div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i64" class="anchor-spacer"></div><h3 class="article-section__title" id="_i64"> Cell culture</h3><div class="NLM_p last">Mouse primary striatal neurons were obtained from postnatal (P1) mice as previously described.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> Briefly, tissues were collected and digested with papain (P4762, Sigma), DNAase (D4527, Sigma), and Dispase (D4818, Sigma). Cells obtained were suspended in Dulbecco’s modified Eagle’s medium (DMEM) supplemented with F12 and 10% fetal bovine serum (FBS) medium and seeded onto plates precoated with poly-<span class="smallcaps smallerCapital">d</span>-lysine (P0899, Sigma). After incubation at 37 °C under 5% CO<sub>2</sub> for 2 h, the medium was changed to Neurobasal medium plus B27, N2 and Glutamax (Life Technologies) for further culture. Animal care and use were in accordance with the general guidelines published by the Association for Assessment and Accreditation of Laboratory Animal Care. The Animal Care and Use Committee of Fudan University approved all animal experimental procedures. The mouse striatal cells (STHdh) were purchased from Coriell Cell Repositories (Camden, NJ). STHdh cells were cultured in DMEM supplemented with 10% FBS at 33 °C under 5% CO<sub>2</sub>. HEK293 cells were obtained from ATCC and cultured in DMEM supplemented with 10% FBS at 37 °C under 5% CO<sub>2</sub>.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i65" class="anchor-spacer"></div><h3 class="article-section__title" id="_i65"> TR-FRET Assay</h3><div class="NLM_p">The time-resolved fluorescence resonance energy transfer (TR-FRET) assay was performed as previously described.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> For measurement of the mouse soluble HTT levels, we applied 2B7-terbium and 2166-D2 antibodies, which target sterically different and unblocked epitopes in HTT. The monoclonal antibody 2B7 binds to the 17 amino acids, immediately followed by the polyQ repeat of HTT. When applied in combination with 2166, an antibody binds to 181–810 amino acids of HTT, and the 2B7/2166 antibody pair specifically detected both wild-type and mutant HTT levels.<a onclick="showRef(event, 'ref9 ref23'); return false;" href="javascript:void(0);" class="ref ref9 ref23">(9,23)</a> For detection of mHTT aggregates, by targeting the polyproline region in exon 1 of mHTT, a mixture of 4C9 antibody labeled with either terbium or D2 fluorophore, respectively, results in a TR-FRET signal for aggregated, but not soluble mHTT.<a onclick="showRef(event, 'ref7 ref38'); return false;" href="javascript:void(0);" class="ref ref7 ref38">(7,38)</a></div><div class="NLM_p last">Then, proteins from cells or tissues were diluted with original lysis buffer, and then incubated with indicated antibody pairs diluted with HTRF buffer at a final concentration of 1.4 ng/μL for D2-labeled acceptor antibody and 0.023 ng/μL for terbium-labeled donor antibody. The signal ratio between 665 and 615 nm was detected as the HTRF signal and the background was corrected by blank samples.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i66" class="anchor-spacer"></div><h3 class="article-section__title" id="_i66"> cAMP Assay</h3><div class="NLM_p">For the inhibitory activity of compound on GPCRs, cells expressing GPCR were harvested and resuspended in DMEM containing 500 μM IBMX at a density of 2 × 10<sup>5</sup> cells/mL. The cells were then plated onto 384-well assay plates at 1000 cells/5 μL/well. Another 5 μL of buffer containing compounds at various concentrations was added to the cells. After incubation at room temperature for 30 min, 5 μL of DMEM containing GPCR agonists was added, and the incubation was continued for another 30 min. Intracellular cAMP levels were tested by a LANCE Ultra cAMP kit (PerkinElmer, TRF0264) and EnVision multiplate reader according to the manufacturer’s instructions.</div><div class="NLM_p last">To test the reversibility of compounds on GPR52, cells expressing GPR52 were seeded onto 96-well plates at a density of 3 × 10<sup>4</sup> cells/well and cultured overnight. Then, the medium was removed from the well. The control group was replaced with 50 μL of DMEM with GPR52 antagonist for 15 min. After that, another 50 μL of DMEM containing GPR52 agonist WO-459 (final 100 nM) and IBMX (500 μM) was added and the incubation was continued for another 30 min. The recovery group was replaced with 50 μL of DMEM with GPR52 antagonist for 15 min, washed, and replaced with 50 μL of fresh DMEM for 60 min. Then, another 50 μL of DMEM containing GPR52 agonist WO-459 (final 100 nM) and IBMX (500 μM) was added for further incubation of 30 min. Finally, the cells were lysed and the cAMP level was tested by a LANCE Ultra cAMP kit and EnVision multiplate reader.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i67" class="anchor-spacer"></div><h3 class="article-section__title" id="_i67"> HD Mice Models and Behavioral Experiments</h3><div class="NLM_p">HdhQ140 knock-in mice were generated and characterized as previously described.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> All mice were maintained in pathogen-free conditions. The Animal Care and Use Committee at Fudan University approved all animal experimental procedures.</div><div class="NLM_p last">All of the behavioral experiments were performed as previously reported with minor revision.<a onclick="showRef(event, 'ref7 ref20'); return false;" href="javascript:void(0);" class="ref ref7 ref20">(7,20)</a> For the rotarod test, mice were pretrained on the rotarod at 4 rpm for 5 min for 3 days. In the following 5-day test, the mice were placed on the speeding rotarod with an accelerating rate from 4 to 40 rpm within 4 min. The test was terminated until the mice fell from the rotarod or the latency exceeded 4 min, and the latency time for each mouse was recorded. For the balance beam test, the mice were pretrained to walk through the balance beam with 100 cm × 3 cm × 50 cm (<i>D</i> × <i>W</i> × <i>H</i>) for two consecutive days. In the following test, we recorded the total time for each mouse to walk through the balance beam. All of the experiments were repeated at least two times with an interval of 60 min and the means were recorded.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i68" class="anchor-spacer"></div><h3 class="article-section__title" id="_i68"> Compound Treatment in Animals</h3><div class="NLM_p">For intracerebroventricular delivery of Comp-<b>43</b>, the 9-month-old HdhQ140 mice and the age-, weight-, and sex-matching wild-type controls were randomly divided into vehicle and Comp-<b>43</b> groups. Intracerebroventricular injection was performed as previously described.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> Briefly, first, the mice were anesthetized and located in a stereotactic instrument. Then, a guide cannula with a cap was implanted into the left lateral ventricle (0.6 mm posterior, 1.5 mm lateral, and 1.7 mm dorsal to Bregma) of each mouse, followed by the postoperative care for 1 week. In the following 3 consecutive weeks injection, an internal injector with extended 0.5 mm was inserted into the guide cannula, allowing direct delivery of 2 μL of Comp-<b>43</b> at the concentration of 0.5 mM into ventricle at a rate of 0.25 μL/min (RWD Life Science) once a day. The final concentration of Comp-<b>43</b> in the brain was approximately 0.25–0.5 μM given that the total cerebroventricular volume is 50–100 μL and the total brain volume is about 3 mL in mice.</div><div class="NLM_p last">For intraperitoneal delivery of compound, the 9-month-old HdhQ140 mice and their matched wild-type controls were injected with 5 mg/kg vehicles or Comp-<b>43</b> diluted in phosphate-buffered saline (PBS) solution. The injection was performed once a day for a continuous 4 weeks, followed by further behavioral and bio-marker study.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i69" class="anchor-spacer"></div><h3 class="article-section__title" id="_i69"> Compound Detection in Mice Brains and Serum</h3><div class="NLM_p last">We randomly divided 4-week-old age-, weight-, and sex-matched C57/BL6 mice into groups treated with the vehicle or Comp-<b>43</b>. Then, intravenous tail vein injection was performed and the mice were anesthetized by chloral hydrate at indicated time points. The heart blood was collected, and the plasma was obtained after centrifugation at 10 000 rpm for 5 min. After perfusion with PBS, the brains were dissected and then homogenized. Further liquid chromatography–mass spectrometry (LC–MS)/MS analysis of Comp-<b>43</b> in heart plasma and brain tissue was performed by the SIM-Servier joint laboratory as reported previously.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a></div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i70" class="anchor-spacer"></div><h3 class="article-section__title" id="_i70"> Statistics</h3><div class="NLM_p last">The quantitative data are presented as mean ± S.E.M. and analyzed by GraphPad Prism 6. Unpaired Student’s <i>t</i>-test was used to compare two groups. One-way ANOVA followed by post hoc Bonferroni’s or Donnet’s multiple comparisons was used to compare more than two groups. Two-way ANOVA followed by post hoc Bonferroni’s multiple comparisons test was used for comparison of a series of data collection among groups.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i71"><a href="/doi/suppl/10.1021/acs.jmedchem.0c01133" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_02327" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_02327" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c01133?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01133</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Supporting figures, tables, and materials and methods; spectroscopic data of compounds (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01133/suppl_file/jm0c01133_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01133/suppl_file/jm0c01133_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01133/suppl_file/jm0c01133_si_001.pdf">jm0c01133_si_001.pdf (5.82 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01133/suppl_file/jm0c01133_si_002.csv">jm0c01133_si_002.csv (2.44 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.0c01133" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_08124" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_08124" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xin Xie</span> - <span class="hlFld-Affiliation affiliation">School
of Chinese Materia Medica, Nanjing University
of Chinese Medicine, Nanjing 210046, China</span>; 
    <span class="hlFld-Affiliation affiliation">CAS
Key Laboratory of Receptor Research, National Center for Drug Screening, Shanghai Institute of Materia Medica, Chinese Academy
of Sciences, Shanghai 201203, China</span>; 
    <span class="hlFld-Affiliation affiliation">School
of Life Science and Technology, ShanghaiTech
University, Shanghai 201210, China</span>; 
    <span class="hlFld-Affiliation affiliation">School of
Pharmaceutical Science and Technology, Hangzhou
Institute for Advanced Study, UCAS, Hangzhou 310024, China</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#e29a9a8b87a2918b8f8fcc8381cc818c"><span class="__cf_email__" data-cfemail="dca4a4b5b99cafb5b1b1f2bdbff2bfb2">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Boxun Lu</span> - <span class="hlFld-Affiliation affiliation">Neurology
Department at Huashan Hospital, State Key Laboratory of Medical Neurobiology
and MOE Frontiers Center for Brain Science, School of Life Sciences, Fudan University, Shanghai 200438, China</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#39554c5b56414c57795f4c5d5857175c5d4c175a57"><span class="__cf_email__" data-cfemail="1b776e7974636e755b7d6e7f7a75357e7f6e357875">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Youhong Hu</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Drug Research, Shanghai
Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</span>; 
    <span class="hlFld-Affiliation affiliation">School of
Pharmaceutical Science and Technology, Hangzhou
Institute for Advanced Study, UCAS, Hangzhou 310024, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-1770-6272" title="Orcid link">http://orcid.org/0000-0003-1770-6272</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#d2abbabaa792a1bbbfbffcb3b1fcb1bc"><span class="__cf_email__" data-cfemail="99e0f1f1ecd9eaf0f4f4b7f8fab7faf7">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Congcong Wang</span> - <span class="hlFld-Affiliation affiliation">Neurology
Department at Huashan Hospital, State Key Laboratory of Medical Neurobiology
and MOE Frontiers Center for Brain Science, School of Life Sciences, Fudan University, Shanghai 200438, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yu-Fang Zhang</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Drug Research, Shanghai
Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Shimeng Guo</span> - <span class="hlFld-Affiliation affiliation">School
of Chinese Materia Medica, Nanjing University
of Chinese Medicine, Nanjing 210046, China</span>; 
    <span class="hlFld-Affiliation affiliation">CAS
Key Laboratory of Receptor Research, National Center for Drug Screening, Shanghai Institute of Materia Medica, Chinese Academy
of Sciences, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Quan Zhao</span> - <span class="hlFld-Affiliation affiliation">Neurology
Department at Huashan Hospital, State Key Laboratory of Medical Neurobiology
and MOE Frontiers Center for Brain Science, School of Life Sciences, Fudan University, Shanghai 200438, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yanping Zeng</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Drug Research, Shanghai
Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Zhicheng Xie</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Drug Research, Shanghai
Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>C.W. designed and performed the functional study of shGPR52 and Comp-43 in cellular levels in vivo, analyzed and interpreted data, and drafted the manuscripts. Y.Z. designed and performed the synthesis and optimization of GPR52 antagonists, analyzed and interpreted data, and drafted the manuscripts. S.G. performed the potency evaluation of synthesized GPR52 antagonists. Q.Z. performed the functional study of Comp-43in vivo. Y.Z. performed the synthesis of Comp-43. Z.X. did the analysis of the molecular docking. B.L., Y.H., and X.X. led the project, designed the experiments, and interpreted the data. All authors contributed to the data analysis, manuscript preparation, and final approval of the version to be submitted.</p><span class="author-information-subsection-header">Author Contributions</span><p>C.W., Y.Z., and S.G. contributed equally and should be considered as co-first authors.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e4773-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i74">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_04146" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_04146" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The authors are grateful to Drs. Gillian Bate and Marian Difiglia for providing them with the S830 antibody and Ab1 antibody, respectively. They appreciate Dr. Chen Ling and Jingwei Chen for the instruction and preparation of AAV expressing shRNAs. They thank the staff members of the Large-scale Protein Preparation System at the National Facility for Protein Science in Shanghai (NFPS), Zhangjiang Lab, China, for providing technical support and assistance in data collection and analysis. They also thank all members of the lab for sharing reagents and advice. This research was supported by National Natural Science Foundation of China grants (81925012, 81870990, 31961130379), the Fudan-SIMM Joint Research Fund (FU-SIMM 20174013), and China Postdoctoral Science Foundation (2018M632010).</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">GPCR</td><td class="NLM_def"><p class="first last">G protein-coupled receptor</p></td></tr><tr><td class="NLM_term">HD</td><td class="NLM_def"><p class="first last">Huntington’s disease</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr><tr><td class="NLM_term">mHTT</td><td class="NLM_def"><p class="first last">mutant huntingtin</p></td></tr><tr><td class="NLM_term">polyQ</td><td class="NLM_def"><p class="first last">polyglutamine tract</p></td></tr><tr><td class="NLM_term">siRNA</td><td class="NLM_def"><p class="first last">small interference RNAs</p></td></tr><tr><td class="NLM_term">shRNAs</td><td class="NLM_def"><p class="first last">short hairpin RNAs</p></td></tr><tr><td class="NLM_term">ASO</td><td class="NLM_def"><p class="first last">antisense oligonucleotides</p></td></tr><tr><td class="NLM_term">wtHTT</td><td class="NLM_def"><p class="first last">wild-type huntingtin</p></td></tr><tr><td class="NLM_term">AAV</td><td class="NLM_def"><p class="first last">adeno-associated viruses</p></td></tr><tr><td class="NLM_term">sh<i>Gpr</i>52</td><td class="NLM_def"><p class="first last">shRNAs targeting <i>Gpr</i>52</p></td></tr><tr><td class="NLM_term">shNT</td><td class="NLM_def"><p class="first last">nontargeting shRNA control</p></td></tr><tr><td class="NLM_term">TR-FRET</td><td class="NLM_def"><p class="first last">time-resolved fluorescence resonance energy transfer</p></td></tr><tr><td class="NLM_term">HTRF</td><td class="NLM_def"><p class="first last">homogeneous time-resolved fluorescence</p></td></tr><tr><td class="NLM_term">WB</td><td class="NLM_def"><p class="first last">western blot</p></td></tr><tr><td class="NLM_term">ICV</td><td class="NLM_def"><p class="first last">intracerebroventricular</p></td></tr><tr><td class="NLM_term">IC<sub>50</sub></td><td class="NLM_def"><p class="first last">half-maximal inhibitory concentration</p></td></tr><tr><td class="NLM_term">cAMP</td><td class="NLM_def"><p class="first last">cyclic adenosine monophosphate</p></td></tr><tr><td class="NLM_term">PAINS</td><td class="NLM_def"><p class="first last">pan assay interference compounds</p></td></tr><tr><td class="NLM_term">β2AR</td><td class="NLM_def"><p class="first last">β-2 adrenergic receptor</p></td></tr><tr><td class="NLM_term">GCGR</td><td class="NLM_def"><p class="first last">glucagon receptor</p></td></tr><tr><td class="NLM_term">GLP-1R</td><td class="NLM_def"><p class="first last">glucagon-like peptide 1 receptor</p></td></tr><tr><td class="NLM_term">CCR5</td><td class="NLM_def"><p class="first last">C–C motif chemokine receptor 5</p></td></tr><tr><td class="NLM_term">α1DAR</td><td class="NLM_def"><p class="first last">α-1D adrenergic receptor</p></td></tr><tr><td class="NLM_term">DRD2</td><td class="NLM_def"><p class="first last">dopamine D2 receptor</p></td></tr><tr><td class="NLM_term">siGPR52</td><td class="NLM_def"><p class="first last">siRNA targeting GPR52</p></td></tr><tr><td class="NLM_term">siNC</td><td class="NLM_def"><p class="first last">nontargeting siRNA control</p></td></tr><tr><td class="NLM_term">DMSO</td><td class="NLM_def"><p class="first last">dimethylsulfoxide</p></td></tr><tr><td class="NLM_term">EA</td><td class="NLM_def"><p class="first last">ethyl acetate</p></td></tr><tr><td class="NLM_term">PE</td><td class="NLM_def"><p class="first last">petroleum ether</p></td></tr><tr><td class="NLM_term">THF</td><td class="NLM_def"><p class="first last">tetrahydrofuran</p></td></tr><tr><td class="NLM_term">DMF</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N</i>-dimethylformamide</p></td></tr><tr><td class="NLM_term">HWE</td><td class="NLM_def"><p class="first last">Horner–Wadsworth–Emmons Reaction</p></td></tr><tr><td class="NLM_term">TLC</td><td class="NLM_def"><p class="first last">thin-layer chromatography</p></td></tr><tr><td class="NLM_term">DIPEA</td><td class="NLM_def"><p class="first last">diisopropylethylamine</p></td></tr><tr><td class="NLM_term">DMAP</td><td class="NLM_def"><p class="first last"><i>N,N</i>-4-dimethylaminopyridine</p></td></tr><tr><td class="NLM_term">EDCI</td><td class="NLM_def"><p class="first last">1-ethyl-3-(3-dimethylaminopropyl)carbodiimide hydrochloride</p></td></tr><tr><td class="NLM_term">HOBt</td><td class="NLM_def"><p class="first last">1-hydroxybenzotriazoleA novel gene containing a trinucleotide repeat that is expanded and unstable on huntington’s disease chromosomes</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i76">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_71350" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_71350" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 46 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">The
Huntington’s Disease Collaborative Research Group</span> <span> </span><span class="NLM_article-title">A novel gene containing a trinucleotide
repeat that is expanded and unstable on Huntington’s disease
chromosomes</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>72</i></span>,  <span class="NLM_fpage">971</span>– <span class="NLM_lpage">983</span>, <span class="refDoi"> DOI: 10.1016/0092-8674(93)90585-E</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1016%2F0092-8674%2893%2990585-E" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=8458085" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=1993&pages=971-983&author=The%0AHuntington%E2%80%99s+Disease+Collaborative+Research+Group&title=A+novel+gene+containing+a+trinucleotide%0Arepeat+that+is+expanded+and+unstable+on+Huntington%E2%80%99s+disease%0Achromosomes&doi=10.1016%2F0092-8674%2893%2990585-E"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2F0092-8674%2893%2990585-E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0092-8674%252893%252990585-E%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DA%2520novel%2520gene%2520containing%2520a%2520trinucleotide%250Arepeat%2520that%2520is%2520expanded%2520and%2520unstable%2520on%2520Huntington%25E2%2580%2599s%2520disease%250Achromosomes%26jtitle%3DCell%26date%3D1993%26volume%3D72%26spage%3D971%26epage%3D983%26doi%3D10.1016%2F0092-8674%2893%2990585-E" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Richfield, E. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maguire-Zeiss, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vonkeman, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voorn, P.</span></span> <span> </span><span class="NLM_article-title">Preferential loss of preproenkephalin versus preprotachykinin neurons from the striatum of Huntington’s disease patients</span>. <i>Ann. Neurol.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">852</span>– <span class="NLM_lpage">861</span>, <span class="refDoi"> DOI: 10.1002/ana.410380605</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1002%2Fana.410380605" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=8526457" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A280%3ADyaK287gtFekug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1995&pages=852-861&author=E.+K.+Richfieldauthor=K.+A.+Maguire-Zeissauthor=H.+E.+Vonkemanauthor=P.+Voorn&title=Preferential+loss+of+preproenkephalin+versus+preprotachykinin+neurons+from+the+striatum+of+Huntington%E2%80%99s+disease+patients&doi=10.1002%2Fana.410380605"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Preferential loss of preproenkephalin versus preprotachykinin neurons from the striatum of Huntington's disease patients</span></div><div class="casAuthors">Richfield E K; Maguire-Zeiss K A; Vonkeman H E; Voorn P</div><div class="citationInfo"><span class="NLM_cas:title">Annals of neurology</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">852-61</span>
        ISSN:<span class="NLM_cas:issn">0364-5134</span>.
    </div><div class="casAbstract">Preferential loss of basal ganglia neurons and terminals occurs in Huntington's disease (HD).  Terminals of preproenkephalin medium-size spiny neurons are more vulnerable than terminals of preprotachykinin neurons, but the peptidergic neurons of origin have not yet been shown to die preferentially.  We sought to determine, in the striatum, whether preproenkephalin neurons were lost to a greater extent than preprotachykinin neurons and to determine whether there were decreases in specific messenger RNA (mRNA) levels of preproenkephalin, preprotachykinin, and calbindin D28k.  We found a grade-related decrease in the number of preprotachykinin- and calbindin D28k-labeled neurons per measuring field in the caudate nucleus of patients with HD.  Three measures of the neuronal level of preprotachykinin mRNA were all significantly reduced (6-65% of control values) in HD caudate nucleus.  No decline in calbindin D28k mRNA levels per neuron were found in HD striata compared to control striata.  We found a greater loss of preproenkephalin neurons per field than preprotachyknin neurons per field in the caudate nucleus of HD brains compared to control brains.  Preprotachykinin neurons are lost in HD in a grade-related manner and surviving preprotachykinin neurons are impaired in function.  However, preproenkephalin neurons are lost to a greater extent than preprotachykinin neurons, which may explain preferential changes found in projection regions of the striatum.  Declines in neuropeptide mRNA may be specific in HD, since calbindin D28k mRNA levels were unchanged.  Alterations in the levels of expression of preproenkephalin and preprotachykinin mRNA may be direct or indirect effects of the HD mutation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ4iCgcs9PWF3l8icjszYl0fW6udTcc2eZtQNgYlRfWdLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK287gtFekug%253D%253D&md5=01150e69fbb4fd2a415eb7d6bfe526d2</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1002%2Fana.410380605&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fana.410380605%26sid%3Dliteratum%253Aachs%26aulast%3DRichfield%26aufirst%3DE.%2BK.%26aulast%3DMaguire-Zeiss%26aufirst%3DK.%2BA.%26aulast%3DVonkeman%26aufirst%3DH.%2BE.%26aulast%3DVoorn%26aufirst%3DP.%26atitle%3DPreferential%2520loss%2520of%2520preproenkephalin%2520versus%2520preprotachykinin%2520neurons%2520from%2520the%2520striatum%2520of%2520Huntington%25E2%2580%2599s%2520disease%2520patients%26jtitle%3DAnn.%2520Neurol.%26date%3D1995%26volume%3D38%26spage%3D852%26epage%3D861%26doi%3D10.1002%2Fana.410380605" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">DiFiglia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sapp, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chase, K. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bates, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vonsattel, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aronin, N.</span></span> <span> </span><span class="NLM_article-title">Aggregation of huntingtin in neuronal intranuclear inclusions and dystrophic neurites in brain</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>277</i></span>,  <span class="NLM_fpage">1990</span>– <span class="NLM_lpage">1993</span>, <span class="refDoi"> DOI: 10.1126/science.277.5334.1990</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1126%2Fscience.277.5334.1990" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=9302293" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A528%3ADyaK2sXmt12rurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=277&publication_year=1997&pages=1990-1993&author=M.+DiFigliaauthor=E.+Sappauthor=K.+O.+Chaseauthor=S.+W.+Daviesauthor=G.+P.+Batesauthor=J.+P.+Vonsattelauthor=N.+Aronin&title=Aggregation+of+huntingtin+in+neuronal+intranuclear+inclusions+and+dystrophic+neurites+in+brain&doi=10.1126%2Fscience.277.5334.1990"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Aggregation of huntingtin in neuronal intranuclear inclusions and dystrophic neurites in brain</span></div><div class="casAuthors">DiFiglia, Marian; Sapp, Ellen; Chase, Kathryn O.; Davies, Stephen W.; Bates, Gillian; Vonsattel, J. P.; Aronin, Neil</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">277</span>
        (<span class="NLM_cas:issue">5334</span>),
    <span class="NLM_cas:pages">1990-1993</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The cause of neurodegeneration in Huntington's disease (HD) is unknown.  Patients with HD have an expanded NH2-terminal polyglutamine region in huntingtin.  An NH2-terminal fragment of mutant huntingtin was localized to neuronal intranuclear inclusions (NIIs) and dystrophic neurites (DNs) in the HD cortex and striatum, which are affected in HD, and polyglutamine length influenced the extent of huntingtin accumulation in these structures.  Ubiquitin was also found in NIIs and DNs, which suggests that abnormal huntingtin is targeted for proteolysis but is resistant to removal.  The aggregation of mutant huntingtin may be part of the pathogenic mechanism in HD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3fQHwJi2eAbVg90H21EOLACvtfcHk0lgOnBPZCjDrXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXmt12rurs%253D&md5=fc9fa3c298625e623b3df0fe144ab2ba</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1126%2Fscience.277.5334.1990&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.277.5334.1990%26sid%3Dliteratum%253Aachs%26aulast%3DDiFiglia%26aufirst%3DM.%26aulast%3DSapp%26aufirst%3DE.%26aulast%3DChase%26aufirst%3DK.%2BO.%26aulast%3DDavies%26aufirst%3DS.%2BW.%26aulast%3DBates%26aufirst%3DG.%2BP.%26aulast%3DVonsattel%26aufirst%3DJ.%2BP.%26aulast%3DAronin%26aufirst%3DN.%26atitle%3DAggregation%2520of%2520huntingtin%2520in%2520neuronal%2520intranuclear%2520inclusions%2520and%2520dystrophic%2520neurites%2520in%2520brain%26jtitle%3DScience%26date%3D1997%26volume%3D277%26spage%3D1990%26epage%3D1993%26doi%3D10.1126%2Fscience.277.5334.1990" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miller, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrasate, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brooks, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Libeu, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Legleiter, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatters, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curtis, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krishnan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitra, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Widjaja, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaby, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lotz, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Newhouse, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osmand, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thulasiramin, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saudou, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Segal, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masliah, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muchowski, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weisgraber, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finkbeiner, S.</span></span> <span> </span><span class="NLM_article-title">Identifying polyglutamine protein species in situ that best predict neurodegeneration</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">925</span>– <span class="NLM_lpage">934</span>, <span class="refDoi"> DOI: 10.1038/nchembio.694</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1038%2Fnchembio.694" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=22037470" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlyisrfE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=925-934&author=J.+Millerauthor=M.+Arrasateauthor=E.+Brooksauthor=C.+P.+Libeuauthor=J.+Legleiterauthor=D.+Hattersauthor=J.+Curtisauthor=K.+Cheungauthor=P.+Krishnanauthor=S.+Mitraauthor=K.+Widjajaauthor=B.+A.+Shabyauthor=G.+P.+Lotzauthor=Y.+Newhouseauthor=E.+J.+Mitchellauthor=A.+Osmandauthor=M.+Grayauthor=V.+Thulasiraminauthor=F.+Saudouauthor=M.+Segalauthor=X.+W.+Yangauthor=E.+Masliahauthor=L.+M.+Thompsonauthor=P.+J.+Muchowskiauthor=K.+H.+Weisgraberauthor=S.+Finkbeiner&title=Identifying+polyglutamine+protein+species+in+situ+that+best+predict+neurodegeneration&doi=10.1038%2Fnchembio.694"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Identifying polyglutamine protein species in situ that best predict neurodegeneration</span></div><div class="casAuthors">Miller, Jason; Arrasate, Montserrat; Brooks, Elizabeth; Libeu, Clare Peters; Legleiter, Justin; Hatters, Danny; Curtis, Jessica; Cheung, Kenneth; Krishnan, Preethi; Mitra, Siddhartha; Widjaja, Kartika; Shaby, Benjamin A.; Lotz, Gregor P.; Newhouse, Yvonne; Mitchell, Emily J.; Osmand, Alex; Gray, Michelle; Thulasiramin, Vanitha; Saudou, Frederic; Segal, Mark; Yang, X. William; Masliah, Eliezer; Thompson, Leslie M.; Muchowski, Paul J.; Weisgraber, Karl H.; Finkbeiner, Steven</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">925-934</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Polyglutamine (polyQ) stretches exceeding a threshold length confer a toxic function to proteins that contain them and cause at least nine neurol. disorders.  The basis for this toxicity threshold is unclear.  Although polyQ expansions render proteins prone to aggregate into inclusion bodies, this may be a neuronal coping response to more toxic forms of polyQ.  The exact structure of these more toxic forms is unknown.  Here the authors show that the monoclonal antibody 3B5H10 recognizes a species of polyQ protein in situ that strongly predicts neuronal death.  The epitope selectively appears among some of the many low-mol.-wt. conformational states assumed by expanded polyQ and disappears in higher-mol.-wt. aggregated forms, such as inclusion bodies.  These results suggest that protein monomers and possibly small oligomers contg. expanded polyQ stretches can adopt a conformation that is recognized by 3B5H10 and is toxic or closely related to a toxic species.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowzCQPLhzGSrVg90H21EOLACvtfcHk0lgOnBPZCjDrXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlyisrfE&md5=a0d31dabd25c57c5c2fa5aaf294ba555</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.694&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.694%26sid%3Dliteratum%253Aachs%26aulast%3DMiller%26aufirst%3DJ.%26aulast%3DArrasate%26aufirst%3DM.%26aulast%3DBrooks%26aufirst%3DE.%26aulast%3DLibeu%26aufirst%3DC.%2BP.%26aulast%3DLegleiter%26aufirst%3DJ.%26aulast%3DHatters%26aufirst%3DD.%26aulast%3DCurtis%26aufirst%3DJ.%26aulast%3DCheung%26aufirst%3DK.%26aulast%3DKrishnan%26aufirst%3DP.%26aulast%3DMitra%26aufirst%3DS.%26aulast%3DWidjaja%26aufirst%3DK.%26aulast%3DShaby%26aufirst%3DB.%2BA.%26aulast%3DLotz%26aufirst%3DG.%2BP.%26aulast%3DNewhouse%26aufirst%3DY.%26aulast%3DMitchell%26aufirst%3DE.%2BJ.%26aulast%3DOsmand%26aufirst%3DA.%26aulast%3DGray%26aufirst%3DM.%26aulast%3DThulasiramin%26aufirst%3DV.%26aulast%3DSaudou%26aufirst%3DF.%26aulast%3DSegal%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DX.%2BW.%26aulast%3DMasliah%26aufirst%3DE.%26aulast%3DThompson%26aufirst%3DL.%2BM.%26aulast%3DMuchowski%26aufirst%3DP.%2BJ.%26aulast%3DWeisgraber%26aufirst%3DK.%2BH.%26aulast%3DFinkbeiner%26aufirst%3DS.%26atitle%3DIdentifying%2520polyglutamine%2520protein%2520species%2520in%2520situ%2520that%2520best%2520predict%2520neurodegeneration%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2011%26volume%3D7%26spage%3D925%26epage%3D934%26doi%3D10.1038%2Fnchembio.694" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucas, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hen, R.</span></span> <span> </span><span class="NLM_article-title">Reversal of neuropathology and motor dysfunction in a conditional model of Huntington’s disease</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>101</i></span>,  <span class="NLM_fpage">57</span>– <span class="NLM_lpage">66</span>, <span class="refDoi"> DOI: 10.1016/S0092-8674(00)80623-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1016%2FS0092-8674%2800%2980623-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10778856" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A528%3ADC%252BD3cXisVaitbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2000&pages=57-66&author=A.+Yamamotoauthor=J.+J.+Lucasauthor=R.+Hen&title=Reversal+of+neuropathology+and+motor+dysfunction+in+a+conditional+model+of+Huntington%E2%80%99s+disease&doi=10.1016%2FS0092-8674%2800%2980623-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Reversal of neuropathology and motor dysfunction in a conditional model of Huntington's disease</span></div><div class="casAuthors">Yamamoto, Ai; Lucas, Jose J.; Hen, Rene</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, Massachusetts)</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">101</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">57-66</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Neurodegenerative disorders like Huntington's disease (HD) are characterized by progressive and putative irreversible clin. and neuropathol. symptoms, including neuronal protein aggregates.  Conditional transgenic models of neurodegenerative diseases therefore could be a powerful means to explore the relationship between mutant protein expression and progression of the disease.  We have created a conditional model of HD by using the tet-regulatable system.  Mice expressing a mutated huntingtin fragment demonstrate neuronal inclusions, characteristic neuropathol., and progressive motor dysfunction.  Blockade of expression in symptomatic mice leads to a disappearance of inclusions and an amelioration of the behavioral phenotype.  We thus demonstrate that a continuous influx of the mutant protein is required to maintain inclusions and symptoms, raising the possibility that HD may be reversible.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpABup9-4sWj7Vg90H21EOLACvtfcHk0lgOnBPZCjDrXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXisVaitbk%253D&md5=589e78e8d13075710115e67df6e5ac17</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2FS0092-8674%2800%2980623-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0092-8674%252800%252980623-6%26sid%3Dliteratum%253Aachs%26aulast%3DYamamoto%26aufirst%3DA.%26aulast%3DLucas%26aufirst%3DJ.%2BJ.%26aulast%3DHen%26aufirst%3DR.%26atitle%3DReversal%2520of%2520neuropathology%2520and%2520motor%2520dysfunction%2520in%2520a%2520conditional%2520model%2520of%2520Huntington%25E2%2580%2599s%2520disease%26jtitle%3DCell%26date%3D2000%26volume%3D101%26spage%3D57%26epage%3D66%26doi%3D10.1016%2FS0092-8674%2800%2980623-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantle, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holley, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greiner, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shirasaki, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cepeda, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levine, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X. W.</span></span> <span> </span><span class="NLM_article-title">Neuronal targets for reducing mutant huntingtin expression to ameliorate disease in a mouse model of Huntington’s disease</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">536</span>– <span class="NLM_lpage">541</span>, <span class="refDoi"> DOI: 10.1038/nm.3514</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1038%2Fnm.3514" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=24784230" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A528%3ADC%252BC2cXntlehs78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2014&pages=536-541&author=N.+Wangauthor=M.+Grayauthor=X.+H.+Luauthor=J.+P.+Cantleauthor=S.+M.+Holleyauthor=E.+Greinerauthor=X.+Guauthor=D.+Shirasakiauthor=C.+Cepedaauthor=Y.+Liauthor=H.+Dongauthor=M.+S.+Levineauthor=X.+W.+Yang&title=Neuronal+targets+for+reducing+mutant+huntingtin+expression+to+ameliorate+disease+in+a+mouse+model+of+Huntington%E2%80%99s+disease&doi=10.1038%2Fnm.3514"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Neuronal targets for reducing mutant huntingtin expression to ameliorate disease in a mouse model of Huntington's disease</span></div><div class="casAuthors">Wang, Nan; Gray, Michelle; Lu, Xiao-Hong; Cantle, Jeffrey P.; Holley, Sandra M.; Greiner, Erin; Gu, Xiaofeng; Shirasaki, Dyna; Cepeda, Carlos; Li, Yuqing; Dong, Hongwei; Levine, Michael S.; Yang, X. William</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">536-541</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Huntington's disease (HD) is a fatal dominantly inherited neurodegenerative disorder caused by a CAG repeat expansion leading to an elongated polyglutamine stretch in huntingtin.  Mutant huntingtin (mHTT) is ubiquitously expressed in all cells but elicits selective cortical and striatal neurodegeneration in HD.  The mechanistic basis for such selective neuronal vulnerability remains unclear.  A necessary step toward resolving this enigma is to define the cell types in which mHTT expression is causally linked to the disease pathogenesis.  Using a conditional transgenic mouse model of HD, in which the mice express full-length human mHTT from a bacterial artificial chromosome transgene (BACHD), we genetically reduced mHTT expression in neuronal populations in the striatum, cortex or both.  We show that redn. of cortical mHTT expression in BACHD mice partially improves motor and psychiatric-like behavioral deficits but does not improve neurodegeneration, whereas redn. of mHTT expression in both neuronal populations consistently ameliorates all behavioral deficits and selective brain atrophy in this HD model.  Furthermore, whereas redn. of mHTT expression in cortical or striatal neurons partially ameliorates corticostriatal synaptic deficits, further restoration of striatal synaptic function can be achieved by redn. of mHTT expression in both neuronal cell types.  Our study demonstrates distinct but interacting roles of cortical and striatal mHTT in HD pathogenesis and suggests that optimal HD therapeutics may require targeting mHTT in both cortical and striatal neurons.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOHZ5lk0IQqrVg90H21EOLACvtfcHk0lhbPOLU6AcPYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXntlehs78%253D&md5=1fbb71d02f95360589804425aabe04c0</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnm.3514&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.3514%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DN.%26aulast%3DGray%26aufirst%3DM.%26aulast%3DLu%26aufirst%3DX.%2BH.%26aulast%3DCantle%26aufirst%3DJ.%2BP.%26aulast%3DHolley%26aufirst%3DS.%2BM.%26aulast%3DGreiner%26aufirst%3DE.%26aulast%3DGu%26aufirst%3DX.%26aulast%3DShirasaki%26aufirst%3DD.%26aulast%3DCepeda%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DLevine%26aufirst%3DM.%2BS.%26aulast%3DYang%26aufirst%3DX.%2BW.%26atitle%3DNeuronal%2520targets%2520for%2520reducing%2520mutant%2520huntingtin%2520expression%2520to%2520ameliorate%2520disease%2520in%2520a%2520mouse%2520model%2520of%2520Huntington%25E2%2580%2599s%2520disease%26jtitle%3DNat.%2520Med.%26date%3D2014%26volume%3D20%26spage%3D536%26epage%3D541%26doi%3D10.1038%2Fnm.3514" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sha, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Difiglia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mei, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fei, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, B.</span></span> <span> </span><span class="NLM_article-title">Allele-selective lowering of mutant HTT protein by HTT-LC3 linker compounds</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>575</i></span>,  <span class="NLM_fpage">203</span>– <span class="NLM_lpage">209</span>, <span class="refDoi"> DOI: 10.1038/s41586-019-1722-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1038%2Fs41586-019-1722-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=31666698" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitV2hsb3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=575&publication_year=2019&pages=203-209&author=Z.+Liauthor=C.+Wangauthor=Z.+Wangauthor=C.+Zhuauthor=J.+Liauthor=T.+Shaauthor=L.+Maauthor=C.+Gaoauthor=Y.+Yangauthor=Y.+Sunauthor=J.+Wangauthor=X.+Sunauthor=C.+Luauthor=M.+Difigliaauthor=Y.+Meiauthor=C.+Dingauthor=S.+Luoauthor=Y.+Dangauthor=Y.+Dingauthor=Y.+Feiauthor=B.+Lu&title=Allele-selective+lowering+of+mutant+HTT+protein+by+HTT-LC3+linker+compounds&doi=10.1038%2Fs41586-019-1722-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Allele-selective lowering of mutant HTT protein by HTT-LC3 linker compounds</span></div><div class="casAuthors">Li, Zhaoyang; Wang, Cen; Wang, Ziying; Zhu, Chenggang; Li, Jie; Sha, Tian; Ma, Lixiang; Gao, Chao; Yang, Yi; Sun, Yimin; Wang, Jian; Sun, Xiaoli; Lu, Chenqi; Difiglia, Marian; Mei, Yanai; Ding, Chen; Luo, Shouqing; Dang, Yongjun; Ding, Yu; Fei, Yiyan; Lu, Boxun</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">575</span>
        (<span class="NLM_cas:issue">7781</span>),
    <span class="NLM_cas:pages">203-209</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Accumulation of mutant proteins is a major cause of many diseases (collectively called proteopathies), and lowering the level of these proteins can be useful for treatment of these diseases.  We hypothesized that compds. that interact with both the autophagosome protein microtubule-assocd. protein 1A/1B light chain 3 (LC3)1 and the disease-causing protein may target the latter for autophagic clearance.  Mutant huntingtin protein (mHTT) contains an expanded polyglutamine (polyQ) tract and causes Huntington's disease, an incurable neurodegenerative disorder2.  Here, using small-mol.-microarray-based screening, we identified four compds. that interact with both LC3 and mHTT, but not with the wild-type HTT protein.  Some of these compds. targeted mHTT to autophagosomes, reduced mHTT levels in an allele-selective manner, and rescued disease-relevant phenotypes in cells and in vivo in fly and mouse models of Huntington's disease.  We further show that these compds. interact with the expanded polyQ stretch and could lower the level of mutant ataxin-3 (ATXN3), another disease-causing protein with an expanded polyQ tract3.  This study presents candidate compds. for lowering mHTT and potentially other disease-causing proteins with polyQ expansions, demonstrating the concept of lowering levels of disease-causing proteins using autophagosome-tethering compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7rSfxy_Wl7rVg90H21EOLACvtfcHk0lhbPOLU6AcPYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitV2hsb3O&md5=98cba22501f8157c83c0181b74ca15e2</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fs41586-019-1722-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-019-1722-1%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DZhu%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DSha%26aufirst%3DT.%26aulast%3DMa%26aufirst%3DL.%26aulast%3DGao%26aufirst%3DC.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DSun%26aufirst%3DX.%26aulast%3DLu%26aufirst%3DC.%26aulast%3DDifiglia%26aufirst%3DM.%26aulast%3DMei%26aufirst%3DY.%26aulast%3DDing%26aufirst%3DC.%26aulast%3DLuo%26aufirst%3DS.%26aulast%3DDang%26aufirst%3DY.%26aulast%3DDing%26aufirst%3DY.%26aulast%3DFei%26aufirst%3DY.%26aulast%3DLu%26aufirst%3DB.%26atitle%3DAllele-selective%2520lowering%2520of%2520mutant%2520HTT%2520protein%2520by%2520HTT-LC3%2520linker%2520compounds%26jtitle%3DNature%26date%3D2019%26volume%3D575%26spage%3D203%26epage%3D209%26doi%3D10.1038%2Fs41586-019-1722-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Ramahi, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valencia, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berenshteyn, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallego-Flores, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ichcho, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lacoste, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hild, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Difiglia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Botas, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palacino, J.</span></span> <span> </span><span class="NLM_article-title">Identification of NUB1 as a suppressor of mutant Huntington toxicity via enhanced protein clearance</span>. <i>Nat. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">562</span>– <span class="NLM_lpage">570</span>, <span class="refDoi"> DOI: 10.1038/nn.3367</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1038%2Fnn.3367" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=23525043" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A528%3ADC%252BC3sXksVOjsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2013&pages=562-570&author=B.+Luauthor=I.+Al-Ramahiauthor=A.+Valenciaauthor=Q.+Wangauthor=F.+Berenshteynauthor=H.+Yangauthor=T.+Gallego-Floresauthor=S.+Ichchoauthor=A.+Lacosteauthor=M.+Hildauthor=M.+Difigliaauthor=J.+Botasauthor=J.+Palacino&title=Identification+of+NUB1+as+a+suppressor+of+mutant+Huntington+toxicity+via+enhanced+protein+clearance&doi=10.1038%2Fnn.3367"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of NUB1 as a suppressor of mutant Huntingtin toxicity via enhanced protein clearance</span></div><div class="casAuthors">Lu, Boxun; Al-Ramahi, Ismael; Valencia, Antonio; Wang, Qiong; Berenshteyn, Frada; Yang, Haidi; Gallego-Flores, Tatiana; Ichcho, Salah; Lacoste, Arnaud; Hild, Marc; DiFiglia, Marian; Botas, Juan; Palacino, James</div><div class="citationInfo"><span class="NLM_cas:title">Nature Neuroscience</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">562-570</span>CODEN:
                <span class="NLM_cas:coden">NANEFN</span>;
        ISSN:<span class="NLM_cas:issn">1097-6256</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Huntington's disease is caused by expanded CAG repeats in HTT, conferring toxic gain of function on mutant HTT (mHTT) protein.  Reducing mHTT amts. is postulated as a strategy for therapeutic intervention.  We conducted genome-wide RNA interference screens for genes modifying mHTT abundance and identified 13 hits.  We tested 10 in vivo in a Drosophila melanogaster Huntington's disease model, and 6 exhibited activity consistent with the in vitro screening results.  Among these, neg. regulator of ubiquitin-like protein 1 (NUB1) overexpression lowered mHTT in neuronal models and rescued mHTT-induced death.  NUB1 reduces mHTT amts. by enhancing polyubiquitination and proteasomal degrdn. of mHTT protein.  The process requires CUL3 and the ubiquitin-like protein NEDD8 necessary for CUL3 activation.  As a potential approach to modulating NUB1 for treatment, interferon-β lowered mHTT and rescued neuronal toxicity through induction of NUB1.  Thus, we have identified genes modifying endogenous mHTT using high-throughput screening and demonstrate NUB1 as an exemplar entry point for therapeutic intervention of Huntington's disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsSr4IEp1EWLVg90H21EOLACvtfcHk0lhbPOLU6AcPYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXksVOjsb4%253D&md5=fd90b314dab577d18223753b40504eca</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnn.3367&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnn.3367%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DB.%26aulast%3DAl-Ramahi%26aufirst%3DI.%26aulast%3DValencia%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DBerenshteyn%26aufirst%3DF.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DGallego-Flores%26aufirst%3DT.%26aulast%3DIchcho%26aufirst%3DS.%26aulast%3DLacoste%26aufirst%3DA.%26aulast%3DHild%26aufirst%3DM.%26aulast%3DDifiglia%26aufirst%3DM.%26aulast%3DBotas%26aufirst%3DJ.%26aulast%3DPalacino%26aufirst%3DJ.%26atitle%3DIdentification%2520of%2520NUB1%2520as%2520a%2520suppressor%2520of%2520mutant%2520Huntington%2520toxicity%2520via%2520enhanced%2520protein%2520clearance%26jtitle%3DNat.%2520Neurosci.%26date%3D2013%26volume%3D16%26spage%3D562%26epage%3D570%26doi%3D10.1038%2Fnn.3367" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abramowski, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bibel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bodner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chopra, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DiFiglia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fox, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kegel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grueninger, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hersch, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Housman, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Regulier, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosas, H. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stefani, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeitlin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bilbe, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paganetti, P.</span></span> <span> </span><span class="NLM_article-title">Single-step detection of mutant huntingtin in animal and human tissues: a bioassay for Huntington’s disease</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>395</i></span>,  <span class="NLM_fpage">8</span>– <span class="NLM_lpage">15</span>, <span class="refDoi"> DOI: 10.1016/j.ab.2009.08.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1016%2Fj.ab.2009.08.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=19664996" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFKnsb7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=395&publication_year=2009&pages=8-15&author=A.+Weissauthor=D.+Abramowskiauthor=M.+Bibelauthor=R.+Bodnerauthor=V.+Chopraauthor=M.+DiFigliaauthor=J.+Foxauthor=K.+Kegelauthor=C.+Kleinauthor=S.+Grueningerauthor=S.+Herschauthor=D.+Housmanauthor=E.+Regulierauthor=H.+D.+Rosasauthor=M.+Stefaniauthor=S.+Zeitlinauthor=G.+Bilbeauthor=P.+Paganetti&title=Single-step+detection+of+mutant+huntingtin+in+animal+and+human+tissues%3A+a+bioassay+for+Huntington%E2%80%99s+disease&doi=10.1016%2Fj.ab.2009.08.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Single-step detection of mutant huntingtin in animal and human tissues: A bioassay for Huntington's disease</span></div><div class="casAuthors">Weiss, Andreas; Abramowski, Dorothee; Bibel, Miriam; Bodner, Ruth; Chopra, Vanita; DiFiglia, Marian; Fox, Jonathan; Kegel, Kimberly; Klein, Corinna; Grueninger, Stephan; Hersch, Steven; Housman, David; Regulier, Etienne; Rosas, H. Diana; Stefani, Muriel; Zeitlin, Scott; Bilbe, Graeme; Paganetti, Paolo</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">395</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">8-15</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The genetic mutation causing Huntington's disease is a polyglutamine expansion in the huntingtin protein where more than 37 glutamines cause disease by formation of toxic intracellular fragments, aggregates, and cell death.  Despite a clear pathogenic role for mutant huntingtin, understanding huntingtin expression during the presymptomatic phase of the disease or during disease progression has remained obscure.  Central to clarifying the role in the pathomechanism of disease is the ability to easily and accurately measure mutant huntingtin in accessible human tissue samples as well as cell and animal models.  Here the authors describe a highly sensitive time-resolved Forster resonance energy transfer (FRET) assay for quantification of sol. mutant huntingtin in brain, plasma, and cerebrospinal fluid.  Surprisingly, in mice, sol. huntingtin levels decrease during disease progression, inversely correlating with brain aggregate load.  Mutant huntingtin is easily detected in human brain and blood-derived fractions, providing a utility to assess mutant huntingtin expression during disease course as well as a pharmacodynamic marker for disease-modifying therapeutics targeting expression, cleavage, or degrdn. of mutant huntingtin.  The design of the homogeneous one-step method for huntingtin detection is such that it can be easily applied to measure other proteins of interest.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrla9646FajzrVg90H21EOLACvtfcHk0lidRuQQSu4zHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFKnsb7I&md5=2fe224880e4761ec0566994b18353c8c</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.ab.2009.08.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ab.2009.08.001%26sid%3Dliteratum%253Aachs%26aulast%3DWeiss%26aufirst%3DA.%26aulast%3DAbramowski%26aufirst%3DD.%26aulast%3DBibel%26aufirst%3DM.%26aulast%3DBodner%26aufirst%3DR.%26aulast%3DChopra%26aufirst%3DV.%26aulast%3DDiFiglia%26aufirst%3DM.%26aulast%3DFox%26aufirst%3DJ.%26aulast%3DKegel%26aufirst%3DK.%26aulast%3DKlein%26aufirst%3DC.%26aulast%3DGrueninger%26aufirst%3DS.%26aulast%3DHersch%26aufirst%3DS.%26aulast%3DHousman%26aufirst%3DD.%26aulast%3DRegulier%26aufirst%3DE.%26aulast%3DRosas%26aufirst%3DH.%2BD.%26aulast%3DStefani%26aufirst%3DM.%26aulast%3DZeitlin%26aufirst%3DS.%26aulast%3DBilbe%26aufirst%3DG.%26aulast%3DPaganetti%26aufirst%3DP.%26atitle%3DSingle-step%2520detection%2520of%2520mutant%2520huntingtin%2520in%2520animal%2520and%2520human%2520tissues%253A%2520a%2520bioassay%2520for%2520Huntington%25E2%2580%2599s%2520disease%26jtitle%3DAnal.%2520Biochem.%26date%3D2009%26volume%3D395%26spage%3D8%26epage%3D15%26doi%3D10.1016%2Fj.ab.2009.08.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Harper, S. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Staber, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eliason, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martins, I. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kotin, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulson, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davidson, B. L.</span></span> <span> </span><span class="NLM_article-title">RNA interference improves motor and neuropathological abnormalities in a Huntington’s disease mouse model</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>102</i></span>,  <span class="NLM_fpage">5820</span>– <span class="NLM_lpage">5825</span>, <span class="refDoi"> DOI: 10.1073/pnas.0501507102</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1073%2Fpnas.0501507102" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=15811941" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A528%3ADC%252BD2MXjslaktbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2005&pages=5820-5825&author=S.+Q.+Harperauthor=P.+D.+Staberauthor=X.+Heauthor=S.+L.+Eliasonauthor=I.+H.+Martinsauthor=Q.+Maoauthor=L.+Yangauthor=R.+M.+Kotinauthor=H.+L.+Paulsonauthor=B.+L.+Davidson&title=RNA+interference+improves+motor+and+neuropathological+abnormalities+in+a+Huntington%E2%80%99s+disease+mouse+model&doi=10.1073%2Fpnas.0501507102"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">RNA interference improves motor and neuropathological abnormalities in a Huntington's disease mouse model</span></div><div class="casAuthors">Harper, Scott Q.; Staber, Patrick D.; He, Xiaohua; Eliason, Steven L.; Martins, Ines H.; Mao, Qinwen; Yang, Linda; Kotin, Robert M.; Paulson, Henry L.; Davidson, Beverly L.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">5820-5825</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Huntington's disease (HD) is a fatal, dominant neurogenetic disorder.  HD results from polyglutamine repeat expansion (CAG codon, Q) in exon 1 of HD, conferring a toxic gain of function on the protein huntingtin (htt).  Currently, no preventative treatment exists for HD.  RNA interference (RNAi) has emerged as a potential therapeutic tool for treating dominant diseases by directly reducing disease gene expression.  Here, authors show that RNAi directed against mutant human htt reduced htt mRNA and protein expression in cell culture and in HD mouse brain.  Importantly, htt gene silencing improved behavioral and neuropathol. abnormalities assocd. with HD.  These data provide support for the further development of RNAi for HD therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYayMgT1-X7bVg90H21EOLACvtfcHk0lidRuQQSu4zHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXjslaktbo%253D&md5=7a70a4e0ff4411d952e94bce0a671e56</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0501507102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0501507102%26sid%3Dliteratum%253Aachs%26aulast%3DHarper%26aufirst%3DS.%2BQ.%26aulast%3DStaber%26aufirst%3DP.%2BD.%26aulast%3DHe%26aufirst%3DX.%26aulast%3DEliason%26aufirst%3DS.%2BL.%26aulast%3DMartins%26aufirst%3DI.%2BH.%26aulast%3DMao%26aufirst%3DQ.%26aulast%3DYang%26aufirst%3DL.%26aulast%3DKotin%26aufirst%3DR.%2BM.%26aulast%3DPaulson%26aufirst%3DH.%2BL.%26aulast%3DDavidson%26aufirst%3DB.%2BL.%26atitle%3DRNA%2520interference%2520improves%2520motor%2520and%2520neuropathological%2520abnormalities%2520in%2520a%2520Huntington%25E2%2580%2599s%2520disease%2520mouse%2520model%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2005%26volume%3D102%26spage%3D5820%26epage%3D5825%26doi%3D10.1073%2Fpnas.0501507102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez-Lebron, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denovan-Wright, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nash, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewin, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mandel, R. J.</span></span> <span> </span><span class="NLM_article-title">Intrastriatal rAAV-mediated delivery of anti-huntingtin shRNAs induces partial reversal of disease progression in R6/1 Huntington’s disease transgenic mice</span>. <i>Mol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">618</span>– <span class="NLM_lpage">633</span>, <span class="refDoi"> DOI: 10.1016/j.ymthe.2005.05.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1016%2Fj.ymthe.2005.05.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=16019264" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtVWgsbzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2005&pages=618-633&author=E.+Rodriguez-Lebronauthor=E.+M.+Denovan-Wrightauthor=K.+Nashauthor=A.+S.+Lewinauthor=R.+J.+Mandel&title=Intrastriatal+rAAV-mediated+delivery+of+anti-huntingtin+shRNAs+induces+partial+reversal+of+disease+progression+in+R6%2F1+Huntington%E2%80%99s+disease+transgenic+mice&doi=10.1016%2Fj.ymthe.2005.05.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Intrastriatal rAAV-mediated delivery of anti-huntingtin shRNAs induces partial reversal of disease progression in R6/1 Huntington's disease transgenic mice</span></div><div class="casAuthors">Rodriguez-Lebron, Edgardo; Denovan-Wright, Eileen M.; Nash, Kevin; Lewin, Alfred S.; Mandel, Ronald J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Therapy</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">618-633</span>CODEN:
                <span class="NLM_cas:coden">MTOHCK</span>;
        ISSN:<span class="NLM_cas:issn">1525-0016</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Huntington's disease (HD) is a fatal neurodegenerative disorder caused by the presence of an abnormally expanded polyglutamine domain in the N-terminus of huntingtin.  The authors developed a recombinant adeno-assocd. viral serotype 5 (rAAV5) gene transfer strategy to posttranscriptionally suppress the levels of striatal mutant huntingtin (mHtt) in the R6/1 HD transgenic mouse via RNA interference.  Transient cotransfection of HEK293 cells with plasmids expressing a portion of human mHtt derived from R6/1 transgenic HD mice and a short-hairpin RNA directed against the 5' UTR of the mHtt mRNA (siHUNT-1) resulted in redn. in the levels of mHtt mRNA (-75%) and protein (-60%).  Long-term in vivo rAAV5-mediated expression of siHUNT-1 in the striatum of R6/1 mice reduced the levels of mHtt mRNA (-78%) and protein (-28%) as detd. by quant. RT-PCR and Western blot anal., resp.  The redn. in mHtt was concomitant with a redn. in the size and no. of neuronal intranuclear inclusions and a small but significant normalization of the steady-state levels of preproenkephalin and dopamine- and cAMP-responsive phosphoprotein 32 kDa mRNA.  Finally, bilateral expression of rAAV5-siHUNT-1 resulted in delayed onset of the rear paw clasping phenotype exhibited by the R6/1 mice.  These results suggest that a redn. in the levels of striatal mHtt can ameliorate the HD phenotype of R6/1 mice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaTvJJnssVGrVg90H21EOLACvtfcHk0lj8EgLEVIAcxA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtVWgsbzL&md5=abf2a0184528767add5547fd471df9fc</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.ymthe.2005.05.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ymthe.2005.05.006%26sid%3Dliteratum%253Aachs%26aulast%3DRodriguez-Lebron%26aufirst%3DE.%26aulast%3DDenovan-Wright%26aufirst%3DE.%2BM.%26aulast%3DNash%26aufirst%3DK.%26aulast%3DLewin%26aufirst%3DA.%2BS.%26aulast%3DMandel%26aufirst%3DR.%2BJ.%26atitle%3DIntrastriatal%2520rAAV-mediated%2520delivery%2520of%2520anti-huntingtin%2520shRNAs%2520induces%2520partial%2520reversal%2520of%2520disease%2520progression%2520in%2520R6%252F1%2520Huntington%25E2%2580%2599s%2520disease%2520transgenic%2520mice%26jtitle%3DMol.%2520Ther.%26date%3D2005%26volume%3D12%26spage%3D618%26epage%3D633%26doi%3D10.1016%2Fj.ymthe.2005.05.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">DiFiglia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sena-Esteves, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chase, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sapp, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pfister, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sass, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoder, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reeves, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pandey, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajeev, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manoharan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sah, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zamore, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aronin, N.</span></span> <span> </span><span class="NLM_article-title">Therapeutic silencing of mutant huntingtin with siRNA attenuates striatal and cortical neuropathology and behavioral deficits</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>104</i></span>,  <span class="NLM_fpage">17204</span>– <span class="NLM_lpage">17209</span>, <span class="refDoi"> DOI: 10.1073/pnas.0708285104</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1073%2Fpnas.0708285104" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=17940007" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A528%3ADC%252BD2sXht1Kit7rK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2007&pages=17204-17209&author=M.+DiFigliaauthor=M.+Sena-Estevesauthor=K.+Chaseauthor=E.+Sappauthor=E.+Pfisterauthor=M.+Sassauthor=J.+Yoderauthor=P.+Reevesauthor=R.+K.+Pandeyauthor=K.+G.+Rajeevauthor=M.+Manoharanauthor=D.+W.+Sahauthor=P.+D.+Zamoreauthor=N.+Aronin&title=Therapeutic+silencing+of+mutant+huntingtin+with+siRNA+attenuates+striatal+and+cortical+neuropathology+and+behavioral+deficits&doi=10.1073%2Fpnas.0708285104"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Therapeutic silencing of mutant huntingtin with siRNA attenuates striatal and cortical neuropathology and behavioral deficits</span></div><div class="casAuthors">DiFiglia, M.; Sena-Esteves, M.; Chase, K.; Sapp, E.; Pfister, E.; Sass, M.; Yoder, J.; Reeves, P.; Pandey, R. K.; Rajeev, K. G.; Manoharan, M.; Sah, D. W. Y.; Zamore, P. D.; Aronin, N.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">43</span>),
    <span class="NLM_cas:pages">17204-17209</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Huntington's disease (HD) is a neurodegenerative disorder caused by expansion of a CAG repeat in the huntingtin (Htt) gene.  HD is autosomal dominant and, in theory, amenable to therapeutic RNA silencing.  We introduced cholesterol-conjugated small interfering RNA duplexes (cc-siRNA) targeting human Htt mRNA (siRNA-Htt) into mouse striata that also received adeno-assocd. virus contg. either expanded (100 CAG) or wild-type (18 CAG) Htt cDNA encoding huntingtin (Htt) 1-400.  Adeno-assocd. virus delivery to striatum and overlying cortex of the mutant Htt gene, but not the wild type, produced neuropathol. and motor deficits.  Treatment with cc-siRNA-Htt in mice with mutant Htt prolonged survival of striatal neurons, reduced neuropil aggregates, diminished inclusion size, and lowered the frequency of clasping and footslips on balance beam. cc-siRNA-Htt was designed to target human wild-type and mutant Htt and decreased levels of both in the striatum.  Our findings indicate that a single administration into the adult striatum of an siRNA targeting Htt can silence mutant Htt, attenuate neuronal pathol., and delay the abnormal behavioral phenotype obsd. in a rapid-onset, viral transgenic mouse model of HD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrQOhggGghq7Vg90H21EOLACvtfcHk0lj8EgLEVIAcxA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXht1Kit7rK&md5=c14be5978ed75dec51270a54b7f566e2</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0708285104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0708285104%26sid%3Dliteratum%253Aachs%26aulast%3DDiFiglia%26aufirst%3DM.%26aulast%3DSena-Esteves%26aufirst%3DM.%26aulast%3DChase%26aufirst%3DK.%26aulast%3DSapp%26aufirst%3DE.%26aulast%3DPfister%26aufirst%3DE.%26aulast%3DSass%26aufirst%3DM.%26aulast%3DYoder%26aufirst%3DJ.%26aulast%3DReeves%26aufirst%3DP.%26aulast%3DPandey%26aufirst%3DR.%2BK.%26aulast%3DRajeev%26aufirst%3DK.%2BG.%26aulast%3DManoharan%26aufirst%3DM.%26aulast%3DSah%26aufirst%3DD.%2BW.%26aulast%3DZamore%26aufirst%3DP.%2BD.%26aulast%3DAronin%26aufirst%3DN.%26atitle%3DTherapeutic%2520silencing%2520of%2520mutant%2520huntingtin%2520with%2520siRNA%2520attenuates%2520striatal%2520and%2520cortical%2520neuropathology%2520and%2520behavioral%2520deficits%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2007%26volume%3D104%26spage%3D17204%26epage%3D17209%26doi%3D10.1073%2Fpnas.0708285104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kordasiewicz, H. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stanek, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wancewicz, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mazur, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McAlonis, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pytel, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Artates, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shihabuddin, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, C. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cleveland, D. W.</span></span> <span> </span><span class="NLM_article-title">Sustained therapeutic reversal of Huntington’s disease by transient repression of huntingtin synthesis</span>. <i>Neuron</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>74</i></span>,  <span class="NLM_fpage">1031</span>– <span class="NLM_lpage">1044</span>, <span class="refDoi"> DOI: 10.1016/j.neuron.2012.05.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1016%2Fj.neuron.2012.05.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=22726834" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A528%3ADC%252BC38XptVKgtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2012&pages=1031-1044&author=H.+B.+Kordasiewiczauthor=L.+M.+Stanekauthor=E.+V.+Wancewiczauthor=C.+Mazurauthor=M.+M.+McAlonisauthor=K.+A.+Pytelauthor=J.+W.+Artatesauthor=A.+Weissauthor=S.+H.+Chengauthor=L.+S.+Shihabuddinauthor=G.+Hungauthor=C.+F.+Bennettauthor=D.+W.+Cleveland&title=Sustained+therapeutic+reversal+of+Huntington%E2%80%99s+disease+by+transient+repression+of+huntingtin+synthesis&doi=10.1016%2Fj.neuron.2012.05.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Sustained Therapeutic Reversal of Huntington's Disease by Transient Repression of Huntingtin Synthesis</span></div><div class="casAuthors">Kordasiewicz, Holly B.; Stanek, Lisa M.; Wancewicz, Edward V.; Mazur, Curt; McAlonis, Melissa M.; Pytel, Kimberly A.; Artates, Jonathan W.; Weiss, Andreas; Cheng, Seng H.; Shihabuddin, Lamya S.; Hung, Gene; Bennett, C. Frank; Cleveland, Don W.</div><div class="citationInfo"><span class="NLM_cas:title">Neuron</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">74</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1031-1044</span>CODEN:
                <span class="NLM_cas:coden">NERNET</span>;
        ISSN:<span class="NLM_cas:issn">0896-6273</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The primary cause of Huntington's disease (HD) is expression of huntingtin with a polyglutamine expansion.  Despite an absence of consensus on the mechanism(s) of toxicity, diminishing the synthesis of mutant huntingtin will abate toxicity if delivered to the key affected cells.  With antisense oligonucleotides (ASOs) that catalyze RNase H-mediated degrdn. of huntingtin mRNA, we demonstrate that transient infusion into the cerebrospinal fluid of symptomatic HD mouse models not only delays disease progression but mediates a sustained reversal of disease phenotype that persists longer than the huntingtin knockdown.  Redn. of wild-type huntingtin, along with mutant huntingtin, produces the same sustained disease reversal.  Similar ASO infusion into nonhuman primates is shown to effectively lower huntingtin in many brain regions targeted by HD pathol.  Rather than requiring continuous treatment, our findings establish a therapeutic strategy for sustained HD disease reversal produced by transient ASO-mediated diminution of huntingtin synthesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrn0_XTuNHKjrVg90H21EOLACvtfcHk0lj8EgLEVIAcxA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XptVKgtbs%253D&md5=f4e2afdce8cbca04dbd3dcbbbe7f0c1b</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.neuron.2012.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neuron.2012.05.009%26sid%3Dliteratum%253Aachs%26aulast%3DKordasiewicz%26aufirst%3DH.%2BB.%26aulast%3DStanek%26aufirst%3DL.%2BM.%26aulast%3DWancewicz%26aufirst%3DE.%2BV.%26aulast%3DMazur%26aufirst%3DC.%26aulast%3DMcAlonis%26aufirst%3DM.%2BM.%26aulast%3DPytel%26aufirst%3DK.%2BA.%26aulast%3DArtates%26aufirst%3DJ.%2BW.%26aulast%3DWeiss%26aufirst%3DA.%26aulast%3DCheng%26aufirst%3DS.%2BH.%26aulast%3DShihabuddin%26aufirst%3DL.%2BS.%26aulast%3DHung%26aufirst%3DG.%26aulast%3DBennett%26aufirst%3DC.%2BF.%26aulast%3DCleveland%26aufirst%3DD.%2BW.%26atitle%3DSustained%2520therapeutic%2520reversal%2520of%2520Huntington%25E2%2580%2599s%2520disease%2520by%2520transient%2520repression%2520of%2520huntingtin%2520synthesis%26jtitle%3DNeuron%26date%3D2012%26volume%3D74%26spage%3D1031%26epage%3D1044%26doi%3D10.1016%2Fj.neuron.2012.05.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X. J.</span></span> <span> </span><span class="NLM_article-title">CRISPR/Cas9-mediated gene editing ameliorates neurotoxicity in mouse model of Huntington’s disease</span>. <i>J. Clin. Invest.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>127</i></span>,  <span class="NLM_fpage">2719</span>– <span class="NLM_lpage">2724</span>, <span class="refDoi"> DOI: 10.1172/JCI92087</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1172%2FJCI92087" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=28628038" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A280%3ADC%252BC1cnps1Wqsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=127&publication_year=2017&pages=2719-2724&author=S.+Yangauthor=R.+Changauthor=H.+Yangauthor=T.+Zhaoauthor=Y.+Hongauthor=H.+E.+Kongauthor=X.+Sunauthor=Z.+Qinauthor=P.+Jinauthor=S.+Liauthor=X.+J.+Li&title=CRISPR%2FCas9-mediated+gene+editing+ameliorates+neurotoxicity+in+mouse+model+of+Huntington%E2%80%99s+disease&doi=10.1172%2FJCI92087"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">CRISPR/Cas9-mediated gene editing ameliorates neurotoxicity in mouse model of Huntington's disease</span></div><div class="casAuthors">Yang Su; Chang Renbao; Yang Huiming; Zhao Ting; Hong Yan; Kong Ha Eun; Jin Peng; Li Shihua; Li Xiao-Jiang; Chang Renbao; Chang Renbao; Sun Xiaobo; Qin Zhaohui; Li Xiao-Jiang</div><div class="citationInfo"><span class="NLM_cas:title">The Journal of clinical investigation</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">127</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2719-2724</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Huntington's disease is a neurodegenerative disorder caused by a polyglutamine repeat in the Huntingtin gene (HTT).  Although suppressing the expression of mutant HTT (mHTT) has been explored as a therapeutic strategy to treat Huntington's disease, considerable efforts have gone into developing allele-specific suppression of mHTT expression, given that loss of Htt in mice can lead to embryonic lethality.  It remains unknown whether depletion of HTT in the adult brain, regardless of its allele, could be a safe therapy.  Here, we report that permanent suppression of endogenous mHTT expression in the striatum of mHTT-expressing mice (HD140Q-knockin mice) using CRISPR/Cas9-mediated inactivation effectively depleted HTT aggregates and attenuated early neuropathology.  The reduction of mHTT expression in striatal neuronal cells in adult HD140Q-knockin mice did not affect viability, but alleviated motor deficits.  Our studies suggest that non-allele-specific CRISPR/Cas9-mediated gene editing could be used to efficiently and permanently eliminate polyglutamine expansion-mediated neuronal toxicity in the adult brain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSclUvCk85BBNbHMBt0mMIXfW6udTcc2eaBetoega6fYbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cnps1Wqsw%253D%253D&md5=0d4ebdb952e51a695b7595292638fba0</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1172%2FJCI92087&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252FJCI92087%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DS.%26aulast%3DChang%26aufirst%3DR.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DZhao%26aufirst%3DT.%26aulast%3DHong%26aufirst%3DY.%26aulast%3DKong%26aufirst%3DH.%2BE.%26aulast%3DSun%26aufirst%3DX.%26aulast%3DQin%26aufirst%3DZ.%26aulast%3DJin%26aufirst%3DP.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DX.%2BJ.%26atitle%3DCRISPR%252FCas9-mediated%2520gene%2520editing%2520ameliorates%2520neurotoxicity%2520in%2520mouse%2520model%2520of%2520Huntington%25E2%2580%2599s%2520disease%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D2017%26volume%3D127%26spage%3D2719%26epage%3D2724%26doi%3D10.1172%2FJCI92087" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Caron, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dorsey, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayden, M. R.</span></span> <span> </span><span class="NLM_article-title">Therapeutic approaches to Huntington disease: from the bench to the clinic</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">729</span>– <span class="NLM_lpage">750</span>, <span class="refDoi"> DOI: 10.1038/nrd.2018.133</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1038%2Fnrd.2018.133" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=30237454" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslOjsrzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2018&pages=729-750&author=N.+S.+Caronauthor=E.+R.+Dorseyauthor=M.+R.+Hayden&title=Therapeutic+approaches+to+Huntington+disease%3A+from+the+bench+to+the+clinic&doi=10.1038%2Fnrd.2018.133"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Therapeutic approaches to Huntington disease: from the bench to the clinic</span></div><div class="casAuthors">Caron, Nicholas S.; Dorsey, E. Ray; Hayden, Michael R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">729-750</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The 25 years since the identification of the gene responsible for Huntington disease (HD) have stood witness to profound discoveries about the nature of the disease and its pathogenesis.  Despite this progress, however, the development of disease-modifying therapies has thus far been slow.  Preclin. validation of the therapeutic potential of disrupted pathways in HD has led to the advancement of pharmacol. agents, both novel and repurposed, for clin. evaluation.  The most promising therapeutic approaches include huntingtin (HTT) lowering and modification as well as modulation of neuroinflammation and synaptic transmission.  With clin. trials for many of these approaches imminent or currently ongoing, the coming years are promising not only for HD but also for more prevalent neurodegenerative disorders, such as Alzheimer and Parkinson disease, in which many of these pathways have been similarly implicated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpx7V2wDUlHvLVg90H21EOLACvtfcHk0lgH5Zc_I63hEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslOjsrzP&md5=c87c8f8ee9144591ff226c05f634dc41</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2018.133&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2018.133%26sid%3Dliteratum%253Aachs%26aulast%3DCaron%26aufirst%3DN.%2BS.%26aulast%3DDorsey%26aufirst%3DE.%2BR.%26aulast%3DHayden%26aufirst%3DM.%2BR.%26atitle%3DTherapeutic%2520approaches%2520to%2520Huntington%2520disease%253A%2520from%2520the%2520bench%2520to%2520the%2520clinic%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2018%26volume%3D17%26spage%3D729%26epage%3D750%26doi%3D10.1038%2Fnrd.2018.133" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Binari, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perrimon, N.</span></span> <span> </span><span class="NLM_article-title">A genomewide RNA interference screen for modifiers of aggregates formation by mutant Huntingtin in Drosophila</span>. <i>Genetics</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>184</i></span>,  <span class="NLM_fpage">1165</span>– <span class="NLM_lpage">1179</span>, <span class="refDoi"> DOI: 10.1534/genetics.109.112516</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1534%2Fgenetics.109.112516" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=20100940" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsFOnurnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=184&publication_year=2010&pages=1165-1179&author=S.+Zhangauthor=R.+Binariauthor=R.+Zhouauthor=N.+Perrimon&title=A+genomewide+RNA+interference+screen+for+modifiers+of+aggregates+formation+by+mutant+Huntingtin+in+Drosophila&doi=10.1534%2Fgenetics.109.112516"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">A genomewide RNA interference screen for modifiers of aggregates formation by mutant huntingtin in Drosophila</span></div><div class="casAuthors">Zhang, Sheng; Binari, Richard; Zhou, Rui; Perrimon, Norbert</div><div class="citationInfo"><span class="NLM_cas:title">Genetics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">184</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1165-1179</span>CODEN:
                <span class="NLM_cas:coden">GENTAE</span>;
        ISSN:<span class="NLM_cas:issn">0016-6731</span>.
    
            (<span class="NLM_cas:orgname">Genetics Society of America</span>)
        </div><div class="casAbstract">Protein aggregates are a common pathol. feature of most neurodegenerative diseases (NDs).  Understanding their formation and regulation will help clarify their controversial roles in disease pathogenesis.  To date, there have been few systematic studies of aggregates formation in Drosophila, a model organism that has been applied extensively in modeling NDs and screening for toxicity modifiers.  We generated transgenic fly lines that express enhanced-GFP-tagged mutant Huntingtin (Htt) fragments with different lengths of polyglutamine (polyQ) tract and showed that these Htt mutants develop protein aggregates in a polyQ-length- and age-dependent manner in Drosophila.  To identify central regulators of protein aggregation, we further generated stable Drosophila cell lines expressing these Htt mutants and also established a cell-based quant. assay that allows automated measurement of aggregates within cells.  We then performed a genomewide RNA interference screen for regulators of mutant Htt aggregation and isolated 126 genes involved in diverse cellular processes.  Interestingly, although our screen focused only on mutant Htt aggregation, several of the identified candidates were known previously as toxicity modifiers of NDs.  Moreover, modulating the in vivo activity of hsp110 (CG6603) or tra1, two hits from the screen, affects neurodegeneration in a dose-dependent manner in a Drosophila model of Huntington's disease.  Thus, other aggregates regulators isolated in our screen may identify addnl. genes involved in the protein-folding pathway and neurotoxicity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoki7Fderw6ZbVg90H21EOLACvtfcHk0lgH5Zc_I63hEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsFOnurnF&md5=daf2e6693999906970162a4f8955ecee</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1534%2Fgenetics.109.112516&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1534%252Fgenetics.109.112516%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DBinari%26aufirst%3DR.%26aulast%3DZhou%26aufirst%3DR.%26aulast%3DPerrimon%26aufirst%3DN.%26atitle%3DA%2520genomewide%2520RNA%2520interference%2520screen%2520for%2520modifiers%2520of%2520aggregates%2520formation%2520by%2520mutant%2520Huntingtin%2520in%2520Drosophila%26jtitle%3DGenetics%26date%3D2010%26volume%3D184%26spage%3D1165%26epage%3D1179%26doi%3D10.1534%2Fgenetics.109.112516" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baldo, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parker, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bibel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paganetti, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaupmann, K.</span></span> <span> </span><span class="NLM_article-title">A screen for enhancers of clearance identifies huntingtin as a heat shock protein 90 (Hsp90) client protein</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>287</i></span>,  <span class="NLM_fpage">1406</span>– <span class="NLM_lpage">1414</span>, <span class="refDoi"> DOI: 10.1074/jbc.M111.294801</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1074%2Fjbc.M111.294801" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=22123826" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A528%3ADC%252BC38Xjslyksw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2012&pages=1406-1414&author=B.+Baldoauthor=A.+Weissauthor=C.+N.+Parkerauthor=M.+Bibelauthor=P.+Paganettiauthor=K.+Kaupmann&title=A+screen+for+enhancers+of+clearance+identifies+huntingtin+as+a+heat+shock+protein+90+%28Hsp90%29+client+protein&doi=10.1074%2Fjbc.M111.294801"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">A Screen for Enhancers of Clearance Identifies Huntingtin as a Heat Shock Protein 90 (Hsp90) Client Protein</span></div><div class="casAuthors">Baldo, Barbara; Weiss, Andreas; Parker, Christian N.; Bibel, Miriam; Paganetti, Paolo; Kaupmann, Klemens</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">287</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">1406-1414</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Mechanisms to reduce the cellular levels of mutant huntingtin (mHtt) provide promising strategies for treating Huntington disease (HD).  To identify compds. enhancing the degrdn. of mHtt, we performed a high throughput screen using a hippocampal HN10 cell line expressing a 573-amino acid mHtt fragment.  Several hit structures were identified as heat shock protein 90 (Hsp90) inhibitors.  Cell treatment with these compds. reduced levels of mHtt without overt toxic effects as measured by time-resolved Foerster resonance energy transfer assays and Western blots.  To characterize the mechanism of mHtt degrdn., we used the potent and selective Hsp90 inhibitor NVP-AUY922.  In HdhQ150 embryonic stem (ES) cells and in ES cell-derived neurons, NVP-AUY922 treatment substantially reduced sol. full-length mHtt levels.  In HN10 cells, Hsp90 inhibition by NVP-AUY922 enhanced mHtt clearance in the absence of any detectable Hsp70 induction.  Furthermore, inhibition of protein synthesis with cycloheximide or overexpression of dominant neg. heat shock factor 1 (Hsf1) in HdhQ150 ES cells attenuated Hsp70 induction but did not affect NVP-AUY922-mediated mHtt clearance.  Together, these data provided evidence that direct inhibition of Hsp90 chaperone function was crucial for mHtt degrdn. rather than heat shock response induction and Hsp70 up-regulation.  Co-immunopptn. expts. revealed a phys. interaction of mutant and wild-type Htt with the Hsp90 chaperone.  Hsp90 inhibition disrupted the interaction and induced clearance of Htt through the ubiquitin-proteasome system.  These data suggest that Htt is an Hsp90 client protein and that Hsp90 inhibition may provide a means to reduce mHtt in HD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBzgxmFERTErVg90H21EOLACvtfcHk0lh_6vb-T4vUcw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xjslyksw%253D%253D&md5=b0cde124dc9c309851719560d6e44f28</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M111.294801&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M111.294801%26sid%3Dliteratum%253Aachs%26aulast%3DBaldo%26aufirst%3DB.%26aulast%3DWeiss%26aufirst%3DA.%26aulast%3DParker%26aufirst%3DC.%2BN.%26aulast%3DBibel%26aufirst%3DM.%26aulast%3DPaganetti%26aufirst%3DP.%26aulast%3DKaupmann%26aufirst%3DK.%26atitle%3DA%2520screen%2520for%2520enhancers%2520of%2520clearance%2520identifies%2520huntingtin%2520as%2520a%2520heat%2520shock%2520protein%252090%2520%2528Hsp90%2529%2520client%2520protein%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2012%26volume%3D287%26spage%3D1406%26epage%3D1414%26doi%3D10.1074%2Fjbc.M111.294801" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Komatsu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maruyama, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shinohara, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakuma, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imaichi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chikatsu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuniyeda, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siu, F. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhuo, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mun, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsumoto, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hashimoto, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyajima, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Itoh, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Habata, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mori, M.</span></span> <span> </span><span class="NLM_article-title">Anatomical transcriptome of G protein-coupled receptors leads to the identification of a novel therapeutic candidate GPR52 for psychiatric disorders</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span>, <span class="NLM_elocation-id">e90134</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0090134</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1371%2Fjournal.pone.0090134" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=24587241" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVKjsbfF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&author=H.+Komatsuauthor=M.+Maruyamaauthor=S.+Yaoauthor=T.+Shinoharaauthor=K.+Sakumaauthor=S.+Imaichiauthor=T.+Chikatsuauthor=K.+Kuniyedaauthor=F.+K.+Siuauthor=L.+S.+Pengauthor=K.+Zhuoauthor=L.+S.+Munauthor=T.+M.+Hanauthor=Y.+Matsumotoauthor=T.+Hashimotoauthor=N.+Miyajimaauthor=Y.+Itohauthor=K.+Ogiauthor=Y.+Habataauthor=M.+Mori&title=Anatomical+transcriptome+of+G+protein-coupled+receptors+leads+to+the+identification+of+a+novel+therapeutic+candidate+GPR52+for+psychiatric+disorders&doi=10.1371%2Fjournal.pone.0090134"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Anatomical transcriptome of G protein-coupled receptors leads to the identification of a novel therapeutic candidate GPR52 for psychiatric disorders</span></div><div class="casAuthors">Komatsu, Hidetoshi; Maruyama, Minoru; Yao, Shuuhei; Shinohara, Tokuyuki; Sakuma, Kensuke; Imaichi, Sachiko; Chikatsu, Tomoko; Kuniyeda, Kanako; Siu, Foo Kok; Peng, Lam Sock; Zhuo, Katherine; Mun, Lay Sock; Han, Tan Min; Matsumoto, Yoshio; Hashimoto, Tadatoshi; Miyajima, Nobuyuki; Itoh, Yasuaki; Ogi, Kazuhiro; Habata, Yugo; Mori, Masaaki</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">e90134/1-e90134/16, 16 pp.</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Many drugs of abuse and most neuropharmacol. agents regulate G protein-coupled receptors (GPCRs) in the central nervous system (CNS)_ENREF_1.  The striatum, in which dopamine D1 and D2 receptors are enriched, is strongly innervated by the ventral tegmental area (VTA), which is the origin of dopaminergic cell bodies of the mesocorticolimbic dopamine system_ENREF_3 and plays a central role in the development of psychiatric disorders_ENREF_4.  Here we report the comprehensive and anatomical transcript profiling of 322 non-odorant GPCRs in mouse tissue by quant. real-time PCR (qPCR), leading to the identification of neurotherapeutic receptors exclusively expressed in the CNS, esp. in the striatum.  Among them, GPR6, GPR52, and GPR88, known as orphan GPCRs, were shown to co-localize either with a D2 receptor alone or with both D1 and D2 receptors in neurons of the basal ganglia.  Intriguingly, we found that GPR52 was well conserved among vertebrates, is Gs-coupled and responsive to the antipsychotic drug, reserpine.  We used three types of transgenic (Tg) mice employing a Cre-lox system under the control of the GPR52 promoter, namely, GPR52-LacZ Tg, human GPR52 (hGPR52) Tg, and hGPR52-GFP Tg mice.  Detailed histol. investigation suggests that GPR52 may modulate dopaminergic and glutamatergic transmission in neuronal circuits responsible for cognitive function and emotion.  In support of our prediction, GPR52 knockout and transgenic mice exhibited psychosis-related and antipsychotic-like behaviors, resp.  Therefore, we propose that GPR52 has the potential of being a therapeutic psychiatric receptor.  This approach may help identify potential therapeutic targets for CNS diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoceoSKPwv1nrVg90H21EOLACvtfcHk0lh_6vb-T4vUcw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVKjsbfF&md5=fdaf1c9f4a80ef12adf400eef5eec2f3</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0090134&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0090134%26sid%3Dliteratum%253Aachs%26aulast%3DKomatsu%26aufirst%3DH.%26aulast%3DMaruyama%26aufirst%3DM.%26aulast%3DYao%26aufirst%3DS.%26aulast%3DShinohara%26aufirst%3DT.%26aulast%3DSakuma%26aufirst%3DK.%26aulast%3DImaichi%26aufirst%3DS.%26aulast%3DChikatsu%26aufirst%3DT.%26aulast%3DKuniyeda%26aufirst%3DK.%26aulast%3DSiu%26aufirst%3DF.%2BK.%26aulast%3DPeng%26aufirst%3DL.%2BS.%26aulast%3DZhuo%26aufirst%3DK.%26aulast%3DMun%26aufirst%3DL.%2BS.%26aulast%3DHan%26aufirst%3DT.%2BM.%26aulast%3DMatsumoto%26aufirst%3DY.%26aulast%3DHashimoto%26aufirst%3DT.%26aulast%3DMiyajima%26aufirst%3DN.%26aulast%3DItoh%26aufirst%3DY.%26aulast%3DOgi%26aufirst%3DK.%26aulast%3DHabata%26aufirst%3DY.%26aulast%3DMori%26aufirst%3DM.%26atitle%3DAnatomical%2520transcriptome%2520of%2520G%2520protein-coupled%2520receptors%2520leads%2520to%2520the%2520identification%2520of%2520a%2520novel%2520therapeutic%2520candidate%2520GPR52%2520for%2520psychiatric%2520disorders%26jtitle%3DPLoS%2520One%26date%3D2014%26volume%3D9%26doi%3D10.1371%2Fjournal.pone.0090134" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yao, Y.</span>; <span class="NLM_string-name">Cui, X.</span>; <span class="NLM_string-name">Al-Ramahi, I.</span>; <span class="NLM_string-name">Sun, X.</span>; <span class="NLM_string-name">Li, B.</span>; <span class="NLM_string-name">Hou, J.</span>; <span class="NLM_string-name">Difiglia, M.</span>; <span class="NLM_string-name">Palacino, J.</span>; <span class="NLM_string-name">Wu, Z. Y.</span>; <span class="NLM_string-name">Ma, L.</span>; <span class="NLM_string-name">Botas, J.</span>; <span class="NLM_string-name">Lu, B.</span></span> <span> </span><span class="NLM_article-title">A striatal-enriched intronic GPCR modulates huntingtin levels and toxicity</span>.  <i>Elife</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume">4</span>, <span class="refDoi"> DOI: 10.7554/eLife.05449</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.7554%2FeLife.05449" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2015&author=Y.+Yao&author=X.+Cui&author=I.+Al-Ramahi&author=X.+Sun&author=B.+Li&author=J.+Hou&author=M.+Difiglia&author=J.+Palacino&author=Z.+Y.+Wu&author=L.+Ma&author=J.+Botas&author=B.+Lu&title=A+striatal-enriched+intronic+GPCR+modulates+huntingtin+levels+and+toxicity&doi=10.7554%2FeLife.05449"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.7554%2FeLife.05449&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.05449%26sid%3Dliteratum%253Aachs%26aulast%3DYao%26aufirst%3DY.%26atitle%3DA%2520striatal-enriched%2520intronic%2520GPCR%2520modulates%2520huntingtin%2520levels%2520and%2520toxicity%26jtitle%3DElife%26date%3D2015%26volume%3D4%26doi%3D10.7554%2FeLife.05449" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Song, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">An, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, B.</span></span> <span> </span><span class="NLM_article-title">Targeting Gpr52 lowers mutant HTT levels and rescues Huntington’s disease-associated phenotypes</span>. <i>Brain</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>141</i></span>,  <span class="NLM_fpage">1782</span>– <span class="NLM_lpage">1798</span>, <span class="refDoi"> DOI: 10.1093/brain/awy081</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1093%2Fbrain%2Fawy081" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=29608652" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A280%3ADC%252BC1MnotVCksg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=141&publication_year=2018&pages=1782-1798&author=H.+Songauthor=H.+Liauthor=S.+Guoauthor=Y.+Panauthor=Y.+Fuauthor=Z.+Zhouauthor=Z.+Liauthor=X.+Wenauthor=X.+Sunauthor=B.+Heauthor=H.+Guauthor=Q.+Zhaoauthor=C.+Wangauthor=P.+Anauthor=S.+Luoauthor=Y.+Huauthor=X.+Xieauthor=B.+Lu&title=Targeting+Gpr52+lowers+mutant+HTT+levels+and+rescues+Huntington%E2%80%99s+disease-associated+phenotypes&doi=10.1093%2Fbrain%2Fawy081"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Gpr52 lowers mutant HTT levels and rescues Huntington's disease-associated phenotypes</span></div><div class="casAuthors">Song Haikun; Li Hexuan; Pan Yuyin; Fu Yuhua; Zhou Zijian; Li Zhaoyang; Wen Xue; Sun Xiaoli; Zhao Quan; Wang Cen; An Ping; Lu Boxun; Guo Shimeng; He Bingqing; Gu Haifeng; Hu Youhong; Xie Xin; He Bingqing; Xie Xin; Luo Shouqing</div><div class="citationInfo"><span class="NLM_cas:title">Brain : a journal of neurology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">141</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1782-1798</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">See Huang and Gitler (doi:10.1093/brain/awy112) for a scientific commentary on this article.Lowering the levels of disease-causing proteins is an attractive treatment strategy for neurodegenerative disorders, among which Huntington's disease is an appealing disease for testing this strategy because of its monogenetic nature.  Huntington's disease is mainly caused by cytotoxicity of the mutant HTT protein with an expanded polyglutamine repeat tract.  Lowering the soluble mutant HTT may reduce its downstream toxicity and provide potential treatment for Huntington's disease.  This is hard to achieve by small-molecule compound drugs because of a lack of effective targets.  Here we demonstrate Gpr52, an orphan G protein-coupled receptor, as a potential Huntington's disease drug target.  Knocking-out Gpr52 significantly reduces mutant HTT levels in the striatum and rescues Huntington's disease-associated behavioural phenotypes in a knock-in Huntington's disease mouse model expressing endogenous mutant Htt.  Importantly, a novel Gpr52 antagonist E7 reduces mutant HTT levels and rescues Huntington's disease-associated phenotypes in cellular and mouse models.  Our study provides an entry point for Huntington's disease drug discovery by targeting Gpr52.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQSulnCGMUk0eRS7coPn-RtfW6udTcc2ea3URwI1oBGU7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MnotVCksg%253D%253D&md5=3e0e745df0131bbc5db9cbeab7de6e3f</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1093%2Fbrain%2Fawy081&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbrain%252Fawy081%26sid%3Dliteratum%253Aachs%26aulast%3DSong%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DGuo%26aufirst%3DS.%26aulast%3DPan%26aufirst%3DY.%26aulast%3DFu%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DWen%26aufirst%3DX.%26aulast%3DSun%26aufirst%3DX.%26aulast%3DHe%26aufirst%3DB.%26aulast%3DGu%26aufirst%3DH.%26aulast%3DZhao%26aufirst%3DQ.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DAn%26aufirst%3DP.%26aulast%3DLuo%26aufirst%3DS.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DXie%26aufirst%3DX.%26aulast%3DLu%26aufirst%3DB.%26atitle%3DTargeting%2520Gpr52%2520lowers%2520mutant%2520HTT%2520levels%2520and%2520rescues%2520Huntington%25E2%2580%2599s%2520disease-associated%2520phenotypes%26jtitle%3DBrain%26date%3D2018%26volume%3D141%26spage%3D1782%26epage%3D1798%26doi%3D10.1093%2Fbrain%2Fawy081" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Trettel, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rigamonti, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilditch-Maguire, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wheeler, V. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharp, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Persichetti, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cattaneo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacDonald, M. E.</span></span> <span> </span><span class="NLM_article-title">Dominant phenotypes produced by the HD mutation in STHdh(Q111) striatal cells</span>. <i>Hum. Mol. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">2799</span>– <span class="NLM_lpage">2809</span>, <span class="refDoi"> DOI: 10.1093/hmg/9.19.2799</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1093%2Fhmg%2F9.19.2799" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=11092756" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A528%3ADC%252BD3cXoslWms7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2000&pages=2799-2809&author=F.+Trettelauthor=D.+Rigamontiauthor=P.+Hilditch-Maguireauthor=V.+C.+Wheelerauthor=A.+H.+Sharpauthor=F.+Persichettiauthor=E.+Cattaneoauthor=M.+E.+MacDonald&title=Dominant+phenotypes+produced+by+the+HD+mutation+in+STHdh%28Q111%29+striatal+cells&doi=10.1093%2Fhmg%2F9.19.2799"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Dominant phenotypes produced by the HD mutation in STHdhQ111 striatal cells</span></div><div class="casAuthors">Trettel, Flavia; Rigamonti, Dorotea; Hilditch-Maguire, Paige; Wheeler, Vanessa C.; Sharp, Alan H.; Persichetti, Francesca; Cattaneo, Elena; MacDonald, Marcy E.</div><div class="citationInfo"><span class="NLM_cas:title">Human Molecular Genetics</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">2799-2809</span>CODEN:
                <span class="NLM_cas:coden">HMGEE5</span>;
        ISSN:<span class="NLM_cas:issn">0964-6906</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Lengthening a glutamine tract in huntingtin confers a dominant attribute that initiates degeneration of striatal neurons in Huntington's disease (HD).  To identify pathways that are candidates for the mutant protein's abnormal function, we compared striatal cell lines established from wild-type and HdhQ111 knock-in embryos.  Alternate versions of full-length huntingtin, distinguished by epitope accessibility, were localized to different sets of nuclear and perinuclear organelles involved in RNA biogenesis and membrane trafficking.  However, mutant STHdhQ111 cells also exhibited addnl. forms of the full-length mutant protein and displayed dominant phenotypes that did not mirror phenotypes caused by either huntingtin deficiency or excess.  These phenotypes indicate a disruption of striatal cell homeostasis by the mutant protein, via a mechanism that is sep. from its normal activity.  They also support specific stress pathways, including elevated p53, endoplasmic reticulum stress response and hypoxia, as potential players in HD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8lnl6CjJK57Vg90H21EOLACvtfcHk0ljwhMsxAJO29A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXoslWms7c%253D&md5=00dabfeb20888d87f44ad2c3762bd3c9</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1093%2Fhmg%2F9.19.2799&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fhmg%252F9.19.2799%26sid%3Dliteratum%253Aachs%26aulast%3DTrettel%26aufirst%3DF.%26aulast%3DRigamonti%26aufirst%3DD.%26aulast%3DHilditch-Maguire%26aufirst%3DP.%26aulast%3DWheeler%26aufirst%3DV.%2BC.%26aulast%3DSharp%26aufirst%3DA.%2BH.%26aulast%3DPersichetti%26aufirst%3DF.%26aulast%3DCattaneo%26aufirst%3DE.%26aulast%3DMacDonald%26aufirst%3DM.%2BE.%26atitle%3DDominant%2520phenotypes%2520produced%2520by%2520the%2520HD%2520mutation%2520in%2520STHdh%2528Q111%2529%2520striatal%2520cells%26jtitle%3DHum.%2520Mol.%2520Genet.%26date%3D2000%26volume%3D9%26spage%3D2799%26epage%3D2809%26doi%3D10.1093%2Fhmg%2F9.19.2799" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Menalled, L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sison, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dragatsis, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeitlin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chesselet, M. F.</span></span> <span> </span><span class="NLM_article-title">Time course of early motor and neuropathological anomalies in a knock-in mouse model of Huntington’s disease with 140 CAG repeats</span>. <i>J. Comp. Neurol.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>465</i></span>,  <span class="NLM_fpage">11</span>– <span class="NLM_lpage">26</span>, <span class="refDoi"> DOI: 10.1002/cne.10776</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1002%2Fcne.10776" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=12926013" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A528%3ADC%252BD3sXnvFSltrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=465&publication_year=2003&pages=11-26&author=L.+B.+Menalledauthor=J.+D.+Sisonauthor=I.+Dragatsisauthor=S.+Zeitlinauthor=M.+F.+Chesselet&title=Time+course+of+early+motor+and+neuropathological+anomalies+in+a+knock-in+mouse+model+of+Huntington%E2%80%99s+disease+with+140+CAG+repeats&doi=10.1002%2Fcne.10776"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Time course of early motor and neuropathological anomalies in a knock-in mouse model of Huntington's disease with 140 CAG repeats</span></div><div class="casAuthors">Menalled, Liliana B.; Sison, Jessica D.; Dragatsis, Ioannis; Zeitlin, Scott; Chesselet, Marie-Francoise</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Comparative Neurology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">465</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">11-26</span>CODEN:
                <span class="NLM_cas:coden">JCNEAM</span>;
        ISSN:<span class="NLM_cas:issn">0021-9967</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">Huntington's disease (HD) is caused by an abnormal expansion of CAG repeats in the gene encoding huntingtin.  The development of therapies for HD requires preclin. testing of drugs in animal models that reproduce the dysfunction and regionally specific pathol. obsd. in HD.  The authors have developed a new knock-in mouse model of HD with a chimeric mouse/human exon 1 contg. 140 CAG repeats inserted in the murine huntingtin gene.  These mice displayed an increased locomotor activity and rearing at 1 mo of age, followed by hypoactivity at 4 mo and gait anomalies at 1 yr.  Behavioral symptoms preceded neuropathol. anomalies, which became intense and widespread only at 4 mo of age.  These consisted of nuclear staining for huntingtin and huntingtin-contg. nuclear and neuropil aggregates that first appeared in the striatum, nucleus accumbens, and olfactory tubercle.  Interestingly, regions with early pathol. all receive dense dopaminergic inputs, supporting accumulating evidence for a role of dopamine in HD pathol.  Nuclear staining and aggregates predominated in striatum and layer II/III and deep layer V of the cerebral cortex, whereas neuropil aggregates were found in the globus pallidus and layer IV/superficial layer V of the cerebral cortex.  The olfactory system displayed early and marked aggregate accumulation, which may be relevant to the early deficit in odor discrimination obsd. in patients with HD.  Because of their early behavioral anomalies and regionally specific pathol., these mice provide a powerful tool with which to evaluate the effectiveness of new therapies and to study the mechanisms involved in the neuropathol. of HD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWln0m5hxNMrVg90H21EOLACvtfcHk0liGY_POPD5xWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXnvFSltrg%253D&md5=a2632e7aff3a687a218858c5612b30b3</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1002%2Fcne.10776&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcne.10776%26sid%3Dliteratum%253Aachs%26aulast%3DMenalled%26aufirst%3DL.%2BB.%26aulast%3DSison%26aufirst%3DJ.%2BD.%26aulast%3DDragatsis%26aufirst%3DI.%26aulast%3DZeitlin%26aufirst%3DS.%26aulast%3DChesselet%26aufirst%3DM.%2BF.%26atitle%3DTime%2520course%2520of%2520early%2520motor%2520and%2520neuropathological%2520anomalies%2520in%2520a%2520knock-in%2520mouse%2520model%2520of%2520Huntington%25E2%2580%2599s%2520disease%2520with%2520140%2520CAG%2520repeats%26jtitle%3DJ.%2520Comp.%2520Neurol.%26date%3D2003%26volume%3D465%26spage%3D11%26epage%3D26%26doi%3D10.1002%2Fcne.10776" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, B.</span></span> <span> </span><span class="NLM_article-title">TR-FRET Assays for endogenous Huntingtin protein level in mouse Cells</span>. <i>J. Huntington’s Dis.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">253</span>– <span class="NLM_lpage">259</span>, <span class="refDoi"> DOI: 10.3233/JHD-140104</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.3233%2FJHD-140104" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=25300329" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs12ku7bN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2014&pages=253-259&author=Y.+Liangauthor=Y.+Yaoauthor=M.+Luauthor=J.+Houauthor=S.+Yuauthor=B.+Lu&title=TR-FRET+Assays+for+endogenous+Huntingtin+protein+level+in+mouse+Cells&doi=10.3233%2FJHD-140104"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">TR-FRET Assays for Endogenous Huntingtin Protein Level in Mouse Cells</span></div><div class="casAuthors">Liang, Yijian; Yao, Yuwei; Lu, Mingxing; Hou, Jiapeng; Yu, Shenliang; Lu, Boxun</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Huntington's Disease</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">253-259</span>CODEN:
                <span class="NLM_cas:coden">JHDOA3</span>;
        ISSN:<span class="NLM_cas:issn">1879-6397</span>.
    
            (<span class="NLM_cas:orgname">IOS Press</span>)
        </div><div class="casAbstract">High-throughput measurement of huntingtin (Htt) levels is useful for Huntington's disease research.  For example, identification of genetic or chem. modifiers that reduce Htt levels by high-throughput screening provides promising strategy for HD drug discovery.  In the human cells, high-throughput measurement of Htt levels has been established based on the Time Resolved-Fluorescence Resonance Energy Transfer (TR-FRET) technol., using the 2B7/MW1 antibody pair.  Unfortunately, application of this assay in the mouse cells has been problematic due to discrepancies between TR-FRET signals and Western-blots, possibly caused by non-specific antibody binding.  Here the authors report TR-FRET assays that are able to detect endogenous Htt levels of the mouse striatal cell line (STHdh).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9z7pPeiC-KLVg90H21EOLACvtfcHk0liGY_POPD5xWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs12ku7bN&md5=fbfff21d2263e3d4079a057a97821101</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.3233%2FJHD-140104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3233%252FJHD-140104%26sid%3Dliteratum%253Aachs%26aulast%3DLiang%26aufirst%3DY.%26aulast%3DYao%26aufirst%3DY.%26aulast%3DLu%26aufirst%3DM.%26aulast%3DHou%26aufirst%3DJ.%26aulast%3DYu%26aufirst%3DS.%26aulast%3DLu%26aufirst%3DB.%26atitle%3DTR-FRET%2520Assays%2520for%2520endogenous%2520Huntingtin%2520protein%2520level%2520in%2520mouse%2520Cells%26jtitle%3DJ.%2520Huntington%25E2%2580%2599s%2520Dis.%26date%3D2014%26volume%3D3%26spage%3D253%26epage%3D259%26doi%3D10.3233%2FJHD-140104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lagoutte, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serba, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abegg, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoch, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adibekian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winssinger, N.</span></span> <span> </span><span class="NLM_article-title">Divergent synthesis and identification of the cellular targets of deoxyelephantopins</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>, <span class="NLM_elocation-id">12470</span> <span class="refDoi"> DOI: 10.1038/ncomms12470</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1038%2Fncomms12470" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=27539788" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlyltbjP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&author=R.+Lagoutteauthor=C.+Serbaauthor=D.+Abeggauthor=D.+G.+Hochauthor=A.+Adibekianauthor=N.+Winssinger&title=Divergent+synthesis+and+identification+of+the+cellular+targets+of+deoxyelephantopins&doi=10.1038%2Fncomms12470"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Divergent synthesis and identification of the cellular targets of deoxyelephantopins</span></div><div class="casAuthors">Lagoutte, Roman; Serba, Christelle; Abegg, Daniel; Hoch, Dominic G.; Adibekian, Alexander; Winssinger, Nicolas</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">12470</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Herbal exts. contg. sesquiterpene lactones have been extensively used in traditional medicine and are known to be rich in α,β-unsatd. functionalities that can covalently engage target proteins.  Here, we report synthetic methodologies to access analogs of deoxyelephantopin, a sesquiterpene lactone with anticancer properties.  Using alkyne-tagged cellular probes and quant. proteomics anal., we identified several cellular targets of deoxyelephantopin.  We further demonstrate that deoxyelephantopin antagonizes PPARγ activity in situ via covalent engagement of a cysteine residue in the zinc-finger motif of this nuclear receptor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKfKTjnY1Un7Vg90H21EOLACvtfcHk0liGY_POPD5xWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlyltbjP&md5=7e624d560c775c49cb593375bc86de5a</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fncomms12470&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms12470%26sid%3Dliteratum%253Aachs%26aulast%3DLagoutte%26aufirst%3DR.%26aulast%3DSerba%26aufirst%3DC.%26aulast%3DAbegg%26aufirst%3DD.%26aulast%3DHoch%26aufirst%3DD.%2BG.%26aulast%3DAdibekian%26aufirst%3DA.%26aulast%3DWinssinger%26aufirst%3DN.%26atitle%3DDivergent%2520synthesis%2520and%2520identification%2520of%2520the%2520cellular%2520targets%2520of%2520deoxyelephantopins%26jtitle%3DNat.%2520Commun.%26date%3D2016%26volume%3D7%26doi%3D10.1038%2Fncomms12470" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yue, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, F.</span></span> <span> </span><span class="NLM_article-title">Structural basis of ligand recognition and self-activation of orphan GPR52</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>579</i></span>,  <span class="NLM_fpage">152</span>– <span class="NLM_lpage">157</span>, <span class="refDoi"> DOI: 10.1038/s41586-020-2019-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1038%2Fs41586-020-2019-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=32076264" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A528%3ADC%252BB3cXjsVejsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=579&publication_year=2020&pages=152-157&author=X.+Linauthor=M.+Liauthor=N.+Wangauthor=Y.+Wuauthor=Z.+Luoauthor=S.+Guoauthor=G.+W.+Hanauthor=S.+Liauthor=Y.+Yueauthor=X.+Weiauthor=X.+Xieauthor=Y.+Chenauthor=S.+Zhaoauthor=J.+Wuauthor=M.+Leiauthor=F.+Xu&title=Structural+basis+of+ligand+recognition+and+self-activation+of+orphan+GPR52&doi=10.1038%2Fs41586-020-2019-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis of ligand recognition and self-activation of orphan GPR52</span></div><div class="casAuthors">Lin, Xi; Li, Mingyue; Wang, Niandong; Wu, Yiran; Luo, Zhipu; Guo, Shimeng; Han, Gye-Won; Li, Shaobai; Yue, Yang; Wei, Xiaohu; Xie, Xin; Chen, Yong; Zhao, Suwen; Wu, Jian; Lei, Ming; Xu, Fei</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">579</span>
        (<span class="NLM_cas:issue">7797</span>),
    <span class="NLM_cas:pages">152-157</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">GPR52 is a class-A orphan G-protein-coupled receptor that is highly expressed in the brain and represents a promising therapeutic target for the treatment of Huntington's disease and several psychiatric disorders.  Pathol. malfunction of GPR52 signaling occurs primarily through the heterotrimeric Gs protein, but it is unclear how GPR52 and Gs couple for signal transduction and whether a native ligand or other activating input is required.  Here we present the high-resoln. structures of human GPR52 in three states: a ligand-free state, a Gs-coupled self-activation state and a potential allosteric ligand-bound state.  Together, our structures reveal that extracellular loop 2 occupies the orthosteric binding pocket and operates as a built-in agonist, conferring an intrinsically high level of basal activity to GPR52.  A fully active state is achieved when Gs is coupled to GPR52 in the absence of an external agonist.  The receptor also features a side pocket for ligand binding.  These insights into the structure and function of GPR52 could improve our understanding of other self-activated GPCRs, enable the identification of endogenous and tool ligands, and guide drug discovery efforts that target GPR52.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotl_2-sJeM_bVg90H21EOLACvtfcHk0liGY_POPD5xWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXjsVejsLg%253D&md5=a0c899853309addd569f16f70c221e7c</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fs41586-020-2019-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-020-2019-0%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DN.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DLuo%26aufirst%3DZ.%26aulast%3DGuo%26aufirst%3DS.%26aulast%3DHan%26aufirst%3DG.%2BW.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DYue%26aufirst%3DY.%26aulast%3DWei%26aufirst%3DX.%26aulast%3DXie%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DS.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DLei%26aufirst%3DM.%26aulast%3DXu%26aufirst%3DF.%26atitle%3DStructural%2520basis%2520of%2520ligand%2520recognition%2520and%2520self-activation%2520of%2520orphan%2520GPR52%26jtitle%3DNature%26date%3D2020%26volume%3D579%26spage%3D152%26epage%3D157%26doi%3D10.1038%2Fs41586-020-2019-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, G.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yue, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, F.</span></span> <span> </span><span class="NLM_article-title">Structural basis of ligand recognition and self-activation of orphan GPR52</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>579</i></span>,  <span class="NLM_fpage">152</span>– <span class="NLM_lpage">157</span>, <span class="refDoi"> DOI: 10.1038/s41586-020-2019-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1038%2Fs41586-020-2019-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=32076264" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A528%3ADC%252BB3cXjsVejsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=579&publication_year=2020&pages=152-157&author=X.+Linauthor=M.+Liauthor=N.+Wangauthor=Y.+Wuauthor=Z.+Luoauthor=S.+Guoauthor=G.-W.+Hanauthor=S.+Liauthor=Y.+Yueauthor=X.+Weiauthor=X.+Xieauthor=Y.+Chenauthor=S.+Zhaoauthor=J.+Wuauthor=M.+Leiauthor=F.+Xu&title=Structural+basis+of+ligand+recognition+and+self-activation+of+orphan+GPR52&doi=10.1038%2Fs41586-020-2019-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis of ligand recognition and self-activation of orphan GPR52</span></div><div class="casAuthors">Lin, Xi; Li, Mingyue; Wang, Niandong; Wu, Yiran; Luo, Zhipu; Guo, Shimeng; Han, Gye-Won; Li, Shaobai; Yue, Yang; Wei, Xiaohu; Xie, Xin; Chen, Yong; Zhao, Suwen; Wu, Jian; Lei, Ming; Xu, Fei</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">579</span>
        (<span class="NLM_cas:issue">7797</span>),
    <span class="NLM_cas:pages">152-157</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">GPR52 is a class-A orphan G-protein-coupled receptor that is highly expressed in the brain and represents a promising therapeutic target for the treatment of Huntington's disease and several psychiatric disorders.  Pathol. malfunction of GPR52 signaling occurs primarily through the heterotrimeric Gs protein, but it is unclear how GPR52 and Gs couple for signal transduction and whether a native ligand or other activating input is required.  Here we present the high-resoln. structures of human GPR52 in three states: a ligand-free state, a Gs-coupled self-activation state and a potential allosteric ligand-bound state.  Together, our structures reveal that extracellular loop 2 occupies the orthosteric binding pocket and operates as a built-in agonist, conferring an intrinsically high level of basal activity to GPR52.  A fully active state is achieved when Gs is coupled to GPR52 in the absence of an external agonist.  The receptor also features a side pocket for ligand binding.  These insights into the structure and function of GPR52 could improve our understanding of other self-activated GPCRs, enable the identification of endogenous and tool ligands, and guide drug discovery efforts that target GPR52.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotl_2-sJeM_bVg90H21EOLACvtfcHk0lgwQ-m5WJRJyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXjsVejsLg%253D&md5=a0c899853309addd569f16f70c221e7c</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fs41586-020-2019-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-020-2019-0%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DN.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DLuo%26aufirst%3DZ.%26aulast%3DGuo%26aufirst%3DS.%26aulast%3DHan%26aufirst%3DG.-W.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DYue%26aufirst%3DY.%26aulast%3DWei%26aufirst%3DX.%26aulast%3DXie%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DS.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DLei%26aufirst%3DM.%26aulast%3DXu%26aufirst%3DF.%26atitle%3DStructural%2520basis%2520of%2520ligand%2520recognition%2520and%2520self-activation%2520of%2520orphan%2520GPR52%26jtitle%3DNature%26date%3D2020%26volume%3D579%26spage%3D152%26epage%3D157%26doi%3D10.1038%2Fs41586-020-2019-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Trott, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, A. J.</span></span> <span> </span><span class="NLM_article-title">AutoDock Vina: improving the speed and accuracy of docking with a new scoring function, efficient optimization, and multithreading</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">455</span>– <span class="NLM_lpage">461</span>, <span class="refDoi"> DOI: 10.1002/jcc.21334</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1002%2Fjcc.21334" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=19499576" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFGnur3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2010&pages=455-461&author=O.+Trottauthor=A.+J.+Olson&title=AutoDock+Vina%3A+improving+the+speed+and+accuracy+of+docking+with+a+new+scoring+function%2C+efficient+optimization%2C+and+multithreading&doi=10.1002%2Fjcc.21334"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">AutoDock Vina: Improving the speed and accuracy of docking with a new scoring function, efficient optimization, and multithreading</span></div><div class="casAuthors">Trott, Oleg; Olson, Arthur J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">455-461</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">AutoDock Vina, a new program for mol. docking and virtual screening, is presented.  AutoDock Vina achieves an approx. 2 orders of magnitude speed-up compared with the mol. docking software previously developed in the authors' lab (AutoDock 4), while also significantly improving the accuracy of the binding mode predictions, judging by the authors' tests on the training set used in AutoDock 4 development.  Further speed-up is achieved from parallelism, by using multithreading on multicore machines.  AutoDock Vina automatically calcs. the grid maps and clusters the results in a way transparent to the user.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLecOIkUxIlrVg90H21EOLACvtfcHk0lgwQ-m5WJRJyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFGnur3O&md5=c6974af8a1235f7aa09918d3e6f70dc4</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1002%2Fjcc.21334&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.21334%26sid%3Dliteratum%253Aachs%26aulast%3DTrott%26aufirst%3DO.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26atitle%3DAutoDock%2520Vina%253A%2520improving%2520the%2520speed%2520and%2520accuracy%2520of%2520docking%2520with%2520a%2520new%2520scoring%2520function%252C%2520efficient%2520optimization%252C%2520and%2520multithreading%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2010%26volume%3D31%26spage%3D455%26epage%3D461%26doi%3D10.1002%2Fjcc.21334" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vonsattel, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myers, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevens, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrante, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bird, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, E. P.,  Jr.</span></span> <span> </span><span class="NLM_article-title">Neuropathological classification of Huntington’s disease</span>. <i>J. Neuropathol. Exp. Neurol.</i> <span class="NLM_year" style="font-weight: bold;">1985</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">559</span>– <span class="NLM_lpage">577</span>, <span class="refDoi"> DOI: 10.1097/00005072-198511000-00003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1097%2F00005072-198511000-00003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=2932539" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A280%3ADyaL28%252Fjs1egsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=1985&pages=559-577&author=J.+P.+Vonsattelauthor=R.+H.+Myersauthor=T.+J.+Stevensauthor=R.+J.+Ferranteauthor=E.+D.+Birdauthor=E.+P.+Richardson&title=Neuropathological+classification+of+Huntington%E2%80%99s+disease&doi=10.1097%2F00005072-198511000-00003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Neuropathological classification of Huntington's disease</span></div><div class="casAuthors">Vonsattel J P; Myers R H; Stevens T J; Ferrante R J; Bird E D; Richardson E P Jr</div><div class="citationInfo"><span class="NLM_cas:title">Journal of neuropathology and experimental neurology</span>
        (<span class="NLM_cas:date">1985</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">559-77</span>
        ISSN:<span class="NLM_cas:issn">0022-3069</span>.
    </div><div class="casAbstract">In postmortem brain specimens from 163 clinically diagnosed cases of Huntington's disease (HD) the striatum exhibited marked variation in the severity of neuropathological involvement.  A system for grading this severity was established by macroscopic and microscopic criteria, resulting in five grades (0-4) designated in ascending order of severity.  The grade correlates closely with the extent of clinical disability as assessed by a rating scale.  In five cases of clinically diagnosed HD there were no discernible neuropathological abnormalities (grade 0), suggesting that the anatomical changes lag behind the development of clinical abnormalities.  In eight cases, neuropathological changes could only be recognized microscopically (grade 1).  The earliest changes were seen in the medial paraventricular portions of the caudate nucleus (CN), in the tail of the CN, and in the dorsal part of the putamen.  Counts of neurons in the CN reveal that 50% are lost in grade 1 and that 95% are lost in grade 4; astrocytes are greatly increased in grades 2-4.  These studies indicate that analyses of the CN in grade 4 would reflect mainly its astrocytic composition with a component of remote neurons projecting to the striatum.  Because of the relative preservation of the lateral half of the head of the CN in grades 1-2, these regions would reflect early cellular and biochemical changes in HD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRSGVuf9RSW3lUIs6wFK5xJfW6udTcc2eazh4Yfo2Ngbrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaL28%252Fjs1egsg%253D%253D&md5=82c0370acf5893966064699d68fc3d20</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1097%2F00005072-198511000-00003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252F00005072-198511000-00003%26sid%3Dliteratum%253Aachs%26aulast%3DVonsattel%26aufirst%3DJ.%2BP.%26aulast%3DMyers%26aufirst%3DR.%2BH.%26aulast%3DStevens%26aufirst%3DT.%2BJ.%26aulast%3DFerrante%26aufirst%3DR.%2BJ.%26aulast%3DBird%26aufirst%3DE.%2BD.%26aulast%3DRichardson%26aufirst%3DE.%2BP.%26atitle%3DNeuropathological%2520classification%2520of%2520Huntington%25E2%2580%2599s%2520disease%26jtitle%3DJ.%2520Neuropathol.%2520Exp.%2520Neurol.%26date%3D1985%26volume%3D44%26spage%3D559%26epage%3D577%26doi%3D10.1097%2F00005072-198511000-00003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vonsattel, J. P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DiFiglia, M.</span></span> <span> </span><span class="NLM_article-title">Huntington disease</span>. <i>J. Neuropathol. Exp. Neurol.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">369</span>– <span class="NLM_lpage">384</span>, <span class="refDoi"> DOI: 10.1097/00005072-199805000-00001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1097%2F00005072-199805000-00001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=9596408" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A280%3ADyaK1c3lsFyntQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=1998&pages=369-384&author=J.+P.+G.+Vonsattelauthor=M.+DiFiglia&title=Huntington+disease&doi=10.1097%2F00005072-199805000-00001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Huntington disease</span></div><div class="casAuthors">Vonsattel J P; DiFiglia M</div><div class="citationInfo"><span class="NLM_cas:title">Journal of neuropathology and experimental neurology</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">369-84</span>
        ISSN:<span class="NLM_cas:issn">0022-3069</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ_UbgFnqTFpOxsqfvU0ziafW6udTcc2eazh4Yfo2Ngbrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK1c3lsFyntQ%253D%253D&md5=3a2a5f5badf87489aa747a6a29a6f8d6</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1097%2F00005072-199805000-00001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252F00005072-199805000-00001%26sid%3Dliteratum%253Aachs%26aulast%3DVonsattel%26aufirst%3DJ.%2BP.%2BG.%26aulast%3DDiFiglia%26aufirst%3DM.%26atitle%3DHuntington%2520disease%26jtitle%3DJ.%2520Neuropathol.%2520Exp.%2520Neurol.%26date%3D1998%26volume%3D57%26spage%3D369%26epage%3D384%26doi%3D10.1097%2F00005072-199805000-00001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Waldvogel, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tippett, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vonsattel, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faull, R. L.</span></span> <span> </span><span class="NLM_article-title">The neuropathology of Huntington’s disease</span>. <i>Curr. Top. Behav. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">33</span>– <span class="NLM_lpage">80</span>, <span class="refDoi"> DOI: 10.1007/7854_2014_354</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1007%2F7854_2014_354" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=25300927" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A528%3ADC%252BC28XmsVOkt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2015&pages=33-80&author=H.+J.+Waldvogelauthor=E.+H.+Kimauthor=L.+J.+Tippettauthor=J.+P.+Vonsattelauthor=R.+L.+Faull&title=The+neuropathology+of+Huntington%E2%80%99s+disease&doi=10.1007%2F7854_2014_354"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">The Neuropathology of Huntington's Disease</span></div><div class="casAuthors">Waldvogel, Henry J.; Kim, Eric H.; Tippett, Lynette J.; Vonsattel, Jean-Paul G.; Faull, Richard L. M.</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Behavioral Neurosciences</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">Behavioral Neurobiology of Huntington's Disease and Parkinson's Disease</span>),
    <span class="NLM_cas:pages">33-80</span>CODEN:
                <span class="NLM_cas:coden">CTBNBW</span>;
        ISSN:<span class="NLM_cas:issn">1866-3389</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The basal ganglia are a highly interconnected set of subcortical nuclei and major atrophy in one or more regions may have major effects on other regions of the brain.  Therefore, the striatum which is preferentially degenerated and receives projections from the entire cortex also affects the regions to which it targets, esp. the globus pallidus and substantia nigra pars reticulata.  Addnl., the cerebral cortex is itself severely affected as are many other regions of the brain, esp. in more advanced cases.  The cell loss in the basal ganglia and the cerebral cortex is extensive.  The most important new findings in Huntington's disease pathol. is the highly variable nature of the degeneration in the brain.  Most interestingly, this variable pattern of pathol. appears to reflect the highly variable symptomatol. of cases with Huntington's disease even among cases possessing the same no. of CAG repeats.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAPVBC4isuNLVg90H21EOLACvtfcHk0lgQGDRdAga3Dg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmsVOkt7w%253D&md5=7b78454249ab10e4debc22f34b0a8f49</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1007%2F7854_2014_354&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F7854_2014_354%26sid%3Dliteratum%253Aachs%26aulast%3DWaldvogel%26aufirst%3DH.%2BJ.%26aulast%3DKim%26aufirst%3DE.%2BH.%26aulast%3DTippett%26aufirst%3DL.%2BJ.%26aulast%3DVonsattel%26aufirst%3DJ.%2BP.%26aulast%3DFaull%26aufirst%3DR.%2BL.%26atitle%3DThe%2520neuropathology%2520of%2520Huntington%25E2%2580%2599s%2520disease%26jtitle%3DCurr.%2520Top.%2520Behav.%2520Neurosci.%26date%3D2015%26volume%3D22%26spage%3D33%26epage%3D80%26doi%3D10.1007%2F7854_2014_354" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Waldvogel, H. J.</span>; <span class="NLM_string-name">Thu, D.</span>; <span class="NLM_string-name">Hogg, V.</span>; <span class="NLM_string-name">Tippett, L.</span>; <span class="NLM_string-name">Faull, R. L.</span></span> <span> </span><span class="NLM_article-title">Selective neurodegeneration, neuropathology and symptom profiles in Huntington’s disease</span>. In  <i>Advances in Experimental Medicine and Biology</i>; <span class="NLM_publisher-name">Springer</span>: <span class="NLM_publisher-loc">New York, NY</span>, <span class="NLM_year">2012</span>; Vol.  <span class="NLM_volume">769</span>, pp  <span class="NLM_fpage">141</span>– <span class="NLM_lpage">152</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&pages=141-152&author=H.+J.+Waldvogel&author=D.+Thu&author=V.+Hogg&author=L.+Tippett&author=R.+L.+Faull&title=Advances+in+Experimental+Medicine+and+Biology"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DWaldvogel%26aufirst%3DH.%2BJ.%26atitle%3DSelective%2520neurodegeneration%252C%2520neuropathology%2520and%2520symptom%2520profiles%2520in%2520Huntington%25E2%2580%2599s%2520disease%26btitle%3DAdvances%2520in%2520Experimental%2520Medicine%2520and%2520Biology%26pub%3DSpringer%26date%3D2012%26volume%3D769%26spage%3D141%26epage%3D152" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Consortium, H. D. i.</span></span> <span> </span><span class="NLM_article-title">Induced pluripotent stem cells from patients with Huntington’s disease show CAG-repeat-expansion-associated phenotypes</span>. <i>Cell Stem Cell</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">264</span>– <span class="NLM_lpage">278</span>, <span class="refDoi"> DOI: 10.1016/j.stem.2012.04.027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1016%2Fj.stem.2012.04.027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=22748968" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=264-278&author=H.+D.+i.+Consortium&title=Induced+pluripotent+stem+cells+from+patients+with+Huntington%E2%80%99s+disease+show+CAG-repeat-expansion-associated+phenotypes&doi=10.1016%2Fj.stem.2012.04.027"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.stem.2012.04.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.stem.2012.04.027%26sid%3Dliteratum%253Aachs%26aulast%3DConsortium%26aufirst%3DH.%2BD.%2Bi.%26atitle%3DInduced%2520pluripotent%2520stem%2520cells%2520from%2520patients%2520with%2520Huntington%25E2%2580%2599s%2520disease%2520show%2520CAG-repeat-expansion-associated%2520phenotypes%26jtitle%3DCell%2520Stem%2520Cell%26date%3D2012%26volume%3D11%26spage%3D264%26epage%3D278%26doi%3D10.1016%2Fj.stem.2012.04.027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palacino, J.</span></span> <span> </span><span class="NLM_article-title">A novel human embryonic stem cell-derived Huntington’s disease neuronal model exhibits mutant huntingtin (mHTT) aggregates and soluble mHTT-dependent neurodegeneration</span>. <i>FASEB J.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">1820</span>– <span class="NLM_lpage">1829</span>, <span class="refDoi"> DOI: 10.1096/fj.12-219220</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1096%2Ffj.12-219220" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=23325320" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A528%3ADC%252BC3sXnsV2ktb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2013&pages=1820-1829&author=B.+Luauthor=J.+Palacino&title=A+novel+human+embryonic+stem+cell-derived+Huntington%E2%80%99s+disease+neuronal+model+exhibits+mutant+huntingtin+%28mHTT%29+aggregates+and+soluble+mHTT-dependent+neurodegeneration&doi=10.1096%2Ffj.12-219220"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">A novel human embryonic stem cell-derived Huntington's disease neuronal model exhibits mutant huntingtin (mHTT) aggregates and soluble mHTT-dependent neurodegeneration</span></div><div class="casAuthors">Lu, Boxun; Palacino, James</div><div class="citationInfo"><span class="NLM_cas:title">FASEB Journal</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1820-1829, 10.1096/fj.12-219220</span>CODEN:
                <span class="NLM_cas:coden">FAJOEC</span>;
        ISSN:<span class="NLM_cas:issn">0892-6638</span>.
    
            (<span class="NLM_cas:orgname">Federation of American Societies for Experimental Biology</span>)
        </div><div class="casAbstract">Most neurodegenerative diseases are linked to aberrant accumulation of aggregation-prone proteins.  Among them, Huntington's disease (HD) is caused by an expanded polyglutamine repeat stretch in the N terminus of the mutant huntingtin protein (mHTT), which gets cleaved and aggregates in the brain.  Recently established human induced pluripotent stem cell-derived HD neurons exhibit some disease-relevant phenotypes and provide tools for HD research.  However, they have limitations such as genetic heterogeneity and an absence of mHTT aggregates and lack a robust neurodegeneration phenotype.  In addn., the relationship between the phenotype and mHTT levels has not been elucidated.  Herein, we present a human embryonic stem cell (hESC)-derived HD neuronal model expressing HTTexon1 fragments, which addresses the deficiencies enumerated above.  The wild-type and HD lines are derived from an isogenic background and exhibit insol. mHTT aggregates and neurodegeneration.  We also demonstrate a quant. relationship between neurodegeneration and sol. monomeric (but not oligomeric or aggregated) mHTT levels.  Redn. of ∼10% of mHTT is sufficient to prevent toxicity, whereas ∼90% redn. of wild-type HTT is safe and well-tolerated in these cells.  A known HD toxicity modifier (Rhes) showed expected rescue of neurodegeneration.  Therefore, the hESC-derived neuronal models complement existing induced pluripotent stem cell-derived neuronal models and provide valuable tools for HD research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUsb9COGsBh7Vg90H21EOLACvtfcHk0liRFbwji5m1Ug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXnsV2ktb0%253D&md5=f3db78085de1e8bad73f202a7637b016</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1096%2Ffj.12-219220&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1096%252Ffj.12-219220%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DB.%26aulast%3DPalacino%26aufirst%3DJ.%26atitle%3DA%2520novel%2520human%2520embryonic%2520stem%2520cell-derived%2520Huntington%25E2%2580%2599s%2520disease%2520neuronal%2520model%2520exhibits%2520mutant%2520huntingtin%2520%2528mHTT%2529%2520aggregates%2520and%2520soluble%2520mHTT-dependent%2520neurodegeneration%26jtitle%3DFASEB%2520J.%26date%3D2013%26volume%3D27%26spage%3D1820%26epage%3D1829%26doi%3D10.1096%2Ffj.12-219220" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tsvetkov, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrasate, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pleiss, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finkbeiner, S.</span></span> <span> </span><span class="NLM_article-title">A small-molecule scaffold induces autophagy in primary neurons and protects against toxicity in a Huntington disease model</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>107</i></span>,  <span class="NLM_fpage">16982</span>– <span class="NLM_lpage">16987</span>, <span class="refDoi"> DOI: 10.1073/pnas.1004498107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1073%2Fpnas.1004498107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=20833817" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1Oht7fP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2010&pages=16982-16987&author=A.+S.+Tsvetkovauthor=J.+Millerauthor=M.+Arrasateauthor=J.+S.+Wongauthor=M.+A.+Pleissauthor=S.+Finkbeiner&title=A+small-molecule+scaffold+induces+autophagy+in+primary+neurons+and+protects+against+toxicity+in+a+Huntington+disease+model&doi=10.1073%2Fpnas.1004498107"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">A small-molecule scaffold induces autophagy in primary neurons and protects against toxicity in a Huntington disease model</span></div><div class="casAuthors">Tsvetkov, Andrey S.; Miller, Jason; Arrasate, Montserrat; Wong, Jinny S.; Pleiss, Michael A.; Finkbeiner, Steven</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue">39</span>),
    <span class="NLM_cas:pages">16982-16987, S16982/1-S16982/6</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Autophagy is an intracellular turnover pathway.  It has special relevance for neurodegenerative proteinopathies, such as Alzheimer disease, Parkinson disease, and Huntington disease (HD), which are characterized by the accumulation of misfolded proteins.  Although induction of autophagy enhances clearance of misfolded protein and has therefore been suggested as a therapy for proteinopathies, neurons appear to be less responsive to classic autophagy inducers than nonneuronal cells.  Searching for improved inducers of neuronal autophagy, we discovered an N10-substituted phenoxazine that, at proper doses, potently and safely up-regulated autophagy in neurons in an Akt- and mTOR-independent fashion.  In a neuron model of HD, this compd. was neuroprotective and decreased the accumulation of diffuse and aggregated misfolded protein.  A structure/activity anal. with structurally similar compds. approved by the US Food and Drug Administration revealed a defined pharmacophore for inducing neuronal autophagy.  This pharmacophore should prove useful in studying autophagy in neurons and in developing therapies for neurodegenerative proteinopathies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtnUfvW65_mrVg90H21EOLACvtfcHk0liRFbwji5m1Ug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1Oht7fP&md5=aaefe7b3fc1c0e12c53b67c660219767</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1004498107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1004498107%26sid%3Dliteratum%253Aachs%26aulast%3DTsvetkov%26aufirst%3DA.%2BS.%26aulast%3DMiller%26aufirst%3DJ.%26aulast%3DArrasate%26aufirst%3DM.%26aulast%3DWong%26aufirst%3DJ.%2BS.%26aulast%3DPleiss%26aufirst%3DM.%2BA.%26aulast%3DFinkbeiner%26aufirst%3DS.%26atitle%3DA%2520small-molecule%2520scaffold%2520induces%2520autophagy%2520in%2520primary%2520neurons%2520and%2520protects%2520against%2520toxicity%2520in%2520a%2520Huntington%2520disease%2520model%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2010%26volume%3D107%26spage%3D16982%26epage%3D16987%26doi%3D10.1073%2Fpnas.1004498107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Landles, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sathasivam, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodman, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moffitt, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finkbeiner, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gafni, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellerby, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trottier, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richards, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osmand, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paganetti, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bates, G. P.</span></span> <span> </span><span class="NLM_article-title">Proteolysis of mutant huntingtin produces an exon 1 fragment that accumulates as an aggregated protein in neuronal nuclei in Huntington disease</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>285</i></span>,  <span class="NLM_fpage">8808</span>– <span class="NLM_lpage">8823</span>, <span class="refDoi"> DOI: 10.1074/jbc.M109.075028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1074%2Fjbc.M109.075028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=20086007" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjtFShsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=8808-8823&author=C.+Landlesauthor=K.+Sathasivamauthor=A.+Weissauthor=B.+Woodmanauthor=H.+Moffittauthor=S.+Finkbeinerauthor=B.+Sunauthor=J.+Gafniauthor=L.+M.+Ellerbyauthor=Y.+Trottierauthor=W.+G.+Richardsauthor=A.+Osmandauthor=P.+Paganettiauthor=G.+P.+Bates&title=Proteolysis+of+mutant+huntingtin+produces+an+exon+1+fragment+that+accumulates+as+an+aggregated+protein+in+neuronal+nuclei+in+Huntington+disease&doi=10.1074%2Fjbc.M109.075028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Proteolysis of Mutant Huntingtin Produces an Exon 1 Fragment That Accumulates as an Aggregated Protein in Neuronal Nuclei in Huntington Disease</span></div><div class="casAuthors">Landles, Christian; Sathasivam, Kirupa; Weiss, Andreas; Woodman, Ben; Moffitt, Hilary; Finkbeiner, Steve; Sun, Banghua; Gafni, Juliette; Ellerby, Lisa M.; Trottier, Yvon; Richards, William G.; Osmand, Alex; Paganetti, Paolo; Bates, Gillian P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">285</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">8808-8823</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Huntingtin proteolysis has been implicated in the mol. pathogenesis of Huntington disease (HD).  Despite an intense effort, the identity of the pathogenic smallest N-terminal fragment has not been detd.  Using a panel of anti-huntingtin antibodies, we employed an unbiased approach to generate proteolytic cleavage maps of mutant and wild-type huntingtin in the HdhQ150 knock-in mouse model of HD.  We identified 14 prominent N-terminal fragments, which, in addn. to the full-length protein, can be readily detected in cytoplasmic but not nuclear fractions.  These fragments were detected at all ages and are not a consequence of the pathogenic process.  We demonstrated that the smallest fragment is an exon 1 huntingtin protein, known to contain a potent nuclear export signal.  Prior to the onset of behavioral phenotypes, the exon 1 protein, and possibly other small fragments, accumulate in neuronal nuclei in the form of a detergent insol. complex, visualized as diffuse granular nuclear staining in tissue sections.  This methodol. can be used to validate the inhibition of specific proteases as therapeutic targets for HD by pharmacol. or genetic approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoiRwmWU6l4i7Vg90H21EOLACvtfcHk0liRFbwji5m1Ug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjtFShsLg%253D&md5=364988ca7c2f1de0ca697b33baf35bec</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M109.075028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M109.075028%26sid%3Dliteratum%253Aachs%26aulast%3DLandles%26aufirst%3DC.%26aulast%3DSathasivam%26aufirst%3DK.%26aulast%3DWeiss%26aufirst%3DA.%26aulast%3DWoodman%26aufirst%3DB.%26aulast%3DMoffitt%26aufirst%3DH.%26aulast%3DFinkbeiner%26aufirst%3DS.%26aulast%3DSun%26aufirst%3DB.%26aulast%3DGafni%26aufirst%3DJ.%26aulast%3DEllerby%26aufirst%3DL.%2BM.%26aulast%3DTrottier%26aufirst%3DY.%26aulast%3DRichards%26aufirst%3DW.%2BG.%26aulast%3DOsmand%26aufirst%3DA.%26aulast%3DPaganetti%26aufirst%3DP.%26aulast%3DBates%26aufirst%3DG.%2BP.%26atitle%3DProteolysis%2520of%2520mutant%2520huntingtin%2520produces%2520an%2520exon%25201%2520fragment%2520that%2520accumulates%2520as%2520an%2520aggregated%2520protein%2520in%2520neuronal%2520nuclei%2520in%2520Huntington%2520disease%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26spage%3D8808%26epage%3D8823%26doi%3D10.1074%2Fjbc.M109.075028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Crotti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glass, C. K.</span></span> <span> </span><span class="NLM_article-title">The choreography of neuroinflammation in Huntington’s disease</span>. <i>Trends Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">364</span>– <span class="NLM_lpage">373</span>, <span class="refDoi"> DOI: 10.1016/j.it.2015.04.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1016%2Fj.it.2015.04.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=26001312" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A528%3ADC%252BC2MXnslOrs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2015&pages=364-373&author=A.+Crottiauthor=C.+K.+Glass&title=The+choreography+of+neuroinflammation+in+Huntington%E2%80%99s+disease&doi=10.1016%2Fj.it.2015.04.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">The choreography of neuroinflammation in Huntington's disease</span></div><div class="casAuthors">Crotti, Andrea; Glass, Christopher K.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Immunology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">364-373</span>CODEN:
                <span class="NLM_cas:coden">TIRMAE</span>;
        ISSN:<span class="NLM_cas:issn">1471-4906</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Currently, the concept of 'neuroinflammation' includes inflammation assocd. with neurodegenerative diseases, in which there is little or no infiltration of blood-derived immune cells into the brain.  The roles of brain-resident and peripheral immune cells in these inflammatory settings are poorly understood, and it is unclear whether neuroinflammation results from immune reaction to neuronal dysfunction/degeneration, and/or represents cell-autonomous phenotypes of dysfunctional immune cells.  Here, we review recent studies examg. these questions in the context of Huntington's disease (HD), where mutant Huntingtin (HTT) is expressed in both neurons and glia.  Insights into the cellular and mol. mechanisms underlying neuroinflammation in HD may provide a better understanding of inflammation in more complex neurodegenerative disorders, and of the contribution of the neuroinflammatory component to neurodegenerative disease pathogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgjoW-X08wH7Vg90H21EOLACvtfcHk0liHQx8ZyLpKCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXnslOrs7s%253D&md5=69297dd08b2c0c2d0618f261292117d2</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.it.2015.04.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.it.2015.04.007%26sid%3Dliteratum%253Aachs%26aulast%3DCrotti%26aufirst%3DA.%26aulast%3DGlass%26aufirst%3DC.%2BK.%26atitle%3DThe%2520choreography%2520of%2520neuroinflammation%2520in%2520Huntington%25E2%2580%2599s%2520disease%26jtitle%3DTrends%2520Immunol.%26date%3D2015%26volume%3D36%26spage%3D364%26epage%3D373%26doi%3D10.1016%2Fj.it.2015.04.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Drouin-Ouellet, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sawiak, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cisbani, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lagace, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuan, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saint-Pierre, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dury, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alata, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">St-Amour, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mason, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calon, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lacroix, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gowland, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francis, S. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barker, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cicchetti, F.</span></span> <span> </span><span class="NLM_article-title">Cerebrovascular and blood-brain barrier impairments in Huntington’s disease: Potential implications for its pathophysiology</span>. <i>Ann. Neurol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>78</i></span>,  <span class="NLM_fpage">160</span>– <span class="NLM_lpage">177</span>, <span class="refDoi"> DOI: 10.1002/ana.24406</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1002%2Fana.24406" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=25866151" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A280%3ADC%252BC2MjjtlWhtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=2015&pages=160-177&author=J.+Drouin-Ouelletauthor=S.+J.+Sawiakauthor=G.+Cisbaniauthor=M.+Lagaceauthor=W.+L.+Kuanauthor=M.+Saint-Pierreauthor=R.+J.+Duryauthor=W.+Alataauthor=I.+St-Amourauthor=S.+L.+Masonauthor=F.+Calonauthor=S.+Lacroixauthor=P.+A.+Gowlandauthor=S.+T.+Francisauthor=R.+A.+Barkerauthor=F.+Cicchetti&title=Cerebrovascular+and+blood-brain+barrier+impairments+in+Huntington%E2%80%99s+disease%3A+Potential+implications+for+its+pathophysiology&doi=10.1002%2Fana.24406"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Cerebrovascular and blood-brain barrier impairments in Huntington's disease: Potential implications for its pathophysiology</span></div><div class="casAuthors">Drouin-Ouellet Janelle; Kuan Wei-Li; Mason Sarah L; Barker Roger A; Sawiak Stephen J; Cisbani Giulia; Lagace Marie; Saint-Pierre Martine; Alata Wael; St-Amour Isabelle; Calon Frederic; Lacroix Steve; Cicchetti Francesca; Dury Richard J; Gowland Penny A; Francis Susan T; Calon Frederic; Lacroix Steve; Cicchetti Francesca</div><div class="citationInfo"><span class="NLM_cas:title">Annals of neurology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">78</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">160-77</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">OBJECTIVE:  Although the underlying cause of Huntington's disease (HD) is well established, the actual pathophysiological processes involved remain to be fully elucidated.  In other proteinopathies such as Alzheimer's and Parkinson's diseases, there is evidence for impairments of the cerebral vasculature as well as the blood-brain barrier (BBB), which have been suggested to contribute to their pathophysiology.  We investigated whether similar changes are also present in HD.  METHODS:  We used 3- and 7-Tesla magnetic resonance imaging as well as postmortem tissue analyses to assess blood vessel impairments in HD patients.  Our findings were further investigated in the R6/2 mouse model using in situ cerebral perfusion, histological analysis, Western blotting, as well as transmission and scanning electron microscopy.  RESULTS:  We found mutant huntingtin protein (mHtt) aggregates to be present in all major components of the neurovascular unit of both R6/2 mice and HD patients.  This was accompanied by an increase in blood vessel density, a reduction in blood vessel diameter, as well as BBB leakage in the striatum of R6/2 mice, which correlated with a reduced expression of tight junction-associated proteins and increased numbers of transcytotic vesicles, which occasionally contained mHtt aggregates.  We confirmed the existence of similar vascular and BBB changes in HD patients.  INTERPRETATION:  Taken together, our results provide evidence for alterations in the cerebral vasculature in HD leading to BBB leakage, both in the R6/2 mouse model and in HD patients, a phenomenon that may, in turn, have important pathophysiological implications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQFAxLe_E6aqGhHGZei4OW9fW6udTcc2eYAH3JizI_Srrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MjjtlWhtA%253D%253D&md5=210d003662c40e370d866e7d04b8bb07</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1002%2Fana.24406&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fana.24406%26sid%3Dliteratum%253Aachs%26aulast%3DDrouin-Ouellet%26aufirst%3DJ.%26aulast%3DSawiak%26aufirst%3DS.%2BJ.%26aulast%3DCisbani%26aufirst%3DG.%26aulast%3DLagace%26aufirst%3DM.%26aulast%3DKuan%26aufirst%3DW.%2BL.%26aulast%3DSaint-Pierre%26aufirst%3DM.%26aulast%3DDury%26aufirst%3DR.%2BJ.%26aulast%3DAlata%26aufirst%3DW.%26aulast%3DSt-Amour%26aufirst%3DI.%26aulast%3DMason%26aufirst%3DS.%2BL.%26aulast%3DCalon%26aufirst%3DF.%26aulast%3DLacroix%26aufirst%3DS.%26aulast%3DGowland%26aufirst%3DP.%2BA.%26aulast%3DFrancis%26aufirst%3DS.%2BT.%26aulast%3DBarker%26aufirst%3DR.%2BA.%26aulast%3DCicchetti%26aufirst%3DF.%26atitle%3DCerebrovascular%2520and%2520blood-brain%2520barrier%2520impairments%2520in%2520Huntington%25E2%2580%2599s%2520disease%253A%2520Potential%2520implications%2520for%2520its%2520pathophysiology%26jtitle%3DAnn.%2520Neurol.%26date%3D2015%26volume%3D78%26spage%3D160%26epage%3D177%26doi%3D10.1002%2Fana.24406" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baldo, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paganetti, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grueninger, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcellin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaltenbach, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lo, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Semmelroth, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zivanovic, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abramowski, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lotz, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bates, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, A.</span></span> <span> </span><span class="NLM_article-title">TR-FRET-based duplex immunoassay reveals an inverse correlation of soluble and aggregated mutant huntingtin in huntington’s disease</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">264</span>– <span class="NLM_lpage">275</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2011.12.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1016%2Fj.chembiol.2011.12.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=22365609" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A528%3ADC%252BC38XjtFWhtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2012&pages=264-275&author=B.+Baldoauthor=P.+Paganettiauthor=S.+Grueningerauthor=D.+Marcellinauthor=L.+S.+Kaltenbachauthor=D.+C.+Loauthor=M.+Semmelrothauthor=A.+Zivanovicauthor=D.+Abramowskiauthor=D.+Smithauthor=G.+P.+Lotzauthor=G.+P.+Batesauthor=A.+Weiss&title=TR-FRET-based+duplex+immunoassay+reveals+an+inverse+correlation+of+soluble+and+aggregated+mutant+huntingtin+in+huntington%E2%80%99s+disease&doi=10.1016%2Fj.chembiol.2011.12.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">TR-FRET-Based Duplex Immunoassay Reveals an Inverse Correlation of Soluble and Aggregated Mutant huntingtin in Huntington's Disease</span></div><div class="casAuthors">Baldo, Barbara; Paganetti, Paolo; Grueninger, Stephan; Marcellin, David; Kaltenbach, Linda S.; Lo, Donald C.; Semmelroth, Martin; Zivanovic, Andjelija; Abramowski, Dorothee; Smith, Donna; Lotz, Gregor P.; Bates, Gillian P.; Weiss, Andreas</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">264-275</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Huntington's disease (HD) is an inherited neurodegenerative disorder caused by the amplification of a polyglutamine stretch at the N terminus of the huntingtin protein.  N-terminal fragments of the mutant huntingtin (mHtt) aggregate and form intracellular inclusions in brain and peripheral tissues.  Aggregates are an important hallmark of the disease, translating into a high need to quantify them in vitro and in vivo.  We developed a one-step TR-FRET-based immunoassay to quantify sol. and aggregated mHtt in cell and tissue homogenates.  Strikingly, quantification revealed a decrease of sol. mHtt correlating with an increase of aggregated protein in primary neuronal cell cultures, transgenic R6/2, and HdhQ150 knock-in HD mice.  These results emphasize the assay's efficiency for highly sensitive and quant. detection of sol. and aggregated mHtt and its application in high-throughput screening and characterization of HD models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHjXLeXbarcrVg90H21EOLACvtfcHk0liHQx8ZyLpKCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjtFWhtLo%253D&md5=ff1f0f82383793c05895bd86ec9cec3a</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2011.12.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2011.12.020%26sid%3Dliteratum%253Aachs%26aulast%3DBaldo%26aufirst%3DB.%26aulast%3DPaganetti%26aufirst%3DP.%26aulast%3DGrueninger%26aufirst%3DS.%26aulast%3DMarcellin%26aufirst%3DD.%26aulast%3DKaltenbach%26aufirst%3DL.%2BS.%26aulast%3DLo%26aufirst%3DD.%2BC.%26aulast%3DSemmelroth%26aufirst%3DM.%26aulast%3DZivanovic%26aufirst%3DA.%26aulast%3DAbramowski%26aufirst%3DD.%26aulast%3DSmith%26aufirst%3DD.%26aulast%3DLotz%26aufirst%3DG.%2BP.%26aulast%3DBates%26aufirst%3DG.%2BP.%26aulast%3DWeiss%26aufirst%3DA.%26atitle%3DTR-FRET-based%2520duplex%2520immunoassay%2520reveals%2520an%2520inverse%2520correlation%2520of%2520soluble%2520and%2520aggregated%2520mutant%2520huntingtin%2520in%2520huntington%25E2%2580%2599s%2520disease%26jtitle%3DChem.%2520Biol.%26date%3D2012%26volume%3D19%26spage%3D264%26epage%3D275%26doi%3D10.1016%2Fj.chembiol.2011.12.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bano, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zanetti, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mende, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicotera, P.</span></span> <span> </span><span class="NLM_article-title">Neurodegenerative processes in Huntington’s disease</span>. <i>Cell Death Dis.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>2</i></span>, <span class="NLM_elocation-id">e228</span> <span class="refDoi"> DOI: 10.1038/cddis.2011.112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1038%2Fcddis.2011.112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=22071633" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A528%3ADC%252BC38XmvVWl" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2011&author=D.+Banoauthor=F.+Zanettiauthor=Y.+Mendeauthor=P.+Nicotera&title=Neurodegenerative+processes+in+Huntington%E2%80%99s+disease&doi=10.1038%2Fcddis.2011.112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Neurodegenerative processes in Huntington's disease</span></div><div class="casAuthors">Bano, D.; Zanetti, F.; Mende, Y.; Nicotera, P.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Death & Disease</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">Nov.</span>),
    <span class="NLM_cas:pages">e228/1-e228/7</span>CODEN:
                <span class="NLM_cas:coden">CDDEA4</span>;
        ISSN:<span class="NLM_cas:issn">2041-4889</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Huntington's disease (HD) is a complex and severe disorder characterized by the gradual and the progressive loss of neurons, predominantly in the striatum, which leads to the typical motor and cognitive impairments assocd. with this pathol.  HD is caused by a highly polymorphic CAG trinucleotide repeat expansion in the exon-1 of the gene encoding for huntingtin protein.  Since the first discovery of the huntingtin gene, investigations with a consistent no. of in-vitro and in-vivo models have provided insights into the toxic events related to the expression of the mutant protein.  In this review, we will summarize the progress made in characterizing the signaling pathways that contribute to neuronal degeneration in HD.  We will highlight the age-dependent loss of proteostasis that is primarily responsible for the formation of aggregates obsd. in HD patients.  The most promising mol. targets for the development of pharmacol. interventions will also be discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrpilHy3akf4rVg90H21EOLACvtfcHk0lgQsY7SlRwTSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmvVWl&md5=3b1bba67bff5b24a312cd85c02378909</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1038%2Fcddis.2011.112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fcddis.2011.112%26sid%3Dliteratum%253Aachs%26aulast%3DBano%26aufirst%3DD.%26aulast%3DZanetti%26aufirst%3DF.%26aulast%3DMende%26aufirst%3DY.%26aulast%3DNicotera%26aufirst%3DP.%26atitle%3DNeurodegenerative%2520processes%2520in%2520Huntington%25E2%2580%2599s%2520disease%26jtitle%3DCell%2520Death%2520Dis.%26date%3D2011%26volume%3D2%26doi%3D10.1038%2Fcddis.2011.112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Labbadia, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morimoto, R. I.</span></span> <span> </span><span class="NLM_article-title">Huntington’s disease: underlying molecular mechanisms and emerging concepts</span>. <i>Trends Biochem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">378</span>– <span class="NLM_lpage">385</span>, <span class="refDoi"> DOI: 10.1016/j.tibs.2013.05.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1016%2Fj.tibs.2013.05.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=23768628" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpsV2nsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2013&pages=378-385&author=J.+Labbadiaauthor=R.+I.+Morimoto&title=Huntington%E2%80%99s+disease%3A+underlying+molecular+mechanisms+and+emerging+concepts&doi=10.1016%2Fj.tibs.2013.05.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Huntington's disease: underlying molecular mechanisms and emerging concepts</span></div><div class="casAuthors">Labbadia, John; Morimoto, Richard I.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">378-385</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Huntington's disease (HD) is a progressive neurodegenerative disorder for which no disease modifying treatments exist.  Many mol. changes and cellular consequences that underlie HD are obsd. in other neurol. disorders, suggesting that common pathol. mechanisms and pathways may exist.  Recent findings have enhanced our understanding of the way cells regulate and respond to expanded polyglutamine proteins such as mutant huntingtin.  These studies demonstrate that in addn. to effects on folding, aggregation, and clearance pathways, a general transcriptional mechanism also dictates the expression of polyglutamine proteins.  Here, we summarize the key pathways and networks that are important in HD in the context of recent therapeutic advances and highlight how their interplay may be of relevance to other protein folding disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEk5nJJvEz67Vg90H21EOLACvtfcHk0lgQsY7SlRwTSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXpsV2nsbk%253D&md5=422e5b7f479bd523abf7afefc135585c</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1016%2Fj.tibs.2013.05.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tibs.2013.05.003%26sid%3Dliteratum%253Aachs%26aulast%3DLabbadia%26aufirst%3DJ.%26aulast%3DMorimoto%26aufirst%3DR.%2BI.%26atitle%3DHuntington%25E2%2580%2599s%2520disease%253A%2520underlying%2520molecular%2520mechanisms%2520and%2520emerging%2520concepts%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D2013%26volume%3D38%26spage%3D378%26epage%3D385%26doi%3D10.1016%2Fj.tibs.2013.05.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaemmerer, W. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grondin, R. C.</span></span> <span> </span><span class="NLM_article-title">The effects of huntingtin-lowering: what do we know so far?</span>. <i>Degener. Neurol. Neuromuscular Dis.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>Volume 9</i></span>,  <span class="NLM_fpage">3</span>– <span class="NLM_lpage">17</span>, <span class="refDoi"> DOI: 10.2147/DNND.S163808</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.2147%2FDNND.S163808" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=Volume+9&publication_year=2019&pages=3-17&author=W.+F.+Kaemmererauthor=R.+C.+Grondin&title=The+effects+of+huntingtin-lowering%3A+what+do+we+know+so+far%3F&doi=10.2147%2FDNND.S163808"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.2147%2FDNND.S163808&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2147%252FDNND.S163808%26sid%3Dliteratum%253Aachs%26aulast%3DKaemmerer%26aufirst%3DW.%2BF.%26aulast%3DGrondin%26aufirst%3DR.%2BC.%26atitle%3DThe%2520effects%2520of%2520huntingtin-lowering%253A%2520what%2520do%2520we%2520know%2520so%2520far%253F%26jtitle%3DDegener.%2520Neurol.%2520Neuromuscular%2520Dis.%26date%3D2019%26volume%3DVolume%25209%26spage%3D3%26epage%3D17%26doi%3D10.2147%2FDNND.S163808" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Drouet, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perrin, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hassig, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dufour, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Auregan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alves, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonvento, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brouillet, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luthi-Carter, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hantraye, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deglon, N.</span></span> <span> </span><span class="NLM_article-title">Sustained effects of nonallele-specific Huntingtin silencing</span>. <i>Ann. Neurol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>65</i></span>,  <span class="NLM_fpage">276</span>– <span class="NLM_lpage">285</span>, <span class="refDoi"> DOI: 10.1002/ana.21569</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1002%2Fana.21569" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=19334076" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A528%3ADC%252BD1MXltVOkt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2009&pages=276-285&author=V.+Drouetauthor=V.+Perrinauthor=R.+Hassigauthor=N.+Dufourauthor=G.+Aureganauthor=S.+Alvesauthor=G.+Bonventoauthor=E.+Brouilletauthor=R.+Luthi-Carterauthor=P.+Hantrayeauthor=N.+Deglon&title=Sustained+effects+of+nonallele-specific+Huntingtin+silencing&doi=10.1002%2Fana.21569"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Sustained effects of nonallele-specific Huntingtin silencing</span></div><div class="casAuthors">Drouet, Valerie; Perrin, Valerie; Hassig, Raymonde; Dufour, Noelle; Auregan, Gwennaelle; Alves, Sandro; Bonvento, Gilles; Brouillet, Emmanuel; Luthi-Carter, Ruth; Hantraye, Philippe; Deglon, Nicole</div><div class="citationInfo"><span class="NLM_cas:title">Annals of Neurology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">276-285</span>CODEN:
                <span class="NLM_cas:coden">ANNED3</span>;
        ISSN:<span class="NLM_cas:issn">0364-5134</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">Objective: Huntington's disease (HD) is a fatal autosomal dominant neurodegenerative disorder caused by a poly-glutamine expansion in the huntingtin (htt) protein.  No cure is available to date to alleviate neurodegeneration.  Recent studies have demonstrated that RNA interference represents a promising approach for the treatment of autosomal dominant disorders.  But whether an allele-specific silencing of mutant htt or a nonallele-specific silencing should be considered has not been addressed.  Methods: the authors developed small hairpin RNA targeting mutant or wild-type htt transcripts, or both.  Results: the authors confirmed the therapeutic potential of sihtt administered with lentiviral vectors in rodent models of HD and showed that initiation of small interfering RNA treatment after the onset of HD symptoms is still efficacious and reduces the HD-like pathol.  The authors then addressed the question of the impact of nonallele-specific silencing and demonstrated that silencing of endogenous htt to 25 to 35% in vivo is altering several pathways assocd. with known htt functions but is not inducing overt toxicity or increasing striatal vulnerability up to 9 mo after treatment.  Interpretation: These data indicate that the coincident silencing of the wild-type and mutant htt may be considered as a therapeutic tool for HD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohEfZnKTt33bVg90H21EOLACvtfcHk0lgQsY7SlRwTSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXltVOkt70%253D&md5=0f66bd61902ff3bbe0a789af727ca651</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1002%2Fana.21569&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fana.21569%26sid%3Dliteratum%253Aachs%26aulast%3DDrouet%26aufirst%3DV.%26aulast%3DPerrin%26aufirst%3DV.%26aulast%3DHassig%26aufirst%3DR.%26aulast%3DDufour%26aufirst%3DN.%26aulast%3DAuregan%26aufirst%3DG.%26aulast%3DAlves%26aufirst%3DS.%26aulast%3DBonvento%26aufirst%3DG.%26aulast%3DBrouillet%26aufirst%3DE.%26aulast%3DLuthi-Carter%26aufirst%3DR.%26aulast%3DHantraye%26aufirst%3DP.%26aulast%3DDeglon%26aufirst%3DN.%26atitle%3DSustained%2520effects%2520of%2520nonallele-specific%2520Huntingtin%2520silencing%26jtitle%3DAnn.%2520Neurol.%26date%3D2009%26volume%3D65%26spage%3D276%26epage%3D285%26doi%3D10.1002%2Fana.21569" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grondin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaytor, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nelson, P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thakker, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heisel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weatherspoon, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blum, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burright, E. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaemmerer, W. F.</span></span> <span> </span><span class="NLM_article-title">Six-month partial suppression of Huntingtin is well tolerated in the adult rhesus striatum</span>. <i>Brain</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>135</i></span>,  <span class="NLM_fpage">1197</span>– <span class="NLM_lpage">1209</span>, <span class="refDoi"> DOI: 10.1093/brain/awr333</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1093%2Fbrain%2Fawr333" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=22252996" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A280%3ADC%252BC38vgs1ajsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2012&pages=1197-1209&author=R.+Grondinauthor=M.+D.+Kaytorauthor=Y.+Aiauthor=P.+T.+Nelsonauthor=D.+R.+Thakkerauthor=J.+Heiselauthor=M.+R.+Weatherspoonauthor=J.+L.+Blumauthor=E.+N.+Burrightauthor=Z.+Zhangauthor=W.+F.+Kaemmerer&title=Six-month+partial+suppression+of+Huntingtin+is+well+tolerated+in+the+adult+rhesus+striatum&doi=10.1093%2Fbrain%2Fawr333"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Six-month partial suppression of Huntingtin is well tolerated in the adult rhesus striatum</span></div><div class="casAuthors">Grondin Richard; Kaytor Michael D; Ai Yi; Nelson Peter T; Thakker Deepak R; Heisel Jennifer; Weatherspoon Marcy R; Blum Janelle L; Burright Eric N; Zhang Zhiming; Kaemmerer William F</div><div class="citationInfo"><span class="NLM_cas:title">Brain : a journal of neurology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">135</span>
        (<span class="NLM_cas:issue">Pt 4</span>),
    <span class="NLM_cas:pages">1197-209</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Huntington's disease is caused by expression of a mutant form of Huntingtin protein containing an expanded polyglutamine repeat.  One possible treatment for Huntington's disease may be to reduce expression of mutant Huntingtin in the brain via RNA interference.  Unless the therapeutic molecule is designed to be allele-specific, both wild-type and mutant protein will be suppressed by an RNA interference treatment.  A key question is whether suppression of wild-type as well as mutant Huntingtin in targeted brain regions can be tolerated and result in a net benefit to patients with Huntington's disease.  Whether Huntingtin performs essential functions in the adult brain is unclear.  Here, we tested the hypothesis that the adult primate brain can tolerate moderately reduced levels of wild-type Huntingtin protein for an extended period of time.  A serotype 2 adeno-associated viral vector encoding for a short hairpin RNA targeting rhesus huntingtin messenger RNA (active vector) was bilaterally injected into the striatum of four adult rhesus monkeys.  Four additional animals received a comparable vector encoding a scrambled control short hairpin RNA (control vector).  General health and motor behaviour were monitored for 6 months.  Upon termination, brain tissues were sampled and assessed blindly for (i) huntingtin messenger RNA knockdown; (ii) Huntingtin protein expression; and (iii) neuropathological changes.  Reduction in wild-type huntingtin messenger RNA levels averaging ∼30% was measured in the striatum of active vector recipients 6 months post-injection.  A widespread reduction in Huntingtin protein levels was also observed by immunohistochemistry in these animals, with an average protein reduction of ∼45% relative to controls measured by western blot analysis in the putamen of active vector recipients.  As with control vector recipients, no adverse effects were observed behaviourally, and no neurodegeneration was found on histological examination of active vector recipients.  Our results suggest that long-term partial suppression of wild-type Huntingtin may be safe, and thus if a comparable level of suppression of mutant Huntingtin is beneficial, then partial suppression of both wild-type and mutant Huntingtin may result in a net benefit in patients with heterozygous Huntington's disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQOr8E8G496IYvKgEt20y1XfW6udTcc2eYAMk77acT4gLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC38vgs1ajsw%253D%253D&md5=6c299a293da20ed60e15316465a904b6</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1093%2Fbrain%2Fawr333&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbrain%252Fawr333%26sid%3Dliteratum%253Aachs%26aulast%3DGrondin%26aufirst%3DR.%26aulast%3DKaytor%26aufirst%3DM.%2BD.%26aulast%3DAi%26aufirst%3DY.%26aulast%3DNelson%26aufirst%3DP.%2BT.%26aulast%3DThakker%26aufirst%3DD.%2BR.%26aulast%3DHeisel%26aufirst%3DJ.%26aulast%3DWeatherspoon%26aufirst%3DM.%2BR.%26aulast%3DBlum%26aufirst%3DJ.%2BL.%26aulast%3DBurright%26aufirst%3DE.%2BN.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DKaemmerer%26aufirst%3DW.%2BF.%26atitle%3DSix-month%2520partial%2520suppression%2520of%2520Huntingtin%2520is%2520well%2520tolerated%2520in%2520the%2520adult%2520rhesus%2520striatum%26jtitle%3DBrain%26date%3D2012%26volume%3D135%26spage%3D1197%26epage%3D1209%26doi%3D10.1093%2Fbrain%2Fawr333" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boudreau, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McBride, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martins, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xing, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davidson, B. L.</span></span> <span> </span><span class="NLM_article-title">Nonallele-specific silencing of mutant and wild-type huntingtin demonstrates therapeutic efficacy in Huntington’s disease mice</span>. <i>Mol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">1053</span>– <span class="NLM_lpage">1063</span>, <span class="refDoi"> DOI: 10.1038/mt.2009.17</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1038%2Fmt.2009.17" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=19240687" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A528%3ADC%252BD1MXit1agtb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2009&pages=1053-1063&author=R.+L.+Boudreauauthor=J.+L.+McBrideauthor=I.+Martinsauthor=S.+Shenauthor=Y.+Xingauthor=B.+J.+Carterauthor=B.+L.+Davidson&title=Nonallele-specific+silencing+of+mutant+and+wild-type+huntingtin+demonstrates+therapeutic+efficacy+in+Huntington%E2%80%99s+disease+mice&doi=10.1038%2Fmt.2009.17"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Nonallele-specific Silencing of Mutant and Wild-type Huntingtin Demonstrates Therapeutic Efficacy in Huntington's Disease Mice</span></div><div class="casAuthors">Boudreau, Ryan L.; McBride, Jodi L.; Martins, Ines; Shen, Shihao; Xing, Yi; Carter, Barrie J.; Davidson, Beverly L.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Therapy</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1053-1063</span>CODEN:
                <span class="NLM_cas:coden">MTOHCK</span>;
        ISSN:<span class="NLM_cas:issn">1525-0016</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Huntington's disease (HD) is a fatal neurodegenerative disease caused by mutant huntingtin (htt) protein, and there are currently no effective treatments.  Recently, we and others demonstrated that silencing mutant htt via RNA interference (RNAi) provides therapeutic benefit in HD mice.  We have since found that silencing wild-type htt in adult mouse striatum is tolerated for at least 4 mo.  However, given the role of htt in various cellular processes, it remains unknown whether nonallele-specific silencing of both wild-type and mutant htt is a viable therapeutic strategy for HD.  Here, we tested whether cosilencing wild-type and mutant htt provides therapeutic benefit and is tolerable in HD mice.  After treatment, HD mice showed significant redns. in wild-type and mutant htt, and demonstrated improved motor coordination and survival.  We performed transcriptional profiling to evaluate the effects of reducing wild-type htt in adult mouse striatum.  We identified gene expression changes that are concordant with previously described roles for htt in various cellular processes.  Also, several abnormally expressed transcripts assocd. with early-stage HD were differentially expressed in our studies, but intriguingly, those involved in neuronal function changed in opposing directions.  Together, these encouraging and surprising findings support further testing of nonallele-specific RNAi therapeutics for HD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmOjGm_mcSILVg90H21EOLACvtfcHk0lgPVzZ2dukbyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXit1agtb4%253D&md5=0d777457472d0ccded0fd75658c4dc7e</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1038%2Fmt.2009.17&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fmt.2009.17%26sid%3Dliteratum%253Aachs%26aulast%3DBoudreau%26aufirst%3DR.%2BL.%26aulast%3DMcBride%26aufirst%3DJ.%2BL.%26aulast%3DMartins%26aufirst%3DI.%26aulast%3DShen%26aufirst%3DS.%26aulast%3DXing%26aufirst%3DY.%26aulast%3DCarter%26aufirst%3DB.%2BJ.%26aulast%3DDavidson%26aufirst%3DB.%2BL.%26atitle%3DNonallele-specific%2520silencing%2520of%2520mutant%2520and%2520wild-type%2520huntingtin%2520demonstrates%2520therapeutic%2520efficacy%2520in%2520Huntington%25E2%2580%2599s%2520disease%2520mice%26jtitle%3DMol.%2520Ther.%26date%3D2009%26volume%3D17%26spage%3D1053%26epage%3D1063%26doi%3D10.1038%2Fmt.2009.17" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Subramaniam, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sixt, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrow, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snyder, S. H.</span></span> <span> </span><span class="NLM_article-title">Rhes, a striatal specific protein, mediates mutant-huntingtin cytotoxicity</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>324</i></span>,  <span class="NLM_fpage">1327</span>– <span class="NLM_lpage">1330</span>, <span class="refDoi"> DOI: 10.1126/science.1172871</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1126%2Fscience.1172871" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=19498170" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A528%3ADC%252BD1MXms12gur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=324&publication_year=2009&pages=1327-1330&author=S.+Subramaniamauthor=K.+M.+Sixtauthor=R.+Barrowauthor=S.+H.+Snyder&title=Rhes%2C+a+striatal+specific+protein%2C+mediates+mutant-huntingtin+cytotoxicity&doi=10.1126%2Fscience.1172871"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Rhes, a striatal specific protein, mediates mutant-huntingtin Cytotoxicity</span></div><div class="casAuthors">Subramaniam, Srinivasa; Sixt, Katherine M.; Barrow, Roxanne; Snyder, Solomon H.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">324</span>
        (<span class="NLM_cas:issue">5932</span>),
    <span class="NLM_cas:pages">1327-1330</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Huntington's disease (HD) is caused by a polyglutamine repeat in the protein huntingtin (Htt) with mutant Htt (mHtt) expressed throughout the body and similarly in all brain regions.  Yet, HD neuropathol. is largely restricted to the corpus striatum.  We report that the small guanine nucleotide-binding protein Rhes, which is localized very selectively to the striatum, binds physiol. to mHtt.  Using cultured cells, we found Rhes induces sumoylation of mHtt, which leads to cytotoxicity.  Thus, Rhes-mHtt interactions can account for the localized neuropathol. of HD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlu-umOLmUV7Vg90H21EOLACvtfcHk0lgPVzZ2dukbyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXms12gur0%253D&md5=521917309cdb18ef239d825da1aa8ea2</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1126%2Fscience.1172871&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1172871%26sid%3Dliteratum%253Aachs%26aulast%3DSubramaniam%26aufirst%3DS.%26aulast%3DSixt%26aufirst%3DK.%2BM.%26aulast%3DBarrow%26aufirst%3DR.%26aulast%3DSnyder%26aufirst%3DS.%2BH.%26atitle%3DRhes%252C%2520a%2520striatal%2520specific%2520protein%252C%2520mediates%2520mutant-huntingtin%2520cytotoxicity%26jtitle%3DScience%26date%3D2009%26volume%3D324%26spage%3D1327%26epage%3D1330%26doi%3D10.1126%2Fscience.1172871" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, G.</span></span> <span> </span><span class="NLM_article-title">alpha-secretase ADAM10 physically interacts with beta-secretase BACE1 in neurons and regulates CHL1 proteolysis</span>. <i>J. Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">411</span>– <span class="NLM_lpage">422</span>, <span class="refDoi"> DOI: 10.1093/jmcb/mjy001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1093%2Fjmcb%2Fmjy001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=29325091" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtlCnt7jO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2018&pages=411-422&author=X.+Wangauthor=C.+Wangauthor=G.+Pei&title=alpha-secretase+ADAM10+physically+interacts+with+beta-secretase+BACE1+in+neurons+and+regulates+CHL1+proteolysis&doi=10.1093%2Fjmcb%2Fmjy001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">α-secretase ADAM10 physically interacts with β-secretase BACE1 in neurons and regulates CHL1 proteolysis</span></div><div class="casAuthors">Wang, Xin; Wang, Congcong; Pei, Gang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Cell Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">411-422</span>CODEN:
                <span class="NLM_cas:coden">JMCBCU</span>;
        ISSN:<span class="NLM_cas:issn">1759-4685</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">α-secretase and β-secretase are known to compete for amyloid precursor protein (APP) processing and thus play a vital role in Alzheimer's disease pathogenesis.  A disintegrin and metalloproteinase 10 (ADAM10) and β-site APP cleaving enzyme 1 (BACE1) mediate the major activities of α-secretase and β-secretase in brain and share various common substrates.  However, whether they function sep. or together is poorly understood.  Here, we show that ADAM10 and BACE1 co-localize in the neurites of mouse primary neurons.  Co-immunopptn. and fluorescence resonance energy transfer anal. revealed that ADAM10 and BACE1 interact with each other under both endogenous and exogenous conditions.  In addn., we found that ADAM10 enhances the proteolysis of neural cell adhesion mol. close homolog of L1 (CHL1) by BACE1.  Further studies found that ADAM10-BACE1 interaction interfering peptide LT52 attenuates the regulation of ADAM10 on BACE1-mediated cleavage of CHL1.  Our data indicate that ADAM10-BACE1 interaction regulates the proteolysis of some specific substrates and may play a potential role in brain function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFdB-6zlz2hrVg90H21EOLACvtfcHk0ljesgo7hGuKog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtlCnt7jO&md5=0bda141754662e1ea9263a0e64253a4a</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1093%2Fjmcb%2Fmjy001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjmcb%252Fmjy001%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DPei%26aufirst%3DG.%26atitle%3Dalpha-secretase%2520ADAM10%2520physically%2520interacts%2520with%2520beta-secretase%2520BACE1%2520in%2520neurons%2520and%2520regulates%2520CHL1%2520proteolysis%26jtitle%3DJ.%2520Mol.%2520Cell.%2520Biol.%26date%3D2018%26volume%3D10%26spage%3D411%26epage%3D422%26doi%3D10.1093%2Fjmcb%2Fmjy001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 1 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="NLM_string-name hlFld-ContribAuthor">Hidetoshi Komatsu</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of the First Druggable GPR52 Antagonist to Treat Huntington’s Disease. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (2)
                                     , 938-940. <a href="https://doi.org/10.1021/acs.jmedchem.0c02235" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02235</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02235&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c02235%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Bthe%252BFirst%252BDruggable%252BGPR52%252BAntagonist%252Bto%252BTreat%252BHuntington%2525E2%252580%252599s%252BDisease%26aulast%3DKomatsu%26aufirst%3DHidetoshi%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D24122020%26date%3D14012021%26volume%3D64%26issue%3D2%26spage%3D938%26epage%3D940" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01133/20210121/images/medium/jm0c01133_0011.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01133/20210121/images/large/jm0c01133_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01133&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01133/20210121/images/medium/jm0c01133_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01133/20210121/images/large/jm0c01133_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Striatal knockdown of GPR52 reduces mHTT levels in adult HdhQ140 mice. (A) Experimental strategy for striatal knockdown of GPR52 in adult HdhQ140 mice. Generally, adeno-associated viruses (AAV) expressing either shRNAs targeting GPR52 (shGPR52) or the nontargeting shRNA control (shNT) were generated, followed by bi-striatal injection into 4-month-old HdhQ140 mice. After 6 weeks, mice striata were collected for further analysis. (B) QPCR analysis of GPR52 mRNA levels in STHdhQ7/Q111 cells after the viral infection with shGPR52-7 (shG7) or shGPR52-10 (shG10) for 72 h. Plots represent mean ± standard error of the mean (S.E.M.) <i>N</i> = 3–4 and the one-way analysis of variance (ANOVA) with the post hoc Bonferroni’s multiple comparisons was used for statistical analysis. *** <i>P</i> < 0.001, versus the shNT control. (C) mRNA levels of GPR52 in striata of adult HdhQ140 mice 6 weeks post bi-striatal injection with shG7 or shG10. Plots represent mean ± S.E.M. <i>N</i> = 3–4 mice per group. ** <i>P</i> < 0.01 as determined by the one-way ANOVA with the post hoc Bonferroni’s multiple comparisons. (D) TR-FRET analysis of mHTT levels with 2B7/2166 antibody pair in striata of HdhQ140 mice. Mice were bi-striatally injected with shG7 or shG10. Six weeks later, mice striata were obtained for further analysis. Plots represent mean ± S.E.M. <i>N</i> = 3–4 per group. The one-way ANOVA with the post hoc Bonferroni’s multiple comparisons was used. * <i>P</i> < 0.05, ** <i>P</i> < 0.01. (E) WB analysis of mHTT levels using Ab1 antibody in striata of HdhQ140 mice. Representative images are shown. (F) Quantification of WB analysis of mHTT levels in (A). Plots represent mean ± S.E.M. <i>N</i> = 3 per group. One-way ANOVA with the post hoc Bonferroni’s multiple comparisons was used. * <i>P</i> < 0.05.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01133/20210121/images/large/jm0c01133_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01133&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01133/20210121/images/medium/jm0c01133_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01133/20210121/images/large/jm0c01133_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. GPR52-specific antagonists <b>F11</b>, Comp-<b>8</b>, and Comp-<b>43</b>. (A) Chemical structures of (<i>E</i>)-1,7-diphenylhept-4-en-3-one (<b>F11</b>), Comp-<b>8</b>, and Comp-<b>43</b>. (B) Dose responses of <b>F11</b>, Comp-<b>8</b>, and Comp-<b>43</b> on <b>WO-459</b> (100 nM)-stimulated cAMP accumulation in HEK293 cells stably expressing GPR52. Plots represent mean ± S.E.M. <i>N</i> = 3 for independently plated and treated wells. (C) Reversibility of <b>F11</b>, Comp-<b>8</b>, and Comp-<b>43</b> for targeting GPR52 in cells. First, HEK293 cells expressing β2AR serve as the positive control. Cells were treated with dimethyl sulfoxide (DMSO) or propranolol (Pro, 100 nM) for 15 min. For the recovery group, the cells were washed to remove propranolol and then incubated with fresh medium, followed by treatment with β2AR agonist isoproterenol (ISO, 100 nM) for 30 min. Cell lysates were then analyzed by cAMP kit. Then, HEK293 cells expressing GPR52 were treated with DMSO or GPR52 antagonists (10 μM <b>F11</b>, 3 μM Comp-<b>8</b>, and 3 μM Comp-<b>43</b>). For the recovery group, the cells were washed to remove GRP52 antagonists and then incubated with fresh medium. Further, the cells were treated with GPR52 agonist WO-459 (100 nM) and cell lysates were analyzed by cAMP kit. Cells treated with DMSO serve as 100% response, and cells treated with propranolol or GPR52 antagonists without washing as indicated serve as 0% response. Plots represent mean ± S.E.M. <i>N</i> = 3 for independently plated and treated wells.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01133/20210121/images/large/jm0c01133_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01133&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01133/20210121/images/medium/jm0c01133_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01133/20210121/images/large/jm0c01133_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Structural alignments of GPR52 antagonists (wheat) in complex with GPR52 (green). (A–D) Structural alignments of GPR52 with compound <b>F11</b> (A), Comp-<b>8</b> (B), Comp-<b>14</b> (C), and Comp-<b>43</b> (D). The PDB code for GPR52 is <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6LI0">6LI0</a><a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> All of the procedures were performed in Autodock Vina,<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> and the figures were generated using PyMOL software.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01133/20210121/images/large/jm0c01133_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01133&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01133/20210121/images/medium/jm0c01133_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01133/20210121/images/large/jm0c01133_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. GPR52-specific antagonist Comp-<b>43</b> decreases mHTT levels in neuronal cells. (A) TR-FRET analysis of HTT levels using 2B7/2166 antibody pair in STHdhQ7/Q111. Cells were treated with Comp-<b>43</b> at the concentrations of 0.3–10 μM for 48 h. Plots represent mean ± S.E.M. <i>N</i> = 3–4, and the one-way ANOVA with the post hoc Bonferroni’s multiple comparisons was used for statistical analysis. * <i>P</i> < 0.05, ** <i>P</i> < 0.01, *** <i>P</i> < 0.001, versus the DMSO control. (B) WB analysis of HTT levels in STHdhQ7/Q111 cells after treatment with Comp-<b>43</b> at the concentration of 1–10 μM for 48 h. Ab1 antibody detects both mHTT and wtHTT. Rapa: Rapamycin, an autophagy enhancer as the positive control. (C) Quantification of WB analysis of mHTT levels in (B). Plots represent mean ± S.E.M. <i>N</i> = 3 per group. One-way ANOVA with the post hoc Bonferroni’s multiple comparisons was used. * <i>P</i> < 0.05, ** <i>P</i> < 0.01, versus the DMSO control. (D) TR-FRET analysis of HTT levels using 2B7/2166 antibody pair in primary HdhQ7/Q140 neurons. Cells were treated with Comp-<b>43</b> at the concentration of 0.1–3 μM for 48 h. Plots represent mean ± S.E.M. <i>N</i> = 5–6 for independently experiments. ** <i>P</i> < 0.01, *** <i>P</i> < 0.001, versus the DMSO control, as determined by the one-way ANOVA with the post hoc Bonferroni’s multiple comparisons. (E) WB analysis of HTT levels in primary HdhQ7/Q140 neurons after treatment with Comp-<b>43</b> at the concentrations of 0.3–3 μM for 48 h. (F) Quantification of WB analysis of mHTT levels in (E). Plots represent mean ± S.E.M. <i>N</i> = 4 per group. One-way ANOVA with the post hoc Donnet’s multiple comparisons was used. * <i>P</i> < 0.05, versus the DMSO control. (G) TR-FRET analysis of HTT levels using 2B7/2166 antibody pair in STHdhQ7/Q111. Cells were transfected with siNC or siGPR52 for 24 h, respectively, followed by treatment with Comp-<b>43</b> at the concentration of 1–10 μM for another 48 h. Plots represent mean ± S.E.M. N = 4, and two-way ANOVA with the post hoc Bonferroni’s multiple comparisons was used for statistical analysis. *** P < 0.001, versus the DMSO control in the siNC group. (H) WB analysis of HTT levels using Ab1 antibody. STHdhQ7/Q111 cells were transfected with siNC or siGPR52 for 24 h and then treated with Comp-<b>43</b> at the concentration of 3 and10 μM for 48 h. (I) Quantification of WB analysis of mHTT levels in (H). Plots represent mean ± S.E.M. <i>N</i> = 4 per group. One-way ANOVA with the post hoc Bonferroni’s multiple comparisons was used. * <i>P</i> < 0.05, ** <i>P</i> < 0.01, versus the DMSO control in the siNC group.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01133/20210121/images/large/jm0c01133_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01133&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01133/20210121/images/medium/jm0c01133_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01133/20210121/images/large/jm0c01133_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Comp-<b>43</b> protects mouse primary striatal neurons from mHTT toxicity. (A) Immunostaining of the neuronal-specific marker NeuN in primary striatal neurons. Primary neurons were obtained from HdhQ7/Q140 P1 mice. At DIV3-DIV5, cells were treated with Comp-<b>43</b> at 3 μM or transfected with siHTT, and then subjected to starvation by withdrawing the growth factors N2 and B27 for another 48 h. Representative images are shown. Scale bar: 100 μm. (B, C) Quantifications of the NeuN<sup>+</sup> intensity (B) and NeuN<sup>+</sup> area (C) in (A). <i>N</i> = 18, 18, and 9 for DMSO, siHTT, and Comp-<b>43</b>, respectively. Plots represent mean ± S.E.M. * <i>P</i> < 0.05, ** <i>P</i> < 0.01, *** <i>P</i> < 0.001, versus the DMSO control, as determined by one-way ANOVA with the post hoc Bonferroni’s multiple comparisons. (D) Neuronal apoptosis at different time points determined by caspase 3 activity in HdhQ7/Q140 primary striatal neurons. Cells were treated with indicated conditions and starved, and then the caspase 3 activity within the cells was monitored using a green fluorescent dye (NucView 488) and corrected by cell confluency. <i>N</i> = 12 per condition. Plots represent mean ± S.E.M. Two-way ANOVA with post hoc Bonferroni’s multiple comparisons was used for statistical analysis. *** <i>P</i> < 0.001, versus the DMSO control.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01133/20210121/images/large/jm0c01133_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01133&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01133/20210121/images/medium/jm0c01133_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01133/20210121/images/large/jm0c01133_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Intracerebroventricular treatment of Comp-<b>43</b> decreases mHTT levels in HdhQ140 mice. (A, B) TR-FRET analysis of HTT levels with 2B7/2166 antibody pair in striata and cortices of HdhQ140 mice (A) and wild-type mice (B), respectively. Nine-month-old mice were subjected to intracerebroventricular injection of Comp-<b>43</b> for 2 μL at 0.5 mM once a day for 3 consecutive weeks. <i>N</i> = 4–5 for each group. Plots represent mean ± S.E.M. * <i>P</i> < 0.05 as determined by one-way ANOVA with post hoc Bonferroni’s multiple comparisons. (C) Representative immunofluorescent images of S830 (left), and the quantification of S830<sup>+</sup> intensity (right) in striatal slices of HdhQ140 mice treated with DMSO or Comp-<b>43</b>. S830 antibody detects aggregated mHTT. Scale bar: 100 μm. For each group, four to five mice were tested with nine images taken from three independent slices. Plots represent mean ± S.E.M. An unpaired two-tailed Student’s <i>t</i> test was used. ** <i>P</i> < 0.01. (D) Representative immunostaining images of Iba1 (left) and the quantification of Iba1<sup>+</sup> intensity (right) in striatal slices of HdhQ140 mice treated with DMSO or Comp-<b>43</b>. Iba1 antibody serves as an activated microglial marker. Scale bar: 100 μm. For each group, four to five mice were tested with nine images taken from three independent slices Plots represent mean ± S.E.M. *** <i>P</i> < 0.001 as determined by an unpaired two-tailed Student’s <i>t</i> test.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01133/20210121/images/large/jm0c01133_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01133&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01133/20210121/images/medium/jm0c01133_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01133/20210121/images/large/jm0c01133_0008.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Intraperitoneal treatment of Comp-<b>43</b> attenuates HD-associated phenotypes in HdhQ140 mice. (A) Rotarod test results for 9-month-old HdhQ140 and the wild-type control mice. Mice were intraperitoneally injected with Comp-<b>43</b> at a dose of 5 mg/kg or the vehicle control once a day for 4 continuous weeks. After that, the mice were trained for 3 days and then tested for 5 days. <i>N</i> = 7–10 per group. Plots represent mean ± S.E.M. Two-way ANOVA with the post hoc Bonferroni’s multiple comparisons was used for statistical analysis. * <i>P</i> < 0.05, versus the result for HdhQ140 mice treatment with DMSO. (B) Balance beam results for 9-month-old HdhQ140 and the wild-type control mice after intraperitoneal treatment with Comp-<b>43</b> or the vehicle control. Balance beam test was performed after training for 2 consecutive days. <i>N</i> = 7–10 per group. Plots represent mean ± S.E.M. * <i>P</i> < 0.05 as determined by two-way ANOVA with the post hoc Bonferroni’s multiple comparisons. (C) Soluble mHTT levels were measured with 2B7/2166 antibody pair in HdhQ140 mice by TR-FRET. After the behavioral tests, mice striata and cortices were obtained, and then the soluble supernatant proteins were extracted after lysed, homogenized, and centrifuged at 20 000<i>g</i> at 4 °C for 20 min. <i>N</i> = 5–6 for each group. Plots represent mean ± S.E.M. * <i>P</i> < 0.05 as determined by an unpaired two-tailed Student’s <i>t</i> test. (D) mHTT aggregation levels were measured with 4C9/4C9 antibody pair in HdhQ140 mice by TR-FRET. After the soluble supernatant proteins from striata and cortices were extracted, the aggregates were resuspended with 2% sodium dodecyl sulfate (SDS) followed by further sonication. After another centrifugation at 20 000<i>g</i> for 20 min, the supernatants were collected and subjected to HTRF analysis. <i>N</i> = 5–6 for each group. Plots represent mean ± S.E.M. An unpaired two-tailed Student’s <i>t</i> test was used. * <i>P</i> < 0.05. (E) Representative immunostaining images of DARPP32 (top) and Iba1 (bottom) in striatal slices of wild-type or HdhQ140 mice, treated with DMSO or Comp-<b>43</b>, respectively. Mice striata slices were obtained and stained using antibodies against DARPP32 or Iba1, followed by DAPI staining. DARPP32 antibody serves as a specific striatal neuron marker. Scale bar: 50 μm. (F) Quantifications of (E). For each group, six to seven mice were tested with nine images taken from three independent slices. Plots represent mean ± S.E.M. * <i>P</i> < 0.05, ** <i>P</i> < 0.01, *** <i>P</i> < 0.001 as determined by the two-way ANOVA with the post hoc Bonferroni’s multiple comparisons.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01133/20210121/images/large/jm0c01133_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01133&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01133/20210121/images/medium/jm0c01133_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01133/20210121/images/large/jm0c01133_0009.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Model illustrating the function of GPR52 antagonist Comp-<b>43</b>. GPR52-specific antagonist Comp-<b>43</b> reduces mHTT levels via GPR52 and protects striatal neurons from mHTT toxicity, and eventually ameliorates HD-related phenotypes in HdhQ140 mice.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-2/acs.jmedchem.0c01133/20210121/images/large/jm0c01133_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01133&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i76">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_43735" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_43735" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 46 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">The
Huntington’s Disease Collaborative Research Group</span> <span> </span><span class="NLM_article-title">A novel gene containing a trinucleotide
repeat that is expanded and unstable on Huntington’s disease
chromosomes</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>72</i></span>,  <span class="NLM_fpage">971</span>– <span class="NLM_lpage">983</span>, <span class="refDoi"> DOI: 10.1016/0092-8674(93)90585-E</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1016%2F0092-8674%2893%2990585-E" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=8458085" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=1993&pages=971-983&author=The%0AHuntington%E2%80%99s+Disease+Collaborative+Research+Group&title=A+novel+gene+containing+a+trinucleotide%0Arepeat+that+is+expanded+and+unstable+on+Huntington%E2%80%99s+disease%0Achromosomes&doi=10.1016%2F0092-8674%2893%2990585-E"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2F0092-8674%2893%2990585-E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0092-8674%252893%252990585-E%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DA%2520novel%2520gene%2520containing%2520a%2520trinucleotide%250Arepeat%2520that%2520is%2520expanded%2520and%2520unstable%2520on%2520Huntington%25E2%2580%2599s%2520disease%250Achromosomes%26jtitle%3DCell%26date%3D1993%26volume%3D72%26spage%3D971%26epage%3D983%26doi%3D10.1016%2F0092-8674%2893%2990585-E" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Richfield, E. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maguire-Zeiss, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vonkeman, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voorn, P.</span></span> <span> </span><span class="NLM_article-title">Preferential loss of preproenkephalin versus preprotachykinin neurons from the striatum of Huntington’s disease patients</span>. <i>Ann. Neurol.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">852</span>– <span class="NLM_lpage">861</span>, <span class="refDoi"> DOI: 10.1002/ana.410380605</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1002%2Fana.410380605" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=8526457" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A280%3ADyaK287gtFekug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1995&pages=852-861&author=E.+K.+Richfieldauthor=K.+A.+Maguire-Zeissauthor=H.+E.+Vonkemanauthor=P.+Voorn&title=Preferential+loss+of+preproenkephalin+versus+preprotachykinin+neurons+from+the+striatum+of+Huntington%E2%80%99s+disease+patients&doi=10.1002%2Fana.410380605"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Preferential loss of preproenkephalin versus preprotachykinin neurons from the striatum of Huntington's disease patients</span></div><div class="casAuthors">Richfield E K; Maguire-Zeiss K A; Vonkeman H E; Voorn P</div><div class="citationInfo"><span class="NLM_cas:title">Annals of neurology</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">852-61</span>
        ISSN:<span class="NLM_cas:issn">0364-5134</span>.
    </div><div class="casAbstract">Preferential loss of basal ganglia neurons and terminals occurs in Huntington's disease (HD).  Terminals of preproenkephalin medium-size spiny neurons are more vulnerable than terminals of preprotachykinin neurons, but the peptidergic neurons of origin have not yet been shown to die preferentially.  We sought to determine, in the striatum, whether preproenkephalin neurons were lost to a greater extent than preprotachykinin neurons and to determine whether there were decreases in specific messenger RNA (mRNA) levels of preproenkephalin, preprotachykinin, and calbindin D28k.  We found a grade-related decrease in the number of preprotachykinin- and calbindin D28k-labeled neurons per measuring field in the caudate nucleus of patients with HD.  Three measures of the neuronal level of preprotachykinin mRNA were all significantly reduced (6-65% of control values) in HD caudate nucleus.  No decline in calbindin D28k mRNA levels per neuron were found in HD striata compared to control striata.  We found a greater loss of preproenkephalin neurons per field than preprotachyknin neurons per field in the caudate nucleus of HD brains compared to control brains.  Preprotachykinin neurons are lost in HD in a grade-related manner and surviving preprotachykinin neurons are impaired in function.  However, preproenkephalin neurons are lost to a greater extent than preprotachykinin neurons, which may explain preferential changes found in projection regions of the striatum.  Declines in neuropeptide mRNA may be specific in HD, since calbindin D28k mRNA levels were unchanged.  Alterations in the levels of expression of preproenkephalin and preprotachykinin mRNA may be direct or indirect effects of the HD mutation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ4iCgcs9PWF3l8icjszYl0fW6udTcc2eaGtgeYqFg59Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK287gtFekug%253D%253D&md5=01150e69fbb4fd2a415eb7d6bfe526d2</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1002%2Fana.410380605&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fana.410380605%26sid%3Dliteratum%253Aachs%26aulast%3DRichfield%26aufirst%3DE.%2BK.%26aulast%3DMaguire-Zeiss%26aufirst%3DK.%2BA.%26aulast%3DVonkeman%26aufirst%3DH.%2BE.%26aulast%3DVoorn%26aufirst%3DP.%26atitle%3DPreferential%2520loss%2520of%2520preproenkephalin%2520versus%2520preprotachykinin%2520neurons%2520from%2520the%2520striatum%2520of%2520Huntington%25E2%2580%2599s%2520disease%2520patients%26jtitle%3DAnn.%2520Neurol.%26date%3D1995%26volume%3D38%26spage%3D852%26epage%3D861%26doi%3D10.1002%2Fana.410380605" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">DiFiglia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sapp, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chase, K. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bates, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vonsattel, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aronin, N.</span></span> <span> </span><span class="NLM_article-title">Aggregation of huntingtin in neuronal intranuclear inclusions and dystrophic neurites in brain</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>277</i></span>,  <span class="NLM_fpage">1990</span>– <span class="NLM_lpage">1993</span>, <span class="refDoi"> DOI: 10.1126/science.277.5334.1990</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1126%2Fscience.277.5334.1990" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=9302293" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A528%3ADyaK2sXmt12rurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=277&publication_year=1997&pages=1990-1993&author=M.+DiFigliaauthor=E.+Sappauthor=K.+O.+Chaseauthor=S.+W.+Daviesauthor=G.+P.+Batesauthor=J.+P.+Vonsattelauthor=N.+Aronin&title=Aggregation+of+huntingtin+in+neuronal+intranuclear+inclusions+and+dystrophic+neurites+in+brain&doi=10.1126%2Fscience.277.5334.1990"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Aggregation of huntingtin in neuronal intranuclear inclusions and dystrophic neurites in brain</span></div><div class="casAuthors">DiFiglia, Marian; Sapp, Ellen; Chase, Kathryn O.; Davies, Stephen W.; Bates, Gillian; Vonsattel, J. P.; Aronin, Neil</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">277</span>
        (<span class="NLM_cas:issue">5334</span>),
    <span class="NLM_cas:pages">1990-1993</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The cause of neurodegeneration in Huntington's disease (HD) is unknown.  Patients with HD have an expanded NH2-terminal polyglutamine region in huntingtin.  An NH2-terminal fragment of mutant huntingtin was localized to neuronal intranuclear inclusions (NIIs) and dystrophic neurites (DNs) in the HD cortex and striatum, which are affected in HD, and polyglutamine length influenced the extent of huntingtin accumulation in these structures.  Ubiquitin was also found in NIIs and DNs, which suggests that abnormal huntingtin is targeted for proteolysis but is resistant to removal.  The aggregation of mutant huntingtin may be part of the pathogenic mechanism in HD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3fQHwJi2eAbVg90H21EOLACvtfcHk0lgkiJ6FcvVJKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXmt12rurs%253D&md5=fc9fa3c298625e623b3df0fe144ab2ba</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1126%2Fscience.277.5334.1990&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.277.5334.1990%26sid%3Dliteratum%253Aachs%26aulast%3DDiFiglia%26aufirst%3DM.%26aulast%3DSapp%26aufirst%3DE.%26aulast%3DChase%26aufirst%3DK.%2BO.%26aulast%3DDavies%26aufirst%3DS.%2BW.%26aulast%3DBates%26aufirst%3DG.%2BP.%26aulast%3DVonsattel%26aufirst%3DJ.%2BP.%26aulast%3DAronin%26aufirst%3DN.%26atitle%3DAggregation%2520of%2520huntingtin%2520in%2520neuronal%2520intranuclear%2520inclusions%2520and%2520dystrophic%2520neurites%2520in%2520brain%26jtitle%3DScience%26date%3D1997%26volume%3D277%26spage%3D1990%26epage%3D1993%26doi%3D10.1126%2Fscience.277.5334.1990" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miller, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrasate, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brooks, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Libeu, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Legleiter, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatters, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curtis, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krishnan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitra, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Widjaja, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaby, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lotz, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Newhouse, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osmand, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thulasiramin, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saudou, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Segal, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masliah, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muchowski, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weisgraber, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finkbeiner, S.</span></span> <span> </span><span class="NLM_article-title">Identifying polyglutamine protein species in situ that best predict neurodegeneration</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">925</span>– <span class="NLM_lpage">934</span>, <span class="refDoi"> DOI: 10.1038/nchembio.694</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1038%2Fnchembio.694" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=22037470" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlyisrfE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=925-934&author=J.+Millerauthor=M.+Arrasateauthor=E.+Brooksauthor=C.+P.+Libeuauthor=J.+Legleiterauthor=D.+Hattersauthor=J.+Curtisauthor=K.+Cheungauthor=P.+Krishnanauthor=S.+Mitraauthor=K.+Widjajaauthor=B.+A.+Shabyauthor=G.+P.+Lotzauthor=Y.+Newhouseauthor=E.+J.+Mitchellauthor=A.+Osmandauthor=M.+Grayauthor=V.+Thulasiraminauthor=F.+Saudouauthor=M.+Segalauthor=X.+W.+Yangauthor=E.+Masliahauthor=L.+M.+Thompsonauthor=P.+J.+Muchowskiauthor=K.+H.+Weisgraberauthor=S.+Finkbeiner&title=Identifying+polyglutamine+protein+species+in+situ+that+best+predict+neurodegeneration&doi=10.1038%2Fnchembio.694"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Identifying polyglutamine protein species in situ that best predict neurodegeneration</span></div><div class="casAuthors">Miller, Jason; Arrasate, Montserrat; Brooks, Elizabeth; Libeu, Clare Peters; Legleiter, Justin; Hatters, Danny; Curtis, Jessica; Cheung, Kenneth; Krishnan, Preethi; Mitra, Siddhartha; Widjaja, Kartika; Shaby, Benjamin A.; Lotz, Gregor P.; Newhouse, Yvonne; Mitchell, Emily J.; Osmand, Alex; Gray, Michelle; Thulasiramin, Vanitha; Saudou, Frederic; Segal, Mark; Yang, X. William; Masliah, Eliezer; Thompson, Leslie M.; Muchowski, Paul J.; Weisgraber, Karl H.; Finkbeiner, Steven</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">925-934</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Polyglutamine (polyQ) stretches exceeding a threshold length confer a toxic function to proteins that contain them and cause at least nine neurol. disorders.  The basis for this toxicity threshold is unclear.  Although polyQ expansions render proteins prone to aggregate into inclusion bodies, this may be a neuronal coping response to more toxic forms of polyQ.  The exact structure of these more toxic forms is unknown.  Here the authors show that the monoclonal antibody 3B5H10 recognizes a species of polyQ protein in situ that strongly predicts neuronal death.  The epitope selectively appears among some of the many low-mol.-wt. conformational states assumed by expanded polyQ and disappears in higher-mol.-wt. aggregated forms, such as inclusion bodies.  These results suggest that protein monomers and possibly small oligomers contg. expanded polyQ stretches can adopt a conformation that is recognized by 3B5H10 and is toxic or closely related to a toxic species.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowzCQPLhzGSrVg90H21EOLACvtfcHk0lgtt2bgdygXuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlyisrfE&md5=a0d31dabd25c57c5c2fa5aaf294ba555</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.694&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.694%26sid%3Dliteratum%253Aachs%26aulast%3DMiller%26aufirst%3DJ.%26aulast%3DArrasate%26aufirst%3DM.%26aulast%3DBrooks%26aufirst%3DE.%26aulast%3DLibeu%26aufirst%3DC.%2BP.%26aulast%3DLegleiter%26aufirst%3DJ.%26aulast%3DHatters%26aufirst%3DD.%26aulast%3DCurtis%26aufirst%3DJ.%26aulast%3DCheung%26aufirst%3DK.%26aulast%3DKrishnan%26aufirst%3DP.%26aulast%3DMitra%26aufirst%3DS.%26aulast%3DWidjaja%26aufirst%3DK.%26aulast%3DShaby%26aufirst%3DB.%2BA.%26aulast%3DLotz%26aufirst%3DG.%2BP.%26aulast%3DNewhouse%26aufirst%3DY.%26aulast%3DMitchell%26aufirst%3DE.%2BJ.%26aulast%3DOsmand%26aufirst%3DA.%26aulast%3DGray%26aufirst%3DM.%26aulast%3DThulasiramin%26aufirst%3DV.%26aulast%3DSaudou%26aufirst%3DF.%26aulast%3DSegal%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DX.%2BW.%26aulast%3DMasliah%26aufirst%3DE.%26aulast%3DThompson%26aufirst%3DL.%2BM.%26aulast%3DMuchowski%26aufirst%3DP.%2BJ.%26aulast%3DWeisgraber%26aufirst%3DK.%2BH.%26aulast%3DFinkbeiner%26aufirst%3DS.%26atitle%3DIdentifying%2520polyglutamine%2520protein%2520species%2520in%2520situ%2520that%2520best%2520predict%2520neurodegeneration%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2011%26volume%3D7%26spage%3D925%26epage%3D934%26doi%3D10.1038%2Fnchembio.694" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucas, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hen, R.</span></span> <span> </span><span class="NLM_article-title">Reversal of neuropathology and motor dysfunction in a conditional model of Huntington’s disease</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>101</i></span>,  <span class="NLM_fpage">57</span>– <span class="NLM_lpage">66</span>, <span class="refDoi"> DOI: 10.1016/S0092-8674(00)80623-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1016%2FS0092-8674%2800%2980623-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10778856" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A528%3ADC%252BD3cXisVaitbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2000&pages=57-66&author=A.+Yamamotoauthor=J.+J.+Lucasauthor=R.+Hen&title=Reversal+of+neuropathology+and+motor+dysfunction+in+a+conditional+model+of+Huntington%E2%80%99s+disease&doi=10.1016%2FS0092-8674%2800%2980623-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Reversal of neuropathology and motor dysfunction in a conditional model of Huntington's disease</span></div><div class="casAuthors">Yamamoto, Ai; Lucas, Jose J.; Hen, Rene</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, Massachusetts)</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">101</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">57-66</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Neurodegenerative disorders like Huntington's disease (HD) are characterized by progressive and putative irreversible clin. and neuropathol. symptoms, including neuronal protein aggregates.  Conditional transgenic models of neurodegenerative diseases therefore could be a powerful means to explore the relationship between mutant protein expression and progression of the disease.  We have created a conditional model of HD by using the tet-regulatable system.  Mice expressing a mutated huntingtin fragment demonstrate neuronal inclusions, characteristic neuropathol., and progressive motor dysfunction.  Blockade of expression in symptomatic mice leads to a disappearance of inclusions and an amelioration of the behavioral phenotype.  We thus demonstrate that a continuous influx of the mutant protein is required to maintain inclusions and symptoms, raising the possibility that HD may be reversible.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpABup9-4sWj7Vg90H21EOLACvtfcHk0lgtt2bgdygXuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXisVaitbk%253D&md5=589e78e8d13075710115e67df6e5ac17</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2FS0092-8674%2800%2980623-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0092-8674%252800%252980623-6%26sid%3Dliteratum%253Aachs%26aulast%3DYamamoto%26aufirst%3DA.%26aulast%3DLucas%26aufirst%3DJ.%2BJ.%26aulast%3DHen%26aufirst%3DR.%26atitle%3DReversal%2520of%2520neuropathology%2520and%2520motor%2520dysfunction%2520in%2520a%2520conditional%2520model%2520of%2520Huntington%25E2%2580%2599s%2520disease%26jtitle%3DCell%26date%3D2000%26volume%3D101%26spage%3D57%26epage%3D66%26doi%3D10.1016%2FS0092-8674%2800%2980623-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantle, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holley, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greiner, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shirasaki, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cepeda, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levine, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X. W.</span></span> <span> </span><span class="NLM_article-title">Neuronal targets for reducing mutant huntingtin expression to ameliorate disease in a mouse model of Huntington’s disease</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">536</span>– <span class="NLM_lpage">541</span>, <span class="refDoi"> DOI: 10.1038/nm.3514</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1038%2Fnm.3514" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=24784230" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A528%3ADC%252BC2cXntlehs78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2014&pages=536-541&author=N.+Wangauthor=M.+Grayauthor=X.+H.+Luauthor=J.+P.+Cantleauthor=S.+M.+Holleyauthor=E.+Greinerauthor=X.+Guauthor=D.+Shirasakiauthor=C.+Cepedaauthor=Y.+Liauthor=H.+Dongauthor=M.+S.+Levineauthor=X.+W.+Yang&title=Neuronal+targets+for+reducing+mutant+huntingtin+expression+to+ameliorate+disease+in+a+mouse+model+of+Huntington%E2%80%99s+disease&doi=10.1038%2Fnm.3514"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Neuronal targets for reducing mutant huntingtin expression to ameliorate disease in a mouse model of Huntington's disease</span></div><div class="casAuthors">Wang, Nan; Gray, Michelle; Lu, Xiao-Hong; Cantle, Jeffrey P.; Holley, Sandra M.; Greiner, Erin; Gu, Xiaofeng; Shirasaki, Dyna; Cepeda, Carlos; Li, Yuqing; Dong, Hongwei; Levine, Michael S.; Yang, X. William</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">536-541</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Huntington's disease (HD) is a fatal dominantly inherited neurodegenerative disorder caused by a CAG repeat expansion leading to an elongated polyglutamine stretch in huntingtin.  Mutant huntingtin (mHTT) is ubiquitously expressed in all cells but elicits selective cortical and striatal neurodegeneration in HD.  The mechanistic basis for such selective neuronal vulnerability remains unclear.  A necessary step toward resolving this enigma is to define the cell types in which mHTT expression is causally linked to the disease pathogenesis.  Using a conditional transgenic mouse model of HD, in which the mice express full-length human mHTT from a bacterial artificial chromosome transgene (BACHD), we genetically reduced mHTT expression in neuronal populations in the striatum, cortex or both.  We show that redn. of cortical mHTT expression in BACHD mice partially improves motor and psychiatric-like behavioral deficits but does not improve neurodegeneration, whereas redn. of mHTT expression in both neuronal populations consistently ameliorates all behavioral deficits and selective brain atrophy in this HD model.  Furthermore, whereas redn. of mHTT expression in cortical or striatal neurons partially ameliorates corticostriatal synaptic deficits, further restoration of striatal synaptic function can be achieved by redn. of mHTT expression in both neuronal cell types.  Our study demonstrates distinct but interacting roles of cortical and striatal mHTT in HD pathogenesis and suggests that optimal HD therapeutics may require targeting mHTT in both cortical and striatal neurons.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOHZ5lk0IQqrVg90H21EOLACvtfcHk0lg7vXogCYv_dw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXntlehs78%253D&md5=1fbb71d02f95360589804425aabe04c0</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnm.3514&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.3514%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DN.%26aulast%3DGray%26aufirst%3DM.%26aulast%3DLu%26aufirst%3DX.%2BH.%26aulast%3DCantle%26aufirst%3DJ.%2BP.%26aulast%3DHolley%26aufirst%3DS.%2BM.%26aulast%3DGreiner%26aufirst%3DE.%26aulast%3DGu%26aufirst%3DX.%26aulast%3DShirasaki%26aufirst%3DD.%26aulast%3DCepeda%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DLevine%26aufirst%3DM.%2BS.%26aulast%3DYang%26aufirst%3DX.%2BW.%26atitle%3DNeuronal%2520targets%2520for%2520reducing%2520mutant%2520huntingtin%2520expression%2520to%2520ameliorate%2520disease%2520in%2520a%2520mouse%2520model%2520of%2520Huntington%25E2%2580%2599s%2520disease%26jtitle%3DNat.%2520Med.%26date%3D2014%26volume%3D20%26spage%3D536%26epage%3D541%26doi%3D10.1038%2Fnm.3514" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sha, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Difiglia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mei, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fei, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, B.</span></span> <span> </span><span class="NLM_article-title">Allele-selective lowering of mutant HTT protein by HTT-LC3 linker compounds</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>575</i></span>,  <span class="NLM_fpage">203</span>– <span class="NLM_lpage">209</span>, <span class="refDoi"> DOI: 10.1038/s41586-019-1722-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1038%2Fs41586-019-1722-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=31666698" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitV2hsb3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=575&publication_year=2019&pages=203-209&author=Z.+Liauthor=C.+Wangauthor=Z.+Wangauthor=C.+Zhuauthor=J.+Liauthor=T.+Shaauthor=L.+Maauthor=C.+Gaoauthor=Y.+Yangauthor=Y.+Sunauthor=J.+Wangauthor=X.+Sunauthor=C.+Luauthor=M.+Difigliaauthor=Y.+Meiauthor=C.+Dingauthor=S.+Luoauthor=Y.+Dangauthor=Y.+Dingauthor=Y.+Feiauthor=B.+Lu&title=Allele-selective+lowering+of+mutant+HTT+protein+by+HTT-LC3+linker+compounds&doi=10.1038%2Fs41586-019-1722-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Allele-selective lowering of mutant HTT protein by HTT-LC3 linker compounds</span></div><div class="casAuthors">Li, Zhaoyang; Wang, Cen; Wang, Ziying; Zhu, Chenggang; Li, Jie; Sha, Tian; Ma, Lixiang; Gao, Chao; Yang, Yi; Sun, Yimin; Wang, Jian; Sun, Xiaoli; Lu, Chenqi; Difiglia, Marian; Mei, Yanai; Ding, Chen; Luo, Shouqing; Dang, Yongjun; Ding, Yu; Fei, Yiyan; Lu, Boxun</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">575</span>
        (<span class="NLM_cas:issue">7781</span>),
    <span class="NLM_cas:pages">203-209</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Accumulation of mutant proteins is a major cause of many diseases (collectively called proteopathies), and lowering the level of these proteins can be useful for treatment of these diseases.  We hypothesized that compds. that interact with both the autophagosome protein microtubule-assocd. protein 1A/1B light chain 3 (LC3)1 and the disease-causing protein may target the latter for autophagic clearance.  Mutant huntingtin protein (mHTT) contains an expanded polyglutamine (polyQ) tract and causes Huntington's disease, an incurable neurodegenerative disorder2.  Here, using small-mol.-microarray-based screening, we identified four compds. that interact with both LC3 and mHTT, but not with the wild-type HTT protein.  Some of these compds. targeted mHTT to autophagosomes, reduced mHTT levels in an allele-selective manner, and rescued disease-relevant phenotypes in cells and in vivo in fly and mouse models of Huntington's disease.  We further show that these compds. interact with the expanded polyQ stretch and could lower the level of mutant ataxin-3 (ATXN3), another disease-causing protein with an expanded polyQ tract3.  This study presents candidate compds. for lowering mHTT and potentially other disease-causing proteins with polyQ expansions, demonstrating the concept of lowering levels of disease-causing proteins using autophagosome-tethering compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7rSfxy_Wl7rVg90H21EOLACvtfcHk0lg7vXogCYv_dw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitV2hsb3O&md5=98cba22501f8157c83c0181b74ca15e2</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fs41586-019-1722-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-019-1722-1%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DZhu%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DSha%26aufirst%3DT.%26aulast%3DMa%26aufirst%3DL.%26aulast%3DGao%26aufirst%3DC.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DSun%26aufirst%3DX.%26aulast%3DLu%26aufirst%3DC.%26aulast%3DDifiglia%26aufirst%3DM.%26aulast%3DMei%26aufirst%3DY.%26aulast%3DDing%26aufirst%3DC.%26aulast%3DLuo%26aufirst%3DS.%26aulast%3DDang%26aufirst%3DY.%26aulast%3DDing%26aufirst%3DY.%26aulast%3DFei%26aufirst%3DY.%26aulast%3DLu%26aufirst%3DB.%26atitle%3DAllele-selective%2520lowering%2520of%2520mutant%2520HTT%2520protein%2520by%2520HTT-LC3%2520linker%2520compounds%26jtitle%3DNature%26date%3D2019%26volume%3D575%26spage%3D203%26epage%3D209%26doi%3D10.1038%2Fs41586-019-1722-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Ramahi, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valencia, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berenshteyn, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallego-Flores, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ichcho, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lacoste, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hild, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Difiglia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Botas, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palacino, J.</span></span> <span> </span><span class="NLM_article-title">Identification of NUB1 as a suppressor of mutant Huntington toxicity via enhanced protein clearance</span>. <i>Nat. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">562</span>– <span class="NLM_lpage">570</span>, <span class="refDoi"> DOI: 10.1038/nn.3367</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1038%2Fnn.3367" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=23525043" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A528%3ADC%252BC3sXksVOjsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2013&pages=562-570&author=B.+Luauthor=I.+Al-Ramahiauthor=A.+Valenciaauthor=Q.+Wangauthor=F.+Berenshteynauthor=H.+Yangauthor=T.+Gallego-Floresauthor=S.+Ichchoauthor=A.+Lacosteauthor=M.+Hildauthor=M.+Difigliaauthor=J.+Botasauthor=J.+Palacino&title=Identification+of+NUB1+as+a+suppressor+of+mutant+Huntington+toxicity+via+enhanced+protein+clearance&doi=10.1038%2Fnn.3367"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of NUB1 as a suppressor of mutant Huntingtin toxicity via enhanced protein clearance</span></div><div class="casAuthors">Lu, Boxun; Al-Ramahi, Ismael; Valencia, Antonio; Wang, Qiong; Berenshteyn, Frada; Yang, Haidi; Gallego-Flores, Tatiana; Ichcho, Salah; Lacoste, Arnaud; Hild, Marc; DiFiglia, Marian; Botas, Juan; Palacino, James</div><div class="citationInfo"><span class="NLM_cas:title">Nature Neuroscience</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">562-570</span>CODEN:
                <span class="NLM_cas:coden">NANEFN</span>;
        ISSN:<span class="NLM_cas:issn">1097-6256</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Huntington's disease is caused by expanded CAG repeats in HTT, conferring toxic gain of function on mutant HTT (mHTT) protein.  Reducing mHTT amts. is postulated as a strategy for therapeutic intervention.  We conducted genome-wide RNA interference screens for genes modifying mHTT abundance and identified 13 hits.  We tested 10 in vivo in a Drosophila melanogaster Huntington's disease model, and 6 exhibited activity consistent with the in vitro screening results.  Among these, neg. regulator of ubiquitin-like protein 1 (NUB1) overexpression lowered mHTT in neuronal models and rescued mHTT-induced death.  NUB1 reduces mHTT amts. by enhancing polyubiquitination and proteasomal degrdn. of mHTT protein.  The process requires CUL3 and the ubiquitin-like protein NEDD8 necessary for CUL3 activation.  As a potential approach to modulating NUB1 for treatment, interferon-β lowered mHTT and rescued neuronal toxicity through induction of NUB1.  Thus, we have identified genes modifying endogenous mHTT using high-throughput screening and demonstrate NUB1 as an exemplar entry point for therapeutic intervention of Huntington's disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsSr4IEp1EWLVg90H21EOLACvtfcHk0lg7vXogCYv_dw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXksVOjsb4%253D&md5=fd90b314dab577d18223753b40504eca</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnn.3367&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnn.3367%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DB.%26aulast%3DAl-Ramahi%26aufirst%3DI.%26aulast%3DValencia%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DBerenshteyn%26aufirst%3DF.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DGallego-Flores%26aufirst%3DT.%26aulast%3DIchcho%26aufirst%3DS.%26aulast%3DLacoste%26aufirst%3DA.%26aulast%3DHild%26aufirst%3DM.%26aulast%3DDifiglia%26aufirst%3DM.%26aulast%3DBotas%26aufirst%3DJ.%26aulast%3DPalacino%26aufirst%3DJ.%26atitle%3DIdentification%2520of%2520NUB1%2520as%2520a%2520suppressor%2520of%2520mutant%2520Huntington%2520toxicity%2520via%2520enhanced%2520protein%2520clearance%26jtitle%3DNat.%2520Neurosci.%26date%3D2013%26volume%3D16%26spage%3D562%26epage%3D570%26doi%3D10.1038%2Fnn.3367" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abramowski, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bibel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bodner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chopra, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DiFiglia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fox, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kegel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grueninger, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hersch, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Housman, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Regulier, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosas, H. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stefani, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeitlin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bilbe, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paganetti, P.</span></span> <span> </span><span class="NLM_article-title">Single-step detection of mutant huntingtin in animal and human tissues: a bioassay for Huntington’s disease</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>395</i></span>,  <span class="NLM_fpage">8</span>– <span class="NLM_lpage">15</span>, <span class="refDoi"> DOI: 10.1016/j.ab.2009.08.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1016%2Fj.ab.2009.08.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=19664996" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFKnsb7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=395&publication_year=2009&pages=8-15&author=A.+Weissauthor=D.+Abramowskiauthor=M.+Bibelauthor=R.+Bodnerauthor=V.+Chopraauthor=M.+DiFigliaauthor=J.+Foxauthor=K.+Kegelauthor=C.+Kleinauthor=S.+Grueningerauthor=S.+Herschauthor=D.+Housmanauthor=E.+Regulierauthor=H.+D.+Rosasauthor=M.+Stefaniauthor=S.+Zeitlinauthor=G.+Bilbeauthor=P.+Paganetti&title=Single-step+detection+of+mutant+huntingtin+in+animal+and+human+tissues%3A+a+bioassay+for+Huntington%E2%80%99s+disease&doi=10.1016%2Fj.ab.2009.08.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Single-step detection of mutant huntingtin in animal and human tissues: A bioassay for Huntington's disease</span></div><div class="casAuthors">Weiss, Andreas; Abramowski, Dorothee; Bibel, Miriam; Bodner, Ruth; Chopra, Vanita; DiFiglia, Marian; Fox, Jonathan; Kegel, Kimberly; Klein, Corinna; Grueninger, Stephan; Hersch, Steven; Housman, David; Regulier, Etienne; Rosas, H. Diana; Stefani, Muriel; Zeitlin, Scott; Bilbe, Graeme; Paganetti, Paolo</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">395</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">8-15</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The genetic mutation causing Huntington's disease is a polyglutamine expansion in the huntingtin protein where more than 37 glutamines cause disease by formation of toxic intracellular fragments, aggregates, and cell death.  Despite a clear pathogenic role for mutant huntingtin, understanding huntingtin expression during the presymptomatic phase of the disease or during disease progression has remained obscure.  Central to clarifying the role in the pathomechanism of disease is the ability to easily and accurately measure mutant huntingtin in accessible human tissue samples as well as cell and animal models.  Here the authors describe a highly sensitive time-resolved Forster resonance energy transfer (FRET) assay for quantification of sol. mutant huntingtin in brain, plasma, and cerebrospinal fluid.  Surprisingly, in mice, sol. huntingtin levels decrease during disease progression, inversely correlating with brain aggregate load.  Mutant huntingtin is easily detected in human brain and blood-derived fractions, providing a utility to assess mutant huntingtin expression during disease course as well as a pharmacodynamic marker for disease-modifying therapeutics targeting expression, cleavage, or degrdn. of mutant huntingtin.  The design of the homogeneous one-step method for huntingtin detection is such that it can be easily applied to measure other proteins of interest.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrla9646FajzrVg90H21EOLACvtfcHk0lgUWGHfiyNUEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFKnsb7I&md5=2fe224880e4761ec0566994b18353c8c</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.ab.2009.08.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ab.2009.08.001%26sid%3Dliteratum%253Aachs%26aulast%3DWeiss%26aufirst%3DA.%26aulast%3DAbramowski%26aufirst%3DD.%26aulast%3DBibel%26aufirst%3DM.%26aulast%3DBodner%26aufirst%3DR.%26aulast%3DChopra%26aufirst%3DV.%26aulast%3DDiFiglia%26aufirst%3DM.%26aulast%3DFox%26aufirst%3DJ.%26aulast%3DKegel%26aufirst%3DK.%26aulast%3DKlein%26aufirst%3DC.%26aulast%3DGrueninger%26aufirst%3DS.%26aulast%3DHersch%26aufirst%3DS.%26aulast%3DHousman%26aufirst%3DD.%26aulast%3DRegulier%26aufirst%3DE.%26aulast%3DRosas%26aufirst%3DH.%2BD.%26aulast%3DStefani%26aufirst%3DM.%26aulast%3DZeitlin%26aufirst%3DS.%26aulast%3DBilbe%26aufirst%3DG.%26aulast%3DPaganetti%26aufirst%3DP.%26atitle%3DSingle-step%2520detection%2520of%2520mutant%2520huntingtin%2520in%2520animal%2520and%2520human%2520tissues%253A%2520a%2520bioassay%2520for%2520Huntington%25E2%2580%2599s%2520disease%26jtitle%3DAnal.%2520Biochem.%26date%3D2009%26volume%3D395%26spage%3D8%26epage%3D15%26doi%3D10.1016%2Fj.ab.2009.08.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Harper, S. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Staber, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eliason, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martins, I. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kotin, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulson, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davidson, B. L.</span></span> <span> </span><span class="NLM_article-title">RNA interference improves motor and neuropathological abnormalities in a Huntington’s disease mouse model</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>102</i></span>,  <span class="NLM_fpage">5820</span>– <span class="NLM_lpage">5825</span>, <span class="refDoi"> DOI: 10.1073/pnas.0501507102</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1073%2Fpnas.0501507102" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=15811941" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A528%3ADC%252BD2MXjslaktbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2005&pages=5820-5825&author=S.+Q.+Harperauthor=P.+D.+Staberauthor=X.+Heauthor=S.+L.+Eliasonauthor=I.+H.+Martinsauthor=Q.+Maoauthor=L.+Yangauthor=R.+M.+Kotinauthor=H.+L.+Paulsonauthor=B.+L.+Davidson&title=RNA+interference+improves+motor+and+neuropathological+abnormalities+in+a+Huntington%E2%80%99s+disease+mouse+model&doi=10.1073%2Fpnas.0501507102"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">RNA interference improves motor and neuropathological abnormalities in a Huntington's disease mouse model</span></div><div class="casAuthors">Harper, Scott Q.; Staber, Patrick D.; He, Xiaohua; Eliason, Steven L.; Martins, Ines H.; Mao, Qinwen; Yang, Linda; Kotin, Robert M.; Paulson, Henry L.; Davidson, Beverly L.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">5820-5825</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Huntington's disease (HD) is a fatal, dominant neurogenetic disorder.  HD results from polyglutamine repeat expansion (CAG codon, Q) in exon 1 of HD, conferring a toxic gain of function on the protein huntingtin (htt).  Currently, no preventative treatment exists for HD.  RNA interference (RNAi) has emerged as a potential therapeutic tool for treating dominant diseases by directly reducing disease gene expression.  Here, authors show that RNAi directed against mutant human htt reduced htt mRNA and protein expression in cell culture and in HD mouse brain.  Importantly, htt gene silencing improved behavioral and neuropathol. abnormalities assocd. with HD.  These data provide support for the further development of RNAi for HD therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYayMgT1-X7bVg90H21EOLACvtfcHk0lgUWGHfiyNUEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXjslaktbo%253D&md5=7a70a4e0ff4411d952e94bce0a671e56</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0501507102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0501507102%26sid%3Dliteratum%253Aachs%26aulast%3DHarper%26aufirst%3DS.%2BQ.%26aulast%3DStaber%26aufirst%3DP.%2BD.%26aulast%3DHe%26aufirst%3DX.%26aulast%3DEliason%26aufirst%3DS.%2BL.%26aulast%3DMartins%26aufirst%3DI.%2BH.%26aulast%3DMao%26aufirst%3DQ.%26aulast%3DYang%26aufirst%3DL.%26aulast%3DKotin%26aufirst%3DR.%2BM.%26aulast%3DPaulson%26aufirst%3DH.%2BL.%26aulast%3DDavidson%26aufirst%3DB.%2BL.%26atitle%3DRNA%2520interference%2520improves%2520motor%2520and%2520neuropathological%2520abnormalities%2520in%2520a%2520Huntington%25E2%2580%2599s%2520disease%2520mouse%2520model%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2005%26volume%3D102%26spage%3D5820%26epage%3D5825%26doi%3D10.1073%2Fpnas.0501507102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez-Lebron, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denovan-Wright, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nash, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewin, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mandel, R. J.</span></span> <span> </span><span class="NLM_article-title">Intrastriatal rAAV-mediated delivery of anti-huntingtin shRNAs induces partial reversal of disease progression in R6/1 Huntington’s disease transgenic mice</span>. <i>Mol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">618</span>– <span class="NLM_lpage">633</span>, <span class="refDoi"> DOI: 10.1016/j.ymthe.2005.05.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1016%2Fj.ymthe.2005.05.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=16019264" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtVWgsbzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2005&pages=618-633&author=E.+Rodriguez-Lebronauthor=E.+M.+Denovan-Wrightauthor=K.+Nashauthor=A.+S.+Lewinauthor=R.+J.+Mandel&title=Intrastriatal+rAAV-mediated+delivery+of+anti-huntingtin+shRNAs+induces+partial+reversal+of+disease+progression+in+R6%2F1+Huntington%E2%80%99s+disease+transgenic+mice&doi=10.1016%2Fj.ymthe.2005.05.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Intrastriatal rAAV-mediated delivery of anti-huntingtin shRNAs induces partial reversal of disease progression in R6/1 Huntington's disease transgenic mice</span></div><div class="casAuthors">Rodriguez-Lebron, Edgardo; Denovan-Wright, Eileen M.; Nash, Kevin; Lewin, Alfred S.; Mandel, Ronald J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Therapy</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">618-633</span>CODEN:
                <span class="NLM_cas:coden">MTOHCK</span>;
        ISSN:<span class="NLM_cas:issn">1525-0016</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Huntington's disease (HD) is a fatal neurodegenerative disorder caused by the presence of an abnormally expanded polyglutamine domain in the N-terminus of huntingtin.  The authors developed a recombinant adeno-assocd. viral serotype 5 (rAAV5) gene transfer strategy to posttranscriptionally suppress the levels of striatal mutant huntingtin (mHtt) in the R6/1 HD transgenic mouse via RNA interference.  Transient cotransfection of HEK293 cells with plasmids expressing a portion of human mHtt derived from R6/1 transgenic HD mice and a short-hairpin RNA directed against the 5' UTR of the mHtt mRNA (siHUNT-1) resulted in redn. in the levels of mHtt mRNA (-75%) and protein (-60%).  Long-term in vivo rAAV5-mediated expression of siHUNT-1 in the striatum of R6/1 mice reduced the levels of mHtt mRNA (-78%) and protein (-28%) as detd. by quant. RT-PCR and Western blot anal., resp.  The redn. in mHtt was concomitant with a redn. in the size and no. of neuronal intranuclear inclusions and a small but significant normalization of the steady-state levels of preproenkephalin and dopamine- and cAMP-responsive phosphoprotein 32 kDa mRNA.  Finally, bilateral expression of rAAV5-siHUNT-1 resulted in delayed onset of the rear paw clasping phenotype exhibited by the R6/1 mice.  These results suggest that a redn. in the levels of striatal mHtt can ameliorate the HD phenotype of R6/1 mice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaTvJJnssVGrVg90H21EOLACvtfcHk0lgUWGHfiyNUEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtVWgsbzL&md5=abf2a0184528767add5547fd471df9fc</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.ymthe.2005.05.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ymthe.2005.05.006%26sid%3Dliteratum%253Aachs%26aulast%3DRodriguez-Lebron%26aufirst%3DE.%26aulast%3DDenovan-Wright%26aufirst%3DE.%2BM.%26aulast%3DNash%26aufirst%3DK.%26aulast%3DLewin%26aufirst%3DA.%2BS.%26aulast%3DMandel%26aufirst%3DR.%2BJ.%26atitle%3DIntrastriatal%2520rAAV-mediated%2520delivery%2520of%2520anti-huntingtin%2520shRNAs%2520induces%2520partial%2520reversal%2520of%2520disease%2520progression%2520in%2520R6%252F1%2520Huntington%25E2%2580%2599s%2520disease%2520transgenic%2520mice%26jtitle%3DMol.%2520Ther.%26date%3D2005%26volume%3D12%26spage%3D618%26epage%3D633%26doi%3D10.1016%2Fj.ymthe.2005.05.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">DiFiglia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sena-Esteves, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chase, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sapp, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pfister, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sass, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoder, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reeves, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pandey, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajeev, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manoharan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sah, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zamore, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aronin, N.</span></span> <span> </span><span class="NLM_article-title">Therapeutic silencing of mutant huntingtin with siRNA attenuates striatal and cortical neuropathology and behavioral deficits</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>104</i></span>,  <span class="NLM_fpage">17204</span>– <span class="NLM_lpage">17209</span>, <span class="refDoi"> DOI: 10.1073/pnas.0708285104</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1073%2Fpnas.0708285104" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=17940007" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A528%3ADC%252BD2sXht1Kit7rK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2007&pages=17204-17209&author=M.+DiFigliaauthor=M.+Sena-Estevesauthor=K.+Chaseauthor=E.+Sappauthor=E.+Pfisterauthor=M.+Sassauthor=J.+Yoderauthor=P.+Reevesauthor=R.+K.+Pandeyauthor=K.+G.+Rajeevauthor=M.+Manoharanauthor=D.+W.+Sahauthor=P.+D.+Zamoreauthor=N.+Aronin&title=Therapeutic+silencing+of+mutant+huntingtin+with+siRNA+attenuates+striatal+and+cortical+neuropathology+and+behavioral+deficits&doi=10.1073%2Fpnas.0708285104"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Therapeutic silencing of mutant huntingtin with siRNA attenuates striatal and cortical neuropathology and behavioral deficits</span></div><div class="casAuthors">DiFiglia, M.; Sena-Esteves, M.; Chase, K.; Sapp, E.; Pfister, E.; Sass, M.; Yoder, J.; Reeves, P.; Pandey, R. K.; Rajeev, K. G.; Manoharan, M.; Sah, D. W. Y.; Zamore, P. D.; Aronin, N.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">43</span>),
    <span class="NLM_cas:pages">17204-17209</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Huntington's disease (HD) is a neurodegenerative disorder caused by expansion of a CAG repeat in the huntingtin (Htt) gene.  HD is autosomal dominant and, in theory, amenable to therapeutic RNA silencing.  We introduced cholesterol-conjugated small interfering RNA duplexes (cc-siRNA) targeting human Htt mRNA (siRNA-Htt) into mouse striata that also received adeno-assocd. virus contg. either expanded (100 CAG) or wild-type (18 CAG) Htt cDNA encoding huntingtin (Htt) 1-400.  Adeno-assocd. virus delivery to striatum and overlying cortex of the mutant Htt gene, but not the wild type, produced neuropathol. and motor deficits.  Treatment with cc-siRNA-Htt in mice with mutant Htt prolonged survival of striatal neurons, reduced neuropil aggregates, diminished inclusion size, and lowered the frequency of clasping and footslips on balance beam. cc-siRNA-Htt was designed to target human wild-type and mutant Htt and decreased levels of both in the striatum.  Our findings indicate that a single administration into the adult striatum of an siRNA targeting Htt can silence mutant Htt, attenuate neuronal pathol., and delay the abnormal behavioral phenotype obsd. in a rapid-onset, viral transgenic mouse model of HD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrQOhggGghq7Vg90H21EOLACvtfcHk0lgHtF2rXzT5UA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXht1Kit7rK&md5=c14be5978ed75dec51270a54b7f566e2</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0708285104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0708285104%26sid%3Dliteratum%253Aachs%26aulast%3DDiFiglia%26aufirst%3DM.%26aulast%3DSena-Esteves%26aufirst%3DM.%26aulast%3DChase%26aufirst%3DK.%26aulast%3DSapp%26aufirst%3DE.%26aulast%3DPfister%26aufirst%3DE.%26aulast%3DSass%26aufirst%3DM.%26aulast%3DYoder%26aufirst%3DJ.%26aulast%3DReeves%26aufirst%3DP.%26aulast%3DPandey%26aufirst%3DR.%2BK.%26aulast%3DRajeev%26aufirst%3DK.%2BG.%26aulast%3DManoharan%26aufirst%3DM.%26aulast%3DSah%26aufirst%3DD.%2BW.%26aulast%3DZamore%26aufirst%3DP.%2BD.%26aulast%3DAronin%26aufirst%3DN.%26atitle%3DTherapeutic%2520silencing%2520of%2520mutant%2520huntingtin%2520with%2520siRNA%2520attenuates%2520striatal%2520and%2520cortical%2520neuropathology%2520and%2520behavioral%2520deficits%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2007%26volume%3D104%26spage%3D17204%26epage%3D17209%26doi%3D10.1073%2Fpnas.0708285104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kordasiewicz, H. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stanek, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wancewicz, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mazur, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McAlonis, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pytel, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Artates, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shihabuddin, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, C. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cleveland, D. W.</span></span> <span> </span><span class="NLM_article-title">Sustained therapeutic reversal of Huntington’s disease by transient repression of huntingtin synthesis</span>. <i>Neuron</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>74</i></span>,  <span class="NLM_fpage">1031</span>– <span class="NLM_lpage">1044</span>, <span class="refDoi"> DOI: 10.1016/j.neuron.2012.05.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1016%2Fj.neuron.2012.05.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=22726834" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A528%3ADC%252BC38XptVKgtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2012&pages=1031-1044&author=H.+B.+Kordasiewiczauthor=L.+M.+Stanekauthor=E.+V.+Wancewiczauthor=C.+Mazurauthor=M.+M.+McAlonisauthor=K.+A.+Pytelauthor=J.+W.+Artatesauthor=A.+Weissauthor=S.+H.+Chengauthor=L.+S.+Shihabuddinauthor=G.+Hungauthor=C.+F.+Bennettauthor=D.+W.+Cleveland&title=Sustained+therapeutic+reversal+of+Huntington%E2%80%99s+disease+by+transient+repression+of+huntingtin+synthesis&doi=10.1016%2Fj.neuron.2012.05.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Sustained Therapeutic Reversal of Huntington's Disease by Transient Repression of Huntingtin Synthesis</span></div><div class="casAuthors">Kordasiewicz, Holly B.; Stanek, Lisa M.; Wancewicz, Edward V.; Mazur, Curt; McAlonis, Melissa M.; Pytel, Kimberly A.; Artates, Jonathan W.; Weiss, Andreas; Cheng, Seng H.; Shihabuddin, Lamya S.; Hung, Gene; Bennett, C. Frank; Cleveland, Don W.</div><div class="citationInfo"><span class="NLM_cas:title">Neuron</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">74</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1031-1044</span>CODEN:
                <span class="NLM_cas:coden">NERNET</span>;
        ISSN:<span class="NLM_cas:issn">0896-6273</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The primary cause of Huntington's disease (HD) is expression of huntingtin with a polyglutamine expansion.  Despite an absence of consensus on the mechanism(s) of toxicity, diminishing the synthesis of mutant huntingtin will abate toxicity if delivered to the key affected cells.  With antisense oligonucleotides (ASOs) that catalyze RNase H-mediated degrdn. of huntingtin mRNA, we demonstrate that transient infusion into the cerebrospinal fluid of symptomatic HD mouse models not only delays disease progression but mediates a sustained reversal of disease phenotype that persists longer than the huntingtin knockdown.  Redn. of wild-type huntingtin, along with mutant huntingtin, produces the same sustained disease reversal.  Similar ASO infusion into nonhuman primates is shown to effectively lower huntingtin in many brain regions targeted by HD pathol.  Rather than requiring continuous treatment, our findings establish a therapeutic strategy for sustained HD disease reversal produced by transient ASO-mediated diminution of huntingtin synthesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrn0_XTuNHKjrVg90H21EOLACvtfcHk0lgHtF2rXzT5UA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XptVKgtbs%253D&md5=f4e2afdce8cbca04dbd3dcbbbe7f0c1b</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.neuron.2012.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neuron.2012.05.009%26sid%3Dliteratum%253Aachs%26aulast%3DKordasiewicz%26aufirst%3DH.%2BB.%26aulast%3DStanek%26aufirst%3DL.%2BM.%26aulast%3DWancewicz%26aufirst%3DE.%2BV.%26aulast%3DMazur%26aufirst%3DC.%26aulast%3DMcAlonis%26aufirst%3DM.%2BM.%26aulast%3DPytel%26aufirst%3DK.%2BA.%26aulast%3DArtates%26aufirst%3DJ.%2BW.%26aulast%3DWeiss%26aufirst%3DA.%26aulast%3DCheng%26aufirst%3DS.%2BH.%26aulast%3DShihabuddin%26aufirst%3DL.%2BS.%26aulast%3DHung%26aufirst%3DG.%26aulast%3DBennett%26aufirst%3DC.%2BF.%26aulast%3DCleveland%26aufirst%3DD.%2BW.%26atitle%3DSustained%2520therapeutic%2520reversal%2520of%2520Huntington%25E2%2580%2599s%2520disease%2520by%2520transient%2520repression%2520of%2520huntingtin%2520synthesis%26jtitle%3DNeuron%26date%3D2012%26volume%3D74%26spage%3D1031%26epage%3D1044%26doi%3D10.1016%2Fj.neuron.2012.05.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X. J.</span></span> <span> </span><span class="NLM_article-title">CRISPR/Cas9-mediated gene editing ameliorates neurotoxicity in mouse model of Huntington’s disease</span>. <i>J. Clin. Invest.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>127</i></span>,  <span class="NLM_fpage">2719</span>– <span class="NLM_lpage">2724</span>, <span class="refDoi"> DOI: 10.1172/JCI92087</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1172%2FJCI92087" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=28628038" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A280%3ADC%252BC1cnps1Wqsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=127&publication_year=2017&pages=2719-2724&author=S.+Yangauthor=R.+Changauthor=H.+Yangauthor=T.+Zhaoauthor=Y.+Hongauthor=H.+E.+Kongauthor=X.+Sunauthor=Z.+Qinauthor=P.+Jinauthor=S.+Liauthor=X.+J.+Li&title=CRISPR%2FCas9-mediated+gene+editing+ameliorates+neurotoxicity+in+mouse+model+of+Huntington%E2%80%99s+disease&doi=10.1172%2FJCI92087"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">CRISPR/Cas9-mediated gene editing ameliorates neurotoxicity in mouse model of Huntington's disease</span></div><div class="casAuthors">Yang Su; Chang Renbao; Yang Huiming; Zhao Ting; Hong Yan; Kong Ha Eun; Jin Peng; Li Shihua; Li Xiao-Jiang; Chang Renbao; Chang Renbao; Sun Xiaobo; Qin Zhaohui; Li Xiao-Jiang</div><div class="citationInfo"><span class="NLM_cas:title">The Journal of clinical investigation</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">127</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2719-2724</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Huntington's disease is a neurodegenerative disorder caused by a polyglutamine repeat in the Huntingtin gene (HTT).  Although suppressing the expression of mutant HTT (mHTT) has been explored as a therapeutic strategy to treat Huntington's disease, considerable efforts have gone into developing allele-specific suppression of mHTT expression, given that loss of Htt in mice can lead to embryonic lethality.  It remains unknown whether depletion of HTT in the adult brain, regardless of its allele, could be a safe therapy.  Here, we report that permanent suppression of endogenous mHTT expression in the striatum of mHTT-expressing mice (HD140Q-knockin mice) using CRISPR/Cas9-mediated inactivation effectively depleted HTT aggregates and attenuated early neuropathology.  The reduction of mHTT expression in striatal neuronal cells in adult HD140Q-knockin mice did not affect viability, but alleviated motor deficits.  Our studies suggest that non-allele-specific CRISPR/Cas9-mediated gene editing could be used to efficiently and permanently eliminate polyglutamine expansion-mediated neuronal toxicity in the adult brain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSclUvCk85BBNbHMBt0mMIXfW6udTcc2ebEfiSFL-2HwLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cnps1Wqsw%253D%253D&md5=0d4ebdb952e51a695b7595292638fba0</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1172%2FJCI92087&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252FJCI92087%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DS.%26aulast%3DChang%26aufirst%3DR.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DZhao%26aufirst%3DT.%26aulast%3DHong%26aufirst%3DY.%26aulast%3DKong%26aufirst%3DH.%2BE.%26aulast%3DSun%26aufirst%3DX.%26aulast%3DQin%26aufirst%3DZ.%26aulast%3DJin%26aufirst%3DP.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DX.%2BJ.%26atitle%3DCRISPR%252FCas9-mediated%2520gene%2520editing%2520ameliorates%2520neurotoxicity%2520in%2520mouse%2520model%2520of%2520Huntington%25E2%2580%2599s%2520disease%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D2017%26volume%3D127%26spage%3D2719%26epage%3D2724%26doi%3D10.1172%2FJCI92087" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Caron, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dorsey, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayden, M. R.</span></span> <span> </span><span class="NLM_article-title">Therapeutic approaches to Huntington disease: from the bench to the clinic</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">729</span>– <span class="NLM_lpage">750</span>, <span class="refDoi"> DOI: 10.1038/nrd.2018.133</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1038%2Fnrd.2018.133" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=30237454" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslOjsrzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2018&pages=729-750&author=N.+S.+Caronauthor=E.+R.+Dorseyauthor=M.+R.+Hayden&title=Therapeutic+approaches+to+Huntington+disease%3A+from+the+bench+to+the+clinic&doi=10.1038%2Fnrd.2018.133"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Therapeutic approaches to Huntington disease: from the bench to the clinic</span></div><div class="casAuthors">Caron, Nicholas S.; Dorsey, E. Ray; Hayden, Michael R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">729-750</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The 25 years since the identification of the gene responsible for Huntington disease (HD) have stood witness to profound discoveries about the nature of the disease and its pathogenesis.  Despite this progress, however, the development of disease-modifying therapies has thus far been slow.  Preclin. validation of the therapeutic potential of disrupted pathways in HD has led to the advancement of pharmacol. agents, both novel and repurposed, for clin. evaluation.  The most promising therapeutic approaches include huntingtin (HTT) lowering and modification as well as modulation of neuroinflammation and synaptic transmission.  With clin. trials for many of these approaches imminent or currently ongoing, the coming years are promising not only for HD but also for more prevalent neurodegenerative disorders, such as Alzheimer and Parkinson disease, in which many of these pathways have been similarly implicated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpx7V2wDUlHvLVg90H21EOLACvtfcHk0lhy2fEmJz84Xw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslOjsrzP&md5=c87c8f8ee9144591ff226c05f634dc41</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2018.133&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2018.133%26sid%3Dliteratum%253Aachs%26aulast%3DCaron%26aufirst%3DN.%2BS.%26aulast%3DDorsey%26aufirst%3DE.%2BR.%26aulast%3DHayden%26aufirst%3DM.%2BR.%26atitle%3DTherapeutic%2520approaches%2520to%2520Huntington%2520disease%253A%2520from%2520the%2520bench%2520to%2520the%2520clinic%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2018%26volume%3D17%26spage%3D729%26epage%3D750%26doi%3D10.1038%2Fnrd.2018.133" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Binari, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perrimon, N.</span></span> <span> </span><span class="NLM_article-title">A genomewide RNA interference screen for modifiers of aggregates formation by mutant Huntingtin in Drosophila</span>. <i>Genetics</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>184</i></span>,  <span class="NLM_fpage">1165</span>– <span class="NLM_lpage">1179</span>, <span class="refDoi"> DOI: 10.1534/genetics.109.112516</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1534%2Fgenetics.109.112516" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=20100940" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsFOnurnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=184&publication_year=2010&pages=1165-1179&author=S.+Zhangauthor=R.+Binariauthor=R.+Zhouauthor=N.+Perrimon&title=A+genomewide+RNA+interference+screen+for+modifiers+of+aggregates+formation+by+mutant+Huntingtin+in+Drosophila&doi=10.1534%2Fgenetics.109.112516"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">A genomewide RNA interference screen for modifiers of aggregates formation by mutant huntingtin in Drosophila</span></div><div class="casAuthors">Zhang, Sheng; Binari, Richard; Zhou, Rui; Perrimon, Norbert</div><div class="citationInfo"><span class="NLM_cas:title">Genetics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">184</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1165-1179</span>CODEN:
                <span class="NLM_cas:coden">GENTAE</span>;
        ISSN:<span class="NLM_cas:issn">0016-6731</span>.
    
            (<span class="NLM_cas:orgname">Genetics Society of America</span>)
        </div><div class="casAbstract">Protein aggregates are a common pathol. feature of most neurodegenerative diseases (NDs).  Understanding their formation and regulation will help clarify their controversial roles in disease pathogenesis.  To date, there have been few systematic studies of aggregates formation in Drosophila, a model organism that has been applied extensively in modeling NDs and screening for toxicity modifiers.  We generated transgenic fly lines that express enhanced-GFP-tagged mutant Huntingtin (Htt) fragments with different lengths of polyglutamine (polyQ) tract and showed that these Htt mutants develop protein aggregates in a polyQ-length- and age-dependent manner in Drosophila.  To identify central regulators of protein aggregation, we further generated stable Drosophila cell lines expressing these Htt mutants and also established a cell-based quant. assay that allows automated measurement of aggregates within cells.  We then performed a genomewide RNA interference screen for regulators of mutant Htt aggregation and isolated 126 genes involved in diverse cellular processes.  Interestingly, although our screen focused only on mutant Htt aggregation, several of the identified candidates were known previously as toxicity modifiers of NDs.  Moreover, modulating the in vivo activity of hsp110 (CG6603) or tra1, two hits from the screen, affects neurodegeneration in a dose-dependent manner in a Drosophila model of Huntington's disease.  Thus, other aggregates regulators isolated in our screen may identify addnl. genes involved in the protein-folding pathway and neurotoxicity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoki7Fderw6ZbVg90H21EOLACvtfcHk0lg-yTkxuj2y-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsFOnurnF&md5=daf2e6693999906970162a4f8955ecee</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1534%2Fgenetics.109.112516&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1534%252Fgenetics.109.112516%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DBinari%26aufirst%3DR.%26aulast%3DZhou%26aufirst%3DR.%26aulast%3DPerrimon%26aufirst%3DN.%26atitle%3DA%2520genomewide%2520RNA%2520interference%2520screen%2520for%2520modifiers%2520of%2520aggregates%2520formation%2520by%2520mutant%2520Huntingtin%2520in%2520Drosophila%26jtitle%3DGenetics%26date%3D2010%26volume%3D184%26spage%3D1165%26epage%3D1179%26doi%3D10.1534%2Fgenetics.109.112516" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baldo, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parker, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bibel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paganetti, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaupmann, K.</span></span> <span> </span><span class="NLM_article-title">A screen for enhancers of clearance identifies huntingtin as a heat shock protein 90 (Hsp90) client protein</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>287</i></span>,  <span class="NLM_fpage">1406</span>– <span class="NLM_lpage">1414</span>, <span class="refDoi"> DOI: 10.1074/jbc.M111.294801</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1074%2Fjbc.M111.294801" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=22123826" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A528%3ADC%252BC38Xjslyksw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2012&pages=1406-1414&author=B.+Baldoauthor=A.+Weissauthor=C.+N.+Parkerauthor=M.+Bibelauthor=P.+Paganettiauthor=K.+Kaupmann&title=A+screen+for+enhancers+of+clearance+identifies+huntingtin+as+a+heat+shock+protein+90+%28Hsp90%29+client+protein&doi=10.1074%2Fjbc.M111.294801"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">A Screen for Enhancers of Clearance Identifies Huntingtin as a Heat Shock Protein 90 (Hsp90) Client Protein</span></div><div class="casAuthors">Baldo, Barbara; Weiss, Andreas; Parker, Christian N.; Bibel, Miriam; Paganetti, Paolo; Kaupmann, Klemens</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">287</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">1406-1414</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Mechanisms to reduce the cellular levels of mutant huntingtin (mHtt) provide promising strategies for treating Huntington disease (HD).  To identify compds. enhancing the degrdn. of mHtt, we performed a high throughput screen using a hippocampal HN10 cell line expressing a 573-amino acid mHtt fragment.  Several hit structures were identified as heat shock protein 90 (Hsp90) inhibitors.  Cell treatment with these compds. reduced levels of mHtt without overt toxic effects as measured by time-resolved Foerster resonance energy transfer assays and Western blots.  To characterize the mechanism of mHtt degrdn., we used the potent and selective Hsp90 inhibitor NVP-AUY922.  In HdhQ150 embryonic stem (ES) cells and in ES cell-derived neurons, NVP-AUY922 treatment substantially reduced sol. full-length mHtt levels.  In HN10 cells, Hsp90 inhibition by NVP-AUY922 enhanced mHtt clearance in the absence of any detectable Hsp70 induction.  Furthermore, inhibition of protein synthesis with cycloheximide or overexpression of dominant neg. heat shock factor 1 (Hsf1) in HdhQ150 ES cells attenuated Hsp70 induction but did not affect NVP-AUY922-mediated mHtt clearance.  Together, these data provided evidence that direct inhibition of Hsp90 chaperone function was crucial for mHtt degrdn. rather than heat shock response induction and Hsp70 up-regulation.  Co-immunopptn. expts. revealed a phys. interaction of mutant and wild-type Htt with the Hsp90 chaperone.  Hsp90 inhibition disrupted the interaction and induced clearance of Htt through the ubiquitin-proteasome system.  These data suggest that Htt is an Hsp90 client protein and that Hsp90 inhibition may provide a means to reduce mHtt in HD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBzgxmFERTErVg90H21EOLACvtfcHk0lg-yTkxuj2y-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xjslyksw%253D%253D&md5=b0cde124dc9c309851719560d6e44f28</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M111.294801&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M111.294801%26sid%3Dliteratum%253Aachs%26aulast%3DBaldo%26aufirst%3DB.%26aulast%3DWeiss%26aufirst%3DA.%26aulast%3DParker%26aufirst%3DC.%2BN.%26aulast%3DBibel%26aufirst%3DM.%26aulast%3DPaganetti%26aufirst%3DP.%26aulast%3DKaupmann%26aufirst%3DK.%26atitle%3DA%2520screen%2520for%2520enhancers%2520of%2520clearance%2520identifies%2520huntingtin%2520as%2520a%2520heat%2520shock%2520protein%252090%2520%2528Hsp90%2529%2520client%2520protein%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2012%26volume%3D287%26spage%3D1406%26epage%3D1414%26doi%3D10.1074%2Fjbc.M111.294801" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Komatsu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maruyama, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shinohara, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakuma, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imaichi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chikatsu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuniyeda, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siu, F. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhuo, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mun, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsumoto, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hashimoto, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyajima, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Itoh, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Habata, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mori, M.</span></span> <span> </span><span class="NLM_article-title">Anatomical transcriptome of G protein-coupled receptors leads to the identification of a novel therapeutic candidate GPR52 for psychiatric disorders</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span>, <span class="NLM_elocation-id">e90134</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0090134</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1371%2Fjournal.pone.0090134" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=24587241" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVKjsbfF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&author=H.+Komatsuauthor=M.+Maruyamaauthor=S.+Yaoauthor=T.+Shinoharaauthor=K.+Sakumaauthor=S.+Imaichiauthor=T.+Chikatsuauthor=K.+Kuniyedaauthor=F.+K.+Siuauthor=L.+S.+Pengauthor=K.+Zhuoauthor=L.+S.+Munauthor=T.+M.+Hanauthor=Y.+Matsumotoauthor=T.+Hashimotoauthor=N.+Miyajimaauthor=Y.+Itohauthor=K.+Ogiauthor=Y.+Habataauthor=M.+Mori&title=Anatomical+transcriptome+of+G+protein-coupled+receptors+leads+to+the+identification+of+a+novel+therapeutic+candidate+GPR52+for+psychiatric+disorders&doi=10.1371%2Fjournal.pone.0090134"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Anatomical transcriptome of G protein-coupled receptors leads to the identification of a novel therapeutic candidate GPR52 for psychiatric disorders</span></div><div class="casAuthors">Komatsu, Hidetoshi; Maruyama, Minoru; Yao, Shuuhei; Shinohara, Tokuyuki; Sakuma, Kensuke; Imaichi, Sachiko; Chikatsu, Tomoko; Kuniyeda, Kanako; Siu, Foo Kok; Peng, Lam Sock; Zhuo, Katherine; Mun, Lay Sock; Han, Tan Min; Matsumoto, Yoshio; Hashimoto, Tadatoshi; Miyajima, Nobuyuki; Itoh, Yasuaki; Ogi, Kazuhiro; Habata, Yugo; Mori, Masaaki</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">e90134/1-e90134/16, 16 pp.</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Many drugs of abuse and most neuropharmacol. agents regulate G protein-coupled receptors (GPCRs) in the central nervous system (CNS)_ENREF_1.  The striatum, in which dopamine D1 and D2 receptors are enriched, is strongly innervated by the ventral tegmental area (VTA), which is the origin of dopaminergic cell bodies of the mesocorticolimbic dopamine system_ENREF_3 and plays a central role in the development of psychiatric disorders_ENREF_4.  Here we report the comprehensive and anatomical transcript profiling of 322 non-odorant GPCRs in mouse tissue by quant. real-time PCR (qPCR), leading to the identification of neurotherapeutic receptors exclusively expressed in the CNS, esp. in the striatum.  Among them, GPR6, GPR52, and GPR88, known as orphan GPCRs, were shown to co-localize either with a D2 receptor alone or with both D1 and D2 receptors in neurons of the basal ganglia.  Intriguingly, we found that GPR52 was well conserved among vertebrates, is Gs-coupled and responsive to the antipsychotic drug, reserpine.  We used three types of transgenic (Tg) mice employing a Cre-lox system under the control of the GPR52 promoter, namely, GPR52-LacZ Tg, human GPR52 (hGPR52) Tg, and hGPR52-GFP Tg mice.  Detailed histol. investigation suggests that GPR52 may modulate dopaminergic and glutamatergic transmission in neuronal circuits responsible for cognitive function and emotion.  In support of our prediction, GPR52 knockout and transgenic mice exhibited psychosis-related and antipsychotic-like behaviors, resp.  Therefore, we propose that GPR52 has the potential of being a therapeutic psychiatric receptor.  This approach may help identify potential therapeutic targets for CNS diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoceoSKPwv1nrVg90H21EOLACvtfcHk0lg-yTkxuj2y-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVKjsbfF&md5=fdaf1c9f4a80ef12adf400eef5eec2f3</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0090134&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0090134%26sid%3Dliteratum%253Aachs%26aulast%3DKomatsu%26aufirst%3DH.%26aulast%3DMaruyama%26aufirst%3DM.%26aulast%3DYao%26aufirst%3DS.%26aulast%3DShinohara%26aufirst%3DT.%26aulast%3DSakuma%26aufirst%3DK.%26aulast%3DImaichi%26aufirst%3DS.%26aulast%3DChikatsu%26aufirst%3DT.%26aulast%3DKuniyeda%26aufirst%3DK.%26aulast%3DSiu%26aufirst%3DF.%2BK.%26aulast%3DPeng%26aufirst%3DL.%2BS.%26aulast%3DZhuo%26aufirst%3DK.%26aulast%3DMun%26aufirst%3DL.%2BS.%26aulast%3DHan%26aufirst%3DT.%2BM.%26aulast%3DMatsumoto%26aufirst%3DY.%26aulast%3DHashimoto%26aufirst%3DT.%26aulast%3DMiyajima%26aufirst%3DN.%26aulast%3DItoh%26aufirst%3DY.%26aulast%3DOgi%26aufirst%3DK.%26aulast%3DHabata%26aufirst%3DY.%26aulast%3DMori%26aufirst%3DM.%26atitle%3DAnatomical%2520transcriptome%2520of%2520G%2520protein-coupled%2520receptors%2520leads%2520to%2520the%2520identification%2520of%2520a%2520novel%2520therapeutic%2520candidate%2520GPR52%2520for%2520psychiatric%2520disorders%26jtitle%3DPLoS%2520One%26date%3D2014%26volume%3D9%26doi%3D10.1371%2Fjournal.pone.0090134" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yao, Y.</span>; <span class="NLM_string-name">Cui, X.</span>; <span class="NLM_string-name">Al-Ramahi, I.</span>; <span class="NLM_string-name">Sun, X.</span>; <span class="NLM_string-name">Li, B.</span>; <span class="NLM_string-name">Hou, J.</span>; <span class="NLM_string-name">Difiglia, M.</span>; <span class="NLM_string-name">Palacino, J.</span>; <span class="NLM_string-name">Wu, Z. Y.</span>; <span class="NLM_string-name">Ma, L.</span>; <span class="NLM_string-name">Botas, J.</span>; <span class="NLM_string-name">Lu, B.</span></span> <span> </span><span class="NLM_article-title">A striatal-enriched intronic GPCR modulates huntingtin levels and toxicity</span>.  <i>Elife</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume">4</span>, <span class="refDoi"> DOI: 10.7554/eLife.05449</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.7554%2FeLife.05449" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2015&author=Y.+Yao&author=X.+Cui&author=I.+Al-Ramahi&author=X.+Sun&author=B.+Li&author=J.+Hou&author=M.+Difiglia&author=J.+Palacino&author=Z.+Y.+Wu&author=L.+Ma&author=J.+Botas&author=B.+Lu&title=A+striatal-enriched+intronic+GPCR+modulates+huntingtin+levels+and+toxicity&doi=10.7554%2FeLife.05449"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.7554%2FeLife.05449&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.05449%26sid%3Dliteratum%253Aachs%26aulast%3DYao%26aufirst%3DY.%26atitle%3DA%2520striatal-enriched%2520intronic%2520GPCR%2520modulates%2520huntingtin%2520levels%2520and%2520toxicity%26jtitle%3DElife%26date%3D2015%26volume%3D4%26doi%3D10.7554%2FeLife.05449" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Song, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">An, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, B.</span></span> <span> </span><span class="NLM_article-title">Targeting Gpr52 lowers mutant HTT levels and rescues Huntington’s disease-associated phenotypes</span>. <i>Brain</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>141</i></span>,  <span class="NLM_fpage">1782</span>– <span class="NLM_lpage">1798</span>, <span class="refDoi"> DOI: 10.1093/brain/awy081</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1093%2Fbrain%2Fawy081" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=29608652" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A280%3ADC%252BC1MnotVCksg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=141&publication_year=2018&pages=1782-1798&author=H.+Songauthor=H.+Liauthor=S.+Guoauthor=Y.+Panauthor=Y.+Fuauthor=Z.+Zhouauthor=Z.+Liauthor=X.+Wenauthor=X.+Sunauthor=B.+Heauthor=H.+Guauthor=Q.+Zhaoauthor=C.+Wangauthor=P.+Anauthor=S.+Luoauthor=Y.+Huauthor=X.+Xieauthor=B.+Lu&title=Targeting+Gpr52+lowers+mutant+HTT+levels+and+rescues+Huntington%E2%80%99s+disease-associated+phenotypes&doi=10.1093%2Fbrain%2Fawy081"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Gpr52 lowers mutant HTT levels and rescues Huntington's disease-associated phenotypes</span></div><div class="casAuthors">Song Haikun; Li Hexuan; Pan Yuyin; Fu Yuhua; Zhou Zijian; Li Zhaoyang; Wen Xue; Sun Xiaoli; Zhao Quan; Wang Cen; An Ping; Lu Boxun; Guo Shimeng; He Bingqing; Gu Haifeng; Hu Youhong; Xie Xin; He Bingqing; Xie Xin; Luo Shouqing</div><div class="citationInfo"><span class="NLM_cas:title">Brain : a journal of neurology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">141</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1782-1798</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">See Huang and Gitler (doi:10.1093/brain/awy112) for a scientific commentary on this article.Lowering the levels of disease-causing proteins is an attractive treatment strategy for neurodegenerative disorders, among which Huntington's disease is an appealing disease for testing this strategy because of its monogenetic nature.  Huntington's disease is mainly caused by cytotoxicity of the mutant HTT protein with an expanded polyglutamine repeat tract.  Lowering the soluble mutant HTT may reduce its downstream toxicity and provide potential treatment for Huntington's disease.  This is hard to achieve by small-molecule compound drugs because of a lack of effective targets.  Here we demonstrate Gpr52, an orphan G protein-coupled receptor, as a potential Huntington's disease drug target.  Knocking-out Gpr52 significantly reduces mutant HTT levels in the striatum and rescues Huntington's disease-associated behavioural phenotypes in a knock-in Huntington's disease mouse model expressing endogenous mutant Htt.  Importantly, a novel Gpr52 antagonist E7 reduces mutant HTT levels and rescues Huntington's disease-associated phenotypes in cellular and mouse models.  Our study provides an entry point for Huntington's disease drug discovery by targeting Gpr52.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQSulnCGMUk0eRS7coPn-RtfW6udTcc2eZg6uAYreCgArntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MnotVCksg%253D%253D&md5=3e0e745df0131bbc5db9cbeab7de6e3f</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1093%2Fbrain%2Fawy081&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbrain%252Fawy081%26sid%3Dliteratum%253Aachs%26aulast%3DSong%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DGuo%26aufirst%3DS.%26aulast%3DPan%26aufirst%3DY.%26aulast%3DFu%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DWen%26aufirst%3DX.%26aulast%3DSun%26aufirst%3DX.%26aulast%3DHe%26aufirst%3DB.%26aulast%3DGu%26aufirst%3DH.%26aulast%3DZhao%26aufirst%3DQ.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DAn%26aufirst%3DP.%26aulast%3DLuo%26aufirst%3DS.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DXie%26aufirst%3DX.%26aulast%3DLu%26aufirst%3DB.%26atitle%3DTargeting%2520Gpr52%2520lowers%2520mutant%2520HTT%2520levels%2520and%2520rescues%2520Huntington%25E2%2580%2599s%2520disease-associated%2520phenotypes%26jtitle%3DBrain%26date%3D2018%26volume%3D141%26spage%3D1782%26epage%3D1798%26doi%3D10.1093%2Fbrain%2Fawy081" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Trettel, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rigamonti, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilditch-Maguire, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wheeler, V. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharp, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Persichetti, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cattaneo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacDonald, M. E.</span></span> <span> </span><span class="NLM_article-title">Dominant phenotypes produced by the HD mutation in STHdh(Q111) striatal cells</span>. <i>Hum. Mol. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">2799</span>– <span class="NLM_lpage">2809</span>, <span class="refDoi"> DOI: 10.1093/hmg/9.19.2799</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1093%2Fhmg%2F9.19.2799" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=11092756" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A528%3ADC%252BD3cXoslWms7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2000&pages=2799-2809&author=F.+Trettelauthor=D.+Rigamontiauthor=P.+Hilditch-Maguireauthor=V.+C.+Wheelerauthor=A.+H.+Sharpauthor=F.+Persichettiauthor=E.+Cattaneoauthor=M.+E.+MacDonald&title=Dominant+phenotypes+produced+by+the+HD+mutation+in+STHdh%28Q111%29+striatal+cells&doi=10.1093%2Fhmg%2F9.19.2799"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Dominant phenotypes produced by the HD mutation in STHdhQ111 striatal cells</span></div><div class="casAuthors">Trettel, Flavia; Rigamonti, Dorotea; Hilditch-Maguire, Paige; Wheeler, Vanessa C.; Sharp, Alan H.; Persichetti, Francesca; Cattaneo, Elena; MacDonald, Marcy E.</div><div class="citationInfo"><span class="NLM_cas:title">Human Molecular Genetics</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">2799-2809</span>CODEN:
                <span class="NLM_cas:coden">HMGEE5</span>;
        ISSN:<span class="NLM_cas:issn">0964-6906</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Lengthening a glutamine tract in huntingtin confers a dominant attribute that initiates degeneration of striatal neurons in Huntington's disease (HD).  To identify pathways that are candidates for the mutant protein's abnormal function, we compared striatal cell lines established from wild-type and HdhQ111 knock-in embryos.  Alternate versions of full-length huntingtin, distinguished by epitope accessibility, were localized to different sets of nuclear and perinuclear organelles involved in RNA biogenesis and membrane trafficking.  However, mutant STHdhQ111 cells also exhibited addnl. forms of the full-length mutant protein and displayed dominant phenotypes that did not mirror phenotypes caused by either huntingtin deficiency or excess.  These phenotypes indicate a disruption of striatal cell homeostasis by the mutant protein, via a mechanism that is sep. from its normal activity.  They also support specific stress pathways, including elevated p53, endoplasmic reticulum stress response and hypoxia, as potential players in HD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8lnl6CjJK57Vg90H21EOLACvtfcHk0lh91DS_4xf-tQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXoslWms7c%253D&md5=00dabfeb20888d87f44ad2c3762bd3c9</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1093%2Fhmg%2F9.19.2799&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fhmg%252F9.19.2799%26sid%3Dliteratum%253Aachs%26aulast%3DTrettel%26aufirst%3DF.%26aulast%3DRigamonti%26aufirst%3DD.%26aulast%3DHilditch-Maguire%26aufirst%3DP.%26aulast%3DWheeler%26aufirst%3DV.%2BC.%26aulast%3DSharp%26aufirst%3DA.%2BH.%26aulast%3DPersichetti%26aufirst%3DF.%26aulast%3DCattaneo%26aufirst%3DE.%26aulast%3DMacDonald%26aufirst%3DM.%2BE.%26atitle%3DDominant%2520phenotypes%2520produced%2520by%2520the%2520HD%2520mutation%2520in%2520STHdh%2528Q111%2529%2520striatal%2520cells%26jtitle%3DHum.%2520Mol.%2520Genet.%26date%3D2000%26volume%3D9%26spage%3D2799%26epage%3D2809%26doi%3D10.1093%2Fhmg%2F9.19.2799" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Menalled, L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sison, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dragatsis, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeitlin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chesselet, M. F.</span></span> <span> </span><span class="NLM_article-title">Time course of early motor and neuropathological anomalies in a knock-in mouse model of Huntington’s disease with 140 CAG repeats</span>. <i>J. Comp. Neurol.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>465</i></span>,  <span class="NLM_fpage">11</span>– <span class="NLM_lpage">26</span>, <span class="refDoi"> DOI: 10.1002/cne.10776</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1002%2Fcne.10776" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=12926013" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A528%3ADC%252BD3sXnvFSltrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=465&publication_year=2003&pages=11-26&author=L.+B.+Menalledauthor=J.+D.+Sisonauthor=I.+Dragatsisauthor=S.+Zeitlinauthor=M.+F.+Chesselet&title=Time+course+of+early+motor+and+neuropathological+anomalies+in+a+knock-in+mouse+model+of+Huntington%E2%80%99s+disease+with+140+CAG+repeats&doi=10.1002%2Fcne.10776"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Time course of early motor and neuropathological anomalies in a knock-in mouse model of Huntington's disease with 140 CAG repeats</span></div><div class="casAuthors">Menalled, Liliana B.; Sison, Jessica D.; Dragatsis, Ioannis; Zeitlin, Scott; Chesselet, Marie-Francoise</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Comparative Neurology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">465</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">11-26</span>CODEN:
                <span class="NLM_cas:coden">JCNEAM</span>;
        ISSN:<span class="NLM_cas:issn">0021-9967</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">Huntington's disease (HD) is caused by an abnormal expansion of CAG repeats in the gene encoding huntingtin.  The development of therapies for HD requires preclin. testing of drugs in animal models that reproduce the dysfunction and regionally specific pathol. obsd. in HD.  The authors have developed a new knock-in mouse model of HD with a chimeric mouse/human exon 1 contg. 140 CAG repeats inserted in the murine huntingtin gene.  These mice displayed an increased locomotor activity and rearing at 1 mo of age, followed by hypoactivity at 4 mo and gait anomalies at 1 yr.  Behavioral symptoms preceded neuropathol. anomalies, which became intense and widespread only at 4 mo of age.  These consisted of nuclear staining for huntingtin and huntingtin-contg. nuclear and neuropil aggregates that first appeared in the striatum, nucleus accumbens, and olfactory tubercle.  Interestingly, regions with early pathol. all receive dense dopaminergic inputs, supporting accumulating evidence for a role of dopamine in HD pathol.  Nuclear staining and aggregates predominated in striatum and layer II/III and deep layer V of the cerebral cortex, whereas neuropil aggregates were found in the globus pallidus and layer IV/superficial layer V of the cerebral cortex.  The olfactory system displayed early and marked aggregate accumulation, which may be relevant to the early deficit in odor discrimination obsd. in patients with HD.  Because of their early behavioral anomalies and regionally specific pathol., these mice provide a powerful tool with which to evaluate the effectiveness of new therapies and to study the mechanisms involved in the neuropathol. of HD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWln0m5hxNMrVg90H21EOLACvtfcHk0lh91DS_4xf-tQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXnvFSltrg%253D&md5=a2632e7aff3a687a218858c5612b30b3</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1002%2Fcne.10776&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcne.10776%26sid%3Dliteratum%253Aachs%26aulast%3DMenalled%26aufirst%3DL.%2BB.%26aulast%3DSison%26aufirst%3DJ.%2BD.%26aulast%3DDragatsis%26aufirst%3DI.%26aulast%3DZeitlin%26aufirst%3DS.%26aulast%3DChesselet%26aufirst%3DM.%2BF.%26atitle%3DTime%2520course%2520of%2520early%2520motor%2520and%2520neuropathological%2520anomalies%2520in%2520a%2520knock-in%2520mouse%2520model%2520of%2520Huntington%25E2%2580%2599s%2520disease%2520with%2520140%2520CAG%2520repeats%26jtitle%3DJ.%2520Comp.%2520Neurol.%26date%3D2003%26volume%3D465%26spage%3D11%26epage%3D26%26doi%3D10.1002%2Fcne.10776" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, B.</span></span> <span> </span><span class="NLM_article-title">TR-FRET Assays for endogenous Huntingtin protein level in mouse Cells</span>. <i>J. Huntington’s Dis.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">253</span>– <span class="NLM_lpage">259</span>, <span class="refDoi"> DOI: 10.3233/JHD-140104</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.3233%2FJHD-140104" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=25300329" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs12ku7bN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2014&pages=253-259&author=Y.+Liangauthor=Y.+Yaoauthor=M.+Luauthor=J.+Houauthor=S.+Yuauthor=B.+Lu&title=TR-FRET+Assays+for+endogenous+Huntingtin+protein+level+in+mouse+Cells&doi=10.3233%2FJHD-140104"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">TR-FRET Assays for Endogenous Huntingtin Protein Level in Mouse Cells</span></div><div class="casAuthors">Liang, Yijian; Yao, Yuwei; Lu, Mingxing; Hou, Jiapeng; Yu, Shenliang; Lu, Boxun</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Huntington's Disease</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">253-259</span>CODEN:
                <span class="NLM_cas:coden">JHDOA3</span>;
        ISSN:<span class="NLM_cas:issn">1879-6397</span>.
    
            (<span class="NLM_cas:orgname">IOS Press</span>)
        </div><div class="casAbstract">High-throughput measurement of huntingtin (Htt) levels is useful for Huntington's disease research.  For example, identification of genetic or chem. modifiers that reduce Htt levels by high-throughput screening provides promising strategy for HD drug discovery.  In the human cells, high-throughput measurement of Htt levels has been established based on the Time Resolved-Fluorescence Resonance Energy Transfer (TR-FRET) technol., using the 2B7/MW1 antibody pair.  Unfortunately, application of this assay in the mouse cells has been problematic due to discrepancies between TR-FRET signals and Western-blots, possibly caused by non-specific antibody binding.  Here the authors report TR-FRET assays that are able to detect endogenous Htt levels of the mouse striatal cell line (STHdh).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9z7pPeiC-KLVg90H21EOLACvtfcHk0ljLjUOesDHTEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs12ku7bN&md5=fbfff21d2263e3d4079a057a97821101</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.3233%2FJHD-140104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3233%252FJHD-140104%26sid%3Dliteratum%253Aachs%26aulast%3DLiang%26aufirst%3DY.%26aulast%3DYao%26aufirst%3DY.%26aulast%3DLu%26aufirst%3DM.%26aulast%3DHou%26aufirst%3DJ.%26aulast%3DYu%26aufirst%3DS.%26aulast%3DLu%26aufirst%3DB.%26atitle%3DTR-FRET%2520Assays%2520for%2520endogenous%2520Huntingtin%2520protein%2520level%2520in%2520mouse%2520Cells%26jtitle%3DJ.%2520Huntington%25E2%2580%2599s%2520Dis.%26date%3D2014%26volume%3D3%26spage%3D253%26epage%3D259%26doi%3D10.3233%2FJHD-140104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lagoutte, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serba, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abegg, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoch, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adibekian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winssinger, N.</span></span> <span> </span><span class="NLM_article-title">Divergent synthesis and identification of the cellular targets of deoxyelephantopins</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>, <span class="NLM_elocation-id">12470</span> <span class="refDoi"> DOI: 10.1038/ncomms12470</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1038%2Fncomms12470" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=27539788" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlyltbjP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&author=R.+Lagoutteauthor=C.+Serbaauthor=D.+Abeggauthor=D.+G.+Hochauthor=A.+Adibekianauthor=N.+Winssinger&title=Divergent+synthesis+and+identification+of+the+cellular+targets+of+deoxyelephantopins&doi=10.1038%2Fncomms12470"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Divergent synthesis and identification of the cellular targets of deoxyelephantopins</span></div><div class="casAuthors">Lagoutte, Roman; Serba, Christelle; Abegg, Daniel; Hoch, Dominic G.; Adibekian, Alexander; Winssinger, Nicolas</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">12470</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Herbal exts. contg. sesquiterpene lactones have been extensively used in traditional medicine and are known to be rich in α,β-unsatd. functionalities that can covalently engage target proteins.  Here, we report synthetic methodologies to access analogs of deoxyelephantopin, a sesquiterpene lactone with anticancer properties.  Using alkyne-tagged cellular probes and quant. proteomics anal., we identified several cellular targets of deoxyelephantopin.  We further demonstrate that deoxyelephantopin antagonizes PPARγ activity in situ via covalent engagement of a cysteine residue in the zinc-finger motif of this nuclear receptor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKfKTjnY1Un7Vg90H21EOLACvtfcHk0ljLjUOesDHTEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlyltbjP&md5=7e624d560c775c49cb593375bc86de5a</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fncomms12470&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms12470%26sid%3Dliteratum%253Aachs%26aulast%3DLagoutte%26aufirst%3DR.%26aulast%3DSerba%26aufirst%3DC.%26aulast%3DAbegg%26aufirst%3DD.%26aulast%3DHoch%26aufirst%3DD.%2BG.%26aulast%3DAdibekian%26aufirst%3DA.%26aulast%3DWinssinger%26aufirst%3DN.%26atitle%3DDivergent%2520synthesis%2520and%2520identification%2520of%2520the%2520cellular%2520targets%2520of%2520deoxyelephantopins%26jtitle%3DNat.%2520Commun.%26date%3D2016%26volume%3D7%26doi%3D10.1038%2Fncomms12470" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yue, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, F.</span></span> <span> </span><span class="NLM_article-title">Structural basis of ligand recognition and self-activation of orphan GPR52</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>579</i></span>,  <span class="NLM_fpage">152</span>– <span class="NLM_lpage">157</span>, <span class="refDoi"> DOI: 10.1038/s41586-020-2019-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1038%2Fs41586-020-2019-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=32076264" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A528%3ADC%252BB3cXjsVejsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=579&publication_year=2020&pages=152-157&author=X.+Linauthor=M.+Liauthor=N.+Wangauthor=Y.+Wuauthor=Z.+Luoauthor=S.+Guoauthor=G.+W.+Hanauthor=S.+Liauthor=Y.+Yueauthor=X.+Weiauthor=X.+Xieauthor=Y.+Chenauthor=S.+Zhaoauthor=J.+Wuauthor=M.+Leiauthor=F.+Xu&title=Structural+basis+of+ligand+recognition+and+self-activation+of+orphan+GPR52&doi=10.1038%2Fs41586-020-2019-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis of ligand recognition and self-activation of orphan GPR52</span></div><div class="casAuthors">Lin, Xi; Li, Mingyue; Wang, Niandong; Wu, Yiran; Luo, Zhipu; Guo, Shimeng; Han, Gye-Won; Li, Shaobai; Yue, Yang; Wei, Xiaohu; Xie, Xin; Chen, Yong; Zhao, Suwen; Wu, Jian; Lei, Ming; Xu, Fei</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">579</span>
        (<span class="NLM_cas:issue">7797</span>),
    <span class="NLM_cas:pages">152-157</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">GPR52 is a class-A orphan G-protein-coupled receptor that is highly expressed in the brain and represents a promising therapeutic target for the treatment of Huntington's disease and several psychiatric disorders.  Pathol. malfunction of GPR52 signaling occurs primarily through the heterotrimeric Gs protein, but it is unclear how GPR52 and Gs couple for signal transduction and whether a native ligand or other activating input is required.  Here we present the high-resoln. structures of human GPR52 in three states: a ligand-free state, a Gs-coupled self-activation state and a potential allosteric ligand-bound state.  Together, our structures reveal that extracellular loop 2 occupies the orthosteric binding pocket and operates as a built-in agonist, conferring an intrinsically high level of basal activity to GPR52.  A fully active state is achieved when Gs is coupled to GPR52 in the absence of an external agonist.  The receptor also features a side pocket for ligand binding.  These insights into the structure and function of GPR52 could improve our understanding of other self-activated GPCRs, enable the identification of endogenous and tool ligands, and guide drug discovery efforts that target GPR52.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotl_2-sJeM_bVg90H21EOLACvtfcHk0lhYG8C2CqnQOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXjsVejsLg%253D&md5=a0c899853309addd569f16f70c221e7c</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fs41586-020-2019-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-020-2019-0%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DN.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DLuo%26aufirst%3DZ.%26aulast%3DGuo%26aufirst%3DS.%26aulast%3DHan%26aufirst%3DG.%2BW.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DYue%26aufirst%3DY.%26aulast%3DWei%26aufirst%3DX.%26aulast%3DXie%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DS.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DLei%26aufirst%3DM.%26aulast%3DXu%26aufirst%3DF.%26atitle%3DStructural%2520basis%2520of%2520ligand%2520recognition%2520and%2520self-activation%2520of%2520orphan%2520GPR52%26jtitle%3DNature%26date%3D2020%26volume%3D579%26spage%3D152%26epage%3D157%26doi%3D10.1038%2Fs41586-020-2019-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, G.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yue, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, F.</span></span> <span> </span><span class="NLM_article-title">Structural basis of ligand recognition and self-activation of orphan GPR52</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>579</i></span>,  <span class="NLM_fpage">152</span>– <span class="NLM_lpage">157</span>, <span class="refDoi"> DOI: 10.1038/s41586-020-2019-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1038%2Fs41586-020-2019-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=32076264" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A528%3ADC%252BB3cXjsVejsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=579&publication_year=2020&pages=152-157&author=X.+Linauthor=M.+Liauthor=N.+Wangauthor=Y.+Wuauthor=Z.+Luoauthor=S.+Guoauthor=G.-W.+Hanauthor=S.+Liauthor=Y.+Yueauthor=X.+Weiauthor=X.+Xieauthor=Y.+Chenauthor=S.+Zhaoauthor=J.+Wuauthor=M.+Leiauthor=F.+Xu&title=Structural+basis+of+ligand+recognition+and+self-activation+of+orphan+GPR52&doi=10.1038%2Fs41586-020-2019-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis of ligand recognition and self-activation of orphan GPR52</span></div><div class="casAuthors">Lin, Xi; Li, Mingyue; Wang, Niandong; Wu, Yiran; Luo, Zhipu; Guo, Shimeng; Han, Gye-Won; Li, Shaobai; Yue, Yang; Wei, Xiaohu; Xie, Xin; Chen, Yong; Zhao, Suwen; Wu, Jian; Lei, Ming; Xu, Fei</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">579</span>
        (<span class="NLM_cas:issue">7797</span>),
    <span class="NLM_cas:pages">152-157</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">GPR52 is a class-A orphan G-protein-coupled receptor that is highly expressed in the brain and represents a promising therapeutic target for the treatment of Huntington's disease and several psychiatric disorders.  Pathol. malfunction of GPR52 signaling occurs primarily through the heterotrimeric Gs protein, but it is unclear how GPR52 and Gs couple for signal transduction and whether a native ligand or other activating input is required.  Here we present the high-resoln. structures of human GPR52 in three states: a ligand-free state, a Gs-coupled self-activation state and a potential allosteric ligand-bound state.  Together, our structures reveal that extracellular loop 2 occupies the orthosteric binding pocket and operates as a built-in agonist, conferring an intrinsically high level of basal activity to GPR52.  A fully active state is achieved when Gs is coupled to GPR52 in the absence of an external agonist.  The receptor also features a side pocket for ligand binding.  These insights into the structure and function of GPR52 could improve our understanding of other self-activated GPCRs, enable the identification of endogenous and tool ligands, and guide drug discovery efforts that target GPR52.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotl_2-sJeM_bVg90H21EOLACvtfcHk0lhYG8C2CqnQOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXjsVejsLg%253D&md5=a0c899853309addd569f16f70c221e7c</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fs41586-020-2019-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-020-2019-0%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DN.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DLuo%26aufirst%3DZ.%26aulast%3DGuo%26aufirst%3DS.%26aulast%3DHan%26aufirst%3DG.-W.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DYue%26aufirst%3DY.%26aulast%3DWei%26aufirst%3DX.%26aulast%3DXie%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DS.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DLei%26aufirst%3DM.%26aulast%3DXu%26aufirst%3DF.%26atitle%3DStructural%2520basis%2520of%2520ligand%2520recognition%2520and%2520self-activation%2520of%2520orphan%2520GPR52%26jtitle%3DNature%26date%3D2020%26volume%3D579%26spage%3D152%26epage%3D157%26doi%3D10.1038%2Fs41586-020-2019-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Trott, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, A. J.</span></span> <span> </span><span class="NLM_article-title">AutoDock Vina: improving the speed and accuracy of docking with a new scoring function, efficient optimization, and multithreading</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">455</span>– <span class="NLM_lpage">461</span>, <span class="refDoi"> DOI: 10.1002/jcc.21334</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1002%2Fjcc.21334" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=19499576" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFGnur3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2010&pages=455-461&author=O.+Trottauthor=A.+J.+Olson&title=AutoDock+Vina%3A+improving+the+speed+and+accuracy+of+docking+with+a+new+scoring+function%2C+efficient+optimization%2C+and+multithreading&doi=10.1002%2Fjcc.21334"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">AutoDock Vina: Improving the speed and accuracy of docking with a new scoring function, efficient optimization, and multithreading</span></div><div class="casAuthors">Trott, Oleg; Olson, Arthur J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">455-461</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">AutoDock Vina, a new program for mol. docking and virtual screening, is presented.  AutoDock Vina achieves an approx. 2 orders of magnitude speed-up compared with the mol. docking software previously developed in the authors' lab (AutoDock 4), while also significantly improving the accuracy of the binding mode predictions, judging by the authors' tests on the training set used in AutoDock 4 development.  Further speed-up is achieved from parallelism, by using multithreading on multicore machines.  AutoDock Vina automatically calcs. the grid maps and clusters the results in a way transparent to the user.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLecOIkUxIlrVg90H21EOLACvtfcHk0lhYG8C2CqnQOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFGnur3O&md5=c6974af8a1235f7aa09918d3e6f70dc4</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1002%2Fjcc.21334&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.21334%26sid%3Dliteratum%253Aachs%26aulast%3DTrott%26aufirst%3DO.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26atitle%3DAutoDock%2520Vina%253A%2520improving%2520the%2520speed%2520and%2520accuracy%2520of%2520docking%2520with%2520a%2520new%2520scoring%2520function%252C%2520efficient%2520optimization%252C%2520and%2520multithreading%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2010%26volume%3D31%26spage%3D455%26epage%3D461%26doi%3D10.1002%2Fjcc.21334" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vonsattel, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myers, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevens, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrante, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bird, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, E. P.,  Jr.</span></span> <span> </span><span class="NLM_article-title">Neuropathological classification of Huntington’s disease</span>. <i>J. Neuropathol. Exp. Neurol.</i> <span class="NLM_year" style="font-weight: bold;">1985</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">559</span>– <span class="NLM_lpage">577</span>, <span class="refDoi"> DOI: 10.1097/00005072-198511000-00003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1097%2F00005072-198511000-00003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=2932539" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A280%3ADyaL28%252Fjs1egsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=1985&pages=559-577&author=J.+P.+Vonsattelauthor=R.+H.+Myersauthor=T.+J.+Stevensauthor=R.+J.+Ferranteauthor=E.+D.+Birdauthor=E.+P.+Richardson&title=Neuropathological+classification+of+Huntington%E2%80%99s+disease&doi=10.1097%2F00005072-198511000-00003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Neuropathological classification of Huntington's disease</span></div><div class="casAuthors">Vonsattel J P; Myers R H; Stevens T J; Ferrante R J; Bird E D; Richardson E P Jr</div><div class="citationInfo"><span class="NLM_cas:title">Journal of neuropathology and experimental neurology</span>
        (<span class="NLM_cas:date">1985</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">559-77</span>
        ISSN:<span class="NLM_cas:issn">0022-3069</span>.
    </div><div class="casAbstract">In postmortem brain specimens from 163 clinically diagnosed cases of Huntington's disease (HD) the striatum exhibited marked variation in the severity of neuropathological involvement.  A system for grading this severity was established by macroscopic and microscopic criteria, resulting in five grades (0-4) designated in ascending order of severity.  The grade correlates closely with the extent of clinical disability as assessed by a rating scale.  In five cases of clinically diagnosed HD there were no discernible neuropathological abnormalities (grade 0), suggesting that the anatomical changes lag behind the development of clinical abnormalities.  In eight cases, neuropathological changes could only be recognized microscopically (grade 1).  The earliest changes were seen in the medial paraventricular portions of the caudate nucleus (CN), in the tail of the CN, and in the dorsal part of the putamen.  Counts of neurons in the CN reveal that 50% are lost in grade 1 and that 95% are lost in grade 4; astrocytes are greatly increased in grades 2-4.  These studies indicate that analyses of the CN in grade 4 would reflect mainly its astrocytic composition with a component of remote neurons projecting to the striatum.  Because of the relative preservation of the lateral half of the head of the CN in grades 1-2, these regions would reflect early cellular and biochemical changes in HD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRSGVuf9RSW3lUIs6wFK5xJfW6udTcc2ebUixJx3Zyiu7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaL28%252Fjs1egsg%253D%253D&md5=82c0370acf5893966064699d68fc3d20</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1097%2F00005072-198511000-00003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252F00005072-198511000-00003%26sid%3Dliteratum%253Aachs%26aulast%3DVonsattel%26aufirst%3DJ.%2BP.%26aulast%3DMyers%26aufirst%3DR.%2BH.%26aulast%3DStevens%26aufirst%3DT.%2BJ.%26aulast%3DFerrante%26aufirst%3DR.%2BJ.%26aulast%3DBird%26aufirst%3DE.%2BD.%26aulast%3DRichardson%26aufirst%3DE.%2BP.%26atitle%3DNeuropathological%2520classification%2520of%2520Huntington%25E2%2580%2599s%2520disease%26jtitle%3DJ.%2520Neuropathol.%2520Exp.%2520Neurol.%26date%3D1985%26volume%3D44%26spage%3D559%26epage%3D577%26doi%3D10.1097%2F00005072-198511000-00003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vonsattel, J. P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DiFiglia, M.</span></span> <span> </span><span class="NLM_article-title">Huntington disease</span>. <i>J. Neuropathol. Exp. Neurol.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">369</span>– <span class="NLM_lpage">384</span>, <span class="refDoi"> DOI: 10.1097/00005072-199805000-00001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1097%2F00005072-199805000-00001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=9596408" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A280%3ADyaK1c3lsFyntQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=1998&pages=369-384&author=J.+P.+G.+Vonsattelauthor=M.+DiFiglia&title=Huntington+disease&doi=10.1097%2F00005072-199805000-00001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Huntington disease</span></div><div class="casAuthors">Vonsattel J P; DiFiglia M</div><div class="citationInfo"><span class="NLM_cas:title">Journal of neuropathology and experimental neurology</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">369-84</span>
        ISSN:<span class="NLM_cas:issn">0022-3069</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ_UbgFnqTFpOxsqfvU0ziafW6udTcc2ebUixJx3Zyiu7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK1c3lsFyntQ%253D%253D&md5=3a2a5f5badf87489aa747a6a29a6f8d6</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1097%2F00005072-199805000-00001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252F00005072-199805000-00001%26sid%3Dliteratum%253Aachs%26aulast%3DVonsattel%26aufirst%3DJ.%2BP.%2BG.%26aulast%3DDiFiglia%26aufirst%3DM.%26atitle%3DHuntington%2520disease%26jtitle%3DJ.%2520Neuropathol.%2520Exp.%2520Neurol.%26date%3D1998%26volume%3D57%26spage%3D369%26epage%3D384%26doi%3D10.1097%2F00005072-199805000-00001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Waldvogel, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tippett, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vonsattel, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faull, R. L.</span></span> <span> </span><span class="NLM_article-title">The neuropathology of Huntington’s disease</span>. <i>Curr. Top. Behav. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">33</span>– <span class="NLM_lpage">80</span>, <span class="refDoi"> DOI: 10.1007/7854_2014_354</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1007%2F7854_2014_354" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=25300927" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A528%3ADC%252BC28XmsVOkt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2015&pages=33-80&author=H.+J.+Waldvogelauthor=E.+H.+Kimauthor=L.+J.+Tippettauthor=J.+P.+Vonsattelauthor=R.+L.+Faull&title=The+neuropathology+of+Huntington%E2%80%99s+disease&doi=10.1007%2F7854_2014_354"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">The Neuropathology of Huntington's Disease</span></div><div class="casAuthors">Waldvogel, Henry J.; Kim, Eric H.; Tippett, Lynette J.; Vonsattel, Jean-Paul G.; Faull, Richard L. M.</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Behavioral Neurosciences</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">Behavioral Neurobiology of Huntington's Disease and Parkinson's Disease</span>),
    <span class="NLM_cas:pages">33-80</span>CODEN:
                <span class="NLM_cas:coden">CTBNBW</span>;
        ISSN:<span class="NLM_cas:issn">1866-3389</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The basal ganglia are a highly interconnected set of subcortical nuclei and major atrophy in one or more regions may have major effects on other regions of the brain.  Therefore, the striatum which is preferentially degenerated and receives projections from the entire cortex also affects the regions to which it targets, esp. the globus pallidus and substantia nigra pars reticulata.  Addnl., the cerebral cortex is itself severely affected as are many other regions of the brain, esp. in more advanced cases.  The cell loss in the basal ganglia and the cerebral cortex is extensive.  The most important new findings in Huntington's disease pathol. is the highly variable nature of the degeneration in the brain.  Most interestingly, this variable pattern of pathol. appears to reflect the highly variable symptomatol. of cases with Huntington's disease even among cases possessing the same no. of CAG repeats.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAPVBC4isuNLVg90H21EOLACvtfcHk0li8fJdzY4Q-Iw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmsVOkt7w%253D&md5=7b78454249ab10e4debc22f34b0a8f49</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1007%2F7854_2014_354&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F7854_2014_354%26sid%3Dliteratum%253Aachs%26aulast%3DWaldvogel%26aufirst%3DH.%2BJ.%26aulast%3DKim%26aufirst%3DE.%2BH.%26aulast%3DTippett%26aufirst%3DL.%2BJ.%26aulast%3DVonsattel%26aufirst%3DJ.%2BP.%26aulast%3DFaull%26aufirst%3DR.%2BL.%26atitle%3DThe%2520neuropathology%2520of%2520Huntington%25E2%2580%2599s%2520disease%26jtitle%3DCurr.%2520Top.%2520Behav.%2520Neurosci.%26date%3D2015%26volume%3D22%26spage%3D33%26epage%3D80%26doi%3D10.1007%2F7854_2014_354" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Waldvogel, H. J.</span>; <span class="NLM_string-name">Thu, D.</span>; <span class="NLM_string-name">Hogg, V.</span>; <span class="NLM_string-name">Tippett, L.</span>; <span class="NLM_string-name">Faull, R. L.</span></span> <span> </span><span class="NLM_article-title">Selective neurodegeneration, neuropathology and symptom profiles in Huntington’s disease</span>. In  <i>Advances in Experimental Medicine and Biology</i>; <span class="NLM_publisher-name">Springer</span>: <span class="NLM_publisher-loc">New York, NY</span>, <span class="NLM_year">2012</span>; Vol.  <span class="NLM_volume">769</span>, pp  <span class="NLM_fpage">141</span>– <span class="NLM_lpage">152</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&pages=141-152&author=H.+J.+Waldvogel&author=D.+Thu&author=V.+Hogg&author=L.+Tippett&author=R.+L.+Faull&title=Advances+in+Experimental+Medicine+and+Biology"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DWaldvogel%26aufirst%3DH.%2BJ.%26atitle%3DSelective%2520neurodegeneration%252C%2520neuropathology%2520and%2520symptom%2520profiles%2520in%2520Huntington%25E2%2580%2599s%2520disease%26btitle%3DAdvances%2520in%2520Experimental%2520Medicine%2520and%2520Biology%26pub%3DSpringer%26date%3D2012%26volume%3D769%26spage%3D141%26epage%3D152" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Consortium, H. D. i.</span></span> <span> </span><span class="NLM_article-title">Induced pluripotent stem cells from patients with Huntington’s disease show CAG-repeat-expansion-associated phenotypes</span>. <i>Cell Stem Cell</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">264</span>– <span class="NLM_lpage">278</span>, <span class="refDoi"> DOI: 10.1016/j.stem.2012.04.027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1016%2Fj.stem.2012.04.027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=22748968" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=264-278&author=H.+D.+i.+Consortium&title=Induced+pluripotent+stem+cells+from+patients+with+Huntington%E2%80%99s+disease+show+CAG-repeat-expansion-associated+phenotypes&doi=10.1016%2Fj.stem.2012.04.027"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.stem.2012.04.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.stem.2012.04.027%26sid%3Dliteratum%253Aachs%26aulast%3DConsortium%26aufirst%3DH.%2BD.%2Bi.%26atitle%3DInduced%2520pluripotent%2520stem%2520cells%2520from%2520patients%2520with%2520Huntington%25E2%2580%2599s%2520disease%2520show%2520CAG-repeat-expansion-associated%2520phenotypes%26jtitle%3DCell%2520Stem%2520Cell%26date%3D2012%26volume%3D11%26spage%3D264%26epage%3D278%26doi%3D10.1016%2Fj.stem.2012.04.027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palacino, J.</span></span> <span> </span><span class="NLM_article-title">A novel human embryonic stem cell-derived Huntington’s disease neuronal model exhibits mutant huntingtin (mHTT) aggregates and soluble mHTT-dependent neurodegeneration</span>. <i>FASEB J.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">1820</span>– <span class="NLM_lpage">1829</span>, <span class="refDoi"> DOI: 10.1096/fj.12-219220</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1096%2Ffj.12-219220" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=23325320" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A528%3ADC%252BC3sXnsV2ktb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2013&pages=1820-1829&author=B.+Luauthor=J.+Palacino&title=A+novel+human+embryonic+stem+cell-derived+Huntington%E2%80%99s+disease+neuronal+model+exhibits+mutant+huntingtin+%28mHTT%29+aggregates+and+soluble+mHTT-dependent+neurodegeneration&doi=10.1096%2Ffj.12-219220"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">A novel human embryonic stem cell-derived Huntington's disease neuronal model exhibits mutant huntingtin (mHTT) aggregates and soluble mHTT-dependent neurodegeneration</span></div><div class="casAuthors">Lu, Boxun; Palacino, James</div><div class="citationInfo"><span class="NLM_cas:title">FASEB Journal</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1820-1829, 10.1096/fj.12-219220</span>CODEN:
                <span class="NLM_cas:coden">FAJOEC</span>;
        ISSN:<span class="NLM_cas:issn">0892-6638</span>.
    
            (<span class="NLM_cas:orgname">Federation of American Societies for Experimental Biology</span>)
        </div><div class="casAbstract">Most neurodegenerative diseases are linked to aberrant accumulation of aggregation-prone proteins.  Among them, Huntington's disease (HD) is caused by an expanded polyglutamine repeat stretch in the N terminus of the mutant huntingtin protein (mHTT), which gets cleaved and aggregates in the brain.  Recently established human induced pluripotent stem cell-derived HD neurons exhibit some disease-relevant phenotypes and provide tools for HD research.  However, they have limitations such as genetic heterogeneity and an absence of mHTT aggregates and lack a robust neurodegeneration phenotype.  In addn., the relationship between the phenotype and mHTT levels has not been elucidated.  Herein, we present a human embryonic stem cell (hESC)-derived HD neuronal model expressing HTTexon1 fragments, which addresses the deficiencies enumerated above.  The wild-type and HD lines are derived from an isogenic background and exhibit insol. mHTT aggregates and neurodegeneration.  We also demonstrate a quant. relationship between neurodegeneration and sol. monomeric (but not oligomeric or aggregated) mHTT levels.  Redn. of ∼10% of mHTT is sufficient to prevent toxicity, whereas ∼90% redn. of wild-type HTT is safe and well-tolerated in these cells.  A known HD toxicity modifier (Rhes) showed expected rescue of neurodegeneration.  Therefore, the hESC-derived neuronal models complement existing induced pluripotent stem cell-derived neuronal models and provide valuable tools for HD research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUsb9COGsBh7Vg90H21EOLACvtfcHk0ljIKWFps70exQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXnsV2ktb0%253D&md5=f3db78085de1e8bad73f202a7637b016</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1096%2Ffj.12-219220&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1096%252Ffj.12-219220%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DB.%26aulast%3DPalacino%26aufirst%3DJ.%26atitle%3DA%2520novel%2520human%2520embryonic%2520stem%2520cell-derived%2520Huntington%25E2%2580%2599s%2520disease%2520neuronal%2520model%2520exhibits%2520mutant%2520huntingtin%2520%2528mHTT%2529%2520aggregates%2520and%2520soluble%2520mHTT-dependent%2520neurodegeneration%26jtitle%3DFASEB%2520J.%26date%3D2013%26volume%3D27%26spage%3D1820%26epage%3D1829%26doi%3D10.1096%2Ffj.12-219220" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tsvetkov, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrasate, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pleiss, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finkbeiner, S.</span></span> <span> </span><span class="NLM_article-title">A small-molecule scaffold induces autophagy in primary neurons and protects against toxicity in a Huntington disease model</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>107</i></span>,  <span class="NLM_fpage">16982</span>– <span class="NLM_lpage">16987</span>, <span class="refDoi"> DOI: 10.1073/pnas.1004498107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1073%2Fpnas.1004498107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=20833817" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1Oht7fP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2010&pages=16982-16987&author=A.+S.+Tsvetkovauthor=J.+Millerauthor=M.+Arrasateauthor=J.+S.+Wongauthor=M.+A.+Pleissauthor=S.+Finkbeiner&title=A+small-molecule+scaffold+induces+autophagy+in+primary+neurons+and+protects+against+toxicity+in+a+Huntington+disease+model&doi=10.1073%2Fpnas.1004498107"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">A small-molecule scaffold induces autophagy in primary neurons and protects against toxicity in a Huntington disease model</span></div><div class="casAuthors">Tsvetkov, Andrey S.; Miller, Jason; Arrasate, Montserrat; Wong, Jinny S.; Pleiss, Michael A.; Finkbeiner, Steven</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue">39</span>),
    <span class="NLM_cas:pages">16982-16987, S16982/1-S16982/6</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Autophagy is an intracellular turnover pathway.  It has special relevance for neurodegenerative proteinopathies, such as Alzheimer disease, Parkinson disease, and Huntington disease (HD), which are characterized by the accumulation of misfolded proteins.  Although induction of autophagy enhances clearance of misfolded protein and has therefore been suggested as a therapy for proteinopathies, neurons appear to be less responsive to classic autophagy inducers than nonneuronal cells.  Searching for improved inducers of neuronal autophagy, we discovered an N10-substituted phenoxazine that, at proper doses, potently and safely up-regulated autophagy in neurons in an Akt- and mTOR-independent fashion.  In a neuron model of HD, this compd. was neuroprotective and decreased the accumulation of diffuse and aggregated misfolded protein.  A structure/activity anal. with structurally similar compds. approved by the US Food and Drug Administration revealed a defined pharmacophore for inducing neuronal autophagy.  This pharmacophore should prove useful in studying autophagy in neurons and in developing therapies for neurodegenerative proteinopathies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtnUfvW65_mrVg90H21EOLACvtfcHk0ljIKWFps70exQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1Oht7fP&md5=aaefe7b3fc1c0e12c53b67c660219767</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1004498107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1004498107%26sid%3Dliteratum%253Aachs%26aulast%3DTsvetkov%26aufirst%3DA.%2BS.%26aulast%3DMiller%26aufirst%3DJ.%26aulast%3DArrasate%26aufirst%3DM.%26aulast%3DWong%26aufirst%3DJ.%2BS.%26aulast%3DPleiss%26aufirst%3DM.%2BA.%26aulast%3DFinkbeiner%26aufirst%3DS.%26atitle%3DA%2520small-molecule%2520scaffold%2520induces%2520autophagy%2520in%2520primary%2520neurons%2520and%2520protects%2520against%2520toxicity%2520in%2520a%2520Huntington%2520disease%2520model%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2010%26volume%3D107%26spage%3D16982%26epage%3D16987%26doi%3D10.1073%2Fpnas.1004498107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Landles, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sathasivam, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodman, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moffitt, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finkbeiner, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gafni, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellerby, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trottier, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richards, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osmand, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paganetti, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bates, G. P.</span></span> <span> </span><span class="NLM_article-title">Proteolysis of mutant huntingtin produces an exon 1 fragment that accumulates as an aggregated protein in neuronal nuclei in Huntington disease</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>285</i></span>,  <span class="NLM_fpage">8808</span>– <span class="NLM_lpage">8823</span>, <span class="refDoi"> DOI: 10.1074/jbc.M109.075028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1074%2Fjbc.M109.075028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=20086007" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjtFShsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=8808-8823&author=C.+Landlesauthor=K.+Sathasivamauthor=A.+Weissauthor=B.+Woodmanauthor=H.+Moffittauthor=S.+Finkbeinerauthor=B.+Sunauthor=J.+Gafniauthor=L.+M.+Ellerbyauthor=Y.+Trottierauthor=W.+G.+Richardsauthor=A.+Osmandauthor=P.+Paganettiauthor=G.+P.+Bates&title=Proteolysis+of+mutant+huntingtin+produces+an+exon+1+fragment+that+accumulates+as+an+aggregated+protein+in+neuronal+nuclei+in+Huntington+disease&doi=10.1074%2Fjbc.M109.075028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Proteolysis of Mutant Huntingtin Produces an Exon 1 Fragment That Accumulates as an Aggregated Protein in Neuronal Nuclei in Huntington Disease</span></div><div class="casAuthors">Landles, Christian; Sathasivam, Kirupa; Weiss, Andreas; Woodman, Ben; Moffitt, Hilary; Finkbeiner, Steve; Sun, Banghua; Gafni, Juliette; Ellerby, Lisa M.; Trottier, Yvon; Richards, William G.; Osmand, Alex; Paganetti, Paolo; Bates, Gillian P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">285</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">8808-8823</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Huntingtin proteolysis has been implicated in the mol. pathogenesis of Huntington disease (HD).  Despite an intense effort, the identity of the pathogenic smallest N-terminal fragment has not been detd.  Using a panel of anti-huntingtin antibodies, we employed an unbiased approach to generate proteolytic cleavage maps of mutant and wild-type huntingtin in the HdhQ150 knock-in mouse model of HD.  We identified 14 prominent N-terminal fragments, which, in addn. to the full-length protein, can be readily detected in cytoplasmic but not nuclear fractions.  These fragments were detected at all ages and are not a consequence of the pathogenic process.  We demonstrated that the smallest fragment is an exon 1 huntingtin protein, known to contain a potent nuclear export signal.  Prior to the onset of behavioral phenotypes, the exon 1 protein, and possibly other small fragments, accumulate in neuronal nuclei in the form of a detergent insol. complex, visualized as diffuse granular nuclear staining in tissue sections.  This methodol. can be used to validate the inhibition of specific proteases as therapeutic targets for HD by pharmacol. or genetic approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoiRwmWU6l4i7Vg90H21EOLACvtfcHk0ljIKWFps70exQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjtFShsLg%253D&md5=364988ca7c2f1de0ca697b33baf35bec</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M109.075028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M109.075028%26sid%3Dliteratum%253Aachs%26aulast%3DLandles%26aufirst%3DC.%26aulast%3DSathasivam%26aufirst%3DK.%26aulast%3DWeiss%26aufirst%3DA.%26aulast%3DWoodman%26aufirst%3DB.%26aulast%3DMoffitt%26aufirst%3DH.%26aulast%3DFinkbeiner%26aufirst%3DS.%26aulast%3DSun%26aufirst%3DB.%26aulast%3DGafni%26aufirst%3DJ.%26aulast%3DEllerby%26aufirst%3DL.%2BM.%26aulast%3DTrottier%26aufirst%3DY.%26aulast%3DRichards%26aufirst%3DW.%2BG.%26aulast%3DOsmand%26aufirst%3DA.%26aulast%3DPaganetti%26aufirst%3DP.%26aulast%3DBates%26aufirst%3DG.%2BP.%26atitle%3DProteolysis%2520of%2520mutant%2520huntingtin%2520produces%2520an%2520exon%25201%2520fragment%2520that%2520accumulates%2520as%2520an%2520aggregated%2520protein%2520in%2520neuronal%2520nuclei%2520in%2520Huntington%2520disease%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26spage%3D8808%26epage%3D8823%26doi%3D10.1074%2Fjbc.M109.075028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Crotti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glass, C. K.</span></span> <span> </span><span class="NLM_article-title">The choreography of neuroinflammation in Huntington’s disease</span>. <i>Trends Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">364</span>– <span class="NLM_lpage">373</span>, <span class="refDoi"> DOI: 10.1016/j.it.2015.04.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1016%2Fj.it.2015.04.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=26001312" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A528%3ADC%252BC2MXnslOrs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2015&pages=364-373&author=A.+Crottiauthor=C.+K.+Glass&title=The+choreography+of+neuroinflammation+in+Huntington%E2%80%99s+disease&doi=10.1016%2Fj.it.2015.04.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">The choreography of neuroinflammation in Huntington's disease</span></div><div class="casAuthors">Crotti, Andrea; Glass, Christopher K.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Immunology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">364-373</span>CODEN:
                <span class="NLM_cas:coden">TIRMAE</span>;
        ISSN:<span class="NLM_cas:issn">1471-4906</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Currently, the concept of 'neuroinflammation' includes inflammation assocd. with neurodegenerative diseases, in which there is little or no infiltration of blood-derived immune cells into the brain.  The roles of brain-resident and peripheral immune cells in these inflammatory settings are poorly understood, and it is unclear whether neuroinflammation results from immune reaction to neuronal dysfunction/degeneration, and/or represents cell-autonomous phenotypes of dysfunctional immune cells.  Here, we review recent studies examg. these questions in the context of Huntington's disease (HD), where mutant Huntingtin (HTT) is expressed in both neurons and glia.  Insights into the cellular and mol. mechanisms underlying neuroinflammation in HD may provide a better understanding of inflammation in more complex neurodegenerative disorders, and of the contribution of the neuroinflammatory component to neurodegenerative disease pathogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgjoW-X08wH7Vg90H21EOLACvtfcHk0lgrIiTl0BlmLg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXnslOrs7s%253D&md5=69297dd08b2c0c2d0618f261292117d2</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.it.2015.04.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.it.2015.04.007%26sid%3Dliteratum%253Aachs%26aulast%3DCrotti%26aufirst%3DA.%26aulast%3DGlass%26aufirst%3DC.%2BK.%26atitle%3DThe%2520choreography%2520of%2520neuroinflammation%2520in%2520Huntington%25E2%2580%2599s%2520disease%26jtitle%3DTrends%2520Immunol.%26date%3D2015%26volume%3D36%26spage%3D364%26epage%3D373%26doi%3D10.1016%2Fj.it.2015.04.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Drouin-Ouellet, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sawiak, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cisbani, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lagace, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuan, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saint-Pierre, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dury, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alata, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">St-Amour, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mason, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calon, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lacroix, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gowland, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francis, S. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barker, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cicchetti, F.</span></span> <span> </span><span class="NLM_article-title">Cerebrovascular and blood-brain barrier impairments in Huntington’s disease: Potential implications for its pathophysiology</span>. <i>Ann. Neurol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>78</i></span>,  <span class="NLM_fpage">160</span>– <span class="NLM_lpage">177</span>, <span class="refDoi"> DOI: 10.1002/ana.24406</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1002%2Fana.24406" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=25866151" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A280%3ADC%252BC2MjjtlWhtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=2015&pages=160-177&author=J.+Drouin-Ouelletauthor=S.+J.+Sawiakauthor=G.+Cisbaniauthor=M.+Lagaceauthor=W.+L.+Kuanauthor=M.+Saint-Pierreauthor=R.+J.+Duryauthor=W.+Alataauthor=I.+St-Amourauthor=S.+L.+Masonauthor=F.+Calonauthor=S.+Lacroixauthor=P.+A.+Gowlandauthor=S.+T.+Francisauthor=R.+A.+Barkerauthor=F.+Cicchetti&title=Cerebrovascular+and+blood-brain+barrier+impairments+in+Huntington%E2%80%99s+disease%3A+Potential+implications+for+its+pathophysiology&doi=10.1002%2Fana.24406"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Cerebrovascular and blood-brain barrier impairments in Huntington's disease: Potential implications for its pathophysiology</span></div><div class="casAuthors">Drouin-Ouellet Janelle; Kuan Wei-Li; Mason Sarah L; Barker Roger A; Sawiak Stephen J; Cisbani Giulia; Lagace Marie; Saint-Pierre Martine; Alata Wael; St-Amour Isabelle; Calon Frederic; Lacroix Steve; Cicchetti Francesca; Dury Richard J; Gowland Penny A; Francis Susan T; Calon Frederic; Lacroix Steve; Cicchetti Francesca</div><div class="citationInfo"><span class="NLM_cas:title">Annals of neurology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">78</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">160-77</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">OBJECTIVE:  Although the underlying cause of Huntington's disease (HD) is well established, the actual pathophysiological processes involved remain to be fully elucidated.  In other proteinopathies such as Alzheimer's and Parkinson's diseases, there is evidence for impairments of the cerebral vasculature as well as the blood-brain barrier (BBB), which have been suggested to contribute to their pathophysiology.  We investigated whether similar changes are also present in HD.  METHODS:  We used 3- and 7-Tesla magnetic resonance imaging as well as postmortem tissue analyses to assess blood vessel impairments in HD patients.  Our findings were further investigated in the R6/2 mouse model using in situ cerebral perfusion, histological analysis, Western blotting, as well as transmission and scanning electron microscopy.  RESULTS:  We found mutant huntingtin protein (mHtt) aggregates to be present in all major components of the neurovascular unit of both R6/2 mice and HD patients.  This was accompanied by an increase in blood vessel density, a reduction in blood vessel diameter, as well as BBB leakage in the striatum of R6/2 mice, which correlated with a reduced expression of tight junction-associated proteins and increased numbers of transcytotic vesicles, which occasionally contained mHtt aggregates.  We confirmed the existence of similar vascular and BBB changes in HD patients.  INTERPRETATION:  Taken together, our results provide evidence for alterations in the cerebral vasculature in HD leading to BBB leakage, both in the R6/2 mouse model and in HD patients, a phenomenon that may, in turn, have important pathophysiological implications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQFAxLe_E6aqGhHGZei4OW9fW6udTcc2eaiXDUzLl_79bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MjjtlWhtA%253D%253D&md5=210d003662c40e370d866e7d04b8bb07</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1002%2Fana.24406&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fana.24406%26sid%3Dliteratum%253Aachs%26aulast%3DDrouin-Ouellet%26aufirst%3DJ.%26aulast%3DSawiak%26aufirst%3DS.%2BJ.%26aulast%3DCisbani%26aufirst%3DG.%26aulast%3DLagace%26aufirst%3DM.%26aulast%3DKuan%26aufirst%3DW.%2BL.%26aulast%3DSaint-Pierre%26aufirst%3DM.%26aulast%3DDury%26aufirst%3DR.%2BJ.%26aulast%3DAlata%26aufirst%3DW.%26aulast%3DSt-Amour%26aufirst%3DI.%26aulast%3DMason%26aufirst%3DS.%2BL.%26aulast%3DCalon%26aufirst%3DF.%26aulast%3DLacroix%26aufirst%3DS.%26aulast%3DGowland%26aufirst%3DP.%2BA.%26aulast%3DFrancis%26aufirst%3DS.%2BT.%26aulast%3DBarker%26aufirst%3DR.%2BA.%26aulast%3DCicchetti%26aufirst%3DF.%26atitle%3DCerebrovascular%2520and%2520blood-brain%2520barrier%2520impairments%2520in%2520Huntington%25E2%2580%2599s%2520disease%253A%2520Potential%2520implications%2520for%2520its%2520pathophysiology%26jtitle%3DAnn.%2520Neurol.%26date%3D2015%26volume%3D78%26spage%3D160%26epage%3D177%26doi%3D10.1002%2Fana.24406" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baldo, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paganetti, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grueninger, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcellin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaltenbach, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lo, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Semmelroth, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zivanovic, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abramowski, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lotz, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bates, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, A.</span></span> <span> </span><span class="NLM_article-title">TR-FRET-based duplex immunoassay reveals an inverse correlation of soluble and aggregated mutant huntingtin in huntington’s disease</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">264</span>– <span class="NLM_lpage">275</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2011.12.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1016%2Fj.chembiol.2011.12.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=22365609" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A528%3ADC%252BC38XjtFWhtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2012&pages=264-275&author=B.+Baldoauthor=P.+Paganettiauthor=S.+Grueningerauthor=D.+Marcellinauthor=L.+S.+Kaltenbachauthor=D.+C.+Loauthor=M.+Semmelrothauthor=A.+Zivanovicauthor=D.+Abramowskiauthor=D.+Smithauthor=G.+P.+Lotzauthor=G.+P.+Batesauthor=A.+Weiss&title=TR-FRET-based+duplex+immunoassay+reveals+an+inverse+correlation+of+soluble+and+aggregated+mutant+huntingtin+in+huntington%E2%80%99s+disease&doi=10.1016%2Fj.chembiol.2011.12.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">TR-FRET-Based Duplex Immunoassay Reveals an Inverse Correlation of Soluble and Aggregated Mutant huntingtin in Huntington's Disease</span></div><div class="casAuthors">Baldo, Barbara; Paganetti, Paolo; Grueninger, Stephan; Marcellin, David; Kaltenbach, Linda S.; Lo, Donald C.; Semmelroth, Martin; Zivanovic, Andjelija; Abramowski, Dorothee; Smith, Donna; Lotz, Gregor P.; Bates, Gillian P.; Weiss, Andreas</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">264-275</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Huntington's disease (HD) is an inherited neurodegenerative disorder caused by the amplification of a polyglutamine stretch at the N terminus of the huntingtin protein.  N-terminal fragments of the mutant huntingtin (mHtt) aggregate and form intracellular inclusions in brain and peripheral tissues.  Aggregates are an important hallmark of the disease, translating into a high need to quantify them in vitro and in vivo.  We developed a one-step TR-FRET-based immunoassay to quantify sol. and aggregated mHtt in cell and tissue homogenates.  Strikingly, quantification revealed a decrease of sol. mHtt correlating with an increase of aggregated protein in primary neuronal cell cultures, transgenic R6/2, and HdhQ150 knock-in HD mice.  These results emphasize the assay's efficiency for highly sensitive and quant. detection of sol. and aggregated mHtt and its application in high-throughput screening and characterization of HD models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHjXLeXbarcrVg90H21EOLACvtfcHk0lgrIiTl0BlmLg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjtFWhtLo%253D&md5=ff1f0f82383793c05895bd86ec9cec3a</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2011.12.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2011.12.020%26sid%3Dliteratum%253Aachs%26aulast%3DBaldo%26aufirst%3DB.%26aulast%3DPaganetti%26aufirst%3DP.%26aulast%3DGrueninger%26aufirst%3DS.%26aulast%3DMarcellin%26aufirst%3DD.%26aulast%3DKaltenbach%26aufirst%3DL.%2BS.%26aulast%3DLo%26aufirst%3DD.%2BC.%26aulast%3DSemmelroth%26aufirst%3DM.%26aulast%3DZivanovic%26aufirst%3DA.%26aulast%3DAbramowski%26aufirst%3DD.%26aulast%3DSmith%26aufirst%3DD.%26aulast%3DLotz%26aufirst%3DG.%2BP.%26aulast%3DBates%26aufirst%3DG.%2BP.%26aulast%3DWeiss%26aufirst%3DA.%26atitle%3DTR-FRET-based%2520duplex%2520immunoassay%2520reveals%2520an%2520inverse%2520correlation%2520of%2520soluble%2520and%2520aggregated%2520mutant%2520huntingtin%2520in%2520huntington%25E2%2580%2599s%2520disease%26jtitle%3DChem.%2520Biol.%26date%3D2012%26volume%3D19%26spage%3D264%26epage%3D275%26doi%3D10.1016%2Fj.chembiol.2011.12.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bano, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zanetti, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mende, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicotera, P.</span></span> <span> </span><span class="NLM_article-title">Neurodegenerative processes in Huntington’s disease</span>. <i>Cell Death Dis.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>2</i></span>, <span class="NLM_elocation-id">e228</span> <span class="refDoi"> DOI: 10.1038/cddis.2011.112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1038%2Fcddis.2011.112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=22071633" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A528%3ADC%252BC38XmvVWl" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2011&author=D.+Banoauthor=F.+Zanettiauthor=Y.+Mendeauthor=P.+Nicotera&title=Neurodegenerative+processes+in+Huntington%E2%80%99s+disease&doi=10.1038%2Fcddis.2011.112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Neurodegenerative processes in Huntington's disease</span></div><div class="casAuthors">Bano, D.; Zanetti, F.; Mende, Y.; Nicotera, P.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Death & Disease</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">Nov.</span>),
    <span class="NLM_cas:pages">e228/1-e228/7</span>CODEN:
                <span class="NLM_cas:coden">CDDEA4</span>;
        ISSN:<span class="NLM_cas:issn">2041-4889</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Huntington's disease (HD) is a complex and severe disorder characterized by the gradual and the progressive loss of neurons, predominantly in the striatum, which leads to the typical motor and cognitive impairments assocd. with this pathol.  HD is caused by a highly polymorphic CAG trinucleotide repeat expansion in the exon-1 of the gene encoding for huntingtin protein.  Since the first discovery of the huntingtin gene, investigations with a consistent no. of in-vitro and in-vivo models have provided insights into the toxic events related to the expression of the mutant protein.  In this review, we will summarize the progress made in characterizing the signaling pathways that contribute to neuronal degeneration in HD.  We will highlight the age-dependent loss of proteostasis that is primarily responsible for the formation of aggregates obsd. in HD patients.  The most promising mol. targets for the development of pharmacol. interventions will also be discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrpilHy3akf4rVg90H21EOLACvtfcHk0lh0jnZrw749ow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmvVWl&md5=3b1bba67bff5b24a312cd85c02378909</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1038%2Fcddis.2011.112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fcddis.2011.112%26sid%3Dliteratum%253Aachs%26aulast%3DBano%26aufirst%3DD.%26aulast%3DZanetti%26aufirst%3DF.%26aulast%3DMende%26aufirst%3DY.%26aulast%3DNicotera%26aufirst%3DP.%26atitle%3DNeurodegenerative%2520processes%2520in%2520Huntington%25E2%2580%2599s%2520disease%26jtitle%3DCell%2520Death%2520Dis.%26date%3D2011%26volume%3D2%26doi%3D10.1038%2Fcddis.2011.112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Labbadia, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morimoto, R. I.</span></span> <span> </span><span class="NLM_article-title">Huntington’s disease: underlying molecular mechanisms and emerging concepts</span>. <i>Trends Biochem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">378</span>– <span class="NLM_lpage">385</span>, <span class="refDoi"> DOI: 10.1016/j.tibs.2013.05.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1016%2Fj.tibs.2013.05.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=23768628" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpsV2nsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2013&pages=378-385&author=J.+Labbadiaauthor=R.+I.+Morimoto&title=Huntington%E2%80%99s+disease%3A+underlying+molecular+mechanisms+and+emerging+concepts&doi=10.1016%2Fj.tibs.2013.05.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Huntington's disease: underlying molecular mechanisms and emerging concepts</span></div><div class="casAuthors">Labbadia, John; Morimoto, Richard I.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">378-385</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Huntington's disease (HD) is a progressive neurodegenerative disorder for which no disease modifying treatments exist.  Many mol. changes and cellular consequences that underlie HD are obsd. in other neurol. disorders, suggesting that common pathol. mechanisms and pathways may exist.  Recent findings have enhanced our understanding of the way cells regulate and respond to expanded polyglutamine proteins such as mutant huntingtin.  These studies demonstrate that in addn. to effects on folding, aggregation, and clearance pathways, a general transcriptional mechanism also dictates the expression of polyglutamine proteins.  Here, we summarize the key pathways and networks that are important in HD in the context of recent therapeutic advances and highlight how their interplay may be of relevance to other protein folding disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEk5nJJvEz67Vg90H21EOLACvtfcHk0lh0jnZrw749ow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXpsV2nsbk%253D&md5=422e5b7f479bd523abf7afefc135585c</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1016%2Fj.tibs.2013.05.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tibs.2013.05.003%26sid%3Dliteratum%253Aachs%26aulast%3DLabbadia%26aufirst%3DJ.%26aulast%3DMorimoto%26aufirst%3DR.%2BI.%26atitle%3DHuntington%25E2%2580%2599s%2520disease%253A%2520underlying%2520molecular%2520mechanisms%2520and%2520emerging%2520concepts%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D2013%26volume%3D38%26spage%3D378%26epage%3D385%26doi%3D10.1016%2Fj.tibs.2013.05.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaemmerer, W. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grondin, R. C.</span></span> <span> </span><span class="NLM_article-title">The effects of huntingtin-lowering: what do we know so far?</span>. <i>Degener. Neurol. Neuromuscular Dis.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>Volume 9</i></span>,  <span class="NLM_fpage">3</span>– <span class="NLM_lpage">17</span>, <span class="refDoi"> DOI: 10.2147/DNND.S163808</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.2147%2FDNND.S163808" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=Volume+9&publication_year=2019&pages=3-17&author=W.+F.+Kaemmererauthor=R.+C.+Grondin&title=The+effects+of+huntingtin-lowering%3A+what+do+we+know+so+far%3F&doi=10.2147%2FDNND.S163808"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.2147%2FDNND.S163808&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2147%252FDNND.S163808%26sid%3Dliteratum%253Aachs%26aulast%3DKaemmerer%26aufirst%3DW.%2BF.%26aulast%3DGrondin%26aufirst%3DR.%2BC.%26atitle%3DThe%2520effects%2520of%2520huntingtin-lowering%253A%2520what%2520do%2520we%2520know%2520so%2520far%253F%26jtitle%3DDegener.%2520Neurol.%2520Neuromuscular%2520Dis.%26date%3D2019%26volume%3DVolume%25209%26spage%3D3%26epage%3D17%26doi%3D10.2147%2FDNND.S163808" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Drouet, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perrin, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hassig, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dufour, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Auregan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alves, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonvento, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brouillet, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luthi-Carter, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hantraye, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deglon, N.</span></span> <span> </span><span class="NLM_article-title">Sustained effects of nonallele-specific Huntingtin silencing</span>. <i>Ann. Neurol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>65</i></span>,  <span class="NLM_fpage">276</span>– <span class="NLM_lpage">285</span>, <span class="refDoi"> DOI: 10.1002/ana.21569</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1002%2Fana.21569" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=19334076" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A528%3ADC%252BD1MXltVOkt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2009&pages=276-285&author=V.+Drouetauthor=V.+Perrinauthor=R.+Hassigauthor=N.+Dufourauthor=G.+Aureganauthor=S.+Alvesauthor=G.+Bonventoauthor=E.+Brouilletauthor=R.+Luthi-Carterauthor=P.+Hantrayeauthor=N.+Deglon&title=Sustained+effects+of+nonallele-specific+Huntingtin+silencing&doi=10.1002%2Fana.21569"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Sustained effects of nonallele-specific Huntingtin silencing</span></div><div class="casAuthors">Drouet, Valerie; Perrin, Valerie; Hassig, Raymonde; Dufour, Noelle; Auregan, Gwennaelle; Alves, Sandro; Bonvento, Gilles; Brouillet, Emmanuel; Luthi-Carter, Ruth; Hantraye, Philippe; Deglon, Nicole</div><div class="citationInfo"><span class="NLM_cas:title">Annals of Neurology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">276-285</span>CODEN:
                <span class="NLM_cas:coden">ANNED3</span>;
        ISSN:<span class="NLM_cas:issn">0364-5134</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">Objective: Huntington's disease (HD) is a fatal autosomal dominant neurodegenerative disorder caused by a poly-glutamine expansion in the huntingtin (htt) protein.  No cure is available to date to alleviate neurodegeneration.  Recent studies have demonstrated that RNA interference represents a promising approach for the treatment of autosomal dominant disorders.  But whether an allele-specific silencing of mutant htt or a nonallele-specific silencing should be considered has not been addressed.  Methods: the authors developed small hairpin RNA targeting mutant or wild-type htt transcripts, or both.  Results: the authors confirmed the therapeutic potential of sihtt administered with lentiviral vectors in rodent models of HD and showed that initiation of small interfering RNA treatment after the onset of HD symptoms is still efficacious and reduces the HD-like pathol.  The authors then addressed the question of the impact of nonallele-specific silencing and demonstrated that silencing of endogenous htt to 25 to 35% in vivo is altering several pathways assocd. with known htt functions but is not inducing overt toxicity or increasing striatal vulnerability up to 9 mo after treatment.  Interpretation: These data indicate that the coincident silencing of the wild-type and mutant htt may be considered as a therapeutic tool for HD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohEfZnKTt33bVg90H21EOLACvtfcHk0lh0jnZrw749ow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXltVOkt70%253D&md5=0f66bd61902ff3bbe0a789af727ca651</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1002%2Fana.21569&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fana.21569%26sid%3Dliteratum%253Aachs%26aulast%3DDrouet%26aufirst%3DV.%26aulast%3DPerrin%26aufirst%3DV.%26aulast%3DHassig%26aufirst%3DR.%26aulast%3DDufour%26aufirst%3DN.%26aulast%3DAuregan%26aufirst%3DG.%26aulast%3DAlves%26aufirst%3DS.%26aulast%3DBonvento%26aufirst%3DG.%26aulast%3DBrouillet%26aufirst%3DE.%26aulast%3DLuthi-Carter%26aufirst%3DR.%26aulast%3DHantraye%26aufirst%3DP.%26aulast%3DDeglon%26aufirst%3DN.%26atitle%3DSustained%2520effects%2520of%2520nonallele-specific%2520Huntingtin%2520silencing%26jtitle%3DAnn.%2520Neurol.%26date%3D2009%26volume%3D65%26spage%3D276%26epage%3D285%26doi%3D10.1002%2Fana.21569" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grondin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaytor, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nelson, P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thakker, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heisel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weatherspoon, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blum, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burright, E. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaemmerer, W. F.</span></span> <span> </span><span class="NLM_article-title">Six-month partial suppression of Huntingtin is well tolerated in the adult rhesus striatum</span>. <i>Brain</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>135</i></span>,  <span class="NLM_fpage">1197</span>– <span class="NLM_lpage">1209</span>, <span class="refDoi"> DOI: 10.1093/brain/awr333</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1093%2Fbrain%2Fawr333" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=22252996" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A280%3ADC%252BC38vgs1ajsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2012&pages=1197-1209&author=R.+Grondinauthor=M.+D.+Kaytorauthor=Y.+Aiauthor=P.+T.+Nelsonauthor=D.+R.+Thakkerauthor=J.+Heiselauthor=M.+R.+Weatherspoonauthor=J.+L.+Blumauthor=E.+N.+Burrightauthor=Z.+Zhangauthor=W.+F.+Kaemmerer&title=Six-month+partial+suppression+of+Huntingtin+is+well+tolerated+in+the+adult+rhesus+striatum&doi=10.1093%2Fbrain%2Fawr333"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Six-month partial suppression of Huntingtin is well tolerated in the adult rhesus striatum</span></div><div class="casAuthors">Grondin Richard; Kaytor Michael D; Ai Yi; Nelson Peter T; Thakker Deepak R; Heisel Jennifer; Weatherspoon Marcy R; Blum Janelle L; Burright Eric N; Zhang Zhiming; Kaemmerer William F</div><div class="citationInfo"><span class="NLM_cas:title">Brain : a journal of neurology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">135</span>
        (<span class="NLM_cas:issue">Pt 4</span>),
    <span class="NLM_cas:pages">1197-209</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Huntington's disease is caused by expression of a mutant form of Huntingtin protein containing an expanded polyglutamine repeat.  One possible treatment for Huntington's disease may be to reduce expression of mutant Huntingtin in the brain via RNA interference.  Unless the therapeutic molecule is designed to be allele-specific, both wild-type and mutant protein will be suppressed by an RNA interference treatment.  A key question is whether suppression of wild-type as well as mutant Huntingtin in targeted brain regions can be tolerated and result in a net benefit to patients with Huntington's disease.  Whether Huntingtin performs essential functions in the adult brain is unclear.  Here, we tested the hypothesis that the adult primate brain can tolerate moderately reduced levels of wild-type Huntingtin protein for an extended period of time.  A serotype 2 adeno-associated viral vector encoding for a short hairpin RNA targeting rhesus huntingtin messenger RNA (active vector) was bilaterally injected into the striatum of four adult rhesus monkeys.  Four additional animals received a comparable vector encoding a scrambled control short hairpin RNA (control vector).  General health and motor behaviour were monitored for 6 months.  Upon termination, brain tissues were sampled and assessed blindly for (i) huntingtin messenger RNA knockdown; (ii) Huntingtin protein expression; and (iii) neuropathological changes.  Reduction in wild-type huntingtin messenger RNA levels averaging ∼30% was measured in the striatum of active vector recipients 6 months post-injection.  A widespread reduction in Huntingtin protein levels was also observed by immunohistochemistry in these animals, with an average protein reduction of ∼45% relative to controls measured by western blot analysis in the putamen of active vector recipients.  As with control vector recipients, no adverse effects were observed behaviourally, and no neurodegeneration was found on histological examination of active vector recipients.  Our results suggest that long-term partial suppression of wild-type Huntingtin may be safe, and thus if a comparable level of suppression of mutant Huntingtin is beneficial, then partial suppression of both wild-type and mutant Huntingtin may result in a net benefit in patients with heterozygous Huntington's disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQOr8E8G496IYvKgEt20y1XfW6udTcc2ebdluyUqRmAZbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC38vgs1ajsw%253D%253D&md5=6c299a293da20ed60e15316465a904b6</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1093%2Fbrain%2Fawr333&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbrain%252Fawr333%26sid%3Dliteratum%253Aachs%26aulast%3DGrondin%26aufirst%3DR.%26aulast%3DKaytor%26aufirst%3DM.%2BD.%26aulast%3DAi%26aufirst%3DY.%26aulast%3DNelson%26aufirst%3DP.%2BT.%26aulast%3DThakker%26aufirst%3DD.%2BR.%26aulast%3DHeisel%26aufirst%3DJ.%26aulast%3DWeatherspoon%26aufirst%3DM.%2BR.%26aulast%3DBlum%26aufirst%3DJ.%2BL.%26aulast%3DBurright%26aufirst%3DE.%2BN.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DKaemmerer%26aufirst%3DW.%2BF.%26atitle%3DSix-month%2520partial%2520suppression%2520of%2520Huntingtin%2520is%2520well%2520tolerated%2520in%2520the%2520adult%2520rhesus%2520striatum%26jtitle%3DBrain%26date%3D2012%26volume%3D135%26spage%3D1197%26epage%3D1209%26doi%3D10.1093%2Fbrain%2Fawr333" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boudreau, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McBride, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martins, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xing, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davidson, B. L.</span></span> <span> </span><span class="NLM_article-title">Nonallele-specific silencing of mutant and wild-type huntingtin demonstrates therapeutic efficacy in Huntington’s disease mice</span>. <i>Mol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">1053</span>– <span class="NLM_lpage">1063</span>, <span class="refDoi"> DOI: 10.1038/mt.2009.17</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1038%2Fmt.2009.17" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=19240687" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A528%3ADC%252BD1MXit1agtb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2009&pages=1053-1063&author=R.+L.+Boudreauauthor=J.+L.+McBrideauthor=I.+Martinsauthor=S.+Shenauthor=Y.+Xingauthor=B.+J.+Carterauthor=B.+L.+Davidson&title=Nonallele-specific+silencing+of+mutant+and+wild-type+huntingtin+demonstrates+therapeutic+efficacy+in+Huntington%E2%80%99s+disease+mice&doi=10.1038%2Fmt.2009.17"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Nonallele-specific Silencing of Mutant and Wild-type Huntingtin Demonstrates Therapeutic Efficacy in Huntington's Disease Mice</span></div><div class="casAuthors">Boudreau, Ryan L.; McBride, Jodi L.; Martins, Ines; Shen, Shihao; Xing, Yi; Carter, Barrie J.; Davidson, Beverly L.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Therapy</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1053-1063</span>CODEN:
                <span class="NLM_cas:coden">MTOHCK</span>;
        ISSN:<span class="NLM_cas:issn">1525-0016</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Huntington's disease (HD) is a fatal neurodegenerative disease caused by mutant huntingtin (htt) protein, and there are currently no effective treatments.  Recently, we and others demonstrated that silencing mutant htt via RNA interference (RNAi) provides therapeutic benefit in HD mice.  We have since found that silencing wild-type htt in adult mouse striatum is tolerated for at least 4 mo.  However, given the role of htt in various cellular processes, it remains unknown whether nonallele-specific silencing of both wild-type and mutant htt is a viable therapeutic strategy for HD.  Here, we tested whether cosilencing wild-type and mutant htt provides therapeutic benefit and is tolerable in HD mice.  After treatment, HD mice showed significant redns. in wild-type and mutant htt, and demonstrated improved motor coordination and survival.  We performed transcriptional profiling to evaluate the effects of reducing wild-type htt in adult mouse striatum.  We identified gene expression changes that are concordant with previously described roles for htt in various cellular processes.  Also, several abnormally expressed transcripts assocd. with early-stage HD were differentially expressed in our studies, but intriguingly, those involved in neuronal function changed in opposing directions.  Together, these encouraging and surprising findings support further testing of nonallele-specific RNAi therapeutics for HD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmOjGm_mcSILVg90H21EOLACvtfcHk0lh8Zuquwk17Ew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXit1agtb4%253D&md5=0d777457472d0ccded0fd75658c4dc7e</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1038%2Fmt.2009.17&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fmt.2009.17%26sid%3Dliteratum%253Aachs%26aulast%3DBoudreau%26aufirst%3DR.%2BL.%26aulast%3DMcBride%26aufirst%3DJ.%2BL.%26aulast%3DMartins%26aufirst%3DI.%26aulast%3DShen%26aufirst%3DS.%26aulast%3DXing%26aufirst%3DY.%26aulast%3DCarter%26aufirst%3DB.%2BJ.%26aulast%3DDavidson%26aufirst%3DB.%2BL.%26atitle%3DNonallele-specific%2520silencing%2520of%2520mutant%2520and%2520wild-type%2520huntingtin%2520demonstrates%2520therapeutic%2520efficacy%2520in%2520Huntington%25E2%2580%2599s%2520disease%2520mice%26jtitle%3DMol.%2520Ther.%26date%3D2009%26volume%3D17%26spage%3D1053%26epage%3D1063%26doi%3D10.1038%2Fmt.2009.17" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Subramaniam, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sixt, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrow, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snyder, S. H.</span></span> <span> </span><span class="NLM_article-title">Rhes, a striatal specific protein, mediates mutant-huntingtin cytotoxicity</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>324</i></span>,  <span class="NLM_fpage">1327</span>– <span class="NLM_lpage">1330</span>, <span class="refDoi"> DOI: 10.1126/science.1172871</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1126%2Fscience.1172871" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=19498170" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A528%3ADC%252BD1MXms12gur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=324&publication_year=2009&pages=1327-1330&author=S.+Subramaniamauthor=K.+M.+Sixtauthor=R.+Barrowauthor=S.+H.+Snyder&title=Rhes%2C+a+striatal+specific+protein%2C+mediates+mutant-huntingtin+cytotoxicity&doi=10.1126%2Fscience.1172871"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Rhes, a striatal specific protein, mediates mutant-huntingtin Cytotoxicity</span></div><div class="casAuthors">Subramaniam, Srinivasa; Sixt, Katherine M.; Barrow, Roxanne; Snyder, Solomon H.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">324</span>
        (<span class="NLM_cas:issue">5932</span>),
    <span class="NLM_cas:pages">1327-1330</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Huntington's disease (HD) is caused by a polyglutamine repeat in the protein huntingtin (Htt) with mutant Htt (mHtt) expressed throughout the body and similarly in all brain regions.  Yet, HD neuropathol. is largely restricted to the corpus striatum.  We report that the small guanine nucleotide-binding protein Rhes, which is localized very selectively to the striatum, binds physiol. to mHtt.  Using cultured cells, we found Rhes induces sumoylation of mHtt, which leads to cytotoxicity.  Thus, Rhes-mHtt interactions can account for the localized neuropathol. of HD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlu-umOLmUV7Vg90H21EOLACvtfcHk0lgfytXo8cfIzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXms12gur0%253D&md5=521917309cdb18ef239d825da1aa8ea2</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1126%2Fscience.1172871&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1172871%26sid%3Dliteratum%253Aachs%26aulast%3DSubramaniam%26aufirst%3DS.%26aulast%3DSixt%26aufirst%3DK.%2BM.%26aulast%3DBarrow%26aufirst%3DR.%26aulast%3DSnyder%26aufirst%3DS.%2BH.%26atitle%3DRhes%252C%2520a%2520striatal%2520specific%2520protein%252C%2520mediates%2520mutant-huntingtin%2520cytotoxicity%26jtitle%3DScience%26date%3D2009%26volume%3D324%26spage%3D1327%26epage%3D1330%26doi%3D10.1126%2Fscience.1172871" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, G.</span></span> <span> </span><span class="NLM_article-title">alpha-secretase ADAM10 physically interacts with beta-secretase BACE1 in neurons and regulates CHL1 proteolysis</span>. <i>J. Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">411</span>– <span class="NLM_lpage">422</span>, <span class="refDoi"> DOI: 10.1093/jmcb/mjy001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=10.1093%2Fjmcb%2Fmjy001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=29325091" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtlCnt7jO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2018&pages=411-422&author=X.+Wangauthor=C.+Wangauthor=G.+Pei&title=alpha-secretase+ADAM10+physically+interacts+with+beta-secretase+BACE1+in+neurons+and+regulates+CHL1+proteolysis&doi=10.1093%2Fjmcb%2Fmjy001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">α-secretase ADAM10 physically interacts with β-secretase BACE1 in neurons and regulates CHL1 proteolysis</span></div><div class="casAuthors">Wang, Xin; Wang, Congcong; Pei, Gang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Cell Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">411-422</span>CODEN:
                <span class="NLM_cas:coden">JMCBCU</span>;
        ISSN:<span class="NLM_cas:issn">1759-4685</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">α-secretase and β-secretase are known to compete for amyloid precursor protein (APP) processing and thus play a vital role in Alzheimer's disease pathogenesis.  A disintegrin and metalloproteinase 10 (ADAM10) and β-site APP cleaving enzyme 1 (BACE1) mediate the major activities of α-secretase and β-secretase in brain and share various common substrates.  However, whether they function sep. or together is poorly understood.  Here, we show that ADAM10 and BACE1 co-localize in the neurites of mouse primary neurons.  Co-immunopptn. and fluorescence resonance energy transfer anal. revealed that ADAM10 and BACE1 interact with each other under both endogenous and exogenous conditions.  In addn., we found that ADAM10 enhances the proteolysis of neural cell adhesion mol. close homolog of L1 (CHL1) by BACE1.  Further studies found that ADAM10-BACE1 interaction interfering peptide LT52 attenuates the regulation of ADAM10 on BACE1-mediated cleavage of CHL1.  Our data indicate that ADAM10-BACE1 interaction regulates the proteolysis of some specific substrates and may play a potential role in brain function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFdB-6zlz2hrVg90H21EOLACvtfcHk0lgfytXo8cfIzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtlCnt7jO&md5=0bda141754662e1ea9263a0e64253a4a</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1093%2Fjmcb%2Fmjy001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjmcb%252Fmjy001%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DPei%26aufirst%3DG.%26atitle%3Dalpha-secretase%2520ADAM10%2520physically%2520interacts%2520with%2520beta-secretase%2520BACE1%2520in%2520neurons%2520and%2520regulates%2520CHL1%2520proteolysis%26jtitle%3DJ.%2520Mol.%2520Cell.%2520Biol.%26date%3D2018%26volume%3D10%26spage%3D411%26epage%3D422%26doi%3D10.1093%2Fjmcb%2Fmjy001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6LI0" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6LI0','PDB','6LI0'); return false;">PDB: 6LI0</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i71"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01133">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_11663"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c01133?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01133</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Supporting figures, tables, and materials and methods; spectroscopic data of compounds (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01133/suppl_file/jm0c01133_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01133/suppl_file/jm0c01133_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01133/suppl_file/jm0c01133_si_001.pdf">jm0c01133_si_001.pdf (5.82 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01133/suppl_file/jm0c01133_si_002.csv">jm0c01133_si_002.csv (2.44 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c01133&amp;pbContext=%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c01133%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2021.64.issue-2%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c01133" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                7MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679a067119b43cdc","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
