
<html lang="en"     class="pb-page"  data-request-id="eb9ab0c2-eb86-4d85-a06b-440f50562560"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;issue:issue:10.1021/jmcmar.2019.62.issue-16;pageGroup:string:Publication Pages;subPage:string:Full Text;article:article:10.1021/acs.jmedchem.9b00625"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Optimization and Mechanistic Characterization of Pyridopyrimidine Inhibitors of Bacterial Biotin Carboxylase" /></meta><meta name="dc.Creator" content="Logan D.  Andrews" /></meta><meta name="dc.Creator" content="Timothy R.  Kane" /></meta><meta name="dc.Creator" content="Paola  Dozzo" /></meta><meta name="dc.Creator" content="Cat M.  Haglund" /></meta><meta name="dc.Creator" content="Darin J.  Hilderbrandt" /></meta><meta name="dc.Creator" content="Martin S.  Linsell" /></meta><meta name="dc.Creator" content="Timothy  Machajewski" /></meta><meta name="dc.Creator" content="Glen  McEnroe" /></meta><meta name="dc.Creator" content="Alisa W.  Serio" /></meta><meta name="dc.Creator" content="Kenneth B.  Wlasichuk" /></meta><meta name="dc.Creator" content="David B.  Neau" /></meta><meta name="dc.Creator" content="Svetlana  Pakhomova" /></meta><meta name="dc.Creator" content="Grover L.  Waldrop" /></meta><meta name="dc.Creator" content="Marc  Sharp" /></meta><meta name="dc.Creator" content="Joe  Pogliano" /></meta><meta name="dc.Creator" content="Ryan T.  Cirz" /></meta><meta name="dc.Creator" content="Frederick  Cohen" /></meta><meta name="dc.Description" content="A major challenge for new antibiotic discovery is predicting the physicochemical properties that enable small molecules to permeate Gram-negative bacterial membranes. We have applied physicochemica..." /></meta><meta name="Description" content="A major challenge for new antibiotic discovery is predicting the physicochemical properties that enable small molecules to permeate Gram-negative bacterial membranes. We have applied physicochemica..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="July 15, 2019" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b00625" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2019 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b00625" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b00625" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b00625" /></link>
        
    
    

<title>Optimization and Mechanistic Characterization of Pyridopyrimidine Inhibitors of Bacterial Biotin Carboxylase | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b00625" /></meta><meta property="og:title" content="Optimization and Mechanistic Characterization of Pyridopyrimidine Inhibitors of Bacterial Biotin Carboxylase" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00625/20190815/images/large/jm-2019-006255_0001.jpeg" /></meta><meta property="og:description" content="A major challenge for new antibiotic discovery is predicting the physicochemical properties that enable small molecules to permeate Gram-negative bacterial membranes. We have applied physicochemical lessons from previous work to redesign and improve the antibacterial potency of pyridopyrimidine inhibitors of biotin carboxylase (BC) by up to 64-fold and 16-fold against Escherichia coli and Pseudomonas aeruginosa, respectively. Antibacterial and enzyme potency assessments in the presence of an outer membrane-permeabilizing agent or in efflux-compromised strains indicate that penetration and efflux properties of many redesigned BC inhibitors could be improved to various extents. Spontaneous resistance to the improved pyridopyrimidine inhibitors in P. aeruginosa occurs at very low frequencies between 10–8 and 10–9. However, resistant isolates had alarmingly high minimum inhibitory concentration shifts (16- to &gt;128-fold) compared to the parent strain. Whole-genome sequencing of resistant isolates revealed that either BC target mutations or efflux pump overexpression can lead to the development of high-level resistance." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b00625"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b00625">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b00625&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b00625&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b00625&amp;href=/doi/10.1021/acs.jmedchem.9b00625" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2019</span><span class="cit-fg-volume">, 62</span><span class="cit-fg-issue">, 16</span><span class="cit-fg-pageRange">, 7489-7505</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/62/16" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.9b00510" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.9b00673" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Optimization and Mechanistic Characterization of Pyridopyrimidine Inhibitors of Bacterial Biotin Carboxylase</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Logan D. Andrews</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Logan D. Andrews</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Former employee of Achaogen Inc., 1 Tower Place, Suite 400, South San Francisco, California 94080, United States</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#610d0e06051304161221060c00080d4f020e0c"><span class="__cf_email__" data-cfemail="7f1310181b0d1a080c3f18121e1613511c1012">[email protected]</span></a> (L.D.A.).</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Logan+D.++Andrews">Logan D. Andrews</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-5092-4530" title="Orcid link">http://orcid.org/0000-0002-5092-4530</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Timothy R. Kane</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Timothy R. Kane</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Former employee of Achaogen Inc., 1 Tower Place, Suite 400, South San Francisco, California 94080, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Timothy+R.++Kane">Timothy R. Kane</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Paola Dozzo</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Paola Dozzo</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Former employee of Achaogen Inc., 1 Tower Place, Suite 400, South San Francisco, California 94080, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Paola++Dozzo">Paola Dozzo</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Cat M. Haglund</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Cat M. Haglund</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Former employee of Achaogen Inc., 1 Tower Place, Suite 400, South San Francisco, California 94080, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Cat+M.++Haglund">Cat M. Haglund</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Darin J. Hilderbrandt</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Darin J. Hilderbrandt</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Former employee of Achaogen Inc., 1 Tower Place, Suite 400, South San Francisco, California 94080, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Darin+J.++Hilderbrandt">Darin J. Hilderbrandt</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Martin S. Linsell</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Martin S. Linsell</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Former employee of Achaogen Inc., 1 Tower Place, Suite 400, South San Francisco, California 94080, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Martin+S.++Linsell">Martin S. Linsell</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Timothy Machajewski</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Timothy Machajewski</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Former employee of Achaogen Inc., 1 Tower Place, Suite 400, South San Francisco, California 94080, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Timothy++Machajewski">Timothy Machajewski</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Glen McEnroe</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Glen McEnroe</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Former employee of Achaogen Inc., 1 Tower Place, Suite 400, South San Francisco, California 94080, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Glen++McEnroe">Glen McEnroe</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Alisa W. Serio</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Alisa W. Serio</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Former employee of Achaogen Inc., 1 Tower Place, Suite 400, South San Francisco, California 94080, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Alisa+W.++Serio">Alisa W. Serio</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Kenneth B. Wlasichuk</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kenneth B. Wlasichuk</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Former employee of Achaogen Inc., 1 Tower Place, Suite 400, South San Francisco, California 94080, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kenneth+B.++Wlasichuk">Kenneth B. Wlasichuk</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">David B. Neau</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">David B. Neau</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Northeastern Collaborative Access Team, Argonne National Laboratory, Argonne, Illinois 60439, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=David+B.++Neau">David B. Neau</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Svetlana Pakhomova</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Svetlana Pakhomova</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biological Sciences, Louisiana State University, Baton Rouge, Louisiana 70803, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Svetlana++Pakhomova">Svetlana Pakhomova</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Grover L. Waldrop</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Grover L. Waldrop</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biological Sciences, Louisiana State University, Baton Rouge, Louisiana 70803, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Grover+L.++Waldrop">Grover L. Waldrop</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Marc Sharp</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Marc Sharp</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Linnaeus Bioscience Inc., 3210 Merryfield Row, San Diego, California 92121, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Marc++Sharp">Marc Sharp</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Joe Pogliano</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Joe Pogliano</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Linnaeus Bioscience Inc., 3210 Merryfield Row, San Diego, California 92121, United States</div><div class="loa-info-affiliations-info">University of California, San Diego, 9500 Gilman Drive, La Jolla, San Diego, California 92093, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Joe++Pogliano">Joe Pogliano</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-8989-2655" title="Orcid link">http://orcid.org/0000-0001-8989-2655</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ryan T. Cirz</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ryan T. Cirz</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Former employee of Achaogen Inc., 1 Tower Place, Suite 400, South San Francisco, California 94080, United States</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#95e7ecf4fbf6fce7efd5f2f8f4fcf9bbf6faf8"><span class="__cf_email__" data-cfemail="5f2d263e313c362d251f38323e3633713c3032">[email protected]</span></a> (R.T.C.).</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ryan+T.++Cirz">Ryan T. Cirz</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Frederick Cohen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Frederick Cohen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Former employee of Achaogen Inc., 1 Tower Place, Suite 400, South San Francisco, California 94080, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Frederick++Cohen">Frederick Cohen</a></span></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b00625&amp;href=/doi/10.1021%2Facs.jmedchem.9b00625" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2019</span></span><span class="cit-volume">, 62</span><span class="cit-issue">, 16</span><span class="cit-pageRange">, 7489–7505</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">July 15, 2019</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>17 April 2019</li><li><span class="item_label"><b>Published</b> online</span>15 July 2019</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 22 August 2019</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b00625" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00625</a></div><div class="article_header-article-copyright"><strong>Copyright © 2019 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D7489%26pageCount%3D17%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DLogan%2BD.%2BAndrews%252C%2BTimothy%2BR.%2BKane%252C%2BPaola%2BDozzo%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D62%26issueNum%3D16%26contentID%3Dacs.jmedchem.9b00625%26title%3DOptimization%2Band%2BMechanistic%2BCharacterization%2Bof%2BPyridopyrimidine%2BInhibitors%2Bof%2BBacterial%2BBiotin%2BCarboxylase%26numPages%3D17%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D7505%26publicationDate%3DAugust%2B2019">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b00625"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1102</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">8</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b00625" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Optimization and Mechanistic Characterization of Pyridopyrimidine Inhibitors of Bacterial Biotin Carboxylase&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Logan&quot;,&quot;last_name&quot;:&quot;D. Andrews&quot;},{&quot;first_name&quot;:&quot;Timothy&quot;,&quot;last_name&quot;:&quot;R. Kane&quot;},{&quot;first_name&quot;:&quot;Paola&quot;,&quot;last_name&quot;:&quot;Dozzo&quot;},{&quot;first_name&quot;:&quot;Cat&quot;,&quot;last_name&quot;:&quot;M. Haglund&quot;},{&quot;first_name&quot;:&quot;Darin&quot;,&quot;last_name&quot;:&quot;J. Hilderbrandt&quot;},{&quot;first_name&quot;:&quot;Martin&quot;,&quot;last_name&quot;:&quot;S. Linsell&quot;},{&quot;first_name&quot;:&quot;Timothy&quot;,&quot;last_name&quot;:&quot;Machajewski&quot;},{&quot;first_name&quot;:&quot;Glen&quot;,&quot;last_name&quot;:&quot;McEnroe&quot;},{&quot;first_name&quot;:&quot;Alisa&quot;,&quot;last_name&quot;:&quot;W. Serio&quot;},{&quot;first_name&quot;:&quot;Kenneth&quot;,&quot;last_name&quot;:&quot;B. Wlasichuk&quot;},{&quot;first_name&quot;:&quot;David&quot;,&quot;last_name&quot;:&quot;B. Neau&quot;},{&quot;first_name&quot;:&quot;Svetlana&quot;,&quot;last_name&quot;:&quot;Pakhomova&quot;},{&quot;first_name&quot;:&quot;Grover&quot;,&quot;last_name&quot;:&quot;L. Waldrop&quot;},{&quot;first_name&quot;:&quot;Marc&quot;,&quot;last_name&quot;:&quot;Sharp&quot;},{&quot;first_name&quot;:&quot;Joe&quot;,&quot;last_name&quot;:&quot;Pogliano&quot;},{&quot;first_name&quot;:&quot;Ryan&quot;,&quot;last_name&quot;:&quot;T. Cirz&quot;},{&quot;first_name&quot;:&quot;Frederick&quot;,&quot;last_name&quot;:&quot;Cohen&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2019&quot;,&quot;month&quot;:&quot;07&quot;,&quot;day&quot;:&quot;15&quot;,&quot;issue&quot;:&quot;16&quot;,&quot;volume&quot;:&quot;62&quot;,&quot;pages&quot;:&quot;7489-7505&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b00625&quot;},&quot;abstract&quot;:&quot;A major challenge for new antibiotic discovery is predicting the physicochemical properties that enable small molecules to permeate Gram-negative bacterial membranes. We have applied physicochemical lessons from previous work to redesign and improve the antibacterial potency of pyridopyrimidine inhibitors of biotin carboxylase (BC) by up to 64-fold and 16-fold against Escherichia coli and Pseudomonas aeruginosa, respectively. Antibacterial and enzyme potency assessments in the presence of an outer membrane-permeabilizing agent or in efflux-compromised strains indicate that penetration and efflux properties of many redesigned BC inhibitors could be improved to various extents. Spontaneous resistance to the improved pyridopyrimidine inhibitors in P. aeruginosa occurs at very low frequencies between 10–8 and 10–9. However, resistant isolates had alarmingly high minimum inhibitory concentration shifts (16- to &gt;128-fold) compared to the parent strain. Whole-genome sequencing of resistant isolates revealed that eit&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00625&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00625" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00625&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00625" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00625&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00625" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b00625&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00625&amp;href=/doi/10.1021/acs.jmedchem.9b00625" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b00625" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b00625" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (7 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00625&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b00625%26sid%3Dliteratum%253Aachs%26pmid%3D31306011%26genre%3Darticle%26aulast%3DAndrews%26date%3D2019%26atitle%3DOptimization%2Band%2BMechanistic%2BCharacterization%2Bof%2BPyridopyrimidine%2BInhibitors%2Bof%2BBacterial%2BBiotin%2BCarboxylase%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D62%26issue%3D16%26spage%3D7489%26epage%3D7505%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=292610" title="Antibiotic resistance">Antibiotic resistance</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291439" title="Genetics">Genetics</a>,</li><li><a href="/action/doSearch?ConceptID=291691" title="Membranes">Membranes</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/62/16" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/jmcmar.2019.62.issue-16/20190822/jmcmar.2019.62.issue-16.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00625/20190815/images/medium/jm-2019-006255_0001.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00625/20190815/images/large/jm-2019-006255_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00625&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">A major challenge for new antibiotic discovery is predicting the physicochemical properties that enable small molecules to permeate Gram-negative bacterial membranes. We have applied physicochemical lessons from previous work to redesign and improve the antibacterial potency of pyridopyrimidine inhibitors of biotin carboxylase (BC) by up to 64-fold and 16-fold against <i>Escherichia coli</i> and <i>Pseudomonas aeruginosa</i>, respectively. Antibacterial and enzyme potency assessments in the presence of an outer membrane-permeabilizing agent or in efflux-compromised strains indicate that penetration and efflux properties of many redesigned BC inhibitors could be improved to various extents. Spontaneous resistance to the improved pyridopyrimidine inhibitors in <i>P. aeruginosa</i> occurs at very low frequencies between 10<sup>–8</sup> and 10<sup>–9</sup>. However, resistant isolates had alarmingly high minimum inhibitory concentration shifts (16- to >128-fold) compared to the parent strain. Whole-genome sequencing of resistant isolates revealed that either BC target mutations or efflux pump overexpression can lead to the development of high-level resistance.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_68215" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_68215" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The spread of multidrug-resistant (MDR) bacteria presents a threat to our health care system. An analysis from the Centers for Disease Control and Prevention indicates that approximately 23,000 people die annually in the U.S. from these pathogens,<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> and more recent reports suggest this estimate is conservative.<a onclick="showRef(event, 'ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref2 ref3">(2,3)</a> In the future, routine life-saving procedures such as the use of coronary stents, anticancer chemotherapy, and organ transplants could carry the risk of life-threatening bacterial infections. The situation is particularly alarming for MDR infections caused by the so-called Gram-negative “ESKAPE” pathogens: <i>Escherichia coli</i>, <i>Klebsiella pneumoniae</i>, <i>Acinetobacter baumannii</i>, and <i>Pseudomonas aeruginosa</i>. This point is illustrated by the recent discovery of a strain of <i>K. pneumoniae</i> from a U.S. ICU patient that is resistant to 26 antibiotics.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> To address this threat from MDR pathogens, a variety of new antibiotic discovery approaches are warranted including optimization of previously discovered compounds with existing, although sometimes low, levels of antibacterial potency.</div><div class="NLM_p">One such previously discovered compound that serves as an enticing starting point for further optimization is the well-characterized biotin carboxylase (BC) inhibitor <b>1</b> that was identified in a high-throughput screen against a membrane-compromised, efflux pump-deficient strain of <i>E. coli</i> (<i>tolC</i>, <i>imp</i>).<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> BC is a member of the fatty acid biosynthesis pathway (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00625/suppl_file/jm9b00625_si_001.pdf" class="ext-link">Figure S1</a>) and has attracted attention over the past decade as a promising target for the development of novel broad-spectrum antibiotics for Gram-negative bacterial infections.<a onclick="showRef(event, 'ref6 ref7 ref8 ref9 ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref6 ref7 ref8 ref9 ref10 ref11">(6−11)</a> The first committed step in this pathway is catalyzed by an essential cytoplasmic acetyl-CoA carboxylase multienzyme complex<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> (see also <a href="/doi/suppl/10.1021/acs.jmedchem.9b00625/suppl_file/jm9b00625_si_001.pdf" class="ext-link">Text S1</a>), and inhibitors of the BC active site of this complex have shown antibacterial activity against several Gram-negative species.<a onclick="showRef(event, 'ref5 ref13 ref14 ref15 ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref5 ref13 ref14 ref15 ref16 ref17">(5,13−17)</a> The BC active site is highly conserved across a number of bacterial pathogens, including <i>P. aeruginosa</i>, <i>E. coli</i>, and <i>Haemophilus influenzae</i>.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> While <b>1</b> lacks the potency to become a therapeutic agent, it is a promising starting point for medicinal chemistry efforts because of its selective on-target activity, low molecular weight, and available cocrystal structures with BC that enable structure-based molecular design.</div><div class="NLM_p">Inhibitors of BC must pass through both the outer and inner membranes of the Gram-negative bacteria to reach this cytoplasmic enzyme. The outer membrane bars the passage of many exogenous small molecules. However, porins in the membrane allow some hydrophilic molecules with a size and shape compatible with the diameter of the porin channels to diffuse into the periplasm of the cell.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> Once in the periplasm, molecules must then pass through the phospholipid inner membrane to reach cytoplasmic targets, a process favored by lower polarity. Because the molecular properties of the outer and inner membranes differ, the two barriers act orthogonally to greatly hinder the access of most potential inhibitors to their cytoplasmic target sites. Even when a molecule successfully penetrates both membranes, efflux pumps that actively remove foreign compounds from the cell further reduce the concentration of an inhibitor reaching a cytoplasmic target.<a onclick="showRef(event, 'ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref20 ref21">(20,21)</a> These hurdles are major challenges to the development of small-molecule inhibitors designed to kill bacteria by acting within the cytoplasm.</div><div class="NLM_p">To address the challenge of discovering inhibitors that can readily access cytoplasmic targets, retrospective chemoinformatic approaches have been used to evaluate the physicochemical properties of molecules that accumulate effectively in Gram-negative cells.<a onclick="showRef(event, 'ref22 ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref22 ref23 ref24">(22−24)</a> A pioneering analysis by O’Shea and Moser showed that, in general, antibiotics with Gram-negative activity are significantly more polar (as measured by relative polar surface area and cLog<i>D</i> values) and have more H-bond acceptor and donor atoms compared to a reference set of drugs from other therapeutic areas.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> For the subset of antibiotics that act against targets in the cytoplasm, the physicochemical differences compared to the reference set were quite diverse and less pronounced for some classes of antibiotics but still trended toward having higher overall polarity.<a onclick="showRef(event, 'ref24 ref25'); return false;" href="javascript:void(0);" class="ref ref24 ref25">(24,25)</a> However, a more recent assessment of Gram-negative cell compound accumulation that examined charge and polarity (cLog<i>D</i>) properties separately suggests that polarity alone is not predictive of cell accumulation.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a></div><div class="NLM_p">To illustrate the physicochemical properties of cytoplasm-acting inhibitors that are sufficient for antibacterial potency, we compared a set of exemplar Gram-negative antibiotics (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). These antibiotics have cLog<i>D</i> values ranging from −3.6 to 2.0, and all contain groups that ionize near physiological pH (7.4).</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Examples of Cytoplasm-Targeted Gram-Negative Agents with Physicochemical Properties of Interest</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00625/20190815/images/medium/jm-2019-006255_0002.gif" alt="" id="gr8" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Acidic groups are colored red; basic groups are colored blue.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">cLog<i>D</i> at pH 7.4. Calculated using ACD/Percepta GALAS algorithm, Advanced Chemistry Development, Inc.</p></div><div class="footnote" id="t1fn3"><sup><sup>c</sup></sup><p class="last">At physiological pH = 7.4.</p></div><div class="footnote" id="t1fn4"><sup><sup>d</sup></sup><p class="last">Ref <a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a>.</p></div><div class="footnote" id="t1fn5"><sup><sup>e</sup></sup><p class="last">Ref <a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a>.</p></div></div><div></div></div><div class="NLM_p">Comparing the property values in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> reveals that BC inhibitor <b>1</b> with a cLog<i>D</i> value of 3.3 is significantly less polar than the proven antibacterials. In addition and perhaps most importantly, <b>1</b> is the only compound in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> that does not contain an ionizable group. These comparisons suggest that the potency of <b>1</b> could be improved by increasing its polarity and installing ionizable groups with p<i>K</i><sub>a</sub> values in the physiological pH range. Further support for this approach was reported recently by Hergenrother’s group, which showed that the addition of a primary amine to a molecule with a p<i>K</i><sub>a</sub> in the physiological pH range can significantly enhance cellular accumulation.<a onclick="showRef(event, 'ref26 ref29'); return false;" href="javascript:void(0);" class="ref ref26 ref29">(26,29)</a></div><div class="NLM_p last">Herein, we report a structure-guided approach that was used to modulate the physicochemical properties of <b>1</b> to favor membrane penetration and efflux avoidance while simultaneously maintaining or improving BC target affinity, resulting in a significant overall improvement in antibacterial potency.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4">Chemistry</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_10108" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_10108" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Design Strategy</h3><div class="NLM_p">The previously determined structure of <b>1</b> bound to <i>E. coli</i> BC suggested two vectors allowing for physicochemical optimization while also permitting the retention, or even improvement, of BC target inhibition. The first vector points from the amine at the 7-position of the pyridopyrimidine ring toward two conserved glutamic acid residues that coordinate Mg<sup>2+</sup> when substrate ATP is bound (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A). The nearest oxygen atoms of these glutamic acid residue side chains are 4.9 and 6.1 Å away from the 7-position nitrogen atom of <b>1</b>. We hypothesized that this vector would provide an opportunity to place an ionizable amine group that would reduce the cLog<i>D</i> value of the compound while also engaging the nearby glutamic acid residues via H-bonding and/or electrostatic interactions. The second vector points off the dihalo-aromatic ring through a small slot in the active site surface toward the solvent (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B). We begin by focusing our attention on the first vector at the 7-position of the aromatic ring system and subsequently return to the second vector for further optimization. In the final phase of the campaign, compounds consisting of derivatizations at both vectors are presented.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00625/20190815/images/medium/jm-2019-006255_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00625/20190815/images/large/jm-2019-006255_0016.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Compound <b>1</b> (carbon, green; nitrogen, blue; bromine, dark red; all rendered as sticks) cocrystallized with <i>E. coli</i> BC at 2.1 Å as reported in Ref <a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2V58">2V58</a>). (A) View of compound <b>1</b> in the BC active site showing the 7-position vector that points toward two proximal glutamate residues as illustrated by the two arrows. The distance from the 7-position nitrogen atom and nearest oxygen atoms of Glu288 and Glu276 are 4.9 and 6.1 Å, respectively. (B) View of the solvent accessible surface representation, in transparent grey, of <i>E. coli</i> BC cocrystallized with <b>1</b>. The second vector points off the dihalo-aromatic ring of <b>1</b> through a small slot in the active site surface toward the solvent. Images prepared using PyMOL.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00625/20190815/images/large/jm-2019-006255_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00625&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Synthesis</h3><div class="NLM_p">Simple 2,7-diaminopyridopyrimidines such as <b>1</b> and <b>4a</b>–<b>4e</b> are readily synthesized in one step from aryl acetonitriles, as shown in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>.</div><figure id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00625/20190815/images/medium/jm-2019-006255_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00625/20190815/images/large/jm-2019-006255_0012.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of 6-Aryl Pyridopyrimidines<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00625/20190815/images/large/jm-2019-006255_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00625&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) Base, solvent; see the <a class="ref internalNav" href="#sec5" aria-label="Experimental Section">Experimental Section</a>.</p></p></figure><div class="NLM_p">To install an amine at the 7-position, we developed a novel synthetic route that allowed for differentiation of the 2- and 7-positions, as illustrated in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>. The route begins with a condensation between aryl acetonitrile <b>2a</b> and pyrimidine <b>5</b>, wherein the eventual amine at the 2-position is masked as a thiomethyl group. Following condensation, the 7-position is converted to pyridone <b>7</b> by nitration and hydrolysis. Chlorination with POCl<sub>3</sub> provided <b>8</b> with a handle for installation of various amines by simple S<sub>n</sub>Ar displacement. The sulfide was converted to an amine by oxidation to the sulfoxide and displacement with ammonia to provide analogs <b>12a</b> and <b>12b</b>. If required, final Boc deprotection was affected with TFA to afford inhibitors <b>13c</b>–<b>13h</b>. This route could also be modified to allow for late-stage diversification via the Suzuki reaction, as illustrated in <a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>, for the synthesis of compounds <b>14a</b>–<b>14e</b>.</div><figure id="sch2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00625/20190815/images/medium/jm-2019-006255_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00625/20190815/images/large/jm-2019-006255_0011.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compounds <b>12a</b>, <b>12b</b>, and <b>13c</b>–<b>13h</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00625/20190815/images/large/jm-2019-006255_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00625&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) NaH, DMF, 45%; (b) NaNO<sub>2</sub>, trifluoroacetic acid, 90%; (c) POCl<sub>3</sub>, DMF (cat), 95 °C, 98%; (d) <b>9a</b>–<b>9h</b>, MeCN, 1,4-dioxane, 80 °C; (e) mCPBA or Davis oxaziridine, DCM; (f) NH<sub>3</sub>, MeOH, THF, 65 °C; (g) trifluoroacetic acid, DCM.</p></p></figure><figure id="sch3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00625/20190815/images/medium/jm-2019-006255_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00625/20190815/images/large/jm-2019-006255_0010.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Compounds <b>14a</b>–<b>14d</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00625/20190815/images/large/jm-2019-006255_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00625&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) NaH, DMF; (b) NaNO<sub>2</sub>, trifluoroacetic acid; (c) POCl<sub>3</sub>, DMF (cat), 95 °C; (d) <b>9h</b>, MeCN, 1,4-dioxane, 80 °C; (e) mCPBA or Davis oxaziridine, DCM; (f) NH<sub>3</sub>, MeOH, THF, 65 °C; (g) trifluoroacetic acid, DCM; (h) R(BOH)<sub>2</sub>, Pd(dppf)Cl<sub>2</sub>, Na<sub>2</sub>CO<sub>3</sub>, dioxane, water, 80 °C.</p></p></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i10">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_80234" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_80234" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec3_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Structure–Activity Relationships for Derivatives off the Pyridopyrimidine 7-Position</h3><div class="NLM_p">Compound <b>4a</b> was chosen as the parent molecule for this first campaign because it is identical to <b>1</b> except for the substitution of the dibromo ring with a dichloro ring, which results in a slightly lower cLog<i>D</i>. In addition, the IC<sub>50</sub> value measured for <b>4a</b> is within 2-fold of the IC<sub>50</sub> value measured here for <b>1</b>, indicating that the ring substitution does not have a large impact on BC target inhibition. As expected based on the potency reported for <b>1</b>, compound <b>4a</b> has limited microbiological activity against wild-type (WT) ESKAPE pathogens <i>A. baumannii</i>, <i>E. coli</i>, and <i>K. pneumoniae</i>, and no activity against <i>P. aeruginosa</i> was detected (minimum inhibitory concentration (MIC) > 256 μg/mL) (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>).</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. MIC Values, IC<sub>50</sub> Values, and Physicochemical Properties of Interest for Compounds Elaborated at the 7-Position</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00625/20190815/images/medium/jm-2019-006255_0009.gif" alt="" id="gr9" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00625/20190815/images/medium/jm-2019-006255_0008.gif" alt="" id="gr10" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">The strains of each bacterial species were as follows: <i>A. baumannii</i> (wild type) ATCC 19606; <i>E. coli</i> (wild type) DM4100; <i>K. pneumoniae</i> (wild type) ATCC 43816; <i>P. aeruginosa</i> (wild type) PAO1.</p></div><div class="footnote" id="t2fn2"><sup><sup>b</sup></sup><p class="last">The <i>E. coli</i> Δ<i>tolC</i> strain contains a targeted knockout of the efflux pump component TolC.</p></div><div class="footnote" id="t2fn3"><sup><sup>c</sup></sup><p class="last">The <i>P. aeruginosa</i> Δ<i>mexAB</i>, etc. strain contains targeted knockouts of efflux pumps MexAB-OprM, MexCD-OprJ, and MexEF-OprN, and the efflux pump components MexXY are expected to be compromised by the absence of OprM.</p></div><div class="footnote" id="t2fn4"><sup><sup>d</sup></sup><p class="last">Calculated p<i>K</i><sub>a</sub> values are in parentheses.</p></div></div><div></div></div><div class="NLM_p">To test the ability of <b>4a</b> to penetrate the Gram-negative outer membrane, we conducted MIC assays in the presence and absence of the permeabilizing agent polymyxin B nonapeptide (PMBN). At the concentration used (8 μg/mL), PMBN does not have an antibacterial effect but is expected to render the outer membrane more permeable by interacting with and disrupting the negatively charged lipopolysaccharide (LPS) molecules that otherwise maintain the outer membrane order.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> When PMBN is included, the MIC value of compound <b>4a</b> against <i>E. coli</i> is reduced 32-fold from 32 to 1 μg/mL. This significant enhancement of potency in the presence of PMBN suggests that when the outer membrane is compromised, a greater proportion of <b>4a</b> is able to reach the BC active site in the cytoplasm. By contrast, <b>ACHN-975</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>), which also targets a cytoplasmic enzyme but has physicochemical properties that may be more conducive to membrane penetration, is only 2- to 4-fold more potent in the presence of PMBN, suggesting that a greater proportion of <b>ACHN-975</b> readily crosses the outer membrane.</div><div class="NLM_p">To test the contribution of efflux to the limited potency of <b>4a</b>, an <i>E. coli</i> (Δ<i>tolC</i>) mutant was tested in the MIC assay. With this key efflux component deleted, the potency of <b>4a</b> is strengthened by 64-fold (32 to 0.5 μg/mL), suggesting that WT <i>E. coli</i> uses a TolC-containing efflux pump to dramatically reduce the intracellular concentration of <b>4a</b>. Similar results were observed when comparing the MIC values of <b>4a</b> against WT and efflux-compromised strains of <i>P. aeruginosa</i> (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>).</div><div class="NLM_p">When PMBN was included with compound <b>4a</b>, the MIC for the <i>E. coli</i> Δ<i>tolC</i> strain is 0.25 μg/mL, the lowest value observed for all of the strains and conditions tested. Because the efflux-compromised and WT strains are expected to have the same levels of LPS in their outer membranes, we initially reasoned that PMBN should reduce the MIC by the same proportion for both strains. Instead, the presence of PMBN induced only a 2-fold reduction in MIC against the efflux-compromised strain (0.5 to 0.25 μg/mL) compared to a 32-fold reduction against the WT strain (32 to 1 μg/mL). This disconnect suggests that when the outer membrane and efflux mechanisms are both compromised, other properties of the bacterial cell and inhibitor, such as the BC enzyme affinity and/or the ability of the inner membrane to exclude the inhibitor, become limiting to the observed MIC potency.</div><div class="NLM_p">Structural elaboration of <b>4a</b> at the 7-position of the pyridopyrimidine ring yielded compounds <b>12a</b>, <b>12b</b>, and <b>13c</b>–<b>13h</b> shown in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>. To test our structural expectation that groups at this position could be readily accommodated by the BC enzyme, we measured the IC<sub>50</sub> values against purified <i>P. aeruginosa</i> BC. The results in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a> show that the IC<sub>50</sub> values measured are all similar or better compared to <b>4a</b>, demonstrating that the 7-position modification does not disrupt BC inhibition and, in one case (<b>13h</b>), improves inhibition by nearly 10-fold.</div><div class="NLM_p">In addition to having similar BC affinity compared to <b>4a</b>, the compounds <b>12a</b> and <b>12b</b>, also have similar cLog<i>D</i> and p<i>K</i><sub>a</sub> values. Thus, it was expected that these compounds would perform similarly to <b>4a</b> in the MIC panel. The MIC data for <b>12a</b> and <b>12b</b> in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a> reflect these expectations, with similarly low potency against WT strains and similarly strong MIC shifts observed in the presence of PMBN and against the efflux mutants. These observations are consistent with the premise that compounds with high cLog<i>D</i> values and no ionizable groups at physiological pH values have difficulty entering cells and avoiding efflux.</div><div class="NLM_p">To test the physicochemical hypothesis further, compound <b>12b</b> was modified with the addition of a nitrogen atom with a p<i>K</i><sub>a</sub> value near physiological pH, resulting in amine-containing compound <b>13c</b>. In addition to becoming ionizable, <b>13c</b> also has a modest decrease in cLog<i>D</i>. Importantly, the BC affinity of <b>13c</b> remains relatively constant compared to the parent compound so that any changes in potency can more likely be attributed to changes in the two important physicochemical properties. <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a> shows that <b>13c</b> does not have a large increase in antibacterial potency against WT strains. However, the presence of PMBN improves the potency of <b>13c</b> by only 2-fold, suggesting that it crosses the outer membrane more easily than compounds <b>4a</b>, <b>12a</b>, and <b>12b</b>, which have MIC shifts of 16- to 32-fold in the presence of PMBN. Likewise, the 16-fold shift observed with <b>13c</b> against the <i>E. coli</i> efflux-compromised strain compared to the WT strain is much lower than the 64- to 128-fold shifts seen with compounds <b>4a</b>, <b>12a</b>, and <b>12b</b>. The lower fold change suggests that compound <b>13c</b> also avoids efflux more readily than the neutral compounds with higher cLog<i>D</i> values. Despite these apparent cellular penetration improvements and maintenance of BC target affinity, the lack of corresponding improvement against WT strains suggests that <b>13c</b> has lost potency due to some other difference(s) compared to <b>4a</b>, <b>12a</b>, and <b>12b</b>. While currently untested, one hypothesis is that <b>13c</b> has a reduced ability to cross the Gram-negative inner membrane relative to <b>4a</b>, <b>12a</b>, and <b>12b</b>. The direct measurement of the accumulation of compound inside the cell could help test this hypothesis.</div><div class="NLM_p">Further derivatives of <b>13c</b> were designed to increase the p<i>K</i><sub>a</sub> and lower the cLog<i>D</i>, again with the expectation that such changes would improve outer membrane penetration and efflux avoidance. Contrary to this expectation, these derivatives (<b>13d</b>–<b>13g</b> in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>) all have higher MIC shifts in the presence of PMBN compared to <b>13c</b> (8-fold vs 2-fold, respectively). In addition, all had equal or larger MIC shifts against the efflux-compromised strain compared to <b>13c</b>. These observations suggest that an unknown counteracting property of these derivatives reduces their ability to penetrate the outer membrane and avoid efflux relative to <b>13c</b>. One possible explanation is that the basicity of the amines of <b>13d</b>–<b>13g</b> is too high since they all have p<i>K</i><sub>a</sub> values greater than <b>13c</b>. If true, this would suggest that there is a narrow p<i>K</i><sub>a</sub> range for this scaffold to achieve optimal cellular penetration. When the majority of the compound population is positively charged at physiological pH, as is the case when the p<i>K</i><sub>a</sub> is >8, the membrane penetration and efflux avoidance properties of the compounds may get worse despite relatively low cLog<i>D</i> values. While this model also remains untested, we focused our further medicinal chemistry efforts in this series on making compounds with amine p<i>K</i><sub>a</sub> values less than 8.5.</div><div class="NLM_p last">After further rounds of elaboration at the 7-position, the most potent compound against the WT strains was identified to be <b>13h</b>, which contained an amino azabicyclohexane group with a p<i>K</i><sub>a</sub> value of 8. This compound had similar physicochemical properties as compounds <b>13d</b>–<b>13g</b> and yielded similar MIC shifts in the presence of PMBN and against the efflux-compromised strain. However, <b>13h</b> achieved the lowest MIC values against the WT strains of interest (except against <i>P. aeruginosa</i>, which remained above the measurable range), presumably due to its approximately 10-fold improvement in BC target affinity compared to the parent <b>4a</b>. The resiliency of the WT <i>P. aeruginosa</i> strain was not unexpected as this species is known to have a less permeable membrane and a more extensive efflux system compared to the other ESKAPE pathogens tested here.<a onclick="showRef(event, 'ref31 ref32 ref33 ref34 ref35'); return false;" href="javascript:void(0);" class="ref ref31 ref32 ref33 ref34 ref35">(31−35)</a></div></div><div id="sec3_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Structure–Activity Relationships of Slot Position Derivatives</h3><div class="NLM_p">In an effort to further improve the potency of parent compound <b>4a</b>, we turned our attention to the second vector of interest pointing off the dihalo-aromatic ring (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B). Structural elaboration at this position occurred in two phases. The first phase involved the addition of various aryl groups off of the dibromo aromatic ring that were designed to fit within the small surface slot of the BC active site (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B). These modifications did not contain groups capable of ionizing near physiological pH nor did they result in large reductions to the cLog<i>D</i>, so it was not expected that these compounds would have enhanced membrane penetration or efflux avoidance.</div><div class="NLM_p">These new compounds were tested against our MIC panel to assess cellular entry properties and overall potency (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). In accordance with expectations, this set of compounds had large MIC shifts when PMBN was present, similar to parent compounds <b>1</b> and <b>4a</b>. These compounds were also much more potent against the <i>E. coli</i> efflux-compromised strain relative to the WT strain, with MIC shifts ranging from 64- to >512-fold. These observations support the prediction that this set of compounds would retain relatively poor outer membrane permeability and high efflux potential because of unoptimized physicochemical properties. However, we note that compound <b>4c</b> has a cLog<i>D</i> value that is a full unit lower than parent <b>1</b>, yet the MIC fold shift for this compound suggests no improvement in either outer membrane penetration or efflux avoidance. While this reduction in cLog<i>D</i> value may not be large enough to yield predictable penetration and efflux avoidance improvements, it may also be that introduction of polarity at this particular position on the molecule is ineffective for reasons not captured by the simple expectation based solely on bulk physicochemical properties. Given that <b>4c</b> has the same cLog<i>D</i> as <b>13c</b>, which did not show large MIC shifts in the presence of PMBN or when efflux was compromised, it may be that the presence of an ionizable amine, as on <b>13c</b> and not <b>4c</b>, is the more important property in determining cellular accumulation compared to the cLog<i>D</i> parameter.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. MIC Values, IC<sub>50</sub> Values, and Physicochemical Properties of Interest for Compounds Elaborated at the Slot Position</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00625/20190815/images/medium/jm-2019-006255_0007.gif" alt="" id="gr11" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00625/20190815/images/medium/jm-2019-006255_0006.gif" alt="" id="gr12" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">The strains of each bacterial species were as follows: <i>A. baumannii</i> (wild type) ATCC 19606; <i>E. coli</i> (wild type) DM4100; <i>K. pneumoniae</i> (wild type) ATCC 43816; <i>P. aeruginosa</i> (wild type) PAO1.</p></div><div class="footnote" id="t3fn2"><sup><sup>b</sup></sup><p class="last">The <i>E. coli</i> Δ<i>tolC</i> strain contains a targeted knockout of the efflux pump component TolC.</p></div><div class="footnote" id="t3fn3"><sup><sup>c</sup></sup><p class="last">The <i>P. aeruginosa</i> Δ<i>mexAB</i>, etc. strain contains targeted knockouts of efflux pumps MexAB-OprM, MexCD-OprJ, and MexEF-OprN, and the efflux pump components MexXY are expected to be compromised by the absence of OprM.</p></div><div class="footnote" id="t3fn4"><sup><sup>d</sup></sup><p class="last">Calculated p<i>K</i><sub>a</sub> values are in parentheses.</p></div></div><div></div></div><div class="NLM_p">To assess the effect of the structural changes off the dibromo aromatic ring on target BC inhibition, IC<sub>50</sub> values were measured using purified <i>P. aeruginosa</i> BC. As these derivatives were designed to sterically fit into a small slot extending from the BC active site but not to make new specific interactions, it was expected that the IC<sub>50</sub> values of the elaborated compounds would be quite similar to parent compound <b>1</b>. As expected, the IC<sub>50</sub> values reported in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a> are approximately equal to or lower by up to 6-fold of <b>1</b>, indicating that the introduction of additional ring systems at this position does not cause major disruptions to target inhibition. Wary of the potential for the additional aromatic ring to reduce solubility, it was discovered that certain aliphatic groups, such as isobutyl <b>4f</b>, could also fit the slot position and afford activity against the enzyme.</div><div class="NLM_p">For the final phase of the structural elaboration off the dihalo-aromatic ring, we chose a starting scaffold with the best amine-containing group identified during the derivatization campaign at the 7-position of the pyridopyrimidine ring, <b>13h</b>. The first two related compounds in the series, <b>14a</b> and <b>14b</b> (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>), maintained the physicochemical properties of the parent compound <b>13h</b> and showed an improvement in BC inhibition, with IC<sub>50</sub> values recorded at the lower limit of detection of ≤15 nM, an improvement of ≥2-fold relative to the parent <b>13h</b>. This BC target inhibition improvement could be responsible for the small MIC improvements observed against the WT strains (with the exception of <i>A. baumannii</i>). No substantially different MIC shifts were observed in the presence of PMBN or when efflux was compromised, as expected given that <b>14a</b> and <b>14b</b> have p<i>K</i><sub>a</sub> and cLog<i>D</i> values similar to the parent comparator <b>13h</b> and all compounds contain an ionizable amine.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. MIC Values, IC<sub>50</sub> Values, and Physicochemical Properties of Interest for Compounds Elaborated in the Final Phase of the Medicinal Chemistry Campaign</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00625/20190815/images/medium/jm-2019-006255_0005.gif" alt="" id="gr13" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00625/20190815/images/medium/jm-2019-006255_0004.gif" alt="" id="gr14" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">The strains of each bacterial species were as follows: <i>A. baumannii</i> (wild type) ATCC 19606; <i>E. coli</i> (wild type) DM4100; <i>K. pneumoniae</i> (wild type) ATCC 43816; <i>P. aeruginosa</i> (wild type) PAO1; <i>Staphylococcus aureus</i> (wild type) ATCC 29213.</p></div><div class="footnote" id="t4fn2"><sup><sup>b</sup></sup><p class="last">The <i>E. coli</i> Δ<i>tolC</i> strain contains a targeted knockout of the efflux pump component TolC.</p></div><div class="footnote" id="t4fn3"><sup><sup>c</sup></sup><p class="last">The <i>P. aeruginosa</i> Δ<i>mexAB</i>, etc. strain contains targeted knockouts of efflux pumps MexAB-OprM, MexCD-OprJ, and MexEF-OprN, and the efflux pump components MexXY are expected to be compromised by the absence of OprM.</p></div><div class="footnote" id="t4fn4"><sup><sup>d</sup></sup><p class="last">Calculated p<i>K</i><sub>a</sub> values are in parentheses.</p></div></div><div></div></div><div class="NLM_p last">The next compounds in the series, <b>14c</b>–<b>14e</b>, all maintained BC IC<sub>50</sub> values below 15 nM and achieved the most potent microbiological activity observed in the series. As these compounds also maintained the general physicochemical properties of the starting scaffold, the MIC shifts observed in the presence of PMBN or against the efflux-compromised strain were similar to the shifts observed with the parent compound.</div></div><div id="sec3_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Structural Biology</h3><div class="NLM_p">To assess the structural consequences of the 7-position modifications, the X-ray cocrystal structure of <b>13f</b> with the WT BC enzyme from <i>E. coli</i> was solved at 2.06 Å (see <a href="/doi/suppl/10.1021/acs.jmedchem.9b00625/suppl_file/jm9b00625_si_001.pdf" class="ext-link">Table S1</a> for crystallographic statistics). <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A shows that the pyridopyrimidine ring scaffold of <b>13f</b> occupies the same position as the previously reported structure of <b>1</b>.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> Thus, neither the dibromo- to dichloro-substitution nor the modification off the 7-position altered the scaffold binding position in the BC active site pocket.</div><figure id="fig2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00625/20190815/images/medium/jm-2019-006255_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00625/20190815/images/large/jm-2019-006255_0015.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. BC cocrystal structures of new pyridopyrimidine inhibitors overlaid with previously identified parent compound <b>1</b> cocrystallized with <i>E. coli</i> BC (shown in green; PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2V58">2V58</a>). (A) Compound <b>13f</b> (carbon, magenta; nitrogen, blue; chlorine, light green; all rendered as sticks) cocrystallized with <i>E. coli</i> BC at 2.06 Å (magenta ribbon cartoon; PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6OI9">6OI9</a>). The core pyridopyrimidine scaffold of <b>13f</b> is found in the identical position as that of parent compound <b>1</b>. The 7-position moiety extending off <b>13f</b> points toward the conserved glutamate residues, placing its proximal amine group nitrogen atom 3.16 and 2.17 Å in mol A and 2.97 and 2.65 Å in mol B away from the nearest oxygen atoms of the carboxylate groups of Glu288 and Glu276, respectively. (B) Compound <b>14a</b> (carbon, light blue; nitrogen, blue; chlorine, light green; all rendered as sticks) cocrystallized with the <i>H. influenzae</i> BC at 2.5 Å (light blue ribbon cartoon; PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6OI8">6OI8</a>). The T-loop (ref <a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a>) in the <i>H. influenzae</i> crystal structure is presumably disordered, as observed previously in other BC crystal structures (ref <a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a>) and not represented by the ribbon cartoon. The core pyridopyrimidine scaffold of <b>14a</b> is found in the identical position as that of parent compound <b>1</b>. The 7-position moiety extending off <b>14a</b> points toward the conserved glutamate residues, placing its proximal amine group nitrogen atom 3.29 and 2.91 Å away from the nearest oxygen atoms of the carboxylate groups of Glu288 and Glu276, respectively. (C) The vector of <b>14a</b> that extends off the dihalo-aromatic ring is positioned, as expected, within the small slot in the active site surface, as shown by the partially transparent solvent accessible surface representation in gray. Images prepared using PyMOL.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00625/20190815/images/large/jm-2019-006255_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00625&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Consistent with our hypothesis based on the structure of <b>1</b>, the 7-position modification of <b>13f</b> extends further into the interior of the BC enzyme, placing the distal amine within H-bonding distance to Glu288 and Glu276. Consistent with this possibility, the IC<sub>50</sub> results show a 5-fold improvement in the inhibition of <b>13f</b> compared to <b>4a</b>, possibly reflecting the binding contribution of the new H-bond interactions to the conserved glutamate residues that are formed by <b>13f</b>; however, mutation studies involving both glutamate residues would be needed to more definitively assign the binding affinity consequences of the 7-position modification.</div><div class="NLM_p last">To investigate the structural consequences of modifications off the dihalo ring, the crystal structure of <b>14a</b> in complex with WT BC enzyme from <i>H. influenzae</i> was solved at 2.5 Å (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B,C; see <a href="/doi/suppl/10.1021/acs.jmedchem.9b00625/suppl_file/jm9b00625_si_001.pdf" class="ext-link">Table S1</a> for crystallographic statistics). The active site of the <i>H. influenzae</i> BC enzyme is nearly identical to other Gram-negative species of interest.<a onclick="showRef(event, 'ref5 ref18'); return false;" href="javascript:void(0);" class="ref ref5 ref18">(5,18)</a> The structure shows that the pyridopyrimidine ring scaffold of <b>14a</b> occupies the same position as <b>13f</b> and the previously reported structure of <b>1</b>, demonstrating that the pyridyl modification off the dihalo ring does not alter the binding conformation of the core ring system. This modification is positioned, as intended, within the slot region of the BC active site, potentially making packing interactions to the residues in the walls of the slot (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>C). The structure also shows that the 7-position moiety of <b>14a</b> extends toward the conserved Glu288 and Glu276 residues, positioning the primary amine of <b>14a</b> within H-bonding distance to the two acid residues (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B). Thus, both modifications found on <b>14a</b> had the intended structural consequences.</div></div><div id="sec3_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Bacterial Cytological Profiling</h3><div class="NLM_p">The optimization of antibacterial compounds may be compromised by off-target activity that would complicate interpretation of structure–activity relationships.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> To test that the improved antibacterial potency of the novel BC inhibitors was due to on-target activity, bacterial cytological profiling was used. Briefly, this technique uses high-resolution fluorescence microscopy to compare the effects on the cytological structure of novel antibiotics to a set of reference control antibiotics with known targets.<a onclick="showRef(event, 'ref39 ref40'); return false;" href="javascript:void(0);" class="ref ref39 ref40">(39,40)</a> Compounds that inhibit different pathways generate reproducible and distinct cytological signatures that can be used to infer the mechanism of action of unknown compounds.<a onclick="showRef(event, 'ref39 ref40 ref41 ref42 ref43 ref44'); return false;" href="javascript:void(0);" class="ref ref39 ref40 ref41 ref42 ref43 ref44">(39−44)</a> When <i>E. coli</i> cells were treated with the previously reported BC inhibitor <b>1</b><a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> at ∼3-fold the MIC for 2 h, they formed short chains of three to eight cells, a phenotype that is characteristic of lipid biogenesis inhibitors (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>). Two novel inhibitors, <b>13f</b> and <b>14e</b>, were tested in the profiling assay. It was found that both showed cytological profiles very similar to each other and to parent compound <b>1</b>, suggesting that they inhibit the same target in vivo<i>.</i></div><figure id="fig3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00625/20190815/images/medium/jm-2019-006255_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00625/20190815/images/large/jm-2019-006255_0014.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Cells treated with BC inhibitors produce chains of cells. Cells were treated with either DMSO or a compound at the indicated concentrations for 2 h. The top panel shows FM 4-64-stained cell membranes (red), and the bottom panel shows both cell membranes and DNA stained with DAPI (blue). Scale bar equals 1 μm.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00625/20190815/images/large/jm-2019-006255_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00625&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec3_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Selectivity Characterization</h3><div class="NLM_p">Compound <b>1</b> is a member of a class of compounds that were originally identified as inhibitors of VEGFR2 and FGFR1 eukaryotic protein kinases.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> However, in the original report,<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a><b>1</b> was shown to have acceptable selectivity for bacterial BC over these and 28 other eukaryotic protein kinases. To test if the potency improvements of the derivative compounds synthesized in the current work came at the expense of losses in selectivity, two particularly promising compounds, <b>14a</b> and <b>14e</b>, were subjected to a similar panel of kinases. Compounds were profiled at a single concentration of 25 μM in the Z′-LYTE kinase activity assay at Invitrogen. The results are shown in <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>. Compared to parent compound <b>1</b>, the novel compounds retain excellent and even improved selectivity against this panel of kinases.</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Percent Inhibition of Eukaryotic Kinases at a Compound Concentration of 25 μM<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00625/20190815/images/medium/jm-2019-006255_0003.gif" alt="" id="gr15" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup><sup>a</sup></sup><p class="last">To facilitate comparisons, the color of each cell of the Table correlates with the percent inhibition value with red indicating a high percent inhibition and green indicating a low percent inhibition.</p></div></div><div></div></div></div><div id="sec3_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Determination of Resistance Liabilities</h3><div class="NLM_p">One concern with inhibitors targeting a single enzyme is that single-point mutations in the target enzyme may lead to high-level resistance. Due to the high bacterial burden and error-prone replication, it is feasible for bacteria to sample every possible point mutation during the course of an infection. If a single-point mutation can disrupt inhibitor binding without drastically impairing enzyme function, then the infection will become resistant (for example, see ref <a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a>). Efflux-mediated resistance is more common than target-based resistance, but upregulation of a single efflux system usually has a moderate impact on potency.</div><div class="NLM_p">To determine whether high-level resistance to our inhibitors could arise in <i>P. aeruginosa</i>, we selected for spontaneous resistant mutants in the efflux-compromised <i>P. aeruginosa</i> strain. The lack of genes encoding the three major antibiotic efflux systems increases the likelihood of identifying rare mutations in other genes. However, it should be noted that <i>P. aeruginosa</i> encodes additional efflux systems that can contribute to resistance.</div><div class="NLM_p">We selected for mutants on agar plates containing 4-, 8-, or 16-fold the MIC of compounds <b>1</b>, <b>14a</b>, or <b>14e</b>. The spontaneous mutation frequency of these compounds in this strain background was similar, ranging from 2 × 10<sup>–9</sup> for <b>14e</b> to 9 × 10<sup>–9</sup> for <b>14a</b> at 8-fold above the MIC. These frequencies are low and similar to those observed for compound <b>1</b> previously.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> We note however that spontaneous mutation frequencies would likely be much higher for a WT <i>P. aeruginosa</i> strain carrying the full suite of efflux systems as these additional components can also mutate to more efficiently efflux lethal compounds.</div><div class="NLM_p">Several resistant colonies of the efflux-compromised <i>P. aeruginosa</i> strain were found to harbor mutations in the BC coding sequence. These mutations encoded changes to amino acids Leu278 and Ile437, which are located in the BC active site according to a structure of the <i>P. aeruginosa</i> enzyme in complex with a substrate analog (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2VQD">2VQD</a>).<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> Both residues are highly conserved across BC species including <i>E. coli</i> and <i>H. influenzae</i>.<a onclick="showRef(event, 'ref5 ref18'); return false;" href="javascript:void(0);" class="ref ref5 ref18">(5,18)</a> The I437N and I437T mutations identified here have also been observed previously when efflux-compromised strains of <i>E. coli</i> and <i>H. influenzae</i> were grown in the presence of <b>1</b>.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> The L278F mutation observed here in <i>P. aeruginosa</i> was not identified in either the <i>E. coli</i> or <i>H. influenzae</i> enzymes.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> However, several other active site mutations were observed in <i>E. coli</i> and <i>H. influenzae</i> that were not observed in the current study, possibly because not enough resistant colonies were screened to exhaustively identify all possible mutants. If this were the case, additional screening would eventually reveal the same set of resistance-conferring mutants for all BC species. Alternatively, the different sets of active site mutations observed in the BC enzymes from different species may reflect the influence of subtle BC structural differences that result in truly different sets of possible resistance-conferring mutations.</div><div class="NLM_p">All three <i>P. aeruginosa</i> BC active site mutations isolated in this study have markedly stronger effects on the potency of <b>14a</b> compared to <b>14e</b> (<a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>). The L278F mutation raised the MIC of <b>14a</b> by 256-fold above the MIC of the parent strain, while both I437 mutations raised the MIC 128-fold. In contrast, <b>14e</b> was less affected by all three mutations, showing 8-fold MIC shifts above the parent strain.</div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. MIC Changes Conferred by Spontaneous <i>P. aeruginosa</i> Mutations</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="3" align="center">MIC (mg L<sup>–1</sup>)</th><th class="rowsep1 colsep0" colspan="3" align="center">fold change from WT</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">strain</th><th class="colsep0 rowsep0" align="center"><b>1</b></th><th class="colsep0 rowsep0" align="center"><b>14a</b></th><th class="colsep0 rowsep0" align="center"><b>14e</b></th><th class="colsep0 rowsep0" align="center"><b>1</b></th><th class="colsep0 rowsep0" align="center"><b>14a</b></th><th class="colsep0 rowsep0" align="center"><b>14e</b></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">WT background<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">16</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">(1)</td><td class="colsep0 rowsep0" align="left">(1)</td><td class="colsep0 rowsep0" align="left">(1)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BC L278F</td><td class="colsep0 rowsep0" align="left">128</td><td class="colsep0 rowsep0" align="left">256</td><td class="colsep0 rowsep0" align="left">64</td><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">256</td><td class="colsep0 rowsep0" align="left">8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BC I437N</td><td class="colsep0 rowsep0" align="left">>128</td><td class="colsep0 rowsep0" align="left">128</td><td class="colsep0 rowsep0" align="left">64</td><td class="colsep0 rowsep0" align="left">>8</td><td class="colsep0 rowsep0" align="left">128</td><td class="colsep0 rowsep0" align="left">8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BC I437T</td><td class="colsep0 rowsep0" align="left">>128</td><td class="colsep0 rowsep0" align="left">128</td><td class="colsep0 rowsep0" align="left">64</td><td class="colsep0 rowsep0" align="left">>8</td><td class="colsep0 rowsep0" align="left">128</td><td class="colsep0 rowsep0" align="left">8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MexT F201I</td><td class="colsep0 rowsep0" align="left">>128</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">>8</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MexT P202L</td><td class="colsep0 rowsep0" align="left">>128</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">>8</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MexT V226L</td><td class="colsep0 rowsep0" align="left">>128</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">>8</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">INV(<i>muxA</i>-<i>mexZ</i>)</td><td class="colsep0 rowsep0" align="left">>128</td><td class="colsep0 rowsep0" align="left">32</td><td class="colsep0 rowsep0" align="left">32</td><td class="colsep0 rowsep0" align="left">>8</td><td class="colsep0 rowsep0" align="left">32</td><td class="colsep0 rowsep0" align="left">4</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t6fn1"><div class="footnote" id="t6fn1"><sup><sup>a</sup></sup><p class="last"><i>P. aeruginosa</i> Δ<i>mexAB</i>-<i>oprM</i>, Δ<i>mexCD</i>-<i>oprJ</i>, and Δ<i>mexEF-oprN</i> (PAM1626 from ref <a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a>).</p></div></div></div><div class="NLM_p">Examination of the <b>14a</b><i>H. influenzae</i> BC X-ray cocrystal structure overlaid with the previously reported <i>P. aeruginosa</i> BC with a substrate analog cocrystal structure<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> offers a speculative model for how the resistance-conferring mutations might disrupt inhibitor binding but not significantly impact substrate turnover (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). In the <i>H. influenzae</i> structure, the pyridopyrimidine ring moiety of the <b>14a</b> ligand extends further toward Leu278 than the corresponding adenine group of the substrate analog. Because of this difference, mutation to the larger phenylalanine amino acid could sterically hinder <b>14a</b> binding while leaving substrate and transition state binding unaffected. By contrast, the structural overlap of the inhibitor and substrate near the 437 position appears quite extensive in the structural alignment shown in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>. As the resistance-conferring mutations at this position introduce more polarity, it may be that electrostatic forces play a large role in affording the differential effect on the substrate/transition state versus inhibitor binding. Further catalytic and inhibition studies as well as additional X-ray crystal structures would be needed to understand the precise consequences of the resistance-conferring mutations. These future studies may also reveal why <b>14e</b> MIC potency is not as largely affected by the resistance-conferring mutations as the other inhibitors.</div><figure id="fig4" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00625/20190815/images/medium/jm-2019-006255_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00625/20190815/images/large/jm-2019-006255_0013.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Alignment of the <i>P. aeruginosa</i> BC enzyme (shown as yellow ribbon; PBD code 2VQD) cocrystallized with substrate analog AMPCP (carbon, yellow; nitrogen, blue; phosphate, orange; oxygen, red; all rendered as sticks) and the <i>H. influenzae</i> BC enzyme (shown as light blue ribbon) cocrystallized with <b>14a</b> (carbon, light blue; nitrogen, blue; chlorine, light green; all rendered as sticks; PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6OI8">6OI8</a>). Resistance-conferring mutants were found at residue positions 278 and 437, which, in the WT enzymes shown here, are leucine and isoleucine, respectively (shown as sticks).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00625/20190815/images/large/jm-2019-006255_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00625&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Additional resistant mutants were subjected to whole-genome sequencing to identify the underlying resistance mechanism. Several mutants were found to have mutations conferring single amino acid changes in MexT, which is a transcriptional regulator affecting many cellular processes.<a onclick="showRef(event, 'ref48 ref49 ref50'); return false;" href="javascript:void(0);" class="ref ref48 ref49 ref50">(48−50)</a> The effect of <i>mexT</i> mutation was highly compound-specific. Many BC inhibitors, including <b>14a</b> and <b>14e</b>, were unaffected by any <i>mexT</i> mutation (<a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>). However, the potency of <b>1</b> was reduced >8-fold by any of the three individual <i>mexT</i> mutations. The <i>mexT</i> mutants also had elevated resistance to ciprofloxacin and levofloxacin (data not shown), suggesting that a general efflux-mediated mechanism is likely responsible for the observed phenotypes.</div><div class="NLM_p last">Sequencing also uncovered a large (647 kb) chromosomal inversion disrupting the coding sequences of two efflux-related genes: <i>mexZ</i>, a repressor of <i>mexXY</i>, and <i>muxA</i>, which encodes the periplasmic membrane fusion component of the MuxABC-OpmB efflux system. The disruption presumably upregulates MexXY production while, at the same time, abrogating the function of MuxABC-OpmB, but these hypotheses were not tested experimentally. The <i>muxA-mexZ</i> inversion affected all tested BC inhibitors, reducing their potency 4- to 32-fold, and also reduced the potency of ciprofloxacin and levofloxacin (data not shown), supporting the premise that resistance is efflux-mediated. The lack of mutations in other pathways suggests that efflux and BC coding mutations are the major resistance mechanisms for this set of compounds.</div></div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i22">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_44266" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_44266" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In this report, the physicochemical properties of starting compound <b>1</b> were rationally optimized to improve Gram-negative cellular accumulation and BC target inhibition, resulting in the overall enhancement of antibacterial activity. Reducing the cLog<i>D</i> value and installing a primary amine with a p<i>K</i><sub>a</sub> value in the physiological pH range resulted in compounds such as <b>14a</b> and <b>14e</b>, which improved potency up to 64-fold against WT <i>E. coli</i>, 32-fold against WT <i>K. pneumoniae</i>, and 16-fold against an efflux-compromised strain of <i>P. aeruginosa</i>.</div><div class="NLM_p">While these MIC improvements demonstrate the potential of BC inhibitors as antibacterial agents, several challenges remain. First, the understanding of the parameters affecting Gram-negative penetration and efflux is still incomplete. For example, <b>13h</b> showed large MIC and cell penetration improvements compared to the parent compound despite having an almost identical cLog<i>D</i> value. This observation lends further support to the conclusion drawn by more recent studies suggesting that cLog<i>D</i> is not as predictive of Gram-negative cellular accumulation as was once proposed.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> In addition, <b>14a</b>, one of the most potent compounds, still had rather large decreases in MIC in the presence of PMBN and against efflux-compromised strains, indicating that cell penetration and efflux are still significant barriers to the potency of this compound despite having “optimal” properties. These results suggest that there are other, as of yet, undiscovered properties that likely contribute to penetration and efflux avoidance. Recent work has begun to provide evidence that these other properties could likely include a molecule’s globularity and number of rotatable bonds.<a onclick="showRef(event, 'ref26 ref29'); return false;" href="javascript:void(0);" class="ref ref26 ref29">(26,29)</a> In addition, the optimization strategy used here was largely ineffective at improving potency against WT <i>A. baumannii</i> and <i>P. aeruginosa</i>. The unique membrane composition and efflux mechanisms of these species may require different or additional molecular properties to be satisfied before adequate antibacterial potencies can be achieved.</div><div class="NLM_p last">A second challenge illustrated by this work is the inherent resistance liability associated with targets encoded by a single-copy gene such as BC. All clinically relevant antibiotics overcome this challenge by inhibiting targets encoded by multiple gene copies such as the ribosome (aminoglycosides, macrolides, etc.) or multiple related targets such as penicillin-binding proteins (β-lactams) and topoisomerases (quinolones) or by binding to nonprotein targets (vancomycin, polymyxins, etc.). The large MIC shifts observed herein from BC active site mutations illustrate the ease by which single amino acid changes can disrupt inhibitor binding without compromising catalytic activity enough to jeopardize cell survival. In principle, if an inhibitor mimicked the molecules along the catalytic pathway extensively enough, then such single-point mutations could not arise. Designing such inhibitors has proved challenging, but compounds such as <b>14e</b> with reduced MIC shifts in the presence of BC active site mutations represent a promising starting point for further optimization.</div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i23">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_78124" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_78124" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> Chemistry</h3><div class="NLM_p">All chemicals were purchased from commercial suppliers and used as received. Flash chromatography was carried out with prepacked silica cartridges from either ISCO or SiliCycle on an ISCO Companion chromatography system using gradient elution. NMR spectra were recorded on an Oxford 400 MHz spectrometer and referenced to tetramethylsilane. Preparative HPLC was performed on a Luna C18 5 μm column (50 mm × 21 mm) eluted with mixtures of water acetonitrile. All final compounds were purified to >95% chemical purity, as assayed by HPLC with UV detection at λ = 254 and 210 nm.</div><div class="NLM_p last">Analytical LCMS was conducted on an Agilent 1200 series UPLC coupled with an Agilent G1946D mass spectrometer and an Agilent 1200 DAD UV detector. All methods utilized the following parameters: capillary voltage, 2500 V; fragmentor/gain, 100; gain, 1.0; drying gas flow, 12.5 L/min; gas temperature, 350 °C; nebulizer pressure, 60 psi; scan range, 100–1000 amu; eluent A, 0.1% TFA in water; eluent B, 0.1% TFA in acetonitrile; flow, 0.8 mL/min; column, Agilent SB-C18 RRHD 1.8 μm; 2.1 × 50 mm; column temperature, 35 °C. LC retention times are based on the following methods: HPLC method 1, 2–50% eluent B (gradient) over 2.0 min; HPLC method 2, 2–99% eluent B (gradient) over 2.0 min; HPLC method 3: 2–30% eluent B (gradient) over 2.0 min.</div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> 2-[2-(3-Pyridyl)phenyl]acetonitrile (<b>2c</b>)</h3><div class="NLM_p last">For general procedure A, 2-(2-bromophenyl)acetonitrile (100 mg, 0.51 mmol), 3-pyridylboronic acid (130 mg, 1.0 mmol), and potassium fluoride (60 mg, 1.0 mmol) were combined in 1,4-dioxane (3 mL) and methanol (0.90 mL) under nitrogen. Pd<sub>2</sub>(dba)<sub>3</sub> (9.3 mg, 0.010 mmol) and dicyclohexyl-[2-(2,6-dimethoxyphenyl)phenyl]phosphane (SPhos) (4.2 mg, 0.010 mmol) were added, and the reaction was heated at 95 °C overnight. After cooling to room temperature, the reaction was filtered, and the filtrate was concentrated and used in the next step without further purification (97 mg, 98%). LCMS (ESI) [M + H]<sup>+</sup>, 195.0.</div></div><div id="sec5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> 2-[2-Chloro-6-(3-pyridyl)phenyl]acetonitrile (<b>2d</b>)</h3><div class="NLM_p last">2-[2-Chloro-6-(3-pyridyl)phenyl]acetonitrile (<b>2d</b>) was prepared from 2-(2-bromo-6-chloro-phenyl)acetonitrile (410 mg, 1.8 mmol) using general procedure A. The crude material was purified by chromatography on silica gel to yield the desired product (330 mg, 82%). LCMS (ESI) [M + H]<sup>+</sup>, 229.0.</div></div><div id="sec5_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> 2-[2-Chloro-6-(6-methoxy-3-pyridyl)phenyl]acetonitrile (<b>2e</b>)</h3><div class="NLM_p last">2-[2-Chloro-6-(6-methoxy-3-pyridyl)phenyl]acetonitrile (<b>2e</b>) was prepared using general procedure A. The crude material was purified by RP-HPLC to yield the desired product (86 mg, 51%). LCMS (ESI) [M + H]<sup>+</sup>, 259.0.</div></div><div id="sec5_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> 2-(2-Chloro-6-isobutoxy-phenyl)acetonitrile (<b>2f</b>)</h3><div class="NLM_p">2-Chloro-6-hydroxy-benzaldehyde (1.0 g, 6.4 mmol) and isobutyliodide (1.9 mL, 16 mmol) were combined in 8.0 mL of anhydrous DMF, treated with potassium carbonate (1.3 g, 9.6 mmol), and heated at 90 °C in a sealed vessel for 2 days. The reaction was cooled to room temperature, diluted with water, and extracted with ethyl acetate. The ethyl acetate extract was washed with aqueous sodium metabisulfite solution and then brine and concentrated under vacuum. The crude residue was purified on a silica gel column to give 2-chloro-6-isobutoxybenzaldehyde (930 mg, 69%). LCMS (ESI) [M + H]<sup>+</sup>, 213.0.</div><div class="NLM_p last">A 200 mL round-bottom flask fitted with an addition funnel and nitrogen-purged was charged with potassium <i>tert</i>-butoxide (20% in THF, 10 mL, 18 mmol) and cooled in a −45 °C bath. A solution of 1-isocyanomethanesulfonyl-4-methyl-benzene (1.7 g, 8.8 mmol) in 12 mL of anhydrous THF was added dropwise over 10 min. After an additional 10 min, a solution of 2-chloro-6-isobutoxy-benzaldehyde (930 mg, 4.4 mmol) in 12 mL of THF was added dropwise over 10 min. After 2 h, the reaction was allowed to warm to room temperature, and then the solvent was removed under vacuum. The residue was taken up in 30 mL of water and acidified to pH 6 with 10% acetic acid. The product was extracted with ethyl acetate (3 × 30 mL), and the combined extracts were washed with saturated aqueous sodium bicarbonate and then brine and then dried over sodium sulfate, filtered, and concentrated under vacuum. The crude residue was purified on a silica gel column to give the desired product (950 mg, 97%). LCMS (ESI) [M + H]<sup>+</sup>, 223.9.</div></div><div id="sec5_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> 6-([1,1′-Biphenyl]-2-yl)pyrido[2,3-<i>d</i>]pyrimidine-2,7-diamine (<b>4b</b>)</h3><div class="NLM_p last">A solution of 2-([1,1′-biphenyl]-2-yl)acetonitrile (34 mg, 0.17 mmol) in ethoxyethanol (0.5 mL) was treated with 25% sodium methoxide in methanol (31 μL, 0.14 mmol) followed by 2,4-diaminopyrimidine-5-carbaldehyde (20 mg, 0.14 mmol) and then heated at 125 °C for 30 min. The reaction was diluted with water and acidified with TFA. The solution was purified by RP-HPLC to give the desired product (30 mg, 49%). HRMS (ESI) calculated [M + H]<sup>+</sup>, 314.1405; found, 314.1399; <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz): δ 8.78 (1H, s), 7.84 (1H, s), 7.57 (1H, t, <i>J</i> = 7.2 Hz), 7.52 (1H, t, <i>J</i> = 7.6 Hz), 7.46 (1H, d, <i>J</i> = 7.2 Hz), 7.39 (1H, d, <i>J</i> = 7.2 Hz), 7.28–7.21 (5H, m).</div></div><div id="sec5_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> 6-[2-(3-Pyridyl)phenyl]pyrido[2,3-<i>d</i>]pyrimidine-2,7-diamine (<b>4c</b>)</h3><div class="NLM_p last">For general procedure B, a solution of 2-[2-(3-pyridyl)phenyl]acetonitrile (97 mg, 0.50 mmol) in ethylene glycol (1.0 mL) was treated with sodium methoxide (25% in methanol) (0.11 mL, 0.50 mmol) and 2,4-diamino-pyrimidine-5-carbaldehyde (62 mg, 0.45 mmol). The reaction was heated at 140 °C for 2 h, then cooled to rt, acidified with TFA (0.08 mL, 1 mmol), and purified by RP-HPLC to give the desired product (65 mg, 30%). HRMS (ESI) calcd [M + H]<sup>+</sup>, 315.1358; found, 315.1355; <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz): δ 8.94 (1H, s), 8.04 (2H, d, <i>J</i> = 8.0 Hz), 7.93 (1H, s), 7.85 (3H, br s), 7.63–7.54 (4H, m), 7.48 (1H, t, <i>J</i> = 6.8 Hz), 1.96 (2H, br s), 1.75 (2H, br s).</div></div><div id="sec5_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> 6-[2-Chloro-6-(3-pyridyl)phenyl]pyrido[2,3-<i>d</i>]pyrimidine-2,7-diamine (<b>4d</b>)</h3><div class="NLM_p last">6-[2-Chloro-6-(3-pyridyl)phenyl]pyrido[2,3-<i>d</i>]pyrimidine-2,7-diamine (<b>4d</b>) was prepared from 2-[2-chloro-6-(3-pyridyl)phenyl]acetonitrile (33 mg, 0.14 mmol) using general procedure B. The crude material was purified by RP-HPLC to yield the desired product (10 mg, 15%). HRMS (ESI) calcd [M + H]<sup>+</sup>, 349.0968; found, 349.0959; <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz): δ 8.78 (1H, s), 8.43–8.41 (2H, m), 7.85 (1H, s), 7.72 (1H, dd, <i>J</i> = 8.0, 0.4 Hz), 7.65–7.60 (2H, m), 7.49 (1H, dd, <i>J</i> = 8.0 ,0.4 Hz), 7.31 (1H, dd, <i>J</i> = 7.6, 5.2 Hz).</div></div><div id="sec5_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> 6-[2-Chloro-6-(6-methoxy-3-pyridyl)phenyl]pyrido[2,3-<i>d</i>]pyrimidine-2,7-diamine (<b>4e</b>)</h3><div class="NLM_p last">6-[2-Chloro-6-(6-methoxy-3-pyridyl)phenyl]pyrido[2,3-<i>d</i>]pyrimidine-2,7-diamine (<b>4e</b>) was prepared using general procedure B (34 mg, 27%). HRMS (ESI) calcd [M + H]<sup>+</sup>, 379.1074; found, 379.1059; <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz): δ 8.80 (1H, s), 8.02 (1H, d, <i>J</i> = 2.4 Hz), 7.86 (1H, s), 7.67 (1H, d, <i>J</i> = 8.0 Hz), 7.59 (1H, t, <i>J</i> = 8.0 Hz) 7.48 (1H, dd, <i>J</i> = 8.8, 2.4 Hz), 7.44 (1H, d, <i>J</i> = 7.6 Hz), 6.70 (1H, d, <i>J</i> = 8.4 Hz), 3.77 (3H, s).</div></div><div id="sec5_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> 6-(2-Chloro-6-isobutoxy-phenyl)pyrido[2,3-<i>d</i>]pyrimidine-2,7-diamine (<b>4f</b>)</h3><div class="NLM_p last">A solution of 6-(2-chloro-6-isobutoxy-phenyl)-2-methylsulfinyl-pyrido[2,3-<i>d</i>]pyrimidin-7-amine (44 mg, 0.11 mmol) in 1.5 mL of anhydrous THF was treated with 5 M ammonia in methanol (0.45 mL, 2.3 mmol) and heated to 70 °C in a sealed vial overnight. The reaction was concentrated under vacuum, and the crude residue was purified by RP-HPLC to give the desired product (23 mg, 60%). HRMS (ESI) calcd [M + H]<sup>+</sup>, 344.1278; found, 344.1276; <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz): δ 8.90 (1H, s), 7.93 (1H, s), 7.46 (1H, t <i>J</i> = 8.0 Hz), 7.21 (1 h, d, <i>J</i> = 8.0 Hz), 7.14 (1H, d, <i>J</i> = 8.4 Hz), 3.80–3.72 (2H, m), 1.86–1.81 (1H, m), 0.79 (6H, d, <i>J</i> = 6.8 Hz).</div></div><div id="sec5_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> 6-(2,6-Dichlorophenyl)-2-methylsulfanyl-pyrido[2,3-<i>d</i>]pyrimidin-7-amine (<b>6a</b>)</h3><div class="NLM_p last">4-Amino-2-methylsulfanyl-pyrimidine-5-carbaldehyde (3.1 g, 18 mmol) was dissolved in 40 mL of anhydrous dimethylformamide and cooled in an ice/methanol bath (−10 °C) for 15 min under a nitrogen atmosphere. Sodium hydride (60% in mineral oil, 0.73 g, 18 mmol) divided into five portions was added over the course of 5 min, keeping the reaction mixture cooled with stirring. After an additional 5 min, a solution of 2,6-dichlorobenzylacetonitrile (4.9 g, 27 mmol) in 12 mL of anhydrous dimethylformamide was added dropwise via a syringe. The reaction mixture was stirred at −10 °C for 30 min and then allowed to warm to room temperature overnight after which time the reaction mixture was cooled in an ice bath and slowly quenched with 50 mL of saturated aqueous ammonium chloride. After stirring for 20 min, the product was extracted into DCM. The combined DCM extracts were dried over anhydrous magnesium sulfate, filtered, and concentrated under reduced pressure. The residue was redissolved in ethyl acetate and washed with brine. The organic layer was filtered through a sintered glass funnel, and the filtrate was dried over anhydrous sodium sulfate, filtered, and concentrated under reduced pressure to dryness. The residue was purified by silica gel chromatography eluted with ethyl acetate/dichloromethane to afford 710 mg (11%) of 6-(2,6-dichlorophenyl)-2-methylsulfanyl-pyrido[2,3-<i>d</i>]pyrimidin-7-amine. LCMS (ESI): [M + H]<sup>+</sup>, 336.9.</div></div><div id="sec5_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> 6-[2-Chloro-6-(3-Pyridyl)Phenyl]-2-Methylsulfanyl-Pyrido[2,3-d]Pyrimidin-7-Amine (<b>6d</b>)</h3><div class="NLM_p last">2-[2-Chloro-6-(3-pyridyl)phenyl]acetonitrile (210 mg, 0.94 mmol) was dissolved in DMF (3 mL) and THF (1.5 mL) and cooled in an ice bath. Sodium hydride (60% in mineral oil, 39 mg, 0.98 mmol) was added followed by 4-amino-2-methylsulfanyl-pyrimidine-5-carbaldehyde (160 mg, 0.94 mmol), and the reaction was allowed to warm to rt. After stirring overnight, the reaction was poured into ice and acidified to pH 4 with 10% aqueous acetic acid. The product was extracted with ethyl acetate (2×), and the combined organics were washed with water and then brine and dried with sodium sulfate, filtered, and concentrated under reduced pressure. The oil was taken up in DCM and purified by silica gel chromatography eluted with methanol/dichloromethane to afford 110 mg (30%) of 6-[2-chloro-6-(3-pyridyl)phenyl]-2-methylsulfanyl-pyrido[2,3-<i>d</i>]pyrimidin-7-amine. LCMS (ESI): [M + H]<sup>+</sup>, 380.0.</div></div><div id="sec5_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> 6-(2-Chloro-6-isobutoxy-phenyl)-2-methylsulfanyl-pyrido[2,3-<i>d</i>]pyrimidin-7-amine (<b>6f</b>)</h3><div class="NLM_p last">A solution of 2-(2-chloro-6-isobutoxy-phenyl)acetonitrile (950 mg, 4.3 mmol) in 6 mL of anhydrous THF was cooled to −10 °C and treated with sodium hydride (60% in mineral oil, 200 mg, 4.9 mmol) (60% oil dispersion). After 10 min, a solution of 4-amino-2-methylsulfanyl-pyrimidine-5-carbaldehyde (720 mg, 4.3 mmol) in 14 mL of anhydrous DMF was added dropwise over 10 min. After 2 h, the reaction was warmed to 60 °C and stirred overnight. After cooling to room temperature, the reaction was diluted with water (30 mL) and extracted with ethyl acetate (3 × 75 mL). The combined extracts were washed with brine (3 × 30 mL), dried with sodium sulfate, and concentrated under vacuum. The crude residue was purified on a silica gel column to give the desired product (950 mg, 60%). LCMS (ESI) [M + H]<sup>+</sup>, 375.0.</div></div><div id="sec5_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> 6-(2-Chloro-6-isobutoxy-phenyl)-2-methylsulfinyl-pyrido[2,3-<i>d</i>]pyrimidin-7-amine (<b>6g</b>)</h3><div class="NLM_p last">A solution of 6-(2-chloro-6-isobutoxy-phenyl)-2-methylsulfanyl-pyrido[2,3-<i>d</i>]pyrimidin-7-amine (43 mg, 0.11 mmol) in DCM (2 mL) was treated with 2-(benzenesulfonyl)-3-phenyl-oxaziridine (30 mg, 0.11 mmol) and stirred overnight at rt. The reaction was concentrated under vacuum and used directly in the next step. LCMS (ESI) [M + H]<sup>+</sup>, 391.0.</div></div><div id="sec5_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> 6-(2,6-Dichlorophenyl)-2-methylsulfanyl-8<i>H</i>-pyrido[2,3-<i>d</i>]pyrimidin-7-one (<b>7a</b>)</h3><div class="NLM_p last">6-(2,6-Dichlorophenyl)-2-methylsulfanyl-pyrido[2,3-<i>d</i>]pyrimidin-7-amine (200 mg, 0.60 mmol) was dissolved in TFA (5 mL) and cooled in an ice bath. Sodium nitrite (110 mg, 1.6 mmol) was added, and the reaction was stirred in the bath for 15 min and then at room temperature for 2.5 h, after which time the reaction was slowly added to a mixture of saturated aqueous sodium bicarbonate, solid sodium carbonate, and ice (15–20 mL). The product was extracted with ethyl acetate (2 × 15 mL), and the combined organic layers were dried with sodium sulfate, filtered, and concentrated under vacuum, giving 190 mg (96%) of 6-(2,6-dichlorophenyl)-2-methylsulfanyl-8<i>H</i>-pyrido[2,3-<i>d</i>]pyrimidin-7-one. LCMS (ESI): [M + H]<sup>+</sup>, 338.0.</div></div><div id="sec5_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> 6-[2-Chloro-6-(3-pyridyl)phenyl]-2-methylsulfanyl-8<i>H</i>-pyrido[2,3-<i>d</i>]pyrimidin-7-one (<b>7d</b>)</h3><div class="NLM_p last">6-[2-Chloro-6-(3-pyridyl)phenyl]-2-methylsulfanyl-pyrido[2,3-<i>d</i>]pyrimidin-7-amine (110 mg, 0.28 mmol) was dissolved in TFA (2 mL) and cooled in an ice bath. Sodium nitrite (52 mg, 0.75 mmol) was added, and the reaction was allowed to warm to rt. After 90 min, the reaction was slowly added to a mixture of saturated aqueous sodium bicarbonate, solid sodium carbonate, and ice. The product was extracted with ethyl acetate (2×), and the combined organics were washed with brine, dried with sodium sulfate, filtered, and concentrated under reduced pressure to give 93 mg (88%) of 6-[2-chloro-6-(3-pyridyl)phenyl]-2-methylsulfanyl-8<i>H</i>-pyrido[2,3-<i>d</i>]pyrimidin-7-one. LCMS (ESI): [M + H]<sup>+</sup>, 381.0.</div></div><div id="sec5_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> 7-Chloro-6-(2,6-dichlorophenyl)-2-methylsulfanyl-pyrido[2,3-<i>d</i>]pyrimidine (<b>8a</b>)</h3><div class="NLM_p last">6-(2,6-Dichlorophenyl)-2-methylsulfanyl-8<i>H</i>-pyrido[2,3-<i>d</i>]pyrimidin-7-one (190 mg, 0.57 mmol) was combined with phosphorous oxychloride (1.1 mL, 11 mmol) and dimethylformamide (1.0 μL) at room temperature. The reaction was heated at 95 °C. An additional charge of phosphorous oxychloride (1.1 mL, 11 mmol) was added after 5 min. After 2 h, phosphorous oxychloride was removed by concentration under reduced pressure. The residue was treated with ice and cold water in an ice bath to obtain a solid that was collected by filtration by rinsing with water. After further drying under vacuum, 200 mg (96%) of 7-chloro-6-(2,6-dichlorophenyl)-2-methylsulfanyl-pyrido[2,3-<i>d</i>]pyrimidine was obtained. LCMS (ESI): [M + H]<sup>+</sup>, 356.0.</div></div><div id="sec5_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> 7-Chloro-6-[2-chloro-6-(3-pyridyl)phenyl]-2-methylsulfanyl-pyrido[2,3-<i>d</i>]pyrimidine (<b>8d</b>)</h3><div class="NLM_p last">6-[2-Chloro-6-(3-pyridyl)phenyl]-2-methylsulfanyl-8<i>H</i>-pyrido[2,3-<i>d</i>]pyrimidin-7-one (93 mg, 0.24 mmol), phosphorous oxychloride (0.89 mL, 9.8 mmol), and dimethyl formamide (1.9 μL, 0.020 mmol) were combined, stirred, and heated at 95 °C. After 4 h, the reaction was concentrated under reduced pressure and dried further under high vacuum. The material was taken up in DCM dripped slowly into ice-cold saturated sodium bicarbonate. The layers were separated, and the aqueous layer was extracted again with DCM. The combined organic layers were dried with sodium sulfate, filtered, and concentrated under reduced pressure to give 68 mg (70%) of 7-chloro-6-[2-chloro-6-(3-pyridyl)phenyl]-2-methylsulfanyl-pyrido[2,3-<i>d</i>]pyrimidine. LCMS (ESI): [M + H]<sup>+</sup>, 399.0.</div></div><div id="sec5_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i42" class="anchor-spacer"></div><h3 class="article-section__title" id="_i42"> <i>tert</i>-Butyl <i>N</i>-[3-[6-(2,6-Dichlorophenyl)-2-methylsulfanyl-pyrido[2,3-<i>d</i>]pyrimidin-7-yl]-3-azabicyclo[3.1.0]hexan-6-yl]carbamate (<b>10h</b>)</h3><div class="NLM_p last">7-Chloro-6-(2,6-dichlorophenyl)-2-methylsulfanyl-pyrido[2,3-<i>d</i>]pyrimidine (35 mg, 0.10 mmol) was combined with acetonitrile (0.30 mL), 1,4-dioxane (0.30 mL), <i>tert</i>-butyl <i>N</i>-(3-azabicyclo[3.1.0]hexan-6-yl)carbamate (21 mg, 0.11 mmol), and <i>N</i>,<i>N″</i>-diisopropylethylamine (0.03 mL, 0.20 mmol), and the reaction was heated at 80 °C. Additional portions of diisopropylethylamine and <i>tert</i>-butyl <i>N</i>-(3-azabicyclo[3.1.0]hexan-6-yl)carbamate were added, and the temperature was increased to 90 °C. After heating overnight, the solvent was removed by concentration under reduced pressure. The residue was taken up in DCM (1 mL) and washed with 1 M citric acid (2 × 1 mL), water, and saturated aqueous sodium bicarbonate, and then it was dried over sodium sulfate, filtered, and concentrated under reduced pressure, yielding 66 mg of crude <i>tert</i>-butyl <i>N</i>-[3-[6-(2,6-dichlorophenyl)-2-methylsulfanyl-pyrido[2,3-<i>d</i>]pyrimidin-7-yl]-3-azabicyclo[3.1.0]hexan-6-yl]carbamate, which was taken forward without further purification. LCMS (ESI): [M + H]<sup>+</sup>, 518.1.</div></div><div id="sec5_20" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> <i>tert</i>-Butyl <i>N</i>-[3-[6-(2,6-Dichlorophenyl)-2-methylsulfinyl-pyrido[2,3-<i>d</i>]pyrimidin-7-yl]-3-azabicyclo[3.1.0]hexan-6-yl]carbamate (<b>11h</b>)</h3><div class="NLM_p last"><i>tert</i>-Butyl <i>N</i>-[3-[6-(2,6-dichlorophenyl)-2-methylsulfanyl-pyrido[2,3-<i>d</i>]pyrimidin-7-yl]-3-azabicyclo[3.1.0]hexan-6-yl]carbamate (51 mg, 0.10 mmol) was dissolved in DCM (1 mL), and 3-chloroperbenzoic acid (15 mg, 0.090 mmol) was added at rt. After 2 h, the reaction was washed with saturated aqueous sodium bicarbonate, dried with sodium sulfate, filtered, and concentrated under reduced pressure to yield 60 mg of crude <i>tert</i>-butyl <i>N</i>-[3-[6-(2,6-dichlorophenyl)-2-methylsulfinyl-pyrido[2,3-<i>d</i>]pyrimidin-7-yl]-3-azabicyclo[3.1.0]hexan-6-yl]carbamate, which was taken forward without further purification. LCMS (ESI): [M + H]<sup>+</sup>, 534.0.</div></div><div id="sec5_21" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> 1-[2-Amino-6-(2,6-dichlorophenyl)pyrido[2,3-<i>d</i>]pyrimidin-7-yl]pyrrolidine-3-carboxamide (<b>12a</b>)</h3><div class="NLM_p last">1-[2-Amino-6-(2,6-dichlorophenyl)pyrido[2,3-<i>d</i>]pyrimidin-7-yl]pyrrolidine-3-carboxamide (<b>12a</b>) was prepared by the method in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a> using pyrrolidine-3-carboxamide hydrochloride in step d and gaseous ammonia instead of 5 M ammonia in methanol in step f. HRMS (ESI) calculated [M + H]<sup>+</sup>, 403.0841; found, 403.0822; <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz): δ 8.94 (1H, s), 7.91 (1H, s), 7.65–7.61 (2H, m), 7.51 (1H, t, <i>J</i> = 8.0 Hz), 7.41 (1H, s), 6.91 (1H, s), 2.90 (1H, br s), 2.02 (1H, br s), 1.91 (1H br s).</div></div><div id="sec5_22" class="NLM_sec NLM_sec_level_2"><div id="ac_i45" class="anchor-spacer"></div><h3 class="article-section__title" id="_i45"> [1-[2-Amino-6-(2,6-dichlorophenyl)pyrido[2,3-<i>d</i>]pyrimidin-7-yl]pyrrolidin-3-yl]methanol (<b>12b</b>)</h3><div class="NLM_p last">[1-[2-Amino-6-(2,6-dichlorophenyl)pyrido[2,3-<i>d</i>]pyrimidin-7-yl]pyrrolidin-3-yl]methanol (<b>12b</b>) was prepared by the method in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a> using pyrrolidin-3-yl methanol in step d and gaseous ammonia instead of 5 M ammonia in methanol in step f<b>.</b> HRMS (ESI) calculated [M + H]<sup>+</sup>, 390.0888; found, 390.0883; <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz): δ 8.88 (1H, s), 7.85 (1H, s), 7.58 (1H, d, <i>J</i> = 4.0 Hz), 7.57 (1H, d, <i>J</i> = 3.6 Hz), 7.46 (1H, t, <i>J</i> = 8.0 Hz), 3.31–3.27 (2H, m), 3.21–3.18 (2H, m), 2.21 (1H, q, <i>J</i> = 6.8 Hz), 1.82–1.79 (1H, m), 1.60–1.55 (1H, m).</div></div><div id="sec5_23" class="NLM_sec NLM_sec_level_2"><div id="ac_i46" class="anchor-spacer"></div><h3 class="article-section__title" id="_i46"> <i>tert</i>-Butyl <i>N</i>-[3-[2-Amino-6-(2,6-dichlorophenyl)pyrido[2,3-<i>d</i>]pyrimidin-7-yl]-3-azabicyclo[3.1.0]hexan-6-yl]carbamate (<b>12h</b>)</h3><div class="NLM_p last"><i>tert</i>-Butyl <i>N</i>-[3-[6-(2,6-dichlorophenyl)-2-methylsulfinyl-pyrido[2,3-<i>d</i>]pyrimidin-7-yl]-3-azabicyclo[3.1.0]hexan-6-yl]carbamate (52 mg, 0.10 mmol) was dissolved in THF (2 mL) and ammonia (5 M in methanol) (0.49 mL, 2.5 mmol) at rt. The reaction was then heated at 65 °C for 15 h, after which time the reaction was concentrated to an oil to give crude <i>tert</i>-butyl <i>N</i>-[3-[2-amino-6-(2,6-dichlorophenyl)pyrido[2,3-<i>d</i>]pyrimidin-7-yl]-3-azabicyclo[3.1.0]hexan-6-yl]carbamate, which was used in the next step without further purification. LCMS (ESI): [M + H]<sup>+</sup>, 487.0.</div></div><div id="sec5_24" class="NLM_sec NLM_sec_level_2"><div id="ac_i47" class="anchor-spacer"></div><h3 class="article-section__title" id="_i47"> [3-Amino-1-[2-amino-6-(2,6-dichlorophenyl)pyrido[2,3-<i>d</i>]pyrimidin-7-yl]pyrrolidin-3-yl]methanol (<b>13c</b>)</h3><div class="NLM_p last">[3-Amino-1-[2-amino-6-(2,6-dichlorophenyl)pyrido[2,3-<i>d</i>]pyrimidin-7-yl]pyrrolidin-3-yl]methanol (<b>13c</b>) was prepared by the method in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a> using <i>tert</i>-butyl <i>N</i>-[3-(hydroxymethyl)pyrrolidin-3-yl]carbamate in step d and gaseous ammonia instead of 5 M ammonia in methanol in step f<b>.</b> HRMS (ESI) calculated [M + H]<sup>+</sup>, 405.0997; found, 405.0979; <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz): δ 8.86 (1H, s), 8.14 (3H, br s), 7.83 (1H, s), 7.59 (2H, appt, <i>J</i> = 8.4 Hz), 7.47 (1H, t, <i>J</i> = 8.0 Hz), 5.63 (1H, s), 3.40–3.18 (8H, m), 2.07–2.00 (1H, m), 1.93–1.88 (1H, m).</div></div><div id="sec5_25" class="NLM_sec NLM_sec_level_2"><div id="ac_i48" class="anchor-spacer"></div><h3 class="article-section__title" id="_i48"> 7-[3-(Aminomethyl)-3-fluoro-pyrrolidin-1-yl]-6-(2,6-dichlorophenyl)pyrido[2,3-<i>d</i>]pyrimidin-2-amine (<b>13d</b>)</h3><div class="NLM_p last">7-[3-(Aminomethyl)-3-fluoro-pyrrolidin-1-yl]-6-(2,6-dichlorophenyl)pyrido[2,3-<i>d</i>]pyrimidin-2-amine (<b>13d</b>) was prepared by the method in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a> using <i>tert</i>-butyl <i>N</i>-[(3-fluoropyrrolidin-3-yl)methyl]carbamate in step d and gaseous ammonia instead of 5 M ammonia in methanol in step f<b>.</b> HRMS (ESI) calculated [M + H]<sup>+</sup>, 407.0954; found, 407.0943; <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz): δ 8.85 (1H, s), 8.13 (3H, br s), 7.83 (1H, s), 7.61 (1H, d, <i>J</i> = 8.0 Hz), 7.57 (1H, d, <i>J</i> = 8.0 Hz), 7.47 (1H, t, <i>J</i> = 8.4 Hz), 3.78–3.68 (2H, m), 3.62 (1H, br s), 3.31 (2H, br s), 3.16 (1H, br s), 3.05 (1H, br s), 2.16–2.10 (1H, m), 2.06–2.01 (1H, m).</div></div><div id="sec5_26" class="NLM_sec NLM_sec_level_2"><div id="ac_i49" class="anchor-spacer"></div><h3 class="article-section__title" id="_i49"> 1-[2-Amino-6-(2,6-dichlorophenyl)pyrido[2,3-<i>d</i>]pyrimidin-7-yl]-3-(aminomethyl)pyrrolidin-3-ol (<b>13e</b>)</h3><div class="NLM_p last">1-[2-Amino-6-(2,6-dichlorophenyl)pyrido[2,3-<i>d</i>]pyrimidin-7-yl]-3-(aminomethyl)pyrrolidin-3-ol (<b>13e</b>) was prepared by the method in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a> using <i>tert</i>-butyl <i>N</i>-[(3-hydroxypyrrolidin-3-yl)methyl]carbamate in step d and gaseous ammonia instead of 5 M ammonia in methanol in step f<b>.</b> HRMS (ESI) calculated [M + H]<sup>+</sup>, 405.0997; found, 405.0982; <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz): δ 8.87 (1H, s), 7.83 (3H, s), 7.64 (1H, d, <i>J</i> = 8.0 Hz), 7.61 (1H, d, <i>J</i> = 8.4 Hz), 7.51 (1H, t, <i>J</i> = 8.0 Hz), 2.92 (2H, s), 1.86 (2H, t, <i>J</i> = 6.8 Hz).</div></div><div id="sec5_27" class="NLM_sec NLM_sec_level_2"><div id="ac_i50" class="anchor-spacer"></div><h3 class="article-section__title" id="_i50"> 7-[3-(Aminomethyl)pyrrolidin-1-yl]-6-(2,6-dichlorophenyl)pyrido[2,3-<i>d</i>]pyrimidin-2-amine (<b>13f</b>)</h3><div class="NLM_p last">7-[3-(Aminomethyl)pyrrolidin-1-yl]-6-(2,6-dichlorophenyl)pyrido[2,3-<i>d</i>]pyrimidin-2-amine (<b>13f</b>) was prepared by the method in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a> using 3-(<i>N</i>-Boc-aminomethyl)pyrrolidine in step f and gaseous ammonia instead of 5 M ammonia in methanol in step f. HRMS (ESI) calculated [M + H]<sup>+</sup>, 389.1048; found, 389.1035; <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz): δ 8.88 (1H, s), 7.85 (1H, s), 7.81 (3H, br s), 7.60–7.57 (2H, m), 7.47 (1H, t, <i>J</i> = 8.0 Hz), 3.57 (1H, br s), 3.07 (2H, br s), 2.82–2.78 (1H, m), 2.75–2.70 (1H, m), 2.38–2.31 (1H, m), 1.96–1.92 (1H, m), 1.65–1.60 (1H, m).</div></div><div id="sec5_28" class="NLM_sec NLM_sec_level_2"><div id="ac_i51" class="anchor-spacer"></div><h3 class="article-section__title" id="_i51"> 7-[4-(Aminomethyl)-1-piperidyl]-6-(2,6-dichlorophenyl)pyrido[2,3-<i>d</i>]pyrimidin-2-amine (<b>13g</b>)</h3><div class="NLM_p last">7-[4-(Aminomethyl)-1-piperidyl]-6-(2,6-dichlorophenyl)pyrido[2,3-<i>d</i>]pyrimidin-2-amine (<b>13g</b>) was prepared by the method in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a> using <i>tert</i>-butyl <i>N</i>-(4-piperidinylmethyl)carbamate in step d and 2-(benzenesulfonyl)-3-phenyl-oxaziridine instead of 3-chloroperbenzoic acid in step e<b>.</b> HRMS (ESI) calculated [M + H]<sup>+</sup>, 403.1205; found, 403.1188; <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz): δ 8.96 (1H, s), 7.94 (1H, s), 7.72–7.64 (5H, m), 3.92 (2H, br d, <i>J</i> = 12 Hz), 2.91 (2H, t, <i>J</i> = 12.8 Hz), 2.68 (2H, t, <i>J</i> = 6.4 Hz), 1.79 (1H, br s), 1.65 (2H, d, <i>J</i> = 11.6 Hz), 1.06 (2H, q, <i>J</i> = 10.0 Hz).</div></div><div id="sec5_29" class="NLM_sec NLM_sec_level_2"><div id="ac_i52" class="anchor-spacer"></div><h3 class="article-section__title" id="_i52"> 7-((1<i>R</i>,5<i>S</i>,6<i>S</i>)-6-Amino-3-azabicyclo[3.1.0]hexan-3-yl)-6-(2,6-dichlorophenyl)pyrido[2,3-<i>d</i>]pyrimidin-2-amine (<b>13h</b>)</h3><div class="NLM_p last"><i>tert</i>-Butyl <i>N</i>-[3-[2-amino-6-(2,6-dichlorophenyl)pyrido[2,3-<i>d</i>]pyrimidin-7-yl]-3-azabicyclo[3.1.0]hexan-6-yl]carbamate (48 mg, 0.10 mmol) was dissolved in DCM (0.50 mL) and TFA (0.50 mL) at room temperature. After 45 min, the reaction was concentrated under vacuum. The residue was taken up in dimethylformamide and purified by reverse-phase HPLC (1″ Phenomenex Luna column, 0.1% TFA in water/ACN) to afford 18 mg (35% over four steps) of 7-[(1<i>R</i>,<i>5S</i>,6<i>S</i>)-6-amino-3-azabicyclo[3.1.0]hexan-3-yl)-6-(2,6-dichlorophenyl)pyrido[2,3-<i>d</i>]pyrimidin-2-amine. HRMS (ESI) calculated [M + H]<sup>+</sup>, 387.0891; found, 387.0886; <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz): δ 8.83 (1H, s), 8.04 (3H, br s), 7.77 (1H, s), 7.59 (2H, d, <i>J</i> = 8.0 Hz), 7.48 (1H, t, <i>J</i> = 8.4 Hz), 3.42–3.33 (4H, m), 2.15 (1H, s), 1.95 (2H, s).</div></div><div id="sec5_30" class="NLM_sec NLM_sec_level_2"><div id="ac_i53" class="anchor-spacer"></div><h3 class="article-section__title" id="_i53"> 7-[(1<i>R</i>,5<i>S</i>)-6-Amino-3-azabicyclo[3.1.0]hexan-3-yl)-6-[2-chloro-6-(3-pyridyl)phenyl]pyrido[2,3-<i>d</i>]pyrimidin-2-amine (<b>14a</b>)</h3><div class="NLM_p last">The compound was prepared according to the procedure for <b>12a</b> to give 17 mg of <b>14a</b>. HRMS (ESI) calcd [M + H]<sup>+</sup>, 430.1547; found, 430.1525; <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz): δ 8.88 (1H, s), 8.41 (1H, d, <i>J</i> = 6.4 Hz), 8.31 (1H, d, <i>J</i> = 2 Hz), 8.05 (3H s), 7.89 (1H, s), 7.74 (1H, d, <i>J</i> = 7.6 Hz), 7.63 (1H, t, <i>J</i> = 8.0 Hz), 7.49 (1H, d, 7.6 Hz), 7.44 (1H, d, <i>J</i> = 8.0 Hz), 7.25 (1H, dd, <i>J</i> = 7.6, 4.8 Hz), 3.34 (1H, br s), 3.21–3.16 (2H, m), 2.09 (1H, br s), 1.95 (2H, br s).</div></div><div id="sec5_31" class="NLM_sec NLM_sec_level_2"><div id="ac_i54" class="anchor-spacer"></div><h3 class="article-section__title" id="_i54"> 7-[(1<i>R</i>,5<i>S</i>)-6-Amino-3-azabicyclo[3.1.0]hexan-3-yl)-6-[2-chloro-6-(4-pyridyl)phenyl]pyrido[2,3-<i>d</i>]pyrimidin-2-amine (<b>14b</b>)</h3><div class="NLM_p last">The compound was prepared according to the procedure for <b>14c</b>. HRMS (ESI) calculated [M + H]<sup>+</sup>, 430.1547; found, 430.1526; <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz): δ 8.88 (1H, s), 8.44 (2H, d, <i>J</i> = 5.6 Hz), 8.06 (3H, br s), 7.87 (1H, s), 7.76 (1H, d, <i>J</i> = 7.2 Hz), 7.65 (1H, t, <i>J</i> = 7.6 Hz), 7.50 (1H, d, <i>J</i> = 7.2 Hz), 7.09 (2H, d, <i>J</i> = 4.8 Hz), 2.12 (1H, s), 1.96 (2H, s).</div></div><div id="sec5_32" class="NLM_sec NLM_sec_level_2"><div id="ac_i55" class="anchor-spacer"></div><h3 class="article-section__title" id="_i55"> 7-[(1<i>R</i>,5<i>S</i>)-6-Amino-3-azabicyclo[3.1.0]hexan-3-yl)-6-[2-chloro-6-(5-fluoro-6-methoxy-3-pyridyl)phenyl]pyrido[2,3-<i>d</i>]pyrimidin-2-amine (<b>14c</b>)</h3><div class="NLM_p"><i>tert</i>-Butyl <i>N</i>-[3-[2-amino-6-(2-bromo-6-chloro-phenyl)pyrido[2,3-<i>d</i>]pyrimidin-7-yl]-3-azabicyclo[3.1.0]hexan-6-yl]carbamate (15 mg, 0.030 mmol), 1,4-dioxane (0.60 mL), 1 M sodium carbonate (0.15 mL), (5-fluoro-6-methoxy-3-pyridyl)boronic acid (5.3 mg, 0.030 mmol), and Pd(dppf)Cl<sub>2</sub> DCM complex (2.3 mg, 0.0028 mmol) were combined in a vial under nitrogen at rt. The reaction was stirred at 100 °C for 1 h. The product was extracted with ethyl acetate, and the combined organic layers were dried with sodium sulfate, filtered, and concentrated under reduced pressure to give 20 mg of crude <i>tert</i>-butyl <i>N</i>-[3-[2-amino-6-[2-chloro-6-(5-fluoro-6-methoxy-3-pyridyl)phenyl]pyrido[2,3-<i>d</i>]pyrimidin-7-yl]-3-azabicyclo[3.1.0]hexan-6-yl]carbamate that was used in the next step without further purification. LCMS (ESI): [M + H]<sup>+</sup>, 578.1.</div><div class="NLM_p last">The compound was prepared according to the procedure for <b>13h</b>. HRMS (ESI) calculated [M + H]<sup>+</sup>, 478.1558; found, 478.1538; <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz): δ8.84 (1H, s), 8.06 (3H, br s), 7.84 (1H, s), 7.71 (2H, appd, <i>J</i> = 7.2 Hz), 7.60 (1H, t, <i>J</i> = 7.6 Hz), 7.47 (1H, d, <i>J</i> = 7.6 Hz), 7.39 (1H, d, <i>J</i> = 11.2 Hz), 3.86 (3H, s), 2.08 (1H, s), 1.94 (2H, s).</div></div><div id="sec5_33" class="NLM_sec NLM_sec_level_2"><div id="ac_i56" class="anchor-spacer"></div><h3 class="article-section__title" id="_i56"> 7-[(1<i>R</i>,5<i>S</i>)-6-Amino-3-azabicyclo[3.1.0]hexan-3-yl)-6-[2-chloro-6-(2-methylthiazol-5-yl)phenyl]pyrido[2,3-<i>d</i>]pyrimidin-2-amine (<b>14d</b>)</h3><div class="NLM_p last">The compound was prepared according to the procedure for <b>14c</b>. HRMS (ESI) calculated [M + H]<sup>+</sup>, 450.1267; found, 450.1254; <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz): δ 8.84 (1H, s), 8.02 (3H, br s), 7.76–7.73 (3H, m), 7.65 (1H, d, <i>J</i> = 8.0 Hz), 7.57 (1H, t, <i>J</i> = 8.0 Hz) 2.03 (1H, s), 1.95 (2H, s).</div></div><div id="sec5_34" class="NLM_sec NLM_sec_level_2"><div id="ac_i57" class="anchor-spacer"></div><h3 class="article-section__title" id="_i57"> 7-[(1<i>R</i>,5<i>S</i>)-6-Amino-3-azabicyclo[3.1.0]hexan-3-yl)-6-(2-chloro-6-isobutoxy-phenyl)pyrido[2,3-<i>d</i>]pyrimidin-2-amine (<b>14e</b>)</h3><div class="NLM_p last">The compound was prepared according to the procedure for <b>12a</b>. HRMS (ESI) calculated [M + H]<sup>+</sup>, 425.1856; found 425.1843, <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz): δ 8.89 (1H, s), 8.07 (3H, s), 7.75 (1H, s), 7.43 (1H, t, <i>J</i> = 8.4 Hz), 7.18 (1H, d, <i>J</i> = 8.0 Hz), 7.08 (1H, d, <i>J</i> = 8.4 Hz), 3.77–3.69 (3H, m), 3.35 (4H, br s), 2.11 (1H, s), 1.98 (2H, s), 1.79–1.76 (1H, m), 0.76–0.70 (6H, m).</div></div><div id="sec5_35" class="NLM_sec NLM_sec_level_2"><div id="ac_i58" class="anchor-spacer"></div><h3 class="article-section__title" id="_i58"> Bacterial Strains</h3><div class="NLM_p last">Wild-type strains of <i>A. baumannii</i> (ATCC 19606) and <i>K. pneumoniae</i> (ATCC 43816) were obtained from the American Type Culture Collection. The <i>E. coli</i> Δ<i>tolC</i> knockout was constructed at Achaogen in the DM4100 background (Coli Genetic Stock Center Database #6333). The <i>P. aeruginosa</i> efflux-compromised strain Δ<i>mexAB-oprM</i>::<i>Cm</i> Δ<i>mexEF-oprN</i>::Ω<i>Hg</i> Δ<i>mexCD-oprJ</i>::<i>Gm</i>, also known as PAM1262, was constructed in a PAO1 background as described in ref <a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> and kindly provided by the authors.</div></div><div id="sec5_36" class="NLM_sec NLM_sec_level_2"><div id="ac_i59" class="anchor-spacer"></div><h3 class="article-section__title" id="_i59"> Minimum Inhibitory Concentrations</h3><div class="NLM_p last">MICs were determined by broth microdilution according to guidelines established by the Clinical and Laboratory Standards Institute.<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> Compounds were serially diluted in twofold increments in 100% DMSO and then diluted 10-fold in water. Inocula were prepared from cells streaked onto Mueller-Hinton agar (MHA) and grown at 35 °C overnight. Ten microliters of antibiotic solution was mixed with 90 μL of inoculum in cation-adjusted Mueller-Hinton broth in 96-well assay plates, with a final inoculum of approximately 5 × 10<sup>4</sup> cells/well. When included, PMBN was diluted in broth prior to addition of cells to a final assay concentration of 8 μg/mL. After incubation of assay plates at 35 °C for 18–20 h, the lowest concentration of antibiotic that prevented visible growth was recorded as the MIC. In general, MIC values reported are geometric means of at least two independent measurements rounded to the nearest standard dilution increment.</div></div><div id="sec5_37" class="NLM_sec NLM_sec_level_2"><div id="ac_i60" class="anchor-spacer"></div><h3 class="article-section__title" id="_i60"> Purification of <i>P. aeruginosa</i> BC</h3><div class="NLM_p">The acetyl-CoA carboxylase (<i>accC</i>) gene was cloned from <i>P. aeruginosa</i> (PAO1) genomic DNA using forward primer with underlined <i>Eco</i>RI restriction site 5′-GACT<u class="uu">GAATTC</u>TATGTTGGAAAAAGTGCTGATCG-3′ and reverse primer with <i>Sac</i>I restriction site 5′-GACT<u class="uu">GAGCTC</u>TCAGTGCTTGTCCATACCC-3′ and inserted into the pET47b(+) expression vector (Novagen). This construct has the following N-terminal sequence preceding the WT BC sequence: MAHHHHHHSAALEVLFQGPGYQDPNS. Recombinant N-terminal His-Tagged BC was overexpressed in Rosetta(DE3)pLysS <i>E. coli</i> cells (Novagen). A starter culture of 25 mL of LB broth containing kanamycin (50 μg/mL) and chloramphenicol (34 μg/mL) was inoculated with a single colony of Rosetta(DE3)pLysS <i>E. coli</i> cells containing the His-Tagged BC construct and grown overnight at 37 °C. Twenty milliliters of starter culture was used to inoculate 2 L of LB broth containing kanamycin (50 μg/mL) and chloramphenicol (34 μg/mL). The culture was grown to an OD<sub>600</sub> of 0.5 at 37 °C, at which point 0.5 nM IPTG was added to induce overexpression for 3 h. Cells were then pelleted at 10,200<i>g</i> for 15 min and resuspended in 10 mL of buffer A (20 mM Tris–HCl (pH 7.9), 500 mM NaCl, and 5 mM imidazole). The resulting cell slurry was lysed either by sonication (duty cycle, 30; power, 6, pulse for 15 s; rest for 45 s for 20 rounds on ice) or using an Avestin Emulsiflex. The lysed solution was then centrifuged at 75,000<i>g</i> for at least 1 h. The supernatant was gravity-flowed over a 3 mL Co<sup>2+</sup> His-binding column (Pierce) and incubated with resin for 30 min with rotation at 4 °C. The column was washed with 30 mL of buffer A, and then 10 mL of buffer B (20 mM Tris–HCl (pH 7.9), 500 mM NaCl, and 170 mM imidazole) was added to the column to elute the protein. The eluent was exchanged into buffer C (10 mM HEPES (pH 8.0), 25 mM KCl, and 1 mM TCEP) using 10,000 MWCO filters (Amicon) or dialysis. The buffer-exchanged sample (5 mL) was applied to a 5 mL Q-Sepharose column (GE Healthcare) at 2 mL/min. After a 50 mL wash with buffer C, protein was eluted with a linear gradient over 50 mL of buffer D (10 mM HEPES (pH 8.0), 250 mM KCl, and 1 mM TCEP). Fractions were collected and assessed using SDS-PAGE (12% gel; Bio-Rad). Fractions containing the desired BC at >95% purity were pooled and exchanged into 10 mM HEPES (pH 7.0) and 500 mM KCl using an Amicon 10,000 MWCO filtration unit. A bicinchoninic acid assay (Pierce) was used to determine the final protein concentration.</div><div class="NLM_p last">Previously reported purification procedures were used to purify the material for the X-ray crystallography studies.<a onclick="showRef(event, 'ref18 ref37'); return false;" href="javascript:void(0);" class="ref ref18 ref37">(18,37)</a></div></div><div id="sec5_38" class="NLM_sec NLM_sec_level_2"><div id="ac_i61" class="anchor-spacer"></div><h3 class="article-section__title" id="_i61"> <i>P. aeruginosa</i> BC Enzyme Assay</h3><div class="NLM_p">The IC<sub>50</sub> assay used to measure a compound’s relative inhibition of <i>P. aeruginosa</i> BC was adapted from the coupled assay reported in ref <a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a>. The previous study used the full ACCase multienzyme complex consisting of BC, carboxytransferase, and biotin carboxyl carrier protein, while the assay herein uses only the isolated BC enzyme. Both BC and ACCase catalytic activities produce inorganic phosphate (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00625/suppl_file/jm9b00625_si_001.pdf" class="ext-link">Figure S1</a>), which, in the IC<sub>50</sub> assay, is coupled to the generation of a spectroscopically active product. However, because the <i>k</i><sub>cat</sub> and <i>K</i><sub>M</sub> values for inorganic phosphate generation by ACCase likely differ from BC in isolation, the apparent IC<sub>50</sub> values measured for a given inhibitor of the two different enzyme preparations are not directly comparable.</div><div class="NLM_p last">Initial rates of BC activity were measured at various concentrations of test inhibitor in 96-well plates with a final reaction volume of 300 μL. An initial solution was made such that the final concentration of each component would be 50 mM HEPES (pH 8.0), 100 mM KCl, 1 mM TCEP, 5 mM MgCl<sub>2</sub>, 0.1 mg/mL BSA, 0.005% (v/v) Tween 20, 150 μM 7-methyl-6-thioguanosine (Barry and Associates), 0.25 U/mL purine nucleoside phosphorylase (Sigma), 2 mM ATP, 10 mM sodium carbonate, and 50 mM biotin. This solution was incubated at room temperature for 30 min. Afterward, purified <i>P. aeruginosa</i> BC was added to achieve a final apparent concentration of 15 nM, and 270 μL of the resulting solution was distributed into the wells of a 96-well plate containing 30 μL of a 10× concentration of test inhibitor (previously diluted in <1% DMSO) in 3-fold serial dilutions across the plate row. Reaction progress was monitored by the increase in absorbance at 360 nm. Initial rates at each inhibitor compound were plotted and fitted to a standard IC<sub>50</sub> equation.</div></div><div id="sec5_39" class="NLM_sec NLM_sec_level_2"><div id="ac_i62" class="anchor-spacer"></div><h3 class="article-section__title" id="_i62"> Crystallization and Data Collection</h3><div class="NLM_p last">Crystallization conditions are summarized in <a href="/doi/suppl/10.1021/acs.jmedchem.9b00625/suppl_file/jm9b00625_si_001.pdf" class="ext-link">Table S1</a>. Diffraction data were collected at 100 K at the 24-ID-E beamline equipped with a CCD-based ADSC Quantum 315 detector. The images were processed using the XDS program suite<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> and scaled using the Scala program.<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> Data collection and data processing statistics are given in <a href="/doi/suppl/10.1021/acs.jmedchem.9b00625/suppl_file/jm9b00625_si_001.pdf" class="ext-link">Table S1</a>.</div></div><div id="sec5_40" class="NLM_sec NLM_sec_level_2"><div id="ac_i63" class="anchor-spacer"></div><h3 class="article-section__title" id="_i63"> Crystal Structure Determination</h3><div class="NLM_p">The molecular replacement procedure was applied to locate a solution for the <b>13f</b> data set using the program Phaser.<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> A monomer of <i>E. coli</i> BC (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2V58">2V58</a>) was used as a search model. The positioned MR model was refined using the maximum likelihood refinement in REFMAC<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> with the TLS parameters generated by the TLSMD server.<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> Program Coot<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> was used for model building throughout the refinement. The final model consists of protein residues 1–447 (molecule A), 1–447 (molecule B), two <b>13f</b> molecules, three ethylene glycol molecules, and 432 water molecules. Alternate conformations have been built for protein residues S68, E71, N90, R111, K116, S118, R314, and R423 (mol A) as well as I5, R37, and R314 (mol B).</div><div class="NLM_p last">A monomer of <i>H. influenzae</i> BC (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4RZQ">4RZQ</a>) was used as a search model to locate a solution for the <b>14a</b> data set. The positioned MR solution was refined as described above. The final model consists of protein residues 1–445, one <b>14a</b> molecule, one ethylene glycol molecule, and 30 water molecules.</div></div><div id="sec5_41" class="NLM_sec NLM_sec_level_2"><div id="ac_i64" class="anchor-spacer"></div><h3 class="article-section__title" id="_i64"> Bacterial Cytological Profiling</h3><div class="NLM_p last">Cultures of <i>E. coli</i> (ATCC 25922) were grown in LB at 30 °C to an OD<sub>600</sub> of ∼0.15. Cultures were then split, and each sample was treated with the test compound for 120 min. The cells were stained with FM 4-64 (membranes), DAPI (DNA), and Sytox Green (cell integrity). Cells were then imaged at 100× magnification using an Applied Precision DeltaVision Deconvolution microscope.</div></div><div id="sec5_42" class="NLM_sec NLM_sec_level_2"><div id="ac_i65" class="anchor-spacer"></div><h3 class="article-section__title" id="_i65"> Resistance Frequency</h3><div class="NLM_p last">Spontaneous resistance frequency was measured by plating approximately 10<sup>9</sup> colony-forming units from five independent cultures of the efflux-compromised strain of <i>P. aeruginosa</i> onto MHA containing <b>1</b> at 4-, 8-, and 16-fold above the MIC. The inoculum density of each culture was determined by plating serial dilutions onto drug-free MHA. Plates were incubated at 35 °C in ambient air. After 48 h, visible colonies on antibiotic-containing plates were counted, and several colonies from each plate were archived for MIC analysis and sequencing. The frequency for each culture and antibiotic level was calculated as the number of colonies on the antibiotic-containing plate divided by the inoculum density for that culture. The reported frequency is the mean of the frequencies from the five independent cultures.</div></div><div id="sec5_43" class="NLM_sec NLM_sec_level_2"><div id="ac_i66" class="anchor-spacer"></div><h3 class="article-section__title" id="_i66"> Genome Sequencing</h3><div class="NLM_p last">Whole-genome sequences of the <i>P. aeruginosa</i> efflux-compromised parent and wild-type ancestral strain were determined on a PacBio RSII system at the Arizona Genomics Institute and assembled into single contigs. Both genomes were resequenced by Illumina HiSeq (read length, 2 × 100 base pairs) to resolve minor homopolymer discrepancies, resulting in a high-quality reference genome for the parent strain. Spontaneous mutants were sequenced by Illumina HiSeq. Raw reads were filtered for quality, trimmed, and assembled against the reference genome by Maverix Biomics. Single-nucleotide variants were reported by Maverix Biomics and verified at Achaogen by inspection of the assembled reads for representative mutants of each class. The inversion junction sites in <i>muxA</i> and <i>mexZ</i> were determined by analysis of the assembled reads.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i67"><a href="/doi/suppl/10.1021/acs.jmedchem.9b00625" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_12226" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_12226" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.9b00625" class="ext-link">10.1021/acs.jmedchem.9b00625</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Reaction catalyzed by BC and the actyl-CoA carboxylase multienzyme complex, tests of BC essentiality in <i>P. aeruginosa</i>, and X-ray crystallographic conditions and statistics (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00625/suppl_file/jm9b00625_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00625/suppl_file/jm9b00625_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00625/suppl_file/jm9b00625_si_001.pdf">jm9b00625_si_001.pdf (179.66 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00625/suppl_file/jm9b00625_si_002.csv">jm9b00625_si_002.csv (1.42 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.9b00625" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_23600" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_23600" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Logan D. Andrews</span> - <span class="hlFld-Affiliation affiliation">Former
employee of Achaogen Inc., 1 Tower Place, Suite 400, South San Francisco, California 94080, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-5092-4530" title="Orcid link">http://orcid.org/0000-0002-5092-4530</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#f4989b939086918387b49399959d98da979b99"><span class="__cf_email__" data-cfemail="8fe3e0e8ebfdeaf8fccfe8e2eee6e3a1ece0e2">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ryan T. Cirz</span> - <span class="hlFld-Affiliation affiliation">Former
employee of Achaogen Inc., 1 Tower Place, Suite 400, South San Francisco, California 94080, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#c8bab1a9a6aba1bab288afa5a9a1a4e6aba7a5"><span class="__cf_email__" data-cfemail="186a6179767b716a62587f75797174367b7775">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Timothy R. Kane</span> - <span class="hlFld-Affiliation affiliation">Former
employee of Achaogen Inc., 1 Tower Place, Suite 400, South San Francisco, California 94080, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Paola Dozzo</span> - <span class="hlFld-Affiliation affiliation">Former
employee of Achaogen Inc., 1 Tower Place, Suite 400, South San Francisco, California 94080, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Cat M. Haglund</span> - <span class="hlFld-Affiliation affiliation">Former
employee of Achaogen Inc., 1 Tower Place, Suite 400, South San Francisco, California 94080, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Darin J. Hilderbrandt</span> - <span class="hlFld-Affiliation affiliation">Former
employee of Achaogen Inc., 1 Tower Place, Suite 400, South San Francisco, California 94080, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Martin S. Linsell</span> - <span class="hlFld-Affiliation affiliation">Former
employee of Achaogen Inc., 1 Tower Place, Suite 400, South San Francisco, California 94080, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Timothy Machajewski</span> - <span class="hlFld-Affiliation affiliation">Former
employee of Achaogen Inc., 1 Tower Place, Suite 400, South San Francisco, California 94080, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Glen McEnroe</span> - <span class="hlFld-Affiliation affiliation">Former
employee of Achaogen Inc., 1 Tower Place, Suite 400, South San Francisco, California 94080, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Alisa W. Serio</span> - <span class="hlFld-Affiliation affiliation">Former
employee of Achaogen Inc., 1 Tower Place, Suite 400, South San Francisco, California 94080, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kenneth B. Wlasichuk</span> - <span class="hlFld-Affiliation affiliation">Former
employee of Achaogen Inc., 1 Tower Place, Suite 400, South San Francisco, California 94080, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">David B. Neau</span> - <span class="hlFld-Affiliation affiliation">Northeastern
Collaborative Access Team, Argonne National
Laboratory, Argonne, Illinois 60439, United
States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Svetlana Pakhomova</span> - <span class="hlFld-Affiliation affiliation">Department
of Biological Sciences, Louisiana State
University, Baton
Rouge, Louisiana 70803, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Grover L. Waldrop</span> - <span class="hlFld-Affiliation affiliation">Department
of Biological Sciences, Louisiana State
University, Baton
Rouge, Louisiana 70803, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Marc Sharp</span> - <span class="hlFld-Affiliation affiliation">Linnaeus
Bioscience Inc., 3210
Merryfield Row, San Diego, California 92121, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Joe Pogliano</span> - <span class="hlFld-Affiliation affiliation">Linnaeus
Bioscience Inc., 3210
Merryfield Row, San Diego, California 92121, United States</span>; 
    <span class="hlFld-Affiliation affiliation">University
of California, San Diego, 9500 Gilman Drive, La Jolla, San Diego, California 92093, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-8989-2655" title="Orcid link">http://orcid.org/0000-0001-8989-2655</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Frederick Cohen</span> - <span class="hlFld-Affiliation affiliation">Former
employee of Achaogen Inc., 1 Tower Place, Suite 400, South San Francisco, California 94080, United States</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>All authors have given approval to this manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br><p class="inlineNote">The coordinates for X-ray cocrystal structures of <b>13f</b> and <b>14a</b> have been deposited in the PDB with codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6OI9">6OI9</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6OI8">6OI8</a>, respectively.</p></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i70">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_37764" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_37764" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">Research reported in this publication was supported by the National Institute of Allergy and Infectious Diseases of the National Institutes of Health under award number R21AI113572. We acknowledge Andrew McCandlish for the help in preparing and managing grant applications. This report also includes research conducted at the Northeastern Collaborative Access Team beamlines (24ID-C and 24ID-E at the Advanced Photon Source), which are funded by the National Institute of General Medical Sciences (P41 GM103403) from the National Institutes of Health. The Pilatus 6M detector on 24ID-C is funded by an NIH-ORIP HEI grant (S10 RR029205) and use of the Advanced Photon Source, an Office of Science User Facility operated for the U.S. Department of Energy by Argonne National Laboratory, is under contract no. DE-AC02-06CH11357. The content is solely the responsibility of the authors and does not necessarily represent the official views of the National Institutes of Health. We acknowledge the computational modeling and scientific advice of Erin Bradley and the microbiological assay contributions of Hoan Lee.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">ICU</td><td class="NLM_def"><p class="first last">intensive care unit</p></td></tr><tr><td class="NLM_term">INV</td><td class="NLM_def"><p class="first last">inversion</p></td></tr><tr><td class="NLM_term">MIC</td><td class="NLM_def"><p class="first last">minimum inhibitory concentration</p></td></tr><tr><td class="NLM_term">K</td><td class="NLM_def"><p class="first last">Kelvin</p></td></tr><tr><td class="NLM_term">BC</td><td class="NLM_def"><p class="first last">biotin carboxylase</p></td></tr><tr><td class="NLM_term">IC</td><td class="NLM_def"><p class="first last">inhibitory concentration</p></td></tr><tr><td class="NLM_term">ATCC</td><td class="NLM_def"><p class="first last">American Type Culture Collection</p></td></tr><tr><td class="NLM_term">CLSI</td><td class="NLM_def"><p class="first last">Clinical and Laboratory Standards Institute</p></td></tr><tr><td class="NLM_term">MHA</td><td class="NLM_def"><p class="first last">Mueller-Hinton agar</p></td></tr><tr><td class="NLM_term">PMBN</td><td class="NLM_def"><p class="first last">polymyxin B nonapeptide</p></td></tr><tr><td class="NLM_term">DMSO</td><td class="NLM_def"><p class="first last">dimethyl sulfoxide</p></td></tr><tr><td class="NLM_term">ATP</td><td class="NLM_def"><p class="first last">adenosine triphosphate</p></td></tr><tr><td class="NLM_term">HTS</td><td class="NLM_def"><p class="first last">high-throughput screen</p></td></tr><tr><td class="NLM_term">MR</td><td class="NLM_def"><p class="first last">molecular replacement</p></td></tr><tr><td class="NLM_term">U.S.</td><td class="NLM_def"><p class="first last">United States</p></td></tr><tr><td class="NLM_term">LB</td><td class="NLM_def"><p class="first last">lysogeny broth</p></td></tr><tr><td class="NLM_term">AMPCP</td><td class="NLM_def"><p class="first last">adenosine 5′-(α,β-methylene)diphosphate</p></td></tr><tr><td class="NLM_term">IPTG</td><td class="NLM_def"><p class="first last">isopropyl β-<span class="smallcaps smallerCapital">d</span>-1-thiogalactopyranoside</p></td></tr><tr><td class="NLM_term">TCEP</td><td class="NLM_def"><p class="first last">tris(2-carboxyethyl)phosphine</p></td></tr><tr><td class="NLM_term">MWCO</td><td class="NLM_def"><p class="first last">molecular weight cutoff</p></td></tr><tr><td class="NLM_term">LPS</td><td class="NLM_def"><p class="first last">lipopolysaccharide</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i72">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_35342" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_35342" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 57 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span> <i>Antibiotic Resistance Threats
in the United States, 2013</i>; <span class="NLM_publisher-name">U.S. Centers
for Disease Control and Prevention</span>: [Online] <a href="http://www.cdc.gov/drugresistance/threat-report-2013" class="extLink">http://www.cdc.gov/drugresistance/threat-report-2013</a> (accessed 27 February, 2019).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Antibiotic+Resistance+Threats%0Ain+the+United+States%2C+2013%3B+U.S.+Centers%0Afor+Disease+Control+and+Prevention%3A+%5BOnline%5D+http%3A%2F%2Fwww.cdc.gov%2Fdrugresistance%2Fthreat-report-2013+%28accessed+27+February%2C+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26jtitle%3DAntibiotic%2520Resistance%2520Threats%250Ain%2520the%2520United%2520States%252C%25202013%26pub%3DU.S.%2520Centers%250Afor%2520Disease%2520Control%2520and%2520Prevention" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burnham, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olsen, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kollef, M. H.</span></span> <span> </span><span class="NLM_article-title">Re-Estimating Annual Deaths Due to Multidrug-Resistant Organism Infections</span>. <i>Infect. Control Hosp. Epidemiol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">112</span>– <span class="NLM_lpage">113</span>, <span class="refDoi"> DOI: 10.1017/ice.2018.304</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1017%2Fice.2018.304" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=30463634" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A280%3ADC%252BB3crjsFaisA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2019&pages=112-113&author=J.+P.+Burnhamauthor=M.+A.+Olsenauthor=M.+H.+Kollef&title=Re-Estimating+Annual+Deaths+Due+to+Multidrug-Resistant+Organism+Infections&doi=10.1017%2Fice.2018.304"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Re-estimating annual deaths due to multidrug-resistant organism infections</span></div><div class="casAuthors">Burnham Jason P; Olsen Margaret A; Kollef Marin H</div><div class="citationInfo"><span class="NLM_cas:title">Infection control and hospital epidemiology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">112-113</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRsbGst4dFdhyO2kE7krGOBfW6udTcc2eY8eU4Uob3u97ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3crjsFaisA%253D%253D&md5=1bbd7d59e205bf2987585dc2df677343</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1017%2Fice.2018.304&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1017%252Fice.2018.304%26sid%3Dliteratum%253Aachs%26aulast%3DBurnham%26aufirst%3DJ.%2BP.%26aulast%3DOlsen%26aufirst%3DM.%2BA.%26aulast%3DKollef%26aufirst%3DM.%2BH.%26atitle%3DRe-Estimating%2520Annual%2520Deaths%2520Due%2520to%2520Multidrug-Resistant%2520Organism%2520Infections%26jtitle%3DInfect.%2520Control%2520Hosp.%2520Epidemiol.%26date%3D2019%26volume%3D40%26spage%3D112%26epage%3D113%26doi%3D10.1017%2Fice.2018.304" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cassini, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Högberg, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plachouras, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quattrocchi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoxha, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simonsen, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colomb-Cotinat, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kretzschmar, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Devleesschauwer, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cecchini, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouakrim, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliveira, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Struelens, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suetens, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monnet, D. L.</span></span>; <span class="NLM_contrib-group">Burden of AMR Collaborative Group</span> <span> </span><span class="NLM_article-title">Attributable Deaths and Disability-Adjusted Life-Years Caused by Infections with Antibiotic-Resistant Bacteria in the EU and the European Economic Area in 2015: a Population-Level Modelling Analysis</span>. <i>Lancet Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">56</span>– <span class="NLM_lpage">66</span>, <span class="refDoi"> DOI: 10.1016/S1473-3099(18)30605-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1016%2FS1473-3099%2818%2930605-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=30409683" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A280%3ADC%252BB3cvotFGluw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2019&pages=56-66&author=A.+Cassiniauthor=L.+D.+H%C3%B6gbergauthor=D.+Plachourasauthor=A.+Quattrocchiauthor=A.+Hoxhaauthor=G.+S.+Simonsenauthor=M.+Colomb-Cotinatauthor=M.+E.+Kretzschmarauthor=B.+Devleesschauwerauthor=M.+Cecchiniauthor=D.+A.+Ouakrimauthor=T.+C.+Oliveiraauthor=M.+J.+Struelensauthor=C.+Suetensauthor=D.+L.+Monnetauthor=Burden+of+AMR+Collaborative+Group&title=Attributable+Deaths+and+Disability-Adjusted+Life-Years+Caused+by+Infections+with+Antibiotic-Resistant+Bacteria+in+the+EU+and+the+European+Economic+Area+in+2015%3A+a+Population-Level+Modelling+Analysis&doi=10.1016%2FS1473-3099%2818%2930605-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Attributable deaths and disability-adjusted life-years caused by infections with antibiotic-resistant bacteria in the EU and the European Economic Area in 2015: a population-level modelling analysis</span></div><div class="casAuthors">Cassini Alessandro; Hogberg Liselotte Diaz; Plachouras Diamantis; Quattrocchi Annalisa; Hoxha Ana; Struelens Marc J; Suetens Carl; Monnet Dominique L; Simonsen Gunnar Skov; Colomb-Cotinat Melanie; Kretzschmar Mirjam E; Devleesschauwer Brecht; Cecchini Michele; Ouakrim Driss Ait; Oliveira Tiago Cravo</div><div class="citationInfo"><span class="NLM_cas:title">The Lancet. Infectious diseases</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">56-66</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Infections due to antibiotic-resistant bacteria are threatening modern health care.  However, estimating their incidence, complications, and attributable mortality is challenging.  We aimed to estimate the burden of infections caused by antibiotic-resistant bacteria of public health concern in countries of the EU and European Economic Area (EEA) in 2015, measured in number of cases, attributable deaths, and disability-adjusted life-years (DALYs).  METHODS:  We estimated the incidence of infections with 16 antibiotic resistance-bacterium combinations from European Antimicrobial Resistance Surveillance Network (EARS-Net) 2015 data that was country-corrected for population coverage.  We multiplied the number of bloodstream infections (BSIs) by a conversion factor derived from the European Centre for Disease Prevention and Control point prevalence survey of health-care-associated infections in European acute care hospitals in 2011-12 to estimate the number of non-BSIs.  We developed disease outcome models for five types of infection on the basis of systematic reviews of the literature.  FINDINGS:  From EARS-Net data collected between Jan 1, 2015, and Dec 31, 2015, we estimated 671 689 (95% uncertainty interval [UI] 583 148-763 966) infections with antibiotic-resistant bacteria, of which 63·5% (426 277 of 671 689) were associated with health care.  These infections accounted for an estimated 33 110 (28 480-38 430) attributable deaths and 874 541 (768 837-989 068) DALYs.  The burden for the EU and EEA was highest in infants (aged <1 year) and people aged 65 years or older, had increased since 2007, and was highest in Italy and Greece.  INTERPRETATION:  Our results present the health burden of five types of infection with antibiotic-resistant bacteria expressed, for the first time, in DALYs.  The estimated burden of infections with antibiotic-resistant bacteria in the EU and EEA is substantial compared with that of other infectious diseases, and has increased since 2007.  Our burden estimates provide useful information for public health decision-makers prioritising interventions for infectious diseases.  FUNDING:  European Centre for Disease Prevention and Control.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR76qXWYwPPXq69lAbTj9rGfW6udTcc2eY8eU4Uob3u97ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3cvotFGluw%253D%253D&md5=151619810730b07e3d84692f1d966b9a</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2FS1473-3099%2818%2930605-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1473-3099%252818%252930605-4%26sid%3Dliteratum%253Aachs%26aulast%3DCassini%26aufirst%3DA.%26aulast%3DH%25C3%25B6gberg%26aufirst%3DL.%2BD.%26aulast%3DPlachouras%26aufirst%3DD.%26aulast%3DQuattrocchi%26aufirst%3DA.%26aulast%3DHoxha%26aufirst%3DA.%26aulast%3DSimonsen%26aufirst%3DG.%2BS.%26aulast%3DColomb-Cotinat%26aufirst%3DM.%26aulast%3DKretzschmar%26aufirst%3DM.%2BE.%26aulast%3DDevleesschauwer%26aufirst%3DB.%26aulast%3DCecchini%26aufirst%3DM.%26aulast%3DOuakrim%26aufirst%3DD.%2BA.%26aulast%3DOliveira%26aufirst%3DT.%2BC.%26aulast%3DStruelens%26aufirst%3DM.%2BJ.%26aulast%3DSuetens%26aufirst%3DC.%26aulast%3DMonnet%26aufirst%3DD.%2BL.%26aulast%3D%26atitle%3DAttributable%2520Deaths%2520and%2520Disability-Adjusted%2520Life-Years%2520Caused%2520by%2520Infections%2520with%2520Antibiotic-Resistant%2520Bacteria%2520in%2520the%2520EU%2520and%2520the%2520European%2520Economic%2520Area%2520in%25202015%253A%2520a%2520Population-Level%2520Modelling%2520Analysis%26jtitle%3DLancet%2520Infect.%2520Dis.%26date%3D2019%26volume%3D19%26spage%3D56%26epage%3D66%26doi%3D10.1016%2FS1473-3099%2818%2930605-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Todd, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiehlbauch, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walters, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kallen, A.</span></span> <span> </span><span class="NLM_article-title">Notes From the Field: Pan-Resistant New Delhi Metallo-Beta-Lactamase-Producing <i>Klebsiella pneumoniae</i> - Washoe County, Nevada, 2016</span>. <i>MMWR Morb. Mortal Wkly. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">33</span>, <span class="refDoi"> DOI: 10.15585/mmwr.mm6601a7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.15585%2Fmmwr.mm6601a7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=28081065" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A280%3ADC%252BC1c7ltFSmtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2017&pages=33&author=L.+Chenauthor=R.+Toddauthor=J.+Kiehlbauchauthor=M.+Waltersauthor=A.+Kallen&title=Notes+From+the+Field%3A+Pan-Resistant+New+Delhi+Metallo-Beta-Lactamase-Producing+Klebsiella+pneumoniae+-+Washoe+County%2C+Nevada%2C+2016&doi=10.15585%2Fmmwr.mm6601a7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Notes from the Field: Pan-Resistant New Delhi Metallo-Beta-Lactamase-Producing Klebsiella pneumoniae - Washoe County, Nevada, 2016</span></div><div class="casAuthors">Chen Lei; Todd Randall; Kiehlbauch Julia; Walters Maroya; Kallen Alexander</div><div class="citationInfo"><span class="NLM_cas:title">MMWR. Morbidity and mortality weekly report</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">33</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRXLWGx5FIPGe98rul4nhSdfW6udTcc2eY8eU4Uob3u97ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1c7ltFSmtg%253D%253D&md5=c2f8b366b1b02600014e0c42e8bafaab</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.15585%2Fmmwr.mm6601a7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.15585%252Fmmwr.mm6601a7%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DL.%26aulast%3DTodd%26aufirst%3DR.%26aulast%3DKiehlbauch%26aufirst%3DJ.%26aulast%3DWalters%26aufirst%3DM.%26aulast%3DKallen%26aufirst%3DA.%26atitle%3DNotes%2520From%2520the%2520Field%253A%2520Pan-Resistant%2520New%2520Delhi%2520Metallo-Beta-Lactamase-Producing%2520Klebsiella%2520pneumoniae%2520-%2520Washoe%2520County%252C%2520Nevada%252C%25202016%26jtitle%3DMMWR%2520Morb.%2520Mortal%2520Wkly.%2520Rep.%26date%3D2017%26volume%3D66%26spage%3D33%26doi%3D10.15585%2Fmmwr.mm6601a7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miller, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunham, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mochalkin, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banotai, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bowman, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buist, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunkle, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanna, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harwood, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huband, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karnovsky, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuhn, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Limberakis, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mehrens, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mueller, W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narasimhan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogden, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohren, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prasad, J. V. N. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shelly, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skerlos, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sulavik, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, V. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">VanderRoest, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitton, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stover, C. K.</span></span> <span> </span><span class="NLM_article-title">A Class of Selective Antibacterials Derived From a Protein Kinase Inhibitor Pharmacophore</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>106</i></span>,  <span class="NLM_fpage">1737</span>– <span class="NLM_lpage">1742</span>, <span class="refDoi"> DOI: 10.1073/pnas.0811275106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1073%2Fpnas.0811275106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=19164768" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BD1MXitV2jurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2009&pages=1737-1742&author=J.+R.+Millerauthor=S.+Dunhamauthor=I.+Mochalkinauthor=C.+Banotaiauthor=M.+Bowmanauthor=S.+Buistauthor=B.+Dunkleauthor=D.+Hannaauthor=H.+J.+Harwoodauthor=M.+D.+Hubandauthor=A.+Karnovskyauthor=M.+Kuhnauthor=C.+Limberakisauthor=J.+Y.+Liuauthor=S.+Mehrensauthor=W.+T.+Muellerauthor=L.+Narasimhanauthor=A.+Ogdenauthor=J.+Ohrenauthor=J.+V.+N.+V.+Prasadauthor=J.+A.+Shellyauthor=L.+Skerlosauthor=M.+Sulavikauthor=V.+H.+Thomasauthor=S.+VanderRoestauthor=L.+Wangauthor=Z.+Wangauthor=A.+Whittonauthor=T.+Zhuauthor=C.+K.+Stover&title=A+Class+of+Selective+Antibacterials+Derived+From+a+Protein+Kinase+Inhibitor+Pharmacophore&doi=10.1073%2Fpnas.0811275106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">A class of selective antibacterials derived from a protein kinase inhibitor pharmacophore</span></div><div class="casAuthors">Miller, J. Richard; Dunham, Steve; Mochalkin, Igor; Banotai, Craig; Bowman, Matthew; Buist, Susan; Dunkle, Bill; Hanna, Debra; Harwood, James; Huband, Michael D.; Karnovsky, Alla; Kuhn, Michael; Limberakis, Chris; Liu, Jia Y.; Mehrens, Shawn; Mueller, W. Thomas; Narasimhan, Lakshmi; Ogden, Adam; Ohren, Jeff; Vara Prasad, J. V. N.; Shelly, John A.; Skerlos, Laura; Sulavik, Mark; Thomas, V. Hayden; VanderRoest, Steve; Wang, LiAnn; Wang, Zhigang; Whitton, Amy; Zhu, Tong; Stover, C. Kendall</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1737-1742</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">As the need for novel antibiotic classes to combat bacterial drug resistance increases, the paucity of leads resulting from target-based antibacterial screening of pharmaceutical compd. libraries is of major concern.  One explanation for this lack of success is that antibacterial screening efforts have not leveraged the eukaryotic bias resulting from more extensive chem. efforts targeting eukaryotic gene families such as G protein-coupled receptors and protein kinases.  Consistent with a focus on antibacterial target space resembling these eukaryotic targets, we used whole-cell screening to identify a series of antibacterial pyridopyrimidines derived from a protein kinase inhibitor pharmacophore.  In bacteria, the pyridopyrimidines target the ATP-binding site of biotin carboxylase (BC), which catalyzes the first enzymic step of fatty acid biosynthesis.  These inhibitors are effective in vitro and in vivo against fastidious Gram-neg. pathogens including Haemophilus influenzae.  Although the BC active site has architectural similarity to those of eukaryotic protein kinases, inhibitor binding to the BC ATP-binding site is distinct from the protein kinase-binding mode, such that the inhibitors are selective for bacterial BC.  In summary, the authors have discovered a promising class of potent antibacterials with a previously undescribed mechanism of action.  In consideration of the eukaryotic bias of pharmaceutical libraries, the findings also suggest that pursuit of a novel inhibitor leads for antibacterial targets with active-site structural similarity to known human targets will likely be more fruitful than the traditional focus on unique bacterial target space, particularly when structure-based and computational methodologies are applied to ensure bacterial selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8jpj8EOtRo7Vg90H21EOLACvtfcHk0lhKV_AWvW-M4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXitV2jurk%253D&md5=06758ee5befbe847825ec35adb177006</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0811275106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0811275106%26sid%3Dliteratum%253Aachs%26aulast%3DMiller%26aufirst%3DJ.%2BR.%26aulast%3DDunham%26aufirst%3DS.%26aulast%3DMochalkin%26aufirst%3DI.%26aulast%3DBanotai%26aufirst%3DC.%26aulast%3DBowman%26aufirst%3DM.%26aulast%3DBuist%26aufirst%3DS.%26aulast%3DDunkle%26aufirst%3DB.%26aulast%3DHanna%26aufirst%3DD.%26aulast%3DHarwood%26aufirst%3DH.%2BJ.%26aulast%3DHuband%26aufirst%3DM.%2BD.%26aulast%3DKarnovsky%26aufirst%3DA.%26aulast%3DKuhn%26aufirst%3DM.%26aulast%3DLimberakis%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DJ.%2BY.%26aulast%3DMehrens%26aufirst%3DS.%26aulast%3DMueller%26aufirst%3DW.%2BT.%26aulast%3DNarasimhan%26aufirst%3DL.%26aulast%3DOgden%26aufirst%3DA.%26aulast%3DOhren%26aufirst%3DJ.%26aulast%3DPrasad%26aufirst%3DJ.%2BV.%2BN.%2BV.%26aulast%3DShelly%26aufirst%3DJ.%2BA.%26aulast%3DSkerlos%26aufirst%3DL.%26aulast%3DSulavik%26aufirst%3DM.%26aulast%3DThomas%26aufirst%3DV.%2BH.%26aulast%3DVanderRoest%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DWhitton%26aufirst%3DA.%26aulast%3DZhu%26aufirst%3DT.%26aulast%3DStover%26aufirst%3DC.%2BK.%26atitle%3DA%2520Class%2520of%2520Selective%2520Antibacterials%2520Derived%2520From%2520a%2520Protein%2520Kinase%2520Inhibitor%2520Pharmacophore%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2009%26volume%3D106%26spage%3D1737%26epage%3D1742%26doi%3D10.1073%2Fpnas.0811275106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Payne, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warren, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holmes, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lonsdale, J. T.</span></span> <span> </span><span class="NLM_article-title">Bacterial Fatty-Acid Biosynthesis: A Genomics-Driven Target for Antibacterial Drug Discovery</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">537</span>– <span class="NLM_lpage">544</span>, <span class="refDoi"> DOI: 10.1016/S1359-6446(01)01774-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1016%2FS1359-6446%2801%2901774-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=11369293" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BD3MXjtlCjsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2001&pages=537-544&author=D.+J.+Payneauthor=P.+V.+Warrenauthor=D.+J.+Holmesauthor=Y.+Jiauthor=J.+T.+Lonsdale&title=Bacterial+Fatty-Acid+Biosynthesis%3A+A+Genomics-Driven+Target+for+Antibacterial+Drug+Discovery&doi=10.1016%2FS1359-6446%2801%2901774-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Bacterial fatty-acid biosynthesis: a genomics-driven target for antibacterial drug discovery</span></div><div class="casAuthors">Payne, D. J.; Warren, P. V.; Holmes, D. J.; Ji, Y.; Lonsdale, J. T.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">537-544</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A review with 46 refs. demonstrating how the interplay of genomics, bioinformatics and genomic technologies has enabled an in-depth anal. of the component enzymes of the bacterial fatty-acid biosynthesis pathway as a source of novel antibacterial targets.  This evaluation has revealed that many of the enzymes are potentially selective, broad-spectrum antibacterial targets.  We also illustrate the suitability of some of these targets for HTS.  Furthermore, we discuss how the availability of a robust selectivity assay, mode-of-action assays and numerous crystal structures provide an excellent set of tools with which to initiate integrated programs of research to identify novel antibiotics targeted at these enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXOsty-wRiwLVg90H21EOLACvtfcHk0lhKV_AWvW-M4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXjtlCjsbo%253D&md5=50cbe09ba8e48f9aca6a5b847477f24c</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2FS1359-6446%2801%2901774-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1359-6446%252801%252901774-3%26sid%3Dliteratum%253Aachs%26aulast%3DPayne%26aufirst%3DD.%2BJ.%26aulast%3DWarren%26aufirst%3DP.%2BV.%26aulast%3DHolmes%26aufirst%3DD.%2BJ.%26aulast%3DJi%26aufirst%3DY.%26aulast%3DLonsdale%26aufirst%3DJ.%2BT.%26atitle%3DBacterial%2520Fatty-Acid%2520Biosynthesis%253A%2520A%2520Genomics-Driven%2520Target%2520for%2520Antibacterial%2520Drug%2520Discovery%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2001%26volume%3D6%26spage%3D537%26epage%3D544%26doi%3D10.1016%2FS1359-6446%2801%2901774-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cronan, J. E.,  Jr.</span></span> <span> </span><span class="NLM_article-title">Bacterial Fatty Acid Biosynthesis: Targets for Antibacterial Drug Discovery</span>. <i>Annu. Rev. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">305</span>– <span class="NLM_lpage">332</span>, <span class="refDoi"> DOI: 10.1146/annurev.micro.55.1.305</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1146%2Fannurev.micro.55.1.305" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=11544358" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BD3MXnslKjt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2001&pages=305-332&author=J.+W.+Campbellauthor=J.+E.+Cronan&title=Bacterial+Fatty+Acid+Biosynthesis%3A+Targets+for+Antibacterial+Drug+Discovery&doi=10.1146%2Fannurev.micro.55.1.305"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Bacterial fatty acid biosynthesis: Targets for antibacterial drug discovery</span></div><div class="casAuthors">Campbell, John W.; Cronan, John E., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Microbiology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">305-332</span>CODEN:
                <span class="NLM_cas:coden">ARMIAZ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4227</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  The increase in drug-resistant pathogenic bacteria has created an urgent demand for new antibiotics.  Among the more attractive targets for the development of new antibacterial compds. are the enzymes of fatty acid biosynthesis.  Although a no. of potent inhibitors of microbial fatty acid biosynthesis have been discovered, few of these are clin. useful drugs.  Several of these fatty acid biosynthesis inhibitors have potential as lead compds. in the development of new antibacterials.  This review encompasses the known inhibitors and prospective targets for new antibacterials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpr8j4LVT_lPLVg90H21EOLACvtfcHk0lh8jtbGjanaEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXnslKjt7Y%253D&md5=e2fb75187125997e3d2ccea844114fde</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1146%2Fannurev.micro.55.1.305&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.micro.55.1.305%26sid%3Dliteratum%253Aachs%26aulast%3DCampbell%26aufirst%3DJ.%2BW.%26aulast%3DCronan%26aufirst%3DJ.%2BE.%26atitle%3DBacterial%2520Fatty%2520Acid%2520Biosynthesis%253A%2520Targets%2520for%2520Antibacterial%2520Drug%2520Discovery%26jtitle%3DAnnu.%2520Rev.%2520Microbiol.%26date%3D2001%26volume%3D55%26spage%3D305%26epage%3D332%26doi%3D10.1146%2Fannurev.micro.55.1.305" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Heath, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rock, C. O.</span></span> <span> </span><span class="NLM_article-title">Lipid Biosynthesis as a Target for Antibacterial Agents</span>. <i>Prog. Lipid Res.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">467</span>– <span class="NLM_lpage">497</span>, <span class="refDoi"> DOI: 10.1016/S0163-7827(01)00012-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1016%2FS0163-7827%2801%2900012-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=11591436" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BD3MXovFSju7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2001&pages=467-497&author=R.+J.+Heathauthor=S.+W.+Whiteauthor=C.+O.+Rock&title=Lipid+Biosynthesis+as+a+Target+for+Antibacterial+Agents&doi=10.1016%2FS0163-7827%2801%2900012-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Lipid biosynthesis as a target for antibacterial agents</span></div><div class="casAuthors">Heath, Richard J.; White, Stephen W.; Rock, Charles O.</div><div class="citationInfo"><span class="NLM_cas:title">Progress in Lipid Research</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">467-497</span>CODEN:
                <span class="NLM_cas:coden">PLIRDW</span>;
        ISSN:<span class="NLM_cas:issn">0163-7827</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A review.  Fatty acid biosynthesis, the first stage in membrane lipid biogenesis, is catalyzed in most bacteria by a series of small, sol. proteins that are each encoded by a discrete gene (Fig. 1; Table 1).  This arrangement is termed the type II fatty acid synthase (FAS) system and contrasts sharply with the type I FAS of eukaryotes which is a dimer of a single large, multifunctional polypeptide.  Thus, the bacterial pathway offers several unique sites for selective inhibition by chemotherapeutic agents.  The site of action of isoniazid, used in the treatment of tuberculosis for 50 yr, and the consumer antimicrobial agent triclosan were revealed recently to be the enoyl-ACP reductase of the type II FAS.  The fungal metabolites, cerulenin and thiolactomycin, target the condensing enzymes of the bacterial pathway while the dehydratase/isomerase is inhibited by a synthetic acetylenic substrate analog.  Transfer of fatty acids to the membrane has also been inhibited via interference with the first acyltransferase step, while a new class of drugs targets lipid A synthesis.  This review will summarize the data generated on these inhibitors to date, and examine where addnl. efforts will be required to develop new chemotherapeutics to help combat microbial infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpycZaAyBQafrVg90H21EOLACvtfcHk0lh8jtbGjanaEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXovFSju7s%253D&md5=c8a9b6692527ea14816034ece956cf51</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2FS0163-7827%2801%2900012-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0163-7827%252801%252900012-1%26sid%3Dliteratum%253Aachs%26aulast%3DHeath%26aufirst%3DR.%2BJ.%26aulast%3DWhite%26aufirst%3DS.%2BW.%26aulast%3DRock%26aufirst%3DC.%2BO.%26atitle%3DLipid%2520Biosynthesis%2520as%2520a%2520Target%2520for%2520Antibacterial%2520Agents%26jtitle%3DProg.%2520Lipid%2520Res.%26date%3D2001%26volume%3D40%26spage%3D467%26epage%3D497%26doi%3D10.1016%2FS0163-7827%2801%2900012-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rock, C. O.</span></span> <span> </span><span class="NLM_article-title">Inhibiting Bacterial Fatty Acid Synthesis</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>281</i></span>,  <span class="NLM_fpage">17541</span>– <span class="NLM_lpage">17544</span>, <span class="refDoi"> DOI: 10.1074/jbc.R600004200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1074%2Fjbc.R600004200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=16648134" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BD28XmtFyks7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=281&publication_year=2006&pages=17541-17544&author=Y.-M.+Zhangauthor=S.+W.+Whiteauthor=C.+O.+Rock&title=Inhibiting+Bacterial+Fatty+Acid+Synthesis&doi=10.1074%2Fjbc.R600004200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibiting Bacterial Fatty Acid Synthesis</span></div><div class="casAuthors">Zhang, Yong-Mei; White, Stephen W.; Rock, Charles O.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">281</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">17541-17544</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">A review.  The type II fatty acid synthase consists of a series of individual enzymes, each encoded by a sep. gene, that catalyze discrete steps in chain elongation.  The formation of fatty acids is vital to bacteria, and each of the essential enzymes and their acyl group carriers represent a potential target for the development of novel antibacterial therapeutics.  High resoln. x-ray and/or NMR structures of representative members of every enzyme in the type II pathway are now available, and these structures are a valuable resource to guide antibacterial drug discovery.  The role of each enzyme in regulating pathway activity and the diversity in the components of the pathway in the major human pathogens are important considerations in deciding the most suitable targets for future drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRu1UJVfxfWrVg90H21EOLACvtfcHk0lh8jtbGjanaEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XmtFyks7w%253D&md5=b617963deae704579e4009b65281c5af</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1074%2Fjbc.R600004200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.R600004200%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DY.-M.%26aulast%3DWhite%26aufirst%3DS.%2BW.%26aulast%3DRock%26aufirst%3DC.%2BO.%26atitle%3DInhibiting%2520Bacterial%2520Fatty%2520Acid%2520Synthesis%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2006%26volume%3D281%26spage%3D17541%26epage%3D17544%26doi%3D10.1074%2Fjbc.R600004200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wright, H. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reynolds, K. A.</span></span> <span> </span><span class="NLM_article-title">Antibacterial Targets in Fatty Acid Biosynthesis</span>. <i>Curr. Opin. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">447</span>– <span class="NLM_lpage">453</span>, <span class="refDoi"> DOI: 10.1016/j.mib.2007.07.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1016%2Fj.mib.2007.07.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=17707686" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BD2sXht1yltb%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2007&pages=447-453&author=H.+T.+Wrightauthor=K.+A.+Reynolds&title=Antibacterial+Targets+in+Fatty+Acid+Biosynthesis&doi=10.1016%2Fj.mib.2007.07.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Antibacterial targets in fatty acid biosynthesis</span></div><div class="casAuthors">Wright, H. Tonie; Reynolds, Kevin A.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Microbiology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">447-453</span>CODEN:
                <span class="NLM_cas:coden">COMIF7</span>;
        ISSN:<span class="NLM_cas:issn">1369-5274</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The fatty acid biosynthesis pathway is an attractive but still largely unexploited target for the development of new antibacterial agents.  The extended use of the antituberculosis drug isoniazid and the antiseptic triclosan, which are inhibitors of fatty acid biosynthesis, validates this pathway as a target for antibacterial development.  Differences in subcellular organization of the bacterial and eukaryotic multi-enzyme fatty acid synthase systems offer the prospect of inhibitors with host vs. target specificity.  Platensimycin, platencin, and phomallenic acids, newly discovered natural product inhibitors of the condensation steps in fatty acid biosynthesis, represent new classes of compds. with antibiotic potential.  An almost complete catalog of crystal structures for the enzymes of the type II fatty acid biosynthesis pathway can now be exploited in the rational design of new inhibitors, as well as the recently published crystal structures of type I FAS complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7buYo89BhsrVg90H21EOLACvtfcHk0lh8jtbGjanaEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXht1yltb%252FK&md5=fd53c737e87aa6c1694858c19e46d6f1</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.mib.2007.07.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.mib.2007.07.001%26sid%3Dliteratum%253Aachs%26aulast%3DWright%26aufirst%3DH.%2BT.%26aulast%3DReynolds%26aufirst%3DK.%2BA.%26atitle%3DAntibacterial%2520Targets%2520in%2520Fatty%2520Acid%2520Biosynthesis%26jtitle%3DCurr.%2520Opin.%2520Microbiol.%26date%3D2007%26volume%3D10%26spage%3D447%26epage%3D453%26doi%3D10.1016%2Fj.mib.2007.07.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Parsons, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rock, C. O.</span></span> <span> </span><span class="NLM_article-title">Is Bacterial Fatty Acid Synthesis a Valid Target for Antibacterial Drug Discovery?</span>. <i>Curr. Opin. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">544</span>– <span class="NLM_lpage">549</span>, <span class="refDoi"> DOI: 10.1016/j.mib.2011.07.029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1016%2Fj.mib.2011.07.029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=21862391" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlCntrjM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2011&pages=544-549&author=J.+B.+Parsonsauthor=C.+O.+Rock&title=Is+Bacterial+Fatty+Acid+Synthesis+a+Valid+Target+for+Antibacterial+Drug+Discovery%3F&doi=10.1016%2Fj.mib.2011.07.029"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Is bacterial fatty acid synthesis a valid target for antibacterial drug discovery?</span></div><div class="casAuthors">Parsons, Joshua B.; Rock, Charles O.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Microbiology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">544-549</span>CODEN:
                <span class="NLM_cas:coden">COMIF7</span>;
        ISSN:<span class="NLM_cas:issn">1369-5274</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The emergence of resistance against most current drugs emphasizes the need to develop new approaches to control bacterial pathogens, particularly Staphylococcus aureus.  Bacterial fatty acid synthesis is one such target that is being actively pursued by several research groups to develop anti-Staphylococcal agents.  Recently, the wisdom of this approach has been challenged based on the ability of a Gram-pos. bacterium to incorporate extracellular fatty acids and thus circumvent the inhibition of de novo fatty acid synthesis.  The generality of this conclusion has been challenged, and there is enough diversity in the enzymes and regulation of fatty acid synthesis in bacteria to conclude that there is not a single organism that can be considered typical and representative of bacteria as a whole.  We are left without a clear resoln. to this ongoing debate and await new basic research to define the pathways for fatty acid uptake and that det. the biochem. and genetic mechanisms for the regulation of fatty acid synthesis in Gram-pos. bacteria.  These crucial expts. will det. whether diversity in the control of this important pathway accounts for the apparently different responses of Gram-pos. bacteria to the inhibition of de novo fatty acid synthesis in presence of extracellular fatty acid supplements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7o8Zc8puOBrVg90H21EOLACvtfcHk0lg1jjMzxPRW4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlCntrjM&md5=2c5a26d05b1174826232917568fdde17</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.mib.2011.07.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.mib.2011.07.029%26sid%3Dliteratum%253Aachs%26aulast%3DParsons%26aufirst%3DJ.%2BB.%26aulast%3DRock%26aufirst%3DC.%2BO.%26atitle%3DIs%2520Bacterial%2520Fatty%2520Acid%2520Synthesis%2520a%2520Valid%2520Target%2520for%2520Antibacterial%2520Drug%2520Discovery%253F%26jtitle%3DCurr.%2520Opin.%2520Microbiol.%26date%3D2011%26volume%3D14%26spage%3D544%26epage%3D549%26doi%3D10.1016%2Fj.mib.2011.07.029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cronan, J. E.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldrop, G. L.</span></span> <span> </span><span class="NLM_article-title">Multi-Subunit Acetyl-CoA Carboxylases</span>. <i>Prog. Lipid Res.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">407</span>– <span class="NLM_lpage">435</span>, <span class="refDoi"> DOI: 10.1016/S0163-7827(02)00007-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1016%2FS0163-7827%2802%2900007-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=12121720" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BD38Xltleku7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2002&pages=407-435&author=J.+E.+Cronanauthor=G.+L.+Waldrop&title=Multi-Subunit+Acetyl-CoA+Carboxylases&doi=10.1016%2FS0163-7827%2802%2900007-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Multi-subunit acetyl-CoA carboxylases</span></div><div class="casAuthors">Cronan, John E.; Waldrop, Grover L.</div><div class="citationInfo"><span class="NLM_cas:title">Progress in Lipid Research</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">407-435</span>CODEN:
                <span class="NLM_cas:coden">PLIRDW</span>;
        ISSN:<span class="NLM_cas:issn">0163-7827</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A review with 118 refs.  Acetyl-CoA carboxylase (ACC) catalyzes the 1st committed step of fatty acid synthesis, the carboxylation of acetyl-CoA to malonyl-CoA.  Two phys. distinct types of enzymes are found in nature.  Bacterial and most plant chloroplasts contain a multi-subunit ACC (MS-ACC) enzyme that is readily dissocd. into its component proteins.  Mammals, fungi, and plant cytosols contain the 2nd type of ACC, a single large multifunctional polypeptide.  This review focuses on the structures, regulation, and enzymic mechanisms of the bacterial and plant MS-ACCs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEQxH8bu-Sf7Vg90H21EOLACvtfcHk0lg1jjMzxPRW4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xltleku7k%253D&md5=621ebd71847b6fca01114df5e72f6a1a</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2FS0163-7827%2802%2900007-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0163-7827%252802%252900007-3%26sid%3Dliteratum%253Aachs%26aulast%3DCronan%26aufirst%3DJ.%2BE.%26aulast%3DWaldrop%26aufirst%3DG.%2BL.%26atitle%3DMulti-Subunit%2520Acetyl-CoA%2520Carboxylases%26jtitle%3DProg.%2520Lipid%2520Res.%26date%3D2002%26volume%3D41%26spage%3D407%26epage%3D435%26doi%3D10.1016%2FS0163-7827%2802%2900007-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mochalkin, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narasimhan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thanabal, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erdman, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prasad, J. V. N. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lightle, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huband, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stover, C. K.</span></span> <span> </span><span class="NLM_article-title">Discovery of Antibacterial Biotin Carboxylase Inhibitors by Virtual Screening and Fragment-Based Approaches</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">473</span>– <span class="NLM_lpage">483</span>, <span class="refDoi"> DOI: 10.1021/cb9000102</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb9000102" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BD1MXmtVGgtbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2009&pages=473-483&author=I.+Mochalkinauthor=J.+R.+Millerauthor=L.+Narasimhanauthor=V.+Thanabalauthor=P.+Erdmanauthor=P.+B.+Coxauthor=J.+V.+N.+V.+Prasadauthor=S.+Lightleauthor=M.+D.+Hubandauthor=C.+K.+Stover&title=Discovery+of+Antibacterial+Biotin+Carboxylase+Inhibitors+by+Virtual+Screening+and+Fragment-Based+Approaches&doi=10.1021%2Fcb9000102"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Antibacterial Biotin Carboxylase Inhibitors by Virtual Screening and Fragment-Based Approaches</span></div><div class="casAuthors">Mochalkin, Igor; Miller, J. Richard; Narasimhan, Lakshmi; Thanabal, Venkataraman; Erdman, Paul; Cox, Philip B.; Prasad, J. V. N. Vara; Lightle, Sandra; Huband, Michael D.; Stover, C. Kendall</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">473-483</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">As part of an effort to inhibit bacterial fatty acid biosynthesis through the recently validated target biotin carboxylase, a unique combination of two emergent lead discovery strategies was employed.  Both de novo fragment-based drug discovery and virtual screening, which employs 3D shape and electrostatic property similarity searching were used.  A collection of unbiased low-mol.-wt. mols. was screened and a structurally diverse collection of weak-binding but ligand-efficient fragments were identified as potential building blocks for biotin carboxylase ATP-competitive inhibitors.  Through iterative cycles of structure-based drug design relying on successive fragment co-structures, the potency of the initial hits was improved by up to 3000-fold while maintaining their ligand-efficiency and desirable physicochem. properties.  In one example, hit-expansion efforts resulted in a series of amino-oxazoles with antibacterial activity.  These results successfully demonstrate that virtual screening approaches can substantially augment fragment-based screening approaches to identify novel antibacterial agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpX9pCqvTXGXLVg90H21EOLACvtfcHk0lg1jjMzxPRW4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXmtVGgtbc%253D&md5=4f7e3886dd925d5b8c402e9072f2efcb</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Fcb9000102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb9000102%26sid%3Dliteratum%253Aachs%26aulast%3DMochalkin%26aufirst%3DI.%26aulast%3DMiller%26aufirst%3DJ.%2BR.%26aulast%3DNarasimhan%26aufirst%3DL.%26aulast%3DThanabal%26aufirst%3DV.%26aulast%3DErdman%26aufirst%3DP.%26aulast%3DCox%26aufirst%3DP.%2BB.%26aulast%3DPrasad%26aufirst%3DJ.%2BV.%2BN.%2BV.%26aulast%3DLightle%26aufirst%3DS.%26aulast%3DHuband%26aufirst%3DM.%2BD.%26aulast%3DStover%26aufirst%3DC.%2BK.%26atitle%3DDiscovery%2520of%2520Antibacterial%2520Biotin%2520Carboxylase%2520Inhibitors%2520by%2520Virtual%2520Screening%2520and%2520Fragment-Based%2520Approaches%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2009%26volume%3D4%26spage%3D473%26epage%3D483%26doi%3D10.1021%2Fcb9000102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shipps, G. W.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawahata, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soucy, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soriano, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orth, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mann, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Black, T.</span></span> <span> </span><span class="NLM_article-title">Discovery and Optimization of Antibacterial AccC Inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">6507</span>– <span class="NLM_lpage">6514</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2009.10.057</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1016%2Fj.bmcl.2009.10.057" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=19875284" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlGisLvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=6507-6514&author=C.+C.+Chengauthor=G.+W.+Shippsauthor=Z.+Yangauthor=B.+Sunauthor=N.+Kawahataauthor=K.+A.+Soucyauthor=A.+Sorianoauthor=P.+Orthauthor=L.+Xiaoauthor=P.+Mannauthor=T.+Black&title=Discovery+and+Optimization+of+Antibacterial+AccC+Inhibitors&doi=10.1016%2Fj.bmcl.2009.10.057"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and optimization of antibacterial AccC inhibitors</span></div><div class="casAuthors">Cheng, Cliff C.; Shipps, Gerald W.; Yang, Zhiwei; Sun, Binyuan; Kawahata, Noriyuki; Soucy, Kyle A.; Soriano, Aileen; Orth, Peter; Xiao, Li; Mann, Paul; Black, Todd</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">6507-6514</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Benzimidazolecarboxamides and imidazopyridinecarboxamides such as I (R first defined; R1 = H; X = N) and I (R second defined; R1 = O2N; X = CH) are prepd. by soln. and solid-phase methods as inhibitors of biotin carboxylase (AccC) for potential use as antibacterial agents.  For example, I (R first defined; R1 = H; X = N) and I (R second defined; R1 = O2N; X = CH) are potent and selective inhibitors of bacterial AccC with IC50 values of 20 nM and 10 nM, resp., and MIC values of 0.8 μg/mL against a sensitized strain of Escherichia coli (HS294 E. coli).  The structures of two benzimidazolecarboxamides bound to biotin carboxylase are detd. by X-ray crystallog.; the structure of an imidazopyridinecarboxamide bound to the active site of biotin carboxylase is detd. by computer-aided modeling calcns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgL8pY9SdU07Vg90H21EOLACvtfcHk0ljKfIw_AW_7HA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlGisLvJ&md5=e811f3f617b9125bd852a4d65ff6e6c2</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2009.10.057&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2009.10.057%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DC.%2BC.%26aulast%3DShipps%26aufirst%3DG.%2BW.%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DSun%26aufirst%3DB.%26aulast%3DKawahata%26aufirst%3DN.%26aulast%3DSoucy%26aufirst%3DK.%2BA.%26aulast%3DSoriano%26aufirst%3DA.%26aulast%3DOrth%26aufirst%3DP.%26aulast%3DXiao%26aufirst%3DL.%26aulast%3DMann%26aufirst%3DP.%26aulast%3DBlack%26aufirst%3DT.%26atitle%3DDiscovery%2520and%2520Optimization%2520of%2520Antibacterial%2520AccC%2520Inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2009%26volume%3D19%26spage%3D6507%26epage%3D6514%26doi%3D10.1016%2Fj.bmcl.2009.10.057" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ogden, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuhn, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dority, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buist, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mehrens, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanna, D.</span></span> <span> </span><span class="NLM_article-title">Evaluation of Pharmacokinetic/Pharmacodynamic Relationships of PD-0162819, a Biotin Carboxylase Inhibitor Representing a New Class of Antibacterial Compounds, Using <i>In Vitro</i> Infection Models</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">124</span>– <span class="NLM_lpage">129</span>, <span class="refDoi"> DOI: 10.1128/AAC.00090-11</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1128%2FAAC.00090-11" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=21986824" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BC38Xjt1Gksw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2012&pages=124-129&author=A.+Ogdenauthor=M.+Kuhnauthor=M.+Dorityauthor=S.+Buistauthor=S.+Mehrensauthor=T.+Zhuauthor=D.+Xiaoauthor=J.+R.+Millerauthor=D.+Hanna&title=Evaluation+of+Pharmacokinetic%2FPharmacodynamic+Relationships+of+PD-0162819%2C+a+Biotin+Carboxylase+Inhibitor+Representing+a+New+Class+of+Antibacterial+Compounds%2C+Using+In+Vitro+Infection+Models&doi=10.1128%2FAAC.00090-11"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluation of pharmacokinetic/pharmacodynamic relationships of PD-0162819, a biotin carboxylase inhibitor representing a new class of antibacterial compounds, using in vitro infection models</span></div><div class="casAuthors">Ogden, Adam; Kuhn, Michael; Dority, Michael; Buist, Susan; Mehrens, Shawn; Zhu, Tong; Xiao, Deqing; Miller, J. Richard; Hanna, Debra</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">124-129</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The present study investigated the pharmacokinetic/pharmacodynamic (PK/PD) relationships of a prototype biotin carboxylase (BC) inhibitor, PD-0162819, against Haemophilus influenzae 3113 in static concn. time-kill (SCTK) and one-compartment chemostat in vitro infection models.  H. influenzae 3113 was exposed to PD-0162819 concns. of 0.5 to 16 × the MIC (MIC = 0.125 μg/mL) and area-under-the-curve (AUC)/MIC ratios of 1 to 1,100 in SCTK and chemostat expts., resp.  Serial samples were collected over 24 h.  For efficacy driver anal., a sigmoid max.-effect (Emax) model was fitted to the relationship between bacterial d. changes over 24 h and corresponding PK/PD indexes.  A semi-mechanistic PK/PD model describing the time course of bacterial growth and death was developed.  The AUC/MIC ratio best explained efficacy (r2 = 0.95) compared to the peak drug concn. (Cmax)/MIC ratio (r2 = 0.76) and time above the MIC (T>MIC) (r2 = 0.88).  Static effects and 99.9% killing were achieved at AUC/MIC values of 500 and 600, resp.  For time course anal., the net bacterial growth rate const., max. bacterial d., and max. kill rate const. were similar in SCTK and chemostat studies, but PD-0162819 was more potent in SCTK than in the chemostat (50% effective concn. [EC50] = 0.046 vs. 0.34 μg/mL).  In conclusion, basic PK/PD relationships for PD-0162819 were established using in vitro dynamic systems.  Although the bacterial growth parameters and max. drug effects were similar in SCTK and the chemostat system, PD-0162819 appeared to be more potent in SCTK, illustrating the importance of understanding the differences in preclin. models.  Addnl. studies are needed to det. the in vivo relevance of these results.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqpvJt2Z1rgYLVg90H21EOLACvtfcHk0ljKfIw_AW_7HA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xjt1Gksw%253D%253D&md5=31471262f87a14100e07c9024cb70b0f</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1128%2FAAC.00090-11&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.00090-11%26sid%3Dliteratum%253Aachs%26aulast%3DOgden%26aufirst%3DA.%26aulast%3DKuhn%26aufirst%3DM.%26aulast%3DDority%26aufirst%3DM.%26aulast%3DBuist%26aufirst%3DS.%26aulast%3DMehrens%26aufirst%3DS.%26aulast%3DZhu%26aufirst%3DT.%26aulast%3DXiao%26aufirst%3DD.%26aulast%3DMiller%26aufirst%3DJ.%2BR.%26aulast%3DHanna%26aufirst%3DD.%26atitle%3DEvaluation%2520of%2520Pharmacokinetic%252FPharmacodynamic%2520Relationships%2520of%2520PD-0162819%252C%2520a%2520Biotin%2520Carboxylase%2520Inhibitor%2520Representing%2520a%2520New%2520Class%2520of%2520Antibacterial%2520Compounds%252C%2520Using%2520In%2520Vitro%2520Infection%2520Models%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2012%26volume%3D56%26spage%3D124%26epage%3D129%26doi%3D10.1128%2FAAC.00090-11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Silvers, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robertson, G. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldrop, G. L.</span></span> <span> </span><span class="NLM_article-title">Design, Synthesis, and Antibacterial Properties of Dual-Ligand Inhibitors of Acetyl-CoA Carboxylase</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">8947</span>– <span class="NLM_lpage">8959</span>, <span class="refDoi"> DOI: 10.1021/jm501082n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501082n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1OktLjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=8947-8959&author=M.+A.+Silversauthor=G.+T.+Robertsonauthor=C.+M.+Taylorauthor=G.+L.+Waldrop&title=Design%2C+Synthesis%2C+and+Antibacterial+Properties+of+Dual-Ligand+Inhibitors+of+Acetyl-CoA+Carboxylase&doi=10.1021%2Fjm501082n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Design, Synthesis, and Antibacterial Properties of Dual-Ligand Inhibitors of Acetyl-CoA Carboxylase</span></div><div class="casAuthors">Silvers, Molly A.; Robertson, Gregory T.; Taylor, Carol M.; Waldrop, Grover L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">8947-8959</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">There is an urgent demand for the development of new antibiotics due to the increase in drug-resistant pathogenic bacteria.  A novel target is the multifunctional enzyme acetyl-CoA carboxylase (ACC), which catalyzes the first committed step in fatty acid synthesis and consists of two enzymes: biotin carboxylase and carboxyltransferase.  Covalently attaching known inhibitors against these enzymes with satd. hydrocarbon linkers of different lengths generated dual-ligand inhibitors.  Kinetic results revealed that the dual-ligands inhibited the ACC complex in the nanomolar range.  Microbiol. assays showed that the dual-ligand with a 15-carbon linker did not exhibit any antibacterial activity, while the dual-ligand with a 7-carbon linker displayed broad-spectrum antibacterial activity as well as a decreased susceptibility in the development of bacterial resistance.  These results suggest that the properties of the linker are vital for antibacterial activity and show how inhibiting two different enzymes with the same compd. increases the overall potency while also impeding the development of resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYCRRNvvd6ULVg90H21EOLACvtfcHk0ljKfIw_AW_7HA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1OktLjL&md5=4e328b5949c3283f33b25baf6f8212f4</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fjm501082n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501082n%26sid%3Dliteratum%253Aachs%26aulast%3DSilvers%26aufirst%3DM.%2BA.%26aulast%3DRobertson%26aufirst%3DG.%2BT.%26aulast%3DTaylor%26aufirst%3DC.%2BM.%26aulast%3DWaldrop%26aufirst%3DG.%2BL.%26atitle%3DDesign%252C%2520Synthesis%252C%2520and%2520Antibacterial%2520Properties%2520of%2520Dual-Ligand%2520Inhibitors%2520of%2520Acetyl-CoA%2520Carboxylase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D8947%26epage%3D8959%26doi%3D10.1021%2Fjm501082n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wallace, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bowlin, N. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mills, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saenkham, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwasny, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Opperman, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rock, C. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bowlin, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moir, D. T.</span></span> <span> </span><span class="NLM_article-title">Discovery of Bacterial Fatty Acid Synthase Type II Inhibitors Using a Novel Cellular Bioluminescent Reporter Assay</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">5775</span>– <span class="NLM_lpage">5787</span>, <span class="refDoi"> DOI: 10.1128/AAC.00686-15</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1128%2FAAC.00686-15" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=26169404" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlOltLzN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2015&pages=5775-5787&author=J.+Wallaceauthor=N.+O.+Bowlinauthor=D.+M.+Millsauthor=P.+Saenkhamauthor=S.+M.+Kwasnyauthor=T.+J.+Oppermanauthor=J.+D.+Williamsauthor=C.+O.+Rockauthor=T.+L.+Bowlinauthor=D.+T.+Moir&title=Discovery+of+Bacterial+Fatty+Acid+Synthase+Type+II+Inhibitors+Using+a+Novel+Cellular+Bioluminescent+Reporter+Assay&doi=10.1128%2FAAC.00686-15"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of bacterial fatty acid synthase type II inhibitors using a novel cellular bioluminescent reporter assay</span></div><div class="casAuthors">Wallace, Joselynn; Bowlin, Nicholas O.; Mills, Debra M.; Saenkham, Panatda; Kwasny, Steven M.; Opperman, Timothy J.; Williams, John D.; Rock, Charles O.; Bowlin, Terry L.; Moir, Donald T.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">5775-5787</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">1098-6596</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Novel, cellular, gain-of-signal, bioluminescent reporter assays for fatty acid synthesis type II (FASII) inhibitors were constructed in an efflux-deficient strain of Pseudomonas aeruginosa and based on the discovery that FASII genes in P. aeruginosa are coordinately upregulated in response to pathway disruption.  A screen of 115,000 compds. identified a series of sulfonamidobenzamide (SABA) analogs, which generated strong luminescent signals in two FASII reporter strains but not in four control reporter strains designed to respond to inhibitors of pathways other than FASII.  The SABA analogs selectively inhibited lipid biosynthesis in P. aeruginosa and exhibited minimal cytotoxicity to mammalian cells (50% cytotoxic concn. [CC50] ≥ 80 μM).  The most potent SABA analogs had MICs of 0.5 to 7.0 μM (0.2 to 3.0 μg/mL) against an efflux-deficient Escherichia coli (ΔtolC) strain but had no detectable MIC against efflux-proficient E. coli or against P. aeruginosa (efflux deficient or proficient).  Genetic, mol. genetic, and biochem. studies revealed that SABA analogs target the enzyme (AccC) catalyzing the biotin carboxylase half-reaction of the acetyl CoA (acetyl-CoA) carboxylase step in the initiation phase of FASII in E. coli and P. aeruginosa.  These results validate the capability and the sensitivity of this novel bioluminescent reporter screen to identify inhibitors of E. coli and P. aeruginosa FASII.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbxuGbUkG3vrVg90H21EOLACvtfcHk0lgO88TgFoq9cA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlOltLzN&md5=80cbafab9c43b2930bbd3bf0b976dddc</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1128%2FAAC.00686-15&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.00686-15%26sid%3Dliteratum%253Aachs%26aulast%3DWallace%26aufirst%3DJ.%26aulast%3DBowlin%26aufirst%3DN.%2BO.%26aulast%3DMills%26aufirst%3DD.%2BM.%26aulast%3DSaenkham%26aufirst%3DP.%26aulast%3DKwasny%26aufirst%3DS.%2BM.%26aulast%3DOpperman%26aufirst%3DT.%2BJ.%26aulast%3DWilliams%26aufirst%3DJ.%2BD.%26aulast%3DRock%26aufirst%3DC.%2BO.%26aulast%3DBowlin%26aufirst%3DT.%2BL.%26aulast%3DMoir%26aufirst%3DD.%2BT.%26atitle%3DDiscovery%2520of%2520Bacterial%2520Fatty%2520Acid%2520Synthase%2520Type%2520II%2520Inhibitors%2520Using%2520a%2520Novel%2520Cellular%2520Bioluminescent%2520Reporter%2520Assay%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2015%26volume%3D59%26spage%3D5775%26epage%3D5787%26doi%3D10.1128%2FAAC.00686-15" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mochalkin, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evdokimov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lightle, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stover, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldrop, G. L.</span></span> <span> </span><span class="NLM_article-title">Structural Evidence for Substrate-Induced Synergism and Half-Sites Reactivity in Biotin Carboxylase</span>. <i>Protein Sci.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">1706</span>– <span class="NLM_lpage">1718</span>, <span class="refDoi"> DOI: 10.1110/ps.035584.108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1110%2Fps.035584.108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=18725455" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BD1cXht1Cit7zI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2008&pages=1706-1718&author=I.+Mochalkinauthor=J.+R.+Millerauthor=A.+Evdokimovauthor=S.+Lightleauthor=C.+Yanauthor=C.+K.+Stoverauthor=G.+L.+Waldrop&title=Structural+Evidence+for+Substrate-Induced+Synergism+and+Half-Sites+Reactivity+in+Biotin+Carboxylase&doi=10.1110%2Fps.035584.108"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Structural evidence for substrate-induced synergism and half-sites reactivity in biotin carboxylase</span></div><div class="casAuthors">Mochalkin, Igor; Miller, J. Richard; Evdokimov, Artem; Lightle, Sandra; Yan, Chunhong; Stover, Charles Ken; Waldrop, Grover L.</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1706-1718</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">0961-8368</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Bacterial acetyl-CoA carboxylase is a multifunctional biotin-dependent enzyme that consists of three sep. proteins: biotin carboxylase (BC), biotin carboxyl carrier protein (BCCP), and carboxyltransferase (CT).  Acetyl-CoA carboxylase is a potentially attractive target for novel antibiotics because it catalyzes the first committed step in fatty acid biosynthesis.  In the first half-reaction, BC catalyzes the ATP-dependent carboxylation of BCCP.  In the second half-reaction, the carboxyl group is transferred from carboxybiotinylated BCCP to acetyl-CoA to produce malonyl-CoA.  A series of structures of BC from several bacteria crystd. in the presence of various ATP analogs is described that addresses three major questions concerning the catalytic mechanism.  The structure of BC bound to AMPPNP and the two catalytically essential magnesium ions resolves inconsistencies between the kinetics of active-site BC mutants and previously reported BC structures.  Another structure of AMPPNP bound to BC shows the polyphosphate chain folded back on itself, and not in the correct (i.e., extended) conformation for catalysis.  This provides the first structural evidence for the hypothesis of substrate-induced synergism, which posits that ATP binds nonproductively to BC in the absence of biotin.  The BC homodimer has been proposed to exhibit half-sites reactivity where the active sites alternate or "flip-flop" their catalytic cycles.  A crystal structure of BC showed the ATP analog AMPPCF2P bound to one subunit while the other subunit was unliganded.  The liganded subunit was in the closed or catalytic conformation while the unliganded subunit was in the open conformation.  This provides the first structural evidence for half-sites reactivity in BC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2hQakHeDxTLVg90H21EOLACvtfcHk0lgO88TgFoq9cA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXht1Cit7zI&md5=93758000a39e0bdc6cb04f479529d58f</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1110%2Fps.035584.108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1110%252Fps.035584.108%26sid%3Dliteratum%253Aachs%26aulast%3DMochalkin%26aufirst%3DI.%26aulast%3DMiller%26aufirst%3DJ.%2BR.%26aulast%3DEvdokimov%26aufirst%3DA.%26aulast%3DLightle%26aufirst%3DS.%26aulast%3DYan%26aufirst%3DC.%26aulast%3DStover%26aufirst%3DC.%2BK.%26aulast%3DWaldrop%26aufirst%3DG.%2BL.%26atitle%3DStructural%2520Evidence%2520for%2520Substrate-Induced%2520Synergism%2520and%2520Half-Sites%2520Reactivity%2520in%2520Biotin%2520Carboxylase%26jtitle%3DProtein%2520Sci.%26date%3D2008%26volume%3D17%26spage%3D1706%26epage%3D1718%26doi%3D10.1110%2Fps.035584.108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nikaido, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaara, M.</span></span> <span> </span><span class="NLM_article-title">Molecular Basis of Bacterial Outer Membrane Permeability</span>. <i>Microbiol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">1985</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">32</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1128%2FMMBR.49.1.1-32.1985" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=2580220" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADyaL2MXhvVWqur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=1985&pages=1-32&author=H.+Nikaidoauthor=M.+Vaara&title=Molecular+Basis+of+Bacterial+Outer+Membrane+Permeability"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular basis of bacterial outer membrane permeability</span></div><div class="casAuthors">Nikaido, Hiroshi; Vaara, Marti</div><div class="citationInfo"><span class="NLM_cas:title">Microbiological Reviews</span>
        (<span class="NLM_cas:date">1985</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-32</span>CODEN:
                <span class="NLM_cas:coden">MBRED3</span>;
        ISSN:<span class="NLM_cas:issn">0146-0749</span>.
    </div><div class="casAbstract">A review with 344 refs. on the chem. compn. of outer membrane, porins as nonspecific protein channels, specific diffusion channels, lipid bilayer as a diffusion barrier, and fusion sites between outer and inner membranes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqeq6AtZlquPrVg90H21EOLACvtfcHk0lgO88TgFoq9cA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2MXhvVWqur4%253D&md5=e0d9879bb50378f4add4df159aa2b5a2</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1128%2FMMBR.49.1.1-32.1985&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMMBR.49.1.1-32.1985%26sid%3Dliteratum%253Aachs%26aulast%3DNikaido%26aufirst%3DH.%26aulast%3DVaara%26aufirst%3DM.%26atitle%3DMolecular%2520Basis%2520of%2520Bacterial%2520Outer%2520Membrane%2520Permeability%26jtitle%3DMicrobiol.%2520Rev.%26date%3D1985%26volume%3D49%26spage%3D1%26epage%3D32" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nikaido, H.</span></span> <span> </span><span class="NLM_article-title">Prevention of Drug Access to Bacterial Targets: Permeability Barriers and Active Efflux</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>264</i></span>,  <span class="NLM_fpage">382</span>– <span class="NLM_lpage">388</span>, <span class="refDoi"> DOI: 10.1126/science.8153625</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1126%2Fscience.8153625" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=8153625" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADyaK2cXis1Ggurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=264&publication_year=1994&pages=382-388&author=H.+Nikaido&title=Prevention+of+Drug+Access+to+Bacterial+Targets%3A+Permeability+Barriers+and+Active+Efflux&doi=10.1126%2Fscience.8153625"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Prevention of drug access to bacterial targets: permeability barriers and active efflux</span></div><div class="casAuthors">Nikaido, Hiroshi</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">264</span>
        (<span class="NLM_cas:issue">5157</span>),
    <span class="NLM_cas:pages">382-8</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    </div><div class="casAbstract">A review with 79 refs.  Some species of bacteria have low-permeability membrane barriers and are thereby intrinsically resistant to many antibiotics.  Some strains of originally antibiotic-susceptible species may also acquire resistance through decreases in the permeability of membrane barriers.  Another mechanism for preventing access of drugs to targets is the membrane-assocd. energy-driven efflux, which plays a major role in drug resistance, esp. in combination with the permeation barrier.  Recent results indicate the existence of bacterial efflux systems of extremely broad substrate specificity, in many ways reminiscent of the multidrug resistance pump of mammalian cells.  One such system seems to play a major role in the intrinsic resistance of Pseudomonas aeruginosa, a common opportunistic pathogen.  As the pharmacol. industry succeeds in producing agents that can overcome specific mechanisms of bacterial resistance, less specific resistance mechanisms such as permeability barriers and multidrug active efflux may become increasingly significant in the clin. setting.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYpbgrhsj0ZrVg90H21EOLACvtfcHk0lgx4siqlXz5Nw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXis1Ggurc%253D&md5=84eedec2b82bfc061f17d0d5b4e6973d</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1126%2Fscience.8153625&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.8153625%26sid%3Dliteratum%253Aachs%26aulast%3DNikaido%26aufirst%3DH.%26atitle%3DPrevention%2520of%2520Drug%2520Access%2520to%2520Bacterial%2520Targets%253A%2520Permeability%2520Barriers%2520and%2520Active%2520Efflux%26jtitle%3DScience%26date%3D1994%26volume%3D264%26spage%3D382%26epage%3D388%26doi%3D10.1126%2Fscience.8153625" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schweizer, H. P.</span></span> <span> </span><span class="NLM_article-title">Bacterial Resistance to Antibiotics: Active Efflux and Reduced Uptake</span>. <i>Adv. Drug Delivery Rev.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">1486</span>– <span class="NLM_lpage">1513</span>, <span class="refDoi"> DOI: 10.1016/j.addr.2005.04.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1016%2Fj.addr.2005.04.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=15939505" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BD2MXlvFant70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2005&pages=1486-1513&author=A.+Kumarauthor=H.+P.+Schweizer&title=Bacterial+Resistance+to+Antibiotics%3A+Active+Efflux+and+Reduced+Uptake&doi=10.1016%2Fj.addr.2005.04.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Bacterial resistance to antibiotics: Active efflux and reduced uptake</span></div><div class="casAuthors">Kumar, Ayush; Schweizer, Herbert P.</div><div class="citationInfo"><span class="NLM_cas:title">Advanced Drug Delivery Reviews</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1486-1513</span>CODEN:
                <span class="NLM_cas:coden">ADDREP</span>;
        ISSN:<span class="NLM_cas:issn">0169-409X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Antibiotic resistance of bacterial pathogens is a fast emerging global crisis and an understanding of the underlying resistance mechanisms is paramount for the design and development of new therapeutic strategies.  Permeability barriers for and active efflux of drug mols. are 2 resistance mechanisms that have been implicated in various infectious outbreaks of antibiotic-resistant pathogens, suggesting that these mechanisms may be good targets for new drugs.  The synergism of reduced uptake and efflux is most evident in the multiplicative action of the outer membrane permeability barrier and active efflux, which results in high-level intrinsic and(or) acquired resistance in many clin. important gram-neg. bacteria.  This review summarizes the current knowledge of these 2 important resistance mechanisms and potential strategies to overcome them.  Recent advances in understanding the phys. structures, function, and regulation of efflux systems will facilitate exploitation of pumps as new drug targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDE1hxxbXPQrVg90H21EOLACvtfcHk0lgx4siqlXz5Nw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXlvFant70%253D&md5=b1b9bed1deab6c70f83062d73abafc4f</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.addr.2005.04.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.addr.2005.04.004%26sid%3Dliteratum%253Aachs%26aulast%3DKumar%26aufirst%3DA.%26aulast%3DSchweizer%26aufirst%3DH.%2BP.%26atitle%3DBacterial%2520Resistance%2520to%2520Antibiotics%253A%2520Active%2520Efflux%2520and%2520Reduced%2520Uptake%26jtitle%3DAdv.%2520Drug%2520Delivery%2520Rev.%26date%3D2005%26volume%3D57%26spage%3D1486%26epage%3D1513%26doi%3D10.1016%2Fj.addr.2005.04.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brown, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">May-Dracka, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gagnon, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tommasi, R.</span></span> <span> </span><span class="NLM_article-title">Trends and Exceptions of Physical Properties on Antibacterial Activity for Gram-Positive and Gram-Negative Pathogens</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">10144</span>– <span class="NLM_lpage">10161</span>, <span class="refDoi"> DOI: 10.1021/jm501552x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501552x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFehtbvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=10144-10161&author=D.+G.+Brownauthor=T.+L.+May-Drackaauthor=M.+M.+Gagnonauthor=R.+Tommasi&title=Trends+and+Exceptions+of+Physical+Properties+on+Antibacterial+Activity+for+Gram-Positive+and+Gram-Negative+Pathogens&doi=10.1021%2Fjm501552x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Trends and Exceptions of Physical Properties on Antibacterial Activity for Gram-Positive and Gram-Negative Pathogens</span></div><div class="casAuthors">Brown, Dean G.; May-Dracka, Tricia L.; Gagnon, Moriah M.; Tommasi, Ruben</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">10144-10161</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">To better understand the difficulties surrounding the identification of novel antibacterial compds. from corporate screening collections, phys. properties of ∼3200 antibacterial project compds. with whole cell activity against Gram-neg. or Gram-pos. pathogens were profiled and compared to actives found from high throughput (HTS) screens conducted on both biochem. and phenotypic bacterial targets.  The output from 23 antibacterial HTS screens illustrated that when compared to the properties of the antibacterial project compds., the HTS actives were significantly more hydrophobic than antibacterial project compds. (typically 2-4 log units higher), and furthermore, for 14/23 HTS screens, the av. clogD was higher than the screening collection av. (screening collection clogD = 2.45).  It was found that the consequences of this were the following: (a) lead identification programs often further gained hydrophobic character with increased biochem. potency, making the sepn. even larger between the physicochem. properties of known antibacterial agents and the HTS active starting point, (b) the probability of plasma protein binding and cytotoxicity are often increased, and (c) cell-based activity in Gram-neg. bacteria was severely limited or, if present, demonstrated significant efflux.  Our anal. illustrated that compds. least susceptible to efflux were those which were highly polar and small in MW or very large and typically zwitterionic.  Hydrophobicity was often the dominant driver for HTS actives but, more often than not, precluded whole cell antibacterial activity.  However, simply designing polar compds. was not sufficient for antibacterial activity and pointed to a lack of understanding of complex and specific bacterial penetration mechanisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-f01NZg_bsbVg90H21EOLACvtfcHk0lgx4siqlXz5Nw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFehtbvN&md5=e107961414dc8cbf696e2287ddf9362d</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fjm501552x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501552x%26sid%3Dliteratum%253Aachs%26aulast%3DBrown%26aufirst%3DD.%2BG.%26aulast%3DMay-Dracka%26aufirst%3DT.%2BL.%26aulast%3DGagnon%26aufirst%3DM.%2BM.%26aulast%3DTommasi%26aufirst%3DR.%26atitle%3DTrends%2520and%2520Exceptions%2520of%2520Physical%2520Properties%2520on%2520Antibacterial%2520Activity%2520for%2520Gram-Positive%2520and%2520Gram-Negative%2520Pathogens%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D10144%26epage%3D10161%26doi%3D10.1021%2Fjm501552x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tommasi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walkup, G. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manchester, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, A. A.</span></span> <span> </span><span class="NLM_article-title">ESKAPEing the Labyrinth of Antibacterial Discovery</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">529</span>– <span class="NLM_lpage">542</span>, <span class="refDoi"> DOI: 10.1038/nrd4572</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1038%2Fnrd4572" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=26139286" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFyks77M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2015&pages=529-542&author=R.+Tommasiauthor=D.+G.+Brownauthor=G.+K.+Walkupauthor=J.+I.+Manchesterauthor=A.+A.+Miller&title=ESKAPEing+the+Labyrinth+of+Antibacterial+Discovery&doi=10.1038%2Fnrd4572"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">ESKAPEing the labyrinth of antibacterial discovery</span></div><div class="casAuthors">Tommasi, Ruben; Brown, Dean G.; Walkup, Grant K.; Manchester, John I.; Miller, Alita A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">529-542</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Antimicrobial drug resistance is a growing threat to global public health.  Multidrug resistance among the 'ESKAPE' organisms - encompassing Enterococcus faecium, Staphylococcus aureus, Klebsiella pneumoniae, Acinetobacter baumannii, Pseudomonas aeruginosa and Enterobacter spp. - is of particular concern because they are responsible for many serious infections in hospitals.  Although some promising agents are in the pipeline, there is an urgent need for new antibiotic scaffolds.  However, antibacterial researchers have struggled to identify new small mols. with meaningful cellular activity, esp. those effective against multidrug-resistant Gram-neg. pathogens.  This difficulty ultimately stems from an incomplete understanding of efflux systems and compd. permeation through bacterial membranes.  This Opinion article describes findings from target-based and phenotypic screening efforts carried out at AstraZeneca over the past decade, discusses some of the subsequent chem. challenges and concludes with a description of new approaches comprising a combination of computational modeling and advanced biol. tools which may pave the way towards the discovery of new antibacterial agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolJg6HXnn7BLVg90H21EOLACvtfcHk0ljH8jtg8_gO0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFyks77M&md5=0d3159250e27926ab280c305ea5fb9ef</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fnrd4572&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4572%26sid%3Dliteratum%253Aachs%26aulast%3DTommasi%26aufirst%3DR.%26aulast%3DBrown%26aufirst%3DD.%2BG.%26aulast%3DWalkup%26aufirst%3DG.%2BK.%26aulast%3DManchester%26aufirst%3DJ.%2BI.%26aulast%3DMiller%26aufirst%3DA.%2BA.%26atitle%3DESKAPEing%2520the%2520Labyrinth%2520of%2520Antibacterial%2520Discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2015%26volume%3D14%26spage%3D529%26epage%3D542%26doi%3D10.1038%2Fnrd4572" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mugumbate, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overington, J. P.</span></span> <span> </span><span class="NLM_article-title">The Relationship Between Target-Class and the Physicochemical Properties of Antibacterial Drugs</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">5218</span>– <span class="NLM_lpage">5224</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2015.04.063</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1016%2Fj.bmc.2015.04.063" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=25975639" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BC2MXnslOjs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2015&pages=5218-5224&author=G.+Mugumbateauthor=J.+P.+Overington&title=The+Relationship+Between+Target-Class+and+the+Physicochemical+Properties+of+Antibacterial+Drugs&doi=10.1016%2Fj.bmc.2015.04.063"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">The relationship between target-class and the physicochemical properties of antibacterial drugs</span></div><div class="casAuthors">Mugumbate, Grace; Overington, John P.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">5218-5224</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The discovery of novel mechanism of action (MOA) antibacterials has been assocd. with the concept that antibacterial drugs occupy a differentiated region of physicochem. space compared to human-targeted drugs.  With, in broad terms, antibacterials having higher mol. wt., lower log P and higher polar surface area (PSA).  By analyzing the physicochem. properties of about 1700 approved drugs listed in the ChEMBL database, we show, that antibacterials for whose targets are riboproteins (i.e., composed of a complex of RNA and protein) fall outside the conventional human 'drug-like' chem. space; whereas antibacterials that modulate bacterial protein targets, generally comply with the 'rule-of-five' guidelines for classical oral human drugs.  Our anal. suggests a strong target-class assocn. for antibacterials - either protein-targeted or riboprotein-targeted.  There is much discussion in the literature on the failure of screening approaches to deliver novel antibacterial lead series, and linkage of this poor success rate for antibacterials with the chem. space properties of screening collections.  Our anal. suggests that consideration of target-class may be an underappreciated factor in antibacterial lead discovery, and that in fact bacterial protein-targets may well have similar binding site characteristics to human protein targets, and questions the assumption that larger, more polar compds. are a key part of successful future antibacterial discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0zRzPd_fDo7Vg90H21EOLACvtfcHk0ljH8jtg8_gO0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXnslOjs7k%253D&md5=b6d32663c593fb09e1235968a5e5096c</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2015.04.063&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2015.04.063%26sid%3Dliteratum%253Aachs%26aulast%3DMugumbate%26aufirst%3DG.%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26atitle%3DThe%2520Relationship%2520Between%2520Target-Class%2520and%2520the%2520Physicochemical%2520Properties%2520of%2520Antibacterial%2520Drugs%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2015%26volume%3D23%26spage%3D5218%26epage%3D5224%26doi%3D10.1016%2Fj.bmc.2015.04.063" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">O’Shea, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moser, H. E.</span></span> <span> </span><span class="NLM_article-title">Physicochemical Properties of Antibacterial Compounds: Implications for Drug Discovery</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">2871</span>– <span class="NLM_lpage">2878</span>, <span class="refDoi"> DOI: 10.1021/jm700967e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm700967e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlvFCrsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=2871-2878&author=R.+O%E2%80%99Sheaauthor=H.+E.+Moser&title=Physicochemical+Properties+of+Antibacterial+Compounds%3A+Implications+for+Drug+Discovery&doi=10.1021%2Fjm700967e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Physicochemical properties of antibacterial compounds: implications for drug discovery</span></div><div class="casAuthors">O'Shea, Rosemarie; Moser, Heinz E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2871-2878</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">With the rise of multidrug-resistant pathogens and the need for novel antibiotics, it is crit. to understand as much as possible from prior efforts and to apply learned lessons to the discovery of future antibiotics.  One important parameter in particular has previously been mentioned but, in the view, not sufficiently analyzed: the physicochem. property space of antibacterial drugs.  The authors selected 147 antibacterially active compds. that encompass both currently used drugs and compds. that are still under clin. investigation (see Methods for details).  Where available, other property values were extd. from the literature, including protein binding and oral bioavailability in humans.  This anal. suggests that natural products should be increasingly investigated again to identify novel antibacterial hits.  Besides their high level of structural diversity, they are likely to better cover the required physicochem. property space for antibacterial compds. compared to synthetic mols. because of an increased d. of polar functionalities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo25EGWMkWdk7Vg90H21EOLACvtfcHk0ljH8jtg8_gO0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlvFCrsLw%253D&md5=8aae7d962df53b70113e0fe6169abd68</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fjm700967e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm700967e%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Shea%26aufirst%3DR.%26aulast%3DMoser%26aufirst%3DH.%2BE.%26atitle%3DPhysicochemical%2520Properties%2520of%2520Antibacterial%2520Compounds%253A%2520Implications%2520for%2520Drug%2520Discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D2871%26epage%3D2878%26doi%3D10.1021%2Fjm700967e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Richter, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drown, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riley, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shirai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svec, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hergenrother, P. J.</span></span> <span> </span><span class="NLM_article-title">Predictive Compound Accumulation Rules Yield a Broad-Spectrum Antibiotic</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>545</i></span>,  <span class="NLM_fpage">299</span>– <span class="NLM_lpage">304</span>, <span class="refDoi"> DOI: 10.1038/nature22308</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1038%2Fnature22308" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=28489819" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BC2sXnsVeis7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=545&publication_year=2017&pages=299-304&author=M.+F.+Richterauthor=B.+S.+Drownauthor=A.+P.+Rileyauthor=A.+Garciaauthor=T.+Shiraiauthor=R.+L.+Svecauthor=P.+J.+Hergenrother&title=Predictive+Compound+Accumulation+Rules+Yield+a+Broad-Spectrum+Antibiotic&doi=10.1038%2Fnature22308"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Predictive compound accumulation rules yield a broad-spectrum antibiotic</span></div><div class="casAuthors">Richter, Michelle F.; Drown, Bryon S.; Riley, Andrew P.; Garcia, Alfredo; Shirai, Tomohiro; Svec, Riley L.; Hergenrother, Paul J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">545</span>
        (<span class="NLM_cas:issue">7654</span>),
    <span class="NLM_cas:pages">299-304</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Most small mols. are unable to rapidly traverse the outer membrane of Gram-neg. bacteria and accumulate inside these cells, making the discovery of much-needed drugs against these pathogens challenging.  Current understanding of the physicochem. properties that dictate small-mol. accumulation in Gram-neg. bacteria is largely based on retrospective analyses of antibacterial agents, which suggest that polarity and mol. wt. are key factors.  Here the authors assess the ability of over 180 diverse compds. to accumulate in Escherichia coli.  Computational anal. of the results reveals major differences from the retrospective studies, namely that the small mols. that are most likely to accumulate contain an amine, are amphiphilic and rigid, and have low globularity.  These guidelines were then applied to convert deoxynybomycin, a natural product that is active only against Gram-pos. organisms, into an antibiotic with activity against a diverse panel of multi-drug-resistant Gram-neg. pathogens.  The authors anticipate that these findings will aid in the discovery and development of antibiotics against Gram-neg. bacteria.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGot2r9sFv94TrVg90H21EOLACvtfcHk0ljBgsPboadnxw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXnsVeis7Y%253D&md5=ae869cc99ef0e6805b96b6595a222cdc</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fnature22308&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature22308%26sid%3Dliteratum%253Aachs%26aulast%3DRichter%26aufirst%3DM.%2BF.%26aulast%3DDrown%26aufirst%3DB.%2BS.%26aulast%3DRiley%26aufirst%3DA.%2BP.%26aulast%3DGarcia%26aufirst%3DA.%26aulast%3DShirai%26aufirst%3DT.%26aulast%3DSvec%26aufirst%3DR.%2BL.%26aulast%3DHergenrother%26aufirst%3DP.%2BJ.%26atitle%3DPredictive%2520Compound%2520Accumulation%2520Rules%2520Yield%2520a%2520Broad-Spectrum%2520Antibiotic%26jtitle%3DNature%26date%3D2017%26volume%3D545%26spage%3D299%26epage%3D304%26doi%3D10.1038%2Fnature22308" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Serio, A. W.</span>; <span class="NLM_string-name">Kubo, A.</span>; <span class="NLM_string-name">Lopez, S.</span>; <span class="NLM_string-name">Gomez, M.</span>; <span class="NLM_string-name">Corey, V. C.</span>; <span class="NLM_string-name">Andrews, L. D.</span>; <span class="NLM_string-name">Schwatz, M. A.</span>; <span class="NLM_string-name">Kasar, R.</span>; <span class="NLM_string-name">McEnRoe, G.</span>; <span class="NLM_string-name">Aggen, J.</span>; <span class="NLM_string-name">Miller, G.</span>; <span class="NLM_string-name">Armstrong, E. S.</span></span> <span> </span><span class="NLM_article-title">Structure, Potency and Bactericidal Activity of ACHN-975, a First-In-Class LpxC Inhibitor</span>. In  <i>53rd Interscience Conference on Antimicrobial Agents and Chemotherapy</i>; <span class="NLM_publisher-name">Achaogen</span>, <span class="NLM_year">2013</span> [poster #F-1226].</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=A.+W.+Serio&author=A.+Kubo&author=S.+Lopez&author=M.+Gomez&author=V.+C.+Corey&author=L.+D.+Andrews&author=M.+A.+Schwatz&author=R.+Kasar&author=G.+McEnRoe&author=J.+Aggen&author=G.+Miller&author=E.+S.+Armstrong&title=Structure%2C+Potency+and+Bactericidal+Activity+of+ACHN-975%2C+a+First-In-Class+LpxC+Inhibitor"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dconf-proc%26sid%3Dliteratum%253Aachs%26aulast%3DSerio%26aufirst%3DA.%2BW.%26atitle%3DStructure%252C%2520Potency%2520and%2520Bactericidal%2520Activity%2520of%2520ACHN-975%252C%2520a%2520First-In-Class%2520LpxC%2520Inhibitor%26jtitle%3D53rd%2520Interscience%2520Conference%2520on%2520Antimicrobial%2520Agents%2520and%2520Chemotherapy%26pub%3DAchaogen%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tari, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trzoss, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bensen, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Creighton, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cunningham, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stidham, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lightstone, F. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, T. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nix, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finn, J.</span></span> <span> </span><span class="NLM_article-title">Pyrrolopyrimidine Inhibitors of DNA Gyrase B (GyrB) and Topoisomerase IV (ParE). Part I: Structure Guided Discovery and Optimization of Dual Targeting Agents with Potent, Broad-Spectrum Enzymatic Activity</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1529</span>– <span class="NLM_lpage">1536</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2012.11.032</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1016%2Fj.bmcl.2012.11.032" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=23352267" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVOrsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2013&pages=1529-1536&author=L.+W.+Tariauthor=M.+Trzossauthor=D.+C.+Bensenauthor=X.+Liauthor=Z.+Chenauthor=T.+Lamauthor=J.+Zhangauthor=C.+J.+Creightonauthor=M.+L.+Cunninghamauthor=B.+Kwanauthor=M.+Stidhamauthor=K.+J.+Shawauthor=F.+C.+Lightstoneauthor=S.+E.+Wongauthor=T.+B.+Nguyenauthor=J.+Nixauthor=J.+Finn&title=Pyrrolopyrimidine+Inhibitors+of+DNA+Gyrase+B+%28GyrB%29+and+Topoisomerase+IV+%28ParE%29.+Part+I%3A+Structure+Guided+Discovery+and+Optimization+of+Dual+Targeting+Agents+with+Potent%2C+Broad-Spectrum+Enzymatic+Activity&doi=10.1016%2Fj.bmcl.2012.11.032"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Pyrrolopyrimidine inhibitors of DNA gyrase B (GyrB) and topoisomerase IV (ParE). Part I: Structure guided discovery and optimization of dual targeting agents with potent, broad-spectrum enzymatic activity</span></div><div class="casAuthors">Tari, Leslie W.; Trzoss, Michael; Bensen, Daniel C.; Li, Xiaoming; Chen, Zhiyong; Lam, Thanh; Zhang, Junhu; Creighton, Christopher J.; Cunningham, Mark L.; Kwan, Bryan; Stidham, Mark; Shaw, Karen J.; Lightstone, Felice C.; Wong, Sergio E.; Nguyen, Toan B.; Nix, Jay; Finn, John</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1529-1536</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The bacterial topoisomerases DNA gyrase (GyrB) and topoisomerase IV (ParE) are essential enzymes that control the topol. state of DNA during replication.  The high degree of conservation in the ATP-binding pockets of these enzymes make them appealing targets for broad-spectrum inhibitor development.  A pyrrolopyrimidine scaffold was identified from a pharmacophore-based fragment screen with optimization potential.  Structural characterization of inhibitor complexes conducted using selected GyrB/ParE orthologs aided in the identification of important steric, dynamic and compositional differences in the ATP-binding pockets of the targets, enabling the design of highly potent pyrrolopyrimidine inhibitors with broad enzymic spectrum and dual targeting activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqn5kRWxZVjhbVg90H21EOLACvtfcHk0ljBgsPboadnxw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVOrsbo%253D&md5=b8d572d3dc5320c2ec385668b3f60387</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2012.11.032&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2012.11.032%26sid%3Dliteratum%253Aachs%26aulast%3DTari%26aufirst%3DL.%2BW.%26aulast%3DTrzoss%26aufirst%3DM.%26aulast%3DBensen%26aufirst%3DD.%2BC.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DLam%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DCreighton%26aufirst%3DC.%2BJ.%26aulast%3DCunningham%26aufirst%3DM.%2BL.%26aulast%3DKwan%26aufirst%3DB.%26aulast%3DStidham%26aufirst%3DM.%26aulast%3DShaw%26aufirst%3DK.%2BJ.%26aulast%3DLightstone%26aufirst%3DF.%2BC.%26aulast%3DWong%26aufirst%3DS.%2BE.%26aulast%3DNguyen%26aufirst%3DT.%2BB.%26aulast%3DNix%26aufirst%3DJ.%26aulast%3DFinn%26aufirst%3DJ.%26atitle%3DPyrrolopyrimidine%2520Inhibitors%2520of%2520DNA%2520Gyrase%2520B%2520%2528GyrB%2529%2520and%2520Topoisomerase%2520IV%2520%2528ParE%2529.%2520Part%2520I%253A%2520Structure%2520Guided%2520Discovery%2520and%2520Optimization%2520of%2520Dual%2520Targeting%2520Agents%2520with%2520Potent%252C%2520Broad-Spectrum%2520Enzymatic%2520Activity%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D23%26spage%3D1529%26epage%3D1536%26doi%3D10.1016%2Fj.bmcl.2012.11.032" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Richter, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hergenrother, P. J.</span></span> <span> </span><span class="NLM_article-title">The Challenge of Converting Gram-Positive-Only Compounds Into Broad-Spectrum Antibiotics</span>. <i>Ann. N. Y. Acad. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>1435</i></span>,  <span class="NLM_fpage">18</span>– <span class="NLM_lpage">38</span>, <span class="refDoi"> DOI: 10.1111/nyas.13598</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1111%2Fnyas.13598" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=29446459" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitlyktw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1435&publication_year=2019&pages=18-38&author=M.+F.+Richterauthor=P.+J.+Hergenrother&title=The+Challenge+of+Converting+Gram-Positive-Only+Compounds+Into+Broad-Spectrum+Antibiotics&doi=10.1111%2Fnyas.13598"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">The challenge of converting Gram-positive-only compounds into broad-spectrum antibiotics</span></div><div class="casAuthors">Richter, Michelle F.; Hergenrother, Paul J.</div><div class="citationInfo"><span class="NLM_cas:title">Annals of the New York Academy of Sciences</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">1435</span>
        (<span class="NLM_cas:issue">1, Antimicrobial Therapeutics Reviews</span>),
    <span class="NLM_cas:pages">18-38</span>CODEN:
                <span class="NLM_cas:coden">ANYAA9</span>;
        ISSN:<span class="NLM_cas:issn">0077-8923</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  Multidrug resistant Gram-neg. bacterial infections are on the rise, and there is a lack of new classes of drugs to treat these pathogens.  This drug shortage is largely due to the challenge of finding antibiotics that can permeate and persist inside Gram-neg. species.  Efforts to understand the mol. properties that enable certain compds. to accumulate in Gram-neg. bacteria based on retrospective studies of known antibiotics have not been generally actionable in the development of new antibiotics.  A recent assessment of the ability of >180 diverse small mols. to accumulate in Escherichia coli led to predictive guidelines for compd. accumulation in E. coli.  These "eNTRy rules" state that compds. are most likely to accumulate if they contain a nonsterically encumbered ionizable Nitrogen (primary amines are the best), have low Three-dimensionality (globularity ≤ 0.25), and are relatively Rigid (rotatable bonds ≤ 5).  In this review, we look back through 50+ years of antibacterial research and 1000s of derivs. and assess this historical data set through the lens of these predictive guidelines.  The results are consistent with the eNTRy rules, suggesting that the eNTRy rules may provide an actionable and general roadmap for the conversion of Gram-pos.-only compds. into broad-spectrum antibiotics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqX3zbgiAcHdbVg90H21EOLACvtfcHk0ljBgsPboadnxw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitlyktw%253D%253D&md5=12fffe0d10e7d3e5c134ac28ce929ab3</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1111%2Fnyas.13598&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fnyas.13598%26sid%3Dliteratum%253Aachs%26aulast%3DRichter%26aufirst%3DM.%2BF.%26aulast%3DHergenrother%26aufirst%3DP.%2BJ.%26atitle%3DThe%2520Challenge%2520of%2520Converting%2520Gram-Positive-Only%2520Compounds%2520Into%2520Broad-Spectrum%2520Antibiotics%26jtitle%3DAnn.%2520N.%2520Y.%2520Acad.%2520Sci.%26date%3D2019%26volume%3D1435%26spage%3D18%26epage%3D38%26doi%3D10.1111%2Fnyas.13598" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vaara, M.</span></span> <span> </span><span class="NLM_article-title">Agents That Increase the Permeability of the Outer Membrane</span>. <i>Microbiol. Mol. Biol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">1992</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">395</span>– <span class="NLM_lpage">411</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADyaK3sXjsFCktQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=1992&pages=395-411&author=M.+Vaara&title=Agents+That+Increase+the+Permeability+of+the+Outer+Membrane"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Agents that increase the permeability of the outer membrane</span></div><div class="casAuthors">Vaara, Martti</div><div class="citationInfo"><span class="NLM_cas:title">Microbiological Reviews</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">395-411</span>CODEN:
                <span class="NLM_cas:coden">MBRED3</span>;
        ISSN:<span class="NLM_cas:issn">0146-0749</span>.
    </div><div class="casAbstract">A review with 241 refs. on cationic agents and chelators as outer membrane permeabilizers, the mol. mechanism of outer membrane permeability increase, and bacterial strains resistant to the permeabilizers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrIuhdsLpq5OLVg90H21EOLACvtfcHk0lhspznD5MnegQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXjsFCktQ%253D%253D&md5=2416c67d643c9be119bfb1c0d2323b62</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DVaara%26aufirst%3DM.%26atitle%3DAgents%2520That%2520Increase%2520the%2520Permeability%2520of%2520the%2520Outer%2520Membrane%26jtitle%3DMicrobiol.%2520Mol.%2520Biol.%2520Rev.%26date%3D1992%26volume%3D56%26spage%3D395%26epage%3D411" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Angus, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carey, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caron, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kropinski, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hancock, R. E.</span></span> <span> </span><span class="NLM_article-title">Outer Membrane Permeability in <i>Pseudomonas aeruginosa</i>: Comparison of a Wild-Type with an Antibiotic-Supersusceptible Mutant</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">1982</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">299</span>– <span class="NLM_lpage">309</span>, <span class="refDoi"> DOI: 10.1128/AAC.21.2.299</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1128%2FAAC.21.2.299" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=6803666" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADyaL38XhtFyruro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=1982&pages=299-309&author=B.+L.+Angusauthor=A.+M.+Careyauthor=D.+A.+Caronauthor=A.+M.+Kropinskiauthor=R.+E.+Hancock&title=Outer+Membrane+Permeability+in+Pseudomonas+aeruginosa%3A+Comparison+of+a+Wild-Type+with+an+Antibiotic-Supersusceptible+Mutant&doi=10.1128%2FAAC.21.2.299"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Outer membrane permeability in Pseudomonas aeruginosa:  comparison of a wild-type with an antibiotic-supersusceptible mutant</span></div><div class="casAuthors">Angus, Barbara L.; Carey, Alice M.; Caron, Dania A.; Kropinski, Andrew M. B.; Hancock, Robert E. W.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">1982</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">299-309</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    </div><div class="casAbstract">Spontaneous revertants of P. aeruginosa were isolated, using gentamicin or carbenicillin as selective agents, and shown to have 2 patterns of susceptibility to a group of 12 antibiotics.  Partial revertants had 2- to 10-fold greater resistance to these antibiotics than mutant Z61, whereas full revertants had antibiotic susceptibilities indistinguishable from those of the wild-type strain K799, from which mutant Z61 had been derived.  Uptake of a chromogenic β-lactam, nitrocefin, was studied in both uninduced and induced cells of all strains by measuring the steady-state rate of nitrocefin hydrolysis by the inducible, periplasmic β-lactamase in both whole and broken cells.  This demonstrated that outer membrane permeability decreased as antibiotic resistance increased in the series: mutant Z61, partial revertants, wild-type, and full revertants.  Low outer membrane permeability, caused by a low proportion of open functional porins in the outer membrane, may be the reason for the high natural antibiotic resistance of wild-type P. aeruginosa strains.  In addn., levels of benzylpenicillin below the minimal inhibitory concn. for mutant Z61 failed to induce β-lactamase prodn.  The possibility that this was related to the obsd. increase in outer membrane permeability is discussed.  Preliminary evidence is presented that the pore-forming outer membrane porin protein F is not altered in mutant Z61.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXV0owNpUYT7Vg90H21EOLACvtfcHk0lhspznD5MnegQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL38XhtFyruro%253D&md5=aa7c0311c93f236f6f184c301713bfb8</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1128%2FAAC.21.2.299&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.21.2.299%26sid%3Dliteratum%253Aachs%26aulast%3DAngus%26aufirst%3DB.%2BL.%26aulast%3DCarey%26aufirst%3DA.%2BM.%26aulast%3DCaron%26aufirst%3DD.%2BA.%26aulast%3DKropinski%26aufirst%3DA.%2BM.%26aulast%3DHancock%26aufirst%3DR.%2BE.%26atitle%3DOuter%2520Membrane%2520Permeability%2520in%2520Pseudomonas%2520aeruginosa%253A%2520Comparison%2520of%2520a%2520Wild-Type%2520with%2520an%2520Antibiotic-Supersusceptible%2520Mutant%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D1982%26volume%3D21%26spage%3D299%26epage%3D309%26doi%3D10.1128%2FAAC.21.2.299" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yoshimura, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikaido, H.</span></span> <span> </span><span class="NLM_article-title">Permeability of <i>Pseudomonas aeruginosa</i> Outer Membrane to Hydrophilic Solutes</span>. <i>J. Bacteriol.</i> <span class="NLM_year" style="font-weight: bold;">1982</span>,  <span class="NLM_volume"><i>152</i></span>,  <span class="NLM_fpage">636</span>– <span class="NLM_lpage">642</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=6813310" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADyaL38XmtFagu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=152&publication_year=1982&pages=636-642&author=F.+Yoshimuraauthor=H.+Nikaido&title=Permeability+of+Pseudomonas+aeruginosa+Outer+Membrane+to+Hydrophilic+Solutes"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Permeability of Pseudomonas aeruginosa outer membrane to hydrophilic solutes</span></div><div class="casAuthors">Yoshimura, Fuminobu; Nikaido, Hiroshi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Bacteriology</span>
        (<span class="NLM_cas:date">1982</span>),
    <span class="NLM_cas:volume">152</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">636-42</span>CODEN:
                <span class="NLM_cas:coden">JOBAAY</span>;
        ISSN:<span class="NLM_cas:issn">0021-9193</span>.
    </div><div class="casAbstract">P. aeruginosa Is usually resistant to a wide variety of antibacterial agents, and it has been inferred, on the basis of indirect evidence, that this was due to the low permeability of its outer membrane.  The permeability of P. aeruginosa outer membrane was detd. directly, by measuring the rates of hydrolysis of cephacetrile, cephaloridine, and various phosphate esters by hydrolytic enzymes located in the periplasm.  The permeability to these compds. was ∼100-fold lower than of the outer membrane of Escherichia coli K-12.  Also, the apparent Km values for active transport of various C and energy source compds. were typically >20 μM in P. aeruginosa, in contrast to E. coli, in which the values are usually <5 μM.  Apparently, the P. aeruginosa outer membrane indeed has a low permeability to most hydrophilic compds., and this membrane acts as a rate-limiting step in active transport processes with high Vmax values.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxJ7cuhYCEVbVg90H21EOLACvtfcHk0lhspznD5MnegQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL38XmtFagu7s%253D&md5=1b90ff40a7309d64b724487ccb6ea0ed</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DYoshimura%26aufirst%3DF.%26aulast%3DNikaido%26aufirst%3DH.%26atitle%3DPermeability%2520of%2520Pseudomonas%2520aeruginosa%2520Outer%2520Membrane%2520to%2520Hydrophilic%2520Solutes%26jtitle%3DJ.%2520Bacteriol.%26date%3D1982%26volume%3D152%26spage%3D636%26epage%3D642" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Delcour, A.
H.</span></span> <span> </span><span class="NLM_article-title">Outer Membrane Permeability and Antibiotic Resistance</span>. <i>Biochim. Biophys. Acta</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>1794</i></span>,  <span class="NLM_fpage">808</span>– <span class="NLM_lpage">816</span>, <span class="refDoi"> DOI: 10.1016/j.bbapap.2008.11.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1016%2Fj.bbapap.2008.11.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=19100346" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BD1MXkvVSrtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1794&publication_year=2009&pages=808-816&author=A.%0AH.+Delcour&title=Outer+Membrane+Permeability+and+Antibiotic+Resistance&doi=10.1016%2Fj.bbapap.2008.11.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Outer membrane permeability and antibiotic resistance</span></div><div class="casAuthors">Delcour, Anne H.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Proteins and Proteomics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">1794</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">808-816</span>CODEN:
                <span class="NLM_cas:coden">BBAPBW</span>;
        ISSN:<span class="NLM_cas:issn">1570-9639</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B. V.</span>)
        </div><div class="casAbstract">A review.  To date most antibiotics are targeted at intracellular processes, and must be able to penetrate the bacterial cell envelope.  In particular, the outer membrane of gram-neg. bacteria provides a formidable barrier that must be overcome.  There are essentially two pathways that antibiotics can take through the outer membrane: a lipid-mediated pathway for hydrophobic antibiotics, and general diffusion porins for hydrophilic antibiotics.  The lipid and protein compns. of the outer membrane have a strong impact on the sensitivity of bacteria to many types of antibiotics, and drug resistance involving modifications of these macromols. is common.  This review describes the mol. mechanisms for permeation of antibiotics through the outer membrane, and the strategies that bacteria have deployed to resist antibiotics by modifications of these pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnujySVWxi3bVg90H21EOLACvtfcHk0lhspznD5MnegQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXkvVSrtr4%253D&md5=b32db36960e4c0d122bef0c3648462e8</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.bbapap.2008.11.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbapap.2008.11.005%26sid%3Dliteratum%253Aachs%26aulast%3DDelcour%26aufirst%3DA.%2BH.%26atitle%3DOuter%2520Membrane%2520Permeability%2520and%2520Antibiotic%2520Resistance%26jtitle%3DBiochim.%2520Biophys.%2520Acta%26date%3D2009%26volume%3D1794%26spage%3D808%26epage%3D816%26doi%3D10.1016%2Fj.bbapap.2008.11.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Page, M. G. P.</span></span> <span> </span><span class="NLM_article-title">The Role of the Outer Membrane of Gram-Negative Bacteria in Antibiotic Resistance: Ajax’ Shield or Achilles’ Heel?</span>. <i>Handb. Exp. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>211</i></span>,  <span class="NLM_fpage">67</span>– <span class="NLM_lpage">86</span>, <span class="refDoi"> DOI: 10.1007/978-3-642-28951-4_5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1007%2F978-3-642-28951-4_5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BC3sXnslWqt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=211&publication_year=2012&pages=67-86&author=M.+G.+P.+Page&title=The+Role+of+the+Outer+Membrane+of+Gram-Negative+Bacteria+in+Antibiotic+Resistance%3A+Ajax%E2%80%99+Shield+or+Achilles%E2%80%99+Heel%3F&doi=10.1007%2F978-3-642-28951-4_5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">The role of the outer membrane of gram-negative bacteria in antibiotic resistance: Ajax' shield of Achilles' heel?</span></div><div class="casAuthors">Page, Malcolm G. P.</div><div class="citationInfo"><span class="NLM_cas:title">Handbook of Experimental Pharmacology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">211</span>
        (<span class="NLM_cas:issue">Antibiotic Resistance</span>),
    <span class="NLM_cas:pages">67-86</span>CODEN:
                <span class="NLM_cas:coden">HEPHD2</span>;
        ISSN:<span class="NLM_cas:issn">0171-2004</span>.
    
            (<span class="NLM_cas:orgname">Springer GmbH</span>)
        </div><div class="casAbstract">A review.  There has been an enormous increase in our knowledge of the fundamental steps in the biosynthesis and assembly of the outer membrane of Gram-neg. bacteria.  Lipopolysaccharide is a major component of the outer membrane of Gram-neg. bacteria as is peptidoglycan.  Porins, efflux pumps and other transport proteins of the outer membrane are also present.  It is clear that there are numerous essential proteins that have the potential to be targets for novel antimicrobial agents.  Progress, however, has been slow.  Much of the emphasis has been on cytoplasmic processes that were better understood earlier on, but have the drawback that two penetration barriers, with different permeability properties, have to be crossed.  With the increased understanding of the late-stage events occurring in the periplasm, it may be possible to shift focus to these more accessible targets.  Nevertheless, getting drugs across the outer membrane will remain a challenge to the ingenuity of the medicinal chemist.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3ryHhzdkuqbVg90H21EOLACvtfcHk0lhZo087yhDHDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXnslWqt7o%253D&md5=2d10c275feb129390c0349297fe8b468</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1007%2F978-3-642-28951-4_5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-3-642-28951-4_5%26sid%3Dliteratum%253Aachs%26aulast%3DPage%26aufirst%3DM.%2BG.%2BP.%26atitle%3DThe%2520Role%2520of%2520the%2520Outer%2520Membrane%2520of%2520Gram-Negative%2520Bacteria%2520in%2520Antibiotic%2520Resistance%253A%2520Ajax%25E2%2580%2599%2520Shield%2520or%2520Achilles%25E2%2580%2599%2520Heel%253F%26jtitle%3DHandb.%2520Exp.%2520Pharmacol.%26date%3D2012%26volume%3D211%26spage%3D67%26epage%3D86%26doi%3D10.1007%2F978-3-642-28951-4_5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Breidenstein, E. B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de la Fuente-Núñez, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hancock, R. E. W.</span></span> <span> </span><span class="NLM_article-title"><i>Pseudomonas aeruginosa</i>: All Roads Lead to Resistance</span>. <i>Trends Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">419</span>– <span class="NLM_lpage">426</span>, <span class="refDoi"> DOI: 10.1016/j.tim.2011.04.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1016%2Fj.tim.2011.04.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=21664819" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BC3MXps1entbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2011&pages=419-426&author=E.+B.+M.+Breidensteinauthor=C.+de+la+Fuente-N%C3%BA%C3%B1ezauthor=R.+E.+W.+Hancock&title=Pseudomonas+aeruginosa%3A+All+Roads+Lead+to+Resistance&doi=10.1016%2Fj.tim.2011.04.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Pseudomonas aeruginosa: all roads lead to resistance</span></div><div class="casAuthors">Breidenstein, Elena B. M.; de la Fuente-Nunez, Cesar; Hancock, Robert E. W.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Microbiology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">419-426</span>CODEN:
                <span class="NLM_cas:coden">TRMIEA</span>;
        ISSN:<span class="NLM_cas:issn">0966-842X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Pseudomonas aeruginosa is often resistant to multiple antibiotics and consequently has joined the ranks of superbugs' due to its enormous capacity to engender resistance.  It demonstrates decreased susceptibility to most antibiotics due to low outer membrane permeability coupled to adaptive mechanisms and can readily achieve clin. resistance.  Newer research, using mutant library screens, microarray technologies and mutation frequency anal., has identified very large collections of genes (the resistome) that when mutated lead to resistance as well as new forms of adaptive resistance that can be triggered by antibiotics themselves, in in vivo growth conditions or complex adaptations such as biofilm growth or swarming motility.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrp-9n40KdDx7Vg90H21EOLACvtfcHk0lhZo087yhDHDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXps1entbo%253D&md5=8af03809d5bc30795cdb3dc403808c33</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.tim.2011.04.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tim.2011.04.005%26sid%3Dliteratum%253Aachs%26aulast%3DBreidenstein%26aufirst%3DE.%2BB.%2BM.%26aulast%3Dde%2Bla%2BFuente-N%25C3%25BA%25C3%25B1ez%26aufirst%3DC.%26aulast%3DHancock%26aufirst%3DR.%2BE.%2BW.%26atitle%3DPseudomonas%2520aeruginosa%253A%2520All%2520Roads%2520Lead%2520to%2520Resistance%26jtitle%3DTrends%2520Microbiol.%26date%3D2011%26volume%3D19%26spage%3D419%26epage%3D426%26doi%3D10.1016%2Fj.tim.2011.04.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Waldrop, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holden, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maurice, M. S.</span></span> <span> </span><span class="NLM_article-title">The Enzymes of Biotin Dependent CO<sub>2</sub> Metabolism: What Structures Reveal About Their Reaction Mechanisms</span>. <i>Protein Sci.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">1597</span>– <span class="NLM_lpage">1619</span>, <span class="refDoi"> DOI: 10.1002/pro.2156</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1002%2Fpro.2156" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=22969052" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFWitrfE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2012&pages=1597-1619&author=G.+L.+Waldropauthor=H.+M.+Holdenauthor=M.+S.+Maurice&title=The+Enzymes+of+Biotin+Dependent+CO2+Metabolism%3A+What+Structures+Reveal+About+Their+Reaction+Mechanisms&doi=10.1002%2Fpro.2156"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">The enzymes of biotin dependent CO2 metabolism: What structures reveal about their reaction mechanisms</span></div><div class="casAuthors">Waldrop, Grover L.; Holden, Hazel M.; St. Maurice, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1597-1619</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">1469-896X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  Biotin is the major cofactor involved in CO2 metab.  Indeed, biotin-dependent enzymes are ubiquitous in nature and are involved in a myriad of metabolic processes including fatty acid synthesis and gluconeogenesis.  The cofactor, itself, is composed of a ureido ring, a tetrahydrothiophene ring, and a valeric acid side-chain.  It is the ureido ring that functions as the CO2 carrier.  A complete understanding of biotin-dependent enzymes is critically important for translational research in light of the fact that some of these enzymes serve as targets for anti-obesity agents, antibiotics, and herbicides.  Prior to 1990, however, there was a dearth of information regarding the mol. architectures of biotin-dependent enzymes.  In recent years there has been an explosion in the no. of 3-dimensional structures reported for these proteins.  Here, the authors review the current understanding of the structures and functions of biotin-dependent enzymes.  In addn., the authors provide a crit. anal. of what these structures have and have not revealed about biotin-dependent catalysis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhq6DUbpmID7Vg90H21EOLACvtfcHk0lhZo087yhDHDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFWitrfE&md5=0f3b44a3b78c67296f7282c7e47c8a0f</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1002%2Fpro.2156&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpro.2156%26sid%3Dliteratum%253Aachs%26aulast%3DWaldrop%26aufirst%3DG.%2BL.%26aulast%3DHolden%26aufirst%3DH.%2BM.%26aulast%3DMaurice%26aufirst%3DM.%2BS.%26atitle%3DThe%2520Enzymes%2520of%2520Biotin%2520Dependent%2520CO2%2520Metabolism%253A%2520What%2520Structures%2520Reveal%2520About%2520Their%2520Reaction%2520Mechanisms%26jtitle%3DProtein%2520Sci.%26date%3D2012%26volume%3D21%26spage%3D1597%26epage%3D1619%26doi%3D10.1002%2Fpro.2156" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Broussard, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pakhomova, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neau, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonnot, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldrop, G. L.</span></span> <span> </span><span class="NLM_article-title">Structural Analysis of Substrate, Reaction Intermediate, and Product Binding in <i>Haemophilus influenzae</i> Biotin Carboxylase</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">3860</span>– <span class="NLM_lpage">3870</span>, <span class="refDoi"> DOI: 10.1021/acs.biochem.5b00340</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.biochem.5b00340" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BC2MXptV2rtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2015&pages=3860-3870&author=T.+C.+Broussardauthor=S.+Pakhomovaauthor=D.+B.+Neauauthor=R.+Bonnotauthor=G.+L.+Waldrop&title=Structural+Analysis+of+Substrate%2C+Reaction+Intermediate%2C+and+Product+Binding+in+Haemophilus+influenzae+Biotin+Carboxylase&doi=10.1021%2Facs.biochem.5b00340"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Analysis of Substrate, Reaction Intermediate, and Product Binding in Haemophilus influenzae Biotin Carboxylase</span></div><div class="casAuthors">Broussard, Tyler C.; Pakhomova, Svetlana; Neau, David B.; Bonnot, Ross; Waldrop, Grover L.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">3860-3870</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Acetyl-CoA carboxylase catalyzes the first and regulated step in fatty acid synthesis.  In most Gram-neg. and Gram-pos. bacteria, the enzyme is composed of three proteins: biotin carboxylase, a biotin carboxyl carrier protein (BCCP), and carboxyltransferase.  The reaction mechanism involves two half-reactions, with biotin carboxylase catalyzing the ATP-dependent carboxylation of biotin-BCCP in the first reaction.  In the second reaction, carboxyltransferase catalyzes the transfer of the carboxyl group from biotin-BCCP to acetyl-CoA to form malonyl-CoA.  In this report, high-resoln. crystal structures of biotin carboxylase from Haemophilus influenzae were detd. with bicarbonate, the ATP analog AMPPCP; the carboxyphosphate intermediate analogs, phosphonoacetamide (PAM) and phosphonoformate (POF); the products ADP and phosphate; and the carboxybiotin analog N1'-methoxycarbonyl biotin Me ester.  The structures have a common theme in that bicarbonate, phosphate, and the Me ester of the carboxyl group of N1'-methoxycarbonyl biotin Me ester all bound in the same pocket in the active site of biotin carboxylase and as such utilize the same set of amino acids for binding.  This finding suggests a catalytic mechanism for biotin carboxylase in which the binding pocket that binds tetrahedral phosphate also accommodates and stabilizes a tetrahedral dianionic transition state resulting from direct transfer of CO2 from the carboxyphosphate intermediate to biotin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2YP7QWNwytrVg90H21EOLACvtfcHk0lj1PS_Lwtc4nQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXptV2rtrw%253D&md5=99925d8aa7204fa50a9bfb12f83edb19</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Facs.biochem.5b00340&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.biochem.5b00340%26sid%3Dliteratum%253Aachs%26aulast%3DBroussard%26aufirst%3DT.%2BC.%26aulast%3DPakhomova%26aufirst%3DS.%26aulast%3DNeau%26aufirst%3DD.%2BB.%26aulast%3DBonnot%26aufirst%3DR.%26aulast%3DWaldrop%26aufirst%3DG.%2BL.%26atitle%3DStructural%2520Analysis%2520of%2520Substrate%252C%2520Reaction%2520Intermediate%252C%2520and%2520Product%2520Binding%2520in%2520Haemophilus%2520influenzae%2520Biotin%2520Carboxylase%26jtitle%3DBiochemistry%26date%3D2015%26volume%3D54%26spage%3D3860%26epage%3D3870%26doi%3D10.1021%2Facs.biochem.5b00340" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cunningham, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwan, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nelson, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bensen, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, K. J.</span></span> <span> </span><span class="NLM_article-title">Distinguishing On-Target Versus Off-Target Activity in Early Antibacterial Drug Discovery Using a Macromolecular Synthesis Assay</span>. <i>J. Biomol. Screening</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">1018</span>– <span class="NLM_lpage">1026</span>, <span class="refDoi"> DOI: 10.1177/1087057113487208</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1177%2F1087057113487208" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=23686103" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslWitr%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2013&pages=1018-1026&author=M.+L.+Cunninghamauthor=B.+P.+Kwanauthor=K.+J.+Nelsonauthor=D.+C.+Bensenauthor=K.+J.+Shaw&title=Distinguishing+On-Target+Versus+Off-Target+Activity+in+Early+Antibacterial+Drug+Discovery+Using+a+Macromolecular+Synthesis+Assay&doi=10.1177%2F1087057113487208"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Distinguishing on-target versus off-target activity in early antibacterial drug discovery using a macromolecular synthesis assay</span></div><div class="casAuthors">Cunningham, Mark L.; Kwan, Bryan P.; Nelson, Kirk J.; Bensen, Daniel C.; Shaw, Karen J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1018-1026, 9</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">The macromol. synthesis assay was optimized in both S. aureus and E. coli imp and used to define patterns of inhibition of DNA, RNA, protein, and cell wall biosynthesis of several drug classes.  The concn. of drug required to elicit pathway inhibition differed among the antimicrobial agents tested, with inhibition detected at concns. significantly below the min. inhibitory concn. (MIC) for tedizolid; within 4-fold of the MIC for ciprofloxacin, cefepime, vancomycin, tetracycline, and chloramphenicol; and significantly above the MIC for rifampicin and kanamycin.  In a DNA gyrase/topoisomerase IV structure-based drug design optimization program, the assay rapidly identified undesirable offtarget activity within certain chemotypes, altering the course of the program to focus on the series that maintained ontarget activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3Es7L9oicRbVg90H21EOLACvtfcHk0lj1PS_Lwtc4nQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslWitr%252FF&md5=ca15beac8968c13ce847b201a52d73d3</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1177%2F1087057113487208&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057113487208%26sid%3Dliteratum%253Aachs%26aulast%3DCunningham%26aufirst%3DM.%2BL.%26aulast%3DKwan%26aufirst%3DB.%2BP.%26aulast%3DNelson%26aufirst%3DK.%2BJ.%26aulast%3DBensen%26aufirst%3DD.%2BC.%26aulast%3DShaw%26aufirst%3DK.%2BJ.%26atitle%3DDistinguishing%2520On-Target%2520Versus%2520Off-Target%2520Activity%2520in%2520Early%2520Antibacterial%2520Drug%2520Discovery%2520Using%2520a%2520Macromolecular%2520Synthesis%2520Assay%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2013%26volume%3D18%26spage%3D1018%26epage%3D1026%26doi%3D10.1177%2F1087057113487208" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nonejuie, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burkart, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pogliano, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pogliano, J.</span></span> <span> </span><span class="NLM_article-title">Bacterial Cytological Profiling Rapidly Identifies the Cellular Pathways Targeted by Antibacterial Molecules</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">16169</span>– <span class="NLM_lpage">16174</span>, <span class="refDoi"> DOI: 10.1073/pnas.1311066110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1073%2Fpnas.1311066110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=24046367" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1SqtbfI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=16169-16174&author=P.+Nonejuieauthor=M.+Burkartauthor=K.+Poglianoauthor=J.+Pogliano&title=Bacterial+Cytological+Profiling+Rapidly+Identifies+the+Cellular+Pathways+Targeted+by+Antibacterial+Molecules&doi=10.1073%2Fpnas.1311066110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Bacterial cytological profiling rapidly identifies the cellular pathways targeted by antibacterial molecules</span></div><div class="casAuthors">Nonejuie, Poochit; Burkart, Michael; Pogliano, Kit; Pogliano, Joe</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">40</span>),
    <span class="NLM_cas:pages">16169-16174,S16169/1-S16169/14</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Identifying the mechanism of action for antibacterial compds. is essential for understanding how bacteria interact with one another and with other cell types and for antibiotic discovery efforts, but detg. a compd.'s mechanism of action remains a serious challenge that limits both basic research and antibacterial discovery programs.  Here, we show that bacterial cytol. profiling (BCP) is a rapid and powerful approach for identifying the cellular pathway affected by antibacterial mols.  BCP can distinguish between inhibitors that affect different cellular pathways as well as different targets within the same pathway.  We use BCP to demonstrate that spirohexenolide A, a spirotetronate that is active against methicillin-resistant Staphylococcus aureus, rapidly collapses the proton motive force.  BCP offers a simple, one-step assay that can be broadly applied, solving the longstanding problem of how to rapidly det. the cellular target of thousands of compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqb4cbb0TVNjLVg90H21EOLACvtfcHk0lj1PS_Lwtc4nQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1SqtbfI&md5=8822d9a313f79b659d28307659057a58</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1311066110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1311066110%26sid%3Dliteratum%253Aachs%26aulast%3DNonejuie%26aufirst%3DP.%26aulast%3DBurkart%26aufirst%3DM.%26aulast%3DPogliano%26aufirst%3DK.%26aulast%3DPogliano%26aufirst%3DJ.%26atitle%3DBacterial%2520Cytological%2520Profiling%2520Rapidly%2520Identifies%2520the%2520Cellular%2520Pathways%2520Targeted%2520by%2520Antibacterial%2520Molecules%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2013%26volume%3D110%26spage%3D16169%26epage%3D16174%26doi%3D10.1073%2Fpnas.1311066110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lamsa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dorrestein, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pogliano, K.</span></span> <span> </span><span class="NLM_article-title">The <i>Bacillus subtilis</i> Cannibalism Toxin SDP Collapses the Proton Motive Force and Induces Autolysis</span>. <i>Mol. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>84</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">500</span>, <span class="refDoi"> DOI: 10.1111/j.1365-2958.2012.08038.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1111%2Fj.1365-2958.2012.08038.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=22469514" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BC38XnvVGqurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=84&publication_year=2012&pages=486-500&author=A.+Lamsaauthor=W.-T.+Liuauthor=P.+C.+Dorresteinauthor=K.+Pogliano&title=The+Bacillus+subtilis+Cannibalism+Toxin+SDP+Collapses+the+Proton+Motive+Force+and+Induces+Autolysis&doi=10.1111%2Fj.1365-2958.2012.08038.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">The Bacillus subtilis cannibalism toxin SDP collapses the proton motive force and induces autolysis</span></div><div class="casAuthors">Lamsa, Anne; Liu, Wei-Ting; Dorrestein, Pieter C.; Pogliano, Kit</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Microbiology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">84</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">486-500</span>CODEN:
                <span class="NLM_cas:coden">MOMIEE</span>;
        ISSN:<span class="NLM_cas:issn">0950-382X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Bacillus subtilis SDP is a peptide toxin that kills cells outside the biofilm to support continued growth.  The authors show that purified SDP acts like endogenously produced SDP; it delays sporulation, and the SdpI immunity protein confers SDP resistance.  SDP kills a variety of Gram-pos. bacteria in the phylum Firmicutes, as well as Escherichia coli with a compromised outer membrane, suggesting it participates in defense of the B. subtilis biofilm against Gram-pos. bacteria as well as cannibalism.  Fluorescence microscopy reveals that the effect of SDP on cells differs from that of nisin, nigericin, valinomycin and vancomycin-KCl, but resembles that of CCCP, DNP and azide.  Indeed, SDP rapidly collapses the PMF as measured by fluorometry and flow cytometry, which triggers the slower process of autolysis.  This secondary consequence of SDP treatment is not required for cell death since the autolysin-defective lytC, lytD, lytE, lytF strain fails to be lysed but is nevertheless killed by SDP.  Collapsing the PMF is an ideal mechanism for a toxin involved in cannibalism and biofilm defense, since this would incapacitate neighboring cells by inhibiting motility and secretion of proteins and toxins.  It would also induce autolysis in many Gram-pos. species, thereby releasing nutrients that promote biofilm growth.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppl3ScLKne07Vg90H21EOLACvtfcHk0ljyBJfMJNjZ6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XnvVGqurc%253D&md5=fc3cb2d274cb75e1d4358df072777b5e</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1111%2Fj.1365-2958.2012.08038.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1365-2958.2012.08038.x%26sid%3Dliteratum%253Aachs%26aulast%3DLamsa%26aufirst%3DA.%26aulast%3DLiu%26aufirst%3DW.-T.%26aulast%3DDorrestein%26aufirst%3DP.%2BC.%26aulast%3DPogliano%26aufirst%3DK.%26atitle%3DThe%2520Bacillus%2520subtilis%2520Cannibalism%2520Toxin%2520SDP%2520Collapses%2520the%2520Proton%2520Motive%2520Force%2520and%2520Induces%2520Autolysis%26jtitle%3DMol.%2520Microbiol.%26date%3D2012%26volume%3D84%26spage%3D486%26epage%3D500%26doi%3D10.1111%2Fj.1365-2958.2012.08038.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lamsa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez-Garrido, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quach, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riley, E. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pogliano, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pogliano, K.</span></span> <span> </span><span class="NLM_article-title">Rapid Inhibition Profiling in Bacillus subtilisto Identify the Mechanism of Action of New Antimicrobials</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">2222</span>– <span class="NLM_lpage">2231</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b01050</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b01050" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BC28XotFaitrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=2222-2231&author=A.+Lamsaauthor=J.+Lopez-Garridoauthor=D.+Quachauthor=E.+P.+Rileyauthor=J.+Poglianoauthor=K.+Pogliano&title=Rapid+Inhibition+Profiling+in+Bacillus+subtilisto+Identify+the+Mechanism+of+Action+of+New+Antimicrobials&doi=10.1021%2Facschembio.5b01050"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid Inhibition Profiling in Bacillus subtilis to Identify the Mechanism of Action of New Antimicrobials</span></div><div class="casAuthors">Lamsa, Anne; Lopez-Garrido, Javier; Quach, Diana; Riley, Eammon P.; Pogliano, Joe; Pogliano, Kit</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2222-2231</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Increasing antimicrobial resistance has become a major public health crisis.  New antimicrobials with novel mechanisms of action (MOA) are desperately needed.  We previously developed a method, bacterial cytol. profiling (BCP), which utilizes fluorescence microscopy to rapidly identify the MOA of antimicrobial compds.  BCP is based upon our discovery that cells treated with antibiotics affecting different metabolic pathways generate different cytol. signatures, providing quant. information that can be used to det. a compd.'s MOA.  Here, we describe a system, rapid inhibition profiling (RIP), for creating cytol. profiles of new antibiotic targets for which there are currently no chem. inhibitors.  RIP consists of the fast, inducible degrdn. of a target protein followed by BCP.  We demonstrate that degrading essential proteins in the major metabolic pathways for DNA replication, transcription, fatty acid biosynthesis, and peptidoglycan biogenesis in Bacillus subtilis rapidly produces cytol. profiles closely matching that of antimicrobials targeting the same pathways.  Addnl., RIP and antibiotics targeting different steps in fatty acid biosynthesis can be differentiated from each other.  We utilize RIP and BCP to show that the antibacterial MOA of four nonsteroidal anti-inflammatory antibiotics differs from that proposed based on in vitro data.  RIP is a versatile method that will extend our knowledge of phenotypes assocd. with inactivating essential bacterial enzymes and thereby allow for screening for mols. that inhibit novel essential targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoH0kSYirX3HLVg90H21EOLACvtfcHk0ljyBJfMJNjZ6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XotFaitrc%253D&md5=cb2c1f57f8e5c1a249a052ca2efda6f3</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b01050&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b01050%26sid%3Dliteratum%253Aachs%26aulast%3DLamsa%26aufirst%3DA.%26aulast%3DLopez-Garrido%26aufirst%3DJ.%26aulast%3DQuach%26aufirst%3DD.%26aulast%3DRiley%26aufirst%3DE.%2BP.%26aulast%3DPogliano%26aufirst%3DJ.%26aulast%3DPogliano%26aufirst%3DK.%26atitle%3DRapid%2520Inhibition%2520Profiling%2520in%2520Bacillus%2520subtilisto%2520Identify%2520the%2520Mechanism%2520of%2520Action%2520of%2520New%2520Antimicrobials%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26volume%3D11%26spage%3D2222%26epage%3D2231%26doi%3D10.1021%2Facschembio.5b01050" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mohammad, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Younis, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peters, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pogliano, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pogliano, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayhoub, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oldfield, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cushman, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seleem, M. N.</span></span> <span> </span><span class="NLM_article-title">Phenylthiazole Antibacterial Agents Targeting Cell Wall Synthesis Exhibit Potent Activity <i>In Vitro</i> and <i>In Vivo</i> Against Vancomycin-Resistant Enterococci</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2425</span>– <span class="NLM_lpage">2438</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01780</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01780" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjs1elu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=2425-2438&author=H.+Mohammadauthor=W.+Younisauthor=L.+Chenauthor=C.+E.+Petersauthor=J.+Poglianoauthor=K.+Poglianoauthor=B.+Cooperauthor=J.+Zhangauthor=A.+Mayhoubauthor=E.+Oldfieldauthor=M.+Cushmanauthor=M.+N.+Seleem&title=Phenylthiazole+Antibacterial+Agents+Targeting+Cell+Wall+Synthesis+Exhibit+Potent+Activity+In+Vitro+and+In+Vivo+Against+Vancomycin-Resistant+Enterococci&doi=10.1021%2Facs.jmedchem.6b01780"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Phenylthiazole Antibacterial Agents Targeting Cell Wall Synthesis Exhibit Potent Activity in Vitro and in Vivo against Vancomycin-Resistant Enterococci</span></div><div class="casAuthors">Mohammad, Haroon; Younis, Waleed; Chen, Lu; Peters, Christine E.; Pogliano, Joe; Pogliano, Kit; Cooper, Bruce; Zhang, Jianan; Mayhoub, Abdelrahman; Oldfield, Eric; Cushman, Mark; Seleem, Mohamed N.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2425-2438</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The emergence of antibiotic-resistant bacterial species, such as vancomycin-resistant enterococci (VRE), necessitates the development of new antimicrobials.  Here, we investigate the spectrum of antibacterial activity of three phenylthiazole-substituted aminoguanidines.  These compds. possess potent activity against VRE, inhibiting growth of clin. isolates at concns. as low as 0.5 μg/mL.  The compds. exerted a rapid bactericidal effect, targeting cell wall synthesis.  Transposon mutagenesis suggested three possible targets: YubA, YubB (undecaprenyl diphosphate phosphatase (UPPP)), and YubD.  Both UPPP as well as undecaprenyl diphosphate synthase were inhibited by compd. 1.  YubA and YubD are annotated as transporters and may also be targets because 1 collapsed the proton motive force in membrane vesicles.  Using Caenorhabditis elegans, we demonstrate that two compds. (1, 3, at 20 μg/mL) retain potent activity in vivo, significantly reducing the burden of VRE in infected worms.  Taken altogether, the results indicate that compds. 1 and 3 warrant further investigation as novel antibacterial agents against drug-resistant enterococci.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoeil1XLiO43rVg90H21EOLACvtfcHk0ljyBJfMJNjZ6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjs1elu7c%253D&md5=d4bc343d2329d28effe1b1210b1b9cf7</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01780&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01780%26sid%3Dliteratum%253Aachs%26aulast%3DMohammad%26aufirst%3DH.%26aulast%3DYounis%26aufirst%3DW.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DPeters%26aufirst%3DC.%2BE.%26aulast%3DPogliano%26aufirst%3DJ.%26aulast%3DPogliano%26aufirst%3DK.%26aulast%3DCooper%26aufirst%3DB.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DMayhoub%26aufirst%3DA.%26aulast%3DOldfield%26aufirst%3DE.%26aulast%3DCushman%26aufirst%3DM.%26aulast%3DSeleem%26aufirst%3DM.%2BN.%26atitle%3DPhenylthiazole%2520Antibacterial%2520Agents%2520Targeting%2520Cell%2520Wall%2520Synthesis%2520Exhibit%2520Potent%2520Activity%2520In%2520Vitro%2520and%2520In%2520Vivo%2520Against%2520Vancomycin-Resistant%2520Enterococci%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D2425%26epage%3D2438%26doi%3D10.1021%2Facs.jmedchem.6b01780" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nonejuie, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munguia, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hollands, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dam, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumaraswamy, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rivera, H.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corriden, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rohde, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hensler, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burkart, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pogliano, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakoulas, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nizet, V.</span></span> <span> </span><span class="NLM_article-title">Azithromycin Synergizes with Cationic Antimicrobial Peptides to Exert Bactericidal and Therapeutic Activity Against Highly Multidrug-Resistant Gram-Negative Bacterial Pathogens</span>. <i>EBioMedicine</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">690</span>– <span class="NLM_lpage">698</span>, <span class="refDoi"> DOI: 10.1016/j.ebiom.2015.05.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1016%2Fj.ebiom.2015.05.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=26288841" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A280%3ADC%252BC287ks1ygsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2015&pages=690-698&author=L.+Linauthor=P.+Nonejuieauthor=J.+Munguiaauthor=A.+Hollandsauthor=J.+Olsonauthor=Q.+Damauthor=M.+Kumaraswamyauthor=H.+Riveraauthor=R.+Corridenauthor=M.+Rohdeauthor=M.+E.+Henslerauthor=M.+D.+Burkartauthor=J.+Poglianoauthor=G.+Sakoulasauthor=V.+Nizet&title=Azithromycin+Synergizes+with+Cationic+Antimicrobial+Peptides+to+Exert+Bactericidal+and+Therapeutic+Activity+Against+Highly+Multidrug-Resistant+Gram-Negative+Bacterial+Pathogens&doi=10.1016%2Fj.ebiom.2015.05.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Azithromycin Synergizes with Cationic Antimicrobial Peptides to Exert Bactericidal and Therapeutic Activity Against Highly Multidrug-Resistant Gram-Negative Bacterial Pathogens</span></div><div class="casAuthors">Lin Leo; Munguia Jason; Hollands Andrew; Olson Joshua; Dam Quang; Corriden Ross; Hensler Mary E; Sakoulas George; Nonejuie Poochit; Pogliano Joe; Kumaraswamy Monika; Rivera Heriberto Jr; Burkart Michael D; Rohde Manfred; Nizet Victor</div><div class="citationInfo"><span class="NLM_cas:title">EBioMedicine</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">690-8</span>
        ISSN:<span class="NLM_cas:issn">2352-3964</span>.
    </div><div class="casAbstract">Antibiotic resistance poses an increasingly grave threat to the public health.  Of pressing concern, rapid spread of carbapenem-resistance among multidrug-resistant (MDR) Gram-negative rods (GNR) is associated with few treatment options and high mortality rates.  Current antibiotic susceptibility testing guiding patient management is performed in a standardized manner, identifying minimum inhibitory concentrations (MIC) in bacteriologic media, but ignoring host immune factors.  Lacking activity in standard MIC testing, azithromycin (AZM), the most commonly prescribed antibiotic in the U.S., is never recommended for MDR GNR infection.  Here we report a potent bactericidal action of AZM against MDR carbapenem-resistant isolates of Pseudomonas aeruginosa, Klebsiella pneumoniae, and Acinetobacter baumannii.  This pharmaceutical activity is associated with enhanced AZM cell penetration in eukaryotic tissue culture media and striking multi-log-fold synergies with host cathelicidin antimicrobial peptide LL-37 or the last line antibiotic colistin.  Finally, AZM monotherapy exerts clear therapeutic effects in murine models of MDR GNR infection.  Our results suggest that AZM, currently ignored as a treatment option, could benefit patients with MDR GNR infections, especially in combination with colistin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR9RCUhukP3WwMthz5_Rx5rfW6udTcc2eZH61R0qA7P47ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC287ks1ygsQ%253D%253D&md5=22b994576cd66910a7d391084fa3c9df</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2Fj.ebiom.2015.05.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ebiom.2015.05.021%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DL.%26aulast%3DNonejuie%26aufirst%3DP.%26aulast%3DMunguia%26aufirst%3DJ.%26aulast%3DHollands%26aufirst%3DA.%26aulast%3DOlson%26aufirst%3DJ.%26aulast%3DDam%26aufirst%3DQ.%26aulast%3DKumaraswamy%26aufirst%3DM.%26aulast%3DRivera%26aufirst%3DH.%26aulast%3DCorriden%26aufirst%3DR.%26aulast%3DRohde%26aufirst%3DM.%26aulast%3DHensler%26aufirst%3DM.%2BE.%26aulast%3DBurkart%26aufirst%3DM.%2BD.%26aulast%3DPogliano%26aufirst%3DJ.%26aulast%3DSakoulas%26aufirst%3DG.%26aulast%3DNizet%26aufirst%3DV.%26atitle%3DAzithromycin%2520Synergizes%2520with%2520Cationic%2520Antimicrobial%2520Peptides%2520to%2520Exert%2520Bactericidal%2520and%2520Therapeutic%2520Activity%2520Against%2520Highly%2520Multidrug-Resistant%2520Gram-Negative%2520Bacterial%2520Pathogens%26jtitle%3DEBioMedicine%26date%3D2015%26volume%3D2%26spage%3D690%26epage%3D698%26doi%3D10.1016%2Fj.ebiom.2015.05.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nonejuie, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trial, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Newton, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamsa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perera, V. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aguilar, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dorrestein, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pogliano, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pogliano, K.</span></span> <span> </span><span class="NLM_article-title">Application of Bacterial Cytological Profiling to Crude Natural Product Extracts Reveals the Antibacterial Arsenal of <i>Bacillus subtilis</i></span>. <i>J. Antibiot.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">353</span>– <span class="NLM_lpage">361</span>, <span class="refDoi"> DOI: 10.1038/ja.2015.116</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1038%2Fja.2015.116" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=26648120" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BC28Xos1anu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2016&pages=353-361&author=P.+Nonejuieauthor=R.+M.+Trialauthor=G.+L.+Newtonauthor=A.+Lamsaauthor=V.+R.+Pereraauthor=J.+Aguilarauthor=W.-T.+Liuauthor=P.+C.+Dorresteinauthor=J.+Poglianoauthor=K.+Pogliano&title=Application+of+Bacterial+Cytological+Profiling+to+Crude+Natural+Product+Extracts+Reveals+the+Antibacterial+Arsenal+of+Bacillus+subtilis&doi=10.1038%2Fja.2015.116"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Application of bacterial cytological profiling to crude natural product extracts reveals the antibacterial arsenal of Bacillus subtilis</span></div><div class="casAuthors">Nonejuie, Poochit; Trial, Rachelle M.; Newton, Gerald L.; Lamsa, Anne; Ranmali Perera, Varahenage; Aguilar, Julieta; Liu, Wei-Ting; Dorrestein, Pieter C.; Pogliano, Joe; Pogliano, Kit</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Antibiotics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">353-361</span>CODEN:
                <span class="NLM_cas:coden">JANTAJ</span>;
        ISSN:<span class="NLM_cas:issn">0021-8820</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Although most clin. used antibiotics are derived from natural products, identifying new antibacterial mols. from natural product exts. is difficult due to the complexity of these exts. and the limited tools to correlate biol. activity with specific mols.  Here, we show that bacterial cytol. profiling (BCP) provides a rapid method for mechanism of action detn. on plates and in complex natural product exts. and for activity-guided purifn.  We prepd. an ext. from Bacillus subtilis 3610 that killed the Escherichia coli lptD mutant and used BCP to observe two types of bioactivities in the unfractionated ext.: inhibition of translation and permeablization of the cytoplasmic membrane.  We used BCP to guide purifn. of the mols. responsible for each activity, identifying the translation inhibitors bacillaene and bacillaene B (glycosylated bacillaene) and demonstrating that two mols. contribute to cell permeablization, the bacteriocin subtilosin and the cyclic peptide sporulation killing factor.  Our results suggest that bacillaene mediates translational arrest, and show that bacillaene B has a min. inhibitory concn. 10 × higher than unmodified bacillaene.  Finally, we show that BCP can be used to screen strains on an agar plate without the need for ext. prepn., greatly saving time and improving throughput.  Thus, BCP simplifies the isolation of novel natural products, by identifying strains, crude exts. and fractions with interesting bioactivities even when multiple activities are present, allowing investigators to focus labor-intensive steps on those with desired activities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVUxWGiUNr6bVg90H21EOLACvtfcHk0ljB8RojJ9VlMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xos1anu7g%253D&md5=c842bc1c7746f8cd189cbc1cd0b39c7b</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1038%2Fja.2015.116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fja.2015.116%26sid%3Dliteratum%253Aachs%26aulast%3DNonejuie%26aufirst%3DP.%26aulast%3DTrial%26aufirst%3DR.%2BM.%26aulast%3DNewton%26aufirst%3DG.%2BL.%26aulast%3DLamsa%26aufirst%3DA.%26aulast%3DPerera%26aufirst%3DV.%2BR.%26aulast%3DAguilar%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DW.-T.%26aulast%3DDorrestein%26aufirst%3DP.%2BC.%26aulast%3DPogliano%26aufirst%3DJ.%26aulast%3DPogliano%26aufirst%3DK.%26atitle%3DApplication%2520of%2520Bacterial%2520Cytological%2520Profiling%2520to%2520Crude%2520Natural%2520Product%2520Extracts%2520Reveals%2520the%2520Antibacterial%2520Arsenal%2520of%2520Bacillus%2520subtilis%26jtitle%3DJ.%2520Antibiot.%26date%3D2016%26volume%3D69%26spage%3D353%26epage%3D361%26doi%3D10.1038%2Fja.2015.116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hamby, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Connolly, C. J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schroeder, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winters, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Showalter, H. D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panek, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Major, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olsewski, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryan, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dahring, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, G. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keiser, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraker, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slintak, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nelson, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fry, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradford, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hallak, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doherty, A. M.</span></span> <span> </span><span class="NLM_article-title">Structure-Activity Relationships for a Novel Series of Pyrido[2,3-d]pyrimidine Tyrosine Kinase Inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">2296</span>– <span class="NLM_lpage">2303</span>, <span class="refDoi"> DOI: 10.1021/jm970367n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm970367n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADyaK2sXktlKisLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=1997&pages=2296-2303&author=J.+M.+Hambyauthor=C.+J.+C.+Connollyauthor=M.+C.+Schroederauthor=R.+T.+Wintersauthor=H.+D.+H.+Showalterauthor=R.+L.+Panekauthor=T.+C.+Majorauthor=B.+Olsewskiauthor=M.+J.+Ryanauthor=T.+Dahringauthor=G.+H.+Luauthor=J.+Keiserauthor=A.+Amarauthor=C.+Shenauthor=A.+J.+Krakerauthor=V.+Slintakauthor=J.+M.+Nelsonauthor=D.+W.+Fryauthor=L.+Bradfordauthor=H.+Hallakauthor=A.+M.+Doherty&title=Structure-Activity+Relationships+for+a+Novel+Series+of+Pyrido%5B2%2C3-d%5Dpyrimidine+Tyrosine+Kinase+Inhibitors&doi=10.1021%2Fjm970367n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Activity Relationships for a Novel Series of Pyrido[2,3-d]pyrimidine Tyrosine Kinase Inhibitors</span></div><div class="casAuthors">Hamby, James M.; Connolly, Cleo J. C.; Schroeder, Mel C.; Winters, R. Thomas; Showalter, H. D. Hollis; Panek, Robert L.; Major, Terry C.; Olsewski, Bronislawa; Ryan, Michael J.; Dahring, Tawny; Lu, Gina H.; Keiser, Joan; Amar, Aneesa; Shen, Cindy; Kraker, Alan J.; Slintak, Veronika; Nelson, James M.; Fry, David W.; Bradford, Laura; Hallak, Hussein; Doherty, Annette M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">2296-2303</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Screening of a compd. library for inhibitors of the fibroblast growth factor (FGFr) and platelet-derived growth factor (PDGFr) receptor tyrosine kinases led to the development of a novel series of ATP competitive pyrido[2,3-d]pyrimidine tyrosine kinase inhibitors.  The initial lead, 1-[2-amino-6-(2,6-dichlorophenyl)pyrido[2,3-d]pyrimidin-7-yl]-3-tert-butylurea (I; PD-089828), was found to be a broadly active tyrosine kinase inhibitor.  I inhibited the PDGFr, FGFr, EGFr, and c-src tyrosine kinases with IC50 values of 1.11, 0.13, 0.45, and 0.22 μM, resp.  Subsequent SAR studies led to the synthesis of new analogs with improved potency, soly., and bioavailability relative to the initial lead.  For example, the introduction of a [4-(diethylamino)butyl]amino side chain into the 2-position of I afforded a compd. (II) with enhanced potency and bioavailability.  II inhibited PDGF-stimulated vascular smooth muscle cell proliferation with an IC50 of 0.3 μM.  Furthermore, replacement of the 6-(2,6-dichlorophenyl) moiety of I with a 6-(3',5'-dimethoxyphenyl) functionality produced a highly selective FGFr tyrosine kinase inhibitor (III).  III inhibited the FGFr tyrosine kinase with an IC50 of 0.060 μM, whereas IC50s for the inhibition of the PDGFr, FGFr, EGFr, c-src, and InsR tyrosine kinases for this compd. were all greater than 50 μM.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJdLcE9N0L8rVg90H21EOLACvtfcHk0ljB8RojJ9VlMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXktlKisLs%253D&md5=f68e00a123b8b450e647761ac1421d77</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Fjm970367n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm970367n%26sid%3Dliteratum%253Aachs%26aulast%3DHamby%26aufirst%3DJ.%2BM.%26aulast%3DConnolly%26aufirst%3DC.%2BJ.%2BC.%26aulast%3DSchroeder%26aufirst%3DM.%2BC.%26aulast%3DWinters%26aufirst%3DR.%2BT.%26aulast%3DShowalter%26aufirst%3DH.%2BD.%2BH.%26aulast%3DPanek%26aufirst%3DR.%2BL.%26aulast%3DMajor%26aufirst%3DT.%2BC.%26aulast%3DOlsewski%26aufirst%3DB.%26aulast%3DRyan%26aufirst%3DM.%2BJ.%26aulast%3DDahring%26aufirst%3DT.%26aulast%3DLu%26aufirst%3DG.%2BH.%26aulast%3DKeiser%26aufirst%3DJ.%26aulast%3DAmar%26aufirst%3DA.%26aulast%3DShen%26aufirst%3DC.%26aulast%3DKraker%26aufirst%3DA.%2BJ.%26aulast%3DSlintak%26aufirst%3DV.%26aulast%3DNelson%26aufirst%3DJ.%2BM.%26aulast%3DFry%26aufirst%3DD.%2BW.%26aulast%3DBradford%26aufirst%3DL.%26aulast%3DHallak%26aufirst%3DH.%26aulast%3DDoherty%26aufirst%3DA.%2BM.%26atitle%3DStructure-Activity%2520Relationships%2520for%2520a%2520Novel%2520Series%2520of%2520Pyrido%255B2%252C3-d%255Dpyrimidine%2520Tyrosine%2520Kinase%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1997%26volume%3D40%26spage%3D2296%26epage%3D2303%26doi%3D10.1021%2Fjm970367n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hernandez, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crépin, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palencia, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cusack, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akama, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freund, Y. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meewan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rock, F. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sexton, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheoran, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nieman, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anugula, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keramane, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savariraj, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Subedi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Leary, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simon, N. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Marsh, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mushtaq, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warner, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Livermore, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alley, M. R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plattner, J. J.</span></span> <span> </span><span class="NLM_article-title">Discovery of a Novel Class of Boron-Based Antibacterials with Activity Against Gram-Negative Bacteria</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">1394</span>– <span class="NLM_lpage">1403</span>, <span class="refDoi"> DOI: 10.1128/AAC.02058-12</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1128%2FAAC.02058-12" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=23295920" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BC3sXltV2htLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2013&pages=1394-1403&author=V.+Hernandezauthor=T.+Cr%C3%A9pinauthor=A.+Palenciaauthor=S.+Cusackauthor=T.+Akamaauthor=S.+J.+Bakerauthor=W.+Buauthor=L.+Fengauthor=Y.+R.+Freundauthor=L.+Liuauthor=M.+Meewanauthor=M.+Mohanauthor=W.+Maoauthor=F.+L.+Rockauthor=H.+Sextonauthor=A.+Sheoranauthor=Y.+Zhangauthor=Y.-K.+Zhangauthor=Y.+Zhouauthor=J.+A.+Niemanauthor=M.+R.+Anugulaauthor=E.+M.+Keramaneauthor=K.+Savarirajauthor=D.+S.+Reddyauthor=R.+Sharmaauthor=R.+Subediauthor=R.+Singhauthor=A.+O%E2%80%99Learyauthor=N.+L.+Simonauthor=P.+L.+De+Marshauthor=S.+Mushtaqauthor=M.+Warnerauthor=D.+M.+Livermoreauthor=M.+R.+K.+Alleyauthor=J.+J.+Plattner&title=Discovery+of+a+Novel+Class+of+Boron-Based+Antibacterials+with+Activity+Against+Gram-Negative+Bacteria&doi=10.1128%2FAAC.02058-12"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a novel class of boron-based antibacterials with activity against Gram-negative bacteria</span></div><div class="casAuthors">Hernandez, Vincent; Crepin, Thibaut; Palencia, Andres; Cusack, Stephen; Akama, Tsutomu; Baker, Stephen J.; Bu, Wei; Feng, Lisa; Freund, Yvonne R.; Liu, Liang; Meewan, Maliwan; Mohan, Manisha; Mao, Weimin; Rock, Fernando L.; Sexton, Holly; Sheoran, Anita; Zhang, Yanchen; Zhang, Yong-Kang; Zhou, Yasheen; Nieman, James A.; Anugula, Mahipal Reddy; Keramane, El Mehdi; Savariraj, Kingsley; Reddy, D. Shekhar; Sharma, Rashmi; Subedi, Rajendra; Singh, Rajeshwar; O'Leary, Ann; Simon, Nerissa L.; De Marsh, Peter L.; Mushtaq, Shazad; Warner, Marina; Livermore, David M.; Alley, M. R. K.; Plattner, Jacob J.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1394-1403</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Gram-neg. bacteria cause approx. 70% of the infections in intensive care units.  A growing no. of bacterial isolates responsible for these infections are resistant to currently available antibiotics and to many in development.  Most agents under development are modifications of existing drug classes, which only partially overcome existing resistance mechanisms.  Therefore, new classes of Gram-neg. antibacterials with truly novel modes of action are needed to circumvent these existing resistance mechanisms.  We have previously identified a new a way to inhibit an aminoacyl-tRNA synthetase, leucyl-tRNA synthetase (LeuRS), in fungi via the oxaborole tRNA trapping (OBORT) mechanism.  Herein, we show how we have modified the OBORT mechanism using a structure-guided approach to develop a new boron-based antibiotic class, the aminomethylbenzoxaboroles, which inhibit bacterial leucyl-tRNA synthetase and have activity against Gram-neg. bacteria by largely evading the main efflux mechanisms in Escherichia coli and Pseudomonas aeruginosa.  The lead analog, AN3365, is active against Gram-neg. bacteria, including Enterobacteriaceae bearing NDM-1 and KPC carbapenemases, as well as P. aeruginosa.  This novel boron-based antibacterial, AN3365, has good mouse pharmacokinetics and was efficacious against E. coli and P. aeruginosa in murine thigh infection models, which suggest that this novel class of antibacterials has the potential to address this unmet medical need.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWl5Oj0jzVnLVg90H21EOLACvtfcHk0lg1TY5Vn_YhNw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXltV2htLg%253D&md5=7ed74bde2bacfa4ae20abb50b745c57d</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1128%2FAAC.02058-12&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.02058-12%26sid%3Dliteratum%253Aachs%26aulast%3DHernandez%26aufirst%3DV.%26aulast%3DCr%25C3%25A9pin%26aufirst%3DT.%26aulast%3DPalencia%26aufirst%3DA.%26aulast%3DCusack%26aufirst%3DS.%26aulast%3DAkama%26aufirst%3DT.%26aulast%3DBaker%26aufirst%3DS.%2BJ.%26aulast%3DBu%26aufirst%3DW.%26aulast%3DFeng%26aufirst%3DL.%26aulast%3DFreund%26aufirst%3DY.%2BR.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DMeewan%26aufirst%3DM.%26aulast%3DMohan%26aufirst%3DM.%26aulast%3DMao%26aufirst%3DW.%26aulast%3DRock%26aufirst%3DF.%2BL.%26aulast%3DSexton%26aufirst%3DH.%26aulast%3DSheoran%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DY.-K.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DNieman%26aufirst%3DJ.%2BA.%26aulast%3DAnugula%26aufirst%3DM.%2BR.%26aulast%3DKeramane%26aufirst%3DE.%2BM.%26aulast%3DSavariraj%26aufirst%3DK.%26aulast%3DReddy%26aufirst%3DD.%2BS.%26aulast%3DSharma%26aufirst%3DR.%26aulast%3DSubedi%26aufirst%3DR.%26aulast%3DSingh%26aufirst%3DR.%26aulast%3DO%25E2%2580%2599Leary%26aufirst%3DA.%26aulast%3DSimon%26aufirst%3DN.%2BL.%26aulast%3DDe%2BMarsh%26aufirst%3DP.%2BL.%26aulast%3DMushtaq%26aufirst%3DS.%26aulast%3DWarner%26aufirst%3DM.%26aulast%3DLivermore%26aufirst%3DD.%2BM.%26aulast%3DAlley%26aufirst%3DM.%2BR.%2BK.%26aulast%3DPlattner%26aufirst%3DJ.%2BJ.%26atitle%3DDiscovery%2520of%2520a%2520Novel%2520Class%2520of%2520Boron-Based%2520Antibacterials%2520with%2520Activity%2520Against%2520Gram-Negative%2520Bacteria%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2013%26volume%3D57%26spage%3D1394%26epage%3D1403%26doi%3D10.1128%2FAAC.02058-12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lomovskaya, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoshino, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishida, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mistry, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warren, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boyer, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chamberland, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, V. J.</span></span> <span> </span><span class="NLM_article-title">Use of a Genetic Approach to Evaluate the Consequences of Inhibition of Efflux Pumps in <i>Pseudomonas aeruginosa</i></span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">1340</span>– <span class="NLM_lpage">1346</span>, <span class="refDoi"> DOI: 10.1128/AAC.43.6.1340</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1128%2FAAC.43.6.1340" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10348749" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADyaK1MXjs12ns70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=1999&pages=1340-1346&author=O.+Lomovskayaauthor=A.+Leeauthor=K.+Hoshinoauthor=H.+Ishidaauthor=A.+Mistryauthor=M.+S.+Warrenauthor=E.+Boyerauthor=S.+Chamberlandauthor=V.+J.+Lee&title=Use+of+a+Genetic+Approach+to+Evaluate+the+Consequences+of+Inhibition+of+Efflux+Pumps+in+Pseudomonas+aeruginosa&doi=10.1128%2FAAC.43.6.1340"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Use of a genetic approach to evaluate the consequences of inhibition of efflux pumps in Pseudomonas aeruginosa</span></div><div class="casAuthors">Lomovskaya, Olga; Lee, Angela; Hoshino, Kazuki; Ishida, Hiroko; Mistry, Anita; Warren, Mark S.; Boyer, Eric; Chamberland, Suzanne; Lee, Ving J.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1340-1346</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Drug efflux pumps in P. aeruginosa were evaluated as potential targets for antibacterial therapy.  The potential effects of pump inhibition on susceptibility to fluoroquinolone antibiotics were studied with isogenic strains that overexpress or lack individual efflux pumps and that have various combinations of efflux-and target-mediated mutations.  Deletions in 3 efflux pump operons were constructed.  As expected, deletion of the MexAB-OprM efflux pump decreased resistance to fluoroquinolones in the wild-type P. aeruginosa (16-fold redn. for levofloxacin [LVX]) or in the strain that overexpressed mexAB-oprM operon (64-fold redn. for LVX).  In addn. to that, resistance to LVX was significantly reduced even for the strains carrying target mutations (64-fold for strains for which LVX MICs were >4 μg/mL).  The frequencies of emergence of LVX-resistant variants from different deletion mutants and the wild-type strain also were studied.  Deletion of individual pumps or pairs of the pumps did not significantly affect the frequency of emergence of resistant variants (at 4 × the MIC for the wild-type strain) compared to that for the wild type (10-6 to 10-7).  In the case of the strain with a triple deletion, the frequency of spontaneous mutants was undetectable (<10-11).  In summary, inhibition of drug efflux pumps would (i) significantly decrease the level of intrinsic resistance, (ii) reverse acquired resistance, and (iii) result in a decreased frequency of emergence of P. aeruginosa strains highly resistant to fluoroquinolones in clin. settings.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqc1gq1c_6DILVg90H21EOLACvtfcHk0lg1TY5Vn_YhNw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXjs12ns70%253D&md5=538a0e96bed5be6be50833f29274057e</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1128%2FAAC.43.6.1340&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.43.6.1340%26sid%3Dliteratum%253Aachs%26aulast%3DLomovskaya%26aufirst%3DO.%26aulast%3DLee%26aufirst%3DA.%26aulast%3DHoshino%26aufirst%3DK.%26aulast%3DIshida%26aufirst%3DH.%26aulast%3DMistry%26aufirst%3DA.%26aulast%3DWarren%26aufirst%3DM.%2BS.%26aulast%3DBoyer%26aufirst%3DE.%26aulast%3DChamberland%26aufirst%3DS.%26aulast%3DLee%26aufirst%3DV.%2BJ.%26atitle%3DUse%2520of%2520a%2520Genetic%2520Approach%2520to%2520Evaluate%2520the%2520Consequences%2520of%2520Inhibition%2520of%2520Efflux%2520Pumps%2520in%2520Pseudomonas%2520aeruginosa%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D1999%26volume%3D43%26spage%3D1340%26epage%3D1346%26doi%3D10.1128%2FAAC.43.6.1340" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tian, Z.-X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mac Aogáin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Connor, H. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fargier, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mooij, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Gara, F.</span></span> <span> </span><span class="NLM_article-title">MexT Modulates Virulence Determinants in <i>Pseudomonas aeruginosa</i> Independent of the MexEF-OprN Efflux Pump</span>. <i>Microb. Pathog.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">237</span>– <span class="NLM_lpage">241</span>, <span class="refDoi"> DOI: 10.1016/j.micpath.2009.08.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1016%2Fj.micpath.2009.08.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=19683048" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFyntLnI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2009&pages=237-241&author=Z.-X.+Tianauthor=M.+Mac+Aog%C3%A1inauthor=H.+F.+O%E2%80%99Connorauthor=E.+Fargierauthor=M.+J.+Mooijauthor=C.+Adamsauthor=Y.-P.+Wangauthor=F.+O%E2%80%99Gara&title=MexT+Modulates+Virulence+Determinants+in+Pseudomonas+aeruginosa+Independent+of+the+MexEF-OprN+Efflux+Pump&doi=10.1016%2Fj.micpath.2009.08.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">MexT modulates virulence determinants in Pseudomonas aeruginosa independent of the MexEF-OprN efflux pump</span></div><div class="casAuthors">Tian, Zhe-Xian; MacAogain, Micheal; O'Connor, Hazel F.; Fargier, Emilie; Mooij, Marlies J.; Adams, Claire; Wang, Yi-Ping; O'Gara, Fergal</div><div class="citationInfo"><span class="NLM_cas:title">Microbial Pathogenesis</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">237-241</span>CODEN:
                <span class="NLM_cas:coden">MIPAEV</span>;
        ISSN:<span class="NLM_cas:issn">0882-4010</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">In the human pathogen Pseudomonas aeruginosa, the LysR-family regulator MexT modulates the induction of the tripartite MexEF-OprN resistance nodulation-division multi-drug efflux system resulting in increased resistance to diverse antibiotics.  The MexEF-OprN system is normally quiescent in wild-type cells, but is highly induced in nfxC-type phenotypic mutants in a MexT dependent manner.  In addn. to antibiotic resistance, induction of mexEF-oprN in nfxC-type mutants has been linked to reduced levels of homoserine lactone-dependent virulence traits, including pyocyanin, elastase, rhamnolipids and PQS and to reduced expression of type three secretion effector proteins.  In this study, MexT is overexpressed in wild-type PAO1 and an isogenic mexEF deletion mutant to det. if MexT regulates diverse virulence phenotypes dependent or independent of MexEF-OprN.  It is shown that MexT regulates type three secretion, pyocyanin prodn. and early surface attachment independent of MexEF-OprN.  In contrast, MexT modulation of the expression of the virulence genes rhlA, lasB and hcnB is dependent on MexEF-OprN, which apparently mediates these effects via efflux of cell-signaling intermediates.  The data presented demonstrates that MexT may play a more global role in modulating P. aeruginosa virulence than previously reported and suggests that MexT regulates diverse targets that mediate phenotypic alterations independent of MexEF-OprN.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoyGW6fcwbDWbVg90H21EOLACvtfcHk0lhfP3chRfoxNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFyntLnI&md5=f7e94dd750d16e6df937a9d4c2cddd1d</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1016%2Fj.micpath.2009.08.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.micpath.2009.08.003%26sid%3Dliteratum%253Aachs%26aulast%3DTian%26aufirst%3DZ.-X.%26aulast%3DMac%2BAog%25C3%25A1in%26aufirst%3DM.%26aulast%3DO%25E2%2580%2599Connor%26aufirst%3DH.%2BF.%26aulast%3DFargier%26aufirst%3DE.%26aulast%3DMooij%26aufirst%3DM.%2BJ.%26aulast%3DAdams%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DY.-P.%26aulast%3DO%25E2%2580%2599Gara%26aufirst%3DF.%26atitle%3DMexT%2520Modulates%2520Virulence%2520Determinants%2520in%2520Pseudomonas%2520aeruginosa%2520Independent%2520of%2520the%2520MexEF-OprN%2520Efflux%2520Pump%26jtitle%3DMicrob.%2520Pathog.%26date%3D2009%26volume%3D47%26spage%3D237%26epage%3D241%26doi%3D10.1016%2Fj.micpath.2009.08.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tian, Z.-X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fargier, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mac Aogáin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Gara, F.</span></span> <span> </span><span class="NLM_article-title">Transcriptome Profiling Defines a Novel Regulon Modulated by the LysR-Type Transcriptional Regulator MexT in <i>Pseudomonas aeruginosa</i></span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">7546</span>– <span class="NLM_lpage">7559</span>, <span class="refDoi"> DOI: 10.1093/nar/gkp828</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1093%2Fnar%2Fgkp828" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=19846594" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFyktrzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2009&pages=7546-7559&author=Z.-X.+Tianauthor=E.+Fargierauthor=M.+Mac+Aog%C3%A1inauthor=C.+Adamsauthor=Y.-P.+Wangauthor=F.+O%E2%80%99Gara&title=Transcriptome+Profiling+Defines+a+Novel+Regulon+Modulated+by+the+LysR-Type+Transcriptional+Regulator+MexT+in+Pseudomonas+aeruginosa&doi=10.1093%2Fnar%2Fgkp828"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Transcriptome profiling defines a novel regulon modulated by the LysR-type transcriptional regulator MexT in Pseudomonas aeruginosa</span></div><div class="casAuthors">Tian, Zhe-Xian; Fargier, Emilie; MacAogain, Micheal; Adams, Claire; Wang, Yi-Ping; O'Gara, Fergal</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">7546-7559</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The LysR-family regulator MexT modulates the expression of the MexEF-OprN efflux system in the human pathogen Pseudomonas aeruginosa.  Recently, we demonstrated that MexT regulates certain virulence phenotypes, including the type-three secretion system and early attachment independent of its role in regulating MexEF-OprN.  In this study, transcriptome profiling was utilized to investigate the global nature of MexT regulation in P. aeruginosa PAO1 and an isogenic mexEF mutant.  Twelve genes of unknown function were highly induced by overexpressing MexT independent of MexEF-OprN.  A well-conserved DNA motif was identified in the upstream regulatory region of nine of these genes and upstream of mexE.  Reporter fusion anal. demonstrated that the expression of the genes was significantly induced by MexT in P. aeruginosa and a heterogenous Escherichia coli strain and that the conserved sequence was required for this induction.  The conserved DNA motif was further characterized as the MexT binding site by site-directed mutagenesis and electrophoretic mobility shift assays.  Genes contg. this conserved regulatory sequence were identified across other Pseudomonas species, and their expression was activated by MexT.  Thus, a novel regulon directly modulated by MexT, that includes but is independent of mexEF-oprN, has been identified.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_qS5Q2HRJDrVg90H21EOLACvtfcHk0lhfP3chRfoxNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFyktrzE&md5=3669c88ab1757d7c607abf66d9fbb486</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkp828&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkp828%26sid%3Dliteratum%253Aachs%26aulast%3DTian%26aufirst%3DZ.-X.%26aulast%3DFargier%26aufirst%3DE.%26aulast%3DMac%2BAog%25C3%25A1in%26aufirst%3DM.%26aulast%3DAdams%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DY.-P.%26aulast%3DO%25E2%2580%2599Gara%26aufirst%3DF.%26atitle%3DTranscriptome%2520Profiling%2520Defines%2520a%2520Novel%2520Regulon%2520Modulated%2520by%2520the%2520LysR-Type%2520Transcriptional%2520Regulator%2520MexT%2520in%2520Pseudomonas%2520aeruginosa%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2009%26volume%3D37%26spage%3D7546%26epage%3D7559%26doi%3D10.1093%2Fnar%2Fgkp828" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Luong, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shogan, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaborin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belogortseva, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shrout, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaborina, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alverdy, J. C.</span></span> <span> </span><span class="NLM_article-title">Emergence of the P2 Phenotype in <i>Pseudomonas aeruginosa</i> PAO1 Strains Involves Various Mutations in MexT or MexF</span>. <i>J. Bacteriol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>196</i></span>,  <span class="NLM_fpage">504</span>– <span class="NLM_lpage">513</span>, <span class="refDoi"> DOI: 10.1128/JB.01050-13</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1128%2FJB.01050-13" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=24244000" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFOlu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=196&publication_year=2014&pages=504-513&author=P.+M.+Luongauthor=B.+D.+Shoganauthor=A.+Zaborinauthor=N.+Belogortsevaauthor=J.+D.+Shroutauthor=O.+Zaborinaauthor=J.+C.+Alverdy&title=Emergence+of+the+P2+Phenotype+in+Pseudomonas+aeruginosa+PAO1+Strains+Involves+Various+Mutations+in+MexT+or+MexF&doi=10.1128%2FJB.01050-13"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Emergence of the P2 phenotype in Pseudomonas aeruginosa PAO1 strains involves various mutations in mexT or mexF</span></div><div class="casAuthors">Luong, Preston M.; Shogan, Benjamin D.; Zaborin, Alexander; Belogortseva, Natalia; Shrout, Joshua D.; Zaborina, Olga; Alverdy, John C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Bacteriology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">196</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">504-513, 11 pp.</span>CODEN:
                <span class="NLM_cas:coden">JOBAAY</span>;
        ISSN:<span class="NLM_cas:issn">1098-5530</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">We recently demonstrated that Pseudomonas aeruginosa PAO1 undergoes a pronounced phenotypic change when introduced into the intestines of rats during surgical injury.  Recovered strains displayed a specific phenotype (termed the P2 phenotype) characterized by altered pyocyanin prodn., high collagenase activity, high swarming motility, low resistance to chloramphenicol, and increased killing of Caenorhabditis elegans compared to the inoculating strain (termed the P1 phenotype).  The aims of this study were to characterize the differences between the P. aeruginosa P1 and P2 phenotypes in quorum sensing and competitiveness.  We then detd. the presence of the P2 phenotype among PAO1 strains from various labs.  Results demonstrated that P2 cells display accelerated growth during early exponential phase and early activation of quorum-sensing systems and overcome the growth of P1 cells in a mixed population.  Among eight PAO1 strains obtained from different labs., four exhibited the P2 phenotype.  Of 27 mutants analyzed from the P. aeruginosa MPAO1 transposon library, 25 displayed P2 phenotypes.  The P2 phenotype in both cases correlated with a lack of expression of mexE or mexF due to mutations in mexT and mexF genes.  In summary, strains possessing the P2 phenotype are distributed among PAO1 strains commonly used across a variety of research labs.  Genetically, they are characterized by various mutations in mexT or mexF.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfJd5RNfcvuLVg90H21EOLACvtfcHk0lhfP3chRfoxNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFOlu78%253D&md5=0013229194244ab996822e5854549998</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1128%2FJB.01050-13&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJB.01050-13%26sid%3Dliteratum%253Aachs%26aulast%3DLuong%26aufirst%3DP.%2BM.%26aulast%3DShogan%26aufirst%3DB.%2BD.%26aulast%3DZaborin%26aufirst%3DA.%26aulast%3DBelogortseva%26aufirst%3DN.%26aulast%3DShrout%26aufirst%3DJ.%2BD.%26aulast%3DZaborina%26aufirst%3DO.%26aulast%3DAlverdy%26aufirst%3DJ.%2BC.%26atitle%3DEmergence%2520of%2520the%2520P2%2520Phenotype%2520in%2520Pseudomonas%2520aeruginosa%2520PAO1%2520Strains%2520Involves%2520Various%2520Mutations%2520in%2520MexT%2520or%2520MexF%26jtitle%3DJ.%2520Bacteriol.%26date%3D2014%26volume%3D196%26spage%3D504%26epage%3D513%26doi%3D10.1128%2FJB.01050-13" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group">CSLI</span>.  <i>CSLI M07-A10: Methods for Dilution Antimicrobial
Susceptibility
Tests for Bacteria That Grow Aerobically; Approved Standard-Tenth
Edition</i>. <span class="NLM_publisher-name">Clinical and Laboratory Standards
Institute</span>: <span class="NLM_publisher-loc">Wayne, PA</span>; <span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=CSLI&title=CSLI+M07-A10%3A+Methods+for+Dilution+Antimicrobial%0ASusceptibility%0ATests+for+Bacteria+That+Grow+Aerobically%3B+Approved+Standard-Tenth%0AEdition"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3D%26btitle%3DCSLI%2520M07-A10%253A%2520Methods%2520for%2520Dilution%2520Antimicrobial%250ASusceptibility%250ATests%2520for%2520Bacteria%2520That%2520Grow%2520Aerobically%253B%2520Approved%2520Standard-Tenth%250AEdition%26pub%3DClinical%2520and%2520Laboratory%2520Standards%250AInstitute%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">XDS</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1107/S0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=XDS&doi=10.1107%2FS0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0lhRPus5T_bKvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXDS%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2FS0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P.</span></span> <span> </span><span class="NLM_article-title">Scaling and Assessment of Data Quality</span>. <i>Acta Crystallogr., Sect. D: Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">72</span>– <span class="NLM_lpage">82</span>, <span class="refDoi"> DOI: 10.1107/S0907444905036693</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1107%2FS0907444905036693" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=16369096" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlant7jM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2006&pages=72-82&author=P.+Evans&title=Scaling+and+Assessment+of+Data+Quality&doi=10.1107%2FS0907444905036693"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Scaling and assessment of data quality</span></div><div class="casAuthors">Evans, Philip</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">D62</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">72-82</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">The various phys. factors affecting measured diffraction intensities are discussed, as are the scaling models which may be used to put the data on a consistent scale.  After scaling, the intensities can be analyzed to set the real resoln. of the data set, to detect bad regions (e.g. bad images), to analyze radiation damage and to assess the overall quality of the data set.  The significance of any anomalous signal may be assessed by probability and correlation anal.  The algorithms used by the CCP4 scaling program SCALA are described.  A requirement for the scaling and merging of intensities is knowledge of the Laue group and point-group symmetries: the possible symmetry of the diffraction pattern may be detd. from scores such as correlation coeffs. between observations which might be symmetry-related.  These scoring functions are implemented in a new program POINTLESS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGravLSLU62RhLVg90H21EOLACvtfcHk0lhRPus5T_bKvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlant7jM&md5=293d3876e534c0c57813990515bb3c76</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1107%2FS0907444905036693&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444905036693%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DP.%26atitle%3DScaling%2520and%2520Assessment%2520of%2520Data%2520Quality%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Struct.%2520Biol.%26date%3D2006%26volume%3D62%26spage%3D72%26epage%3D82%26doi%3D10.1107%2FS0907444905036693" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunkóczi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echols, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oeffner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, P. H.</span></span> <span> </span><span class="NLM_article-title">PHENIX:A Comprehensive Python-Based System for Macromolecular Structure Solution</span>. <i>Acta Crystallogr., Sect. D: Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">213</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1107/S0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunk%C3%B3cziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.-W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3AA+Comprehensive+Python-Based+System+for+Macromolecular+Structure+Solution&doi=10.1107%2FS0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0lhRPus5T_bKvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunk%25C3%25B3czi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.-W.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253AA%2520Comprehensive%2520Python-Based%2520System%2520for%2520Macromolecular%2520Structure%2520Solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Struct.%2520Biol.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2FS0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group">Collaborative
Computation Project, Number 4</span> <span> </span><span class="NLM_article-title">The CCP4 Suite: Programs for Protein Crystallography</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">760</span>– <span class="NLM_lpage">763</span>, <span class="refDoi"> DOI: 10.1107/S0907444994003112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1107%2FS0907444994003112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=15299374" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=1994&pages=760-763&author=Collaborative%0AComputation+Project%2C+Number+4&title=The+CCP4+Suite%3A+Programs+for+Protein+Crystallography&doi=10.1107%2FS0907444994003112"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1107%2FS0907444994003112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444994003112%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DThe%2520CCP4%2520Suite%253A%2520Programs%2520for%2520Protein%2520Crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1994%26volume%3D50%26spage%3D760%26epage%3D763%26doi%3D10.1107%2FS0907444994003112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Painter, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merritt, E. A.</span></span> <span> </span><span class="NLM_article-title">TLSMD Web Server for the Generation of Multi-Group TLS Models</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">109</span>– <span class="NLM_lpage">11</span>, <span class="refDoi"> DOI: 10.1107/S0021889805038987</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1107%2FS0021889805038987" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BD28XmvVyjsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2006&pages=109-11&author=J.+Painterauthor=E.+A.+Merritt&title=TLSMD+Web+Server+for+the+Generation+of+Multi-Group+TLS+Models&doi=10.1107%2FS0021889805038987"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">TLSMD web server for the generation of multi-group TLS models</span></div><div class="casAuthors">Painter, Jay; Merritt, Ethan A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">109-111</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">The TLSMD web server exts. information about dynamic properties of a protein based on information derived from a single-crystal structure.  It does so by analyzing the spatial distribution of individual at. thermal parameters present in an input structural model.  The server partitions the protein structure into multiple, contiguous chain segments, each segment corresponding to one group in a multi-group description of the protein's overall dynamic motion.  For each polypeptide chain of the input protein, the anal. generates the optimal partition into two segments, three segments,... up to 20 segments.  Each such partition is optimal in the sense that it is the best approxn. of the overall spatial distribution of input thermal parameters in terms of N chain segments, each acting as a rigid group undergoing TLS (translation/libration/screw) motion.  This multi-group TLS model may be used as a starting point for further crystallog. refinement, or as the basis for analyzing inter-domain and other large-scale motions implied by the crystal structure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhkwf2sopjVrVg90H21EOLACvtfcHk0liDYDz--qldag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XmvVyjsA%253D%253D&md5=5d16d5d6310baa5434064bdf0ec81e24</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1107%2FS0021889805038987&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889805038987%26sid%3Dliteratum%253Aachs%26aulast%3DPainter%26aufirst%3DJ.%26aulast%3DMerritt%26aufirst%3DE.%2BA.%26atitle%3DTLSMD%2520Web%2520Server%2520for%2520the%2520Generation%2520of%2520Multi-Group%2520TLS%2520Models%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2006%26volume%3D39%26spage%3D109%26epage%3D11%26doi%3D10.1107%2FS0021889805038987" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Coot: Model-Building Tools for Molecular Graphics</span>. <i>Acta Crystallogr., Sect. D: Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2126</span>– <span class="NLM_lpage">2132</span>, <span class="refDoi"> DOI: 10.1107/S0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+Model-Building+Tools+for+Molecular+Graphics&doi=10.1107%2FS0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0liDYDz--qldag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520Model-Building%2520Tools%2520for%2520Molecular%2520Graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Struct.%2520Biol.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2FS0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 8 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Kristen A. Muñoz, <span class="NLM_string-name hlFld-ContribAuthor">Paul J. Hergenrother</span>. </span><span class="cited-content_cbyCitation_article-title">Facilitating Compound Entry as a Means to Discover Antibiotics for Gram-Negative Bacteria. </span><span class="cited-content_cbyCitation_journal-name">Accounts of Chemical Research</span><span> <strong>2021,</strong> <em>54 </em>
                                    (6)
                                     , 1322-1333. <a href="https://doi.org/10.1021/acs.accounts.0c00895" title="DOI URL">https://doi.org/10.1021/acs.accounts.0c00895</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.accounts.0c00895&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.accounts.0c00895%26sid%3Dliteratum%253Aachs%26jtitle%3DAccounts%2520of%2520Chemical%2520Research%26atitle%3DFacilitating%252BCompound%252BEntry%252Bas%252Ba%252BMeans%252Bto%252BDiscover%252BAntibiotics%252Bfor%252BGram-Negative%252BBacteria%26aulast%3DMu%25C3%25B1oz%26aufirst%3DKristen%2BA.%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D04012021%26date%3D26022021%26volume%3D54%26issue%3D6%26spage%3D1322%26epage%3D1333" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Sarah J. Perlmutter, Emily J. Geddes, Bryon S. Drown, Stephen E. Motika, Myung Ryul Lee, <span class="NLM_string-name hlFld-ContribAuthor">Paul J. Hergenrother</span>. </span><span class="cited-content_cbyCitation_article-title">Compound Uptake into E. coli Can Be Facilitated by N-Alkyl Guanidiniums and Pyridiniums. </span><span class="cited-content_cbyCitation_journal-name">ACS Infectious Diseases</span><span> <strong>2021,</strong> <em>7 </em>
                                    (1)
                                     , 162-173. <a href="https://doi.org/10.1021/acsinfecdis.0c00715" title="DOI URL">https://doi.org/10.1021/acsinfecdis.0c00715</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsinfecdis.0c00715&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsinfecdis.0c00715%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Infectious%2520Diseases%26atitle%3DCompound%252BUptake%252Binto%252BE.%2525C2%2525A0coli%252BCan%252BBe%252BFacilitated%252Bby%252BN-Alkyl%252BGuanidiniums%252Band%252BPyridiniums%26aulast%3DPerlmutter%26aufirst%3DSarah%2BJ.%26date%3D2021%26date%3D2020%26date%3D2020%26date%3D13102020%26date%3D23112020%26volume%3D7%26issue%3D1%26spage%3D162%26epage%3D173" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhixu  Zhou</span>, <span class="hlFld-ContribAuthor ">Ye  Liu</span>, <span class="hlFld-ContribAuthor ">Qian  Ren</span>, <span class="hlFld-ContribAuthor ">Dehou  Yu</span>, <span class="hlFld-ContribAuthor ">Hongguang  Lu</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis, crystal structure and DFT study of a novel compound N-(4-(2,4-dimorpholinopyrido[2,3-d]pyrimidin-6-yl)phenyl)pyrrolidine-1-carboxamide. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular Structure</span><span> <strong>2021,</strong> <em>1235 </em>, 130261. <a href="https://doi.org/10.1016/j.molstruc.2021.130261" title="DOI URL">https://doi.org/10.1016/j.molstruc.2021.130261</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.molstruc.2021.130261&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.molstruc.2021.130261%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520Structure%26atitle%3DSynthesis%25252C%252Bcrystal%252Bstructure%252Band%252BDFT%252Bstudy%252Bof%252Ba%252Bnovel%252Bcompound%252BN-%2525284-%2525282%25252C4-dimorpholinopyrido%25255B2%25252C3-d%25255Dpyrimidin-6-yl%252529phenyl%252529pyrrolidine-1-carboxamide%26aulast%3DZhou%26aufirst%3DZhixu%26date%3D2021%26volume%3D1235%26spage%3D130261" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Henni-Karoliina  Ropponen</span>, <span class="hlFld-ContribAuthor ">Eleonora  Diamanti</span>, <span class="hlFld-ContribAuthor ">Alexandra  Siemens</span>, <span class="hlFld-ContribAuthor ">Boris  Illarionov</span>, <span class="hlFld-ContribAuthor ">Jörg  Haupenthal</span>, <span class="hlFld-ContribAuthor ">Markus  Fischer</span>, <span class="hlFld-ContribAuthor ">Matthias  Rottmann</span>, <span class="hlFld-ContribAuthor ">Matthias  Witschel</span>, <span class="hlFld-ContribAuthor ">Anna K. H.  Hirsch</span>. </span><span class="cited-content_cbyCitation_article-title">Assessment of the rules related to gaining activity against Gram-negative bacteria. </span><span class="cited-content_cbyCitation_journal-name">RSC Medicinal Chemistry</span><span> <strong>2021,</strong> <em>12 </em>
                                    (4)
                                     , 593-601. <a href="https://doi.org/10.1039/D0MD00409J" title="DOI URL">https://doi.org/10.1039/D0MD00409J</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0MD00409J&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0MD00409J%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Medicinal%2520Chemistry%26atitle%3DAssessment%252Bof%252Bthe%252Brules%252Brelated%252Bto%252Bgaining%252Bactivity%252Bagainst%252BGram-negative%252Bbacteria%26aulast%3DRopponen%26aufirst%3DHenni-Karoliina%26date%3D2021%26date%3D2021%26volume%3D12%26issue%3D4%26spage%3D593%26epage%3D601" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sara S.  El Zahed</span>, <span class="hlFld-ContribAuthor ">Shawn  French</span>, <span class="hlFld-ContribAuthor ">Maya A.  Farha</span>, <span class="hlFld-ContribAuthor ">Garima  Kumar</span>, <span class="hlFld-ContribAuthor ">Eric D.  Brown</span>. </span><span class="cited-content_cbyCitation_article-title">Physicochemical and Structural Parameters Contributing to the Antibacterial Activity and Efflux Susceptibility of Small-Molecule Inhibitors of Escherichia coli. </span><span class="cited-content_cbyCitation_journal-name">Antimicrobial Agents and Chemotherapy</span><span> <strong>2021,</strong> <em>65 </em>
                                    (4)
                                     <a href="https://doi.org/10.1128/AAC.01925-20" title="DOI URL">https://doi.org/10.1128/AAC.01925-20</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1128/AAC.01925-20&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1128%2FAAC.01925-20%26sid%3Dliteratum%253Aachs%26jtitle%3DAntimicrobial%2520Agents%2520and%2520Chemotherapy%26atitle%3DPhysicochemical%252Band%252BStructural%252BParameters%252BContributing%252Bto%252Bthe%252BAntibacterial%252BActivity%252Band%252BEfflux%252BSusceptibility%252Bof%252BSmall-Molecule%252BInhibitors%252Bof%252BEscherichia%252Bcoli%26aulast%3DEl%2BZahed%26aufirst%3DSara%2BS.%26date%3D2021%26volume%3D65%26issue%3D4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yanlin  Jian</span>, <span class="hlFld-ContribAuthor ">Romain  Merceron</span>, <span class="hlFld-ContribAuthor ">Steven  De Munck</span>, <span class="hlFld-ContribAuthor ">He Eun  Forbes</span>, <span class="hlFld-ContribAuthor ">Fabian  Hulpia</span>, <span class="hlFld-ContribAuthor ">Martijn D.P.  Risseeuw</span>, <span class="hlFld-ContribAuthor ">Kristof  Van Hecke</span>, <span class="hlFld-ContribAuthor ">Savvas N.  Savvides</span>, <span class="hlFld-ContribAuthor ">Hélène  Munier-Lehmann</span>, <span class="hlFld-ContribAuthor ">Helena.I.M.  Boshoff</span>, <span class="hlFld-ContribAuthor ">Serge  Van Calenbergh</span>. </span><span class="cited-content_cbyCitation_article-title">Endeavors towards transformation of M. tuberculosis thymidylate kinase (MtbTMPK) inhibitors into potential antimycobacterial agents. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>206 </em>, 112659. <a href="https://doi.org/10.1016/j.ejmech.2020.112659" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112659</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112659&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112659%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DEndeavors%252Btowards%252Btransformation%252Bof%252BM.%2525C2%2525A0tuberculosis%252Bthymidylate%252Bkinase%252B%252528MtbTMPK%252529%252Binhibitors%252Binto%252Bpotential%252Bantimycobacterial%252Bagents%26aulast%3DJian%26aufirst%3DYanlin%26date%3D2020%26volume%3D206%26spage%3D112659" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kyle E  Murphy</span>, <span class="hlFld-ContribAuthor ">Grace F  Sloan</span>, <span class="hlFld-ContribAuthor ">Grace V  Lawhern</span>, <span class="hlFld-ContribAuthor ">Grace E  Volk</span>, <span class="hlFld-ContribAuthor ">Jacob T  Shumate</span>, <span class="hlFld-ContribAuthor ">Amanda L  Wolfe</span>. </span><span class="cited-content_cbyCitation_article-title">Advances in antibiotic drug discovery: reducing the barriers for antibiotic development. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2020,</strong> <em>12 </em>
                                    (22)
                                     , 2067-2087. <a href="https://doi.org/10.4155/fmc-2020-0247" title="DOI URL">https://doi.org/10.4155/fmc-2020-0247</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc-2020-0247&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc-2020-0247%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DAdvances%252Bin%252Bantibiotic%252Bdrug%252Bdiscovery%25253A%252Breducing%252Bthe%252Bbarriers%252Bfor%252Bantibiotic%252Bdevelopment%26aulast%3DMurphy%26aufirst%3DKyle%2BE%26date%3D2020%26volume%3D12%26issue%3D22%26spage%3D2067%26epage%3D2087" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sridhar  Chirumarry</span>, <span class="hlFld-ContribAuthor ">Nak-Kyun  Soung</span>, <span class="hlFld-ContribAuthor ">Junyeol  Han</span>, <span class="hlFld-ContribAuthor ">Eun Young  Kim</span>, <span class="hlFld-ContribAuthor ">Eun Kyoung  Ryu</span>, <span class="hlFld-ContribAuthor ">Young-Ho  Lee</span>, <span class="hlFld-ContribAuthor ">Song Yub  Shin</span>, <span class="hlFld-ContribAuthor ">Pethaiah  Gunasekaran</span>, <span class="hlFld-ContribAuthor ">Jeong Kyu  Bang</span>. </span><span class="cited-content_cbyCitation_article-title">Antibacterial AZT derivative regulates metastasis of breast cancer cells. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>193 </em>, 112233. <a href="https://doi.org/10.1016/j.ejmech.2020.112233" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112233</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112233&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112233%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DAntibacterial%252BAZT%252Bderivative%252Bregulates%252Bmetastasis%252Bof%252Bbreast%252Bcancer%252Bcells%26aulast%3DChirumarry%26aufirst%3DSridhar%26date%3D2020%26volume%3D193%26spage%3D112233" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00625/20190815/images/medium/jm-2019-006255_0001.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00625/20190815/images/large/jm-2019-006255_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00625&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00625/20190815/images/medium/jm-2019-006255_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00625/20190815/images/large/jm-2019-006255_0016.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Compound <b>1</b> (carbon, green; nitrogen, blue; bromine, dark red; all rendered as sticks) cocrystallized with <i>E. coli</i> BC at 2.1 Å as reported in Ref <a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2V58">2V58</a>). (A) View of compound <b>1</b> in the BC active site showing the 7-position vector that points toward two proximal glutamate residues as illustrated by the two arrows. The distance from the 7-position nitrogen atom and nearest oxygen atoms of Glu288 and Glu276 are 4.9 and 6.1 Å, respectively. (B) View of the solvent accessible surface representation, in transparent grey, of <i>E. coli</i> BC cocrystallized with <b>1</b>. The second vector points off the dihalo-aromatic ring of <b>1</b> through a small slot in the active site surface toward the solvent. Images prepared using PyMOL.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00625/20190815/images/large/jm-2019-006255_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00625&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00625/20190815/images/medium/jm-2019-006255_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00625/20190815/images/large/jm-2019-006255_0012.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of 6-Aryl Pyridopyrimidines<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00625/20190815/images/large/jm-2019-006255_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00625&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) Base, solvent; see the <a class="ref internalNav" href="#sec5" aria-label="Experimental Section">Experimental Section</a>.</p></p></figure><figure data-id="sch2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00625/20190815/images/medium/jm-2019-006255_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00625/20190815/images/large/jm-2019-006255_0011.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compounds <b>12a</b>, <b>12b</b>, and <b>13c</b>–<b>13h</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00625/20190815/images/large/jm-2019-006255_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00625&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) NaH, DMF, 45%; (b) NaNO<sub>2</sub>, trifluoroacetic acid, 90%; (c) POCl<sub>3</sub>, DMF (cat), 95 °C, 98%; (d) <b>9a</b>–<b>9h</b>, MeCN, 1,4-dioxane, 80 °C; (e) mCPBA or Davis oxaziridine, DCM; (f) NH<sub>3</sub>, MeOH, THF, 65 °C; (g) trifluoroacetic acid, DCM.</p></p></figure><figure data-id="sch3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00625/20190815/images/medium/jm-2019-006255_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00625/20190815/images/large/jm-2019-006255_0010.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Compounds <b>14a</b>–<b>14d</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00625/20190815/images/large/jm-2019-006255_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00625&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) NaH, DMF; (b) NaNO<sub>2</sub>, trifluoroacetic acid; (c) POCl<sub>3</sub>, DMF (cat), 95 °C; (d) <b>9h</b>, MeCN, 1,4-dioxane, 80 °C; (e) mCPBA or Davis oxaziridine, DCM; (f) NH<sub>3</sub>, MeOH, THF, 65 °C; (g) trifluoroacetic acid, DCM; (h) R(BOH)<sub>2</sub>, Pd(dppf)Cl<sub>2</sub>, Na<sub>2</sub>CO<sub>3</sub>, dioxane, water, 80 °C.</p></p></figure><figure data-id="fig2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00625/20190815/images/medium/jm-2019-006255_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00625/20190815/images/large/jm-2019-006255_0015.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. BC cocrystal structures of new pyridopyrimidine inhibitors overlaid with previously identified parent compound <b>1</b> cocrystallized with <i>E. coli</i> BC (shown in green; PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2V58">2V58</a>). (A) Compound <b>13f</b> (carbon, magenta; nitrogen, blue; chlorine, light green; all rendered as sticks) cocrystallized with <i>E. coli</i> BC at 2.06 Å (magenta ribbon cartoon; PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6OI9">6OI9</a>). The core pyridopyrimidine scaffold of <b>13f</b> is found in the identical position as that of parent compound <b>1</b>. The 7-position moiety extending off <b>13f</b> points toward the conserved glutamate residues, placing its proximal amine group nitrogen atom 3.16 and 2.17 Å in mol A and 2.97 and 2.65 Å in mol B away from the nearest oxygen atoms of the carboxylate groups of Glu288 and Glu276, respectively. (B) Compound <b>14a</b> (carbon, light blue; nitrogen, blue; chlorine, light green; all rendered as sticks) cocrystallized with the <i>H. influenzae</i> BC at 2.5 Å (light blue ribbon cartoon; PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6OI8">6OI8</a>). The T-loop (ref <a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a>) in the <i>H. influenzae</i> crystal structure is presumably disordered, as observed previously in other BC crystal structures (ref <a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a>) and not represented by the ribbon cartoon. The core pyridopyrimidine scaffold of <b>14a</b> is found in the identical position as that of parent compound <b>1</b>. The 7-position moiety extending off <b>14a</b> points toward the conserved glutamate residues, placing its proximal amine group nitrogen atom 3.29 and 2.91 Å away from the nearest oxygen atoms of the carboxylate groups of Glu288 and Glu276, respectively. (C) The vector of <b>14a</b> that extends off the dihalo-aromatic ring is positioned, as expected, within the small slot in the active site surface, as shown by the partially transparent solvent accessible surface representation in gray. Images prepared using PyMOL.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00625/20190815/images/large/jm-2019-006255_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00625&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00625/20190815/images/medium/jm-2019-006255_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00625/20190815/images/large/jm-2019-006255_0014.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Cells treated with BC inhibitors produce chains of cells. Cells were treated with either DMSO or a compound at the indicated concentrations for 2 h. The top panel shows FM 4-64-stained cell membranes (red), and the bottom panel shows both cell membranes and DNA stained with DAPI (blue). Scale bar equals 1 μm.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00625/20190815/images/large/jm-2019-006255_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00625&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00625/20190815/images/medium/jm-2019-006255_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00625/20190815/images/large/jm-2019-006255_0013.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Alignment of the <i>P. aeruginosa</i> BC enzyme (shown as yellow ribbon; PBD code 2VQD) cocrystallized with substrate analog AMPCP (carbon, yellow; nitrogen, blue; phosphate, orange; oxygen, red; all rendered as sticks) and the <i>H. influenzae</i> BC enzyme (shown as light blue ribbon) cocrystallized with <b>14a</b> (carbon, light blue; nitrogen, blue; chlorine, light green; all rendered as sticks; PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6OI8">6OI8</a>). Resistance-conferring mutants were found at residue positions 278 and 437, which, in the WT enzymes shown here, are leucine and isoleucine, respectively (shown as sticks).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-16/acs.jmedchem.9b00625/20190815/images/large/jm-2019-006255_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00625&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i72">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_38171" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_38171" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 57 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span> <i>Antibiotic Resistance Threats
in the United States, 2013</i>; <span class="NLM_publisher-name">U.S. Centers
for Disease Control and Prevention</span>: [Online] <a href="http://www.cdc.gov/drugresistance/threat-report-2013" class="extLink">http://www.cdc.gov/drugresistance/threat-report-2013</a> (accessed 27 February, 2019).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Antibiotic+Resistance+Threats%0Ain+the+United+States%2C+2013%3B+U.S.+Centers%0Afor+Disease+Control+and+Prevention%3A+%5BOnline%5D+http%3A%2F%2Fwww.cdc.gov%2Fdrugresistance%2Fthreat-report-2013+%28accessed+27+February%2C+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26jtitle%3DAntibiotic%2520Resistance%2520Threats%250Ain%2520the%2520United%2520States%252C%25202013%26pub%3DU.S.%2520Centers%250Afor%2520Disease%2520Control%2520and%2520Prevention" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burnham, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olsen, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kollef, M. H.</span></span> <span> </span><span class="NLM_article-title">Re-Estimating Annual Deaths Due to Multidrug-Resistant Organism Infections</span>. <i>Infect. Control Hosp. Epidemiol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">112</span>– <span class="NLM_lpage">113</span>, <span class="refDoi"> DOI: 10.1017/ice.2018.304</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1017%2Fice.2018.304" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=30463634" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A280%3ADC%252BB3crjsFaisA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2019&pages=112-113&author=J.+P.+Burnhamauthor=M.+A.+Olsenauthor=M.+H.+Kollef&title=Re-Estimating+Annual+Deaths+Due+to+Multidrug-Resistant+Organism+Infections&doi=10.1017%2Fice.2018.304"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Re-estimating annual deaths due to multidrug-resistant organism infections</span></div><div class="casAuthors">Burnham Jason P; Olsen Margaret A; Kollef Marin H</div><div class="citationInfo"><span class="NLM_cas:title">Infection control and hospital epidemiology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">112-113</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRsbGst4dFdhyO2kE7krGOBfW6udTcc2eZaaEs2TimUkLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3crjsFaisA%253D%253D&md5=1bbd7d59e205bf2987585dc2df677343</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1017%2Fice.2018.304&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1017%252Fice.2018.304%26sid%3Dliteratum%253Aachs%26aulast%3DBurnham%26aufirst%3DJ.%2BP.%26aulast%3DOlsen%26aufirst%3DM.%2BA.%26aulast%3DKollef%26aufirst%3DM.%2BH.%26atitle%3DRe-Estimating%2520Annual%2520Deaths%2520Due%2520to%2520Multidrug-Resistant%2520Organism%2520Infections%26jtitle%3DInfect.%2520Control%2520Hosp.%2520Epidemiol.%26date%3D2019%26volume%3D40%26spage%3D112%26epage%3D113%26doi%3D10.1017%2Fice.2018.304" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cassini, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Högberg, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plachouras, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quattrocchi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoxha, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simonsen, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colomb-Cotinat, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kretzschmar, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Devleesschauwer, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cecchini, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouakrim, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliveira, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Struelens, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suetens, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monnet, D. L.</span></span>; <span class="NLM_contrib-group">Burden of AMR Collaborative Group</span> <span> </span><span class="NLM_article-title">Attributable Deaths and Disability-Adjusted Life-Years Caused by Infections with Antibiotic-Resistant Bacteria in the EU and the European Economic Area in 2015: a Population-Level Modelling Analysis</span>. <i>Lancet Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">56</span>– <span class="NLM_lpage">66</span>, <span class="refDoi"> DOI: 10.1016/S1473-3099(18)30605-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1016%2FS1473-3099%2818%2930605-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=30409683" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A280%3ADC%252BB3cvotFGluw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2019&pages=56-66&author=A.+Cassiniauthor=L.+D.+H%C3%B6gbergauthor=D.+Plachourasauthor=A.+Quattrocchiauthor=A.+Hoxhaauthor=G.+S.+Simonsenauthor=M.+Colomb-Cotinatauthor=M.+E.+Kretzschmarauthor=B.+Devleesschauwerauthor=M.+Cecchiniauthor=D.+A.+Ouakrimauthor=T.+C.+Oliveiraauthor=M.+J.+Struelensauthor=C.+Suetensauthor=D.+L.+Monnetauthor=Burden+of+AMR+Collaborative+Group&title=Attributable+Deaths+and+Disability-Adjusted+Life-Years+Caused+by+Infections+with+Antibiotic-Resistant+Bacteria+in+the+EU+and+the+European+Economic+Area+in+2015%3A+a+Population-Level+Modelling+Analysis&doi=10.1016%2FS1473-3099%2818%2930605-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Attributable deaths and disability-adjusted life-years caused by infections with antibiotic-resistant bacteria in the EU and the European Economic Area in 2015: a population-level modelling analysis</span></div><div class="casAuthors">Cassini Alessandro; Hogberg Liselotte Diaz; Plachouras Diamantis; Quattrocchi Annalisa; Hoxha Ana; Struelens Marc J; Suetens Carl; Monnet Dominique L; Simonsen Gunnar Skov; Colomb-Cotinat Melanie; Kretzschmar Mirjam E; Devleesschauwer Brecht; Cecchini Michele; Ouakrim Driss Ait; Oliveira Tiago Cravo</div><div class="citationInfo"><span class="NLM_cas:title">The Lancet. Infectious diseases</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">56-66</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Infections due to antibiotic-resistant bacteria are threatening modern health care.  However, estimating their incidence, complications, and attributable mortality is challenging.  We aimed to estimate the burden of infections caused by antibiotic-resistant bacteria of public health concern in countries of the EU and European Economic Area (EEA) in 2015, measured in number of cases, attributable deaths, and disability-adjusted life-years (DALYs).  METHODS:  We estimated the incidence of infections with 16 antibiotic resistance-bacterium combinations from European Antimicrobial Resistance Surveillance Network (EARS-Net) 2015 data that was country-corrected for population coverage.  We multiplied the number of bloodstream infections (BSIs) by a conversion factor derived from the European Centre for Disease Prevention and Control point prevalence survey of health-care-associated infections in European acute care hospitals in 2011-12 to estimate the number of non-BSIs.  We developed disease outcome models for five types of infection on the basis of systematic reviews of the literature.  FINDINGS:  From EARS-Net data collected between Jan 1, 2015, and Dec 31, 2015, we estimated 671 689 (95% uncertainty interval [UI] 583 148-763 966) infections with antibiotic-resistant bacteria, of which 63·5% (426 277 of 671 689) were associated with health care.  These infections accounted for an estimated 33 110 (28 480-38 430) attributable deaths and 874 541 (768 837-989 068) DALYs.  The burden for the EU and EEA was highest in infants (aged <1 year) and people aged 65 years or older, had increased since 2007, and was highest in Italy and Greece.  INTERPRETATION:  Our results present the health burden of five types of infection with antibiotic-resistant bacteria expressed, for the first time, in DALYs.  The estimated burden of infections with antibiotic-resistant bacteria in the EU and EEA is substantial compared with that of other infectious diseases, and has increased since 2007.  Our burden estimates provide useful information for public health decision-makers prioritising interventions for infectious diseases.  FUNDING:  European Centre for Disease Prevention and Control.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR76qXWYwPPXq69lAbTj9rGfW6udTcc2eakED1BRzfQeLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3cvotFGluw%253D%253D&md5=151619810730b07e3d84692f1d966b9a</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2FS1473-3099%2818%2930605-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1473-3099%252818%252930605-4%26sid%3Dliteratum%253Aachs%26aulast%3DCassini%26aufirst%3DA.%26aulast%3DH%25C3%25B6gberg%26aufirst%3DL.%2BD.%26aulast%3DPlachouras%26aufirst%3DD.%26aulast%3DQuattrocchi%26aufirst%3DA.%26aulast%3DHoxha%26aufirst%3DA.%26aulast%3DSimonsen%26aufirst%3DG.%2BS.%26aulast%3DColomb-Cotinat%26aufirst%3DM.%26aulast%3DKretzschmar%26aufirst%3DM.%2BE.%26aulast%3DDevleesschauwer%26aufirst%3DB.%26aulast%3DCecchini%26aufirst%3DM.%26aulast%3DOuakrim%26aufirst%3DD.%2BA.%26aulast%3DOliveira%26aufirst%3DT.%2BC.%26aulast%3DStruelens%26aufirst%3DM.%2BJ.%26aulast%3DSuetens%26aufirst%3DC.%26aulast%3DMonnet%26aufirst%3DD.%2BL.%26aulast%3D%26atitle%3DAttributable%2520Deaths%2520and%2520Disability-Adjusted%2520Life-Years%2520Caused%2520by%2520Infections%2520with%2520Antibiotic-Resistant%2520Bacteria%2520in%2520the%2520EU%2520and%2520the%2520European%2520Economic%2520Area%2520in%25202015%253A%2520a%2520Population-Level%2520Modelling%2520Analysis%26jtitle%3DLancet%2520Infect.%2520Dis.%26date%3D2019%26volume%3D19%26spage%3D56%26epage%3D66%26doi%3D10.1016%2FS1473-3099%2818%2930605-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Todd, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiehlbauch, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walters, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kallen, A.</span></span> <span> </span><span class="NLM_article-title">Notes From the Field: Pan-Resistant New Delhi Metallo-Beta-Lactamase-Producing <i>Klebsiella pneumoniae</i> - Washoe County, Nevada, 2016</span>. <i>MMWR Morb. Mortal Wkly. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">33</span>, <span class="refDoi"> DOI: 10.15585/mmwr.mm6601a7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.15585%2Fmmwr.mm6601a7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=28081065" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A280%3ADC%252BC1c7ltFSmtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2017&pages=33&author=L.+Chenauthor=R.+Toddauthor=J.+Kiehlbauchauthor=M.+Waltersauthor=A.+Kallen&title=Notes+From+the+Field%3A+Pan-Resistant+New+Delhi+Metallo-Beta-Lactamase-Producing+Klebsiella+pneumoniae+-+Washoe+County%2C+Nevada%2C+2016&doi=10.15585%2Fmmwr.mm6601a7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Notes from the Field: Pan-Resistant New Delhi Metallo-Beta-Lactamase-Producing Klebsiella pneumoniae - Washoe County, Nevada, 2016</span></div><div class="casAuthors">Chen Lei; Todd Randall; Kiehlbauch Julia; Walters Maroya; Kallen Alexander</div><div class="citationInfo"><span class="NLM_cas:title">MMWR. Morbidity and mortality weekly report</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">33</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRXLWGx5FIPGe98rul4nhSdfW6udTcc2eakED1BRzfQeLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1c7ltFSmtg%253D%253D&md5=c2f8b366b1b02600014e0c42e8bafaab</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.15585%2Fmmwr.mm6601a7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.15585%252Fmmwr.mm6601a7%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DL.%26aulast%3DTodd%26aufirst%3DR.%26aulast%3DKiehlbauch%26aufirst%3DJ.%26aulast%3DWalters%26aufirst%3DM.%26aulast%3DKallen%26aufirst%3DA.%26atitle%3DNotes%2520From%2520the%2520Field%253A%2520Pan-Resistant%2520New%2520Delhi%2520Metallo-Beta-Lactamase-Producing%2520Klebsiella%2520pneumoniae%2520-%2520Washoe%2520County%252C%2520Nevada%252C%25202016%26jtitle%3DMMWR%2520Morb.%2520Mortal%2520Wkly.%2520Rep.%26date%3D2017%26volume%3D66%26spage%3D33%26doi%3D10.15585%2Fmmwr.mm6601a7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miller, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunham, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mochalkin, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banotai, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bowman, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buist, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunkle, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanna, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harwood, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huband, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karnovsky, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuhn, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Limberakis, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mehrens, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mueller, W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narasimhan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogden, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohren, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prasad, J. V. N. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shelly, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skerlos, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sulavik, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, V. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">VanderRoest, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitton, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stover, C. K.</span></span> <span> </span><span class="NLM_article-title">A Class of Selective Antibacterials Derived From a Protein Kinase Inhibitor Pharmacophore</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>106</i></span>,  <span class="NLM_fpage">1737</span>– <span class="NLM_lpage">1742</span>, <span class="refDoi"> DOI: 10.1073/pnas.0811275106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1073%2Fpnas.0811275106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=19164768" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BD1MXitV2jurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2009&pages=1737-1742&author=J.+R.+Millerauthor=S.+Dunhamauthor=I.+Mochalkinauthor=C.+Banotaiauthor=M.+Bowmanauthor=S.+Buistauthor=B.+Dunkleauthor=D.+Hannaauthor=H.+J.+Harwoodauthor=M.+D.+Hubandauthor=A.+Karnovskyauthor=M.+Kuhnauthor=C.+Limberakisauthor=J.+Y.+Liuauthor=S.+Mehrensauthor=W.+T.+Muellerauthor=L.+Narasimhanauthor=A.+Ogdenauthor=J.+Ohrenauthor=J.+V.+N.+V.+Prasadauthor=J.+A.+Shellyauthor=L.+Skerlosauthor=M.+Sulavikauthor=V.+H.+Thomasauthor=S.+VanderRoestauthor=L.+Wangauthor=Z.+Wangauthor=A.+Whittonauthor=T.+Zhuauthor=C.+K.+Stover&title=A+Class+of+Selective+Antibacterials+Derived+From+a+Protein+Kinase+Inhibitor+Pharmacophore&doi=10.1073%2Fpnas.0811275106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">A class of selective antibacterials derived from a protein kinase inhibitor pharmacophore</span></div><div class="casAuthors">Miller, J. Richard; Dunham, Steve; Mochalkin, Igor; Banotai, Craig; Bowman, Matthew; Buist, Susan; Dunkle, Bill; Hanna, Debra; Harwood, James; Huband, Michael D.; Karnovsky, Alla; Kuhn, Michael; Limberakis, Chris; Liu, Jia Y.; Mehrens, Shawn; Mueller, W. Thomas; Narasimhan, Lakshmi; Ogden, Adam; Ohren, Jeff; Vara Prasad, J. V. N.; Shelly, John A.; Skerlos, Laura; Sulavik, Mark; Thomas, V. Hayden; VanderRoest, Steve; Wang, LiAnn; Wang, Zhigang; Whitton, Amy; Zhu, Tong; Stover, C. Kendall</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1737-1742</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">As the need for novel antibiotic classes to combat bacterial drug resistance increases, the paucity of leads resulting from target-based antibacterial screening of pharmaceutical compd. libraries is of major concern.  One explanation for this lack of success is that antibacterial screening efforts have not leveraged the eukaryotic bias resulting from more extensive chem. efforts targeting eukaryotic gene families such as G protein-coupled receptors and protein kinases.  Consistent with a focus on antibacterial target space resembling these eukaryotic targets, we used whole-cell screening to identify a series of antibacterial pyridopyrimidines derived from a protein kinase inhibitor pharmacophore.  In bacteria, the pyridopyrimidines target the ATP-binding site of biotin carboxylase (BC), which catalyzes the first enzymic step of fatty acid biosynthesis.  These inhibitors are effective in vitro and in vivo against fastidious Gram-neg. pathogens including Haemophilus influenzae.  Although the BC active site has architectural similarity to those of eukaryotic protein kinases, inhibitor binding to the BC ATP-binding site is distinct from the protein kinase-binding mode, such that the inhibitors are selective for bacterial BC.  In summary, the authors have discovered a promising class of potent antibacterials with a previously undescribed mechanism of action.  In consideration of the eukaryotic bias of pharmaceutical libraries, the findings also suggest that pursuit of a novel inhibitor leads for antibacterial targets with active-site structural similarity to known human targets will likely be more fruitful than the traditional focus on unique bacterial target space, particularly when structure-based and computational methodologies are applied to ensure bacterial selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8jpj8EOtRo7Vg90H21EOLACvtfcHk0liHS-JzyMuq5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXitV2jurk%253D&md5=06758ee5befbe847825ec35adb177006</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0811275106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0811275106%26sid%3Dliteratum%253Aachs%26aulast%3DMiller%26aufirst%3DJ.%2BR.%26aulast%3DDunham%26aufirst%3DS.%26aulast%3DMochalkin%26aufirst%3DI.%26aulast%3DBanotai%26aufirst%3DC.%26aulast%3DBowman%26aufirst%3DM.%26aulast%3DBuist%26aufirst%3DS.%26aulast%3DDunkle%26aufirst%3DB.%26aulast%3DHanna%26aufirst%3DD.%26aulast%3DHarwood%26aufirst%3DH.%2BJ.%26aulast%3DHuband%26aufirst%3DM.%2BD.%26aulast%3DKarnovsky%26aufirst%3DA.%26aulast%3DKuhn%26aufirst%3DM.%26aulast%3DLimberakis%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DJ.%2BY.%26aulast%3DMehrens%26aufirst%3DS.%26aulast%3DMueller%26aufirst%3DW.%2BT.%26aulast%3DNarasimhan%26aufirst%3DL.%26aulast%3DOgden%26aufirst%3DA.%26aulast%3DOhren%26aufirst%3DJ.%26aulast%3DPrasad%26aufirst%3DJ.%2BV.%2BN.%2BV.%26aulast%3DShelly%26aufirst%3DJ.%2BA.%26aulast%3DSkerlos%26aufirst%3DL.%26aulast%3DSulavik%26aufirst%3DM.%26aulast%3DThomas%26aufirst%3DV.%2BH.%26aulast%3DVanderRoest%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DWhitton%26aufirst%3DA.%26aulast%3DZhu%26aufirst%3DT.%26aulast%3DStover%26aufirst%3DC.%2BK.%26atitle%3DA%2520Class%2520of%2520Selective%2520Antibacterials%2520Derived%2520From%2520a%2520Protein%2520Kinase%2520Inhibitor%2520Pharmacophore%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2009%26volume%3D106%26spage%3D1737%26epage%3D1742%26doi%3D10.1073%2Fpnas.0811275106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Payne, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warren, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holmes, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lonsdale, J. T.</span></span> <span> </span><span class="NLM_article-title">Bacterial Fatty-Acid Biosynthesis: A Genomics-Driven Target for Antibacterial Drug Discovery</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">537</span>– <span class="NLM_lpage">544</span>, <span class="refDoi"> DOI: 10.1016/S1359-6446(01)01774-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1016%2FS1359-6446%2801%2901774-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=11369293" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BD3MXjtlCjsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2001&pages=537-544&author=D.+J.+Payneauthor=P.+V.+Warrenauthor=D.+J.+Holmesauthor=Y.+Jiauthor=J.+T.+Lonsdale&title=Bacterial+Fatty-Acid+Biosynthesis%3A+A+Genomics-Driven+Target+for+Antibacterial+Drug+Discovery&doi=10.1016%2FS1359-6446%2801%2901774-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Bacterial fatty-acid biosynthesis: a genomics-driven target for antibacterial drug discovery</span></div><div class="casAuthors">Payne, D. J.; Warren, P. V.; Holmes, D. J.; Ji, Y.; Lonsdale, J. T.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">537-544</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A review with 46 refs. demonstrating how the interplay of genomics, bioinformatics and genomic technologies has enabled an in-depth anal. of the component enzymes of the bacterial fatty-acid biosynthesis pathway as a source of novel antibacterial targets.  This evaluation has revealed that many of the enzymes are potentially selective, broad-spectrum antibacterial targets.  We also illustrate the suitability of some of these targets for HTS.  Furthermore, we discuss how the availability of a robust selectivity assay, mode-of-action assays and numerous crystal structures provide an excellent set of tools with which to initiate integrated programs of research to identify novel antibiotics targeted at these enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXOsty-wRiwLVg90H21EOLACvtfcHk0liGDWTEN8T_Jg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXjtlCjsbo%253D&md5=50cbe09ba8e48f9aca6a5b847477f24c</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2FS1359-6446%2801%2901774-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1359-6446%252801%252901774-3%26sid%3Dliteratum%253Aachs%26aulast%3DPayne%26aufirst%3DD.%2BJ.%26aulast%3DWarren%26aufirst%3DP.%2BV.%26aulast%3DHolmes%26aufirst%3DD.%2BJ.%26aulast%3DJi%26aufirst%3DY.%26aulast%3DLonsdale%26aufirst%3DJ.%2BT.%26atitle%3DBacterial%2520Fatty-Acid%2520Biosynthesis%253A%2520A%2520Genomics-Driven%2520Target%2520for%2520Antibacterial%2520Drug%2520Discovery%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2001%26volume%3D6%26spage%3D537%26epage%3D544%26doi%3D10.1016%2FS1359-6446%2801%2901774-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cronan, J. E.,  Jr.</span></span> <span> </span><span class="NLM_article-title">Bacterial Fatty Acid Biosynthesis: Targets for Antibacterial Drug Discovery</span>. <i>Annu. Rev. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">305</span>– <span class="NLM_lpage">332</span>, <span class="refDoi"> DOI: 10.1146/annurev.micro.55.1.305</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1146%2Fannurev.micro.55.1.305" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=11544358" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BD3MXnslKjt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2001&pages=305-332&author=J.+W.+Campbellauthor=J.+E.+Cronan&title=Bacterial+Fatty+Acid+Biosynthesis%3A+Targets+for+Antibacterial+Drug+Discovery&doi=10.1146%2Fannurev.micro.55.1.305"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Bacterial fatty acid biosynthesis: Targets for antibacterial drug discovery</span></div><div class="casAuthors">Campbell, John W.; Cronan, John E., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Microbiology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">305-332</span>CODEN:
                <span class="NLM_cas:coden">ARMIAZ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4227</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  The increase in drug-resistant pathogenic bacteria has created an urgent demand for new antibiotics.  Among the more attractive targets for the development of new antibacterial compds. are the enzymes of fatty acid biosynthesis.  Although a no. of potent inhibitors of microbial fatty acid biosynthesis have been discovered, few of these are clin. useful drugs.  Several of these fatty acid biosynthesis inhibitors have potential as lead compds. in the development of new antibacterials.  This review encompasses the known inhibitors and prospective targets for new antibacterials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpr8j4LVT_lPLVg90H21EOLACvtfcHk0liGDWTEN8T_Jg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXnslKjt7Y%253D&md5=e2fb75187125997e3d2ccea844114fde</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1146%2Fannurev.micro.55.1.305&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.micro.55.1.305%26sid%3Dliteratum%253Aachs%26aulast%3DCampbell%26aufirst%3DJ.%2BW.%26aulast%3DCronan%26aufirst%3DJ.%2BE.%26atitle%3DBacterial%2520Fatty%2520Acid%2520Biosynthesis%253A%2520Targets%2520for%2520Antibacterial%2520Drug%2520Discovery%26jtitle%3DAnnu.%2520Rev.%2520Microbiol.%26date%3D2001%26volume%3D55%26spage%3D305%26epage%3D332%26doi%3D10.1146%2Fannurev.micro.55.1.305" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Heath, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rock, C. O.</span></span> <span> </span><span class="NLM_article-title">Lipid Biosynthesis as a Target for Antibacterial Agents</span>. <i>Prog. Lipid Res.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">467</span>– <span class="NLM_lpage">497</span>, <span class="refDoi"> DOI: 10.1016/S0163-7827(01)00012-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1016%2FS0163-7827%2801%2900012-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=11591436" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BD3MXovFSju7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2001&pages=467-497&author=R.+J.+Heathauthor=S.+W.+Whiteauthor=C.+O.+Rock&title=Lipid+Biosynthesis+as+a+Target+for+Antibacterial+Agents&doi=10.1016%2FS0163-7827%2801%2900012-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Lipid biosynthesis as a target for antibacterial agents</span></div><div class="casAuthors">Heath, Richard J.; White, Stephen W.; Rock, Charles O.</div><div class="citationInfo"><span class="NLM_cas:title">Progress in Lipid Research</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">467-497</span>CODEN:
                <span class="NLM_cas:coden">PLIRDW</span>;
        ISSN:<span class="NLM_cas:issn">0163-7827</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A review.  Fatty acid biosynthesis, the first stage in membrane lipid biogenesis, is catalyzed in most bacteria by a series of small, sol. proteins that are each encoded by a discrete gene (Fig. 1; Table 1).  This arrangement is termed the type II fatty acid synthase (FAS) system and contrasts sharply with the type I FAS of eukaryotes which is a dimer of a single large, multifunctional polypeptide.  Thus, the bacterial pathway offers several unique sites for selective inhibition by chemotherapeutic agents.  The site of action of isoniazid, used in the treatment of tuberculosis for 50 yr, and the consumer antimicrobial agent triclosan were revealed recently to be the enoyl-ACP reductase of the type II FAS.  The fungal metabolites, cerulenin and thiolactomycin, target the condensing enzymes of the bacterial pathway while the dehydratase/isomerase is inhibited by a synthetic acetylenic substrate analog.  Transfer of fatty acids to the membrane has also been inhibited via interference with the first acyltransferase step, while a new class of drugs targets lipid A synthesis.  This review will summarize the data generated on these inhibitors to date, and examine where addnl. efforts will be required to develop new chemotherapeutics to help combat microbial infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpycZaAyBQafrVg90H21EOLACvtfcHk0liGDWTEN8T_Jg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXovFSju7s%253D&md5=c8a9b6692527ea14816034ece956cf51</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2FS0163-7827%2801%2900012-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0163-7827%252801%252900012-1%26sid%3Dliteratum%253Aachs%26aulast%3DHeath%26aufirst%3DR.%2BJ.%26aulast%3DWhite%26aufirst%3DS.%2BW.%26aulast%3DRock%26aufirst%3DC.%2BO.%26atitle%3DLipid%2520Biosynthesis%2520as%2520a%2520Target%2520for%2520Antibacterial%2520Agents%26jtitle%3DProg.%2520Lipid%2520Res.%26date%3D2001%26volume%3D40%26spage%3D467%26epage%3D497%26doi%3D10.1016%2FS0163-7827%2801%2900012-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rock, C. O.</span></span> <span> </span><span class="NLM_article-title">Inhibiting Bacterial Fatty Acid Synthesis</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>281</i></span>,  <span class="NLM_fpage">17541</span>– <span class="NLM_lpage">17544</span>, <span class="refDoi"> DOI: 10.1074/jbc.R600004200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1074%2Fjbc.R600004200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=16648134" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BD28XmtFyks7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=281&publication_year=2006&pages=17541-17544&author=Y.-M.+Zhangauthor=S.+W.+Whiteauthor=C.+O.+Rock&title=Inhibiting+Bacterial+Fatty+Acid+Synthesis&doi=10.1074%2Fjbc.R600004200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibiting Bacterial Fatty Acid Synthesis</span></div><div class="casAuthors">Zhang, Yong-Mei; White, Stephen W.; Rock, Charles O.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">281</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">17541-17544</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">A review.  The type II fatty acid synthase consists of a series of individual enzymes, each encoded by a sep. gene, that catalyze discrete steps in chain elongation.  The formation of fatty acids is vital to bacteria, and each of the essential enzymes and their acyl group carriers represent a potential target for the development of novel antibacterial therapeutics.  High resoln. x-ray and/or NMR structures of representative members of every enzyme in the type II pathway are now available, and these structures are a valuable resource to guide antibacterial drug discovery.  The role of each enzyme in regulating pathway activity and the diversity in the components of the pathway in the major human pathogens are important considerations in deciding the most suitable targets for future drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRu1UJVfxfWrVg90H21EOLACvtfcHk0ljvH4UlcuDjmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XmtFyks7w%253D&md5=b617963deae704579e4009b65281c5af</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1074%2Fjbc.R600004200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.R600004200%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DY.-M.%26aulast%3DWhite%26aufirst%3DS.%2BW.%26aulast%3DRock%26aufirst%3DC.%2BO.%26atitle%3DInhibiting%2520Bacterial%2520Fatty%2520Acid%2520Synthesis%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2006%26volume%3D281%26spage%3D17541%26epage%3D17544%26doi%3D10.1074%2Fjbc.R600004200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wright, H. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reynolds, K. A.</span></span> <span> </span><span class="NLM_article-title">Antibacterial Targets in Fatty Acid Biosynthesis</span>. <i>Curr. Opin. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">447</span>– <span class="NLM_lpage">453</span>, <span class="refDoi"> DOI: 10.1016/j.mib.2007.07.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1016%2Fj.mib.2007.07.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=17707686" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BD2sXht1yltb%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2007&pages=447-453&author=H.+T.+Wrightauthor=K.+A.+Reynolds&title=Antibacterial+Targets+in+Fatty+Acid+Biosynthesis&doi=10.1016%2Fj.mib.2007.07.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Antibacterial targets in fatty acid biosynthesis</span></div><div class="casAuthors">Wright, H. Tonie; Reynolds, Kevin A.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Microbiology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">447-453</span>CODEN:
                <span class="NLM_cas:coden">COMIF7</span>;
        ISSN:<span class="NLM_cas:issn">1369-5274</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The fatty acid biosynthesis pathway is an attractive but still largely unexploited target for the development of new antibacterial agents.  The extended use of the antituberculosis drug isoniazid and the antiseptic triclosan, which are inhibitors of fatty acid biosynthesis, validates this pathway as a target for antibacterial development.  Differences in subcellular organization of the bacterial and eukaryotic multi-enzyme fatty acid synthase systems offer the prospect of inhibitors with host vs. target specificity.  Platensimycin, platencin, and phomallenic acids, newly discovered natural product inhibitors of the condensation steps in fatty acid biosynthesis, represent new classes of compds. with antibiotic potential.  An almost complete catalog of crystal structures for the enzymes of the type II fatty acid biosynthesis pathway can now be exploited in the rational design of new inhibitors, as well as the recently published crystal structures of type I FAS complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7buYo89BhsrVg90H21EOLACvtfcHk0ljvH4UlcuDjmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXht1yltb%252FK&md5=fd53c737e87aa6c1694858c19e46d6f1</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.mib.2007.07.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.mib.2007.07.001%26sid%3Dliteratum%253Aachs%26aulast%3DWright%26aufirst%3DH.%2BT.%26aulast%3DReynolds%26aufirst%3DK.%2BA.%26atitle%3DAntibacterial%2520Targets%2520in%2520Fatty%2520Acid%2520Biosynthesis%26jtitle%3DCurr.%2520Opin.%2520Microbiol.%26date%3D2007%26volume%3D10%26spage%3D447%26epage%3D453%26doi%3D10.1016%2Fj.mib.2007.07.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Parsons, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rock, C. O.</span></span> <span> </span><span class="NLM_article-title">Is Bacterial Fatty Acid Synthesis a Valid Target for Antibacterial Drug Discovery?</span>. <i>Curr. Opin. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">544</span>– <span class="NLM_lpage">549</span>, <span class="refDoi"> DOI: 10.1016/j.mib.2011.07.029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1016%2Fj.mib.2011.07.029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=21862391" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlCntrjM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2011&pages=544-549&author=J.+B.+Parsonsauthor=C.+O.+Rock&title=Is+Bacterial+Fatty+Acid+Synthesis+a+Valid+Target+for+Antibacterial+Drug+Discovery%3F&doi=10.1016%2Fj.mib.2011.07.029"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Is bacterial fatty acid synthesis a valid target for antibacterial drug discovery?</span></div><div class="casAuthors">Parsons, Joshua B.; Rock, Charles O.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Microbiology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">544-549</span>CODEN:
                <span class="NLM_cas:coden">COMIF7</span>;
        ISSN:<span class="NLM_cas:issn">1369-5274</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The emergence of resistance against most current drugs emphasizes the need to develop new approaches to control bacterial pathogens, particularly Staphylococcus aureus.  Bacterial fatty acid synthesis is one such target that is being actively pursued by several research groups to develop anti-Staphylococcal agents.  Recently, the wisdom of this approach has been challenged based on the ability of a Gram-pos. bacterium to incorporate extracellular fatty acids and thus circumvent the inhibition of de novo fatty acid synthesis.  The generality of this conclusion has been challenged, and there is enough diversity in the enzymes and regulation of fatty acid synthesis in bacteria to conclude that there is not a single organism that can be considered typical and representative of bacteria as a whole.  We are left without a clear resoln. to this ongoing debate and await new basic research to define the pathways for fatty acid uptake and that det. the biochem. and genetic mechanisms for the regulation of fatty acid synthesis in Gram-pos. bacteria.  These crucial expts. will det. whether diversity in the control of this important pathway accounts for the apparently different responses of Gram-pos. bacteria to the inhibition of de novo fatty acid synthesis in presence of extracellular fatty acid supplements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7o8Zc8puOBrVg90H21EOLACvtfcHk0li_rlofhyesJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlCntrjM&md5=2c5a26d05b1174826232917568fdde17</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.mib.2011.07.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.mib.2011.07.029%26sid%3Dliteratum%253Aachs%26aulast%3DParsons%26aufirst%3DJ.%2BB.%26aulast%3DRock%26aufirst%3DC.%2BO.%26atitle%3DIs%2520Bacterial%2520Fatty%2520Acid%2520Synthesis%2520a%2520Valid%2520Target%2520for%2520Antibacterial%2520Drug%2520Discovery%253F%26jtitle%3DCurr.%2520Opin.%2520Microbiol.%26date%3D2011%26volume%3D14%26spage%3D544%26epage%3D549%26doi%3D10.1016%2Fj.mib.2011.07.029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cronan, J. E.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldrop, G. L.</span></span> <span> </span><span class="NLM_article-title">Multi-Subunit Acetyl-CoA Carboxylases</span>. <i>Prog. Lipid Res.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">407</span>– <span class="NLM_lpage">435</span>, <span class="refDoi"> DOI: 10.1016/S0163-7827(02)00007-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1016%2FS0163-7827%2802%2900007-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=12121720" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BD38Xltleku7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2002&pages=407-435&author=J.+E.+Cronanauthor=G.+L.+Waldrop&title=Multi-Subunit+Acetyl-CoA+Carboxylases&doi=10.1016%2FS0163-7827%2802%2900007-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Multi-subunit acetyl-CoA carboxylases</span></div><div class="casAuthors">Cronan, John E.; Waldrop, Grover L.</div><div class="citationInfo"><span class="NLM_cas:title">Progress in Lipid Research</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">407-435</span>CODEN:
                <span class="NLM_cas:coden">PLIRDW</span>;
        ISSN:<span class="NLM_cas:issn">0163-7827</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A review with 118 refs.  Acetyl-CoA carboxylase (ACC) catalyzes the 1st committed step of fatty acid synthesis, the carboxylation of acetyl-CoA to malonyl-CoA.  Two phys. distinct types of enzymes are found in nature.  Bacterial and most plant chloroplasts contain a multi-subunit ACC (MS-ACC) enzyme that is readily dissocd. into its component proteins.  Mammals, fungi, and plant cytosols contain the 2nd type of ACC, a single large multifunctional polypeptide.  This review focuses on the structures, regulation, and enzymic mechanisms of the bacterial and plant MS-ACCs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEQxH8bu-Sf7Vg90H21EOLACvtfcHk0li_rlofhyesJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xltleku7k%253D&md5=621ebd71847b6fca01114df5e72f6a1a</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2FS0163-7827%2802%2900007-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0163-7827%252802%252900007-3%26sid%3Dliteratum%253Aachs%26aulast%3DCronan%26aufirst%3DJ.%2BE.%26aulast%3DWaldrop%26aufirst%3DG.%2BL.%26atitle%3DMulti-Subunit%2520Acetyl-CoA%2520Carboxylases%26jtitle%3DProg.%2520Lipid%2520Res.%26date%3D2002%26volume%3D41%26spage%3D407%26epage%3D435%26doi%3D10.1016%2FS0163-7827%2802%2900007-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mochalkin, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narasimhan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thanabal, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erdman, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prasad, J. V. N. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lightle, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huband, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stover, C. K.</span></span> <span> </span><span class="NLM_article-title">Discovery of Antibacterial Biotin Carboxylase Inhibitors by Virtual Screening and Fragment-Based Approaches</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">473</span>– <span class="NLM_lpage">483</span>, <span class="refDoi"> DOI: 10.1021/cb9000102</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb9000102" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BD1MXmtVGgtbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2009&pages=473-483&author=I.+Mochalkinauthor=J.+R.+Millerauthor=L.+Narasimhanauthor=V.+Thanabalauthor=P.+Erdmanauthor=P.+B.+Coxauthor=J.+V.+N.+V.+Prasadauthor=S.+Lightleauthor=M.+D.+Hubandauthor=C.+K.+Stover&title=Discovery+of+Antibacterial+Biotin+Carboxylase+Inhibitors+by+Virtual+Screening+and+Fragment-Based+Approaches&doi=10.1021%2Fcb9000102"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Antibacterial Biotin Carboxylase Inhibitors by Virtual Screening and Fragment-Based Approaches</span></div><div class="casAuthors">Mochalkin, Igor; Miller, J. Richard; Narasimhan, Lakshmi; Thanabal, Venkataraman; Erdman, Paul; Cox, Philip B.; Prasad, J. V. N. Vara; Lightle, Sandra; Huband, Michael D.; Stover, C. Kendall</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">473-483</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">As part of an effort to inhibit bacterial fatty acid biosynthesis through the recently validated target biotin carboxylase, a unique combination of two emergent lead discovery strategies was employed.  Both de novo fragment-based drug discovery and virtual screening, which employs 3D shape and electrostatic property similarity searching were used.  A collection of unbiased low-mol.-wt. mols. was screened and a structurally diverse collection of weak-binding but ligand-efficient fragments were identified as potential building blocks for biotin carboxylase ATP-competitive inhibitors.  Through iterative cycles of structure-based drug design relying on successive fragment co-structures, the potency of the initial hits was improved by up to 3000-fold while maintaining their ligand-efficiency and desirable physicochem. properties.  In one example, hit-expansion efforts resulted in a series of amino-oxazoles with antibacterial activity.  These results successfully demonstrate that virtual screening approaches can substantially augment fragment-based screening approaches to identify novel antibacterial agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpX9pCqvTXGXLVg90H21EOLACvtfcHk0li_rlofhyesJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXmtVGgtbc%253D&md5=4f7e3886dd925d5b8c402e9072f2efcb</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Fcb9000102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb9000102%26sid%3Dliteratum%253Aachs%26aulast%3DMochalkin%26aufirst%3DI.%26aulast%3DMiller%26aufirst%3DJ.%2BR.%26aulast%3DNarasimhan%26aufirst%3DL.%26aulast%3DThanabal%26aufirst%3DV.%26aulast%3DErdman%26aufirst%3DP.%26aulast%3DCox%26aufirst%3DP.%2BB.%26aulast%3DPrasad%26aufirst%3DJ.%2BV.%2BN.%2BV.%26aulast%3DLightle%26aufirst%3DS.%26aulast%3DHuband%26aufirst%3DM.%2BD.%26aulast%3DStover%26aufirst%3DC.%2BK.%26atitle%3DDiscovery%2520of%2520Antibacterial%2520Biotin%2520Carboxylase%2520Inhibitors%2520by%2520Virtual%2520Screening%2520and%2520Fragment-Based%2520Approaches%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2009%26volume%3D4%26spage%3D473%26epage%3D483%26doi%3D10.1021%2Fcb9000102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shipps, G. W.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawahata, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soucy, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soriano, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orth, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mann, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Black, T.</span></span> <span> </span><span class="NLM_article-title">Discovery and Optimization of Antibacterial AccC Inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">6507</span>– <span class="NLM_lpage">6514</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2009.10.057</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1016%2Fj.bmcl.2009.10.057" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=19875284" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlGisLvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=6507-6514&author=C.+C.+Chengauthor=G.+W.+Shippsauthor=Z.+Yangauthor=B.+Sunauthor=N.+Kawahataauthor=K.+A.+Soucyauthor=A.+Sorianoauthor=P.+Orthauthor=L.+Xiaoauthor=P.+Mannauthor=T.+Black&title=Discovery+and+Optimization+of+Antibacterial+AccC+Inhibitors&doi=10.1016%2Fj.bmcl.2009.10.057"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and optimization of antibacterial AccC inhibitors</span></div><div class="casAuthors">Cheng, Cliff C.; Shipps, Gerald W.; Yang, Zhiwei; Sun, Binyuan; Kawahata, Noriyuki; Soucy, Kyle A.; Soriano, Aileen; Orth, Peter; Xiao, Li; Mann, Paul; Black, Todd</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">6507-6514</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Benzimidazolecarboxamides and imidazopyridinecarboxamides such as I (R first defined; R1 = H; X = N) and I (R second defined; R1 = O2N; X = CH) are prepd. by soln. and solid-phase methods as inhibitors of biotin carboxylase (AccC) for potential use as antibacterial agents.  For example, I (R first defined; R1 = H; X = N) and I (R second defined; R1 = O2N; X = CH) are potent and selective inhibitors of bacterial AccC with IC50 values of 20 nM and 10 nM, resp., and MIC values of 0.8 μg/mL against a sensitized strain of Escherichia coli (HS294 E. coli).  The structures of two benzimidazolecarboxamides bound to biotin carboxylase are detd. by X-ray crystallog.; the structure of an imidazopyridinecarboxamide bound to the active site of biotin carboxylase is detd. by computer-aided modeling calcns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgL8pY9SdU07Vg90H21EOLACvtfcHk0ljgJaNhpkS_7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlGisLvJ&md5=e811f3f617b9125bd852a4d65ff6e6c2</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2009.10.057&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2009.10.057%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DC.%2BC.%26aulast%3DShipps%26aufirst%3DG.%2BW.%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DSun%26aufirst%3DB.%26aulast%3DKawahata%26aufirst%3DN.%26aulast%3DSoucy%26aufirst%3DK.%2BA.%26aulast%3DSoriano%26aufirst%3DA.%26aulast%3DOrth%26aufirst%3DP.%26aulast%3DXiao%26aufirst%3DL.%26aulast%3DMann%26aufirst%3DP.%26aulast%3DBlack%26aufirst%3DT.%26atitle%3DDiscovery%2520and%2520Optimization%2520of%2520Antibacterial%2520AccC%2520Inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2009%26volume%3D19%26spage%3D6507%26epage%3D6514%26doi%3D10.1016%2Fj.bmcl.2009.10.057" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ogden, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuhn, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dority, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buist, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mehrens, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanna, D.</span></span> <span> </span><span class="NLM_article-title">Evaluation of Pharmacokinetic/Pharmacodynamic Relationships of PD-0162819, a Biotin Carboxylase Inhibitor Representing a New Class of Antibacterial Compounds, Using <i>In Vitro</i> Infection Models</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">124</span>– <span class="NLM_lpage">129</span>, <span class="refDoi"> DOI: 10.1128/AAC.00090-11</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1128%2FAAC.00090-11" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=21986824" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BC38Xjt1Gksw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2012&pages=124-129&author=A.+Ogdenauthor=M.+Kuhnauthor=M.+Dorityauthor=S.+Buistauthor=S.+Mehrensauthor=T.+Zhuauthor=D.+Xiaoauthor=J.+R.+Millerauthor=D.+Hanna&title=Evaluation+of+Pharmacokinetic%2FPharmacodynamic+Relationships+of+PD-0162819%2C+a+Biotin+Carboxylase+Inhibitor+Representing+a+New+Class+of+Antibacterial+Compounds%2C+Using+In+Vitro+Infection+Models&doi=10.1128%2FAAC.00090-11"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluation of pharmacokinetic/pharmacodynamic relationships of PD-0162819, a biotin carboxylase inhibitor representing a new class of antibacterial compounds, using in vitro infection models</span></div><div class="casAuthors">Ogden, Adam; Kuhn, Michael; Dority, Michael; Buist, Susan; Mehrens, Shawn; Zhu, Tong; Xiao, Deqing; Miller, J. Richard; Hanna, Debra</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">124-129</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The present study investigated the pharmacokinetic/pharmacodynamic (PK/PD) relationships of a prototype biotin carboxylase (BC) inhibitor, PD-0162819, against Haemophilus influenzae 3113 in static concn. time-kill (SCTK) and one-compartment chemostat in vitro infection models.  H. influenzae 3113 was exposed to PD-0162819 concns. of 0.5 to 16 × the MIC (MIC = 0.125 μg/mL) and area-under-the-curve (AUC)/MIC ratios of 1 to 1,100 in SCTK and chemostat expts., resp.  Serial samples were collected over 24 h.  For efficacy driver anal., a sigmoid max.-effect (Emax) model was fitted to the relationship between bacterial d. changes over 24 h and corresponding PK/PD indexes.  A semi-mechanistic PK/PD model describing the time course of bacterial growth and death was developed.  The AUC/MIC ratio best explained efficacy (r2 = 0.95) compared to the peak drug concn. (Cmax)/MIC ratio (r2 = 0.76) and time above the MIC (T>MIC) (r2 = 0.88).  Static effects and 99.9% killing were achieved at AUC/MIC values of 500 and 600, resp.  For time course anal., the net bacterial growth rate const., max. bacterial d., and max. kill rate const. were similar in SCTK and chemostat studies, but PD-0162819 was more potent in SCTK than in the chemostat (50% effective concn. [EC50] = 0.046 vs. 0.34 μg/mL).  In conclusion, basic PK/PD relationships for PD-0162819 were established using in vitro dynamic systems.  Although the bacterial growth parameters and max. drug effects were similar in SCTK and the chemostat system, PD-0162819 appeared to be more potent in SCTK, illustrating the importance of understanding the differences in preclin. models.  Addnl. studies are needed to det. the in vivo relevance of these results.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqpvJt2Z1rgYLVg90H21EOLACvtfcHk0ljgJaNhpkS_7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xjt1Gksw%253D%253D&md5=31471262f87a14100e07c9024cb70b0f</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1128%2FAAC.00090-11&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.00090-11%26sid%3Dliteratum%253Aachs%26aulast%3DOgden%26aufirst%3DA.%26aulast%3DKuhn%26aufirst%3DM.%26aulast%3DDority%26aufirst%3DM.%26aulast%3DBuist%26aufirst%3DS.%26aulast%3DMehrens%26aufirst%3DS.%26aulast%3DZhu%26aufirst%3DT.%26aulast%3DXiao%26aufirst%3DD.%26aulast%3DMiller%26aufirst%3DJ.%2BR.%26aulast%3DHanna%26aufirst%3DD.%26atitle%3DEvaluation%2520of%2520Pharmacokinetic%252FPharmacodynamic%2520Relationships%2520of%2520PD-0162819%252C%2520a%2520Biotin%2520Carboxylase%2520Inhibitor%2520Representing%2520a%2520New%2520Class%2520of%2520Antibacterial%2520Compounds%252C%2520Using%2520In%2520Vitro%2520Infection%2520Models%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2012%26volume%3D56%26spage%3D124%26epage%3D129%26doi%3D10.1128%2FAAC.00090-11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Silvers, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robertson, G. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldrop, G. L.</span></span> <span> </span><span class="NLM_article-title">Design, Synthesis, and Antibacterial Properties of Dual-Ligand Inhibitors of Acetyl-CoA Carboxylase</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">8947</span>– <span class="NLM_lpage">8959</span>, <span class="refDoi"> DOI: 10.1021/jm501082n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501082n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1OktLjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=8947-8959&author=M.+A.+Silversauthor=G.+T.+Robertsonauthor=C.+M.+Taylorauthor=G.+L.+Waldrop&title=Design%2C+Synthesis%2C+and+Antibacterial+Properties+of+Dual-Ligand+Inhibitors+of+Acetyl-CoA+Carboxylase&doi=10.1021%2Fjm501082n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Design, Synthesis, and Antibacterial Properties of Dual-Ligand Inhibitors of Acetyl-CoA Carboxylase</span></div><div class="casAuthors">Silvers, Molly A.; Robertson, Gregory T.; Taylor, Carol M.; Waldrop, Grover L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">8947-8959</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">There is an urgent demand for the development of new antibiotics due to the increase in drug-resistant pathogenic bacteria.  A novel target is the multifunctional enzyme acetyl-CoA carboxylase (ACC), which catalyzes the first committed step in fatty acid synthesis and consists of two enzymes: biotin carboxylase and carboxyltransferase.  Covalently attaching known inhibitors against these enzymes with satd. hydrocarbon linkers of different lengths generated dual-ligand inhibitors.  Kinetic results revealed that the dual-ligands inhibited the ACC complex in the nanomolar range.  Microbiol. assays showed that the dual-ligand with a 15-carbon linker did not exhibit any antibacterial activity, while the dual-ligand with a 7-carbon linker displayed broad-spectrum antibacterial activity as well as a decreased susceptibility in the development of bacterial resistance.  These results suggest that the properties of the linker are vital for antibacterial activity and show how inhibiting two different enzymes with the same compd. increases the overall potency while also impeding the development of resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYCRRNvvd6ULVg90H21EOLACvtfcHk0lgqem5uxH47kQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1OktLjL&md5=4e328b5949c3283f33b25baf6f8212f4</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fjm501082n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501082n%26sid%3Dliteratum%253Aachs%26aulast%3DSilvers%26aufirst%3DM.%2BA.%26aulast%3DRobertson%26aufirst%3DG.%2BT.%26aulast%3DTaylor%26aufirst%3DC.%2BM.%26aulast%3DWaldrop%26aufirst%3DG.%2BL.%26atitle%3DDesign%252C%2520Synthesis%252C%2520and%2520Antibacterial%2520Properties%2520of%2520Dual-Ligand%2520Inhibitors%2520of%2520Acetyl-CoA%2520Carboxylase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D8947%26epage%3D8959%26doi%3D10.1021%2Fjm501082n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wallace, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bowlin, N. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mills, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saenkham, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwasny, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Opperman, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rock, C. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bowlin, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moir, D. T.</span></span> <span> </span><span class="NLM_article-title">Discovery of Bacterial Fatty Acid Synthase Type II Inhibitors Using a Novel Cellular Bioluminescent Reporter Assay</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">5775</span>– <span class="NLM_lpage">5787</span>, <span class="refDoi"> DOI: 10.1128/AAC.00686-15</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1128%2FAAC.00686-15" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=26169404" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlOltLzN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2015&pages=5775-5787&author=J.+Wallaceauthor=N.+O.+Bowlinauthor=D.+M.+Millsauthor=P.+Saenkhamauthor=S.+M.+Kwasnyauthor=T.+J.+Oppermanauthor=J.+D.+Williamsauthor=C.+O.+Rockauthor=T.+L.+Bowlinauthor=D.+T.+Moir&title=Discovery+of+Bacterial+Fatty+Acid+Synthase+Type+II+Inhibitors+Using+a+Novel+Cellular+Bioluminescent+Reporter+Assay&doi=10.1128%2FAAC.00686-15"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of bacterial fatty acid synthase type II inhibitors using a novel cellular bioluminescent reporter assay</span></div><div class="casAuthors">Wallace, Joselynn; Bowlin, Nicholas O.; Mills, Debra M.; Saenkham, Panatda; Kwasny, Steven M.; Opperman, Timothy J.; Williams, John D.; Rock, Charles O.; Bowlin, Terry L.; Moir, Donald T.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">5775-5787</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">1098-6596</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Novel, cellular, gain-of-signal, bioluminescent reporter assays for fatty acid synthesis type II (FASII) inhibitors were constructed in an efflux-deficient strain of Pseudomonas aeruginosa and based on the discovery that FASII genes in P. aeruginosa are coordinately upregulated in response to pathway disruption.  A screen of 115,000 compds. identified a series of sulfonamidobenzamide (SABA) analogs, which generated strong luminescent signals in two FASII reporter strains but not in four control reporter strains designed to respond to inhibitors of pathways other than FASII.  The SABA analogs selectively inhibited lipid biosynthesis in P. aeruginosa and exhibited minimal cytotoxicity to mammalian cells (50% cytotoxic concn. [CC50] ≥ 80 μM).  The most potent SABA analogs had MICs of 0.5 to 7.0 μM (0.2 to 3.0 μg/mL) against an efflux-deficient Escherichia coli (ΔtolC) strain but had no detectable MIC against efflux-proficient E. coli or against P. aeruginosa (efflux deficient or proficient).  Genetic, mol. genetic, and biochem. studies revealed that SABA analogs target the enzyme (AccC) catalyzing the biotin carboxylase half-reaction of the acetyl CoA (acetyl-CoA) carboxylase step in the initiation phase of FASII in E. coli and P. aeruginosa.  These results validate the capability and the sensitivity of this novel bioluminescent reporter screen to identify inhibitors of E. coli and P. aeruginosa FASII.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbxuGbUkG3vrVg90H21EOLACvtfcHk0lgqem5uxH47kQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlOltLzN&md5=80cbafab9c43b2930bbd3bf0b976dddc</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1128%2FAAC.00686-15&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.00686-15%26sid%3Dliteratum%253Aachs%26aulast%3DWallace%26aufirst%3DJ.%26aulast%3DBowlin%26aufirst%3DN.%2BO.%26aulast%3DMills%26aufirst%3DD.%2BM.%26aulast%3DSaenkham%26aufirst%3DP.%26aulast%3DKwasny%26aufirst%3DS.%2BM.%26aulast%3DOpperman%26aufirst%3DT.%2BJ.%26aulast%3DWilliams%26aufirst%3DJ.%2BD.%26aulast%3DRock%26aufirst%3DC.%2BO.%26aulast%3DBowlin%26aufirst%3DT.%2BL.%26aulast%3DMoir%26aufirst%3DD.%2BT.%26atitle%3DDiscovery%2520of%2520Bacterial%2520Fatty%2520Acid%2520Synthase%2520Type%2520II%2520Inhibitors%2520Using%2520a%2520Novel%2520Cellular%2520Bioluminescent%2520Reporter%2520Assay%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2015%26volume%3D59%26spage%3D5775%26epage%3D5787%26doi%3D10.1128%2FAAC.00686-15" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mochalkin, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evdokimov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lightle, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stover, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldrop, G. L.</span></span> <span> </span><span class="NLM_article-title">Structural Evidence for Substrate-Induced Synergism and Half-Sites Reactivity in Biotin Carboxylase</span>. <i>Protein Sci.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">1706</span>– <span class="NLM_lpage">1718</span>, <span class="refDoi"> DOI: 10.1110/ps.035584.108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1110%2Fps.035584.108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=18725455" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BD1cXht1Cit7zI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2008&pages=1706-1718&author=I.+Mochalkinauthor=J.+R.+Millerauthor=A.+Evdokimovauthor=S.+Lightleauthor=C.+Yanauthor=C.+K.+Stoverauthor=G.+L.+Waldrop&title=Structural+Evidence+for+Substrate-Induced+Synergism+and+Half-Sites+Reactivity+in+Biotin+Carboxylase&doi=10.1110%2Fps.035584.108"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Structural evidence for substrate-induced synergism and half-sites reactivity in biotin carboxylase</span></div><div class="casAuthors">Mochalkin, Igor; Miller, J. Richard; Evdokimov, Artem; Lightle, Sandra; Yan, Chunhong; Stover, Charles Ken; Waldrop, Grover L.</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1706-1718</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">0961-8368</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Bacterial acetyl-CoA carboxylase is a multifunctional biotin-dependent enzyme that consists of three sep. proteins: biotin carboxylase (BC), biotin carboxyl carrier protein (BCCP), and carboxyltransferase (CT).  Acetyl-CoA carboxylase is a potentially attractive target for novel antibiotics because it catalyzes the first committed step in fatty acid biosynthesis.  In the first half-reaction, BC catalyzes the ATP-dependent carboxylation of BCCP.  In the second half-reaction, the carboxyl group is transferred from carboxybiotinylated BCCP to acetyl-CoA to produce malonyl-CoA.  A series of structures of BC from several bacteria crystd. in the presence of various ATP analogs is described that addresses three major questions concerning the catalytic mechanism.  The structure of BC bound to AMPPNP and the two catalytically essential magnesium ions resolves inconsistencies between the kinetics of active-site BC mutants and previously reported BC structures.  Another structure of AMPPNP bound to BC shows the polyphosphate chain folded back on itself, and not in the correct (i.e., extended) conformation for catalysis.  This provides the first structural evidence for the hypothesis of substrate-induced synergism, which posits that ATP binds nonproductively to BC in the absence of biotin.  The BC homodimer has been proposed to exhibit half-sites reactivity where the active sites alternate or "flip-flop" their catalytic cycles.  A crystal structure of BC showed the ATP analog AMPPCF2P bound to one subunit while the other subunit was unliganded.  The liganded subunit was in the closed or catalytic conformation while the unliganded subunit was in the open conformation.  This provides the first structural evidence for half-sites reactivity in BC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2hQakHeDxTLVg90H21EOLACvtfcHk0liW9G9BfDcRiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXht1Cit7zI&md5=93758000a39e0bdc6cb04f479529d58f</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1110%2Fps.035584.108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1110%252Fps.035584.108%26sid%3Dliteratum%253Aachs%26aulast%3DMochalkin%26aufirst%3DI.%26aulast%3DMiller%26aufirst%3DJ.%2BR.%26aulast%3DEvdokimov%26aufirst%3DA.%26aulast%3DLightle%26aufirst%3DS.%26aulast%3DYan%26aufirst%3DC.%26aulast%3DStover%26aufirst%3DC.%2BK.%26aulast%3DWaldrop%26aufirst%3DG.%2BL.%26atitle%3DStructural%2520Evidence%2520for%2520Substrate-Induced%2520Synergism%2520and%2520Half-Sites%2520Reactivity%2520in%2520Biotin%2520Carboxylase%26jtitle%3DProtein%2520Sci.%26date%3D2008%26volume%3D17%26spage%3D1706%26epage%3D1718%26doi%3D10.1110%2Fps.035584.108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nikaido, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaara, M.</span></span> <span> </span><span class="NLM_article-title">Molecular Basis of Bacterial Outer Membrane Permeability</span>. <i>Microbiol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">1985</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">32</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1128%2FMMBR.49.1.1-32.1985" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=2580220" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADyaL2MXhvVWqur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=1985&pages=1-32&author=H.+Nikaidoauthor=M.+Vaara&title=Molecular+Basis+of+Bacterial+Outer+Membrane+Permeability"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular basis of bacterial outer membrane permeability</span></div><div class="casAuthors">Nikaido, Hiroshi; Vaara, Marti</div><div class="citationInfo"><span class="NLM_cas:title">Microbiological Reviews</span>
        (<span class="NLM_cas:date">1985</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-32</span>CODEN:
                <span class="NLM_cas:coden">MBRED3</span>;
        ISSN:<span class="NLM_cas:issn">0146-0749</span>.
    </div><div class="casAbstract">A review with 344 refs. on the chem. compn. of outer membrane, porins as nonspecific protein channels, specific diffusion channels, lipid bilayer as a diffusion barrier, and fusion sites between outer and inner membranes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqeq6AtZlquPrVg90H21EOLACvtfcHk0liW9G9BfDcRiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2MXhvVWqur4%253D&md5=e0d9879bb50378f4add4df159aa2b5a2</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1128%2FMMBR.49.1.1-32.1985&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMMBR.49.1.1-32.1985%26sid%3Dliteratum%253Aachs%26aulast%3DNikaido%26aufirst%3DH.%26aulast%3DVaara%26aufirst%3DM.%26atitle%3DMolecular%2520Basis%2520of%2520Bacterial%2520Outer%2520Membrane%2520Permeability%26jtitle%3DMicrobiol.%2520Rev.%26date%3D1985%26volume%3D49%26spage%3D1%26epage%3D32" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nikaido, H.</span></span> <span> </span><span class="NLM_article-title">Prevention of Drug Access to Bacterial Targets: Permeability Barriers and Active Efflux</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>264</i></span>,  <span class="NLM_fpage">382</span>– <span class="NLM_lpage">388</span>, <span class="refDoi"> DOI: 10.1126/science.8153625</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1126%2Fscience.8153625" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=8153625" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADyaK2cXis1Ggurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=264&publication_year=1994&pages=382-388&author=H.+Nikaido&title=Prevention+of+Drug+Access+to+Bacterial+Targets%3A+Permeability+Barriers+and+Active+Efflux&doi=10.1126%2Fscience.8153625"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Prevention of drug access to bacterial targets: permeability barriers and active efflux</span></div><div class="casAuthors">Nikaido, Hiroshi</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">264</span>
        (<span class="NLM_cas:issue">5157</span>),
    <span class="NLM_cas:pages">382-8</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    </div><div class="casAbstract">A review with 79 refs.  Some species of bacteria have low-permeability membrane barriers and are thereby intrinsically resistant to many antibiotics.  Some strains of originally antibiotic-susceptible species may also acquire resistance through decreases in the permeability of membrane barriers.  Another mechanism for preventing access of drugs to targets is the membrane-assocd. energy-driven efflux, which plays a major role in drug resistance, esp. in combination with the permeation barrier.  Recent results indicate the existence of bacterial efflux systems of extremely broad substrate specificity, in many ways reminiscent of the multidrug resistance pump of mammalian cells.  One such system seems to play a major role in the intrinsic resistance of Pseudomonas aeruginosa, a common opportunistic pathogen.  As the pharmacol. industry succeeds in producing agents that can overcome specific mechanisms of bacterial resistance, less specific resistance mechanisms such as permeability barriers and multidrug active efflux may become increasingly significant in the clin. setting.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYpbgrhsj0ZrVg90H21EOLACvtfcHk0liW9G9BfDcRiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXis1Ggurc%253D&md5=84eedec2b82bfc061f17d0d5b4e6973d</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1126%2Fscience.8153625&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.8153625%26sid%3Dliteratum%253Aachs%26aulast%3DNikaido%26aufirst%3DH.%26atitle%3DPrevention%2520of%2520Drug%2520Access%2520to%2520Bacterial%2520Targets%253A%2520Permeability%2520Barriers%2520and%2520Active%2520Efflux%26jtitle%3DScience%26date%3D1994%26volume%3D264%26spage%3D382%26epage%3D388%26doi%3D10.1126%2Fscience.8153625" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schweizer, H. P.</span></span> <span> </span><span class="NLM_article-title">Bacterial Resistance to Antibiotics: Active Efflux and Reduced Uptake</span>. <i>Adv. Drug Delivery Rev.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">1486</span>– <span class="NLM_lpage">1513</span>, <span class="refDoi"> DOI: 10.1016/j.addr.2005.04.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1016%2Fj.addr.2005.04.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=15939505" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BD2MXlvFant70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2005&pages=1486-1513&author=A.+Kumarauthor=H.+P.+Schweizer&title=Bacterial+Resistance+to+Antibiotics%3A+Active+Efflux+and+Reduced+Uptake&doi=10.1016%2Fj.addr.2005.04.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Bacterial resistance to antibiotics: Active efflux and reduced uptake</span></div><div class="casAuthors">Kumar, Ayush; Schweizer, Herbert P.</div><div class="citationInfo"><span class="NLM_cas:title">Advanced Drug Delivery Reviews</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1486-1513</span>CODEN:
                <span class="NLM_cas:coden">ADDREP</span>;
        ISSN:<span class="NLM_cas:issn">0169-409X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Antibiotic resistance of bacterial pathogens is a fast emerging global crisis and an understanding of the underlying resistance mechanisms is paramount for the design and development of new therapeutic strategies.  Permeability barriers for and active efflux of drug mols. are 2 resistance mechanisms that have been implicated in various infectious outbreaks of antibiotic-resistant pathogens, suggesting that these mechanisms may be good targets for new drugs.  The synergism of reduced uptake and efflux is most evident in the multiplicative action of the outer membrane permeability barrier and active efflux, which results in high-level intrinsic and(or) acquired resistance in many clin. important gram-neg. bacteria.  This review summarizes the current knowledge of these 2 important resistance mechanisms and potential strategies to overcome them.  Recent advances in understanding the phys. structures, function, and regulation of efflux systems will facilitate exploitation of pumps as new drug targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDE1hxxbXPQrVg90H21EOLACvtfcHk0liW9G9BfDcRiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXlvFant70%253D&md5=b1b9bed1deab6c70f83062d73abafc4f</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.addr.2005.04.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.addr.2005.04.004%26sid%3Dliteratum%253Aachs%26aulast%3DKumar%26aufirst%3DA.%26aulast%3DSchweizer%26aufirst%3DH.%2BP.%26atitle%3DBacterial%2520Resistance%2520to%2520Antibiotics%253A%2520Active%2520Efflux%2520and%2520Reduced%2520Uptake%26jtitle%3DAdv.%2520Drug%2520Delivery%2520Rev.%26date%3D2005%26volume%3D57%26spage%3D1486%26epage%3D1513%26doi%3D10.1016%2Fj.addr.2005.04.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brown, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">May-Dracka, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gagnon, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tommasi, R.</span></span> <span> </span><span class="NLM_article-title">Trends and Exceptions of Physical Properties on Antibacterial Activity for Gram-Positive and Gram-Negative Pathogens</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">10144</span>– <span class="NLM_lpage">10161</span>, <span class="refDoi"> DOI: 10.1021/jm501552x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501552x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFehtbvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=10144-10161&author=D.+G.+Brownauthor=T.+L.+May-Drackaauthor=M.+M.+Gagnonauthor=R.+Tommasi&title=Trends+and+Exceptions+of+Physical+Properties+on+Antibacterial+Activity+for+Gram-Positive+and+Gram-Negative+Pathogens&doi=10.1021%2Fjm501552x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Trends and Exceptions of Physical Properties on Antibacterial Activity for Gram-Positive and Gram-Negative Pathogens</span></div><div class="casAuthors">Brown, Dean G.; May-Dracka, Tricia L.; Gagnon, Moriah M.; Tommasi, Ruben</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">10144-10161</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">To better understand the difficulties surrounding the identification of novel antibacterial compds. from corporate screening collections, phys. properties of ∼3200 antibacterial project compds. with whole cell activity against Gram-neg. or Gram-pos. pathogens were profiled and compared to actives found from high throughput (HTS) screens conducted on both biochem. and phenotypic bacterial targets.  The output from 23 antibacterial HTS screens illustrated that when compared to the properties of the antibacterial project compds., the HTS actives were significantly more hydrophobic than antibacterial project compds. (typically 2-4 log units higher), and furthermore, for 14/23 HTS screens, the av. clogD was higher than the screening collection av. (screening collection clogD = 2.45).  It was found that the consequences of this were the following: (a) lead identification programs often further gained hydrophobic character with increased biochem. potency, making the sepn. even larger between the physicochem. properties of known antibacterial agents and the HTS active starting point, (b) the probability of plasma protein binding and cytotoxicity are often increased, and (c) cell-based activity in Gram-neg. bacteria was severely limited or, if present, demonstrated significant efflux.  Our anal. illustrated that compds. least susceptible to efflux were those which were highly polar and small in MW or very large and typically zwitterionic.  Hydrophobicity was often the dominant driver for HTS actives but, more often than not, precluded whole cell antibacterial activity.  However, simply designing polar compds. was not sufficient for antibacterial activity and pointed to a lack of understanding of complex and specific bacterial penetration mechanisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-f01NZg_bsbVg90H21EOLACvtfcHk0lgR45Bb4o9igg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFehtbvN&md5=e107961414dc8cbf696e2287ddf9362d</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fjm501552x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501552x%26sid%3Dliteratum%253Aachs%26aulast%3DBrown%26aufirst%3DD.%2BG.%26aulast%3DMay-Dracka%26aufirst%3DT.%2BL.%26aulast%3DGagnon%26aufirst%3DM.%2BM.%26aulast%3DTommasi%26aufirst%3DR.%26atitle%3DTrends%2520and%2520Exceptions%2520of%2520Physical%2520Properties%2520on%2520Antibacterial%2520Activity%2520for%2520Gram-Positive%2520and%2520Gram-Negative%2520Pathogens%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D10144%26epage%3D10161%26doi%3D10.1021%2Fjm501552x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tommasi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walkup, G. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manchester, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, A. A.</span></span> <span> </span><span class="NLM_article-title">ESKAPEing the Labyrinth of Antibacterial Discovery</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">529</span>– <span class="NLM_lpage">542</span>, <span class="refDoi"> DOI: 10.1038/nrd4572</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1038%2Fnrd4572" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=26139286" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFyks77M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2015&pages=529-542&author=R.+Tommasiauthor=D.+G.+Brownauthor=G.+K.+Walkupauthor=J.+I.+Manchesterauthor=A.+A.+Miller&title=ESKAPEing+the+Labyrinth+of+Antibacterial+Discovery&doi=10.1038%2Fnrd4572"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">ESKAPEing the labyrinth of antibacterial discovery</span></div><div class="casAuthors">Tommasi, Ruben; Brown, Dean G.; Walkup, Grant K.; Manchester, John I.; Miller, Alita A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">529-542</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Antimicrobial drug resistance is a growing threat to global public health.  Multidrug resistance among the 'ESKAPE' organisms - encompassing Enterococcus faecium, Staphylococcus aureus, Klebsiella pneumoniae, Acinetobacter baumannii, Pseudomonas aeruginosa and Enterobacter spp. - is of particular concern because they are responsible for many serious infections in hospitals.  Although some promising agents are in the pipeline, there is an urgent need for new antibiotic scaffolds.  However, antibacterial researchers have struggled to identify new small mols. with meaningful cellular activity, esp. those effective against multidrug-resistant Gram-neg. pathogens.  This difficulty ultimately stems from an incomplete understanding of efflux systems and compd. permeation through bacterial membranes.  This Opinion article describes findings from target-based and phenotypic screening efforts carried out at AstraZeneca over the past decade, discusses some of the subsequent chem. challenges and concludes with a description of new approaches comprising a combination of computational modeling and advanced biol. tools which may pave the way towards the discovery of new antibacterial agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolJg6HXnn7BLVg90H21EOLACvtfcHk0lgR45Bb4o9igg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFyks77M&md5=0d3159250e27926ab280c305ea5fb9ef</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fnrd4572&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4572%26sid%3Dliteratum%253Aachs%26aulast%3DTommasi%26aufirst%3DR.%26aulast%3DBrown%26aufirst%3DD.%2BG.%26aulast%3DWalkup%26aufirst%3DG.%2BK.%26aulast%3DManchester%26aufirst%3DJ.%2BI.%26aulast%3DMiller%26aufirst%3DA.%2BA.%26atitle%3DESKAPEing%2520the%2520Labyrinth%2520of%2520Antibacterial%2520Discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2015%26volume%3D14%26spage%3D529%26epage%3D542%26doi%3D10.1038%2Fnrd4572" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mugumbate, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overington, J. P.</span></span> <span> </span><span class="NLM_article-title">The Relationship Between Target-Class and the Physicochemical Properties of Antibacterial Drugs</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">5218</span>– <span class="NLM_lpage">5224</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2015.04.063</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1016%2Fj.bmc.2015.04.063" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=25975639" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BC2MXnslOjs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2015&pages=5218-5224&author=G.+Mugumbateauthor=J.+P.+Overington&title=The+Relationship+Between+Target-Class+and+the+Physicochemical+Properties+of+Antibacterial+Drugs&doi=10.1016%2Fj.bmc.2015.04.063"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">The relationship between target-class and the physicochemical properties of antibacterial drugs</span></div><div class="casAuthors">Mugumbate, Grace; Overington, John P.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">5218-5224</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The discovery of novel mechanism of action (MOA) antibacterials has been assocd. with the concept that antibacterial drugs occupy a differentiated region of physicochem. space compared to human-targeted drugs.  With, in broad terms, antibacterials having higher mol. wt., lower log P and higher polar surface area (PSA).  By analyzing the physicochem. properties of about 1700 approved drugs listed in the ChEMBL database, we show, that antibacterials for whose targets are riboproteins (i.e., composed of a complex of RNA and protein) fall outside the conventional human 'drug-like' chem. space; whereas antibacterials that modulate bacterial protein targets, generally comply with the 'rule-of-five' guidelines for classical oral human drugs.  Our anal. suggests a strong target-class assocn. for antibacterials - either protein-targeted or riboprotein-targeted.  There is much discussion in the literature on the failure of screening approaches to deliver novel antibacterial lead series, and linkage of this poor success rate for antibacterials with the chem. space properties of screening collections.  Our anal. suggests that consideration of target-class may be an underappreciated factor in antibacterial lead discovery, and that in fact bacterial protein-targets may well have similar binding site characteristics to human protein targets, and questions the assumption that larger, more polar compds. are a key part of successful future antibacterial discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0zRzPd_fDo7Vg90H21EOLACvtfcHk0lhRklI67D84AQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXnslOjs7k%253D&md5=b6d32663c593fb09e1235968a5e5096c</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2015.04.063&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2015.04.063%26sid%3Dliteratum%253Aachs%26aulast%3DMugumbate%26aufirst%3DG.%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26atitle%3DThe%2520Relationship%2520Between%2520Target-Class%2520and%2520the%2520Physicochemical%2520Properties%2520of%2520Antibacterial%2520Drugs%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2015%26volume%3D23%26spage%3D5218%26epage%3D5224%26doi%3D10.1016%2Fj.bmc.2015.04.063" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">O’Shea, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moser, H. E.</span></span> <span> </span><span class="NLM_article-title">Physicochemical Properties of Antibacterial Compounds: Implications for Drug Discovery</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">2871</span>– <span class="NLM_lpage">2878</span>, <span class="refDoi"> DOI: 10.1021/jm700967e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm700967e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlvFCrsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=2871-2878&author=R.+O%E2%80%99Sheaauthor=H.+E.+Moser&title=Physicochemical+Properties+of+Antibacterial+Compounds%3A+Implications+for+Drug+Discovery&doi=10.1021%2Fjm700967e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Physicochemical properties of antibacterial compounds: implications for drug discovery</span></div><div class="casAuthors">O'Shea, Rosemarie; Moser, Heinz E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2871-2878</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">With the rise of multidrug-resistant pathogens and the need for novel antibiotics, it is crit. to understand as much as possible from prior efforts and to apply learned lessons to the discovery of future antibiotics.  One important parameter in particular has previously been mentioned but, in the view, not sufficiently analyzed: the physicochem. property space of antibacterial drugs.  The authors selected 147 antibacterially active compds. that encompass both currently used drugs and compds. that are still under clin. investigation (see Methods for details).  Where available, other property values were extd. from the literature, including protein binding and oral bioavailability in humans.  This anal. suggests that natural products should be increasingly investigated again to identify novel antibacterial hits.  Besides their high level of structural diversity, they are likely to better cover the required physicochem. property space for antibacterial compds. compared to synthetic mols. because of an increased d. of polar functionalities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo25EGWMkWdk7Vg90H21EOLACvtfcHk0lhRklI67D84AQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlvFCrsLw%253D&md5=8aae7d962df53b70113e0fe6169abd68</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fjm700967e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm700967e%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Shea%26aufirst%3DR.%26aulast%3DMoser%26aufirst%3DH.%2BE.%26atitle%3DPhysicochemical%2520Properties%2520of%2520Antibacterial%2520Compounds%253A%2520Implications%2520for%2520Drug%2520Discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D2871%26epage%3D2878%26doi%3D10.1021%2Fjm700967e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Richter, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drown, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riley, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shirai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svec, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hergenrother, P. J.</span></span> <span> </span><span class="NLM_article-title">Predictive Compound Accumulation Rules Yield a Broad-Spectrum Antibiotic</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>545</i></span>,  <span class="NLM_fpage">299</span>– <span class="NLM_lpage">304</span>, <span class="refDoi"> DOI: 10.1038/nature22308</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1038%2Fnature22308" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=28489819" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BC2sXnsVeis7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=545&publication_year=2017&pages=299-304&author=M.+F.+Richterauthor=B.+S.+Drownauthor=A.+P.+Rileyauthor=A.+Garciaauthor=T.+Shiraiauthor=R.+L.+Svecauthor=P.+J.+Hergenrother&title=Predictive+Compound+Accumulation+Rules+Yield+a+Broad-Spectrum+Antibiotic&doi=10.1038%2Fnature22308"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Predictive compound accumulation rules yield a broad-spectrum antibiotic</span></div><div class="casAuthors">Richter, Michelle F.; Drown, Bryon S.; Riley, Andrew P.; Garcia, Alfredo; Shirai, Tomohiro; Svec, Riley L.; Hergenrother, Paul J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">545</span>
        (<span class="NLM_cas:issue">7654</span>),
    <span class="NLM_cas:pages">299-304</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Most small mols. are unable to rapidly traverse the outer membrane of Gram-neg. bacteria and accumulate inside these cells, making the discovery of much-needed drugs against these pathogens challenging.  Current understanding of the physicochem. properties that dictate small-mol. accumulation in Gram-neg. bacteria is largely based on retrospective analyses of antibacterial agents, which suggest that polarity and mol. wt. are key factors.  Here the authors assess the ability of over 180 diverse compds. to accumulate in Escherichia coli.  Computational anal. of the results reveals major differences from the retrospective studies, namely that the small mols. that are most likely to accumulate contain an amine, are amphiphilic and rigid, and have low globularity.  These guidelines were then applied to convert deoxynybomycin, a natural product that is active only against Gram-pos. organisms, into an antibiotic with activity against a diverse panel of multi-drug-resistant Gram-neg. pathogens.  The authors anticipate that these findings will aid in the discovery and development of antibiotics against Gram-neg. bacteria.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGot2r9sFv94TrVg90H21EOLACvtfcHk0lhRklI67D84AQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXnsVeis7Y%253D&md5=ae869cc99ef0e6805b96b6595a222cdc</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fnature22308&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature22308%26sid%3Dliteratum%253Aachs%26aulast%3DRichter%26aufirst%3DM.%2BF.%26aulast%3DDrown%26aufirst%3DB.%2BS.%26aulast%3DRiley%26aufirst%3DA.%2BP.%26aulast%3DGarcia%26aufirst%3DA.%26aulast%3DShirai%26aufirst%3DT.%26aulast%3DSvec%26aufirst%3DR.%2BL.%26aulast%3DHergenrother%26aufirst%3DP.%2BJ.%26atitle%3DPredictive%2520Compound%2520Accumulation%2520Rules%2520Yield%2520a%2520Broad-Spectrum%2520Antibiotic%26jtitle%3DNature%26date%3D2017%26volume%3D545%26spage%3D299%26epage%3D304%26doi%3D10.1038%2Fnature22308" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Serio, A. W.</span>; <span class="NLM_string-name">Kubo, A.</span>; <span class="NLM_string-name">Lopez, S.</span>; <span class="NLM_string-name">Gomez, M.</span>; <span class="NLM_string-name">Corey, V. C.</span>; <span class="NLM_string-name">Andrews, L. D.</span>; <span class="NLM_string-name">Schwatz, M. A.</span>; <span class="NLM_string-name">Kasar, R.</span>; <span class="NLM_string-name">McEnRoe, G.</span>; <span class="NLM_string-name">Aggen, J.</span>; <span class="NLM_string-name">Miller, G.</span>; <span class="NLM_string-name">Armstrong, E. S.</span></span> <span> </span><span class="NLM_article-title">Structure, Potency and Bactericidal Activity of ACHN-975, a First-In-Class LpxC Inhibitor</span>. In  <i>53rd Interscience Conference on Antimicrobial Agents and Chemotherapy</i>; <span class="NLM_publisher-name">Achaogen</span>, <span class="NLM_year">2013</span> [poster #F-1226].</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=A.+W.+Serio&author=A.+Kubo&author=S.+Lopez&author=M.+Gomez&author=V.+C.+Corey&author=L.+D.+Andrews&author=M.+A.+Schwatz&author=R.+Kasar&author=G.+McEnRoe&author=J.+Aggen&author=G.+Miller&author=E.+S.+Armstrong&title=Structure%2C+Potency+and+Bactericidal+Activity+of+ACHN-975%2C+a+First-In-Class+LpxC+Inhibitor"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dconf-proc%26sid%3Dliteratum%253Aachs%26aulast%3DSerio%26aufirst%3DA.%2BW.%26atitle%3DStructure%252C%2520Potency%2520and%2520Bactericidal%2520Activity%2520of%2520ACHN-975%252C%2520a%2520First-In-Class%2520LpxC%2520Inhibitor%26jtitle%3D53rd%2520Interscience%2520Conference%2520on%2520Antimicrobial%2520Agents%2520and%2520Chemotherapy%26pub%3DAchaogen%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tari, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trzoss, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bensen, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Creighton, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cunningham, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stidham, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lightstone, F. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, T. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nix, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finn, J.</span></span> <span> </span><span class="NLM_article-title">Pyrrolopyrimidine Inhibitors of DNA Gyrase B (GyrB) and Topoisomerase IV (ParE). Part I: Structure Guided Discovery and Optimization of Dual Targeting Agents with Potent, Broad-Spectrum Enzymatic Activity</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1529</span>– <span class="NLM_lpage">1536</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2012.11.032</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1016%2Fj.bmcl.2012.11.032" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=23352267" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVOrsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2013&pages=1529-1536&author=L.+W.+Tariauthor=M.+Trzossauthor=D.+C.+Bensenauthor=X.+Liauthor=Z.+Chenauthor=T.+Lamauthor=J.+Zhangauthor=C.+J.+Creightonauthor=M.+L.+Cunninghamauthor=B.+Kwanauthor=M.+Stidhamauthor=K.+J.+Shawauthor=F.+C.+Lightstoneauthor=S.+E.+Wongauthor=T.+B.+Nguyenauthor=J.+Nixauthor=J.+Finn&title=Pyrrolopyrimidine+Inhibitors+of+DNA+Gyrase+B+%28GyrB%29+and+Topoisomerase+IV+%28ParE%29.+Part+I%3A+Structure+Guided+Discovery+and+Optimization+of+Dual+Targeting+Agents+with+Potent%2C+Broad-Spectrum+Enzymatic+Activity&doi=10.1016%2Fj.bmcl.2012.11.032"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Pyrrolopyrimidine inhibitors of DNA gyrase B (GyrB) and topoisomerase IV (ParE). Part I: Structure guided discovery and optimization of dual targeting agents with potent, broad-spectrum enzymatic activity</span></div><div class="casAuthors">Tari, Leslie W.; Trzoss, Michael; Bensen, Daniel C.; Li, Xiaoming; Chen, Zhiyong; Lam, Thanh; Zhang, Junhu; Creighton, Christopher J.; Cunningham, Mark L.; Kwan, Bryan; Stidham, Mark; Shaw, Karen J.; Lightstone, Felice C.; Wong, Sergio E.; Nguyen, Toan B.; Nix, Jay; Finn, John</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1529-1536</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The bacterial topoisomerases DNA gyrase (GyrB) and topoisomerase IV (ParE) are essential enzymes that control the topol. state of DNA during replication.  The high degree of conservation in the ATP-binding pockets of these enzymes make them appealing targets for broad-spectrum inhibitor development.  A pyrrolopyrimidine scaffold was identified from a pharmacophore-based fragment screen with optimization potential.  Structural characterization of inhibitor complexes conducted using selected GyrB/ParE orthologs aided in the identification of important steric, dynamic and compositional differences in the ATP-binding pockets of the targets, enabling the design of highly potent pyrrolopyrimidine inhibitors with broad enzymic spectrum and dual targeting activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqn5kRWxZVjhbVg90H21EOLACvtfcHk0li7zyDoNJikrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVOrsbo%253D&md5=b8d572d3dc5320c2ec385668b3f60387</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2012.11.032&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2012.11.032%26sid%3Dliteratum%253Aachs%26aulast%3DTari%26aufirst%3DL.%2BW.%26aulast%3DTrzoss%26aufirst%3DM.%26aulast%3DBensen%26aufirst%3DD.%2BC.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DLam%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DCreighton%26aufirst%3DC.%2BJ.%26aulast%3DCunningham%26aufirst%3DM.%2BL.%26aulast%3DKwan%26aufirst%3DB.%26aulast%3DStidham%26aufirst%3DM.%26aulast%3DShaw%26aufirst%3DK.%2BJ.%26aulast%3DLightstone%26aufirst%3DF.%2BC.%26aulast%3DWong%26aufirst%3DS.%2BE.%26aulast%3DNguyen%26aufirst%3DT.%2BB.%26aulast%3DNix%26aufirst%3DJ.%26aulast%3DFinn%26aufirst%3DJ.%26atitle%3DPyrrolopyrimidine%2520Inhibitors%2520of%2520DNA%2520Gyrase%2520B%2520%2528GyrB%2529%2520and%2520Topoisomerase%2520IV%2520%2528ParE%2529.%2520Part%2520I%253A%2520Structure%2520Guided%2520Discovery%2520and%2520Optimization%2520of%2520Dual%2520Targeting%2520Agents%2520with%2520Potent%252C%2520Broad-Spectrum%2520Enzymatic%2520Activity%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D23%26spage%3D1529%26epage%3D1536%26doi%3D10.1016%2Fj.bmcl.2012.11.032" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Richter, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hergenrother, P. J.</span></span> <span> </span><span class="NLM_article-title">The Challenge of Converting Gram-Positive-Only Compounds Into Broad-Spectrum Antibiotics</span>. <i>Ann. N. Y. Acad. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>1435</i></span>,  <span class="NLM_fpage">18</span>– <span class="NLM_lpage">38</span>, <span class="refDoi"> DOI: 10.1111/nyas.13598</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1111%2Fnyas.13598" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=29446459" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitlyktw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1435&publication_year=2019&pages=18-38&author=M.+F.+Richterauthor=P.+J.+Hergenrother&title=The+Challenge+of+Converting+Gram-Positive-Only+Compounds+Into+Broad-Spectrum+Antibiotics&doi=10.1111%2Fnyas.13598"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">The challenge of converting Gram-positive-only compounds into broad-spectrum antibiotics</span></div><div class="casAuthors">Richter, Michelle F.; Hergenrother, Paul J.</div><div class="citationInfo"><span class="NLM_cas:title">Annals of the New York Academy of Sciences</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">1435</span>
        (<span class="NLM_cas:issue">1, Antimicrobial Therapeutics Reviews</span>),
    <span class="NLM_cas:pages">18-38</span>CODEN:
                <span class="NLM_cas:coden">ANYAA9</span>;
        ISSN:<span class="NLM_cas:issn">0077-8923</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  Multidrug resistant Gram-neg. bacterial infections are on the rise, and there is a lack of new classes of drugs to treat these pathogens.  This drug shortage is largely due to the challenge of finding antibiotics that can permeate and persist inside Gram-neg. species.  Efforts to understand the mol. properties that enable certain compds. to accumulate in Gram-neg. bacteria based on retrospective studies of known antibiotics have not been generally actionable in the development of new antibiotics.  A recent assessment of the ability of >180 diverse small mols. to accumulate in Escherichia coli led to predictive guidelines for compd. accumulation in E. coli.  These "eNTRy rules" state that compds. are most likely to accumulate if they contain a nonsterically encumbered ionizable Nitrogen (primary amines are the best), have low Three-dimensionality (globularity ≤ 0.25), and are relatively Rigid (rotatable bonds ≤ 5).  In this review, we look back through 50+ years of antibacterial research and 1000s of derivs. and assess this historical data set through the lens of these predictive guidelines.  The results are consistent with the eNTRy rules, suggesting that the eNTRy rules may provide an actionable and general roadmap for the conversion of Gram-pos.-only compds. into broad-spectrum antibiotics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqX3zbgiAcHdbVg90H21EOLACvtfcHk0lidwOfJV9EnUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitlyktw%253D%253D&md5=12fffe0d10e7d3e5c134ac28ce929ab3</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1111%2Fnyas.13598&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fnyas.13598%26sid%3Dliteratum%253Aachs%26aulast%3DRichter%26aufirst%3DM.%2BF.%26aulast%3DHergenrother%26aufirst%3DP.%2BJ.%26atitle%3DThe%2520Challenge%2520of%2520Converting%2520Gram-Positive-Only%2520Compounds%2520Into%2520Broad-Spectrum%2520Antibiotics%26jtitle%3DAnn.%2520N.%2520Y.%2520Acad.%2520Sci.%26date%3D2019%26volume%3D1435%26spage%3D18%26epage%3D38%26doi%3D10.1111%2Fnyas.13598" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vaara, M.</span></span> <span> </span><span class="NLM_article-title">Agents That Increase the Permeability of the Outer Membrane</span>. <i>Microbiol. Mol. Biol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">1992</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">395</span>– <span class="NLM_lpage">411</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADyaK3sXjsFCktQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=1992&pages=395-411&author=M.+Vaara&title=Agents+That+Increase+the+Permeability+of+the+Outer+Membrane"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Agents that increase the permeability of the outer membrane</span></div><div class="casAuthors">Vaara, Martti</div><div class="citationInfo"><span class="NLM_cas:title">Microbiological Reviews</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">395-411</span>CODEN:
                <span class="NLM_cas:coden">MBRED3</span>;
        ISSN:<span class="NLM_cas:issn">0146-0749</span>.
    </div><div class="casAbstract">A review with 241 refs. on cationic agents and chelators as outer membrane permeabilizers, the mol. mechanism of outer membrane permeability increase, and bacterial strains resistant to the permeabilizers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrIuhdsLpq5OLVg90H21EOLACvtfcHk0lidwOfJV9EnUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXjsFCktQ%253D%253D&md5=2416c67d643c9be119bfb1c0d2323b62</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DVaara%26aufirst%3DM.%26atitle%3DAgents%2520That%2520Increase%2520the%2520Permeability%2520of%2520the%2520Outer%2520Membrane%26jtitle%3DMicrobiol.%2520Mol.%2520Biol.%2520Rev.%26date%3D1992%26volume%3D56%26spage%3D395%26epage%3D411" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Angus, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carey, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caron, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kropinski, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hancock, R. E.</span></span> <span> </span><span class="NLM_article-title">Outer Membrane Permeability in <i>Pseudomonas aeruginosa</i>: Comparison of a Wild-Type with an Antibiotic-Supersusceptible Mutant</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">1982</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">299</span>– <span class="NLM_lpage">309</span>, <span class="refDoi"> DOI: 10.1128/AAC.21.2.299</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1128%2FAAC.21.2.299" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=6803666" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADyaL38XhtFyruro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=1982&pages=299-309&author=B.+L.+Angusauthor=A.+M.+Careyauthor=D.+A.+Caronauthor=A.+M.+Kropinskiauthor=R.+E.+Hancock&title=Outer+Membrane+Permeability+in+Pseudomonas+aeruginosa%3A+Comparison+of+a+Wild-Type+with+an+Antibiotic-Supersusceptible+Mutant&doi=10.1128%2FAAC.21.2.299"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Outer membrane permeability in Pseudomonas aeruginosa:  comparison of a wild-type with an antibiotic-supersusceptible mutant</span></div><div class="casAuthors">Angus, Barbara L.; Carey, Alice M.; Caron, Dania A.; Kropinski, Andrew M. B.; Hancock, Robert E. W.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">1982</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">299-309</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    </div><div class="casAbstract">Spontaneous revertants of P. aeruginosa were isolated, using gentamicin or carbenicillin as selective agents, and shown to have 2 patterns of susceptibility to a group of 12 antibiotics.  Partial revertants had 2- to 10-fold greater resistance to these antibiotics than mutant Z61, whereas full revertants had antibiotic susceptibilities indistinguishable from those of the wild-type strain K799, from which mutant Z61 had been derived.  Uptake of a chromogenic β-lactam, nitrocefin, was studied in both uninduced and induced cells of all strains by measuring the steady-state rate of nitrocefin hydrolysis by the inducible, periplasmic β-lactamase in both whole and broken cells.  This demonstrated that outer membrane permeability decreased as antibiotic resistance increased in the series: mutant Z61, partial revertants, wild-type, and full revertants.  Low outer membrane permeability, caused by a low proportion of open functional porins in the outer membrane, may be the reason for the high natural antibiotic resistance of wild-type P. aeruginosa strains.  In addn., levels of benzylpenicillin below the minimal inhibitory concn. for mutant Z61 failed to induce β-lactamase prodn.  The possibility that this was related to the obsd. increase in outer membrane permeability is discussed.  Preliminary evidence is presented that the pore-forming outer membrane porin protein F is not altered in mutant Z61.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXV0owNpUYT7Vg90H21EOLACvtfcHk0lidwOfJV9EnUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL38XhtFyruro%253D&md5=aa7c0311c93f236f6f184c301713bfb8</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1128%2FAAC.21.2.299&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.21.2.299%26sid%3Dliteratum%253Aachs%26aulast%3DAngus%26aufirst%3DB.%2BL.%26aulast%3DCarey%26aufirst%3DA.%2BM.%26aulast%3DCaron%26aufirst%3DD.%2BA.%26aulast%3DKropinski%26aufirst%3DA.%2BM.%26aulast%3DHancock%26aufirst%3DR.%2BE.%26atitle%3DOuter%2520Membrane%2520Permeability%2520in%2520Pseudomonas%2520aeruginosa%253A%2520Comparison%2520of%2520a%2520Wild-Type%2520with%2520an%2520Antibiotic-Supersusceptible%2520Mutant%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D1982%26volume%3D21%26spage%3D299%26epage%3D309%26doi%3D10.1128%2FAAC.21.2.299" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yoshimura, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikaido, H.</span></span> <span> </span><span class="NLM_article-title">Permeability of <i>Pseudomonas aeruginosa</i> Outer Membrane to Hydrophilic Solutes</span>. <i>J. Bacteriol.</i> <span class="NLM_year" style="font-weight: bold;">1982</span>,  <span class="NLM_volume"><i>152</i></span>,  <span class="NLM_fpage">636</span>– <span class="NLM_lpage">642</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=6813310" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADyaL38XmtFagu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=152&publication_year=1982&pages=636-642&author=F.+Yoshimuraauthor=H.+Nikaido&title=Permeability+of+Pseudomonas+aeruginosa+Outer+Membrane+to+Hydrophilic+Solutes"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Permeability of Pseudomonas aeruginosa outer membrane to hydrophilic solutes</span></div><div class="casAuthors">Yoshimura, Fuminobu; Nikaido, Hiroshi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Bacteriology</span>
        (<span class="NLM_cas:date">1982</span>),
    <span class="NLM_cas:volume">152</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">636-42</span>CODEN:
                <span class="NLM_cas:coden">JOBAAY</span>;
        ISSN:<span class="NLM_cas:issn">0021-9193</span>.
    </div><div class="casAbstract">P. aeruginosa Is usually resistant to a wide variety of antibacterial agents, and it has been inferred, on the basis of indirect evidence, that this was due to the low permeability of its outer membrane.  The permeability of P. aeruginosa outer membrane was detd. directly, by measuring the rates of hydrolysis of cephacetrile, cephaloridine, and various phosphate esters by hydrolytic enzymes located in the periplasm.  The permeability to these compds. was ∼100-fold lower than of the outer membrane of Escherichia coli K-12.  Also, the apparent Km values for active transport of various C and energy source compds. were typically >20 μM in P. aeruginosa, in contrast to E. coli, in which the values are usually <5 μM.  Apparently, the P. aeruginosa outer membrane indeed has a low permeability to most hydrophilic compds., and this membrane acts as a rate-limiting step in active transport processes with high Vmax values.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxJ7cuhYCEVbVg90H21EOLACvtfcHk0lhuzXddfoub7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL38XmtFagu7s%253D&md5=1b90ff40a7309d64b724487ccb6ea0ed</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DYoshimura%26aufirst%3DF.%26aulast%3DNikaido%26aufirst%3DH.%26atitle%3DPermeability%2520of%2520Pseudomonas%2520aeruginosa%2520Outer%2520Membrane%2520to%2520Hydrophilic%2520Solutes%26jtitle%3DJ.%2520Bacteriol.%26date%3D1982%26volume%3D152%26spage%3D636%26epage%3D642" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Delcour, A.
H.</span></span> <span> </span><span class="NLM_article-title">Outer Membrane Permeability and Antibiotic Resistance</span>. <i>Biochim. Biophys. Acta</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>1794</i></span>,  <span class="NLM_fpage">808</span>– <span class="NLM_lpage">816</span>, <span class="refDoi"> DOI: 10.1016/j.bbapap.2008.11.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1016%2Fj.bbapap.2008.11.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=19100346" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BD1MXkvVSrtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1794&publication_year=2009&pages=808-816&author=A.%0AH.+Delcour&title=Outer+Membrane+Permeability+and+Antibiotic+Resistance&doi=10.1016%2Fj.bbapap.2008.11.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Outer membrane permeability and antibiotic resistance</span></div><div class="casAuthors">Delcour, Anne H.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Proteins and Proteomics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">1794</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">808-816</span>CODEN:
                <span class="NLM_cas:coden">BBAPBW</span>;
        ISSN:<span class="NLM_cas:issn">1570-9639</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B. V.</span>)
        </div><div class="casAbstract">A review.  To date most antibiotics are targeted at intracellular processes, and must be able to penetrate the bacterial cell envelope.  In particular, the outer membrane of gram-neg. bacteria provides a formidable barrier that must be overcome.  There are essentially two pathways that antibiotics can take through the outer membrane: a lipid-mediated pathway for hydrophobic antibiotics, and general diffusion porins for hydrophilic antibiotics.  The lipid and protein compns. of the outer membrane have a strong impact on the sensitivity of bacteria to many types of antibiotics, and drug resistance involving modifications of these macromols. is common.  This review describes the mol. mechanisms for permeation of antibiotics through the outer membrane, and the strategies that bacteria have deployed to resist antibiotics by modifications of these pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnujySVWxi3bVg90H21EOLACvtfcHk0lhuzXddfoub7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXkvVSrtr4%253D&md5=b32db36960e4c0d122bef0c3648462e8</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.bbapap.2008.11.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbapap.2008.11.005%26sid%3Dliteratum%253Aachs%26aulast%3DDelcour%26aufirst%3DA.%2BH.%26atitle%3DOuter%2520Membrane%2520Permeability%2520and%2520Antibiotic%2520Resistance%26jtitle%3DBiochim.%2520Biophys.%2520Acta%26date%3D2009%26volume%3D1794%26spage%3D808%26epage%3D816%26doi%3D10.1016%2Fj.bbapap.2008.11.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Page, M. G. P.</span></span> <span> </span><span class="NLM_article-title">The Role of the Outer Membrane of Gram-Negative Bacteria in Antibiotic Resistance: Ajax’ Shield or Achilles’ Heel?</span>. <i>Handb. Exp. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>211</i></span>,  <span class="NLM_fpage">67</span>– <span class="NLM_lpage">86</span>, <span class="refDoi"> DOI: 10.1007/978-3-642-28951-4_5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1007%2F978-3-642-28951-4_5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BC3sXnslWqt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=211&publication_year=2012&pages=67-86&author=M.+G.+P.+Page&title=The+Role+of+the+Outer+Membrane+of+Gram-Negative+Bacteria+in+Antibiotic+Resistance%3A+Ajax%E2%80%99+Shield+or+Achilles%E2%80%99+Heel%3F&doi=10.1007%2F978-3-642-28951-4_5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">The role of the outer membrane of gram-negative bacteria in antibiotic resistance: Ajax' shield of Achilles' heel?</span></div><div class="casAuthors">Page, Malcolm G. P.</div><div class="citationInfo"><span class="NLM_cas:title">Handbook of Experimental Pharmacology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">211</span>
        (<span class="NLM_cas:issue">Antibiotic Resistance</span>),
    <span class="NLM_cas:pages">67-86</span>CODEN:
                <span class="NLM_cas:coden">HEPHD2</span>;
        ISSN:<span class="NLM_cas:issn">0171-2004</span>.
    
            (<span class="NLM_cas:orgname">Springer GmbH</span>)
        </div><div class="casAbstract">A review.  There has been an enormous increase in our knowledge of the fundamental steps in the biosynthesis and assembly of the outer membrane of Gram-neg. bacteria.  Lipopolysaccharide is a major component of the outer membrane of Gram-neg. bacteria as is peptidoglycan.  Porins, efflux pumps and other transport proteins of the outer membrane are also present.  It is clear that there are numerous essential proteins that have the potential to be targets for novel antimicrobial agents.  Progress, however, has been slow.  Much of the emphasis has been on cytoplasmic processes that were better understood earlier on, but have the drawback that two penetration barriers, with different permeability properties, have to be crossed.  With the increased understanding of the late-stage events occurring in the periplasm, it may be possible to shift focus to these more accessible targets.  Nevertheless, getting drugs across the outer membrane will remain a challenge to the ingenuity of the medicinal chemist.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3ryHhzdkuqbVg90H21EOLACvtfcHk0lh5evFrRL4yuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXnslWqt7o%253D&md5=2d10c275feb129390c0349297fe8b468</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1007%2F978-3-642-28951-4_5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-3-642-28951-4_5%26sid%3Dliteratum%253Aachs%26aulast%3DPage%26aufirst%3DM.%2BG.%2BP.%26atitle%3DThe%2520Role%2520of%2520the%2520Outer%2520Membrane%2520of%2520Gram-Negative%2520Bacteria%2520in%2520Antibiotic%2520Resistance%253A%2520Ajax%25E2%2580%2599%2520Shield%2520or%2520Achilles%25E2%2580%2599%2520Heel%253F%26jtitle%3DHandb.%2520Exp.%2520Pharmacol.%26date%3D2012%26volume%3D211%26spage%3D67%26epage%3D86%26doi%3D10.1007%2F978-3-642-28951-4_5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Breidenstein, E. B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de la Fuente-Núñez, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hancock, R. E. W.</span></span> <span> </span><span class="NLM_article-title"><i>Pseudomonas aeruginosa</i>: All Roads Lead to Resistance</span>. <i>Trends Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">419</span>– <span class="NLM_lpage">426</span>, <span class="refDoi"> DOI: 10.1016/j.tim.2011.04.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1016%2Fj.tim.2011.04.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=21664819" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BC3MXps1entbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2011&pages=419-426&author=E.+B.+M.+Breidensteinauthor=C.+de+la+Fuente-N%C3%BA%C3%B1ezauthor=R.+E.+W.+Hancock&title=Pseudomonas+aeruginosa%3A+All+Roads+Lead+to+Resistance&doi=10.1016%2Fj.tim.2011.04.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Pseudomonas aeruginosa: all roads lead to resistance</span></div><div class="casAuthors">Breidenstein, Elena B. M.; de la Fuente-Nunez, Cesar; Hancock, Robert E. W.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Microbiology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">419-426</span>CODEN:
                <span class="NLM_cas:coden">TRMIEA</span>;
        ISSN:<span class="NLM_cas:issn">0966-842X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Pseudomonas aeruginosa is often resistant to multiple antibiotics and consequently has joined the ranks of superbugs' due to its enormous capacity to engender resistance.  It demonstrates decreased susceptibility to most antibiotics due to low outer membrane permeability coupled to adaptive mechanisms and can readily achieve clin. resistance.  Newer research, using mutant library screens, microarray technologies and mutation frequency anal., has identified very large collections of genes (the resistome) that when mutated lead to resistance as well as new forms of adaptive resistance that can be triggered by antibiotics themselves, in in vivo growth conditions or complex adaptations such as biofilm growth or swarming motility.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrp-9n40KdDx7Vg90H21EOLACvtfcHk0lh5evFrRL4yuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXps1entbo%253D&md5=8af03809d5bc30795cdb3dc403808c33</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.tim.2011.04.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tim.2011.04.005%26sid%3Dliteratum%253Aachs%26aulast%3DBreidenstein%26aufirst%3DE.%2BB.%2BM.%26aulast%3Dde%2Bla%2BFuente-N%25C3%25BA%25C3%25B1ez%26aufirst%3DC.%26aulast%3DHancock%26aufirst%3DR.%2BE.%2BW.%26atitle%3DPseudomonas%2520aeruginosa%253A%2520All%2520Roads%2520Lead%2520to%2520Resistance%26jtitle%3DTrends%2520Microbiol.%26date%3D2011%26volume%3D19%26spage%3D419%26epage%3D426%26doi%3D10.1016%2Fj.tim.2011.04.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Waldrop, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holden, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maurice, M. S.</span></span> <span> </span><span class="NLM_article-title">The Enzymes of Biotin Dependent CO<sub>2</sub> Metabolism: What Structures Reveal About Their Reaction Mechanisms</span>. <i>Protein Sci.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">1597</span>– <span class="NLM_lpage">1619</span>, <span class="refDoi"> DOI: 10.1002/pro.2156</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1002%2Fpro.2156" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=22969052" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFWitrfE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2012&pages=1597-1619&author=G.+L.+Waldropauthor=H.+M.+Holdenauthor=M.+S.+Maurice&title=The+Enzymes+of+Biotin+Dependent+CO2+Metabolism%3A+What+Structures+Reveal+About+Their+Reaction+Mechanisms&doi=10.1002%2Fpro.2156"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">The enzymes of biotin dependent CO2 metabolism: What structures reveal about their reaction mechanisms</span></div><div class="casAuthors">Waldrop, Grover L.; Holden, Hazel M.; St. Maurice, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1597-1619</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">1469-896X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  Biotin is the major cofactor involved in CO2 metab.  Indeed, biotin-dependent enzymes are ubiquitous in nature and are involved in a myriad of metabolic processes including fatty acid synthesis and gluconeogenesis.  The cofactor, itself, is composed of a ureido ring, a tetrahydrothiophene ring, and a valeric acid side-chain.  It is the ureido ring that functions as the CO2 carrier.  A complete understanding of biotin-dependent enzymes is critically important for translational research in light of the fact that some of these enzymes serve as targets for anti-obesity agents, antibiotics, and herbicides.  Prior to 1990, however, there was a dearth of information regarding the mol. architectures of biotin-dependent enzymes.  In recent years there has been an explosion in the no. of 3-dimensional structures reported for these proteins.  Here, the authors review the current understanding of the structures and functions of biotin-dependent enzymes.  In addn., the authors provide a crit. anal. of what these structures have and have not revealed about biotin-dependent catalysis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhq6DUbpmID7Vg90H21EOLACvtfcHk0ljk7GLS42Nbag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFWitrfE&md5=0f3b44a3b78c67296f7282c7e47c8a0f</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1002%2Fpro.2156&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpro.2156%26sid%3Dliteratum%253Aachs%26aulast%3DWaldrop%26aufirst%3DG.%2BL.%26aulast%3DHolden%26aufirst%3DH.%2BM.%26aulast%3DMaurice%26aufirst%3DM.%2BS.%26atitle%3DThe%2520Enzymes%2520of%2520Biotin%2520Dependent%2520CO2%2520Metabolism%253A%2520What%2520Structures%2520Reveal%2520About%2520Their%2520Reaction%2520Mechanisms%26jtitle%3DProtein%2520Sci.%26date%3D2012%26volume%3D21%26spage%3D1597%26epage%3D1619%26doi%3D10.1002%2Fpro.2156" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Broussard, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pakhomova, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neau, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonnot, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldrop, G. L.</span></span> <span> </span><span class="NLM_article-title">Structural Analysis of Substrate, Reaction Intermediate, and Product Binding in <i>Haemophilus influenzae</i> Biotin Carboxylase</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">3860</span>– <span class="NLM_lpage">3870</span>, <span class="refDoi"> DOI: 10.1021/acs.biochem.5b00340</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.biochem.5b00340" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BC2MXptV2rtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2015&pages=3860-3870&author=T.+C.+Broussardauthor=S.+Pakhomovaauthor=D.+B.+Neauauthor=R.+Bonnotauthor=G.+L.+Waldrop&title=Structural+Analysis+of+Substrate%2C+Reaction+Intermediate%2C+and+Product+Binding+in+Haemophilus+influenzae+Biotin+Carboxylase&doi=10.1021%2Facs.biochem.5b00340"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Analysis of Substrate, Reaction Intermediate, and Product Binding in Haemophilus influenzae Biotin Carboxylase</span></div><div class="casAuthors">Broussard, Tyler C.; Pakhomova, Svetlana; Neau, David B.; Bonnot, Ross; Waldrop, Grover L.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">3860-3870</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Acetyl-CoA carboxylase catalyzes the first and regulated step in fatty acid synthesis.  In most Gram-neg. and Gram-pos. bacteria, the enzyme is composed of three proteins: biotin carboxylase, a biotin carboxyl carrier protein (BCCP), and carboxyltransferase.  The reaction mechanism involves two half-reactions, with biotin carboxylase catalyzing the ATP-dependent carboxylation of biotin-BCCP in the first reaction.  In the second reaction, carboxyltransferase catalyzes the transfer of the carboxyl group from biotin-BCCP to acetyl-CoA to form malonyl-CoA.  In this report, high-resoln. crystal structures of biotin carboxylase from Haemophilus influenzae were detd. with bicarbonate, the ATP analog AMPPCP; the carboxyphosphate intermediate analogs, phosphonoacetamide (PAM) and phosphonoformate (POF); the products ADP and phosphate; and the carboxybiotin analog N1'-methoxycarbonyl biotin Me ester.  The structures have a common theme in that bicarbonate, phosphate, and the Me ester of the carboxyl group of N1'-methoxycarbonyl biotin Me ester all bound in the same pocket in the active site of biotin carboxylase and as such utilize the same set of amino acids for binding.  This finding suggests a catalytic mechanism for biotin carboxylase in which the binding pocket that binds tetrahedral phosphate also accommodates and stabilizes a tetrahedral dianionic transition state resulting from direct transfer of CO2 from the carboxyphosphate intermediate to biotin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2YP7QWNwytrVg90H21EOLACvtfcHk0ljk7GLS42Nbag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXptV2rtrw%253D&md5=99925d8aa7204fa50a9bfb12f83edb19</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Facs.biochem.5b00340&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.biochem.5b00340%26sid%3Dliteratum%253Aachs%26aulast%3DBroussard%26aufirst%3DT.%2BC.%26aulast%3DPakhomova%26aufirst%3DS.%26aulast%3DNeau%26aufirst%3DD.%2BB.%26aulast%3DBonnot%26aufirst%3DR.%26aulast%3DWaldrop%26aufirst%3DG.%2BL.%26atitle%3DStructural%2520Analysis%2520of%2520Substrate%252C%2520Reaction%2520Intermediate%252C%2520and%2520Product%2520Binding%2520in%2520Haemophilus%2520influenzae%2520Biotin%2520Carboxylase%26jtitle%3DBiochemistry%26date%3D2015%26volume%3D54%26spage%3D3860%26epage%3D3870%26doi%3D10.1021%2Facs.biochem.5b00340" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cunningham, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwan, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nelson, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bensen, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, K. J.</span></span> <span> </span><span class="NLM_article-title">Distinguishing On-Target Versus Off-Target Activity in Early Antibacterial Drug Discovery Using a Macromolecular Synthesis Assay</span>. <i>J. Biomol. Screening</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">1018</span>– <span class="NLM_lpage">1026</span>, <span class="refDoi"> DOI: 10.1177/1087057113487208</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1177%2F1087057113487208" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=23686103" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslWitr%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2013&pages=1018-1026&author=M.+L.+Cunninghamauthor=B.+P.+Kwanauthor=K.+J.+Nelsonauthor=D.+C.+Bensenauthor=K.+J.+Shaw&title=Distinguishing+On-Target+Versus+Off-Target+Activity+in+Early+Antibacterial+Drug+Discovery+Using+a+Macromolecular+Synthesis+Assay&doi=10.1177%2F1087057113487208"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Distinguishing on-target versus off-target activity in early antibacterial drug discovery using a macromolecular synthesis assay</span></div><div class="casAuthors">Cunningham, Mark L.; Kwan, Bryan P.; Nelson, Kirk J.; Bensen, Daniel C.; Shaw, Karen J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1018-1026, 9</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">The macromol. synthesis assay was optimized in both S. aureus and E. coli imp and used to define patterns of inhibition of DNA, RNA, protein, and cell wall biosynthesis of several drug classes.  The concn. of drug required to elicit pathway inhibition differed among the antimicrobial agents tested, with inhibition detected at concns. significantly below the min. inhibitory concn. (MIC) for tedizolid; within 4-fold of the MIC for ciprofloxacin, cefepime, vancomycin, tetracycline, and chloramphenicol; and significantly above the MIC for rifampicin and kanamycin.  In a DNA gyrase/topoisomerase IV structure-based drug design optimization program, the assay rapidly identified undesirable offtarget activity within certain chemotypes, altering the course of the program to focus on the series that maintained ontarget activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3Es7L9oicRbVg90H21EOLACvtfcHk0liZV53d3qojBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslWitr%252FF&md5=ca15beac8968c13ce847b201a52d73d3</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1177%2F1087057113487208&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057113487208%26sid%3Dliteratum%253Aachs%26aulast%3DCunningham%26aufirst%3DM.%2BL.%26aulast%3DKwan%26aufirst%3DB.%2BP.%26aulast%3DNelson%26aufirst%3DK.%2BJ.%26aulast%3DBensen%26aufirst%3DD.%2BC.%26aulast%3DShaw%26aufirst%3DK.%2BJ.%26atitle%3DDistinguishing%2520On-Target%2520Versus%2520Off-Target%2520Activity%2520in%2520Early%2520Antibacterial%2520Drug%2520Discovery%2520Using%2520a%2520Macromolecular%2520Synthesis%2520Assay%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2013%26volume%3D18%26spage%3D1018%26epage%3D1026%26doi%3D10.1177%2F1087057113487208" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nonejuie, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burkart, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pogliano, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pogliano, J.</span></span> <span> </span><span class="NLM_article-title">Bacterial Cytological Profiling Rapidly Identifies the Cellular Pathways Targeted by Antibacterial Molecules</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">16169</span>– <span class="NLM_lpage">16174</span>, <span class="refDoi"> DOI: 10.1073/pnas.1311066110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1073%2Fpnas.1311066110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=24046367" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1SqtbfI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=16169-16174&author=P.+Nonejuieauthor=M.+Burkartauthor=K.+Poglianoauthor=J.+Pogliano&title=Bacterial+Cytological+Profiling+Rapidly+Identifies+the+Cellular+Pathways+Targeted+by+Antibacterial+Molecules&doi=10.1073%2Fpnas.1311066110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Bacterial cytological profiling rapidly identifies the cellular pathways targeted by antibacterial molecules</span></div><div class="casAuthors">Nonejuie, Poochit; Burkart, Michael; Pogliano, Kit; Pogliano, Joe</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">40</span>),
    <span class="NLM_cas:pages">16169-16174,S16169/1-S16169/14</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Identifying the mechanism of action for antibacterial compds. is essential for understanding how bacteria interact with one another and with other cell types and for antibiotic discovery efforts, but detg. a compd.'s mechanism of action remains a serious challenge that limits both basic research and antibacterial discovery programs.  Here, we show that bacterial cytol. profiling (BCP) is a rapid and powerful approach for identifying the cellular pathway affected by antibacterial mols.  BCP can distinguish between inhibitors that affect different cellular pathways as well as different targets within the same pathway.  We use BCP to demonstrate that spirohexenolide A, a spirotetronate that is active against methicillin-resistant Staphylococcus aureus, rapidly collapses the proton motive force.  BCP offers a simple, one-step assay that can be broadly applied, solving the longstanding problem of how to rapidly det. the cellular target of thousands of compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqb4cbb0TVNjLVg90H21EOLACvtfcHk0liZV53d3qojBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1SqtbfI&md5=8822d9a313f79b659d28307659057a58</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1311066110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1311066110%26sid%3Dliteratum%253Aachs%26aulast%3DNonejuie%26aufirst%3DP.%26aulast%3DBurkart%26aufirst%3DM.%26aulast%3DPogliano%26aufirst%3DK.%26aulast%3DPogliano%26aufirst%3DJ.%26atitle%3DBacterial%2520Cytological%2520Profiling%2520Rapidly%2520Identifies%2520the%2520Cellular%2520Pathways%2520Targeted%2520by%2520Antibacterial%2520Molecules%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2013%26volume%3D110%26spage%3D16169%26epage%3D16174%26doi%3D10.1073%2Fpnas.1311066110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lamsa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dorrestein, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pogliano, K.</span></span> <span> </span><span class="NLM_article-title">The <i>Bacillus subtilis</i> Cannibalism Toxin SDP Collapses the Proton Motive Force and Induces Autolysis</span>. <i>Mol. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>84</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">500</span>, <span class="refDoi"> DOI: 10.1111/j.1365-2958.2012.08038.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1111%2Fj.1365-2958.2012.08038.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=22469514" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BC38XnvVGqurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=84&publication_year=2012&pages=486-500&author=A.+Lamsaauthor=W.-T.+Liuauthor=P.+C.+Dorresteinauthor=K.+Pogliano&title=The+Bacillus+subtilis+Cannibalism+Toxin+SDP+Collapses+the+Proton+Motive+Force+and+Induces+Autolysis&doi=10.1111%2Fj.1365-2958.2012.08038.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">The Bacillus subtilis cannibalism toxin SDP collapses the proton motive force and induces autolysis</span></div><div class="casAuthors">Lamsa, Anne; Liu, Wei-Ting; Dorrestein, Pieter C.; Pogliano, Kit</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Microbiology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">84</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">486-500</span>CODEN:
                <span class="NLM_cas:coden">MOMIEE</span>;
        ISSN:<span class="NLM_cas:issn">0950-382X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Bacillus subtilis SDP is a peptide toxin that kills cells outside the biofilm to support continued growth.  The authors show that purified SDP acts like endogenously produced SDP; it delays sporulation, and the SdpI immunity protein confers SDP resistance.  SDP kills a variety of Gram-pos. bacteria in the phylum Firmicutes, as well as Escherichia coli with a compromised outer membrane, suggesting it participates in defense of the B. subtilis biofilm against Gram-pos. bacteria as well as cannibalism.  Fluorescence microscopy reveals that the effect of SDP on cells differs from that of nisin, nigericin, valinomycin and vancomycin-KCl, but resembles that of CCCP, DNP and azide.  Indeed, SDP rapidly collapses the PMF as measured by fluorometry and flow cytometry, which triggers the slower process of autolysis.  This secondary consequence of SDP treatment is not required for cell death since the autolysin-defective lytC, lytD, lytE, lytF strain fails to be lysed but is nevertheless killed by SDP.  Collapsing the PMF is an ideal mechanism for a toxin involved in cannibalism and biofilm defense, since this would incapacitate neighboring cells by inhibiting motility and secretion of proteins and toxins.  It would also induce autolysis in many Gram-pos. species, thereby releasing nutrients that promote biofilm growth.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppl3ScLKne07Vg90H21EOLACvtfcHk0liZV53d3qojBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XnvVGqurc%253D&md5=fc3cb2d274cb75e1d4358df072777b5e</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1111%2Fj.1365-2958.2012.08038.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1365-2958.2012.08038.x%26sid%3Dliteratum%253Aachs%26aulast%3DLamsa%26aufirst%3DA.%26aulast%3DLiu%26aufirst%3DW.-T.%26aulast%3DDorrestein%26aufirst%3DP.%2BC.%26aulast%3DPogliano%26aufirst%3DK.%26atitle%3DThe%2520Bacillus%2520subtilis%2520Cannibalism%2520Toxin%2520SDP%2520Collapses%2520the%2520Proton%2520Motive%2520Force%2520and%2520Induces%2520Autolysis%26jtitle%3DMol.%2520Microbiol.%26date%3D2012%26volume%3D84%26spage%3D486%26epage%3D500%26doi%3D10.1111%2Fj.1365-2958.2012.08038.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lamsa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez-Garrido, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quach, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riley, E. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pogliano, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pogliano, K.</span></span> <span> </span><span class="NLM_article-title">Rapid Inhibition Profiling in Bacillus subtilisto Identify the Mechanism of Action of New Antimicrobials</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">2222</span>– <span class="NLM_lpage">2231</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b01050</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b01050" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BC28XotFaitrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=2222-2231&author=A.+Lamsaauthor=J.+Lopez-Garridoauthor=D.+Quachauthor=E.+P.+Rileyauthor=J.+Poglianoauthor=K.+Pogliano&title=Rapid+Inhibition+Profiling+in+Bacillus+subtilisto+Identify+the+Mechanism+of+Action+of+New+Antimicrobials&doi=10.1021%2Facschembio.5b01050"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid Inhibition Profiling in Bacillus subtilis to Identify the Mechanism of Action of New Antimicrobials</span></div><div class="casAuthors">Lamsa, Anne; Lopez-Garrido, Javier; Quach, Diana; Riley, Eammon P.; Pogliano, Joe; Pogliano, Kit</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2222-2231</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Increasing antimicrobial resistance has become a major public health crisis.  New antimicrobials with novel mechanisms of action (MOA) are desperately needed.  We previously developed a method, bacterial cytol. profiling (BCP), which utilizes fluorescence microscopy to rapidly identify the MOA of antimicrobial compds.  BCP is based upon our discovery that cells treated with antibiotics affecting different metabolic pathways generate different cytol. signatures, providing quant. information that can be used to det. a compd.'s MOA.  Here, we describe a system, rapid inhibition profiling (RIP), for creating cytol. profiles of new antibiotic targets for which there are currently no chem. inhibitors.  RIP consists of the fast, inducible degrdn. of a target protein followed by BCP.  We demonstrate that degrading essential proteins in the major metabolic pathways for DNA replication, transcription, fatty acid biosynthesis, and peptidoglycan biogenesis in Bacillus subtilis rapidly produces cytol. profiles closely matching that of antimicrobials targeting the same pathways.  Addnl., RIP and antibiotics targeting different steps in fatty acid biosynthesis can be differentiated from each other.  We utilize RIP and BCP to show that the antibacterial MOA of four nonsteroidal anti-inflammatory antibiotics differs from that proposed based on in vitro data.  RIP is a versatile method that will extend our knowledge of phenotypes assocd. with inactivating essential bacterial enzymes and thereby allow for screening for mols. that inhibit novel essential targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoH0kSYirX3HLVg90H21EOLACvtfcHk0lhppTsBPlwrQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XotFaitrc%253D&md5=cb2c1f57f8e5c1a249a052ca2efda6f3</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b01050&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b01050%26sid%3Dliteratum%253Aachs%26aulast%3DLamsa%26aufirst%3DA.%26aulast%3DLopez-Garrido%26aufirst%3DJ.%26aulast%3DQuach%26aufirst%3DD.%26aulast%3DRiley%26aufirst%3DE.%2BP.%26aulast%3DPogliano%26aufirst%3DJ.%26aulast%3DPogliano%26aufirst%3DK.%26atitle%3DRapid%2520Inhibition%2520Profiling%2520in%2520Bacillus%2520subtilisto%2520Identify%2520the%2520Mechanism%2520of%2520Action%2520of%2520New%2520Antimicrobials%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26volume%3D11%26spage%3D2222%26epage%3D2231%26doi%3D10.1021%2Facschembio.5b01050" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mohammad, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Younis, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peters, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pogliano, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pogliano, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayhoub, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oldfield, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cushman, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seleem, M. N.</span></span> <span> </span><span class="NLM_article-title">Phenylthiazole Antibacterial Agents Targeting Cell Wall Synthesis Exhibit Potent Activity <i>In Vitro</i> and <i>In Vivo</i> Against Vancomycin-Resistant Enterococci</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2425</span>– <span class="NLM_lpage">2438</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01780</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01780" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjs1elu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=2425-2438&author=H.+Mohammadauthor=W.+Younisauthor=L.+Chenauthor=C.+E.+Petersauthor=J.+Poglianoauthor=K.+Poglianoauthor=B.+Cooperauthor=J.+Zhangauthor=A.+Mayhoubauthor=E.+Oldfieldauthor=M.+Cushmanauthor=M.+N.+Seleem&title=Phenylthiazole+Antibacterial+Agents+Targeting+Cell+Wall+Synthesis+Exhibit+Potent+Activity+In+Vitro+and+In+Vivo+Against+Vancomycin-Resistant+Enterococci&doi=10.1021%2Facs.jmedchem.6b01780"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Phenylthiazole Antibacterial Agents Targeting Cell Wall Synthesis Exhibit Potent Activity in Vitro and in Vivo against Vancomycin-Resistant Enterococci</span></div><div class="casAuthors">Mohammad, Haroon; Younis, Waleed; Chen, Lu; Peters, Christine E.; Pogliano, Joe; Pogliano, Kit; Cooper, Bruce; Zhang, Jianan; Mayhoub, Abdelrahman; Oldfield, Eric; Cushman, Mark; Seleem, Mohamed N.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2425-2438</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The emergence of antibiotic-resistant bacterial species, such as vancomycin-resistant enterococci (VRE), necessitates the development of new antimicrobials.  Here, we investigate the spectrum of antibacterial activity of three phenylthiazole-substituted aminoguanidines.  These compds. possess potent activity against VRE, inhibiting growth of clin. isolates at concns. as low as 0.5 μg/mL.  The compds. exerted a rapid bactericidal effect, targeting cell wall synthesis.  Transposon mutagenesis suggested three possible targets: YubA, YubB (undecaprenyl diphosphate phosphatase (UPPP)), and YubD.  Both UPPP as well as undecaprenyl diphosphate synthase were inhibited by compd. 1.  YubA and YubD are annotated as transporters and may also be targets because 1 collapsed the proton motive force in membrane vesicles.  Using Caenorhabditis elegans, we demonstrate that two compds. (1, 3, at 20 μg/mL) retain potent activity in vivo, significantly reducing the burden of VRE in infected worms.  Taken altogether, the results indicate that compds. 1 and 3 warrant further investigation as novel antibacterial agents against drug-resistant enterococci.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoeil1XLiO43rVg90H21EOLACvtfcHk0lhppTsBPlwrQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjs1elu7c%253D&md5=d4bc343d2329d28effe1b1210b1b9cf7</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01780&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01780%26sid%3Dliteratum%253Aachs%26aulast%3DMohammad%26aufirst%3DH.%26aulast%3DYounis%26aufirst%3DW.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DPeters%26aufirst%3DC.%2BE.%26aulast%3DPogliano%26aufirst%3DJ.%26aulast%3DPogliano%26aufirst%3DK.%26aulast%3DCooper%26aufirst%3DB.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DMayhoub%26aufirst%3DA.%26aulast%3DOldfield%26aufirst%3DE.%26aulast%3DCushman%26aufirst%3DM.%26aulast%3DSeleem%26aufirst%3DM.%2BN.%26atitle%3DPhenylthiazole%2520Antibacterial%2520Agents%2520Targeting%2520Cell%2520Wall%2520Synthesis%2520Exhibit%2520Potent%2520Activity%2520In%2520Vitro%2520and%2520In%2520Vivo%2520Against%2520Vancomycin-Resistant%2520Enterococci%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D2425%26epage%3D2438%26doi%3D10.1021%2Facs.jmedchem.6b01780" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nonejuie, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munguia, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hollands, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dam, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumaraswamy, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rivera, H.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corriden, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rohde, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hensler, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burkart, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pogliano, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakoulas, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nizet, V.</span></span> <span> </span><span class="NLM_article-title">Azithromycin Synergizes with Cationic Antimicrobial Peptides to Exert Bactericidal and Therapeutic Activity Against Highly Multidrug-Resistant Gram-Negative Bacterial Pathogens</span>. <i>EBioMedicine</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">690</span>– <span class="NLM_lpage">698</span>, <span class="refDoi"> DOI: 10.1016/j.ebiom.2015.05.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1016%2Fj.ebiom.2015.05.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=26288841" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A280%3ADC%252BC287ks1ygsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2015&pages=690-698&author=L.+Linauthor=P.+Nonejuieauthor=J.+Munguiaauthor=A.+Hollandsauthor=J.+Olsonauthor=Q.+Damauthor=M.+Kumaraswamyauthor=H.+Riveraauthor=R.+Corridenauthor=M.+Rohdeauthor=M.+E.+Henslerauthor=M.+D.+Burkartauthor=J.+Poglianoauthor=G.+Sakoulasauthor=V.+Nizet&title=Azithromycin+Synergizes+with+Cationic+Antimicrobial+Peptides+to+Exert+Bactericidal+and+Therapeutic+Activity+Against+Highly+Multidrug-Resistant+Gram-Negative+Bacterial+Pathogens&doi=10.1016%2Fj.ebiom.2015.05.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Azithromycin Synergizes with Cationic Antimicrobial Peptides to Exert Bactericidal and Therapeutic Activity Against Highly Multidrug-Resistant Gram-Negative Bacterial Pathogens</span></div><div class="casAuthors">Lin Leo; Munguia Jason; Hollands Andrew; Olson Joshua; Dam Quang; Corriden Ross; Hensler Mary E; Sakoulas George; Nonejuie Poochit; Pogliano Joe; Kumaraswamy Monika; Rivera Heriberto Jr; Burkart Michael D; Rohde Manfred; Nizet Victor</div><div class="citationInfo"><span class="NLM_cas:title">EBioMedicine</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">690-8</span>
        ISSN:<span class="NLM_cas:issn">2352-3964</span>.
    </div><div class="casAbstract">Antibiotic resistance poses an increasingly grave threat to the public health.  Of pressing concern, rapid spread of carbapenem-resistance among multidrug-resistant (MDR) Gram-negative rods (GNR) is associated with few treatment options and high mortality rates.  Current antibiotic susceptibility testing guiding patient management is performed in a standardized manner, identifying minimum inhibitory concentrations (MIC) in bacteriologic media, but ignoring host immune factors.  Lacking activity in standard MIC testing, azithromycin (AZM), the most commonly prescribed antibiotic in the U.S., is never recommended for MDR GNR infection.  Here we report a potent bactericidal action of AZM against MDR carbapenem-resistant isolates of Pseudomonas aeruginosa, Klebsiella pneumoniae, and Acinetobacter baumannii.  This pharmaceutical activity is associated with enhanced AZM cell penetration in eukaryotic tissue culture media and striking multi-log-fold synergies with host cathelicidin antimicrobial peptide LL-37 or the last line antibiotic colistin.  Finally, AZM monotherapy exerts clear therapeutic effects in murine models of MDR GNR infection.  Our results suggest that AZM, currently ignored as a treatment option, could benefit patients with MDR GNR infections, especially in combination with colistin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR9RCUhukP3WwMthz5_Rx5rfW6udTcc2eafLpFedmHmsrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC287ks1ygsQ%253D%253D&md5=22b994576cd66910a7d391084fa3c9df</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2Fj.ebiom.2015.05.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ebiom.2015.05.021%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DL.%26aulast%3DNonejuie%26aufirst%3DP.%26aulast%3DMunguia%26aufirst%3DJ.%26aulast%3DHollands%26aufirst%3DA.%26aulast%3DOlson%26aufirst%3DJ.%26aulast%3DDam%26aufirst%3DQ.%26aulast%3DKumaraswamy%26aufirst%3DM.%26aulast%3DRivera%26aufirst%3DH.%26aulast%3DCorriden%26aufirst%3DR.%26aulast%3DRohde%26aufirst%3DM.%26aulast%3DHensler%26aufirst%3DM.%2BE.%26aulast%3DBurkart%26aufirst%3DM.%2BD.%26aulast%3DPogliano%26aufirst%3DJ.%26aulast%3DSakoulas%26aufirst%3DG.%26aulast%3DNizet%26aufirst%3DV.%26atitle%3DAzithromycin%2520Synergizes%2520with%2520Cationic%2520Antimicrobial%2520Peptides%2520to%2520Exert%2520Bactericidal%2520and%2520Therapeutic%2520Activity%2520Against%2520Highly%2520Multidrug-Resistant%2520Gram-Negative%2520Bacterial%2520Pathogens%26jtitle%3DEBioMedicine%26date%3D2015%26volume%3D2%26spage%3D690%26epage%3D698%26doi%3D10.1016%2Fj.ebiom.2015.05.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nonejuie, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trial, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Newton, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamsa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perera, V. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aguilar, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dorrestein, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pogliano, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pogliano, K.</span></span> <span> </span><span class="NLM_article-title">Application of Bacterial Cytological Profiling to Crude Natural Product Extracts Reveals the Antibacterial Arsenal of <i>Bacillus subtilis</i></span>. <i>J. Antibiot.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">353</span>– <span class="NLM_lpage">361</span>, <span class="refDoi"> DOI: 10.1038/ja.2015.116</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1038%2Fja.2015.116" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=26648120" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BC28Xos1anu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2016&pages=353-361&author=P.+Nonejuieauthor=R.+M.+Trialauthor=G.+L.+Newtonauthor=A.+Lamsaauthor=V.+R.+Pereraauthor=J.+Aguilarauthor=W.-T.+Liuauthor=P.+C.+Dorresteinauthor=J.+Poglianoauthor=K.+Pogliano&title=Application+of+Bacterial+Cytological+Profiling+to+Crude+Natural+Product+Extracts+Reveals+the+Antibacterial+Arsenal+of+Bacillus+subtilis&doi=10.1038%2Fja.2015.116"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Application of bacterial cytological profiling to crude natural product extracts reveals the antibacterial arsenal of Bacillus subtilis</span></div><div class="casAuthors">Nonejuie, Poochit; Trial, Rachelle M.; Newton, Gerald L.; Lamsa, Anne; Ranmali Perera, Varahenage; Aguilar, Julieta; Liu, Wei-Ting; Dorrestein, Pieter C.; Pogliano, Joe; Pogliano, Kit</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Antibiotics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">353-361</span>CODEN:
                <span class="NLM_cas:coden">JANTAJ</span>;
        ISSN:<span class="NLM_cas:issn">0021-8820</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Although most clin. used antibiotics are derived from natural products, identifying new antibacterial mols. from natural product exts. is difficult due to the complexity of these exts. and the limited tools to correlate biol. activity with specific mols.  Here, we show that bacterial cytol. profiling (BCP) provides a rapid method for mechanism of action detn. on plates and in complex natural product exts. and for activity-guided purifn.  We prepd. an ext. from Bacillus subtilis 3610 that killed the Escherichia coli lptD mutant and used BCP to observe two types of bioactivities in the unfractionated ext.: inhibition of translation and permeablization of the cytoplasmic membrane.  We used BCP to guide purifn. of the mols. responsible for each activity, identifying the translation inhibitors bacillaene and bacillaene B (glycosylated bacillaene) and demonstrating that two mols. contribute to cell permeablization, the bacteriocin subtilosin and the cyclic peptide sporulation killing factor.  Our results suggest that bacillaene mediates translational arrest, and show that bacillaene B has a min. inhibitory concn. 10 × higher than unmodified bacillaene.  Finally, we show that BCP can be used to screen strains on an agar plate without the need for ext. prepn., greatly saving time and improving throughput.  Thus, BCP simplifies the isolation of novel natural products, by identifying strains, crude exts. and fractions with interesting bioactivities even when multiple activities are present, allowing investigators to focus labor-intensive steps on those with desired activities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVUxWGiUNr6bVg90H21EOLACvtfcHk0lh9ZWHTh2vRIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xos1anu7g%253D&md5=c842bc1c7746f8cd189cbc1cd0b39c7b</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1038%2Fja.2015.116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fja.2015.116%26sid%3Dliteratum%253Aachs%26aulast%3DNonejuie%26aufirst%3DP.%26aulast%3DTrial%26aufirst%3DR.%2BM.%26aulast%3DNewton%26aufirst%3DG.%2BL.%26aulast%3DLamsa%26aufirst%3DA.%26aulast%3DPerera%26aufirst%3DV.%2BR.%26aulast%3DAguilar%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DW.-T.%26aulast%3DDorrestein%26aufirst%3DP.%2BC.%26aulast%3DPogliano%26aufirst%3DJ.%26aulast%3DPogliano%26aufirst%3DK.%26atitle%3DApplication%2520of%2520Bacterial%2520Cytological%2520Profiling%2520to%2520Crude%2520Natural%2520Product%2520Extracts%2520Reveals%2520the%2520Antibacterial%2520Arsenal%2520of%2520Bacillus%2520subtilis%26jtitle%3DJ.%2520Antibiot.%26date%3D2016%26volume%3D69%26spage%3D353%26epage%3D361%26doi%3D10.1038%2Fja.2015.116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hamby, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Connolly, C. J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schroeder, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winters, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Showalter, H. D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panek, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Major, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olsewski, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryan, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dahring, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, G. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keiser, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraker, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slintak, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nelson, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fry, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradford, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hallak, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doherty, A. M.</span></span> <span> </span><span class="NLM_article-title">Structure-Activity Relationships for a Novel Series of Pyrido[2,3-d]pyrimidine Tyrosine Kinase Inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">2296</span>– <span class="NLM_lpage">2303</span>, <span class="refDoi"> DOI: 10.1021/jm970367n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm970367n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADyaK2sXktlKisLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=1997&pages=2296-2303&author=J.+M.+Hambyauthor=C.+J.+C.+Connollyauthor=M.+C.+Schroederauthor=R.+T.+Wintersauthor=H.+D.+H.+Showalterauthor=R.+L.+Panekauthor=T.+C.+Majorauthor=B.+Olsewskiauthor=M.+J.+Ryanauthor=T.+Dahringauthor=G.+H.+Luauthor=J.+Keiserauthor=A.+Amarauthor=C.+Shenauthor=A.+J.+Krakerauthor=V.+Slintakauthor=J.+M.+Nelsonauthor=D.+W.+Fryauthor=L.+Bradfordauthor=H.+Hallakauthor=A.+M.+Doherty&title=Structure-Activity+Relationships+for+a+Novel+Series+of+Pyrido%5B2%2C3-d%5Dpyrimidine+Tyrosine+Kinase+Inhibitors&doi=10.1021%2Fjm970367n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Activity Relationships for a Novel Series of Pyrido[2,3-d]pyrimidine Tyrosine Kinase Inhibitors</span></div><div class="casAuthors">Hamby, James M.; Connolly, Cleo J. C.; Schroeder, Mel C.; Winters, R. Thomas; Showalter, H. D. Hollis; Panek, Robert L.; Major, Terry C.; Olsewski, Bronislawa; Ryan, Michael J.; Dahring, Tawny; Lu, Gina H.; Keiser, Joan; Amar, Aneesa; Shen, Cindy; Kraker, Alan J.; Slintak, Veronika; Nelson, James M.; Fry, David W.; Bradford, Laura; Hallak, Hussein; Doherty, Annette M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">2296-2303</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Screening of a compd. library for inhibitors of the fibroblast growth factor (FGFr) and platelet-derived growth factor (PDGFr) receptor tyrosine kinases led to the development of a novel series of ATP competitive pyrido[2,3-d]pyrimidine tyrosine kinase inhibitors.  The initial lead, 1-[2-amino-6-(2,6-dichlorophenyl)pyrido[2,3-d]pyrimidin-7-yl]-3-tert-butylurea (I; PD-089828), was found to be a broadly active tyrosine kinase inhibitor.  I inhibited the PDGFr, FGFr, EGFr, and c-src tyrosine kinases with IC50 values of 1.11, 0.13, 0.45, and 0.22 μM, resp.  Subsequent SAR studies led to the synthesis of new analogs with improved potency, soly., and bioavailability relative to the initial lead.  For example, the introduction of a [4-(diethylamino)butyl]amino side chain into the 2-position of I afforded a compd. (II) with enhanced potency and bioavailability.  II inhibited PDGF-stimulated vascular smooth muscle cell proliferation with an IC50 of 0.3 μM.  Furthermore, replacement of the 6-(2,6-dichlorophenyl) moiety of I with a 6-(3',5'-dimethoxyphenyl) functionality produced a highly selective FGFr tyrosine kinase inhibitor (III).  III inhibited the FGFr tyrosine kinase with an IC50 of 0.060 μM, whereas IC50s for the inhibition of the PDGFr, FGFr, EGFr, c-src, and InsR tyrosine kinases for this compd. were all greater than 50 μM.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJdLcE9N0L8rVg90H21EOLACvtfcHk0lh9ZWHTh2vRIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXktlKisLs%253D&md5=f68e00a123b8b450e647761ac1421d77</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Fjm970367n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm970367n%26sid%3Dliteratum%253Aachs%26aulast%3DHamby%26aufirst%3DJ.%2BM.%26aulast%3DConnolly%26aufirst%3DC.%2BJ.%2BC.%26aulast%3DSchroeder%26aufirst%3DM.%2BC.%26aulast%3DWinters%26aufirst%3DR.%2BT.%26aulast%3DShowalter%26aufirst%3DH.%2BD.%2BH.%26aulast%3DPanek%26aufirst%3DR.%2BL.%26aulast%3DMajor%26aufirst%3DT.%2BC.%26aulast%3DOlsewski%26aufirst%3DB.%26aulast%3DRyan%26aufirst%3DM.%2BJ.%26aulast%3DDahring%26aufirst%3DT.%26aulast%3DLu%26aufirst%3DG.%2BH.%26aulast%3DKeiser%26aufirst%3DJ.%26aulast%3DAmar%26aufirst%3DA.%26aulast%3DShen%26aufirst%3DC.%26aulast%3DKraker%26aufirst%3DA.%2BJ.%26aulast%3DSlintak%26aufirst%3DV.%26aulast%3DNelson%26aufirst%3DJ.%2BM.%26aulast%3DFry%26aufirst%3DD.%2BW.%26aulast%3DBradford%26aufirst%3DL.%26aulast%3DHallak%26aufirst%3DH.%26aulast%3DDoherty%26aufirst%3DA.%2BM.%26atitle%3DStructure-Activity%2520Relationships%2520for%2520a%2520Novel%2520Series%2520of%2520Pyrido%255B2%252C3-d%255Dpyrimidine%2520Tyrosine%2520Kinase%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1997%26volume%3D40%26spage%3D2296%26epage%3D2303%26doi%3D10.1021%2Fjm970367n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hernandez, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crépin, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palencia, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cusack, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akama, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freund, Y. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meewan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rock, F. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sexton, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheoran, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nieman, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anugula, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keramane, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savariraj, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Subedi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Leary, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simon, N. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Marsh, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mushtaq, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warner, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Livermore, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alley, M. R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plattner, J. J.</span></span> <span> </span><span class="NLM_article-title">Discovery of a Novel Class of Boron-Based Antibacterials with Activity Against Gram-Negative Bacteria</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">1394</span>– <span class="NLM_lpage">1403</span>, <span class="refDoi"> DOI: 10.1128/AAC.02058-12</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1128%2FAAC.02058-12" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=23295920" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BC3sXltV2htLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2013&pages=1394-1403&author=V.+Hernandezauthor=T.+Cr%C3%A9pinauthor=A.+Palenciaauthor=S.+Cusackauthor=T.+Akamaauthor=S.+J.+Bakerauthor=W.+Buauthor=L.+Fengauthor=Y.+R.+Freundauthor=L.+Liuauthor=M.+Meewanauthor=M.+Mohanauthor=W.+Maoauthor=F.+L.+Rockauthor=H.+Sextonauthor=A.+Sheoranauthor=Y.+Zhangauthor=Y.-K.+Zhangauthor=Y.+Zhouauthor=J.+A.+Niemanauthor=M.+R.+Anugulaauthor=E.+M.+Keramaneauthor=K.+Savarirajauthor=D.+S.+Reddyauthor=R.+Sharmaauthor=R.+Subediauthor=R.+Singhauthor=A.+O%E2%80%99Learyauthor=N.+L.+Simonauthor=P.+L.+De+Marshauthor=S.+Mushtaqauthor=M.+Warnerauthor=D.+M.+Livermoreauthor=M.+R.+K.+Alleyauthor=J.+J.+Plattner&title=Discovery+of+a+Novel+Class+of+Boron-Based+Antibacterials+with+Activity+Against+Gram-Negative+Bacteria&doi=10.1128%2FAAC.02058-12"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a novel class of boron-based antibacterials with activity against Gram-negative bacteria</span></div><div class="casAuthors">Hernandez, Vincent; Crepin, Thibaut; Palencia, Andres; Cusack, Stephen; Akama, Tsutomu; Baker, Stephen J.; Bu, Wei; Feng, Lisa; Freund, Yvonne R.; Liu, Liang; Meewan, Maliwan; Mohan, Manisha; Mao, Weimin; Rock, Fernando L.; Sexton, Holly; Sheoran, Anita; Zhang, Yanchen; Zhang, Yong-Kang; Zhou, Yasheen; Nieman, James A.; Anugula, Mahipal Reddy; Keramane, El Mehdi; Savariraj, Kingsley; Reddy, D. Shekhar; Sharma, Rashmi; Subedi, Rajendra; Singh, Rajeshwar; O'Leary, Ann; Simon, Nerissa L.; De Marsh, Peter L.; Mushtaq, Shazad; Warner, Marina; Livermore, David M.; Alley, M. R. K.; Plattner, Jacob J.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1394-1403</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Gram-neg. bacteria cause approx. 70% of the infections in intensive care units.  A growing no. of bacterial isolates responsible for these infections are resistant to currently available antibiotics and to many in development.  Most agents under development are modifications of existing drug classes, which only partially overcome existing resistance mechanisms.  Therefore, new classes of Gram-neg. antibacterials with truly novel modes of action are needed to circumvent these existing resistance mechanisms.  We have previously identified a new a way to inhibit an aminoacyl-tRNA synthetase, leucyl-tRNA synthetase (LeuRS), in fungi via the oxaborole tRNA trapping (OBORT) mechanism.  Herein, we show how we have modified the OBORT mechanism using a structure-guided approach to develop a new boron-based antibiotic class, the aminomethylbenzoxaboroles, which inhibit bacterial leucyl-tRNA synthetase and have activity against Gram-neg. bacteria by largely evading the main efflux mechanisms in Escherichia coli and Pseudomonas aeruginosa.  The lead analog, AN3365, is active against Gram-neg. bacteria, including Enterobacteriaceae bearing NDM-1 and KPC carbapenemases, as well as P. aeruginosa.  This novel boron-based antibacterial, AN3365, has good mouse pharmacokinetics and was efficacious against E. coli and P. aeruginosa in murine thigh infection models, which suggest that this novel class of antibacterials has the potential to address this unmet medical need.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWl5Oj0jzVnLVg90H21EOLACvtfcHk0ljmwmwoenE0PA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXltV2htLg%253D&md5=7ed74bde2bacfa4ae20abb50b745c57d</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1128%2FAAC.02058-12&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.02058-12%26sid%3Dliteratum%253Aachs%26aulast%3DHernandez%26aufirst%3DV.%26aulast%3DCr%25C3%25A9pin%26aufirst%3DT.%26aulast%3DPalencia%26aufirst%3DA.%26aulast%3DCusack%26aufirst%3DS.%26aulast%3DAkama%26aufirst%3DT.%26aulast%3DBaker%26aufirst%3DS.%2BJ.%26aulast%3DBu%26aufirst%3DW.%26aulast%3DFeng%26aufirst%3DL.%26aulast%3DFreund%26aufirst%3DY.%2BR.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DMeewan%26aufirst%3DM.%26aulast%3DMohan%26aufirst%3DM.%26aulast%3DMao%26aufirst%3DW.%26aulast%3DRock%26aufirst%3DF.%2BL.%26aulast%3DSexton%26aufirst%3DH.%26aulast%3DSheoran%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DY.-K.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DNieman%26aufirst%3DJ.%2BA.%26aulast%3DAnugula%26aufirst%3DM.%2BR.%26aulast%3DKeramane%26aufirst%3DE.%2BM.%26aulast%3DSavariraj%26aufirst%3DK.%26aulast%3DReddy%26aufirst%3DD.%2BS.%26aulast%3DSharma%26aufirst%3DR.%26aulast%3DSubedi%26aufirst%3DR.%26aulast%3DSingh%26aufirst%3DR.%26aulast%3DO%25E2%2580%2599Leary%26aufirst%3DA.%26aulast%3DSimon%26aufirst%3DN.%2BL.%26aulast%3DDe%2BMarsh%26aufirst%3DP.%2BL.%26aulast%3DMushtaq%26aufirst%3DS.%26aulast%3DWarner%26aufirst%3DM.%26aulast%3DLivermore%26aufirst%3DD.%2BM.%26aulast%3DAlley%26aufirst%3DM.%2BR.%2BK.%26aulast%3DPlattner%26aufirst%3DJ.%2BJ.%26atitle%3DDiscovery%2520of%2520a%2520Novel%2520Class%2520of%2520Boron-Based%2520Antibacterials%2520with%2520Activity%2520Against%2520Gram-Negative%2520Bacteria%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2013%26volume%3D57%26spage%3D1394%26epage%3D1403%26doi%3D10.1128%2FAAC.02058-12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lomovskaya, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoshino, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishida, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mistry, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warren, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boyer, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chamberland, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, V. J.</span></span> <span> </span><span class="NLM_article-title">Use of a Genetic Approach to Evaluate the Consequences of Inhibition of Efflux Pumps in <i>Pseudomonas aeruginosa</i></span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">1340</span>– <span class="NLM_lpage">1346</span>, <span class="refDoi"> DOI: 10.1128/AAC.43.6.1340</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1128%2FAAC.43.6.1340" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10348749" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADyaK1MXjs12ns70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=1999&pages=1340-1346&author=O.+Lomovskayaauthor=A.+Leeauthor=K.+Hoshinoauthor=H.+Ishidaauthor=A.+Mistryauthor=M.+S.+Warrenauthor=E.+Boyerauthor=S.+Chamberlandauthor=V.+J.+Lee&title=Use+of+a+Genetic+Approach+to+Evaluate+the+Consequences+of+Inhibition+of+Efflux+Pumps+in+Pseudomonas+aeruginosa&doi=10.1128%2FAAC.43.6.1340"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Use of a genetic approach to evaluate the consequences of inhibition of efflux pumps in Pseudomonas aeruginosa</span></div><div class="casAuthors">Lomovskaya, Olga; Lee, Angela; Hoshino, Kazuki; Ishida, Hiroko; Mistry, Anita; Warren, Mark S.; Boyer, Eric; Chamberland, Suzanne; Lee, Ving J.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1340-1346</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Drug efflux pumps in P. aeruginosa were evaluated as potential targets for antibacterial therapy.  The potential effects of pump inhibition on susceptibility to fluoroquinolone antibiotics were studied with isogenic strains that overexpress or lack individual efflux pumps and that have various combinations of efflux-and target-mediated mutations.  Deletions in 3 efflux pump operons were constructed.  As expected, deletion of the MexAB-OprM efflux pump decreased resistance to fluoroquinolones in the wild-type P. aeruginosa (16-fold redn. for levofloxacin [LVX]) or in the strain that overexpressed mexAB-oprM operon (64-fold redn. for LVX).  In addn. to that, resistance to LVX was significantly reduced even for the strains carrying target mutations (64-fold for strains for which LVX MICs were >4 μg/mL).  The frequencies of emergence of LVX-resistant variants from different deletion mutants and the wild-type strain also were studied.  Deletion of individual pumps or pairs of the pumps did not significantly affect the frequency of emergence of resistant variants (at 4 × the MIC for the wild-type strain) compared to that for the wild type (10-6 to 10-7).  In the case of the strain with a triple deletion, the frequency of spontaneous mutants was undetectable (<10-11).  In summary, inhibition of drug efflux pumps would (i) significantly decrease the level of intrinsic resistance, (ii) reverse acquired resistance, and (iii) result in a decreased frequency of emergence of P. aeruginosa strains highly resistant to fluoroquinolones in clin. settings.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqc1gq1c_6DILVg90H21EOLACvtfcHk0ljmwmwoenE0PA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXjs12ns70%253D&md5=538a0e96bed5be6be50833f29274057e</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1128%2FAAC.43.6.1340&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.43.6.1340%26sid%3Dliteratum%253Aachs%26aulast%3DLomovskaya%26aufirst%3DO.%26aulast%3DLee%26aufirst%3DA.%26aulast%3DHoshino%26aufirst%3DK.%26aulast%3DIshida%26aufirst%3DH.%26aulast%3DMistry%26aufirst%3DA.%26aulast%3DWarren%26aufirst%3DM.%2BS.%26aulast%3DBoyer%26aufirst%3DE.%26aulast%3DChamberland%26aufirst%3DS.%26aulast%3DLee%26aufirst%3DV.%2BJ.%26atitle%3DUse%2520of%2520a%2520Genetic%2520Approach%2520to%2520Evaluate%2520the%2520Consequences%2520of%2520Inhibition%2520of%2520Efflux%2520Pumps%2520in%2520Pseudomonas%2520aeruginosa%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D1999%26volume%3D43%26spage%3D1340%26epage%3D1346%26doi%3D10.1128%2FAAC.43.6.1340" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tian, Z.-X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mac Aogáin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Connor, H. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fargier, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mooij, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Gara, F.</span></span> <span> </span><span class="NLM_article-title">MexT Modulates Virulence Determinants in <i>Pseudomonas aeruginosa</i> Independent of the MexEF-OprN Efflux Pump</span>. <i>Microb. Pathog.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">237</span>– <span class="NLM_lpage">241</span>, <span class="refDoi"> DOI: 10.1016/j.micpath.2009.08.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1016%2Fj.micpath.2009.08.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=19683048" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFyntLnI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2009&pages=237-241&author=Z.-X.+Tianauthor=M.+Mac+Aog%C3%A1inauthor=H.+F.+O%E2%80%99Connorauthor=E.+Fargierauthor=M.+J.+Mooijauthor=C.+Adamsauthor=Y.-P.+Wangauthor=F.+O%E2%80%99Gara&title=MexT+Modulates+Virulence+Determinants+in+Pseudomonas+aeruginosa+Independent+of+the+MexEF-OprN+Efflux+Pump&doi=10.1016%2Fj.micpath.2009.08.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">MexT modulates virulence determinants in Pseudomonas aeruginosa independent of the MexEF-OprN efflux pump</span></div><div class="casAuthors">Tian, Zhe-Xian; MacAogain, Micheal; O'Connor, Hazel F.; Fargier, Emilie; Mooij, Marlies J.; Adams, Claire; Wang, Yi-Ping; O'Gara, Fergal</div><div class="citationInfo"><span class="NLM_cas:title">Microbial Pathogenesis</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">237-241</span>CODEN:
                <span class="NLM_cas:coden">MIPAEV</span>;
        ISSN:<span class="NLM_cas:issn">0882-4010</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">In the human pathogen Pseudomonas aeruginosa, the LysR-family regulator MexT modulates the induction of the tripartite MexEF-OprN resistance nodulation-division multi-drug efflux system resulting in increased resistance to diverse antibiotics.  The MexEF-OprN system is normally quiescent in wild-type cells, but is highly induced in nfxC-type phenotypic mutants in a MexT dependent manner.  In addn. to antibiotic resistance, induction of mexEF-oprN in nfxC-type mutants has been linked to reduced levels of homoserine lactone-dependent virulence traits, including pyocyanin, elastase, rhamnolipids and PQS and to reduced expression of type three secretion effector proteins.  In this study, MexT is overexpressed in wild-type PAO1 and an isogenic mexEF deletion mutant to det. if MexT regulates diverse virulence phenotypes dependent or independent of MexEF-OprN.  It is shown that MexT regulates type three secretion, pyocyanin prodn. and early surface attachment independent of MexEF-OprN.  In contrast, MexT modulation of the expression of the virulence genes rhlA, lasB and hcnB is dependent on MexEF-OprN, which apparently mediates these effects via efflux of cell-signaling intermediates.  The data presented demonstrates that MexT may play a more global role in modulating P. aeruginosa virulence than previously reported and suggests that MexT regulates diverse targets that mediate phenotypic alterations independent of MexEF-OprN.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoyGW6fcwbDWbVg90H21EOLACvtfcHk0ljnmJxOZDVtGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFyntLnI&md5=f7e94dd750d16e6df937a9d4c2cddd1d</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1016%2Fj.micpath.2009.08.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.micpath.2009.08.003%26sid%3Dliteratum%253Aachs%26aulast%3DTian%26aufirst%3DZ.-X.%26aulast%3DMac%2BAog%25C3%25A1in%26aufirst%3DM.%26aulast%3DO%25E2%2580%2599Connor%26aufirst%3DH.%2BF.%26aulast%3DFargier%26aufirst%3DE.%26aulast%3DMooij%26aufirst%3DM.%2BJ.%26aulast%3DAdams%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DY.-P.%26aulast%3DO%25E2%2580%2599Gara%26aufirst%3DF.%26atitle%3DMexT%2520Modulates%2520Virulence%2520Determinants%2520in%2520Pseudomonas%2520aeruginosa%2520Independent%2520of%2520the%2520MexEF-OprN%2520Efflux%2520Pump%26jtitle%3DMicrob.%2520Pathog.%26date%3D2009%26volume%3D47%26spage%3D237%26epage%3D241%26doi%3D10.1016%2Fj.micpath.2009.08.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tian, Z.-X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fargier, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mac Aogáin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Gara, F.</span></span> <span> </span><span class="NLM_article-title">Transcriptome Profiling Defines a Novel Regulon Modulated by the LysR-Type Transcriptional Regulator MexT in <i>Pseudomonas aeruginosa</i></span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">7546</span>– <span class="NLM_lpage">7559</span>, <span class="refDoi"> DOI: 10.1093/nar/gkp828</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1093%2Fnar%2Fgkp828" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=19846594" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFyktrzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2009&pages=7546-7559&author=Z.-X.+Tianauthor=E.+Fargierauthor=M.+Mac+Aog%C3%A1inauthor=C.+Adamsauthor=Y.-P.+Wangauthor=F.+O%E2%80%99Gara&title=Transcriptome+Profiling+Defines+a+Novel+Regulon+Modulated+by+the+LysR-Type+Transcriptional+Regulator+MexT+in+Pseudomonas+aeruginosa&doi=10.1093%2Fnar%2Fgkp828"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Transcriptome profiling defines a novel regulon modulated by the LysR-type transcriptional regulator MexT in Pseudomonas aeruginosa</span></div><div class="casAuthors">Tian, Zhe-Xian; Fargier, Emilie; MacAogain, Micheal; Adams, Claire; Wang, Yi-Ping; O'Gara, Fergal</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">7546-7559</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The LysR-family regulator MexT modulates the expression of the MexEF-OprN efflux system in the human pathogen Pseudomonas aeruginosa.  Recently, we demonstrated that MexT regulates certain virulence phenotypes, including the type-three secretion system and early attachment independent of its role in regulating MexEF-OprN.  In this study, transcriptome profiling was utilized to investigate the global nature of MexT regulation in P. aeruginosa PAO1 and an isogenic mexEF mutant.  Twelve genes of unknown function were highly induced by overexpressing MexT independent of MexEF-OprN.  A well-conserved DNA motif was identified in the upstream regulatory region of nine of these genes and upstream of mexE.  Reporter fusion anal. demonstrated that the expression of the genes was significantly induced by MexT in P. aeruginosa and a heterogenous Escherichia coli strain and that the conserved sequence was required for this induction.  The conserved DNA motif was further characterized as the MexT binding site by site-directed mutagenesis and electrophoretic mobility shift assays.  Genes contg. this conserved regulatory sequence were identified across other Pseudomonas species, and their expression was activated by MexT.  Thus, a novel regulon directly modulated by MexT, that includes but is independent of mexEF-oprN, has been identified.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_qS5Q2HRJDrVg90H21EOLACvtfcHk0ljnmJxOZDVtGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFyktrzE&md5=3669c88ab1757d7c607abf66d9fbb486</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkp828&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkp828%26sid%3Dliteratum%253Aachs%26aulast%3DTian%26aufirst%3DZ.-X.%26aulast%3DFargier%26aufirst%3DE.%26aulast%3DMac%2BAog%25C3%25A1in%26aufirst%3DM.%26aulast%3DAdams%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DY.-P.%26aulast%3DO%25E2%2580%2599Gara%26aufirst%3DF.%26atitle%3DTranscriptome%2520Profiling%2520Defines%2520a%2520Novel%2520Regulon%2520Modulated%2520by%2520the%2520LysR-Type%2520Transcriptional%2520Regulator%2520MexT%2520in%2520Pseudomonas%2520aeruginosa%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2009%26volume%3D37%26spage%3D7546%26epage%3D7559%26doi%3D10.1093%2Fnar%2Fgkp828" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Luong, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shogan, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaborin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belogortseva, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shrout, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaborina, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alverdy, J. C.</span></span> <span> </span><span class="NLM_article-title">Emergence of the P2 Phenotype in <i>Pseudomonas aeruginosa</i> PAO1 Strains Involves Various Mutations in MexT or MexF</span>. <i>J. Bacteriol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>196</i></span>,  <span class="NLM_fpage">504</span>– <span class="NLM_lpage">513</span>, <span class="refDoi"> DOI: 10.1128/JB.01050-13</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1128%2FJB.01050-13" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=24244000" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFOlu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=196&publication_year=2014&pages=504-513&author=P.+M.+Luongauthor=B.+D.+Shoganauthor=A.+Zaborinauthor=N.+Belogortsevaauthor=J.+D.+Shroutauthor=O.+Zaborinaauthor=J.+C.+Alverdy&title=Emergence+of+the+P2+Phenotype+in+Pseudomonas+aeruginosa+PAO1+Strains+Involves+Various+Mutations+in+MexT+or+MexF&doi=10.1128%2FJB.01050-13"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Emergence of the P2 phenotype in Pseudomonas aeruginosa PAO1 strains involves various mutations in mexT or mexF</span></div><div class="casAuthors">Luong, Preston M.; Shogan, Benjamin D.; Zaborin, Alexander; Belogortseva, Natalia; Shrout, Joshua D.; Zaborina, Olga; Alverdy, John C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Bacteriology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">196</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">504-513, 11 pp.</span>CODEN:
                <span class="NLM_cas:coden">JOBAAY</span>;
        ISSN:<span class="NLM_cas:issn">1098-5530</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">We recently demonstrated that Pseudomonas aeruginosa PAO1 undergoes a pronounced phenotypic change when introduced into the intestines of rats during surgical injury.  Recovered strains displayed a specific phenotype (termed the P2 phenotype) characterized by altered pyocyanin prodn., high collagenase activity, high swarming motility, low resistance to chloramphenicol, and increased killing of Caenorhabditis elegans compared to the inoculating strain (termed the P1 phenotype).  The aims of this study were to characterize the differences between the P. aeruginosa P1 and P2 phenotypes in quorum sensing and competitiveness.  We then detd. the presence of the P2 phenotype among PAO1 strains from various labs.  Results demonstrated that P2 cells display accelerated growth during early exponential phase and early activation of quorum-sensing systems and overcome the growth of P1 cells in a mixed population.  Among eight PAO1 strains obtained from different labs., four exhibited the P2 phenotype.  Of 27 mutants analyzed from the P. aeruginosa MPAO1 transposon library, 25 displayed P2 phenotypes.  The P2 phenotype in both cases correlated with a lack of expression of mexE or mexF due to mutations in mexT and mexF genes.  In summary, strains possessing the P2 phenotype are distributed among PAO1 strains commonly used across a variety of research labs.  Genetically, they are characterized by various mutations in mexT or mexF.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfJd5RNfcvuLVg90H21EOLACvtfcHk0ljnmJxOZDVtGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFOlu78%253D&md5=0013229194244ab996822e5854549998</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1128%2FJB.01050-13&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJB.01050-13%26sid%3Dliteratum%253Aachs%26aulast%3DLuong%26aufirst%3DP.%2BM.%26aulast%3DShogan%26aufirst%3DB.%2BD.%26aulast%3DZaborin%26aufirst%3DA.%26aulast%3DBelogortseva%26aufirst%3DN.%26aulast%3DShrout%26aufirst%3DJ.%2BD.%26aulast%3DZaborina%26aufirst%3DO.%26aulast%3DAlverdy%26aufirst%3DJ.%2BC.%26atitle%3DEmergence%2520of%2520the%2520P2%2520Phenotype%2520in%2520Pseudomonas%2520aeruginosa%2520PAO1%2520Strains%2520Involves%2520Various%2520Mutations%2520in%2520MexT%2520or%2520MexF%26jtitle%3DJ.%2520Bacteriol.%26date%3D2014%26volume%3D196%26spage%3D504%26epage%3D513%26doi%3D10.1128%2FJB.01050-13" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group">CSLI</span>.  <i>CSLI M07-A10: Methods for Dilution Antimicrobial
Susceptibility
Tests for Bacteria That Grow Aerobically; Approved Standard-Tenth
Edition</i>. <span class="NLM_publisher-name">Clinical and Laboratory Standards
Institute</span>: <span class="NLM_publisher-loc">Wayne, PA</span>; <span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=CSLI&title=CSLI+M07-A10%3A+Methods+for+Dilution+Antimicrobial%0ASusceptibility%0ATests+for+Bacteria+That+Grow+Aerobically%3B+Approved+Standard-Tenth%0AEdition"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3D%26btitle%3DCSLI%2520M07-A10%253A%2520Methods%2520for%2520Dilution%2520Antimicrobial%250ASusceptibility%250ATests%2520for%2520Bacteria%2520That%2520Grow%2520Aerobically%253B%2520Approved%2520Standard-Tenth%250AEdition%26pub%3DClinical%2520and%2520Laboratory%2520Standards%250AInstitute%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">XDS</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1107/S0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=XDS&doi=10.1107%2FS0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0liQYGNIPYurmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXDS%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2FS0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P.</span></span> <span> </span><span class="NLM_article-title">Scaling and Assessment of Data Quality</span>. <i>Acta Crystallogr., Sect. D: Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">72</span>– <span class="NLM_lpage">82</span>, <span class="refDoi"> DOI: 10.1107/S0907444905036693</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1107%2FS0907444905036693" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=16369096" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlant7jM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2006&pages=72-82&author=P.+Evans&title=Scaling+and+Assessment+of+Data+Quality&doi=10.1107%2FS0907444905036693"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Scaling and assessment of data quality</span></div><div class="casAuthors">Evans, Philip</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">D62</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">72-82</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">The various phys. factors affecting measured diffraction intensities are discussed, as are the scaling models which may be used to put the data on a consistent scale.  After scaling, the intensities can be analyzed to set the real resoln. of the data set, to detect bad regions (e.g. bad images), to analyze radiation damage and to assess the overall quality of the data set.  The significance of any anomalous signal may be assessed by probability and correlation anal.  The algorithms used by the CCP4 scaling program SCALA are described.  A requirement for the scaling and merging of intensities is knowledge of the Laue group and point-group symmetries: the possible symmetry of the diffraction pattern may be detd. from scores such as correlation coeffs. between observations which might be symmetry-related.  These scoring functions are implemented in a new program POINTLESS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGravLSLU62RhLVg90H21EOLACvtfcHk0liQYGNIPYurmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlant7jM&md5=293d3876e534c0c57813990515bb3c76</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1107%2FS0907444905036693&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444905036693%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DP.%26atitle%3DScaling%2520and%2520Assessment%2520of%2520Data%2520Quality%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Struct.%2520Biol.%26date%3D2006%26volume%3D62%26spage%3D72%26epage%3D82%26doi%3D10.1107%2FS0907444905036693" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunkóczi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echols, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oeffner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, P. H.</span></span> <span> </span><span class="NLM_article-title">PHENIX:A Comprehensive Python-Based System for Macromolecular Structure Solution</span>. <i>Acta Crystallogr., Sect. D: Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">213</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1107/S0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunk%C3%B3cziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.-W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3AA+Comprehensive+Python-Based+System+for+Macromolecular+Structure+Solution&doi=10.1107%2FS0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0liQYGNIPYurmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunk%25C3%25B3czi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.-W.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253AA%2520Comprehensive%2520Python-Based%2520System%2520for%2520Macromolecular%2520Structure%2520Solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Struct.%2520Biol.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2FS0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group">Collaborative
Computation Project, Number 4</span> <span> </span><span class="NLM_article-title">The CCP4 Suite: Programs for Protein Crystallography</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">760</span>– <span class="NLM_lpage">763</span>, <span class="refDoi"> DOI: 10.1107/S0907444994003112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1107%2FS0907444994003112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=15299374" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=1994&pages=760-763&author=Collaborative%0AComputation+Project%2C+Number+4&title=The+CCP4+Suite%3A+Programs+for+Protein+Crystallography&doi=10.1107%2FS0907444994003112"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1107%2FS0907444994003112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444994003112%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DThe%2520CCP4%2520Suite%253A%2520Programs%2520for%2520Protein%2520Crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1994%26volume%3D50%26spage%3D760%26epage%3D763%26doi%3D10.1107%2FS0907444994003112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Painter, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merritt, E. A.</span></span> <span> </span><span class="NLM_article-title">TLSMD Web Server for the Generation of Multi-Group TLS Models</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">109</span>– <span class="NLM_lpage">11</span>, <span class="refDoi"> DOI: 10.1107/S0021889805038987</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1107%2FS0021889805038987" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BD28XmvVyjsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2006&pages=109-11&author=J.+Painterauthor=E.+A.+Merritt&title=TLSMD+Web+Server+for+the+Generation+of+Multi-Group+TLS+Models&doi=10.1107%2FS0021889805038987"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">TLSMD web server for the generation of multi-group TLS models</span></div><div class="casAuthors">Painter, Jay; Merritt, Ethan A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">109-111</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">The TLSMD web server exts. information about dynamic properties of a protein based on information derived from a single-crystal structure.  It does so by analyzing the spatial distribution of individual at. thermal parameters present in an input structural model.  The server partitions the protein structure into multiple, contiguous chain segments, each segment corresponding to one group in a multi-group description of the protein's overall dynamic motion.  For each polypeptide chain of the input protein, the anal. generates the optimal partition into two segments, three segments,... up to 20 segments.  Each such partition is optimal in the sense that it is the best approxn. of the overall spatial distribution of input thermal parameters in terms of N chain segments, each acting as a rigid group undergoing TLS (translation/libration/screw) motion.  This multi-group TLS model may be used as a starting point for further crystallog. refinement, or as the basis for analyzing inter-domain and other large-scale motions implied by the crystal structure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhkwf2sopjVrVg90H21EOLACvtfcHk0liIr8NjJScj3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XmvVyjsA%253D%253D&md5=5d16d5d6310baa5434064bdf0ec81e24</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1107%2FS0021889805038987&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889805038987%26sid%3Dliteratum%253Aachs%26aulast%3DPainter%26aufirst%3DJ.%26aulast%3DMerritt%26aufirst%3DE.%2BA.%26atitle%3DTLSMD%2520Web%2520Server%2520for%2520the%2520Generation%2520of%2520Multi-Group%2520TLS%2520Models%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2006%26volume%3D39%26spage%3D109%26epage%3D11%26doi%3D10.1107%2FS0021889805038987" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Coot: Model-Building Tools for Molecular Graphics</span>. <i>Acta Crystallogr., Sect. D: Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2126</span>– <span class="NLM_lpage">2132</span>, <span class="refDoi"> DOI: 10.1107/S0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+Model-Building+Tools+for+Molecular+Graphics&doi=10.1107%2FS0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0liIr8NjJScj3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520Model-Building%2520Tools%2520for%2520Molecular%2520Graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Struct.%2520Biol.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2FS0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2V58" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2V58','PDB','2V58'); return false;">PDB: 2V58</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6OI9" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6OI9','PDB','6OI9'); return false;">PDB: 6OI9</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6OI8" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6OI8','PDB','6OI8'); return false;">PDB: 6OI8</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2VQD" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2VQD','PDB','2VQD'); return false;">PDB: 2VQD</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4RZQ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4RZQ','PDB','4RZQ'); return false;">PDB: 4RZQ</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i67"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00625">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_76275"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.9b00625">10.1021/acs.jmedchem.9b00625</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Reaction catalyzed by BC and the actyl-CoA carboxylase multienzyme complex, tests of BC essentiality in <i>P. aeruginosa</i>, and X-ray crystallographic conditions and statistics (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00625/suppl_file/jm9b00625_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00625/suppl_file/jm9b00625_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00625/suppl_file/jm9b00625_si_001.pdf">jm9b00625_si_001.pdf (179.66 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00625/suppl_file/jm9b00625_si_002.csv">jm9b00625_si_002.csv (1.42 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b00625&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2019.62.issue-16%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b00625" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b00625" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                7MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799e470eeb03c46","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
