
<html lang="en"     class="pb-page"  data-request-id="3af18214-edf1-4d9e-99fb-9076714f57ac"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.9b01210;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;issue:issue:10.1021/jmcmar.2019.62.issue-22"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Structure–Activity Relationship of Peptide-Conjugated Chloramphenicol for Inhibiting Escherichia coli" /></meta><meta name="dc.Creator" content="Jiaqing  Wang" /></meta><meta name="dc.Creator" content="Adrianna  Shy" /></meta><meta name="dc.Creator" content="Difei  Wu" /></meta><meta name="dc.Creator" content="Deani L.  Cooper" /></meta><meta name="dc.Creator" content="Jiashu  Xu" /></meta><meta name="dc.Creator" content="Hongjian  He" /></meta><meta name="dc.Creator" content="Wenjun  Zhan" /></meta><meta name="dc.Creator" content="Shenghuan  Sun" /></meta><meta name="dc.Creator" content="Susan T.  Lovett" /></meta><meta name="dc.Creator" content="Bing  Xu" /></meta><meta name="dc.Description" content="Intravenous administration of a prodrug, chloramphenicol succinate (CLsu), is ineffective. Recently, we have shown that conjugation of diglycine of CLsu (CLsuGG) not only increases the antibiotic e..." /></meta><meta name="Description" content="Intravenous administration of a prodrug, chloramphenicol succinate (CLsu), is ineffective. Recently, we have shown that conjugation of diglycine of CLsu (CLsuGG) not only increases the antibiotic e..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="October 31, 2019" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b01210" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2019 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b01210" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b01210" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b01210" /></link>
        
    
    

<title>Structure–Activity Relationship of Peptide-Conjugated Chloramphenicol for Inhibiting Escherichia coli | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b01210" /></meta><meta property="og:title" content="Structure–Activity Relationship of Peptide-Conjugated Chloramphenicol for Inhibiting Escherichia coli" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01210/20191122/images/large/jm9b01210_0008.jpeg" /></meta><meta property="og:description" content="Intravenous administration of a prodrug, chloramphenicol succinate (CLsu), is ineffective. Recently, we have shown that conjugation of diglycine of CLsu (CLsuGG) not only increases the antibiotic efficacy against Escherichia coli but also reduces adverse drug effects against bone marrow stromal cells. Here, we report the synthesis of structural analogues of CLsuGG and their activities against E. coli. These analogues reveal several trends: (i) except the water-insoluble analogues, the attachment of peptides to CLsu enhances the efficacy of the prodrugs; (ii) negative charges, high steric hindrance in the side chains, or a rigid diester decreases the activities of prodrugs in comparison to CLsuGG; (iii) dipeptides apparently increase the efficacy of the prodrugs most effectively; and so forth. This work suggests that conjugating peptides to CLsu effectively modulates the properties of prodrugs. The structure–activity relationship of these new conjugates may provide useful insights for expanding the pool of antibiotics." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b01210"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01210">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b01210&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b01210&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b01210&amp;href=/doi/10.1021/acs.jmedchem.9b01210" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2019</span><span class="cit-fg-volume">, 62</span><span class="cit-fg-issue">, 22</span><span class="cit-fg-pageRange">, 10245-10257</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/62/22" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.9b01201" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.9b01310" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Structure–Activity Relationship of Peptide-Conjugated Chloramphenicol for Inhibiting <i>Escherichia coli</i></span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Jiaqing Wang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jiaqing Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, Brandeis University, 415 South Street, Waltham, Massachusetts 02453, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jiaqing++Wang">Jiaqing Wang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Adrianna Shy</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Adrianna Shy</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, Brandeis University, 415 South Street, Waltham, Massachusetts 02453, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Adrianna++Shy">Adrianna Shy</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Difei Wu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Difei Wu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, Brandeis University, 415 South Street, Waltham, Massachusetts 02453, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Difei++Wu">Difei Wu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Deani L. Cooper</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Deani L. Cooper</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biology, Brandeis University, 415 South Street, Waltham, Massachusetts 02453, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Deani+L.++Cooper">Deani L. Cooper</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jiashu Xu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jiashu Xu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, Brandeis University, 415 South Street, Waltham, Massachusetts 02453, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jiashu++Xu">Jiashu Xu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Hongjian He</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hongjian He</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, Brandeis University, 415 South Street, Waltham, Massachusetts 02453, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hongjian++He">Hongjian He</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Wenjun Zhan</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Wenjun Zhan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, Brandeis University, 415 South Street, Waltham, Massachusetts 02453, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Wenjun++Zhan">Wenjun Zhan</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Shenghuan Sun</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Shenghuan Sun</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, Brandeis University, 415 South Street, Waltham, Massachusetts 02453, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Shenghuan++Sun">Shenghuan Sun</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Susan T. Lovett</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Susan T. Lovett</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biology, Brandeis University, 415 South Street, Waltham, Massachusetts 02453, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Susan+T.++Lovett">Susan T. Lovett</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Bing Xu</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Bing Xu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, Brandeis University, 415 South Street, Waltham, Massachusetts 02453, United States</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#f2908a87b29080939c96979b81dc979687"><span class="__cf_email__" data-cfemail="096b717c496b7b68676d6c607a276c6d7c">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Bing++Xu">Bing Xu</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-4639-387X" title="Orcid link">http://orcid.org/0000-0002-4639-387X</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b01210&amp;href=/doi/10.1021%2Facs.jmedchem.9b01210" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2019</span></span><span class="cit-volume">, 62</span><span class="cit-issue">, 22</span><span class="cit-pageRange">, 10245–10257</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">October 31, 2019</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>25 July 2019</li><li><span class="item_label"><b>Published</b> online</span>31 October 2019</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 27 November 2019</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b01210" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01210</a></div><div class="article_header-article-copyright"><strong>Copyright © 2019 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D10245%26pageCount%3D13%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DJiaqing%2BWang%252C%2BAdrianna%2BShy%252C%2BDifei%2BWu%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D62%26issueNum%3D22%26contentID%3Dacs.jmedchem.9b01210%26title%3DStructure%25E2%2580%2593Activity%2BRelationship%2Bof%2BPeptide-Conjugated%2BChloramphenicol%2Bfor%2BInhibiting%2BEscherichia%2Bcoli%26numPages%3D13%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D10257%26publicationDate%3DNovember%2B2019">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b01210"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">919</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">2</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b01210" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Structure–Activity Relationship of Peptide-Conjugated Chloramphenicol for Inhibiting Escherichia coli&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Jiaqing&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Adrianna&quot;,&quot;last_name&quot;:&quot;Shy&quot;},{&quot;first_name&quot;:&quot;Difei&quot;,&quot;last_name&quot;:&quot;Wu&quot;},{&quot;first_name&quot;:&quot;Deani&quot;,&quot;last_name&quot;:&quot;L. Cooper&quot;},{&quot;first_name&quot;:&quot;Jiashu&quot;,&quot;last_name&quot;:&quot;Xu&quot;},{&quot;first_name&quot;:&quot;Hongjian&quot;,&quot;last_name&quot;:&quot;He&quot;},{&quot;first_name&quot;:&quot;Wenjun&quot;,&quot;last_name&quot;:&quot;Zhan&quot;},{&quot;first_name&quot;:&quot;Shenghuan&quot;,&quot;last_name&quot;:&quot;Sun&quot;},{&quot;first_name&quot;:&quot;Susan&quot;,&quot;last_name&quot;:&quot;T. Lovett&quot;},{&quot;first_name&quot;:&quot;Bing&quot;,&quot;last_name&quot;:&quot;Xu&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2019&quot;,&quot;month&quot;:&quot;10&quot;,&quot;day&quot;:&quot;31&quot;,&quot;issue&quot;:&quot;22&quot;,&quot;volume&quot;:&quot;62&quot;,&quot;pages&quot;:&quot;10245-10257&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b01210&quot;},&quot;abstract&quot;:&quot;Intravenous administration of a prodrug, chloramphenicol succinate (CLsu), is ineffective. Recently, we have shown that conjugation of diglycine of CLsu (CLsuGG) not only increases the antibiotic efficacy against Escherichia coli but also reduces adverse drug effects against bone marrow stromal cells. Here, we report the synthesis of structural analogues of CLsuGG and their activities against E. coli. These analogues reveal several trends: (i) except the water-insoluble analogues, the attachment of peptides to CLsu enhances the efficacy of the prodrugs; (ii) negative charges, high steric hindrance in the side chains, or a rigid diester decreases the activities of prodrugs in comparison to CLsuGG; (iii) dipeptides apparently increase the efficacy of the prodrugs most effectively; and so forth. This work suggests that conjugating peptides to CLsu effectively modulates the properties of prodrugs. The structure–activity relationship of these new conjugates may provide useful insights for expanding the pool of ant&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01210&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01210" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01210&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01210" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01210&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01210" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b01210&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01210&amp;href=/doi/10.1021/acs.jmedchem.9b01210" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b01210" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b01210" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (3 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01210&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b01210%26sid%3Dliteratum%253Aachs%26pmid%3D31670952%26genre%3Darticle%26aulast%3DWang%26date%3D2019%26atitle%3DStructure%25E2%2580%2593Activity%2BRelationship%2Bof%2BPeptide-Conjugated%2BChloramphenicol%2Bfor%2BInhibiting%2BEscherichia%2Bcoli%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D62%26issue%3D22%26spage%3D10245%26epage%3D10257%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291362" title="Antimicrobial agents">Antimicrobial agents</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291208" title="Hydrolysis">Hydrolysis</a>,</li><li><a href="/action/doSearch?ConceptID=291070" title="Pharmaceuticals">Pharmaceuticals</a>,</li><li><a href="/action/doSearch?ConceptID=291290" title="Conjugate acid-base pairs">Conjugate acid-base pairs</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/62/22" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/jmcmar.2019.62.issue-22/20191127/jmcmar.2019.62.issue-22.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01210/20191122/images/medium/jm9b01210_0008.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01210/20191122/images/large/jm9b01210_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01210&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Intravenous administration of a prodrug, chloramphenicol succinate (CLsu), is ineffective. Recently, we have shown that conjugation of diglycine of CLsu (CLsuGG) not only increases the antibiotic efficacy against <i>Escherichia coli</i> but also reduces adverse drug effects against bone marrow stromal cells. Here, we report the synthesis of structural analogues of CLsuGG and their activities against <i>E. coli</i>. These analogues reveal several trends: (i) except the water-insoluble analogues, the attachment of peptides to CLsu enhances the efficacy of the prodrugs; (ii) negative charges, high steric hindrance in the side chains, or a rigid diester decreases the activities of prodrugs in comparison to CLsuGG; (iii) dipeptides apparently increase the efficacy of the prodrugs most effectively; and so forth. This work suggests that conjugating peptides to CLsu effectively modulates the properties of prodrugs. The structure–activity relationship of these new conjugates may provide useful insights for expanding the pool of antibiotics.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_10653" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_10653" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Resulting from a large amount of antibiotics used for human and animal treatment, multidrug resistance (MDR) in bacteria remains a serious threat in public health.<a onclick="showRef(event, 'ref1 ref2 ref3 ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3 ref4 ref5">(1−5)</a> Drugs recently developed to thwart emerging antibiotic resistances, such as linezolid and the latest β-lactams, to vancomycin, have already lost effectiveness against some bacterial strains.<a onclick="showRef(event, 'ref6 ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref6 ref7 ref8">(6−8)</a> An even more serious threat may be the emergence of MDR Gram-negative bacteria that are resistant to essentially all of the available agents.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> Even more discouraging is that the development of alternatives to the existing strategies for killing pathogenic bacteria has slowed dramatically over the past decades, which fails to keep pace with the outbreak of resistance. Moreover, newer, successfully developed alternatives are strictly reserved to treat only the most serious infections. These factors make the antibiotic supply in the clinical pipeline remain limited.<a onclick="showRef(event, 'ref4 ref10'); return false;" href="javascript:void(0);" class="ref ref4 ref10">(4,10)</a> Thus, there is an urgent need for developing new antimicrobial approaches against MDR bacterial pathogens.</div><div class="NLM_p">Different strategies are being used to discover and develop novel drugs to fight against bacteria, which mainly include (1) drug derivatives, which enhance the efficacy and safety of the existing antibacterial agents via the modification of the drugs<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> or increase the specificity (e.g., efflux pump inhibitors);<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> (2) discovery of new antibacterial agents, which involves the development of new tools to discover genomic or target-based antibiotics via previously unexplored mechanisms,<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> and a classical or whole-cell antibacterial assay to find antibiotics produced by microorganisms of different sources;<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> (3) bacteriophages or enzybiotics, which utilize bacteriophages or phage-lytic enzymes;<a onclick="showRef(event, 'ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref14 ref15">(14,15)</a> and (4) ecology/evolutionary biology approaches, which target the ecology and evolution of antibiotic resistance.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> In comparison to developing a new drug that requires years of extensive testing, modifying the existing antibiotics to achieve higher efficacy is safer, faster, and of low cost. Therefore, development of feasible drug derivatives has become an active research focus.</div><div class="NLM_p">Recently, the development of antibiotic derivatives has made significant progress. For example, Boger et al. reported that modifying the binding pocket of vancomycin, introducing a quaternary ammonium salt to the peripheral C-terminal, and linking (4-chlorobiphenyl)methyl to the vancomycin disaccharide provide dual target binding and three independent action mechanisms, which directly overcome the molecular basis of vancomycin resistance.<a onclick="showRef(event, 'ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref17 ref18">(17,18)</a> The vancomycin analogues provide improvements in the antibacterial potency against vancomycin-resistant enterococci and display little propensity for resistance. Kahne et al. reported the synthesis of novobiocin derivatives, which were more potent than novobiocin when being used in combination with polymyxin, the drug of last resort for treating Gram-negative infections.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> The novobiocin analogues not only decrease the dosage of polymyxin but also increase its efficacy and safety via inhibiting DNA gyrase, binding LptB, and disrupting the outer membrane. Nolan et al. developed a synthetic siderophore–antibiotic conjugate (Ent-Cipro), which affords targeted antibacterial activity against pathogenic <i>Escherichia coli</i> strains.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> Ent-Cipro provides an excellent selectivity between nonpathogenic and pathogenic <i>E. coli</i>. Ramström and Yan synthesized a series of ciprofloxacin derivatives, which exhibited enhanced antibacterial activities against both sensitive and resistant <i>E. coli</i>.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> In addition, there are reports to conjugate CL to amino acid. For example, in the early years, Khanna et al. reported that the direct conjugation of ploy(amino acids) to the reduced nitro group of CL generated conjugated antigens, which provided conventional ways for the production of antibodies specific for CL.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> Recently, Gu et al. directly attached antimicrobial peptides to CL to form conjugates (i.e., CL-UBI<sub>29-41</sub> hybrid), which are tested in mouse models.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a></div><div class="NLM_p">Our recent study shows that conjugating diglycine to a prodrug of antibiotics [i.e., chloramphenicol (CL) and ciprofloxacin] drastically accelerates intrabacterial hydrolysis of ester bonds, regenerating the antibiotics against <i>E. coli</i>. Moreover, the ester conjugate of CL and diglycine (CLsuGG, <b>1</b>) also exhibits reduced toxicity to bone marrow cells, a major side effect of CL.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> In addition, mechanistic investigation reveals that <b>1</b> shows different hydrolysis rates upon treated by the mammalian and bacterial esterases. The effects of the structures of conjugated peptides on intrabacterial hydrolysis, as well as on the antibacterial activity, however, remain unexplored.</div><div class="NLM_p">To understand the structure–activity relationship of the peptide-conjugated CL for inhibiting bacteria, we synthesized 34 structural analogues of <b>1</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>) and examined their activities against <i>E. coli</i>. Our results show that conjugating peptides to chloramphenicol succinate (CLsu) enhances the efficacy of prodrugs to varying degrees, apart from water-insoluble analogues. As shown in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>, negative charges, high steric hindrance in the side chains of peptides, or a rigid diester (i.e., cyclohexane-1,2-dicarboxylic acid) results in lower activities than that of <b>1</b>. Moreover, the investigation of conjugated peptides from single amino acid to pentapeptides indicates that dipeptides most effectively increase the efficacy of CLsu. In addition, capping the C-terminal of the peptide with a <i>N</i>-methylacetamide group scarcely lowers the activity of the prodrug in comparison to the uncapped peptide conjugate. <span class="smallcaps smallerCapital">d</span>-Amino acid residues, in general, are more favorable than <span class="smallcaps smallerCapital">l</span>-amino acid residues for increasing the activity of the prodrugs, suggesting that <span class="smallcaps smallerCapital">d</span>-peptides slightly affect the hydrolysis rate of the ester bond in the prodrugs. Besides establishing conjugation of peptides as a simple and effective way for modulating the properties of the prodrug CLsu, the structure–activity relationship of these peptide-conjugated prodrugs may provide useful insights for designing peptide conjugates<a onclick="showRef(event, 'ref25 ref26 ref27 ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref25 ref26 ref27 ref28 ref29">(25−29)</a> of other antibiotics.</div><figure id="sch1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01210/20191122/images/medium/jm9b01210_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01210/20191122/images/large/jm9b01210_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Illustration of Peptide-Conjugated Antibiotics for Intrabacterial Activation and the Trends of Efficiency Associated with the Structures of the Conjugates (LPS: Lipopolysaccharide; OM: Outer Membrane; and IM: Inner Membrane)</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01210/20191122/images/large/jm9b01210_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01210&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Structures of CL, CLsu, and Peptide-Conjugated Prodrugs</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01210/20191122/images/medium/jm9b01210_0006.gif" alt="" id="GRAPHIC-d7e351-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01210/20191122/images/medium/jm9b01210_0007.gif" alt="" id="gr6" /></img><div></div></div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i5">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_22317" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_22317" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Molecular Design</h3><div class="NLM_p last">We designed the peptide-conjugated CL in a straightforward manner for easy production and convenient optimization. As shown in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, the conjugates have a general formula of antibiotic—succinate—peptide (or amino acid). The peptides fall into three main categories: a variety of neutral peptides (<b>1a–1q</b>), from di- to pentapeptides, for studying the influence of numbers of same amino acid residues or the side chains of the amino acid residues on the activity; charged peptides (e.g., arginine, lysine, glutamic acid, or aspartic acid (<b>2a–2j</b>)) for investigating the effect of charges; and naphthyl-containing peptides (<b>3a–3c</b>) for understanding the roles of strong hydrophobicity. Besides peptide conjugates, there are conjugates bearing a single amino acid (<b>4a</b>), or only capping the C-terminal of CLsu using a methyl (<b>4b</b>), sulfate (<b>4c</b>), or phosphate (<b>4d</b>) group. Moreover, considering the stereochemistry of the peptides as a determining factor for biological activities, we generated <span class="smallcaps smallerCapital">d</span>-peptide conjugates in the first two categories (e.g., <b>1g</b>, <b>1i</b>, <b>2b</b>, and <b>2h</b>).</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Synthesis</h3><div class="NLM_p last">As shown in <a href="/doi/suppl/10.1021/acs.jmedchem.9b01210/suppl_file/jm9b01210_si_001.pdf" class="ext-link">Scheme S1</a>, the commercially available CLsu sodium, after acidification, is suitable for solid-phase peptide synthesis (SPPS) to produce all peptide-conjugated analogues of <b>1</b>. Briefly, being attached to the 2-chlorotrityl chloride resin, the fluorenylmethyloxycarbonyl (Fmoc)-protected amino acid, after releasing the Fmoc group, reacts with the next amino acid or CLsu for expanding the chain of the peptide or for introducing CL. Then, trifluoroacetic acid cleaves the products from the resin. Dropwise addition of bromotrimethylsilane to the methanol solution of CLsu generates <b>4b</b>. Solution-phase amidation of CLsu with taurine and phosphorylethanolamine results in <b>4c</b> and <b>4d</b>, respectively.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Antibacterial Activity</h3><div class="NLM_p">After obtaining the pure conjugates, we tested these analogues for their antibacterial activity against a wild-type <i>E. coli</i> strain (K-12). As shown in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, the first category, relatively hydrophilic neutral peptides conjugated to CLsu (<b>1a–1q</b>), exhibits higher inhibitory activity than other categories of the conjugates, with minimum inhibitory concentrations (MICs) ranging from 20 to 200 μM, which is comparable to that of CL but much lower than that of CLsu (standard compounds). Our previous work demonstrates that diglycine-conjugating CLsu apparently increases the efficacy of the prodrugs most effectively.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> The replacement of C-terminal glycine on <b>1</b> by leucine, alanine, or serine generates <b>1a</b>, <b>1b</b>, and <b>1c</b>, respectively, which all exhibit an MIC of 20 μM. Switching the location of glycine and serine in <b>1c</b> produces <b>1d</b>, which exhibits an MIC (40 μM) twice that of <b>1c</b>. These results suggest that directly linking the carboxyl group of CLsu by a glycine residue is important for the high activity of the prodrugs. Capping the carboxylic acid of <b>1</b> by <i>N</i>-methylacetamide forms <b>1e</b>, which exhibits an MIC of 20 μM. The activity of <b>1e</b> is comparable to that of <b>1</b>, indicating that capping the C-terminal of <b>1</b> by <i>N</i>-methylacetamide has less effect on the hydrolysis of prodrugs.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. MIC of Peptide-Conjugated CLsu against <i>E. coli</i> (K-12)</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">#</th><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center">MIC (μM)</th><th class="colsep0 rowsep0" align="center">#</th><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center">MIC (μM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">CL</td><td class="colsep0 rowsep0" align="left">20</td><td class="colsep0 rowsep0" align="left"><b>1q</b></td><td class="colsep0 rowsep0" align="left">CLsu-GGGGG</td><td class="colsep0 rowsep0" align="left">40</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">CLsu</td><td class="colsep0 rowsep0" align="left">>200</td><td class="colsep0 rowsep0" align="left"><b>2a</b></td><td class="colsep0 rowsep0" align="left">CLsu-rr</td><td class="colsep0 rowsep0" align="left">200</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1a</b></td><td class="colsep0 rowsep0" align="left">CLsu-Gl</td><td class="colsep0 rowsep0" align="left">20</td><td class="colsep0 rowsep0" align="left"><b>2b</b></td><td class="colsep0 rowsep0" align="left">CLsu-kk</td><td class="colsep0 rowsep0" align="left">200</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1b</b></td><td class="colsep0 rowsep0" align="left">CLsu-Ga</td><td class="colsep0 rowsep0" align="left">20</td><td class="colsep0 rowsep0" align="left"><b>2c</b></td><td class="colsep0 rowsep0" align="left">CLsu-KK</td><td class="colsep0 rowsep0" align="left">200</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1c</b></td><td class="colsep0 rowsep0" align="left">CLsu-Gs</td><td class="colsep0 rowsep0" align="left">20</td><td class="colsep0 rowsep0" align="left"><b>2d</b></td><td class="colsep0 rowsep0" align="left">CLsu-ffrr</td><td class="colsep0 rowsep0" align="left">200</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1d</b></td><td class="colsep0 rowsep0" align="left">CLsu-sG</td><td class="colsep0 rowsep0" align="left">40</td><td class="colsep0 rowsep0" align="left"><b>2e</b></td><td class="colsep0 rowsep0" align="left">CLsu-ffkk</td><td class="colsep0 rowsep0" align="left">200</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1e</b></td><td class="colsep0 rowsep0" align="left">CLsu-GGNHMe</td><td class="colsep0 rowsep0" align="left">20</td><td class="colsep0 rowsep0" align="left"><b>2f</b></td><td class="colsep0 rowsep0" align="left">CLsu-GK</td><td class="colsep0 rowsep0" align="left">20</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1f</b></td><td class="colsep0 rowsep0" align="left">CLsu-GGGG</td><td class="colsep0 rowsep0" align="left">20</td><td class="colsep0 rowsep0" align="left"><b>2g</b></td><td class="colsep0 rowsep0" align="left">CLsu-ee</td><td class="colsep0 rowsep0" align="left">>200</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1g</b></td><td class="colsep0 rowsep0" align="left">CLsu-aa</td><td class="colsep0 rowsep0" align="left">80</td><td class="colsep0 rowsep0" align="left"><b>2h</b></td><td class="colsep0 rowsep0" align="left">CLsu-dd</td><td class="colsep0 rowsep0" align="left">>200</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1h</b></td><td class="colsep0 rowsep0" align="left">CLsu-AA</td><td class="colsep0 rowsep0" align="left">100</td><td class="colsep0 rowsep0" align="left"><b>2i</b></td><td class="colsep0 rowsep0" align="left">CLsu-DD</td><td class="colsep0 rowsep0" align="left">200</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1i</b></td><td class="colsep0 rowsep0" align="left">CLsu-aaa</td><td class="colsep0 rowsep0" align="left">80</td><td class="colsep0 rowsep0" align="left"><b>2j</b></td><td class="colsep0 rowsep0" align="left">CLsu-GD</td><td class="colsep0 rowsep0" align="left">20</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1j</b></td><td class="colsep0 rowsep0" align="left">CLsu-AAA</td><td class="colsep0 rowsep0" align="left">100</td><td class="colsep0 rowsep0" align="left"><b>3a</b></td><td class="colsep0 rowsep0" align="left">CLsu-ff</td><td class="colsep0 rowsep0" align="left">100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1k</b></td><td class="colsep0 rowsep0" align="left">CLsu-GGf</td><td class="colsep0 rowsep0" align="left">20</td><td class="colsep0 rowsep0" align="left"><b>3b</b></td><td class="colsep0 rowsep0" align="left">CLsu-ff(2-NaI)</td><td class="colsep0 rowsep0" align="left">>200</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1l</b></td><td class="colsep0 rowsep0" align="left">CLsu-GGF</td><td class="colsep0 rowsep0" align="left">20</td><td class="colsep0 rowsep0" align="left"><b>3c</b></td><td class="colsep0 rowsep0" align="left">CLsu-ff(2-NaI)GG</td><td class="colsep0 rowsep0" align="left">>200</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1m</b></td><td class="colsep0 rowsep0" align="left">CLsu-GFG</td><td class="colsep0 rowsep0" align="left">20</td><td class="colsep0 rowsep0" align="left"><b>4a</b></td><td class="colsep0 rowsep0" align="left">CLsu-a</td><td class="colsep0 rowsep0" align="left">>200</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1n</b></td><td class="colsep0 rowsep0" align="left">CLsu-FGG</td><td class="colsep0 rowsep0" align="left">80</td><td class="colsep0 rowsep0" align="left"><b>4b</b></td><td class="colsep0 rowsep0" align="left">CLsu-OMe</td><td class="colsep0 rowsep0" align="left">>200</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1o</b></td><td class="colsep0 rowsep0" align="left">CLsu-GGff</td><td class="colsep0 rowsep0" align="left">20</td><td class="colsep0 rowsep0" align="left"><b>4c</b></td><td class="colsep0 rowsep0" align="left">CLsu-Tau</td><td class="colsep0 rowsep0" align="left">40</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1p</b></td><td class="colsep0 rowsep0" align="left">CLsu-GGFF</td><td class="colsep0 rowsep0" align="left">20</td><td class="colsep0 rowsep0" align="left"><b>4d</b></td><td class="colsep0 rowsep0" align="left">CLsu-ep</td><td class="colsep0 rowsep0" align="left">100</td></tr></tbody></table></div></div><div class="NLM_p">Di-<span class="smallcaps smallerCapital">d</span>-alanine and di-<span class="smallcaps smallerCapital">l</span>-alanine replace the diglycine in <b>1</b> to generate <b>1g</b> and <b>1h</b>, which exhibit MICs of 80 and 100 μM, respectively. Being considerably higher than that of <b>1</b> but close to each other, the MICs of <b>1g</b> and <b>1h</b> suggest that the side chains, on the amino acid residues of the peptides, likely affect the hydrolysis of prodrugs by the esterases. <b>1i</b> and <b>1j</b>, containing three alanine residues and being enantiomers, exhibit MICs of 80 and 100 μM, respectively. These results indicate that <span class="smallcaps smallerCapital">d</span>-alanine slightly enhances the activity of the prodrugs and agree with our previous observation that dipeptide (i.e., diglycine) is optimal for enhancing the antibacterial efficacy of the conjugates.</div><div class="NLM_p">Inserting one phenylalanine in <b>1</b> at different positions produces <b>1k</b>, <b>1l</b>, <b>1m</b>, and <b>1n</b>, which exhibit MICs of 20, 20, 20, and 80 μM, respectively. While the activity of <b>1l</b> is comparable to that of <b>1</b>, the activity of <b>1n</b> is 10 times lower than that of <b>1</b>. This result indicates that phenylalanine should be away from the ester linkage to favor the hydrolysis of the prodrug. The activities of <b>1k</b> and <b>1m</b> are comparable to that of <b>1</b>, indicating that the position of phenylalanine slightly affects the activities of the prodrugs when glycine is the first residue connected to succinate. The attachment of two <span class="smallcaps smallerCapital">d</span>- or <span class="smallcaps smallerCapital">l</span>-diphenylalanines in the carboxylic acid end of <b>1</b> generates <b>1o</b> and <b>1p</b>, both of which exhibit an MIC of 20 μM, which is similar to that of <b>1k</b> and <b>1l</b>, indicating that the number of phenylalanines may slightly affect the hydrolysis. While the conjugate (<b>1f</b>) containing four glycine residues shows a comparable activity (MIC = 20 μM) to that of <b>1</b>, the conjugate (<b>1q</b>) containing five glycine residues and exhibiting an MIC of 40 μM is considerably less active than <b>1</b>. This result again agrees with our earlier conclusions that diglycine is optimal for enhancing the antibacterial efficacy of CL.</div><div class="NLM_p">Considering the electrostatic interactions with a negatively charged bacterial membrane,<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> we designed the second category of conjugates (<b>2a–2j</b>) by attaching charged peptides to CLsu. The incorporation of two <span class="smallcaps smallerCapital">d</span>-arginine and two <span class="smallcaps smallerCapital">d</span>-lysine residues generates <b>2a</b> and <b>2b</b>, with MICs of 200 and 200 μM, respectively, which are similar to <b>1g</b> (80 μM) and <b>1h</b> (100 μM) in the first category. With two <span class="smallcaps smallerCapital">l</span>-lysine residues in the conjugates, the MIC of <b>2c</b> is 200 μM. This result indicates that the <span class="smallcaps smallerCapital">d</span>- and <span class="smallcaps smallerCapital">l</span>-lysine residues would have similar influences on the activities of the prodrugs. Moreover, the insertion of two <span class="smallcaps smallerCapital">d</span>-phenylalanine residues in <b>2a</b> and <b>2b</b> generates <b>2d</b> and <b>2e</b>, respectively, which still shows comparable activity with that of <b>2a</b> and <b>2b</b>, both with an MIC of 200 μM. Using one glycine to replace one <span class="smallcaps smallerCapital">l</span>-lysine in <b>2c</b> leads to <b>2f</b>, which shows much higher inhibitory activity (20 μM) than <b>2c</b> (200 μM).</div><div class="NLM_p">Besides introducing the above positively charged peptides, we also attached various negatively charged peptides in the conjugates and compared their antibacterial efficiency with the positively charged ones. Containing glutamic acid or aspartic acid residues, <b>2g</b>, <b>2h</b>, and <b>2i</b> all exhibit high MICs (<b>2g</b>: >200 μM; <b>2h</b>: >200 μM; and <b>2i</b>: 200 μM), indicating that multiple negative charges decrease the activities of the prodrugs. This notion is supported by the lower MIC (20 μM) of <b>2j</b> than that of <b>2i</b>. These results are consistent with the idea that the negatively charged bacterial membrane may electrostatically repel the prodrugs bearing multiple negative charges.</div><div class="NLM_p">Introducing either hydrophilic neutral peptides or charged peptides to CLsu results in water-soluble conjugates. To understand the roles of solubility, we created conjugates attached with hydrophobic peptides or steric hindrance in side chains to CLsu (<b>3a–3c</b>). The direct attachment of diphenylalanine in the carboxylic acid end of CLsu generates <b>3a</b>, which shows low inhibitory activity against <i>E. coli</i> with an MIC of 100 μM. Furthermore, attaching a hydrophobic amino acid (i.e., naphthylalanine) produces <b>3b</b>, which is more hydrophobic than <b>3a</b>, and hardly inhibits <i>E. coli</i> growth, and the concentration is higher than 200 μM. In addition, the insertion of diphenylalanine and naphthylalanine in <b>1</b> forms <b>3c</b>, which also results in an MIC higher than 200 μM. These observations imply that the incorporation of hydrophobic peptides significantly impairs the efficiency of the prodrugs, if the solubility of the conjugates is low. This observation is reasonable because low solubility increases the difficulty to enter bacterial membranes and decreases the rate of hydrolysis of the ester bond by the esterases.</div><div class="NLM_p last">Besides peptide conjugates, there are conjugates bearing a single alanine (<b>4a</b>), or only capping the C-terminal of CLsu with a methyl (<b>4b</b>), sulfate (<b>4c</b>), or phosphate group (<b>4d</b>). <b>4a</b> exhibits a high MIC (>200 μM), implying that the direct attachment of one amino acid residue to CLsu has less influence on the hydrolysis of the prodrugs. Blocking the carboxyl group with methyl group generates water-insoluble <b>4b</b> with two ester bonds. The high MIC of <b>4b</b> (>200 μM) suggests that multiple ester bonds and/or low solubility likely reduces the rate of hydrolysis by esterases. <b>4c</b> and <b>4d</b> exhibit MICs of 40 and 100 μM, respectively, which are lower than the prodrug [i.e., CLsu (>200 μM)] but still higher than <b>1</b>. The result is not only consistent with the fact that conjugating taurine<a onclick="showRef(event, 'ref32 ref33'); return false;" href="javascript:void(0);" class="ref ref32 ref33">(32,33)</a> plays a role in boosting cellular uptake but also supports the conjugation of peptides to prodrugs as an effective way for enhancing the efficacy of the prodrugs. Although the antibacterial activity of these peptide-conjugated CLs, such as CL itself, is not as potent as that of some other broad-spectrum antibiotics (e.g., ciprofloxacin, trimethoprim, and telithromycin with MICs of 0.01, 1, and 0.6 μg/mL against <i>E. coli</i> K-12, respectively),<a onclick="showRef(event, 'ref34 ref35 ref36'); return false;" href="javascript:void(0);" class="ref ref34 ref35 ref36">(34−36)</a> they not only show improved potency in comparison with CLsu, the clinically failed prodrug of CL,<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> but also exhibit better water solubility than that of CL. This discovery may provide potential clinical applications for intravenous injection of the new prodrugs of CL.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Hydrolysis Rate</h3><div class="NLM_p">To further understand the effects of the structures of conjugated peptides on hydrolysis, as well as on the antibacterial activity, first, we compared the antibacterial activity of <b>1c</b>, <b>1d</b>, <b>2b</b>, and <b>2c</b> before and after the treatment of a commercially available mammalian esterase (porcine liver esterase, PLE) (1 U/mL) and found that the addition of exogenous PLE hardly changes the inhibitory efficacy of <b>1c</b>, <b>1d</b>, <b>2b</b>, and <b>2c</b> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A). Then, we measured the hydrolysis rate of the ester bond of <b>1c</b>, <b>1d</b>, <b>2b</b>, and <b>2c</b> by <i>E. coli</i> or HepG2 (a hepatocyte cell which overexpresses mammalian esterases<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a>) lysates, after normalizing their activities. As shown in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B, catalyzed by the lysate of either <i>E. coli</i> or HepG2, all peptide-conjugated prodrugs are able to undergo complete hydrolysis for regenerating the active drug, which further supports our conclusion that the attachment of peptides to CLsu enhances the efficacy of the prodrugs. In detail, under the condition of <i>E. coli</i> lysates, the different conjugated peptides (with slightly different apparent first-order rate constants of <b>1c</b>: 3.763 h<sup>–1</sup>, <b>1d</b>: 1.267 h<sup>–1</sup>, <b>2b</b>: 0.317 h<sup>–1</sup>, and <b>2c</b>: 0.310 h<sup>–1</sup>) lead to different inhibitory activities of prodrugs with the tendency that faster hydrolysis results in higher efficacy (<b>1c</b>: 20 μM, <b>1d</b>: 40 μM, <b>2b</b>: 200 μM and <b>2c</b>: 200 μM).</div><figure id="fig1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01210/20191122/images/medium/jm9b01210_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01210/20191122/images/large/jm9b01210_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. (A) Antibacterial activity of <b>1c</b>, <b>1d</b>, <b>2b</b>, and <b>2c</b> before and after the treatment of PLE; [PLE] = 1 U/mL. (B) Hydrolysis curves of <b>1c</b>, <b>1d</b>, <b>2b</b>, and <b>2c</b> with the addition of the lysate of HepG2 cells or the lysate of <i>E. coli</i> (K-12); [<b>1c</b>] = [<b>1d</b>] = [<b>2b</b>] = [<b>2c</b>] = 200 μM, [HepG2 lysate] = [<i>E. coli</i> lysate] = 0.1 U/mL. (C) Static light-scattering signals of the solution of <b>1c</b>, <b>1d</b>, <b>2b</b>, and <b>2c</b> before and after the addition of PLE; [<b>1c</b>] = [<b>1d</b>] = [<b>2b</b>] = [<b>2c</b>] = 200 μM, [PLE] = 1 U/mL, <i>t</i> = 24 h. (D) Transmission electron microscopy (TEM) images of <b>1c</b>, <b>1d</b>, <b>2b</b>, and <b>2c</b> before and after the addition of PLE; [<b>1c</b>] = [<b>1d</b>] = [<b>2b</b>] = [<b>2c</b>] = 500 μM, [PLE] = 1 U/mL, <i>t</i> = 24 h, scale bar = 500 nm.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01210/20191122/images/large/jm9b01210_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01210&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The intensities of the static light-scattering signals increase drastically at different magnitudes after the addition of PLE to the solutions of <b>1c</b>, <b>1d</b>, <b>2b</b>, and <b>2c</b> for 24 h (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C), agreeing with the TEM images, which show an increased amount of nanoparticles after the addition of PLE (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>D). Besides confirming that the precursors become active antibiotics (i.e., CL) after the hydrolysis catalyzed by esterases, these results indicate that soluble precursors are able to undergo intrabacterial hydrolysis to form CL. Because of the poor solubility of CL, such a conversion enhances the retention of CL inside <i>E. coli</i>.</div><div class="NLM_p last">Furthermore, we examined the serum stability of CLsu, <b>1c</b>, <b>1d</b>, <b>2b</b>, <b>2c</b>, and several other most effective conjugates (e.g., <b>1k</b>, <b>1o</b>, and <b>2f</b>). We incubated CLsu and these conjugates in human serum (from human male AB plasma) at a concentration of 200 μM at 37 °C for 2 and 24 h, respectively (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01210/suppl_file/jm9b01210_si_001.pdf" class="ext-link">Table S2</a>). The liquid chromatography–mass spectrometry (LC–MS) results show that all the conjugates hydrolyzed to form CL after 24 h while the conversion of CLsu only achieved less than 15% after 24 h. Furthermore, <b>1c</b>, <b>1k</b>, and <b>2f</b> were able to hydrolyze completely within 2 h. The discovery is consistent with the fact that the faster hydrolysis results in higher efficacy and further confirms that the attachment of peptides to CLsu would enhance the efficacy of the prodrugs. In addition, the results also show that the different conjugated peptides may lead to a slightly different stability in serum environment. Therefore, these conjugates may achieve adequate serum concentrations in a short time if being administrated intravenously, which may address the shortcoming of CLsu and its clinical failure. However, the balance between serum stability outside of bacteria and targeting hydrolysis inside the bacteria remains optimized, which may be achieved by further investigating the peptide–antibiotic conjugates.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Uptake Mechanism</h3><div class="NLM_p">Our previous work demonstrated that the diglycine-conjugated CLsu enters the bacteria via multiple paths, including <i>ydgR</i> (i.e., inner membrane oligopeptide transporters involving in the uptake of di- and tripeptides),<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a><i>fepA</i> (i.e., a siderophore transporter),<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> and passive diffusion. We chose several representative peptides conjugating CLsu (e.g., <b>1c</b>, <b>1d</b>, <b>2b</b>, and <b>2c</b>) to study the effects of conjugated peptides on the uptake of the prodrugs. In addition to testing the activities of these conjugates against <i>E. coli</i> mutants that have <i>ydgR</i> or <i>fepA</i> deleted, we also measured the activities of these conjugates against <i>E. coli</i> mutants that have <i>ompF</i> (i.e., outer membrane porin transporters mediating the nonspecific diffusion of small solutes)<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> or <i>acrA</i> (i.e., the periplasmic lipoprotein component of multidrug efflux pumps)<a onclick="showRef(event, 'ref42 ref43'); return false;" href="javascript:void(0);" class="ref ref42 ref43">(42,43)</a> deleted. As shown in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>, we compared the activities of <b>1c</b>, <b>1d</b>, <b>2b</b>, and <b>2c</b> against <i>ydgR</i>, <i>fepA</i>, <i>ompF</i>, and <i>acrA</i> mutants and wild-type <i>E. coli</i>. When the mutants are treated by <b>1c</b>, <b>1d</b>, <b>2b</b>, and <b>2c</b>, the deletion of <i>ydgR</i>, <i>fepA</i>, or <i>ompF</i> reduces the viability of bacteria at different magnitudes and hardly rescues the bacteria, suggesting that <i>ydgR</i>, <i>fepA</i>, and <i>ompF</i> are unlikely to be the major contributors for the precursors entering the bacteria, and mutation of one transporter may not lead to drug resistance of these precursors. However, the deletion of <i>acrA</i> significantly reduces the viabilities of the mutant treated by the conjugates (<b>1c</b>: 30%, <b>1d</b>: 40%, <b>2b</b>: 20%, and <b>2c</b>: 50%), indicating that <i>acrA</i> likely plays a key role in pumping out the precursors, thus agreeing with the suggestion that <i>acrA</i> contributes to bacterial resistance.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a></div><figure id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01210/20191122/images/medium/jm9b01210_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01210/20191122/images/large/jm9b01210_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Antibacterial activity of <b>1c</b>, <b>1d</b>, <b>2b</b>, and <b>2c</b> against <i>ydgR</i>, <i>fepA</i>, and <i>ompF</i> transporter deletion mutants or <i>acrA</i> efflux pump deletion mutant of <i>E. coli</i>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01210/20191122/images/large/jm9b01210_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01210&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Intrabacterial Hydrolysis</h3><div class="NLM_p">To examine the role of different bacterial esterases on the hydrolysis of different peptides conjugated to CLsu, we measured the activities of <b>1c</b>, <b>1d</b>, <b>2b</b>, and <b>2c</b> against eight <i>E. coli</i> mutants that have one of the bacterial esterase genes (i.e., <i>bioH</i>, <i>yjfP</i>, <i>frsA</i>, <i>ybfF</i>, <i>yfbB</i>, <i>ypfH</i>, <i>yeiG</i>, or <i>yqiA</i>)<a onclick="showRef(event, 'ref45 ref46 ref47'); return false;" href="javascript:void(0);" class="ref ref45 ref46 ref47">(45−47)</a> deleted. As shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>, the deletion of the cytoplasmic esterase of <i>E. coli</i> reduces the antibacterial activities of <b>1c</b>, <b>1d</b>, <b>2b</b>, and <b>2c</b> moderately, at about 20, 10, 20, and 20%, respectively. These studies confirm that various esterases in bacterial cytoplasm convert precursors to the active antibiotic agent (i.e., CL) after the precursors enter <i>E. coli</i>.</div><figure id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01210/20191122/images/medium/jm9b01210_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01210/20191122/images/large/jm9b01210_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Antibacterial activity of <b>1c</b>, <b>1d</b>, <b>2b</b>, and <b>2c</b> against esterase (<i>bioH</i>, <i>yjfP</i>, <i>frsA</i>, <i>ybfF</i>, <i>yfbB</i>, <i>ypfH</i>, <i>yeiG</i>, or <i>yqiA</i>) deletion mutants of <i>E. coli</i>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01210/20191122/images/large/jm9b01210_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01210&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Cytotoxicity Study</h3><div class="NLM_p">From previous in vitro cytotoxicity studies against HS-5 (a bone marrow stromal cell line), HepG2 (a hepatocyte cell line), and HEK293 (a kidney cell line) cells, we found that diglycine conjugating CLsu would likely reduce the major adverse effect of CL (i.e., bone marrow suppression), whereas it hardly alters the cytotoxicity of CL against HepG2 and HEK293 cell lines. To assess the major side effects of these conjugates, we compared the cytotoxicities of the most effective conjugates, which exhibit an MIC of 20 μM (i.e., <b>1a</b>, <b>1b</b>, <b>1c</b>, <b>1e</b>, <b>1f</b>, <b>1k</b>, <b>1l</b>, <b>1m</b>, <b>1o</b>, <b>1p</b>, <b>2f</b>, and <b>2j</b>) with the control compounds (i.e., ciprofloxacin, trimethoprim, and telithromycin) against HS-5 and HEK293 cells. As shown in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A,B, like the prodrug CLsu, most of the conjugates show lower cytotoxicity than CL against HS-5 cells, further confirming that peptide-conjugated CLsu would likely reduce the major adverse effect of CL. Meanwhile, ciprofloxacin (Cipro), trimethoprim (TMP), and telithromycin (TEL) also show relatively low cytotoxicity against HS-5 cells at a concentration of around MICs. In addition, all conjugates show about the same cytotoxicity as CL against HEK293 cells, further confirming that the peptide-conjugated CLsu scarcely alters the cytotoxicity of CL against HEK293. Furthermore, we increased the concentration of these conjugates to 10-fold and 100-fold of MICs (i.e., 200 μM and 2 mM) to examine their cytotoxicities against HS-5 cells. As shown in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C,D, all of them still show relatively lower cytotoxicity than CL against HS-5 cells, implying the possibility of clinical administration.</div><figure id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01210/20191122/images/medium/jm9b01210_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01210/20191122/images/large/jm9b01210_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Cell viability of (A) HS-5 cells and (B) HEK293 cells incubated with <b>1a</b>, <b>1b</b>, <b>1c</b>, <b>1e</b>, <b>1f</b>, <b>1k</b>, <b>1l</b>, <b>1m</b>, <b>1o</b>, <b>1p</b>, <b>2f</b>, and <b>2j</b> for 24 h. [CL] = [CLsu] = [<b>1a</b>] = [<b>1b</b>] = [<b>1c</b>] = [<b>1e</b>] = [<b>1f</b>] = [<b>1k</b>] = [<b>1l</b>] = [<b>1m</b>] = [<b>1o</b>] = [<b>1p</b>] = [<b>2f</b>] = [<b>2j</b>] = 20 μM; [Cipro] = [TMP] = [TEL] = 0.5 μM. (C) Cell viability of HS-5 cells incubated with CL, CLsu, <b>1a</b>, <b>1b</b>, <b>1c</b>, <b>1e</b>, <b>1f</b>, <b>1k</b>, <b>1l</b>, <b>1m</b>, <b>1o</b>, <b>1p</b>, <b>2f</b>, and <b>2j</b> for 24 h. [CL] = [CLsu] = [<b>1a</b>] = [<b>1b</b>] = [<b>1c</b>] = [<b>1e</b>] = [<b>1f</b>] = [<b>1k</b>] = [<b>1l</b>] = [<b>1m</b>] = [<b>1o</b>] = [<b>1p</b>] = [<b>2f</b>] = [<b>2j</b>] = 200 μM; [Cipro] = [TMP] = [TEL] = 5 μM. (D) Cell viability of HS-5 cells incubated with CL, CLsu, <b>1a</b>, <b>1b</b>, <b>1c</b>, <b>1e</b>, <b>1f</b>, <b>1k</b>, <b>1l</b>, <b>1m</b>, <b>1o</b>, <b>1p</b>, <b>2f</b>, and <b>2j</b> for 24 h. [CL] = [CLsu] = [<b>1a</b>] = [<b>1b</b>] = [<b>1c</b>] = [<b>1e</b>] = [<b>1f</b>] = [<b>1k</b>] = [<b>1l</b>] = [<b>1m</b>] = [<b>1o</b>] = [<b>1p</b>] = [<b>2f</b>] = [<b>2j</b>] = 2000 μM; [Cipro] = [TMP] = [TEL] = 50 μM.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01210/20191122/images/large/jm9b01210_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01210&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i14">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_12584" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_12584" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">In summary, we have developed a series of peptide-conjugated prodrugs of CL via an ester bond, which modulates the properties of the prodrugs. The antibacterial activity study of these conjugates against <i>E. coli</i> demonstrated the structure–activity relationship of these peptide-conjugated CLs, which may provide a powerful way for developing potential antibiotic agents that treat bacterial infections. In addition, further hydrolysis investigation of the conjugates suggests that rapid intrabacterial hydrolysis is a useful way to increase the retention of active drugs inside bacteria and enhance the efficacy of prodrugs, which deserves further exploration. Additionally, our investigation showed that <i>acrA</i> efflux pumps lead to the bacterial resistance of these peptide-conjugated prodrugs, and various cytoplasmic esterases contributed to form active antibiotic agents after precursors entered <i>E. coli</i>, which indicates the future direction of molecular design based on the functions of bacterial esterases<a onclick="showRef(event, 'ref48 ref49'); return false;" href="javascript:void(0);" class="ref ref48 ref49">(48,49)</a> or bacterial efflux pumps.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> This work may also provide a potential strategy to design functional peptides<a onclick="showRef(event, 'ref51 ref52 ref53'); return false;" href="javascript:void(0);" class="ref ref51 ref52 ref53">(51−53)</a> or peptidomimetic<a onclick="showRef(event, 'ref54 ref55 ref56 ref57 ref58'); return false;" href="javascript:void(0);" class="ref ref54 ref55 ref56 ref57 ref58">(54−58)</a> modified prodrugs. For example, conjugating tissue-specific targeting peptides to antibiotic agents and regenerating the active antibiotics in the target tissues via rapid hydrolysis might be an effective approach to achieve targeted treatment of bacterial infections with lower adverse effects toward others tissues.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_42618" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_42618" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> General Information</h3><div class="NLM_p last">2-Chlorotrityl chloride resin (1.0–1.2 mmol/g), <i>O</i>-(benzotriazol-1-yl)-<i>N</i>,<i>N</i>,<i>N</i>′,<i>N</i>′-tetramethyluronium hexafluorophosphate (HBTU), and Fmoc-protected amino acids were purchased from GL Biochem (Shanghai, China). Other chemical reagents and solvents were purchased from Fisher Scientific. Dulbecco’s modified Eagle medium, fetal bovine serum, and Gibco penicillin–streptomycin were purchased from Life Technologies. All precursors were purified with an Agilent 1100 Series Liquid Chromatograph system, equipped with an XTerra C18 RP column and a Variable Wavelength Detector. The LC–MS spectra were obtained with a Waters Acquity Ultra Performance LC with Waters MICROMASS detector.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Synthesis of Desired Compounds</h3><div class="NLM_p">We synthesized CLsu by acidifying the commercially available CLsu sodium. CLsu sodium (150 mg) was dissolved in distilled water (3 mL), and HCl (1 M) was added dropwise until the pH of the mixture was adjusted to 2.0. The precipitate was washed several times with distilled water and dried for further use.</div><div class="NLM_p">We used SPPS<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a> for the synthesis of all peptide-conjugated prodrugs. 2-Chlorotrityl chloride resin (500 mg, 0.5 mmol) was swelled in 10 mL of dichloromethane (DCM) for 20 min. The attachment of the first Fmoc-protected amino acid (0.5 mmol) to the resin was achieved by adding <i>N</i>,<i>N</i>-diisopropylethylamine (DIPEA) (413 μL, 2.5 mmol) to the beads in the reaction vessel, which was allowed to shake at room temperature for 1 h. After that, the reaction solution was drained, followed by washing with dimethylformamide (DMF; 10 mL × 3) and DCM (10 mL × 3). The unreacted 2-chlorotrityl chloride moieties were capped with a solution of methanol/DIPEA/DCM (v/v/v: 3/1/16) for 30 min. The beads were washed with DMF (10 mL × 3). The Fmoc group was removed by treating the beads with 20% piperidine/DMF (v/v) solution for 20 min at room temperature. The solution was drained and washed with DMF (10 mL × 3). The beads were reacted with the next Fmoc-protected amino acid or CLsu (0.5 mmol) by adding HBTU (190 mg, 0.5 mmol) and DIPEA (413 μL, 2.5 mmol) to the beads in the reaction vessel, which was allowed to shake at room temperature for 1 h, and the solution was drained. Then, the product was cleaved from resin with 10 mL of trifluoroacetic acid for 2 h. The solution was collected, and the remaining beads were washed with 5 mL of trifluoroacetic acid three times. All the solution was combined and trifluoroacetic acid was removed with the N<sub>2</sub> flow. The residue was then precipitated with diethyl ether. The crude product was purified by reverse-phase high-performance liquid chromatography (HPLC) using HPLC grade acetonitrile and water with a supplement of 0.1% trifluoroacetic acid as the eluents. The purity of the compounds was determined to be >95% by LC–MS with a Waters Acquity Ultra Performance LC (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01210/suppl_file/jm9b01210_si_001.pdf" class="ext-link">Table S1</a>). The nuclear magnetic resonance (NMR) spectra were obtained on a Varian Unity Inova 400 instrument (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01210/suppl_file/jm9b01210_si_001.pdf" class="ext-link">Figures S1–S34 and S69–S102</a>).</div><div id="sec4_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18"> Compound <b>1a</b></h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, 25 °C, ppm): δ 8.51 (d, <i>J</i> = 8.0 Hz, 1H), 8.16 (m, 3H), 8.04 (d, <i>J</i> = 8.0 Hz, 1H), 7.64 (d, <i>J</i> = 8.0 Hz, 2H), 6.44 (m, 1H), 5.03 (s, 1H), 4.22 (m, 3H), 4.08 (m, 2H), 3.73 (m, 2H), 2.44 (m, 4H), 1.56 (m, 3H), 0.84 (m, 6H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>, 25 °C, ppm): δ 174.34 (<i>C</i>O carboxyl), 172.63 (<i>C</i>O carboxyl), 171.47 (<i>C</i>ONH amide), 169.26 (<i>C</i>ONH amide), 164.02 (<i>C</i>ONH CL), 150.65 (<i>C</i>CH phenyl), 146.99 (<i>C</i>NO<sub>2</sub> phenyl), 127.87 (<i>C</i>H phenyl), 123.30 (<i>C</i>H phenyl), 69.85 (<i>C</i>H CL), 66.67 (<i>C</i>H CL), 63.38 (<i>C</i>H<sub>2</sub> CL), 53.90 (<i>C</i>H), 50.25 (<i>C</i>H<sub>2</sub>), 42.08 (<i>C</i>H<sub>2</sub>), 30.05 (<i>C</i>H<sub>2</sub>), 29.33 (<i>C</i>H<sub>2</sub>), 24.61 (<i>C</i>H<sub>2</sub>), 23.19 (<i>C</i>H), 21.73 (<i>C</i>H<sub>3</sub>); ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>23</sub>H<sub>30</sub>Cl<sub>2</sub>N<sub>4</sub>O<sub>10</sub> [M]<sup>+</sup><i>m</i>/<i>z</i>, 592.13; found [M – H]<sup>−</sup>, 591.27.</div></div><div id="sec4_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> Compound <b>1b</b></h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, 25 °C, ppm): δ 8.51 (d, <i>J</i> = 12.0 Hz, 1H), 8.16 (m, 3H), 8.10 (d, <i>J</i> = 8.0 Hz, 1H), 7.64 (d, <i>J</i> = 8.0 Hz, 2H), 6.44 (s, 1H), 5.03 (s, 1H), 4.20 (m, 3H), 4.09 (m, 2H), 3.72 (m, 2H), 2.45 (m, 4H), 1.25 (d, <i>J</i> = 8.0 Hz, 3H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>, 25 °C, ppm): δ 174.38 (<i>C</i>O carboxyl), 172.66 (<i>C</i>O carboxyl), 171.51 (<i>C</i>ONH amide), 169.04 (<i>C</i>ONH amide), 164.05 (<i>C</i>ONH CL), 150.67 (<i>C</i>CH phenyl), 147.02 (<i>C</i>NO<sub>2</sub> phenyl), 127.89 (<i>C</i>H phenyl), 123.33 (<i>C</i>H phenyl), 69.88 (<i>C</i>H CL), 66.70 (<i>C</i>H CL), 63.42 (<i>C</i>H<sub>2</sub> CL), 53.91 (<i>C</i>H, CL), 47.84 (<i>C</i>H), 42.11 (<i>C</i>H<sub>2</sub>), 30.07 (<i>C</i>H<sub>2</sub>), 29.35 (<i>C</i>H<sub>2</sub>), 17.63 (<i>C</i>H<sub>3</sub>); ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>20</sub>H<sub>24</sub>Cl<sub>2</sub>N<sub>4</sub>O<sub>10</sub> [M]<sup>+</sup><i>m</i>/<i>z</i>, 550.09; found [M – H]<sup>−</sup>, 549.28.</div></div><div id="sec4_2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> Compound <b>1c</b></h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, 25 °C, ppm): δ 12.63 (s, 1H), 8.51 (d, <i>J</i> = 8.0 Hz, 1H), 8.15 (m, 3H), 7.99 (d, <i>J</i> = 8.0 Hz, 1H), 7.63 (d, <i>J</i> = 12.0 Hz, 2H), 6.44 (s, 1H), 6.20 (s, 1H), 5.02 (s, 1H), 4.24 (m, 3H), 4.09 (m, 1H), 3.77 (m, 2H), 3.69 (m, 1H), 3.60 (m, 1H), 2.43 (m, 4H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>, 25 °C, ppm): <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>, 25 °C, ppm): δ 172.58 (<i>C</i>O carboxyl), 172.23 (<i>C</i>O carboxyl), 171.46 (<i>C</i>ONH amide), 169.34 (<i>C</i>ONH amide), 164.04 (<i>C</i>ONH CL), 150.66 (<i>C</i>CH phenyl), 147.01 (<i>C</i>NO<sub>2</sub> phenyl), 127.88 (<i>C</i>H phenyl), 123.32 (<i>C</i>H phenyl), 69.88 (<i>C</i>H CL), 66.69 (<i>C</i>H CL), 63.44 (<i>C</i>H<sub>2</sub> CL), 61.72 (<i>C</i>H<sub>2</sub>), 54.95 (<i>C</i>H), 53.92 (<i>C</i>H, CL), 42.17 (<i>C</i>H<sub>2</sub>), 30.07 (<i>C</i>H<sub>2</sub>), 29.36 (<i>C</i>H<sub>2</sub>); ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>20</sub>H<sub>24</sub>Cl<sub>2</sub>N<sub>4</sub>O<sub>11</sub> [M]<sup>+</sup><i>m</i>/<i>z</i>, 566.08; found [M – H]<sup>−</sup>, 565.27.</div></div><div id="sec4_2_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> Compound <b>1d</b></h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, 25 °C, ppm): δ 8.53 (m, 1H), 8.15 (m, 3H), 8.01 (m, 1H), 7.64 (m, 2H), 6.43 (m, 1H), 5.03 (d, <i>J</i> = 8.0 Hz, 1H), 4.22 (m, 6H), 3.73 (m, 2H), 3.57 (m, 2H), 2.48 (m, 4H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>, 25 °C, ppm): δ 172.70 (<i>C</i>O carboxyl), 171.49 (<i>C</i>O carboxyl), 171.38 (<i>C</i>ONH amide), 170.83 (<i>C</i>ONH amide), 164.04 (<i>C</i>ONH CL), 150.67 (<i>C</i>CH phenyl), 147.01 (<i>C</i>NO<sub>2</sub> phenyl), 127.89 (<i>C</i>H phenyl), 123.32 (<i>C</i>H phenyl), 69.85 (<i>C</i>H CL), 66.69 (<i>C</i>H CL), 63.38 (<i>C</i>H<sub>2</sub> CL), 62.16 (<i>C</i>H<sub>2</sub>), 55.49 (<i>C</i>H), 53.90 (<i>C</i>H, CL), 41.12 (<i>C</i>H<sub>2</sub>), 30.15 (<i>C</i>H<sub>2</sub>), 29.35 (<i>C</i>H<sub>2</sub>); ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>20</sub>H<sub>24</sub>Cl<sub>2</sub>N<sub>4</sub>O<sub>11</sub> [M]<sup>+</sup><i>m</i>/<i>z</i>, 566.08; found [M – H]<sup>−</sup>, 565.27.</div></div><div id="sec4_2_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> Compound <b>1e</b></h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, 25 °C, ppm): δ 8.51 (d, <i>J</i> = 8.0 Hz, 1H), 8.22 (m, 1H), 8.17 (d, <i>J</i> = 8.0 Hz, 2H), 8.11 (m, 1H), 7.63 (d, <i>J</i> = 8.0 Hz, 2H), 6.44 (s, 1H), 5.02 (s, 1H), 4.21 (m, 2H), 4.09 (m, 1H), 3.72 (m, 3H), 3.63 (d, <i>J</i> = 4.0 Hz, 1H), 2.57 (d, <i>J</i> = 4.0 Hz, 2H), 2.45 (d, 4H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>, 25 °C, ppm): δ 172.63 (<i>C</i>O carboxyl), 171.90 (<i>C</i>O carboxyl), 171.53 (<i>C</i>ONH amide), 169.69 (<i>C</i>ONH amide), 164.05 (<i>C</i>ONH CL), 150.66 (<i>C</i>CH phenyl), 147.02 (<i>C</i>NO<sub>2</sub> phenyl), 127.88 (<i>C</i>H phenyl), 123.33 (<i>C</i>H phenyl), 69.89 (<i>C</i>H CL), 66.70 (<i>C</i>H CL), 63.48 (<i>C</i>H<sub>2</sub> CL), 53.93 (<i>C</i>H, CL), 49.02 (<i>C</i>H<sub>2</sub>), 42.42 (<i>C</i>H<sub>2</sub>), 30.08 (<i>C</i>H<sub>2</sub>), 29.32 (<i>C</i>H<sub>2</sub>), 25.86 (<i>C</i>H<sub>3</sub>); ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>20</sub>H<sub>25</sub>Cl<sub>2</sub>N<sub>5</sub>O<sub>9</sub> [M]<sup>+</sup><i>m</i>/<i>z</i>, 549.10; found [M – H]<sup>−</sup>, 548.20.</div></div><div id="sec4_2_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> Compound <b>1f</b></h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, 25 °C, ppm): δ 8.51 (d, <i>J</i> = 8.0 Hz, 1H), 8.16 (m, 6H), 7.63 (d, <i>J</i> = 8.0 Hz, 2H), 6.44 (s, 1H), 5.02 (s, 1H), 4.20 (m, 2H), 4.09 (m, 2H), 3.74 (dd, <i>J</i> = 4.0, 8.0 Hz, 8H), 2.46 (m, 4H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>, 25 °C, ppm): δ 172.71 (<i>C</i>O carboxyl), 171.82 (<i>C</i>O carboxyl), 171.58 (<i>C</i>ONH amide), 169.90 (<i>C</i>ONH amide), 169.62 (<i>C</i>ONH amide), 164.14 (<i>C</i>ONH CL), 150.75 (<i>C</i>CH phenyl), 147.10 (<i>C</i>NO<sub>2</sub> phenyl), 127.97 (<i>C</i>H phenyl), 123.42 (<i>C</i>H phenyl), 69.98 (<i>C</i>H CL), 66.78 (<i>C</i>H CL), 63.55 (<i>C</i>H<sub>2</sub> CL), 54.02 (<i>C</i>H CL), 42.65 (<i>C</i>H<sub>2</sub>), 42.51 (<i>C</i>H<sub>2</sub>), 42.24 (<i>C</i>H<sub>2</sub>), 41.07 (<i>C</i>H<sub>2</sub>), 30.16 (<i>C</i>H<sub>2</sub>), 29.40 (<i>C</i>H<sub>2</sub>); ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>23</sub>H<sub>28</sub>Cl<sub>2</sub>N<sub>6</sub>O<sub>12</sub> [M]<sup>+</sup><i>m</i>/<i>z</i>, 650.11; found [M – H]<sup>−</sup>, 649.20.</div></div><div id="sec4_2_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> Compound <b>1g</b></h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, 25 °C, ppm): δ 8.51 (d, <i>J</i> = 8.0 Hz, 1H), 8.17 (d, <i>J</i> = 12.0 Hz, 2H), 8.13 (d, <i>J</i> = 8.0 Hz, 1H), 8.07 (d, <i>J</i> = 8.0 Hz, 1H), 7.64 (d, <i>J</i> = 8.0 Hz, 2H), 6.44 (s, 1H), 5.03 (s, 1H), 4.31 (m, 1H), 4.17 (m, 3H), 4.09 (m, 1H), 2.44 (m, 4H), 1.23 (d, <i>J</i> = 4.0 Hz, 3H), 1.19 (d, <i>J</i> = 8.0 Hz, 3H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>, 25 °C, ppm): δ 174.50 (<i>C</i>O carboxyl), 172.76 (<i>C</i>O carboxyl), 172.66 (<i>C</i>ONH amide), 170.94 (<i>C</i>ONH amide), 164.13 (<i>C</i>ONH CL), 150.79 (<i>C</i>CH phenyl), 147.11 (<i>C</i>NO<sub>2</sub> phenyl), 128.00 (<i>C</i>H phenyl), 123.43 (<i>C</i>H phenyl), 69.91 (<i>C</i>H CL), 66.79 (<i>C</i>H CL), 63.38 (<i>C</i>H<sub>2</sub> CL), 53.97 (<i>C</i>H CL), 48.29 (<i>C</i>H), 47.88 (<i>C</i>H), 30.12 (<i>C</i>H<sub>2</sub>), 29.41 (<i>C</i>H<sub>2</sub>), 18.72 (<i>C</i>H<sub>3</sub>), 17.54 (<i>C</i>H<sub>3</sub>); ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>21</sub>H<sub>26</sub>Cl<sub>2</sub>N<sub>4</sub>O<sub>10</sub> [M]<sup>+</sup><i>m</i>/<i>z</i>, 564.10; found [M – H]<sup>−</sup>, 563.35.</div></div><div id="sec4_2_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> Compound <b>1h</b></h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, 25 °C, ppm): δ 8.51 (d, <i>J</i> = 12.0 Hz, 1H), 8.17 (d, <i>J</i> = 8.0 Hz, 2H), 8.12 (d, <i>J</i> = 8.0 Hz, 1H), 8.06 (d, <i>J</i> = 8.0 Hz, 1H), 7.64 (d, <i>J</i> = 8.0 Hz, 2H), 6.44 (s, 1H), 5.03 (s, 1H), 4.31 (m, 1H), 4.18 (m, 3H), 4.09 (m, 1H), 2.44 (m, 4H), 1.26 (d, <i>J</i> = 8.0 Hz, 3H), 1.18 (d, <i>J</i> = 8.0 Hz, 3H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>, 25 °C, ppm): δ 174.41 (<i>C</i>O carboxyl), 172.61 (<i>C</i>O carboxyl), 172.54 (<i>C</i>ONH amide), 170.82 (<i>C</i>ONH amide), 164.07 (<i>C</i>ONH CL), 150.65 (<i>C</i>CH phenyl), 147.02 (<i>C</i>NO<sub>2</sub> phenyl), 127.89 (<i>C</i>H phenyl), 123.32 (<i>C</i>H phenyl), 69.93 (<i>C</i>H CL), 66.70 (<i>C</i>H CL), 63.47 (<i>C</i>H<sub>2</sub> CL), 53.95 (<i>C</i>H CL), 48.15 (<i>C</i>H), 47.81 (<i>C</i>H), 30.05 (<i>C</i>H<sub>2</sub>), 29.36 (<i>C</i>H<sub>2</sub>), 18.65 (<i>C</i>H<sub>3</sub>), 17.46 (<i>C</i>H<sub>3</sub>); ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>21</sub>H<sub>26</sub>Cl<sub>2</sub>N<sub>4</sub>O<sub>10</sub> [M]<sup>+</sup><i>m</i>/<i>z</i>, 564.10; found [M – H]<sup>−</sup>, 563.06.</div></div><div id="sec4_2_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> Compound <b>1i</b></h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, 25 °C, ppm): δ 8.51 (d, <i>J</i> = 8.0 Hz, 1H), 8.17 (d, <i>J</i> = 8.0 Hz, 2H), 8.12 (d, <i>J</i> = 8.0 Hz, 1H), 7.98 (d, <i>J</i> = 4.0 Hz, 1H), 7.91 (d, <i>J</i> = 8.0 Hz, 1H), 7.64 (d, <i>J</i> = 8.0 Hz, 2H), 6.43 (s, 1H), 5.02 (s, 1H), 4.21 (m, 5H), 4.09 (m, 1H), 2.45 (m, 4H), 1.22 (m, 9H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>, 25 °C, ppm): δ 174.33 (<i>C</i>O carboxyl), 172.69 (<i>C</i>O carboxyl), 172.28 (<i>C</i>ONH amide), 171.14 (<i>C</i>ONH amide), 164.02 (<i>C</i>ONH CL), 150.67 (<i>C</i>CH phenyl), 147.00 (<i>C</i>NO<sub>2</sub> phenyl), 127.89 (<i>C</i>H phenyl), 123.33 (<i>C</i>H phenyl), 69.84 (<i>C</i>H CL), 66.68 (<i>C</i>H CL), 63.37 (<i>C</i>H<sub>2</sub> CL), 53.87 (<i>C</i>H CL), 49.01 (<i>C</i>H<sub>3</sub> methanol), 48.64 (<i>C</i>H), 48.08 (<i>C</i>H), 47.83 (<i>C</i>H), 30.01 (<i>C</i>H<sub>2</sub>), 29.29 (<i>C</i>H<sub>2</sub>), 18.44 (<i>C</i>H<sub>3</sub>), 17.50 (<i>C</i>H<sub>3</sub>); ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>24</sub>H<sub>31</sub>Cl<sub>2</sub>N<sub>5</sub>O<sub>11</sub> [M]<sup>+</sup><i>m</i>/<i>z</i>, 635.14; found [M – H]<sup>−</sup>, 634.30.</div></div><div id="sec4_2_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> Compound <b>1j</b></h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, 25 °C, ppm): δ 8.51 (d, <i>J</i> = 12.0 Hz, 1H), 8.17 (d, <i>J</i> = 8.0 Hz, 2H), 8.11 (d, <i>J</i> = 8.0 Hz, 1H), 8.00 (d, <i>J</i> = 4.0 Hz, 1H), 7.90 (d, <i>J</i> = 8.0 Hz, 1H), 7.63 (d, <i>J</i> = 8.0 Hz, 2H), 6.43 (s, 1H), 5.02 (s, 1H), 4.22 (m, 5H), 4.08 (m, 1H), 2.44 (m, 4H), 1.22 (m, 9H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>, 25 °C, ppm): δ 174.45 (<i>C</i>O carboxyl), 172.76 (<i>C</i>O carboxyl), 172.40 (<i>C</i>ONH amide), 171.26 (<i>C</i>ONH amide), 164.18 (<i>C</i>ONH CL), 150.74 (<i>C</i>CH phenyl), 147.13 (<i>C</i>NO<sub>2</sub> phenyl), 127.99 (<i>C</i>H phenyl), 123.43 (<i>C</i>H phenyl), 70.07 (<i>C</i>H CL), 66.80 (<i>C</i>H CL), 63.68 (<i>C</i>H<sub>2</sub> CL), 54.07 (<i>C</i>H CL), 49.12 (<i>C</i>H<sub>3</sub> methanol), 48.73 (<i>C</i>H), 48.24 (<i>C</i>H), 47.94 (<i>C</i>H), 30.15 (<i>C</i>H<sub>2</sub>), 29.44 (<i>C</i>H<sub>2</sub>), 18.56 (<i>C</i>H<sub>3</sub>), 17.62 (<i>C</i>H<sub>3</sub>); ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>24</sub>H<sub>31</sub>Cl<sub>2</sub>N<sub>5</sub>O<sub>11</sub> [M]<sup>+</sup><i>m</i>/<i>z</i>, 635.14; found [M – H]<sup>−</sup>, 634.30.</div></div><div id="sec4_2_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> Compound <b>1k</b></h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, 25 °C, ppm): δ 8.51 (d, <i>J</i> = 8.0 Hz, 1H), 8.16 (m, 4H), 8.03 (dd, <i>J</i> = 4.0, 8.0 Hz, 1H), 7.63 (d, <i>J</i> = 8.0 Hz, 2H), 7.27 (m, 2H), 7.20 (m, 3H), 6.44 (s, 1H), 5.02 (s, 1H), 4.41 (m, 1H), 4.20 (m, 2H), 4.08 (m, 1H), 3.68 (m, 4H), 3.04 (m, 1H), 2.88 (m, 1H), 2.46 (m, 4H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>, 25 °C, ppm): δ 173.14 (<i>C</i>O carboxyl), 172.61 (<i>C</i>O carboxyl), 171.71 (<i>C</i>ONH amide), 169.56 (<i>C</i>ONH amide), 169.05 (<i>C</i>ONH amide), 164.05 (<i>C</i>ONH CL), 150.65 (<i>C</i>CH phenyl), 147.01 (<i>C</i>NO<sub>2</sub> phenyl), 137.85 (<i>C</i>CH<sub>2</sub> phenyl), 129.52 (<i>C</i>H phenyl), 128.62 (<i>C</i>H phenyl), 127.87 (<i>C</i>H phenyl), 126.86 (<i>C</i>H phenyl), 123.32 (<i>C</i>H phenyl), 69.89 (<i>C</i>H CL), 66.68 (<i>C</i>H CL), 63.45 (<i>C</i>H<sub>2</sub> CL), 53.90 (<i>C</i>H CL), 42.52 (<i>C</i>H<sub>2</sub>), 41.97 (<i>C</i>H<sub>2</sub>), 37.19 (<i>C</i>H<sub>2</sub>), 30.06 (<i>C</i>H<sub>2</sub>), 29.29 (<i>C</i>H<sub>2</sub>); ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>28</sub>H<sub>31</sub>Cl<sub>2</sub>N<sub>5</sub>O<sub>11</sub> [M]<sup>+</sup><i>m</i>/<i>z</i>, 683.14; found [M – H]<sup>−</sup>, 682.30.</div></div><div id="sec4_2_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> Compound <b>1l</b></h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, 25 °C, ppm): δ 8.51 (d, <i>J</i> = 8.0 Hz, 1H), 8.16 (m, 4H), 8.03 (dd, <i>J</i> = 4.0, 8.0 Hz, 1H), 7.63 (d, <i>J</i> = 8.0 Hz, 2H), 7.27 (m, 2H), 7.20 (m, 3H), 6.43 (s, 1H), 5.02 (s, 1H), 4.41 (m, 1H), 4.20 (m, 2H), 4.08 (m, 1H), 3.67 (m, 4H), 3.04 (dd, <i>J</i> = 4.0, 8.0 Hz, 1H), 2.87 (dd, <i>J</i> = 8.0, 4.0 Hz, 1H), 2.46 (m, 4H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>, 25 °C, ppm): δ 173.14 (<i>C</i>O carboxyl), 172.61 (<i>C</i>O carboxyl), 171.71 (<i>C</i>ONH amide), 169.56 (<i>C</i>ONH amide), 169.05 (<i>C</i>ONH amide), 164.05 (<i>C</i>ONH CL), 150.65 (<i>C</i>CH phenyl), 147.01 (<i>C</i>NO<sub>2</sub> phenyl), 137.85 (<i>C</i>CH<sub>2</sub> phenyl), 129.52 (<i>C</i>H phenyl), 128.61 (<i>C</i>H phenyl), 127.87 (<i>C</i>H phenyl), 126.86 (<i>C</i>H phenyl), 123.32 (<i>C</i>H phenyl), 69.89 (<i>C</i>H CL), 66.69 (<i>C</i>H CL), 63.45 (<i>C</i>H<sub>2</sub> CL), 53.90 (<i>C</i>H CL), 42.52 (<i>C</i>H<sub>2</sub>), 41.97 (<i>C</i>H<sub>2</sub>), 37.19 (<i>C</i>H<sub>2</sub>), 30.06 (<i>C</i>H<sub>2</sub>), 29.29 (<i>C</i>H<sub>2</sub>); ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>28</sub>H<sub>31</sub>Cl<sub>2</sub>N<sub>5</sub>O<sub>11</sub> [M]<sup>+</sup><i>m</i>/<i>z</i>, 683.14; found [M – H]<sup>−</sup>, 682.39.</div></div><div id="sec4_2_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> Compound <b>1m</b></h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, 25 °C, ppm): δ 8.50 (d, <i>J</i> = 12.0 Hz, 1H), 8.36 (s, 1H), 8.16 (m, 2H), 8.06 (m, 2H), 7.63 (d, <i>J</i> = 8.0 Hz, 2H), 7.20 (m, 5H), 6.44 (s, 1H), 5.01 (s, 1H), 4.53 (m, 1H), 4.21 (m, 2H), 4.08 (m, 1H), 3.73 (m, 3H), 3.56 (m, 1H), 3.03 (d, <i>J</i> = 12.0 Hz, 1H), 2.76 (t, <i>J</i> = 12.0, 1H), 2.43 (m, 4H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>, 25 °C, ppm): δ 172.62 (<i>C</i>O carboxyl), 171.80 (<i>C</i>O carboxyl), 171.53 (<i>C</i>ONH amide), 171.44 (<i>C</i>ONH amide), 169.08 (<i>C</i>ONH amide), 164.05 (<i>C</i>ONH CL), 150.63 (<i>C</i>CH phenyl), 147.01 (<i>C</i>NO<sub>2</sub> phenyl), 138.20 (<i>C</i>CH<sub>2</sub> phenyl), 129.56 (<i>C</i>H phenyl), 128.45 (<i>C</i>H phenyl), 127.87 (<i>C</i>H phenyl), 126.68 (<i>C</i>H phenyl), 123.32 (<i>C</i>H phenyl), 69.92 (<i>C</i>H CL), 66.69 (<i>C</i>H CL), 63.49 (<i>C</i>H<sub>2</sub> CL), 54.13 (<i>C</i>H), 53.94 (<i>C</i>H CL), 42.32 (<i>C</i>H<sub>2</sub>), 41.06 (<i>C</i>H<sub>2</sub>), 38.03 (<i>C</i>H<sub>2</sub>), 30.01 (<i>C</i>H<sub>2</sub>), 29.30 (<i>C</i>H<sub>2</sub>); ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>28</sub>H<sub>31</sub>Cl<sub>2</sub>N<sub>5</sub>O<sub>11</sub> [M]<sup>+</sup><i>m</i>/<i>z</i>, 683.14; found [M – H]<sup>−</sup>, 682.17.</div></div><div id="sec4_2_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> Compound <b>1n</b></h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, 25 °C, ppm): δ 8.50 (d, <i>J</i> = 8.0 Hz, 1H), 8.29 (t, <i>J</i> = 4.0 Hz, 1H), 8.17 (m, 3H), 8.05 (dd, <i>J</i> = 8.0, 4.0 Hz, 1H), 7.63 (d, <i>J</i> = 8.0 Hz, 2H), 7.21 (m, 5H), 6.43 (s, 1H), 5.01 (s, 1H), 4.50 (m, 1H), 4.20 (m, 2H), 4.07 (m, 1H), 3.74 (m, 4H), 3.04 (dd, <i>J</i> = 4.0, 8.0 Hz, 1H), 2.76 (m, 1H), 2.35 (m, 4H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>, 25 °C, ppm): δ 172.53 (<i>C</i>O carboxyl), 171.94 (<i>C</i>O carboxyl), 171.48 (<i>C</i>ONH amide), 171.29 (<i>C</i>ONH amide), 169.46 (<i>C</i>ONH amide), 164.07 (<i>C</i>ONH CL), 150.64 (<i>C</i>CH phenyl), 147.01 (<i>C</i>NO<sub>2</sub> phenyl), 138.36 (<i>C</i>CH<sub>2</sub> phenyl), 129.54 (<i>C</i>H phenyl), 128.43 (<i>C</i>H phenyl), 127.88 (<i>C</i>H phenyl), 126.63 (<i>C</i>H phenyl), 123.32 (<i>C</i>H phenyl), 69.93 (<i>C</i>H CL), 66.69 (<i>C</i>H CL), 63.51 (<i>C</i>H<sub>2</sub> CL), 54.54 (<i>C</i>H), 53.93 (<i>C</i>H CL), 42.22 (<i>C</i>H<sub>2</sub>), 41.00 (<i>C</i>H<sub>2</sub>), 37.86 (<i>C</i>H<sub>2</sub>), 30.13 (<i>C</i>H<sub>2</sub>), 29.35 (<i>C</i>H<sub>2</sub>); ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>28</sub>H<sub>31</sub>Cl<sub>2</sub>N<sub>5</sub>O<sub>11</sub> [M]<sup>+</sup><i>m</i>/<i>z</i>, 683.14; found [M – H]<sup>−</sup>, 682.30.</div></div><div id="sec4_2_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> Compound <b>1o</b></h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, 25 °C, ppm): δ 8.51 (d, <i>J</i> = 8.0 Hz, 1H), 8.32 (d, <i>J</i> = 8.0 Hz, 1H), 8.15 (m, 3H), 7.99 (m, 2H), 7.63 (d, <i>J</i> = 8.0 Hz, 2H), 7.21 (m, 11H), 6.43 (d, <i>J</i> = 4.0 Hz, 1H), 5.02 (s, 1H), 4.53 (m, 1H), 4.42 (m, 1H), 4.20 (m, 2H), 4.09 (m, 1H), 3.69 (m, 3H), 2.99 (m, 5H), 2.71 (m, 2H), 2.44 (m, 2H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>, 25 °C, ppm): δ 173.08 (<i>C</i>O carboxyl), 172.60 (<i>C</i>O carboxyl), 171.69 (<i>C</i>ONH amide), 171.43 (<i>C</i>ONH amide), 169.60 (<i>C</i>ONH amide), 168.74 (<i>C</i>ONH amide), 164.05 (<i>C</i>ONH CL), 150.64 (<i>C</i>CH phenyl), 147.01 (<i>C</i>NO<sub>2</sub> phenyl), 138.10 (<i>C</i>CH<sub>2</sub> phenyl), 137.81 (<i>C</i>H phenyl), 129.60 (<i>C</i>H phenyl), 129.52 (<i>C</i>H phenyl), 128.63 (<i>C</i>H phenyl), 128.41 (<i>C</i>H phenyl), 127.87 (<i>C</i>H phenyl), 126.87 (<i>C</i>H phenyl), 126.65 (<i>C</i>H phenyl), 123.32 (<i>C</i>H phenyl), 69.89 (<i>C</i>H CL), 66.68 (<i>C</i>H CL), 63.45 (<i>C</i>H<sub>2</sub> CL), 53.96 (<i>C</i>H CL), 42.50 (<i>C</i>H<sub>2</sub>), 42.10 (<i>C</i>H<sub>2</sub>), 37.94 (<i>C</i>H<sub>2</sub>), 37.06 (<i>C</i>H<sub>2</sub>), 30.05 (<i>C</i>H<sub>2</sub>), 29.28 (<i>C</i>H<sub>2</sub>); ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>37</sub>H<sub>40</sub>Cl<sub>2</sub>N<sub>6</sub>O<sub>12</sub> [M]<sup>+</sup><i>m</i>/<i>z</i>, 830.21; found [M – H]<sup>−</sup>, 829.29.</div></div><div id="sec4_2_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> Compound <b>1p</b></h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, 25 °C, ppm): δ 8.51 (m, 1H), 8.31 (m, 1H), 8.14 (m, 3H), 7.98 (m, 2H), 7.62 (m, 2H), 7.23 (m, 10H), 6.42 (d, <i>J</i> = 16.0 Hz, 1H), 5.00 (d, <i>J</i> = 16.0 Hz, 1H), 4.46 (m, 4H), 4.14 (m, 2H), 3.68 (m, 3H), 3.55 (m, 1H), 2.97 (m, 3H), 2.68 (m, 1H), 2.47 (m, 4H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>, 25 °C, ppm): δ 173.08 (<i>C</i>O carboxyl), 172.60 (<i>C</i>O carboxyl), 171.68 (<i>C</i>ONH amide), 171.42 (<i>C</i>ONH amide), 169.59 (<i>C</i>ONH amide), 168.73 (<i>C</i>ONH amide), 164.04 (<i>C</i>ONH CL), 150.64 (<i>C</i>CH phenyl), 147.01 (<i>C</i>NO<sub>2</sub> phenyl), 138.10 (<i>C</i>CH<sub>2</sub> phenyl), 137.82 (<i>C</i>H phenyl), 129.60 (<i>C</i>H phenyl), 129.52 (<i>C</i>H phenyl), 128.63 (<i>C</i>H phenyl), 128.41 (<i>C</i>H phenyl), 127.87 (<i>C</i>H phenyl), 126.87 (<i>C</i>H phenyl), 126.65 (<i>C</i>H phenyl), 123.32 (<i>C</i>H phenyl), 69.88 (<i>C</i>H CL), 66.68 (<i>C</i>H CL), 63.44 (<i>C</i>H<sub>2</sub> CL), 53.95 (<i>C</i>H CL), 42.49 (<i>C</i>H<sub>2</sub>), 42.09 (<i>C</i>H<sub>2</sub>), 37.94 (<i>C</i>H<sub>2</sub>), 37.05 (<i>C</i>H<sub>2</sub>), 30.05 (<i>C</i>H<sub>2</sub>), 29.28 (<i>C</i>H<sub>2</sub>); ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>37</sub>H<sub>40</sub>Cl<sub>2</sub>N<sub>6</sub>O<sub>12</sub> [M]<sup>+</sup><i>m</i>/<i>z</i>, 830.21; found [M – H]<sup>−</sup>, 829.29.</div></div><div id="sec4_2_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> Compound <b>1q</b></h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, 25 °C, ppm): δ 12.54 (s, 1H), 8.51 (d, <i>J</i> = 12.0 Hz, 1H), 8.16 (m, 7H), 7.64 (d, <i>J</i> = 8.0 Hz, 2H), 6.43 (d, <i>J</i> = 4 Hz, 1H), 6.20 (s, 1H), 5.03 (s, 1H), 4.21 (m, 2H), 4.11 (m, 1H), 3.74 (s, 10H), 2.46 (m, 4H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>, 25 °C, ppm): δ 172.63 (<i>C</i>O carboxyl), 171.73 (<i>C</i>O carboxyl), 171.49 (<i>C</i>ONH amide), 169.84 (<i>C</i>ONH amide), 169.63 (<i>C</i>ONH amide), 169.53 (<i>C</i>ONH amide), 169.47 (<i>C</i>ONH amide), 164.05 (<i>C</i>ONH CL), 150.66 (<i>C</i>CH phenyl), 147.01 (<i>C</i>NO<sub>2</sub> phenyl), 127.88 (<i>C</i>H phenyl), 123.32 (<i>C</i>H phenyl), 69.89 (<i>C</i>H CL), 66.69 (<i>C</i>H CL), 63.47 (<i>C</i>H<sub>2</sub> CL), 53.93 (<i>C</i>H CL), 42.56 (<i>C</i>H<sub>2</sub>), 42.43 (<i>C</i>H<sub>2</sub>), 42.13 (<i>C</i>H<sub>2</sub>), 40.97 (<i>C</i>H<sub>2</sub>), 30.07 (<i>C</i>H<sub>2</sub>), 29.30 (<i>C</i>H<sub>2</sub>); ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>25</sub>H<sub>31</sub>Cl<sub>2</sub>N<sub>7</sub>O<sub>13</sub> [M]<sup>+</sup><i>m</i>/<i>z</i>, 707.14; found [M – H]<sup>−</sup>, 706.22.</div></div><div id="sec4_2_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> Compound <b>2a</b></h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, 25 °C, ppm): δ 8.55 (d, <i>J</i> = 8.0 Hz, 1H), 8.22 (d, <i>J</i> = 8.0 Hz, 1H), 8.17 (d, <i>J</i> = 8.0 Hz, 2H), 8.09 (d, <i>J</i> = 8.0 Hz, 1H), 7.59 (m, 4H), 6.45 (s, 1H), 5.03 (s, 1H), 4.32 (m, 2H), 4.20 (m, 4H), 4.09 (m, 2H), 3.08 (m, 4H), 2.44 (m, 4H), 1.64 (m, 8H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>, 25 °C, ppm): δ 173.62 (<i>C</i>O carboxyl), 172.01 (<i>C</i>O carboxyl), 171.18 (<i>C</i>ONH amide), 164.13 (<i>C</i>ONH amide), 164.10 (<i>C</i>O trifluoroacetic acid) 158.95 (NH<sub>2</sub><i>C</i>(NH)<sub>2</sub>), 157.10 (<i>C</i>ONH amide), 150.61 (<i>C</i>CH phenyl), 147.02 (<i>C</i>NO<sub>2</sub> phenyl), 127.87 (<i>C</i>H phenyl), 123.33 (<i>C</i>H phenyl), 109.99 (<i>C</i>F<sub>3</sub> trifluoroacetic acid), 69.93 (<i>C</i>H CL), 66.68 (<i>C</i>H CL), 63.53 (<i>C</i>H<sub>2</sub> CL), 53.93 (<i>C</i>H CL), 52.28 (<i>C</i>H), 51.92 (<i>C</i>H), 49.01 (<i>C</i>H<sub>3</sub> methanol), 40.93 (<i>C</i>H<sub>2</sub>), 40.70 (<i>C</i>H<sub>2</sub>), 30.02 (<i>C</i>H<sub>2</sub>), 29.84 (<i>C</i>H<sub>2</sub>), 29.30 (<i>C</i>H<sub>2</sub>), 28.46 (<i>C</i>H<sub>2</sub>), 25.56 (<i>C</i>H<sub>2</sub>), 25.34 (<i>C</i>H<sub>2</sub>); ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>27</sub>H<sub>42</sub>Cl<sub>2</sub>N<sub>10</sub>O<sub>10</sub> [M]<sup>+</sup><i>m</i>/<i>z</i>, 736.25; found [M – 3H]<sup>3–</sup>, 733.43.</div></div><div id="sec4_2_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> Compound <b>2b</b></h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, 25 °C, ppm): δ 8.55 (d, <i>J</i> = 8.0 Hz, 1H), 8.17 (t, <i>J</i> = 8.0 Hz, 3H), 8.05 (d, <i>J</i> = 8.0 Hz, 1H), 7.70 (s, 6H), 7.64 (d, <i>J</i> = 8.0 Hz, 2H), 6.45 (s, 1H), 6.24 (d, <i>J</i> = 8.0 Hz, 1H), 5.03 (s, 1H), 4.19 (m, 5H), 2.74 (m, 4H), 2.44 (m, 4H), 1.53 (m, 6H), 1.34 (m, 6H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>, 25 °C, ppm): δ 173.84 (<i>C</i>O carboxyl), 172.65 (<i>C</i>O carboxyl), 172.22 (<i>C</i>ONH amide), 171.16 (<i>C</i>ONH amide), 164.10 (<i>C</i>ONH amide), 150.64 (<i>C</i>CH phenyl), 147.02 (<i>C</i>NO<sub>2</sub> phenyl), 127.88 (<i>C</i>H phenyl), 123.34 (<i>C</i>H phenyl), 69.89 (<i>C</i>H CL), 66.69 (<i>C</i>H CL), 63.45 (<i>C</i>H<sub>2</sub> CL), 53.91 (<i>C</i>H CL), 52.48 (<i>C</i>H), 52.00 (<i>C</i>H), 39.17 (<i>C</i>H<sub>2</sub>), 39.02 (<i>C</i>H<sub>2</sub>), 32.01 (<i>C</i>H<sub>2</sub>), 30.74 (<i>C</i>H<sub>2</sub>), 30.06 (<i>C</i>H<sub>2</sub>), 29.35 (<i>C</i>H<sub>2</sub>), 27.14 (<i>C</i>H<sub>2</sub>), 26.92 (<i>C</i>H<sub>2</sub>), 22.79 (<i>C</i>H<sub>2</sub>), 22.63 (<i>C</i>H<sub>2</sub>); ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>27</sub>H<sub>42</sub>Cl<sub>2</sub>N<sub>6</sub>O<sub>10</sub> [M]<sup>+</sup><i>m</i>/<i>z</i>, 680.23; found [M – 3H]<sup>3–</sup>, 677.27.</div></div><div id="sec4_2_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> Compound <b>2c</b></h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, 25 °C, ppm): δ 12.61 (s, 1H), 8.56 (d, <i>J</i> = 8.0 Hz, 1H), 8.17 (d, <i>J</i> = 8.0 Hz, 3H), 8.06 (d, <i>J</i> = 8.0 Hz, 1H), 7.73 (s, 6H), 7.63 (d, <i>J</i> = 12.0 Hz, 2H), 6.45 (s, 1H), 6.24 (d, <i>J</i> = 8.0 Hz, 1H), 5.02 (s, 1H), 4.20 (m, 5H), 2.74 (s, 4H), 2.43 (m, 4H), 1.51 (m, 6H), 1.35 (m, 6H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>, 25 °C, ppm): δ 173.84 (<i>C</i>O carboxyl), 172.60 (<i>C</i>O carboxyl), 172.21 (<i>C</i>ONH amide), 171.15 (<i>C</i>ONH amide), 164.13 (<i>C</i>ONH amide), 150.62 (<i>C</i>CH phenyl), 147.02 (<i>C</i>NO<sub>2</sub> phenyl), 127.88 (<i>C</i>H phenyl), 123.34 (<i>C</i>H phenyl), 69.98 (<i>C</i>H CL), 66.70 (<i>C</i>H CL), 63.61 (<i>C</i>H<sub>2</sub> CL), 53.97 (<i>C</i>H CL), 52.46 (<i>C</i>H), 52.02 (<i>C</i>H), 39.12 (<i>C</i>H<sub>2</sub>), 39.01 (<i>C</i>H<sub>2</sub>), 31.99 (<i>C</i>H<sub>2</sub>), 30.71 (<i>C</i>H<sub>2</sub>), 30.07 (<i>C</i>H<sub>2</sub>), 29.40 (<i>C</i>H<sub>2</sub>), 27.11 (<i>C</i>H<sub>2</sub>), 26.92 (<i>C</i>H<sub>2</sub>), 22.79 (<i>C</i>H<sub>2</sub>), 22.60 (<i>C</i>H<sub>2</sub>); ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>27</sub>H<sub>42</sub>Cl<sub>2</sub>N<sub>6</sub>O<sub>10</sub> [M]<sup>+</sup><i>m</i>/<i>z</i>, 680.23; found [M – 3H]<sup>3–</sup>, 677.42.</div></div><div id="sec4_2_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> Compound <b>2d</b></h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, 25 °C, ppm): δ 12.71 (s, 1H), 8.52 (d, <i>J</i> = 8.0 Hz, 1H), 8.22 (d, <i>J</i> = 8.0 Hz, 1H), 8.14 (m, 3H), 8.05 (m, 2H), 7.62 (m, 3H), 7.57 (t, <i>J</i> = 4.0 Hz, 1H), 7.19 (m, 14H), 6.42 (m, 1H), 5.01 (s, 1H), 4.50 (m, 2H), 4.32 (m, 1H), 4.18 (m, 3H), 4.08 (m, 1H), 2.94 (m, 8H), 2.67 (m, 2H), 2.32 (m, 4H), 1.75 (m, 2H), 1.57 (m, 6H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>, 25 °C, ppm): δ 173.58 (<i>C</i>O carboxyl), 172.51 (<i>C</i>O carboxyl), 171.68 (<i>C</i>ONH amide), 171.53 (<i>C</i>ONH amide), 171.22 (<i>C</i>ONH amide), 171.11 (<i>C</i>ONH amide), 164.11 (<i>C</i>ONH amide), 157.10 (NH<sub>2</sub><i>C</i>(NH)<sub>2</sub>), 157.07 (NH<sub>2</sub><i>C</i>(NH)<sub>2</sub>), 150.58 (<i>C</i>CH phenyl), 147.01 (<i>C</i>NO<sub>2</sub> phenyl), 138.16 (<i>C</i>CH<sub>2</sub> phenyl), 137.88 (<i>C</i>CH<sub>2</sub> phenyl), 129.59 (<i>C</i>H phenyl), 129.48 (<i>C</i>H phenyl), 128.45 (<i>C</i>H phenyl), 128.39 (<i>C</i>H phenyl), 127.86 (<i>C</i>H phenyl), 123.32 (<i>C</i>H phenyl), 69.99 (<i>C</i>H CL), 66.68 (<i>C</i>H CL), 54.17 (<i>C</i>H<sub>2</sub> CL), 53.95 (<i>C</i>H CL), 52.34 (<i>C</i>H), 52.00 (<i>C</i>H), 30.13 (<i>C</i>H<sub>2</sub>), 29.32 (<i>C</i>H<sub>2</sub>), 25.56 (<i>C</i>H<sub>2</sub>), 25.25 (<i>C</i>H<sub>2</sub>); ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>45</sub>H<sub>60</sub>Cl<sub>2</sub>N<sub>12</sub>O<sub>12</sub> [M]<sup>+</sup><i>m</i>/<i>z</i>, 1030.38; found [M – 3H]<sup>3–</sup>, 1027.59.</div></div><div id="sec4_2_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> Compound <b>2e</b></h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, 25 °C, ppm): δ 8.52 (d, <i>J</i> = 8.0 Hz, 1H), 8.16 (d, <i>J</i> = 8.0 Hz, 2H), 8.06 (m, 3H), 7.70 (s, 5H), 7.62 (d, <i>J</i> = 8.0 Hz, 2H), 7.19 (m, 10H), 6.44 (s, 1H), 5.01 (s, 1H), 4.49 (m, 2H), 4.19 (m, 5H), 3.03 (m, 1H), 2.93 (m, 1H), 2.73 (m, 6H), 2.32 (m, 4H), 1.67 (m, 4H), 1.52 (m, 4H), 1.35 (m, 2H), 1.24 (s, 2H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>, 25 °C, ppm): δ 173.77 (<i>C</i>O carboxyl), 172.76 (<i>C</i>O carboxyl), 172.50 (<i>C</i>ONH amide), 171.52 (<i>C</i>ONH amide), 171.15 (<i>C</i>ONH amide), 171.06 (<i>C</i>ONH amide), 164.09 (<i>C</i>ONH amide), 150.60 (<i>C</i>CH phenyl), 147.02 (<i>C</i>NO<sub>2</sub> phenyl), 138.20 (<i>C</i>CH<sub>2</sub> phenyl), 137.99 (<i>C</i>CH<sub>2</sub> phenyl), 129.60 (<i>C</i>H phenyl), 129.49 (<i>C</i>H phenyl), 128.45 (<i>C</i>H phenyl), 127.87 (<i>C</i>H phenyl), 126.71 (<i>C</i>H phenyl), 126.60 (<i>C</i>H phenyl), 123.33 (<i>C</i>H phenyl), 69.98 (<i>C</i>H CL), 66.69 (<i>C</i>H CL), 63.65 (<i>C</i>H<sub>2</sub> CL), 54.18 (<i>C</i>H), 53.95 (<i>C</i>H CL), 52.31 (<i>C</i>H), 52.15 (<i>C</i>H), 52.04 (<i>C</i>H), 37.66 (<i>C</i>H<sub>2</sub>), 31.95 (<i>C</i>H<sub>2</sub>), 30.79 (<i>C</i>H<sub>2</sub>), 30.14 (<i>C</i>H<sub>2</sub>), 29.34 (<i>C</i>H<sub>2</sub>), 27.11 (<i>C</i>H<sub>2</sub>), 26.94 (<i>C</i>H<sub>2</sub>), 26.90 (<i>C</i>H<sub>2</sub>), 22.77 (<i>C</i>H<sub>2</sub>), 22.67 (<i>C</i>H<sub>2</sub>), 22.48 (<i>C</i>H<sub>2</sub>); ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>45</sub>H<sub>60</sub>Cl<sub>2</sub>N<sub>8</sub>O<sub>12</sub> [M]<sup>+</sup><i>m</i>/<i>z</i>, 974.37; found [M – 3H]<sup>3–</sup>, 971.37.</div></div><div id="sec4_2_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> Compound <b>2f</b></h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, 25 °C, ppm): δ 12.67 (s, 1H), 8.52 (m, 1H), 8.13 (m, 4H), 7.63 (m, 5H), 6.44 (d, <i>J</i> = 8.0 Hz, 1H), 6.22 (t, <i>J</i> = 4.0 Hz, 1H), 5.03 (s, 1H), 4.15 (m, 4H), 3.73 (m, 2H), 2.75 (dd, <i>J</i> = 8.0, 4.0 Hz, 2H), 2.44 (m, 4H), 1.71 (m, 2H), 1.54 (m, 2H), 1.33 (m, 2H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>, 25 °C, ppm): δ 173.82 (<i>C</i>O carboxyl), 172.62 (<i>C</i>O carboxyl), 171.56 (<i>C</i>ONH amide), 169.34 (<i>C</i>ONH amide), 164.08 (<i>C</i>ONH amide), 150.64 (<i>C</i>CH phenyl), 147.02 (<i>C</i>NO<sub>2</sub> phenyl), 127.88 (<i>C</i>H phenyl), 123.33 (<i>C</i>H phenyl), 69.92 (<i>C</i>H CL), 66.70 (<i>C</i>H CL), 63.50 (<i>C</i>H<sub>2</sub> CL), 53.95 (<i>C</i>H CL), 51.88 (<i>C</i>H), 42.16 (<i>C</i>H<sub>2</sub>), 39.06 (<i>C</i>H<sub>2</sub>), 30.92 (<i>C</i>H<sub>2</sub>), 30.07 (<i>C</i>H<sub>2</sub>), 29.35 (<i>C</i>H<sub>2</sub>), 26.90 (<i>C</i>H<sub>2</sub>), 22.69 (<i>C</i>H<sub>2</sub>); ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>23</sub>H<sub>32</sub>Cl<sub>2</sub>N<sub>5</sub>O<sub>10</sub> [M]<sup>+</sup><i>m</i>/<i>z</i>, 608.15; found [M – 2H]<sup>2–</sup>, 606.33.</div></div><div id="sec4_2_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> Compound <b>2g</b></h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, 25 °C, ppm): δ 8.51 (d, <i>J</i> = 8.0 Hz, 1H), 8.17 (d, <i>J</i> = 8.0 Hz, 3H), 8.06 (d, <i>J</i> = 8.0 Hz, 1H), 7.64 (d, <i>J</i> = 8.0 Hz, 2H), 6.44 (s, 1H), 6.19 (d, <i>J</i> = 4.0 Hz, 1H), 5.02 (s, 1H), 4.20 (m, 5H), 2.42 (m, 4H), 2.26 (t, <i>J</i> = 8.0 Hz, 4H), 1.92 (m, 2H), 1.75 (m, 2H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>, 25 °C, ppm): δ 174.43 (<i>C</i>O carboxyl), 174.11 (<i>C</i>O carboxyl), 173.50 (<i>C</i>O carboxyl), 172.64 (<i>C</i>O carboxyl), 171.81 (<i>C</i>ONH amide), 171.16 (<i>C</i>ONH amide), 164.05 (<i>C</i>ONH amide), 150.63 (<i>C</i>CH phenyl), 147.01 (<i>C</i>NO<sub>2</sub> phenyl), 127.87 (<i>C</i>H phenyl), 123.32 (<i>C</i>H phenyl), 69.89 (<i>C</i>H CL), 66.69 (<i>C</i>H CL), 63.45 (<i>C</i>H<sub>2</sub> CL), 53.91 (<i>C</i>H CL), 51.99 (<i>C</i>H), 51.60 (<i>C</i>H), 30.46 (<i>C</i>H<sub>2</sub>), 30.41 (<i>C</i>H<sub>2</sub>), 30.04 (<i>C</i>H<sub>2</sub>), 29.32 (<i>C</i>H<sub>2</sub>), 28.02 (<i>C</i>H<sub>2</sub>), 26.50 (<i>C</i>H<sub>2</sub>); ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>25</sub>H<sub>30</sub>Cl<sub>2</sub>N<sub>4</sub>O<sub>14</sub> [M]<sup>+</sup><i>m</i>/<i>z</i>, 680.11; found [M – H]<sup>−</sup>, 679.23.</div></div><div id="sec4_2_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> Compound <b>2h</b></h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, 25 °C, ppm): δ 12.44 (s, 3H), 8.52 (d, <i>J</i> = 8.0 Hz, 1H), 8.23 (d, <i>J</i> = 8.0 Hz, 1H), 8.17 (d, <i>J</i> = 8.0 Hz, 2H), 8.05 (d, <i>J</i> = 8.0 Hz, 1H), 7.64 (d, <i>J</i> = 8.0 Hz, 2H), 6.44 (s, 1H), 5.04 (s, 1H), 4.60 (m, 1H), 4.50 (m, 1H), 4.22 (m, 2H), 4.10 (t, <i>J</i> = 8.0 Hz, 1H), 2.66 (m, 4H), 2.43 (m, 4H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>, 25 °C, ppm): δ 172.62 (<i>C</i>O carboxyl), 172.54 (<i>C</i>O carboxyl), 172.09 (<i>C</i>O carboxyl), 172.05 (<i>C</i>O carboxyl), 171.25 (<i>C</i>ONH amide), 170.96 (<i>C</i>ONH amide), 164.06 (<i>C</i>ONH amide), 150.65 (<i>C</i>CH phenyl), 147.01 (<i>C</i>NO<sub>2</sub> phenyl), 127.89 (<i>C</i>H phenyl), 123.32 (<i>C</i>H phenyl), 69.89 (<i>C</i>H CL), 66.68 (<i>C</i>H CL), 63.53 (<i>C</i>H<sub>2</sub> CL), 53.91 (<i>C</i>H CL), 49.62 (<i>C</i>H), 48.97 (<i>C</i>H), 36.57 (<i>C</i>H<sub>2</sub>), 36.23 (<i>C</i>H<sub>2</sub>), 30.14 (<i>C</i>H<sub>2</sub>), 29.31 (<i>C</i>H<sub>2</sub>); ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>23</sub>H<sub>26</sub>Cl<sub>2</sub>N<sub>4</sub>O<sub>14</sub> [M]<sup>+</sup><i>m</i>/<i>z</i>, 652.08; found [M – H]<sup>−</sup>, 651.15.</div></div><div id="sec4_2_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> Compound <b>2i</b></h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, 25 °C, ppm): δ 12.40 (s, 2H), 8.52 (d, <i>J</i> = 8.0 Hz, 1H), 8.23 (d, <i>J</i> = 8.0 Hz, 1H), 8.16 (d, <i>J</i> = 8.0 Hz, 2H), 8.01 (d, <i>J</i> = 4.0 Hz, 1H), 7.63 (d, <i>J</i> = 8.0 Hz, 2H), 6.44 (s, 1H), 6.21 (s, 1H), 5.03 (s, 1H), 4.58 (dd, <i>J</i> = 8.0, 4.0 Hz, 1H), 4.49 (m, 1H), 4.22 (m, 2H), 4.10 (m, 1H), 2.66 (m, 2H), 2.55 (m, 2H), 2.42 (m, 4H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>, 25 °C, ppm): δ 172.59 (<i>C</i>O carboxyl), 172.11 (<i>C</i>O carboxyl), 172.05 (<i>C</i>O carboxyl), 171.25 (<i>C</i>ONH amide), 170.90 (<i>C</i>ONH amide), 164.08 (<i>C</i>ONH amide), 150.62 (<i>C</i>CH phenyl), 147.02 (<i>C</i>NO<sub>2</sub> phenyl), 127.88 (<i>C</i>H phenyl), 123.32 (<i>C</i>H phenyl), 69.97 (<i>C</i>H CL), 66.69 (<i>C</i>H CL), 63.62 (<i>C</i>H<sub>2</sub> CL), 53.95 (<i>C</i>H CL), 49.62 (<i>C</i>H), 49.00 (<i>C</i>H), 36.56 (<i>C</i>H<sub>2</sub>), 30.15 (<i>C</i>H<sub>2</sub>), 29.35 (<i>C</i>H<sub>2</sub>); ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>23</sub>H<sub>26</sub>Cl<sub>2</sub>N<sub>4</sub>O<sub>14</sub> [M]<sup>+</sup><i>m</i>/<i>z</i>, 652.08; found [M – H]<sup>−</sup>, 651.25.</div></div><div id="sec4_2_27" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> Compound <b>2j</b></h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub> 25 °C, ppm): δ 8.51 (d, <i>J</i> = 4.0 Hz, 1H), 8.17 (m, 4H), 7.63 (m, 2H), 6.43 (d, <i>J</i> = 4.0 Hz, 1H), 5.03 (s, 1H), 4.54 (m, 1H), 4.22 (m, 2H), 4.09 (m, 1H), 3.16 (d, <i>J</i> = 4.0 Hz, 3H), 2.65 (m, 2H), 2.44 (m, 4H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>, 25 °C, ppm): δ 172.63 (<i>C</i>O carboxyl), 172.60 (<i>C</i>O carboxyl), 172.05 (<i>C</i>O carboxyl), 171.47 (<i>C</i>ONH amide), 169.17 (<i>C</i>ONH amide), 164.05 (<i>C</i>ONH amide), 150.64 (<i>C</i>CH phenyl), 147.02 (<i>C</i>NO<sub>2</sub> phenyl), 127.88 (<i>C</i>H phenyl), 123.32 (<i>C</i>H phenyl), 69.91 (<i>C</i>H CL), 66.69 (<i>C</i>H CL), 63.49 (<i>C</i>H<sub>2</sub> CL), 53.93 (<i>C</i>H CL), 49.01 (<i>C</i>H), 42.09 (<i>C</i>H<sub>2</sub>), 36.42 (<i>C</i>H<sub>2</sub>), 30.06 (<i>C</i>H<sub>2</sub>), 29.32 (<i>C</i>H<sub>2</sub>); ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>21</sub>H<sub>24</sub>Cl<sub>2</sub>N<sub>4</sub>O<sub>12</sub> [M]<sup>+</sup><i>m</i>/<i>z</i>, 594.08; found [M – H]<sup>−</sup>, 593.32.</div></div><div id="sec4_2_28" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> Compound <b>3a</b></h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, 25 °C, ppm): δ 8.50 (d, <i>J</i> = 8.0 Hz, 1H), 8.26 (d, <i>J</i> = 8.0 Hz, 1H), 8.17 (m, 2H), 8.07 (d, <i>J</i> = 8.0 Hz, 1H), 7.63 (d, <i>J</i> = 8.0 Hz, 2H), 7.22 (m, 10H), 6.44 (s, 1H), 5.01 (s, 1H), 4.53 (m, 1H), 4.43 (m, 1H), 4.20 (m, 2H), 4.09 (m, 1H), 2.98 (m, 3H), 2.69 (t, <i>J</i> = 12.0 Hz, 1H), 2.33 (m, 4H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>, 25 °C, ppm): δ 173.12 (<i>C</i>O carboxyl), 172.51 (<i>C</i>O carboxyl), 171.66 (<i>C</i>ONH amide), 170.88 (<i>C</i>ONH amide), 164.05 (<i>C</i>ONH amide), 150.63 (<i>C</i>CH phenyl), 147.01 (<i>C</i>NO<sub>2</sub> phenyl), 138.24 (<i>C</i>CH<sub>2</sub> phenyl), 137.80 (<i>C</i>CH<sub>2</sub> phenyl), 129.57 (<i>C</i>H phenyl), 129.53 (<i>C</i>H phenyl), 128.60 (<i>C</i>H phenyl), 128.37 (<i>C</i>H phenyl), 127.89 (<i>C</i>H phenyl), 126.87 (<i>C</i>H phenyl), 126.59 (<i>C</i>H phenyl), 123.33 (<i>C</i>H phenyl), 69.94 (<i>C</i>H CL), 66.70 (<i>C</i>H CL), 63.49 (<i>C</i>H<sub>2</sub> CL), 53.91 (<i>C</i>H CL), 37.92 (<i>C</i>H<sub>2</sub>), 37.06 (<i>C</i>H<sub>2</sub>), 30.14 (<i>C</i>H<sub>2</sub>), 29.38 (<i>C</i>H<sub>2</sub>); ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>33</sub>H<sub>34</sub>Cl<sub>2</sub>N<sub>4</sub>O<sub>10</sub> [M]<sup>+</sup><i>m</i>/<i>z</i>, 716.17; found [M – H]<sup>−</sup>, 715.39.</div></div><div id="sec4_2_29" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> Compound <b>3b</b></h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, 25 °C, ppm): δ 8.49 (d, <i>J</i> = 8.0 Hz, 1H), 8.34 (d, <i>J</i> = 8.0 Hz, 1H), 8.15 (d, <i>J</i> = 8.0 Hz, 2H), 8.01 (dd, <i>J</i> = 8.0, 12.0 Hz, 2H), 7.83 (m, 3H), 7.74 (s, 1H), 7.61 (d, <i>J</i> = 8.0 Hz, 2H), 7.44 (m, 3H), 7.15 (m, 9H), 6.43 (s, 1H), 6.18 (d, <i>J</i> = 4.0 Hz, 1H), 4.99 (s, 1H), 4.56 (m, 2H), 4.43 (m, 1H), 4.21 (m, 2H), 4.07 (m, 1H), 3.24 (m, 1H), 3.11 (m, 1H), 2.99 (m, 1H), 2.88 (m, 1H), 2.76 (m, 1H), 2.62 (m, 1H), 2.30 (m, 4H), 1.23 (s, 1H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>, 25 °C, ppm): δ 173.03 (<i>C</i>O carboxyl), 172.47 (<i>C</i>O carboxyl), 171.39 (<i>C</i>ONH amide), 171.30 (<i>C</i>ONH amide), 170.84 (<i>C</i>ONH amide), 164.05 (<i>C</i>ONH amide), 150.61 (<i>C</i>CH phenyl), 147.00 (<i>C</i>NO<sub>2</sub> phenyl), 138.24 (<i>C</i>CH<sub>2</sub> naphthyl), 137.97 (<i>C</i>CH<sub>2</sub> phenyl), 135.43 (<i>C</i>CH<sub>2</sub> phenyl), 133.39 (<i>C</i> naphthyl), 132.30 (<i>C</i> naphthyl), 129.60 (<i>C</i>H phenyl), 129.50 (<i>C</i>H phenyl), 128.39 (<i>C</i>H phenyl), 128.28 (<i>C</i>H phenyl), 128.11 (<i>C</i>H phenyl), 128.04 (<i>C</i>H phenyl), 127.90 (<i>C</i>H phenyl), 127.87 (<i>C</i>H phenyl), 127.84 (<i>C</i>H phenyl), 126.64 (<i>C</i>H naphthyl), 126.48 (<i>C</i>H naphthyl), 126.37 (<i>C</i>H naphthyl), 125.90 (<i>C</i>H naphthyl), 123.31 (<i>C</i>H phenyl), 69.96 (<i>C</i>H CL), 66.70 (<i>C</i>H CL), 63.53 (<i>C</i>H<sub>2</sub> CL), 54.08 (<i>C</i>H), 54.01 (<i>C</i>H), 53.91 (<i>C</i>H CL), 37.93 (<i>C</i>H<sub>2</sub>), 37.76 (<i>C</i>H<sub>2</sub>), 37.31 (<i>C</i>H<sub>2</sub>), 30.11 (<i>C</i>H<sub>2</sub>), 29.35 (<i>C</i>H<sub>2</sub>); ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>46</sub>H<sub>45</sub>Cl<sub>2</sub>N<sub>5</sub>O<sub>11</sub> [M]<sup>+</sup><i>m</i>/<i>z</i>, 913.25; found [M – H]<sup>−</sup>, 912.45.</div></div><div id="sec4_2_30" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> Compound <b>3c</b></h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, 25 °C, ppm): δ 8.50 (d, <i>J</i> = 8.0 Hz, 1H), 8.24 (m, 2H), 8.15 (m, 2H), 8.08 (m, 2H), 8.01 (d, <i>J</i> = 8.0 Hz, 1H), 7.82 (m, 3H), 7.75 (s, 1H), 7.62 (m, 2H), 7.44 (m, 2H), 7.15 (m, 10H), 6.43 (s, 1H), 5.00 (s, 1H), 4.68 (m, 1H), 4.48 (m, 2H), 4.19 (m, 2H), 4.07 (m, 1H), 3.75 (d, <i>J</i> = 4.0 Hz, 4H), 3.23 (m, 1H), 3.00 (m, 2H), 2.86 (m, 1H), 2.76 (m, 1H), 2.61 (m, 1H), 2.30 (m, 4H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>, 25 °C, ppm): δ 172.49 (<i>C</i>O carboxyl), 171.49 (<i>C</i>O carboxyl), 171.44 (<i>C</i>ONH amide), 171.37 (<i>C</i>ONH amide), 170.89 (<i>C</i>ONH amide), 169.39 (<i>C</i>ONH amide), 164.07 (<i>C</i>ONH amide), 150.60 (<i>C</i>CH phenyl), 147.01 (<i>C</i>NO<sub>2</sub> phenyl), 138.23 (<i>C</i>CH<sub>2</sub> naphthyl), 137.98 (<i>C</i>CH<sub>2</sub> phenyl), 135.72 (<i>C</i>CH<sub>2</sub> phenyl), 133.39 (<i>C</i> naphthyl), 132.25 (<i>C</i> naphthyl), 129.56 (<i>C</i>H phenyl), 129.51 (<i>C</i>H phenyl), 128.41 (<i>C</i>H phenyl), 128.28 (<i>C</i>H phenyl), 127.88 (<i>C</i>H phenyl), 127.82 (<i>C</i>H phenyl), 126.63 (<i>C</i>H naphthyl), 126.50 (<i>C</i>H naphthyl), 126.29 (<i>C</i>H naphthyl), 125.79 (<i>C</i>H naphthyl), 123.32 (<i>C</i>H phenyl), 69.98 (<i>C</i>H CL), 66.70 (<i>C</i>H CL), 63.55 (<i>C</i>H<sub>2</sub> CL), 54.36 (<i>C</i>H), 54.29 (<i>C</i>H), 53.99 (<i>C</i>H), 53.95 (<i>C</i>H CL), 42.24 (<i>C</i>H<sub>2</sub>), 41.00 (<i>C</i>H<sub>2</sub>), 38.04 (<i>C</i>H<sub>2</sub>), 37.81 (<i>C</i>H<sub>2</sub>), 37.74 (<i>C</i>H<sub>2</sub>), 30.11 (<i>C</i>H<sub>2</sub>), 29.35 (<i>C</i>H<sub>2</sub>); ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>50</sub>H<sub>51</sub>Cl<sub>2</sub>N<sub>7</sub>O<sub>13</sub> [M]<sup>+</sup><i>m</i>/<i>z</i>, 1027.29; found [M – H]<sup>−</sup>, 1026.49.</div></div><div id="sec4_2_31" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> Compound <b>4a</b></h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, 25 °C, ppm): δ 8.51 (d, <i>J</i> = 8.0 Hz, 1H), 8.18 (m, 3H), 7.63 (d, <i>J</i> = 12.0 Hz, 2H), 6.43 (m, 1H), 5.03 (s, 1H), 4.14 (m, 5H), 2.44 (m, 4H), 1.24 (m, 3H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>, 25 °C, ppm): δ 174.69 (<i>C</i>O carboxyl), 172.62 (<i>C</i>O carboxyl), 171.06 (<i>C</i>ONH amide), 164.13 (<i>C</i>ONH amide), 150.78 (<i>C</i>CH phenyl), 147.10 (<i>C</i>NO<sub>2</sub> phenyl), 127.98 (<i>C</i>H phenyl), 123.40 (<i>C</i>H phenyl), 69.93 (<i>C</i>H CL), 66.78 (<i>C</i>H CL), 63.48 (<i>C</i>H<sub>2</sub> CL), 54.00 (<i>C</i>H CL), 48.00 (<i>C</i>H), 30.00 (<i>C</i>H<sub>2</sub>), 29.37 (<i>C</i>H<sub>2</sub>), 17.72 (<i>C</i>H<sub>3</sub>); ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>18</sub>H<sub>21</sub>Cl<sub>2</sub>N<sub>3</sub>O<sub>9</sub> [M]<sup>+</sup><i>m</i>/<i>z</i>, 493.07; found [M – H]<sup>−</sup>, 492.29.</div></div><div id="sec4_2_32" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> Compound <b>4b</b></h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, 25 °C, ppm): δ 8.52 (d, <i>J</i> = 4.0 Hz, 1H), 8.17 (m, 2H), 7.62 (m, 2H), 6.43 (d, <i>J</i> = 4.0 Hz, 1H), 6.22 (d, <i>J</i> = 4.0 Hz, 1H), 5.01 (s, 1H), 4.24 (m, 2H), 4.11 (m, 1H), 3.59 (d, <i>J</i> = 4.0 Hz, 3H), 2.55 (s, 4H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>, 25 °C, ppm): δ 172.87 (<i>C</i>O carboxyl), 172.19 (<i>C</i>O carboxyl), 164.17 (<i>C</i>ONH amide), 150.65 (<i>C</i>CH phenyl), 147.12 (<i>C</i>NO<sub>2</sub> phenyl), 127.94 (<i>C</i>H phenyl), 123.43 (<i>C</i>H phenyl), 70.07 (<i>C</i>H CL), 66.78 (<i>C</i>H CL), 63.90 (<i>C</i>H<sub>2</sub> CL), 54.03 (<i>C</i>H CL), 52.01 (<i>C</i>H<sub>3</sub>), 29.09 (<i>C</i>H<sub>2</sub>), 28.84 (<i>C</i>H<sub>2</sub>); ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>16</sub>H<sub>18</sub>Cl<sub>2</sub>N<sub>2</sub>O<sub>8</sub> [M]<sup>+</sup><i>m</i>/<i>z</i>, 436.04; found [M – H]<sup>−</sup>, 435.15.</div></div><div id="sec4_2_33" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> Compound <b>4c</b></h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, 25 °C, ppm): δ 8.51 (d, <i>J</i> = 12.0 Hz, 1H), 8.16 (m, 2H), 7.77 (s, 1H), 7.64 (m, 2H), 6.44 (d, <i>J</i> = 4.0 Hz, 1H), 5.03 (s, 1H), 4.14 (m, 5H), 3.29 (m, 2H), 2.55 (m, 2H), 2.33 (m, 2H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>, 25 °C, ppm): δ 172.54 (<i>C</i>O carboxyl), 170.68 (<i>C</i>ONH amide), 164.03 (<i>C</i>ONH amide), 150.70 (<i>C</i>CH phenyl), 147.02 (<i>C</i>NO<sub>2</sub> phenyl), 127.92 (<i>C</i>H phenyl), 123.33 (<i>C</i>H phenyl), 69.85 (<i>C</i>H CL), 66.71 (<i>C</i>H CL), 63.33 (<i>C</i>H<sub>2</sub> CL), 53.97 (<i>C</i>H CL), 50.93 (<i>C</i>H<sub>2</sub>), 36.04 (<i>C</i>H<sub>2</sub>), 30.36 (<i>C</i>H<sub>2</sub>), 29.45 (<i>C</i>H<sub>2</sub>); ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>17</sub>H<sub>21</sub>Cl<sub>2</sub>N<sub>3</sub>O<sub>10</sub>S [M]<sup>+</sup><i>m</i>/<i>z</i>, 529.03; found [M – H]<sup>−</sup>, 528.25.</div></div><div id="sec4_2_34" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> Compound <b>4d</b></h4><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, 25 °C, ppm): δ 8.51 (m, 1H), 8.16 (m, 2H), 8.03 (m, 1H), 7.63 (m, 2H), 6.43 (m, 1H), 5.02 (d, <i>J</i> = 8.0 Hz, 1H), 4.21 (s, 2H), 4.10 (m, 1H), 3.79 (m, 2H), 3.25 (m, 2H), 2.47 (m, 2H), 2.37 (m, 2H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>, 25 °C, ppm): δ 172.59 (<i>C</i>O carboxyl), 171.40 (<i>C</i>ONH amide), 164.05 (<i>C</i>ONH amide), 150.66 (<i>C</i>CH phenyl), 147.02 (<i>C</i>NO<sub>2</sub> phenyl), 127.87 (<i>C</i>H phenyl), 123.32 (<i>C</i>H phenyl), 69.89 (<i>C</i>H CL), 66.69 (<i>C</i>H CL), 64.18 (<i>C</i>H<sub>2</sub>), 64.13 (<i>C</i>H<sub>2</sub> CL), 63.45 (<i>C</i>H CL), 53.93 (<i>C</i>H<sub>2</sub>), 30.10 (<i>C</i>H<sub>2</sub>), 29.32 (<i>C</i>H<sub>2</sub>); ESI-MS <i>m</i>/<i>z</i>: calcd for C<sub>17</sub>H<sub>22</sub>Cl<sub>2</sub>N<sub>3</sub>O<sub>11</sub>P [M]<sup>+</sup><i>m</i>/<i>z</i>, 545.04; found [M – H]<sup>−</sup>, 544.24.</div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i52"><a href="/doi/suppl/10.1021/acs.jmedchem.9b01210" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_87061" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_87061" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.9b01210" class="ext-link">10.1021/acs.jmedchem.9b01210</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline"><sup>1</sup>H NMR spectra, <sup>13</sup>C NMR spectra, LC–MS traces and MS of all compounds, and experimental details (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01210/suppl_file/jm9b01210_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and some data (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01210/suppl_file/jm9b01210_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01210/suppl_file/jm9b01210_si_001.pdf">jm9b01210_si_001.pdf (12.41 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01210/suppl_file/jm9b01210_si_002.csv">jm9b01210_si_002.csv (4.05 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.9b01210" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_77601" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_77601" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Bing Xu</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry and , Brandeis University, 415 South Street, Waltham, Massachusetts 02453, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-4639-387X" title="Orcid link">http://orcid.org/0000-0002-4639-387X</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#a4c6dcd1e4c6d6c5cac0c1cdd78ac1c0d1"><span class="__cf_email__" data-cfemail="d8baa0ad98baaab9b6bcbdb1abf6bdbcad">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jiaqing Wang</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry and , Brandeis University, 415 South Street, Waltham, Massachusetts 02453, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Adrianna Shy</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry and , Brandeis University, 415 South Street, Waltham, Massachusetts 02453, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Difei Wu</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry and , Brandeis University, 415 South Street, Waltham, Massachusetts 02453, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Deani L. Cooper</span> - <span class="hlFld-Affiliation affiliation">Department of Biology and , Brandeis University, 415 South Street, Waltham, Massachusetts 02453, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jiashu Xu</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry and , Brandeis University, 415 South Street, Waltham, Massachusetts 02453, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hongjian He</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry and , Brandeis University, 415 South Street, Waltham, Massachusetts 02453, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Wenjun Zhan</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry and , Brandeis University, 415 South Street, Waltham, Massachusetts 02453, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Shenghuan Sun</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry and , Brandeis University, 415 South Street, Waltham, Massachusetts 02453, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Susan T. Lovett</span> - <span class="hlFld-Affiliation affiliation">Department of Biology and , Brandeis University, 415 South Street, Waltham, Massachusetts 02453, United States</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>This manuscript was written through contributions from all authors. All authors have given approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i54">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_47673" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_47673" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was partially supported by NIH (R21AI130560) and NSF (DMR-1420382).</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term"><i>E. coli</i></td><td class="NLM_def"><p class="first last"><i>Escherichia coli</i></p></td></tr><tr><td class="NLM_term">MDR</td><td class="NLM_def"><p class="first last">multidrug resistance</p></td></tr><tr><td class="NLM_term">LPS</td><td class="NLM_def"><p class="first last">lipopolysaccharide</p></td></tr><tr><td class="NLM_term">OM</td><td class="NLM_def"><p class="first last">outer membrane;</p></td></tr><tr><td class="NLM_term">IM</td><td class="NLM_def"><p class="first last">inner membrane</p></td></tr><tr><td class="NLM_term">MIC</td><td class="NLM_def"><p class="first last">minimum inhibitory concentration</p></td></tr><tr><td class="NLM_term">CL</td><td class="NLM_def"><p class="first last">chloramphenicol</p></td></tr><tr><td class="NLM_term">CLsu</td><td class="NLM_def"><p class="first last">chloramphenicol succinate</p></td></tr><tr><td class="NLM_term">PLE</td><td class="NLM_def"><p class="first last">porcine liver esterase</p></td></tr><tr><td class="NLM_term">HBTU</td><td class="NLM_def"><p class="first last"><i>O</i>-(benzotriazol-1-yl)-<i>N</i>,<i>N</i>,<i>N</i>′,<i>N</i>′-tetramethyluronium hexafluorophosphate</p></td></tr><tr><td class="NLM_term">DIPEA</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N</i>-diisopropylethylamine</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i56">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_78832" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_78832" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 59 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Neu, H. C.</span></span> <span> </span><span class="NLM_article-title">The crisis in antibiotic resistance</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1992</span>,  <span class="NLM_volume"><i>257</i></span>,  <span class="NLM_fpage">1064</span>– <span class="NLM_lpage">1073</span>, <span class="refDoi"> DOI: 10.1126/science.257.5073.1064</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1126%2Fscience.257.5073.1064" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1509257" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADyaK38Xls1OhsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=257&publication_year=1992&pages=1064-1073&author=H.+C.+Neu&title=The+crisis+in+antibiotic+resistance&doi=10.1126%2Fscience.257.5073.1064"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The crisis in antibiotic resistance</span></div><div class="casAuthors">Neu, Harold C.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">257</span>
        (<span class="NLM_cas:issue">5073</span>),
    <span class="NLM_cas:pages">1064-72</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    </div><div class="casAbstract">A review with >64 refs.  The synthesis of large nos. of antibiotics over the past three decades has caused complacency about the threat of bacterial resistance.  Bacteria have become resistant to antimicrobial agents as a result of chromosomal changes or the exchange of genetic material via plasmids and transposons.  Streptococcus pneumoniae, Streptococcus pyogenes, and staphylococci, organisms that cause respiratory and cutaneous infections, and members of the Enterobacteriaceae and Pseudomonas families, organisms that cause diarrhea, urinary infection, and sepsis, are now resistant to virtually all of the older antibiotics.  The extensive use of antibiotics in the community and hospitals has fueled this crisis.  Mechanisms such as antibiotic control programs, better hygiene, and synthesis of agents with improved antimicrobial activity need to be adopted in order to limit bacterial resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmntDmhDYAKrVg90H21EOLACvtfcHk0lhCDS-MSE2D2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38Xls1OhsbY%253D&md5=02b98676a76f669ffde4dbb1f10a3876</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1126%2Fscience.257.5073.1064&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.257.5073.1064%26sid%3Dliteratum%253Aachs%26aulast%3DNeu%26aufirst%3DH.%2BC.%26atitle%3DThe%2520crisis%2520in%2520antibiotic%2520resistance%26jtitle%3DScience%26date%3D1992%26volume%3D257%26spage%3D1064%26epage%3D1073%26doi%3D10.1126%2Fscience.257.5073.1064" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ghafourian, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sadeghifard, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soheili, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sekawi, Z.</span></span> <span> </span><span class="NLM_article-title">Extended spectrum beta-lactamases: definition, classification and epidemiology</span>. <i>Curr. Issues Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">11</span>– <span class="NLM_lpage">22</span>, <span class="refDoi"> DOI: 10.21775/cimb.017.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.21775%2Fcimb.017.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=24821872" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A280%3ADC%252BC2cjitFWqtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2015&pages=11-22&author=S.+Ghafourianauthor=N.+Sadeghifardauthor=S.+Soheiliauthor=Z.+Sekawi&title=Extended+spectrum+beta-lactamases%3A+definition%2C+classification+and+epidemiology&doi=10.21775%2Fcimb.017.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Extended Spectrum Beta-lactamases: Definition, Classification and Epidemiology</span></div><div class="casAuthors">Ghafourian Sobhan; Sadeghifard Nourkhoda; Soheili Sara; Sekawi Zamberi</div><div class="citationInfo"><span class="NLM_cas:title">Current issues in molecular biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">11-21</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Extended spectrum beta-lactamases (ESBLs) are defined as enzymes produced by certain bacteria that are able to hydrolyze extended spectrum cephalosporin.  They are therefore effective against beta-lactam antibiotics such as ceftazidime, ceftriaxone, cefotaxime and oxyimino-monobactam.  The objective of the current review is to provide a better understanding of ESBL and the epidemiology of ESBL producing organisms which are among those responsible for antibiotic resistant strains.  Globally, ESBLs are considered to be problematic, particularly in hospitalized patients.  There is an increasing frequency of ESBL in different parts of the world.  The high risk patients are those contaminated with ESBL producer strains as it renders treatment to be ineffective in these patients.  Thus, there an immediate needs to identify EBSL and formulate strategic policy initiatives to reduce their prevalence.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQCRwqOHh6OXxcr74wzj5o5fW6udTcc2eYBqz4-NvoB27ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2cjitFWqtg%253D%253D&md5=3af33b6ae498e7849578e0af6b62a6f4</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.21775%2Fcimb.017.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.21775%252Fcimb.017.011%26sid%3Dliteratum%253Aachs%26aulast%3DGhafourian%26aufirst%3DS.%26aulast%3DSadeghifard%26aufirst%3DN.%26aulast%3DSoheili%26aufirst%3DS.%26aulast%3DSekawi%26aufirst%3DZ.%26atitle%3DExtended%2520spectrum%2520beta-lactamases%253A%2520definition%252C%2520classification%2520and%2520epidemiology%26jtitle%3DCurr.%2520Issues%2520Mol.%2520Biol.%26date%3D2015%26volume%3D17%26spage%3D11%26epage%3D22%26doi%3D10.21775%2Fcimb.017.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Carattoli, A.</span></span> <span> </span><span class="NLM_article-title">Plasmids and the spread of resistance</span>. <i>Int. J. Med. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>303</i></span>,  <span class="NLM_fpage">298</span>– <span class="NLM_lpage">304</span>, <span class="refDoi"> DOI: 10.1016/j.ijmm.2013.02.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1016%2Fj.ijmm.2013.02.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=23499304" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BC3sXktlOgs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=303&publication_year=2013&pages=298-304&author=A.+Carattoli&title=Plasmids+and+the+spread+of+resistance&doi=10.1016%2Fj.ijmm.2013.02.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Plasmids and the spread of resistance</span></div><div class="casAuthors">Carattoli, Alessandra</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Medical Microbiology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">303</span>
        (<span class="NLM_cas:issue">6-7</span>),
    <span class="NLM_cas:pages">298-304</span>CODEN:
                <span class="NLM_cas:coden">IMEMFV</span>;
        ISSN:<span class="NLM_cas:issn">1438-4221</span>.
    
            (<span class="NLM_cas:orgname">Elsevier GmbH</span>)
        </div><div class="casAbstract">A review.  Plasmids represent one of the most difficult challenge for counteracting the dissemination of antimicrobial resistance.  They contribute to the spread of relevant resistance determinants, promoting horizontal gene transfer among unrelated bacteria.  Undistinguishable plasmids were identified in unrelated bacterial strains isolated at huge geog. distant area, with no apparent epidemiol. links.  These plasmids belong to families that are largely prevalent in naturally occurring bacteria, usually carry multiple phys. linked genetic determinants, conferring resistance to different classes of antibiotics simultaneously.  Plasmids also harbor virulence factors and addiction systems, promoting their stability and maintenance in the bacterial host, in different environmental conditions.  The characteristics of the most successful plasmids that were at the origin of the spread of carbapenemase, expanded-spectrum β-lactamase, and plasmid-mediated quinolone resistance genes are discussed in this review.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFn5H47TRIhLVg90H21EOLACvtfcHk0lhCDS-MSE2D2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXktlOgs7s%253D&md5=5aa53c713a9210482f99e19a082eb629</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.ijmm.2013.02.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ijmm.2013.02.001%26sid%3Dliteratum%253Aachs%26aulast%3DCarattoli%26aufirst%3DA.%26atitle%3DPlasmids%2520and%2520the%2520spread%2520of%2520resistance%26jtitle%3DInt.%2520J.%2520Med.%2520Microbiol.%26date%3D2013%26volume%3D303%26spage%3D298%26epage%3D304%26doi%3D10.1016%2Fj.ijmm.2013.02.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Butler, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blaskovich, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, M. A.</span></span> <span> </span><span class="NLM_article-title">Antibiotics in the clinical pipeline at the end of 2015</span>. <i>J. Antibiot.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>70</i></span>,  <span class="NLM_fpage">3</span>– <span class="NLM_lpage">24</span>, <span class="refDoi"> DOI: 10.1038/ja.2016.72</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1038%2Fja.2016.72" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=27353164" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BC2sXislOgsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=2017&pages=3-24&author=M.+S.+Butlerauthor=M.+A.+Blaskovichauthor=M.+A.+Cooper&title=Antibiotics+in+the+clinical+pipeline+at+the+end+of+2015&doi=10.1038%2Fja.2016.72"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Antibiotics in the clinical pipeline at the end of 2015</span></div><div class="casAuthors">Butler, Mark S.; Blaskovich, Mark AT; Cooper, Matthew A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Antibiotics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">70</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">3-24</span>CODEN:
                <span class="NLM_cas:coden">JANTAJ</span>;
        ISSN:<span class="NLM_cas:issn">0021-8820</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  There is growing global recognition that the continued emergence of multidrug-resistant bacteria poses a serious threat to human health.  Action plans released by the World Health Organization and governments of the UK and USA in particular recognize that discovering new antibiotics, particularly those with new modes of action, is one essential element required to avert future catastrophic pandemics.  This review lists the 30 antibiotics and two β-lactamase/β-lactam combinations first launched since 2000, and analyzes in depth seven new antibiotics and two new β-lactam/β-lactamase inhibitor combinations launched since 2013.  The development status, mode of action, spectra of activity and genesis (natural product, natural product-derived, synthetic or protein/mammalian peptide) of the 37 compds. and six β-lactamase/β-lactam combinations being evaluated in clin. trials between 2013 and 2015 are discussed.  Compds. discontinued from clin. development since 2013 and new antibacterial pharmacophores are also reviewed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0BwmpOtSH87Vg90H21EOLACvtfcHk0lhCDS-MSE2D2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXislOgsg%253D%253D&md5=067fba1f69db183c588a45ec888b6294</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fja.2016.72&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fja.2016.72%26sid%3Dliteratum%253Aachs%26aulast%3DButler%26aufirst%3DM.%2BS.%26aulast%3DBlaskovich%26aufirst%3DM.%2BA.%26aulast%3DCooper%26aufirst%3DM.%2BA.%26atitle%3DAntibiotics%2520in%2520the%2520clinical%2520pipeline%2520at%2520the%2520end%2520of%25202015%26jtitle%3DJ.%2520Antibiot.%26date%3D2017%26volume%3D70%26spage%3D3%26epage%3D24%26doi%3D10.1038%2Fja.2016.72" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">World Health Organization</span>.  <i>WHO List of Critically Important Antimicrobials
for Human Medicine (WHO CIA List)</i>. No. WHO/NMH/FOS/FZD/19.1; <span class="NLM_publisher-name">World Health Organization</span>, <span class="NLM_year">2019</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&author=World+Health+Organization&title=WHO+List+of+Critically+Important+Antimicrobials%0Afor+Human+Medicine+%28WHO+CIA+List%29"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3D%26btitle%3DWHO%2520List%2520of%2520Critically%2520Important%2520Antimicrobials%250Afor%2520Human%2520Medicine%2520%2528WHO%2520CIA%2520List%2529%26pub%3DWorld%2520Health%2520Organization%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Appelbaum, P. C.</span></span> <span> </span><span class="NLM_article-title">2012 and beyond: potential for the start of a second pre-antibiotic era?</span>. <i>J. Antimicrob. Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">2062</span>– <span class="NLM_lpage">2068</span>, <span class="refDoi"> DOI: 10.1093/jac/dks213</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1093%2Fjac%2Fdks213" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=22687888" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1egt73M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2012&pages=2062-2068&author=P.+C.+Appelbaum&title=2012+and+beyond%3A+potential+for+the+start+of+a+second+pre-antibiotic+era%3F&doi=10.1093%2Fjac%2Fdks213"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">2012 and beyond: potential for the start of a second pre-antibiotic era?</span></div><div class="casAuthors">Appelbaum, Peter C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Antimicrobial Chemotherapy</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2062-2068</span>CODEN:
                <span class="NLM_cas:coden">JACHDX</span>;
        ISSN:<span class="NLM_cas:issn">0305-7453</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The past decade has seen an alarming confluence of circumstances antithetical to development of new antibacterials, and most of the very few new drugs under development have problems with their microbiol., toxicity, or pharmacokinetics and pharmacodynamics.  Most large pharmaceutical companies have divested themselves of their antibiotic portfolios, and the promise that start-up companies would fill the niche has not been fulfilled.  What is left of the field is being stifled by bureaucratic regulations, problems with approval, lack of expertise and a general lack of understanding of how serious the situation is.  In paediatrics, in particular, there is no sign of any new antibacterial in the foreseeable future.  Waiting for initiatives and deliberations will probably take too long, and a vigorous and deliberate effort to educate the public-who in their turn can apply the needed pressure-seems to be the only way of achieving a rapid turnaround of this most dangerous situation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmZRXWCxg-e7Vg90H21EOLACvtfcHk0lhoxJBlKaYEJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1egt73M&md5=254248181dd6ad25cf4c7ef7ba74ce7c</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1093%2Fjac%2Fdks213&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjac%252Fdks213%26sid%3Dliteratum%253Aachs%26aulast%3DAppelbaum%26aufirst%3DP.%2BC.%26atitle%3D2012%2520and%2520beyond%253A%2520potential%2520for%2520the%2520start%2520of%2520a%2520second%2520pre-antibiotic%2520era%253F%26jtitle%3DJ.%2520Antimicrob.%2520Chemother.%26date%3D2012%26volume%3D67%26spage%3D2062%26epage%3D2068%26doi%3D10.1093%2Fjac%2Fdks213" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arias, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, B. E.</span></span> <span> </span><span class="NLM_article-title">Emergence and management of drug-resistant enterococcal infections</span>. <i>Expert Rev. Anti-Infect. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">637</span>– <span class="NLM_lpage">655</span>, <span class="refDoi"> DOI: 10.1586/14787210.6.5.637</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1586%2F14787210.6.5.637" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=18847403" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BD1cXht1Glu7vJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2008&pages=637-655&author=C.+A.+Ariasauthor=B.+E.+Murray&title=Emergence+and+management+of+drug-resistant+enterococcal+infections&doi=10.1586%2F14787210.6.5.637"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Emergence and management of drug-resistant enterococcal infections</span></div><div class="casAuthors">Arias, Cesar A.; Murray, Barbara E.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Review of Anti-Infective Therapy</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">637-655</span>CODEN:
                <span class="NLM_cas:coden">ERATCK</span>;
        ISSN:<span class="NLM_cas:issn">1478-7210</span>.
    
            (<span class="NLM_cas:orgname">Expert Reviews Ltd.</span>)
        </div><div class="casAbstract">A review.  The treatment of multidrug-resistant enterococcal infections continues to be a challenge for clinicians.  Glycopeptide and β-lactam resistance is now a common feature of the majority of Enterococcus faecium hospital isolates, and resistance to aminoglycosides, quinupristin-dalfopristin, linezolid and daptomycin further complicates the problem.  New antibiotics, such as tigecycline, lipoglycopeptides (dalbavancin, oritavancin and telavancin) and cephalosporins with activity against Enterococcus faecalis (ceftobiprole and ceftaroline), may have potential activity against certain resistant enterococcal strains in specific clin. settings, as may some older antibiotics, such as ampicillin, chloramphenicol, doxycycline, minocycline and nitrofurantoin.  However, the treatment of endovascular infections (particularly endocarditis, where bactericidal therapy is important for optimal cure rates) caused by resistant enterococci continues to be an immense challenge even with the availability of new agents.  The optimal therapy for these infections is not well established and clin. data are usually limited to case reports with conflicting results.  Therefore, treatment decisions may have to be based on animal models and sporadic experiences and the best approach is for the physician to consider carefully each patient on a case by case manner and gather all the clin. and microbiol. information possible regarding species identification and susceptibilities in order to choose a therapeutic regimen that would appear to be active.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSBKIGI357NbVg90H21EOLACvtfcHk0lhoxJBlKaYEJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXht1Glu7vJ&md5=901c7c8d6d85071cd0655938be231d36</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1586%2F14787210.6.5.637&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1586%252F14787210.6.5.637%26sid%3Dliteratum%253Aachs%26aulast%3DArias%26aufirst%3DC.%2BA.%26aulast%3DMurray%26aufirst%3DB.%2BE.%26atitle%3DEmergence%2520and%2520management%2520of%2520drug-resistant%2520enterococcal%2520infections%26jtitle%3DExpert%2520Rev.%2520Anti-Infect.%2520Ther.%26date%3D2008%26volume%3D6%26spage%3D637%26epage%3D655%26doi%3D10.1586%2F14787210.6.5.637" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yong, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toleman, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giske, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sundman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walsh, T. R.</span></span> <span> </span><span class="NLM_article-title">Characterization of a new metallo-β-lactamase gene, blaNDM-1, and a novel erythromycin esterase gene carried on a unique genetic structure in Klebsiella pneumoniae sequence type 14 from India</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">5046</span>– <span class="NLM_lpage">5054</span>, <span class="refDoi"> DOI: 10.1128/aac.00774-09</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1128%2Faac.00774-09" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=19770275" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsF2gs7jK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2009&pages=5046-5054&author=D.+Yongauthor=M.+A.+Tolemanauthor=C.+G.+Giskeauthor=H.+S.+Choauthor=K.+Sundmanauthor=K.+Leeauthor=T.+R.+Walsh&title=Characterization+of+a+new+metallo-%CE%B2-lactamase+gene%2C+blaNDM-1%2C+and+a+novel+erythromycin+esterase+gene+carried+on+a+unique+genetic+structure+in+Klebsiella+pneumoniae+sequence+type+14+from+India&doi=10.1128%2Faac.00774-09"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of a new metallo-β-lactamase gene, blaNDM-1, and a novel erythromycin esterase gene carried on a unique genetic structure in Klebsiella pneumoniae sequence type 14 from India</span></div><div class="casAuthors">Yong, Dongeun; Toleman, Mark A.; Giske, Christian G.; Cho, Hyun S.; Sundman, Kristina; Lee, Kyungwon; Walsh, Timothy R.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">5046-5054</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">A Swedish patient of Indian origin traveled to New Delhi, India, and acquired a urinary tract infection caused by a carbapenem-resistant Klebsiella pneumoniae strain that typed to the sequence type 14 complex.  The isolate, Klebsiella pneumoniae 05-506, was shown to possess a metallo-β-lactamase (MBL) but was neg. for previously known MBL genes.  Gene libraries and amplification of class 1 integrons revealed three resistance-conferring regions; the first contained blaCMY-4 flanked by ISEcP1 and blc.  The second region of 4.8 kb contained a complex class 1 integron with the gene cassettes arr-2, a new erythromycin esterase gene; ereC; aadA1; and cmlA7.  An intact ISCR1 element was shown to be downstream from the qac/sul genes.  The third region consisted of a new MBL gene, designated blaNDM-1, flanked on one side by K. pneumoniae DNA and a truncated IS26 element on its other side.  The last two regions lie adjacent to one another, and all three regions are found on a 180-kb region that is easily transferable to recipient strains and that confers resistance to all antibiotics except fluoroquinolones and colistin.  NDM-1 shares very little identity with other MBLs, with the most similar MBLs being VIM-1/VIM-2, with which it has only 32.4% identity.  As well as possessing unique residues near the active site, NDM-1 also has an addnl. insert between positions 162 and 166 not present in other MBLs.  NDM-1 has a mol. mass of 28 kDa, is monomeric, and can hydrolyze all β-lactams except aztreonam.  Compared to VIM-2, NDM-1 displays tighter binding to most cephalosporins, in particular, cefuroxime, cefotaxime, and cephalothin (cefalotin), and also to the penicillins.  NDM-1 does not bind to the carbapenems as tightly as IMP-1 or VIM-2 and turns over the carbapenems at a rate similar to that of VIM-2.  In addn. to K. pneumoniae 05-506, blaNDM-1 was found on a 140-kb plasmid in an Escherichia coli strain isolated from the patient's feces, inferring the possibility of in vivo conjugation.  The broad resistance carried on these plasmids is a further worrying development for India, which already has high levels of antibiotic resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxWiGWRq2gPrVg90H21EOLACvtfcHk0lhoxJBlKaYEJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsF2gs7jK&md5=ff1df02887753da53b870d4c6edfde39</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1128%2Faac.00774-09&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252Faac.00774-09%26sid%3Dliteratum%253Aachs%26aulast%3DYong%26aufirst%3DD.%26aulast%3DToleman%26aufirst%3DM.%2BA.%26aulast%3DGiske%26aufirst%3DC.%2BG.%26aulast%3DCho%26aufirst%3DH.%2BS.%26aulast%3DSundman%26aufirst%3DK.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DWalsh%26aufirst%3DT.%2BR.%26atitle%3DCharacterization%2520of%2520a%2520new%2520metallo-%25CE%25B2-lactamase%2520gene%252C%2520blaNDM-1%252C%2520and%2520a%2520novel%2520erythromycin%2520esterase%2520gene%2520carried%2520on%2520a%2520unique%2520genetic%2520structure%2520in%2520Klebsiella%2520pneumoniae%2520sequence%2520type%252014%2520from%2520India%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2009%26volume%3D53%26spage%3D5046%26epage%3D5054%26doi%3D10.1128%2Faac.00774-09" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Livermore, D. M.</span></span> <span> </span><span class="NLM_article-title">The need for new antibiotics</span>. <i>Clin. Microbiol. Infect.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">9</span>, <span class="refDoi"> DOI: 10.1111/j.1465-0691.2004.1004.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1111%2Fj.1465-0691.2004.1004.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=15522034" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A280%3ADC%252BD2crltVKjtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2004&pages=1-9&author=D.+M.+Livermore&title=The+need+for+new+antibiotics&doi=10.1111%2Fj.1465-0691.2004.1004.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">The need for new antibiotics</span></div><div class="casAuthors">Livermore D M</div><div class="citationInfo"><span class="NLM_cas:title">Clinical microbiology and infection : the official publication of the European Society of Clinical Microbiology and Infectious Diseases</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">10 Suppl 4</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1-9</span>
        ISSN:<span class="NLM_cas:issn">1198-743X</span>.
    </div><div class="casAbstract">Politicians and public health officials have joined specialist professionals in recognising antibiotic resistance as a threat to modern medicine.  Their response has centred on minimising unnecessary antibiotic prescribing, aiming to reduce selection pressure for resistance.  Despite a few hopeful trends (e.g., declining penicillin resistance among pneumococci in the UK), established resistance is proving hard to displace; moreover, new resistances continue to emerge and to proliferate at new sites.  There consequently remains a strong need for new antibiotics, particularly those directed against multiresistant Gram-negative bacteria in hospitals.  Already some nonfermenters of the genera Acinetobacter and Pseudomonas are resistant to all good antibiotics and many Enterobacteriaceae are resistant to all except carbapenems.  There is also a growing need for new agents against community-acquired pathogens, including the agents of tuberculosis, gonorrhoea and urinary tract infections.  Unless antibacterial development is re-energised, there is a serious risk that a growing proportion of infections, especially in hospitals, will become effectively untreatable.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRta44EV8rLicjl5L8hwNX0fW6udTcc2eZYoYLjc7Pfzrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2crltVKjtg%253D%253D&md5=ed66c9ae026dbb5f5b79bc5ad56106b5</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1111%2Fj.1465-0691.2004.1004.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1465-0691.2004.1004.x%26sid%3Dliteratum%253Aachs%26aulast%3DLivermore%26aufirst%3DD.%2BM.%26atitle%3DThe%2520need%2520for%2520new%2520antibiotics%26jtitle%3DClin.%2520Microbiol.%2520Infect.%26date%3D2004%26volume%3D10%26spage%3D1%26epage%3D9%26doi%3D10.1111%2Fj.1465-0691.2004.1004.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blaskovich, M. A.</span></span> <span> </span><span class="NLM_article-title">The diminished antimicrobial pipeline</span>. <i>Microbiol. Aust.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">92</span>– <span class="NLM_lpage">96</span>, <span class="refDoi"> DOI: 10.1071/ma19025</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1071%2Fma19025" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2019&pages=92-96&author=M.+A.+Blaskovich&title=The+diminished+antimicrobial+pipeline&doi=10.1071%2Fma19025"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1071%2Fma19025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1071%252Fma19025%26sid%3Dliteratum%253Aachs%26aulast%3DBlaskovich%26aufirst%3DM.%2BA.%26atitle%3DThe%2520diminished%2520antimicrobial%2520pipeline%26jtitle%3DMicrobiol.%2520Aust.%26date%3D2019%26volume%3D40%26spage%3D92%26epage%3D96%26doi%3D10.1071%2Fma19025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Walsh, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wencewicz, T. A.</span></span> <span> </span><span class="NLM_article-title">Prospects for new antibiotics: a molecule-centered perspective</span>. <i>J. Antibiot.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">7</span>– <span class="NLM_lpage">22</span>, <span class="refDoi"> DOI: 10.1038/ja.2013.49</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1038%2Fja.2013.49" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=23756684" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht1yitrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2014&pages=7-22&author=C.+T.+Walshauthor=T.+A.+Wencewicz&title=Prospects+for+new+antibiotics%3A+a+molecule-centered+perspective&doi=10.1038%2Fja.2013.49"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Prospects for new antibiotics: a molecule-centered perspective</span></div><div class="casAuthors">Walsh, Christopher T.; Wencewicz, Timothy A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Antibiotics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">7-22</span>CODEN:
                <span class="NLM_cas:coden">JANTAJ</span>;
        ISSN:<span class="NLM_cas:issn">0021-8820</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  There is a continuous need for iterative cycles of antibiotic discovery and development to deal with the selection of resistant pathogens that emerge as therapeutic application of an antibiotic becomes widespread.  A short golden age of antibiotic discovery from nature followed by a subsequent golden half century of medicinal chem. optimization of existing mol. scaffolds emphasizes the need for new antibiotic mol. frameworks.  The authors bring a mol.-centered perspective to the questions of where will new scaffolds come from, when will chemogenetic approaches yield useful new antibiotics and what existing bacterial targets merit contemporary re-examn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8hr8BTzzMG7Vg90H21EOLACvtfcHk0ljHwm-uDgbG_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht1yitrc%253D&md5=fb100c0954679a78210c497daa4fab1d</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fja.2013.49&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fja.2013.49%26sid%3Dliteratum%253Aachs%26aulast%3DWalsh%26aufirst%3DC.%2BT.%26aulast%3DWencewicz%26aufirst%3DT.%2BA.%26atitle%3DProspects%2520for%2520new%2520antibiotics%253A%2520a%2520molecule-centered%2520perspective%26jtitle%3DJ.%2520Antibiot.%26date%3D2014%26volume%3D67%26spage%3D7%26epage%3D22%26doi%3D10.1038%2Fja.2013.49" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Goemaere, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melet, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larue, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lieutaud, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alves de
Sousa, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chevalier, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yimga-Djapa, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giglione, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huguet, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alimi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meinnel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dardel, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Artaud, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pagès, J.-M.</span></span> <span> </span><span class="NLM_article-title">New peptide deformylase inhibitors and cooperative interaction: a combination to improve antibacterial activity</span>. <i>J. Antimicrob. Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">1392</span>– <span class="NLM_lpage">1400</span>, <span class="refDoi"> DOI: 10.1093/jac/dks058</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1093%2Fjac%2Fdks058" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=22378679" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BC38XmvF2qsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2012&pages=1392-1400&author=E.+Goemaereauthor=A.+Meletauthor=V.+Larueauthor=A.+Lieutaudauthor=R.+Alves+de%0ASousaauthor=J.+Chevalierauthor=L.+Yimga-Djapaauthor=C.+Giglioneauthor=F.+Huguetauthor=M.+Alimiauthor=T.+Meinnelauthor=F.+Dardelauthor=I.+Artaudauthor=J.-M.+Pag%C3%A8s&title=New+peptide+deformylase+inhibitors+and+cooperative+interaction%3A+a+combination+to+improve+antibacterial+activity&doi=10.1093%2Fjac%2Fdks058"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">New peptide deformylase inhibitors and cooperative interaction: a combination to improve antibacterial activity</span></div><div class="casAuthors">Goemaere, Emilie; Melet, Armelle; Larue, Valery; Lieutaud, Aurelie; Alves de Sousa, Rodolphe; Chevalier, Jacqueline; Yimga-Djapa, Liselotte; Giglione, Carmela; Huguet, Florian; Alimi, Mickael; Meinnel, Thierry; Dardel, Frederic; Artaud, Isabelle; Pages, Jean-Marie</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Antimicrobial Chemotherapy</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1392-1400</span>CODEN:
                <span class="NLM_cas:coden">JACHDX</span>;
        ISSN:<span class="NLM_cas:issn">0305-7453</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Bacterial drug resistance is a worrying public health problem and there is an urgent need for research and development to provide new antibacterial mols.  Peptide deformylase (PDF) is now a well-described intracellular target selected for the design of a new antibiotic group, PDF inhibitors (PDFIs).  The initial bacterial susceptibility to an inhibitor of a cytoplasmic target is directly assocd. with the diffusion of the compd. through the membrane barrier of Gram-neg. bacteria and with its cytosolic accumulation at the required concn.  The authors have recently demonstrated that the activity of different PDFIs is strongly dependent on the accumulation of the active mols. by using permeabilizing agents, efflux inhibitors or efflux-mutated strains.  Here, they assessed various combination protocols using different putative inhibitors (PDFIs, methionine aminopeptidase inhibitors etc.) to improve antibacterial activity against various resistant Gram-neg. bacteria.  The max. effect was obsd. when combining actinonin with a dual inhibitor of methionine aminopeptidase and PDF, this mol. being also able to interact with the target while actinonin is bound to the PDF active site.  Thus, such a combination of inhibitors acting on two tightly assocd. metabolic steps results in a cooperative effect on bacterial cells and opens an original way to combat multidrug-resistant bacteria.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoh9Jnoz6NUNrVg90H21EOLACvtfcHk0ljHwm-uDgbG_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmvF2qsb4%253D&md5=fa53cc46eb9e4193a62f34c27609254c</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1093%2Fjac%2Fdks058&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjac%252Fdks058%26sid%3Dliteratum%253Aachs%26aulast%3DGoemaere%26aufirst%3DE.%26aulast%3DMelet%26aufirst%3DA.%26aulast%3DLarue%26aufirst%3DV.%26aulast%3DLieutaud%26aufirst%3DA.%26aulast%3DAlves%2Bde%2BSousa%26aufirst%3DR.%26aulast%3DChevalier%26aufirst%3DJ.%26aulast%3DYimga-Djapa%26aufirst%3DL.%26aulast%3DGiglione%26aufirst%3DC.%26aulast%3DHuguet%26aufirst%3DF.%26aulast%3DAlimi%26aufirst%3DM.%26aulast%3DMeinnel%26aufirst%3DT.%26aulast%3DDardel%26aufirst%3DF.%26aulast%3DArtaud%26aufirst%3DI.%26aulast%3DPag%25C3%25A8s%26aufirst%3DJ.-M.%26atitle%3DNew%2520peptide%2520deformylase%2520inhibitors%2520and%2520cooperative%2520interaction%253A%2520a%2520combination%2520to%2520improve%2520antibacterial%2520activity%26jtitle%3DJ.%2520Antimicrob.%2520Chemother.%26date%3D2012%26volume%3D67%26spage%3D1392%26epage%3D1400%26doi%3D10.1093%2Fjac%2Fdks058" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Singh, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miesel, L.</span></span> <span> </span><span class="NLM_article-title">Screening strategies for discovery of antibacterial natural products</span>. <i>Expert Rev. Anti-Infect. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">589</span>– <span class="NLM_lpage">613</span>, <span class="refDoi"> DOI: 10.1586/eri.11.81</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1586%2Feri.11.81" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=21819327" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtVSlu7nF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2011&pages=589-613&author=S.+B.+Singhauthor=K.+Youngauthor=L.+Miesel&title=Screening+strategies+for+discovery+of+antibacterial+natural+products&doi=10.1586%2Feri.11.81"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Screening strategies for discovery of antibacterial natural products</span></div><div class="casAuthors">Singh, Sheo B.; Young, Katherine; Miesel, Lynn</div><div class="citationInfo"><span class="NLM_cas:title">Expert Review of Anti-Infective Therapy</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">589-613</span>CODEN:
                <span class="NLM_cas:coden">ERATCK</span>;
        ISSN:<span class="NLM_cas:issn">1478-7210</span>.
    
            (<span class="NLM_cas:orgname">Expert Reviews Ltd.</span>)
        </div><div class="casAbstract">A review.  Microbial-derived natural products have been a traditional source of antibiotics and antibiotic leads and continue to be effective sources of antibiotics today.  The most important of these discoveries were made about 50 years ago.  Chem. modifications of natural products discovered during those years continue to produce new clin. agents but their value is now, unfortunately, fading away owing to the exhaustion of opportunities of chem. modifications.  The discovery of new natural antibiotics is directly linked to new screening technologies, particularly technologies that can help to eliminate the rediscovery of known antibiotics.  In this article, we have reviewed the screening technologies from recent literature as well as originating from authors labs. that were used for the screening of natural products.  The article covers the entire spectrum of screening strategies, including classical empiric whole-cell assays to more sophisticated antisense based hypersensitive Staphylococcus aureus Fitness Test assays designed to screen all targets simultaneously.  These technologies have led to the discovery of a series of natural product antibiotics, which have been summarized, including the discovery of platensimycin, platencin, nocathiacins, philipimycin, cyclothialidine and muryamycins.  It is quite clear that natural products provide a tremendous opportunity to discover new antibiotics when combined with new hyper-sensitive whole-cell technologies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2PhKCdqbtRLVg90H21EOLACvtfcHk0liLW1mI1TKdmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtVSlu7nF&md5=53cc27c37e75a31dbdac87159e198c68</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1586%2Feri.11.81&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1586%252Feri.11.81%26sid%3Dliteratum%253Aachs%26aulast%3DSingh%26aufirst%3DS.%2BB.%26aulast%3DYoung%26aufirst%3DK.%26aulast%3DMiesel%26aufirst%3DL.%26atitle%3DScreening%2520strategies%2520for%2520discovery%2520of%2520antibacterial%2520natural%2520products%26jtitle%3DExpert%2520Rev.%2520Anti-Infect.%2520Ther.%26date%3D2011%26volume%3D9%26spage%3D589%26epage%3D613%26doi%3D10.1586%2Feri.11.81" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pastagia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuch, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischetti, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, D. B.</span></span> <span> </span><span class="NLM_article-title">Lysins: the arrival of pathogen-directed anti-infectives</span>. <i>J. Med. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">1506</span>– <span class="NLM_lpage">1516</span>, <span class="refDoi"> DOI: 10.1099/jmm.0.061028-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1099%2Fjmm.0.061028-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=23813275" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslymtL7P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2013&pages=1506-1516&author=M.+Pastagiaauthor=R.+Schuchauthor=V.+A.+Fischettiauthor=D.+B.+Huang&title=Lysins%3A+the+arrival+of+pathogen-directed+anti-infectives&doi=10.1099%2Fjmm.0.061028-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Lysins: the arrival of pathogen-directed anti-infectives</span></div><div class="casAuthors">Pastagia, Mina; Schuch, Raymond; Fischetti, Vincent A.; Huang, David B.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medical Microbiology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1506-1516</span>CODEN:
                <span class="NLM_cas:coden">JMMIAV</span>;
        ISSN:<span class="NLM_cas:issn">0022-2615</span>.
    
            (<span class="NLM_cas:orgname">Society for General Microbiology</span>)
        </div><div class="casAbstract">A review.  Lysins represent a novel class of anti-infectives derived from bacteriophage.  Lysins are bacterial cell-wall hydrolytic enzymes that selectively and rapidly kill (≥3 log c.f.u. in 30 min) specific Gram-pos. bacteria providing a targeted therapeutic approach with minimal impact on unrelated commensal flora.  The potential for bacterial resistance to lysins is considered low due to targeting of highly conserved peptidoglycan components.  Through cutting-edge genetic engineering, lysins can be assembled into large libraries of anti-infective agents tailored to any bacterium of interest including drug-resistant Gram-pos. pathogens such as meticillin- and vancomycin-resistant Staphylococcus aureus, vancomycin-resistant Enterococcus faecalis and Enterococcus faecium, and penicillin-resistant Streptococcus pneumoniae.  Lysins can eliminate bacteria systemically and topically from mucosal surfaces and biofilms, as evidenced by exptl. models of sepsis, endocarditis, pneumonia, meningitis, and nasopharyngeal, skin and vaginal decolonization.  Furthermore, lysins can act synergistically with antibiotics and, in the process, resensitize bacteria to non-susceptible antibiotics.  Clin. trials are being prepd. to assess the safety and pharmacokinetic properties of lysins in humans.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowK9VT81rblbVg90H21EOLACvtfcHk0liLW1mI1TKdmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslymtL7P&md5=be25021fa548b979f8aac39f2b9cf69e</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1099%2Fjmm.0.061028-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1099%252Fjmm.0.061028-0%26sid%3Dliteratum%253Aachs%26aulast%3DPastagia%26aufirst%3DM.%26aulast%3DSchuch%26aufirst%3DR.%26aulast%3DFischetti%26aufirst%3DV.%2BA.%26aulast%3DHuang%26aufirst%3DD.%2BB.%26atitle%3DLysins%253A%2520the%2520arrival%2520of%2520pathogen-directed%2520anti-infectives%26jtitle%3DJ.%2520Med.%2520Microbiol.%26date%3D2013%26volume%3D62%26spage%3D1506%26epage%3D1516%26doi%3D10.1099%2Fjmm.0.061028-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bragg, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Westhuizen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coetsee, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boucher, C.</span></span> <span> </span><span class="NLM_article-title">Bacteriophages as potential treatment option for antibiotic resistant bacteria</span>. <i>Adv. Exp. Med. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>807</i></span>,  <span class="NLM_fpage">97</span>– <span class="NLM_lpage">110</span>, <span class="refDoi"> DOI: 10.1007/978-81-322-1777-0_7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1007%2F978-81-322-1777-0_7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=24619620" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BC2MXms12hsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=807&publication_year=2014&pages=97-110&author=R.+Braggauthor=W.+van+der+Westhuizenauthor=J.-Y.+Leeauthor=E.+Coetseeauthor=C.+Boucher&title=Bacteriophages+as+potential+treatment+option+for+antibiotic+resistant+bacteria&doi=10.1007%2F978-81-322-1777-0_7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Bacteriophages as Potential Treatment Option for Antibiotic Resistant Bacteria</span></div><div class="casAuthors">Bragg, Robert; van der Westhuizen, Wouter; Lee, Ji-Yun; Coetsee, Elke; Boucher, Charlotte</div><div class="citationInfo"><span class="NLM_cas:title">Advances in Experimental Medicine and Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">807</span>
        (<span class="NLM_cas:issue">Infectious Diseases and Nanomedicine I</span>),
    <span class="NLM_cas:pages">97-110</span>CODEN:
                <span class="NLM_cas:coden">AEMBAP</span>;
        ISSN:<span class="NLM_cas:issn">2214-8019</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  The world is facing an ever-increasing problem with antibiotic resistant bacteria and we are rapidly heading for a post-antibiotic era.  There is an urgent need to investigate alterative treatment options while there are still a few antibiotics left.  Bacteriophages are viruses that specifically target bacteria.  Before the development of antibiotics, some efforts were made to use bacteriophages as a treatment option, but most of this research stopped soon after the discovery of antibiotics.  There are two different replication options which bacteriophages employ.  These are the lytic and lysogenic life cycles.  Both these life cycles have potential as treatment options.  There are various advantages and disadvantages to the use of bacteriophages as treatment options.  The main advantage is the specificity of bacteriophages and treatments can be designed to specifically target pathogenic bacteria while not neg. affecting the normal microbiota.  There are various advantages to this.  However, the high level of specificity also creates potential problems, the main being the requirement of highly specific diagnostic procedures.  Another potential problem with phage therapy includes the development of immunity and limitations with the registration of phage therapy options.  The latter is driving research toward the expression of phage genes which break the bacterial cell wall, which could then be used as a treatment option.  Various aspects of phage therapy have been investigated in studies undertaken by our research group.  We have investigated specificity of phages to various avian pathogenic E. coli isolates.  Furthermore, the exciting NanoSAM technol. has been employed to investigate bacteriophage replication and aspects of this will be discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_73G00hXLbrVg90H21EOLACvtfcHk0liLW1mI1TKdmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXms12hsL4%253D&md5=590f57e33bf11b319e774d6e1a9212a9</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1007%2F978-81-322-1777-0_7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-81-322-1777-0_7%26sid%3Dliteratum%253Aachs%26aulast%3DBragg%26aufirst%3DR.%26aulast%3Dvan%2Bder%2BWesthuizen%26aufirst%3DW.%26aulast%3DLee%26aufirst%3DJ.-Y.%26aulast%3DCoetsee%26aufirst%3DE.%26aulast%3DBoucher%26aufirst%3DC.%26atitle%3DBacteriophages%2520as%2520potential%2520treatment%2520option%2520for%2520antibiotic%2520resistant%2520bacteria%26jtitle%3DAdv.%2520Exp.%2520Med.%2520Biol.%26date%3D2014%26volume%3D807%26spage%3D97%26epage%3D110%26doi%3D10.1007%2F978-81-322-1777-0_7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mosqueda, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gato, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roca, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">López, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Alegría, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernández
Cuenca, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martínez-Martínez, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pachón, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cisneros, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodríguez-Baño, J.</span></span> <span> </span><span class="NLM_article-title">Characterization of plasmids carrying the blaOXA-24/40 carbapenemase gene and the genes encoding the AbkA/AbkB proteins of a toxin/antitoxin system</span>. <i>J. Antimicrob. Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">2629</span>– <span class="NLM_lpage">2633</span>, <span class="refDoi"> DOI: 10.1093/jac/dku179</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1093%2Fjac%2Fdku179" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=24879663" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BC2MXis1Ciurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2014&pages=2629-2633&author=N.+Mosquedaauthor=E.+Gatoauthor=I.+Rocaauthor=M.+L%C3%B3pezauthor=C.+R.+de+Alegr%C3%ADaauthor=F.+Fern%C3%A1ndez%0ACuencaauthor=L.+Mart%C3%ADnez-Mart%C3%ADnezauthor=J.+Pach%C3%B3nauthor=J.+M.+Cisnerosauthor=J.+Rodr%C3%ADguez-Ba%C3%B1o&title=Characterization+of+plasmids+carrying+the+blaOXA-24%2F40+carbapenemase+gene+and+the+genes+encoding+the+AbkA%2FAbkB+proteins+of+a+toxin%2Fantitoxin+system&doi=10.1093%2Fjac%2Fdku179"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of plasmids carrying the blaOXA-24/40 carbapenemase gene and the genes encoding the AbkA/AbkB proteins of a toxin/antitoxin system</span></div><div class="casAuthors">Mosqueda, Noraida; Gato, Eva; Roca, Ignasi; Lopez, Maria; Ruiz de Alegria, Carlos; Cuenca, Felipe Fernandez; Martinez-Martinez, Luis; Pachon, Jeronimo; Cisneros, Jose Miguel; Rodriguez-Bano, Jesus; Pascual, Alvaro; Vila, Jordi; Bou, German; Tomas, Maria</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Antimicrobial Chemotherapy</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2629-2633</span>CODEN:
                <span class="NLM_cas:coden">JACHDX</span>;
        ISSN:<span class="NLM_cas:issn">0305-7453</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Background: Carbapenem-resistant Acinetobacter baumannii (CRAb) is a major source of nosocomial infections in Spain assocd. with the prodn. of OXA-58-like or OXA-24/40-like β-lactamase enzymes.  We analyzed the plasmids carrying the blaOXA-24/40-like gene in CRAb isolates obtained a decade apart.  Methods: The presence of β-lactamases was screened for by PCR (metallo-β-lactamases, carbapenem-hydrolyzing class D β-lactamases, GES and KPC) in 101 CRAb isolates obtained in two multicentre studies (GEIH/REIPI-Ab-2000 and GEIH/REIPI-Ab-2010; n = 493 Acinetobacter spp).  We analyzed the distribution and characterization of the plasmids carrying the blaOXA-24/40-like gene and sequenced two plasmids, AbATCC223p (2000) and AbATCC329p (2010) from A. baumannii ATCC 17978 transformants.  Results: Acquisition of the blaOXA-24/40-like gene was the main mechanism underlying resistance to carbapenems (48.7% in 2000 compared with 51.6% in 2010).  This gene was mainly isolated in ST2 A. baumannii strains in both studies, although some novel STs (ST79 and ST80) appeared in 2010.  The gene was located in plasmids (8-12 kbp) assocd. with the repAci2 or repAci2/repGR12 types.  The sequences of AbATCC223p (8840 bp) and AbATCC329p (8842 bp) plasmids were similar, particularly regarding the presence of the genes encoding the AbkA/AbkB proteins assocd. with the toxin/antitoxin system.  Moreover, the abkA/abkB gene sequences (>96% identity) were also located in plasmids harbouring the blaOXA-58-like gene.  Conclusions: The action of OXA-24/40 and OXA-58 β-lactamase-like enzymes represents the main mechanism underlying resistance to carbapenems in Spain in the last decade.  AbkA/AbkB proteins in the toxin/antitoxin system may be involved in the successful dissemination of plasmids carrying the blaOXA-24/40-like gene, and probably also the blaOXA-58-like gene, thus contributing to the plasmid stability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqiPMaW9VPKNrVg90H21EOLACvtfcHk0lhhL420JRZpeg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXis1Ciurk%253D&md5=5f034329f78f54bb3f11f6102a3f49d7</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1093%2Fjac%2Fdku179&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjac%252Fdku179%26sid%3Dliteratum%253Aachs%26aulast%3DMosqueda%26aufirst%3DN.%26aulast%3DGato%26aufirst%3DE.%26aulast%3DRoca%26aufirst%3DI.%26aulast%3DL%25C3%25B3pez%26aufirst%3DM.%26aulast%3Dde%2BAlegr%25C3%25ADa%26aufirst%3DC.%2BR.%26aulast%3DFern%25C3%25A1ndez%2BCuenca%26aufirst%3DF.%26aulast%3DMart%25C3%25ADnez-Mart%25C3%25ADnez%26aufirst%3DL.%26aulast%3DPach%25C3%25B3n%26aufirst%3DJ.%26aulast%3DCisneros%26aufirst%3DJ.%2BM.%26aulast%3DRodr%25C3%25ADguez-Ba%25C3%25B1o%26aufirst%3DJ.%26atitle%3DCharacterization%2520of%2520plasmids%2520carrying%2520the%2520blaOXA-24%252F40%2520carbapenemase%2520gene%2520and%2520the%2520genes%2520encoding%2520the%2520AbkA%252FAbkB%2520proteins%2520of%2520a%2520toxin%252Fantitoxin%2520system%26jtitle%3DJ.%2520Antimicrob.%2520Chemother.%26date%3D2014%26volume%3D69%26spage%3D2629%26epage%3D2633%26doi%3D10.1093%2Fjac%2Fdku179" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Okano, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Isley, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boger, D. L.</span></span> <span> </span><span class="NLM_article-title">Peripheral modifications of [Ψ[CH<sub>2</sub>NH]Tpg<sup>4</sup>] vancomycin with added synergistic mechanisms of action provide durable and potent antibiotics</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">E5052</span>– <span class="NLM_lpage">E5061</span>, <span class="refDoi"> DOI: 10.1073/pnas.1704125114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1073%2Fpnas.1704125114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=28559345" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BC2sXovVSitr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2017&pages=E5052-E5061&author=A.+Okanoauthor=N.+A.+Isleyauthor=D.+L.+Boger&title=Peripheral+modifications+of+%5B%CE%A8%5BCH2NH%5DTpg4%5D+vancomycin+with+added+synergistic+mechanisms+of+action+provide+durable+and+potent+antibiotics&doi=10.1073%2Fpnas.1704125114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Peripheral modifications of [Ψ[CH2NH]Tpg4]vancomycin with added synergistic mechanisms of action provide durable and potent antibiotics</span></div><div class="casAuthors">Okano, Akinori; Isley, Nicholas A.; Boger, Dale L.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">E5052-E5061</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Subsequent to binding pocket modifications designed to provide dual d-Ala-d-Ala/d-Ala-d-Lac binding that directly overcome the mol. basis of vancomycin resistance, peripheral structural changes have been explored to improve antimicrobial potency and provide addnl. synergistic mechanisms of action.  A C-terminal peripheral modification, introducing a quaternary ammonium salt, is reported and was found to provide a binding pocket-modified vancomycin analog with a second mechanism of action that is independent of d-Ala-d-Ala/d-Ala-d-Lac binding.  This modification, which induces cell wall permeability and is complementary to the glycopeptide inhibition of cell wall synthesis, was found to provide improvements in antimicrobial potency (200-fold) against vancomycin-resistant Enterococci (VRE).  Furthermore, it is shown that this type of C-terminal modification may be combined with a second peripheral (4-chlorobiphenyl)methyl (CBP) addn. to the vancomycin disaccharide to provide even more potent antimicrobial agents [VRE min. inhibitory concn. (MIC) = 0.01-0.005 μg/mL] with activity that can be attributed to three independent and synergistic mechanisms of action, only one of which requires d-Ala-d-Ala/d-Ala-d-Lac binding.  Finally, it is shown that such peripherally and binding pocket-modified vancomycin analogs display little propensity for acquired resistance by VRE and that their durability against such challenges as well as their antimicrobial potency follow now predictable trends (three > two > one mechanisms of action).  Such antibiotics are expected to display durable antimicrobial activity not prone to rapidly acquired clin. resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwl7lZFEjyJrVg90H21EOLACvtfcHk0lhhL420JRZpeg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXovVSitr8%253D&md5=c98e49ee55fa7f42ab921a43e6d66ed2</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1704125114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1704125114%26sid%3Dliteratum%253Aachs%26aulast%3DOkano%26aufirst%3DA.%26aulast%3DIsley%26aufirst%3DN.%2BA.%26aulast%3DBoger%26aufirst%3DD.%2BL.%26atitle%3DPeripheral%2520modifications%2520of%2520%255B%25CE%25A8%255BCH2NH%255DTpg4%255D%2520vancomycin%2520with%2520added%2520synergistic%2520mechanisms%2520of%2520action%2520provide%2520durable%2520and%2520potent%2520antibiotics%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2017%26volume%3D114%26spage%3DE5052%26epage%3DE5061%26doi%3D10.1073%2Fpnas.1704125114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Okano, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakayama, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindsey, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schammel, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boger, D. L.</span></span> <span> </span><span class="NLM_article-title">Total syntheses and initial evaluation of [Ψ[C(=S)NH]Tpg<sup>4</sup>]vancomycin, [Ψ[C(=NH) NH]Tpg<sup>4</sup>]vancomycin, [Ψ[CH<sub>2</sub>NH]Tpg<sup>4</sup>]vancomycin, and their (4-Chlorobiphenyl)methyl derivatives: synergistic binding pocket and peripheral modifications for the glycopeptide antibiotics</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>137</i></span>,  <span class="NLM_fpage">3693</span>– <span class="NLM_lpage">3704</span>, <span class="refDoi"> DOI: 10.1021/jacs.5b01008</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.5b01008" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BC2MXktVGkt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=137&publication_year=2015&pages=3693-3704&author=A.+Okanoauthor=A.+Nakayamaauthor=K.+Wuauthor=E.+A.+Lindseyauthor=A.+W.+Schammelauthor=Y.+Fengauthor=K.+C.+Collinsauthor=D.+L.+Boger&title=Total+syntheses+and+initial+evaluation+of+%5B%CE%A8%5BC%28%3DS%29NH%5DTpg4%5Dvancomycin%2C+%5B%CE%A8%5BC%28%3DNH%29+NH%5DTpg4%5Dvancomycin%2C+%5B%CE%A8%5BCH2NH%5DTpg4%5Dvancomycin%2C+and+their+%284-Chlorobiphenyl%29methyl+derivatives%3A+synergistic+binding+pocket+and+peripheral+modifications+for+the+glycopeptide+antibiotics&doi=10.1021%2Fjacs.5b01008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Total Syntheses and Initial Evaluation of [Ψ[C(=S)NH]Tpg4]vancomycin, [Ψ[C(=NH)NH]Tpg4]vancomycin, [Ψ[CH2NH]Tpg4]vancomycin, and Their (4-Chlorobiphenyl)methyl Derivatives: Synergistic Binding Pocket and Peripheral Modifications for the Glycopeptide Antibiotics</span></div><div class="casAuthors">Okano, Akinori; Nakayama, Atsushi; Wu, Kejia; Lindsey, Erick A.; Schammel, Alex W.; Feng, Yiqing; Collins, Karen C.; Boger, Dale L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">137</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3693-3704</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Full details of studies are disclosed on the total syntheses of binding pocket analogs of vancomycin bearing the peripheral L-vancosaminyl-1,2-D-glucosyl disaccharide that contain changes to a key single atom in the residue-4 amide (residue-4 carbonyl O → S, NH, H2) designed to directly address the underlying mol. basis of resistance to vancomycin.  Also disclosed are studies piloting the late-stage transformations conducted on the synthetically more accessible C-terminus hydroxymethyl aglycon derivs. and full details of the peripheral chlorobiphenyl functionalization of all of the binding-pocket-modified vancomycin analogs designed for dual D-Ala-D-Ala/D-Ala-D-Lac binding.  Their collective assessment indicates that combined binding pocket and chlorobiphenyl peripherally modified analogs exhibit a remarkable spectrum of antimicrobial activity (VSSA, MRSA, and VanA and VanB VRE) and impressive potencies against both vancomycin-sensitive and vancomycin-resistant bacteria (MICs = 0.06-0.005 and 0.5-0.06 μg/mL for the amidine and methylene analogs, resp.) and likely benefit from two independent and synergistic mechanisms of action, only one of which is dependent on D-Ala-D-Ala/D-Ala-D-Lac binding.  Such analogs are likely to display esp. durable antibiotic activity that is not prone to rapidly acquired clin. resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBtCGg4ti2Z7Vg90H21EOLACvtfcHk0lhRKTo6ubFo8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXktVGkt7Y%253D&md5=9287dfd1c6aca3bc9f3541ce554817fb</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fjacs.5b01008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.5b01008%26sid%3Dliteratum%253Aachs%26aulast%3DOkano%26aufirst%3DA.%26aulast%3DNakayama%26aufirst%3DA.%26aulast%3DWu%26aufirst%3DK.%26aulast%3DLindsey%26aufirst%3DE.%2BA.%26aulast%3DSchammel%26aufirst%3DA.%2BW.%26aulast%3DFeng%26aufirst%3DY.%26aulast%3DCollins%26aufirst%3DK.%2BC.%26aulast%3DBoger%26aufirst%3DD.%2BL.%26atitle%3DTotal%2520syntheses%2520and%2520initial%2520evaluation%2520of%2520%255B%25CE%25A8%255BC%2528%253DS%2529NH%255DTpg4%255Dvancomycin%252C%2520%255B%25CE%25A8%255BC%2528%253DNH%2529%2520NH%255DTpg4%255Dvancomycin%252C%2520%255B%25CE%25A8%255BCH2NH%255DTpg4%255Dvancomycin%252C%2520and%2520their%2520%25284-Chlorobiphenyl%2529methyl%2520derivatives%253A%2520synergistic%2520binding%2520pocket%2520and%2520peripheral%2520modifications%2520for%2520the%2520glycopeptide%2520antibiotics%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2015%26volume%3D137%26spage%3D3693%26epage%3D3704%26doi%3D10.1021%2Fjacs.5b01008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mandler, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baidin, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pahil, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owens, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kahne, D.</span></span> <span> </span><span class="NLM_article-title">Novobiocin enhances polymyxin activity by stimulating lipopolysaccharide transport</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>140</i></span>,  <span class="NLM_fpage">6749</span>– <span class="NLM_lpage">6753</span>, <span class="refDoi"> DOI: 10.1021/jacs.8b02283</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.8b02283" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BC1cXpt12hsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=140&publication_year=2018&pages=6749-6753&author=M.+D.+Mandlerauthor=V.+Baidinauthor=J.+Leeauthor=K.+S.+Pahilauthor=T.+W.+Owensauthor=D.+Kahne&title=Novobiocin+enhances+polymyxin+activity+by+stimulating+lipopolysaccharide+transport&doi=10.1021%2Fjacs.8b02283"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Novobiocin Enhances Polymyxin Activity by Stimulating Lipopolysaccharide Transport</span></div><div class="casAuthors">Mandler, Michael D.; Baidin, Vadim; Lee, James; Pahil, Karanbir S.; Owens, Tristan W.; Kahne, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">140</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">6749-6753</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Gram-neg. bacteria are challenging to kill with antibiotics due to their impenetrable outer membrane contg. lipopolysaccharide (LPS).  The polymyxins, including colistin, are the drugs of last resort for treating Gram-neg. infections.  These drugs bind LPS and disrupt the outer membrane; however, their toxicity limits their usefulness.  Polymyxin has been shown to synergize with many antibiotics including novobiocin, which inhibits DNA gyrase, by facilitating transport of these antibiotics across the outer membrane.  Recently, we have shown that novobiocin not only inhibits DNA gyrase but also binds and stimulates LptB, the ATPase that powers LPS transport.  Here, we report the synthesis of novobiocin derivs. that sep. these two activities.  One analog retains LptB-stimulatory activity but is unable to inhibit DNA gyrase.  This analog, which is not toxic on its own, nevertheless enhances the lethality of polymyxin by binding LptB and stimulating LPS transport.  Therefore, LPS transport agonism contributes substantially to novobiocin-polymyxin synergy.  We also report other novobiocin analogs that inhibit DNA gyrase better than or equal to novobiocin, but bind better to LptB and therefore have even greater LptB stimulatory activity.  These compds. are more potent than novobiocin when used in combination with polymyxin.  Novobiocin analogs optimized for both gyrase inhibition and LPS transport agonism may allow the use of lower doses of polymyxin, increasing its efficacy and safety.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUlbaNzErHFbVg90H21EOLACvtfcHk0lhRKTo6ubFo8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXpt12hsLw%253D&md5=5b22525ded3bcabbb9e4affb02f5e6be</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fjacs.8b02283&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.8b02283%26sid%3Dliteratum%253Aachs%26aulast%3DMandler%26aufirst%3DM.%2BD.%26aulast%3DBaidin%26aufirst%3DV.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DPahil%26aufirst%3DK.%2BS.%26aulast%3DOwens%26aufirst%3DT.%2BW.%26aulast%3DKahne%26aufirst%3DD.%26atitle%3DNovobiocin%2520enhances%2520polymyxin%2520activity%2520by%2520stimulating%2520lipopolysaccharide%2520transport%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2018%26volume%3D140%26spage%3D6749%26epage%3D6753%26doi%3D10.1021%2Fjacs.8b02283" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Neumann, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sassone-Corsi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raffatellu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nolan, E. M.</span></span> <span> </span><span class="NLM_article-title">Esterase-catalyzed siderophore hydrolysis activates an enterobactin-ciprofloxacin conjugate and confers targeted antibacterial activity</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>140</i></span>,  <span class="NLM_fpage">5193</span>– <span class="NLM_lpage">5201</span>, <span class="refDoi"> DOI: 10.1021/jacs.8b01042</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.8b01042" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BC1cXmtVWrurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=140&publication_year=2018&pages=5193-5201&author=W.+Neumannauthor=M.+Sassone-Corsiauthor=M.+Raffatelluauthor=E.+M.+Nolan&title=Esterase-catalyzed+siderophore+hydrolysis+activates+an+enterobactin-ciprofloxacin+conjugate+and+confers+targeted+antibacterial+activity&doi=10.1021%2Fjacs.8b01042"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Esterase-Catalyzed Siderophore Hydrolysis Activates an Enterobactin-Ciprofloxacin Conjugate and Confers Targeted Antibacterial Activity</span></div><div class="casAuthors">Neumann, Wilma; Sassone-Corsi, Martina; Raffatellu, Manuela; Nolan, Elizabeth M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">140</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">5193-5201</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Enteric Gram-neg. bacteria, including Escherichia coli, biosynthesize and deploy the triscatecholate siderophore enterobactin (Ent) in the vertebrate host to acquire iron, an essential nutrient.  We report that Ent-Cipro, a synthetic siderophore-antibiotic conjugate based on the native Ent platform that harbors an alkyl linker at one of the catechols with a ciprofloxacin cargo attached, affords targeted antibacterial activity against E. coli strains that express the pathogen-assocd. iroA gene cluster.  Attachment of the siderophore to ciprofloxacin, a DNA gyrase inhibitor and broad-spectrum antibiotic that is used to treat infections caused by E. coli, generates an inactive prodrug and guides the antibiotic into the cytoplasm of bacteria that express the Ent uptake machinery (FepABCDG).  Intracellular hydrolysis of the siderophore restores the activity of the antibiotic.  Remarkably, Fes, the cytoplasmic Ent hydrolase expressed by all E. coli, does not contribute to Ent-Cipro activation.  Instead, this processing step requires IroD, a cytoplasmic hydrolase that is only expressed by E. coli that harbor the iroA gene cluster and are predominantly pathogenic.  In the uropathogenic E. coli UTI89 and CFT073, Ent-Cipro provides antibacterial activity comparable to unmodified ciprofloxacin.  This work highlights the potential of leveraging and targeting pathogen-assocd. microbial enzymes in narrow-spectrum antibacterial approaches.  Moreover, because E. coli include harmless gut commensals as well as resident microbes that can contribute to disease, Ent-Cipro may provide a valuable chem. tool for strain-selective modulation of the microbiota.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqL1RwaM3wdnrVg90H21EOLACvtfcHk0lhRKTo6ubFo8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXmtVWrurY%253D&md5=d5fe2b8f151a0e611b9d5a25c4d45c85</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjacs.8b01042&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.8b01042%26sid%3Dliteratum%253Aachs%26aulast%3DNeumann%26aufirst%3DW.%26aulast%3DSassone-Corsi%26aufirst%3DM.%26aulast%3DRaffatellu%26aufirst%3DM.%26aulast%3DNolan%26aufirst%3DE.%2BM.%26atitle%3DEsterase-catalyzed%2520siderophore%2520hydrolysis%2520activates%2520an%2520enterobactin-ciprofloxacin%2520conjugate%2520and%2520confers%2520targeted%2520antibacterial%2520activity%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2018%26volume%3D140%26spage%3D5193%26epage%3D5201%26doi%3D10.1021%2Fjacs.8b01042" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xie, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manuguri, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Proietti, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inge, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Häll, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramström, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, M.</span></span> <span> </span><span class="NLM_article-title">Design and synthesis of theranostic antibiotic nanodrugs that display enhanced antibacterial activity and luminescence</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">8464</span>– <span class="NLM_lpage">8469</span>, <span class="refDoi"> DOI: 10.1073/pnas.1708556114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1073%2Fpnas.1708556114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=28743748" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1akt7%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2017&pages=8464-8469&author=S.+Xieauthor=S.+Manuguriauthor=G.+Proiettiauthor=J.+Romsonauthor=Y.+Fuauthor=A.+K.+Ingeauthor=B.+Wuauthor=Y.+Zhangauthor=D.+H%C3%A4llauthor=O.+Ramstr%C3%B6mauthor=M.+Yan&title=Design+and+synthesis+of+theranostic+antibiotic+nanodrugs+that+display+enhanced+antibacterial+activity+and+luminescence&doi=10.1073%2Fpnas.1708556114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Design and synthesis of theranostic antibiotic nanodrugs that display enhanced antibacterial activity and luminescence</span></div><div class="casAuthors">Xie, Sheng; Manuguri, Sesha; Proietti, Giampiero; Romson, Joakim; Fu, Ying; Inge, A. Ken; Wu, Bin; Zhang, Yang; Haell, Daniel; Ramstroem, Olof; Yan, Mingdi</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">32</span>),
    <span class="NLM_cas:pages">8464-8469</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">We report the modular formulation of ciprofloxacin-based pure theranostic nanodrugs that display enhanced antibacterial activities, as well as aggregation-induced emission (AIE) enhancement that was successfully used to image bacteria.  The drug derivs., consisting of ciprofloxacin, a perfluoroaryl ring, and a Ph ring linked by an amidine bond, were efficiently synthesized by a straightforward protocol from a perfluoroaryl azide, ciprofloxacin, and an aldehyde in acetone at room temp.  These compds. are propeller-shaped, and upon pptn. into water, readily assembled into stable nanoaggregates that transformed ciprofloxacin derivs. into AIE-active luminogens.  The nanoaggregates displayed increased luminescence and were successfully used to image bacteria.  In addn., these nanodrugs showed enhanced antibacterial activities, lowering the min. inhibitory concn. (MIC) by more than one order of magnitude against both sensitive and resistant Escherichia coli.  The study represents a strategy in the design and development of pure theranostic nanodrugs for combating drug-resistant bacterial infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1DDzoLoklYbVg90H21EOLACvtfcHk0lgLLDcY4pq7pg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1akt7%252FE&md5=1688d7d82216ab04699df95fb3a16d80</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1708556114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1708556114%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DS.%26aulast%3DManuguri%26aufirst%3DS.%26aulast%3DProietti%26aufirst%3DG.%26aulast%3DRomson%26aufirst%3DJ.%26aulast%3DFu%26aufirst%3DY.%26aulast%3DInge%26aufirst%3DA.%2BK.%26aulast%3DWu%26aufirst%3DB.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DH%25C3%25A4ll%26aufirst%3DD.%26aulast%3DRamstr%25C3%25B6m%26aufirst%3DO.%26aulast%3DYan%26aufirst%3DM.%26atitle%3DDesign%2520and%2520synthesis%2520of%2520theranostic%2520antibiotic%2520nanodrugs%2520that%2520display%2520enhanced%2520antibacterial%2520activity%2520and%2520luminescence%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2017%26volume%3D114%26spage%3D8464%26epage%3D8469%26doi%3D10.1073%2Fpnas.1708556114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Khanna, P.</span>; <span class="NLM_string-name">Snyder, E. S.</span>; <span class="NLM_string-name">Singh, P.</span></span> <span> </span><span class="NLM_article-title">Chloramphenicol derivatives</span>. U.S. Patent <span class="NLM_patent">4,489,157 A</span>, <span class="NLM_year">1984</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1984&author=P.+Khanna&author=E.+S.+Snyder&author=P.+Singh&title=Chloramphenicol+derivatives"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DKhanna%26aufirst%3DP.%26atitle%3DChloramphenicol%2520derivatives%26date%3D1984" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madrid, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, Y.</span></span> <span> </span><span class="NLM_article-title">Bacteria-targeting conjugates based on antimicrobial peptide for bacteria diagnosis and therapy</span>. <i>Mol. Pharm.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">2505</span>– <span class="NLM_lpage">2516</span>, <span class="refDoi"> DOI: 10.1021/acs.molpharmaceut.5b00053</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.molpharmaceut.5b00053" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BC2MXptlOgu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2015&pages=2505-2516&author=H.+Chenauthor=C.+Liuauthor=D.+Chenauthor=K.+Madridauthor=S.+Pengauthor=X.+Dongauthor=M.+Zhangauthor=Y.+Gu&title=Bacteria-targeting+conjugates+based+on+antimicrobial+peptide+for+bacteria+diagnosis+and+therapy&doi=10.1021%2Facs.molpharmaceut.5b00053"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Bacteria-Targeting Conjugates Based on Antimicrobial Peptide for Bacteria Diagnosis and Therapy</span></div><div class="casAuthors">Chen, Haiyan; Liu, Cuicui; Chen, Dan; Madrid, Kyle; Peng, Shuwen; Dong, Xinyue; Zhang, Min; Gu, Yueqing</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmaceutics</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2505-2516</span>CODEN:
                <span class="NLM_cas:coden">MPOHBP</span>;
        ISSN:<span class="NLM_cas:issn">1543-8384</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Chloramphenicol (CAP) is one of the most effective antimicrobial agents, but its therapeutic efficacy is greatly limited by its nonspecific distribution and consequent side effects in neutrophils.  Targeting to the infection sites, and thus restricting CAP nonselective delivery, provides an alternative way to overcome this limitation.  The antibacterial peptide fragment UBI29-41 was identified to have a high bacterial affinity.  However, no research so far has been carried out to utilize UBI29-41 as a ligand for bacteria-targeting therapies.  In this article, we first labeled a near-IR fluorescent dye (ICG02) with UBI29-41 to investigate its targeting capability in different bacteria (S. aureus, E. coli, and P. auruginosa) and bacteria-infected mouse models.  Subsequently, UBI29-41 was conjugated with the typical antibiotic (CAP) through the linker glutaric anhydride to form the conjugate CAP-UBI29-41 for the bacteria-targeting therapy.  In vitro studies demonstrated the enhanced antibacterial effects of CAP-UBI29-41 on S. aureus and E.coil.  Meanwhile, the toxicity of CAP-UBI29-41 on normal cells decreased distinctly in comparison with CAP.  Most importantly, CAP-UBI29-41 exhibited more favorable antibacterial efficacy than CAP in bacteria-bearing mouse models.  All these results demonstrated that UBI29-41 is an ideal targeting ligand to construct antibacterial agents for bacteria-targeting therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1DQ4CZtNRl7Vg90H21EOLACvtfcHk0lgLLDcY4pq7pg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXptlOgu7s%253D&md5=b3f3bded0087efdc8d35ab3977619f2c</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Facs.molpharmaceut.5b00053&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.molpharmaceut.5b00053%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DD.%26aulast%3DMadrid%26aufirst%3DK.%26aulast%3DPeng%26aufirst%3DS.%26aulast%3DDong%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DM.%26aulast%3DGu%26aufirst%3DY.%26atitle%3DBacteria-targeting%2520conjugates%2520based%2520on%2520antimicrobial%2520peptide%2520for%2520bacteria%2520diagnosis%2520and%2520therapy%26jtitle%3DMol.%2520Pharm.%26date%3D2015%26volume%3D12%26spage%3D2505%26epage%3D2516%26doi%3D10.1021%2Facs.molpharmaceut.5b00053" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yunis, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, U. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Restrepo, A.</span></span> <span> </span><span class="NLM_article-title">Reversible bone marrow suppression from chloramphenicol. A consequence of mitochondrial injury</span>. <i>Arch. Intern. Med.</i> <span class="NLM_year" style="font-weight: bold;">1970</span>,  <span class="NLM_volume"><i>126</i></span>,  <span class="NLM_fpage">272</span>– <span class="NLM_lpage">275</span>, <span class="refDoi"> DOI: 10.1001/archinte.126.2.272</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1001%2Farchinte.126.2.272" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=4914780" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A280%3ADyaE3c3ltV2rtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=126&publication_year=1970&pages=272-275&author=A.+A.+Yunisauthor=U.+S.+Smithauthor=A.+Restrepo&title=Reversible+bone+marrow+suppression+from+chloramphenicol.+A+consequence+of+mitochondrial+injury&doi=10.1001%2Farchinte.126.2.272"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Reversible bone marrow suppression from chloramphenicol. A consequence of mitochondrial injury</span></div><div class="casAuthors">Yunis A A; Smith U S; Restrepo A</div><div class="citationInfo"><span class="NLM_cas:title">Archives of internal medicine</span>
        (<span class="NLM_cas:date">1970</span>),
    <span class="NLM_cas:volume">126</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">272-5</span>
        ISSN:<span class="NLM_cas:issn">0003-9926</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTWItyHU5NDm0ewzGPswiWOfW6udTcc2eYVRslrQ_Bx37ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaE3c3ltV2rtg%253D%253D&md5=bde3fea9e17833d2fee5478aef5a22a1</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1001%2Farchinte.126.2.272&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1001%252Farchinte.126.2.272%26sid%3Dliteratum%253Aachs%26aulast%3DYunis%26aufirst%3DA.%2BA.%26aulast%3DSmith%26aufirst%3DU.%2BS.%26aulast%3DRestrepo%26aufirst%3DA.%26atitle%3DReversible%2520bone%2520marrow%2520suppression%2520from%2520chloramphenicol.%2520A%2520consequence%2520of%2520mitochondrial%2520injury%26jtitle%3DArch.%2520Intern.%2520Med.%26date%3D1970%26volume%3D126%26spage%3D272%26epage%3D275%26doi%3D10.1001%2Farchinte.126.2.272" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheetham, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Angacian, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, H.</span></span> <span> </span><span class="NLM_article-title">Peptide-drug conjugates as effective prodrug strategies for targeted delivery</span>. <i>Adv. Drug Delivery Rev.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>110–111</i></span>,  <span class="NLM_fpage">112</span>– <span class="NLM_lpage">126</span>, <span class="refDoi"> DOI: 10.1016/j.addr.2016.06.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1016%2Fj.addr.2016.06.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=27370248" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFSgtbvL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110%E2%80%93111&publication_year=2017&pages=112-126&author=Y.+Wangauthor=A.+G.+Cheethamauthor=G.+Angacianauthor=H.+Suauthor=L.+Xieauthor=H.+Cui&title=Peptide-drug+conjugates+as+effective+prodrug+strategies+for+targeted+delivery&doi=10.1016%2Fj.addr.2016.06.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Peptide-drug conjugates as effective prodrug strategies for targeted delivery</span></div><div class="casAuthors">Wang, Yin; Cheetham, Andrew G.; Angacian, Garren; Su, Hao; Xie, Lisi; Cui, Honggang</div><div class="citationInfo"><span class="NLM_cas:title">Advanced Drug Delivery Reviews</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">110-111</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">112-126</span>CODEN:
                <span class="NLM_cas:coden">ADDREP</span>;
        ISSN:<span class="NLM_cas:issn">0169-409X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Peptide-drug conjugates (PDCs) represent an important class of therapeutic agents that combine one or more drug mols. with a short peptide through a biodegradable linker.  This prodrug strategy uniquely and specifically exploits the biol. activities and self-assembling potential of small-mol. peptides to improve the treatment efficacy of medicinal compds.  We review here the recent progress in the design and synthesis of peptide-drug conjugates in the context of targeted drug delivery and cancer chemotherapy.  We analyze carefully the key design features in choosing the peptide sequence and linker chem. for the drug of interest, as well as the strategies to optimize the conjugate design.  We highlight the recent progress in the design and synthesis of self-assembling peptide-drug amphiphiles to construct supramol. nanomedicine and nanofiber hydrogels for both systemic and topical delivery of active pharmaceutical ingredients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxcgDx1p75N7Vg90H21EOLACvtfcHk0liPjVeDbqIftA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFSgtbvL&md5=3a23bd362031d78b92bc167a44c62c96</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.addr.2016.06.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.addr.2016.06.015%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DY.%26aulast%3DCheetham%26aufirst%3DA.%2BG.%26aulast%3DAngacian%26aufirst%3DG.%26aulast%3DSu%26aufirst%3DH.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DCui%26aufirst%3DH.%26atitle%3DPeptide-drug%2520conjugates%2520as%2520effective%2520prodrug%2520strategies%2520for%2520targeted%2520delivery%26jtitle%3DAdv.%2520Drug%2520Delivery%2520Rev.%26date%3D2017%26volume%3D110%25E2%2580%2593111%26spage%3D112%26epage%3D126%26doi%3D10.1016%2Fj.addr.2016.06.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheetham, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moon, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moxley, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Y.-a.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, H.</span></span> <span> </span><span class="NLM_article-title">Controlled release of free doxorubicin from peptide-drug conjugates by drug loading</span>. <i>J. Control. Release</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>191</i></span>,  <span class="NLM_fpage">123</span>– <span class="NLM_lpage">130</span>, <span class="refDoi"> DOI: 10.1016/j.jconrel.2014.05.051</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1016%2Fj.jconrel.2014.05.051" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=24892976" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVWqsL%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=191&publication_year=2014&pages=123-130&author=Z.+Chenauthor=P.+Zhangauthor=A.+G.+Cheethamauthor=J.+H.+Moonauthor=J.+W.+Moxleyauthor=Y.-a.+Linauthor=H.+Cui&title=Controlled+release+of+free+doxorubicin+from+peptide-drug+conjugates+by+drug+loading&doi=10.1016%2Fj.jconrel.2014.05.051"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Controlled release of free doxorubicin from peptide-drug conjugates by drug loading</span></div><div class="casAuthors">Chen, Zhipeng; Zhang, Pengcheng; Cheetham, Andrew G.; Moon, Jae Hyon; Moxley, James W.; Lin, Yi-an; Cui, Honggang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Controlled Release</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">191</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">123-130</span>CODEN:
                <span class="NLM_cas:coden">JCREEC</span>;
        ISSN:<span class="NLM_cas:issn">0168-3659</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Covalent modification of a drug with a peptide moiety has been extensively used as an effective strategy to improve the drug's therapeutic outcome.  One important consideration in the design of such a prodrug is the release of the free drug from the covalently bound form in a desired fashion.  In most cases, the free drug release rate is controlled by the use of various chem. linkers that bridge the drug to the auxiliary segment.  We report here that the degree of drug conjugation per peptide could also regulate the drug release in addn. to its apparent effect on drug loading of the resulting conjugates.  In this work, we synthesized three peptide-drug conjugates (NTD, d-NTD and q-NTD) in which the cell penetrating peptide Tat is covalently connected to one, two, or four doxorubicin, resp., through a cathepsin B degradable tetrapeptide linker (-Gly-Phe-Leu-Gly-).  We found that the no. of doxorubicin within the conjugate impacts the release of doxorubicin in a significant way, with q-NTD showing the slowest release rate while NTD showing the fastest release rate.  Our cellular uptake expts. reveal that q-NTD accumulated most effectively within cancer cells while NTD shows the lowest intracellular accumulation concn.  Interestingly, our cell viability assessment using a SRB assay reveals that d-NTD is the most potent conjugate against HepG2 human liver cancer cells.  These results suggest that intracellular accumulation efficiency and the free drug release rate are two important factors that det. the in vitro efficacy of drug conjugates.  To further validate this conclusion, we conjugated a short hydrocarbon onto the NTD to improve its cellular uptake, and found that the resulting conjugate, C16NTD, exhibited comparable intracellular accumulation as the q-NTD conjugate but superior anticancer activity due to its more effective release of free doxorubicin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofw3k2yd8YPrVg90H21EOLACvtfcHk0liPjVeDbqIftA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVWqsL%252FM&md5=482b2cbfdac59b40d885d8cf76e3426c</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.jconrel.2014.05.051&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jconrel.2014.05.051%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DP.%26aulast%3DCheetham%26aufirst%3DA.%2BG.%26aulast%3DMoon%26aufirst%3DJ.%2BH.%26aulast%3DMoxley%26aufirst%3DJ.%2BW.%26aulast%3DLin%26aufirst%3DY.-a.%26aulast%3DCui%26aufirst%3DH.%26atitle%3DControlled%2520release%2520of%2520free%2520doxorubicin%2520from%2520peptide-drug%2520conjugates%2520by%2520drug%2520loading%26jtitle%3DJ.%2520Control.%2520Release%26date%3D2014%26volume%3D191%26spage%3D123%26epage%3D130%26doi%3D10.1016%2Fj.jconrel.2014.05.051" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheetham, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Y.-a.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lock, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, H.</span></span> <span> </span><span class="NLM_article-title">Supramolecular nanostructures formed by anticancer drug assembly</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>135</i></span>,  <span class="NLM_fpage">2907</span>– <span class="NLM_lpage">2910</span>, <span class="refDoi"> DOI: 10.1021/ja3115983</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja3115983" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVymsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2013&pages=2907-2910&author=A.+G.+Cheethamauthor=P.+Zhangauthor=Y.-a.+Linauthor=L.+L.+Lockauthor=H.+Cui&title=Supramolecular+nanostructures+formed+by+anticancer+drug+assembly&doi=10.1021%2Fja3115983"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Supramolecular Nanostructures Formed by Anticancer Drug Assembly</span></div><div class="casAuthors">Cheetham, Andrew G.; Zhang, Pengcheng; Lin, Yi-an; Lock, Lye Lin; Cui, Honggang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">135</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2907-2910</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We report here a supramol. strategy to directly assemble the small mol. hydrophobic anticancer drug camptothecin (CPT) into discrete, stable, well-defined nanostructures with a high and quant. drug loading.  Depending on the no. of CPTs in the mol. design, the resulting nanostructures can be either nanofibers or nanotubes, and have a fixed CPT loading content ranging from 23% to 38%.  We found that formation of nanostructures provides protection for both the CPT drug and the biodegradable linker from the external environment and thus offers a mechanism for controlled release of CPT.  Under tumor-relevant conditions, these drug nanostructures can release the bioactive form of CPT and show in vitro efficacy against a no. of cancer cell lines.  This strategy can be extended to construct nanostructures of other types of anticancer drugs and thus presents new opportunities for the development of self-delivering drugs for cancer therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrjrlQs4Vc6kbVg90H21EOLACvtfcHk0liPjVeDbqIftA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvVymsbc%253D&md5=59ef9bc13c8a82ed9727a9d989baa959</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fja3115983&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja3115983%26sid%3Dliteratum%253Aachs%26aulast%3DCheetham%26aufirst%3DA.%2BG.%26aulast%3DZhang%26aufirst%3DP.%26aulast%3DLin%26aufirst%3DY.-a.%26aulast%3DLock%26aufirst%3DL.%2BL.%26aulast%3DCui%26aufirst%3DH.%26atitle%3DSupramolecular%2520nanostructures%2520formed%2520by%2520anticancer%2520drug%2520assembly%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2013%26volume%3D135%26spage%3D2907%26epage%3D2910%26doi%3D10.1021%2Fja3115983" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheetham, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chakroun, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, H.</span></span> <span> </span><span class="NLM_article-title">Self-assembling prodrugs</span>. <i>Chem. Soc. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">6638</span>– <span class="NLM_lpage">6663</span>, <span class="refDoi"> DOI: 10.1039/c7cs00521k</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1039%2FC7CS00521K" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=29019492" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1Chs7bK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2017&pages=6638-6663&author=A.+G.+Cheethamauthor=R.+W.+Chakrounauthor=W.+Maauthor=H.+Cui&title=Self-assembling+prodrugs&doi=10.1039%2Fc7cs00521k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Self-assembling prodrugs</span></div><div class="casAuthors">Cheetham, Andrew G.; Chakroun, Rami W.; Ma, Wang; Cui, Honggang</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Society Reviews</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">6638-6663</span>CODEN:
                <span class="NLM_cas:coden">CSRVBR</span>;
        ISSN:<span class="NLM_cas:issn">0306-0012</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Covalent modification of therapeutic compds. is a clin. proven strategy to devise prodrugs with enhanced treatment efficacies.  This prodrug strategy relies on the modified drugs that possess advantageous pharmacokinetic properties and administration routes over their parent drug.  Self-assembling prodrugs represent an emerging class of therapeutic agents capable of spontaneously assocg. into well-defined supramol. nanostructures in aq. solns.  The self-assembly of prodrugs expands the functional space of conventional prodrug design, affording a possible pathway to more effective therapies as the assembled nanostructure possesses distinct physicochem. properties and interaction potentials that can be tailored to specific administration routes and disease treatment.  In this review, we will discuss the various types of self-assembling prodrugs in development, providing an overview of the methods used to control their structure and function and, ultimately, our perspective on their current and future potential.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVWv5tijAnLbVg90H21EOLACvtfcHk0ljkRF1lN3n5lQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1Chs7bK&md5=c57c113dd4215e71d808d02d365311d9</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1039%2FC7CS00521K&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7CS00521K%26sid%3Dliteratum%253Aachs%26aulast%3DCheetham%26aufirst%3DA.%2BG.%26aulast%3DChakroun%26aufirst%3DR.%2BW.%26aulast%3DMa%26aufirst%3DW.%26aulast%3DCui%26aufirst%3DH.%26atitle%3DSelf-assembling%2520prodrugs%26jtitle%3DChem.%2520Soc.%2520Rev.%26date%3D2017%26volume%3D46%26spage%3D6638%26epage%3D6663%26doi%3D10.1039%2Fc7cs00521k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">FengZhao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span> <span> </span><span class="NLM_article-title">Disulfide bond reduction-triggered molecular hydrogels of folic acid-Taxol conjugates</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">6946</span>– <span class="NLM_lpage">6951</span>, <span class="refDoi"> DOI: 10.1039/c3ob40969d</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1039%2Fc3ob40969d" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=23989242" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFWksbjP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2013&pages=6946-6951&author=C.+Yangauthor=D.+Liauthor=Q.+FengZhaoauthor=L.+Wangauthor=L.+Wangauthor=Z.+Yang&title=Disulfide+bond+reduction-triggered+molecular+hydrogels+of+folic+acid-Taxol+conjugates&doi=10.1039%2Fc3ob40969d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Disulfide bond reduction-triggered molecular hydrogels of folic acid-Taxol conjugates</span></div><div class="casAuthors">Yang, Chengbiao; Li, Dongxia; Qianqi, Feng Zhao; Wang, Lianyong; Wang, Ling; Yang, Zhimou</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">40</span>),
    <span class="NLM_cas:pages">6946-6951</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Mol. hydrogels of therapeutic agents are a novel kind of self-delivery system that can sustain release of drugs or pro-drugs.  We have previously developed a mol. hydrogelator of folic acid (FA)-Taxol conjugate triggered by phosphatase.  In this paper, we report a novel mol. hydrogelator system of FA-Taxol conjugates with improved synthetic strategy.  The hydrogels are formed by the redn. of disulfide bond by glutathione (GSH).  These hydrogels could sustain release of Taxol through ester bond hydrolysis.  Compared with i.v. (i.v.) injection of clin. used Taxol with four times the dosage, our hydrogel could inhibit tumor growth more efficiently by a single dose of intra-tumor (i.t.) administration.  These observations suggested the big potential of this novel gelation system of Taxol for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5mkDu5LBPZrVg90H21EOLACvtfcHk0ljkRF1lN3n5lQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFWksbjP&md5=f9a178df76dfc49e1ee4ad984cb67e9d</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1039%2Fc3ob40969d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc3ob40969d%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DD.%26aulast%3DFengZhao%26aufirst%3DQ.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DZ.%26atitle%3DDisulfide%2520bond%2520reduction-triggered%2520molecular%2520hydrogels%2520of%2520folic%2520acid-Taxol%2520conjugates%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2013%26volume%3D11%26spage%3D6946%26epage%3D6951%26doi%3D10.1039%2Fc3ob40969d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lovett, S. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, B.</span></span> <span> </span><span class="NLM_article-title">Diglycine enables rapid intrabacterial hydrolysis for activating anbiotics against gram-negative bacteria</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">10631</span>– <span class="NLM_lpage">10634</span>, <span class="refDoi"> DOI: 10.1002/anie.201905230</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1002%2Fanie.201905230" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BC1MXht1Oqt7zI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2019&pages=10631-10634&author=J.+Wangauthor=D.+L.+Cooperauthor=W.+Zhanauthor=D.+Wuauthor=H.+Heauthor=S.+Sunauthor=S.+T.+Lovettauthor=B.+Xu&title=Diglycine+enables+rapid+intrabacterial+hydrolysis+for+activating+anbiotics+against+gram-negative+bacteria&doi=10.1002%2Fanie.201905230"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Diglycine Enables Rapid Intrabacterial Hydrolysis for Activating Anbiotics against Gram-negative Bacteria</span></div><div class="casAuthors">Wang, Jiaqing; Cooper, Deani L.; Zhan, Wenjun; Wu, Difei; He, Hongjian; Sun, Shenghuan; Lovett, Susan T.; Xu, Bing</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">10631-10634</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Antimicrobial drug resistance demands novel approaches for improving the efficacy of antibiotics, esp. against Gram-neg. bacteria.  We report that conjugating a diglycine (GG) to an antibiotic prodrug drastically accelerates intrabacterial ester-bond hydrolysis required for activating the antibiotic.  Specifically, the attachment of GG to chloramphenicol succinate (CLsu) generates CLsuGG, which exhibits about an order of magnitude higher inhibitory efficacy than CLsu against Escherichia coli.  Further studies reveal that CLsuGG undergoes rapid hydrolysis, catalyzed by intrabacterial esterases (e.g., BioH and YjfP), to generate chloramphenicol (CL) in E. coli.  Importantly, the conjugate exhibits lower cytotoxicity to bone marrow stromal cells than CL.  Structural analogs of CLsuGG indicate that the conjugation of GG to an antibiotic prodrug is an effective strategy for accelerating enzymic prodrug hydrolysis and enhancing the antibacterial efficacy of antibiotics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7QnuC8K8iL7Vg90H21EOLACvtfcHk0ljkRF1lN3n5lQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXht1Oqt7zI&md5=1b64f171d468ac55d219c8f5c822d22a</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1002%2Fanie.201905230&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201905230%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DCooper%26aufirst%3DD.%2BL.%26aulast%3DZhan%26aufirst%3DW.%26aulast%3DWu%26aufirst%3DD.%26aulast%3DHe%26aufirst%3DH.%26aulast%3DSun%26aufirst%3DS.%26aulast%3DLovett%26aufirst%3DS.%2BT.%26aulast%3DXu%26aufirst%3DB.%26atitle%3DDiglycine%2520enables%2520rapid%2520intrabacterial%2520hydrolysis%2520for%2520activating%2520anbiotics%2520against%2520gram-negative%2520bacteria%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2019%26volume%3D58%26spage%3D10631%26epage%3D10634%26doi%3D10.1002%2Fanie.201905230" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dickson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koohmaraie, M.</span></span> <span> </span><span class="NLM_article-title">Cell surface charge characteristics and their relationship to bacterial attachment to meat surfaces</span>. <i>Appl. Environ. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">832</span>– <span class="NLM_lpage">836</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1128%2FAEM.55.4.832-836.1989" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=2499255" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A280%3ADyaL1M3ntFyrtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=1989&pages=832-836&author=J.+S.+Dicksonauthor=M.+Koohmaraie&title=Cell+surface+charge+characteristics+and+their+relationship+to+bacterial+attachment+to+meat+surfaces"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Cell surface charge characteristics and their relationship to bacterial attachment to meat surfaces</span></div><div class="casAuthors">Dickson J S; Koohmaraie M</div><div class="citationInfo"><span class="NLM_cas:title">Applied and environmental microbiology</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">832-6</span>
        ISSN:<span class="NLM_cas:issn">0099-2240</span>.
    </div><div class="casAbstract">Cell surface charge and hydrophobicity of Bacillus subtilis, Escherichia coli O157:H7, Listeria monocytogenes, Salmonella typhimurium, Serratia marcescens, Staphylococcus aureus, and Staphylococcus epidermidis were determined by hydrocarbon adherence, hydrophobic interaction, and electrostatic interaction chromatography.  Surface charge and hydrophobicity were compared with the initial attachment values and rates of attachment of the bacteria to meat surfaces.  There was a linear correlation between the relative negative charge on the bacterial cell surface and initial attachment to lean beef muscle (r2 = 0.885) and fat tissue (r2 = 0.777).  Hydrophobicity correlated well with attachment to fat tissue only.  The relative hydrophobicity of each bacterium was dependent on the specific method of determination, with wide variations noted between methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRkv7qQYFIC6svLc4ARHEmzfW6udTcc2ebcaRPvdSzCf7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaL1M3ntFyrtA%253D%253D&md5=6690ecaf36a7400ea36b00a26bb3c0ab</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1128%2FAEM.55.4.832-836.1989&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAEM.55.4.832-836.1989%26sid%3Dliteratum%253Aachs%26aulast%3DDickson%26aufirst%3DJ.%2BS.%26aulast%3DKoohmaraie%26aufirst%3DM.%26atitle%3DCell%2520surface%2520charge%2520characteristics%2520and%2520their%2520relationship%2520to%2520bacterial%2520attachment%2520to%2520meat%2520surfaces%26jtitle%3DAppl.%2520Environ.%2520Microbiol.%26date%3D1989%26volume%3D55%26spage%3D832%26epage%3D836" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamagata, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, B.</span></span> <span> </span><span class="NLM_article-title">Taurine Boosts Cellular Uptake of Smalld-Peptides for Enzyme-Instructed Intracellular Molecular Self-Assembly</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>137</i></span>,  <span class="NLM_fpage">10040</span>– <span class="NLM_lpage">10043</span>, <span class="refDoi"> DOI: 10.1021/jacs.5b06181</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.5b06181" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Kms77K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=137&publication_year=2015&pages=10040-10043&author=J.+Zhouauthor=X.+Duauthor=J.+Liauthor=N.+Yamagataauthor=B.+Xu&title=Taurine+Boosts+Cellular+Uptake+of+Smalld-Peptides+for+Enzyme-Instructed+Intracellular+Molecular+Self-Assembly&doi=10.1021%2Fjacs.5b06181"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Taurine Boosts Cellular Uptake of Small D-Peptides for Enzyme-Instructed Intracellular Molecular Self-Assembly</span></div><div class="casAuthors">Zhou, Jie; Du, Xuewen; Li, Jie; Yamagata, Natsuko; Xu, Bing</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">137</span>
        (<span class="NLM_cas:issue">32</span>),
    <span class="NLM_cas:pages">10040-10043</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Due to their biostability, D-peptides are emerging as an important mol. platform for biomedical applications.  Being proteolytically resistant, D-peptides lack interactions with endogenous transporters and hardly enter cells.  Here we show that taurine, a natural amino acid, drastically boosts the cellular uptake of small D-peptides in mammalian cells by >10-fold, from 118 μM (without conjugating taurine) to >1.6 mM (after conjugating taurine).  The uptake of a large amt. of the ester conjugate of taurine and D-peptide allows intracellular esterase to trigger intracellular self-assembly of the D-peptide deriv., further enhancing their cellular accumulation.  The study on the mechanism of the uptake reveals that the conjugates enter cells via both dynamin-dependent endocytosis and macropinocytosis, but likely not relying on taurine transporters.  Differing fundamentally from the pos. charged cell-penetrating peptides, the biocompatibility, stability, and simplicity of the enzyme-cleavable taurine motif promise new ways to promote the uptake of bioactive mols. for countering the action of efflux pump and contributing to intracellular mol. self-assembly.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjESPZzhIKYrVg90H21EOLACvtfcHk0lhH4RYlLlENZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Kms77K&md5=bda970b866249440a59db6b5448de40a</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fjacs.5b06181&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.5b06181%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DDu%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DYamagata%26aufirst%3DN.%26aulast%3DXu%26aufirst%3DB.%26atitle%3DTaurine%2520Boosts%2520Cellular%2520Uptake%2520of%2520Smalld-Peptides%2520for%2520Enzyme-Instructed%2520Intracellular%2520Molecular%2520Self-Assembly%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2015%26volume%3D137%26spage%3D10040%26epage%3D10043%26doi%3D10.1021%2Fjacs.5b06181" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Medina, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dinulescu, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, B.</span></span> <span> </span><span class="NLM_article-title">Enzyme-instructed intracellular molecular self-assembly to boost activity of cisplatin against drug-resistant ovarian cancer cells</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">13307</span>– <span class="NLM_lpage">13311</span>, <span class="refDoi"> DOI: 10.1002/anie.201507157</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1002%2Fanie.201507157" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsV2nsb7M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2015&pages=13307-13311&author=J.+Liauthor=Y.+Kuangauthor=J.+Shiauthor=J.+Zhouauthor=J.+E.+Medinaauthor=R.+Zhouauthor=D.+Yuanauthor=C.+Yangauthor=H.+Wangauthor=Z.+Yangauthor=J.+Liuauthor=D.+M.+Dinulescuauthor=B.+Xu&title=Enzyme-instructed+intracellular+molecular+self-assembly+to+boost+activity+of+cisplatin+against+drug-resistant+ovarian+cancer+cells&doi=10.1002%2Fanie.201507157"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Enzyme-Instructed Intracellular Molecular Self-Assembly to Boost Activity of Cisplatin against Drug-Resistant Ovarian Cancer Cells</span></div><div class="casAuthors">Li, Jie; Kuang, Yi; Shi, Junfeng; Zhou, Jie; Medina, Jamie E.; Zhou, Rong; Yuan, Dan; Yang, Cuihong; Wang, Huaimin; Yang, Zhimou; Liu, Jianfeng; Dinulescu, Daniela M.; Xu, Bing</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">45</span>),
    <span class="NLM_cas:pages">13307-13311</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Anticancer drug resistance demands innovative approaches that boost the activity of drugs against drug-resistant cancers without increasing the systemic toxicity.  Use of enzyme-instructed self-assembly (EISA) to generate intracellular supramol. assemblies that drastically boost the activity of cisplatin against drug-resistant ovarian cancer cells.  The authors design and synthesize small peptide precursors as the substrates of carboxylesterase (CES).  CES cleaves the ester bond pre-installed on the precursors to form the peptides that self-assemble in water to form nanofibers.  At the optimal concns., the precursors themselves are innocuous to cells, but they double or triple the activity of cisplatin against the drug-resistant ovarian cancer cells.  This work illustrates a simple, yet fundamental, new way to introduce non-cytotoxic components into combination therapies with cisplatin without increasing the systemic burden or side effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNfBq1jJrEr7Vg90H21EOLACvtfcHk0lhH4RYlLlENZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsV2nsb7M&md5=793f2c0ebf856fa2c21383b0b43975e8</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1002%2Fanie.201507157&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201507157%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DKuang%26aufirst%3DY.%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DMedina%26aufirst%3DJ.%2BE.%26aulast%3DZhou%26aufirst%3DR.%26aulast%3DYuan%26aufirst%3DD.%26aulast%3DYang%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DDinulescu%26aufirst%3DD.%2BM.%26aulast%3DXu%26aufirst%3DB.%26atitle%3DEnzyme-instructed%2520intracellular%2520molecular%2520self-assembly%2520to%2520boost%2520activity%2520of%2520cisplatin%2520against%2520drug-resistant%2520ovarian%2520cancer%2520cells%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2015%26volume%3D54%26spage%3D13307%26epage%3D13311%26doi%3D10.1002%2Fanie.201507157" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sulavik, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Houseweart, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cramer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiwani, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murgolo, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greene, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DiDomenico, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, G. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hare, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimer, G.</span></span> <span> </span><span class="NLM_article-title">Antibiotic susceptibility profiles of Escherichia coli strains lacking multidrug efflux pump genes</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">1126</span>– <span class="NLM_lpage">1136</span>, <span class="refDoi"> DOI: 10.1128/aac.45.4.1126-1136.2001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1128%2Faac.45.4.1126-1136.2001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=11257026" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BD3MXit1Omsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2001&pages=1126-1136&author=M.+C.+Sulavikauthor=C.+Houseweartauthor=C.+Cramerauthor=N.+Jiwaniauthor=N.+Murgoloauthor=J.+Greeneauthor=B.+DiDomenicoauthor=K.+J.+Shawauthor=G.+H.+Millerauthor=R.+Hareauthor=G.+Shimer&title=Antibiotic+susceptibility+profiles+of+Escherichia+coli+strains+lacking+multidrug+efflux+pump+genes&doi=10.1128%2Faac.45.4.1126-1136.2001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Antibiotic susceptibility profiles of Escherichia coli strains lacking multidrug efflux pump genes</span></div><div class="casAuthors">Sulavik, Mark C.; Houseweart, Chad; Cramer, Christina; Jiwani, Nilofer; Murgolo, Nicholas; Greene, Jonathan; DiDomenico, Beth; Shaw, Karen Joy; Miller, George H.; Hare, Roberta; Shimer, George</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1126-1136</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The contribution of seven known and nine predicted genes or operons assocd. with multidrug resistance to the susceptibility of Escherichia coli W3110 was assessed for 20 different classes of antimicrobial compds. that include antibiotics, antiseptics, detergents, and dyes.  Strains were constructed with deletions for genes in the major facilitator superfamily, the resistance nodulation-cell division family, the small multidrug resistance family, the ATP-binding cassette family, and outer membrane factors.  The agar diln. MICs of 35 compds. were detd. for strains with deletions for multidrug resistance (MDR) pumps.  Deletions in acrAB or tolC resulted in increased susceptibilities to the majority of compds. tested.  The remaining MDR pump gene deletions resulted in increased susceptibilities to far fewer compds.  The results identify which MDR pumps contribute to intrinsic resistance under the conditions tested and supply practical information useful for designing sensitive assay strains for cell-based screening of antibacterial compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2tjRRrBEqUrVg90H21EOLACvtfcHk0lhXFVhzMs2VGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXit1Omsrg%253D&md5=86d025f77f3e79d83d868f9c727785e0</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1128%2Faac.45.4.1126-1136.2001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252Faac.45.4.1126-1136.2001%26sid%3Dliteratum%253Aachs%26aulast%3DSulavik%26aufirst%3DM.%2BC.%26aulast%3DHouseweart%26aufirst%3DC.%26aulast%3DCramer%26aufirst%3DC.%26aulast%3DJiwani%26aufirst%3DN.%26aulast%3DMurgolo%26aufirst%3DN.%26aulast%3DGreene%26aufirst%3DJ.%26aulast%3DDiDomenico%26aufirst%3DB.%26aulast%3DShaw%26aufirst%3DK.%2BJ.%26aulast%3DMiller%26aufirst%3DG.%2BH.%26aulast%3DHare%26aufirst%3DR.%26aulast%3DShimer%26aufirst%3DG.%26atitle%3DAntibiotic%2520susceptibility%2520profiles%2520of%2520Escherichia%2520coli%2520strains%2520lacking%2520multidrug%2520efflux%2520pump%2520genes%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2001%26volume%3D45%26spage%3D1126%26epage%3D1136%26doi%3D10.1128%2Faac.45.4.1126-1136.2001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Greenwood, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Grady, F.</span></span> <span> </span><span class="NLM_article-title">Activity and interaction of trimethoprim and sulphamethoxazole against Escherichia coli</span>. <i>J. Clin. Pathol.</i> <span class="NLM_year" style="font-weight: bold;">1976</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">162</span>– <span class="NLM_lpage">166</span>, <span class="refDoi"> DOI: 10.1136/jcp.29.2.162</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1136%2Fjcp.29.2.162" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=777036" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADyaE28Xhs1Sjtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=1976&pages=162-166&author=D.+Greenwoodauthor=F.+O%E2%80%99Grady&title=Activity+and+interaction+of+trimethoprim+and+sulphamethoxazole+against+Escherichia+coli&doi=10.1136%2Fjcp.29.2.162"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Activity and interaction of trimethoprim and sulfamethoxazole against Escherichia coli</span></div><div class="casAuthors">Greenwood, D.; O'Grady, F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Pathology</span>
        (<span class="NLM_cas:date">1976</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">162-6</span>CODEN:
                <span class="NLM_cas:coden">JCPAAK</span>;
        ISSN:<span class="NLM_cas:issn">0021-9746</span>.
    </div><div class="casAbstract">Studies of the activity of trimethoprim lactate (I lactate) [23256-42-0] and sulfamethoxazole (II) [723-46-6] alone and in combination [8064-90-2] against a sensitive strain of Escherichia coli showed that in a static system in which the conditions of exposure of bacteria to drug resembled those of conventional min. inhibitory concn. (MIC) titrns., both I and II exhibited antibacterial activity at concns. well below the conventionally detd. MIC, but regrowth occurred at these concns. during the overnight incubation period due to the emergence of adaptively resistant bacteria.  Tests of combined drug action in the static turbidimetric system revealed even more synergic interaction than was apparent in conventional MIC tests, suggesting that an important component of overall synergic interaction is the mutual suppression of adaptive resistance to the other agent.  Studies in  an in vitro model which simulates the hydrokinetic features of the urinary bladder showed that concns. of I and II below the conventionally detd. MIC inhibited the growth even of extremely dense bacterial populations so long as the concn. of drug was maintained.  The response of cultures exposed to combinations of I and II in this system was so dominated by the effect of I that no synergic interaction with II was noted at concns. of the drugs which are achievable in urine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUr86Nh9pFcrVg90H21EOLACvtfcHk0lhXFVhzMs2VGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE28Xhs1Sjtbo%253D&md5=eb3124d10717d33e9b3e06df4854f4f0</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1136%2Fjcp.29.2.162&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1136%252Fjcp.29.2.162%26sid%3Dliteratum%253Aachs%26aulast%3DGreenwood%26aufirst%3DD.%26aulast%3DO%25E2%2580%2599Grady%26aufirst%3DF.%26atitle%3DActivity%2520and%2520interaction%2520of%2520trimethoprim%2520and%2520sulphamethoxazole%2520against%2520Escherichia%2520coli%26jtitle%3DJ.%2520Clin.%2520Pathol.%26date%3D1976%26volume%3D29%26spage%3D162%26epage%3D166%26doi%3D10.1136%2Fjcp.29.2.162" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Glassford, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagh, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velvadapu, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paul, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sandelin, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeBrosse, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klepacki, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Small, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacKerell, A. D.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrade, R. B.</span></span> <span> </span><span class="NLM_article-title">Desmethyl macrolides: synthesis and evaluation of 4-desmethyl telithromycin</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">1021</span>– <span class="NLM_lpage">1026</span>, <span class="refDoi"> DOI: 10.1021/ml5002097</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml5002097" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFKmtLzM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=1021-1026&author=I.+Glassfordauthor=M.+Leeauthor=B.+Waghauthor=V.+Velvadapuauthor=T.+Paulauthor=G.+Sandelinauthor=C.+DeBrosseauthor=D.+Klepackiauthor=M.+C.+Smallauthor=A.+D.+MacKerellauthor=R.+B.+Andrade&title=Desmethyl+macrolides%3A+synthesis+and+evaluation+of+4-desmethyl+telithromycin&doi=10.1021%2Fml5002097"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Desmethyl Macrolides: Synthesis and Evaluation of 4-Desmethyl Telithromycin</span></div><div class="casAuthors">Glassford, Ian; Lee, Miseon; Wagh, Bharat; Velvadapu, Venkata; Paul, Tapas; Sandelin, Gary; DeBrosse, Charles; Klepacki, Dorota; Small, Meagan C.; MacKerell, Alexander D.; Andrade, Rodrigo B.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1021-1026</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Novel sources of antibiotics are needed to address the serious threat of bacterial resistance.  Accordingly, we have launched a structure-based drug design program featuring a desmethylation strategy wherein Me groups have been replaced with hydrogens.  Herein we report the total synthesis, mol. modeling, and biol. evaluation of 4-desmethyl telithromycin (I), a novel desmethyl analog of the third-generation ketolide antibiotic telithromycin and our final analog in this series.  While I was found to be equipotent with telithromycin against wild-type bacteria, it was 4-fold less potent against the A2058G mutant.  These findings reveal that strategically replacing the C4-Me group with hydrogen (i.e., desmethylation) did not address this mechanism of resistance.  Throughout the desmethyl series, the sequential addn. of Me to the 14-membered macrolactone resulted in improved bioactivity.  Mol. modeling methods indicate that changes in conformational flexibility dominate the increased biol. activity; moreover, they reveal I adopts a different conformation once bound to the A2058G ribosome, thus impacting noncovalent interactions reflected in a lower MIC value.  Finally, fluorescence polarization expts. of I with E. coli ribosomes confirmed I is indeed binding the ribosome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5fEOWybrdp7Vg90H21EOLACvtfcHk0lhXFVhzMs2VGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFKmtLzM&md5=e2dcb2b85f8ee956cbce59076b08300a</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fml5002097&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml5002097%26sid%3Dliteratum%253Aachs%26aulast%3DGlassford%26aufirst%3DI.%26aulast%3DLee%26aufirst%3DM.%26aulast%3DWagh%26aufirst%3DB.%26aulast%3DVelvadapu%26aufirst%3DV.%26aulast%3DPaul%26aufirst%3DT.%26aulast%3DSandelin%26aufirst%3DG.%26aulast%3DDeBrosse%26aufirst%3DC.%26aulast%3DKlepacki%26aufirst%3DD.%26aulast%3DSmall%26aufirst%3DM.%2BC.%26aulast%3DMacKerell%26aufirst%3DA.%2BD.%26aulast%3DAndrade%26aufirst%3DR.%2BB.%26atitle%3DDesmethyl%2520macrolides%253A%2520synthesis%2520and%2520evaluation%2520of%25204-desmethyl%2520telithromycin%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D5%26spage%3D1021%26epage%3D1026%26doi%3D10.1021%2Fml5002097" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yogev, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolling, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, T.</span></span> <span> </span><span class="NLM_article-title">Pharmacokinetic comparison of intravenous and oral chloramphenicol in patients with Haemophilus influenzae meningitis</span>. <i>Pediatrics</i> <span class="NLM_year" style="font-weight: bold;">1981</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">656</span>– <span class="NLM_lpage">660</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=6973130" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A280%3ADyaL3M3ks1Gqtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=1981&pages=656-660&author=R.+Yogevauthor=W.+M.+Kollingauthor=T.+Williams&title=Pharmacokinetic+comparison+of+intravenous+and+oral+chloramphenicol+in+patients+with+Haemophilus+influenzae+meningitis"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacokinetic comparison of intravenous and oral chloramphenicol in patients with Haemophilus influenzae meningitis</span></div><div class="casAuthors">Yogev R; Kolling W M; Williams T</div><div class="citationInfo"><span class="NLM_cas:title">Pediatrics</span>
        (<span class="NLM_cas:date">1981</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">656-60</span>
        ISSN:<span class="NLM_cas:issn">0031-4005</span>.
    </div><div class="casAbstract">The pharmacokinetics of chloramphenicol following intravenous and oral administration were studied in 14 infants with Haemophilus influenzae meningitis.  Following five days of treatment with intravenous chloramphenicol (100 mg/kg/day every six hours), oral chloramphenicol was substituted at the same dose.  Multiple serum levels of chloramphenicol were determined after an intravenous dose on day 4 and after an oral dose on day 10.  CSF levels were measured six hours after intravenous or oral chloramphenicol dose on those days (CSF trough).  Following intravenous administration, the mean peak serum level of 15.0 micrograms/ml was reached at 45 minutes.  In comparison, after oral chloramphenicol in the same dosage, the mean peak serum level of 18.5 micrograms/ml was achieved at two to three hours.  The mean serum half-life of the drug (6.5 hours) was significantly longer after oral administration than after intravenous chloramphenicol (4.0 hours) (P less than .001).  The increased serum half-life following orally administered chloramphenicol was occasionally associated with drug accumulation.  In addition, mean trough CSF levels were somewhat higher when the patient received oral medication (6.6 micrograms/ml) compared to intravenous administration (4.2 micrograms/ml) (P less than .001).  For any treatment regimen for H influenzae meningitis that includes a period of oral chloramphenicol therapy the patient should be hospitalized to ensure compliance.  Because of the wide range of individual variation in serum half-life that may result in accumulation, periodic monitoring of serum chloramphenicol levels is also recommended.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTfTBzE2TvtT02yPDyEFiigfW6udTcc2ebwcc7PK4AmAbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaL3M3ks1Gqtw%253D%253D&md5=c833524469c559a0fc11de532f0451f8</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DYogev%26aufirst%3DR.%26aulast%3DKolling%26aufirst%3DW.%2BM.%26aulast%3DWilliams%26aufirst%3DT.%26atitle%3DPharmacokinetic%2520comparison%2520of%2520intravenous%2520and%2520oral%2520chloramphenicol%2520in%2520patients%2520with%2520Haemophilus%2520influenzae%2520meningitis%26jtitle%3DPediatrics%26date%3D1981%26volume%3D67%26spage%3D656%26epage%3D660" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Medina, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dinulescu, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, B.</span></span> <span> </span><span class="NLM_article-title">Selectively inducing cancer cell death by intracellular enzyme-instructed self-assembly (EISA) of dipeptide derivatives</span>. <i>Adv. Healthcare Mater.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">1601400</span>, <span class="refDoi"> DOI: 10.1002/adhm.201601400</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1002%2Fadhm.201601400" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2017&pages=1601400&author=J.+Liauthor=J.+Shiauthor=J.+E.+Medinaauthor=J.+Zhouauthor=X.+Duauthor=H.+Wangauthor=C.+Yangauthor=J.+Liuauthor=Z.+Yangauthor=D.+M.+Dinulescuauthor=B.+Xu&title=Selectively+inducing+cancer+cell+death+by+intracellular+enzyme-instructed+self-assembly+%28EISA%29+of+dipeptide+derivatives&doi=10.1002%2Fadhm.201601400"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1002%2Fadhm.201601400&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fadhm.201601400%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DMedina%26aufirst%3DJ.%2BE.%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DDu%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DYang%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DDinulescu%26aufirst%3DD.%2BM.%26aulast%3DXu%26aufirst%3DB.%26atitle%3DSelectively%2520inducing%2520cancer%2520cell%2520death%2520by%2520intracellular%2520enzyme-instructed%2520self-assembly%2520%2528EISA%2529%2520of%2520dipeptide%2520derivatives%26jtitle%3DAdv.%2520Healthcare%2520Mater.%26date%3D2017%26volume%3D6%26spage%3D1601400%26doi%3D10.1002%2Fadhm.201601400" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Prabhala, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aduri, N. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaheen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iqbal, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brasen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gajhede, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rahman, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirza, O.</span></span> <span> </span><span class="NLM_article-title">The prototypical proton-coupled oligopeptide transporter YdgR fromEscherichia colifacilitates chloramphenicol uptake into bacterial cells</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>293</i></span>,  <span class="NLM_fpage">1007</span>– <span class="NLM_lpage">1017</span>, <span class="refDoi"> DOI: 10.1074/jbc.m117.805960</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1074%2Fjbc.m117.805960" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=29150447" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1yjsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=293&publication_year=2018&pages=1007-1017&author=B.+K.+Prabhalaauthor=N.+G.+Aduriauthor=N.+Sharmaauthor=A.+Shaheenauthor=A.+Sharmaauthor=M.+Iqbalauthor=P.+R.+Hansenauthor=C.+Brasenauthor=M.+Gajhedeauthor=M.+Rahmanauthor=O.+Mirza&title=The+prototypical+proton-coupled+oligopeptide+transporter+YdgR+fromEscherichia+colifacilitates+chloramphenicol+uptake+into+bacterial+cells&doi=10.1074%2Fjbc.m117.805960"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">The prototypical proton-coupled oligopeptide transporter YdgR from Escherichia coli facilitates chloramphenicol uptake into bacterial cells</span></div><div class="casAuthors">Prabhala, Bala K.; Aduri, Nanda G.; Sharma, Neha; Shaheen, Aqsa; Sharma, Arpan; Iqbal, Mazhar; Hansen, Paul R.; Brasen, Christoffer; Gajhede, Michael; Rahman, Moazur; Mirza, Osman</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">293</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1007-1017</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Chloramphenicol (Cam) is a broad-spectrum antibiotic used to combat bacterial infections in humans and animals.  Cam export from bacterial cells is one of the mechanisms by which pathogens resist Cam's antibacterial effects, and several different proteins are known to facilitate this process.  However, to date no report exists on any specific transport protein that facilitates Cam uptake.  The proton-coupled oligopeptide transporter (POT) YdgR from Escherichia coli is a prototypical member of the POT family, functioning in proton-coupled uptake of di- and tripeptides.  By following bacterial growth and conducting LC-MS-based assays we show here that YdgR facilitates Cam uptake.  Some YdgR variants displaying reduced peptide uptake also exhibited reduced Cam uptake, indicating that peptides and Cam bind YdgR at similar regions.  Homol. modeling of YdgR, Cam docking, and mutational studies suggested a binding mode that resembles that of Cam binding to the multidrug resistance transporter MdfA.  To our knowledge, this is the first report of Cam uptake into bacterial cells mediated by a specific transporter protein.  Our findings suggest a specific bacterial transporter for drug uptake that might be targeted to promote greater antibiotic influx to increase cytoplasmic antibiotic concn. for enhanced cytotoxicity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqc9zM2x0kV67Vg90H21EOLACvtfcHk0lhGxyhYyauGBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1yjsro%253D&md5=4c6c9a2275886ca46238b581580da8fa</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1074%2Fjbc.m117.805960&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.m117.805960%26sid%3Dliteratum%253Aachs%26aulast%3DPrabhala%26aufirst%3DB.%2BK.%26aulast%3DAduri%26aufirst%3DN.%2BG.%26aulast%3DSharma%26aufirst%3DN.%26aulast%3DShaheen%26aufirst%3DA.%26aulast%3DSharma%26aufirst%3DA.%26aulast%3DIqbal%26aufirst%3DM.%26aulast%3DHansen%26aufirst%3DP.%2BR.%26aulast%3DBrasen%26aufirst%3DC.%26aulast%3DGajhede%26aufirst%3DM.%26aulast%3DRahman%26aufirst%3DM.%26aulast%3DMirza%26aufirst%3DO.%26atitle%3DThe%2520prototypical%2520proton-coupled%2520oligopeptide%2520transporter%2520YdgR%2520fromEscherichia%2520colifacilitates%2520chloramphenicol%2520uptake%2520into%2520bacterial%2520cells%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2018%26volume%3D293%26spage%3D1007%26epage%3D1017%26doi%3D10.1074%2Fjbc.m117.805960" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Buchanan, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venkatramani, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Esser, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palnitkar, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chakraborty, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Helm, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deisenhofer, J.</span></span> <span> </span><span class="NLM_article-title">Crystal structure of the outer membrane active transporter FepA from Escherichia coli</span>. <i>Nat. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">56</span>– <span class="NLM_lpage">63</span>, <span class="refDoi"> DOI: 10.1038/4931</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1038%2F4931" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=9886293" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADyaK1MXjvVOjuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1999&pages=56-63&author=S.+K.+Buchananauthor=B.+S.+Smithauthor=L.+Venkatramaniauthor=D.+Xiaauthor=L.+Esserauthor=M.+Palnitkarauthor=R.+Chakrabortyauthor=D.+van+der+Helmauthor=J.+Deisenhofer&title=Crystal+structure+of+the+outer+membrane+active+transporter+FepA+from+Escherichia+coli&doi=10.1038%2F4931"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of the outer membrane active transporter FepA from Escherichia coli</span></div><div class="casAuthors">Buchanan, Susan K.; Smith, Barbara S.; Venkatramani, Lalitha; Xia, Di; Esser, Lothar; Palnitkar, Maya; Chakraborty, Ranjan; Van Der Helm, Dick; Deisenhofer, Johann</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural Biology</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">56-63</span>CODEN:
                <span class="NLM_cas:coden">NSBIEW</span>;
        ISSN:<span class="NLM_cas:issn">1072-8368</span>.
    
            (<span class="NLM_cas:orgname">Nature America</span>)
        </div><div class="casAbstract">Integral outer membrane receptors for iron chelates and vitamin B12 carry out specific ligand transport against a concn. gradient.  Energy for active transport is obtained from the proton-motive force of the inner membrane through phys. interaction with TonB-ExbB-ExbD, an inner membrane complex.  Here we report the crystal structure of an active transport, outer membrane receptor at 2.4 A resoln.  Two distinct functional domains are revealed: (i) a 22-stranded β-barrel that spans the outer membrane and contains large extracellular loops which appear to function in ligand binding; and (ii) a globular N-terminal domain that folds into the barrel pore, inhibiting access to the periplasm and contributing two addnl. loops for potential ligand binding.  These loops could provide a signaling pathway between the processes of ligand recognition and TonB-mediated transport.  The blockage of the pore suggests that the N-terminal domain must undergo a conformational rearrangement to allow ligand transport into the periplasm.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoU1RPTePfa67Vg90H21EOLACvtfcHk0lhvVW3qPxmMsw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXjvVOjuw%253D%253D&md5=4cb575c15e4c828137778c0d2186e0e3</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1038%2F4931&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F4931%26sid%3Dliteratum%253Aachs%26aulast%3DBuchanan%26aufirst%3DS.%2BK.%26aulast%3DSmith%26aufirst%3DB.%2BS.%26aulast%3DVenkatramani%26aufirst%3DL.%26aulast%3DXia%26aufirst%3DD.%26aulast%3DEsser%26aufirst%3DL.%26aulast%3DPalnitkar%26aufirst%3DM.%26aulast%3DChakraborty%26aufirst%3DR.%26aulast%3Dvan%2Bder%2BHelm%26aufirst%3DD.%26aulast%3DDeisenhofer%26aufirst%3DJ.%26atitle%3DCrystal%2520structure%2520of%2520the%2520outer%2520membrane%2520active%2520transporter%2520FepA%2520from%2520Escherichia%2520coli%26jtitle%3DNat.%2520Struct.%2520Biol.%26date%3D1999%26volume%3D6%26spage%3D56%26epage%3D63%26doi%3D10.1038%2F4931" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Masi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pagès, J.-M.</span></span> <span> </span><span class="NLM_article-title">Structure, function and regulation of outer membrane proteins involved in drug transport in enterobactericeae: the OmpF/C - TolC case</span>. <i>Open Microbiol. J.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">22</span>– <span class="NLM_lpage">33</span>, <span class="refDoi"> DOI: 10.2174/1874285801307010022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.2174%2F1874285801307010022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=23569467" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFGqsLrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2013&pages=22-33&author=M.+Masiauthor=J.-M.+Pag%C3%A8s&title=Structure%2C+function+and+regulation+of+outer+membrane+proteins+involved+in+drug+transport+in+enterobactericeae%3A+the+OmpF%2FC+-+TolC+case&doi=10.2174%2F1874285801307010022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Structure, function and regulation of outer membrane proteins involved in drug transport in Enterobactericeae: the OmpF/C - TolC case</span></div><div class="casAuthors">Masi, Muriel; Pages, Jean-Marie</div><div class="citationInfo"><span class="NLM_cas:title">Open Microbiology Journal</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">22-33</span>CODEN:
                <span class="NLM_cas:coden">OMJPAO</span>;
        ISSN:<span class="NLM_cas:issn">1874-2858</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Antibiotic translocation across membranes of Gram-neg. bacteria is a key step for the activity on their specific intracellular targets.  Resistant bacteria control their membrane permeability as a first line of defense to protect themselves against external toxic compds. such as antibiotics and biocides.  On one hand, resistance to small hydrophilic antibiotics such as β-lactams and fluoroquinolones frequently results from the "closing" of their way in: the general outer membrane porins.  On the other hand, an effective way out for a wide range of antibiotics is provided by TolC-like proteins, which are outer membrane components of multidrug efflux pumps.  Accordingly, altered membrane permeability, including porin modifications and/or efflux pumps' overexpression, is always assocd. to multidrug resistance (MDR) in a no. of clin. isolates.  Several recent studies have highlighted our current understanding of porins/TolC structures and functions in Enterobacteriaceae.  Here, we review the transport of antibiotics through the OmpF/C general porins and the TolC-like channels with regards to recent data on their structure, function, assembly, regulation and contribution to bacterial resistance.  Because MDR strains have evolved global strategies to identify and fight our antibiotic arsenal, it is important to constantly update our global knowledge on antibiotic transport.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVvvxKCgz2e7Vg90H21EOLACvtfcHk0lhvVW3qPxmMsw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFGqsLrJ&md5=6fc21e8bbbdff1a67f441b7515dc7f3f</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.2174%2F1874285801307010022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1874285801307010022%26sid%3Dliteratum%253Aachs%26aulast%3DMasi%26aufirst%3DM.%26aulast%3DPag%25C3%25A8s%26aufirst%3DJ.-M.%26atitle%3DStructure%252C%2520function%2520and%2520regulation%2520of%2520outer%2520membrane%2520proteins%2520involved%2520in%2520drug%2520transport%2520in%2520enterobactericeae%253A%2520the%2520OmpF%252FC%2520-%2520TolC%2520case%26jtitle%3DOpen%2520Microbiol.%2520J.%26date%3D2013%26volume%3D7%26spage%3D22%26epage%3D33%26doi%3D10.2174%2F1874285801307010022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nishino, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamaguchi, A.</span></span> <span> </span><span class="NLM_article-title">Analysis of a complete library of putative drug transporter genes in Escherichia coli</span>. <i>J. Bacteriol.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>183</i></span>,  <span class="NLM_fpage">5803</span>– <span class="NLM_lpage">5812</span>, <span class="refDoi"> DOI: 10.1128/jb.183.20.5803-5812.2001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1128%2Fjb.183.20.5803-5812.2001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=11566977" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BD3MXntlanurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=183&publication_year=2001&pages=5803-5812&author=K.+Nishinoauthor=A.+Yamaguchi&title=Analysis+of+a+complete+library+of+putative+drug+transporter+genes+in+Escherichia+coli&doi=10.1128%2Fjb.183.20.5803-5812.2001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Analysis of a complete library of putative drug transporter genes in Escherichia coli</span></div><div class="casAuthors">Nishino, Kunihiko; Yamaguchi, Akihito</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Bacteriology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">183</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">5803-5812</span>CODEN:
                <span class="NLM_cas:coden">JOBAAY</span>;
        ISSN:<span class="NLM_cas:issn">0021-9193</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The complete sequencing of bacterial genomes has revealed a large no. of drug transporter genes.  In Escherichia coli, there are 37 open reading frames (ORFs) assumed to be drug transporter genes on the basis of sequence similarities, although the transport capabilities of most of them have not been established yet.  The authors cloned all 37 putative drug transporter genes in E. coli and investigated their drug resistance phenotypes using an E. coli drug-sensitive mutant as a host.  E. coli cells transformed with a plasmid carrying one of 20 ORFs, i.e., fsr, mdfA, yceE, yceL, bcr, emrKY, emrAB, emrD, yidY, yjiO, ydhE, acrAB, cusA (formerly ybdE), yegMNO, acrD, acrEF, yhiUV, emrE, ydgFE, and ybjYZ, exhibited increased resistance to some of the 26 representative antimicrobial agents and chem. compds. tested in this study.  Of these 20 ORFs, cusA, yegMNO, ydgFE, yceE, yceL, yidY, and ybjYZ are novel drug resistance genes.  The fsr, bcr, yjiO, ydhE, acrD, and yhiUV genes gave broader resistance spectra than previously reported.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHqUeniJrxvLVg90H21EOLACvtfcHk0lhvVW3qPxmMsw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXntlanurs%253D&md5=0076182ce947886a3f4f5ec915cbf08a</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1128%2Fjb.183.20.5803-5812.2001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252Fjb.183.20.5803-5812.2001%26sid%3Dliteratum%253Aachs%26aulast%3DNishino%26aufirst%3DK.%26aulast%3DYamaguchi%26aufirst%3DA.%26atitle%3DAnalysis%2520of%2520a%2520complete%2520library%2520of%2520putative%2520drug%2520transporter%2520genes%2520in%2520Escherichia%2520coli%26jtitle%3DJ.%2520Bacteriol.%26date%3D2001%26volume%3D183%26spage%3D5803%26epage%3D5812%26doi%3D10.1128%2Fjb.183.20.5803-5812.2001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tikhonova, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zgurskaya, H. I.</span></span> <span> </span><span class="NLM_article-title">AcrA, AcrB, and TolC ofEscherichia coliForm a Stable Intermembrane Multidrug Efflux Complex</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>279</i></span>,  <span class="NLM_fpage">32116</span>– <span class="NLM_lpage">32124</span>, <span class="refDoi"> DOI: 10.1074/jbc.m402230200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1074%2Fjbc.m402230200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=15155734" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmtVCiu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=279&publication_year=2004&pages=32116-32124&author=E.+B.+Tikhonovaauthor=H.+I.+Zgurskaya&title=AcrA%2C+AcrB%2C+and+TolC+ofEscherichia+coliForm+a+Stable+Intermembrane+Multidrug+Efflux+Complex&doi=10.1074%2Fjbc.m402230200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">AcrA, AcrB, and TolC of Escherichia coli form a stable intermembrane multidrug efflux complex</span></div><div class="casAuthors">Tikhonova, Elena B.; Zgurskaya, Helen I.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">279</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">32116-32124</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Many transporters of Gram-neg. bacteria involved in the extracellular secretion of proteins and the efflux of toxic mols. operate by forming intermembrane complexes.  These complexes are proposed to span both inner and outer membranes and create a bridge across the periplasm.  In this study, we analyzed interactions between the inner and outer membrane components of the tri-partite multidrug efflux pump AcrAB-TolC from Escherichia coli.  We found that, once assembled, the intermembrane AcrAB-TolC complex is stable during the sepn. of the inner and outer membranes and subsequent purifn.  All three components of the complex co-purify when the affinity tag is attached to either of the proteins suggesting bi-partite interactions between AcrA, AcrB, and TolC.  We show that antibiotics, the substrates of AcrAB-TolC, stabilize interactions within the complex.  However, the formation of the AcrAB-TolC complex does not require an input of energy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcRm1Wtm5dL7Vg90H21EOLACvtfcHk0lhvVW3qPxmMsw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmtVCiu7o%253D&md5=a8969321f1326c367b6f16d613293c73</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1074%2Fjbc.m402230200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.m402230200%26sid%3Dliteratum%253Aachs%26aulast%3DTikhonova%26aufirst%3DE.%2BB.%26aulast%3DZgurskaya%26aufirst%3DH.%2BI.%26atitle%3DAcrA%252C%2520AcrB%252C%2520and%2520TolC%2520ofEscherichia%2520coliForm%2520a%2520Stable%2520Intermembrane%2520Multidrug%2520Efflux%2520Complex%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2004%26volume%3D279%26spage%3D32116%26epage%3D32124%26doi%3D10.1074%2Fjbc.m402230200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.-Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plésiat, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikaido, H.</span></span> <span> </span><span class="NLM_article-title">The challenge of efflux-mediated antibiotic resistance in Gram-negative bacteria</span>. <i>Clin. Microbiol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">337</span>– <span class="NLM_lpage">418</span>, <span class="refDoi"> DOI: 10.1128/cmr.00117-14</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1128%2FCMR.00117-14" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=25788514" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BC1cXltlyhs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2015&pages=337-418&author=X.-Z.+Liauthor=P.+Pl%C3%A9siatauthor=H.+Nikaido&title=The+challenge+of+efflux-mediated+antibiotic+resistance+in+Gram-negative+bacteria&doi=10.1128%2Fcmr.00117-14"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">The challenge of efflux-mediated antibiotic resistance in Gram-negative bacteria</span></div><div class="casAuthors">Li, Xian-Zhi; Plesiat, Patrick; Nikaido, Hiroshi</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Microbiology Reviews</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">337-418</span>CODEN:
                <span class="NLM_cas:coden">CMIREX</span>;
        ISSN:<span class="NLM_cas:issn">1098-6618</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The global emergence of multidrug-resistant Gram-neg. bacteria is a growing threat to antibiotic therapy.  The chromosomally encoded drug efflux mechanisms that are ubiquitous in these bacteria greatly contribute to antibiotic resistance and present a major challenge for antibiotic development.  Multidrug pumps, particularly those represented by the clin. relevant AcrAB-TolC and Mex pumps of the resistance-nodulation-division (RND) superfamily, not only mediate intrinsic and acquired multidrug resistance (MDR) but also are involved in other functions, including the bacterial stress response and pathogenicity.  Addnl., efflux pumps interact synergistically with other resistance mechanisms (e.g., with the outer membrane permeability barrier) to increase resistance levels.  Since the discovery of RND pumps in the early 1990s, remarkable scientific and technol. advances have allowed for an in-depth understanding of the structural and biochem. basis, substrate profiles, mol. regulation, and inhibition of MDR pumps.  However, the development of clin. useful efflux pump inhibitors and/or new antibiotics that can bypass pump effects continues to be a challenge.  Plasmid-borne efflux pump genes (including those for RND pumps) have increasingly been identified.  This article highlights the recent progress obtained for organisms of clin. significance, together with methodol. considerations for the characterization of MDR pumps.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8mBa4pi9DV7Vg90H21EOLACvtfcHk0lgQuqcBdZrvGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXltlyhs7o%253D&md5=f481bdc9d3813f008c4c69502e9c9639</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1128%2FCMR.00117-14&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FCMR.00117-14%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DX.-Z.%26aulast%3DPl%25C3%25A9siat%26aufirst%3DP.%26aulast%3DNikaido%26aufirst%3DH.%26atitle%3DThe%2520challenge%2520of%2520efflux-mediated%2520antibiotic%2520resistance%2520in%2520Gram-negative%2520bacteria%26jtitle%3DClin.%2520Microbiol.%2520Rev.%26date%3D2015%26volume%3D28%26spage%3D337%26epage%3D418%26doi%3D10.1128%2Fcmr.00117-14" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Godinho, L. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reis, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tepper, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poelarends, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quax, W. J.</span></span> <span> </span><span class="NLM_article-title">Discovery of an Escherichia coli esterase with high activity and enantioselectivity toward 1,2-O-isopropylideneglycerol esters</span>. <i>Appl. Environ. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>77</i></span>,  <span class="NLM_fpage">6094</span>– <span class="NLM_lpage">6099</span>, <span class="refDoi"> DOI: 10.1128/aem.05122-11</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1128%2Faem.05122-11" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=21764964" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1OjurnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2011&pages=6094-6099&author=L.+F.+Godinhoauthor=C.+R.+Reisauthor=P.+G.+Tepperauthor=G.+J.+Poelarendsauthor=W.+J.+Quax&title=Discovery+of+an+Escherichia+coli+esterase+with+high+activity+and+enantioselectivity+toward+1%2C2-O-isopropylideneglycerol+esters&doi=10.1128%2Faem.05122-11"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of an Escherichia coli esterase with high activity and enantioselectivity toward 1,2-O-isopropylideneglycerol esters</span></div><div class="casAuthors">Godinho, Luis F.; Reis, Carlos R.; Tepper, Pieter G.; Poelarends, Gerrit J.; Quax, Wim J.</div><div class="citationInfo"><span class="NLM_cas:title">Applied and Environmental Microbiology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">77</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">6094-6099</span>CODEN:
                <span class="NLM_cas:coden">AEMIDF</span>;
        ISSN:<span class="NLM_cas:issn">0099-2240</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Escherichia coli has been widely used as an expression host for the identification of desired biocatalysts through screening or selection assays.  We have previously used E. coli in growth selection and screening assays for identification of Bacillus subtilis lipase variants (located in the periplasm) with improved activity and enantioselectivity toward 1,2-O-isopropylideneglycerol (IPG) esters.  In the course of these studies, we discovered that E. coli itself exhibits significant cytoplasmic esterase activity toward IPG esters.  In order to identify the enzyme (or enzymes) responsible for this esterase activity, we analyzed eight E. coli knockout strains, in which single esterase genes were deleted, for their ability to hydrolyze IPG butyrate.  This approach led to the identification of esterase YbfF as the major E. coli enzyme responsible for the hydrolytic activity toward IPG esters.  The gene coding for YbfF was cloned and overexpressed in E. coli, and the corresponding protein was purified and characterized for its biocatalytic performance.  YbfF displays a high level of activity toward IPG butyrate and IPG caprylate and prefers the R-enantiomer of these substrates, producing the S-enantiomer of the IPG product with high enantiomeric excess (72 to 94% ee).  The enantioselectivity of YbfF for IPG caprylate (E = 40) could be significantly enhanced when using DMF or DMSO as cosolvents in kinetic resoln. expts.  The enzyme also shows high enantioselectivity toward 1-phenylethyl acetate (E ≥ 200), giving the chiral product (R)-1-phenylethanol with >99% ee.  The high activity and enantioselectivity of YbfF make it an attractive enzyme for org. synthesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMEA7m3V6QbbVg90H21EOLACvtfcHk0lgQuqcBdZrvGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1OjurnE&md5=b9fa66958cd4460a4b68d5babbb265e5</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1128%2Faem.05122-11&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252Faem.05122-11%26sid%3Dliteratum%253Aachs%26aulast%3DGodinho%26aufirst%3DL.%2BF.%26aulast%3DReis%26aufirst%3DC.%2BR.%26aulast%3DTepper%26aufirst%3DP.%2BG.%26aulast%3DPoelarends%26aufirst%3DG.%2BJ.%26aulast%3DQuax%26aufirst%3DW.%2BJ.%26atitle%3DDiscovery%2520of%2520an%2520Escherichia%2520coli%2520esterase%2520with%2520high%2520activity%2520and%2520enantioselectivity%2520toward%25201%252C2-O-isopropylideneglycerol%2520esters%26jtitle%3DAppl.%2520Environ.%2520Microbiol.%26date%3D2011%26volume%3D77%26spage%3D6094%26epage%3D6099%26doi%3D10.1128%2Faem.05122-11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tomczyk, N. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nettleship, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baxter, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crichton, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Webster, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campopiano, D. J.</span></span> <span> </span><span class="NLM_article-title">Purification and characterisation of the BIOH protein from the biotin biosynthetic pathway</span>. <i>FEBS Lett.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>513</i></span>,  <span class="NLM_fpage">299</span>– <span class="NLM_lpage">304</span>, <span class="refDoi"> DOI: 10.1016/s0014-5793(02)02342-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1016%2Fs0014-5793%2802%2902342-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=11904168" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BD38XitFejsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=513&publication_year=2002&pages=299-304&author=N.+H.+Tomczykauthor=J.+E.+Nettleshipauthor=R.+L.+Baxterauthor=H.+J.+Crichtonauthor=S.+P.+Websterauthor=D.+J.+Campopiano&title=Purification+and+characterisation+of+the+BIOH+protein+from+the+biotin+biosynthetic+pathway&doi=10.1016%2Fs0014-5793%2802%2902342-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Purification and characterisation of the BIOH protein from the biotin biosynthetic pathway</span></div><div class="casAuthors">Tomczyk, Nicholas H.; Nettleship, Joanne E.; Baxter, Robert L.; Crichton, Hilary J.; Webster, Scott P.; Campopiano, Dominic J.</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Letters</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">513</span>
        (<span class="NLM_cas:issue">2,3</span>),
    <span class="NLM_cas:pages">299-304</span>CODEN:
                <span class="NLM_cas:coden">FEBLAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-5793</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">Conversion of pimeloyl-CoA (CoA) to biotin in Escherichia coli requires at least four enzymes encoded by genes in the bio operon.  One gene, bioH, which is not present in the bioABFCD operon, is required for the synthesis of pimeloyl-CoA but its exact role in formation of this intermediate is unknown.  To investigate this further, we have overexpressed and purified the bioH gene products from both E. coli (BIOH EC) and Neisseria meningitis (BIOH NM) in E. coli.  When purified BIOH was incubated with excess CoA and analyzed by electrospray mass spectrometry a species of mass corresponding to a BIOH:CoA complex was obsd.  Mutation of a conserved serine residue to alanine (BIOH EC S82A) did not prevent CoA binding.  This is the first report of the purifn. of BIOH and the observation of a small mol. bound to the protein provides clues to its role in pimeloyl-CoA synthesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6_E-LdXBQmbVg90H21EOLACvtfcHk0lgQuqcBdZrvGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XitFejsb8%253D&md5=09d7a34f7d89b010211b063e51c7824d</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2Fs0014-5793%2802%2902342-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fs0014-5793%252802%252902342-6%26sid%3Dliteratum%253Aachs%26aulast%3DTomczyk%26aufirst%3DN.%2BH.%26aulast%3DNettleship%26aufirst%3DJ.%2BE.%26aulast%3DBaxter%26aufirst%3DR.%2BL.%26aulast%3DCrichton%26aufirst%3DH.%2BJ.%26aulast%3DWebster%26aufirst%3DS.%2BP.%26aulast%3DCampopiano%26aufirst%3DD.%2BJ.%26atitle%3DPurification%2520and%2520characterisation%2520of%2520the%2520BIOH%2520protein%2520from%2520the%2520biotin%2520biosynthetic%2520pathway%26jtitle%3DFEBS%2520Lett.%26date%3D2002%26volume%3D513%26spage%3D299%26epage%3D304%26doi%3D10.1016%2Fs0014-5793%2802%2902342-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kuznetsova, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Proudfoot, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanders, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reinking, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savchenko, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yakunin, A.</span></span> <span> </span><span class="NLM_article-title">Enzyme genomics: Application of general enzymatic screens to discover new enzymes</span>. <i>FEMS Microbiol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">263</span>– <span class="NLM_lpage">279</span>, <span class="refDoi"> DOI: 10.1016/j.fmrre.2004.12.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1016%2Fj.fmrre.2004.12.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=15808744" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BD2MXivV2qurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2005&pages=263-279&author=E.+Kuznetsovaauthor=M.+Proudfootauthor=S.+Sandersauthor=J.+Reinkingauthor=A.+Savchenkoauthor=C.+Arrowsmithauthor=A.+Edwardsauthor=A.+Yakunin&title=Enzyme+genomics%3A+Application+of+general+enzymatic+screens+to+discover+new+enzymes&doi=10.1016%2Fj.fmrre.2004.12.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Enzyme genomics: Application of general enzymatic screens to discover new enzymes</span></div><div class="casAuthors">Kuznetsova, Ekaterina; Proudfoot, Michael; Sanders, Stephen A.; Reinking, Jeffrey; Savchenko, Alexei; Arrowsmith, Cheryl H.; Edwards, Aled M.; Yakunin, Alexander F.</div><div class="citationInfo"><span class="NLM_cas:title">FEMS Microbiology Reviews</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">263-279</span>CODEN:
                <span class="NLM_cas:coden">FMREE4</span>;
        ISSN:<span class="NLM_cas:issn">0168-6445</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  In all sequenced genomes, a large fraction of predicted genes encodes proteins of unknown biochem. function and up to 15% of the genes with "known" function are mis-annotated.  Several global approaches are routinely employed to predict function, including sophisticated sequence anal., gene expression, protein interaction, and protein structure.  In the first coupling of genomics and enzymol., Phizicky and colleagues undertook a screen for specific enzymes using large pools of partially purified proteins and specific enzymic assays.  Here we present an overview of the further developments of this approach, which involve the use of general enzymic assays to screen individually purified proteins for enzymic activity.  The assays have relaxed substrate specificity and are designed to identify the subclass or sub-subclasses of enzymes (phosphatase, phosphodiesterase/nuclease, protease, esterase, dehydrogenase, and oxidase) to which the unknown protein belongs.  Further biochem. characterization of proteins can be facilitated by the application of secondary screens with natural substrates (substrate profiling).  We demonstrate here the feasibility and merits of this approach for hydrolases and oxidoreductases, two very broad and important classes of enzymes.  Application of general enzymic screens and substrate profiling can greatly speed up the identification of biochem. function of unknown proteins and the exptl. verification of functional predictions produced by other functional genomics approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrd9B3Mv7L_RbVg90H21EOLACvtfcHk0lidJ-etib-aEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXivV2qurs%253D&md5=27f821afaa58bacb8add2ba0ad471d6f</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.fmrre.2004.12.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.fmrre.2004.12.006%26sid%3Dliteratum%253Aachs%26aulast%3DKuznetsova%26aufirst%3DE.%26aulast%3DProudfoot%26aufirst%3DM.%26aulast%3DSanders%26aufirst%3DS.%26aulast%3DReinking%26aufirst%3DJ.%26aulast%3DSavchenko%26aufirst%3DA.%26aulast%3DArrowsmith%26aufirst%3DC.%26aulast%3DEdwards%26aufirst%3DA.%26aulast%3DYakunin%26aufirst%3DA.%26atitle%3DEnzyme%2520genomics%253A%2520Application%2520of%2520general%2520enzymatic%2520screens%2520to%2520discover%2520new%2520enzymes%26jtitle%3DFEMS%2520Microbiol.%2520Rev.%26date%3D2005%26volume%3D29%26spage%3D263%26epage%3D279%26doi%3D10.1016%2Fj.fmrre.2004.12.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arpigny, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaeger, K.-E.</span></span> <span> </span><span class="NLM_article-title">Bacterial lipolytic enzymes: classification and properties</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>343</i></span>,  <span class="NLM_fpage">177</span>– <span class="NLM_lpage">183</span>, <span class="refDoi"> DOI: 10.1042/bj3430177</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1042%2Fbj3430177" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10493927" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADyaK1MXmvVyju7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=343&publication_year=1999&pages=177-183&author=J.+L.+Arpignyauthor=K.-E.+Jaeger&title=Bacterial+lipolytic+enzymes%3A+classification+and+properties&doi=10.1042%2Fbj3430177"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Bacterial lipolytic enzymes: classification and properties</span></div><div class="casAuthors">Arpigny, Jean Louis; Jaeger, Karl-Erich</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">343</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">177-183</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">Knowledge of bacterial lipolytic enzymes is increasing at a rapid and exciting rate.  To obtain an overview of this industrially very important class of enzymes and their characteristics, we have collected and classified the information available from protein and nucleotide databases.  Here we propose an updated and extensive classification of bacterial esterases and lipases based mainly on a comparison of their amino acid sequences and some fundamental biol. properties.  These new insights result in the identification of eight different families with the largest being further divided into six subfamilies.  Moreover, the classification enables us to predict (1) important structural features such as residues forming the catalytic site or the presence of disulfide bonds, (2) types of secretion mechanism and requirement for lipase-specific foldases, and (3) the potential relationship to other enzyme families.  This work will therefore contribute to a faster identification and to an easier characterization of novel bacterial lipolytic enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopK9Mikju05bVg90H21EOLACvtfcHk0lidJ-etib-aEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXmvVyju7o%253D&md5=090196a9865b862342ec6964d61101df</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1042%2Fbj3430177&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252Fbj3430177%26sid%3Dliteratum%253Aachs%26aulast%3DArpigny%26aufirst%3DJ.%2BL.%26aulast%3DJaeger%26aufirst%3DK.-E.%26atitle%3DBacterial%2520lipolytic%2520enzymes%253A%2520classification%2520and%2520properties%26jtitle%3DBiochem.%2520J.%26date%3D1999%26volume%3D343%26spage%3D177%26epage%3D183%26doi%3D10.1042%2Fbj3430177" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bornscheuer, U. T.</span></span> <span> </span><span class="NLM_article-title">Microbial carboxyl esterases: classification, properties and application in biocatalysis</span>. <i>FEMS Microbiol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">73</span>– <span class="NLM_lpage">81</span>, <span class="refDoi"> DOI: 10.1111/j.1574-6976.2002.tb00599.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1111%2Fj.1574-6976.2002.tb00599.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=12007643" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BD38Xjtl2itL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2002&pages=73-81&author=U.+T.+Bornscheuer&title=Microbial+carboxyl+esterases%3A+classification%2C+properties+and+application+in+biocatalysis&doi=10.1111%2Fj.1574-6976.2002.tb00599.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Microbial carboxyl esterases: classification, properties and application in biocatalysis</span></div><div class="casAuthors">Bornscheuer, Uwe T.</div><div class="citationInfo"><span class="NLM_cas:title">FEMS Microbiology Reviews</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">73-81</span>CODEN:
                <span class="NLM_cas:coden">FMREE4</span>;
        ISSN:<span class="NLM_cas:issn">0168-6445</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">A review with 79 refs.  Esterases (EC 3.1.1.x) represent a diverse group of hydrolases catalyzing the cleavage and formation of ester bonds and are widely distributed in animals, plants, and microorganisms.  In addn. to lipases, a considerable no. of microbial carboxyl esterases have also been discovered and overexpressed.  This review summarizes their properties and classification.  Special emphasis is given on their application in org. synthesis for the resoln. of racemates and pro-stereogenic compds.  In addn., recent results for altering their properties by directed evolution are presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpl1yo2gShFArVg90H21EOLACvtfcHk0lidJ-etib-aEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xjtl2itL4%253D&md5=a852ea9888822ea974713fc655258a11</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1111%2Fj.1574-6976.2002.tb00599.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1574-6976.2002.tb00599.x%26sid%3Dliteratum%253Aachs%26aulast%3DBornscheuer%26aufirst%3DU.%2BT.%26atitle%3DMicrobial%2520carboxyl%2520esterases%253A%2520classification%252C%2520properties%2520and%2520application%2520in%2520biocatalysis%26jtitle%3DFEMS%2520Microbiol.%2520Rev.%26date%3D2002%26volume%3D26%26spage%3D73%26epage%3D81%26doi%3D10.1111%2Fj.1574-6976.2002.tb00599.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sjuts, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vargiu, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwasny, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, S. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ornik, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruggerone, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bowlin, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikaido, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pos, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Opperman, T. J.</span></span> <span> </span><span class="NLM_article-title">Molecular basis for inhibition of AcrB multidrug efflux pump by novel and powerful pyranopyridine derivatives</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">3509</span>– <span class="NLM_lpage">3514</span>, <span class="refDoi"> DOI: 10.1073/pnas.1602472113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1073%2Fpnas.1602472113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=26976576" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BC28XktFemtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2016&pages=3509-3514&author=H.+Sjutsauthor=A.+V.+Vargiuauthor=S.+M.+Kwasnyauthor=S.+T.+Nguyenauthor=H.-S.+Kimauthor=X.+Dingauthor=A.+R.+Ornikauthor=P.+Ruggeroneauthor=T.+L.+Bowlinauthor=H.+Nikaidoauthor=K.+M.+Posauthor=T.+J.+Opperman&title=Molecular+basis+for+inhibition+of+AcrB+multidrug+efflux+pump+by+novel+and+powerful+pyranopyridine+derivatives&doi=10.1073%2Fpnas.1602472113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular basis for inhibition of AcrB multidrug efflux pump by novel and powerful pyranopyridine derivatives</span></div><div class="casAuthors">Sjuts, Hanno; Vargiu, Attilio V.; Kwasny, Steven M.; Nguyen, Son T.; Kim, Hong-Suk; Ding, Xiaoyuan; Ornik, Alina R.; Ruggerone, Paolo; Bowlin, Terry L.; Nikaido, Hiroshi; Pos, Klaas M.; Opperman, Timothy J.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">3509-3514</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The Escherichia coli AcrAB-TolC efflux pump is the archetype of the resistance nodulation cell division (RND) exporters from Gram-neg. bacteria.  Overexpression of RND-type efflux pumps is a major factor in multidrug resistance (MDR), which makes these pumps important antibacterial drug discovery targets.  The authors have recently developed novel pyranopyridine-based inhibitors of AcrB, which are orders of magnitude more powerful than the previously known inhibitors.  However, further development of such inhibitors has been hindered by the lack of structural information for rational drug design.  Although only the sol., periplasmic part of AcrB binds and exports the ligands, the presence of the membrane-embedded domain in AcrB and its polyspecific binding behavior have made cocrystn. with drugs challenging.  To overcome this obstacle, the authors have engineered and produced a sol. version of AcrB [AcrB periplasmic domain (AcrBper)], which is highly congruent in structure with the periplasmic part of the full-length protein, and is capable of binding substrates and potent inhibitors.  Here, the authors describe the mol. basis for pyranopyridine-based inhibition of AcrB using a combination of cellular, x-ray crystallog., and mol. dynamics (MD) simulations studies.  The pyranopyridines bind within a phenylalanine-rich cage that branches from the deep binding pocket of AcrB, where they form extensive hydrophobic interactions.  Moreover, the increasing potency of improved inhibitors correlates with the formation of a delicate protein- and water-mediated hydrogen bond network.  These detailed insights provide a mol. platform for the development of novel combinational therapies using efflux pump inhibitors for combating multidrug resistant Gram-neg. pathogens.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAJfI8XTMkfrVg90H21EOLACvtfcHk0lj5wVpmW-IvFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XktFemtrw%253D&md5=49593df0938148f8ec4a84a4ad30cecd</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1602472113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1602472113%26sid%3Dliteratum%253Aachs%26aulast%3DSjuts%26aufirst%3DH.%26aulast%3DVargiu%26aufirst%3DA.%2BV.%26aulast%3DKwasny%26aufirst%3DS.%2BM.%26aulast%3DNguyen%26aufirst%3DS.%2BT.%26aulast%3DKim%26aufirst%3DH.-S.%26aulast%3DDing%26aufirst%3DX.%26aulast%3DOrnik%26aufirst%3DA.%2BR.%26aulast%3DRuggerone%26aufirst%3DP.%26aulast%3DBowlin%26aufirst%3DT.%2BL.%26aulast%3DNikaido%26aufirst%3DH.%26aulast%3DPos%26aufirst%3DK.%2BM.%26aulast%3DOpperman%26aufirst%3DT.%2BJ.%26atitle%3DMolecular%2520basis%2520for%2520inhibition%2520of%2520AcrB%2520multidrug%2520efflux%2520pump%2520by%2520novel%2520and%2520powerful%2520pyranopyridine%2520derivatives%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2016%26volume%3D113%26spage%3D3509%26epage%3D3514%26doi%3D10.1073%2Fpnas.1602472113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span> <span> </span><span class="NLM_article-title">Supramolecular nanofibers with superior bioactivity to insulin-like growth factor-I</span>. <i>Nano Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">1560</span>– <span class="NLM_lpage">1569</span>, <span class="refDoi"> DOI: 10.1021/acs.nanolett.8b04406</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.nanolett.8b04406" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjsVeisL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2019&pages=1560-1569&author=Y.+Shangauthor=D.+Zhiauthor=G.+Fengauthor=Z.+Wangauthor=D.+Maoauthor=S.+Guoauthor=R.+Liuauthor=L.+Liuauthor=S.+Zhangauthor=S.+Sunauthor=K.+Wangauthor=D.+Kongauthor=J.+Gaoauthor=Z.+Yang&title=Supramolecular+nanofibers+with+superior+bioactivity+to+insulin-like+growth+factor-I&doi=10.1021%2Facs.nanolett.8b04406"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Supramolecular Nanofibers with Superior Bioactivity to Insulin-Like Growth Factor-I</span></div><div class="casAuthors">Shang, Yuna; Zhi, Dengke; Feng, Guowei; Wang, Zhongyan; Mao, Duo; Guo, Shuang; Liu, Ruihua; Liu, Lulu; Zhang, Shuhao; Sun, Shenghuan; Wang, Kai; Kong, Deling; Gao, Jie; Yang, Zhimou</div><div class="citationInfo"><span class="NLM_cas:title">Nano Letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1560-1569</span>CODEN:
                <span class="NLM_cas:coden">NALEFD</span>;
        ISSN:<span class="NLM_cas:issn">1530-6984</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bioactive peptides derived from proteins generally need to be folded into secondary structures to activate downstream signaling pathways.  However, synthetic peptides typically form random-coils, thus losing their bioactivities.  Here, we show that by introducing a self-assembling peptide motif and using different prepn. pathways, a peptide from insulin-like growth factor-I (IGF-1) can be folded into an α-helix and β-sheet.  The β-sheet one exhibits a low dissocn. const. to the IGF-1 receptor (IGF-1R, 11.5 nM), which is only about 3 times higher than that of IGF-1 (4.3 nM).  However, the α-helical one and the peptide without self-assembling motif show weak affinities to IGF-1R (KD = 179.1 and 321.6 nM, resp.).  At 10 nM, the β-sheet one efficiently activates the IGF-1 downstream pathway, significantly enhancing HUVEC proliferation and preventing cell apoptosis.  The β-sheet peptide shows superior performance to IGF-1 in vivo, and it improves ischemic hind-limb salvage by significantly reducing muscle degrdn. and enhancing limb vascularization.  Our study provides a useful strategy to constrain peptides into different conformations, which may lead to the development of supramol. nanomaterials mimicking biofunctional proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0gIjwARXkoLVg90H21EOLACvtfcHk0lj5wVpmW-IvFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjsVeisL4%253D&md5=42e72e0fe56a128a407f6d0a94768387</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Facs.nanolett.8b04406&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.nanolett.8b04406%26sid%3Dliteratum%253Aachs%26aulast%3DShang%26aufirst%3DY.%26aulast%3DZhi%26aufirst%3DD.%26aulast%3DFeng%26aufirst%3DG.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DMao%26aufirst%3DD.%26aulast%3DGuo%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DR.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DSun%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DK.%26aulast%3DKong%26aufirst%3DD.%26aulast%3DGao%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DZ.%26atitle%3DSupramolecular%2520nanofibers%2520with%2520superior%2520bioactivity%2520to%2520insulin-like%2520growth%2520factor-I%26jtitle%3DNano%2520Lett.%26date%3D2019%26volume%3D19%26spage%3D1560%26epage%3D1569%26doi%3D10.1021%2Facs.nanolett.8b04406" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hua, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ou, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, M.</span></span> <span> </span><span class="NLM_article-title">Folic acid derived hydrogel enhances the survival and promotes therapeutic efficacy of iPS cells for acute myocardial infarction</span>. <i>ACS Appl. Mater. Interfaces</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">24459</span>– <span class="NLM_lpage">24468</span>, <span class="refDoi"> DOI: 10.1021/acsami.8b08659</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsami.8b08659" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1Knsr7L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2018&pages=24459-24468&author=H.+Liauthor=J.+Gaoauthor=Y.+Shangauthor=Y.+Huaauthor=M.+Yeauthor=Z.+Yangauthor=C.+Ouauthor=M.+Chen&title=Folic+acid+derived+hydrogel+enhances+the+survival+and+promotes+therapeutic+efficacy+of+iPS+cells+for+acute+myocardial+infarction&doi=10.1021%2Facsami.8b08659"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Folic Acid Derived Hydrogel Enhances the Survival and Promotes Therapeutic Efficacy of iPS Cells for Acute Myocardial Infarction</span></div><div class="casAuthors">Li, Hekai; Gao, Jie; Shang, Yuna; Hua, Yongquan; Ye, Min; Yang, Zhimou; Ou, Caiwen; Chen, Minsheng</div><div class="citationInfo"><span class="NLM_cas:title">ACS Applied Materials & Interfaces</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">29</span>),
    <span class="NLM_cas:pages">24459-24468</span>CODEN:
                <span class="NLM_cas:coden">AAMICK</span>;
        ISSN:<span class="NLM_cas:issn">1944-8244</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Stem cell therapy has obtained extensive consensus to be an effective method for post myocardial infarction (MI) intervention.  Induced pluripotent stem (iPS) cells, which are able to differentiate into multiple cell types, have the potential to generate cardiovascular lineage cells for myocardial repair after ischemic damage, but their poor retention rate significantly hinders the therapeutic efficacy.  In the present study, we developed a supramol. hydrogel which is formed by the self-assembly of folic acid (FA)-modified peptide via a biocompatible method (glutathione redn.) and suitable for cell encapsulation and transplantation.  The iPS cells labeled with CM-Dil were transplanted into the MI hearts of mice with or without FA hydrogel encapsulation.  The results corroborated that the FA hydrogel significantly improved the retention and survival of iPS cells in MI hearts post injection, leading to augmentation of the therapeutic efficacy of iPS cells including better cardiac function and much less adverse heart remodeling, by subsequent differentiation toward cardiac cells and stimulation of neovascularization.  This study reported a novel supramol. hydrogel based on FA-peptides capable of improving the therapeutic capacity of iPS cells, which held big potential in the treatment of MI.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoy3dxgJRp4CbVg90H21EOLACvtfcHk0lj5wVpmW-IvFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1Knsr7L&md5=ef582ee438e2092661c78267836714cc</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Facsami.8b08659&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsami.8b08659%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DH.%26aulast%3DGao%26aufirst%3DJ.%26aulast%3DShang%26aufirst%3DY.%26aulast%3DHua%26aufirst%3DY.%26aulast%3DYe%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DOu%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DM.%26atitle%3DFolic%2520acid%2520derived%2520hydrogel%2520enhances%2520the%2520survival%2520and%2520promotes%2520therapeutic%2520efficacy%2520of%2520iPS%2520cells%2520for%2520acute%2520myocardial%2520infarction%26jtitle%3DACS%2520Appl.%2520Mater.%2520Interfaces%26date%3D2018%26volume%3D10%26spage%3D24459%26epage%3D24468%26doi%3D10.1021%2Facsami.8b08659" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span> <span> </span><span class="NLM_article-title">Tandem molecular self-assembly in liver cancer cells</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">1813</span>– <span class="NLM_lpage">1816</span>, <span class="refDoi"> DOI: 10.1002/anie.201710237</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1002%2Fanie.201710237" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlKrsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2018&pages=1813-1816&author=J.+Zhanauthor=Y.+Caiauthor=S.+Heauthor=L.+Wangauthor=Z.+Yang&title=Tandem+molecular+self-assembly+in+liver+cancer+cells&doi=10.1002%2Fanie.201710237"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Tandem Molecular Self-Assembly in Liver Cancer Cells</span></div><div class="casAuthors">Zhan, Jie; Cai, Yanbin; He, Shuangshuang; Wang, Ling; Yang, Zhimou</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1813-1816</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The authors herein describe the tandem mol. self-assembly of a peptide deriv. (1) that is controlled by a combination of enzymic and chem. reactions.  In phosphate-buffered saline (PBS), compd. 1 self-assembles first into nanoparticles by phosphatase and then into nanofibers by glutathione.  Liver cancer cells exhibit higher concns. of both phosphatase and GSH than normal cells.  Therefore, the tandem self-assembly of 1 also occurs in the liver cancer cell lines HepG2 and QGY7703; compd. 1 first forms nanoparticles around the cells and then forms nanofibers inside the cells.  Owing to this self-assembly mechanism, compd. 1 exhibits large ratios for cellular uptake and inhibition of cell viability between liver cancer cells and normal liver cells.  The authors envision that using both extracellular and intracellular reactions to trigger tandem mol. self-assembly could lead to the development of supramol. nanomaterials with improved performance in cancer diagnostics and therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpowoReFd-OdbVg90H21EOLACvtfcHk0lhJ2DllcuwFQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlKrsLY%253D&md5=06fcf89d59333758e65cfe0d42727f03</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1002%2Fanie.201710237&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201710237%26sid%3Dliteratum%253Aachs%26aulast%3DZhan%26aufirst%3DJ.%26aulast%3DCai%26aufirst%3DY.%26aulast%3DHe%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DZ.%26atitle%3DTandem%2520molecular%2520self-assembly%2520in%2520liver%2520cancer%2520cells%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2018%26volume%3D57%26spage%3D1813%26epage%3D1816%26doi%3D10.1002%2Fanie.201710237" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gellman, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeGrado, W. F.</span></span> <span> </span><span class="NLM_article-title">β-Peptides: from structure to function</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>101</i></span>,  <span class="NLM_fpage">3219</span>– <span class="NLM_lpage">3232</span>, <span class="refDoi"> DOI: 10.1021/cr000045i</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr000045i" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BD3MXntlGntbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2001&pages=3219-3232&author=R.+P.+Chengauthor=S.+H.+Gellmanauthor=W.+F.+DeGrado&title=%CE%B2-Peptides%3A+from+structure+to+function&doi=10.1021%2Fcr000045i"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">β-Peptides: From structure to function</span></div><div class="casAuthors">Cheng, Richard P.; Gellman, Samuel H.; DeGrado, William F.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, D. C.)</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">101</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3219-3232</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review with 187 refs., of the conformational properties and biol. functions, esp. as peptidomimetics, of β-peptides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTfycZ30AaKLVg90H21EOLACvtfcHk0lhJ2DllcuwFQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXntlGntbg%253D&md5=40cbf3f3d3c00ec8d6250adf407a663a</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1021%2Fcr000045i&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr000045i%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DR.%2BP.%26aulast%3DGellman%26aufirst%3DS.%2BH.%26aulast%3DDeGrado%26aufirst%3DW.%2BF.%26atitle%3D%25CE%25B2-Peptides%253A%2520from%2520structure%2520to%2520function%26jtitle%3DChem.%2520Rev.%26date%3D2001%26volume%3D101%26spage%3D3219%26epage%3D3232%26doi%3D10.1021%2Fcr000045i" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kreitler, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinkruger, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mortenson, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mittal, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Travis, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forest, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gellman, S. H.</span></span> <span> </span><span class="NLM_article-title">A hendecad motif is preferred for heterochiral coiled-coil formation</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>141</i></span>,  <span class="NLM_fpage">1583</span>– <span class="NLM_lpage">1592</span>, <span class="refDoi"> DOI: 10.1021/jacs.8b11246</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.8b11246" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BC1MXpsFejtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=141&publication_year=2019&pages=1583-1592&author=D.+F.+Kreitlerauthor=Z.+Yaoauthor=J.+D.+Steinkrugerauthor=D.+E.+Mortensonauthor=L.+Huangauthor=R.+Mittalauthor=B.+R.+Travisauthor=K.+T.+Forestauthor=S.+H.+Gellman&title=A+hendecad+motif+is+preferred+for+heterochiral+coiled-coil+formation&doi=10.1021%2Fjacs.8b11246"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">A hendecad motif is preferred for heterochiral coiled-coil formation</span></div><div class="casAuthors">Kreitler, Dale F.; Yao, Zhihui; Steinkruger, Jay D.; Mortenson, David E.; Huang, Lijun; Mittal, Ritesh; Travis, Benjamin R.; Forest, Katrina T.; Gellman, Samuel H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">141</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1583-1592</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The structural principles that govern interactions between L- and D-peptides are not well understood.  Among natural proteins, coiled-coil assemblies formed between or among α-helixes are the most regular feature of tertiary and quaternary structures.  We recently reported the 1st high-resoln. structures for heterochiral coiled-coil dimers, which represent a starting point for understanding assocns. of L- and D-polypeptides.  These structures were an unexpected outcome from crystn. of a racemic peptide corresponding to the transmembrane domain of the influenza A M2 protein (M2-TM).  The reported structures raised the possibility that heterochiral coiled-coil dimers prefer an 11-residue (hendecad) sequence repeat, in contrast to the 7-residue (heptad) sequence repeat that is dominant among natural coiled-coils.  To gain insight on sequence repeat preferences of heterochiral coiled-coils, we examd. 3 M2-TM variants contg. substitutions intended to minimize steric clashes between side-chains at the coiled-coil interface.  In each of the 3 new crystal structures, we obsd. heterochiral coiled-coil assocns. that closely matched a hendecad sequence motif, which strengthens the conclusion that this motif is intrinsic to the pairing of α-helixes with opposite handedness.  In each case, the presence of a hendecad motif was established by comparing the obsd. helical frequency to that of an ideal hendecad.  This comparison revealed that decreasing the size of the amino acid side-chain at positions that project toward the superhelical axis produces tighter packing, as detd. by the size of the coiled-coil radius.  These results provide a basis for future design of heterochiral coiled-coil pairings.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXI23Fe-4ZMLVg90H21EOLACvtfcHk0lhJ2DllcuwFQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXpsFejtQ%253D%253D&md5=223f0d8d062f3cf76af6f3a9453a8e89</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Fjacs.8b11246&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.8b11246%26sid%3Dliteratum%253Aachs%26aulast%3DKreitler%26aufirst%3DD.%2BF.%26aulast%3DYao%26aufirst%3DZ.%26aulast%3DSteinkruger%26aufirst%3DJ.%2BD.%26aulast%3DMortenson%26aufirst%3DD.%2BE.%26aulast%3DHuang%26aufirst%3DL.%26aulast%3DMittal%26aufirst%3DR.%26aulast%3DTravis%26aufirst%3DB.%2BR.%26aulast%3DForest%26aufirst%3DK.%2BT.%26aulast%3DGellman%26aufirst%3DS.%2BH.%26atitle%3DA%2520hendecad%2520motif%2520is%2520preferred%2520for%2520heterochiral%2520coiled-coil%2520formation%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2019%26volume%3D141%26spage%3D1583%26epage%3D1592%26doi%3D10.1021%2Fjacs.8b11246" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abdulkadir, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">Inhibition of β-catenin/B cell lymphoma 9 protein–protein interaction using α-helix–mimicking sulfono-γ-AApeptide inhibitors</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">10757</span>– <span class="NLM_lpage">10762</span>, <span class="refDoi"> DOI: 10.1073/pnas.1819663116</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1073%2Fpnas.1819663116" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=31088961" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtVChsr%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2019&pages=10757-10762&author=P.+Sangauthor=M.+Zhangauthor=Y.+Shiauthor=C.+Liauthor=S.+Abdulkadirauthor=Q.+Liauthor=H.+Jiauthor=J.+Cai&title=Inhibition+of+%CE%B2-catenin%2FB+cell+lymphoma+9+protein%E2%80%93protein+interaction+using+%CE%B1-helix%E2%80%93mimicking+sulfono-%CE%B3-AApeptide+inhibitors&doi=10.1073%2Fpnas.1819663116"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of β-catenin/B cell lymphoma 9 protein-protein interaction using α-helix-mimicking sulfono-γ-AApeptide inhibitors</span></div><div class="casAuthors">Sang, Peng; Zhang, Min; Shi, Yan; Li, Chunpu; Abdulkadir, Sami; Li, Qi; Ji, Haitao; Cai, Jianfeng</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">10757-10762</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The rational design of α-helix-mimicking peptidomimetics provides a streamlined approach to discover potent inhibitors for protein-protein interactions (PPIs).  However, designing cell-penetrating long peptidomimetic scaffolds equipped with various functional groups necessary for interacting with large protein-binding interfaces remains challenging.  This is particularly true for targeting β-catenin/BCL9 PPIs.  Here the authors designed a series of unprecedented helical sulfono-γ-AApeptides that mimic the binding mode of the α-helical HD2 domain of B Cell Lymphoma 9 (BCL9).  The studies show that sulfono-γ-AApeptides can structurally and functionally mimic the α-helical domain of BCL9 and selectively disrupt β-catenin/BCL9 PPIs with even higher potency.  More intriguingly, these sulfono-γ-AApeptides can enter cancer cells, bind with β-catenin and disrupt β-catenin/BCL9 PPIs, and exhibit excellent cellular activity, which is much more potent than the BCL9 peptide.  Furthermore, the enzymic stability studies demonstrate the remarkable stability of the helical sulfono-γ-AApeptides, with no degrdn. in the presence of pronase for 24 h, augmenting their biol. potential.  This work represents not only an example of helical sulfono-γ-AApeptides that mimic α-helix and disrupt protein-protein interactions, but also an excellent example of potent, selective, and cell-permeable unnatural foldameric peptidomimetics that disrupt the β-catenin/BCL9 PPI.  The design of helical sulfono-γ-AApeptides may lead to a new strategy to modulate a myriad of protein-protein interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_pKlgpb-_U7Vg90H21EOLACvtfcHk0li1uFMUY8ti0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtVChsr%252FE&md5=145172aa8475bfe4a20b97d6b88d12dd</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1819663116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1819663116%26sid%3Dliteratum%253Aachs%26aulast%3DSang%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DM.%26aulast%3DShi%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DAbdulkadir%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DJi%26aufirst%3DH.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3DInhibition%2520of%2520%25CE%25B2-catenin%252FB%2520cell%2520lymphoma%25209%2520protein%25E2%2580%2593protein%2520interaction%2520using%2520%25CE%25B1-helix%25E2%2580%2593mimicking%2520sulfono-%25CE%25B3-AApeptide%2520inhibitors%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2019%26volume%3D116%26spage%3D10757%26epage%3D10762%26doi%3D10.1073%2Fpnas.1819663116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhattarai, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, F.-S.</span></span> <span> </span><span class="NLM_article-title">Cyclic peptidomimetics as inhibitor for miR-155 biogenesis</span>. <i>Mol. Pharm.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">914</span>– <span class="NLM_lpage">920</span>, <span class="refDoi"> DOI: 10.1021/acs.molpharmaceut.8b01247</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.molpharmaceut.8b01247" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtlSmsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2019&pages=914-920&author=H.+Yanauthor=M.+Zhouauthor=U.+Bhattaraiauthor=Y.+Songauthor=M.+Zhengauthor=J.+Caiauthor=F.-S.+Liang&title=Cyclic+peptidomimetics+as+inhibitor+for+miR-155+biogenesis&doi=10.1021%2Facs.molpharmaceut.8b01247"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclic Peptidomimetics as Inhibitor for miR-155 Biogenesis</span></div><div class="casAuthors">Yan, Hao; Zhou, Mi; Bhattarai, Umesh; Song, Yabin; Zheng, Mengmeng; Cai, Jianfeng; Liang, Fu-Sen</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmaceutics</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">914-920</span>CODEN:
                <span class="NLM_cas:coden">MPOHBP</span>;
        ISSN:<span class="NLM_cas:issn">1543-8384</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">MiR-155 plays key promoting roles in several cancers and emerges as an important anticancer therapeutic target.  However, the discovery of small mols. that target RNAs is challenging.  Peptidomimetics have been shown to be a rich source for discovering novel ligands to regulate cellular proteins.  However, the potential of using peptidomimetics for RNA targeting is relatively unexplored.  To this end, we designed and synthesized members of a novel 320 000 compd. macrocyclic peptidomimetic library.  An affinity-based screening protocol led to the identification of a pre-miR-155 binder that inhibits oncogenic miR-155 maturation in vitro and in cell and induces cancer cell apoptosis.  The results of this investigation demonstrate that macrocyclic peptidomimetics could serve as a new scaffold for RNA targeting.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqa1182LkJrALVg90H21EOLACvtfcHk0li1uFMUY8ti0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtlSmsg%253D%253D&md5=0ac2dec99b95d9efbbaef045bf9e9e4f</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Facs.molpharmaceut.8b01247&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.molpharmaceut.8b01247%26sid%3Dliteratum%253Aachs%26aulast%3DYan%26aufirst%3DH.%26aulast%3DZhou%26aufirst%3DM.%26aulast%3DBhattarai%26aufirst%3DU.%26aulast%3DSong%26aufirst%3DY.%26aulast%3DZheng%26aufirst%3DM.%26aulast%3DCai%26aufirst%3DJ.%26aulast%3DLiang%26aufirst%3DF.-S.%26atitle%3DCyclic%2520peptidomimetics%2520as%2520inhibitor%2520for%2520miR-155%2520biogenesis%26jtitle%3DMol.%2520Pharm.%26date%3D2019%26volume%3D16%26spage%3D914%26epage%3D920%26doi%3D10.1021%2Facs.molpharmaceut.8b01247" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">She, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teng, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peguero-Tejada, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Odom, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gjonaj, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wojtas, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Vaart, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">De novo left-handed synthetic peptidomimetic foldamers</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">9916</span>– <span class="NLM_lpage">9920</span>, <span class="refDoi"> DOI: 10.1002/anie.201805184</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1002%2Fanie.201805184" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1yjt7bL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2018&pages=9916-9920&author=F.+Sheauthor=P.+Tengauthor=A.+Peguero-Tejadaauthor=M.+Wangauthor=N.+Maauthor=T.+Odomauthor=M.+Zhouauthor=E.+Gjonajauthor=L.+Wojtasauthor=A.+van+der+Vaartauthor=J.+Cai&title=De+novo+left-handed+synthetic+peptidomimetic+foldamers&doi=10.1002%2Fanie.201805184"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">De novo left-handed synthetic peptidomimetic foldamers</span></div><div class="casAuthors">She, Fengyu; Teng, Peng; Peguero-Tejada, Alfredo; Wang, Minghui; Ma, Ning; Odom, Timothy; Zhou, Mi; Gjonaj, Erald; Wojtas, Lukasz; van der Vaart, Arjan; Cai, Jianfeng</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">9916-9920</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The development of peptidomimetic helical foldamers with a wide repertoire of functions is of significant interest.  Herein, we report the X-ray crystal structures of a series of homogeneous L-sulfono-γ-AA foldamers and elucidate their folding conformation at the at. level.  Single-crystal X-ray crystallog. revealed that this class of oligomers fold into unprecedented dragon-boat-shaped and unexpected left-handed helixes, which are stabilized by the 14-hydrogen-bonding pattern present in all sequences.  These L-sulfono-γ-AA-peptides have a helical pitch of 5.1 Å and exactly four side chains per turn, and the side chains lie perfectly on top of each other along the helical axis.  2D NMR spectroscopy, computational simulations, and CD studies support the folding conformation in soln.  Our results provide a structural basis at the at. level for the design of novel biomimetics with a precise arrangement of functional groups in three dimensions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGru2vbXzMss1rVg90H21EOLACvtfcHk0li1uFMUY8ti0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1yjt7bL&md5=c45cc7e77dfa2887f5fec63d8e3dbe7e</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1002%2Fanie.201805184&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201805184%26sid%3Dliteratum%253Aachs%26aulast%3DShe%26aufirst%3DF.%26aulast%3DTeng%26aufirst%3DP.%26aulast%3DPeguero-Tejada%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DMa%26aufirst%3DN.%26aulast%3DOdom%26aufirst%3DT.%26aulast%3DZhou%26aufirst%3DM.%26aulast%3DGjonaj%26aufirst%3DE.%26aulast%3DWojtas%26aufirst%3DL.%26aulast%3Dvan%2Bder%2BVaart%26aufirst%3DA.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3DDe%2520novo%2520left-handed%2520synthetic%2520peptidomimetic%2520foldamers%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2018%26volume%3D57%26spage%3D9916%26epage%3D9920%26doi%3D10.1002%2Fanie.201805184" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chan, W. C.</span>; <span class="NLM_string-name">White, P. D.</span></span> <i>Fmoc Solid Phase Peptide Synthesis: A Practical Approach</i>; <span class="NLM_publisher-name">Oxford University Press</span>: <span class="NLM_publisher-loc">Oxford</span>, <span class="NLM_year">2000</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2000&author=W.+C.+Chan&author=P.+D.+White&title=Fmoc+Solid+Phase+Peptide+Synthesis%3A+A+Practical+Approach"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DChan%26aufirst%3DW.%2BC.%26btitle%3DFmoc%2520Solid%2520Phase%2520Peptide%2520Synthesis%253A%2520A%2520Practical%2520Approach%26pub%3DOxford%2520University%2520Press%26date%3D2000" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 2 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ziqi  Liu</span>, <span class="hlFld-ContribAuthor ">Xuan  Tang</span>, <span class="hlFld-ContribAuthor ">Feng  Feng</span>, <span class="hlFld-ContribAuthor ">Jing  Xu</span>, <span class="hlFld-ContribAuthor ">Can  Wu</span>, <span class="hlFld-ContribAuthor ">Guoru  Dai</span>, <span class="hlFld-ContribAuthor ">Wanqing  Yue</span>, <span class="hlFld-ContribAuthor ">Wenying  Zhong</span>, <span class="hlFld-ContribAuthor ">Keming  Xu</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular design of peptide amphiphiles for controlled self-assembly and drug release. </span><span class="cited-content_cbyCitation_journal-name">Journal of Materials Chemistry B</span><span> <strong>2021,</strong> <em>9 </em>
                                    (15)
                                     , 3326-3334. <a href="https://doi.org/10.1039/D1TB00173F" title="DOI URL">https://doi.org/10.1039/D1TB00173F</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D1TB00173F&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD1TB00173F%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Materials%2520Chemistry%2520B%26atitle%3DMolecular%252Bdesign%252Bof%252Bpeptide%252Bamphiphiles%252Bfor%252Bcontrolled%252Bself-assembly%252Band%252Bdrug%252Brelease%26aulast%3DLiu%26aufirst%3DZiqi%26date%3D2021%26date%3D2021%26volume%3D9%26issue%3D15%26spage%3D3326%26epage%3D3334" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Anna N.  Tevyashova</span>. </span><span class="cited-content_cbyCitation_article-title">Recent Trends in Synthesis of Chloramphenicol New Derivatives. </span><span class="cited-content_cbyCitation_journal-name">Antibiotics</span><span> <strong>2021,</strong> <em>10 </em>
                                    (4)
                                     , 370. <a href="https://doi.org/10.3390/antibiotics10040370" title="DOI URL">https://doi.org/10.3390/antibiotics10040370</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/antibiotics10040370&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fantibiotics10040370%26sid%3Dliteratum%253Aachs%26jtitle%3DAntibiotics%26atitle%3DRecent%252BTrends%252Bin%252BSynthesis%252Bof%252BChloramphenicol%252BNew%252BDerivatives%26aulast%3DTevyashova%26aufirst%3DAnna%2BN.%26date%3D2021%26date%3D2021%26volume%3D10%26issue%3D4%26spage%3D370" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01210/20191122/images/medium/jm9b01210_0008.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01210/20191122/images/large/jm9b01210_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01210&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01210/20191122/images/medium/jm9b01210_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01210/20191122/images/large/jm9b01210_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Illustration of Peptide-Conjugated Antibiotics for Intrabacterial Activation and the Trends of Efficiency Associated with the Structures of the Conjugates (LPS: Lipopolysaccharide; OM: Outer Membrane; and IM: Inner Membrane)</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01210/20191122/images/large/jm9b01210_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01210&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01210/20191122/images/medium/jm9b01210_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01210/20191122/images/large/jm9b01210_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. (A) Antibacterial activity of <b>1c</b>, <b>1d</b>, <b>2b</b>, and <b>2c</b> before and after the treatment of PLE; [PLE] = 1 U/mL. (B) Hydrolysis curves of <b>1c</b>, <b>1d</b>, <b>2b</b>, and <b>2c</b> with the addition of the lysate of HepG2 cells or the lysate of <i>E. coli</i> (K-12); [<b>1c</b>] = [<b>1d</b>] = [<b>2b</b>] = [<b>2c</b>] = 200 μM, [HepG2 lysate] = [<i>E. coli</i> lysate] = 0.1 U/mL. (C) Static light-scattering signals of the solution of <b>1c</b>, <b>1d</b>, <b>2b</b>, and <b>2c</b> before and after the addition of PLE; [<b>1c</b>] = [<b>1d</b>] = [<b>2b</b>] = [<b>2c</b>] = 200 μM, [PLE] = 1 U/mL, <i>t</i> = 24 h. (D) Transmission electron microscopy (TEM) images of <b>1c</b>, <b>1d</b>, <b>2b</b>, and <b>2c</b> before and after the addition of PLE; [<b>1c</b>] = [<b>1d</b>] = [<b>2b</b>] = [<b>2c</b>] = 500 μM, [PLE] = 1 U/mL, <i>t</i> = 24 h, scale bar = 500 nm.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01210/20191122/images/large/jm9b01210_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01210&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01210/20191122/images/medium/jm9b01210_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01210/20191122/images/large/jm9b01210_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Antibacterial activity of <b>1c</b>, <b>1d</b>, <b>2b</b>, and <b>2c</b> against <i>ydgR</i>, <i>fepA</i>, and <i>ompF</i> transporter deletion mutants or <i>acrA</i> efflux pump deletion mutant of <i>E. coli</i>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01210/20191122/images/large/jm9b01210_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01210&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01210/20191122/images/medium/jm9b01210_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01210/20191122/images/large/jm9b01210_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Antibacterial activity of <b>1c</b>, <b>1d</b>, <b>2b</b>, and <b>2c</b> against esterase (<i>bioH</i>, <i>yjfP</i>, <i>frsA</i>, <i>ybfF</i>, <i>yfbB</i>, <i>ypfH</i>, <i>yeiG</i>, or <i>yqiA</i>) deletion mutants of <i>E. coli</i>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01210/20191122/images/large/jm9b01210_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01210&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01210/20191122/images/medium/jm9b01210_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01210/20191122/images/large/jm9b01210_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Cell viability of (A) HS-5 cells and (B) HEK293 cells incubated with <b>1a</b>, <b>1b</b>, <b>1c</b>, <b>1e</b>, <b>1f</b>, <b>1k</b>, <b>1l</b>, <b>1m</b>, <b>1o</b>, <b>1p</b>, <b>2f</b>, and <b>2j</b> for 24 h. [CL] = [CLsu] = [<b>1a</b>] = [<b>1b</b>] = [<b>1c</b>] = [<b>1e</b>] = [<b>1f</b>] = [<b>1k</b>] = [<b>1l</b>] = [<b>1m</b>] = [<b>1o</b>] = [<b>1p</b>] = [<b>2f</b>] = [<b>2j</b>] = 20 μM; [Cipro] = [TMP] = [TEL] = 0.5 μM. (C) Cell viability of HS-5 cells incubated with CL, CLsu, <b>1a</b>, <b>1b</b>, <b>1c</b>, <b>1e</b>, <b>1f</b>, <b>1k</b>, <b>1l</b>, <b>1m</b>, <b>1o</b>, <b>1p</b>, <b>2f</b>, and <b>2j</b> for 24 h. [CL] = [CLsu] = [<b>1a</b>] = [<b>1b</b>] = [<b>1c</b>] = [<b>1e</b>] = [<b>1f</b>] = [<b>1k</b>] = [<b>1l</b>] = [<b>1m</b>] = [<b>1o</b>] = [<b>1p</b>] = [<b>2f</b>] = [<b>2j</b>] = 200 μM; [Cipro] = [TMP] = [TEL] = 5 μM. (D) Cell viability of HS-5 cells incubated with CL, CLsu, <b>1a</b>, <b>1b</b>, <b>1c</b>, <b>1e</b>, <b>1f</b>, <b>1k</b>, <b>1l</b>, <b>1m</b>, <b>1o</b>, <b>1p</b>, <b>2f</b>, and <b>2j</b> for 24 h. [CL] = [CLsu] = [<b>1a</b>] = [<b>1b</b>] = [<b>1c</b>] = [<b>1e</b>] = [<b>1f</b>] = [<b>1k</b>] = [<b>1l</b>] = [<b>1m</b>] = [<b>1o</b>] = [<b>1p</b>] = [<b>2f</b>] = [<b>2j</b>] = 2000 μM; [Cipro] = [TMP] = [TEL] = 50 μM.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-22/acs.jmedchem.9b01210/20191122/images/large/jm9b01210_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01210&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i56">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_57204" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_57204" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 59 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Neu, H. C.</span></span> <span> </span><span class="NLM_article-title">The crisis in antibiotic resistance</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1992</span>,  <span class="NLM_volume"><i>257</i></span>,  <span class="NLM_fpage">1064</span>– <span class="NLM_lpage">1073</span>, <span class="refDoi"> DOI: 10.1126/science.257.5073.1064</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1126%2Fscience.257.5073.1064" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1509257" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADyaK38Xls1OhsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=257&publication_year=1992&pages=1064-1073&author=H.+C.+Neu&title=The+crisis+in+antibiotic+resistance&doi=10.1126%2Fscience.257.5073.1064"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The crisis in antibiotic resistance</span></div><div class="casAuthors">Neu, Harold C.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">257</span>
        (<span class="NLM_cas:issue">5073</span>),
    <span class="NLM_cas:pages">1064-72</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    </div><div class="casAbstract">A review with >64 refs.  The synthesis of large nos. of antibiotics over the past three decades has caused complacency about the threat of bacterial resistance.  Bacteria have become resistant to antimicrobial agents as a result of chromosomal changes or the exchange of genetic material via plasmids and transposons.  Streptococcus pneumoniae, Streptococcus pyogenes, and staphylococci, organisms that cause respiratory and cutaneous infections, and members of the Enterobacteriaceae and Pseudomonas families, organisms that cause diarrhea, urinary infection, and sepsis, are now resistant to virtually all of the older antibiotics.  The extensive use of antibiotics in the community and hospitals has fueled this crisis.  Mechanisms such as antibiotic control programs, better hygiene, and synthesis of agents with improved antimicrobial activity need to be adopted in order to limit bacterial resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmntDmhDYAKrVg90H21EOLACvtfcHk0lgECUZnjiKvow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38Xls1OhsbY%253D&md5=02b98676a76f669ffde4dbb1f10a3876</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1126%2Fscience.257.5073.1064&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.257.5073.1064%26sid%3Dliteratum%253Aachs%26aulast%3DNeu%26aufirst%3DH.%2BC.%26atitle%3DThe%2520crisis%2520in%2520antibiotic%2520resistance%26jtitle%3DScience%26date%3D1992%26volume%3D257%26spage%3D1064%26epage%3D1073%26doi%3D10.1126%2Fscience.257.5073.1064" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ghafourian, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sadeghifard, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soheili, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sekawi, Z.</span></span> <span> </span><span class="NLM_article-title">Extended spectrum beta-lactamases: definition, classification and epidemiology</span>. <i>Curr. Issues Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">11</span>– <span class="NLM_lpage">22</span>, <span class="refDoi"> DOI: 10.21775/cimb.017.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.21775%2Fcimb.017.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=24821872" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A280%3ADC%252BC2cjitFWqtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2015&pages=11-22&author=S.+Ghafourianauthor=N.+Sadeghifardauthor=S.+Soheiliauthor=Z.+Sekawi&title=Extended+spectrum+beta-lactamases%3A+definition%2C+classification+and+epidemiology&doi=10.21775%2Fcimb.017.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Extended Spectrum Beta-lactamases: Definition, Classification and Epidemiology</span></div><div class="casAuthors">Ghafourian Sobhan; Sadeghifard Nourkhoda; Soheili Sara; Sekawi Zamberi</div><div class="citationInfo"><span class="NLM_cas:title">Current issues in molecular biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">11-21</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Extended spectrum beta-lactamases (ESBLs) are defined as enzymes produced by certain bacteria that are able to hydrolyze extended spectrum cephalosporin.  They are therefore effective against beta-lactam antibiotics such as ceftazidime, ceftriaxone, cefotaxime and oxyimino-monobactam.  The objective of the current review is to provide a better understanding of ESBL and the epidemiology of ESBL producing organisms which are among those responsible for antibiotic resistant strains.  Globally, ESBLs are considered to be problematic, particularly in hospitalized patients.  There is an increasing frequency of ESBL in different parts of the world.  The high risk patients are those contaminated with ESBL producer strains as it renders treatment to be ineffective in these patients.  Thus, there an immediate needs to identify EBSL and formulate strategic policy initiatives to reduce their prevalence.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQCRwqOHh6OXxcr74wzj5o5fW6udTcc2eYPVNZQVOHHs7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2cjitFWqtg%253D%253D&md5=3af33b6ae498e7849578e0af6b62a6f4</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.21775%2Fcimb.017.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.21775%252Fcimb.017.011%26sid%3Dliteratum%253Aachs%26aulast%3DGhafourian%26aufirst%3DS.%26aulast%3DSadeghifard%26aufirst%3DN.%26aulast%3DSoheili%26aufirst%3DS.%26aulast%3DSekawi%26aufirst%3DZ.%26atitle%3DExtended%2520spectrum%2520beta-lactamases%253A%2520definition%252C%2520classification%2520and%2520epidemiology%26jtitle%3DCurr.%2520Issues%2520Mol.%2520Biol.%26date%3D2015%26volume%3D17%26spage%3D11%26epage%3D22%26doi%3D10.21775%2Fcimb.017.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Carattoli, A.</span></span> <span> </span><span class="NLM_article-title">Plasmids and the spread of resistance</span>. <i>Int. J. Med. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>303</i></span>,  <span class="NLM_fpage">298</span>– <span class="NLM_lpage">304</span>, <span class="refDoi"> DOI: 10.1016/j.ijmm.2013.02.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1016%2Fj.ijmm.2013.02.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=23499304" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BC3sXktlOgs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=303&publication_year=2013&pages=298-304&author=A.+Carattoli&title=Plasmids+and+the+spread+of+resistance&doi=10.1016%2Fj.ijmm.2013.02.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Plasmids and the spread of resistance</span></div><div class="casAuthors">Carattoli, Alessandra</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Medical Microbiology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">303</span>
        (<span class="NLM_cas:issue">6-7</span>),
    <span class="NLM_cas:pages">298-304</span>CODEN:
                <span class="NLM_cas:coden">IMEMFV</span>;
        ISSN:<span class="NLM_cas:issn">1438-4221</span>.
    
            (<span class="NLM_cas:orgname">Elsevier GmbH</span>)
        </div><div class="casAbstract">A review.  Plasmids represent one of the most difficult challenge for counteracting the dissemination of antimicrobial resistance.  They contribute to the spread of relevant resistance determinants, promoting horizontal gene transfer among unrelated bacteria.  Undistinguishable plasmids were identified in unrelated bacterial strains isolated at huge geog. distant area, with no apparent epidemiol. links.  These plasmids belong to families that are largely prevalent in naturally occurring bacteria, usually carry multiple phys. linked genetic determinants, conferring resistance to different classes of antibiotics simultaneously.  Plasmids also harbor virulence factors and addiction systems, promoting their stability and maintenance in the bacterial host, in different environmental conditions.  The characteristics of the most successful plasmids that were at the origin of the spread of carbapenemase, expanded-spectrum β-lactamase, and plasmid-mediated quinolone resistance genes are discussed in this review.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFn5H47TRIhLVg90H21EOLACvtfcHk0lgECUZnjiKvow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXktlOgs7s%253D&md5=5aa53c713a9210482f99e19a082eb629</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.ijmm.2013.02.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ijmm.2013.02.001%26sid%3Dliteratum%253Aachs%26aulast%3DCarattoli%26aufirst%3DA.%26atitle%3DPlasmids%2520and%2520the%2520spread%2520of%2520resistance%26jtitle%3DInt.%2520J.%2520Med.%2520Microbiol.%26date%3D2013%26volume%3D303%26spage%3D298%26epage%3D304%26doi%3D10.1016%2Fj.ijmm.2013.02.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Butler, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blaskovich, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, M. A.</span></span> <span> </span><span class="NLM_article-title">Antibiotics in the clinical pipeline at the end of 2015</span>. <i>J. Antibiot.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>70</i></span>,  <span class="NLM_fpage">3</span>– <span class="NLM_lpage">24</span>, <span class="refDoi"> DOI: 10.1038/ja.2016.72</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1038%2Fja.2016.72" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=27353164" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BC2sXislOgsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=2017&pages=3-24&author=M.+S.+Butlerauthor=M.+A.+Blaskovichauthor=M.+A.+Cooper&title=Antibiotics+in+the+clinical+pipeline+at+the+end+of+2015&doi=10.1038%2Fja.2016.72"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Antibiotics in the clinical pipeline at the end of 2015</span></div><div class="casAuthors">Butler, Mark S.; Blaskovich, Mark AT; Cooper, Matthew A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Antibiotics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">70</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">3-24</span>CODEN:
                <span class="NLM_cas:coden">JANTAJ</span>;
        ISSN:<span class="NLM_cas:issn">0021-8820</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  There is growing global recognition that the continued emergence of multidrug-resistant bacteria poses a serious threat to human health.  Action plans released by the World Health Organization and governments of the UK and USA in particular recognize that discovering new antibiotics, particularly those with new modes of action, is one essential element required to avert future catastrophic pandemics.  This review lists the 30 antibiotics and two β-lactamase/β-lactam combinations first launched since 2000, and analyzes in depth seven new antibiotics and two new β-lactam/β-lactamase inhibitor combinations launched since 2013.  The development status, mode of action, spectra of activity and genesis (natural product, natural product-derived, synthetic or protein/mammalian peptide) of the 37 compds. and six β-lactamase/β-lactam combinations being evaluated in clin. trials between 2013 and 2015 are discussed.  Compds. discontinued from clin. development since 2013 and new antibacterial pharmacophores are also reviewed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0BwmpOtSH87Vg90H21EOLACvtfcHk0lhChBvXNcf6wQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXislOgsg%253D%253D&md5=067fba1f69db183c588a45ec888b6294</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fja.2016.72&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fja.2016.72%26sid%3Dliteratum%253Aachs%26aulast%3DButler%26aufirst%3DM.%2BS.%26aulast%3DBlaskovich%26aufirst%3DM.%2BA.%26aulast%3DCooper%26aufirst%3DM.%2BA.%26atitle%3DAntibiotics%2520in%2520the%2520clinical%2520pipeline%2520at%2520the%2520end%2520of%25202015%26jtitle%3DJ.%2520Antibiot.%26date%3D2017%26volume%3D70%26spage%3D3%26epage%3D24%26doi%3D10.1038%2Fja.2016.72" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">World Health Organization</span>.  <i>WHO List of Critically Important Antimicrobials
for Human Medicine (WHO CIA List)</i>. No. WHO/NMH/FOS/FZD/19.1; <span class="NLM_publisher-name">World Health Organization</span>, <span class="NLM_year">2019</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&author=World+Health+Organization&title=WHO+List+of+Critically+Important+Antimicrobials%0Afor+Human+Medicine+%28WHO+CIA+List%29"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3D%26btitle%3DWHO%2520List%2520of%2520Critically%2520Important%2520Antimicrobials%250Afor%2520Human%2520Medicine%2520%2528WHO%2520CIA%2520List%2529%26pub%3DWorld%2520Health%2520Organization%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Appelbaum, P. C.</span></span> <span> </span><span class="NLM_article-title">2012 and beyond: potential for the start of a second pre-antibiotic era?</span>. <i>J. Antimicrob. Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">2062</span>– <span class="NLM_lpage">2068</span>, <span class="refDoi"> DOI: 10.1093/jac/dks213</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1093%2Fjac%2Fdks213" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=22687888" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1egt73M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2012&pages=2062-2068&author=P.+C.+Appelbaum&title=2012+and+beyond%3A+potential+for+the+start+of+a+second+pre-antibiotic+era%3F&doi=10.1093%2Fjac%2Fdks213"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">2012 and beyond: potential for the start of a second pre-antibiotic era?</span></div><div class="casAuthors">Appelbaum, Peter C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Antimicrobial Chemotherapy</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2062-2068</span>CODEN:
                <span class="NLM_cas:coden">JACHDX</span>;
        ISSN:<span class="NLM_cas:issn">0305-7453</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The past decade has seen an alarming confluence of circumstances antithetical to development of new antibacterials, and most of the very few new drugs under development have problems with their microbiol., toxicity, or pharmacokinetics and pharmacodynamics.  Most large pharmaceutical companies have divested themselves of their antibiotic portfolios, and the promise that start-up companies would fill the niche has not been fulfilled.  What is left of the field is being stifled by bureaucratic regulations, problems with approval, lack of expertise and a general lack of understanding of how serious the situation is.  In paediatrics, in particular, there is no sign of any new antibacterial in the foreseeable future.  Waiting for initiatives and deliberations will probably take too long, and a vigorous and deliberate effort to educate the public-who in their turn can apply the needed pressure-seems to be the only way of achieving a rapid turnaround of this most dangerous situation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmZRXWCxg-e7Vg90H21EOLACvtfcHk0lhChBvXNcf6wQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1egt73M&md5=254248181dd6ad25cf4c7ef7ba74ce7c</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1093%2Fjac%2Fdks213&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjac%252Fdks213%26sid%3Dliteratum%253Aachs%26aulast%3DAppelbaum%26aufirst%3DP.%2BC.%26atitle%3D2012%2520and%2520beyond%253A%2520potential%2520for%2520the%2520start%2520of%2520a%2520second%2520pre-antibiotic%2520era%253F%26jtitle%3DJ.%2520Antimicrob.%2520Chemother.%26date%3D2012%26volume%3D67%26spage%3D2062%26epage%3D2068%26doi%3D10.1093%2Fjac%2Fdks213" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arias, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, B. E.</span></span> <span> </span><span class="NLM_article-title">Emergence and management of drug-resistant enterococcal infections</span>. <i>Expert Rev. Anti-Infect. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">637</span>– <span class="NLM_lpage">655</span>, <span class="refDoi"> DOI: 10.1586/14787210.6.5.637</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1586%2F14787210.6.5.637" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=18847403" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BD1cXht1Glu7vJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2008&pages=637-655&author=C.+A.+Ariasauthor=B.+E.+Murray&title=Emergence+and+management+of+drug-resistant+enterococcal+infections&doi=10.1586%2F14787210.6.5.637"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Emergence and management of drug-resistant enterococcal infections</span></div><div class="casAuthors">Arias, Cesar A.; Murray, Barbara E.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Review of Anti-Infective Therapy</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">637-655</span>CODEN:
                <span class="NLM_cas:coden">ERATCK</span>;
        ISSN:<span class="NLM_cas:issn">1478-7210</span>.
    
            (<span class="NLM_cas:orgname">Expert Reviews Ltd.</span>)
        </div><div class="casAbstract">A review.  The treatment of multidrug-resistant enterococcal infections continues to be a challenge for clinicians.  Glycopeptide and β-lactam resistance is now a common feature of the majority of Enterococcus faecium hospital isolates, and resistance to aminoglycosides, quinupristin-dalfopristin, linezolid and daptomycin further complicates the problem.  New antibiotics, such as tigecycline, lipoglycopeptides (dalbavancin, oritavancin and telavancin) and cephalosporins with activity against Enterococcus faecalis (ceftobiprole and ceftaroline), may have potential activity against certain resistant enterococcal strains in specific clin. settings, as may some older antibiotics, such as ampicillin, chloramphenicol, doxycycline, minocycline and nitrofurantoin.  However, the treatment of endovascular infections (particularly endocarditis, where bactericidal therapy is important for optimal cure rates) caused by resistant enterococci continues to be an immense challenge even with the availability of new agents.  The optimal therapy for these infections is not well established and clin. data are usually limited to case reports with conflicting results.  Therefore, treatment decisions may have to be based on animal models and sporadic experiences and the best approach is for the physician to consider carefully each patient on a case by case manner and gather all the clin. and microbiol. information possible regarding species identification and susceptibilities in order to choose a therapeutic regimen that would appear to be active.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSBKIGI357NbVg90H21EOLACvtfcHk0lhChBvXNcf6wQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXht1Glu7vJ&md5=901c7c8d6d85071cd0655938be231d36</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1586%2F14787210.6.5.637&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1586%252F14787210.6.5.637%26sid%3Dliteratum%253Aachs%26aulast%3DArias%26aufirst%3DC.%2BA.%26aulast%3DMurray%26aufirst%3DB.%2BE.%26atitle%3DEmergence%2520and%2520management%2520of%2520drug-resistant%2520enterococcal%2520infections%26jtitle%3DExpert%2520Rev.%2520Anti-Infect.%2520Ther.%26date%3D2008%26volume%3D6%26spage%3D637%26epage%3D655%26doi%3D10.1586%2F14787210.6.5.637" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yong, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toleman, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giske, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sundman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walsh, T. R.</span></span> <span> </span><span class="NLM_article-title">Characterization of a new metallo-β-lactamase gene, blaNDM-1, and a novel erythromycin esterase gene carried on a unique genetic structure in Klebsiella pneumoniae sequence type 14 from India</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">5046</span>– <span class="NLM_lpage">5054</span>, <span class="refDoi"> DOI: 10.1128/aac.00774-09</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1128%2Faac.00774-09" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=19770275" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsF2gs7jK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2009&pages=5046-5054&author=D.+Yongauthor=M.+A.+Tolemanauthor=C.+G.+Giskeauthor=H.+S.+Choauthor=K.+Sundmanauthor=K.+Leeauthor=T.+R.+Walsh&title=Characterization+of+a+new+metallo-%CE%B2-lactamase+gene%2C+blaNDM-1%2C+and+a+novel+erythromycin+esterase+gene+carried+on+a+unique+genetic+structure+in+Klebsiella+pneumoniae+sequence+type+14+from+India&doi=10.1128%2Faac.00774-09"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of a new metallo-β-lactamase gene, blaNDM-1, and a novel erythromycin esterase gene carried on a unique genetic structure in Klebsiella pneumoniae sequence type 14 from India</span></div><div class="casAuthors">Yong, Dongeun; Toleman, Mark A.; Giske, Christian G.; Cho, Hyun S.; Sundman, Kristina; Lee, Kyungwon; Walsh, Timothy R.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">5046-5054</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">A Swedish patient of Indian origin traveled to New Delhi, India, and acquired a urinary tract infection caused by a carbapenem-resistant Klebsiella pneumoniae strain that typed to the sequence type 14 complex.  The isolate, Klebsiella pneumoniae 05-506, was shown to possess a metallo-β-lactamase (MBL) but was neg. for previously known MBL genes.  Gene libraries and amplification of class 1 integrons revealed three resistance-conferring regions; the first contained blaCMY-4 flanked by ISEcP1 and blc.  The second region of 4.8 kb contained a complex class 1 integron with the gene cassettes arr-2, a new erythromycin esterase gene; ereC; aadA1; and cmlA7.  An intact ISCR1 element was shown to be downstream from the qac/sul genes.  The third region consisted of a new MBL gene, designated blaNDM-1, flanked on one side by K. pneumoniae DNA and a truncated IS26 element on its other side.  The last two regions lie adjacent to one another, and all three regions are found on a 180-kb region that is easily transferable to recipient strains and that confers resistance to all antibiotics except fluoroquinolones and colistin.  NDM-1 shares very little identity with other MBLs, with the most similar MBLs being VIM-1/VIM-2, with which it has only 32.4% identity.  As well as possessing unique residues near the active site, NDM-1 also has an addnl. insert between positions 162 and 166 not present in other MBLs.  NDM-1 has a mol. mass of 28 kDa, is monomeric, and can hydrolyze all β-lactams except aztreonam.  Compared to VIM-2, NDM-1 displays tighter binding to most cephalosporins, in particular, cefuroxime, cefotaxime, and cephalothin (cefalotin), and also to the penicillins.  NDM-1 does not bind to the carbapenems as tightly as IMP-1 or VIM-2 and turns over the carbapenems at a rate similar to that of VIM-2.  In addn. to K. pneumoniae 05-506, blaNDM-1 was found on a 140-kb plasmid in an Escherichia coli strain isolated from the patient's feces, inferring the possibility of in vivo conjugation.  The broad resistance carried on these plasmids is a further worrying development for India, which already has high levels of antibiotic resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxWiGWRq2gPrVg90H21EOLACvtfcHk0ljnOSyaZqFiRg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsF2gs7jK&md5=ff1df02887753da53b870d4c6edfde39</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1128%2Faac.00774-09&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252Faac.00774-09%26sid%3Dliteratum%253Aachs%26aulast%3DYong%26aufirst%3DD.%26aulast%3DToleman%26aufirst%3DM.%2BA.%26aulast%3DGiske%26aufirst%3DC.%2BG.%26aulast%3DCho%26aufirst%3DH.%2BS.%26aulast%3DSundman%26aufirst%3DK.%26aulast%3DLee%26aufirst%3DK.%26aulast%3DWalsh%26aufirst%3DT.%2BR.%26atitle%3DCharacterization%2520of%2520a%2520new%2520metallo-%25CE%25B2-lactamase%2520gene%252C%2520blaNDM-1%252C%2520and%2520a%2520novel%2520erythromycin%2520esterase%2520gene%2520carried%2520on%2520a%2520unique%2520genetic%2520structure%2520in%2520Klebsiella%2520pneumoniae%2520sequence%2520type%252014%2520from%2520India%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2009%26volume%3D53%26spage%3D5046%26epage%3D5054%26doi%3D10.1128%2Faac.00774-09" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Livermore, D. M.</span></span> <span> </span><span class="NLM_article-title">The need for new antibiotics</span>. <i>Clin. Microbiol. Infect.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">9</span>, <span class="refDoi"> DOI: 10.1111/j.1465-0691.2004.1004.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1111%2Fj.1465-0691.2004.1004.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=15522034" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A280%3ADC%252BD2crltVKjtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2004&pages=1-9&author=D.+M.+Livermore&title=The+need+for+new+antibiotics&doi=10.1111%2Fj.1465-0691.2004.1004.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">The need for new antibiotics</span></div><div class="casAuthors">Livermore D M</div><div class="citationInfo"><span class="NLM_cas:title">Clinical microbiology and infection : the official publication of the European Society of Clinical Microbiology and Infectious Diseases</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">10 Suppl 4</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1-9</span>
        ISSN:<span class="NLM_cas:issn">1198-743X</span>.
    </div><div class="casAbstract">Politicians and public health officials have joined specialist professionals in recognising antibiotic resistance as a threat to modern medicine.  Their response has centred on minimising unnecessary antibiotic prescribing, aiming to reduce selection pressure for resistance.  Despite a few hopeful trends (e.g., declining penicillin resistance among pneumococci in the UK), established resistance is proving hard to displace; moreover, new resistances continue to emerge and to proliferate at new sites.  There consequently remains a strong need for new antibiotics, particularly those directed against multiresistant Gram-negative bacteria in hospitals.  Already some nonfermenters of the genera Acinetobacter and Pseudomonas are resistant to all good antibiotics and many Enterobacteriaceae are resistant to all except carbapenems.  There is also a growing need for new agents against community-acquired pathogens, including the agents of tuberculosis, gonorrhoea and urinary tract infections.  Unless antibacterial development is re-energised, there is a serious risk that a growing proportion of infections, especially in hospitals, will become effectively untreatable.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRta44EV8rLicjl5L8hwNX0fW6udTcc2eaLG-9_k122DLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2crltVKjtg%253D%253D&md5=ed66c9ae026dbb5f5b79bc5ad56106b5</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1111%2Fj.1465-0691.2004.1004.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1465-0691.2004.1004.x%26sid%3Dliteratum%253Aachs%26aulast%3DLivermore%26aufirst%3DD.%2BM.%26atitle%3DThe%2520need%2520for%2520new%2520antibiotics%26jtitle%3DClin.%2520Microbiol.%2520Infect.%26date%3D2004%26volume%3D10%26spage%3D1%26epage%3D9%26doi%3D10.1111%2Fj.1465-0691.2004.1004.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blaskovich, M. A.</span></span> <span> </span><span class="NLM_article-title">The diminished antimicrobial pipeline</span>. <i>Microbiol. Aust.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">92</span>– <span class="NLM_lpage">96</span>, <span class="refDoi"> DOI: 10.1071/ma19025</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1071%2Fma19025" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2019&pages=92-96&author=M.+A.+Blaskovich&title=The+diminished+antimicrobial+pipeline&doi=10.1071%2Fma19025"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1071%2Fma19025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1071%252Fma19025%26sid%3Dliteratum%253Aachs%26aulast%3DBlaskovich%26aufirst%3DM.%2BA.%26atitle%3DThe%2520diminished%2520antimicrobial%2520pipeline%26jtitle%3DMicrobiol.%2520Aust.%26date%3D2019%26volume%3D40%26spage%3D92%26epage%3D96%26doi%3D10.1071%2Fma19025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Walsh, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wencewicz, T. A.</span></span> <span> </span><span class="NLM_article-title">Prospects for new antibiotics: a molecule-centered perspective</span>. <i>J. Antibiot.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">7</span>– <span class="NLM_lpage">22</span>, <span class="refDoi"> DOI: 10.1038/ja.2013.49</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1038%2Fja.2013.49" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=23756684" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht1yitrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2014&pages=7-22&author=C.+T.+Walshauthor=T.+A.+Wencewicz&title=Prospects+for+new+antibiotics%3A+a+molecule-centered+perspective&doi=10.1038%2Fja.2013.49"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Prospects for new antibiotics: a molecule-centered perspective</span></div><div class="casAuthors">Walsh, Christopher T.; Wencewicz, Timothy A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Antibiotics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">7-22</span>CODEN:
                <span class="NLM_cas:coden">JANTAJ</span>;
        ISSN:<span class="NLM_cas:issn">0021-8820</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  There is a continuous need for iterative cycles of antibiotic discovery and development to deal with the selection of resistant pathogens that emerge as therapeutic application of an antibiotic becomes widespread.  A short golden age of antibiotic discovery from nature followed by a subsequent golden half century of medicinal chem. optimization of existing mol. scaffolds emphasizes the need for new antibiotic mol. frameworks.  The authors bring a mol.-centered perspective to the questions of where will new scaffolds come from, when will chemogenetic approaches yield useful new antibiotics and what existing bacterial targets merit contemporary re-examn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8hr8BTzzMG7Vg90H21EOLACvtfcHk0ljnOSyaZqFiRg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht1yitrc%253D&md5=fb100c0954679a78210c497daa4fab1d</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fja.2013.49&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fja.2013.49%26sid%3Dliteratum%253Aachs%26aulast%3DWalsh%26aufirst%3DC.%2BT.%26aulast%3DWencewicz%26aufirst%3DT.%2BA.%26atitle%3DProspects%2520for%2520new%2520antibiotics%253A%2520a%2520molecule-centered%2520perspective%26jtitle%3DJ.%2520Antibiot.%26date%3D2014%26volume%3D67%26spage%3D7%26epage%3D22%26doi%3D10.1038%2Fja.2013.49" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Goemaere, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melet, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larue, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lieutaud, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alves de
Sousa, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chevalier, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yimga-Djapa, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giglione, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huguet, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alimi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meinnel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dardel, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Artaud, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pagès, J.-M.</span></span> <span> </span><span class="NLM_article-title">New peptide deformylase inhibitors and cooperative interaction: a combination to improve antibacterial activity</span>. <i>J. Antimicrob. Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">1392</span>– <span class="NLM_lpage">1400</span>, <span class="refDoi"> DOI: 10.1093/jac/dks058</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1093%2Fjac%2Fdks058" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=22378679" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BC38XmvF2qsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2012&pages=1392-1400&author=E.+Goemaereauthor=A.+Meletauthor=V.+Larueauthor=A.+Lieutaudauthor=R.+Alves+de%0ASousaauthor=J.+Chevalierauthor=L.+Yimga-Djapaauthor=C.+Giglioneauthor=F.+Huguetauthor=M.+Alimiauthor=T.+Meinnelauthor=F.+Dardelauthor=I.+Artaudauthor=J.-M.+Pag%C3%A8s&title=New+peptide+deformylase+inhibitors+and+cooperative+interaction%3A+a+combination+to+improve+antibacterial+activity&doi=10.1093%2Fjac%2Fdks058"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">New peptide deformylase inhibitors and cooperative interaction: a combination to improve antibacterial activity</span></div><div class="casAuthors">Goemaere, Emilie; Melet, Armelle; Larue, Valery; Lieutaud, Aurelie; Alves de Sousa, Rodolphe; Chevalier, Jacqueline; Yimga-Djapa, Liselotte; Giglione, Carmela; Huguet, Florian; Alimi, Mickael; Meinnel, Thierry; Dardel, Frederic; Artaud, Isabelle; Pages, Jean-Marie</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Antimicrobial Chemotherapy</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1392-1400</span>CODEN:
                <span class="NLM_cas:coden">JACHDX</span>;
        ISSN:<span class="NLM_cas:issn">0305-7453</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Bacterial drug resistance is a worrying public health problem and there is an urgent need for research and development to provide new antibacterial mols.  Peptide deformylase (PDF) is now a well-described intracellular target selected for the design of a new antibiotic group, PDF inhibitors (PDFIs).  The initial bacterial susceptibility to an inhibitor of a cytoplasmic target is directly assocd. with the diffusion of the compd. through the membrane barrier of Gram-neg. bacteria and with its cytosolic accumulation at the required concn.  The authors have recently demonstrated that the activity of different PDFIs is strongly dependent on the accumulation of the active mols. by using permeabilizing agents, efflux inhibitors or efflux-mutated strains.  Here, they assessed various combination protocols using different putative inhibitors (PDFIs, methionine aminopeptidase inhibitors etc.) to improve antibacterial activity against various resistant Gram-neg. bacteria.  The max. effect was obsd. when combining actinonin with a dual inhibitor of methionine aminopeptidase and PDF, this mol. being also able to interact with the target while actinonin is bound to the PDF active site.  Thus, such a combination of inhibitors acting on two tightly assocd. metabolic steps results in a cooperative effect on bacterial cells and opens an original way to combat multidrug-resistant bacteria.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoh9Jnoz6NUNrVg90H21EOLACvtfcHk0ljgVWHyLe7TLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmvF2qsb4%253D&md5=fa53cc46eb9e4193a62f34c27609254c</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1093%2Fjac%2Fdks058&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjac%252Fdks058%26sid%3Dliteratum%253Aachs%26aulast%3DGoemaere%26aufirst%3DE.%26aulast%3DMelet%26aufirst%3DA.%26aulast%3DLarue%26aufirst%3DV.%26aulast%3DLieutaud%26aufirst%3DA.%26aulast%3DAlves%2Bde%2BSousa%26aufirst%3DR.%26aulast%3DChevalier%26aufirst%3DJ.%26aulast%3DYimga-Djapa%26aufirst%3DL.%26aulast%3DGiglione%26aufirst%3DC.%26aulast%3DHuguet%26aufirst%3DF.%26aulast%3DAlimi%26aufirst%3DM.%26aulast%3DMeinnel%26aufirst%3DT.%26aulast%3DDardel%26aufirst%3DF.%26aulast%3DArtaud%26aufirst%3DI.%26aulast%3DPag%25C3%25A8s%26aufirst%3DJ.-M.%26atitle%3DNew%2520peptide%2520deformylase%2520inhibitors%2520and%2520cooperative%2520interaction%253A%2520a%2520combination%2520to%2520improve%2520antibacterial%2520activity%26jtitle%3DJ.%2520Antimicrob.%2520Chemother.%26date%3D2012%26volume%3D67%26spage%3D1392%26epage%3D1400%26doi%3D10.1093%2Fjac%2Fdks058" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Singh, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miesel, L.</span></span> <span> </span><span class="NLM_article-title">Screening strategies for discovery of antibacterial natural products</span>. <i>Expert Rev. Anti-Infect. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">589</span>– <span class="NLM_lpage">613</span>, <span class="refDoi"> DOI: 10.1586/eri.11.81</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1586%2Feri.11.81" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=21819327" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtVSlu7nF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2011&pages=589-613&author=S.+B.+Singhauthor=K.+Youngauthor=L.+Miesel&title=Screening+strategies+for+discovery+of+antibacterial+natural+products&doi=10.1586%2Feri.11.81"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Screening strategies for discovery of antibacterial natural products</span></div><div class="casAuthors">Singh, Sheo B.; Young, Katherine; Miesel, Lynn</div><div class="citationInfo"><span class="NLM_cas:title">Expert Review of Anti-Infective Therapy</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">589-613</span>CODEN:
                <span class="NLM_cas:coden">ERATCK</span>;
        ISSN:<span class="NLM_cas:issn">1478-7210</span>.
    
            (<span class="NLM_cas:orgname">Expert Reviews Ltd.</span>)
        </div><div class="casAbstract">A review.  Microbial-derived natural products have been a traditional source of antibiotics and antibiotic leads and continue to be effective sources of antibiotics today.  The most important of these discoveries were made about 50 years ago.  Chem. modifications of natural products discovered during those years continue to produce new clin. agents but their value is now, unfortunately, fading away owing to the exhaustion of opportunities of chem. modifications.  The discovery of new natural antibiotics is directly linked to new screening technologies, particularly technologies that can help to eliminate the rediscovery of known antibiotics.  In this article, we have reviewed the screening technologies from recent literature as well as originating from authors labs. that were used for the screening of natural products.  The article covers the entire spectrum of screening strategies, including classical empiric whole-cell assays to more sophisticated antisense based hypersensitive Staphylococcus aureus Fitness Test assays designed to screen all targets simultaneously.  These technologies have led to the discovery of a series of natural product antibiotics, which have been summarized, including the discovery of platensimycin, platencin, nocathiacins, philipimycin, cyclothialidine and muryamycins.  It is quite clear that natural products provide a tremendous opportunity to discover new antibiotics when combined with new hyper-sensitive whole-cell technologies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2PhKCdqbtRLVg90H21EOLACvtfcHk0ljgVWHyLe7TLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtVSlu7nF&md5=53cc27c37e75a31dbdac87159e198c68</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1586%2Feri.11.81&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1586%252Feri.11.81%26sid%3Dliteratum%253Aachs%26aulast%3DSingh%26aufirst%3DS.%2BB.%26aulast%3DYoung%26aufirst%3DK.%26aulast%3DMiesel%26aufirst%3DL.%26atitle%3DScreening%2520strategies%2520for%2520discovery%2520of%2520antibacterial%2520natural%2520products%26jtitle%3DExpert%2520Rev.%2520Anti-Infect.%2520Ther.%26date%3D2011%26volume%3D9%26spage%3D589%26epage%3D613%26doi%3D10.1586%2Feri.11.81" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pastagia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuch, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischetti, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, D. B.</span></span> <span> </span><span class="NLM_article-title">Lysins: the arrival of pathogen-directed anti-infectives</span>. <i>J. Med. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">1506</span>– <span class="NLM_lpage">1516</span>, <span class="refDoi"> DOI: 10.1099/jmm.0.061028-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1099%2Fjmm.0.061028-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=23813275" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslymtL7P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2013&pages=1506-1516&author=M.+Pastagiaauthor=R.+Schuchauthor=V.+A.+Fischettiauthor=D.+B.+Huang&title=Lysins%3A+the+arrival+of+pathogen-directed+anti-infectives&doi=10.1099%2Fjmm.0.061028-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Lysins: the arrival of pathogen-directed anti-infectives</span></div><div class="casAuthors">Pastagia, Mina; Schuch, Raymond; Fischetti, Vincent A.; Huang, David B.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medical Microbiology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1506-1516</span>CODEN:
                <span class="NLM_cas:coden">JMMIAV</span>;
        ISSN:<span class="NLM_cas:issn">0022-2615</span>.
    
            (<span class="NLM_cas:orgname">Society for General Microbiology</span>)
        </div><div class="casAbstract">A review.  Lysins represent a novel class of anti-infectives derived from bacteriophage.  Lysins are bacterial cell-wall hydrolytic enzymes that selectively and rapidly kill (≥3 log c.f.u. in 30 min) specific Gram-pos. bacteria providing a targeted therapeutic approach with minimal impact on unrelated commensal flora.  The potential for bacterial resistance to lysins is considered low due to targeting of highly conserved peptidoglycan components.  Through cutting-edge genetic engineering, lysins can be assembled into large libraries of anti-infective agents tailored to any bacterium of interest including drug-resistant Gram-pos. pathogens such as meticillin- and vancomycin-resistant Staphylococcus aureus, vancomycin-resistant Enterococcus faecalis and Enterococcus faecium, and penicillin-resistant Streptococcus pneumoniae.  Lysins can eliminate bacteria systemically and topically from mucosal surfaces and biofilms, as evidenced by exptl. models of sepsis, endocarditis, pneumonia, meningitis, and nasopharyngeal, skin and vaginal decolonization.  Furthermore, lysins can act synergistically with antibiotics and, in the process, resensitize bacteria to non-susceptible antibiotics.  Clin. trials are being prepd. to assess the safety and pharmacokinetic properties of lysins in humans.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowK9VT81rblbVg90H21EOLACvtfcHk0ljgVWHyLe7TLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslymtL7P&md5=be25021fa548b979f8aac39f2b9cf69e</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1099%2Fjmm.0.061028-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1099%252Fjmm.0.061028-0%26sid%3Dliteratum%253Aachs%26aulast%3DPastagia%26aufirst%3DM.%26aulast%3DSchuch%26aufirst%3DR.%26aulast%3DFischetti%26aufirst%3DV.%2BA.%26aulast%3DHuang%26aufirst%3DD.%2BB.%26atitle%3DLysins%253A%2520the%2520arrival%2520of%2520pathogen-directed%2520anti-infectives%26jtitle%3DJ.%2520Med.%2520Microbiol.%26date%3D2013%26volume%3D62%26spage%3D1506%26epage%3D1516%26doi%3D10.1099%2Fjmm.0.061028-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bragg, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Westhuizen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coetsee, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boucher, C.</span></span> <span> </span><span class="NLM_article-title">Bacteriophages as potential treatment option for antibiotic resistant bacteria</span>. <i>Adv. Exp. Med. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>807</i></span>,  <span class="NLM_fpage">97</span>– <span class="NLM_lpage">110</span>, <span class="refDoi"> DOI: 10.1007/978-81-322-1777-0_7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1007%2F978-81-322-1777-0_7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=24619620" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BC2MXms12hsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=807&publication_year=2014&pages=97-110&author=R.+Braggauthor=W.+van+der+Westhuizenauthor=J.-Y.+Leeauthor=E.+Coetseeauthor=C.+Boucher&title=Bacteriophages+as+potential+treatment+option+for+antibiotic+resistant+bacteria&doi=10.1007%2F978-81-322-1777-0_7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Bacteriophages as Potential Treatment Option for Antibiotic Resistant Bacteria</span></div><div class="casAuthors">Bragg, Robert; van der Westhuizen, Wouter; Lee, Ji-Yun; Coetsee, Elke; Boucher, Charlotte</div><div class="citationInfo"><span class="NLM_cas:title">Advances in Experimental Medicine and Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">807</span>
        (<span class="NLM_cas:issue">Infectious Diseases and Nanomedicine I</span>),
    <span class="NLM_cas:pages">97-110</span>CODEN:
                <span class="NLM_cas:coden">AEMBAP</span>;
        ISSN:<span class="NLM_cas:issn">2214-8019</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  The world is facing an ever-increasing problem with antibiotic resistant bacteria and we are rapidly heading for a post-antibiotic era.  There is an urgent need to investigate alterative treatment options while there are still a few antibiotics left.  Bacteriophages are viruses that specifically target bacteria.  Before the development of antibiotics, some efforts were made to use bacteriophages as a treatment option, but most of this research stopped soon after the discovery of antibiotics.  There are two different replication options which bacteriophages employ.  These are the lytic and lysogenic life cycles.  Both these life cycles have potential as treatment options.  There are various advantages and disadvantages to the use of bacteriophages as treatment options.  The main advantage is the specificity of bacteriophages and treatments can be designed to specifically target pathogenic bacteria while not neg. affecting the normal microbiota.  There are various advantages to this.  However, the high level of specificity also creates potential problems, the main being the requirement of highly specific diagnostic procedures.  Another potential problem with phage therapy includes the development of immunity and limitations with the registration of phage therapy options.  The latter is driving research toward the expression of phage genes which break the bacterial cell wall, which could then be used as a treatment option.  Various aspects of phage therapy have been investigated in studies undertaken by our research group.  We have investigated specificity of phages to various avian pathogenic E. coli isolates.  Furthermore, the exciting NanoSAM technol. has been employed to investigate bacteriophage replication and aspects of this will be discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_73G00hXLbrVg90H21EOLACvtfcHk0lgmLAocWqEgKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXms12hsL4%253D&md5=590f57e33bf11b319e774d6e1a9212a9</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1007%2F978-81-322-1777-0_7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-81-322-1777-0_7%26sid%3Dliteratum%253Aachs%26aulast%3DBragg%26aufirst%3DR.%26aulast%3Dvan%2Bder%2BWesthuizen%26aufirst%3DW.%26aulast%3DLee%26aufirst%3DJ.-Y.%26aulast%3DCoetsee%26aufirst%3DE.%26aulast%3DBoucher%26aufirst%3DC.%26atitle%3DBacteriophages%2520as%2520potential%2520treatment%2520option%2520for%2520antibiotic%2520resistant%2520bacteria%26jtitle%3DAdv.%2520Exp.%2520Med.%2520Biol.%26date%3D2014%26volume%3D807%26spage%3D97%26epage%3D110%26doi%3D10.1007%2F978-81-322-1777-0_7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mosqueda, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gato, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roca, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">López, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Alegría, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernández
Cuenca, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martínez-Martínez, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pachón, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cisneros, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodríguez-Baño, J.</span></span> <span> </span><span class="NLM_article-title">Characterization of plasmids carrying the blaOXA-24/40 carbapenemase gene and the genes encoding the AbkA/AbkB proteins of a toxin/antitoxin system</span>. <i>J. Antimicrob. Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">2629</span>– <span class="NLM_lpage">2633</span>, <span class="refDoi"> DOI: 10.1093/jac/dku179</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1093%2Fjac%2Fdku179" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=24879663" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BC2MXis1Ciurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2014&pages=2629-2633&author=N.+Mosquedaauthor=E.+Gatoauthor=I.+Rocaauthor=M.+L%C3%B3pezauthor=C.+R.+de+Alegr%C3%ADaauthor=F.+Fern%C3%A1ndez%0ACuencaauthor=L.+Mart%C3%ADnez-Mart%C3%ADnezauthor=J.+Pach%C3%B3nauthor=J.+M.+Cisnerosauthor=J.+Rodr%C3%ADguez-Ba%C3%B1o&title=Characterization+of+plasmids+carrying+the+blaOXA-24%2F40+carbapenemase+gene+and+the+genes+encoding+the+AbkA%2FAbkB+proteins+of+a+toxin%2Fantitoxin+system&doi=10.1093%2Fjac%2Fdku179"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of plasmids carrying the blaOXA-24/40 carbapenemase gene and the genes encoding the AbkA/AbkB proteins of a toxin/antitoxin system</span></div><div class="casAuthors">Mosqueda, Noraida; Gato, Eva; Roca, Ignasi; Lopez, Maria; Ruiz de Alegria, Carlos; Cuenca, Felipe Fernandez; Martinez-Martinez, Luis; Pachon, Jeronimo; Cisneros, Jose Miguel; Rodriguez-Bano, Jesus; Pascual, Alvaro; Vila, Jordi; Bou, German; Tomas, Maria</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Antimicrobial Chemotherapy</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2629-2633</span>CODEN:
                <span class="NLM_cas:coden">JACHDX</span>;
        ISSN:<span class="NLM_cas:issn">0305-7453</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Background: Carbapenem-resistant Acinetobacter baumannii (CRAb) is a major source of nosocomial infections in Spain assocd. with the prodn. of OXA-58-like or OXA-24/40-like β-lactamase enzymes.  We analyzed the plasmids carrying the blaOXA-24/40-like gene in CRAb isolates obtained a decade apart.  Methods: The presence of β-lactamases was screened for by PCR (metallo-β-lactamases, carbapenem-hydrolyzing class D β-lactamases, GES and KPC) in 101 CRAb isolates obtained in two multicentre studies (GEIH/REIPI-Ab-2000 and GEIH/REIPI-Ab-2010; n = 493 Acinetobacter spp).  We analyzed the distribution and characterization of the plasmids carrying the blaOXA-24/40-like gene and sequenced two plasmids, AbATCC223p (2000) and AbATCC329p (2010) from A. baumannii ATCC 17978 transformants.  Results: Acquisition of the blaOXA-24/40-like gene was the main mechanism underlying resistance to carbapenems (48.7% in 2000 compared with 51.6% in 2010).  This gene was mainly isolated in ST2 A. baumannii strains in both studies, although some novel STs (ST79 and ST80) appeared in 2010.  The gene was located in plasmids (8-12 kbp) assocd. with the repAci2 or repAci2/repGR12 types.  The sequences of AbATCC223p (8840 bp) and AbATCC329p (8842 bp) plasmids were similar, particularly regarding the presence of the genes encoding the AbkA/AbkB proteins assocd. with the toxin/antitoxin system.  Moreover, the abkA/abkB gene sequences (>96% identity) were also located in plasmids harbouring the blaOXA-58-like gene.  Conclusions: The action of OXA-24/40 and OXA-58 β-lactamase-like enzymes represents the main mechanism underlying resistance to carbapenems in Spain in the last decade.  AbkA/AbkB proteins in the toxin/antitoxin system may be involved in the successful dissemination of plasmids carrying the blaOXA-24/40-like gene, and probably also the blaOXA-58-like gene, thus contributing to the plasmid stability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqiPMaW9VPKNrVg90H21EOLACvtfcHk0lgmLAocWqEgKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXis1Ciurk%253D&md5=5f034329f78f54bb3f11f6102a3f49d7</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1093%2Fjac%2Fdku179&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjac%252Fdku179%26sid%3Dliteratum%253Aachs%26aulast%3DMosqueda%26aufirst%3DN.%26aulast%3DGato%26aufirst%3DE.%26aulast%3DRoca%26aufirst%3DI.%26aulast%3DL%25C3%25B3pez%26aufirst%3DM.%26aulast%3Dde%2BAlegr%25C3%25ADa%26aufirst%3DC.%2BR.%26aulast%3DFern%25C3%25A1ndez%2BCuenca%26aufirst%3DF.%26aulast%3DMart%25C3%25ADnez-Mart%25C3%25ADnez%26aufirst%3DL.%26aulast%3DPach%25C3%25B3n%26aufirst%3DJ.%26aulast%3DCisneros%26aufirst%3DJ.%2BM.%26aulast%3DRodr%25C3%25ADguez-Ba%25C3%25B1o%26aufirst%3DJ.%26atitle%3DCharacterization%2520of%2520plasmids%2520carrying%2520the%2520blaOXA-24%252F40%2520carbapenemase%2520gene%2520and%2520the%2520genes%2520encoding%2520the%2520AbkA%252FAbkB%2520proteins%2520of%2520a%2520toxin%252Fantitoxin%2520system%26jtitle%3DJ.%2520Antimicrob.%2520Chemother.%26date%3D2014%26volume%3D69%26spage%3D2629%26epage%3D2633%26doi%3D10.1093%2Fjac%2Fdku179" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Okano, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Isley, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boger, D. L.</span></span> <span> </span><span class="NLM_article-title">Peripheral modifications of [Ψ[CH<sub>2</sub>NH]Tpg<sup>4</sup>] vancomycin with added synergistic mechanisms of action provide durable and potent antibiotics</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">E5052</span>– <span class="NLM_lpage">E5061</span>, <span class="refDoi"> DOI: 10.1073/pnas.1704125114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1073%2Fpnas.1704125114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=28559345" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BC2sXovVSitr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2017&pages=E5052-E5061&author=A.+Okanoauthor=N.+A.+Isleyauthor=D.+L.+Boger&title=Peripheral+modifications+of+%5B%CE%A8%5BCH2NH%5DTpg4%5D+vancomycin+with+added+synergistic+mechanisms+of+action+provide+durable+and+potent+antibiotics&doi=10.1073%2Fpnas.1704125114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Peripheral modifications of [Ψ[CH2NH]Tpg4]vancomycin with added synergistic mechanisms of action provide durable and potent antibiotics</span></div><div class="casAuthors">Okano, Akinori; Isley, Nicholas A.; Boger, Dale L.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">E5052-E5061</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Subsequent to binding pocket modifications designed to provide dual d-Ala-d-Ala/d-Ala-d-Lac binding that directly overcome the mol. basis of vancomycin resistance, peripheral structural changes have been explored to improve antimicrobial potency and provide addnl. synergistic mechanisms of action.  A C-terminal peripheral modification, introducing a quaternary ammonium salt, is reported and was found to provide a binding pocket-modified vancomycin analog with a second mechanism of action that is independent of d-Ala-d-Ala/d-Ala-d-Lac binding.  This modification, which induces cell wall permeability and is complementary to the glycopeptide inhibition of cell wall synthesis, was found to provide improvements in antimicrobial potency (200-fold) against vancomycin-resistant Enterococci (VRE).  Furthermore, it is shown that this type of C-terminal modification may be combined with a second peripheral (4-chlorobiphenyl)methyl (CBP) addn. to the vancomycin disaccharide to provide even more potent antimicrobial agents [VRE min. inhibitory concn. (MIC) = 0.01-0.005 μg/mL] with activity that can be attributed to three independent and synergistic mechanisms of action, only one of which requires d-Ala-d-Ala/d-Ala-d-Lac binding.  Finally, it is shown that such peripherally and binding pocket-modified vancomycin analogs display little propensity for acquired resistance by VRE and that their durability against such challenges as well as their antimicrobial potency follow now predictable trends (three > two > one mechanisms of action).  Such antibiotics are expected to display durable antimicrobial activity not prone to rapidly acquired clin. resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwl7lZFEjyJrVg90H21EOLACvtfcHk0lgmLAocWqEgKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXovVSitr8%253D&md5=c98e49ee55fa7f42ab921a43e6d66ed2</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1704125114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1704125114%26sid%3Dliteratum%253Aachs%26aulast%3DOkano%26aufirst%3DA.%26aulast%3DIsley%26aufirst%3DN.%2BA.%26aulast%3DBoger%26aufirst%3DD.%2BL.%26atitle%3DPeripheral%2520modifications%2520of%2520%255B%25CE%25A8%255BCH2NH%255DTpg4%255D%2520vancomycin%2520with%2520added%2520synergistic%2520mechanisms%2520of%2520action%2520provide%2520durable%2520and%2520potent%2520antibiotics%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2017%26volume%3D114%26spage%3DE5052%26epage%3DE5061%26doi%3D10.1073%2Fpnas.1704125114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Okano, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakayama, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindsey, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schammel, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boger, D. L.</span></span> <span> </span><span class="NLM_article-title">Total syntheses and initial evaluation of [Ψ[C(=S)NH]Tpg<sup>4</sup>]vancomycin, [Ψ[C(=NH) NH]Tpg<sup>4</sup>]vancomycin, [Ψ[CH<sub>2</sub>NH]Tpg<sup>4</sup>]vancomycin, and their (4-Chlorobiphenyl)methyl derivatives: synergistic binding pocket and peripheral modifications for the glycopeptide antibiotics</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>137</i></span>,  <span class="NLM_fpage">3693</span>– <span class="NLM_lpage">3704</span>, <span class="refDoi"> DOI: 10.1021/jacs.5b01008</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.5b01008" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BC2MXktVGkt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=137&publication_year=2015&pages=3693-3704&author=A.+Okanoauthor=A.+Nakayamaauthor=K.+Wuauthor=E.+A.+Lindseyauthor=A.+W.+Schammelauthor=Y.+Fengauthor=K.+C.+Collinsauthor=D.+L.+Boger&title=Total+syntheses+and+initial+evaluation+of+%5B%CE%A8%5BC%28%3DS%29NH%5DTpg4%5Dvancomycin%2C+%5B%CE%A8%5BC%28%3DNH%29+NH%5DTpg4%5Dvancomycin%2C+%5B%CE%A8%5BCH2NH%5DTpg4%5Dvancomycin%2C+and+their+%284-Chlorobiphenyl%29methyl+derivatives%3A+synergistic+binding+pocket+and+peripheral+modifications+for+the+glycopeptide+antibiotics&doi=10.1021%2Fjacs.5b01008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Total Syntheses and Initial Evaluation of [Ψ[C(=S)NH]Tpg4]vancomycin, [Ψ[C(=NH)NH]Tpg4]vancomycin, [Ψ[CH2NH]Tpg4]vancomycin, and Their (4-Chlorobiphenyl)methyl Derivatives: Synergistic Binding Pocket and Peripheral Modifications for the Glycopeptide Antibiotics</span></div><div class="casAuthors">Okano, Akinori; Nakayama, Atsushi; Wu, Kejia; Lindsey, Erick A.; Schammel, Alex W.; Feng, Yiqing; Collins, Karen C.; Boger, Dale L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">137</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3693-3704</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Full details of studies are disclosed on the total syntheses of binding pocket analogs of vancomycin bearing the peripheral L-vancosaminyl-1,2-D-glucosyl disaccharide that contain changes to a key single atom in the residue-4 amide (residue-4 carbonyl O → S, NH, H2) designed to directly address the underlying mol. basis of resistance to vancomycin.  Also disclosed are studies piloting the late-stage transformations conducted on the synthetically more accessible C-terminus hydroxymethyl aglycon derivs. and full details of the peripheral chlorobiphenyl functionalization of all of the binding-pocket-modified vancomycin analogs designed for dual D-Ala-D-Ala/D-Ala-D-Lac binding.  Their collective assessment indicates that combined binding pocket and chlorobiphenyl peripherally modified analogs exhibit a remarkable spectrum of antimicrobial activity (VSSA, MRSA, and VanA and VanB VRE) and impressive potencies against both vancomycin-sensitive and vancomycin-resistant bacteria (MICs = 0.06-0.005 and 0.5-0.06 μg/mL for the amidine and methylene analogs, resp.) and likely benefit from two independent and synergistic mechanisms of action, only one of which is dependent on D-Ala-D-Ala/D-Ala-D-Lac binding.  Such analogs are likely to display esp. durable antibiotic activity that is not prone to rapidly acquired clin. resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBtCGg4ti2Z7Vg90H21EOLACvtfcHk0lg-w0SUTSraNg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXktVGkt7Y%253D&md5=9287dfd1c6aca3bc9f3541ce554817fb</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fjacs.5b01008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.5b01008%26sid%3Dliteratum%253Aachs%26aulast%3DOkano%26aufirst%3DA.%26aulast%3DNakayama%26aufirst%3DA.%26aulast%3DWu%26aufirst%3DK.%26aulast%3DLindsey%26aufirst%3DE.%2BA.%26aulast%3DSchammel%26aufirst%3DA.%2BW.%26aulast%3DFeng%26aufirst%3DY.%26aulast%3DCollins%26aufirst%3DK.%2BC.%26aulast%3DBoger%26aufirst%3DD.%2BL.%26atitle%3DTotal%2520syntheses%2520and%2520initial%2520evaluation%2520of%2520%255B%25CE%25A8%255BC%2528%253DS%2529NH%255DTpg4%255Dvancomycin%252C%2520%255B%25CE%25A8%255BC%2528%253DNH%2529%2520NH%255DTpg4%255Dvancomycin%252C%2520%255B%25CE%25A8%255BCH2NH%255DTpg4%255Dvancomycin%252C%2520and%2520their%2520%25284-Chlorobiphenyl%2529methyl%2520derivatives%253A%2520synergistic%2520binding%2520pocket%2520and%2520peripheral%2520modifications%2520for%2520the%2520glycopeptide%2520antibiotics%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2015%26volume%3D137%26spage%3D3693%26epage%3D3704%26doi%3D10.1021%2Fjacs.5b01008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mandler, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baidin, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pahil, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owens, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kahne, D.</span></span> <span> </span><span class="NLM_article-title">Novobiocin enhances polymyxin activity by stimulating lipopolysaccharide transport</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>140</i></span>,  <span class="NLM_fpage">6749</span>– <span class="NLM_lpage">6753</span>, <span class="refDoi"> DOI: 10.1021/jacs.8b02283</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.8b02283" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BC1cXpt12hsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=140&publication_year=2018&pages=6749-6753&author=M.+D.+Mandlerauthor=V.+Baidinauthor=J.+Leeauthor=K.+S.+Pahilauthor=T.+W.+Owensauthor=D.+Kahne&title=Novobiocin+enhances+polymyxin+activity+by+stimulating+lipopolysaccharide+transport&doi=10.1021%2Fjacs.8b02283"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Novobiocin Enhances Polymyxin Activity by Stimulating Lipopolysaccharide Transport</span></div><div class="casAuthors">Mandler, Michael D.; Baidin, Vadim; Lee, James; Pahil, Karanbir S.; Owens, Tristan W.; Kahne, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">140</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">6749-6753</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Gram-neg. bacteria are challenging to kill with antibiotics due to their impenetrable outer membrane contg. lipopolysaccharide (LPS).  The polymyxins, including colistin, are the drugs of last resort for treating Gram-neg. infections.  These drugs bind LPS and disrupt the outer membrane; however, their toxicity limits their usefulness.  Polymyxin has been shown to synergize with many antibiotics including novobiocin, which inhibits DNA gyrase, by facilitating transport of these antibiotics across the outer membrane.  Recently, we have shown that novobiocin not only inhibits DNA gyrase but also binds and stimulates LptB, the ATPase that powers LPS transport.  Here, we report the synthesis of novobiocin derivs. that sep. these two activities.  One analog retains LptB-stimulatory activity but is unable to inhibit DNA gyrase.  This analog, which is not toxic on its own, nevertheless enhances the lethality of polymyxin by binding LptB and stimulating LPS transport.  Therefore, LPS transport agonism contributes substantially to novobiocin-polymyxin synergy.  We also report other novobiocin analogs that inhibit DNA gyrase better than or equal to novobiocin, but bind better to LptB and therefore have even greater LptB stimulatory activity.  These compds. are more potent than novobiocin when used in combination with polymyxin.  Novobiocin analogs optimized for both gyrase inhibition and LPS transport agonism may allow the use of lower doses of polymyxin, increasing its efficacy and safety.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUlbaNzErHFbVg90H21EOLACvtfcHk0lg-w0SUTSraNg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXpt12hsLw%253D&md5=5b22525ded3bcabbb9e4affb02f5e6be</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fjacs.8b02283&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.8b02283%26sid%3Dliteratum%253Aachs%26aulast%3DMandler%26aufirst%3DM.%2BD.%26aulast%3DBaidin%26aufirst%3DV.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DPahil%26aufirst%3DK.%2BS.%26aulast%3DOwens%26aufirst%3DT.%2BW.%26aulast%3DKahne%26aufirst%3DD.%26atitle%3DNovobiocin%2520enhances%2520polymyxin%2520activity%2520by%2520stimulating%2520lipopolysaccharide%2520transport%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2018%26volume%3D140%26spage%3D6749%26epage%3D6753%26doi%3D10.1021%2Fjacs.8b02283" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Neumann, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sassone-Corsi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raffatellu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nolan, E. M.</span></span> <span> </span><span class="NLM_article-title">Esterase-catalyzed siderophore hydrolysis activates an enterobactin-ciprofloxacin conjugate and confers targeted antibacterial activity</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>140</i></span>,  <span class="NLM_fpage">5193</span>– <span class="NLM_lpage">5201</span>, <span class="refDoi"> DOI: 10.1021/jacs.8b01042</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.8b01042" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BC1cXmtVWrurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=140&publication_year=2018&pages=5193-5201&author=W.+Neumannauthor=M.+Sassone-Corsiauthor=M.+Raffatelluauthor=E.+M.+Nolan&title=Esterase-catalyzed+siderophore+hydrolysis+activates+an+enterobactin-ciprofloxacin+conjugate+and+confers+targeted+antibacterial+activity&doi=10.1021%2Fjacs.8b01042"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Esterase-Catalyzed Siderophore Hydrolysis Activates an Enterobactin-Ciprofloxacin Conjugate and Confers Targeted Antibacterial Activity</span></div><div class="casAuthors">Neumann, Wilma; Sassone-Corsi, Martina; Raffatellu, Manuela; Nolan, Elizabeth M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">140</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">5193-5201</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Enteric Gram-neg. bacteria, including Escherichia coli, biosynthesize and deploy the triscatecholate siderophore enterobactin (Ent) in the vertebrate host to acquire iron, an essential nutrient.  We report that Ent-Cipro, a synthetic siderophore-antibiotic conjugate based on the native Ent platform that harbors an alkyl linker at one of the catechols with a ciprofloxacin cargo attached, affords targeted antibacterial activity against E. coli strains that express the pathogen-assocd. iroA gene cluster.  Attachment of the siderophore to ciprofloxacin, a DNA gyrase inhibitor and broad-spectrum antibiotic that is used to treat infections caused by E. coli, generates an inactive prodrug and guides the antibiotic into the cytoplasm of bacteria that express the Ent uptake machinery (FepABCDG).  Intracellular hydrolysis of the siderophore restores the activity of the antibiotic.  Remarkably, Fes, the cytoplasmic Ent hydrolase expressed by all E. coli, does not contribute to Ent-Cipro activation.  Instead, this processing step requires IroD, a cytoplasmic hydrolase that is only expressed by E. coli that harbor the iroA gene cluster and are predominantly pathogenic.  In the uropathogenic E. coli UTI89 and CFT073, Ent-Cipro provides antibacterial activity comparable to unmodified ciprofloxacin.  This work highlights the potential of leveraging and targeting pathogen-assocd. microbial enzymes in narrow-spectrum antibacterial approaches.  Moreover, because E. coli include harmless gut commensals as well as resident microbes that can contribute to disease, Ent-Cipro may provide a valuable chem. tool for strain-selective modulation of the microbiota.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqL1RwaM3wdnrVg90H21EOLACvtfcHk0lg-w0SUTSraNg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXmtVWrurY%253D&md5=d5fe2b8f151a0e611b9d5a25c4d45c85</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjacs.8b01042&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.8b01042%26sid%3Dliteratum%253Aachs%26aulast%3DNeumann%26aufirst%3DW.%26aulast%3DSassone-Corsi%26aufirst%3DM.%26aulast%3DRaffatellu%26aufirst%3DM.%26aulast%3DNolan%26aufirst%3DE.%2BM.%26atitle%3DEsterase-catalyzed%2520siderophore%2520hydrolysis%2520activates%2520an%2520enterobactin-ciprofloxacin%2520conjugate%2520and%2520confers%2520targeted%2520antibacterial%2520activity%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2018%26volume%3D140%26spage%3D5193%26epage%3D5201%26doi%3D10.1021%2Fjacs.8b01042" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xie, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manuguri, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Proietti, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inge, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Häll, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramström, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, M.</span></span> <span> </span><span class="NLM_article-title">Design and synthesis of theranostic antibiotic nanodrugs that display enhanced antibacterial activity and luminescence</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">8464</span>– <span class="NLM_lpage">8469</span>, <span class="refDoi"> DOI: 10.1073/pnas.1708556114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1073%2Fpnas.1708556114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=28743748" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1akt7%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2017&pages=8464-8469&author=S.+Xieauthor=S.+Manuguriauthor=G.+Proiettiauthor=J.+Romsonauthor=Y.+Fuauthor=A.+K.+Ingeauthor=B.+Wuauthor=Y.+Zhangauthor=D.+H%C3%A4llauthor=O.+Ramstr%C3%B6mauthor=M.+Yan&title=Design+and+synthesis+of+theranostic+antibiotic+nanodrugs+that+display+enhanced+antibacterial+activity+and+luminescence&doi=10.1073%2Fpnas.1708556114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Design and synthesis of theranostic antibiotic nanodrugs that display enhanced antibacterial activity and luminescence</span></div><div class="casAuthors">Xie, Sheng; Manuguri, Sesha; Proietti, Giampiero; Romson, Joakim; Fu, Ying; Inge, A. Ken; Wu, Bin; Zhang, Yang; Haell, Daniel; Ramstroem, Olof; Yan, Mingdi</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">32</span>),
    <span class="NLM_cas:pages">8464-8469</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">We report the modular formulation of ciprofloxacin-based pure theranostic nanodrugs that display enhanced antibacterial activities, as well as aggregation-induced emission (AIE) enhancement that was successfully used to image bacteria.  The drug derivs., consisting of ciprofloxacin, a perfluoroaryl ring, and a Ph ring linked by an amidine bond, were efficiently synthesized by a straightforward protocol from a perfluoroaryl azide, ciprofloxacin, and an aldehyde in acetone at room temp.  These compds. are propeller-shaped, and upon pptn. into water, readily assembled into stable nanoaggregates that transformed ciprofloxacin derivs. into AIE-active luminogens.  The nanoaggregates displayed increased luminescence and were successfully used to image bacteria.  In addn., these nanodrugs showed enhanced antibacterial activities, lowering the min. inhibitory concn. (MIC) by more than one order of magnitude against both sensitive and resistant Escherichia coli.  The study represents a strategy in the design and development of pure theranostic nanodrugs for combating drug-resistant bacterial infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1DDzoLoklYbVg90H21EOLACvtfcHk0lihNDiYRbI5Dg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1akt7%252FE&md5=1688d7d82216ab04699df95fb3a16d80</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1708556114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1708556114%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DS.%26aulast%3DManuguri%26aufirst%3DS.%26aulast%3DProietti%26aufirst%3DG.%26aulast%3DRomson%26aufirst%3DJ.%26aulast%3DFu%26aufirst%3DY.%26aulast%3DInge%26aufirst%3DA.%2BK.%26aulast%3DWu%26aufirst%3DB.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DH%25C3%25A4ll%26aufirst%3DD.%26aulast%3DRamstr%25C3%25B6m%26aufirst%3DO.%26aulast%3DYan%26aufirst%3DM.%26atitle%3DDesign%2520and%2520synthesis%2520of%2520theranostic%2520antibiotic%2520nanodrugs%2520that%2520display%2520enhanced%2520antibacterial%2520activity%2520and%2520luminescence%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2017%26volume%3D114%26spage%3D8464%26epage%3D8469%26doi%3D10.1073%2Fpnas.1708556114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Khanna, P.</span>; <span class="NLM_string-name">Snyder, E. S.</span>; <span class="NLM_string-name">Singh, P.</span></span> <span> </span><span class="NLM_article-title">Chloramphenicol derivatives</span>. U.S. Patent <span class="NLM_patent">4,489,157 A</span>, <span class="NLM_year">1984</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1984&author=P.+Khanna&author=E.+S.+Snyder&author=P.+Singh&title=Chloramphenicol+derivatives"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DKhanna%26aufirst%3DP.%26atitle%3DChloramphenicol%2520derivatives%26date%3D1984" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madrid, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, Y.</span></span> <span> </span><span class="NLM_article-title">Bacteria-targeting conjugates based on antimicrobial peptide for bacteria diagnosis and therapy</span>. <i>Mol. Pharm.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">2505</span>– <span class="NLM_lpage">2516</span>, <span class="refDoi"> DOI: 10.1021/acs.molpharmaceut.5b00053</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.molpharmaceut.5b00053" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BC2MXptlOgu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2015&pages=2505-2516&author=H.+Chenauthor=C.+Liuauthor=D.+Chenauthor=K.+Madridauthor=S.+Pengauthor=X.+Dongauthor=M.+Zhangauthor=Y.+Gu&title=Bacteria-targeting+conjugates+based+on+antimicrobial+peptide+for+bacteria+diagnosis+and+therapy&doi=10.1021%2Facs.molpharmaceut.5b00053"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Bacteria-Targeting Conjugates Based on Antimicrobial Peptide for Bacteria Diagnosis and Therapy</span></div><div class="casAuthors">Chen, Haiyan; Liu, Cuicui; Chen, Dan; Madrid, Kyle; Peng, Shuwen; Dong, Xinyue; Zhang, Min; Gu, Yueqing</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmaceutics</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2505-2516</span>CODEN:
                <span class="NLM_cas:coden">MPOHBP</span>;
        ISSN:<span class="NLM_cas:issn">1543-8384</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Chloramphenicol (CAP) is one of the most effective antimicrobial agents, but its therapeutic efficacy is greatly limited by its nonspecific distribution and consequent side effects in neutrophils.  Targeting to the infection sites, and thus restricting CAP nonselective delivery, provides an alternative way to overcome this limitation.  The antibacterial peptide fragment UBI29-41 was identified to have a high bacterial affinity.  However, no research so far has been carried out to utilize UBI29-41 as a ligand for bacteria-targeting therapies.  In this article, we first labeled a near-IR fluorescent dye (ICG02) with UBI29-41 to investigate its targeting capability in different bacteria (S. aureus, E. coli, and P. auruginosa) and bacteria-infected mouse models.  Subsequently, UBI29-41 was conjugated with the typical antibiotic (CAP) through the linker glutaric anhydride to form the conjugate CAP-UBI29-41 for the bacteria-targeting therapy.  In vitro studies demonstrated the enhanced antibacterial effects of CAP-UBI29-41 on S. aureus and E.coil.  Meanwhile, the toxicity of CAP-UBI29-41 on normal cells decreased distinctly in comparison with CAP.  Most importantly, CAP-UBI29-41 exhibited more favorable antibacterial efficacy than CAP in bacteria-bearing mouse models.  All these results demonstrated that UBI29-41 is an ideal targeting ligand to construct antibacterial agents for bacteria-targeting therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1DQ4CZtNRl7Vg90H21EOLACvtfcHk0lihNDiYRbI5Dg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXptlOgu7s%253D&md5=b3f3bded0087efdc8d35ab3977619f2c</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Facs.molpharmaceut.5b00053&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.molpharmaceut.5b00053%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DD.%26aulast%3DMadrid%26aufirst%3DK.%26aulast%3DPeng%26aufirst%3DS.%26aulast%3DDong%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DM.%26aulast%3DGu%26aufirst%3DY.%26atitle%3DBacteria-targeting%2520conjugates%2520based%2520on%2520antimicrobial%2520peptide%2520for%2520bacteria%2520diagnosis%2520and%2520therapy%26jtitle%3DMol.%2520Pharm.%26date%3D2015%26volume%3D12%26spage%3D2505%26epage%3D2516%26doi%3D10.1021%2Facs.molpharmaceut.5b00053" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yunis, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, U. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Restrepo, A.</span></span> <span> </span><span class="NLM_article-title">Reversible bone marrow suppression from chloramphenicol. A consequence of mitochondrial injury</span>. <i>Arch. Intern. Med.</i> <span class="NLM_year" style="font-weight: bold;">1970</span>,  <span class="NLM_volume"><i>126</i></span>,  <span class="NLM_fpage">272</span>– <span class="NLM_lpage">275</span>, <span class="refDoi"> DOI: 10.1001/archinte.126.2.272</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1001%2Farchinte.126.2.272" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=4914780" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A280%3ADyaE3c3ltV2rtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=126&publication_year=1970&pages=272-275&author=A.+A.+Yunisauthor=U.+S.+Smithauthor=A.+Restrepo&title=Reversible+bone+marrow+suppression+from+chloramphenicol.+A+consequence+of+mitochondrial+injury&doi=10.1001%2Farchinte.126.2.272"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Reversible bone marrow suppression from chloramphenicol. A consequence of mitochondrial injury</span></div><div class="casAuthors">Yunis A A; Smith U S; Restrepo A</div><div class="citationInfo"><span class="NLM_cas:title">Archives of internal medicine</span>
        (<span class="NLM_cas:date">1970</span>),
    <span class="NLM_cas:volume">126</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">272-5</span>
        ISSN:<span class="NLM_cas:issn">0003-9926</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTWItyHU5NDm0ewzGPswiWOfW6udTcc2eayQ6SVAarRNbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaE3c3ltV2rtg%253D%253D&md5=bde3fea9e17833d2fee5478aef5a22a1</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1001%2Farchinte.126.2.272&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1001%252Farchinte.126.2.272%26sid%3Dliteratum%253Aachs%26aulast%3DYunis%26aufirst%3DA.%2BA.%26aulast%3DSmith%26aufirst%3DU.%2BS.%26aulast%3DRestrepo%26aufirst%3DA.%26atitle%3DReversible%2520bone%2520marrow%2520suppression%2520from%2520chloramphenicol.%2520A%2520consequence%2520of%2520mitochondrial%2520injury%26jtitle%3DArch.%2520Intern.%2520Med.%26date%3D1970%26volume%3D126%26spage%3D272%26epage%3D275%26doi%3D10.1001%2Farchinte.126.2.272" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheetham, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Angacian, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, H.</span></span> <span> </span><span class="NLM_article-title">Peptide-drug conjugates as effective prodrug strategies for targeted delivery</span>. <i>Adv. Drug Delivery Rev.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>110–111</i></span>,  <span class="NLM_fpage">112</span>– <span class="NLM_lpage">126</span>, <span class="refDoi"> DOI: 10.1016/j.addr.2016.06.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1016%2Fj.addr.2016.06.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=27370248" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFSgtbvL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110%E2%80%93111&publication_year=2017&pages=112-126&author=Y.+Wangauthor=A.+G.+Cheethamauthor=G.+Angacianauthor=H.+Suauthor=L.+Xieauthor=H.+Cui&title=Peptide-drug+conjugates+as+effective+prodrug+strategies+for+targeted+delivery&doi=10.1016%2Fj.addr.2016.06.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Peptide-drug conjugates as effective prodrug strategies for targeted delivery</span></div><div class="casAuthors">Wang, Yin; Cheetham, Andrew G.; Angacian, Garren; Su, Hao; Xie, Lisi; Cui, Honggang</div><div class="citationInfo"><span class="NLM_cas:title">Advanced Drug Delivery Reviews</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">110-111</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">112-126</span>CODEN:
                <span class="NLM_cas:coden">ADDREP</span>;
        ISSN:<span class="NLM_cas:issn">0169-409X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Peptide-drug conjugates (PDCs) represent an important class of therapeutic agents that combine one or more drug mols. with a short peptide through a biodegradable linker.  This prodrug strategy uniquely and specifically exploits the biol. activities and self-assembling potential of small-mol. peptides to improve the treatment efficacy of medicinal compds.  We review here the recent progress in the design and synthesis of peptide-drug conjugates in the context of targeted drug delivery and cancer chemotherapy.  We analyze carefully the key design features in choosing the peptide sequence and linker chem. for the drug of interest, as well as the strategies to optimize the conjugate design.  We highlight the recent progress in the design and synthesis of self-assembling peptide-drug amphiphiles to construct supramol. nanomedicine and nanofiber hydrogels for both systemic and topical delivery of active pharmaceutical ingredients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxcgDx1p75N7Vg90H21EOLACvtfcHk0ljYlaL4CQbsww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFSgtbvL&md5=3a23bd362031d78b92bc167a44c62c96</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.addr.2016.06.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.addr.2016.06.015%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DY.%26aulast%3DCheetham%26aufirst%3DA.%2BG.%26aulast%3DAngacian%26aufirst%3DG.%26aulast%3DSu%26aufirst%3DH.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DCui%26aufirst%3DH.%26atitle%3DPeptide-drug%2520conjugates%2520as%2520effective%2520prodrug%2520strategies%2520for%2520targeted%2520delivery%26jtitle%3DAdv.%2520Drug%2520Delivery%2520Rev.%26date%3D2017%26volume%3D110%25E2%2580%2593111%26spage%3D112%26epage%3D126%26doi%3D10.1016%2Fj.addr.2016.06.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheetham, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moon, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moxley, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Y.-a.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, H.</span></span> <span> </span><span class="NLM_article-title">Controlled release of free doxorubicin from peptide-drug conjugates by drug loading</span>. <i>J. Control. Release</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>191</i></span>,  <span class="NLM_fpage">123</span>– <span class="NLM_lpage">130</span>, <span class="refDoi"> DOI: 10.1016/j.jconrel.2014.05.051</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1016%2Fj.jconrel.2014.05.051" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=24892976" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVWqsL%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=191&publication_year=2014&pages=123-130&author=Z.+Chenauthor=P.+Zhangauthor=A.+G.+Cheethamauthor=J.+H.+Moonauthor=J.+W.+Moxleyauthor=Y.-a.+Linauthor=H.+Cui&title=Controlled+release+of+free+doxorubicin+from+peptide-drug+conjugates+by+drug+loading&doi=10.1016%2Fj.jconrel.2014.05.051"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Controlled release of free doxorubicin from peptide-drug conjugates by drug loading</span></div><div class="casAuthors">Chen, Zhipeng; Zhang, Pengcheng; Cheetham, Andrew G.; Moon, Jae Hyon; Moxley, James W.; Lin, Yi-an; Cui, Honggang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Controlled Release</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">191</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">123-130</span>CODEN:
                <span class="NLM_cas:coden">JCREEC</span>;
        ISSN:<span class="NLM_cas:issn">0168-3659</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Covalent modification of a drug with a peptide moiety has been extensively used as an effective strategy to improve the drug's therapeutic outcome.  One important consideration in the design of such a prodrug is the release of the free drug from the covalently bound form in a desired fashion.  In most cases, the free drug release rate is controlled by the use of various chem. linkers that bridge the drug to the auxiliary segment.  We report here that the degree of drug conjugation per peptide could also regulate the drug release in addn. to its apparent effect on drug loading of the resulting conjugates.  In this work, we synthesized three peptide-drug conjugates (NTD, d-NTD and q-NTD) in which the cell penetrating peptide Tat is covalently connected to one, two, or four doxorubicin, resp., through a cathepsin B degradable tetrapeptide linker (-Gly-Phe-Leu-Gly-).  We found that the no. of doxorubicin within the conjugate impacts the release of doxorubicin in a significant way, with q-NTD showing the slowest release rate while NTD showing the fastest release rate.  Our cellular uptake expts. reveal that q-NTD accumulated most effectively within cancer cells while NTD shows the lowest intracellular accumulation concn.  Interestingly, our cell viability assessment using a SRB assay reveals that d-NTD is the most potent conjugate against HepG2 human liver cancer cells.  These results suggest that intracellular accumulation efficiency and the free drug release rate are two important factors that det. the in vitro efficacy of drug conjugates.  To further validate this conclusion, we conjugated a short hydrocarbon onto the NTD to improve its cellular uptake, and found that the resulting conjugate, C16NTD, exhibited comparable intracellular accumulation as the q-NTD conjugate but superior anticancer activity due to its more effective release of free doxorubicin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofw3k2yd8YPrVg90H21EOLACvtfcHk0ljYlaL4CQbsww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVWqsL%252FM&md5=482b2cbfdac59b40d885d8cf76e3426c</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.jconrel.2014.05.051&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jconrel.2014.05.051%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DP.%26aulast%3DCheetham%26aufirst%3DA.%2BG.%26aulast%3DMoon%26aufirst%3DJ.%2BH.%26aulast%3DMoxley%26aufirst%3DJ.%2BW.%26aulast%3DLin%26aufirst%3DY.-a.%26aulast%3DCui%26aufirst%3DH.%26atitle%3DControlled%2520release%2520of%2520free%2520doxorubicin%2520from%2520peptide-drug%2520conjugates%2520by%2520drug%2520loading%26jtitle%3DJ.%2520Control.%2520Release%26date%3D2014%26volume%3D191%26spage%3D123%26epage%3D130%26doi%3D10.1016%2Fj.jconrel.2014.05.051" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheetham, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Y.-a.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lock, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, H.</span></span> <span> </span><span class="NLM_article-title">Supramolecular nanostructures formed by anticancer drug assembly</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>135</i></span>,  <span class="NLM_fpage">2907</span>– <span class="NLM_lpage">2910</span>, <span class="refDoi"> DOI: 10.1021/ja3115983</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja3115983" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVymsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2013&pages=2907-2910&author=A.+G.+Cheethamauthor=P.+Zhangauthor=Y.-a.+Linauthor=L.+L.+Lockauthor=H.+Cui&title=Supramolecular+nanostructures+formed+by+anticancer+drug+assembly&doi=10.1021%2Fja3115983"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Supramolecular Nanostructures Formed by Anticancer Drug Assembly</span></div><div class="casAuthors">Cheetham, Andrew G.; Zhang, Pengcheng; Lin, Yi-an; Lock, Lye Lin; Cui, Honggang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">135</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2907-2910</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We report here a supramol. strategy to directly assemble the small mol. hydrophobic anticancer drug camptothecin (CPT) into discrete, stable, well-defined nanostructures with a high and quant. drug loading.  Depending on the no. of CPTs in the mol. design, the resulting nanostructures can be either nanofibers or nanotubes, and have a fixed CPT loading content ranging from 23% to 38%.  We found that formation of nanostructures provides protection for both the CPT drug and the biodegradable linker from the external environment and thus offers a mechanism for controlled release of CPT.  Under tumor-relevant conditions, these drug nanostructures can release the bioactive form of CPT and show in vitro efficacy against a no. of cancer cell lines.  This strategy can be extended to construct nanostructures of other types of anticancer drugs and thus presents new opportunities for the development of self-delivering drugs for cancer therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrjrlQs4Vc6kbVg90H21EOLACvtfcHk0ljYlaL4CQbsww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvVymsbc%253D&md5=59ef9bc13c8a82ed9727a9d989baa959</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fja3115983&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja3115983%26sid%3Dliteratum%253Aachs%26aulast%3DCheetham%26aufirst%3DA.%2BG.%26aulast%3DZhang%26aufirst%3DP.%26aulast%3DLin%26aufirst%3DY.-a.%26aulast%3DLock%26aufirst%3DL.%2BL.%26aulast%3DCui%26aufirst%3DH.%26atitle%3DSupramolecular%2520nanostructures%2520formed%2520by%2520anticancer%2520drug%2520assembly%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2013%26volume%3D135%26spage%3D2907%26epage%3D2910%26doi%3D10.1021%2Fja3115983" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheetham, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chakroun, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, H.</span></span> <span> </span><span class="NLM_article-title">Self-assembling prodrugs</span>. <i>Chem. Soc. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">6638</span>– <span class="NLM_lpage">6663</span>, <span class="refDoi"> DOI: 10.1039/c7cs00521k</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1039%2FC7CS00521K" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=29019492" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1Chs7bK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2017&pages=6638-6663&author=A.+G.+Cheethamauthor=R.+W.+Chakrounauthor=W.+Maauthor=H.+Cui&title=Self-assembling+prodrugs&doi=10.1039%2Fc7cs00521k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Self-assembling prodrugs</span></div><div class="casAuthors">Cheetham, Andrew G.; Chakroun, Rami W.; Ma, Wang; Cui, Honggang</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Society Reviews</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">6638-6663</span>CODEN:
                <span class="NLM_cas:coden">CSRVBR</span>;
        ISSN:<span class="NLM_cas:issn">0306-0012</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Covalent modification of therapeutic compds. is a clin. proven strategy to devise prodrugs with enhanced treatment efficacies.  This prodrug strategy relies on the modified drugs that possess advantageous pharmacokinetic properties and administration routes over their parent drug.  Self-assembling prodrugs represent an emerging class of therapeutic agents capable of spontaneously assocg. into well-defined supramol. nanostructures in aq. solns.  The self-assembly of prodrugs expands the functional space of conventional prodrug design, affording a possible pathway to more effective therapies as the assembled nanostructure possesses distinct physicochem. properties and interaction potentials that can be tailored to specific administration routes and disease treatment.  In this review, we will discuss the various types of self-assembling prodrugs in development, providing an overview of the methods used to control their structure and function and, ultimately, our perspective on their current and future potential.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVWv5tijAnLbVg90H21EOLACvtfcHk0ljnhkV_YyC-mA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1Chs7bK&md5=c57c113dd4215e71d808d02d365311d9</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1039%2FC7CS00521K&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7CS00521K%26sid%3Dliteratum%253Aachs%26aulast%3DCheetham%26aufirst%3DA.%2BG.%26aulast%3DChakroun%26aufirst%3DR.%2BW.%26aulast%3DMa%26aufirst%3DW.%26aulast%3DCui%26aufirst%3DH.%26atitle%3DSelf-assembling%2520prodrugs%26jtitle%3DChem.%2520Soc.%2520Rev.%26date%3D2017%26volume%3D46%26spage%3D6638%26epage%3D6663%26doi%3D10.1039%2Fc7cs00521k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">FengZhao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span> <span> </span><span class="NLM_article-title">Disulfide bond reduction-triggered molecular hydrogels of folic acid-Taxol conjugates</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">6946</span>– <span class="NLM_lpage">6951</span>, <span class="refDoi"> DOI: 10.1039/c3ob40969d</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1039%2Fc3ob40969d" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=23989242" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFWksbjP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2013&pages=6946-6951&author=C.+Yangauthor=D.+Liauthor=Q.+FengZhaoauthor=L.+Wangauthor=L.+Wangauthor=Z.+Yang&title=Disulfide+bond+reduction-triggered+molecular+hydrogels+of+folic+acid-Taxol+conjugates&doi=10.1039%2Fc3ob40969d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Disulfide bond reduction-triggered molecular hydrogels of folic acid-Taxol conjugates</span></div><div class="casAuthors">Yang, Chengbiao; Li, Dongxia; Qianqi, Feng Zhao; Wang, Lianyong; Wang, Ling; Yang, Zhimou</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">40</span>),
    <span class="NLM_cas:pages">6946-6951</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Mol. hydrogels of therapeutic agents are a novel kind of self-delivery system that can sustain release of drugs or pro-drugs.  We have previously developed a mol. hydrogelator of folic acid (FA)-Taxol conjugate triggered by phosphatase.  In this paper, we report a novel mol. hydrogelator system of FA-Taxol conjugates with improved synthetic strategy.  The hydrogels are formed by the redn. of disulfide bond by glutathione (GSH).  These hydrogels could sustain release of Taxol through ester bond hydrolysis.  Compared with i.v. (i.v.) injection of clin. used Taxol with four times the dosage, our hydrogel could inhibit tumor growth more efficiently by a single dose of intra-tumor (i.t.) administration.  These observations suggested the big potential of this novel gelation system of Taxol for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5mkDu5LBPZrVg90H21EOLACvtfcHk0ljnhkV_YyC-mA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFWksbjP&md5=f9a178df76dfc49e1ee4ad984cb67e9d</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1039%2Fc3ob40969d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc3ob40969d%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DD.%26aulast%3DFengZhao%26aufirst%3DQ.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DZ.%26atitle%3DDisulfide%2520bond%2520reduction-triggered%2520molecular%2520hydrogels%2520of%2520folic%2520acid-Taxol%2520conjugates%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2013%26volume%3D11%26spage%3D6946%26epage%3D6951%26doi%3D10.1039%2Fc3ob40969d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lovett, S. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, B.</span></span> <span> </span><span class="NLM_article-title">Diglycine enables rapid intrabacterial hydrolysis for activating anbiotics against gram-negative bacteria</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">10631</span>– <span class="NLM_lpage">10634</span>, <span class="refDoi"> DOI: 10.1002/anie.201905230</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1002%2Fanie.201905230" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BC1MXht1Oqt7zI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2019&pages=10631-10634&author=J.+Wangauthor=D.+L.+Cooperauthor=W.+Zhanauthor=D.+Wuauthor=H.+Heauthor=S.+Sunauthor=S.+T.+Lovettauthor=B.+Xu&title=Diglycine+enables+rapid+intrabacterial+hydrolysis+for+activating+anbiotics+against+gram-negative+bacteria&doi=10.1002%2Fanie.201905230"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Diglycine Enables Rapid Intrabacterial Hydrolysis for Activating Anbiotics against Gram-negative Bacteria</span></div><div class="casAuthors">Wang, Jiaqing; Cooper, Deani L.; Zhan, Wenjun; Wu, Difei; He, Hongjian; Sun, Shenghuan; Lovett, Susan T.; Xu, Bing</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">10631-10634</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Antimicrobial drug resistance demands novel approaches for improving the efficacy of antibiotics, esp. against Gram-neg. bacteria.  We report that conjugating a diglycine (GG) to an antibiotic prodrug drastically accelerates intrabacterial ester-bond hydrolysis required for activating the antibiotic.  Specifically, the attachment of GG to chloramphenicol succinate (CLsu) generates CLsuGG, which exhibits about an order of magnitude higher inhibitory efficacy than CLsu against Escherichia coli.  Further studies reveal that CLsuGG undergoes rapid hydrolysis, catalyzed by intrabacterial esterases (e.g., BioH and YjfP), to generate chloramphenicol (CL) in E. coli.  Importantly, the conjugate exhibits lower cytotoxicity to bone marrow stromal cells than CL.  Structural analogs of CLsuGG indicate that the conjugation of GG to an antibiotic prodrug is an effective strategy for accelerating enzymic prodrug hydrolysis and enhancing the antibacterial efficacy of antibiotics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7QnuC8K8iL7Vg90H21EOLACvtfcHk0ljnhkV_YyC-mA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXht1Oqt7zI&md5=1b64f171d468ac55d219c8f5c822d22a</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1002%2Fanie.201905230&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201905230%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DCooper%26aufirst%3DD.%2BL.%26aulast%3DZhan%26aufirst%3DW.%26aulast%3DWu%26aufirst%3DD.%26aulast%3DHe%26aufirst%3DH.%26aulast%3DSun%26aufirst%3DS.%26aulast%3DLovett%26aufirst%3DS.%2BT.%26aulast%3DXu%26aufirst%3DB.%26atitle%3DDiglycine%2520enables%2520rapid%2520intrabacterial%2520hydrolysis%2520for%2520activating%2520anbiotics%2520against%2520gram-negative%2520bacteria%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2019%26volume%3D58%26spage%3D10631%26epage%3D10634%26doi%3D10.1002%2Fanie.201905230" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dickson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koohmaraie, M.</span></span> <span> </span><span class="NLM_article-title">Cell surface charge characteristics and their relationship to bacterial attachment to meat surfaces</span>. <i>Appl. Environ. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">832</span>– <span class="NLM_lpage">836</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1128%2FAEM.55.4.832-836.1989" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=2499255" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A280%3ADyaL1M3ntFyrtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=1989&pages=832-836&author=J.+S.+Dicksonauthor=M.+Koohmaraie&title=Cell+surface+charge+characteristics+and+their+relationship+to+bacterial+attachment+to+meat+surfaces"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Cell surface charge characteristics and their relationship to bacterial attachment to meat surfaces</span></div><div class="casAuthors">Dickson J S; Koohmaraie M</div><div class="citationInfo"><span class="NLM_cas:title">Applied and environmental microbiology</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">832-6</span>
        ISSN:<span class="NLM_cas:issn">0099-2240</span>.
    </div><div class="casAbstract">Cell surface charge and hydrophobicity of Bacillus subtilis, Escherichia coli O157:H7, Listeria monocytogenes, Salmonella typhimurium, Serratia marcescens, Staphylococcus aureus, and Staphylococcus epidermidis were determined by hydrocarbon adherence, hydrophobic interaction, and electrostatic interaction chromatography.  Surface charge and hydrophobicity were compared with the initial attachment values and rates of attachment of the bacteria to meat surfaces.  There was a linear correlation between the relative negative charge on the bacterial cell surface and initial attachment to lean beef muscle (r2 = 0.885) and fat tissue (r2 = 0.777).  Hydrophobicity correlated well with attachment to fat tissue only.  The relative hydrophobicity of each bacterium was dependent on the specific method of determination, with wide variations noted between methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRkv7qQYFIC6svLc4ARHEmzfW6udTcc2eYz2XxcuSZByrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaL1M3ntFyrtA%253D%253D&md5=6690ecaf36a7400ea36b00a26bb3c0ab</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1128%2FAEM.55.4.832-836.1989&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAEM.55.4.832-836.1989%26sid%3Dliteratum%253Aachs%26aulast%3DDickson%26aufirst%3DJ.%2BS.%26aulast%3DKoohmaraie%26aufirst%3DM.%26atitle%3DCell%2520surface%2520charge%2520characteristics%2520and%2520their%2520relationship%2520to%2520bacterial%2520attachment%2520to%2520meat%2520surfaces%26jtitle%3DAppl.%2520Environ.%2520Microbiol.%26date%3D1989%26volume%3D55%26spage%3D832%26epage%3D836" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamagata, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, B.</span></span> <span> </span><span class="NLM_article-title">Taurine Boosts Cellular Uptake of Smalld-Peptides for Enzyme-Instructed Intracellular Molecular Self-Assembly</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>137</i></span>,  <span class="NLM_fpage">10040</span>– <span class="NLM_lpage">10043</span>, <span class="refDoi"> DOI: 10.1021/jacs.5b06181</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.5b06181" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Kms77K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=137&publication_year=2015&pages=10040-10043&author=J.+Zhouauthor=X.+Duauthor=J.+Liauthor=N.+Yamagataauthor=B.+Xu&title=Taurine+Boosts+Cellular+Uptake+of+Smalld-Peptides+for+Enzyme-Instructed+Intracellular+Molecular+Self-Assembly&doi=10.1021%2Fjacs.5b06181"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Taurine Boosts Cellular Uptake of Small D-Peptides for Enzyme-Instructed Intracellular Molecular Self-Assembly</span></div><div class="casAuthors">Zhou, Jie; Du, Xuewen; Li, Jie; Yamagata, Natsuko; Xu, Bing</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">137</span>
        (<span class="NLM_cas:issue">32</span>),
    <span class="NLM_cas:pages">10040-10043</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Due to their biostability, D-peptides are emerging as an important mol. platform for biomedical applications.  Being proteolytically resistant, D-peptides lack interactions with endogenous transporters and hardly enter cells.  Here we show that taurine, a natural amino acid, drastically boosts the cellular uptake of small D-peptides in mammalian cells by >10-fold, from 118 μM (without conjugating taurine) to >1.6 mM (after conjugating taurine).  The uptake of a large amt. of the ester conjugate of taurine and D-peptide allows intracellular esterase to trigger intracellular self-assembly of the D-peptide deriv., further enhancing their cellular accumulation.  The study on the mechanism of the uptake reveals that the conjugates enter cells via both dynamin-dependent endocytosis and macropinocytosis, but likely not relying on taurine transporters.  Differing fundamentally from the pos. charged cell-penetrating peptides, the biocompatibility, stability, and simplicity of the enzyme-cleavable taurine motif promise new ways to promote the uptake of bioactive mols. for countering the action of efflux pump and contributing to intracellular mol. self-assembly.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjESPZzhIKYrVg90H21EOLACvtfcHk0ljqMsoYYV3ojA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Kms77K&md5=bda970b866249440a59db6b5448de40a</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fjacs.5b06181&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.5b06181%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DDu%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DYamagata%26aufirst%3DN.%26aulast%3DXu%26aufirst%3DB.%26atitle%3DTaurine%2520Boosts%2520Cellular%2520Uptake%2520of%2520Smalld-Peptides%2520for%2520Enzyme-Instructed%2520Intracellular%2520Molecular%2520Self-Assembly%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2015%26volume%3D137%26spage%3D10040%26epage%3D10043%26doi%3D10.1021%2Fjacs.5b06181" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Medina, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dinulescu, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, B.</span></span> <span> </span><span class="NLM_article-title">Enzyme-instructed intracellular molecular self-assembly to boost activity of cisplatin against drug-resistant ovarian cancer cells</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">13307</span>– <span class="NLM_lpage">13311</span>, <span class="refDoi"> DOI: 10.1002/anie.201507157</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1002%2Fanie.201507157" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsV2nsb7M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2015&pages=13307-13311&author=J.+Liauthor=Y.+Kuangauthor=J.+Shiauthor=J.+Zhouauthor=J.+E.+Medinaauthor=R.+Zhouauthor=D.+Yuanauthor=C.+Yangauthor=H.+Wangauthor=Z.+Yangauthor=J.+Liuauthor=D.+M.+Dinulescuauthor=B.+Xu&title=Enzyme-instructed+intracellular+molecular+self-assembly+to+boost+activity+of+cisplatin+against+drug-resistant+ovarian+cancer+cells&doi=10.1002%2Fanie.201507157"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Enzyme-Instructed Intracellular Molecular Self-Assembly to Boost Activity of Cisplatin against Drug-Resistant Ovarian Cancer Cells</span></div><div class="casAuthors">Li, Jie; Kuang, Yi; Shi, Junfeng; Zhou, Jie; Medina, Jamie E.; Zhou, Rong; Yuan, Dan; Yang, Cuihong; Wang, Huaimin; Yang, Zhimou; Liu, Jianfeng; Dinulescu, Daniela M.; Xu, Bing</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">45</span>),
    <span class="NLM_cas:pages">13307-13311</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Anticancer drug resistance demands innovative approaches that boost the activity of drugs against drug-resistant cancers without increasing the systemic toxicity.  Use of enzyme-instructed self-assembly (EISA) to generate intracellular supramol. assemblies that drastically boost the activity of cisplatin against drug-resistant ovarian cancer cells.  The authors design and synthesize small peptide precursors as the substrates of carboxylesterase (CES).  CES cleaves the ester bond pre-installed on the precursors to form the peptides that self-assemble in water to form nanofibers.  At the optimal concns., the precursors themselves are innocuous to cells, but they double or triple the activity of cisplatin against the drug-resistant ovarian cancer cells.  This work illustrates a simple, yet fundamental, new way to introduce non-cytotoxic components into combination therapies with cisplatin without increasing the systemic burden or side effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNfBq1jJrEr7Vg90H21EOLACvtfcHk0ljqMsoYYV3ojA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsV2nsb7M&md5=793f2c0ebf856fa2c21383b0b43975e8</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1002%2Fanie.201507157&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201507157%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DKuang%26aufirst%3DY.%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DMedina%26aufirst%3DJ.%2BE.%26aulast%3DZhou%26aufirst%3DR.%26aulast%3DYuan%26aufirst%3DD.%26aulast%3DYang%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DDinulescu%26aufirst%3DD.%2BM.%26aulast%3DXu%26aufirst%3DB.%26atitle%3DEnzyme-instructed%2520intracellular%2520molecular%2520self-assembly%2520to%2520boost%2520activity%2520of%2520cisplatin%2520against%2520drug-resistant%2520ovarian%2520cancer%2520cells%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2015%26volume%3D54%26spage%3D13307%26epage%3D13311%26doi%3D10.1002%2Fanie.201507157" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sulavik, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Houseweart, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cramer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiwani, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murgolo, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greene, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DiDomenico, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, G. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hare, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimer, G.</span></span> <span> </span><span class="NLM_article-title">Antibiotic susceptibility profiles of Escherichia coli strains lacking multidrug efflux pump genes</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">1126</span>– <span class="NLM_lpage">1136</span>, <span class="refDoi"> DOI: 10.1128/aac.45.4.1126-1136.2001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1128%2Faac.45.4.1126-1136.2001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=11257026" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BD3MXit1Omsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2001&pages=1126-1136&author=M.+C.+Sulavikauthor=C.+Houseweartauthor=C.+Cramerauthor=N.+Jiwaniauthor=N.+Murgoloauthor=J.+Greeneauthor=B.+DiDomenicoauthor=K.+J.+Shawauthor=G.+H.+Millerauthor=R.+Hareauthor=G.+Shimer&title=Antibiotic+susceptibility+profiles+of+Escherichia+coli+strains+lacking+multidrug+efflux+pump+genes&doi=10.1128%2Faac.45.4.1126-1136.2001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Antibiotic susceptibility profiles of Escherichia coli strains lacking multidrug efflux pump genes</span></div><div class="casAuthors">Sulavik, Mark C.; Houseweart, Chad; Cramer, Christina; Jiwani, Nilofer; Murgolo, Nicholas; Greene, Jonathan; DiDomenico, Beth; Shaw, Karen Joy; Miller, George H.; Hare, Roberta; Shimer, George</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1126-1136</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The contribution of seven known and nine predicted genes or operons assocd. with multidrug resistance to the susceptibility of Escherichia coli W3110 was assessed for 20 different classes of antimicrobial compds. that include antibiotics, antiseptics, detergents, and dyes.  Strains were constructed with deletions for genes in the major facilitator superfamily, the resistance nodulation-cell division family, the small multidrug resistance family, the ATP-binding cassette family, and outer membrane factors.  The agar diln. MICs of 35 compds. were detd. for strains with deletions for multidrug resistance (MDR) pumps.  Deletions in acrAB or tolC resulted in increased susceptibilities to the majority of compds. tested.  The remaining MDR pump gene deletions resulted in increased susceptibilities to far fewer compds.  The results identify which MDR pumps contribute to intrinsic resistance under the conditions tested and supply practical information useful for designing sensitive assay strains for cell-based screening of antibacterial compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2tjRRrBEqUrVg90H21EOLACvtfcHk0ljqMsoYYV3ojA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXit1Omsrg%253D&md5=86d025f77f3e79d83d868f9c727785e0</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1128%2Faac.45.4.1126-1136.2001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252Faac.45.4.1126-1136.2001%26sid%3Dliteratum%253Aachs%26aulast%3DSulavik%26aufirst%3DM.%2BC.%26aulast%3DHouseweart%26aufirst%3DC.%26aulast%3DCramer%26aufirst%3DC.%26aulast%3DJiwani%26aufirst%3DN.%26aulast%3DMurgolo%26aufirst%3DN.%26aulast%3DGreene%26aufirst%3DJ.%26aulast%3DDiDomenico%26aufirst%3DB.%26aulast%3DShaw%26aufirst%3DK.%2BJ.%26aulast%3DMiller%26aufirst%3DG.%2BH.%26aulast%3DHare%26aufirst%3DR.%26aulast%3DShimer%26aufirst%3DG.%26atitle%3DAntibiotic%2520susceptibility%2520profiles%2520of%2520Escherichia%2520coli%2520strains%2520lacking%2520multidrug%2520efflux%2520pump%2520genes%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2001%26volume%3D45%26spage%3D1126%26epage%3D1136%26doi%3D10.1128%2Faac.45.4.1126-1136.2001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Greenwood, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Grady, F.</span></span> <span> </span><span class="NLM_article-title">Activity and interaction of trimethoprim and sulphamethoxazole against Escherichia coli</span>. <i>J. Clin. Pathol.</i> <span class="NLM_year" style="font-weight: bold;">1976</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">162</span>– <span class="NLM_lpage">166</span>, <span class="refDoi"> DOI: 10.1136/jcp.29.2.162</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1136%2Fjcp.29.2.162" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=777036" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADyaE28Xhs1Sjtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=1976&pages=162-166&author=D.+Greenwoodauthor=F.+O%E2%80%99Grady&title=Activity+and+interaction+of+trimethoprim+and+sulphamethoxazole+against+Escherichia+coli&doi=10.1136%2Fjcp.29.2.162"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Activity and interaction of trimethoprim and sulfamethoxazole against Escherichia coli</span></div><div class="casAuthors">Greenwood, D.; O'Grady, F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Pathology</span>
        (<span class="NLM_cas:date">1976</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">162-6</span>CODEN:
                <span class="NLM_cas:coden">JCPAAK</span>;
        ISSN:<span class="NLM_cas:issn">0021-9746</span>.
    </div><div class="casAbstract">Studies of the activity of trimethoprim lactate (I lactate) [23256-42-0] and sulfamethoxazole (II) [723-46-6] alone and in combination [8064-90-2] against a sensitive strain of Escherichia coli showed that in a static system in which the conditions of exposure of bacteria to drug resembled those of conventional min. inhibitory concn. (MIC) titrns., both I and II exhibited antibacterial activity at concns. well below the conventionally detd. MIC, but regrowth occurred at these concns. during the overnight incubation period due to the emergence of adaptively resistant bacteria.  Tests of combined drug action in the static turbidimetric system revealed even more synergic interaction than was apparent in conventional MIC tests, suggesting that an important component of overall synergic interaction is the mutual suppression of adaptive resistance to the other agent.  Studies in  an in vitro model which simulates the hydrokinetic features of the urinary bladder showed that concns. of I and II below the conventionally detd. MIC inhibited the growth even of extremely dense bacterial populations so long as the concn. of drug was maintained.  The response of cultures exposed to combinations of I and II in this system was so dominated by the effect of I that no synergic interaction with II was noted at concns. of the drugs which are achievable in urine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUr86Nh9pFcrVg90H21EOLACvtfcHk0ljMycDs7NXnYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE28Xhs1Sjtbo%253D&md5=eb3124d10717d33e9b3e06df4854f4f0</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1136%2Fjcp.29.2.162&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1136%252Fjcp.29.2.162%26sid%3Dliteratum%253Aachs%26aulast%3DGreenwood%26aufirst%3DD.%26aulast%3DO%25E2%2580%2599Grady%26aufirst%3DF.%26atitle%3DActivity%2520and%2520interaction%2520of%2520trimethoprim%2520and%2520sulphamethoxazole%2520against%2520Escherichia%2520coli%26jtitle%3DJ.%2520Clin.%2520Pathol.%26date%3D1976%26volume%3D29%26spage%3D162%26epage%3D166%26doi%3D10.1136%2Fjcp.29.2.162" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Glassford, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagh, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velvadapu, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paul, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sandelin, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeBrosse, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klepacki, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Small, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacKerell, A. D.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrade, R. B.</span></span> <span> </span><span class="NLM_article-title">Desmethyl macrolides: synthesis and evaluation of 4-desmethyl telithromycin</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">1021</span>– <span class="NLM_lpage">1026</span>, <span class="refDoi"> DOI: 10.1021/ml5002097</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml5002097" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFKmtLzM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=1021-1026&author=I.+Glassfordauthor=M.+Leeauthor=B.+Waghauthor=V.+Velvadapuauthor=T.+Paulauthor=G.+Sandelinauthor=C.+DeBrosseauthor=D.+Klepackiauthor=M.+C.+Smallauthor=A.+D.+MacKerellauthor=R.+B.+Andrade&title=Desmethyl+macrolides%3A+synthesis+and+evaluation+of+4-desmethyl+telithromycin&doi=10.1021%2Fml5002097"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Desmethyl Macrolides: Synthesis and Evaluation of 4-Desmethyl Telithromycin</span></div><div class="casAuthors">Glassford, Ian; Lee, Miseon; Wagh, Bharat; Velvadapu, Venkata; Paul, Tapas; Sandelin, Gary; DeBrosse, Charles; Klepacki, Dorota; Small, Meagan C.; MacKerell, Alexander D.; Andrade, Rodrigo B.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1021-1026</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Novel sources of antibiotics are needed to address the serious threat of bacterial resistance.  Accordingly, we have launched a structure-based drug design program featuring a desmethylation strategy wherein Me groups have been replaced with hydrogens.  Herein we report the total synthesis, mol. modeling, and biol. evaluation of 4-desmethyl telithromycin (I), a novel desmethyl analog of the third-generation ketolide antibiotic telithromycin and our final analog in this series.  While I was found to be equipotent with telithromycin against wild-type bacteria, it was 4-fold less potent against the A2058G mutant.  These findings reveal that strategically replacing the C4-Me group with hydrogen (i.e., desmethylation) did not address this mechanism of resistance.  Throughout the desmethyl series, the sequential addn. of Me to the 14-membered macrolactone resulted in improved bioactivity.  Mol. modeling methods indicate that changes in conformational flexibility dominate the increased biol. activity; moreover, they reveal I adopts a different conformation once bound to the A2058G ribosome, thus impacting noncovalent interactions reflected in a lower MIC value.  Finally, fluorescence polarization expts. of I with E. coli ribosomes confirmed I is indeed binding the ribosome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5fEOWybrdp7Vg90H21EOLACvtfcHk0ljMycDs7NXnYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFKmtLzM&md5=e2dcb2b85f8ee956cbce59076b08300a</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fml5002097&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml5002097%26sid%3Dliteratum%253Aachs%26aulast%3DGlassford%26aufirst%3DI.%26aulast%3DLee%26aufirst%3DM.%26aulast%3DWagh%26aufirst%3DB.%26aulast%3DVelvadapu%26aufirst%3DV.%26aulast%3DPaul%26aufirst%3DT.%26aulast%3DSandelin%26aufirst%3DG.%26aulast%3DDeBrosse%26aufirst%3DC.%26aulast%3DKlepacki%26aufirst%3DD.%26aulast%3DSmall%26aufirst%3DM.%2BC.%26aulast%3DMacKerell%26aufirst%3DA.%2BD.%26aulast%3DAndrade%26aufirst%3DR.%2BB.%26atitle%3DDesmethyl%2520macrolides%253A%2520synthesis%2520and%2520evaluation%2520of%25204-desmethyl%2520telithromycin%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D5%26spage%3D1021%26epage%3D1026%26doi%3D10.1021%2Fml5002097" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yogev, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolling, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, T.</span></span> <span> </span><span class="NLM_article-title">Pharmacokinetic comparison of intravenous and oral chloramphenicol in patients with Haemophilus influenzae meningitis</span>. <i>Pediatrics</i> <span class="NLM_year" style="font-weight: bold;">1981</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">656</span>– <span class="NLM_lpage">660</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=6973130" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A280%3ADyaL3M3ks1Gqtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=1981&pages=656-660&author=R.+Yogevauthor=W.+M.+Kollingauthor=T.+Williams&title=Pharmacokinetic+comparison+of+intravenous+and+oral+chloramphenicol+in+patients+with+Haemophilus+influenzae+meningitis"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacokinetic comparison of intravenous and oral chloramphenicol in patients with Haemophilus influenzae meningitis</span></div><div class="casAuthors">Yogev R; Kolling W M; Williams T</div><div class="citationInfo"><span class="NLM_cas:title">Pediatrics</span>
        (<span class="NLM_cas:date">1981</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">656-60</span>
        ISSN:<span class="NLM_cas:issn">0031-4005</span>.
    </div><div class="casAbstract">The pharmacokinetics of chloramphenicol following intravenous and oral administration were studied in 14 infants with Haemophilus influenzae meningitis.  Following five days of treatment with intravenous chloramphenicol (100 mg/kg/day every six hours), oral chloramphenicol was substituted at the same dose.  Multiple serum levels of chloramphenicol were determined after an intravenous dose on day 4 and after an oral dose on day 10.  CSF levels were measured six hours after intravenous or oral chloramphenicol dose on those days (CSF trough).  Following intravenous administration, the mean peak serum level of 15.0 micrograms/ml was reached at 45 minutes.  In comparison, after oral chloramphenicol in the same dosage, the mean peak serum level of 18.5 micrograms/ml was achieved at two to three hours.  The mean serum half-life of the drug (6.5 hours) was significantly longer after oral administration than after intravenous chloramphenicol (4.0 hours) (P less than .001).  The increased serum half-life following orally administered chloramphenicol was occasionally associated with drug accumulation.  In addition, mean trough CSF levels were somewhat higher when the patient received oral medication (6.6 micrograms/ml) compared to intravenous administration (4.2 micrograms/ml) (P less than .001).  For any treatment regimen for H influenzae meningitis that includes a period of oral chloramphenicol therapy the patient should be hospitalized to ensure compliance.  Because of the wide range of individual variation in serum half-life that may result in accumulation, periodic monitoring of serum chloramphenicol levels is also recommended.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTfTBzE2TvtT02yPDyEFiigfW6udTcc2ea_79Qp0AA8pLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaL3M3ks1Gqtw%253D%253D&md5=c833524469c559a0fc11de532f0451f8</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DYogev%26aufirst%3DR.%26aulast%3DKolling%26aufirst%3DW.%2BM.%26aulast%3DWilliams%26aufirst%3DT.%26atitle%3DPharmacokinetic%2520comparison%2520of%2520intravenous%2520and%2520oral%2520chloramphenicol%2520in%2520patients%2520with%2520Haemophilus%2520influenzae%2520meningitis%26jtitle%3DPediatrics%26date%3D1981%26volume%3D67%26spage%3D656%26epage%3D660" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Medina, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dinulescu, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, B.</span></span> <span> </span><span class="NLM_article-title">Selectively inducing cancer cell death by intracellular enzyme-instructed self-assembly (EISA) of dipeptide derivatives</span>. <i>Adv. Healthcare Mater.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">1601400</span>, <span class="refDoi"> DOI: 10.1002/adhm.201601400</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1002%2Fadhm.201601400" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2017&pages=1601400&author=J.+Liauthor=J.+Shiauthor=J.+E.+Medinaauthor=J.+Zhouauthor=X.+Duauthor=H.+Wangauthor=C.+Yangauthor=J.+Liuauthor=Z.+Yangauthor=D.+M.+Dinulescuauthor=B.+Xu&title=Selectively+inducing+cancer+cell+death+by+intracellular+enzyme-instructed+self-assembly+%28EISA%29+of+dipeptide+derivatives&doi=10.1002%2Fadhm.201601400"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1002%2Fadhm.201601400&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fadhm.201601400%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DMedina%26aufirst%3DJ.%2BE.%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DDu%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DYang%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DDinulescu%26aufirst%3DD.%2BM.%26aulast%3DXu%26aufirst%3DB.%26atitle%3DSelectively%2520inducing%2520cancer%2520cell%2520death%2520by%2520intracellular%2520enzyme-instructed%2520self-assembly%2520%2528EISA%2529%2520of%2520dipeptide%2520derivatives%26jtitle%3DAdv.%2520Healthcare%2520Mater.%26date%3D2017%26volume%3D6%26spage%3D1601400%26doi%3D10.1002%2Fadhm.201601400" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Prabhala, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aduri, N. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaheen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iqbal, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brasen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gajhede, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rahman, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirza, O.</span></span> <span> </span><span class="NLM_article-title">The prototypical proton-coupled oligopeptide transporter YdgR fromEscherichia colifacilitates chloramphenicol uptake into bacterial cells</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>293</i></span>,  <span class="NLM_fpage">1007</span>– <span class="NLM_lpage">1017</span>, <span class="refDoi"> DOI: 10.1074/jbc.m117.805960</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1074%2Fjbc.m117.805960" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=29150447" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1yjsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=293&publication_year=2018&pages=1007-1017&author=B.+K.+Prabhalaauthor=N.+G.+Aduriauthor=N.+Sharmaauthor=A.+Shaheenauthor=A.+Sharmaauthor=M.+Iqbalauthor=P.+R.+Hansenauthor=C.+Brasenauthor=M.+Gajhedeauthor=M.+Rahmanauthor=O.+Mirza&title=The+prototypical+proton-coupled+oligopeptide+transporter+YdgR+fromEscherichia+colifacilitates+chloramphenicol+uptake+into+bacterial+cells&doi=10.1074%2Fjbc.m117.805960"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">The prototypical proton-coupled oligopeptide transporter YdgR from Escherichia coli facilitates chloramphenicol uptake into bacterial cells</span></div><div class="casAuthors">Prabhala, Bala K.; Aduri, Nanda G.; Sharma, Neha; Shaheen, Aqsa; Sharma, Arpan; Iqbal, Mazhar; Hansen, Paul R.; Brasen, Christoffer; Gajhede, Michael; Rahman, Moazur; Mirza, Osman</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">293</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1007-1017</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Chloramphenicol (Cam) is a broad-spectrum antibiotic used to combat bacterial infections in humans and animals.  Cam export from bacterial cells is one of the mechanisms by which pathogens resist Cam's antibacterial effects, and several different proteins are known to facilitate this process.  However, to date no report exists on any specific transport protein that facilitates Cam uptake.  The proton-coupled oligopeptide transporter (POT) YdgR from Escherichia coli is a prototypical member of the POT family, functioning in proton-coupled uptake of di- and tripeptides.  By following bacterial growth and conducting LC-MS-based assays we show here that YdgR facilitates Cam uptake.  Some YdgR variants displaying reduced peptide uptake also exhibited reduced Cam uptake, indicating that peptides and Cam bind YdgR at similar regions.  Homol. modeling of YdgR, Cam docking, and mutational studies suggested a binding mode that resembles that of Cam binding to the multidrug resistance transporter MdfA.  To our knowledge, this is the first report of Cam uptake into bacterial cells mediated by a specific transporter protein.  Our findings suggest a specific bacterial transporter for drug uptake that might be targeted to promote greater antibiotic influx to increase cytoplasmic antibiotic concn. for enhanced cytotoxicity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqc9zM2x0kV67Vg90H21EOLACvtfcHk0lhkYZzNz6rlnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1yjsro%253D&md5=4c6c9a2275886ca46238b581580da8fa</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1074%2Fjbc.m117.805960&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.m117.805960%26sid%3Dliteratum%253Aachs%26aulast%3DPrabhala%26aufirst%3DB.%2BK.%26aulast%3DAduri%26aufirst%3DN.%2BG.%26aulast%3DSharma%26aufirst%3DN.%26aulast%3DShaheen%26aufirst%3DA.%26aulast%3DSharma%26aufirst%3DA.%26aulast%3DIqbal%26aufirst%3DM.%26aulast%3DHansen%26aufirst%3DP.%2BR.%26aulast%3DBrasen%26aufirst%3DC.%26aulast%3DGajhede%26aufirst%3DM.%26aulast%3DRahman%26aufirst%3DM.%26aulast%3DMirza%26aufirst%3DO.%26atitle%3DThe%2520prototypical%2520proton-coupled%2520oligopeptide%2520transporter%2520YdgR%2520fromEscherichia%2520colifacilitates%2520chloramphenicol%2520uptake%2520into%2520bacterial%2520cells%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2018%26volume%3D293%26spage%3D1007%26epage%3D1017%26doi%3D10.1074%2Fjbc.m117.805960" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Buchanan, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venkatramani, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Esser, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palnitkar, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chakraborty, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Helm, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deisenhofer, J.</span></span> <span> </span><span class="NLM_article-title">Crystal structure of the outer membrane active transporter FepA from Escherichia coli</span>. <i>Nat. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">56</span>– <span class="NLM_lpage">63</span>, <span class="refDoi"> DOI: 10.1038/4931</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1038%2F4931" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=9886293" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADyaK1MXjvVOjuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1999&pages=56-63&author=S.+K.+Buchananauthor=B.+S.+Smithauthor=L.+Venkatramaniauthor=D.+Xiaauthor=L.+Esserauthor=M.+Palnitkarauthor=R.+Chakrabortyauthor=D.+van+der+Helmauthor=J.+Deisenhofer&title=Crystal+structure+of+the+outer+membrane+active+transporter+FepA+from+Escherichia+coli&doi=10.1038%2F4931"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of the outer membrane active transporter FepA from Escherichia coli</span></div><div class="casAuthors">Buchanan, Susan K.; Smith, Barbara S.; Venkatramani, Lalitha; Xia, Di; Esser, Lothar; Palnitkar, Maya; Chakraborty, Ranjan; Van Der Helm, Dick; Deisenhofer, Johann</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural Biology</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">56-63</span>CODEN:
                <span class="NLM_cas:coden">NSBIEW</span>;
        ISSN:<span class="NLM_cas:issn">1072-8368</span>.
    
            (<span class="NLM_cas:orgname">Nature America</span>)
        </div><div class="casAbstract">Integral outer membrane receptors for iron chelates and vitamin B12 carry out specific ligand transport against a concn. gradient.  Energy for active transport is obtained from the proton-motive force of the inner membrane through phys. interaction with TonB-ExbB-ExbD, an inner membrane complex.  Here we report the crystal structure of an active transport, outer membrane receptor at 2.4 A resoln.  Two distinct functional domains are revealed: (i) a 22-stranded β-barrel that spans the outer membrane and contains large extracellular loops which appear to function in ligand binding; and (ii) a globular N-terminal domain that folds into the barrel pore, inhibiting access to the periplasm and contributing two addnl. loops for potential ligand binding.  These loops could provide a signaling pathway between the processes of ligand recognition and TonB-mediated transport.  The blockage of the pore suggests that the N-terminal domain must undergo a conformational rearrangement to allow ligand transport into the periplasm.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoU1RPTePfa67Vg90H21EOLACvtfcHk0lhkYZzNz6rlnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXjvVOjuw%253D%253D&md5=4cb575c15e4c828137778c0d2186e0e3</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1038%2F4931&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F4931%26sid%3Dliteratum%253Aachs%26aulast%3DBuchanan%26aufirst%3DS.%2BK.%26aulast%3DSmith%26aufirst%3DB.%2BS.%26aulast%3DVenkatramani%26aufirst%3DL.%26aulast%3DXia%26aufirst%3DD.%26aulast%3DEsser%26aufirst%3DL.%26aulast%3DPalnitkar%26aufirst%3DM.%26aulast%3DChakraborty%26aufirst%3DR.%26aulast%3Dvan%2Bder%2BHelm%26aufirst%3DD.%26aulast%3DDeisenhofer%26aufirst%3DJ.%26atitle%3DCrystal%2520structure%2520of%2520the%2520outer%2520membrane%2520active%2520transporter%2520FepA%2520from%2520Escherichia%2520coli%26jtitle%3DNat.%2520Struct.%2520Biol.%26date%3D1999%26volume%3D6%26spage%3D56%26epage%3D63%26doi%3D10.1038%2F4931" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Masi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pagès, J.-M.</span></span> <span> </span><span class="NLM_article-title">Structure, function and regulation of outer membrane proteins involved in drug transport in enterobactericeae: the OmpF/C - TolC case</span>. <i>Open Microbiol. J.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">22</span>– <span class="NLM_lpage">33</span>, <span class="refDoi"> DOI: 10.2174/1874285801307010022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.2174%2F1874285801307010022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=23569467" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFGqsLrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2013&pages=22-33&author=M.+Masiauthor=J.-M.+Pag%C3%A8s&title=Structure%2C+function+and+regulation+of+outer+membrane+proteins+involved+in+drug+transport+in+enterobactericeae%3A+the+OmpF%2FC+-+TolC+case&doi=10.2174%2F1874285801307010022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Structure, function and regulation of outer membrane proteins involved in drug transport in Enterobactericeae: the OmpF/C - TolC case</span></div><div class="casAuthors">Masi, Muriel; Pages, Jean-Marie</div><div class="citationInfo"><span class="NLM_cas:title">Open Microbiology Journal</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">22-33</span>CODEN:
                <span class="NLM_cas:coden">OMJPAO</span>;
        ISSN:<span class="NLM_cas:issn">1874-2858</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Antibiotic translocation across membranes of Gram-neg. bacteria is a key step for the activity on their specific intracellular targets.  Resistant bacteria control their membrane permeability as a first line of defense to protect themselves against external toxic compds. such as antibiotics and biocides.  On one hand, resistance to small hydrophilic antibiotics such as β-lactams and fluoroquinolones frequently results from the "closing" of their way in: the general outer membrane porins.  On the other hand, an effective way out for a wide range of antibiotics is provided by TolC-like proteins, which are outer membrane components of multidrug efflux pumps.  Accordingly, altered membrane permeability, including porin modifications and/or efflux pumps' overexpression, is always assocd. to multidrug resistance (MDR) in a no. of clin. isolates.  Several recent studies have highlighted our current understanding of porins/TolC structures and functions in Enterobacteriaceae.  Here, we review the transport of antibiotics through the OmpF/C general porins and the TolC-like channels with regards to recent data on their structure, function, assembly, regulation and contribution to bacterial resistance.  Because MDR strains have evolved global strategies to identify and fight our antibiotic arsenal, it is important to constantly update our global knowledge on antibiotic transport.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVvvxKCgz2e7Vg90H21EOLACvtfcHk0lh0yh0QJf9PQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFGqsLrJ&md5=6fc21e8bbbdff1a67f441b7515dc7f3f</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.2174%2F1874285801307010022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1874285801307010022%26sid%3Dliteratum%253Aachs%26aulast%3DMasi%26aufirst%3DM.%26aulast%3DPag%25C3%25A8s%26aufirst%3DJ.-M.%26atitle%3DStructure%252C%2520function%2520and%2520regulation%2520of%2520outer%2520membrane%2520proteins%2520involved%2520in%2520drug%2520transport%2520in%2520enterobactericeae%253A%2520the%2520OmpF%252FC%2520-%2520TolC%2520case%26jtitle%3DOpen%2520Microbiol.%2520J.%26date%3D2013%26volume%3D7%26spage%3D22%26epage%3D33%26doi%3D10.2174%2F1874285801307010022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nishino, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamaguchi, A.</span></span> <span> </span><span class="NLM_article-title">Analysis of a complete library of putative drug transporter genes in Escherichia coli</span>. <i>J. Bacteriol.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>183</i></span>,  <span class="NLM_fpage">5803</span>– <span class="NLM_lpage">5812</span>, <span class="refDoi"> DOI: 10.1128/jb.183.20.5803-5812.2001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1128%2Fjb.183.20.5803-5812.2001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=11566977" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BD3MXntlanurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=183&publication_year=2001&pages=5803-5812&author=K.+Nishinoauthor=A.+Yamaguchi&title=Analysis+of+a+complete+library+of+putative+drug+transporter+genes+in+Escherichia+coli&doi=10.1128%2Fjb.183.20.5803-5812.2001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Analysis of a complete library of putative drug transporter genes in Escherichia coli</span></div><div class="casAuthors">Nishino, Kunihiko; Yamaguchi, Akihito</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Bacteriology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">183</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">5803-5812</span>CODEN:
                <span class="NLM_cas:coden">JOBAAY</span>;
        ISSN:<span class="NLM_cas:issn">0021-9193</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The complete sequencing of bacterial genomes has revealed a large no. of drug transporter genes.  In Escherichia coli, there are 37 open reading frames (ORFs) assumed to be drug transporter genes on the basis of sequence similarities, although the transport capabilities of most of them have not been established yet.  The authors cloned all 37 putative drug transporter genes in E. coli and investigated their drug resistance phenotypes using an E. coli drug-sensitive mutant as a host.  E. coli cells transformed with a plasmid carrying one of 20 ORFs, i.e., fsr, mdfA, yceE, yceL, bcr, emrKY, emrAB, emrD, yidY, yjiO, ydhE, acrAB, cusA (formerly ybdE), yegMNO, acrD, acrEF, yhiUV, emrE, ydgFE, and ybjYZ, exhibited increased resistance to some of the 26 representative antimicrobial agents and chem. compds. tested in this study.  Of these 20 ORFs, cusA, yegMNO, ydgFE, yceE, yceL, yidY, and ybjYZ are novel drug resistance genes.  The fsr, bcr, yjiO, ydhE, acrD, and yhiUV genes gave broader resistance spectra than previously reported.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHqUeniJrxvLVg90H21EOLACvtfcHk0lh0yh0QJf9PQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXntlanurs%253D&md5=0076182ce947886a3f4f5ec915cbf08a</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1128%2Fjb.183.20.5803-5812.2001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252Fjb.183.20.5803-5812.2001%26sid%3Dliteratum%253Aachs%26aulast%3DNishino%26aufirst%3DK.%26aulast%3DYamaguchi%26aufirst%3DA.%26atitle%3DAnalysis%2520of%2520a%2520complete%2520library%2520of%2520putative%2520drug%2520transporter%2520genes%2520in%2520Escherichia%2520coli%26jtitle%3DJ.%2520Bacteriol.%26date%3D2001%26volume%3D183%26spage%3D5803%26epage%3D5812%26doi%3D10.1128%2Fjb.183.20.5803-5812.2001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tikhonova, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zgurskaya, H. I.</span></span> <span> </span><span class="NLM_article-title">AcrA, AcrB, and TolC ofEscherichia coliForm a Stable Intermembrane Multidrug Efflux Complex</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>279</i></span>,  <span class="NLM_fpage">32116</span>– <span class="NLM_lpage">32124</span>, <span class="refDoi"> DOI: 10.1074/jbc.m402230200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1074%2Fjbc.m402230200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=15155734" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmtVCiu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=279&publication_year=2004&pages=32116-32124&author=E.+B.+Tikhonovaauthor=H.+I.+Zgurskaya&title=AcrA%2C+AcrB%2C+and+TolC+ofEscherichia+coliForm+a+Stable+Intermembrane+Multidrug+Efflux+Complex&doi=10.1074%2Fjbc.m402230200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">AcrA, AcrB, and TolC of Escherichia coli form a stable intermembrane multidrug efflux complex</span></div><div class="casAuthors">Tikhonova, Elena B.; Zgurskaya, Helen I.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">279</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">32116-32124</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Many transporters of Gram-neg. bacteria involved in the extracellular secretion of proteins and the efflux of toxic mols. operate by forming intermembrane complexes.  These complexes are proposed to span both inner and outer membranes and create a bridge across the periplasm.  In this study, we analyzed interactions between the inner and outer membrane components of the tri-partite multidrug efflux pump AcrAB-TolC from Escherichia coli.  We found that, once assembled, the intermembrane AcrAB-TolC complex is stable during the sepn. of the inner and outer membranes and subsequent purifn.  All three components of the complex co-purify when the affinity tag is attached to either of the proteins suggesting bi-partite interactions between AcrA, AcrB, and TolC.  We show that antibiotics, the substrates of AcrAB-TolC, stabilize interactions within the complex.  However, the formation of the AcrAB-TolC complex does not require an input of energy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcRm1Wtm5dL7Vg90H21EOLACvtfcHk0lh0yh0QJf9PQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmtVCiu7o%253D&md5=a8969321f1326c367b6f16d613293c73</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1074%2Fjbc.m402230200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.m402230200%26sid%3Dliteratum%253Aachs%26aulast%3DTikhonova%26aufirst%3DE.%2BB.%26aulast%3DZgurskaya%26aufirst%3DH.%2BI.%26atitle%3DAcrA%252C%2520AcrB%252C%2520and%2520TolC%2520ofEscherichia%2520coliForm%2520a%2520Stable%2520Intermembrane%2520Multidrug%2520Efflux%2520Complex%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2004%26volume%3D279%26spage%3D32116%26epage%3D32124%26doi%3D10.1074%2Fjbc.m402230200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.-Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plésiat, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikaido, H.</span></span> <span> </span><span class="NLM_article-title">The challenge of efflux-mediated antibiotic resistance in Gram-negative bacteria</span>. <i>Clin. Microbiol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">337</span>– <span class="NLM_lpage">418</span>, <span class="refDoi"> DOI: 10.1128/cmr.00117-14</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1128%2FCMR.00117-14" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=25788514" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BC1cXltlyhs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2015&pages=337-418&author=X.-Z.+Liauthor=P.+Pl%C3%A9siatauthor=H.+Nikaido&title=The+challenge+of+efflux-mediated+antibiotic+resistance+in+Gram-negative+bacteria&doi=10.1128%2Fcmr.00117-14"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">The challenge of efflux-mediated antibiotic resistance in Gram-negative bacteria</span></div><div class="casAuthors">Li, Xian-Zhi; Plesiat, Patrick; Nikaido, Hiroshi</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Microbiology Reviews</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">337-418</span>CODEN:
                <span class="NLM_cas:coden">CMIREX</span>;
        ISSN:<span class="NLM_cas:issn">1098-6618</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The global emergence of multidrug-resistant Gram-neg. bacteria is a growing threat to antibiotic therapy.  The chromosomally encoded drug efflux mechanisms that are ubiquitous in these bacteria greatly contribute to antibiotic resistance and present a major challenge for antibiotic development.  Multidrug pumps, particularly those represented by the clin. relevant AcrAB-TolC and Mex pumps of the resistance-nodulation-division (RND) superfamily, not only mediate intrinsic and acquired multidrug resistance (MDR) but also are involved in other functions, including the bacterial stress response and pathogenicity.  Addnl., efflux pumps interact synergistically with other resistance mechanisms (e.g., with the outer membrane permeability barrier) to increase resistance levels.  Since the discovery of RND pumps in the early 1990s, remarkable scientific and technol. advances have allowed for an in-depth understanding of the structural and biochem. basis, substrate profiles, mol. regulation, and inhibition of MDR pumps.  However, the development of clin. useful efflux pump inhibitors and/or new antibiotics that can bypass pump effects continues to be a challenge.  Plasmid-borne efflux pump genes (including those for RND pumps) have increasingly been identified.  This article highlights the recent progress obtained for organisms of clin. significance, together with methodol. considerations for the characterization of MDR pumps.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8mBa4pi9DV7Vg90H21EOLACvtfcHk0lh0yh0QJf9PQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXltlyhs7o%253D&md5=f481bdc9d3813f008c4c69502e9c9639</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1128%2FCMR.00117-14&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FCMR.00117-14%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DX.-Z.%26aulast%3DPl%25C3%25A9siat%26aufirst%3DP.%26aulast%3DNikaido%26aufirst%3DH.%26atitle%3DThe%2520challenge%2520of%2520efflux-mediated%2520antibiotic%2520resistance%2520in%2520Gram-negative%2520bacteria%26jtitle%3DClin.%2520Microbiol.%2520Rev.%26date%3D2015%26volume%3D28%26spage%3D337%26epage%3D418%26doi%3D10.1128%2Fcmr.00117-14" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Godinho, L. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reis, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tepper, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poelarends, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quax, W. J.</span></span> <span> </span><span class="NLM_article-title">Discovery of an Escherichia coli esterase with high activity and enantioselectivity toward 1,2-O-isopropylideneglycerol esters</span>. <i>Appl. Environ. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>77</i></span>,  <span class="NLM_fpage">6094</span>– <span class="NLM_lpage">6099</span>, <span class="refDoi"> DOI: 10.1128/aem.05122-11</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1128%2Faem.05122-11" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=21764964" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1OjurnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2011&pages=6094-6099&author=L.+F.+Godinhoauthor=C.+R.+Reisauthor=P.+G.+Tepperauthor=G.+J.+Poelarendsauthor=W.+J.+Quax&title=Discovery+of+an+Escherichia+coli+esterase+with+high+activity+and+enantioselectivity+toward+1%2C2-O-isopropylideneglycerol+esters&doi=10.1128%2Faem.05122-11"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of an Escherichia coli esterase with high activity and enantioselectivity toward 1,2-O-isopropylideneglycerol esters</span></div><div class="casAuthors">Godinho, Luis F.; Reis, Carlos R.; Tepper, Pieter G.; Poelarends, Gerrit J.; Quax, Wim J.</div><div class="citationInfo"><span class="NLM_cas:title">Applied and Environmental Microbiology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">77</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">6094-6099</span>CODEN:
                <span class="NLM_cas:coden">AEMIDF</span>;
        ISSN:<span class="NLM_cas:issn">0099-2240</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Escherichia coli has been widely used as an expression host for the identification of desired biocatalysts through screening or selection assays.  We have previously used E. coli in growth selection and screening assays for identification of Bacillus subtilis lipase variants (located in the periplasm) with improved activity and enantioselectivity toward 1,2-O-isopropylideneglycerol (IPG) esters.  In the course of these studies, we discovered that E. coli itself exhibits significant cytoplasmic esterase activity toward IPG esters.  In order to identify the enzyme (or enzymes) responsible for this esterase activity, we analyzed eight E. coli knockout strains, in which single esterase genes were deleted, for their ability to hydrolyze IPG butyrate.  This approach led to the identification of esterase YbfF as the major E. coli enzyme responsible for the hydrolytic activity toward IPG esters.  The gene coding for YbfF was cloned and overexpressed in E. coli, and the corresponding protein was purified and characterized for its biocatalytic performance.  YbfF displays a high level of activity toward IPG butyrate and IPG caprylate and prefers the R-enantiomer of these substrates, producing the S-enantiomer of the IPG product with high enantiomeric excess (72 to 94% ee).  The enantioselectivity of YbfF for IPG caprylate (E = 40) could be significantly enhanced when using DMF or DMSO as cosolvents in kinetic resoln. expts.  The enzyme also shows high enantioselectivity toward 1-phenylethyl acetate (E ≥ 200), giving the chiral product (R)-1-phenylethanol with >99% ee.  The high activity and enantioselectivity of YbfF make it an attractive enzyme for org. synthesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMEA7m3V6QbbVg90H21EOLACvtfcHk0lhuNjy-omt0lg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1OjurnE&md5=b9fa66958cd4460a4b68d5babbb265e5</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1128%2Faem.05122-11&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252Faem.05122-11%26sid%3Dliteratum%253Aachs%26aulast%3DGodinho%26aufirst%3DL.%2BF.%26aulast%3DReis%26aufirst%3DC.%2BR.%26aulast%3DTepper%26aufirst%3DP.%2BG.%26aulast%3DPoelarends%26aufirst%3DG.%2BJ.%26aulast%3DQuax%26aufirst%3DW.%2BJ.%26atitle%3DDiscovery%2520of%2520an%2520Escherichia%2520coli%2520esterase%2520with%2520high%2520activity%2520and%2520enantioselectivity%2520toward%25201%252C2-O-isopropylideneglycerol%2520esters%26jtitle%3DAppl.%2520Environ.%2520Microbiol.%26date%3D2011%26volume%3D77%26spage%3D6094%26epage%3D6099%26doi%3D10.1128%2Faem.05122-11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tomczyk, N. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nettleship, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baxter, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crichton, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Webster, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campopiano, D. J.</span></span> <span> </span><span class="NLM_article-title">Purification and characterisation of the BIOH protein from the biotin biosynthetic pathway</span>. <i>FEBS Lett.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>513</i></span>,  <span class="NLM_fpage">299</span>– <span class="NLM_lpage">304</span>, <span class="refDoi"> DOI: 10.1016/s0014-5793(02)02342-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1016%2Fs0014-5793%2802%2902342-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=11904168" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BD38XitFejsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=513&publication_year=2002&pages=299-304&author=N.+H.+Tomczykauthor=J.+E.+Nettleshipauthor=R.+L.+Baxterauthor=H.+J.+Crichtonauthor=S.+P.+Websterauthor=D.+J.+Campopiano&title=Purification+and+characterisation+of+the+BIOH+protein+from+the+biotin+biosynthetic+pathway&doi=10.1016%2Fs0014-5793%2802%2902342-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Purification and characterisation of the BIOH protein from the biotin biosynthetic pathway</span></div><div class="casAuthors">Tomczyk, Nicholas H.; Nettleship, Joanne E.; Baxter, Robert L.; Crichton, Hilary J.; Webster, Scott P.; Campopiano, Dominic J.</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Letters</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">513</span>
        (<span class="NLM_cas:issue">2,3</span>),
    <span class="NLM_cas:pages">299-304</span>CODEN:
                <span class="NLM_cas:coden">FEBLAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-5793</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">Conversion of pimeloyl-CoA (CoA) to biotin in Escherichia coli requires at least four enzymes encoded by genes in the bio operon.  One gene, bioH, which is not present in the bioABFCD operon, is required for the synthesis of pimeloyl-CoA but its exact role in formation of this intermediate is unknown.  To investigate this further, we have overexpressed and purified the bioH gene products from both E. coli (BIOH EC) and Neisseria meningitis (BIOH NM) in E. coli.  When purified BIOH was incubated with excess CoA and analyzed by electrospray mass spectrometry a species of mass corresponding to a BIOH:CoA complex was obsd.  Mutation of a conserved serine residue to alanine (BIOH EC S82A) did not prevent CoA binding.  This is the first report of the purifn. of BIOH and the observation of a small mol. bound to the protein provides clues to its role in pimeloyl-CoA synthesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6_E-LdXBQmbVg90H21EOLACvtfcHk0lhuNjy-omt0lg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XitFejsb8%253D&md5=09d7a34f7d89b010211b063e51c7824d</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2Fs0014-5793%2802%2902342-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fs0014-5793%252802%252902342-6%26sid%3Dliteratum%253Aachs%26aulast%3DTomczyk%26aufirst%3DN.%2BH.%26aulast%3DNettleship%26aufirst%3DJ.%2BE.%26aulast%3DBaxter%26aufirst%3DR.%2BL.%26aulast%3DCrichton%26aufirst%3DH.%2BJ.%26aulast%3DWebster%26aufirst%3DS.%2BP.%26aulast%3DCampopiano%26aufirst%3DD.%2BJ.%26atitle%3DPurification%2520and%2520characterisation%2520of%2520the%2520BIOH%2520protein%2520from%2520the%2520biotin%2520biosynthetic%2520pathway%26jtitle%3DFEBS%2520Lett.%26date%3D2002%26volume%3D513%26spage%3D299%26epage%3D304%26doi%3D10.1016%2Fs0014-5793%2802%2902342-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kuznetsova, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Proudfoot, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanders, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reinking, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savchenko, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yakunin, A.</span></span> <span> </span><span class="NLM_article-title">Enzyme genomics: Application of general enzymatic screens to discover new enzymes</span>. <i>FEMS Microbiol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">263</span>– <span class="NLM_lpage">279</span>, <span class="refDoi"> DOI: 10.1016/j.fmrre.2004.12.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1016%2Fj.fmrre.2004.12.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=15808744" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BD2MXivV2qurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2005&pages=263-279&author=E.+Kuznetsovaauthor=M.+Proudfootauthor=S.+Sandersauthor=J.+Reinkingauthor=A.+Savchenkoauthor=C.+Arrowsmithauthor=A.+Edwardsauthor=A.+Yakunin&title=Enzyme+genomics%3A+Application+of+general+enzymatic+screens+to+discover+new+enzymes&doi=10.1016%2Fj.fmrre.2004.12.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Enzyme genomics: Application of general enzymatic screens to discover new enzymes</span></div><div class="casAuthors">Kuznetsova, Ekaterina; Proudfoot, Michael; Sanders, Stephen A.; Reinking, Jeffrey; Savchenko, Alexei; Arrowsmith, Cheryl H.; Edwards, Aled M.; Yakunin, Alexander F.</div><div class="citationInfo"><span class="NLM_cas:title">FEMS Microbiology Reviews</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">263-279</span>CODEN:
                <span class="NLM_cas:coden">FMREE4</span>;
        ISSN:<span class="NLM_cas:issn">0168-6445</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  In all sequenced genomes, a large fraction of predicted genes encodes proteins of unknown biochem. function and up to 15% of the genes with "known" function are mis-annotated.  Several global approaches are routinely employed to predict function, including sophisticated sequence anal., gene expression, protein interaction, and protein structure.  In the first coupling of genomics and enzymol., Phizicky and colleagues undertook a screen for specific enzymes using large pools of partially purified proteins and specific enzymic assays.  Here we present an overview of the further developments of this approach, which involve the use of general enzymic assays to screen individually purified proteins for enzymic activity.  The assays have relaxed substrate specificity and are designed to identify the subclass or sub-subclasses of enzymes (phosphatase, phosphodiesterase/nuclease, protease, esterase, dehydrogenase, and oxidase) to which the unknown protein belongs.  Further biochem. characterization of proteins can be facilitated by the application of secondary screens with natural substrates (substrate profiling).  We demonstrate here the feasibility and merits of this approach for hydrolases and oxidoreductases, two very broad and important classes of enzymes.  Application of general enzymic screens and substrate profiling can greatly speed up the identification of biochem. function of unknown proteins and the exptl. verification of functional predictions produced by other functional genomics approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrd9B3Mv7L_RbVg90H21EOLACvtfcHk0lhuNjy-omt0lg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXivV2qurs%253D&md5=27f821afaa58bacb8add2ba0ad471d6f</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.fmrre.2004.12.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.fmrre.2004.12.006%26sid%3Dliteratum%253Aachs%26aulast%3DKuznetsova%26aufirst%3DE.%26aulast%3DProudfoot%26aufirst%3DM.%26aulast%3DSanders%26aufirst%3DS.%26aulast%3DReinking%26aufirst%3DJ.%26aulast%3DSavchenko%26aufirst%3DA.%26aulast%3DArrowsmith%26aufirst%3DC.%26aulast%3DEdwards%26aufirst%3DA.%26aulast%3DYakunin%26aufirst%3DA.%26atitle%3DEnzyme%2520genomics%253A%2520Application%2520of%2520general%2520enzymatic%2520screens%2520to%2520discover%2520new%2520enzymes%26jtitle%3DFEMS%2520Microbiol.%2520Rev.%26date%3D2005%26volume%3D29%26spage%3D263%26epage%3D279%26doi%3D10.1016%2Fj.fmrre.2004.12.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arpigny, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaeger, K.-E.</span></span> <span> </span><span class="NLM_article-title">Bacterial lipolytic enzymes: classification and properties</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>343</i></span>,  <span class="NLM_fpage">177</span>– <span class="NLM_lpage">183</span>, <span class="refDoi"> DOI: 10.1042/bj3430177</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1042%2Fbj3430177" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10493927" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADyaK1MXmvVyju7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=343&publication_year=1999&pages=177-183&author=J.+L.+Arpignyauthor=K.-E.+Jaeger&title=Bacterial+lipolytic+enzymes%3A+classification+and+properties&doi=10.1042%2Fbj3430177"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Bacterial lipolytic enzymes: classification and properties</span></div><div class="casAuthors">Arpigny, Jean Louis; Jaeger, Karl-Erich</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">343</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">177-183</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">Knowledge of bacterial lipolytic enzymes is increasing at a rapid and exciting rate.  To obtain an overview of this industrially very important class of enzymes and their characteristics, we have collected and classified the information available from protein and nucleotide databases.  Here we propose an updated and extensive classification of bacterial esterases and lipases based mainly on a comparison of their amino acid sequences and some fundamental biol. properties.  These new insights result in the identification of eight different families with the largest being further divided into six subfamilies.  Moreover, the classification enables us to predict (1) important structural features such as residues forming the catalytic site or the presence of disulfide bonds, (2) types of secretion mechanism and requirement for lipase-specific foldases, and (3) the potential relationship to other enzyme families.  This work will therefore contribute to a faster identification and to an easier characterization of novel bacterial lipolytic enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopK9Mikju05bVg90H21EOLACvtfcHk0lhaPEwh1fHP-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXmvVyju7o%253D&md5=090196a9865b862342ec6964d61101df</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1042%2Fbj3430177&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252Fbj3430177%26sid%3Dliteratum%253Aachs%26aulast%3DArpigny%26aufirst%3DJ.%2BL.%26aulast%3DJaeger%26aufirst%3DK.-E.%26atitle%3DBacterial%2520lipolytic%2520enzymes%253A%2520classification%2520and%2520properties%26jtitle%3DBiochem.%2520J.%26date%3D1999%26volume%3D343%26spage%3D177%26epage%3D183%26doi%3D10.1042%2Fbj3430177" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bornscheuer, U. T.</span></span> <span> </span><span class="NLM_article-title">Microbial carboxyl esterases: classification, properties and application in biocatalysis</span>. <i>FEMS Microbiol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">73</span>– <span class="NLM_lpage">81</span>, <span class="refDoi"> DOI: 10.1111/j.1574-6976.2002.tb00599.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1111%2Fj.1574-6976.2002.tb00599.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=12007643" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BD38Xjtl2itL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2002&pages=73-81&author=U.+T.+Bornscheuer&title=Microbial+carboxyl+esterases%3A+classification%2C+properties+and+application+in+biocatalysis&doi=10.1111%2Fj.1574-6976.2002.tb00599.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Microbial carboxyl esterases: classification, properties and application in biocatalysis</span></div><div class="casAuthors">Bornscheuer, Uwe T.</div><div class="citationInfo"><span class="NLM_cas:title">FEMS Microbiology Reviews</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">73-81</span>CODEN:
                <span class="NLM_cas:coden">FMREE4</span>;
        ISSN:<span class="NLM_cas:issn">0168-6445</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">A review with 79 refs.  Esterases (EC 3.1.1.x) represent a diverse group of hydrolases catalyzing the cleavage and formation of ester bonds and are widely distributed in animals, plants, and microorganisms.  In addn. to lipases, a considerable no. of microbial carboxyl esterases have also been discovered and overexpressed.  This review summarizes their properties and classification.  Special emphasis is given on their application in org. synthesis for the resoln. of racemates and pro-stereogenic compds.  In addn., recent results for altering their properties by directed evolution are presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpl1yo2gShFArVg90H21EOLACvtfcHk0lhaPEwh1fHP-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xjtl2itL4%253D&md5=a852ea9888822ea974713fc655258a11</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1111%2Fj.1574-6976.2002.tb00599.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1574-6976.2002.tb00599.x%26sid%3Dliteratum%253Aachs%26aulast%3DBornscheuer%26aufirst%3DU.%2BT.%26atitle%3DMicrobial%2520carboxyl%2520esterases%253A%2520classification%252C%2520properties%2520and%2520application%2520in%2520biocatalysis%26jtitle%3DFEMS%2520Microbiol.%2520Rev.%26date%3D2002%26volume%3D26%26spage%3D73%26epage%3D81%26doi%3D10.1111%2Fj.1574-6976.2002.tb00599.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sjuts, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vargiu, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwasny, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, S. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ornik, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruggerone, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bowlin, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikaido, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pos, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Opperman, T. J.</span></span> <span> </span><span class="NLM_article-title">Molecular basis for inhibition of AcrB multidrug efflux pump by novel and powerful pyranopyridine derivatives</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">3509</span>– <span class="NLM_lpage">3514</span>, <span class="refDoi"> DOI: 10.1073/pnas.1602472113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1073%2Fpnas.1602472113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=26976576" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BC28XktFemtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2016&pages=3509-3514&author=H.+Sjutsauthor=A.+V.+Vargiuauthor=S.+M.+Kwasnyauthor=S.+T.+Nguyenauthor=H.-S.+Kimauthor=X.+Dingauthor=A.+R.+Ornikauthor=P.+Ruggeroneauthor=T.+L.+Bowlinauthor=H.+Nikaidoauthor=K.+M.+Posauthor=T.+J.+Opperman&title=Molecular+basis+for+inhibition+of+AcrB+multidrug+efflux+pump+by+novel+and+powerful+pyranopyridine+derivatives&doi=10.1073%2Fpnas.1602472113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular basis for inhibition of AcrB multidrug efflux pump by novel and powerful pyranopyridine derivatives</span></div><div class="casAuthors">Sjuts, Hanno; Vargiu, Attilio V.; Kwasny, Steven M.; Nguyen, Son T.; Kim, Hong-Suk; Ding, Xiaoyuan; Ornik, Alina R.; Ruggerone, Paolo; Bowlin, Terry L.; Nikaido, Hiroshi; Pos, Klaas M.; Opperman, Timothy J.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">3509-3514</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The Escherichia coli AcrAB-TolC efflux pump is the archetype of the resistance nodulation cell division (RND) exporters from Gram-neg. bacteria.  Overexpression of RND-type efflux pumps is a major factor in multidrug resistance (MDR), which makes these pumps important antibacterial drug discovery targets.  The authors have recently developed novel pyranopyridine-based inhibitors of AcrB, which are orders of magnitude more powerful than the previously known inhibitors.  However, further development of such inhibitors has been hindered by the lack of structural information for rational drug design.  Although only the sol., periplasmic part of AcrB binds and exports the ligands, the presence of the membrane-embedded domain in AcrB and its polyspecific binding behavior have made cocrystn. with drugs challenging.  To overcome this obstacle, the authors have engineered and produced a sol. version of AcrB [AcrB periplasmic domain (AcrBper)], which is highly congruent in structure with the periplasmic part of the full-length protein, and is capable of binding substrates and potent inhibitors.  Here, the authors describe the mol. basis for pyranopyridine-based inhibition of AcrB using a combination of cellular, x-ray crystallog., and mol. dynamics (MD) simulations studies.  The pyranopyridines bind within a phenylalanine-rich cage that branches from the deep binding pocket of AcrB, where they form extensive hydrophobic interactions.  Moreover, the increasing potency of improved inhibitors correlates with the formation of a delicate protein- and water-mediated hydrogen bond network.  These detailed insights provide a mol. platform for the development of novel combinational therapies using efflux pump inhibitors for combating multidrug resistant Gram-neg. pathogens.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAJfI8XTMkfrVg90H21EOLACvtfcHk0lhaPEwh1fHP-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XktFemtrw%253D&md5=49593df0938148f8ec4a84a4ad30cecd</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1602472113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1602472113%26sid%3Dliteratum%253Aachs%26aulast%3DSjuts%26aufirst%3DH.%26aulast%3DVargiu%26aufirst%3DA.%2BV.%26aulast%3DKwasny%26aufirst%3DS.%2BM.%26aulast%3DNguyen%26aufirst%3DS.%2BT.%26aulast%3DKim%26aufirst%3DH.-S.%26aulast%3DDing%26aufirst%3DX.%26aulast%3DOrnik%26aufirst%3DA.%2BR.%26aulast%3DRuggerone%26aufirst%3DP.%26aulast%3DBowlin%26aufirst%3DT.%2BL.%26aulast%3DNikaido%26aufirst%3DH.%26aulast%3DPos%26aufirst%3DK.%2BM.%26aulast%3DOpperman%26aufirst%3DT.%2BJ.%26atitle%3DMolecular%2520basis%2520for%2520inhibition%2520of%2520AcrB%2520multidrug%2520efflux%2520pump%2520by%2520novel%2520and%2520powerful%2520pyranopyridine%2520derivatives%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2016%26volume%3D113%26spage%3D3509%26epage%3D3514%26doi%3D10.1073%2Fpnas.1602472113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span> <span> </span><span class="NLM_article-title">Supramolecular nanofibers with superior bioactivity to insulin-like growth factor-I</span>. <i>Nano Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">1560</span>– <span class="NLM_lpage">1569</span>, <span class="refDoi"> DOI: 10.1021/acs.nanolett.8b04406</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.nanolett.8b04406" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjsVeisL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2019&pages=1560-1569&author=Y.+Shangauthor=D.+Zhiauthor=G.+Fengauthor=Z.+Wangauthor=D.+Maoauthor=S.+Guoauthor=R.+Liuauthor=L.+Liuauthor=S.+Zhangauthor=S.+Sunauthor=K.+Wangauthor=D.+Kongauthor=J.+Gaoauthor=Z.+Yang&title=Supramolecular+nanofibers+with+superior+bioactivity+to+insulin-like+growth+factor-I&doi=10.1021%2Facs.nanolett.8b04406"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Supramolecular Nanofibers with Superior Bioactivity to Insulin-Like Growth Factor-I</span></div><div class="casAuthors">Shang, Yuna; Zhi, Dengke; Feng, Guowei; Wang, Zhongyan; Mao, Duo; Guo, Shuang; Liu, Ruihua; Liu, Lulu; Zhang, Shuhao; Sun, Shenghuan; Wang, Kai; Kong, Deling; Gao, Jie; Yang, Zhimou</div><div class="citationInfo"><span class="NLM_cas:title">Nano Letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1560-1569</span>CODEN:
                <span class="NLM_cas:coden">NALEFD</span>;
        ISSN:<span class="NLM_cas:issn">1530-6984</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bioactive peptides derived from proteins generally need to be folded into secondary structures to activate downstream signaling pathways.  However, synthetic peptides typically form random-coils, thus losing their bioactivities.  Here, we show that by introducing a self-assembling peptide motif and using different prepn. pathways, a peptide from insulin-like growth factor-I (IGF-1) can be folded into an α-helix and β-sheet.  The β-sheet one exhibits a low dissocn. const. to the IGF-1 receptor (IGF-1R, 11.5 nM), which is only about 3 times higher than that of IGF-1 (4.3 nM).  However, the α-helical one and the peptide without self-assembling motif show weak affinities to IGF-1R (KD = 179.1 and 321.6 nM, resp.).  At 10 nM, the β-sheet one efficiently activates the IGF-1 downstream pathway, significantly enhancing HUVEC proliferation and preventing cell apoptosis.  The β-sheet peptide shows superior performance to IGF-1 in vivo, and it improves ischemic hind-limb salvage by significantly reducing muscle degrdn. and enhancing limb vascularization.  Our study provides a useful strategy to constrain peptides into different conformations, which may lead to the development of supramol. nanomaterials mimicking biofunctional proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0gIjwARXkoLVg90H21EOLACvtfcHk0lgVzwt9HdRi-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjsVeisL4%253D&md5=42e72e0fe56a128a407f6d0a94768387</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Facs.nanolett.8b04406&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.nanolett.8b04406%26sid%3Dliteratum%253Aachs%26aulast%3DShang%26aufirst%3DY.%26aulast%3DZhi%26aufirst%3DD.%26aulast%3DFeng%26aufirst%3DG.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DMao%26aufirst%3DD.%26aulast%3DGuo%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DR.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DSun%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DK.%26aulast%3DKong%26aufirst%3DD.%26aulast%3DGao%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DZ.%26atitle%3DSupramolecular%2520nanofibers%2520with%2520superior%2520bioactivity%2520to%2520insulin-like%2520growth%2520factor-I%26jtitle%3DNano%2520Lett.%26date%3D2019%26volume%3D19%26spage%3D1560%26epage%3D1569%26doi%3D10.1021%2Facs.nanolett.8b04406" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hua, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ou, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, M.</span></span> <span> </span><span class="NLM_article-title">Folic acid derived hydrogel enhances the survival and promotes therapeutic efficacy of iPS cells for acute myocardial infarction</span>. <i>ACS Appl. Mater. Interfaces</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">24459</span>– <span class="NLM_lpage">24468</span>, <span class="refDoi"> DOI: 10.1021/acsami.8b08659</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsami.8b08659" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1Knsr7L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2018&pages=24459-24468&author=H.+Liauthor=J.+Gaoauthor=Y.+Shangauthor=Y.+Huaauthor=M.+Yeauthor=Z.+Yangauthor=C.+Ouauthor=M.+Chen&title=Folic+acid+derived+hydrogel+enhances+the+survival+and+promotes+therapeutic+efficacy+of+iPS+cells+for+acute+myocardial+infarction&doi=10.1021%2Facsami.8b08659"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Folic Acid Derived Hydrogel Enhances the Survival and Promotes Therapeutic Efficacy of iPS Cells for Acute Myocardial Infarction</span></div><div class="casAuthors">Li, Hekai; Gao, Jie; Shang, Yuna; Hua, Yongquan; Ye, Min; Yang, Zhimou; Ou, Caiwen; Chen, Minsheng</div><div class="citationInfo"><span class="NLM_cas:title">ACS Applied Materials & Interfaces</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">29</span>),
    <span class="NLM_cas:pages">24459-24468</span>CODEN:
                <span class="NLM_cas:coden">AAMICK</span>;
        ISSN:<span class="NLM_cas:issn">1944-8244</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Stem cell therapy has obtained extensive consensus to be an effective method for post myocardial infarction (MI) intervention.  Induced pluripotent stem (iPS) cells, which are able to differentiate into multiple cell types, have the potential to generate cardiovascular lineage cells for myocardial repair after ischemic damage, but their poor retention rate significantly hinders the therapeutic efficacy.  In the present study, we developed a supramol. hydrogel which is formed by the self-assembly of folic acid (FA)-modified peptide via a biocompatible method (glutathione redn.) and suitable for cell encapsulation and transplantation.  The iPS cells labeled with CM-Dil were transplanted into the MI hearts of mice with or without FA hydrogel encapsulation.  The results corroborated that the FA hydrogel significantly improved the retention and survival of iPS cells in MI hearts post injection, leading to augmentation of the therapeutic efficacy of iPS cells including better cardiac function and much less adverse heart remodeling, by subsequent differentiation toward cardiac cells and stimulation of neovascularization.  This study reported a novel supramol. hydrogel based on FA-peptides capable of improving the therapeutic capacity of iPS cells, which held big potential in the treatment of MI.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoy3dxgJRp4CbVg90H21EOLACvtfcHk0lgVzwt9HdRi-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1Knsr7L&md5=ef582ee438e2092661c78267836714cc</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Facsami.8b08659&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsami.8b08659%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DH.%26aulast%3DGao%26aufirst%3DJ.%26aulast%3DShang%26aufirst%3DY.%26aulast%3DHua%26aufirst%3DY.%26aulast%3DYe%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DOu%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DM.%26atitle%3DFolic%2520acid%2520derived%2520hydrogel%2520enhances%2520the%2520survival%2520and%2520promotes%2520therapeutic%2520efficacy%2520of%2520iPS%2520cells%2520for%2520acute%2520myocardial%2520infarction%26jtitle%3DACS%2520Appl.%2520Mater.%2520Interfaces%26date%3D2018%26volume%3D10%26spage%3D24459%26epage%3D24468%26doi%3D10.1021%2Facsami.8b08659" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span> <span> </span><span class="NLM_article-title">Tandem molecular self-assembly in liver cancer cells</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">1813</span>– <span class="NLM_lpage">1816</span>, <span class="refDoi"> DOI: 10.1002/anie.201710237</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1002%2Fanie.201710237" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlKrsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2018&pages=1813-1816&author=J.+Zhanauthor=Y.+Caiauthor=S.+Heauthor=L.+Wangauthor=Z.+Yang&title=Tandem+molecular+self-assembly+in+liver+cancer+cells&doi=10.1002%2Fanie.201710237"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Tandem Molecular Self-Assembly in Liver Cancer Cells</span></div><div class="casAuthors">Zhan, Jie; Cai, Yanbin; He, Shuangshuang; Wang, Ling; Yang, Zhimou</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1813-1816</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The authors herein describe the tandem mol. self-assembly of a peptide deriv. (1) that is controlled by a combination of enzymic and chem. reactions.  In phosphate-buffered saline (PBS), compd. 1 self-assembles first into nanoparticles by phosphatase and then into nanofibers by glutathione.  Liver cancer cells exhibit higher concns. of both phosphatase and GSH than normal cells.  Therefore, the tandem self-assembly of 1 also occurs in the liver cancer cell lines HepG2 and QGY7703; compd. 1 first forms nanoparticles around the cells and then forms nanofibers inside the cells.  Owing to this self-assembly mechanism, compd. 1 exhibits large ratios for cellular uptake and inhibition of cell viability between liver cancer cells and normal liver cells.  The authors envision that using both extracellular and intracellular reactions to trigger tandem mol. self-assembly could lead to the development of supramol. nanomaterials with improved performance in cancer diagnostics and therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpowoReFd-OdbVg90H21EOLACvtfcHk0lgVzwt9HdRi-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlKrsLY%253D&md5=06fcf89d59333758e65cfe0d42727f03</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1002%2Fanie.201710237&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201710237%26sid%3Dliteratum%253Aachs%26aulast%3DZhan%26aufirst%3DJ.%26aulast%3DCai%26aufirst%3DY.%26aulast%3DHe%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DZ.%26atitle%3DTandem%2520molecular%2520self-assembly%2520in%2520liver%2520cancer%2520cells%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2018%26volume%3D57%26spage%3D1813%26epage%3D1816%26doi%3D10.1002%2Fanie.201710237" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gellman, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeGrado, W. F.</span></span> <span> </span><span class="NLM_article-title">β-Peptides: from structure to function</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>101</i></span>,  <span class="NLM_fpage">3219</span>– <span class="NLM_lpage">3232</span>, <span class="refDoi"> DOI: 10.1021/cr000045i</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr000045i" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BD3MXntlGntbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2001&pages=3219-3232&author=R.+P.+Chengauthor=S.+H.+Gellmanauthor=W.+F.+DeGrado&title=%CE%B2-Peptides%3A+from+structure+to+function&doi=10.1021%2Fcr000045i"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">β-Peptides: From structure to function</span></div><div class="casAuthors">Cheng, Richard P.; Gellman, Samuel H.; DeGrado, William F.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, D. C.)</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">101</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3219-3232</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review with 187 refs., of the conformational properties and biol. functions, esp. as peptidomimetics, of β-peptides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTfycZ30AaKLVg90H21EOLACvtfcHk0ljRiCXt6QJZKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXntlGntbg%253D&md5=40cbf3f3d3c00ec8d6250adf407a663a</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1021%2Fcr000045i&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr000045i%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DR.%2BP.%26aulast%3DGellman%26aufirst%3DS.%2BH.%26aulast%3DDeGrado%26aufirst%3DW.%2BF.%26atitle%3D%25CE%25B2-Peptides%253A%2520from%2520structure%2520to%2520function%26jtitle%3DChem.%2520Rev.%26date%3D2001%26volume%3D101%26spage%3D3219%26epage%3D3232%26doi%3D10.1021%2Fcr000045i" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kreitler, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinkruger, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mortenson, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mittal, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Travis, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forest, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gellman, S. H.</span></span> <span> </span><span class="NLM_article-title">A hendecad motif is preferred for heterochiral coiled-coil formation</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>141</i></span>,  <span class="NLM_fpage">1583</span>– <span class="NLM_lpage">1592</span>, <span class="refDoi"> DOI: 10.1021/jacs.8b11246</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.8b11246" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BC1MXpsFejtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=141&publication_year=2019&pages=1583-1592&author=D.+F.+Kreitlerauthor=Z.+Yaoauthor=J.+D.+Steinkrugerauthor=D.+E.+Mortensonauthor=L.+Huangauthor=R.+Mittalauthor=B.+R.+Travisauthor=K.+T.+Forestauthor=S.+H.+Gellman&title=A+hendecad+motif+is+preferred+for+heterochiral+coiled-coil+formation&doi=10.1021%2Fjacs.8b11246"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">A hendecad motif is preferred for heterochiral coiled-coil formation</span></div><div class="casAuthors">Kreitler, Dale F.; Yao, Zhihui; Steinkruger, Jay D.; Mortenson, David E.; Huang, Lijun; Mittal, Ritesh; Travis, Benjamin R.; Forest, Katrina T.; Gellman, Samuel H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">141</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1583-1592</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The structural principles that govern interactions between L- and D-peptides are not well understood.  Among natural proteins, coiled-coil assemblies formed between or among α-helixes are the most regular feature of tertiary and quaternary structures.  We recently reported the 1st high-resoln. structures for heterochiral coiled-coil dimers, which represent a starting point for understanding assocns. of L- and D-polypeptides.  These structures were an unexpected outcome from crystn. of a racemic peptide corresponding to the transmembrane domain of the influenza A M2 protein (M2-TM).  The reported structures raised the possibility that heterochiral coiled-coil dimers prefer an 11-residue (hendecad) sequence repeat, in contrast to the 7-residue (heptad) sequence repeat that is dominant among natural coiled-coils.  To gain insight on sequence repeat preferences of heterochiral coiled-coils, we examd. 3 M2-TM variants contg. substitutions intended to minimize steric clashes between side-chains at the coiled-coil interface.  In each of the 3 new crystal structures, we obsd. heterochiral coiled-coil assocns. that closely matched a hendecad sequence motif, which strengthens the conclusion that this motif is intrinsic to the pairing of α-helixes with opposite handedness.  In each case, the presence of a hendecad motif was established by comparing the obsd. helical frequency to that of an ideal hendecad.  This comparison revealed that decreasing the size of the amino acid side-chain at positions that project toward the superhelical axis produces tighter packing, as detd. by the size of the coiled-coil radius.  These results provide a basis for future design of heterochiral coiled-coil pairings.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXI23Fe-4ZMLVg90H21EOLACvtfcHk0ljRiCXt6QJZKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXpsFejtQ%253D%253D&md5=223f0d8d062f3cf76af6f3a9453a8e89</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Fjacs.8b11246&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.8b11246%26sid%3Dliteratum%253Aachs%26aulast%3DKreitler%26aufirst%3DD.%2BF.%26aulast%3DYao%26aufirst%3DZ.%26aulast%3DSteinkruger%26aufirst%3DJ.%2BD.%26aulast%3DMortenson%26aufirst%3DD.%2BE.%26aulast%3DHuang%26aufirst%3DL.%26aulast%3DMittal%26aufirst%3DR.%26aulast%3DTravis%26aufirst%3DB.%2BR.%26aulast%3DForest%26aufirst%3DK.%2BT.%26aulast%3DGellman%26aufirst%3DS.%2BH.%26atitle%3DA%2520hendecad%2520motif%2520is%2520preferred%2520for%2520heterochiral%2520coiled-coil%2520formation%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2019%26volume%3D141%26spage%3D1583%26epage%3D1592%26doi%3D10.1021%2Fjacs.8b11246" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abdulkadir, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">Inhibition of β-catenin/B cell lymphoma 9 protein–protein interaction using α-helix–mimicking sulfono-γ-AApeptide inhibitors</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">10757</span>– <span class="NLM_lpage">10762</span>, <span class="refDoi"> DOI: 10.1073/pnas.1819663116</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1073%2Fpnas.1819663116" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=31088961" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtVChsr%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2019&pages=10757-10762&author=P.+Sangauthor=M.+Zhangauthor=Y.+Shiauthor=C.+Liauthor=S.+Abdulkadirauthor=Q.+Liauthor=H.+Jiauthor=J.+Cai&title=Inhibition+of+%CE%B2-catenin%2FB+cell+lymphoma+9+protein%E2%80%93protein+interaction+using+%CE%B1-helix%E2%80%93mimicking+sulfono-%CE%B3-AApeptide+inhibitors&doi=10.1073%2Fpnas.1819663116"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of β-catenin/B cell lymphoma 9 protein-protein interaction using α-helix-mimicking sulfono-γ-AApeptide inhibitors</span></div><div class="casAuthors">Sang, Peng; Zhang, Min; Shi, Yan; Li, Chunpu; Abdulkadir, Sami; Li, Qi; Ji, Haitao; Cai, Jianfeng</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">10757-10762</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The rational design of α-helix-mimicking peptidomimetics provides a streamlined approach to discover potent inhibitors for protein-protein interactions (PPIs).  However, designing cell-penetrating long peptidomimetic scaffolds equipped with various functional groups necessary for interacting with large protein-binding interfaces remains challenging.  This is particularly true for targeting β-catenin/BCL9 PPIs.  Here the authors designed a series of unprecedented helical sulfono-γ-AApeptides that mimic the binding mode of the α-helical HD2 domain of B Cell Lymphoma 9 (BCL9).  The studies show that sulfono-γ-AApeptides can structurally and functionally mimic the α-helical domain of BCL9 and selectively disrupt β-catenin/BCL9 PPIs with even higher potency.  More intriguingly, these sulfono-γ-AApeptides can enter cancer cells, bind with β-catenin and disrupt β-catenin/BCL9 PPIs, and exhibit excellent cellular activity, which is much more potent than the BCL9 peptide.  Furthermore, the enzymic stability studies demonstrate the remarkable stability of the helical sulfono-γ-AApeptides, with no degrdn. in the presence of pronase for 24 h, augmenting their biol. potential.  This work represents not only an example of helical sulfono-γ-AApeptides that mimic α-helix and disrupt protein-protein interactions, but also an excellent example of potent, selective, and cell-permeable unnatural foldameric peptidomimetics that disrupt the β-catenin/BCL9 PPI.  The design of helical sulfono-γ-AApeptides may lead to a new strategy to modulate a myriad of protein-protein interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_pKlgpb-_U7Vg90H21EOLACvtfcHk0ljRiCXt6QJZKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtVChsr%252FE&md5=145172aa8475bfe4a20b97d6b88d12dd</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1819663116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1819663116%26sid%3Dliteratum%253Aachs%26aulast%3DSang%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DM.%26aulast%3DShi%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DAbdulkadir%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DJi%26aufirst%3DH.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3DInhibition%2520of%2520%25CE%25B2-catenin%252FB%2520cell%2520lymphoma%25209%2520protein%25E2%2580%2593protein%2520interaction%2520using%2520%25CE%25B1-helix%25E2%2580%2593mimicking%2520sulfono-%25CE%25B3-AApeptide%2520inhibitors%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2019%26volume%3D116%26spage%3D10757%26epage%3D10762%26doi%3D10.1073%2Fpnas.1819663116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhattarai, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, F.-S.</span></span> <span> </span><span class="NLM_article-title">Cyclic peptidomimetics as inhibitor for miR-155 biogenesis</span>. <i>Mol. Pharm.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">914</span>– <span class="NLM_lpage">920</span>, <span class="refDoi"> DOI: 10.1021/acs.molpharmaceut.8b01247</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.molpharmaceut.8b01247" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtlSmsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2019&pages=914-920&author=H.+Yanauthor=M.+Zhouauthor=U.+Bhattaraiauthor=Y.+Songauthor=M.+Zhengauthor=J.+Caiauthor=F.-S.+Liang&title=Cyclic+peptidomimetics+as+inhibitor+for+miR-155+biogenesis&doi=10.1021%2Facs.molpharmaceut.8b01247"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclic Peptidomimetics as Inhibitor for miR-155 Biogenesis</span></div><div class="casAuthors">Yan, Hao; Zhou, Mi; Bhattarai, Umesh; Song, Yabin; Zheng, Mengmeng; Cai, Jianfeng; Liang, Fu-Sen</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmaceutics</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">914-920</span>CODEN:
                <span class="NLM_cas:coden">MPOHBP</span>;
        ISSN:<span class="NLM_cas:issn">1543-8384</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">MiR-155 plays key promoting roles in several cancers and emerges as an important anticancer therapeutic target.  However, the discovery of small mols. that target RNAs is challenging.  Peptidomimetics have been shown to be a rich source for discovering novel ligands to regulate cellular proteins.  However, the potential of using peptidomimetics for RNA targeting is relatively unexplored.  To this end, we designed and synthesized members of a novel 320 000 compd. macrocyclic peptidomimetic library.  An affinity-based screening protocol led to the identification of a pre-miR-155 binder that inhibits oncogenic miR-155 maturation in vitro and in cell and induces cancer cell apoptosis.  The results of this investigation demonstrate that macrocyclic peptidomimetics could serve as a new scaffold for RNA targeting.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqa1182LkJrALVg90H21EOLACvtfcHk0lhwjBvj23XQCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtlSmsg%253D%253D&md5=0ac2dec99b95d9efbbaef045bf9e9e4f</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Facs.molpharmaceut.8b01247&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.molpharmaceut.8b01247%26sid%3Dliteratum%253Aachs%26aulast%3DYan%26aufirst%3DH.%26aulast%3DZhou%26aufirst%3DM.%26aulast%3DBhattarai%26aufirst%3DU.%26aulast%3DSong%26aufirst%3DY.%26aulast%3DZheng%26aufirst%3DM.%26aulast%3DCai%26aufirst%3DJ.%26aulast%3DLiang%26aufirst%3DF.-S.%26atitle%3DCyclic%2520peptidomimetics%2520as%2520inhibitor%2520for%2520miR-155%2520biogenesis%26jtitle%3DMol.%2520Pharm.%26date%3D2019%26volume%3D16%26spage%3D914%26epage%3D920%26doi%3D10.1021%2Facs.molpharmaceut.8b01247" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">She, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teng, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peguero-Tejada, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Odom, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gjonaj, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wojtas, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Vaart, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, J.</span></span> <span> </span><span class="NLM_article-title">De novo left-handed synthetic peptidomimetic foldamers</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">9916</span>– <span class="NLM_lpage">9920</span>, <span class="refDoi"> DOI: 10.1002/anie.201805184</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=10.1002%2Fanie.201805184" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1yjt7bL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2018&pages=9916-9920&author=F.+Sheauthor=P.+Tengauthor=A.+Peguero-Tejadaauthor=M.+Wangauthor=N.+Maauthor=T.+Odomauthor=M.+Zhouauthor=E.+Gjonajauthor=L.+Wojtasauthor=A.+van+der+Vaartauthor=J.+Cai&title=De+novo+left-handed+synthetic+peptidomimetic+foldamers&doi=10.1002%2Fanie.201805184"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">De novo left-handed synthetic peptidomimetic foldamers</span></div><div class="casAuthors">She, Fengyu; Teng, Peng; Peguero-Tejada, Alfredo; Wang, Minghui; Ma, Ning; Odom, Timothy; Zhou, Mi; Gjonaj, Erald; Wojtas, Lukasz; van der Vaart, Arjan; Cai, Jianfeng</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">9916-9920</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The development of peptidomimetic helical foldamers with a wide repertoire of functions is of significant interest.  Herein, we report the X-ray crystal structures of a series of homogeneous L-sulfono-γ-AA foldamers and elucidate their folding conformation at the at. level.  Single-crystal X-ray crystallog. revealed that this class of oligomers fold into unprecedented dragon-boat-shaped and unexpected left-handed helixes, which are stabilized by the 14-hydrogen-bonding pattern present in all sequences.  These L-sulfono-γ-AA-peptides have a helical pitch of 5.1 Å and exactly four side chains per turn, and the side chains lie perfectly on top of each other along the helical axis.  2D NMR spectroscopy, computational simulations, and CD studies support the folding conformation in soln.  Our results provide a structural basis at the at. level for the design of novel biomimetics with a precise arrangement of functional groups in three dimensions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGru2vbXzMss1rVg90H21EOLACvtfcHk0lhwjBvj23XQCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1yjt7bL&md5=c45cc7e77dfa2887f5fec63d8e3dbe7e</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1002%2Fanie.201805184&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201805184%26sid%3Dliteratum%253Aachs%26aulast%3DShe%26aufirst%3DF.%26aulast%3DTeng%26aufirst%3DP.%26aulast%3DPeguero-Tejada%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DMa%26aufirst%3DN.%26aulast%3DOdom%26aufirst%3DT.%26aulast%3DZhou%26aufirst%3DM.%26aulast%3DGjonaj%26aufirst%3DE.%26aulast%3DWojtas%26aufirst%3DL.%26aulast%3Dvan%2Bder%2BVaart%26aufirst%3DA.%26aulast%3DCai%26aufirst%3DJ.%26atitle%3DDe%2520novo%2520left-handed%2520synthetic%2520peptidomimetic%2520foldamers%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2018%26volume%3D57%26spage%3D9916%26epage%3D9920%26doi%3D10.1002%2Fanie.201805184" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chan, W. C.</span>; <span class="NLM_string-name">White, P. D.</span></span> <i>Fmoc Solid Phase Peptide Synthesis: A Practical Approach</i>; <span class="NLM_publisher-name">Oxford University Press</span>: <span class="NLM_publisher-loc">Oxford</span>, <span class="NLM_year">2000</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2000&author=W.+C.+Chan&author=P.+D.+White&title=Fmoc+Solid+Phase+Peptide+Synthesis%3A+A+Practical+Approach"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DChan%26aufirst%3DW.%2BC.%26btitle%3DFmoc%2520Solid%2520Phase%2520Peptide%2520Synthesis%253A%2520A%2520Practical%2520Approach%26pub%3DOxford%2520University%2520Press%26date%3D2000" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i52"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01210">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_31830"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.9b01210">10.1021/acs.jmedchem.9b01210</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline"><sup>1</sup>H NMR spectra, <sup>13</sup>C NMR spectra, LC–MS traces and MS of all compounds, and experimental details (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01210/suppl_file/jm9b01210_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and some data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01210/suppl_file/jm9b01210_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01210/suppl_file/jm9b01210_si_001.pdf">jm9b01210_si_001.pdf (12.41 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01210/suppl_file/jm9b01210_si_002.csv">jm9b01210_si_002.csv (4.05 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b01210&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b01210%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2019.62.issue-22" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b01210" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                3MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679a7e94e8e2198b","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
