
<html lang="en"     class="pb-page"  data-request-id="64eefb86-c99f-4af4-9378-41c531895e62"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;issue:issue:10.1021/jmcmar.2020.63.issue-11;article:article:10.1021/acs.jmedchem.0c00699"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Identification of the New Covalent Allosteric Binding Site of Fructose-1,6-bisphosphatase with Disulfiram Derivatives toward Glucose Reduction" /></meta><meta name="dc.Creator" content="Yunyuan  Huang" /></meta><meta name="dc.Creator" content="Yixiang  Xu" /></meta><meta name="dc.Creator" content="Rongrong  Song" /></meta><meta name="dc.Creator" content="Shuaishuai  Ni" /></meta><meta name="dc.Creator" content="Jiaqi  Liu" /></meta><meta name="dc.Creator" content="Yanhong  Xu" /></meta><meta name="dc.Creator" content="Yanliang  Ren" /></meta><meta name="dc.Creator" content="Li  Rao" /></meta><meta name="dc.Creator" content="Yingjie  Wang" /></meta><meta name="dc.Creator" content="Lin  Wei" /></meta><meta name="dc.Creator" content="Lingling  Feng" /></meta><meta name="dc.Creator" content="Chen  Su" /></meta><meta name="dc.Creator" content="Chao  Peng" /></meta><meta name="dc.Creator" content="Jian  Li" /></meta><meta name="dc.Creator" content="Jian  Wan" /></meta><meta name="dc.Description" content="Fructose 1,6-bisphosphatase (FBPase) has attracted substantial interest as a target associated with cancer and type 2 diabetes. Herein, we found that disulfiram and its derivatives can potently inh..." /></meta><meta name="Description" content="Fructose 1,6-bisphosphatase (FBPase) has attracted substantial interest as a target associated with cancer and type 2 diabetes. Herein, we found that disulfiram and its derivatives can potently inh..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="May 7, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c00699" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2020 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00699" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c00699" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00699" /></link>
        
    
    

<title>Identification of the New Covalent Allosteric Binding Site of Fructose-1,6-bisphosphatase with Disulfiram Derivatives toward Glucose Reduction | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c00699" /></meta><meta property="og:title" content="Identification of the New Covalent Allosteric Binding Site of Fructose-1,6-bisphosphatase with Disulfiram Derivatives toward Glucose Reduction" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00699/20200604/images/large/jm0c00699_0007.jpeg" /></meta><meta property="og:description" content="Fructose 1,6-bisphosphatase (FBPase) has attracted substantial interest as a target associated with cancer and type 2 diabetes. Herein, we found that disulfiram and its derivatives can potently inhibit FBPase by covalently binding to a new C128 allosteric site distinct from the original C128 site in APO FBPase. Further identification of the allosteric inhibition mechanism reveals that the covalent binding of a fragment of 214 will result in the movement of C128 and the dissociation of helix H4 (123–128), which in turn allows S123 to more easily form new hydrogen bonds with K71 and D74 in helix H3 (69–72), thereby inhibiting FBPase activity. Notably, both disulfiram and 212 might moderately reduce blood glucose output in vivo. Therefore, our current findings not only identify a new covalent allosteric site of FBPase but also establish a structural foundation and provide a promising way for the design of covalent allosteric drugs for glucose reduction." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c00699"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00699">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c00699&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c00699&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c00699&amp;href=/doi/10.1021/acs.jmedchem.0c00699" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 11</span><span class="cit-fg-pageRange">, 6238-6247</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/11" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c00648" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/jmv063i011_1386111" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Identification of the New Covalent Allosteric Binding Site of Fructose-1,6-bisphosphatase with Disulfiram Derivatives toward Glucose Reduction</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Yunyuan Huang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yunyuan Huang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Key Laboratory of Pesticide & Chemical Biology (CCNU), Ministry of Education, College of Chemistry, Central China Normal University, Wuhan 430079, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yunyuan++Huang">Yunyuan Huang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yixiang Xu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yixiang Xu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Shanghai Key Laboratory of New Drug Design, School of Pharmacy, East China University of Science and Technology, Shanghai 200237, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yixiang++Xu">Yixiang Xu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Rongrong Song</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Rongrong Song</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Key Laboratory of Pesticide & Chemical Biology (CCNU), Ministry of Education, College of Chemistry, Central China Normal University, Wuhan 430079, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Rongrong++Song">Rongrong Song</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Shuaishuai Ni</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Shuaishuai Ni</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cancer Institute, Longhua Hospital, Shanghai University of Traditional Chinese Medicine, Shanghai 200237, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Shuaishuai++Ni">Shuaishuai Ni</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jiaqi Liu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jiaqi Liu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Key Laboratory of Pesticide & Chemical Biology (CCNU), Ministry of Education, College of Chemistry, Central China Normal University, Wuhan 430079, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jiaqi++Liu">Jiaqi Liu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yanhong Xu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yanhong Xu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Key Laboratory of Pesticide & Chemical Biology (CCNU), Ministry of Education, College of Chemistry, Central China Normal University, Wuhan 430079, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yanhong++Xu">Yanhong Xu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yanliang Ren</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yanliang Ren</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Key Laboratory of Pesticide & Chemical Biology (CCNU), Ministry of Education, College of Chemistry, Central China Normal University, Wuhan 430079, China</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#6517000b1c092508040c094b06060b104b0001104b060b"><span class="__cf_email__" data-cfemail="394b5c5740557954585055175a5a574c175c5d4c175a57">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yanliang++Ren">Yanliang Ren</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-8565-2152" title="Orcid link">http://orcid.org/0000-0001-8565-2152</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Li Rao</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Li Rao</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Key Laboratory of Pesticide & Chemical Biology (CCNU), Ministry of Education, College of Chemistry, Central China Normal University, Wuhan 430079, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Li++Rao">Li Rao</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yingjie Wang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yingjie Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Shenzhen Bay Laboratory, Shenzhen 518055, Guangdong, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yingjie++Wang">Yingjie Wang</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9800-8163" title="Orcid link">http://orcid.org/0000-0001-9800-8163</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Lin Wei</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lin Wei</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Key Laboratory of Pesticide & Chemical Biology (CCNU), Ministry of Education, College of Chemistry, Central China Normal University, Wuhan 430079, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lin++Wei">Lin Wei</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Lingling Feng</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lingling Feng</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Key Laboratory of Pesticide & Chemical Biology (CCNU), Ministry of Education, College of Chemistry, Central China Normal University, Wuhan 430079, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lingling++Feng">Lingling Feng</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-1260-4656" title="Orcid link">http://orcid.org/0000-0003-1260-4656</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Chen Su</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Chen Su</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Facility for Protein Science in Shanghai, Zhangjiang Lab, Shanghai 201210, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Chen++Su">Chen Su</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Chao Peng</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Chao Peng</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Facility for Protein Science in Shanghai, Zhangjiang Lab, Shanghai 201210, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Chao++Peng">Chao Peng</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jian Li</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jian Li</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Shanghai Key Laboratory of New Drug Design, School of Pharmacy, East China University of Science and Technology, Shanghai 200237, China</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#e08a89818e8c89a08583959394ce858495ce838e"><span class="__cf_email__" data-cfemail="177d7e76797b7e57727462646339727362397479">[email protected]</span></a>. Tel: +86-21-64252584. Fax: +86-21-64252584.</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jian++Li">Jian Li</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-7521-8798" title="Orcid link">http://orcid.org/0000-0002-7521-8798</a></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Jian Wan</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jian Wan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Key Laboratory of Pesticide & Chemical Biology (CCNU), Ministry of Education, College of Chemistry, Central China Normal University, Wuhan 430079, China</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#244e4d454a53454a6449454d480a47474a510a4140510a474a"><span class="__cf_email__" data-cfemail="89e3e0e8e7fee8e7c9e4e8e0e5a7eaeae7fca7ecedfca7eae7">[email protected]</span></a>. Tel: +86-27-67862022. Fax: +86-27-67862022.</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jian++Wan">Jian Wan</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-4172-1392" title="Orcid link">http://orcid.org/0000-0003-4172-1392</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00699&amp;href=/doi/10.1021%2Facs.jmedchem.0c00699" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 11</span><span class="cit-pageRange">, 6238–6247</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">May 7, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>27 April 2020</li><li><span class="item_label"><b>Published</b> online</span>7 May 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 11 June 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c00699" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00699</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D6238%26pageCount%3D10%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DYunyuan%2BHuang%252C%2BYixiang%2BXu%252C%2BRongrong%2BSong%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D11%26contentID%3Dacs.jmedchem.0c00699%26title%3DIdentification%2Bof%2Bthe%2BNew%2BCovalent%2BAllosteric%2BBinding%2BSite%2Bof%2BFructose-1%252C6-bisphosphatase%2Bwith%2BDisulfiram%2BDerivatives%2Btoward%2BGlucose%2BReduction%26numPages%3D10%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D6247%26publicationDate%3DJune%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c00699"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1033</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">2</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c00699" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Identification of the New Covalent Allosteric Binding Site of Fructose-1,6-bisphosphatase with Disulfiram Derivatives toward Glucose Reduction&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Yunyuan&quot;,&quot;last_name&quot;:&quot;Huang&quot;},{&quot;first_name&quot;:&quot;Yixiang&quot;,&quot;last_name&quot;:&quot;Xu&quot;},{&quot;first_name&quot;:&quot;Rongrong&quot;,&quot;last_name&quot;:&quot;Song&quot;},{&quot;first_name&quot;:&quot;Shuaishuai&quot;,&quot;last_name&quot;:&quot;Ni&quot;},{&quot;first_name&quot;:&quot;Jiaqi&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Yanhong&quot;,&quot;last_name&quot;:&quot;Xu&quot;},{&quot;first_name&quot;:&quot;Yanliang&quot;,&quot;last_name&quot;:&quot;Ren&quot;},{&quot;first_name&quot;:&quot;Li&quot;,&quot;last_name&quot;:&quot;Rao&quot;},{&quot;first_name&quot;:&quot;Yingjie&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Lin&quot;,&quot;last_name&quot;:&quot;Wei&quot;},{&quot;first_name&quot;:&quot;Lingling&quot;,&quot;last_name&quot;:&quot;Feng&quot;},{&quot;first_name&quot;:&quot;Chen&quot;,&quot;last_name&quot;:&quot;Su&quot;},{&quot;first_name&quot;:&quot;Chao&quot;,&quot;last_name&quot;:&quot;Peng&quot;},{&quot;first_name&quot;:&quot;Jian&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Jian&quot;,&quot;last_name&quot;:&quot;Wan&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;05&quot;,&quot;day&quot;:&quot;07&quot;,&quot;issue&quot;:&quot;11&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;6238-6247&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c00699&quot;},&quot;abstract&quot;:&quot;Fructose 1,6-bisphosphatase (FBPase) has attracted substantial interest as a target associated with cancer and type 2 diabetes. Herein, we found that disulfiram and its derivatives can potently inhibit FBPase by covalently binding to a new C128 allosteric site distinct from the original C128 site in APO FBPase. Further identification of the allosteric inhibition mechanism reveals that the covalent binding of a fragment of 214 will result in the movement of C128 and the dissociation of helix H4 (123–128), which in turn allows S123 to more easily form new hydrogen bonds with K71 and D74 in helix H3 (69–72), thereby inhibiting FBPase activity. Notably, both disulfiram and 212 might moderately reduce blood glucose output in vivo. Therefore, our current findings not only identify a new covalent allosteric site of FBPase but also establish a structural foundation and provide a promising way for the design of covalent allosteric drugs for glucose reduction.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00699&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00699" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00699&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00699" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00699&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00699" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00699&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00699&amp;href=/doi/10.1021/acs.jmedchem.0c00699" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c00699" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c00699" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (5 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00699&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c00699%26sid%3Dliteratum%253Aachs%26pmid%3D32375478%26genre%3Darticle%26aulast%3DHuang%26date%3D2020%26atitle%3DIdentification%2Bof%2Bthe%2BNew%2BCovalent%2BAllosteric%2BBinding%2BSite%2Bof%2BFructose-1%252C6-bisphosphatase%2Bwith%2BDisulfiram%2BDerivatives%2Btoward%2BGlucose%2BReduction%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D11%26spage%3D6238%26epage%3D6247%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (5)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=290758" title="Anatomy">Anatomy</a>,</li><li><a href="/action/doSearch?ConceptID=290925" title="Carbohydrates">Carbohydrates</a>,</li><li><a href="/action/doSearch?ConceptID=291880" title="Crystal structure">Crystal structure</a>,</li><li><a href="/action/doSearch?ConceptID=292464" title="Rodent models">Rodent models</a>,</li><li><a href="/action/doSearch?ConceptID=291763" title="Noncovalent interactions">Noncovalent interactions</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/11" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/jmcmar.2020.63.issue-11/20200611/jmcmar.2020.63.issue-11.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00699/20200604/images/medium/jm0c00699_0007.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00699/20200604/images/large/jm0c00699_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00699&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Fructose 1,6-bisphosphatase (FBPase) has attracted substantial interest as a target associated with cancer and type 2 diabetes. Herein, we found that disulfiram and its derivatives can potently inhibit FBPase by covalently binding to a new C128 allosteric site distinct from the original C128 site in APO FBPase. Further identification of the allosteric inhibition mechanism reveals that the covalent binding of a fragment of <b>214</b> will result in the movement of C128 and the dissociation of helix H4 (123–128), which in turn allows S123 to more easily form new hydrogen bonds with K71 and D74 in helix H3 (69–72), thereby inhibiting FBPase activity. Notably, both disulfiram and <b>212</b> might moderately reduce blood glucose output <i>in vivo</i>. Therefore, our current findings not only identify a new covalent allosteric site of FBPase but also establish a structural foundation and provide a promising way for the design of covalent allosteric drugs for glucose reduction.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_83742" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_83742" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Fructose-1,6-bisphosphatase (FBPase) acts as an important rate-limiting enzyme central to the gluconeogenic (GNG) pathway and regulates metabolism in dysfunctional cells or tissues, such as hepatocytes,<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> renal carcinoma cells,<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> soft tissue sarcomas,<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> cancer stem cells,<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> natural killer cells,<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> and hematopoietic progenitor cells.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> FBPase inhibitors have been developed as a potential strategy for treating cancer<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> and type 2 diabetes (T2D).<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> Moreover, many efforts have been made to discover inhibitors that bind to the AMP allosteric site of FBPase to treat T2D.<a onclick="showRef(event, 'ref7 ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref7 ref8 ref9">(7−9)</a> Among these inhibitors, MB06322 was tested in a phase II clinical trial for the treatment of T2D.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> However, it has been suggested that a metabolite (MB05099)<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> of MB06322 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00699/suppl_file/jm0c00699_si_001.pdf" class="ext-link">Figure S1</a>) leads to lactic acidosis in humans. Additionally, a previous study<a onclick="showRef(event, 'ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref12 ref13">(12,13)</a> reported that the AMP site was a difficult target due to its high dependence on electrostatic binding interactions and lack of selectivity relative to other important enzymes involved in anabolic and catabolic pathways. To address this concern, discovering and identifying a novel site that can regulate the catalytic activity of FBPase is of utmost importance.</div><div class="NLM_p">Recently, the discovery of new allosteric sites and modulators has greatly facilitated the design of new drugs.<a onclick="showRef(event, 'ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref14 ref15">(14,15)</a> Discovering drugs that covalently bind target protein residues (e.g., cysteine and serine) has become an indispensable strategy for drug discovery due to the high biological activity, high selectivity, and long action time<a onclick="showRef(event, 'ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref16 ref17">(16,17)</a> of such compounds. Thus, covalent allosteric modulators, which combine the merits of allosteric and covalent drugs, have recently emerged as a research area that has shown great potential for drug discovery.<a onclick="showRef(event, 'ref14 ref18 ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref14 ref18 ref19 ref20">(14,18−20)</a> KRAS (G12C) was considered to be an undruggable target for cancer until the discovery of new covalent allosteric modulators.<a onclick="showRef(event, 'ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref21 ref22">(21,22)</a> In past years, AMG 510, a KRAS(G12C) allosteric covalent modulator, was approved for clinical trials.<a onclick="showRef(event, 'ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref23 ref24">(23,24)</a> In our previous work,<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> we found that several nitrostyrene derivatives could potently inhibit FBPase by covalently binding to the original C128 site,<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> but the details of their covalent allosteric mechanism remain unclear.</div><div class="NLM_p last">Facing a high risk of failure, high costs, and a long period during the research and development (R&D) of novel antidiabetic agents, drug repurposing has been identified as a faster and more economical strategy. Herein, disulfiram, a well-known antialcoholism<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> and anticancer drug,<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> and its derivatives (<b>212</b>/<b>214</b>) were used to clarify the covalent allosteric mechanism of the C128 site. It is found that the disulfiram derivative <b>214</b> potently inhibits FBPase activity by covalently binding to a novel allosteric binding site, which is distinct from the original C128 site in APO FBPase<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> and not exploited previously. Furthermore, the covalent allosteric mechanism of this new site has been confirmed by integrating the crystallographic structure of FBPase–C128 modified by <b>214</b> (a disulfiram derivative), LC–MS, site-directed mutagenesis, and molecular dynamics (MD) simulations. Considering the better FBPase inhibitory activities of disulfiram and <b>212</b>, the hypoglycemic efficacies of these two compounds in ICR and <i>db/db</i> mice were further evaluated.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_82463" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_82463" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Synthesis of Compounds <b>212</b>/<b>214</b></h3><div class="NLM_p">The syntheses of compounds <b>212</b> and <b>214</b> are shown in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>, and the procedure generally followed previous works.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> Commercially available thiazole-2-thiol was added to a freshly prepared solution of 4-chlorophenyl hypochlorothioite in anhydrous ethyl ether, and the reaction mixture was stirred for 12 h at room temperature to generate compound <b>212</b>. Compound <b>214</b> was obtained by the oxidation of benzo[<i>d</i>]thiazole-2-thiol and ethanethiol by using DDQ (2,3-dichloro-5,6-dicyano1,4-benzoquinone) as the oxidizing agent. The synthetic procedures are described in the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>.</div><figure id="sch1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00699/20200604/images/medium/jm0c00699_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00699/20200604/images/large/jm0c00699_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Compounds <b>212</b> and <b>214</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00699/20200604/images/large/jm0c00699_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00699&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a)SO<sub>2</sub>Cl<sub>2</sub>, DCM, 0 °C, 1 h; (b) Et<sub>2</sub>O, room temperature, 12 h; (c) DDQ, DCM, 0 °C, 1 h.</p></p></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Design of Compounds <b>212</b>/<b>214</b></h3><div class="NLM_p">Targeting cysteines is a practical strategy for discovering covalent inhibitors<a onclick="showRef(event, 'ref17 ref29'); return false;" href="javascript:void(0);" class="ref ref17 ref29">(17,29)</a> because of their low occurrence (2.3%) in the human proteome<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> and high selectivity. Thus, several well-established drugs that could covalently modify cysteines were used to explore their alternative functions as novel FBPase inhibitors.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> Amongst them, disulfiram (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00699/suppl_file/jm0c00699_si_001.pdf" class="ext-link">Figure S2</a>) exhibited good biological activity against FBPase (IC<sub>50</sub> = 1.5 μM). The thiazole ring is a very common framework in FDA-approved drugs<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> and disulfiram derivatives,<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> Thus, we used a cyclization strategy to further elucidate the inhibitory mechanism of disulfiram and its derivatives (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). In addition, the synthesis of asymmetric disulfide derivatives (<b>212</b> and <b>214</b>) helped us to effectively verify this covalent reaction mechanism. As illustrated in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>, disulfiram derivatives <b>212</b> and <b>214</b> also potently inhibited FBPase (IC<sub>50</sub> = 0.54 μM for <b>212</b> and 1.6 μM for <b>214</b>).</div><figure id="fig1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00699/20200604/images/medium/jm0c00699_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00699/20200604/images/large/jm0c00699_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Designing strategy of asymmetric disulfide compounds <b>212/214</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00699/20200604/images/large/jm0c00699_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00699&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Determination and Analysis of FBPase Crystal Structures</h3><div class="NLM_p">To elaborate the allosteric inhibitory mechanism of the disulfiram derivatives, three different FBPase structures (APO PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1FTA">1FTA</a>, FBP-bound PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ZWK">5ZWK</a>, <b>214b</b>-bound PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6LS5">6LS5</a>) shown in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a> were considered. As illustrated in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A, S123–I126 (2.9 Å) and N125–C128 (3.0 Å) hydrogen bonds could be observed in the crystal structure of APO FBPase. These two hydrogen bonds contribute to the formation of the 3<sub>10</sub>-helix structure, named helix H4. In addition, a new hydrogen bond between the substrate (FBP) and S123 appeared in the crystal structure of FBP-bound FBPase. Although the hydrogen bonds of S123–I126 (3.3 Å) and N125–C128 (3.5 Å) could also be observed in the crystal structure of FBP-bound FBPase (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B), these two hydrogen bonds were longer than those in APO FBPase. Moreover, the helical content<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> of H4 in the FBP-bound FBPase structure was decreased compared with that of APO FBPase. Both evidence indicate that the hydrogen bond between S123 and FBP could affect the helical content of H4.</div><figure id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00699/20200604/images/medium/jm0c00699_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00699/20200604/images/large/jm0c00699_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Crystal structures of FBPase. (A) Interaction networks between helix H3 (73–88) and helix H4 (123–128) in APO FBPase (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1FTA">1FTA</a>, green). (B) Interaction networks between helix H3 and helix H4 in FBP-bound FBPase (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ZWK">5ZWK</a>, salmon.) (C) Electron density (2Fo-Fc omit map contoured at 0.8σ) for <b>214b</b> bound to C128. (D) Interaction networks between FBPase and <b>214b</b>. (E) Pocket variation from the original C128 site in APO FBPase/FBP-bound FBPase to the new allosteric binding site in <b>214b</b>-bound FBPase. (F) Interaction networks between helix H3 and helix H4 in <b>214b</b>-bound FBPase (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6LS5">6LS5</a>, gray) (G) Conformation comparison among <b>214b</b>-FBPase (gray cartoon, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6LS5">6LS5</a>), FBP-bound FBPase (salmon cartoon, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ZWK">5ZWK</a>), and APO FBPase (green cartoon, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1FTA">1FTA</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00699/20200604/images/large/jm0c00699_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00699&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The structure of the FBPase cocrystallized with <b>214b</b> (a fragment of <b>214</b>) was successfully determined at 2.0 Å resolution by X-ray crystallography to identify the allosteric inhibition mechanism of this novel C128 site for the first time. The crystallographic data are listed in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Crystallographic Data Collection and Refinement Statistics</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">parameter</th><th class="colsep0 rowsep0" align="center">Hu-FBPase-<b>214b</b> (6LS5)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="2" align="left">Data collection<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">space group</td><td class="colsep0 rowsep0" align="left"><i>P</i>2<sub>1</sub>2<sub>1</sub>2<sub>1</sub></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">wavelength</td><td class="colsep0 rowsep0" align="left">1.06</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">cell dimensions</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">a, b, c (Å)</td><td class="colsep0 rowsep0" align="left">67.44, 83.40, 276.48</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">α, β, γ (°)</td><td class="colsep0 rowsep0" align="left">90, 90, 90</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">resolution (Å)</td><td class="colsep0 rowsep0" align="left">42.8–2.03 (2.04–2.03)<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>R</i><sub>merge</sub></td><td class="colsep0 rowsep0" align="left">11.5 (63.2)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>I</i>/σ (I)</td><td class="colsep0 rowsep0" align="left">11.3 (2.3)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">completeness (%)</td><td class="colsep0 rowsep0" align="left">99.6 (98.6)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">redundancy</td><td class="colsep0 rowsep0" align="left">8.1 (7.4)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="2" align="left">Refinement</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">no. reflections</td><td class="colsep0 rowsep0" align="left">101,851</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>R</i><sub>work</sub>/<i>R</i><sub>free</sub></td><td class="colsep0 rowsep0" align="left">19.42/23.73</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">no. atoms</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">protein</td><td class="colsep0 rowsep0" align="left">9909</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ligand/ion</td><td class="colsep0 rowsep0" align="left">99</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">water</td><td class="colsep0 rowsep0" align="left">640</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">all atom</td><td class="colsep0 rowsep0" align="left">10,648</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">B-factors</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">protein</td><td class="colsep0 rowsep0" align="left">32.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ligand/ion</td><td class="colsep0 rowsep0" align="left">41.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">water</td><td class="colsep0 rowsep0" align="left">37.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">all Atom</td><td class="colsep0 rowsep0" align="left">33.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">rms deviations</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">bond lengths (Å)</td><td class="colsep0 rowsep0" align="left">0.011</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">bond angles (°)</td><td class="colsep0 rowsep0" align="left">1.119</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Data were collected from a single crystal.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">Values in parentheses are for the highest-resolution shell.</p></div></div></div><div class="NLM_p">A covalent linkage between C128 and <b>214b</b> could be observed from the structure of <b>214b</b> in complex with FBPase (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>C). <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>D shows that the C128 pocket for <b>214b</b> modification involves residues L129, M172, M185, D197, H253, and V257. As shown in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>E, the addition of <b>214b</b> to C128 results in the movement of C128 residue, the formation of a new allosteric binding site in <b>214b</b>-bound FBPase, and the disappearance of the original C128 site<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> in APO FBPase and FBP-bound FBPase. This is most likely to be the driving force for the deactivation of FBPase. Compared with the crystal structures of APO FBPase (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A) and FBP-bound FBPase (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B), the structure of <b>214b</b>-bound FBPase (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>F) lacks the hydrogen bonds of S123–FBP, S123–I126, and N125–C128, and helix H4 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>F,G) was broken. Instead, a new hydrogen bonding network between S123 and D74 and K71 and water was formed. Simultaneously, the helical content of helix H3 in <b>214b</b>-bound FBPase was increased compared with those in APO FBPase and FBP-bound FBPase (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>G).</div><div class="NLM_p">Furthermore, three FBPase structures (APO, FBP-bound, and <b>214b</b>-bound) were further optimized by MD simulations for 100 ns. As shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A, the 3<sub>10</sub>-helix of H4 in the APO FBPase crystal (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A) was optimized to form a more stable H4 α-helix by MD simulations (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A). The MD structures of FBP-bound FBPase (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B) and <b>214b</b>-bound FBPase (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C) are similar to the crystal structures (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B,F). Notably, there was only one hydrogen bond between S123 and K71 in the MD structure of <b>214b</b>-bound FBPase instead of the hydrogen bonding network in the <b>214b</b>-bound FBPase crystal structure, indicating that the interaction between S123 and K71 was important for the elongation of the new H3 helix (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C,D). The protein dynamics that contribute to allosteric regulation were further investigated, and plots of the distances (S124:O–C128:N), helical contents (helix H4), and population densities of S123-Chi are illustrated in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>E–G, respectively. The magenta line represents the helical content of H4, the distance of S124:O–C128:N, and the population density of S123-Chi in APO FBPase, while the orange line represents those in FBP-bound FBPase, and the blue line represents those in <b>214b</b>-bound FBPase. As shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a> (magenta line), in the APO FBPase structure, the S124:O–C128:N distance was approximately 3 Å (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>E), and the helical content was approximately 3.1 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>F), both indicating a typical α-helical configuration. Moreover, the population density of S123-Chi in both swing directions (gauche<sup>–</sup> and gauche<sup>+</sup>) was approximately 0.5 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>G). However, when FBP was bound to FBPase (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>, orange line), the helical content (120–129) was decreased to 0.9, and the S124:O–C128:N distance was increased to approximately 6 Å. This indicates that the binding of FBP to FBPase results in the partial dissociation of the 120–129 helix compared with that in APO FBPase. As expected, the swing in/out direction and probability of the S123 side chain (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>G, orange line) show that the S123 side chain in FBP-bound FBPase has an almost gauche<sup>+</sup> direction when forming hydrogen bonds with FBP.</div><figure id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00699/20200604/images/medium/jm0c00699_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00699/20200604/images/large/jm0c00699_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Analysis of FBPase structures simulated by 100 ns molecular dynamics (MD). (A) Interaction networks between APO FBPase structure helix H3 and helix H4. (B) Interaction networks between FBP-bound FBPase structure helix H3 and helix H4. (C) Interaction networks between <b>214b</b>-bound FBPase structure helix H3 and helix H4. (D) Conformation comparison among <b>214b</b>-FBPase (blue cartoon), FBP-bound FBPase (orange cartoon), and APO FBPase (magenta cartoon). (E) Distance change (S124:O–C128:N) of three FBPase structures (APO: magenta line, FBP-bound: orange line, and <b>214b-</b>bound: blue line). (F) Helical content (residues 120–129) change of three FBPase structures (APO: magenta line, FBP-bound: orange line, and <b>214b-</b>bound: blue line). (G) S123 side chain population density of three FBPase structures (APO: magenta line, FBP-bound: orange line, and <b>214b-</b>bound: blue line) in molecular dynamics balance.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00699/20200604/images/large/jm0c00699_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00699&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">As shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a> (blue line), the S124:O–C128:N distance was increased to approximately 9 Å (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>E), and the helical content in <b>214b</b>-bound FBPase was reduced to approximately 0.3 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>F). This demonstrates that the covalent addition of <b>214b</b> to C128 results in the dissociation of helix H4 by disrupting the hydrogen bond between S124 and C128. Based on the above evidence, we inferred that the hydrogen bond (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>E) between S124 in position i and C128 in position i + 4 significantly contributes to the formation of helix H4 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>F). Consistent with our crystal structure of <b>214</b>-bound FBPase (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>F), the addition of <b>214b</b> to C128 led to significant repulsion between C128 and the −CONH<sub>2</sub> group of N125; thus, N125 was far from C128, and the S124–C128 hydrogen bond was broken, which in turn led to the disruption of helix H4. As expected, the population density of the S123 side chain increases to approximately 1.0 in the gauche<sup>+</sup> direction but decreases to approximately 0.3 in the gauche<sup>–</sup> direction (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>G), indicating that most S123 is pointed away from FBP and forms hydrogen bonds with K71, D74, and water in helices 69–72 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>F, gray cartoon). Overall, upon addition of <b>214b</b> to C128, N125 is repelled from C128, in turn disrupting the S124–C128 hydrogen bond and helix H4. Helix H4 is very important for the movement of S123, and the dissociation of helix H4 allows S123 to more easily form a new hydrogen bond with K71 in elongated helix H3 (69–72). The loss of the S123–FBP hydrogen bond leads to the inactivity of FBPase. Therefore, N125 and S123 are critical residues for the inhibition of FBPase by disulfiram derivatives.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> LC–MS and Mutation Experiments</h3><div class="NLM_p">Liquid chromatography–mass spectrometry (LC–MS) was performed to verify whether these compounds covalently bind C128 to FBPase. As illustrated in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A, the molecular weight of FBPase is 38,009 Da. Correspondingly, the peaks at 38,129 Da (FBPase + 2×<b>214b</b>) and 38,189 Da (FBPase + 3×<b>214b</b>) in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B match the molecular weight of FBPase covalently bound to two and three <b>214b</b> fragments of <b>214</b>, respectively, which indicates that <b>214b</b> could covalently bind to multiple cysteines in FBPase. The peak of 37,993 Da in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C corresponds to the molecular weight of the C128S mutation (when the C128 residue in FBPase was mutated to serine). Notably, <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>D shows peaks of 38,053 Da (C128S + <b>214b</b>) and 38,113 Da (C128S + 2×<b>214b</b>), which indicate that <b>214b</b> could covalently bind with cysteines in FBPase other than C128. The peaks in the LC–MS spectrum of wild-type FBPase (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B) show that one equivalent of the <b>214b</b> fragment was lost in the peaks of C128S (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>D), which implied that <b>214b</b> was most likely covalently bound to C128. In fact, the crystallographic data mentioned above have already lent a solid credit for this finding. In addition, the possible reaction pathway between FBPase and <b>214</b> is illustrated in <a href="/doi/suppl/10.1021/acs.jmedchem.0c00699/suppl_file/jm0c00699_si_001.pdf" class="ext-link">Figure S3</a>. In addition, the possible reaction mechanism between FBPase and compound <b>212</b> was also predicted by using XO methods at a ωB97X-D/6-31G*//ωB97X-D/6–311-G** theoretical level, as shown in <a href="/doi/suppl/10.1021/acs.jmedchem.0c00699/suppl_file/jm0c00699_si_001.pdf" class="ext-link">Figure S4</a>.</div><figure id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00699/20200604/images/medium/jm0c00699_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00699/20200604/images/large/jm0c00699_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Full-scan electrospray mass spectrum of the parent ion region of <b>214</b> covalent binding of WT (wild-type) FBPase and C128S. (A) Molecular weight of WT FBPase. (B) Mass weight of FBPase + <i>n</i><b>214b</b> (<i>n</i> = 2 and 3). (C) Mass weight of C128S mutation. (D) Mass weight of C128S + <i>n</i><b>214b</b> (<i>n</i> = 1 and 2).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00699/20200604/images/large/jm0c00699_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00699&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To identify the central and important role of cysteine in covalent inhibitory regulation when FBPase was covalently modified by disulfiram and its derivatives, the IC<sub>50</sub> values of these three compounds against cysteine mutants were measured systematically.</div><div class="NLM_p">As shown in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, the IC<sub>50</sub> values of <b>212</b>, <b>214</b>, and disulfiram against the C38S, C92S, C116S, C179S, C183S, and C281S mutants were almost the same as their IC<sub>50</sub> values against the wild-type (WT) protein, and only the C128S mutation resulted in a significant increase (>600-fold) in the IC<sub>50</sub> values compared to that of the WT protein. The above mutant assays and crystallographic data show that C128 plays a unique and crucial role in the covalent allosteric inhibition of FBPase when disulfiram, <b>212</b>, and <b>214</b> covalently bind with FBPase.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. IC<sub>50</sub> of Disulfiram/<b>212/214</b> against Cysteine Mutation and WT FBPase</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"><sub>parameter</sub></th><th class="colsep0 rowsep0" align="center">WT</th><th class="colsep0 rowsep0" align="center">C38S</th><th class="colsep0 rowsep0" align="center">C92S</th><th class="colsep0 rowsep0" align="center">C116S</th><th class="colsep0 rowsep0" align="center">C128S</th><th class="colsep0 rowsep0" align="center">C179S</th><th class="colsep0 rowsep0" align="center">C183S</th><th class="colsep0 rowsep0" align="center">C281S</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>k</i><sub>cat</sub> (s<sup>–1</sup>)</td><td class="colsep0 rowsep0" align="left">2.5 ± 0.1</td><td class="colsep0 rowsep0" align="left">0.80 ± 0.06</td><td class="colsep0 rowsep0" align="left">0.99 ± 0.06</td><td class="colsep0 rowsep0" align="left">0.99 ± 0.12</td><td class="colsep0 rowsep0" align="left">1.7 ± 0.1</td><td class="colsep0 rowsep0" align="left">1.4 ± 0.1</td><td class="colsep0 rowsep0" align="left">2.3 ± 0.1</td><td class="colsep0 rowsep0" align="left">2.2 ± 0.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">disulfiram IC<sub>50</sub> (μM)</td><td class="colsep0 rowsep0" align="left">1.5 ± 0.1</td><td class="colsep0 rowsep0" align="left">0.60 ± 0.03</td><td class="colsep0 rowsep0" align="left">1.2 ± 0.1</td><td class="colsep0 rowsep0" align="left">1.3 ± 0.06</td><td class="colsep0 rowsep0" align="left">>1000<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">1.9 ± 0.2</td><td class="colsep0 rowsep0" align="left">1.5 ± 0.2</td><td class="colsep0 rowsep0" align="left">1.1 ± 0.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">disulfiram (IC<sup>M</sup><sub>50</sub>/IC<sup>W</sup><sub>50</sub>)</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">0.4</td><td class="colsep0 rowsep0" align="left">0.8</td><td class="colsep0 rowsep0" align="left">0.9</td><td class="colsep0 rowsep0" align="left">>667</td><td class="colsep0 rowsep0" align="left">1.3</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">0.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>212</b> IC<sub>50</sub> (μM)</td><td class="colsep0 rowsep0" align="left">0.54 ± 0.06</td><td class="colsep0 rowsep0" align="left">2.1 ± 0.7</td><td class="colsep0 rowsep0" align="left">0.64 ± 0.12</td><td class="colsep0 rowsep0" align="left">0.85 ± 0.08</td><td class="colsep0 rowsep0" align="left">>1000<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">0.67 ± 0.06</td><td class="colsep0 rowsep0" align="left">1.1 ± 0.1</td><td class="colsep0 rowsep0" align="left">1.3 ± 0.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>212</b> (IC<sup>M</sup><sub>50</sub>/IC<sup>W</sup><sub>50</sub>)</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">3.8</td><td class="colsep0 rowsep0" align="left">1.2</td><td class="colsep0 rowsep0" align="left">1.6</td><td class="colsep0 rowsep0" align="left">>1800</td><td class="colsep0 rowsep0" align="left">1.2</td><td class="colsep0 rowsep0" align="left">2.1</td><td class="colsep0 rowsep0" align="left">2.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>214</b> IC<sub>50</sub> (μM)</td><td class="colsep0 rowsep0" align="left">1.6 ± 0.2</td><td class="colsep0 rowsep0" align="left">6.6 ± 1.0</td><td class="colsep0 rowsep0" align="left">0.98 ± 0.26</td><td class="colsep0 rowsep0" align="left">1.9 ± 0.3</td><td class="colsep0 rowsep0" align="left">>1000<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">1.8 ± 0.2</td><td class="colsep0 rowsep0" align="left">1.2 ± 0.3</td><td class="colsep0 rowsep0" align="left">1.5 ± 0.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>214</b> (IC<sup>M</sup><sub>50</sub>/IC<sup>W</sup><sub>50</sub>)</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">4.1</td><td class="colsep0 rowsep0" align="left">0.6</td><td class="colsep0 rowsep0" align="left">1.2</td><td class="colsep0 rowsep0" align="left">>625</td><td class="colsep0 rowsep0" align="left">1.1</td><td class="colsep0 rowsep0" align="left">0.75</td><td class="colsep0 rowsep0" align="left">0.93</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">1000 μM inhibition rate is less than 50%</p></div></div></div><div class="NLM_p last">In addition, a series of Ala mutation assays of residues S123, S124, and N125 in helix H4 were carried out, and the corresponding IC<sub>50</sub> values of disulfiram, <b>212</b>, and <b>214</b> against these mutants were also determined. The enzymatic activities of S123A, S124A, and N125A were determined as described in our previous report.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> Compared with the WT (<i>k</i><sub>cat</sub> = 2.5 s<sup>–1</sup>), S124A (<i>k</i><sub>cat</sub> = 3.1 s<sup>–1</sup>) and N125A (<i>k</i><sub>cat</sub> = 0.93 s<sup>–1</sup>) retained comparable catalytic activities, while S123A (<i>k</i><sub>cat</sub> = 0.04 s<sup>–1</sup>) lost almost all of its catalytic activity.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> Thus, these results demonstrated that among the above three residues, only S123 was crucial for the catalytic activity of the enzyme, which was consistent with the crystallographic structure of FBP-bound (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ZWK">5ZWK</a>), showing a direct interaction between S123 and the substrate. As listed in <a href="/doi/suppl/10.1021/acs.jmedchem.0c00699/suppl_file/jm0c00699_si_001.pdf" class="ext-link">Table S1</a>, the IC<sub>50</sub> values of disulfiram, <b>212</b>, and <b>214</b> against the S124A mutant of FBPase showed almost no changes compared to those of WT FBPase; however, the N125A mutation led to 320-, 900-, and 313-fold increases in the IC<sub>50</sub> values of disulfiram, <b>212</b>, and <b>214</b>, respectively, indicating that N125 not S124 played an important role in the covalent allosteric inhibitory regulation of helix H4 when C128 was covalently bound with these disulfiram derivatives. Notably, the results of this mutation assay are consistent with the abovementioned MD results and helpful for further understanding the inhibitory regulation mechanism of new covalent allosteric binding sites through the dissociation of helix H4 in FBPase.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Glucose Reduction in ICR and <i>db</i>/<i>db</i> Mice</h3><div class="NLM_p">Considering the better FBPase inhibitory activities of disulfiram and <b>212</b>, it is imperative to further evaluate the hypoglycemic efficacies of these two compounds in ICR and <i>db/db</i> mice. As shown in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A, in an oral administration model, disulfiram at doses of 100 mg/kg significantly reduced blood glucose at 4 h (<i>P</i> < 0.01 vs vehicle), and the area under the curve (AUC)<sub>0-8h</sub> of the 100 mg/kg disulfiram-treated group was reduced by 12.8% compared to that of the vehicle group (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B). However, compound <b>212</b> at oral doses of 100 mg/kg failed to display a hypoglycemic effect, most likely due to its poor oral bioavailability. Interestingly, we found that compound <b>212</b> at an intraperitoneal dose of 50 mg/kg could remarkably reduce blood glucose in mice during 2–6 h compared to the vehicle group at 50 mg/kg (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>C). AUC<sub>0-8h</sub> analysis suggested that the 50 mg/kg <b>212</b>-treated group exhibited a 36.3% reduction in blood glucose (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>D). Unfortunately, disulfiram cannot be administered intraperitoneally due to its poor water solubility. Therefore, <b>212</b> was selected for further pharmacodynamics studies. To better evaluate glucose tolerance in response to treatment with <b>212</b>, an oral glucose tolerance test (OGTT) was carried out in ICR mice. Glucose challenge (2 g/kg) dramatically increased the blood glucose levels in the vehicle group, whereas the <b>212</b>-administered groups exhibited much smaller increases in blood glucose, especially at 30 min after glucose administration (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>E). The AUC<sub>0-2h</sub> analysis suggested that 50 mg/kg <b>212</b> observably increased glucose tolerance by reducing the AUC<sub>0-2h</sub> by 61.7% relative to that of the vehicle group, and the pharmaceutical effect was superior to that of 25 mg/kg <b>212</b> (ΔAUC<sub>0–2h</sub> = 19.8%) (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>F). These results clearly demonstrated that disulfiram and <b>212</b> have hypoglycemic effects in ICR mice.</div><figure id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00699/20200604/images/medium/jm0c00699_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00699/20200604/images/large/jm0c00699_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Effect of single administration of disulfiram or compound <b>212</b> on blood glucose in 7–9 week old ICR or <i>db/db</i> mice (<i>n</i> = 5–6 in each group). (A) Blood glucose changes and (B) AUC of blood glucose between 0 and 8 h in 12 h-fasted ICR mice after oral administration of disulfiram and compound <b>212</b>. (C) Blood glucose changes and (D) AUC of blood glucose between 0 and 8 h in 12 h-fasted ICR mice after intraperitoneal administration of compound <b>212</b>. (E) Blood glucose responses to an oral glucose challenge (2 g/kg) and (F) AUC of blood glucose between 0 and 2 h in 12 h-fasted ICR mice. Compound <b>212</b> was intraperitoneally administered 1.5 h prior to oral glucose challenge. (G) Blood glucose changes and (H) AUC of blood glucose between 0 and 6 h in freely feeding <i>db/db</i> mice after intraperitoneal administration of compound <b>212</b>. (I) Blood glucose responses to an oral glucose challenge (2 g/kg) and (J) AUC of blood glucose between 0 and 2 h in 12 h-fasted <i>db/db</i> mice. Compound <b>212</b> was administered 1.5 h prior to oral glucose challenge. Data are presented as the mean ± SEM (*<i>P</i> < 0.05, **<i>P</i> < 0.01, ***<i>P</i> < 0.001 vs vehicle; student’s <i>t</i> test).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00699/20200604/images/large/jm0c00699_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00699&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">As an important pathway for glucose homeostasis, GNG can convert lactate and other carbon substrates into glucose (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00699/suppl_file/jm0c00699_si_001.pdf" class="ext-link">Figure S5A</a>). Therefore, the inhibition of FBPase likely blocks the GNG pathway and leads to decreased glucose and lactate concentrations. To preliminarily determine whether compound <b>212</b> could inhibit FBPase <i>in vivo</i>, we measured several parameters in ICR mice, including the concentrations of blood glucose, blood lactate, and the FBPase activity. As illustrated in <a href="/doi/suppl/10.1021/acs.jmedchem.0c00699/suppl_file/jm0c00699_si_001.pdf" class="ext-link">Figure S5B</a>, compound <b>212</b> at a dose of 50 mg/kg had hypoglycemic effects (<i>P</i> < 0.05 vs vehicle) 2 h after ip administration. At the same time, the blood lactate concentration of the <b>212</b>-treated group (4.1 mM) was elevated relative to that of the vehicle group (2.0 mM) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00699/suppl_file/jm0c00699_si_001.pdf" class="ext-link">Figure S5C</a>), but lactic acidosis was not observed (>5 mM<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a>). Moreover, <b>212</b> significantly inhibited the activity of FBPase at 50 mg/kg (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00699/suppl_file/jm0c00699_si_001.pdf" class="ext-link">Figure S5D</a>). These results further indicate that compound <b>212</b> could block GNG by inhibiting FBPase.</div><div class="NLM_p last">In light of the favorable hypoglycemic effect of <b>212</b> in ICR mice, the ability of <b>212</b> to lower glucose was further evaluated in diabetic <i>db/db</i> mice. Low doses (25 and 50 mg/kg) of <b>212</b> had hypoglycemic effects at 1–4 h compared to the vehicle (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>G) and remarkably reduced blood glucose AUC<sub>0-6h</sub> by 21.7% and 25.5%, respectively (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>H). An OGTT was also performed to evaluate the glucose tolerance ability of <b>212</b>. AUC<sub>0-2h</sub> analysis suggested that compound <b>212</b> at a dose of 25 mg/kg observably increased glucose tolerance by reducing the AUC<sub>0-2h</sub> by 21.8% relative to the vehicle, which was similar to the effect of <b>212</b> at 50 mg/kg (ΔAUC<sub>0–2h</sub> = 19.7%) (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>I,J). These experiments suggested that <b>212</b> could also improve glucose homeostasis in diabetic <i>db/db</i> mice.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i12">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_88265" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_88265" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">In this study, a novel covalent allosteric binding site, which is distinct from the original C128 site in APO FBPase and not exploited by previous works, was discovered based on the cocrystal structure of FBPase-<b>214b</b>. Disulfiram, a well-known anti-alcoholism drug, and its derivatives (<b>212</b> and <b>214)</b> were found to potently inhibit FBPase by covalently binding to this new C128 site. Crystallographic and MD studies revealed that the addition of <b>214b</b> to C128 would repel N125 away from C128 and disrupt the S124–C128 hydrogen bond and helix H4 (123–128). The dissociation of helix H4 allows S123 to more easily move away from FBP and form a new hydrogen bond with K71, which in turn results in the extension of helix H3 (69–72). The hydrogen bond of S123–FBP is essential for the catalytic activity of FBPase, and the addition of <b>214b</b> to C128 could disrupt the S123–FBP hydrogen bond, thereby inhibiting FBPase. To the best of our knowledge, this is the first report that disulfiram or its derivatives act as covalent allosteric modulators of FBPase and exhibit hypoglycemic effects. In particular, disulfiram and <b>212</b> moderately reduced blood glucose in an ICR mouse model and a <i>db/db</i> mouse model. Our current findings identify a new covalent allosteric binding site of FBPase and establish a structural foundation for the discovery of new covalent allosteric drugs that inhibit FBPase.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i13">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_84312" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_84312" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Chemistry</h3><div class="NLM_p">The synthetic starting materials, reagents, and solvents were obtained from a commercial supplier, such as Energy Chemical, J&K, and TCI, at the highest commercial quality and were used without further purification. The reaction progress was monitored using an analytical thin-layer chromatograph (TLC) (HSGF 254; 150–200 μm thickness; Yantai Huiyou Co., China); UV light (254 nm) and I2 were used to visualize the components. <sup>1</sup>H NMR and <sup>13</sup>C NMR spectra were recorded on Bruker AMX 400 and AMX 600 spectrometers in DMSO-<i>d</i><sub>6</sub> or CDCl<sub>3</sub> with TMS (tetramethylsilane) as an internal standard. Chemical shifts are reported in parts per million (ppm, δ) downfield from TMS. Proton coupling patterns are described as singlet (s), doublet (d), triplet (t), quartet (q), and multiplet (m). High-resolution mass spectrometry (HRMS) data were obtained by electron ionization (EI) using a Waters GCT Premier. All final compounds were purified to >95% purity, as determined by high-performance liquid chromatography (HPLC) on an Agilent 1100 instrument with a quaternary pump and a diode array detector (DAD), MeOH:H<sub>2</sub>O = 95:5 (v/v) as the eluent, and a flow rate of 0.5 mL/min.</div><div id="sec4_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i15" class="anchor-spacer"></div><h4 class="article-section__title" id="_i15"> 2-((4-Chlorophenyl)disulfanyl)thiazole (<b>212</b>)</h4><div class="NLM_p last">To a stirred solution of 4-chlorobenzenethiol (1.4 g, 10 mmol) in 20 mL of anhydrous dichloromethane was added dropwise sulfuryl chloride (810 μL, 10 mmol) at 0 °C. The reaction mixture was stirred for 1 h at the same temperature and then concentrated under reduced pressure to afford 4-chlorophenyl hypochlorothioite as a brownish yellow liquid, which was used in the next step without purification. A solution of freshly prepared 4-chlorophenyl hypochlorothioite (5 mmol) in 10 mL of anhydrous ethyl ether was added dropwise to a solution of thiazole-2-thiol (5 mmol) in 10 mL of anhydrous ethyl ether at room temperature. The reaction mixture was stirred for 12 h, and removal of the solvents produced a liquid that was purified using column chromatography to afford compound <b>212</b>. Yellow liquid. 75% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.73 (d, <i>J</i> = 3.3 Hz, 1H), 7.57–7.50 (m, 2H), 7.34–7.29 (m, 3H). <sup>13</sup>C NMR (151 MHz, DMSO) δ 144.93, 130.91, 130.03, 129.92, 129.62, 124.03. HRMS (EI): <i>m</i>/<i>z</i> calcd C<sub>9</sub>H<sub>6</sub>ClNS<sub>3</sub> (M<sup>+</sup>), 258.9351; found, 258.9352. HPLC purity: 99.2%.</div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i16" class="anchor-spacer"></div><h4 class="article-section__title" id="_i16"> 2-(Ethyldisulfanyl)benzo[<i>d</i>]thiazole (<b>214</b>)</h4><div class="NLM_p last">2-Mercaptobenzothiazole (1.0 mmol) and ethanethiol (1.5 mmol) were dissolved in 5 mL of DCM (dichloromethane) at 0 °C. DDQ (1.0 mmol) was then slowly added in portions to the solution, and the reaction mixture was stirred for 0.5–1 h on an ice bath. Upon completion of the reaction, the solvents were evaporated under vacuum, and the crude compound was purified via column chromatography to give compound <b>214</b>. Colorless oil. 85% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.87 (d, <i>J</i> = 8.1 Hz, 1H), 7.80 (dd, <i>J</i> = 8.0, 0.5 Hz, 1H), 7.47–7.40 (m, 1H), 7.36–7.30 (m, 1H), 2.98 (q, <i>J</i> = 7.3 Hz, 2H), 1.42 (t, <i>J</i> = 7.3 Hz, 3H). <sup>13</sup>C NMR (150 MHz, DMSO-<i>d</i><sub>6</sub>) δ 172.79, 155.13, 135.72, 126.99, 125.25, 122.46, 122.18, 33.26, 14.62. HRMS (EI) <i>m</i>/<i>z</i> calcd C<sub>9</sub>H<sub>9</sub>NS<sub>3</sub> [M]<sup>+</sup>, 226.9897; found, 226.9894. HPLC purity: 99.9%.</div></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Drugs and Chemicals</h3><div class="NLM_p last">Mercaptopurine, furazolidone, disulfiram, carmofur, lipoic acid, afatinib, disodium phosphonomycin, eflornithine, warfarin, and phenoxybenzamine were obtained from Sigma-Aldrich.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Enzyme Activity Assays and Mutagenesis</h3><div class="NLM_p">Full-length WT human liver FBPase (GenBank: <a href="https://www.ncbi.nlm.nih.gov/gquery?Term=D26055.1">D26055.1</a>) was expressed in <i>E. coli</i> BL21 (DE3) and purified with a HisTrap_FF_5 mL [Global] column following the standard ÄKTA pure system, and this process was performed as described in our previous work.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> Enzymatic reactions were performed with purified enzyme (11.2 μg/mL) and FBP (0.4 mM/L) in 30 μL of buffer (50 mM/L Tris, 0.8 mM/L Mg<sup>2+</sup>). After reacting for 5 min at 37 °C, 15 μL of 1 M perchloric acid was used to stop the reaction, and then malachite green (0.35% polyvinyl alcohol and 0.0035% malachite green) was added to quantify the inorganic phosphate. The A620 was measured after these additions.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> The kinetic constant values or half-maximal inhibitory concentrations (IC<sub>50</sub>) were identified using logistic equations (nonlinear analysis) or Hill kinetics equations in the Origin 7.5 program package.</div><div class="NLM_p last">Mutations were performed by introducing specific base changes into a double-stranded DNA plasmid. The mutant constructs were generated using the two-step PCR method. DNA encoding WT human liver FBPase cloned into pET-21b was used as a template for mutagenesis. The parental methylated and hemimethylated DNA were digested by Nde I and Xho l restriction enzymes. Then, the mutant constructs were ligated into a previous plasmid. The plasmids of the recombinant mutant were transformed into DH5α competent cells. All of the mutations were confirmed by DNA sequencing. The verified plasmids with mutations were transformed into strain <i>E. coli</i> BL21 (DE3) cells. The mutant FBPase proteins were purified in the same manner as WT FBPase.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Crystallization, Data Collection, and Structure Determination</h3><div class="NLM_p">APO FBPase crystals were obtained after 5–7 days at 18 °C by the hanging drop vapor diffusion method against 500 μL of well solution using 24 well crystallization plates. The crystallization drops contained 1 μL of protein solution (10 mg/mL FBPase, 0.1 M NaCl, 1 mM AMP, and 10 mM Mg<sup>2+</sup>) mixed with 1 μL of reservoir solution containing 0.05 M Tris (pH 6.8) and 10% (v/v) polyethylene glycol 8000. <b>214</b> was soaked into a drop of APO FBPase crystals by adding 0.2 μL of the compound (5 mM) in the same reservoir solution for 10 h.</div><div class="NLM_p last">Crystals were cryoprotected with 20% (v/v) glycerol and flash-cooled in liquid nitrogen. The data were acquired at the Shanghai Synchrotron Radiation Facility on beamline BL19U. The data were processed with XDS packages.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> Furthermore, the structure was solved by molecular replacement using the CCP4<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> program, and a previous human FBPase structure (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ZWK">5ZWK</a>) was used as the search model. Model analysis, building, and refinement were performed with the Coot<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> and Phenix<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> programs.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Animals</h3><div class="NLM_p last">Male ICR mice (6–7 weeks) and male BKS. Cg-+Leprdb/+Leprdb/JclSlac mice (6–8 weeks) were purchased from Shanghai JSJ Lab Animal, Ltd. and Shanghai SLAC Laboratory Animal Co., respectively. The animals were housed under specific pathogen-free conditions with a 12 h light–dark cycle at a temperature of 25 °C, humidity of 55%, and free access to water and food. The animal study was approved by the Animal Care and Use Committee of Central China Normal University.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Glucose Reduction in ICR Mice</h3><div class="NLM_p last">ICR mice that had been fasted for 12 h (<i>n</i> = 6 in each group) were administered vehicle (10% ricinus oil in water), <b>212</b> (100 mg/kg, po), disulfiram (100 mg/kg, po), or compound <b>212</b> (25 mg/kg and 50 mg/kg, ip). Food was withheld throughout the study. Blood samples were obtained from the tail vein at 0, 2, 4, 6, and 8 h and were analyzed using a glucometer (Sanicare).</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Glucose Reduction in <i>db/db</i> Mice</h3><div class="NLM_p last">Freely fed <i>db/db</i> mice (<i>n</i> = 6 in each group) were intraperitoneally administered compound <b>212</b> (25 mg/kg and 50 mg/kg), metformin (250 mg/kg), or vehicle (10% ricinus oil in water). Food was withheld after the administration of the test compounds. Blood samples were obtained from the tail vein at 0, 2, 4, and 6 h and were analyzed using a glucometer (Sanicare).</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> OGTT in ICR and <i>db/db</i> Mice</h3><div class="NLM_p last">ICR or <i>db/db</i> mice that had been fasted for 12 h (<i>n</i> = 6 in each group) were intraperitoneally administered compound <b>212</b> (25 and 50 mg/kg), metformin (250 mg/kg), or vehicle (10% ricinus oil in water) 1.5 h before glucose challenge. Glucose (2 g/kg) was orally administered at 0 h, and blood samples were drawn from the tail vein at 0, 0.5, 1, 2, 3, and 4.5 h after glucose administration. Plasma glucose was measured using a glucometer (Sanicare). Food was withheld throughout the study.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> Inhibition of FBPase <i>ex Vivo</i></h3><div class="NLM_p last">ICR mice received the test compound via ip administration and were euthanized by decapitation after 2 h. The liver was removed, weighed, and homogenized in double distilled water. Blood samples were obtained from the tail vein at 0 and 2 h. The activities of liver FBPase were tested as previously described.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Blood glucose was measured using a glucometer, and blood lactate was measured using a Lactic Acid LD Assay Kit (Nanjing Jiancheng Bioengineering Institute).</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> Mass Spectrometry</h3><div class="NLM_p">WT FBPase (50 μM) or C128S FBPase (50 μM) was incubated with 500 μM disulfiram and <b>212</b> for 2 h. Then, the samples were analyzed using an Agilent 6550 quadrupole-time-of-flight (QTOF) mass spectrometer (Santa Clara, CA) coupled with an Agilent 1260 high-performance liquid chromatograph (HPLC; Santa Clara, CA).</div><div class="NLM_p last">The samples were separated on a Phenomenex Jupiter C4 300 Å LC column (2 × 150 mm, 5 μm) over 15 min using a gradient from 5 to 100% acetonitrile containing 0.1% formic acid at a flow rate of 0.5 mL/min. Data were acquired in positive-ion mode with a Dual Agilent Jet Stream electrospray voltage using a capillary temperature of 250 °C, a fragmentor voltage of 175 V, a capillary voltage of 3000 V, and a <i>m</i>/<i>z</i> 600–3200 mass window. Mass deconvolution was then performed using Agilent Mass Hunter Qualitative Analysis B.06.00 software.</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> Molecular Dynamics Simulations</h3><div class="NLM_p last">The crystal structures of the APO state (1FTA), FBP-bound state (5ZWK), and FBP/<b>214b</b>-bound state obtained in this study were used as templates to set up the system for MD simulations. The CHARMM<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> and CGENFF<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> force fields were used for the protein and the ligands (including FBP and <b>214b</b>), respectively. The whole system was surrounded by TIP3P water molecules<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> in a truncated octahedral box 10 Å from any solute atoms. The system was neutralized with K<sup>+</sup> counterions. MD simulations of 100 ns were performed at 300 K. LINCS<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> was used to constrain bonds involving hydrogen atoms, and the time step was 2.0 fs. The nonbonded cutoff was set to 10 Å. The long-range electrostatics were calculated by the particle mesh Ewald (PME) algorithm.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> Parallel simulations were performed simultaneously using GROMACS 5.1.4.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> To characterize the changes in the secondary structures at helix E, <i>i.e</i>., residues 120–129, the collective variable ALPHARMSD<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> was used to probe the number of six-residue segments in the α-helical configuration.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes2"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i27"><a href="/doi/suppl/10.1021/acs.jmedchem.0c00699" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_72184" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_72184" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c00699?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00699</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Molecular dynamic results (PDB); <sup>1</sup>H NMR, <sup>13</sup>C NMR, and high-resolution mass spectra (HRMS) data; HPLC spectra data for <b>212</b>/<b>214</b>; and WT and mutant enzyme data and proposed reaction mechanism calculation for <b>212</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00699/suppl_file/jm0c00699_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00699/suppl_file/jm0c00699_si_002.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">Molecular dynamic result for <b>214</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00699/suppl_file/jm0c00699_si_003.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Molecular dynamic result for APO (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00699/suppl_file/jm0c00699_si_004.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Molecular dynamic result for FBP (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00699/suppl_file/jm0c00699_si_005.pdb" class="ext-link">PDB</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00699/suppl_file/jm0c00699_si_001.pdf">jm0c00699_si_001.pdf (832.16 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00699/suppl_file/jm0c00699_si_002.csv">jm0c00699_si_002.csv (0.84 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00699/suppl_file/jm0c00699_si_003.pdb">jm0c00699_si_003.pdb (779.7 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00699/suppl_file/jm0c00699_si_004.pdb">jm0c00699_si_004.pdb (819.21 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00699/suppl_file/jm0c00699_si_005.pdb">jm0c00699_si_005.pdb (823.16 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The atomic coordinates and structure factors have been deposited into the RCSB Protein Data Bank with accession numbers 6LS5. Authors will release the atomic coordinates and experimental data upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.0c00699" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_63725" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_63725" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yanliang Ren</span> - <span class="hlFld-Affiliation affiliation">Key
Laboratory of Pesticide & Chemical Biology (CCNU), Ministry of
Education, College of Chemistry, Central
China Normal University, Wuhan 430079, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-8565-2152" title="Orcid link">http://orcid.org/0000-0001-8565-2152</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#681a0d0611042805090104460b0b061d460d0c1d460b06"><span class="__cf_email__" data-cfemail="acdec9c2d5c0ecc1cdc5c082cfcfc2d982c9c8d982cfc2">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jian Li</span> - <span class="hlFld-Affiliation affiliation">Shanghai
Key Laboratory of New Drug Design, School of Pharmacy, East China University of Science and Technology, Shanghai 200237, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-7521-8798" title="Orcid link">http://orcid.org/0000-0002-7521-8798</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#02686b636c6e6b4267617771762c6766772c616c"><span class="__cf_email__" data-cfemail="39535058575550795c5a4c4a4d175c5d4c175a57">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jian Wan</span> - <span class="hlFld-Affiliation affiliation">Key
Laboratory of Pesticide & Chemical Biology (CCNU), Ministry of
Education, College of Chemistry, Central
China Normal University, Wuhan 430079, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-4172-1392" title="Orcid link">http://orcid.org/0000-0003-4172-1392</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#a5cfccc4cbd2c4cbe5c8c4ccc98bc6c6cbd08bc0c1d08bc6cb"><span class="__cf_email__" data-cfemail="88e2e1e9e6ffe9e6c8e5e9e1e4a6ebebe6fda6edecfda6ebe6">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yunyuan Huang</span> - <span class="hlFld-Affiliation affiliation">Key
Laboratory of Pesticide & Chemical Biology (CCNU), Ministry of
Education, College of Chemistry, Central
China Normal University, Wuhan 430079, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yixiang Xu</span> - <span class="hlFld-Affiliation affiliation">Shanghai
Key Laboratory of New Drug Design, School of Pharmacy, East China University of Science and Technology, Shanghai 200237, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rongrong Song</span> - <span class="hlFld-Affiliation affiliation">Key
Laboratory of Pesticide & Chemical Biology (CCNU), Ministry of
Education, College of Chemistry, Central
China Normal University, Wuhan 430079, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Shuaishuai Ni</span> - <span class="hlFld-Affiliation affiliation">Cancer
Institute, Longhua Hospital, Shanghai University
of Traditional Chinese Medicine, Shanghai 200237, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jiaqi Liu</span> - <span class="hlFld-Affiliation affiliation">Key
Laboratory of Pesticide & Chemical Biology (CCNU), Ministry of
Education, College of Chemistry, Central
China Normal University, Wuhan 430079, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yanhong Xu</span> - <span class="hlFld-Affiliation affiliation">Key
Laboratory of Pesticide & Chemical Biology (CCNU), Ministry of
Education, College of Chemistry, Central
China Normal University, Wuhan 430079, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Li Rao</span> - <span class="hlFld-Affiliation affiliation">Key
Laboratory of Pesticide & Chemical Biology (CCNU), Ministry of
Education, College of Chemistry, Central
China Normal University, Wuhan 430079, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yingjie Wang</span> - <span class="hlFld-Affiliation affiliation">Shenzhen
Bay Laboratory, Shenzhen 518055, Guangdong, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9800-8163" title="Orcid link">http://orcid.org/0000-0001-9800-8163</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lin Wei</span> - <span class="hlFld-Affiliation affiliation">Key
Laboratory of Pesticide & Chemical Biology (CCNU), Ministry of
Education, College of Chemistry, Central
China Normal University, Wuhan 430079, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lingling Feng</span> - <span class="hlFld-Affiliation affiliation">Key
Laboratory of Pesticide & Chemical Biology (CCNU), Ministry of
Education, College of Chemistry, Central
China Normal University, Wuhan 430079, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-1260-4656" title="Orcid link">http://orcid.org/0000-0003-1260-4656</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chen Su</span> - <span class="hlFld-Affiliation affiliation">National
Facility for Protein Science in Shanghai, Zhangjiang Lab, Shanghai 201210, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chao Peng</span> - <span class="hlFld-Affiliation affiliation">National
Facility for Protein Science in Shanghai, Zhangjiang Lab, Shanghai 201210, China</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>Y.H., Y.X., R.S., and S.N. contributed equally to this work.</p><span class="author-information-subsection-header">Author Contributions</span><p>All authors contributed to the writing of the manuscript and have approved the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i31">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_16008" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_16008" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was supported by the Natural Science Foundation of China (nos. 21877046, 21873035, 21572077, and 21472061), the Program for the PCSIRT (no. IRT0953), and the self-determined research funds of CCNU from the colleges’ basic research and operation of MOE (nos. CCNU19TS011, CCNU18TS010, 2018YBZZ019, and CCNU16A02041). The support from the Program of Introducing Talents of Discipline to Universities of China (111 Program, B17019) is also appreciated. We thank the staff of the BL19U1 beamline of the NCPSS at the Shanghai Synchrotron Radiation Facility for assistance during data collection.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">FBPase</td><td class="NLM_def"><p class="first last">fructose-1,6-bisphosphatase</p></td></tr><tr><td class="NLM_term">FBP</td><td class="NLM_def"><p class="first last">fructose-1,6-biphosphate</p></td></tr><tr><td class="NLM_term">F6P</td><td class="NLM_def"><p class="first last">fructose-6-biphosphate</p></td></tr><tr><td class="NLM_term">IC<sub>50</sub></td><td class="NLM_def"><p class="first last">half maximal inhibitory concentration</p></td></tr><tr><td class="NLM_term">T2D</td><td class="NLM_def"><p class="first last">type 2 diabetes</p></td></tr><tr><td class="NLM_term">AMP</td><td class="NLM_def"><p class="first last">adenosine monophosphate</p></td></tr><tr><td class="NLM_term">ICR</td><td class="NLM_def"><p class="first last">Institute of Cancer Research</p></td></tr><tr><td class="NLM_term">OGTT</td><td class="NLM_def"><p class="first last">oral glucose tolerance test</p></td></tr><tr><td class="NLM_term">LC–MS</td><td class="NLM_def"><p class="first last">liquid chromatography–mass spectrometry</p></td></tr><tr><td class="NLM_term">GNG</td><td class="NLM_def"><p class="first last">gluconeogenesis</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i33">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_02670" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_02670" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 44 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hunter, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughey, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lantier, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sundelin, E. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peggie, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeqiraj, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sicheri, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jessen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wasserman, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakamoto, K.</span></span> <span> </span><span class="NLM_article-title">Metformin reduces liver glucose production by inhibition of fructose-1-6-bisphosphatase</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">1395</span>– <span class="NLM_lpage">1406</span>, <span class="refDoi"> DOI: 10.1038/s41591-018-0159-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=10.1038%2Fs41591-018-0159-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=30150719" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsF2nsrvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2018&pages=1395-1406&author=R.+W.+Hunterauthor=C.+C.+Hugheyauthor=L.+Lantierauthor=E.+I.+Sundelinauthor=M.+Peggieauthor=E.+Zeqirajauthor=F.+Sicheriauthor=N.+Jessenauthor=D.+H.+Wassermanauthor=K.+Sakamoto&title=Metformin+reduces+liver+glucose+production+by+inhibition+of+fructose-1-6-bisphosphatase&doi=10.1038%2Fs41591-018-0159-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Metformin reduces liver glucose production by inhibition of fructose-1-6-bisphosphatase</span></div><div class="casAuthors">Hunter, Roger W.; Hughey, Curtis C.; Lantier, Louise; Sundelin, Elias I.; Peggie, Mark; Zeqiraj, Elton; Sicheri, Frank; Jessen, Niels; Wasserman, David H.; Sakamoto, Kei</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1395-1406</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Metformin is a first-line drug for the treatment of individuals with type 2 diabetes, yet its precise mechanism of action remains unclear.  Metformin exerts its antihyperglycemic action primarily through lowering hepatic glucose prodn. (HGP).  This suppression is thought to be mediated through inhibition of mitochondrial respiratory complex I, and thus elevation of 5'-adenosine monophosphate (AMP) levels and the activation of AMP-activated protein kinase (AMPK), though this proposition has been challenged given results in mice lacking hepatic AMPK.  Here we report that the AMP-inhibited enzyme fructose-1,6-bisphosphatase-1 (FBP1), a rate-controlling enzyme in gluconeogenesis, functions as a major contributor to the therapeutic action of metformin.  We identified a point mutation in FBP1 that renders it insensitive to AMP while sparing regulation by fructose-2,6-bisphosphate (F-2,6-P2), and knock-in (KI) of this mutant in mice significantly reduces their response to metformin treatment.  We observe this during a metformin tolerance test and in a metformin-euglycemic clamp that we have developed.  The antihyperglycemic effect of metformin in high-fat diet-fed diabetic FBP1-KI mice was also significantly blunted compared to wild-type controls.  Collectively, we show a new mechanism of action for metformin and provide further evidence that mol. targeting of FBP1 can have antihyperglycemic effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHU4MMYmFinrVg90H21EOLACvtfcHk0lglnQynMHP0wQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsF2nsrvJ&md5=c1696431e641d352c2c2619862d6e4bb</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fs41591-018-0159-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41591-018-0159-7%26sid%3Dliteratum%253Aachs%26aulast%3DHunter%26aufirst%3DR.%2BW.%26aulast%3DHughey%26aufirst%3DC.%2BC.%26aulast%3DLantier%26aufirst%3DL.%26aulast%3DSundelin%26aufirst%3DE.%2BI.%26aulast%3DPeggie%26aufirst%3DM.%26aulast%3DZeqiraj%26aufirst%3DE.%26aulast%3DSicheri%26aufirst%3DF.%26aulast%3DJessen%26aufirst%3DN.%26aulast%3DWasserman%26aufirst%3DD.%2BH.%26aulast%3DSakamoto%26aufirst%3DK.%26atitle%3DMetformin%2520reduces%2520liver%2520glucose%2520production%2520by%2520inhibition%2520of%2520fructose-1-6-bisphosphatase%26jtitle%3DNat.%2520Med.%26date%3D2018%26volume%3D24%26spage%3D1395%26epage%3D1406%26doi%3D10.1038%2Fs41591-018-0159-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, D. S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walton, Z. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ochocki, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathew, L. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mancuso, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gade, T. P. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keith, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nissim, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simon, M. C.</span></span> <span> </span><span class="NLM_article-title">Fructose-1,6-bisphosphatase opposes renal carcinoma progression</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>513</i></span>,  <span class="NLM_fpage">251</span>– <span class="NLM_lpage">255</span>, <span class="refDoi"> DOI: 10.1038/nature13557</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=10.1038%2Fnature13557" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=25043030" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFOltrjK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=513&publication_year=2014&pages=251-255&author=B.+Liauthor=B.+Qiuauthor=D.+S.+M.+Leeauthor=Z.+E.+Waltonauthor=J.+D.+Ochockiauthor=L.+K.+Mathewauthor=A.+Mancusoauthor=T.+P.+F.+Gadeauthor=B.+Keithauthor=I.+Nissimauthor=M.+C.+Simon&title=Fructose-1%2C6-bisphosphatase+opposes+renal+carcinoma+progression&doi=10.1038%2Fnature13557"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Fructose-1,6-bisphosphatase opposes renal carcinoma progression</span></div><div class="casAuthors">Li, Bo; Qiu, Bo; Lee, David S. M.; Walton, Zandra E.; Ochocki, Joshua D.; Mathew, Lijoy K.; Mancuso, Anthony; Gade, Terence P. F.; Keith, Brian; Nissim, Itzhak; Simon, M. Celeste</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">513</span>
        (<span class="NLM_cas:issue">7517</span>),
    <span class="NLM_cas:pages">251-255</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Clear cell renal cell carcinoma (ccRCC), the most common form of kidney cancer, is characterized by elevated glycogen levels and fat deposition.  These consistent metabolic alterations are assocd. with normoxic stabilization of hypoxia-inducible factors (HIFs) secondary to von Hippel-Lindau (VHL) mutations that occur in over 90% of ccRCC tumors.  However, kidney-specific VHL deletion in mice fails to elicit ccRCC-specific metabolic phenotypes and tumor formation, suggesting that addnl. mechanisms are essential.  Recent large-scale sequencing analyses revealed the loss of several chromatin remodelling enzymes in a subset of ccRCC (these included polybromo-1, SET domain contg. 2 and BRCA1-assocd. protein-1, among others), indicating that epigenetic perturbations are probably important contributors to the natural history of this disease.  Here we used an integrative approach comprising pan-metabolomic profiling and metabolic gene set anal. and detd. that the gluconeogenic enzyme fructose-1,6-bisphosphatase 1 (FBP1) is uniformly depleted in over six hundred ccRCC tumors examd.  Notably, the human FBP1 locus resides on chromosome 9q22, the loss of which is assocd. with poor prognosis for ccRCC patients.  Our data further indicate that FBP1 inhibits ccRCC progression through two distinct mechanisms.  First, FBP1 antagonizes glycolytic flux in renal tubular epithelial cells, the presumptive ccRCC cell of origin, thereby inhibiting a potential Warburg effect.  Second, in pVHL (the protein encoded by the VHL gene)-deficient ccRCC cells, FBP1 restrains cell proliferation, glycolysis and the pentose phosphate pathway in a catalytic-activity-independent manner, by inhibiting nuclear HIF function via direct interaction with the HIF inhibitory domain.  This unique dual function of the FBP1 protein explains its ubiquitous loss in ccRCC, distinguishing FBP1 from previously identified tumor suppressors that are not consistently mutated in all tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTwq6rXdgYK7Vg90H21EOLACvtfcHk0lhDPylsbYpnaQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFOltrjK&md5=1e0d6f2a5d7dc7fc058866b690feaa9b</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnature13557&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13557%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DB.%26aulast%3DQiu%26aufirst%3DB.%26aulast%3DLee%26aufirst%3DD.%2BS.%2BM.%26aulast%3DWalton%26aufirst%3DZ.%2BE.%26aulast%3DOchocki%26aufirst%3DJ.%2BD.%26aulast%3DMathew%26aufirst%3DL.%2BK.%26aulast%3DMancuso%26aufirst%3DA.%26aulast%3DGade%26aufirst%3DT.%2BP.%2BF.%26aulast%3DKeith%26aufirst%3DB.%26aulast%3DNissim%26aufirst%3DI.%26aulast%3DSimon%26aufirst%3DM.%2BC.%26atitle%3DFructose-1%252C6-bisphosphatase%2520opposes%2520renal%2520carcinoma%2520progression%26jtitle%3DNature%26date%3D2014%26volume%3D513%26spage%3D251%26epage%3D255%26doi%3D10.1038%2Fnature13557" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huangyang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nissim, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weljie, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mancuso, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keith, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoon, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simon, M. C.</span></span> <span> </span><span class="NLM_article-title">Fructose-1,6-bisphosphatase 2 inhibits sarcoma progression by restraining mitochondrial biogenesis</span>. <i>Cell Metab.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">174</span>– <span class="NLM_lpage">188.e7</span>, <span class="refDoi"> DOI: 10.1016/j.cmet.2019.10.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=10.1016%2Fj.cmet.2019.10.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=31761563" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BC1MXit1Krs7fM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2020&pages=174-188.e7&author=P.+Huangyangauthor=F.+Liauthor=P.+Leeauthor=I.+Nissimauthor=A.+M.+Weljieauthor=A.+Mancusoauthor=B.+Liauthor=B.+Keithauthor=S.+S.+Yoonauthor=M.+C.+Simon&title=Fructose-1%2C6-bisphosphatase+2+inhibits+sarcoma+progression+by+restraining+mitochondrial+biogenesis&doi=10.1016%2Fj.cmet.2019.10.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Fructose-1,6-Bisphosphatase 2 Inhibits Sarcoma Progression by Restraining Mitochondrial Biogenesis</span></div><div class="casAuthors">Huangyang, Peiwei; Li, Fuming; Lee, Pearl; Nissim, Itzhak; Weljie, Aalim M.; Mancuso, Anthony; Li, Bo; Keith, Brian; Yoon, Sam S.; Celeste Simon, M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Metabolism</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">174-188.e7</span>CODEN:
                <span class="NLM_cas:coden">CMEEB5</span>;
        ISSN:<span class="NLM_cas:issn">1550-4131</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">The remarkable cellular and genetic heterogeneity of soft tissue sarcomas (STSs) limits the clin. benefit of targeted therapies.  Here, we show that expression of the gluconeogenic isoenzyme fructose-1,6-bisphosphatase 2 (FBP2) is silenced in a broad spectrum of sarcoma subtypes, revealing an apparent common metabolic feature shared by diverse STSs.  Enforced FBP2 expression inhibits sarcoma cell and tumor growth through two distinct mechanisms.  First, cytosolic FBP2 antagonizes elevated glycolysis assocd. with the "Warburg effect", thereby inhibiting sarcoma cell proliferation.  Second, nuclear-localized FBP2 restrains mitochondrial biogenesis and respiration in a catalytic-activity-independent manner by inhibiting the expression of nuclear respiratory factor and mitochondrial transcription factor A (TFAM).  Specifically, nuclear FBP2 colocalizes with the c-Myc transcription factor at the TFAM locus and represses c-Myc-dependent TFAM expression.  This unique dual function of FBP2 provides a rationale for its selective suppression in STSs, identifying a potential metabolic vulnerability of this malignancy and possible therapeutic target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDl8wisDCTFbVg90H21EOLACvtfcHk0lhDPylsbYpnaQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXit1Krs7fM&md5=62f6640a2d1ccdb7535f940ccd5bc958</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.cmet.2019.10.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cmet.2019.10.012%26sid%3Dliteratum%253Aachs%26aulast%3DHuangyang%26aufirst%3DP.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DLee%26aufirst%3DP.%26aulast%3DNissim%26aufirst%3DI.%26aulast%3DWeljie%26aufirst%3DA.%2BM.%26aulast%3DMancuso%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DB.%26aulast%3DKeith%26aufirst%3DB.%26aulast%3DYoon%26aufirst%3DS.%2BS.%26aulast%3DSimon%26aufirst%3DM.%2BC.%26atitle%3DFructose-1%252C6-bisphosphatase%25202%2520inhibits%2520sarcoma%2520progression%2520by%2520restraining%2520mitochondrial%2520biogenesis%26jtitle%3DCell%2520Metab.%26date%3D2020%26volume%3D31%26spage%3D174%26epage%3D188.e7%26doi%3D10.1016%2Fj.cmet.2019.10.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dong, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, T. W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miriyala, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lorkiewicz, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">St Clair, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, M.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evers, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, B. P.</span></span> <span> </span><span class="NLM_article-title">Loss of FBP1 by Snail-mediated repression provides metabolic advantages in basal-like breast cancer</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">316</span>– <span class="NLM_lpage">331</span>, <span class="refDoi"> DOI: 10.1016/j.ccr.2013.01.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=10.1016%2Fj.ccr.2013.01.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=23453623" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtl2jtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2013&pages=316-331&author=C.+Dongauthor=T.+Yuanauthor=Y.+Wuauthor=Y.+Wangauthor=T.+W.+M.+Fanauthor=S.+Miriyalaauthor=Y.+Linauthor=J.+Yaoauthor=J.+Shiauthor=T.+Kangauthor=P.+Lorkiewiczauthor=D.+St+Clairauthor=M.-C.+Hungauthor=B.+M.+Eversauthor=B.+P.+Zhou&title=Loss+of+FBP1+by+Snail-mediated+repression+provides+metabolic+advantages+in+basal-like+breast+cancer&doi=10.1016%2Fj.ccr.2013.01.022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Loss of FBP1 by Snail-Mediated Repression Provides Metabolic Advantages in Basal-like Breast Cancer</span></div><div class="casAuthors">Dong, Chenfang; Yuan, Tingting; Wu, Yadi; Wang, Yifan; Fan, Teresa W. M.; Miriyala, Sumitra; Lin, Yiwei; Yao, Jun; Shi, Jian; Kang, Tiebang; Lorkiewicz, Pawel; St. Clair, Daret; Hung, Mien-Chie; Evers, B. Mark; Zhou, Binhua P.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">316-331</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">The epithelial-mesenchymal transition (EMT) enhances cancer invasiveness and confers tumor cells with cancer stem cell (CSC)-like characteristics.  We show that the Snail-G9a-Dnmt1 complex, which is crit. for E-cadherin promoter silencing, is also required for the promoter methylation of fructose-1,6-biphosphatase (FBP1) in basal-like breast cancer (BLBC).  Loss of FBP1 induces glycolysis and results in increased glucose uptake, macromol. biosynthesis, formation of tetrameric PKM2, and maintenance of ATP prodn. under hypoxia.  Loss of FBP1 also inhibits oxygen consumption and reactive oxygen species prodn. by suppressing mitochondrial complex I activity; this metabolic reprogramming results in an increased CSC-like property and tumorigenicity by enhancing the interaction of β-catenin with T-cell factor.  Our study indicates that the loss of FBP1 is a crit. oncogenic event in EMT and BLBC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWkPRV9TNYgbVg90H21EOLACvtfcHk0ljYC8a0YhH4hg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtl2jtrs%253D&md5=8f9ede71407f07c62e4d3a811890d30e</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2013.01.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2013.01.022%26sid%3Dliteratum%253Aachs%26aulast%3DDong%26aufirst%3DC.%26aulast%3DYuan%26aufirst%3DT.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DFan%26aufirst%3DT.%2BW.%2BM.%26aulast%3DMiriyala%26aufirst%3DS.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DYao%26aufirst%3DJ.%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DKang%26aufirst%3DT.%26aulast%3DLorkiewicz%26aufirst%3DP.%26aulast%3DSt%2BClair%26aufirst%3DD.%26aulast%3DHung%26aufirst%3DM.-C.%26aulast%3DEvers%26aufirst%3DB.%2BM.%26aulast%3DZhou%26aufirst%3DB.%2BP.%26atitle%3DLoss%2520of%2520FBP1%2520by%2520Snail-mediated%2520repression%2520provides%2520metabolic%2520advantages%2520in%2520basal-like%2520breast%2520cancer%26jtitle%3DCancer%2520Cell%26date%3D2013%26volume%3D23%26spage%3D316%26epage%3D331%26doi%3D10.1016%2Fj.ccr.2013.01.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, H.</span></span> <span> </span><span class="NLM_article-title">Dysfunction of natural killer cells by FBP1-induced inhibition of glycolysis during lung cancer progression</span>. <i>Cell Metab.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">243</span>– <span class="NLM_lpage">255.e5</span>, <span class="refDoi"> DOI: 10.1016/j.cmet.2018.06.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=10.1016%2Fj.cmet.2018.06.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=30033198" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlGlur%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2018&pages=243-255.e5&author=J.+Congauthor=X.+Wangauthor=X.+Zhengauthor=D.+Wangauthor=B.+Fuauthor=R.+Sunauthor=Z.+Tianauthor=H.+Wei&title=Dysfunction+of+natural+killer+cells+by+FBP1-induced+inhibition+of+glycolysis+during+lung+cancer+progression&doi=10.1016%2Fj.cmet.2018.06.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Dysfunction of Natural Killer Cells by FBP1-Induced Inhibition of Glycolysis during Lung Cancer Progression</span></div><div class="casAuthors">Cong, Jingjing; Wang, Xianwei; Zheng, Xiaohu; Wang, Dong; Fu, Binqing; Sun, Rui; Tian, Zhigang; Wei, Haiming</div><div class="citationInfo"><span class="NLM_cas:title">Cell Metabolism</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">243-255.e5</span>CODEN:
                <span class="NLM_cas:coden">CMEEB5</span>;
        ISSN:<span class="NLM_cas:issn">1550-4131</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Natural killer (NK) cells are effector lymphocytes with pivotal roles in the resistance against various tumors; dysfunction of NK cells often results in advanced tumor progression.  Tumors develop in three stages comprising initiation, promotion, and progression, but little is known about the interrelationships between NK cells and tumor cells at different stages of tumor development.  Here, we demonstrated that NK cells prevented tumor initiation potently but did not prevent tumor promotion or tumor progression in Kras-driven lung cancer.  Moreover, loss of the antitumor effect in NK cells was closely assocd. with their dysfunctional state during tumor promotion and progression.  Mechanistically, aberrant fructose-1,6-bisphosphatase (FBP1) expression in NK cells elicited their dysfunction by inhibiting glycolysis and impairing viability.  Thus, our results show dynamic alterations of NK cells during tumor development and uncover a novel mechanism involved in NK cell dysfunction, suggesting potential directions for NK cell-based cancer immunotherapy involving FBP1 targeting.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoD-cwjrsMu87Vg90H21EOLACvtfcHk0ljYC8a0YhH4hg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlGlur%252FP&md5=5d13cd38666d2c2b184babd70011bcbe</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.cmet.2018.06.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cmet.2018.06.021%26sid%3Dliteratum%253Aachs%26aulast%3DCong%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DZheng%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DD.%26aulast%3DFu%26aufirst%3DB.%26aulast%3DSun%26aufirst%3DR.%26aulast%3DTian%26aufirst%3DZ.%26aulast%3DWei%26aufirst%3DH.%26atitle%3DDysfunction%2520of%2520natural%2520killer%2520cells%2520by%2520FBP1-induced%2520inhibition%2520of%2520glycolysis%2520during%2520lung%2520cancer%2520progression%26jtitle%3DCell%2520Metab.%26date%3D2018%26volume%3D28%26spage%3D243%26epage%3D255.e5%26doi%3D10.1016%2Fj.cmet.2018.06.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guo, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Broxmeyer, H. E.</span></span> <span> </span><span class="NLM_article-title">Antagonism of PPAR-γ signaling expands human hematopoietic stem and progenitor cells by enhancing glycolysis</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">360</span>– <span class="NLM_lpage">367</span>, <span class="refDoi"> DOI: 10.1038/nm.4477</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=10.1038%2Fnm.4477" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=29377004" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvVequrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2018&pages=360-367&author=B.+Guoauthor=X.+Huangauthor=M.+R.+Leeauthor=S.+A.+Leeauthor=H.+E.+Broxmeyer&title=Antagonism+of+PPAR-%CE%B3+signaling+expands+human+hematopoietic+stem+and+progenitor+cells+by+enhancing+glycolysis&doi=10.1038%2Fnm.4477"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Antagonism of PPAR-γ signaling expands human hematopoietic stem and progenitor cells by enhancing glycolysis</span></div><div class="casAuthors">Guo, Bin; Huang, Xinxin; Lee, Man Ryul; Lee, Sang A.; Broxmeyer, Hal E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">360-367</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Hematopoietic stem cells (HSCs) quiescently reside in bone marrow niches and have the capacity to self-renew or differentiate to form all of the blood cells throughout the lifespan of an animal.  Allogeneic HSC transplantation is a life-saving treatment for malignant and nonmalignant disorders.  HSCs isolated from umbilical cord blood (CB) are used for hematopoietic cell transplantation (HCT), but due to the limited nos. of HSCs in single units of umbilical CB, a no. of methods have been proposed for ex vivo expansion of human HSCs.  We show here that antagonism of peroxisome proliferator-activated receptor (PPAR)-γ promotes ex vivo expansion of phenotypically and functionally defined subsets of human CB HSCs and hematopoietic progenitor cells (HSPCs).  PPAR-γ antagonism in CB HSPCs strongly downregulated expression of several differentiation-assocd. genes, as well as fructose-bisphosphatase 1 (FBP1; which encodes a neg. regulator of glycolysis), and enhanced glycolysis without compromising mitochondrial metab.  The expansion of CB HSPCs by PPAR-γ antagonism was completely suppressed by removal of glucose or inhibition of glycolysis.  Moreover, knockdown of FBP1 expression promoted glycolysis and ex vivo expansion of long-term repopulating CB HSPCs, whereas overexpression of FBP1 suppressed the expansion of CB HSPCs that was induced by PPAR-γ antagonism.  Our study suggests the possibility for a new and simple means for metabolic reprogramming of CB HSPCs to improve the efficacy of HCT.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7LkTosYldjbVg90H21EOLACvtfcHk0ljYC8a0YhH4hg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvVequrY%253D&md5=52b707113e8ed91ce1a6d7897789f889</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnm.4477&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.4477%26sid%3Dliteratum%253Aachs%26aulast%3DGuo%26aufirst%3DB.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DLee%26aufirst%3DM.%2BR.%26aulast%3DLee%26aufirst%3DS.%2BA.%26aulast%3DBroxmeyer%26aufirst%3DH.%2BE.%26atitle%3DAntagonism%2520of%2520PPAR-%25CE%25B3%2520signaling%2520expands%2520human%2520hematopoietic%2520stem%2520and%2520progenitor%2520cells%2520by%2520enhancing%2520glycolysis%26jtitle%3DNat.%2520Med.%26date%3D2018%26volume%3D24%26spage%3D360%26epage%3D367%26doi%3D10.1038%2Fnm.4477" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rines, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharabi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tavares, C. D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Puigserver, P.</span></span> <span> </span><span class="NLM_article-title">Targeting hepatic glucose metabolism in the treatment of type 2 diabetes</span>. <i>Nat. Rev. Drug Discov.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">786</span>– <span class="NLM_lpage">804</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.151</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=10.1038%2Fnrd.2016.151" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=27516169" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhtlehur7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=786-804&author=A.+K.+Rinesauthor=K.+Sharabiauthor=C.+D.+J.+Tavaresauthor=P.+Puigserver&title=Targeting+hepatic+glucose+metabolism+in+the+treatment+of+type+2+diabetes&doi=10.1038%2Fnrd.2016.151"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting hepatic glucose metabolism in the treatment of type 2 diabetes</span></div><div class="casAuthors">Rines, Amy K.; Sharabi, Kfir; Tavares, Clint D. J.; Puigserver, Pere</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">786-804</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Type 2 diabetes mellitus is characterized by the dysregulation of glucose homeostasis, resulting in hyperglycemia.  Although current diabetes treatments have exhibited some success in lowering blood glucose levels, their effect is not always sustained and their use may be assocd. with undesirable side effects, such as hypoglycemia.  Novel antidiabetic drugs, which may be used in combination with existing therapies, are therefore needed.  The potential of specifically targeting the liver to normalize blood glucose levels has not been fully exploited.  Here, we review the mol. mechanisms controlling hepatic gluconeogenesis and glycogen storage, and assess the prospect of therapeutically targeting assocd. pathways to treat type 2 diabetes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZigCtBTlD37Vg90H21EOLACvtfcHk0li4PPURTQ-2dw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhtlehur7J&md5=a51c9b6149bb4a6729f2bac583171865</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.151&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.151%26sid%3Dliteratum%253Aachs%26aulast%3DRines%26aufirst%3DA.%2BK.%26aulast%3DSharabi%26aufirst%3DK.%26aulast%3DTavares%26aufirst%3DC.%2BD.%2BJ.%26aulast%3DPuigserver%26aufirst%3DP.%26atitle%3DTargeting%2520hepatic%2520glucose%2520metabolism%2520in%2520the%2520treatment%2520of%2520type%25202%2520diabetes%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discov.%26date%3D2016%26volume%3D15%26spage%3D786%26epage%3D804%26doi%3D10.1038%2Fnrd.2016.151" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaur, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dahiya, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, M.</span></span> <span> </span><span class="NLM_article-title">Fructose-1,6-bisphosphatase inhibitors: A new valid approach for management of type 2 diabetes mellitus</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>141</i></span>,  <span class="NLM_fpage">473</span>– <span class="NLM_lpage">505</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2017.09.029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=10.1016%2Fj.ejmech.2017.09.029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=29055870" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslSiu73E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=141&publication_year=2017&pages=473-505&author=R.+Kaurauthor=L.+Dahiyaauthor=M.+Kumar&title=Fructose-1%2C6-bisphosphatase+inhibitors%3A+A+new+valid+approach+for+management+of+type+2+diabetes+mellitus&doi=10.1016%2Fj.ejmech.2017.09.029"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Fructose-1,6-bisphosphatase inhibitors: A new valid approach for management of type 2 diabetes mellitus</span></div><div class="casAuthors">Kaur, Ramandeep; Dahiya, Lalita; Kumar, Manoj</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">141</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">473-505</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A review.  The rising incidence of diabetes and confines allied with clin. therapies emphasized the need to explore new mol. targets to develop novel, effective and safer antihyperglycemic agents.  Excessive endogenous glucose prodn. by gluconeogenesis is a primary determinant of hyperglycemia in patients with type 2 diabetes.  But not even a single current medication acts directly to reduce gluconeogenesis.  Fructose-1,6-bisphosphatase (FBPase), a well recognized rate controlling enzyme of gluconeogenesis, has emerged as legitimate mol. level target to control gluconeogenesis mediated glucose overprodn. and its inhibitors are likely to fulfill an unmet medical need.  In this compilation various chem. classes of FBPase inhibitors have been reviewed which mainly acts through uncompetitive and non-competitive manner.  A detailed account on structure activity relationship studies of inhibitors have been presented along with their mol. level interactions at binding sites of enzyme.  Three Dimensional Quant. Structure Activity relationship (3D-QSAR) studies, performed to optimize the lead, have been summarized at some places.  In this assemblage, FBPase inhibitors patented in past have been compiled in tabular form.  This review highlights the new insight into the therapeutic utility of FBPase inhibitors and their potential as a new class of antidiabetic drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1W8b0O6Cbl7Vg90H21EOLACvtfcHk0li4PPURTQ-2dw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslSiu73E&md5=e2736803222e3b48cfd01bc230455742</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2017.09.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2017.09.029%26sid%3Dliteratum%253Aachs%26aulast%3DKaur%26aufirst%3DR.%26aulast%3DDahiya%26aufirst%3DL.%26aulast%3DKumar%26aufirst%3DM.%26atitle%3DFructose-1%252C6-bisphosphatase%2520inhibitors%253A%2520A%2520new%2520valid%2520approach%2520for%2520management%2520of%2520type%25202%2520diabetes%2520mellitus%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2017%26volume%3D141%26spage%3D473%26epage%3D505%26doi%3D10.1016%2Fj.ejmech.2017.09.029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bie, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Z.</span></span> <span> </span><span class="NLM_article-title">Discovery of novel indole derivatives as allosteric inhibitors of fructose-1,6-bisphosphatase</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>90</i></span>,  <span class="NLM_fpage">394</span>– <span class="NLM_lpage">405</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2014.11.049</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=10.1016%2Fj.ejmech.2014.11.049" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=25461330" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitVWhs7nL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=90&publication_year=2015&pages=394-405&author=J.+Bieauthor=S.+Liuauthor=Z.+Liauthor=Y.+Muauthor=B.+Xuauthor=Z.+Shen&title=Discovery+of+novel+indole+derivatives+as+allosteric+inhibitors+of+fructose-1%2C6-bisphosphatase&doi=10.1016%2Fj.ejmech.2014.11.049"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of novel indole derivatives as allosteric inhibitors of fructose-1,6-bisphosphatase</span></div><div class="casAuthors">Bie, Jianbo; Liu, Shuainan; Li, Zhanmei; Mu, Yongzhao; Xu, Bailing; Shen, Zhufang</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">90</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">394-405</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A series of novel indole derivs. was designed and synthesized as inhibitors of fructose-1,6-bisphosphatase (FBPase).  The most potent compd. 3-(2-Carboxyethyl)-5-ethyl-7-nitro-1H-indole-2-carboxylic acid was identified with an IC50 value of 0.10 μM by testing the inhibitory activity against recombinant human FBPase.  The structure-activity relations were studied on the substitution at 4- and 5-position of the indole scaffold.  The binding interactions of the title compds. at AMP binding site of FBPase were predicted using CDOCKER algorithm.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoINKqL176MDrVg90H21EOLACvtfcHk0li4PPURTQ-2dw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitVWhs7nL&md5=ce4e81ee3f0753d3013dae9e57252133</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2014.11.049&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2014.11.049%26sid%3Dliteratum%253Aachs%26aulast%3DBie%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DMu%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DB.%26aulast%3DShen%26aufirst%3DZ.%26atitle%3DDiscovery%2520of%2520novel%2520indole%2520derivatives%2520as%2520allosteric%2520inhibitors%2520of%2520fructose-1%252C6-bisphosphatase%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2015%26volume%3D90%26spage%3D394%26epage%3D405%26doi%3D10.1016%2Fj.ejmech.2014.11.049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">van
Poelje, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potter, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandramouli, V. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landau, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erion, M. D.</span></span> <span> </span><span class="NLM_article-title">Inhibition of fructose 1,6-bisphosphatase reduces excessive endogenous glucose production and attenuates hyperglycemia in Zucker diabetic fatty rats</span>. <i>Diabetes</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">1747</span>– <span class="NLM_lpage">1754</span>, <span class="refDoi"> DOI: 10.2337/db05-1443</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=10.2337%2Fdb05-1443" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=16731838" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BD28XlsF2rurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2006&pages=1747-1754&author=P.+D.+van%0APoeljeauthor=S.+C.+Potterauthor=V.+C.+Chandramouliauthor=B.+R.+Landauauthor=Q.+Dangauthor=M.+D.+Erion&title=Inhibition+of+fructose+1%2C6-bisphosphatase+reduces+excessive+endogenous+glucose+production+and+attenuates+hyperglycemia+in+Zucker+diabetic+fatty+rats&doi=10.2337%2Fdb05-1443"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of fructose 1,6-bisphosphatase reduces excessive endogenous glucose production and attenuates hyperglycemia in Zucker diabetic fatty rats</span></div><div class="casAuthors">van Poelje, Paul D.; Potter, Scott C.; Chandramouli, Visvanathan C.; Landau, Bernard R.; Dang, Qun; Erion, Mark D.</div><div class="citationInfo"><span class="NLM_cas:title">Diabetes</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1747-1754</span>CODEN:
                <span class="NLM_cas:coden">DIAEAZ</span>;
        ISSN:<span class="NLM_cas:issn">0012-1797</span>.
    
            (<span class="NLM_cas:orgname">American Diabetes Association</span>)
        </div><div class="casAbstract">Gluconeogenesis is increased in type 2 diabetes and contributes significantly to fasting and postprandial hyperglycemia.  We recently reported the discovery of the 1st potent and selective inhibitors of fructose 1,6-bisphosphatase (FBPase), a rate-controlling enzyme of gluconeogenesis.  Herein we describe acute and chronic effects of the lead inhibitor, MB06322 (CS-917), in rodent models of type 2 diabetes.  In fasting male ZDF rats with overt diabetes, a single dose of MB06322 inhibited gluconeogenesis by 70% and overall endogenous glucose prodn. by 46%, leading to a redn. in blood glucose of >200 mg/dL.  Chronic treatment of freely feeding 6-wk-old male Zucker diabetic fatty (ZDF) rats delayed the development of hyperglycemia and preserved pancreatic function.  Elevation of lactate (∼1.5-fold) occurred after 4 wk of treatment, as did the apparent shunting of precursors into triglycerides.  Profound glucose lowering (∼44%) and similar metabolic ramifications were assocd. with 2-wk intervention therapy of 10-wk-old male ZDF rats.  In high-fat diet-fed female ZDF rats, MB06322 treatment for 2 wk fully attenuated hyperglycemia without evidence of metabolic perturbation other than a modest redn. in glycogen stores (∼20%).  The studies confirm that excessive gluconeogenesis plays an integral role in the pathophysiol. of type 2 diabetes and suggest that FBPase inhibitors may provide a future treatment option.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoW3d61gJUTBrVg90H21EOLACvtfcHk0li4PPURTQ-2dw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XlsF2rurc%253D&md5=f85ef1bcc9b9e2de186efe2afe45643c</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.2337%2Fdb05-1443&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2337%252Fdb05-1443%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2BPoelje%26aufirst%3DP.%2BD.%26aulast%3DPotter%26aufirst%3DS.%2BC.%26aulast%3DChandramouli%26aufirst%3DV.%2BC.%26aulast%3DLandau%26aufirst%3DB.%2BR.%26aulast%3DDang%26aufirst%3DQ.%26aulast%3DErion%26aufirst%3DM.%2BD.%26atitle%3DInhibition%2520of%2520fructose%25201%252C6-bisphosphatase%2520reduces%2520excessive%2520endogenous%2520glucose%2520production%2520and%2520attenuates%2520hyperglycemia%2520in%2520Zucker%2520diabetic%2520fatty%2520rats%26jtitle%3DDiabetes%26date%3D2006%26volume%3D55%26spage%3D1747%26epage%3D1754%26doi%3D10.2337%2Fdb05-1443" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dang, Q.</span>; <span class="NLM_string-name">Van
Poelje, P. D.</span>; <span class="NLM_string-name">Erion, M. D.</span></span> <span> </span><span class="NLM_article-title">The discovery and development of MB07803, a second-generation fructose-1,6-bisphosphatase inhibitor with improved pharmacokinetic properties, as a potential treatment of type 2 diabetes</span>. In  <i>New Therapeutic Strategies for Type 2 Diabetes: Small Molecule Approaches</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robert, M. J.</span></span>; <span class="NLM_publisher-name">Royal Society of Chemistry</span>, <span class="NLM_publisher-loc">London</span> <span class="NLM_year">2012</span>, pp  <span class="NLM_fpage">306</span>– <span class="NLM_lpage">323</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=10.1039%2F9781849735322-00306" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&pages=306-323&author=Q.+Dang&author=P.+D.+Van%0APoelje&author=M.+D.+Erionauthor=M.+J.+Robert&title=New+Therapeutic+Strategies+for+Type+2+Diabetes%3A+Small+Molecule+Approaches"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1039%2F9781849735322-00306&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1039%252F9781849735322-00306%26sid%3Dliteratum%253Aachs%26aulast%3DDang%26aufirst%3DQ.%26atitle%3DThe%2520discovery%2520and%2520development%2520of%2520MB07803%252C%2520a%2520second-generation%2520fructose-1%252C6-bisphosphatase%2520inhibitor%2520with%2520improved%2520pharmacokinetic%2520properties%252C%2520as%2520a%2520potential%2520treatment%2520of%2520type%25202%2520diabetes%26btitle%3DNew%2520Therapeutic%2520Strategies%2520for%2520Type%25202%2520Diabetes%253A%2520Small%2520Molecule%2520Approaches%26aulast%3DRobert%26aufirst%3DM.%2BJ.%26pub%3DRoyal%2520Society%2520of%2520Chemistry%26date%3D2012%26spage%3D306%26epage%3D323" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasibhatla, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potter, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujitaki, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Poelje, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lipscomb, W. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erion, M. D.</span></span> <span> </span><span class="NLM_article-title">Discovery of potent and specific fructose-1,6-bisphosphatase inhibitors and a series of orally-bioavailable phosphoramidase-sensitive prodrugs for the treatment of type 2 diabetes</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>129</i></span>,  <span class="NLM_fpage">15491</span>– <span class="NLM_lpage">15502</span>, <span class="refDoi"> DOI: 10.1021/ja074871l</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja074871l" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlOgsrjP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=129&publication_year=2007&pages=15491-15502&author=Q.+Dangauthor=S.+R.+Kasibhatlaauthor=K.+R.+Reddyauthor=T.+Jiangauthor=M.+R.+Reddyauthor=S.+C.+Potterauthor=J.+M.+Fujitakiauthor=P.+D.+van+Poeljeauthor=J.+Huangauthor=W.+N.+Lipscombauthor=M.+D.+Erion&title=Discovery+of+potent+and+specific+fructose-1%2C6-bisphosphatase+inhibitors+and+a+series+of+orally-bioavailable+phosphoramidase-sensitive+prodrugs+for+the+treatment+of+type+2+diabetes&doi=10.1021%2Fja074871l"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Potent and Specific Fructose-1,6-Bisphosphatase Inhibitors and a Series of Orally-Bioavailable Phosphoramidase-Sensitive Prodrugs for the Treatment of Type 2 Diabetes</span></div><div class="casAuthors">Dang, Qun; Rao Kasibhatla, Srinivas; Reddy, K. Raja; Jiang, Tao; Reddy, M. Rami; Potter, Scott C.; Fujitaki, James M.; van Poelje, Paul D.; Huang, Jingwei; Lipscomb, William N.; Erion, Mark D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">129</span>
        (<span class="NLM_cas:issue">50</span>),
    <span class="NLM_cas:pages">15491-15502</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Excessive glucose prodn. by the liver coupled with decreased glucose uptake and metab. by muscle, fat, and liver results in chronically elevated blood glucose levels in patients with type 2 diabetes.  Efforts to treat diabetes by reducing glucose prodn. have largely focused on the gluconeogenesis pathway and rate-limiting enzymes within this pathway such as fructose-1,6-bisphosphatase (FBPase).  The first potent FBPase inhibitors were identified using a structure-guided drug design strategy but proved difficult to deliver orally.  Herein, we report the synthesis and characterization of a series of orally bioavailable FBPase inhibitors identified following the combined discoveries of a low mol. wt. inhibitor series with increased potency and a phosphonate prodrug class suitable for their oral delivery.  The lead inhibitor, 10A (I), was designed with the aid of X-ray crystallog. and mol. modeling to bind to the allosteric AMP binding site of FBPase.  High potency (IC50 = 16 nM) and FBPase specificity were achieved by linking a 2-aminothiazole with a phosphonic acid.  Free-energy perturbation calcns. provided insight into the factors that contributed to the high binding affinity.  10A and std. phosphonate prodrugs of 10A exhibited poor oral bioavailability (0.2-11%).  Improved oral bioavailability (22-47%) was achieved using phosphonate diamides that convert to the corresponding phosphonic acid by sequential action of an esterase and a phosphoramidase.  Oral administration of the lead prodrug, MB06322 (30, CS-917), to Zucker Diabetic Fatty rats led to dose-dependent inhibition of gluconeogenesis and endogenous glucose prodn. and consequently to significant blood glucose redn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopA52xpsptwLVg90H21EOLACvtfcHk0lhu0QrDYmD_SA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlOgsrjP&md5=ea5856c4481a445d06f45882a8562def</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Fja074871l&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja074871l%26sid%3Dliteratum%253Aachs%26aulast%3DDang%26aufirst%3DQ.%26aulast%3DKasibhatla%26aufirst%3DS.%2BR.%26aulast%3DReddy%26aufirst%3DK.%2BR.%26aulast%3DJiang%26aufirst%3DT.%26aulast%3DReddy%26aufirst%3DM.%2BR.%26aulast%3DPotter%26aufirst%3DS.%2BC.%26aulast%3DFujitaki%26aufirst%3DJ.%2BM.%26aulast%3Dvan%2BPoelje%26aufirst%3DP.%2BD.%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DLipscomb%26aufirst%3DW.%2BN.%26aulast%3DErion%26aufirst%3DM.%2BD.%26atitle%3DDiscovery%2520of%2520potent%2520and%2520specific%2520fructose-1%252C6-bisphosphatase%2520inhibitors%2520and%2520a%2520series%2520of%2520orally-bioavailable%2520phosphoramidase-sensitive%2520prodrugs%2520for%2520the%2520treatment%2520of%2520type%25202%2520diabetes%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2007%26volume%3D129%26spage%3D15491%26epage%3D15502%26doi%3D10.1021%2Fja074871l" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Erion, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasibhatla, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lipscomb, W. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Poelje, P. D.</span></span> <span> </span><span class="NLM_article-title">Structure-guided design of AMP mimics that inhibit fructose-1,6-bisphosphatase with high affinity and specificity</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>129</i></span>,  <span class="NLM_fpage">15480</span>– <span class="NLM_lpage">15490</span>, <span class="refDoi"> DOI: 10.1021/ja074869u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja074869u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlOgsrrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=129&publication_year=2007&pages=15480-15490&author=M.+D.+Erionauthor=Q.+Dangauthor=M.+R.+Reddyauthor=S.+R.+Kasibhatlaauthor=J.+Huangauthor=W.+N.+Lipscombauthor=P.+D.+van+Poelje&title=Structure-guided+design+of+AMP+mimics+that+inhibit+fructose-1%2C6-bisphosphatase+with+high+affinity+and+specificity&doi=10.1021%2Fja074869u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Guided Design of AMP Mimics That Inhibit Fructose-1,6-bisphosphatase with High Affinity and Specificity</span></div><div class="casAuthors">Erion, Mark D.; Dang, Qun; Reddy, M. Rami; Kasibhatla, Srinivas Rao; Huang, Jingwei; Lipscomb, William N.; van Poelje, Paul D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">129</span>
        (<span class="NLM_cas:issue">50</span>),
    <span class="NLM_cas:pages">15480-15490</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">AMP binding sites are commonly used by nature for allosteric regulation of enzymes controlling the prodn. and metab. of carbohydrates and lipids.  Since many of these enzymes represent potential drug targets for metabolic diseases, efforts were initiated to discover AMP mimics that bind to AMP-binding sites with high affinity and high enzyme specificity.  Herein the authors report the structure-guided design of potent fructose 1,6-bisphosphatase (FBPase) inhibitors that interact with the AMP binding site on FBPase despite their structural dissimilarity to AMP.  Mol. modeling, free-energy perturbation calcns., x-ray crystallog., and enzyme kinetic data guided the redesign of AMP, which began by replacing the 5'-phosphate with a phosphonic acid attached to C8 of the adenine base via a 3-atom spacer.  Addnl. binding affinity was gained by replacing the ribose with an alkyl group that formed van der Waals interactions with a hydrophobic region within the AMP binding site and by replacing the purine nitrogens N1 and N3 with carbons to minimize desolvation energy expenditures.  The resulting benzimidazole phosphonic acid, 16, inhibited human FBPase (IC50 = 90 nM) 11-fold more potently than AMP and exhibited high specificity for the AMP binding site on FBPase.  16 Also inhibited FBPase in primary rat hepatocytes and correspondingly resulted in concn.-dependent inhibition of the gluconeogenesis pathway.  Accordingly, these results suggest that the AMP site of FBPase may represent a potential drug target for reducing the excessive glucose produced by the gluconeogenesis pathway in patients with type 2 diabetes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5aPGR2PR-9bVg90H21EOLACvtfcHk0lhu0QrDYmD_SA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlOgsrrJ&md5=15c15e648a7188fd3280be2c8c17b35f</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Fja074869u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja074869u%26sid%3Dliteratum%253Aachs%26aulast%3DErion%26aufirst%3DM.%2BD.%26aulast%3DDang%26aufirst%3DQ.%26aulast%3DReddy%26aufirst%3DM.%2BR.%26aulast%3DKasibhatla%26aufirst%3DS.%2BR.%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DLipscomb%26aufirst%3DW.%2BN.%26aulast%3Dvan%2BPoelje%26aufirst%3DP.%2BD.%26atitle%3DStructure-guided%2520design%2520of%2520AMP%2520mimics%2520that%2520inhibit%2520fructose-1%252C6-bisphosphatase%2520with%2520high%2520affinity%2520and%2520specificity%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2007%26volume%3D129%26spage%3D15480%26epage%3D15490%26doi%3D10.1021%2Fja074869u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nussinov, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, C. J.</span></span> <span> </span><span class="NLM_article-title">Allostery in disease and in drug discovery</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>153</i></span>,  <span class="NLM_fpage">293</span>– <span class="NLM_lpage">305</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2013.03.034</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=10.1016%2Fj.cell.2013.03.034" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=23582321" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlvVChu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=153&publication_year=2013&pages=293-305&author=R.+Nussinovauthor=C.+J.+Tsai&title=Allostery+in+disease+and+in+drug+discovery&doi=10.1016%2Fj.cell.2013.03.034"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Allostery in Disease and in Drug Discovery</span></div><div class="casAuthors">Nussinov, Ruth; Tsai, Chung-Jung</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">153</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">293-305</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Allostery is largely assocd. with conformational and functional transitions in individual proteins.  This concept can be extended to consider the impact of conformational perturbations on cellular function and disease states.  Here, we clarify the concept of allostery and how it controls physiol. activities.  We focus on the challenging questions of how allostery can both cause disease and contribute to development of new therapeutics.  We aim to increase the awareness of the linkage between disease symptoms on the cellular level and specific aberrant allosteric actions on the mol. level and to emphasize the potential of allosteric drugs in innovative therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrz5bXPgUjfFLVg90H21EOLACvtfcHk0liP7vYoZ2_HQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlvVChu70%253D&md5=14a91c0d364187044e5ad3e048868514</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2013.03.034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2013.03.034%26sid%3Dliteratum%253Aachs%26aulast%3DNussinov%26aufirst%3DR.%26aulast%3DTsai%26aufirst%3DC.%2BJ.%26atitle%3DAllostery%2520in%2520disease%2520and%2520in%2520drug%2520discovery%26jtitle%3DCell%26date%3D2013%26volume%3D153%26spage%3D293%26epage%3D305%26doi%3D10.1016%2Fj.cell.2013.03.034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">Allosteric methods and their applications: facilitating the discovery of allosteric drugs and the investigation of allosteric mechanisms</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">492</span>– <span class="NLM_lpage">500</span>, <span class="refDoi"> DOI: 10.1021/acs.accounts.8b00570</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.accounts.8b00570" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvVWrt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2019&pages=492-500&author=S.+Luauthor=Q.+Shenauthor=J.+Zhang&title=Allosteric+methods+and+their+applications%3A+facilitating+the+discovery+of+allosteric+drugs+and+the+investigation+of+allosteric+mechanisms&doi=10.1021%2Facs.accounts.8b00570"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric Methods and Their Applications: Facilitating the Discovery of Allosteric Drugs and the Investigation of Allosteric Mechanisms</span></div><div class="casAuthors">Lu, Shaoyong; Shen, Qiancheng; Zhang, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">492-500</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Allostery, or allosteric regulation, is the phenomenon in which protein functional activity is altered by the binding of an effector at an allosteric site that is topog. distinct from the orthosteric, active site.  As one of the most direct and efficient ways to regulate protein function, allostery has played a fundamental role in innumerable biol. processes of all living organisms, including enzyme catalysis, signal transduction, cell metab., and gene transcription.  It is thus considered as "the second secret of life".  The abnormality of allosteric communication networks between allosteric and orthosteric sites is assocd. with the pathogenesis of human diseases.  Allosteric modulators, by attaching to structurally diverse allosteric sites, offer the potential for differential selectivity and improved safety compared with orthosteric drugs that bind to conserved orthosteric sites.  Harnessing allostery has thus been regarded as a novel strategy for drug discovery.  Despite much progress having been made in the repertoire of allostery since the turn of the millennium, the identification of allosteric drugs for therapeutic targets and the elucidation of allosteric mechanisms still present substantial challenges.  These challenges are derived from the difficulties in the identification of allosteric sites and mutations, the assessment of allosteric protein-modulator interactions, the screening of allosteric modulators, and the elucidation of allosteric mechanisms in biol. systems.To address these issues, we have developed a panel of allosteric services for specific allosteric applications over the past decade, including (i) the creation of the Allosteric Database, with the aim of providing comprehensive allosteric information such as allosteric proteins, modulators, sites, pathways, etc., (ii) the construction of the ASBench benchmark of high-quality allosteric sites for the development of computational methods for predicting allosteric sites, (iii) the development of Allosite and AllositePro for the prediction of the location of allosteric sites in proteins, (iv) the development of the Alloscore scoring function for the evaluation of allosteric protein-modulator interactions, (v) the development of Allosterome for evolutionary anal. of query allosteric sites/modulators within the human proteome, (vi) the development of AlloDriver for the prediction of allosteric mutagenesis, and (vii) the development of AlloFinder for the virtual screening of allosteric modulators and the investigation of allosteric mechanisms.  Importantly, we have validated computationally predicted allosteric sites, mutations, and modulators in the real cases of sirtuin 6, casein kinase 2α, phosphodiesterase 10A, and signal transduction and activation of transcription 3.  Furthermore, our developed allosteric methods have been widely exploited by other users around the world for allosteric research.  Therefore, these allosteric services are expected to expedite the discovery of allosteric drugs and the investigation of allosteric mechanisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrlHNG36v7_bVg90H21EOLACvtfcHk0liP7vYoZ2_HQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvVWrt7o%253D&md5=a23fcee3def606164d2a22b0681b5837</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Facs.accounts.8b00570&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.accounts.8b00570%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DS.%26aulast%3DShen%26aufirst%3DQ.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DAllosteric%2520methods%2520and%2520their%2520applications%253A%2520facilitating%2520the%2520discovery%2520of%2520allosteric%2520drugs%2520and%2520the%2520investigation%2520of%2520allosteric%2520mechanisms%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2019%26volume%3D52%26spage%3D492%26epage%3D500%26doi%3D10.1021%2Facs.accounts.8b00570" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lonsdale, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, R. A.</span></span> <span> </span><span class="NLM_article-title">Structure-based design of targeted covalent inhibitors</span>. <i>Chem. Soc. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">3816</span>– <span class="NLM_lpage">3830</span>, <span class="refDoi"> DOI: 10.1039/C7CS00220C</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=10.1039%2FC7CS00220C" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=29620097" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BC1cXntVGiurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2018&pages=3816-3830&author=R.+Lonsdaleauthor=R.+A.+Ward&title=Structure-based+design+of+targeted+covalent+inhibitors&doi=10.1039%2FC7CS00220C"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based design of targeted covalent inhibitors</span></div><div class="casAuthors">Lonsdale, Richard; Ward, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Society Reviews</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3816-3830</span>CODEN:
                <span class="NLM_cas:coden">CSRVBR</span>;
        ISSN:<span class="NLM_cas:issn">0306-0012</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Covalent inhibition is a rapidly growing discipline within drug discovery.  Many historical covalent inhibitors were discovered by serendipity, with such a mechanism of action often regarded as undesirable due to potential toxicity issues.  Recent progress has seen a major shift in this outlook, as covalent inhibition shows promise for targets where previous efforts to identify non-covalent small mol. inhibitors have failed.  Targeted covalent inhibitors (TCIs) can offer drug discovery scientists the ability to increase the potency and/or selectivity of small mol. inhibitors, by attachment of reactive functional groups designed to covalently bind to specific sites in a target.  In this tutorial review we introduce the broader concept of covalent inhibition, discuss the potential benefits and challenges of such an approach, and provide an overview of the current status of the field.  We also describe some strategies and computational tools to enable successful covalent drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqnSIGMuS3vm7Vg90H21EOLACvtfcHk0liP7vYoZ2_HQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXntVGiurs%253D&md5=a7e31b23cf8ecfcad1397ad10ea10b7b</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1039%2FC7CS00220C&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7CS00220C%26sid%3Dliteratum%253Aachs%26aulast%3DLonsdale%26aufirst%3DR.%26aulast%3DWard%26aufirst%3DR.%2BA.%26atitle%3DStructure-based%2520design%2520of%2520targeted%2520covalent%2520inhibitors%26jtitle%3DChem.%2520Soc.%2520Rev.%26date%3D2018%26volume%3D47%26spage%3D3816%26epage%3D3830%26doi%3D10.1039%2FC7CS00220C" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Singh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petter, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baillie, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitty, A.</span></span> <span> </span><span class="NLM_article-title">The resurgence of covalent drugs</span>. <i>Nat. Rev. Drug Discov.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">307</span>– <span class="NLM_lpage">317</span>, <span class="refDoi"> DOI: 10.1038/nrd3410</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=10.1038%2Fnrd3410" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=21455239" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktVGmu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=307-317&author=J.+Singhauthor=R.+C.+Petterauthor=T.+A.+Baillieauthor=A.+Whitty&title=The+resurgence+of+covalent+drugs&doi=10.1038%2Fnrd3410"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">The resurgence of covalent drugs</span></div><div class="casAuthors">Singh, Juswinder; Petter, Russell C.; Baillie, Thomas A.; Whitty, Adrian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">307-317</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Covalent drugs haveproved to be successful therapies for various indications, but largely owing to safety concerns, they are rarely considered when initiating a target-directed drug discovery project.  There is a need to reassess this important class of drugs, and to reconcile the discordance between the historic success of covalent drugs and the reluctance of most drug discovery teams to include them in their armamentarium.  This Review surveys the prevalence and pharmacol. advantages of covalent drugs, discusses how potential risks and challenges may be addressed through innovative design, and presents the broad opportunities provided by targeted covalent inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreDCEmORV_NbVg90H21EOLACvtfcHk0lgI_-H35YuvtQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktVGmu7g%253D&md5=2190289081e151416c097be4a5b04460</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fnrd3410&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3410%26sid%3Dliteratum%253Aachs%26aulast%3DSingh%26aufirst%3DJ.%26aulast%3DPetter%26aufirst%3DR.%2BC.%26aulast%3DBaillie%26aufirst%3DT.%2BA.%26aulast%3DWhitty%26aufirst%3DA.%26atitle%3DThe%2520resurgence%2520of%2520covalent%2520drugs%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discov.%26date%3D2011%26volume%3D10%26spage%3D307%26epage%3D317%26doi%3D10.1038%2Fnrd3410" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">Designed covalent allosteric modulators: an emerging paradigm in drug discovery</span>. <i>Drug Discov. Today</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">447</span>– <span class="NLM_lpage">453</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2016.11.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=10.1016%2Fj.drudis.2016.11.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=27888140" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFOju7zJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2017&pages=447-453&author=S.+Luauthor=J.+Zhang&title=Designed+covalent+allosteric+modulators%3A+an+emerging+paradigm+in+drug+discovery&doi=10.1016%2Fj.drudis.2016.11.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Designed covalent allosteric modulators: an emerging paradigm in drug discovery</span></div><div class="casAuthors">Lu, Shaoyong; Zhang, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">447-453</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Covalent allosteric modulators possess the pharmacol. advantages (high potency, extended duration of action and low drug resistance) of covalent ligands and the addnl. benefit of the higher specificity and lower toxicity of allosteric modulators.  This approach is gaining increasing recognition as a valuable tool in drug discovery.  Here, we review the recent advances in the design of covalent allosteric modulators with an emphasis on successful examples.  A broad spectrum of protein targets capable of being modulated by them reflects the prevalence of this strategy.  We also discuss the challenges and future directions in the development of covalent allosteric modulators.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqnMtSOrLWxdrVg90H21EOLACvtfcHk0lgI_-H35YuvtQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFOju7zJ&md5=bc656114f3f956ce44ba06c1a7669da4</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2016.11.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2016.11.013%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DDesigned%2520covalent%2520allosteric%2520modulators%253A%2520an%2520emerging%2520paradigm%2520in%2520drug%2520discovery%26jtitle%3DDrug%2520Discov.%2520Today%26date%3D2017%26volume%3D22%26spage%3D447%26epage%3D453%26doi%3D10.1016%2Fj.drudis.2016.11.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nussinov, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, C. J.</span></span> <span> </span><span class="NLM_article-title">The design of covalent allosteric drugs</span>. <i>Annu. Rev. Pharmacol. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">249</span>– <span class="NLM_lpage">267</span>, <span class="refDoi"> DOI: 10.1146/annurev-pharmtox-010814-124401</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=10.1146%2Fannurev-pharmtox-010814-124401" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=25149918" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BC2MXkt12qsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=249-267&author=R.+Nussinovauthor=C.+J.+Tsai&title=The+design+of+covalent+allosteric+drugs&doi=10.1146%2Fannurev-pharmtox-010814-124401"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">The design of covalent allosteric drugs</span></div><div class="casAuthors">Nussinov, Ruth; Tsai, Chung-Jung</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Pharmacology and Toxicology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">249-267</span>CODEN:
                <span class="NLM_cas:coden">ARPTDI</span>;
        ISSN:<span class="NLM_cas:issn">0362-1642</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">A review.  A key issue in drug discovery is how to reduce drug dosage and increase specificity while retaining or increasing efficacy, as high dosage is often linked to toxicity.  There are two types of drugs on the market: orthosteric and allosteric.  Orthosteric drugs can be noncovalent or covalent.  The latter are advantageous because they may be prescribed in lower doses, but their potential off-target toxicity is a primary concern.  The chief advantages of allosteric drugs are their higher specificity and their consequently lower chance of toxic side effects.  Covalent allosteric drugs combine the pharmacol. merits of covalent drugs with the addnl. benefit of the higher specificity of allosteric drugs.  In a recent promising step in therapeutic drug development, allosteric, disulfide-tethered fragments successfully modulated the activity of a protein kinase and K-Ras.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCurPEiYvhQrVg90H21EOLACvtfcHk0lgI_-H35YuvtQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXkt12qsrc%253D&md5=b85640442af2255165ba382bf00abd3d</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1146%2Fannurev-pharmtox-010814-124401&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-pharmtox-010814-124401%26sid%3Dliteratum%253Aachs%26aulast%3DNussinov%26aufirst%3DR.%26aulast%3DTsai%26aufirst%3DC.%2BJ.%26atitle%3DThe%2520design%2520of%2520covalent%2520allosteric%2520drugs%26jtitle%3DAnnu.%2520Rev.%2520Pharmacol.%2520Toxicol.%26date%3D2015%26volume%3D55%26spage%3D249%26epage%3D267%26doi%3D10.1146%2Fannurev-pharmtox-010814-124401" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Uhlenbrock, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weisner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landel, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindemann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hardick, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gontla, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scheinpflug, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czodrowski, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janning, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Depta, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quambusch, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engels, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rauh, D.</span></span> <span> </span><span class="NLM_article-title">Structural and chemical insights into the covalent-allosteric inhibition of the protein kinase Akt</span>. <i>Chem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">3573</span>– <span class="NLM_lpage">3585</span>, <span class="refDoi"> DOI: 10.1039/C8SC05212C</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=10.1039%2FC8SC05212C" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=30996949" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BC1MXivVOlu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=3573-3585&author=N.+Uhlenbrockauthor=S.+Smithauthor=J.+Weisnerauthor=I.+Landelauthor=M.+Lindemannauthor=T.+A.+Leauthor=J.+Hardickauthor=R.+Gontlaauthor=R.+Scheinpflugauthor=P.+Czodrowskiauthor=P.+Janningauthor=L.+Deptaauthor=L.+Quambuschauthor=M.+P.+M%C3%BCllerauthor=B.+Engelsauthor=D.+Rauh&title=Structural+and+chemical+insights+into+the+covalent-allosteric+inhibition+of+the+protein+kinase+Akt&doi=10.1039%2FC8SC05212C"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Structural and chemical insights into the covalent-allosteric inhibition of the protein kinase Akt</span></div><div class="casAuthors">Uhlenbrock, Niklas; Smith, Steven; Weisner, Joern; Landel, Ina; Lindemann, Marius; Le, Thien Anh; Hardick, Julia; Gontla, Rajesh; Scheinpflug, Rebekka; Czodrowski, Paul; Janning, Petra; Depta, Laura; Quambusch, Lena; Mueller, Matthias P.; Engels, Bernd; Rauh, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3573-3585</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The Ser/Thr kinase Akt (Protein Kinase B/PKB) is a master switch in cellular signal transduction pathways.  Its downstream signaling influences cell proliferation, cell growth, and apoptosis, rendering Akt a prominent drug target.  The unique activation mechanism of Akt involves a change of the relative orientation of its N-terminal pleckstrin homol. (PH) and the kinase domain and makes this kinase suitable for highly specific allosteric modulation.  Here we present a unique set of crystal structures of covalent-allosteric interdomain inhibitors in complex with full-length Akt and report the structure-based design, synthesis, biol. and pharmacol. evaluation of a focused library of these innovative inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxp6UFlZqI5bVg90H21EOLACvtfcHk0lgI_-H35YuvtQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXivVOlu7w%253D&md5=4ff6ee8b9d7f5e7ac9840158fafc32ef</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1039%2FC8SC05212C&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC8SC05212C%26sid%3Dliteratum%253Aachs%26aulast%3DUhlenbrock%26aufirst%3DN.%26aulast%3DSmith%26aufirst%3DS.%26aulast%3DWeisner%26aufirst%3DJ.%26aulast%3DLandel%26aufirst%3DI.%26aulast%3DLindemann%26aufirst%3DM.%26aulast%3DLe%26aufirst%3DT.%2BA.%26aulast%3DHardick%26aufirst%3DJ.%26aulast%3DGontla%26aufirst%3DR.%26aulast%3DScheinpflug%26aufirst%3DR.%26aulast%3DCzodrowski%26aufirst%3DP.%26aulast%3DJanning%26aufirst%3DP.%26aulast%3DDepta%26aufirst%3DL.%26aulast%3DQuambusch%26aufirst%3DL.%26aulast%3DM%25C3%25BCller%26aufirst%3DM.%2BP.%26aulast%3DEngels%26aufirst%3DB.%26aulast%3DRauh%26aufirst%3DD.%26atitle%3DStructural%2520and%2520chemical%2520insights%2520into%2520the%2520covalent-allosteric%2520inhibition%2520of%2520the%2520protein%2520kinase%2520Akt%26jtitle%3DChem.%2520Sci.%26date%3D2019%26volume%3D10%26spage%3D3573%26epage%3D3585%26doi%3D10.1039%2FC8SC05212C" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ostrem, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peters, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sos, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shokat, K. M.</span></span> <span> </span><span class="NLM_article-title">K-Ras(G12C) inhibitors allosterically control GTP affinity and effector interactions</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>503</i></span>,  <span class="NLM_fpage">548</span>– <span class="NLM_lpage">551</span>, <span class="refDoi"> DOI: 10.1038/nature12796</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=10.1038%2Fnature12796" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=24256730" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVGqs7jO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=503&publication_year=2013&pages=548-551&author=J.+M.+Ostremauthor=U.+Petersauthor=M.+L.+Sosauthor=J.+A.+Wellsauthor=K.+M.+Shokat&title=K-Ras%28G12C%29+inhibitors+allosterically+control+GTP+affinity+and+effector+interactions&doi=10.1038%2Fnature12796"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">K-Ras(G12C) inhibitors allosterically control GTP affinity and effector interactions</span></div><div class="casAuthors">Ostrem, Jonathan M.; Peters, Ulf; Sos, Martin L.; Wells, James A.; Shokat, Kevan M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">503</span>
        (<span class="NLM_cas:issue">7477</span>),
    <span class="NLM_cas:pages">548-551</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Somatic mutations in the small GTPase K-Ras are the most common activating lesions found in human cancer, and are generally assocd. with poor response to std. therapies.  Efforts to target this oncogene directly have faced difficulties owing to its picomolar affinity for GTP/GDP and the absence of known allosteric regulatory sites.  Oncogenic mutations result in functional activation of Ras family proteins by impairing GTP hydrolysis.  With diminished regulation by GTPase activity, the nucleotide state of Ras becomes more dependent on relative nucleotide affinity and concn.  This gives GTP an advantage over GDP and increases the proportion of active GTP-bound Ras.  Here we report the development of small mols. that irreversibly bind to a common oncogenic mutant, K-Ras(G12C).  These compds. rely on the mutant cysteine for binding and therefore do not affect the wild-type protein.  Crystallog. studies reveal the formation of a new pocket that is not apparent in previous structures of Ras, beneath the effector binding switch-II region.  Binding of these inhibitors to K-Ras(G12C) disrupts both switch-I and switch-II, subverting the native nucleotide preference to favor GDP over GTP and impairing binding to Raf.  Our data provide structure-based validation of a new allosteric regulatory site on Ras that is targetable in a mutant-specific manner.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokkgcVXuBn_rVg90H21EOLACvtfcHk0lht5O9PSaD7cg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvVGqs7jO&md5=27c1e7d8ef6b7e4b2919e9f00a073923</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fnature12796&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12796%26sid%3Dliteratum%253Aachs%26aulast%3DOstrem%26aufirst%3DJ.%2BM.%26aulast%3DPeters%26aufirst%3DU.%26aulast%3DSos%26aufirst%3DM.%2BL.%26aulast%3DWells%26aufirst%3DJ.%2BA.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26atitle%3DK-Ras%2528G12C%2529%2520inhibitors%2520allosterically%2520control%2520GTP%2520affinity%2520and%2520effector%2520interactions%26jtitle%3DNature%26date%3D2013%26volume%3D503%26spage%3D548%26epage%3D551%26doi%3D10.1038%2Fnature12796" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Janes, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peters, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Babbar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Firdaus, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darjania, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, Y. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thach, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zarieh, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ely, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kucharski, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kessler, L. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zarrinkar, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brehmer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhanak, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lorenzi, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu-Lowe, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patricelli, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span> <span> </span><span class="NLM_article-title">Targeting KRAS mutant cancers with a covalent G12C-specific inhibitor</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>172</i></span>,  <span class="NLM_fpage">578</span>– <span class="NLM_lpage">589.e17</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2018.01.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=10.1016%2Fj.cell.2018.01.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=29373830" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BC1cXislyisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=172&publication_year=2018&pages=578-589.e17&author=M.+R.+Janesauthor=J.+Zhangauthor=L.+S.+Liauthor=R.+Hansenauthor=U.+Petersauthor=X.+Guoauthor=Y.+Chenauthor=A.+Babbarauthor=S.+J.+Firdausauthor=L.+Darjaniaauthor=J.+Fengauthor=J.+H.+Chenauthor=S.+Liauthor=S.+Liauthor=Y.+O.+Longauthor=C.+Thachauthor=Y.+Liuauthor=A.+Zariehauthor=T.+Elyauthor=J.+M.+Kucharskiauthor=L.+V.+Kesslerauthor=T.+Wuauthor=K.+Yuauthor=Y.+Wangauthor=Y.+Yaoauthor=X.+Dengauthor=P.+P.+Zarrinkarauthor=D.+Brehmerauthor=D.+Dhanakauthor=M.+V.+Lorenziauthor=D.+Hu-Loweauthor=M.+P.+Patricelliauthor=P.+Renauthor=Y.+Liu&title=Targeting+KRAS+mutant+cancers+with+a+covalent+G12C-specific+inhibitor&doi=10.1016%2Fj.cell.2018.01.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting KRAS Mutant Cancers with a Covalent G12C-Specific Inhibitor</span></div><div class="casAuthors">Janes, Matthew R.; Zhang, Jingchuan; Li, Lian-Sheng; Hansen, Rasmus; Peters, Ulf; Guo, Xin; Chen, Yuching; Babbar, Anjali; Firdaus, Sarah J.; Darjania, Levan; Feng, Jun; Chen, Jeffrey H.; Li, Shuangwei; Li, Shisheng; Long, Yun O.; Thach, Carol; Liu, Yuan; Zarieh, Ata; Ely, Tess; Kucharski, Jeff M.; Kessler, Linda V.; Wu, Tao; Yu, Ke; Wang, Yi; Yao, Yvonne; Deng, Xiaohu; Zarrinkar, Patrick P.; Brehmer, Dirk; Dhanak, Dashyant; Lorenzi, Matthew V.; Hu-Lowe, Dana; Patricelli, Matthew P.; Ren, Pingda; Liu, Yi</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">172</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">578-589.e17</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">KRASG12C was recently identified to be potentially druggable by allele-specific covalent targeting of Cys 12 in vicinity to an inducible allosteric switch II pocket (S-IIP).  Success of this approach requires active cycling of KRASG12C between its active-GTP and inactive-GDP conformations as accessibility of the S-IIP is restricted only to the GDP-bound state.  This strategy proved feasible for inhibiting mutant KRAS in vitro; however, it is uncertain whether this approach would translate to in vivo.  Here, the authors describe structure-based design and identification of ARS-1620, a covalent compd. with high potency and selectivity for KRASG12C.  ARS-1620 achieves rapid and sustained in vivo target occupancy to induce tumor regression.  The authors use ARS-1620 to dissect oncogenic KRAS dependency and demonstrate that monolayer culture formats significantly underestimate KRAS dependency in vivo.  This study provides in vivo evidence that mutant KRAS can be selectively targeted and reveals ARS-1620 as representing a new generation of KRASG12C-specific inhibitors with promising therapeutic potential.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVmv4KAl4_dLVg90H21EOLACvtfcHk0lht5O9PSaD7cg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXislyisb8%253D&md5=d0883bde981b51e41a794179ffae87be</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2018.01.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2018.01.006%26sid%3Dliteratum%253Aachs%26aulast%3DJanes%26aufirst%3DM.%2BR.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DL.%2BS.%26aulast%3DHansen%26aufirst%3DR.%26aulast%3DPeters%26aufirst%3DU.%26aulast%3DGuo%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DBabbar%26aufirst%3DA.%26aulast%3DFirdaus%26aufirst%3DS.%2BJ.%26aulast%3DDarjania%26aufirst%3DL.%26aulast%3DFeng%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DJ.%2BH.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DLong%26aufirst%3DY.%2BO.%26aulast%3DThach%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DZarieh%26aufirst%3DA.%26aulast%3DEly%26aufirst%3DT.%26aulast%3DKucharski%26aufirst%3DJ.%2BM.%26aulast%3DKessler%26aufirst%3DL.%2BV.%26aulast%3DWu%26aufirst%3DT.%26aulast%3DYu%26aufirst%3DK.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DYao%26aufirst%3DY.%26aulast%3DDeng%26aufirst%3DX.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26aulast%3DBrehmer%26aufirst%3DD.%26aulast%3DDhanak%26aufirst%3DD.%26aulast%3DLorenzi%26aufirst%3DM.%2BV.%26aulast%3DHu-Lowe%26aufirst%3DD.%26aulast%3DPatricelli%26aufirst%3DM.%2BP.%26aulast%3DRen%26aufirst%3DP.%26aulast%3DLiu%26aufirst%3DY.%26atitle%3DTargeting%2520KRAS%2520mutant%2520cancers%2520with%2520a%2520covalent%2520G12C-specific%2520inhibitor%26jtitle%3DCell%26date%3D2018%26volume%3D172%26spage%3D578%26epage%3D589.e17%26doi%3D10.1016%2Fj.cell.2018.01.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Canon, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rex, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saiki, A. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohr, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooke, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bagal, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaida, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holt, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knutson, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koppada, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lanman, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rapaport, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">San Miguel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortiz, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osgood, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, J.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCarter, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Volak, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Houk, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fakih, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Neil, B. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Falchook, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Desai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Govindan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouyang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henary, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arvedson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cee, V. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lipford, J. R.</span></span> <span> </span><span class="NLM_article-title">The clinical KRAS(G12C) inhibitor AMG 510 drives anti-tumour immunity</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>575</i></span>,  <span class="NLM_fpage">217</span>– <span class="NLM_lpage">223</span>, <span class="refDoi"> DOI: 10.1038/s41586-019-1694-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=10.1038%2Fs41586-019-1694-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=31666701" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitV2hsb7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=575&publication_year=2019&pages=217-223&author=J.+Canonauthor=K.+Rexauthor=A.+Y.+Saikiauthor=C.+Mohrauthor=K.+Cookeauthor=D.+Bagalauthor=K.+Gaidaauthor=T.+Holtauthor=C.+G.+Knutsonauthor=N.+Koppadaauthor=B.+A.+Lanmanauthor=J.+Wernerauthor=A.+S.+Rapaportauthor=T.+San+Miguelauthor=R.+Ortizauthor=T.+Osgoodauthor=J.-R.+Sunauthor=X.+Zhuauthor=J.+D.+McCarterauthor=L.+P.+Volakauthor=B.+E.+Houkauthor=M.+G.+Fakihauthor=B.+H.+O%E2%80%99Neilauthor=T.+J.+Priceauthor=G.+S.+Falchookauthor=J.+Desaiauthor=J.+Kuoauthor=R.+Govindanauthor=D.+S.+Hongauthor=W.+Ouyangauthor=H.+Henaryauthor=T.+Arvedsonauthor=V.+J.+Ceeauthor=J.+R.+Lipford&title=The+clinical+KRAS%28G12C%29+inhibitor+AMG+510+drives+anti-tumour+immunity&doi=10.1038%2Fs41586-019-1694-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">The clinical KRAS(G12C) inhibitor AMG 510 drives anti-tumour immunity</span></div><div class="casAuthors">Canon, Jude; Rex, Karen; Saiki, Anne Y.; Mohr, Christopher; Cooke, Keegan; Bagal, Dhanashri; Gaida, Kevin; Holt, Tyler; Knutson, Charles G.; Koppada, Neelima; Lanman, Brian A.; Werner, Jonathan; Rapaport, Aaron S.; San Miguel, Tisha; Ortiz, Roberto; Osgood, Tao; Sun, Ji-Rong; Zhu, Xiaochun; McCarter, John D.; Volak, Laurie P.; Houk, Brett E.; Fakih, Marwan G.; O'Neil, Bert H.; Price, Timothy J.; Falchook, Gerald S.; Desai, Jayesh; Kuo, James; Govindan, Ramaswamy; Hong, David S.; Ouyang, Wenjun; Henary, Haby; Arvedson, Tara; Cee, Victor J.; Lipford, J. Russell</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">575</span>
        (<span class="NLM_cas:issue">7781</span>),
    <span class="NLM_cas:pages">217-223</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">KRAS is the most frequently mutated oncogene in cancer and encodes a key signalling protein in tumors1,2.  The KRAS(G12C) mutant has a cysteine residue that has been exploited to design covalent inhibitors that have promising preclin. activity3-5.  Here we optimized a series of inhibitors, using novel binding interactions to markedly enhance their potency and selectivity.  Our efforts have led to the discovery of AMG 510, which is, to our knowledge, the first KRAS(G12C) inhibitor in clin. development.  In preclin. analyses, treatment with AMG 510 led to the regression of KRASG12C tumors and improved the anti-tumor efficacy of chemotherapy and targeted agents.  In immune-competent mice, treatment with AMG 510 resulted in a pro-inflammatory tumor microenvironment and produced durable cures alone as well as in combination with immune-checkpoint inhibitors.  Cured mice rejected the growth of isogenic KRASG12D tumors, which suggests adaptive immunity against shared antigens.  Furthermore, in clin. trials, AMG 510 demonstrated anti-tumor activity in the first dosing cohorts and represents a potentially transformative therapy for patients for whom effective treatments are lacking.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTHdBftQfnhbVg90H21EOLACvtfcHk0ljnIXS0TM7qyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitV2hsb7F&md5=3819175980905976ba87ff545b9506e1</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fs41586-019-1694-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-019-1694-1%26sid%3Dliteratum%253Aachs%26aulast%3DCanon%26aufirst%3DJ.%26aulast%3DRex%26aufirst%3DK.%26aulast%3DSaiki%26aufirst%3DA.%2BY.%26aulast%3DMohr%26aufirst%3DC.%26aulast%3DCooke%26aufirst%3DK.%26aulast%3DBagal%26aufirst%3DD.%26aulast%3DGaida%26aufirst%3DK.%26aulast%3DHolt%26aufirst%3DT.%26aulast%3DKnutson%26aufirst%3DC.%2BG.%26aulast%3DKoppada%26aufirst%3DN.%26aulast%3DLanman%26aufirst%3DB.%2BA.%26aulast%3DWerner%26aufirst%3DJ.%26aulast%3DRapaport%26aufirst%3DA.%2BS.%26aulast%3DSan%2BMiguel%26aufirst%3DT.%26aulast%3DOrtiz%26aufirst%3DR.%26aulast%3DOsgood%26aufirst%3DT.%26aulast%3DSun%26aufirst%3DJ.-R.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DMcCarter%26aufirst%3DJ.%2BD.%26aulast%3DVolak%26aufirst%3DL.%2BP.%26aulast%3DHouk%26aufirst%3DB.%2BE.%26aulast%3DFakih%26aufirst%3DM.%2BG.%26aulast%3DO%25E2%2580%2599Neil%26aufirst%3DB.%2BH.%26aulast%3DPrice%26aufirst%3DT.%2BJ.%26aulast%3DFalchook%26aufirst%3DG.%2BS.%26aulast%3DDesai%26aufirst%3DJ.%26aulast%3DKuo%26aufirst%3DJ.%26aulast%3DGovindan%26aufirst%3DR.%26aulast%3DHong%26aufirst%3DD.%2BS.%26aulast%3DOuyang%26aufirst%3DW.%26aulast%3DHenary%26aufirst%3DH.%26aulast%3DArvedson%26aufirst%3DT.%26aulast%3DCee%26aufirst%3DV.%2BJ.%26aulast%3DLipford%26aufirst%3DJ.%2BR.%26atitle%3DThe%2520clinical%2520KRAS%2528G12C%2529%2520inhibitor%2520AMG%2520510%2520drives%2520anti-tumour%2520immunity%26jtitle%3DNature%26date%3D2019%26volume%3D575%26spage%3D217%26epage%3D223%26doi%3D10.1038%2Fs41586-019-1694-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lanman, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amegadzie, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashton, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Booker, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frohn, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodman, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kopecky, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Low, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minatti, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishimura, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pickrell, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reed, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siegmund, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamayo, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tegley, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walton, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wurz, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Achanta, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartberger, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Canon, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hollis, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCarter, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohr, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rex, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saiki, A. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">San Miguel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Volak, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whittington, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zech, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lipford, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cee, V. J.</span></span> <span> </span><span class="NLM_article-title">Discovery of a covalent inhibitor of KRAS<sup>G12C</sup>(AMG 510) for the treatment of solid tumors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">52</span>– <span class="NLM_lpage">65</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01180</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01180" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitlGitrrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=52-65&author=B.+A.+Lanmanauthor=J.+R.+Allenauthor=J.+G.+Allenauthor=A.+K.+Amegadzieauthor=K.+S.+Ashtonauthor=S.+K.+Bookerauthor=J.+J.+Chenauthor=N.+Chenauthor=M.+J.+Frohnauthor=G.+Goodmanauthor=D.+J.+Kopeckyauthor=L.+Liuauthor=P.+Lopezauthor=J.+D.+Lowauthor=V.+Maauthor=A.+E.+Minattiauthor=T.+T.+Nguyenauthor=N.+Nishimuraauthor=A.+J.+Pickrellauthor=A.+B.+Reedauthor=Y.+Shinauthor=A.+C.+Siegmundauthor=N.+A.+Tamayoauthor=C.+M.+Tegleyauthor=M.+C.+Waltonauthor=H.+L.+Wangauthor=R.+P.+Wurzauthor=M.+Xueauthor=K.+C.+Yangauthor=P.+Achantaauthor=M.+D.+Bartbergerauthor=J.+Canonauthor=L.+S.+Hollisauthor=J.+D.+McCarterauthor=C.+Mohrauthor=K.+Rexauthor=A.+Y.+Saikiauthor=T.+San+Miguelauthor=L.+P.+Volakauthor=K.+H.+Wangauthor=D.+A.+Whittingtonauthor=S.+G.+Zechauthor=J.+R.+Lipfordauthor=V.+J.+Cee&title=Discovery+of+a+covalent+inhibitor+of+KRASG12C%28AMG+510%29+for+the+treatment+of+solid+tumors&doi=10.1021%2Facs.jmedchem.9b01180"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Covalent Inhibitor of KRASG12C (AMG 510) for the Treatment of Solid Tumors</span></div><div class="casAuthors">Lanman, Brian A.; Allen, Jennifer R.; Allen, John G.; Amegadzie, Albert K.; Ashton, Kate S.; Booker, Shon K.; Chen, Jian Jeffrey; Chen, Ning; Frohn, Michael J.; Goodman, Guy; Kopecky, David J.; Liu, Longbin; Lopez, Patricia; Low, Jonathan D.; Ma, Vu; Minatti, Ana E.; Nguyen, Thomas T.; Nishimura, Nobuko; Pickrell, Alexander J.; Reed, Anthony B.; Shin, Youngsook; Siegmund, Aaron C.; Tamayo, Nuria A.; Tegley, Christopher M.; Walton, Mary C.; Wang, Hui-Ling; Wurz, Ryan P.; Xue, May; Yang, Kevin C.; Achanta, Pragathi; Bartberger, Michael D.; Canon, Jude; Hollis, L. Steven; McCarter, John D.; Mohr, Christopher; Rex, Karen; Saiki, Anne Y.; San Miguel, Tisha; Volak, Laurie P.; Wang, Kevin H.; Whittington, Douglas A.; Zech, Stephan G.; Lipford, J. Russell; Cee, Victor J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">52-65</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">KRASG12C has emerged as a promising target in the treatment of solid tumors.  Covalent inhibitors targeting the mutant cysteine-12 residue have been shown to disrupt signaling by this long-"undruggable" target; however clin. viable inhibitors have yet to be identified.  Here, we report efforts to exploit a cryptic pocket (H95/Y96/Q99) we identified in KRASG12C to identify inhibitors suitable for clin. development.  Structure-based design efforts leading to the identification of a novel quinazolinone scaffold are described, along with optimization efforts that overcame a configurational stability issue arising from restricted rotation about an axially chiral biaryl bond.  Biopharmaceutical optimization of the resulting leads culminated in the identification of AMG 510, a highly potent, selective, and well-tolerated KRASG12C inhibitor currently in phase I clin. trials (NCT03600883).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGeWqRRXX6lbVg90H21EOLACvtfcHk0ljnIXS0TM7qyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitlGitrrK&md5=d973189be70bb913020edb4989f6ba7d</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01180&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01180%26sid%3Dliteratum%253Aachs%26aulast%3DLanman%26aufirst%3DB.%2BA.%26aulast%3DAllen%26aufirst%3DJ.%2BR.%26aulast%3DAllen%26aufirst%3DJ.%2BG.%26aulast%3DAmegadzie%26aufirst%3DA.%2BK.%26aulast%3DAshton%26aufirst%3DK.%2BS.%26aulast%3DBooker%26aufirst%3DS.%2BK.%26aulast%3DChen%26aufirst%3DJ.%2BJ.%26aulast%3DChen%26aufirst%3DN.%26aulast%3DFrohn%26aufirst%3DM.%2BJ.%26aulast%3DGoodman%26aufirst%3DG.%26aulast%3DKopecky%26aufirst%3DD.%2BJ.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DLopez%26aufirst%3DP.%26aulast%3DLow%26aufirst%3DJ.%2BD.%26aulast%3DMa%26aufirst%3DV.%26aulast%3DMinatti%26aufirst%3DA.%2BE.%26aulast%3DNguyen%26aufirst%3DT.%2BT.%26aulast%3DNishimura%26aufirst%3DN.%26aulast%3DPickrell%26aufirst%3DA.%2BJ.%26aulast%3DReed%26aufirst%3DA.%2BB.%26aulast%3DShin%26aufirst%3DY.%26aulast%3DSiegmund%26aufirst%3DA.%2BC.%26aulast%3DTamayo%26aufirst%3DN.%2BA.%26aulast%3DTegley%26aufirst%3DC.%2BM.%26aulast%3DWalton%26aufirst%3DM.%2BC.%26aulast%3DWang%26aufirst%3DH.%2BL.%26aulast%3DWurz%26aufirst%3DR.%2BP.%26aulast%3DXue%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DK.%2BC.%26aulast%3DAchanta%26aufirst%3DP.%26aulast%3DBartberger%26aufirst%3DM.%2BD.%26aulast%3DCanon%26aufirst%3DJ.%26aulast%3DHollis%26aufirst%3DL.%2BS.%26aulast%3DMcCarter%26aufirst%3DJ.%2BD.%26aulast%3DMohr%26aufirst%3DC.%26aulast%3DRex%26aufirst%3DK.%26aulast%3DSaiki%26aufirst%3DA.%2BY.%26aulast%3DSan%2BMiguel%26aufirst%3DT.%26aulast%3DVolak%26aufirst%3DL.%2BP.%26aulast%3DWang%26aufirst%3DK.%2BH.%26aulast%3DWhittington%26aufirst%3DD.%2BA.%26aulast%3DZech%26aufirst%3DS.%2BG.%26aulast%3DLipford%26aufirst%3DJ.%2BR.%26aulast%3DCee%26aufirst%3DV.%2BJ.%26atitle%3DDiscovery%2520of%2520a%2520covalent%2520inhibitor%2520of%2520KRASG12C%2528AMG%2520510%2529%2520for%2520the%2520treatment%2520of%2520solid%2520tumors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D52%26epage%3D65%26doi%3D10.1021%2Facs.jmedchem.9b01180" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wan, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of novel allosteric site and covalent inhibitors of FBPase with potent hypoglycemic effects</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>184</i></span>,  <span class="NLM_fpage">111749</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2019.111749</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=10.1016%2Fj.ejmech.2019.111749" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=31589992" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvFert7jK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=184&publication_year=2019&pages=111749&author=Y.+Huangauthor=L.+Weiauthor=X.+Hanauthor=H.+Chenauthor=Y.+Renauthor=Y.+Xuauthor=R.+Songauthor=L.+Raoauthor=C.+Suauthor=C.+Pengauthor=L.+Fengauthor=J.+Wan&title=Discovery+of+novel+allosteric+site+and+covalent+inhibitors+of+FBPase+with+potent+hypoglycemic+effects&doi=10.1016%2Fj.ejmech.2019.111749"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of novel allosteric site and covalent inhibitors of FBPase with potent hypoglycemic effects</span></div><div class="casAuthors">Huang, Yunyuan; Wei, Lin; Han, Xinya; Chen, Haifeng; Ren, Yanliang; Xu, Yanhong; Song, Rongrong; Rao, Li; Su, Chen; Peng, Chao; Feng, Lingling; Wan, Jian</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">184</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">111749</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Fructose-1,6-bisphosphatase (FBPase) is an essential enzyme of GNG pathway.  Significant advances demonstrate the FBPase plays a crit. role in treatment of diabetes.  Numerous FBPase inhibitors were developed by targeting AMP site, nevertheless, none of these inhibitors has exhibited suitable potency and druggability.  Herein, a new allosteric site (C128) on FBPase was discovered, and several nitrostyrene compds. exhibiting potent FBPase inhibitions were found covalently bind to C128 site on FBPase.  Mutagenesis suggest that C128 is the only cysteine that can influence FBPase inhibition, the N125-S124-S123 pathway was most likely involved in allosteric signaling transmission between C128 and active site.  However, these nitrostyrenes may bind with multiple cysteine besides C128 in FBPase.  To improve pocket selectivity, a series of novel compds. (14a-14n) were re-designed rationally by integrating fragment-based covalent virtual screening and machine-learning-based synthetic complexity evaluation.  As expected, the mass spectrometry validated that the proportion of title compds. binding to the C128 in FBPase was significantly higher than that of nitrostyrenes.  Notably, under physiol. and pathol. conditions, the treatment of compds. 14b(I), 14c(II), 14i(III) or 14n(IV) led to potent inhibition of glucose prodn., as well as decreased triglyceride and total cholesterol levels in mouse primary hepatocytes.  We highlight a novel paradigm that mol. targeting C128 site on FBPase can have potent hypoglycemic effect.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4rA2SCokJuLVg90H21EOLACvtfcHk0lh6TnIDHUAOFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvFert7jK&md5=a3c8f7ca2413e19243e1ad424676135a</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2019.111749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2019.111749%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DWei%26aufirst%3DL.%26aulast%3DHan%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DRen%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DSong%26aufirst%3DR.%26aulast%3DRao%26aufirst%3DL.%26aulast%3DSu%26aufirst%3DC.%26aulast%3DPeng%26aufirst%3DC.%26aulast%3DFeng%26aufirst%3DL.%26aulast%3DWan%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520novel%2520allosteric%2520site%2520and%2520covalent%2520inhibitors%2520of%2520FBPase%2520with%2520potent%2520hypoglycemic%2520effects%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2019%26volume%3D184%26spage%3D111749%26doi%3D10.1016%2Fj.ejmech.2019.111749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Veverka, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mays, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lipsky, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naylor, S.</span></span> <span> </span><span class="NLM_article-title">Inhibition of aldehyde dehydrogenase by disulfiram and its metabolite methyl diethylthiocarbamoyl-sulfoxide</span>. <i>Biochem. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">511</span>– <span class="NLM_lpage">518</span>, <span class="refDoi"> DOI: 10.1016/S0006-2952(96)00767-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=10.1016%2FS0006-2952%2896%2900767-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=9105402" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADyaK2sXitVajsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=511-518&author=K.+A.+Veverkaauthor=K.+L.+Johnsonauthor=D.+C.+Maysauthor=J.+J.+Lipskyauthor=S.+Naylor&title=Inhibition+of+aldehyde+dehydrogenase+by+disulfiram+and+its+metabolite+methyl+diethylthiocarbamoyl-sulfoxide&doi=10.1016%2FS0006-2952%2896%2900767-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of aldehyde dehydrogenase by disulfiram and its metabolite methyl diethylthiocarbamoyl-sulfoxide</span></div><div class="casAuthors">Veverka, Karen A.; Johnson, Kenneth L.; Mays, Dennis C.; Lipsky, James J.; Naylor, Stephen</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Pharmacology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">511-518</span>CODEN:
                <span class="NLM_cas:coden">BCPCA6</span>;
        ISSN:<span class="NLM_cas:issn">0006-2952</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Disulfiram (DSF) is presently the only available drug used in the aversion therapy of recovering alcoholics.  It acts by inhibiting aldehyde dehydrogenase (ALDH), leading to high blood levels of acetaldehyde.  The in vitro inhibition of ALDH by DSF and its metabolites was systematically studied by combined enzyme inhibition assay with direct mol. wt. detn. of the same sample using electrospray ionization-mass spectrometry (ESI-MS).  Enzyme activity was measured after incubating yeast ALDH (yALDH) with excess concns. of DSF, Me diethyldithiocarbamate (MeDDC) and Me diethylthiocarbamoyl-sulfoxide (MeDTC-SO) and then subjected to anal. by ESI-MS.  Addn. of DSF resulted in complete enzyme inhibition; however, ESI-MS anal. demonstrated no discernible shift in mol. wt., indicating that no intermol. adduct was formed with the protein.  Treatment of yALDH with MeDTC-SO also completely abolished yALDH activity with a concomitant increase of +≈100 Da in the mol. mass of the enzyme.  This indicated formation of a covalent carbamoyl protein adduct.  Furthermore, the effects of dithiothreitol (DTT) were examd. on samples of inhibited protein in vitro.  At pH 7.5, DDT completely reversed inhibition after DSF treatment.  YALDH inhibited by MeDTC-SO could not be recovered by DTT at pH 7.5, but at pH 9 the enzymic activity was fully restored and a mass loss of ∼100 Da was noted.  This observations are consistent with mechanisms where inhibition of yALDH by DSF in vitro involves oxidn. of the active site, whereas MeDTC-SO forms a covalent adduct with the protein in vitro resulting in cessation of enzyme activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWw03Q2vgZmbVg90H21EOLACvtfcHk0lh6TnIDHUAOFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXitVajsbY%253D&md5=7893c1c4a675f7d94fea0b143434e4c3</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2FS0006-2952%2896%2900767-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0006-2952%252896%252900767-8%26sid%3Dliteratum%253Aachs%26aulast%3DVeverka%26aufirst%3DK.%2BA.%26aulast%3DJohnson%26aufirst%3DK.%2BL.%26aulast%3DMays%26aufirst%3DD.%2BC.%26aulast%3DLipsky%26aufirst%3DJ.%2BJ.%26aulast%3DNaylor%26aufirst%3DS.%26atitle%3DInhibition%2520of%2520aldehyde%2520dehydrogenase%2520by%2520disulfiram%2520and%2520its%2520metabolite%2520methyl%2520diethylthiocarbamoyl-sulfoxide%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D1997%26volume%3D53%26spage%3D511%26epage%3D518%26doi%3D10.1016%2FS0006-2952%2896%2900767-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Skrott, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mistrik, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andersen, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friis, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majera, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gursky, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozdian, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartkova, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turi, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moudry, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraus, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michalova, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaclavkova, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dzubak, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vrobel, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pouckova, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sedlacek, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miklovicova, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kutt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mattova, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Driessen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dou, Q. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olsen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hajduch, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cvek, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshaies, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartek, J.</span></span> <span> </span><span class="NLM_article-title">Alcohol-abuse drug disulfiram targets cancer via p97 segregase adaptor NPL4</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>552</i></span>,  <span class="NLM_fpage">194</span>– <span class="NLM_lpage">199</span>, <span class="refDoi"> DOI: 10.1038/nature25016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=10.1038%2Fnature25016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=29211715" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFGlsbrN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=552&publication_year=2017&pages=194-199&author=Z.+Skrottauthor=M.+Mistrikauthor=K.+K.+Andersenauthor=S.+Friisauthor=D.+Majeraauthor=J.+Gurskyauthor=T.+Ozdianauthor=J.+Bartkovaauthor=Z.+Turiauthor=P.+Moudryauthor=M.+Krausauthor=M.+Michalovaauthor=J.+Vaclavkovaauthor=P.+Dzubakauthor=I.+Vrobelauthor=P.+Pouckovaauthor=J.+Sedlacekauthor=A.+Miklovicovaauthor=A.+Kuttauthor=J.+Liauthor=J.+Mattovaauthor=C.+Driessenauthor=Q.+P.+Douauthor=J.+Olsenauthor=M.+Hajduchauthor=B.+Cvekauthor=R.+J.+Deshaiesauthor=J.+Bartek&title=Alcohol-abuse+drug+disulfiram+targets+cancer+via+p97+segregase+adaptor+NPL4&doi=10.1038%2Fnature25016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Alcohol-abuse drug disulfiram targets cancer via p97 segregase adaptor NPL4</span></div><div class="casAuthors">Skrott, Zdenek; Mistrik, Martin; Andersen, Klaus Kaae; Friis, Soeren; Majera, Dusana; Gursky, Jan; Ozdian, Tomas; Bartkova, Jirina; Turi, Zsofia; Moudry, Pavel; Kraus, Marianne; Michalova, Martina; Vaclavkova, Jana; Dzubak, Petr; Vrobel, Ivo; Pouckova, Pavla; Sedlacek, Jindrich; Miklovicova, Andrea; Kutt, Anne; Li, Jing; Mattova, Jana; Driessen, Christoph; Dou, Q. Ping; Olsen, Joergen; Hajduch, Marian; Cvek, Boris; Deshaies, Raymond J.; Bartek, Jiri</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">552</span>
        (<span class="NLM_cas:issue">7684</span>),
    <span class="NLM_cas:pages">194-199</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Cancer incidence is rising and this global challenge is further exacerbated by tumor resistance to available medicines.  A promising approach to meet the need for improved cancer treatment is drug repurposing.  Here we highlight the potential for repurposing disulfiram (also known by the trade name Antabuse), an old alc.-aversion drug that has been shown to be effective against diverse cancer types in preclin. studies.  Our nationwide epidemiol. study reveals that patients who continuously used disulfiram have a lower risk of death from cancer compared to those who stopped using the drug at their diagnosis.  Moreover, we identify the ditiocarb-copper complex as the metabolite of disulfiram that is responsible for its anti-cancer effects, and provide methods to detect preferential accumulation of the complex in tumors and candidate biomarkers to analyze its effect on cells and tissues.  Finally, our functional and biophys. analyses reveal the mol. target of disulfiram's tumor-suppressing effects as NPL4, an adaptor of p97 (also known as VCP) segregase, which is essential for the turnover of proteins involved in multiple regulatory and stress-response pathways in cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqp_sAn4LKB57Vg90H21EOLACvtfcHk0li0BazpNw0vIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFGlsbrN&md5=b53235723e8478ab49b7d569d0dd35ac</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1038%2Fnature25016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature25016%26sid%3Dliteratum%253Aachs%26aulast%3DSkrott%26aufirst%3DZ.%26aulast%3DMistrik%26aufirst%3DM.%26aulast%3DAndersen%26aufirst%3DK.%2BK.%26aulast%3DFriis%26aufirst%3DS.%26aulast%3DMajera%26aufirst%3DD.%26aulast%3DGursky%26aufirst%3DJ.%26aulast%3DOzdian%26aufirst%3DT.%26aulast%3DBartkova%26aufirst%3DJ.%26aulast%3DTuri%26aufirst%3DZ.%26aulast%3DMoudry%26aufirst%3DP.%26aulast%3DKraus%26aufirst%3DM.%26aulast%3DMichalova%26aufirst%3DM.%26aulast%3DVaclavkova%26aufirst%3DJ.%26aulast%3DDzubak%26aufirst%3DP.%26aulast%3DVrobel%26aufirst%3DI.%26aulast%3DPouckova%26aufirst%3DP.%26aulast%3DSedlacek%26aufirst%3DJ.%26aulast%3DMiklovicova%26aufirst%3DA.%26aulast%3DKutt%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DMattova%26aufirst%3DJ.%26aulast%3DDriessen%26aufirst%3DC.%26aulast%3DDou%26aufirst%3DQ.%2BP.%26aulast%3DOlsen%26aufirst%3DJ.%26aulast%3DHajduch%26aufirst%3DM.%26aulast%3DCvek%26aufirst%3DB.%26aulast%3DDeshaies%26aufirst%3DR.%2BJ.%26aulast%3DBartek%26aufirst%3DJ.%26atitle%3DAlcohol-abuse%2520drug%2520disulfiram%2520targets%2520cancer%2520via%2520p97%2520segregase%2520adaptor%2520NPL4%26jtitle%3DNature%26date%3D2017%26volume%3D552%26spage%3D194%26epage%3D199%26doi%3D10.1038%2Fnature25016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bao, B. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, G. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, F. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J. G.</span></span> <span> </span><span class="NLM_article-title">Discovery of unsymmetrical aromatic disulfides as novel inhibitors of SARS-CoV main protease: Chemical synthesis, biological evaluation, molecular docking and 3D-QSAR study</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>137</i></span>,  <span class="NLM_fpage">450</span>– <span class="NLM_lpage">461</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2017.05.045</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=10.1016%2Fj.ejmech.2017.05.045" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=28624700" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVertbrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=137&publication_year=2017&pages=450-461&author=L.+Wangauthor=B.+B.+Baoauthor=G.+Q.+Songauthor=C.+Chenauthor=X.+M.+Zhangauthor=W.+Luauthor=Z.+Wangauthor=Y.+Caiauthor=S.+Liauthor=S.+Fuauthor=F.+H.+Songauthor=H.+Yangauthor=J.+G.+Wang&title=Discovery+of+unsymmetrical+aromatic+disulfides+as+novel+inhibitors+of+SARS-CoV+main+protease%3A+Chemical+synthesis%2C+biological+evaluation%2C+molecular+docking+and+3D-QSAR+study&doi=10.1016%2Fj.ejmech.2017.05.045"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of unsymmetrical aromatic disulfides as novel inhibitors of SARS-CoV main protease: Chemical synthesis, biological evaluation, molecular docking and 3D-QSAR study</span></div><div class="casAuthors">Wang, Li; Bao, Bo-Bo; Song, Guo-Qing; Chen, Cheng; Zhang, Xu-Meng; Lu, Wei; Wang, Zefang; Cai, Yan; Li, Shuang; Fu, Sheng; Song, Fu-Hang; Yang, Haitao; Wang, Jian-Guo</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">137</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">450-461</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">The worldwide outbreak of severe acute respiratory syndrome (SARS) in 2003 had caused a high rate of mortality.  Main protease (Mpro) of SARS-assocd. coronavirus (SARS-CoV) is an important target to discover pharmaceutical compds. for the therapy of this life-threatening disease.  During the course of screening new anti-SARS agents, a series of unsym. arom. disulfides inhibited SARS-CoV Mpro significantly for the first time have been identified.  Novel unsym. arom. disulfides R1SSR2 [R1 = 1,3-thiazol-2-ylsulfanyl, 1,3,4-oxadiazol-2-ylsulfanyl, pyrimidin-2-ylsulfanyl, etc.; R2 = phenylsulfanyl, 4-bromophenylsulfanyl, [2-(methoxycarbonyl)phenyl]sulfanyl, etc.] were synthesized chem. and their biol. activities were evaluated in vitro against SARS-CoV Mpro.  These novel compds. displayed excellent IC50 data in the range of 0.516-5.954 μM.  Preliminary studies indicated that these disulfides are reversible and competitive inhibitors.  A possible binding mode was generated via mol. docking simulation and a comparative field anal. (CoMFA) model was constructed to understand the structure-activity relationships.  The present research therefore has provided some meaningful guidance to design and identify anti-SARS drugs with totally new chem. structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqd14DP7qY8zrVg90H21EOLACvtfcHk0li0BazpNw0vIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVertbrO&md5=07198a9f4ec7718a9f2299a3184dc8c4</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2017.05.045&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2017.05.045%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.%26aulast%3DBao%26aufirst%3DB.%2BB.%26aulast%3DSong%26aufirst%3DG.%2BQ.%26aulast%3DChen%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DX.%2BM.%26aulast%3DLu%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DCai%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DFu%26aufirst%3DS.%26aulast%3DSong%26aufirst%3DF.%2BH.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DJ.%2BG.%26atitle%3DDiscovery%2520of%2520unsymmetrical%2520aromatic%2520disulfides%2520as%2520novel%2520inhibitors%2520of%2520SARS-CoV%2520main%2520protease%253A%2520Chemical%2520synthesis%252C%2520biological%2520evaluation%252C%2520molecular%2520docking%2520and%25203D-QSAR%2520study%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2017%26volume%3D137%26spage%3D450%26epage%3D461%26doi%3D10.1016%2Fj.ejmech.2017.05.045" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hallenbeck, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turner, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renslo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arkin, M.</span></span> <span> </span><span class="NLM_article-title">Targeting non-catalytic cysteine residues through structure-guided drug discovery</span>. <i>Curr. Top. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">4</span>– <span class="NLM_lpage">15</span>, <span class="refDoi"> DOI: 10.2174/1568026616666160719163839</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=10.2174%2F1568026616666160719163839" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=27449257" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFCmurzK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2017&pages=4-15&author=K.+Hallenbeckauthor=D.+Turnerauthor=A.+Rensloauthor=M.+Arkin&title=Targeting+non-catalytic+cysteine+residues+through+structure-guided+drug+discovery&doi=10.2174%2F1568026616666160719163839"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Non-Catalytic Cysteine Residues Through Structure-Guided Drug Discovery</span></div><div class="casAuthors">Hallenbeck, Kenneth K.; Turner, David M.; Renslo, Adam R.; Arkin, Michelle R.</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">4-15</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  The targeting of non-catalytic cysteine residues with small mols. is drawing increased attention from drug discovery scientists and chem. biologists.  From a biol. perspective, genomic and proteomic studies have revealed the presence of cysteine mutations in several oncogenic proteins, suggesting both a functional role for these residues and also a strategy for targeting them in an 'allele specific' manner.  For the medicinal chemist, the structure-guided design of cysteine- reactive mols. is an appealing strategy to realize improved selectivity and pharmacodynamic properties in drug leads.  Finally, for chem. biologists, the modification of cysteine residues provides a unique means to probe protein structure and allosteric regulation.  Here, we review three applications of cysteinemodifying small mols.: 1. the optimization of existing drug leads, 2. the discovery of new lead compds., and 3. the use of cysteine-reactive mols. as probes of protein dynamics.  In each case, structure-guided design plays a key role in detg. which cysteine residue(s) to target and in designing compds. with the proper geometry to enable both covalent interaction with the targeted cysteine and productive non-covalent interactions with nearby protein residues.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdyF3jwe8kzLVg90H21EOLACvtfcHk0li0BazpNw0vIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFCmurzK&md5=adbdf5b5afd5fc53ba0f360e695b6fa5</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.2174%2F1568026616666160719163839&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1568026616666160719163839%26sid%3Dliteratum%253Aachs%26aulast%3DHallenbeck%26aufirst%3DK.%26aulast%3DTurner%26aufirst%3DD.%26aulast%3DRenslo%26aufirst%3DA.%26aulast%3DArkin%26aufirst%3DM.%26atitle%3DTargeting%2520non-catalytic%2520cysteine%2520residues%2520through%2520structure-guided%2520drug%2520discovery%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2017%26volume%3D17%26spage%3D4%26epage%3D15%26doi%3D10.2174%2F1568026616666160719163839" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacCoss, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawson, A. D. G.</span></span> <span> </span><span class="NLM_article-title">Rings in drugs</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">5845</span>– <span class="NLM_lpage">5859</span>, <span class="refDoi"> DOI: 10.1021/jm4017625</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm4017625" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlCntbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=5845-5859&author=R.+D.+Taylorauthor=M.+MacCossauthor=A.+D.+G.+Lawson&title=Rings+in+drugs&doi=10.1021%2Fjm4017625"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Rings in Drugs</span></div><div class="casAuthors">Taylor, Richard D.; MacCoss, Malcolm; Lawson, Alastair D. G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">5845-5859</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  We have analyzed the rings, ring systems, and frameworks in drugs listed in the FDA Orange Book to understand the frequency, time-lines, mol. property space, and the application of these rings in different therapeutic areas and target classes.  This anal. shows that there are only 351 ring systems and 1197 frameworks in drugs that came onto the market before 2013.  Furthermore, on av. six new ring systems enter drug space each year and approx. 28% of new drugs contain a new ring system.  Moreover, it is very unusual for a drug to contain more than one new ring system and the majority of the most frequently used ring systems (83%) were first used in drugs developed prior to 1983.  These observations give insight into the chem. novelty of drugs and potentially efficient ways to assess compd. libraries and develop compds. from hit identification to lead optimization and beyond.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoC_e_L4z4XCrVg90H21EOLACvtfcHk0lgsxAPVZJwQpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlCntbc%253D&md5=c84ce84b8698eb6fad38fa82b0ab4912</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fjm4017625&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm4017625%26sid%3Dliteratum%253Aachs%26aulast%3DTaylor%26aufirst%3DR.%2BD.%26aulast%3DMacCoss%26aufirst%3DM.%26aulast%3DLawson%26aufirst%3DA.%2BD.%2BG.%26atitle%3DRings%2520in%2520drugs%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D5845%26epage%3D5859%26doi%3D10.1021%2Fjm4017625" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span> <span> </span><span class="NLM_article-title">Development of the first generation of disulfide-based subtype-selective and potent covalent pyruvate dehydrogenase kinase 1 (PDK1) inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2227</span>– <span class="NLM_lpage">2244</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01245</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01245" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjtlyjtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=2227-2244&author=Y.+Liuauthor=Z.+Xieauthor=D.+Zhaoauthor=J.+Zhuauthor=F.+Maoauthor=S.+Tangauthor=H.+Xuauthor=C.+Luoauthor=M.+Gengauthor=M.+Huangauthor=J.+Li&title=Development+of+the+first+generation+of+disulfide-based+subtype-selective+and+potent+covalent+pyruvate+dehydrogenase+kinase+1+%28PDK1%29+inhibitors&doi=10.1021%2Facs.jmedchem.6b01245"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Development of the First Generation of Disulfide-Based Subtype-Selective and Potent Covalent Pyruvate Dehydrogenase Kinase 1 (PDK1) Inhibitors</span></div><div class="casAuthors">Liu, Yifu; Xie, Zuoquan; Zhao, Dan; Zhu, Jin; Mao, Fei; Tang, Shuai; Xu, Hui; Luo, Cheng; Geng, Meiyu; Huang, Min; Li, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2227-2244</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Pyruvate dehydrogenase kinases (PDKs) are overexpressed in most cancer cells and are responsible for aberrant glucose metab.  The authors previously described Bis(4-morpholinyl thiocarbonyl)-disulfide (JX06, 16) as the first covalent inhibitor of PDK1.  Here, based on the scaffold of JX06, the authors identify two novel types of disulfide-based PDK1 inhibitors.  The most potent analog, bis(1,4-oxazepan-4-yl-thiocarbonyl)disulfide (3a), effectively inhibits PDK1 both at the mol. (kinact/Ki = 4.17 × 103 M-1s-1) and the cellular level (down to 0.1 μM).  In contrast to JX06, 3a is a potent and subtype-selective inhibitor of PDK1 with >40-fold selectivity for PDK2-4.  Compd. 3a also significantly alters glucose metabolic pathways in A549 cells by decreasing ECAR and increasing ROS.  Moreover, in the xenograft models, 3a shows significant antitumor activity with no neg. effect to the mice wt.  Collectively, these data demonstrate that 3a may be an excellent lead compd. for the treatment of cancer as a first-generation subtype-selective and covalent PDK1 inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEpZNCUKK6MrVg90H21EOLACvtfcHk0lgsxAPVZJwQpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjtlyjtr0%253D&md5=7529ce2fc9d1a138a8d1405a23f7e4e8</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01245&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01245%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DXie%26aufirst%3DZ.%26aulast%3DZhao%26aufirst%3DD.%26aulast%3DZhu%26aufirst%3DJ.%26aulast%3DMao%26aufirst%3DF.%26aulast%3DTang%26aufirst%3DS.%26aulast%3DXu%26aufirst%3DH.%26aulast%3DLuo%26aufirst%3DC.%26aulast%3DGeng%26aufirst%3DM.%26aulast%3DHuang%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DJ.%26atitle%3DDevelopment%2520of%2520the%2520first%2520generation%2520of%2520disulfide-based%2520subtype-selective%2520and%2520potent%2520covalent%2520pyruvate%2520dehydrogenase%2520kinase%25201%2520%2528PDK1%2529%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D2227%26epage%3D2244%26doi%3D10.1021%2Facs.jmedchem.6b01245" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pietrucci, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laio, A.</span></span> <span> </span><span class="NLM_article-title">A collective variable for the efficient exploration of protein beta-sheet structures: application to SH3 and GB1</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">2197</span>– <span class="NLM_lpage">2201</span>, <span class="refDoi"> DOI: 10.1021/ct900202f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct900202f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BD1MXptleltbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=2197-2201&author=F.+Pietrucciauthor=A.+Laio&title=A+collective+variable+for+the+efficient+exploration+of+protein+beta-sheet+structures%3A+application+to+SH3+and+GB1&doi=10.1021%2Fct900202f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">A Collective Variable for the Efficient Exploration of Protein Beta-Sheet Structures: Application to SH3 and GB1</span></div><div class="casAuthors">Pietrucci, Fabio; Laio, Alessandro</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2197-2201</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The authors introduce a new class of collective variables which allow forming efficiently beta-sheet structures in all-atom explicit-solvent simulations of proteins.  By this approach the authors are able to systematically fold a 16-residue beta hairpin using metadynamics on a single replica.  Application to the 56-residue SH3 and GB1 proteins show that, starting from extended states, in ∼100 ns tens of structures contg. more than 30% beta-sheet are obtained, including parts of the native fold.  Using these variables may allow folding moderate size proteins with an accurate explicit solvent description.  Moreover, it may allow investigating the presence of misfolded states that are relevant for diseases (e.g., prion and Alzheimer) and studying beta-aggregation (amyloid diseases).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUydogD9f81LVg90H21EOLACvtfcHk0lgsxAPVZJwQpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXptleltbo%253D&md5=3d937a81ee3db1084794d47b96b41e35</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fct900202f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct900202f%26sid%3Dliteratum%253Aachs%26aulast%3DPietrucci%26aufirst%3DF.%26aulast%3DLaio%26aufirst%3DA.%26atitle%3DA%2520collective%2520variable%2520for%2520the%2520efficient%2520exploration%2520of%2520protein%2520beta-sheet%2520structures%253A%2520application%2520to%2520SH3%2520and%2520GB1%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2009%26volume%3D5%26spage%3D2197%26epage%3D2201%26doi%3D10.1021%2Fct900202f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lalau, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Race, J. M.</span></span> <span> </span><span class="NLM_article-title">Lactic acidosis in metformin-treated patients. Prognostic value of arterial lactate levels and plasma metformin concentrations</span>. <i>Drug Saf.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">377</span>– <span class="NLM_lpage">384</span>, <span class="refDoi"> DOI: 10.2165/00002018-199920040-00006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=10.2165%2F00002018-199920040-00006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=10230584" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADyaK1MXjtVGhs74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=1999&pages=377-384&author=J.+D.+Lalauauthor=J.+M.+Race&title=Lactic+acidosis+in+metformin-treated+patients.+Prognostic+value+of+arterial+lactate+levels+and+plasma+metformin+concentrations&doi=10.2165%2F00002018-199920040-00006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Lactic acidosis in metformin-treated patients: prognostic value of arterial lactate levels and plasma metformin concentrations</span></div><div class="casAuthors">Lalau, Jean-Daniel; Race, Jean-M.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Safety</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">377-384</span>CODEN:
                <span class="NLM_cas:coden">DRSAEA</span>;
        ISSN:<span class="NLM_cas:issn">0114-5916</span>.
    
            (<span class="NLM_cas:orgname">Adis International Ltd.</span>)
        </div><div class="casAbstract">The antidiabetic drug metformin has been assocd. in a small no. of patients with lactic acidosis, a serious condition with a poor prognosis.  However, because of lack of data, the prognostic significance of hyperlactatemia in metformin-treated patients is not known.  Data were collected from 49 metformin-treated patients with lactic acidosis (arterial lactate level ≥5 mmol/L and blood pH ≤7.35) and available plasma metformin concn. data to investigate the assocn. of arterial lactate levels and plasma metformin concns. with mortality.  The overall mortality rate in this patient sample was 45% and the median arterial lactate level was 13.1 mmol/L.  Median lactate levels were similar in patients who survived (13 mmol/L) and those who died (14.3 mmol/L), whereas the median plasma metformin concn. was 3 times higher in patients who survived (20.6 mg/L vs. 6.3 mg/L).  In this, the largest series of metformin-treated patients with lactic acidosis yet reported, 55% of patients survived and these patients had a median arterial lactate level of 13.1 mmol/L.  Neither arterial lactate levels nor plasma metformin concns. were of prognostic significance in relation to mortality in this sample of metformin-treated patients with lactic acidosis.  Death in these patients appeared instead to be assocd. with other hypoxic disease or underlying ill health.  These observations suggest that accumulation of metformin may not be as significant with respect to high arterial levels of lactate and their effects as has been traditionally thought.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAFd7jfy_hDLVg90H21EOLACvtfcHk0lgpJPS3SDQNVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXjtVGhs74%253D&md5=5f2fb705fa49e4c2f3bf58a993c5e814</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.2165%2F00002018-199920040-00006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2165%252F00002018-199920040-00006%26sid%3Dliteratum%253Aachs%26aulast%3DLalau%26aufirst%3DJ.%2BD.%26aulast%3DRace%26aufirst%3DJ.%2BM.%26atitle%3DLactic%2520acidosis%2520in%2520metformin-treated%2520patients.%2520Prognostic%2520value%2520of%2520arterial%2520lactate%2520levels%2520and%2520plasma%2520metformin%2520concentrations%26jtitle%3DDrug%2520Saf.%26date%3D1999%26volume%3D20%26spage%3D377%26epage%3D384%26doi%3D10.2165%2F00002018-199920040-00006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Han, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wan, J.</span></span> <span> </span><span class="NLM_article-title">New insight into the binding modes of TNP-AMP to human liver fructose-1,6-bisphosphatase</span>. <i>Spectrochim. Acta. A Mol. Biomol. Spectrosc.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>165</i></span>,  <span class="NLM_fpage">155</span>– <span class="NLM_lpage">160</span>, <span class="refDoi"> DOI: 10.1016/j.saa.2016.04.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=10.1016%2Fj.saa.2016.04.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=27137358" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BC28XntFyjsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=165&publication_year=2016&pages=155-160&author=X.+Hanauthor=Y.+Huangauthor=R.+Zhangauthor=S.+Xiaoauthor=S.+Zhuauthor=N.+Qinauthor=Z.+Hongauthor=L.+Weiauthor=J.+Fengauthor=Y.+Renauthor=L.+Fengauthor=J.+Wan&title=New+insight+into+the+binding+modes+of+TNP-AMP+to+human+liver+fructose-1%2C6-bisphosphatase&doi=10.1016%2Fj.saa.2016.04.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">New insight into the binding modes of TNP-AMP to human liver fructose-1,6-bisphosphatase</span></div><div class="casAuthors">Han, Xinya; Huang, Yunyuan; Zhang, Rui; Xiao, San; Zhu, Shuaihuan; Qin, Nian; Hong, Zongqin; Wei, Lin; Feng, Jiangtao; Ren, Yanliang; Feng, Lingling; Wan, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Spectrochimica Acta, Part A: Molecular and Biomolecular Spectroscopy</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">165</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">155-160</span>CODEN:
                <span class="NLM_cas:coden">SAMCAS</span>;
        ISSN:<span class="NLM_cas:issn">1386-1425</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Human liver fructose 1,6-bisphosphatase (FBPase) contains 2 binding sites, a fructose 1,6-bisphosphate (FBP) active site and an AMP allosteric site.  The FBP active site works by stabilizing the FBPase, and the allosteric site impairs the activity of FBPase through its binding of a non-substrate mol.  The fluorescent AMP analog, 2',3'-O-(2,4,6-trinitrophenyl)AMP (TNP-AMP) is used as a fluorescent probe as it is able to competitively inhibit AMP binding to the AMP allosteric site and, therefore, could be used for exploring the binding modes of inhibitors targeted on the allosteric site.  Here, the authors re-examd. the binding modes of TNP-AMP to FBPase.  However, the present enzyme kinetic assays showed that AMP and FBP both could reduce the fluorescence from the bound TNP-AMP through competition for FBPase, suggesting that TNP-AMP binds not only to the AMP allosteric site but also to the FBP active site.  Mutagenesis assays of K274L (located in the FBP active site) showed that the Lys-274 residue is very important for TNP-AMP to bind to the active site of FBPase.  The results further proved that TNP-AMP was able to bind individually to the both sites.  Thus, the present study provides a new insight into the binding mechanism of TNP-AMP to FBPase.  The TNP-AMP fluorescent probe could be used to examine the binding site of an inhibitor (the active site or the allosteric site) using FBPase satd. by AMP and FBP, resp., or the K247L mutant FBPase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJ6O9415rhJLVg90H21EOLACvtfcHk0lgpJPS3SDQNVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XntFyjsLc%253D&md5=dbe1c1be3db825878066a33bde03e901</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.saa.2016.04.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.saa.2016.04.002%26sid%3Dliteratum%253Aachs%26aulast%3DHan%26aufirst%3DX.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DR.%26aulast%3DXiao%26aufirst%3DS.%26aulast%3DZhu%26aufirst%3DS.%26aulast%3DQin%26aufirst%3DN.%26aulast%3DHong%26aufirst%3DZ.%26aulast%3DWei%26aufirst%3DL.%26aulast%3DFeng%26aufirst%3DJ.%26aulast%3DRen%26aufirst%3DY.%26aulast%3DFeng%26aufirst%3DL.%26aulast%3DWan%26aufirst%3DJ.%26atitle%3DNew%2520insight%2520into%2520the%2520binding%2520modes%2520of%2520TNP-AMP%2520to%2520human%2520liver%2520fructose-1%252C6-bisphosphatase%26jtitle%3DSpectrochim.%2520Acta.%2520A%2520Mol.%2520Biomol.%2520Spectrosc.%26date%3D2016%26volume%3D165%26spage%3D155%26epage%3D160%26doi%3D10.1016%2Fj.saa.2016.04.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">Xds</span>. <i>Acta Crystallogr. D</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1107/S0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=Xds&doi=10.1107%2FS0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0lgpJPS3SDQNVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXds%26jtitle%3DActa%2520Crystallogr.%2520D%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2FS0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballard, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keegan, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krissinel, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leslie, A. G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNicholas, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannu, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potterton, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, K. S.</span></span> <span> </span><span class="NLM_article-title">Overview of theCCP4 suite and current developments</span>. <i>Acta Crystallogr. D</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">242</span>, <span class="refDoi"> DOI: 10.1107/S0907444910045749</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=10.1107%2FS0907444910045749" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=21460441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=235-242&author=M.+D.+Winnauthor=C.+C.+Ballardauthor=K.+D.+Cowtanauthor=E.+J.+Dodsonauthor=P.+Emsleyauthor=P.+R.+Evansauthor=R.+M.+Keeganauthor=E.+B.+Krissinelauthor=A.+G.+W.+Leslieauthor=A.+McCoyauthor=S.+J.+McNicholasauthor=G.+N.+Murshudovauthor=N.+S.+Pannuauthor=E.+A.+Pottertonauthor=H.+R.+Powellauthor=R.+J.+Readauthor=A.+Vaginauthor=K.+S.+Wilson&title=Overview+of+theCCP4+suite+and+current+developments&doi=10.1107%2FS0907444910045749"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Overview of the CCP4 suite and current developments</span></div><div class="casAuthors">Winn, Martyn D.; Ballard, Charles C.; Cowtan, Kevin D.; Dodson, Eleanor J.; Emsley, Paul; Evans, Phil R.; Keegan, Ronan M.; Krissinel, Eugene B.; Leslie, Andrew G. W.; McCoy, Airlie; McNicholas, Stuart J.; Murshudov, Garib N.; Pannu, Navraj S.; Potterton, Elizabeth A.; Powell, Harold R.; Read, Randy J.; Vagin, Alexei; Wilson, Keith S.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  The CCP4 (Collaborative Computational Project, No. 4) software suite is a collection of programs and assocd. data and software libraries which can be used for macromol. structure detn. by X-ray crystallog.  The suite is designed to be flexible, allowing users a no. of methods of achieving their aims.  The programs are from a wide variety of sources but are connected by a common infrastructure provided by std. file formats, data objects and graphical interfaces.  Structure soln. by macromol. crystallog. is becoming increasingly automated and the CCP4 suite includes several automation pipelines.  After giving a brief description of the evolution of CCP4 over the last 30 years, an overview of the current suite is given.  While detailed descriptions are given in the accompanying articles, here it is shown how the individual programs contribute to a complete software package.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_wWMMIol6UrVg90H21EOLACvtfcHk0liD4eCWpUilqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D&md5=c407e4d47bef46864be336d60147c17d</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1107%2FS0907444910045749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910045749%26sid%3Dliteratum%253Aachs%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DBallard%26aufirst%3DC.%2BC.%26aulast%3DCowtan%26aufirst%3DK.%2BD.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DKeegan%26aufirst%3DR.%2BM.%26aulast%3DKrissinel%26aufirst%3DE.%2BB.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%26aulast%3DMcNicholas%26aufirst%3DS.%2BJ.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DPotterton%26aufirst%3DE.%2BA.%26aulast%3DPowell%26aufirst%3DH.%2BR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DWilson%26aufirst%3DK.%2BS.%26atitle%3DOverview%2520of%2520theCCP4%2520suite%2520and%2520current%2520developments%26jtitle%3DActa%2520Crystallogr.%2520D%26date%3D2011%26volume%3D67%26spage%3D235%26epage%3D242%26doi%3D10.1107%2FS0907444910045749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Coot: model-building tools for molecular graphics</span>. <i>Acta Crystallogr. D</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2126</span>– <span class="NLM_lpage">2132</span>, <span class="refDoi"> DOI: 10.1107/S0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+model-building+tools+for+molecular+graphics&doi=10.1107%2FS0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0liD4eCWpUilqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%2520D%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2FS0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ioerger, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sacchettini, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sauter, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span> <span> </span><span class="NLM_article-title">PHENIX: building new software for automated crystallographic structure determination</span>. <i>Acta Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">1948</span>– <span class="NLM_lpage">1954</span>, <span class="refDoi"> DOI: 10.1107/S0907444902016657</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=10.1107%2FS0907444902016657" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=12393927" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BD38XotVCksbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2002&pages=1948-1954&author=P.+D.+Adamsauthor=R.+W.+Grosse-Kunstleveauthor=L.-W.+Hungauthor=T.+R.+Ioergerauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+J.+Readauthor=J.+C.+Sacchettiniauthor=N.+K.+Sauterauthor=T.+C.+Terwilliger&title=PHENIX%3A+building+new+software+for+automated+crystallographic+structure+determination&doi=10.1107%2FS0907444902016657"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: building new software for automated crystallographic structure determination</span></div><div class="casAuthors">Adams, Paul D.; Grosse-Kunstleve, Ralf W.; Hung, Li Wei; Ioerger, Thomas R.; McCoy, Airlie J.; Moriarty, Nigel W.; Read, Randy J.; Sacchettini, James C.; Sauter, Nicholas K.; Terwilliger, Thomas C.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">D58</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1948-1954</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Munksgaard</span>)
        </div><div class="casAbstract">Structural genomics seeks to rapidly expand the no. of protein structures to ext. the max. amt. of information from genomic sequence databases.  The advent of several large-scale projects worldwide leads to many new challenges in the field of crystallog. macromol. structure detn.  A novel software package, called PHENIX (Python-based Hierarchical Environment for Integrated Xtallog.), is therefore being developed.  This new software will provide the necessary algorithms to proceed from reduced intensity data to a refined mol. model and to facilitate structure soln. for both the novice and expert crystallographer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOtuUfHXPmCbVg90H21EOLACvtfcHk0liD4eCWpUilqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XotVCksbc%253D&md5=9a3b66e1f7d064183f7a9ce108cb4e5e</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1107%2FS0907444902016657&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444902016657%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DHung%26aufirst%3DL.-W.%26aulast%3DIoerger%26aufirst%3DT.%2BR.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DSacchettini%26aufirst%3DJ.%2BC.%26aulast%3DSauter%26aufirst%3DN.%2BK.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26atitle%3DPHENIX%253A%2520building%2520new%2520software%2520for%2520automated%2520crystallographic%2520structure%2520determination%26jtitle%3DActa%2520Crystallogr.%26date%3D2002%26volume%3D58%26spage%3D1948%26epage%3D1954%26doi%3D10.1107%2FS0907444902016657" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Best, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shim, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopes, P. E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mittal, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feig, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacKerell, A. D.,  Jr.</span></span> <span> </span><span class="NLM_article-title">Optimization of the Additive CHARMM all-atom protein force field targeting improved sampling of the backbone ϕ, ψ and side-chain χ1 and χ2 dihedral angles</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">3257</span>– <span class="NLM_lpage">3273</span>, <span class="refDoi"> DOI: 10.1021/ct300400x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct300400x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVKqurfP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=3257-3273&author=R.+B.+Bestauthor=X.+Zhuauthor=J.+Shimauthor=P.+E.+M.+Lopesauthor=J.+Mittalauthor=M.+Feigauthor=A.+D.+MacKerell&title=Optimization+of+the+Additive+CHARMM+all-atom+protein+force+field+targeting+improved+sampling+of+the+backbone+%CF%95%2C+%CF%88+and+side-chain+%CF%871+and+%CF%872+dihedral+angles&doi=10.1021%2Fct300400x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of the Additive CHARMM All-Atom Protein Force Field Targeting Improved Sampling of the Backbone .vphi., ψ and Side-Chain χ1 and χ2 Dihedral Angles</span></div><div class="casAuthors">Best, Robert B.; Zhu, Xiao; Shim, Jihyun; Lopes, Pedro E. M.; Mittal, Jeetain; Feig, Michael; MacKerell, Alexander D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3257-3273</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">While the quality of the current CHARMM22/CMAP additive force field for proteins has been demonstrated in a large no. of applications, limitations in the model with respect to the equil. between the sampling of helical and extended conformations in folding simulations have been noted.  To overcome this, as well as make other improvements in the model, we present a combination of refinements that should result in enhanced accuracy in simulations of proteins.  The common (non-Gly, -Pro) backbone CMAP potential has been refined against exptl. soln. NMR data for weakly structured peptides, resulting in a rebalancing of the energies of the α-helix and extended regions of the Ramachandran map, correcting the α-helical bias of CHARMM22/CMAP.  The Gly and Pro CMAPs have been refitted to more accurate quantum-mech. energy surfaces.  Side-chain torsion parameters have been optimized by fitting to backbone-dependent quantum-mech. energy surfaces, followed by addnl. empirical optimization targeting NMR scalar couplings for unfolded proteins.  A comprehensive validation of the revised force field was then performed against a collection of exptl. data: (i) comparison of simulations of eight proteins in their crystal environments with crystal structures; (ii) comparison with backbone scalar couplings for weakly structured peptides; (iii) comparison with NMR residual dipolar couplings and scalar couplings for both backbone and side-chains in folded proteins; (iv) equil. folding of mini-proteins.  The results indicate that the revised CHARMM 36 parameters represent an improved model for modeling and simulation studies of proteins, including studies of protein folding, assembly, and functionally relevant conformational changes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGnwGJMAMFvLVg90H21EOLACvtfcHk0liy9HWnKHXmAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVKqurfP&md5=9a48a0c5770fb1e887c3bb34d45b1354</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fct300400x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct300400x%26sid%3Dliteratum%253Aachs%26aulast%3DBest%26aufirst%3DR.%2BB.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DShim%26aufirst%3DJ.%26aulast%3DLopes%26aufirst%3DP.%2BE.%2BM.%26aulast%3DMittal%26aufirst%3DJ.%26aulast%3DFeig%26aufirst%3DM.%26aulast%3DMacKerell%26aufirst%3DA.%2BD.%26atitle%3DOptimization%2520of%2520the%2520Additive%2520CHARMM%2520all-atom%2520protein%2520force%2520field%2520targeting%2520improved%2520sampling%2520of%2520the%2520backbone%2520%25CF%2595%252C%2520%25CF%2588%2520and%2520side-chain%2520%25CF%25871%2520and%2520%25CF%25872%2520dihedral%2520angles%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2012%26volume%3D8%26spage%3D3257%26epage%3D3273%26doi%3D10.1021%2Fct300400x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vanommeslaeghe, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatcher, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Acharya, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kundu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shim, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darian, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guvench, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopes, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vorobyov, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mackerell, A. D.,  Jr.</span></span> <span> </span><span class="NLM_article-title">CHARMM general force field: A force field for drug-like molecules compatible with the CHARMM all-atom additive biological force fields</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">671</span>– <span class="NLM_lpage">690</span>, <span class="refDoi"> DOI: 10.1002/jcc.21367</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=10.1002%2Fjcc.21367" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=19575467" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlentbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2010&pages=671-690&author=K.+Vanommeslaegheauthor=E.+Hatcherauthor=C.+Acharyaauthor=S.+Kunduauthor=S.+Zhongauthor=J.+Shimauthor=E.+Darianauthor=O.+Guvenchauthor=P.+Lopesauthor=I.+Vorobyovauthor=A.+D.+Mackerell&title=CHARMM+general+force+field%3A+A+force+field+for+drug-like+molecules+compatible+with+the+CHARMM+all-atom+additive+biological+force+fields&doi=10.1002%2Fjcc.21367"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">CHARMM general force field: A force field for drug-like molecules compatible with the CHARMM all-atom additive biological force fields</span></div><div class="casAuthors">Vanommeslaeghe, K.; Hatcher, E.; Acharya, C.; Kundu, S.; Zhong, S.; Shim, J.; Darian, E.; Guvench, O.; Lopes, P.; Vorobyov, I.; Mackerell, A. D., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">671-690</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The widely used CHARMM additive all-atom force field includes parameters for proteins, nucleic acids, lipids, and carbohydrates.  In the present article, an extension of the CHARMM force field to drug-like mols. is presented.  The resulting CHARMM General Force Field (CGenFF) covers a wide range of chem. groups present in biomols. and drug-like mols., including a large no. of heterocyclic scaffolds.  The parametrization philosophy behind the force field focuses on quality at the expense of transferability, with the implementation concg. on an extensible force field.  Statistics related to the quality of the parametrization with a focus on exptl. validation are presented.  Addnl., the parametrization procedure, described fully in the present article in the context of the model systems, pyrrolidine, and 3-phenoxymethyl-pyrrolidine will allow users to readily extend the force field to chem. groups that are not explicitly covered in the force field as well as add functional groups to and link together mols. already available in the force field.  CGenFF thus makes it possible to perform "all-CHARMM" simulations on drug-target interactions thereby extending the utility of CHARMM force fields to medicinally relevant systems. © 2009 Wiley Periodicals, Inc.J Comput Chem, 2010.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCuh0CLdmxN7Vg90H21EOLACvtfcHk0liy9HWnKHXmAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlentbc%253D&md5=26212e0e4f73bded0c89d4b411cd3833</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1002%2Fjcc.21367&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.21367%26sid%3Dliteratum%253Aachs%26aulast%3DVanommeslaeghe%26aufirst%3DK.%26aulast%3DHatcher%26aufirst%3DE.%26aulast%3DAcharya%26aufirst%3DC.%26aulast%3DKundu%26aufirst%3DS.%26aulast%3DZhong%26aufirst%3DS.%26aulast%3DShim%26aufirst%3DJ.%26aulast%3DDarian%26aufirst%3DE.%26aulast%3DGuvench%26aufirst%3DO.%26aulast%3DLopes%26aufirst%3DP.%26aulast%3DVorobyov%26aufirst%3DI.%26aulast%3DMackerell%26aufirst%3DA.%2BD.%26atitle%3DCHARMM%2520general%2520force%2520field%253A%2520A%2520force%2520field%2520for%2520drug-like%2520molecules%2520compatible%2520with%2520the%2520CHARMM%2520all-atom%2520additive%2520biological%2520force%2520fields%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2010%26volume%3D31%26spage%3D671%26epage%3D690%26doi%3D10.1002%2Fjcc.21367" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandrasekhar, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madura, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Impey, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, M. L.</span></span> <span> </span><span class="NLM_article-title">Comparison of simple potential functions for simulating liquid water</span>. <i>J. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">1983</span>,  <span class="NLM_volume"><i>79</i></span>,  <span class="NLM_fpage">926</span>– <span class="NLM_lpage">935</span>, <span class="refDoi"> DOI: 10.1063/1.445869</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=10.1063%2F1.445869" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADyaL3sXksF2htL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=1983&pages=926-935&author=W.+L.+Jorgensenauthor=J.+Chandrasekharauthor=J.+D.+Maduraauthor=R.+W.+Impeyauthor=M.+L.+Klein&title=Comparison+of+simple+potential+functions+for+simulating+liquid+water&doi=10.1063%2F1.445869"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of simple potential functions for simulating liquid water</span></div><div class="casAuthors">Jorgensen, William L.; Chandrasekhar, Jayaraman; Madura, Jeffry D.; Impey, Roger W.; Klein, Michael L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1983</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">926-35</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">Classical Monte Carlo simulations were carried out for liq. H2O in the NPT ensemble at 25° and 1 atm using 6 of the simpler intermol. potential functions for the dimer.  Comparisons were made with exptl. thermodn. and structural data including the neutron diffraction results of Thiessen and Narten (1982).  The computed densities and potential energies agree with expt. except for the original Bernal-Fowler model, which yields an 18% overest. of the d. and poor structural results.  The discrepancy may be due to the correction terms needed in processing the neutron data or to an effect uniformly neglected in the computations.  Comparisons were made for the self-diffusion coeffs. obtained from mol. dynamics simulations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpK4QUFMvwYFrVg90H21EOLACvtfcHk0liy9HWnKHXmAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3sXksF2htL4%253D&md5=a1161334e381746be8c9b15a5e56f704</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1063%2F1.445869&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.445869%26sid%3Dliteratum%253Aachs%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DChandrasekhar%26aufirst%3DJ.%26aulast%3DMadura%26aufirst%3DJ.%2BD.%26aulast%3DImpey%26aufirst%3DR.%2BW.%26aulast%3DKlein%26aufirst%3DM.%2BL.%26atitle%3DComparison%2520of%2520simple%2520potential%2520functions%2520for%2520simulating%2520liquid%2520water%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1983%26volume%3D79%26spage%3D926%26epage%3D935%26doi%3D10.1063%2F1.445869" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hess, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bekker, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berendsen, H. J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fraaije, J. G. E. M.</span></span> <span> </span><span class="NLM_article-title">LINCS: A linear constraint solver for molecular simulations</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">1463</span>– <span class="NLM_lpage">1472</span>, <span class="refDoi"> DOI: 10.1002/(SICI)1096-987X(199709)18:12<1463::AID-JCC4>3.0.CO;2-H</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=10.1002%2F%28SICI%291096-987X%28199709%2918%3A12%3C1463%3A%3AAID-JCC4%3E3.0.CO%3B2-H" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADyaK2sXlvV2nu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=1997&pages=1463-1472&author=B.+Hessauthor=H.+Bekkerauthor=H.+J.+C.+Berendsenauthor=J.+G.+E.+M.+Fraaije&title=LINCS%3A+A+linear+constraint+solver+for+molecular+simulations&doi=10.1002%2F%28SICI%291096-987X%28199709%2918%3A12%3C1463%3A%3AAID-JCC4%3E3.0.CO%3B2-H"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">LINCS: a linear constraint solver for molecular simulations</span></div><div class="casAuthors">Hess, Berk; Bekker, Henk; Berendsen, Herman J. C.; Fraaije, Johannes G. E. M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1463-1472</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">Wiley</span>)
        </div><div class="casAbstract">We present a new LINear Constraint Solver (LINCS) for mol. simulations with bond constraints using the enzyme lysozyme and a 32-residue peptide as test systems.  The algorithm is inherently stable, as the constraints themselves are reset instead of derivs. of the constraints, thereby eliminating drift.  Although the derivation of the algorithm is presented in terms of matrixes, no matrix matrix multiplications are needed and only the nonzero matrix elements have to be stored, making the method useful for very large mols.  At the same accuracy, the LINCS algorithm is 3-4 times faster than the SHAKE algorithm.  Parallelization of the algorithm is straightforward.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpk6h4pdV0Nx7Vg90H21EOLACvtfcHk0ljjTS0HW1QXxw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXlvV2nu7g%253D&md5=890f8af0d2ca1f65aa93db5a3a0bacf2</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291096-987X%28199709%2918%3A12%3C1463%3A%3AAID-JCC4%3E3.0.CO%3B2-H&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291096-987X%2528199709%252918%253A12%253C1463%253A%253AAID-JCC4%253E3.0.CO%253B2-H%26sid%3Dliteratum%253Aachs%26aulast%3DHess%26aufirst%3DB.%26aulast%3DBekker%26aufirst%3DH.%26aulast%3DBerendsen%26aufirst%3DH.%2BJ.%2BC.%26aulast%3DFraaije%26aufirst%3DJ.%2BG.%2BE.%2BM.%26atitle%3DLINCS%253A%2520A%2520linear%2520constraint%2520solver%2520for%2520molecular%2520simulations%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1997%26volume%3D18%26spage%3D1463%26epage%3D1472%26doi%3D10.1002%2F%28SICI%291096-987X%28199709%2918%3A12%3C1463%3A%3AAID-JCC4%3E3.0.CO%3B2-H" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Darden, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">York, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pedersen, L.</span></span> <span> </span><span class="NLM_article-title">Particle mesh Ewald: An N·log(N) method for Ewald sums in large systems</span>. <i>J. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>98</i></span>,  <span class="NLM_fpage">10089</span>– <span class="NLM_lpage">10092</span>, <span class="refDoi"> DOI: 10.1063/1.464397</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=10.1063%2F1.464397" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADyaK3sXks1Ohsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=1993&pages=10089-10092&author=T.+Dardenauthor=D.+Yorkauthor=L.+Pedersen&title=Particle+mesh+Ewald%3A+An+N%C2%B7log%28N%29+method+for+Ewald+sums+in+large+systems&doi=10.1063%2F1.464397"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Particle mesh Ewald: an N·log(N) method for Ewald sums in large systems</span></div><div class="casAuthors">Darden, Tom; York, Darrin; Pedersen, Lee</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">10089-92</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">An N·log(N) method for evaluating electrostatic energies and forces of large periodic systems is presented.  The method is based on interpolation of the reciprocal space Ewald sums and evaluation of the resulting convolution using fast Fourier transforms.  Timings and accuracies are presented for three large cryst. ionic systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDIJavf7AZPbVg90H21EOLACvtfcHk0ljjTS0HW1QXxw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXks1Ohsr0%253D&md5=3c9f230bd01b7b714fd096d4d2e755f6</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1063%2F1.464397&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.464397%26sid%3Dliteratum%253Aachs%26aulast%3DDarden%26aufirst%3DT.%26aulast%3DYork%26aufirst%3DD.%26aulast%3DPedersen%26aufirst%3DL.%26atitle%3DParticle%2520mesh%2520Ewald%253A%2520An%2520N%25C2%25B7log%2528N%2529%2520method%2520for%2520Ewald%2520sums%2520in%2520large%2520systems%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1993%26volume%3D98%26spage%3D10089%26epage%3D10092%26doi%3D10.1063%2F1.464397" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hess, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kutzner, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der
Spoel, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindahl, E.</span></span> <span> </span><span class="NLM_article-title">GROMACS 4: Algorithms for highly efficient, load-balanced, and scalable molecular simulation</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">435</span>– <span class="NLM_lpage">447</span>, <span class="refDoi"> DOI: 10.1021/ct700301q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct700301q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVSqurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2008&pages=435-447&author=B.+Hessauthor=C.+Kutznerauthor=D.+van+der%0ASpoelauthor=E.+Lindahl&title=GROMACS+4%3A+Algorithms+for+highly+efficient%2C+load-balanced%2C+and+scalable+molecular+simulation&doi=10.1021%2Fct700301q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">GROMACS 4: Algorithms for Highly Efficient, Load-Balanced, and Scalable Molecular Simulation</span></div><div class="casAuthors">Hess, Berk; Kutzner, Carsten; van der Spoel, David; Lindahl, Erik</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">435-447</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. simulation is an extremely useful, but computationally very expensive tool for studies of chem. and biomol. systems.  Here, we present a new implementation of our mol. simulation toolkit GROMACS which now both achieves extremely high performance on single processors from algorithmic optimizations and hand-coded routines and simultaneously scales very well on parallel machines.  The code encompasses a minimal-communication domain decompn. algorithm, full dynamic load balancing, a state-of-the-art parallel constraint solver, and efficient virtual site algorithms that allow removal of hydrogen atom degrees of freedom to enable integration time steps up to 5 fs for atomistic simulations also in parallel.  To improve the scaling properties of the common particle mesh Ewald electrostatics algorithms, we have in addn. used a Multiple-Program, Multiple-Data approach, with sep. node domains responsible for direct and reciprocal space interactions.  Not only does this combination of algorithms enable extremely long simulations of large systems but also it provides that simulation performance on quite modest nos. of std. cluster nodes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrajjWdk6mUFrVg90H21EOLACvtfcHk0ljjTS0HW1QXxw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVSqurc%253D&md5=d53c94901386260221792ea30f151c5f</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Fct700301q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct700301q%26sid%3Dliteratum%253Aachs%26aulast%3DHess%26aufirst%3DB.%26aulast%3DKutzner%26aufirst%3DC.%26aulast%3Dvan%2Bder%2BSpoel%26aufirst%3DD.%26aulast%3DLindahl%26aufirst%3DE.%26atitle%3DGROMACS%25204%253A%2520Algorithms%2520for%2520highly%2520efficient%252C%2520load-balanced%252C%2520and%2520scalable%2520molecular%2520simulation%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2008%26volume%3D4%26spage%3D435%26epage%3D447%26doi%3D10.1021%2Fct700301q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 2 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiao  Yu</span>, <span class="hlFld-ContribAuthor ">Fan  Zhang</span>, <span class="hlFld-ContribAuthor ">Ting  Liu</span>, <span class="hlFld-ContribAuthor ">Zhigang  Liu</span>, <span class="hlFld-ContribAuthor ">Qingjian  Dong</span>, <span class="hlFld-ContribAuthor ">Ding  Li</span>. </span><span class="cited-content_cbyCitation_article-title">Exploring efficacy of natural-derived acetylphenol scaffold inhibitors for α-glucosidase: Synthesis, in vitro and in vivo biochemical studies. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2020,</strong> <em>30 </em>
                                    (23)
                                     , 127528. <a href="https://doi.org/10.1016/j.bmcl.2020.127528" title="DOI URL">https://doi.org/10.1016/j.bmcl.2020.127528</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2020.127528&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2020.127528%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DExploring%252Befficacy%252Bof%252Bnatural-derived%252Bacetylphenol%252Bscaffold%252Binhibitors%252Bfor%252B%2525CE%2525B1-glucosidase%25253A%252BSynthesis%25252C%252Bin%252Bvitro%252Band%252Bin%252Bvivo%252Bbiochemical%252Bstudies%26aulast%3DYu%26aufirst%3DXiao%26date%3D2020%26volume%3D30%26issue%3D23%26spage%3D127528" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yi-xiang  Xu</span>, <span class="hlFld-ContribAuthor ">Yun-yuan  Huang</span>, <span class="hlFld-ContribAuthor ">Rong-rong  Song</span>, <span class="hlFld-ContribAuthor ">Yan-liang  Ren</span>, <span class="hlFld-ContribAuthor ">Xin  Chen</span>, <span class="hlFld-ContribAuthor ">Chao  Zhang</span>, <span class="hlFld-ContribAuthor ">Fei  Mao</span>, <span class="hlFld-ContribAuthor ">Xiao-kang  Li</span>, <span class="hlFld-ContribAuthor ">Jin  Zhu</span>, <span class="hlFld-ContribAuthor ">Shuai-shuai  Ni</span>, <span class="hlFld-ContribAuthor ">Jian  Wan</span>, <span class="hlFld-ContribAuthor ">Jian  Li</span>. </span><span class="cited-content_cbyCitation_article-title">Development of disulfide-derived fructose-1,6-bisphosphatase (FBPase) covalent inhibitors for the treatment of type 2 diabetes. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>203 </em>, 112500. <a href="https://doi.org/10.1016/j.ejmech.2020.112500" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112500</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112500&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112500%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDevelopment%252Bof%252Bdisulfide-derived%252Bfructose-1%25252C6-bisphosphatase%252B%252528FBPase%252529%252Bcovalent%252Binhibitors%252Bfor%252Bthe%252Btreatment%252Bof%252Btype%252B2%252Bdiabetes%26aulast%3DXu%26aufirst%3DYi-xiang%26date%3D2020%26volume%3D203%26spage%3D112500" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00699/20200604/images/medium/jm0c00699_0007.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00699/20200604/images/large/jm0c00699_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00699&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00699/20200604/images/medium/jm0c00699_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00699/20200604/images/large/jm0c00699_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Compounds <b>212</b> and <b>214</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00699/20200604/images/large/jm0c00699_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00699&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a)SO<sub>2</sub>Cl<sub>2</sub>, DCM, 0 °C, 1 h; (b) Et<sub>2</sub>O, room temperature, 12 h; (c) DDQ, DCM, 0 °C, 1 h.</p></p></figure><figure data-id="fig1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00699/20200604/images/medium/jm0c00699_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00699/20200604/images/large/jm0c00699_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Designing strategy of asymmetric disulfide compounds <b>212/214</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00699/20200604/images/large/jm0c00699_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00699&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00699/20200604/images/medium/jm0c00699_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00699/20200604/images/large/jm0c00699_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Crystal structures of FBPase. (A) Interaction networks between helix H3 (73–88) and helix H4 (123–128) in APO FBPase (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1FTA">1FTA</a>, green). (B) Interaction networks between helix H3 and helix H4 in FBP-bound FBPase (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ZWK">5ZWK</a>, salmon.) (C) Electron density (2Fo-Fc omit map contoured at 0.8σ) for <b>214b</b> bound to C128. (D) Interaction networks between FBPase and <b>214b</b>. (E) Pocket variation from the original C128 site in APO FBPase/FBP-bound FBPase to the new allosteric binding site in <b>214b</b>-bound FBPase. (F) Interaction networks between helix H3 and helix H4 in <b>214b</b>-bound FBPase (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6LS5">6LS5</a>, gray) (G) Conformation comparison among <b>214b</b>-FBPase (gray cartoon, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6LS5">6LS5</a>), FBP-bound FBPase (salmon cartoon, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ZWK">5ZWK</a>), and APO FBPase (green cartoon, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1FTA">1FTA</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00699/20200604/images/large/jm0c00699_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00699&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00699/20200604/images/medium/jm0c00699_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00699/20200604/images/large/jm0c00699_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Analysis of FBPase structures simulated by 100 ns molecular dynamics (MD). (A) Interaction networks between APO FBPase structure helix H3 and helix H4. (B) Interaction networks between FBP-bound FBPase structure helix H3 and helix H4. (C) Interaction networks between <b>214b</b>-bound FBPase structure helix H3 and helix H4. (D) Conformation comparison among <b>214b</b>-FBPase (blue cartoon), FBP-bound FBPase (orange cartoon), and APO FBPase (magenta cartoon). (E) Distance change (S124:O–C128:N) of three FBPase structures (APO: magenta line, FBP-bound: orange line, and <b>214b-</b>bound: blue line). (F) Helical content (residues 120–129) change of three FBPase structures (APO: magenta line, FBP-bound: orange line, and <b>214b-</b>bound: blue line). (G) S123 side chain population density of three FBPase structures (APO: magenta line, FBP-bound: orange line, and <b>214b-</b>bound: blue line) in molecular dynamics balance.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00699/20200604/images/large/jm0c00699_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00699&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00699/20200604/images/medium/jm0c00699_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00699/20200604/images/large/jm0c00699_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Full-scan electrospray mass spectrum of the parent ion region of <b>214</b> covalent binding of WT (wild-type) FBPase and C128S. (A) Molecular weight of WT FBPase. (B) Mass weight of FBPase + <i>n</i><b>214b</b> (<i>n</i> = 2 and 3). (C) Mass weight of C128S mutation. (D) Mass weight of C128S + <i>n</i><b>214b</b> (<i>n</i> = 1 and 2).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00699/20200604/images/large/jm0c00699_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00699&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00699/20200604/images/medium/jm0c00699_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00699/20200604/images/large/jm0c00699_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Effect of single administration of disulfiram or compound <b>212</b> on blood glucose in 7–9 week old ICR or <i>db/db</i> mice (<i>n</i> = 5–6 in each group). (A) Blood glucose changes and (B) AUC of blood glucose between 0 and 8 h in 12 h-fasted ICR mice after oral administration of disulfiram and compound <b>212</b>. (C) Blood glucose changes and (D) AUC of blood glucose between 0 and 8 h in 12 h-fasted ICR mice after intraperitoneal administration of compound <b>212</b>. (E) Blood glucose responses to an oral glucose challenge (2 g/kg) and (F) AUC of blood glucose between 0 and 2 h in 12 h-fasted ICR mice. Compound <b>212</b> was intraperitoneally administered 1.5 h prior to oral glucose challenge. (G) Blood glucose changes and (H) AUC of blood glucose between 0 and 6 h in freely feeding <i>db/db</i> mice after intraperitoneal administration of compound <b>212</b>. (I) Blood glucose responses to an oral glucose challenge (2 g/kg) and (J) AUC of blood glucose between 0 and 2 h in 12 h-fasted <i>db/db</i> mice. Compound <b>212</b> was administered 1.5 h prior to oral glucose challenge. Data are presented as the mean ± SEM (*<i>P</i> < 0.05, **<i>P</i> < 0.01, ***<i>P</i> < 0.001 vs vehicle; student’s <i>t</i> test).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-11/acs.jmedchem.0c00699/20200604/images/large/jm0c00699_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00699&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i33">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_21387" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_21387" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 44 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hunter, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughey, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lantier, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sundelin, E. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peggie, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeqiraj, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sicheri, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jessen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wasserman, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakamoto, K.</span></span> <span> </span><span class="NLM_article-title">Metformin reduces liver glucose production by inhibition of fructose-1-6-bisphosphatase</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">1395</span>– <span class="NLM_lpage">1406</span>, <span class="refDoi"> DOI: 10.1038/s41591-018-0159-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=10.1038%2Fs41591-018-0159-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=30150719" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsF2nsrvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2018&pages=1395-1406&author=R.+W.+Hunterauthor=C.+C.+Hugheyauthor=L.+Lantierauthor=E.+I.+Sundelinauthor=M.+Peggieauthor=E.+Zeqirajauthor=F.+Sicheriauthor=N.+Jessenauthor=D.+H.+Wassermanauthor=K.+Sakamoto&title=Metformin+reduces+liver+glucose+production+by+inhibition+of+fructose-1-6-bisphosphatase&doi=10.1038%2Fs41591-018-0159-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Metformin reduces liver glucose production by inhibition of fructose-1-6-bisphosphatase</span></div><div class="casAuthors">Hunter, Roger W.; Hughey, Curtis C.; Lantier, Louise; Sundelin, Elias I.; Peggie, Mark; Zeqiraj, Elton; Sicheri, Frank; Jessen, Niels; Wasserman, David H.; Sakamoto, Kei</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1395-1406</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Metformin is a first-line drug for the treatment of individuals with type 2 diabetes, yet its precise mechanism of action remains unclear.  Metformin exerts its antihyperglycemic action primarily through lowering hepatic glucose prodn. (HGP).  This suppression is thought to be mediated through inhibition of mitochondrial respiratory complex I, and thus elevation of 5'-adenosine monophosphate (AMP) levels and the activation of AMP-activated protein kinase (AMPK), though this proposition has been challenged given results in mice lacking hepatic AMPK.  Here we report that the AMP-inhibited enzyme fructose-1,6-bisphosphatase-1 (FBP1), a rate-controlling enzyme in gluconeogenesis, functions as a major contributor to the therapeutic action of metformin.  We identified a point mutation in FBP1 that renders it insensitive to AMP while sparing regulation by fructose-2,6-bisphosphate (F-2,6-P2), and knock-in (KI) of this mutant in mice significantly reduces their response to metformin treatment.  We observe this during a metformin tolerance test and in a metformin-euglycemic clamp that we have developed.  The antihyperglycemic effect of metformin in high-fat diet-fed diabetic FBP1-KI mice was also significantly blunted compared to wild-type controls.  Collectively, we show a new mechanism of action for metformin and provide further evidence that mol. targeting of FBP1 can have antihyperglycemic effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHU4MMYmFinrVg90H21EOLACvtfcHk0lhrzoH90MFpEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsF2nsrvJ&md5=c1696431e641d352c2c2619862d6e4bb</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fs41591-018-0159-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41591-018-0159-7%26sid%3Dliteratum%253Aachs%26aulast%3DHunter%26aufirst%3DR.%2BW.%26aulast%3DHughey%26aufirst%3DC.%2BC.%26aulast%3DLantier%26aufirst%3DL.%26aulast%3DSundelin%26aufirst%3DE.%2BI.%26aulast%3DPeggie%26aufirst%3DM.%26aulast%3DZeqiraj%26aufirst%3DE.%26aulast%3DSicheri%26aufirst%3DF.%26aulast%3DJessen%26aufirst%3DN.%26aulast%3DWasserman%26aufirst%3DD.%2BH.%26aulast%3DSakamoto%26aufirst%3DK.%26atitle%3DMetformin%2520reduces%2520liver%2520glucose%2520production%2520by%2520inhibition%2520of%2520fructose-1-6-bisphosphatase%26jtitle%3DNat.%2520Med.%26date%3D2018%26volume%3D24%26spage%3D1395%26epage%3D1406%26doi%3D10.1038%2Fs41591-018-0159-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, D. S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walton, Z. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ochocki, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathew, L. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mancuso, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gade, T. P. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keith, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nissim, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simon, M. C.</span></span> <span> </span><span class="NLM_article-title">Fructose-1,6-bisphosphatase opposes renal carcinoma progression</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>513</i></span>,  <span class="NLM_fpage">251</span>– <span class="NLM_lpage">255</span>, <span class="refDoi"> DOI: 10.1038/nature13557</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=10.1038%2Fnature13557" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=25043030" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFOltrjK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=513&publication_year=2014&pages=251-255&author=B.+Liauthor=B.+Qiuauthor=D.+S.+M.+Leeauthor=Z.+E.+Waltonauthor=J.+D.+Ochockiauthor=L.+K.+Mathewauthor=A.+Mancusoauthor=T.+P.+F.+Gadeauthor=B.+Keithauthor=I.+Nissimauthor=M.+C.+Simon&title=Fructose-1%2C6-bisphosphatase+opposes+renal+carcinoma+progression&doi=10.1038%2Fnature13557"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Fructose-1,6-bisphosphatase opposes renal carcinoma progression</span></div><div class="casAuthors">Li, Bo; Qiu, Bo; Lee, David S. M.; Walton, Zandra E.; Ochocki, Joshua D.; Mathew, Lijoy K.; Mancuso, Anthony; Gade, Terence P. F.; Keith, Brian; Nissim, Itzhak; Simon, M. Celeste</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">513</span>
        (<span class="NLM_cas:issue">7517</span>),
    <span class="NLM_cas:pages">251-255</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Clear cell renal cell carcinoma (ccRCC), the most common form of kidney cancer, is characterized by elevated glycogen levels and fat deposition.  These consistent metabolic alterations are assocd. with normoxic stabilization of hypoxia-inducible factors (HIFs) secondary to von Hippel-Lindau (VHL) mutations that occur in over 90% of ccRCC tumors.  However, kidney-specific VHL deletion in mice fails to elicit ccRCC-specific metabolic phenotypes and tumor formation, suggesting that addnl. mechanisms are essential.  Recent large-scale sequencing analyses revealed the loss of several chromatin remodelling enzymes in a subset of ccRCC (these included polybromo-1, SET domain contg. 2 and BRCA1-assocd. protein-1, among others), indicating that epigenetic perturbations are probably important contributors to the natural history of this disease.  Here we used an integrative approach comprising pan-metabolomic profiling and metabolic gene set anal. and detd. that the gluconeogenic enzyme fructose-1,6-bisphosphatase 1 (FBP1) is uniformly depleted in over six hundred ccRCC tumors examd.  Notably, the human FBP1 locus resides on chromosome 9q22, the loss of which is assocd. with poor prognosis for ccRCC patients.  Our data further indicate that FBP1 inhibits ccRCC progression through two distinct mechanisms.  First, FBP1 antagonizes glycolytic flux in renal tubular epithelial cells, the presumptive ccRCC cell of origin, thereby inhibiting a potential Warburg effect.  Second, in pVHL (the protein encoded by the VHL gene)-deficient ccRCC cells, FBP1 restrains cell proliferation, glycolysis and the pentose phosphate pathway in a catalytic-activity-independent manner, by inhibiting nuclear HIF function via direct interaction with the HIF inhibitory domain.  This unique dual function of the FBP1 protein explains its ubiquitous loss in ccRCC, distinguishing FBP1 from previously identified tumor suppressors that are not consistently mutated in all tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTwq6rXdgYK7Vg90H21EOLACvtfcHk0lhKHBzT0frI8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFOltrjK&md5=1e0d6f2a5d7dc7fc058866b690feaa9b</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnature13557&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13557%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DB.%26aulast%3DQiu%26aufirst%3DB.%26aulast%3DLee%26aufirst%3DD.%2BS.%2BM.%26aulast%3DWalton%26aufirst%3DZ.%2BE.%26aulast%3DOchocki%26aufirst%3DJ.%2BD.%26aulast%3DMathew%26aufirst%3DL.%2BK.%26aulast%3DMancuso%26aufirst%3DA.%26aulast%3DGade%26aufirst%3DT.%2BP.%2BF.%26aulast%3DKeith%26aufirst%3DB.%26aulast%3DNissim%26aufirst%3DI.%26aulast%3DSimon%26aufirst%3DM.%2BC.%26atitle%3DFructose-1%252C6-bisphosphatase%2520opposes%2520renal%2520carcinoma%2520progression%26jtitle%3DNature%26date%3D2014%26volume%3D513%26spage%3D251%26epage%3D255%26doi%3D10.1038%2Fnature13557" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huangyang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nissim, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weljie, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mancuso, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keith, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoon, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simon, M. C.</span></span> <span> </span><span class="NLM_article-title">Fructose-1,6-bisphosphatase 2 inhibits sarcoma progression by restraining mitochondrial biogenesis</span>. <i>Cell Metab.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">174</span>– <span class="NLM_lpage">188.e7</span>, <span class="refDoi"> DOI: 10.1016/j.cmet.2019.10.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=10.1016%2Fj.cmet.2019.10.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=31761563" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BC1MXit1Krs7fM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2020&pages=174-188.e7&author=P.+Huangyangauthor=F.+Liauthor=P.+Leeauthor=I.+Nissimauthor=A.+M.+Weljieauthor=A.+Mancusoauthor=B.+Liauthor=B.+Keithauthor=S.+S.+Yoonauthor=M.+C.+Simon&title=Fructose-1%2C6-bisphosphatase+2+inhibits+sarcoma+progression+by+restraining+mitochondrial+biogenesis&doi=10.1016%2Fj.cmet.2019.10.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Fructose-1,6-Bisphosphatase 2 Inhibits Sarcoma Progression by Restraining Mitochondrial Biogenesis</span></div><div class="casAuthors">Huangyang, Peiwei; Li, Fuming; Lee, Pearl; Nissim, Itzhak; Weljie, Aalim M.; Mancuso, Anthony; Li, Bo; Keith, Brian; Yoon, Sam S.; Celeste Simon, M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Metabolism</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">174-188.e7</span>CODEN:
                <span class="NLM_cas:coden">CMEEB5</span>;
        ISSN:<span class="NLM_cas:issn">1550-4131</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">The remarkable cellular and genetic heterogeneity of soft tissue sarcomas (STSs) limits the clin. benefit of targeted therapies.  Here, we show that expression of the gluconeogenic isoenzyme fructose-1,6-bisphosphatase 2 (FBP2) is silenced in a broad spectrum of sarcoma subtypes, revealing an apparent common metabolic feature shared by diverse STSs.  Enforced FBP2 expression inhibits sarcoma cell and tumor growth through two distinct mechanisms.  First, cytosolic FBP2 antagonizes elevated glycolysis assocd. with the "Warburg effect", thereby inhibiting sarcoma cell proliferation.  Second, nuclear-localized FBP2 restrains mitochondrial biogenesis and respiration in a catalytic-activity-independent manner by inhibiting the expression of nuclear respiratory factor and mitochondrial transcription factor A (TFAM).  Specifically, nuclear FBP2 colocalizes with the c-Myc transcription factor at the TFAM locus and represses c-Myc-dependent TFAM expression.  This unique dual function of FBP2 provides a rationale for its selective suppression in STSs, identifying a potential metabolic vulnerability of this malignancy and possible therapeutic target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDl8wisDCTFbVg90H21EOLACvtfcHk0lhKHBzT0frI8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXit1Krs7fM&md5=62f6640a2d1ccdb7535f940ccd5bc958</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.cmet.2019.10.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cmet.2019.10.012%26sid%3Dliteratum%253Aachs%26aulast%3DHuangyang%26aufirst%3DP.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DLee%26aufirst%3DP.%26aulast%3DNissim%26aufirst%3DI.%26aulast%3DWeljie%26aufirst%3DA.%2BM.%26aulast%3DMancuso%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DB.%26aulast%3DKeith%26aufirst%3DB.%26aulast%3DYoon%26aufirst%3DS.%2BS.%26aulast%3DSimon%26aufirst%3DM.%2BC.%26atitle%3DFructose-1%252C6-bisphosphatase%25202%2520inhibits%2520sarcoma%2520progression%2520by%2520restraining%2520mitochondrial%2520biogenesis%26jtitle%3DCell%2520Metab.%26date%3D2020%26volume%3D31%26spage%3D174%26epage%3D188.e7%26doi%3D10.1016%2Fj.cmet.2019.10.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dong, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, T. W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miriyala, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lorkiewicz, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">St Clair, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, M.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evers, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, B. P.</span></span> <span> </span><span class="NLM_article-title">Loss of FBP1 by Snail-mediated repression provides metabolic advantages in basal-like breast cancer</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">316</span>– <span class="NLM_lpage">331</span>, <span class="refDoi"> DOI: 10.1016/j.ccr.2013.01.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=10.1016%2Fj.ccr.2013.01.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=23453623" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtl2jtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2013&pages=316-331&author=C.+Dongauthor=T.+Yuanauthor=Y.+Wuauthor=Y.+Wangauthor=T.+W.+M.+Fanauthor=S.+Miriyalaauthor=Y.+Linauthor=J.+Yaoauthor=J.+Shiauthor=T.+Kangauthor=P.+Lorkiewiczauthor=D.+St+Clairauthor=M.-C.+Hungauthor=B.+M.+Eversauthor=B.+P.+Zhou&title=Loss+of+FBP1+by+Snail-mediated+repression+provides+metabolic+advantages+in+basal-like+breast+cancer&doi=10.1016%2Fj.ccr.2013.01.022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Loss of FBP1 by Snail-Mediated Repression Provides Metabolic Advantages in Basal-like Breast Cancer</span></div><div class="casAuthors">Dong, Chenfang; Yuan, Tingting; Wu, Yadi; Wang, Yifan; Fan, Teresa W. M.; Miriyala, Sumitra; Lin, Yiwei; Yao, Jun; Shi, Jian; Kang, Tiebang; Lorkiewicz, Pawel; St. Clair, Daret; Hung, Mien-Chie; Evers, B. Mark; Zhou, Binhua P.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">316-331</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">The epithelial-mesenchymal transition (EMT) enhances cancer invasiveness and confers tumor cells with cancer stem cell (CSC)-like characteristics.  We show that the Snail-G9a-Dnmt1 complex, which is crit. for E-cadherin promoter silencing, is also required for the promoter methylation of fructose-1,6-biphosphatase (FBP1) in basal-like breast cancer (BLBC).  Loss of FBP1 induces glycolysis and results in increased glucose uptake, macromol. biosynthesis, formation of tetrameric PKM2, and maintenance of ATP prodn. under hypoxia.  Loss of FBP1 also inhibits oxygen consumption and reactive oxygen species prodn. by suppressing mitochondrial complex I activity; this metabolic reprogramming results in an increased CSC-like property and tumorigenicity by enhancing the interaction of β-catenin with T-cell factor.  Our study indicates that the loss of FBP1 is a crit. oncogenic event in EMT and BLBC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWkPRV9TNYgbVg90H21EOLACvtfcHk0lhKHBzT0frI8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtl2jtrs%253D&md5=8f9ede71407f07c62e4d3a811890d30e</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2013.01.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2013.01.022%26sid%3Dliteratum%253Aachs%26aulast%3DDong%26aufirst%3DC.%26aulast%3DYuan%26aufirst%3DT.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DFan%26aufirst%3DT.%2BW.%2BM.%26aulast%3DMiriyala%26aufirst%3DS.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DYao%26aufirst%3DJ.%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DKang%26aufirst%3DT.%26aulast%3DLorkiewicz%26aufirst%3DP.%26aulast%3DSt%2BClair%26aufirst%3DD.%26aulast%3DHung%26aufirst%3DM.-C.%26aulast%3DEvers%26aufirst%3DB.%2BM.%26aulast%3DZhou%26aufirst%3DB.%2BP.%26atitle%3DLoss%2520of%2520FBP1%2520by%2520Snail-mediated%2520repression%2520provides%2520metabolic%2520advantages%2520in%2520basal-like%2520breast%2520cancer%26jtitle%3DCancer%2520Cell%26date%3D2013%26volume%3D23%26spage%3D316%26epage%3D331%26doi%3D10.1016%2Fj.ccr.2013.01.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, H.</span></span> <span> </span><span class="NLM_article-title">Dysfunction of natural killer cells by FBP1-induced inhibition of glycolysis during lung cancer progression</span>. <i>Cell Metab.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">243</span>– <span class="NLM_lpage">255.e5</span>, <span class="refDoi"> DOI: 10.1016/j.cmet.2018.06.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=10.1016%2Fj.cmet.2018.06.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=30033198" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlGlur%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2018&pages=243-255.e5&author=J.+Congauthor=X.+Wangauthor=X.+Zhengauthor=D.+Wangauthor=B.+Fuauthor=R.+Sunauthor=Z.+Tianauthor=H.+Wei&title=Dysfunction+of+natural+killer+cells+by+FBP1-induced+inhibition+of+glycolysis+during+lung+cancer+progression&doi=10.1016%2Fj.cmet.2018.06.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Dysfunction of Natural Killer Cells by FBP1-Induced Inhibition of Glycolysis during Lung Cancer Progression</span></div><div class="casAuthors">Cong, Jingjing; Wang, Xianwei; Zheng, Xiaohu; Wang, Dong; Fu, Binqing; Sun, Rui; Tian, Zhigang; Wei, Haiming</div><div class="citationInfo"><span class="NLM_cas:title">Cell Metabolism</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">243-255.e5</span>CODEN:
                <span class="NLM_cas:coden">CMEEB5</span>;
        ISSN:<span class="NLM_cas:issn">1550-4131</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Natural killer (NK) cells are effector lymphocytes with pivotal roles in the resistance against various tumors; dysfunction of NK cells often results in advanced tumor progression.  Tumors develop in three stages comprising initiation, promotion, and progression, but little is known about the interrelationships between NK cells and tumor cells at different stages of tumor development.  Here, we demonstrated that NK cells prevented tumor initiation potently but did not prevent tumor promotion or tumor progression in Kras-driven lung cancer.  Moreover, loss of the antitumor effect in NK cells was closely assocd. with their dysfunctional state during tumor promotion and progression.  Mechanistically, aberrant fructose-1,6-bisphosphatase (FBP1) expression in NK cells elicited their dysfunction by inhibiting glycolysis and impairing viability.  Thus, our results show dynamic alterations of NK cells during tumor development and uncover a novel mechanism involved in NK cell dysfunction, suggesting potential directions for NK cell-based cancer immunotherapy involving FBP1 targeting.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoD-cwjrsMu87Vg90H21EOLACvtfcHk0ljE9YKhZ6-joA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlGlur%252FP&md5=5d13cd38666d2c2b184babd70011bcbe</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.cmet.2018.06.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cmet.2018.06.021%26sid%3Dliteratum%253Aachs%26aulast%3DCong%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DZheng%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DD.%26aulast%3DFu%26aufirst%3DB.%26aulast%3DSun%26aufirst%3DR.%26aulast%3DTian%26aufirst%3DZ.%26aulast%3DWei%26aufirst%3DH.%26atitle%3DDysfunction%2520of%2520natural%2520killer%2520cells%2520by%2520FBP1-induced%2520inhibition%2520of%2520glycolysis%2520during%2520lung%2520cancer%2520progression%26jtitle%3DCell%2520Metab.%26date%3D2018%26volume%3D28%26spage%3D243%26epage%3D255.e5%26doi%3D10.1016%2Fj.cmet.2018.06.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guo, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Broxmeyer, H. E.</span></span> <span> </span><span class="NLM_article-title">Antagonism of PPAR-γ signaling expands human hematopoietic stem and progenitor cells by enhancing glycolysis</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">360</span>– <span class="NLM_lpage">367</span>, <span class="refDoi"> DOI: 10.1038/nm.4477</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=10.1038%2Fnm.4477" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=29377004" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvVequrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2018&pages=360-367&author=B.+Guoauthor=X.+Huangauthor=M.+R.+Leeauthor=S.+A.+Leeauthor=H.+E.+Broxmeyer&title=Antagonism+of+PPAR-%CE%B3+signaling+expands+human+hematopoietic+stem+and+progenitor+cells+by+enhancing+glycolysis&doi=10.1038%2Fnm.4477"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Antagonism of PPAR-γ signaling expands human hematopoietic stem and progenitor cells by enhancing glycolysis</span></div><div class="casAuthors">Guo, Bin; Huang, Xinxin; Lee, Man Ryul; Lee, Sang A.; Broxmeyer, Hal E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">360-367</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Hematopoietic stem cells (HSCs) quiescently reside in bone marrow niches and have the capacity to self-renew or differentiate to form all of the blood cells throughout the lifespan of an animal.  Allogeneic HSC transplantation is a life-saving treatment for malignant and nonmalignant disorders.  HSCs isolated from umbilical cord blood (CB) are used for hematopoietic cell transplantation (HCT), but due to the limited nos. of HSCs in single units of umbilical CB, a no. of methods have been proposed for ex vivo expansion of human HSCs.  We show here that antagonism of peroxisome proliferator-activated receptor (PPAR)-γ promotes ex vivo expansion of phenotypically and functionally defined subsets of human CB HSCs and hematopoietic progenitor cells (HSPCs).  PPAR-γ antagonism in CB HSPCs strongly downregulated expression of several differentiation-assocd. genes, as well as fructose-bisphosphatase 1 (FBP1; which encodes a neg. regulator of glycolysis), and enhanced glycolysis without compromising mitochondrial metab.  The expansion of CB HSPCs by PPAR-γ antagonism was completely suppressed by removal of glucose or inhibition of glycolysis.  Moreover, knockdown of FBP1 expression promoted glycolysis and ex vivo expansion of long-term repopulating CB HSPCs, whereas overexpression of FBP1 suppressed the expansion of CB HSPCs that was induced by PPAR-γ antagonism.  Our study suggests the possibility for a new and simple means for metabolic reprogramming of CB HSPCs to improve the efficacy of HCT.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7LkTosYldjbVg90H21EOLACvtfcHk0ljE9YKhZ6-joA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvVequrY%253D&md5=52b707113e8ed91ce1a6d7897789f889</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnm.4477&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.4477%26sid%3Dliteratum%253Aachs%26aulast%3DGuo%26aufirst%3DB.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DLee%26aufirst%3DM.%2BR.%26aulast%3DLee%26aufirst%3DS.%2BA.%26aulast%3DBroxmeyer%26aufirst%3DH.%2BE.%26atitle%3DAntagonism%2520of%2520PPAR-%25CE%25B3%2520signaling%2520expands%2520human%2520hematopoietic%2520stem%2520and%2520progenitor%2520cells%2520by%2520enhancing%2520glycolysis%26jtitle%3DNat.%2520Med.%26date%3D2018%26volume%3D24%26spage%3D360%26epage%3D367%26doi%3D10.1038%2Fnm.4477" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rines, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharabi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tavares, C. D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Puigserver, P.</span></span> <span> </span><span class="NLM_article-title">Targeting hepatic glucose metabolism in the treatment of type 2 diabetes</span>. <i>Nat. Rev. Drug Discov.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">786</span>– <span class="NLM_lpage">804</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.151</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=10.1038%2Fnrd.2016.151" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=27516169" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhtlehur7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=786-804&author=A.+K.+Rinesauthor=K.+Sharabiauthor=C.+D.+J.+Tavaresauthor=P.+Puigserver&title=Targeting+hepatic+glucose+metabolism+in+the+treatment+of+type+2+diabetes&doi=10.1038%2Fnrd.2016.151"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting hepatic glucose metabolism in the treatment of type 2 diabetes</span></div><div class="casAuthors">Rines, Amy K.; Sharabi, Kfir; Tavares, Clint D. J.; Puigserver, Pere</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">786-804</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Type 2 diabetes mellitus is characterized by the dysregulation of glucose homeostasis, resulting in hyperglycemia.  Although current diabetes treatments have exhibited some success in lowering blood glucose levels, their effect is not always sustained and their use may be assocd. with undesirable side effects, such as hypoglycemia.  Novel antidiabetic drugs, which may be used in combination with existing therapies, are therefore needed.  The potential of specifically targeting the liver to normalize blood glucose levels has not been fully exploited.  Here, we review the mol. mechanisms controlling hepatic gluconeogenesis and glycogen storage, and assess the prospect of therapeutically targeting assocd. pathways to treat type 2 diabetes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZigCtBTlD37Vg90H21EOLACvtfcHk0ljE9YKhZ6-joA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhtlehur7J&md5=a51c9b6149bb4a6729f2bac583171865</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.151&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.151%26sid%3Dliteratum%253Aachs%26aulast%3DRines%26aufirst%3DA.%2BK.%26aulast%3DSharabi%26aufirst%3DK.%26aulast%3DTavares%26aufirst%3DC.%2BD.%2BJ.%26aulast%3DPuigserver%26aufirst%3DP.%26atitle%3DTargeting%2520hepatic%2520glucose%2520metabolism%2520in%2520the%2520treatment%2520of%2520type%25202%2520diabetes%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discov.%26date%3D2016%26volume%3D15%26spage%3D786%26epage%3D804%26doi%3D10.1038%2Fnrd.2016.151" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaur, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dahiya, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, M.</span></span> <span> </span><span class="NLM_article-title">Fructose-1,6-bisphosphatase inhibitors: A new valid approach for management of type 2 diabetes mellitus</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>141</i></span>,  <span class="NLM_fpage">473</span>– <span class="NLM_lpage">505</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2017.09.029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=10.1016%2Fj.ejmech.2017.09.029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=29055870" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslSiu73E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=141&publication_year=2017&pages=473-505&author=R.+Kaurauthor=L.+Dahiyaauthor=M.+Kumar&title=Fructose-1%2C6-bisphosphatase+inhibitors%3A+A+new+valid+approach+for+management+of+type+2+diabetes+mellitus&doi=10.1016%2Fj.ejmech.2017.09.029"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Fructose-1,6-bisphosphatase inhibitors: A new valid approach for management of type 2 diabetes mellitus</span></div><div class="casAuthors">Kaur, Ramandeep; Dahiya, Lalita; Kumar, Manoj</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">141</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">473-505</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A review.  The rising incidence of diabetes and confines allied with clin. therapies emphasized the need to explore new mol. targets to develop novel, effective and safer antihyperglycemic agents.  Excessive endogenous glucose prodn. by gluconeogenesis is a primary determinant of hyperglycemia in patients with type 2 diabetes.  But not even a single current medication acts directly to reduce gluconeogenesis.  Fructose-1,6-bisphosphatase (FBPase), a well recognized rate controlling enzyme of gluconeogenesis, has emerged as legitimate mol. level target to control gluconeogenesis mediated glucose overprodn. and its inhibitors are likely to fulfill an unmet medical need.  In this compilation various chem. classes of FBPase inhibitors have been reviewed which mainly acts through uncompetitive and non-competitive manner.  A detailed account on structure activity relationship studies of inhibitors have been presented along with their mol. level interactions at binding sites of enzyme.  Three Dimensional Quant. Structure Activity relationship (3D-QSAR) studies, performed to optimize the lead, have been summarized at some places.  In this assemblage, FBPase inhibitors patented in past have been compiled in tabular form.  This review highlights the new insight into the therapeutic utility of FBPase inhibitors and their potential as a new class of antidiabetic drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1W8b0O6Cbl7Vg90H21EOLACvtfcHk0lhHj9dS0wMb-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslSiu73E&md5=e2736803222e3b48cfd01bc230455742</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2017.09.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2017.09.029%26sid%3Dliteratum%253Aachs%26aulast%3DKaur%26aufirst%3DR.%26aulast%3DDahiya%26aufirst%3DL.%26aulast%3DKumar%26aufirst%3DM.%26atitle%3DFructose-1%252C6-bisphosphatase%2520inhibitors%253A%2520A%2520new%2520valid%2520approach%2520for%2520management%2520of%2520type%25202%2520diabetes%2520mellitus%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2017%26volume%3D141%26spage%3D473%26epage%3D505%26doi%3D10.1016%2Fj.ejmech.2017.09.029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bie, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Z.</span></span> <span> </span><span class="NLM_article-title">Discovery of novel indole derivatives as allosteric inhibitors of fructose-1,6-bisphosphatase</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>90</i></span>,  <span class="NLM_fpage">394</span>– <span class="NLM_lpage">405</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2014.11.049</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=10.1016%2Fj.ejmech.2014.11.049" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=25461330" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitVWhs7nL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=90&publication_year=2015&pages=394-405&author=J.+Bieauthor=S.+Liuauthor=Z.+Liauthor=Y.+Muauthor=B.+Xuauthor=Z.+Shen&title=Discovery+of+novel+indole+derivatives+as+allosteric+inhibitors+of+fructose-1%2C6-bisphosphatase&doi=10.1016%2Fj.ejmech.2014.11.049"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of novel indole derivatives as allosteric inhibitors of fructose-1,6-bisphosphatase</span></div><div class="casAuthors">Bie, Jianbo; Liu, Shuainan; Li, Zhanmei; Mu, Yongzhao; Xu, Bailing; Shen, Zhufang</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">90</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">394-405</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A series of novel indole derivs. was designed and synthesized as inhibitors of fructose-1,6-bisphosphatase (FBPase).  The most potent compd. 3-(2-Carboxyethyl)-5-ethyl-7-nitro-1H-indole-2-carboxylic acid was identified with an IC50 value of 0.10 μM by testing the inhibitory activity against recombinant human FBPase.  The structure-activity relations were studied on the substitution at 4- and 5-position of the indole scaffold.  The binding interactions of the title compds. at AMP binding site of FBPase were predicted using CDOCKER algorithm.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoINKqL176MDrVg90H21EOLACvtfcHk0lhHj9dS0wMb-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitVWhs7nL&md5=ce4e81ee3f0753d3013dae9e57252133</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2014.11.049&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2014.11.049%26sid%3Dliteratum%253Aachs%26aulast%3DBie%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DMu%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DB.%26aulast%3DShen%26aufirst%3DZ.%26atitle%3DDiscovery%2520of%2520novel%2520indole%2520derivatives%2520as%2520allosteric%2520inhibitors%2520of%2520fructose-1%252C6-bisphosphatase%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2015%26volume%3D90%26spage%3D394%26epage%3D405%26doi%3D10.1016%2Fj.ejmech.2014.11.049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">van
Poelje, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potter, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandramouli, V. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landau, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erion, M. D.</span></span> <span> </span><span class="NLM_article-title">Inhibition of fructose 1,6-bisphosphatase reduces excessive endogenous glucose production and attenuates hyperglycemia in Zucker diabetic fatty rats</span>. <i>Diabetes</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">1747</span>– <span class="NLM_lpage">1754</span>, <span class="refDoi"> DOI: 10.2337/db05-1443</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=10.2337%2Fdb05-1443" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=16731838" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BD28XlsF2rurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2006&pages=1747-1754&author=P.+D.+van%0APoeljeauthor=S.+C.+Potterauthor=V.+C.+Chandramouliauthor=B.+R.+Landauauthor=Q.+Dangauthor=M.+D.+Erion&title=Inhibition+of+fructose+1%2C6-bisphosphatase+reduces+excessive+endogenous+glucose+production+and+attenuates+hyperglycemia+in+Zucker+diabetic+fatty+rats&doi=10.2337%2Fdb05-1443"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of fructose 1,6-bisphosphatase reduces excessive endogenous glucose production and attenuates hyperglycemia in Zucker diabetic fatty rats</span></div><div class="casAuthors">van Poelje, Paul D.; Potter, Scott C.; Chandramouli, Visvanathan C.; Landau, Bernard R.; Dang, Qun; Erion, Mark D.</div><div class="citationInfo"><span class="NLM_cas:title">Diabetes</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1747-1754</span>CODEN:
                <span class="NLM_cas:coden">DIAEAZ</span>;
        ISSN:<span class="NLM_cas:issn">0012-1797</span>.
    
            (<span class="NLM_cas:orgname">American Diabetes Association</span>)
        </div><div class="casAbstract">Gluconeogenesis is increased in type 2 diabetes and contributes significantly to fasting and postprandial hyperglycemia.  We recently reported the discovery of the 1st potent and selective inhibitors of fructose 1,6-bisphosphatase (FBPase), a rate-controlling enzyme of gluconeogenesis.  Herein we describe acute and chronic effects of the lead inhibitor, MB06322 (CS-917), in rodent models of type 2 diabetes.  In fasting male ZDF rats with overt diabetes, a single dose of MB06322 inhibited gluconeogenesis by 70% and overall endogenous glucose prodn. by 46%, leading to a redn. in blood glucose of >200 mg/dL.  Chronic treatment of freely feeding 6-wk-old male Zucker diabetic fatty (ZDF) rats delayed the development of hyperglycemia and preserved pancreatic function.  Elevation of lactate (∼1.5-fold) occurred after 4 wk of treatment, as did the apparent shunting of precursors into triglycerides.  Profound glucose lowering (∼44%) and similar metabolic ramifications were assocd. with 2-wk intervention therapy of 10-wk-old male ZDF rats.  In high-fat diet-fed female ZDF rats, MB06322 treatment for 2 wk fully attenuated hyperglycemia without evidence of metabolic perturbation other than a modest redn. in glycogen stores (∼20%).  The studies confirm that excessive gluconeogenesis plays an integral role in the pathophysiol. of type 2 diabetes and suggest that FBPase inhibitors may provide a future treatment option.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoW3d61gJUTBrVg90H21EOLACvtfcHk0lhHj9dS0wMb-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XlsF2rurc%253D&md5=f85ef1bcc9b9e2de186efe2afe45643c</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.2337%2Fdb05-1443&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2337%252Fdb05-1443%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2BPoelje%26aufirst%3DP.%2BD.%26aulast%3DPotter%26aufirst%3DS.%2BC.%26aulast%3DChandramouli%26aufirst%3DV.%2BC.%26aulast%3DLandau%26aufirst%3DB.%2BR.%26aulast%3DDang%26aufirst%3DQ.%26aulast%3DErion%26aufirst%3DM.%2BD.%26atitle%3DInhibition%2520of%2520fructose%25201%252C6-bisphosphatase%2520reduces%2520excessive%2520endogenous%2520glucose%2520production%2520and%2520attenuates%2520hyperglycemia%2520in%2520Zucker%2520diabetic%2520fatty%2520rats%26jtitle%3DDiabetes%26date%3D2006%26volume%3D55%26spage%3D1747%26epage%3D1754%26doi%3D10.2337%2Fdb05-1443" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dang, Q.</span>; <span class="NLM_string-name">Van
Poelje, P. D.</span>; <span class="NLM_string-name">Erion, M. D.</span></span> <span> </span><span class="NLM_article-title">The discovery and development of MB07803, a second-generation fructose-1,6-bisphosphatase inhibitor with improved pharmacokinetic properties, as a potential treatment of type 2 diabetes</span>. In  <i>New Therapeutic Strategies for Type 2 Diabetes: Small Molecule Approaches</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robert, M. J.</span></span>; <span class="NLM_publisher-name">Royal Society of Chemistry</span>, <span class="NLM_publisher-loc">London</span> <span class="NLM_year">2012</span>, pp  <span class="NLM_fpage">306</span>– <span class="NLM_lpage">323</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=10.1039%2F9781849735322-00306" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&pages=306-323&author=Q.+Dang&author=P.+D.+Van%0APoelje&author=M.+D.+Erionauthor=M.+J.+Robert&title=New+Therapeutic+Strategies+for+Type+2+Diabetes%3A+Small+Molecule+Approaches"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1039%2F9781849735322-00306&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1039%252F9781849735322-00306%26sid%3Dliteratum%253Aachs%26aulast%3DDang%26aufirst%3DQ.%26atitle%3DThe%2520discovery%2520and%2520development%2520of%2520MB07803%252C%2520a%2520second-generation%2520fructose-1%252C6-bisphosphatase%2520inhibitor%2520with%2520improved%2520pharmacokinetic%2520properties%252C%2520as%2520a%2520potential%2520treatment%2520of%2520type%25202%2520diabetes%26btitle%3DNew%2520Therapeutic%2520Strategies%2520for%2520Type%25202%2520Diabetes%253A%2520Small%2520Molecule%2520Approaches%26aulast%3DRobert%26aufirst%3DM.%2BJ.%26pub%3DRoyal%2520Society%2520of%2520Chemistry%26date%3D2012%26spage%3D306%26epage%3D323" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasibhatla, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potter, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujitaki, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Poelje, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lipscomb, W. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erion, M. D.</span></span> <span> </span><span class="NLM_article-title">Discovery of potent and specific fructose-1,6-bisphosphatase inhibitors and a series of orally-bioavailable phosphoramidase-sensitive prodrugs for the treatment of type 2 diabetes</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>129</i></span>,  <span class="NLM_fpage">15491</span>– <span class="NLM_lpage">15502</span>, <span class="refDoi"> DOI: 10.1021/ja074871l</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja074871l" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlOgsrjP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=129&publication_year=2007&pages=15491-15502&author=Q.+Dangauthor=S.+R.+Kasibhatlaauthor=K.+R.+Reddyauthor=T.+Jiangauthor=M.+R.+Reddyauthor=S.+C.+Potterauthor=J.+M.+Fujitakiauthor=P.+D.+van+Poeljeauthor=J.+Huangauthor=W.+N.+Lipscombauthor=M.+D.+Erion&title=Discovery+of+potent+and+specific+fructose-1%2C6-bisphosphatase+inhibitors+and+a+series+of+orally-bioavailable+phosphoramidase-sensitive+prodrugs+for+the+treatment+of+type+2+diabetes&doi=10.1021%2Fja074871l"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Potent and Specific Fructose-1,6-Bisphosphatase Inhibitors and a Series of Orally-Bioavailable Phosphoramidase-Sensitive Prodrugs for the Treatment of Type 2 Diabetes</span></div><div class="casAuthors">Dang, Qun; Rao Kasibhatla, Srinivas; Reddy, K. Raja; Jiang, Tao; Reddy, M. Rami; Potter, Scott C.; Fujitaki, James M.; van Poelje, Paul D.; Huang, Jingwei; Lipscomb, William N.; Erion, Mark D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">129</span>
        (<span class="NLM_cas:issue">50</span>),
    <span class="NLM_cas:pages">15491-15502</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Excessive glucose prodn. by the liver coupled with decreased glucose uptake and metab. by muscle, fat, and liver results in chronically elevated blood glucose levels in patients with type 2 diabetes.  Efforts to treat diabetes by reducing glucose prodn. have largely focused on the gluconeogenesis pathway and rate-limiting enzymes within this pathway such as fructose-1,6-bisphosphatase (FBPase).  The first potent FBPase inhibitors were identified using a structure-guided drug design strategy but proved difficult to deliver orally.  Herein, we report the synthesis and characterization of a series of orally bioavailable FBPase inhibitors identified following the combined discoveries of a low mol. wt. inhibitor series with increased potency and a phosphonate prodrug class suitable for their oral delivery.  The lead inhibitor, 10A (I), was designed with the aid of X-ray crystallog. and mol. modeling to bind to the allosteric AMP binding site of FBPase.  High potency (IC50 = 16 nM) and FBPase specificity were achieved by linking a 2-aminothiazole with a phosphonic acid.  Free-energy perturbation calcns. provided insight into the factors that contributed to the high binding affinity.  10A and std. phosphonate prodrugs of 10A exhibited poor oral bioavailability (0.2-11%).  Improved oral bioavailability (22-47%) was achieved using phosphonate diamides that convert to the corresponding phosphonic acid by sequential action of an esterase and a phosphoramidase.  Oral administration of the lead prodrug, MB06322 (30, CS-917), to Zucker Diabetic Fatty rats led to dose-dependent inhibition of gluconeogenesis and endogenous glucose prodn. and consequently to significant blood glucose redn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopA52xpsptwLVg90H21EOLACvtfcHk0ljuwDoT9mfPUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlOgsrjP&md5=ea5856c4481a445d06f45882a8562def</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Fja074871l&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja074871l%26sid%3Dliteratum%253Aachs%26aulast%3DDang%26aufirst%3DQ.%26aulast%3DKasibhatla%26aufirst%3DS.%2BR.%26aulast%3DReddy%26aufirst%3DK.%2BR.%26aulast%3DJiang%26aufirst%3DT.%26aulast%3DReddy%26aufirst%3DM.%2BR.%26aulast%3DPotter%26aufirst%3DS.%2BC.%26aulast%3DFujitaki%26aufirst%3DJ.%2BM.%26aulast%3Dvan%2BPoelje%26aufirst%3DP.%2BD.%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DLipscomb%26aufirst%3DW.%2BN.%26aulast%3DErion%26aufirst%3DM.%2BD.%26atitle%3DDiscovery%2520of%2520potent%2520and%2520specific%2520fructose-1%252C6-bisphosphatase%2520inhibitors%2520and%2520a%2520series%2520of%2520orally-bioavailable%2520phosphoramidase-sensitive%2520prodrugs%2520for%2520the%2520treatment%2520of%2520type%25202%2520diabetes%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2007%26volume%3D129%26spage%3D15491%26epage%3D15502%26doi%3D10.1021%2Fja074871l" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Erion, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasibhatla, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lipscomb, W. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Poelje, P. D.</span></span> <span> </span><span class="NLM_article-title">Structure-guided design of AMP mimics that inhibit fructose-1,6-bisphosphatase with high affinity and specificity</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>129</i></span>,  <span class="NLM_fpage">15480</span>– <span class="NLM_lpage">15490</span>, <span class="refDoi"> DOI: 10.1021/ja074869u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja074869u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlOgsrrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=129&publication_year=2007&pages=15480-15490&author=M.+D.+Erionauthor=Q.+Dangauthor=M.+R.+Reddyauthor=S.+R.+Kasibhatlaauthor=J.+Huangauthor=W.+N.+Lipscombauthor=P.+D.+van+Poelje&title=Structure-guided+design+of+AMP+mimics+that+inhibit+fructose-1%2C6-bisphosphatase+with+high+affinity+and+specificity&doi=10.1021%2Fja074869u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Guided Design of AMP Mimics That Inhibit Fructose-1,6-bisphosphatase with High Affinity and Specificity</span></div><div class="casAuthors">Erion, Mark D.; Dang, Qun; Reddy, M. Rami; Kasibhatla, Srinivas Rao; Huang, Jingwei; Lipscomb, William N.; van Poelje, Paul D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">129</span>
        (<span class="NLM_cas:issue">50</span>),
    <span class="NLM_cas:pages">15480-15490</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">AMP binding sites are commonly used by nature for allosteric regulation of enzymes controlling the prodn. and metab. of carbohydrates and lipids.  Since many of these enzymes represent potential drug targets for metabolic diseases, efforts were initiated to discover AMP mimics that bind to AMP-binding sites with high affinity and high enzyme specificity.  Herein the authors report the structure-guided design of potent fructose 1,6-bisphosphatase (FBPase) inhibitors that interact with the AMP binding site on FBPase despite their structural dissimilarity to AMP.  Mol. modeling, free-energy perturbation calcns., x-ray crystallog., and enzyme kinetic data guided the redesign of AMP, which began by replacing the 5'-phosphate with a phosphonic acid attached to C8 of the adenine base via a 3-atom spacer.  Addnl. binding affinity was gained by replacing the ribose with an alkyl group that formed van der Waals interactions with a hydrophobic region within the AMP binding site and by replacing the purine nitrogens N1 and N3 with carbons to minimize desolvation energy expenditures.  The resulting benzimidazole phosphonic acid, 16, inhibited human FBPase (IC50 = 90 nM) 11-fold more potently than AMP and exhibited high specificity for the AMP binding site on FBPase.  16 Also inhibited FBPase in primary rat hepatocytes and correspondingly resulted in concn.-dependent inhibition of the gluconeogenesis pathway.  Accordingly, these results suggest that the AMP site of FBPase may represent a potential drug target for reducing the excessive glucose produced by the gluconeogenesis pathway in patients with type 2 diabetes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5aPGR2PR-9bVg90H21EOLACvtfcHk0ljuwDoT9mfPUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlOgsrrJ&md5=15c15e648a7188fd3280be2c8c17b35f</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Fja074869u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja074869u%26sid%3Dliteratum%253Aachs%26aulast%3DErion%26aufirst%3DM.%2BD.%26aulast%3DDang%26aufirst%3DQ.%26aulast%3DReddy%26aufirst%3DM.%2BR.%26aulast%3DKasibhatla%26aufirst%3DS.%2BR.%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DLipscomb%26aufirst%3DW.%2BN.%26aulast%3Dvan%2BPoelje%26aufirst%3DP.%2BD.%26atitle%3DStructure-guided%2520design%2520of%2520AMP%2520mimics%2520that%2520inhibit%2520fructose-1%252C6-bisphosphatase%2520with%2520high%2520affinity%2520and%2520specificity%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2007%26volume%3D129%26spage%3D15480%26epage%3D15490%26doi%3D10.1021%2Fja074869u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nussinov, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, C. J.</span></span> <span> </span><span class="NLM_article-title">Allostery in disease and in drug discovery</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>153</i></span>,  <span class="NLM_fpage">293</span>– <span class="NLM_lpage">305</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2013.03.034</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=10.1016%2Fj.cell.2013.03.034" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=23582321" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlvVChu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=153&publication_year=2013&pages=293-305&author=R.+Nussinovauthor=C.+J.+Tsai&title=Allostery+in+disease+and+in+drug+discovery&doi=10.1016%2Fj.cell.2013.03.034"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Allostery in Disease and in Drug Discovery</span></div><div class="casAuthors">Nussinov, Ruth; Tsai, Chung-Jung</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">153</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">293-305</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Allostery is largely assocd. with conformational and functional transitions in individual proteins.  This concept can be extended to consider the impact of conformational perturbations on cellular function and disease states.  Here, we clarify the concept of allostery and how it controls physiol. activities.  We focus on the challenging questions of how allostery can both cause disease and contribute to development of new therapeutics.  We aim to increase the awareness of the linkage between disease symptoms on the cellular level and specific aberrant allosteric actions on the mol. level and to emphasize the potential of allosteric drugs in innovative therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrz5bXPgUjfFLVg90H21EOLACvtfcHk0lhCvOc9sUgGHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlvVChu70%253D&md5=14a91c0d364187044e5ad3e048868514</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2013.03.034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2013.03.034%26sid%3Dliteratum%253Aachs%26aulast%3DNussinov%26aufirst%3DR.%26aulast%3DTsai%26aufirst%3DC.%2BJ.%26atitle%3DAllostery%2520in%2520disease%2520and%2520in%2520drug%2520discovery%26jtitle%3DCell%26date%3D2013%26volume%3D153%26spage%3D293%26epage%3D305%26doi%3D10.1016%2Fj.cell.2013.03.034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">Allosteric methods and their applications: facilitating the discovery of allosteric drugs and the investigation of allosteric mechanisms</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">492</span>– <span class="NLM_lpage">500</span>, <span class="refDoi"> DOI: 10.1021/acs.accounts.8b00570</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.accounts.8b00570" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvVWrt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2019&pages=492-500&author=S.+Luauthor=Q.+Shenauthor=J.+Zhang&title=Allosteric+methods+and+their+applications%3A+facilitating+the+discovery+of+allosteric+drugs+and+the+investigation+of+allosteric+mechanisms&doi=10.1021%2Facs.accounts.8b00570"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric Methods and Their Applications: Facilitating the Discovery of Allosteric Drugs and the Investigation of Allosteric Mechanisms</span></div><div class="casAuthors">Lu, Shaoyong; Shen, Qiancheng; Zhang, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">492-500</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Allostery, or allosteric regulation, is the phenomenon in which protein functional activity is altered by the binding of an effector at an allosteric site that is topog. distinct from the orthosteric, active site.  As one of the most direct and efficient ways to regulate protein function, allostery has played a fundamental role in innumerable biol. processes of all living organisms, including enzyme catalysis, signal transduction, cell metab., and gene transcription.  It is thus considered as "the second secret of life".  The abnormality of allosteric communication networks between allosteric and orthosteric sites is assocd. with the pathogenesis of human diseases.  Allosteric modulators, by attaching to structurally diverse allosteric sites, offer the potential for differential selectivity and improved safety compared with orthosteric drugs that bind to conserved orthosteric sites.  Harnessing allostery has thus been regarded as a novel strategy for drug discovery.  Despite much progress having been made in the repertoire of allostery since the turn of the millennium, the identification of allosteric drugs for therapeutic targets and the elucidation of allosteric mechanisms still present substantial challenges.  These challenges are derived from the difficulties in the identification of allosteric sites and mutations, the assessment of allosteric protein-modulator interactions, the screening of allosteric modulators, and the elucidation of allosteric mechanisms in biol. systems.To address these issues, we have developed a panel of allosteric services for specific allosteric applications over the past decade, including (i) the creation of the Allosteric Database, with the aim of providing comprehensive allosteric information such as allosteric proteins, modulators, sites, pathways, etc., (ii) the construction of the ASBench benchmark of high-quality allosteric sites for the development of computational methods for predicting allosteric sites, (iii) the development of Allosite and AllositePro for the prediction of the location of allosteric sites in proteins, (iv) the development of the Alloscore scoring function for the evaluation of allosteric protein-modulator interactions, (v) the development of Allosterome for evolutionary anal. of query allosteric sites/modulators within the human proteome, (vi) the development of AlloDriver for the prediction of allosteric mutagenesis, and (vii) the development of AlloFinder for the virtual screening of allosteric modulators and the investigation of allosteric mechanisms.  Importantly, we have validated computationally predicted allosteric sites, mutations, and modulators in the real cases of sirtuin 6, casein kinase 2α, phosphodiesterase 10A, and signal transduction and activation of transcription 3.  Furthermore, our developed allosteric methods have been widely exploited by other users around the world for allosteric research.  Therefore, these allosteric services are expected to expedite the discovery of allosteric drugs and the investigation of allosteric mechanisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrlHNG36v7_bVg90H21EOLACvtfcHk0lhCvOc9sUgGHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvVWrt7o%253D&md5=a23fcee3def606164d2a22b0681b5837</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Facs.accounts.8b00570&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.accounts.8b00570%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DS.%26aulast%3DShen%26aufirst%3DQ.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DAllosteric%2520methods%2520and%2520their%2520applications%253A%2520facilitating%2520the%2520discovery%2520of%2520allosteric%2520drugs%2520and%2520the%2520investigation%2520of%2520allosteric%2520mechanisms%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2019%26volume%3D52%26spage%3D492%26epage%3D500%26doi%3D10.1021%2Facs.accounts.8b00570" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lonsdale, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, R. A.</span></span> <span> </span><span class="NLM_article-title">Structure-based design of targeted covalent inhibitors</span>. <i>Chem. Soc. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">3816</span>– <span class="NLM_lpage">3830</span>, <span class="refDoi"> DOI: 10.1039/C7CS00220C</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=10.1039%2FC7CS00220C" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=29620097" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BC1cXntVGiurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2018&pages=3816-3830&author=R.+Lonsdaleauthor=R.+A.+Ward&title=Structure-based+design+of+targeted+covalent+inhibitors&doi=10.1039%2FC7CS00220C"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based design of targeted covalent inhibitors</span></div><div class="casAuthors">Lonsdale, Richard; Ward, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Society Reviews</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3816-3830</span>CODEN:
                <span class="NLM_cas:coden">CSRVBR</span>;
        ISSN:<span class="NLM_cas:issn">0306-0012</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Covalent inhibition is a rapidly growing discipline within drug discovery.  Many historical covalent inhibitors were discovered by serendipity, with such a mechanism of action often regarded as undesirable due to potential toxicity issues.  Recent progress has seen a major shift in this outlook, as covalent inhibition shows promise for targets where previous efforts to identify non-covalent small mol. inhibitors have failed.  Targeted covalent inhibitors (TCIs) can offer drug discovery scientists the ability to increase the potency and/or selectivity of small mol. inhibitors, by attachment of reactive functional groups designed to covalently bind to specific sites in a target.  In this tutorial review we introduce the broader concept of covalent inhibition, discuss the potential benefits and challenges of such an approach, and provide an overview of the current status of the field.  We also describe some strategies and computational tools to enable successful covalent drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqnSIGMuS3vm7Vg90H21EOLACvtfcHk0lhCvOc9sUgGHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXntVGiurs%253D&md5=a7e31b23cf8ecfcad1397ad10ea10b7b</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1039%2FC7CS00220C&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7CS00220C%26sid%3Dliteratum%253Aachs%26aulast%3DLonsdale%26aufirst%3DR.%26aulast%3DWard%26aufirst%3DR.%2BA.%26atitle%3DStructure-based%2520design%2520of%2520targeted%2520covalent%2520inhibitors%26jtitle%3DChem.%2520Soc.%2520Rev.%26date%3D2018%26volume%3D47%26spage%3D3816%26epage%3D3830%26doi%3D10.1039%2FC7CS00220C" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Singh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petter, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baillie, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitty, A.</span></span> <span> </span><span class="NLM_article-title">The resurgence of covalent drugs</span>. <i>Nat. Rev. Drug Discov.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">307</span>– <span class="NLM_lpage">317</span>, <span class="refDoi"> DOI: 10.1038/nrd3410</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=10.1038%2Fnrd3410" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=21455239" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktVGmu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=307-317&author=J.+Singhauthor=R.+C.+Petterauthor=T.+A.+Baillieauthor=A.+Whitty&title=The+resurgence+of+covalent+drugs&doi=10.1038%2Fnrd3410"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">The resurgence of covalent drugs</span></div><div class="casAuthors">Singh, Juswinder; Petter, Russell C.; Baillie, Thomas A.; Whitty, Adrian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">307-317</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Covalent drugs haveproved to be successful therapies for various indications, but largely owing to safety concerns, they are rarely considered when initiating a target-directed drug discovery project.  There is a need to reassess this important class of drugs, and to reconcile the discordance between the historic success of covalent drugs and the reluctance of most drug discovery teams to include them in their armamentarium.  This Review surveys the prevalence and pharmacol. advantages of covalent drugs, discusses how potential risks and challenges may be addressed through innovative design, and presents the broad opportunities provided by targeted covalent inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreDCEmORV_NbVg90H21EOLACvtfcHk0lhCvOc9sUgGHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktVGmu7g%253D&md5=2190289081e151416c097be4a5b04460</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fnrd3410&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3410%26sid%3Dliteratum%253Aachs%26aulast%3DSingh%26aufirst%3DJ.%26aulast%3DPetter%26aufirst%3DR.%2BC.%26aulast%3DBaillie%26aufirst%3DT.%2BA.%26aulast%3DWhitty%26aufirst%3DA.%26atitle%3DThe%2520resurgence%2520of%2520covalent%2520drugs%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discov.%26date%3D2011%26volume%3D10%26spage%3D307%26epage%3D317%26doi%3D10.1038%2Fnrd3410" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">Designed covalent allosteric modulators: an emerging paradigm in drug discovery</span>. <i>Drug Discov. Today</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">447</span>– <span class="NLM_lpage">453</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2016.11.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=10.1016%2Fj.drudis.2016.11.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=27888140" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFOju7zJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2017&pages=447-453&author=S.+Luauthor=J.+Zhang&title=Designed+covalent+allosteric+modulators%3A+an+emerging+paradigm+in+drug+discovery&doi=10.1016%2Fj.drudis.2016.11.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Designed covalent allosteric modulators: an emerging paradigm in drug discovery</span></div><div class="casAuthors">Lu, Shaoyong; Zhang, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">447-453</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Covalent allosteric modulators possess the pharmacol. advantages (high potency, extended duration of action and low drug resistance) of covalent ligands and the addnl. benefit of the higher specificity and lower toxicity of allosteric modulators.  This approach is gaining increasing recognition as a valuable tool in drug discovery.  Here, we review the recent advances in the design of covalent allosteric modulators with an emphasis on successful examples.  A broad spectrum of protein targets capable of being modulated by them reflects the prevalence of this strategy.  We also discuss the challenges and future directions in the development of covalent allosteric modulators.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqnMtSOrLWxdrVg90H21EOLACvtfcHk0lhPF4vFfLRuFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFOju7zJ&md5=bc656114f3f956ce44ba06c1a7669da4</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2016.11.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2016.11.013%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DDesigned%2520covalent%2520allosteric%2520modulators%253A%2520an%2520emerging%2520paradigm%2520in%2520drug%2520discovery%26jtitle%3DDrug%2520Discov.%2520Today%26date%3D2017%26volume%3D22%26spage%3D447%26epage%3D453%26doi%3D10.1016%2Fj.drudis.2016.11.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nussinov, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, C. J.</span></span> <span> </span><span class="NLM_article-title">The design of covalent allosteric drugs</span>. <i>Annu. Rev. Pharmacol. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">249</span>– <span class="NLM_lpage">267</span>, <span class="refDoi"> DOI: 10.1146/annurev-pharmtox-010814-124401</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=10.1146%2Fannurev-pharmtox-010814-124401" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=25149918" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BC2MXkt12qsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=249-267&author=R.+Nussinovauthor=C.+J.+Tsai&title=The+design+of+covalent+allosteric+drugs&doi=10.1146%2Fannurev-pharmtox-010814-124401"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">The design of covalent allosteric drugs</span></div><div class="casAuthors">Nussinov, Ruth; Tsai, Chung-Jung</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Pharmacology and Toxicology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">249-267</span>CODEN:
                <span class="NLM_cas:coden">ARPTDI</span>;
        ISSN:<span class="NLM_cas:issn">0362-1642</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">A review.  A key issue in drug discovery is how to reduce drug dosage and increase specificity while retaining or increasing efficacy, as high dosage is often linked to toxicity.  There are two types of drugs on the market: orthosteric and allosteric.  Orthosteric drugs can be noncovalent or covalent.  The latter are advantageous because they may be prescribed in lower doses, but their potential off-target toxicity is a primary concern.  The chief advantages of allosteric drugs are their higher specificity and their consequently lower chance of toxic side effects.  Covalent allosteric drugs combine the pharmacol. merits of covalent drugs with the addnl. benefit of the higher specificity of allosteric drugs.  In a recent promising step in therapeutic drug development, allosteric, disulfide-tethered fragments successfully modulated the activity of a protein kinase and K-Ras.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCurPEiYvhQrVg90H21EOLACvtfcHk0lhPF4vFfLRuFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXkt12qsrc%253D&md5=b85640442af2255165ba382bf00abd3d</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1146%2Fannurev-pharmtox-010814-124401&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-pharmtox-010814-124401%26sid%3Dliteratum%253Aachs%26aulast%3DNussinov%26aufirst%3DR.%26aulast%3DTsai%26aufirst%3DC.%2BJ.%26atitle%3DThe%2520design%2520of%2520covalent%2520allosteric%2520drugs%26jtitle%3DAnnu.%2520Rev.%2520Pharmacol.%2520Toxicol.%26date%3D2015%26volume%3D55%26spage%3D249%26epage%3D267%26doi%3D10.1146%2Fannurev-pharmtox-010814-124401" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Uhlenbrock, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weisner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landel, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindemann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hardick, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gontla, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scheinpflug, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czodrowski, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janning, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Depta, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quambusch, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engels, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rauh, D.</span></span> <span> </span><span class="NLM_article-title">Structural and chemical insights into the covalent-allosteric inhibition of the protein kinase Akt</span>. <i>Chem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">3573</span>– <span class="NLM_lpage">3585</span>, <span class="refDoi"> DOI: 10.1039/C8SC05212C</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=10.1039%2FC8SC05212C" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=30996949" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BC1MXivVOlu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=3573-3585&author=N.+Uhlenbrockauthor=S.+Smithauthor=J.+Weisnerauthor=I.+Landelauthor=M.+Lindemannauthor=T.+A.+Leauthor=J.+Hardickauthor=R.+Gontlaauthor=R.+Scheinpflugauthor=P.+Czodrowskiauthor=P.+Janningauthor=L.+Deptaauthor=L.+Quambuschauthor=M.+P.+M%C3%BCllerauthor=B.+Engelsauthor=D.+Rauh&title=Structural+and+chemical+insights+into+the+covalent-allosteric+inhibition+of+the+protein+kinase+Akt&doi=10.1039%2FC8SC05212C"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Structural and chemical insights into the covalent-allosteric inhibition of the protein kinase Akt</span></div><div class="casAuthors">Uhlenbrock, Niklas; Smith, Steven; Weisner, Joern; Landel, Ina; Lindemann, Marius; Le, Thien Anh; Hardick, Julia; Gontla, Rajesh; Scheinpflug, Rebekka; Czodrowski, Paul; Janning, Petra; Depta, Laura; Quambusch, Lena; Mueller, Matthias P.; Engels, Bernd; Rauh, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3573-3585</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The Ser/Thr kinase Akt (Protein Kinase B/PKB) is a master switch in cellular signal transduction pathways.  Its downstream signaling influences cell proliferation, cell growth, and apoptosis, rendering Akt a prominent drug target.  The unique activation mechanism of Akt involves a change of the relative orientation of its N-terminal pleckstrin homol. (PH) and the kinase domain and makes this kinase suitable for highly specific allosteric modulation.  Here we present a unique set of crystal structures of covalent-allosteric interdomain inhibitors in complex with full-length Akt and report the structure-based design, synthesis, biol. and pharmacol. evaluation of a focused library of these innovative inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxp6UFlZqI5bVg90H21EOLACvtfcHk0lhPF4vFfLRuFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXivVOlu7w%253D&md5=4ff6ee8b9d7f5e7ac9840158fafc32ef</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1039%2FC8SC05212C&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC8SC05212C%26sid%3Dliteratum%253Aachs%26aulast%3DUhlenbrock%26aufirst%3DN.%26aulast%3DSmith%26aufirst%3DS.%26aulast%3DWeisner%26aufirst%3DJ.%26aulast%3DLandel%26aufirst%3DI.%26aulast%3DLindemann%26aufirst%3DM.%26aulast%3DLe%26aufirst%3DT.%2BA.%26aulast%3DHardick%26aufirst%3DJ.%26aulast%3DGontla%26aufirst%3DR.%26aulast%3DScheinpflug%26aufirst%3DR.%26aulast%3DCzodrowski%26aufirst%3DP.%26aulast%3DJanning%26aufirst%3DP.%26aulast%3DDepta%26aufirst%3DL.%26aulast%3DQuambusch%26aufirst%3DL.%26aulast%3DM%25C3%25BCller%26aufirst%3DM.%2BP.%26aulast%3DEngels%26aufirst%3DB.%26aulast%3DRauh%26aufirst%3DD.%26atitle%3DStructural%2520and%2520chemical%2520insights%2520into%2520the%2520covalent-allosteric%2520inhibition%2520of%2520the%2520protein%2520kinase%2520Akt%26jtitle%3DChem.%2520Sci.%26date%3D2019%26volume%3D10%26spage%3D3573%26epage%3D3585%26doi%3D10.1039%2FC8SC05212C" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ostrem, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peters, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sos, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shokat, K. M.</span></span> <span> </span><span class="NLM_article-title">K-Ras(G12C) inhibitors allosterically control GTP affinity and effector interactions</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>503</i></span>,  <span class="NLM_fpage">548</span>– <span class="NLM_lpage">551</span>, <span class="refDoi"> DOI: 10.1038/nature12796</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=10.1038%2Fnature12796" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=24256730" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVGqs7jO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=503&publication_year=2013&pages=548-551&author=J.+M.+Ostremauthor=U.+Petersauthor=M.+L.+Sosauthor=J.+A.+Wellsauthor=K.+M.+Shokat&title=K-Ras%28G12C%29+inhibitors+allosterically+control+GTP+affinity+and+effector+interactions&doi=10.1038%2Fnature12796"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">K-Ras(G12C) inhibitors allosterically control GTP affinity and effector interactions</span></div><div class="casAuthors">Ostrem, Jonathan M.; Peters, Ulf; Sos, Martin L.; Wells, James A.; Shokat, Kevan M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">503</span>
        (<span class="NLM_cas:issue">7477</span>),
    <span class="NLM_cas:pages">548-551</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Somatic mutations in the small GTPase K-Ras are the most common activating lesions found in human cancer, and are generally assocd. with poor response to std. therapies.  Efforts to target this oncogene directly have faced difficulties owing to its picomolar affinity for GTP/GDP and the absence of known allosteric regulatory sites.  Oncogenic mutations result in functional activation of Ras family proteins by impairing GTP hydrolysis.  With diminished regulation by GTPase activity, the nucleotide state of Ras becomes more dependent on relative nucleotide affinity and concn.  This gives GTP an advantage over GDP and increases the proportion of active GTP-bound Ras.  Here we report the development of small mols. that irreversibly bind to a common oncogenic mutant, K-Ras(G12C).  These compds. rely on the mutant cysteine for binding and therefore do not affect the wild-type protein.  Crystallog. studies reveal the formation of a new pocket that is not apparent in previous structures of Ras, beneath the effector binding switch-II region.  Binding of these inhibitors to K-Ras(G12C) disrupts both switch-I and switch-II, subverting the native nucleotide preference to favor GDP over GTP and impairing binding to Raf.  Our data provide structure-based validation of a new allosteric regulatory site on Ras that is targetable in a mutant-specific manner.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokkgcVXuBn_rVg90H21EOLACvtfcHk0lhDuQDYVN4xlw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvVGqs7jO&md5=27c1e7d8ef6b7e4b2919e9f00a073923</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fnature12796&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12796%26sid%3Dliteratum%253Aachs%26aulast%3DOstrem%26aufirst%3DJ.%2BM.%26aulast%3DPeters%26aufirst%3DU.%26aulast%3DSos%26aufirst%3DM.%2BL.%26aulast%3DWells%26aufirst%3DJ.%2BA.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26atitle%3DK-Ras%2528G12C%2529%2520inhibitors%2520allosterically%2520control%2520GTP%2520affinity%2520and%2520effector%2520interactions%26jtitle%3DNature%26date%3D2013%26volume%3D503%26spage%3D548%26epage%3D551%26doi%3D10.1038%2Fnature12796" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Janes, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peters, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Babbar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Firdaus, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darjania, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, Y. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thach, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zarieh, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ely, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kucharski, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kessler, L. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zarrinkar, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brehmer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhanak, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lorenzi, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu-Lowe, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patricelli, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span> <span> </span><span class="NLM_article-title">Targeting KRAS mutant cancers with a covalent G12C-specific inhibitor</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>172</i></span>,  <span class="NLM_fpage">578</span>– <span class="NLM_lpage">589.e17</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2018.01.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=10.1016%2Fj.cell.2018.01.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=29373830" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BC1cXislyisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=172&publication_year=2018&pages=578-589.e17&author=M.+R.+Janesauthor=J.+Zhangauthor=L.+S.+Liauthor=R.+Hansenauthor=U.+Petersauthor=X.+Guoauthor=Y.+Chenauthor=A.+Babbarauthor=S.+J.+Firdausauthor=L.+Darjaniaauthor=J.+Fengauthor=J.+H.+Chenauthor=S.+Liauthor=S.+Liauthor=Y.+O.+Longauthor=C.+Thachauthor=Y.+Liuauthor=A.+Zariehauthor=T.+Elyauthor=J.+M.+Kucharskiauthor=L.+V.+Kesslerauthor=T.+Wuauthor=K.+Yuauthor=Y.+Wangauthor=Y.+Yaoauthor=X.+Dengauthor=P.+P.+Zarrinkarauthor=D.+Brehmerauthor=D.+Dhanakauthor=M.+V.+Lorenziauthor=D.+Hu-Loweauthor=M.+P.+Patricelliauthor=P.+Renauthor=Y.+Liu&title=Targeting+KRAS+mutant+cancers+with+a+covalent+G12C-specific+inhibitor&doi=10.1016%2Fj.cell.2018.01.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting KRAS Mutant Cancers with a Covalent G12C-Specific Inhibitor</span></div><div class="casAuthors">Janes, Matthew R.; Zhang, Jingchuan; Li, Lian-Sheng; Hansen, Rasmus; Peters, Ulf; Guo, Xin; Chen, Yuching; Babbar, Anjali; Firdaus, Sarah J.; Darjania, Levan; Feng, Jun; Chen, Jeffrey H.; Li, Shuangwei; Li, Shisheng; Long, Yun O.; Thach, Carol; Liu, Yuan; Zarieh, Ata; Ely, Tess; Kucharski, Jeff M.; Kessler, Linda V.; Wu, Tao; Yu, Ke; Wang, Yi; Yao, Yvonne; Deng, Xiaohu; Zarrinkar, Patrick P.; Brehmer, Dirk; Dhanak, Dashyant; Lorenzi, Matthew V.; Hu-Lowe, Dana; Patricelli, Matthew P.; Ren, Pingda; Liu, Yi</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">172</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">578-589.e17</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">KRASG12C was recently identified to be potentially druggable by allele-specific covalent targeting of Cys 12 in vicinity to an inducible allosteric switch II pocket (S-IIP).  Success of this approach requires active cycling of KRASG12C between its active-GTP and inactive-GDP conformations as accessibility of the S-IIP is restricted only to the GDP-bound state.  This strategy proved feasible for inhibiting mutant KRAS in vitro; however, it is uncertain whether this approach would translate to in vivo.  Here, the authors describe structure-based design and identification of ARS-1620, a covalent compd. with high potency and selectivity for KRASG12C.  ARS-1620 achieves rapid and sustained in vivo target occupancy to induce tumor regression.  The authors use ARS-1620 to dissect oncogenic KRAS dependency and demonstrate that monolayer culture formats significantly underestimate KRAS dependency in vivo.  This study provides in vivo evidence that mutant KRAS can be selectively targeted and reveals ARS-1620 as representing a new generation of KRASG12C-specific inhibitors with promising therapeutic potential.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVmv4KAl4_dLVg90H21EOLACvtfcHk0lhDuQDYVN4xlw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXislyisb8%253D&md5=d0883bde981b51e41a794179ffae87be</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2018.01.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2018.01.006%26sid%3Dliteratum%253Aachs%26aulast%3DJanes%26aufirst%3DM.%2BR.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DL.%2BS.%26aulast%3DHansen%26aufirst%3DR.%26aulast%3DPeters%26aufirst%3DU.%26aulast%3DGuo%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DBabbar%26aufirst%3DA.%26aulast%3DFirdaus%26aufirst%3DS.%2BJ.%26aulast%3DDarjania%26aufirst%3DL.%26aulast%3DFeng%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DJ.%2BH.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DLong%26aufirst%3DY.%2BO.%26aulast%3DThach%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DZarieh%26aufirst%3DA.%26aulast%3DEly%26aufirst%3DT.%26aulast%3DKucharski%26aufirst%3DJ.%2BM.%26aulast%3DKessler%26aufirst%3DL.%2BV.%26aulast%3DWu%26aufirst%3DT.%26aulast%3DYu%26aufirst%3DK.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DYao%26aufirst%3DY.%26aulast%3DDeng%26aufirst%3DX.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26aulast%3DBrehmer%26aufirst%3DD.%26aulast%3DDhanak%26aufirst%3DD.%26aulast%3DLorenzi%26aufirst%3DM.%2BV.%26aulast%3DHu-Lowe%26aufirst%3DD.%26aulast%3DPatricelli%26aufirst%3DM.%2BP.%26aulast%3DRen%26aufirst%3DP.%26aulast%3DLiu%26aufirst%3DY.%26atitle%3DTargeting%2520KRAS%2520mutant%2520cancers%2520with%2520a%2520covalent%2520G12C-specific%2520inhibitor%26jtitle%3DCell%26date%3D2018%26volume%3D172%26spage%3D578%26epage%3D589.e17%26doi%3D10.1016%2Fj.cell.2018.01.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Canon, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rex, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saiki, A. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohr, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooke, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bagal, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaida, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holt, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knutson, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koppada, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lanman, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rapaport, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">San Miguel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortiz, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osgood, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, J.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCarter, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Volak, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Houk, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fakih, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Neil, B. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Falchook, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Desai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Govindan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouyang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henary, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arvedson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cee, V. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lipford, J. R.</span></span> <span> </span><span class="NLM_article-title">The clinical KRAS(G12C) inhibitor AMG 510 drives anti-tumour immunity</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>575</i></span>,  <span class="NLM_fpage">217</span>– <span class="NLM_lpage">223</span>, <span class="refDoi"> DOI: 10.1038/s41586-019-1694-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=10.1038%2Fs41586-019-1694-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=31666701" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitV2hsb7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=575&publication_year=2019&pages=217-223&author=J.+Canonauthor=K.+Rexauthor=A.+Y.+Saikiauthor=C.+Mohrauthor=K.+Cookeauthor=D.+Bagalauthor=K.+Gaidaauthor=T.+Holtauthor=C.+G.+Knutsonauthor=N.+Koppadaauthor=B.+A.+Lanmanauthor=J.+Wernerauthor=A.+S.+Rapaportauthor=T.+San+Miguelauthor=R.+Ortizauthor=T.+Osgoodauthor=J.-R.+Sunauthor=X.+Zhuauthor=J.+D.+McCarterauthor=L.+P.+Volakauthor=B.+E.+Houkauthor=M.+G.+Fakihauthor=B.+H.+O%E2%80%99Neilauthor=T.+J.+Priceauthor=G.+S.+Falchookauthor=J.+Desaiauthor=J.+Kuoauthor=R.+Govindanauthor=D.+S.+Hongauthor=W.+Ouyangauthor=H.+Henaryauthor=T.+Arvedsonauthor=V.+J.+Ceeauthor=J.+R.+Lipford&title=The+clinical+KRAS%28G12C%29+inhibitor+AMG+510+drives+anti-tumour+immunity&doi=10.1038%2Fs41586-019-1694-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">The clinical KRAS(G12C) inhibitor AMG 510 drives anti-tumour immunity</span></div><div class="casAuthors">Canon, Jude; Rex, Karen; Saiki, Anne Y.; Mohr, Christopher; Cooke, Keegan; Bagal, Dhanashri; Gaida, Kevin; Holt, Tyler; Knutson, Charles G.; Koppada, Neelima; Lanman, Brian A.; Werner, Jonathan; Rapaport, Aaron S.; San Miguel, Tisha; Ortiz, Roberto; Osgood, Tao; Sun, Ji-Rong; Zhu, Xiaochun; McCarter, John D.; Volak, Laurie P.; Houk, Brett E.; Fakih, Marwan G.; O'Neil, Bert H.; Price, Timothy J.; Falchook, Gerald S.; Desai, Jayesh; Kuo, James; Govindan, Ramaswamy; Hong, David S.; Ouyang, Wenjun; Henary, Haby; Arvedson, Tara; Cee, Victor J.; Lipford, J. Russell</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">575</span>
        (<span class="NLM_cas:issue">7781</span>),
    <span class="NLM_cas:pages">217-223</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">KRAS is the most frequently mutated oncogene in cancer and encodes a key signalling protein in tumors1,2.  The KRAS(G12C) mutant has a cysteine residue that has been exploited to design covalent inhibitors that have promising preclin. activity3-5.  Here we optimized a series of inhibitors, using novel binding interactions to markedly enhance their potency and selectivity.  Our efforts have led to the discovery of AMG 510, which is, to our knowledge, the first KRAS(G12C) inhibitor in clin. development.  In preclin. analyses, treatment with AMG 510 led to the regression of KRASG12C tumors and improved the anti-tumor efficacy of chemotherapy and targeted agents.  In immune-competent mice, treatment with AMG 510 resulted in a pro-inflammatory tumor microenvironment and produced durable cures alone as well as in combination with immune-checkpoint inhibitors.  Cured mice rejected the growth of isogenic KRASG12D tumors, which suggests adaptive immunity against shared antigens.  Furthermore, in clin. trials, AMG 510 demonstrated anti-tumor activity in the first dosing cohorts and represents a potentially transformative therapy for patients for whom effective treatments are lacking.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTHdBftQfnhbVg90H21EOLACvtfcHk0ljIcjhF6i2Gmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitV2hsb7F&md5=3819175980905976ba87ff545b9506e1</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fs41586-019-1694-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-019-1694-1%26sid%3Dliteratum%253Aachs%26aulast%3DCanon%26aufirst%3DJ.%26aulast%3DRex%26aufirst%3DK.%26aulast%3DSaiki%26aufirst%3DA.%2BY.%26aulast%3DMohr%26aufirst%3DC.%26aulast%3DCooke%26aufirst%3DK.%26aulast%3DBagal%26aufirst%3DD.%26aulast%3DGaida%26aufirst%3DK.%26aulast%3DHolt%26aufirst%3DT.%26aulast%3DKnutson%26aufirst%3DC.%2BG.%26aulast%3DKoppada%26aufirst%3DN.%26aulast%3DLanman%26aufirst%3DB.%2BA.%26aulast%3DWerner%26aufirst%3DJ.%26aulast%3DRapaport%26aufirst%3DA.%2BS.%26aulast%3DSan%2BMiguel%26aufirst%3DT.%26aulast%3DOrtiz%26aufirst%3DR.%26aulast%3DOsgood%26aufirst%3DT.%26aulast%3DSun%26aufirst%3DJ.-R.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DMcCarter%26aufirst%3DJ.%2BD.%26aulast%3DVolak%26aufirst%3DL.%2BP.%26aulast%3DHouk%26aufirst%3DB.%2BE.%26aulast%3DFakih%26aufirst%3DM.%2BG.%26aulast%3DO%25E2%2580%2599Neil%26aufirst%3DB.%2BH.%26aulast%3DPrice%26aufirst%3DT.%2BJ.%26aulast%3DFalchook%26aufirst%3DG.%2BS.%26aulast%3DDesai%26aufirst%3DJ.%26aulast%3DKuo%26aufirst%3DJ.%26aulast%3DGovindan%26aufirst%3DR.%26aulast%3DHong%26aufirst%3DD.%2BS.%26aulast%3DOuyang%26aufirst%3DW.%26aulast%3DHenary%26aufirst%3DH.%26aulast%3DArvedson%26aufirst%3DT.%26aulast%3DCee%26aufirst%3DV.%2BJ.%26aulast%3DLipford%26aufirst%3DJ.%2BR.%26atitle%3DThe%2520clinical%2520KRAS%2528G12C%2529%2520inhibitor%2520AMG%2520510%2520drives%2520anti-tumour%2520immunity%26jtitle%3DNature%26date%3D2019%26volume%3D575%26spage%3D217%26epage%3D223%26doi%3D10.1038%2Fs41586-019-1694-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lanman, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amegadzie, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashton, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Booker, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frohn, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodman, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kopecky, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Low, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minatti, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishimura, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pickrell, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reed, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siegmund, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamayo, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tegley, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walton, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wurz, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Achanta, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartberger, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Canon, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hollis, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCarter, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohr, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rex, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saiki, A. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">San Miguel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Volak, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whittington, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zech, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lipford, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cee, V. J.</span></span> <span> </span><span class="NLM_article-title">Discovery of a covalent inhibitor of KRAS<sup>G12C</sup>(AMG 510) for the treatment of solid tumors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">52</span>– <span class="NLM_lpage">65</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01180</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01180" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitlGitrrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=52-65&author=B.+A.+Lanmanauthor=J.+R.+Allenauthor=J.+G.+Allenauthor=A.+K.+Amegadzieauthor=K.+S.+Ashtonauthor=S.+K.+Bookerauthor=J.+J.+Chenauthor=N.+Chenauthor=M.+J.+Frohnauthor=G.+Goodmanauthor=D.+J.+Kopeckyauthor=L.+Liuauthor=P.+Lopezauthor=J.+D.+Lowauthor=V.+Maauthor=A.+E.+Minattiauthor=T.+T.+Nguyenauthor=N.+Nishimuraauthor=A.+J.+Pickrellauthor=A.+B.+Reedauthor=Y.+Shinauthor=A.+C.+Siegmundauthor=N.+A.+Tamayoauthor=C.+M.+Tegleyauthor=M.+C.+Waltonauthor=H.+L.+Wangauthor=R.+P.+Wurzauthor=M.+Xueauthor=K.+C.+Yangauthor=P.+Achantaauthor=M.+D.+Bartbergerauthor=J.+Canonauthor=L.+S.+Hollisauthor=J.+D.+McCarterauthor=C.+Mohrauthor=K.+Rexauthor=A.+Y.+Saikiauthor=T.+San+Miguelauthor=L.+P.+Volakauthor=K.+H.+Wangauthor=D.+A.+Whittingtonauthor=S.+G.+Zechauthor=J.+R.+Lipfordauthor=V.+J.+Cee&title=Discovery+of+a+covalent+inhibitor+of+KRASG12C%28AMG+510%29+for+the+treatment+of+solid+tumors&doi=10.1021%2Facs.jmedchem.9b01180"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Covalent Inhibitor of KRASG12C (AMG 510) for the Treatment of Solid Tumors</span></div><div class="casAuthors">Lanman, Brian A.; Allen, Jennifer R.; Allen, John G.; Amegadzie, Albert K.; Ashton, Kate S.; Booker, Shon K.; Chen, Jian Jeffrey; Chen, Ning; Frohn, Michael J.; Goodman, Guy; Kopecky, David J.; Liu, Longbin; Lopez, Patricia; Low, Jonathan D.; Ma, Vu; Minatti, Ana E.; Nguyen, Thomas T.; Nishimura, Nobuko; Pickrell, Alexander J.; Reed, Anthony B.; Shin, Youngsook; Siegmund, Aaron C.; Tamayo, Nuria A.; Tegley, Christopher M.; Walton, Mary C.; Wang, Hui-Ling; Wurz, Ryan P.; Xue, May; Yang, Kevin C.; Achanta, Pragathi; Bartberger, Michael D.; Canon, Jude; Hollis, L. Steven; McCarter, John D.; Mohr, Christopher; Rex, Karen; Saiki, Anne Y.; San Miguel, Tisha; Volak, Laurie P.; Wang, Kevin H.; Whittington, Douglas A.; Zech, Stephan G.; Lipford, J. Russell; Cee, Victor J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">52-65</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">KRASG12C has emerged as a promising target in the treatment of solid tumors.  Covalent inhibitors targeting the mutant cysteine-12 residue have been shown to disrupt signaling by this long-"undruggable" target; however clin. viable inhibitors have yet to be identified.  Here, we report efforts to exploit a cryptic pocket (H95/Y96/Q99) we identified in KRASG12C to identify inhibitors suitable for clin. development.  Structure-based design efforts leading to the identification of a novel quinazolinone scaffold are described, along with optimization efforts that overcame a configurational stability issue arising from restricted rotation about an axially chiral biaryl bond.  Biopharmaceutical optimization of the resulting leads culminated in the identification of AMG 510, a highly potent, selective, and well-tolerated KRASG12C inhibitor currently in phase I clin. trials (NCT03600883).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGeWqRRXX6lbVg90H21EOLACvtfcHk0ljIcjhF6i2Gmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitlGitrrK&md5=d973189be70bb913020edb4989f6ba7d</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01180&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01180%26sid%3Dliteratum%253Aachs%26aulast%3DLanman%26aufirst%3DB.%2BA.%26aulast%3DAllen%26aufirst%3DJ.%2BR.%26aulast%3DAllen%26aufirst%3DJ.%2BG.%26aulast%3DAmegadzie%26aufirst%3DA.%2BK.%26aulast%3DAshton%26aufirst%3DK.%2BS.%26aulast%3DBooker%26aufirst%3DS.%2BK.%26aulast%3DChen%26aufirst%3DJ.%2BJ.%26aulast%3DChen%26aufirst%3DN.%26aulast%3DFrohn%26aufirst%3DM.%2BJ.%26aulast%3DGoodman%26aufirst%3DG.%26aulast%3DKopecky%26aufirst%3DD.%2BJ.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DLopez%26aufirst%3DP.%26aulast%3DLow%26aufirst%3DJ.%2BD.%26aulast%3DMa%26aufirst%3DV.%26aulast%3DMinatti%26aufirst%3DA.%2BE.%26aulast%3DNguyen%26aufirst%3DT.%2BT.%26aulast%3DNishimura%26aufirst%3DN.%26aulast%3DPickrell%26aufirst%3DA.%2BJ.%26aulast%3DReed%26aufirst%3DA.%2BB.%26aulast%3DShin%26aufirst%3DY.%26aulast%3DSiegmund%26aufirst%3DA.%2BC.%26aulast%3DTamayo%26aufirst%3DN.%2BA.%26aulast%3DTegley%26aufirst%3DC.%2BM.%26aulast%3DWalton%26aufirst%3DM.%2BC.%26aulast%3DWang%26aufirst%3DH.%2BL.%26aulast%3DWurz%26aufirst%3DR.%2BP.%26aulast%3DXue%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DK.%2BC.%26aulast%3DAchanta%26aufirst%3DP.%26aulast%3DBartberger%26aufirst%3DM.%2BD.%26aulast%3DCanon%26aufirst%3DJ.%26aulast%3DHollis%26aufirst%3DL.%2BS.%26aulast%3DMcCarter%26aufirst%3DJ.%2BD.%26aulast%3DMohr%26aufirst%3DC.%26aulast%3DRex%26aufirst%3DK.%26aulast%3DSaiki%26aufirst%3DA.%2BY.%26aulast%3DSan%2BMiguel%26aufirst%3DT.%26aulast%3DVolak%26aufirst%3DL.%2BP.%26aulast%3DWang%26aufirst%3DK.%2BH.%26aulast%3DWhittington%26aufirst%3DD.%2BA.%26aulast%3DZech%26aufirst%3DS.%2BG.%26aulast%3DLipford%26aufirst%3DJ.%2BR.%26aulast%3DCee%26aufirst%3DV.%2BJ.%26atitle%3DDiscovery%2520of%2520a%2520covalent%2520inhibitor%2520of%2520KRASG12C%2528AMG%2520510%2529%2520for%2520the%2520treatment%2520of%2520solid%2520tumors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D52%26epage%3D65%26doi%3D10.1021%2Facs.jmedchem.9b01180" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wan, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of novel allosteric site and covalent inhibitors of FBPase with potent hypoglycemic effects</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>184</i></span>,  <span class="NLM_fpage">111749</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2019.111749</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=10.1016%2Fj.ejmech.2019.111749" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=31589992" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvFert7jK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=184&publication_year=2019&pages=111749&author=Y.+Huangauthor=L.+Weiauthor=X.+Hanauthor=H.+Chenauthor=Y.+Renauthor=Y.+Xuauthor=R.+Songauthor=L.+Raoauthor=C.+Suauthor=C.+Pengauthor=L.+Fengauthor=J.+Wan&title=Discovery+of+novel+allosteric+site+and+covalent+inhibitors+of+FBPase+with+potent+hypoglycemic+effects&doi=10.1016%2Fj.ejmech.2019.111749"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of novel allosteric site and covalent inhibitors of FBPase with potent hypoglycemic effects</span></div><div class="casAuthors">Huang, Yunyuan; Wei, Lin; Han, Xinya; Chen, Haifeng; Ren, Yanliang; Xu, Yanhong; Song, Rongrong; Rao, Li; Su, Chen; Peng, Chao; Feng, Lingling; Wan, Jian</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">184</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">111749</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Fructose-1,6-bisphosphatase (FBPase) is an essential enzyme of GNG pathway.  Significant advances demonstrate the FBPase plays a crit. role in treatment of diabetes.  Numerous FBPase inhibitors were developed by targeting AMP site, nevertheless, none of these inhibitors has exhibited suitable potency and druggability.  Herein, a new allosteric site (C128) on FBPase was discovered, and several nitrostyrene compds. exhibiting potent FBPase inhibitions were found covalently bind to C128 site on FBPase.  Mutagenesis suggest that C128 is the only cysteine that can influence FBPase inhibition, the N125-S124-S123 pathway was most likely involved in allosteric signaling transmission between C128 and active site.  However, these nitrostyrenes may bind with multiple cysteine besides C128 in FBPase.  To improve pocket selectivity, a series of novel compds. (14a-14n) were re-designed rationally by integrating fragment-based covalent virtual screening and machine-learning-based synthetic complexity evaluation.  As expected, the mass spectrometry validated that the proportion of title compds. binding to the C128 in FBPase was significantly higher than that of nitrostyrenes.  Notably, under physiol. and pathol. conditions, the treatment of compds. 14b(I), 14c(II), 14i(III) or 14n(IV) led to potent inhibition of glucose prodn., as well as decreased triglyceride and total cholesterol levels in mouse primary hepatocytes.  We highlight a novel paradigm that mol. targeting C128 site on FBPase can have potent hypoglycemic effect.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4rA2SCokJuLVg90H21EOLACvtfcHk0lhHOLsT_3QxvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvFert7jK&md5=a3c8f7ca2413e19243e1ad424676135a</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2019.111749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2019.111749%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DWei%26aufirst%3DL.%26aulast%3DHan%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DRen%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DSong%26aufirst%3DR.%26aulast%3DRao%26aufirst%3DL.%26aulast%3DSu%26aufirst%3DC.%26aulast%3DPeng%26aufirst%3DC.%26aulast%3DFeng%26aufirst%3DL.%26aulast%3DWan%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520novel%2520allosteric%2520site%2520and%2520covalent%2520inhibitors%2520of%2520FBPase%2520with%2520potent%2520hypoglycemic%2520effects%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2019%26volume%3D184%26spage%3D111749%26doi%3D10.1016%2Fj.ejmech.2019.111749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Veverka, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mays, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lipsky, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naylor, S.</span></span> <span> </span><span class="NLM_article-title">Inhibition of aldehyde dehydrogenase by disulfiram and its metabolite methyl diethylthiocarbamoyl-sulfoxide</span>. <i>Biochem. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">511</span>– <span class="NLM_lpage">518</span>, <span class="refDoi"> DOI: 10.1016/S0006-2952(96)00767-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=10.1016%2FS0006-2952%2896%2900767-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=9105402" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADyaK2sXitVajsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=511-518&author=K.+A.+Veverkaauthor=K.+L.+Johnsonauthor=D.+C.+Maysauthor=J.+J.+Lipskyauthor=S.+Naylor&title=Inhibition+of+aldehyde+dehydrogenase+by+disulfiram+and+its+metabolite+methyl+diethylthiocarbamoyl-sulfoxide&doi=10.1016%2FS0006-2952%2896%2900767-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of aldehyde dehydrogenase by disulfiram and its metabolite methyl diethylthiocarbamoyl-sulfoxide</span></div><div class="casAuthors">Veverka, Karen A.; Johnson, Kenneth L.; Mays, Dennis C.; Lipsky, James J.; Naylor, Stephen</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Pharmacology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">511-518</span>CODEN:
                <span class="NLM_cas:coden">BCPCA6</span>;
        ISSN:<span class="NLM_cas:issn">0006-2952</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Disulfiram (DSF) is presently the only available drug used in the aversion therapy of recovering alcoholics.  It acts by inhibiting aldehyde dehydrogenase (ALDH), leading to high blood levels of acetaldehyde.  The in vitro inhibition of ALDH by DSF and its metabolites was systematically studied by combined enzyme inhibition assay with direct mol. wt. detn. of the same sample using electrospray ionization-mass spectrometry (ESI-MS).  Enzyme activity was measured after incubating yeast ALDH (yALDH) with excess concns. of DSF, Me diethyldithiocarbamate (MeDDC) and Me diethylthiocarbamoyl-sulfoxide (MeDTC-SO) and then subjected to anal. by ESI-MS.  Addn. of DSF resulted in complete enzyme inhibition; however, ESI-MS anal. demonstrated no discernible shift in mol. wt., indicating that no intermol. adduct was formed with the protein.  Treatment of yALDH with MeDTC-SO also completely abolished yALDH activity with a concomitant increase of +≈100 Da in the mol. mass of the enzyme.  This indicated formation of a covalent carbamoyl protein adduct.  Furthermore, the effects of dithiothreitol (DTT) were examd. on samples of inhibited protein in vitro.  At pH 7.5, DDT completely reversed inhibition after DSF treatment.  YALDH inhibited by MeDTC-SO could not be recovered by DTT at pH 7.5, but at pH 9 the enzymic activity was fully restored and a mass loss of ∼100 Da was noted.  This observations are consistent with mechanisms where inhibition of yALDH by DSF in vitro involves oxidn. of the active site, whereas MeDTC-SO forms a covalent adduct with the protein in vitro resulting in cessation of enzyme activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWw03Q2vgZmbVg90H21EOLACvtfcHk0lhHOLsT_3QxvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXitVajsbY%253D&md5=7893c1c4a675f7d94fea0b143434e4c3</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2FS0006-2952%2896%2900767-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0006-2952%252896%252900767-8%26sid%3Dliteratum%253Aachs%26aulast%3DVeverka%26aufirst%3DK.%2BA.%26aulast%3DJohnson%26aufirst%3DK.%2BL.%26aulast%3DMays%26aufirst%3DD.%2BC.%26aulast%3DLipsky%26aufirst%3DJ.%2BJ.%26aulast%3DNaylor%26aufirst%3DS.%26atitle%3DInhibition%2520of%2520aldehyde%2520dehydrogenase%2520by%2520disulfiram%2520and%2520its%2520metabolite%2520methyl%2520diethylthiocarbamoyl-sulfoxide%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D1997%26volume%3D53%26spage%3D511%26epage%3D518%26doi%3D10.1016%2FS0006-2952%2896%2900767-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Skrott, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mistrik, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andersen, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friis, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majera, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gursky, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozdian, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartkova, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turi, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moudry, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraus, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michalova, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaclavkova, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dzubak, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vrobel, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pouckova, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sedlacek, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miklovicova, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kutt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mattova, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Driessen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dou, Q. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olsen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hajduch, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cvek, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshaies, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartek, J.</span></span> <span> </span><span class="NLM_article-title">Alcohol-abuse drug disulfiram targets cancer via p97 segregase adaptor NPL4</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>552</i></span>,  <span class="NLM_fpage">194</span>– <span class="NLM_lpage">199</span>, <span class="refDoi"> DOI: 10.1038/nature25016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=10.1038%2Fnature25016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=29211715" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFGlsbrN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=552&publication_year=2017&pages=194-199&author=Z.+Skrottauthor=M.+Mistrikauthor=K.+K.+Andersenauthor=S.+Friisauthor=D.+Majeraauthor=J.+Gurskyauthor=T.+Ozdianauthor=J.+Bartkovaauthor=Z.+Turiauthor=P.+Moudryauthor=M.+Krausauthor=M.+Michalovaauthor=J.+Vaclavkovaauthor=P.+Dzubakauthor=I.+Vrobelauthor=P.+Pouckovaauthor=J.+Sedlacekauthor=A.+Miklovicovaauthor=A.+Kuttauthor=J.+Liauthor=J.+Mattovaauthor=C.+Driessenauthor=Q.+P.+Douauthor=J.+Olsenauthor=M.+Hajduchauthor=B.+Cvekauthor=R.+J.+Deshaiesauthor=J.+Bartek&title=Alcohol-abuse+drug+disulfiram+targets+cancer+via+p97+segregase+adaptor+NPL4&doi=10.1038%2Fnature25016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Alcohol-abuse drug disulfiram targets cancer via p97 segregase adaptor NPL4</span></div><div class="casAuthors">Skrott, Zdenek; Mistrik, Martin; Andersen, Klaus Kaae; Friis, Soeren; Majera, Dusana; Gursky, Jan; Ozdian, Tomas; Bartkova, Jirina; Turi, Zsofia; Moudry, Pavel; Kraus, Marianne; Michalova, Martina; Vaclavkova, Jana; Dzubak, Petr; Vrobel, Ivo; Pouckova, Pavla; Sedlacek, Jindrich; Miklovicova, Andrea; Kutt, Anne; Li, Jing; Mattova, Jana; Driessen, Christoph; Dou, Q. Ping; Olsen, Joergen; Hajduch, Marian; Cvek, Boris; Deshaies, Raymond J.; Bartek, Jiri</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">552</span>
        (<span class="NLM_cas:issue">7684</span>),
    <span class="NLM_cas:pages">194-199</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Cancer incidence is rising and this global challenge is further exacerbated by tumor resistance to available medicines.  A promising approach to meet the need for improved cancer treatment is drug repurposing.  Here we highlight the potential for repurposing disulfiram (also known by the trade name Antabuse), an old alc.-aversion drug that has been shown to be effective against diverse cancer types in preclin. studies.  Our nationwide epidemiol. study reveals that patients who continuously used disulfiram have a lower risk of death from cancer compared to those who stopped using the drug at their diagnosis.  Moreover, we identify the ditiocarb-copper complex as the metabolite of disulfiram that is responsible for its anti-cancer effects, and provide methods to detect preferential accumulation of the complex in tumors and candidate biomarkers to analyze its effect on cells and tissues.  Finally, our functional and biophys. analyses reveal the mol. target of disulfiram's tumor-suppressing effects as NPL4, an adaptor of p97 (also known as VCP) segregase, which is essential for the turnover of proteins involved in multiple regulatory and stress-response pathways in cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqp_sAn4LKB57Vg90H21EOLACvtfcHk0lig8ScyWXxbkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFGlsbrN&md5=b53235723e8478ab49b7d569d0dd35ac</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1038%2Fnature25016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature25016%26sid%3Dliteratum%253Aachs%26aulast%3DSkrott%26aufirst%3DZ.%26aulast%3DMistrik%26aufirst%3DM.%26aulast%3DAndersen%26aufirst%3DK.%2BK.%26aulast%3DFriis%26aufirst%3DS.%26aulast%3DMajera%26aufirst%3DD.%26aulast%3DGursky%26aufirst%3DJ.%26aulast%3DOzdian%26aufirst%3DT.%26aulast%3DBartkova%26aufirst%3DJ.%26aulast%3DTuri%26aufirst%3DZ.%26aulast%3DMoudry%26aufirst%3DP.%26aulast%3DKraus%26aufirst%3DM.%26aulast%3DMichalova%26aufirst%3DM.%26aulast%3DVaclavkova%26aufirst%3DJ.%26aulast%3DDzubak%26aufirst%3DP.%26aulast%3DVrobel%26aufirst%3DI.%26aulast%3DPouckova%26aufirst%3DP.%26aulast%3DSedlacek%26aufirst%3DJ.%26aulast%3DMiklovicova%26aufirst%3DA.%26aulast%3DKutt%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DMattova%26aufirst%3DJ.%26aulast%3DDriessen%26aufirst%3DC.%26aulast%3DDou%26aufirst%3DQ.%2BP.%26aulast%3DOlsen%26aufirst%3DJ.%26aulast%3DHajduch%26aufirst%3DM.%26aulast%3DCvek%26aufirst%3DB.%26aulast%3DDeshaies%26aufirst%3DR.%2BJ.%26aulast%3DBartek%26aufirst%3DJ.%26atitle%3DAlcohol-abuse%2520drug%2520disulfiram%2520targets%2520cancer%2520via%2520p97%2520segregase%2520adaptor%2520NPL4%26jtitle%3DNature%26date%3D2017%26volume%3D552%26spage%3D194%26epage%3D199%26doi%3D10.1038%2Fnature25016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bao, B. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, G. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, F. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J. G.</span></span> <span> </span><span class="NLM_article-title">Discovery of unsymmetrical aromatic disulfides as novel inhibitors of SARS-CoV main protease: Chemical synthesis, biological evaluation, molecular docking and 3D-QSAR study</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>137</i></span>,  <span class="NLM_fpage">450</span>– <span class="NLM_lpage">461</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2017.05.045</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=10.1016%2Fj.ejmech.2017.05.045" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=28624700" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVertbrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=137&publication_year=2017&pages=450-461&author=L.+Wangauthor=B.+B.+Baoauthor=G.+Q.+Songauthor=C.+Chenauthor=X.+M.+Zhangauthor=W.+Luauthor=Z.+Wangauthor=Y.+Caiauthor=S.+Liauthor=S.+Fuauthor=F.+H.+Songauthor=H.+Yangauthor=J.+G.+Wang&title=Discovery+of+unsymmetrical+aromatic+disulfides+as+novel+inhibitors+of+SARS-CoV+main+protease%3A+Chemical+synthesis%2C+biological+evaluation%2C+molecular+docking+and+3D-QSAR+study&doi=10.1016%2Fj.ejmech.2017.05.045"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of unsymmetrical aromatic disulfides as novel inhibitors of SARS-CoV main protease: Chemical synthesis, biological evaluation, molecular docking and 3D-QSAR study</span></div><div class="casAuthors">Wang, Li; Bao, Bo-Bo; Song, Guo-Qing; Chen, Cheng; Zhang, Xu-Meng; Lu, Wei; Wang, Zefang; Cai, Yan; Li, Shuang; Fu, Sheng; Song, Fu-Hang; Yang, Haitao; Wang, Jian-Guo</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">137</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">450-461</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">The worldwide outbreak of severe acute respiratory syndrome (SARS) in 2003 had caused a high rate of mortality.  Main protease (Mpro) of SARS-assocd. coronavirus (SARS-CoV) is an important target to discover pharmaceutical compds. for the therapy of this life-threatening disease.  During the course of screening new anti-SARS agents, a series of unsym. arom. disulfides inhibited SARS-CoV Mpro significantly for the first time have been identified.  Novel unsym. arom. disulfides R1SSR2 [R1 = 1,3-thiazol-2-ylsulfanyl, 1,3,4-oxadiazol-2-ylsulfanyl, pyrimidin-2-ylsulfanyl, etc.; R2 = phenylsulfanyl, 4-bromophenylsulfanyl, [2-(methoxycarbonyl)phenyl]sulfanyl, etc.] were synthesized chem. and their biol. activities were evaluated in vitro against SARS-CoV Mpro.  These novel compds. displayed excellent IC50 data in the range of 0.516-5.954 μM.  Preliminary studies indicated that these disulfides are reversible and competitive inhibitors.  A possible binding mode was generated via mol. docking simulation and a comparative field anal. (CoMFA) model was constructed to understand the structure-activity relationships.  The present research therefore has provided some meaningful guidance to design and identify anti-SARS drugs with totally new chem. structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqd14DP7qY8zrVg90H21EOLACvtfcHk0lig8ScyWXxbkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVertbrO&md5=07198a9f4ec7718a9f2299a3184dc8c4</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2017.05.045&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2017.05.045%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.%26aulast%3DBao%26aufirst%3DB.%2BB.%26aulast%3DSong%26aufirst%3DG.%2BQ.%26aulast%3DChen%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DX.%2BM.%26aulast%3DLu%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DCai%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DFu%26aufirst%3DS.%26aulast%3DSong%26aufirst%3DF.%2BH.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DJ.%2BG.%26atitle%3DDiscovery%2520of%2520unsymmetrical%2520aromatic%2520disulfides%2520as%2520novel%2520inhibitors%2520of%2520SARS-CoV%2520main%2520protease%253A%2520Chemical%2520synthesis%252C%2520biological%2520evaluation%252C%2520molecular%2520docking%2520and%25203D-QSAR%2520study%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2017%26volume%3D137%26spage%3D450%26epage%3D461%26doi%3D10.1016%2Fj.ejmech.2017.05.045" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hallenbeck, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turner, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renslo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arkin, M.</span></span> <span> </span><span class="NLM_article-title">Targeting non-catalytic cysteine residues through structure-guided drug discovery</span>. <i>Curr. Top. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">4</span>– <span class="NLM_lpage">15</span>, <span class="refDoi"> DOI: 10.2174/1568026616666160719163839</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=10.2174%2F1568026616666160719163839" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=27449257" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFCmurzK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2017&pages=4-15&author=K.+Hallenbeckauthor=D.+Turnerauthor=A.+Rensloauthor=M.+Arkin&title=Targeting+non-catalytic+cysteine+residues+through+structure-guided+drug+discovery&doi=10.2174%2F1568026616666160719163839"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Non-Catalytic Cysteine Residues Through Structure-Guided Drug Discovery</span></div><div class="casAuthors">Hallenbeck, Kenneth K.; Turner, David M.; Renslo, Adam R.; Arkin, Michelle R.</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">4-15</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  The targeting of non-catalytic cysteine residues with small mols. is drawing increased attention from drug discovery scientists and chem. biologists.  From a biol. perspective, genomic and proteomic studies have revealed the presence of cysteine mutations in several oncogenic proteins, suggesting both a functional role for these residues and also a strategy for targeting them in an 'allele specific' manner.  For the medicinal chemist, the structure-guided design of cysteine- reactive mols. is an appealing strategy to realize improved selectivity and pharmacodynamic properties in drug leads.  Finally, for chem. biologists, the modification of cysteine residues provides a unique means to probe protein structure and allosteric regulation.  Here, we review three applications of cysteinemodifying small mols.: 1. the optimization of existing drug leads, 2. the discovery of new lead compds., and 3. the use of cysteine-reactive mols. as probes of protein dynamics.  In each case, structure-guided design plays a key role in detg. which cysteine residue(s) to target and in designing compds. with the proper geometry to enable both covalent interaction with the targeted cysteine and productive non-covalent interactions with nearby protein residues.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdyF3jwe8kzLVg90H21EOLACvtfcHk0lgVlCVndsTk0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFCmurzK&md5=adbdf5b5afd5fc53ba0f360e695b6fa5</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.2174%2F1568026616666160719163839&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1568026616666160719163839%26sid%3Dliteratum%253Aachs%26aulast%3DHallenbeck%26aufirst%3DK.%26aulast%3DTurner%26aufirst%3DD.%26aulast%3DRenslo%26aufirst%3DA.%26aulast%3DArkin%26aufirst%3DM.%26atitle%3DTargeting%2520non-catalytic%2520cysteine%2520residues%2520through%2520structure-guided%2520drug%2520discovery%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2017%26volume%3D17%26spage%3D4%26epage%3D15%26doi%3D10.2174%2F1568026616666160719163839" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacCoss, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawson, A. D. G.</span></span> <span> </span><span class="NLM_article-title">Rings in drugs</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">5845</span>– <span class="NLM_lpage">5859</span>, <span class="refDoi"> DOI: 10.1021/jm4017625</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm4017625" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlCntbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=5845-5859&author=R.+D.+Taylorauthor=M.+MacCossauthor=A.+D.+G.+Lawson&title=Rings+in+drugs&doi=10.1021%2Fjm4017625"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Rings in Drugs</span></div><div class="casAuthors">Taylor, Richard D.; MacCoss, Malcolm; Lawson, Alastair D. G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">5845-5859</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  We have analyzed the rings, ring systems, and frameworks in drugs listed in the FDA Orange Book to understand the frequency, time-lines, mol. property space, and the application of these rings in different therapeutic areas and target classes.  This anal. shows that there are only 351 ring systems and 1197 frameworks in drugs that came onto the market before 2013.  Furthermore, on av. six new ring systems enter drug space each year and approx. 28% of new drugs contain a new ring system.  Moreover, it is very unusual for a drug to contain more than one new ring system and the majority of the most frequently used ring systems (83%) were first used in drugs developed prior to 1983.  These observations give insight into the chem. novelty of drugs and potentially efficient ways to assess compd. libraries and develop compds. from hit identification to lead optimization and beyond.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoC_e_L4z4XCrVg90H21EOLACvtfcHk0lgVlCVndsTk0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlCntbc%253D&md5=c84ce84b8698eb6fad38fa82b0ab4912</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fjm4017625&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm4017625%26sid%3Dliteratum%253Aachs%26aulast%3DTaylor%26aufirst%3DR.%2BD.%26aulast%3DMacCoss%26aufirst%3DM.%26aulast%3DLawson%26aufirst%3DA.%2BD.%2BG.%26atitle%3DRings%2520in%2520drugs%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D5845%26epage%3D5859%26doi%3D10.1021%2Fjm4017625" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span> <span> </span><span class="NLM_article-title">Development of the first generation of disulfide-based subtype-selective and potent covalent pyruvate dehydrogenase kinase 1 (PDK1) inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2227</span>– <span class="NLM_lpage">2244</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01245</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01245" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjtlyjtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=2227-2244&author=Y.+Liuauthor=Z.+Xieauthor=D.+Zhaoauthor=J.+Zhuauthor=F.+Maoauthor=S.+Tangauthor=H.+Xuauthor=C.+Luoauthor=M.+Gengauthor=M.+Huangauthor=J.+Li&title=Development+of+the+first+generation+of+disulfide-based+subtype-selective+and+potent+covalent+pyruvate+dehydrogenase+kinase+1+%28PDK1%29+inhibitors&doi=10.1021%2Facs.jmedchem.6b01245"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Development of the First Generation of Disulfide-Based Subtype-Selective and Potent Covalent Pyruvate Dehydrogenase Kinase 1 (PDK1) Inhibitors</span></div><div class="casAuthors">Liu, Yifu; Xie, Zuoquan; Zhao, Dan; Zhu, Jin; Mao, Fei; Tang, Shuai; Xu, Hui; Luo, Cheng; Geng, Meiyu; Huang, Min; Li, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2227-2244</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Pyruvate dehydrogenase kinases (PDKs) are overexpressed in most cancer cells and are responsible for aberrant glucose metab.  The authors previously described Bis(4-morpholinyl thiocarbonyl)-disulfide (JX06, 16) as the first covalent inhibitor of PDK1.  Here, based on the scaffold of JX06, the authors identify two novel types of disulfide-based PDK1 inhibitors.  The most potent analog, bis(1,4-oxazepan-4-yl-thiocarbonyl)disulfide (3a), effectively inhibits PDK1 both at the mol. (kinact/Ki = 4.17 × 103 M-1s-1) and the cellular level (down to 0.1 μM).  In contrast to JX06, 3a is a potent and subtype-selective inhibitor of PDK1 with >40-fold selectivity for PDK2-4.  Compd. 3a also significantly alters glucose metabolic pathways in A549 cells by decreasing ECAR and increasing ROS.  Moreover, in the xenograft models, 3a shows significant antitumor activity with no neg. effect to the mice wt.  Collectively, these data demonstrate that 3a may be an excellent lead compd. for the treatment of cancer as a first-generation subtype-selective and covalent PDK1 inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEpZNCUKK6MrVg90H21EOLACvtfcHk0lgVlCVndsTk0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjtlyjtr0%253D&md5=7529ce2fc9d1a138a8d1405a23f7e4e8</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01245&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01245%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DXie%26aufirst%3DZ.%26aulast%3DZhao%26aufirst%3DD.%26aulast%3DZhu%26aufirst%3DJ.%26aulast%3DMao%26aufirst%3DF.%26aulast%3DTang%26aufirst%3DS.%26aulast%3DXu%26aufirst%3DH.%26aulast%3DLuo%26aufirst%3DC.%26aulast%3DGeng%26aufirst%3DM.%26aulast%3DHuang%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DJ.%26atitle%3DDevelopment%2520of%2520the%2520first%2520generation%2520of%2520disulfide-based%2520subtype-selective%2520and%2520potent%2520covalent%2520pyruvate%2520dehydrogenase%2520kinase%25201%2520%2528PDK1%2529%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D2227%26epage%3D2244%26doi%3D10.1021%2Facs.jmedchem.6b01245" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pietrucci, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laio, A.</span></span> <span> </span><span class="NLM_article-title">A collective variable for the efficient exploration of protein beta-sheet structures: application to SH3 and GB1</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">2197</span>– <span class="NLM_lpage">2201</span>, <span class="refDoi"> DOI: 10.1021/ct900202f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct900202f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BD1MXptleltbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=2197-2201&author=F.+Pietrucciauthor=A.+Laio&title=A+collective+variable+for+the+efficient+exploration+of+protein+beta-sheet+structures%3A+application+to+SH3+and+GB1&doi=10.1021%2Fct900202f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">A Collective Variable for the Efficient Exploration of Protein Beta-Sheet Structures: Application to SH3 and GB1</span></div><div class="casAuthors">Pietrucci, Fabio; Laio, Alessandro</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2197-2201</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The authors introduce a new class of collective variables which allow forming efficiently beta-sheet structures in all-atom explicit-solvent simulations of proteins.  By this approach the authors are able to systematically fold a 16-residue beta hairpin using metadynamics on a single replica.  Application to the 56-residue SH3 and GB1 proteins show that, starting from extended states, in ∼100 ns tens of structures contg. more than 30% beta-sheet are obtained, including parts of the native fold.  Using these variables may allow folding moderate size proteins with an accurate explicit solvent description.  Moreover, it may allow investigating the presence of misfolded states that are relevant for diseases (e.g., prion and Alzheimer) and studying beta-aggregation (amyloid diseases).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUydogD9f81LVg90H21EOLACvtfcHk0liA4JeW-Pb2iQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXptleltbo%253D&md5=3d937a81ee3db1084794d47b96b41e35</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fct900202f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct900202f%26sid%3Dliteratum%253Aachs%26aulast%3DPietrucci%26aufirst%3DF.%26aulast%3DLaio%26aufirst%3DA.%26atitle%3DA%2520collective%2520variable%2520for%2520the%2520efficient%2520exploration%2520of%2520protein%2520beta-sheet%2520structures%253A%2520application%2520to%2520SH3%2520and%2520GB1%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2009%26volume%3D5%26spage%3D2197%26epage%3D2201%26doi%3D10.1021%2Fct900202f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lalau, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Race, J. M.</span></span> <span> </span><span class="NLM_article-title">Lactic acidosis in metformin-treated patients. Prognostic value of arterial lactate levels and plasma metformin concentrations</span>. <i>Drug Saf.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">377</span>– <span class="NLM_lpage">384</span>, <span class="refDoi"> DOI: 10.2165/00002018-199920040-00006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=10.2165%2F00002018-199920040-00006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=10230584" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADyaK1MXjtVGhs74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=1999&pages=377-384&author=J.+D.+Lalauauthor=J.+M.+Race&title=Lactic+acidosis+in+metformin-treated+patients.+Prognostic+value+of+arterial+lactate+levels+and+plasma+metformin+concentrations&doi=10.2165%2F00002018-199920040-00006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Lactic acidosis in metformin-treated patients: prognostic value of arterial lactate levels and plasma metformin concentrations</span></div><div class="casAuthors">Lalau, Jean-Daniel; Race, Jean-M.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Safety</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">377-384</span>CODEN:
                <span class="NLM_cas:coden">DRSAEA</span>;
        ISSN:<span class="NLM_cas:issn">0114-5916</span>.
    
            (<span class="NLM_cas:orgname">Adis International Ltd.</span>)
        </div><div class="casAbstract">The antidiabetic drug metformin has been assocd. in a small no. of patients with lactic acidosis, a serious condition with a poor prognosis.  However, because of lack of data, the prognostic significance of hyperlactatemia in metformin-treated patients is not known.  Data were collected from 49 metformin-treated patients with lactic acidosis (arterial lactate level ≥5 mmol/L and blood pH ≤7.35) and available plasma metformin concn. data to investigate the assocn. of arterial lactate levels and plasma metformin concns. with mortality.  The overall mortality rate in this patient sample was 45% and the median arterial lactate level was 13.1 mmol/L.  Median lactate levels were similar in patients who survived (13 mmol/L) and those who died (14.3 mmol/L), whereas the median plasma metformin concn. was 3 times higher in patients who survived (20.6 mg/L vs. 6.3 mg/L).  In this, the largest series of metformin-treated patients with lactic acidosis yet reported, 55% of patients survived and these patients had a median arterial lactate level of 13.1 mmol/L.  Neither arterial lactate levels nor plasma metformin concns. were of prognostic significance in relation to mortality in this sample of metformin-treated patients with lactic acidosis.  Death in these patients appeared instead to be assocd. with other hypoxic disease or underlying ill health.  These observations suggest that accumulation of metformin may not be as significant with respect to high arterial levels of lactate and their effects as has been traditionally thought.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAFd7jfy_hDLVg90H21EOLACvtfcHk0liA4JeW-Pb2iQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXjtVGhs74%253D&md5=5f2fb705fa49e4c2f3bf58a993c5e814</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.2165%2F00002018-199920040-00006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2165%252F00002018-199920040-00006%26sid%3Dliteratum%253Aachs%26aulast%3DLalau%26aufirst%3DJ.%2BD.%26aulast%3DRace%26aufirst%3DJ.%2BM.%26atitle%3DLactic%2520acidosis%2520in%2520metformin-treated%2520patients.%2520Prognostic%2520value%2520of%2520arterial%2520lactate%2520levels%2520and%2520plasma%2520metformin%2520concentrations%26jtitle%3DDrug%2520Saf.%26date%3D1999%26volume%3D20%26spage%3D377%26epage%3D384%26doi%3D10.2165%2F00002018-199920040-00006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Han, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wan, J.</span></span> <span> </span><span class="NLM_article-title">New insight into the binding modes of TNP-AMP to human liver fructose-1,6-bisphosphatase</span>. <i>Spectrochim. Acta. A Mol. Biomol. Spectrosc.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>165</i></span>,  <span class="NLM_fpage">155</span>– <span class="NLM_lpage">160</span>, <span class="refDoi"> DOI: 10.1016/j.saa.2016.04.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=10.1016%2Fj.saa.2016.04.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=27137358" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BC28XntFyjsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=165&publication_year=2016&pages=155-160&author=X.+Hanauthor=Y.+Huangauthor=R.+Zhangauthor=S.+Xiaoauthor=S.+Zhuauthor=N.+Qinauthor=Z.+Hongauthor=L.+Weiauthor=J.+Fengauthor=Y.+Renauthor=L.+Fengauthor=J.+Wan&title=New+insight+into+the+binding+modes+of+TNP-AMP+to+human+liver+fructose-1%2C6-bisphosphatase&doi=10.1016%2Fj.saa.2016.04.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">New insight into the binding modes of TNP-AMP to human liver fructose-1,6-bisphosphatase</span></div><div class="casAuthors">Han, Xinya; Huang, Yunyuan; Zhang, Rui; Xiao, San; Zhu, Shuaihuan; Qin, Nian; Hong, Zongqin; Wei, Lin; Feng, Jiangtao; Ren, Yanliang; Feng, Lingling; Wan, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Spectrochimica Acta, Part A: Molecular and Biomolecular Spectroscopy</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">165</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">155-160</span>CODEN:
                <span class="NLM_cas:coden">SAMCAS</span>;
        ISSN:<span class="NLM_cas:issn">1386-1425</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Human liver fructose 1,6-bisphosphatase (FBPase) contains 2 binding sites, a fructose 1,6-bisphosphate (FBP) active site and an AMP allosteric site.  The FBP active site works by stabilizing the FBPase, and the allosteric site impairs the activity of FBPase through its binding of a non-substrate mol.  The fluorescent AMP analog, 2',3'-O-(2,4,6-trinitrophenyl)AMP (TNP-AMP) is used as a fluorescent probe as it is able to competitively inhibit AMP binding to the AMP allosteric site and, therefore, could be used for exploring the binding modes of inhibitors targeted on the allosteric site.  Here, the authors re-examd. the binding modes of TNP-AMP to FBPase.  However, the present enzyme kinetic assays showed that AMP and FBP both could reduce the fluorescence from the bound TNP-AMP through competition for FBPase, suggesting that TNP-AMP binds not only to the AMP allosteric site but also to the FBP active site.  Mutagenesis assays of K274L (located in the FBP active site) showed that the Lys-274 residue is very important for TNP-AMP to bind to the active site of FBPase.  The results further proved that TNP-AMP was able to bind individually to the both sites.  Thus, the present study provides a new insight into the binding mechanism of TNP-AMP to FBPase.  The TNP-AMP fluorescent probe could be used to examine the binding site of an inhibitor (the active site or the allosteric site) using FBPase satd. by AMP and FBP, resp., or the K247L mutant FBPase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJ6O9415rhJLVg90H21EOLACvtfcHk0liA4JeW-Pb2iQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XntFyjsLc%253D&md5=dbe1c1be3db825878066a33bde03e901</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.saa.2016.04.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.saa.2016.04.002%26sid%3Dliteratum%253Aachs%26aulast%3DHan%26aufirst%3DX.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DR.%26aulast%3DXiao%26aufirst%3DS.%26aulast%3DZhu%26aufirst%3DS.%26aulast%3DQin%26aufirst%3DN.%26aulast%3DHong%26aufirst%3DZ.%26aulast%3DWei%26aufirst%3DL.%26aulast%3DFeng%26aufirst%3DJ.%26aulast%3DRen%26aufirst%3DY.%26aulast%3DFeng%26aufirst%3DL.%26aulast%3DWan%26aufirst%3DJ.%26atitle%3DNew%2520insight%2520into%2520the%2520binding%2520modes%2520of%2520TNP-AMP%2520to%2520human%2520liver%2520fructose-1%252C6-bisphosphatase%26jtitle%3DSpectrochim.%2520Acta.%2520A%2520Mol.%2520Biomol.%2520Spectrosc.%26date%3D2016%26volume%3D165%26spage%3D155%26epage%3D160%26doi%3D10.1016%2Fj.saa.2016.04.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">Xds</span>. <i>Acta Crystallogr. D</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1107/S0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=Xds&doi=10.1107%2FS0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0ljANkqcYa0fWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXds%26jtitle%3DActa%2520Crystallogr.%2520D%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2FS0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballard, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keegan, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krissinel, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leslie, A. G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNicholas, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannu, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potterton, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, K. S.</span></span> <span> </span><span class="NLM_article-title">Overview of theCCP4 suite and current developments</span>. <i>Acta Crystallogr. D</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">242</span>, <span class="refDoi"> DOI: 10.1107/S0907444910045749</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=10.1107%2FS0907444910045749" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=21460441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=235-242&author=M.+D.+Winnauthor=C.+C.+Ballardauthor=K.+D.+Cowtanauthor=E.+J.+Dodsonauthor=P.+Emsleyauthor=P.+R.+Evansauthor=R.+M.+Keeganauthor=E.+B.+Krissinelauthor=A.+G.+W.+Leslieauthor=A.+McCoyauthor=S.+J.+McNicholasauthor=G.+N.+Murshudovauthor=N.+S.+Pannuauthor=E.+A.+Pottertonauthor=H.+R.+Powellauthor=R.+J.+Readauthor=A.+Vaginauthor=K.+S.+Wilson&title=Overview+of+theCCP4+suite+and+current+developments&doi=10.1107%2FS0907444910045749"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Overview of the CCP4 suite and current developments</span></div><div class="casAuthors">Winn, Martyn D.; Ballard, Charles C.; Cowtan, Kevin D.; Dodson, Eleanor J.; Emsley, Paul; Evans, Phil R.; Keegan, Ronan M.; Krissinel, Eugene B.; Leslie, Andrew G. W.; McCoy, Airlie; McNicholas, Stuart J.; Murshudov, Garib N.; Pannu, Navraj S.; Potterton, Elizabeth A.; Powell, Harold R.; Read, Randy J.; Vagin, Alexei; Wilson, Keith S.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  The CCP4 (Collaborative Computational Project, No. 4) software suite is a collection of programs and assocd. data and software libraries which can be used for macromol. structure detn. by X-ray crystallog.  The suite is designed to be flexible, allowing users a no. of methods of achieving their aims.  The programs are from a wide variety of sources but are connected by a common infrastructure provided by std. file formats, data objects and graphical interfaces.  Structure soln. by macromol. crystallog. is becoming increasingly automated and the CCP4 suite includes several automation pipelines.  After giving a brief description of the evolution of CCP4 over the last 30 years, an overview of the current suite is given.  While detailed descriptions are given in the accompanying articles, here it is shown how the individual programs contribute to a complete software package.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_wWMMIol6UrVg90H21EOLACvtfcHk0ljANkqcYa0fWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D&md5=c407e4d47bef46864be336d60147c17d</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1107%2FS0907444910045749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910045749%26sid%3Dliteratum%253Aachs%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DBallard%26aufirst%3DC.%2BC.%26aulast%3DCowtan%26aufirst%3DK.%2BD.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DKeegan%26aufirst%3DR.%2BM.%26aulast%3DKrissinel%26aufirst%3DE.%2BB.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%26aulast%3DMcNicholas%26aufirst%3DS.%2BJ.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DPotterton%26aufirst%3DE.%2BA.%26aulast%3DPowell%26aufirst%3DH.%2BR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DWilson%26aufirst%3DK.%2BS.%26atitle%3DOverview%2520of%2520theCCP4%2520suite%2520and%2520current%2520developments%26jtitle%3DActa%2520Crystallogr.%2520D%26date%3D2011%26volume%3D67%26spage%3D235%26epage%3D242%26doi%3D10.1107%2FS0907444910045749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Coot: model-building tools for molecular graphics</span>. <i>Acta Crystallogr. D</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2126</span>– <span class="NLM_lpage">2132</span>, <span class="refDoi"> DOI: 10.1107/S0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+model-building+tools+for+molecular+graphics&doi=10.1107%2FS0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0ljANkqcYa0fWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%2520D%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2FS0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ioerger, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sacchettini, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sauter, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span> <span> </span><span class="NLM_article-title">PHENIX: building new software for automated crystallographic structure determination</span>. <i>Acta Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">1948</span>– <span class="NLM_lpage">1954</span>, <span class="refDoi"> DOI: 10.1107/S0907444902016657</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=10.1107%2FS0907444902016657" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=12393927" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BD38XotVCksbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2002&pages=1948-1954&author=P.+D.+Adamsauthor=R.+W.+Grosse-Kunstleveauthor=L.-W.+Hungauthor=T.+R.+Ioergerauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+J.+Readauthor=J.+C.+Sacchettiniauthor=N.+K.+Sauterauthor=T.+C.+Terwilliger&title=PHENIX%3A+building+new+software+for+automated+crystallographic+structure+determination&doi=10.1107%2FS0907444902016657"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: building new software for automated crystallographic structure determination</span></div><div class="casAuthors">Adams, Paul D.; Grosse-Kunstleve, Ralf W.; Hung, Li Wei; Ioerger, Thomas R.; McCoy, Airlie J.; Moriarty, Nigel W.; Read, Randy J.; Sacchettini, James C.; Sauter, Nicholas K.; Terwilliger, Thomas C.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">D58</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1948-1954</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Munksgaard</span>)
        </div><div class="casAbstract">Structural genomics seeks to rapidly expand the no. of protein structures to ext. the max. amt. of information from genomic sequence databases.  The advent of several large-scale projects worldwide leads to many new challenges in the field of crystallog. macromol. structure detn.  A novel software package, called PHENIX (Python-based Hierarchical Environment for Integrated Xtallog.), is therefore being developed.  This new software will provide the necessary algorithms to proceed from reduced intensity data to a refined mol. model and to facilitate structure soln. for both the novice and expert crystallographer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOtuUfHXPmCbVg90H21EOLACvtfcHk0lg6iLWq1DLufw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XotVCksbc%253D&md5=9a3b66e1f7d064183f7a9ce108cb4e5e</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1107%2FS0907444902016657&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444902016657%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DHung%26aufirst%3DL.-W.%26aulast%3DIoerger%26aufirst%3DT.%2BR.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DSacchettini%26aufirst%3DJ.%2BC.%26aulast%3DSauter%26aufirst%3DN.%2BK.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26atitle%3DPHENIX%253A%2520building%2520new%2520software%2520for%2520automated%2520crystallographic%2520structure%2520determination%26jtitle%3DActa%2520Crystallogr.%26date%3D2002%26volume%3D58%26spage%3D1948%26epage%3D1954%26doi%3D10.1107%2FS0907444902016657" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Best, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shim, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopes, P. E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mittal, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feig, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacKerell, A. D.,  Jr.</span></span> <span> </span><span class="NLM_article-title">Optimization of the Additive CHARMM all-atom protein force field targeting improved sampling of the backbone ϕ, ψ and side-chain χ1 and χ2 dihedral angles</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">3257</span>– <span class="NLM_lpage">3273</span>, <span class="refDoi"> DOI: 10.1021/ct300400x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct300400x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVKqurfP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=3257-3273&author=R.+B.+Bestauthor=X.+Zhuauthor=J.+Shimauthor=P.+E.+M.+Lopesauthor=J.+Mittalauthor=M.+Feigauthor=A.+D.+MacKerell&title=Optimization+of+the+Additive+CHARMM+all-atom+protein+force+field+targeting+improved+sampling+of+the+backbone+%CF%95%2C+%CF%88+and+side-chain+%CF%871+and+%CF%872+dihedral+angles&doi=10.1021%2Fct300400x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of the Additive CHARMM All-Atom Protein Force Field Targeting Improved Sampling of the Backbone .vphi., ψ and Side-Chain χ1 and χ2 Dihedral Angles</span></div><div class="casAuthors">Best, Robert B.; Zhu, Xiao; Shim, Jihyun; Lopes, Pedro E. M.; Mittal, Jeetain; Feig, Michael; MacKerell, Alexander D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3257-3273</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">While the quality of the current CHARMM22/CMAP additive force field for proteins has been demonstrated in a large no. of applications, limitations in the model with respect to the equil. between the sampling of helical and extended conformations in folding simulations have been noted.  To overcome this, as well as make other improvements in the model, we present a combination of refinements that should result in enhanced accuracy in simulations of proteins.  The common (non-Gly, -Pro) backbone CMAP potential has been refined against exptl. soln. NMR data for weakly structured peptides, resulting in a rebalancing of the energies of the α-helix and extended regions of the Ramachandran map, correcting the α-helical bias of CHARMM22/CMAP.  The Gly and Pro CMAPs have been refitted to more accurate quantum-mech. energy surfaces.  Side-chain torsion parameters have been optimized by fitting to backbone-dependent quantum-mech. energy surfaces, followed by addnl. empirical optimization targeting NMR scalar couplings for unfolded proteins.  A comprehensive validation of the revised force field was then performed against a collection of exptl. data: (i) comparison of simulations of eight proteins in their crystal environments with crystal structures; (ii) comparison with backbone scalar couplings for weakly structured peptides; (iii) comparison with NMR residual dipolar couplings and scalar couplings for both backbone and side-chains in folded proteins; (iv) equil. folding of mini-proteins.  The results indicate that the revised CHARMM 36 parameters represent an improved model for modeling and simulation studies of proteins, including studies of protein folding, assembly, and functionally relevant conformational changes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGnwGJMAMFvLVg90H21EOLACvtfcHk0lg6iLWq1DLufw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVKqurfP&md5=9a48a0c5770fb1e887c3bb34d45b1354</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fct300400x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct300400x%26sid%3Dliteratum%253Aachs%26aulast%3DBest%26aufirst%3DR.%2BB.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DShim%26aufirst%3DJ.%26aulast%3DLopes%26aufirst%3DP.%2BE.%2BM.%26aulast%3DMittal%26aufirst%3DJ.%26aulast%3DFeig%26aufirst%3DM.%26aulast%3DMacKerell%26aufirst%3DA.%2BD.%26atitle%3DOptimization%2520of%2520the%2520Additive%2520CHARMM%2520all-atom%2520protein%2520force%2520field%2520targeting%2520improved%2520sampling%2520of%2520the%2520backbone%2520%25CF%2595%252C%2520%25CF%2588%2520and%2520side-chain%2520%25CF%25871%2520and%2520%25CF%25872%2520dihedral%2520angles%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2012%26volume%3D8%26spage%3D3257%26epage%3D3273%26doi%3D10.1021%2Fct300400x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vanommeslaeghe, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatcher, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Acharya, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kundu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shim, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darian, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guvench, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopes, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vorobyov, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mackerell, A. D.,  Jr.</span></span> <span> </span><span class="NLM_article-title">CHARMM general force field: A force field for drug-like molecules compatible with the CHARMM all-atom additive biological force fields</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">671</span>– <span class="NLM_lpage">690</span>, <span class="refDoi"> DOI: 10.1002/jcc.21367</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=10.1002%2Fjcc.21367" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=19575467" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlentbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2010&pages=671-690&author=K.+Vanommeslaegheauthor=E.+Hatcherauthor=C.+Acharyaauthor=S.+Kunduauthor=S.+Zhongauthor=J.+Shimauthor=E.+Darianauthor=O.+Guvenchauthor=P.+Lopesauthor=I.+Vorobyovauthor=A.+D.+Mackerell&title=CHARMM+general+force+field%3A+A+force+field+for+drug-like+molecules+compatible+with+the+CHARMM+all-atom+additive+biological+force+fields&doi=10.1002%2Fjcc.21367"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">CHARMM general force field: A force field for drug-like molecules compatible with the CHARMM all-atom additive biological force fields</span></div><div class="casAuthors">Vanommeslaeghe, K.; Hatcher, E.; Acharya, C.; Kundu, S.; Zhong, S.; Shim, J.; Darian, E.; Guvench, O.; Lopes, P.; Vorobyov, I.; Mackerell, A. D., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">671-690</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The widely used CHARMM additive all-atom force field includes parameters for proteins, nucleic acids, lipids, and carbohydrates.  In the present article, an extension of the CHARMM force field to drug-like mols. is presented.  The resulting CHARMM General Force Field (CGenFF) covers a wide range of chem. groups present in biomols. and drug-like mols., including a large no. of heterocyclic scaffolds.  The parametrization philosophy behind the force field focuses on quality at the expense of transferability, with the implementation concg. on an extensible force field.  Statistics related to the quality of the parametrization with a focus on exptl. validation are presented.  Addnl., the parametrization procedure, described fully in the present article in the context of the model systems, pyrrolidine, and 3-phenoxymethyl-pyrrolidine will allow users to readily extend the force field to chem. groups that are not explicitly covered in the force field as well as add functional groups to and link together mols. already available in the force field.  CGenFF thus makes it possible to perform "all-CHARMM" simulations on drug-target interactions thereby extending the utility of CHARMM force fields to medicinally relevant systems. © 2009 Wiley Periodicals, Inc.J Comput Chem, 2010.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCuh0CLdmxN7Vg90H21EOLACvtfcHk0lg6iLWq1DLufw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlentbc%253D&md5=26212e0e4f73bded0c89d4b411cd3833</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1002%2Fjcc.21367&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.21367%26sid%3Dliteratum%253Aachs%26aulast%3DVanommeslaeghe%26aufirst%3DK.%26aulast%3DHatcher%26aufirst%3DE.%26aulast%3DAcharya%26aufirst%3DC.%26aulast%3DKundu%26aufirst%3DS.%26aulast%3DZhong%26aufirst%3DS.%26aulast%3DShim%26aufirst%3DJ.%26aulast%3DDarian%26aufirst%3DE.%26aulast%3DGuvench%26aufirst%3DO.%26aulast%3DLopes%26aufirst%3DP.%26aulast%3DVorobyov%26aufirst%3DI.%26aulast%3DMackerell%26aufirst%3DA.%2BD.%26atitle%3DCHARMM%2520general%2520force%2520field%253A%2520A%2520force%2520field%2520for%2520drug-like%2520molecules%2520compatible%2520with%2520the%2520CHARMM%2520all-atom%2520additive%2520biological%2520force%2520fields%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2010%26volume%3D31%26spage%3D671%26epage%3D690%26doi%3D10.1002%2Fjcc.21367" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandrasekhar, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madura, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Impey, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, M. L.</span></span> <span> </span><span class="NLM_article-title">Comparison of simple potential functions for simulating liquid water</span>. <i>J. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">1983</span>,  <span class="NLM_volume"><i>79</i></span>,  <span class="NLM_fpage">926</span>– <span class="NLM_lpage">935</span>, <span class="refDoi"> DOI: 10.1063/1.445869</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=10.1063%2F1.445869" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADyaL3sXksF2htL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=1983&pages=926-935&author=W.+L.+Jorgensenauthor=J.+Chandrasekharauthor=J.+D.+Maduraauthor=R.+W.+Impeyauthor=M.+L.+Klein&title=Comparison+of+simple+potential+functions+for+simulating+liquid+water&doi=10.1063%2F1.445869"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of simple potential functions for simulating liquid water</span></div><div class="casAuthors">Jorgensen, William L.; Chandrasekhar, Jayaraman; Madura, Jeffry D.; Impey, Roger W.; Klein, Michael L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1983</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">926-35</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">Classical Monte Carlo simulations were carried out for liq. H2O in the NPT ensemble at 25° and 1 atm using 6 of the simpler intermol. potential functions for the dimer.  Comparisons were made with exptl. thermodn. and structural data including the neutron diffraction results of Thiessen and Narten (1982).  The computed densities and potential energies agree with expt. except for the original Bernal-Fowler model, which yields an 18% overest. of the d. and poor structural results.  The discrepancy may be due to the correction terms needed in processing the neutron data or to an effect uniformly neglected in the computations.  Comparisons were made for the self-diffusion coeffs. obtained from mol. dynamics simulations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpK4QUFMvwYFrVg90H21EOLACvtfcHk0lhLosrv2fXesg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3sXksF2htL4%253D&md5=a1161334e381746be8c9b15a5e56f704</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1063%2F1.445869&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.445869%26sid%3Dliteratum%253Aachs%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DChandrasekhar%26aufirst%3DJ.%26aulast%3DMadura%26aufirst%3DJ.%2BD.%26aulast%3DImpey%26aufirst%3DR.%2BW.%26aulast%3DKlein%26aufirst%3DM.%2BL.%26atitle%3DComparison%2520of%2520simple%2520potential%2520functions%2520for%2520simulating%2520liquid%2520water%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1983%26volume%3D79%26spage%3D926%26epage%3D935%26doi%3D10.1063%2F1.445869" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hess, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bekker, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berendsen, H. J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fraaije, J. G. E. M.</span></span> <span> </span><span class="NLM_article-title">LINCS: A linear constraint solver for molecular simulations</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">1463</span>– <span class="NLM_lpage">1472</span>, <span class="refDoi"> DOI: 10.1002/(SICI)1096-987X(199709)18:12<1463::AID-JCC4>3.0.CO;2-H</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=10.1002%2F%28SICI%291096-987X%28199709%2918%3A12%3C1463%3A%3AAID-JCC4%3E3.0.CO%3B2-H" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADyaK2sXlvV2nu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=1997&pages=1463-1472&author=B.+Hessauthor=H.+Bekkerauthor=H.+J.+C.+Berendsenauthor=J.+G.+E.+M.+Fraaije&title=LINCS%3A+A+linear+constraint+solver+for+molecular+simulations&doi=10.1002%2F%28SICI%291096-987X%28199709%2918%3A12%3C1463%3A%3AAID-JCC4%3E3.0.CO%3B2-H"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">LINCS: a linear constraint solver for molecular simulations</span></div><div class="casAuthors">Hess, Berk; Bekker, Henk; Berendsen, Herman J. C.; Fraaije, Johannes G. E. M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1463-1472</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">Wiley</span>)
        </div><div class="casAbstract">We present a new LINear Constraint Solver (LINCS) for mol. simulations with bond constraints using the enzyme lysozyme and a 32-residue peptide as test systems.  The algorithm is inherently stable, as the constraints themselves are reset instead of derivs. of the constraints, thereby eliminating drift.  Although the derivation of the algorithm is presented in terms of matrixes, no matrix matrix multiplications are needed and only the nonzero matrix elements have to be stored, making the method useful for very large mols.  At the same accuracy, the LINCS algorithm is 3-4 times faster than the SHAKE algorithm.  Parallelization of the algorithm is straightforward.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpk6h4pdV0Nx7Vg90H21EOLACvtfcHk0lhLosrv2fXesg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXlvV2nu7g%253D&md5=890f8af0d2ca1f65aa93db5a3a0bacf2</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291096-987X%28199709%2918%3A12%3C1463%3A%3AAID-JCC4%3E3.0.CO%3B2-H&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291096-987X%2528199709%252918%253A12%253C1463%253A%253AAID-JCC4%253E3.0.CO%253B2-H%26sid%3Dliteratum%253Aachs%26aulast%3DHess%26aufirst%3DB.%26aulast%3DBekker%26aufirst%3DH.%26aulast%3DBerendsen%26aufirst%3DH.%2BJ.%2BC.%26aulast%3DFraaije%26aufirst%3DJ.%2BG.%2BE.%2BM.%26atitle%3DLINCS%253A%2520A%2520linear%2520constraint%2520solver%2520for%2520molecular%2520simulations%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1997%26volume%3D18%26spage%3D1463%26epage%3D1472%26doi%3D10.1002%2F%28SICI%291096-987X%28199709%2918%3A12%3C1463%3A%3AAID-JCC4%3E3.0.CO%3B2-H" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Darden, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">York, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pedersen, L.</span></span> <span> </span><span class="NLM_article-title">Particle mesh Ewald: An N·log(N) method for Ewald sums in large systems</span>. <i>J. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>98</i></span>,  <span class="NLM_fpage">10089</span>– <span class="NLM_lpage">10092</span>, <span class="refDoi"> DOI: 10.1063/1.464397</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=10.1063%2F1.464397" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADyaK3sXks1Ohsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=1993&pages=10089-10092&author=T.+Dardenauthor=D.+Yorkauthor=L.+Pedersen&title=Particle+mesh+Ewald%3A+An+N%C2%B7log%28N%29+method+for+Ewald+sums+in+large+systems&doi=10.1063%2F1.464397"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Particle mesh Ewald: an N·log(N) method for Ewald sums in large systems</span></div><div class="casAuthors">Darden, Tom; York, Darrin; Pedersen, Lee</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">10089-92</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">An N·log(N) method for evaluating electrostatic energies and forces of large periodic systems is presented.  The method is based on interpolation of the reciprocal space Ewald sums and evaluation of the resulting convolution using fast Fourier transforms.  Timings and accuracies are presented for three large cryst. ionic systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDIJavf7AZPbVg90H21EOLACvtfcHk0lhLosrv2fXesg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXks1Ohsr0%253D&md5=3c9f230bd01b7b714fd096d4d2e755f6</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1063%2F1.464397&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.464397%26sid%3Dliteratum%253Aachs%26aulast%3DDarden%26aufirst%3DT.%26aulast%3DYork%26aufirst%3DD.%26aulast%3DPedersen%26aufirst%3DL.%26atitle%3DParticle%2520mesh%2520Ewald%253A%2520An%2520N%25C2%25B7log%2528N%2529%2520method%2520for%2520Ewald%2520sums%2520in%2520large%2520systems%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1993%26volume%3D98%26spage%3D10089%26epage%3D10092%26doi%3D10.1063%2F1.464397" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hess, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kutzner, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der
Spoel, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindahl, E.</span></span> <span> </span><span class="NLM_article-title">GROMACS 4: Algorithms for highly efficient, load-balanced, and scalable molecular simulation</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">435</span>– <span class="NLM_lpage">447</span>, <span class="refDoi"> DOI: 10.1021/ct700301q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct700301q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVSqurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2008&pages=435-447&author=B.+Hessauthor=C.+Kutznerauthor=D.+van+der%0ASpoelauthor=E.+Lindahl&title=GROMACS+4%3A+Algorithms+for+highly+efficient%2C+load-balanced%2C+and+scalable+molecular+simulation&doi=10.1021%2Fct700301q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">GROMACS 4: Algorithms for Highly Efficient, Load-Balanced, and Scalable Molecular Simulation</span></div><div class="casAuthors">Hess, Berk; Kutzner, Carsten; van der Spoel, David; Lindahl, Erik</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">435-447</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. simulation is an extremely useful, but computationally very expensive tool for studies of chem. and biomol. systems.  Here, we present a new implementation of our mol. simulation toolkit GROMACS which now both achieves extremely high performance on single processors from algorithmic optimizations and hand-coded routines and simultaneously scales very well on parallel machines.  The code encompasses a minimal-communication domain decompn. algorithm, full dynamic load balancing, a state-of-the-art parallel constraint solver, and efficient virtual site algorithms that allow removal of hydrogen atom degrees of freedom to enable integration time steps up to 5 fs for atomistic simulations also in parallel.  To improve the scaling properties of the common particle mesh Ewald electrostatics algorithms, we have in addn. used a Multiple-Program, Multiple-Data approach, with sep. node domains responsible for direct and reciprocal space interactions.  Not only does this combination of algorithms enable extremely long simulations of large systems but also it provides that simulation performance on quite modest nos. of std. cluster nodes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrajjWdk6mUFrVg90H21EOLACvtfcHk0lhLosrv2fXesg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVSqurc%253D&md5=d53c94901386260221792ea30f151c5f</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Fct700301q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct700301q%26sid%3Dliteratum%253Aachs%26aulast%3DHess%26aufirst%3DB.%26aulast%3DKutzner%26aufirst%3DC.%26aulast%3Dvan%2Bder%2BSpoel%26aufirst%3DD.%26aulast%3DLindahl%26aufirst%3DE.%26atitle%3DGROMACS%25204%253A%2520Algorithms%2520for%2520highly%2520efficient%252C%2520load-balanced%252C%2520and%2520scalable%2520molecular%2520simulation%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2008%26volume%3D4%26spage%3D435%26epage%3D447%26doi%3D10.1021%2Fct700301q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1FTA" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1FTA','PDB','1FTA'); return false;">PDB: 1FTA</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ZWK" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5ZWK','PDB','5ZWK'); return false;">PDB: 5ZWK</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6LS5" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6LS5','PDB','6LS5'); return false;">PDB: 6LS5</a></li><li><a href="http://www.ncbi.nlm.nih.gov/gquery?Term=D26055.1" onclick="trackOutboundLink('http://www.ncbi.nlm.nih.gov/gquery?Term=D26055.1','Genbank','D26055.1'); return false;">Genbank: D26055.1</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes2"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i27"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00699">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_74126"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c00699?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00699</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Molecular dynamic results (PDB); <sup>1</sup>H NMR, <sup>13</sup>C NMR, and high-resolution mass spectra (HRMS) data; HPLC spectra data for <b>212</b>/<b>214</b>; and WT and mutant enzyme data and proposed reaction mechanism calculation for <b>212</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00699/suppl_file/jm0c00699_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00699/suppl_file/jm0c00699_si_002.csv">CSV</a>)</p></li><li><p class="inline">Molecular dynamic result for <b>214</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00699/suppl_file/jm0c00699_si_003.pdb">PDB</a>)</p></li><li><p class="inline">Molecular dynamic result for APO (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00699/suppl_file/jm0c00699_si_004.pdb">PDB</a>)</p></li><li><p class="inline">Molecular dynamic result for FBP (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00699/suppl_file/jm0c00699_si_005.pdb">PDB</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00699/suppl_file/jm0c00699_si_001.pdf">jm0c00699_si_001.pdf (832.16 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00699/suppl_file/jm0c00699_si_002.csv">jm0c00699_si_002.csv (0.84 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00699/suppl_file/jm0c00699_si_003.pdb">jm0c00699_si_003.pdb (779.7 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00699/suppl_file/jm0c00699_si_004.pdb">jm0c00699_si_004.pdb (819.21 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00699/suppl_file/jm0c00699_si_005.pdb">jm0c00699_si_005.pdb (823.16 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The atomic coordinates and structure factors have been deposited into the RCSB Protein Data Bank with accession numbers 6LS5. Authors will release the atomic coordinates and experimental data upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c00699&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-11%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c00699" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c00699" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                5MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679ac456892d2398","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
