
<html lang="en"     class="pb-page"  data-request-id="6a2d755b-f575-444c-af76-8e0d18657433"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;issue:issue:10.1021/jmcmar.2016.59.issue-1;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.5b01361;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="3-(Piperidin-4-ylmethoxy)pyridine Containing Compounds Are Potent Inhibitors of Lysine Specific Demethylase 1" /></meta><meta name="dc.Creator" content="Fangrui  Wu" /></meta><meta name="dc.Creator" content="Chao  Zhou" /></meta><meta name="dc.Creator" content="Yuan  Yao" /></meta><meta name="dc.Creator" content="Liping  Wei" /></meta><meta name="dc.Creator" content="Zizhen  Feng" /></meta><meta name="dc.Creator" content="Lisheng  Deng" /></meta><meta name="dc.Creator" content="Yongcheng  Song" /></meta><meta name="dc.Description" content="Methylation of histone lysine residues plays important roles in gene expression regulation as well as cancer initiation. Lysine specific demethylase 1 (LSD1) is responsible for maintaining balanced..." /></meta><meta name="Description" content="Methylation of histone lysine residues plays important roles in gene expression regulation as well as cancer initiation. Lysine specific demethylase 1 (LSD1) is responsible for maintaining balanced..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="December 24, 2015" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.5b01361" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2015 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.5b01361" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.5b01361" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.5b01361" /></link>
        
    
    

<title>3-(Piperidin-4-ylmethoxy)pyridine Containing Compounds Are Potent Inhibitors of Lysine Specific Demethylase 1 | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.5b01361" /></meta><meta property="og:title" content="3-(Piperidin-4-ylmethoxy)pyridine Containing Compounds Are Potent Inhibitors of Lysine Specific Demethylase 1" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-1/acs.jmedchem.5b01361/20160113/images/large/jm-2015-013613_0014.jpeg" /></meta><meta property="og:description" content="Methylation of histone lysine residues plays important roles in gene expression regulation as well as cancer initiation. Lysine specific demethylase 1 (LSD1) is responsible for maintaining balanced methylation levels at histone H3 lysine 4 (H3K4). LSD1 is a drug target for certain cancers, due to important functions of methylated H3K4 or LSD1 overexpression. We report the design, synthesis, and structure–activity relationships of 3-(piperidin-4-ylmethoxy)pyridine containing compounds as potent LSD1 inhibitors with Ki values as low as 29 nM. These compounds exhibited high selectivity (&gt;160×) against related monoamine oxidase A and B. Enzyme kinetics and docking studies suggested they are competitive inhibitors against a dimethylated H3K4 substrate and provided a possible binding mode. The potent LSD1 inhibitors can increase cellular H3K4 methylation and strongly inhibit proliferation of several leukemia and solid tumor cells with EC50 values as low as 280 nM, while they had negligible effects on normal cells." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.5b01361"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.5b01361">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.5b01361&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.5b01361&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.5b01361&amp;href=/doi/10.1021/acs.jmedchem.5b01361" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2016</span><span class="cit-fg-volume">, 59</span><span class="cit-fg-issue">, 1</span><span class="cit-fg-pageRange">, 253-263</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/59/1" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.5b01284" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.5b01374" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">3-(Piperidin-4-ylmethoxy)pyridine Containing Compounds Are Potent Inhibitors of Lysine Specific Demethylase 1</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Fangrui++Wu">Fangrui Wu</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Chao++Zhou">Chao Zhou</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Yuan++Yao">Yuan Yao</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Liping++Wei">Liping Wei</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Zizhen++Feng">Zizhen Feng</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Lisheng++Deng">Lisheng Deng</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Yongcheng++Song">Yongcheng Song</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-text"><sup>†</sup>Department of Pharmacology, and <sup>‡</sup>Dan L. Duncan Cancer Center, Baylor College of Medicine, 1 Baylor Plaza, Houston, Texas 77030, United States</span></div><div class="corresp-info"><strong>*</strong>Phone: 713-798-7415. E-mail: <a href="/cdn-cgi/l/email-protection#c6bfb5a9a8a186a4a5abe8a3a2b3"><span class="__cf_email__" data-cfemail="a4ddd7cbcac3e4c6c7c98ac1c0d1">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.5b01361&amp;href=/doi/10.1021%2Facs.jmedchem.5b01361" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2016</span></span><span class="cit-volume">, 59</span><span class="cit-issue">, 1</span><span class="cit-pageRange">, 253–263</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">December 10, 2015</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>3 September 2015</li><li><span class="item_label"><b>Published</b> online</span>24 December 2015</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 14 January 2016</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.5b01361" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.5b01361</a></div><div class="article_header-article-copyright"><strong>Copyright © 2015 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D253%26pageCount%3D11%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DFangrui%2BWu%252C%2BChao%2BZhou%252C%2BYuan%2BYao%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D59%26issueNum%3D1%26contentID%3Dacs.jmedchem.5b01361%26title%3D3-%2528Piperidin-4-ylmethoxy%2529pyridine%2BContaining%2BCompounds%2BAre%2BPotent%2BInhibitors%2Bof%2BLysine%2BSpecific%2BDemethylase%2B1%26numPages%3D11%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D263%26publicationDate%3DJanuary%2B2016">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.5b01361"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2285</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">48</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.5b01361" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;3-(Piperidin-4-ylmethoxy)pyridine Containing Compounds Are Potent Inhibitors of Lysine Specific Demethylase 1&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Fangrui&quot;,&quot;last_name&quot;:&quot;Wu&quot;},{&quot;first_name&quot;:&quot;Chao&quot;,&quot;last_name&quot;:&quot;Zhou&quot;},{&quot;first_name&quot;:&quot;Yuan&quot;,&quot;last_name&quot;:&quot;Yao&quot;},{&quot;first_name&quot;:&quot;Liping&quot;,&quot;last_name&quot;:&quot;Wei&quot;},{&quot;first_name&quot;:&quot;Zizhen&quot;,&quot;last_name&quot;:&quot;Feng&quot;},{&quot;first_name&quot;:&quot;Lisheng&quot;,&quot;last_name&quot;:&quot;Deng&quot;},{&quot;first_name&quot;:&quot;Yongcheng&quot;,&quot;last_name&quot;:&quot;Song&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2015&quot;,&quot;month&quot;:&quot;12&quot;,&quot;day&quot;:&quot;24&quot;,&quot;issue&quot;:&quot;1&quot;,&quot;volume&quot;:&quot;59&quot;,&quot;pages&quot;:&quot;253-263&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.5b01361&quot;},&quot;abstract&quot;:&quot;Methylation of histone lysine residues plays important roles in gene expression regulation as well as cancer initiation. Lysine specific demethylase 1 (LSD1) is responsible for maintaining balanced methylation levels at histone H3 lysine 4 (H3K4). LSD1 is a drug target for certain cancers, due to important functions of methylated H3K4 or LSD1 overexpression. We report the design, synthesis, and structure–activity relationships of 3-(piperidin-4-ylmethoxy)pyridine containing compounds as potent LSD1 inhibitors with Ki values as low as 29 nM. These compounds exhibited high selectivity (&gt;160×) against related monoamine oxidase A and B. Enzyme kinetics and docking studies suggested they are competitive inhibitors against a dimethylated H3K4 substrate and provided a possible binding mode. The potent LSD1 inhibitors can increase cellular H3K4 methylation and strongly inhibit proliferation of several leukemia and solid tumor cells with EC50 values as low as 280 nM, while they had negligible effects on normal cells.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b01361&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01361" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b01361&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01361" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b01361&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01361" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.5b01361&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01361&amp;href=/doi/10.1021/acs.jmedchem.5b01361" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.5b01361" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.5b01361" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (3 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.5b01361&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.5b01361%26sid%3Dliteratum%253Aachs%26pmid%3D26652247%26genre%3Darticle%26aulast%3DWu%26date%3D2016%26atitle%3D3-%2528Piperidin-4-ylmethoxy%2529pyridine%2BContaining%2BCompounds%2BAre%2BPotent%2BInhibitors%2Bof%2BLysine%2BSpecific%2BDemethylase%2B1%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D59%26issue%3D1%26spage%3D253%26epage%3D263%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=292258" title="Cancer">Cancer</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291711" title="Selectivity">Selectivity</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/59/1" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-1/jmcmar.2016.59.issue-1/20160114/jmcmar.2016.59.issue-1.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-1/acs.jmedchem.5b01361/20160113/images/medium/jm-2015-013613_0014.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-1/acs.jmedchem.5b01361/20160113/images/large/jm-2015-013613_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01361&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Methylation of histone lysine residues plays important roles in gene expression regulation as well as cancer initiation. Lysine specific demethylase 1 (LSD1) is responsible for maintaining balanced methylation levels at histone H3 lysine 4 (H3K4). LSD1 is a drug target for certain cancers, due to important functions of methylated H3K4 or LSD1 overexpression. We report the design, synthesis, and structure–activity relationships of 3-(piperidin-4-ylmethoxy)pyridine containing compounds as potent LSD1 inhibitors with <i>K</i><sub>i</sub> values as low as 29 nM. These compounds exhibited high selectivity (>160×) against related monoamine oxidase A and B. Enzyme kinetics and docking studies suggested they are competitive inhibitors against a dimethylated H3K4 substrate and provided a possible binding mode. The potent LSD1 inhibitors can increase cellular H3K4 methylation and strongly inhibit proliferation of several leukemia and solid tumor cells with EC<sub>50</sub> values as low as 280 nM, while they had negligible effects on normal cells.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_60761" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_60761" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Post-translational modifications of the ε-amino group of histone lysine residues, such as acetylation and methylation, play important roles in gene expression regulation in normal physiology as well as diseases.<a onclick="showRef(event, 'ref1 ref2 ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3 ref4">(1-4)</a> Small molecule modulators of histone modifying enzymes are therefore chemical probes for biological studies and potentially useful therapeutics for such diseases.<a onclick="showRef(event, 'ref5 ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref5 ref6 ref7">(5-7)</a> Histone H3 lysine 4 (H3K4) methylation, including mono-, di-, and trimethylation (H3K4me1, -2, and -3) has been found to be generally a biomarker for active gene transcription.<a onclick="showRef(event, 'ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref8 ref9">(8, 9)</a> Mixed lineage leukemia (MLL), a large, multidomain protein containing ∼4000 amino acids, can methylate H3K4, with its C-terminal SET domain being catalytically active.<a onclick="showRef(event, 'ref9 ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref9 ref10 ref11">(9-11)</a> In addition, cellular H3K4 methylation levels are tightly regulated because of their important functions. Lysine specific demethylase 1 (LSD1, also known as KDM1a), has been found to be a member of an MLL transcription complex. LSD1 catalyzes an oxidative demethylation reaction from H3K4me1 and -2 (but not H3K4me3) and thus counteracts the enzyme function of MLL.<a onclick="showRef(event, 'ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref12 ref13">(12, 13)</a> Both MLL and LSD1 are essential for hematopoiesis, showing a balanced cellular H3K4 methylation is crucial for normal hematopoiesis.<a onclick="showRef(event, 'ref11 ref14'); return false;" href="javascript:void(0);" class="ref ref11 ref14">(11, 14)</a> In addition, H3K9<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> and other proteins such as DNMT1<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> were reported as a substrate of LSD1.</div><div class="NLM_p">Dysregulated histone lysine methylations may lead to a disease, such as cancer.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> For example, ∼75% acute leukemia in infants and 10% of that in adults contain an MLL gene translocation and this subtype of leukemia shows a poor prognosis.<a onclick="showRef(event, 'ref17 ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref17 ref18 ref19">(17-19)</a> All MLL translocations identified to date produce an onco-MLL fusion protein without the C-terminal SET domain, which cannot methylate H3K4.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> Rather, the fusion partners can recruit DOT1L, a histone H3 lysine 79 (H3K79) methyltransferase, which causes aberrant H3K79 methylation at MLL target gene loci and overexpression of these genes, eventually leading to the initiation of the leukemia.<a onclick="showRef(event, 'ref20 ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref20 ref21 ref22">(20-22)</a> Pharmacological inhibition of DOT1L has been found to block proliferation of MLL-rearranged leukemia cells, showing effectiveness of the targeted epigenetic therapy.<a onclick="showRef(event, 'ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref23 ref24">(23, 24)</a></div><div class="NLM_p">LSD1 is a flavin adenine dinucleotide (FAD) dependent oxidase, related to monoamine oxidase A and B (MAO-A and -B).<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a><a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>A schematically shows the mechanism of catalysis of LSD1. Upon binding to LSD1, the methyl group of a methylated histone lysine residue (e.g., H3K4me2) is oxidized by the cofactor FAD, followed by hydrolysis to give formaldehyde and demethylated lysine. FADH<sub>2</sub>, the reduced form of FAD, is oxidized by O<sub>2</sub> in the solvent to give H<sub>2</sub>O<sub>2</sub> and FAD to complete a catalytic cycle.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> A number of LSD1 inhibitors with several chemotypes, including cyclopropylamine, propargylamine, hydrazine, and triazole dithiocarbamate, have been reported in journals and patents,<a onclick="showRef(event, 'ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref26 ref27">(26, 27)</a> as representatively shown in <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>B. Most LSD1 inhibitors are cyclopropylamine containing compounds, derived from tranylcypromine (<b>1</b>), an inhibitor of MAO-A and -B as well as an FDA-approved antidepression drug. Mechanistically, the α-C atom of the cyclopropyl ring of these compounds is oxidized, causing opening of the constrained ring followed by covalent link to FAD (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01361/suppl_file/jm5b01361_si_001.pdf" class="ext-link">Supporting Information Figure S1</a>).<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> Reversible LSD1 inhibitors have also been reported.<a onclick="showRef(event, 'cit26b'); return false;" href="javascript:void(0);" class="ref cit26b">(26b)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-1/acs.jmedchem.5b01361/20160113/images/medium/jm-2015-013613_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-1/acs.jmedchem.5b01361/20160113/images/large/jm-2015-013613_0005.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. (A) Mechanism of catalysis for LSD1. (B) Structures of representative LSD1 inhibitors, among which compounds in the upper panel are irreversible inhibitors and those in the lower panel are reversible.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-1/acs.jmedchem.5b01361/20160113/images/large/jm-2015-013613_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01361&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">LSD1 has been suggested to be a drug target for several types of cancers (e.g., prostate and breast) due to its overexpression in these malignancies.<a onclick="showRef(event, 'ref28 ref29 ref30'); return false;" href="javascript:void(0);" class="ref ref28 ref29 ref30">(28-30)</a> In addition, LSD1 was recently found to be required for an MLL-rearranged leukemia.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> Knockdown of LSD1 can abrogate the transforming ability of the onco-MLL fusion gene and increase the H3K4me2 levels. Using a cyclopropylamine based LSD1 inhibitor also showed activity against the MLL-rearranged leukemia in cell and mouse models.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> However, this compound was reported to have severe toxicities. Nonetheless, these previous studies have indicated LSD1 is a validated drug target and more chemotypes of inhibitors are therefore needed.<a onclick="showRef(event, 'ref32 ref33'); return false;" href="javascript:void(0);" class="ref ref32 ref33">(32, 33)</a></div><div class="NLM_p last">Here, we report the discovery, synthesis, and structure–activity relationships (SARs) of 3-(piperidin-4-ylmethoxy)pyridine containing compounds as potent LSD1 inhibitors. Enzyme kinetics and molecular modeling studies were used to investigate the possible mechanism of inhibition as well as the binding mode of these compounds. Molecular and cell biology experiments were performed to characterize the biological activities of selected compounds. During the revision of this manuscript, we noticed that scientists at GlaxoSmithKline found a structurally similar LSD1 inhibitor and presented its preliminary data at the 2013 American Association of Cancer Research annual meeting.<a onclick="showRef(event, 'ref34 cit26b cit27i'); return false;" href="javascript:void(0);" class="ref ref34 cit26b cit27i">(34, 26b, 27i)</a></div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_02655" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_02655" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3"> Inhibitor Design and Discovery</h3><div class="NLM_p">On the basis of the results from previous<a onclick="showRef(event, 'cit27a'); return false;" href="javascript:void(0);" class="ref cit27a">(27a)</a> and our own studies on cyclopropylamine containing inhibitors of LSD1, it is clear that a second basic N-substituent, such as the piperidin-4-ylmethyl group (in red dashed circle, <a class="ref internalNav" href="#cht1" aria-label="Chart 1">Chart 1</a>) in compound <b>2</b>, can significantly increase the inhibitory activity as well as selectivity for LSD1. For example, compound <b>2</b> (IC<sub>50</sub> = 62 nM) is ∼250-fold more active against human recombinant LSD1 than compound <b>1</b> (IC<sub>50</sub> = 15.7 μM <a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a>) and, moreover, the piperidin-4-ylmethyl group enhances the enzyme selectivity for LSD1 by >1500-fold. Similar features have been observed by us and others<a onclick="showRef(event, 'cit27a'); return false;" href="javascript:void(0);" class="ref cit27a">(27a)</a> for compounds <b>S1</b> and <b>S2</b> for the (4-methylpiperazin-1-yl)carbonylmethyl group in compound <b>S2</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01361/suppl_file/jm5b01361_si_001.pdf" class="ext-link">Supporting Information Figure S2</a>). These data strongly support an important role of these basic amine groups in inhibitor recognition and/or binding to LSD1. However, there has been no structural information as to how these groups bind to LSD1, since X-ray crystallography might not be suited for this study because these groups are to be eliminated upon completion of the reaction (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01361/suppl_file/jm5b01361_si_001.pdf" class="ext-link">Figure S1</a>). Nevertheless, given its important role in inhibitory potency and selectivity for LSD1, a basic amine group such as piperidin-4-ylmethyl should be included for the design of new LSD1 inhibitors.</div><figure id="cht1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Chart 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-1/acs.jmedchem.5b01361/20160113/images/medium/jm-2015-013613_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-1/acs.jmedchem.5b01361/20160113/images/large/jm-2015-013613_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Chart 1. Inhibitor Design Leading to the Finding of New LSD1 Inhibitors<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-1/acs.jmedchem.5b01361/20160113/images/large/jm-2015-013613_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01361&amp;id=cht1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Chart a</span>IC<sub>50</sub> values of compound <b>1</b> against MAO-A and -B are from ref <a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">35</a>.</p></p></figure><div class="NLM_p">We wished to find competitive, druglike inhibitors of LSD1 that do not covalently bind to FAD or the enzyme. Compound <b>3</b> (<a class="ref internalNav" href="#cht1" aria-label="Chart 1">Chart 1</a>) was designed in which the piperidin-4-ylmethyl and the phenyl group in the potent inhibitor <b>2</b> are retained, while the reactive cyclopropylamine moiety is replaced by a chemically stable pyridine core. Our modeling (docking) study (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>A) predicted that the protonated piperidine amino group (at the physiological pH) could form a hydrogen bond with Asp555 and the pyridine ring could have favorable electrostatic and hydrophobic interactions with the flavin ring of FAD as well as Tyr761. Compound <b>3</b> was found to be inactive presumably due to the large active site of LSD1. Compound <b>4</b> was next designed with the rationale that an additional phenyl group could increase the binding affinity to LSD1, as suggested by docking (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>B). The use of the pyrazine ring in <b>4</b> was intended for quick synthesis (see the <a class="ref internalNav" href="#sec2_2" aria-label="Synthesis">Synthesis</a> section below). Compound <b>4</b> was found to be a weak inhibitor with an apparent <i>K</i><sub>i</sub> value of 47.8 μM. We next performed a small scale of SAR studies (described below) based on compound <b>4</b>, which yielded a good inhibitor <b>5</b> with a <i>K</i><sub>i</sub> value of 2.3 μM. Enzyme kinetics studies (described below) suggested that compound <b>5</b> is a competitive inhibitor of LSD1 against a dimethylated H3K4 peptide substrate. These findings, together with its druglike chemical structure, show that compound <b>5</b> represents a promising new scaffold for further inhibitor optimization.</div><figure id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-1/acs.jmedchem.5b01361/20160113/images/medium/jm-2015-013613_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-1/acs.jmedchem.5b01361/20160113/images/large/jm-2015-013613_0006.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Docking studies of potential LSD1 inhibitors, using an LSD1 crystal structure (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2V1D">2V1D</a>) as the docking template. For clarity, only key residues (in light blue) in the active site of LSD1 are shown, together with the cofactor FAD (light blue tube model). Hydrogen bonds are shown as orange dashed lines. (A) Lwest-energy docking structure of compound <b>3</b> (tube model with C atoms in green), suggesting favorable interactions between its pyridine ring and FAD and Y761. (B) Lowest-energy docking structure of <b>4</b> (in yellow), showing introducing an additional 5-phenyl group could be favorable.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-1/acs.jmedchem.5b01361/20160113/images/large/jm-2015-013613_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01361&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Synthesis</h3><div class="NLM_p"><a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a> illustrates a general method for synthesis of pyrazine containing inhibitors. A commercially available benzil compound was reacted with an equal molar amount of 2-aminoacetamide in the presence of sodium hydroxide to give 2-hydroxy-5,6-diphenylpyrazine. Upon treatment with POCl<sub>3</sub>, its 2-hydroxy group was converted to a -Cl. The compound thus obtained was subjected to a substitution reaction using a sodium salt of <i>tert</i>-butyloxycarbonyl (BOC) protected 4-(hydroxymethyl)piperidine. Pyrazine containing inhibitors (such as compound <b>4</b>) were synthesized after deprotection of the BOC group. In addition, treatment of the BOC protected intermediate compound (R = Br) with zinc cyanide in the presence of a catalytic amount of Pd(PPh<sub>3</sub>)<sub>4</sub> to produce a mixture of monocyano and dicyano substituted compounds, which can be carefully separated by column chromatography to give, upon deprotection, compounds <b>8</b>–<b>10</b> (described below).</div><figure id="sch1" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-1/acs.jmedchem.5b01361/20160113/images/medium/jm-2015-013613_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-1/acs.jmedchem.5b01361/20160113/images/large/jm-2015-013613_0010.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. General Synthetic Method for Pyrazine Containing Inhibitors<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-1/acs.jmedchem.5b01361/20160113/images/large/jm-2015-013613_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01361&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (i) 2-aminoacetamide, NaOH; (ii) POCl<sub>3</sub>; (iii) BOC protected 4-(hydroxymethyl)piperidine, NaH; (iv) HCl in 1,4-dioxane; (v) Pd(PPh<sub>3</sub>)<sub>4</sub>, Zn(CN)<sub>2</sub>.</p></p></figure><div class="NLM_p"><a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a> shows a general synthesis of 3-(piperidin-4-ylmethoxy)pyridine containing LSD1 inhibitors (such as compound <b>5</b>). A Mitsunobu reaction between 5-bromo-6-chloropyridin-3-ol and BOC protected 4-(hydroxymethyl)piperidine introduces a BOC-protected 3-(piperidin-4-ylmethoxy) R<sup>3</sup> substituent. A selective Suzuki coupling reaction was performed to give R<sup>5</sup>-substituted compound, which underwent a second Suzuki coupling reaction with its 6-Cl group to add a R<sup>6</sup>-substituent. Upon deprotection of the BOC group, 3-(piperidin-4-ylmethoxy)pyridine containing compounds <b>5</b>, <b>11</b>–<b>40</b> can be obtained. In addition, by use of similar methods, related compounds <b>41</b>–<b>45</b> can be readily synthesized (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01361/suppl_file/jm5b01361_si_001.pdf" class="ext-link">Supporting Information Experimental Section</a>).</div><figure id="sch2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-1/acs.jmedchem.5b01361/20160113/images/medium/jm-2015-013613_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-1/acs.jmedchem.5b01361/20160113/images/large/jm-2015-013613_0011.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. General Synthetic Method for Pyridine Containing Inhibitors<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-1/acs.jmedchem.5b01361/20160113/images/large/jm-2015-013613_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01361&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (i) BOC protected 4-(hydroxymethyl)piperidine, PPh<sub>3</sub>, diisopropyl azodicarboxylate; (ii) R<sup>5</sup>-B(OH)<sub>2</sub>, Pd(PPh<sub>3</sub>)<sub>4</sub>, Na<sub>2</sub>CO<sub>3</sub>, 80 °C; (iii) R<sup>6</sup>-B(OH)<sub>2</sub>, Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>3</sub>PO<sub>4</sub>, 140 °C; (iv) HCl in 1,4-dioxane.</p></p></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> SAR Studies</h3><div class="NLM_p">Upon obtaining compound <b>4</b> with an apparent <i>K</i><sub>i</sub> value of 47.8 μM, we synthesized several analogs of <b>4</b> shown in <a class="ref internalNav" href="#cht2" aria-label="Chart 2">Chart 2</a>. Compound <b>6</b> with 5,6-di-(4-tolyl) and compound <b>7</b> with 5,6-di-(4-bromophenyl) substituents are more active (<i>K</i><sub>i</sub> = 32.2 and 26.2 μM, respectively), showing the two methyl or bromo groups are favorable. Synthesized from <b>7</b> using a Pd-catalyzed reaction, compound <b>8</b> with 5,6-di-(4-cyanophenyl) was found to be considerably more active with a <i>K</i><sub>i</sub> of 9.1 μM. In addition, monocyano substituted compounds <b>9</b> and <b>10</b> were also synthesized, which can be separated by column chromatography. One isomer <b>9</b> exhibited potent inhibitory activity against LSD1 with a <i>K</i><sub>i</sub> value of 540 nM, while the other isomer <b>10</b> (<i>K</i><sub>i</sub> = 16.5 μM) is less active than compound <b>8</b>. This indicated that 5- and 6-substituent in compounds <b>4</b>, <b>6</b>–<b>10</b> should be individually optimized for LSD1 inhibition. However, this is not synthetically feasible for compounds with a pyrazine core.</div><figure id="cht2" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Chart 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-1/acs.jmedchem.5b01361/20160113/images/medium/jm-2015-013613_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-1/acs.jmedchem.5b01361/20160113/images/large/jm-2015-013613_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Chart 2. Structures and <i>K</i><sub>i</sub> Values (in Parentheses) of Compounds <b>5</b>–<b>12</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-1/acs.jmedchem.5b01361/20160113/images/large/jm-2015-013613_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01361&amp;id=cht2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The pyridine core structure, in which the two substituents can be modified separately, was used for SAR studies. Although compound <b>11</b> (<a class="ref internalNav" href="#cht2" aria-label="Chart 2">Chart 2</a>, which corresponds to the pyrazine compound <b>4</b>) is inactive, compound <b>12</b> with a <i>K</i><sub>i</sub> value of 4.0 μM is a good LSD1 inhibitor and stronger than the corresponding pyrazine compound <b>8</b>. This showed the pyridine core can be a good scaffold for activity optimization.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Structures and Activity of Compounds <b>5</b>, <b>12</b>–<b>40</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-1/acs.jmedchem.5b01361/20160113/images/medium/jm-2015-013613_0012.gif" alt="" id="fx1" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-1/acs.jmedchem.5b01361/20160113/images/medium/jm-2015-013613_0013.gif" alt="" id="fx2" /></img><div></div></div><div class="NLM_p">The 6-position of the pyridine ring was optimized, and the results are summarized in <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>. Compound <b>5</b> with an unsubstituted 6-phenyl group was found to be slightly more active (<i>K</i><sub>i</sub> = 2.3 μM) than <b>12</b>. Compound <b>13</b> having a 4-fluorophenyl group exhibited potent inhibitory activity with a <i>K</i><sub>i</sub> of 220 nM, ∼10× more active than the parent compound <b>5</b>. Adding an additional -F at the 3- or 2-position in compound <b>14</b> (<i>K</i><sub>i</sub> = 570 nM) or <b>15</b> (<i>K</i><sub>i</sub> = 1.2 μM), respectively, resulted in a ∼2.5× or 5.5× activity reduction, suggesting that 3- or 2-substitution of the phenyl ring seems to be disfavored. The 4-trifluoromethyl group in compound <b>16</b> considerably enhanced the inhibitory potency to 58 nM. Change to a 4-methyl group in compound <b>17</b> (<i>K</i><sub>i</sub> = 29 nM) further caused a ∼2-fold activity increase. Move of the methyl group to 3-position in compound <b>18</b> (<i>K</i><sub>i</sub> = 800 nM) drastically decreased the activity by ∼27-fold, showing again substitution at the 4-position of the 6-phenyl ring is favorable. Compounds <b>19</b> and <b>20</b> with a 4-ethyl and 4-isopropyl group, respectively, were found to be ∼7× and 3× less active (<i>K</i><sub>i</sub> = 200 and 88 nM) than <b>17</b> with a 4-methyl, showing a bigger alkyl group at this position is less favorable. As compared to the most potent inhibitor <b>17</b>, while compound <b>21</b> (<i>K</i><sub>i</sub> = 740 nM) having a 4-methoxy group showed a significant activity reduction, compound <b>22</b> with a 4-trifluoromethoxy substituent exhibited a similar inhibitory activity (<i>K</i><sub>i</sub> = 46 nM). Activities of compounds <b>23</b>–<b>26</b> indicated that a 6-pyridinyl ring is less favorable than a phenyl ring. Compounds <b>23</b> and <b>24</b> (<i>K</i><sub>i</sub> = 5.0 and 3.2 μM) with a pyridin-4-yl and -3-yl ring, respectively, are slightly less active than is compound <b>5</b> (<i>K</i><sub>i</sub> = 2.3 μM). More noticeably, compound <b>25</b> (<i>K</i><sub>i</sub> = 1.8 μM) with a 4-fluoropyridin-3-yl substituent exhibited a ∼8-fold activity reduction as compared to the corresponding phenyl compound <b>13</b>. Although the 4-methyl group in compound <b>26</b> resulted in a significant activity increase (<i>K</i><sub>i</sub> = 380 nM, as compared to 3.2 μM for compound <b>24</b>), compound <b>26</b> is still ∼13× less active than the corresponding phenyl compound <b>17</b>. In addition, introducing a 3-thiophenyl ring to the 6-position produced compound <b>27</b>, which exhibited a comparable inhibitory activity (<i>K</i><sub>i</sub> = 2.4 μM) to that of the phenyl compound <b>5</b>.</div><div class="NLM_p">Next, compounds <b>28</b>–<b>40</b> were synthesized and tested for LSD1 inhibition for SAR studies of the 5-position of the most potent inhibitor <b>17</b>. As summarized in <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>, the 4-cyano substituent of the 5-phenyl ring has turned out to be critically important for LSD1 inhibition. Compound <b>28</b> with a 4-ethynyl group was surprisingly found to suffer from a complete (>1700×) activity loss, despite a similar size of the R<sup>5</sup> groups. This could be due to different electrostatic properties between the ethynyl (in <b>28</b>) and the cyano group (in <b>17</b>). Compounds <b>29</b> and <b>30</b> with a 4-carboxy and 4-hydroxymethyl group in the 5-phenyl ring also completely lost inhibitory activities. In addition, compounds <b>31</b>–<b>34</b>, which contain a 4-amide group with a range of steric, hydrophobic, and electronic properties, were found to be inactive. Compounds <b>35</b>–<b>38</b> having a 4-aminomethyl group retained some inhibitory activities against LSD1. Compound <b>35</b> with a smaller ethylaminomethyl group is a weak inhibitor with a <i>K</i><sub>i</sub> of 22.3 μM, while compounds <b>36</b> and <b>37</b> having a bulkier piperidin-1-ylmethyl and morpholin-4-ylmethyl substituent, respectively, are almost inactive. Interestingly, compound <b>38</b> with a dibasic piperazin-1-ylmethyl group is a fairly good inhibitor with a <i>K</i><sub>i</sub> of 7.6 μM, although it is >250× less active than compound <b>17</b> having the corresponding -CN substituent. Moreover, compared to <b>17</b>, adding a -F into the 2- and 3-position of the 5-phenyl ring in compounds <b>39</b> and <b>40</b> (<i>K</i><sub>i</sub> = 220 and 150 nM), respectively, resulted in ∼8× and 5× activity reductions.</div><div class="NLM_p">Finally, several compounds shown in <a class="ref internalNav" href="#cht3" aria-label="Chart 3">Chart 3</a> were synthesized for the SAR study of the 3-(piperidin-4-ylmethoxy)pyridine core structure. Compound <b>41</b> (<i>K</i><sub>i</sub> = 4.9 μM) with a benzene core structure is ∼170-fold less potent than compound <b>17</b>, suggesting the pyridine core is of importance for the activity. Although it is a racemic mixture, the 650 nM <i>K</i><sub>i</sub> value of compound <b>42</b> shows that the piperidin-3-yl substituent is significantly less favorable than the piperidin-4-yl group in <b>17</b>. Greatly reduced activity of compound <b>43</b> (<i>K</i><sub>i</sub> = 1.2 μM) also indicates that change of an -O- linkage in compound <b>17</b> to a -NH- is highly disfavored. Loss of activity for compounds <b>44</b> (with a 3-OH group) and <b>45</b> (with a 3-benzyloxy group) clearly shows the critical role of the basic piperidine side chain in LSD1 inhibition.</div><figure id="cht3" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Chart 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-1/acs.jmedchem.5b01361/20160113/images/medium/jm-2015-013613_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-1/acs.jmedchem.5b01361/20160113/images/large/jm-2015-013613_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Chart 3. Structures and <i>K</i><sub>i</sub> Values (in Parentheses) of Compounds <b>41</b>–<b>45</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-1/acs.jmedchem.5b01361/20160113/images/large/jm-2015-013613_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01361&amp;id=cht3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Enzyme Kinetics Studies</h3><div class="NLM_p">A steady-state enzyme kinetics study of compound <b>5</b>, which is a representative compound for this novel series of LSD1 inhibitors, was performed to determine the possible mechanism of action. Since the cofactor FAD has a very high affinity to LSD1, as evidenced by its presence in the enzyme even after a lengthy expression and purification process, the kinetics study was performed against a dimethylated H3K4 peptide substrate with a sequence of ARTK(Me2)QTARKSTGGKAPRKQKA. Initial velocities of the LSD1 catalyzed reaction were measured with increasing concentrations of compound <b>5</b> and the substrate. The program SigmaPlot was used for enzyme kinetics modeling. The reaction rate and ligand concentration data were imported into the program and fitted to either competitive, uncompetitive, or noncompetitive inhibition models. The best model, as judged by <i>R</i><sup>2</sup> and AICc values, would show the mode of action for the inhibitor. Thus, compound <b>5</b> was found to be a competitive inhibitor of LSD1 against the substrate because the kinetics data for <b>5</b> were best fitted to the competitive mode of action (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>), as compared to the uncompetitive and noncompetitive fitting (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01361/suppl_file/jm5b01361_si_001.pdf" class="ext-link">Figure S3</a>).</div><figure id="fig3" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-1/acs.jmedchem.5b01361/20160113/images/medium/jm-2015-013613_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-1/acs.jmedchem.5b01361/20160113/images/large/jm-2015-013613_0007.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Lineweaver–Burk plot of competitive inhibition model for compound <b>5</b> with variable concentrations of the dimethylated H3K4 peptide substrate, with an enlarged figure in the left panel showing details near the origin. The <i>K</i><sub>i</sub> was determined to be 1.8 μM in this experiment.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-1/acs.jmedchem.5b01361/20160113/images/large/jm-2015-013613_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01361&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Docking Studies</h3><div class="NLM_p">We carried out molecular modeling (docking) studies to predict how the most potent inhibitor <b>17</b> binds to LSD1. A number of X-ray crystal structures of LSD1 in complex with covalent inhibitors,<a onclick="showRef(event, 'ref25 ref36'); return false;" href="javascript:void(0);" class="ref ref25 ref36">(25, 36)</a> a reversible inhibitor,<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> and substrate-like peptide<a onclick="showRef(event, 'ref38 ref39'); return false;" href="javascript:void(0);" class="ref ref38 ref39">(38, 39)</a> are available for the study. The docking program Glide in Schrödinger Suite molecular modeling software package was used for docking.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> The crystal structure (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2V1D">2V1D</a>) of human LSD1 in complex with a H3K4 mimetic peptide<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> (with a K4M mutation) was used as a docking template, which was prepared by adding hydrogen atoms to the structure, removing the peptide and water molecules, while keeping FAD as an integrated component of the protein. Compound <b>17</b> can be well docked into the active site of LSD1, with its 20 docking structures with the lowest energies tightly clustered to one another and having similar binding features (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>A). The pyridine core, tolyl (R<sup>6</sup>), 4-cyanophenyl (R<sup>5</sup>), and piperidin-4-ylmethoxy (R<sup>3</sup>) groups of these 20 structures are located in the same site or pocket in LSD1. <a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>B and <a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>C show two views of the binding conformation of <b>17</b> with the lowest docking energy in the active site of LSD1. Its central pyridine ring is predicted to have favorable hydrophobic as well as electrostatic interactions with the flavin ring, Tyr761, Ala809, Thr810, and Ala539. The tolyl R<sup>6</sup> group was found to be located in a mostly hydrophobic pocket surrounded by flavin, Phe538, Val333, Tyr761, Met332, Leu659, Lys661, and Trp695. The 4-cyanophenyl R<sup>5</sup> group occupies a pocket surrounded by with Ala539, Phe538, Trp695, His564, and Thr335 and has favorable interactions with these residues. The protonated amine of the piperidin-4-ylmethoxy group was found to have a hydrogen bond as well as electrostatic interactions with the negatively charged side chain of Asp555. In addition, the skeleton of the piperidin-4-ylmethoxy group possesses hydrophobic interactions with Ala809, Pro808, Ala539, and Asn540.</div><figure id="fig4" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-1/acs.jmedchem.5b01361/20160113/images/medium/jm-2015-013613_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-1/acs.jmedchem.5b01361/20160113/images/large/jm-2015-013613_0008.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Docking results of compound <b>17</b>, using an LSD1 structure (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2V1D">2V1D</a>) as the docking template: (A) 20 lowest-energy docking structures of compound <b>17</b>, with the most stable conformation shown as an orange tube model; (B) lowest-energy docking structure of <b>17</b>, showing the interacting key residues (in light cyan) of LSD1 and FAD (light cyan tube model); (C) lowest-energy docking structure of <b>17</b> (as a space-filing model) in the active site of LSD1 (shown as a meshed electrostatic potential surface model).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-1/acs.jmedchem.5b01361/20160113/images/large/jm-2015-013613_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01361&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Of interest is that the docking results could be used to rationalize our experimental SAR. For example, the predicted strong interactions between the protonated piperidin-4-ylmethoxy group and Asp555 (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>B) could explain the importance of this group for LSD1 inhibition. As comparof ied to compound <b>41</b> with a benzene core structure, possibly enhanced interactions between the pyridine core in <b>17</b> and flavin as well as Tyr761 (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>B) could account for the increased activity of <b>17</b>. For SAR on the R<sup>6</sup> group, as compared to compound <b>5</b>, the increased activity of <b>17</b> could be due to the favorable interactions of the methyl group in <b>17</b> with Phe538, Val333, Met332, and Lys661 (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>B and <a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>C). In addition, the methyl is predicted to range from ∼3.3 Å away from Met332 to ∼4.8 Å from Trp695 by the docking study. This limited space might account for somewhat decreased activities of compounds <b>19</b> and <b>20</b> with a bigger -Et and -<i>i</i>-Pr group, respectively.</div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Enzyme Selectivity</h3><div class="NLM_p">LSD1 belongs to a family of FAD-dependent monoamine oxidases including MAO-A and -B. Due to an important role of MAO-A and -B in degradation of neurotransmitters in the central nervous system, selective inhibition of LSD1 is highly desirable. We investigated enzyme selectivity of several representative 3-(piperidin-4-ylmethoxy)pyridine containing LSD1 inhibitors against MAO-A and -B. Thus, compounds <b>5</b>, <b>16</b>, <b>17</b>, and <b>22</b> were tested for their activity against recombinant human MAO-A and -B and their selectivity indices were also calculated. These results are summarized in <a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>. All four potent LSD1 inhibitors did not significantly inhibit MAO-A with <i>K</i><sub>i</sub> > 50 μM, showing excellent selectivity. For MAO-B, compounds <b>5</b>, <b>16</b>, and <b>17</b> had only weak activities with <i>K</i><sub>i</sub> values of 9.7–18.7 μM. Due to its relatively low activity against LSD1, compound <b>5</b> showed 4.6× selectivity against MAO-B. However, potent LSD1 inhibitors <b>16</b> and <b>17</b> exhibited very high selectivity of >160- and >640-fold against MAO-B, respectively. Moreover, despite having a relatively good activity against MAO-B (<i>K</i><sub>i</sub> = 2.6 μM), compound <b>22</b> is still a highly selective LSD1 inhibitor with a selectivity index of >56 against MAO-B, because of its potent inhibition of LSD1. Structurally, the 4-CF<sub>3</sub>, -Me, and -OCF<sub>3</sub> of the 6-phenyl ring in compounds <b>16</b>, <b>17</b>, and <b>22</b>, respectively, account for the high selectivity against MAO-B because as compared to the parent compound <b>5</b>, these substituents have less impact (<4×) on MAO-B inhibition but increase the inhibitory potency against LSD1 by >39-fold.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Inhibitory Activity (<i>K</i><sub>i</sub>, μM) against MAO-A and -B and Selectivity Index for LSD1</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">LSD1</th><th class="colsep0 rowsep0" align="center">MAO-A</th><th class="colsep0 rowsep0" align="center">MAO-B</th><th class="colsep0 rowsep0" align="center">selectivity index</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5</b></td><td class="colsep0 rowsep0" align="left">2.3</td><td class="colsep0 rowsep0" align="left">>50</td><td class="colsep0 rowsep0" align="left">10.6 ± 2.8</td><td class="colsep0 rowsep0" align="left">>4.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>16</b></td><td class="colsep0 rowsep0" align="left">0.058</td><td class="colsep0 rowsep0" align="left">>50</td><td class="colsep0 rowsep0" align="left">9.7 ± 2.9</td><td class="colsep0 rowsep0" align="left">>160</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17</b></td><td class="colsep0 rowsep0" align="left">0.029</td><td class="colsep0 rowsep0" align="left">>50</td><td class="colsep0 rowsep0" align="left">18.7 ± 6.9</td><td class="colsep0 rowsep0" align="left">>640</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>22</b></td><td class="colsep0 rowsep0" align="left">0.046</td><td class="colsep0 rowsep0" align="left">>50</td><td class="colsep0 rowsep0" align="left">2.6 ± 1.2</td><td class="colsep0 rowsep0" align="left">>56</td></tr></tbody></table></div></div><div class="NLM_p last">Structurally similar 5,6-diaryl-3-(aminoalkyloxy)pyridine compounds were reported to be potent inhibitors of Akt kinase,<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> which is also a drug target for cancer. We examined the enzyme selectivity of the most potent compound <b>17</b> against human Akt. Compound <b>17</b> was found to have a weak activity against Akt with an IC<sub>50</sub> value of 87.6 ± 21.6 μM, showing a high enzyme selectivity for LSD1. In addition, these compounds were reported as nicotinic receptor ligands<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> as well as antituberculosis agents.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> Although these do not directly relate to the antitumor purpose of this study, a comprehensive enzyme selectivity profiling could be useful for future development of these compounds.</div></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Cell Activity Testing</h3><div class="NLM_p">The most potent compound <b>17</b> was tested to see whether it can change the cellular H3K4 methylation level in a leukemia cell line MV4-11. Thus, cells were treated with <b>17</b> at 0.2 and 1 μM and histones were then collected, separated by electrophoresis, and probed by a specific antibody of H3K4me2, as well as that of histone H3 as input controls. As can be seen in <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>, compound <b>17</b> appeared to increase the cellular level of H3K4me2 at 1 μM, suggesting it is cell membrane permeable and inhibits LSD1 in cells. It is noted because this method is semiquantitative. More quantitative investigation could be useful to characterize how compound <b>17</b> affects cellular histone methylations globally and at specific gene loci.</div><figure id="fig5" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-1/acs.jmedchem.5b01361/20160113/images/medium/jm-2015-013613_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-1/acs.jmedchem.5b01361/20160113/images/large/jm-2015-013613_0009.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Treatment of MV4-11 cells with potent LSD1 inhibitor <b>17</b> caused an increased cellular level of H3K4me2.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-1/acs.jmedchem.5b01361/20160113/images/large/jm-2015-013613_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01361&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Next, selected potent LSD1 inhibitors, including compound <b>S2</b> (which is a cyclopropylamine-containing inhibitor, <a href="/doi/suppl/10.1021/acs.jmedchem.5b01361/suppl_file/jm5b01361_si_001.pdf" class="ext-link">Figure S2</a>) as a positive control, were tested for their ability to inhibit proliferation of four tumor cell lines and a normal fibroblast cell line WI-38. Among the four tumor cells, MV4-11 and Molm-13 are leukemia cell lines with an MLL gene translocation. Two representative breast cancer cells MCF-7 (estrogen receptor positive) and MDA-MB231 (estrogen receptor negative) were also included in the study, since LSD1 is often overexpressed in breast cancers. The results are summarized in <a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>, together with the LSD1 inhibitory activity of these compounds. Of interest is that the five competitive inhibitors exhibited strong antitumor activities against the four tumor cells with EC<sub>50</sub> values of 280 nM to 8.6 μM. However, these compounds had weak or no activity against the growth of normal fibroblast cells WI-38. Particularly, LSD1 inhibitors <b>16</b>, <b>17</b>, <b>20</b>, and <b>22</b> showed potent activity against proliferation of MV4-11 leukemia cells with EC<sub>50</sub> values ranging from 280 nM to 480 nM, while less potent compound <b>13</b> exhibited a reduced activity (EC<sub>50</sub> = 1.1 μM). These compounds were also found to have strong activity against the growth of Molm-13 leukemia cells with EC<sub>50</sub> values of 0.84–4.4 μM. For MCF-7 breast cancer, more potent LSD1 inhibitors <b>16</b>, <b>17</b>, <b>20</b>, and <b>22</b> appeared to be superior (EC<sub>50</sub> of 3.3–3.6 μM) to compound <b>13</b> (EC<sub>50</sub> of 6.9 μM). These compounds also showed fairly good antiproliferative activity against MDA-MB231 breast cancer cells with EC<sub>50</sub> values of 5.6–8.6 μM. In addition, these five competitive LSD1 inhibitors showed similar antitumor activities as compared to the covalent inhibitor <b>S2</b>, which also exhibited strong antiproliferative activity (EC<sub>50</sub> of 0.084–8.9 μM) against the tumor cells, while it was inactive against the normal cells. Overall, the strong and selective antitumor activity suggested that this class of LSD1 inhibitors could be potentially useful cancer therapeutics and further efforts of compound development are therefore warranted.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Antiproliferative Activity (EC<sub>50</sub>, μM) of LSD1 Inhibitors against Cancer and Normal Cells</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">LSD1 <i>K</i><sub>i</sub> (μM)</th><th class="colsep0 rowsep0" align="center">MV4-11</th><th class="colsep0 rowsep0" align="center">Molm-13</th><th class="colsep0 rowsep0" align="center">MCF-7</th><th class="colsep0 rowsep0" align="center">MDA-MB231</th><th class="colsep0 rowsep0" align="center">WI-38</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>13</b></td><td class="colsep0 rowsep0" align="left">0.22</td><td class="colsep0 rowsep0" align="left">1.1 ± 0.12</td><td class="colsep0 rowsep0" align="left">4.4 ± 1.1</td><td class="colsep0 rowsep0" align="left">6.9 ± 1.8</td><td class="colsep0 rowsep0" align="left">8.6 ± 2.4</td><td class="colsep0 rowsep0" align="left">>50</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>16</b></td><td class="colsep0 rowsep0" align="left">0.058</td><td class="colsep0 rowsep0" align="left">0.28 ± 0.03</td><td class="colsep0 rowsep0" align="left">0.84 ± 0.18</td><td class="colsep0 rowsep0" align="left">3.6 ± 1.5</td><td class="colsep0 rowsep0" align="left">6.9 ± 1.9</td><td class="colsep0 rowsep0" align="left">30.0 ± 9.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17</b></td><td class="colsep0 rowsep0" align="left">0.029</td><td class="colsep0 rowsep0" align="left">0.36 ± 0.13</td><td class="colsep0 rowsep0" align="left">3.4 ± 1.2</td><td class="colsep0 rowsep0" align="left">3.6 ± 1.5</td><td class="colsep0 rowsep0" align="left">5.6 ± 1.0</td><td class="colsep0 rowsep0" align="left">26.6 ± 6.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20</b></td><td class="colsep0 rowsep0" align="left">0.088</td><td class="colsep0 rowsep0" align="left">0.30 ± 0.02</td><td class="colsep0 rowsep0" align="left">3.2 ± 1.1</td><td class="colsep0 rowsep0" align="left">3.3 ± 0.8</td><td class="colsep0 rowsep0" align="left">7.6 ± 2.9</td><td class="colsep0 rowsep0" align="left">20.0 ± 4.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>22</b></td><td class="colsep0 rowsep0" align="left">0.046</td><td class="colsep0 rowsep0" align="left">0.48 ± 0.12</td><td class="colsep0 rowsep0" align="left">3.6 ± 0.5</td><td class="colsep0 rowsep0" align="left">5.3 ± 2.2</td><td class="colsep0 rowsep0" align="left">7.5 ± 2.2</td><td class="colsep0 rowsep0" align="left">17.5 ± 4.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>S2</b></td><td class="colsep0 rowsep0" align="left">0.077<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">0.084 ± 0.026</td><td class="colsep0 rowsep0" align="left">0.32 ± 0.1</td><td class="colsep0 rowsep0" align="left">6.6 ± 2.0</td><td class="colsep0 rowsep0" align="left">8.9 ± 2.5</td><td class="colsep0 rowsep0" align="left">>50</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup>a</sup><p class="last">IC<sub>50</sub> value for compound <b>S2</b>, which is a covalent inhibitor of LSD1.</p></div></div></div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i18">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_25138" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_25138" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">LSD1 plays an important role in maintaining a balanced methylated H3K4, a key “histone code” for active gene transcription. It has also been found to be overexpressed in certain cancers such as breast cancer.<a onclick="showRef(event, 'ref28 ref29 ref30'); return false;" href="javascript:void(0);" class="ref ref28 ref29 ref30">(28-30)</a> Therefore, LSD1 represents a drug target for intervention and more chemotypes of potent LSD1 inhibitors are needed.</div><div class="NLM_p">On the basis of previous SAR of cyclopropylamine containing LSD1 inhibitors, a series of 3-(piperidin-4-ylmethoxy)pyridine and related compounds were found to be new LSD1 inhibitors. Structure guided molecular modeling and medicinal chemistry studies have led to the identification of several potent LSD1 inhibitors (e.g., <b>16</b>, <b>17</b>, <b>20</b>, and <b>22</b>) with <i>K</i><sub>i</sub> values as low as 29 nM. SAR investigation of this series of compounds showed that (1) a pyridine or pyrazine core structure is superior to a benzene ring; (2) a basic 3-(piperidin-4-ylmethoxy) R<sup>3</sup> group is important to LSD1 inhibition; (3) 4-cyanophenyl group is the best R<sup>5</sup> substituent in this study, while many other groups such as amide or alkyl-containing phenyl groups significantly reduce the inhibitory activity; (4) a phenyl R<sup>6</sup> group is superior to a corresponding pyridine group; and (5) among the substituted phenyl R<sup>6</sup> groups, several para-substituents (e.g., -Me, CF<sub>3</sub>, and -OCF<sub>3</sub> in <b>17</b>, <b>16</b>, and <b>22</b>) can significantly enhance LSD1 inhibition (by as high as 79×), while meta- or ortho-substituents are less effective.</div><div class="NLM_p last">Enzyme kinetics studies suggested that these compounds are competitive inhibitors against the dimethylated H3K4 peptide substrate. Consistent with the kinetics data, docking studies showed that these compounds can be well docked into the substrate binding site of LSD1. In addition, the docking results could be used to rationalize many of the observed SARs. Several representative LSD1 inhibitors were found to exhibit very high selectivity against related MAO-A and -B enzymes. This feature is desirable for this series of LSD1 inhibitors, due to important functions of MAO-A and -B in the central nervous system. The most potent LSD1 inhibitor <b>17</b> was found to increase the cellular level of H3K4me2, showing on-target activity in cells. However, it did not significantly affect the methylation levels of several other histone lysine residues (including H3K4me3), presumably because these are not cellular substrates of LSD1. Finally, five potent LSD1 inhibitors were found to exhibit potent to good activity against proliferation of two MLL-rearranged leukemia cells and two breast cancer cells with EC<sub>50</sub> values ranging from 0.28 to 8.6 μM, while no or very weak activity of these compounds against the growth of normal fibroblast cells was observed.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i19">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_64650" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_64650" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">All chemicals for synthesis were purchased from Alfa Aesar (Ward Hill, MA) or Aldrich (Milwaukee, WI). All compounds were characterized by <sup>1</sup>H NMR on a Varian (Palo Alto, CA) 400-MR spectrometer. Identification of the most active compounds was confirmed by HRMS using an Agilent 6550 iFunnel quadrupole-time-of-flight (Q-TOF) mass spectrometer with electrospray ionization (ESI). The purities of synthesized compounds were determined by a Shimadzu Prominence HPLC with a Zorbax C18 column (4.6 mm × 250 mm) monitored by UV at 254 nm. The purities of the reported compounds were found to be >95%. HPLC–MS experiments were performed using a Shimadzu Prominence HPLC connected to a LCMS-2020 MS system.</div><div class="NLM_p"><b>Synthesis and characterization of compounds 4</b>–<b>45</b> can be found in <a href="/doi/suppl/10.1021/acs.jmedchem.5b01361/suppl_file/jm5b01361_si_001.pdf" class="ext-link">Supporting Information Experimental Section</a>.</div><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> LSD1 Inhibition</h3><div class="NLM_p">The catalytic domain (172–833) of human LSD1 was cloned, inserted into pGEX-KG vector, and the DNA sequence was verified by sequencing. The pGEX-KG-LSD1 plasmid thus generated was transformed into <i>E. coli</i> BL21-CodonPlus strain (Agilent) and cultured at 37 °C in LB medium containing ampicillin (50 μg/mL) and chloramphenicol (34 μg/mL). When the optical density of the bacterial culture approached ∼0.9 at 600 nm, LSD1 expression was induced by adding 0.2 mM isopropylthiogalactoside (IPTG) at 25 °C for 20 h. Cells were next collected, lysed, centrifuged for 20 min at 20 000 rpm. The resulting supernatant was subjected to an affinity column chromatography with glutathione sepharose resins. The recombinant GST-LSD1 fusion protein was obtained in ∼90% purity (SDS–PAGE) by elution with 10 mM glutathione solution and a secondary purification using a Superdex 200 gel filtration column chromatography followed by concentration.</div><div class="NLM_p">A mass spectrometry based biochemical assay for LSD1 was developed, with the amount of demethylated product peptide being quantitatively determined by HPLC–MS. Increasing concentrations of a compound were incubated with LSD1 (150 nM) in 50 mM phosphate buffer (pH = 7.0) containing 0.01% Brij-35 for 10 min at 25 °C, before initiation of the reaction by adding 10 μM of dimethylated peptide substrate ARTK(Me2)QTARKSTGGKAPRKQKA. The total volume of the reaction mixture was 60 μL. Reactions were terminated after 30 min by adding 6 N formic acid (5 μL), and 20 μL of the reaction mixture was subjected to HPLC–MS to separate and determine the amount of the reaction product. HPLC was run using a Phenomenex C18 column (250 mm × 4.6 mm, 5 μm) with acetonitrile/water (40:60, containing 0.1% TFA) as an eluent at a flow rate of 0.5 mL/min. A selected ion monitoring (SIM) for 1142 Da was used to detect and quantitate the amount of the product ARTK(Me1)QTARKSTGGKAPRKQKA (parameters: interface voltage, 3.0 kV; detector voltage, 1.3 kV; nebulizing gas, 1.5 L/min; drying gas, 15 L/min; desolvation line temperature, 300 °C; heat block temperature, 300 °C; Pirani gauge vacuum, 150 Pa; ion gauge vacuum: 5 × 10<sup>–4</sup> Pa). To ensure an initial velocity was determined, ∼5% of the dimethylated substrate was consumed before the reaction was stopped. Therefore, monomethylated peptide was obtained almost as the only product. No significant amount of the nonmethylated product was detected.</div><div class="NLM_p last">Data were imported into Prism 5.0 (GraphPad), and the IC<sub>50</sub> values were determined using the sigmoidal dose response fitting in the program. For compounds with IC<sub>50</sub> > 1.5 μM (i.e., ≫[LSD1]), <i>K</i><sub>i</sub> values were calculated using the Cheng–Prusoff equation <i>K</i><sub>i</sub> = IC<sub>50</sub>/(1 + [S]/<i>K</i><sub>m</sub>), where [S] is the concentration of the peptide substrate (10 μM) and <i>K</i><sub>m</sub> is a reported value of 10 μM.<a onclick="showRef(event, 'cit27a'); return false;" href="javascript:void(0);" class="ref cit27a">(27a)</a> For compounds with IC<sub>50</sub> < 1.5 μM, <i>K</i><sub>i</sub> values were calculated using the Morrison tight inhibition modeling in Prism. The reported <i>K</i><sub>i</sub> values were the mean values of at least three independent experiments.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Enzyme Kinetics Study</h3><div class="NLM_p last">A steady-state kinetics study was conducted by measuring the initial velocities of reactions catalyzed by LSD1 while varying the concentrations of compound <b>5</b> (0, 2 μM, 4 μM, and 6 μM) and the peptide substrate (2 μM, 3 μM, 5 μM, 10 μM, 50 μM, and 100 μM). Data were imported into SigmaPlot and fitted into the competitive, noncompetitive, and uncompetitive inhibition models. The best kinetic model was determined by the highest <i>R</i><sup>2</sup> and lowest AICc values. Lineweaver–Burk plots were generated by SigmaPlot.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Molecular Modeling</h3><div class="NLM_p last">Docking studies were performed with our previous published methods<a onclick="showRef(event, 'ref44 ref45 ref46'); return false;" href="javascript:void(0);" class="ref ref44 ref45 ref46">(44-46)</a> using Schrödinger suite (version 2015),<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> which includes all of the programs described below. The crystal structure of LSD1 in complex with an H3K4 peptide (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2V1D">2V1D</a>) was prepared using the module “Protein Preparation Wizard” in Maestro (version 10.1) using the default protein parameters. Hydrogen atoms were added, the H3K4 peptide ligand and all water molecules were extracted, and FAD was retained in the protein structure for docking. H-bonds were next optimized, the partial charges for all atoms were assigned, and the protein was energy-minimized using OPLS-2005 force field. A receptor grid, which is large enough to contain the active site, was generated with the program Glide without any constraints. Inhibitor compounds were built, energy-minimized using OPLS-2005 force field in Maestro, and then docked into the prepared protein structure using Glide (docking parameters: standard-precision and dock flexibly).</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Inhibition of MAO-A/-B</h3><div class="NLM_p last">Human recombinant MAO-A and -B enzymes were purchased from Sigma, and their biochemical activity and inhibition were determined using a MAO-Glo assay kit from Promega. Following the manufacturer’s protocol, the assays were performed in 384-well white plates by adding all reagents in the kit, MAO-A or -B (100 nM) and increasing concentrations of inhibitors to a final volume of 20 μL. After 60 min, the reactions were stopped by adding a reconstituted luciferin detection reagent (20 μL/well). Upon incubation for an additional 20 min, luminescence of each well was determined using a Beckman DTX-880 microplate reader. Using <i>K</i><sub>m</sub> values provided in the kit, <i>K</i><sub>i</sub> values were similarly calculated as described above.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> Inhibition of Akt Kinase</h3><div class="NLM_p last">The inhibitory activity of compound <b>17</b> against human recombinant Akt kinase was determined using a commercially available ELISA kit (Enzo life Sciences, Farmingdale, NY). Following the manufacturer’s protocol, Akt kinase (5 nM), ATP, and increasing concentrations of <b>17</b> (0.01–100 μM) were added to a final volume of 30 μL in a 96-well plate precoated with a substrate. After a 60 min incubation in 30 °C, the reactions were stopped by removal of all reactants. The primary antibody against the phosphorylated substrate and a peroxidase conjugated secondary antibody were added sequentially. The optical absorbance at 450 nM of each well was developed with tetramethylbenzidine substrate (TMB) and determined using a Beckman DTX-880 microplate reader. The data were imported into Prism (version 5.0, GraphPad) and IC<sub>50</sub> values were calculated by using a standard dose response curve fitting. The reported value is the average of three independent experiments.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> Western Blot</h3><div class="NLM_p last">10<sup>6</sup> MV4-11 cells/well were incubated with compounds <b>17</b> and <b>28</b> at 0.2 and 1 μM for 3 days. Histone proteins were extracted using EpiQuik total histone extraction kit (Epigentek) according to the manufacturer’s protocol. Equal amounts of histones (2 μg) were separated on SDS–PAGE and transferred to PVDF membranes. The blots were probed with primary antibodies against H3K4me2 and -me3, H3K27me2, H3K36me2, H4K20me2, and H3 (Cell Signaling), followed by anti-rabbit IgG (Thermo Scientific) secondary antibodies. Quantitation of these blots were performed using the program ImageJ.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> Cell Growth Inhibition</h3><div class="NLM_p last">The antiproliferation assays were done using our previous method.<a onclick="showRef(event, 'ref47 ref48'); return false;" href="javascript:void(0);" class="ref ref47 ref48">(47, 48)</a> In brief, for leukemia cells, 10<sup>6</sup> cells per well were added into 96-well plates and cultured with increasing concentrations of a compound in RPMI-1640 medium supplemented with 10% fetal bovine serum and penicillin (100 U/mL) and streptomycin (100 μg/mL) at 37 °C in a 5% CO<sub>2</sub> atmosphere with 100% humidity. For solid tumor and normal cells, 10<sup>5</sup> cells per well were added into 96-well plates and cultured in Dulbecco’s modified Eagle medium (DMEM) supplemented with 10% fetal bovine serum and penicillin (100 U/mL) and streptomycin (100 μg/mL) overnight for cell attachment. After addition of increasing concentration of a compound, plates were incubated for a given period of time. Cell viability was assessed by using an XTT assay kit (Roche) for leukemia cells or an MTT assay (Sigma) for attachment cells. Compound IC<sub>50</sub> values were calculated from dose response curves using Prism 5.0.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-2"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i27"><a href="/doi/suppl/10.1021/acs.jmedchem.5b01361">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_65851" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_65851" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.5b01361" class="ext-link">10.1021/acs.jmedchem.5b01361</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Figures S1–S3 and Experimental Section showing detailed compound synthesis and characterization (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01361/suppl_file/jm5b01361_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01361/suppl_file/jm5b01361_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b01361/suppl_file/jm5b01361_si_001.pdf">jm5b01361_si_001.pdf (517.55 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b01361/suppl_file/jm5b01361_si_002.csv">jm5b01361_si_002.csv (2.98 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.5b01361" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_85454" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_85454" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yongcheng Song</span> - <span class="hlFld-Affiliation affiliation">†Department of Pharmacology, and ‡Dan L. Duncan Cancer Center, Baylor College of Medicine, 1 Baylor Plaza, Houston, Texas 77030, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#b6cfc5d9d8d1f6d4d5db98d3d2c3"><span class="__cf_email__" data-cfemail="b9c0cad6d7def9dbdad497dcddcc">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Fangrui Wu</span> - <span class="hlFld-Affiliation affiliation">†Department of Pharmacology, and ‡Dan L. Duncan Cancer Center, Baylor College of Medicine, 1 Baylor Plaza, Houston, Texas 77030, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chao Zhou</span> - <span class="hlFld-Affiliation affiliation">†Department of Pharmacology, and ‡Dan L. Duncan Cancer Center, Baylor College of Medicine, 1 Baylor Plaza, Houston, Texas 77030, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yuan Yao</span> - <span class="hlFld-Affiliation affiliation">†Department of Pharmacology, and ‡Dan L. Duncan Cancer Center, Baylor College of Medicine, 1 Baylor Plaza, Houston, Texas 77030, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Liping Wei</span> - <span class="hlFld-Affiliation affiliation">†Department of Pharmacology, and ‡Dan L. Duncan Cancer Center, Baylor College of Medicine, 1 Baylor Plaza, Houston, Texas 77030, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Zizhen Feng</span> - <span class="hlFld-Affiliation affiliation">†Department of Pharmacology, and ‡Dan L. Duncan Cancer Center, Baylor College of Medicine, 1 Baylor Plaza, Houston, Texas 77030, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lisheng Deng</span> - <span class="hlFld-Affiliation affiliation">†Department of Pharmacology, and ‡Dan L. Duncan Cancer Center, Baylor College of Medicine, 1 Baylor Plaza, Houston, Texas 77030, United States</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>F.W. and C.Z. contributed equally.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d2188e1979-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i29">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_41433" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_41433" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was supported by Grant RP150129 from Cancer Prevention and Research Institute of Texas (CPRIT) and Grant R01NS080963 from National Institute of Neurological Disorders and Stroke (NINDS/NIH) to Y.S. We also thank Dr. Feng Li of BCM Metabolomics Core for performing HRMS.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i30" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i30"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i31" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i31"> Abbrevations Used</h2><tr><td class="NLM_term">BOC</td><td class="NLM_def"><p class="first last"><i>tert</i>-butyloxycarbonyl</p></td></tr><tr><td class="NLM_term">FAD</td><td class="NLM_def"><p class="first last">flavin adenine dinucleotide</p></td></tr><tr><td class="NLM_term">H3K4</td><td class="NLM_def"><p class="first last">histone H3 lysine 4</p></td></tr><tr><td class="NLM_term">H3K4me1, -2, or -3</td><td class="NLM_def"><p class="first last">mono-, di-, or trimethylated H3K4</p></td></tr><tr><td class="NLM_term">LSD1</td><td class="NLM_def"><p class="first last">lysine specific demethylase 1</p></td></tr><tr><td class="NLM_term">MAO</td><td class="NLM_def"><p class="first last">monoamine oxidase</p></td></tr><tr><td class="NLM_term">MLL</td><td class="NLM_def"><p class="first last">mixed lineage leukemia</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i32">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_41002" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_41002" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 48 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><div class="note"><p class="first last">For a recent review, see the following:</p></div><span class="NLM_contrib-group">Kouzarides, T.</span><span> </span><span class="NLM_article-title">Chromatin modifications and their function</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">128</span><span class="NLM_x">, </span> <span class="NLM_fpage">693</span><span class="NLM_x">–</span> <span class="NLM_lpage">705</span><span class="refDoi"> DOI: 10.1016/j.cell.2007.02.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1016%2Fj.cell.2007.02.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=17320507" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BD2sXis12ju7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2007&pages=693-705&author=T.+Kouzarides&title=Chromatin+modifications+and+their+function&doi=10.1016%2Fj.cell.2007.02.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Chromatin modifications and their function</span></div><div class="casAuthors">Kouzarides, Tony</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">693-705</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The surface of nucleosomes is studded with a multiplicity of modifications.  At least eight different classes have been characterized to date and many different sites have been identified for each class.  Operationally, modifications function either by disrupting chromatin contacts or by affecting the recruitment of nonhistone proteins to chromatin.  Their presence on histones can dictate the higher-order chromatin structure in which DNA is packaged and can orchestrate the ordered recruitment of enzyme complexes to manipulate DNA.  In this way, histone modifications have the potential to influence many fundamental biol. processes, some of which may be epigenetically inherited.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAH9AX2dGicbVg90H21EOLACvtfcHk0lh93yQJbZhGSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXis12ju7Y%253D&md5=c5995fe68fd9473f4268d59c01d3b9de</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2007.02.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2007.02.005%26sid%3Dliteratum%253Aachs%26aulast%3DKouzarides%26aufirst%3DT.%26atitle%3DChromatin%2520modifications%2520and%2520their%2520function%26jtitle%3DCell%26date%3D2007%26volume%3D128%26spage%3D693%26epage%3D705%26doi%3D10.1016%2Fj.cell.2007.02.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Jones, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baylin, S. B.</span><span> </span><span class="NLM_article-title">The epigenomics of cancer</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">128</span><span class="NLM_x">, </span> <span class="NLM_fpage">683</span><span class="NLM_x">–</span> <span class="NLM_lpage">692</span><span class="refDoi"> DOI: 10.1016/j.cell.2007.01.029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1016%2Fj.cell.2007.01.029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=17320506" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BD2sXis12ju7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2007&pages=683-692&author=P.+A.+Jonesauthor=S.+B.+Baylin&title=The+epigenomics+of+cancer&doi=10.1016%2Fj.cell.2007.01.029"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">The epigenomics of cancer</span></div><div class="casAuthors">Jones, Peter A.; Baylin, Stephen B.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">683-692</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Aberrant gene function and altered patterns of gene expression are key features of cancer.  Growing evidence shows that acquired epigenetic abnormalities participate with genetic alterations to cause this dysregulation.  Here, we review recent advances in understanding how epigenetic alterations participate in the earliest stages of neoplasia, including stem/precursor cell contributions, and discuss the growing implications of these advances for strategies to control cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPBpHNAkcSPbVg90H21EOLACvtfcHk0lh93yQJbZhGSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXis12ju7k%253D&md5=78eac8ecc093f1aed0acbfd578f34717</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2007.01.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2007.01.029%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DP.%2BA.%26aulast%3DBaylin%26aufirst%3DS.%2BB.%26atitle%3DThe%2520epigenomics%2520of%2520cancer%26jtitle%3DCell%26date%3D2007%26volume%3D128%26spage%3D683%26epage%3D692%26doi%3D10.1016%2Fj.cell.2007.01.029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Wilson, C. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rowell, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sekimata, M.</span><span> </span><span class="NLM_article-title">Epigenetic control of T-helper-cell differentiation</span> <span class="citation_source-journal">Nat. Rev. Immunol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">91</span><span class="NLM_x">–</span> <span class="NLM_lpage">105</span><span class="refDoi"> DOI: 10.1038/nri2487</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1038%2Fnri2487" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=19151746" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BD1MXntFGnug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2009&pages=91-105&author=C.+B.+Wilsonauthor=E.+Rowellauthor=M.+Sekimata&title=Epigenetic+control+of+T-helper-cell+differentiation&doi=10.1038%2Fnri2487"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetic control of T-helper-cell differentiation</span></div><div class="casAuthors">Wilson, Christopher B.; Rowell, Emily; Sekimata, Masayuki</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Immunology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">91-105</span>CODEN:
                <span class="NLM_cas:coden">NRIABX</span>;
        ISSN:<span class="NLM_cas:issn">1474-1733</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Naive CD4+ T cells give rise to T-helper-cell subsets with functions that are tailored to their resp. roles in host defense.  The specification of T-helper-cell subsets is controlled by networks of lineage-specifying transcription factors, which bind to regulatory elements in genes that encode cytokines and other transcription factors.  The nuclear context in which these transcription factors act is affected by epigenetic processes, which allow programs of gene expression to be inherited by progeny cells that at the same time retain the potential for change in response to altered environmental signals.  In this Review, the authors describe these epigenetic processes and discuss how they collaborate to govern the fate and function of T helper cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZxqgRtS65ibVg90H21EOLACvtfcHk0ljk1fx69RXr2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXntFGnug%253D%253D&md5=14cb4ee035389a460489f7e530d0e41b</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnri2487&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnri2487%26sid%3Dliteratum%253Aachs%26aulast%3DWilson%26aufirst%3DC.%2BB.%26aulast%3DRowell%26aufirst%3DE.%26aulast%3DSekimata%26aufirst%3DM.%26atitle%3DEpigenetic%2520control%2520of%2520T-helper-cell%2520differentiation%26jtitle%3DNat.%2520Rev.%2520Immunol.%26date%3D2009%26volume%3D9%26spage%3D91%26epage%3D105%26doi%3D10.1038%2Fnri2487" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Tsankova, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Renthal, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kumar, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nestler, E. J.</span><span> </span><span class="NLM_article-title">Epigenetic regulation in psychiatric disorders</span> <span class="citation_source-journal">Nat. Rev. Neurosci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">355</span><span class="NLM_x">–</span> <span class="NLM_lpage">367</span><span class="refDoi"> DOI: 10.1038/nrn2132</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1038%2Fnrn2132" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=17453016" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BD2sXksFSis7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2007&pages=355-367&author=N.+Tsankovaauthor=W.+Renthalauthor=A.+Kumarauthor=E.+J.+Nestler&title=Epigenetic+regulation+in+psychiatric+disorders&doi=10.1038%2Fnrn2132"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetic regulation in psychiatric disorders</span></div><div class="casAuthors">Tsankova, Nadia; Renthal, William; Kumar, Arvind; Nestler, Eric J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Neuroscience</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">355-367</span>CODEN:
                <span class="NLM_cas:coden">NRNAAN</span>;
        ISSN:<span class="NLM_cas:issn">1471-003X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Many neurol. and most psychiatric disorders are not due to mutations in a single gene; rather, they involve mol. disturbances entailing multiple genes and signals that control their expression.  Recent research has demonstrated that complex 'epigenetic' mechanisms, which regulate gene activity without altering the DNA code, have long-lasting effects within mature neurons.  This review summarizes recent evidence for the existence of sustained epigenetic mechanisms of gene regulation in neurons that have been implicated in the regulation of complex behavior, including abnormalities in several psychiatric disorders such as depression, drug addiction and schizophrenia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQG0gqjbj_Z7Vg90H21EOLACvtfcHk0ljk1fx69RXr2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXksFSis7g%253D&md5=6af012ebb4b554beafcb4771e3e913ba</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnrn2132&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrn2132%26sid%3Dliteratum%253Aachs%26aulast%3DTsankova%26aufirst%3DN.%26aulast%3DRenthal%26aufirst%3DW.%26aulast%3DKumar%26aufirst%3DA.%26aulast%3DNestler%26aufirst%3DE.%2BJ.%26atitle%3DEpigenetic%2520regulation%2520in%2520psychiatric%2520disorders%26jtitle%3DNat.%2520Rev.%2520Neurosci.%26date%3D2007%26volume%3D8%26spage%3D355%26epage%3D367%26doi%3D10.1038%2Fnrn2132" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Solomon, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richon, V. M.</span><span> </span><span class="NLM_article-title">Protein methyltransferases as a target class for drug discovery</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">724</span><span class="NLM_x">–</span> <span class="NLM_lpage">732</span><span class="refDoi"> DOI: 10.1038/nrd2974</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1038%2Fnrd2974" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=19721445" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVKmsrvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2009&pages=724-732&author=R.+A.+Copelandauthor=M.+E.+Solomonauthor=V.+M.+Richon&title=Protein+methyltransferases+as+a+target+class+for+drug+discovery&doi=10.1038%2Fnrd2974"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Protein methyltransferases as a target class for drug discovery</span></div><div class="casAuthors">Copeland, Robert A.; Solomon, Michael E.; Richon, Victoria M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">724-732</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The protein methyltransferases (PMTs) - which methylate protein lysine and arginine residues and have crucial roles in gene transcription - are emerging as an important group of enzymes that play key parts in normal physiol. and human diseases.  The collection of human PMTs is a large and diverse group of enzymes that have a common mechanism of catalysis.  Here, we review the biol., biochem. and structural data that together present PMTs as a novel, chem. tractable target class for drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpySXIqKk9Lw7Vg90H21EOLACvtfcHk0ljk1fx69RXr2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVKmsrvI&md5=0626e14396442b47c7da2aaa6a0ac6e3</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnrd2974&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2974%26sid%3Dliteratum%253Aachs%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DSolomon%26aufirst%3DM.%2BE.%26aulast%3DRichon%26aufirst%3DV.%2BM.%26atitle%3DProtein%2520methyltransferases%2520as%2520a%2520target%2520class%2520for%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2009%26volume%3D8%26spage%3D724%26epage%3D732%26doi%3D10.1038%2Fnrd2974" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Cole, P. A.</span><span> </span><span class="NLM_article-title">Chemical probes for histone-modifying enzymes</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">590</span><span class="NLM_x">–</span> <span class="NLM_lpage">597</span><span class="refDoi"> DOI: 10.1038/nchembio.111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1038%2Fnchembio.111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=18800048" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtFamtbrL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2008&pages=590-597&author=P.+A.+Cole&title=Chemical+probes+for+histone-modifying+enzymes&doi=10.1038%2Fnchembio.111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical probes for histone-modifying enzymes</span></div><div class="casAuthors">Cole, Philip A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">590-597</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The histone-modifying enzymes that catalyze reversible lysine acetylation and methylation are central to the epigenetic regulation of chromatin remodeling.  From the early discovery of histone deacetylase inhibitors to the more recent identification of histone demethylase blockers, chem. approaches offer increasingly sophisticated tools for the investigation of the structure and function of these lysine-modifying enzymes.  This review summarizes progress to date on compds. identified from screens or by design that can modulate the activity of classical histone deacetylases, sirtuins, histone acetyltransferases, histone methyltransferases and histone demethylases.  We highlight applications of compds. to mechanistic and functional studies involving these enzymes and discuss future challenges regarding target specificity and general utility.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJZ5RKIrQsO7Vg90H21EOLACvtfcHk0ljk1fx69RXr2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtFamtbrL&md5=71fe323c232af42b67a7dfc134c4d49b</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.111%26sid%3Dliteratum%253Aachs%26aulast%3DCole%26aufirst%3DP.%2BA.%26atitle%3DChemical%2520probes%2520for%2520histone-modifying%2520enzymes%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2008%26volume%3D4%26spage%3D590%26epage%3D597%26doi%3D10.1038%2Fnchembio.111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Falkenberg, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnstone, R. W.</span><span> </span><span class="NLM_article-title">Histone deacetylases and their inhibitors in cancer, neurological diseases and immune disorders</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">673</span><span class="NLM_x">–</span> <span class="NLM_lpage">691</span><span class="refDoi"> DOI: 10.1038/nrd4360</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1038%2Fnrd4360" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=25131830" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlKrtLjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=673-691&author=K.+J.+Falkenbergauthor=R.+W.+Johnstone&title=Histone+deacetylases+and+their+inhibitors+in+cancer%2C+neurological+diseases+and+immune+disorders&doi=10.1038%2Fnrd4360"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Histone deacetylases and their inhibitors in cancer, neurological diseases and immune disorders</span></div><div class="casAuthors">Falkenberg, Katrina J.; Johnstone, Ricky W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">673-691</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Epigenetic aberrations, which are recognized as key drivers of several human diseases, are often caused by genetic defects that result in functional deregulation of epigenetic proteins, their altered expression and/or their atypical recruitment to certain gene promoters.  Importantly, epigenetic changes are reversible, and epigenetic enzymes and regulatory proteins can be targeted using small mols.  This Review discusses the role of altered expression and/or function of one class of epigenetic regulators - histone deacetylases (HDACs) - and their role in cancer, neurol. diseases and immune disorders.  We highlight the development of small-mol. HDAC inhibitors and their use in the lab., in preclin. models and in the clinic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXiWEG49LhebVg90H21EOLACvtfcHk0li1JX8OoolH2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlKrtLjI&md5=ccfb68fa3fd4685e2305cdf8a6a9c81a</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnrd4360&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4360%26sid%3Dliteratum%253Aachs%26aulast%3DFalkenberg%26aufirst%3DK.%2BJ.%26aulast%3DJohnstone%26aufirst%3DR.%2BW.%26atitle%3DHistone%2520deacetylases%2520and%2520their%2520inhibitors%2520in%2520cancer%252C%2520neurological%2520diseases%2520and%2520immune%2520disorders%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D673%26epage%3D691%26doi%3D10.1038%2Fnrd4360" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Milne, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Briggs, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brock, H. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gibbs, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allis, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hess, J. L.</span><span> </span><span class="NLM_article-title">MLL targets SET domain methyltransferase activity to Hox gene promoters</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">1107</span><span class="NLM_x">–</span> <span class="NLM_lpage">1117</span><span class="refDoi"> DOI: 10.1016/S1097-2765(02)00741-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1016%2FS1097-2765%2802%2900741-4" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2002&pages=1107-1117&author=T.+A.+Milneauthor=S.+D.+Briggsauthor=H.+W.+Brockauthor=M.+E.+Martinauthor=D.+Gibbsauthor=C.+D.+Allisauthor=J.+L.+Hess&title=MLL+targets+SET+domain+methyltransferase+activity+to+Hox+gene+promoters&doi=10.1016%2FS1097-2765%2802%2900741-4"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2FS1097-2765%2802%2900741-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1097-2765%252802%252900741-4%26sid%3Dliteratum%253Aachs%26aulast%3DMilne%26aufirst%3DT.%2BA.%26aulast%3DBriggs%26aufirst%3DS.%2BD.%26aulast%3DBrock%26aufirst%3DH.%2BW.%26aulast%3DMartin%26aufirst%3DM.%2BE.%26aulast%3DGibbs%26aufirst%3DD.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26aulast%3DHess%26aufirst%3DJ.%2BL.%26atitle%3DMLL%2520targets%2520SET%2520domain%2520methyltransferase%2520activity%2520to%2520Hox%2520gene%2520promoters%26jtitle%3DMol.%2520Cell%26date%3D2002%26volume%3D10%26spage%3D1107%26epage%3D1117%26doi%3D10.1016%2FS1097-2765%2802%2900741-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Briggs, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bryk, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Strahl, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheung, W. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davie, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dent, S. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winston, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allis, C. D.</span><span> </span><span class="NLM_article-title">Histone H3 lysine 4 methylation is mediated by Set1 and required for cell growth and rDNA silencing in Saccharomyces cerevisiae</span> <span class="citation_source-journal">Genes Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">3286</span><span class="NLM_x">–</span> <span class="NLM_lpage">3295</span><span class="refDoi"> DOI: 10.1101/gad.940201</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1101%2Fgad.940201" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2001&pages=3286-3295&author=S.+D.+Briggsauthor=M.+Brykauthor=B.+D.+Strahlauthor=W.+L.+Cheungauthor=J.+K.+Davieauthor=S.+Y.+Dentauthor=F.+Winstonauthor=C.+D.+Allis&title=Histone+H3+lysine+4+methylation+is+mediated+by+Set1+and+required+for+cell+growth+and+rDNA+silencing+in+Saccharomyces+cerevisiae&doi=10.1101%2Fgad.940201"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1101%2Fgad.940201&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.940201%26sid%3Dliteratum%253Aachs%26aulast%3DBriggs%26aufirst%3DS.%2BD.%26aulast%3DBryk%26aufirst%3DM.%26aulast%3DStrahl%26aufirst%3DB.%2BD.%26aulast%3DCheung%26aufirst%3DW.%2BL.%26aulast%3DDavie%26aufirst%3DJ.%2BK.%26aulast%3DDent%26aufirst%3DS.%2BY.%26aulast%3DWinston%26aufirst%3DF.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26atitle%3DHistone%2520H3%2520lysine%25204%2520methylation%2520is%2520mediated%2520by%2520Set1%2520and%2520required%2520for%2520cell%2520growth%2520and%2520rDNA%2520silencing%2520in%2520Saccharomyces%2520cerevisiae%26jtitle%3DGenes%2520Dev.%26date%3D2001%26volume%3D15%26spage%3D3286%26epage%3D3295%26doi%3D10.1101%2Fgad.940201" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Milne, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Briggs, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brock, H. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gibbs, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allis, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hess, J. L.</span><span> </span><span class="NLM_article-title">MLL targets SET domain methyltransferase activity to Hox gene promoters</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">1107</span><span class="NLM_x">–</span> <span class="NLM_lpage">1117</span><span class="refDoi"> DOI: 10.1016/S1097-2765(02)00741-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1016%2FS1097-2765%2802%2900741-4" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2002&pages=1107-1117&author=T.+A.+Milneauthor=S.+D.+Briggsauthor=H.+W.+Brockauthor=M.+E.+Martinauthor=D.+Gibbsauthor=C.+D.+Allisauthor=J.+L.+Hess&title=MLL+targets+SET+domain+methyltransferase+activity+to+Hox+gene+promoters&doi=10.1016%2FS1097-2765%2802%2900741-4"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2FS1097-2765%2802%2900741-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1097-2765%252802%252900741-4%26sid%3Dliteratum%253Aachs%26aulast%3DMilne%26aufirst%3DT.%2BA.%26aulast%3DBriggs%26aufirst%3DS.%2BD.%26aulast%3DBrock%26aufirst%3DH.%2BW.%26aulast%3DMartin%26aufirst%3DM.%2BE.%26aulast%3DGibbs%26aufirst%3DD.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26aulast%3DHess%26aufirst%3DJ.%2BL.%26atitle%3DMLL%2520targets%2520SET%2520domain%2520methyltransferase%2520activity%2520to%2520Hox%2520gene%2520promoters%26jtitle%3DMol.%2520Cell%26date%3D2002%26volume%3D10%26spage%3D1107%26epage%3D1117%26doi%3D10.1016%2FS1097-2765%2802%2900741-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Krivtsov, A. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Armstrong, S. A.</span><span> </span><span class="NLM_article-title">MLL translocations, histone modifications and leukaemia stem-cell development</span> <span class="citation_source-journal">Nat. Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">823</span><span class="NLM_x">–</span> <span class="NLM_lpage">833</span><span class="refDoi"> DOI: 10.1038/nrc2253</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1038%2Fnrc2253" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=17957188" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BD2sXht1Wgsb%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2007&pages=823-833&author=A.+V.+Krivtsovauthor=S.+A.+Armstrong&title=MLL+translocations%2C+histone+modifications+and+leukaemia+stem-cell+development&doi=10.1038%2Fnrc2253"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">MLL translocations, histone modifications and leukaemia stem-cell development</span></div><div class="casAuthors">Krivtsov, Andrei V.; Armstrong, Scott A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">823-833</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Mixed lineage leukemia (MLL) has histone methyltransferase activity and regulates the expression of genes such as Hox genes.  This activity is lost in MLL fusion proteins resulting from inter-chromosomal translocations, which are leukemogenic.  How do MLL fusions function and what is their role in leukemia stem cells.  Translocations that involve the mixed lineage leukemia (MLL) gene identify a unique group of acute leukemias, and often predict a poor prognosis.  The MLL gene encodes a DNA-binding protein that methylates histone H3 lysine 4 (H3K4), and pos. regulates gene expression including multiple Hox genes.  Leukaemogenic MLL translocations encode MLL fusion proteins that have lost H3K4 methyltransferase activity.  A key feature of MLL fusion proteins is their ability to efficiently transform haematopoietic cells into leukemia stem cells.  The link between a chromatin modulator and leukemia stem cells provides support for epigenetic landscapes as an important part of leukemia and normal stem-cell development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqm9NXrcNtrd7Vg90H21EOLACvtfcHk0li1JX8OoolH2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXht1Wgsb%252FJ&md5=56efc9bbbb9f24fb4233e40ed86628fd</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fnrc2253&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2253%26sid%3Dliteratum%253Aachs%26aulast%3DKrivtsov%26aufirst%3DA.%2BV.%26aulast%3DArmstrong%26aufirst%3DS.%2BA.%26atitle%3DMLL%2520translocations%252C%2520histone%2520modifications%2520and%2520leukaemia%2520stem-cell%2520development%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2007%26volume%3D7%26spage%3D823%26epage%3D833%26doi%3D10.1038%2Fnrc2253" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Nakamura, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mori, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tada, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krajewski, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rozovskaia, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wassell, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dubois, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mazo, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Croce, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Canaani, E.</span><span> </span><span class="NLM_article-title">ALL-1 is a histone methyltransferase that assembles a supercomplex of proteins involved in transcriptional regulation</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">1119</span><span class="NLM_x">–</span> <span class="NLM_lpage">1128</span><span class="refDoi"> DOI: 10.1016/S1097-2765(02)00740-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1016%2FS1097-2765%2802%2900740-2" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2002&pages=1119-1128&author=T.+Nakamuraauthor=T.+Moriauthor=S.+Tadaauthor=W.+Krajewskiauthor=T.+Rozovskaiaauthor=R.+Wassellauthor=G.+Duboisauthor=A.+Mazoauthor=C.+M.+Croceauthor=E.+Canaani&title=ALL-1+is+a+histone+methyltransferase+that+assembles+a+supercomplex+of+proteins+involved+in+transcriptional+regulation&doi=10.1016%2FS1097-2765%2802%2900740-2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2FS1097-2765%2802%2900740-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1097-2765%252802%252900740-2%26sid%3Dliteratum%253Aachs%26aulast%3DNakamura%26aufirst%3DT.%26aulast%3DMori%26aufirst%3DT.%26aulast%3DTada%26aufirst%3DS.%26aulast%3DKrajewski%26aufirst%3DW.%26aulast%3DRozovskaia%26aufirst%3DT.%26aulast%3DWassell%26aufirst%3DR.%26aulast%3DDubois%26aufirst%3DG.%26aulast%3DMazo%26aufirst%3DA.%26aulast%3DCroce%26aufirst%3DC.%2BM.%26aulast%3DCanaani%26aufirst%3DE.%26atitle%3DALL-1%2520is%2520a%2520histone%2520methyltransferase%2520that%2520assembles%2520a%2520supercomplex%2520of%2520proteins%2520involved%2520in%2520transcriptional%2520regulation%26jtitle%3DMol.%2520Cell%26date%3D2002%26volume%3D10%26spage%3D1119%26epage%3D1128%26doi%3D10.1016%2FS1097-2765%2802%2900740-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Shi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lan, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mulligan, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whetstine, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cole, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casero, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Y.</span><span> </span><span class="NLM_article-title">Histone demethylation mediated by the nuclear amine oxidase homolog LSD1</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">119</span><span class="NLM_x">, </span> <span class="NLM_fpage">941</span><span class="NLM_x">–</span> <span class="NLM_lpage">953</span><span class="refDoi"> DOI: 10.1016/j.cell.2004.12.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1016%2Fj.cell.2004.12.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=15620353" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BD2MXltlSitg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2004&pages=941-953&author=Y.+Shiauthor=F.+Lanauthor=C.+Matsonauthor=P.+Mulliganauthor=J.+R.+Whetstineauthor=P.+A.+Coleauthor=R.+A.+Caseroauthor=Y.+Shi&title=Histone+demethylation+mediated+by+the+nuclear+amine+oxidase+homolog+LSD1&doi=10.1016%2Fj.cell.2004.12.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Histone demethylation mediated by the nuclear amine oxidase homolog LSD1</span></div><div class="casAuthors">Shi, Yujiang; Lan, Fei; Matson, Caitlin; Mulligan, Peter; Whetstine, Johnathan R.; Cole, Philip A.; Casero, Robert A.; Shi, Yang</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">941-953</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Posttranslational modifications of histone N-terminal tails impact chromatin structure and gene transcription.  While the extent of histone acetylation is detd. by both acetyltransferases and deacetylases, it has been unclear whether histone methylation is also regulated by enzymes with opposing activities.  Here, we provide evidence that LSD1 (KIAA0601), a nuclear homolog of amine oxidases, functions as a histone demethylase and transcriptional corepressor.  LSD1 specifically demethylates histone H3 lysine 4, which is linked to active transcription.  Lysine demethylation occurs via an oxidn. reaction that generates formaldehyde.  Importantly, RNAi inhibition of LSD1 causes an increase in H3 lysine 4 methylation and concomitant derepression of target genes, suggesting that LSD1 represses transcription via histone demethylation.  The results thus identify a histone demethylase conserved from S. pombe to human and reveal dynamic regulation of histone methylation by both histone methylases and demethylases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXHr-8kwGdXrVg90H21EOLACvtfcHk0liJEupEtXss8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXltlSitg%253D%253D&md5=3474569204255e90a832667ba654748b</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2004.12.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2004.12.012%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DY.%26aulast%3DLan%26aufirst%3DF.%26aulast%3DMatson%26aufirst%3DC.%26aulast%3DMulligan%26aufirst%3DP.%26aulast%3DWhetstine%26aufirst%3DJ.%2BR.%26aulast%3DCole%26aufirst%3DP.%2BA.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26aulast%3DShi%26aufirst%3DY.%26atitle%3DHistone%2520demethylation%2520mediated%2520by%2520the%2520nuclear%2520amine%2520oxidase%2520homolog%2520LSD1%26jtitle%3DCell%26date%3D2004%26volume%3D119%26spage%3D941%26epage%3D953%26doi%3D10.1016%2Fj.cell.2004.12.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Kerenyi, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shao, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hsu, Y. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luc, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Brien, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fujiwara, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peng, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Orkin, S. H.</span><span> </span><span class="NLM_article-title">Histone demethylase Lsd1 represses hematopoietic stem and progenitor cell signatures during blood cell maturation</span> <span class="citation_source-journal">eLife</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">e00633</span><span class="refDoi"> DOI: 10.7554/eLife.00633</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.7554%2FeLife.00633" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=23795291" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjtlWktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2013&pages=e00633&author=M.+A.+Kerenyiauthor=Z.+Shaoauthor=Y.+J.+Hsuauthor=G.+Guoauthor=S.+Lucauthor=K.+O%E2%80%99Brienauthor=Y.+Fujiwaraauthor=C.+Pengauthor=M.+Nguyenauthor=S.+H.+Orkin&title=Histone+demethylase+Lsd1+represses+hematopoietic+stem+and+progenitor+cell+signatures+during+blood+cell+maturation&doi=10.7554%2FeLife.00633"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Histone demethylase Lsd1 represses hematopoietic stem and progenitor cell signatures during blood cell maturation</span></div><div class="casAuthors">Kerenyi, Marc A.; Shao, Zhen; Hsu, Yu-Jung; Guo, Guoji; Luc, Sidinh; O'Brien, Kassandra; Fujiwara, Yuko; Peng, Cong; Nguyen, Minh; Orkin, Stuart H.</div><div class="citationInfo"><span class="NLM_cas:title">eLife</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">e00633/1-e00633/23</span>CODEN:
                <span class="NLM_cas:coden">ELIFA8</span>;
        ISSN:<span class="NLM_cas:issn">2050-084X</span>.
    
            (<span class="NLM_cas:orgname">eLife Sciences Publications Ltd.</span>)
        </div><div class="casAbstract">Here, we describe that lysine-specific demethylase 1 (Lsd1/KDM1a), which demethylates histone H3 on Lys4 or Lys9 (H3K4/K9), is an indispensable epigenetic governor of hematopoietic differentiation.  Integrative genomic anal., combining global occupancy of Lsd1, genome-wide anal. of its substrates H3K4 monomethylation and dimethylation, and gene expression profiling, reveals that Lsd1 represses hematopoietic stem and progenitor cell (HSPC) gene expression programs during hematopoietic differentiation.  We found that Lsd1 acts at transcription start sites, as well as enhancer regions.  Loss of Lsd1 was assocd. with increased H3K4me1 and H3K4me2 methylation on HSPC genes and gene derepression.  Failure to fully silence HSPC genes compromised differentiation of hematopoietic stem cells as well as mature blood cell lineages.  Collectively, our data indicate that Lsd1-mediated concurrent repression of enhancer and promoter activity of stem and progenitor cell genes is a pivotal epigenetic mechanism required for proper hematopoietic maturation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrx7esSx55Ug7Vg90H21EOLACvtfcHk0liJEupEtXss8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjtlWktg%253D%253D&md5=2b810bfcc09f686619523fa7f3ebde06</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.7554%2FeLife.00633&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.00633%26sid%3Dliteratum%253Aachs%26aulast%3DKerenyi%26aufirst%3DM.%2BA.%26aulast%3DShao%26aufirst%3DZ.%26aulast%3DHsu%26aufirst%3DY.%2BJ.%26aulast%3DGuo%26aufirst%3DG.%26aulast%3DLuc%26aufirst%3DS.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DK.%26aulast%3DFujiwara%26aufirst%3DY.%26aulast%3DPeng%26aufirst%3DC.%26aulast%3DNguyen%26aufirst%3DM.%26aulast%3DOrkin%26aufirst%3DS.%2BH.%26atitle%3DHistone%2520demethylase%2520Lsd1%2520represses%2520hematopoietic%2520stem%2520and%2520progenitor%2520cell%2520signatures%2520during%2520blood%2520cell%2520maturation%26jtitle%3DeLife%26date%3D2013%26volume%3D2%26spage%3De00633%26doi%3D10.7554%2FeLife.00633" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Metzger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wissmann, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yin, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Müller, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneider, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peters, A. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Günther, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buettner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schüle, R.</span><span> </span><span class="NLM_article-title">LSD1 demethylates repressive histone marks to promote androgen-receptor-dependent transcription</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">437</span><span class="NLM_x">, </span> <span class="NLM_fpage">436</span><span class="NLM_x">–</span> <span class="NLM_lpage">439</span><span class="refDoi"> DOI: 10.1038/nature04020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1038%2Fnature04020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=16079795" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BD2MXpvFOrtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=437&publication_year=2005&pages=436-439&author=E.+Metzgerauthor=M.+Wissmannauthor=N.+Yinauthor=J.+M.+M%C3%BCllerauthor=R.+Schneiderauthor=A.+H.+Petersauthor=T.+G%C3%BCntherauthor=R.+Buettnerauthor=R.+Sch%C3%BCle&title=LSD1+demethylates+repressive+histone+marks+to+promote+androgen-receptor-dependent+transcription&doi=10.1038%2Fnature04020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">LSD1 demethylates repressive histone marks to promote androgen-receptor-dependent transcription</span></div><div class="casAuthors">Metzger, Eric; Wissmann, Melanie; Yin, Na; Mueller, Judith M.; Schneider, Robert; Peters, Antoine H. F. M.; Guenther, Thomas; Buettner, Reinhard; Schuele, Roland</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">437</span>
        (<span class="NLM_cas:issue">7057</span>),
    <span class="NLM_cas:pages">436-439</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Gene regulation in eukaryotes requires the coordinate interaction of chromatin-modulating proteins with specific transcription factors such as the androgen receptor.  Gene activation and repression is specifically regulated by histone methylation status at distinct lysine residues.  Here the authors show that lysine-specific demethylase 1 (LSD1; also known as BHC110) co-localizes with the androgen receptor in normal human prostate and prostate tumor.  LSD1 interacts with androgen receptor in vitro and in vivo, and stimulates androgen-receptor-dependent transcription.  Conversely, knockdown of LSD1 protein levels abrogates androgen-induced transcriptional activation and cell proliferation.  Chromatin immunopptn. analyses demonstrate that androgen receptor and LSD1 form chromatin-assocd. complexes in a ligand-dependent manner.  LSD1 relieves repressive histone marks by demethylation of histone H3 at lysine 9 (H3-K9), thereby leading to de-repression of androgen receptor target genes.  Furthermore, the authors identify pargyline as an inhibitor of LSD1.  Pargyline blocks demethylation of H3-K9 by LSD1 and consequently androgen-receptor-dependent transcription.  Thus, modulation of LSD1 activity offers a new strategy to regulate androgen receptor functions.  Here, the authors link demethylation of a repressive histone mark with androgen-receptor-dependent gene activation, thus providing a mechanism by which demethylases control specific gene expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgFgaYIdjiFbVg90H21EOLACvtfcHk0liJEupEtXss8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXpvFOrtL4%253D&md5=6cac1e6f6c21a80f59de27796cc9ee22</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fnature04020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature04020%26sid%3Dliteratum%253Aachs%26aulast%3DMetzger%26aufirst%3DE.%26aulast%3DWissmann%26aufirst%3DM.%26aulast%3DYin%26aufirst%3DN.%26aulast%3DM%25C3%25BCller%26aufirst%3DJ.%2BM.%26aulast%3DSchneider%26aufirst%3DR.%26aulast%3DPeters%26aufirst%3DA.%2BH.%26aulast%3DG%25C3%25BCnther%26aufirst%3DT.%26aulast%3DBuettner%26aufirst%3DR.%26aulast%3DSch%25C3%25BCle%26aufirst%3DR.%26atitle%3DLSD1%2520demethylates%2520repressive%2520histone%2520marks%2520to%2520promote%2520androgen-receptor-dependent%2520transcription%26jtitle%3DNature%26date%3D2005%26volume%3D437%26spage%3D436%26epage%3D439%26doi%3D10.1038%2Fnature04020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hevi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurash, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lei, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gay, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajko, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Su, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gaudet, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, T.</span><span> </span><span class="NLM_article-title">The lysine demethylase LSD1 (KDM1) is required for maintenance of global DNA methylation</span> <span class="citation_source-journal">Nat. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">125</span><span class="NLM_x">–</span> <span class="NLM_lpage">129</span><span class="refDoi"> DOI: 10.1038/ng.268</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1038%2Fng.268" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=19098913" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsFajtbbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2009&pages=125-129&author=J.+Wangauthor=S.+Heviauthor=J.+K.+Kurashauthor=H.+Leiauthor=F.+Gayauthor=J.+Bajkoauthor=H.+Suauthor=W.+Sunauthor=H.+Changauthor=G.+Xuauthor=F.+Gaudetauthor=E.+Liauthor=T.+Chen&title=The+lysine+demethylase+LSD1+%28KDM1%29+is+required+for+maintenance+of+global+DNA+methylation&doi=10.1038%2Fng.268"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">The lysine demethylase LSD1 (KDM1) is required for maintenance of global DNA methylation</span></div><div class="casAuthors">Wang, Jing; Hevi, Sarah; Kurash, Julia K.; Lei, Hong; Gay, Frederique; Bajko, Jeffrey; Su, Hui; Sun, Weitao; Chang, Hua; Xu, Guoliang; Gaudet, Francois; Li, En; Chen, Taiping</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">125-129</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Histone methylation and DNA methylation cooperatively regulate chromatin structure and gene activity.  How these two systems coordinate with each other remains unclear.  Here we study the biol. function of lysine-specific demethylase 1 (LSD1, also known as KDM1 and AOF2), which has been shown to demethylate histone H3 on lysine 4 (H3K4) and lysine 9 (H3K9).  We show that LSD1 is required for gastrulation during mouse embryogenesis.  Notably, targeted deletion of the gene encoding LSD1 (namely, Aof2) in embryonic stem (ES) cells induces progressive loss of DNA methylation.  This loss correlates with a decrease in DNA methyltransferase 1 (Dnmt1) protein, as a result of reduced Dnmt1 stability.  Dnmt1 protein is methylated in vivo, and its methylation is enhanced in the absence of LSD1.  Furthermore, Dnmt1 can be methylated by Set7/9 (also known as KMT7) and demethylated by LSD1 in vitro.  Our findings suggest that LSD1 demethylates and stabilizes Dnmt1, thus providing a previously unknown mechanistic link between the histone and DNA methylation systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSJgebl5_48LVg90H21EOLACvtfcHk0lhDKMjMc9jzhg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsFajtbbK&md5=37722ddd4564a79a0cd1c7b148c370d8</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fng.268&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fng.268%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DHevi%26aufirst%3DS.%26aulast%3DKurash%26aufirst%3DJ.%2BK.%26aulast%3DLei%26aufirst%3DH.%26aulast%3DGay%26aufirst%3DF.%26aulast%3DBajko%26aufirst%3DJ.%26aulast%3DSu%26aufirst%3DH.%26aulast%3DSun%26aufirst%3DW.%26aulast%3DChang%26aufirst%3DH.%26aulast%3DXu%26aufirst%3DG.%26aulast%3DGaudet%26aufirst%3DF.%26aulast%3DLi%26aufirst%3DE.%26aulast%3DChen%26aufirst%3DT.%26atitle%3DThe%2520lysine%2520demethylase%2520LSD1%2520%2528KDM1%2529%2520is%2520required%2520for%2520maintenance%2520of%2520global%2520DNA%2520methylation%26jtitle%3DNat.%2520Genet.%26date%3D2009%26volume%3D41%26spage%3D125%26epage%3D129%26doi%3D10.1038%2Fng.268" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Chen, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sorensen, P. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Domer, P. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reaman, G. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korsmeyer, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heerema, N. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hammond, G. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kersey, J. H.</span><span> </span><span class="NLM_article-title">Molecular rearrangements on chromosome 11q23 predominate in infant acute lymphoblastic leukemia and are associated with specific biologic variables and poor outcome</span> <span class="citation_source-journal">Blood</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">81</span><span class="NLM_x">, </span> <span class="NLM_fpage">2386</span><span class="NLM_x">–</span> <span class="NLM_lpage">2393</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=1993&pages=2386-2393&author=C.+S.+Chenauthor=P.+H.+Sorensenauthor=P.+H.+Domerauthor=G.+H.+Reamanauthor=S.+J.+Korsmeyerauthor=N.+A.+Heeremaauthor=G.+D.+Hammondauthor=J.+H.+Kersey&title=Molecular+rearrangements+on+chromosome+11q23+predominate+in+infant+acute+lymphoblastic+leukemia+and+are+associated+with+specific+biologic+variables+and+poor+outcome"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DC.%2BS.%26aulast%3DSorensen%26aufirst%3DP.%2BH.%26aulast%3DDomer%26aufirst%3DP.%2BH.%26aulast%3DReaman%26aufirst%3DG.%2BH.%26aulast%3DKorsmeyer%26aufirst%3DS.%2BJ.%26aulast%3DHeerema%26aufirst%3DN.%2BA.%26aulast%3DHammond%26aufirst%3DG.%2BD.%26aulast%3DKersey%26aufirst%3DJ.%2BH.%26atitle%3DMolecular%2520rearrangements%2520on%2520chromosome%252011q23%2520predominate%2520in%2520infant%2520acute%2520lymphoblastic%2520leukemia%2520and%2520are%2520associated%2520with%2520specific%2520biologic%2520variables%2520and%2520poor%2520outcome%26jtitle%3DBlood%26date%3D1993%26volume%3D81%26spage%3D2386%26epage%3D2393" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Mrozek, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heinonen, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lawrence, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carroll, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koduru, P. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rao, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Strout, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hutchison, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moore, J. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mayer, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schiffer, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bloomfield, C. D.</span><span> </span><span class="NLM_article-title">Adult patients with de novo acute myeloid leukemia and t(9; 11)(p22; q23) have a superior outcome to patients with other translocations involving band 11q23: a cancer and leukemia group B study</span> <span class="citation_source-journal">Blood</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">90</span><span class="NLM_x">, </span> <span class="NLM_fpage">4532</span><span class="NLM_x">–</span> <span class="NLM_lpage">4538</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=90&publication_year=1997&pages=4532-4538&author=K.+Mrozekauthor=K.+Heinonenauthor=D.+Lawrenceauthor=A.+J.+Carrollauthor=P.+R.+Koduruauthor=K.+W.+Raoauthor=M.+P.+Stroutauthor=R.+E.+Hutchisonauthor=J.+O.+Mooreauthor=R.+J.+Mayerauthor=C.+A.+Schifferauthor=C.+D.+Bloomfield&title=Adult+patients+with+de+novo+acute+myeloid+leukemia+and+t%289%3B+11%29%28p22%3B+q23%29+have+a+superior+outcome+to+patients+with+other+translocations+involving+band+11q23%3A+a+cancer+and+leukemia+group+B+study"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMrozek%26aufirst%3DK.%26aulast%3DHeinonen%26aufirst%3DK.%26aulast%3DLawrence%26aufirst%3DD.%26aulast%3DCarroll%26aufirst%3DA.%2BJ.%26aulast%3DKoduru%26aufirst%3DP.%2BR.%26aulast%3DRao%26aufirst%3DK.%2BW.%26aulast%3DStrout%26aufirst%3DM.%2BP.%26aulast%3DHutchison%26aufirst%3DR.%2BE.%26aulast%3DMoore%26aufirst%3DJ.%2BO.%26aulast%3DMayer%26aufirst%3DR.%2BJ.%26aulast%3DSchiffer%26aufirst%3DC.%2BA.%26aulast%3DBloomfield%26aufirst%3DC.%2BD.%26atitle%3DAdult%2520patients%2520with%2520de%2520novo%2520acute%2520myeloid%2520leukemia%2520and%2520t%25289%253B%252011%2529%2528p22%253B%2520q23%2529%2520have%2520a%2520superior%2520outcome%2520to%2520patients%2520with%2520other%2520translocations%2520involving%2520band%252011q23%253A%2520a%2520cancer%2520and%2520leukemia%2520group%2520B%2520study%26jtitle%3DBlood%26date%3D1997%26volume%3D90%26spage%3D4532%26epage%3D4538" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Hilden, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dinndorf, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meerbaum, S. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sather, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Villaluna, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heerema, N. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McGlennen, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, F. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woods, W. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salzer, W. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnstone, H. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dreyer, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reaman, G. H.</span><span> </span><span class="NLM_article-title">Analysis of prognostic factors of acute lymphoblastic leukemia in infants: report on CCG 1953 from the Children’s Oncology Group</span> <span class="citation_source-journal">Blood</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">108</span><span class="NLM_x">, </span> <span class="NLM_fpage">441</span><span class="NLM_x">–</span> <span class="NLM_lpage">451</span><span class="refDoi"> DOI: 10.1182/blood-2005-07-3011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1182%2Fblood-2005-07-3011" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2006&pages=441-451&author=J.+M.+Hildenauthor=P.+A.+Dinndorfauthor=S.+O.+Meerbaumauthor=H.+Satherauthor=D.+Villalunaauthor=N.+A.+Heeremaauthor=R.+McGlennenauthor=F.+O.+Smithauthor=W.+G.+Woodsauthor=W.+L.+Salzerauthor=H.+S.+Johnstoneauthor=Z.+Dreyerauthor=G.+H.+Reaman&title=Analysis+of+prognostic+factors+of+acute+lymphoblastic+leukemia+in+infants%3A+report+on+CCG+1953+from+the+Children%E2%80%99s+Oncology+Group&doi=10.1182%2Fblood-2005-07-3011"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1182%2Fblood-2005-07-3011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2005-07-3011%26sid%3Dliteratum%253Aachs%26aulast%3DHilden%26aufirst%3DJ.%2BM.%26aulast%3DDinndorf%26aufirst%3DP.%2BA.%26aulast%3DMeerbaum%26aufirst%3DS.%2BO.%26aulast%3DSather%26aufirst%3DH.%26aulast%3DVillaluna%26aufirst%3DD.%26aulast%3DHeerema%26aufirst%3DN.%2BA.%26aulast%3DMcGlennen%26aufirst%3DR.%26aulast%3DSmith%26aufirst%3DF.%2BO.%26aulast%3DWoods%26aufirst%3DW.%2BG.%26aulast%3DSalzer%26aufirst%3DW.%2BL.%26aulast%3DJohnstone%26aufirst%3DH.%2BS.%26aulast%3DDreyer%26aufirst%3DZ.%26aulast%3DReaman%26aufirst%3DG.%2BH.%26atitle%3DAnalysis%2520of%2520prognostic%2520factors%2520of%2520acute%2520lymphoblastic%2520leukemia%2520in%2520infants%253A%2520report%2520on%2520CCG%25201953%2520from%2520the%2520Children%25E2%2580%2599s%2520Oncology%2520Group%26jtitle%3DBlood%26date%3D2006%26volume%3D108%26spage%3D441%26epage%3D451%26doi%3D10.1182%2Fblood-2005-07-3011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Okada, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coffield, V. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Su, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span> </span><span class="NLM_article-title">hDOT1L links histone methylation to leukemogenesis</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">121</span><span class="NLM_x">, </span> <span class="NLM_fpage">167</span><span class="NLM_x">–</span> <span class="NLM_lpage">178</span><span class="refDoi"> DOI: 10.1016/j.cell.2005.02.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1016%2Fj.cell.2005.02.020" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=121&publication_year=2005&pages=167-178&author=Y.+Okadaauthor=Q.+Fengauthor=Y.+Linauthor=Q.+Jiangauthor=Y.+Liauthor=V.+M.+Coffieldauthor=L.+Suauthor=G.+Xuauthor=Y.+Zhang&title=hDOT1L+links+histone+methylation+to+leukemogenesis&doi=10.1016%2Fj.cell.2005.02.020"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2005.02.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2005.02.020%26sid%3Dliteratum%253Aachs%26aulast%3DOkada%26aufirst%3DY.%26aulast%3DFeng%26aufirst%3DQ.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DJiang%26aufirst%3DQ.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DCoffield%26aufirst%3DV.%2BM.%26aulast%3DSu%26aufirst%3DL.%26aulast%3DXu%26aufirst%3DG.%26aulast%3DZhang%26aufirst%3DY.%26atitle%3DhDOT1L%2520links%2520histone%2520methylation%2520to%2520leukemogenesis%26jtitle%3DCell%26date%3D2005%26volume%3D121%26spage%3D167%26epage%3D178%26doi%3D10.1016%2Fj.cell.2005.02.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Bitoun, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oliver, P. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davies, K. E.</span><span> </span><span class="NLM_article-title">The mixed-lineage leukemia fusion partner AF4 stimulates RNA polymerase II transcriptional elongation and mediates coordinated chromatin remodeling</span> <span class="citation_source-journal">Hum. Mol. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">92</span><span class="NLM_x">–</span> <span class="NLM_lpage">106</span><span class="refDoi"> DOI: 10.1093/hmg/ddl444</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1093%2Fhmg%2Fddl444" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=17135274" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BD2sXotVWqsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2007&pages=92-106&author=E.+Bitounauthor=P.+L.+Oliverauthor=K.+E.+Davies&title=The+mixed-lineage+leukemia+fusion+partner+AF4+stimulates+RNA+polymerase+II+transcriptional+elongation+and+mediates+coordinated+chromatin+remodeling&doi=10.1093%2Fhmg%2Fddl444"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">The mixed-lineage leukemia fusion partner AF4 stimulates RNA polymerase II transcriptional elongation and mediates coordinated chromatin remodeling</span></div><div class="casAuthors">Bitoun, Emmanuelle; Oliver, Peter L.; Davies, Kay E.</div><div class="citationInfo"><span class="NLM_cas:title">Human Molecular Genetics</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">92-106</span>CODEN:
                <span class="NLM_cas:coden">HMGEE5</span>;
        ISSN:<span class="NLM_cas:issn">0964-6906</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">AF4 gene, frequently translocated with mixed-lineage leukemia (MLL) in childhood acute leukemia, encodes a putative transcriptional activator of the AF4/LAF4/FMR2 (ALF) protein family previously implicated in lymphopoiesis and Purkinje cell function in the cerebellum.  Here, we provide the first evidence for a direct role of AF4 in the regulation of transcriptional elongation by RNA polymerase II (Pol II).  We demonstrate that mouse Af4 functions as a pos. regulator of Pol II transcription elongation factor b (P-TEFb) kinase and, in complex with MLL fusion partners Af9, Enl and Af10, as a mediator of histone H3-K79 methylation by recruiting Dot1 to elongating Pol II.  These pathways are interconnected and tightly regulated by the P-TEFb-dependent phosphorylation of Af4, Af9 and Enl which controls their transactivation activity and/or protein stability.  Consistently, increased levels of phosphorylated Pol II and methylated H3-K79 are obsd. in the ataxic mouse mutant robotic, an over-expression model of Af4.  Finally, we confirm the functional relevance of Af4, Enl and Af9 to the regulation of gene transcription as their over-expression strongly stimulates P-TEFb-dependent transcription of a luciferase reporter gene.  Our findings uncover a central role for these proteins in the regulation of transcriptional elongation and coordinated histone methylation, providing valuable insight into their contribution to leukemogenesis and neurodegeneration.  Since these activities likely extend to the entire ALF protein family, this study also significantly inputs our understanding of the mol. basis of FRAXE mental retardation syndrome in which FMR2 expression is silenced.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTx1-wiI5sbrVg90H21EOLACvtfcHk0lhsqsMPUxJCfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXotVWqsg%253D%253D&md5=82f9f9ce37a791034adc19c0eaeea078</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1093%2Fhmg%2Fddl444&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fhmg%252Fddl444%26sid%3Dliteratum%253Aachs%26aulast%3DBitoun%26aufirst%3DE.%26aulast%3DOliver%26aufirst%3DP.%2BL.%26aulast%3DDavies%26aufirst%3DK.%2BE.%26atitle%3DThe%2520mixed-lineage%2520leukemia%2520fusion%2520partner%2520AF4%2520stimulates%2520RNA%2520polymerase%2520II%2520transcriptional%2520elongation%2520and%2520mediates%2520coordinated%2520chromatin%2520remodeling%26jtitle%3DHum.%2520Mol.%2520Genet.%26date%3D2007%26volume%3D16%26spage%3D92%26epage%3D106%26doi%3D10.1093%2Fhmg%2Fddl444" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Krivtsov, A. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lemieux, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Faber, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vempati, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sinha, A. U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xia, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jesneck, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bracken, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Silverman, L. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kutok, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kung, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Armstrong, S. A.</span><span> </span><span class="NLM_article-title">H3K79 methylation profiles define murine and human MLL-AF4 leukemias</span> <span class="citation_source-journal">Cancer Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">355</span><span class="NLM_x">–</span> <span class="NLM_lpage">368</span><span class="refDoi"> DOI: 10.1016/j.ccr.2008.10.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1016%2Fj.ccr.2008.10.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=18977325" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtlOjsL3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2008&pages=355-368&author=A.+V.+Krivtsovauthor=Z.+Fengauthor=M.+E.+Lemieuxauthor=J.+Faberauthor=S.+Vempatiauthor=A.+U.+Sinhaauthor=X.+Xiaauthor=J.+Jesneckauthor=A.+P.+Brackenauthor=L.+B.+Silvermanauthor=J.+L.+Kutokauthor=A.+L.+Kungauthor=S.+A.+Armstrong&title=H3K79+methylation+profiles+define+murine+and+human+MLL-AF4+leukemias&doi=10.1016%2Fj.ccr.2008.10.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">H3K79 methylation profiles define murine and human MLL-AF4 leukemias</span></div><div class="casAuthors">Krivtsov, Andrei V.; Feng, Zhaohui; Lemieux, Madeleine E.; Faber, Joerg; Vempati, Sridhar; Sinha, Amit U.; Xia, Xiaobo; Jesneck, Jonathan; Bracken, Adrian P.; Silverman, Lewis B.; Kutok, Jeffery L.; Kung, Andrew L.; Armstrong, Scott A.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">355-368</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">We created a mouse model wherein conditional expression of an MLL-AF4 fusion oncogene induces B precursor acute lymphoblastic (ALL) or acute myeloid leukemias (AML).  Gene expression profile anal. of the ALL cells demonstrated significant overlap with human MLL-rearranged ALL.  ChIP-chip anal. demonstrated histone H3 lysine 79 (H3K79) methylation profiles that correlated with MLL-AF4-assocd. gene expression profiles in murine ALLs and in human MLL-rearranged leukemias.  Human MLL-rearranged ALLs could be distinguished from other ALLs by their H3K79 profiles, and suppression of the H3K79 methyltransferase DOT1L inhibited expression of crit. MLL-AF4 target genes.  We thus demonstrate that ectopic H3K79 methylation is a distinguishing feature of murine and human MLL-AF4 ALLs and is important for maintenance of MLL-AF4-driven gene expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouUg_fkZBiabVg90H21EOLACvtfcHk0lhsqsMPUxJCfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtlOjsL3F&md5=755a12b8819e8e4a798b434a6818c647</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2008.10.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2008.10.001%26sid%3Dliteratum%253Aachs%26aulast%3DKrivtsov%26aufirst%3DA.%2BV.%26aulast%3DFeng%26aufirst%3DZ.%26aulast%3DLemieux%26aufirst%3DM.%2BE.%26aulast%3DFaber%26aufirst%3DJ.%26aulast%3DVempati%26aufirst%3DS.%26aulast%3DSinha%26aufirst%3DA.%2BU.%26aulast%3DXia%26aufirst%3DX.%26aulast%3DJesneck%26aufirst%3DJ.%26aulast%3DBracken%26aufirst%3DA.%2BP.%26aulast%3DSilverman%26aufirst%3DL.%2BB.%26aulast%3DKutok%26aufirst%3DJ.%2BL.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DArmstrong%26aufirst%3DS.%2BA.%26atitle%3DH3K79%2520methylation%2520profiles%2520define%2520murine%2520and%2520human%2520MLL-AF4%2520leukemias%26jtitle%3DCancer%2520Cell%26date%3D2008%26volume%3D14%26spage%3D355%26epage%3D368%26doi%3D10.1016%2Fj.ccr.2008.10.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="note"><p class="first last">For work from us, see the following:</p></div><div class="NLM_citation" id="cit23a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Yao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anglin, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prasad, B. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, Y.</span><span> </span><span class="NLM_article-title">Selective inhibitors of histone methyltransferase DOT1L: design, synthesis, and crystallographic studies</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">133</span><span class="NLM_x">, </span> <span class="NLM_fpage">16746</span><span class="NLM_x">–</span> <span class="NLM_lpage">16749</span><span class="refDoi"> DOI: 10.1021/ja206312b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja206312b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2011&pages=16746-16749&author=Y.+Yaoauthor=P.+Chenauthor=J.+Diaoauthor=G.+Chengauthor=L.+Dengauthor=J.+L.+Anglinauthor=B.+V.+Prasadauthor=Y.+Song&title=Selective+inhibitors+of+histone+methyltransferase+DOT1L%3A+design%2C+synthesis%2C+and+crystallographic+studies&doi=10.1021%2Fja206312b"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23a&amp;dbid=16384&amp;doi=10.1021%2Fja206312b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja206312b%26sid%3Dliteratum%253Aachs%26aulast%3DYao%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DP.%26aulast%3DDiao%26aufirst%3DJ.%26aulast%3DCheng%26aufirst%3DG.%26aulast%3DDeng%26aufirst%3DL.%26aulast%3DAnglin%26aufirst%3DJ.%2BL.%26aulast%3DPrasad%26aufirst%3DB.%2BV.%26aulast%3DSong%26aufirst%3DY.%26atitle%3DSelective%2520inhibitors%2520of%2520histone%2520methyltransferase%2520DOT1L%253A%2520design%252C%2520synthesis%252C%2520and%2520crystallographic%2520studies%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2011%26volume%3D133%26spage%3D16746%26epage%3D16749%26doi%3D10.1021%2Fja206312b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit23b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Anglin, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cai, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, Y.</span><span> </span><span class="NLM_article-title">Synthesis and structure-activity relationship investigation of adenosine-containing inhibitors of histone methyltransferase DOT1L</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">8066</span><span class="NLM_x">–</span> <span class="NLM_lpage">8074</span><span class="refDoi"> DOI: 10.1021/jm300917h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm300917h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1KjsrjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=8066-8074&author=J.+L.+Anglinauthor=L.+Dengauthor=Y.+Yaoauthor=G.+Caiauthor=Z.+Liuauthor=H.+Jiangauthor=G.+Chengauthor=P.+Chenauthor=S.+Dongauthor=Y.+Song&title=Synthesis+and+structure-activity+relationship+investigation+of+adenosine-containing+inhibitors+of+histone+methyltransferase+DOT1L&doi=10.1021%2Fjm300917h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23bR"><div class="casContent"><span class="casTitleNuber">23b</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and Structure-Activity Relationship Investigation of Adenosine-Containing Inhibitors of Histone Methyltransferase DOT1L</span></div><div class="casAuthors">Anglin, Justin L.; Deng, Lisheng; Yao, Yuan; Cai, Guobin; Liu, Zhen; Jiang, Hong; Cheng, Gang; Chen, Pinhong; Dong, Shuo; Song, Yongcheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">8066-8074</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Histone-3-lysine79 (H3K79) methyltransferase DOT1L has been found to be a drug target for acute leukemia with MLL (mixed lineage leukemia) gene translocation.  A total of 55 adenosine-contg. compds., e.g. I, were designed and synthesized, among which several potent DOT1L inhibitors were identified with Ki values as low as 0.5 nM.  These compds. also show high selectivity (>4500-fold) over three other histone methyltransferases.  Structure-activity relationships (SAR) of these compds. for their inhibitory activities against DOT1L are discussed.  Potent DOT1L inhibitors exhibit selective activity against the proliferation of MLL-translocated leukemia cell lines with EC50 values of 4-11 μM.  Isothermal titrn. calorimetry studies showed that two representative inhibitors bind with a high affinity to the DOT1L:nucleosome complex and only compete with the enzyme cofactor SAM (S-adenosyl-L-methionine) but not the substrate nucleosome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3lGfG_WNicbVg90H21EOLACvtfcHk0ljEI7kNd9vrSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1KjsrjO&md5=d869fdfbf94d9fa7282086d8f507f11b</span></div><a href="/servlet/linkout?suffix=cit23b&amp;dbid=16384&amp;doi=10.1021%2Fjm300917h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm300917h%26sid%3Dliteratum%253Aachs%26aulast%3DAnglin%26aufirst%3DJ.%2BL.%26aulast%3DDeng%26aufirst%3DL.%26aulast%3DYao%26aufirst%3DY.%26aulast%3DCai%26aufirst%3DG.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DCheng%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DP.%26aulast%3DDong%26aufirst%3DS.%26aulast%3DSong%26aufirst%3DY.%26atitle%3DSynthesis%2520and%2520structure-activity%2520relationship%2520investigation%2520of%2520adenosine-containing%2520inhibitors%2520of%2520histone%2520methyltransferase%2520DOT1L%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D8066%26epage%3D8074%26doi%3D10.1021%2Fjm300917h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit23c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Deng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Redell, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, Y.</span><span> </span><span class="NLM_article-title">Synthesis, activity and metabolic stability of non-ribose containing inhibitors of histone methyltransferase DOT1L</span> <span class="citation_source-journal">MedChemComm</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">822</span><span class="NLM_x">–</span> <span class="NLM_lpage">826</span><span class="refDoi"> DOI: 10.1039/c3md00021d</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1039%2Fc3md00021d" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=23795283" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BC3sXmvVygs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=822-826&author=L.+Dengauthor=L.+Zhangauthor=Y.+Yaoauthor=C.+Wangauthor=M.+S.+Redellauthor=S.+Dongauthor=Y.+Song&title=Synthesis%2C+activity+and+metabolic+stability+of+non-ribose+containing+inhibitors+of+histone+methyltransferase+DOT1L&doi=10.1039%2Fc3md00021d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23cR"><div class="casContent"><span class="casTitleNuber">23c</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis, activity and metabolic stability of non-ribose containing inhibitors of histone methyltransferase DOT1L</span></div><div class="casAuthors">Deng, Lisheng; Zhang, Li; Yao, Yuan; Wang, Cong; Redell, Michele S.; Dong, Shuo; Song, Yongcheng</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">822-826</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Histone methyltransferase DOT1L is a drug target for MLL leukemia.  We report an efficient synthesis of a cyclopentane-contg. adenosine nucleoside compd. (I) that potently and selectively inhibits DOT1L (Ki = 1.1 nM) as well as H3K79 methylation (IC50 ∼ 200 nM).  Importantly, this compd. exhibits a high stability in plasma and liver microsomes, suggesting it is a better drug candidate.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-mJfoEIVrvLVg90H21EOLACvtfcHk0ljEI7kNd9vrSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXmvVygs7o%253D&md5=202e6bf4c9eeed38982b0f036a253425</span></div><a href="/servlet/linkout?suffix=cit23c&amp;dbid=16384&amp;doi=10.1039%2Fc3md00021d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc3md00021d%26sid%3Dliteratum%253Aachs%26aulast%3DDeng%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DYao%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DRedell%26aufirst%3DM.%2BS.%26aulast%3DDong%26aufirst%3DS.%26aulast%3DSong%26aufirst%3DY.%26atitle%3DSynthesis%252C%2520activity%2520and%2520metabolic%2520stability%2520of%2520non-ribose%2520containing%2520inhibitors%2520of%2520histone%2520methyltransferase%2520DOT1L%26jtitle%3DMedChemComm%26date%3D2013%26volume%3D4%26spage%3D822%26epage%3D826%26doi%3D10.1039%2Fc3md00021d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit23d"><span><span class="NLM_label">(d) </span><span class="NLM_contrib-group">Liu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Redell, M.</span><span> </span><span class="NLM_article-title">DOT1L Inhibition Sensitizes MLL-Rearranged AML to Chemotherapy</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">e98270</span><span class="refDoi"> DOI: 10.1371/journal.pone.0098270</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23d&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1371%2Fjournal.pone.0098270" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=e98270&author=W.+Liuauthor=L.+Dengauthor=Y.+Songauthor=M.+Redell&title=DOT1L+Inhibition+Sensitizes+MLL-Rearranged+AML+to+Chemotherapy&doi=10.1371%2Fjournal.pone.0098270"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23d&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0098270&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0098270%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DDeng%26aufirst%3DL.%26aulast%3DSong%26aufirst%3DY.%26aulast%3DRedell%26aufirst%3DM.%26atitle%3DDOT1L%2520Inhibition%2520Sensitizes%2520MLL-Rearranged%2520AML%2520to%2520Chemotherapy%26jtitle%3DPLoS%2520One%26date%3D2014%26volume%3D9%26spage%3De98270%26doi%3D10.1371%2Fjournal.pone.0098270" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="note"><p class="first last">For work from other groups, seethe following:</p></div><div class="NLM_citation" id="cit24a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Daigle, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olhava, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Therkelsen, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Majer, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sneeringer, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnston, L. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuntz, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chesworth, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moyer, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bernt, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tseng, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kung, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Armstrong, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richon, V. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollock, R. M.</span><span> </span><span class="NLM_article-title">Selective killing of mixed lineage leukemia cells by a potent small-molecule DOT1L inhibitor</span> <span class="citation_source-journal">Cancer Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">53</span><span class="NLM_x">–</span> <span class="NLM_lpage">65</span><span class="refDoi"> DOI: 10.1016/j.ccr.2011.06.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1016%2Fj.ccr.2011.06.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=21741596" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BC3MXoslCms7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2011&pages=53-65&author=S.+R.+Daigleauthor=E.+J.+Olhavaauthor=C.+A.+Therkelsenauthor=C.+R.+Majerauthor=C.+J.+Sneeringerauthor=J.+Songauthor=L.+D.+Johnstonauthor=M.+P.+Scottauthor=J.+J.+Smithauthor=Y.+Xiaoauthor=L.+Jinauthor=K.+W.+Kuntzauthor=R.+Chesworthauthor=M.+P.+Moyerauthor=K.+M.+Berntauthor=J.+C.+Tsengauthor=A.+L.+Kungauthor=S.+A.+Armstrongauthor=R.+A.+Copelandauthor=V.+M.+Richonauthor=R.+M.+Pollock&title=Selective+killing+of+mixed+lineage+leukemia+cells+by+a+potent+small-molecule+DOT1L+inhibitor&doi=10.1016%2Fj.ccr.2011.06.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24aR"><div class="casContent"><span class="casTitleNuber">24a</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Killing of Mixed Lineage Leukemia Cells by a Potent Small-Molecule DOT1L Inhibitor</span></div><div class="casAuthors">Daigle, Scott R.; Olhava, Edward J.; Therkelsen, Carly A.; Majer, Christina R.; Sneeringer, Christopher J.; Song, Jeffrey; Johnston, L. Danielle; Scott, Margaret Porter; Smith, Jesse J.; Xiao, Yonghong; Jin, Lei; Kuntz, Kevin W.; Chesworth, Richard; Moyer, Mikel P.; Bernt, Kathrin M.; Tseng, Jen-Chieh; Kung, Andrew L.; Armstrong, Scott A.; Copeland, Robert A.; Richon, Victoria M.; Pollock, Roy M.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">53-65</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Mislocated enzymic activity of DOT1L has been proposed as a driver of leukemogenesis in mixed lineage leukemia (MLL).  The characterization of EPZ004777, a potent, selective inhibitor of DOT1L is reported.  Treatment of MLL cells with the compd. selectively inhibits H3K79 methylation and blocks expression of leukemogenic genes.  Exposure of leukemic cells to EPZ004777 results in selective killing of those cells bearing the MLL gene translocation, with little effect on non-MLL-translocated cells.  Finally, in vivo administration of EPZ004777 leads to extension of survival in a mouse MLL xenograft model.  These results provide compelling support for DOT1L inhibition as a basis for targeted therapeutics against MLL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodbBbxHFmyybVg90H21EOLACvtfcHk0ljEI7kNd9vrSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXoslCms7c%253D&md5=1d39f99c2983b6b5fccc079110092c3d</span></div><a href="/servlet/linkout?suffix=cit24a&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2011.06.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2011.06.009%26sid%3Dliteratum%253Aachs%26aulast%3DDaigle%26aufirst%3DS.%2BR.%26aulast%3DOlhava%26aufirst%3DE.%2BJ.%26aulast%3DTherkelsen%26aufirst%3DC.%2BA.%26aulast%3DMajer%26aufirst%3DC.%2BR.%26aulast%3DSneeringer%26aufirst%3DC.%2BJ.%26aulast%3DSong%26aufirst%3DJ.%26aulast%3DJohnston%26aufirst%3DL.%2BD.%26aulast%3DScott%26aufirst%3DM.%2BP.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DXiao%26aufirst%3DY.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DKuntz%26aufirst%3DK.%2BW.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DBernt%26aufirst%3DK.%2BM.%26aulast%3DTseng%26aufirst%3DJ.%2BC.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DArmstrong%26aufirst%3DS.%2BA.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DRichon%26aufirst%3DV.%2BM.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26atitle%3DSelective%2520killing%2520of%2520mixed%2520lineage%2520leukemia%2520cells%2520by%2520a%2520potent%2520small-molecule%2520DOT1L%2520inhibitor%26jtitle%3DCancer%2520Cell%26date%3D2011%26volume%3D20%26spage%3D53%26epage%3D65%26doi%3D10.1016%2Fj.ccr.2011.06.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit24b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Basavapathruni, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Daigle, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Majer, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Therkelsen, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wigle, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuntz, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chesworth, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollock, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moyer, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richon, V. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olhava, E. J.</span><span> </span><span class="NLM_article-title">Conformational adaptation drives potent, selective and durable inhibition of the human protein methyltransferase DOT1L</span> <span class="citation_source-journal">Chem. Biol. Drug Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">80</span><span class="NLM_x">, </span> <span class="NLM_fpage">971</span><span class="NLM_x">–</span> <span class="NLM_lpage">980</span><span class="refDoi"> DOI: 10.1111/cbdd.12050</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1111%2Fcbdd.12050" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=22978415" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslClsr7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=80&publication_year=2012&pages=971-980&author=A.+Basavapathruniauthor=L.+Jinauthor=S.+R.+Daigleauthor=C.+R.+Majerauthor=C.+A.+Therkelsenauthor=T.+J.+Wigleauthor=K.+W.+Kuntzauthor=R.+Chesworthauthor=R.+M.+Pollockauthor=M.+P.+Scottauthor=M.+P.+Moyerauthor=V.+M.+Richonauthor=R.+A.+Copelandauthor=E.+J.+Olhava&title=Conformational+adaptation+drives+potent%2C+selective+and+durable+inhibition+of+the+human+protein+methyltransferase+DOT1L&doi=10.1111%2Fcbdd.12050"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24bR"><div class="casContent"><span class="casTitleNuber">24b</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational adaptation drives potent, selective and durable inhibition of the human protein methyltransferase DOT1L</span></div><div class="casAuthors">Basavapathruni, Aravind; Jin, Lei; Daigle, Scott R.; Majer, Christina R. A.; Therkelsen, Carly A.; Wigle, Tim J.; Kuntz, Kevin W.; Chesworth, Richard; Pollock, Roy M.; Scott, Margaret P.; Moyer, Mikel P.; Richon, Victoria M.; Copeland, Robert A.; Olhava, Edward J.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">80</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">971-980</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">DOT1L is the human protein methyltransferase responsible for catalyzing the methylation of histone H3 on lysine 79 (H3K79).  The ectopic activity of DOT1L, assocd. with the chromosomal translocation that is a universal hallmark of MLL-rearranged leukemia, is a required driver of leukemogenesis in this malignancy.  Here, we present studies on the structure-activity relation of aminonucleoside-based DOT1L inhibitors.  Within this series, we find that improvements in target enzyme affinity and selectivity are driven entirely by diminution of the dissocn. rate const. for the enzyme-inhibitor complex, leading to long residence times for the binary complex.  The biochem. Ki and residence times measured for these inhibitors correlate well with their effects on intracellular H3K79 methylation and MLL-rearranged leukemic cell killing.  Crystallog. studies reveal a conformational adaptation mechanism assocd. with high-affinity inhibitor binding and prolonged residence time; these studies also suggest that conformational adaptation likewise plays a crit. role in natural ligand interactions with the enzyme, hence, facilitating enzyme turnover.  These results provide crit. insights into the role of conformational adaptation in the enzymic mechanism of catalysis and in pharmacol. intervention for DOT1L and other members of this enzyme class.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJm_d3iX3p77Vg90H21EOLACvtfcHk0lgpPa5ovD9ayA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslClsr7O&md5=de628fcd2c37b5f6496cc3b483125309</span></div><a href="/servlet/linkout?suffix=cit24b&amp;dbid=16384&amp;doi=10.1111%2Fcbdd.12050&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fcbdd.12050%26sid%3Dliteratum%253Aachs%26aulast%3DBasavapathruni%26aufirst%3DA.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DDaigle%26aufirst%3DS.%2BR.%26aulast%3DMajer%26aufirst%3DC.%2BR.%26aulast%3DTherkelsen%26aufirst%3DC.%2BA.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DKuntz%26aufirst%3DK.%2BW.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26aulast%3DScott%26aufirst%3DM.%2BP.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DRichon%26aufirst%3DV.%2BM.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DOlhava%26aufirst%3DE.%2BJ.%26atitle%3DConformational%2520adaptation%2520drives%2520potent%252C%2520selective%2520and%2520durable%2520inhibition%2520of%2520the%2520human%2520protein%2520methyltransferase%2520DOT1L%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2012%26volume%3D80%26spage%3D971%26epage%3D980%26doi%3D10.1111%2Fcbdd.12050" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit24c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Yu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chory, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wernimont, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tempel, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scopton, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Federation, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marineau, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marcellus, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iacob, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Engen, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Griffin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aman, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wienholds, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pineda, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Estiu, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shatseva, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajian, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-Awar, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dick, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bradner, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span> </span><span class="NLM_article-title">Catalytic site remodelling of the DOT1L methyltransferase by selective inhibitors</span> <span class="citation_source-journal">Nat. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">1288</span><span class="refDoi"> DOI: 10.1038/ncomms2304</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1038%2Fncomms2304" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=23250418" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A280%3ADC%252BC3s3jtVWlsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2012&pages=1288&author=W.+Yuauthor=E.+J.+Choryauthor=A.+K.+Wernimontauthor=W.+Tempelauthor=A.+Scoptonauthor=A.+Federationauthor=J.+J.+Marineauauthor=J.+Qiauthor=D.+Barsyte-Lovejoyauthor=J.+Yiauthor=R.+Marcellusauthor=R.+E.+Iacobauthor=J.+R.+Engenauthor=C.+Griffinauthor=A.+Amanauthor=E.+Wienholdsauthor=F.+Liauthor=J.+Pinedaauthor=G.+Estiuauthor=T.+Shatsevaauthor=T.+Hajianauthor=R.+Al-Awarauthor=J.+E.+Dickauthor=M.+Vedadiauthor=P.+J.+Brownauthor=C.+H.+Arrowsmithauthor=J.+E.+Bradnerauthor=M.+Schapira&title=Catalytic+site+remodelling+of+the+DOT1L+methyltransferase+by+selective+inhibitors&doi=10.1038%2Fncomms2304"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24cR"><div class="casContent"><span class="casTitleNuber">24c</span><div class="casTitle"><span class="NLM_cas:atitle">Catalytic site remodelling of the DOT1L methyltransferase by selective inhibitors</span></div><div class="casAuthors">Yu Wenyu; Chory Emma J; Wernimont Amy K; Tempel Wolfram; Scopton Alex; Federation Alexander; Marineau Jason J; Qi Jun; Barsyte-Lovejoy Dalia; Yi Joanna; Marcellus Richard; Iacob Roxana E; Engen John R; Griffin Carly; Aman Ahmed; Wienholds Erno; Li Fengling; Pineda Javier; Estiu Guillermina; Shatseva Tatiana; Hajian Taraneh; Al-Awar Rima; Dick John E; Vedadi Masoud; Brown Peter J; Arrowsmith Cheryl H; Bradner James E; Schapira Matthieu</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1288</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Selective inhibition of protein methyltransferases is a promising new approach to drug discovery.  An attractive strategy towards this goal is the development of compounds that selectively inhibit binding of the cofactor, S-adenosylmethionine, within specific protein methyltransferases.  Here we report the three-dimensional structure of the protein methyltransferase DOT1L bound to EPZ004777, the first S-adenosylmethionine-competitive inhibitor of a protein methyltransferase with in vivo efficacy.  This structure and those of four new analogues reveal remodelling of the catalytic site.  EPZ004777 and a brominated analogue, SGC0946, inhibit DOT1L in vitro and selectively kill mixed lineage leukaemia cells, in which DOT1L is aberrantly localized via interaction with an oncogenic MLL fusion protein.  These data provide important new insight into mechanisms of cell-active S-adenosylmethionine-competitive protein methyltransferase inhibitors, and establish a foundation for the further development of drug-like inhibitors of DOT1L for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcST2GJQO5XUmjDnmmyUf5qOfW6udTcc2eaqe33fVHfg1Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3s3jtVWlsQ%253D%253D&md5=f7b501ac4188b14b48fc435637e0e192</span></div><a href="/servlet/linkout?suffix=cit24c&amp;dbid=16384&amp;doi=10.1038%2Fncomms2304&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms2304%26sid%3Dliteratum%253Aachs%26aulast%3DYu%26aufirst%3DW.%26aulast%3DChory%26aufirst%3DE.%2BJ.%26aulast%3DWernimont%26aufirst%3DA.%2BK.%26aulast%3DTempel%26aufirst%3DW.%26aulast%3DScopton%26aufirst%3DA.%26aulast%3DFederation%26aufirst%3DA.%26aulast%3DMarineau%26aufirst%3DJ.%2BJ.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DYi%26aufirst%3DJ.%26aulast%3DMarcellus%26aufirst%3DR.%26aulast%3DIacob%26aufirst%3DR.%2BE.%26aulast%3DEngen%26aufirst%3DJ.%2BR.%26aulast%3DGriffin%26aufirst%3DC.%26aulast%3DAman%26aufirst%3DA.%26aulast%3DWienholds%26aufirst%3DE.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DPineda%26aufirst%3DJ.%26aulast%3DEstiu%26aufirst%3DG.%26aulast%3DShatseva%26aufirst%3DT.%26aulast%3DHajian%26aufirst%3DT.%26aulast%3DAl-Awar%26aufirst%3DR.%26aulast%3DDick%26aufirst%3DJ.%2BE.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DSchapira%26aufirst%3DM.%26atitle%3DCatalytic%2520site%2520remodelling%2520of%2520the%2520DOT1L%2520methyltransferase%2520by%2520selective%2520inhibitors%26jtitle%3DNat.%2520Commun.%26date%3D2012%26volume%3D3%26spage%3D1288%26doi%3D10.1038%2Fncomms2304" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit24d"><span><span class="NLM_label">(d) </span><span class="NLM_contrib-group">Daigle, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olhava, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Therkelsen, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Basavapathruni, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boriack-Sjodin, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allain, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klaus, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raimondi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waters, N. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chesworth, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moyer, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richon, V. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollock, R. M.</span><span> </span><span class="NLM_article-title">Potent inhibition of DOT1L as treatment for MLL-fusion leukemia</span> <span class="citation_source-journal">Blood</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">122</span><span class="NLM_x">, </span> <span class="NLM_fpage">1017</span><span class="NLM_x">–</span> <span class="NLM_lpage">1025</span><span class="refDoi"> DOI: 10.1182/blood-2013-04-497644</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24d&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1182%2Fblood-2013-04-497644" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=122&publication_year=2013&pages=1017-1025&author=S.+R.+Daigleauthor=E.+J.+Olhavaauthor=C.+A.+Therkelsenauthor=A.+Basavapathruniauthor=L.+Jinauthor=P.+A.+Boriack-Sjodinauthor=C.+J.+Allainauthor=C.+R.+Klausauthor=A.+Raimondiauthor=M.+P.+Scottauthor=N.+J.+Watersauthor=R.+Chesworthauthor=M.+P.+Moyerauthor=R.+A.+Copelandauthor=V.+M.+Richonauthor=R.+M.+Pollock&title=Potent+inhibition+of+DOT1L+as+treatment+for+MLL-fusion+leukemia&doi=10.1182%2Fblood-2013-04-497644"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24d&amp;dbid=16384&amp;doi=10.1182%2Fblood-2013-04-497644&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2013-04-497644%26sid%3Dliteratum%253Aachs%26aulast%3DDaigle%26aufirst%3DS.%2BR.%26aulast%3DOlhava%26aufirst%3DE.%2BJ.%26aulast%3DTherkelsen%26aufirst%3DC.%2BA.%26aulast%3DBasavapathruni%26aufirst%3DA.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DBoriack-Sjodin%26aufirst%3DP.%2BA.%26aulast%3DAllain%26aufirst%3DC.%2BJ.%26aulast%3DKlaus%26aufirst%3DC.%2BR.%26aulast%3DRaimondi%26aufirst%3DA.%26aulast%3DScott%26aufirst%3DM.%2BP.%26aulast%3DWaters%26aufirst%3DN.%2BJ.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DRichon%26aufirst%3DV.%2BM.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26atitle%3DPotent%2520inhibition%2520of%2520DOT1L%2520as%2520treatment%2520for%2520MLL-fusion%2520leukemia%26jtitle%3DBlood%26date%3D2013%26volume%3D122%26spage%3D1017%26epage%3D1025%26doi%3D10.1182%2Fblood-2013-04-497644" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Binda, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valente, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Romanenghi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pilotto, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cirilli, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karytinos, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciossani, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Botrugno, O. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Forneris, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tardugno, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edmondson, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Minucci, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattevi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mai, A.</span><span> </span><span class="NLM_article-title">Biochemical, structural, and biological evaluation of tranylcypromine derivatives as inhibitors of histone demethylases LSD1 and LSD2</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">132</span><span class="NLM_x">, </span> <span class="NLM_fpage">6827</span><span class="NLM_x">–</span> <span class="NLM_lpage">6833</span><span class="refDoi"> DOI: 10.1021/ja101557k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja101557k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BC3cXltFChur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2010&pages=6827-6833&author=C.+Bindaauthor=S.+Valenteauthor=M.+Romanenghiauthor=S.+Pilottoauthor=R.+Cirilliauthor=A.+Karytinosauthor=G.+Ciossaniauthor=O.+A.+Botrugnoauthor=F.+Fornerisauthor=M.+Tardugnoauthor=D.+E.+Edmondsonauthor=S.+Minucciauthor=A.+Matteviauthor=A.+Mai&title=Biochemical%2C+structural%2C+and+biological+evaluation+of+tranylcypromine+derivatives+as+inhibitors+of+histone+demethylases+LSD1+and+LSD2&doi=10.1021%2Fja101557k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Biochemical, Structural, and Biological Evaluation of Tranylcypromine Derivatives as Inhibitors of Histone Demethylases LSD1 and LSD2</span></div><div class="casAuthors">Binda, Claudia; Valente, Sergio; Romanenghi, Mauro; Pilotto, Simona; Cirilli, Roberto; Karytinos, Aristotele; Ciossani, Giuseppe; Botrugno, Oronza A.; Forneris, Federico; Tardugno, Maria; Edmondson, Dale E.; Minucci, Saverio; Mattevi, Andrea; Mai, Antonello</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">132</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">6827-6833</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">LSD1 and LSD2 histone demethylases are implicated in a no. of physiol. and pathol. processes, ranging from tumorigenesis to herpes virus infection.  A comprehensive structural, biochem., and cellular study is presented here to probe the potential of these enzymes for epigenetic therapies.  This approach employs tranylcypromine as a chem. scaffold for the design of novel demethylase inhibitors.  This drug is a clin. validated antidepressant known to target monoamine oxidases A and B.  These two flavoenzymes are structurally related to LSD1 and LSD2.  Mechanistic and crystallog. studies of tranylcypromine inhibition reveal a lack of selectivity and differing covalent modifications of the FAD cofactor depending on the enantiomeric form.  These findings are pharmacol. relevant, since tranylcypromine is currently administered as a racemic mixt.  A large set of tranylcypromine analogs were synthesized and screened for inhibitory activities.  We found that the common evolutionary origin of LSD and MAO enzymes, despite their unrelated functions and substrate specificities, is reflected in related ligand-binding properties.  A few compds. with partial enzyme selectivity were identified.  The biol. activity of one of these new inhibitors was evaluated with a cellular model of acute promyelocytic leukemia chosen since its pathogenesis includes aberrant activities of several chromatin modifiers.  Marked effects on cell differentiation and an unprecedented synergistic activity with antileukemia drugs were obsd.  These data demonstrate that these LSD1/2 inhibitors are of potential relevance for the treatment of promyelocytic leukemia and, more generally, as tools to alter chromatin state with promise of a block of tumor progression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqH4F9uwn5Jx7Vg90H21EOLACvtfcHk0lj0K_9Wzdk49g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXltFChur0%253D&md5=d6b28616bbb0455e139625fdb43696d3</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fja101557k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja101557k%26sid%3Dliteratum%253Aachs%26aulast%3DBinda%26aufirst%3DC.%26aulast%3DValente%26aufirst%3DS.%26aulast%3DRomanenghi%26aufirst%3DM.%26aulast%3DPilotto%26aufirst%3DS.%26aulast%3DCirilli%26aufirst%3DR.%26aulast%3DKarytinos%26aufirst%3DA.%26aulast%3DCiossani%26aufirst%3DG.%26aulast%3DBotrugno%26aufirst%3DO.%2BA.%26aulast%3DForneris%26aufirst%3DF.%26aulast%3DTardugno%26aufirst%3DM.%26aulast%3DEdmondson%26aufirst%3DD.%2BE.%26aulast%3DMinucci%26aufirst%3DS.%26aulast%3DMattevi%26aufirst%3DA.%26aulast%3DMai%26aufirst%3DA.%26atitle%3DBiochemical%252C%2520structural%252C%2520and%2520biological%2520evaluation%2520of%2520tranylcypromine%2520derivatives%2520as%2520inhibitors%2520of%2520histone%2520demethylases%2520LSD1%2520and%2520LSD2%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2010%26volume%3D132%26spage%3D6827%26epage%3D6833%26doi%3D10.1021%2Fja101557k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="note"><p class="first last">See recent reviews and the references therein.</p></div><div class="NLM_citation" id="cit26a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Suzuki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyata, N.</span><span> </span><span class="NLM_article-title">Lysine demethylases inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">8236</span><span class="NLM_x">–</span> <span class="NLM_lpage">8250</span><span class="refDoi"> DOI: 10.1021/jm201048w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm201048w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1OrtrnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=8236-8250&author=T.+Suzukiauthor=N.+Miyata&title=Lysine+demethylases+inhibitors&doi=10.1021%2Fjm201048w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26aR"><div class="casContent"><span class="casTitleNuber">26a</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine Demethylases Inhibitors</span></div><div class="casAuthors">Suzuki, Takayoshi; Miyata, Naoki</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">8236-8250</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review, on the use of lysine demethylases inhibitors in therapy of various diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplp9MjSdvnn7Vg90H21EOLACvtfcHk0lj0K_9Wzdk49g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1OrtrnF&md5=194d4bbc2e52672f5d5f019de0bb760c</span></div><a href="/servlet/linkout?suffix=cit26a&amp;dbid=16384&amp;doi=10.1021%2Fjm201048w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm201048w%26sid%3Dliteratum%253Aachs%26aulast%3DSuzuki%26aufirst%3DT.%26aulast%3DMiyata%26aufirst%3DN.%26atitle%3DLysine%2520demethylases%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D8236%26epage%3D8250%26doi%3D10.1021%2Fjm201048w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit26b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Mould, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McGonagle, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wiseman, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, E. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jordan, A. M.</span><span> </span><span class="NLM_article-title">Reversible inhibitors of LSD1 as therapeutic agents in acute myeloid leukemia: clinical significance and progress to date</span> <span class="citation_source-journal">Med. Res. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">35</span><span class="NLM_x">, </span> <span class="NLM_fpage">586</span><span class="NLM_x">–</span> <span class="NLM_lpage">618</span><span class="refDoi"> DOI: 10.1002/med.21334</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1002%2Fmed.21334" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=25418875" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlvFSju7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2015&pages=586-618&author=D.+P.+Mouldauthor=A.+E.+McGonagleauthor=D.+H.+Wisemanauthor=E.+L.+Williamsauthor=A.+M.+Jordan&title=Reversible+inhibitors+of+LSD1+as+therapeutic+agents+in+acute+myeloid+leukemia%3A+clinical+significance+and+progress+to+date&doi=10.1002%2Fmed.21334"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26bR"><div class="casContent"><span class="casTitleNuber">26b</span><div class="casTitle"><span class="NLM_cas:atitle">Reversible Inhibitors of LSD1 as Therapeutic Agents in Acute Myeloid Leukemia: Clinical Significance and Progress to Date</span></div><div class="casAuthors">Mould, Daniel P.; McGonagle, Alison E.; Wiseman, Daniel H.; Williams, Emma L.; Jordan, Allan M.</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">586-618</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">In the 10 years since the discovery of lysine-specific demethylase 1 (LSD1), this epigenetic eraser has emerged as an important target of interest in oncol.  More specifically, research has demonstrated that it plays an essential role in the self-renewal of leukemic stem cells in acute myeloid leukemia (AML).  This review will cover clin. aspects of AML, the role of epigenetics in the disease, and discuss the research that led to the first irreversible inhibitors of LSD1 entering clin. trials for the treatment of AML in 2014.  We also review recent achievements and progress in the development of potent and selective reversible inhibitors of LSD1.  These compds. differ in their mode of action from tranylcypromine derivs. and could facilitate novel biochem. studies to probe the pathways mediated by LSD1.  In this review, we will critically evaluate the strengths and weaknesses of published series of reversible LSD1 inhibitors.  Overall, while the development of reversible inhibitors to date has been less fruitful than that of irreversible inhibitors, there is still the possibility for their use to facilitate further research into the roles and functions of LSD1 and to expand the therapeutic applications of LSD1 inhibitors in the clinic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTFqxWxhZyUrVg90H21EOLACvtfcHk0lj0K_9Wzdk49g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlvFSju7c%253D&md5=b669a61231659817709013572e8fdcf7</span></div><a href="/servlet/linkout?suffix=cit26b&amp;dbid=16384&amp;doi=10.1002%2Fmed.21334&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21334%26sid%3Dliteratum%253Aachs%26aulast%3DMould%26aufirst%3DD.%2BP.%26aulast%3DMcGonagle%26aufirst%3DA.%2BE.%26aulast%3DWiseman%26aufirst%3DD.%2BH.%26aulast%3DWilliams%26aufirst%3DE.%2BL.%26aulast%3DJordan%26aufirst%3DA.%2BM.%26atitle%3DReversible%2520inhibitors%2520of%2520LSD1%2520as%2520therapeutic%2520agents%2520in%2520acute%2520myeloid%2520leukemia%253A%2520clinical%2520significance%2520and%2520progress%2520to%2520date%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2015%26volume%3D35%26spage%3D586%26epage%3D618%26doi%3D10.1002%2Fmed.21334" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="note"><p class="first last">For more recent LSD1 inhibitors, see the following:</p></div><div class="NLM_citation" id="cit27a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Neelamegam, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ricq, E. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Malvaez, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patnaik, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norton, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carlin, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hill, I. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wood, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haggarty, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hooker, J. M.</span><span> </span><span class="NLM_article-title">Brain-penetrant LSD1 inhibitors can block memory consolidation</span> <span class="citation_source-journal">ACS Chem. Neurosci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">120</span><span class="NLM_x">–</span> <span class="NLM_lpage">128</span><span class="refDoi"> DOI: 10.1021/cn200104y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cn200104y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVyrs77E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2012&pages=120-128&author=R.+Neelamegamauthor=E.+L.+Ricqauthor=M.+Malvaezauthor=D.+Patnaikauthor=S.+Nortonauthor=S.+M.+Carlinauthor=I.+T.+Hillauthor=M.+A.+Woodauthor=S.+J.+Haggartyauthor=J.+M.+Hooker&title=Brain-penetrant+LSD1+inhibitors+can+block+memory+consolidation&doi=10.1021%2Fcn200104y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27aR"><div class="casContent"><span class="casTitleNuber">27a</span><div class="casTitle"><span class="NLM_cas:atitle">Brain-Penetrant LSD1 Inhibitors Can Block Memory Consolidation</span></div><div class="casAuthors">Neelamegam, Ramesh; Ricq, Emily L.; Malvaez, Melissa; Patnaik, Debasis; Norton, Stephanie; Carlin, Stephen M.; Hill, Ian T.; Wood, Marcelo A.; Haggarty, Stephen J.; Hooker, Jacob M.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Neuroscience</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">120-128</span>CODEN:
                <span class="NLM_cas:coden">ACNCDM</span>;
        ISSN:<span class="NLM_cas:issn">1948-7193</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Modulation of histone modifications in the brain may represent a new mechanism for brain disorder therapy.  Post-translational modifications of histones regulate gene expression, affecting major cellular processes such as proliferation, differentiation, and function.  An important enzyme involved in one of these histone modifications is lysine specific demethylase 1 (LSD1).  This enzyme is flavin-dependent and exhibits homol. to amine oxidases.  Parnate (2-phenylcyclopropylamine (2-PCPA); tranylcypromine) is a potent inhibitor of monoamine oxidases, and derivs. of 2-PCPA have been used for development of selective LSD1 inhibitors based on the ability to form covalent adducts with FAD.  Here we report the synthesis and in vitro characterization of LSD1 inhibitors that bond covalently to FAD.  The two most potent and selective inhibitors were used to demonstrate brain penetration when administered systemically to rodents.  First, radiosynthesis of a positron-emitting analog was used to obtain preliminary biodistribution data and whole brain time-activity curves.  Second, we demonstrate that this series of LSD1 inhibitors is capable of producing a cognitive effect in a mouse model.  By using a memory formation paradigm, novel object recognition, we show that LSD1 inhibition can abolish long-term memory formation without affecting short-term memory, providing further evidence for the importance of reversible histone methylation in the function of the nervous system.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRTx9XGRzjgrVg90H21EOLACvtfcHk0ljbmdaKP2WzKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVyrs77E&md5=cd365810b397d0ece732ea853a3e93da</span></div><a href="/servlet/linkout?suffix=cit27a&amp;dbid=16384&amp;doi=10.1021%2Fcn200104y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcn200104y%26sid%3Dliteratum%253Aachs%26aulast%3DNeelamegam%26aufirst%3DR.%26aulast%3DRicq%26aufirst%3DE.%2BL.%26aulast%3DMalvaez%26aufirst%3DM.%26aulast%3DPatnaik%26aufirst%3DD.%26aulast%3DNorton%26aufirst%3DS.%26aulast%3DCarlin%26aufirst%3DS.%2BM.%26aulast%3DHill%26aufirst%3DI.%2BT.%26aulast%3DWood%26aufirst%3DM.%2BA.%26aulast%3DHaggarty%26aufirst%3DS.%2BJ.%26aulast%3DHooker%26aufirst%3DJ.%2BM.%26atitle%3DBrain-penetrant%2520LSD1%2520inhibitors%2520can%2520block%2520memory%2520consolidation%26jtitle%3DACS%2520Chem.%2520Neurosci.%26date%3D2012%26volume%3D3%26spage%3D120%26epage%3D128%26doi%3D10.1021%2Fcn200104y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit27b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Zheng, Y. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duan, Y. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zi, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lv, W. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ye, X. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mobley, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, Y. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Z. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, H. M.</span><span> </span><span class="NLM_article-title">Triazole-dithiocarbamate based selective lysine specific demethylase 1 (LSD1) inactivators inhibit gastric cancer cell growth, invasion, and migration</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">8543</span><span class="NLM_x">–</span> <span class="NLM_lpage">8560</span><span class="refDoi"> DOI: 10.1021/jm401002r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401002r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1ejt7nL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=8543-8560&author=Y.+C.+Zhengauthor=Y.+C.+Duanauthor=J.+L.+Maauthor=R.+M.+Xuauthor=X.+Ziauthor=W.+L.+Lvauthor=M.+M.+Wangauthor=X.+W.+Yeauthor=S.+Zhuauthor=D.+Mobleyauthor=Y.+Y.+Zhuauthor=J.+W.+Wangauthor=J.+F.+Liauthor=Z.+R.+Wangauthor=W.+Zhaoauthor=H.+M.+Liu&title=Triazole-dithiocarbamate+based+selective+lysine+specific+demethylase+1+%28LSD1%29+inactivators+inhibit+gastric+cancer+cell+growth%2C+invasion%2C+and+migration&doi=10.1021%2Fjm401002r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27bR"><div class="casContent"><span class="casTitleNuber">27b</span><div class="casTitle"><span class="NLM_cas:atitle">Triazole-Dithiocarbamate Based Selective Lysine Specific Demethylase 1 (LSD1) Inactivators Inhibit Gastric Cancer Cell Growth, Invasion, and Migration</span></div><div class="casAuthors">Zheng, Yi-Chao; Duan, Ying-Chao; Ma, Jin-Lian; Xu, Rui-Min; Zi, Xiaolin; Lv, Wen-Lei; Wang, Meng-Meng; Ye, Xian-Wei; Zhu, Shun; Mobley, David; Zhu, Yan-Yan; Wang, Jun-Wei; Li, Jin-Feng; Wang, Zhi-Ru; Zhao, Wen; Liu, Hong-Min</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">8543-8560</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Lysine specific demethylase 1 (LSD1), the first identified histone demethylase, plays an important role in epigenetic regulation of gene activation and repression.  The up-regulated LSD1's expression has been reported in several malignant tumors.  In the current study, we designed and synthesized five series of 1,2,3-triazole-dithiocarbamate hybrids and screened their inhibitory activity toward LSD1.  We found that some of these compds., esp. compd. (I), exhibited the most specific and robust inhibition of LSD1.  Interestingly, compd. I also showed potent and selective cytotoxicity against LSD1 overexpressing gastric cancer cell lines MGC-803 and HGC-27, as well as marked inhibition of cell migration and invasion, compared to 2-PCPA.  Furthermore, compd. I effectively reduced the tumor growth bared by human gastric cancer cells in vivo with no signs of adverse side effects.  These findings suggested that compd. I deserves further investigation as a lead compd. in the treatment of LSD1 overexpressing gastric cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjGdOvultbebVg90H21EOLACvtfcHk0ljbmdaKP2WzKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1ejt7nL&md5=4e4594271ff6b62237ff17a87d6f04d8</span></div><a href="/servlet/linkout?suffix=cit27b&amp;dbid=16384&amp;doi=10.1021%2Fjm401002r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401002r%26sid%3Dliteratum%253Aachs%26aulast%3DZheng%26aufirst%3DY.%2BC.%26aulast%3DDuan%26aufirst%3DY.%2BC.%26aulast%3DMa%26aufirst%3DJ.%2BL.%26aulast%3DXu%26aufirst%3DR.%2BM.%26aulast%3DZi%26aufirst%3DX.%26aulast%3DLv%26aufirst%3DW.%2BL.%26aulast%3DWang%26aufirst%3DM.%2BM.%26aulast%3DYe%26aufirst%3DX.%2BW.%26aulast%3DZhu%26aufirst%3DS.%26aulast%3DMobley%26aufirst%3DD.%26aulast%3DZhu%26aufirst%3DY.%2BY.%26aulast%3DWang%26aufirst%3DJ.%2BW.%26aulast%3DLi%26aufirst%3DJ.%2BF.%26aulast%3DWang%26aufirst%3DZ.%2BR.%26aulast%3DZhao%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DH.%2BM.%26atitle%3DTriazole-dithiocarbamate%2520based%2520selective%2520lysine%2520specific%2520demethylase%25201%2520%2528LSD1%2529%2520inactivators%2520inhibit%2520gastric%2520cancer%2520cell%2520growth%252C%2520invasion%252C%2520and%2520migration%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D8543%26epage%3D8560%26doi%3D10.1021%2Fjm401002r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit27c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Sorna, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Theisen, E. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stephens, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Warner, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bearss, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vankayalapati, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharma, S.</span><span> </span><span class="NLM_article-title">High-throughput virtual screening identifies novel N′-(1-phenylethylidene)-benzohydrazides as potent, specific, and reversible LSD1 inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">9496</span><span class="NLM_x">–</span> <span class="NLM_lpage">9508</span><span class="refDoi"> DOI: 10.1021/jm400870h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400870h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslyks7fF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=9496-9508&author=V.+Sornaauthor=E.+R.+Theisenauthor=B.+Stephensauthor=S.+L.+Warnerauthor=D.+J.+Bearssauthor=H.+Vankayalapatiauthor=S.+Sharma&title=High-throughput+virtual+screening+identifies+novel+N%E2%80%B2-%281-phenylethylidene%29-benzohydrazides+as+potent%2C+specific%2C+and+reversible+LSD1+inhibitors&doi=10.1021%2Fjm400870h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27cR"><div class="casContent"><span class="casTitleNuber">27c</span><div class="casTitle"><span class="NLM_cas:atitle">High-Throughput Virtual Screening Identifies Novel N'-(1-Phenylethylidene)-benzohydrazides as Potent, Specific, and Reversible LSD1 Inhibitors</span></div><div class="casAuthors">Sorna, Venkataswamy; Theisen, Emily R.; Stephens, Bret; Warner, Steven L.; Bearss, David J.; Vankayalapati, Hariprasad; Sharma, Sunil</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">9496-9508</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Lysine specific demethylase 1 (LSD1) plays an important role in regulating histone lysine methylation at residues K4 and K9 on histone H3 and is an attractive therapeutic target in multiple malignancies.  Here we report a structure-based virtual screen of a compd. library contg. ∼2 million small mol. entities.  Computational docking and scoring followed by biochem. screening led to the identification of a novel N'-(1-phenylethylidene)-benzohydrazide series of LSD1 inhibitors with hits showing biochem. IC50s in the 200-400 nM range.  Hit-to-lead optimization and structure-activity relation studies aided in the discovery of compd. (I), with a Ki of 31 nM.  Compd. I is reversible and specific for LSD1 as compared to the monoamine oxidases shows minimal inhibition of CYPs and hERG and inhibits proliferation and survival in several cancer cell lines, including breast and colorectal cancer.  Compd. I may be used to probe LSD1's biol. role in these cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp97P3ef8ur6rVg90H21EOLACvtfcHk0ljbmdaKP2WzKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslyks7fF&md5=47293fd790c5bc9fd944943ff4b1a58a</span></div><a href="/servlet/linkout?suffix=cit27c&amp;dbid=16384&amp;doi=10.1021%2Fjm400870h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400870h%26sid%3Dliteratum%253Aachs%26aulast%3DSorna%26aufirst%3DV.%26aulast%3DTheisen%26aufirst%3DE.%2BR.%26aulast%3DStephens%26aufirst%3DB.%26aulast%3DWarner%26aufirst%3DS.%2BL.%26aulast%3DBearss%26aufirst%3DD.%2BJ.%26aulast%3DVankayalapati%26aufirst%3DH.%26aulast%3DSharma%26aufirst%3DS.%26atitle%3DHigh-throughput%2520virtual%2520screening%2520identifies%2520novel%2520N%25E2%2580%25B2-%25281-phenylethylidene%2529-benzohydrazides%2520as%2520potent%252C%2520specific%252C%2520and%2520reversible%2520LSD1%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D9496%26epage%3D9508%26doi%3D10.1021%2Fjm400870h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit27d"><span><span class="NLM_label">(d) </span><span class="NLM_contrib-group">Prusevich, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kalin, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ming, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Basso, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Givens, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cieniewicz, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hsiao, P. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roberson, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adejola, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Avery, L. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casero, R. A.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taverna, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qian, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tackett, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ratan, R. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDonald, O. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feinberg, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cole, P. A.</span><span> </span><span class="NLM_article-title">A Selective Phenelzine Analogue Inhibitor of Histone Demethylase LSD1</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">1284</span><span class="NLM_x">–</span> <span class="NLM_lpage">1293</span><span class="refDoi"> DOI: 10.1021/cb500018s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb500018s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27d&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BC2cXls1Crtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=1284-1293&author=P.+Prusevichauthor=J.+H.+Kalinauthor=S.+A.+Mingauthor=M.+Bassoauthor=J.+Givensauthor=X.+Liauthor=J.+Huauthor=M.+S.+Taylorauthor=A.+M.+Cieniewiczauthor=P.+Y.+Hsiaoauthor=R.+Huangauthor=H.+Robersonauthor=N.+Adejolaauthor=L.+B.+Averyauthor=R.+A.+Caseroauthor=S.+D.+Tavernaauthor=J.+Qianauthor=A.+J.+Tackettauthor=R.+R.+Ratanauthor=O.+G.+McDonaldauthor=A.+P.+Feinbergauthor=P.+A.+Cole&title=A+Selective+Phenelzine+Analogue+Inhibitor+of+Histone+Demethylase+LSD1&doi=10.1021%2Fcb500018s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27dR"><div class="casContent"><span class="casTitleNuber">27d</span><div class="casTitle"><span class="NLM_cas:atitle">A Selective Phenelzine Analogue Inhibitor of Histone Demethylase LSD1</span></div><div class="casAuthors">Prusevich, Polina; Kalin, Jay H.; Ming, Shonoi A.; Basso, Manuela; Givens, Jeffrey; Li, Xin; Hu, Jianfei; Taylor, Martin S.; Cieniewicz, Anne M.; Hsiao, Po-Yuan; Huang, Rong; Roberson, Heather; Adejola, Nkosi; Avery, Lindsay B.; Casero, Robert A., Jr.; Taverna, Sean D.; Qian, Jiang; Tackett, Alan J.; Ratan, Rajiv R.; McDonald, Oliver G.; Feinberg, Andrew P.; Cole, Philip A.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1284-1293</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Lysine-specific demethylase 1 (LSD1) is an epigenetic enzyme that oxidatively cleaves Me groups from monomethyl and di-Me Lys 4 of histone H3 (H3K4Me1, H3K4Me2) and can contribute to gene silencing.  This study describes the design and synthesis of analogs of a monoamine oxidase antidepressant, phenelzine, and their LSD1 inhibitory properties.  A novel phenelzine analog (bizine) contg. a phenyl-butyrylamide appendage was shown to be a potent LSD1 inhibitor in vitro and was selective vs. monoamine oxidases A/B and the LSD1 homolog, LSD2.  Bizine was found to be effective at modulating bulk histone methylation in cancer cells, and ChIP-seq expts. revealed a statistically significant overlap in the H3K4 methylation pattern of genes affected by bizine and those altered in LSD1-/- cells.  Treatment of two cancer cell lines, LNCaP and H460, with bizine conferred a redn. in proliferation rate, and bizine showed additive to synergistic effects on cell growth when used in combination with two out of five HDAC inhibitors tested.  Moreover, neurons exposed to oxidative stress were protected by the presence of bizine, suggesting potential applications in neurodegenerative disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2Z1w5uKT7qbVg90H21EOLACvtfcHk0lhgXOB_txHJgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXls1Crtr0%253D&md5=36c9f3fb9b28579fa3cad41052cf335a</span></div><a href="/servlet/linkout?suffix=cit27d&amp;dbid=16384&amp;doi=10.1021%2Fcb500018s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb500018s%26sid%3Dliteratum%253Aachs%26aulast%3DPrusevich%26aufirst%3DP.%26aulast%3DKalin%26aufirst%3DJ.%2BH.%26aulast%3DMing%26aufirst%3DS.%2BA.%26aulast%3DBasso%26aufirst%3DM.%26aulast%3DGivens%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DTaylor%26aufirst%3DM.%2BS.%26aulast%3DCieniewicz%26aufirst%3DA.%2BM.%26aulast%3DHsiao%26aufirst%3DP.%2BY.%26aulast%3DHuang%26aufirst%3DR.%26aulast%3DRoberson%26aufirst%3DH.%26aulast%3DAdejola%26aufirst%3DN.%26aulast%3DAvery%26aufirst%3DL.%2BB.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26aulast%3DTaverna%26aufirst%3DS.%2BD.%26aulast%3DQian%26aufirst%3DJ.%26aulast%3DTackett%26aufirst%3DA.%2BJ.%26aulast%3DRatan%26aufirst%3DR.%2BR.%26aulast%3DMcDonald%26aufirst%3DO.%2BG.%26aulast%3DFeinberg%26aufirst%3DA.%2BP.%26aulast%3DCole%26aufirst%3DP.%2BA.%26atitle%3DA%2520Selective%2520Phenelzine%2520Analogue%2520Inhibitor%2520of%2520Histone%2520Demethylase%2520LSD1%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2014%26volume%3D9%26spage%3D1284%26epage%3D1293%26doi%3D10.1021%2Fcb500018s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit27e"><span><span class="NLM_label">(e) </span><span class="NLM_contrib-group">Huang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greene, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stewart, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goodwin, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baylin, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woster, P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casero, R. A.</span><span> </span><span class="NLM_article-title">Inhibition of lysine-specific demethylase 1 by polyamine analogues results in reexpression of aberrantly silenced genes</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">104</span><span class="NLM_x">, </span> <span class="NLM_fpage">8023</span><span class="NLM_x">–</span> <span class="NLM_lpage">8028</span><span class="refDoi"> DOI: 10.1073/pnas.0700720104</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27e&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1073%2Fpnas.0700720104" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27e&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=17463086" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27e&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BD2sXmtFCiurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2007&pages=8023-8028&author=Y.+Huangauthor=E.+Greeneauthor=T.+M.+Stewartauthor=A.+C.+Goodwinauthor=S.+B.+Baylinauthor=P.+M.+Wosterauthor=R.+A.+Casero&title=Inhibition+of+lysine-specific+demethylase+1+by+polyamine+analogues+results+in+reexpression+of+aberrantly+silenced+genes&doi=10.1073%2Fpnas.0700720104"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27eR"><div class="casContent"><span class="casTitleNuber">27e</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of lysine-specific demethylase 1 by polyamine analogues results in reexpression of aberrantly silenced genes</span></div><div class="casAuthors">Huang, Yi; Greene, Eriko; Stewart, Tracy Murray; Goodwin, Andrew C.; Baylin, Stephen B.; Woster, Patrick M.; Casero, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8023-8028</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Epigenetic chromatin modification is a major regulator of eukaryotic gene expression, and aberrant epigenetic silencing of gene expression contributes to tumorigenesis.  Histone modifications include acetylation, phosphorylation, and methylation, resulting in a combination of histone marks known collectively as the histone code.  The chromatin marks at a given promoter det., in part, whether specific promoters are in an open/active conformation or closed/repressed conformation.  Dimethyl-lysine 4 histone H3 (H3K4me2) is a transcription-activating chromatin mark at gene promoters, and demethylation of this mark by the lysine-specific demethylase 1 (LSD1), a homolog of polyamine oxidases, may broadly repress gene expression.  We now report that novel biguanide and bisguanidine polyamine analogs are potent inhibitors of LSD1.  These analogs inhibit LSD1 in human colon carcinoma cells and affect a reexpression of multiple, aberrantly silenced genes important in the development of colon cancer, including members of the secreted frizzle-related proteins (SFRPs) and the GATA family of transcription factors.  Furthermore, we demonstrate by chromatin immunopptn. anal. that the reexpression is concurrent with increased H3K4me2 and acetyl-H3K9 marks, decreased H3K9me1 and H3K9me2 repressive marks.  We thus define important new agents for reversing aberrant repression of gene transcription.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoC96zJoYEK67Vg90H21EOLACvtfcHk0lhgXOB_txHJgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXmtFCiurg%253D&md5=4c7863ecb038e3e10c5774cba2c28a45</span></div><a href="/servlet/linkout?suffix=cit27e&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0700720104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0700720104%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DGreene%26aufirst%3DE.%26aulast%3DStewart%26aufirst%3DT.%2BM.%26aulast%3DGoodwin%26aufirst%3DA.%2BC.%26aulast%3DBaylin%26aufirst%3DS.%2BB.%26aulast%3DWoster%26aufirst%3DP.%2BM.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26atitle%3DInhibition%2520of%2520lysine-specific%2520demethylase%25201%2520by%2520polyamine%2520analogues%2520results%2520in%2520reexpression%2520of%2520aberrantly%2520silenced%2520genes%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2007%26volume%3D104%26spage%3D8023%26epage%3D8028%26doi%3D10.1073%2Fpnas.0700720104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit27f"><span><span class="NLM_label">(f) </span><span class="NLM_contrib-group">Huang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stewart, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baylin, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marton, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perkins, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woster, P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casero, R. A.,  Jr.</span><span> </span><span class="NLM_article-title">Novel oligoamine analogues inhibit lysine-specific demethylase 1 and induce reexpression of epigenetically silenced genes</span> <span class="citation_source-journal">Clin. Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">7217</span><span class="NLM_x">–</span> <span class="NLM_lpage">7228</span><span class="refDoi"> DOI: 10.1158/1078-0432.CCR-09-1293</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27f&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1158%2F1078-0432.CCR-09-1293" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27f&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=19934284" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27f&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFSnu7zE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2009&pages=7217-7228&author=Y.+Huangauthor=T.+M.+Stewartauthor=Y.+Wuauthor=S.+B.+Baylinauthor=L.+J.+Martonauthor=B.+Perkinsauthor=R.+J.+Jonesauthor=P.+M.+Wosterauthor=R.+A.+Casero&title=Novel+oligoamine+analogues+inhibit+lysine-specific+demethylase+1+and+induce+reexpression+of+epigenetically+silenced+genes&doi=10.1158%2F1078-0432.CCR-09-1293"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27fR"><div class="casContent"><span class="casTitleNuber">27f</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Oligoamine Analogues Inhibit Lysine-Specific Demethylase 1 and Induce Reexpression of Epigenetically Silenced Genes</span></div><div class="casAuthors">Huang, Yi; Stewart, Tracy Murray; Wu, Yu; Baylin, Stephen B.; Marton, Laurence J.; Perkins, Brandy; Jones, Richard J.; Woster, Patrick M.; Casero, Robert A., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">7217-7228</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: Abnormal DNA CpG island hypermethylation and transcriptionally repressive histone modifications are assocd. with the aberrant silencing of tumor suppressor genes.  Lysine methylation is a dynamic, enzymically controlled process.  Lysine-specific demethylase 1 (LSD1) has recently been identified as a histone lysine demethylase.  LSD1 specifically catalyzes demethylation of mono- and dimethyl-lysine 4 of histone 3 (H3K4), key pos. chromatin marks assocd. with transcriptional activation.  We hypothesized that a novel class of oligoamine analogs would effectively inhibit LSD1 and thus cause the reexpression of aberrantly silenced genes.  Exptl. Design: Human colorectal cancer cells were treated with the oligoamines and changes in mono- and dimethyl-H3K4 and other chromatin marks were monitored.  In addn., treated cells were evaluated for the reexpression of the aberrantly silenced secreted frizzled-related proteins (SFRP) Wnt signaling pathway antagonist genes.  Finally, the effects of the LSD1 inhibitors were evaluated in an in vivo xenograft model.  Results: Treatment of HCT116 human colon adenocarcinoma cells in vitro resulted in increased H3K4 methylation and reexpression of silenced SFRP genes.  This reexpression is also accompanied by a decrease in H3K9me2 repressive mark.  Importantly, cotreatment with low doses of oligoamines and a DNA methyltransferase inhibitor highly induces the reexpression of the aberrantly silenced SFRP2 gene and results in significant inhibition of the growth of established tumors in a human colon tumor model in vivo.  Conclusions: The use of LSD1-inhibiting oligoamine analogs in combination with DNA methyltransferase inhibitors represents a highly promising and novel approach for epigenetic therapy of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrk1naS7xW5GbVg90H21EOLACvtfcHk0ljWSKnhBknQbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFSnu7zE&md5=0f86cd2d3482aa0a36adacc4f10329dd</span></div><a href="/servlet/linkout?suffix=cit27f&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-09-1293&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-09-1293%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DStewart%26aufirst%3DT.%2BM.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DBaylin%26aufirst%3DS.%2BB.%26aulast%3DMarton%26aufirst%3DL.%2BJ.%26aulast%3DPerkins%26aufirst%3DB.%26aulast%3DJones%26aufirst%3DR.%2BJ.%26aulast%3DWoster%26aufirst%3DP.%2BM.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26atitle%3DNovel%2520oligoamine%2520analogues%2520inhibit%2520lysine-specific%2520demethylase%25201%2520and%2520induce%2520reexpression%2520of%2520epigenetically%2520silenced%2520genes%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2009%26volume%3D15%26spage%3D7217%26epage%3D7228%26doi%3D10.1158%2F1078-0432.CCR-09-1293" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit27g"><span><span class="NLM_label">(g) </span><span class="NLM_contrib-group">Sharma, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steinbergs, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crowley, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hanson, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casero, R. A.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woster, P. M.</span><span> </span><span class="NLM_article-title">(Bis)urea and (Bis)thiourea Inhibitors of Lysine-Specific Demethylase 1 as Epigenetic Modulators</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">5197</span><span class="NLM_x">–</span> <span class="NLM_lpage">5212</span><span class="refDoi"> DOI: 10.1021/jm100217a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm100217a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27g&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BC3cXnvVKqsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=5197-5212&author=S.+K.+Sharmaauthor=Y.+Wuauthor=N.+Steinbergsauthor=M.+L.+Crowleyauthor=A.+S.+Hansonauthor=R.+A.+Caseroauthor=P.+M.+Woster&title=%28Bis%29urea+and+%28Bis%29thiourea+Inhibitors+of+Lysine-Specific+Demethylase+1+as+Epigenetic+Modulators&doi=10.1021%2Fjm100217a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27gR"><div class="casContent"><span class="casTitleNuber">27g</span><div class="casTitle"><span class="NLM_cas:atitle">(Bis)urea and (Bis)thiourea Inhibitors of Lysine-Specific Demethylase 1 as Epigenetic Modulators</span></div><div class="casAuthors">Sharma, Shiv K.; Wu, Yu; Steinbergs, Nora; Crowley, Michael L.; Hanson, Allison S.; Casero, Robert A., Jr.; Woster, Patrick M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">5197-5212</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The recently discovered enzyme lysine-specific demethylase 1 (LSD1) plays an important role in the epigenetic control of gene expression, and aberrant gene silencing secondary to LSD1 overexpression is thought to contribute to the development of cancer.  We recently reported a series of (bis)guanidines and (bis)biguanides that are potent inhibitors of LSD1 and induce the re-expression of aberrantly silenced tumor suppressor genes in tumor cells in vitro.  We now report a series of isosteric ureas and thioureas that are also potent inhibitors of LSD1.  These compds. induce increases in methylation at the histone 3 lysine 4 (H3K4) chromatin mark, a specific target of LSD1, in Calu-6 lung carcinoma cells.  In addn., these analogs increase cellular levels of secreted frizzle-related proteins (SFRP) 2 and 5 and transcription factor GATA4.  These compds. represent an important new series of epigenetic modulators with the potential for use as antitumor agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruUR48vA8XgrVg90H21EOLACvtfcHk0ljWSKnhBknQbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXnvVKqsbo%253D&md5=4e19e747331276e3682f7a2d02f73b47</span></div><a href="/servlet/linkout?suffix=cit27g&amp;dbid=16384&amp;doi=10.1021%2Fjm100217a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100217a%26sid%3Dliteratum%253Aachs%26aulast%3DSharma%26aufirst%3DS.%2BK.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DSteinbergs%26aufirst%3DN.%26aulast%3DCrowley%26aufirst%3DM.%2BL.%26aulast%3DHanson%26aufirst%3DA.%2BS.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26aulast%3DWoster%26aufirst%3DP.%2BM.%26atitle%3D%2528Bis%2529urea%2520and%2520%2528Bis%2529thiourea%2520Inhibitors%2520of%2520Lysine-Specific%2520Demethylase%25201%2520as%2520Epigenetic%2520Modulators%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D5197%26epage%3D5212%26doi%3D10.1021%2Fjm100217a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit27h"><span><span class="NLM_label">(h) </span><span class="NLM_contrib-group">Kutz, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holshouser, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marrow, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woster, P. M.</span><span> </span><span class="NLM_article-title">3,5-Diamino-1,2,4-triazoles as a novel scaffold for potent, reversible LSD1 (KDM1A) inhibitors</span> <span class="citation_source-journal">MedChemComm</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">1863</span><span class="NLM_x">–</span> <span class="NLM_lpage">1870</span><span class="refDoi"> DOI: 10.1039/C4MD00283K</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27h&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1039%2FC4MD00283K" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27h&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=25580204" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27h&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVSqtL7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=1863-1870&author=C.+J.+Kutzauthor=S.+L.+Holshouserauthor=E.+A.+Marrowauthor=P.+M.+Woster&title=3%2C5-Diamino-1%2C2%2C4-triazoles+as+a+novel+scaffold+for+potent%2C+reversible+LSD1+%28KDM1A%29+inhibitors&doi=10.1039%2FC4MD00283K"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27hR"><div class="casContent"><span class="casTitleNuber">27h</span><div class="casTitle"><span class="NLM_cas:atitle">3,5-Diamino-1,2,4-triazoles as a novel scaffold for potent, reversible LSD1 (KDM1A) inhibitors</span></div><div class="casAuthors">Kutz, Craig J.; Holshouser, Steven L.; Marrow, Ethan A.; Woster, Patrick M.</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1863-1870</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The chromatin remodeling amine oxidase lysine-specific demethylase 1 (LSD1) has become an attractive target for the design of specific inhibitors with therapeutic potential.  We, and others, have described LSD1 inhibitors that have potential as antitumor agents.  Many of the currently known LSD1 inhibitors are poor drug candidates, or are structurally based on the tranylcypromine backbone, thus increasing the potential for off-target effects mediated by other amine oxidases.  We now describe a series of potent LSD1 inhibitors based on a novel 1,2,4-triazole scaffold; these inhibitors show a high degree of specificity for LSD1 in vitro, and cause increases in cellular histone 3 dimethyllysine 4 (H3K4me2), a gene transcription activating mark.  Importantly, these inhibitors exhibit low toxicity towards mammalian cells in vitro, and thus they may show utility in the treatment of epigenetically-based diseases where cell death is not a desired endpoint.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXpwx29ujiWbVg90H21EOLACvtfcHk0ljWSKnhBknQbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVSqtL7N&md5=3a6314468bdb665cb30ae86505e5eb15</span></div><a href="/servlet/linkout?suffix=cit27h&amp;dbid=16384&amp;doi=10.1039%2FC4MD00283K&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC4MD00283K%26sid%3Dliteratum%253Aachs%26aulast%3DKutz%26aufirst%3DC.%2BJ.%26aulast%3DHolshouser%26aufirst%3DS.%2BL.%26aulast%3DMarrow%26aufirst%3DE.%2BA.%26aulast%3DWoster%26aufirst%3DP.%2BM.%26atitle%3D3%252C5-Diamino-1%252C2%252C4-triazoles%2520as%2520a%2520novel%2520scaffold%2520for%2520potent%252C%2520reversible%2520LSD1%2520%2528KDM1A%2529%2520inhibitors%26jtitle%3DMedChemComm%26date%3D2014%26volume%3D5%26spage%3D1863%26epage%3D1870%26doi%3D10.1039%2FC4MD00283K" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit27i"><span><span class="NLM_label">(i) </span><span class="NLM_contrib-group">Hitchin, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blagg, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burke, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burns, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cockerill, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fairweather, E. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hutton, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jordan, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McAndrew, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mirza, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mould, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomson, G. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waddell, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ogilvie, D. J.</span><span> </span><span class="NLM_article-title">Development and evaluation of selective, reversible LSD1 inhibitors derived from fragments</span> <span class="citation_source-journal">MedChemComm</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">1513</span><span class="NLM_x">–</span> <span class="NLM_lpage">1522</span><span class="refDoi"> DOI: 10.1039/c3md00226h</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27i&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1039%2Fc3md00226h" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27i&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1OrsrvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=1513-1522&author=J.+R.+Hitchinauthor=J.+Blaggauthor=R.+Burkeauthor=S.+Burnsauthor=M.+J.+Cockerillauthor=E.+E.+Fairweatherauthor=C.+Huttonauthor=A.+M.+Jordanauthor=C.+McAndrewauthor=A.+Mirzaauthor=D.+Mouldauthor=G.+T.+Thomsonauthor=I.+Waddellauthor=D.+J.+Ogilvie&title=Development+and+evaluation+of+selective%2C+reversible+LSD1+inhibitors+derived+from+fragments&doi=10.1039%2Fc3md00226h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27iR"><div class="casContent"><span class="casTitleNuber">27i</span><div class="casTitle"><span class="NLM_cas:atitle">Development and evaluation of selective, reversible LSD1 inhibitors derived from fragments</span></div><div class="casAuthors">Hitchin, James R.; Blagg, Julian; Burke, Rosemary; Burns, Samantha; Cockerill, Mark J.; Fairweather, Emma E.; Hutton, Colin; Jordan, Allan M.; McAndrew, Craig; Mirza, Amin; Mould, Daniel; Thomson, Graeme J.; Waddell, Ian; Ogilvie, Donald J.</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1513-1522</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Two series of aminothiazoles have been developed as reversible inhibitors of lysine specific demethylase 1 (LSD1) through the expansion of a hit derived from a high concn. biochem. fragment based screen of 2466 compds.  The potency of the initial fragment hit was increased 32-fold through synthesis, with one series of compds. showing clear structure-activity relationships and inhibitory activities in the range of 7 to 187 μM in a biochem. assay.  This series also showed selectivity against the related FAD-dependent enzyme mono-amine oxidase A (MAO-A).  Although a wide range of irreversible inhibitors of LSD1 have been reported with activities in the low nanomolar range, this work represents one of the first reported examples of a reversible small mol. inhibitor of LSD1 with clear SAR and selectivity against MAO-A, and could provide a platform for the development of more potent reversible inhibitors.  Herein, we also report the use of a recently developed cell-based assay for profiling LSD1 inhibitors, and present results on our own compds. as well as a selection of recently described reversible LSD1 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUgrL8DOIuDbVg90H21EOLACvtfcHk0ljWSKnhBknQbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1OrsrvK&md5=bec441bbb94f970eab94b96026db7fbe</span></div><a href="/servlet/linkout?suffix=cit27i&amp;dbid=16384&amp;doi=10.1039%2Fc3md00226h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc3md00226h%26sid%3Dliteratum%253Aachs%26aulast%3DHitchin%26aufirst%3DJ.%2BR.%26aulast%3DBlagg%26aufirst%3DJ.%26aulast%3DBurke%26aufirst%3DR.%26aulast%3DBurns%26aufirst%3DS.%26aulast%3DCockerill%26aufirst%3DM.%2BJ.%26aulast%3DFairweather%26aufirst%3DE.%2BE.%26aulast%3DHutton%26aufirst%3DC.%26aulast%3DJordan%26aufirst%3DA.%2BM.%26aulast%3DMcAndrew%26aufirst%3DC.%26aulast%3DMirza%26aufirst%3DA.%26aulast%3DMould%26aufirst%3DD.%26aulast%3DThomson%26aufirst%3DG.%2BT.%26aulast%3DWaddell%26aufirst%3DI.%26aulast%3DOgilvie%26aufirst%3DD.%2BJ.%26atitle%3DDevelopment%2520and%2520evaluation%2520of%2520selective%252C%2520reversible%2520LSD1%2520inhibitors%2520derived%2520from%2520fragments%26jtitle%3DMedChemComm%26date%3D2013%26volume%3D4%26spage%3D1513%26epage%3D1522%26doi%3D10.1039%2Fc3md00226h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit27j"><span><span class="NLM_label">(j) </span><span class="NLM_contrib-group">Culhane, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yen, P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cole, P. A.</span><span> </span><span class="NLM_article-title">Comparative analysis of small molecules and histone substrate analogues as LSD1 lysine demethylase inhibitors</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">132</span><span class="NLM_x">, </span> <span class="NLM_fpage">3164</span><span class="NLM_x">–</span> <span class="NLM_lpage">3176</span><span class="refDoi"> DOI: 10.1021/ja909996p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja909996p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2010&pages=3164-3176&author=J.+C.+Culhaneauthor=D.+Wangauthor=P.+M.+Yenauthor=P.+A.+Cole&title=Comparative+analysis+of+small+molecules+and+histone+substrate+analogues+as+LSD1+lysine+demethylase+inhibitors&doi=10.1021%2Fja909996p"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27j&amp;dbid=16384&amp;doi=10.1021%2Fja909996p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja909996p%26sid%3Dliteratum%253Aachs%26aulast%3DCulhane%26aufirst%3DJ.%2BC.%26aulast%3DWang%26aufirst%3DD.%26aulast%3DYen%26aufirst%3DP.%2BM.%26aulast%3DCole%26aufirst%3DP.%2BA.%26atitle%3DComparative%2520analysis%2520of%2520small%2520molecules%2520and%2520histone%2520substrate%2520analogues%2520as%2520LSD1%2520lysine%2520demethylase%2520inhibitors%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2010%26volume%3D132%26spage%3D3164%26epage%3D3176%26doi%3D10.1021%2Fja909996p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Kahl, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gullotti, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heukamp, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolf, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friedrichs, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vorreuther, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Solleder, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bastian, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ellinger, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metzger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schule, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buettner, R.</span><span> </span><span class="NLM_article-title">Androgen receptor coactivators lysine-specific histone demethylase 1 and four and a half LIM domain protein 2 predict risk of prostate cancer recurrence</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">11341</span><span class="NLM_x">–</span> <span class="NLM_lpage">11347</span><span class="refDoi"> DOI: 10.1158/0008-5472.CAN-06-1570</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1158%2F0008-5472.CAN-06-1570" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2006&pages=11341-11347&author=P.+Kahlauthor=L.+Gullottiauthor=L.+C.+Heukampauthor=S.+Wolfauthor=N.+Friedrichsauthor=R.+Vorreutherauthor=G.+Sollederauthor=P.+J.+Bastianauthor=J.+Ellingerauthor=E.+Metzgerauthor=R.+Schuleauthor=R.+Buettner&title=Androgen+receptor+coactivators+lysine-specific+histone+demethylase+1+and+four+and+a+half+LIM+domain+protein+2+predict+risk+of+prostate+cancer+recurrence&doi=10.1158%2F0008-5472.CAN-06-1570"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-06-1570&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-06-1570%26sid%3Dliteratum%253Aachs%26aulast%3DKahl%26aufirst%3DP.%26aulast%3DGullotti%26aufirst%3DL.%26aulast%3DHeukamp%26aufirst%3DL.%2BC.%26aulast%3DWolf%26aufirst%3DS.%26aulast%3DFriedrichs%26aufirst%3DN.%26aulast%3DVorreuther%26aufirst%3DR.%26aulast%3DSolleder%26aufirst%3DG.%26aulast%3DBastian%26aufirst%3DP.%2BJ.%26aulast%3DEllinger%26aufirst%3DJ.%26aulast%3DMetzger%26aufirst%3DE.%26aulast%3DSchule%26aufirst%3DR.%26aulast%3DBuettner%26aufirst%3DR.%26atitle%3DAndrogen%2520receptor%2520coactivators%2520lysine-specific%2520histone%2520demethylase%25201%2520and%2520four%2520and%2520a%2520half%2520LIM%2520domain%2520protein%25202%2520predict%2520risk%2520of%2520prostate%2520cancer%2520recurrence%26jtitle%3DCancer%2520Res.%26date%3D2006%26volume%3D66%26spage%3D11341%26epage%3D11347%26doi%3D10.1158%2F0008-5472.CAN-06-1570" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Schulte, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lim, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schramm, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friedrichs, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koster, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Versteeg, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ora, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pajtler, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klein-Hitpass, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhfittig-Kulle, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metzger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schule, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eggert, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buettner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kirfel, J.</span><span> </span><span class="NLM_article-title">Lysine-specific demethylase 1 is strongly expressed in poorly differentiated neuroblastoma: implications for therapy</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">69</span><span class="NLM_x">, </span> <span class="NLM_fpage">2065</span><span class="NLM_x">–</span> <span class="NLM_lpage">2071</span><span class="refDoi"> DOI: 10.1158/0008-5472.CAN-08-1735</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1158%2F0008-5472.CAN-08-1735" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=19223552" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BD1MXisFSruro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2009&pages=2065-2071&author=J.+H.+Schulteauthor=S.+Limauthor=A.+Schrammauthor=N.+Friedrichsauthor=J.+Kosterauthor=R.+Versteegauthor=I.+Oraauthor=K.+Pajtlerauthor=L.+Klein-Hitpassauthor=S.+Kuhfittig-Kulleauthor=E.+Metzgerauthor=R.+Schuleauthor=A.+Eggertauthor=R.+Buettnerauthor=J.+Kirfel&title=Lysine-specific+demethylase+1+is+strongly+expressed+in+poorly+differentiated+neuroblastoma%3A+implications+for+therapy&doi=10.1158%2F0008-5472.CAN-08-1735"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine-specific demethylase 1 is strongly expressed in poorly differentiated neuroblastoma: Implications for therapy</span></div><div class="casAuthors">Schulte, Johannes H.; Lim, Soyoung; Schramm, Alexander; Friedrichs, Nicolaus; Koster, Jan; Versteeg, Rogier; Ora, Ingrid; Pajtler, Kristian; Klein-Hitpass, Ludger; Kuhfittig-Kulle, Steffi; Metzger, Eric; Schuele, Roland; Eggert, Angelika; Buettner, Reinhard; Kirfel, Jutta</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2065-2071</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Aberrant epigenetic changes in DNA methylation and histone acetylation are hallmarks of most cancers, whereas histone methylation was previously considered to be irreversible and less versatile.  Recently, several histone demethylases were identified catalyzing the removal of Me groups from histone H3 lysine residues and thereby influencing gene expression.  Neuroblastomas continue to remain a clin. challenge despite advances in multimodal therapy.  Here, we address the functional significance of the chromatin-modifying enzyme lysine-specific demethylase 1 (LSD1) in neuroblastoma.  LSD1 expression correlated with adverse outcome and was inversely correlated with differentiation in neuroblastic tumors.  Differentiation of neuroblastoma cells resulted in down-regulation of LSD1.  Small interfering RNA-mediated knockdown of LSD1 decreased cellular growth, induced expression of differentiation-assocd. genes, and increased target gene-specific H3K4 methylation.  Moreover, LSD1 inhibition using monoamine oxidase inhibitors resulted in an increase of global H3K4 methylation and growth inhibition of neuroblastoma cells in vitro.  Finally, targeting LSD1 reduced neuroblastoma xenograft growth in vivo.  Here, we provide the first evidence that a histone demethylase, LSD1, is involved in maintaining the undifferentiated, malignant phenotype of neuroblastoma cells.  We show that inhibition of LSD1 reprograms the transcriptome of neuroblastoma cells and inhibits neuroblastoma xenograft growth.  Our results suggest that targeting histone demethylases may provide a novel option for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphjmZreB7Q67Vg90H21EOLACvtfcHk0li74QIa6VsINg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXisFSruro%253D&md5=4118d7f651c8abf8c32e296cc0030656</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-08-1735&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-08-1735%26sid%3Dliteratum%253Aachs%26aulast%3DSchulte%26aufirst%3DJ.%2BH.%26aulast%3DLim%26aufirst%3DS.%26aulast%3DSchramm%26aufirst%3DA.%26aulast%3DFriedrichs%26aufirst%3DN.%26aulast%3DKoster%26aufirst%3DJ.%26aulast%3DVersteeg%26aufirst%3DR.%26aulast%3DOra%26aufirst%3DI.%26aulast%3DPajtler%26aufirst%3DK.%26aulast%3DKlein-Hitpass%26aufirst%3DL.%26aulast%3DKuhfittig-Kulle%26aufirst%3DS.%26aulast%3DMetzger%26aufirst%3DE.%26aulast%3DSchule%26aufirst%3DR.%26aulast%3DEggert%26aufirst%3DA.%26aulast%3DBuettner%26aufirst%3DR.%26aulast%3DKirfel%26aufirst%3DJ.%26atitle%3DLysine-specific%2520demethylase%25201%2520is%2520strongly%2520expressed%2520in%2520poorly%2520differentiated%2520neuroblastoma%253A%2520implications%2520for%2520therapy%26jtitle%3DCancer%2520Res.%26date%3D2009%26volume%3D69%26spage%3D2065%26epage%3D2071%26doi%3D10.1158%2F0008-5472.CAN-08-1735" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Lim, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janzer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Becker, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zimmer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schule, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buettner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kirfel, J.</span><span> </span><span class="NLM_article-title">Lysine-specific demethylase 1 (LSD1) is highly expressed in ER-negative breast cancers and a biomarker predicting aggressive biology</span> <span class="citation_source-journal">Carcinogenesis</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">512</span><span class="NLM_x">–</span> <span class="NLM_lpage">520</span><span class="refDoi"> DOI: 10.1093/carcin/bgp324</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1093%2Fcarcin%2Fbgp324" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=20042638" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BC3cXivFGmsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2010&pages=512-520&author=S.+Limauthor=A.+Janzerauthor=A.+Beckerauthor=A.+Zimmerauthor=R.+Schuleauthor=R.+Buettnerauthor=J.+Kirfel&title=Lysine-specific+demethylase+1+%28LSD1%29+is+highly+expressed+in+ER-negative+breast+cancers+and+a+biomarker+predicting+aggressive+biology&doi=10.1093%2Fcarcin%2Fbgp324"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine-specific demethylase 1 (LSD1) is highly expressed in ER-negative breast cancers and a biomarker predicting aggressive biology</span></div><div class="casAuthors">Lim, Soyoung; Janzer, Andreas; Becker, Astrid; Zimmer, Andreas; Schuele, Roland; Buettner, Reinhard; Kirfel, Jutta</div><div class="citationInfo"><span class="NLM_cas:title">Carcinogenesis</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">512-520</span>CODEN:
                <span class="NLM_cas:coden">CRNGDP</span>;
        ISSN:<span class="NLM_cas:issn">0143-3334</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Breast carcinogenesis is a multistep process involving both genetic and epigenetic changes.  Since epigenetic changes like histone modifications are potentially reversible processes, much effort has been directed toward understanding this mechanism with the goal of finding novel therapies as well as more refined diagnostic and prognostic tools in breast cancer.  Lysine-specific demethylase 1 (LSD1) plays a key role in the regulation of gene expression by removing the Me groups from methylated lysine 4 of histone H3 and lysine 9 of histone H3.  LSD1 is essential for mammalian development and involved in many biol. processes.  Considering recent evidence that LSD1 is involved in carcinogenesis, we investigated the role of LSD1 in breast cancer.  Therefore, we developed an ELISA to det. LSD1 protein levels in tissue specimens of breast cancer and measured very high LSD1 levels in estrogen receptor (ER)-neg. tumors.  Pharmacol. LSD1 inhibition resulted in growth inhibition of breast cancer cells.  Knockdown of LSD1 using small interfering RNA approach induced regulation of several proliferation-assocd. genes like p21, ERBB2 and CCNA2.  Addnl., we found that LSD1 is recruited to the promoters of these genes.  In summary, our data indicate that LSD1 may provide a predictive marker for aggressive biol. and a novel attractive therapeutic target for treatment of ER-neg. breast cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoeX6QvvaygFLVg90H21EOLACvtfcHk0li74QIa6VsINg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXivFGmsrw%253D&md5=b94c60b52c6a440d71cdf5a8007ea97b</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1093%2Fcarcin%2Fbgp324&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fcarcin%252Fbgp324%26sid%3Dliteratum%253Aachs%26aulast%3DLim%26aufirst%3DS.%26aulast%3DJanzer%26aufirst%3DA.%26aulast%3DBecker%26aufirst%3DA.%26aulast%3DZimmer%26aufirst%3DA.%26aulast%3DSchule%26aufirst%3DR.%26aulast%3DBuettner%26aufirst%3DR.%26aulast%3DKirfel%26aufirst%3DJ.%26atitle%3DLysine-specific%2520demethylase%25201%2520%2528LSD1%2529%2520is%2520highly%2520expressed%2520in%2520ER-negative%2520breast%2520cancers%2520and%2520a%2520biomarker%2520predicting%2520aggressive%2520biology%26jtitle%3DCarcinogenesis%26date%3D2010%26volume%3D31%26spage%3D512%26epage%3D520%26doi%3D10.1093%2Fcarcin%2Fbgp324" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Harris, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lynch, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spencer, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hitchin, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciceri, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blaser, J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greystoke, B. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jordan, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ogilvie, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Somervaille, T. C.</span><span> </span><span class="NLM_article-title">The histone demethylase KDM1A sustains the oncogenic potential of MLL-AF9 leukemia stem cells</span> <span class="citation_source-journal">Cancer Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">473</span><span class="NLM_x">–</span> <span class="NLM_lpage">487</span><span class="refDoi"> DOI: 10.1016/j.ccr.2012.03.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1016%2Fj.ccr.2012.03.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=22464800" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BC38XkvFyqsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2012&pages=473-487&author=W.+J.+Harrisauthor=X.+Huangauthor=J.+T.+Lynchauthor=G.+J.+Spencerauthor=J.+R.+Hitchinauthor=Y.+Liauthor=F.+Ciceriauthor=J.+G.+Blaserauthor=B.+F.+Greystokeauthor=A.+M.+Jordanauthor=C.+J.+Millerauthor=D.+J.+Ogilvieauthor=T.+C.+Somervaille&title=The+histone+demethylase+KDM1A+sustains+the+oncogenic+potential+of+MLL-AF9+leukemia+stem+cells&doi=10.1016%2Fj.ccr.2012.03.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">The Histone Demethylase KDM1A Sustains the Oncogenic Potential of MLL-AF9 Leukemia Stem Cells</span></div><div class="casAuthors">Harris, William J.; Huang, Xu; Lynch, James T.; Spencer, Gary J.; Hitchin, James R.; Li, Yaoyong; Ciceri, Filippo; Blaser, Julian G.; Greystoke, Brigit F.; Jordan, Allan M.; Miller, Crispin J.; Ogilvie, Donald J.; Somervaille, Tim C. P.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">473-487</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Using a mouse model of human MLL-AF9 leukemia, we identified the lysine-specific demethylase KDM1A (LSD1 or AOF2) as an essential regulator of leukemia stem cell (LSC) potential.  KDM1A acts at genomic loci bound by MLL-AF9 to sustain expression of the assocd. oncogenic program, thus preventing differentiation and apoptosis.  In vitro and in vivo pharmacol. targeting of KDM1A using tranylcypromine analogs active in the nanomolar range phenocopied Kdm1a knockdown in both murine and primary human AML cells exhibiting MLL translocations.  By contrast, the clonogenic and repopulating potential of normal hematopoietic stem and progenitor cells was spared.  These data establish KDM1A as a key effector of the differentiation block in MLL leukemia, which may be selectively targeted to therapeutic effect.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiZ65foej58rVg90H21EOLACvtfcHk0lik2DF6yUAiAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkvFyqsrc%253D&md5=3227f2ac24a884a8ebadbce71198830c</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2012.03.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2012.03.014%26sid%3Dliteratum%253Aachs%26aulast%3DHarris%26aufirst%3DW.%2BJ.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DLynch%26aufirst%3DJ.%2BT.%26aulast%3DSpencer%26aufirst%3DG.%2BJ.%26aulast%3DHitchin%26aufirst%3DJ.%2BR.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DCiceri%26aufirst%3DF.%26aulast%3DBlaser%26aufirst%3DJ.%2BG.%26aulast%3DGreystoke%26aufirst%3DB.%2BF.%26aulast%3DJordan%26aufirst%3DA.%2BM.%26aulast%3DMiller%26aufirst%3DC.%2BJ.%26aulast%3DOgilvie%26aufirst%3DD.%2BJ.%26aulast%3DSomervaille%26aufirst%3DT.%2BC.%26atitle%3DThe%2520histone%2520demethylase%2520KDM1A%2520sustains%2520the%2520oncogenic%2520potential%2520of%2520MLL-AF9%2520leukemia%2520stem%2520cells%26jtitle%3DCancer%2520Cell%26date%3D2012%26volume%3D21%26spage%3D473%26epage%3D487%26doi%3D10.1016%2Fj.ccr.2012.03.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Dent, S. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chandra, J.</span><span> </span><span class="NLM_article-title">The lasting influence of LSD1 in the blood</span> <span class="citation_source-journal">eLife</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">e00963</span><span class="refDoi"> DOI: 10.7554/eLife.00963</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.7554%2FeLife.00963" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2013&pages=e00963&author=S.+Y.+Dentauthor=J.+Chandra&title=The+lasting+influence+of+LSD1+in+the+blood&doi=10.7554%2FeLife.00963"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.7554%2FeLife.00963&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.00963%26sid%3Dliteratum%253Aachs%26aulast%3DDent%26aufirst%3DS.%2BY.%26aulast%3DChandra%26aufirst%3DJ.%26atitle%3DThe%2520lasting%2520influence%2520of%2520LSD1%2520in%2520the%2520blood%26jtitle%3DeLife%26date%3D2013%26volume%3D2%26spage%3De00963%26doi%3D10.7554%2FeLife.00963" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Lynch, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harris, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Somervaille, T. C.</span><span> </span><span class="NLM_article-title">LSD1 inhibition: a therapeutic strategy in cancer?</span> <span class="citation_source-journal">Expert Opin. Ther. Targets</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">1239</span><span class="NLM_x">–</span> <span class="NLM_lpage">1249</span><span class="refDoi"> DOI: 10.1517/14728222.2012.722206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1517%2F14728222.2012.722206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=22957941" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs12jtrzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2012&pages=1239-1249&author=J.+T.+Lynchauthor=W.+J.+Harrisauthor=T.+C.+Somervaille&title=LSD1+inhibition%3A+a+therapeutic+strategy+in+cancer%3F&doi=10.1517%2F14728222.2012.722206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">LSD1 inhibition: a therapeutic strategy in cancer?</span></div><div class="casAuthors">Lynch, James T.; Harris, William J.; Somervaille, Tim C. P.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Targets</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1239-1249</span>CODEN:
                <span class="NLM_cas:coden">EOTTAO</span>;
        ISSN:<span class="NLM_cas:issn">1472-8222</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Introduction: The role of epigenetic dysfunction in cancer is increasingly appreciated.  This has raised the question as to whether enzymes that regulate the structure and function of chromatin might represent novel therapeutic targets.  The histone demethylase LSD1 is one such candidate and novel, potent inhibitors are under development.Areas covered: The literature on LSD1 (also known as KDM1A, AOF2, BHC110 or KIAA0601) was identified in Pubmed and is herein discussed.  Areas covered include the structure and enzymic activity of LSD1, its role in chromatin regulatory complexes, its functional roles in normal and malignant tissue, pharmacol. inhibitors of its activity and their putative therapeutic roles.Expert opinion: Pre-clin. data supporting a therapeutic role for LSD1 inhibitors are most encouraging in acute myeloid leukemia, although optimal dosing strategies and beneficial combinations with other agents remain unclear.  Studies making use of potent, selective LSD1 inhibitors active in the nanomolar range are required to establish therapeutic indications in other subtypes of haematol. malignancy, and in solid tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2afxNLYupwrVg90H21EOLACvtfcHk0lik2DF6yUAiAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs12jtrzI&md5=53afa8f4337b2374ec6e5a2d3fe41736</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1517%2F14728222.2012.722206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F14728222.2012.722206%26sid%3Dliteratum%253Aachs%26aulast%3DLynch%26aufirst%3DJ.%2BT.%26aulast%3DHarris%26aufirst%3DW.%2BJ.%26aulast%3DSomervaille%26aufirst%3DT.%2BC.%26atitle%3DLSD1%2520inhibition%253A%2520a%2520therapeutic%2520strategy%2520in%2520cancer%253F%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Targets%26date%3D2012%26volume%3D16%26spage%3D1239%26epage%3D1249%26doi%3D10.1517%2F14728222.2012.722206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Dhanak, D.</span><span> </span><span class="NLM_article-title">Drugging the Cancer Epigenome</span>.  <span class="citation_source-book">Proceedings of the 104th Annual Meeting of the American Association for Cancer Research</span>, Washington, DC, April 6–10, 2013; <span class="NLM_publisher-name">AACR</span>: <span class="NLM_publisher-loc">Philadelphia, PA</span>,<span class="NLM_x"> </span><span class="NLM_year">2013</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=D.+Dhanak&title=Proceedings+of+the+104th+Annual+Meeting+of+the+American+Association+for+Cancer+Research"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DDhanak%26aufirst%3DD.%26atitle%3DDrugging%2520the%2520Cancer%2520Epigenome%26btitle%3DProceedings%2520of%2520the%2520104th%2520Annual%2520Meeting%2520of%2520the%2520American%2520Association%2520for%2520Cancer%2520Research%26pub%3DAACR%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Khan, M. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyata, N.</span><span> </span><span class="NLM_article-title">An overview of phenylcyclopropylamine derivatives: biochemical and biological significance and recent developments</span> <span class="citation_source-journal">Med. Res. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x">, </span> <span class="NLM_fpage">873</span><span class="NLM_x">–</span> <span class="NLM_lpage">910</span><span class="refDoi"> DOI: 10.1002/med.21269</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1002%2Fmed.21269" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=22893613" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BC3sXptFersL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2013&pages=873-910&author=M.+N.+Khanauthor=T.+Suzukiauthor=N.+Miyata&title=An+overview+of+phenylcyclopropylamine+derivatives%3A+biochemical+and+biological+significance+and+recent+developments&doi=10.1002%2Fmed.21269"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">An Overview of Phenylcyclopropylamine Derivatives: Biochemical and Biological Significance and Recent Developments</span></div><div class="casAuthors">Khan, Mohammed Naseer Ahmed; Suzuki, Takayoshi; Miyata, Naoki</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">873-910</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  Trans-2-Phencylcyclopropylamine (2-PCPA), a potent, clin. used antidepressant, affects monoamine neurotransmitter levels by inhibiting the main metabolizing enzymes, monoamine oxidases (MAOs).  However, the antidepressant action of this compd. was not fully explained by its effects on MAOs due to its wide variety of biol. effects. 2-PCPA also affects depression-assocd. pathophysiol. pathways, and linked with increased levels of trace amines in brain, upregulation of GABAB receptors (where GABA is gamma amino butyric acid), modulation of phospholipid metab., and interference with various cytochrome P 450 (CYP) enzymes.  Consequently, despite its adverse effects and limited clin. applicability, 2-PCPA has attracted interest as a structural scaffold for the development of mechanism-based inhibitors of various enzymes, including lysine-specific demethylase 1 (LSD1), which is a possible target for cancer chemotherapy.  In the recent years, many reports have appeared in the literature based on 2-PCPA scaffold and their potential medicinal implications.  This review mainly focuses on the medicinal chem. aspects including drug design, structure-activity relationships (SAR), biol. and biochem. properties, and mechanism of actions of 2-PCPA and its derivs.  Furthermore, we also highlight recent advance in this area and discuss their future applications for beneficial therapeutic effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqIZJFRem7GlrVg90H21EOLACvtfcHk0lik2DF6yUAiAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXptFersL0%253D&md5=d257f3b954efb33ae6f824c254ae1c30</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1002%2Fmed.21269&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21269%26sid%3Dliteratum%253Aachs%26aulast%3DKhan%26aufirst%3DM.%2BN.%26aulast%3DSuzuki%26aufirst%3DT.%26aulast%3DMiyata%26aufirst%3DN.%26atitle%3DAn%2520overview%2520of%2520phenylcyclopropylamine%2520derivatives%253A%2520biochemical%2520and%2520biological%2520significance%2520and%2520recent%2520developments%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2013%26volume%3D33%26spage%3D873%26epage%3D910%26doi%3D10.1002%2Fmed.21269" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Mimasu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Umezawa, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sato, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Higuchi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Umehara, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yokoyama, S.</span><span> </span><span class="NLM_article-title">Structurally designed trans-2-phenylcyclopropylamine derivatives potently inhibit histone demethylase LSD1/KDM1</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">6494</span><span class="NLM_x">–</span> <span class="NLM_lpage">6503</span><span class="refDoi"> DOI: 10.1021/bi100299r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi100299r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BC3cXosFCmur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2010&pages=6494-6503&author=S.+Mimasuauthor=N.+Umezawaauthor=S.+Satoauthor=T.+Higuchiauthor=T.+Umeharaauthor=S.+Yokoyama&title=Structurally+designed+trans-2-phenylcyclopropylamine+derivatives+potently+inhibit+histone+demethylase+LSD1%2FKDM1&doi=10.1021%2Fbi100299r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Structurally Designed trans-2-Phenylcyclopropylamine Derivatives Potently Inhibit Histone Demethylase LSD1/KDM1</span></div><div class="casAuthors">Mimasu, Shinya; Umezawa, Naoki; Sato, Shin; Higuchi, Tsunehiko; Umehara, Takashi; Yokoyama, Shigeyuki</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">6494-6503</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Lysine-specific demethylase 1 (LSD1/KDM1) demethylates histone H3, in addn. to tumor suppressor p53 and DNA methyltransferase 1 (Dnmt1), thus regulating eukaryotic gene expression by altering chromatin structure.  Specific inhibitors of LSD1 are desired as anticancer agents, because LSD1 aberrations are assocd. with several cancers, and LSD1 inhibition restores the expression of abnormally silenced genes in cancerous cells.  In this study, we designed and synthesized several candidate compds. to inhibit LSD1, based on the structures of LSD1 and monoamine oxidase B (MAO-B), in complex with an antidepressant tranylcypromine (2-PCPA) deriv.  Compd. S2101 exhibited stronger LSD1 inhibition than tranylcypromine and the known small LSD1 inhibitors in LSD1 demethylation assays, with a kinact/KI value of 4560 M-1 s-1.  In comparison with tranylcypromine, the compd. displayed weaker inhibition to the monoamine oxidases.  The inhibition modes of the two 2-PCPA derivs., 2-PFPA and S1201, were identified by detn. of the inhibitor-bound LSD1 structures, which revealed the enhanced stability of the inhibitor-FAD adducts by their interactions with the surrounding LSD1 residues.  These mols. are potential pharmaceutical candidates for cancer or latent virus infection, as well as research tools for LSD1-related biol. investigations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqIFRdqSvmn97Vg90H21EOLACvtfcHk0lhZgvfyaQz3XA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXosFCmur8%253D&md5=26732b0cd1928d44263f8bb8790ace54</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fbi100299r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi100299r%26sid%3Dliteratum%253Aachs%26aulast%3DMimasu%26aufirst%3DS.%26aulast%3DUmezawa%26aufirst%3DN.%26aulast%3DSato%26aufirst%3DS.%26aulast%3DHiguchi%26aufirst%3DT.%26aulast%3DUmehara%26aufirst%3DT.%26aulast%3DYokoyama%26aufirst%3DS.%26atitle%3DStructurally%2520designed%2520trans-2-phenylcyclopropylamine%2520derivatives%2520potently%2520inhibit%2520histone%2520demethylase%2520LSD1%252FKDM1%26jtitle%3DBiochemistry%26date%3D2010%26volume%3D49%26spage%3D6494%26epage%3D6503%26doi%3D10.1021%2Fbi100299r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Tortorici, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borrello, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tardugno, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chiarelli, L. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pilotto, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciossani, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vellore, N. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bailey, S. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Connell, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crabb, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Packham, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mai, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baron, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ganesan, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattevi, A.</span><span> </span><span class="NLM_article-title">Protein recognition by short peptide reversible inhibitors of the chromatin-modifying LSD1/CoREST lysine demethylase</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">1677</span><span class="NLM_x">–</span> <span class="NLM_lpage">1682</span><span class="refDoi"> DOI: 10.1021/cb4001926</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb4001926" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BC3sXosVCmt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=1677-1682&author=M.+Tortoriciauthor=M.+T.+Borrelloauthor=M.+Tardugnoauthor=L.+R.+Chiarelliauthor=S.+Pilottoauthor=G.+Ciossaniauthor=N.+A.+Velloreauthor=S.+G.+Baileyauthor=J.+Cowanauthor=M.+O%E2%80%99Connellauthor=S.+J.+Crabbauthor=G.+Packhamauthor=A.+Maiauthor=R.+Baronauthor=A.+Ganesanauthor=A.+Mattevi&title=Protein+recognition+by+short+peptide+reversible+inhibitors+of+the+chromatin-modifying+LSD1%2FCoREST+lysine+demethylase&doi=10.1021%2Fcb4001926"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Protein recognition by short peptide reversible inhibitors of the chromatin-modifying LSD1/CoREST lysine demethylase</span></div><div class="casAuthors">Tortorici, Marcello; Borrello, Maria Teresa; Tardugno, Maria; Chiarelli, Laurent R.; Pilotto, Simona; Ciossani, Giuseppe; Vellore, Nadeem A.; Bailey, Sarah G.; Cowan, Jonathan; O'Connell, Maria; Crabb, Simon J.; Packham, Graham; Mai, Antonello; Baron, Riccardo; Ganesan, A.; Mattevi, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1677-1682</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The combinatorial assembly of protein complexes is at the heart of chromatin biol.  Lysine demethylase LSD1(KDM1A)/CoREST beautifully exemplifies this concept.  The active site of the enzyme tightly assocs. to the N-terminal domain of transcription factors of the SNAIL1 family, which therefore can competitively inhibit the binding of the N-terminal tail of the histone substrate.  Our enzymic, crystallog., spectroscopic, and computational studies reveal that LSD1/CoREST can bind to a hexapeptide derived from the SNAIL sequence through recognition of a pos. charged α-helical turn that forms upon binding to the enzyme.  Variations in sequence and length of this six amino acid ligand modulate affinities enabling the same binding site to differentially interact with proteins that exert distinct biol. functions.  The discovered short peptide inhibitors exhibit antiproliferative activities and lay the foundation for the development of peptidomimetic small mol. inhibitors of LSD1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7SufQOwK-y7Vg90H21EOLACvtfcHk0lhZgvfyaQz3XA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXosVCmt7g%253D&md5=a8479b513e392e8acb78979e116dcffa</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fcb4001926&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb4001926%26sid%3Dliteratum%253Aachs%26aulast%3DTortorici%26aufirst%3DM.%26aulast%3DBorrello%26aufirst%3DM.%2BT.%26aulast%3DTardugno%26aufirst%3DM.%26aulast%3DChiarelli%26aufirst%3DL.%2BR.%26aulast%3DPilotto%26aufirst%3DS.%26aulast%3DCiossani%26aufirst%3DG.%26aulast%3DVellore%26aufirst%3DN.%2BA.%26aulast%3DBailey%26aufirst%3DS.%2BG.%26aulast%3DCowan%26aufirst%3DJ.%26aulast%3DO%25E2%2580%2599Connell%26aufirst%3DM.%26aulast%3DCrabb%26aufirst%3DS.%2BJ.%26aulast%3DPackham%26aufirst%3DG.%26aulast%3DMai%26aufirst%3DA.%26aulast%3DBaron%26aufirst%3DR.%26aulast%3DGanesan%26aufirst%3DA.%26aulast%3DMattevi%26aufirst%3DA.%26atitle%3DProtein%2520recognition%2520by%2520short%2520peptide%2520reversible%2520inhibitors%2520of%2520the%2520chromatin-modifying%2520LSD1%252FCoREST%2520lysine%2520demethylase%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2013%26volume%3D8%26spage%3D1677%26epage%3D1682%26doi%3D10.1021%2Fcb4001926" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Stavropoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blobel, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoelz, A.</span><span> </span><span class="NLM_article-title">Crystal structure and mechanism of human lysine-specific demethylase-1</span> <span class="citation_source-journal">Nat. Struct. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">626</span><span class="NLM_x">–</span> <span class="NLM_lpage">632</span><span class="refDoi"> DOI: 10.1038/nsmb1113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1038%2Fnsmb1113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=16799558" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BD28Xms1eltLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2006&pages=626-632&author=P.+Stavropoulosauthor=G.+Blobelauthor=A.+Hoelz&title=Crystal+structure+and+mechanism+of+human+lysine-specific+demethylase-1&doi=10.1038%2Fnsmb1113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure and mechanism of human lysine-specific demethylase-1</span></div><div class="casAuthors">Stavropoulos, Pete; Blobel, Guenter; Hoelz, Andre</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">626-632</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The reversible methylation of specific Lys residues in histone tails is crucial in epigenetic gene regulation.  LSD1, the 1st known lysine-specific demethylase, selectively removes mono-Me and di-Me, but not tri-Me modifications of Lys-4 or Lys-9 in histone-3.  Here, the authors present the crystal structure of LSD1 at 2.9 Å resoln.  LSD1 was found to form a highly asym., closely packed domain structure from which a long helical 'tower' domain protruded.  The active site cavity was spacious enough to accommodate several residues of the histone tail substrate, but did not appear capable of recognizing the different methylation states of the substrate, lysine.  This supported the hypothesis that trimethylated lysine was chem. rather than sterically discriminated.  The authors presented a biochem. anal. of LSD1 mutants that identified crucial residues in the active site cavity and showed the importance of the SWIRM and tower domains for catalysis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqG8Vp2AlsWQLVg90H21EOLACvtfcHk0lhZgvfyaQz3XA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xms1eltLs%253D&md5=34cbc6f0936b642a8668d1d76abd1e0e</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1038%2Fnsmb1113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb1113%26sid%3Dliteratum%253Aachs%26aulast%3DStavropoulos%26aufirst%3DP.%26aulast%3DBlobel%26aufirst%3DG.%26aulast%3DHoelz%26aufirst%3DA.%26atitle%3DCrystal%2520structure%2520and%2520mechanism%2520of%2520human%2520lysine-specific%2520demethylase-1%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2006%26volume%3D13%26spage%3D626%26epage%3D632%26doi%3D10.1038%2Fnsmb1113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Forneris, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Binda, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adamo, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Battaglioli, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattevi, A.</span><span> </span><span class="NLM_article-title">Structural basis of LSD1-CoREST selectivity in histone H3 recognition</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">282</span><span class="NLM_x">, </span> <span class="NLM_fpage">20070</span><span class="NLM_x">–</span> <span class="NLM_lpage">20074</span><span class="refDoi"> DOI: 10.1074/jbc.C700100200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1074%2Fjbc.C700100200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=17537733" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnsFWntr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=282&publication_year=2007&pages=20070-20074&author=F.+Fornerisauthor=C.+Bindaauthor=A.+Adamoauthor=E.+Battaglioliauthor=A.+Mattevi&title=Structural+basis+of+LSD1-CoREST+selectivity+in+histone+H3+recognition&doi=10.1074%2Fjbc.C700100200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis of LSD1-CoREST Selectivity in Histone H3 Recognition</span></div><div class="casAuthors">Forneris, Federico; Binda, Claudia; Adamo, Antonio; Battaglioli, Elena; Mattevi, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">282</span>
        (<span class="NLM_cas:issue">28</span>),
    <span class="NLM_cas:pages">20070-20074</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Histone demethylase LSD1 regulates transcription by demethylating Lys4 of histone H3.  The crystal structure of the enzyme in complex with CoREST and a substrate-like peptide inhibitor highlights an intricate network of interactions and a folded conformation of the bound peptide.  The core of the peptide structure is formed by Arg2, Gln5, and Ser10, which are engaged in specific intramol. H-bonds.  Several charged side chains on the surface of the substrate-binding pocket establish electrostatic interactions with the peptide.  The three-dimensional structure predicts that methylated Lys4 binds in a solvent inaccessible position in front of the flavin cofactor.  This geometry is fully consistent with the demethylation reaction being catalyzed through a flavin-mediated oxidn. of the substrate amino-Me group.  These features dictate the exquisite substrate specificity of LSD1 and provide a structural framework to explain the fine tuning of its catalytic activity and the active role of CoREST in substrate recognition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpD8PHQpzggArVg90H21EOLACvtfcHk0lji9Z35hUdJ_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnsFWntr0%253D&md5=a8af3e0892441bf3fc64ce80c19d0c3b</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1074%2Fjbc.C700100200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.C700100200%26sid%3Dliteratum%253Aachs%26aulast%3DForneris%26aufirst%3DF.%26aulast%3DBinda%26aufirst%3DC.%26aulast%3DAdamo%26aufirst%3DA.%26aulast%3DBattaglioli%26aufirst%3DE.%26aulast%3DMattevi%26aufirst%3DA.%26atitle%3DStructural%2520basis%2520of%2520LSD1-CoREST%2520selectivity%2520in%2520histone%2520H3%2520recognition%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2007%26volume%3D282%26spage%3D20070%26epage%3D20074%26doi%3D10.1074%2Fjbc.C700100200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span>Schrödinger Suite, version <span class="NLM_edition">2015</span>; <span class="NLM_publisher-name">Schrödinger, LLC</span>: <span class="NLM_publisher-loc">New York, NY</span>,<span class="NLM_x"> </span><span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Schr%C3%B6dinger+Suite%2C+version+2015%3B+Schr%C3%B6dinger%2C+LLC%3A+New+York%2C+NY%2C+2015."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DSchr%25C3%25B6dinger%2520Suite%26pub%3DSchr%25C3%25B6dinger%252C%2520LLC%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">Lin, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamashita, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nidarmarthy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luengo, J. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rhodes, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knick, V. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choudhry, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lai, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Minthorn, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Strum, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wood, E. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elkins, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Concha, N. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heerding, D. A.</span><span> </span><span class="NLM_article-title">2,3,5-Trisubstituted pyridines as selective AKT inhibitors-Part I: Substitution at 2-position of the core pyridine for ROCK1 selectivity</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">673</span><span class="NLM_x">–</span> <span class="NLM_lpage">678</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2009.11.064</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1016%2Fj.bmcl.2009.11.064" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2010&pages=673-678&author=H.+Linauthor=D.+S.+Yamashitaauthor=J.+Zengauthor=R.+Xieauthor=W.+Wangauthor=S.+Nidarmarthyauthor=J.+I.+Luengoauthor=N.+Rhodesauthor=V.+B.+Knickauthor=A.+E.+Choudhryauthor=Z.+Laiauthor=E.+A.+Minthornauthor=S.+L.+Strumauthor=E.+R.+Woodauthor=P.+A.+Elkinsauthor=N.+O.+Conchaauthor=D.+A.+Heerding&title=2%2C3%2C5-Trisubstituted+pyridines+as+selective+AKT+inhibitors-Part+I%3A+Substitution+at+2-position+of+the+core+pyridine+for+ROCK1+selectivity&doi=10.1016%2Fj.bmcl.2009.11.064"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2009.11.064&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2009.11.064%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DH.%26aulast%3DYamashita%26aufirst%3DD.%2BS.%26aulast%3DZeng%26aufirst%3DJ.%26aulast%3DXie%26aufirst%3DR.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DNidarmarthy%26aufirst%3DS.%26aulast%3DLuengo%26aufirst%3DJ.%2BI.%26aulast%3DRhodes%26aufirst%3DN.%26aulast%3DKnick%26aufirst%3DV.%2BB.%26aulast%3DChoudhry%26aufirst%3DA.%2BE.%26aulast%3DLai%26aufirst%3DZ.%26aulast%3DMinthorn%26aufirst%3DE.%2BA.%26aulast%3DStrum%26aufirst%3DS.%2BL.%26aulast%3DWood%26aufirst%3DE.%2BR.%26aulast%3DElkins%26aufirst%3DP.%2BA.%26aulast%3DConcha%26aufirst%3DN.%2BO.%26aulast%3DHeerding%26aufirst%3DD.%2BA.%26atitle%3D2%252C3%252C5-Trisubstituted%2520pyridines%2520as%2520selective%2520AKT%2520inhibitors-Part%2520I%253A%2520Substitution%2520at%25202-position%2520of%2520the%2520core%2520pyridine%2520for%2520ROCK1%2520selectivity%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2010%26volume%3D20%26spage%3D673%26epage%3D678%26doi%3D10.1016%2Fj.bmcl.2009.11.064" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group">Lin, N.-H.; He, Y.; Holladay, M. W.; Ryther, K.; Li, Y.</span><span> </span><span class="NLM_article-title">3-Pyridyloxymethyl heterocyclic ether compounds useful in controlling chemical synaptic transmission</span>. U.S. Patent 5,629,325,<span class="NLM_x"> </span><span class="NLM_year">1997</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1997&author=N.-H.+Lin&author=Y.+He&author=M.+W.+Holladay&author=K.+Ryther&author=Y.+Li&title=3-Pyridyloxymethyl+heterocyclic+ether+compounds+useful+in+controlling+chemical+synaptic+transmission"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DN.-H.%26atitle%3D3-Pyridyloxymethyl%2520heterocyclic%2520ether%2520compounds%2520useful%2520in%2520controlling%2520chemical%2520synaptic%2520transmission%26date%3D1997" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group">Ballell, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bates, R. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alvarez-Gomez, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alvarez-Ruiz, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barroso, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blanco, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crespo, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Escribano, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">González, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lozano, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huss, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santos-Villarejo, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martín-Plaza, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mendoza, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rebollo-Lopez, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Remuiñan-Blanco, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lavandera, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pérez-Herran, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gamo-Benito, F. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">García-Bustos, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barros, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Castro, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cammack, N.</span><span> </span><span class="NLM_article-title">Fueling open-source drug discovery: 177 small-molecule leads against tuberculosis</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">313</span><span class="NLM_x">–</span> <span class="NLM_lpage">321</span><span class="refDoi"> DOI: 10.1002/cmdc.201200428</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1002%2Fcmdc.201200428" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=23307663" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BC3sXls1Omug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=313-321&author=L.+Ballellauthor=R.+H.+Batesauthor=R.+J.+Youngauthor=D.+Alvarez-Gomezauthor=E.+Alvarez-Ruizauthor=V.+Barrosoauthor=D.+Blancoauthor=B.+Crespoauthor=J.+Escribanoauthor=R.+Gonz%C3%A1lezauthor=S.+Lozanoauthor=S.+Hussauthor=A.+Santos-Villarejoauthor=J.+J.+Mart%C3%ADn-Plazaauthor=A.+Mendozaauthor=M.+J.+Rebollo-Lopezauthor=M.+Remui%C3%B1an-Blancoauthor=J.+L.+Lavanderaauthor=E.+P%C3%A9rez-Herranauthor=F.+J.+Gamo-Benitoauthor=J.+F.+Garc%C3%ADa-Bustosauthor=D.+Barrosauthor=J.+P.+Castroauthor=N.+Cammack&title=Fueling+open-source+drug+discovery%3A+177+small-molecule+leads+against+tuberculosis&doi=10.1002%2Fcmdc.201200428"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Fueling Open-Source Drug Discovery: 177 Small-Molecule Leads against Tuberculosis</span></div><div class="casAuthors">Ballell, Lluis; Bates, Robert H.; Young, Rob J.; Alvarez-Gomez, Daniel; Alvarez-Ruiz, Emilio; Barroso, Vanessa; Blanco, Delia; Crespo, Benigno; Escribano, Jaime; Gonzalez, Ruben; Lozano, Sonia; Huss, Sophie; Santos-Villarejo, Angel; Martin-Plaza, Jose Julio; Mendoza, Alfonso; Rebollo-Lopez, Maria Jose; Remuinan-Blanco, Modesto; Lavandera, Jose Luis; Perez-Herran, Esther; Gamo-Benito, Francisco Javier; Garcia-Bustos, Jose Francisco; Barros, David; Castro, Julia P.; Cammack, Nicholas</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">313-321</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">With the aim of fuelling open-source, translational, early-stage drug discovery activities, the results of the recently completed antimycobacterial phenotypic screening campaign against Mycobacterium bovis BCG with hit confirmation in M. tuberculosis H37Rv were made publicly accessible.  A set of 177 potent non-cytotoxic H37Rv hits was identified and will be made available to maximize the potential impact of the compds. toward a chem. genetics/proteomics exercise, while at the same time providing a plethora of potential starting points for new synthetic lead-generation activities.  Two addnl. drug-discovery-relevant datasets are included: (a) a drug-like property anal. reflecting the latest lead-like guidelines and (b) an early lead-generation package of the most promising hits within the clusters identified.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUJ2sVPWfPALVg90H21EOLACvtfcHk0lji9Z35hUdJ_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXls1Omug%253D%253D&md5=32c975dbe183849358c701ca697681d3</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201200428&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201200428%26sid%3Dliteratum%253Aachs%26aulast%3DBallell%26aufirst%3DL.%26aulast%3DBates%26aufirst%3DR.%2BH.%26aulast%3DYoung%26aufirst%3DR.%2BJ.%26aulast%3DAlvarez-Gomez%26aufirst%3DD.%26aulast%3DAlvarez-Ruiz%26aufirst%3DE.%26aulast%3DBarroso%26aufirst%3DV.%26aulast%3DBlanco%26aufirst%3DD.%26aulast%3DCrespo%26aufirst%3DB.%26aulast%3DEscribano%26aufirst%3DJ.%26aulast%3DGonz%25C3%25A1lez%26aufirst%3DR.%26aulast%3DLozano%26aufirst%3DS.%26aulast%3DHuss%26aufirst%3DS.%26aulast%3DSantos-Villarejo%26aufirst%3DA.%26aulast%3DMart%25C3%25ADn-Plaza%26aufirst%3DJ.%2BJ.%26aulast%3DMendoza%26aufirst%3DA.%26aulast%3DRebollo-Lopez%26aufirst%3DM.%2BJ.%26aulast%3DRemui%25C3%25B1an-Blanco%26aufirst%3DM.%26aulast%3DLavandera%26aufirst%3DJ.%2BL.%26aulast%3DP%25C3%25A9rez-Herran%26aufirst%3DE.%26aulast%3DGamo-Benito%26aufirst%3DF.%2BJ.%26aulast%3DGarc%25C3%25ADa-Bustos%26aufirst%3DJ.%2BF.%26aulast%3DBarros%26aufirst%3DD.%26aulast%3DCastro%26aufirst%3DJ.%2BP.%26aulast%3DCammack%26aufirst%3DN.%26atitle%3DFueling%2520open-source%2520drug%2520discovery%253A%2520177%2520small-molecule%2520leads%2520against%2520tuberculosis%26jtitle%3DChemMedChem%26date%3D2013%26volume%3D8%26spage%3D313%26epage%3D321%26doi%3D10.1002%2Fcmdc.201200428" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group">Zheng, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anglin, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prasad, B. V. V</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, Y.</span><span> </span><span class="NLM_article-title">Crystallographic Investigation and Selective Inhibition of Mutant Isocitrate Dehydrogenase</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">542</span><span class="NLM_x">–</span> <span class="NLM_lpage">546</span><span class="refDoi"> DOI: 10.1021/ml400036z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml400036z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BC3sXntVWrtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=542-546&author=B.+Zhengauthor=Y.+Yaoauthor=Z.+Liuauthor=L.+Dengauthor=J.+L.+Anglinauthor=H.+Jiangauthor=B.+V.+V+Prasadauthor=Y.+Song&title=Crystallographic+Investigation+and+Selective+Inhibition+of+Mutant+Isocitrate+Dehydrogenase&doi=10.1021%2Fml400036z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Crystallographic Investigation and Selective Inhibition of Mutant Isocitrate Dehydrogenase</span></div><div class="casAuthors">Zheng, Baisong; Yao, Yuan; Liu, Zhen; Deng, Lisheng; Anglin, Justin L.; Jiang, Hong; Prasad, B. V. Venkataram; Song, Yongcheng</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">542-546</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mutations in isocitrate dehydrogenase (IDH), a key enzyme in the tricarboxylic acid cycle, have recently been found in ∼75% glioma and ∼20% acute myeloid leukemia.  Different from the wild-type enzyme, mutant IDH1 catalyzes the redn. of α-ketoglutaric acid to D-2-hydroxyglutaric acid.  Strong evidence has shown mutant IDH1 represents a novel target for this type of cancer.  We found two 1-hydroxypyridin-2-one compds. that are potent inhibitors of R132H and R132C IDH1 mutants with Ki values as low as 120 nM.  These compds. exhibit >60-fold selectivity against wild-type IDH1 and can inhibit the prodn. of D-2-hydroxyglutaric acid in IDH1 mutated cells, representing novel chem. probes for cancer biol. studies.  We also report the first inhibitor-bound crystal structures of IDH1(R132H), showing these inhibitors have H-bond, electrostatic, and hydrophobic interactions with the mutant enzyme.  Comparison with the substrate-bound IDH1 structures revealed the structural basis for the high enzyme selectivity of these compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqH0vzVLGCPZLVg90H21EOLACvtfcHk0li8CBKz5Y-nXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXntVWrtLg%253D&md5=c34c67949fe816bdf151525a578535f3</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Fml400036z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml400036z%26sid%3Dliteratum%253Aachs%26aulast%3DZheng%26aufirst%3DB.%26aulast%3DYao%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DDeng%26aufirst%3DL.%26aulast%3DAnglin%26aufirst%3DJ.%2BL.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DPrasad%26aufirst%3DB.%2BV.%2BV%26aulast%3DSong%26aufirst%3DY.%26atitle%3DCrystallographic%2520Investigation%2520and%2520Selective%2520Inhibition%2520of%2520Mutant%2520Isocitrate%2520Dehydrogenase%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D4%26spage%3D542%26epage%3D546%26doi%3D10.1021%2Fml400036z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group">Deng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pujari, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crick, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prasad, B. V. V</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, Y.</span><span> </span><span class="NLM_article-title">Inhibition of 1-Deoxy-D-Xylulose-5-Phosphate Reductoisomerase by Lipophilic Phosphonates: SAR, QSAR and Crystallographic Studies</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">4721</span><span class="NLM_x">–</span> <span class="NLM_lpage">34</span><span class="refDoi"> DOI: 10.1021/jm200363d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm200363d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=4721-34&author=L.+Dengauthor=J.+Diaoauthor=P.+Chenauthor=V.+Pujariauthor=Y.+Yaoauthor=G.+Chengauthor=D.+C.+Crickauthor=B.+V.+V+Prasadauthor=Y.+Song&title=Inhibition+of+1-Deoxy-D-Xylulose-5-Phosphate+Reductoisomerase+by+Lipophilic+Phosphonates%3A+SAR%2C+QSAR+and+Crystallographic+Studies&doi=10.1021%2Fjm200363d"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Fjm200363d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200363d%26sid%3Dliteratum%253Aachs%26aulast%3DDeng%26aufirst%3DL.%26aulast%3DDiao%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DP.%26aulast%3DPujari%26aufirst%3DV.%26aulast%3DYao%26aufirst%3DY.%26aulast%3DCheng%26aufirst%3DG.%26aulast%3DCrick%26aufirst%3DD.%2BC.%26aulast%3DPrasad%26aufirst%3DB.%2BV.%2BV%26aulast%3DSong%26aufirst%3DY.%26atitle%3DInhibition%2520of%25201-Deoxy-D-Xylulose-5-Phosphate%2520Reductoisomerase%2520by%2520Lipophilic%2520Phosphonates%253A%2520SAR%252C%2520QSAR%2520and%2520Crystallographic%2520Studies%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D4721%26epage%3D34%26doi%3D10.1021%2Fjm200363d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group">Deng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sundriyal, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rubio, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, Y.</span><span> </span><span class="NLM_article-title">Coordination Chemistry Based Approach to Lipophilic Inhibitors of 1-Deoxy-<i>D</i>-xylulose-5-phosphate Reductoisomerase</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">6539</span><span class="NLM_x">–</span> <span class="NLM_lpage">42</span><span class="refDoi"> DOI: 10.1021/jm9012592</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm9012592" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=6539-42&author=L.+Dengauthor=S.+Sundriyalauthor=V.+Rubioauthor=Z.+Shiauthor=Y.+Song&title=Coordination+Chemistry+Based+Approach+to+Lipophilic+Inhibitors+of+1-Deoxy-D-xylulose-5-phosphate+Reductoisomerase&doi=10.1021%2Fjm9012592"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Fjm9012592&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm9012592%26sid%3Dliteratum%253Aachs%26aulast%3DDeng%26aufirst%3DL.%26aulast%3DSundriyal%26aufirst%3DS.%26aulast%3DRubio%26aufirst%3DV.%26aulast%3DShi%26aufirst%3DZ.%26aulast%3DSong%26aufirst%3DY.%26atitle%3DCoordination%2520Chemistry%2520Based%2520Approach%2520to%2520Lipophilic%2520Inhibitors%2520of%25201-Deoxy-D-xylulose-5-phosphate%2520Reductoisomerase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D6539%26epage%3D42%26doi%3D10.1021%2Fjm9012592" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group">Zhang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wei, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mo, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, Y.</span><span> </span><span class="NLM_article-title">Inhibition of Histone H3K79 Methylation Selectively Inhibits Proliferation, Self-renewal and Metastatic Potential of Breast Cancer</span> <span class="citation_source-journal">Oncotarget</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">10665</span><span class="NLM_x">–</span> <span class="NLM_lpage">10677</span><span class="refDoi"> DOI: 10.18632/oncotarget.2496</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.18632%2Foncotarget.2496" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=25359765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A280%3ADC%252BC2M3kslylsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=10665-10677&author=L.+Zhangauthor=L.+Dengauthor=F.+Chenauthor=Y.+Yaoauthor=B.+Wuauthor=L.+Weiauthor=Q.+Moauthor=Y.+Song&title=Inhibition+of+Histone+H3K79+Methylation+Selectively+Inhibits+Proliferation%2C+Self-renewal+and+Metastatic+Potential+of+Breast+Cancer&doi=10.18632%2Foncotarget.2496"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of histone H3K79 methylation selectively inhibits proliferation, self-renewal and metastatic potential of breast cancer</span></div><div class="casAuthors">Zhang Li; Deng Lisheng; Yao Yuan; Wu Bulan; Wei Liping; Chen Fengju; Mo Qianxing; Song Yongcheng</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">10665-77</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Histone lysine methylation regulates gene expression and cancer initiation.  Bioinformatics analysis suggested that DOT1L, a histone H3-lysine79 (H3K79) methyltransferase, plays a potentially important role in breast cancer.  DOT1L inhibition selectively inhibited proliferation, self-renewal, metastatic potential of breast cancer cells and induced cell differentiation.  In addition, inhibitors of S-adenosylhomocysteine hydrolase (SAHH), such as neplanocin and 3-deazaneplanocin, also inhibited both H3K79 methylation and proliferation of breast cancer cells in vitro and in vivo.  The activity of SAHH inhibitors was previously attributed to inhibition of H3K27 methyltransferase EZH2.  However, inhibition of EZH2 by a specific inhibitor did not contribute to cell death.  SAHH inhibitors had only weak activity against H3K27 methylation and their activity is therefore mainly due to DOT1L/H3K79 methylation inhibition.  Overall, we showed that DOT1L is a potential drug target for breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcToXKVEBdgQcLHHuJhbyicZfW6udTcc2eac9uQj_4jK7Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2M3kslylsg%253D%253D&md5=c495f4521165f0fb79399b80d8181955</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.2496&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.2496%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DDeng%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DF.%26aulast%3DYao%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DB.%26aulast%3DWei%26aufirst%3DL.%26aulast%3DMo%26aufirst%3DQ.%26aulast%3DSong%26aufirst%3DY.%26atitle%3DInhibition%2520of%2520Histone%2520H3K79%2520Methylation%2520Selectively%2520Inhibits%2520Proliferation%252C%2520Self-renewal%2520and%2520Metastatic%2520Potential%2520of%2520Breast%2520Cancer%26jtitle%3DOncotarget%26date%3D2014%26volume%3D5%26spage%3D10665%26epage%3D10677%26doi%3D10.18632%2Foncotarget.2496" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group">Liu, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kogiso, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qiu, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lv, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gallo, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, X.-N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, Y.</span><span> </span><span class="NLM_article-title">Inhibition of Cancer-Associated Mutant Isocitrate Dehydrogenases: Synthesis, SAR and Selective Antitumor Activity</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">8307</span><span class="NLM_x">–</span> <span class="NLM_lpage">8318</span><span class="refDoi"> DOI: 10.1021/jm500660f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm500660f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=8307-8318&author=Z.+Liuauthor=Y.+Yaoauthor=M.+Kogisoauthor=B.+Zhengauthor=L.+Dengauthor=J.+J.+Qiuauthor=S.+Dongauthor=H.+Lvauthor=J.+M.+Galloauthor=X.-N.+Liauthor=Y.+Song&title=Inhibition+of+Cancer-Associated+Mutant+Isocitrate+Dehydrogenases%3A+Synthesis%2C+SAR+and+Selective+Antitumor+Activity&doi=10.1021%2Fjm500660f"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Fjm500660f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm500660f%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DYao%26aufirst%3DY.%26aulast%3DKogiso%26aufirst%3DM.%26aulast%3DZheng%26aufirst%3DB.%26aulast%3DDeng%26aufirst%3DL.%26aulast%3DQiu%26aufirst%3DJ.%2BJ.%26aulast%3DDong%26aufirst%3DS.%26aulast%3DLv%26aufirst%3DH.%26aulast%3DGallo%26aufirst%3DJ.%2BM.%26aulast%3DLi%26aufirst%3DX.-N.%26aulast%3DSong%26aufirst%3DY.%26atitle%3DInhibition%2520of%2520Cancer-Associated%2520Mutant%2520Isocitrate%2520Dehydrogenases%253A%2520Synthesis%252C%2520SAR%2520and%2520Selective%2520Antitumor%2520Activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D8307%26epage%3D8318%26doi%3D10.1021%2Fjm500660f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23a','cit23b','cit23c','cit23d'],'ref24':['cit24a','cit24b','cit24c','cit24d'],'ref25':['cit25'],'ref26':['cit26a','cit26b'],'ref27':['cit27a','cit27b','cit27c','cit27d','cit27e','cit27f','cit27g','cit27h','cit27i','cit27j'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 48 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Shenyou Nie, Yuan Yao, Fangrui Wu, Xiaowei Wu, Jidong Zhao, Yuanda Hua, Jingyu Wu, Tong Huo, Yi-Lun Lin, Alexander R. Kneubehl, Megan B. Vogt, Josephine Ferreon, Rebecca Rico-Hesse, <span class="NLM_string-name hlFld-ContribAuthor">Yongcheng Song</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis, Structure–Activity Relationships, and Antiviral Activity of Allosteric Inhibitors of Flavivirus NS2B–NS3 Protease. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (5)
                                     , 2777-2800. <a href="https://doi.org/10.1021/acs.jmedchem.0c02070" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02070</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02070&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c02070%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DSynthesis%25252C%252BStructure%2525E2%252580%252593Activity%252BRelationships%25252C%252Band%252BAntiviral%252BActivity%252Bof%252BAllosteric%252BInhibitors%252Bof%252BFlavivirus%252BNS2B%2525E2%252580%252593NS3%252BProtease%26aulast%3DNie%26aufirst%3DShenyou%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D28112020%26date%3D17022021%26volume%3D64%26issue%3D5%26spage%3D2777%26epage%3D2800" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Xing-Jie Dai, Ying Liu, Lei-Peng Xue, Xiao-Peng Xiong, Ying Zhou, Yi-Chao Zheng, <span class="NLM_string-name hlFld-ContribAuthor">Hong-Min Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Reversible Lysine Specific Demethylase 1 (LSD1) Inhibitors: A Promising Wrench to Impair LSD1. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (5)
                                     , 2466-2488. <a href="https://doi.org/10.1021/acs.jmedchem.0c02176" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02176</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02176&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c02176%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DReversible%252BLysine%252BSpecific%252BDemethylase%252B1%252B%252528LSD1%252529%252BInhibitors%25253A%252BA%252BPromising%252BWrench%252Bto%252BImpair%252BLSD1%26aulast%3DDai%26aufirst%3DXing-Jie%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D16122020%26date%3D23022021%26volume%3D64%26issue%3D5%26spage%3D2466%26epage%3D2488" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Fangrui Wu, Yuanda Hua, Salma Kaochar, Shenyou Nie, Yi-Lun Lin, Yuan Yao, Jingyu Wu, Xiaowei Wu, Xiaoyong Fu, Rachel Schiff, Christel M. Davis, Matthew Robertson, Erik A. Ehli, Cristian Coarfa, Nicholas Mitsiades, <span class="NLM_string-name hlFld-ContribAuthor">Yongcheng Song</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery, Structure–Activity Relationship, and Biological Activity of Histone-Competitive Inhibitors of Histone Acetyltransferases P300/CBP. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (9)
                                     , 4716-4731. <a href="https://doi.org/10.1021/acs.jmedchem.9b02164" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b02164</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b02164&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b02164%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%25252C%252BStructure%2525E2%252580%252593Activity%252BRelationship%25252C%252Band%252BBiological%252BActivity%252Bof%252BHistone-Competitive%252BInhibitors%252Bof%252BHistone%252BAcetyltransferases%252BP300%25252FCBP%26aulast%3DWu%26aufirst%3DFangrui%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D26122019%26date%3D21042020%26volume%3D63%26issue%3D9%26spage%3D4716%26epage%3D4731" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Yuan Yao, Tong Huo, Yi-Lun Lin, Shenyou Nie, Fangrui Wu, Yuanda Hua, Jingyu Wu, Alexander R. Kneubehl, Megan B. Vogt, Rebecca Rico-Hesse, <span class="NLM_string-name hlFld-ContribAuthor">Yongcheng Song</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery, X-ray Crystallography and Antiviral Activity of Allosteric Inhibitors of Flavivirus NS2B-NS3 Protease. </span><span class="cited-content_cbyCitation_journal-name">Journal of the American Chemical Society</span><span> <strong>2019,</strong> <em>141 </em>
                                    (17)
                                     , 6832-6836. <a href="https://doi.org/10.1021/jacs.9b02505" title="DOI URL">https://doi.org/10.1021/jacs.9b02505</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jacs.9b02505&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjacs.9b02505%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520the%2520American%2520Chemical%2520Society%26atitle%3DDiscovery%25252C%252BX-ray%252BCrystallography%252Band%252BAntiviral%252BActivity%252Bof%252BAllosteric%252BInhibitors%252Bof%252BFlavivirus%252BNS2B-NS3%252BProtease%26aulast%3DYao%26aufirst%3DYuan%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D06032019%26date%3D23042019%26volume%3D141%26issue%3D17%26spage%3D6832%26epage%3D6836" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">H. Ümit  Kaniskan</span>, <span class="hlFld-ContribAuthor ">Michael L.  Martini</span>, and <span class="hlFld-ContribAuthor ">Jian  Jin</span>  . </span><span class="cited-content_cbyCitation_article-title">Inhibitors of Protein Methyltransferases and Demethylases. </span><span class="cited-content_cbyCitation_journal-name">Chemical Reviews</span><span> <strong>2018,</strong> <em>118 </em>
                                    (3)
                                     , 989-1068. <a href="https://doi.org/10.1021/acs.chemrev.6b00801" title="DOI URL">https://doi.org/10.1021/acs.chemrev.6b00801</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.chemrev.6b00801&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.chemrev.6b00801%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Reviews%26atitle%3DInhibitors%252Bof%252BProtein%252BMethyltransferases%252Band%252BDemethylases%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259Cmit%26date%3D2018%26date%3D2017%26date%3D2016%26date%3D30112016%26date%3D24032017%26date%3D14022018%26volume%3D118%26issue%3D3%26spage%3D989%26epage%3D1068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Daniel P.  Mould</span>, <span class="hlFld-ContribAuthor ">Cristina  Alli</span>, <span class="hlFld-ContribAuthor ">Ulf  Bremberg</span>, <span class="hlFld-ContribAuthor ">Sharon  Cartic</span>, <span class="hlFld-ContribAuthor ">Allan M.  Jordan</span>, <span class="hlFld-ContribAuthor ">Matthis  Geitmann</span>, <span class="hlFld-ContribAuthor ">Alba  Maiques-Diaz</span>, <span class="hlFld-ContribAuthor ">Alison E.  McGonagle</span>, <span class="hlFld-ContribAuthor ">Tim C. P.  Somervaille</span>, <span class="hlFld-ContribAuthor ">Gary J.  Spencer</span>, <span class="hlFld-ContribAuthor ">Fabrice  Turlais</span>, and <span class="hlFld-ContribAuthor ">Donald  Ogilvie</span>  . </span><span class="cited-content_cbyCitation_article-title">Development of (4-Cyanophenyl)glycine Derivatives as Reversible Inhibitors of Lysine Specific Demethylase 1. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2017,</strong> <em>60 </em>
                                    (19)
                                     , 7984-7999. <a href="https://doi.org/10.1021/acs.jmedchem.7b00462" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b00462</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b00462&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.7b00462%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDevelopment%252Bof%252B%2525284-Cyanophenyl%252529glycine%252BDerivatives%252Bas%252BReversible%252BInhibitors%252Bof%252BLysine%252BSpecific%252BDemethylase%252B1%26aulast%3DMould%26aufirst%3DDaniel%2BP.%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D28032017%26date%3D21092017%26date%3D12102017%26date%3D11092017%26volume%3D60%26issue%3D19%26spage%3D7984%26epage%3D7999" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhong-Hua  Li</span>, <span class="hlFld-ContribAuthor ">Xue-Qi  Liu</span>, <span class="hlFld-ContribAuthor ">Peng-Fei  Geng</span>, <span class="hlFld-ContribAuthor ">Feng-Zhi  Suo</span>, <span class="hlFld-ContribAuthor ">Jin-Lian  Ma</span>, <span class="hlFld-ContribAuthor ">Bin  Yu</span>, <span class="hlFld-ContribAuthor ">Tao-Qian  Zhao</span>, <span class="hlFld-ContribAuthor ">Zhao-Qing  Zhou</span>, <span class="hlFld-ContribAuthor ">Chen-Xi  Huang</span>, <span class="hlFld-ContribAuthor ">Yi-Chao  Zheng</span>, and <span class="hlFld-ContribAuthor ">Hong-Min  Liu</span>  . </span><span class="cited-content_cbyCitation_article-title">Discovery of [1,2,3]Triazolo[4,5-d]pyrimidine Derivatives as Novel LSD1 Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2017,</strong> <em>8 </em>
                                    (4)
                                     , 384-389. <a href="https://doi.org/10.1021/acsmedchemlett.6b00423" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.6b00423</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.6b00423&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.6b00423%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DDiscovery%252Bof%252B%25255B1%25252C2%25252C3%25255DTriazolo%25255B4%25252C5-d%25255Dpyrimidine%252BDerivatives%252Bas%252BNovel%252BLSD1%252BInhibitors%26aulast%3DLi%26aufirst%3DZhong-Hua%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D2016%26date%3D25102016%26date%3D06032017%26date%3D08032017%26date%3D13042017%26date%3D06032017%26volume%3D8%26issue%3D4%26spage%3D384%26epage%3D389" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Paola  Vianello</span>, <span class="hlFld-ContribAuthor ">Luca  Sartori</span>, <span class="hlFld-ContribAuthor ">Federica  Amigoni</span>, <span class="hlFld-ContribAuthor ">Anna  Cappa</span>, <span class="hlFld-ContribAuthor ">Giovanni  Fagá</span>, <span class="hlFld-ContribAuthor ">Raimondo  Fattori</span>, <span class="hlFld-ContribAuthor ">Elena  Legnaghi</span>, <span class="hlFld-ContribAuthor ">Giuseppe  Ciossani</span>, <span class="hlFld-ContribAuthor ">Andrea  Mattevi</span>, <span class="hlFld-ContribAuthor ">Giuseppe  Meroni</span>, <span class="hlFld-ContribAuthor ">Loris  Moretti</span>, <span class="hlFld-ContribAuthor ">Valentina  Cecatiello</span>, <span class="hlFld-ContribAuthor ">Sebastiano  Pasqualato</span>, <span class="hlFld-ContribAuthor ">Alessia  Romussi</span>, <span class="hlFld-ContribAuthor ">Florian  Thaler</span>, <span class="hlFld-ContribAuthor ">Paolo  Trifiró</span>, <span class="hlFld-ContribAuthor ">Manuela  Villa</span>, <span class="hlFld-ContribAuthor ">Oronza A.  Botrugno</span>, <span class="hlFld-ContribAuthor ">Paola  Dessanti</span>, <span class="hlFld-ContribAuthor ">Saverio  Minucci</span>, <span class="hlFld-ContribAuthor ">Stefania  Vultaggio</span>, <span class="hlFld-ContribAuthor ">Elisa  Zagarrí</span>, <span class="hlFld-ContribAuthor ">Mario  Varasi</span>, and <span class="hlFld-ContribAuthor ">Ciro  Mercurio</span>  . </span><span class="cited-content_cbyCitation_article-title">Thieno[3,2-b]pyrrole-5-carboxamides as New Reversible Inhibitors of Histone Lysine Demethylase KDM1A/LSD1. Part 2: Structure-Based Drug Design and Structure–Activity Relationship. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2017,</strong> <em>60 </em>
                                    (5)
                                     , 1693-1715. <a href="https://doi.org/10.1021/acs.jmedchem.6b01019" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b01019</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b01019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.6b01019%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DThieno%25255B3%25252C2-b%25255Dpyrrole-5-carboxamides%252Bas%252BNew%252BReversible%252BInhibitors%252Bof%252BHistone%252BLysine%252BDemethylase%252BKDM1A%25252FLSD1.%252BPart%252B2%25253A%252BStructure-Based%252BDrug%252BDesign%252Band%252BStructure%2525E2%252580%252593Activity%252BRelationship%26aulast%3DVianello%26aufirst%3DPaola%26date%3D2017%26date%3D2017%26date%3D2016%26date%3D26072016%26date%3D27022017%26date%3D09032017%26date%3D10022017%26volume%3D60%26issue%3D5%26spage%3D1693%26epage%3D1715" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiangyu  Zhang</span>, <span class="hlFld-ContribAuthor ">Hailan  Huang</span>, <span class="hlFld-ContribAuthor ">Ziheng  Zhang</span>, <span class="hlFld-ContribAuthor ">Jiangkun  Yan</span>, <span class="hlFld-ContribAuthor ">Tianxiao  Wu</span>, <span class="hlFld-ContribAuthor ">Wenbo  Yin</span>, <span class="hlFld-ContribAuthor ">Yixiang  Sun</span>, <span class="hlFld-ContribAuthor ">Xinran  Wang</span>, <span class="hlFld-ContribAuthor ">Yanting  Gu</span>, <span class="hlFld-ContribAuthor ">Dongmei  Zhao</span>, <span class="hlFld-ContribAuthor ">Maosheng  Cheng</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis and biological evaluation of novel benzofuran derivatives as potent LSD1 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>220 </em>, 113501. <a href="https://doi.org/10.1016/j.ejmech.2021.113501" title="DOI URL">https://doi.org/10.1016/j.ejmech.2021.113501</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2021.113501&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2021.113501%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDesign%25252C%252Bsynthesis%252Band%252Bbiological%252Bevaluation%252Bof%252Bnovel%252Bbenzofuran%252Bderivatives%252Bas%252Bpotent%252BLSD1%252Binhibitors%26aulast%3DZhang%26aufirst%3DXiangyu%26date%3D2021%26volume%3D220%26spage%3D113501" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiangyu  Zhang</span>, <span class="hlFld-ContribAuthor ">Jiangkun  Yan</span>, <span class="hlFld-ContribAuthor ">Hanxun  Wang</span>, <span class="hlFld-ContribAuthor ">Ying  Wang</span>, <span class="hlFld-ContribAuthor ">Jian  Wang</span>, <span class="hlFld-ContribAuthor ">Dongmei  Zhao</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular docking, 3D-QSAR, and molecular dynamics simulations of thieno[3,2-b]pyrrole derivatives against anticancer targets of KDM1A/LSD1. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biomolecular Structure and Dynamics</span><span> <strong>2021,</strong> <em>39 </em>
                                    (4)
                                     , 1189-1202. <a href="https://doi.org/10.1080/07391102.2020.1726819" title="DOI URL">https://doi.org/10.1080/07391102.2020.1726819</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/07391102.2020.1726819&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F07391102.2020.1726819%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biomolecular%2520Structure%2520and%2520Dynamics%26atitle%3DMolecular%252Bdocking%25252C%252B3D-QSAR%25252C%252Band%252Bmolecular%252Bdynamics%252Bsimulations%252Bof%252Bthieno%25255B3%25252C2-b%25255Dpyrrole%252Bderivatives%252Bagainst%252Banticancer%252Btargets%252Bof%252BKDM1A%25252FLSD1%26aulast%3DZhang%26aufirst%3DXiangyu%26date%3D2021%26date%3D2020%26volume%3D39%26issue%3D4%26spage%3D1189%26epage%3D1202" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shujing  Zhang</span>, <span class="hlFld-ContribAuthor ">Menghan  Liu</span>, <span class="hlFld-ContribAuthor ">Yongfang  Yao</span>, <span class="hlFld-ContribAuthor ">Bin  Yu</span>, <span class="hlFld-ContribAuthor ">Hongmin  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting LSD1 for acute myeloid leukemia (AML) treatment. </span><span class="cited-content_cbyCitation_journal-name">Pharmacological Research</span><span> <strong>2021,</strong> <em>164 </em>, 105335. <a href="https://doi.org/10.1016/j.phrs.2020.105335" title="DOI URL">https://doi.org/10.1016/j.phrs.2020.105335</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.phrs.2020.105335&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.phrs.2020.105335%26sid%3Dliteratum%253Aachs%26jtitle%3DPharmacological%2520Research%26atitle%3DTargeting%252BLSD1%252Bfor%252Bacute%252Bmyeloid%252Bleukemia%252B%252528AML%252529%252Btreatment%26aulast%3DZhang%26aufirst%3DShujing%26date%3D2021%26volume%3D164%26spage%3D105335" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Eleni  Pitsillou</span>, <span class="hlFld-ContribAuthor ">Julia  Liang</span>, <span class="hlFld-ContribAuthor ">Andrew  Hung</span>, <span class="hlFld-ContribAuthor ">Tom C.  Karagiannis</span>. </span><span class="cited-content_cbyCitation_article-title">The circadian machinery links metabolic disorders and depression: A review of pathways, proteins and potential pharmacological interventions. </span><span class="cited-content_cbyCitation_journal-name">Life Sciences</span><span> <strong>2021,</strong> <em>265 </em>, 118809. <a href="https://doi.org/10.1016/j.lfs.2020.118809" title="DOI URL">https://doi.org/10.1016/j.lfs.2020.118809</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.lfs.2020.118809&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.lfs.2020.118809%26sid%3Dliteratum%253Aachs%26jtitle%3DLife%2520Sciences%26atitle%3DThe%252Bcircadian%252Bmachinery%252Blinks%252Bmetabolic%252Bdisorders%252Band%252Bdepression%25253A%252BA%252Breview%252Bof%252Bpathways%25252C%252Bproteins%252Band%252Bpotential%252Bpharmacological%252Binterventions%26aulast%3DPitsillou%26aufirst%3DEleni%26date%3D2021%26volume%3D265%26spage%3D118809" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ao  Shen</span>, <span class="hlFld-ContribAuthor ">Xi-Yong  Yu</span>. </span><span class="cited-content_cbyCitation_article-title">Histone lysine demethylase inhibitor (HDMi) as chemo-sensitizing agent. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 41-55. <a href="https://doi.org/10.1016/B978-0-12-823684-0.00007-4" title="DOI URL">https://doi.org/10.1016/B978-0-12-823684-0.00007-4</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-823684-0.00007-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-823684-0.00007-4%26sid%3Dliteratum%253Aachs%26atitle%3DHistone%252Blysine%252Bdemethylase%252Binhibitor%252B%252528HDMi%252529%252Bas%252Bchemo-sensitizing%252Bagent%26aulast%3DShen%26aufirst%3DAo%26date%3D2021%26spage%3D41%26epage%3D55%26pub%3DElsevier%26atitle%3DEpigenetic%252BRegulation%252Bin%252BOvercoming%252BChemoresistance%26date%3D2021%26volume%3D128" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Samir  Mehndiratta</span>, <span class="hlFld-ContribAuthor ">Jing-Ping  Liou</span>. </span><span class="cited-content_cbyCitation_article-title">Histone lysine specific demethylase 1 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">RSC Medicinal Chemistry</span><span> <strong>2020,</strong> <em>11 </em>
                                    (9)
                                     , 969-981. <a href="https://doi.org/10.1039/D0MD00141D" title="DOI URL">https://doi.org/10.1039/D0MD00141D</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0MD00141D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0MD00141D%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Medicinal%2520Chemistry%26atitle%3DHistone%252Blysine%252Bspecific%252Bdemethylase%252B1%252Binhibitors%26aulast%3DMehndiratta%26aufirst%3DSamir%26date%3D2020%26date%3D2020%26volume%3D11%26issue%3D9%26spage%3D969%26epage%3D981" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Liyun  Liang</span>, <span class="hlFld-ContribAuthor ">Haiwen  Wang</span>, <span class="hlFld-ContribAuthor ">Yongliang  Du</span>, <span class="hlFld-ContribAuthor ">Bingling  Luo</span>, <span class="hlFld-ContribAuthor ">Ning  Meng</span>, <span class="hlFld-ContribAuthor ">Meifeng  Cen</span>, <span class="hlFld-ContribAuthor ">Peng  Huang</span>, <span class="hlFld-ContribAuthor ">A.  Ganesan</span>, <span class="hlFld-ContribAuthor ">Shijun  Wen</span>. </span><span class="cited-content_cbyCitation_article-title">New tranylcypromine derivatives containing sulfonamide motif as potent LSD1 inhibitors to target acute myeloid leukemia: Design, synthesis and biological evaluation. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic Chemistry</span><span> <strong>2020,</strong> <em>99 </em>, 103808. <a href="https://doi.org/10.1016/j.bioorg.2020.103808" title="DOI URL">https://doi.org/10.1016/j.bioorg.2020.103808</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bioorg.2020.103808&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bioorg.2020.103808%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520Chemistry%26atitle%3DNew%252Btranylcypromine%252Bderivatives%252Bcontaining%252Bsulfonamide%252Bmotif%252Bas%252Bpotent%252BLSD1%252Binhibitors%252Bto%252Btarget%252Bacute%252Bmyeloid%252Bleukemia%25253A%252BDesign%25252C%252Bsynthesis%252Band%252Bbiological%252Bevaluation%26aulast%3DLiang%26aufirst%3DLiyun%26date%3D2020%26volume%3D99%26spage%3D103808" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Eleni  Pitsillou</span>, <span class="hlFld-ContribAuthor ">Julia  Liang</span>, <span class="hlFld-ContribAuthor ">Andrew  Hung</span>, <span class="hlFld-ContribAuthor ">Tom C.  Karagiannis</span>. </span><span class="cited-content_cbyCitation_article-title">Chromatin modification by olive phenolics: In silico molecular docking studies utilising the phenolic groups categorised in the OliveNet™ database against lysine specific demethylase enzymes. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular Graphics and Modelling</span><span> <strong>2020,</strong> <em>97 </em>, 107575. <a href="https://doi.org/10.1016/j.jmgm.2020.107575" title="DOI URL">https://doi.org/10.1016/j.jmgm.2020.107575</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jmgm.2020.107575&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jmgm.2020.107575%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520Graphics%2520and%2520Modelling%26atitle%3DChromatin%252Bmodification%252Bby%252Bolive%252Bphenolics%25253A%252BIn%252Bsilico%252Bmolecular%252Bdocking%252Bstudies%252Butilising%252Bthe%252Bphenolic%252Bgroups%252Bcategorised%252Bin%252Bthe%252BOliveNet%2525E2%252584%2525A2%252Bdatabase%252Bagainst%252Blysine%252Bspecific%252Bdemethylase%252Benzymes%26aulast%3DPitsillou%26aufirst%3DEleni%26date%3D2020%26volume%3D97%26spage%3D107575" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xinran  Wang</span>, <span class="hlFld-ContribAuthor ">Cai  Zhang</span>, <span class="hlFld-ContribAuthor ">Xiangyu  Zhang</span>, <span class="hlFld-ContribAuthor ">Jiangkun  Yan</span>, <span class="hlFld-ContribAuthor ">Jiming  Wang</span>, <span class="hlFld-ContribAuthor ">Qinwen  Jiang</span>, <span class="hlFld-ContribAuthor ">Liyu  Zhao</span>, <span class="hlFld-ContribAuthor ">Dongmei  Zhao</span>, <span class="hlFld-ContribAuthor ">Maosheng  Cheng</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis and biological evaluation of tetrahydroquinoline-based reversible LSD1 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>194 </em>, 112243. <a href="https://doi.org/10.1016/j.ejmech.2020.112243" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112243</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112243&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112243%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDesign%25252C%252Bsynthesis%252Band%252Bbiological%252Bevaluation%252Bof%252Btetrahydroquinoline-based%252Breversible%252BLSD1%252Binhibitors%26aulast%3DWang%26aufirst%3DXinran%26date%3D2020%26volume%3D194%26spage%3D112243" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yongtao  Xu</span>, <span class="hlFld-ContribAuthor ">Zihao  He</span>, <span class="hlFld-ContribAuthor ">Hongyi  Liu</span>, <span class="hlFld-ContribAuthor ">Yifan  Chen</span>, <span class="hlFld-ContribAuthor ">Yunlong  Gao</span>, <span class="hlFld-ContribAuthor ">Songjie  Zhang</span>, <span class="hlFld-ContribAuthor ">Meiting  Wang</span>, <span class="hlFld-ContribAuthor ">Xiaoyuan  Lu</span>, <span class="hlFld-ContribAuthor ">Chang  Wang</span>, <span class="hlFld-ContribAuthor ">Zongya  Zhao</span>, <span class="hlFld-ContribAuthor ">Yan  Liu</span>, <span class="hlFld-ContribAuthor ">Junqiang  Zhao</span>, <span class="hlFld-ContribAuthor ">Yi  Yu</span>, <span class="hlFld-ContribAuthor ">Min  Yang</span>. </span><span class="cited-content_cbyCitation_article-title">3D-QSAR, molecular docking, and molecular dynamics simulation study of thieno[3,2-
              b
              ]pyrrole-5-carboxamide derivatives as LSD1 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">RSC Advances</span><span> <strong>2020,</strong> <em>10 </em>
                                    (12)
                                     , 6927-6943. <a href="https://doi.org/10.1039/C9RA10085G" title="DOI URL">https://doi.org/10.1039/C9RA10085G</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C9RA10085G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC9RA10085G%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Advances%26atitle%3D3D-QSAR%25252C%252Bmolecular%252Bdocking%25252C%252Band%252Bmolecular%252Bdynamics%252Bsimulation%252Bstudy%252Bof%252Bthieno%25255B3%25252C2-%252Bb%252B%25255Dpyrrole-5-carboxamide%252Bderivatives%252Bas%252BLSD1%252Binhibitors%26aulast%3DXu%26aufirst%3DYongtao%26date%3D2020%26date%3D2020%26volume%3D10%26issue%3D12%26spage%3D6927%26epage%3D6943" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhong-Hua  Li</span>, <span class="hlFld-ContribAuthor ">Jin-Lian  Ma</span>, <span class="hlFld-ContribAuthor ">Gai-Zhi  Liu</span>, <span class="hlFld-ContribAuthor ">Xin-Hui  Zhang</span>, <span class="hlFld-ContribAuthor ">Ting-Ting  Qin</span>, <span class="hlFld-ContribAuthor ">Wei-Hong  Ren</span>, <span class="hlFld-ContribAuthor ">Tao-Qian  Zhao</span>, <span class="hlFld-ContribAuthor ">Xiao-Hui  Chen</span>, <span class="hlFld-ContribAuthor ">Zhen-Qiang  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">[1,2,3]Triazolo[4,5-d]pyrimidine derivatives incorporating (thio)urea moiety as a novel scaffold for LSD1 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>187 </em>, 111989. <a href="https://doi.org/10.1016/j.ejmech.2019.111989" title="DOI URL">https://doi.org/10.1016/j.ejmech.2019.111989</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2019.111989&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2019.111989%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3D%25255B1%25252C2%25252C3%25255DTriazolo%25255B4%25252C5-d%25255Dpyrimidine%252Bderivatives%252Bincorporating%252B%252528thio%252529urea%252Bmoiety%252Bas%252Ba%252Bnovel%252Bscaffold%252Bfor%252BLSD1%252Binhibitors%26aulast%3DLi%26aufirst%3DZhong-Hua%26date%3D2020%26volume%3D187%26spage%3D111989" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yuan  Fang</span>, <span class="hlFld-ContribAuthor ">Guochao  Liao</span>, <span class="hlFld-ContribAuthor ">Bin  Yu</span>. </span><span class="cited-content_cbyCitation_article-title">LSD1/KDM1A inhibitors in clinical trials: advances and prospects. </span><span class="cited-content_cbyCitation_journal-name">Journal of Hematology & Oncology</span><span> <strong>2019,</strong> <em>12 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13045-019-0811-9" title="DOI URL">https://doi.org/10.1186/s13045-019-0811-9</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13045-019-0811-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13045-019-0811-9%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Hematology%2520%2526%2520Oncology%26atitle%3DLSD1%25252FKDM1A%252Binhibitors%252Bin%252Bclinical%252Btrials%25253A%252Badvances%252Band%252Bprospects%26aulast%3DFang%26aufirst%3DYuan%26date%3D2019%26date%3D2019%26volume%3D12%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yongtao  Xu</span>, <span class="hlFld-ContribAuthor ">Zihao  He</span>, <span class="hlFld-ContribAuthor ">Min  Yang</span>, <span class="hlFld-ContribAuthor ">Yunlong  Gao</span>, <span class="hlFld-ContribAuthor ">Linfeng  Jin</span>, <span class="hlFld-ContribAuthor ">Meiting  Wang</span>, <span class="hlFld-ContribAuthor ">Yichao  Zheng</span>, <span class="hlFld-ContribAuthor ">Xiaoyuan  Lu</span>, <span class="hlFld-ContribAuthor ">Songjie  Zhang</span>, <span class="hlFld-ContribAuthor ">Chang  Wang</span>, <span class="hlFld-ContribAuthor ">Zongya  Zhao</span>, <span class="hlFld-ContribAuthor ">Junqiang  Zhao</span>, <span class="hlFld-ContribAuthor ">Qinghe  Gao</span>, <span class="hlFld-ContribAuthor ">Yingchao  Duan</span>. </span><span class="cited-content_cbyCitation_article-title">Investigating the Binding Mode of Reversible LSD1 Inhibitors Derived from Stilbene Derivatives by 3D-QSAR, Molecular Docking, and Molecular Dynamics Simulation. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2019,</strong> <em>24 </em>
                                    (24)
                                     , 4479. <a href="https://doi.org/10.3390/molecules24244479" title="DOI URL">https://doi.org/10.3390/molecules24244479</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules24244479&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules24244479%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DInvestigating%252Bthe%252BBinding%252BMode%252Bof%252BReversible%252BLSD1%252BInhibitors%252BDerived%252Bfrom%252BStilbene%252BDerivatives%252Bby%252B3D-QSAR%25252C%252BMolecular%252BDocking%25252C%252Band%252BMolecular%252BDynamics%252BSimulation%26aulast%3DXu%26aufirst%3DYongtao%26date%3D2019%26date%3D2019%26volume%3D24%26issue%3D24%26spage%3D4479" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xu-Dong  Sun</span>, <span class="hlFld-ContribAuthor ">Yi-Chao  Zheng</span>, <span class="hlFld-ContribAuthor ">Chao-Ya  Ma</span>, <span class="hlFld-ContribAuthor ">Jing  Yang</span>, <span class="hlFld-ContribAuthor ">Qi-Bing  Gao</span>, <span class="hlFld-ContribAuthor ">Ying  Yan</span>, <span class="hlFld-ContribAuthor ">Zhi-Zheng  Wang</span>, <span class="hlFld-ContribAuthor ">Wen  Li</span>, <span class="hlFld-ContribAuthor ">Wen  Zhao</span>, <span class="hlFld-ContribAuthor ">Hong-Min  Liu</span>, <span class="hlFld-ContribAuthor ">Lina  Ding</span>. </span><span class="cited-content_cbyCitation_article-title">Identifying the novel inhibitors of lysine-specific demethylase 1 (LSD1) combining pharmacophore-based and structure-based virtual screening. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biomolecular Structure and Dynamics</span><span> <strong>2019,</strong> <em>37 </em>
                                    (16)
                                     , 4200-4214. <a href="https://doi.org/10.1080/07391102.2018.1538903" title="DOI URL">https://doi.org/10.1080/07391102.2018.1538903</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/07391102.2018.1538903&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F07391102.2018.1538903%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biomolecular%2520Structure%2520and%2520Dynamics%26atitle%3DIdentifying%252Bthe%252Bnovel%252Binhibitors%252Bof%252Blysine-specific%252Bdemethylase%252B1%252B%252528LSD1%252529%252Bcombining%252Bpharmacophore-based%252Band%252Bstructure-based%252Bvirtual%252Bscreening%26aulast%3DSun%26aufirst%3DXu-Dong%26date%3D2019%26date%3D2018%26volume%3D37%26issue%3D16%26spage%3D4200%26epage%3D4214" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhi-Zheng  Wang</span>, <span class="hlFld-ContribAuthor ">Jing  Yang</span>, <span class="hlFld-ContribAuthor ">Xu-Dong  Sun</span>, <span class="hlFld-ContribAuthor ">Chao-Ya  Ma</span>, <span class="hlFld-ContribAuthor ">Qi-Bing  Gao</span>, <span class="hlFld-ContribAuthor ">Lina  Ding</span>, <span class="hlFld-ContribAuthor ">Hong-Min  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Probing the binding mechanism of substituted pyridine derivatives as effective and selective lysine-specific demethylase 1 inhibitors using 3D-QSAR, molecular docking and molecular dynamics simulations. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biomolecular Structure and Dynamics</span><span> <strong>2019,</strong> <em>37 </em>
                                    (13)
                                     , 3482-3495. <a href="https://doi.org/10.1080/07391102.2018.1518158" title="DOI URL">https://doi.org/10.1080/07391102.2018.1518158</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/07391102.2018.1518158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F07391102.2018.1518158%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biomolecular%2520Structure%2520and%2520Dynamics%26atitle%3DProbing%252Bthe%252Bbinding%252Bmechanism%252Bof%252Bsubstituted%252Bpyridine%252Bderivatives%252Bas%252Beffective%252Band%252Bselective%252Blysine-specific%252Bdemethylase%252B1%252Binhibitors%252Busing%252B3D-QSAR%25252C%252Bmolecular%252Bdocking%252Band%252Bmolecular%252Bdynamics%252Bsimulations%26aulast%3DWang%26aufirst%3DZhi-Zheng%26date%3D2019%26date%3D2018%26volume%3D37%26issue%3D13%26spage%3D3482%26epage%3D3495" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shuai  Wang</span>, <span class="hlFld-ContribAuthor ">Zhong-Rui  Li</span>, <span class="hlFld-ContribAuthor ">Feng-Zhi  Suo</span>, <span class="hlFld-ContribAuthor ">Xiao-Han  Yuan</span>, <span class="hlFld-ContribAuthor ">Bin  Yu</span>, <span class="hlFld-ContribAuthor ">Hong-Min  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis, structure-activity relationship studies and biological characterization of new [1,2,4]triazolo[1,5-a]pyrimidine-based LSD1/KDM1A inhibitors. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>167 </em>, 388-401. <a href="https://doi.org/10.1016/j.ejmech.2019.02.039" title="DOI URL">https://doi.org/10.1016/j.ejmech.2019.02.039</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2019.02.039&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2019.02.039%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DSynthesis%25252C%252Bstructure-activity%252Brelationship%252Bstudies%252Band%252Bbiological%252Bcharacterization%252Bof%252Bnew%252B%25255B1%25252C2%25252C4%25255Dtriazolo%25255B1%25252C5-a%25255Dpyrimidine-based%252BLSD1%25252FKDM1A%252Binhibitors%26aulast%3DWang%26aufirst%3DShuai%26date%3D2019%26volume%3D167%26spage%3D388%26epage%3D401" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xuan-Xuan  Du</span>, <span class="hlFld-ContribAuthor ">Quan-Xing  Zi</span>, <span class="hlFld-ContribAuthor ">Yu-Meng  Wu</span>, <span class="hlFld-ContribAuthor ">Yi  Jin</span>, <span class="hlFld-ContribAuthor ">Jun  Lin</span>, <span class="hlFld-ContribAuthor ">Sheng-Jiao  Yan</span>. </span><span class="cited-content_cbyCitation_article-title">An environmentally benign multi-component reaction: regioselective synthesis of fluorinated 2-aminopyridines using diverse properties of the nitro group. </span><span class="cited-content_cbyCitation_journal-name">Green Chemistry</span><span> <strong>2019,</strong> <em>21 </em>
                                    (6)
                                     , 1505-1516. <a href="https://doi.org/10.1039/C8GC03698E" title="DOI URL">https://doi.org/10.1039/C8GC03698E</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C8GC03698E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC8GC03698E%26sid%3Dliteratum%253Aachs%26jtitle%3DGreen%2520Chemistry%26atitle%3DAn%252Benvironmentally%252Bbenign%252Bmulti-component%252Breaction%25253A%252Bregioselective%252Bsynthesis%252Bof%252Bfluorinated%252B2-aminopyridines%252Busing%252Bdiverse%252Bproperties%252Bof%252Bthe%252Bnitro%252Bgroup%26aulast%3DDu%26aufirst%3DXuan-Xuan%26date%3D2019%26date%3D2019%26volume%3D21%26issue%3D6%26spage%3D1505%26epage%3D1516" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Adam  Lee</span>, <span class="hlFld-ContribAuthor ">M. Teresa  Borrello</span>, <span class="hlFld-ContribAuthor ">A.  Ganesan</span>. </span><span class="cited-content_cbyCitation_article-title">LSD
              (Lysine‐Specific Demethylase): A Decade‐Long Trip from Discovery to Clinical Trials. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 221-261. <a href="https://doi.org/10.1002/9783527809257.ch10" title="DOI URL">https://doi.org/10.1002/9783527809257.ch10</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9783527809257.ch10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9783527809257.ch10%26sid%3Dliteratum%253Aachs%26atitle%3DLSD%252B%252528Lysine%2525E2%252580%252590Specific%252BDemethylase%252529%25253A%252BA%252BDecade%2525E2%252580%252590Long%252BTrip%252Bfrom%252BDiscovery%252Bto%252BClinical%252BTrials%26aulast%3DLee%26aufirst%3DAdam%26date%3D2019%26date%3D2019%26spage%3D221%26epage%3D261%26pub%3DWiley%26atitle%3DEpigenetic%252BDrug%252BDiscovery%26aulast%3DSippl%26aufirst%3DWolfgang%26date%3D2019%26date%3D2019%26volume%3D115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Christian  Feller</span>, <span class="hlFld-ContribAuthor ">David  Weigt</span>, <span class="hlFld-ContribAuthor ">Carsten  Hopf</span>. </span><span class="cited-content_cbyCitation_article-title">Mass Spectrometry and Chemical Biology in Epigenetics Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 79-106. <a href="https://doi.org/10.1002/9783527809257.ch4" title="DOI URL">https://doi.org/10.1002/9783527809257.ch4</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9783527809257.ch4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9783527809257.ch4%26sid%3Dliteratum%253Aachs%26atitle%3DMass%252BSpectrometry%252Band%252BChemical%252BBiology%252Bin%252BEpigenetics%252BDrug%252BDiscovery%26aulast%3DFeller%26aufirst%3DChristian%26date%3D2019%26date%3D2019%26spage%3D79%26epage%3D106%26pub%3DWiley%26atitle%3DEpigenetic%252BDrug%252BDiscovery%26aulast%3DSippl%26aufirst%3DWolfgang%26date%3D2019%26date%3D2019%26volume%3D11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Takayoshi  Suzuki</span>. </span><span class="cited-content_cbyCitation_article-title">Lysine-Specific Histone Demethylases 1/2 (LSD1/2) and Their Inhibitors. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 197-219. <a href="https://doi.org/10.1007/7355_2019_74" title="DOI URL">https://doi.org/10.1007/7355_2019_74</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/7355_2019_74&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F7355_2019_74%26sid%3Dliteratum%253Aachs%26atitle%3DLysine-Specific%252BHistone%252BDemethylases%252B1%25252F2%252B%252528LSD1%25252F2%252529%252Band%252BTheir%252BInhibitors%26aulast%3DSuzuki%26aufirst%3DTakayoshi%26date%3D2019%26date%3D2020%26spage%3D197%26epage%3D219%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DChemical%252BEpigenetics%26aulast%3DMai%26aufirst%3DAntonello%26date%3D2020%26volume%3D33" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dina  Robaa</span>, <span class="hlFld-ContribAuthor ">Jelena  Melesina</span>, <span class="hlFld-ContribAuthor ">Chiara  Luise</span>, <span class="hlFld-ContribAuthor ">Wolfgang  Sippl</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-Based Design of Epigenetic Inhibitors. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 455-486. <a href="https://doi.org/10.1007/7355_2019_80" title="DOI URL">https://doi.org/10.1007/7355_2019_80</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/7355_2019_80&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F7355_2019_80%26sid%3Dliteratum%253Aachs%26atitle%3DStructure-Based%252BDesign%252Bof%252BEpigenetic%252BInhibitors%26aulast%3DRobaa%26aufirst%3DDina%26date%3D2019%26date%3D2019%26spage%3D455%26epage%3D486%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DChemical%252BEpigenetics%26aulast%3DMai%26aufirst%3DAntonello%26date%3D2020%26volume%3D33" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Qi-Sheng  Ma</span>, <span class="hlFld-ContribAuthor ">Yongfang  Yao</span>, <span class="hlFld-ContribAuthor ">Yi-Chao  Zheng</span>, <span class="hlFld-ContribAuthor ">Siqi  Feng</span>, <span class="hlFld-ContribAuthor ">Junbiao  Chang</span>, <span class="hlFld-ContribAuthor ">Bin  Yu</span>, <span class="hlFld-ContribAuthor ">Hong-Min  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Ligand-based design, synthesis and biological evaluation of xanthine derivatives as LSD1/KDM1A inhibitors. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>162 </em>, 555-567. <a href="https://doi.org/10.1016/j.ejmech.2018.11.035" title="DOI URL">https://doi.org/10.1016/j.ejmech.2018.11.035</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2018.11.035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2018.11.035%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DLigand-based%252Bdesign%25252C%252Bsynthesis%252Band%252Bbiological%252Bevaluation%252Bof%252Bxanthine%252Bderivatives%252Bas%252BLSD1%25252FKDM1A%252Binhibitors%26aulast%3DMa%26aufirst%3DQi-Sheng%26date%3D2019%26volume%3D162%26spage%3D555%26epage%3D567" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xudong  Sun</span>, <span class="hlFld-ContribAuthor ">Lina  Ding</span>, <span class="hlFld-ContribAuthor ">Hong-Min  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Probing the binding mode and unbinding mechanism of LSD1 inhibitors by combined computational methods. </span><span class="cited-content_cbyCitation_journal-name">Physical Chemistry Chemical Physics</span><span> <strong>2018,</strong> <em>20 </em>
                                    (47)
                                     , 29833-29846. <a href="https://doi.org/10.1039/C8CP03090A" title="DOI URL">https://doi.org/10.1039/C8CP03090A</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C8CP03090A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC8CP03090A%26sid%3Dliteratum%253Aachs%26jtitle%3DPhysical%2520Chemistry%2520Chemical%2520Physics%26atitle%3DProbing%252Bthe%252Bbinding%252Bmode%252Band%252Bunbinding%252Bmechanism%252Bof%252BLSD1%252Binhibitors%252Bby%252Bcombined%252Bcomputational%252Bmethods%26aulast%3DSun%26aufirst%3DXudong%26date%3D2018%26date%3D2018%26volume%3D20%26issue%3D47%26spage%3D29833%26epage%3D29846" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yingchao  Duan</span>, <span class="hlFld-ContribAuthor ">Wenping  Qin</span>, <span class="hlFld-ContribAuthor ">Fengzhi  Suo</span>, <span class="hlFld-ContribAuthor ">Xiaoyu  Zhai</span>, <span class="hlFld-ContribAuthor ">Yuanyuan  Guan</span>, <span class="hlFld-ContribAuthor ">Xiaojuan  Wang</span>, <span class="hlFld-ContribAuthor ">Yichao  Zheng</span>, <span class="hlFld-ContribAuthor ">Hongmin  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis and in vitro evaluation of stilbene derivatives as novel LSD1 inhibitors for AML therapy. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2018,</strong> <em>26 </em>
                                    (23-24)
                                     , 6000-6014. <a href="https://doi.org/10.1016/j.bmc.2018.10.037" title="DOI URL">https://doi.org/10.1016/j.bmc.2018.10.037</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2018.10.037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2018.10.037%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DDesign%25252C%252Bsynthesis%252Band%252Bin%252Bvitro%252Bevaluation%252Bof%252Bstilbene%252Bderivatives%252Bas%252Bnovel%252BLSD1%252Binhibitors%252Bfor%252BAML%252Btherapy%26aulast%3DDuan%26aufirst%3DYingchao%26date%3D2018%26volume%3D26%26issue%3D23-24%26spage%3D6000%26epage%3D6014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Siyuan  Xu</span>, <span class="hlFld-ContribAuthor ">Chen  Zhou</span>, <span class="hlFld-ContribAuthor ">Rongfeng  Liu</span>, <span class="hlFld-ContribAuthor ">Qihua  Zhu</span>, <span class="hlFld-ContribAuthor ">Yungen  Xu</span>, <span class="hlFld-ContribAuthor ">Fei  Lan</span>, <span class="hlFld-ContribAuthor ">Xiaoming  Zha</span>. </span><span class="cited-content_cbyCitation_article-title">Optimization of 5-arylidene barbiturates as potent, selective, reversible LSD1 inhibitors for the treatment of acute promyelocytic leukemia. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2018,</strong> <em>26 </em>
                                    (17)
                                     , 4871-4880. <a href="https://doi.org/10.1016/j.bmc.2018.08.026" title="DOI URL">https://doi.org/10.1016/j.bmc.2018.08.026</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2018.08.026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2018.08.026%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DOptimization%252Bof%252B5-arylidene%252Bbarbiturates%252Bas%252Bpotent%25252C%252Bselective%25252C%252Breversible%252BLSD1%252Binhibitors%252Bfor%252Bthe%252Btreatment%252Bof%252Bacute%252Bpromyelocytic%252Bleukemia%26aulast%3DXu%26aufirst%3DSiyuan%26date%3D2018%26volume%3D26%26issue%3D17%26spage%3D4871%26epage%3D4880" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jiayue  Xi</span>, <span class="hlFld-ContribAuthor ">Siyuan  Xu</span>, <span class="hlFld-ContribAuthor ">Lulu  Zhang</span>, <span class="hlFld-ContribAuthor ">Xueyuan  Bi</span>, <span class="hlFld-ContribAuthor ">Yanshen  Ren</span>, <span class="hlFld-ContribAuthor ">Yu-Chih  Liu</span>, <span class="hlFld-ContribAuthor ">Yueqing  Gu</span>, <span class="hlFld-ContribAuthor ">Yungen  Xu</span>, <span class="hlFld-ContribAuthor ">Fei  Lan</span>, <span class="hlFld-ContribAuthor ">Xiaoming  Zha</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis and biological activity of 4-(4-benzyloxy)phenoxypiperidines as selective and reversible LSD1 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic Chemistry</span><span> <strong>2018,</strong> <em>78 </em>, 7-16. <a href="https://doi.org/10.1016/j.bioorg.2018.02.016" title="DOI URL">https://doi.org/10.1016/j.bioorg.2018.02.016</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bioorg.2018.02.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bioorg.2018.02.016%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520Chemistry%26atitle%3DDesign%25252C%252Bsynthesis%252Band%252Bbiological%252Bactivity%252Bof%252B4-%2525284-benzyloxy%252529phenoxypiperidines%252Bas%252Bselective%252Band%252Breversible%252BLSD1%252Binhibitors%26aulast%3DXi%26aufirst%3DJiayue%26date%3D2018%26volume%3D78%26spage%3D7%26epage%3D16" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hideaki  Niwa</span>, <span class="hlFld-ContribAuthor ">Shin  Sato</span>, <span class="hlFld-ContribAuthor ">Tomoko  Hashimoto</span>, <span class="hlFld-ContribAuthor ">Kenji  Matsuno</span>, <span class="hlFld-ContribAuthor ">Takashi  Umehara</span>. </span><span class="cited-content_cbyCitation_article-title">Crystal Structure of LSD1 in Complex with 4-[5-(Piperidin-4-ylmethoxy)-2-(p-tolyl)pyridin-3-yl]benzonitrile. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2018,</strong> <em>23 </em>
                                    (7)
                                     , 1538. <a href="https://doi.org/10.3390/molecules23071538" title="DOI URL">https://doi.org/10.3390/molecules23071538</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules23071538&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules23071538%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DCrystal%252BStructure%252Bof%252BLSD1%252Bin%252BComplex%252Bwith%252B4-%25255B5-%252528Piperidin-4-ylmethoxy%252529-2-%252528p-tolyl%252529pyridin-3-yl%25255Dbenzonitrile%26aulast%3DNiwa%26aufirst%3DHideaki%26date%3D2018%26date%3D2018%26volume%3D23%26issue%3D7%26spage%3D1538" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ying  Shi</span>, <span class="hlFld-ContribAuthor ">Yan-Ran  Wu</span>, <span class="hlFld-ContribAuthor ">Ming-Bo  Su</span>, <span class="hlFld-ContribAuthor ">Dong-Hao  Shen</span>, <span class="hlFld-ContribAuthor ">Hendra  Gunosewoyo</span>, <span class="hlFld-ContribAuthor ">Fan  Yang</span>, <span class="hlFld-ContribAuthor ">Jia  Li</span>, <span class="hlFld-ContribAuthor ">Jie  Tang</span>, <span class="hlFld-ContribAuthor ">Yu-Bo  Zhou</span>, <span class="hlFld-ContribAuthor ">Li-Fang  Yu</span>. </span><span class="cited-content_cbyCitation_article-title">Novel spirocyclic tranylcypromine derivatives as lysine-specific demethylase 1 (LSD1) inhibitors. </span><span class="cited-content_cbyCitation_journal-name">RSC Advances</span><span> <strong>2018,</strong> <em>8 </em>
                                    (3)
                                     , 1666-1676. <a href="https://doi.org/10.1039/C7RA13097J" title="DOI URL">https://doi.org/10.1039/C7RA13097J</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C7RA13097J&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC7RA13097J%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Advances%26atitle%3DNovel%252Bspirocyclic%252Btranylcypromine%252Bderivatives%252Bas%252Blysine-specific%252Bdemethylase%252B1%252B%252528LSD1%252529%252Binhibitors%26aulast%3DShi%26aufirst%3DYing%26date%3D2018%26date%3D2018%26volume%3D8%26issue%3D3%26spage%3D1666%26epage%3D1676" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Johannes  Schulz-Fincke</span>, <span class="hlFld-ContribAuthor ">Mirjam  Hau</span>, <span class="hlFld-ContribAuthor ">Jessica  Barth</span>, <span class="hlFld-ContribAuthor ">Dina  Robaa</span>, <span class="hlFld-ContribAuthor ">Dominica  Willmann</span>, <span class="hlFld-ContribAuthor ">Andreas  Kürner</span>, <span class="hlFld-ContribAuthor ">Julian  Haas</span>, <span class="hlFld-ContribAuthor ">Gabriele  Greve</span>, <span class="hlFld-ContribAuthor ">Tinka  Haydn</span>, <span class="hlFld-ContribAuthor ">Simone  Fulda</span>, <span class="hlFld-ContribAuthor ">Michael  Lübbert</span>, <span class="hlFld-ContribAuthor ">Steffen  Lüdeke</span>, <span class="hlFld-ContribAuthor ">Tobias  Berg</span>, <span class="hlFld-ContribAuthor ">Wolfgang  Sippl</span>, <span class="hlFld-ContribAuthor ">Roland  Schüle</span>, <span class="hlFld-ContribAuthor ">Manfred  Jung</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-activity studies on N -Substituted tranylcypromine derivatives lead to selective inhibitors of lysine specific demethylase 1 (LSD1) and potent inducers of leukemic cell differentiation. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>144 </em>, 52-67. <a href="https://doi.org/10.1016/j.ejmech.2017.12.001" title="DOI URL">https://doi.org/10.1016/j.ejmech.2017.12.001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2017.12.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2017.12.001%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DStructure-activity%252Bstudies%252Bon%252BN%252B-Substituted%252Btranylcypromine%252Bderivatives%252Blead%252Bto%252Bselective%252Binhibitors%252Bof%252Blysine%252Bspecific%252Bdemethylase%252B1%252B%252528LSD1%252529%252Band%252Bpotent%252Binducers%252Bof%252Bleukemic%252Bcell%252Bdifferentiation%26aulast%3DSchulz-Fincke%26aufirst%3DJohannes%26date%3D2018%26volume%3D144%26spage%3D52%26epage%3D67" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sheraz  Gul</span>. </span><span class="cited-content_cbyCitation_article-title">Epigenetic assays for chemical biology and drug discovery. </span><span class="cited-content_cbyCitation_journal-name">Clinical Epigenetics</span><span> <strong>2017,</strong> <em>9 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13148-017-0342-6" title="DOI URL">https://doi.org/10.1186/s13148-017-0342-6</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13148-017-0342-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13148-017-0342-6%26sid%3Dliteratum%253Aachs%26jtitle%3DClinical%2520Epigenetics%26atitle%3DEpigenetic%252Bassays%252Bfor%252Bchemical%252Bbiology%252Band%252Bdrug%252Bdiscovery%26aulast%3DGul%26aufirst%3DSheraz%26date%3D2017%26date%3D2017%26volume%3D9%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ying-Chao  Duan</span>, <span class="hlFld-ContribAuthor ">Yong-Cheng  Ma</span>, <span class="hlFld-ContribAuthor ">Wen-Ping  Qin</span>, <span class="hlFld-ContribAuthor ">Li-Na  Ding</span>, <span class="hlFld-ContribAuthor ">Yi-Chao  Zheng</span>, <span class="hlFld-ContribAuthor ">Ying-Li  Zhu</span>, <span class="hlFld-ContribAuthor ">Xiao-Yu  Zhai</span>, <span class="hlFld-ContribAuthor ">Jing  Yang</span>, <span class="hlFld-ContribAuthor ">Chao-Ya  Ma</span>, <span class="hlFld-ContribAuthor ">Yuan-Yuan  Guan</span>. </span><span class="cited-content_cbyCitation_article-title">Design and synthesis of tranylcypromine derivatives as novel LSD1/HDACs dual inhibitors for cancer treatment. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2017,</strong> <em>140 </em>, 392-402. <a href="https://doi.org/10.1016/j.ejmech.2017.09.038" title="DOI URL">https://doi.org/10.1016/j.ejmech.2017.09.038</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2017.09.038&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2017.09.038%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDesign%252Band%252Bsynthesis%252Bof%252Btranylcypromine%252Bderivatives%252Bas%252Bnovel%252BLSD1%25252FHDACs%252Bdual%252Binhibitors%252Bfor%252Bcancer%252Btreatment%26aulast%3DDuan%26aufirst%3DYing-Chao%26date%3D2017%26volume%3D140%26spage%3D392%26epage%3D402" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lina  Ding</span>, <span class="hlFld-ContribAuthor ">Zhi-Zheng  Wang</span>, <span class="hlFld-ContribAuthor ">Xu-Dong  Sun</span>, <span class="hlFld-ContribAuthor ">Jing  Yang</span>, <span class="hlFld-ContribAuthor ">Chao-Ya  Ma</span>, <span class="hlFld-ContribAuthor ">Wen  Li</span>, <span class="hlFld-ContribAuthor ">Hong-Min  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">3D-QSAR (CoMFA, CoMSIA), molecular docking and molecular dynamics simulations study of 6-aryl-5-cyano-pyrimidine derivatives to explore the structure requirements of LSD1 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2017,</strong> <em>27 </em>
                                    (15)
                                     , 3521-3528. <a href="https://doi.org/10.1016/j.bmcl.2017.05.065" title="DOI URL">https://doi.org/10.1016/j.bmcl.2017.05.065</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2017.05.065&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2017.05.065%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3D3D-QSAR%252B%252528CoMFA%25252C%252BCoMSIA%252529%25252C%252Bmolecular%252Bdocking%252Band%252Bmolecular%252Bdynamics%252Bsimulations%252Bstudy%252Bof%252B6-aryl-5-cyano-pyrimidine%252Bderivatives%252Bto%252Bexplore%252Bthe%252Bstructure%252Brequirements%252Bof%252BLSD1%252Binhibitors%26aulast%3DDing%26aufirst%3DLina%26date%3D2017%26volume%3D27%26issue%3D15%26spage%3D3521%26epage%3D3528" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiaoli  Fu</span>, <span class="hlFld-ContribAuthor ">Peng  Zhang</span>, <span class="hlFld-ContribAuthor ">Bin  Yu</span>. </span><span class="cited-content_cbyCitation_article-title">Advances toward LSD1 inhibitors for cancer therapy. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2017,</strong> <em>9 </em>
                                    (11)
                                     , 1227-1242. <a href="https://doi.org/10.4155/fmc-2017-0068" title="DOI URL">https://doi.org/10.4155/fmc-2017-0068</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc-2017-0068&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc-2017-0068%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DAdvances%252Btoward%252BLSD1%252Binhibitors%252Bfor%252Bcancer%252Btherapy%26aulast%3DFu%26aufirst%3DXiaoli%26date%3D2017%26volume%3D9%26issue%3D11%26spage%3D1227%26epage%3D1242" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jiayue  Xi</span>, <span class="hlFld-ContribAuthor ">Siyuan  Xu</span>, <span class="hlFld-ContribAuthor ">Liming  Wu</span>, <span class="hlFld-ContribAuthor ">Tianfang  Ma</span>, <span class="hlFld-ContribAuthor ">Rongfeng  Liu</span>, <span class="hlFld-ContribAuthor ">Yu-Chih  Liu</span>, <span class="hlFld-ContribAuthor ">Dawei  Deng</span>, <span class="hlFld-ContribAuthor ">Yueqing  Gu</span>, <span class="hlFld-ContribAuthor ">Jinpei  Zhou</span>, <span class="hlFld-ContribAuthor ">Fei  Lan</span>, <span class="hlFld-ContribAuthor ">Xiaoming  Zha</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis and biological activity of 3-oxoamino-benzenesulfonamides as selective and reversible LSD1 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic Chemistry</span><span> <strong>2017,</strong> <em>72 </em>, 182-189. <a href="https://doi.org/10.1016/j.bioorg.2017.04.006" title="DOI URL">https://doi.org/10.1016/j.bioorg.2017.04.006</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bioorg.2017.04.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bioorg.2017.04.006%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520Chemistry%26atitle%3DDesign%25252C%252Bsynthesis%252Band%252Bbiological%252Bactivity%252Bof%252B3-oxoamino-benzenesulfonamides%252Bas%252Bselective%252Band%252Breversible%252BLSD1%252Binhibitors%26aulast%3DXi%26aufirst%3DJiayue%26date%3D2017%26volume%3D72%26spage%3D182%26epage%3D189" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hideaki  Niwa</span>, <span class="hlFld-ContribAuthor ">Takashi  Umehara</span>. </span><span class="cited-content_cbyCitation_article-title">Structural insight into inhibitors of flavin adenine dinucleotide-dependent lysine demethylases. </span><span class="cited-content_cbyCitation_journal-name">Epigenetics</span><span> <strong>2017,</strong> <em>12 </em>
                                    (5)
                                     , 340-352. <a href="https://doi.org/10.1080/15592294.2017.1290032" title="DOI URL">https://doi.org/10.1080/15592294.2017.1290032</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/15592294.2017.1290032&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F15592294.2017.1290032%26sid%3Dliteratum%253Aachs%26jtitle%3DEpigenetics%26atitle%3DStructural%252Binsight%252Binto%252Binhibitors%252Bof%252Bflavin%252Badenine%252Bdinucleotide-dependent%252Blysine%252Bdemethylases%26aulast%3DNiwa%26aufirst%3DHideaki%26date%3D2017%26date%3D2017%26volume%3D12%26issue%3D5%26spage%3D340%26epage%3D352" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ying-Chao  Duan</span>, <span class="hlFld-ContribAuthor ">Yuan-Yuan  Guan</span>, <span class="hlFld-ContribAuthor ">Xiao-Yu  Zhai</span>, <span class="hlFld-ContribAuthor ">Li-Na  Ding</span>, <span class="hlFld-ContribAuthor ">Wen-Ping  Qin</span>, <span class="hlFld-ContribAuthor ">Dan-Dan  Shen</span>, <span class="hlFld-ContribAuthor ">Xue-Qi  Liu</span>, <span class="hlFld-ContribAuthor ">Xu-Dong  Sun</span>, <span class="hlFld-ContribAuthor ">Yi-Chao  Zheng</span>, <span class="hlFld-ContribAuthor ">Hong-Min  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of resveratrol derivatives as novel LSD1 inhibitors: Design, synthesis and their biological evaluation. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2017,</strong> <em>126 </em>, 246-258. <a href="https://doi.org/10.1016/j.ejmech.2016.11.035" title="DOI URL">https://doi.org/10.1016/j.ejmech.2016.11.035</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2016.11.035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2016.11.035%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Bresveratrol%252Bderivatives%252Bas%252Bnovel%252BLSD1%252Binhibitors%25253A%252BDesign%25252C%252Bsynthesis%252Band%252Btheir%252Bbiological%252Bevaluation%26aulast%3DDuan%26aufirst%3DYing-Chao%26date%3D2017%26volume%3D126%26spage%3D246%26epage%3D258" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ashwini  Jambhekar</span>, <span class="hlFld-ContribAuthor ">Jamie N.  Anastas</span>, <span class="hlFld-ContribAuthor ">Yang  Shi</span>. </span><span class="cited-content_cbyCitation_article-title">Histone Lysine Demethylase Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Cold Spring Harbor Perspectives in Medicine</span><span> <strong>2017,</strong> <em>7 </em>
                                    (1)
                                     , a026484. <a href="https://doi.org/10.1101/cshperspect.a026484" title="DOI URL">https://doi.org/10.1101/cshperspect.a026484</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1101/cshperspect.a026484&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1101%2Fcshperspect.a026484%26sid%3Dliteratum%253Aachs%26jtitle%3DCold%2520Spring%2520Harbor%2520Perspectives%2520in%2520Medicine%26atitle%3DHistone%252BLysine%252BDemethylase%252BInhibitors%26aulast%3DJambhekar%26aufirst%3DAshwini%26date%3D2017%26date%3D2016%26volume%3D7%26issue%3D1%26spage%3Da026484" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yongcheng  Song</span>, <span class="hlFld-ContribAuthor ">Fangrui  Wu</span>, <span class="hlFld-ContribAuthor ">Jingyu  Wu</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting histone methylation for cancer therapy: enzymes, inhibitors, biological activity and perspectives. </span><span class="cited-content_cbyCitation_journal-name">Journal of Hematology & Oncology</span><span> <strong>2016,</strong> <em>9 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13045-016-0279-9" title="DOI URL">https://doi.org/10.1186/s13045-016-0279-9</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13045-016-0279-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13045-016-0279-9%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Hematology%2520%2526%2520Oncology%26atitle%3DTargeting%252Bhistone%252Bmethylation%252Bfor%252Bcancer%252Btherapy%25253A%252Benzymes%25252C%252Binhibitors%25252C%252Bbiological%252Bactivity%252Band%252Bperspectives%26aulast%3DSong%26aufirst%3DYongcheng%26date%3D2016%26date%3D2016%26volume%3D9%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lu  Wang</span>, <span class="hlFld-ContribAuthor ">Gaoyuan  Zhu</span>, <span class="hlFld-ContribAuthor ">Weifang  Tang</span>, <span class="hlFld-ContribAuthor ">Tao  Lu</span>, <span class="hlFld-ContribAuthor ">Ding  Du</span>. </span><span class="cited-content_cbyCitation_article-title">DMAP-promoted in situ activation of bromoacetic acid as a 2-carbon synthon for facile synthesis of pyridines and fused pyridin-2-ones. </span><span class="cited-content_cbyCitation_journal-name">Tetrahedron</span><span> <strong>2016,</strong> <em>72 </em>
                                    (41)
                                     , 6510-6517. <a href="https://doi.org/10.1016/j.tet.2016.08.062" title="DOI URL">https://doi.org/10.1016/j.tet.2016.08.062</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.tet.2016.08.062&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.tet.2016.08.062%26sid%3Dliteratum%253Aachs%26jtitle%3DTetrahedron%26atitle%3DDMAP-promoted%252Bin%252Bsitu%252Bactivation%252Bof%252Bbromoacetic%252Bacid%252Bas%252Ba%252B2-carbon%252Bsynthon%252Bfor%252Bfacile%252Bsynthesis%252Bof%252Bpyridines%252Band%252Bfused%252Bpyridin-2-ones%26aulast%3DWang%26aufirst%3DLu%26date%3D2016%26volume%3D72%26issue%3D41%26spage%3D6510%26epage%3D6517" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Leonid L.  Fershtat</span>, <span class="hlFld-ContribAuthor ">Alexander A.  Larin</span>, <span class="hlFld-ContribAuthor ">Margarita A.  Epishina</span>, <span class="hlFld-ContribAuthor ">Igor V.  Ovchinnikov</span>, <span class="hlFld-ContribAuthor ">Alexander S.  Kulikov</span>, <span class="hlFld-ContribAuthor ">Ivan V.  Ananyev</span>, <span class="hlFld-ContribAuthor ">Nina N.  Makhova</span>. </span><span class="cited-content_cbyCitation_article-title">Design of hybrid heterocyclic systems with a furoxanylpyridine core via tandem hetero-Diels–Alder/retro-Diels–Alder reactions of (1,2,4-triazin-3-yl)furoxans. </span><span class="cited-content_cbyCitation_journal-name">RSC Advances</span><span> <strong>2016,</strong> <em>6 </em>
                                    (37)
                                     , 31526-31539. <a href="https://doi.org/10.1039/C6RA05110C" title="DOI URL">https://doi.org/10.1039/C6RA05110C</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C6RA05110C&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC6RA05110C%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Advances%26atitle%3DDesign%252Bof%252Bhybrid%252Bheterocyclic%252Bsystems%252Bwith%252Ba%252Bfuroxanylpyridine%252Bcore%252Bvia%252Btandem%252Bhetero-Diels%2525E2%252580%252593Alder%25252Fretro-Diels%2525E2%252580%252593Alder%252Breactions%252Bof%252B%2525281%25252C2%25252C4-triazin-3-yl%252529furoxans%26aulast%3DFershtat%26aufirst%3DLeonid%2BL.%26date%3D2016%26date%3D2016%26volume%3D6%26issue%3D37%26spage%3D31526%26epage%3D31539" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-1/acs.jmedchem.5b01361/20160113/images/medium/jm-2015-013613_0014.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-1/acs.jmedchem.5b01361/20160113/images/large/jm-2015-013613_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01361&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-1/acs.jmedchem.5b01361/20160113/images/medium/jm-2015-013613_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-1/acs.jmedchem.5b01361/20160113/images/large/jm-2015-013613_0005.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. (A) Mechanism of catalysis for LSD1. (B) Structures of representative LSD1 inhibitors, among which compounds in the upper panel are irreversible inhibitors and those in the lower panel are reversible.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-1/acs.jmedchem.5b01361/20160113/images/large/jm-2015-013613_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01361&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="cht1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Chart 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-1/acs.jmedchem.5b01361/20160113/images/medium/jm-2015-013613_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-1/acs.jmedchem.5b01361/20160113/images/large/jm-2015-013613_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Chart 1. Inhibitor Design Leading to the Finding of New LSD1 Inhibitors<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-1/acs.jmedchem.5b01361/20160113/images/large/jm-2015-013613_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01361&amp;id=cht1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Chart a</span>IC<sub>50</sub> values of compound <b>1</b> against MAO-A and -B are from ref <a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">35</a>.</p></p></figure><figure data-id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-1/acs.jmedchem.5b01361/20160113/images/medium/jm-2015-013613_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-1/acs.jmedchem.5b01361/20160113/images/large/jm-2015-013613_0006.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Docking studies of potential LSD1 inhibitors, using an LSD1 crystal structure (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2V1D">2V1D</a>) as the docking template. For clarity, only key residues (in light blue) in the active site of LSD1 are shown, together with the cofactor FAD (light blue tube model). Hydrogen bonds are shown as orange dashed lines. (A) Lwest-energy docking structure of compound <b>3</b> (tube model with C atoms in green), suggesting favorable interactions between its pyridine ring and FAD and Y761. (B) Lowest-energy docking structure of <b>4</b> (in yellow), showing introducing an additional 5-phenyl group could be favorable.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-1/acs.jmedchem.5b01361/20160113/images/large/jm-2015-013613_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01361&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-1/acs.jmedchem.5b01361/20160113/images/medium/jm-2015-013613_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-1/acs.jmedchem.5b01361/20160113/images/large/jm-2015-013613_0010.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. General Synthetic Method for Pyrazine Containing Inhibitors<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-1/acs.jmedchem.5b01361/20160113/images/large/jm-2015-013613_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01361&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (i) 2-aminoacetamide, NaOH; (ii) POCl<sub>3</sub>; (iii) BOC protected 4-(hydroxymethyl)piperidine, NaH; (iv) HCl in 1,4-dioxane; (v) Pd(PPh<sub>3</sub>)<sub>4</sub>, Zn(CN)<sub>2</sub>.</p></p></figure><figure data-id="sch2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-1/acs.jmedchem.5b01361/20160113/images/medium/jm-2015-013613_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-1/acs.jmedchem.5b01361/20160113/images/large/jm-2015-013613_0011.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. General Synthetic Method for Pyridine Containing Inhibitors<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-1/acs.jmedchem.5b01361/20160113/images/large/jm-2015-013613_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01361&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (i) BOC protected 4-(hydroxymethyl)piperidine, PPh<sub>3</sub>, diisopropyl azodicarboxylate; (ii) R<sup>5</sup>-B(OH)<sub>2</sub>, Pd(PPh<sub>3</sub>)<sub>4</sub>, Na<sub>2</sub>CO<sub>3</sub>, 80 °C; (iii) R<sup>6</sup>-B(OH)<sub>2</sub>, Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>3</sub>PO<sub>4</sub>, 140 °C; (iv) HCl in 1,4-dioxane.</p></p></figure><figure data-id="cht2" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Chart 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-1/acs.jmedchem.5b01361/20160113/images/medium/jm-2015-013613_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-1/acs.jmedchem.5b01361/20160113/images/large/jm-2015-013613_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Chart 2. Structures and <i>K</i><sub>i</sub> Values (in Parentheses) of Compounds <b>5</b>–<b>12</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-1/acs.jmedchem.5b01361/20160113/images/large/jm-2015-013613_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01361&amp;id=cht2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="cht3" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Chart 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-1/acs.jmedchem.5b01361/20160113/images/medium/jm-2015-013613_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-1/acs.jmedchem.5b01361/20160113/images/large/jm-2015-013613_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Chart 3. Structures and <i>K</i><sub>i</sub> Values (in Parentheses) of Compounds <b>41</b>–<b>45</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-1/acs.jmedchem.5b01361/20160113/images/large/jm-2015-013613_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01361&amp;id=cht3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-1/acs.jmedchem.5b01361/20160113/images/medium/jm-2015-013613_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-1/acs.jmedchem.5b01361/20160113/images/large/jm-2015-013613_0007.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Lineweaver–Burk plot of competitive inhibition model for compound <b>5</b> with variable concentrations of the dimethylated H3K4 peptide substrate, with an enlarged figure in the left panel showing details near the origin. The <i>K</i><sub>i</sub> was determined to be 1.8 μM in this experiment.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-1/acs.jmedchem.5b01361/20160113/images/large/jm-2015-013613_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01361&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-1/acs.jmedchem.5b01361/20160113/images/medium/jm-2015-013613_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-1/acs.jmedchem.5b01361/20160113/images/large/jm-2015-013613_0008.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Docking results of compound <b>17</b>, using an LSD1 structure (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2V1D">2V1D</a>) as the docking template: (A) 20 lowest-energy docking structures of compound <b>17</b>, with the most stable conformation shown as an orange tube model; (B) lowest-energy docking structure of <b>17</b>, showing the interacting key residues (in light cyan) of LSD1 and FAD (light cyan tube model); (C) lowest-energy docking structure of <b>17</b> (as a space-filing model) in the active site of LSD1 (shown as a meshed electrostatic potential surface model).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-1/acs.jmedchem.5b01361/20160113/images/large/jm-2015-013613_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01361&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-1/acs.jmedchem.5b01361/20160113/images/medium/jm-2015-013613_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-1/acs.jmedchem.5b01361/20160113/images/large/jm-2015-013613_0009.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Treatment of MV4-11 cells with potent LSD1 inhibitor <b>17</b> caused an increased cellular level of H3K4me2.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-1/acs.jmedchem.5b01361/20160113/images/large/jm-2015-013613_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01361&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i32">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_45547" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_45547" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 48 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><div class="note"><p class="first last">For a recent review, see the following:</p></div><span class="NLM_contrib-group">Kouzarides, T.</span><span> </span><span class="NLM_article-title">Chromatin modifications and their function</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">128</span><span class="NLM_x">, </span> <span class="NLM_fpage">693</span><span class="NLM_x">–</span> <span class="NLM_lpage">705</span><span class="refDoi"> DOI: 10.1016/j.cell.2007.02.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1016%2Fj.cell.2007.02.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=17320507" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BD2sXis12ju7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2007&pages=693-705&author=T.+Kouzarides&title=Chromatin+modifications+and+their+function&doi=10.1016%2Fj.cell.2007.02.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Chromatin modifications and their function</span></div><div class="casAuthors">Kouzarides, Tony</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">693-705</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The surface of nucleosomes is studded with a multiplicity of modifications.  At least eight different classes have been characterized to date and many different sites have been identified for each class.  Operationally, modifications function either by disrupting chromatin contacts or by affecting the recruitment of nonhistone proteins to chromatin.  Their presence on histones can dictate the higher-order chromatin structure in which DNA is packaged and can orchestrate the ordered recruitment of enzyme complexes to manipulate DNA.  In this way, histone modifications have the potential to influence many fundamental biol. processes, some of which may be epigenetically inherited.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAH9AX2dGicbVg90H21EOLACvtfcHk0lia6rchHhny8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXis12ju7Y%253D&md5=c5995fe68fd9473f4268d59c01d3b9de</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2007.02.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2007.02.005%26sid%3Dliteratum%253Aachs%26aulast%3DKouzarides%26aufirst%3DT.%26atitle%3DChromatin%2520modifications%2520and%2520their%2520function%26jtitle%3DCell%26date%3D2007%26volume%3D128%26spage%3D693%26epage%3D705%26doi%3D10.1016%2Fj.cell.2007.02.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Jones, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baylin, S. B.</span><span> </span><span class="NLM_article-title">The epigenomics of cancer</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">128</span><span class="NLM_x">, </span> <span class="NLM_fpage">683</span><span class="NLM_x">–</span> <span class="NLM_lpage">692</span><span class="refDoi"> DOI: 10.1016/j.cell.2007.01.029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1016%2Fj.cell.2007.01.029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=17320506" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BD2sXis12ju7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2007&pages=683-692&author=P.+A.+Jonesauthor=S.+B.+Baylin&title=The+epigenomics+of+cancer&doi=10.1016%2Fj.cell.2007.01.029"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">The epigenomics of cancer</span></div><div class="casAuthors">Jones, Peter A.; Baylin, Stephen B.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">683-692</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Aberrant gene function and altered patterns of gene expression are key features of cancer.  Growing evidence shows that acquired epigenetic abnormalities participate with genetic alterations to cause this dysregulation.  Here, we review recent advances in understanding how epigenetic alterations participate in the earliest stages of neoplasia, including stem/precursor cell contributions, and discuss the growing implications of these advances for strategies to control cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPBpHNAkcSPbVg90H21EOLACvtfcHk0lhBQhbX8dP3rw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXis12ju7k%253D&md5=78eac8ecc093f1aed0acbfd578f34717</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2007.01.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2007.01.029%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DP.%2BA.%26aulast%3DBaylin%26aufirst%3DS.%2BB.%26atitle%3DThe%2520epigenomics%2520of%2520cancer%26jtitle%3DCell%26date%3D2007%26volume%3D128%26spage%3D683%26epage%3D692%26doi%3D10.1016%2Fj.cell.2007.01.029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Wilson, C. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rowell, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sekimata, M.</span><span> </span><span class="NLM_article-title">Epigenetic control of T-helper-cell differentiation</span> <span class="citation_source-journal">Nat. Rev. Immunol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">91</span><span class="NLM_x">–</span> <span class="NLM_lpage">105</span><span class="refDoi"> DOI: 10.1038/nri2487</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1038%2Fnri2487" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=19151746" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BD1MXntFGnug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2009&pages=91-105&author=C.+B.+Wilsonauthor=E.+Rowellauthor=M.+Sekimata&title=Epigenetic+control+of+T-helper-cell+differentiation&doi=10.1038%2Fnri2487"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetic control of T-helper-cell differentiation</span></div><div class="casAuthors">Wilson, Christopher B.; Rowell, Emily; Sekimata, Masayuki</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Immunology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">91-105</span>CODEN:
                <span class="NLM_cas:coden">NRIABX</span>;
        ISSN:<span class="NLM_cas:issn">1474-1733</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Naive CD4+ T cells give rise to T-helper-cell subsets with functions that are tailored to their resp. roles in host defense.  The specification of T-helper-cell subsets is controlled by networks of lineage-specifying transcription factors, which bind to regulatory elements in genes that encode cytokines and other transcription factors.  The nuclear context in which these transcription factors act is affected by epigenetic processes, which allow programs of gene expression to be inherited by progeny cells that at the same time retain the potential for change in response to altered environmental signals.  In this Review, the authors describe these epigenetic processes and discuss how they collaborate to govern the fate and function of T helper cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZxqgRtS65ibVg90H21EOLACvtfcHk0lhBQhbX8dP3rw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXntFGnug%253D%253D&md5=14cb4ee035389a460489f7e530d0e41b</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnri2487&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnri2487%26sid%3Dliteratum%253Aachs%26aulast%3DWilson%26aufirst%3DC.%2BB.%26aulast%3DRowell%26aufirst%3DE.%26aulast%3DSekimata%26aufirst%3DM.%26atitle%3DEpigenetic%2520control%2520of%2520T-helper-cell%2520differentiation%26jtitle%3DNat.%2520Rev.%2520Immunol.%26date%3D2009%26volume%3D9%26spage%3D91%26epage%3D105%26doi%3D10.1038%2Fnri2487" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Tsankova, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Renthal, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kumar, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nestler, E. J.</span><span> </span><span class="NLM_article-title">Epigenetic regulation in psychiatric disorders</span> <span class="citation_source-journal">Nat. Rev. Neurosci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">355</span><span class="NLM_x">–</span> <span class="NLM_lpage">367</span><span class="refDoi"> DOI: 10.1038/nrn2132</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1038%2Fnrn2132" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=17453016" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BD2sXksFSis7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2007&pages=355-367&author=N.+Tsankovaauthor=W.+Renthalauthor=A.+Kumarauthor=E.+J.+Nestler&title=Epigenetic+regulation+in+psychiatric+disorders&doi=10.1038%2Fnrn2132"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetic regulation in psychiatric disorders</span></div><div class="casAuthors">Tsankova, Nadia; Renthal, William; Kumar, Arvind; Nestler, Eric J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Neuroscience</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">355-367</span>CODEN:
                <span class="NLM_cas:coden">NRNAAN</span>;
        ISSN:<span class="NLM_cas:issn">1471-003X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Many neurol. and most psychiatric disorders are not due to mutations in a single gene; rather, they involve mol. disturbances entailing multiple genes and signals that control their expression.  Recent research has demonstrated that complex 'epigenetic' mechanisms, which regulate gene activity without altering the DNA code, have long-lasting effects within mature neurons.  This review summarizes recent evidence for the existence of sustained epigenetic mechanisms of gene regulation in neurons that have been implicated in the regulation of complex behavior, including abnormalities in several psychiatric disorders such as depression, drug addiction and schizophrenia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQG0gqjbj_Z7Vg90H21EOLACvtfcHk0lhBQhbX8dP3rw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXksFSis7g%253D&md5=6af012ebb4b554beafcb4771e3e913ba</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnrn2132&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrn2132%26sid%3Dliteratum%253Aachs%26aulast%3DTsankova%26aufirst%3DN.%26aulast%3DRenthal%26aufirst%3DW.%26aulast%3DKumar%26aufirst%3DA.%26aulast%3DNestler%26aufirst%3DE.%2BJ.%26atitle%3DEpigenetic%2520regulation%2520in%2520psychiatric%2520disorders%26jtitle%3DNat.%2520Rev.%2520Neurosci.%26date%3D2007%26volume%3D8%26spage%3D355%26epage%3D367%26doi%3D10.1038%2Fnrn2132" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Solomon, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richon, V. M.</span><span> </span><span class="NLM_article-title">Protein methyltransferases as a target class for drug discovery</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">724</span><span class="NLM_x">–</span> <span class="NLM_lpage">732</span><span class="refDoi"> DOI: 10.1038/nrd2974</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1038%2Fnrd2974" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=19721445" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVKmsrvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2009&pages=724-732&author=R.+A.+Copelandauthor=M.+E.+Solomonauthor=V.+M.+Richon&title=Protein+methyltransferases+as+a+target+class+for+drug+discovery&doi=10.1038%2Fnrd2974"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Protein methyltransferases as a target class for drug discovery</span></div><div class="casAuthors">Copeland, Robert A.; Solomon, Michael E.; Richon, Victoria M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">724-732</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The protein methyltransferases (PMTs) - which methylate protein lysine and arginine residues and have crucial roles in gene transcription - are emerging as an important group of enzymes that play key parts in normal physiol. and human diseases.  The collection of human PMTs is a large and diverse group of enzymes that have a common mechanism of catalysis.  Here, we review the biol., biochem. and structural data that together present PMTs as a novel, chem. tractable target class for drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpySXIqKk9Lw7Vg90H21EOLACvtfcHk0ljoVLEG1xtFIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVKmsrvI&md5=0626e14396442b47c7da2aaa6a0ac6e3</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnrd2974&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2974%26sid%3Dliteratum%253Aachs%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DSolomon%26aufirst%3DM.%2BE.%26aulast%3DRichon%26aufirst%3DV.%2BM.%26atitle%3DProtein%2520methyltransferases%2520as%2520a%2520target%2520class%2520for%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2009%26volume%3D8%26spage%3D724%26epage%3D732%26doi%3D10.1038%2Fnrd2974" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Cole, P. A.</span><span> </span><span class="NLM_article-title">Chemical probes for histone-modifying enzymes</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">590</span><span class="NLM_x">–</span> <span class="NLM_lpage">597</span><span class="refDoi"> DOI: 10.1038/nchembio.111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1038%2Fnchembio.111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=18800048" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtFamtbrL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2008&pages=590-597&author=P.+A.+Cole&title=Chemical+probes+for+histone-modifying+enzymes&doi=10.1038%2Fnchembio.111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical probes for histone-modifying enzymes</span></div><div class="casAuthors">Cole, Philip A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">590-597</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The histone-modifying enzymes that catalyze reversible lysine acetylation and methylation are central to the epigenetic regulation of chromatin remodeling.  From the early discovery of histone deacetylase inhibitors to the more recent identification of histone demethylase blockers, chem. approaches offer increasingly sophisticated tools for the investigation of the structure and function of these lysine-modifying enzymes.  This review summarizes progress to date on compds. identified from screens or by design that can modulate the activity of classical histone deacetylases, sirtuins, histone acetyltransferases, histone methyltransferases and histone demethylases.  We highlight applications of compds. to mechanistic and functional studies involving these enzymes and discuss future challenges regarding target specificity and general utility.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJZ5RKIrQsO7Vg90H21EOLACvtfcHk0ljoVLEG1xtFIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtFamtbrL&md5=71fe323c232af42b67a7dfc134c4d49b</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.111%26sid%3Dliteratum%253Aachs%26aulast%3DCole%26aufirst%3DP.%2BA.%26atitle%3DChemical%2520probes%2520for%2520histone-modifying%2520enzymes%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2008%26volume%3D4%26spage%3D590%26epage%3D597%26doi%3D10.1038%2Fnchembio.111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Falkenberg, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnstone, R. W.</span><span> </span><span class="NLM_article-title">Histone deacetylases and their inhibitors in cancer, neurological diseases and immune disorders</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">673</span><span class="NLM_x">–</span> <span class="NLM_lpage">691</span><span class="refDoi"> DOI: 10.1038/nrd4360</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1038%2Fnrd4360" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=25131830" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlKrtLjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=673-691&author=K.+J.+Falkenbergauthor=R.+W.+Johnstone&title=Histone+deacetylases+and+their+inhibitors+in+cancer%2C+neurological+diseases+and+immune+disorders&doi=10.1038%2Fnrd4360"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Histone deacetylases and their inhibitors in cancer, neurological diseases and immune disorders</span></div><div class="casAuthors">Falkenberg, Katrina J.; Johnstone, Ricky W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">673-691</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Epigenetic aberrations, which are recognized as key drivers of several human diseases, are often caused by genetic defects that result in functional deregulation of epigenetic proteins, their altered expression and/or their atypical recruitment to certain gene promoters.  Importantly, epigenetic changes are reversible, and epigenetic enzymes and regulatory proteins can be targeted using small mols.  This Review discusses the role of altered expression and/or function of one class of epigenetic regulators - histone deacetylases (HDACs) - and their role in cancer, neurol. diseases and immune disorders.  We highlight the development of small-mol. HDAC inhibitors and their use in the lab., in preclin. models and in the clinic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXiWEG49LhebVg90H21EOLACvtfcHk0ljj2CDYvSPTww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlKrtLjI&md5=ccfb68fa3fd4685e2305cdf8a6a9c81a</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnrd4360&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4360%26sid%3Dliteratum%253Aachs%26aulast%3DFalkenberg%26aufirst%3DK.%2BJ.%26aulast%3DJohnstone%26aufirst%3DR.%2BW.%26atitle%3DHistone%2520deacetylases%2520and%2520their%2520inhibitors%2520in%2520cancer%252C%2520neurological%2520diseases%2520and%2520immune%2520disorders%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D673%26epage%3D691%26doi%3D10.1038%2Fnrd4360" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Milne, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Briggs, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brock, H. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gibbs, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allis, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hess, J. L.</span><span> </span><span class="NLM_article-title">MLL targets SET domain methyltransferase activity to Hox gene promoters</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">1107</span><span class="NLM_x">–</span> <span class="NLM_lpage">1117</span><span class="refDoi"> DOI: 10.1016/S1097-2765(02)00741-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1016%2FS1097-2765%2802%2900741-4" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2002&pages=1107-1117&author=T.+A.+Milneauthor=S.+D.+Briggsauthor=H.+W.+Brockauthor=M.+E.+Martinauthor=D.+Gibbsauthor=C.+D.+Allisauthor=J.+L.+Hess&title=MLL+targets+SET+domain+methyltransferase+activity+to+Hox+gene+promoters&doi=10.1016%2FS1097-2765%2802%2900741-4"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2FS1097-2765%2802%2900741-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1097-2765%252802%252900741-4%26sid%3Dliteratum%253Aachs%26aulast%3DMilne%26aufirst%3DT.%2BA.%26aulast%3DBriggs%26aufirst%3DS.%2BD.%26aulast%3DBrock%26aufirst%3DH.%2BW.%26aulast%3DMartin%26aufirst%3DM.%2BE.%26aulast%3DGibbs%26aufirst%3DD.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26aulast%3DHess%26aufirst%3DJ.%2BL.%26atitle%3DMLL%2520targets%2520SET%2520domain%2520methyltransferase%2520activity%2520to%2520Hox%2520gene%2520promoters%26jtitle%3DMol.%2520Cell%26date%3D2002%26volume%3D10%26spage%3D1107%26epage%3D1117%26doi%3D10.1016%2FS1097-2765%2802%2900741-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Briggs, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bryk, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Strahl, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheung, W. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davie, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dent, S. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winston, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allis, C. D.</span><span> </span><span class="NLM_article-title">Histone H3 lysine 4 methylation is mediated by Set1 and required for cell growth and rDNA silencing in Saccharomyces cerevisiae</span> <span class="citation_source-journal">Genes Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">3286</span><span class="NLM_x">–</span> <span class="NLM_lpage">3295</span><span class="refDoi"> DOI: 10.1101/gad.940201</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1101%2Fgad.940201" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2001&pages=3286-3295&author=S.+D.+Briggsauthor=M.+Brykauthor=B.+D.+Strahlauthor=W.+L.+Cheungauthor=J.+K.+Davieauthor=S.+Y.+Dentauthor=F.+Winstonauthor=C.+D.+Allis&title=Histone+H3+lysine+4+methylation+is+mediated+by+Set1+and+required+for+cell+growth+and+rDNA+silencing+in+Saccharomyces+cerevisiae&doi=10.1101%2Fgad.940201"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1101%2Fgad.940201&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.940201%26sid%3Dliteratum%253Aachs%26aulast%3DBriggs%26aufirst%3DS.%2BD.%26aulast%3DBryk%26aufirst%3DM.%26aulast%3DStrahl%26aufirst%3DB.%2BD.%26aulast%3DCheung%26aufirst%3DW.%2BL.%26aulast%3DDavie%26aufirst%3DJ.%2BK.%26aulast%3DDent%26aufirst%3DS.%2BY.%26aulast%3DWinston%26aufirst%3DF.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26atitle%3DHistone%2520H3%2520lysine%25204%2520methylation%2520is%2520mediated%2520by%2520Set1%2520and%2520required%2520for%2520cell%2520growth%2520and%2520rDNA%2520silencing%2520in%2520Saccharomyces%2520cerevisiae%26jtitle%3DGenes%2520Dev.%26date%3D2001%26volume%3D15%26spage%3D3286%26epage%3D3295%26doi%3D10.1101%2Fgad.940201" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Milne, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Briggs, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brock, H. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gibbs, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allis, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hess, J. L.</span><span> </span><span class="NLM_article-title">MLL targets SET domain methyltransferase activity to Hox gene promoters</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">1107</span><span class="NLM_x">–</span> <span class="NLM_lpage">1117</span><span class="refDoi"> DOI: 10.1016/S1097-2765(02)00741-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1016%2FS1097-2765%2802%2900741-4" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2002&pages=1107-1117&author=T.+A.+Milneauthor=S.+D.+Briggsauthor=H.+W.+Brockauthor=M.+E.+Martinauthor=D.+Gibbsauthor=C.+D.+Allisauthor=J.+L.+Hess&title=MLL+targets+SET+domain+methyltransferase+activity+to+Hox+gene+promoters&doi=10.1016%2FS1097-2765%2802%2900741-4"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2FS1097-2765%2802%2900741-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1097-2765%252802%252900741-4%26sid%3Dliteratum%253Aachs%26aulast%3DMilne%26aufirst%3DT.%2BA.%26aulast%3DBriggs%26aufirst%3DS.%2BD.%26aulast%3DBrock%26aufirst%3DH.%2BW.%26aulast%3DMartin%26aufirst%3DM.%2BE.%26aulast%3DGibbs%26aufirst%3DD.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26aulast%3DHess%26aufirst%3DJ.%2BL.%26atitle%3DMLL%2520targets%2520SET%2520domain%2520methyltransferase%2520activity%2520to%2520Hox%2520gene%2520promoters%26jtitle%3DMol.%2520Cell%26date%3D2002%26volume%3D10%26spage%3D1107%26epage%3D1117%26doi%3D10.1016%2FS1097-2765%2802%2900741-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Krivtsov, A. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Armstrong, S. A.</span><span> </span><span class="NLM_article-title">MLL translocations, histone modifications and leukaemia stem-cell development</span> <span class="citation_source-journal">Nat. Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">823</span><span class="NLM_x">–</span> <span class="NLM_lpage">833</span><span class="refDoi"> DOI: 10.1038/nrc2253</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1038%2Fnrc2253" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=17957188" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BD2sXht1Wgsb%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2007&pages=823-833&author=A.+V.+Krivtsovauthor=S.+A.+Armstrong&title=MLL+translocations%2C+histone+modifications+and+leukaemia+stem-cell+development&doi=10.1038%2Fnrc2253"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">MLL translocations, histone modifications and leukaemia stem-cell development</span></div><div class="casAuthors">Krivtsov, Andrei V.; Armstrong, Scott A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">823-833</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Mixed lineage leukemia (MLL) has histone methyltransferase activity and regulates the expression of genes such as Hox genes.  This activity is lost in MLL fusion proteins resulting from inter-chromosomal translocations, which are leukemogenic.  How do MLL fusions function and what is their role in leukemia stem cells.  Translocations that involve the mixed lineage leukemia (MLL) gene identify a unique group of acute leukemias, and often predict a poor prognosis.  The MLL gene encodes a DNA-binding protein that methylates histone H3 lysine 4 (H3K4), and pos. regulates gene expression including multiple Hox genes.  Leukaemogenic MLL translocations encode MLL fusion proteins that have lost H3K4 methyltransferase activity.  A key feature of MLL fusion proteins is their ability to efficiently transform haematopoietic cells into leukemia stem cells.  The link between a chromatin modulator and leukemia stem cells provides support for epigenetic landscapes as an important part of leukemia and normal stem-cell development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqm9NXrcNtrd7Vg90H21EOLACvtfcHk0ljj2CDYvSPTww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXht1Wgsb%252FJ&md5=56efc9bbbb9f24fb4233e40ed86628fd</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fnrc2253&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2253%26sid%3Dliteratum%253Aachs%26aulast%3DKrivtsov%26aufirst%3DA.%2BV.%26aulast%3DArmstrong%26aufirst%3DS.%2BA.%26atitle%3DMLL%2520translocations%252C%2520histone%2520modifications%2520and%2520leukaemia%2520stem-cell%2520development%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2007%26volume%3D7%26spage%3D823%26epage%3D833%26doi%3D10.1038%2Fnrc2253" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Nakamura, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mori, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tada, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krajewski, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rozovskaia, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wassell, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dubois, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mazo, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Croce, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Canaani, E.</span><span> </span><span class="NLM_article-title">ALL-1 is a histone methyltransferase that assembles a supercomplex of proteins involved in transcriptional regulation</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">1119</span><span class="NLM_x">–</span> <span class="NLM_lpage">1128</span><span class="refDoi"> DOI: 10.1016/S1097-2765(02)00740-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1016%2FS1097-2765%2802%2900740-2" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2002&pages=1119-1128&author=T.+Nakamuraauthor=T.+Moriauthor=S.+Tadaauthor=W.+Krajewskiauthor=T.+Rozovskaiaauthor=R.+Wassellauthor=G.+Duboisauthor=A.+Mazoauthor=C.+M.+Croceauthor=E.+Canaani&title=ALL-1+is+a+histone+methyltransferase+that+assembles+a+supercomplex+of+proteins+involved+in+transcriptional+regulation&doi=10.1016%2FS1097-2765%2802%2900740-2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2FS1097-2765%2802%2900740-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1097-2765%252802%252900740-2%26sid%3Dliteratum%253Aachs%26aulast%3DNakamura%26aufirst%3DT.%26aulast%3DMori%26aufirst%3DT.%26aulast%3DTada%26aufirst%3DS.%26aulast%3DKrajewski%26aufirst%3DW.%26aulast%3DRozovskaia%26aufirst%3DT.%26aulast%3DWassell%26aufirst%3DR.%26aulast%3DDubois%26aufirst%3DG.%26aulast%3DMazo%26aufirst%3DA.%26aulast%3DCroce%26aufirst%3DC.%2BM.%26aulast%3DCanaani%26aufirst%3DE.%26atitle%3DALL-1%2520is%2520a%2520histone%2520methyltransferase%2520that%2520assembles%2520a%2520supercomplex%2520of%2520proteins%2520involved%2520in%2520transcriptional%2520regulation%26jtitle%3DMol.%2520Cell%26date%3D2002%26volume%3D10%26spage%3D1119%26epage%3D1128%26doi%3D10.1016%2FS1097-2765%2802%2900740-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Shi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lan, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mulligan, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whetstine, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cole, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casero, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Y.</span><span> </span><span class="NLM_article-title">Histone demethylation mediated by the nuclear amine oxidase homolog LSD1</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">119</span><span class="NLM_x">, </span> <span class="NLM_fpage">941</span><span class="NLM_x">–</span> <span class="NLM_lpage">953</span><span class="refDoi"> DOI: 10.1016/j.cell.2004.12.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1016%2Fj.cell.2004.12.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=15620353" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BD2MXltlSitg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2004&pages=941-953&author=Y.+Shiauthor=F.+Lanauthor=C.+Matsonauthor=P.+Mulliganauthor=J.+R.+Whetstineauthor=P.+A.+Coleauthor=R.+A.+Caseroauthor=Y.+Shi&title=Histone+demethylation+mediated+by+the+nuclear+amine+oxidase+homolog+LSD1&doi=10.1016%2Fj.cell.2004.12.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Histone demethylation mediated by the nuclear amine oxidase homolog LSD1</span></div><div class="casAuthors">Shi, Yujiang; Lan, Fei; Matson, Caitlin; Mulligan, Peter; Whetstine, Johnathan R.; Cole, Philip A.; Casero, Robert A.; Shi, Yang</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">941-953</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Posttranslational modifications of histone N-terminal tails impact chromatin structure and gene transcription.  While the extent of histone acetylation is detd. by both acetyltransferases and deacetylases, it has been unclear whether histone methylation is also regulated by enzymes with opposing activities.  Here, we provide evidence that LSD1 (KIAA0601), a nuclear homolog of amine oxidases, functions as a histone demethylase and transcriptional corepressor.  LSD1 specifically demethylates histone H3 lysine 4, which is linked to active transcription.  Lysine demethylation occurs via an oxidn. reaction that generates formaldehyde.  Importantly, RNAi inhibition of LSD1 causes an increase in H3 lysine 4 methylation and concomitant derepression of target genes, suggesting that LSD1 represses transcription via histone demethylation.  The results thus identify a histone demethylase conserved from S. pombe to human and reveal dynamic regulation of histone methylation by both histone methylases and demethylases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXHr-8kwGdXrVg90H21EOLACvtfcHk0liWGepu8nUdsw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXltlSitg%253D%253D&md5=3474569204255e90a832667ba654748b</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2004.12.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2004.12.012%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DY.%26aulast%3DLan%26aufirst%3DF.%26aulast%3DMatson%26aufirst%3DC.%26aulast%3DMulligan%26aufirst%3DP.%26aulast%3DWhetstine%26aufirst%3DJ.%2BR.%26aulast%3DCole%26aufirst%3DP.%2BA.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26aulast%3DShi%26aufirst%3DY.%26atitle%3DHistone%2520demethylation%2520mediated%2520by%2520the%2520nuclear%2520amine%2520oxidase%2520homolog%2520LSD1%26jtitle%3DCell%26date%3D2004%26volume%3D119%26spage%3D941%26epage%3D953%26doi%3D10.1016%2Fj.cell.2004.12.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Kerenyi, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shao, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hsu, Y. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luc, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Brien, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fujiwara, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peng, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Orkin, S. H.</span><span> </span><span class="NLM_article-title">Histone demethylase Lsd1 represses hematopoietic stem and progenitor cell signatures during blood cell maturation</span> <span class="citation_source-journal">eLife</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">e00633</span><span class="refDoi"> DOI: 10.7554/eLife.00633</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.7554%2FeLife.00633" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=23795291" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjtlWktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2013&pages=e00633&author=M.+A.+Kerenyiauthor=Z.+Shaoauthor=Y.+J.+Hsuauthor=G.+Guoauthor=S.+Lucauthor=K.+O%E2%80%99Brienauthor=Y.+Fujiwaraauthor=C.+Pengauthor=M.+Nguyenauthor=S.+H.+Orkin&title=Histone+demethylase+Lsd1+represses+hematopoietic+stem+and+progenitor+cell+signatures+during+blood+cell+maturation&doi=10.7554%2FeLife.00633"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Histone demethylase Lsd1 represses hematopoietic stem and progenitor cell signatures during blood cell maturation</span></div><div class="casAuthors">Kerenyi, Marc A.; Shao, Zhen; Hsu, Yu-Jung; Guo, Guoji; Luc, Sidinh; O'Brien, Kassandra; Fujiwara, Yuko; Peng, Cong; Nguyen, Minh; Orkin, Stuart H.</div><div class="citationInfo"><span class="NLM_cas:title">eLife</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">e00633/1-e00633/23</span>CODEN:
                <span class="NLM_cas:coden">ELIFA8</span>;
        ISSN:<span class="NLM_cas:issn">2050-084X</span>.
    
            (<span class="NLM_cas:orgname">eLife Sciences Publications Ltd.</span>)
        </div><div class="casAbstract">Here, we describe that lysine-specific demethylase 1 (Lsd1/KDM1a), which demethylates histone H3 on Lys4 or Lys9 (H3K4/K9), is an indispensable epigenetic governor of hematopoietic differentiation.  Integrative genomic anal., combining global occupancy of Lsd1, genome-wide anal. of its substrates H3K4 monomethylation and dimethylation, and gene expression profiling, reveals that Lsd1 represses hematopoietic stem and progenitor cell (HSPC) gene expression programs during hematopoietic differentiation.  We found that Lsd1 acts at transcription start sites, as well as enhancer regions.  Loss of Lsd1 was assocd. with increased H3K4me1 and H3K4me2 methylation on HSPC genes and gene derepression.  Failure to fully silence HSPC genes compromised differentiation of hematopoietic stem cells as well as mature blood cell lineages.  Collectively, our data indicate that Lsd1-mediated concurrent repression of enhancer and promoter activity of stem and progenitor cell genes is a pivotal epigenetic mechanism required for proper hematopoietic maturation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrx7esSx55Ug7Vg90H21EOLACvtfcHk0liWGepu8nUdsw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjtlWktg%253D%253D&md5=2b810bfcc09f686619523fa7f3ebde06</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.7554%2FeLife.00633&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.00633%26sid%3Dliteratum%253Aachs%26aulast%3DKerenyi%26aufirst%3DM.%2BA.%26aulast%3DShao%26aufirst%3DZ.%26aulast%3DHsu%26aufirst%3DY.%2BJ.%26aulast%3DGuo%26aufirst%3DG.%26aulast%3DLuc%26aufirst%3DS.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DK.%26aulast%3DFujiwara%26aufirst%3DY.%26aulast%3DPeng%26aufirst%3DC.%26aulast%3DNguyen%26aufirst%3DM.%26aulast%3DOrkin%26aufirst%3DS.%2BH.%26atitle%3DHistone%2520demethylase%2520Lsd1%2520represses%2520hematopoietic%2520stem%2520and%2520progenitor%2520cell%2520signatures%2520during%2520blood%2520cell%2520maturation%26jtitle%3DeLife%26date%3D2013%26volume%3D2%26spage%3De00633%26doi%3D10.7554%2FeLife.00633" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Metzger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wissmann, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yin, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Müller, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneider, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peters, A. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Günther, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buettner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schüle, R.</span><span> </span><span class="NLM_article-title">LSD1 demethylates repressive histone marks to promote androgen-receptor-dependent transcription</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">437</span><span class="NLM_x">, </span> <span class="NLM_fpage">436</span><span class="NLM_x">–</span> <span class="NLM_lpage">439</span><span class="refDoi"> DOI: 10.1038/nature04020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1038%2Fnature04020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=16079795" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BD2MXpvFOrtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=437&publication_year=2005&pages=436-439&author=E.+Metzgerauthor=M.+Wissmannauthor=N.+Yinauthor=J.+M.+M%C3%BCllerauthor=R.+Schneiderauthor=A.+H.+Petersauthor=T.+G%C3%BCntherauthor=R.+Buettnerauthor=R.+Sch%C3%BCle&title=LSD1+demethylates+repressive+histone+marks+to+promote+androgen-receptor-dependent+transcription&doi=10.1038%2Fnature04020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">LSD1 demethylates repressive histone marks to promote androgen-receptor-dependent transcription</span></div><div class="casAuthors">Metzger, Eric; Wissmann, Melanie; Yin, Na; Mueller, Judith M.; Schneider, Robert; Peters, Antoine H. F. M.; Guenther, Thomas; Buettner, Reinhard; Schuele, Roland</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">437</span>
        (<span class="NLM_cas:issue">7057</span>),
    <span class="NLM_cas:pages">436-439</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Gene regulation in eukaryotes requires the coordinate interaction of chromatin-modulating proteins with specific transcription factors such as the androgen receptor.  Gene activation and repression is specifically regulated by histone methylation status at distinct lysine residues.  Here the authors show that lysine-specific demethylase 1 (LSD1; also known as BHC110) co-localizes with the androgen receptor in normal human prostate and prostate tumor.  LSD1 interacts with androgen receptor in vitro and in vivo, and stimulates androgen-receptor-dependent transcription.  Conversely, knockdown of LSD1 protein levels abrogates androgen-induced transcriptional activation and cell proliferation.  Chromatin immunopptn. analyses demonstrate that androgen receptor and LSD1 form chromatin-assocd. complexes in a ligand-dependent manner.  LSD1 relieves repressive histone marks by demethylation of histone H3 at lysine 9 (H3-K9), thereby leading to de-repression of androgen receptor target genes.  Furthermore, the authors identify pargyline as an inhibitor of LSD1.  Pargyline blocks demethylation of H3-K9 by LSD1 and consequently androgen-receptor-dependent transcription.  Thus, modulation of LSD1 activity offers a new strategy to regulate androgen receptor functions.  Here, the authors link demethylation of a repressive histone mark with androgen-receptor-dependent gene activation, thus providing a mechanism by which demethylases control specific gene expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgFgaYIdjiFbVg90H21EOLACvtfcHk0liWGepu8nUdsw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXpvFOrtL4%253D&md5=6cac1e6f6c21a80f59de27796cc9ee22</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fnature04020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature04020%26sid%3Dliteratum%253Aachs%26aulast%3DMetzger%26aufirst%3DE.%26aulast%3DWissmann%26aufirst%3DM.%26aulast%3DYin%26aufirst%3DN.%26aulast%3DM%25C3%25BCller%26aufirst%3DJ.%2BM.%26aulast%3DSchneider%26aufirst%3DR.%26aulast%3DPeters%26aufirst%3DA.%2BH.%26aulast%3DG%25C3%25BCnther%26aufirst%3DT.%26aulast%3DBuettner%26aufirst%3DR.%26aulast%3DSch%25C3%25BCle%26aufirst%3DR.%26atitle%3DLSD1%2520demethylates%2520repressive%2520histone%2520marks%2520to%2520promote%2520androgen-receptor-dependent%2520transcription%26jtitle%3DNature%26date%3D2005%26volume%3D437%26spage%3D436%26epage%3D439%26doi%3D10.1038%2Fnature04020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hevi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurash, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lei, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gay, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajko, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Su, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gaudet, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, T.</span><span> </span><span class="NLM_article-title">The lysine demethylase LSD1 (KDM1) is required for maintenance of global DNA methylation</span> <span class="citation_source-journal">Nat. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">125</span><span class="NLM_x">–</span> <span class="NLM_lpage">129</span><span class="refDoi"> DOI: 10.1038/ng.268</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1038%2Fng.268" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=19098913" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsFajtbbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2009&pages=125-129&author=J.+Wangauthor=S.+Heviauthor=J.+K.+Kurashauthor=H.+Leiauthor=F.+Gayauthor=J.+Bajkoauthor=H.+Suauthor=W.+Sunauthor=H.+Changauthor=G.+Xuauthor=F.+Gaudetauthor=E.+Liauthor=T.+Chen&title=The+lysine+demethylase+LSD1+%28KDM1%29+is+required+for+maintenance+of+global+DNA+methylation&doi=10.1038%2Fng.268"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">The lysine demethylase LSD1 (KDM1) is required for maintenance of global DNA methylation</span></div><div class="casAuthors">Wang, Jing; Hevi, Sarah; Kurash, Julia K.; Lei, Hong; Gay, Frederique; Bajko, Jeffrey; Su, Hui; Sun, Weitao; Chang, Hua; Xu, Guoliang; Gaudet, Francois; Li, En; Chen, Taiping</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">125-129</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Histone methylation and DNA methylation cooperatively regulate chromatin structure and gene activity.  How these two systems coordinate with each other remains unclear.  Here we study the biol. function of lysine-specific demethylase 1 (LSD1, also known as KDM1 and AOF2), which has been shown to demethylate histone H3 on lysine 4 (H3K4) and lysine 9 (H3K9).  We show that LSD1 is required for gastrulation during mouse embryogenesis.  Notably, targeted deletion of the gene encoding LSD1 (namely, Aof2) in embryonic stem (ES) cells induces progressive loss of DNA methylation.  This loss correlates with a decrease in DNA methyltransferase 1 (Dnmt1) protein, as a result of reduced Dnmt1 stability.  Dnmt1 protein is methylated in vivo, and its methylation is enhanced in the absence of LSD1.  Furthermore, Dnmt1 can be methylated by Set7/9 (also known as KMT7) and demethylated by LSD1 in vitro.  Our findings suggest that LSD1 demethylates and stabilizes Dnmt1, thus providing a previously unknown mechanistic link between the histone and DNA methylation systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSJgebl5_48LVg90H21EOLACvtfcHk0lg0kD-4YTNh_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsFajtbbK&md5=37722ddd4564a79a0cd1c7b148c370d8</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fng.268&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fng.268%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DHevi%26aufirst%3DS.%26aulast%3DKurash%26aufirst%3DJ.%2BK.%26aulast%3DLei%26aufirst%3DH.%26aulast%3DGay%26aufirst%3DF.%26aulast%3DBajko%26aufirst%3DJ.%26aulast%3DSu%26aufirst%3DH.%26aulast%3DSun%26aufirst%3DW.%26aulast%3DChang%26aufirst%3DH.%26aulast%3DXu%26aufirst%3DG.%26aulast%3DGaudet%26aufirst%3DF.%26aulast%3DLi%26aufirst%3DE.%26aulast%3DChen%26aufirst%3DT.%26atitle%3DThe%2520lysine%2520demethylase%2520LSD1%2520%2528KDM1%2529%2520is%2520required%2520for%2520maintenance%2520of%2520global%2520DNA%2520methylation%26jtitle%3DNat.%2520Genet.%26date%3D2009%26volume%3D41%26spage%3D125%26epage%3D129%26doi%3D10.1038%2Fng.268" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Chen, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sorensen, P. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Domer, P. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reaman, G. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korsmeyer, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heerema, N. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hammond, G. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kersey, J. H.</span><span> </span><span class="NLM_article-title">Molecular rearrangements on chromosome 11q23 predominate in infant acute lymphoblastic leukemia and are associated with specific biologic variables and poor outcome</span> <span class="citation_source-journal">Blood</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">81</span><span class="NLM_x">, </span> <span class="NLM_fpage">2386</span><span class="NLM_x">–</span> <span class="NLM_lpage">2393</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=1993&pages=2386-2393&author=C.+S.+Chenauthor=P.+H.+Sorensenauthor=P.+H.+Domerauthor=G.+H.+Reamanauthor=S.+J.+Korsmeyerauthor=N.+A.+Heeremaauthor=G.+D.+Hammondauthor=J.+H.+Kersey&title=Molecular+rearrangements+on+chromosome+11q23+predominate+in+infant+acute+lymphoblastic+leukemia+and+are+associated+with+specific+biologic+variables+and+poor+outcome"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DC.%2BS.%26aulast%3DSorensen%26aufirst%3DP.%2BH.%26aulast%3DDomer%26aufirst%3DP.%2BH.%26aulast%3DReaman%26aufirst%3DG.%2BH.%26aulast%3DKorsmeyer%26aufirst%3DS.%2BJ.%26aulast%3DHeerema%26aufirst%3DN.%2BA.%26aulast%3DHammond%26aufirst%3DG.%2BD.%26aulast%3DKersey%26aufirst%3DJ.%2BH.%26atitle%3DMolecular%2520rearrangements%2520on%2520chromosome%252011q23%2520predominate%2520in%2520infant%2520acute%2520lymphoblastic%2520leukemia%2520and%2520are%2520associated%2520with%2520specific%2520biologic%2520variables%2520and%2520poor%2520outcome%26jtitle%3DBlood%26date%3D1993%26volume%3D81%26spage%3D2386%26epage%3D2393" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Mrozek, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heinonen, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lawrence, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carroll, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koduru, P. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rao, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Strout, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hutchison, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moore, J. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mayer, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schiffer, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bloomfield, C. D.</span><span> </span><span class="NLM_article-title">Adult patients with de novo acute myeloid leukemia and t(9; 11)(p22; q23) have a superior outcome to patients with other translocations involving band 11q23: a cancer and leukemia group B study</span> <span class="citation_source-journal">Blood</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">90</span><span class="NLM_x">, </span> <span class="NLM_fpage">4532</span><span class="NLM_x">–</span> <span class="NLM_lpage">4538</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=90&publication_year=1997&pages=4532-4538&author=K.+Mrozekauthor=K.+Heinonenauthor=D.+Lawrenceauthor=A.+J.+Carrollauthor=P.+R.+Koduruauthor=K.+W.+Raoauthor=M.+P.+Stroutauthor=R.+E.+Hutchisonauthor=J.+O.+Mooreauthor=R.+J.+Mayerauthor=C.+A.+Schifferauthor=C.+D.+Bloomfield&title=Adult+patients+with+de+novo+acute+myeloid+leukemia+and+t%289%3B+11%29%28p22%3B+q23%29+have+a+superior+outcome+to+patients+with+other+translocations+involving+band+11q23%3A+a+cancer+and+leukemia+group+B+study"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMrozek%26aufirst%3DK.%26aulast%3DHeinonen%26aufirst%3DK.%26aulast%3DLawrence%26aufirst%3DD.%26aulast%3DCarroll%26aufirst%3DA.%2BJ.%26aulast%3DKoduru%26aufirst%3DP.%2BR.%26aulast%3DRao%26aufirst%3DK.%2BW.%26aulast%3DStrout%26aufirst%3DM.%2BP.%26aulast%3DHutchison%26aufirst%3DR.%2BE.%26aulast%3DMoore%26aufirst%3DJ.%2BO.%26aulast%3DMayer%26aufirst%3DR.%2BJ.%26aulast%3DSchiffer%26aufirst%3DC.%2BA.%26aulast%3DBloomfield%26aufirst%3DC.%2BD.%26atitle%3DAdult%2520patients%2520with%2520de%2520novo%2520acute%2520myeloid%2520leukemia%2520and%2520t%25289%253B%252011%2529%2528p22%253B%2520q23%2529%2520have%2520a%2520superior%2520outcome%2520to%2520patients%2520with%2520other%2520translocations%2520involving%2520band%252011q23%253A%2520a%2520cancer%2520and%2520leukemia%2520group%2520B%2520study%26jtitle%3DBlood%26date%3D1997%26volume%3D90%26spage%3D4532%26epage%3D4538" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Hilden, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dinndorf, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meerbaum, S. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sather, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Villaluna, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heerema, N. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McGlennen, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, F. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woods, W. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salzer, W. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnstone, H. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dreyer, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reaman, G. H.</span><span> </span><span class="NLM_article-title">Analysis of prognostic factors of acute lymphoblastic leukemia in infants: report on CCG 1953 from the Children’s Oncology Group</span> <span class="citation_source-journal">Blood</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">108</span><span class="NLM_x">, </span> <span class="NLM_fpage">441</span><span class="NLM_x">–</span> <span class="NLM_lpage">451</span><span class="refDoi"> DOI: 10.1182/blood-2005-07-3011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1182%2Fblood-2005-07-3011" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2006&pages=441-451&author=J.+M.+Hildenauthor=P.+A.+Dinndorfauthor=S.+O.+Meerbaumauthor=H.+Satherauthor=D.+Villalunaauthor=N.+A.+Heeremaauthor=R.+McGlennenauthor=F.+O.+Smithauthor=W.+G.+Woodsauthor=W.+L.+Salzerauthor=H.+S.+Johnstoneauthor=Z.+Dreyerauthor=G.+H.+Reaman&title=Analysis+of+prognostic+factors+of+acute+lymphoblastic+leukemia+in+infants%3A+report+on+CCG+1953+from+the+Children%E2%80%99s+Oncology+Group&doi=10.1182%2Fblood-2005-07-3011"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1182%2Fblood-2005-07-3011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2005-07-3011%26sid%3Dliteratum%253Aachs%26aulast%3DHilden%26aufirst%3DJ.%2BM.%26aulast%3DDinndorf%26aufirst%3DP.%2BA.%26aulast%3DMeerbaum%26aufirst%3DS.%2BO.%26aulast%3DSather%26aufirst%3DH.%26aulast%3DVillaluna%26aufirst%3DD.%26aulast%3DHeerema%26aufirst%3DN.%2BA.%26aulast%3DMcGlennen%26aufirst%3DR.%26aulast%3DSmith%26aufirst%3DF.%2BO.%26aulast%3DWoods%26aufirst%3DW.%2BG.%26aulast%3DSalzer%26aufirst%3DW.%2BL.%26aulast%3DJohnstone%26aufirst%3DH.%2BS.%26aulast%3DDreyer%26aufirst%3DZ.%26aulast%3DReaman%26aufirst%3DG.%2BH.%26atitle%3DAnalysis%2520of%2520prognostic%2520factors%2520of%2520acute%2520lymphoblastic%2520leukemia%2520in%2520infants%253A%2520report%2520on%2520CCG%25201953%2520from%2520the%2520Children%25E2%2580%2599s%2520Oncology%2520Group%26jtitle%3DBlood%26date%3D2006%26volume%3D108%26spage%3D441%26epage%3D451%26doi%3D10.1182%2Fblood-2005-07-3011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Okada, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coffield, V. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Su, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span> </span><span class="NLM_article-title">hDOT1L links histone methylation to leukemogenesis</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">121</span><span class="NLM_x">, </span> <span class="NLM_fpage">167</span><span class="NLM_x">–</span> <span class="NLM_lpage">178</span><span class="refDoi"> DOI: 10.1016/j.cell.2005.02.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1016%2Fj.cell.2005.02.020" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=121&publication_year=2005&pages=167-178&author=Y.+Okadaauthor=Q.+Fengauthor=Y.+Linauthor=Q.+Jiangauthor=Y.+Liauthor=V.+M.+Coffieldauthor=L.+Suauthor=G.+Xuauthor=Y.+Zhang&title=hDOT1L+links+histone+methylation+to+leukemogenesis&doi=10.1016%2Fj.cell.2005.02.020"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2005.02.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2005.02.020%26sid%3Dliteratum%253Aachs%26aulast%3DOkada%26aufirst%3DY.%26aulast%3DFeng%26aufirst%3DQ.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DJiang%26aufirst%3DQ.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DCoffield%26aufirst%3DV.%2BM.%26aulast%3DSu%26aufirst%3DL.%26aulast%3DXu%26aufirst%3DG.%26aulast%3DZhang%26aufirst%3DY.%26atitle%3DhDOT1L%2520links%2520histone%2520methylation%2520to%2520leukemogenesis%26jtitle%3DCell%26date%3D2005%26volume%3D121%26spage%3D167%26epage%3D178%26doi%3D10.1016%2Fj.cell.2005.02.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Bitoun, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oliver, P. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davies, K. E.</span><span> </span><span class="NLM_article-title">The mixed-lineage leukemia fusion partner AF4 stimulates RNA polymerase II transcriptional elongation and mediates coordinated chromatin remodeling</span> <span class="citation_source-journal">Hum. Mol. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">92</span><span class="NLM_x">–</span> <span class="NLM_lpage">106</span><span class="refDoi"> DOI: 10.1093/hmg/ddl444</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1093%2Fhmg%2Fddl444" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=17135274" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BD2sXotVWqsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2007&pages=92-106&author=E.+Bitounauthor=P.+L.+Oliverauthor=K.+E.+Davies&title=The+mixed-lineage+leukemia+fusion+partner+AF4+stimulates+RNA+polymerase+II+transcriptional+elongation+and+mediates+coordinated+chromatin+remodeling&doi=10.1093%2Fhmg%2Fddl444"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">The mixed-lineage leukemia fusion partner AF4 stimulates RNA polymerase II transcriptional elongation and mediates coordinated chromatin remodeling</span></div><div class="casAuthors">Bitoun, Emmanuelle; Oliver, Peter L.; Davies, Kay E.</div><div class="citationInfo"><span class="NLM_cas:title">Human Molecular Genetics</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">92-106</span>CODEN:
                <span class="NLM_cas:coden">HMGEE5</span>;
        ISSN:<span class="NLM_cas:issn">0964-6906</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">AF4 gene, frequently translocated with mixed-lineage leukemia (MLL) in childhood acute leukemia, encodes a putative transcriptional activator of the AF4/LAF4/FMR2 (ALF) protein family previously implicated in lymphopoiesis and Purkinje cell function in the cerebellum.  Here, we provide the first evidence for a direct role of AF4 in the regulation of transcriptional elongation by RNA polymerase II (Pol II).  We demonstrate that mouse Af4 functions as a pos. regulator of Pol II transcription elongation factor b (P-TEFb) kinase and, in complex with MLL fusion partners Af9, Enl and Af10, as a mediator of histone H3-K79 methylation by recruiting Dot1 to elongating Pol II.  These pathways are interconnected and tightly regulated by the P-TEFb-dependent phosphorylation of Af4, Af9 and Enl which controls their transactivation activity and/or protein stability.  Consistently, increased levels of phosphorylated Pol II and methylated H3-K79 are obsd. in the ataxic mouse mutant robotic, an over-expression model of Af4.  Finally, we confirm the functional relevance of Af4, Enl and Af9 to the regulation of gene transcription as their over-expression strongly stimulates P-TEFb-dependent transcription of a luciferase reporter gene.  Our findings uncover a central role for these proteins in the regulation of transcriptional elongation and coordinated histone methylation, providing valuable insight into their contribution to leukemogenesis and neurodegeneration.  Since these activities likely extend to the entire ALF protein family, this study also significantly inputs our understanding of the mol. basis of FRAXE mental retardation syndrome in which FMR2 expression is silenced.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTx1-wiI5sbrVg90H21EOLACvtfcHk0liDAkWxLVc8-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXotVWqsg%253D%253D&md5=82f9f9ce37a791034adc19c0eaeea078</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1093%2Fhmg%2Fddl444&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fhmg%252Fddl444%26sid%3Dliteratum%253Aachs%26aulast%3DBitoun%26aufirst%3DE.%26aulast%3DOliver%26aufirst%3DP.%2BL.%26aulast%3DDavies%26aufirst%3DK.%2BE.%26atitle%3DThe%2520mixed-lineage%2520leukemia%2520fusion%2520partner%2520AF4%2520stimulates%2520RNA%2520polymerase%2520II%2520transcriptional%2520elongation%2520and%2520mediates%2520coordinated%2520chromatin%2520remodeling%26jtitle%3DHum.%2520Mol.%2520Genet.%26date%3D2007%26volume%3D16%26spage%3D92%26epage%3D106%26doi%3D10.1093%2Fhmg%2Fddl444" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Krivtsov, A. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lemieux, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Faber, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vempati, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sinha, A. U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xia, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jesneck, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bracken, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Silverman, L. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kutok, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kung, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Armstrong, S. A.</span><span> </span><span class="NLM_article-title">H3K79 methylation profiles define murine and human MLL-AF4 leukemias</span> <span class="citation_source-journal">Cancer Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">355</span><span class="NLM_x">–</span> <span class="NLM_lpage">368</span><span class="refDoi"> DOI: 10.1016/j.ccr.2008.10.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1016%2Fj.ccr.2008.10.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=18977325" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtlOjsL3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2008&pages=355-368&author=A.+V.+Krivtsovauthor=Z.+Fengauthor=M.+E.+Lemieuxauthor=J.+Faberauthor=S.+Vempatiauthor=A.+U.+Sinhaauthor=X.+Xiaauthor=J.+Jesneckauthor=A.+P.+Brackenauthor=L.+B.+Silvermanauthor=J.+L.+Kutokauthor=A.+L.+Kungauthor=S.+A.+Armstrong&title=H3K79+methylation+profiles+define+murine+and+human+MLL-AF4+leukemias&doi=10.1016%2Fj.ccr.2008.10.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">H3K79 methylation profiles define murine and human MLL-AF4 leukemias</span></div><div class="casAuthors">Krivtsov, Andrei V.; Feng, Zhaohui; Lemieux, Madeleine E.; Faber, Joerg; Vempati, Sridhar; Sinha, Amit U.; Xia, Xiaobo; Jesneck, Jonathan; Bracken, Adrian P.; Silverman, Lewis B.; Kutok, Jeffery L.; Kung, Andrew L.; Armstrong, Scott A.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">355-368</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">We created a mouse model wherein conditional expression of an MLL-AF4 fusion oncogene induces B precursor acute lymphoblastic (ALL) or acute myeloid leukemias (AML).  Gene expression profile anal. of the ALL cells demonstrated significant overlap with human MLL-rearranged ALL.  ChIP-chip anal. demonstrated histone H3 lysine 79 (H3K79) methylation profiles that correlated with MLL-AF4-assocd. gene expression profiles in murine ALLs and in human MLL-rearranged leukemias.  Human MLL-rearranged ALLs could be distinguished from other ALLs by their H3K79 profiles, and suppression of the H3K79 methyltransferase DOT1L inhibited expression of crit. MLL-AF4 target genes.  We thus demonstrate that ectopic H3K79 methylation is a distinguishing feature of murine and human MLL-AF4 ALLs and is important for maintenance of MLL-AF4-driven gene expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouUg_fkZBiabVg90H21EOLACvtfcHk0liDAkWxLVc8-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtlOjsL3F&md5=755a12b8819e8e4a798b434a6818c647</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2008.10.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2008.10.001%26sid%3Dliteratum%253Aachs%26aulast%3DKrivtsov%26aufirst%3DA.%2BV.%26aulast%3DFeng%26aufirst%3DZ.%26aulast%3DLemieux%26aufirst%3DM.%2BE.%26aulast%3DFaber%26aufirst%3DJ.%26aulast%3DVempati%26aufirst%3DS.%26aulast%3DSinha%26aufirst%3DA.%2BU.%26aulast%3DXia%26aufirst%3DX.%26aulast%3DJesneck%26aufirst%3DJ.%26aulast%3DBracken%26aufirst%3DA.%2BP.%26aulast%3DSilverman%26aufirst%3DL.%2BB.%26aulast%3DKutok%26aufirst%3DJ.%2BL.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DArmstrong%26aufirst%3DS.%2BA.%26atitle%3DH3K79%2520methylation%2520profiles%2520define%2520murine%2520and%2520human%2520MLL-AF4%2520leukemias%26jtitle%3DCancer%2520Cell%26date%3D2008%26volume%3D14%26spage%3D355%26epage%3D368%26doi%3D10.1016%2Fj.ccr.2008.10.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="note"><p class="first last">For work from us, see the following:</p></div><div class="NLM_citation" id="rightTab-cit23a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Yao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anglin, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prasad, B. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, Y.</span><span> </span><span class="NLM_article-title">Selective inhibitors of histone methyltransferase DOT1L: design, synthesis, and crystallographic studies</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">133</span><span class="NLM_x">, </span> <span class="NLM_fpage">16746</span><span class="NLM_x">–</span> <span class="NLM_lpage">16749</span><span class="refDoi"> DOI: 10.1021/ja206312b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja206312b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2011&pages=16746-16749&author=Y.+Yaoauthor=P.+Chenauthor=J.+Diaoauthor=G.+Chengauthor=L.+Dengauthor=J.+L.+Anglinauthor=B.+V.+Prasadauthor=Y.+Song&title=Selective+inhibitors+of+histone+methyltransferase+DOT1L%3A+design%2C+synthesis%2C+and+crystallographic+studies&doi=10.1021%2Fja206312b"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23a&amp;dbid=16384&amp;doi=10.1021%2Fja206312b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja206312b%26sid%3Dliteratum%253Aachs%26aulast%3DYao%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DP.%26aulast%3DDiao%26aufirst%3DJ.%26aulast%3DCheng%26aufirst%3DG.%26aulast%3DDeng%26aufirst%3DL.%26aulast%3DAnglin%26aufirst%3DJ.%2BL.%26aulast%3DPrasad%26aufirst%3DB.%2BV.%26aulast%3DSong%26aufirst%3DY.%26atitle%3DSelective%2520inhibitors%2520of%2520histone%2520methyltransferase%2520DOT1L%253A%2520design%252C%2520synthesis%252C%2520and%2520crystallographic%2520studies%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2011%26volume%3D133%26spage%3D16746%26epage%3D16749%26doi%3D10.1021%2Fja206312b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit23b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Anglin, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cai, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, Y.</span><span> </span><span class="NLM_article-title">Synthesis and structure-activity relationship investigation of adenosine-containing inhibitors of histone methyltransferase DOT1L</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">8066</span><span class="NLM_x">–</span> <span class="NLM_lpage">8074</span><span class="refDoi"> DOI: 10.1021/jm300917h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm300917h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1KjsrjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=8066-8074&author=J.+L.+Anglinauthor=L.+Dengauthor=Y.+Yaoauthor=G.+Caiauthor=Z.+Liuauthor=H.+Jiangauthor=G.+Chengauthor=P.+Chenauthor=S.+Dongauthor=Y.+Song&title=Synthesis+and+structure-activity+relationship+investigation+of+adenosine-containing+inhibitors+of+histone+methyltransferase+DOT1L&doi=10.1021%2Fjm300917h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23bR"><div class="casContent"><span class="casTitleNuber">23b</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and Structure-Activity Relationship Investigation of Adenosine-Containing Inhibitors of Histone Methyltransferase DOT1L</span></div><div class="casAuthors">Anglin, Justin L.; Deng, Lisheng; Yao, Yuan; Cai, Guobin; Liu, Zhen; Jiang, Hong; Cheng, Gang; Chen, Pinhong; Dong, Shuo; Song, Yongcheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">8066-8074</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Histone-3-lysine79 (H3K79) methyltransferase DOT1L has been found to be a drug target for acute leukemia with MLL (mixed lineage leukemia) gene translocation.  A total of 55 adenosine-contg. compds., e.g. I, were designed and synthesized, among which several potent DOT1L inhibitors were identified with Ki values as low as 0.5 nM.  These compds. also show high selectivity (>4500-fold) over three other histone methyltransferases.  Structure-activity relationships (SAR) of these compds. for their inhibitory activities against DOT1L are discussed.  Potent DOT1L inhibitors exhibit selective activity against the proliferation of MLL-translocated leukemia cell lines with EC50 values of 4-11 μM.  Isothermal titrn. calorimetry studies showed that two representative inhibitors bind with a high affinity to the DOT1L:nucleosome complex and only compete with the enzyme cofactor SAM (S-adenosyl-L-methionine) but not the substrate nucleosome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3lGfG_WNicbVg90H21EOLACvtfcHk0liDAkWxLVc8-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1KjsrjO&md5=d869fdfbf94d9fa7282086d8f507f11b</span></div><a href="/servlet/linkout?suffix=cit23b&amp;dbid=16384&amp;doi=10.1021%2Fjm300917h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm300917h%26sid%3Dliteratum%253Aachs%26aulast%3DAnglin%26aufirst%3DJ.%2BL.%26aulast%3DDeng%26aufirst%3DL.%26aulast%3DYao%26aufirst%3DY.%26aulast%3DCai%26aufirst%3DG.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DCheng%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DP.%26aulast%3DDong%26aufirst%3DS.%26aulast%3DSong%26aufirst%3DY.%26atitle%3DSynthesis%2520and%2520structure-activity%2520relationship%2520investigation%2520of%2520adenosine-containing%2520inhibitors%2520of%2520histone%2520methyltransferase%2520DOT1L%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D8066%26epage%3D8074%26doi%3D10.1021%2Fjm300917h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit23c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Deng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Redell, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, Y.</span><span> </span><span class="NLM_article-title">Synthesis, activity and metabolic stability of non-ribose containing inhibitors of histone methyltransferase DOT1L</span> <span class="citation_source-journal">MedChemComm</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">822</span><span class="NLM_x">–</span> <span class="NLM_lpage">826</span><span class="refDoi"> DOI: 10.1039/c3md00021d</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1039%2Fc3md00021d" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=23795283" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BC3sXmvVygs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=822-826&author=L.+Dengauthor=L.+Zhangauthor=Y.+Yaoauthor=C.+Wangauthor=M.+S.+Redellauthor=S.+Dongauthor=Y.+Song&title=Synthesis%2C+activity+and+metabolic+stability+of+non-ribose+containing+inhibitors+of+histone+methyltransferase+DOT1L&doi=10.1039%2Fc3md00021d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23cR"><div class="casContent"><span class="casTitleNuber">23c</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis, activity and metabolic stability of non-ribose containing inhibitors of histone methyltransferase DOT1L</span></div><div class="casAuthors">Deng, Lisheng; Zhang, Li; Yao, Yuan; Wang, Cong; Redell, Michele S.; Dong, Shuo; Song, Yongcheng</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">822-826</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Histone methyltransferase DOT1L is a drug target for MLL leukemia.  We report an efficient synthesis of a cyclopentane-contg. adenosine nucleoside compd. (I) that potently and selectively inhibits DOT1L (Ki = 1.1 nM) as well as H3K79 methylation (IC50 ∼ 200 nM).  Importantly, this compd. exhibits a high stability in plasma and liver microsomes, suggesting it is a better drug candidate.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-mJfoEIVrvLVg90H21EOLACvtfcHk0lixylFZsU15fA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXmvVygs7o%253D&md5=202e6bf4c9eeed38982b0f036a253425</span></div><a href="/servlet/linkout?suffix=cit23c&amp;dbid=16384&amp;doi=10.1039%2Fc3md00021d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc3md00021d%26sid%3Dliteratum%253Aachs%26aulast%3DDeng%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DYao%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DRedell%26aufirst%3DM.%2BS.%26aulast%3DDong%26aufirst%3DS.%26aulast%3DSong%26aufirst%3DY.%26atitle%3DSynthesis%252C%2520activity%2520and%2520metabolic%2520stability%2520of%2520non-ribose%2520containing%2520inhibitors%2520of%2520histone%2520methyltransferase%2520DOT1L%26jtitle%3DMedChemComm%26date%3D2013%26volume%3D4%26spage%3D822%26epage%3D826%26doi%3D10.1039%2Fc3md00021d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit23d"><span><span class="NLM_label">(d) </span><span class="NLM_contrib-group">Liu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Redell, M.</span><span> </span><span class="NLM_article-title">DOT1L Inhibition Sensitizes MLL-Rearranged AML to Chemotherapy</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">e98270</span><span class="refDoi"> DOI: 10.1371/journal.pone.0098270</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23d&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1371%2Fjournal.pone.0098270" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=e98270&author=W.+Liuauthor=L.+Dengauthor=Y.+Songauthor=M.+Redell&title=DOT1L+Inhibition+Sensitizes+MLL-Rearranged+AML+to+Chemotherapy&doi=10.1371%2Fjournal.pone.0098270"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23d&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0098270&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0098270%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DDeng%26aufirst%3DL.%26aulast%3DSong%26aufirst%3DY.%26aulast%3DRedell%26aufirst%3DM.%26atitle%3DDOT1L%2520Inhibition%2520Sensitizes%2520MLL-Rearranged%2520AML%2520to%2520Chemotherapy%26jtitle%3DPLoS%2520One%26date%3D2014%26volume%3D9%26spage%3De98270%26doi%3D10.1371%2Fjournal.pone.0098270" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="note"><p class="first last">For work from other groups, seethe following:</p></div><div class="NLM_citation" id="rightTab-cit24a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Daigle, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olhava, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Therkelsen, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Majer, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sneeringer, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnston, L. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuntz, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chesworth, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moyer, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bernt, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tseng, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kung, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Armstrong, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richon, V. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollock, R. M.</span><span> </span><span class="NLM_article-title">Selective killing of mixed lineage leukemia cells by a potent small-molecule DOT1L inhibitor</span> <span class="citation_source-journal">Cancer Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">53</span><span class="NLM_x">–</span> <span class="NLM_lpage">65</span><span class="refDoi"> DOI: 10.1016/j.ccr.2011.06.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1016%2Fj.ccr.2011.06.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=21741596" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BC3MXoslCms7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2011&pages=53-65&author=S.+R.+Daigleauthor=E.+J.+Olhavaauthor=C.+A.+Therkelsenauthor=C.+R.+Majerauthor=C.+J.+Sneeringerauthor=J.+Songauthor=L.+D.+Johnstonauthor=M.+P.+Scottauthor=J.+J.+Smithauthor=Y.+Xiaoauthor=L.+Jinauthor=K.+W.+Kuntzauthor=R.+Chesworthauthor=M.+P.+Moyerauthor=K.+M.+Berntauthor=J.+C.+Tsengauthor=A.+L.+Kungauthor=S.+A.+Armstrongauthor=R.+A.+Copelandauthor=V.+M.+Richonauthor=R.+M.+Pollock&title=Selective+killing+of+mixed+lineage+leukemia+cells+by+a+potent+small-molecule+DOT1L+inhibitor&doi=10.1016%2Fj.ccr.2011.06.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24aR"><div class="casContent"><span class="casTitleNuber">24a</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Killing of Mixed Lineage Leukemia Cells by a Potent Small-Molecule DOT1L Inhibitor</span></div><div class="casAuthors">Daigle, Scott R.; Olhava, Edward J.; Therkelsen, Carly A.; Majer, Christina R.; Sneeringer, Christopher J.; Song, Jeffrey; Johnston, L. Danielle; Scott, Margaret Porter; Smith, Jesse J.; Xiao, Yonghong; Jin, Lei; Kuntz, Kevin W.; Chesworth, Richard; Moyer, Mikel P.; Bernt, Kathrin M.; Tseng, Jen-Chieh; Kung, Andrew L.; Armstrong, Scott A.; Copeland, Robert A.; Richon, Victoria M.; Pollock, Roy M.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">53-65</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Mislocated enzymic activity of DOT1L has been proposed as a driver of leukemogenesis in mixed lineage leukemia (MLL).  The characterization of EPZ004777, a potent, selective inhibitor of DOT1L is reported.  Treatment of MLL cells with the compd. selectively inhibits H3K79 methylation and blocks expression of leukemogenic genes.  Exposure of leukemic cells to EPZ004777 results in selective killing of those cells bearing the MLL gene translocation, with little effect on non-MLL-translocated cells.  Finally, in vivo administration of EPZ004777 leads to extension of survival in a mouse MLL xenograft model.  These results provide compelling support for DOT1L inhibition as a basis for targeted therapeutics against MLL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodbBbxHFmyybVg90H21EOLACvtfcHk0lixylFZsU15fA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXoslCms7c%253D&md5=1d39f99c2983b6b5fccc079110092c3d</span></div><a href="/servlet/linkout?suffix=cit24a&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2011.06.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2011.06.009%26sid%3Dliteratum%253Aachs%26aulast%3DDaigle%26aufirst%3DS.%2BR.%26aulast%3DOlhava%26aufirst%3DE.%2BJ.%26aulast%3DTherkelsen%26aufirst%3DC.%2BA.%26aulast%3DMajer%26aufirst%3DC.%2BR.%26aulast%3DSneeringer%26aufirst%3DC.%2BJ.%26aulast%3DSong%26aufirst%3DJ.%26aulast%3DJohnston%26aufirst%3DL.%2BD.%26aulast%3DScott%26aufirst%3DM.%2BP.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DXiao%26aufirst%3DY.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DKuntz%26aufirst%3DK.%2BW.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DBernt%26aufirst%3DK.%2BM.%26aulast%3DTseng%26aufirst%3DJ.%2BC.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DArmstrong%26aufirst%3DS.%2BA.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DRichon%26aufirst%3DV.%2BM.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26atitle%3DSelective%2520killing%2520of%2520mixed%2520lineage%2520leukemia%2520cells%2520by%2520a%2520potent%2520small-molecule%2520DOT1L%2520inhibitor%26jtitle%3DCancer%2520Cell%26date%3D2011%26volume%3D20%26spage%3D53%26epage%3D65%26doi%3D10.1016%2Fj.ccr.2011.06.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit24b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Basavapathruni, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Daigle, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Majer, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Therkelsen, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wigle, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuntz, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chesworth, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollock, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moyer, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richon, V. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olhava, E. J.</span><span> </span><span class="NLM_article-title">Conformational adaptation drives potent, selective and durable inhibition of the human protein methyltransferase DOT1L</span> <span class="citation_source-journal">Chem. Biol. Drug Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">80</span><span class="NLM_x">, </span> <span class="NLM_fpage">971</span><span class="NLM_x">–</span> <span class="NLM_lpage">980</span><span class="refDoi"> DOI: 10.1111/cbdd.12050</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1111%2Fcbdd.12050" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=22978415" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslClsr7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=80&publication_year=2012&pages=971-980&author=A.+Basavapathruniauthor=L.+Jinauthor=S.+R.+Daigleauthor=C.+R.+Majerauthor=C.+A.+Therkelsenauthor=T.+J.+Wigleauthor=K.+W.+Kuntzauthor=R.+Chesworthauthor=R.+M.+Pollockauthor=M.+P.+Scottauthor=M.+P.+Moyerauthor=V.+M.+Richonauthor=R.+A.+Copelandauthor=E.+J.+Olhava&title=Conformational+adaptation+drives+potent%2C+selective+and+durable+inhibition+of+the+human+protein+methyltransferase+DOT1L&doi=10.1111%2Fcbdd.12050"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24bR"><div class="casContent"><span class="casTitleNuber">24b</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational adaptation drives potent, selective and durable inhibition of the human protein methyltransferase DOT1L</span></div><div class="casAuthors">Basavapathruni, Aravind; Jin, Lei; Daigle, Scott R.; Majer, Christina R. A.; Therkelsen, Carly A.; Wigle, Tim J.; Kuntz, Kevin W.; Chesworth, Richard; Pollock, Roy M.; Scott, Margaret P.; Moyer, Mikel P.; Richon, Victoria M.; Copeland, Robert A.; Olhava, Edward J.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">80</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">971-980</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">DOT1L is the human protein methyltransferase responsible for catalyzing the methylation of histone H3 on lysine 79 (H3K79).  The ectopic activity of DOT1L, assocd. with the chromosomal translocation that is a universal hallmark of MLL-rearranged leukemia, is a required driver of leukemogenesis in this malignancy.  Here, we present studies on the structure-activity relation of aminonucleoside-based DOT1L inhibitors.  Within this series, we find that improvements in target enzyme affinity and selectivity are driven entirely by diminution of the dissocn. rate const. for the enzyme-inhibitor complex, leading to long residence times for the binary complex.  The biochem. Ki and residence times measured for these inhibitors correlate well with their effects on intracellular H3K79 methylation and MLL-rearranged leukemic cell killing.  Crystallog. studies reveal a conformational adaptation mechanism assocd. with high-affinity inhibitor binding and prolonged residence time; these studies also suggest that conformational adaptation likewise plays a crit. role in natural ligand interactions with the enzyme, hence, facilitating enzyme turnover.  These results provide crit. insights into the role of conformational adaptation in the enzymic mechanism of catalysis and in pharmacol. intervention for DOT1L and other members of this enzyme class.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJm_d3iX3p77Vg90H21EOLACvtfcHk0ljMvDMTehQg1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslClsr7O&md5=de628fcd2c37b5f6496cc3b483125309</span></div><a href="/servlet/linkout?suffix=cit24b&amp;dbid=16384&amp;doi=10.1111%2Fcbdd.12050&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fcbdd.12050%26sid%3Dliteratum%253Aachs%26aulast%3DBasavapathruni%26aufirst%3DA.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DDaigle%26aufirst%3DS.%2BR.%26aulast%3DMajer%26aufirst%3DC.%2BR.%26aulast%3DTherkelsen%26aufirst%3DC.%2BA.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DKuntz%26aufirst%3DK.%2BW.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26aulast%3DScott%26aufirst%3DM.%2BP.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DRichon%26aufirst%3DV.%2BM.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DOlhava%26aufirst%3DE.%2BJ.%26atitle%3DConformational%2520adaptation%2520drives%2520potent%252C%2520selective%2520and%2520durable%2520inhibition%2520of%2520the%2520human%2520protein%2520methyltransferase%2520DOT1L%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2012%26volume%3D80%26spage%3D971%26epage%3D980%26doi%3D10.1111%2Fcbdd.12050" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit24c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Yu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chory, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wernimont, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tempel, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scopton, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Federation, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marineau, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marcellus, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iacob, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Engen, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Griffin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aman, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wienholds, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pineda, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Estiu, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shatseva, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajian, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-Awar, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dick, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bradner, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span> </span><span class="NLM_article-title">Catalytic site remodelling of the DOT1L methyltransferase by selective inhibitors</span> <span class="citation_source-journal">Nat. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">1288</span><span class="refDoi"> DOI: 10.1038/ncomms2304</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1038%2Fncomms2304" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=23250418" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A280%3ADC%252BC3s3jtVWlsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2012&pages=1288&author=W.+Yuauthor=E.+J.+Choryauthor=A.+K.+Wernimontauthor=W.+Tempelauthor=A.+Scoptonauthor=A.+Federationauthor=J.+J.+Marineauauthor=J.+Qiauthor=D.+Barsyte-Lovejoyauthor=J.+Yiauthor=R.+Marcellusauthor=R.+E.+Iacobauthor=J.+R.+Engenauthor=C.+Griffinauthor=A.+Amanauthor=E.+Wienholdsauthor=F.+Liauthor=J.+Pinedaauthor=G.+Estiuauthor=T.+Shatsevaauthor=T.+Hajianauthor=R.+Al-Awarauthor=J.+E.+Dickauthor=M.+Vedadiauthor=P.+J.+Brownauthor=C.+H.+Arrowsmithauthor=J.+E.+Bradnerauthor=M.+Schapira&title=Catalytic+site+remodelling+of+the+DOT1L+methyltransferase+by+selective+inhibitors&doi=10.1038%2Fncomms2304"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24cR"><div class="casContent"><span class="casTitleNuber">24c</span><div class="casTitle"><span class="NLM_cas:atitle">Catalytic site remodelling of the DOT1L methyltransferase by selective inhibitors</span></div><div class="casAuthors">Yu Wenyu; Chory Emma J; Wernimont Amy K; Tempel Wolfram; Scopton Alex; Federation Alexander; Marineau Jason J; Qi Jun; Barsyte-Lovejoy Dalia; Yi Joanna; Marcellus Richard; Iacob Roxana E; Engen John R; Griffin Carly; Aman Ahmed; Wienholds Erno; Li Fengling; Pineda Javier; Estiu Guillermina; Shatseva Tatiana; Hajian Taraneh; Al-Awar Rima; Dick John E; Vedadi Masoud; Brown Peter J; Arrowsmith Cheryl H; Bradner James E; Schapira Matthieu</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1288</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Selective inhibition of protein methyltransferases is a promising new approach to drug discovery.  An attractive strategy towards this goal is the development of compounds that selectively inhibit binding of the cofactor, S-adenosylmethionine, within specific protein methyltransferases.  Here we report the three-dimensional structure of the protein methyltransferase DOT1L bound to EPZ004777, the first S-adenosylmethionine-competitive inhibitor of a protein methyltransferase with in vivo efficacy.  This structure and those of four new analogues reveal remodelling of the catalytic site.  EPZ004777 and a brominated analogue, SGC0946, inhibit DOT1L in vitro and selectively kill mixed lineage leukaemia cells, in which DOT1L is aberrantly localized via interaction with an oncogenic MLL fusion protein.  These data provide important new insight into mechanisms of cell-active S-adenosylmethionine-competitive protein methyltransferase inhibitors, and establish a foundation for the further development of drug-like inhibitors of DOT1L for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcST2GJQO5XUmjDnmmyUf5qOfW6udTcc2ealtyFRw9YDfrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3s3jtVWlsQ%253D%253D&md5=f7b501ac4188b14b48fc435637e0e192</span></div><a href="/servlet/linkout?suffix=cit24c&amp;dbid=16384&amp;doi=10.1038%2Fncomms2304&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms2304%26sid%3Dliteratum%253Aachs%26aulast%3DYu%26aufirst%3DW.%26aulast%3DChory%26aufirst%3DE.%2BJ.%26aulast%3DWernimont%26aufirst%3DA.%2BK.%26aulast%3DTempel%26aufirst%3DW.%26aulast%3DScopton%26aufirst%3DA.%26aulast%3DFederation%26aufirst%3DA.%26aulast%3DMarineau%26aufirst%3DJ.%2BJ.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DYi%26aufirst%3DJ.%26aulast%3DMarcellus%26aufirst%3DR.%26aulast%3DIacob%26aufirst%3DR.%2BE.%26aulast%3DEngen%26aufirst%3DJ.%2BR.%26aulast%3DGriffin%26aufirst%3DC.%26aulast%3DAman%26aufirst%3DA.%26aulast%3DWienholds%26aufirst%3DE.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DPineda%26aufirst%3DJ.%26aulast%3DEstiu%26aufirst%3DG.%26aulast%3DShatseva%26aufirst%3DT.%26aulast%3DHajian%26aufirst%3DT.%26aulast%3DAl-Awar%26aufirst%3DR.%26aulast%3DDick%26aufirst%3DJ.%2BE.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DSchapira%26aufirst%3DM.%26atitle%3DCatalytic%2520site%2520remodelling%2520of%2520the%2520DOT1L%2520methyltransferase%2520by%2520selective%2520inhibitors%26jtitle%3DNat.%2520Commun.%26date%3D2012%26volume%3D3%26spage%3D1288%26doi%3D10.1038%2Fncomms2304" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit24d"><span><span class="NLM_label">(d) </span><span class="NLM_contrib-group">Daigle, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olhava, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Therkelsen, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Basavapathruni, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boriack-Sjodin, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allain, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klaus, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raimondi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waters, N. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chesworth, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moyer, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richon, V. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollock, R. M.</span><span> </span><span class="NLM_article-title">Potent inhibition of DOT1L as treatment for MLL-fusion leukemia</span> <span class="citation_source-journal">Blood</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">122</span><span class="NLM_x">, </span> <span class="NLM_fpage">1017</span><span class="NLM_x">–</span> <span class="NLM_lpage">1025</span><span class="refDoi"> DOI: 10.1182/blood-2013-04-497644</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24d&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1182%2Fblood-2013-04-497644" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=122&publication_year=2013&pages=1017-1025&author=S.+R.+Daigleauthor=E.+J.+Olhavaauthor=C.+A.+Therkelsenauthor=A.+Basavapathruniauthor=L.+Jinauthor=P.+A.+Boriack-Sjodinauthor=C.+J.+Allainauthor=C.+R.+Klausauthor=A.+Raimondiauthor=M.+P.+Scottauthor=N.+J.+Watersauthor=R.+Chesworthauthor=M.+P.+Moyerauthor=R.+A.+Copelandauthor=V.+M.+Richonauthor=R.+M.+Pollock&title=Potent+inhibition+of+DOT1L+as+treatment+for+MLL-fusion+leukemia&doi=10.1182%2Fblood-2013-04-497644"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24d&amp;dbid=16384&amp;doi=10.1182%2Fblood-2013-04-497644&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2013-04-497644%26sid%3Dliteratum%253Aachs%26aulast%3DDaigle%26aufirst%3DS.%2BR.%26aulast%3DOlhava%26aufirst%3DE.%2BJ.%26aulast%3DTherkelsen%26aufirst%3DC.%2BA.%26aulast%3DBasavapathruni%26aufirst%3DA.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DBoriack-Sjodin%26aufirst%3DP.%2BA.%26aulast%3DAllain%26aufirst%3DC.%2BJ.%26aulast%3DKlaus%26aufirst%3DC.%2BR.%26aulast%3DRaimondi%26aufirst%3DA.%26aulast%3DScott%26aufirst%3DM.%2BP.%26aulast%3DWaters%26aufirst%3DN.%2BJ.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DRichon%26aufirst%3DV.%2BM.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26atitle%3DPotent%2520inhibition%2520of%2520DOT1L%2520as%2520treatment%2520for%2520MLL-fusion%2520leukemia%26jtitle%3DBlood%26date%3D2013%26volume%3D122%26spage%3D1017%26epage%3D1025%26doi%3D10.1182%2Fblood-2013-04-497644" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Binda, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valente, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Romanenghi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pilotto, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cirilli, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karytinos, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciossani, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Botrugno, O. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Forneris, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tardugno, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edmondson, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Minucci, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattevi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mai, A.</span><span> </span><span class="NLM_article-title">Biochemical, structural, and biological evaluation of tranylcypromine derivatives as inhibitors of histone demethylases LSD1 and LSD2</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">132</span><span class="NLM_x">, </span> <span class="NLM_fpage">6827</span><span class="NLM_x">–</span> <span class="NLM_lpage">6833</span><span class="refDoi"> DOI: 10.1021/ja101557k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja101557k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BC3cXltFChur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2010&pages=6827-6833&author=C.+Bindaauthor=S.+Valenteauthor=M.+Romanenghiauthor=S.+Pilottoauthor=R.+Cirilliauthor=A.+Karytinosauthor=G.+Ciossaniauthor=O.+A.+Botrugnoauthor=F.+Fornerisauthor=M.+Tardugnoauthor=D.+E.+Edmondsonauthor=S.+Minucciauthor=A.+Matteviauthor=A.+Mai&title=Biochemical%2C+structural%2C+and+biological+evaluation+of+tranylcypromine+derivatives+as+inhibitors+of+histone+demethylases+LSD1+and+LSD2&doi=10.1021%2Fja101557k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Biochemical, Structural, and Biological Evaluation of Tranylcypromine Derivatives as Inhibitors of Histone Demethylases LSD1 and LSD2</span></div><div class="casAuthors">Binda, Claudia; Valente, Sergio; Romanenghi, Mauro; Pilotto, Simona; Cirilli, Roberto; Karytinos, Aristotele; Ciossani, Giuseppe; Botrugno, Oronza A.; Forneris, Federico; Tardugno, Maria; Edmondson, Dale E.; Minucci, Saverio; Mattevi, Andrea; Mai, Antonello</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">132</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">6827-6833</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">LSD1 and LSD2 histone demethylases are implicated in a no. of physiol. and pathol. processes, ranging from tumorigenesis to herpes virus infection.  A comprehensive structural, biochem., and cellular study is presented here to probe the potential of these enzymes for epigenetic therapies.  This approach employs tranylcypromine as a chem. scaffold for the design of novel demethylase inhibitors.  This drug is a clin. validated antidepressant known to target monoamine oxidases A and B.  These two flavoenzymes are structurally related to LSD1 and LSD2.  Mechanistic and crystallog. studies of tranylcypromine inhibition reveal a lack of selectivity and differing covalent modifications of the FAD cofactor depending on the enantiomeric form.  These findings are pharmacol. relevant, since tranylcypromine is currently administered as a racemic mixt.  A large set of tranylcypromine analogs were synthesized and screened for inhibitory activities.  We found that the common evolutionary origin of LSD and MAO enzymes, despite their unrelated functions and substrate specificities, is reflected in related ligand-binding properties.  A few compds. with partial enzyme selectivity were identified.  The biol. activity of one of these new inhibitors was evaluated with a cellular model of acute promyelocytic leukemia chosen since its pathogenesis includes aberrant activities of several chromatin modifiers.  Marked effects on cell differentiation and an unprecedented synergistic activity with antileukemia drugs were obsd.  These data demonstrate that these LSD1/2 inhibitors are of potential relevance for the treatment of promyelocytic leukemia and, more generally, as tools to alter chromatin state with promise of a block of tumor progression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqH4F9uwn5Jx7Vg90H21EOLACvtfcHk0lhnqy0mCJ_jRQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXltFChur0%253D&md5=d6b28616bbb0455e139625fdb43696d3</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fja101557k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja101557k%26sid%3Dliteratum%253Aachs%26aulast%3DBinda%26aufirst%3DC.%26aulast%3DValente%26aufirst%3DS.%26aulast%3DRomanenghi%26aufirst%3DM.%26aulast%3DPilotto%26aufirst%3DS.%26aulast%3DCirilli%26aufirst%3DR.%26aulast%3DKarytinos%26aufirst%3DA.%26aulast%3DCiossani%26aufirst%3DG.%26aulast%3DBotrugno%26aufirst%3DO.%2BA.%26aulast%3DForneris%26aufirst%3DF.%26aulast%3DTardugno%26aufirst%3DM.%26aulast%3DEdmondson%26aufirst%3DD.%2BE.%26aulast%3DMinucci%26aufirst%3DS.%26aulast%3DMattevi%26aufirst%3DA.%26aulast%3DMai%26aufirst%3DA.%26atitle%3DBiochemical%252C%2520structural%252C%2520and%2520biological%2520evaluation%2520of%2520tranylcypromine%2520derivatives%2520as%2520inhibitors%2520of%2520histone%2520demethylases%2520LSD1%2520and%2520LSD2%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2010%26volume%3D132%26spage%3D6827%26epage%3D6833%26doi%3D10.1021%2Fja101557k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="note"><p class="first last">See recent reviews and the references therein.</p></div><div class="NLM_citation" id="rightTab-cit26a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Suzuki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyata, N.</span><span> </span><span class="NLM_article-title">Lysine demethylases inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">8236</span><span class="NLM_x">–</span> <span class="NLM_lpage">8250</span><span class="refDoi"> DOI: 10.1021/jm201048w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm201048w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1OrtrnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=8236-8250&author=T.+Suzukiauthor=N.+Miyata&title=Lysine+demethylases+inhibitors&doi=10.1021%2Fjm201048w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26aR"><div class="casContent"><span class="casTitleNuber">26a</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine Demethylases Inhibitors</span></div><div class="casAuthors">Suzuki, Takayoshi; Miyata, Naoki</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">8236-8250</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review, on the use of lysine demethylases inhibitors in therapy of various diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplp9MjSdvnn7Vg90H21EOLACvtfcHk0lhnqy0mCJ_jRQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1OrtrnF&md5=194d4bbc2e52672f5d5f019de0bb760c</span></div><a href="/servlet/linkout?suffix=cit26a&amp;dbid=16384&amp;doi=10.1021%2Fjm201048w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm201048w%26sid%3Dliteratum%253Aachs%26aulast%3DSuzuki%26aufirst%3DT.%26aulast%3DMiyata%26aufirst%3DN.%26atitle%3DLysine%2520demethylases%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D8236%26epage%3D8250%26doi%3D10.1021%2Fjm201048w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit26b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Mould, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McGonagle, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wiseman, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, E. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jordan, A. M.</span><span> </span><span class="NLM_article-title">Reversible inhibitors of LSD1 as therapeutic agents in acute myeloid leukemia: clinical significance and progress to date</span> <span class="citation_source-journal">Med. Res. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">35</span><span class="NLM_x">, </span> <span class="NLM_fpage">586</span><span class="NLM_x">–</span> <span class="NLM_lpage">618</span><span class="refDoi"> DOI: 10.1002/med.21334</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1002%2Fmed.21334" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=25418875" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlvFSju7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2015&pages=586-618&author=D.+P.+Mouldauthor=A.+E.+McGonagleauthor=D.+H.+Wisemanauthor=E.+L.+Williamsauthor=A.+M.+Jordan&title=Reversible+inhibitors+of+LSD1+as+therapeutic+agents+in+acute+myeloid+leukemia%3A+clinical+significance+and+progress+to+date&doi=10.1002%2Fmed.21334"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26bR"><div class="casContent"><span class="casTitleNuber">26b</span><div class="casTitle"><span class="NLM_cas:atitle">Reversible Inhibitors of LSD1 as Therapeutic Agents in Acute Myeloid Leukemia: Clinical Significance and Progress to Date</span></div><div class="casAuthors">Mould, Daniel P.; McGonagle, Alison E.; Wiseman, Daniel H.; Williams, Emma L.; Jordan, Allan M.</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">586-618</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">In the 10 years since the discovery of lysine-specific demethylase 1 (LSD1), this epigenetic eraser has emerged as an important target of interest in oncol.  More specifically, research has demonstrated that it plays an essential role in the self-renewal of leukemic stem cells in acute myeloid leukemia (AML).  This review will cover clin. aspects of AML, the role of epigenetics in the disease, and discuss the research that led to the first irreversible inhibitors of LSD1 entering clin. trials for the treatment of AML in 2014.  We also review recent achievements and progress in the development of potent and selective reversible inhibitors of LSD1.  These compds. differ in their mode of action from tranylcypromine derivs. and could facilitate novel biochem. studies to probe the pathways mediated by LSD1.  In this review, we will critically evaluate the strengths and weaknesses of published series of reversible LSD1 inhibitors.  Overall, while the development of reversible inhibitors to date has been less fruitful than that of irreversible inhibitors, there is still the possibility for their use to facilitate further research into the roles and functions of LSD1 and to expand the therapeutic applications of LSD1 inhibitors in the clinic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTFqxWxhZyUrVg90H21EOLACvtfcHk0lhnqy0mCJ_jRQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlvFSju7c%253D&md5=b669a61231659817709013572e8fdcf7</span></div><a href="/servlet/linkout?suffix=cit26b&amp;dbid=16384&amp;doi=10.1002%2Fmed.21334&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21334%26sid%3Dliteratum%253Aachs%26aulast%3DMould%26aufirst%3DD.%2BP.%26aulast%3DMcGonagle%26aufirst%3DA.%2BE.%26aulast%3DWiseman%26aufirst%3DD.%2BH.%26aulast%3DWilliams%26aufirst%3DE.%2BL.%26aulast%3DJordan%26aufirst%3DA.%2BM.%26atitle%3DReversible%2520inhibitors%2520of%2520LSD1%2520as%2520therapeutic%2520agents%2520in%2520acute%2520myeloid%2520leukemia%253A%2520clinical%2520significance%2520and%2520progress%2520to%2520date%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2015%26volume%3D35%26spage%3D586%26epage%3D618%26doi%3D10.1002%2Fmed.21334" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="note"><p class="first last">For more recent LSD1 inhibitors, see the following:</p></div><div class="NLM_citation" id="rightTab-cit27a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Neelamegam, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ricq, E. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Malvaez, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patnaik, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norton, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carlin, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hill, I. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wood, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haggarty, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hooker, J. M.</span><span> </span><span class="NLM_article-title">Brain-penetrant LSD1 inhibitors can block memory consolidation</span> <span class="citation_source-journal">ACS Chem. Neurosci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">120</span><span class="NLM_x">–</span> <span class="NLM_lpage">128</span><span class="refDoi"> DOI: 10.1021/cn200104y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cn200104y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVyrs77E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2012&pages=120-128&author=R.+Neelamegamauthor=E.+L.+Ricqauthor=M.+Malvaezauthor=D.+Patnaikauthor=S.+Nortonauthor=S.+M.+Carlinauthor=I.+T.+Hillauthor=M.+A.+Woodauthor=S.+J.+Haggartyauthor=J.+M.+Hooker&title=Brain-penetrant+LSD1+inhibitors+can+block+memory+consolidation&doi=10.1021%2Fcn200104y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27aR"><div class="casContent"><span class="casTitleNuber">27a</span><div class="casTitle"><span class="NLM_cas:atitle">Brain-Penetrant LSD1 Inhibitors Can Block Memory Consolidation</span></div><div class="casAuthors">Neelamegam, Ramesh; Ricq, Emily L.; Malvaez, Melissa; Patnaik, Debasis; Norton, Stephanie; Carlin, Stephen M.; Hill, Ian T.; Wood, Marcelo A.; Haggarty, Stephen J.; Hooker, Jacob M.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Neuroscience</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">120-128</span>CODEN:
                <span class="NLM_cas:coden">ACNCDM</span>;
        ISSN:<span class="NLM_cas:issn">1948-7193</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Modulation of histone modifications in the brain may represent a new mechanism for brain disorder therapy.  Post-translational modifications of histones regulate gene expression, affecting major cellular processes such as proliferation, differentiation, and function.  An important enzyme involved in one of these histone modifications is lysine specific demethylase 1 (LSD1).  This enzyme is flavin-dependent and exhibits homol. to amine oxidases.  Parnate (2-phenylcyclopropylamine (2-PCPA); tranylcypromine) is a potent inhibitor of monoamine oxidases, and derivs. of 2-PCPA have been used for development of selective LSD1 inhibitors based on the ability to form covalent adducts with FAD.  Here we report the synthesis and in vitro characterization of LSD1 inhibitors that bond covalently to FAD.  The two most potent and selective inhibitors were used to demonstrate brain penetration when administered systemically to rodents.  First, radiosynthesis of a positron-emitting analog was used to obtain preliminary biodistribution data and whole brain time-activity curves.  Second, we demonstrate that this series of LSD1 inhibitors is capable of producing a cognitive effect in a mouse model.  By using a memory formation paradigm, novel object recognition, we show that LSD1 inhibition can abolish long-term memory formation without affecting short-term memory, providing further evidence for the importance of reversible histone methylation in the function of the nervous system.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRTx9XGRzjgrVg90H21EOLACvtfcHk0lj2Fyh0FeCPyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVyrs77E&md5=cd365810b397d0ece732ea853a3e93da</span></div><a href="/servlet/linkout?suffix=cit27a&amp;dbid=16384&amp;doi=10.1021%2Fcn200104y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcn200104y%26sid%3Dliteratum%253Aachs%26aulast%3DNeelamegam%26aufirst%3DR.%26aulast%3DRicq%26aufirst%3DE.%2BL.%26aulast%3DMalvaez%26aufirst%3DM.%26aulast%3DPatnaik%26aufirst%3DD.%26aulast%3DNorton%26aufirst%3DS.%26aulast%3DCarlin%26aufirst%3DS.%2BM.%26aulast%3DHill%26aufirst%3DI.%2BT.%26aulast%3DWood%26aufirst%3DM.%2BA.%26aulast%3DHaggarty%26aufirst%3DS.%2BJ.%26aulast%3DHooker%26aufirst%3DJ.%2BM.%26atitle%3DBrain-penetrant%2520LSD1%2520inhibitors%2520can%2520block%2520memory%2520consolidation%26jtitle%3DACS%2520Chem.%2520Neurosci.%26date%3D2012%26volume%3D3%26spage%3D120%26epage%3D128%26doi%3D10.1021%2Fcn200104y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit27b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Zheng, Y. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duan, Y. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zi, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lv, W. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ye, X. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mobley, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, Y. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Z. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, H. M.</span><span> </span><span class="NLM_article-title">Triazole-dithiocarbamate based selective lysine specific demethylase 1 (LSD1) inactivators inhibit gastric cancer cell growth, invasion, and migration</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">8543</span><span class="NLM_x">–</span> <span class="NLM_lpage">8560</span><span class="refDoi"> DOI: 10.1021/jm401002r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401002r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1ejt7nL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=8543-8560&author=Y.+C.+Zhengauthor=Y.+C.+Duanauthor=J.+L.+Maauthor=R.+M.+Xuauthor=X.+Ziauthor=W.+L.+Lvauthor=M.+M.+Wangauthor=X.+W.+Yeauthor=S.+Zhuauthor=D.+Mobleyauthor=Y.+Y.+Zhuauthor=J.+W.+Wangauthor=J.+F.+Liauthor=Z.+R.+Wangauthor=W.+Zhaoauthor=H.+M.+Liu&title=Triazole-dithiocarbamate+based+selective+lysine+specific+demethylase+1+%28LSD1%29+inactivators+inhibit+gastric+cancer+cell+growth%2C+invasion%2C+and+migration&doi=10.1021%2Fjm401002r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27bR"><div class="casContent"><span class="casTitleNuber">27b</span><div class="casTitle"><span class="NLM_cas:atitle">Triazole-Dithiocarbamate Based Selective Lysine Specific Demethylase 1 (LSD1) Inactivators Inhibit Gastric Cancer Cell Growth, Invasion, and Migration</span></div><div class="casAuthors">Zheng, Yi-Chao; Duan, Ying-Chao; Ma, Jin-Lian; Xu, Rui-Min; Zi, Xiaolin; Lv, Wen-Lei; Wang, Meng-Meng; Ye, Xian-Wei; Zhu, Shun; Mobley, David; Zhu, Yan-Yan; Wang, Jun-Wei; Li, Jin-Feng; Wang, Zhi-Ru; Zhao, Wen; Liu, Hong-Min</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">8543-8560</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Lysine specific demethylase 1 (LSD1), the first identified histone demethylase, plays an important role in epigenetic regulation of gene activation and repression.  The up-regulated LSD1's expression has been reported in several malignant tumors.  In the current study, we designed and synthesized five series of 1,2,3-triazole-dithiocarbamate hybrids and screened their inhibitory activity toward LSD1.  We found that some of these compds., esp. compd. (I), exhibited the most specific and robust inhibition of LSD1.  Interestingly, compd. I also showed potent and selective cytotoxicity against LSD1 overexpressing gastric cancer cell lines MGC-803 and HGC-27, as well as marked inhibition of cell migration and invasion, compared to 2-PCPA.  Furthermore, compd. I effectively reduced the tumor growth bared by human gastric cancer cells in vivo with no signs of adverse side effects.  These findings suggested that compd. I deserves further investigation as a lead compd. in the treatment of LSD1 overexpressing gastric cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjGdOvultbebVg90H21EOLACvtfcHk0lj2Fyh0FeCPyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1ejt7nL&md5=4e4594271ff6b62237ff17a87d6f04d8</span></div><a href="/servlet/linkout?suffix=cit27b&amp;dbid=16384&amp;doi=10.1021%2Fjm401002r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401002r%26sid%3Dliteratum%253Aachs%26aulast%3DZheng%26aufirst%3DY.%2BC.%26aulast%3DDuan%26aufirst%3DY.%2BC.%26aulast%3DMa%26aufirst%3DJ.%2BL.%26aulast%3DXu%26aufirst%3DR.%2BM.%26aulast%3DZi%26aufirst%3DX.%26aulast%3DLv%26aufirst%3DW.%2BL.%26aulast%3DWang%26aufirst%3DM.%2BM.%26aulast%3DYe%26aufirst%3DX.%2BW.%26aulast%3DZhu%26aufirst%3DS.%26aulast%3DMobley%26aufirst%3DD.%26aulast%3DZhu%26aufirst%3DY.%2BY.%26aulast%3DWang%26aufirst%3DJ.%2BW.%26aulast%3DLi%26aufirst%3DJ.%2BF.%26aulast%3DWang%26aufirst%3DZ.%2BR.%26aulast%3DZhao%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DH.%2BM.%26atitle%3DTriazole-dithiocarbamate%2520based%2520selective%2520lysine%2520specific%2520demethylase%25201%2520%2528LSD1%2529%2520inactivators%2520inhibit%2520gastric%2520cancer%2520cell%2520growth%252C%2520invasion%252C%2520and%2520migration%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D8543%26epage%3D8560%26doi%3D10.1021%2Fjm401002r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit27c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Sorna, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Theisen, E. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stephens, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Warner, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bearss, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vankayalapati, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharma, S.</span><span> </span><span class="NLM_article-title">High-throughput virtual screening identifies novel N′-(1-phenylethylidene)-benzohydrazides as potent, specific, and reversible LSD1 inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">9496</span><span class="NLM_x">–</span> <span class="NLM_lpage">9508</span><span class="refDoi"> DOI: 10.1021/jm400870h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400870h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslyks7fF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=9496-9508&author=V.+Sornaauthor=E.+R.+Theisenauthor=B.+Stephensauthor=S.+L.+Warnerauthor=D.+J.+Bearssauthor=H.+Vankayalapatiauthor=S.+Sharma&title=High-throughput+virtual+screening+identifies+novel+N%E2%80%B2-%281-phenylethylidene%29-benzohydrazides+as+potent%2C+specific%2C+and+reversible+LSD1+inhibitors&doi=10.1021%2Fjm400870h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27cR"><div class="casContent"><span class="casTitleNuber">27c</span><div class="casTitle"><span class="NLM_cas:atitle">High-Throughput Virtual Screening Identifies Novel N'-(1-Phenylethylidene)-benzohydrazides as Potent, Specific, and Reversible LSD1 Inhibitors</span></div><div class="casAuthors">Sorna, Venkataswamy; Theisen, Emily R.; Stephens, Bret; Warner, Steven L.; Bearss, David J.; Vankayalapati, Hariprasad; Sharma, Sunil</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">9496-9508</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Lysine specific demethylase 1 (LSD1) plays an important role in regulating histone lysine methylation at residues K4 and K9 on histone H3 and is an attractive therapeutic target in multiple malignancies.  Here we report a structure-based virtual screen of a compd. library contg. ∼2 million small mol. entities.  Computational docking and scoring followed by biochem. screening led to the identification of a novel N'-(1-phenylethylidene)-benzohydrazide series of LSD1 inhibitors with hits showing biochem. IC50s in the 200-400 nM range.  Hit-to-lead optimization and structure-activity relation studies aided in the discovery of compd. (I), with a Ki of 31 nM.  Compd. I is reversible and specific for LSD1 as compared to the monoamine oxidases shows minimal inhibition of CYPs and hERG and inhibits proliferation and survival in several cancer cell lines, including breast and colorectal cancer.  Compd. I may be used to probe LSD1's biol. role in these cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp97P3ef8ur6rVg90H21EOLACvtfcHk0lj2Fyh0FeCPyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslyks7fF&md5=47293fd790c5bc9fd944943ff4b1a58a</span></div><a href="/servlet/linkout?suffix=cit27c&amp;dbid=16384&amp;doi=10.1021%2Fjm400870h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400870h%26sid%3Dliteratum%253Aachs%26aulast%3DSorna%26aufirst%3DV.%26aulast%3DTheisen%26aufirst%3DE.%2BR.%26aulast%3DStephens%26aufirst%3DB.%26aulast%3DWarner%26aufirst%3DS.%2BL.%26aulast%3DBearss%26aufirst%3DD.%2BJ.%26aulast%3DVankayalapati%26aufirst%3DH.%26aulast%3DSharma%26aufirst%3DS.%26atitle%3DHigh-throughput%2520virtual%2520screening%2520identifies%2520novel%2520N%25E2%2580%25B2-%25281-phenylethylidene%2529-benzohydrazides%2520as%2520potent%252C%2520specific%252C%2520and%2520reversible%2520LSD1%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D9496%26epage%3D9508%26doi%3D10.1021%2Fjm400870h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit27d"><span><span class="NLM_label">(d) </span><span class="NLM_contrib-group">Prusevich, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kalin, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ming, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Basso, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Givens, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cieniewicz, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hsiao, P. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roberson, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adejola, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Avery, L. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casero, R. A.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taverna, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qian, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tackett, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ratan, R. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDonald, O. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feinberg, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cole, P. A.</span><span> </span><span class="NLM_article-title">A Selective Phenelzine Analogue Inhibitor of Histone Demethylase LSD1</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">1284</span><span class="NLM_x">–</span> <span class="NLM_lpage">1293</span><span class="refDoi"> DOI: 10.1021/cb500018s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb500018s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27d&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BC2cXls1Crtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=1284-1293&author=P.+Prusevichauthor=J.+H.+Kalinauthor=S.+A.+Mingauthor=M.+Bassoauthor=J.+Givensauthor=X.+Liauthor=J.+Huauthor=M.+S.+Taylorauthor=A.+M.+Cieniewiczauthor=P.+Y.+Hsiaoauthor=R.+Huangauthor=H.+Robersonauthor=N.+Adejolaauthor=L.+B.+Averyauthor=R.+A.+Caseroauthor=S.+D.+Tavernaauthor=J.+Qianauthor=A.+J.+Tackettauthor=R.+R.+Ratanauthor=O.+G.+McDonaldauthor=A.+P.+Feinbergauthor=P.+A.+Cole&title=A+Selective+Phenelzine+Analogue+Inhibitor+of+Histone+Demethylase+LSD1&doi=10.1021%2Fcb500018s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27dR"><div class="casContent"><span class="casTitleNuber">27d</span><div class="casTitle"><span class="NLM_cas:atitle">A Selective Phenelzine Analogue Inhibitor of Histone Demethylase LSD1</span></div><div class="casAuthors">Prusevich, Polina; Kalin, Jay H.; Ming, Shonoi A.; Basso, Manuela; Givens, Jeffrey; Li, Xin; Hu, Jianfei; Taylor, Martin S.; Cieniewicz, Anne M.; Hsiao, Po-Yuan; Huang, Rong; Roberson, Heather; Adejola, Nkosi; Avery, Lindsay B.; Casero, Robert A., Jr.; Taverna, Sean D.; Qian, Jiang; Tackett, Alan J.; Ratan, Rajiv R.; McDonald, Oliver G.; Feinberg, Andrew P.; Cole, Philip A.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1284-1293</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Lysine-specific demethylase 1 (LSD1) is an epigenetic enzyme that oxidatively cleaves Me groups from monomethyl and di-Me Lys 4 of histone H3 (H3K4Me1, H3K4Me2) and can contribute to gene silencing.  This study describes the design and synthesis of analogs of a monoamine oxidase antidepressant, phenelzine, and their LSD1 inhibitory properties.  A novel phenelzine analog (bizine) contg. a phenyl-butyrylamide appendage was shown to be a potent LSD1 inhibitor in vitro and was selective vs. monoamine oxidases A/B and the LSD1 homolog, LSD2.  Bizine was found to be effective at modulating bulk histone methylation in cancer cells, and ChIP-seq expts. revealed a statistically significant overlap in the H3K4 methylation pattern of genes affected by bizine and those altered in LSD1-/- cells.  Treatment of two cancer cell lines, LNCaP and H460, with bizine conferred a redn. in proliferation rate, and bizine showed additive to synergistic effects on cell growth when used in combination with two out of five HDAC inhibitors tested.  Moreover, neurons exposed to oxidative stress were protected by the presence of bizine, suggesting potential applications in neurodegenerative disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2Z1w5uKT7qbVg90H21EOLACvtfcHk0lh_iFiVl2p6_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXls1Crtr0%253D&md5=36c9f3fb9b28579fa3cad41052cf335a</span></div><a href="/servlet/linkout?suffix=cit27d&amp;dbid=16384&amp;doi=10.1021%2Fcb500018s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb500018s%26sid%3Dliteratum%253Aachs%26aulast%3DPrusevich%26aufirst%3DP.%26aulast%3DKalin%26aufirst%3DJ.%2BH.%26aulast%3DMing%26aufirst%3DS.%2BA.%26aulast%3DBasso%26aufirst%3DM.%26aulast%3DGivens%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DTaylor%26aufirst%3DM.%2BS.%26aulast%3DCieniewicz%26aufirst%3DA.%2BM.%26aulast%3DHsiao%26aufirst%3DP.%2BY.%26aulast%3DHuang%26aufirst%3DR.%26aulast%3DRoberson%26aufirst%3DH.%26aulast%3DAdejola%26aufirst%3DN.%26aulast%3DAvery%26aufirst%3DL.%2BB.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26aulast%3DTaverna%26aufirst%3DS.%2BD.%26aulast%3DQian%26aufirst%3DJ.%26aulast%3DTackett%26aufirst%3DA.%2BJ.%26aulast%3DRatan%26aufirst%3DR.%2BR.%26aulast%3DMcDonald%26aufirst%3DO.%2BG.%26aulast%3DFeinberg%26aufirst%3DA.%2BP.%26aulast%3DCole%26aufirst%3DP.%2BA.%26atitle%3DA%2520Selective%2520Phenelzine%2520Analogue%2520Inhibitor%2520of%2520Histone%2520Demethylase%2520LSD1%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2014%26volume%3D9%26spage%3D1284%26epage%3D1293%26doi%3D10.1021%2Fcb500018s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit27e"><span><span class="NLM_label">(e) </span><span class="NLM_contrib-group">Huang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greene, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stewart, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goodwin, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baylin, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woster, P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casero, R. A.</span><span> </span><span class="NLM_article-title">Inhibition of lysine-specific demethylase 1 by polyamine analogues results in reexpression of aberrantly silenced genes</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">104</span><span class="NLM_x">, </span> <span class="NLM_fpage">8023</span><span class="NLM_x">–</span> <span class="NLM_lpage">8028</span><span class="refDoi"> DOI: 10.1073/pnas.0700720104</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27e&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1073%2Fpnas.0700720104" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27e&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=17463086" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27e&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BD2sXmtFCiurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2007&pages=8023-8028&author=Y.+Huangauthor=E.+Greeneauthor=T.+M.+Stewartauthor=A.+C.+Goodwinauthor=S.+B.+Baylinauthor=P.+M.+Wosterauthor=R.+A.+Casero&title=Inhibition+of+lysine-specific+demethylase+1+by+polyamine+analogues+results+in+reexpression+of+aberrantly+silenced+genes&doi=10.1073%2Fpnas.0700720104"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27eR"><div class="casContent"><span class="casTitleNuber">27e</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of lysine-specific demethylase 1 by polyamine analogues results in reexpression of aberrantly silenced genes</span></div><div class="casAuthors">Huang, Yi; Greene, Eriko; Stewart, Tracy Murray; Goodwin, Andrew C.; Baylin, Stephen B.; Woster, Patrick M.; Casero, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8023-8028</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Epigenetic chromatin modification is a major regulator of eukaryotic gene expression, and aberrant epigenetic silencing of gene expression contributes to tumorigenesis.  Histone modifications include acetylation, phosphorylation, and methylation, resulting in a combination of histone marks known collectively as the histone code.  The chromatin marks at a given promoter det., in part, whether specific promoters are in an open/active conformation or closed/repressed conformation.  Dimethyl-lysine 4 histone H3 (H3K4me2) is a transcription-activating chromatin mark at gene promoters, and demethylation of this mark by the lysine-specific demethylase 1 (LSD1), a homolog of polyamine oxidases, may broadly repress gene expression.  We now report that novel biguanide and bisguanidine polyamine analogs are potent inhibitors of LSD1.  These analogs inhibit LSD1 in human colon carcinoma cells and affect a reexpression of multiple, aberrantly silenced genes important in the development of colon cancer, including members of the secreted frizzle-related proteins (SFRPs) and the GATA family of transcription factors.  Furthermore, we demonstrate by chromatin immunopptn. anal. that the reexpression is concurrent with increased H3K4me2 and acetyl-H3K9 marks, decreased H3K9me1 and H3K9me2 repressive marks.  We thus define important new agents for reversing aberrant repression of gene transcription.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoC96zJoYEK67Vg90H21EOLACvtfcHk0lh_iFiVl2p6_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXmtFCiurg%253D&md5=4c7863ecb038e3e10c5774cba2c28a45</span></div><a href="/servlet/linkout?suffix=cit27e&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0700720104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0700720104%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DGreene%26aufirst%3DE.%26aulast%3DStewart%26aufirst%3DT.%2BM.%26aulast%3DGoodwin%26aufirst%3DA.%2BC.%26aulast%3DBaylin%26aufirst%3DS.%2BB.%26aulast%3DWoster%26aufirst%3DP.%2BM.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26atitle%3DInhibition%2520of%2520lysine-specific%2520demethylase%25201%2520by%2520polyamine%2520analogues%2520results%2520in%2520reexpression%2520of%2520aberrantly%2520silenced%2520genes%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2007%26volume%3D104%26spage%3D8023%26epage%3D8028%26doi%3D10.1073%2Fpnas.0700720104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit27f"><span><span class="NLM_label">(f) </span><span class="NLM_contrib-group">Huang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stewart, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baylin, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marton, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perkins, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woster, P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casero, R. A.,  Jr.</span><span> </span><span class="NLM_article-title">Novel oligoamine analogues inhibit lysine-specific demethylase 1 and induce reexpression of epigenetically silenced genes</span> <span class="citation_source-journal">Clin. Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">7217</span><span class="NLM_x">–</span> <span class="NLM_lpage">7228</span><span class="refDoi"> DOI: 10.1158/1078-0432.CCR-09-1293</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27f&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1158%2F1078-0432.CCR-09-1293" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27f&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=19934284" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27f&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFSnu7zE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2009&pages=7217-7228&author=Y.+Huangauthor=T.+M.+Stewartauthor=Y.+Wuauthor=S.+B.+Baylinauthor=L.+J.+Martonauthor=B.+Perkinsauthor=R.+J.+Jonesauthor=P.+M.+Wosterauthor=R.+A.+Casero&title=Novel+oligoamine+analogues+inhibit+lysine-specific+demethylase+1+and+induce+reexpression+of+epigenetically+silenced+genes&doi=10.1158%2F1078-0432.CCR-09-1293"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27fR"><div class="casContent"><span class="casTitleNuber">27f</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Oligoamine Analogues Inhibit Lysine-Specific Demethylase 1 and Induce Reexpression of Epigenetically Silenced Genes</span></div><div class="casAuthors">Huang, Yi; Stewart, Tracy Murray; Wu, Yu; Baylin, Stephen B.; Marton, Laurence J.; Perkins, Brandy; Jones, Richard J.; Woster, Patrick M.; Casero, Robert A., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">7217-7228</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: Abnormal DNA CpG island hypermethylation and transcriptionally repressive histone modifications are assocd. with the aberrant silencing of tumor suppressor genes.  Lysine methylation is a dynamic, enzymically controlled process.  Lysine-specific demethylase 1 (LSD1) has recently been identified as a histone lysine demethylase.  LSD1 specifically catalyzes demethylation of mono- and dimethyl-lysine 4 of histone 3 (H3K4), key pos. chromatin marks assocd. with transcriptional activation.  We hypothesized that a novel class of oligoamine analogs would effectively inhibit LSD1 and thus cause the reexpression of aberrantly silenced genes.  Exptl. Design: Human colorectal cancer cells were treated with the oligoamines and changes in mono- and dimethyl-H3K4 and other chromatin marks were monitored.  In addn., treated cells were evaluated for the reexpression of the aberrantly silenced secreted frizzled-related proteins (SFRP) Wnt signaling pathway antagonist genes.  Finally, the effects of the LSD1 inhibitors were evaluated in an in vivo xenograft model.  Results: Treatment of HCT116 human colon adenocarcinoma cells in vitro resulted in increased H3K4 methylation and reexpression of silenced SFRP genes.  This reexpression is also accompanied by a decrease in H3K9me2 repressive mark.  Importantly, cotreatment with low doses of oligoamines and a DNA methyltransferase inhibitor highly induces the reexpression of the aberrantly silenced SFRP2 gene and results in significant inhibition of the growth of established tumors in a human colon tumor model in vivo.  Conclusions: The use of LSD1-inhibiting oligoamine analogs in combination with DNA methyltransferase inhibitors represents a highly promising and novel approach for epigenetic therapy of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrk1naS7xW5GbVg90H21EOLACvtfcHk0lh_iFiVl2p6_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFSnu7zE&md5=0f86cd2d3482aa0a36adacc4f10329dd</span></div><a href="/servlet/linkout?suffix=cit27f&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-09-1293&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-09-1293%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DStewart%26aufirst%3DT.%2BM.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DBaylin%26aufirst%3DS.%2BB.%26aulast%3DMarton%26aufirst%3DL.%2BJ.%26aulast%3DPerkins%26aufirst%3DB.%26aulast%3DJones%26aufirst%3DR.%2BJ.%26aulast%3DWoster%26aufirst%3DP.%2BM.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26atitle%3DNovel%2520oligoamine%2520analogues%2520inhibit%2520lysine-specific%2520demethylase%25201%2520and%2520induce%2520reexpression%2520of%2520epigenetically%2520silenced%2520genes%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2009%26volume%3D15%26spage%3D7217%26epage%3D7228%26doi%3D10.1158%2F1078-0432.CCR-09-1293" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit27g"><span><span class="NLM_label">(g) </span><span class="NLM_contrib-group">Sharma, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steinbergs, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crowley, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hanson, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casero, R. A.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woster, P. M.</span><span> </span><span class="NLM_article-title">(Bis)urea and (Bis)thiourea Inhibitors of Lysine-Specific Demethylase 1 as Epigenetic Modulators</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">5197</span><span class="NLM_x">–</span> <span class="NLM_lpage">5212</span><span class="refDoi"> DOI: 10.1021/jm100217a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm100217a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27g&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BC3cXnvVKqsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=5197-5212&author=S.+K.+Sharmaauthor=Y.+Wuauthor=N.+Steinbergsauthor=M.+L.+Crowleyauthor=A.+S.+Hansonauthor=R.+A.+Caseroauthor=P.+M.+Woster&title=%28Bis%29urea+and+%28Bis%29thiourea+Inhibitors+of+Lysine-Specific+Demethylase+1+as+Epigenetic+Modulators&doi=10.1021%2Fjm100217a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27gR"><div class="casContent"><span class="casTitleNuber">27g</span><div class="casTitle"><span class="NLM_cas:atitle">(Bis)urea and (Bis)thiourea Inhibitors of Lysine-Specific Demethylase 1 as Epigenetic Modulators</span></div><div class="casAuthors">Sharma, Shiv K.; Wu, Yu; Steinbergs, Nora; Crowley, Michael L.; Hanson, Allison S.; Casero, Robert A., Jr.; Woster, Patrick M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">5197-5212</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The recently discovered enzyme lysine-specific demethylase 1 (LSD1) plays an important role in the epigenetic control of gene expression, and aberrant gene silencing secondary to LSD1 overexpression is thought to contribute to the development of cancer.  We recently reported a series of (bis)guanidines and (bis)biguanides that are potent inhibitors of LSD1 and induce the re-expression of aberrantly silenced tumor suppressor genes in tumor cells in vitro.  We now report a series of isosteric ureas and thioureas that are also potent inhibitors of LSD1.  These compds. induce increases in methylation at the histone 3 lysine 4 (H3K4) chromatin mark, a specific target of LSD1, in Calu-6 lung carcinoma cells.  In addn., these analogs increase cellular levels of secreted frizzle-related proteins (SFRP) 2 and 5 and transcription factor GATA4.  These compds. represent an important new series of epigenetic modulators with the potential for use as antitumor agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruUR48vA8XgrVg90H21EOLACvtfcHk0lj0SLaRR3xfAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXnvVKqsbo%253D&md5=4e19e747331276e3682f7a2d02f73b47</span></div><a href="/servlet/linkout?suffix=cit27g&amp;dbid=16384&amp;doi=10.1021%2Fjm100217a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100217a%26sid%3Dliteratum%253Aachs%26aulast%3DSharma%26aufirst%3DS.%2BK.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DSteinbergs%26aufirst%3DN.%26aulast%3DCrowley%26aufirst%3DM.%2BL.%26aulast%3DHanson%26aufirst%3DA.%2BS.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26aulast%3DWoster%26aufirst%3DP.%2BM.%26atitle%3D%2528Bis%2529urea%2520and%2520%2528Bis%2529thiourea%2520Inhibitors%2520of%2520Lysine-Specific%2520Demethylase%25201%2520as%2520Epigenetic%2520Modulators%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D5197%26epage%3D5212%26doi%3D10.1021%2Fjm100217a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit27h"><span><span class="NLM_label">(h) </span><span class="NLM_contrib-group">Kutz, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holshouser, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marrow, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woster, P. M.</span><span> </span><span class="NLM_article-title">3,5-Diamino-1,2,4-triazoles as a novel scaffold for potent, reversible LSD1 (KDM1A) inhibitors</span> <span class="citation_source-journal">MedChemComm</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">1863</span><span class="NLM_x">–</span> <span class="NLM_lpage">1870</span><span class="refDoi"> DOI: 10.1039/C4MD00283K</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27h&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1039%2FC4MD00283K" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27h&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=25580204" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27h&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVSqtL7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=1863-1870&author=C.+J.+Kutzauthor=S.+L.+Holshouserauthor=E.+A.+Marrowauthor=P.+M.+Woster&title=3%2C5-Diamino-1%2C2%2C4-triazoles+as+a+novel+scaffold+for+potent%2C+reversible+LSD1+%28KDM1A%29+inhibitors&doi=10.1039%2FC4MD00283K"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27hR"><div class="casContent"><span class="casTitleNuber">27h</span><div class="casTitle"><span class="NLM_cas:atitle">3,5-Diamino-1,2,4-triazoles as a novel scaffold for potent, reversible LSD1 (KDM1A) inhibitors</span></div><div class="casAuthors">Kutz, Craig J.; Holshouser, Steven L.; Marrow, Ethan A.; Woster, Patrick M.</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1863-1870</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The chromatin remodeling amine oxidase lysine-specific demethylase 1 (LSD1) has become an attractive target for the design of specific inhibitors with therapeutic potential.  We, and others, have described LSD1 inhibitors that have potential as antitumor agents.  Many of the currently known LSD1 inhibitors are poor drug candidates, or are structurally based on the tranylcypromine backbone, thus increasing the potential for off-target effects mediated by other amine oxidases.  We now describe a series of potent LSD1 inhibitors based on a novel 1,2,4-triazole scaffold; these inhibitors show a high degree of specificity for LSD1 in vitro, and cause increases in cellular histone 3 dimethyllysine 4 (H3K4me2), a gene transcription activating mark.  Importantly, these inhibitors exhibit low toxicity towards mammalian cells in vitro, and thus they may show utility in the treatment of epigenetically-based diseases where cell death is not a desired endpoint.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXpwx29ujiWbVg90H21EOLACvtfcHk0lj0SLaRR3xfAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVSqtL7N&md5=3a6314468bdb665cb30ae86505e5eb15</span></div><a href="/servlet/linkout?suffix=cit27h&amp;dbid=16384&amp;doi=10.1039%2FC4MD00283K&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC4MD00283K%26sid%3Dliteratum%253Aachs%26aulast%3DKutz%26aufirst%3DC.%2BJ.%26aulast%3DHolshouser%26aufirst%3DS.%2BL.%26aulast%3DMarrow%26aufirst%3DE.%2BA.%26aulast%3DWoster%26aufirst%3DP.%2BM.%26atitle%3D3%252C5-Diamino-1%252C2%252C4-triazoles%2520as%2520a%2520novel%2520scaffold%2520for%2520potent%252C%2520reversible%2520LSD1%2520%2528KDM1A%2529%2520inhibitors%26jtitle%3DMedChemComm%26date%3D2014%26volume%3D5%26spage%3D1863%26epage%3D1870%26doi%3D10.1039%2FC4MD00283K" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit27i"><span><span class="NLM_label">(i) </span><span class="NLM_contrib-group">Hitchin, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blagg, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burke, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burns, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cockerill, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fairweather, E. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hutton, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jordan, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McAndrew, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mirza, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mould, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomson, G. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waddell, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ogilvie, D. J.</span><span> </span><span class="NLM_article-title">Development and evaluation of selective, reversible LSD1 inhibitors derived from fragments</span> <span class="citation_source-journal">MedChemComm</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">1513</span><span class="NLM_x">–</span> <span class="NLM_lpage">1522</span><span class="refDoi"> DOI: 10.1039/c3md00226h</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27i&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1039%2Fc3md00226h" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27i&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1OrsrvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=1513-1522&author=J.+R.+Hitchinauthor=J.+Blaggauthor=R.+Burkeauthor=S.+Burnsauthor=M.+J.+Cockerillauthor=E.+E.+Fairweatherauthor=C.+Huttonauthor=A.+M.+Jordanauthor=C.+McAndrewauthor=A.+Mirzaauthor=D.+Mouldauthor=G.+T.+Thomsonauthor=I.+Waddellauthor=D.+J.+Ogilvie&title=Development+and+evaluation+of+selective%2C+reversible+LSD1+inhibitors+derived+from+fragments&doi=10.1039%2Fc3md00226h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27iR"><div class="casContent"><span class="casTitleNuber">27i</span><div class="casTitle"><span class="NLM_cas:atitle">Development and evaluation of selective, reversible LSD1 inhibitors derived from fragments</span></div><div class="casAuthors">Hitchin, James R.; Blagg, Julian; Burke, Rosemary; Burns, Samantha; Cockerill, Mark J.; Fairweather, Emma E.; Hutton, Colin; Jordan, Allan M.; McAndrew, Craig; Mirza, Amin; Mould, Daniel; Thomson, Graeme J.; Waddell, Ian; Ogilvie, Donald J.</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1513-1522</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Two series of aminothiazoles have been developed as reversible inhibitors of lysine specific demethylase 1 (LSD1) through the expansion of a hit derived from a high concn. biochem. fragment based screen of 2466 compds.  The potency of the initial fragment hit was increased 32-fold through synthesis, with one series of compds. showing clear structure-activity relationships and inhibitory activities in the range of 7 to 187 μM in a biochem. assay.  This series also showed selectivity against the related FAD-dependent enzyme mono-amine oxidase A (MAO-A).  Although a wide range of irreversible inhibitors of LSD1 have been reported with activities in the low nanomolar range, this work represents one of the first reported examples of a reversible small mol. inhibitor of LSD1 with clear SAR and selectivity against MAO-A, and could provide a platform for the development of more potent reversible inhibitors.  Herein, we also report the use of a recently developed cell-based assay for profiling LSD1 inhibitors, and present results on our own compds. as well as a selection of recently described reversible LSD1 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUgrL8DOIuDbVg90H21EOLACvtfcHk0lj0SLaRR3xfAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1OrsrvK&md5=bec441bbb94f970eab94b96026db7fbe</span></div><a href="/servlet/linkout?suffix=cit27i&amp;dbid=16384&amp;doi=10.1039%2Fc3md00226h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc3md00226h%26sid%3Dliteratum%253Aachs%26aulast%3DHitchin%26aufirst%3DJ.%2BR.%26aulast%3DBlagg%26aufirst%3DJ.%26aulast%3DBurke%26aufirst%3DR.%26aulast%3DBurns%26aufirst%3DS.%26aulast%3DCockerill%26aufirst%3DM.%2BJ.%26aulast%3DFairweather%26aufirst%3DE.%2BE.%26aulast%3DHutton%26aufirst%3DC.%26aulast%3DJordan%26aufirst%3DA.%2BM.%26aulast%3DMcAndrew%26aufirst%3DC.%26aulast%3DMirza%26aufirst%3DA.%26aulast%3DMould%26aufirst%3DD.%26aulast%3DThomson%26aufirst%3DG.%2BT.%26aulast%3DWaddell%26aufirst%3DI.%26aulast%3DOgilvie%26aufirst%3DD.%2BJ.%26atitle%3DDevelopment%2520and%2520evaluation%2520of%2520selective%252C%2520reversible%2520LSD1%2520inhibitors%2520derived%2520from%2520fragments%26jtitle%3DMedChemComm%26date%3D2013%26volume%3D4%26spage%3D1513%26epage%3D1522%26doi%3D10.1039%2Fc3md00226h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit27j"><span><span class="NLM_label">(j) </span><span class="NLM_contrib-group">Culhane, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yen, P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cole, P. A.</span><span> </span><span class="NLM_article-title">Comparative analysis of small molecules and histone substrate analogues as LSD1 lysine demethylase inhibitors</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">132</span><span class="NLM_x">, </span> <span class="NLM_fpage">3164</span><span class="NLM_x">–</span> <span class="NLM_lpage">3176</span><span class="refDoi"> DOI: 10.1021/ja909996p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja909996p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2010&pages=3164-3176&author=J.+C.+Culhaneauthor=D.+Wangauthor=P.+M.+Yenauthor=P.+A.+Cole&title=Comparative+analysis+of+small+molecules+and+histone+substrate+analogues+as+LSD1+lysine+demethylase+inhibitors&doi=10.1021%2Fja909996p"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27j&amp;dbid=16384&amp;doi=10.1021%2Fja909996p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja909996p%26sid%3Dliteratum%253Aachs%26aulast%3DCulhane%26aufirst%3DJ.%2BC.%26aulast%3DWang%26aufirst%3DD.%26aulast%3DYen%26aufirst%3DP.%2BM.%26aulast%3DCole%26aufirst%3DP.%2BA.%26atitle%3DComparative%2520analysis%2520of%2520small%2520molecules%2520and%2520histone%2520substrate%2520analogues%2520as%2520LSD1%2520lysine%2520demethylase%2520inhibitors%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2010%26volume%3D132%26spage%3D3164%26epage%3D3176%26doi%3D10.1021%2Fja909996p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Kahl, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gullotti, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heukamp, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolf, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friedrichs, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vorreuther, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Solleder, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bastian, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ellinger, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metzger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schule, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buettner, R.</span><span> </span><span class="NLM_article-title">Androgen receptor coactivators lysine-specific histone demethylase 1 and four and a half LIM domain protein 2 predict risk of prostate cancer recurrence</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">11341</span><span class="NLM_x">–</span> <span class="NLM_lpage">11347</span><span class="refDoi"> DOI: 10.1158/0008-5472.CAN-06-1570</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1158%2F0008-5472.CAN-06-1570" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2006&pages=11341-11347&author=P.+Kahlauthor=L.+Gullottiauthor=L.+C.+Heukampauthor=S.+Wolfauthor=N.+Friedrichsauthor=R.+Vorreutherauthor=G.+Sollederauthor=P.+J.+Bastianauthor=J.+Ellingerauthor=E.+Metzgerauthor=R.+Schuleauthor=R.+Buettner&title=Androgen+receptor+coactivators+lysine-specific+histone+demethylase+1+and+four+and+a+half+LIM+domain+protein+2+predict+risk+of+prostate+cancer+recurrence&doi=10.1158%2F0008-5472.CAN-06-1570"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-06-1570&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-06-1570%26sid%3Dliteratum%253Aachs%26aulast%3DKahl%26aufirst%3DP.%26aulast%3DGullotti%26aufirst%3DL.%26aulast%3DHeukamp%26aufirst%3DL.%2BC.%26aulast%3DWolf%26aufirst%3DS.%26aulast%3DFriedrichs%26aufirst%3DN.%26aulast%3DVorreuther%26aufirst%3DR.%26aulast%3DSolleder%26aufirst%3DG.%26aulast%3DBastian%26aufirst%3DP.%2BJ.%26aulast%3DEllinger%26aufirst%3DJ.%26aulast%3DMetzger%26aufirst%3DE.%26aulast%3DSchule%26aufirst%3DR.%26aulast%3DBuettner%26aufirst%3DR.%26atitle%3DAndrogen%2520receptor%2520coactivators%2520lysine-specific%2520histone%2520demethylase%25201%2520and%2520four%2520and%2520a%2520half%2520LIM%2520domain%2520protein%25202%2520predict%2520risk%2520of%2520prostate%2520cancer%2520recurrence%26jtitle%3DCancer%2520Res.%26date%3D2006%26volume%3D66%26spage%3D11341%26epage%3D11347%26doi%3D10.1158%2F0008-5472.CAN-06-1570" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Schulte, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lim, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schramm, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friedrichs, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koster, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Versteeg, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ora, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pajtler, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klein-Hitpass, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhfittig-Kulle, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metzger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schule, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eggert, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buettner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kirfel, J.</span><span> </span><span class="NLM_article-title">Lysine-specific demethylase 1 is strongly expressed in poorly differentiated neuroblastoma: implications for therapy</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">69</span><span class="NLM_x">, </span> <span class="NLM_fpage">2065</span><span class="NLM_x">–</span> <span class="NLM_lpage">2071</span><span class="refDoi"> DOI: 10.1158/0008-5472.CAN-08-1735</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1158%2F0008-5472.CAN-08-1735" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=19223552" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BD1MXisFSruro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2009&pages=2065-2071&author=J.+H.+Schulteauthor=S.+Limauthor=A.+Schrammauthor=N.+Friedrichsauthor=J.+Kosterauthor=R.+Versteegauthor=I.+Oraauthor=K.+Pajtlerauthor=L.+Klein-Hitpassauthor=S.+Kuhfittig-Kulleauthor=E.+Metzgerauthor=R.+Schuleauthor=A.+Eggertauthor=R.+Buettnerauthor=J.+Kirfel&title=Lysine-specific+demethylase+1+is+strongly+expressed+in+poorly+differentiated+neuroblastoma%3A+implications+for+therapy&doi=10.1158%2F0008-5472.CAN-08-1735"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine-specific demethylase 1 is strongly expressed in poorly differentiated neuroblastoma: Implications for therapy</span></div><div class="casAuthors">Schulte, Johannes H.; Lim, Soyoung; Schramm, Alexander; Friedrichs, Nicolaus; Koster, Jan; Versteeg, Rogier; Ora, Ingrid; Pajtler, Kristian; Klein-Hitpass, Ludger; Kuhfittig-Kulle, Steffi; Metzger, Eric; Schuele, Roland; Eggert, Angelika; Buettner, Reinhard; Kirfel, Jutta</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2065-2071</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Aberrant epigenetic changes in DNA methylation and histone acetylation are hallmarks of most cancers, whereas histone methylation was previously considered to be irreversible and less versatile.  Recently, several histone demethylases were identified catalyzing the removal of Me groups from histone H3 lysine residues and thereby influencing gene expression.  Neuroblastomas continue to remain a clin. challenge despite advances in multimodal therapy.  Here, we address the functional significance of the chromatin-modifying enzyme lysine-specific demethylase 1 (LSD1) in neuroblastoma.  LSD1 expression correlated with adverse outcome and was inversely correlated with differentiation in neuroblastic tumors.  Differentiation of neuroblastoma cells resulted in down-regulation of LSD1.  Small interfering RNA-mediated knockdown of LSD1 decreased cellular growth, induced expression of differentiation-assocd. genes, and increased target gene-specific H3K4 methylation.  Moreover, LSD1 inhibition using monoamine oxidase inhibitors resulted in an increase of global H3K4 methylation and growth inhibition of neuroblastoma cells in vitro.  Finally, targeting LSD1 reduced neuroblastoma xenograft growth in vivo.  Here, we provide the first evidence that a histone demethylase, LSD1, is involved in maintaining the undifferentiated, malignant phenotype of neuroblastoma cells.  We show that inhibition of LSD1 reprograms the transcriptome of neuroblastoma cells and inhibits neuroblastoma xenograft growth.  Our results suggest that targeting histone demethylases may provide a novel option for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphjmZreB7Q67Vg90H21EOLACvtfcHk0lgIDN80G2UsGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXisFSruro%253D&md5=4118d7f651c8abf8c32e296cc0030656</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-08-1735&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-08-1735%26sid%3Dliteratum%253Aachs%26aulast%3DSchulte%26aufirst%3DJ.%2BH.%26aulast%3DLim%26aufirst%3DS.%26aulast%3DSchramm%26aufirst%3DA.%26aulast%3DFriedrichs%26aufirst%3DN.%26aulast%3DKoster%26aufirst%3DJ.%26aulast%3DVersteeg%26aufirst%3DR.%26aulast%3DOra%26aufirst%3DI.%26aulast%3DPajtler%26aufirst%3DK.%26aulast%3DKlein-Hitpass%26aufirst%3DL.%26aulast%3DKuhfittig-Kulle%26aufirst%3DS.%26aulast%3DMetzger%26aufirst%3DE.%26aulast%3DSchule%26aufirst%3DR.%26aulast%3DEggert%26aufirst%3DA.%26aulast%3DBuettner%26aufirst%3DR.%26aulast%3DKirfel%26aufirst%3DJ.%26atitle%3DLysine-specific%2520demethylase%25201%2520is%2520strongly%2520expressed%2520in%2520poorly%2520differentiated%2520neuroblastoma%253A%2520implications%2520for%2520therapy%26jtitle%3DCancer%2520Res.%26date%3D2009%26volume%3D69%26spage%3D2065%26epage%3D2071%26doi%3D10.1158%2F0008-5472.CAN-08-1735" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Lim, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janzer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Becker, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zimmer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schule, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buettner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kirfel, J.</span><span> </span><span class="NLM_article-title">Lysine-specific demethylase 1 (LSD1) is highly expressed in ER-negative breast cancers and a biomarker predicting aggressive biology</span> <span class="citation_source-journal">Carcinogenesis</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">512</span><span class="NLM_x">–</span> <span class="NLM_lpage">520</span><span class="refDoi"> DOI: 10.1093/carcin/bgp324</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1093%2Fcarcin%2Fbgp324" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=20042638" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BC3cXivFGmsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2010&pages=512-520&author=S.+Limauthor=A.+Janzerauthor=A.+Beckerauthor=A.+Zimmerauthor=R.+Schuleauthor=R.+Buettnerauthor=J.+Kirfel&title=Lysine-specific+demethylase+1+%28LSD1%29+is+highly+expressed+in+ER-negative+breast+cancers+and+a+biomarker+predicting+aggressive+biology&doi=10.1093%2Fcarcin%2Fbgp324"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine-specific demethylase 1 (LSD1) is highly expressed in ER-negative breast cancers and a biomarker predicting aggressive biology</span></div><div class="casAuthors">Lim, Soyoung; Janzer, Andreas; Becker, Astrid; Zimmer, Andreas; Schuele, Roland; Buettner, Reinhard; Kirfel, Jutta</div><div class="citationInfo"><span class="NLM_cas:title">Carcinogenesis</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">512-520</span>CODEN:
                <span class="NLM_cas:coden">CRNGDP</span>;
        ISSN:<span class="NLM_cas:issn">0143-3334</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Breast carcinogenesis is a multistep process involving both genetic and epigenetic changes.  Since epigenetic changes like histone modifications are potentially reversible processes, much effort has been directed toward understanding this mechanism with the goal of finding novel therapies as well as more refined diagnostic and prognostic tools in breast cancer.  Lysine-specific demethylase 1 (LSD1) plays a key role in the regulation of gene expression by removing the Me groups from methylated lysine 4 of histone H3 and lysine 9 of histone H3.  LSD1 is essential for mammalian development and involved in many biol. processes.  Considering recent evidence that LSD1 is involved in carcinogenesis, we investigated the role of LSD1 in breast cancer.  Therefore, we developed an ELISA to det. LSD1 protein levels in tissue specimens of breast cancer and measured very high LSD1 levels in estrogen receptor (ER)-neg. tumors.  Pharmacol. LSD1 inhibition resulted in growth inhibition of breast cancer cells.  Knockdown of LSD1 using small interfering RNA approach induced regulation of several proliferation-assocd. genes like p21, ERBB2 and CCNA2.  Addnl., we found that LSD1 is recruited to the promoters of these genes.  In summary, our data indicate that LSD1 may provide a predictive marker for aggressive biol. and a novel attractive therapeutic target for treatment of ER-neg. breast cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoeX6QvvaygFLVg90H21EOLACvtfcHk0lgIDN80G2UsGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXivFGmsrw%253D&md5=b94c60b52c6a440d71cdf5a8007ea97b</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1093%2Fcarcin%2Fbgp324&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fcarcin%252Fbgp324%26sid%3Dliteratum%253Aachs%26aulast%3DLim%26aufirst%3DS.%26aulast%3DJanzer%26aufirst%3DA.%26aulast%3DBecker%26aufirst%3DA.%26aulast%3DZimmer%26aufirst%3DA.%26aulast%3DSchule%26aufirst%3DR.%26aulast%3DBuettner%26aufirst%3DR.%26aulast%3DKirfel%26aufirst%3DJ.%26atitle%3DLysine-specific%2520demethylase%25201%2520%2528LSD1%2529%2520is%2520highly%2520expressed%2520in%2520ER-negative%2520breast%2520cancers%2520and%2520a%2520biomarker%2520predicting%2520aggressive%2520biology%26jtitle%3DCarcinogenesis%26date%3D2010%26volume%3D31%26spage%3D512%26epage%3D520%26doi%3D10.1093%2Fcarcin%2Fbgp324" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Harris, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lynch, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spencer, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hitchin, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciceri, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blaser, J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greystoke, B. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jordan, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ogilvie, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Somervaille, T. C.</span><span> </span><span class="NLM_article-title">The histone demethylase KDM1A sustains the oncogenic potential of MLL-AF9 leukemia stem cells</span> <span class="citation_source-journal">Cancer Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">473</span><span class="NLM_x">–</span> <span class="NLM_lpage">487</span><span class="refDoi"> DOI: 10.1016/j.ccr.2012.03.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1016%2Fj.ccr.2012.03.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=22464800" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BC38XkvFyqsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2012&pages=473-487&author=W.+J.+Harrisauthor=X.+Huangauthor=J.+T.+Lynchauthor=G.+J.+Spencerauthor=J.+R.+Hitchinauthor=Y.+Liauthor=F.+Ciceriauthor=J.+G.+Blaserauthor=B.+F.+Greystokeauthor=A.+M.+Jordanauthor=C.+J.+Millerauthor=D.+J.+Ogilvieauthor=T.+C.+Somervaille&title=The+histone+demethylase+KDM1A+sustains+the+oncogenic+potential+of+MLL-AF9+leukemia+stem+cells&doi=10.1016%2Fj.ccr.2012.03.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">The Histone Demethylase KDM1A Sustains the Oncogenic Potential of MLL-AF9 Leukemia Stem Cells</span></div><div class="casAuthors">Harris, William J.; Huang, Xu; Lynch, James T.; Spencer, Gary J.; Hitchin, James R.; Li, Yaoyong; Ciceri, Filippo; Blaser, Julian G.; Greystoke, Brigit F.; Jordan, Allan M.; Miller, Crispin J.; Ogilvie, Donald J.; Somervaille, Tim C. P.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">473-487</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Using a mouse model of human MLL-AF9 leukemia, we identified the lysine-specific demethylase KDM1A (LSD1 or AOF2) as an essential regulator of leukemia stem cell (LSC) potential.  KDM1A acts at genomic loci bound by MLL-AF9 to sustain expression of the assocd. oncogenic program, thus preventing differentiation and apoptosis.  In vitro and in vivo pharmacol. targeting of KDM1A using tranylcypromine analogs active in the nanomolar range phenocopied Kdm1a knockdown in both murine and primary human AML cells exhibiting MLL translocations.  By contrast, the clonogenic and repopulating potential of normal hematopoietic stem and progenitor cells was spared.  These data establish KDM1A as a key effector of the differentiation block in MLL leukemia, which may be selectively targeted to therapeutic effect.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiZ65foej58rVg90H21EOLACvtfcHk0lhDhL5GNLfKdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkvFyqsrc%253D&md5=3227f2ac24a884a8ebadbce71198830c</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2012.03.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2012.03.014%26sid%3Dliteratum%253Aachs%26aulast%3DHarris%26aufirst%3DW.%2BJ.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DLynch%26aufirst%3DJ.%2BT.%26aulast%3DSpencer%26aufirst%3DG.%2BJ.%26aulast%3DHitchin%26aufirst%3DJ.%2BR.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DCiceri%26aufirst%3DF.%26aulast%3DBlaser%26aufirst%3DJ.%2BG.%26aulast%3DGreystoke%26aufirst%3DB.%2BF.%26aulast%3DJordan%26aufirst%3DA.%2BM.%26aulast%3DMiller%26aufirst%3DC.%2BJ.%26aulast%3DOgilvie%26aufirst%3DD.%2BJ.%26aulast%3DSomervaille%26aufirst%3DT.%2BC.%26atitle%3DThe%2520histone%2520demethylase%2520KDM1A%2520sustains%2520the%2520oncogenic%2520potential%2520of%2520MLL-AF9%2520leukemia%2520stem%2520cells%26jtitle%3DCancer%2520Cell%26date%3D2012%26volume%3D21%26spage%3D473%26epage%3D487%26doi%3D10.1016%2Fj.ccr.2012.03.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Dent, S. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chandra, J.</span><span> </span><span class="NLM_article-title">The lasting influence of LSD1 in the blood</span> <span class="citation_source-journal">eLife</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">e00963</span><span class="refDoi"> DOI: 10.7554/eLife.00963</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.7554%2FeLife.00963" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2013&pages=e00963&author=S.+Y.+Dentauthor=J.+Chandra&title=The+lasting+influence+of+LSD1+in+the+blood&doi=10.7554%2FeLife.00963"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.7554%2FeLife.00963&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.00963%26sid%3Dliteratum%253Aachs%26aulast%3DDent%26aufirst%3DS.%2BY.%26aulast%3DChandra%26aufirst%3DJ.%26atitle%3DThe%2520lasting%2520influence%2520of%2520LSD1%2520in%2520the%2520blood%26jtitle%3DeLife%26date%3D2013%26volume%3D2%26spage%3De00963%26doi%3D10.7554%2FeLife.00963" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Lynch, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harris, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Somervaille, T. C.</span><span> </span><span class="NLM_article-title">LSD1 inhibition: a therapeutic strategy in cancer?</span> <span class="citation_source-journal">Expert Opin. Ther. Targets</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">1239</span><span class="NLM_x">–</span> <span class="NLM_lpage">1249</span><span class="refDoi"> DOI: 10.1517/14728222.2012.722206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1517%2F14728222.2012.722206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=22957941" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs12jtrzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2012&pages=1239-1249&author=J.+T.+Lynchauthor=W.+J.+Harrisauthor=T.+C.+Somervaille&title=LSD1+inhibition%3A+a+therapeutic+strategy+in+cancer%3F&doi=10.1517%2F14728222.2012.722206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">LSD1 inhibition: a therapeutic strategy in cancer?</span></div><div class="casAuthors">Lynch, James T.; Harris, William J.; Somervaille, Tim C. P.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Targets</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1239-1249</span>CODEN:
                <span class="NLM_cas:coden">EOTTAO</span>;
        ISSN:<span class="NLM_cas:issn">1472-8222</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Introduction: The role of epigenetic dysfunction in cancer is increasingly appreciated.  This has raised the question as to whether enzymes that regulate the structure and function of chromatin might represent novel therapeutic targets.  The histone demethylase LSD1 is one such candidate and novel, potent inhibitors are under development.Areas covered: The literature on LSD1 (also known as KDM1A, AOF2, BHC110 or KIAA0601) was identified in Pubmed and is herein discussed.  Areas covered include the structure and enzymic activity of LSD1, its role in chromatin regulatory complexes, its functional roles in normal and malignant tissue, pharmacol. inhibitors of its activity and their putative therapeutic roles.Expert opinion: Pre-clin. data supporting a therapeutic role for LSD1 inhibitors are most encouraging in acute myeloid leukemia, although optimal dosing strategies and beneficial combinations with other agents remain unclear.  Studies making use of potent, selective LSD1 inhibitors active in the nanomolar range are required to establish therapeutic indications in other subtypes of haematol. malignancy, and in solid tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2afxNLYupwrVg90H21EOLACvtfcHk0lhDhL5GNLfKdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs12jtrzI&md5=53afa8f4337b2374ec6e5a2d3fe41736</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1517%2F14728222.2012.722206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F14728222.2012.722206%26sid%3Dliteratum%253Aachs%26aulast%3DLynch%26aufirst%3DJ.%2BT.%26aulast%3DHarris%26aufirst%3DW.%2BJ.%26aulast%3DSomervaille%26aufirst%3DT.%2BC.%26atitle%3DLSD1%2520inhibition%253A%2520a%2520therapeutic%2520strategy%2520in%2520cancer%253F%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Targets%26date%3D2012%26volume%3D16%26spage%3D1239%26epage%3D1249%26doi%3D10.1517%2F14728222.2012.722206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Dhanak, D.</span><span> </span><span class="NLM_article-title">Drugging the Cancer Epigenome</span>.  <span class="citation_source-book">Proceedings of the 104th Annual Meeting of the American Association for Cancer Research</span>, Washington, DC, April 6–10, 2013; <span class="NLM_publisher-name">AACR</span>: <span class="NLM_publisher-loc">Philadelphia, PA</span>,<span class="NLM_x"> </span><span class="NLM_year">2013</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=D.+Dhanak&title=Proceedings+of+the+104th+Annual+Meeting+of+the+American+Association+for+Cancer+Research"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DDhanak%26aufirst%3DD.%26atitle%3DDrugging%2520the%2520Cancer%2520Epigenome%26btitle%3DProceedings%2520of%2520the%2520104th%2520Annual%2520Meeting%2520of%2520the%2520American%2520Association%2520for%2520Cancer%2520Research%26pub%3DAACR%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Khan, M. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyata, N.</span><span> </span><span class="NLM_article-title">An overview of phenylcyclopropylamine derivatives: biochemical and biological significance and recent developments</span> <span class="citation_source-journal">Med. Res. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x">, </span> <span class="NLM_fpage">873</span><span class="NLM_x">–</span> <span class="NLM_lpage">910</span><span class="refDoi"> DOI: 10.1002/med.21269</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1002%2Fmed.21269" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=22893613" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BC3sXptFersL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2013&pages=873-910&author=M.+N.+Khanauthor=T.+Suzukiauthor=N.+Miyata&title=An+overview+of+phenylcyclopropylamine+derivatives%3A+biochemical+and+biological+significance+and+recent+developments&doi=10.1002%2Fmed.21269"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">An Overview of Phenylcyclopropylamine Derivatives: Biochemical and Biological Significance and Recent Developments</span></div><div class="casAuthors">Khan, Mohammed Naseer Ahmed; Suzuki, Takayoshi; Miyata, Naoki</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">873-910</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  Trans-2-Phencylcyclopropylamine (2-PCPA), a potent, clin. used antidepressant, affects monoamine neurotransmitter levels by inhibiting the main metabolizing enzymes, monoamine oxidases (MAOs).  However, the antidepressant action of this compd. was not fully explained by its effects on MAOs due to its wide variety of biol. effects. 2-PCPA also affects depression-assocd. pathophysiol. pathways, and linked with increased levels of trace amines in brain, upregulation of GABAB receptors (where GABA is gamma amino butyric acid), modulation of phospholipid metab., and interference with various cytochrome P 450 (CYP) enzymes.  Consequently, despite its adverse effects and limited clin. applicability, 2-PCPA has attracted interest as a structural scaffold for the development of mechanism-based inhibitors of various enzymes, including lysine-specific demethylase 1 (LSD1), which is a possible target for cancer chemotherapy.  In the recent years, many reports have appeared in the literature based on 2-PCPA scaffold and their potential medicinal implications.  This review mainly focuses on the medicinal chem. aspects including drug design, structure-activity relationships (SAR), biol. and biochem. properties, and mechanism of actions of 2-PCPA and its derivs.  Furthermore, we also highlight recent advance in this area and discuss their future applications for beneficial therapeutic effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqIZJFRem7GlrVg90H21EOLACvtfcHk0lhDhL5GNLfKdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXptFersL0%253D&md5=d257f3b954efb33ae6f824c254ae1c30</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1002%2Fmed.21269&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21269%26sid%3Dliteratum%253Aachs%26aulast%3DKhan%26aufirst%3DM.%2BN.%26aulast%3DSuzuki%26aufirst%3DT.%26aulast%3DMiyata%26aufirst%3DN.%26atitle%3DAn%2520overview%2520of%2520phenylcyclopropylamine%2520derivatives%253A%2520biochemical%2520and%2520biological%2520significance%2520and%2520recent%2520developments%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2013%26volume%3D33%26spage%3D873%26epage%3D910%26doi%3D10.1002%2Fmed.21269" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Mimasu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Umezawa, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sato, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Higuchi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Umehara, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yokoyama, S.</span><span> </span><span class="NLM_article-title">Structurally designed trans-2-phenylcyclopropylamine derivatives potently inhibit histone demethylase LSD1/KDM1</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">6494</span><span class="NLM_x">–</span> <span class="NLM_lpage">6503</span><span class="refDoi"> DOI: 10.1021/bi100299r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi100299r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BC3cXosFCmur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2010&pages=6494-6503&author=S.+Mimasuauthor=N.+Umezawaauthor=S.+Satoauthor=T.+Higuchiauthor=T.+Umeharaauthor=S.+Yokoyama&title=Structurally+designed+trans-2-phenylcyclopropylamine+derivatives+potently+inhibit+histone+demethylase+LSD1%2FKDM1&doi=10.1021%2Fbi100299r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Structurally Designed trans-2-Phenylcyclopropylamine Derivatives Potently Inhibit Histone Demethylase LSD1/KDM1</span></div><div class="casAuthors">Mimasu, Shinya; Umezawa, Naoki; Sato, Shin; Higuchi, Tsunehiko; Umehara, Takashi; Yokoyama, Shigeyuki</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">6494-6503</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Lysine-specific demethylase 1 (LSD1/KDM1) demethylates histone H3, in addn. to tumor suppressor p53 and DNA methyltransferase 1 (Dnmt1), thus regulating eukaryotic gene expression by altering chromatin structure.  Specific inhibitors of LSD1 are desired as anticancer agents, because LSD1 aberrations are assocd. with several cancers, and LSD1 inhibition restores the expression of abnormally silenced genes in cancerous cells.  In this study, we designed and synthesized several candidate compds. to inhibit LSD1, based on the structures of LSD1 and monoamine oxidase B (MAO-B), in complex with an antidepressant tranylcypromine (2-PCPA) deriv.  Compd. S2101 exhibited stronger LSD1 inhibition than tranylcypromine and the known small LSD1 inhibitors in LSD1 demethylation assays, with a kinact/KI value of 4560 M-1 s-1.  In comparison with tranylcypromine, the compd. displayed weaker inhibition to the monoamine oxidases.  The inhibition modes of the two 2-PCPA derivs., 2-PFPA and S1201, were identified by detn. of the inhibitor-bound LSD1 structures, which revealed the enhanced stability of the inhibitor-FAD adducts by their interactions with the surrounding LSD1 residues.  These mols. are potential pharmaceutical candidates for cancer or latent virus infection, as well as research tools for LSD1-related biol. investigations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqIFRdqSvmn97Vg90H21EOLACvtfcHk0lhGaDGDfywAHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXosFCmur8%253D&md5=26732b0cd1928d44263f8bb8790ace54</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fbi100299r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi100299r%26sid%3Dliteratum%253Aachs%26aulast%3DMimasu%26aufirst%3DS.%26aulast%3DUmezawa%26aufirst%3DN.%26aulast%3DSato%26aufirst%3DS.%26aulast%3DHiguchi%26aufirst%3DT.%26aulast%3DUmehara%26aufirst%3DT.%26aulast%3DYokoyama%26aufirst%3DS.%26atitle%3DStructurally%2520designed%2520trans-2-phenylcyclopropylamine%2520derivatives%2520potently%2520inhibit%2520histone%2520demethylase%2520LSD1%252FKDM1%26jtitle%3DBiochemistry%26date%3D2010%26volume%3D49%26spage%3D6494%26epage%3D6503%26doi%3D10.1021%2Fbi100299r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Tortorici, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borrello, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tardugno, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chiarelli, L. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pilotto, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciossani, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vellore, N. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bailey, S. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Connell, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crabb, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Packham, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mai, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baron, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ganesan, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattevi, A.</span><span> </span><span class="NLM_article-title">Protein recognition by short peptide reversible inhibitors of the chromatin-modifying LSD1/CoREST lysine demethylase</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">1677</span><span class="NLM_x">–</span> <span class="NLM_lpage">1682</span><span class="refDoi"> DOI: 10.1021/cb4001926</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb4001926" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BC3sXosVCmt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=1677-1682&author=M.+Tortoriciauthor=M.+T.+Borrelloauthor=M.+Tardugnoauthor=L.+R.+Chiarelliauthor=S.+Pilottoauthor=G.+Ciossaniauthor=N.+A.+Velloreauthor=S.+G.+Baileyauthor=J.+Cowanauthor=M.+O%E2%80%99Connellauthor=S.+J.+Crabbauthor=G.+Packhamauthor=A.+Maiauthor=R.+Baronauthor=A.+Ganesanauthor=A.+Mattevi&title=Protein+recognition+by+short+peptide+reversible+inhibitors+of+the+chromatin-modifying+LSD1%2FCoREST+lysine+demethylase&doi=10.1021%2Fcb4001926"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Protein recognition by short peptide reversible inhibitors of the chromatin-modifying LSD1/CoREST lysine demethylase</span></div><div class="casAuthors">Tortorici, Marcello; Borrello, Maria Teresa; Tardugno, Maria; Chiarelli, Laurent R.; Pilotto, Simona; Ciossani, Giuseppe; Vellore, Nadeem A.; Bailey, Sarah G.; Cowan, Jonathan; O'Connell, Maria; Crabb, Simon J.; Packham, Graham; Mai, Antonello; Baron, Riccardo; Ganesan, A.; Mattevi, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1677-1682</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The combinatorial assembly of protein complexes is at the heart of chromatin biol.  Lysine demethylase LSD1(KDM1A)/CoREST beautifully exemplifies this concept.  The active site of the enzyme tightly assocs. to the N-terminal domain of transcription factors of the SNAIL1 family, which therefore can competitively inhibit the binding of the N-terminal tail of the histone substrate.  Our enzymic, crystallog., spectroscopic, and computational studies reveal that LSD1/CoREST can bind to a hexapeptide derived from the SNAIL sequence through recognition of a pos. charged α-helical turn that forms upon binding to the enzyme.  Variations in sequence and length of this six amino acid ligand modulate affinities enabling the same binding site to differentially interact with proteins that exert distinct biol. functions.  The discovered short peptide inhibitors exhibit antiproliferative activities and lay the foundation for the development of peptidomimetic small mol. inhibitors of LSD1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7SufQOwK-y7Vg90H21EOLACvtfcHk0lhGaDGDfywAHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXosVCmt7g%253D&md5=a8479b513e392e8acb78979e116dcffa</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fcb4001926&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb4001926%26sid%3Dliteratum%253Aachs%26aulast%3DTortorici%26aufirst%3DM.%26aulast%3DBorrello%26aufirst%3DM.%2BT.%26aulast%3DTardugno%26aufirst%3DM.%26aulast%3DChiarelli%26aufirst%3DL.%2BR.%26aulast%3DPilotto%26aufirst%3DS.%26aulast%3DCiossani%26aufirst%3DG.%26aulast%3DVellore%26aufirst%3DN.%2BA.%26aulast%3DBailey%26aufirst%3DS.%2BG.%26aulast%3DCowan%26aufirst%3DJ.%26aulast%3DO%25E2%2580%2599Connell%26aufirst%3DM.%26aulast%3DCrabb%26aufirst%3DS.%2BJ.%26aulast%3DPackham%26aufirst%3DG.%26aulast%3DMai%26aufirst%3DA.%26aulast%3DBaron%26aufirst%3DR.%26aulast%3DGanesan%26aufirst%3DA.%26aulast%3DMattevi%26aufirst%3DA.%26atitle%3DProtein%2520recognition%2520by%2520short%2520peptide%2520reversible%2520inhibitors%2520of%2520the%2520chromatin-modifying%2520LSD1%252FCoREST%2520lysine%2520demethylase%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2013%26volume%3D8%26spage%3D1677%26epage%3D1682%26doi%3D10.1021%2Fcb4001926" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Stavropoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blobel, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoelz, A.</span><span> </span><span class="NLM_article-title">Crystal structure and mechanism of human lysine-specific demethylase-1</span> <span class="citation_source-journal">Nat. Struct. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">626</span><span class="NLM_x">–</span> <span class="NLM_lpage">632</span><span class="refDoi"> DOI: 10.1038/nsmb1113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1038%2Fnsmb1113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=16799558" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BD28Xms1eltLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2006&pages=626-632&author=P.+Stavropoulosauthor=G.+Blobelauthor=A.+Hoelz&title=Crystal+structure+and+mechanism+of+human+lysine-specific+demethylase-1&doi=10.1038%2Fnsmb1113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure and mechanism of human lysine-specific demethylase-1</span></div><div class="casAuthors">Stavropoulos, Pete; Blobel, Guenter; Hoelz, Andre</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">626-632</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The reversible methylation of specific Lys residues in histone tails is crucial in epigenetic gene regulation.  LSD1, the 1st known lysine-specific demethylase, selectively removes mono-Me and di-Me, but not tri-Me modifications of Lys-4 or Lys-9 in histone-3.  Here, the authors present the crystal structure of LSD1 at 2.9 Å resoln.  LSD1 was found to form a highly asym., closely packed domain structure from which a long helical 'tower' domain protruded.  The active site cavity was spacious enough to accommodate several residues of the histone tail substrate, but did not appear capable of recognizing the different methylation states of the substrate, lysine.  This supported the hypothesis that trimethylated lysine was chem. rather than sterically discriminated.  The authors presented a biochem. anal. of LSD1 mutants that identified crucial residues in the active site cavity and showed the importance of the SWIRM and tower domains for catalysis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqG8Vp2AlsWQLVg90H21EOLACvtfcHk0lhGaDGDfywAHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xms1eltLs%253D&md5=34cbc6f0936b642a8668d1d76abd1e0e</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1038%2Fnsmb1113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb1113%26sid%3Dliteratum%253Aachs%26aulast%3DStavropoulos%26aufirst%3DP.%26aulast%3DBlobel%26aufirst%3DG.%26aulast%3DHoelz%26aufirst%3DA.%26atitle%3DCrystal%2520structure%2520and%2520mechanism%2520of%2520human%2520lysine-specific%2520demethylase-1%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2006%26volume%3D13%26spage%3D626%26epage%3D632%26doi%3D10.1038%2Fnsmb1113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Forneris, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Binda, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adamo, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Battaglioli, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattevi, A.</span><span> </span><span class="NLM_article-title">Structural basis of LSD1-CoREST selectivity in histone H3 recognition</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">282</span><span class="NLM_x">, </span> <span class="NLM_fpage">20070</span><span class="NLM_x">–</span> <span class="NLM_lpage">20074</span><span class="refDoi"> DOI: 10.1074/jbc.C700100200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1074%2Fjbc.C700100200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=17537733" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnsFWntr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=282&publication_year=2007&pages=20070-20074&author=F.+Fornerisauthor=C.+Bindaauthor=A.+Adamoauthor=E.+Battaglioliauthor=A.+Mattevi&title=Structural+basis+of+LSD1-CoREST+selectivity+in+histone+H3+recognition&doi=10.1074%2Fjbc.C700100200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis of LSD1-CoREST Selectivity in Histone H3 Recognition</span></div><div class="casAuthors">Forneris, Federico; Binda, Claudia; Adamo, Antonio; Battaglioli, Elena; Mattevi, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">282</span>
        (<span class="NLM_cas:issue">28</span>),
    <span class="NLM_cas:pages">20070-20074</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Histone demethylase LSD1 regulates transcription by demethylating Lys4 of histone H3.  The crystal structure of the enzyme in complex with CoREST and a substrate-like peptide inhibitor highlights an intricate network of interactions and a folded conformation of the bound peptide.  The core of the peptide structure is formed by Arg2, Gln5, and Ser10, which are engaged in specific intramol. H-bonds.  Several charged side chains on the surface of the substrate-binding pocket establish electrostatic interactions with the peptide.  The three-dimensional structure predicts that methylated Lys4 binds in a solvent inaccessible position in front of the flavin cofactor.  This geometry is fully consistent with the demethylation reaction being catalyzed through a flavin-mediated oxidn. of the substrate amino-Me group.  These features dictate the exquisite substrate specificity of LSD1 and provide a structural framework to explain the fine tuning of its catalytic activity and the active role of CoREST in substrate recognition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpD8PHQpzggArVg90H21EOLACvtfcHk0lgKc6FU3JLTEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnsFWntr0%253D&md5=a8af3e0892441bf3fc64ce80c19d0c3b</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1074%2Fjbc.C700100200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.C700100200%26sid%3Dliteratum%253Aachs%26aulast%3DForneris%26aufirst%3DF.%26aulast%3DBinda%26aufirst%3DC.%26aulast%3DAdamo%26aufirst%3DA.%26aulast%3DBattaglioli%26aufirst%3DE.%26aulast%3DMattevi%26aufirst%3DA.%26atitle%3DStructural%2520basis%2520of%2520LSD1-CoREST%2520selectivity%2520in%2520histone%2520H3%2520recognition%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2007%26volume%3D282%26spage%3D20070%26epage%3D20074%26doi%3D10.1074%2Fjbc.C700100200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span>Schrödinger Suite, version <span class="NLM_edition">2015</span>; <span class="NLM_publisher-name">Schrödinger, LLC</span>: <span class="NLM_publisher-loc">New York, NY</span>,<span class="NLM_x"> </span><span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Schr%C3%B6dinger+Suite%2C+version+2015%3B+Schr%C3%B6dinger%2C+LLC%3A+New+York%2C+NY%2C+2015."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DSchr%25C3%25B6dinger%2520Suite%26pub%3DSchr%25C3%25B6dinger%252C%2520LLC%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">Lin, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamashita, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nidarmarthy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luengo, J. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rhodes, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knick, V. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choudhry, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lai, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Minthorn, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Strum, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wood, E. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elkins, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Concha, N. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heerding, D. A.</span><span> </span><span class="NLM_article-title">2,3,5-Trisubstituted pyridines as selective AKT inhibitors-Part I: Substitution at 2-position of the core pyridine for ROCK1 selectivity</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">673</span><span class="NLM_x">–</span> <span class="NLM_lpage">678</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2009.11.064</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1016%2Fj.bmcl.2009.11.064" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2010&pages=673-678&author=H.+Linauthor=D.+S.+Yamashitaauthor=J.+Zengauthor=R.+Xieauthor=W.+Wangauthor=S.+Nidarmarthyauthor=J.+I.+Luengoauthor=N.+Rhodesauthor=V.+B.+Knickauthor=A.+E.+Choudhryauthor=Z.+Laiauthor=E.+A.+Minthornauthor=S.+L.+Strumauthor=E.+R.+Woodauthor=P.+A.+Elkinsauthor=N.+O.+Conchaauthor=D.+A.+Heerding&title=2%2C3%2C5-Trisubstituted+pyridines+as+selective+AKT+inhibitors-Part+I%3A+Substitution+at+2-position+of+the+core+pyridine+for+ROCK1+selectivity&doi=10.1016%2Fj.bmcl.2009.11.064"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2009.11.064&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2009.11.064%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DH.%26aulast%3DYamashita%26aufirst%3DD.%2BS.%26aulast%3DZeng%26aufirst%3DJ.%26aulast%3DXie%26aufirst%3DR.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DNidarmarthy%26aufirst%3DS.%26aulast%3DLuengo%26aufirst%3DJ.%2BI.%26aulast%3DRhodes%26aufirst%3DN.%26aulast%3DKnick%26aufirst%3DV.%2BB.%26aulast%3DChoudhry%26aufirst%3DA.%2BE.%26aulast%3DLai%26aufirst%3DZ.%26aulast%3DMinthorn%26aufirst%3DE.%2BA.%26aulast%3DStrum%26aufirst%3DS.%2BL.%26aulast%3DWood%26aufirst%3DE.%2BR.%26aulast%3DElkins%26aufirst%3DP.%2BA.%26aulast%3DConcha%26aufirst%3DN.%2BO.%26aulast%3DHeerding%26aufirst%3DD.%2BA.%26atitle%3D2%252C3%252C5-Trisubstituted%2520pyridines%2520as%2520selective%2520AKT%2520inhibitors-Part%2520I%253A%2520Substitution%2520at%25202-position%2520of%2520the%2520core%2520pyridine%2520for%2520ROCK1%2520selectivity%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2010%26volume%3D20%26spage%3D673%26epage%3D678%26doi%3D10.1016%2Fj.bmcl.2009.11.064" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group">Lin, N.-H.; He, Y.; Holladay, M. W.; Ryther, K.; Li, Y.</span><span> </span><span class="NLM_article-title">3-Pyridyloxymethyl heterocyclic ether compounds useful in controlling chemical synaptic transmission</span>. U.S. Patent 5,629,325,<span class="NLM_x"> </span><span class="NLM_year">1997</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1997&author=N.-H.+Lin&author=Y.+He&author=M.+W.+Holladay&author=K.+Ryther&author=Y.+Li&title=3-Pyridyloxymethyl+heterocyclic+ether+compounds+useful+in+controlling+chemical+synaptic+transmission"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DN.-H.%26atitle%3D3-Pyridyloxymethyl%2520heterocyclic%2520ether%2520compounds%2520useful%2520in%2520controlling%2520chemical%2520synaptic%2520transmission%26date%3D1997" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group">Ballell, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bates, R. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alvarez-Gomez, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alvarez-Ruiz, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barroso, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blanco, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crespo, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Escribano, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">González, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lozano, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huss, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santos-Villarejo, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martín-Plaza, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mendoza, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rebollo-Lopez, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Remuiñan-Blanco, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lavandera, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pérez-Herran, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gamo-Benito, F. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">García-Bustos, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barros, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Castro, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cammack, N.</span><span> </span><span class="NLM_article-title">Fueling open-source drug discovery: 177 small-molecule leads against tuberculosis</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">313</span><span class="NLM_x">–</span> <span class="NLM_lpage">321</span><span class="refDoi"> DOI: 10.1002/cmdc.201200428</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.1002%2Fcmdc.201200428" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=23307663" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BC3sXls1Omug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=313-321&author=L.+Ballellauthor=R.+H.+Batesauthor=R.+J.+Youngauthor=D.+Alvarez-Gomezauthor=E.+Alvarez-Ruizauthor=V.+Barrosoauthor=D.+Blancoauthor=B.+Crespoauthor=J.+Escribanoauthor=R.+Gonz%C3%A1lezauthor=S.+Lozanoauthor=S.+Hussauthor=A.+Santos-Villarejoauthor=J.+J.+Mart%C3%ADn-Plazaauthor=A.+Mendozaauthor=M.+J.+Rebollo-Lopezauthor=M.+Remui%C3%B1an-Blancoauthor=J.+L.+Lavanderaauthor=E.+P%C3%A9rez-Herranauthor=F.+J.+Gamo-Benitoauthor=J.+F.+Garc%C3%ADa-Bustosauthor=D.+Barrosauthor=J.+P.+Castroauthor=N.+Cammack&title=Fueling+open-source+drug+discovery%3A+177+small-molecule+leads+against+tuberculosis&doi=10.1002%2Fcmdc.201200428"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Fueling Open-Source Drug Discovery: 177 Small-Molecule Leads against Tuberculosis</span></div><div class="casAuthors">Ballell, Lluis; Bates, Robert H.; Young, Rob J.; Alvarez-Gomez, Daniel; Alvarez-Ruiz, Emilio; Barroso, Vanessa; Blanco, Delia; Crespo, Benigno; Escribano, Jaime; Gonzalez, Ruben; Lozano, Sonia; Huss, Sophie; Santos-Villarejo, Angel; Martin-Plaza, Jose Julio; Mendoza, Alfonso; Rebollo-Lopez, Maria Jose; Remuinan-Blanco, Modesto; Lavandera, Jose Luis; Perez-Herran, Esther; Gamo-Benito, Francisco Javier; Garcia-Bustos, Jose Francisco; Barros, David; Castro, Julia P.; Cammack, Nicholas</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">313-321</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">With the aim of fuelling open-source, translational, early-stage drug discovery activities, the results of the recently completed antimycobacterial phenotypic screening campaign against Mycobacterium bovis BCG with hit confirmation in M. tuberculosis H37Rv were made publicly accessible.  A set of 177 potent non-cytotoxic H37Rv hits was identified and will be made available to maximize the potential impact of the compds. toward a chem. genetics/proteomics exercise, while at the same time providing a plethora of potential starting points for new synthetic lead-generation activities.  Two addnl. drug-discovery-relevant datasets are included: (a) a drug-like property anal. reflecting the latest lead-like guidelines and (b) an early lead-generation package of the most promising hits within the clusters identified.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUJ2sVPWfPALVg90H21EOLACvtfcHk0lgKc6FU3JLTEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXls1Omug%253D%253D&md5=32c975dbe183849358c701ca697681d3</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201200428&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201200428%26sid%3Dliteratum%253Aachs%26aulast%3DBallell%26aufirst%3DL.%26aulast%3DBates%26aufirst%3DR.%2BH.%26aulast%3DYoung%26aufirst%3DR.%2BJ.%26aulast%3DAlvarez-Gomez%26aufirst%3DD.%26aulast%3DAlvarez-Ruiz%26aufirst%3DE.%26aulast%3DBarroso%26aufirst%3DV.%26aulast%3DBlanco%26aufirst%3DD.%26aulast%3DCrespo%26aufirst%3DB.%26aulast%3DEscribano%26aufirst%3DJ.%26aulast%3DGonz%25C3%25A1lez%26aufirst%3DR.%26aulast%3DLozano%26aufirst%3DS.%26aulast%3DHuss%26aufirst%3DS.%26aulast%3DSantos-Villarejo%26aufirst%3DA.%26aulast%3DMart%25C3%25ADn-Plaza%26aufirst%3DJ.%2BJ.%26aulast%3DMendoza%26aufirst%3DA.%26aulast%3DRebollo-Lopez%26aufirst%3DM.%2BJ.%26aulast%3DRemui%25C3%25B1an-Blanco%26aufirst%3DM.%26aulast%3DLavandera%26aufirst%3DJ.%2BL.%26aulast%3DP%25C3%25A9rez-Herran%26aufirst%3DE.%26aulast%3DGamo-Benito%26aufirst%3DF.%2BJ.%26aulast%3DGarc%25C3%25ADa-Bustos%26aufirst%3DJ.%2BF.%26aulast%3DBarros%26aufirst%3DD.%26aulast%3DCastro%26aufirst%3DJ.%2BP.%26aulast%3DCammack%26aufirst%3DN.%26atitle%3DFueling%2520open-source%2520drug%2520discovery%253A%2520177%2520small-molecule%2520leads%2520against%2520tuberculosis%26jtitle%3DChemMedChem%26date%3D2013%26volume%3D8%26spage%3D313%26epage%3D321%26doi%3D10.1002%2Fcmdc.201200428" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group">Zheng, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anglin, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prasad, B. V. V</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, Y.</span><span> </span><span class="NLM_article-title">Crystallographic Investigation and Selective Inhibition of Mutant Isocitrate Dehydrogenase</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">542</span><span class="NLM_x">–</span> <span class="NLM_lpage">546</span><span class="refDoi"> DOI: 10.1021/ml400036z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml400036z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A528%3ADC%252BC3sXntVWrtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=542-546&author=B.+Zhengauthor=Y.+Yaoauthor=Z.+Liuauthor=L.+Dengauthor=J.+L.+Anglinauthor=H.+Jiangauthor=B.+V.+V+Prasadauthor=Y.+Song&title=Crystallographic+Investigation+and+Selective+Inhibition+of+Mutant+Isocitrate+Dehydrogenase&doi=10.1021%2Fml400036z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Crystallographic Investigation and Selective Inhibition of Mutant Isocitrate Dehydrogenase</span></div><div class="casAuthors">Zheng, Baisong; Yao, Yuan; Liu, Zhen; Deng, Lisheng; Anglin, Justin L.; Jiang, Hong; Prasad, B. V. Venkataram; Song, Yongcheng</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">542-546</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mutations in isocitrate dehydrogenase (IDH), a key enzyme in the tricarboxylic acid cycle, have recently been found in ∼75% glioma and ∼20% acute myeloid leukemia.  Different from the wild-type enzyme, mutant IDH1 catalyzes the redn. of α-ketoglutaric acid to D-2-hydroxyglutaric acid.  Strong evidence has shown mutant IDH1 represents a novel target for this type of cancer.  We found two 1-hydroxypyridin-2-one compds. that are potent inhibitors of R132H and R132C IDH1 mutants with Ki values as low as 120 nM.  These compds. exhibit >60-fold selectivity against wild-type IDH1 and can inhibit the prodn. of D-2-hydroxyglutaric acid in IDH1 mutated cells, representing novel chem. probes for cancer biol. studies.  We also report the first inhibitor-bound crystal structures of IDH1(R132H), showing these inhibitors have H-bond, electrostatic, and hydrophobic interactions with the mutant enzyme.  Comparison with the substrate-bound IDH1 structures revealed the structural basis for the high enzyme selectivity of these compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqH0vzVLGCPZLVg90H21EOLACvtfcHk0lig6zTLy__EFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXntVWrtLg%253D&md5=c34c67949fe816bdf151525a578535f3</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Fml400036z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml400036z%26sid%3Dliteratum%253Aachs%26aulast%3DZheng%26aufirst%3DB.%26aulast%3DYao%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DDeng%26aufirst%3DL.%26aulast%3DAnglin%26aufirst%3DJ.%2BL.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DPrasad%26aufirst%3DB.%2BV.%2BV%26aulast%3DSong%26aufirst%3DY.%26atitle%3DCrystallographic%2520Investigation%2520and%2520Selective%2520Inhibition%2520of%2520Mutant%2520Isocitrate%2520Dehydrogenase%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D4%26spage%3D542%26epage%3D546%26doi%3D10.1021%2Fml400036z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group">Deng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pujari, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crick, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prasad, B. V. V</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, Y.</span><span> </span><span class="NLM_article-title">Inhibition of 1-Deoxy-D-Xylulose-5-Phosphate Reductoisomerase by Lipophilic Phosphonates: SAR, QSAR and Crystallographic Studies</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">4721</span><span class="NLM_x">–</span> <span class="NLM_lpage">34</span><span class="refDoi"> DOI: 10.1021/jm200363d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm200363d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=4721-34&author=L.+Dengauthor=J.+Diaoauthor=P.+Chenauthor=V.+Pujariauthor=Y.+Yaoauthor=G.+Chengauthor=D.+C.+Crickauthor=B.+V.+V+Prasadauthor=Y.+Song&title=Inhibition+of+1-Deoxy-D-Xylulose-5-Phosphate+Reductoisomerase+by+Lipophilic+Phosphonates%3A+SAR%2C+QSAR+and+Crystallographic+Studies&doi=10.1021%2Fjm200363d"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Fjm200363d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200363d%26sid%3Dliteratum%253Aachs%26aulast%3DDeng%26aufirst%3DL.%26aulast%3DDiao%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DP.%26aulast%3DPujari%26aufirst%3DV.%26aulast%3DYao%26aufirst%3DY.%26aulast%3DCheng%26aufirst%3DG.%26aulast%3DCrick%26aufirst%3DD.%2BC.%26aulast%3DPrasad%26aufirst%3DB.%2BV.%2BV%26aulast%3DSong%26aufirst%3DY.%26atitle%3DInhibition%2520of%25201-Deoxy-D-Xylulose-5-Phosphate%2520Reductoisomerase%2520by%2520Lipophilic%2520Phosphonates%253A%2520SAR%252C%2520QSAR%2520and%2520Crystallographic%2520Studies%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D4721%26epage%3D34%26doi%3D10.1021%2Fjm200363d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group">Deng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sundriyal, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rubio, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, Y.</span><span> </span><span class="NLM_article-title">Coordination Chemistry Based Approach to Lipophilic Inhibitors of 1-Deoxy-<i>D</i>-xylulose-5-phosphate Reductoisomerase</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">6539</span><span class="NLM_x">–</span> <span class="NLM_lpage">42</span><span class="refDoi"> DOI: 10.1021/jm9012592</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm9012592" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=6539-42&author=L.+Dengauthor=S.+Sundriyalauthor=V.+Rubioauthor=Z.+Shiauthor=Y.+Song&title=Coordination+Chemistry+Based+Approach+to+Lipophilic+Inhibitors+of+1-Deoxy-D-xylulose-5-phosphate+Reductoisomerase&doi=10.1021%2Fjm9012592"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Fjm9012592&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm9012592%26sid%3Dliteratum%253Aachs%26aulast%3DDeng%26aufirst%3DL.%26aulast%3DSundriyal%26aufirst%3DS.%26aulast%3DRubio%26aufirst%3DV.%26aulast%3DShi%26aufirst%3DZ.%26aulast%3DSong%26aufirst%3DY.%26atitle%3DCoordination%2520Chemistry%2520Based%2520Approach%2520to%2520Lipophilic%2520Inhibitors%2520of%25201-Deoxy-D-xylulose-5-phosphate%2520Reductoisomerase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D6539%26epage%3D42%26doi%3D10.1021%2Fjm9012592" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group">Zhang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wei, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mo, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, Y.</span><span> </span><span class="NLM_article-title">Inhibition of Histone H3K79 Methylation Selectively Inhibits Proliferation, Self-renewal and Metastatic Potential of Breast Cancer</span> <span class="citation_source-journal">Oncotarget</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">10665</span><span class="NLM_x">–</span> <span class="NLM_lpage">10677</span><span class="refDoi"> DOI: 10.18632/oncotarget.2496</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=10.18632%2Foncotarget.2496" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=25359765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;key=1%3ACAS%3A280%3ADC%252BC2M3kslylsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=10665-10677&author=L.+Zhangauthor=L.+Dengauthor=F.+Chenauthor=Y.+Yaoauthor=B.+Wuauthor=L.+Weiauthor=Q.+Moauthor=Y.+Song&title=Inhibition+of+Histone+H3K79+Methylation+Selectively+Inhibits+Proliferation%2C+Self-renewal+and+Metastatic+Potential+of+Breast+Cancer&doi=10.18632%2Foncotarget.2496"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of histone H3K79 methylation selectively inhibits proliferation, self-renewal and metastatic potential of breast cancer</span></div><div class="casAuthors">Zhang Li; Deng Lisheng; Yao Yuan; Wu Bulan; Wei Liping; Chen Fengju; Mo Qianxing; Song Yongcheng</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">10665-77</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Histone lysine methylation regulates gene expression and cancer initiation.  Bioinformatics analysis suggested that DOT1L, a histone H3-lysine79 (H3K79) methyltransferase, plays a potentially important role in breast cancer.  DOT1L inhibition selectively inhibited proliferation, self-renewal, metastatic potential of breast cancer cells and induced cell differentiation.  In addition, inhibitors of S-adenosylhomocysteine hydrolase (SAHH), such as neplanocin and 3-deazaneplanocin, also inhibited both H3K79 methylation and proliferation of breast cancer cells in vitro and in vivo.  The activity of SAHH inhibitors was previously attributed to inhibition of H3K27 methyltransferase EZH2.  However, inhibition of EZH2 by a specific inhibitor did not contribute to cell death.  SAHH inhibitors had only weak activity against H3K27 methylation and their activity is therefore mainly due to DOT1L/H3K79 methylation inhibition.  Overall, we showed that DOT1L is a potential drug target for breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcToXKVEBdgQcLHHuJhbyicZfW6udTcc2eYIqyZmfwY9dLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2M3kslylsg%253D%253D&md5=c495f4521165f0fb79399b80d8181955</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.2496&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.2496%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DDeng%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DF.%26aulast%3DYao%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DB.%26aulast%3DWei%26aufirst%3DL.%26aulast%3DMo%26aufirst%3DQ.%26aulast%3DSong%26aufirst%3DY.%26atitle%3DInhibition%2520of%2520Histone%2520H3K79%2520Methylation%2520Selectively%2520Inhibits%2520Proliferation%252C%2520Self-renewal%2520and%2520Metastatic%2520Potential%2520of%2520Breast%2520Cancer%26jtitle%3DOncotarget%26date%3D2014%26volume%3D5%26spage%3D10665%26epage%3D10677%26doi%3D10.18632%2Foncotarget.2496" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group">Liu, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kogiso, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qiu, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lv, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gallo, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, X.-N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, Y.</span><span> </span><span class="NLM_article-title">Inhibition of Cancer-Associated Mutant Isocitrate Dehydrogenases: Synthesis, SAR and Selective Antitumor Activity</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">8307</span><span class="NLM_x">–</span> <span class="NLM_lpage">8318</span><span class="refDoi"> DOI: 10.1021/jm500660f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm500660f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=8307-8318&author=Z.+Liuauthor=Y.+Yaoauthor=M.+Kogisoauthor=B.+Zhengauthor=L.+Dengauthor=J.+J.+Qiuauthor=S.+Dongauthor=H.+Lvauthor=J.+M.+Galloauthor=X.-N.+Liauthor=Y.+Song&title=Inhibition+of+Cancer-Associated+Mutant+Isocitrate+Dehydrogenases%3A+Synthesis%2C+SAR+and+Selective+Antitumor+Activity&doi=10.1021%2Fjm500660f"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Fjm500660f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm500660f%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DYao%26aufirst%3DY.%26aulast%3DKogiso%26aufirst%3DM.%26aulast%3DZheng%26aufirst%3DB.%26aulast%3DDeng%26aufirst%3DL.%26aulast%3DQiu%26aufirst%3DJ.%2BJ.%26aulast%3DDong%26aufirst%3DS.%26aulast%3DLv%26aufirst%3DH.%26aulast%3DGallo%26aufirst%3DJ.%2BM.%26aulast%3DLi%26aufirst%3DX.-N.%26aulast%3DSong%26aufirst%3DY.%26atitle%3DInhibition%2520of%2520Cancer-Associated%2520Mutant%2520Isocitrate%2520Dehydrogenases%253A%2520Synthesis%252C%2520SAR%2520and%2520Selective%2520Antitumor%2520Activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D8307%26epage%3D8318%26doi%3D10.1021%2Fjm500660f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2V1D" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2V1D','PDB','2V1D'); return false;">PDB: 2V1D</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-2"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i27"><a href="/doi/suppl/10.1021/acs.jmedchem.5b01361">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_80015"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.5b01361">10.1021/acs.jmedchem.5b01361</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Figures S1–S3 and Experimental Section showing detailed compound synthesis and characterization (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.5b01361/suppl_file/jm5b01361_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.5b01361/suppl_file/jm5b01361_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b01361/suppl_file/jm5b01361_si_001.pdf">jm5b01361_si_001.pdf (517.55 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b01361/suppl_file/jm5b01361_si_002.csv">jm5b01361_si_002.csv (2.98 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.5b01361&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2016.59.issue-1%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.5b01361%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.5b01361" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                3MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679937930b373db8","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
