
<html lang="en"     class="pb-page"  data-request-id="a4a5c6db-0e23-4a99-b357-ee9f66eafa51"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;article:article:10.1021/acs.jmedchem.1c00460;issue:issue:10.1021/jmcmar.2021.64.issue-14"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Design, Synthesis, and Evaluation of VHL-Based EZH2 Degraders to Enhance Therapeutic Activity against Lymphoma" /></meta><meta name="dc.Creator" content="Yalin  Tu" /></meta><meta name="dc.Creator" content="Yameng  Sun" /></meta><meta name="dc.Creator" content="Shuang  Qiao" /></meta><meta name="dc.Creator" content="Yao  Luo" /></meta><meta name="dc.Creator" content="Panpan  Liu" /></meta><meta name="dc.Creator" content="Zhong-Xing  Jiang" /></meta><meta name="dc.Creator" content="Yumin  Hu" /></meta><meta name="dc.Creator" content="Zifeng  Wang" /></meta><meta name="dc.Creator" content="Peng  Huang" /></meta><meta name="dc.Creator" content="Shijun  Wen" /></meta><meta name="dc.Description" content="Traditional EZH2 inhibitors are developed to suppress the enzymatic methylation activity, and they may have therapeutic limitations due to the nonenzymatic functions of EZH2 in cancer development. ..." /></meta><meta name="Description" content="Traditional EZH2 inhibitors are developed to suppress the enzymatic methylation activity, and they may have therapeutic limitations due to the nonenzymatic functions of EZH2 in cancer development. ..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="July 1, 2021" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.1c00460" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2021 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.1c00460" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.1c00460" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.1c00460" /></link>
        
    
    

<title>Design, Synthesis, and Evaluation of VHL-Based EZH2 Degraders to Enhance Therapeutic Activity against Lymphoma | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.1c00460" /></meta><meta property="og:title" content="Design, Synthesis, and Evaluation of VHL-Based EZH2 Degraders to Enhance Therapeutic Activity against Lymphoma" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00460/20210716/images/large/jm1c00460_0012.jpeg" /></meta><meta property="og:description" content="Traditional EZH2 inhibitors are developed to suppress the enzymatic methylation activity, and they may have therapeutic limitations due to the nonenzymatic functions of EZH2 in cancer development. Here, we report proteolysis-target chimera (PROTAC)-based EZH2 degraders to target the whole EZH2 in lymphoma. Two series of EZH2 degraders were designed and synthesized to hijack E3 ligase systems containing either von Hippel–Lindau (VHL) or cereblon (CRBN), and some VHL-based compounds were able to mediate EZH2 degradation. Two best degraders, YM181 and YM281, induced robust cell viability inhibition in diffuse large B-cell lymphoma (DLBCL) and other subtypes of lymphomas, outperforming a clinically used EZH2 inhibitor EPZ6438 (tazemetostat) that was only effective against DLBCL. The EZH2 degraders displayed promising antitumor activities in lymphoma xenografts and patient-derived primary lymphoma cells. Our study demonstrates that EZH2 degraders have better therapeutic activity than EZH2 inhibitors, which may provide a potential anticancer strategy to treat lymphoma." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.1c00460"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00460">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.1c00460&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.1c00460&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.1c00460&amp;href=/doi/10.1021/acs.jmedchem.1c00460" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, 64</span><span class="cit-fg-issue">, 14</span><span class="cit-fg-pageRange">, 10167-10184</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/64/14" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.1c00437" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.1c00449" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Design, Synthesis, and Evaluation of VHL-Based EZH2 Degraders to Enhance Therapeutic Activity against Lymphoma</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Yalin Tu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yalin Tu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Oncology in South China, Collaborative Innovation Center for Cancer Medicine, Sun Yat-sen University Cancer Center, 651 Dongfeng East Road, Guangzhou 510060, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yalin++Tu">Yalin Tu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yameng Sun</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yameng Sun</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Oncology in South China, Collaborative Innovation Center for Cancer Medicine, Sun Yat-sen University Cancer Center, 651 Dongfeng East Road, Guangzhou 510060, China</div><div class="loa-info-affiliations-info">The School of Pharmaceutical Sciences, Wuhan University, Wuhan 430071, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yameng++Sun">Yameng Sun</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Shuang Qiao</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Shuang Qiao</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Oncology in South China, Collaborative Innovation Center for Cancer Medicine, Sun Yat-sen University Cancer Center, 651 Dongfeng East Road, Guangzhou 510060, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Shuang++Qiao">Shuang Qiao</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yao Luo</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yao Luo</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Oncology in South China, Collaborative Innovation Center for Cancer Medicine, Sun Yat-sen University Cancer Center, 651 Dongfeng East Road, Guangzhou 510060, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yao++Luo">Yao Luo</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Panpan Liu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Panpan Liu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Oncology in South China, Collaborative Innovation Center for Cancer Medicine, Sun Yat-sen University Cancer Center, 651 Dongfeng East Road, Guangzhou 510060, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Panpan++Liu">Panpan Liu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Zhong-Xing Jiang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Zhong-Xing Jiang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">The School of Pharmaceutical Sciences, Wuhan University, Wuhan 430071, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Zhong-Xing++Jiang">Zhong-Xing Jiang</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-2601-4366" title="Orcid link">https://orcid.org/0000-0003-2601-4366</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yumin Hu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yumin Hu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Oncology in South China, Collaborative Innovation Center for Cancer Medicine, Sun Yat-sen University Cancer Center, 651 Dongfeng East Road, Guangzhou 510060, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yumin++Hu">Yumin Hu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Zifeng Wang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Zifeng Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Oncology in South China, Collaborative Innovation Center for Cancer Medicine, Sun Yat-sen University Cancer Center, 651 Dongfeng East Road, Guangzhou 510060, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Zifeng++Wang">Zifeng Wang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Peng Huang</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Peng Huang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Oncology in South China, Collaborative Innovation Center for Cancer Medicine, Sun Yat-sen University Cancer Center, 651 Dongfeng East Road, Guangzhou 510060, China</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#c8a0bda9a6afb8ada6af88bbb1bbbdababe6a7baafe6aba6"><span class="__cf_email__" data-cfemail="7e160b1f10190e1b10193e0d070d0b1d1d50110c19501d10">[email protected]</span></a>+86 20 87343511.</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Peng++Huang">Peng Huang</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Shijun Wen</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Shijun Wen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Oncology in South China, Collaborative Innovation Center for Cancer Medicine, Sun Yat-sen University Cancer Center, 651 Dongfeng East Road, Guangzhou 510060, China</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#1166747f62797b516268626472723f7e63763f727f"><span class="__cf_email__" data-cfemail="0374666d706b6943707a707660602d6c71642d606d">[email protected]</span></a>. Tel. +86 20 87342283.</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Shijun++Wen">Shijun Wen</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-9347-8243" title="Orcid link">https://orcid.org/0000-0002-9347-8243</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.1c00460&amp;href=/doi/10.1021%2Facs.jmedchem.1c00460" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, 64</span><span class="cit-issue">, 14</span><span class="cit-pageRange">, 10167–10184</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">July 1, 2021</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>13 March 2021</li><li><span class="item_label"><b>Published</b> online</span>1 July 2021</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 22 July 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.1c00460" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00460</a></div><div class="article_header-article-copyright"><strong>Copyright © 2021 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D10167%26pageCount%3D18%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DYalin%2BTu%252C%2BYameng%2BSun%252C%2BShuang%2BQiao%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D64%26issueNum%3D14%26contentID%3Dacs.jmedchem.1c00460%26title%3DDesign%252C%2BSynthesis%252C%2Band%2BEvaluation%2Bof%2BVHL-Based%2BEZH2%2BDegraders%2Bto%2BEnhance%2BTherapeutic%2BActivity%2Bagainst%2BLymphoma%26numPages%3D18%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D10184%26publicationDate%3DJuly%2B2021">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.1c00460"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1113</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val">-</div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.1c00460" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Design, Synthesis, and Evaluation of VHL-Based EZH2 Degraders to Enhance Therapeutic Activity against Lymphoma&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Yalin&quot;,&quot;last_name&quot;:&quot;Tu&quot;},{&quot;first_name&quot;:&quot;Yameng&quot;,&quot;last_name&quot;:&quot;Sun&quot;},{&quot;first_name&quot;:&quot;Shuang&quot;,&quot;last_name&quot;:&quot;Qiao&quot;},{&quot;first_name&quot;:&quot;Yao&quot;,&quot;last_name&quot;:&quot;Luo&quot;},{&quot;first_name&quot;:&quot;Panpan&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Zhong-Xing&quot;,&quot;last_name&quot;:&quot;Jiang&quot;},{&quot;first_name&quot;:&quot;Yumin&quot;,&quot;last_name&quot;:&quot;Hu&quot;},{&quot;first_name&quot;:&quot;Zifeng&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Peng&quot;,&quot;last_name&quot;:&quot;Huang&quot;},{&quot;first_name&quot;:&quot;Shijun&quot;,&quot;last_name&quot;:&quot;Wen&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2021&quot;,&quot;month&quot;:&quot;07&quot;,&quot;day&quot;:&quot;01&quot;,&quot;issue&quot;:&quot;14&quot;,&quot;volume&quot;:&quot;64&quot;,&quot;pages&quot;:&quot;10167-10184&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.1c00460&quot;},&quot;abstract&quot;:&quot;Traditional EZH2 inhibitors are developed to suppress the enzymatic methylation activity, and they may have therapeutic limitations due to the nonenzymatic functions of EZH2 in cancer development. Here, we report proteolysis-target chimera (PROTAC)-based EZH2 degraders to target the whole EZH2 in lymphoma. Two series of EZH2 degraders were designed and synthesized to hijack E3 ligase systems containing either von Hippel–Lindau (VHL) or cereblon (CRBN), and some VHL-based compounds were able to mediate EZH2 degradation. Two best degraders, YM181 and YM281, induced robust cell viability inhibition in diffuse large B-cell lymphoma (DLBCL) and other subtypes of lymphomas, outperforming a clinically used EZH2 inhibitor EPZ6438 (tazemetostat) that was only effective against DLBCL. The EZH2 degraders displayed promising antitumor activities in lymphoma xenografts and patient-derived primary lymphoma cells. Our study demonstrates that EZH2 degraders have better therapeutic activity than EZH2 inhibitors, which may pro&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00460&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00460" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00460&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00460" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00460&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00460" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.1c00460&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00460&amp;href=/doi/10.1021/acs.jmedchem.1c00460" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.1c00460" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.1c00460" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (5 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00460&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.1c00460%26sid%3Dliteratum%253Aachs%26pmid%3D34196564%26genre%3Darticle%26aulast%3DTu%26date%3D2021%26atitle%3DDesign%252C%2BSynthesis%252C%2Band%2BEvaluation%2Bof%2BVHL-Based%2BEZH2%2BDegraders%2Bto%2BEnhance%2BTherapeutic%2BActivity%2Bagainst%2BLymphoma%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D64%26issue%3D14%26spage%3D10167%26epage%3D10184%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (3)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291316" title="Mixtures">Mixtures</a>,</li><li><a href="/action/doSearch?ConceptID=291781" title="Degradation">Degradation</a>,</li><li><a href="/action/doSearch?ConceptID=290896" title="Cells">Cells</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=292153" title="Tumors">Tumors</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/64/14" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00460/20210716/images/medium/jm1c00460_0012.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00460/20210716/images/large/jm1c00460_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00460&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Traditional EZH2 inhibitors are developed to suppress the enzymatic methylation activity, and they may have therapeutic limitations due to the nonenzymatic functions of EZH2 in cancer development. Here, we report proteolysis-target chimera (PROTAC)-based EZH2 degraders to target the whole EZH2 in lymphoma. Two series of EZH2 degraders were designed and synthesized to hijack E3 ligase systems containing either von Hippel–Lindau (VHL) or cereblon (CRBN), and some VHL-based compounds were able to mediate EZH2 degradation. Two best degraders, YM181 and YM281, induced robust cell viability inhibition in diffuse large B-cell lymphoma (DLBCL) and other subtypes of lymphomas, outperforming a clinically used EZH2 inhibitor EPZ6438 (tazemetostat) that was only effective against DLBCL. The EZH2 degraders displayed promising antitumor activities in lymphoma xenografts and patient-derived primary lymphoma cells. Our study demonstrates that EZH2 degraders have better therapeutic activity than EZH2 inhibitors, which may provide a potential anticancer strategy to treat lymphoma.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_34422" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_34422" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">EZH2 is the enzymatic subunit of polycomb repressive complex 2 (PRC2) that mainly trimethylate lysine 27 of histone H3 (H3K27) to silence the gene transcription.<a onclick="showRef(event, 'ref1 ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3">(1−3)</a> Either overexpression or gain-of-function mutation of EZH2 is observed in various tumors, including lymphoma, especially diffuse large B-cell lymphoma (DLBCL), T-cell acute lymphoblastic leukemia, breast cancer, and prostate cancer.<a onclick="showRef(event, 'ref4 ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref4 ref5 ref6">(4−6)</a> The oncogenic roles of EZH2 are often attributed to its end-product H3K27me3-mediated epigenetic silencing of tumor suppressor genes.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> The loss of mutation of specific subunits of other chromatin remodeling complexes that antagonize PRC2 activity, like the SWI/SNF complex, is another major reason for the activation of EZH2.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> For example, the loss of the SWI/SNF subunit INI1 (encoded by <i>SMARCB1</i>) leads to constitutive EZH2 activation in nearly all epithelioid sarcomas.<a onclick="showRef(event, 'ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref8 ref9">(8,9)</a> A broad role for EZH2 in the progression of cancers with mutations of the SWI/SNF subunits ARID1A, PBRM1, and SMARCA4 has also been demonstrated in both cell lines and in vivo models.<a onclick="showRef(event, 'ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref10 ref11">(10,11)</a></div><div class="NLM_p">Many compounds that inhibit the methylation enzymatic activity of EZH2 are developed and show promising antitumor efficacy in preclinical and clinical trials in hematologic malignancies as well as solid tumors that exhibit strong EZH2 dependencies due to synthetic lethality with the aforementioned SWI/SNF subunit mutations.<a onclick="showRef(event, 'ref6 ref12'); return false;" href="javascript:void(0);" class="ref ref6 ref12">(6,12)</a> In 2020, the most advanced EZH2 inhibitor EPZ6438 (tazemetostat) has been approved by FDA for the treatment of metastatic and locally advanced epithelioid sarcoma.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> These inhibitors often bind to the SET domain of EZH2 and compete with the cofactor <i>S</i>-adenosylmethionine (SAM) without affecting the protein stability of EZH2.<a onclick="showRef(event, 'ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref14 ref15">(14,15)</a> However, increasing evidence has revealed that the oncogenic function of EZH2 is not entirely dependent on its enzymatic activity. The whole EZH2 protein itself is also correlated with tumor proliferation, independent of its H3K27 trimethylation activity.<a onclick="showRef(event, 'ref5 ref10 ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref5 ref10 ref16 ref17">(5,10,16,17)</a> Meanwhile, some drawbacks of current EZH2 inhibitors are uncovered, for example, excessive dosage, acquired resistance, and drug insensitivity to most solid tumors.<a onclick="showRef(event, 'ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref18 ref19">(18,19)</a> Therefore, we hypothesized that the therapeutic gains from abrogating the whole EZH2 protein rather than simply inhibiting its enzymatic activity might be better in EZH2-dependent cancers, and the development of EZH2-targeting degraders is demanded.</div><div class="NLM_p">Cells can maintain proteins in a well-preserved homeostasis using the natural ubiquitin–proteasome degradation system in which ubiquitination is the initial key process. When ubiquitin, a small protein, tags the targeted proteins that are accumulated abnormally, the ubiquitylated proteins will be subjected to the proteasome for degradation to remove the protein redundancy. Proteolysis targeting chimeras (PROTACs) can achieve selective protein degradation by artificially hijacking natural ubiquitin–proteasome systems. PROTAC-based degraders have two crucial warheads: one binding to a protein of interest (POI) and the other binding to E3 ubiquitin ligase. These special bifunctional molecules force a handshake between an E3 ligase and the POI to artificially ubiquitylate the POI that is subsequently subjected to the proteasome-mediated depletion.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> The PROTAC technology has been employed to degrade several proteins like BRD4,<a onclick="showRef(event, 'ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref21 ref22">(21,22)</a> AR,<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> and Stat3.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> PROTAC-based degraders not only intervene in the enzymatic activities that are targeted by the traditional inhibitors but also disturb nonenzymatic functions of the interested protein, and they may substantially improve the therapeutic efficacy and overcome the inhibitor-mediated drug resistance.<a onclick="showRef(event, 'ref20 ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref20 ref25 ref26">(20,25,26)</a> Moreover, the potential advantages of PROTACs in the epigenetic context have also been attracting the attention of researchers. Compared with the parental epigenetic inhibitors, epigenetic PROTACs endowed with higher target selectivity, increased potency, prolonged action, reduced side-effects, and risk of resistance would offer more potential therapeutic strategies for the epigenetic POI-dependent cancers and would be very useful as chemical tools for dissecting the biological roles of the epigenetic POI.<a onclick="showRef(event, 'ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref27 ref28">(27,28)</a></div><div class="NLM_p last">Herein, we report the development of specific EZH2 degraders YM181 and YM281 based on PROTAC and further elaborate their better antitumor effects against not only DLBCL but also other types of lymphomas in vitro and in vivo compared to the parental EZH2 inhibitor EPZ6438. Meanwhile, YM281 caused robust cell death and substantial viability inhibition of primary lymphoma cells from patients. Our work provides novel EZH2 degraders to target a broad range of lymphomas of which EZH2 inhibitors only exhibit their limited efficacy to DLBCL.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_26121" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_26121" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Design and Screening of EZH2 Degraders via the PROTAC Strategy</h3><div class="NLM_p">In our previous work, we found that the most advanced EZH2 inhibitor EPZ6438 required high doses to reach reasonable in vitro antiproliferative activity against acute myeloid leukemia, although the compound was reported to suppress EZH2 activity largely at nanomolar concentration levels.<a onclick="showRef(event, 'ref16 ref29'); return false;" href="javascript:void(0);" class="ref ref16 ref29">(16,29)</a> Since lymphomas are believed to be more sensitive to EZH2 inhibitors, EPZ6438 was used to screen our laboratory available lymphoma cell lines, including Burkitt’s lymphoma (BL), lymphoblastic lymphoma (LBL), mantle cell lymphoma (MCL), and DLBCL cell lines. However, most lymphoma cell lines and even one DLBCL cell line Toledo did not exhibit obvious responses to EZH2 inhibition, although EZH2 protein levels are similar in all cell lines (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A,B). Moreover, EPZ6438 could not completely suppress the cell growth of the tested DLBCL (SU-DHL-2, SU-DHL-4, and SU-DHL-6), which are widely reported to be sensitive to EZH2 inhibition, with at least 25% cells remaining viable even at a high dose (10 μM). Meanwhile, the EZH2 knockdown almost completely inhibited the cell growth of SU-DHL-2 (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C). These observations confirmed the necessity of the development of EZH2 degraders. Using the conventional PROTAC strategy (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>D), our molecular docking model indicated a possibility of the morpholine end to be extended to an E3 ligase binder to assemble EZH2 degraders (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>E).<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00460/20210716/images/medium/jm1c00460_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00460/20210716/images/large/jm1c00460_0002.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Strategy to develop EZH2 degraders based on PROTAC technology. (A) MTS cell viability analysis in different lymphoma cell lines treated with indicated concentrations of EPZ6438 for 5 days. (B) Western blot analysis of EZH2 protein levels in different lymphoma cell lines. Tubulin was used as a loading control. (C) Cell numbers were measured by cell count after the transfection of control shRNA or shEZH2 in SU-DHL-2 cells for the indicated days (left). EZH2 and H3K27me3 levels were measured by western blot assay. H3 and tubulin were used as loading controls (right). (D) Schematic representation of the EZH2 degrader design. (E) Docking conformation of EPZ6438 in the catalytic domain of EZH2 (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LS6">5LS6</a>, left) and the chemical structure of EPZ6438 (right). Note: EPZ stands for EPZ6438 in this figure and the following figures.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00460/20210716/images/large/jm1c00460_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00460&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We removed the morpholine motif with a direct linker connected to EPZ6438 to minimize the overall final molecular size for proper cellular permeability. Then, two series of PROTAC-based EZH2 degraders were designed and synthesized to hijack two widely studied E3 ligase systems containing either von Hippel–Lindau (VHL) or cereblon (CRBN). Compounds <b>V1</b>–<b>V7</b> were linked in different lengths to a small peptide-like fragment that can bind to VHL (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A).<a onclick="showRef(event, 'ref31 ref32'); return false;" href="javascript:void(0);" class="ref ref31 ref32">(31,32)</a> Detection of the EZH2 protein level and its catalytic activity mark H3K27me3 by western blot was performed to screen the synthetic compounds at a 0.3–3 μM concentration range to avoid the so-called “hook effect”.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> YM281 (also named as <b>V2</b>), YM181 (also named as <b>V3</b>), and <b>V4</b> significantly decreased the H3K27me3 degree at 1 μM, while <b>V1</b> and <b>V5</b> required high doses to reach the same level (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B,C). However, only YM181 and YM281 depleted the half protein level of EZH2 at 1 μM (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B,D). Neither compound <b>V6</b> nor <b>V7</b> with a longer linker at either 14 or 17 atoms length had an impact on both levels of EZH2 and H3K27me3, indicating that the over-long linker might sacrifice the binding capacity to EZH2, the cell permeability of compounds, and ternary complex formation. Our second series of designed EZH2 degraders <b>G1</b>–<b>G6</b> were linked to the thalidomide motif that binds to CRBN. However, none of them showed promising EZH2 degradation capacity, while most of them retained the EZH2 enzymatic inhibition activities (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>E and <a href="/doi/suppl/10.1021/acs.jmedchem.1c00460/suppl_file/jm1c00460_si_001.pdf" class="ext-link">S1</a>). It seemed that the VHL hijacking approach is more efficient to degrade EZH2.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> In some cases, a more diverse arsenal of E3 ligase ligands may maximize the opportunity for complementary surfaces between the E3 ligase and POI and thus improve the degradation efficiency of PROTACs.<a onclick="showRef(event, 'ref34 ref35 ref36'); return false;" href="javascript:void(0);" class="ref ref34 ref35 ref36">(34−36)</a> Notwithstanding, more investigation is needed to elucidate the exact reason for the failure of G series compounds to degrade EZH2. Based on the EZH2 degradation efficiency, the two best compounds, YM181 and YM281, were chosen for further investigation.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00460/20210716/images/medium/jm1c00460_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00460/20210716/images/large/jm1c00460_0003.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Chemical structures and degradation efficacy of the EZH2 degraders. (A) Chemical structures and linker lengths of VHL-based PROTACs. (B) Western blot analysis of EZH2 and H3K27me3 levels in 22Rv1 cells treated with indicated concentrations of compounds or dimethyl sulfoxide (DMSO) for 48 h. H3 and tubulin were used as loading controls. (C, D) Scatter plots showing the relative (C) H3K27me3 and (D) EZH2 bands abundance in B. (E) Chemical structures and degradation efficiency of CRBN-based PROTACs. N, no significant effects on the related target and Y, significant effects on the related target.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00460/20210716/images/large/jm1c00460_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00460&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> YM181 and YM281 Degraded the EZH2 Protein through the VHL-Dependent Ubiquitin–Proteasome System</h3><div class="NLM_p">As shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A, YM181 and YM281 abrogated both the EZH2 protein level and the H3K27me3 degree in a concentration-dependent manner in 24 h, and moreover had no significant effect on the protein level of EZH1 that is homologous to EZH2 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B and <a href="/doi/suppl/10.1021/acs.jmedchem.1c00460/suppl_file/jm1c00460_si_001.pdf" class="ext-link">S2</a>). The maximum degradation efficacy reached 80% at a concentration of 2 μM. A time-course study demonstrated that the EZH2 degradation could be detected in 2 h, and this effect reached a maximum in 4 h (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C). It is reported that the PRC2 complex collapse in the absence of any core subunit.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> To test the influence of EZH2 degraders on the PRC2 complex, the protein levels of the other two subunits, EED and SUZ12, were examined in both SU-DHL-2 and 22Rv1 cells treated with YM181 at 2 μM (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>D). Within 24 h, YM181 substantially reduced the levels of EED and SUZ12. The results demonstrated that YM181 and YM281 rapidly deleted the EZH2 protein and consequently destabilized the PRC2 complex due to the loss of integrity.<a onclick="showRef(event, 'ref25 ref37 ref38'); return false;" href="javascript:void(0);" class="ref ref25 ref37 ref38">(25,37,38)</a> Alternatively, it is also possible that EED and/or SUZ12 in proximity to EZH2 may be ubiquitinated by the EZH2-PROTAC-mediated ternary complex formation.<a onclick="showRef(event, 'ref37 ref38'); return false;" href="javascript:void(0);" class="ref ref37 ref38">(37,38)</a> In the following experiments, as shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>E, the degradation effect of YM181 could be rescued by adding either a proteasome inhibitor MG132 or MLN-4924, an inhibitor of the neddylation that is essential to activate the VHL E3 ligase system.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> Meanwhile, the EZH2 inhibitor EPZ6438 competed with YM181 to occupy the catalytic pocket and subsequently prevented protein degradation. Furthermore, the addition of a synthetic VHL ligand VH032 also reversed the EZH2 degradation caused by YM181. To further confirm the requirement of VHL for YM181-induced EZH2 degradation, VHL was knocked down by siRNAs in 22Rv1 cells. The exposure of the VHL knockdown cells to YM181 did not reduce the EZH2 protein level that was largely degraded in the control cells (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>F). Additionally, Co-IP experiments clearly displayed a significant increase of EZH2 ubiquitination mediated by both YM181 and YM281, while the whole protein level was decreased within 12 h (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>G). These results confirmed that the degradation of EZH2 induced by the two compounds was indeed associated with the VHL-dependent ubiquitin–proteasome system.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00460/20210716/images/medium/jm1c00460_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00460/20210716/images/large/jm1c00460_0004.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. EZH2 degraders abrogated the EZH2 protein level and the PRC2 complex through the VHL-dependent ubiquitin–proteasome system. (A) Chemical structures of YM181 and YM281. (B) Relative protein bands abundance in 22Rv1 cells treated with indicated concentrations of YM181 and YM281 for 24 h. (C) Western blot analysis of EZH2 and H3K27me3 levels in 22Rv1 cells treated with YM181 (2 μM) for the indicated exposure time. H3 and tubulin were used as loading controls. (D) Western blot analysis of the indicated protein levels in 22Rv1 and SU-DHL-2 cells treated with YM181 (2 μM) for 24 h. H3 was used as a loading control. (E) Western blot analysis of EZH2 levels in 22Rv1 cells treated with YM181 (2 μM) for 24 h after 2 h pretreatment with DMSO, MG132 (0.5 μM), MLN-4924 (0.4 μM), EPZ6438 (2 μM), and VH032 (2 μM). Tubulin was used as a loading control. (F) Western blot analysis of EZH2 and VHL levels after the transfection of nontargeting (NC) or VHL siRNAs for 48 h, followed by a 24 h YM181 treatment (2 μM) in 22Rv1 cells. Tubulin was used as a loading control. (G) Immunoprecipitation-western blot analysis of ubiquitylated EZH2 levels in 22Rv1 cells treated with 2 μM YM181 and YM281 for 12 h. Tubulin was used as a loading control.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00460/20210716/images/large/jm1c00460_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00460&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> EZH2 Degraders Displayed Stronger Anticancer Effects than EPZ6438 in Lymphoma Cell Lines</h3><div class="NLM_p">Given that EZH2 inhibitors are reportedly effective in the intervention of DLBCL, anticancer effects of EZH2 degraders were first tested in three DLBCL cell lines, SU-DHL-2, SU-DHL-4, and SU-DHL-6. The MTS assays confirmed that both YM181 and YM281 caused cell viability to decrease stronger than EPZ6438 in all three cell lines (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A). YM181 and YM281 could induce nearly complete cell viability inhibition compared to EPZ6438, although their initial effective concentrations were slightly higher. Meanwhile, EPZ6438 only achieved half inhibition in SU-DHL-2 and SU-DHL-4 cells and 70% inhibition in SU-DHL-6. Moreover, all of the other lymphoma cell lines that were resistant to the EZH2 inhibitor also showed a complete response to the EZH2 degraders (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00460/suppl_file/jm1c00460_si_001.pdf" class="ext-link">Figure S3A</a>). Western blot analysis further confirmed that YM181 induced substantial EZH2 degradation (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B and <a href="/doi/suppl/10.1021/acs.jmedchem.1c00460/suppl_file/jm1c00460_si_001.pdf" class="ext-link">S3B,C</a>). EPZ6438 robustly reduced the H3K27me3 levels at low nanomolar concentrations, while YM181 required high doses. The installment of both the linker and the VHL ligand motif might cause an EZH2 inhibitory activity loss, explaining the reason why EZH2 degraders needed a higher effect-starting concentration. To study whether the linker and the VHL ligand warhead would bring off-target effects, we also evaluated the cytostatic effects of V6 and V7 that are structurally similar but do not induce EZH2 degradation. Neither V6 nor V7 induced significant cell viability inhibition (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00460/suppl_file/jm1c00460_si_001.pdf" class="ext-link">Figure S3D,E</a>). Meanwhile, we synthesized YM620, an isomer of YM281, with alterations of two stereocenters in hydroxyproline, which diminish the binding affinity to VHL (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00460/suppl_file/jm1c00460_si_001.pdf" class="ext-link">Figure S3F</a>). Indeed, YM620 did not have an apparent effect on the EZH2 protein level, while it substantially inhibited EZH2 enzymatic activity (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00460/suppl_file/jm1c00460_si_001.pdf" class="ext-link">Figure S3G</a>). Compared to YM281, YM620 had a weaker antiproliferative capacity in two tested cancer cell lines (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00460/suppl_file/jm1c00460_si_001.pdf" class="ext-link">Figure S3H</a>). All of these data excluded the off-target effects of our EZH2 degraders.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00460/20210716/images/medium/jm1c00460_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00460/20210716/images/large/jm1c00460_0005.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. EZH2 degraders displayed stronger anticancer abilities than EPZ6438 in lymphoma cell lines. (A) MTS cell viability curves in different DLBCL cell lines treated with indicated compounds for 5 days. (B) Western blot analysis of EZH2 and H3K27me3 levels in SU-DHL-2 cells treated with the indicated compounds for 24 h. H3 and tubulin were used as loading controls. (C) Cell cycle analysis in SU-DHL-6 cells treated with DMSO or indicated compounds for 24 h. (D) Western blot analysis of PARP, caspase-3, and cleaved-caspase-3 levels in SU-DHL-6 cells treated with indicated compounds for 48 h. Tubulin was used as a loading control. (E) Proportions of PI<sup>+</sup> and/or Annexin-V<sup>+</sup> apoptotic cells in SU-DHL-6 cells treated with indicated compounds for 48 h as measured by flow cytometry.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00460/20210716/images/large/jm1c00460_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00460&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">To investigate how the EZH2 degraders intervene in cell growth of lymphoma cells, both the cell cycle and apoptosis analyses were conducted. After a 24 h treatment, EZP6438 at the tested doses (1, 3, 5 μM) slightly caused a G0/G1 phase arrest of SU-DHL-6 cells without significant observation of sub-G1 increase (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C and <a href="/doi/suppl/10.1021/acs.jmedchem.1c00460/suppl_file/jm1c00460_si_001.pdf" class="ext-link">S4A</a>). However, YM181 and YM281 at the same doses led to a concentration-dependent cell cycle arrest and a profound sub-G1 population increase. The caspase-3/7 assay demonstrated that EZH2 degraders increased the activity of caspase-3/7, and western blot also showed that the cleaved caspase-3 and PARP were significantly increased, indicating the apoptotic event (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>D and <a href="/doi/suppl/10.1021/acs.jmedchem.1c00460/suppl_file/jm1c00460_si_001.pdf" class="ext-link">S4B,C</a>). The Annexin-V/PI assay further verified that apoptosis was clearly induced by both EZH2 degraders in a dose-dependent manner, while EZP6438 did not (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>E and <a href="/doi/suppl/10.1021/acs.jmedchem.1c00460/suppl_file/jm1c00460_si_001.pdf" class="ext-link">S4D</a>). These results demonstrated that EZH2 degraders induced obvious cell cycle arrest and apoptosis, which might be the reasons why YM181 and YM281 could achieve a complete cell viability inhibition, whereas EPZ6438 could not if only inhibiting the methylation function of EZH2.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> EZH2 Degraders Reduced Tumor Growth In Vivo and Decreased Cell Viability in Primary Lymphoma Patient Cells</h3><div class="NLM_p">To investigate the antitumor activity of EZH2 degraders in vivo, a xenograft mouse model of DLBCL cell line SU-DHL-6 was first conducted. Consistent with in vitro results, YM281 (80 mg/kg) administered by intraperitoneal injection 6 times weekly for 3 weeks remarkably suppressed the tumor volume (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A). However, EPZ6438 at the equal molar dose to YM281 (42.5 mg/kg) failed. Neither EPZ6438 nor YM281 caused significant weight loss in mice (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00460/suppl_file/jm1c00460_si_001.pdf" class="ext-link">Figure S5A</a>). Western blot analysis of tumor tissue showed that YM281 significantly reduced the EZH2 protein and H3K27me3 levels (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B). EPZ36438 did not prevent the tumor growth, although it substantially decreased the H3K27me3 level, indicating that the enzymatic inhibition of EZH2 might not be enough to attenuate lymphoma cells in vivo. The significant destabilization of EZH2 and inhibition of tumor proliferation (indicated by Ki67 staining) induced by YM281 was further confirmed by immunohistochemistry in the tumor slices (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>C). The in vivo anticancer efficacy in DLBCL was validated in the mice xenograft model of Jeko-1, a mantle cell lymphoma cell line (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>D–F and <a href="/doi/suppl/10.1021/acs.jmedchem.1c00460/suppl_file/jm1c00460_si_001.pdf" class="ext-link">S5B</a>). More importantly, the tumor weight was clearly associated with the EZH2 protein level in YM281 treated mice (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00460/suppl_file/jm1c00460_si_001.pdf" class="ext-link">Figure S5C,D</a>). Finally, to evaluate the potential clinical implication of EZH2 degraders that showed promising efficacy in all different lymphoma cell lines, as shown in <a href="/doi/suppl/10.1021/acs.jmedchem.1c00460/suppl_file/jm1c00460_si_001.pdf" class="ext-link">Figure S3A</a>, primary lymphoma cells extracted from various lymphoma patient samples were used for further tests. First, EZH2 degrader YM281 induced dose-dependent EZH2 degradation in primary cells from one DLBCL patient (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>G and <a href="/doi/suppl/10.1021/acs.jmedchem.1c00460/suppl_file/jm1c00460_si_001.pdf" class="ext-link">Table S1</a>). Furthermore, lymphoma primary cells from 11 cases of lymphoma patients, including two cases of BLBCL, were tested for the efficacy of YM281. Compared to EPZ6438, YM281 increased the activity of caspase-3 and -7 and meanwhile reduced the cell viability observed in adenosine triphosphate (ATP) assays (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>H,I).</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00460/20210716/images/medium/jm1c00460_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00460/20210716/images/large/jm1c00460_0006.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. EZH2 degraders reduced tumor growth in vivo and decreased cell viability in primary lymphoma patient cells. (A) Tumor volume of Balb/c nude mice bearing SU-DHL-6 xenograft administrated intraperitoneally with a vehicle, YM281 (80 mg/kg) or EPZ (42.5 mg/kg), for 3 weeks. (B) Western blot analysis of EZH2 and H3K27me3 levels in the representative SU-DHL-6 model excised tumors. GAPDH was used as a loading control. Representative tumor images were presented. (C) Immunohistochemistry analysis of EZH2, H3K27me3, and Ki67 levels in the representative excised tumors from A. (D) Tumor volume of Balb/c nude mice bearing the Jeko-1 xenograft administrated intraperitoneally with a vehicle, YM281 (100 mg/kg) or EPZ (50 mg/kg), for 30 days. (E) Western blot analysis of EZH2 and H3K27me3 in the representative Jeko-1 model excised tumors. Tubulin was used as a loading control. Representative tumor images were presented. (F) Immunohistochemistry analysis of EZH2 and Ki67 levels in the representative Jeko-1 model excised tumors. (G) Western blot analysis of EZH2 and H3K27me3 levels in a DLBCL patient sample’s cells treated with YM281 at indicated concentrations for 24 h. H3 and tubulin were used as loading controls. (H) Caspase-3/7 activity of patient-derived primary lymphoma cells treated with each compound at indicated concentrations for 48 h in 11 lymphoma patient cases. (I) Quantitation of the present ATP level in patient-derived cells with each compound at indicated concentrations for 48 h in 11 lymphoma patient cases.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00460/20210716/images/large/jm1c00460_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00460&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i8">Chemistry</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_01404" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_01404" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The synthesis of the key intermediates <b>6</b>, <b>7</b>, VHL ligands <b>16</b> and VH032, and CRBN ligands <b>32</b> and <b>33</b> have been previously described (<a class="ref internalNav" href="#sch1" aria-label="Schemes 1">Schemes 1</a>, <a class="ref internalNav" href="#sch2" aria-label="2">2</a>, and <a class="ref internalNav" href="#sch5" aria-label="5">5</a>).<a onclick="showRef(event, 'ref31 ref40 ref41 ref42 ref43'); return false;" href="javascript:void(0);" class="ref ref31 ref40 ref41 ref42 ref43">(31,40−43)</a> As shown in <a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>, compound <b>V2</b> (YM281) was used as an example for the synthesis of series V compounds. The commercially available propane-1,3-diol (<b>17b</b>) was protected by the benzyl group on one side hydroxyl and then reacted with <i>t</i>-butyl bromoacetate. The removal of the benzyl group under catalytic hydrogenation conditions followed by a substitution reaction of hydroxyl with tosyl chloride gave compound <b>19b</b>. Alkylation of 4-hydroxyphenylboronic acid pinacol ester with <b>19b</b> afforded <b>20b</b>, and consequential Suzuki coupling with compound <b>6</b> and the removal of the Boc group under acidic conditions gave the acid <b>21b</b>. The final compound <b>V2</b> (YM281) was obtained via standard amide coupling between acid <b>21b</b> and the amine <b>16</b>. In a similar manner, other V series compounds <b>V1</b>, <b>V3</b> (YM181), <b>V4</b>-<b>V7</b>, and YM620 modified with the varying linker were prepared, as shown in <a class="ref internalNav" href="#sch3" aria-label="Schemes 3">Schemes 3</a> and <a class="ref internalNav" href="#sch4" aria-label="4">4</a>.</div><figure id="sch1" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00460/20210716/images/medium/jm1c00460_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00460/20210716/images/large/jm1c00460_0007.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Intermediate Compounds <b>6</b> and <b>7</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00460/20210716/images/large/jm1c00460_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00460&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) Fe, NH<sub>4</sub>Cl, MeOH, 90 °C; (b) tetrahydro-4<i>H</i>-pyran-4-one, AcOH, Na(AcO)<sub>3</sub>BH, 1,2-dichloroethane; (c) acetaldehyde, AcOH, Na(AcO)<sub>3</sub>BH, 1,2-dichloroethane; (d) Boc<sub>2</sub>O, H<sub>2</sub>, Raney-Ni, MeOH; (e) HCl/MeOH; (f) NaOH, EtOH, 60 °C; (g) HOBt, EDCI, NMM, DMSO; (h) 4-methoxycarbonylphenylboronic acid, K<sub>2</sub>CO<sub>3</sub>, Pd(PPh<sub>3</sub>)<sub>4</sub>, <i>N</i>,<i>N</i>-dimethylformamide (DMF), 90 °C; and (i) NaOH, EtOH, 60 °C. Note: unless stated, reactions underwent at room temperature in all synthetic schemes.</p></p></figure><figure id="sch2" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00460/20210716/images/medium/jm1c00460_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00460/20210716/images/large/jm1c00460_0008.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of VHL Ligands <b>15</b>, <b>16</b>, and <b>VH032</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00460/20210716/images/large/jm1c00460_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00460&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) NaOH, Boc<sub>2</sub>O, MeOH; (b) Pd(OAc)<sub>2</sub>, KOAc, <i>N</i>,<i>N</i>-dimethylacetamide (DMA), 150 °C; (c) HCl/MeOH; (d) <b>12</b>, HATU, <i>N</i>,<i>N</i>-diisopropylethylamine (DIPEA), tetrahydrofuran (THF); (e) HCl/MeOH; (f) <b>14</b>, HATU, DIPEA, DMF; (g) HCl/MeOH; and (h) DIPEA, acetic anhydride, CH<sub>2</sub>Cl<sub>2</sub>.</p></p></figure><figure id="sch3" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00460/20210716/images/medium/jm1c00460_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00460/20210716/images/large/jm1c00460_0009.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Series V Compounds<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00460/20210716/images/large/jm1c00460_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00460&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) BnBr, NaH, THF; (b) <i>tert</i>-butyl bromoacetate, NaOH, TBACl, CH<sub>2</sub>Cl<sub>2</sub>; (c) Pd/C, H<sub>2</sub>, EtOH; (d) TsCl, DMAP, TEA, CH<sub>2</sub>Cl<sub>2</sub>; (e) 4-hydroxyphenylboronic acid pinacol ester, K<sub>2</sub>CO<sub>3,</sub> DMF, 70 °C; (f) <b>6</b>, K<sub>3</sub>PO<sub>4</sub>, Pd(PPh)<sub>4,</sub> DMF, 90 °C; (g) 20% TFA, CH<sub>2</sub>Cl<sub>2</sub>; (h) <b>16</b>, HATU, DIPEA, DMF; (i) NaH, <i>tert</i>-butyl bromoacetate, DMF; (j) TsCl, DMAP, TEA, CH<sub>2</sub>Cl<sub>2</sub>; (k) 4-hydroxyphenylboronic acid pinacol ester, K<sub>2</sub>CO<sub>3</sub>, DMF, 70 °C; (l) <b>6</b>, K<sub>3</sub>PO<sub>4</sub>, Pd(PPh)<sub>4</sub>, DMF, 90 °C; (m) 20% TFA, CH<sub>2</sub>Cl<sub>2</sub>; (n) <b>16</b>, HATU, DIPEA, DMF; (o) diethylamine, CH<sub>2</sub>Cl<sub>2</sub>; (p) Boc<sub>2</sub>O, NaOH, CH<sub>2</sub>Cl<sub>2</sub>; (q) <b>16</b>, HATU, DIPEA, DMF; (r) HCl/MeOH; and (s) <b>7</b>, HATU, DIPEA, DMF.</p></p></figure><figure id="sch4" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00460/20210716/images/medium/jm1c00460_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00460/20210716/images/large/jm1c00460_0010.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of <b>YM620</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00460/20210716/images/large/jm1c00460_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00460&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <b>11</b>, HATU, DIPEA, THF; (b) HCl/MeOH; (c) <b>14</b>, HATU, DIPEA; (d) HCl/MeOH; and (e) <b>21b</b>, HATU, DIPEA, DMF.</p></p></figure><div class="NLM_p">Compound <b>G5</b> was used as an example for the synthesis of series G compounds (<a class="ref internalNav" href="#sch5" aria-label="Scheme 5">Scheme 5</a>). The commercially available octane-1,8-diamine was protected by Boc on one side amine to give <b>37b</b>. Amide coupling with <b>36</b><a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> and subsequent removal of the Boc group under acidic conditions afforded amine <b>38b</b>. The final compound <b>G5</b> was obtained via standard amide coupling between acid <b>7</b> and amine <b>38b</b>. In a similar manner, other series G compound <b>G1–G4</b> and <b>G6</b> modified with the varying linker were prepared.</div><figure id="sch5" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00460/20210716/images/medium/jm1c00460_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00460/20210716/images/large/jm1c00460_0011.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of Series G Compounds<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00460/20210716/images/large/jm1c00460_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00460&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <b>29</b>, NaOAc, AcOH, 120 °C reflux, or <b>30</b>, pyridine, 110 °C; (b) <b>34a–b</b>, DIPEA, DMF 90 °C or <b>34c–d</b>, Na<sub>2</sub>CO<sub>3</sub>, DMF, 80 °C; (c) 20% TFA, CH<sub>2</sub>Cl<sub>2</sub>; (d) <b>7</b>, HATU, DIPEA, DMF; (e) <i>ter</i>t-butyl bromoacetate, K<sub>2</sub>CO<sub>3</sub>, DMF; (f) 20% TFA, CH<sub>2</sub>Cl<sub>2</sub>; (g) HATU, DIPEA, DMF; (h) 20% TFA, CH<sub>2</sub>Cl<sub>2</sub>, 50 °C; and (i) <b>7</b>, HATU, DIPEA, DMF.</p></p></figure></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i14">Discussion and Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_37688" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_37688" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">As widely conceptualized, EZH2 inhibitors that simply suppress EZH2 enzymatic activity exhibit antitumor effects by inhibiting trimethylation of H3K27 and thereby activating tumor suppressor genes.<a onclick="showRef(event, 'ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref6 ref7">(6,7)</a> This class of EZH2 inhibitors reduces H3K27 methylation levels at very low concentrations. However, their therapeutic efficacy varies largely in different types of tumors; for example, DLBCL is the major subtype of lymphomas that are most sensitive to EZH2 inhibitors. Previous research studies indicated that the whole EZH2 protein itself is involved in tumor proliferation.<a onclick="showRef(event, 'ref5 ref10 ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref5 ref10 ref16 ref17">(5,10,16,17)</a> Therefore, abrogating the whole EZH2 protein may have a significant therapeutic advantage over simply inhibiting its enzymatic activity. Chen and co-workers have reported that gambogenic acid derivatives could covalently bind to Cys668 within the EZH2-SET domain, triggering EZH2 degradation to inhibit head and neck cancer cells.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> But the selectivity of gambogenic acid derivatives for EZH2 is questionable.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> Very recently, Jin’s group reported a hydrophobic tagging-based EZH2 degrader (MS1943) that showed better anticancer capacity in triple-negative breast cancer than the parental EZH2 inhibitor.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> Surprisingly, MS1943 was reported to reduce the protein level of SUZ12 without affecting the EED protein level, which is different from our EZH2 PROTACs that reduce the EED protein level as well.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> The concomitant degradation of EED induced by YM281 and YM181 could either result from collateral ubiquitination of EED by the VHL E3 ligase due to its proximity to EZH2 or from the reduced stability of the PRC2 complex to eject the EED subunit after the first EZH2 degradation.<a onclick="showRef(event, 'ref25 ref39 ref47'); return false;" href="javascript:void(0);" class="ref ref25 ref39 ref47">(25,39,47)</a> The exact mechanism remains to be elucidated in the future. More recently, some CRBN-based PROTACs were reported while our revised manuscript was under review.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> However, their compounds remained to be evaluated in the animal study. Meanwhile, PROTAC-based degradation of EED, another subunit of the PRC2 complex, was also reported recently.<a onclick="showRef(event, 'ref37 ref38'); return false;" href="javascript:void(0);" class="ref ref37 ref38">(37,38)</a> However, the reported EED degraders did not show superior in vitro anticancer activities compared with their parental EED inhibitors, whereas more in vivo experiments are also needed to validate their efficacy.</div><div class="NLM_p">In our observation, the therapeutic efficacy of EZH2 inhibitors was limited to DLBCL cell lines among the tested lymphoma cell lines (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A). We wonder whether the direct EZH2 degradation via PROTAC technology could be developed to improve their targeting capacity to the other types of lymphoma cells. In our current study, we developed PROTAC-based EZH2 degraders and investigated their efficiency of EZH2 degradation and therapeutic efficacy in various types of lymphoma in vitro and in vivo. Our study revealed that only VHL-targeting compounds enabled the EZH2 degradation with an appropriate linker at 7 or 9 atoms length. Compared to the parental EZH2 inhibitor EPZ6438, our two best EZH2 degraders YM181 and YM281 selectively degraded EZH2 over EZH1, and they exhibited effective antiproliferative activity both in DLBCL and other types of lymphoma cell lines. Furthermore, the EZH2 degrader showed an apparent advantage to prevent in vivo tumor growth in lymphoma xenografts without obvious toxicity at the efficacious doses.</div><div class="NLM_p last">However, the incomplete EZH2 degradation and the modest cellular potencies for YM281 and YM181 in the inhibition of cell viability at low concentrations suggest that there is still room for further optimization. In the future, a structure and activity relationship study on different linker scaffoldings with the same linking length as YM181 and YM281 may be warranted to obtain more potent EZH2 degraders. Meanwhile, cancer cells that may not depend on EZH2 for their tumorigenesis, for example, pancreatic cancer cell AsPC1 and lung cancer cell NCI-H460, are not sensitive to YM181 and YM181, although the compounds were able to decrease their EZH2 levels (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00460/suppl_file/jm1c00460_si_001.pdf" class="ext-link">Figure S6</a>). By measuring cell permeability with Caco-2 cells, both YM181 and YM281 showed their apparent permeability ability largely compromised compared to the parental EHZ2 inhibitor EPZ6438 (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00460/suppl_file/jm1c00460_si_001.pdf" class="ext-link">Table S2</a>), indicating the importance to improve their oral bioavailability through further structural optimization. Overall, our results demonstrate that EZH2 degraders may have better therapeutic potential than EZH2 inhibitors against lymphomas. The exact mechanism of action and the application of our EZH2 degraders in other cancers are under investigation.</div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_81064" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_81064" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Chemistry: General Experiment and Information</h3><div class="NLM_p">Synthesis details for the key final compounds are described here. All solvents were commercially available and were used without further purification unless stated. The chemicals used were either purchased from commercial sources or prepared according to literature procedures. The <sup>1</sup>H and <sup>13</sup>C nuclear magnetic resonance (NMR) spectra were recorded on a Bruker Avance spectrometer 400 at 400 MHz and 100 MHz or a Bruker Avance spectrometer 500 at 500 MHz and 125 MHz respectively. Chemical shifts are given in ppm (δ) referenced to CDCl<sub>3</sub> with 7.26 for <sup>1</sup>H and 77.10 for <sup>13</sup>C, and to <i>d</i><sub>6</sub>-DMSO with 2.50 for <sup>1</sup>H and 39.5 for <sup>13</sup>C. In the case of multiplet, the signals are reported as intervals. Signals are abbreviated as follows: s, singlet; d, doublet; t, triplet; q, quartet; and m, multiplet. Coupling constants are expressed in hertz. High-resolution mass spectra (HRMS) were recorded on a BRUKER VPEXII spectrometer (the ESI mode). The progress of the reactions was monitored by thin-layer chromatography on a glass plate coated with silica gel with a fluorescent indicator (GF254). Flash column chromatography was performed on silica gel (200–300 mesh). An Agilent 1100 HPLC system equipped with an Eclipse XDB-C18 column was used to determine the purity of all of the final key products. Gradient elution: 10–90% MeOH against H<sub>2</sub>O with 0.1% TFA over 15 min (the ratio of MeOH/H<sub>2</sub>O from 10 to 90% in 8 min, and finally 90% for 7 min) at a flow rate of 1.0 mL/min; detection wavelength: 254 nm. All biologically tested compounds had a purity of more than 95%.</div><div id="sec5_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i17" class="anchor-spacer"></div><h4 class="article-section__title" id="_i17"> 5-Bromo-<i>N</i>-((4,6-dimethyl-2-oxo-1,2-dihydropyridin-3-yl)methyl)-3-(ethyl(tetrahydro-2<i>H</i>-pyran-4-yl)amino)-2-methylbenzamide (<b>6</b>)<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a></h4><div class="NLM_p">To a solution of methyl 5-bromo-2-methyl-3-nitrobenzoate (5 g, 18.24 mmol) in MeOH (50 mL), Fe (5.09 g, 91.22 mmol) and NH<sub>4</sub>Cl (1.95 g, 36.49 mmol) were added sequentially. The mixture was stirred at 90 °C overnight. In the end, the solid precipitated was filtered off and washed with MeOH (3 × 80 mL). The combined filtrate was dried and concentrated under reduced pressure to give the desired crude <b>2</b> as a yellow liquid (4.34 g, 99.6% yield). To a solution of <b>2</b> (4.34 g, 17.78 mmol) in 1,2-dichloroethane (50 mL) were added tetrahydro-4<i>H</i>-pyran-4-one (2.46 mL, 26.67 mmol) and AcOH (6.10 mL, 106.68 mmol). The mixture was stirred at r.t. for 0.5 h before Na(AcO)<sub>3</sub>BH (11.31 g, 53.34 mmol) was added at 0 °C. The mixture was warmed to r.t. and stirred for 24 h. EtOAc and water were added, and the aqueous phase was extracted with EtOAc twice. The combined organic phases were dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. The residue was purified by flash column chromatography with EtOAc/PE (10–35%) to afford the product as a yellow solid (5.34 g, 91.5% yield). To a solution of the product (5.34 g, 16.27 mmol) in 1,2-dichloroethane (50 mL) were added acetaldehyde (2.76 mL, 48.81 mmol) and AcOH (5.58 mL, 97.62 mmol). The mixture was stirred at r.t. for 0.5 h before Na(AcO)<sub>3</sub>BH (10.35 g, 48.81 mmol) was added at 0 °C. The mixture was warmed to r.t. and stirred for 24 h before EtOAc (200 mL) and water (100 mL) were added. The aqueous layer was extracted with EtOAc (2 × 150 mL). The combined organic phases were dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. The residue was purified by flash column chromatography with EtOAc/PE (10–35%) to afford <b>3</b> as a yellow solid (5.18 g, 89.4% yield).</div><div class="NLM_p">To a solution of 4,6-dimethyl-2-oxo-1,2-dihydro-3-pyridinecarbonitrile <b>4</b> (3.0 g, 20.25 mmol) in MeOH (120 mL) was added a catalytic amount of Raney nickel and Boc<sub>2</sub>O (5.3 g, 24.30 mmol). The mixture was purged and refilled with H<sub>2</sub> three times and stirred at r.t. for 12 h under H<sub>2</sub>. After filtration and concentration, the residue was purified by flash column chromatography with MeOH/CH<sub>2</sub>Cl<sub>2</sub> (2–5%) to afford a white solid (5.1 g, 99.6% yield). Acetyl chloride (5.74 mL, 80.68 mmol) was added to MeOH (28 mL) at 0 °C for 1 h to prepare a fresh HCl solution in methanol, and then the above-obtained compound (5.09 g, 20.17 mmol) was added. The solution was warmed to r.t., stirred for 3 h, and finally concentrated under reduced pressure to give <b>5</b> (4.1 g, 100% yield) without further purification.</div><div class="NLM_p">To a solution of <b>3</b> (5.18 g, 14.54 mmol) in ethanol (50 mL) was added aqueous NaOH solution. The reaction mixture was stirred at 60 °C for 3 h before it was concentrated under reduced pressure. Then, CH<sub>2</sub>Cl<sub>2</sub> (100 mL) and water (150 mL) were added, and the mixture was washed with CH<sub>2</sub>Cl<sub>2</sub> (2 × 100 mL). The aqueous phase was acidified by adding aq. HCl (1 M) until pH = 3–4. Then, the aqueous solution was extracted with CH<sub>2</sub>Cl<sub>2</sub> (3 × 200 mL). The combined organic phases were concentrated under reduced pressure to give the intermediate acid as a white solid (4.55 g, 91.4% yield).</div><div class="NLM_p last">To a solution of <b>5</b> (3.03 g, 19.93 mmol) and the above-obtained acid (4.55 g, 13.29 mmol) in DMSO (50 mL), were added HOBt (2.15 g, 15.94 mmol), EDCI (3.06 g, 15.94 mmol), and NMM (8.76 mL, 79.72 mmol). The mixture was stirred at r.t. overnight before EtOAc (150 mL) and water (100 mL) were added. The aqueous phase was extracted with EtOAc (2 × 100 mL). The combined organic phases were dried over anhydrous Na<sub>2</sub>SO<sub>4</sub> and concentrated under reduced pressure. The residue was purified by silica gel flash column chromatography with EtOAc/PE (10–35%) to afford <b>6</b> as a yellow solid (3.7 g, 58.3% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 11.12 (s, 1H), 7.22 (d, <i>J</i> = 1.8 Hz, 1H), 7.18 (d, <i>J</i> = 1.8 Hz, 1H), 7.12 (t, <i>J</i> = 5.6 Hz, 1H), 5.95 (s, 1H), 4.52 (d, <i>J</i> = 5.9 Hz, 2H), 3.95 (d, <i>J</i> = 11.4 Hz, 2H), 3.35–3.27 (m, 2H), 3.02 (q, <i>J</i> = 6.9 Hz, 2H), 2.93 (m, 1H), 2.39 (s, 3H), 2.24 (d, <i>J</i> = 2.3 Hz, 6H), 1.66 (m, 4H), 0.85 (t, <i>J</i> = 7.0 Hz, 3H).</div></div><div id="sec5_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18"> 3′-(((4,6-Dimethyl-2-oxo-1,2-dihydropyridin-3-yl)methyl)carbamoyl)-5′-(ethyl(tetrahydro-2<i>H</i>-pyran-4-yl)amino)-4′-methyl-[1,1′-biphenyl]-4-carboxylic Acid (<b>7</b>)<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a></h4><div class="NLM_p last">To a solution of <b>6</b> (1 g, 2.10 mmol) in DMF (20 mL) were added 4-methoxycarbonylphenylboronic acid (453 mg, 2.52 mmol), K<sub>2</sub>CO<sub>3</sub> (1.16 g, 8.40 mmol), and Pd(PPh<sub>3</sub>)<sub>4</sub> (121 mg, 104.95 μmol). The solution was purged and refilled with argon three times. Then, the solution was stirred at 90 °C for 9 h before EtOAc (70 mL) and water (30 mL) were added at r.t. The aqueous phase was extracted with EtOAc (2 × 70 mL). The combined organic phases were dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. The residue was purified by flash column chromatography with MeOH/CH<sub>2</sub>Cl<sub>2</sub> (0–10%) to afford the coupled product as a yellow solid (0.82 g, 73.5% yield). The obtained product was hydrolyzed under NaOH in EtOH at 60 °C to give the final acid <b>7</b>. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 12.96 (s, 1H), 8.17 (d, <i>J</i> = 8.2 Hz, 2H), 7.69 (m, 3H), 7.51 (s, 1H), 7.41 (s, 1H), 6.12 (s, 1H), 4.51 (d, <i>J</i> = 5.8 Hz, 2H), 3.96 (d, <i>J</i> = 11.3 Hz, 2H), 3.33 (t, <i>J</i> = 11.0 Hz, 2H), 3.15 (s, 2H), 3.06 (s, 1H), 2.50 (d, <i>J</i> = 15.6 Hz, 6H), 2.37 (s, 3H), 1.72 (m, 4H), 0.92 (t, <i>J</i> = 6.4 Hz, 3H).</div></div><div id="sec5_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> <i>tert</i>-Butyl((<i>S</i>)-1-((2<i>S</i>,4<i>R</i>)-4-hydroxy-2-((4-(4-methylthiazol-5-yl)benzyl)carbamoyl)pyrrolidin-1-yl)-3,3-dimethyl-1-oxobutan-2-yl)carbamate (<b>15</b>)<a onclick="showRef(event, 'ref31 ref43'); return false;" href="javascript:void(0);" class="ref ref31 ref43">(31,43)</a></h4><div class="NLM_p">To a solution of (4-bromophenyl)methanamine <b>8</b> (6.79 mL, 53.75 mmol) in MeOH (80 mL) were added Boc<sub>2</sub>O (18.52 mL, 80.62 mmol) and NaOH (107.5 mmol, 1 M) at 0 °C. The reaction mixture was warmed to r.t. and stirred for 5 h. MeOH was removed under reduced pressure, and the residue was acidified by adding aq. HCl until pH = 5. The aqueous solution was extracted with CH<sub>2</sub>Cl<sub>2</sub> (3 × 150 mL). The combined organic phases were washed with brine, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, and filtered. The concentration under reduced pressure gave a colorless liquid (14.4 g, 93.6% yield), which was used in the next step without further purification.</div><div class="NLM_p last">To a solution of the above-obtained liquid (14.4 g, 50.42 mmol) in DMA (50 mL) under argon were added Pd(AcO)<sub>2</sub> (226.45 mg, 1.01 mmol), potassium acetate (9.9 g, 100.86 mmol), and 4-methylthiazole <b>9</b> (4.59 mL, 50.43 mmol). The reaction mixture was stirred at 150 °C for 10 h. Then, CH<sub>2</sub>Cl<sub>2</sub> (150 mL) and water (100 mL) were added. The aqueous phase was extracted with CH<sub>2</sub>Cl<sub>2</sub> (2 × 150 mL). The combined organic phases were dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, concentrated under reduced pressure to give <b>10</b> as a yellow liquid (14.35 g, 93.7% yield), which was used in the next step without further purification. After the removal of the Boc group in <b>10</b> (6.5 g. 21.35 mmol) under acidic conditions, a free amine residue (5.14 g, 20.77 mmol) was obtained. To a stirred solution of <b>12</b> (5.92 g, 25.62 mmol) and DIPEA (10.59 mL, 64.05 mmol) in anhydrous THF (75 mL) at 0 °C were added the above-obtained amine (5.14 g, 20.77 mmol) and HATU (9.74, 25.62 mmol). The resulting mixture was warmed to r.t. and stirred for 2 h. After THF was removed under reduced pressure, CH<sub>2</sub>Cl<sub>2</sub> (120 mL) and water (80 mL) were added. The aqueous phase was extracted with CH<sub>2</sub>Cl<sub>2</sub> (2 × 100 mL). The combined organic phases were dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure to give <b>13</b> as a yellow liquid (8.67 g, 82.5% yield), which was used in the next step without further purification. After the removal of the Boc group in <b>13</b> (5 g, 11.98 mmol) under acidic conditions, the obtained free amine residue was added to the stirred solution of <i>N</i>-Boc-<span class="smallcaps smallerCapital">l</span>-<i>tert</i>-Leucine <b>14</b> (2.76 g, 11.93 mmol), DIPEA (5.91 mL, 35.78 mmol), and HATU (6.8 g, 17.89 mmol) in DMF (45 mL). The reaction mixture was stirred at r.t. for 5 h. CH<sub>2</sub>Cl<sub>2</sub> (120 mL) and water (60 mL) were added, and the aqueous phase was extracted with CH<sub>2</sub>Cl<sub>2</sub> (2 × 100 mL). The combined organic phases were dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. The residue was purified by column chromatography on silica gel with MeOH/CH<sub>2</sub>Cl<sub>2</sub> (0–10%) to give <b>15</b> as a yellow solid (4.5 g, 70.8% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.68 (s, 1H), 7.47 (m, 1H), 7.33 (m, 4H), 5.20 (d, <i>J</i> = 9.0 Hz, 1H), 4.74 (m, 1H), 4.53(m, 2H), 4.30 (dd, <i>J</i> = 15.0, 5.1 Hz, 1H), 4.17 (d, <i>J</i> = 9.1 Hz, 1H), 4.02 (d, <i>J</i> = 11.3 Hz, 1H), 3.60 (dd, <i>J</i> = 11.3, 3.4 Hz, 1H), 2.50 (s, 3H), 2.50 (m, 1H), 2.10 (m, 1H), 1.40 (s, 9H), 0.91 (s, 9H).</div></div><div id="sec5_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-Acetamido-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (<b>VH032</b>)<a onclick="showRef(event, 'ref31 ref43'); return false;" href="javascript:void(0);" class="ref ref31 ref43">(31,43)</a></h4><div class="NLM_p last">Removal of the Boc group in <b>15</b> (50 mg, 94.22 mmol) under acidic conditions afforded amine compound <b>16</b>. To a solution of <b>16</b> (40 mg, 94.22 μmol) in CH<sub>2</sub>Cl<sub>2</sub> was added DIPEA (46.06 μL, 278.70 μmol) at 0 °C. After the mixture was stirred for 0.5 h at r.t., acetic anhydride (13.17 μL, 139.35 μmol) was added. The mixture was continued to stir at r.t. for 2 h. CH<sub>2</sub>Cl<sub>2</sub> (10 mL) and water (5 mL) were added, and the mixture was extracted with CH<sub>2</sub>Cl<sub>2</sub> (2 × 10 mL). The combined organic phases were dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. The residue was purified by column chromatography on silica gel with MeOH/CH<sub>2</sub>Cl<sub>2</sub> (0–10%) to give <b>VH032</b> as a yellow solid (32 mg, 67.7% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.68 (s, 1H), 7.36 (q, <i>J</i> = 8.3 Hz, 4H), 6.32 (d, <i>J</i> = 8.6 Hz, 1H), 4.70 (t, <i>J</i> = 8.0 Hz, 1H), 4.65–4.47 (m, 3H), 4.33 (dd, <i>J</i> = 15.0, 5.1 Hz, 1H), 4.10 (d, <i>J</i> = 11.5 Hz, 1H), 3.61 (dd, <i>J</i> = 11.4, 3.3 Hz, 1H), 2.51 (s, 3H), 2.51 (m, 1H), 2.14 (dd, <i>J</i> = 13.6, 8.4 Hz, 1H), 1.98 (s, 3H), 0.93 (s, 9H).</div></div><div id="sec5_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> 2-(2,6-Dioxopiperidin-3-yl)-4-fluoroisoindoline-1,3-dione (<b>32</b>)<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a></h4><div class="NLM_p last">To a suspension of 4-fluoroisobenzofuran-1,3-dione <b>29</b> (1 g, 6.02 mmol) and 3-aminopiperidine-2,6-dione hydrochloride <b>31</b> (0.99 g, 6.02 mmol) in AcOH (20 mL) was added sodium acetate (0.59 g, 7.22 mmol). The reaction mixture was heated to 120 °C for 12 h. After cooling to r.t., AcOH was removed under reduced pressure. The residue was purified by flash column chromatography with MeOH/CH<sub>2</sub>Cl<sub>2</sub> (0–10%) to give <b>32</b> as a white solid (1.33 g, 79.7% yield). <sup>1</sup>H NMR (400 MHz, <i>d</i><sub>6</sub>-DMSO) δ 11.14 (s, 1H), 7.98–7.91 (m, 1H), 7.79 (d, <i>J</i> = 7.3 Hz, 1H), 7.74 (t, <i>J</i> = 8.9 Hz, 1H), 5.16 (dd, <i>J</i> = 12.9, 5.4 Hz, 1H), 2.89 (ddd, <i>J</i> = 17.2, 14.0, 5.5 Hz, 1H), 2.68–2.52 (m, 2H), 2.06 (ddd, <i>J</i> = 10.7, 5.5, 3.1 Hz, 1H).</div></div><div id="sec5_1_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> 2-(2,6-Dioxopiperidin-3-yl)-4-hydroxyisoindoline-1,3-dione (<b>33</b>)<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a></h4><div class="NLM_p last">3-Hydroxyphthalic anhydride <b>30</b> (3 g, 18.28 mmol) and 3-aminopiperidine-2,6-dione hydrochloride <b>31</b> (2.99 g, 18.28 mmol) were dissolved in pyridine (80 mL) and heated to 110 °C. After 14 h, the mixture was cooled to r.t. and concentrated under reduced pressure. The residue was purified by flash column chromatography with MeOH/CH<sub>2</sub>Cl<sub>2</sub> (0–10%) to give <b>33</b> as a tan solid (4.02 g, 80.2% yield). <sup>1</sup>H NMR (500 MHz, <i>d</i><sub>6</sub>-DMSO) δ 11.18 (s, 1H), 11.09 (s, 1H), 7.65 (t, <i>J</i> = 7.7 Hz, 1H), 7.32 (d, <i>J</i> = 7.0 Hz, 1H), 7.25 (d, <i>J</i> = 8.4 Hz, 1H), 5.07 (dd, <i>J</i> = 12.8, 5.2 Hz, 1H), 2.93–2.83 (m, 1H), 2.55 (dd, <i>J</i> = 27.8, 11.8 Hz, 2H), 2.01 (dd, <i>J</i> = 13.6, 7.8 Hz, 1H).</div></div><div id="sec5_1_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-(2-(3-((3′-(((4,6-Dimethyl-2-oxo-1,2-dihydropyridin-3-yl)methyl)carbamoyl)-5′-(ethyl(tetrahydro-2<i>H</i>-pyran-4-yl)amino)-4′-methyl-[1,1′-biphenyl]-4-yl)oxy)propoxy)acetamido)-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (<b>V2</b>, <b>YM281</b>)</h4><div class="NLM_p">To a solution of 1,3-propanediol <b>17b</b> (2 g, 26.28 mmol) in THF (10 mL) was added NaH (60%, 525.61 mg, 131.14 mmol) at 0 °C. The reaction mixture was stirred for 1 h before BnBr (1.04 mL, 8.76 mmol) was added at 0 °C. The reaction mixture was gradually warmed to r.t., stirred overnight, and quenched with saturated aq. NH<sub>4</sub>Cl (30 mL). The mixture was extracted with EtOAc (3 × 80 mL). The combined organic phases were dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. The residue was purified by flash column chromatography with EtOAc/PE (10–50%) to afford 3-(benzyloxy)propan-1-ol as a colorless oil (0.82 g, 56.3% yield).</div><div class="NLM_p">A mixture of 3-(benzyloxy)propan-1-ol (0.81 g, 4.87 mmol), <i>tert</i>-butyl bromoacetate (2.84 mL, 19.5 mmol), aqueous sodium hydroxide (35%, 20 mL), and TBACl (1.38 mL, 4.87 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (20 mL) was stirred vigorously at r.t. overnight. The reaction mixture was poured into water (15 mL) and then was extracted with CH<sub>2</sub>Cl<sub>2</sub> (3 × 20 mL). The combined organic phases were washed with water (15 mL × 2) and brine (2 × 15 mL), dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, and concentrated under reduced pressure. The residue was purified by flash column chromatography with EtOAc/PE (0–20%) to afford <b>18b</b> as a colorless oil (1.2 g, 87.8% yield).</div><div class="NLM_p">A mixture of <b>18b</b> (670 mg, 2.39 mmol) and palladium on carbon (10%, 70 mg) in ethanol (8 mL) was stirred at r.t. overnight under a hydrogen atmosphere. The mixture was filtered and washed with EtOAc (3 × 30 mL). The combined filtrate was concentrated under reduced pressure to afford a hydroxyl product as a colorless oil, which was used in the next step without further purification. The hydroxyl product (454 mg, 2.39 mmol) was dissolved in CH<sub>2</sub>Cl<sub>2</sub> (5 mL), followed by the addition of 4-toluenesulfonyl chloride (901 mg, 4.73 mmol), TEA (658 μL, 4.73 mmol), and DMAP (145 mg, 1.18 mmol). The reaction mixture was stirred at r.t. for 3 h before it was diluted with CH<sub>2</sub>Cl<sub>2</sub> (100 mL). The organic phase was washed with water (3 × 10 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated under reduced pressure. The residue was purified by flash column chromatography with EtOAc/PE (20–50%) to afford <b>19b</b> as a colorless oil (450 mg, 54.8% yield).</div><div class="NLM_p">To a solution of <b>19b</b> (150 mg, 435 μmol) in DMF (2 mL) were added 4-hydroxyphenylboronic acid pinacol ester (144 mg, 653 μmol) and K<sub>2</sub>CO<sub>3</sub> (120 mg, 871 μmol). The reaction mixture was stirred at 70 °C for 5 h and then cooled to r.t. before EtOAc (30 mL) and water (10 mL) were added. The mixture was extracted with EtOAc (2 × 30 mL). The combined organic phases were dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. The residue was purified by flash column chromatography with EtOAc/PE (20–50%) to afford <b>20b</b> as a colorless solid (145 mg, 84.9% yield).</div><div class="NLM_p">To a solution of <b>6</b> (121.4 mg, 254.9 μmol) in DMF (2 mL) were added <b>20b</b> (100 mg, 254.91 μmol), K<sub>2</sub>CO<sub>3</sub> (108.2 mg, 509 μmol), and Pd(PPh<sub>3</sub>)<sub>4</sub> (14.7 mg, 12.75 μmol). The mixture was purged and refilled with argon three times. The mixture was stirred at 90 °C for 9 h before EtOAc (30 mL) and water (10 mL) were added at r.t. The aqueous phase was extracted with EtOAc (2 × 30 mL). The combined organic phases were dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. The residue was purified by flash column chromatography with MeOH/CH<sub>2</sub>Cl<sub>2</sub> (0–10%) to afford the coupled product as a yellow solid (120 mg, 71.1% yield).</div><div class="NLM_p last">Then, the obtained product (120 mg, 181.32 μmol) was dissolved in CH<sub>2</sub>Cl<sub>2</sub> (1 mL), and TFA (135 μL, 1.81 mmol) was added before the mixture was stirred at r.t. for 3 h. The concentration of the solvent provided acid <b>21b</b>, which was used for the following steps without further purification. To a stirred solution of <b>21b</b> in DMF (2 mL), amine <b>16</b> (92.98 mg, 215.94 mmol), DIPEA (178.45 μL, 1.08 mmol), and HATU (68.42 mg 179.95 μmol) were added sequentially. The resulting mixture was stirred at r.t. for 10 h, and then brine (5 mL) was added and extracted with EtOAc (3 × 15 mL). The combined organic phases were washed with brine, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. The residue was purified by flash column chromatography with MeOH/CH<sub>2</sub>Cl<sub>2</sub> (0–10%) to give <b>V2</b> (<b>YM281</b>) as a white powder (118 mg, 64.4% yield). <sup>1</sup>H NMR (500 MHz, <i>d</i><sub>6</sub>-DMSO) δ 11.47 (s, 1H), 8.95 (s, 1H), 8.60 (s, 1H), 8.18 (s, 1H), 7.51 (d, <i>J</i> = 7.8 Hz, 2H), 7.44 (d, <i>J</i> = 8.8 Hz, 1H), 7.38 (m, 4H), 7.32 (s, 1H), 7.16 (s, 1H), 7.01 (d, <i>J</i> = 7.4 Hz, 2H), 5.85 (s, 1H), 5.17 (s, 1H), 4.57 (d, <i>J</i> = 9.3 Hz, 1H), 4.46 (t, <i>J</i> = 8.1 Hz, 1H), 4.38 (d, <i>J</i> = 15.9 Hz, 2H), 4.33–4.21 (m, 3H), 4.10 (s, 2H), 3.97 (s, 2H), 3.81 (d, <i>J</i> = 9.9 Hz, 2H), 3.71–3.58 (m, 4H), 3.23 (t, <i>J</i> = 11.2 Hz, 2H), 3.06 (d, <i>J</i> = 6.3 Hz, 2H), 2.99 (s, 1H), 2.41 (s, 3H), 2.23 (s, 3H), 2.20 (s, 3H), 2.10 (s, 3H), 2.07–1.98 (m, 3H), 1.91 (s, 1H), 1.64 (d, <i>J</i> = 11.2 Hz, 2H), 1.50 (d, <i>J</i> = 10.6 Hz, 2H), 0.94 (s, 9H), 0.81 (t, <i>J</i> = 5.9 Hz, 3H). <sup>13</sup>C NMR (125 MHz, <i>d</i><sub>6</sub>-DMSO) δ 171.7, 169.2, 169.1, 168.4, 163.0, 158.1, 151.4, 149.5, 148.8, 147.7, 142.7, 139.6, 139.4, 136.9, 132.3, 131.9, 131.1, 129.7, 128.9, 128.7, 128.2, 127.7, 127.5, 122.5, 121.6, 120.5, 114.9, 107.4, 69.5, 68.9, 67.7, 66.3, 64.6, 58.8, 57.9, 56.6, 55.7, 41.7, 41.2, 37.9, 35.8, 34.9, 30.3, 29.0, 26.2, 19.0, 18.2, 15.9, 14.5, 12.7. HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>56</sub>H<sub>72</sub>N<sub>7</sub>O<sub>9</sub>S [M + H]<sup>+</sup>: 1018.5107, found: 1018.5121. Purity: 98.1%.</div></div><div id="sec5_1_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-(2-(2-((3′-(((4,6-Dimethyl-2-oxo-1,2-dihydropyridin-3-yl)methyl)carbamoyl)-5′-(ethyl(tetrahydro-2<i>H</i>-pyran-4-yl)amino)-4′-methyl-[1,1′-biphenyl]-4-yl)oxy)ethoxy)acetamido)-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (<b>V1</b>)</h4><div class="NLM_p last"><b>V1</b> was prepared following the procedure for the synthesis of compound <b>V2</b> (<b>YM281</b>). <sup>1</sup>H NMR (400 MHz, <i>d</i><sub>6</sub>-DMSO) δ 11.47 (s, 1H), 8.92 (s, 1H), 8.63 (t, <i>J</i> = 5.9 Hz, 1H), 8.17 (t, <i>J</i> = 4.8 Hz, 1H), 7.53–7.46 (m, 3H), 7.42 (d, <i>J</i> = 8.3 Hz, 2H), 7.36 (d, <i>J</i> = 8.2 Hz, 2H), 7.29 (s, 1H), 7.15 (s, 1H), 7.09 (d, <i>J</i> = 8.7 Hz, 2H), 5.85 (s, 1H), 5.18 (d, <i>J</i> = 3.4 Hz, 1H), 4.61 (d, <i>J</i> = 9.6 Hz, 1H), 4.51–4.34 (m, 3H), 4.30 (d, <i>J</i> = 4.8 Hz, 2H), 4.26 (d, <i>J</i> = 5.5 Hz, 1H), 4.24–4.12 (m, 2H), 4.06 (s, 2H), 3.86 (t, <i>J</i> = 4.3 Hz, 2H), 3.80 (d, <i>J</i> = 11.7 Hz, 2H), 3.69 (dd, <i>J</i> = 10.5, 3.7 Hz, 1H), 3.62 (d, <i>J</i> = 10.8 Hz, 1H), 3.22 (t, <i>J</i> = 11.0 Hz, 2H), 3.08–3.01 (m, 2H), 3.01–2.92 (m, 1H), 2.38 (s, 3H), 2.22 (s, 3H), 2.21 (s, 3H), 2.10 (s, 3H), 2.09–2.02 (m, 1H), 1.96–1.88 (m, 1H), 1.63 (d, <i>J</i> = 11.4 Hz, 2H), 1.55–1.45 (m, 2H), 0.94 (s, 9H), 0.79 (t, <i>J</i> = 7.0 Hz, 3H). <sup>13</sup>C NMR (101 MHz, <i>d</i><sub>6</sub>-DMSO) δ 171.8, 169.1, 168.4, 163.0, 158.0, 151.3, 149.5, 148.8, 147.6, 142.7, 139.5, 139.4, 136.8, 132.4, 131.9, 131.1, 129.6, 128.8, 128.6, 128.1, 127.6, 127.4, 122.5, 121.6, 120.5, 115.0, 107.4, 69.6, 69.5, 68.9, 70.0, 66.3, 58.8, 57.8, 56.6, 55.7, 41.7, 41.2, 37.9, 35.8, 34.9, 30.3, 26.2, 18.9, 18.9, 18.2, 15.9, 14.5, 12.7. HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>55</sub>H<sub>70</sub>N<sub>7</sub>O<sub>9</sub>S [M + H]<sup>+</sup>: 1004.4805, found: 1004.4805. Purity: 98.1%.</div></div><div id="sec5_1_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-(2-(2-(2-((3′-(((4,6-Dimethyl-2-oxo-1,2-dihydropyridin-3-yl)methyl)carbamoyl)-5′-(ethyl(tetrahydro-2<i>H</i>-pyran-4-yl)amino)-4′-methyl-[1,1′-biphenyl]-4-yl)oxy)ethoxy)ethoxy)acetamido)-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (<b>V3</b>, <b>YM181</b>)</h4><div class="NLM_p last"><b>V3</b> was prepared following the procedure for the synthesis of compound <b>V2</b> (<b>YM281</b>). <sup>1</sup>H NMR (500 MHz, <i>d</i><sub>6</sub>-DMSO) δ 11.46 (s, 1H), 8.95 (s, 1H), 8.59 (t, <i>J</i> = 5.6 Hz, 1H), 8.17 (s, 1H), 7.51 (d, <i>J</i> = 8.1 Hz, 2H), 7.45 (d, <i>J</i> = 9.5 Hz, 1H), 7.42–7.36 (m, 4H), 7.33 (s, 1H), 7.17 (s, 1H), 6.98 (d, <i>J</i> = 8.0 Hz, 2H), 5.85 (s, 1H), 5.16 (s, 1H), 4.57 (d, <i>J</i> = 9.5 Hz, 1H), 4.45 (t, <i>J</i> = 8.0 Hz, 1H), 4.38 (dd, <i>J</i> = 16.4, 6.1 Hz, 2H), 4.29 (d, <i>J</i> = 4.2 Hz, 2H), 4.25 (dd, <i>J</i> = 16.0, 5.3 Hz, 1H), 4.13 (s, 2H), 3.97 (d, <i>J</i> = 16.9 Hz, 2H), 3.80 (t, <i>J</i> = 14.1 Hz, 4H), 3.70–3.59 (m, 4H), 3.23 (t, <i>J</i> = 11.4 Hz, 2H), 3.06 (d, <i>J</i> = 6.6 Hz, 2H), 3.01 (d, <i>J</i> = 10.2 Hz, 1H), 2.42 (s, 3H), 2.23 (s, 3H), 2.20 (s, 3H), 2.10 (s, 3H), 2.05 (d, <i>J</i> = 7.3 Hz, 1H), 1.98 (dd, <i>J</i> = 17.1, 10.8 Hz, 1H), 1.91 (dd, <i>J</i> = 14.8, 6.5 Hz, 1H), 1.65 (d, <i>J</i> = 11.4 Hz, 2H), 1.57–1.44 (m, 2H), 0.94 (s, 9H), 0.82 (t, <i>J</i> = 6.7 Hz, 3H). <sup>13</sup>C NMR (101 MHz, <i>d</i><sub>6</sub>-DMSO) δ 171.8, 169.2, 168.7, 163.1, 158.1, 151.4, 149.6, 147.8, 142.8, 139.5, 139.4, 136.9, 132.3, 132.0, 131.2, 129.7, 128.7, 127.7, 127.5, 122.6, 121.6, 120.5, 114.9, 107.5, 70.5, 69.8, 69.7, 69.1, 68.9, 67.2, 66.4, 58.8, 57.9, 56.6, 55.8, 41.7, 37.9, 35.8, 34.9, 31.3, 30.4, 29.0, 26.2, 25.2, 22.1, 21.4, 19.0, 18.2, 15.9, 14.5, 14.0, 12.7. HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>57</sub>H<sub>74</sub>N<sub>7</sub>O<sub>10</sub>S [M + H]<sup>+</sup>: 1048.5212, found: 1048.5216. Purity: 96.3%.</div></div><div id="sec5_1_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> <i>N</i><sup>3</sup>-((4,6-Dimethyl-2-oxo-1,2-dihydropyridin-3-yl)methyl)-5-(ethyl(tetrahydro-2<i>H</i>-pyran-4-yl)amino)-<i>N</i><sup><i>4</i></sup>′-(2-(2-(2-(((<i>S</i>)-1-((2<i>S</i>,4<i>R</i>)-4-hydroxy-2-((4-(4-methylthiazol-5-yl)benzyl)carbamoyl)pyrrolidin-1-yl)-3,3-dimethyl-1-oxobutan-2-yl)amino)-2-oxoethoxy)ethoxy)ethyl)-4-methyl-[1,1′-biphenyl]-3,4′-dicarboxamide (<b>V4</b>)</h4><div class="NLM_p last"><b>V4</b> was prepared following the procedure for the synthesis of compound <b>V2</b> (<b>YM281</b>). <sup>1</sup>H NMR (400 MHz, <i>d</i><sub>6</sub>-DMSO) δ 11.44 (s, 1H), 8.95 (s, 1H), 8.60–8.50 (m, 2H), 8.20 (t, <i>J</i> = 5.0 Hz, 1H), 7.91 (d, <i>J</i> = 8.4 Hz, 2H), 7.70 (d, <i>J</i> = 8.4 Hz, 2H), 7.44 (d, <i>J</i> = 9.9 Hz, 2H), 7.38 (brs, 4H), 7.29 (s, 1H), 5.86 (s, 1H), 5.16 (d, <i>J</i> = 3.5 Hz, 1H), 4.57 (d, <i>J</i> = 9.6 Hz, 1H), 4.47 (d, <i>J</i> = 8.1 Hz, 1H), 4.35 (d, <i>J</i> = 6.0 Hz, 2H), 4.30 (d, <i>J</i> = 4.8 Hz, 2H), 4.25 (dd, <i>J</i> = 15.8, 5.7 Hz, 1H), 3.97 (s, 2H), 3.82 (d, <i>J</i> = 10.3 Hz, 2H), 3.70–3.55 (m, 8H), 3.46 (d, <i>J</i> = 5.8 Hz, 2H), 3.25 (t, <i>J</i> = 11.2 Hz, 2H), 3.15–2.97 (m, 3H), 2.43 (s, 3H), 2.25 (s, 3H), 2.21 (s, 3H), 2.10 (s, 3H), 2.06 (s, 1H), 1.90 (s, 1H), 1.66 (d, <i>J</i> = 11.7 Hz, 2H), 1.52 (d, <i>J</i> = 8.8 Hz, 2H), 0.93 (s, 9H), 0.83 (t, <i>J</i> = 7.0 Hz, 3H). <sup>13</sup>C NMR (101 MHz, <i>d</i><sub>6</sub>-DMSO) δ 171.7, 169.2, 168.9, 168.6, 165.9, 163.0, 151.4, 149.5, 149.0, 147.7, 142.3, 139.7, 139.4, 136.2, 133.1, 131.1, 129.7, 128.7, 127.8, 127.4, 126.4, 123.0, 121.0, 99.5, 70.4, 69.6, 69.4, 69.0, 68.8, 66.3, 58.7, 57.9, 56.6, 55.7, 41.7, 41.2, 37.9, 35.7, 34.9, 30.3, 26.2, 18.9, 18.2, 15.9, 14.6, 12.8. HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>58</sub>H<sub>73</sub>N<sub>8</sub>O<sub>10</sub>S [M – H]<sup>−</sup>: 1073.5176, found: 1073.5195. Purity: 95.8%.</div></div><div id="sec5_1_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> (2<i>S</i>,4<i>R</i>)-1-((<i>S</i>)-2-(<i>tert</i>-Butyl)-14-((3′-(((4,6-dimethyl-2-oxo-1,2-dihydropyridin-3-yl)methyl)carbamoyl)-5′-(ethyl(tetrahydro-2<i>H</i>-pyran-4-yl)amino)-4′-methyl-[1,1′-biphenyl]-4-yl)oxy)-4-oxo-6,9,12-trioxa-3-azatetradecanoyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (<b>V5</b>)</h4><div class="NLM_p last"><b>V5</b> was prepared following the procedure for the synthesis of compound <b>V2</b> (<b>YM281</b>). <sup>1</sup>H NMR (400 MHz, <i>d</i><sub>6</sub>-DMSO) δ 11.47 (s, 1H), 8.97 (d, <i>J</i> = 3.9 Hz, 1H), 8.60 (t, <i>J</i> = 5.9 Hz, 1H), 8.17 (t, <i>J</i> = 4.8 Hz, 1H), 7.53 (d, <i>J</i> = 8.6 Hz, 2H), 7.43 (d, <i>J</i> = 4.6 Hz, 1H), 7.41 (s, 1H), 7.38 (d, <i>J</i> = 9.1 Hz, 4H), 7.35 (s, 1H), 7.18 (s, 1H), 6.99 (d, <i>J</i> = 8.7 Hz, 2H), 5.85 (s, 1H), 5.17 (s, 1H), 4.57 (d, <i>J</i> = 9.6 Hz, 1H), 4.50–4.39 (m, 1H), 4.37 (d, <i>J</i> = 6.1 Hz, 2H), 4.33–4.20 (m, 3H), 4.13–4.06 (m, 2H), 3.97 (s, 2H), 3.82 (d, <i>J</i> = 10.2 Hz, 3H), 3.76–3.71 (m, 2H), 3.60 (m, 6H), 3.24 (t, <i>J</i> = 11.2 Hz, 2H), 3.17 (s, 2H), 3.11-3.03 (m, 2H), 3.01 (s, 1H), 2.43 (s, 3H), 2.23 (s, 3H), 2.21 (s, 3H), 2.10 (s, 3H), 2.08–2.03 (m, 1H), 1.95–1.86 (m, 1H), 1.65 (d, <i>J</i> = 10.9 Hz, 2H), 1.57–1.45 (m, 2H), 0.94 (s, 9H), 0.82 (t, <i>J</i> = 6.9 Hz, 3H). <sup>13</sup>C NMR (101 MHz, <i>d</i><sub>6</sub>-DMSO) δ 171.8, 169.1, 168.6, 163.0, 158.0, 151.4, 149.5, 148.8, 147.7, 14.7, 139.5, 139.4, 136.9, 132.3, 132.0, 131.1, 129.7, 128.7, 127.6, 127.5, 122.5, 121.6, 120.5, 114.9, 107.4, 70.5, 69.9, 69.9, 69.6, 68.9, 68.9, 67.2, 66.3, 58.8, 57.9, 56.6, 55.7, 48.6, 41.7, 41.2, 37.9, 35.7, 34.9, 30.3, 26.2, 18.9, 18.2, 15.9, 14.5, 12.7 HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>59</sub>H<sub>76</sub>N<sub>7</sub>O<sub>11</sub>S [M – H]<sup>−</sup>: 1090.5329, found: 1090.5319. Purity: 98.4%.</div></div><div id="sec5_1_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> <i>N</i><sup>3</sup>-((4,6-Dimethyl-2-oxo-1,2-dihydropyridin-3-yl)methyl)-5-(ethyl(tetrahydro-2<i>H</i>-pyran-4-yl)amino)-<i>N</i><sup><i>4</i></sup>′-((<i>S</i>)-14-((2<i>S</i>,4<i>R</i>)-4-hydroxy-2-((4-(4-methylthiazol-5-yl)benzyl)carbamoyl)pyrrolidine-1-carbonyl)-15,15-dimethyl-12-oxo-3,6,9-trioxa-13-azahexadecyl)-4-methyl-[1,1′-biphenyl]-3,4′-dicarboxamide (<b>V6</b>)</h4><div class="NLM_p last"><b>V6</b> was prepared following the procedure for the synthesis of compound <b>V2</b> (YM281). <sup>1</sup>H NMR (400 MHz, <i>d</i><sub>6</sub>-DMSO) δ 11.45 (s, 1H), 8.97 (s, 1H), 8.55 (t, <i>J</i> = 5.7 Hz, 2H), 8.21 (t, <i>J</i> = 4.9 Hz, 2H), 7.91 (d, <i>J</i> = 8.4 Hz, 2H), 7.89 (d, <i>J</i> = 9.5 Hz, 1H), 7.72 (d, <i>J</i> = 8.3 Hz, 2H), 7.46 (s, 1H), 7.40 (q, <i>J</i> = 8.3 Hz, 4H), 7.30 (s, 1H), 5.87 (s, 1H), 5.14 (d, <i>J</i> = 3.5 Hz, 1H), 4.55 (d, <i>J</i> = 9.4 Hz, 1H), 4.42 (m, 2H), 4.35 (s, 1H), 4.30 (d, <i>J</i> = 4.9 Hz, 2H), 4.22 (dd, <i>J</i> = 15.8, 5.4 Hz, 2H), 4.01–3.95 (m, 1H), 3.83 (d, <i>J</i> = 10.8 Hz, 2H), 3.70–3.41 (m, 15H), 3.29–3.20 (m, 2H), 3.15–2.99 (m, 3H), 2.44 (s, 3H), 2.26 (s, 3H), 2.22 (s, 3H), 2.11 (s, 3H), 2.03 (m, 1H), 1.97–1.86 (m, 1H), 1.67 (d, <i>J</i> = 11.3 Hz, 2H), 1.54–1.48 (m, 2H), 0.93 (s, 9H), 0.83 (t, <i>J</i> = 7.0 Hz, 3H). <sup>13</sup>C NMR (101 MHz, <i>d</i><sub>6</sub>-DMSO) δ 171.9, 170.0, 169.6, 169.0, 166.0, 163.0, 151.4, 149.6, 149.0, 147.7, 142.8, 142.8, 142.3, 139.7, 139.5, 136.2, 133.5, 133.1, 131.2, 129.7, 128.6, 127.8, 127.4, 126.4, 124.0, 123.1, 121.6, 121.0, 107.4, 69.7, 69.6, 69.5, 68.9, 66.9, 66.3, 58.7, 57.9, 56.3, 41.7, 41.3, 41.2, 37.9, 35.7, 35.3, 34.9, 30.3, 26.3, 19.0, 18.6, 18.2, 15.9, 14.6, 12.8. HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>61</sub>H<sub>79</sub>N<sub>8</sub>O<sub>11</sub>S [M – H]<sup>−</sup>: 1131.5594, found: 1131.5584. Purity: 99.1%.</div></div><div id="sec5_1_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> <i>N</i><sup>3</sup>-((4,6-Dimethyl-2-oxo-1,2-dihydropyridin-3-yl)methyl)-5-(ethyl(tetrahydro-2<i>H</i>-pyran-4-yl)amino)-<i>N</i><sup><i>4</i></sup>′-((<i>S</i>)-17-((2<i>S</i>,4<i>R</i>)-4-hydroxy-2-((4-(4-methylthiazol-5-yl) benzyl) carbamoyl) pyrrolidine-1-carbonyl)-18,18-dimethyl-15-oxo-3,6,9,12-tetraoxa-16-azanonadecyl)-4-methyl-[1,1′-biphenyl]-3,4′-dicarboxamide (<b>V7</b>)</h4><div class="NLM_p last"><b>V7</b> was prepared following the procedure for the synthesis of compound <b>V2</b> (<b>YM281</b>). <sup>1</sup>H NMR (400 MHz, <i>d</i><sub>6</sub>-DMSO) δ 11.47 (s, 1H), 8.98 (s, 1H), 8.58 (s, 2H), 8.23 (s, 1H), 7.93 (d, <i>J</i> = 8.2 Hz, 3H), 7.72 (d, <i>J</i> = 8.0 Hz, 2H), 7.46 (s, 1H), 7.39 (dd, <i>J</i> = 16.2, 7.9 Hz, 4H), 7.29 (s, 1H), 5.86 (s, 1H), 5.15 (s, 1H), 4.55 (d, <i>J</i> = 9.3 Hz, 1H), 4.42 (d, <i>J</i> = 4.8 Hz, 2H), 4.35 (s, 1H), 4.30 (d, <i>J</i> = 4.0 Hz, 2H), 4.26–4.16 (m, 1H), 4.08–4.01 (m, 1H), 3.98 (t, <i>J</i> = 6.5 Hz, 1H), 3.83 (d, <i>J</i> = 10.2 Hz, 2H), 3.70–3.59 (m, 4H), 3.55-3.48 (m, 8H), 3.45 (d, <i>J</i> = 4.9 Hz, 2H), 3.25 (t, <i>J</i> = 11.2 Hz, 3H), 3.17–2.96 (m, 3H), 2.44 (s, 3H), 2.25 (s, 3H), 2.21 (s, 3H), 2.10 (s, 3H), 2.07–2.00 (m, 1H), 1.99 (s, 1H), 1.90 (s, 1H), 1.66 (d, <i>J</i> = 10.9 Hz, 2H), 1.52 (d, <i>J</i> = 7.4 Hz, 4H), 1.31 (d, <i>J</i> = 6.9 Hz, 2H), 0.93 (s, 9H), 0.83 (t, <i>J</i> = 7.0 Hz, 3H). <sup>13</sup>C NMR (101 MHz, <i>d</i><sub>6</sub>-DMSO) δ 172.0, 170.1, 169.6, 169.3, 169.0, 166.0, 163.1, 151.5, 149.7, 149.1, 147.7, 142.8, 142.4, 139.8, 139.5, 136.3, 133.5, 133.1, 131.2, 129.7, 128.7, 127.9, 127.5, 126.4, 123.1, 121.6, 121.1, 107.5, 72.9, 69.8, 69.8, 69.7, 69.7, 69.5, 68.9, 68.9, 66.9, 66.4, 63.8, 58.8, 57.9, 56.4, 48.6, 43.1, 41.7, 41.3, 38.0, 35.7, 35.4, 35.0, 30.4, 30.1, 30.0, 26.4, 19.0, 18.6, 18.2, 15.9, 14.7, 13.6, 12.8. HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>63</sub>H<sub>83</sub>N<sub>8</sub>O<sub>12</sub>S [M – H]<sup>−</sup>: 1175.5857, found: 1175.5874. Purity: 97.7%.</div></div><div id="sec5_1_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> (2<i>R</i>,4<i>S</i>)-1-((<i>S</i>)-2-(2-(3-((3′-(((4,6-Dimethyl-2-oxo-1,2-dihydropyridin-3-yl)methyl)carbamoyl)-5′-(ethyl(tetrahydro-2<i>H</i>-pyran-4-yl)amino)-4′-methyl-[1,1′-biphenyl]-4-yl)oxy)propoxy)acetamido)-3,3-dimethylbutanoyl)-4-hydroxy-<i>N</i>-(4-(4-methylthiazol-5-yl)benzyl)pyrrolidine-2-carboxamide (<b>YM620</b>)</h4><div class="NLM_p last"><b>YM620</b> was prepared from <b>21b</b> and <b>41</b> following the procedure for the synthesis of compound <b>V2</b> (<b>YM281</b>). <sup>1</sup>H NMR (400 MHz, <i>d</i><sub>6</sub>-DMSO) δ 11.46 (s, 1H), 8.96 (s, 1H), 8.35 (t, <i>J</i> = 5.9 Hz, 1H), 8.17 (t, <i>J</i> = 4.9 Hz, 1H), 7.52 (d, <i>J</i> = 8.6 Hz, 2H), 7.45–7.37 (m, 4H), 7.32 (t, <i>J</i> = 6.3 Hz, 3H), 7.18 (s, 1H), 6.97 (d, <i>J</i> = 8.7 Hz, 2H), 5.86 (s, 1H), 5.14 (d, <i>J</i> = 3.5 Hz, 1H), 4.54 (d, <i>J</i> = 9.0 Hz, 1H), 4.44–4.33 (m, 3H), 4.30 (d, <i>J</i> = 4.5 Hz, 3H), 4.25 (d, <i>J</i> = 5.8 Hz, 1H), 4.04 (t, <i>J</i> = 6.3 Hz, 2H), 3.82 (d, <i>J</i> = 10.5 Hz, 2H), 3.75 (dd, <i>J</i> = 10.3, 4.7 Hz, 1H), 3.59 (d, <i>J</i> = 6.5 Hz, 3H), 3.24 (t, <i>J</i> = 11.1 Hz, 3H), 3.12–3.04 (m, 2H), 3.01 (s, 1H), 2.43 (s, 3H), 2.24 (s, 3H), 2.21 (s, 3H), 2.11 (s, 3H), 2.06 (dd, <i>J</i> = 8.2, 4.2 Hz, 1H), 1.99–1.91 (m, 3H), 1.66 (d, <i>J</i> = 10.7 Hz, 2H), 1.52 (d, <i>J</i> = 8.6 Hz, 2H), 0.97 (s, 9H), 0.84–0.80 (m, 3H).<sup>13</sup>C NMR (101 MHz, <i>d</i><sub>6</sub>-DMSO) δ 171.5, 169.1, 168.9, 163.0, 158.0, 151.4, 149.4, 148.8, 147.7, 142.7, 139.5, 139.3, 136.8, 132.2, 131.9, 131.1, 129.7, 128.7, 127.6, 127.4, 122.5, 121.6, 120.4, 114.8, 107.3, 69.4, 68.5, 67.6, 66.3, 64.5, 58.8, 57.8, 55.9, 55.4, 41.5, 41.2, 37.9, 35.0, 34.9, 30.3, 29.0, 26.4, 18.9, 18.2, 15.9, 14.5, 12.7. HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>56</sub>H<sub>70</sub>N<sub>7</sub>O<sub>9</sub>S [M – H]<sup>−</sup>: 1016.4961, found: 1016.4929. Purity: 99.4%.</div></div><div id="sec5_1_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> <i>N</i><sup>3</sup>-((4,6-Dimethyl-2-oxo-1,2-dihydropyridin-3-yl)methyl)-<i>N</i><sup><i>4</i></sup>′-(8-(2-((2-(2,6-dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)oxy)acetamido)octyl)-5-(ethyl(tetrahydro-2<i>H</i>-pyran-4-yl)amino)-4-methyl-[1,1′-biphenyl]-3,4′-dicarboxamide (<b>G5</b>)</h4><div class="NLM_p">To a solution of Boc<sub>2</sub>O (1 g, 4.58 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (10 mL) was added a solution of octane-1,8-diamine (2.64 g, 18.33 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (10 mL) dropwise at 0 °C. Then, the reaction mixture was warmed to r.t. and stirred overnight. Finally, CH<sub>2</sub>Cl<sub>2</sub> (50 mL) and water (20 mL) were added. The mixture was separated, and the organic phase was washed with brine (3 × 10 mL). The organic phase was dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated under reduced pressure. The crude residue was purified by flash column chromatography MeOH/CH<sub>2</sub>Cl<sub>2</sub> (0–10%) to give compound <b>37b</b> as a white solid (0.93 g, 63.5% yield).</div><div class="NLM_p">To a solution of <b>36</b> (100 mg, 301 μmol) in DMF (3 mL) were added a solution of <b>37b</b> (110.3 mg, 451 μmol) in DMF (4.5 mL), DIPEA (116.69 mg, 149.22 μmol), and HATU (114.44 mg, 300.96 μmol). The reaction mixture was stirred for 19 h at r.t. and was then diluted with EtOAc (30 mL). The organic phase was washed sequentially with 10% aq. citric acid (10 mL), saturated aq. sodium bicarbonate (10 mL), water (10 mL), and brine (10 mL × 2). The organic phase was dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. Purification by column chromatography with MeOH/CH<sub>2</sub>Cl<sub>2</sub> (0–5%) gave the desired coupling product as a yellow solid (95 mg, 56.5% yield). Then, the obtained compound was dissolved in TFA (1.5 mL, 0.1 M) and heated to 50 °C. After 1 h, the mixture was cooled to r.t., diluted with MeOH, and concentrated under reduced pressure. The crude residue was precipitated with diethyl ether and dried under vacuum to give <b>38b</b> as a yellow solid (95 mg, 100%).</div><div class="NLM_p last">To the solution of <b>7</b> (50 mg, 99.29 μmol) in DMF (2 mL) were added <b>38b</b> (71.6 mg, 119 μmol), DIPEA (73.84 μL, 446.79 μmol), and HATU (36.73 mg, 99.29 μmol). The mixture was stirred at r.t. for 22 h before EtOAc and water were added. The mixture was extracted with EtOAc twice. The combined organic phases were dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. The residue was purified by flash column chromatography with MeOH/CH<sub>2</sub>Cl<sub>2</sub> (0–10%) to afford <b>G5</b> as a yellow solid (32 mg, 34.6% yield). <sup>1</sup>H NMR (400 MHz, <i>d</i><sub>6</sub>-DMSO) δ 11.45 (s, 1H), 11.10 (s, 1H), 8.45 (t, <i>J</i> = 5.5 Hz, 1H), 8.21 (t, <i>J</i> = 4.7 Hz, 1H), 7.91 (d, <i>J</i> = 8.5 Hz, 3H), 7.84–7.78 (m, 1H), 7.71 (d, <i>J</i> = 8.3 Hz, 2H), 7.49 (d, <i>J</i> = 7.2 Hz, 1H), 7.46 (s, 1H), 7.40 (d, <i>J</i> = 8.5 Hz, 1H), 7.29 (s, 1H), 5.86 (s, 1H), 5.12 (dd, <i>J</i> = 12.8, 5.4 Hz, 1H), 4.76 (s, 2H), 4.30 (d, <i>J</i> = 4.7 Hz, 2H), 3.83 (d, <i>J</i> = 10.8 Hz, 2H), 3.29–3.20 (m, 4H), 3.18–3.06 (m, 4H), 2.96–2.84 (m, 3H), 2.60–2.53 (m, 2H), 2.26 (s, 3H), 2.21 (s, 3H), 2.11 (s, 3H), 2.05 (dd, <i>J</i> = 12.5, 4.2 Hz, 2H), 1.67 (d, <i>J</i> = 11.1 Hz, 2H), 1.52 (s, 3H), 1.44 (s, 2H), 1.28 (s, 8H), 0.83 (t, <i>J</i> = 6.8 Hz, 3H). <sup>13</sup>C NMR (101 MHz, <i>d</i><sub>6</sub>-DMSO) δ 172.7, 169.8, 169.0, 166.7, 166.6, 165.7, 165.5, 163.0, 155.0, 149.6, 149.1, 142.8, 142.2, 139.7, 136.9, 136.3, 133.4, 133.0, 127.8, 126.3, 123.1, 121.6, 121.0, 120.4, 116.9, 116.1, 107.4, 67.7, 66.3, 57.9, 48.8, 41.2, 38.3, 34.9, 31.0, 30.3, 29.1, 29.0, 28.7, 28.7, 26.5, 26.3, 22.0, 18.9, 18.2, 14.6, 12.8. HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>53</sub>H<sub>64</sub>N<sub>7</sub>O<sub>10</sub> [M + H]<sup>+</sup>: 958.4709, found: 958.4677. Purity: 96.0%.</div></div><div id="sec5_1_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> <i>N</i><sup>3</sup>-((4,6-Dimethyl-2-oxo-1,2-dihydropyridin-3-yl)methyl)-<i>N</i><sup><i>4</i></sup>′-(4-((2-(2,6-dioxotetrahydro-2<i>H</i>-pyran-3-yl)-1,3-dioxoisoindolin-4-yl)amino)butyl)-5-(ethyl(tetrahydro-2<i>H</i>-pyran-4-yl)amino)-4-methyl-[1,1′-biphenyl]-3,4′-dicarboxamide (<b>G1</b>)</h4><div class="NLM_p last"><b>G1</b> was prepared using the similar procedure for the synthesis of compound <b>G5</b>. <sup>1</sup>H NMR (400 MHz, <i>d</i><sub>6</sub>-DMSO) δ 11.44 (s, 1H), 11.07 (s, 1H), 8.52 (s, 1H), 8.20 (s, 1H), 7.90 (d, <i>J</i> = 8.4 Hz, 2H), 7.71 (d, <i>J</i> = 8.4 Hz, 2H), 7.60–7.53 (m, 1H), 7.46 (s, 1H), 7.29 (s, 1H), 7.12 (d, <i>J</i> = 8.6 Hz, 1H), 7.01 (d, <i>J</i> = 7.0 Hz, 1H), 6.57 (s, 1H), 5.86 (s, 1H), 5.04 (dd, <i>J</i> = 12.8, 5.4 Hz, 1H), 4.30 (d, <i>J</i> = 4.9 Hz, 2H), 3.83 (d, <i>J</i> = 10.4 Hz, 3H), 3.14–3.03 (m, 4H), 2.89 (dd, <i>J</i> = 22.5, 8.5 Hz, 2H), 2.58 (d, <i>J</i> = 16.9 Hz, 2H), 2.25 (s, 3H), 2.21 (d, <i>J</i> = 5.4 Hz, 3H), 2.11 (s, 3H), 2.08–1.99 (m, 2H), 1.65 (d, <i>J</i> = 20.5 Hz, 6H), 1.53 (d, <i>J</i> = 8.2 Hz, 2H), 0.84 (t, <i>J</i> = 6.9 Hz, 3H). <sup>13</sup>C NMR (101 MHz, <i>d</i><sub>6</sub>-DMSO) δ 172.8, 170.1, 169.0, 167.3, 165.8, 163.0, 149.6, 149.1, 146.4, 142.3, 139.8, 136.3, 133.4, 133.3, 132.2, 127.8, 126.4, 123.1, 121.6, 121.0, 117.3, 110.4, 109.1, 107.4, 83.9, 66.3, 57.9, 48.6, 41.6, 41.2, 34.9, 31.0, 30.3, 26.6, 26.3, 22.2, 19.0, 18.2, 14.6, 12.8. HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>47</sub>H<sub>51</sub>N<sub>6</sub>O<sub>9</sub> [M – H]<sup>−</sup>: 843.3723, found: 843.3684. Purity: 95.2%.</div></div><div id="sec5_1_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> <i>N</i><sup>3</sup>-((4,6-Dimethyl-2-oxo-1,2-dihydropyridin-3-yl)methyl)-<i>N</i><sup><i>4</i></sup>′-(2-(2-((2-(2,6-dioxotetrahydro-2<i>H</i>-pyran-3-yl)-1,3-dioxoisoindolin-4-yl)amino)ethoxy)ethyl)-5-(ethyl(tetrahydro-2<i>H</i>-pyran-4-yl)amino)-4-methyl-[1,1′-biphenyl]-3,4′-dicarboxamide (<b>G2</b>)</h4><div class="NLM_p last"><b>G2</b> was prepared using the similar procedure for the synthesis of compound <b>G5</b>. <sup>1</sup>H NMR (400 MHz, <i>d</i><sub>6</sub>-DMSO) δ 11.47 (s, 1H), 11.07 (s, 1H), 8.53 (t, <i>J</i> = 5.5 Hz, 1H), 8.21 (t, <i>J</i> = 4.7 Hz, 1H), 7.89 (d, <i>J</i> = 8.3 Hz, 2H), 7.69 (d, <i>J</i> = 8.4 Hz, 2H), 7.58–7.52 (m, 1H), 7.46 (s, 1H), 7.29 (s, 1H), 7.14 (d, <i>J</i> = 8.6 Hz, 1H), 7.01 (d, <i>J</i> = 7.0 Hz, 1H), 6.62 (t, <i>J</i> = 5.6 Hz, 1H), 5.86 (s, 1H), 5.03 (dd, <i>J</i> = 12.9, 5.3 Hz, 1H), 4.30 (d, <i>J</i> = 4.9 Hz, 2H), 3.83 (d, <i>J</i> = 9.6 Hz, 2H), 3.66 (t, <i>J</i> = 5.3 Hz, 2H), 3.61 (d, <i>J</i> = 6.0 Hz, 2H), 3.47 (dd, <i>J</i> = 10.3, 5.3 Hz, 6H), 3.10 (dd, <i>J</i> = 13.9, 6.8 Hz, 2H), 3.03 (s, 1H), 2.92–2.80 (m, 1H), 2.56 (d, <i>J</i> = 16.1 Hz, 2H), 2.25 (s, 3H), 2.21 (s, 3H), 2.11 (s, 3H), 2.04–1.94 (m, 1H), 1.67 (d, <i>J</i> = 10.2 Hz, 2H), 1.60–1.46 (m, 2H), 0.83 (t, <i>J</i> = 6.9 Hz, 3H). <sup>13</sup>C NMR (101 MHz, <i>d</i><sub>6</sub>-DMSO) δ 172.8, 170.1, 169.0, 167.3, 166.0, 163.0, 158.1, 157.8, 149.6, 149.1, 146.5, 142.8, 142.3, 139.7, 136.3, 133.5, 133.1, 132.1, 127.8, 126.4, 123.1, 121.6, 121.1, 117.5, 110.7, 109.3, 107.5, 68.8, 66.4, 57.9, 48.6, 41.7, 41.3, 34.9, 31.0, 30.4, 22.2, 19.0, 18.2, 14.7, 12.8. HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>47</sub>H<sub>51</sub>N<sub>6</sub>O<sub>10</sub> [M – H]<sup>−</sup>: 859.3672, found: 859.3682. Purity: 96.7%.</div></div><div id="sec5_1_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> <i>N</i><sup>3</sup>-((4,6-Dimethyl-2-oxo-1,2-dihydropyridin-3-yl)methyl)-<i>N</i><sup><i>4</i></sup>′-(2-(2-(2-((2-(2,6-dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)oxy)ethoxy)ethoxy)ethyl)-5-(ethyl(tetrahydro-2<i>H</i>-pyran-4-yl)amino)-4-methyl-[1,1′-biphenyl]-3,4′-dicarboxamide (<b>G3</b>)</h4><div class="NLM_p last"><b>G3</b> was prepared following the similar procedure for the synthesis of compound <b>G5</b>. <sup>1</sup>H NMR (400 MHz, <i>d</i><sub>6</sub>-DMSO) δ 11.46 (s, 1H), 11.09 (s, 1H), 8.52 (d, <i>J</i> = 5.7 Hz, 1H), 8.21 (s, 1H), 7.91 (d, <i>J</i> = 8.3 Hz, 2H), 7.82–7.75 (m, 1H), 7.71 (d, <i>J</i> = 8.2 Hz, 2H), 7.52–7.40 (m, 3H), 7.29 (s, 1H), 5.86 (s, 1H), 5.08 (dd, <i>J</i> = 12.8, 5.3 Hz, 1H), 4.31 (s, 4H), 3.84-3.82 (m, 3H), 3.70–3.63 (m, 2H), 3.57 (dd, <i>J</i> = 11.1, 5.5 Hz, 5H), 3.43 (d, <i>J</i> = 5.4 Hz, 2H), 3.09 (d, <i>J</i> = 6.8 Hz, 2H), 3.01 (d, <i>J</i> = 11.0 Hz, 1H), 2.94–2.79 (m, 2H), 2.63–2.53 (m, 2H), 2.26 (s, 2H), 2.21 (d, <i>J</i> = 5.2 Hz, 3H), 2.11 (s, 3H), 2.07 (d, <i>J</i> = 3.3 Hz, 1H), 2.06–1.98 (m, 2H), 1.66 (d, <i>J</i> = 10.2 Hz, 2H), 1.52 (d, <i>J</i> = 8.2 Hz, 2H), 0.83 (t, <i>J</i> = 6.8 Hz, 3H). <sup>13</sup>C NMR (101 MHz, <i>d</i><sub>6</sub>-DMSO) δ 172.8, 170.0, 169.0, 166.8, 165.9, 165.3, 163.0, 155.8, 149.5, 149.1, 142.8, 142.3, 139.7, 136.9, 136.2, 133.5, 133.2, 133.1, 127.8, 126.4, 123.1, 121.6, 121.0, 120.0, 116.3, 115.4, 107.4, 70.1, 69.7, 68.9, 68.9, 68.7, 66.3, 57.9, 48.8, 41.2, 34.9, 30.9, 30.3, 22.0, 18.9, 18.2, 14.6, 12.8. HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>49</sub>H<sub>57</sub>N<sub>6</sub>O<sub>11</sub> [M + H]<sup>+</sup> 905.4080, found: 905.4096. Purity: 95.0%.</div></div><div id="sec5_1_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> <i>N</i><sup>3</sup>-((4,6-Dimethyl-2-oxo-1,2-dihydropyridin-3-yl)methyl)-<i>N</i><sup><i>4</i></sup>′-(6-(2-((2-(2,6-dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)oxy)acetamido)hexyl)-5-(ethyl(tetrahydro-2<i>H</i>-pyran-4-yl)amino)-4-methyl-[1,1′-biphenyl]-3,4′-dicarboxamide (<b>G4</b>)</h4><div class="NLM_p last"><b>G4</b> was prepared following the similar procedure for the synthesis of compound <b>G5</b>. <sup>1</sup>H NMR (500 MHz, <i>d</i><sub>6</sub>-DMSO) δ 11.47 (s, 1H), 11.12 (s, 1H), 8.47 (t, <i>J</i> = 5.3 Hz, 1H), 8.22 (t, <i>J</i> = 4.7 Hz, 1H), 7.95 (t, <i>J</i> = 5.4 Hz, 1H), 7.90 (d, <i>J</i> = 8.2 Hz, 2H), 7.81 (t, <i>J</i> = 7.9 Hz, 1H), 7.71 (d, <i>J</i> = 8.2 Hz, 2H), 7.51–7.43 (m, 2H), 7.39 (d, <i>J</i> = 8.5 Hz, 1H), 7.29 (s, 1H), 5.86 (s, 1H), 5.12 (dd, <i>J</i> = 12.7, 5.4 Hz, 1H), 4.77 (s, 2H), 4.30 (d, <i>J</i> = 4.7 Hz, 2H), 3.83 (d, <i>J</i> = 11.6 Hz, 2H), 3.25 (t, <i>J</i> = 9.3 Hz, 4H), 3.15 (dd, <i>J</i> = 12.5, 6.2 Hz, 2H), 3.10 (d, <i>J</i> = 6.9 Hz, 2H), 3.02 (t, <i>J</i> = 10.8 Hz, 1H), 2.90 (dd, <i>J</i> = 23.2, 7.3 Hz, 3H), 2.56 (dd, <i>J</i> = 24.7, 11.7 Hz, 2H), 2.25 (s, 3H), 2.21 (s, 3H), 2.10 (s, 3H), 2.08–2.00 (m, 1H), 1.66 (d, <i>J</i> = 11.1 Hz, 2H), 1.53 (d, <i>J</i> = 7.4 Hz, 2H), 1.45 (m, 2H), 1.31 (m, 4H), 0.83 (t, <i>J</i> = 6.9 Hz, 3H). <sup>13</sup>C NMR (126 MHz, <i>d</i><sub>6</sub>-DMSO) δ 172.8, 169.9, 169.0, 166.8, 166.7, 165.8, 165.6, 163.1, 155.1, 149.6, 149.1, 142.8, 142.2, 139.8, 137.0, 136.3, 133.4, 133.1, 127.8, 126.4, 123.1, 121.6, 121.0, 120.4, 116.9, 116.1, 107.4, 67.7, 66.4, 57.9, 48.8, 41.2, 38.3, 34.9, 31.0, 30.3, 29.2, 29.0, 26.2, 26.1, 22.0, 19.0, 18.2, 14.7, 12.8. HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>51</sub>H<sub>60</sub>N<sub>7</sub>O<sub>10</sub> [M + H]<sup>+</sup>: 930.4396, found: 930.4413. Purity: 95.9%.</div></div><div id="sec5_1_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> <i>N</i><sup>3</sup>-((4,6-Dimethyl-2-oxo-1,2-dihydropyridin-3-yl)methyl)-<i>N</i><sup><i>4</i></sup>′-(2-(2-(2-(2-((2-(2,6-dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)oxy)ethoxy)ethoxy)ethoxy)ethyl)-5-(ethyl(tetrahydro-2<i>H</i>-pyran-4-yl)amino)-4-methyl-[1,1′-biphenyl]-3,4′-dicarboxamide (<b>G6</b>)</h4><div class="NLM_p last"><b>G6</b> was prepared following the similar procedure for the synthesis of compound <b>G5</b>. <sup>1</sup>H NMR (400 MHz, <i>d</i><sub>6</sub>-DMSO) δ 11.45 (s, 1H), 11.09 (s, 1H), 8.52 (dd, <i>J</i> = 12.1, 6.3 Hz, 1H), 8.20 (t, <i>J</i> = 4.9 Hz, 1H), 7.90 (t, <i>J</i> = 9.3 Hz, 2H), 7.81–7.76 (m, 1H), 7.74–7.67 (m, 2H), 7.50 (d, <i>J</i> = 8.5 Hz, 1H), 7.44 (d, <i>J</i> = 7.3 Hz, 2H), 7.30 (s, 1H), 5.86 (s, 1H), 5.08 (dd, <i>J</i> = 12.8, 5.4 Hz, 1H), 4.30 (dd, <i>J</i> = 11.9, 6.1 Hz, 4H), 3.85 (d, <i>J</i> = 5.9 Hz, 1H), 3.82–3.75 (m, 2H), 3.66–3.60 (m, 2H), 3.54 (m, 8H), 3.46–3.39 (m, 2H), 3.09 (d, <i>J</i> = 7.0 Hz, 2H), 3.03 (s, 1H), 2.94–2.80 (m, 2H), 2.66–2.53 (m, 2H), 2.25 (s, 2H), 2.21 (d, <i>J</i> = 5.4 Hz, 4H), 2.10 (s, 3H), 2.09–1.97 (m, 3H), 1.90 (d, <i>J</i> = 11.7 Hz, 1H), 1.66 (d, <i>J</i> = 10.6 Hz, 1H), 1.59–1.47 (m, 2H), 0.83 (t, <i>J</i> = 7.0 Hz, 3H). <sup>13</sup>C NMR (101 MHz, <i>d</i><sub>6</sub>-DMSO) <i>δ</i> 172.7, 169.9, 166.8, 165.9, 165.2, 163.0, 155.8, 137.0, 133.2, 127.8, 126.4, 123.4, 121.5, 121.0, 120.0, 116.3, 115.4, 107.4, 99.4, 70.2, 69.8, 69.7, 69.6, 69.2, 68.9, 68.9, 68.7, 66.3, 66.1, 48.8, 34.9, 32.7, 30.9, 30.3, 22.0, 18.9, 18.9, 18.2, 14.6, 12.8. HRMS (ESI, <i>m</i>/<i>z</i>) calcd for C<sub>51</sub>H<sub>61</sub>N<sub>6</sub>O<sub>12</sub> [M + H]<sup>+</sup>: 949.4342, found 949.4323. Purity: 95.0%.</div></div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> Cell Lines</h3><div class="NLM_p last">The human prostate cancer cell lines 22rv1; DLBCL cell lines SU-DHL-2, SU-DHL-4, and SU-DHL-6; BL cell lines NCI-BL209, Daudi, Raji, and Namalwa; LBL cell lines JVM-2; and MCL cell lines MINO and Jeko-1 were obtained from ATCC. All cell lines were cultured in RPMI-1640, supplemented with 10% FBS at 37 °C and 5% CO<sub>2</sub>. All of the cell lines used in the study had been tested for mycoplasma contamination every three weeks.</div></div><div id="sec5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> Cell Number Count</h3><div class="NLM_p last">Cell number was counted by Cellometer Auto T4 cell counter (Nexcelom Bioscience, Lawrence, Massachusetts).</div></div><div id="sec5_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> Cell Viability Assay</h3><div class="NLM_p last">Cell viability was determined by the MTS assay, as we reported previously.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a></div></div><div id="sec5_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> Cell Cycle and Apoptosis</h3><div class="NLM_p last">The cell cycle was measured by flow cytometry according to the commercial cell cycle analysis kit (KeyGEN BioTECH, Jiang Su, China). Annexin-V and propidium iodide-based apoptosis analyses were measured by flow cytometry according to the commercial cell apoptosis analysis kit (BD, Franklin Lakes, NJ). The activity of Caspase-3/7 was evaluated by the Caspase-Glo 3/7 Assay (Promega, Madison, WI) through the manufacturer’s instructions.</div></div><div id="sec5_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> Measurement of Cellular ATP</h3><div class="NLM_p last">The cellular ATP concentration was detected using an ATP-based CellTiter-Glo Luminescent Cell Viability Kit (Promega, Madison, WI) according to the manufacturer’s instructions.</div></div><div id="sec5_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i42" class="anchor-spacer"></div><h3 class="article-section__title" id="_i42"> Western Blot</h3><div class="NLM_p last">The cells were washed with phosphate-buffered saline (PBS) and then lysed in lysis buffer with a protease inhibitor cocktail (Selleck). After the protein concentration normalization using a bicinchoninic acid (BCA) protein assay (ThermoFisher, Rockford, IL), the samples were separated by standard sodium dodecyl sulfate polyacrylamide gel electrophoresis (SDS-PAGE) and transferred to poly(vinylidene fluoride) (PVDF) membranes (Millipore, Billerica, MA). Subsequently, the membranes were blocked with 5% milk for 1 h at r.t. before blotting with the indicated first antibodies overnight at 4 °C. After being washed with TBST (Tris-buffered saline containing 0.1% Tween-20), the membranes were probed with the horseradish peroxidase-conjugated secondary antibodies for 1 h at r.t. Enhanced chemiluminescence was used for signal detection. The intensities of bands were performed using Image Lab (Bio-Rad, Hercules, CA). All the antibodies used for western blot are from Cell Signaling Technology.</div></div><div id="sec5_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> Immunohistochemistry</h3><div class="NLM_p last">Tumors were harvested in formalin and dehydrated and embedded in paraffin. Tissue slides were then deparaffinized, hydrated, and rinsed in PBS. After being boiled in citrate buffer for 4–6 min for antigen retrieval, peroxide blocking was performed with 3% H<sub>2</sub>O<sub>2</sub> at r.t. for 20 min. The sections were incubated with EZH2, H3K27me3 (Cell Signaling Technology), and Ki67 (Abcam), following the suggested concentration at 37 °C overnight. Then, the samples were probed with a secondary antibody at 37 °C for 1 h. All slides were stained with 3,3′-diaminobenzidine and counterstained with hematoxylin.</div></div><div id="sec5_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> Immunoprecipitation</h3><div class="NLM_p last">The cells were harvested using NP40 (Beyotime Biotechnology, Shanghai, China) containing a protease inhibitor cocktail. After 30 min, cell lysates were centrifuged at 4000<i>g</i> for 15 min at 4 °C, and the supernatants were incubated with EZH2 antibody or IgG (Cell Signaling Technology) overnight at 4 °C. Subsequent incubation with protein A/G-coated agarose beads (Merck, Germany) continued for an additional 3 h at 4 °C. After the samples were washed six times with ice-cold NP40, the supernatants were removed by centrifugation at 800<i>g</i> for 2 min. The proteins were then separated from the beads using immunoblotting loading buffer for 5 min at 95 °C. The supernatants were collected for subsequent immunoblotting analysis after SDS gel separation.</div></div><div id="sec5_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i45" class="anchor-spacer"></div><h3 class="article-section__title" id="_i45"> Lentiviral Constructs and siRNAs</h3><div class="NLM_p last">The shEZH2 lentiviral constructs were purchased from GeneCopoeia (GeneCopoeia, Inc, Rockville, Maryland) and transduced into SU-DHL-2 cells according to the literature.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> VHL siRNA (target sequence: GCTCTACGAAGATCTGGAA) was from RIBOBIO (Guangzhou, China). 22Rv1 cells were transfected with siRNA using Lipofectamine RNAiMAX (ThermoFisher, Rockford, IL), following the manufacturer’s instructions.</div></div><div id="sec5_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i46" class="anchor-spacer"></div><h3 class="article-section__title" id="_i46"> Caco-2 Cell Permeability Assay</h3><div class="NLM_p last">The Caco-2 cells were seeded onto polycarbonate 12-well Transwell filters at a density of 2 × 10<sup>5</sup> cells/well. The confluent monolayers obtained at 21 days were utilized to assess the in vitro permeability. Culture media in the apical and basolateral compartments were replaced every 2 days, and the integrity of the monolayer was detected by fluorescein. Before the experiments, the culture medium in both chambers was replaced with prewarmed Hank’s balanced salt solution (HBSS). The cultures were then stabilized at 37 °C for 30 min. For the permeation studies, 0.2 mL of drug formulation diluted with HBSS was added to the apical side, and the basolateral side was replaced with 1 mL fresh HBSS. The treated cells were incubated at 37 ± 0.5 °C. The amount of permeated drug was determined by collecting 50 μL of samples from the basolateral compartment, followed by replacement with 50 μL of fresh HBSS at 1, 2, and 3 h. The collected samples were evaporated and reconstituted with methanol, then the concentrations of drugs in the samples were determined by liquid chromatography–mass spectrometry (LC-MS). The apparent permeability (<i>P</i><sub>app</sub>) of ETP in various formulations was calculated as follows: <span class="NLM_inline-formula"><img class="imgWithoutFigViewer inline-formula-styles" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00460/20210716/images/medium/jm1c00460_m001.gif" alt="" id="_i47" /></img></span> where d<i>Q</i>/d<i>t</i> is the slope of the cumulative drug permeated versus time (μg/s), <i>A</i> is the surface area of the monolayer (1.12 cm<sup>2</sup>), <i>V</i> is the volume of basolateral side HBSS (1 mL), and <i>C</i><sub>0</sub> is the initial concentration of compounds on the apical side (μg/mL).</div></div><div id="sec5_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i48" class="anchor-spacer"></div><h3 class="article-section__title" id="_i48"> Animal Experiments</h3><div class="NLM_p last">The animal experiment was conducted in compliance with a protocol approved by the Institutional Animals Care and Use Committee of Sun Yat-sen University Cancer Center and was carried out in the Center of Experiment Animal of Sun Yat-sen University (North Campus, approval no.: L102012019050K). Balb/c nude mice (female) were bought from Beijing Vital River Laboratory Animal Technology Co., Ltd. For SU-DHL-6 xenografts, and five million cells were injected subcutaneously. After 2 weeks, when tumors reached 100–200 mm<sup>3</sup>, mice were randomly divided into three groups (6 mice per group) and administrated with a vehicle control (80% PBS, 10% castor oil, and 10% DMSO) or indicated doses of compounds (YM281: 80 mg/kg; EPZ6438: 42.5 mg/kg) through intraperitoneal injection 6 times weekly. Tumor sizes and animal weights were measured 2–3 times per week. The mice were sacrificed after 3 weeks’ drug administration, and the tumor tissue was harvested for analyses. For Jeko-1 xenografts, two million cells were injected subcutaneously. Ten days later, most of the tumors grow about 100 mm<sup>3</sup>, mice were then randomly divided into three groups (5 mice per group), and intraperitoneally administrated daily with the control (70% PBS, 20% castor oil, and 10% DMSO) or indicated doses of compounds (YM281: 100 mg/kg; EPZ6438: 50 mg/kg). Tumor sizes and animal weights were measured every 3 days. The mice were sacrificed after 30 days’ drug treatment, and then the tumor tissue was harvested for analyses.</div></div><div id="sec5_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i49" class="anchor-spacer"></div><h3 class="article-section__title" id="_i49"> Isolation and Culture of Clinically Derived Lymphoma Samples</h3><div class="NLM_p last">Tumor tissues from patients with lymphoma at the Sun Yat-sen University cancer center after proper informed consent were collected for cellular Caspase-3/7 activity, ATP content detection, and Western blot analyses (see <a href="/doi/suppl/10.1021/acs.jmedchem.1c00460/suppl_file/jm1c00460_si_001.pdf" class="ext-link">Table S2</a> for the details of patient information). The tumor tissues were immediately minced with fine scissors, and single cells were isolated through a 70 μm strainer (BD Falcon) and cultured in RPMI-1640, supplemented with 10% FBS, 50 μM β-mercaptoethanol, and penicillin/streptomycin solution at 37 °C and 5% CO<sub>2</sub>. Studies using human specimen were approved (No. GZR2018-089) by the Institutional Ethical Committee of Sun Yat-sen University Cancer Center.</div></div><div id="sec5_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i50" class="anchor-spacer"></div><h3 class="article-section__title" id="_i50"> Molecular Docking</h3><div class="NLM_p last">Molecular Operating Environment (MOE 2014) (Chemical Computing Group Inc, Montreal, Quebec, Canada) was used for molecular docking. EPZ6438 was constructed using the builder module, and energy was minimized using Force Field MMFF94x and saved as the MDB file. The crystal structure of EZH2 (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LS6">5LS6</a>) was downloaded from the protein data bank, and water molecules were removed. The residual crystal structure was prepared using ligX with the default parameters of MOE [gradient: 0.1, Force Field: MMFF94X]. A 5Å radius area around the bounded inhibitor in the crystal structure was defined as the active site. The prepared ligand structure was flexibly docked into the EZH2 binding site, with a triangle matcher as the placement methodology, London Δ<i>G</i> or ASE as a scoring methodology, force field refinement was selected, and dock calculations were run automatically. The obtained 30 conformations were generated and stored in a database, and the best conformations were analyzed for the binding interaction analysis.</div></div><div id="sec5_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i51" class="anchor-spacer"></div><h3 class="article-section__title" id="_i51"> Statistical Analysis</h3><div class="NLM_p last">Statistical analysis was performed by GraphPad Prism 8 software (GraphPad Software, Inc, San Diego, CA). All data are presented as mean ± standard error of the mean (SEM). The statistical significance of differences was determined using Student’s <i>t</i>-test. Differences are considered statistically significant when the <i>p</i> values are less than 0.05.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-2"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i52"><a href="/doi/suppl/10.1021/acs.jmedchem.1c00460" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_02075" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_02075" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.1c00460?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00460</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Western blot analysis of EZH2, EZH1, and H3K27me3 levels affected by tested compounds; MTS cell viability analysis in different lymphoma cell lines treated with YM281; the structure of YM620 and its effects on EZH2 and cell proliferation; flow cytometry analysis of the cell cycle and cell apoptosis; Western blot analysis of PARP, Casp-3, and C-casp-3 levels; the body weights of the tested compound treated mice; nonsensitive cancer cell lines NIC-H460 and AsPC1 treated by the tested compounds (Figures S1–S6); patient sample information and corresponding analytical methods; apparent permeability of EPZ6438, YM181, and YM281 (Tables S1–S2); NMR spectra of V1–V7, YM620, G1–G6, and key intermediates; HRMS spectra and HPLC results of YM181 and YM281 (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00460/suppl_file/jm1c00460_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">The docking session for EPZ6438 in EZH2 crystal <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LS6">5LS6</a> (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00460/suppl_file/jm1c00460_si_002.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00460/suppl_file/jm1c00460_si_003.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00460/suppl_file/jm1c00460_si_001.pdf">jm1c00460_si_001.pdf (4.14 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00460/suppl_file/jm1c00460_si_002.pdb">jm1c00460_si_002.pdb (1.37 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00460/suppl_file/jm1c00460_si_003.csv">jm1c00460_si_003.csv (3.11 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.1c00460" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_23851" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_23851" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Peng Huang</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Oncology in South China, Collaborative Innovation
Center for Cancer Medicine, Sun Yat-sen
University Cancer Center, 651 Dongfeng East Road, Guangzhou 510060, China</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#3e564b5f50594e5b50597e4d474d4b5d5d10514c59105d50"><span class="__cf_email__" data-cfemail="c1a9b4a0afa6b1a4afa681b2b8b2b4a2a2efaeb3a6efa2af">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Shijun Wen</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Oncology in South China, Collaborative Innovation
Center for Cancer Medicine, Sun Yat-sen
University Cancer Center, 651 Dongfeng East Road, Guangzhou 510060, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-9347-8243" title="Orcid link">https://orcid.org/0000-0002-9347-8243</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#295e4c475a4143695a505a5c4a4a07465b4e074a47"><span class="__cf_email__" data-cfemail="c8bfada6bba0a288bbb1bbbdababe6a7baafe6aba6">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yalin Tu</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Oncology in South China, Collaborative Innovation
Center for Cancer Medicine, Sun Yat-sen
University Cancer Center, 651 Dongfeng East Road, Guangzhou 510060, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yameng Sun</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Oncology in South China, Collaborative Innovation
Center for Cancer Medicine, Sun Yat-sen
University Cancer Center, 651 Dongfeng East Road, Guangzhou 510060, China</span>; 
    <span class="hlFld-Affiliation affiliation">The
School of Pharmaceutical Sciences, Wuhan
University, Wuhan 430071, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Shuang Qiao</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Oncology in South China, Collaborative Innovation
Center for Cancer Medicine, Sun Yat-sen
University Cancer Center, 651 Dongfeng East Road, Guangzhou 510060, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yao Luo</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Oncology in South China, Collaborative Innovation
Center for Cancer Medicine, Sun Yat-sen
University Cancer Center, 651 Dongfeng East Road, Guangzhou 510060, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Panpan Liu</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Oncology in South China, Collaborative Innovation
Center for Cancer Medicine, Sun Yat-sen
University Cancer Center, 651 Dongfeng East Road, Guangzhou 510060, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Zhong-Xing Jiang</span> - <span class="hlFld-Affiliation affiliation">The
School of Pharmaceutical Sciences, Wuhan
University, Wuhan 430071, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-2601-4366" title="Orcid link">https://orcid.org/0000-0003-2601-4366</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yumin Hu</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Oncology in South China, Collaborative Innovation
Center for Cancer Medicine, Sun Yat-sen
University Cancer Center, 651 Dongfeng East Road, Guangzhou 510060, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Zifeng Wang</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Oncology in South China, Collaborative Innovation
Center for Cancer Medicine, Sun Yat-sen
University Cancer Center, 651 Dongfeng East Road, Guangzhou 510060, China</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>Y.T., Y.S., and S.Q. contributed equally to this work.</p><span class="author-information-subsection-header">Author Contributions</span><p>Conceptualization, S.W. and Y.T.; funding acquisition, S.W. and P.H.; methodology, Y.T., Y.S., Y.H., Z.W., and S.Q.; project administration, Y.T., Y.S., and S.Q.; resources, S.W.; Supervision, S.W. and P.H.; validation, Y.T., Y.S., and S.Q.; visualization, Y.T., Y.S., and S.Q.; writing—original draft, Y.T. and Y.S.; writing—review & editing, Y.T., S.W., P.H., and S.Q.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e3873-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i55">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_18110" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_18110" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We are grateful for the grant support from the National Natural Science Foundation of China (81672952, 81872440), the Guangdong Science and Technology Program (2017A020215198), and the Guangzhou Science and Technology Program (201807010041).</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">EZH2</td><td class="NLM_def"><p class="first last">enhancer of zeste homologue 2</p></td></tr><tr><td class="NLM_term">PROTAC</td><td class="NLM_def"><p class="first last">proteolysis-target chimera</p></td></tr><tr><td class="NLM_term">VHL</td><td class="NLM_def"><p class="first last">von Hippel–Lindau</p></td></tr><tr><td class="NLM_term">CRBN</td><td class="NLM_def"><p class="first last">cereblon</p></td></tr><tr><td class="NLM_term">DLBCL</td><td class="NLM_def"><p class="first last">diffuse large B-cell lymphoma</p></td></tr><tr><td class="NLM_term">PRC2</td><td class="NLM_def"><p class="first last">polycomb repressive complex 2</p></td></tr><tr><td class="NLM_term">SAM</td><td class="NLM_def"><p class="first last"><i>S</i>-adenosylmethionine</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i57">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_73556" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_73556" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 49 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Margueron, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reinberg, D.</span></span> <span> </span><span class="NLM_article-title">The Polycomb complex PRC2 and its mark in life</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>469</i></span>,  <span class="NLM_fpage">343</span>– <span class="NLM_lpage">349</span>, <span class="refDoi"> DOI: 10.1038/nature09784</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1038%2Fnature09784" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=21248841" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC3MXnvVOisA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=469&publication_year=2011&pages=343-349&author=R.+Margueronauthor=D.+Reinberg&title=The+Polycomb+complex+PRC2+and+its+mark+in+life&doi=10.1038%2Fnature09784"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The Polycomb complex PRC2 and its mark in life</span></div><div class="casAuthors">Margueron, Raphael; Reinberg, Danny</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">469</span>
        (<span class="NLM_cas:issue">7330</span>),
    <span class="NLM_cas:pages">343-349</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Polycomb group proteins maintain the gene-expression pattern of different cells that is set during early development by regulating chromatin structure.  In mammals, 2 main Polycomb group complexes exist: Polycomb repressive complex 1 (PRC1) and 2 (PRC2).  PRC1 compacts chromatin and catalyzes the monoubiquitination of histone H2A.  PRC2 also contributes to chromatin compaction, and catalyzes the methylation of histone H3 at Lys-27.  PRC2 is involved in various biol. processes, including differentiation, maintaining cell identity and proliferation, and stem-cell plasticity.  Recent studies of PRC2 have expanded the perspectives on its function and regulation, and uncovered a role for non-coding RNA in the recruitment of PRC2 to target genes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7fYmuBqnMA7Vg90H21EOLACvtfcHk0lh62tDiPtzLKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXnvVOisA%253D%253D&md5=5f15e91cc6ff765f649842041f333841</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnature09784&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09784%26sid%3Dliteratum%253Aachs%26aulast%3DMargueron%26aufirst%3DR.%26aulast%3DReinberg%26aufirst%3DD.%26atitle%3DThe%2520Polycomb%2520complex%2520PRC2%2520and%2520its%2520mark%2520in%2520life%26jtitle%3DNature%26date%3D2011%26volume%3D469%26spage%3D343%26epage%3D349%26doi%3D10.1038%2Fnature09784" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Di
Croce, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helin, K.</span></span> <span> </span><span class="NLM_article-title">Transcriptional regulation by Polycomb group proteins</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">1147</span>– <span class="NLM_lpage">1155</span>, <span class="refDoi"> DOI: 10.1038/nsmb.2669</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1038%2Fnsmb.2669" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=24096405" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFOiurrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2013&pages=1147-1155&author=L.+Di%0ACroceauthor=K.+Helin&title=Transcriptional+regulation+by+Polycomb+group+proteins&doi=10.1038%2Fnsmb.2669"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Transcriptional regulation by Polycomb group proteins</span></div><div class="casAuthors">Di Croce, Luciano; Helin, Kristian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1147-1155</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Polycomb group (PcG) proteins are epigenetic regulators of transcription that have key roles in stem-cell identity, differentiation and disease.  Mechanistically, they function within multiprotein complexes, called Polycomb repressive complexes (PRCs), which modify histones (and other proteins) and silence target genes.  The dynamics of PRC1 and PRC2 components has been the focus of recent research.  Here we discuss our current knowledge of the PRC complexes, how they are targeted to chromatin and how the high diversity of the PcG proteins allows these complexes to influence cell identity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrArNyrbNU897Vg90H21EOLACvtfcHk0lh62tDiPtzLKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFOiurrP&md5=004c2682edbda681e9f6de84ba87e639</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.2669&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.2669%26sid%3Dliteratum%253Aachs%26aulast%3DDi%2BCroce%26aufirst%3DL.%26aulast%3DHelin%26aufirst%3DK.%26atitle%3DTranscriptional%2520regulation%2520by%2520Polycomb%2520group%2520proteins%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2013%26volume%3D20%26spage%3D1147%26epage%3D1155%26doi%3D10.1038%2Fnsmb.2669" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Velichutina, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaknovich, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gascoyne, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melnick, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elemento, O.</span></span> <span> </span><span class="NLM_article-title">EZH2-mediated epigenetic silencing in germinal center B cells contributes to proliferation and lymphomagenesis</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">5247</span>– <span class="NLM_lpage">5255</span>, <span class="refDoi"> DOI: 10.1182/blood-2010-04-280149</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1182%2Fblood-2010-04-280149" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=20736451" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC3MXot1I%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2010&pages=5247-5255&author=I.+Velichutinaauthor=R.+Shaknovichauthor=H.+Gengauthor=N.+A.+Johnsonauthor=R.+D.+Gascoyneauthor=A.+M.+Melnickauthor=O.+Elemento&title=EZH2-mediated+epigenetic+silencing+in+germinal+center+B+cells+contributes+to+proliferation+and+lymphomagenesis&doi=10.1182%2Fblood-2010-04-280149"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">EZH2-mediated epigenetic silencing in germinal center B cells contributes to proliferation and lymphomagenesis</span></div><div class="casAuthors">Velichutina, Irina; Shaknovich, Rita; Geng, Huimin; Johnson, Nathalie A.; Gascoyne, Randy D.; Melnick, Ari M.; Elemento, Olivier</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">5247-5255</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">EZH2 is the catalytic subunit of the PRC2 Polycomb complex and mediates transcriptional repression through its histone methyltransferase activity.  EZH2 is up-regulated in normal germinal center (GC) B cells and is implicated in lymphomagenesis.  To explore the transcriptional programs controlled by EZH2, we performed chromatin immunopptn. (ChIP-on-chip) in GC cells and found that it binds approx. 1800 promoters, often assocd. with DNA sequences similar to Drosophila Polycomb response elements.  While EZH2 targets overlapped extensively between GC B cells and embryonic stem cells, we also obsd. a large GC-specific EZH2 regulatory program.  These genes are preferentially histone 3 lysine 27-trimethylated and repressed in GC B cells and include several key cell cycle-related tumor suppressor genes.  Accordingly, siRNA-mediated down-regulation of EZH2 in diffuse large B-cell lymphoma (DLBCL) cells resulted in acute cell cycle arrest at the G1/S transition and up-regulation of its tumor suppressor target genes.  At the DNA level, EZH2-bound promoters are hypomethylated in GC B cells, but many of them are aberrantly hypermethylated in DLBCL, suggesting disruption of normal epigenetic processes in these cells.  EZH2 is thus involved in regulating a specific epigenetic program in normal GCs, including silencing of antiproliferative genes, which may contribute to the malignant transformation of GC B cells into DLBCLs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobqjlF41Jsh7Vg90H21EOLACvtfcHk0lh62tDiPtzLKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXot1I%253D&md5=4f50ce144655d0b3a3b4de12c069ee22</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1182%2Fblood-2010-04-280149&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2010-04-280149%26sid%3Dliteratum%253Aachs%26aulast%3DVelichutina%26aufirst%3DI.%26aulast%3DShaknovich%26aufirst%3DR.%26aulast%3DGeng%26aufirst%3DH.%26aulast%3DJohnson%26aufirst%3DN.%2BA.%26aulast%3DGascoyne%26aufirst%3DR.%2BD.%26aulast%3DMelnick%26aufirst%3DA.%2BM.%26aulast%3DElemento%26aufirst%3DO.%26atitle%3DEZH2-mediated%2520epigenetic%2520silencing%2520in%2520germinal%2520center%2520B%2520cells%2520contributes%2520to%2520proliferation%2520and%2520lymphomagenesis%26jtitle%3DBlood%26date%3D2010%26volume%3D116%26spage%3D5247%26epage%3D5255%26doi%3D10.1182%2Fblood-2010-04-280149" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ntziachristos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsirigos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Vlierberghe, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nedjic, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trimarchi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flaherty, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferres-Marco, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">da Ros, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siegle, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asp, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hadler, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rigo, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Keersmaecker, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huynh, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Utro, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poglio, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samon, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paietta, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Racevskis, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rowe, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabadan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levine, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pflumio, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dominguez, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrando, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aifantis, I.</span></span> <span> </span><span class="NLM_article-title">Genetic inactivation of the polycomb repressive complex 2 in T cell acute lymphoblastic leukemia</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">298</span>– <span class="NLM_lpage">301</span>, <span class="refDoi"> DOI: 10.1038/nm.2651</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1038%2Fnm.2651" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=22237151" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC38XlsFyrtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2012&pages=298-301&author=P.+Ntziachristosauthor=A.+Tsirigosauthor=P.+Van+Vlierbergheauthor=J.+Nedjicauthor=T.+Trimarchiauthor=M.+S.+Flahertyauthor=D.+Ferres-Marcoauthor=V.+da+Rosauthor=Z.+Tangauthor=J.+Siegleauthor=P.+Aspauthor=M.+Hadlerauthor=I.+Rigoauthor=K.+De+Keersmaeckerauthor=J.+Patelauthor=T.+Huynhauthor=F.+Utroauthor=S.+Poglioauthor=J.+B.+Samonauthor=E.+Paiettaauthor=J.+Racevskisauthor=J.+M.+Roweauthor=R.+Rabadanauthor=R.+L.+Levineauthor=S.+Brownauthor=F.+Pflumioauthor=M.+Dominguezauthor=A.+Ferrandoauthor=I.+Aifantis&title=Genetic+inactivation+of+the+polycomb+repressive+complex+2+in+T+cell+acute+lymphoblastic+leukemia&doi=10.1038%2Fnm.2651"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Genetic inactivation of the polycomb repressive complex 2 in T cell acute lymphoblastic leukemia</span></div><div class="casAuthors">Ntziachristos, Panagiotis; Tsirigos, Aristotelis; Van Vlierberghe, Pieter; Nedjic, Jelena; Trimarchi, Thomas; Sol Flaherty, Maria; Ferres-Marco, Dolors; da Ros, Vanina; Tang, Zuojian; Siegle, Jasmin; Asp, Patrik; Hadler, Michael; Rigo, Isaura; De Keersmaecker, Kim; Patel, Jay; Huynh, Tien; Utro, Filippo; Poglio, Sandrine; Samon, Jeremy B.; Paietta, Elisabeth; Racevskis, Janis; Rowe, Jacob M.; Rabadan, Raul; Levine, Ross L.; Brown, Stuart; Pflumio, Francoise; Dominguez, Maria; Ferrando, Adolfo; Aifantis, Iannis</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">298-303</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">T cell acute lymphoblastic leukemia (T-ALL) is an immature hematopoietic malignancy driven mainly by oncogenic activation of NOTCH1 signaling.  In this study we report the presence of loss-of-function mutations and deletions of the EZH2 and SUZ12 genes, which encode crucial components of the Polycomb repressive complex 2 (PRC2), in 25% of T-ALLs.  To further study the role of PRC2 in T-ALL, we used NOTCH1-dependent mouse models of the disease, as well as human T-ALL samples, and combined locus-specific and global anal. of NOTCH1-driven epigenetic changes.  These studies demonstrated that activation of NOTCH1 specifically induces loss of the repressive mark Lys27 trimethylation of histone 3 (H3K27me3) by antagonizing the activity of PRC2.  These studies suggest a tumor suppressor role for PRC2 in human leukemia and suggest a hitherto unrecognized dynamic interplay between oncogenic NOTCH1 and PRC2 function for the regulation of gene expression and cell transformation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNZkInF-IGxbVg90H21EOLACvtfcHk0lgBoPzfgCLTsw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XlsFyrtQ%253D%253D&md5=061baa595535b99856ded964c146091e</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnm.2651&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.2651%26sid%3Dliteratum%253Aachs%26aulast%3DNtziachristos%26aufirst%3DP.%26aulast%3DTsirigos%26aufirst%3DA.%26aulast%3DVan%2BVlierberghe%26aufirst%3DP.%26aulast%3DNedjic%26aufirst%3DJ.%26aulast%3DTrimarchi%26aufirst%3DT.%26aulast%3DFlaherty%26aufirst%3DM.%2BS.%26aulast%3DFerres-Marco%26aufirst%3DD.%26aulast%3Dda%2BRos%26aufirst%3DV.%26aulast%3DTang%26aufirst%3DZ.%26aulast%3DSiegle%26aufirst%3DJ.%26aulast%3DAsp%26aufirst%3DP.%26aulast%3DHadler%26aufirst%3DM.%26aulast%3DRigo%26aufirst%3DI.%26aulast%3DDe%2BKeersmaecker%26aufirst%3DK.%26aulast%3DPatel%26aufirst%3DJ.%26aulast%3DHuynh%26aufirst%3DT.%26aulast%3DUtro%26aufirst%3DF.%26aulast%3DPoglio%26aufirst%3DS.%26aulast%3DSamon%26aufirst%3DJ.%2BB.%26aulast%3DPaietta%26aufirst%3DE.%26aulast%3DRacevskis%26aufirst%3DJ.%26aulast%3DRowe%26aufirst%3DJ.%2BM.%26aulast%3DRabadan%26aufirst%3DR.%26aulast%3DLevine%26aufirst%3DR.%2BL.%26aulast%3DBrown%26aufirst%3DS.%26aulast%3DPflumio%26aufirst%3DF.%26aulast%3DDominguez%26aufirst%3DM.%26aulast%3DFerrando%26aufirst%3DA.%26aulast%3DAifantis%26aufirst%3DI.%26atitle%3DGenetic%2520inactivation%2520of%2520the%2520polycomb%2520repressive%2520complex%25202%2520in%2520T%2520cell%2520acute%2520lymphoblastic%2520leukemia%26jtitle%3DNat.%2520Med.%26date%3D2012%26volume%3D18%26spage%3D298%26epage%3D301%26doi%3D10.1038%2Fnm.2651" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Z. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groner, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lis, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stack, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cato, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thornton, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gregory, R. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morrissey, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vessella, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montironi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magi-Galluzzi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kantoff, P. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balk, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, M.</span></span> <span> </span><span class="NLM_article-title">EZH2 oncogenic activity in castration-resistant prostate cancer cells is Polycomb-independent</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>338</i></span>,  <span class="NLM_fpage">1465</span>– <span class="NLM_lpage">1469</span>, <span class="refDoi"> DOI: 10.1126/science.1227604</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1126%2Fscience.1227604" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=23239736" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVSktLfF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=338&publication_year=2012&pages=1465-1469&author=K.+Xuauthor=Z.+J.+Wuauthor=A.+C.+Gronerauthor=H.+H.+Heauthor=C.+Caiauthor=R.+T.+Lisauthor=X.+Wuauthor=E.+C.+Stackauthor=M.+Lodaauthor=T.+Liuauthor=H.+Xuauthor=L.+Catoauthor=J.+E.+Thorntonauthor=R.+I.+Gregoryauthor=C.+Morrisseyauthor=R.+L.+Vessellaauthor=R.+Montironiauthor=C.+Magi-Galluzziauthor=P.+W.+Kantoffauthor=S.+P.+Balkauthor=X.+S.+Liuauthor=M.+Brown&title=EZH2+oncogenic+activity+in+castration-resistant+prostate+cancer+cells+is+Polycomb-independent&doi=10.1126%2Fscience.1227604"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">EZH2 oncogenic activity in castration-resistant prostate cancer cells is polycomb-independent</span></div><div class="casAuthors">Xu, Kexin; Wu, Zhenhua Jeremy; Groner, Anna C.; He, Housheng Hansen; Cai, Changmeng; Lis, Rosina T.; Wu, Xiaoqiu; Stack, Edward C.; Loda, Massimo; Liu, Tao; Xu, Han; Cato, Laura; Thornton, James E.; Gregory, Richard I.; Morrissey, Colm; Vessella, Robert L.; Montironi, Rodolfo; Magi-Galluzzi, Cristina; Kantoff, Philip W.; Balk, Steven P.; Liu, X. Shirley; Brown, Myles</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">338</span>
        (<span class="NLM_cas:issue">6113</span>),
    <span class="NLM_cas:pages">1465-1469</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Epigenetic regulators represent a promising new class of therapeutic targets for cancer.  Enhancer of zeste homolog 2 (EZH2), a subunit of Polycomb repressive complex 2 (PRC2), silences gene expression via its histone methyltransferase activity.  We found that the oncogenic function of EZH2 in cells of castration-resistant prostate cancer is independent of its role as a transcriptional repressor.  Instead, it involves the ability of EZH2 to act as a coactivator for crit. transcription factors including the androgen receptor.  This functional switch is dependent on phosphorylation of EZH2 and requires an intact methyltransferase domain.  Hence, targeting the non-PRC2 function of EZH2 may have therapeutic efficacy for treating metastatic, hormone-refractory prostate cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoOB2b-7A-UaLVg90H21EOLACvtfcHk0lgBoPzfgCLTsw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVSktLfF&md5=98399f2e6a4c9b07121059772650d2d3</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1126%2Fscience.1227604&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1227604%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DK.%26aulast%3DWu%26aufirst%3DZ.%2BJ.%26aulast%3DGroner%26aufirst%3DA.%2BC.%26aulast%3DHe%26aufirst%3DH.%2BH.%26aulast%3DCai%26aufirst%3DC.%26aulast%3DLis%26aufirst%3DR.%2BT.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DStack%26aufirst%3DE.%2BC.%26aulast%3DLoda%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DT.%26aulast%3DXu%26aufirst%3DH.%26aulast%3DCato%26aufirst%3DL.%26aulast%3DThornton%26aufirst%3DJ.%2BE.%26aulast%3DGregory%26aufirst%3DR.%2BI.%26aulast%3DMorrissey%26aufirst%3DC.%26aulast%3DVessella%26aufirst%3DR.%2BL.%26aulast%3DMontironi%26aufirst%3DR.%26aulast%3DMagi-Galluzzi%26aufirst%3DC.%26aulast%3DKantoff%26aufirst%3DP.%2BW.%26aulast%3DBalk%26aufirst%3DS.%2BP.%26aulast%3DLiu%26aufirst%3DX.%2BS.%26aulast%3DBrown%26aufirst%3DM.%26atitle%3DEZH2%2520oncogenic%2520activity%2520in%2520castration-resistant%2520prostate%2520cancer%2520cells%2520is%2520Polycomb-independent%26jtitle%3DScience%26date%3D2012%26volume%3D338%26spage%3D1465%26epage%3D1469%26doi%3D10.1126%2Fscience.1227604" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, C. W.</span></span> <span> </span><span class="NLM_article-title">Targeting EZH2 in cancer</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">128</span>– <span class="NLM_lpage">134</span>, <span class="refDoi"> DOI: 10.1038/nm.4036</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1038%2Fnm.4036" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=26845405" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC28Xit1Wktr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2016&pages=128-134&author=K.+H.+Kimauthor=C.+W.+Roberts&title=Targeting+EZH2+in+cancer&doi=10.1038%2Fnm.4036"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting EZH2 in cancer</span></div><div class="casAuthors">Kim, Kimberly H.; Roberts, Charles W. M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">128-134</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Recent genomic studies have resulted in an emerging understanding of the role of chromatin regulators in the development of cancer.  EZH2, a histone Me transferase subunit of a Polycomb repressor complex, is recurrently mutated in several forms of cancer and is highly expressed in numerous others.  Notably, both gain-of-function and loss-of-function mutations occur in cancers but are assocd. with distinct cancer types.  Here we review the spectrum of EZH2-assocd. mutations, discuss the mechanisms underlying EZH2 function, and synthesize a unifying perspective that the promotion of cancer arises from disruption of the role of EZH2 as a master regulator of transcription.  We further discuss EZH2 inhibitors that are now showing early signs of promise in clin. trials and also addnl. strategies to combat roles of EZH2 in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2UJCv3jp9-rVg90H21EOLACvtfcHk0lidHPsmJBEm8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xit1Wktr4%253D&md5=757ae5f38dc560920f66ced2c818e9e1</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnm.4036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.4036%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DK.%2BH.%26aulast%3DRoberts%26aufirst%3DC.%2BW.%26atitle%3DTargeting%2520EZH2%2520in%2520cancer%26jtitle%3DNat.%2520Med.%26date%3D2016%26volume%3D22%26spage%3D128%26epage%3D134%26doi%3D10.1038%2Fnm.4036" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bonasio, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reinberg, D.</span></span> <span> </span><span class="NLM_article-title">Molecular signals of epigenetic states</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>330</i></span>,  <span class="NLM_fpage">612</span>– <span class="NLM_lpage">616</span>, <span class="refDoi"> DOI: 10.1126/science.1191078</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1126%2Fscience.1191078" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=21030644" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlaqt7bE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=330&publication_year=2010&pages=612-616&author=R.+Bonasioauthor=S.+Tuauthor=D.+Reinberg&title=Molecular+signals+of+epigenetic+states&doi=10.1126%2Fscience.1191078"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Signals of Epigenetic States</span></div><div class="casAuthors">Bonasio, Roberto; Tu, Shengjiang; Reinberg, Danny</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">330</span>
        (<span class="NLM_cas:issue">6004</span>),
    <span class="NLM_cas:pages">612-616</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A review.  Epigenetic signals are responsible for the establishment, maintenance, and reversal of metastable transcriptional states that are fundamental for the cell's ability to "remember" past events, such as changes in the external environment or developmental cues.  Complex epigenetic states are orchestrated by several converging and reinforcing signals, including transcription factors, noncoding RNAs, DNA methylation, and histone modifications.  Although all of these pathways modulate transcription from chromatin in vivo, the mechanisms by which epigenetic information is transmitted through cell division remain unclear.  Because epigenetic states are metastable and change in response to the appropriate signals, a deeper understanding of their mol. framework will allow us to tackle the dysregulation of epigenetics in disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwvUAbah6n1LVg90H21EOLACvtfcHk0lidHPsmJBEm8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlaqt7bE&md5=85a662dbd8372bebbc8255f8db6dad0b</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1126%2Fscience.1191078&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1191078%26sid%3Dliteratum%253Aachs%26aulast%3DBonasio%26aufirst%3DR.%26aulast%3DTu%26aufirst%3DS.%26aulast%3DReinberg%26aufirst%3DD.%26atitle%3DMolecular%2520signals%2520of%2520epigenetic%2520states%26jtitle%3DScience%26date%3D2010%26volume%3D330%26spage%3D612%26epage%3D616%26doi%3D10.1126%2Fscience.1191078" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bates, S. E.</span></span> <span> </span><span class="NLM_article-title">Epigenetic Therapies for Cancer</span>. <i>N. Engl. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>383</i></span>,  <span class="NLM_fpage">650</span>– <span class="NLM_lpage">663</span>, <span class="refDoi"> DOI: 10.1056/NEJMra1805035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1056%2FNEJMra1805035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=32786190" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhs1SltrbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=383&publication_year=2020&pages=650-663&author=S.+E.+Bates&title=Epigenetic+Therapies+for+Cancer&doi=10.1056%2FNEJMra1805035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetic therapies for cancer</span></div><div class="casAuthors">Bates, Susan E.</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">383</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">650-663</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">1533-4406</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">A review.  Chromatin is one of the earliest identified targets for cancer therapeutics.  A more precise understanding of the complexity of chromatin and its role in oncogenesis began to emerge when sequencing of the cancer genome revealed mutations in numerous genes encoding proteins that regulate chromatin.  In many cases, these mutations proved to be crit. in maintaining the malignant process, an observation that led to new therapeutics.  This review summarizes approved agents and their clin. activity, describes therapies in development, and delineates challenges in the field of epigenetics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiRHFKx786ibVg90H21EOLACvtfcHk0lidHPsmJBEm8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhs1SltrbL&md5=3cd39b4cc794c12b3bd45ac22a909db6</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1056%2FNEJMra1805035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMra1805035%26sid%3Dliteratum%253Aachs%26aulast%3DBates%26aufirst%3DS.%2BE.%26atitle%3DEpigenetic%2520Therapies%2520for%2520Cancer%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2020%26volume%3D383%26spage%3D650%26epage%3D663%26doi%3D10.1056%2FNEJMra1805035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kohashi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oda, Y.</span></span> <span> </span><span class="NLM_article-title">Oncogenic roles of SMARCB1/INI1 and its deficient tumors</span>. <i>Cancer Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>108</i></span>,  <span class="NLM_fpage">547</span>– <span class="NLM_lpage">552</span>, <span class="refDoi"> DOI: 10.1111/cas.13173</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1111%2Fcas.13173" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=28109176" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC2sXlvFClsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2017&pages=547-552&author=K.+Kohashiauthor=Y.+Oda&title=Oncogenic+roles+of+SMARCB1%2FINI1+and+its+deficient+tumors&doi=10.1111%2Fcas.13173"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Oncogenic roles of SMARCB1/INI1 and its deficient tumors</span></div><div class="casAuthors">Kohashi, Kenichi; Oda, Yoshinao</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Science</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">547-552</span>CODEN:
                <span class="NLM_cas:coden">CSACCM</span>;
        ISSN:<span class="NLM_cas:issn">1349-7006</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">SMARCB1/INI1 is one of the core subunit proteins of the ATP-dependent SWI/SNF chromatin remodeling complex, and is identified as a potent and bona fide tumor suppressor.  Interactions have been demonstrated between SMARCB1/INI1 and key proteins in various pathways related to tumor proliferation and progression: the p16-RB pathway, WNT signaling pathway, sonic hedgehog signaling pathway and Polycomb pathway.  Initially, no detectable SMARCB1/INI1 protein expression was found in malignant rhabdoid tumor cells, whereas all other kinds of tumor cells and non-tumorous tissue showed SMARCB1/INI1 protein expression.  Therefore, immunohistochem. testing for the SMARCB1/INI1 antibody has been considered useful in confirming the histol. diagnosis of malignant rhabdoid tumors.  However, recently, aberrant expression of SMARCB1/INI1 has been found in various tumors such as epithelioid sarcomas, schwannomatosis, synovial sarcomas, and so on.  In addn., it has been reported that aberrant expression can be classified into three patterns: complete loss, mosaic expression and reduced expression.  Although the various pathways related to mechanisms of tumorigenesis and tumor proliferation are complexly intertwined, the clarification of these mechanisms may contribute to therapeutic strategies in SMARCB1/INI1-deficient tumors.  In terms of pathol. classifications, SMARCB1/INI1-deficient tumors may be re-classified by genetic backgrounds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbJZuuZkjIIrVg90H21EOLACvtfcHk0lidHPsmJBEm8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXlvFClsLo%253D&md5=d9f76311d904561cc4f1547e13b918b4</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1111%2Fcas.13173&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fcas.13173%26sid%3Dliteratum%253Aachs%26aulast%3DKohashi%26aufirst%3DK.%26aulast%3DOda%26aufirst%3DY.%26atitle%3DOncogenic%2520roles%2520of%2520SMARCB1%252FINI1%2520and%2520its%2520deficient%2520tumors%26jtitle%3DCancer%2520Sci.%26date%3D2017%26volume%3D108%26spage%3D547%26epage%3D552%26doi%3D10.1111%2Fcas.13173" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howard, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vazquez, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsherniak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haswell, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walensky, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hahn, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orkin, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, C. W.</span></span> <span> </span><span class="NLM_article-title">SWI/SNF-mutant cancers depend on catalytic and non-catalytic activity of EZH2</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">1491</span>– <span class="NLM_lpage">1496</span>, <span class="refDoi"> DOI: 10.1038/nm.3968</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1038%2Fnm.3968" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=26552009" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslKgt7nK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=1491-1496&author=K.+H.+Kimauthor=W.+Kimauthor=T.+P.+Howardauthor=F.+Vazquezauthor=A.+Tsherniakauthor=J.+N.+Wuauthor=W.+Wangauthor=J.+R.+Haswellauthor=L.+D.+Walenskyauthor=W.+C.+Hahnauthor=S.+H.+Orkinauthor=C.+W.+Roberts&title=SWI%2FSNF-mutant+cancers+depend+on+catalytic+and+non-catalytic+activity+of+EZH2&doi=10.1038%2Fnm.3968"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">SWI/SNF-mutant cancers depend on catalytic and non-catalytic activity of EZH2</span></div><div class="casAuthors">Kim, Kimberly H.; Kim, Woojin; Howard, Thomas P.; Vazquez, Francisca; Tsherniak, Aviad; Wu, Jennifer N.; Wang, Weishan; Haswell, Jeffrey R.; Walensky, Loren D.; Hahn, William C.; Orkin, Stuart H.; Roberts, Charles W. M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1491-1496</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Human cancer genome sequencing has recently revealed that genes that encode subunits of SWI/SNF chromatin remodeling complexes are frequently mutated across a wide variety of cancers, and several subunits of the complex have been shown to have bona fide tumor suppressor activity.  However, whether mutations in SWI/SNF subunits result in shared dependencies is unknown.  Here we show that EZH2, a catalytic subunit of the polycomb repressive complex 2 (PRC2), is essential in all tested cancer cell lines and xenografts harboring mutations of the SWI/SNF subunits ARID1A, PBRM1, and SMARCA4, which are several of the most frequently mutated SWI/SNF subunits in human cancer, but that co-occurrence of a Ras pathway mutation is correlated with abrogation of this dependence.  Notably, we demonstrate that SWI/SNF-mutant cancer cells are primarily dependent on a non-catalytic role of EZH2 in the stabilization of the PRC2 complex, and that they are only partially dependent on EZH2 histone methyltransferase activity.  These results not only reveal a shared dependency of cancers with genetic alterations in SWI/SNF subunits, but also suggest that EZH2 enzymic inhibitors now in clin. development may not fully suppress the oncogenic activity of EZH2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrNjhQKP01U5bVg90H21EOLACvtfcHk0lhigLE7LVyoyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslKgt7nK&md5=a30c11fe7ce3c20e982e824105b8d640</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnm.3968&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.3968%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DK.%2BH.%26aulast%3DKim%26aufirst%3DW.%26aulast%3DHoward%26aufirst%3DT.%2BP.%26aulast%3DVazquez%26aufirst%3DF.%26aulast%3DTsherniak%26aufirst%3DA.%26aulast%3DWu%26aufirst%3DJ.%2BN.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DHaswell%26aufirst%3DJ.%2BR.%26aulast%3DWalensky%26aufirst%3DL.%2BD.%26aulast%3DHahn%26aufirst%3DW.%2BC.%26aulast%3DOrkin%26aufirst%3DS.%2BH.%26aulast%3DRoberts%26aufirst%3DC.%2BW.%26atitle%3DSWI%252FSNF-mutant%2520cancers%2520depend%2520on%2520catalytic%2520and%2520non-catalytic%2520activity%2520of%2520EZH2%26jtitle%3DNat.%2520Med.%26date%3D2015%26volume%3D21%26spage%3D1491%26epage%3D1496%26doi%3D10.1038%2Fnm.3968" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bitler, B. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aird, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garipov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amatangelo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kossenkov, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schultz, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shih Ie, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conejo-Garcia, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Speicher, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, R.</span></span> <span> </span><span class="NLM_article-title">Synthetic lethality by targeting EZH2 methyltransferase activity in ARID1A-mutated cancers</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">231</span>– <span class="NLM_lpage">238</span>, <span class="refDoi"> DOI: 10.1038/nm.3799</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1038%2Fnm.3799" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=25686104" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC2MXivVSit7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=231-238&author=B.+G.+Bitlerauthor=K.+M.+Airdauthor=A.+Garipovauthor=H.+Liauthor=M.+Amatangeloauthor=A.+V.+Kossenkovauthor=D.+C.+Schultzauthor=Q.+Liuauthor=M.+Shih+Ieauthor=J.+R.+Conejo-Garciaauthor=D.+W.+Speicherauthor=R.+Zhang&title=Synthetic+lethality+by+targeting+EZH2+methyltransferase+activity+in+ARID1A-mutated+cancers&doi=10.1038%2Fnm.3799"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Synthetic lethality by targeting EZH2 methyltransferase activity in ARID1A-mutated cancers</span></div><div class="casAuthors">Bitler, Benjamin G.; Aird, Katherine M.; Garipov, Azat; Li, Hua; Amatangelo, Michael; Kossenkov, Andrew V.; Schultz, David C.; Liu, Qin; Shih, Ie-Ming; Conejo-Garcia, Jose R.; Speicher, David W.; Zhang, Rugang</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">231-238</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The gene encoding ARID1A, a chromatin remodeler, shows one of the highest mutation rates across many cancer types.  Notably, ARID1A is mutated in over 50% of ovarian clear cell carcinomas, which currently have no effective therapy.  To date, clin. applicable targeted cancer therapy based on ARID1A mutational status has not been described.  Here we show that inhibition of the EZH2 methyltransferase acts in a synthetic lethal manner in ARID1A-mutated ovarian cancer cells and that ARID1A mutational status correlated with response to the EZH2 inhibitor.  We identified PIK3IP1 as a direct target of ARID1A and EZH2 that is upregulated by EZH2 inhibition and contributed to the obsd. synthetic lethality by inhibiting PI3K-AKT signaling.  Importantly, EZH2 inhibition caused regression of ARID1A-mutated ovarian tumors in vivo.  To our knowledge, this is the first data set to demonstrate a synthetic lethality between ARID1A mutation and EZH2 inhibition.  Our data indicate that pharmacol. inhibition of EZH2 represents a novel treatment strategy for cancers involving ARID1A mutations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEiQfgiO8t4LVg90H21EOLACvtfcHk0lhigLE7LVyoyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXivVSit7k%253D&md5=2521906b5e6ef9b7fc51a09ceb1f0c9d</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fnm.3799&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.3799%26sid%3Dliteratum%253Aachs%26aulast%3DBitler%26aufirst%3DB.%2BG.%26aulast%3DAird%26aufirst%3DK.%2BM.%26aulast%3DGaripov%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DAmatangelo%26aufirst%3DM.%26aulast%3DKossenkov%26aufirst%3DA.%2BV.%26aulast%3DSchultz%26aufirst%3DD.%2BC.%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DShih%2BIe%26aufirst%3DM.%26aulast%3DConejo-Garcia%26aufirst%3DJ.%2BR.%26aulast%3DSpeicher%26aufirst%3DD.%2BW.%26aulast%3DZhang%26aufirst%3DR.%26atitle%3DSynthetic%2520lethality%2520by%2520targeting%2520EZH2%2520methyltransferase%2520activity%2520in%2520ARID1A-mutated%2520cancers%26jtitle%3DNat.%2520Med.%26date%3D2015%26volume%3D21%26spage%3D231%26epage%3D238%26doi%3D10.1038%2Fnm.3799" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yamagishi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uchimaru, K.</span></span> <span> </span><span class="NLM_article-title">Targeting EZH2 in cancer therapy</span>. <i>Curr. Opin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">375</span>– <span class="NLM_lpage">381</span>, <span class="refDoi"> DOI: 10.1097/CCO.0000000000000390</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1097%2FCCO.0000000000000390" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=28665819" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht12iu7zO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2017&pages=375-381&author=M.+Yamagishiauthor=K.+Uchimaru&title=Targeting+EZH2+in+cancer+therapy&doi=10.1097%2FCCO.0000000000000390"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting EZH2 in cancer therapy</span></div><div class="casAuthors">Yamagishi, Makoto; Uchimaru, Kaoru</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Oncology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">375-381</span>CODEN:
                <span class="NLM_cas:coden">CUOOE8</span>;
        ISSN:<span class="NLM_cas:issn">1040-8746</span>.
    
            (<span class="NLM_cas:orgname">Lippincott Williams & Wilkins</span>)
        </div><div class="casAbstract">A review.  The present review introduces recent outstanding progress pertaining to Enhancer of zeste homolog 2 (EZH2), esp. regarding its mode of action as a master regulator of chromatin, and provides mol.-based evidence for targeting EZH2 in cancer therapy.  We discuss the active development of small mols. targeting the enzymic activity of EZH2/polycomb repressive complex 2 (PRC2).  Recent findings: Genetic, transcriptional, and posttranscriptional dysregulation of EZH2 is frequently obsd. in many cancer types.  EZH2 promotes tumorigenesis by altering the expression of numerous tumor suppressor genes.  Furthermore, the executive mol. processes initiated by EZH2, such as NF-B activation, microRNA silencing, tumor immune evasion, and noncanonical transcription regulation, appear to be the fundamental characteristics of each cancer.  Systematic investigations have suggested coordinated regulation of the cancer epigenome wherein antagonistic complexes of both polycomb and SWI/SNF are involved.  Frequent loss-of-function mutations in epigenetic factors, such as ARID1A, SMARCA4, SMARCB1, BAP1, and KDM6A, are likely to elicit the EZH2/PRC2-addicted situation.  Our comprehensive understanding encourages the development of advanced strategies for the appropriate manipulation of the cancer epigenome.  Moreover, a couple of small mols. that can effectively inhibit the enzymic activity of EZH2/PRC2 have been translated into early-phase clin. trials.  The EZH2-mediated epigenome and subsequent transcriptome define cellular identity.  Effective and specific strategies for the manipulation of EZH2/PRC2 may lead to the development of more precise cancer medicines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNgCSp7SIPsLVg90H21EOLACvtfcHk0lhigLE7LVyoyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht12iu7zO&md5=a0089ba316841e0a4a3b67d3038465b9</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1097%2FCCO.0000000000000390&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252FCCO.0000000000000390%26sid%3Dliteratum%253Aachs%26aulast%3DYamagishi%26aufirst%3DM.%26aulast%3DUchimaru%26aufirst%3DK.%26atitle%3DTargeting%2520EZH2%2520in%2520cancer%2520therapy%26jtitle%3DCurr.%2520Opin.%2520Oncol.%26date%3D2017%26volume%3D29%26spage%3D375%26epage%3D381%26doi%3D10.1097%2FCCO.0000000000000390" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span> </span><span class="NLM_article-title">First EZH2 Inhibitor Approved-for Rare Sarcoma</span>.  <i>Cancer Discovery</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume">10</span>,  <span class="NLM_fpage">333</span>  <span class="NLM_lpage">334</span>. <span class="refDoi"> DOI: 10.1158/2159-8290.CD-NB2020-006</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1158%2F2159-8290.CD-NB2020-006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=32041739" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=First+EZH2+Inhibitor+Approved-for+Rare+Sarcoma.+Cancer+Discovery+2020%2C+10%2C+333+334.+10.1158%2F2159-8290.CD-NB2020-006."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1158%2F2159-8290.CD-NB2020-006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F2159-8290.CD-NB2020-006%26sid%3Dliteratum%253Aachs%26atitle%3DFirst%2520EZH2%2520Inhibitor%2520Approved-for%2520Rare%2520Sarcoma%26jtitle%3DCancer%2520Discovery%26date%3D2020%26volume%3D10%26spage%3D333%26epage%3D334%26doi%3D10.1158%2F2159-8290.CD-NB2020-006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Knutson, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wigle, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warholic, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sneeringer, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allain, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klaus, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sacks, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raimondi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majer, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olhava, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chesworth, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moyer, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richon, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keilhack, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollock, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuntz, K. W.</span></span> <span> </span><span class="NLM_article-title">A selective inhibitor of EZH2 blocks H3K27 methylation and kills mutant lymphoma cells</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">890</span>– <span class="NLM_lpage">896</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1084</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1038%2Fnchembio.1084" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=23023262" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVaqtrvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=890-896&author=S.+K.+Knutsonauthor=T.+J.+Wigleauthor=N.+M.+Warholicauthor=C.+J.+Sneeringerauthor=C.+J.+Allainauthor=C.+R.+Klausauthor=J.+D.+Sacksauthor=A.+Raimondiauthor=C.+R.+Majerauthor=J.+Songauthor=M.+P.+Scottauthor=L.+Jinauthor=J.+J.+Smithauthor=E.+J.+Olhavaauthor=R.+Chesworthauthor=M.+P.+Moyerauthor=V.+M.+Richonauthor=R.+A.+Copelandauthor=H.+Keilhackauthor=R.+M.+Pollockauthor=K.+W.+Kuntz&title=A+selective+inhibitor+of+EZH2+blocks+H3K27+methylation+and+kills+mutant+lymphoma+cells&doi=10.1038%2Fnchembio.1084"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">A selective inhibitor of EZH2 blocks H3K27 methylation and kills mutant lymphoma cells</span></div><div class="casAuthors">Knutson, Sarah K.; Wigle, Tim J.; Warholic, Natalie M.; Sneeringer, Christopher J.; Allain, Christina J.; Klaus, Christine R.; Sacks, Joelle D.; Raimondi, Alejandra; Majer, Christina R.; Song, Jeffrey; Scott, Margaret Porter; Jin, Lei; Smith, Jesse J.; Olhava, Edward J.; Chesworth, Richard; Moyer, Mikel P.; Richon, Victoria M.; Copeland, Robert A.; Keilhack, Heike; Pollock, Roy M.; Kuntz, Kevin W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">890-896</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">EZH2 catalyzes trimethylation of histone H3 lysine 27 (H3K27).  Point mutations of EZH2 at Tyr 641 and Ala 677 occur in subpopulations of non-Hodgkin's lymphoma, where they drive H3K27 hypertrimethylation.  Here the authors report the discovery of EPZ005687, a potent inhibitor of EZH2 (Ki of 24 nM).  EPZ005687 has greater than 500-fold selectivity against 15 other protein methyltransferases and has 50-fold selectivity against the closely related enzyme EZH1.  The compd. reduces H3K27 methylation in various lymphoma cells; this translates into apoptotic cell killing in heterozygous Tyr 641 or Ala 677 mutant cells, with minimal effects on the proliferation of wild-type cells.  These data suggest that genetic alteration of EZH2 (for example, mutations at Tyr 641 or Ala 677) results in a crit. dependency on enzymic activity for proliferation (i.e., the equiv. of oncogene addiction), thus portending the clin. use of EZH2 inhibitors for cancers in which EZH2 is genetically altered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohlduVbTgWJrVg90H21EOLACvtfcHk0ljOM2pSlhD97A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVaqtrvP&md5=a9ad7a41505e3fd4aa087d9f5dae8825</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1084&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1084%26sid%3Dliteratum%253Aachs%26aulast%3DKnutson%26aufirst%3DS.%2BK.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DWarholic%26aufirst%3DN.%2BM.%26aulast%3DSneeringer%26aufirst%3DC.%2BJ.%26aulast%3DAllain%26aufirst%3DC.%2BJ.%26aulast%3DKlaus%26aufirst%3DC.%2BR.%26aulast%3DSacks%26aufirst%3DJ.%2BD.%26aulast%3DRaimondi%26aufirst%3DA.%26aulast%3DMajer%26aufirst%3DC.%2BR.%26aulast%3DSong%26aufirst%3DJ.%26aulast%3DScott%26aufirst%3DM.%2BP.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DOlhava%26aufirst%3DE.%2BJ.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DRichon%26aufirst%3DV.%2BM.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DKeilhack%26aufirst%3DH.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26aulast%3DKuntz%26aufirst%3DK.%2BW.%26atitle%3DA%2520selective%2520inhibitor%2520of%2520EZH2%2520blocks%2520H3K27%2520methylation%2520and%2520kills%2520mutant%2520lymphoma%2520cells%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2012%26volume%3D8%26spage%3D890%26epage%3D896%26doi%3D10.1038%2Fnchembio.1084" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCabe, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ott, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ganji, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korenchuk, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Aller, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graves, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Della Pietra, A.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diaz, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LaFrance, L. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mellinger, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duquenne, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kruger, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McHugh, C. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brandt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhanak, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verma, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tummino, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Creasy, C. L.</span></span> <span> </span><span class="NLM_article-title">EZH2 inhibition as a therapeutic strategy for lymphoma with EZH2-activating mutations</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>492</i></span>,  <span class="NLM_fpage">108</span>– <span class="NLM_lpage">112</span>, <span class="refDoi"> DOI: 10.1038/nature11606</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1038%2Fnature11606" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=23051747" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVyktbjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=492&publication_year=2012&pages=108-112&author=M.+T.+McCabeauthor=H.+M.+Ottauthor=G.+Ganjiauthor=S.+Korenchukauthor=C.+Thompsonauthor=G.+S.+Van+Allerauthor=Y.+Liuauthor=A.+P.+Gravesauthor=A.+Della+Pietraauthor=E.+Diazauthor=L.+V.+LaFranceauthor=M.+Mellingerauthor=C.+Duquenneauthor=X.+Tianauthor=R.+G.+Krugerauthor=C.+F.+McHughauthor=M.+Brandtauthor=W.+H.+Millerauthor=D.+Dhanakauthor=S.+K.+Vermaauthor=P.+J.+Tumminoauthor=C.+L.+Creasy&title=EZH2+inhibition+as+a+therapeutic+strategy+for+lymphoma+with+EZH2-activating+mutations&doi=10.1038%2Fnature11606"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">EZH2 inhibition as a therapeutic strategy for lymphoma with EZH2-activating mutations</span></div><div class="casAuthors">McCabe, Michael T.; Ott, Heidi M.; Ganji, Gopinath; Korenchuk, Susan; Thompson, Christine; Van Aller, Glenn S.; Liu, Yan; Graves, Alan P.; Della Pietra, Anthony , III; Diaz, Elsie; LaFrance, Louis V.; Mellinger, Mark; Duquenne, Celine; Tian, Xinrong; Kruger, Ryan G.; McHugh, Charles F.; Brandt, Martin; Miller, William H.; Dhanak, Dashyant; Verma, Sharad K.; Tummino, Peter J.; Creasy, Caretha L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">492</span>
        (<span class="NLM_cas:issue">7427</span>),
    <span class="NLM_cas:pages">108-112</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">In eukaryotes, post-translational modification of histones is crit. for regulation of chromatin structure and gene expression.  EZH2 is the catalytic subunit of the polycomb repressive complex 2 (PRC2) and is involved in repressing gene expression through methylation of histone H3 on lysine 27 (H3K27).  EZH2 overexpression is implicated in tumorigenesis and correlates with poor prognosis in several tumor types.  Addnl., somatic heterozygous mutations of Y641 and A677 residues within the catalytic SET domain of EZH2 occur in diffuse large B-cell lymphoma (DLBCL) and follicular lymphoma.  The Y641 residue is the most frequently mutated residue, with up to 22% of germinal center B-cell DLBCL and follicular lymphoma harbouring mutations at this site.  These lymphomas have increased H3K27 tri-methylation (H3K27me3) owing to altered substrate preferences of the mutant enzymes.  However, it is unknown whether specific, direct inhibition of EZH2 methyltransferase activity will be effective in treating EZH2 mutant lymphomas.  Here we demonstrate that GSK126, a potent, highly selective, S-adenosyl-methionine-competitive, small-mol. inhibitor of EZH2 methyltransferase activity, decreases global H3K27me3 levels and reactivates silenced PRC2 target genes.  GSK126 effectively inhibits the proliferation of EZH2 mutant DLBCL cell lines and markedly inhibits the growth of EZH2 mutant DLBCL xenografts in mice.  Together, these data demonstrate that pharmacol. inhibition of EZH2 activity may provide a promising treatment for EZH2 mutant lymphoma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxw5cLmW3Lc7Vg90H21EOLACvtfcHk0ljOM2pSlhD97A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVyktbjL&md5=a1dff99556de004c48aae88bc45dffab</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fnature11606&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11606%26sid%3Dliteratum%253Aachs%26aulast%3DMcCabe%26aufirst%3DM.%2BT.%26aulast%3DOtt%26aufirst%3DH.%2BM.%26aulast%3DGanji%26aufirst%3DG.%26aulast%3DKorenchuk%26aufirst%3DS.%26aulast%3DThompson%26aufirst%3DC.%26aulast%3DVan%2BAller%26aufirst%3DG.%2BS.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DGraves%26aufirst%3DA.%2BP.%26aulast%3DDella%2BPietra%26aufirst%3DA.%26aulast%3DDiaz%26aufirst%3DE.%26aulast%3DLaFrance%26aufirst%3DL.%2BV.%26aulast%3DMellinger%26aufirst%3DM.%26aulast%3DDuquenne%26aufirst%3DC.%26aulast%3DTian%26aufirst%3DX.%26aulast%3DKruger%26aufirst%3DR.%2BG.%26aulast%3DMcHugh%26aufirst%3DC.%2BF.%26aulast%3DBrandt%26aufirst%3DM.%26aulast%3DMiller%26aufirst%3DW.%2BH.%26aulast%3DDhanak%26aufirst%3DD.%26aulast%3DVerma%26aufirst%3DS.%2BK.%26aulast%3DTummino%26aufirst%3DP.%2BJ.%26aulast%3DCreasy%26aufirst%3DC.%2BL.%26atitle%3DEZH2%2520inhibition%2520as%2520a%2520therapeutic%2520strategy%2520for%2520lymphoma%2520with%2520EZH2-activating%2520mutations%26jtitle%3DNature%26date%3D2012%26volume%3D492%26spage%3D108%26epage%3D112%26doi%3D10.1038%2Fnature11606" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bate-Eya, L. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gierman, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ebus, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koster, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caron, H. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Versteeg, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dolman, M. E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molenaar, J. J.</span></span> <span> </span><span class="NLM_article-title">Enhancer of zeste homologue 2 plays an important role in neuroblastoma cell survival independent of its histone methyltransferase activity</span>. <i>Eur. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>75</i></span>,  <span class="NLM_fpage">63</span>– <span class="NLM_lpage">72</span>, <span class="refDoi"> DOI: 10.1016/j.ejca.2016.12.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1016%2Fj.ejca.2016.12.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=28214660" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC2sXit1eru70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2017&pages=63-72&author=L.+T.+Bate-Eyaauthor=H.+J.+Giermanauthor=M.+E.+Ebusauthor=J.+Kosterauthor=H.+N.+Caronauthor=R.+Versteegauthor=M.+E.+M.+Dolmanauthor=J.+J.+Molenaar&title=Enhancer+of+zeste+homologue+2+plays+an+important+role+in+neuroblastoma+cell+survival+independent+of+its+histone+methyltransferase+activity&doi=10.1016%2Fj.ejca.2016.12.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Enhancer of zeste homologue 2 plays an important role in neuroblastoma cell survival independent of its histone methyltransferase activity</span></div><div class="casAuthors">Bate-Eya, Laurel T.; Gierman, Hinco J.; Ebus, Marli E.; Koster, Jan; Caron, Huib N.; Versteeg, Rogier; Dolman, M. Emmy M.; Molenaar, Jan J.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Cancer</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">75</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">63-72</span>CODEN:
                <span class="NLM_cas:coden">EJCAEL</span>;
        ISSN:<span class="NLM_cas:issn">0959-8049</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Neuroblastoma is predominantly characterized by chromosomal rearrangements.  Next to V-Myc Avian Myelocytomatosis Viral Oncogene Neuroblastoma Derived Homolog (MYCN) amplification, chromosome 7 and 17q gains are frequently obsd.  We identified a neuroblastoma patient with a regional 7q36 gain, encompassing the enhancer of zeste homolog 2 (EZH2) gene.  EZH2 is the histone methyltransferase of lysine 27 of histone H3 (H3K27me3) that forms the catalytic subunit of the polycomb repressive complex 2.  H3K27me3 is commonly assocd. with the silencing of genes involved in cellular processes such as cell cycle regulation, cellular differentiation and cancer.  High EZH2 expression correlated with poor prognosis and overall survival independent of MYCN amplification status.  Unexpectedly, treatment of 3 EZH2-high expressing neuroblastoma cell lines (IMR32, CHP134 and NMB), with EZH2-specific inhibitors (GSK126 and EPZ6438) resulted in only a slight G1 arrest, despite max. histone methyltransferase activity inhibition.  Furthermore, colony formation in cell lines treated with the inhibitors was reduced only at concns. much higher than necessary for complete inhibition of EZH2 histone methyltransferase activity.  Knockdown of the complete protein with three independent shRNAs resulted in a strong apoptotic response and decreased cyclin D1 levels.  This apoptotic response could be rescued by overexpressing EZH2ΔSET, a truncated form of wild-type EZH2 lacking the SET transactivation domain necessary for histone methyltransferase activity.  Our findings suggest that high EZH2 expression, at least in neuroblastoma, has a survival function independent of its methyltransferase activity.  This important finding highlights the need for studies on EZH2 beyond its methyltransferase function and the requirement for compds. that will target EZH2 as a complete protein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPeYIEEK1oFLVg90H21EOLACvtfcHk0ljttbuTCOUsHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXit1eru70%253D&md5=65cbff2c51dc7c1af9d96cbeaf12cf11</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.ejca.2016.12.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejca.2016.12.019%26sid%3Dliteratum%253Aachs%26aulast%3DBate-Eya%26aufirst%3DL.%2BT.%26aulast%3DGierman%26aufirst%3DH.%2BJ.%26aulast%3DEbus%26aufirst%3DM.%2BE.%26aulast%3DKoster%26aufirst%3DJ.%26aulast%3DCaron%26aufirst%3DH.%2BN.%26aulast%3DVersteeg%26aufirst%3DR.%26aulast%3DDolman%26aufirst%3DM.%2BE.%2BM.%26aulast%3DMolenaar%26aufirst%3DJ.%2BJ.%26atitle%3DEnhancer%2520of%2520zeste%2520homologue%25202%2520plays%2520an%2520important%2520role%2520in%2520neuroblastoma%2520cell%2520survival%2520independent%2520of%2520its%2520histone%2520methyltransferase%2520activity%26jtitle%3DEur.%2520J.%2520Cancer%26date%3D2017%26volume%3D75%26spage%3D63%26epage%3D72%26doi%3D10.1016%2Fj.ejca.2016.12.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fong, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Licht, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span> <span> </span><span class="NLM_article-title">Polycomb- and Methylation-Independent Roles of EZH2 as a Transcription Activator</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">2808</span>– <span class="NLM_lpage">2820e4</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2018.11.035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1016%2Fj.celrep.2018.11.035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=30517868" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisVCit7bL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=2808-2820e4&author=J.+Kimauthor=Y.+Leeauthor=X.+Luauthor=B.+Songauthor=K.+W.+Fongauthor=Q.+Caoauthor=J.+D.+Lichtauthor=J.+C.+Zhaoauthor=J.+Yu&title=Polycomb-+and+Methylation-Independent+Roles+of+EZH2+as+a+Transcription+Activator&doi=10.1016%2Fj.celrep.2018.11.035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Polycomb- and Methylation-Independent Roles of EZH2 as a Transcription Activator</span></div><div class="casAuthors">Kim, Jung; Lee, Yongik; Lu, Xiaodong; Song, Bing; Fong, Ka-Wing; Cao, Qi; Licht, Jonathan D.; Zhao, Jonathan C.; Yu, Jindan</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2808-2820.e4</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Enhancer of Zeste 2 (EZH2) is the enzymic subunit of Polycomb Repressive Complex 2 (PRC2), which catalyzes histone H3 lysine 27 trimethylation (H3K27me3) at target promoters for gene silencing.  Here, we report that EZH2 activates androgen receptor (AR) gene transcription through direct occupancy at its promoter.  Importantly, this activating role of EZH2 is independent of PRC2 and its methyltransferase activities.  Genome-wide assays revealed extensive EZH2 occupancy at promoters marked by either H3K27ac or H3K27me3, leading to gene activation or repression, resp.  Last, we demonstrate enhanced efficacy of enzymic EZH2 inhibitors when used in combination with AR antagonists in blocking the dual roles of EZH2 and suppressing prostate cancer progression in vitro and in vivo.  Taken together, our study reports EZH2 as a transcriptional activator, a key target of which is AR, and suggests a drug-combinatory approach to treat advanced prostate cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGogFd6y_QesoLVg90H21EOLACvtfcHk0ljttbuTCOUsHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisVCit7bL&md5=795cd5933915f46a31e298075abaa247</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2018.11.035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2018.11.035%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DY.%26aulast%3DLu%26aufirst%3DX.%26aulast%3DSong%26aufirst%3DB.%26aulast%3DFong%26aufirst%3DK.%2BW.%26aulast%3DCao%26aufirst%3DQ.%26aulast%3DLicht%26aufirst%3DJ.%2BD.%26aulast%3DZhao%26aufirst%3DJ.%2BC.%26aulast%3DYu%26aufirst%3DJ.%26atitle%3DPolycomb-%2520and%2520Methylation-Independent%2520Roles%2520of%2520EZH2%2520as%2520a%2520Transcription%2520Activator%26jtitle%3DCell%2520Rep.%26date%3D2018%26volume%3D25%26spage%3D2808%26epage%3D2820e4%26doi%3D10.1016%2Fj.celrep.2018.11.035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, X. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cong, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geng, M.</span></span> <span> </span><span class="NLM_article-title">Targeting Epigenetic Crosstalk as a Therapeutic Strategy for EZH2-Aberrant Solid Tumors</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>175</i></span>,  <span class="NLM_fpage">186</span>– <span class="NLM_lpage">199.e19</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2018.08.058</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1016%2Fj.cell.2018.08.058" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=30220457" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslahs7%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=175&publication_year=2018&pages=186-199.e19&author=X.+Huangauthor=J.+Yanauthor=M.+Zhangauthor=Y.+Wangauthor=Y.+Chenauthor=X.+Fuauthor=R.+Weiauthor=X.+L.+Zhengauthor=Z.+Liuauthor=X.+Zhangauthor=H.+Yangauthor=B.+Haoauthor=Y.+Y.+Shenauthor=Y.+Suauthor=X.+Congauthor=M.+Huangauthor=M.+Tanauthor=J.+Dingauthor=M.+Geng&title=Targeting+Epigenetic+Crosstalk+as+a+Therapeutic+Strategy+for+EZH2-Aberrant+Solid+Tumors&doi=10.1016%2Fj.cell.2018.08.058"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Epigenetic Crosstalk as a Therapeutic Strategy for EZH2-Aberrant Solid Tumors</span></div><div class="casAuthors">Huang, Xun; Yan, Juan; Zhang, Min; Wang, Yafang; Chen, Yi; Fu, Xuhong; Wei, Rongrui; Zheng, Xing-ling; Liu, Zhiwei; Zhang, Xiong; Yang, Hong; Hao, Bingbing; Shen, Yan-yan; Su, Yi; Cong, Xiaoji; Huang, Min; Tan, Minjia; Ding, Jian; Geng, Meiyu</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">175</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">186-199.e19</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Mutations or aberrant upregulation of EZH2 occur frequently in human cancers, yet clin. benefits of EZH2 inhibitor (EZH2i) remain unsatisfactory and limited to certain hematol. malignancies.  We profile global posttranslational histone modification changes across a large panel of cancer cell lines with various sensitivities to EZH2i.  We report here oncogenic transcriptional reprogramming mediated by MLL1's interaction with the p300/CBP complex, which directs H3K27me loss to reciprocal H3K27ac gain and restricts EZH2i response.  Concurrent inhibition of H3K27me and H3K27ac results in transcriptional repression and MAPK pathway dependency in cancer subsets.  In preclin. models encompassing a broad spectrum of EZH2-aberrant solid tumors, a combination of EZH2 and BRD4 inhibitors, or a triple-combination including MAPK inhibition display robust efficacy with very tolerable toxicity.  Our results suggest an attractive precision treatment strategy for EZH2-aberrant tumors on the basis of tumor-intrinsic MLL1 expression and concurrent inhibition of epigenetic crosstalk and feedback MAPK activation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgroiKQYI5RLVg90H21EOLACvtfcHk0ljttbuTCOUsHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslahs7%252FM&md5=1879411de33f088e3c197d6175e02e18</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2018.08.058&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2018.08.058%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DYan%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DFu%26aufirst%3DX.%26aulast%3DWei%26aufirst%3DR.%26aulast%3DZheng%26aufirst%3DX.%2BL.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DHao%26aufirst%3DB.%26aulast%3DShen%26aufirst%3DY.%2BY.%26aulast%3DSu%26aufirst%3DY.%26aulast%3DCong%26aufirst%3DX.%26aulast%3DHuang%26aufirst%3DM.%26aulast%3DTan%26aufirst%3DM.%26aulast%3DDing%26aufirst%3DJ.%26aulast%3DGeng%26aufirst%3DM.%26atitle%3DTargeting%2520Epigenetic%2520Crosstalk%2520as%2520a%2520Therapeutic%2520Strategy%2520for%2520EZH2-Aberrant%2520Solid%2520Tumors%26jtitle%3DCell%26date%3D2018%26volume%3D175%26spage%3D186%26epage%3D199.e19%26doi%3D10.1016%2Fj.cell.2018.08.058" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gibaja, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harari, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korn, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruddy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saenz-Vash, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhai, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rejtar, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paris, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hassan, A. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, H. M.</span></span> <span> </span><span class="NLM_article-title">Development of secondary mutations in wild-type and mutant EZH2 alleles cooperates to confer resistance to EZH2 inhibitors</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">558</span>– <span class="NLM_lpage">566</span>, <span class="refDoi"> DOI: 10.1038/onc.2015.114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1038%2Fonc.2015.114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=25893294" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFynt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2016&pages=558-566&author=V.+Gibajaauthor=F.+Shenauthor=J.+Harariauthor=J.+Kornauthor=D.+Ruddyauthor=V.+Saenz-Vashauthor=H.+Zhaiauthor=T.+Rejtarauthor=C.+G.+Parisauthor=Z.+Yuauthor=M.+Liraauthor=D.+Kingauthor=W.+Qiauthor=N.+Keenauthor=A.+Q.+Hassanauthor=H.+M.+Chan&title=Development+of+secondary+mutations+in+wild-type+and+mutant+EZH2+alleles+cooperates+to+confer+resistance+to+EZH2+inhibitors&doi=10.1038%2Fonc.2015.114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Development of secondary mutations in wild-type and mutant EZH2 alleles cooperates to confer resistance to EZH2 inhibitors</span></div><div class="casAuthors">Gibaja, V.; Shen, F.; Harari, J.; Korn, J.; Ruddy, D.; Saenz-Vash, V.; Zhai, H.; Rejtar, T.; Paris, C. G.; Yu, Z.; Lira, M.; King, D.; Qi, W.; Keen, N.; Hassan, A. Q.; Chan, H. M.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">558-566</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The histone methyltransferase Enhancer of Zeste Homolog 2 (EZH2) is frequently dysregulated in cancers, and gain-of-function (GOF) EZH2 mutations have been identified in non-Hodgkin lymphomas.  Small-mol. inhibitors against EZH2 demonstrated anti-tumor activity in EZH2-mutated lymphomas and entered clin. trials.  Here, we developed models of acquired resistance to EZH2 inhibitor EI1 with EZH2-mutated lymphoma cells.  Resistance was generated by secondary mutations in both wild-type (WT) and GOF Y641N EZH2 alleles.  These EZH2 mutants retained the substrate specificity of their predecessor complexes but became refractory to biochem. inhibition by EZH2 inhibitors.  Resistant cells were able to maintain a high level of H3K27Me3 in the presence of inhibitors.  Interestingly, mutation of EZH2 WT alone generated an intermediate resistance phenotype, which is consistent with a previously proposed model of cooperation between EZH2 WT and Y641N mutants to promote tumorigenesis.  In addn., the findings presented here have implications for the clin. translation of EZH2 inhibitors and underscore the need to develop novel EZH2 inhibitors to target potential resistance emerging in clin. settings.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUaTpSYCtEf7Vg90H21EOLACvtfcHk0liAqznafoGjCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFynt74%253D&md5=fcfebb482305effcb67f258ce8c589e8</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fonc.2015.114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2015.114%26sid%3Dliteratum%253Aachs%26aulast%3DGibaja%26aufirst%3DV.%26aulast%3DShen%26aufirst%3DF.%26aulast%3DHarari%26aufirst%3DJ.%26aulast%3DKorn%26aufirst%3DJ.%26aulast%3DRuddy%26aufirst%3DD.%26aulast%3DSaenz-Vash%26aufirst%3DV.%26aulast%3DZhai%26aufirst%3DH.%26aulast%3DRejtar%26aufirst%3DT.%26aulast%3DParis%26aufirst%3DC.%2BG.%26aulast%3DYu%26aufirst%3DZ.%26aulast%3DLira%26aufirst%3DM.%26aulast%3DKing%26aufirst%3DD.%26aulast%3DQi%26aufirst%3DW.%26aulast%3DKeen%26aufirst%3DN.%26aulast%3DHassan%26aufirst%3DA.%2BQ.%26aulast%3DChan%26aufirst%3DH.%2BM.%26atitle%3DDevelopment%2520of%2520secondary%2520mutations%2520in%2520wild-type%2520and%2520mutant%2520EZH2%2520alleles%2520cooperates%2520to%2520confer%2520resistance%2520to%2520EZH2%2520inhibitors%26jtitle%3DOncogene%26date%3D2016%26volume%3D35%26spage%3D558%26epage%3D566%26doi%3D10.1038%2Fonc.2015.114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burslem, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Proteolysis-Targeting Chimeras as Therapeutics and Tools for Biological Discovery</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>181</i></span>,  <span class="NLM_fpage">102</span>– <span class="NLM_lpage">114</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2019.11.031</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1016%2Fj.cell.2019.11.031" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=31955850" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BB3cXht1Sitrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=181&publication_year=2020&pages=102-114&author=G.+M.+Burslemauthor=C.+M.+Crews&title=Proteolysis-Targeting+Chimeras+as+Therapeutics+and+Tools+for+Biological+Discovery&doi=10.1016%2Fj.cell.2019.11.031"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Proteolysis-Targeting Chimeras as Therapeutics and Tools for Biological Discovery</span></div><div class="casAuthors">Burslem, George M.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">181</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">102-114</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  New biol. tools provide new techniques to probe fundamental biol. processes.  Here we describe the burgeoning field of proteolysis-targeting chimeras (PROTACs), which are capable of modulating protein concns. at a post-translational level by co-opting the ubiquitin-proteasome system.  We describe the PROTAC technol. and its application to drug discovery and provide examples where PROTACs have enabled novel biol. insights.  Furthermore, we provide a workflow for PROTAC development and use and discuss the benefits and issues assocd. with PROTACs.  Finally, we compare PROTAC-mediated protein-level modulation with other technologies, such as RNAi and genome editing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXgumsEakVWrVg90H21EOLACvtfcHk0liAqznafoGjCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXht1Sitrk%253D&md5=3010929d2979cf04dc6003e66575f1b0</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2019.11.031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2019.11.031%26sid%3Dliteratum%253Aachs%26aulast%3DBurslem%26aufirst%3DG.%2BM.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DProteolysis-Targeting%2520Chimeras%2520as%2520Therapeutics%2520and%2520Tools%2520for%2520Biological%2520Discovery%26jtitle%3DCell%26date%3D2020%26volume%3D181%26spage%3D102%26epage%3D114%26doi%3D10.1016%2Fj.cell.2019.11.031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altieri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raina, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Hijacking the E3 Ubiquitin Ligase Cereblon to Efficiently Target BRD4</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">755</span>– <span class="NLM_lpage">763</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2015.05.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1016%2Fj.chembiol.2015.05.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=26051217" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVWrtbrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2015&pages=755-763&author=J.+Luauthor=Y.+Qianauthor=M.+Altieriauthor=H.+Dongauthor=J.+Wangauthor=K.+Rainaauthor=J.+Hinesauthor=J.+D.+Winklerauthor=A.+P.+Crewauthor=K.+Colemanauthor=C.+M.+Crews&title=Hijacking+the+E3+Ubiquitin+Ligase+Cereblon+to+Efficiently+Target+BRD4&doi=10.1016%2Fj.chembiol.2015.05.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Hijacking the E3 Ubiquitin Ligase Cereblon to Efficiently Target BRD4</span></div><div class="casAuthors">Lu, Jing; Qian, Yimin; Altieri, Martha; Dong, Hanqing; Wang, Jing; Raina, Kanak; Hines, John; Winkler, James D.; Crew, Andrew P.; Coleman, Kevin; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">755-763</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">BRD4, a bromodomain and extraterminal domain (BET) family member, is an attractive target in multiple pathol. settings, particularly cancer.  While BRD4 inhibitors have shown some promise in MYC-driven malignancies such as Burkitt's lymphoma (BL), we show that BRD4 inhibitors lead to robust BRD4 protein accumulation, which may account for their limited suppression of MYC expression, modest antiproliferative activity, and lack of apoptotic induction.  To address these limitations we designed ARV-825, a hetero-bifunctional PROTAC (Proteolysis Targeting Chimera) that recruits BRD4 to the E3 ubiquitin ligase cereblon, leading to fast, efficient, and prolonged degrdn. of BRD4 in all BL cell lines tested.  Consequently, ARV-825 more effectively suppresses c-MYC levels and downstream signaling than small-mol. BRD4 inhibitors, resulting in more effective cell proliferation inhibition and apoptosis induction in BL.  Our findings provide strong evidence that cereblon-based PROTACs provide a better and more efficient strategy in targeting BRD4 than traditional small-mol. inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6Ol4grCG2RbVg90H21EOLACvtfcHk0ljpwbh6Y8GylA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVWrtbrM&md5=a64675e7a0cd006a8737be9730f447e3</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2015.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2015.05.009%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DAltieri%26aufirst%3DM.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DRaina%26aufirst%3DK.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DColeman%26aufirst%3DK.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DHijacking%2520the%2520E3%2520Ubiquitin%2520Ligase%2520Cereblon%2520to%2520Efficiently%2520Target%2520BRD4%26jtitle%3DChem.%2520Biol.%26date%3D2015%26volume%3D22%26spage%3D755%26epage%3D763%26doi%3D10.1016%2Fj.chembiol.2015.05.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Souza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhe-Paganon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">DRUG DEVELOPMENT. Phthalimide conjugation as a strategy for in vivo target protein degradation</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>348</i></span>,  <span class="NLM_fpage">1376</span>– <span class="NLM_lpage">1381</span>, <span class="refDoi"> DOI: 10.1126/science.aab1433</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1126%2Fscience.aab1433" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=25999370" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC2MXpvFansLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=348&publication_year=2015&pages=1376-1381&author=G.+E.+Winterauthor=D.+L.+Buckleyauthor=J.+Paulkauthor=J.+M.+Robertsauthor=A.+Souzaauthor=S.+Dhe-Paganonauthor=J.+E.+Bradner&title=DRUG+DEVELOPMENT.+Phthalimide+conjugation+as+a+strategy+for+in+vivo+target+protein+degradation&doi=10.1126%2Fscience.aab1433"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Phthalimide conjugation as a strategy for in vivo target protein degradation</span></div><div class="casAuthors">Winter, Georg E.; Buckley, Dennis L.; Paulk, Joshiawa; Roberts, Justin M.; Souza, Amanda; Dhe-Paganon, Sirano; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">348</span>
        (<span class="NLM_cas:issue">6241</span>),
    <span class="NLM_cas:pages">1376-1381</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The development of effective pharmacol. inhibitors of multidomain scaffold proteins, notably transcription factors, is a particularly challenging problem.  In part, this is because many small-mol. antagonists disrupt the activity of only one domain in the target protein.  The authors devised a chem. strategy that promotes ligand-dependent target protein degrdn. using as an example the transcriptional coactivator BRD4, a protein crit. for cancer cell growth and survival.  The authors appended a competitive antagonist of BET bromodomains to a phthalimide moiety to hijack the cereblon E3 ubiquitin ligase complex.  The resultant compd., dBET1, induced highly selective cereblon-dependent BET protein degrdn. in vitro and in vivo and delayed leukemia progression in mice.  A second series of probes resulted in selective degrdn. of the cytosolic protein FKBP12.  This chem. strategy for controlling target protein stability may have implications for therapeutically targeting previously intractable proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1ZW4zjRjXi7Vg90H21EOLACvtfcHk0ljpwbh6Y8GylA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXpvFansLk%253D&md5=b0277cc95d318d926b96e7a119066e3b</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1126%2Fscience.aab1433&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aab1433%26sid%3Dliteratum%253Aachs%26aulast%3DWinter%26aufirst%3DG.%2BE.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DRoberts%26aufirst%3DJ.%2BM.%26aulast%3DSouza%26aufirst%3DA.%26aulast%3DDhe-Paganon%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DDRUG%2520DEVELOPMENT.%2520Phthalimide%2520conjugation%2520as%2520a%2520strategy%2520for%2520in%2520vivo%2520target%2520protein%2520degradation%26jtitle%3DScience%26date%3D2015%26volume%3D348%26spage%3D1376%26epage%3D1381%26doi%3D10.1126%2Fscience.aab1433" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Han, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Salas, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaswamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delproposto, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of ARD-69 as a Highly Potent Proteolysis Targeting Chimera (PROTAC) Degrader of Androgen Receptor (AR) for the Treatment of Prostate Cancer</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">941</span>– <span class="NLM_lpage">964</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01631</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01631" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC1MXlvFCjtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=941-964&author=X.+Hanauthor=C.+Wangauthor=C.+Qinauthor=W.+Xiangauthor=E.+Fernandez-Salasauthor=C.+Y.+Yangauthor=M.+Wangauthor=L.+Zhaoauthor=T.+Xuauthor=K.+Chinnaswamyauthor=J.+Delpropostoauthor=J.+Stuckeyauthor=S.+Wang&title=Discovery+of+ARD-69+as+a+Highly+Potent+Proteolysis+Targeting+Chimera+%28PROTAC%29+Degrader+of+Androgen+Receptor+%28AR%29+for+the+Treatment+of+Prostate+Cancer&doi=10.1021%2Facs.jmedchem.8b01631"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of ARD-69 as a highly potent proteolysis targeting chimera (PROTAC) degrader of androgen receptor (AR) for the treatment of prostate cancer</span></div><div class="casAuthors">Han, Xin; Wang, Chao; Qin, Chong; Xiang, Weiguo; Fernandez-Salas, Ester; Yang, Chao-Yie; Wang, Mi; Zhao, Lijie; Xu, Tianfeng; Chinnaswamy, Krishnapriya; Delproposto, James; Stuckey, Jeanne; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">941-964</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We report herein the discovery of highly potent PROTAC degraders of androgen receptor (AR), as exemplified by IARD-69.  I induces degrdn. of AR protein in AR-pos. prostate cancer cell lines in a dose- and time-dependent manner.  I achieves DC50 values of 0.86, 0.76, and 10.4 nM in LNCaP, VCaP, and 22Rv1 AR+ prostate cancer cell lines, resp.  I is capable of reducing the AR protein level by >95% in these prostate cancer cell lines and effectively suppressing AR-regulated gene expression.  I potently inhibits cell growth in these AR-pos. prostate cancer cell lines and is >100 times more potent than AR antagonists.  A single dose of I effectively reduces the level of AR protein in xenograft tumor tissue in mice.  Further optimization of I may ultimately lead to a new therapy for AR+, castration-resistant prostate cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTlOD69tfhtbVg90H21EOLACvtfcHk0ljpwbh6Y8GylA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXlvFCjtQ%253D%253D&md5=2376573fb1a1e19c517d64c70199700a</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01631&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01631%26sid%3Dliteratum%253Aachs%26aulast%3DHan%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DQin%26aufirst%3DC.%26aulast%3DXiang%26aufirst%3DW.%26aulast%3DFernandez-Salas%26aufirst%3DE.%26aulast%3DYang%26aufirst%3DC.%2BY.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DZhao%26aufirst%3DL.%26aulast%3DXu%26aufirst%3DT.%26aulast%3DChinnaswamy%26aufirst%3DK.%26aulast%3DDelproposto%26aufirst%3DJ.%26aulast%3DStuckey%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520ARD-69%2520as%2520a%2520Highly%2520Potent%2520Proteolysis%2520Targeting%2520Chimera%2520%2528PROTAC%2529%2520Degrader%2520of%2520Androgen%2520Receptor%2520%2528AR%2529%2520for%2520the%2520Treatment%2520of%2520Prostate%2520Cancer%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D941%26epage%3D964%26doi%3D10.1021%2Facs.jmedchem.8b01631" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaswamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meagher, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">A Potent and Selective Small-Molecule Degrader of STAT3 Achieves Complete Tumor Regression In Vivo</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">498</span>– <span class="NLM_lpage">511e17</span>, <span class="refDoi"> DOI: 10.1016/j.ccell.2019.10.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1016%2Fj.ccell.2019.10.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=31715132" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitFCjtrjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2019&pages=498-511e17&author=L.+Baiauthor=H.+Zhouauthor=R.+Xuauthor=Y.+Zhaoauthor=K.+Chinnaswamyauthor=D.+McEachernauthor=J.+Chenauthor=C.+Y.+Yangauthor=Z.+Liuauthor=M.+Wangauthor=L.+Liuauthor=H.+Jiangauthor=B.+Wenauthor=P.+Kumarauthor=J.+L.+Meagherauthor=D.+Sunauthor=J.+A.+Stuckeyauthor=S.+Wang&title=A+Potent+and+Selective+Small-Molecule+Degrader+of+STAT3+Achieves+Complete+Tumor+Regression+In+Vivo&doi=10.1016%2Fj.ccell.2019.10.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">A Potent and Selective Small-Molecule Degrader of STAT3 Achieves Complete Tumor Regression In Vivo</span></div><div class="casAuthors">Bai, Longchuan; Zhou, Haibin; Xu, Renqi; Zhao, Yujun; Chinnaswamy, Krishnapriya; McEachern, Donna; Chen, Jianyong; Yang, Chao-Yie; Liu, Zhaomin; Wang, Mi; Liu, Liu; Jiang, Hui; Wen, Bo; Kumar, Praveen; Meagher, Jennifer L.; Sun, Duxin; Stuckey, Jeanne A.; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">498-511.e17</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Signal transducer and activator of transcription 3 (STAT3) is an attractive cancer therapeutic target.  Here we report the discovery of SD-36(I), a small-mol. degrader of STAT3.  SD-36 potently induces the degrdn. of STAT3 protein in vitro and in vivo and demonstrates high selectivity over other STAT members.  Induced degrdn. of STAT3 results in a strong suppression of its transcription network in leukemia and lymphoma cells.  SD-36 inhibits the growth of a subset of acute myeloid leukemia and anaplastic large-cell lymphoma cell lines by inducing cell-cycle arrest and/or apoptosis.  SD-36 achieves complete and long-lasting tumor regression in multiple xenograft mouse models at well-tolerated dose schedules.  Degrdn. of STAT3 protein, therefore, is a promising cancer therapeutic strategy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUaP49Ap9DxLVg90H21EOLACvtfcHk0lgfVVxue3qQ8w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitFCjtrjO&md5=71252c17d50a1a8be829352942175bd6</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2019.10.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2019.10.002%26sid%3Dliteratum%253Aachs%26aulast%3DBai%26aufirst%3DL.%26aulast%3DZhou%26aufirst%3DH.%26aulast%3DXu%26aufirst%3DR.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DChinnaswamy%26aufirst%3DK.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DC.%2BY.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DKumar%26aufirst%3DP.%26aulast%3DMeagher%26aufirst%3DJ.%2BL.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DStuckey%26aufirst%3DJ.%2BA.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DA%2520Potent%2520and%2520Selective%2520Small-Molecule%2520Degrader%2520of%2520STAT3%2520Achieves%2520Complete%2520Tumor%2520Regression%2520In%2520Vivo%26jtitle%3DCancer%2520Cell%26date%3D2019%26volume%3D36%26spage%3D498%26epage%3D511e17%26doi%3D10.1016%2Fj.ccell.2019.10.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Farnaby, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koegl, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roy, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitworth, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diers, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trainor, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zollman, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steurer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karolyi-Oezguer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riedmueller, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gmaschitz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wachter, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dank, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galant, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharps, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rumpel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Traxler, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerstberger, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schnitzer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petermann, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greb, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinstabl, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bader, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoephel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss-Puxbaum, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ehrenhofer-Wolfer, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wohrle, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boehmelt, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rinnenthal, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnhof, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiechens, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, M. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owen-Hughes, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ettmayer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McConnell, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">BAF complex vulnerabilities in cancer demonstrated via structure-based PROTAC design</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">672</span>– <span class="NLM_lpage">680</span>, <span class="refDoi"> DOI: 10.1038/s41589-019-0294-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1038%2Fs41589-019-0294-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=31178587" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtFeqtLfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2019&pages=672-680&author=W.+Farnabyauthor=M.+Koeglauthor=M.+J.+Royauthor=C.+Whitworthauthor=E.+Diersauthor=N.+Trainorauthor=D.+Zollmanauthor=S.+Steurerauthor=J.+Karolyi-Oezguerauthor=C.+Riedmuellerauthor=T.+Gmaschitzauthor=J.+Wachterauthor=C.+Dankauthor=M.+Galantauthor=B.+Sharpsauthor=K.+Rumpelauthor=E.+Traxlerauthor=T.+Gerstbergerauthor=R.+Schnitzerauthor=O.+Petermannauthor=P.+Grebauthor=H.+Weinstablauthor=G.+Baderauthor=A.+Zoephelauthor=A.+Weiss-Puxbaumauthor=K.+Ehrenhofer-Wolferauthor=S.+Wohrleauthor=G.+Boehmeltauthor=J.+Rinnenthalauthor=H.+Arnhofauthor=N.+Wiechensauthor=M.+Y.+Wuauthor=T.+Owen-Hughesauthor=P.+Ettmayerauthor=M.+Pearsonauthor=D.+B.+McConnellauthor=A.+Ciulli&title=BAF+complex+vulnerabilities+in+cancer+demonstrated+via+structure-based+PROTAC+design&doi=10.1038%2Fs41589-019-0294-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">BAF complex vulnerabilities in cancer demonstrated via structure-based PROTAC design</span></div><div class="casAuthors">Farnaby, William; Koegl, Manfred; Roy, Michael J.; Whitworth, Claire; Diers, Emelyne; Trainor, Nicole; Zollman, David; Steurer, Steffen; Karolyi-Oezguer, Jale; Riedmueller, Carina; Gmaschitz, Teresa; Wachter, Johannes; Dank, Christian; Galant, Michael; Sharps, Bernadette; Rumpel, Klaus; Traxler, Elisabeth; Gerstberger, Thomas; Schnitzer, Renate; Petermann, Oliver; Greb, Peter; Weinstabl, Harald; Bader, Gerd; Zoephel, Andreas; Weiss-Puxbaum, Alexander; Ehrenhofer-Wolfer, Katharina; Wohrle, Simon; Boehmelt, Guido; Rinnenthal, Joerg; Arnhof, Heribert; Wiechens, Nicola; Wu, Meng-Ying; Owen-Hughes, Tom; Ettmayer, Peter; Pearson, Mark; McConnell, Darryl B.; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">672-680</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Targeting subunits of BAF/PBAF chromatin remodeling complexes has been proposed as an approach to exploit cancer vulnerabilities.  Here, we develop proteolysis targeting chimera (PROTAC) degraders of the BAF ATPase subunits SMARCA2 and SMARCA4 using a bromodomain ligand and recruitment of the E3 ubiquitin ligase VHL.  High-resoln. ternary complex crystal structures and biophys. investigation guided rational and efficient optimization toward ACBI1(I), a potent and cooperative degrader of SMARCA2, SMARCA4 and PBRM1.  I induced anti-proliferative effects and cell death caused by SMARCA2 depletion in SMARCA4 mutant cancer cells, and in acute myeloid leukemia cells dependent on SMARCA4 ATPase activity.  These findings exemplify a successful biophysics- and structure-based PROTAC design approach to degrade high profile drug targets, and pave the way toward new therapeutics for the treatment of tumors sensitive to the loss of BAF complex ATPases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonEh4pux4Bs7Vg90H21EOLACvtfcHk0lgfVVxue3qQ8w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtFeqtLfJ&md5=e95fad12c727aedcf08aed9320dae905</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fs41589-019-0294-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-019-0294-6%26sid%3Dliteratum%253Aachs%26aulast%3DFarnaby%26aufirst%3DW.%26aulast%3DKoegl%26aufirst%3DM.%26aulast%3DRoy%26aufirst%3DM.%2BJ.%26aulast%3DWhitworth%26aufirst%3DC.%26aulast%3DDiers%26aufirst%3DE.%26aulast%3DTrainor%26aufirst%3DN.%26aulast%3DZollman%26aufirst%3DD.%26aulast%3DSteurer%26aufirst%3DS.%26aulast%3DKarolyi-Oezguer%26aufirst%3DJ.%26aulast%3DRiedmueller%26aufirst%3DC.%26aulast%3DGmaschitz%26aufirst%3DT.%26aulast%3DWachter%26aufirst%3DJ.%26aulast%3DDank%26aufirst%3DC.%26aulast%3DGalant%26aufirst%3DM.%26aulast%3DSharps%26aufirst%3DB.%26aulast%3DRumpel%26aufirst%3DK.%26aulast%3DTraxler%26aufirst%3DE.%26aulast%3DGerstberger%26aufirst%3DT.%26aulast%3DSchnitzer%26aufirst%3DR.%26aulast%3DPetermann%26aufirst%3DO.%26aulast%3DGreb%26aufirst%3DP.%26aulast%3DWeinstabl%26aufirst%3DH.%26aulast%3DBader%26aufirst%3DG.%26aulast%3DZoephel%26aufirst%3DA.%26aulast%3DWeiss-Puxbaum%26aufirst%3DA.%26aulast%3DEhrenhofer-Wolfer%26aufirst%3DK.%26aulast%3DWohrle%26aufirst%3DS.%26aulast%3DBoehmelt%26aufirst%3DG.%26aulast%3DRinnenthal%26aufirst%3DJ.%26aulast%3DArnhof%26aufirst%3DH.%26aulast%3DWiechens%26aufirst%3DN.%26aulast%3DWu%26aufirst%3DM.%2BY.%26aulast%3DOwen-Hughes%26aufirst%3DT.%26aulast%3DEttmayer%26aufirst%3DP.%26aulast%3DPearson%26aufirst%3DM.%26aulast%3DMcConnell%26aufirst%3DD.%2BB.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DBAF%2520complex%2520vulnerabilities%2520in%2520cancer%2520demonstrated%2520via%2520structure-based%2520PROTAC%2520design%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2019%26volume%3D15%26spage%3D672%26epage%3D680%26doi%3D10.1038%2Fs41589-019-0294-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Y.</span></span> <span> </span><span class="NLM_article-title">PROTAC-induced BTK degradation as a novel therapy for mutated BTK C481S induced ibrutinib-resistant B-cell malignancies</span>. <i>Cell Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">779</span>– <span class="NLM_lpage">781</span>, <span class="refDoi"> DOI: 10.1038/s41422-018-0055-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1038%2Fs41422-018-0055-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=29875397" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1ektbzO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2018&pages=779-781&author=Y.+Sunauthor=X.+Zhaoauthor=N.+Dingauthor=H.+Gaoauthor=Y.+Wuauthor=Y.+Yangauthor=M.+Zhaoauthor=J.+Hwangauthor=Y.+Songauthor=W.+Liuauthor=Y.+Rao&title=PROTAC-induced+BTK+degradation+as+a+novel+therapy+for+mutated+BTK+C481S+induced+ibrutinib-resistant+B-cell+malignancies&doi=10.1038%2Fs41422-018-0055-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">PROTAC-induced BTK degradation as a novel therapy for mutated BTK C481S induced ibrutinib-resistant B-cell malignancies</span></div><div class="casAuthors">Sun, Yonghui; Zhao, Xingwang; Ding, Ning; Gao, Hongying; Wu, Yue; Yang, Yiqing; Zhao, Meng; Hwang, Jinseok; Song, Yuqin; Liu, Wanli; Rao, Yu</div><div class="citationInfo"><span class="NLM_cas:title">Cell Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">779-781</span>CODEN:
                <span class="NLM_cas:coden">CREEB6</span>;
        ISSN:<span class="NLM_cas:issn">1001-0602</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">In this study, for the first time, we report the development of BTK-targeting degraders using the PROTAC strategy.  These PROTACs could ef fi ciently degrade ibrutinib-sensitive BTK-WT (wild type).  More importantly, our newly designed PROTACs also significantly induced the degrdn. of ibrutinib resistant BTK-C481S (50% degrdn. efficiency at 30 nM).  Furthermore, our PROTAC mols. ef fi ciently inhibited cell proliferation and colony formation, while exhibited no obvious inhibition (>1000 nM) of ITK, EGFR, and TEC, which are major off-targets of ibrutinib.  These data demonstrate the strong potential for developing PROTAC-based therapeutic mols.  The results indicate that the PROTAC strategy (protein degrdn. instead of inhibition) could be adopted as a general and powerful therapeutic treatment for drug-resistant cancers in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqg_FLdaUP_HLVg90H21EOLACvtfcHk0lge8NcaC4XAaQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1ektbzO&md5=1cca51ac76ecb3b491da0117b21271fb</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fs41422-018-0055-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41422-018-0055-1%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DDing%26aufirst%3DN.%26aulast%3DGao%26aufirst%3DH.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DM.%26aulast%3DHwang%26aufirst%3DJ.%26aulast%3DSong%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DRao%26aufirst%3DY.%26atitle%3DPROTAC-induced%2520BTK%2520degradation%2520as%2520a%2520novel%2520therapy%2520for%2520mutated%2520BTK%2520C481S%2520induced%2520ibrutinib-resistant%2520B-cell%2520malignancies%26jtitle%3DCell%2520Res.%26date%3D2018%26volume%3D28%26spage%3D779%26epage%3D781%26doi%3D10.1038%2Fs41422-018-0055-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tomaselli, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mautone, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rotili, D.</span></span> <span> </span><span class="NLM_article-title">Recent advances in epigenetic proteolysis targeting chimeras (Epi-PROTACs)</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>207</i></span>, <span class="NLM_elocation-id">112750</span> <span class="refDoi"> DOI: 10.1016/j.ejmech.2020.112750</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1016%2Fj.ejmech.2020.112750" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=32871345" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhvVWhtbjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=207&publication_year=2020&author=D.+Tomaselliauthor=N.+Mautoneauthor=A.+Maiauthor=D.+Rotili&title=Recent+advances+in+epigenetic+proteolysis+targeting+chimeras+%28Epi-PROTACs%29&doi=10.1016%2Fj.ejmech.2020.112750"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Recent advances in epigenetic proteolysis targeting chimeras (Epi-PROTACs)</span></div><div class="casAuthors">Tomaselli, Daniela; Mautone, Nicola; Mai, Antonello; Rotili, Dante</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">207</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">112750</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A review.  PROteolysis TArgeting Chimeras (PROTACs) are heterobifunctional mols. that trigger the poly-ubiquitination of the protein of interest (POI) inducing its degrdn. via the recruitment of the ubiquitin-proteasome system, thus suppressing the POI's intracellular levels and indirectly all its functions.  Recently, one of the fields where the protein knockdown induced by PROTACs has demonstrated to serve as a promising biochem. tool and to provide new opportunities for drug discovery is the epigenetics (epi-PROTACs).  A full inhibition of the functions of all domains of a specific epigenetic POI (e-POI), rather than just the block of its catalytic/single domain activity, is in fact a new more effective modality to hit an e-POI and, in principle, the complex it belongs to, and potentially to treat the related diseases, first cancer.  In this review, we will present the most relevant progresses made, esp. in the last two years, in the application of PROTACs technol. to the three main classes of e-POIs: "writers", "erasers" and "readers".  Emphasis will be devoted to the medicinal chem. aspects of the epi-PROTACs design, prepn., and optimization and to the comparison with small mol. epi-drugs for both epi-targets functional annotation and potential anticancer therapy purposes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrd2A6SgsU0hbVg90H21EOLACvtfcHk0lge8NcaC4XAaQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhvVWhtbjI&md5=d75dbf0e84fee63f89c9f79177ce1982</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2020.112750&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2020.112750%26sid%3Dliteratum%253Aachs%26aulast%3DTomaselli%26aufirst%3DD.%26aulast%3DMautone%26aufirst%3DN.%26aulast%3DMai%26aufirst%3DA.%26aulast%3DRotili%26aufirst%3DD.%26atitle%3DRecent%2520advances%2520in%2520epigenetic%2520proteolysis%2520targeting%2520chimeras%2520%2528Epi-PROTACs%2529%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2020%26volume%3D207%26doi%3D10.1016%2Fj.ejmech.2020.112750" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vogelmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robaa, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sippl, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, M.</span></span> <span> </span><span class="NLM_article-title">Proteolysis targeting chimeras (PROTACs) for epigenetics research</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">8</span>– <span class="NLM_lpage">16</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2020.01.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1016%2Fj.cbpa.2020.01.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=32146413" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BB3cXksFeitrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2020&pages=8-16&author=A.+Vogelmannauthor=D.+Robaaauthor=W.+Sipplauthor=M.+Jung&title=Proteolysis+targeting+chimeras+%28PROTACs%29+for+epigenetics+research&doi=10.1016%2Fj.cbpa.2020.01.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Proteolysis targeting chimeras (PROTACs) for epigenetics research</span></div><div class="casAuthors">Vogelmann, Anja; Robaa, Dina; Sippl, Wolfgang; Jung, Manfred</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">8-16</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Proteolysis targeting chimeras (PROTACs) are heterobifunctional mols. and allow selective protein degrdn. by addressing the natural ubiquitin proteasome system.  As this new strategy of chem. induced protein degrdn. can serve as a biol. tool and provides new possibilities for drug discovery, it has been applied to a variety of targets including (nuclear) receptors, kinases, and epigenetic proteins.  A lot of PROTACs have already been designed in the field of epigenetics, and their synthesis and characterization highly contributed to structural optimization and improved mechanistic understanding of these mols.  In this review, we will discuss and summarize recent advances in PROTAC discovery with focus on epigenetic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqW2o_9L2Y8GbVg90H21EOLACvtfcHk0lge8NcaC4XAaQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXksFeitrw%253D&md5=6a4f3b70d6c6a446c744b9788b7582cf</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2020.01.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2020.01.010%26sid%3Dliteratum%253Aachs%26aulast%3DVogelmann%26aufirst%3DA.%26aulast%3DRobaa%26aufirst%3DD.%26aulast%3DSippl%26aufirst%3DW.%26aulast%3DJung%26aufirst%3DM.%26atitle%3DProteolysis%2520targeting%2520chimeras%2520%2528PROTACs%2529%2520for%2520epigenetics%2520research%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2020%26volume%3D57%26spage%3D8%26epage%3D16%26doi%3D10.1016%2Fj.cbpa.2020.01.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, P.</span></span> <span> </span><span class="NLM_article-title">Novel combination of histone methylation modulators with therapeutic synergy against acute myeloid leukemia in vitro and in vivo</span>. <i>Cancer Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>413</i></span>,  <span class="NLM_fpage">35</span>– <span class="NLM_lpage">45</span>, <span class="refDoi"> DOI: 10.1016/j.canlet.2017.10.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1016%2Fj.canlet.2017.10.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=29069576" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslagtbbE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=413&publication_year=2018&pages=35-45&author=S.+Wenauthor=J.+Wangauthor=P.+Liuauthor=Y.+Liauthor=W.+Luauthor=Y.+Huauthor=J.+Liuauthor=Z.+Heauthor=P.+Huang&title=Novel+combination+of+histone+methylation+modulators+with+therapeutic+synergy+against+acute+myeloid+leukemia+in+vitro+and+in+vivo&doi=10.1016%2Fj.canlet.2017.10.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Novel combination of histone methylation modulators with therapeutic synergy against acute myeloid leukemia in vitro and in vivo</span></div><div class="casAuthors">Wen, Shijun; Wang, Jiankang; Liu, Panpan; Li, Yiqing; Lu, Wenhua; Hu, Yumin; Liu, Jinyun; He, Zhiyuan; Huang, Peng</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Letters (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">413</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">35-45</span>CODEN:
                <span class="NLM_cas:coden">CALEDQ</span>;
        ISSN:<span class="NLM_cas:issn">0304-3835</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Acute myeloid leukemia (AML) is a hematol. malignancy with rapid disease progression and often becomes lethal without treatment.  Development of effective new therapies is essential to improve the clin. outcome of AML patients.  Enhancer of zeste homolog 2 (EZH2) and lysine specific demethylase 1 (LSD1) play important roles in epigenetic regulation and their altered expressions have been obsd. in cancer.  Although EZH2 and LSD1 have opposite histone methylation functions, we found that both enzymes were paradoxically up-regulated in AML cells.  Importantly, a combined inhibition of EZH2 and LSD1 resulted in a synergistic activity against AML in vitro and in vivo.  Such synergy was mechanistically correlated with up-regulation of H3K4me1/2 and H3K9Ac and down-regulation of H3K27me3, leading to a decrease of anti-apoptotic protein Bcl-2.  These epigenetic alterations also compromised the mitochondrial respiration capacity and glycolytic activity and resulted in ATP depletion, a key event contributing to the potent cytotoxic effect of the drug combination.  Taken together, our work identified a novel therapeutic approach against AML by combining two small mols. that inhibit different histone methylation-modulating proteins with apparently opposite enzyme activities.  Such a new drug combination strategy likely has significant clin. implications since epigenetic modulators are currently in clin. trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp78k_2d8kiwrVg90H21EOLACvtfcHk0lhQEmUefX9Y8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslagtbbE&md5=e79654ad4081eae952118c3e00afb778</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.canlet.2017.10.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.canlet.2017.10.015%26sid%3Dliteratum%253Aachs%26aulast%3DWen%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DP.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DLu%26aufirst%3DW.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DHe%26aufirst%3DZ.%26aulast%3DHuang%26aufirst%3DP.%26atitle%3DNovel%2520combination%2520of%2520histone%2520methylation%2520modulators%2520with%2520therapeutic%2520synergy%2520against%2520acute%2520myeloid%2520leukemia%2520in%2520vitro%2520and%2520in%2520vivo%26jtitle%3DCancer%2520Lett.%26date%3D2018%26volume%3D413%26spage%3D35%26epage%3D45%26doi%3D10.1016%2Fj.canlet.2017.10.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kuntz, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keilhack, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollock, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knutson, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter-Scott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richon, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sneeringer, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wigle, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allain, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majer, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moyer, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chesworth, R.</span></span> <span> </span><span class="NLM_article-title">The Importance of Being Me: Magic Methyls, Methyltransferase Inhibitors, and the Discovery of Tazemetostat</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1556</span>– <span class="NLM_lpage">1564</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01501</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01501" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC28XotF2qsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1556-1564&author=K.+W.+Kuntzauthor=J.+E.+Campbellauthor=H.+Keilhackauthor=R.+M.+Pollockauthor=S.+K.+Knutsonauthor=M.+Porter-Scottauthor=V.+M.+Richonauthor=C.+J.+Sneeringerauthor=T.+J.+Wigleauthor=C.+J.+Allainauthor=C.+R.+Majerauthor=M.+P.+Moyerauthor=R.+A.+Copelandauthor=R.+Chesworth&title=The+Importance+of+Being+Me%3A+Magic+Methyls%2C+Methyltransferase+Inhibitors%2C+and+the+Discovery+of+Tazemetostat&doi=10.1021%2Facs.jmedchem.5b01501"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">The Importance of Being Me: Magic Methyls, Methyltransferase Inhibitors, and the Discovery of Tazemetostat</span></div><div class="casAuthors">Kuntz, Kevin W.; Campbell, John E.; Keilhack, Heike; Pollock, Roy M.; Knutson, Sarah K.; Porter-Scott, Margaret; Richon, Victoria M.; Sneeringer, Chris J.; Wigle, Tim J.; Allain, Christina J.; Majer, Christina R.; Moyer, Mikel P.; Copeland, Robert A.; Chesworth, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1556-1564</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Posttranslational methylation of histones plays a crit. role in gene regulation.  Misregulation of histone methylation can lead to oncogenic transformation.  Enhancer of Zeste homolog 2 (EZH2) methylates histone 3 at lysine 27 (H3K27) and abnormal methylation of this site is found in many cancers.  Tazemetostat, an EHZ2 inhibitor in clin. development, has shown activity in both preclin. models of cancer as well as in patients with lymphoma or INI1-deficient solid tumors.  Herein we report the structure-activity relationships from identification of an initial hit in a high-throughput screen through selection of tazemetostat for clin. development.  The importance of several Me groups to the potency of the inhibitors is highlighted as well as the importance of balancing pharmacokinetic properties with potency.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGryASbIgMDxAbVg90H21EOLACvtfcHk0lhQEmUefX9Y8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XotF2qsA%253D%253D&md5=a26ccc4a0109f57fbb64f7fded8965a1</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01501&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01501%26sid%3Dliteratum%253Aachs%26aulast%3DKuntz%26aufirst%3DK.%2BW.%26aulast%3DCampbell%26aufirst%3DJ.%2BE.%26aulast%3DKeilhack%26aufirst%3DH.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26aulast%3DKnutson%26aufirst%3DS.%2BK.%26aulast%3DPorter-Scott%26aufirst%3DM.%26aulast%3DRichon%26aufirst%3DV.%2BM.%26aulast%3DSneeringer%26aufirst%3DC.%2BJ.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DAllain%26aufirst%3DC.%2BJ.%26aulast%3DMajer%26aufirst%3DC.%2BR.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DChesworth%26aufirst%3DR.%26atitle%3DThe%2520Importance%2520of%2520Being%2520Me%253A%2520Magic%2520Methyls%252C%2520Methyltransferase%2520Inhibitors%252C%2520and%2520the%2520Discovery%2520of%2520Tazemetostat%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1556%26epage%3D1564%26doi%3D10.1021%2Facs.jmedchem.5b01501" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Galdeano, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gadd, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soares, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scaffidi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Molle, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birced, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hewitt, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dias, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Structure-guided design and optimization of small molecules targeting the protein-protein interaction between the von Hippel-Lindau (VHL) E3 ubiquitin ligase and the hypoxia inducible factor (HIF) alpha subunit with in vitro nanomolar affinities</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">8657</span>– <span class="NLM_lpage">8663</span>, <span class="refDoi"> DOI: 10.1021/jm5011258</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm5011258" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVGhtb%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=8657-8663&author=C.+Galdeanoauthor=M.+S.+Gaddauthor=P.+Soaresauthor=S.+Scaffidiauthor=I.+Van+Molleauthor=I.+Bircedauthor=S.+Hewittauthor=D.+M.+Diasauthor=A.+Ciulli&title=Structure-guided+design+and+optimization+of+small+molecules+targeting+the+protein-protein+interaction+between+the+von+Hippel-Lindau+%28VHL%29+E3+ubiquitin+ligase+and+the+hypoxia+inducible+factor+%28HIF%29+alpha+subunit+with+in+vitro+nanomolar+affinities&doi=10.1021%2Fjm5011258"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Guided Design and Optimization of Small Molecules Targeting the Protein-Protein Interaction between the von Hippel-Lindau (VHL) E3 Ubiquitin Ligase and the Hypoxia Inducible Factor (HIF) Alpha Subunit with in Vitro Nanomolar Affinities</span></div><div class="casAuthors">Galdeano, Carles; Gadd, Morgan S.; Soares, Pedro; Scaffidi, Salvatore; Van Molle, Inge; Birced, Ipek; Hewitt, Sarah; Dias, David M.; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">8657-8663</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">E3 ubiquitin ligases are attractive targets in the ubiquitin-proteasome system, however, the development of small-mol. ligands has been rewarded with limited success.  The von Hippel-Lindau protein (pVHL) is the substrate recognition subunit of the VHL E3 ligase that targets HIF-1α for degrdn.  The authors recently reported inhibitors of the pVHL:HIF-1α interaction, however they exhibited moderate potency.  Herein, the authors report the design and optimization, guided by x-ray crystal structures, of a ligand series with nanomolar binding affinities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpixK3qp1ctJrVg90H21EOLACvtfcHk0lhQEmUefX9Y8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVGhtb%252FP&md5=831aa2387d94cb03d735a4be7bdd2cdd</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fjm5011258&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm5011258%26sid%3Dliteratum%253Aachs%26aulast%3DGaldeano%26aufirst%3DC.%26aulast%3DGadd%26aufirst%3DM.%2BS.%26aulast%3DSoares%26aufirst%3DP.%26aulast%3DScaffidi%26aufirst%3DS.%26aulast%3DVan%2BMolle%26aufirst%3DI.%26aulast%3DBirced%26aufirst%3DI.%26aulast%3DHewitt%26aufirst%3DS.%26aulast%3DDias%26aufirst%3DD.%2BM.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DStructure-guided%2520design%2520and%2520optimization%2520of%2520small%2520molecules%2520targeting%2520the%2520protein-protein%2520interaction%2520between%2520the%2520von%2520Hippel-Lindau%2520%2528VHL%2529%2520E3%2520ubiquitin%2520ligase%2520and%2520the%2520hypoxia%2520inducible%2520factor%2520%2528HIF%2529%2520alpha%2520subunit%2520with%2520in%2520vitro%2520nanomolar%2520affinities%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D8657%26epage%3D8663%26doi%3D10.1021%2Fjm5011258" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Frost, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galdeano, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soares, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gadd, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grzes, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Epemolu, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimamura, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bantscheff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantrell, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rocha, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Potent and selective chemical probe of hypoxic signalling downstream of HIF-alpha hydroxylation via VHL inhibition</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>, <span class="NLM_elocation-id">13312</span> <span class="refDoi"> DOI: 10.1038/ncomms13312</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1038%2Fncomms13312" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=27811928" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVSitLfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&author=J.+Frostauthor=C.+Galdeanoauthor=P.+Soaresauthor=M.+S.+Gaddauthor=K.+M.+Grzesauthor=L.+Ellisauthor=O.+Epemoluauthor=S.+Shimamuraauthor=M.+Bantscheffauthor=P.+Grandiauthor=K.+D.+Readauthor=D.+A.+Cantrellauthor=S.+Rochaauthor=A.+Ciulli&title=Potent+and+selective+chemical+probe+of+hypoxic+signalling+downstream+of+HIF-alpha+hydroxylation+via+VHL+inhibition&doi=10.1038%2Fncomms13312"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Potent and selective chemical probe of hypoxic signalling downstream of HIF-α hydroxylation via VHL inhibition</span></div><div class="casAuthors">Frost, Julianty; Galdeano, Carles; Soares, Pedro; Gadd, Morgan S.; Grzes, Katarzyna M.; Ellis, Lucy; Epemolu, Ola; Shimamura, Satoko; Bantscheff, Marcus; Grandi, Paola; Read, Kevin D.; Cantrell, Doreen A.; Rocha, Sonia; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">13312</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Chem. strategies to using small mols. to stimulate hypoxia inducible factors (HIFs) activity and trigger a hypoxic response under normoxic conditions, such as iron chelators and inhibitors of prolyl hydroxylase domain (PHD) enzymes, have broad-spectrum activities and off-target effects.  Here we disclose VH298, a potent VHL inhibitor that stabilizes HIF-α and elicits a hypoxic response via a different mechanism, that is the blockade of the VHL:HIF-α protein-protein interaction downstream of HIF-α hydroxylation by PHD enzymes.  We show that VH298 engages with high affinity and specificity with VHL as its only major cellular target, leading to selective on-target accumulation of hydroxylated HIF-α in a concn.- and time-dependent fashion in different cell lines, with subsequent upregulation of HIF-target genes at both mRNA and protein levels.  VH298 represents a high-quality chem. probe of the HIF signalling cascade and an attractive starting point to the development of potential new therapeutics targeting hypoxia signalling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLB1WvVF4EFLVg90H21EOLACvtfcHk0liBtEkWPRfYbA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVSitLfJ&md5=7a884b6193b3ca5e119fecea2eb31d3e</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1038%2Fncomms13312&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms13312%26sid%3Dliteratum%253Aachs%26aulast%3DFrost%26aufirst%3DJ.%26aulast%3DGaldeano%26aufirst%3DC.%26aulast%3DSoares%26aufirst%3DP.%26aulast%3DGadd%26aufirst%3DM.%2BS.%26aulast%3DGrzes%26aufirst%3DK.%2BM.%26aulast%3DEllis%26aufirst%3DL.%26aulast%3DEpemolu%26aufirst%3DO.%26aulast%3DShimamura%26aufirst%3DS.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DRead%26aufirst%3DK.%2BD.%26aulast%3DCantrell%26aufirst%3DD.%2BA.%26aulast%3DRocha%26aufirst%3DS.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DPotent%2520and%2520selective%2520chemical%2520probe%2520of%2520hypoxic%2520signalling%2520downstream%2520of%2520HIF-alpha%2520hydroxylation%2520via%2520VHL%2520inhibition%26jtitle%3DNat.%2520Commun.%26date%3D2016%26volume%3D7%26doi%3D10.1038%2Fncomms13312" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zorba, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Starr, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borzilleri, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farley, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiemer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uccello, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia-Irrizary, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czabaniuk, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuff, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliver, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montgomery, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayward, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coe, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niosi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luthra, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El-Kattan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noe, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanmugasundaram, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilbert, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calabrese, M. F.</span></span> <span> </span><span class="NLM_article-title">Delineating the role of cooperativity in the design of potent PROTACs for BTK</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>115</i></span>,  <span class="NLM_fpage">E7285</span>– <span class="NLM_lpage">E7292</span>, <span class="refDoi"> DOI: 10.1073/pnas.1803662115</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1073%2Fpnas.1803662115" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=30012605" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=115&publication_year=2018&pages=E7285-E7292&author=A.+Zorbaauthor=C.+Nguyenauthor=Y.+Xuauthor=J.+Starrauthor=K.+Borzilleriauthor=J.+Smithauthor=H.+Zhuauthor=K.+A.+Farleyauthor=W.+Dingauthor=J.+Schiemerauthor=X.+Fengauthor=J.+S.+Changauthor=D.+P.+Uccelloauthor=J.+A.+Youngauthor=C.+N.+Garcia-Irrizaryauthor=L.+Czabaniukauthor=B.+Schuffauthor=R.+Oliverauthor=J.+Montgomeryauthor=M.+M.+Haywardauthor=J.+Coeauthor=J.+Chenauthor=M.+Niosiauthor=S.+Luthraauthor=J.+C.+Shahauthor=A.+El-Kattanauthor=X.+Qiuauthor=G.+M.+Westauthor=M.+C.+Noeauthor=V.+Shanmugasundaramauthor=A.+M.+Gilbertauthor=M.+F.+Brownauthor=M.+F.+Calabrese&title=Delineating+the+role+of+cooperativity+in+the+design+of+potent+PROTACs+for+BTK&doi=10.1073%2Fpnas.1803662115"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1803662115&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1803662115%26sid%3Dliteratum%253Aachs%26aulast%3DZorba%26aufirst%3DA.%26aulast%3DNguyen%26aufirst%3DC.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DStarr%26aufirst%3DJ.%26aulast%3DBorzilleri%26aufirst%3DK.%26aulast%3DSmith%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DH.%26aulast%3DFarley%26aufirst%3DK.%2BA.%26aulast%3DDing%26aufirst%3DW.%26aulast%3DSchiemer%26aufirst%3DJ.%26aulast%3DFeng%26aufirst%3DX.%26aulast%3DChang%26aufirst%3DJ.%2BS.%26aulast%3DUccello%26aufirst%3DD.%2BP.%26aulast%3DYoung%26aufirst%3DJ.%2BA.%26aulast%3DGarcia-Irrizary%26aufirst%3DC.%2BN.%26aulast%3DCzabaniuk%26aufirst%3DL.%26aulast%3DSchuff%26aufirst%3DB.%26aulast%3DOliver%26aufirst%3DR.%26aulast%3DMontgomery%26aufirst%3DJ.%26aulast%3DHayward%26aufirst%3DM.%2BM.%26aulast%3DCoe%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DNiosi%26aufirst%3DM.%26aulast%3DLuthra%26aufirst%3DS.%26aulast%3DShah%26aufirst%3DJ.%2BC.%26aulast%3DEl-Kattan%26aufirst%3DA.%26aulast%3DQiu%26aufirst%3DX.%26aulast%3DWest%26aufirst%3DG.%2BM.%26aulast%3DNoe%26aufirst%3DM.%2BC.%26aulast%3DShanmugasundaram%26aufirst%3DV.%26aulast%3DGilbert%26aufirst%3DA.%2BM.%26aulast%3DBrown%26aufirst%3DM.%2BF.%26aulast%3DCalabrese%26aufirst%3DM.%2BF.%26atitle%3DDelineating%2520the%2520role%2520of%2520cooperativity%2520in%2520the%2520design%2520of%2520potent%2520PROTACs%2520for%2520BTK%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2018%26volume%3D115%26spage%3DE7285%26epage%3DE7292%26doi%3D10.1073%2Fpnas.1803662115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bondeson, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burslem, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buhimschi, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaime-Figueroa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamman, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishchenko, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Lessons in PROTAC Design from Selective Degradation with a Promiscuous Warhead</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">78</span>– <span class="NLM_lpage">87.e5</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.09.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1016%2Fj.chembiol.2017.09.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=29129718" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=78-87.e5&author=D.+P.+Bondesonauthor=B.+E.+Smithauthor=G.+M.+Burslemauthor=A.+D.+Buhimschiauthor=J.+Hinesauthor=S.+Jaime-Figueroaauthor=J.+Wangauthor=B.+D.+Hammanauthor=A.+Ishchenkoauthor=C.+M.+Crews&title=Lessons+in+PROTAC+Design+from+Selective+Degradation+with+a+Promiscuous+Warhead&doi=10.1016%2Fj.chembiol.2017.09.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Lessons in PROTAC Design from Selective Degradation with a Promiscuous Warhead</span></div><div class="casAuthors">Bondeson, Daniel P.; Smith, Blake E.; Burslem, George M.; Buhimschi, Alexandru D.; Hines, John; Jaime-Figueroa, Saul; Wang, Jing; Hamman, Brian D.; Ishchenko, Alexey; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">78-87.e5</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Inhibiting protein function selectively is a major goal of modern drug discovery.  Here, we report a previously understudied benefit of small mol. proteolysis-targeting chimeras (PROTACs) that recruit E3 ubiquitin ligases to target proteins for their ubiquitination and subsequent proteasome-mediated degrdn.  Using promiscuous CRBN- and VHL-recruiting PROTACs that bind >50 kinases, we show that only a subset of bound targets is degraded.  The basis of this selectivity relies on protein-protein interactions between the E3 ubiquitin ligase and the target protein, as illustrated by engaged proteins that are not degraded as a result of unstable ternary complexes with PROTAC-recruited E3 ligases.  In contrast, weak PROTAC:target protein affinity can be stabilized by high-affinity target:PROTAC:ligase trimer interactions, leading to efficient degrdn.  This study highlights design guidelines for generating potent PROTACs as well as possibilities for degrading undruggable proteins immune to traditional small-mol. inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoISOBudOLyFrVg90H21EOLACvtfcHk0liBtEkWPRfYbA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjN&md5=30fac502705a2344c4261cf531e6bf9d</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.09.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.09.010%26sid%3Dliteratum%253Aachs%26aulast%3DBondeson%26aufirst%3DD.%2BP.%26aulast%3DSmith%26aufirst%3DB.%2BE.%26aulast%3DBurslem%26aufirst%3DG.%2BM.%26aulast%3DBuhimschi%26aufirst%3DA.%2BD.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DJaime-Figueroa%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DHamman%26aufirst%3DB.%2BD.%26aulast%3DIshchenko%26aufirst%3DA.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DLessons%2520in%2520PROTAC%2520Design%2520from%2520Selective%2520Degradation%2520with%2520a%2520Promiscuous%2520Warhead%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2018%26volume%3D25%26spage%3D78%26epage%3D87.e5%26doi%3D10.1016%2Fj.chembiol.2017.09.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calabrese, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bullock, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Targeted protein degradation: expanding the toolbox</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">949</span>– <span class="NLM_lpage">963</span>, <span class="refDoi"> DOI: 10.1038/s41573-019-0047-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1038%2Fs41573-019-0047-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=31666732" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitVKrtLrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=949-963&author=M.+Schapiraauthor=M.+F.+Calabreseauthor=A.+N.+Bullockauthor=C.+M.+Crews&title=Targeted+protein+degradation%3A+expanding+the+toolbox&doi=10.1038%2Fs41573-019-0047-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted protein degradation: expanding the toolbox</span></div><div class="casAuthors">Schapira, Matthieu; Calabrese, Matthew F.; Bullock, Alex N.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">949-963</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review. Proteolysis-targeting chimeras (PROTACs) and related mols. that induce targeted protein degrdn. by the ubiquitin-proteasome system represent a new therapeutic modality and are the focus of great interest, owing to potential advantages over traditional occupancy-based inhibitors with respect to dosing, side effects, drug resistance and modulating 'undruggable' targets.  However, the technol. is still maturing, and the design elements for successful PROTAC-based drugs are currently being elucidated.  Importantly, fewer than 10 of the more than 600 E3 ubiquitin ligases have so far been exploited for targeted protein degrdn., and expansion of knowledge in this area is a key opportunity.  Here, we briefly discuss lessons learned about targeted protein degrdn. in chem. biol. and drug discovery and systematically review the expression profile, domain architecture and chem. tractability of human E3 ligases that could expand the toolbox for PROTAC discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwUvM27MU3WrVg90H21EOLACvtfcHk0lhQlFFoaIYbEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitVKrtLrM&md5=9bfd48c6c47fc1cce15129495fea7079</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1038%2Fs41573-019-0047-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41573-019-0047-y%26sid%3Dliteratum%253Aachs%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DCalabrese%26aufirst%3DM.%2BF.%26aulast%3DBullock%26aufirst%3DA.%2BN.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DTargeted%2520protein%2520degradation%253A%2520expanding%2520the%2520toolbox%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2019%26volume%3D18%26spage%3D949%26epage%3D963%26doi%3D10.1038%2Fs41573-019-0047-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zoppi, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maniaci, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Testa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gmaschitz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wieshofer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koegl, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riching, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniels, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spallarossa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Iterative Design and Optimization of Initially Inactive Proteolysis Targeting Chimeras (PROTACs) Identify VZ185 as a Potent, Fast, and Selective von Hippel-Lindau (VHL) Based Dual Degrader Probe of BRD9 and BRD7</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">699</span>– <span class="NLM_lpage">726</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01413</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01413" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisVyhsLjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=699-726&author=V.+Zoppiauthor=S.+J.+Hughesauthor=C.+Maniaciauthor=A.+Testaauthor=T.+Gmaschitzauthor=C.+Wieshoferauthor=M.+Koeglauthor=K.+M.+Richingauthor=D.+L.+Danielsauthor=A.+Spallarossaauthor=A.+Ciulli&title=Iterative+Design+and+Optimization+of+Initially+Inactive+Proteolysis+Targeting+Chimeras+%28PROTACs%29+Identify+VZ185+as+a+Potent%2C+Fast%2C+and+Selective+von+Hippel-Lindau+%28VHL%29+Based+Dual+Degrader+Probe+of+BRD9+and+BRD7&doi=10.1021%2Facs.jmedchem.8b01413"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Iterative Design and Optimization of Initially Inactive Proteolysis Targeting Chimeras (PROTACs) Identify VZ185 as a Potent, Fast, and Selective von Hippel-Lindau (VHL) Based Dual Degrader Probe of BRD9 and BRD7</span></div><div class="casAuthors">Zoppi, Vittoria; Hughes, Scott J.; Maniaci, Chiara; Testa, Andrea; Gmaschitz, Teresa; Wieshofer, Corinna; Koegl, Manfred; Riching, Kristin M.; Daniels, Danette L.; Spallarossa, Andrea; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">699-726</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Developing PROTACs to redirect the ubiquitination activity of E3 ligases and potently degrade a target protein within cells can be a lengthy and unpredictable process, and it remains unclear whether any combination of E3 and target might be productive for degrdn.  The authors describe a probe-quality degrader for a ligase-target pair deemed unsuitable: the von Hippel-Lindau (VHL) and BRD9, a bromodomain-contg. subunit of the SWI/SNF chromatin remodeling complex BAF.  VHL-based degraders could be optimized from suboptimal compds. in two rounds by systematically varying conjugation patterns and linkers and monitoring cellular degrdn. activities, kinetic profiles, and ubiquitination, as well as ternary complex formation thermodn.  The emerged structure-activity relations guided the discovery of VZ185, a potent, fast, and selective degrader of BRD9 and of its close homolog BRD7.  The authors' findings qualify a new chem. tool for BRD7/9 knockdown and provide a roadmap for PROTAC development against seemingly incompatible target-ligase combinations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqiPnajotKvqLVg90H21EOLACvtfcHk0lhQlFFoaIYbEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisVyhsLjN&md5=be7938c26807a2535b7f9b7de8ebfbb0</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01413&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01413%26sid%3Dliteratum%253Aachs%26aulast%3DZoppi%26aufirst%3DV.%26aulast%3DHughes%26aufirst%3DS.%2BJ.%26aulast%3DManiaci%26aufirst%3DC.%26aulast%3DTesta%26aufirst%3DA.%26aulast%3DGmaschitz%26aufirst%3DT.%26aulast%3DWieshofer%26aufirst%3DC.%26aulast%3DKoegl%26aufirst%3DM.%26aulast%3DRiching%26aufirst%3DK.%2BM.%26aulast%3DDaniels%26aufirst%3DD.%2BL.%26aulast%3DSpallarossa%26aufirst%3DA.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DIterative%2520Design%2520and%2520Optimization%2520of%2520Initially%2520Inactive%2520Proteolysis%2520Targeting%2520Chimeras%2520%2528PROTACs%2529%2520Identify%2520VZ185%2520as%2520a%2520Potent%252C%2520Fast%252C%2520and%2520Selective%2520von%2520Hippel-Lindau%2520%2528VHL%2529%2520Based%2520Dual%2520Degrader%2520Probe%2520of%2520BRD9%2520and%2520BRD7%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D699%26epage%3D726%26doi%3D10.1021%2Facs.jmedchem.8b01413" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Potjewyd, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turner, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beri, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rectenwald, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norris-Drouin, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cholensky, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Margolis, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearce, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herring, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, L. I.</span></span> <span> </span><span class="NLM_article-title">Degradation of Polycomb Repressive Complex 2 with an EED-Targeted Bivalent Chemical Degrader</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">47</span>– <span class="NLM_lpage">56.e15</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2019.11.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1016%2Fj.chembiol.2019.11.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=31831267" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitlOmt7bM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2020&pages=47-56.e15&author=F.+Potjewydauthor=A.+W.+Turnerauthor=J.+Beriauthor=J.+M.+Rectenwaldauthor=J.+L.+Norris-Drouinauthor=S.+H.+Cholenskyauthor=D.+M.+Margolisauthor=K.+H.+Pearceauthor=L.+E.+Herringauthor=L.+I.+James&title=Degradation+of+Polycomb+Repressive+Complex+2+with+an+EED-Targeted+Bivalent+Chemical+Degrader&doi=10.1016%2Fj.chembiol.2019.11.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Degradation of Polycomb Repressive Complex 2 with an EED-Targeted Bivalent Chemical Degrader</span></div><div class="casAuthors">Potjewyd, Frances; Turner, Anne-Marie W.; Beri, Joshua; Rectenwald, Justin M.; Norris-Drouin, Jacqueline L.; Cholensky, Stephanie H.; Margolis, David M.; Pearce, Kenneth H.; Herring, Laura E.; James, Lindsey I.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">47-56.e15</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Protein degrdn. via the use of bivalent chem. degraders provides an alternative strategy to block protein function and assess the biol. roles of putative drug targets.  This approach capitalizes on the advantages of small-mol. inhibitors while moving beyond the restrictions of traditional pharmacol.  Here, we report a chem. degrader (UNC6852) that targets polycomb repressive complex 2 (PRC2).  UNC6852 contains an EED226-derived ligand and a ligand for VHL which bind to the WD40 arom. cage of EED and CRL2VHL, resp., to induce proteasomal degrdn. of PRC2 components, EED, EZH2, and SUZ12.  Degrdn. of PRC2 with UNC6852 blocks the histone methyltransferase activity of EZH2, decreasing H3K27me3 levels in HeLa cells and diffuse large B cell lymphoma (DLBCL) cells contg. EZH2 gain-of-function mutations.  UNC6852 degrades both wild-type and mutant EZH2, and addnl. displays anti-proliferative effects in this cancer model system.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqF3p-HFdD9KLVg90H21EOLACvtfcHk0lhQlFFoaIYbEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitlOmt7bM&md5=7d9ca2726bad4e40003310951edc09d4</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2019.11.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2019.11.006%26sid%3Dliteratum%253Aachs%26aulast%3DPotjewyd%26aufirst%3DF.%26aulast%3DTurner%26aufirst%3DA.%2BW.%26aulast%3DBeri%26aufirst%3DJ.%26aulast%3DRectenwald%26aufirst%3DJ.%2BM.%26aulast%3DNorris-Drouin%26aufirst%3DJ.%2BL.%26aulast%3DCholensky%26aufirst%3DS.%2BH.%26aulast%3DMargolis%26aufirst%3DD.%2BM.%26aulast%3DPearce%26aufirst%3DK.%2BH.%26aulast%3DHerring%26aufirst%3DL.%2BE.%26aulast%3DJames%26aufirst%3DL.%2BI.%26atitle%3DDegradation%2520of%2520Polycomb%2520Repressive%2520Complex%25202%2520with%2520an%2520EED-Targeted%2520Bivalent%2520Chemical%2520Degrader%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2020%26volume%3D27%26spage%3D47%26epage%3D56.e15%26doi%3D10.1016%2Fj.chembiol.2019.11.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hsu, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rasmusson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pachl, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawatkar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DH, O. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bagal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Code, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rawlins, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Argyrou, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomlinson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiarparin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacques, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woods, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bednarski, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drew, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castaldi, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fawell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bloecher, A.</span></span> <span> </span><span class="NLM_article-title">EED-Targeted PROTACs Degrade EED, EZH2, and SUZ12 in the PRC2 Complex</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">41</span>– <span class="NLM_lpage">46.e17</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2019.11.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1016%2Fj.chembiol.2019.11.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=31786184" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC1MXit1KqsrnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2020&pages=41-46.e17&author=J.+H.+Hsuauthor=T.+Rasmussonauthor=J.+Robinsonauthor=F.+Pachlauthor=J.+Readauthor=S.+Kawatkarauthor=O.+D.+DHauthor=S.+Bagalauthor=E.+Codeauthor=P.+Rawlinsauthor=A.+Argyrouauthor=R.+Tomlinsonauthor=N.+Gaoauthor=X.+Zhuauthor=E.+Chiarparinauthor=K.+Jacquesauthor=M.+Shenauthor=H.+Woodsauthor=E.+Bednarskiauthor=D.+M.+Wilsonauthor=L.+Drewauthor=M.+P.+Castaldiauthor=S.+Fawellauthor=A.+Bloecher&title=EED-Targeted+PROTACs+Degrade+EED%2C+EZH2%2C+and+SUZ12+in+the+PRC2+Complex&doi=10.1016%2Fj.chembiol.2019.11.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">EED-Targeted PROTACs Degrade EED, EZH2, and SUZ12 in the PRC2 Complex</span></div><div class="casAuthors">Hsu, Jessie Hao-Ru; Rasmusson, Timothy; Robinson, James; Pachl, Fiona; Read, Jon; Kawatkar, Sameer; O' Donovan, Daniel H.; Bagal, Sharan; Code, Erin; Rawlins, Philip; Argyrou, Argyrides; Tomlinson, Ronald; Gao, Ning; Zhu, Xiahui; Chiarparin, Elisabetta; Jacques, Kelly; Shen, Minhui; Woods, Haley; Bednarski, Emma; Wilson, David M.; Drew, Lisa; Castaldi, M. Paola; Fawell, Stephen; Bloecher, Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">41-46.e17</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Deregulation of the PRC2 complex, comprised of the core subunits EZH2, SUZ12, and EED, drives aberrant hypermethylation of H3K27 and tumorigenicity of many cancers.  Although inhibitors of EZH2 have shown promising clin. activity, preclin. data suggest that resistance can be acquired through secondary mutations in EZH2 that abrogate drug target engagement.  To address these limitations, we have designed several hetero-bifunctional PROTACs (proteolysis-targeting chimera) to efficiently target EED for elimination.  Our PROTACs bind to EED (pKD ~ 9.0) and promote ternary complex formation with the E3 ubiquitin ligase.  The PROTACs potently inhibit PRC2 enzyme activity (pIC50 ~ 8.1) and induce rapid degrdn. of not only EED but also EZH2 and SUZ12 within the PRC2 complex.  Furthermore, the PROTACs selectively inhibit proliferation of PRC2-dependent cancer cells (half maximal growth inhibition [GI50] = 49-58 nM).  In summary, our data demonstrate a therapeutic modality to target PRC2-dependent cancer through a PROTAC-mediated degrdn. mechanism.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4WGn283nfDLVg90H21EOLACvtfcHk0ljCM2IqLVFX5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXit1KqsrnO&md5=791238d7d1423f4a715529cfa14bfb65</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2019.11.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2019.11.004%26sid%3Dliteratum%253Aachs%26aulast%3DHsu%26aufirst%3DJ.%2BH.%26aulast%3DRasmusson%26aufirst%3DT.%26aulast%3DRobinson%26aufirst%3DJ.%26aulast%3DPachl%26aufirst%3DF.%26aulast%3DRead%26aufirst%3DJ.%26aulast%3DKawatkar%26aufirst%3DS.%26aulast%3DDH%26aufirst%3DO.%2BD.%26aulast%3DBagal%26aufirst%3DS.%26aulast%3DCode%26aufirst%3DE.%26aulast%3DRawlins%26aufirst%3DP.%26aulast%3DArgyrou%26aufirst%3DA.%26aulast%3DTomlinson%26aufirst%3DR.%26aulast%3DGao%26aufirst%3DN.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DChiarparin%26aufirst%3DE.%26aulast%3DJacques%26aufirst%3DK.%26aulast%3DShen%26aufirst%3DM.%26aulast%3DWoods%26aufirst%3DH.%26aulast%3DBednarski%26aufirst%3DE.%26aulast%3DWilson%26aufirst%3DD.%2BM.%26aulast%3DDrew%26aufirst%3DL.%26aulast%3DCastaldi%26aufirst%3DM.%2BP.%26aulast%3DFawell%26aufirst%3DS.%26aulast%3DBloecher%26aufirst%3DA.%26atitle%3DEED-Targeted%2520PROTACs%2520Degrade%2520EED%252C%2520EZH2%252C%2520and%2520SUZ12%2520in%2520the%2520PRC2%2520Complex%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2020%26volume%3D27%26spage%3D41%26epage%3D46.e17%26doi%3D10.1016%2Fj.chembiol.2019.11.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maniaci, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Testa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamont, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rocha, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alessi, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romeo, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Homo-PROTACs: bivalent small-molecule dimerizers of the VHL E3 ubiquitin ligase to induce self-degradation</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">830</span> <span class="refDoi"> DOI: 10.1038/s41467-017-00954-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1038%2Fs41467-017-00954-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=29018234" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A280%3ADC%252BC1M%252Fptlaiuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&author=C.+Maniaciauthor=S.+J.+Hughesauthor=A.+Testaauthor=W.+Chenauthor=D.+J.+Lamontauthor=S.+Rochaauthor=D.+R.+Alessiauthor=R.+Romeoauthor=A.+Ciulli&title=Homo-PROTACs%3A+bivalent+small-molecule+dimerizers+of+the+VHL+E3+ubiquitin+ligase+to+induce+self-degradation&doi=10.1038%2Fs41467-017-00954-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Homo-PROTACs: bivalent small-molecule dimerizers of the VHL E3 ubiquitin ligase to induce self-degradation</span></div><div class="casAuthors">Maniaci Chiara; Hughes Scott J; Testa Andrea; Chen Wenzhang; Lamont Douglas J; Ciulli Alessio; Maniaci Chiara; Alessi Dario R; Rocha Sonia; Romeo Roberto</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">830</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">E3 ubiquitin ligases are key enzymes within the ubiquitin proteasome system which catalyze the ubiquitination of proteins, targeting them for proteasomal degradation.  E3 ligases are gaining importance as targets to small molecules, both for direct inhibition and to be hijacked to induce the degradation of non-native neo-substrates using bivalent compounds known as PROTACs (for 'proteolysis-targeting chimeras').  We describe Homo-PROTACs as an approach to dimerize an E3 ligase to trigger its suicide-type chemical knockdown inside cells.  We provide proof-of-concept of Homo-PROTACs using diverse molecules composed of two instances of a ligand for the von Hippel-Lindau (VHL) E3 ligase.  The most active compound, CM11, dimerizes VHL with high avidity in vitro and induces potent, rapid and proteasome-dependent self-degradation of VHL in different cell lines, in a highly isoform-selective fashion and without triggering a hypoxic response.  This approach offers a novel chemical probe for selective VHL knockdown, and demonstrates the potential for a new modality of chemical intervention on E3 ligases.Targeting the ubiquitin proteasome system to modulate protein homeostasis using small molecules has promising therapeutic potential.  Here the authors describe Homo-PROTACS: small molecules that can induce the homo-dimerization of E3 ubiquitin ligases and cause their proteasome-dependent degradation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRVJKELZUTFBpPq5LRczm8ffW6udTcc2eY8OoSX2C4b_7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M%252Fptlaiuw%253D%253D&md5=0ed79d9ff14eb7b6132a152f0413c4ef</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1038%2Fs41467-017-00954-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-017-00954-1%26sid%3Dliteratum%253Aachs%26aulast%3DManiaci%26aufirst%3DC.%26aulast%3DHughes%26aufirst%3DS.%2BJ.%26aulast%3DTesta%26aufirst%3DA.%26aulast%3DChen%26aufirst%3DW.%26aulast%3DLamont%26aufirst%3DD.%2BJ.%26aulast%3DRocha%26aufirst%3DS.%26aulast%3DAlessi%26aufirst%3DD.%2BR.%26aulast%3DRomeo%26aufirst%3DR.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DHomo-PROTACs%253A%2520bivalent%2520small-molecule%2520dimerizers%2520of%2520the%2520VHL%2520E3%2520ubiquitin%2520ligase%2520to%2520induce%2520self-degradation%26jtitle%3DNat.%2520Commun.%26date%3D2017%26volume%3D8%26doi%3D10.1038%2Fs41467-017-00954-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Knutson, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warholic, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wigle, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klaus, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allain, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raimondi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter Scott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chesworth, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moyer, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richon, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollock, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuntz, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keilhack, H.</span></span> <span> </span><span class="NLM_article-title">Durable tumor regression in genetically altered malignant rhabdoid tumors by inhibition of methyltransferase EZH2</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">7922</span>– <span class="NLM_lpage">7927</span>, <span class="refDoi"> DOI: 10.1073/pnas.1303800110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1073%2Fpnas.1303800110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=23620515" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC3sXptFGrurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=7922-7927&author=S.+K.+Knutsonauthor=N.+M.+Warholicauthor=T.+J.+Wigleauthor=C.+R.+Klausauthor=C.+J.+Allainauthor=A.+Raimondiauthor=M.+Porter+Scottauthor=R.+Chesworthauthor=M.+P.+Moyerauthor=R.+A.+Copelandauthor=V.+M.+Richonauthor=R.+M.+Pollockauthor=K.+W.+Kuntzauthor=H.+Keilhack&title=Durable+tumor+regression+in+genetically+altered+malignant+rhabdoid+tumors+by+inhibition+of+methyltransferase+EZH2&doi=10.1073%2Fpnas.1303800110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Durable tumor regression in genetically altered malignant rhabdoid tumors by inhibition of methyltransferase EZH2</span></div><div class="casAuthors">Knutson, Sarah K.; Warholic, Natalie M.; Wigle, Tim J.; Klaus, Christine R.; Allain, Christina J.; Raimondi, Alejandra; Scott, Margaret Porter; Chesworth, Richard; Moyer, Mikel P.; Copeland, Robert A.; Richon, Victoria M.; Pollock, Roy M.; Kuntz, Kevin W.; Keilhack, Heike</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">7922-7927, S7922/1-S7922/5</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Inactivation of the switch/sucrose nonfermentable complex component SMARCB1 is extremely prevalent in pediatric malignant rhabdoid tumors (MRTs) or atypical teratoid rhabdoid tumors.  This alteration is hypothesized to confer oncogenic dependency on EZH2 in these cancers.  The authors report the discovery of a potent, selective, and orally bioavailable small-mol. inhibitor of EZH2 enzymic activity, (N-((4,6-dimethyl-2-oxo-1,2-dihydropyridin-3-yl)methyl)-5-(ethyl(tetrahydro-2H-pyran-4-yl)amino)-4-methyl-4'-(morpholinomethyl)-[1,1'-biphenyl]-3-carboxamide).  The compd. induces apoptosis and differentiation specifically in SMARCB1-deleted MRT cells.  Treatment of xenograft-bearing mice with (N-((4,6-dimethyl-2-oxo-1,2-dihydropyridin-3-yl)methyl)-5-(ethyl(tetrahydro-2H-pyran-4-yl)amino)-4-methyl-4'-(morpholinomethyl)-[1,1'-biphenyl]-3-carboxamide) leads to dose-dependent regression of MRTs with correlative diminution of intratumoral trimethylation levels of lysine 27 on histone H3, and prevention of tumor regrowth after dosing cessation.  These data demonstrate the dependency of SMARCB1 mutant MRTs on EZH2 enzymic activity and portend the utility of EZH2-targeted drugs for the treatment of these genetically defined cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWRIJfPhwLA7Vg90H21EOLACvtfcHk0ljCM2IqLVFX5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXptFGrurw%253D&md5=c2c6470d48a13ab8343db0a09465ba42</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1303800110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1303800110%26sid%3Dliteratum%253Aachs%26aulast%3DKnutson%26aufirst%3DS.%2BK.%26aulast%3DWarholic%26aufirst%3DN.%2BM.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DKlaus%26aufirst%3DC.%2BR.%26aulast%3DAllain%26aufirst%3DC.%2BJ.%26aulast%3DRaimondi%26aufirst%3DA.%26aulast%3DPorter%2BScott%26aufirst%3DM.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DRichon%26aufirst%3DV.%2BM.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26aulast%3DKuntz%26aufirst%3DK.%2BW.%26aulast%3DKeilhack%26aufirst%3DH.%26atitle%3DDurable%2520tumor%2520regression%2520in%2520genetically%2520altered%2520malignant%2520rhabdoid%2520tumors%2520by%2520inhibition%2520of%2520methyltransferase%2520EZH2%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2013%26volume%3D110%26spage%3D7922%26epage%3D7927%26doi%3D10.1073%2Fpnas.1303800110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Remillard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brien, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sonnett, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seo, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dastjerdi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wuhr, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhe-Paganon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armstrong, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Degradation of the BAF Complex Factor BRD9 by Heterobifunctional Ligands</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">5738</span>– <span class="NLM_lpage">5743</span>, <span class="refDoi"> DOI: 10.1002/anie.201611281</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1002%2Fanie.201611281" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmtFGrurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2017&pages=5738-5743&author=D.+Remillardauthor=D.+L.+Buckleyauthor=J.+Paulkauthor=G.+L.+Brienauthor=M.+Sonnettauthor=H.+S.+Seoauthor=S.+Dastjerdiauthor=M.+Wuhrauthor=S.+Dhe-Paganonauthor=S.+A.+Armstrongauthor=J.+E.+Bradner&title=Degradation+of+the+BAF+Complex+Factor+BRD9+by+Heterobifunctional+Ligands&doi=10.1002%2Fanie.201611281"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Degradation of the BAF Complex Factor BRD9 by Heterobifunctional Ligands</span></div><div class="casAuthors">Remillard, David; Buckley, Dennis L.; Paulk, Joshiawa; Brien, Gerard L.; Sonnett, Matthew; Seo, Hyuk-Soo; Dastierdi, Shiva; Wuehr, Martin; Dhe-Paganon, Sirano; Armstrong, Scott A.; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">5738-5743</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The bromodomain-contg. protein BRD9, a subunit of the human BAF (SWI/SNF) nucleosome remodeling complex, has emerged as an attractive therapeutic target in cancer.  Despite the development of chem. probes targeting the BRD9 bromodomain, there is a limited understanding of BRD9 function beyond acetyl-lysine recognition.  We have therefore created the first BRD9-directed chem. degraders, through iterative design and testing of heterobifunctional ligands that bridge the BRD9 bromodomain and the cereblon E3 ubiquitin ligase complex.  Degraders of BRD9 exhibit markedly enhanced potency compared to parental ligands (10- to 100-fold).  Parallel study of degraders with divergent BRD9-binding chemotypes in models of acute myeloid leukemia resolves bromodomain polypharmacol. in this emerging drug class.  Together, these findings reveal the tractability of non-BET bromodomain contg. proteins to chem. degrdn., and highlight lead compd. dBRD9 as a tool for the study of BRD9.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQ9FU2Ipaq27Vg90H21EOLACvtfcHk0lgQtLVLjUzUAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmtFGrurk%253D&md5=2eb6562cc9d976c7ae6070c9a419e9ea</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1002%2Fanie.201611281&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201611281%26sid%3Dliteratum%253Aachs%26aulast%3DRemillard%26aufirst%3DD.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DBrien%26aufirst%3DG.%2BL.%26aulast%3DSonnett%26aufirst%3DM.%26aulast%3DSeo%26aufirst%3DH.%2BS.%26aulast%3DDastjerdi%26aufirst%3DS.%26aulast%3DWuhr%26aufirst%3DM.%26aulast%3DDhe-Paganon%26aufirst%3DS.%26aulast%3DArmstrong%26aufirst%3DS.%2BA.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DDegradation%2520of%2520the%2520BAF%2520Complex%2520Factor%2520BRD9%2520by%2520Heterobifunctional%2520Ligands%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2017%26volume%3D56%26spage%3D5738%26epage%3D5743%26doi%3D10.1002%2Fanie.201611281" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Salas, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Przybranowski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of a Small-Molecule Degrader of Bromodomain and Extra-Terminal (BET) Proteins with Picomolar Cellular Potencies and Capable of Achieving Tumor Regression</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">462</span>– <span class="NLM_lpage">481</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01816</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01816" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkvVajsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=462-481&author=B.+Zhouauthor=J.+Huauthor=F.+Xuauthor=Z.+Chenauthor=L.+Baiauthor=E.+Fernandez-Salasauthor=M.+Linauthor=L.+Liuauthor=C.+Y.+Yangauthor=Y.+Zhaoauthor=D.+McEachernauthor=S.+Przybranowskiauthor=B.+Wenauthor=D.+Sunauthor=S.+Wang&title=Discovery+of+a+Small-Molecule+Degrader+of+Bromodomain+and+Extra-Terminal+%28BET%29+Proteins+with+Picomolar+Cellular+Potencies+and+Capable+of+Achieving+Tumor+Regression&doi=10.1021%2Facs.jmedchem.6b01816"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Small-Molecule Degrader of Bromodomain and Extra-Terminal (BET) Proteins with Picomolar Cellular Potencies and Capable of Achieving Tumor Regression</span></div><div class="casAuthors">Zhou, Bing; Hu, Jiantao; Xu, Fuming; Chen, Zhuo; Bai, Longchuan; Fernandez-Salas, Ester; Lin, Mei; Liu, Liu; Yang, Chao-Yie; Zhao, Yujun; McEachern, Donna; Przybranowski, Sally; Wen, Bo; Sun, Duxin; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">462-481</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bromodomain and extra-terminal (BET) family proteins, consisting of BRD2, BRD3, BRD4, and testis-specific BRDT members, are epigenetic "readers" and play a key role in the regulation of gene transcription.  BET proteins are considered to be attractive therapeutic targets for cancer and other human diseases.  Recently, heterobifunctional small-mol. BET degraders have been designed based upon the proteolysis targeting chimera (PROTAC) concept to induce BET protein degrdn.  Herein, we present our design, synthesis, and evaluation of a new class of PROTAC BET degraders.  One of the most promising compds., 23, effectively degrades BRD4 protein at concns. as low as 30 pM in the RS4;11 leukemia cell line, achieves an IC50 value of 51 pM in inhibition of RS4;11 cell growth and induces rapid tumor regression in vivo against RS4;11 xenograft tumors.  These data establish that compd. 23 (BETd-260/ZBC260) is a highly potent and efficacious BET degrader.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqz2Lwp52x1rVg90H21EOLACvtfcHk0lgQtLVLjUzUAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkvVajsb8%253D&md5=6b3a10ecd192a5812ef581dca1fdc82a</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01816&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01816%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DB.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DF.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DBai%26aufirst%3DL.%26aulast%3DFernandez-Salas%26aufirst%3DE.%26aulast%3DLin%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DC.%2BY.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DPrzybranowski%26aufirst%3DS.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520a%2520Small-Molecule%2520Degrader%2520of%2520Bromodomain%2520and%2520Extra-Terminal%2520%2528BET%2529%2520Proteins%2520with%2520Picomolar%2520Cellular%2520Potencies%2520and%2520Capable%2520of%2520Achieving%2520Tumor%2520Regression%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D462%26epage%3D481%26doi%3D10.1021%2Facs.jmedchem.6b01816" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zengerle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Selective Small Molecule Induced Degradation of the BET Bromodomain Protein BRD4</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1770</span>– <span class="NLM_lpage">1777</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00216</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00216" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC2MXptlOgu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=1770-1777&author=M.+Zengerleauthor=K.+H.+Chanauthor=A.+Ciulli&title=Selective+Small+Molecule+Induced+Degradation+of+the+BET+Bromodomain+Protein+BRD4&doi=10.1021%2Facschembio.5b00216"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Small Molecule Induced Degradation of the BET Bromodomain Protein BRD4</span></div><div class="casAuthors">Zengerle, Michael; Chan, Kwok-Ho; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1770-1777</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The Bromo- and Extra-Terminal (BET) proteins BRD2, BRD3, and BRD4 play important roles in transcriptional regulation, epigenetics, and cancer and are the targets of pan-BET selective bromodomain inhibitor JQ1.  However, the lack of intra-BET selectivity limits the scope of current inhibitors as probes for target validation and could lead to unwanted side effects or toxicity in a therapeutic setting.  We designed Proteolysis Targeted Chimeras (PROTACs) that tether JQ1 to a ligand for the E3 ubiquitin ligase VHL, aimed at triggering the intracellular destruction of BET proteins.  Compd. MZ1 potently and rapidly induces reversible, long-lasting, and unexpectedly selective removal of BRD4 over BRD2 and BRD3.  The activity of MZ1 is dependent on binding to VHL but is achieved at a sufficiently low concn. not to induce stabilization of HIF-1α.  Gene expression profiles of selected cancer-related genes responsive to JQ1 reveal distinct and more limited transcriptional responses induced by MZ1, consistent with selective suppression of BRD4.  Our discovery opens up new opportunities to elucidate the cellular phenotypes and therapeutic implications assocd. with selective targeting of BRD4.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEmSv7p9MZOrVg90H21EOLACvtfcHk0litTpNPmiJDiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXptlOgu70%253D&md5=b7514fd274eff8e4bb6a771f5aaf9ab5</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00216&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00216%26sid%3Dliteratum%253Aachs%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DChan%26aufirst%3DK.%2BH.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DSelective%2520Small%2520Molecule%2520Induced%2520Degradation%2520of%2520the%2520BET%2520Bromodomain%2520Protein%2520BRD4%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D1770%26epage%3D1777%26doi%3D10.1021%2Facschembio.5b00216" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W.</span></span> <span> </span><span class="NLM_article-title">A covalently bound inhibitor triggers EZH2 degradation through CHIP-mediated ubiquitination</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">1243</span>– <span class="NLM_lpage">1260</span>, <span class="refDoi"> DOI: 10.15252/embj.201694058</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.15252%2Fembj.201694058" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=28320739" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC2sXksFSisr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2017&pages=1243-1260&author=X.+Wangauthor=W.+Caoauthor=J.+Zhangauthor=M.+Yanauthor=Q.+Xuauthor=X.+Wuauthor=L.+Wanauthor=Z.+Zhangauthor=C.+Zhangauthor=X.+Qinauthor=M.+Xiaoauthor=D.+Yeauthor=Y.+Liuauthor=Z.+Hanauthor=S.+Wangauthor=L.+Maoauthor=W.+Weiauthor=W.+Chen&title=A+covalently+bound+inhibitor+triggers+EZH2+degradation+through+CHIP-mediated+ubiquitination&doi=10.15252%2Fembj.201694058"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">A covalently bound inhibitor triggers EZH2 degradation through CHIP-mediated ubiquitination</span></div><div class="casAuthors">Wang, Xu; Cao, Wei; Zhang, Jianjun; Yan, Ming; Xu, Qin; Wu, Xiangbing; Wan, Lixin; Zhang, Zhiyuan; Zhang, Chenping; Qin, Xing; Xiao, Meng; Ye, Dongxia; Liu, Yuyang; Han, Zeguang; Wang, Shaomeng; Mao, Li; Wei, Wenyi; Chen, Wantao</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1243-1260</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Enhancer of zeste homolog 2 (EZH2) has been characterized as a crit. oncogene and a promising drug target in human malignant tumors.  The current EZH2 inhibitors strongly suppress the enhanced enzymic function of mutant EZH2 in some lymphomas.  However, the recent identification of a PRC2- and methyltransferase-independent role of EZH2 indicates that a complete suppression of all oncogenic functions of EZH2 is needed.  Here, we report a unique EZH2-targeting strategy by identifying a gambogenic acid (GNA) deriv. as a novel agent that specifically and covalently bound to Cys668 within the EZH2-SET domain, triggering EZH2 degrdn. through COOH terminus of Hsp70-interacting protein (CHIP)-mediated ubiquitination.  This class of inhibitors significantly suppressed H3K27Me3 and effectively reactivated polycomb repressor complex 2 (PRC2)-silenced tumor suppressor genes.  Moreover, the novel inhibitors significantly suppressed tumor growth in an EZH2-dependent manner, and tumors bearing a non-GNA-interacting C668S-EZH2 mutation exhibited resistance to the inhibitors.  Together, our results identify the inhibition of the signaling pathway that governs GNA-mediated destruction of EZH2 as a promising anti-cancer strategy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRQknSdWy5T7Vg90H21EOLACvtfcHk0litTpNPmiJDiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXksFSisr0%253D&md5=b9b8a266537e2922648784d9cd4edf02</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.15252%2Fembj.201694058&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.15252%252Fembj.201694058%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DX.%26aulast%3DCao%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DYan%26aufirst%3DM.%26aulast%3DXu%26aufirst%3DQ.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DWan%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DQin%26aufirst%3DX.%26aulast%3DXiao%26aufirst%3DM.%26aulast%3DYe%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DHan%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DMao%26aufirst%3DL.%26aulast%3DWei%26aufirst%3DW.%26aulast%3DChen%26aufirst%3DW.%26atitle%3DA%2520covalently%2520bound%2520inhibitor%2520triggers%2520EZH2%2520degradation%2520through%2520CHIP-mediated%2520ubiquitination%26jtitle%3DEMBO%2520J.%26date%3D2017%26volume%3D36%26spage%3D1243%26epage%3D1260%26doi%3D10.15252%2Fembj.201694058" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kasibhatla, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jessen, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maliartchouk, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">English, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewe, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Archer, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ponce, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sirisoma, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gehlsen, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, S. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tseng, B.</span></span> <span> </span><span class="NLM_article-title">A role for transferrin receptor in triggering apoptosis when targeted with gambogic acid</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>102</i></span>,  <span class="NLM_fpage">12095</span>– <span class="NLM_lpage">12100</span>, <span class="refDoi"> DOI: 10.1073/pnas.0406731102</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1073%2Fpnas.0406731102" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=16103367" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BD2MXps1yqsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2005&pages=12095-12100&author=S.+Kasibhatlaauthor=K.+A.+Jessenauthor=S.+Maliartchoukauthor=J.+Y.+Wangauthor=N.+M.+Englishauthor=J.+Dreweauthor=L.+Qiuauthor=S.+P.+Archerauthor=A.+E.+Ponceauthor=N.+Sirisomaauthor=S.+Jiangauthor=H.+Z.+Zhangauthor=K.+R.+Gehlsenauthor=S.+X.+Caiauthor=D.+R.+Greenauthor=B.+Tseng&title=A+role+for+transferrin+receptor+in+triggering+apoptosis+when+targeted+with+gambogic+acid&doi=10.1073%2Fpnas.0406731102"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">A role for transferrin receptor in triggering apoptosis when targeted with gambogic acid</span></div><div class="casAuthors">Kasibhatla, Shailaja; Jessen, Katayoun A.; Maliartchouk, Sergei; Wang, Jean Yu; English, Nicole M.; Drewe, John; Qiu, Ling; Archer, Shannon P.; Ponce, Anthony E.; Sirisoma, Nilantha; Jiang, Songchun; Zhang, Han-Zhong; Gehlsen, Kurt R.; Cai, Sui Xiong; Green, Douglas R.; Tseng, Ben</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">12095-12100</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Transferrin receptor (TfR) has been shown to be significantly overexpressed in different types of cancers.  We discovered TfR as a target for gambogic acid (GA), used in traditional Chinese medicine and a previously undiscovered link between TfR and the rapid activation of apoptosis.  The binding site of GA on TfR is independent of the transferrin binding site, and it appears that GA potentially inhibits TfR internalization.  Down-regulation of TfR by RNA interference decreases sensitivity to GA-induced apoptosis, further supporting TfR as the primary GA receptor.  In summary, GA binding to TfR induces a unique signal leading to rapid apoptosis of tumor cells.  These results suggest that GA may provide an addnl. approach for targeting the TfR and its use in cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPax6NHbKWnLVg90H21EOLACvtfcHk0litTpNPmiJDiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXps1yqsr4%253D&md5=f6cf82bfa7f841ca71046457dd7074c0</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0406731102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0406731102%26sid%3Dliteratum%253Aachs%26aulast%3DKasibhatla%26aufirst%3DS.%26aulast%3DJessen%26aufirst%3DK.%2BA.%26aulast%3DMaliartchouk%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DJ.%2BY.%26aulast%3DEnglish%26aufirst%3DN.%2BM.%26aulast%3DDrewe%26aufirst%3DJ.%26aulast%3DQiu%26aufirst%3DL.%26aulast%3DArcher%26aufirst%3DS.%2BP.%26aulast%3DPonce%26aufirst%3DA.%2BE.%26aulast%3DSirisoma%26aufirst%3DN.%26aulast%3DJiang%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DH.%2BZ.%26aulast%3DGehlsen%26aufirst%3DK.%2BR.%26aulast%3DCai%26aufirst%3DS.%2BX.%26aulast%3DGreen%26aufirst%3DD.%2BR.%26aulast%3DTseng%26aufirst%3DB.%26atitle%3DA%2520role%2520for%2520transferrin%2520receptor%2520in%2520triggering%2520apoptosis%2520when%2520targeted%2520with%2520gambogic%2520acid%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2005%26volume%3D102%26spage%3D12095%26epage%3D12100%26doi%3D10.1073%2Fpnas.0406731102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ma, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stratikopoulos, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwarz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leshchenko, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rialdi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dale, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lagana, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guccione, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parekh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parsons, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of a first-in-class EZH2 selective degrader</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">214</span>– <span class="NLM_lpage">222</span>, <span class="refDoi"> DOI: 10.1038/s41589-019-0421-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1038%2Fs41589-019-0421-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=31819273" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitlCht7zJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2020&pages=214-222&author=A.+Maauthor=E.+Stratikopoulosauthor=K.+S.+Parkauthor=J.+Weiauthor=T.+C.+Martinauthor=X.+Yangauthor=M.+Schwarzauthor=V.+Leshchenkoauthor=A.+Rialdiauthor=B.+Daleauthor=A.+Laganaauthor=E.+Guccioneauthor=S.+Parekhauthor=R.+Parsonsauthor=J.+Jin&title=Discovery+of+a+first-in-class+EZH2+selective+degrader&doi=10.1038%2Fs41589-019-0421-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a first-in-class EZH2 selective degrader</span></div><div class="casAuthors">Ma, Anqi; Stratikopoulos, Elias; Park, Kwang-Su; Wei, Jieli; Martin, Tiphaine C.; Yang, Xiaobao; Schwarz, Megan; Leshchenko, Violetta; Rialdi, Alexander; Dale, Brandon; Lagana, Alessandro; Guccione, Ernesto; Parekh, Samir; Parsons, Ramon; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">214-222</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The enhancer of zeste homolog 2 (EZH2) is the main enzymic subunit of the PRC2 complex, which catalyzes trimethylation of histone H3 lysine 27 (H3K27me3) to promote transcriptional silencing.  EZH2 is overexpressed in multiple types of cancer including triple-neg. breast cancer (TNBC), and high expression levels correlate with poor prognosis.  Several EZH2 inhibitors, which inhibit the methyltransferase activity of EZH2, have shown promise in treating sarcoma and follicular lymphoma in clinics.  However, EZH2 inhibitors are ineffective at blocking proliferation of TNBC cells, even though they effectively reduce the H3K27me3 mark.  Using a hydrophobic tagging approach, we generated MS1943, a first-in-class EZH2 selective degrader that effectively reduces EZH2 levels in cells.  Importantly, MS1943 has a profound cytotoxic effect in multiple TNBC cells, while sparing normal cells, and is efficacious in vivo, suggesting that pharmacol. degrdn. of EZH2 can be advantageous for treating the cancers that are dependent on EZH2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPb6bK3v_gQrVg90H21EOLACvtfcHk0lhwR7Dt0bmBPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitlCht7zJ&md5=e1ccae457eba9b298f8b0e583bbaa833</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1038%2Fs41589-019-0421-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-019-0421-4%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DA.%26aulast%3DStratikopoulos%26aufirst%3DE.%26aulast%3DPark%26aufirst%3DK.%2BS.%26aulast%3DWei%26aufirst%3DJ.%26aulast%3DMartin%26aufirst%3DT.%2BC.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DSchwarz%26aufirst%3DM.%26aulast%3DLeshchenko%26aufirst%3DV.%26aulast%3DRialdi%26aufirst%3DA.%26aulast%3DDale%26aufirst%3DB.%26aulast%3DLagana%26aufirst%3DA.%26aulast%3DGuccione%26aufirst%3DE.%26aulast%3DParekh%26aufirst%3DS.%26aulast%3DParsons%26aufirst%3DR.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520a%2520first-in-class%2520EZH2%2520selective%2520degrader%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2020%26volume%3D16%26spage%3D214%26epage%3D222%26doi%3D10.1038%2Fs41589-019-0421-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobrovolsky, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weisberg, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doctor, Z. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ko, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffin, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treon, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">A Chemoproteomic Approach to Query the Degradable Kinome Using a Multi-kinase Degrader</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">88</span>– <span class="NLM_lpage">99.e6</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.10.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1016%2Fj.chembiol.2017.10.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=29129717" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=88-99.e6&author=H.+T.+Huangauthor=D.+Dobrovolskyauthor=J.+Paulkauthor=G.+Yangauthor=E.+L.+Weisbergauthor=Z.+M.+Doctorauthor=D.+L.+Buckleyauthor=J.+H.+Choauthor=E.+Koauthor=J.+Jangauthor=K.+Shiauthor=H.+G.+Choiauthor=J.+D.+Griffinauthor=Y.+Liauthor=S.+P.+Treonauthor=E.+S.+Fischerauthor=J.+E.+Bradnerauthor=L.+Tanauthor=N.+S.+Gray&title=A+Chemoproteomic+Approach+to+Query+the+Degradable+Kinome+Using+a+Multi-kinase+Degrader&doi=10.1016%2Fj.chembiol.2017.10.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">A Chemoproteomic Approach to Query the Degradable Kinome Using a Multi-kinase Degrader</span></div><div class="casAuthors">Huang, Hai-Tsang; Dobrovolsky, Dennis; Paulk, Joshiawa; Yang, Guang; Weisberg, Ellen L.; Doctor, Zainab M.; Buckley, Dennis L.; Cho, Joong-Heui; Ko, Eunhwa; Jang, Jaebong; Shi, Kun; Choi, Hwan Geun; Griffin, James D.; Li, Ying; Treon, Steven P.; Fischer, Eric S.; Bradner, James E.; Tan, Li; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">88-99.e6</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Heterobifunctional mols. that recruit E3 ubiquitin ligases, such as cereblon, for targeted protein degrdn. represent an emerging pharmacol. strategy.  A major unanswered question is how generally applicable this strategy is to all protein targets.  In this study, we designed a multi-kinase degrader by conjugating a highly promiscuous kinase inhibitor with a cereblon-binding ligand, and used quant. proteomics to discover 28 kinases, including BTK, PTK2, PTK2B, FLT3, AURKA, AURKB, TEC, ULK1, ITK, and nine members of the CDK family, as degradable.  This set of kinases is only a fraction of the intracellular targets bound by the degrader, demonstrating that successful degrdn. requires more than target engagement.  The results guided us to develop selective degraders for FLT3 and BTK, with potentials to improve disease treatment.  Together, this study demonstrates an efficient approach to triage a gene family of interest to identify readily degradable targets for further studies and pre-clin. developments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYNxhoPV3127Vg90H21EOLACvtfcHk0lhwR7Dt0bmBPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjO&md5=703cfbaca3edd24176472158563f895f</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.10.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.10.005%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DH.%2BT.%26aulast%3DDobrovolsky%26aufirst%3DD.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DG.%26aulast%3DWeisberg%26aufirst%3DE.%2BL.%26aulast%3DDoctor%26aufirst%3DZ.%2BM.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DCho%26aufirst%3DJ.%2BH.%26aulast%3DKo%26aufirst%3DE.%26aulast%3DJang%26aufirst%3DJ.%26aulast%3DShi%26aufirst%3DK.%26aulast%3DChoi%26aufirst%3DH.%2BG.%26aulast%3DGriffin%26aufirst%3DJ.%2BD.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DTreon%26aufirst%3DS.%2BP.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DTan%26aufirst%3DL.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DA%2520Chemoproteomic%2520Approach%2520to%2520Query%2520the%2520Degradable%2520Kinome%2520Using%2520a%2520Multi-kinase%2520Degrader%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2018%26volume%3D25%26spage%3D88%26epage%3D99.e6%26doi%3D10.1016%2Fj.chembiol.2017.10.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, L.</span></span> <span> </span><span class="NLM_article-title">Design and Synthesis of EZH2-Based PROTACs to Degrade the PRC2 Complex for Targeting the Noncatalytic Activity of EZH2</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>64</i></span>,  <span class="NLM_fpage">2829</span>– <span class="NLM_lpage">2848</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c02234</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c02234" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BB3MXksV2hsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2021&pages=2829-2848&author=Z.+Liuauthor=X.+Huauthor=Q.+Wangauthor=X.+Wuauthor=Q.+Zhangauthor=W.+Weiauthor=X.+Suauthor=H.+Heauthor=S.+Zhouauthor=R.+Huauthor=T.+Yeauthor=Y.+Zhuauthor=N.+Wangauthor=L.+Yu&title=Design+and+Synthesis+of+EZH2-Based+PROTACs+to+Degrade+the+PRC2+Complex+for+Targeting+the+Noncatalytic+Activity+of+EZH2&doi=10.1021%2Facs.jmedchem.0c02234"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Synthesis of EZH2-Based PROTACs to Degrade the Complex for Targeting the Noncatalytic Activity of EZH2</span></div><div class="casAuthors">Liu, Zhihao; Hu, Xi; Wang, Qiwei; Wu, Xiuli; Zhang, Qiangsheng; Wei, Wei; Su, Xingping; He, Hualong; Zhou, Shuyan; Hu, Rong; Ye, Tinghong; Zhu, Yongxia; Wang, Ningyu; Yu, Luoting</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2829-2848</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">EZH2 mediates both PRC2-dependent gene silencing via catalyzing H3K27me3 and PRC2-independent transcriptional activation in various cancers.  Given its oncogenic role in cancers, EZH2 has constituted a compelling target for anticancer therapy.  However, current EZH2 inhibitors only target its methyltransferase activity to downregulate H3K27me3 levels and show limited efficacy because of inadequate suppression of the EZH2 oncogenic activity.  Therefore, therapeutic strategies to completely block the oncogenic activity of EZH2 are urgently needed.  Herein, we report a series of EZH2-targeted proteolysis targeting chimeras (PROTACs) that induce proteasomal degrdn. of PRC2 components, including EZH2, EED, SUZ12, and RbAp48.  Preliminary assessment identified E7 as the most active PROTAC mol., which decreased PRC2 subunits and H3K27me2/3 levels in various cancer cells.  Furthermore, E7 strongly inhibited transcriptional silencing mediated by EZH2 dependent on PRC2 and transcriptional activation mediated by EZH2 independent of PRC2, showing significant antiproliferative activities against cancer cell lines dependent on the enzymic and nonenzymic activities of EZH2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBGvGI0J8IlbVg90H21EOLACvtfcHk0lhwR7Dt0bmBPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXksV2hsrs%253D&md5=6be23721044612d4de7d1446af35f698</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c02234&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c02234%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DHu%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DWei%26aufirst%3DW.%26aulast%3DSu%26aufirst%3DX.%26aulast%3DHe%26aufirst%3DH.%26aulast%3DZhou%26aufirst%3DS.%26aulast%3DHu%26aufirst%3DR.%26aulast%3DYe%26aufirst%3DT.%26aulast%3DZhu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DN.%26aulast%3DYu%26aufirst%3DL.%26atitle%3DDesign%2520and%2520Synthesis%2520of%2520EZH2-Based%2520PROTACs%2520to%2520Degrade%2520the%2520PRC2%2520Complex%2520for%2520Targeting%2520the%2520Noncatalytic%2520Activity%2520of%2520EZH2%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2021%26volume%3D64%26spage%3D2829%26epage%3D2848%26doi%3D10.1021%2Facs.jmedchem.0c02234" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glorieux, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, P.</span></span> <span> </span><span class="NLM_article-title">Chemotherapy induces tumor immune evasion by upregulation of programmed cell death ligand 1 expression in bone marrow stromal cells</span>. <i>Mol. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">358</span>– <span class="NLM_lpage">372</span>, <span class="refDoi"> DOI: 10.1002/1878-0261.12032</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1002%2F1878-0261.12032" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=28218497" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC2sXnsV2jsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2017&pages=358-372&author=M.+Yangauthor=P.+Liuauthor=K.+Wangauthor=C.+Glorieuxauthor=Y.+Huauthor=S.+Wenauthor=W.+Jiangauthor=P.+Huang&title=Chemotherapy+induces+tumor+immune+evasion+by+upregulation+of+programmed+cell+death+ligand+1+expression+in+bone+marrow+stromal+cells&doi=10.1002%2F1878-0261.12032"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Chemotherapy induces tumor immune evasion by upregulation of programmed cell death ligand 1 expression in bone marrow stromal cells</span></div><div class="casAuthors">Yang, Mengqi; Liu, Panpan; Wang, Kefeng; Glorieux, Christophe; Hu, Yumin; Wen, Shijun; Jiang, Wenqi; Huang, Peng</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Oncology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">358-372</span>CODEN:
                <span class="NLM_cas:coden">MOONC3</span>;
        ISSN:<span class="NLM_cas:issn">1878-0261</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons Ltd.</span>)
        </div><div class="casAbstract">Programmed cell death ligand 1 (PD-L1) is a neg. regulator of the immune response that enables tumor cells to escape T-cell immunity.  Although PD-L1 expression in cancer cells has been extensively studied, the expression of PD-L1 in stromal cells and its clin. significance remain largely unknown.  Here, we show that bone marrow stromal cells express a low level of PD-L1 and that this mol. is significantly upregulated by key drugs used in the treatment of lymphoma at clin. relevant concns.  Mechanistically, chemotherapeutic drugs induce PD-L1 expression in stromal cells through upregulation of granulocyte macrophage colony-stimulating factor and activation of the extracellular signal-regulated kinase (ERK) 1/2 signaling pathway.  Suppression of ERK by a chem. inhibitor or genetic silencing of ERK2 expression prevents drug-induced PD-L1 expression.  PD-L1 expression is upregulated in the bone marrow stromal cells of mice treated with doxorubicin and in drug-treated bone marrow specimens from lymphoma patients.  Drug-induced PD-L1 expression in stromal cells can cause significant impairment of T-cell functions.  Overall, our study reveals a previously unrecognized mechanism by which chemotherapy induces tumor immune evasion by upregulation of PD-L1 in bone marrow stromal cells, and provides new evidence for the combination of chemotherapy and anti-PD-L1/PD-1 as an effective strategy for treatment of lymphoma and other cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqe6PgoFUeWp7Vg90H21EOLACvtfcHk0liLSCWa6CqQDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXnsV2jsrg%253D&md5=d56540bfdcd424ee43174ef3c940a061</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1002%2F1878-0261.12032&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F1878-0261.12032%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DP.%26aulast%3DWang%26aufirst%3DK.%26aulast%3DGlorieux%26aufirst%3DC.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DWen%26aufirst%3DS.%26aulast%3DJiang%26aufirst%3DW.%26aulast%3DHuang%26aufirst%3DP.%26atitle%3DChemotherapy%2520induces%2520tumor%2520immune%2520evasion%2520by%2520upregulation%2520of%2520programmed%2520cell%2520death%2520ligand%25201%2520expression%2520in%2520bone%2520marrow%2520stromal%2520cells%26jtitle%3DMol.%2520Oncol.%26date%3D2017%26volume%3D11%26spage%3D358%26epage%3D372%26doi%3D10.1002%2F1878-0261.12032" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article has not yet been cited by other publications.</p></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00460/20210716/images/medium/jm1c00460_0012.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00460/20210716/images/large/jm1c00460_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00460&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00460/20210716/images/medium/jm1c00460_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00460/20210716/images/large/jm1c00460_0002.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Strategy to develop EZH2 degraders based on PROTAC technology. (A) MTS cell viability analysis in different lymphoma cell lines treated with indicated concentrations of EPZ6438 for 5 days. (B) Western blot analysis of EZH2 protein levels in different lymphoma cell lines. Tubulin was used as a loading control. (C) Cell numbers were measured by cell count after the transfection of control shRNA or shEZH2 in SU-DHL-2 cells for the indicated days (left). EZH2 and H3K27me3 levels were measured by western blot assay. H3 and tubulin were used as loading controls (right). (D) Schematic representation of the EZH2 degrader design. (E) Docking conformation of EPZ6438 in the catalytic domain of EZH2 (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LS6">5LS6</a>, left) and the chemical structure of EPZ6438 (right). Note: EPZ stands for EPZ6438 in this figure and the following figures.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00460/20210716/images/large/jm1c00460_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00460&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00460/20210716/images/medium/jm1c00460_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00460/20210716/images/large/jm1c00460_0003.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Chemical structures and degradation efficacy of the EZH2 degraders. (A) Chemical structures and linker lengths of VHL-based PROTACs. (B) Western blot analysis of EZH2 and H3K27me3 levels in 22Rv1 cells treated with indicated concentrations of compounds or dimethyl sulfoxide (DMSO) for 48 h. H3 and tubulin were used as loading controls. (C, D) Scatter plots showing the relative (C) H3K27me3 and (D) EZH2 bands abundance in B. (E) Chemical structures and degradation efficiency of CRBN-based PROTACs. N, no significant effects on the related target and Y, significant effects on the related target.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00460/20210716/images/large/jm1c00460_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00460&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00460/20210716/images/medium/jm1c00460_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00460/20210716/images/large/jm1c00460_0004.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. EZH2 degraders abrogated the EZH2 protein level and the PRC2 complex through the VHL-dependent ubiquitin–proteasome system. (A) Chemical structures of YM181 and YM281. (B) Relative protein bands abundance in 22Rv1 cells treated with indicated concentrations of YM181 and YM281 for 24 h. (C) Western blot analysis of EZH2 and H3K27me3 levels in 22Rv1 cells treated with YM181 (2 μM) for the indicated exposure time. H3 and tubulin were used as loading controls. (D) Western blot analysis of the indicated protein levels in 22Rv1 and SU-DHL-2 cells treated with YM181 (2 μM) for 24 h. H3 was used as a loading control. (E) Western blot analysis of EZH2 levels in 22Rv1 cells treated with YM181 (2 μM) for 24 h after 2 h pretreatment with DMSO, MG132 (0.5 μM), MLN-4924 (0.4 μM), EPZ6438 (2 μM), and VH032 (2 μM). Tubulin was used as a loading control. (F) Western blot analysis of EZH2 and VHL levels after the transfection of nontargeting (NC) or VHL siRNAs for 48 h, followed by a 24 h YM181 treatment (2 μM) in 22Rv1 cells. Tubulin was used as a loading control. (G) Immunoprecipitation-western blot analysis of ubiquitylated EZH2 levels in 22Rv1 cells treated with 2 μM YM181 and YM281 for 12 h. Tubulin was used as a loading control.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00460/20210716/images/large/jm1c00460_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00460&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00460/20210716/images/medium/jm1c00460_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00460/20210716/images/large/jm1c00460_0005.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. EZH2 degraders displayed stronger anticancer abilities than EPZ6438 in lymphoma cell lines. (A) MTS cell viability curves in different DLBCL cell lines treated with indicated compounds for 5 days. (B) Western blot analysis of EZH2 and H3K27me3 levels in SU-DHL-2 cells treated with the indicated compounds for 24 h. H3 and tubulin were used as loading controls. (C) Cell cycle analysis in SU-DHL-6 cells treated with DMSO or indicated compounds for 24 h. (D) Western blot analysis of PARP, caspase-3, and cleaved-caspase-3 levels in SU-DHL-6 cells treated with indicated compounds for 48 h. Tubulin was used as a loading control. (E) Proportions of PI<sup>+</sup> and/or Annexin-V<sup>+</sup> apoptotic cells in SU-DHL-6 cells treated with indicated compounds for 48 h as measured by flow cytometry.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00460/20210716/images/large/jm1c00460_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00460&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00460/20210716/images/medium/jm1c00460_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00460/20210716/images/large/jm1c00460_0006.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. EZH2 degraders reduced tumor growth in vivo and decreased cell viability in primary lymphoma patient cells. (A) Tumor volume of Balb/c nude mice bearing SU-DHL-6 xenograft administrated intraperitoneally with a vehicle, YM281 (80 mg/kg) or EPZ (42.5 mg/kg), for 3 weeks. (B) Western blot analysis of EZH2 and H3K27me3 levels in the representative SU-DHL-6 model excised tumors. GAPDH was used as a loading control. Representative tumor images were presented. (C) Immunohistochemistry analysis of EZH2, H3K27me3, and Ki67 levels in the representative excised tumors from A. (D) Tumor volume of Balb/c nude mice bearing the Jeko-1 xenograft administrated intraperitoneally with a vehicle, YM281 (100 mg/kg) or EPZ (50 mg/kg), for 30 days. (E) Western blot analysis of EZH2 and H3K27me3 in the representative Jeko-1 model excised tumors. Tubulin was used as a loading control. Representative tumor images were presented. (F) Immunohistochemistry analysis of EZH2 and Ki67 levels in the representative Jeko-1 model excised tumors. (G) Western blot analysis of EZH2 and H3K27me3 levels in a DLBCL patient sample’s cells treated with YM281 at indicated concentrations for 24 h. H3 and tubulin were used as loading controls. (H) Caspase-3/7 activity of patient-derived primary lymphoma cells treated with each compound at indicated concentrations for 48 h in 11 lymphoma patient cases. (I) Quantitation of the present ATP level in patient-derived cells with each compound at indicated concentrations for 48 h in 11 lymphoma patient cases.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00460/20210716/images/large/jm1c00460_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00460&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00460/20210716/images/medium/jm1c00460_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00460/20210716/images/large/jm1c00460_0007.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Intermediate Compounds <b>6</b> and <b>7</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00460/20210716/images/large/jm1c00460_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00460&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) Fe, NH<sub>4</sub>Cl, MeOH, 90 °C; (b) tetrahydro-4<i>H</i>-pyran-4-one, AcOH, Na(AcO)<sub>3</sub>BH, 1,2-dichloroethane; (c) acetaldehyde, AcOH, Na(AcO)<sub>3</sub>BH, 1,2-dichloroethane; (d) Boc<sub>2</sub>O, H<sub>2</sub>, Raney-Ni, MeOH; (e) HCl/MeOH; (f) NaOH, EtOH, 60 °C; (g) HOBt, EDCI, NMM, DMSO; (h) 4-methoxycarbonylphenylboronic acid, K<sub>2</sub>CO<sub>3</sub>, Pd(PPh<sub>3</sub>)<sub>4</sub>, <i>N</i>,<i>N</i>-dimethylformamide (DMF), 90 °C; and (i) NaOH, EtOH, 60 °C. Note: unless stated, reactions underwent at room temperature in all synthetic schemes.</p></p></figure><figure data-id="sch2" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00460/20210716/images/medium/jm1c00460_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00460/20210716/images/large/jm1c00460_0008.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of VHL Ligands <b>15</b>, <b>16</b>, and <b>VH032</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00460/20210716/images/large/jm1c00460_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00460&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) NaOH, Boc<sub>2</sub>O, MeOH; (b) Pd(OAc)<sub>2</sub>, KOAc, <i>N</i>,<i>N</i>-dimethylacetamide (DMA), 150 °C; (c) HCl/MeOH; (d) <b>12</b>, HATU, <i>N</i>,<i>N</i>-diisopropylethylamine (DIPEA), tetrahydrofuran (THF); (e) HCl/MeOH; (f) <b>14</b>, HATU, DIPEA, DMF; (g) HCl/MeOH; and (h) DIPEA, acetic anhydride, CH<sub>2</sub>Cl<sub>2</sub>.</p></p></figure><figure data-id="sch3" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00460/20210716/images/medium/jm1c00460_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00460/20210716/images/large/jm1c00460_0009.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Series V Compounds<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00460/20210716/images/large/jm1c00460_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00460&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) BnBr, NaH, THF; (b) <i>tert</i>-butyl bromoacetate, NaOH, TBACl, CH<sub>2</sub>Cl<sub>2</sub>; (c) Pd/C, H<sub>2</sub>, EtOH; (d) TsCl, DMAP, TEA, CH<sub>2</sub>Cl<sub>2</sub>; (e) 4-hydroxyphenylboronic acid pinacol ester, K<sub>2</sub>CO<sub>3,</sub> DMF, 70 °C; (f) <b>6</b>, K<sub>3</sub>PO<sub>4</sub>, Pd(PPh)<sub>4,</sub> DMF, 90 °C; (g) 20% TFA, CH<sub>2</sub>Cl<sub>2</sub>; (h) <b>16</b>, HATU, DIPEA, DMF; (i) NaH, <i>tert</i>-butyl bromoacetate, DMF; (j) TsCl, DMAP, TEA, CH<sub>2</sub>Cl<sub>2</sub>; (k) 4-hydroxyphenylboronic acid pinacol ester, K<sub>2</sub>CO<sub>3</sub>, DMF, 70 °C; (l) <b>6</b>, K<sub>3</sub>PO<sub>4</sub>, Pd(PPh)<sub>4</sub>, DMF, 90 °C; (m) 20% TFA, CH<sub>2</sub>Cl<sub>2</sub>; (n) <b>16</b>, HATU, DIPEA, DMF; (o) diethylamine, CH<sub>2</sub>Cl<sub>2</sub>; (p) Boc<sub>2</sub>O, NaOH, CH<sub>2</sub>Cl<sub>2</sub>; (q) <b>16</b>, HATU, DIPEA, DMF; (r) HCl/MeOH; and (s) <b>7</b>, HATU, DIPEA, DMF.</p></p></figure><figure data-id="sch4" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00460/20210716/images/medium/jm1c00460_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00460/20210716/images/large/jm1c00460_0010.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of <b>YM620</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00460/20210716/images/large/jm1c00460_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00460&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <b>11</b>, HATU, DIPEA, THF; (b) HCl/MeOH; (c) <b>14</b>, HATU, DIPEA; (d) HCl/MeOH; and (e) <b>21b</b>, HATU, DIPEA, DMF.</p></p></figure><figure data-id="sch5" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00460/20210716/images/medium/jm1c00460_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00460/20210716/images/large/jm1c00460_0011.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of Series G Compounds<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/acs.jmedchem.1c00460/20210716/images/large/jm1c00460_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00460&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <b>29</b>, NaOAc, AcOH, 120 °C reflux, or <b>30</b>, pyridine, 110 °C; (b) <b>34a–b</b>, DIPEA, DMF 90 °C or <b>34c–d</b>, Na<sub>2</sub>CO<sub>3</sub>, DMF, 80 °C; (c) 20% TFA, CH<sub>2</sub>Cl<sub>2</sub>; (d) <b>7</b>, HATU, DIPEA, DMF; (e) <i>ter</i>t-butyl bromoacetate, K<sub>2</sub>CO<sub>3</sub>, DMF; (f) 20% TFA, CH<sub>2</sub>Cl<sub>2</sub>; (g) HATU, DIPEA, DMF; (h) 20% TFA, CH<sub>2</sub>Cl<sub>2</sub>, 50 °C; and (i) <b>7</b>, HATU, DIPEA, DMF.</p></p></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i57">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_23517" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_23517" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 49 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Margueron, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reinberg, D.</span></span> <span> </span><span class="NLM_article-title">The Polycomb complex PRC2 and its mark in life</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>469</i></span>,  <span class="NLM_fpage">343</span>– <span class="NLM_lpage">349</span>, <span class="refDoi"> DOI: 10.1038/nature09784</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1038%2Fnature09784" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=21248841" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC3MXnvVOisA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=469&publication_year=2011&pages=343-349&author=R.+Margueronauthor=D.+Reinberg&title=The+Polycomb+complex+PRC2+and+its+mark+in+life&doi=10.1038%2Fnature09784"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The Polycomb complex PRC2 and its mark in life</span></div><div class="casAuthors">Margueron, Raphael; Reinberg, Danny</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">469</span>
        (<span class="NLM_cas:issue">7330</span>),
    <span class="NLM_cas:pages">343-349</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Polycomb group proteins maintain the gene-expression pattern of different cells that is set during early development by regulating chromatin structure.  In mammals, 2 main Polycomb group complexes exist: Polycomb repressive complex 1 (PRC1) and 2 (PRC2).  PRC1 compacts chromatin and catalyzes the monoubiquitination of histone H2A.  PRC2 also contributes to chromatin compaction, and catalyzes the methylation of histone H3 at Lys-27.  PRC2 is involved in various biol. processes, including differentiation, maintaining cell identity and proliferation, and stem-cell plasticity.  Recent studies of PRC2 have expanded the perspectives on its function and regulation, and uncovered a role for non-coding RNA in the recruitment of PRC2 to target genes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7fYmuBqnMA7Vg90H21EOLACvtfcHk0lhCSYHGykVz0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXnvVOisA%253D%253D&md5=5f15e91cc6ff765f649842041f333841</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnature09784&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09784%26sid%3Dliteratum%253Aachs%26aulast%3DMargueron%26aufirst%3DR.%26aulast%3DReinberg%26aufirst%3DD.%26atitle%3DThe%2520Polycomb%2520complex%2520PRC2%2520and%2520its%2520mark%2520in%2520life%26jtitle%3DNature%26date%3D2011%26volume%3D469%26spage%3D343%26epage%3D349%26doi%3D10.1038%2Fnature09784" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Di
Croce, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helin, K.</span></span> <span> </span><span class="NLM_article-title">Transcriptional regulation by Polycomb group proteins</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">1147</span>– <span class="NLM_lpage">1155</span>, <span class="refDoi"> DOI: 10.1038/nsmb.2669</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1038%2Fnsmb.2669" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=24096405" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFOiurrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2013&pages=1147-1155&author=L.+Di%0ACroceauthor=K.+Helin&title=Transcriptional+regulation+by+Polycomb+group+proteins&doi=10.1038%2Fnsmb.2669"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Transcriptional regulation by Polycomb group proteins</span></div><div class="casAuthors">Di Croce, Luciano; Helin, Kristian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1147-1155</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Polycomb group (PcG) proteins are epigenetic regulators of transcription that have key roles in stem-cell identity, differentiation and disease.  Mechanistically, they function within multiprotein complexes, called Polycomb repressive complexes (PRCs), which modify histones (and other proteins) and silence target genes.  The dynamics of PRC1 and PRC2 components has been the focus of recent research.  Here we discuss our current knowledge of the PRC complexes, how they are targeted to chromatin and how the high diversity of the PcG proteins allows these complexes to influence cell identity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrArNyrbNU897Vg90H21EOLACvtfcHk0lhCSYHGykVz0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFOiurrP&md5=004c2682edbda681e9f6de84ba87e639</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.2669&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.2669%26sid%3Dliteratum%253Aachs%26aulast%3DDi%2BCroce%26aufirst%3DL.%26aulast%3DHelin%26aufirst%3DK.%26atitle%3DTranscriptional%2520regulation%2520by%2520Polycomb%2520group%2520proteins%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2013%26volume%3D20%26spage%3D1147%26epage%3D1155%26doi%3D10.1038%2Fnsmb.2669" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Velichutina, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaknovich, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gascoyne, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melnick, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elemento, O.</span></span> <span> </span><span class="NLM_article-title">EZH2-mediated epigenetic silencing in germinal center B cells contributes to proliferation and lymphomagenesis</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">5247</span>– <span class="NLM_lpage">5255</span>, <span class="refDoi"> DOI: 10.1182/blood-2010-04-280149</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1182%2Fblood-2010-04-280149" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=20736451" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC3MXot1I%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2010&pages=5247-5255&author=I.+Velichutinaauthor=R.+Shaknovichauthor=H.+Gengauthor=N.+A.+Johnsonauthor=R.+D.+Gascoyneauthor=A.+M.+Melnickauthor=O.+Elemento&title=EZH2-mediated+epigenetic+silencing+in+germinal+center+B+cells+contributes+to+proliferation+and+lymphomagenesis&doi=10.1182%2Fblood-2010-04-280149"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">EZH2-mediated epigenetic silencing in germinal center B cells contributes to proliferation and lymphomagenesis</span></div><div class="casAuthors">Velichutina, Irina; Shaknovich, Rita; Geng, Huimin; Johnson, Nathalie A.; Gascoyne, Randy D.; Melnick, Ari M.; Elemento, Olivier</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">5247-5255</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">EZH2 is the catalytic subunit of the PRC2 Polycomb complex and mediates transcriptional repression through its histone methyltransferase activity.  EZH2 is up-regulated in normal germinal center (GC) B cells and is implicated in lymphomagenesis.  To explore the transcriptional programs controlled by EZH2, we performed chromatin immunopptn. (ChIP-on-chip) in GC cells and found that it binds approx. 1800 promoters, often assocd. with DNA sequences similar to Drosophila Polycomb response elements.  While EZH2 targets overlapped extensively between GC B cells and embryonic stem cells, we also obsd. a large GC-specific EZH2 regulatory program.  These genes are preferentially histone 3 lysine 27-trimethylated and repressed in GC B cells and include several key cell cycle-related tumor suppressor genes.  Accordingly, siRNA-mediated down-regulation of EZH2 in diffuse large B-cell lymphoma (DLBCL) cells resulted in acute cell cycle arrest at the G1/S transition and up-regulation of its tumor suppressor target genes.  At the DNA level, EZH2-bound promoters are hypomethylated in GC B cells, but many of them are aberrantly hypermethylated in DLBCL, suggesting disruption of normal epigenetic processes in these cells.  EZH2 is thus involved in regulating a specific epigenetic program in normal GCs, including silencing of antiproliferative genes, which may contribute to the malignant transformation of GC B cells into DLBCLs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobqjlF41Jsh7Vg90H21EOLACvtfcHk0lhCSYHGykVz0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXot1I%253D&md5=4f50ce144655d0b3a3b4de12c069ee22</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1182%2Fblood-2010-04-280149&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2010-04-280149%26sid%3Dliteratum%253Aachs%26aulast%3DVelichutina%26aufirst%3DI.%26aulast%3DShaknovich%26aufirst%3DR.%26aulast%3DGeng%26aufirst%3DH.%26aulast%3DJohnson%26aufirst%3DN.%2BA.%26aulast%3DGascoyne%26aufirst%3DR.%2BD.%26aulast%3DMelnick%26aufirst%3DA.%2BM.%26aulast%3DElemento%26aufirst%3DO.%26atitle%3DEZH2-mediated%2520epigenetic%2520silencing%2520in%2520germinal%2520center%2520B%2520cells%2520contributes%2520to%2520proliferation%2520and%2520lymphomagenesis%26jtitle%3DBlood%26date%3D2010%26volume%3D116%26spage%3D5247%26epage%3D5255%26doi%3D10.1182%2Fblood-2010-04-280149" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ntziachristos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsirigos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Vlierberghe, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nedjic, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trimarchi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flaherty, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferres-Marco, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">da Ros, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siegle, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asp, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hadler, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rigo, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Keersmaecker, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huynh, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Utro, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poglio, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samon, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paietta, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Racevskis, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rowe, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabadan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levine, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pflumio, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dominguez, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrando, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aifantis, I.</span></span> <span> </span><span class="NLM_article-title">Genetic inactivation of the polycomb repressive complex 2 in T cell acute lymphoblastic leukemia</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">298</span>– <span class="NLM_lpage">301</span>, <span class="refDoi"> DOI: 10.1038/nm.2651</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1038%2Fnm.2651" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=22237151" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC38XlsFyrtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2012&pages=298-301&author=P.+Ntziachristosauthor=A.+Tsirigosauthor=P.+Van+Vlierbergheauthor=J.+Nedjicauthor=T.+Trimarchiauthor=M.+S.+Flahertyauthor=D.+Ferres-Marcoauthor=V.+da+Rosauthor=Z.+Tangauthor=J.+Siegleauthor=P.+Aspauthor=M.+Hadlerauthor=I.+Rigoauthor=K.+De+Keersmaeckerauthor=J.+Patelauthor=T.+Huynhauthor=F.+Utroauthor=S.+Poglioauthor=J.+B.+Samonauthor=E.+Paiettaauthor=J.+Racevskisauthor=J.+M.+Roweauthor=R.+Rabadanauthor=R.+L.+Levineauthor=S.+Brownauthor=F.+Pflumioauthor=M.+Dominguezauthor=A.+Ferrandoauthor=I.+Aifantis&title=Genetic+inactivation+of+the+polycomb+repressive+complex+2+in+T+cell+acute+lymphoblastic+leukemia&doi=10.1038%2Fnm.2651"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Genetic inactivation of the polycomb repressive complex 2 in T cell acute lymphoblastic leukemia</span></div><div class="casAuthors">Ntziachristos, Panagiotis; Tsirigos, Aristotelis; Van Vlierberghe, Pieter; Nedjic, Jelena; Trimarchi, Thomas; Sol Flaherty, Maria; Ferres-Marco, Dolors; da Ros, Vanina; Tang, Zuojian; Siegle, Jasmin; Asp, Patrik; Hadler, Michael; Rigo, Isaura; De Keersmaecker, Kim; Patel, Jay; Huynh, Tien; Utro, Filippo; Poglio, Sandrine; Samon, Jeremy B.; Paietta, Elisabeth; Racevskis, Janis; Rowe, Jacob M.; Rabadan, Raul; Levine, Ross L.; Brown, Stuart; Pflumio, Francoise; Dominguez, Maria; Ferrando, Adolfo; Aifantis, Iannis</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">298-303</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">T cell acute lymphoblastic leukemia (T-ALL) is an immature hematopoietic malignancy driven mainly by oncogenic activation of NOTCH1 signaling.  In this study we report the presence of loss-of-function mutations and deletions of the EZH2 and SUZ12 genes, which encode crucial components of the Polycomb repressive complex 2 (PRC2), in 25% of T-ALLs.  To further study the role of PRC2 in T-ALL, we used NOTCH1-dependent mouse models of the disease, as well as human T-ALL samples, and combined locus-specific and global anal. of NOTCH1-driven epigenetic changes.  These studies demonstrated that activation of NOTCH1 specifically induces loss of the repressive mark Lys27 trimethylation of histone 3 (H3K27me3) by antagonizing the activity of PRC2.  These studies suggest a tumor suppressor role for PRC2 in human leukemia and suggest a hitherto unrecognized dynamic interplay between oncogenic NOTCH1 and PRC2 function for the regulation of gene expression and cell transformation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNZkInF-IGxbVg90H21EOLACvtfcHk0liHRQqjvG45iw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XlsFyrtQ%253D%253D&md5=061baa595535b99856ded964c146091e</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnm.2651&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.2651%26sid%3Dliteratum%253Aachs%26aulast%3DNtziachristos%26aufirst%3DP.%26aulast%3DTsirigos%26aufirst%3DA.%26aulast%3DVan%2BVlierberghe%26aufirst%3DP.%26aulast%3DNedjic%26aufirst%3DJ.%26aulast%3DTrimarchi%26aufirst%3DT.%26aulast%3DFlaherty%26aufirst%3DM.%2BS.%26aulast%3DFerres-Marco%26aufirst%3DD.%26aulast%3Dda%2BRos%26aufirst%3DV.%26aulast%3DTang%26aufirst%3DZ.%26aulast%3DSiegle%26aufirst%3DJ.%26aulast%3DAsp%26aufirst%3DP.%26aulast%3DHadler%26aufirst%3DM.%26aulast%3DRigo%26aufirst%3DI.%26aulast%3DDe%2BKeersmaecker%26aufirst%3DK.%26aulast%3DPatel%26aufirst%3DJ.%26aulast%3DHuynh%26aufirst%3DT.%26aulast%3DUtro%26aufirst%3DF.%26aulast%3DPoglio%26aufirst%3DS.%26aulast%3DSamon%26aufirst%3DJ.%2BB.%26aulast%3DPaietta%26aufirst%3DE.%26aulast%3DRacevskis%26aufirst%3DJ.%26aulast%3DRowe%26aufirst%3DJ.%2BM.%26aulast%3DRabadan%26aufirst%3DR.%26aulast%3DLevine%26aufirst%3DR.%2BL.%26aulast%3DBrown%26aufirst%3DS.%26aulast%3DPflumio%26aufirst%3DF.%26aulast%3DDominguez%26aufirst%3DM.%26aulast%3DFerrando%26aufirst%3DA.%26aulast%3DAifantis%26aufirst%3DI.%26atitle%3DGenetic%2520inactivation%2520of%2520the%2520polycomb%2520repressive%2520complex%25202%2520in%2520T%2520cell%2520acute%2520lymphoblastic%2520leukemia%26jtitle%3DNat.%2520Med.%26date%3D2012%26volume%3D18%26spage%3D298%26epage%3D301%26doi%3D10.1038%2Fnm.2651" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Z. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groner, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lis, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stack, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loda, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cato, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thornton, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gregory, R. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morrissey, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vessella, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montironi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magi-Galluzzi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kantoff, P. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balk, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, M.</span></span> <span> </span><span class="NLM_article-title">EZH2 oncogenic activity in castration-resistant prostate cancer cells is Polycomb-independent</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>338</i></span>,  <span class="NLM_fpage">1465</span>– <span class="NLM_lpage">1469</span>, <span class="refDoi"> DOI: 10.1126/science.1227604</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1126%2Fscience.1227604" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=23239736" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVSktLfF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=338&publication_year=2012&pages=1465-1469&author=K.+Xuauthor=Z.+J.+Wuauthor=A.+C.+Gronerauthor=H.+H.+Heauthor=C.+Caiauthor=R.+T.+Lisauthor=X.+Wuauthor=E.+C.+Stackauthor=M.+Lodaauthor=T.+Liuauthor=H.+Xuauthor=L.+Catoauthor=J.+E.+Thorntonauthor=R.+I.+Gregoryauthor=C.+Morrisseyauthor=R.+L.+Vessellaauthor=R.+Montironiauthor=C.+Magi-Galluzziauthor=P.+W.+Kantoffauthor=S.+P.+Balkauthor=X.+S.+Liuauthor=M.+Brown&title=EZH2+oncogenic+activity+in+castration-resistant+prostate+cancer+cells+is+Polycomb-independent&doi=10.1126%2Fscience.1227604"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">EZH2 oncogenic activity in castration-resistant prostate cancer cells is polycomb-independent</span></div><div class="casAuthors">Xu, Kexin; Wu, Zhenhua Jeremy; Groner, Anna C.; He, Housheng Hansen; Cai, Changmeng; Lis, Rosina T.; Wu, Xiaoqiu; Stack, Edward C.; Loda, Massimo; Liu, Tao; Xu, Han; Cato, Laura; Thornton, James E.; Gregory, Richard I.; Morrissey, Colm; Vessella, Robert L.; Montironi, Rodolfo; Magi-Galluzzi, Cristina; Kantoff, Philip W.; Balk, Steven P.; Liu, X. Shirley; Brown, Myles</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">338</span>
        (<span class="NLM_cas:issue">6113</span>),
    <span class="NLM_cas:pages">1465-1469</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Epigenetic regulators represent a promising new class of therapeutic targets for cancer.  Enhancer of zeste homolog 2 (EZH2), a subunit of Polycomb repressive complex 2 (PRC2), silences gene expression via its histone methyltransferase activity.  We found that the oncogenic function of EZH2 in cells of castration-resistant prostate cancer is independent of its role as a transcriptional repressor.  Instead, it involves the ability of EZH2 to act as a coactivator for crit. transcription factors including the androgen receptor.  This functional switch is dependent on phosphorylation of EZH2 and requires an intact methyltransferase domain.  Hence, targeting the non-PRC2 function of EZH2 may have therapeutic efficacy for treating metastatic, hormone-refractory prostate cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoOB2b-7A-UaLVg90H21EOLACvtfcHk0liHRQqjvG45iw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVSktLfF&md5=98399f2e6a4c9b07121059772650d2d3</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1126%2Fscience.1227604&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1227604%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DK.%26aulast%3DWu%26aufirst%3DZ.%2BJ.%26aulast%3DGroner%26aufirst%3DA.%2BC.%26aulast%3DHe%26aufirst%3DH.%2BH.%26aulast%3DCai%26aufirst%3DC.%26aulast%3DLis%26aufirst%3DR.%2BT.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DStack%26aufirst%3DE.%2BC.%26aulast%3DLoda%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DT.%26aulast%3DXu%26aufirst%3DH.%26aulast%3DCato%26aufirst%3DL.%26aulast%3DThornton%26aufirst%3DJ.%2BE.%26aulast%3DGregory%26aufirst%3DR.%2BI.%26aulast%3DMorrissey%26aufirst%3DC.%26aulast%3DVessella%26aufirst%3DR.%2BL.%26aulast%3DMontironi%26aufirst%3DR.%26aulast%3DMagi-Galluzzi%26aufirst%3DC.%26aulast%3DKantoff%26aufirst%3DP.%2BW.%26aulast%3DBalk%26aufirst%3DS.%2BP.%26aulast%3DLiu%26aufirst%3DX.%2BS.%26aulast%3DBrown%26aufirst%3DM.%26atitle%3DEZH2%2520oncogenic%2520activity%2520in%2520castration-resistant%2520prostate%2520cancer%2520cells%2520is%2520Polycomb-independent%26jtitle%3DScience%26date%3D2012%26volume%3D338%26spage%3D1465%26epage%3D1469%26doi%3D10.1126%2Fscience.1227604" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, C. W.</span></span> <span> </span><span class="NLM_article-title">Targeting EZH2 in cancer</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">128</span>– <span class="NLM_lpage">134</span>, <span class="refDoi"> DOI: 10.1038/nm.4036</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1038%2Fnm.4036" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=26845405" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC28Xit1Wktr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2016&pages=128-134&author=K.+H.+Kimauthor=C.+W.+Roberts&title=Targeting+EZH2+in+cancer&doi=10.1038%2Fnm.4036"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting EZH2 in cancer</span></div><div class="casAuthors">Kim, Kimberly H.; Roberts, Charles W. M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">128-134</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Recent genomic studies have resulted in an emerging understanding of the role of chromatin regulators in the development of cancer.  EZH2, a histone Me transferase subunit of a Polycomb repressor complex, is recurrently mutated in several forms of cancer and is highly expressed in numerous others.  Notably, both gain-of-function and loss-of-function mutations occur in cancers but are assocd. with distinct cancer types.  Here we review the spectrum of EZH2-assocd. mutations, discuss the mechanisms underlying EZH2 function, and synthesize a unifying perspective that the promotion of cancer arises from disruption of the role of EZH2 as a master regulator of transcription.  We further discuss EZH2 inhibitors that are now showing early signs of promise in clin. trials and also addnl. strategies to combat roles of EZH2 in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2UJCv3jp9-rVg90H21EOLACvtfcHk0liHRQqjvG45iw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xit1Wktr4%253D&md5=757ae5f38dc560920f66ced2c818e9e1</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnm.4036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.4036%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DK.%2BH.%26aulast%3DRoberts%26aufirst%3DC.%2BW.%26atitle%3DTargeting%2520EZH2%2520in%2520cancer%26jtitle%3DNat.%2520Med.%26date%3D2016%26volume%3D22%26spage%3D128%26epage%3D134%26doi%3D10.1038%2Fnm.4036" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bonasio, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reinberg, D.</span></span> <span> </span><span class="NLM_article-title">Molecular signals of epigenetic states</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>330</i></span>,  <span class="NLM_fpage">612</span>– <span class="NLM_lpage">616</span>, <span class="refDoi"> DOI: 10.1126/science.1191078</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1126%2Fscience.1191078" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=21030644" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlaqt7bE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=330&publication_year=2010&pages=612-616&author=R.+Bonasioauthor=S.+Tuauthor=D.+Reinberg&title=Molecular+signals+of+epigenetic+states&doi=10.1126%2Fscience.1191078"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Signals of Epigenetic States</span></div><div class="casAuthors">Bonasio, Roberto; Tu, Shengjiang; Reinberg, Danny</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">330</span>
        (<span class="NLM_cas:issue">6004</span>),
    <span class="NLM_cas:pages">612-616</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A review.  Epigenetic signals are responsible for the establishment, maintenance, and reversal of metastable transcriptional states that are fundamental for the cell's ability to "remember" past events, such as changes in the external environment or developmental cues.  Complex epigenetic states are orchestrated by several converging and reinforcing signals, including transcription factors, noncoding RNAs, DNA methylation, and histone modifications.  Although all of these pathways modulate transcription from chromatin in vivo, the mechanisms by which epigenetic information is transmitted through cell division remain unclear.  Because epigenetic states are metastable and change in response to the appropriate signals, a deeper understanding of their mol. framework will allow us to tackle the dysregulation of epigenetics in disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwvUAbah6n1LVg90H21EOLACvtfcHk0ljS-KF3ctkd3g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlaqt7bE&md5=85a662dbd8372bebbc8255f8db6dad0b</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1126%2Fscience.1191078&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1191078%26sid%3Dliteratum%253Aachs%26aulast%3DBonasio%26aufirst%3DR.%26aulast%3DTu%26aufirst%3DS.%26aulast%3DReinberg%26aufirst%3DD.%26atitle%3DMolecular%2520signals%2520of%2520epigenetic%2520states%26jtitle%3DScience%26date%3D2010%26volume%3D330%26spage%3D612%26epage%3D616%26doi%3D10.1126%2Fscience.1191078" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bates, S. E.</span></span> <span> </span><span class="NLM_article-title">Epigenetic Therapies for Cancer</span>. <i>N. Engl. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>383</i></span>,  <span class="NLM_fpage">650</span>– <span class="NLM_lpage">663</span>, <span class="refDoi"> DOI: 10.1056/NEJMra1805035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1056%2FNEJMra1805035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=32786190" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhs1SltrbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=383&publication_year=2020&pages=650-663&author=S.+E.+Bates&title=Epigenetic+Therapies+for+Cancer&doi=10.1056%2FNEJMra1805035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Epigenetic therapies for cancer</span></div><div class="casAuthors">Bates, Susan E.</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">383</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">650-663</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">1533-4406</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">A review.  Chromatin is one of the earliest identified targets for cancer therapeutics.  A more precise understanding of the complexity of chromatin and its role in oncogenesis began to emerge when sequencing of the cancer genome revealed mutations in numerous genes encoding proteins that regulate chromatin.  In many cases, these mutations proved to be crit. in maintaining the malignant process, an observation that led to new therapeutics.  This review summarizes approved agents and their clin. activity, describes therapies in development, and delineates challenges in the field of epigenetics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiRHFKx786ibVg90H21EOLACvtfcHk0ljS-KF3ctkd3g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhs1SltrbL&md5=3cd39b4cc794c12b3bd45ac22a909db6</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1056%2FNEJMra1805035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMra1805035%26sid%3Dliteratum%253Aachs%26aulast%3DBates%26aufirst%3DS.%2BE.%26atitle%3DEpigenetic%2520Therapies%2520for%2520Cancer%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2020%26volume%3D383%26spage%3D650%26epage%3D663%26doi%3D10.1056%2FNEJMra1805035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kohashi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oda, Y.</span></span> <span> </span><span class="NLM_article-title">Oncogenic roles of SMARCB1/INI1 and its deficient tumors</span>. <i>Cancer Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>108</i></span>,  <span class="NLM_fpage">547</span>– <span class="NLM_lpage">552</span>, <span class="refDoi"> DOI: 10.1111/cas.13173</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1111%2Fcas.13173" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=28109176" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC2sXlvFClsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2017&pages=547-552&author=K.+Kohashiauthor=Y.+Oda&title=Oncogenic+roles+of+SMARCB1%2FINI1+and+its+deficient+tumors&doi=10.1111%2Fcas.13173"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Oncogenic roles of SMARCB1/INI1 and its deficient tumors</span></div><div class="casAuthors">Kohashi, Kenichi; Oda, Yoshinao</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Science</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">547-552</span>CODEN:
                <span class="NLM_cas:coden">CSACCM</span>;
        ISSN:<span class="NLM_cas:issn">1349-7006</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">SMARCB1/INI1 is one of the core subunit proteins of the ATP-dependent SWI/SNF chromatin remodeling complex, and is identified as a potent and bona fide tumor suppressor.  Interactions have been demonstrated between SMARCB1/INI1 and key proteins in various pathways related to tumor proliferation and progression: the p16-RB pathway, WNT signaling pathway, sonic hedgehog signaling pathway and Polycomb pathway.  Initially, no detectable SMARCB1/INI1 protein expression was found in malignant rhabdoid tumor cells, whereas all other kinds of tumor cells and non-tumorous tissue showed SMARCB1/INI1 protein expression.  Therefore, immunohistochem. testing for the SMARCB1/INI1 antibody has been considered useful in confirming the histol. diagnosis of malignant rhabdoid tumors.  However, recently, aberrant expression of SMARCB1/INI1 has been found in various tumors such as epithelioid sarcomas, schwannomatosis, synovial sarcomas, and so on.  In addn., it has been reported that aberrant expression can be classified into three patterns: complete loss, mosaic expression and reduced expression.  Although the various pathways related to mechanisms of tumorigenesis and tumor proliferation are complexly intertwined, the clarification of these mechanisms may contribute to therapeutic strategies in SMARCB1/INI1-deficient tumors.  In terms of pathol. classifications, SMARCB1/INI1-deficient tumors may be re-classified by genetic backgrounds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbJZuuZkjIIrVg90H21EOLACvtfcHk0ljS-KF3ctkd3g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXlvFClsLo%253D&md5=d9f76311d904561cc4f1547e13b918b4</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1111%2Fcas.13173&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fcas.13173%26sid%3Dliteratum%253Aachs%26aulast%3DKohashi%26aufirst%3DK.%26aulast%3DOda%26aufirst%3DY.%26atitle%3DOncogenic%2520roles%2520of%2520SMARCB1%252FINI1%2520and%2520its%2520deficient%2520tumors%26jtitle%3DCancer%2520Sci.%26date%3D2017%26volume%3D108%26spage%3D547%26epage%3D552%26doi%3D10.1111%2Fcas.13173" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howard, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vazquez, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsherniak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haswell, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walensky, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hahn, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orkin, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, C. W.</span></span> <span> </span><span class="NLM_article-title">SWI/SNF-mutant cancers depend on catalytic and non-catalytic activity of EZH2</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">1491</span>– <span class="NLM_lpage">1496</span>, <span class="refDoi"> DOI: 10.1038/nm.3968</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1038%2Fnm.3968" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=26552009" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslKgt7nK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=1491-1496&author=K.+H.+Kimauthor=W.+Kimauthor=T.+P.+Howardauthor=F.+Vazquezauthor=A.+Tsherniakauthor=J.+N.+Wuauthor=W.+Wangauthor=J.+R.+Haswellauthor=L.+D.+Walenskyauthor=W.+C.+Hahnauthor=S.+H.+Orkinauthor=C.+W.+Roberts&title=SWI%2FSNF-mutant+cancers+depend+on+catalytic+and+non-catalytic+activity+of+EZH2&doi=10.1038%2Fnm.3968"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">SWI/SNF-mutant cancers depend on catalytic and non-catalytic activity of EZH2</span></div><div class="casAuthors">Kim, Kimberly H.; Kim, Woojin; Howard, Thomas P.; Vazquez, Francisca; Tsherniak, Aviad; Wu, Jennifer N.; Wang, Weishan; Haswell, Jeffrey R.; Walensky, Loren D.; Hahn, William C.; Orkin, Stuart H.; Roberts, Charles W. M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1491-1496</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Human cancer genome sequencing has recently revealed that genes that encode subunits of SWI/SNF chromatin remodeling complexes are frequently mutated across a wide variety of cancers, and several subunits of the complex have been shown to have bona fide tumor suppressor activity.  However, whether mutations in SWI/SNF subunits result in shared dependencies is unknown.  Here we show that EZH2, a catalytic subunit of the polycomb repressive complex 2 (PRC2), is essential in all tested cancer cell lines and xenografts harboring mutations of the SWI/SNF subunits ARID1A, PBRM1, and SMARCA4, which are several of the most frequently mutated SWI/SNF subunits in human cancer, but that co-occurrence of a Ras pathway mutation is correlated with abrogation of this dependence.  Notably, we demonstrate that SWI/SNF-mutant cancer cells are primarily dependent on a non-catalytic role of EZH2 in the stabilization of the PRC2 complex, and that they are only partially dependent on EZH2 histone methyltransferase activity.  These results not only reveal a shared dependency of cancers with genetic alterations in SWI/SNF subunits, but also suggest that EZH2 enzymic inhibitors now in clin. development may not fully suppress the oncogenic activity of EZH2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrNjhQKP01U5bVg90H21EOLACvtfcHk0ljS-KF3ctkd3g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslKgt7nK&md5=a30c11fe7ce3c20e982e824105b8d640</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnm.3968&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.3968%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DK.%2BH.%26aulast%3DKim%26aufirst%3DW.%26aulast%3DHoward%26aufirst%3DT.%2BP.%26aulast%3DVazquez%26aufirst%3DF.%26aulast%3DTsherniak%26aufirst%3DA.%26aulast%3DWu%26aufirst%3DJ.%2BN.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DHaswell%26aufirst%3DJ.%2BR.%26aulast%3DWalensky%26aufirst%3DL.%2BD.%26aulast%3DHahn%26aufirst%3DW.%2BC.%26aulast%3DOrkin%26aufirst%3DS.%2BH.%26aulast%3DRoberts%26aufirst%3DC.%2BW.%26atitle%3DSWI%252FSNF-mutant%2520cancers%2520depend%2520on%2520catalytic%2520and%2520non-catalytic%2520activity%2520of%2520EZH2%26jtitle%3DNat.%2520Med.%26date%3D2015%26volume%3D21%26spage%3D1491%26epage%3D1496%26doi%3D10.1038%2Fnm.3968" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bitler, B. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aird, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garipov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amatangelo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kossenkov, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schultz, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shih Ie, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conejo-Garcia, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Speicher, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, R.</span></span> <span> </span><span class="NLM_article-title">Synthetic lethality by targeting EZH2 methyltransferase activity in ARID1A-mutated cancers</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">231</span>– <span class="NLM_lpage">238</span>, <span class="refDoi"> DOI: 10.1038/nm.3799</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1038%2Fnm.3799" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=25686104" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC2MXivVSit7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=231-238&author=B.+G.+Bitlerauthor=K.+M.+Airdauthor=A.+Garipovauthor=H.+Liauthor=M.+Amatangeloauthor=A.+V.+Kossenkovauthor=D.+C.+Schultzauthor=Q.+Liuauthor=M.+Shih+Ieauthor=J.+R.+Conejo-Garciaauthor=D.+W.+Speicherauthor=R.+Zhang&title=Synthetic+lethality+by+targeting+EZH2+methyltransferase+activity+in+ARID1A-mutated+cancers&doi=10.1038%2Fnm.3799"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Synthetic lethality by targeting EZH2 methyltransferase activity in ARID1A-mutated cancers</span></div><div class="casAuthors">Bitler, Benjamin G.; Aird, Katherine M.; Garipov, Azat; Li, Hua; Amatangelo, Michael; Kossenkov, Andrew V.; Schultz, David C.; Liu, Qin; Shih, Ie-Ming; Conejo-Garcia, Jose R.; Speicher, David W.; Zhang, Rugang</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">231-238</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The gene encoding ARID1A, a chromatin remodeler, shows one of the highest mutation rates across many cancer types.  Notably, ARID1A is mutated in over 50% of ovarian clear cell carcinomas, which currently have no effective therapy.  To date, clin. applicable targeted cancer therapy based on ARID1A mutational status has not been described.  Here we show that inhibition of the EZH2 methyltransferase acts in a synthetic lethal manner in ARID1A-mutated ovarian cancer cells and that ARID1A mutational status correlated with response to the EZH2 inhibitor.  We identified PIK3IP1 as a direct target of ARID1A and EZH2 that is upregulated by EZH2 inhibition and contributed to the obsd. synthetic lethality by inhibiting PI3K-AKT signaling.  Importantly, EZH2 inhibition caused regression of ARID1A-mutated ovarian tumors in vivo.  To our knowledge, this is the first data set to demonstrate a synthetic lethality between ARID1A mutation and EZH2 inhibition.  Our data indicate that pharmacol. inhibition of EZH2 represents a novel treatment strategy for cancers involving ARID1A mutations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEiQfgiO8t4LVg90H21EOLACvtfcHk0li6xOnG9IrW0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXivVSit7k%253D&md5=2521906b5e6ef9b7fc51a09ceb1f0c9d</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fnm.3799&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.3799%26sid%3Dliteratum%253Aachs%26aulast%3DBitler%26aufirst%3DB.%2BG.%26aulast%3DAird%26aufirst%3DK.%2BM.%26aulast%3DGaripov%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DAmatangelo%26aufirst%3DM.%26aulast%3DKossenkov%26aufirst%3DA.%2BV.%26aulast%3DSchultz%26aufirst%3DD.%2BC.%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DShih%2BIe%26aufirst%3DM.%26aulast%3DConejo-Garcia%26aufirst%3DJ.%2BR.%26aulast%3DSpeicher%26aufirst%3DD.%2BW.%26aulast%3DZhang%26aufirst%3DR.%26atitle%3DSynthetic%2520lethality%2520by%2520targeting%2520EZH2%2520methyltransferase%2520activity%2520in%2520ARID1A-mutated%2520cancers%26jtitle%3DNat.%2520Med.%26date%3D2015%26volume%3D21%26spage%3D231%26epage%3D238%26doi%3D10.1038%2Fnm.3799" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yamagishi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uchimaru, K.</span></span> <span> </span><span class="NLM_article-title">Targeting EZH2 in cancer therapy</span>. <i>Curr. Opin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">375</span>– <span class="NLM_lpage">381</span>, <span class="refDoi"> DOI: 10.1097/CCO.0000000000000390</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1097%2FCCO.0000000000000390" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=28665819" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht12iu7zO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2017&pages=375-381&author=M.+Yamagishiauthor=K.+Uchimaru&title=Targeting+EZH2+in+cancer+therapy&doi=10.1097%2FCCO.0000000000000390"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting EZH2 in cancer therapy</span></div><div class="casAuthors">Yamagishi, Makoto; Uchimaru, Kaoru</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Oncology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">375-381</span>CODEN:
                <span class="NLM_cas:coden">CUOOE8</span>;
        ISSN:<span class="NLM_cas:issn">1040-8746</span>.
    
            (<span class="NLM_cas:orgname">Lippincott Williams & Wilkins</span>)
        </div><div class="casAbstract">A review.  The present review introduces recent outstanding progress pertaining to Enhancer of zeste homolog 2 (EZH2), esp. regarding its mode of action as a master regulator of chromatin, and provides mol.-based evidence for targeting EZH2 in cancer therapy.  We discuss the active development of small mols. targeting the enzymic activity of EZH2/polycomb repressive complex 2 (PRC2).  Recent findings: Genetic, transcriptional, and posttranscriptional dysregulation of EZH2 is frequently obsd. in many cancer types.  EZH2 promotes tumorigenesis by altering the expression of numerous tumor suppressor genes.  Furthermore, the executive mol. processes initiated by EZH2, such as NF-B activation, microRNA silencing, tumor immune evasion, and noncanonical transcription regulation, appear to be the fundamental characteristics of each cancer.  Systematic investigations have suggested coordinated regulation of the cancer epigenome wherein antagonistic complexes of both polycomb and SWI/SNF are involved.  Frequent loss-of-function mutations in epigenetic factors, such as ARID1A, SMARCA4, SMARCB1, BAP1, and KDM6A, are likely to elicit the EZH2/PRC2-addicted situation.  Our comprehensive understanding encourages the development of advanced strategies for the appropriate manipulation of the cancer epigenome.  Moreover, a couple of small mols. that can effectively inhibit the enzymic activity of EZH2/PRC2 have been translated into early-phase clin. trials.  The EZH2-mediated epigenome and subsequent transcriptome define cellular identity.  Effective and specific strategies for the manipulation of EZH2/PRC2 may lead to the development of more precise cancer medicines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqNgCSp7SIPsLVg90H21EOLACvtfcHk0li6xOnG9IrW0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht12iu7zO&md5=a0089ba316841e0a4a3b67d3038465b9</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1097%2FCCO.0000000000000390&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252FCCO.0000000000000390%26sid%3Dliteratum%253Aachs%26aulast%3DYamagishi%26aufirst%3DM.%26aulast%3DUchimaru%26aufirst%3DK.%26atitle%3DTargeting%2520EZH2%2520in%2520cancer%2520therapy%26jtitle%3DCurr.%2520Opin.%2520Oncol.%26date%3D2017%26volume%3D29%26spage%3D375%26epage%3D381%26doi%3D10.1097%2FCCO.0000000000000390" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span> </span><span class="NLM_article-title">First EZH2 Inhibitor Approved-for Rare Sarcoma</span>.  <i>Cancer Discovery</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume">10</span>,  <span class="NLM_fpage">333</span>  <span class="NLM_lpage">334</span>. <span class="refDoi"> DOI: 10.1158/2159-8290.CD-NB2020-006</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1158%2F2159-8290.CD-NB2020-006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=32041739" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=First+EZH2+Inhibitor+Approved-for+Rare+Sarcoma.+Cancer+Discovery+2020%2C+10%2C+333+334.+10.1158%2F2159-8290.CD-NB2020-006."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1158%2F2159-8290.CD-NB2020-006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F2159-8290.CD-NB2020-006%26sid%3Dliteratum%253Aachs%26atitle%3DFirst%2520EZH2%2520Inhibitor%2520Approved-for%2520Rare%2520Sarcoma%26jtitle%3DCancer%2520Discovery%26date%3D2020%26volume%3D10%26spage%3D333%26epage%3D334%26doi%3D10.1158%2F2159-8290.CD-NB2020-006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Knutson, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wigle, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warholic, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sneeringer, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allain, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klaus, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sacks, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raimondi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majer, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olhava, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chesworth, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moyer, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richon, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keilhack, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollock, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuntz, K. W.</span></span> <span> </span><span class="NLM_article-title">A selective inhibitor of EZH2 blocks H3K27 methylation and kills mutant lymphoma cells</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">890</span>– <span class="NLM_lpage">896</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1084</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1038%2Fnchembio.1084" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=23023262" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVaqtrvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=890-896&author=S.+K.+Knutsonauthor=T.+J.+Wigleauthor=N.+M.+Warholicauthor=C.+J.+Sneeringerauthor=C.+J.+Allainauthor=C.+R.+Klausauthor=J.+D.+Sacksauthor=A.+Raimondiauthor=C.+R.+Majerauthor=J.+Songauthor=M.+P.+Scottauthor=L.+Jinauthor=J.+J.+Smithauthor=E.+J.+Olhavaauthor=R.+Chesworthauthor=M.+P.+Moyerauthor=V.+M.+Richonauthor=R.+A.+Copelandauthor=H.+Keilhackauthor=R.+M.+Pollockauthor=K.+W.+Kuntz&title=A+selective+inhibitor+of+EZH2+blocks+H3K27+methylation+and+kills+mutant+lymphoma+cells&doi=10.1038%2Fnchembio.1084"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">A selective inhibitor of EZH2 blocks H3K27 methylation and kills mutant lymphoma cells</span></div><div class="casAuthors">Knutson, Sarah K.; Wigle, Tim J.; Warholic, Natalie M.; Sneeringer, Christopher J.; Allain, Christina J.; Klaus, Christine R.; Sacks, Joelle D.; Raimondi, Alejandra; Majer, Christina R.; Song, Jeffrey; Scott, Margaret Porter; Jin, Lei; Smith, Jesse J.; Olhava, Edward J.; Chesworth, Richard; Moyer, Mikel P.; Richon, Victoria M.; Copeland, Robert A.; Keilhack, Heike; Pollock, Roy M.; Kuntz, Kevin W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">890-896</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">EZH2 catalyzes trimethylation of histone H3 lysine 27 (H3K27).  Point mutations of EZH2 at Tyr 641 and Ala 677 occur in subpopulations of non-Hodgkin's lymphoma, where they drive H3K27 hypertrimethylation.  Here the authors report the discovery of EPZ005687, a potent inhibitor of EZH2 (Ki of 24 nM).  EPZ005687 has greater than 500-fold selectivity against 15 other protein methyltransferases and has 50-fold selectivity against the closely related enzyme EZH1.  The compd. reduces H3K27 methylation in various lymphoma cells; this translates into apoptotic cell killing in heterozygous Tyr 641 or Ala 677 mutant cells, with minimal effects on the proliferation of wild-type cells.  These data suggest that genetic alteration of EZH2 (for example, mutations at Tyr 641 or Ala 677) results in a crit. dependency on enzymic activity for proliferation (i.e., the equiv. of oncogene addiction), thus portending the clin. use of EZH2 inhibitors for cancers in which EZH2 is genetically altered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohlduVbTgWJrVg90H21EOLACvtfcHk0lgExuwNR7jNew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVaqtrvP&md5=a9ad7a41505e3fd4aa087d9f5dae8825</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1084&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1084%26sid%3Dliteratum%253Aachs%26aulast%3DKnutson%26aufirst%3DS.%2BK.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DWarholic%26aufirst%3DN.%2BM.%26aulast%3DSneeringer%26aufirst%3DC.%2BJ.%26aulast%3DAllain%26aufirst%3DC.%2BJ.%26aulast%3DKlaus%26aufirst%3DC.%2BR.%26aulast%3DSacks%26aufirst%3DJ.%2BD.%26aulast%3DRaimondi%26aufirst%3DA.%26aulast%3DMajer%26aufirst%3DC.%2BR.%26aulast%3DSong%26aufirst%3DJ.%26aulast%3DScott%26aufirst%3DM.%2BP.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DOlhava%26aufirst%3DE.%2BJ.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DRichon%26aufirst%3DV.%2BM.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DKeilhack%26aufirst%3DH.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26aulast%3DKuntz%26aufirst%3DK.%2BW.%26atitle%3DA%2520selective%2520inhibitor%2520of%2520EZH2%2520blocks%2520H3K27%2520methylation%2520and%2520kills%2520mutant%2520lymphoma%2520cells%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2012%26volume%3D8%26spage%3D890%26epage%3D896%26doi%3D10.1038%2Fnchembio.1084" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCabe, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ott, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ganji, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korenchuk, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Aller, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graves, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Della Pietra, A.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diaz, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LaFrance, L. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mellinger, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duquenne, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kruger, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McHugh, C. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brandt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhanak, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verma, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tummino, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Creasy, C. L.</span></span> <span> </span><span class="NLM_article-title">EZH2 inhibition as a therapeutic strategy for lymphoma with EZH2-activating mutations</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>492</i></span>,  <span class="NLM_fpage">108</span>– <span class="NLM_lpage">112</span>, <span class="refDoi"> DOI: 10.1038/nature11606</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1038%2Fnature11606" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=23051747" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVyktbjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=492&publication_year=2012&pages=108-112&author=M.+T.+McCabeauthor=H.+M.+Ottauthor=G.+Ganjiauthor=S.+Korenchukauthor=C.+Thompsonauthor=G.+S.+Van+Allerauthor=Y.+Liuauthor=A.+P.+Gravesauthor=A.+Della+Pietraauthor=E.+Diazauthor=L.+V.+LaFranceauthor=M.+Mellingerauthor=C.+Duquenneauthor=X.+Tianauthor=R.+G.+Krugerauthor=C.+F.+McHughauthor=M.+Brandtauthor=W.+H.+Millerauthor=D.+Dhanakauthor=S.+K.+Vermaauthor=P.+J.+Tumminoauthor=C.+L.+Creasy&title=EZH2+inhibition+as+a+therapeutic+strategy+for+lymphoma+with+EZH2-activating+mutations&doi=10.1038%2Fnature11606"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">EZH2 inhibition as a therapeutic strategy for lymphoma with EZH2-activating mutations</span></div><div class="casAuthors">McCabe, Michael T.; Ott, Heidi M.; Ganji, Gopinath; Korenchuk, Susan; Thompson, Christine; Van Aller, Glenn S.; Liu, Yan; Graves, Alan P.; Della Pietra, Anthony , III; Diaz, Elsie; LaFrance, Louis V.; Mellinger, Mark; Duquenne, Celine; Tian, Xinrong; Kruger, Ryan G.; McHugh, Charles F.; Brandt, Martin; Miller, William H.; Dhanak, Dashyant; Verma, Sharad K.; Tummino, Peter J.; Creasy, Caretha L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">492</span>
        (<span class="NLM_cas:issue">7427</span>),
    <span class="NLM_cas:pages">108-112</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">In eukaryotes, post-translational modification of histones is crit. for regulation of chromatin structure and gene expression.  EZH2 is the catalytic subunit of the polycomb repressive complex 2 (PRC2) and is involved in repressing gene expression through methylation of histone H3 on lysine 27 (H3K27).  EZH2 overexpression is implicated in tumorigenesis and correlates with poor prognosis in several tumor types.  Addnl., somatic heterozygous mutations of Y641 and A677 residues within the catalytic SET domain of EZH2 occur in diffuse large B-cell lymphoma (DLBCL) and follicular lymphoma.  The Y641 residue is the most frequently mutated residue, with up to 22% of germinal center B-cell DLBCL and follicular lymphoma harbouring mutations at this site.  These lymphomas have increased H3K27 tri-methylation (H3K27me3) owing to altered substrate preferences of the mutant enzymes.  However, it is unknown whether specific, direct inhibition of EZH2 methyltransferase activity will be effective in treating EZH2 mutant lymphomas.  Here we demonstrate that GSK126, a potent, highly selective, S-adenosyl-methionine-competitive, small-mol. inhibitor of EZH2 methyltransferase activity, decreases global H3K27me3 levels and reactivates silenced PRC2 target genes.  GSK126 effectively inhibits the proliferation of EZH2 mutant DLBCL cell lines and markedly inhibits the growth of EZH2 mutant DLBCL xenografts in mice.  Together, these data demonstrate that pharmacol. inhibition of EZH2 activity may provide a promising treatment for EZH2 mutant lymphoma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxw5cLmW3Lc7Vg90H21EOLACvtfcHk0lhRoaxHcc7g9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVyktbjL&md5=a1dff99556de004c48aae88bc45dffab</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fnature11606&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11606%26sid%3Dliteratum%253Aachs%26aulast%3DMcCabe%26aufirst%3DM.%2BT.%26aulast%3DOtt%26aufirst%3DH.%2BM.%26aulast%3DGanji%26aufirst%3DG.%26aulast%3DKorenchuk%26aufirst%3DS.%26aulast%3DThompson%26aufirst%3DC.%26aulast%3DVan%2BAller%26aufirst%3DG.%2BS.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DGraves%26aufirst%3DA.%2BP.%26aulast%3DDella%2BPietra%26aufirst%3DA.%26aulast%3DDiaz%26aufirst%3DE.%26aulast%3DLaFrance%26aufirst%3DL.%2BV.%26aulast%3DMellinger%26aufirst%3DM.%26aulast%3DDuquenne%26aufirst%3DC.%26aulast%3DTian%26aufirst%3DX.%26aulast%3DKruger%26aufirst%3DR.%2BG.%26aulast%3DMcHugh%26aufirst%3DC.%2BF.%26aulast%3DBrandt%26aufirst%3DM.%26aulast%3DMiller%26aufirst%3DW.%2BH.%26aulast%3DDhanak%26aufirst%3DD.%26aulast%3DVerma%26aufirst%3DS.%2BK.%26aulast%3DTummino%26aufirst%3DP.%2BJ.%26aulast%3DCreasy%26aufirst%3DC.%2BL.%26atitle%3DEZH2%2520inhibition%2520as%2520a%2520therapeutic%2520strategy%2520for%2520lymphoma%2520with%2520EZH2-activating%2520mutations%26jtitle%3DNature%26date%3D2012%26volume%3D492%26spage%3D108%26epage%3D112%26doi%3D10.1038%2Fnature11606" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bate-Eya, L. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gierman, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ebus, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koster, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caron, H. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Versteeg, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dolman, M. E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molenaar, J. J.</span></span> <span> </span><span class="NLM_article-title">Enhancer of zeste homologue 2 plays an important role in neuroblastoma cell survival independent of its histone methyltransferase activity</span>. <i>Eur. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>75</i></span>,  <span class="NLM_fpage">63</span>– <span class="NLM_lpage">72</span>, <span class="refDoi"> DOI: 10.1016/j.ejca.2016.12.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1016%2Fj.ejca.2016.12.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=28214660" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC2sXit1eru70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2017&pages=63-72&author=L.+T.+Bate-Eyaauthor=H.+J.+Giermanauthor=M.+E.+Ebusauthor=J.+Kosterauthor=H.+N.+Caronauthor=R.+Versteegauthor=M.+E.+M.+Dolmanauthor=J.+J.+Molenaar&title=Enhancer+of+zeste+homologue+2+plays+an+important+role+in+neuroblastoma+cell+survival+independent+of+its+histone+methyltransferase+activity&doi=10.1016%2Fj.ejca.2016.12.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Enhancer of zeste homologue 2 plays an important role in neuroblastoma cell survival independent of its histone methyltransferase activity</span></div><div class="casAuthors">Bate-Eya, Laurel T.; Gierman, Hinco J.; Ebus, Marli E.; Koster, Jan; Caron, Huib N.; Versteeg, Rogier; Dolman, M. Emmy M.; Molenaar, Jan J.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Cancer</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">75</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">63-72</span>CODEN:
                <span class="NLM_cas:coden">EJCAEL</span>;
        ISSN:<span class="NLM_cas:issn">0959-8049</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Neuroblastoma is predominantly characterized by chromosomal rearrangements.  Next to V-Myc Avian Myelocytomatosis Viral Oncogene Neuroblastoma Derived Homolog (MYCN) amplification, chromosome 7 and 17q gains are frequently obsd.  We identified a neuroblastoma patient with a regional 7q36 gain, encompassing the enhancer of zeste homolog 2 (EZH2) gene.  EZH2 is the histone methyltransferase of lysine 27 of histone H3 (H3K27me3) that forms the catalytic subunit of the polycomb repressive complex 2.  H3K27me3 is commonly assocd. with the silencing of genes involved in cellular processes such as cell cycle regulation, cellular differentiation and cancer.  High EZH2 expression correlated with poor prognosis and overall survival independent of MYCN amplification status.  Unexpectedly, treatment of 3 EZH2-high expressing neuroblastoma cell lines (IMR32, CHP134 and NMB), with EZH2-specific inhibitors (GSK126 and EPZ6438) resulted in only a slight G1 arrest, despite max. histone methyltransferase activity inhibition.  Furthermore, colony formation in cell lines treated with the inhibitors was reduced only at concns. much higher than necessary for complete inhibition of EZH2 histone methyltransferase activity.  Knockdown of the complete protein with three independent shRNAs resulted in a strong apoptotic response and decreased cyclin D1 levels.  This apoptotic response could be rescued by overexpressing EZH2ΔSET, a truncated form of wild-type EZH2 lacking the SET transactivation domain necessary for histone methyltransferase activity.  Our findings suggest that high EZH2 expression, at least in neuroblastoma, has a survival function independent of its methyltransferase activity.  This important finding highlights the need for studies on EZH2 beyond its methyltransferase function and the requirement for compds. that will target EZH2 as a complete protein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPeYIEEK1oFLVg90H21EOLACvtfcHk0lhRoaxHcc7g9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXit1eru70%253D&md5=65cbff2c51dc7c1af9d96cbeaf12cf11</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.ejca.2016.12.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejca.2016.12.019%26sid%3Dliteratum%253Aachs%26aulast%3DBate-Eya%26aufirst%3DL.%2BT.%26aulast%3DGierman%26aufirst%3DH.%2BJ.%26aulast%3DEbus%26aufirst%3DM.%2BE.%26aulast%3DKoster%26aufirst%3DJ.%26aulast%3DCaron%26aufirst%3DH.%2BN.%26aulast%3DVersteeg%26aufirst%3DR.%26aulast%3DDolman%26aufirst%3DM.%2BE.%2BM.%26aulast%3DMolenaar%26aufirst%3DJ.%2BJ.%26atitle%3DEnhancer%2520of%2520zeste%2520homologue%25202%2520plays%2520an%2520important%2520role%2520in%2520neuroblastoma%2520cell%2520survival%2520independent%2520of%2520its%2520histone%2520methyltransferase%2520activity%26jtitle%3DEur.%2520J.%2520Cancer%26date%3D2017%26volume%3D75%26spage%3D63%26epage%3D72%26doi%3D10.1016%2Fj.ejca.2016.12.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fong, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Licht, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span> <span> </span><span class="NLM_article-title">Polycomb- and Methylation-Independent Roles of EZH2 as a Transcription Activator</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">2808</span>– <span class="NLM_lpage">2820e4</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2018.11.035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1016%2Fj.celrep.2018.11.035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=30517868" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisVCit7bL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=2808-2820e4&author=J.+Kimauthor=Y.+Leeauthor=X.+Luauthor=B.+Songauthor=K.+W.+Fongauthor=Q.+Caoauthor=J.+D.+Lichtauthor=J.+C.+Zhaoauthor=J.+Yu&title=Polycomb-+and+Methylation-Independent+Roles+of+EZH2+as+a+Transcription+Activator&doi=10.1016%2Fj.celrep.2018.11.035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Polycomb- and Methylation-Independent Roles of EZH2 as a Transcription Activator</span></div><div class="casAuthors">Kim, Jung; Lee, Yongik; Lu, Xiaodong; Song, Bing; Fong, Ka-Wing; Cao, Qi; Licht, Jonathan D.; Zhao, Jonathan C.; Yu, Jindan</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2808-2820.e4</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Enhancer of Zeste 2 (EZH2) is the enzymic subunit of Polycomb Repressive Complex 2 (PRC2), which catalyzes histone H3 lysine 27 trimethylation (H3K27me3) at target promoters for gene silencing.  Here, we report that EZH2 activates androgen receptor (AR) gene transcription through direct occupancy at its promoter.  Importantly, this activating role of EZH2 is independent of PRC2 and its methyltransferase activities.  Genome-wide assays revealed extensive EZH2 occupancy at promoters marked by either H3K27ac or H3K27me3, leading to gene activation or repression, resp.  Last, we demonstrate enhanced efficacy of enzymic EZH2 inhibitors when used in combination with AR antagonists in blocking the dual roles of EZH2 and suppressing prostate cancer progression in vitro and in vivo.  Taken together, our study reports EZH2 as a transcriptional activator, a key target of which is AR, and suggests a drug-combinatory approach to treat advanced prostate cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGogFd6y_QesoLVg90H21EOLACvtfcHk0lhRoaxHcc7g9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisVCit7bL&md5=795cd5933915f46a31e298075abaa247</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2018.11.035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2018.11.035%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DY.%26aulast%3DLu%26aufirst%3DX.%26aulast%3DSong%26aufirst%3DB.%26aulast%3DFong%26aufirst%3DK.%2BW.%26aulast%3DCao%26aufirst%3DQ.%26aulast%3DLicht%26aufirst%3DJ.%2BD.%26aulast%3DZhao%26aufirst%3DJ.%2BC.%26aulast%3DYu%26aufirst%3DJ.%26atitle%3DPolycomb-%2520and%2520Methylation-Independent%2520Roles%2520of%2520EZH2%2520as%2520a%2520Transcription%2520Activator%26jtitle%3DCell%2520Rep.%26date%3D2018%26volume%3D25%26spage%3D2808%26epage%3D2820e4%26doi%3D10.1016%2Fj.celrep.2018.11.035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, X. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cong, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geng, M.</span></span> <span> </span><span class="NLM_article-title">Targeting Epigenetic Crosstalk as a Therapeutic Strategy for EZH2-Aberrant Solid Tumors</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>175</i></span>,  <span class="NLM_fpage">186</span>– <span class="NLM_lpage">199.e19</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2018.08.058</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1016%2Fj.cell.2018.08.058" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=30220457" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslahs7%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=175&publication_year=2018&pages=186-199.e19&author=X.+Huangauthor=J.+Yanauthor=M.+Zhangauthor=Y.+Wangauthor=Y.+Chenauthor=X.+Fuauthor=R.+Weiauthor=X.+L.+Zhengauthor=Z.+Liuauthor=X.+Zhangauthor=H.+Yangauthor=B.+Haoauthor=Y.+Y.+Shenauthor=Y.+Suauthor=X.+Congauthor=M.+Huangauthor=M.+Tanauthor=J.+Dingauthor=M.+Geng&title=Targeting+Epigenetic+Crosstalk+as+a+Therapeutic+Strategy+for+EZH2-Aberrant+Solid+Tumors&doi=10.1016%2Fj.cell.2018.08.058"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Epigenetic Crosstalk as a Therapeutic Strategy for EZH2-Aberrant Solid Tumors</span></div><div class="casAuthors">Huang, Xun; Yan, Juan; Zhang, Min; Wang, Yafang; Chen, Yi; Fu, Xuhong; Wei, Rongrui; Zheng, Xing-ling; Liu, Zhiwei; Zhang, Xiong; Yang, Hong; Hao, Bingbing; Shen, Yan-yan; Su, Yi; Cong, Xiaoji; Huang, Min; Tan, Minjia; Ding, Jian; Geng, Meiyu</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">175</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">186-199.e19</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Mutations or aberrant upregulation of EZH2 occur frequently in human cancers, yet clin. benefits of EZH2 inhibitor (EZH2i) remain unsatisfactory and limited to certain hematol. malignancies.  We profile global posttranslational histone modification changes across a large panel of cancer cell lines with various sensitivities to EZH2i.  We report here oncogenic transcriptional reprogramming mediated by MLL1's interaction with the p300/CBP complex, which directs H3K27me loss to reciprocal H3K27ac gain and restricts EZH2i response.  Concurrent inhibition of H3K27me and H3K27ac results in transcriptional repression and MAPK pathway dependency in cancer subsets.  In preclin. models encompassing a broad spectrum of EZH2-aberrant solid tumors, a combination of EZH2 and BRD4 inhibitors, or a triple-combination including MAPK inhibition display robust efficacy with very tolerable toxicity.  Our results suggest an attractive precision treatment strategy for EZH2-aberrant tumors on the basis of tumor-intrinsic MLL1 expression and concurrent inhibition of epigenetic crosstalk and feedback MAPK activation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgroiKQYI5RLVg90H21EOLACvtfcHk0lgcOAranxkBiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslahs7%252FM&md5=1879411de33f088e3c197d6175e02e18</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2018.08.058&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2018.08.058%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DYan%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DFu%26aufirst%3DX.%26aulast%3DWei%26aufirst%3DR.%26aulast%3DZheng%26aufirst%3DX.%2BL.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DHao%26aufirst%3DB.%26aulast%3DShen%26aufirst%3DY.%2BY.%26aulast%3DSu%26aufirst%3DY.%26aulast%3DCong%26aufirst%3DX.%26aulast%3DHuang%26aufirst%3DM.%26aulast%3DTan%26aufirst%3DM.%26aulast%3DDing%26aufirst%3DJ.%26aulast%3DGeng%26aufirst%3DM.%26atitle%3DTargeting%2520Epigenetic%2520Crosstalk%2520as%2520a%2520Therapeutic%2520Strategy%2520for%2520EZH2-Aberrant%2520Solid%2520Tumors%26jtitle%3DCell%26date%3D2018%26volume%3D175%26spage%3D186%26epage%3D199.e19%26doi%3D10.1016%2Fj.cell.2018.08.058" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gibaja, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harari, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korn, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruddy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saenz-Vash, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhai, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rejtar, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paris, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hassan, A. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, H. M.</span></span> <span> </span><span class="NLM_article-title">Development of secondary mutations in wild-type and mutant EZH2 alleles cooperates to confer resistance to EZH2 inhibitors</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">558</span>– <span class="NLM_lpage">566</span>, <span class="refDoi"> DOI: 10.1038/onc.2015.114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1038%2Fonc.2015.114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=25893294" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFynt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2016&pages=558-566&author=V.+Gibajaauthor=F.+Shenauthor=J.+Harariauthor=J.+Kornauthor=D.+Ruddyauthor=V.+Saenz-Vashauthor=H.+Zhaiauthor=T.+Rejtarauthor=C.+G.+Parisauthor=Z.+Yuauthor=M.+Liraauthor=D.+Kingauthor=W.+Qiauthor=N.+Keenauthor=A.+Q.+Hassanauthor=H.+M.+Chan&title=Development+of+secondary+mutations+in+wild-type+and+mutant+EZH2+alleles+cooperates+to+confer+resistance+to+EZH2+inhibitors&doi=10.1038%2Fonc.2015.114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Development of secondary mutations in wild-type and mutant EZH2 alleles cooperates to confer resistance to EZH2 inhibitors</span></div><div class="casAuthors">Gibaja, V.; Shen, F.; Harari, J.; Korn, J.; Ruddy, D.; Saenz-Vash, V.; Zhai, H.; Rejtar, T.; Paris, C. G.; Yu, Z.; Lira, M.; King, D.; Qi, W.; Keen, N.; Hassan, A. Q.; Chan, H. M.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">558-566</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The histone methyltransferase Enhancer of Zeste Homolog 2 (EZH2) is frequently dysregulated in cancers, and gain-of-function (GOF) EZH2 mutations have been identified in non-Hodgkin lymphomas.  Small-mol. inhibitors against EZH2 demonstrated anti-tumor activity in EZH2-mutated lymphomas and entered clin. trials.  Here, we developed models of acquired resistance to EZH2 inhibitor EI1 with EZH2-mutated lymphoma cells.  Resistance was generated by secondary mutations in both wild-type (WT) and GOF Y641N EZH2 alleles.  These EZH2 mutants retained the substrate specificity of their predecessor complexes but became refractory to biochem. inhibition by EZH2 inhibitors.  Resistant cells were able to maintain a high level of H3K27Me3 in the presence of inhibitors.  Interestingly, mutation of EZH2 WT alone generated an intermediate resistance phenotype, which is consistent with a previously proposed model of cooperation between EZH2 WT and Y641N mutants to promote tumorigenesis.  In addn., the findings presented here have implications for the clin. translation of EZH2 inhibitors and underscore the need to develop novel EZH2 inhibitors to target potential resistance emerging in clin. settings.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUaTpSYCtEf7Vg90H21EOLACvtfcHk0lgcOAranxkBiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFynt74%253D&md5=fcfebb482305effcb67f258ce8c589e8</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fonc.2015.114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2015.114%26sid%3Dliteratum%253Aachs%26aulast%3DGibaja%26aufirst%3DV.%26aulast%3DShen%26aufirst%3DF.%26aulast%3DHarari%26aufirst%3DJ.%26aulast%3DKorn%26aufirst%3DJ.%26aulast%3DRuddy%26aufirst%3DD.%26aulast%3DSaenz-Vash%26aufirst%3DV.%26aulast%3DZhai%26aufirst%3DH.%26aulast%3DRejtar%26aufirst%3DT.%26aulast%3DParis%26aufirst%3DC.%2BG.%26aulast%3DYu%26aufirst%3DZ.%26aulast%3DLira%26aufirst%3DM.%26aulast%3DKing%26aufirst%3DD.%26aulast%3DQi%26aufirst%3DW.%26aulast%3DKeen%26aufirst%3DN.%26aulast%3DHassan%26aufirst%3DA.%2BQ.%26aulast%3DChan%26aufirst%3DH.%2BM.%26atitle%3DDevelopment%2520of%2520secondary%2520mutations%2520in%2520wild-type%2520and%2520mutant%2520EZH2%2520alleles%2520cooperates%2520to%2520confer%2520resistance%2520to%2520EZH2%2520inhibitors%26jtitle%3DOncogene%26date%3D2016%26volume%3D35%26spage%3D558%26epage%3D566%26doi%3D10.1038%2Fonc.2015.114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burslem, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Proteolysis-Targeting Chimeras as Therapeutics and Tools for Biological Discovery</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>181</i></span>,  <span class="NLM_fpage">102</span>– <span class="NLM_lpage">114</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2019.11.031</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1016%2Fj.cell.2019.11.031" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=31955850" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BB3cXht1Sitrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=181&publication_year=2020&pages=102-114&author=G.+M.+Burslemauthor=C.+M.+Crews&title=Proteolysis-Targeting+Chimeras+as+Therapeutics+and+Tools+for+Biological+Discovery&doi=10.1016%2Fj.cell.2019.11.031"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Proteolysis-Targeting Chimeras as Therapeutics and Tools for Biological Discovery</span></div><div class="casAuthors">Burslem, George M.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">181</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">102-114</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  New biol. tools provide new techniques to probe fundamental biol. processes.  Here we describe the burgeoning field of proteolysis-targeting chimeras (PROTACs), which are capable of modulating protein concns. at a post-translational level by co-opting the ubiquitin-proteasome system.  We describe the PROTAC technol. and its application to drug discovery and provide examples where PROTACs have enabled novel biol. insights.  Furthermore, we provide a workflow for PROTAC development and use and discuss the benefits and issues assocd. with PROTACs.  Finally, we compare PROTAC-mediated protein-level modulation with other technologies, such as RNAi and genome editing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXgumsEakVWrVg90H21EOLACvtfcHk0lgcOAranxkBiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXht1Sitrk%253D&md5=3010929d2979cf04dc6003e66575f1b0</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2019.11.031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2019.11.031%26sid%3Dliteratum%253Aachs%26aulast%3DBurslem%26aufirst%3DG.%2BM.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DProteolysis-Targeting%2520Chimeras%2520as%2520Therapeutics%2520and%2520Tools%2520for%2520Biological%2520Discovery%26jtitle%3DCell%26date%3D2020%26volume%3D181%26spage%3D102%26epage%3D114%26doi%3D10.1016%2Fj.cell.2019.11.031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altieri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raina, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Hijacking the E3 Ubiquitin Ligase Cereblon to Efficiently Target BRD4</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">755</span>– <span class="NLM_lpage">763</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2015.05.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1016%2Fj.chembiol.2015.05.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=26051217" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVWrtbrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2015&pages=755-763&author=J.+Luauthor=Y.+Qianauthor=M.+Altieriauthor=H.+Dongauthor=J.+Wangauthor=K.+Rainaauthor=J.+Hinesauthor=J.+D.+Winklerauthor=A.+P.+Crewauthor=K.+Colemanauthor=C.+M.+Crews&title=Hijacking+the+E3+Ubiquitin+Ligase+Cereblon+to+Efficiently+Target+BRD4&doi=10.1016%2Fj.chembiol.2015.05.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Hijacking the E3 Ubiquitin Ligase Cereblon to Efficiently Target BRD4</span></div><div class="casAuthors">Lu, Jing; Qian, Yimin; Altieri, Martha; Dong, Hanqing; Wang, Jing; Raina, Kanak; Hines, John; Winkler, James D.; Crew, Andrew P.; Coleman, Kevin; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">755-763</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">BRD4, a bromodomain and extraterminal domain (BET) family member, is an attractive target in multiple pathol. settings, particularly cancer.  While BRD4 inhibitors have shown some promise in MYC-driven malignancies such as Burkitt's lymphoma (BL), we show that BRD4 inhibitors lead to robust BRD4 protein accumulation, which may account for their limited suppression of MYC expression, modest antiproliferative activity, and lack of apoptotic induction.  To address these limitations we designed ARV-825, a hetero-bifunctional PROTAC (Proteolysis Targeting Chimera) that recruits BRD4 to the E3 ubiquitin ligase cereblon, leading to fast, efficient, and prolonged degrdn. of BRD4 in all BL cell lines tested.  Consequently, ARV-825 more effectively suppresses c-MYC levels and downstream signaling than small-mol. BRD4 inhibitors, resulting in more effective cell proliferation inhibition and apoptosis induction in BL.  Our findings provide strong evidence that cereblon-based PROTACs provide a better and more efficient strategy in targeting BRD4 than traditional small-mol. inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6Ol4grCG2RbVg90H21EOLACvtfcHk0ljHSGoCU1jeAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVWrtbrM&md5=a64675e7a0cd006a8737be9730f447e3</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2015.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2015.05.009%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DAltieri%26aufirst%3DM.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DRaina%26aufirst%3DK.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DColeman%26aufirst%3DK.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DHijacking%2520the%2520E3%2520Ubiquitin%2520Ligase%2520Cereblon%2520to%2520Efficiently%2520Target%2520BRD4%26jtitle%3DChem.%2520Biol.%26date%3D2015%26volume%3D22%26spage%3D755%26epage%3D763%26doi%3D10.1016%2Fj.chembiol.2015.05.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Souza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhe-Paganon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">DRUG DEVELOPMENT. Phthalimide conjugation as a strategy for in vivo target protein degradation</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>348</i></span>,  <span class="NLM_fpage">1376</span>– <span class="NLM_lpage">1381</span>, <span class="refDoi"> DOI: 10.1126/science.aab1433</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1126%2Fscience.aab1433" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=25999370" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC2MXpvFansLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=348&publication_year=2015&pages=1376-1381&author=G.+E.+Winterauthor=D.+L.+Buckleyauthor=J.+Paulkauthor=J.+M.+Robertsauthor=A.+Souzaauthor=S.+Dhe-Paganonauthor=J.+E.+Bradner&title=DRUG+DEVELOPMENT.+Phthalimide+conjugation+as+a+strategy+for+in+vivo+target+protein+degradation&doi=10.1126%2Fscience.aab1433"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Phthalimide conjugation as a strategy for in vivo target protein degradation</span></div><div class="casAuthors">Winter, Georg E.; Buckley, Dennis L.; Paulk, Joshiawa; Roberts, Justin M.; Souza, Amanda; Dhe-Paganon, Sirano; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">348</span>
        (<span class="NLM_cas:issue">6241</span>),
    <span class="NLM_cas:pages">1376-1381</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The development of effective pharmacol. inhibitors of multidomain scaffold proteins, notably transcription factors, is a particularly challenging problem.  In part, this is because many small-mol. antagonists disrupt the activity of only one domain in the target protein.  The authors devised a chem. strategy that promotes ligand-dependent target protein degrdn. using as an example the transcriptional coactivator BRD4, a protein crit. for cancer cell growth and survival.  The authors appended a competitive antagonist of BET bromodomains to a phthalimide moiety to hijack the cereblon E3 ubiquitin ligase complex.  The resultant compd., dBET1, induced highly selective cereblon-dependent BET protein degrdn. in vitro and in vivo and delayed leukemia progression in mice.  A second series of probes resulted in selective degrdn. of the cytosolic protein FKBP12.  This chem. strategy for controlling target protein stability may have implications for therapeutically targeting previously intractable proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1ZW4zjRjXi7Vg90H21EOLACvtfcHk0ljHSGoCU1jeAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXpvFansLk%253D&md5=b0277cc95d318d926b96e7a119066e3b</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1126%2Fscience.aab1433&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aab1433%26sid%3Dliteratum%253Aachs%26aulast%3DWinter%26aufirst%3DG.%2BE.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DRoberts%26aufirst%3DJ.%2BM.%26aulast%3DSouza%26aufirst%3DA.%26aulast%3DDhe-Paganon%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DDRUG%2520DEVELOPMENT.%2520Phthalimide%2520conjugation%2520as%2520a%2520strategy%2520for%2520in%2520vivo%2520target%2520protein%2520degradation%26jtitle%3DScience%26date%3D2015%26volume%3D348%26spage%3D1376%26epage%3D1381%26doi%3D10.1126%2Fscience.aab1433" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Han, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Salas, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaswamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delproposto, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of ARD-69 as a Highly Potent Proteolysis Targeting Chimera (PROTAC) Degrader of Androgen Receptor (AR) for the Treatment of Prostate Cancer</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">941</span>– <span class="NLM_lpage">964</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01631</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01631" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC1MXlvFCjtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=941-964&author=X.+Hanauthor=C.+Wangauthor=C.+Qinauthor=W.+Xiangauthor=E.+Fernandez-Salasauthor=C.+Y.+Yangauthor=M.+Wangauthor=L.+Zhaoauthor=T.+Xuauthor=K.+Chinnaswamyauthor=J.+Delpropostoauthor=J.+Stuckeyauthor=S.+Wang&title=Discovery+of+ARD-69+as+a+Highly+Potent+Proteolysis+Targeting+Chimera+%28PROTAC%29+Degrader+of+Androgen+Receptor+%28AR%29+for+the+Treatment+of+Prostate+Cancer&doi=10.1021%2Facs.jmedchem.8b01631"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of ARD-69 as a highly potent proteolysis targeting chimera (PROTAC) degrader of androgen receptor (AR) for the treatment of prostate cancer</span></div><div class="casAuthors">Han, Xin; Wang, Chao; Qin, Chong; Xiang, Weiguo; Fernandez-Salas, Ester; Yang, Chao-Yie; Wang, Mi; Zhao, Lijie; Xu, Tianfeng; Chinnaswamy, Krishnapriya; Delproposto, James; Stuckey, Jeanne; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">941-964</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We report herein the discovery of highly potent PROTAC degraders of androgen receptor (AR), as exemplified by IARD-69.  I induces degrdn. of AR protein in AR-pos. prostate cancer cell lines in a dose- and time-dependent manner.  I achieves DC50 values of 0.86, 0.76, and 10.4 nM in LNCaP, VCaP, and 22Rv1 AR+ prostate cancer cell lines, resp.  I is capable of reducing the AR protein level by >95% in these prostate cancer cell lines and effectively suppressing AR-regulated gene expression.  I potently inhibits cell growth in these AR-pos. prostate cancer cell lines and is >100 times more potent than AR antagonists.  A single dose of I effectively reduces the level of AR protein in xenograft tumor tissue in mice.  Further optimization of I may ultimately lead to a new therapy for AR+, castration-resistant prostate cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTlOD69tfhtbVg90H21EOLACvtfcHk0lg6uzpkGh4s-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXlvFCjtQ%253D%253D&md5=2376573fb1a1e19c517d64c70199700a</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01631&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01631%26sid%3Dliteratum%253Aachs%26aulast%3DHan%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DQin%26aufirst%3DC.%26aulast%3DXiang%26aufirst%3DW.%26aulast%3DFernandez-Salas%26aufirst%3DE.%26aulast%3DYang%26aufirst%3DC.%2BY.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DZhao%26aufirst%3DL.%26aulast%3DXu%26aufirst%3DT.%26aulast%3DChinnaswamy%26aufirst%3DK.%26aulast%3DDelproposto%26aufirst%3DJ.%26aulast%3DStuckey%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520ARD-69%2520as%2520a%2520Highly%2520Potent%2520Proteolysis%2520Targeting%2520Chimera%2520%2528PROTAC%2529%2520Degrader%2520of%2520Androgen%2520Receptor%2520%2528AR%2529%2520for%2520the%2520Treatment%2520of%2520Prostate%2520Cancer%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D941%26epage%3D964%26doi%3D10.1021%2Facs.jmedchem.8b01631" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaswamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meagher, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">A Potent and Selective Small-Molecule Degrader of STAT3 Achieves Complete Tumor Regression In Vivo</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">498</span>– <span class="NLM_lpage">511e17</span>, <span class="refDoi"> DOI: 10.1016/j.ccell.2019.10.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1016%2Fj.ccell.2019.10.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=31715132" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitFCjtrjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2019&pages=498-511e17&author=L.+Baiauthor=H.+Zhouauthor=R.+Xuauthor=Y.+Zhaoauthor=K.+Chinnaswamyauthor=D.+McEachernauthor=J.+Chenauthor=C.+Y.+Yangauthor=Z.+Liuauthor=M.+Wangauthor=L.+Liuauthor=H.+Jiangauthor=B.+Wenauthor=P.+Kumarauthor=J.+L.+Meagherauthor=D.+Sunauthor=J.+A.+Stuckeyauthor=S.+Wang&title=A+Potent+and+Selective+Small-Molecule+Degrader+of+STAT3+Achieves+Complete+Tumor+Regression+In+Vivo&doi=10.1016%2Fj.ccell.2019.10.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">A Potent and Selective Small-Molecule Degrader of STAT3 Achieves Complete Tumor Regression In Vivo</span></div><div class="casAuthors">Bai, Longchuan; Zhou, Haibin; Xu, Renqi; Zhao, Yujun; Chinnaswamy, Krishnapriya; McEachern, Donna; Chen, Jianyong; Yang, Chao-Yie; Liu, Zhaomin; Wang, Mi; Liu, Liu; Jiang, Hui; Wen, Bo; Kumar, Praveen; Meagher, Jennifer L.; Sun, Duxin; Stuckey, Jeanne A.; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">498-511.e17</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Signal transducer and activator of transcription 3 (STAT3) is an attractive cancer therapeutic target.  Here we report the discovery of SD-36(I), a small-mol. degrader of STAT3.  SD-36 potently induces the degrdn. of STAT3 protein in vitro and in vivo and demonstrates high selectivity over other STAT members.  Induced degrdn. of STAT3 results in a strong suppression of its transcription network in leukemia and lymphoma cells.  SD-36 inhibits the growth of a subset of acute myeloid leukemia and anaplastic large-cell lymphoma cell lines by inducing cell-cycle arrest and/or apoptosis.  SD-36 achieves complete and long-lasting tumor regression in multiple xenograft mouse models at well-tolerated dose schedules.  Degrdn. of STAT3 protein, therefore, is a promising cancer therapeutic strategy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUaP49Ap9DxLVg90H21EOLACvtfcHk0lg6uzpkGh4s-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitFCjtrjO&md5=71252c17d50a1a8be829352942175bd6</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2019.10.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2019.10.002%26sid%3Dliteratum%253Aachs%26aulast%3DBai%26aufirst%3DL.%26aulast%3DZhou%26aufirst%3DH.%26aulast%3DXu%26aufirst%3DR.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DChinnaswamy%26aufirst%3DK.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DC.%2BY.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DKumar%26aufirst%3DP.%26aulast%3DMeagher%26aufirst%3DJ.%2BL.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DStuckey%26aufirst%3DJ.%2BA.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DA%2520Potent%2520and%2520Selective%2520Small-Molecule%2520Degrader%2520of%2520STAT3%2520Achieves%2520Complete%2520Tumor%2520Regression%2520In%2520Vivo%26jtitle%3DCancer%2520Cell%26date%3D2019%26volume%3D36%26spage%3D498%26epage%3D511e17%26doi%3D10.1016%2Fj.ccell.2019.10.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Farnaby, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koegl, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roy, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitworth, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diers, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trainor, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zollman, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steurer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karolyi-Oezguer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riedmueller, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gmaschitz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wachter, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dank, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galant, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharps, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rumpel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Traxler, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerstberger, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schnitzer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petermann, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greb, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinstabl, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bader, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoephel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss-Puxbaum, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ehrenhofer-Wolfer, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wohrle, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boehmelt, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rinnenthal, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnhof, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiechens, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, M. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owen-Hughes, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ettmayer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McConnell, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">BAF complex vulnerabilities in cancer demonstrated via structure-based PROTAC design</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">672</span>– <span class="NLM_lpage">680</span>, <span class="refDoi"> DOI: 10.1038/s41589-019-0294-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1038%2Fs41589-019-0294-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=31178587" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtFeqtLfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2019&pages=672-680&author=W.+Farnabyauthor=M.+Koeglauthor=M.+J.+Royauthor=C.+Whitworthauthor=E.+Diersauthor=N.+Trainorauthor=D.+Zollmanauthor=S.+Steurerauthor=J.+Karolyi-Oezguerauthor=C.+Riedmuellerauthor=T.+Gmaschitzauthor=J.+Wachterauthor=C.+Dankauthor=M.+Galantauthor=B.+Sharpsauthor=K.+Rumpelauthor=E.+Traxlerauthor=T.+Gerstbergerauthor=R.+Schnitzerauthor=O.+Petermannauthor=P.+Grebauthor=H.+Weinstablauthor=G.+Baderauthor=A.+Zoephelauthor=A.+Weiss-Puxbaumauthor=K.+Ehrenhofer-Wolferauthor=S.+Wohrleauthor=G.+Boehmeltauthor=J.+Rinnenthalauthor=H.+Arnhofauthor=N.+Wiechensauthor=M.+Y.+Wuauthor=T.+Owen-Hughesauthor=P.+Ettmayerauthor=M.+Pearsonauthor=D.+B.+McConnellauthor=A.+Ciulli&title=BAF+complex+vulnerabilities+in+cancer+demonstrated+via+structure-based+PROTAC+design&doi=10.1038%2Fs41589-019-0294-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">BAF complex vulnerabilities in cancer demonstrated via structure-based PROTAC design</span></div><div class="casAuthors">Farnaby, William; Koegl, Manfred; Roy, Michael J.; Whitworth, Claire; Diers, Emelyne; Trainor, Nicole; Zollman, David; Steurer, Steffen; Karolyi-Oezguer, Jale; Riedmueller, Carina; Gmaschitz, Teresa; Wachter, Johannes; Dank, Christian; Galant, Michael; Sharps, Bernadette; Rumpel, Klaus; Traxler, Elisabeth; Gerstberger, Thomas; Schnitzer, Renate; Petermann, Oliver; Greb, Peter; Weinstabl, Harald; Bader, Gerd; Zoephel, Andreas; Weiss-Puxbaum, Alexander; Ehrenhofer-Wolfer, Katharina; Wohrle, Simon; Boehmelt, Guido; Rinnenthal, Joerg; Arnhof, Heribert; Wiechens, Nicola; Wu, Meng-Ying; Owen-Hughes, Tom; Ettmayer, Peter; Pearson, Mark; McConnell, Darryl B.; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">672-680</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Targeting subunits of BAF/PBAF chromatin remodeling complexes has been proposed as an approach to exploit cancer vulnerabilities.  Here, we develop proteolysis targeting chimera (PROTAC) degraders of the BAF ATPase subunits SMARCA2 and SMARCA4 using a bromodomain ligand and recruitment of the E3 ubiquitin ligase VHL.  High-resoln. ternary complex crystal structures and biophys. investigation guided rational and efficient optimization toward ACBI1(I), a potent and cooperative degrader of SMARCA2, SMARCA4 and PBRM1.  I induced anti-proliferative effects and cell death caused by SMARCA2 depletion in SMARCA4 mutant cancer cells, and in acute myeloid leukemia cells dependent on SMARCA4 ATPase activity.  These findings exemplify a successful biophysics- and structure-based PROTAC design approach to degrade high profile drug targets, and pave the way toward new therapeutics for the treatment of tumors sensitive to the loss of BAF complex ATPases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonEh4pux4Bs7Vg90H21EOLACvtfcHk0lg6uzpkGh4s-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtFeqtLfJ&md5=e95fad12c727aedcf08aed9320dae905</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fs41589-019-0294-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-019-0294-6%26sid%3Dliteratum%253Aachs%26aulast%3DFarnaby%26aufirst%3DW.%26aulast%3DKoegl%26aufirst%3DM.%26aulast%3DRoy%26aufirst%3DM.%2BJ.%26aulast%3DWhitworth%26aufirst%3DC.%26aulast%3DDiers%26aufirst%3DE.%26aulast%3DTrainor%26aufirst%3DN.%26aulast%3DZollman%26aufirst%3DD.%26aulast%3DSteurer%26aufirst%3DS.%26aulast%3DKarolyi-Oezguer%26aufirst%3DJ.%26aulast%3DRiedmueller%26aufirst%3DC.%26aulast%3DGmaschitz%26aufirst%3DT.%26aulast%3DWachter%26aufirst%3DJ.%26aulast%3DDank%26aufirst%3DC.%26aulast%3DGalant%26aufirst%3DM.%26aulast%3DSharps%26aufirst%3DB.%26aulast%3DRumpel%26aufirst%3DK.%26aulast%3DTraxler%26aufirst%3DE.%26aulast%3DGerstberger%26aufirst%3DT.%26aulast%3DSchnitzer%26aufirst%3DR.%26aulast%3DPetermann%26aufirst%3DO.%26aulast%3DGreb%26aufirst%3DP.%26aulast%3DWeinstabl%26aufirst%3DH.%26aulast%3DBader%26aufirst%3DG.%26aulast%3DZoephel%26aufirst%3DA.%26aulast%3DWeiss-Puxbaum%26aufirst%3DA.%26aulast%3DEhrenhofer-Wolfer%26aufirst%3DK.%26aulast%3DWohrle%26aufirst%3DS.%26aulast%3DBoehmelt%26aufirst%3DG.%26aulast%3DRinnenthal%26aufirst%3DJ.%26aulast%3DArnhof%26aufirst%3DH.%26aulast%3DWiechens%26aufirst%3DN.%26aulast%3DWu%26aufirst%3DM.%2BY.%26aulast%3DOwen-Hughes%26aufirst%3DT.%26aulast%3DEttmayer%26aufirst%3DP.%26aulast%3DPearson%26aufirst%3DM.%26aulast%3DMcConnell%26aufirst%3DD.%2BB.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DBAF%2520complex%2520vulnerabilities%2520in%2520cancer%2520demonstrated%2520via%2520structure-based%2520PROTAC%2520design%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2019%26volume%3D15%26spage%3D672%26epage%3D680%26doi%3D10.1038%2Fs41589-019-0294-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Y.</span></span> <span> </span><span class="NLM_article-title">PROTAC-induced BTK degradation as a novel therapy for mutated BTK C481S induced ibrutinib-resistant B-cell malignancies</span>. <i>Cell Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">779</span>– <span class="NLM_lpage">781</span>, <span class="refDoi"> DOI: 10.1038/s41422-018-0055-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1038%2Fs41422-018-0055-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=29875397" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1ektbzO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2018&pages=779-781&author=Y.+Sunauthor=X.+Zhaoauthor=N.+Dingauthor=H.+Gaoauthor=Y.+Wuauthor=Y.+Yangauthor=M.+Zhaoauthor=J.+Hwangauthor=Y.+Songauthor=W.+Liuauthor=Y.+Rao&title=PROTAC-induced+BTK+degradation+as+a+novel+therapy+for+mutated+BTK+C481S+induced+ibrutinib-resistant+B-cell+malignancies&doi=10.1038%2Fs41422-018-0055-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">PROTAC-induced BTK degradation as a novel therapy for mutated BTK C481S induced ibrutinib-resistant B-cell malignancies</span></div><div class="casAuthors">Sun, Yonghui; Zhao, Xingwang; Ding, Ning; Gao, Hongying; Wu, Yue; Yang, Yiqing; Zhao, Meng; Hwang, Jinseok; Song, Yuqin; Liu, Wanli; Rao, Yu</div><div class="citationInfo"><span class="NLM_cas:title">Cell Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">779-781</span>CODEN:
                <span class="NLM_cas:coden">CREEB6</span>;
        ISSN:<span class="NLM_cas:issn">1001-0602</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">In this study, for the first time, we report the development of BTK-targeting degraders using the PROTAC strategy.  These PROTACs could ef fi ciently degrade ibrutinib-sensitive BTK-WT (wild type).  More importantly, our newly designed PROTACs also significantly induced the degrdn. of ibrutinib resistant BTK-C481S (50% degrdn. efficiency at 30 nM).  Furthermore, our PROTAC mols. ef fi ciently inhibited cell proliferation and colony formation, while exhibited no obvious inhibition (>1000 nM) of ITK, EGFR, and TEC, which are major off-targets of ibrutinib.  These data demonstrate the strong potential for developing PROTAC-based therapeutic mols.  The results indicate that the PROTAC strategy (protein degrdn. instead of inhibition) could be adopted as a general and powerful therapeutic treatment for drug-resistant cancers in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqg_FLdaUP_HLVg90H21EOLACvtfcHk0lidc5ixkvvljQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1ektbzO&md5=1cca51ac76ecb3b491da0117b21271fb</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fs41422-018-0055-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41422-018-0055-1%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DDing%26aufirst%3DN.%26aulast%3DGao%26aufirst%3DH.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DM.%26aulast%3DHwang%26aufirst%3DJ.%26aulast%3DSong%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DRao%26aufirst%3DY.%26atitle%3DPROTAC-induced%2520BTK%2520degradation%2520as%2520a%2520novel%2520therapy%2520for%2520mutated%2520BTK%2520C481S%2520induced%2520ibrutinib-resistant%2520B-cell%2520malignancies%26jtitle%3DCell%2520Res.%26date%3D2018%26volume%3D28%26spage%3D779%26epage%3D781%26doi%3D10.1038%2Fs41422-018-0055-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tomaselli, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mautone, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rotili, D.</span></span> <span> </span><span class="NLM_article-title">Recent advances in epigenetic proteolysis targeting chimeras (Epi-PROTACs)</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>207</i></span>, <span class="NLM_elocation-id">112750</span> <span class="refDoi"> DOI: 10.1016/j.ejmech.2020.112750</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1016%2Fj.ejmech.2020.112750" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=32871345" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhvVWhtbjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=207&publication_year=2020&author=D.+Tomaselliauthor=N.+Mautoneauthor=A.+Maiauthor=D.+Rotili&title=Recent+advances+in+epigenetic+proteolysis+targeting+chimeras+%28Epi-PROTACs%29&doi=10.1016%2Fj.ejmech.2020.112750"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Recent advances in epigenetic proteolysis targeting chimeras (Epi-PROTACs)</span></div><div class="casAuthors">Tomaselli, Daniela; Mautone, Nicola; Mai, Antonello; Rotili, Dante</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">207</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">112750</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A review.  PROteolysis TArgeting Chimeras (PROTACs) are heterobifunctional mols. that trigger the poly-ubiquitination of the protein of interest (POI) inducing its degrdn. via the recruitment of the ubiquitin-proteasome system, thus suppressing the POI's intracellular levels and indirectly all its functions.  Recently, one of the fields where the protein knockdown induced by PROTACs has demonstrated to serve as a promising biochem. tool and to provide new opportunities for drug discovery is the epigenetics (epi-PROTACs).  A full inhibition of the functions of all domains of a specific epigenetic POI (e-POI), rather than just the block of its catalytic/single domain activity, is in fact a new more effective modality to hit an e-POI and, in principle, the complex it belongs to, and potentially to treat the related diseases, first cancer.  In this review, we will present the most relevant progresses made, esp. in the last two years, in the application of PROTACs technol. to the three main classes of e-POIs: "writers", "erasers" and "readers".  Emphasis will be devoted to the medicinal chem. aspects of the epi-PROTACs design, prepn., and optimization and to the comparison with small mol. epi-drugs for both epi-targets functional annotation and potential anticancer therapy purposes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrd2A6SgsU0hbVg90H21EOLACvtfcHk0lidc5ixkvvljQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhvVWhtbjI&md5=d75dbf0e84fee63f89c9f79177ce1982</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2020.112750&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2020.112750%26sid%3Dliteratum%253Aachs%26aulast%3DTomaselli%26aufirst%3DD.%26aulast%3DMautone%26aufirst%3DN.%26aulast%3DMai%26aufirst%3DA.%26aulast%3DRotili%26aufirst%3DD.%26atitle%3DRecent%2520advances%2520in%2520epigenetic%2520proteolysis%2520targeting%2520chimeras%2520%2528Epi-PROTACs%2529%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2020%26volume%3D207%26doi%3D10.1016%2Fj.ejmech.2020.112750" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vogelmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robaa, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sippl, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, M.</span></span> <span> </span><span class="NLM_article-title">Proteolysis targeting chimeras (PROTACs) for epigenetics research</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">8</span>– <span class="NLM_lpage">16</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2020.01.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1016%2Fj.cbpa.2020.01.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=32146413" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BB3cXksFeitrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2020&pages=8-16&author=A.+Vogelmannauthor=D.+Robaaauthor=W.+Sipplauthor=M.+Jung&title=Proteolysis+targeting+chimeras+%28PROTACs%29+for+epigenetics+research&doi=10.1016%2Fj.cbpa.2020.01.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Proteolysis targeting chimeras (PROTACs) for epigenetics research</span></div><div class="casAuthors">Vogelmann, Anja; Robaa, Dina; Sippl, Wolfgang; Jung, Manfred</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">8-16</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Proteolysis targeting chimeras (PROTACs) are heterobifunctional mols. and allow selective protein degrdn. by addressing the natural ubiquitin proteasome system.  As this new strategy of chem. induced protein degrdn. can serve as a biol. tool and provides new possibilities for drug discovery, it has been applied to a variety of targets including (nuclear) receptors, kinases, and epigenetic proteins.  A lot of PROTACs have already been designed in the field of epigenetics, and their synthesis and characterization highly contributed to structural optimization and improved mechanistic understanding of these mols.  In this review, we will discuss and summarize recent advances in PROTAC discovery with focus on epigenetic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqW2o_9L2Y8GbVg90H21EOLACvtfcHk0lg_R-HXqZz6ug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXksFeitrw%253D&md5=6a4f3b70d6c6a446c744b9788b7582cf</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2020.01.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2020.01.010%26sid%3Dliteratum%253Aachs%26aulast%3DVogelmann%26aufirst%3DA.%26aulast%3DRobaa%26aufirst%3DD.%26aulast%3DSippl%26aufirst%3DW.%26aulast%3DJung%26aufirst%3DM.%26atitle%3DProteolysis%2520targeting%2520chimeras%2520%2528PROTACs%2529%2520for%2520epigenetics%2520research%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2020%26volume%3D57%26spage%3D8%26epage%3D16%26doi%3D10.1016%2Fj.cbpa.2020.01.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, P.</span></span> <span> </span><span class="NLM_article-title">Novel combination of histone methylation modulators with therapeutic synergy against acute myeloid leukemia in vitro and in vivo</span>. <i>Cancer Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>413</i></span>,  <span class="NLM_fpage">35</span>– <span class="NLM_lpage">45</span>, <span class="refDoi"> DOI: 10.1016/j.canlet.2017.10.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1016%2Fj.canlet.2017.10.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=29069576" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslagtbbE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=413&publication_year=2018&pages=35-45&author=S.+Wenauthor=J.+Wangauthor=P.+Liuauthor=Y.+Liauthor=W.+Luauthor=Y.+Huauthor=J.+Liuauthor=Z.+Heauthor=P.+Huang&title=Novel+combination+of+histone+methylation+modulators+with+therapeutic+synergy+against+acute+myeloid+leukemia+in+vitro+and+in+vivo&doi=10.1016%2Fj.canlet.2017.10.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Novel combination of histone methylation modulators with therapeutic synergy against acute myeloid leukemia in vitro and in vivo</span></div><div class="casAuthors">Wen, Shijun; Wang, Jiankang; Liu, Panpan; Li, Yiqing; Lu, Wenhua; Hu, Yumin; Liu, Jinyun; He, Zhiyuan; Huang, Peng</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Letters (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">413</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">35-45</span>CODEN:
                <span class="NLM_cas:coden">CALEDQ</span>;
        ISSN:<span class="NLM_cas:issn">0304-3835</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Acute myeloid leukemia (AML) is a hematol. malignancy with rapid disease progression and often becomes lethal without treatment.  Development of effective new therapies is essential to improve the clin. outcome of AML patients.  Enhancer of zeste homolog 2 (EZH2) and lysine specific demethylase 1 (LSD1) play important roles in epigenetic regulation and their altered expressions have been obsd. in cancer.  Although EZH2 and LSD1 have opposite histone methylation functions, we found that both enzymes were paradoxically up-regulated in AML cells.  Importantly, a combined inhibition of EZH2 and LSD1 resulted in a synergistic activity against AML in vitro and in vivo.  Such synergy was mechanistically correlated with up-regulation of H3K4me1/2 and H3K9Ac and down-regulation of H3K27me3, leading to a decrease of anti-apoptotic protein Bcl-2.  These epigenetic alterations also compromised the mitochondrial respiration capacity and glycolytic activity and resulted in ATP depletion, a key event contributing to the potent cytotoxic effect of the drug combination.  Taken together, our work identified a novel therapeutic approach against AML by combining two small mols. that inhibit different histone methylation-modulating proteins with apparently opposite enzyme activities.  Such a new drug combination strategy likely has significant clin. implications since epigenetic modulators are currently in clin. trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp78k_2d8kiwrVg90H21EOLACvtfcHk0lg_R-HXqZz6ug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslagtbbE&md5=e79654ad4081eae952118c3e00afb778</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.canlet.2017.10.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.canlet.2017.10.015%26sid%3Dliteratum%253Aachs%26aulast%3DWen%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DP.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DLu%26aufirst%3DW.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DHe%26aufirst%3DZ.%26aulast%3DHuang%26aufirst%3DP.%26atitle%3DNovel%2520combination%2520of%2520histone%2520methylation%2520modulators%2520with%2520therapeutic%2520synergy%2520against%2520acute%2520myeloid%2520leukemia%2520in%2520vitro%2520and%2520in%2520vivo%26jtitle%3DCancer%2520Lett.%26date%3D2018%26volume%3D413%26spage%3D35%26epage%3D45%26doi%3D10.1016%2Fj.canlet.2017.10.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kuntz, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keilhack, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollock, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knutson, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter-Scott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richon, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sneeringer, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wigle, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allain, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majer, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moyer, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chesworth, R.</span></span> <span> </span><span class="NLM_article-title">The Importance of Being Me: Magic Methyls, Methyltransferase Inhibitors, and the Discovery of Tazemetostat</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1556</span>– <span class="NLM_lpage">1564</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01501</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01501" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC28XotF2qsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=1556-1564&author=K.+W.+Kuntzauthor=J.+E.+Campbellauthor=H.+Keilhackauthor=R.+M.+Pollockauthor=S.+K.+Knutsonauthor=M.+Porter-Scottauthor=V.+M.+Richonauthor=C.+J.+Sneeringerauthor=T.+J.+Wigleauthor=C.+J.+Allainauthor=C.+R.+Majerauthor=M.+P.+Moyerauthor=R.+A.+Copelandauthor=R.+Chesworth&title=The+Importance+of+Being+Me%3A+Magic+Methyls%2C+Methyltransferase+Inhibitors%2C+and+the+Discovery+of+Tazemetostat&doi=10.1021%2Facs.jmedchem.5b01501"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">The Importance of Being Me: Magic Methyls, Methyltransferase Inhibitors, and the Discovery of Tazemetostat</span></div><div class="casAuthors">Kuntz, Kevin W.; Campbell, John E.; Keilhack, Heike; Pollock, Roy M.; Knutson, Sarah K.; Porter-Scott, Margaret; Richon, Victoria M.; Sneeringer, Chris J.; Wigle, Tim J.; Allain, Christina J.; Majer, Christina R.; Moyer, Mikel P.; Copeland, Robert A.; Chesworth, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1556-1564</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Posttranslational methylation of histones plays a crit. role in gene regulation.  Misregulation of histone methylation can lead to oncogenic transformation.  Enhancer of Zeste homolog 2 (EZH2) methylates histone 3 at lysine 27 (H3K27) and abnormal methylation of this site is found in many cancers.  Tazemetostat, an EHZ2 inhibitor in clin. development, has shown activity in both preclin. models of cancer as well as in patients with lymphoma or INI1-deficient solid tumors.  Herein we report the structure-activity relationships from identification of an initial hit in a high-throughput screen through selection of tazemetostat for clin. development.  The importance of several Me groups to the potency of the inhibitors is highlighted as well as the importance of balancing pharmacokinetic properties with potency.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGryASbIgMDxAbVg90H21EOLACvtfcHk0lg_R-HXqZz6ug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XotF2qsA%253D%253D&md5=a26ccc4a0109f57fbb64f7fded8965a1</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01501&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01501%26sid%3Dliteratum%253Aachs%26aulast%3DKuntz%26aufirst%3DK.%2BW.%26aulast%3DCampbell%26aufirst%3DJ.%2BE.%26aulast%3DKeilhack%26aufirst%3DH.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26aulast%3DKnutson%26aufirst%3DS.%2BK.%26aulast%3DPorter-Scott%26aufirst%3DM.%26aulast%3DRichon%26aufirst%3DV.%2BM.%26aulast%3DSneeringer%26aufirst%3DC.%2BJ.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DAllain%26aufirst%3DC.%2BJ.%26aulast%3DMajer%26aufirst%3DC.%2BR.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DChesworth%26aufirst%3DR.%26atitle%3DThe%2520Importance%2520of%2520Being%2520Me%253A%2520Magic%2520Methyls%252C%2520Methyltransferase%2520Inhibitors%252C%2520and%2520the%2520Discovery%2520of%2520Tazemetostat%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D1556%26epage%3D1564%26doi%3D10.1021%2Facs.jmedchem.5b01501" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Galdeano, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gadd, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soares, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scaffidi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Molle, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birced, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hewitt, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dias, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Structure-guided design and optimization of small molecules targeting the protein-protein interaction between the von Hippel-Lindau (VHL) E3 ubiquitin ligase and the hypoxia inducible factor (HIF) alpha subunit with in vitro nanomolar affinities</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">8657</span>– <span class="NLM_lpage">8663</span>, <span class="refDoi"> DOI: 10.1021/jm5011258</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm5011258" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVGhtb%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=8657-8663&author=C.+Galdeanoauthor=M.+S.+Gaddauthor=P.+Soaresauthor=S.+Scaffidiauthor=I.+Van+Molleauthor=I.+Bircedauthor=S.+Hewittauthor=D.+M.+Diasauthor=A.+Ciulli&title=Structure-guided+design+and+optimization+of+small+molecules+targeting+the+protein-protein+interaction+between+the+von+Hippel-Lindau+%28VHL%29+E3+ubiquitin+ligase+and+the+hypoxia+inducible+factor+%28HIF%29+alpha+subunit+with+in+vitro+nanomolar+affinities&doi=10.1021%2Fjm5011258"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Guided Design and Optimization of Small Molecules Targeting the Protein-Protein Interaction between the von Hippel-Lindau (VHL) E3 Ubiquitin Ligase and the Hypoxia Inducible Factor (HIF) Alpha Subunit with in Vitro Nanomolar Affinities</span></div><div class="casAuthors">Galdeano, Carles; Gadd, Morgan S.; Soares, Pedro; Scaffidi, Salvatore; Van Molle, Inge; Birced, Ipek; Hewitt, Sarah; Dias, David M.; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">8657-8663</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">E3 ubiquitin ligases are attractive targets in the ubiquitin-proteasome system, however, the development of small-mol. ligands has been rewarded with limited success.  The von Hippel-Lindau protein (pVHL) is the substrate recognition subunit of the VHL E3 ligase that targets HIF-1α for degrdn.  The authors recently reported inhibitors of the pVHL:HIF-1α interaction, however they exhibited moderate potency.  Herein, the authors report the design and optimization, guided by x-ray crystal structures, of a ligand series with nanomolar binding affinities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpixK3qp1ctJrVg90H21EOLACvtfcHk0lgNzr6kO7Et_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVGhtb%252FP&md5=831aa2387d94cb03d735a4be7bdd2cdd</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fjm5011258&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm5011258%26sid%3Dliteratum%253Aachs%26aulast%3DGaldeano%26aufirst%3DC.%26aulast%3DGadd%26aufirst%3DM.%2BS.%26aulast%3DSoares%26aufirst%3DP.%26aulast%3DScaffidi%26aufirst%3DS.%26aulast%3DVan%2BMolle%26aufirst%3DI.%26aulast%3DBirced%26aufirst%3DI.%26aulast%3DHewitt%26aufirst%3DS.%26aulast%3DDias%26aufirst%3DD.%2BM.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DStructure-guided%2520design%2520and%2520optimization%2520of%2520small%2520molecules%2520targeting%2520the%2520protein-protein%2520interaction%2520between%2520the%2520von%2520Hippel-Lindau%2520%2528VHL%2529%2520E3%2520ubiquitin%2520ligase%2520and%2520the%2520hypoxia%2520inducible%2520factor%2520%2528HIF%2529%2520alpha%2520subunit%2520with%2520in%2520vitro%2520nanomolar%2520affinities%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D8657%26epage%3D8663%26doi%3D10.1021%2Fjm5011258" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Frost, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galdeano, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soares, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gadd, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grzes, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Epemolu, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimamura, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bantscheff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantrell, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rocha, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Potent and selective chemical probe of hypoxic signalling downstream of HIF-alpha hydroxylation via VHL inhibition</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>, <span class="NLM_elocation-id">13312</span> <span class="refDoi"> DOI: 10.1038/ncomms13312</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1038%2Fncomms13312" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=27811928" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVSitLfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&author=J.+Frostauthor=C.+Galdeanoauthor=P.+Soaresauthor=M.+S.+Gaddauthor=K.+M.+Grzesauthor=L.+Ellisauthor=O.+Epemoluauthor=S.+Shimamuraauthor=M.+Bantscheffauthor=P.+Grandiauthor=K.+D.+Readauthor=D.+A.+Cantrellauthor=S.+Rochaauthor=A.+Ciulli&title=Potent+and+selective+chemical+probe+of+hypoxic+signalling+downstream+of+HIF-alpha+hydroxylation+via+VHL+inhibition&doi=10.1038%2Fncomms13312"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Potent and selective chemical probe of hypoxic signalling downstream of HIF-α hydroxylation via VHL inhibition</span></div><div class="casAuthors">Frost, Julianty; Galdeano, Carles; Soares, Pedro; Gadd, Morgan S.; Grzes, Katarzyna M.; Ellis, Lucy; Epemolu, Ola; Shimamura, Satoko; Bantscheff, Marcus; Grandi, Paola; Read, Kevin D.; Cantrell, Doreen A.; Rocha, Sonia; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">13312</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Chem. strategies to using small mols. to stimulate hypoxia inducible factors (HIFs) activity and trigger a hypoxic response under normoxic conditions, such as iron chelators and inhibitors of prolyl hydroxylase domain (PHD) enzymes, have broad-spectrum activities and off-target effects.  Here we disclose VH298, a potent VHL inhibitor that stabilizes HIF-α and elicits a hypoxic response via a different mechanism, that is the blockade of the VHL:HIF-α protein-protein interaction downstream of HIF-α hydroxylation by PHD enzymes.  We show that VH298 engages with high affinity and specificity with VHL as its only major cellular target, leading to selective on-target accumulation of hydroxylated HIF-α in a concn.- and time-dependent fashion in different cell lines, with subsequent upregulation of HIF-target genes at both mRNA and protein levels.  VH298 represents a high-quality chem. probe of the HIF signalling cascade and an attractive starting point to the development of potential new therapeutics targeting hypoxia signalling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLB1WvVF4EFLVg90H21EOLACvtfcHk0lgNzr6kO7Et_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVSitLfJ&md5=7a884b6193b3ca5e119fecea2eb31d3e</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1038%2Fncomms13312&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms13312%26sid%3Dliteratum%253Aachs%26aulast%3DFrost%26aufirst%3DJ.%26aulast%3DGaldeano%26aufirst%3DC.%26aulast%3DSoares%26aufirst%3DP.%26aulast%3DGadd%26aufirst%3DM.%2BS.%26aulast%3DGrzes%26aufirst%3DK.%2BM.%26aulast%3DEllis%26aufirst%3DL.%26aulast%3DEpemolu%26aufirst%3DO.%26aulast%3DShimamura%26aufirst%3DS.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DRead%26aufirst%3DK.%2BD.%26aulast%3DCantrell%26aufirst%3DD.%2BA.%26aulast%3DRocha%26aufirst%3DS.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DPotent%2520and%2520selective%2520chemical%2520probe%2520of%2520hypoxic%2520signalling%2520downstream%2520of%2520HIF-alpha%2520hydroxylation%2520via%2520VHL%2520inhibition%26jtitle%3DNat.%2520Commun.%26date%3D2016%26volume%3D7%26doi%3D10.1038%2Fncomms13312" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zorba, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Starr, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borzilleri, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farley, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiemer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uccello, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia-Irrizary, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czabaniuk, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuff, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliver, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montgomery, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayward, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coe, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niosi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luthra, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El-Kattan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noe, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanmugasundaram, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilbert, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calabrese, M. F.</span></span> <span> </span><span class="NLM_article-title">Delineating the role of cooperativity in the design of potent PROTACs for BTK</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>115</i></span>,  <span class="NLM_fpage">E7285</span>– <span class="NLM_lpage">E7292</span>, <span class="refDoi"> DOI: 10.1073/pnas.1803662115</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1073%2Fpnas.1803662115" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=30012605" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=115&publication_year=2018&pages=E7285-E7292&author=A.+Zorbaauthor=C.+Nguyenauthor=Y.+Xuauthor=J.+Starrauthor=K.+Borzilleriauthor=J.+Smithauthor=H.+Zhuauthor=K.+A.+Farleyauthor=W.+Dingauthor=J.+Schiemerauthor=X.+Fengauthor=J.+S.+Changauthor=D.+P.+Uccelloauthor=J.+A.+Youngauthor=C.+N.+Garcia-Irrizaryauthor=L.+Czabaniukauthor=B.+Schuffauthor=R.+Oliverauthor=J.+Montgomeryauthor=M.+M.+Haywardauthor=J.+Coeauthor=J.+Chenauthor=M.+Niosiauthor=S.+Luthraauthor=J.+C.+Shahauthor=A.+El-Kattanauthor=X.+Qiuauthor=G.+M.+Westauthor=M.+C.+Noeauthor=V.+Shanmugasundaramauthor=A.+M.+Gilbertauthor=M.+F.+Brownauthor=M.+F.+Calabrese&title=Delineating+the+role+of+cooperativity+in+the+design+of+potent+PROTACs+for+BTK&doi=10.1073%2Fpnas.1803662115"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1803662115&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1803662115%26sid%3Dliteratum%253Aachs%26aulast%3DZorba%26aufirst%3DA.%26aulast%3DNguyen%26aufirst%3DC.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DStarr%26aufirst%3DJ.%26aulast%3DBorzilleri%26aufirst%3DK.%26aulast%3DSmith%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DH.%26aulast%3DFarley%26aufirst%3DK.%2BA.%26aulast%3DDing%26aufirst%3DW.%26aulast%3DSchiemer%26aufirst%3DJ.%26aulast%3DFeng%26aufirst%3DX.%26aulast%3DChang%26aufirst%3DJ.%2BS.%26aulast%3DUccello%26aufirst%3DD.%2BP.%26aulast%3DYoung%26aufirst%3DJ.%2BA.%26aulast%3DGarcia-Irrizary%26aufirst%3DC.%2BN.%26aulast%3DCzabaniuk%26aufirst%3DL.%26aulast%3DSchuff%26aufirst%3DB.%26aulast%3DOliver%26aufirst%3DR.%26aulast%3DMontgomery%26aufirst%3DJ.%26aulast%3DHayward%26aufirst%3DM.%2BM.%26aulast%3DCoe%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DNiosi%26aufirst%3DM.%26aulast%3DLuthra%26aufirst%3DS.%26aulast%3DShah%26aufirst%3DJ.%2BC.%26aulast%3DEl-Kattan%26aufirst%3DA.%26aulast%3DQiu%26aufirst%3DX.%26aulast%3DWest%26aufirst%3DG.%2BM.%26aulast%3DNoe%26aufirst%3DM.%2BC.%26aulast%3DShanmugasundaram%26aufirst%3DV.%26aulast%3DGilbert%26aufirst%3DA.%2BM.%26aulast%3DBrown%26aufirst%3DM.%2BF.%26aulast%3DCalabrese%26aufirst%3DM.%2BF.%26atitle%3DDelineating%2520the%2520role%2520of%2520cooperativity%2520in%2520the%2520design%2520of%2520potent%2520PROTACs%2520for%2520BTK%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2018%26volume%3D115%26spage%3DE7285%26epage%3DE7292%26doi%3D10.1073%2Fpnas.1803662115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bondeson, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burslem, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buhimschi, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaime-Figueroa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamman, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishchenko, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Lessons in PROTAC Design from Selective Degradation with a Promiscuous Warhead</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">78</span>– <span class="NLM_lpage">87.e5</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.09.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1016%2Fj.chembiol.2017.09.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=29129718" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=78-87.e5&author=D.+P.+Bondesonauthor=B.+E.+Smithauthor=G.+M.+Burslemauthor=A.+D.+Buhimschiauthor=J.+Hinesauthor=S.+Jaime-Figueroaauthor=J.+Wangauthor=B.+D.+Hammanauthor=A.+Ishchenkoauthor=C.+M.+Crews&title=Lessons+in+PROTAC+Design+from+Selective+Degradation+with+a+Promiscuous+Warhead&doi=10.1016%2Fj.chembiol.2017.09.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Lessons in PROTAC Design from Selective Degradation with a Promiscuous Warhead</span></div><div class="casAuthors">Bondeson, Daniel P.; Smith, Blake E.; Burslem, George M.; Buhimschi, Alexandru D.; Hines, John; Jaime-Figueroa, Saul; Wang, Jing; Hamman, Brian D.; Ishchenko, Alexey; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">78-87.e5</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Inhibiting protein function selectively is a major goal of modern drug discovery.  Here, we report a previously understudied benefit of small mol. proteolysis-targeting chimeras (PROTACs) that recruit E3 ubiquitin ligases to target proteins for their ubiquitination and subsequent proteasome-mediated degrdn.  Using promiscuous CRBN- and VHL-recruiting PROTACs that bind >50 kinases, we show that only a subset of bound targets is degraded.  The basis of this selectivity relies on protein-protein interactions between the E3 ubiquitin ligase and the target protein, as illustrated by engaged proteins that are not degraded as a result of unstable ternary complexes with PROTAC-recruited E3 ligases.  In contrast, weak PROTAC:target protein affinity can be stabilized by high-affinity target:PROTAC:ligase trimer interactions, leading to efficient degrdn.  This study highlights design guidelines for generating potent PROTACs as well as possibilities for degrading undruggable proteins immune to traditional small-mol. inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoISOBudOLyFrVg90H21EOLACvtfcHk0liU-Bi6PCNu4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjN&md5=30fac502705a2344c4261cf531e6bf9d</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.09.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.09.010%26sid%3Dliteratum%253Aachs%26aulast%3DBondeson%26aufirst%3DD.%2BP.%26aulast%3DSmith%26aufirst%3DB.%2BE.%26aulast%3DBurslem%26aufirst%3DG.%2BM.%26aulast%3DBuhimschi%26aufirst%3DA.%2BD.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DJaime-Figueroa%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DHamman%26aufirst%3DB.%2BD.%26aulast%3DIshchenko%26aufirst%3DA.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DLessons%2520in%2520PROTAC%2520Design%2520from%2520Selective%2520Degradation%2520with%2520a%2520Promiscuous%2520Warhead%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2018%26volume%3D25%26spage%3D78%26epage%3D87.e5%26doi%3D10.1016%2Fj.chembiol.2017.09.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calabrese, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bullock, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Targeted protein degradation: expanding the toolbox</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">949</span>– <span class="NLM_lpage">963</span>, <span class="refDoi"> DOI: 10.1038/s41573-019-0047-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1038%2Fs41573-019-0047-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=31666732" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitVKrtLrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=949-963&author=M.+Schapiraauthor=M.+F.+Calabreseauthor=A.+N.+Bullockauthor=C.+M.+Crews&title=Targeted+protein+degradation%3A+expanding+the+toolbox&doi=10.1038%2Fs41573-019-0047-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted protein degradation: expanding the toolbox</span></div><div class="casAuthors">Schapira, Matthieu; Calabrese, Matthew F.; Bullock, Alex N.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">949-963</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review. Proteolysis-targeting chimeras (PROTACs) and related mols. that induce targeted protein degrdn. by the ubiquitin-proteasome system represent a new therapeutic modality and are the focus of great interest, owing to potential advantages over traditional occupancy-based inhibitors with respect to dosing, side effects, drug resistance and modulating 'undruggable' targets.  However, the technol. is still maturing, and the design elements for successful PROTAC-based drugs are currently being elucidated.  Importantly, fewer than 10 of the more than 600 E3 ubiquitin ligases have so far been exploited for targeted protein degrdn., and expansion of knowledge in this area is a key opportunity.  Here, we briefly discuss lessons learned about targeted protein degrdn. in chem. biol. and drug discovery and systematically review the expression profile, domain architecture and chem. tractability of human E3 ligases that could expand the toolbox for PROTAC discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwUvM27MU3WrVg90H21EOLACvtfcHk0liU-Bi6PCNu4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitVKrtLrM&md5=9bfd48c6c47fc1cce15129495fea7079</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1038%2Fs41573-019-0047-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41573-019-0047-y%26sid%3Dliteratum%253Aachs%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DCalabrese%26aufirst%3DM.%2BF.%26aulast%3DBullock%26aufirst%3DA.%2BN.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DTargeted%2520protein%2520degradation%253A%2520expanding%2520the%2520toolbox%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2019%26volume%3D18%26spage%3D949%26epage%3D963%26doi%3D10.1038%2Fs41573-019-0047-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zoppi, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maniaci, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Testa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gmaschitz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wieshofer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koegl, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riching, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniels, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spallarossa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Iterative Design and Optimization of Initially Inactive Proteolysis Targeting Chimeras (PROTACs) Identify VZ185 as a Potent, Fast, and Selective von Hippel-Lindau (VHL) Based Dual Degrader Probe of BRD9 and BRD7</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">699</span>– <span class="NLM_lpage">726</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01413</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01413" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisVyhsLjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=699-726&author=V.+Zoppiauthor=S.+J.+Hughesauthor=C.+Maniaciauthor=A.+Testaauthor=T.+Gmaschitzauthor=C.+Wieshoferauthor=M.+Koeglauthor=K.+M.+Richingauthor=D.+L.+Danielsauthor=A.+Spallarossaauthor=A.+Ciulli&title=Iterative+Design+and+Optimization+of+Initially+Inactive+Proteolysis+Targeting+Chimeras+%28PROTACs%29+Identify+VZ185+as+a+Potent%2C+Fast%2C+and+Selective+von+Hippel-Lindau+%28VHL%29+Based+Dual+Degrader+Probe+of+BRD9+and+BRD7&doi=10.1021%2Facs.jmedchem.8b01413"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Iterative Design and Optimization of Initially Inactive Proteolysis Targeting Chimeras (PROTACs) Identify VZ185 as a Potent, Fast, and Selective von Hippel-Lindau (VHL) Based Dual Degrader Probe of BRD9 and BRD7</span></div><div class="casAuthors">Zoppi, Vittoria; Hughes, Scott J.; Maniaci, Chiara; Testa, Andrea; Gmaschitz, Teresa; Wieshofer, Corinna; Koegl, Manfred; Riching, Kristin M.; Daniels, Danette L.; Spallarossa, Andrea; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">699-726</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Developing PROTACs to redirect the ubiquitination activity of E3 ligases and potently degrade a target protein within cells can be a lengthy and unpredictable process, and it remains unclear whether any combination of E3 and target might be productive for degrdn.  The authors describe a probe-quality degrader for a ligase-target pair deemed unsuitable: the von Hippel-Lindau (VHL) and BRD9, a bromodomain-contg. subunit of the SWI/SNF chromatin remodeling complex BAF.  VHL-based degraders could be optimized from suboptimal compds. in two rounds by systematically varying conjugation patterns and linkers and monitoring cellular degrdn. activities, kinetic profiles, and ubiquitination, as well as ternary complex formation thermodn.  The emerged structure-activity relations guided the discovery of VZ185, a potent, fast, and selective degrader of BRD9 and of its close homolog BRD7.  The authors' findings qualify a new chem. tool for BRD7/9 knockdown and provide a roadmap for PROTAC development against seemingly incompatible target-ligase combinations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqiPnajotKvqLVg90H21EOLACvtfcHk0ljT7ucU1SVF0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisVyhsLjN&md5=be7938c26807a2535b7f9b7de8ebfbb0</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01413&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01413%26sid%3Dliteratum%253Aachs%26aulast%3DZoppi%26aufirst%3DV.%26aulast%3DHughes%26aufirst%3DS.%2BJ.%26aulast%3DManiaci%26aufirst%3DC.%26aulast%3DTesta%26aufirst%3DA.%26aulast%3DGmaschitz%26aufirst%3DT.%26aulast%3DWieshofer%26aufirst%3DC.%26aulast%3DKoegl%26aufirst%3DM.%26aulast%3DRiching%26aufirst%3DK.%2BM.%26aulast%3DDaniels%26aufirst%3DD.%2BL.%26aulast%3DSpallarossa%26aufirst%3DA.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DIterative%2520Design%2520and%2520Optimization%2520of%2520Initially%2520Inactive%2520Proteolysis%2520Targeting%2520Chimeras%2520%2528PROTACs%2529%2520Identify%2520VZ185%2520as%2520a%2520Potent%252C%2520Fast%252C%2520and%2520Selective%2520von%2520Hippel-Lindau%2520%2528VHL%2529%2520Based%2520Dual%2520Degrader%2520Probe%2520of%2520BRD9%2520and%2520BRD7%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D699%26epage%3D726%26doi%3D10.1021%2Facs.jmedchem.8b01413" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Potjewyd, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turner, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beri, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rectenwald, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norris-Drouin, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cholensky, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Margolis, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearce, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herring, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, L. I.</span></span> <span> </span><span class="NLM_article-title">Degradation of Polycomb Repressive Complex 2 with an EED-Targeted Bivalent Chemical Degrader</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">47</span>– <span class="NLM_lpage">56.e15</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2019.11.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1016%2Fj.chembiol.2019.11.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=31831267" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitlOmt7bM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2020&pages=47-56.e15&author=F.+Potjewydauthor=A.+W.+Turnerauthor=J.+Beriauthor=J.+M.+Rectenwaldauthor=J.+L.+Norris-Drouinauthor=S.+H.+Cholenskyauthor=D.+M.+Margolisauthor=K.+H.+Pearceauthor=L.+E.+Herringauthor=L.+I.+James&title=Degradation+of+Polycomb+Repressive+Complex+2+with+an+EED-Targeted+Bivalent+Chemical+Degrader&doi=10.1016%2Fj.chembiol.2019.11.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Degradation of Polycomb Repressive Complex 2 with an EED-Targeted Bivalent Chemical Degrader</span></div><div class="casAuthors">Potjewyd, Frances; Turner, Anne-Marie W.; Beri, Joshua; Rectenwald, Justin M.; Norris-Drouin, Jacqueline L.; Cholensky, Stephanie H.; Margolis, David M.; Pearce, Kenneth H.; Herring, Laura E.; James, Lindsey I.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">47-56.e15</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Protein degrdn. via the use of bivalent chem. degraders provides an alternative strategy to block protein function and assess the biol. roles of putative drug targets.  This approach capitalizes on the advantages of small-mol. inhibitors while moving beyond the restrictions of traditional pharmacol.  Here, we report a chem. degrader (UNC6852) that targets polycomb repressive complex 2 (PRC2).  UNC6852 contains an EED226-derived ligand and a ligand for VHL which bind to the WD40 arom. cage of EED and CRL2VHL, resp., to induce proteasomal degrdn. of PRC2 components, EED, EZH2, and SUZ12.  Degrdn. of PRC2 with UNC6852 blocks the histone methyltransferase activity of EZH2, decreasing H3K27me3 levels in HeLa cells and diffuse large B cell lymphoma (DLBCL) cells contg. EZH2 gain-of-function mutations.  UNC6852 degrades both wild-type and mutant EZH2, and addnl. displays anti-proliferative effects in this cancer model system.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqF3p-HFdD9KLVg90H21EOLACvtfcHk0ljT7ucU1SVF0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitlOmt7bM&md5=7d9ca2726bad4e40003310951edc09d4</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2019.11.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2019.11.006%26sid%3Dliteratum%253Aachs%26aulast%3DPotjewyd%26aufirst%3DF.%26aulast%3DTurner%26aufirst%3DA.%2BW.%26aulast%3DBeri%26aufirst%3DJ.%26aulast%3DRectenwald%26aufirst%3DJ.%2BM.%26aulast%3DNorris-Drouin%26aufirst%3DJ.%2BL.%26aulast%3DCholensky%26aufirst%3DS.%2BH.%26aulast%3DMargolis%26aufirst%3DD.%2BM.%26aulast%3DPearce%26aufirst%3DK.%2BH.%26aulast%3DHerring%26aufirst%3DL.%2BE.%26aulast%3DJames%26aufirst%3DL.%2BI.%26atitle%3DDegradation%2520of%2520Polycomb%2520Repressive%2520Complex%25202%2520with%2520an%2520EED-Targeted%2520Bivalent%2520Chemical%2520Degrader%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2020%26volume%3D27%26spage%3D47%26epage%3D56.e15%26doi%3D10.1016%2Fj.chembiol.2019.11.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hsu, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rasmusson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pachl, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawatkar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DH, O. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bagal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Code, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rawlins, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Argyrou, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomlinson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiarparin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacques, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woods, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bednarski, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drew, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castaldi, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fawell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bloecher, A.</span></span> <span> </span><span class="NLM_article-title">EED-Targeted PROTACs Degrade EED, EZH2, and SUZ12 in the PRC2 Complex</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">41</span>– <span class="NLM_lpage">46.e17</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2019.11.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1016%2Fj.chembiol.2019.11.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=31786184" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC1MXit1KqsrnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2020&pages=41-46.e17&author=J.+H.+Hsuauthor=T.+Rasmussonauthor=J.+Robinsonauthor=F.+Pachlauthor=J.+Readauthor=S.+Kawatkarauthor=O.+D.+DHauthor=S.+Bagalauthor=E.+Codeauthor=P.+Rawlinsauthor=A.+Argyrouauthor=R.+Tomlinsonauthor=N.+Gaoauthor=X.+Zhuauthor=E.+Chiarparinauthor=K.+Jacquesauthor=M.+Shenauthor=H.+Woodsauthor=E.+Bednarskiauthor=D.+M.+Wilsonauthor=L.+Drewauthor=M.+P.+Castaldiauthor=S.+Fawellauthor=A.+Bloecher&title=EED-Targeted+PROTACs+Degrade+EED%2C+EZH2%2C+and+SUZ12+in+the+PRC2+Complex&doi=10.1016%2Fj.chembiol.2019.11.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">EED-Targeted PROTACs Degrade EED, EZH2, and SUZ12 in the PRC2 Complex</span></div><div class="casAuthors">Hsu, Jessie Hao-Ru; Rasmusson, Timothy; Robinson, James; Pachl, Fiona; Read, Jon; Kawatkar, Sameer; O' Donovan, Daniel H.; Bagal, Sharan; Code, Erin; Rawlins, Philip; Argyrou, Argyrides; Tomlinson, Ronald; Gao, Ning; Zhu, Xiahui; Chiarparin, Elisabetta; Jacques, Kelly; Shen, Minhui; Woods, Haley; Bednarski, Emma; Wilson, David M.; Drew, Lisa; Castaldi, M. Paola; Fawell, Stephen; Bloecher, Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">41-46.e17</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Deregulation of the PRC2 complex, comprised of the core subunits EZH2, SUZ12, and EED, drives aberrant hypermethylation of H3K27 and tumorigenicity of many cancers.  Although inhibitors of EZH2 have shown promising clin. activity, preclin. data suggest that resistance can be acquired through secondary mutations in EZH2 that abrogate drug target engagement.  To address these limitations, we have designed several hetero-bifunctional PROTACs (proteolysis-targeting chimera) to efficiently target EED for elimination.  Our PROTACs bind to EED (pKD ~ 9.0) and promote ternary complex formation with the E3 ubiquitin ligase.  The PROTACs potently inhibit PRC2 enzyme activity (pIC50 ~ 8.1) and induce rapid degrdn. of not only EED but also EZH2 and SUZ12 within the PRC2 complex.  Furthermore, the PROTACs selectively inhibit proliferation of PRC2-dependent cancer cells (half maximal growth inhibition [GI50] = 49-58 nM).  In summary, our data demonstrate a therapeutic modality to target PRC2-dependent cancer through a PROTAC-mediated degrdn. mechanism.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4WGn283nfDLVg90H21EOLACvtfcHk0lgzbLeTWHxIzA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXit1KqsrnO&md5=791238d7d1423f4a715529cfa14bfb65</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2019.11.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2019.11.004%26sid%3Dliteratum%253Aachs%26aulast%3DHsu%26aufirst%3DJ.%2BH.%26aulast%3DRasmusson%26aufirst%3DT.%26aulast%3DRobinson%26aufirst%3DJ.%26aulast%3DPachl%26aufirst%3DF.%26aulast%3DRead%26aufirst%3DJ.%26aulast%3DKawatkar%26aufirst%3DS.%26aulast%3DDH%26aufirst%3DO.%2BD.%26aulast%3DBagal%26aufirst%3DS.%26aulast%3DCode%26aufirst%3DE.%26aulast%3DRawlins%26aufirst%3DP.%26aulast%3DArgyrou%26aufirst%3DA.%26aulast%3DTomlinson%26aufirst%3DR.%26aulast%3DGao%26aufirst%3DN.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DChiarparin%26aufirst%3DE.%26aulast%3DJacques%26aufirst%3DK.%26aulast%3DShen%26aufirst%3DM.%26aulast%3DWoods%26aufirst%3DH.%26aulast%3DBednarski%26aufirst%3DE.%26aulast%3DWilson%26aufirst%3DD.%2BM.%26aulast%3DDrew%26aufirst%3DL.%26aulast%3DCastaldi%26aufirst%3DM.%2BP.%26aulast%3DFawell%26aufirst%3DS.%26aulast%3DBloecher%26aufirst%3DA.%26atitle%3DEED-Targeted%2520PROTACs%2520Degrade%2520EED%252C%2520EZH2%252C%2520and%2520SUZ12%2520in%2520the%2520PRC2%2520Complex%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2020%26volume%3D27%26spage%3D41%26epage%3D46.e17%26doi%3D10.1016%2Fj.chembiol.2019.11.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maniaci, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Testa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamont, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rocha, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alessi, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romeo, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Homo-PROTACs: bivalent small-molecule dimerizers of the VHL E3 ubiquitin ligase to induce self-degradation</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">830</span> <span class="refDoi"> DOI: 10.1038/s41467-017-00954-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1038%2Fs41467-017-00954-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=29018234" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A280%3ADC%252BC1M%252Fptlaiuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&author=C.+Maniaciauthor=S.+J.+Hughesauthor=A.+Testaauthor=W.+Chenauthor=D.+J.+Lamontauthor=S.+Rochaauthor=D.+R.+Alessiauthor=R.+Romeoauthor=A.+Ciulli&title=Homo-PROTACs%3A+bivalent+small-molecule+dimerizers+of+the+VHL+E3+ubiquitin+ligase+to+induce+self-degradation&doi=10.1038%2Fs41467-017-00954-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Homo-PROTACs: bivalent small-molecule dimerizers of the VHL E3 ubiquitin ligase to induce self-degradation</span></div><div class="casAuthors">Maniaci Chiara; Hughes Scott J; Testa Andrea; Chen Wenzhang; Lamont Douglas J; Ciulli Alessio; Maniaci Chiara; Alessi Dario R; Rocha Sonia; Romeo Roberto</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">830</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">E3 ubiquitin ligases are key enzymes within the ubiquitin proteasome system which catalyze the ubiquitination of proteins, targeting them for proteasomal degradation.  E3 ligases are gaining importance as targets to small molecules, both for direct inhibition and to be hijacked to induce the degradation of non-native neo-substrates using bivalent compounds known as PROTACs (for 'proteolysis-targeting chimeras').  We describe Homo-PROTACs as an approach to dimerize an E3 ligase to trigger its suicide-type chemical knockdown inside cells.  We provide proof-of-concept of Homo-PROTACs using diverse molecules composed of two instances of a ligand for the von Hippel-Lindau (VHL) E3 ligase.  The most active compound, CM11, dimerizes VHL with high avidity in vitro and induces potent, rapid and proteasome-dependent self-degradation of VHL in different cell lines, in a highly isoform-selective fashion and without triggering a hypoxic response.  This approach offers a novel chemical probe for selective VHL knockdown, and demonstrates the potential for a new modality of chemical intervention on E3 ligases.Targeting the ubiquitin proteasome system to modulate protein homeostasis using small molecules has promising therapeutic potential.  Here the authors describe Homo-PROTACS: small molecules that can induce the homo-dimerization of E3 ubiquitin ligases and cause their proteasome-dependent degradation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRVJKELZUTFBpPq5LRczm8ffW6udTcc2eaZq5cOoNNx4rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M%252Fptlaiuw%253D%253D&md5=0ed79d9ff14eb7b6132a152f0413c4ef</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1038%2Fs41467-017-00954-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-017-00954-1%26sid%3Dliteratum%253Aachs%26aulast%3DManiaci%26aufirst%3DC.%26aulast%3DHughes%26aufirst%3DS.%2BJ.%26aulast%3DTesta%26aufirst%3DA.%26aulast%3DChen%26aufirst%3DW.%26aulast%3DLamont%26aufirst%3DD.%2BJ.%26aulast%3DRocha%26aufirst%3DS.%26aulast%3DAlessi%26aufirst%3DD.%2BR.%26aulast%3DRomeo%26aufirst%3DR.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DHomo-PROTACs%253A%2520bivalent%2520small-molecule%2520dimerizers%2520of%2520the%2520VHL%2520E3%2520ubiquitin%2520ligase%2520to%2520induce%2520self-degradation%26jtitle%3DNat.%2520Commun.%26date%3D2017%26volume%3D8%26doi%3D10.1038%2Fs41467-017-00954-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Knutson, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warholic, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wigle, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klaus, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allain, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raimondi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter Scott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chesworth, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moyer, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richon, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollock, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuntz, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keilhack, H.</span></span> <span> </span><span class="NLM_article-title">Durable tumor regression in genetically altered malignant rhabdoid tumors by inhibition of methyltransferase EZH2</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">7922</span>– <span class="NLM_lpage">7927</span>, <span class="refDoi"> DOI: 10.1073/pnas.1303800110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1073%2Fpnas.1303800110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=23620515" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC3sXptFGrurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=7922-7927&author=S.+K.+Knutsonauthor=N.+M.+Warholicauthor=T.+J.+Wigleauthor=C.+R.+Klausauthor=C.+J.+Allainauthor=A.+Raimondiauthor=M.+Porter+Scottauthor=R.+Chesworthauthor=M.+P.+Moyerauthor=R.+A.+Copelandauthor=V.+M.+Richonauthor=R.+M.+Pollockauthor=K.+W.+Kuntzauthor=H.+Keilhack&title=Durable+tumor+regression+in+genetically+altered+malignant+rhabdoid+tumors+by+inhibition+of+methyltransferase+EZH2&doi=10.1073%2Fpnas.1303800110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Durable tumor regression in genetically altered malignant rhabdoid tumors by inhibition of methyltransferase EZH2</span></div><div class="casAuthors">Knutson, Sarah K.; Warholic, Natalie M.; Wigle, Tim J.; Klaus, Christine R.; Allain, Christina J.; Raimondi, Alejandra; Scott, Margaret Porter; Chesworth, Richard; Moyer, Mikel P.; Copeland, Robert A.; Richon, Victoria M.; Pollock, Roy M.; Kuntz, Kevin W.; Keilhack, Heike</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">7922-7927, S7922/1-S7922/5</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Inactivation of the switch/sucrose nonfermentable complex component SMARCB1 is extremely prevalent in pediatric malignant rhabdoid tumors (MRTs) or atypical teratoid rhabdoid tumors.  This alteration is hypothesized to confer oncogenic dependency on EZH2 in these cancers.  The authors report the discovery of a potent, selective, and orally bioavailable small-mol. inhibitor of EZH2 enzymic activity, (N-((4,6-dimethyl-2-oxo-1,2-dihydropyridin-3-yl)methyl)-5-(ethyl(tetrahydro-2H-pyran-4-yl)amino)-4-methyl-4'-(morpholinomethyl)-[1,1'-biphenyl]-3-carboxamide).  The compd. induces apoptosis and differentiation specifically in SMARCB1-deleted MRT cells.  Treatment of xenograft-bearing mice with (N-((4,6-dimethyl-2-oxo-1,2-dihydropyridin-3-yl)methyl)-5-(ethyl(tetrahydro-2H-pyran-4-yl)amino)-4-methyl-4'-(morpholinomethyl)-[1,1'-biphenyl]-3-carboxamide) leads to dose-dependent regression of MRTs with correlative diminution of intratumoral trimethylation levels of lysine 27 on histone H3, and prevention of tumor regrowth after dosing cessation.  These data demonstrate the dependency of SMARCB1 mutant MRTs on EZH2 enzymic activity and portend the utility of EZH2-targeted drugs for the treatment of these genetically defined cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWRIJfPhwLA7Vg90H21EOLACvtfcHk0liFevoWDNIrqQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXptFGrurw%253D&md5=c2c6470d48a13ab8343db0a09465ba42</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1303800110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1303800110%26sid%3Dliteratum%253Aachs%26aulast%3DKnutson%26aufirst%3DS.%2BK.%26aulast%3DWarholic%26aufirst%3DN.%2BM.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DKlaus%26aufirst%3DC.%2BR.%26aulast%3DAllain%26aufirst%3DC.%2BJ.%26aulast%3DRaimondi%26aufirst%3DA.%26aulast%3DPorter%2BScott%26aufirst%3DM.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DRichon%26aufirst%3DV.%2BM.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26aulast%3DKuntz%26aufirst%3DK.%2BW.%26aulast%3DKeilhack%26aufirst%3DH.%26atitle%3DDurable%2520tumor%2520regression%2520in%2520genetically%2520altered%2520malignant%2520rhabdoid%2520tumors%2520by%2520inhibition%2520of%2520methyltransferase%2520EZH2%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2013%26volume%3D110%26spage%3D7922%26epage%3D7927%26doi%3D10.1073%2Fpnas.1303800110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Remillard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brien, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sonnett, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seo, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dastjerdi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wuhr, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhe-Paganon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armstrong, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Degradation of the BAF Complex Factor BRD9 by Heterobifunctional Ligands</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">5738</span>– <span class="NLM_lpage">5743</span>, <span class="refDoi"> DOI: 10.1002/anie.201611281</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1002%2Fanie.201611281" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmtFGrurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2017&pages=5738-5743&author=D.+Remillardauthor=D.+L.+Buckleyauthor=J.+Paulkauthor=G.+L.+Brienauthor=M.+Sonnettauthor=H.+S.+Seoauthor=S.+Dastjerdiauthor=M.+Wuhrauthor=S.+Dhe-Paganonauthor=S.+A.+Armstrongauthor=J.+E.+Bradner&title=Degradation+of+the+BAF+Complex+Factor+BRD9+by+Heterobifunctional+Ligands&doi=10.1002%2Fanie.201611281"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Degradation of the BAF Complex Factor BRD9 by Heterobifunctional Ligands</span></div><div class="casAuthors">Remillard, David; Buckley, Dennis L.; Paulk, Joshiawa; Brien, Gerard L.; Sonnett, Matthew; Seo, Hyuk-Soo; Dastierdi, Shiva; Wuehr, Martin; Dhe-Paganon, Sirano; Armstrong, Scott A.; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">5738-5743</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The bromodomain-contg. protein BRD9, a subunit of the human BAF (SWI/SNF) nucleosome remodeling complex, has emerged as an attractive therapeutic target in cancer.  Despite the development of chem. probes targeting the BRD9 bromodomain, there is a limited understanding of BRD9 function beyond acetyl-lysine recognition.  We have therefore created the first BRD9-directed chem. degraders, through iterative design and testing of heterobifunctional ligands that bridge the BRD9 bromodomain and the cereblon E3 ubiquitin ligase complex.  Degraders of BRD9 exhibit markedly enhanced potency compared to parental ligands (10- to 100-fold).  Parallel study of degraders with divergent BRD9-binding chemotypes in models of acute myeloid leukemia resolves bromodomain polypharmacol. in this emerging drug class.  Together, these findings reveal the tractability of non-BET bromodomain contg. proteins to chem. degrdn., and highlight lead compd. dBRD9 as a tool for the study of BRD9.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQ9FU2Ipaq27Vg90H21EOLACvtfcHk0liFevoWDNIrqQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmtFGrurk%253D&md5=2eb6562cc9d976c7ae6070c9a419e9ea</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1002%2Fanie.201611281&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201611281%26sid%3Dliteratum%253Aachs%26aulast%3DRemillard%26aufirst%3DD.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DBrien%26aufirst%3DG.%2BL.%26aulast%3DSonnett%26aufirst%3DM.%26aulast%3DSeo%26aufirst%3DH.%2BS.%26aulast%3DDastjerdi%26aufirst%3DS.%26aulast%3DWuhr%26aufirst%3DM.%26aulast%3DDhe-Paganon%26aufirst%3DS.%26aulast%3DArmstrong%26aufirst%3DS.%2BA.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DDegradation%2520of%2520the%2520BAF%2520Complex%2520Factor%2520BRD9%2520by%2520Heterobifunctional%2520Ligands%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2017%26volume%3D56%26spage%3D5738%26epage%3D5743%26doi%3D10.1002%2Fanie.201611281" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Salas, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Przybranowski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of a Small-Molecule Degrader of Bromodomain and Extra-Terminal (BET) Proteins with Picomolar Cellular Potencies and Capable of Achieving Tumor Regression</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">462</span>– <span class="NLM_lpage">481</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01816</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01816" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkvVajsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=462-481&author=B.+Zhouauthor=J.+Huauthor=F.+Xuauthor=Z.+Chenauthor=L.+Baiauthor=E.+Fernandez-Salasauthor=M.+Linauthor=L.+Liuauthor=C.+Y.+Yangauthor=Y.+Zhaoauthor=D.+McEachernauthor=S.+Przybranowskiauthor=B.+Wenauthor=D.+Sunauthor=S.+Wang&title=Discovery+of+a+Small-Molecule+Degrader+of+Bromodomain+and+Extra-Terminal+%28BET%29+Proteins+with+Picomolar+Cellular+Potencies+and+Capable+of+Achieving+Tumor+Regression&doi=10.1021%2Facs.jmedchem.6b01816"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Small-Molecule Degrader of Bromodomain and Extra-Terminal (BET) Proteins with Picomolar Cellular Potencies and Capable of Achieving Tumor Regression</span></div><div class="casAuthors">Zhou, Bing; Hu, Jiantao; Xu, Fuming; Chen, Zhuo; Bai, Longchuan; Fernandez-Salas, Ester; Lin, Mei; Liu, Liu; Yang, Chao-Yie; Zhao, Yujun; McEachern, Donna; Przybranowski, Sally; Wen, Bo; Sun, Duxin; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">462-481</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bromodomain and extra-terminal (BET) family proteins, consisting of BRD2, BRD3, BRD4, and testis-specific BRDT members, are epigenetic "readers" and play a key role in the regulation of gene transcription.  BET proteins are considered to be attractive therapeutic targets for cancer and other human diseases.  Recently, heterobifunctional small-mol. BET degraders have been designed based upon the proteolysis targeting chimera (PROTAC) concept to induce BET protein degrdn.  Herein, we present our design, synthesis, and evaluation of a new class of PROTAC BET degraders.  One of the most promising compds., 23, effectively degrades BRD4 protein at concns. as low as 30 pM in the RS4;11 leukemia cell line, achieves an IC50 value of 51 pM in inhibition of RS4;11 cell growth and induces rapid tumor regression in vivo against RS4;11 xenograft tumors.  These data establish that compd. 23 (BETd-260/ZBC260) is a highly potent and efficacious BET degrader.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqz2Lwp52x1rVg90H21EOLACvtfcHk0liFevoWDNIrqQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkvVajsb8%253D&md5=6b3a10ecd192a5812ef581dca1fdc82a</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01816&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01816%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DB.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DF.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DBai%26aufirst%3DL.%26aulast%3DFernandez-Salas%26aufirst%3DE.%26aulast%3DLin%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DC.%2BY.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DPrzybranowski%26aufirst%3DS.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520a%2520Small-Molecule%2520Degrader%2520of%2520Bromodomain%2520and%2520Extra-Terminal%2520%2528BET%2529%2520Proteins%2520with%2520Picomolar%2520Cellular%2520Potencies%2520and%2520Capable%2520of%2520Achieving%2520Tumor%2520Regression%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D462%26epage%3D481%26doi%3D10.1021%2Facs.jmedchem.6b01816" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zengerle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Selective Small Molecule Induced Degradation of the BET Bromodomain Protein BRD4</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1770</span>– <span class="NLM_lpage">1777</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00216</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00216" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC2MXptlOgu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=1770-1777&author=M.+Zengerleauthor=K.+H.+Chanauthor=A.+Ciulli&title=Selective+Small+Molecule+Induced+Degradation+of+the+BET+Bromodomain+Protein+BRD4&doi=10.1021%2Facschembio.5b00216"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Small Molecule Induced Degradation of the BET Bromodomain Protein BRD4</span></div><div class="casAuthors">Zengerle, Michael; Chan, Kwok-Ho; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1770-1777</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The Bromo- and Extra-Terminal (BET) proteins BRD2, BRD3, and BRD4 play important roles in transcriptional regulation, epigenetics, and cancer and are the targets of pan-BET selective bromodomain inhibitor JQ1.  However, the lack of intra-BET selectivity limits the scope of current inhibitors as probes for target validation and could lead to unwanted side effects or toxicity in a therapeutic setting.  We designed Proteolysis Targeted Chimeras (PROTACs) that tether JQ1 to a ligand for the E3 ubiquitin ligase VHL, aimed at triggering the intracellular destruction of BET proteins.  Compd. MZ1 potently and rapidly induces reversible, long-lasting, and unexpectedly selective removal of BRD4 over BRD2 and BRD3.  The activity of MZ1 is dependent on binding to VHL but is achieved at a sufficiently low concn. not to induce stabilization of HIF-1α.  Gene expression profiles of selected cancer-related genes responsive to JQ1 reveal distinct and more limited transcriptional responses induced by MZ1, consistent with selective suppression of BRD4.  Our discovery opens up new opportunities to elucidate the cellular phenotypes and therapeutic implications assocd. with selective targeting of BRD4.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEmSv7p9MZOrVg90H21EOLACvtfcHk0lhX8kgXqiUQnw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXptlOgu70%253D&md5=b7514fd274eff8e4bb6a771f5aaf9ab5</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00216&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00216%26sid%3Dliteratum%253Aachs%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DChan%26aufirst%3DK.%2BH.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DSelective%2520Small%2520Molecule%2520Induced%2520Degradation%2520of%2520the%2520BET%2520Bromodomain%2520Protein%2520BRD4%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D1770%26epage%3D1777%26doi%3D10.1021%2Facschembio.5b00216" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W.</span></span> <span> </span><span class="NLM_article-title">A covalently bound inhibitor triggers EZH2 degradation through CHIP-mediated ubiquitination</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">1243</span>– <span class="NLM_lpage">1260</span>, <span class="refDoi"> DOI: 10.15252/embj.201694058</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.15252%2Fembj.201694058" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=28320739" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC2sXksFSisr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2017&pages=1243-1260&author=X.+Wangauthor=W.+Caoauthor=J.+Zhangauthor=M.+Yanauthor=Q.+Xuauthor=X.+Wuauthor=L.+Wanauthor=Z.+Zhangauthor=C.+Zhangauthor=X.+Qinauthor=M.+Xiaoauthor=D.+Yeauthor=Y.+Liuauthor=Z.+Hanauthor=S.+Wangauthor=L.+Maoauthor=W.+Weiauthor=W.+Chen&title=A+covalently+bound+inhibitor+triggers+EZH2+degradation+through+CHIP-mediated+ubiquitination&doi=10.15252%2Fembj.201694058"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">A covalently bound inhibitor triggers EZH2 degradation through CHIP-mediated ubiquitination</span></div><div class="casAuthors">Wang, Xu; Cao, Wei; Zhang, Jianjun; Yan, Ming; Xu, Qin; Wu, Xiangbing; Wan, Lixin; Zhang, Zhiyuan; Zhang, Chenping; Qin, Xing; Xiao, Meng; Ye, Dongxia; Liu, Yuyang; Han, Zeguang; Wang, Shaomeng; Mao, Li; Wei, Wenyi; Chen, Wantao</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1243-1260</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Enhancer of zeste homolog 2 (EZH2) has been characterized as a crit. oncogene and a promising drug target in human malignant tumors.  The current EZH2 inhibitors strongly suppress the enhanced enzymic function of mutant EZH2 in some lymphomas.  However, the recent identification of a PRC2- and methyltransferase-independent role of EZH2 indicates that a complete suppression of all oncogenic functions of EZH2 is needed.  Here, we report a unique EZH2-targeting strategy by identifying a gambogenic acid (GNA) deriv. as a novel agent that specifically and covalently bound to Cys668 within the EZH2-SET domain, triggering EZH2 degrdn. through COOH terminus of Hsp70-interacting protein (CHIP)-mediated ubiquitination.  This class of inhibitors significantly suppressed H3K27Me3 and effectively reactivated polycomb repressor complex 2 (PRC2)-silenced tumor suppressor genes.  Moreover, the novel inhibitors significantly suppressed tumor growth in an EZH2-dependent manner, and tumors bearing a non-GNA-interacting C668S-EZH2 mutation exhibited resistance to the inhibitors.  Together, our results identify the inhibition of the signaling pathway that governs GNA-mediated destruction of EZH2 as a promising anti-cancer strategy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRQknSdWy5T7Vg90H21EOLACvtfcHk0lhX8kgXqiUQnw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXksFSisr0%253D&md5=b9b8a266537e2922648784d9cd4edf02</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.15252%2Fembj.201694058&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.15252%252Fembj.201694058%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DX.%26aulast%3DCao%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DYan%26aufirst%3DM.%26aulast%3DXu%26aufirst%3DQ.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DWan%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DQin%26aufirst%3DX.%26aulast%3DXiao%26aufirst%3DM.%26aulast%3DYe%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DHan%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DMao%26aufirst%3DL.%26aulast%3DWei%26aufirst%3DW.%26aulast%3DChen%26aufirst%3DW.%26atitle%3DA%2520covalently%2520bound%2520inhibitor%2520triggers%2520EZH2%2520degradation%2520through%2520CHIP-mediated%2520ubiquitination%26jtitle%3DEMBO%2520J.%26date%3D2017%26volume%3D36%26spage%3D1243%26epage%3D1260%26doi%3D10.15252%2Fembj.201694058" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kasibhatla, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jessen, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maliartchouk, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">English, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewe, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Archer, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ponce, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sirisoma, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gehlsen, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, S. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tseng, B.</span></span> <span> </span><span class="NLM_article-title">A role for transferrin receptor in triggering apoptosis when targeted with gambogic acid</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>102</i></span>,  <span class="NLM_fpage">12095</span>– <span class="NLM_lpage">12100</span>, <span class="refDoi"> DOI: 10.1073/pnas.0406731102</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1073%2Fpnas.0406731102" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=16103367" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BD2MXps1yqsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2005&pages=12095-12100&author=S.+Kasibhatlaauthor=K.+A.+Jessenauthor=S.+Maliartchoukauthor=J.+Y.+Wangauthor=N.+M.+Englishauthor=J.+Dreweauthor=L.+Qiuauthor=S.+P.+Archerauthor=A.+E.+Ponceauthor=N.+Sirisomaauthor=S.+Jiangauthor=H.+Z.+Zhangauthor=K.+R.+Gehlsenauthor=S.+X.+Caiauthor=D.+R.+Greenauthor=B.+Tseng&title=A+role+for+transferrin+receptor+in+triggering+apoptosis+when+targeted+with+gambogic+acid&doi=10.1073%2Fpnas.0406731102"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">A role for transferrin receptor in triggering apoptosis when targeted with gambogic acid</span></div><div class="casAuthors">Kasibhatla, Shailaja; Jessen, Katayoun A.; Maliartchouk, Sergei; Wang, Jean Yu; English, Nicole M.; Drewe, John; Qiu, Ling; Archer, Shannon P.; Ponce, Anthony E.; Sirisoma, Nilantha; Jiang, Songchun; Zhang, Han-Zhong; Gehlsen, Kurt R.; Cai, Sui Xiong; Green, Douglas R.; Tseng, Ben</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">12095-12100</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Transferrin receptor (TfR) has been shown to be significantly overexpressed in different types of cancers.  We discovered TfR as a target for gambogic acid (GA), used in traditional Chinese medicine and a previously undiscovered link between TfR and the rapid activation of apoptosis.  The binding site of GA on TfR is independent of the transferrin binding site, and it appears that GA potentially inhibits TfR internalization.  Down-regulation of TfR by RNA interference decreases sensitivity to GA-induced apoptosis, further supporting TfR as the primary GA receptor.  In summary, GA binding to TfR induces a unique signal leading to rapid apoptosis of tumor cells.  These results suggest that GA may provide an addnl. approach for targeting the TfR and its use in cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPax6NHbKWnLVg90H21EOLACvtfcHk0lg_awr95Din6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXps1yqsr4%253D&md5=f6cf82bfa7f841ca71046457dd7074c0</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0406731102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0406731102%26sid%3Dliteratum%253Aachs%26aulast%3DKasibhatla%26aufirst%3DS.%26aulast%3DJessen%26aufirst%3DK.%2BA.%26aulast%3DMaliartchouk%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DJ.%2BY.%26aulast%3DEnglish%26aufirst%3DN.%2BM.%26aulast%3DDrewe%26aufirst%3DJ.%26aulast%3DQiu%26aufirst%3DL.%26aulast%3DArcher%26aufirst%3DS.%2BP.%26aulast%3DPonce%26aufirst%3DA.%2BE.%26aulast%3DSirisoma%26aufirst%3DN.%26aulast%3DJiang%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DH.%2BZ.%26aulast%3DGehlsen%26aufirst%3DK.%2BR.%26aulast%3DCai%26aufirst%3DS.%2BX.%26aulast%3DGreen%26aufirst%3DD.%2BR.%26aulast%3DTseng%26aufirst%3DB.%26atitle%3DA%2520role%2520for%2520transferrin%2520receptor%2520in%2520triggering%2520apoptosis%2520when%2520targeted%2520with%2520gambogic%2520acid%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2005%26volume%3D102%26spage%3D12095%26epage%3D12100%26doi%3D10.1073%2Fpnas.0406731102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ma, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stratikopoulos, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwarz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leshchenko, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rialdi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dale, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lagana, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guccione, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parekh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parsons, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of a first-in-class EZH2 selective degrader</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">214</span>– <span class="NLM_lpage">222</span>, <span class="refDoi"> DOI: 10.1038/s41589-019-0421-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1038%2Fs41589-019-0421-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=31819273" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitlCht7zJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2020&pages=214-222&author=A.+Maauthor=E.+Stratikopoulosauthor=K.+S.+Parkauthor=J.+Weiauthor=T.+C.+Martinauthor=X.+Yangauthor=M.+Schwarzauthor=V.+Leshchenkoauthor=A.+Rialdiauthor=B.+Daleauthor=A.+Laganaauthor=E.+Guccioneauthor=S.+Parekhauthor=R.+Parsonsauthor=J.+Jin&title=Discovery+of+a+first-in-class+EZH2+selective+degrader&doi=10.1038%2Fs41589-019-0421-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a first-in-class EZH2 selective degrader</span></div><div class="casAuthors">Ma, Anqi; Stratikopoulos, Elias; Park, Kwang-Su; Wei, Jieli; Martin, Tiphaine C.; Yang, Xiaobao; Schwarz, Megan; Leshchenko, Violetta; Rialdi, Alexander; Dale, Brandon; Lagana, Alessandro; Guccione, Ernesto; Parekh, Samir; Parsons, Ramon; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">214-222</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The enhancer of zeste homolog 2 (EZH2) is the main enzymic subunit of the PRC2 complex, which catalyzes trimethylation of histone H3 lysine 27 (H3K27me3) to promote transcriptional silencing.  EZH2 is overexpressed in multiple types of cancer including triple-neg. breast cancer (TNBC), and high expression levels correlate with poor prognosis.  Several EZH2 inhibitors, which inhibit the methyltransferase activity of EZH2, have shown promise in treating sarcoma and follicular lymphoma in clinics.  However, EZH2 inhibitors are ineffective at blocking proliferation of TNBC cells, even though they effectively reduce the H3K27me3 mark.  Using a hydrophobic tagging approach, we generated MS1943, a first-in-class EZH2 selective degrader that effectively reduces EZH2 levels in cells.  Importantly, MS1943 has a profound cytotoxic effect in multiple TNBC cells, while sparing normal cells, and is efficacious in vivo, suggesting that pharmacol. degrdn. of EZH2 can be advantageous for treating the cancers that are dependent on EZH2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPb6bK3v_gQrVg90H21EOLACvtfcHk0lg_awr95Din6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitlCht7zJ&md5=e1ccae457eba9b298f8b0e583bbaa833</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1038%2Fs41589-019-0421-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-019-0421-4%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DA.%26aulast%3DStratikopoulos%26aufirst%3DE.%26aulast%3DPark%26aufirst%3DK.%2BS.%26aulast%3DWei%26aufirst%3DJ.%26aulast%3DMartin%26aufirst%3DT.%2BC.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DSchwarz%26aufirst%3DM.%26aulast%3DLeshchenko%26aufirst%3DV.%26aulast%3DRialdi%26aufirst%3DA.%26aulast%3DDale%26aufirst%3DB.%26aulast%3DLagana%26aufirst%3DA.%26aulast%3DGuccione%26aufirst%3DE.%26aulast%3DParekh%26aufirst%3DS.%26aulast%3DParsons%26aufirst%3DR.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520a%2520first-in-class%2520EZH2%2520selective%2520degrader%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2020%26volume%3D16%26spage%3D214%26epage%3D222%26doi%3D10.1038%2Fs41589-019-0421-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobrovolsky, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weisberg, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doctor, Z. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ko, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffin, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treon, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">A Chemoproteomic Approach to Query the Degradable Kinome Using a Multi-kinase Degrader</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">88</span>– <span class="NLM_lpage">99.e6</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.10.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1016%2Fj.chembiol.2017.10.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=29129717" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=88-99.e6&author=H.+T.+Huangauthor=D.+Dobrovolskyauthor=J.+Paulkauthor=G.+Yangauthor=E.+L.+Weisbergauthor=Z.+M.+Doctorauthor=D.+L.+Buckleyauthor=J.+H.+Choauthor=E.+Koauthor=J.+Jangauthor=K.+Shiauthor=H.+G.+Choiauthor=J.+D.+Griffinauthor=Y.+Liauthor=S.+P.+Treonauthor=E.+S.+Fischerauthor=J.+E.+Bradnerauthor=L.+Tanauthor=N.+S.+Gray&title=A+Chemoproteomic+Approach+to+Query+the+Degradable+Kinome+Using+a+Multi-kinase+Degrader&doi=10.1016%2Fj.chembiol.2017.10.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">A Chemoproteomic Approach to Query the Degradable Kinome Using a Multi-kinase Degrader</span></div><div class="casAuthors">Huang, Hai-Tsang; Dobrovolsky, Dennis; Paulk, Joshiawa; Yang, Guang; Weisberg, Ellen L.; Doctor, Zainab M.; Buckley, Dennis L.; Cho, Joong-Heui; Ko, Eunhwa; Jang, Jaebong; Shi, Kun; Choi, Hwan Geun; Griffin, James D.; Li, Ying; Treon, Steven P.; Fischer, Eric S.; Bradner, James E.; Tan, Li; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">88-99.e6</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Heterobifunctional mols. that recruit E3 ubiquitin ligases, such as cereblon, for targeted protein degrdn. represent an emerging pharmacol. strategy.  A major unanswered question is how generally applicable this strategy is to all protein targets.  In this study, we designed a multi-kinase degrader by conjugating a highly promiscuous kinase inhibitor with a cereblon-binding ligand, and used quant. proteomics to discover 28 kinases, including BTK, PTK2, PTK2B, FLT3, AURKA, AURKB, TEC, ULK1, ITK, and nine members of the CDK family, as degradable.  This set of kinases is only a fraction of the intracellular targets bound by the degrader, demonstrating that successful degrdn. requires more than target engagement.  The results guided us to develop selective degraders for FLT3 and BTK, with potentials to improve disease treatment.  Together, this study demonstrates an efficient approach to triage a gene family of interest to identify readily degradable targets for further studies and pre-clin. developments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYNxhoPV3127Vg90H21EOLACvtfcHk0lhmPHkDE_bt2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjO&md5=703cfbaca3edd24176472158563f895f</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.10.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.10.005%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DH.%2BT.%26aulast%3DDobrovolsky%26aufirst%3DD.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DG.%26aulast%3DWeisberg%26aufirst%3DE.%2BL.%26aulast%3DDoctor%26aufirst%3DZ.%2BM.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DCho%26aufirst%3DJ.%2BH.%26aulast%3DKo%26aufirst%3DE.%26aulast%3DJang%26aufirst%3DJ.%26aulast%3DShi%26aufirst%3DK.%26aulast%3DChoi%26aufirst%3DH.%2BG.%26aulast%3DGriffin%26aufirst%3DJ.%2BD.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DTreon%26aufirst%3DS.%2BP.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DTan%26aufirst%3DL.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DA%2520Chemoproteomic%2520Approach%2520to%2520Query%2520the%2520Degradable%2520Kinome%2520Using%2520a%2520Multi-kinase%2520Degrader%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2018%26volume%3D25%26spage%3D88%26epage%3D99.e6%26doi%3D10.1016%2Fj.chembiol.2017.10.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, L.</span></span> <span> </span><span class="NLM_article-title">Design and Synthesis of EZH2-Based PROTACs to Degrade the PRC2 Complex for Targeting the Noncatalytic Activity of EZH2</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>64</i></span>,  <span class="NLM_fpage">2829</span>– <span class="NLM_lpage">2848</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c02234</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c02234" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BB3MXksV2hsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2021&pages=2829-2848&author=Z.+Liuauthor=X.+Huauthor=Q.+Wangauthor=X.+Wuauthor=Q.+Zhangauthor=W.+Weiauthor=X.+Suauthor=H.+Heauthor=S.+Zhouauthor=R.+Huauthor=T.+Yeauthor=Y.+Zhuauthor=N.+Wangauthor=L.+Yu&title=Design+and+Synthesis+of+EZH2-Based+PROTACs+to+Degrade+the+PRC2+Complex+for+Targeting+the+Noncatalytic+Activity+of+EZH2&doi=10.1021%2Facs.jmedchem.0c02234"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Synthesis of EZH2-Based PROTACs to Degrade the Complex for Targeting the Noncatalytic Activity of EZH2</span></div><div class="casAuthors">Liu, Zhihao; Hu, Xi; Wang, Qiwei; Wu, Xiuli; Zhang, Qiangsheng; Wei, Wei; Su, Xingping; He, Hualong; Zhou, Shuyan; Hu, Rong; Ye, Tinghong; Zhu, Yongxia; Wang, Ningyu; Yu, Luoting</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2829-2848</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">EZH2 mediates both PRC2-dependent gene silencing via catalyzing H3K27me3 and PRC2-independent transcriptional activation in various cancers.  Given its oncogenic role in cancers, EZH2 has constituted a compelling target for anticancer therapy.  However, current EZH2 inhibitors only target its methyltransferase activity to downregulate H3K27me3 levels and show limited efficacy because of inadequate suppression of the EZH2 oncogenic activity.  Therefore, therapeutic strategies to completely block the oncogenic activity of EZH2 are urgently needed.  Herein, we report a series of EZH2-targeted proteolysis targeting chimeras (PROTACs) that induce proteasomal degrdn. of PRC2 components, including EZH2, EED, SUZ12, and RbAp48.  Preliminary assessment identified E7 as the most active PROTAC mol., which decreased PRC2 subunits and H3K27me2/3 levels in various cancer cells.  Furthermore, E7 strongly inhibited transcriptional silencing mediated by EZH2 dependent on PRC2 and transcriptional activation mediated by EZH2 independent of PRC2, showing significant antiproliferative activities against cancer cell lines dependent on the enzymic and nonenzymic activities of EZH2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBGvGI0J8IlbVg90H21EOLACvtfcHk0lhmPHkDE_bt2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXksV2hsrs%253D&md5=6be23721044612d4de7d1446af35f698</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c02234&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c02234%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DHu%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DWei%26aufirst%3DW.%26aulast%3DSu%26aufirst%3DX.%26aulast%3DHe%26aufirst%3DH.%26aulast%3DZhou%26aufirst%3DS.%26aulast%3DHu%26aufirst%3DR.%26aulast%3DYe%26aufirst%3DT.%26aulast%3DZhu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DN.%26aulast%3DYu%26aufirst%3DL.%26atitle%3DDesign%2520and%2520Synthesis%2520of%2520EZH2-Based%2520PROTACs%2520to%2520Degrade%2520the%2520PRC2%2520Complex%2520for%2520Targeting%2520the%2520Noncatalytic%2520Activity%2520of%2520EZH2%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2021%26volume%3D64%26spage%3D2829%26epage%3D2848%26doi%3D10.1021%2Facs.jmedchem.0c02234" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glorieux, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, P.</span></span> <span> </span><span class="NLM_article-title">Chemotherapy induces tumor immune evasion by upregulation of programmed cell death ligand 1 expression in bone marrow stromal cells</span>. <i>Mol. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">358</span>– <span class="NLM_lpage">372</span>, <span class="refDoi"> DOI: 10.1002/1878-0261.12032</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=10.1002%2F1878-0261.12032" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=28218497" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;key=1%3ACAS%3A528%3ADC%252BC2sXnsV2jsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2017&pages=358-372&author=M.+Yangauthor=P.+Liuauthor=K.+Wangauthor=C.+Glorieuxauthor=Y.+Huauthor=S.+Wenauthor=W.+Jiangauthor=P.+Huang&title=Chemotherapy+induces+tumor+immune+evasion+by+upregulation+of+programmed+cell+death+ligand+1+expression+in+bone+marrow+stromal+cells&doi=10.1002%2F1878-0261.12032"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Chemotherapy induces tumor immune evasion by upregulation of programmed cell death ligand 1 expression in bone marrow stromal cells</span></div><div class="casAuthors">Yang, Mengqi; Liu, Panpan; Wang, Kefeng; Glorieux, Christophe; Hu, Yumin; Wen, Shijun; Jiang, Wenqi; Huang, Peng</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Oncology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">358-372</span>CODEN:
                <span class="NLM_cas:coden">MOONC3</span>;
        ISSN:<span class="NLM_cas:issn">1878-0261</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons Ltd.</span>)
        </div><div class="casAbstract">Programmed cell death ligand 1 (PD-L1) is a neg. regulator of the immune response that enables tumor cells to escape T-cell immunity.  Although PD-L1 expression in cancer cells has been extensively studied, the expression of PD-L1 in stromal cells and its clin. significance remain largely unknown.  Here, we show that bone marrow stromal cells express a low level of PD-L1 and that this mol. is significantly upregulated by key drugs used in the treatment of lymphoma at clin. relevant concns.  Mechanistically, chemotherapeutic drugs induce PD-L1 expression in stromal cells through upregulation of granulocyte macrophage colony-stimulating factor and activation of the extracellular signal-regulated kinase (ERK) 1/2 signaling pathway.  Suppression of ERK by a chem. inhibitor or genetic silencing of ERK2 expression prevents drug-induced PD-L1 expression.  PD-L1 expression is upregulated in the bone marrow stromal cells of mice treated with doxorubicin and in drug-treated bone marrow specimens from lymphoma patients.  Drug-induced PD-L1 expression in stromal cells can cause significant impairment of T-cell functions.  Overall, our study reveals a previously unrecognized mechanism by which chemotherapy induces tumor immune evasion by upregulation of PD-L1 in bone marrow stromal cells, and provides new evidence for the combination of chemotherapy and anti-PD-L1/PD-1 as an effective strategy for treatment of lymphoma and other cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqe6PgoFUeWp7Vg90H21EOLACvtfcHk0lhmPHkDE_bt2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXnsV2jsrg%253D&md5=d56540bfdcd424ee43174ef3c940a061</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1002%2F1878-0261.12032&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F1878-0261.12032%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DP.%26aulast%3DWang%26aufirst%3DK.%26aulast%3DGlorieux%26aufirst%3DC.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DWen%26aufirst%3DS.%26aulast%3DJiang%26aufirst%3DW.%26aulast%3DHuang%26aufirst%3DP.%26atitle%3DChemotherapy%2520induces%2520tumor%2520immune%2520evasion%2520by%2520upregulation%2520of%2520programmed%2520cell%2520death%2520ligand%25201%2520expression%2520in%2520bone%2520marrow%2520stromal%2520cells%26jtitle%3DMol.%2520Oncol.%26date%3D2017%26volume%3D11%26spage%3D358%26epage%3D372%26doi%3D10.1002%2F1878-0261.12032" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LS6" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LS6','PDB','5LS6'); return false;">PDB: 5LS6</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-2"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i52"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00460">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_62830"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.1c00460?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00460</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Western blot analysis of EZH2, EZH1, and H3K27me3 levels affected by tested compounds; MTS cell viability analysis in different lymphoma cell lines treated with YM281; the structure of YM620 and its effects on EZH2 and cell proliferation; flow cytometry analysis of the cell cycle and cell apoptosis; Western blot analysis of PARP, Casp-3, and C-casp-3 levels; the body weights of the tested compound treated mice; nonsensitive cancer cell lines NIC-H460 and AsPC1 treated by the tested compounds (Figures S1–S6); patient sample information and corresponding analytical methods; apparent permeability of EPZ6438, YM181, and YM281 (Tables S1–S2); NMR spectra of V1–V7, YM620, G1–G6, and key intermediates; HRMS spectra and HPLC results of YM181 and YM281 (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00460/suppl_file/jm1c00460_si_001.pdf">PDF</a>)</p></li><li><p class="inline">The docking session for EPZ6438 in EZH2 crystal <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LS6">5LS6</a> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00460/suppl_file/jm1c00460_si_002.pdb">PDB</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00460/suppl_file/jm1c00460_si_003.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00460/suppl_file/jm1c00460_si_001.pdf">jm1c00460_si_001.pdf (4.14 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00460/suppl_file/jm1c00460_si_002.pdb">jm1c00460_si_002.pdb (1.37 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00460/suppl_file/jm1c00460_si_003.csv">jm1c00460_si_003.csv (3.11 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.1c00460&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.1c00460%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2021.64.issue-14" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.1c00460" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                5MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799b1cbeb9f3c40","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
